<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                          RESEARCH ARTICLE <br />  <br />  <br />                                                           Polarity-Dependent Asymmetric Distribution <br />                                                           MEX-5/6&#226;&#8364;&#8220;Mediated Translational <br />                                                           Activation Era-1 mRNA C. elegans <br />                                                           Embryos <br />                                                           Zolt&#195;&#161;n Spir&#195;&#179;, Pierre G&#195;&#182;nczy* <br />                                                           Swiss Institute Experimental Cancer Research (ISREC), School Life Sciences, Swiss Federal Institute <br />                                                           Technology (EPFL) Lausanne, Lausanne, Switzerland <br />  <br />                                                           * Pierre.Gonczy@epfl.ch <br />  <br />  <br />  <br />  <br />                                                           <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                                           The early C. elegans embryo attractive model investigate fundamental de- <br />                                                           velopmental processes. With exception mex-3 mRNA, maternally contributed <br />     OPEN ACCESS                                           mRNAs thought distributed uniformly one-cell embryo. Here, report and <br />                                                           characterize striking distribution mRNA encoding novel protein ERA-1. We <br /> Citation: Spir&#195;&#179; Z, G&#195;&#182;nczy P (2015) Polarity- <br /> Dependent Asymmetric Distribution MEX-5/6&#226;&#8364;&#8220;            era-1 mRNA enriched anterior one-cell embryo present solely <br /> Mediated Translational Activation Era-1 mRNA       anterior blastomeres thereafter. Although era-1 essential gene, uncovered <br /> in C. elegans Embryos. PLoS ONE 10(3): e0120984.          era-1 null mutant embryos sensitive slight impairment embryonic polarity. We <br /> doi:10.1371/journal.pone.0120984 <br />                                                           asymmetric distribution era-1 mRNA depends anterior-posterior polarity <br /> Academic Editor: Myon-Hee Lee, East Carolina              cues era-1 3&#226;&#8364;&#8482;UTR. Similarly era-1 mRNA, YFP-ERA-1 protein en- <br /> University, UNITED STATES <br />                                                           riched anterior blastomeres. Interestingly, RNA-binding protein MEX-5 <br /> Received: October 9, 2014                                 required era-1 mRNA asymmetry. Furthermore, MEX-5, its <br /> Accepted: February 9, 2015                                partially redundant partner MEX-6, needed activate era-1 mRNA translation anteri- <br /> Published: March 30, 2015                                 blastomeres. These findings lead propose MEX-5/6&#226;&#8364;&#8220;mediated regulation of <br />                                                           era-1 mRNA contributes robust embryonic development. <br /> Copyright: &#194;&#169; 2015 Spir&#195;&#179;, G&#195;&#182;nczy. This open <br /> access article distributed terms the <br /> Creative Commons Attribution License, permits <br /> unrestricted use, distribution, reproduction   Introduction <br /> medium, provided original author source are <br /> credited.                                                 Before zygotic transcription begins, restricted distribution maternally provided mRNAs and <br />                                                           translational regulation widely used mechanisms orchestrate early developmental <br /> Data Availability Statement: All relevant data are <br /> within paper Supporting Information files.    processes, including one-cell embryo (hereafter referred zygote) (reviewed in <br />                                                           [1&#226;&#8364;&#8220;3]). Compared metazoan organisms issue investigated, C. <br /> Funding: The work supported grant <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                           elegans stands apart mex-3 [4], mRNA restricted distribution the <br /> the Swiss National Science Foundation (grant <br /> number: 3100A0-122500/1, website: http://www.snf. <br />                                                           zygote reported date. <br /> ch/en/Pages/default.aspx). The funders role        The C. elegans zygote polarized shortly fertilization anterior-poste- <br /> study design, data collection analysis, decision   rior (A-P) embryonic axis action PAR proteins associated components, <br /> publish, preparation manuscript.                localize anterior (PAR-6, PAR-3, PKC-3) posterior (PAR-2, PAR-1) cell cortex <br /> Competing Interests: The authors declared            (reviewed [5]). Further A-P asymmetries generated result, including anterior en- <br /> that competing interests exist.                        richment partially redundant CCCH RNA-binding proteins MEX-5 MEX-6, <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120984 March 30, 2015                                                                                                      1 / 14 <br />                                                                                           Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br />                                          function reinforce polarity [6,7]. Such anterior enrichment requires phosphorylation <br />                                          MEX-5 PAR-1 kinase, results increased MEX-5 diffusion posterior <br />                                          A-P MEX-5 gradient [8]. As result proper A-P polarity, zygote divides <br />                                          unequally, yielding larger anterior cell AB smaller posterior cell P1. Thereafter, AB <br />                                          divides symmetrically, generating ABa ABp, P1 divides asymmetrically, resulting <br />                                          EMS P2. <br />                                              Translational regulation plays important role C. elegans development past the <br />                                          two-cell stage. Thus, MEX-5/6, MEX-3 SPN-4, regulate translation of <br />                                          mRNAs anterior blastomeres, POS-1 regulates mRNAs posterior <br />                                          blastomeres [9&#226;&#8364;&#8220;11]. In general, translational repressors activators act binding the <br />                                          3&#226;&#8364;&#8482;UTR target mRNAs. For example, translation uniformly localized mRNA encoding <br />                                          Notch receptor GLP-1 activated ABa ABp binding MEX-5, SPN-4 and <br />                                          MEX-3 glp-1 3&#226;&#8364;&#8482;UTR, repressed, 3&#226;&#8364;&#8482;UTR, POS-1 EMS <br />                                          P2 [11,12]. Similarly, homogenously distributed zif-1mRNA, encodes E3 ubi- <br />                                          quitin protein ligase complex component, dual regulation, MEX-5-me- <br />                                          diated activation winning POS-1-directed repression anterior blastomeres, thereby <br />                                          giving rise asymmetric ZIF-1 protein distribution [13]. <br />                                              Although majority maternally provided mRNAs exhibits uniform distribution in <br />                                          early C. elegans embryos (so-called class I mRNAs [14]), identified po- <br />                                          larized embryos past 2-cell stage. Amongst them, class II mRNAs cey-2, skn-1 or <br />                                          pos-1 distributed uniform manner zygote, enriched P-lineage <br />                                          following preferential degradation cells [10,14]. Single cell mRNA <br />                                          deep sequencing experiments revealed 4 mRNAs abundant P1 AB, <br />                                          converse true 13 mRNAs [15]. Whether mRNAs exhibits <br />                                          asymmetric distribution zygote addressed. Therefore, mex-3 mRNA, <br />                                          enriched anterior zygote present solely anterior blastomeres <br />                                          thereafter, mRNA known exhibit restricted distribution C. elegans zygote, <br />                                          mechanisms governing distribution understood [4,9] <br />  <br />  <br />                                          Materials Methods <br />                                          Worm strains molecular biology <br />                                          Wild type (N2) animals maintained 20&#226;&#8364;&#8220;24&#194;&#176;C, transgenic worms expressing YFP-ERA- <br />                                          1[3&#226;&#8364;&#8482;era-1] UTR, YFP[3&#226;&#8364;&#8482;era-1] YFP-ERA-1[3&#226;&#8364;&#8482;pie-1] 24&#194;&#176;C. All constructs subcloned <br />                                          pCFJ178 [15], expresses C. briggsae unc-119 genomic sequence. The yfp gene <br />                                          3 introns [16] fused era-1 genomic sequence, start codon end <br />                                          174 nt-long 3&#226;&#8364;&#8482;UTR (yfp-era-1[3&#226;&#8364;&#8482;era-1) stop codon open reading frame <br />                                          (yfp-era-1[3&#226;&#8364;&#8482;pie-1]). The 120 nt-long pie-1 3&#226;&#8364;&#8482;UTR used yfp-era-1[3&#226;&#8364;&#8482;pie-1]. Transgenic <br />                                          lines generated ballistic bombardment unc-119(ed3) animals [17]; designations <br />                                          resulting strains follows: GZ1146&#226;&#8364;&#8221;isIs101[pie-1::yfp-era-1[3&apos;era-1] unc-119(+); <br />                                          GZ1280 -isIs102[pie-1::yfp[3&apos;era-1] unc-119(+) GZ1169&#226;&#8364;&#8221;isIs103[pie-1::yfp-era-1[3&apos;pie-1] <br />                                          unc-119(+). Note strain (GZ1169), phenotypically wild-type animals segre- <br />                                          gate *50% wild-type, *25% Uncs *25% sterile animals protruding vulva (Ste/Pvl). <br />                                          Because Ste/Pvl <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span>phenotype</span> suppressed gfp(RNAi), surmise does not <br />                                          stem presence YFP-ERA-1 protein, instead disruption, transgene <br />                                          insertion, gene important vulva development. However, exclude possibil- <br />                                          ity GZ1169 carries extrachromosomal array. The era-1(tm5852) era-1(tm6426) <br />                                          strains provided National Bioresource Project (NBRP) maintained 20 <br />                                          24&#194;&#176;C. Bacterial feeding strains era-1, mex-5, par-3 Vidal library [18], that <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120984 March 30, 2015                                                                           2 / 14 <br />                                                                                           Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br />                                          par-2 Ahringer library [19]. The feeding clones mex-6 par-1 gen- <br />                                          erated amplifying region mex-6 (between position 241 1014) par-1 (between <br />                                          positions 1288 2395) cDNAs, cloned L4440 vector [20]. Feeding <br />                                          RNAi performed 24 hours 24&#194;&#176;C, par-1(RNAi), 48 hours incubation <br />                                          24&#194;&#176;C needed. Double depletion achieved mixing IPTG-induced individual <br />                                          bacterial cultures 1:1 ratio. For RNAi injection soaking, dsRNA targeting en- <br />                                          tire era-1 mRNA synthesized SP6 T7 vitro synthesis kits (Promega). Soak- <br />                                          ing performed L3 animals 24 hours 20&#194;&#176;C resulting embryos <br />                                          investigated 24 hours recovery. A similar recovery period followed injection L4 animals <br />                                          era-1 dsRNA. era-1 RNAi feeding performed 20&#226;&#8364;&#8220;24 hours 24&#194;&#176;C. <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> partially <br />                                          deplete par-2 par-3, L4 animals fed bacteria expressing relevant dsRNAs 4 <br />                                          hours 24&#194;&#176;C transferred plates containing OP50 20&#194;&#176;C, laid eggs <br />                                          scored embryonic viability. <br />  <br />  <br />                                          Live imaging <br />                                          Dual time-lapse DIC fluorescence imaging performed Zeiss Axioplan 2 de- <br />                                          scribed [21]. The motorized filter wheel, external shutters, 1,392 x 1,040 pixels <br />                                          12-bit Photometrics CoolSNAP ES2 controlled &#206;&#188;Manager [22]. Images acquired <br />                                          exposure time 10&#226;&#8364;&#8220;100ms DIC 300 ms fluorescence channel using <br />                                          Zeiss Filter Set 10. Embryos allowed develop coverslip imaging <br />                                          snapshots taken indicating times. <br />  <br />  <br />                                          Immunostaining <br />                                          For immunostaining, embryos fixed methanol &#226;&#710;&#8217;20&#194;&#176;C 30 minutes followed in- <br />                                          cubating overnight 4&#194;&#176;C primary antibodies. Primary mouse antibodies &#206;&#177;-tubu- <br />                                          lin (1/300; DM1A, Sigma) GFP (1/100; MAB3580, Millipore) used rabbit <br />                                          antibodies GPA-16 (1/200, [23] PAR-2 (1:200, [24]. Secondary antibodies Alex- <br />                                          a488-conjugated goat anti-mouse (1/500; Life Technologies, A11001) Cy3-conjugated goat <br />                                          anti-rabbit (1/1000; Jackson ImmunoResearch, 711-165-152). Slides counterstained with <br />                                          1mg/ml Hoechst 33258 (Sigma) visualize DNA. Images acquired LSM700 confo- <br />                                          cal microscope (Zeiss) processed ImageJ. <br />                                              For quantification cell membrane enrichment, fluorescence intensities YFP and <br />                                          GPA-16 measured membrane middle plane embryo values ex- <br />                                          pressed ratio YFP GPA-16 intensities. <br />  <br />  <br />                                          RT-qPCR <br />                                          Embryos bleached total mRNA isolated using trizol-chloroform extraction [25]. <br />                                          200 &#206;&#188;g total RNA reverse transcribed using poly-dT random hexamer primers. <br />                                          Specific primer pairs used amplify cDNA Power SYBR Green PCR mix <br />                                          (Applied Biosystems). The PCR reaction performed 7900HT Fast Real-Time PCR <br />                                          System machine (Applied Biosystems) SDS2.4 software using 60&#194;&#176;C annealing and <br />                                          72&#194;&#176;C elongation. The &#206;&#8221;&#206;&#8221;CT method used quantify mRNA levels relative refer- <br />                                          ence gene, act-1. The primers follow: <br />                                           act-1 F &#226;&#8364;&#8220; AGTCCGCCGGAATCCACGAG <br />                                           act-1 R &#226;&#8364;&#8220; CTTGATCTTCATGGTTGATG <br />                                           yfp F &#226;&#8364;&#8220; GGGAACTACAAGACACGTGC <br />                                           yfp R &#226;&#8364;&#8220; GTGTCCAAGAATGTTTCC <br />  <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120984 March 30, 2015                                                                           3 / 14 <br />                                                                                           Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br />                                          Digoxigenin situ hybridization <br />                                          Digoxigenin-labeled single stranded DNA probes prepared using PCR DIG Probe <br />                                          Synthesis Kit (Roche) following manufacturer&#226;&#8364;&#8482;s instructions. Briefly, plasmid carrying <br />                                          gene served template PCR-based reaction, reverse primer binding <br />                                          3&#226;&#8364;&#8482;UTR added DIG-11-dUTP containing dNTP mix. Note did <br />                                          linearize plasmid preparing probes used circular form asym- <br />                                          metric PCR instead, reasoning non-specific sequences probe mixture help <br />                                          block non-specific binding sites. The resulting PCR product ethanol-precipitated re- <br />                                          suspended hybridization buffer (50% deionized formamide, 5x SSC, 100 &#206;&#188;g/ml sonicated <br />                                          salmon testis DNA, 100 &#206;&#188;g/ml yeast tRNA, 100 &#206;&#188;g/ml heparin, 0.1% Tween-20) concentra- <br />                                          tion 50 ng/&#206;&#188;l, boiled 95&#194;&#176;C 75 minutes reduce size (Seydoux Fire, 1995) and <br />                                          stored &#226;&#710;&#8217;20&#194;&#176;C. <br />                                              The situ hybridization protocol developed combination previously available <br />                                          procedures [26,27]. Gravid hermaphrodites bleached resulting embryos placed <br />                                          poly-D-lysine coated slide total volume 7&#206;&#188;l diluted water. Then coverslip was <br />                                          gently deposited embryos, freeze-crack method used remove eggshell. <br />                                          Subsequent steps performed plastic slide holder containing 150 ml indicated <br />                                          solutions, unless stated. First, slides submerged &#226;&#710;&#8217;20&#194;&#176;C methanol <br />                                          5 minutes sequential mixtures methanol Hepes-PBS-formalde- <br />                                          hyde (HPF) solutions 4&#194;&#176;C (1x PBS, 75mM Hepes pH = 6.9, 0.03% EGTA, 1.5 mM MgSO4, <br />                                          3.6% formaldehyde); ratios water:methanol:Hepes-PBS-formaldehyde 50:15:35, <br />                                          50:25:25 finally 50:35:15), 2 minutes 4&#194;&#176;C, fixation 3.6% formalde- <br />                                          hyde HPF solution 20 minutes 4&#194;&#176;C [26]. Following wash PBS-Tween (0.1%, <br />                                          PBS-Tw) 5 minutes room temperature, Proteinase K (5.6 &#206;&#188;g/ml) digestion performed <br />                                          10 minutes 24&#194;&#176;C PBS-Tw stopped submerging slides 2 g/l glycine <br />                                          dissolved PBS 2 min followed 2 washes PBS-Tw 2 minutes each. The slides were <br />                                          re-fixed HPF solution 50 minutes room temperature, washed twice PBS-Tw <br />                                          5 minutes each, treated 2 mg/ml glycine 5 minutes washed with <br />                                          PBS-T. Slides submerged Formamide-SSC solution (50% formamide-5xSSC <br />                                          (pH = 7) -100 &#206;&#188;g/ml heparin, 0.1% Tween-20), mixed PBS-Tw 1:1 ratio 10 min- <br />                                          utes Formamide-SSC 10 minutes. Pre-hybridization carried <br />                                          50 &#206;&#188;l hybridization buffer (50% deionized formamide 50%, 5xSSC, 100 &#206;&#188;g/ml sonicated <br />                                          salmon testis DNA, 100 &#206;&#188;g/ml yeast tRNA, 100 &#206;&#188;g/ml, heparin 0.1% Tween 20) 1 hour at <br />                                          48&#194;&#176;C, probe applied concentration 40 ng/&#206;&#188;l 50 &#206;&#188;l overnight, again <br />                                          48&#194;&#176;C. To prevent evaporation, parafilm squares glued rubber cement mounted on <br />                                          slides. The probe washed twice Formamide-SSC:PBS-Tw (1:1) 10 minutes <br />                                          4 times using 0.1% Chaps 0.8x PBS 48&#194;&#176;C 20 minutes [26], a <br />                                          blocking step included using 0.1%BSA&#226;&#8364;&#8221;0.1%Triton-X 1xPBS (PBS-BSA-Tx) 1.5 <br />                                          hours room temperature. The alkaline-phosphatase (AP)-conjugated anti-DIG antibody <br />                                          (Roche) diluted 1:2000 1x PBS incubated 4&#194;&#176;C overnight. The slides washed <br />                                          5 times PBS-BSA-Tx twice staining buffer (100 mM Tris-HCl (pH = 9.5), <br />                                          100mM NaCl, 5mM MgCl2, 1mM Levamisole, 0.1% Tween-20); signal developed <br />                                          using mixture 333 &#206;&#188;g/ml NBT (Roche) 70 &#206;&#188;g/ml BCIP (AppliChem) 20&#226;&#8364;&#8220;60 min- <br />                                          utes room temperature. The slides washed PBS-Tw 5 minutes at <br />                                          room temperature, stained 1mg/ml Hoechst 33258 (Sigma) PBS-Tw, washed once <br />                                          PBS, mounted stored 4&#194;&#176;C. All situ hybridization experiments performed at <br />                                          twice representative embryos shown figure panels. Control embryos were <br />                                          included slide instance. <br />  <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120984 March 30, 2015                                                                           4 / 14 <br />                                                                                            Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br />                                             Quantification mRNA levels performed measuring mean intensities cyto- <br />                                          plasm AB P1 (A, P), background outside embryos (B), later stage <br />                                          embryos, hybridization did signal (C). The value B subtracted from <br />                                          A, P C, resulting corrected values used determine ratios A/C P/C, which <br />                                          reported figures arbitrary units (A.U.). In instances, one-cell embryo <br />                                          shown figures surrounded multicellular embryos possessing high Hoechst signal <br />                                          intensities, leading overflow blue signal embryo interest. In cases, <br />                                          background subtracted Hoechst image divided Gaussian blurred value, <br />                                          resulting clearing non-pertinent signal. Then brightness enhanced clearer <br />                                          visualization relevant DNA signal. <br />  <br />  <br />                                          Results Discussion <br />                                          The era-1 mRNA enriched anterior C. elegans zygote and <br />                                          anterior blastomeres thereafter <br />                                          To determine localized mRNAs zygote apart mex-3, analyzed <br />                                          distribution 11&#226;&#8364;&#8482;237 mRNAs available in situ hybridization database (Nematode <br />                                          Expression Database (NEXTDB), http://nematode.lab.nig.ac.jp).This database contains typical- <br />                                          ly 4&#226;&#8364;&#8220;6 images, 7&#226;&#8364;&#8220;8 embryos different stages development. We analyzed <br />                                          images low magnification; non-uniform distribution spotted in <br />                                          early stage embryos manner, images magnified assess mRNA distri- <br />                                          bution. This analysis led uncover transcript uncharacterized open reading <br />                                          frame W02F12.3 displays suggestive distribution, bias anterior zy- <br />                                          gote embryos subsequent stages embryogenesis. We dubbed gene era-1 (for <br />                                          embryonic mRNA anterior) investigated further. <br />                                              We ascertained asymmetric distribution era-1 mRNA digoxygenin-alkaline phos- <br />                                          phatase situ hybridization. We asymmetric enrichment era-1 mRNA becomes <br />                                          apparent prophase cell cycle (Fig. 1A-B). Moreover, era-1 mRNA enriched <br />                                          anterior blastomere AB compared posterior blastomere P1 two-cell embryos <br />                                          (Fig. 1C, 1I). This difference persists comparing AB daughters ABa ABp the <br />                                          P1 daughters EMS P2 (Fig. 1D). No signal detected era-1(RNAi) embryos, indicating <br />                                          probe specificity (Fig. 1F, 1I). <br />                                              Intriguingly, era-1 genes identified single cell mRNA sequencing as <br />                                          enriched AB versus P1 [28]. Conversely, 13 mRNAs identified study as <br />                                          enriched AB versus P1, mex-3 mRNA shows clear asymmetry Nematode <br />                                          Expression Database. Another 6 mRNAs appear uniformly distributed, 5 been <br />                                          tested 1 analyzable oversaturated signal. Therefore, mRNA sequencing <br />                                          large-scale situ hybridization methods provide complementary means identify <br />                                          mRNAs asymmetric distribution C. elegans embryos. <br />  <br />  <br />                                          Structure function era-1 <br />                                          The era-1 gene predicted encode 388 amino acids long polypeptide, clear protein <br />                                          domains (S1A Fig.). We distant relatives ERA-1 nematodes (S1B Fig.), al- <br />                                          bona fide orthologues, performing BLAST analysis against <br />                                          C. elegans proteome did identify ERA-1 hit, indicating fast divergence from <br />                                          common ancestor. <br />                                             We set address function era-1 C. elegans. A comprehensive RNAi-based <br />                                          functional genomic screen did reveal requirement era-1 [29]. Accordingly, found <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120984 March 30, 2015                                                                             5 / 14 <br />                                                                                                        Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br />  <br /> Fig 1. era-1 mRNA enriched anterior early C. elegans embryos. A-D era-1 mRNA distribution wild-type zygote (P0), prophase (A) or <br /> prometaphase (B), 2-cell (C) 4-cell (D) embryos. Here figures, mRNA appears dark grey, DNA light blue scale bars <br /> represent 10 microns. E-H era-1 mRNA distribution control (E), era-1(RNAi) (F), par-3(RNAi) (G) mex-5(RNAi) (H) 2-cell embryos. I Corresponding <br /> quantifications mRNA levels AB P1 (Materials Methods). Number embryos quantified: wild type, n = 9; era-1(RNAi), n = 12; par-3(RNAi), <br /> n = 10; mex-5(RNAi), n = 7. In figures, experiments performed twice, mean values &#194;&#177; standard deviations (SD) shown, with <br /> exact values given S1 Table. Statistical analysis performed using unpaired Student&#226;&#8364;&#8482;s t-test compare mRNA levels AB versus P1, yielding the <br /> following p-values: wild type&#226;&#8364;&#8221;p = 5.67&#195;&#8212;10&#226;&#710;&#8217;10, era-1(RNAi), p = 0.326; par-3(RNAi), p = 0.907; mex-5(RNAi), p = 0.287. <br /> doi:10.1371/journal.pone.0120984.g001 <br />  <br />                                                performing era-1(RNAi) injection, soaking feeding did result lethality an <br />                                                obvious phenotype resulting progeny. Furthermore, deletion alleles <br />                                                era-1(tm5852) era-1(tm6426) viable fertile exhibit apparent pheno- <br />                                                type embryo (S1A, S1C Fig.). We conclude era-1 essential gene stan- <br />                                                dard laboratory conditions. <br />  <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120984 March 30, 2015                                                                                            6 / 14 <br />                                                                                           Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br />                                             Because intriguing polarized distribution era-1 mRNA, rapid <br />                                          sequence divergence nematodes, explored era-1 exhibit synthetic le- <br />                                          thality genes required embryonic polarity C. elegans. Intriguingly, that <br />                                          era-1(tm6426) embryos sensitive wild-type partial RNAi-mediated <br />                                          depletion PAR-3 PAR-2 (S1C Fig.). Whereas future work needed determine the <br />                                          mechanism underlying synthetic lethality, conclude ERA-1 plays non-essential <br />                                          role ensuring robust embryonic development. <br />  <br />                                          era-1 mRNA distribution requires MEX-5 era-1 3&#226;&#8364;&#8482;UTR <br />                                          We sought address regulates remarkable distribution era-1 mRNA. First, we <br />                                          asked anterior enrichment polarity-dependent. As shown Fig. 1G 1I, that <br />                                          case, era-1 mRNA distribution uniform par-3(RNAi) embryos. What <br />                                          trans-acting factor govern anterior enrichment? The anteriorly localized <br />                                          CCCH finger protein MEX-5 plausible candidate likewise enriched the <br />                                          anterior one-cell stage embryos [9]. Accordingly, era-1 mRNA localized in <br />                                          uniform manner mex-5(RNAi) embryos (Fig. 1H, 1I). Although merely reflect a <br />                                          role MEX-5 polarity establishment, evidence shown suggests MEX-5 plays a <br />                                          direct role regulating era-1 mRNA. <br />                                              To identify cis-acting element regulating era-1 mRNA distribution, fused sequence <br />                                          coding yellow fluorescent protein (YFP) era-1 genomic sequence (exons, introns <br />                                          3&#226;&#8364;&#8482;UTR) expression vector driven pie-1 promoter (yielding yfp-era-1[3&#226;&#8364;&#8482;era-1], <br />                                          Fig. 2A). We generated corresponding transgenic line performed situ hybridization <br />                                          yfp mRNA. We yfp-era-1[3&#226;&#8364;&#8482;era-1] mRNA enriched anterior <br />                                          zygote inherited anterior blastomeres (Fig. 2B, 2G). <br />                                              Sequences 3&#226;&#8364;&#8482;UTR responsible localizing mRNAs [30,31]. To test the <br />                                          contribution era-1 3&#226;&#8364;&#8482;UTR era-1 mRNA distribution, generated transgenic animals <br />                                          expressing yfp fused directly era-1 3&#226;&#8364;&#8482;UTR (yielding yfp[3&#226;&#8364;&#8482;era-1], Fig. 2C, 2D). As shown <br />                                          Fig. 2G, analysis uncovered significant anterior bias yfp[3&#226;&#8364;&#8482;era-1] mRNA, although <br />                                          extent enrichment AB versus P1 pronounced yfp-era-1[3&#226;&#8364;&#8482;era-1] <br />                                          (compare Fig. 2D 2B). We conclude era-1 3&#226;&#8364;&#8482;UTR, sufficient for, <br />                                          contributes mRNA anterior distribution, help elements located coding <br />                                          sequence. <br />                                              To test era-1 3&#226;&#8364;&#8482;UTR necessary anterior mRNA distribution, modified yfp- <br />                                          era-1[3&#226;&#8364;&#8482;era-1] replacing era-1 3&#226;&#8364;&#8482;UTR pie-1 3&#226;&#8364;&#8482;UTR (yielding yfp-era-1[3&#226;&#8364;&#8482;pie-1], <br />                                          Fig. 2E), generated corresponding transgenic line. Strikingly, yfp-era-1 <br />                                          [3&#226;&#8364;&#8482;pie-1] mRNA localizes homogenous manner, difference AB P1 <br />                                          (Fig. 2F, 2G). Therefore, era-1 3&#226;&#8364;&#8482;UTR necessary anterior mRNA distribution. <br />  <br />                                          YFP-ERA-1 protein enriched anterior blastomeres <br />                                          To address anterior enrichment era-1 mRNA accompanied likewise pro- <br />                                          tein distribution, analyzed embryos expressing yfp-era-1[3&#226;&#8364;&#8482;era-1] using dual differential in- <br />                                          terference contrast (DIC) fluorescent time-lapse microscopy. This revealed YFP-ERA- <br />                                          1[3&#226;&#8364;&#8482;era-1] protein detected starting *10 cell stage solely descendants the <br />                                          anterior blastomere AB (Fig. 3A). We performed immunofluorescence analysis using anti- <br />                                          bodies YFP offer sensitive detection means. We YFP-ERA- <br />                                          1[3&#226;&#8364;&#8482;era-1] distributed uniformly zygote enriched AB cell (S1D, S1E <br />                                          Fig.), anterior bias pronounced 4-cell stage embryos (Fig. 4A, 4E). In- <br />                                          triguingly, noted YFP-ERA-1[3&#226;&#8364;&#8482;era-1] slightly enriched cell cortex, <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120984 March 30, 2015                                                                           7 / 14 <br />                                                                                                          Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br />  <br /> Fig 2. yfp-tagged era-1 mRNA recapitulates endogenous era-1 mRNA distribution 3&#226;&#8364;&#8482;UTR-dependent manner. A, C, E Schematic representation <br /> of yfp-era-1[3&#226;&#8364;&#8482; era-1] (A), yfp[3&#226;&#8364;&#8482;era-1] (C) yfp-era-1[3&#226;&#8364;&#8482;pie-1] (E) constructs. B, D, F Localization constructs 2-cell embryos. G Corresponding <br /> quantifications mRNA levels AB P1 (Materials Methods). Number embryos quantified: yfp-era-1[3&#226;&#8364;&#8482;era-1], n = 9;. yfp[3&#226;&#8364;&#8482;era-1], n = 8; yfp-era- <br /> 1[3&#226;&#8364;&#8482;pie-1], n = 10. Statistical analysis performed using unpaired Student&#226;&#8364;&#8482;s t-test compare mRNA levels AB versus P1, yielding following p- <br /> values: yfp-era-1[3&#226;&#8364;&#8482; era-1], p = 1.03&#195;&#8212;10&#226;&#710;&#8217;7; yfp[3&#226;&#8364;&#8482;era-1], p = 2&#195;&#8212;10&#226;&#710;&#8217;4; yfp-era-1[3&#226;&#8364;&#8482;pie-1], p = 0.362. Note anterior enrichment yfp-era-1[3&#226;&#8364;&#8482;era-1] is <br /> somewhat pronounced endogenous era-1 mRNA, era-1 5&#226;&#8364;&#8482;UTR contributes mRNA distribution the <br /> presence yfp perturbs localization. <br /> doi:10.1371/journal.pone.0120984.g002 <br />  <br />                                                 apparent boundary anterior blastomeres (Fig. 4A), rais- <br />                                                 ing possibility ERA-1 cortical protein. To address anterior enrichment of <br />                                                 YFP-ERA-1[3&#226;&#8364;&#8482;era-1] protein dependent A-P polarity, analyzed embryos depleted of <br />                                                 PAR-3. As anticipated uniform distribution era-1 mRNA embryos (see <br />                                                 Fig. 1I), YFP-ERA-1[3&#226;&#8364;&#8482;era-1] present comparable levels blastomeres of <br />                                                 par-3(RNAi) embryos (Fig. 4C, 4E). <br />                                                    To test era-1 3&#226;&#8364;&#8482;UTR important restricting fusion protein anterior blasto- <br />                                                 meres, examined embryos expressing yfp-era-1[3&#226;&#8364;&#8482;pie-1]. Interestingly, that <br />                                                 YFP-ERA-1[3&#226;&#8364;&#8482;pie-1] protein present uniform manner zygote (Figs. <br />                                                 3B, 4F, 4H). We conclude era-1 3&#226;&#8364;&#8482;UTR important restricting presence the <br />                                                 fusion protein anterior blastomeres embryogenesis. Moreover, noted that <br />                                                 YFP-ERA-1[3&#226;&#8364;&#8482;pie-1] protein levels higher YFP-ERA-1[3&#226;&#8364;&#8482;era-1], in <br />                                                 anterior blastomeres (compare Figs. 3A 3B 4A 4F). This despite yfp-era-1 <br />                                                 [3&#226;&#8364;&#8482;pie-1] mRNA levels lower yfp-era[3&#226;&#8364;&#8482;era-1] (Fig. 2G). Because pro- <br />                                                 tein present newly fertilized zygote comparing YFP-ERA-1[3&#226;&#8364;&#8482;pie-1] with <br />                                                 YFP-ERA-1[3&#226;&#8364;&#8482;era-1] (compare Fig. 3B 3A, row), findings indicate the <br />                                                 era-1 3&#226;&#8364;&#8482;UTR prevents translation oogenesis. <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120984 March 30, 2015                                                                                               8 / 14 <br />                                                                                                       Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br />  <br /> Fig 3. YFP-ERA-1 enriched anterior blastomeres 3&#226;&#8364;&#8482;UTR- MEX-5/6-dependent manner. A-B Snapshots dual DIC fluorescent <br /> time-lapse microscopy recordings embryos expressing YFP-ERA-1[3&#226;&#8364;&#8482;era-1] (A) YFP-ERA-1[3&#226;&#8364;&#8482;pie-1] (B). Time indicated min:sec, t = 0 <br /> corresponding cytokinesis onset P0. Arrowheads points YFP signal cell cortex. Brightness contrast lowered panel B avoid <br /> saturating signal. C-D Snapshots late stage mex-5/6(RNAi) embryos expressing YFP-ERA-1[3&#226;&#8364;&#8482;era-1] (C) YFP-ERA-1[3&#226;&#8364;&#8482;pie-1] (D). <br /> doi:10.1371/journal.pone.0120984.g003 <br />  <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120984 March 30, 2015                                                                                               9 / 14 <br />                                                               Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120984 March 30, 2015                                              10 / 14 <br />                                                                                                          Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br /> Fig 4. MEX-5/6-mediated translational activation era-1. A-D 4-cell embryos expressing YFP-ERA-1[3&#226;&#8364;&#8482; era-1] control (A), mex-5/6(RNAi) (B), par-3 <br /> (RNAi) (C) par-1(RNAi) (D) conditions, stained antibodies GFP (green) GPA-16 (red); DNA viewed blue. E Signal intensities the <br /> cell-cell boundaries ABa/ABp EMS/P2 relative GPA-16 (Materials Methods). Number embryos quantified: control, n = 10; mex-5/6(RNAi), <br /> n = 10; par-3(RNAi), n = 9; par-1(RNAi), n = 10. Statistical analysis performed using unpaired Student&#226;&#8364;&#8482;s t-test compare control RNAi conditions for <br /> both anterior posterior blastomeres, yielding following p-values: mex-5/6(RNAi), anterior: p = 4.42&#195;&#8212;10&#226;&#8364;&#8220;7; posterior: p = 1.46&#195;&#8212;10&#226;&#8364;&#8220;3; par-3(RNAi), <br /> anterior: p = 9.32&#195;&#8212;10&#226;&#8364;&#8220;3; posterior: p = 0.04; par-1(RNAi), anterior: p = 0.741; posterior: p = 1.81&#195;&#8212;10&#226;&#8364;&#8220;5. F-G 4-cell embryos expressing YFP-ERA-1[3&#226;&#8364;&#8482; pie-1] in <br /> control (F) mex-5/6(RNAi) (G). Red arrowheads F indicate cytoplasmic foci resembling endomembranes, suggesting ERA-1 traffics through <br /> endosomal compartments. H Signal intensities cell-cell boundaries ABa/ABp EMS/P2 relative GPA-16 (Materials Methods). Number of <br /> embryos quantified: control, n = 10; mex-5/6(RNAi), n = 8. Statistical analysis performed using unpaired Student&#226;&#8364;&#8482;s t-test compare control mex-5/6 <br /> (RNAi) anterior posterior blastomeres, yielding following p-values: mex-5/6 (RNAi), anterior: p = 0.113; posterior: p = 0.205. I Working model <br /> of MEX-5/6-mediated control era-1 mRNA asymmetric distribution P0 translational activation 2-cell embryos; main text. <br /> doi:10.1371/journal.pone.0120984.g004 <br />  <br />  <br />                                                 MEX-5/6 positively regulates era-1 translation era-1 3&#226;&#8364;&#8482;UTR <br />                                                 Because era-1 mRNA YFP-ERA-1 protein enriched anterior blastomeres, the <br />                                                 distribution protein dictated merely mRNA. Alternatively in <br />                                                 addition, translational activation occur anterior blastomeres. What factors could <br />                                                 responsible putative activation? MEX-5 MEX-6 activate translation mRNAs <br />                                                 mex-3 glp-1 anterior blastomeres [9]. Compatible possibility, found <br />                                                 MEX-5/6 target consensus sequences (i.e.  6 U 13 nucleotides [12] era-1 <br />                                                 3&#226;&#8364;&#8482;UTR, additional ones era-1 coding sequence (S2A Fig.). <br />                                                    We depleted MEX-5/6 explore regulate era-1 mRNA translation. Strik- <br />                                                 ingly, YFP-ERA-1[3&#226;&#8364;&#8482;era-1] protein levels greatly reduced mex-5/6(RNAi) <br />                                                 embryos (Figs. 3C, 4B, 4E). RT-qPCR analysis revealed slight statistically non-significant <br />                                                 decrease mRNA levels (S2B Fig.), suggesting reduced YFP signal to <br />                                                 mRNA degradation. Given MEX-5/6 play role downstream PAR proteins in <br />                                                 establishing A-P polarity [9], given era-1 mRNA distribution homogenous mex-5 <br />                                                 (RNAi) embryos, principle uniform distribution merely reflect polarity defects. <br />                                                 However, view unlikely, YFP-ERA-1[3&#226;&#8364;&#8482;era-1] levels lower mex-5/6 <br />                                                 (RNAi) embryos par-3(RNAi) embryos, polarity likewise affected (Fig. 4B, <br />                                                 4C, 4E). Of note, depletion translational regulator MEX-3 did affect YFP-ERA-1 <br />                                                 [3&#226;&#8364;&#8482;era-1] protein levels (S2C Fig.), indicating specificity effect incurred following MEX- <br />                                                 5/6 depletion. Furthermore, levels YFP-ERA-1[3&#226;&#8364;&#8482;pie-1] protein unaf- <br />                                                 fected mex-5/6(RNAi) embryos (Figs. 3D, 4G), indicating translational activation occurs <br />                                                 era-1 3&#226;&#8364;&#8482;UTR. <br />                                                    We reasoned MEX-5/6 promote translational activation era-1 mRNA suggested <br />                                                 experiments, posterior YFP-ERA-1[3&#226;&#8364;&#8482;era-1] protein levels higher em- <br />                                                 bryos depleted PAR-1, MEX-5/6 activity increased posterior [8]. As shown <br />                                                 Fig. 4D 4E, case indeed. Overall, observations lead to <br />                                                 conclude MEX-5/6 needed activate translation era-1 mRNA its <br />                                                 3&#226;&#8364;&#8482;UTR embryogenesis. <br />  <br />                                                 Conclusions <br />                                                 Our findings suggest following working model regulation era-1 mRNA pro- <br />                                                 tein asymmetric distribution (Fig. 4I). In model, era-1 mRNA anterior enrichment regu- <br />                                                 lated A-P polarity cues, MEX-5/6 era-1 3&#226;&#8364;&#8482;UTR. MEX-5/6 potentially <br />                                                 transport era-1 mRNA anterior, giving rise asymmetric distribution (Fig. 4I). Fur- <br />                                                 thermore, MEX-5/6 act 3&#226;&#8364;&#8482;UTR, directly indirectly, activate translation the <br />                                                 era-1 mRNA embryo. These findings provide novel insights mechanisms by <br />                                                 mRNA distribution coupled translational regulation contribute robust develop- <br />                                                 ment C. elegans embryos. <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120984 March 30, 2015                                                                                              11 / 14 <br />                                                                                             Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br />                                          Supporting Information <br />                                          S1 Fig. The C. elegans specific era-1 gene contributes robust embryonic development. A. <br />                                          Schematic wild-type era-1 gene era-1(tm5852) era-1(tm6426) mutant alleles, <br />                                          indication premature STOP codons (lack translation depicted grey). Analysis <br />                                          Scanprosit [1] failed identify clear protein domains ERA-1. B. Phylogenetic relation- <br />                                          ships ERA-1-related proteins select nematodes identified Blast search <br />                                          (Ensembl). The tree built Clustal W2 software (http://www.ebi.ac.uk/Tools/ <br />                                          phylogeny/clustalw2_phylogeny/) sequence alignment performed MAFFT server <br />                                          (http://mafft.cbrc.jp/alignment/server/). The values indicate <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> branch leading to <br />                                          previous node number substitutions proportion alignment <br />                                          length. C. Embryonic lethality following partial depletion PAR-3 PAR-2 wild type and <br />                                          era-1(tm6426) worms. &gt;800 embryos scored 8 independent experiments par-3 <br />                                          (RNAi), &gt;200 embryos 3 independent experiments par-2(RNAi). Note enhance- <br />                                          ment observed glp-1(RNAi) apx-1(RNAi) compared wild type, indicating <br />                                          enhanced lethality observed par-3(RNAi) par-2(RNAi) generic effect <br />                                          response RNAi. Statistical analysis performed using unpaired Student&#226;&#8364;&#8482;s t-test com- <br />                                          pare embryonic lethality wild type versus era-1(tm6426) indicated conditions, yielding <br />                                          following p-values: partial par-3(RNAi), p = 4.6&#195;&#8212;10&#226;&#710;&#8217;3; partial par-2(RNAi), p = 0.048. D-E. <br />                                          YFP-ERA-1[3&#226;&#8364;&#8482;era-1] protein telophase cell division (D) 2-cell stage (E). <br />                                          The upper images YFP signal alone, lower ones merge. Scale bar represents <br />                                          10 microns. <br />                                          1. Castro E, Sigrist CJ, Gattiker A, Bulliard V, Langendijk-Genevaux PS, Gasteiger E, et al. <br />                                          (2006) ScanProsite: detection PROSITE signature matches ProRule-associated function- <br />                                          al structural residues proteins. Nucleic acids research 34: W362&#226;&#8364;&#8220;365. <br />                                          (PDF) <br />                                          S2 Fig. MEX-5-dependent translational regulation era-1 mRNA potential MEX-5 <br />                                          binding sites. A. Putative MEX-5 binding sites (red) era-1 exons (blue) the <br />                                          3&#226;&#8364;&#8482;UTR (green). The sequences regions numbered 1&#226;&#8364;&#8220;7 shown red. B. mRNA <br />                                          levels yfp-era-1[3&#226;&#8364;&#8482;era-1], yfp[3&#226;&#8364;&#8482;era-1] yfp-era-1[3&#226;&#8364;&#8482;pie-1] measured RT-qPCR or <br />                                          MEX-5 depletion (Materials Methods). Values shown relative act-1 mRNA. <br />                                          The experiment performed 3 times. Average values indicated error bars represent- <br />                                          ing standard error mean. Statistical analysis performed using unpaired Student&#226;&#8364;&#8482;s <br />                                          t-test compare mRNA levels control mex-5(RNAi) conditions, yielding following <br />                                          p-values: yfp-era-1[3&#226;&#8364;&#8482;era-1], p = 0.429; yfp[3&#226;&#8364;&#8482;era-1], p = 0.234, yfp-era-1[3&#226;&#8364;&#8482;pie-1], p = 0.284. <br />                                          C. Quantification membrane enrichment relative GPA-16 4-cell stage embryos express- <br />                                          ing YFP-ERA-1[3&#226;&#8364;&#8482;era-1] mex-5 mex-3 depletion, control embryos (Materials <br />                                          Methods). Number embryos quantified: control, n = 11; mex-5(RNAi), n = 7; mex-3 <br />                                          (RNAi), n = 7. Statistical analysis performed using unpaired Student&#226;&#8364;&#8482;s t-test compare <br />                                          control RNAi conditions anterior posterior blastomeres, yielding follow- <br />                                          ing p-values: mex-5(RNAi), anterior: p = 0.021; posterior: p = 0.012; mex-3(RNAi), anterior: <br />                                          p = 0.465; posterior: p = 0.124 Note relative levels (i.e. levels YFP-ERA-1 <br />                                          [3&#226;&#8364;&#8482;era-1] versus GPA-16) lower Fig. 4E variation signal intensities <br />                                          experiment experiment, ratio anterior posterior YFP-ERA-1[3&#226;&#8364;&#8482;era-1] <br />                                          signals comparable experiments. <br />                                          (PDF) <br />                                          S1 Table. Exact values statistical analysis. The designation experiment (first col- <br />                                          umn), nature measurement (second column), genotype/RNAi condition (A: <br />  <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120984 March 30, 2015                                                                            12 / 14 <br />                                                                                                  Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br />                                          anterior side, P: posterior side) (third column), actual value (fourth column), number <br />                                          embryos experiments analyzed (fifth column), p-value using unpaired Student&#226;&#8364;&#8482;s t-test <br />                                          (sixth column) corresponding Figure (last column) reported. <br />                                          (PDF) <br />  <br />  <br />                                          Acknowledgments <br />                                          We thank Geraldine Seydoux advice situ hybridization Yuji Kohara pub- <br />                                          licly available database. We grateful Fernando R. Balestra, Sachin Kotak Jian Qiu for <br />                                          comments manuscript. Some strains obtained Caenorhabditis Genetics <br />                                          Center, funded NIH National Center Research Resources (NCRR). <br />  <br />  <br />                                          Author Contributions <br />                                          Conceived designed experiments: ZS PG. Performed experiments: ZS. Analyzed <br />                                          data: ZS PG. Contributed reagents/materials/analysis tools: ZS PG. Wrote paper: ZS <br />                                          PG. <br />  <br />  <br />                                          References <br />                                           1.   Lasko P (2012) mRNA localization translational control Drosophila oogenesis. Cold Spring Har- <br />                                                bor perspectives biology 4. <br />                                           2.   Kumano G (2012) Polarizing animal cells mRNA localization oogenesis early development. <br />                                                Development, growth &amp; differentiation 54: 1&#226;&#8364;&#8220;18. <br />                                           3.   Hwang SY, Rose LS (2010) Control asymmetric cell division early C. elegans embryogenesis: <br />                                                teaming-up translational repression protein degradation. BMB reports 43: 69&#226;&#8364;&#8220;78. PMID: <br />                                                20193124 <br />                                           4.   Draper BW, Mello CC, Bowerman B, Hardin J, Priess JR (1996) MEX-3 KH domain protein reg- <br />                                                ulates blastomere identity early C. elegans embryos. Cell 87: 205&#226;&#8364;&#8220;216. PMID: 8861905 <br />                                           5.   G&#195;&#182;nczy P, Rose LS (2005) Asymmetric cell division axis formation embryo. WormBook: the <br />                                                online review C elegans biology: 1&#226;&#8364;&#8220;20. <br />                                           6.   Cuenca AA, Schetter A, Aceto D, Kemphues K, Seydoux G (2003) Polarization C. elegans zy- <br />                                                gote proceeds distinct establishment maintenance phases. Development 130: 1255&#226;&#8364;&#8220;1265. <br />                                                PMID: 12588843 <br />                                           7.   Daniels BR, Dobrowsky TM, Perkins EM, Sun SX, Wirtz D (2010) MEX-5 enrichment C. elegans <br />                                                early embryo mediated differential diffusion. Development 137: 2579&#226;&#8364;&#8220;2585. doi: 10.1242/dev. <br />                                                051326 PMID: 20627961 <br />                                           8.   Griffin EE, Odde DJ, Seydoux G (2011) Regulation MEX-5 gradient spatially segregated ki- <br />                                                nase/phosphatase cycle. Cell 146: 955&#226;&#8364;&#8220;968. doi: 10.1016/j.cell.2011.08.012 PMID: 21925318 <br />                                           9.   Schubert CM, Lin R, Vries CJ, Plasterk RH, Priess JR (2000) MEX-5 MEX-6 function estab- <br />                                                lish soma/germline asymmetry early C. elegans embryos. Molecular cell 5: 671&#226;&#8364;&#8220;682. PMID: <br />                                                10882103 <br />                                          10.   Tabara H, Hill RJ, Mello CC, Priess JR, Kohara Y (1999) pos-1 encodes cytoplasmic zinc-finger pro- <br />                                                tein essential germline specification C. elegans. Development 126: 1&#226;&#8364;&#8220;11. PMID: 9834181 <br />                                          11.   Ogura K, Kishimoto N, Mitani S, Gengyo-Ando K, Kohara Y (2003) Translational control maternal <br />                                                glp-1 mRNA POS-1 interacting protein SPN-4 Caenorhabditis elegans. Development 130: <br />                                                2495&#226;&#8364;&#8220;2503. PMID: 12702662 <br />                                          12.   Pagano JM, Farley BM, McCoig LM, Ryder SP (2007) Molecular basis RNA recognition em- <br />                                                bryonic polarity determinant MEX-5. The Journal biological chemistry 282: 8883&#226;&#8364;&#8220;8894. PMID: <br />                                                17264081 <br />                                          13.   Oldenbroek M, Robertson SM, Guven-Ozkan T, Gore S, Nishi Y, Lin R (2012) Multiple RNA-binding <br />                                                proteins function combinatorially control soma-restricted expression <span id='am-163' about='xsp:pattern' typeof='owl:Thing'>pattern</span> E3 ligase sub- <br />                                                <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> ZIF-1. Developmental biology 363: 388&#226;&#8364;&#8220;398. doi: 10.1016/j.ydbio.2012.01.002 PMID: 22265679 <br />                                          14.   Seydoux G, Fire A (1994) Soma-germline asymmetry distributions embryonic RNAs Caenor- <br />                                                habditis elegans. Development 120: 2823&#226;&#8364;&#8220;2834. PMID: 7607073 <br />  <br />  <br />  <br />  <br /> PLOS ONE | DOI:10.1371/journal.pone.0120984 March 30, 2015                                                                                       13 / 14 <br />                                                                                                  Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br />                                          15.   Frokjaer-Jensen C, Davis MW, Hopkins CE, Newman BJ, Thummel JM, Olesen SP, et al. (2008) Sin- <br />                                                gle-copy insertion transgenes Caenorhabditis elegans. Nature genetics 40: 1375&#226;&#8364;&#8220;1383. doi: 10. <br />                                                1038/ng.248 PMID: 18953339 <br />                                          16.   Redemann S, Schloissnig S, Ernst S, Pozniakowsky A, Ayloo S, Hyman AA, et al. (2011) Codon adap- <br />                                                tation-based control protein expression C. elegans. Nature methods 8: 250&#226;&#8364;&#8220;252. doi: 10.1038/ <br />                                                nmeth.1565 PMID: 21278743 <br />                                          17.   Praitis V, Casey E, Collar D, Austin J (2001) Creation low-copy integrated transgenic lines Caenor- <br />                                                habditis elegans. Genetics 157: 1217&#226;&#8364;&#8220;1226. PMID: 11238406 <br />                                          18.   Rual JF, Ceron J, Koreth J, Hao T, Nicot AS, Hirozane-Kishikawa T, et al. (2004) Toward improving <br />                                                Caenorhabditis elegans phenome mapping ORFeome-based RNAi library. Genome research <br />                                                14: 2162&#226;&#8364;&#8220;2168. PMID: 15489339 <br />                                          19.   Fraser AG, Kamath RS, Zipperlen P, Martinez-Campos M, Sohrmann M, Ahringer J (2000) Functional <br />                                                genomic analysis C. elegans chromosome I systematic <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span>. Nature 408: 325&#226;&#8364;&#8220;330. <br />                                                PMID: 11099033 <br />                                          20.   Timmons L, Tabara H, Mello CC, Fire AZ (2003) Inducible systemic RNA silencing Caenorhabditis <br />                                                elegans. Molecular biology cell 14: 2972&#226;&#8364;&#8220;2983. PMID: 12857879 <br />                                          21.   Bellanger JM, G&#195;&#182;nczy P (2003) TAC-1 ZYG-9 form complex promotes microtubule assembly <br />                                                C. elegans embryos. Current biology: CB 13: 1488&#226;&#8364;&#8220;1498. PMID: 12956950 <br />                                          22.   Edelstein A, Amodaj N, Hoover K, Vale R, Stuurman N (2010) Computer control microscopes using <br />                                                microManager. Current protocols molecular biology / edited Frederick M Ausubel [et al] Chapter <br />                                                14: Unit14 20. <br />              <br /> </body></html>