<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Lee et al. Microbial Cell Factories 2014, 13:61 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br />  RESEARCH                                                                                                                                     Open Access <br />  <br /> Genome-scale metabolic network reconstruction <br /> and silico flux analysis thermophilic <br /> bacterium Thermus thermophilus HB27 <br /> Na-Rae Lee1&#226;&#8364;&#160;, Meiyappan Lakshmanan2&#226;&#8364;&#160;, Shilpi Aggarwal2, Ji-Won Song1, Iftekhar A Karimi2, Dong-Yup Lee2,3* <br /> and Jin-Byung Park1,4* <br />  <br />  <br />   <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />   Background: Thermus thermophilus, extremely thermophilic bacterium, widely recognized model <br />   organism studying microbes survive adapt high temperature environment. However, the <br />   thermotolerant mechanisms cellular metabolism remains unravelled. Thus, highly required <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> <br />   consider systems biological approaches T. thermophilus metabolic network model employed together <br />   high throughput experimental data elucidating physiological characteristics harsh <br />   conditions. <br />   Results: We reconstructed genome-scale metabolic model T. thermophilus, iTT548, large-scale <br />   network thermophilic bacterium, accounting 548 unique genes, 796 reactions 635 unique metabolites. <br />   Our initial comparative analysis model Escherichia coli revealed distinctive metabolic reactions, <br />   mainly amino acid metabolism carotenoid biosynthesis, producing relevant compounds retain cellular <br />   membrane withstanding high temperature. Constraints-based flux analysis was, then, applied simulate the <br />   metabolic state glucose minimal amino acid rich media. Remarkably, resulting growth predictions were <br />   highly consistent experimental observations. The subsequent comparative flux analysis different <br />   environmental conditions highlighted cells consumed branched chain amino acids preferably utilized <br />   directly relevant anabolic pathways fatty acid synthesis. Finally, gene essentiality study also <br />   conducted single gene deletion analysis, identify conditional essential genes glucose minimal and <br />   complex media. <br />   Conclusions: The reconstructed genome-scale metabolic model elucidates phenotypes T. thermophilus, thus <br />   allowing gain valuable insights cellular metabolism silico simulations. The information obtained <br />   analysis shed light understanding physiology thermophiles helps to <br />   devise metabolic engineering strategies develop T. thermophilus thermostable microbial cell factory. <br />   Keywords: Thermus thermophilus, Thermophile, Genome-scale metabolic model, Constraints-based flux analysis, Ethanol <br />  <br />  <br />  <br />  <br /> * Correspondence: cheld@nus.edu.sg; jbpark06@ewha.ac.kr <br /> &#226;&#8364;&#160; <br />  Equal contributors <br /> 2 <br />  Department Chemical Biomolecular Engineering, National University <br /> of Singapore, 4 Engineering Drive 4, Singapore 117585, Singapore <br /> 1 <br />  Department Food Science &amp; Engineering, Ewha Womans University, 11-1 <br /> Daehyun-dong, Seodaemun-gu, Seoul 120-750, Korea <br /> Full list author information available end article <br />  <br />                                        &#194;&#169; 2014 Lee et al.; licensee BioMed Central Ltd. This Open Access article distributed terms Creative <br />                                        Commons Attribution License (http://creativecommons.org/licenses/by/4.0), permits unrestricted use, distribution, and <br />                                        reproduction medium, provided original work properly credited. The Creative Commons Public Domain <br />                                        Dedication waiver (http://creativecommons.org/publicdomain/zero/1.0/) applies data available article, <br />                                        unless stated. <br />  Lee et al. Microbial Cell Factories 2014, 13:61                                                                  Page 2 14 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br /> Background                                                      utilized postulate various strain improvement strategies <br /> Thermus thermophilus gram-negative, obligate aerobic       [17-19,24,25]. Thus, development T. thermophilus <br /> bacterium, representing best-studied ther-           GSMM based currently available biochemical and <br /> mophiles. It usually colonizes terrestrial volcanic hot     genomic information subsequent silico analysis <br /> springs (grows optimally 65 72&#194;&#176;C)           enables elucidate unique metabolic behaviour. <br /> originally isolated Japanese thermal spa [1]. In           In thermophilic microbes regard, a <br /> addition ability surviving high tempera-      initial attempts model cellular metabolisms. <br /> tures, T. thermophilus resistant stress     First, silico model Thermotoga maritima pre- <br /> harsh chemical conditions [2]. These properties motivated       sented, covering central metabolism 3D <br /> researchers extract isolate numerous proteins        structures enzymes accounted network <br /> T. thermophilus, making model organism struc-        [26]. Recently, genome-scale metabolic model of <br /> tural genomics significant industrial potential [3-6].     thermophilic archeon, Sulfolobus solfataricus, also <br /> For example, thermostable proteins used     developed, used autotrophic growth <br /> in commercial processes, including DNA polymerase        bicarbonate hydroxypropionate-hydroxybutyrate <br /> PCR techniques, &#206;&#177;-amylases glucose isomerases            cycle aerobic conditions [27]. However, models <br /> starch processing, esterases, lipases proteases          mature explain molecular mecha- <br /> organic synthesis, xylanases paper pulp              nisms high temperature adaptations con- <br /> manufacturing [7,8]. Moreover, T. thermophilus         sider detailed biosynthetic machinery biomolecules <br /> recognized potential microbial cell factory low    help retain integrity cell wall <br /> cost ethanol fermentation lignocellulosic waste ma-        membranes. Therefore, work, reconstructed the <br /> terials grow utilizing C5/C6        genome-scale metabolic model T. thermophilus based <br /> carbon sources relatively high temperatures, i.e. 70&#226;&#8364;&#8220;80&#194;&#176;     genome annotation HB27 wild-type strain [28] <br /> C, reducing energy costs: cooling step re-       investigating unique characteristics thermophilic <br /> quired following enzymatic hydrolysis, rendering easier      microbes. Additionally, model functionally char- <br /> to distil subsequent fermentations [9].                         acterized gene essentiality studies identify essential <br />    Despite enormous potentials biotechnological ap-         genes cellular growth growing glucose <br /> plications, current knowledge unique cellular        minimal amino acid supplemented complex media. <br /> physiology T. thermophilus limited; date, <br /> the production distinctive carotenoid molecules [10]         Results <br /> and use adaptive protein synthesis strategies [11]       Reconstruction T. thermophilus genome-scale <br /> are notable traits unravelled molecular         metabolic network <br /> level. Such limited studies technical     The genome-scale metabolic network T. thermophilus <br /> difficulties cultivating analysing thermophilic mi-      HB27 reconstructed step procedure: <br /> crobes; cell culture experiments require high         (1) construction draft network compilation genes, <br /> energy maintain optimal growth conditions. Hence,        reactions pathway information biochemical data- <br /> it required develop systematic approaches     bases based genome annotation T. thermophilus <br /> for improving understanding T. thermophilus cellu-       HB27, (2) manual curation draft model verifying <br /> lar behaviour. In regard, constraints-based silico      elemental balances reactions assigning proper <br /> metabolic modeling analysis considered        gene-reaction relationships, (3) gap filling using <br /> of promising techniques characterize physio-         organism specific knowledge (see Methods). During the <br /> logical behaviour metabolic states organism           reconstruction process, significant efforts highly re- <br /> upon various environmental/genetic changes sys-         quired identify resolve network gaps across <br /> tematically capture genotype-<span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> relationships        various metabolic pathways. Such gaps exist the <br /> <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> entire genome information [12,13]. As result,        incomplete genome annotations result missing <br /> several genome-scale metabolic models (GSMMs)           biochemical reactions dead ends. These gaps be <br /> available describing metabolic organization vari-    appropriately filled addition new reactions based <br /> ous organisms including Escherichia coli [14], Bacillus         information obtained literature inferred by <br /> subtilis [15], Saccharomyces cerevisiae [16], Pichia pastoris   genome annotation organisms. For example, <br /> [17], Corynebacterium glutamicum [18], Ralstonia eutro-         initial model contained metabolic gaps the <br /> pha [19], Pseudomonas aeruginosa [20],             synthetic pathway thermozeaxanthin thermobis- <br /> multicellular eukaryotes Mus musculus [21]          zeaxanthin, unique type carotenoids found <br /> Homo sapiens [22]. Moreover, availability           thermophiles, enabling cellular membrane to <br /> several conveniently accessible constraints-based model-        retain fluidity high temperatures [29-31]. <br /> ing software tools [23], models largely         Therefore, order gaps, added reactions <br />  Lee et al. Microbial Cell Factories 2014, 13:61                                                                        Page 3 14 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br /> corresponding glycosyltransferase acyltransferase en- <br /> zymes Staphylococcus aureus subsp. aureus 11819&#226;&#8364;&#8220;97 <br /> and Halobacillus halophilus DSM 2266 sequential <br /> glycosylation esterification zeaxanthin glucose <br /> and branched-chain fatty acids, producing thermozeax- <br /> anthin. Similarly, draft model gaps in <br /> the carbon assimilatory pathways, unable to <br /> consume carbon sources, namely, trehalose, palatinose, <br /> isomaltose, cellobiose, glutamate mannose. However, <br /> earlier studies reported T. thermophilus grow <br /> on carbon sources [32,33]. Such discrepancies <br /> were resolved adding new reactions corresponding <br /> to &#206;&#177;-glucosidase mannokinase enzymes based <br /> on information available KEGG MetaCyc <br /> databases. Overall, included 74 new reactions 63 <br /> enzymes, improving network connectivity <br /> (see Additional file 1 complete list reactions added). <br /> The gap filling draft model followed identifi-      Figure 1 Distribution reactions genes various <br /> cation genetic evidence newly added enzymes            metabolic subsystems iTT548. <br /> via sequence-based homology searches. For purpose, <br /> a BLASTp search performed NCBI database for <br /> the enzymes resolve network gaps using            comparison, 363 enzymes/genes common <br /> their amino acid sequences collected various          E. coli T. thermophilus, belonging to <br /> organisms non-redundant protein sequences            central metabolic pathways glycolysis, pentose <br /> of T. thermophilus HB27 genome. In way, 63         phosphate pathway TCA cycle. However, certain <br /> enzymes added, assign putative locus 10           notable differences observed amino acid <br /> of them, providing possible new annotations (see            synthetic pathways T. thermophilus. For instance, lysine <br /> Additional file 1 list new annotations). The          synthesized alpha-aminoadipate pathway instead <br /> final genome-scale metabolic network T. thermophilus,         diaminopimelate pathway. Similarly, upstream of <br /> iTT548, contains 548 unique genes (ORF coverage &#226;&#8364;&#8220;                methionine synthetic pathway conserved E. <br /> 24%), 796 reactions 635 unique metabolites. In               coli: precursor molecule, homocysteine, produced <br /> iTT548, 796 reactions classified seven         O-acetyl-L-homoserine hydrogen sulphide via <br /> major metabolic subsystems: carbohydrates, amino acid,           O-acetyl-L-(homo) serine sulfhydrylase T. thermophilus <br /> energy cofactors, lipids, nucleotides, carotenoids       O-succinyl-L-homoserine cysteine through <br /> transport. Among them, amino acid metabolism largest         O-succinyl-homoserine lyase E. coli. The comparison <br /> number reactions genes, followed carbohydrates         transport reactions revealed T. <br /> and energy cofactors metabolism (Figure 1). The de-          thermophilus lacks phosphoenolpyruvate-dependent <br /> tailed list completely curated T. thermophilus HB27           phosphotransferase (PTS), typical bacterial trans- <br /> metabolic network containing various genes, reactions,       port system, consumes carbohy- <br /> and metabolites obtained Additional file 1,          drates including glucose ATP-binding cassette (ABC) <br /> and available Systems Biology Markup Language            transporters. <br /> (SBML) file (level 2, version 1, http://sbml.org/) (Additional      Interestingly, unique feature T. thermophilus, <br /> file 2).                                                         iTT548 contains necessary biosynthetic machinery <br />                                                                  synthesizing molecules help in <br /> Network characteristics iTT548 comparison        habituating high temperatures. Unlike gram- <br /> E. coli GSMM                                                     negative bacteria, T. thermophilus does contain lipo- <br /> Figure 2A presents overall network features iTT548        polysaccharides cell outer membrane [34]. Instead, it <br /> and comparison E. coli using metabolic model        embeds complex carotenoid glucoside esters various <br /> (iAF1260) [14] terms EC numbers. Here,              branched chain fatty acids, known thermozeaxanthins <br /> should noted unable fairly compare        thermobiszeaxanthins, lipid bilayers. Such an <br /> metabolic characteristics T. thermophilus          arrangement offers multiple advantages including the <br /> thermophiles T. maritima model limited         retention membrane fluidity high temperatures <br /> to central metabolism S. solfataricus GSMM             reduction oxygen diffusion membrane <br /> was accessible link provided. From            preventing oxidation damage [10,35]. Furthermore, <br />  Lee et al. Microbial Cell Factories 2014, 13:61                                                                                       Page 4 14 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br />  Figure 2 Metabolic organization biomass composition T. thermophilus E. coli. (A) General features iTT548 comparison <br />  E. coli iAF1260 GSMM (Feist et al. 2007), (B) Central metabolic network T. thermophilus, (C) amino acid composition (mol%) (D) fatty <br />  acid composition (mol%). The numbers Venn diagram represents enzymes organism. The common unique pathways T. <br />  thermophilus highlighted blue red backgrounds, respectively. The number unique common enzymes identified using the <br />  EC numbers. The biomass data E. coli obtained iAF1260 GSMM. See supplementary 1 metabolite enzyme abbreviations used <br />  network diagram. <br />  <br />  <br /> T. thermophilus synthesizes unique polyamines                      branched chain fatty acids amino acids as <br /> such thermine, spermine, thermospermine caldo-                      valine, leucine isoleucine ketoisovalerate oxi- <br /> pentamine using distinct pathway L-arginine                     doreductase (vorA) gram-positive bacteria as <br /> aminopropyl agmatine [36]. These polyamines essen-                     Bacillus. Table 1 list unique carotenoid poly- <br /> tial high temperature protein synthesis ensuring                amine molecules, relevant genes corresponding <br /> proper structure formation initiation complex                       synthetic pathways accounted iTT548; Figure 2 illus- <br /> among 30S ribosomal subunit, messenger,                        trates central metabolic network T. thermophilus <br /> initial aminoacyl-tRNA. As notable exception gram-                    branched chain fatty acids carotenoid syn- <br /> negative bacterium, T. thermophilus synthesizes                       thetic pathways highlighted. <br />  Lee et al. Microbial Cell Factories 2014, 13:61                                                                                Page 5 14 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br /> Table 1 Biosynthetic machinery unique molecules T. thermophilus <br /> Molecules                                                 No.    Genes involved                              Major metabolic <br />                                                           reactions                                             precursors <br /> Branched fatty acids                                      19       ilvE (TTC1870), vorA (TTC1756) vorB      Leucine, valine, isoleucine <br />                                                                    (TTC1757), FabF (TTC0049 TTC0045), FabG   acetyl-coA <br /> (i-15:0, ai-15:0, i-16:0, i-17:0, ai-17:1)                         (TTC0047 TTC0394), FabZ (TTC1463), FabI <br />                                                                    (TTC0343) <br /> Carotenoids                                               18       CrtE (TTC1986), CrtB (TT_P0057), CrtI         Isopentenyl diphosphate <br />                                                                    (TT_P0066), CrtY (TT_P0060), CrtZ (TT_P0059), (IPP), UDP-glucose and <br /> (thermocryptoxanthin, thermozeaxanthin and <br />                                                                    CruC (TT_P0062), CruD (TT_P0061)              branched fatty acids <br /> thermobiszeaxanthin) <br /> Polyamines                                                16       SpeA (TTC1277), SpeB (TTC0764 TTC1132), Putrescine, spermidine and <br />                                                                    SpeD1 (TTC0473), SpeD2 (TTC1093), SpeE     spermine <br /> (1,3-diaminopropane, norspermidine, sym-homospermidine,            (TTC0472) <br /> thermine, thermospermine, homospermine, caldopentamine, <br /> thermopentamine, homocaldopentamine, caldohexamine, <br /> homocaldohexamine, tris(3-aminopropyl)amine, <br /> tetrakis(3-aminopropyl)ammonium) <br />  <br />  <br />   In iTT548, included biomass equation               70&#194;&#176;C. It noted medium supple- <br /> based amino acid compositional analysis              mented 20 amino acids. The <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> consump- <br /> data obtained literature. Importantly, biomass            tion profiles indicated glucose consumed first. <br /> composition carefully formulated avoid               Subsequently, carbohydrates trehalose and <br /> erroneous conclusions flux balance analysis                amino acids assimilated (Figure 3B). Notably, cells <br /> [37]. It noted earlier thermophile               did consume amino acid supplemented the <br /> models, T. maritima S. solfataricus, adopted bio-           medium preferred branched chain amino acids ahead <br /> mass equation E. coli M. bakeri, respectively,             amino acids, possibly synthesize branched <br /> both grow 37&#194;&#176;C optimal grow-             chain fatty acids (Figures 3C D). Furthermore, the <br /> ing temperature range thermophiles (50 ~ 80&#194;&#176;C). In               complex medium did appreciable lag phase <br /> this regard, compositions amino acids, val-             cells grew twice fast minimal <br /> ine, lysine, threonine, lysine, glutamine isoleucine,           media. <br /> are distinctive T. thermophilus E. coli              To simulate cellular growth minimal medium, <br /> (Figure 2C). Similarly, fatty acid compositions T.           biomass equation maximized flux analysis <br /> thermophilus different E. coli; linear           simulations simultaneously constraining glucose <br /> chain fatty acids composition negligible              uptake rates measured exponential phase based <br /> T. thermophilus branched-chain fatty acids                assumption wild-type organisms typically <br /> such iso-C17:0 iso-C15:0, present              evolve maximization cellular growth during <br /> in E. coli, contribute bulk total lipid compositions         exponential phase [38]. The exchange fluxes NH3, <br /> (Figure 2D). Collectively, results highlight need         phosphate, sulphite, H2O, Fe2+, Mg H+ left <br /> for careful estimation biomass equation mod-           unconstrained provide basic nutrients minerals for <br /> eling thermophiles.                                                 cell growth. The oxygen uptake rate constrained at <br />                                                                     average specific uptake rate 10 mmol g&#226;&#710;&#8217;1 DCW <br /> Model validation using minimal complex media                    hr&#226;&#710;&#8217;1 based previous publication [39]. Additionally, <br /> during batch cultures                                               lactate exchange flux constrained the <br /> We validated iTT548 using data batch cultures              measured uptake/secretion rates phase. A growth <br /> of T. thermophilus growing glucose minimal                   associated maintenance (GAM) value 58.34 mmol g&#226;&#710;&#8217;1 <br /> complex media. In case glucose minimal medium,                   DCW h&#226;&#710;&#8217;1, E. coli GSMM, NGAM requirement <br /> cells cultured DMM containing 0.6% (w/v) glu-             14 mmol g&#226;&#710;&#8217;1 DCW h&#226;&#710;&#8217;1, calculated based established <br /> cose 70&#194;&#176;C. The residual concentration glucose             methods [14], used simulations (see <br /> the cell density monitored (Figure 3A). Initially,           Additional file 3 detailed calculation NGAM calcu- <br /> prolonged lag phase 2 d observed, followed             lations). Here, noted comparing <br /> exponential growth phase 8 h stationary               NGAM requirement T. thermophilus E. coli <br /> phase reached. The cell cultures indicated             [14], possess high value (14 compared of <br /> presence acetate, lactate ethanol trace amounts           8 mmol g&#226;&#710;&#8217;1 DCW h&#226;&#710;&#8217;1) possibly differences in <br /> during lag exponential growth phases. In order               H+ permeability cytoplasmic membrane optimal <br /> analyse growth behaviour T. thermophilus rich           growth temperature; T. thermophilus cytoplasmic mem- <br /> medium, cells grown complex TM medium                 brane highly permeable H+, thus, leaks protons <br />  Lee et al. Microbial Cell Factories 2014, 13:61                                                                                    Page 6 14 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br />  Figure 3 Batch fermentation profile optical density various nutrients glucose minimal complex medium. (A) Profiles of <br />  optical density residual concentrations glucose, acetate lactate glucose minimal medium, (B) optical density residual <br />  concentrations glucose, trehalose, lactate acetate complex medium, (C) concentrations amino acids rapidly consumed in <br />  complex media (D) amino acids completely consumed. Highlighted regions correspond exponential growth phases of <br />  cultures corresponding <span id='am-3' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> consumption/secretion profiles used silico simulations. <br />  <br />  <br />  <br /> without ATP synthesis ATPase [40]. Notably, simula- <br /> tion results highly consistent observed growth <br /> rates (Figure 4). For complex media, the <br /> biomass equation maximized simultaneously <br /> constraining uptake/secretion rates nutrients <br /> (glucose, trehalose, lactate, acetate amino acids) <br /> during exponential growth phase. The silico pre- <br /> dicted cell growth 0.66 h&#226;&#710;&#8217;1 close (within the <br /> acceptable error range 10%) experimentally <br /> observed specific growth rate 0.64 h&#226;&#710;&#8217;1, clearly <br /> indicating high predictive ability iTT548 in <br /> complex media. <br />  <br /> In silico comparative metabolic flux analysis minimal <br /> and complex media <br /> When comparing cellular growth rates T. thermophi-                  Figure 4 Comparison silico growth rate experimentally <br />                                                                            observed growth rate exponential phase cell culture <br /> lus minimal complex media, surprisingly, was <br />                                                                            glucose minimal medium. <br /> significantly higher owing availability of <br />  Lee et al. Microbial Cell Factories 2014, 13:61                                                                    Page 7 14 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br /> rich carbon nitrogen sources form various           media, (ii) conditionally essential &#226;&#8364;&#8220; required one <br /> amino acids. Since microbe            media (iii) non-essential &#226;&#8364;&#8220; dispensable both <br /> amino acids directly medium, possible        media. The gene essentiality study revealed 23.5% <br /> that protein biosynthetic demand partially ful-        19.5% total 548 genes iTT548 essential <br /> filled. In order confirm hypothesis, compared          cell growth minimal complex media, respect- <br /> the simulated metabolic fluxes minimal (phase P1)         ively (see Additional file 4 complete list essential <br /> and complex media, observed lower fluxes                 genes). A total 107 genes essential condi- <br /> through relevant biosynthetic reactions amino acids        tions additional 21 genes essential in <br /> in complex medium (Table 2). Herein,             minimal media. In order understand knock- <br /> noted conducted flux variability analysis        functional category genes crucial <br /> [41] re-assure confidence simulated metabolic       cell viability condition, identified the <br /> states environmental condition.                           distribution essential genes various metabolic <br />    Interestingly, 20 amino acids supple-        processes (Figure 5). Interestingly, carotenoids metab- <br /> mented complex medium, cells consumed             olism contained lethal genes (82%), suggesting <br /> a preferably. In order understand cel-        unique pathway synthesizing thermozeax- <br /> lular behavior, compared uptake rate individual         anthin thermobiszeaxanthin does al- <br /> amino acids actual biosynthetic demands.               ternative routes quite rigid T. thermophilus. <br /> Methionine cysteine consumed medium             Following carotenoids metabolism, nucleotides and <br /> according biosynthetic demands alanine,            lipids metabolism second highest number of <br /> valine, leucine, isoleucine glutamate consumed           completely essential genes (32.7% 24.6%, respectively). <br /> in excess individual biosynthetic requirements           On hand, examination conditionally essential <br /> (Table 3). Further analysis revealed surplus valine/     genes, i.e. genes essential minimal <br /> leucine/isoleucine alanine/glutamate utilized         media, revealed amino acid metabolism contains <br /> synthesize branched chain fatty acids amino         genes (Figure 5). Since complex media <br /> acids, respectively. Notably, leucine consumed           supplemented amino acids, them <br /> more biosynthetic demand (almost 7 times                 directly consumed media utilizing <br /> higher) mainly synthesize iso-15 iso-17 fatty acids,       biosynthetic pathways, classifying genes <br /> which constitute major composition T. thermophi-           pathways non-essential condi- <br /> lus. Remarkably, amino acid utilization <span id='am-242' about='xsp:pattern' typeof='owl:Thing'>pattern</span>           tions. However, time, distribution certain <br /> completely different E. coli grown complex media,         completely essential genes amino acid metabolism also <br /> which consumed serine, aspartate, glycine threonine           indicates biosynthetic pathways tryptophan, proline <br /> ahead amino acids [42]. The subsequent silico         tyrosine crucial cell growth they <br /> analysis E. coli revealed rapidly depleted serine     synthesized cell albeit availabil- <br /> was primarily converted pyruvate serine deaminase,       ity complex medium. Interestingly, gene dele- <br /> and acetate producing ATP similarly           tion analyses showed high number completely <br /> observed microbes C. glutamicum          essential genes oxidative phosphorylation and <br /> and Lactococcus lactis. In regard, interesting      TCA cycle. Oxygen key electron acceptor cellular <br /> note T. thermophilus possess unique <span id='am-5' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> con-            metabolism accept electrons redox <br /> sumption pattern, preferably synthesizing branched chain          cofactors involved TCA cycle generates energy <br /> fatty acids improving energy production further.      oxidative phosphorylation. If oxygen devoid or <br /> Overall, silico analysis highlights consumed   oxidative phosphorylation TCA cycle reac- <br /> amino acids contribute directly indirectly      tions perturbed, generally, bacteria regen- <br /> biomass synthesis, strategies increase uptake non-      erate redox cofactors switching fermentative <br /> consumed amino acids tyrosine, lysine, tryptophan         growth help substrate level phosphorylation. <br /> and histidine postulated enhance cellular           In regard, T. thermophilus lacks PTS and <br /> growth complex medium.                                         utilizes ABC transporters glucose uptake, could <br />                                                                   possible microbe switch fermenta- <br /> Analysis essential genes T. thermophilus                       tive metabolism completely thus, requires oxygen to <br /> We analysed essentiality individual genes T.        generate sufficient energy cell growth TCA cycle <br /> thermophilus glucose minimal complex TT                 oxidative phosphorylation. <br /> medium conditions using iTT548 model deleting every <br /> gene time (see Methods). The genes             Discussion <br /> categorized classes: (i) completely essential &#226;&#8364;&#8220;        Thermophilic microbes represent unique class organ- <br /> genes required cellular growth          isms distinct cell wall assembly enables them <br />                                                                                                                                                                                                                           http://www.microbialcellfactories.com/content/13/1/61 <br />                                                                                                                                                                                                                          Lee et al. Microbial Cell Factories 2014, 13:61 <br /> Table 2 Comparison metabolic reaction fluxes amino acids biosynthetic reactions minimal complex media <br /> Reaction ID Enzyme (Gene)                                              Reaction details                                                              Complex media                  Minimal media <br />                                                                                                                                                           (LB &lt; L &lt; UB)                  (LB &lt; L &lt; UB) <br /> R192           Alanine dehydrogenase (ald)                                  H[c] + NADH[c] + PYR[c] + NH4[c] &lt; = &gt; L_ALA[c] + NAD[c] + H2O[c] &#226;&#710;&#8217;0.031 &lt; &#226;&#710;&#8217;0.031 &lt; &#226;&#710;&#8217;0.031 0.234 &lt; 0.234 &lt; 2.416 <br /> R521           Asparagine synthase (asnB)                                   L_ASP[c] + L_GLN[c] + ATP[c] + H2O[c] &lt; = &gt; L_GLU[c] + L_ASN[c] +             &#226;&#710;&#8217;1.808 &lt; &#226;&#710;&#8217;1.808 &lt; &#226;&#710;&#8217;1.783       &#226;&#710;&#8217;0.537 &lt; &#226;&#710;&#8217;0.537 &lt; &#226;&#710;&#8217;0.522 <br />                                                                             PPi[c] + H[c] + AMP[c] <br /> R198           Aspartate aminotransferase (aspC)                            L_ASP[c] + AKG[c] &lt; = &gt; L_GLU[c] + OAC[c]                                     &#226;&#710;&#8217;1.782 &lt; &#226;&#710;&#8217;1.782 &lt; &#226;&#710;&#8217;1.782       &#226;&#710;&#8217;0.416 &lt; &#226;&#710;&#8217;0.416 &lt; &#226;&#710;&#8217;0.416 <br /> R205           Glutamine synthetase (gln)                                   L_GLU[c] + ATP[c] + NH4[c] - &gt; L_GLN[c] + ADP[c] + Pi[c] + H[c]               &#226;&#710;&#8217;1.793 &lt; &#226;&#710;&#8217;1.793 &lt; &#226;&#710;&#8217;1.793       &#226;&#710;&#8217;0.474 &lt; &#226;&#710;&#8217;0.474 &lt; &#226;&#710;&#8217;0.474 <br /> R202           Glutamate dehydrogenase (gdh)                                L_GLU[c] + NADP[c] + H2O[c] &lt; = &gt; H[c] + NADPH[c] + AKG[c] + NH4[c]           &#226;&#710;&#8217;9.830 &lt; &#226;&#710;&#8217;9.145 &lt; 4.713        &#226;&#710;&#8217;9.288 &lt; &#226;&#710;&#8217;8.813 &lt; 2.977 <br /> R227           Cysteine synthase (cysM)                                     OAC_SER[c] + H2S[c] - &gt; L_CYS[c] + H[c] + AC[c]                               0&lt;0&lt;0                          0 &lt; 0.005 &lt; 0.005 <br /> R235           Methionine synthase (metH)                                   L_HCYS[c] + 5MTHF[c] - &gt; L_MET[c] + H[c] + THF[c]                             0&lt;0&lt;0                          0 &lt; 0.009 &lt; 0.009 <br /> R272           N2-acetyl-L-lysine deacetylase (lysK)                        H2O[c] + N2AC_L_LYS[c] - &gt; L_LYS[c] + AC[c]                                   0.229 &lt; 0.229 &lt; 0.229          0.05 &lt; 0.05 &lt; 0.05 <br /> R651           3-phosphoserine phosphatase (serB)                           H2O[c] + PPTD_SER[c] - &gt; L_SER[c] + Pi[c]                                     0&lt;0&lt;0                          0 &lt; 1.072 &lt; 2.545 <br /> R655           Serine hydroxymethyltransferase (glyA)                       L_SER[c] + THF[c] - &gt; L_GLY[c] + H2O[c] + METHF[c]                            0&lt;0&lt;0                          0 &lt; 0.512 &lt; 6.42 <br /> R213           Serine hydroxymethyltransferase (glyA)                       L_GLY[c] + H2O[c] + 5_10_MNTHF[c] &lt; = &gt; L_SER[c] + THF[c]                     0.135 &lt; 0.135 &lt; 0.135          &#226;&#710;&#8217;1.787 &lt; &#226;&#710;&#8217;0.442 &lt; 3.993 <br /> R212           Tryptophan synthase (trpA)                                   L_SER[c] + IND_GLYP[c] &lt; = &gt; L_TRP[c] + H2O[c] + GAP[c]                       0.054 &lt; 0.054 &lt; 0.054          0.026 &lt; 0.026 &lt; 0.026 <br /> R221           Threonine synthase (thrC)                                    H2O[c] + OP_HSER[c] - &gt; L_THR[c] + Pi[c]                                      0.0714 &lt; 0.0714 &lt; 0.0714       0.063 &lt; 0.063 &lt; 0.063 <br /> R245           Branched-chain amino acid aminotransferase (ilvE) L_VAL[c] + AKG[c] &lt; = &gt; L_GLU[c] + 2K3M_BUT[c]                                           0.113 &lt; 0.113 &lt; 0.113          &#226;&#710;&#8217;0.068 &lt; &#226;&#710;&#8217;0.068 &lt; &#226;&#710;&#8217;0.068 <br /> R244           Branched-chain amino acid aminotransferase (ilvE) L_GLU[c] + 4M_2OPENTN[c] &lt; = &gt; L_LEU[c] + AKG[c]                                         &#226;&#710;&#8217;1.179 &lt; &#226;&#710;&#8217;1.179 &lt; &#226;&#710;&#8217;1.179 0.154 &lt; 0.154 &lt; 0.154 <br /> R246           Branched-chain amino acid aminotransferase (ilvE) L_ILE[c] + AKG[c] &lt; = &gt; L_GLU[c] + 3M_2OPENTN[c]                                         0.478 &lt; 0.478 &lt; 0.478          &#226;&#710;&#8217;0.024 &lt; &#226;&#710;&#8217;0.024 &lt; &#226;&#710;&#8217;0.024 <br /> R341           Pyrroline-5-carboxylate reductase (ProC)                     L_PRO[c] + NADP[c] &lt; = &gt; 2 H[c] + NADPH[c] + PYRR5CARB[c]                     &#226;&#710;&#8217;0.432 &lt; &#226;&#710;&#8217;0.432 &lt; &#226;&#710;&#8217;0.432       &#226;&#710;&#8217;0.119 &lt; &#226;&#710;&#8217;0.119 &lt; &#226;&#710;&#8217;0.119 <br /> R195           Argininosuccinate lyase (argH)                               L_ARG_SUCC[c] - &gt; L_ARG[c] + FUM[c]                                           1.144 &lt; 1.144 &lt; 1.144          0.132 &lt; 0.132 &lt; 0.132 <br /> R297           Aspartate aminotransferase (aat)                             L_PHE[c] + AKG[c] &lt; = &gt; L_GLU[c] + PPYR[c]                                    0&lt;0&lt;0                          &#226;&#710;&#8217;0.054 &lt; &#226;&#710;&#8217;0.054 &lt; &#226;&#710;&#8217;0.054 <br /> R284           Histidinol dehydrogenase (hisD)                              2 NAD[c] + H2O[c] + L_HISN[c] - &gt; L_HIS[c] + 3 H[c] + 2 NADH[c]               0.025 &lt; 0.025 &lt; 0.025          0.015 &lt; 0.015 &lt; 0.015 <br /> The reactions highlighted bold possess flux complex media. FVA conducted re-assure simulated fluxes results provided attainable lower upper bounds identified <br /> from analysis. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                        Page 8 14 <br />  Lee et al. Microbial Cell Factories 2014, 13:61                                                                                          Page 9 14 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br /> Table 3 Consumption production pattern amino acids complex media <br /> Amino acids     Supply    Biosynthetic    Contribution <br />                 medium         demand <br />                                                To     From     Remarks <br /> Alanine         0.7399         0.355           0.3853                        Surplus alanine contributes glycine synthesis <br /> Glycine         0.116          0.308                         0.1918          Some produced alanine <br /> Valine          0.224          0.109           0.1151                        Excess valine utilized synthesis anteiso-17:1 anteiso-15:0 <br />                                                                              fatty acids <br /> Leucine         1.427          0.244           1.1827                        Excess leucine utilized iso-15:0 iso-17:0 fatty acids synthesis <br /> Isoleucine      0.5181         0.039           0.4794                        Excess isoleucine utilized synthesis anteiso fatty acid synthesis <br /> Proline         &#226;&#710;&#8217;0.24          0.189                         0.4291          Synthesized glutamate <br /> Methionine      0.015          0.015                                         Utilized media biosynthetic demand <br /> Serine          0.057          0.135                         0.0785          Synthesized glycine <br /> Threonine       0.03           0.100                         0.0699          Synthesized aspartate <br /> Phenylalanine   0.086          0.086                                         Utilized media biosynthetic demand <br /> Aspartate       &#226;&#710;&#8217;0.12          0.103                         0.2232          Synthesized oxaloacetate glutamate <br /> Cysteine        0.0087         0.009                                         Utilized media biosynthetic demand <br /> Glutamate       1.48           0.176           1.3041                        Excess glutamate utilized amino acid synthesis <br /> Arginine        &#226;&#710;&#8217;0.93          0.211                         1.1409          Synthesized aspartate <br /> Asparagine      &#226;&#710;&#8217;0.023         0.103                         0.1262          Synthesized aspartate <br /> Lysine          &#226;&#710;&#8217;0.141         0.087                         0.2283          Synthesized oxoglutarate <br /> Glutamine       0              0.177                         0.1771          Synthesized glutamate <br /> Histidine       0              0.024                         0.0245          Synthetsized PRPP <br /> Tyrosine        &#226;&#710;&#8217;0.03204       0.069                         0.1007          Synthesized PEP, E4P <br /> Tryptophan      &#226;&#710;&#8217;0.01116       0.043                         0.0538          Synthesized serine <br />  <br />  <br />  <br /> to maintain cellular membrane integrity                 temperatures [10]. In regard, iTT548 completely cap- <br /> high temperatures. It reported thermophilic                 tures biosynthetic pathways thermozeaxanthins, <br /> organisms, including T. maritima S. solfataricus,                    addition metabolic routes biomass <br /> synthesize ether lipids long chain dicarboxylic                     precursors amino acids, nucleotides lipids. <br /> fatty acids fatty alcohols [43]. However, Thermus                    Similarly, iTT548 contains unique biosynthetic <br /> sp. synthesize ether lipids produce unique                    pathways unusual polyamines help in <br /> carotenoid molecules thermozeaxanthin                        stabilizing nucleotide strands proteins synthesis <br /> thermobizeaxanthin, embed lipid bi-layer                 high temperatures. It earlier reported that <br /> to attain required cellular membrane fluidity high                T. thermophilus unique polyamine synthesis: even <br />  <br />  <br />  <br />  <br />  Figure 5 Distribution essential genes T. thermophilus metabolic subsystems. Black, grey white colors indicate completely-, <br />  conditionally- non-essential genes, respectively. The numbers parenthesis represent number genes subsystem. <br />  Lee et al. Microbial Cell Factories 2014, 13:61                                                                           Page 10 14 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br /> the extreme thermophiles S. solfataricus pro-                low temperature range, hypothesize that <br /> duces relatively shorter polyamines [36]. Collectively,              thermophiles likely adjust biomass <br /> these results clearly detailed metabolic cover-             composition selectively better adapt growth <br /> age iTT548 thermophiles comparing                environment. Therefore, use appropriate biomass <br /> preceding GSMMs.                                                     equations simulating cellular growth correspond- <br />   Furthermore, work includes prudently drafted                ing temperature ranges crucial reliable prediction. <br /> biomass equation specific thermophiles, espe-                The gene deletion analyses iTT548 revealed <br /> cially Thermus sp. As mentioned earlier, comparative             interesting traits function deleted <br /> analysis T. thermophilus E. coli biomass composi-             genes respect overall cellular metabolism T. <br /> tions highlighted significant differences               thermophilus. Among them, notable rela- <br /> amino acid fatty acid compositions. Noticeably,              tively high percentage essential genes compared <br /> T. thermophilus biomass analysis revealed con-              E. coli (23% 13%), possibly smaller <br /> centration thermolabile amino acids              Open Reading Frames (ORF) content (only 2,263 com- <br /> as threonine histidine substantially lesser E.          pared 4,623) despite possessing necessary modules <br /> coli proline concentration higher. It            cell viable high temperatures. Furthermore, <br /> should highlighted observations                 observation highlights fact T. ther- <br /> good agreement earlier reports suggested              mophilus thrives higher temperatures other <br /> selective usage amino acid residues key             microbes, fitness relatively competitive <br /> adaptive strategy employed thermophiles [44,45].                  rigid network organization. Interestingly, the <br /> Arguably, cellular compositions thermophilic                  gene essentiality analyses indicated carotenoids <br /> microbes change depending growth temperature;                 metabolism functionally quite fragile of <br /> the current biomass equation derived based com-               genes essential cellular growth. However, it <br /> positional analysis T. thermophilus grown 70&#194;&#176;C. In             noted gene deletion analysis results are <br /> order clarify temperature dependent compositional             sensitive parameters silico medium <br /> change biomass, measured amino acid compositions               setup biomass composition. In regard, current <br /> in T. thermophilus 45&#194;&#176;C. Their comparison com-               biomass composition obtain T. thermophilus at <br /> positional data 70&#194;&#176;C clearly indicated           optimum growth temperatures, i.e. 70&#194;&#176;C, gene <br /> significant difference individual overall amino          deletion results current study applicable to <br /> acid concentrations (Figure 6A). Similarly, com-             condition. <br /> pared fatty acid compositions 40&#194;&#176;C 70&#194;&#176;C <br /> using data literature [30]. Very interestingly,             Conclusions <br /> unlike amino acid comparison, fatty acid compositions,               We presented genome-scale metabolic network T. <br /> both overall individual lower 40&#194;&#176;C                  thermophilus, iTT548, representing ther- <br /> (Figure 6B). Although able make complete            mophiles, containing 548 unique genes, 796 reactions <br /> comparison low high growth temperatures                  635 unique metabolites. As unique feature of <br /> since data available cellular constituents           T. thermophilus, iTT548 contains necessary metabolic <br /> such peptidoglycans thermotolerant carotenoids             pathways synthesizing unique carotenoids <br />  <br />  <br />  <br />  <br />  Figure 6 Influence temperature T. thermophilus biomass composition. (A) amino acid composition (mol%) 70&#194;&#176;C 45&#194;&#176;C and <br />  (B) fatty acid composition (mol%) 70&#194;&#176;C 40&#194;&#176;C. <br />  Lee et al. Microbial Cell Factories 2014, 13:61                                                               Page 11 14 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br /> and polyamines help habituating high            butyldimethylsilyl-trifluoro-acetamide (MBDSTFA) in <br /> temperatures. The reconstructed metabolic model           DMF 80&#194;&#176;C 30 min. After centrifugation 14,000 g <br /> subsequently validated batch culture experiments     5 min, supernatant injected GC/MS split <br /> on glucose minimal complex medium            injection mode (1:10 split ratio). <br /> silico growth predictions iTT548 good <br /> agreement observed experimental results. The         Metabolic network reconstruction <br /> comparative flux analysis minimal complex         The genome-scale metabolic network T. thermophilus <br /> media highlighted consumption utilization     HB27 reconstructed using published genome <br /> branched chain amino acids directly relevant fatty     annotation [28] information collected vari- <br /> acids anabolic pathways, resulting higher growth      ous biological genomic databases basis the <br /> rates rich medium. A gene essentiality study       established procedure [48]. First, initial draft model <br /> also conducted silico simulation studies   constructed compiling annotated metabolic <br /> minimal complex media, highlighting high           genes corresponding biochemical reactions <br /> percentage lethal genes comparison E. coli,        KEGG [49] MetaCyc [50]. Then, reactions <br /> suggesting metabolic backbone T. thermo-          corrected elemental imbalances mapped <br /> philus quite rigid. Overall, metabolic        appropriate genes devise proper gene-protein- <br /> network presented current study expected         reaction (GPR) relationships. Additionally, spontan- <br /> be significant contribution systems analysis     eous non-gene-associated reactions including <br /> thermophiles metabolic model utilized        metabolite transport incorporated the <br /> along high throughput datasets better un-        model based physiological evidence literature <br /> derstanding organism.                                      databases. The connectivity draft network was <br />                                                               checked using GapFind algorithm the <br /> Methods                                                       gaps [51]. The identified missing links filled either <br /> Microorganism culture conditions                          introduction sink reactions allow material <br /> T. thermophilus HB27 strain used model organ-        exchange cell surrounding environ- <br /> ism. For fermentation complex medium, single colony      ment adding reactions similar microbes <br /> was cultivated overnight 70&#194;&#176;C 150 rpm 5 mL      close knowledge gaps. <br /> the TM medium [46], culture transferred a <br /> 500 mL baffled-flask containing 100 mL TM broth. In <br /> case cultivation defined glucose minimal medium,        Biomass composition <br /> the overnight seed grown 70&#194;&#176;C 150 rpm 5 mL         Cellular biomass composition important pre- <br /> of TM medium transferred 500 mL             requisite silico flux analysis, especially during <br /> baffled-flask containing 100 mL defined minimal            exponential growth phase, primary cellular <br /> medium (DMM) 0.6% (w/v) glucose cultivated           objective maximize growth. Amino acid composition <br /> for 24 hours. Then, 5 mL flask culture DMM in-      T. thermophilus HB27 estimated hydrolyzing <br /> oculated 100 mL fresh DMM. During fermentations,        cell pellets 6 N HCl 24 h 130&#194;&#176;C, subse- <br /> cell growth monitored measuring optical density    quently analysing hydrolysates using HPLC equipped <br /> at 600 nm. The dry cell weight (DCW) estimated       UV-detector C18 column. Cell wall lipid <br /> by predetermined conversion factor 0.34.                 compositions obtained previous publications <br />                                                               Thermus sp. [30,31,35,52]. The overall DNA RNA <br /> Analytical methods                                            composition assumed E. coli [14] since <br /> Concentrations glucose, organic acids ethanol       data available Thermus sp. The individual <br /> the culture broth measured high performance           weights nucleotides DNA RNA calcu- <br /> liquid chromatography (HPLC) (Waters, Milford, MA)            lated based reported G + C content 69.4% [28]. <br /> equipped HPX-87H column (Bio-Rad, Hercules,           Detailed information biomass composition calculations <br /> CA), dual &#206;&#187; absorbance detector. The collected samples      Additional file 3. <br /> were centrifuged 14,000 g 4&#194;&#176;C 5 min the <br /> supernatant analyzed column using 5 mM           Constraints-based flux analysis <br /> sulfuric acid mobile phase 0.6 mL&#226;&#710;&#8217;1 min. Concen-      We implemented constraints-based flux analysis to <br /> trations amino acids determined gas chromato-      simulate T. thermophilus metabolism varying <br /> graph/mass spectrophotometer (GC/MS) (Agilent, Santa          environmental conditions. The biomass reaction was <br /> Ciara, CA) equipped HP-5MS column (Agilent),          maximized simulate exponential growth phase <br /> as previously reported [47]. In brief, samples       described [53-55]. Mathematically, op- <br /> centrifuged, dried derivatized methyl-N-t-           timization problem, i.e. maximization biomass subjected <br />  Lee et al. Microbial Cell Factories 2014, 13:61                                                                                  Page 12 14 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br /> to stoichiometric capacity constraints, formu-           supplied carbon source based complex <br /> lated follows:                                                   media based <span id='am-6' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> consumption profile. The simu- <br />                X                                                    lation results subsequently analyzed identify the <br />     max Z &#194;&#188;       cj vj                                             essential genes gene classified essential <br />                        j <br />                                                                     resulting cell growth prediction correspond- <br />            X                                                        ing mutant equal 5% wild-type. Note <br />     s:t:          S ij vj &#194;&#188; 0 &#226;&#710;&#8364; metabolite i <br />             j <br />                                                                     gene deletion analysis study per- <br />                                                                     formed using COBRA toolbox. <br />     vmin <br />      j   &#226;&#8240;&#164; vj &#226;&#8240;&#164; vmax <br />                  j               &#226;&#710;&#8364; reaction j <br />                                                                     Additional files <br /> where Sij refers stoichiometric coefficient me- <br /> tabolite involved reaction j, vj denotes flux            Additional file 1: Details iTT548 containing genes, reactions, <br /> or specific rate metabolic reaction j, vmin j   vmax <br />                                                           j <br />                                                                      metabolites. A list reactions added gap-filling possible <br />                                                                      new annotations identified study provided. <br /> represent lower upper limits flux of <br />                                                                      Additional file 2: SBML file iTT548. <br /> reaction j, respectively; Z corresponds cellular <br />                                                                      Additional file 3: Biomass composition T. thermophilus HB27 and <br /> objective linear function metabolic reactions        NGAM calculations. <br /> where relative weights determined coeffi-             Additional file 4: List essential genes glucose minimal and <br /> cient cj. In study, constraints-based flux analysis         complex media. <br /> problems solved using COBRA toolbox [56]. <br />                                                                     Competing interests <br /> Flux variability analysis                                           The authors declare competing interests. <br /> As constraints-based flux analysis optimization <br />                                                                     Authors&#226;&#8364;&#8482; contributions <br /> based technique, possible multiple              NRL, ML, JBP DYL conceived designed study. NRL JWS <br /> flux distributions attaining physiological state.          performed batch culture experiments. NRL SA created draft <br /> Therefore, order confirm plausibility internal         model. ML refined model performed simulations. NRL, ML, IAK, <br />                                                                     JBP DYL wrote manuscript. JBP DYL coordinated and <br /> metabolic fluxes simulated minimal complex media             directed project. All authors read approved final <br /> by flux analysis, performed flux variability analysis        manuscript. <br /> (FVA) identify possible range fluxes while <br />                                                                     Acknowledgements <br /> simulating particular phenotypic state. Mathematically,           This work supported National University Singapore, Biomedical <br /> the optimization problem specific FVA repre-              Research Council A*STAR (Agency Science, Technology Research), <br /> sented follows:                                                  Singapore, Korea Research Foundation (KRF- 2010&#226;&#8364;&#8220;0009169), Republic of <br />                                                                     Korea, grant Next-Generation BioGreen 21 Program (SSAC, <br />      max= min              vj                                       No. PJ009520), Rural Development Administration, Republic Korea. <br />                            X <br />                 s:t:              S ij vj &#194;&#188; 0                       Author details <br />                                                                     1 <br />                                                                      Department Food Science &amp; Engineering, Ewha Womans University, 11-1 <br />                              j <br />                            X                                        Daehyun-dong, Seodaemun-gu, Seoul 120-750, Korea. 2Department of <br />                                   cj vj &#194;&#188; Z obj                     Chemical Biomolecular Engineering, National University Singapore, 4 <br />                              j                                      Engineering Drive 4, Singapore 117585, Singapore. 3Bioprocessing <br />                                                                     Technology Institute, Agency Science, Technology Research <br />                            vmin <br />                             j   &#226;&#8240;&#164; vj &#226;&#8240;&#164; vmax <br />                                         j         j &#194;&#188; 1; &#226;&#8364;&#166;; n   (A*STAR), 20 Biopolis Way, #06-01, Centros, Singapore 138668, Singapore. <br />                                                                     4 <br />                                                                      Global Top5 Research Program, Ewha Womans University, 11-1 <br />                                                                     Daehyeon-dong, Seodaemun-gu, Seoul 120750, Korea. <br /> where Zobj denotes value objective calculated by <br /> flux analysis n number fluxes. The upper              Received: 14 February 2014 Accepted: 17 April 2014 <br /> range fluxes identified maximizing objective           Published: 28 April 2014 <br /> whereas lower range obtained minimizing the <br />                                                                     References <br /> same. In study, FVA implemented using                  1. Oshima T, Imahori K: Description Themus thermophilus (Yoshida and <br /> COBRA toolbox.                                                          Oshima) comb. nov., nonsporulation thermophilic bacterium a <br />                                                                         Japanese thermal spa. Int J Syst Bacteriol 1974, 24:102&#226;&#8364;&#8220;112. <br />                                                                     2. Cava F, Hidalgo A, Berenguer J: Thermus thermophilus biological model. <br /> Gene deletion analysis                                                  Extremophiles 2009, 13:213&#226;&#8364;&#8220;231. <br /> Gene deletion simulations performed maximiz-                3. Wimberly BT, Brodersen DE, Clemons WM Jr, Morgan-Warren RJ, Carter AP, <br /> ing cellular biomass constraining flux                Vonrhein C, Hartsch T, Ramakrishnan V: Structure 30S ribosomal <br />                                                                         subunit. Nature 2000, 407:327&#226;&#8364;&#8220;339. <br /> the corresponding reaction(s) zero GPR re-            4. Sazanov LA, Hinchliffe P: Structure hydrophilic domain of <br /> lationships defined nutrient uptake rates. In case                respiratory complex I Thermus thermophilus. Science 2006, <br /> of glucose minimal medium, glucose fueled                   311:1430&#226;&#8364;&#8220;1436. <br />                                                                     5. Selmer M, Dunham CM, Murphy FV, Weixlbaumer A, Petry S, Kelley AC, Weir <br /> carbon source. On hand, glucose, trehalose                    JR, Ramakrishnan V: Structure 70S ribosome complexed with <br /> and amino acids valine, leucine isoleucine                  mRNA tRNA. Science 2006, 313:1935&#226;&#8364;&#8220;1942. <br />  Lee et al. Microbial Cell Factories 2014, 13:61                                                                                                      Page 13 14 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br /> 6.    Yokoyama K, Ohkuma S, Taguchi H, Yasunaga T, Wakabayashi T, Yoshida M:         28. Henne A, Bruggemann H, Raasch C, Wiezer A, Hartsch T, Liesegang H, <br />       V-type H + &#226;&#710;&#8217;ATPase/synthase thermophilic eubacterium, Thermus               Johann A, Lienard T, Gohl O, Martinez-Arias R, Jacobi C, Starkuviene V, <br />       thermophilus - Subunit structure operon. J Biol Chem 2000,                     Schlenczeck S, Dencker S, Huber R, Klenk HP, Kramer W, Merkl R, Gottschalk <br />       275:13955&#226;&#8364;&#8220;13961.                                                                   G, Fritz HJ: The genome sequence extreme thermophile Thermus <br /> 7.    Pantazaki AA, Pritsa AA, Kyriakidis DA: Biotechnologically relevant enzymes        thermophilus. Nat Biotechnol 2004, 22:547&#226;&#8364;&#8220;553. <br />       Thermus thermophilus. Appl Microbiol Biotechnol 2002, 58:1&#226;&#8364;&#8220;12.            29. Kaneda T: Iso- anteiso-fatty acids bacteria: biosynthesis, function, <br /> 8.    Niehaus F, Bertoldo C, Kahler M, Antranikian G: Extremophiles source          taxonomic significance. Microbiological Rev 1991, 55:288&#226;&#8364;&#8220;302. <br />       novel enzymes industrial application. Appl Microbiol Biotechnol 1999,   30. Nordstrom KM, Laakso SV: Effect growth temperature fatty acid <br />       51:711&#226;&#8364;&#8220;729.                                                                        composition Thermus strains. Appl Environ Microbiol 1992, <br /> 9.    Riyanti EI: Genetic manipulation thermophiles ethanol production.           58:1656&#226;&#8364;&#8220;1660. <br />       In PhD Thesis. The University New South Wales: School biotechnology      31. Pask-Hughes RA, Shaw N: Glycolipids extreme thermophilic <br />       biomolecular sciences; 2007.                                                   bacteria belonging geus Thermus. J Bacteriol 1982, 149:54&#226;&#8364;&#8220;58. <br /> 10.   Tian B, Hua Y: Carotenoid biosynthesis extremophilic Deinococcus&#226;&#8364;&#8220;Thermus    32. Alarico S, da Costa MS, Empadinhas N: Molecular physiological role of <br />       bacteria. Trends Microbiol 2010, 18:512&#226;&#8364;&#8220;520.                                       trehalose-hydrolyzing alpha-glucosidase Thermus thermophilus <br /> 11.   Shigi N, Suzuki T, Tamakoshi M, Oshima T, Watanabe K: Conserved bases           HB27. J Bacteriol 2008, 190:2298&#226;&#8364;&#8220;2305. <br />       TPsi C loop tRNA determinants thermophile-specific              33. Lengsfeld C, Schonert S, Dippel R, Boos W: Glucose- glucokinase- <br />       2-thiouridylation position 54. J Biol Chem 2002, 277:39128&#226;&#8364;&#8220;39135.               controlled mal gene expression Escherichia coli. J Bacteriol 2009, <br /> 12.   Lewis NE, Nagarajan H, Palsson BO: Constraining metabolic                      191:701&#226;&#8364;&#8220;712. <br />       genotype-phenotype relationship using phylogeny silico                 34. Holst O: Structure Lipopolysaccharide Core Region. In Bacterial <br />       methods. Nat Rev Microbiol 2012, 10:291&#226;&#8364;&#8220;305.                                       Lipopolysaccharides. Edited Knirel YA, Valvano MA. Wien: Springer-Verlag; <br /> 13.   Liu L, Agren R, Bordel S, Nielsen J: Use genome-scale metabolic models          2011:21&#226;&#8364;&#8220;39. <br />       understanding microbial physiology. FEBS Letters 2010, 584:2556&#226;&#8364;&#8220;2564.      35. Mandelli F, Yamashita F, Pereira JL, Mercadante AZ: Evaluation biomass <br /> 14.   Feist AM, Henry CS, Reed JL, Krummenacker M, Joyce AR, Karp PD,                    production, carotenoid level antioxidant capacity produced by <br />       Broadbelt LJ, Hatzimanikatis V, Palsson BO: A genome-scale metabolic               Thermus filiformis using fractional factorial design. Braz J Microbiol 2012, <br />       reconstruction Escherichia coli K-12 MG1655 accounts 1260             43:126&#226;&#8364;&#8220;134. <br />       ORFs thermodynamic information. Mol Syst Biol 2007, 3:121.                 36. Oshima T: Unique polyamines produced extreme thermophile, <br /> 15.   Oh YK, Palsson BO, Park SM, Schilling CH, Mahadevan R: Genome-scale                Thermus thermophilus. Amino Acids 2007, 33:367&#226;&#8364;&#8220;372. <br />       reconstruction metabolic network Bacillus subtilis based              37. Raghunathan AU, Perez-Correa JR, Bieger LT: Data reconciliation and <br />       high-throughput phenotyping gene essentiality data. J Biol Chem                parameter estimation flux-balance analysis. Biotechnol Bioeng 2003, <br />       2007, 282:28791&#226;&#8364;&#8220;28799.                                                             84:700&#226;&#8364;&#8220;709. <br /> 16.   Mo ML, Palsson BO, Herrgard MJ: Connecting extracellular metabolomic           38. Schuster S, Pfeiffer T, Fell DA: Is maximization molar yield metabolic <br />       measurements intracellular flux states yeast. BMC Syst Biol 2009,            networks favoured evolution? J Theor Biol 2008, 252:497&#226;&#8364;&#8220;504. <br />       3:37.                                                                          39. Demirtas MU, Kolhatkar A, Kilbane JJ 2nd: Effect aeration agitation <br /> 17.   Chung BK, Selvarasu S, Andrea C, Ryu J, Lee H, Ahn J, Lee DY:                      growth rate Thermus thermophilus batch mode. J Biosci Bioeng <br />       Genome-scale metabolic reconstruction silico analysis                    2003, 95:113&#226;&#8364;&#8220;117. <br />       methylotrophic yeast Pichia pastoris strain improvement. Microb            40. Mckay A, Quilter J, Jones CW: Energy conservation extreme <br />       Cell Fact 2010, 9:50.                                                              thermophile Thermus Thermophilus HB8. Arch Microbiol 1982, 131:43&#226;&#8364;&#8220;50. <br /> 18.   Kjeldsen KR, Nielsen J: In silico genome-scale reconstruction              41. Mahadevan R, Schilling CH: The effects alternate optimal solutions in <br />       validation Corynebacterium glutamicum metabolic network.                    constraint-based genome-scale metabolic model. Metab Eng 2003, <br />       Biotechnol Bioeng 2009, 102:583&#226;&#8364;&#8220;597.                                               5:264&#226;&#8364;&#8220;276. <br /> 19.   Park JM, Kim TY, Lee SY: Genome-scale reconstruction silico             42. Selvarasu S, Ow DS-W, Lee SY, Lee MM, Oh SK-W, Karimi IA, Lee D-Y: <br />       analysis Ralstonia eutropha H16 polyhydroxyalkanoate                    Characterizing Escherichia coli DH5&#206;&#177; growth metabolism a <br />       synthesis, lithoautotrophic growth, 2-methyl citric acid production.           complex medium using genome-scale flux analysis. Biotechnol Bioeng <br />       BMC Syst Biol 2011, 5:101.                                                         2009, 102:923&#226;&#8364;&#8220;934. <br /> 20.   Oberhardt MA, Puchalka J, Fryer KE, Martins dosSantos VA, Papin JA:            43. Koga Y: Thermal adaptation archaeal bacterial lipid <br />       Genome-scale metabolic network analysis opportunistic pathogen              membranes. Archaea 2012, 2012:789652. <br />       Pseudomonas aeruginosa PAO1. J Bacteriol 2008, 190:2790&#226;&#8364;&#8220;2803.                  44. Kumar S, Tsai CJ, Nussinov R: Factors enhancing protein thermostability. <br /> 21.   Selvarasu S, Karimi IA, Ghim GH, Lee DY: Genome-scale modeling              Protein Eng 2000, 13:179&#226;&#8364;&#8220;191. <br />       silico analysis mouse cell metabolic network. Mol bioSyst 2010,             45. Mallik S, Kundu S: A comparison structural evolutionary attributes <br />       6:152&#226;&#8364;&#8220;161.                                                                         Escherichia coli Thermus thermophilus small ribosomal subunits: <br /> 22.   Duarte NC, Becker SA, Jamshidi N, Thiele I, Mo ML, Vo TD, Srivas R, Palsson        signatures thermal adaptation. PloS One 2013, 8:e69898. <br />       BO: Global reconstruction human metabolic network based              46. Koyama Y, Hoshino T, Tomizuka N, Furukawa K: Genetic transformation of <br />       genomic bibliomic data. Proc Natl Acad Sci U S A 2007,                         extreme thermophile Thermus thermophilus Thermus <br />       104:1777&#226;&#8364;&#8220;1782.                                                                     spp. J Bacteriol 1986, 166:338&#226;&#8364;&#8220;340. <br /> 23.   Lakshmanan M, Koh G, Chung BK, Lee DY: Software applications flux          47. Yang KM, Lee NR, Woo JM, Choi W, Zimmermann M, Blank LM, Park JB: <br />       balance analysis. Brief Bioinform 2014, 15:108&#226;&#8364;&#8220;122.                                Ethanol reduces mitochondrial membrane integrity impacts <br /> 24.   Lee SJ, Lee DY, Kim TY, Kim BH, Lee J, Lee SY: Metabolic engineering            carbon metabolism Saccharomyces cerevisiae. FEMS Yeast Res 2012, <br />       Escherichia coli enhanced production succinic acid, based                12:675&#226;&#8364;&#8220;684. <br />       genome comparison silico gene <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>knockout</span> simulation. Appl Environ         48. Thiele I, Palsson BO: A protocol generating high-quality genome-scale <br />       Microbiol 2005, 71:7880&#226;&#8364;&#8220;7887.                                                      metabolic reconstruction. Nat Protocol 2010, 5:93&#226;&#8364;&#8220;121. <br /> 25.   Matsuda F, Furusawa C, Kondo T, Ishii J, Shimizu H, Kondo A: Engineering       49. Kanehisa M, Goto S, Hattori M, Aoki-Kinoshita KF, Itoh M, Kawashima S, <br />       strategy yeast metabolism higher alcohol production. Microb Cell            Katayama T, Araki M, Hirakawa M: From genomics chemical genomics: <br />       Fact 2011, 10:70.                                                                  new developments KEGG. Nucleic Acids Res 2006, 34:D354&#226;&#8364;&#8220;D357. <br /> 26.   Zhang Y, Thiele I, Weekes D, Li Z, Jaroszewski L, Ginalski K, Deacon AM,       50. Caspi R, Altman T, Dreher K, Fulcher CA, Subhraveti P, Keseler IM, Kothari <br />       Wooley J, Lesley SA, Wilson IA, Palsson BO, Osterman A, Godzik A:                  A, Krummenacker M, Latendresse M, Mueller LA, Ong Q, Paley S, Pujar A, <br />       Three-dimensional structure view central metabolic network               Shearer AG, Travers M, Weerasinghe D, Zhang P, Karp PD: The MetaCyc <br />       Thermotoga maritima. Science 2009, 325:1544&#226;&#8364;&#8220;1549.                                  database metabolic pathways enzymes BioCyc <br /> 27.   Ulas T, Riemer SA, Zaparty M, Siebers B, Schomburg D: Genome-scale                 collection pathway/genome databases. Nucleic Acids Res 2012, <br />       reconstruction analysis metabolic network                        40:D742&#226;&#8364;&#8220;D753. <br />       hyperthermophilic archaeon Sulfolobus solfataricus. PloS One 2012,             51. Satish Kumar V, Dasika MS, Maranas CD: Optimization based automated <br />       7:e43401.                                                                          curation metabolic reconstructions. BMC Bioinform 2007, 8:212. <br />  Lee et al. Microbial Cell Factories 2014, 13:61                                                                                              Page 14 14 <br /> http://www.microbialcellfactories.com/content/13/1/61 <br />  <br />  <br />  <br />  <br /> 52. Ray PH, White DC, Brock TD: Effect growth temper <br /> </body></html>