<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>On Evolution Yeti, Drosophila melanogaster <br /> Heterochromatin Gene <br /> Roberta Moschetti1, Emanuele Celauro2, Fulvio Cruciani2, Ruggiero Caizzi1, Patrizio Dimitri2* <br /> 1 Dipartimento di Biologia, Universita` degli Studi di Bari, Bari, Italy, 2 Dipartimento di Biologia e Biotecnologie &#226;&#8364;&#732;&#226;&#8364;&#732;Charles Darwin&#226;&#8364;&#8482;&#226;&#8364;&#8482; Istituto Pasteur Fondazione Cenci- <br /> Bolognetti, Sapienza Universita` di Roma, Roma, Italy <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Constitutive heterochromatin ubiquitous unveiled component eukaryotic genomes, it <br />      comprises large portions. Although constitutive heterochromatin generally considered transcriptionally silent, it <br />      contains significant variety sequences expressed, 300 single-copy coding genes been <br />      identified genetic genomic analyses decades. Here, report results evolutionary analysis of <br />      Yeti, essential gene Drosophila melanogaster located deep pericentromeric region chromosome 2R. By FISH, <br />      showed Yeti maintains heterochromatin location D. simulans D. sechellia species, closely related D. <br />      melanogaster, distant species e.g., D. pseudoobscura D. virilis, euchromatin, the <br />      syntenic chromosome Muller C, corresponds 2R arm D. melanogaster chromosome 2. Thus, evolutionary <br />      time, Yeti resident chromosomal element, progressively moved closer pericentric <br />      regions. Moreover, silico reconstruction Yeti gene structure 19 Drosophila species 5 non-drosophilid <br />      dipterans shows stable organization evolution. Accordingly, PCR analysis sequencing, that <br />      single intron Yeti does undergo major intraspecies interspecies size changes, unlike introns other <br />      essential Drosophila heterochromatin genes, light Dbp80. This implicates diverse evolutionary forces shaping <br />      structural organization genes heterochromatin. Finally, results dS - dN tests Yeti under <br />      negative selection heterochromatin euchromatin, indicate change genomic location did not <br />      affected significantly molecular evolution gene. Together, results work contribute our <br />      understanding evolutionary dynamics constitutive heterochromatin genomes higher eukaryotes. <br />  <br />   Citation: Moschetti R, Celauro E, Cruciani F, Caizzi R, Dimitri P (2014) On Evolution Yeti, Drosophila melanogaster Heterochromatin Gene. PLoS ONE 9(11): <br />   e113010. doi:10.1371/journal.pone.0113010 <br />   Editor: Igor V. Sharakhov, Virginia Tech, United States America <br />   Received August 11, 2014; Accepted October 22, 2014; Published November 18, 2014 <br />   Copyright: &#195;&#376; 2014 Moschetti et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Data Availability: The authors confirm data underlying findings fully available restriction. All relevant data paper. <br />   Funding: This research supported grants Ministero dell&#226;&#8364;&#8482;Universita` e della Ricerca- Miur (Prin 2008), Istituto Pasteur-Fondazione Cenci-Bolognetti and <br />   Fondazione Roma terzo settore. The funders role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * Email: patrizio.dimitri@uniroma1.it <br />  <br />  <br />  <br /> Introduction                                                                              so-called &#226;&#8364;&#732;&#226;&#8364;&#732;junk&#226;&#8364;&#8482;&#226;&#8364;&#8482; repetitive DNA, satellite sequences and <br />                                                                                           truncated transposable element remnants. <br />    Constitutive heterochromatin commonly large blocks                            Together, properties led view constitutive <br /> near centromeres telomeres; consists repetitive                          heterochromatin &#226;&#8364;&#732;&#226;&#8364;&#732;desert&#226;&#8364;&#8482;&#226;&#8364;&#8482; genetic functions [5]. In last <br /> DNA sequences maintains characteristic organization                            decades, however, studies primarily conducted Drosophila <br /> both homologous chromosomes. It ubiquitous component                              melanogaster shown constitutive heterochromatin does <br /> eukaryotic genomes and, species, comprises large                                  fact play roles important cellular functions, as <br /> chromosomal portions, entire chromosomes. For example,                            chromosome organization inheritance [6,7,8,9,10,11]. Al- <br /> about 30% Drosophila human genomes, 70&#226;&#8364;&#8220;                              generally regarded transcriptionally silent, constitutive <br /> 90% certain nematode plant genomes,                                 heterochromatin contain actively transcribed <br /> constitutive heterochromatin [1,2,3], reasons                             genes [3]. For example, Drosophila melanogaster, 40 <br /> widespread occurrence unclear.                                                  genes essential viability fertility mapped <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br />    Heterochromatin originally defined cytological level                         pericentric heterochromatin [12,13,14,15,16,17]. <br /> the chromosome portion stains deeply prophase                                    In decade, release D. melanogaster heterochro- <br /> maintains compact organization stages                             matin sequence Berkeley Drosophila Genome Project <br /> mitotic cell cycle [4]. Historically, distinctive antagonistic proper-                    (http://www.fruitfly.org/) Drosophila Heterochromatin Ge- <br /> ties compared rest genome identified: 1) strongly                      nome Project (http://www.dhgp.org/index_release_notes.html) <br /> reduced level meiotic recombination; 2) low gene density; 3)                           greatly facilitated studies mapping, molecular organization <br /> mosaic inactivation expression euchromatic genes                           function genes located pericentromeric heterochromatin <br /> moved nearby (position effect variegation, PEV); 4) late replication                      [18]. <br /> during S phase; 5) transcriptional inactivity; 6) enrichment the <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                            1                         November 2014 | Volume 9 | Issue 11 | e113010 <br />                                                                                                           Evolution Drosophila Yeti Gene <br />  <br />  <br />  <br />  <br /> Figure 1. Cytogenetic mapping heterochromatin genes chromosome 2. The map modified shown previous papers <br /> [3,45]. The diagram shows essential genes defined mutational analyses (below) annotated genes defined heterochromatin genome <br /> project (above). Shades blue correspond intensity DAPI staining, darkest blue blocks representing regions strong <br /> fluorescence intensity open blocks representing non fluorescent regions. The different cytological regions numbered. <br /> doi:10.1371/journal.pone.0113010.g001 <br />  <br />    More recently, improved genome shotgun assembly               transcription profiles mapped sequences microarray analysis <br /> [19] produced, includes 20.7 Mb draft-quality           revealed region-specific temporal patterns transcription <br /> heterochromatin sequence. In years, 15 Mb                heterochromatin oogenesis early embryonic <br /> sequence improved completed [20]               development. <br /> BAC-based physical map 13 Mb pericentric heterochroma-                 Evolutionary studies shown D. melanogaster hetero- <br /> tin, cytogenetic map locates 11 Mb         chromatin genes, light others, originated <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> specific heterochromatin regions, constructed [20].             progenitors originally located euchromatin in <br /> About 250 protein-coding genes defined release 5.1            drosophilid lineage [23,24]. Here focussed study <br /> annotation currently sequenced heterochromatin DNA                 evolutionary origin Yeti, essential heterochromatin <br /> [21]. According results, number active genes           gene D. melanogaster, encodes protein belonging the <br /> constitutive heterochromatin D. melanogaster appears             evolutionarily conserved BCNT family chromatin remodellers <br /> higher defined genetic analysis. Notably, genes             [25,26]. We report Yeti locates euchromatin distant <br /> encode proteins involved important cellular developmental          species, e.g. D. pseudoobscura D. virilis, similarly has <br /> processes [3].                                                            light genes [23,24]. Moreover, found <br />    Further mapping D. melanogaster heterochromatin                 Yeti gene structure remains stable the <br /> performed comparative genomic hybridization [22]. The                  evolution Drosophila species. In particular, second exon that <br />  <br />  <br /> PLOS ONE | www.plosone.org                                            2                     November 2014 | Volume 9 | Issue 11 | e113010 <br />                                                                                                                         Evolution Drosophila Yeti Gene <br />  <br />  <br />  <br />  Table 1. List Yeti orthologs encoded proteins. <br />  <br />  <br />                                                                                                                                     Uniprot         Amino <br />  Species                Gene ID              Database Location                                     GenBank A.C. (position)          reference       acids <br />  <br />  D.melanogaster         FBgn0128734          2Rh:1343403..1345119                                  NW_001848856.1                   B4J7U2          241 <br />  D.simulans             FBgn0191193          3R:38,269..39,056 [+]                                 NT_167061.1                      B4QUX4          241 <br />  D.sechellia            FBgn0166002          2h;scaffold_170:38,673..3,460 [2].                    NW_001999858.1                   B4IME9          241 <br />  D.yakuba               FBgn0236265          2h;v2_chr2h_random_005:668,463..673,172.              NW_002052891.1                   B4IT75          215 <br />  D.erecta               FBgn0103193          scaffold 4929                                         NW_001956548.1                   B3N457          236 <br />  D.eugracilis           available        available                                         KB464511.1 (19716..18949)        available   235 <br />  D.biarmipes            available        available                                         KB462255.1 (18839..19580)        available   229** <br />  D.takahashi            available        available                                         KB460683.1 (11044..10283)        available   235** <br />  D.elegans              available        available                                         KB458177.1 (6204..6991)          available   244** <br />  D.ananassae            FBgn0088065          2R euchromatic region scaffold 13266                  NW_001939294.1                   B3MBR1a         272a <br />  D.bipectinata          available        available                                         KB464371.1 (48012..48773)        available   232 <br />  D.pseudoobscura        FBgn0245984          2R euchromatic region                                 NC_009006.2                      B5E0W2          275 <br />  D.persimilis           FBgn0148669          available                                         NW_001985955.1                   B4GBJ4          275 <br />  D.miranda              available        available                                         CM001519.2 (366522..367496)      available   275 <br />  D.willistoni           FBgn0212772          2R euchromatic arm                                    NW_002032340.1                   B4MJ08          273 <br />  D.mojavensis           FBgn0143134          2R euchromatic arm                                    NW_001979114.1                   B4KQ8           295 <br />  D.virilis              FBgn0209341          2R euchromatic arm                                    NW_002014420.1                   B4LKP7          300 <br />  D.albomicans           available        available                                         JH859027.1 (5894805..5895706)    available   279 <br />  D.grimshawi            FBgn0128734          2R euchromatic arm                                    NW_001961673.1                   B4J7U2          285 <br />  C.quinquefasciatus     CPIJ018830           available                                         NW_001888048.1                   B0XH12          284 <br />  A.aegypti              AAEL007422           supercontig 1.255, euchromatic region 2p25 [46]       NW_001810963.1                   Q0IF03          279 <br />  A.darlingi             available        available                                         ADMH02000690.1 (45710.. 44829)   available <br />  A.gambiae              AGAP005152           2L:euchromatic region 21E                             NT_078265.2                      Q7PPY4          293 <br />  M.destructor           available        available                                         GL501532.1 (369211.. 368309)     available   300 <br />  <br />  **ORF defective. Amino acids deducted bestfit alignments. <br />  a) <br />    Bestfit protein 236 aminoacids long. <br />  doi:10.1371/journal.pone.0113010.t001 <br />  <br />  <br /> encodes 30 aminoacids conserved BCNT domain                        probes amplified conserved region located outside <br /> invariably 91 bp-long. Finally, single intron                    C-terminal BCNT coding domain YETI protein (see <br /> Yeti does undergo major size changes D. melanogaster                       Materials Methods). <br /> closely related species, unlike introns essential                           The results analysis shown Figure 2. In D. <br /> Drosophila heterochromatin genes [27].                                                melanogaster, D. simulans D. sechellia Yeti cDNA probe <br />                                                                                       produces signal mapping <span id='am-5' about='xsp:base' typeof='owl:Thing'>base</span> division 41A, right <br /> Results                                                                               arm chromosome 2 (Figure 2A,B,C). Notably, signals show <br />                                                                                       large diffuse structure different sharp hybridization <br /> Evolutionary repositioning Yeti gene                                      signals usually seen euchromatic probes; morphology <br /> euchromatin pericentric heterochromatin                                            distinctive mark sequences derived partially <br />   The single-copy Yeti gene D. melanogaster maps region                     polytenized heterochromatin regions [30,31]. Together, FISH <br /> h41 chromosome 2R mitotic heterochromatin (Figure 1;                               results indicate Yeti maintains heterochromatic location in <br /> Table 1), corresponds division 41A salivary gland                         D. simulans D. sechellia. The FlyBase localization Yeti D. <br /> polytene chromosomes [25,26,28,29].                                                   sechellia 2 h (scaffold_170:38,673..3,460; Table 1), accord <br />   To characterize chromosomal location Yeti                              mapping results, D. simulans 3R <br /> Drosophila genus species, performed fluorescent situ                            (38,269..39,056; Table 1) apparently conflicting reflect <br /> hybridization (FISH) experiments polytene chromosomes                           assembly error D. simulans genome sequences (see <br /> D. simulans, D. sechellia, sibling species D. melanogaster,                    discussion), reported Schaffer et al. [32]. In D. <br /> and distantly related species: D. pseudoobscura, belonging                     pseudoobscura D. virilis single FISH signal observed in <br /> to Sophophora subgenus D. virilis, belonging                           euchromatic arms polytene chromosomes, agreement <br /> Drosophila subgenus. These species cover nearly 40 million years                    FlyBase (Figure 2D, E; Table 1). In D. pseudoobscura, the <br /> divergence time represent wide spectrum                             Yeti PCR probe produced sharp signal maps region 63C <br /> evolutionary history Yeti.                                                         proximal euchromatin chromosome 3, D. Virilis <br />    To map Yeti D. simulans D. sechellia, used D.                        Yeti signal region 53E, distal euchromatin of <br /> melanogaster Yeti cDNA probe (RE36623), PCR species-                            chromosome 5. Thus, independently genome localization <br /> specific probes used D. pseudoobscura D. virilis. PCR                     (heterochromatin euchromatin), analysed species Yeti lies <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                        3                     November 2014 | Volume 9 | Issue 11 | e113010 <br />                                                                                                                     Evolution Drosophila Yeti Gene <br />  <br />  <br />                                                                                    59 end gene, probably errors sequencing that <br />                                                                                    needs improved. However, case 39end the <br />                                                                                    gene, containing BCNT domain-coding region, detected <br />                                                                                    (Figure 3). The protein sequence alignments strong <br />                                                                                    conservation BCNT domain Drosophila genus and <br />                                                                                    non-drosophilid dipterans (Figure 4). <br />                                                                                       We compared reconstructed molecular organization of <br />                                                                                    Yeti gene above-mentioned species, study it <br />                                                                                    underwent substantial structural changes evolution. The <br />                                                                                    results analysis shown Figure 5. It appears the <br />                                                                                    structure Yeti orthologs, exons intron <br />                                                                                    remained highly conserved divergence lineages in <br />                                                                                    Drosophila genus, exception D. willistoni, where <br />                                                                                    additional intron 75 bp present. The gene size identical <br />                                                                                    D. melanogaster, D. simulans D. sechellia, only <br />                                                                                    detectable difference represented intron D. <br />                                                                                    melanogaster 7 bp longer. In general, Drosophila genus <br />                                                                                    exon single intron undergo changes size: <br />                                                                                    exon varies 557 bp (D. yakuba) 812 bp (D. virilis), <br />                                                                                    intron spans 51 bp (D. biarmipes) 70 bp (D. <br />                                                                                    willistoni). Notably, size second exon, encodes the <br />                                                                                    portion BCNT domain, shows striking conservation in <br />                                                                                    Drosophila genus, invariably 91 bp long. Finally, the <br />                                                                                    genome species non-drosophilid dipterans (C. quinquefas- <br /> Figure 2. Examples FISH mapping Yeti probes polytene                      ciatus, A. Aegypti, A. darlingi, A. gambiae M. destructor) the <br /> chromosomes Drosophila species. Salivary gland polytene                         organization Yeti differs intron disappears a <br /> chromosomes stained DAPI pseudocolorated blue;                    single exon present. <br /> fluorescent signals pseudocolorated red. In D. melanogaster (A) <br /> and closely related D. simulans (B) D. sechellia (C) species, the <br /> Yeti cDNA probe maps 2Rh base polytene division 41. The               Characterization Yeti intron Drosophila species <br /> large diffuse morphology Yeti signal species,               Intraspecific interspecific size polymorphisms hetero- <br /> reflects disorganized poorly banded structure                       chromatin gene introns Drosophila, are <br /> heterochromatin chromocenter. The arrows point base                  likely associated novo insertions TE-related <br /> 2Rh. In D. pseudobscura, hybridization signal Yeti PCR probe maps           sequences [27]. We asked intron Yeti prone <br /> to region 63C proximal euchromatin (D). In D. virilis Yeti              TE insertions gross changes <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span>. To answer this <br /> hybridization signal maps region 53E, distal euchromatin of <br /> chromosome 5 (E).                                                                  question, PCR amplified region 180 bp comprising <br /> doi:10.1371/journal.pone.0113010.g002                                              Yeti intron genomic DNAs extracted D. <br />                                                                                    melanogaster D. simulans strains. Most analyzed strains <br /> in syntenic chromosome Muller C, corresponds 2R                    derived geographically distant natural populations (see <br /> arm D. melanogaster chromosome 2.                                               materials methods complete list). We included <br />                                                                                    analysis single strain D. sechellia D. teissieri. The <br />                                                                                    rationale experiments insertion targeted <br /> In silico reconstruction Yeti gene organization                              intron, turn increased expected size the <br /> different sequenced genomes                                                        amplified region. <br />    To study evolutionary conservation Yeti gene                            We analysed 25 wild type strains 4 laboratory stocks D. <br /> organization, characterized structure Yeti                      melanogaster, 9 wild type strains D. simulans (see <br /> orthologs 19 Drosophila species (D. melanogaster, D. simulans,                  materials methods). As shown Figure 6, PCR amplification <br /> D. sechellia, D. yakuba, D. erecta, D. eugracilis, D. biarmipes, D                 Yeti produces prominent band expected size the <br /> takahashii, D. elegans, D. ananassae, D. bipectinata, D. pseudobs-                 different strains D. melanogaster (Figure 6A) D. simulans <br /> cura, D. persimilis, D. miranda, D. willistoni, D. mojavensisi, D.                 (Figure 6B) D. sechellia D. tessieri strains <br /> virilis, D. albomicans, D. grimshawi) non-                         (Figure 6C). Sequencing purified PCR products Iso <br /> drosophilid dipterans (C. quinquefasciatus, A. Aegypti, A. darlingi,               Scansano (D. melanogaster), Chicharo Death Valley (D. <br /> A. gambiae M. destructor pest crop).                                           simulans), D. sechellia D. tessieri confirmed they <br />    All Yeti DNA sequences retrieved FlyBase. In                      correspond Yeti intron-containing region (Figure 7). In <br /> case annotated genes, YETI protein sequences                           conclusion, results analysis suggest small intron <br /> extracted Ortho DB database [33],                          Yeti does frequently undergo significant increase size, <br /> reported orthologous sequences belonging BCNT family                     unlike essential heterochromatin genes Drosophila [27]. <br /> complex [26,34]. For recently sequenced genomes, Yeti                      This conclusion agreement observation Yeti <br /> DNA sequences recovered TblastN procedure, using                       gene structure tends stable evolution the <br /> the D. melanogaster YETI protein sequence. The alignments                       Drosophila genus, single intron retains short size <br /> the retrieved Yeti orthologs shown Figure 3                       (Figure 5). <br /> coordinates reported Table 1. <br />    We able recover deducted complete protein sequence                    Yeti negative selection <br /> except recently added genomes D. biarmipes, D.                            We asked Yeti evolved negative <br /> takahashi D. elegans, frame-shift mutations                   (purifying) positive selection, change in <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                     4                      November 2014 | Volume 9 | Issue 11 | e113010 <br />                                                                                                                 Evolution Drosophila Yeti Gene <br />  <br />  <br />  <br />  <br /> Figure 3. Alignment Yeti ortholog sequences encoding BCNT-C domain. The grey area corresponds intron present the <br /> Drosophila species. <br /> doi:10.1371/journal.pone.0113010.g003 <br />  <br /> location euchromatin heterochromatin affected                 change genomic location does appear affected <br /> the molecular evolution gene. To aim performed                significantly molecular evolution/function Yeti (Table 2). <br /> dS - dN tests, codon-based test selection (for details material <br /> and methods), using DNA sequences representative Yeti                Discussion <br /> genes: located heterochromatin (Dmel\Yeti, Dsec\Yeti and <br /> Dere\Yeti) euchromatin (Dpse\Yeti Dvir\Yeti). The                  In paper, studied evolutionary origin Yeti, an <br /> results suggest Yeti purifying (negative) selection         essential gene Drosophila melanogaster (Figure 1) located the <br /> present heterochromatin euchromatin. Thus, the <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                 5                     November 2014 | Volume 9 | Issue 11 | e113010 <br />                                                                                                               Evolution Drosophila Yeti Gene <br />  <br />  <br />  <br />  <br /> Figure 4. Alignment BCNT domain YETI proteins species. The arrow points intron position corresponding coding <br /> region Drosophila species. <br /> doi:10.1371/journal.pone.0113010.g004 <br />  <br /> deep heterochromatin chromosome 2 [25,26,28,29] found <br /> that evolved euchromatic ancestor drosophilids. <br />    Our FISH analysis shows Yeti maintains heterochromatin <br /> location 2Rh, base division 41A, D. simulans and <br /> D. sechellia sibling species D. melanogaster (Figure 2). The FlyBase <br /> localization Yeti D. sechellia scaffold_170:38,673..3,460), in <br /> accord mapping results, D. simulans Yeti reported to <br /> map 3R arm (3,765..73,764; Table 1) 70 kb gene-poor <br /> genomic region. However, confident heterochro- <br /> matin location Yeti D. simulans following reasons: First, <br /> as discussed result section, FISH signal morphology <br /> produced Yeti probe different usually seen with <br /> euchromatic probes represents distinctive mark sequences <br /> derived polytenized heterochromatin [30,31]. Second, our <br /> FISH mapping Yeti D. simulans based reproducible <br /> experiments, carried polytene chromosome <br /> figures obtained 10 larvae. Finally, paucity genes <br /> around D. simulans Yeti se highly suggestive heterochromatin <br /> localization. Thus, possible apparent discrepancy <br /> between data FlyBase reflect assembly error that <br /> occurred D. simulans genome sequence assembly, reported <br /> by Schaffer et al. [32]. <br />    Our FISH analysis distantly related species, D. <br /> pseudoobscura D. virilis, Yeti located euchromatin <br /> (Figure 2). In D. pseudobscura, Yeti maps chromosome 3 at <br /> polytene division 63C, D. virilis in <br /> chromosome 5, polytene division 53E. Interestingly, Yeti lies <br /> in syntenic chromosome Muller C corresponds 2R <br /> arm D. melanogaster chromosome 2. Together, results of <br /> our analysis indicate evolution Drosophila <br /> genus, Yeti resident chromosomal element, <br /> but time progressively moved closer pericentric <br /> regions. Such movements occurred 40 million <br />                                                                               Figure 5. Comparison Yeti gene structure among <br /> of years, estimated divergence time D. melanogaster               sequenced genomes. Only coding regions showed. Exons <br /> and D. virilis (Figure 8). A similar evolutionary trend reported          boxes numbers refer nucleotides. Symbols: 1, annotated <br /> for light neighboring genes 2L heterochromatin [23]              genes; ?, study; **, defective ORF. The grey area 39 end <br /> and genes chromosome 3 heterochromatin [24].                     represents conserved BCNT-C domain protein. <br />                                                                               doi:10.1371/journal.pone.0113010.g005 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                6                     November 2014 | Volume 9 | Issue 11 | e113010 <br />                                                                                                                    Evolution Drosophila Yeti Gene <br />  <br />  <br />  <br />  <br /> Figure 6. PCR amplification genomic region containing Yeti intron Drosophila species. A single PCR product 180 bp <br /> was found, D. melanogaster (A) D. simulans, (B) D. sechellia (C) D. tessieri (C) related species. M = Marker; Frib = Friburgo; Mal = Mali; Scan <br /> = Scansano; Iso = y1; cn1 bw1sp1 isogenic strain; OR = Oregon-R; Bej = Bejin; Chi = Chicharo; DV = Death Valley; Gen = Genoa; Mor = Moruya; Kyo <br /> = Kyogle; Arm = Armidale; Can = Canaries; Sech = D. Sechellia; tes = D. tessieri. Molecular weight bps. <br /> doi:10.1371/journal.pone.0113010.g006 <br />  <br />    A striking difference heterochromatin euchroma-                  TE copies (or TE &#226;&#8364;&#732;&#226;&#8364;&#732;remnants&#226;&#8364;&#8482;&#226;&#8364;&#8482;) [27,35,36,37]. In context, the <br /> tin genes lies generally larger size complex molecular               Yeti gene D. melanogaster 900 bp-long genomic region <br /> structure compared latter. The example              represents exception [26]. The true RpL38, RpL5 <br /> &#226;&#8364;&#732;giant&#226;&#8364;&#8482; Y-chromosome fertility factors D. melanogaster men-                  RpL15, essential ribosomal protein-coding genes located <br /> tioned paradigmatic respect [12]. Some                  heterochromatin chromosomes 2 3, are <br /> essential heterochromatin genes chromosomes 2 3                 relatively small size [24,26,37,38]. How observa- <br /> large presence large introns harbour truncated               tions explained? <br />  <br />  <br />  <br />  <br /> Figure 7. Sequencing purified PCR products Drosophila species. Sequence alignments Iso Scansano (D. melanogaster), <br /> Chicharo Death Valley (D. Simulans), D. sechellia D. tessieri. Sequence analysis confirmed correspond Yeti intron containing <br /> region. The Yeti intron shown normal text, flanking exons bold. The D. simulans D. sechellia intron lacks 7 bp stretch (see the <br /> gap), agreement genome sequence data (see results Figure 5). The D. Tessieri intron sequence identical D. simulans D. <br /> sechellia. <br /> doi:10.1371/journal.pone.0113010.g007 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                  7                      November 2014 | Volume 9 | Issue 11 | e113010 <br />                                                                                                                                 Evolution Drosophila Yeti Gene <br />  <br />  <br />  <br />  Table 2. Codon-based test selection. <br />  <br />  <br />  Number species compared                                                  Number codonsa                                    dS - dN                Pb <br />  <br />  5                                                                           232                                                  10.2                   ,1026 <br />  3 (heterochromatic sequences)                                               233                                                  5.3                    ,1026 <br />  2 (euchromatic sequences)                                                   275                                                  8.7                    ,1026 <br />  <br />  a <br />   All positions containing gaps eliminated. <br />  b <br />   The probability (P) rejecting null hypothesis strict-neutrality (dN = dS) favor alternative hypothesis (purifying selection, dN,dS) shown. <br />  doi:10.1371/journal.pone.0113010.t002 <br />  <br />  <br />    One imagine that, evolution, genes increased                        proper development. It unreasonable speculate these <br /> size targets reiterated transposable-element                            genes maintained original structure, spite transition <br /> insertions intronic regions, depending time                          heterochromatin massive chromosomal rearrangements <br /> residence heterochromatin. This, however, does                           occurred time, requirement their <br /> have case. In fact, light Yeti genes,                         efficient expression early development. In addition, is <br /> having likely resided heterochromatin comparable                               possible heterochromatin, given sequence be <br /> evolutionary time (less 30 million years), underwent                          differentially targeted transposable elements, some <br /> different molecular architecture; light gene structure dramat-                      sequences refractory others. These observations <br /> ically changed evolutionary transition euchroma-                        suggest evolutionary forces acted shaping the <br /> tin heterochromatin, remarkable increase size                     structural organization genes currently D. melanogaster <br /> introns targeted TEs [23]; Yeti retained original organization                   heterochromatin molecularly diverse. <br /> in analysed species, short genomic region carrying                            Finally, results dS - dN tests, showing Yeti under <br /> single short intron (Figure 5). In addition, PCR analysis                         negative selection heterochromatin euchromatin <br /> found Yeti intron does undergo significant interspecies                    (Table 2), accord evolutionary conserved function <br /> or intraspecies changes physical size (Figure 6). Similarly,                 suggest change genomic location did affected <br /> RpL15 gene shows conserved structure Drosophila                                 significantly molecular evolution gene. <br /> species, independently genomic location [24].                                     Together, results contribute expand understanding <br />    How explain different behaviour heterochromatin                            molecular dynamics driving evolution hetero- <br /> genes? In particular, selective pressure                     chromatin genome higher eukaryotes. <br /> maintain genes short size (with few, short introns) despite <br /> of genomic location, owing particular functional                         Materials Methods <br /> requirement: interestingly respect, highly expressed genes <br /> are known harbour substantially shorter introns genes                      Drosophila strains <br /> are expressed low levels [39]. This case Yeti                         Fly cultures carried 25uC standard cornmeal <br /> RpL38, RpL5 RpL15 heterochromatic genes Drosophila                               yeast medium. D. melanogaster strains derived natural <br /> melanogaster, highly expressed                        populations: Altamura I Altamura II (South Italy), Beijing <br /> short size carry short introns [26,37,38]. Yeti itself encodes                   (Cina), California California I (USA), Charolles (France), <br /> important chromatin-remodeling factor required development                          Charolles 1999 (France), Friburgo 1997 (Germany), HJ30 <br /> [26] ribosomal protein coding genes essential                          (France), Hobo (Grece) Israel-4 (Israel), Lanuvio (Italy), Luminy <br />                                                                                         (France), Mali Mali I (West Africa), Marrakesh (North Africa), <br />                                                                                         Scansano (Central Italy), Vallecas (Spain), W15 (USA), W30 <br />                                                                                         (USA), W90 (USA), W130 (USA), W135 (USA), Zimbabwe <br />                                                                                         (Africa), Gaiano (North Italy). D. melanogaster laboratory strains <br />                                                                                         (separated comma): isogenized y1; cn1,bw1,sp1 strain (Iso) [40]. <br />                                                                                         l(2)LP2/SM1,Cy. Cy/Sp;Sb,Delta2-3,ry506/TM6,ry506. l(2)EMS- <br />                                                                                         31/SM1,Cy. D. simulans strains derived natural populations: <br />                                                                                         Armidale (New South Wales, Australia), Canaries (Atlantic Spain), <br />                                                                                         Can River (Australia) Chantal (France), Chicharo (Portugal), <br />                                                                                         Death Valley (USA), Genoa (Italy), Kyogle (Australia) Moruya <br />                                                                                         (New South Wales, Australia). D. melanogaster D. simulans <br />                                                                                         wild-type strains derived natural populations collected <br />                                                                                         wild end year 2000 gifts Sergio <br />                                                                                         Pimpinelli, Nikolaj Junakovic, Chantal Vaury Pierre Capy. <br />  <br /> Figure 8. Evolutionary repositioning Yeti gene. Schematic                        Cytology fluorescent situ hybridization <br /> representation Yeti gene transition euchromatin                              Polytene chromosomes prepared according Pardue [41] were <br /> heterochromatin. The arrows point chromosomal position Yeti.                     stained DAPI. The D. melanogaster RE36623 cDNA Yeti <br /> It appears Yeti resident Muller C chromosomal <br /> element, evolutionary time progressively approached                      probe labelled nick-translation Cy3-dCTP (Amer- <br /> pericentric heterochromatin D. melanogaster                   sham). Species-specific PCR probes used FISH D. <br /> deep portion 2Rh.                                                                    pseudobscura D. virilis (see below). FISH procedures were <br /> doi:10.1371/journal.pone.0113010.g008                                                   performed according Dimitri [42]. Digital images obtained <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                          8                          November 2014 | Volume 9 | Issue 11 | e113010 <br />                                                                                                                                  Evolution Drosophila Yeti Gene <br />  <br />  <br /> using Olympus epifluorescence microscope equipped                              Testing signatures selection <br /> cooled CCD camera. Gray scale images, obtained separately                                   A codon-based test selection conducted MEGA 6.0 <br /> recording Cy3 DAPI fluorescence specific filters,                            [43], using Nei-Gojobori method [44]. The statistic test (dS &#226;&#8364;&#8220; <br /> pseudo colored merged final image using Adobe                            dN) expected zero null hypothesis neutrality. <br /> Photoshop software.                                                                      dS dN numbers synonymous nonsynonymous <br />                                                                                          substitutions site, respectively. Three analyses performed. <br /> Nucleic acid manipulation sequence alignments                                        The analysis involved coding sequences Yeti gene <br />    Genomic DNA extraction PCR performed according                               different Drosophila species (D. melanogaster, D. <br /> to protocol Berkeley Drosophila Genome Project (http://                           simulans, D. erecta, D. pseudoobscura D. virilis). The other <br /> www.fruitfly.org/about/methods/index.html). To PCR amplify                               analyses involved Drosophila species where <br /> Yeti genomic fragments D. melanogaster, D. simulans strains,                        Yeti gene heterochromatic (D. melanogaster, D. simulans and <br /> D. sechellia D. tessieri following primers used: F: 59-                     D. erecta) species Yeti gene euchromatic (D. <br /> TTAGTATGGCCTGCGAGACA-39; R: 59-TGTGCTCGCA-                                               pseudoobscura D. virilis). <br /> TAACAAAGGC-39. <br />    PCR cycle were: 400 98uC, 40 x (100 98uC, 300 58uC, 80 <br />                                                                                          Acknowledgments <br /> 72uC). Amplified fragments gel purified sequenced by <br /> Bio-Fab research s.r.l. The Yeti probes D. pseudoobscura                        We grateful Patrizia Lavia critical reading manuscript. We <br /> D. virilis generated PCR genomic DNA                               thank Sergio Pimpinelli, Chantal Vaury Pierre Capy gift of <br /> following primers:                                                                       Drosophila strains. <br />    Dpse-F 59-GCGACGATGATAGCATCAAT-39; Dpse-R 59- <br /> GTGAGTGCTCAGCTGCTCAT-39                                                                  Author Contributions <br />    Dvir-F 59-AGCTAAACGTAGCACGCGTC-39; Dvir-R 59-                                         Conceived designed experiments: RC EC FC RM PD. Performed <br /> TGTGTACGCAGATCCTCGTC-39                                                                  experiments: RC EC FC RM PD. Analyzed data: RC EC FC RM <br />    PCR cycle were: 49 94uC, 35 x (300 95uC, 450 60uC, 300                             PD. Contributed reagents/materials/analysis tools: RC FC PD. Wrote the <br /> 72uC). Amplified fragments cloned pGEM-T vector                                  paper: PD. <br /> (Promega) verified DNA sequencing. <br />    Multiple sequence alignment performed ClustalW <br /> procedure available EMBL-EBI (http://www.ebi.ac.uk) with <br /> the multialin interface http://multalin.toulouse.inra.fr. <br />  <br /> References <br />  1. Moritz KB, Roth GE (1976) Complexity germline somatic DNA                  19. Hoskins RA, Smith CD, Carlson JW, Carvalho AB, Halpern A, et al. (2002) <br />     Ascaris. Nature 259: 55&#226;&#8364;&#8220;57.                                                              Heterochromatic sequences Drosophila whole-genome shotgun assembly. <br />  2. Peterson DG, Pearson WR, Stack SM (1998) Characterization tomato                  Genome Biology 3: research0085.1&#226;&#8364;&#8220;0085.16. <br />     (Lycopersicon esculentum) genome using vitro situ DNA reassociation.       20. Hoskins RA, Carlson JW, Kennedy C, Acevedo D, Evans-Holm M, et al. (2007) <br />     Genome 41: 346&#226;&#8364;&#8220;356.                                                                      Sequence finishing mapping Drosophila melanogaster heterochromatin. <br />  3. Dimitri P, Caizzi R, Giordano E, Carmela Accardo M, Lattanzi, et al. (2009)              Science 316: 1625&#226;&#8364;&#8220;1628. <br />     Constitutive heterochromatin: surprising variety expressed sequences.           21. Smith CD, Shu S, Mungall CJ, Karpen GH (2007) The Release 5.1 annotation <br />     Chromosoma 118: 419&#226;&#8364;&#8220;35.                                                                  Drosophila melanogaster heterochromatin. Science 316: 1586&#226;&#8364;&#8220;1591. <br />  4. Heitz E (1928) Das Heterochromatin der Moose. Jb Wiss Bot 69: 762&#226;&#8364;&#8220;818.               22. He B, Caudy A, Parsons L, Rosebrock A, Pane A, et al. (2012) Mapping the <br />  5. John B (1988) The biology heterochromatin. In Heterochromatin: Molecular              pericentric heterochromatin comparative genomic hybridization analysis and <br />     Structural aspects ed. V. R. S., Cambridge University Press. p. 1&#226;&#8364;&#8220;128.               chromosome deletions Drosophila melanogaster. Genome Res 22: 2507&#226;&#8364;&#8220; <br />  6. Williams SM, Robbins LG (1992) Molecular genetic analysis Drosophila                  2519. <br />     rRNA arrays. Trends Genet 8: 335&#226;&#8364;&#8220;340.                                                23. Yasuhara JC, DeCrease CH, Wakimoto BT (2005) Evolution heterochro- <br />  7. Dernburg AF, Sedat JW, Hawley RS (1996). Direct evidence role                   matic genes Drosophila. Proc Natl Acad Sci USA 102: 10958&#226;&#8364;&#8220;10963. <br />     heterochromatin meiotic chromosome segregation. Cell 86: 135&#226;&#8364;&#8220;146.                 24. Schulze SR, McAllister BF, Sinclair DA, Fitzpatrick KA, Marchetti M, et al. <br />  8. Elgin SCR (1996) Heterochromatin gene regulation Drosophila. Current              (2006) Heterochromatic genes Drosophila: comparative analysis two <br />     Opinion Genetics Development 6: 193&#226;&#8364;&#8220;200.                                          genes. Genetics 173: 1433&#226;&#8364;&#8220;1445. <br />  9. Karpen GH, Le MG, Le H (1996) Centric heterochromatin efficiency          25. Cenci G, Belloni G, Dimitri P (2003) 1(2)41Aa, heterochromatic gene of <br />     achiasmate disjunction Drosophila female meiosis. Science 273: 118&#226;&#8364;&#8220;122.               Drosophila melanogaster, required mitotic meiotic chromosome <br /> 10. Pardue ML (2007) Following chromosome path garden genome.              condensation. Genet Res 81: 15&#226;&#8364;&#8220;24. <br />     Annu Rev Cell Dev Biol. 2007; 23: 1&#226;&#8364;&#8220;22.                                              26. Messina G, Damia E, Fanti L, Atterrato MT, Celauro E, et al. (2014) Yeti, an <br /> 11. Villasante A, Mende&#194;&#180;z-Lago M, Abad JP, Montejo Garc&#196;&#177;`ni E (2007) The birth            essential Drosophila melanogaster gene, encodes protein required for <br />     centromere. Cell Cycle 6: 2872&#226;&#8364;&#8220;2876.                                              chromatin organization. J Cell Sci 127: 2577&#226;&#8364;&#8220;2588. doi:10.1242/jcs.150243 <br /> 12. Gatti M, Pimpinelli S (1992). Functional elements Drosophila melanogaster             Epub 2014 Mar 20. <br />     heterochromatin. Annu Rev Genet 26: 239&#226;&#8364;&#8220;275.                                         27. Dimitri P, Junakovic N, Arca` B (2003) Colonization heterochromatic genes by <br /> 13. Hilliker AJ (1976) Genetic analysis centromeric heterochromatin                transposable elements Drosophila. Mol Biol Evol 20: 503&#226;&#8364;&#8220;512. <br />     chromosome 2 Drosophila melanogaster: deficiency mapping EMS-induced           28. Dimitri P, Corradini N, Rossi F, Vern&#196;&#177;` F, Cenci G, et al. (2003) Vital genes in <br />     lethal complementation groups. Genetics 83: 765&#226;&#8364;&#8220;782.                                     heterochromatin chromosomes 2 3 Drosophila melanogaster. <br /> 14. Marchant GE, Holm DG (1988) Genetic analysis heterochromatin                   Genetica. 117: 209&#226;&#8364;&#8220;215. <br />     chromosome 3 Drosophila melanogaster. II. Vital loci identified EMS       29. Rossi F, Moschetti R, Caizzi R, Corradini N, Dimitri P (2007) Cytogenetic and <br />     mutagenesis. Genetics 120: 519&#226;&#8364;&#8220;532.                                                      molecular characterization heterochromatin gene models Drosophila <br /> 15. Dimitri P (1991) Cytogenetic analysis second chromosome heterochro-               melanogaster. Genetics 175: 595&#226;&#8364;&#8220;607. <br />     matin Drosophila melanogaster. Genetics 127: 553&#226;&#8364;&#8220;564.                             30. Zhang P, Spradling AC (1995) The Drosophila salivary gland chromocenter <br /> 16. Koryakov DE, Zhimulev IF, Dimitri P (2002) Cytogenetic analysis             contains highly polytenized subdomains mitotic heterochromatin. Genetics. <br />     chromosome heterochromatin Drosophila melanogaster. Genetics 160: 509&#226;&#8364;&#8220;                1995 Feb; 139(2): 659&#226;&#8364;&#8220;70. <br />     517.                                                                                 31. Berghella L, Dimitri P (1996) The heterochromatic rolled gene Drosophila <br /> 17. Fitzpatrick KA, Sinclair DA, Schulze SR, Syrzycka M, Honda BM (2005) A                   melanogaster extensively polytenized transcriptionally active the <br />     genetic molecular profile chromosome centric heterochromatin             salivary gland chromocenter. Genetics. 1996 Sep; 144(1): 117&#226;&#8364;&#8220;25. <br />     Drosophila melanogaster. Genome 48: 571&#226;&#8364;&#8220;584.                                         32. Schaeffer SW, Bhutkar A, McAllister BF, Matsuda M, Matzkin LM, et al. (2008) <br /> 18. Adams MD, Celniker SE, Holt RA, Evans CA, Gocayne JD, et al. (2000) The                  Polytene chromosomal maps 11 Drosophila species: order genomic <br />     genome sequence Drosophila melanogaster. Science 287: 2185&#226;&#8364;&#8220;2195.                      scaffolds inferred genetic physical maps. Genetics 179: 1601&#226;&#8364;&#8220;55. <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                           9                         November 2014 | Volume 9 | Issue 11 | e113010 <br />                                                                                                                                  Evolution Drosophila Yeti Gene <br />  <br />  <br /> 33. Waterhouse RM, Zdobnov EM, Tegenfeldt F, Li J, Kriventseva EV (2011)                  40. Brizuela BJ, Elfring L, Ballard J, Tamkun JW, Kennison JA (1994) Genetic <br />     OrthoDB: hierarchical catalog eukaryotic orthologs 2011. Nucleic                analysis brahma gene Drosophila melanogaster polytene <br />     Acids Res. 39(Database issue): D283&#226;&#8364;&#8220;8.                                                    chromosome subdivisions 72AB. Genetics 137: 803&#226;&#8364;&#8220;813. <br /> 34. Iwashita S, Osada N, Itoh T, Sezaki M, Oshima K, et al. (2003). A transposable        41. Pardue ML (1986) In situ hybridization DNA chromosome nuclei. In: <br />     element-mediated gene divergence directly produces novel type bovine               Roberts DB (ed) Drosophila: practical approach. IRL Press, Oxford, pp. 111&#226;&#8364;&#8220; <br />     Bcnt protein including endonuclease domain RTE-1. Mol Biol Evol 20:                137. <br />     1556&#226;&#8364;&#8220;1563.                                                               <br /> </body></html>