<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Mol Cell. 2008 April 25; 30(2): 248&#226;&#8364;&#8220;258. doi:10.1016/j.molcel.2008.02.021. <br />  <br />  <br />  <br />                            Toward Comprehensive Temperature-Sensitive Mutant <br />                            Repository Essential Genes Saccharomyces cerevisiae <br />                            Shay Ben-Aroya1, Candice Coombes2, Teresa Kwok1, Kathryn A. O&#226;&#8364;&#8482;Donnell2, Jef D. <br />                            Boeke2,*, Philip Hieter1,* <br />                            1Michael    Smith Laboratories, University British Columbia, Vancouver, BC V6T 1Z4, Canada <br />                            2HighThroughput Biology Center, The Johns Hopkins University School Medicine, Baltimore, <br />                            MD 21205, USA <br />  <br />                            SUMMARY <br />                                 The Saccharomyces cereivisiae gene deletion project revealed approximately 20% yeast <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 genes required viability. The analysis essential genes traditionally relies conditional <br />                                 mutants, typically temperature-sensitive (ts) alleles. We developed systematic approach (termed <br />                                 &#226;&#8364;&#339;diploid shuffle&#226;&#8364;?) useful generating ts allele essential gene S. cerevisiae for <br />                                 improved genetic manipulation mutant alleles gene constructs general. Importantly, each <br />                                 ts allele resides normal genomic locus, flanked specific cognate UPTAG DNTAG bar <br />                                 codes. A subset 250 ts mutants, including ts alleles uncharacterized essential genes and <br />                                 prioritized genes human counterparts, ready distribution. The importance of <br />                                 collection demonstrated biochemical genetic screens reveal essential genes <br />                                 involved RNA processing maintenance chromosomal stability. <br />  <br />  <br />                            INTRODUCTION <br />                                              Deciphering functional roles encoded gene products genome central <br />                                              challenge understanding biology organism. The study mutant phenotypes has <br />                                              proven fundamental approach obtaining detailed understanding gene <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              function. While genetic screens based random mutagenesis seminal in <br />                                              identifying genes involved biological processes, random mutagenesis approach rarely <br />                                              achieves saturation, mutability, resulting haploid viable mutants, varies widely <br />                                              genes. <br />  <br />                                              The efficiency mutant <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>phenotype</span> studies greatly accelerated yeast <br />                                              Saccharomyces cerevisiae following creation comprehensive Saccharomyces gene <br />                                              deletion (YKO) collection consortium laboratories. This collection includes isogenic <br />                                              knockouts virtually annotated yeast ORFs, replaced kanMX4 <br />  <br />  <br />  <br />                            &#194;&#169; 2008 Elsevier Inc. <br />                            * <br />                              Correspondence: jboeke@jhmi.edu (J.D.B.), hieter@msl.ubc.ca (P.H.). <br />                            SUPPLEMENTAL DATA <br />                            Supplemental Data include Supplemental Experimental Procedures, figures, tables article <br />                            online http://www.molecule.org/cgi/content/full/30/2/248/DC1/. <br />                             Ben-Aroya et al.                                                                                              Page 2 <br />  <br />  <br />                                               cassette confers resistance G418 (Giaever et al., 2002). One powerful <br />                                               features YKO collection systematic incorporation 20-mer molecular bar <br />                                               codes (UPTAG DNTAG) unique YKO strain (Shoemaker et al., 1996; Winzeler <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               et al., 1999). Functional profiling populations using TAG microarrays greatly <br />                                               expedited genetic screens quantitative (Birrell et al., 2001; Lum et al., <br />                                               2004; Ooi et al., 2001). <br />  <br />                                               The YKO mutants valuable resource genome-wide functional analysis 4700 <br />                                               nonessential genes; however, equivalent systematic bar-coded mutant collection exists <br />                                               1107 essential genes. Such collection essential gene haploid mutants be <br />                                               scientifically valuable, enabling systematic genome-wide genetic screens, study of <br />                                               individual genes, comprehensive view genetic interaction networks. <br />  <br />                                               Genetic analysis essential genes typically relies conditional mutants carrying point <br />                                               mutations, retain function specific gene set conditions <br />                                               (permissive), lack function different set conditions (nonpermissive), and <br />                                               exhibit partial (hypomorphic) function semipermissive conditions. Conditional alleles <br />                                               generated replacement native promoter gene can <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               rapidly repressed, tetracycline (tet)-regulatable promoter. This approach has <br />                                               recently used generate collection promoter-shutoff strains two-thirds the <br />                                               essential yeast gene (Mnaimneh et al., 2004). Another approach technique termed <br />                                               decreased abundance mRNA perturbation (DAmP); method, 3&#226;&#8364;&#178;UTR an <br />                                               essential gene disrupted insertion antibiotic-resistance marker, destabilizing the <br />                                               target mRNA. The result protein potentially natural transcriptional regulation <br />                                               reduced levels (Schuldiner et al., 2005). <br />  <br />                                               Traditionally, temperature-sensitive (ts) alleles used great success the <br />                                               genetic analysis essential proteins, remain mainstay genetic analysis <br />                                               (Hartwell et al., 1970; Simchen, 1978). In vitro mutagenesis method known as <br />                                               &#226;&#8364;&#339;plasmid shuffling&#226;&#8364;? widely used generate ts mutations individual genes interest <br />                                               (Sikorski Boeke, 1991); however, method easily scaled encompass the <br />                                               genome, generated alleles integrated genome not <br />                                               capitalize bar <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span> strategy. An alternative method constructing ts alleles involves <br />                                               engineering chromosomal locus gene amino terminus encoded <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               protein fused &#226;&#8364;&#339;heat-inducible degron.&#226;&#8364;? The fusion protein targeted degradation <br />                                               37&#194;&#176;C, modulates stability protein (Kanemaki et al., 2003; Labib et al., <br />                                               2000). The corresponding mutants, termed td (for temperature-activated degron) (Dohmen <br />                                               Varshavsky, 2005), exhibit lethality high temperature approximately 50% the <br />                                               essential genes tested. <br />  <br />                                               In paper, efficient, scalable method generate ts alleles the <br />                                               essential genes S. cerevisiae. The resultant strains isogenic YKO deletion <br />                                               collection. Each ts alleles present normal genomic locus flanked by <br />                                               specific cognate UPTAG DNTAG bar codes. In addition, basic principles the <br />                                               diploid shuffle method corollaries broadly applicable yeast molecular genetic <br />                                               analysis. We used method generate ts alleles 250 essential genes, 45 of <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                           Page 3 <br />  <br />  <br />                                               encode proteins relatively unknown function remaining alleles <br />                                               prioritized based sequence similarity human proteins. We biochemical <br />                                               genetic screens ts mutants corresponding 45 genes unknown function <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               used. These screens reveal new mutants affect RNA processing chromosome <br />                                               transmission strongly validate use ordered array resource biological <br />                                               discovery. <br />  <br />  <br />                               RESULTS <br />                               Diploid Shuffle&#226;&#8364;&#8221;A Systematic Method Generating ts Alleles Essential Genes in <br />                               S. cereviviae <br />                                               Our goal develop systematic approach screen missense mutations result <br />                                               ts alleles essential gene endogenous chromosomal location flanked with <br />                                               appropriate bar codes. <br />  <br />                                               The step generating ts allele Your Favorite Essential Gene (YFEG) use of <br />                                               genomic DNA template amplify open reading frame (ORF) including 5&#226;&#8364;&#178; <br />                                               promoter 3&#226;&#8364;&#178; terminator regions using mutagenic amplification (Figure 1A). The <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               mutagenized PCR product step 1 cloned SB221+Topo-TA. This plasmid <br />                                               contains URA3 gene flanked 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; regions KanMX. The Topo-TA cloning <br />                                               site (Invitrogen) inserted KanMX 5&#226;&#8364;&#178; end URA3 gene. This <br />                                               site allows direct cloning of PCR products need further <br />                                               modifications. The result cloning step library mutagenized YFEG, is <br />                                               transformed E. coli digested release linear fragments (following DNA <br />                                               purification). The linear fragments directly transformed corresponding strain <br />                                               haploid convertible &#226;&#8364;&#339;heterozygous&#226;&#8364;? diploid YKO collection (Figures 1B 1C). <br />                                               This collection individually incorporating modified SGA reporter <br />                                               can1&#206;&#8221;::LEU2-MFA1pr-HIS3 set heterozygous diploid YKOs originally <br />                                               constructed Saccharomyces Genome Deletion project (available Open <br />                                               Biosystems, J.D.B., S. Sookhai-Mahadeo, X. Wang, C. Tiffany, O. Chen, F.A. Spencer, D.S. <br />                                               Yuan, P.B. Meluh, unpublished data). The ~700 bp KanMX5&#226;&#8364;&#178; KanMX3&#226;&#8364;&#178; fragments <br />                                               direct mutagenized YFEG library yfeg&#206;&#8221;::KanMX genomic locus homologous <br />                                               recombination (Figure 1C). Ura+ transformants selected. Pools Ura+ cells containing <br />                                               mutant alleles sporulated (Figure 1D). Spores spread haploid-selective <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               medium incubated 25&#194;&#176;C 5 days colony formation (Figure 1D). Only haploid <br />                                               MATa Ura+ spores containing integrated mutagenized allele YFEG grow this <br />                                               medium. Colonies formed selective medium replica plated incubated 37&#194;&#176;C for <br />                                               2 days (Figure 1E). Colonies growing 25&#194;&#176;C, 37&#194;&#176;C, selected potential ts <br />                                               alleles restreaked temperatures retesting. Retesting backcrossing <br />                                               potential ts allele donor strain (see Experimental Procedures). Following <br />                                               sporulation tetrad dissection, confirmed (1) ts phenotype segregates a <br />                                               Mendelian manner (2:2), indicative depends single mutated gene; (2) linked <br />                                               URA3 cosegregates mutated PCR product; (3) mutagenized <br />                                               PCR product integrated correct genomic locus, rendering cells G418 sensitive <br />                                               (5&#226;&#8364;&#178;kanMX::yfeg-ts-URA3::3&#226;&#8364;&#178;kanMX). <br />  <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                             Page 4 <br />  <br />  <br />                                               In summary, final product method described confirmed MATa <br />                                               strain YKO collection genetic background containing URA3-marked ts allele a <br />                                               specific essential gene integrated endogenous locus flanked bar codes. In <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               addition, diploid shuffle method variations (see Figures S3 S4 available <br />                                               online Discussion) represent significantly improved methods generating strains <br />                                               carrying specific alleles, transferring ts alleles variety strain backgrounds and <br />                                               genetic contexts, introduction gene constructs fusion proteins, <br />                                               heterozygous expression cassettes, etc. <br />  <br />                               Construction ts Strains 45 Essential Genes Unknown Function <br />                                               We generated initial collection ts strains provide early-stage resource value for <br />                                               community genetic phenotypic screening. We chose 45 yeast ORFs known time <br />                                               started essential viability defined S. cerevisiae Genome Data <br />                                               <span id='am-239' about='xsp:base' typeof='owl:Thing'>Base</span> (SGD) uncharacterized ORFs, proteins unknown function unknown <br />                                               molecular function. Thirty ORFs (67%) significant human homolog an <br />                                               E value ranges 3.00E-05 1.00E-261 (Table S1). <br />  <br />                                               For gene, ~15,000 independent PCR-mutagenized clones screened potential ts <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               phenotypes. In 42 45 cases, successfully isolated ts allele round of <br />                                               screening. In remaining cases, second round mutagenesis required, <br />                                               resulting successful isolation ts mutants genes attempted. By <br />                                               sequencing 12 independent ts alleles, average <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>mutation</span> frequency 0.54 <br />                                               base substitution 100 bp, translates average 0.9 mutations 100 amino <br />                                               acids (Figure S1). <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> demonstrate potential new discovery, screened this <br />                                               collection 45 ts mutants using established biochemical genetic screens used <br />                                               extensively past. <br />  <br />                               Screen RNA-Processing Defects Reveals Previously Unidentified Mutants <br />                                               Production ribosomal RNA eukaryotic cells considerably complex in <br />                                               prokaryotes, single enzyme, RNase III, makes needed cleavages ribosome <br />                                               assembly cotranscriptional. Although eukaryotic rRNA organized similarly, 35S <br />                                               precursor, cleavages carried protein ribonucleoprotein complexes <br />                                               (including RNase III [Kufel et al., 1999]). In eukaryotes, molecularly intertwined steps <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               rRNA processing ribosome assembly slower, steps are <br />                                               posttranscriptional. The combined molecular weight small nucleolar ribonucloprotein <br />                                               particle (snoRNP) complexes involved ribosome assembly (not present prokaryotes) <br />                                               exceeds molecular weight ribosome &gt;15-fold (Venema Tollervey, 1999). <br />                                               Production rRNA nucleolus, transport r proteins (and of) the <br />                                               nucleus, imposes requirements assembly absent prokaryotes (Hage Tollervey, <br />                                               2004). At 170 proteins 100 small RNAs estimated participate eukaryotic <br />                                               rRNA processing ribosome assembly (Andersen et al., 2002; Fromont-Racine et al., <br />                                               2003; Venema Tollervey, 1999) addition 79 proteins RNAs make <br />                                               ribosome. <br />  <br />  <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                         Page 5 <br />  <br />  <br />                                               Because complexity RNA-processing defects easily identified by <br />                                               simple RNA-blotting techniques (Lindahl et al., 1992; Peng et al., 2003), hypothesized <br />                                               unknown essential genes contribute RNA processing. We <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               performed specific assays detect defects rRNA, mRNA, snoRNA, snRNA <br />                                               processing collection 45 ts mutants (relatively) unknown function (Table S1) <br />                                               positive control nme1 ts mutant (a kind gift Mark Schmitt). Mutants were <br />                                               grown permissive temperature shifted nonpermissive temperature 2 hr, and <br />                                               total RNA harvested, resolved appropriate gels, blotted series of <br />                                               hybridization probes detect appropriate RNA precursors. Processing defects are <br />                                               readily revealed buildup intermediates larger mature RNA species under <br />                                               study (Table S2). Consistent large number known genes affecting rRNA <br />                                               processing, identified new ts mutants obvious rRNA-processing defects (Figure <br />                                               4). These revealed probing internal transcribed spacer 1 (ITS1) sequences <br />                                               (Figure 2) mature rRNA probes (Figure S2, Table 1). No ts mutants severely affecting <br />                                               pre-mRNA, snoRNA, snRNA-processing pathways detected set of <br />                                               mutants. <br />  <br />                                               As apparent Figure 2 Table 1, rRNA-processing mutants ranging nearly as <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               severe control (the nme1 mutant) milder defects observed as <br />                                               hyperaccumulation 35S, 33S, and, cases, reduced accumulation 27S precursor <br />                                               rRNA species. Most mutants significant deficit mature 18S 25S <br />                                               rRNA, suggesting direct involvement rRNA processing (Figure S2 Table S3). The <br />                                               proteins corresponding mutants, ydr339c yor004w, biochemically <br />                                               identified important components, Utp23p 24p, small-subunit rRNA-processing <br />                                               machinery (Bleichert et al., 2006; Rempola et al., 2006) course these <br />                                               experiments, providing critical internal validation assay approach. <br />                                               While clearly integral parts rRNA-processing machinery, mutants may <br />                                               affect rRNA processing directly. There mechanisms; half all <br />                                               yeast&#226;&#8364;&#8482;s energy goes ribosomes. For example, defects secretion downregulate <br />                                               ribosome biogenesis (Mizuta Warner, 1994). <br />  <br />                               Crosstalk RNA-Processing Pathways <br />                                               We searched pathway connections newly identified ts mutants. YJL069C <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               (UTP18) previously predicted function processing U3 snoRNA (Bernstein et al., <br />                                               2004; Samanta Liang, 2003). The rRNA-processing defect yjl069-ts provides genetic <br />                                               evidence supporting hypothesis, U3 snoRNA essential rRNA processing. <br />                                               Similarly, YGR145W (ENP2) gene encodes protein known interact MPP10p <br />                                               component SSU processome (SGD); SSU processome responsible for <br />                                               processing 18S rRNA. The protein encoded YKR022C (NTR2) reportedly a <br />                                               component Prp43 helicase spliceosome disassembly complex (Tsai et al., 2005); our <br />                                               data suggest facilitate rRNA processing, accord recent studies <br />                                               (Combs et al., 2006; Lebaron et al., 2005; Leeds et al., 2006). Interestingly, YKR022C <br />                                               gene identified here, YLR132C, predicted pre-mRNA splicing factors a <br />                                               computational analysis (Hazbun et al., 2003), observed defects. The <br />                                               YAL043C (PTA1) gene encodes subunit holo-CPF complex involved mRNA 3&#226;&#8364;&#178; <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                            Page 6 <br />  <br />  <br />                                               end formation/polyadenylation (Zhao et al., 1999) tRNA biogenesis (O&#226;&#8364;&#8482;Connor and <br />                                               Peebles, 1992). The discovery set proteins appear function multiple RNA- <br />                                               processing networks exciting, consistent prior studies (Burckin et al., 2005; Leeds et <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               al., 2006), provides new evidence cross-RNA-processing pathway interactions. <br />  <br />                               A Connection Coenzyme A Biosynthesis rRNA Processing? <br />                                               Two rRNA-processing mutants ts genes predicted roles CoA <br />                                               biosynthesis (YDR196C YGR277C), based homology prokaryotic CoA <br />                                               biosynthesis proteins. The observed defects relatively specific rRNA processing and <br />                                               represent general pleiotropic defect. It likely CoA derivative, rather <br />                                               proteins se, affects (directly indirectly) rRNA processing. <br />  <br />                               Screens Chromosome Instability <br />                                               S. cerevisiae attractive identifying characterizing genes involved chromosome <br />                                               segregation. Recently, systematic screening 4700 nonessential gene deletion mutants <br />                                               identified 200 nonessential genes contributing faithful chromosome <br />                                               segregation (Yuen et al., 2007), systematic screening essential gene mutants has <br />                                               possible. We decided search novel CIN genes screening 45 ts <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               mutant collection CIN phenotype. Three different tests used (Figure 3): the <br />                                               chromosome transmission fidelity (CTF) (Spencer et al., 1990) a-like faker (ALF) <br />                                               (Gerring et al., 1990) assays haploids diploid bimater test (BiM) (Kouprina et al., <br />                                               1988) MATa/MAT&#206;&#177; diploids. Screening different assays broadens the <br />                                               spectrum identifiable genes allowing various defects chromosome transmission to <br />                                               detected (Yuen et al., 2007). <br />  <br />                                               For CIN assays, specific ts alleles tested wide range semi- and <br />                                               nonpermissive temperatures (25&#194;&#176;C, 30&#194;&#176;C, 32&#194;&#176;C, 34&#194;&#176;C, 37&#194;&#176;C). In total, 45 ts <br />                                               mutants (20%) exhibited increased chromosome instability (CIN) defined new <br />                                               CIN genes (Table 2 Figure 4). Typical results presented mutants <br />                                               (Figure 3). The severity CIN phenotypes quantified Table 2. Six mutants <br />                                               (ynl152w, yhr020w, keg1, yhr122w, dre2, yor262w) define uncharacterized ORFs. <br />                                               Three additional mutants, swc4, trs23, kae1, previously identified coding for <br />                                               components Swr1 (Krogan et al., 2003), TRAPP (Sacher et al., 2000), KEOPS <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               (Bianchi Shore, 2006) complexes, respectively, molecular function still <br />                                               unknown. <br />  <br />                                               We wanted estimate frequently expect ts mutant (isolated random <br />                                               mutagenesis procedure) subsequently reveal detectable hypomorphic CIN phenotype at <br />                                               reduced (semipermissive) temperature. This addresses issue false negatives upon <br />                                               phenotypic screening hypomorphs. To accomplish this, generated ts alleles PSF2 <br />                                               NSE4 tested CIN phenotype range reduced temperatures. Psf2 <br />                                               represents known subunit GINS complex (Sld5p, Psf1p, Psf2p, Psf3p) (Takayama et <br />                                               al., 2003). Loss Psf2p function depletion deletion shown cause <br />                                               chromosome missegregation Schizosaccharomyces pombe human cells (Huang et <br />                                               al., 2005). Nse4 known interact Nse3 Nse5 components Mms21- <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                            Page 7 <br />  <br />  <br />                                               Smc5-Smc6 DNA repair complex. It suggested Nse4 plays role in <br />                                               maintenance higher-order chromosome structure (Hu et al., 2005). As shown Figure 4 <br />                                               Table 2, newly constructed ts alleles psf2 nse4 expected CIN <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               phenotype semipermissive temperatures. In addition, similar phenotypes caused by <br />                                               newly constructed ts alleles components GINS complex (Psf1 Sld5) <br />                                               Mms21-Smc5-Smc6 complex (Nse3 Nse5). Thus, able isolate new ts <br />                                               alleles subsequently detect CIN phenotype semipermissive temperatures all <br />                                               alleles genes tested. <br />  <br />                               Four Previously Unidentified Essential Genes Involved Sister Chromatid Cohesion <br />                                               To test sister chromatid cohesion defects specific CIN mutants, mutant was <br />                                               crossed tet repressor-GFP/tet operator repeat strain (Michaelis et al., 1997). The <br />                                               markers strain allow visualization specific chromosomal locus assess whether <br />                                               sister chromatids juxtaposition separated. The resulting diploids were <br />                                               sporulated, haploid strains containing appropriate markers identified and <br />                                               analyzed cohesion defects. <br />  <br />                                               In wild-type, 8% cells arrested G2/M GFP signals, indicating sister <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               chromatids efficiently paired prior anaphase. As expected (Mayer et al., 2001), ctf18 <br />                                               control strain, 35% cells arrested G2/M GFP signals, indicating sister <br />                                               chromatid cohesion defect. Of new ts mutants exhibiting CIN phenotype, four <br />                                               (yor262w, dre2, swc4, yhr122w) associated sister chromatid cohesion defects <br />                                               values ranging 23% 44% (Figure 5A). These defects sister <br />                                               chromatid cohesion defects opposed aneuploidy; 4%&#226;&#8364;&#8220;7% wild-type ts <br />                                               mutant cells arrested G1 GFP signals. <br />  <br />                               YHR122W, SWC4, YOR262W Are Required Establishment Sister Chromatid <br />                               Cohesion <br />                                               Sister chromatid cohesion established S phase maintained anaphase onset. <br />                                               Three mutants sister chromatid cohesion defects (yhr122w, swc4, yor262w) were <br />                                               used temperature-shift experiments assess roles establishment maintenance of <br />                                               cohesion. The experimental design described Figure 5B. When gene interest <br />                                               affects establishment, results obtained sample 3 resemble sample 1, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               whereas, maintenance defects, results sample 3 resemble sample 2. For all <br />                                               ts mutants, values GFP signals sample 3 similar obtained in <br />                                               sample 1, sample 2 (Figure 5C). We conclude essential genes may <br />                                               play role establishment sister chromatid cohesion. <br />  <br />                               A Resource 250 Essential Gene ts Mutants <br />                                               We expanded initial collection 45 ts mutants additional 205 essential genes. We <br />                                               decided focus essential genes ts alleles available the <br />                                               yeast community. To assess ts allele available individual ORF, we <br />                                               referred updated list ts alleles collected Boone laboratory (C. Boone, personal <br />                                               communication). This collection represents ~40% essential gene spectrum <br />                                               corresponding 450 essential genes. These extant alleles generated yeast <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                            Page 8 <br />  <br />  <br />                                               community past 40 years transferred YKO genetic <br />                                               background non-bar-coded format (C. Boone, personal communication). Of the <br />                                               remaining 657 essential genes ts alleles currently available, chose 131 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               genes homology human sequence (E value range 3.00E-05 1.00E-261) 74 <br />                                               additional genes (chosen randomly) add collection (see Table S4) generated ts <br />                                               alleles them. In total, 250 ts mutants array format available the <br />                                               community request. For ts allele, matched MATa MAT&#206;&#177; strains available. <br />  <br />  <br />                               DISCUSSION <br />                                               We method making comprehensive set ts conditional mutants of <br />                                               essential genes Saccharomyces cerevisiae. To demonstrate feasibility create an <br />                                               initial community resource, generated subset ts alleles essential genes with <br />                                               unknown function. We used biochemical (RNA processing) genetic (chromosomal <br />                                               stability sister chromatid cohesion) screens identify new genes important these <br />                                               essential functions. The initial collection 45 ts mutants expanded 250 strains, in <br />                                               essential genes significant human homologs prioritized. The collection is <br />                                               ready distribution. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Generating ts alleles individual genes conventional methods tedious readily <br />                                               scalable genome-wide. The approach described exploits features &#226;&#8364;&#339;haploid- <br />                                               convertible&#226;&#8364;? heterozygous diploid collection, using SGA reporter select an <br />                                               integrated ts-allele flanked corresponding bar codes. We method diploid <br />                                               shuffle distinguish standard plasmid shuffle method. The Topo-TA modified <br />                                               plasmid saves time effort cloning step mutagenized library. The approach <br />                                               validated generating ts alleles; however, used applications. Using <br />                                               Topo-TA plasmid protocol described (Figure 1), extant ts allele easily <br />                                               transferred deletion collection genetic background (&#226;&#8364;&#339;allele transfer-in&#226;&#8364;?). The result is <br />                                               integrated allele, marked URA3, flanked appropriate bar codes (see Figure <br />                                               S3 details). Indeed, using Topo-TA plasmid, PCR product (mutant allele, <br />                                               fusion protein, heterologous gene expression cassette, etc.) introduced the <br />                                               6000 genomic sites carrying KanMX replacement cassette integration site, <br />                                               depending specific heterozygote chosen recipient strain. In addition, using <br />                                               primers external primers used original mutagenesis, URA3-marked ts allele <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               gene construct easily transferred deletion set genetic background any <br />                                               strain (&#226;&#8364;&#339;allele transfer-out,&#226;&#8364;? Figure S4). Thus, ts mutation gene <br />                                               construct analyzed specific phenotypes various genetic <br />                                               contexts. <br />  <br />                                               In model organisms, function known gene predicted ORF analyzed by <br />                                               assessing specific phenotype gene ORF deleted inactivated. A most <br />                                               important milestone yeast functional genomics complete set deletion mutants <br />                                               (Giaever et al., 2002). This collection widely used direct screening recessive <br />                                               phenotypes haploids establishing genetic interactions haploid double mutants. <br />                                               However, systematic screens currently possible ~4700 nonessential <br />                                               genes. For essential genes, haploids carrying conditional hypomorphic alleles are <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                                Page 9 <br />  <br />  <br />                                               required. The screens first-release ts allele resource described paper <br />                                               demonstrate value systematic ts mutant resource. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               A screen ribosomal RNA processing netted 11 mutants affecting process relatively <br />                                               severely specifically. This considerably expands list approximately 110 genes <br />                                               previously identified using tet-regulated essential gene constructs microarray approach <br />                                               detecting rRNA precursors displaying increases rRNA precursor abundance at <br />                                               10-fold (Peng et al., 2003). In addition genes previously identified <br />                                               biochemically components processome, additional genes identified. <br />                                               Several gene products implicated forms RNA processing, implying <br />                                               crosstalk central rRNA, pre-mRNA, snoRNA, tRNA-processing pathways. <br />                                               Many identified genes products localized nucleus nucleolus expected <br />                                               proteins directly involved rRNA processing, showed significant defect in <br />                                               mature rRNA. However, mutants, reduction activity affected <br />                                               protein affect rRNA processing indirectly, e.g., affect processing snoRNAs, <br />                                               direct rRNA processing. <br />  <br />                                               Two mutants predicted affect CoA biosynthesis strongly affected rRNA processing, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               suggesting requirement CoA derivative rRNA processing. Two possible <br />                                               explanations specific protein acetylation regulates rRNA processing, or, <br />                                               alternatively, free CoA derivative serve signal metabolic <br />                                               state cell. It make sense decrease readily usable cellular carbon sources <br />                                               downregulated ribosomes/translation machinery. <br />  <br />                                               In paper, CIN screening ts mutant collection resource. Over the <br />                                               past 20 years, phenotype-based screens chromosome missegregation generated a <br />                                               number mutant collections (CTF, CHL, MCM, CIN), identifying genes that <br />                                               functionally conserved eukaryotes (Hoyt et al., 1990; Kouprina et al., 1988; Maine <br />                                               et al., 1984; Meeks-Wagner et al., 1986; Ouspenski et al., 1999; Spencer et al., 1990; Yuen <br />                                               et al., 2007). Results extended effort isolate mutants affect chromosome <br />                                               stability suggest yeast S. cerevisiae close saturation respect. Using <br />                                               subset ts alleles new collection, isolated new CIN mutants. The <br />                                               original ctf collection identified ~50 genes necessary high-fidelity segregation of <br />                                               chromosomes (Spencer et al., 1990). To date, 25 genes represented original ctf <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               collection cloned characterized, encode functions important for <br />                                               sister chromatid cohesion (40%). The connection sister chromatid cohesion <br />                                               function CIN emphasized secondary screen, nine <br />                                               new CIN mutants associated cohesion defect (Figure 5A). We a <br />                                               potential role cohesion mutants establishment stage (Skibbens et al., <br />                                               1999). Since Ctf7 shown involved establishment sister chromatid <br />                                               cohesion, essential genes role described. Based these <br />                                               results, believe early release subset 250 ts mutants provide valuable <br />                                               resource community enrich spectrum mutants affecting various cellular <br />                                               functions. <br />  <br />  <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                            Page 10 <br />  <br />  <br />                                               The obvious use resource study function individual genes, including <br />                                               poorly characterized ORFs. In broader view, collection used directly, no <br />                                               modifications, complete genome-wide studies used nonessential gene <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               deletion collection resource. Other applications include incorporation hypomorphic <br />                                               alleles synthetic genetic network. The MATa version collection contains a <br />                                               uniform genetic background (see Experimental Procedures), strains <br />                                               contains LEU2-marked SGA reporter. Therefore, ts collection used directly <br />                                               synthetic genetic array (SGA) (Tong et al., 2001) epistatic miniarray profiles (eMAPs) <br />                                               (Schuldiner et al., 2005). In cases, MATa strains provided array form to <br />                                               enrich deletion set. In addition, individual mutants used query for <br />                                               SGA analysis. An alternative approach SGA SLAM (Ooi et al., 2003). One the <br />                                               unique features collection integrated alleles flanked bar <br />                                               codes. Since growth rates ts mutants differ nonpermissive temperatures of <br />                                               different alleles vary, believe diploid version SLAM, dSLAM, preferable <br />                                               (Pan et al., 2004). In addition, ts mutants introduced query mutation <br />                                               transformation assess synthetic lethal profile (using allele transfer-out method, <br />                                               Figure S4). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Several approaches developed yeast community study essential <br />                                               genes, resulted collections available community: <br />                                               DAmP alleles (Schuldiner et al., 2005), N-degron (Kanemaki et al., 2003; Labib et al., <br />                                               2000), tet promoter collections (Mnaimneh et al., 2004). The main drawback our <br />                                               approach fact that, following integration mutagenized PCR product, a <br />                                               relatively laborious screening process needed isolate ts allele. In methods, <br />                                               final product ready use following simple step homologous recombination, <br />                                               avoiding screening. However, approach important advantage&#226;&#8364;&#8221;it promises a <br />                                               high rate success. If certain gene ts isolated round, second round <br />                                               screening, changing mutagenesis conditions, usually result positive isolate. <br />                                               The criterion defining mutant ts clear (inability grow high <br />                                               temperatures), ts mutations isolated gene interest. For other <br />                                               approaches, step needed modified gene, resultant allele not <br />                                               associated clear phenotype. For tet promoter collection, rapid and/or <br />                                               sufficient elimination protein function problematic large fraction proteins <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               (Aparicio, 2003). In nearly half cases tested, shutting promoter did affect <br />                                               phenotypes gave slight phenotype (Mnaimneh et al., 2004). When 104 <br />                                               essential genes fused N-degron sequences, nearly 40% did result in <br />                                               inviability nonpermissive temperature (Kanemaki et al., 2003), indicating the <br />                                               rapid protein depletion N-degron uniform proteome. Similarly, the <br />                                               DAmP approach works approximately one-third essential genes and <br />                                               majority. For cases effect observed following the <br />                                               modification, experimental options limited, rate success of <br />                                               methods lower obtained ts alleles. <br />  <br />                                               Genetic analysis essential genes typically relies conditional mutants. In cases, ts <br />                                               alleles provide wide spectrum activity ranging permissive nonpermissive <br />  <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                               Page 11 <br />  <br />  <br />                                               temperatures, providing flexibility phenotype assessment. In tet promoter strains, the <br />                                               promoter state, making difficult achieve intermediate expression <br />                                               levels (Schuldiner et al., 2005). DAmP alleles conditional alleles&#226;&#8364;&#8221;reduced <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               expression constitutive. Finally, method, integrated allele controlled its <br />                                               natural promoter terminator protein wild-type structure lacking any <br />                                               extensions tags. For tet DAmP strains, promoter terminator modified. <br />                                               In degron system, amino terminus encoded protein fused 20 KD <br />                                               fragment, affect function targeted protein. <br />  <br />                                               Taking considerations together, comprehensive collection ts missense mutations, <br />                                               integrated, bar-coded format, represents excellent resource analyzing essential <br />                                               gene function significantly enhance discovery, genome crossreferencing, and <br />                                               functional analysis. <br />  <br />  <br />                               EXPERIMENTAL PROCEDURES <br />                                               For details, Supplemental Experimental Procedures. <br />  <br />                               Yeast Strains <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               ts strains derived yeast heterozygous gene deletion collection carrying the <br />                                               haploid selection marker can1&#206;&#8221;::LEU2-MFA1pr-HIS3 (Open Biosystems catalogue number <br />                                               YSC4428). <br />  <br />                               Plasmids <br />                                               SB221 modified starting plasmid M4758 (Voth et al., 2003) allow more <br />                                               efficient targeted integration KanMX locus heterozygous deletion strains. The <br />                                               BamHII site M4758 modified Topo-TA site easier recombinant cloning. <br />  <br />                               RNA-Processing Mutant Screens <br />                                               Two independent clones mutant strain grown permissive and <br />                                               nonpermissive temperature prior harvesting pellets. RNA prepared, standard <br />                                               northern blot procedure preformed. RNA blots probed biotin-labeled <br />                                               riboprobes. Subsequently, blots reprobed probe containing entire <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               yeast rDNA. <br />  <br />                               Construction ts Mutants <br />                                               ts strains constructed described Results; detailed procedures, the <br />                                               Supplemental Experimental Procedures. <br />  <br />                               CIN Assays <br />                                               CTF, genomic BiM, ALF performed previously described (Yuen et al., 2007). <br />  <br />  <br />                               Supplementary Material <br />                                               Refer Web version PubMed Central supplementary material. <br />  <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                                          Page 12 <br />  <br />  <br />                               Acknowledgments <br />                                               We thank A. Carter, M. Tao, M. Kofoed, B. Le, L. Tang technical assistance; M. Kupiec, K. McManus, G. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Simchen, X. Pan helpful discussions; J. Woolford helpful comments earlier version the <br />                                               manuscript. Supported National Institutes Health (NIH) grant P01 CA16519 P.H. J.D.B., <br />                                               Canadian Institute Health Research (CIHR) grant MOP-38096 P.H., NIH roadmap grant U54 RR020839 <br />                                               (to J.D.B.). S.B.-A. supported Human Frontier Science Program (HFSP) long-term fellowship a <br />                                               research grant Killam trust. <br />  <br />  <br />                               References <br />                                               Andersen JS, Lyon CE, Fox AH, Leung AK, Lam YW, Steen H, Mann M, Lamond AI. Directed <br />                                                 proteomic analysis human nucleolus. Curr Biol. 2002; 12:1&#226;&#8364;&#8220;11. [PubMed: 11790298] <br />                                               Aparicio OM. Tackling essential problem functional proteomics Saccharomyces cerevisiae. <br />                                                 Genome Biol. 2003; 4:230. Published online September 24, 2003. 10.1186/gb-2003-4-10-230 <br />                                                 [PubMed: 14519191] <br />                                               Bernstein KA, Gallagher JE, Mitchell BM, Granneman S, Baserga SJ. The small-subunit processome <br />                                                 ribosome assembly intermediate. Eukaryot Cell. 2004; 3:1619&#226;&#8364;&#8220;1626. [PubMed: 15590835] <br />                                               Bianchi A, Shore D. The KEOPS complex: rosetta stone telomere regulation? Cell. 2006; <br />                                                 124:1125&#226;&#8364;&#8220;1128. [PubMed: 16564005] <br />                                               Birrell GW, Giaever G, Chu AM, Davis RW, Brown JM. A genome-wide screen Saccharomyces <br />                                                 cerevisiae genes affecting UV radiation sensitivity. Proc Natl Acad Sci USA. 2001; 98:12608&#226;&#8364;&#8220; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 12613. [PubMed: 11606770] <br />                                               Bleichert F, Granneman S, Osheim YN, Beyer AL, Baserga SJ. The PINc domain protein Utp24, a <br />                                                 putative nuclease, required early cleavage steps 18S rRNA maturation. Proc Natl Acad <br />                                                 Sci USA. 2006; 103:9464&#226;&#8364;&#8220;9469. [PubMed: 16769905] <br />                                               Burckin T, Nagel R, Mandel-Gutfreund Y, Shiue L, Clark TA, Chong JL, Chang TH, Squazzo S, <br />                                                 Hartzog G, Ares M Jr. Exploring functional relationships components gene <br />                                                 expression machinery. Nat Struct Mol Biol. 2005; 12:175&#226;&#8364;&#8220;182. [PubMed: 15702072] <br />                                               Combs DJ, Nagel RJ, Ares M Jr, Stevens SW. Prp43p DEAH-box spliceosome disassembly factor <br />                                                 essential ribosome biogenesis. Mol Cell Biol. 2006; 26:523&#226;&#8364;&#8220;534. [PubMed: 16382144] <br />                                               Dohmen RJ, Varshavsky A. Heat-inducible degron making conditional mutants. Methods <br />                                                 Enzymol. 2005; 399:799&#226;&#8364;&#8220;822. [PubMed: 16338396] <br />                                               Fromont-Racine M, Senger B, Saveanu C, Fasiolo F. Ribosome assembly eukaryotes. Gene. 2003; <br />                                                   313:17&#226;&#8364;&#8220;42. [PubMed: 12957375] <br />                                               Gerring SL, Spencer F, Hieter P. The CHL 1 (CTF 1) gene product Saccharomyces cerevisiae is <br />                                                   important chromosome transmission normal cell cycle progression G2/M. EMBO J. <br />                                                   1990; 9:4347&#226;&#8364;&#8220;4358. [PubMed: 2265610] <br />                                               Giaever G, Chu AM, Ni L, Connelly C, Riles L, Veronneau S, Dow S, Lucau-Danila A, Anderson K, <br />                                                   Andre B, et al. Functional profiling Saccharomyces cerevisiae genome. Nature. 2002; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   418:387&#226;&#8364;&#8220;391. [PubMed: 12140549] <br />                                               Hage AE, Tollervey D. A surfeit factors: ribosome assembly complicated in <br />                                                   eukaryotes bacteria? RNA Biol. 2004; 1:10&#226;&#8364;&#8220;15. [PubMed: 17194932] <br />                                               Hartwell LH, Culotti J, Reid B. Genetic control cell-division cycle yeast. I Detection of <br />                                                   mutants. Proc Natl Acad Sci USA. 1970; 66:352&#226;&#8364;&#8220;359. [PubMed: 5271168] <br />                                               Hazbun TR, Malmstrom L, Anderson S, Graczyk BJ, Fox B, Riffle M, Sundin BA, Aranda JD, <br />                                                   McDonald WH, Chiu CH, et al. Assigning function yeast proteins integration of <br />                                                   technologies. Mol Cell. 2003; 12:1353&#226;&#8364;&#8220;1365. [PubMed: 14690591] <br />                                               Hoyt MA, Stearns T, Botstein D. Chromosome instability mutants Saccharomyces cerevisiae that <br />                                                   defective microtubule-mediated processes. Mol Cell Biol. 1990; 10:223&#226;&#8364;&#8220;234. [PubMed: <br />                                                   2403635] <br />                                               Hu B, Liao C, Millson SH, Mollapour M, Prodromou C, Pearl LH, Piper PW, Panaretou B. Qri2/Nse4, <br />                                                   component essential Smc5/6 DNA repair complex. Mol Microbiol. 2005; 55:1735&#226;&#8364;&#8220;1750. <br />                                                   [PubMed: 15752197] <br />  <br />  <br />  <br />                                                      Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                                   Page 13 <br />  <br />  <br />                                               Huang HK, Bailis JM, Leverson JD, Gomez EB, Forsburg SL, Hunter T. Suppressors Bir1p <br />                                                   (Survivin) identify roles chromosomal passenger protein Pic1p (INCENP) the <br />                                                   replication initiation factor Psf2p chromosome segregation. Mol Cell Biol. 2005; 25:9000&#226;&#8364;&#8220;9015. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   [PubMed: 16199877] <br />                                               Kanemaki M, Sanchez-Diaz A, Gambus A, Labib K. Functional proteomic identification DNA <br />                                                   replication proteins induced proteolysis vivo. Nature. 2003; 423:720&#226;&#8364;&#8220;724. [PubMed: <br />                                                   12768207] <br />                                               Kouprina N, Pashina OB, Nikolaishwili NT, tsouladze AM, Larionov VL. Genetic control of <br />                                                   chromosome stability yeast Saccharomyces cerevisiae. Yeast. 1988; 4:257&#226;&#8364;&#8220;269. [PubMed: <br />                                                   3064490] <br />                                               Krogan NJ, Keogh MC, Datta N, Sawa C, Ryan OW, Ding H, Haw RA, Pootoolal J, Tong A, <br />                                                   Canadien V, et al. A Snf2 family ATPase complex required recruitment histone H2A <br />                                                   variant Htz1. Mol Cell. 2003; 12:1565&#226;&#8364;&#8220;1576. [PubMed: 14690608] <br />                                               Kufel J, Dichtl B, Tollervey D. Yeast Rnt1p required cleavage pre-ribosomal RNA the <br />                                                   3&#226;&#8364;&#178; ETS 5&#226;&#8364;&#178; ETS. RNA. 1999; 5:909&#226;&#8364;&#8220;917. [PubMed: 10411134] <br />                                               Labib K, Tercero JA, Diffley JF. Uninterrupted MCM2&#226;&#8364;&#8220;7 function required DNA replication fork <br />                                                   progression. Science. 2000; 288:1643&#226;&#8364;&#8220;1647. [PubMed: 10834843] <br />                                               Lebaron S, Froment C, Fromont-Racine M, Rain JC, Monsarrat B, Caizergues-Ferrer M, Henry Y. The <br />                                                   splicing ATPase prp43p component multiple preribosomal particles. Mol Cell Biol. 2005; <br />                                                   25:9269&#226;&#8364;&#8220;9282. [PubMed: 16227579] <br />                                               Leeds NB, Small EC, Hiley SL, Hughes TR, Staley JP. The splicing factor Prp43p, DEAH box <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   ATPase, functions ribosome biogenesis. Mol Cell Biol. 2006; 26:513&#226;&#8364;&#8220;522. [PubMed: <br />                                                   16382143] <br />                                               Lindahl L, Archer RH, Zengel JM. A new rRNA processing mutant Saccharomyces cerevisiae. <br />                                                   Nucleic Acids Res. 1992; 20:295&#226;&#8364;&#8220;301. [PubMed: 1741255] <br />                                               Lum PY, Armour CD, Stepaniants SB, Cavet G, Wolf MK, Butler JS, Hinshaw JC, Garnier P, <br />                                                   Prestwich GD, Leonardson A, et al. Discovering modes action therapeutic compounds using <br />                                                   genome-wide screen yeast heterozygotes. Cell. 2004; 116:121&#226;&#8364;&#8220;137. [PubMed: 14718172] <br />                                               Maine GT, Sinha P, Tye BK. Mutants S. cerevisiae defective maintenance of <br />                                                   minichromosomes. Genetics. 1984; 106:365&#226;&#8364;&#8220;385. [PubMed: 6323245] <br />                                               Mayer ML, Gygi SP, Aebersold R, Hieter P. Identification RFC(Ctf18p, Ctf8p, Dcc1p): an <br />                                                   alternative RFC complex required sister chromatid cohesion S. cerevisiae. Mol Cell. 2001; <br />                                                   7:959&#226;&#8364;&#8220;970. [PubMed: 11389843] <br />                                               Meeks-Wagner D, Wood JS, Garvik B, Hartwell LH. Isolation genes affect mitotic <br />                                                   chromosome transmission S. cerevisiae. Cell. 1986; 44:53&#226;&#8364;&#8220;63. [PubMed: 3510080] <br />                                               Michaelis C, Ciosk R, Nasmyth K. Cohesins: chromosomal proteins prevent premature separation <br />                                                   sister chromatids. Cell. 1997; 91:35&#226;&#8364;&#8220;45. [PubMed: 9335333] <br />                                               Mizuta K, Warner JR. Continued functioning secretory pathway essential ribosome <br />                                                   synthesis. Mol Cell Biol. 1994; 14:2493&#226;&#8364;&#8220;2502. [PubMed: 8139552] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Mnaimneh S, Davierwala AP, Haynes J, Moffat J, Peng WT, Zhang W, Yang X, Pootoolal J, Chua G, <br />                                                   Lopez A, et al. Exploration essential gene functions titratable promoter alleles. Cell. 2004; <br />                                                   118:31&#226;&#8364;&#8220;44. [PubMed: 15242642] <br />                                               O&#226;&#8364;&#8482;Connor JP, Peebles CL. PTA1, essential gene Saccharomyces cerevisiae affecting pre-tRNA <br />                                                   processing. Mol Cell Biol. 1992; 12:3843&#226;&#8364;&#8220;3856. [PubMed: 1508188] <br />                                               Ooi SL, Shoemaker DD, Boeke JD. A DNA microarray-based genetic screen nonhomologous end- <br />                                                   joining mutants Saccharomyces cerevisiae. Science. 2001; 294:2552&#226;&#8364;&#8220;2556. [PubMed: <br />                                                   11701889] <br />                                               Ooi SL, Shoemaker DD, Boeke JD. DNA helicase gene interaction network defined using synthetic <br />                                                   lethality analyzed microarray. Nat Genet. 2003; 35:277&#226;&#8364;&#8220;286. [PubMed: 14566339] <br />                                               Ouspenski II, Elledge SJ, Brinkley BR. New yeast genes important chromosome integrity and <br />                                                   segregation identified dosage effects genome stability. Nucleic Acids Res. 1999; 27:3001&#226;&#8364;&#8220; <br />                                                   3008. [PubMed: 10454593] <br />  <br />  <br />  <br />  <br />                                                      Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                                 Page 14 <br />  <br />  <br />                                               Pan X, Yuan DS, Xiang D, Wang X, Sookhai-Mahadeo S, Bader JS, Hieter P, Spencer F, Boeke JD. A <br />                                                   robust toolkit functional profiling yeast genome. Mol Cell. 2004; 16:487&#226;&#8364;&#8220;496. [PubMed: <br />                                                   15525520] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Peng WT, Robinson MD, Mnaimneh S, Krogan NJ, Cagney G, Morris Q, Davierwala AP, Grigull J, <br />                                                   Yang X, Zhang W, et al. A panoramic view yeast noncoding RNA processing. Cell. 2003; <br />                                                   113:919&#226;&#8364;&#8220;933. [PubMed: 12837249] <br />                                               Rempola B, Karkusiewicz I, Piekarska I, Rytka J. Fcf1p Fcf2p novel nucleolar Saccharomyces <br />                                                   cerevisiae proteins involved pre-rRNA processing. Biochem Biophys Res Commun. 2006; <br />                                                   346:546&#226;&#8364;&#8220;554. [PubMed: 16762320] <br />                                               Sacher M, Barrowman J, Schieltz D, Yates JR 3rd, Ferro-Novick S. Identification characterization <br />                                                   new subunits TRAPP. Eur J Cell Biol. 2000; 79:71&#226;&#8364;&#8220;80. [PubMed: 10727015] <br />                                               Samanta MP, Liang S. Predicting protein functions redundancies large-scale protein <br />                                                   interaction networks. Proc Natl Acad Sci USA. 2003; 100:12579&#226;&#8364;&#8220;12583. [PubMed: 14566057] <br />                                               Schuldiner M, Collins SR, Thompson NJ, Denic V, Bhamidipati A, Punna T, Ihmels J, Andrews B, <br />                                                   Boone C, Greenblatt JF, et al. Exploration function organization yeast early <br />                                                   secretory pathway epistatic miniarray profile. Cell. 2005; 123:507&#226;&#8364;&#8220;519. [PubMed: <br />                                                   16269340] <br />                                               Shoemaker DD, Lashkari DA, Morris D, Mittmann M, Davis RW. Quantitative phenotypic analysis of <br />                                                   yeast deletion mutants using highly parallel molecular bar-coding strategy. Nat Genet. 1996; <br />                                                   14:450&#226;&#8364;&#8220;456. [PubMed: 8944025] <br />                                               Sikorski RS, Boeke JD. In vitro mutagenesis plasmid shuffling: cloned gene mutant yeast. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Methods Enzymol. 1991; 194:302&#226;&#8364;&#8220;318. [PubMed: 2005795] <br />                                               Simchen G. Cell cycle mutants. Annu Rev Genet. 1978; 12:161&#226;&#8364;&#8220;191. [PubMed: 106766] <br />                                               Skibbens RV, Corson LB, Koshland D, Hieter P. Ctf7p essential sister chromatid cohesion and <br />                                                   links mitotic chromosome structure DNA replication machinery. Genes Dev. 1999; 13:307&#226;&#8364;&#8220; <br />                                                   319. [PubMed: 9990855] <br />                                               Spencer F, Gerring SL, Connelly C, Hieter P. Mitotic chromosome transmission fidelity mutants in <br />                                                   Saccharomyces cerevisiae. Genetics. 1990; 124:237&#226;&#8364;&#8220;249. [PubMed: 2407610] <br />                                               Takayama Y, Kamimura Y, Okawa M, Muramatsu S, Sugino A, Araki H. GINS, novel multiprotein <br />                                                   complex required chromosomal DNA replication budding yeast. Genes Dev. 2003; 17:1153&#226;&#8364;&#8220; <br />                                                   1165. [PubMed: 12730134] <br />                                               Tong AH, Evangelista M, Parsons AB, Xu H, Bader GD, Page N, Robinson M, Raghibizadeh S, <br />                                                   Hogue CW, Bussey H, et al. Systematic genetic analysis ordered arrays yeast deletion <br />                                                   mutants. Science. 2001; 294:2364&#226;&#8364;&#8220;2368. [PubMed: 11743205] <br />                                               Tsai RT, Fu RH, Yeh FL, Tseng CK, Lin YC, Huang YH, Cheng SC. Spliceosome disassembly <br />                                                   catalyzed Prp43 associated components Ntr1 Ntr2. Genes Dev. 2005; 19:2991&#226;&#8364;&#8220; <br />                                                   3003. [PubMed: 16357217] <br />                                               Venema J, Tollervey D. Ribosome synthesis Saccharomyces cerevisiae. Annu Rev Genet. 1999; <br />                                                   33:261&#226;&#8364;&#8220;311. [PubMed: 10690410] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Voth WP, Jiang YW, Stillman DJ. New &#226;&#8364;&#732;marker swap&#226;&#8364;&#8482; plasmids converting selectable markers on <br />                                                   budding yeast gene disruptions plasmids. Yeast. 2003; 20:985&#226;&#8364;&#8220;993. [PubMed: 12898713] <br />                                               Winzeler EA, Lee B, McCusker JH, Davis RW. Whole genome genetic-typing yeast using high- <br />                                                   density oligonucleotide arrays. Parasitology (Suppl). 1999; 118:S73&#226;&#8364;&#8220;S80. <br />                                               Yuen KW, Warren CD, Chen O, Kwok T, Hieter P, Spencer FA. Systematic genome instability <br />                                                   screens yeast potential relevance cancer. Proc Natl Acad Sci USA. 2007; 104:3925&#226;&#8364;&#8220; <br />                                                   3930. [PubMed: 17360454] <br />                                               Zhao J, Kessler M, Helmling S, O&#226;&#8364;&#8482;Connor JP, Moore C. Pta1, component yeast CF II, required <br />                                                   cleavage poly(A) addition mRNA precursor. Mol Cell Biol. 1999; 19:7733&#226;&#8364;&#8220;7740. <br />                                                   [PubMed: 10523662] <br />  <br />  <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                             Page 15 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Figure 1. Diploid Shuffle&#226;&#8364;&#8221;A Method Generating ts Alleles Essential Genes in <br />                                               Saccharomyces cereviviae <br />                                               (A) Genomic DNA containing YFEG 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; regions used template for <br />                                               PCR mutagenesis. Two black horizontal arrows represent gene-specific primers used. <br />                                               The mutagenized PCR product cloned vector SB221+ Topo-TA (mutations are <br />                                               represented black stars). The Topo-TA cloning site represented red T, A <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               overhang protruding PCR product represented red A. Left beige bar <br />                                               represents 5&#226;&#8364;&#178; half KanMX selectable marker (Kan), right beige bar <br />                                               represents half KanMX selectable marker (MX). The NotI restriction sites <br />                                               indicated diagonal black arrows. <br />                                               (B) The product cloning step library mutagenized YFEG. The library then <br />                                               transformed E. coli digested NotI release linear fragments (following DNA <br />                                               purification). <br />                                               (C) The linearized library transformed corresponding heterozygous diploid strain. <br />                                               Blue red bars flank KanMX <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>knockout</span> represent bar codes. <br />                                               (D) Heterozygous diploid transformants sporulated (following meiosis), MATa Ura+ <br />                                               haploids spores selected haploid-selective medium 25&#194;&#176;C. <br />  <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                         Page 16 <br />  <br />  <br />                                               (E) Selection ts candidates following replica plating incubating 25&#194;&#176;C 37&#194;&#176;C. <br />                                               Black arrows identify potential ts allele. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                          Page 17 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Figure 2. Ribosomal RNA-Processing Screen <br />                                               (A) Scheme rRNA processing. Ribosomal RNA encoded ~9 kb tandem <br />                                               repeat yeast processed 35S precursor multistep pathway outlined here. <br />                                               The cartoon indicates rDNA repeat structure positions mature 25S, 5.8S, 18S <br />                                               rRNAs (black bars DNA precursor forms, gray bars represent mature forms) derived <br />                                               precursor, separately transcribed 5S rRNA. 3&#226;&#8364;&#178; ends indicated by <br />                                               arrowheads. Intermediates pathways rRNA processing identified are <br />                                               shown the panel. Dotted lines indicate effects processing; an <br />                                               alternate pathway formation mature 5.8S 25S rRNAs topmost 27S <br />                                               precursor indicated square bracket. Note multiple species mobilities <br />                                               27S 7S; experiments distinguish these. The internal <br />                                               transcribed spacer (ITS) regions used probes detect precursor forms indicated the <br />                                               inset. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               (B) Isolation ribosomal RNA-processing-defective mutants. RNA preparations were <br />                                               resolved 1% agarose formaldehyde gels blotted nylon membranes probed with <br />                                               ITS2 probe. Duplicate isolates wild-type identified new mutants shown. <br />                                               Mutant YAL043 set 45 unknown mutants studied. <br />  <br />  <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                         Page 18 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Figure 3. Screening Mutants Defective Faithful Chromosome Segregation <br />                                               (A) A CTF assay: (Aa) A supernumerary linear artificial chromosome fragment (CF) <br />                                               containing SUP11 gene serves sensitive indicator chromosome stability its <br />                                               ability suppress ade2-101 ochre mutation (resulting white cells). (Ab) Chromosome <br />                                               loss indicated streaking cells media non-selective artificial chromosome <br />                                               examination appearance white/red sectored colonies. The sectoring phenotype <br />                                               dre2 starts appear 32&#194;&#176;C; higher temperature (34&#194;&#176;C) phenotype more <br />                                               severe. <br />                                               (B) An ALF assay. (Ba) ALF based fact default mating type yeast is <br />                                               MATa. If MAT&#206;&#177; cells ts mutants lose MAT&#206;&#177; locus (due loss of <br />                                               chromosome III), mate MAT&#206;&#177; tester MATa called &#226;&#8364;&#339;a-like fakers.&#226;&#8364;? <br />                                               (Bb) Three patches MAT&#206;&#177; ts isolate control strains (WT Alpha bim1) were <br />                                               replica plated lawn MAT&#206;&#177; tester strain. Growing colonies indication the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               ability mate tester strains. As shown, ALF swi4 appears 32&#194;&#176;C compared <br />                                               controls. <br />                                               (C) A BiM assay. (Ca) Normally, diploid cells heterozygous mating type locus <br />                                               (MATa/MAT&#206;&#177;) mate MATa MAT&#206;&#177; haploids. However, loss the <br />                                               MATa MAT&#206;&#177; mating type locus generates higher frequency mating-competent <br />                                               cells population cells, exhibits bimating phenotype. (Cb) Three <br />                                               independent isolates homozygous diploid ts isolate plus control strains (WT <br />                                               homozygous diploid homozygous diploid chl1 deletion) patched replica <br />                                               plated MATa (top) MAT&#206;&#177; (bottom) testers. The ability mate diploid is <br />                                               assessed number colonies tester strains. The BiM phenotype ynl152w <br />                                               appears 34&#194;&#176;C, compared control strains. The mechanisms lead CF <br />                                               loss, ALF, BiM shown (Ac), (Bc), (Cc), respectively. <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                           Page 19 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Figure 4. A Summary ts Mutant Phenotypes, Showing CIN RNA-Processing Defects <br />                                               Central nodes colored red, green, blue, light purple represent CIN (BiM, ALF, <br />                                               CTF) RNA-processing screens, respectively. Arrows project central nodes <br />                                               point mutants positive specific screen. Small circles colored blue <br />                                               represent ts mutants unknown function defective CIN RNA processing <br />                                               semipermissive temperature. The ORFs used controls CIN assay were <br />                                               colored red. SSU components isolated bicohemically work progress are <br />                                               colored purple. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                         Page 20 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Figure 5. Identification Genes Critical Proper Sister Chromatid Cohesion <br />                                               (A) A secondary screen ts mutants defective sister chromatid cohesion. Cells were <br />                                               arrested G1 G2/M. The number GFP signals scored wild-type, ctf18 <br />                                               mutant, CIN mutants. The data shown represent percentage cells with <br />                                               GFP signals. One cells counted G1-arrested cells 300&#226;&#8364;&#8220;400 <br />                                               cells G2/M-arrested cells. <br />                                               (B C) Sister chromatid cohesion mutants required establishment sister <br />                                               chromatids: samples released &#206;&#177; factor arrest nocodazole-containing <br />                                               media allow arrest G2/M (labeled stop sign). Samples 1 (black arrow), 2 (red <br />                                               arrow) released arrested 25&#194;&#176;C semipermissive temperature, respectively. <br />                                               Sample 3 (blue arrow) completed S phase 25&#194;&#176;C; however, completion S <br />                                               phase, shifted semipermissive temperature, arrest G2/M. Three hours after <br />                                               arrest, percentage cells GFP signals counted (100 cells). The results <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               presented (C). In cases, numbers obtained samples 1 3 similar, <br />                                               indicative potential role sister chromatid establishment. <br />  <br />  <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                                                Page 21 <br />  <br />  <br />  <br />  <br />                                                                                              Table 1 <br />  <br />                            Genes Identified ts Mutants Affect rRNA Processing <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                ORF Name    Gene Name(s)      Description                                                    rRNA Defect       Percent Reduction <br />                                YAL043C     PTA1, FUN39       Subunit holo-CPF, mRNA, snoRNA pre-tRNA processing; binds   18Sa                    22.4 <br />                                                              phospho-CTD RNAPII <br />  <br />                                YDR196C                       Coenzyme A biosynthesis; mitochondrial                         18S                     11.3 <br />  <br />                                YDR339C     FCF1, UTP24       SSU processome component                                       18S                      11 <br />  <br />                                YGR145W     ENP2              Nucleolar                                                      18S                     23.5 <br />  <br />                                YGR277C                       Coenzyme A biosynthesis                                        Not significant <br />  <br />                                YJL069C     UTP18             Possible U3 snoRNP protein                                     25S                      5.9 <br />  <br />                                YKL033W     FMP47             Mitochondrial                                                  18S                     16.9 <br />  <br />                                YKR022C     NTR2              Spliceosome disassembly                                        18S                      9 <br />  <br />                                YLR132C                       Mitochondria nucleus                                       Not significant <br />  <br />                                YOL022C                       Cytoplasmic                                                    18S                      9.1 <br />  <br />                                YOR004W     UTP23             SSU processome component                                       18S                      8.1 <br />  <br />                            a <br />                             Underrepresented mature rRNA indicated; Figure S2 Table S3 data. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                           Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                                                                  NIH-PA Author Manuscript                                                     NIH-PA Author Manuscript                                            NIH-PA Author Manuscript <br />  <br />  <br />                                                                                                                                                                     Table 2 <br />  <br />                                                                 Genes Identified ts Mutants Affect CIN, Quantification CIN Phenotype <br />  <br />                                                                     ORF Name        Gene Name       Description                                                     ALF     BiM     CTF <br />                                                                     YKR038C         KAE1            A component KEOPS protein complex                         0b     0c      3d <br />                                                                                                                                                                                                                                                                                Ben-Aroya et al. <br />  <br />  <br />  <br />  <br />                                                                     YHR020W                         Protein unknown function                                      32     4       3 <br />                                                                     YOR262W                         Cytoplasmic protein unknown function                          0      0       3 <br />                                                                     YDR246W         TRS23           One subunits transport protein particle (TRAPP)    6      2       2 <br />                                                                     YHR122W                         Putative protein unknown function                             23     3       3 <br />                                                                     YGR002C         SWC4            Component Swr1p complex                                   19     4       2 <br />                                                                     YFR042W         KEG1            Protein unknown function                                      34     6       2 <br />                                                                     YNL152W                         Protein unknown function                                      59     6       0 <br />  <br />                                                                     YKR071Ca        DRE2            Protein unknown function                                      0      0       3 <br />  <br />                                                                     YJL072Ca        PSF2            Subunit GINS complex                                      61     0       2 <br />  <br />                                                                     YDR013Wa        PSF1            Subunit GINS complex                                      52     6       3 <br />  <br />                                                                     YDR489Wa        SLD5            Subunit GINS complex                                      0      5       3 <br />  <br />                                                                     YDR288Wa        NSE3            Subunit Mms21-Smc5-Smc6 complex                           52     3       1 <br />  <br />                                                                     YML023Ca        NSE5            Subunit Mms21-Smc5-Smc6 complex                           0      6       0 <br />  <br />                                                                     YDL105Wa        NSE4            Subunit Mms21-Smc5-Smc6 complex                           18     4       1 <br />  <br />                                                                 a <br />                                                  <br /> </body></html>