<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                          RESEARCH ARTICLE <br />  <br />  <br />                                                           Probing Metabolic Network in <br />                                                           Bloodstream-Form Trypanosoma brucei <br />                                                           Using Untargeted Metabolomics Stable <br />                                                           Isotope Labelled Glucose <br />                                                           Darren J. Creek1, Muriel Mazet2, Fiona Achcar3, Jana Anderson4, Dong-Hyun Kim5, <br />                                                           Ruwida Kamour6, Pauline Morand2&#194;&#164;a, Yoann Millerioux2, Marc Biran2, Eduard <br />                                                           J. Kerkhoven7, Achuthanunni Chokkathukalam8, Stefan K. Weidt8, Karl E. V. Burgess8, <br />        a11111                                             Rainer Breitling9, David G. Watson10, Fr&#195;&#169;d&#195;&#169;ric Bringaud2, Michael P. Barrett3,8* <br />                                                           1 Drug Delivery, Disposition Dynamics, Monash Institute Pharmaceutical Sciences, Monash <br />                                                           University, Parkville Campus, Parkville, Victoria, Australia, 2 Centre R&#195;&#169;sonance Magn&#195;&#169;tique des <br />                                                           Syst&#195;&#168;mes Biologiques, Universit&#195;&#169; Bordeaux, CNRS UMR-5536, Bordeaux, France, 3 Wellcome Trust <br />                                                           Centre Molecular Parasitology, Institute Infection, Immunity Inflammation, College Medical, <br />                                                           Veterinary Life Sciences, University Glasgow, Glasgow, United Kingdom, 4 Department Public <br />                                                           Health, Institute Health Wellbeing, College Medical, Veterinary Life Sciences, University of <br />                                                           Glasgow, Glasgow, United Kingdom, 5 Centre Analytical Bioscience, School Pharmacy, University of <br />     OPEN ACCESS <br />                                                           Nottingham, University Park, Nottingham, United Kingdom, 6 Department Medicinal Pharmaceutical <br /> Citation: Creek DJ, Mazet M, Achcar F, Anderson J,        Chemistry, Faculty Pharmacy, University Tripoli, Tripoli, Libya, 7 Systems Synthetic Biology, <br /> Kim D-H, Kamour R, et al. (2015) Probing              Department Chemical Biological Engineering, Chalmers University Technology, G&#195;&#182;teborg, Sweden, <br /> Metabolic Network Bloodstream-Form                     8 Glasgow Polyomics, Wolfson Wohl Cancer Research Centre, Garscube Campus, College Medical <br />                                                           Veterinary &amp; Life Sciences, University Glasgow, Glasgow, United Kingdom, 9 Manchester Institute of <br /> Trypanosoma brucei Using Untargeted Metabolomics <br />                                                           Biotechnology, Faculty Life Sciences, University Manchester, Manchester, United Kingdom, <br /> with Stable Isotope Labelled Glucose. PLoS Pathog <br />                                                           10 Strathclyde Institute Pharmacy Biomedical Sciences, University Strathclyde, Glasgow, United <br /> 11(3): e1004689. doi:10.1371/journal.ppat.1004689         Kingdom <br /> Editor: Scott Landfear, Oregon Health &amp; Science <br />                                                           &#194;&#164;a Current Address: UCLA Medicine Cardiology, MRL Building, Los Angeles, California, United States of <br /> University, UNITED STATES <br />                                                           America <br /> Received: September 10, 2014                              * Michael.Barrett@glasgow.ac.uk <br />  <br /> Accepted: January 19, 2015 <br />  <br /> Published: March 16, 2015 <br />                                                           <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br /> Copyright: &#194;&#169; 2015 Creek et al. This open <br /> access article distributed terms         Metabolomics coupled heavy-atom isotope-labelled glucose used probe <br /> Creative Commons Attribution License, permits       metabolic pathways active cultured bloodstream form trypomastigotes Trypano- <br /> unrestricted use, distribution, reproduction any <br /> medium, provided original author source are <br />                                                           soma brucei, parasite responsible human African trypanosomiasis. Glucose enters <br /> credited.                                                 branches metabolism glycolysis, widely held sole <br /> Data Availability Statement: Relevant mass <br />                                                           route glucose metabolism. Whilst pyruvate major end-product glucose catabo- <br /> spectrometry data available Supporting          lism, transamination product, alanine, produced significant quantities. The oxi- <br /> Information files. Raw data files             dative branch pentose phosphate pathway operative, non-oxidative <br /> available contacting corresponding author at <br />                                                           branch not. Ribose 5-phosphate generated pathway distributes widely into <br /> michael.barrett@glasgow.ac.uk. Only processed <br /> files, available Supporting              nucleotide synthesis branches metabolism. Acetate, derived glucose, is <br /> Information, used producing paper.           associated range acetylated amino acids and, lesser extent, fatty acids; <br /> Funding: This work supported Australian         labelled glycerol glycerophospholipids. Glucose enters inositol <br /> NHMRC fellowship awarded DJC, SysMO                sugar nucleotides serve precursors macromolecule biosynthesis. Al- <br /> SilicoTryp Project (MPB, FA, DHK, RB), ParaMet        Krebs cycle operative, malate, fumarate succinate, primarily labelled in <br /> PhD programme Marie Curie Initial Training <br />                                                           carbons, present, indicating origin phosphoenolpyruvate via <br /> Network (MPB FB). EK SULSA PhD <br />  <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                                              1 / 25 <br />                                                                                                                          Trypanosome Glucose Metabolism <br />  <br />  <br />  <br /> student. JA, DGW, FB RK supported      oxaloacetate. Interestingly, enzyme responsible conversion phosphoenolpyruvate <br /> BBSRC/ANR Systryp Project. MPB          oxaloacetate, phosphoenolpyruvate carboxykinase, shown essential the <br /> Wellcome Trust Centre Molecular Parasitology <br /> (supported grant 085349). FB&#226;&#8364;&#8482;s group also <br />                                                        bloodstream form trypanosomes, demonstrated lethal <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span>phenotype</span> induced by <br /> supported Centre National la Recherche       RNAi-mediated downregulation expression. In addition, glucose derivatives enter py- <br /> Scientifique (CNRS), Universit&#195;&#169; Bordeaux,   rimidine biosynthesis oxaloacetate precursor aspartate orotate. <br /> Laboratoire d&#226;&#8364;&#8482;Excellence (LabEx) ParaFrap ANR-11- <br /> LABX-0024, Agence Nationale la <br /> Recherche (ANR) grant ACETOTRYP the <br /> ANR-BLANC-2010. The funders role study <br /> design, data collection analysis, decision        Author Summary <br /> publish, preparation manuscript. <br />                                                          In work followed distribution carbon derived glucose blood- <br /> Competing Interests: The authors declared <br />                                                          stream form trypanosomes, causative agent African trypanosomiasis, revealing <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> <br /> that competing interests exist <br />                                                          enter diverse range metabolites. The work involved using 13C-labelled glucose fol- <br />                                                          lowing fate labelled carbon LC-MS based metabolomics platform. Be- <br />                                                          yond glycolysis oxidative branch pentose phosphate pathway <span id='am-5' about='rdfs:label' typeof='owl:Thing'>label</span> <br />                                                          entered lipid biosynthesis glycerol 3-phosphate acetate. Glucose de- <br />                                                          rived carbon entered nucleotide synthesis ribose pyrimidine synthesis <br />                                                          oxaloacetate-derived aspartate. Appreciable quantities carboxylic acids suc- <br />                                                          cinate malate identified, labelling patterns indicate TCA <br />                                                          cycle derived. Amino sugars sugar nucleotides labelled inositol used <br />                                                          protein modification inositol phospholipid headgroup production. We con- <br />                                                          firm active essential oxaloacetate production bloodstream form trypanosomes and <br />                                                          phosphoenolpyruvate carboxykinase essential parasites using RNA <br />                                                          interference. The glucose entering metabolites minor compared the <br />                                                          quantity enters pyruvate excreted cell, observation enzymes con- <br />                                                          tributing metabolism glucose glycolysis essential offers potential <br />                                                          new targets chemotherapy trypanosomiasis. <br />  <br />  <br />  <br />                                                        Introduction <br />                                                        Trypanosoma brucei protozoan parasite, sub-species responsible human <br />                                                        African trypanosomiasis (HAT) animal African trypanosomiasis (Nagana) [1]. Current <br />                                                        treatments HAT inadequate, new preventative therapeutic options urgently <br />                                                        required [2]. The parasite transmitted mammalian hosts tsetse fly vector. The <br />                                                        physiological environments mammalian blood tsetse midgut diverge significantly, <br />                                                        bloodstream form insect form parasites reveal adaptive differences biochemistry to <br />                                                        allow optimised <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span>survival</span> environments [3,4]. <br />                                                            T. brucei possess glycosomes, peroxisome-derived organelles contain enzymes required <br />                                                        glycolysis [5]. Within tsetse fly, glucose generally scarce proline key energy <br />                                                        source [6]. Procyclic form (PCF) trypanosomes preferentially use proline source carbon <br />                                                        energy [6,7,8], glucose-rich culture medium preferentially utilize glucose <br />                                                        glycolysis [7,8]. For years widely accepted bloodstream form <br />                                                        (BSF) trypanosomes exhibit greatly reduced metabolic potential, glycosomal glucose uti- <br />                                                        lization glycolysis sole energy source [4,5,9,10]. Under aerobic conditions pyru- <br />                                                        vate considered sole end-product, redox balance maintained a <br />                                                        mitochondrial alternative oxidase shunt acting regenerate dihydroxyacetone phosphate <br />                                                        (DHAP) glycerol 3-phosphate [11]. Under anaerobiosis, equal quantities pyruvate and <br />                                                        glycerol produced, reverse reaction glycerol kinase (GK) generates glycerol <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                                  2 / 25 <br />                                                                                                               Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                           glycerol 3-phosphate [9]. Early studies BSF trypanosomes isolated rodents iden- <br />                                           tified secreted metabolites, including succinate, aspartate alanine [12&#226;&#8364;&#8220;14]. However, <br />                                           fact products present small amounts stumpy form trypano- <br />                                           somes, non-replicative form parasite pre-adapted life tsetse fly [15,16], could <br />                                           contaminate slender BSF preparations, led observations considered limit- <br />                                           ed relevance formulating models simplified pathway glucose catabolism slen- <br />                                           der replicative form T. brucei [9]. <br />                                               Evidence diverse fate glucose BSF trypanosome metabolism has, however, <br />                                           emerged. For example, NMR analysis glucose metabolism [17,18] confirmed glycerol py- <br />                                           ruvate major end products glycolysis, detecting significant amounts ala- <br />                                           nine. The conversion pyruvate alanine essential BSF trypanosomes, it <br />                                           possible <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>knockout</span> alanine aminotransferase (AAT) gene [19], signifi- <br />                                           reduction transcript RNAi possible [19]. An oxidative branch pentose <br />                                           phosphate pathway (PPP) operative [20], 6-phosphogluconate dehydrogenase essential <br />                                           forms [21,22]. Glucose-derived ribose shown incorporated cellular <br />                                           nucleotides BSF trypanosomes [23,24], inositol derived glucose incorporated <br />                                           glycosylphosphatidyl inositol anchors attached abundant variant surface glyco- <br />                                           protein coat cells [25,26]. Recently shown pyruvate dehydrogenase (PDH) <br />                                           essential slender bloodstream forms deprived threonine source acetate <br />                                           [27]; situation PDH needed generate acetate glucose metabolism. <br />                                               The systematic application reverse genetics PCF trypanosomes combined NMR <br />                                           analyses end products metabolism demonstrated importance succinate fermenta- <br />                                           tion pathways glycosome mitochondrion, route independent the <br />                                           classical TCA cycle [28&#226;&#8364;&#8220;30]. A glycosomal succinate shunt converts phosphoenolpyruvate (PEP) <br />                                           produced cytosol oxaloacetate using PEP carboxykinase (PEPCK) following glycosomal <br />                                           reuptake. PEP converted pyruvate, enters mitochondrion feed the <br />                                           PDH complex production acetyl-CoA, converted mitochondrial acetate:suc- <br />                                           cinate CoA-transferase (ASCT) acetyl-CoA thioesterase (ACH) produce acetate [30,31]. <br />                                               The advent metabolomics technology [32&#226;&#8364;&#8220;34] allows unbiased analysis metabolism <br />                                           diverse pathways T. brucei. Stable-isotope labelling permits direct detection active <br />                                           metabolic pathways live cell [35&#226;&#8364;&#8220;39] offers direct route follow dis- <br />                                           tribution atoms precursor substrate metabolic network. We previ- <br />                                           ously applied approach PCF trypanosomes [38], extend methodology to <br />                                           make comprehensive assessment distribution glucose-derived carbon the <br />                                           BSF trypanosome. <br />  <br />                                           Results <br />                                           Extensive glucose utilization anabolic processes bloodstream form <br />                                           T. brucei <br />                                           BSF T. brucei fed 50:50 mixture U-12C U-13C-glucose major carbon source <br />                                           hour 24 hours. Glycolytic intermediates rapidly labelled, expected re- <br />                                           sult high flux glycolytic pathway [10], complete (50%) labelling the <br />                                           major end-product, pyruvate. Significant labelling observed metabolites other <br />                                           pathways, indicating extensive anabolic utilization glucose. In total, 150 metabolites <br />                                           detected glucose-derived carbon labelling, including sugar phosphates, sugar nucleo- <br />                                           tides, lipids secondary metabolites derived glycolytic intermediates end-products <br />                                           (S1 Table). Whilst excreted pyruvate accounts majority catabolised glucose [10], glu- <br />                                           cose-derived pyruvate serves major source alanine acetyl-CoA cell. <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                        3 / 25 <br />                                                                                                                          Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                                Furthermore, metabolism glucose oxidative branch PPP provides ribose <br />                                                5-phosphate, evidenced 50% U-13C-labelled, turn utilized nucleo- <br />                                                tide biosynthesis. Of significant labelling numerous carboxylic acids includ- <br />                                                ing malate, fumarate succinate, extensive labelling aspartate various <br />                                                pyrimidines. In all, results demonstrate extensive utilization glucose anabolic pur- <br />                                                poses bloodstream form T. brucei (Fig 1). <br />  <br />  <br />                                                Revisiting glycolysis bloodstream form T. brucei <br />                                                The glycolytic pathway highly active T. brucei, explains 6-carbon labelling in <br />                                                glucose 6-phosphate (G6P), fructose 6-phosphate (F6P) fructose-1,6-bisphosphate (FBP), <br />                                                3-carbon labelling triose phosphate intermediates lower glycolysis (Fig 2). <br />                                                    Interestingly, additional 3-carbon labelled isotopologues hexose phosphates observed. <br />                                                These isotopologues represent 10% intracellular G6P F6P, 40% FBP. These <br />                                                metabolites identified exact mass retention time using authentic standards, although <br />                                                absolutely rule metabolites mass retention time (i.e. unan- <br />                                                ticipated hexose phosphate isomers) co-eluting metabolites. These findings incon- <br />                                                sistent unidirectional glucose flow Embden-Meyerhof-Parnas glycolytic <br />  <br />  <br />  <br />  <br /> Fig 1. Overview metabolites labelled D-glucose bloodstream form T. brucei. Metabolic network derived detected metabolites and <br /> annotated enzymes T. brucei. Red nodes indicate labelled metabolites 50% U-13C-glucose, darker red indicating higher percentage labelling. <br /> Grey nodes indicate labelling detected. Nodes spots indicate metabolite detected, presence suggested adjacent <br /> metabolites network. <br /> doi:10.1371/journal.ppat.1004689.g001 <br />  <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                                     4 / 25 <br />                                                                                                                          Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />  <br /> Fig 2. Labelling glycolytic intermediates 50% U-13C-glucose BSF T. brucei. (A) Schematic glycolytic pathway BSF T. brucei. Black <br /> font = detected LCMS, grey font = inferred intermediates detected assay. (B) Heatmap relative isotopologue abundances glycolytic <br /> intermediates labelling 24 hours 50% U-13C-glucose HMI11. 0 = carbons unlabelled (e.g. U-12C-glucose), 6 = carbons labelled (e.g. <br /> U-13C-glucose). (C) Isotopologue abundances hexose phosphates labelling 24 hours 50% U-13C-glucose (n = 3, mean &#194;&#177; SD). (D) <br /> Isotopologue abundances glucose 6-phosphate labelling 24 hours 100% U-12C-glucose (white columns), 50% U-13C-glucose (hatched <br /> columns) 100% U-13C-glucose (black columns) (n = 3, mean &#194;&#177; SD). Abbreviations: G6P, glucose 6-phosphate; F6P, fructose 6-phosphate; FBP, fructose- <br /> 1,6-bisphosphate; DHAP, dihydroxyacetone phosphate; GA3P, glyceraldehyde 3-phosphate; 1,3BPG, 1,3-bisphosphoglycerate; 3PG, 3-phosphoglycerate; <br /> 2PG, 2-phosphoglycerate; PEP, phosphoenolpyruvate; G3P, glycerol 3-phosphate. Enzymes: 1, hexokinase; 2, glucose-6-phosphate isomerase; 3, <br /> phosphofructokinase; 4, fructose-1,6-bisphosphatase; 5, aldolase; 6, triosephosphate isomerase; 7, glyceraldehyde 3-phosphate dehydrogenase; 8, <br /> phosphoglycerate kinase; 9, phosphoglycerate mutase; 10, enolase; 11, pyruvate kinase; 12, alanine aminotransferase; 13, glycerol-3-phosphate <br /> dehydrogenase; 14, glycerol kinase; 15, mitochondrial FAD-dependent glycerol-3-phosphate dehydrogenase; 16, methylglyoxal detoxification pathway. <br /> doi:10.1371/journal.ppat.1004689.g002 <br />  <br />  <br />                                                pathway. Labelling studies 100% U-13C-glucose produced little 3-labelled hexose phos- <br />                                                phate (~2% hexose phosphate 3-labelled ~97% glucose labelled <br />                                                inside cells), indicating carbons hexose phosphates originally derived <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                glucose (Fig 2d). Considering lack source absence reactions that <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                                      5 / 25 <br />                                                                                                             Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                           produce 3-labelled hexose-phosphates, 3-labelled hexose-phosphates likely de- <br />                                           rive 3-labelled fructose bisphosphate (possibly reverse phosphofructokinase re- <br />                                           action contribution fructose bisphosphatase ruled out). <br />                                           Gluconeogenesis amino acids carbon sources identified, consistent pre- <br />                                           vious observations [20]. <br />                                               The high percentage 3-carbon labelled FBP using 50:50 mix U-13C-glucose <br />                                           U-12C-glucose counter-intuitive, given strong net forward flux gly- <br />                                           colysis high concentration FBP. However, computational analysis shows it <br />                                           impossible. The aldolase reaction reversible (the net aldolase flux = forward flux&#226;&#8364;&#8221; <br />                                           reverse flux). The observed labelling require the FBP pool created the <br />                                           &#226;&#8364;&#339;reverse&#226;&#8364;? aldolase reaction, working direction FBP production, represent 85% <br />                                           (in HMI medium) 97% (in CMM medium) total FBP pool (see S1 Text detailed ex- <br />                                           planations). These large contributions reverse reaction possible certain condi- <br />                                           tions, spite net flux strongly favouring forward direction. Simulations using a <br />                                           previously published collection models described in [40] under <br />                                           classical assumption glycosomes impermeable, reverse aldolase fluxes vary <br />                                           widely plausible kinetic parameter values, frequently 0 60% the <br />                                           FBP pool coming &#226;&#8364;&#339;reverse&#226;&#8364;? aldolase reaction (i.e. 0 29% potentially <br />                                           3-labelled). However, glycosome semi-permeable metabolites smaller FBP itself <br />                                           (model 3 ref 40) non-selective pore-forming channels glycosomal membrane <br />                                           [41], high reverse aldolase flux, accounting 60&#226;&#8364;&#8220;99% FBP pool favoured (S1 Text). <br />                                           This excellent agreement observed labelling <span id='am-476' about='xsp:pattern' typeof='owl:Thing'>pattern</span>. <br />                                               In addition pyruvate, transamination product alanine produced, probably via <br />                                           AAT, enzyme previously shown essential BSF trypanosomes [19]. Very low levels of <br />                                           labelled lactate detected, carbons labelled. No lactate dehydrogenase <br />                                           identified T. brucei, L-lactate production shown occur <br />                                           unusual methylglyoxal detoxification pathway trypanosomes [42], may <br />                                           correspond metabolite corresponding mass. <br />                                               A number potential pyruvate-derived conjugated metabolites observed, although <br />                                           significance unknown. For example, abundant metabolite 3C 6C label- <br />                                           ling corresponding molecular formula C6H8O6, putatively identified parapyruvate <br />                                           [43]. Pyruvate appeared form adducts basic amino acids produce metabolites <br />                                           mass, labelling predicted retention times consistent carboxyethyl-L-arginine and <br />                                           carboxyethyl-L-ornithine (S1 Table). Another abundant novel 3-carbon labelled metabolite <br />                                           detected molecular formula C6H9NO4S, likely condensation product pyru- <br />                                           vate cysteine, C7H11NO5 correspond formation pyruvate-threonine <br />                                           adduct. These metabolites abundant cells grown HMI11 compared CMM, <br />                                           consistent greater abundance L-cysteine L-threonine richer medium. In <br />                                           absence authentic standards, however, analysis required confirm the <br />                                           structure physiological relevance compounds. <br />  <br />  <br />                                           Pentose phosphate pathway <br />                                           The pentose phosphate pathway intermediates (6-phosphogluconate ribose 5-phosphate) <br />                                           fully labelled, indicating linear oxidative PPP derived glucose 6-phosphate (Fig 3). <br />                                           The complete (5-carbon) labelling ribose purine nucleotides confirms pathway is <br />                                           major source ribose 5-phosphate nucleotide synthesis (Fig 3e). This contrasts with <br />                                           situation PCF T. brucei, possess active non-oxidative branch PPP <br />                                           [20] characteristic 2-, 3- 5-carbon ribose 5-phosphate labelling a <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                     6 / 25 <br />                                                                                                                       Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />  <br />                                           Fig 3. Labelling pentose phosphate pathway purine nucleotides 50% U-13C-glucose in <br />                                           BSF T. brucei. (A) PPP BSF T. brucei. Black font = detected LCMS, grey font = inferred intermediates <br />                                           detected assay. (B) Heatmap relative isotopologue abundances pentose phosphate pathway <br />                                           intermediates labelling 24 hours 50% U-13C-glucose HMI11. (C) Representative mass spectra <br />                                           octulose 8-phosphate T. brucei incubated unlabelled (U-12C) glucose (top) labelled (U-13C) <br />                                           glucose (bottom). The +3, +5 +8 isotopologues predominant labelled sample. (D) <br />                                           Chromatograms octulose 8-phosphate isotopologues T. brucei incubated unlabelled (U-12C) <br />                                           glucose, labelled (U-13C) glucose, unlabelled (U-12C) ribose labelled (U-13C) ribose. (n = 3). (E) Heatmap <br />                                           relative isotopologue abundances purine nucleotides cofactors labelling 24 hours 50% <br />                                           U-13C-glucose HMI11. Abbreviations: G6P, glucose 6-phosphate; F6P, fructose 6-phosphate; 6PG, 6- <br />                                           phosphogluconate; Ru5P, ribulose 5-phosphate; R5P, ribose 5-phosphate; O8P, octulose 8-phosphate; <br />                                           N9P, nonulose 9-phosphate; GA3P, glyceraldehyde 3-phosphate. (F) The T. brucei transaldolase gene was <br />                                           heterologously expressed E. coli protein purified (left hand panel, crude E. coli extract on <br />                                           left purified protein right). Purified transaldolase shown produce octulose 8-phosphate <br />                                           provided ribose 5-phosphate fructose 6-phosphate acceptor donor substrates <br />                                           respectively (right hand panel). Enzymes: 1, Hexokinase; 2, glucose 6-phosphate isomerase; 17, glucose 6- <br />  <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                                    7 / 25 <br />                                                                                                               Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                           phosphate dehydrogenase; 18, 6-phosphogluconate dehydrogenase; 19, ribose 5-phosphate isomerase; 20, <br />                                           ribokinase; 21, transaldolase. <br />                                           doi:10.1371/journal.ppat.1004689.g003 <br />  <br />                                           consequence transaldolase transketolase (TKT) reactions combine fully labelled <br />                                           fully unlabelled precursors [38]. The lack isotopologues confirms lack of <br />                                           TKT BSFs [20,44]. <br />                                               Two high molecular weight sugar phosphates, octulose 8-phosphate (O8P) nonulose 9- <br />                                           phosphate (N9P) detected putatively identified accurate mass predicted reten- <br />                                           tion time (as authentic standards available). The labelling patterns (Fig 3c-d) indicate <br />                                           production transaldolase activity carbon <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> donated aldose donor to <br />                                           pentose phosphate (to produce O8P) hexose phosphate (to produce N9P). Interest- <br />                                           ingly, BSF T. brucei previously shown possess transaldolase activity [20], spite <br />                                           absence TKT. The absence 2C- 6C- labelled O8P BSFs, present <br />                                           PCFs [38], indicates PPP intermediate precursor metabolite. We ex- <br />                                           pressed purified gene encoding T. brucei transaldolase (Tb927.8.5600) Escherichia <br />                                           coli. When enzyme mixed ribose 5-phosphate F6P substrates, mass spec- <br />                                           trometry revealed O8P produced, confirming ability enzyme carry out <br />                                           reaction (Fig 3f). Moreover, cells labelled U-13C-ribose resulted exten- <br />                                           sive labelling ribose 5-phosphate (produced ribokinase), significant 5-carbon label- <br />                                           ling O8P (Fig 3d). <br />                                               The detection metabolites formulas consistent sedoheptulose 7-phosphate <br />                                           (C7H15O10P) erythrose 4-phosphate (C4H9O7P) surprising cell TKT activ- <br />                                           ity. However, 50% U-13C-glucose labelling reveals 3-carbon labelled isotopologue <br />                                           each, confirming products TKT. Further investigation the <br />                                           abundance C7H15O10P peak low samples, retention time of <br />                                           C4H9O7P inconsistent authentic standard erythrose 4-phosphate. More work <br />                                           required elucidate identity source metabolites, confirm <br />                                           non-canonical pathway involved sugar phosphate remodelling exists trypanosomes. <br />  <br />                                           Nucleotide salvage <br />                                           The lack uniform labelling purine bases, nucleosides nucleotides confirms re- <br />                                           quirement exogenous purine bases trypanosomatids lack purine biosynthetic <br />                                           pathways (Fig 3e). However, glucose-derived ribose produces 5-carbon labelled purine nucleo- <br />                                           tides, purine salvage pathways highly active [45]. Transporters efficient uptake of <br />                                           purine nucleosides characterized, guanosine inosine appear major <br />                                           purine sources in vitro culture, presence excess hypoxanthine [46]. These la- <br />                                           belling data confirm nucleoside hydrolases [47] form purine bases prima- <br />                                           ry route nucleotide salvage imported nucleosides glucose-derived ribose <br />                                           5-phosphate added purine phosphoribosyltransferase [48]. The complete la- <br />                                           belling ribose moiety adenosine nucleotides indicates minimal purine salvage ex- <br />                                           ogenous adenosine adenosine kinase vitro growth conditions, spite its <br />                                           high activity cells [49]. <br />                                              Intracellular synthesis various cofactors, including NAD+ S-adenosyl methionine <br />                                           (AdoMet), confirmed incorporation labelled ribose nucleotide subunits. <br />                                           AdoMet important precursor polyamine synthesis, resulting production of <br />                                           5&#226;&#8364;&#8482;methylthioadenosine (MTA). The 5-carbon labelling metabolites confirms that <br />                                           labelling ribose moiety. However, absence labelling methionine precludes <br />                                           activity methionine salvage pathway previously proposed [50]. Methionine uptake from <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                         8 / 25 <br />                                                                                                             Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                           serum [51] apparently fulfils methionine requirements BSF parasites. The absence la- <br />                                           belling &#206;&#177;-ketomethylthiobutyrate (KMTB), intermediate methionine salvage <br />                                           pathway, confirms lack function proposed pathway. KMTB probably <br />                                           arises instead transamination methionine, previously shown excreted <br />                                           parasites amino acid-derived keto acids [46]. <br />  <br />                                           Carboxylic acid derivatives catabolised glucose <br />                                           An active TCA cycle reportedly operative BSF T. brucei, observation &lt;1% <br />                                           detected 2-oxoglutarate contains label glucose corroborates (S1 Table). A small <br />                                           acetyl-coA oxaloacetate were, nevertheless, directed citrate, seen/ob- <br />                                           served 2-, 3- 5-carbon labelling excess natural abundance 13C total <br />                                           approximately 1% cells grown HMI11, 5% CMM-grown cells (S2 Table). The <br />                                           relevance low level citrate production known. PCF T. brucei use classi- <br />                                           cal citrate&#226;&#8364;&#8221;malate shuttle, instead utilizing unique acetate shuttle provide acetyl-CoA to <br />                                           cytosol [31]. <br />                                              Isotope analysis intracellular metabolites cells incubated 50% U-13C-labelled <br />                                           glucose reveals three-carbon labelled isotopologues succinate malate, representing 35% <br />                                           (70% corrected 50% labelled glucose precursor) intracellular malate (Fig 4) and <br />                                           26% succinate (52% corrected). This labelling pattern consistent production of <br />                                           succinate succinate fermentation pathway. The enzyme pathway, PEPCK, is <br />                                           BSF trypanosomes, albeit lower level PCF [52,53]. Although oxaloacetate is <br />                                           detectable analytical platform, observation 3-carbon labelled isotopologue <br />                                           aspartate (a transamination product oxaloacetate) supports production oxaloace- <br />                                           tate PEP. The total labelling aspartate (62% corrected) comparable malate (70% <br />                                           corrected), demonstrating intracellular aspartate primarily synthesized glucose-de- <br />                                           rived oxaloacetate, taken surrounding medium, contains <br />                                           aspartate 100 &#206;&#188;M [46]. BSF trypanosomes failed demonstrate appreciable aspartate trans- <br />                                           port capability [54]. T. brucei encodes separate isoforms malate dehydrogenase <br />                                           (MDH), localized mitochondrion, glycosomes cytosol, respectively [55]. The gly- <br />                                           cosomal isoform reportedly absent BSF, cytosolic isoform present at <br />                                           higher levels BSF procyclics [55]. Additionally small mitochondrial iso- <br />                                           form reportedly present [55&#226;&#8364;&#8220;57]. The succinate fermentation pathway in- <br />                                           volve oxaloacetate produced glycosome PEPCK re-enter cytosol and <br />                                           possibly mitochondrion metabolism. Further work, systematically removing or <br />                                           knocking expression different isoforms, followed metabolic profiling, re- <br />                                           quired deconvolute sources partially reduced products glucose catabolism in <br />                                           BSF trypanosome. <br />  <br />                                           Aspartate use nucleotide biosynthesis <br />                                           Oxaloacetate-derived aspartate key precursor pyrimidine synthesis, converted to <br />                                           orotate dihydroorotate. Dihydroorotate orotate detected isotope enrichment <br />                                           equivalent aspartate, ~35% 3-labelled carbons (70% corrected), confirming novo synthesis <br />                                           pyrimidines way. Uracil labelling slightly lower (44% corrected), suggesting some <br />                                           uptake uracil medium. However, higher levels 2-labelling UMP UDP sug- <br />                                           gest novo synthesized orotate major source pyrimidines conditions <br />                                           (Fig 4). Labelling uridine cytidine nucleotides significantly different between <br />                                           different culture media used study, HMI11 CMM. However, significant dif- <br />                                           ferences observed thymidine nucleotides. dTTP dTMP abundance CMM-grown <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                      9 / 25 <br />                                                                                                                     Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />  <br />                                           Fig 4. Glucose-derived dicarboxylic acid pyrimidine metabolic pathways BSF T. brucei indicate <br />                                           activity PEPCK. (A) Metabolic pathways dicarboxylic acid pyrimidine synthesis BSF T. brucei. <br />                                           Red carbons structures indicate predominant 13C labelling. (B) Heatmap relative isotopologue <br />                                           abundances dicarboxylic acid pyrimidine intermediates labelling 24 hours 50% U-13C- <br />                                           glucose HMI11. (C) Isotopologue abundances dTTP labelling 0 (U), 1 (L1) 24 (L24) hours <br />                                           50% U-13C-glucose HMI11 (H) CMM (M) (n = 2&#226;&#8364;&#8220;3, mean &#194;&#177; SD). Extensive novo dTTP synthesis <br />                                           observed CMM (i.e. absence exogenous thymidine). Abbreviation: PEP, <br />                                           phosphoenolpyruvate; Pyr, pyruvate; OxAc, oxaloacetate; Mal, malate; Fum, fumarate; Succ, succinate; Asp, <br />                                           Aspartate; AdSucc, adenylosuccinate; DHO, dihydroorotate; Oro, orotate; Ura, uracil; Urd, uridine; dUrd, <br />                                           deoxyuridine; Tmd, thymidine. Enzymes: 11, pyruvate kinase; 22, malate dehydrogenase; 23, fumarase; 24, <br />                                           NADH-dependent fumarate reductase); 25, aspartate aminotransferase; 26, adenylosuccinate synthase; 27, <br />                                           adenylosuccinate lyase; 28, aspartate carbamoyltransferase dihydroorotase; 29, dihydroorotate <br />                                           dehydrogenase; 30, orotate phosphoribosyltransferase orotidine 5-phosphate decarboxylase; 31, <br />                                           nucleoside diphosphatase; 32, ribonucleoside-diphosphate reductase; 33, thymidylate kinase; 34, <br />                                           thymidylate synthase; 35, thymidine kinase; 36, uracil phosphoribosyltransferase; 37, uridine phosphorylase; <br />                                           38, nucleoside diphosphate kinase; 39, cytidine triphosphate synthase. <br />                                           doi:10.1371/journal.ppat.1004689.g004 <br />  <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                               10 / 25 <br />                                                                                                             Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                           cells half observed HMI11, contains 20 &#206;&#188;M added thymidine. The iso- <br />                                           tope labelling patterns dTTP dTMP CMM matched UMP py- <br />                                           rimidine nucleotides, indicating complete synthesis novo synthesized pyrimidines by <br />                                           thymidylate synthetase thymidine-poor media [58,59]. dTMP dTTP HMI11-grown <br />                                           cells 10% labelled, confirming preference thymidine salvage action <br />                                           thymidine kinase exogenous thymidine available (Fig 4c). <br />                                               Aspartate important intermediate purine salvage pathway, for- <br />                                           mation adenylosuccinate release fumarate results nitrogen transfer aspartate <br />                                           IMP, producing AMP. Adenylosuccinate detected 3-carbon labelling, con- <br />                                           firms role glucose-derived aspartate purine salvage pathway (Fig 3e). <br />  <br />                                           PEPCK essential BSF T. brucei <br />                                           The discovery glucose enters amino acid nucleotide pathways oxaloacetate indi- <br />                                           cates PEPCK play unexpectedly important role BSF trypanosomes. Indeed, a <br />                                           genome-wide RNAi screen indicated gene essential [60]. To test further, west- <br />                                           ern blot analysis performed anti-PEPCK immune serum, showing PEPCK ex- <br />                                           pressed BSF, albeit lower abundance PCF (Fig 5). Two cell lines <br />                                           (RNAiPEPCK-B3 RNAiPEPCK-D6) generated investigate role PEPCK in <br />                                           BSF T. brucei RNAi down-regulation pepck gene expression. Both cell lines showed a <br />                                           growth arrest days tetracycline induction. A phenotypic reversion 5 days post-induc- <br />                                           tion correlated re-expression PEPCK (Fig 5). <br />  <br />                                           RNAi <br />                                                PEPCK labelled metabolomics <br />                                           In order measure metabolic role PEPCK, production succinate glucose in <br />                                           BSF trypanosomes measured using 1H-NMR, previously described PCF T. brucei <br />                                           [61]. The parental BSF 427 strain incubated 5 hours PBS/NaHCO3 medium contain- <br />                                           ing 4 mM D-glucose carbon source incubation medium analyzed by <br />                                           1 <br />                                            H-NMR spectroscopy. In addition pyruvate, detectable amounts succinate quanti- <br />                                           tatively measured, levels ~2.7% excreted pyruvate (Fig 6a Table 1). The of <br />                                           excreted succinate decreased 4.3 fold RNAiPEPCK-B3 mutant days in- <br />                                           duction compared wild-type cells (47 &#194;&#177;17 versus 201 &#194;&#177;89 nmol/h/mg protein). Succi- <br />                                           nate secretion increased days induction (79 &#194;&#177;18 nmol/h/mg protein) (Fig 6a and <br />                                           Table 1), consequence pepck re-expression (Fig 5). <br />                                               To confirm reduction succinate production RNAiPEPCK-B3 tetracycline-induced <br />                                           cell line (at 2 days), determined incorporation 13C-labelling U-13C-glucose in- <br />                                           tracellular metabolites LC-MS. A reduction 13C3 incorporation malate (24 &#194;&#177; 10%) <br />                                           succinate (29 &#194;&#177; 2%) observed induced RNAiPEPCK-B3 cell line compared the <br />                                           non-induced cells (Fig 6b). Labelling products PEPCK-derived oxaloacetate was <br />                                           significantly reduced, including aspartate (15-fold), uracil (3-fold) UTP (2-fold). In <br />                                           contrast, 13C incorporation metabolites produced upstream PEPCK step (G6P, 3PG <br />                                           pyruvate), pathways derived glycolytic intermediates (alanine, ace- <br />                                           tyl-CoA, ribose 5-phosphate) significantly altered (Fig 6b). <br />  <br />                                           Acetate production BSF T. brucei <br />                                           In addition secretion conversion alanine, pyruvate major source acetyl-CoA <br />                                           BSF trypanosomes. Although intracellular concentration acetyl-CoA the <br />                                           limit detection analytical method, evidence production provided 2-hydro- <br />                                           xyethyl thiamine pyrophosphate (Fig 7a), intermediate pyruvate&#226;&#8364;&#8482;s conversion to <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                    11 / 25 <br />                                                                                                                          Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />  <br /> Fig 5. PEPCK essential bloodstream form T. brucei. (A) Growth curve different clones showing down-regulation pepck RNAi <br /> (RNAiPEPCK-D6 RNAiPEPCK-B3) parental 427 BSF strain (WT) incubated presence (.i) absence (.ni) 10 &#206;&#188;g/ml tetracycline. <br /> Cells maintained exponential growth phase (between 105 2x106 cells/ml) cumulative cell numbers normalized dilution <br /> during cultivation. (B) Western blot analyses parental (WT) mutant cell lines anti-PEPCK anti-enolase sera containing antibodies as <br /> indicated left margin. (C) Western blot demonstrates significantly higher levels PEPCK procyclic form (PF) compared bloodstream form (BF) WT <br /> T. brucei. *Analyses performed 5 x 106 cells sample HSP-60 shown loading control. <br /> doi:10.1371/journal.ppat.1004689.g005 <br />  <br />  <br />  <br />                                                acetyl-CoA PDH. The presence labelled carbons indicates formation directly from <br />                                                glucose-derived pyruvate. Labelling minimal hour, complete labelling ob- <br />                                                served 24 hours, suggesting flux pathway significantly slower than <br />                                                pathways (Fig 7a). It recently shown [27] PDH knockdown <br />                                                conditionally lethal BSF T. brucei. Both threonine, threonine dehydrogenase (TDH) <br />                                                pathway, glucose, PDH, provide acetate route essential <br />                                                threonine absent, redundant loss TDH [27]. Furthermore, acetylated me- <br />                                                tabolites including acetylcarnitine, acetyllysine, acetylornithine acetylglutamine all <br />                                                detected 2 labelled carbons glucose (S1 Table). The unlabelled proportion these <br />                                                metabolites probably relates threonine&#226;&#8364;&#8482;s key role acetate provision [27]. <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                                     12 / 25 <br />                                                                                                                      Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />  <br />                                           Fig 6. Metabolic profile RNAiPEPCK-B3 BSF T. brucei confirms depletion dicarboxylic acids and <br />                                           pyrimidines. (A) Proton (1H) NMR analysis excreted pyruvate succinate glucose. Pyruvate (Pyr) <br />                                           succinate (Suc) excreted bloodstream form 427 cell line (WT), RNAiPEPCK-B3 mutant <br />                                           non-induced (ni), tetracycline-induced 2 days (i 2 days), tetracycline-induced 5 days (i 5 days) was <br />                                           determined 1H-NMR. The cells incubated PBS containing 4 mM glucose 4 hours. Each <br />                                           spectrum corresponds representative experiment set 3. A spectrum <br />                                           ranging 2.2 ppm 2.4 ppm shown. (B) LC-MS analysis intracellular glycolytic intermediates. <br />                                           Relative isotopologue abundances following incubation PBS containing 4 mM U-13C-glucose 4 hours <br />                                           metabolic intermediates RNAiPEPCK-B3 mutant non-induced (NI) tetracycline-induced 2 days <br />                                           (I). Significant decreases labelling observed PEPCK-derived metabolites including 3-carbon <br />                                           labelling dicarboxylic acids aspartate, 2-carbon labelling pyrimidine nucleotides (or 7-carbon <br />                                           labelling ribose-containing metabolites). Labelling significantly decreased glycolytic <br />                                           intermediates glucose-derived metabolites, including alanine, acetate (shown acetyllysine) <br />                                           ribose (shown 5-carbon labelling UTP). <br />                                           doi:10.1371/journal.ppat.1004689.g006 <br />  <br />  <br />  <br />                                           Glucose contributions fatty acid lipid biosynthesis <br />                                           De novo fatty acid synthesis detected BSF T. brucei incorporation labelled car- <br />                                           bon units free fatty acids phospholipids (Fig 7b). However, low level labelling <br />                                           fatty acids suggests novo fatty acid synthesis minor compared salvage <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                                 13 / 25 <br />                                                                                                                    Trypanosome Glucose Metabolism <br />  <br />  <br />  <br /> Table 1. Production succinate, pyruvate, alanine acetate glucose PEPCK RNAi T. brucei measured 1H-NMR. <br />  <br /> Cell line                                   n                                           Excreted molecules glucose <br />  <br />                                                              Pyruvate                       Succinate             Alanine                 Acetate <br />                                                                  nmol / h / mg protein <br /> WTa                                         3                7534&#194;&#177;1535                      201&#194;&#177;89                821&#194;&#177;174                 564&#194;&#177;42 <br /> RNAi <br />       PEPCK-3.nib                           6                7281&#194;&#177;634                       127&#194;&#177;26                754&#194;&#177;49                  328&#194;&#177;51 <br /> RNAi <br />       PEPCK-B3.ic 2days                     7                6913&#194;&#177;760                       47&#194;&#177;17                 731&#194;&#177;8                   361&#194;&#177;18 <br /> RNAi <br />       PEPCK-B3.ic 5days                     3                6880&#194;&#177;797                       79&#194;&#177;18                 722&#194;&#177;64                  281&#194;&#177;44 <br /> a <br />     Strain 427 bloodstream-form T. brucei <br /> b <br />     Non-induced RNAiPEPCK-B3 (without added tetracycline) <br /> c <br />     Induced RNAiPEPCK-B3 (with added tetracycline) <br />  <br /> doi:10.1371/journal.ppat.1004689.t001 <br />  <br />  <br />                                                 mechanisms, high level lysophospholipid uptake represent principal source of <br />                                                 fatty acids cells [62] elongation accounting modelling fatty acids different <br />                                                 chain <span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span> [63]. <br />                                                     The additional 3-carbon labelling observed glycerophospholipids demonstrates lipid <br />                                                 salvage utilizes glycolysis-derived glycerol 3-phosphate synthesis phospholipid head- <br />                                                 groups. Despite requirement salvage host (or culture medium) lipids provi- <br />                                                 sion choline, fatty acids, half glycerol phosphate lipid head-groups <br />                                                 derived glycolysis (ranging 8 100% corrected labelling detected PC (phos- <br />                                                 phatidylcholine) PE (phosphatidylethanolamine) lipids; Fig 7 S1 Table, variability <br />                                                 relating fact relative contributions salvage novo synthesis pathways <br />                                                 uniform lipid species. <br />  <br />                                                 Nucleotide sugars glycoconjugate precursors <br />                                                 Several labelled metabolites detected (S1 Table) indicating glucose enters <br />                                                 amino sugars (N-acetyl-D-glucosamine N-acetyl-D-glucosamine 6-P), sugar nucleotides <br />                                                 (UDP-Glc, UDP-Gal GDP-man UDP-GlcNAC; Fig 7c) low-level labelling other <br />                                                 sugars (tentatively identified myo-inositol, fructose, glucuronate arabinonate, although <br />                                                 accurate identification sugars difficult using accurate mass-based metabolomics, many <br />                                                 isomers theoretically possible carbohydrate metabolites; hence, additional orthog- <br />                                                 onal approaches required confidently identify structures. Nothwithstanding their <br />                                                 abundance clearly low compared glucose fructose primary hexoses. Mannose <br />                                                 absent free form, generation occurring nucleotide conjugated in <br />                                                 phosphorylated form [24]). Targeted studies sugar nucleotide biosynthesis already <br />                                                 demonstrated novo biosynthesis UDP-Glc [64] GDP-Man [24] glucose, and <br />                                                 analysis inositol metabolism demonstrated novo synthesis inositol protein glycosyl- <br />                                                 ation, addition inositol salvage lipid biosynthesis [25,26]. Data current study <br />                                                 demonstrated 10% labelling inositol, confirmed inositol moiety the <br />                                                 lipid headgroup glycerol-phosphoinositol unlabelled (i.e. formed novo synthesis <br />                                                 glucose, consistent targeted analyses conducted investigate origin of <br />                                                 inositol T. brucei [25,26]). <br />  <br />                                                 Discussion <br />                                                 The introduction metabolomic technologies, particularly conjunction stable isotope <br />                                                 tracing has, recent years, transformed ability analyse metabolism biological <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                            14 / 25 <br />                                                                                                                       Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />  <br />                                           Fig 7. Glucose carbon source acetate, lipids sugar nucleotides WT BSF T. brucei. (A) <br />                                           Time-dependent incorporation glucose diverse pathways. Relative isotopologue abundances for <br />                                           unlabelled cells (0) cells incubated 50% U-13C-glucose HMI11 hour (1) 24 hours (24). <br />                                           Labelling reaches equilibrium 1 hour pyruvate, alanine, malate, aspartate ribose 5-phosphate. <br />                                           Minimal labelling observed 2-hydroxyethyl-TPP 1 hour, complete (50%) labelling occurs within <br />                                           24 hours, indicating relatively slow flux acetate production pathway. (n = 3, mean &#194;&#177; SD). (B) <br />                                           Heatmap relative isotopologue abundances lipids labelling 24 hours 50% U-13C-glucose in <br />                                           HMI11 indicating combination salvage mechanisms novo synthesized glycerol phosphate and <br />                                           limited incorporation glucose-derived acetyl-CoA. After manual curation, isotopologues containing more <br />                                           6 labelled carbons confirmed detected lipids. (C) Isotopologue abundances UDP-N- <br />                                           acetyl glucosamine (UDP-GlcNAc) labelling 24 hours 50% U-13C-glucose HMI11 reveal <br />                                           incorporation labelled carbon glucosamine, acetate, ribose uracil. Structures represent the <br />                                           likely isotopologue(s) mass 13C atoms bonds shown red. Additional minor isotopologues <br />                                           omitted clarity, abundant isotopologues containing 3-labelled <br />                                           glucosamine 6-phosphate. <br />                                           doi:10.1371/journal.ppat.1004689.g007 <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                                  15 / 25 <br />                                                                                                              Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                           systems. For example, isotopologue studies Leishmania [65], Plasmodium [36,37] and <br />                                           Toxoplasma [35] identified novel pathways resolved long-standing questions me- <br />                                           tabolism. The African trypanosome, Trypanosoma brucei, causative agent human Afri- <br />                                           trypanosomiasis, neglected tropical disease sub-Saharan Africa, new drugs <br />                                           urgently needed. The slender bloodstream form parasites generally con- <br />                                           sidered highly streamlined glucose catabolic pathway [3&#226;&#8364;&#8220;5,9,10]. The localization of <br />                                           seven enzymes glycolytic pathway membrane-bounded peroxisome-like or- <br />                                           ganelles known glycosomes [5] proposed instrumental regulation of <br />                                           flux glycolysis providing environment ATP/ADP NAD/NADH <br />                                           balanced [9,10]. This arrangement enabled production well-defined dynam- <br />                                           ic mathematical model glycolysis [9,10] models able predict im- <br />                                           portant phenomena, contribution different enzymes overall control the <br />                                           pathway. Recent iterations included addition parameters explicitly ac- <br />                                           count uncertainty related [66] activity pentose phosphate <br />                                           pathway [22] presence permeability pores allow free diffusion glyco- <br />                                           lytic intermediates cytosol glycosome [40]. <br />                                               Here taken systematic route analyse extent glucose metabolism in <br />                                           long slender BSF trypanosomes. <br />                                               Extensive pyruvate production observed, consistent existing models glycolytic <br />                                           flux energy production. However, 3-carbon labelled isotopologues observed glycolyt- <br />                                           ic hexose phosphates reveals previously unreported complexity pathway. The lack of <br />                                           transketolase [20,42] indicates canonical non-oxidative pentose phosphate cycle miss- <br />                                           ing cells source 3-labelled hexose phosphates. The role of <br />                                           transaldolase BSFs uncertain, transaldolase-mediated production 3-labelled fruc- <br />                                           tose 6-phosphate (from F6P GA3P) hypothesized, appears a <br />                                           redundant reaction. Alternatively, aldolase [67], acting combine U-13C U-12C <br />                                           DHAP U-13C U-12C GA3P produce relative proportions 0-, 3- 6- <br />                                           labelled fructose bisphosphate identified here. The production 3-labelled F6P (and <br />                                           G6P) FBP require fructose-1,6-bisphosphatase activity, transcriptomic <br />                                           [68] proteomic [69] data indicate enzyme present BSF T. brucei. A family of <br />                                           models, simulating flux glycolysis whilst taking account uncertainties about <br />                                           distribution glycosome cytosol [40], reveals metabolites <br />                                           smaller FBP able exchange compartments FBP not, signifi- <br />                                           proportion total FBP cell arise condensation GA3P <br />                                           DHAP. The data, therefore, supports proposed role pores creating semi-permeable <br />                                           glycosome [41]. Further work required fully investigate intermediate fluxes glycolyt- <br />                                           ic metabolism determine temporal spatial aspects metabolism rise to <br />                                           unexpected observations. <br />                                               At point rule definitively production compound mass <br />                                           retention time FBP artefact experimental system. The true of <br />                                           exotic, unexpected species identified study. However, non-enzymatic aldol con- <br />                                           densation triose phosphates highly unlikely, given non-enzymatic condensations and <br />                                           additions generally require extreme pH conditions (&lt;1 &gt; 9). To confirm this, DHAP and <br />                                           GA3P mixed CMM medium followed extraction conditions used extract <br />                                           cellular lysates storage seven days -80&#194;&#176;C, FBP potential conden- <br />                                           sation addition products observed LC-MS. We consider likely that <br />                                           metabolites bona fide products trypanosome metabo- <br />                                           lism, incorporation carbon-13 label confirms endogenous production these <br />                                           metabolites (compared unlabelled metabolites primarily acquired serum the <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                     16 / 25 <br />                                                                                                             Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                           growth media), physiological function, relevance remains unknown. <br />                                           Metabolites produced enzymatically chemically cells inevitable by-prod- <br />                                           ucts metabolism, methylglyoxal example discussed [42]. <br />                                               Apart identification numerous unexpected metabolites pathways operat- <br />                                           ing BSF trypanosomes, experiments rule operation pathways pre- <br />                                           viously assumed functional. For example, rapid deamination aromatic amino acids <br />                                           believed contribute methionine cycle involved provision decarboxylated <br />                                           S-adenosylmethionine used polyamine synthesis [50]. Here, however, demonstrate <br />                                           carbons glucose enter methionine, provided exclusively through <br />                                           cellular uptake instead [51]. This means pathways methylthioadenosine detoxifi- <br />                                           cation remain elucidated BSF trypanosome. <br />                                               While majority glucose flows secreted pyruvate aerobic culture conditions, <br />                                           glucose metabolism pervasive BSF parasites. In addition pyruvate and <br />                                           glycerol, various metabolites secreted, including alanine, transaminated product <br />                                           pyruvate. This reaction appears essential alanine transaminase gene not <br />                                           knocked T. brucei genome [19], knockdown RNAi possible. Suc- <br />                                           cinate secreted. The succinate produced BSF trypanosomes primarily labelled in <br />                                           carbons, indicates derivation glycolytic phosphoenolpyruvate, than <br />                                           canonical TCA cycle, produce primarily 2-carbon 4-carbon labelled <br />                                           isotopologues. Analysis phosphoproteome BSFs identified glycosomal NADH-depen- <br />                                           dent fumarate dehydrogenase [70]. However, possible distinguish glycoso- <br />                                           mal, cytosolic potentially mitochondrial derivation metabolites, systematic <br />                                           metabolite profiling following <span id='am-242' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-243' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span>gene knockout</span> knockdown various enzymes neces- <br />                                           sary deconvolute pathways. <br />                                               Although suggested glucose makes negligible contribution anabolic pro- <br />                                           cesses BSF trypanosomes [10], numerous isolated studies revealed glucose serves <br />                                           precursor processes including sugar nucleotide glycoconjugate synthesis [23], inosi- <br />                                           tol production [25,26] recently acetate production [27]. Glucose enters pyrimidine <br />                                           biosynthesis oxaloacetate lipid metabolism glycerol 3-phosphate acetate <br />                                           production. RNAi knockdown PEPCK confirmed activity essential BSF try- <br />                                           panosomes. The enzyme essential PCFs, contribution major in <br />                                           parasitic insect form [71]. We recently demonstrated loss bound phosphate from <br />                                           glycolytic pathway glycosome PPP requires mechanism retain bound <br />                                           phosphate balance [22]. PEPCK offers obvious means restore glycosomal ATP, this <br />                                           link PPP explain unexpectedly essential nature enzyme the <br />                                           BSF trypanosome. <br />                                               Although untargeted mass spectrometry platform use intrinsically unable to <br />                                           quantify total flux glucose carbon various pathways discussed here, clearly <br />                                           case overwhelming majority glucose catabolised pyruvate (Table 1). The <br />                                           total glucose entering pathways order just percent <br />                                           total consumed, agreement measurements Haanstra et al [10]. Although <br />                                           small minority total glucose consumption, however, glucose major carbon <br />                                           source production of extra-glycolytic metabolites, flux clearly <br />                                           important given fact enzymes, including PEPCK shown here, turn es- <br />                                           sential metabolism parasites. <br />                                               The identification novel essential enzymes BSF trypanosome important <br />                                           implications chemotherapy. Glycolysis long considered attractive target given the <br />                                           absolute dependency glucose energy carbon provision cells [72]; results <br />                                           additional pathways periphery glycolysis equally promising. <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                     17 / 25 <br />                                                                                                              Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                           Materials Methods <br />                                           Parasite culture <br />                                           T. brucei brucei bloodstream form (s427) cultured vitro HMI11 [73] CMM [44] <br />                                           supplemented 10% fetal calf serum (FCS) Gold (PAA, Piscataway, NJ) 37&#194;&#176;C, 5% CO2. <br />                                           Initial 5 ml cultures 25 ml vented flasks (Corning) grown maximum density of <br />                                           4 &#195;&#8212; 106 cells/ml subcultured 1-in-100 1-in-1,000 dilution 2 3 days, respec- <br />                                           tively. A hemocytometer (Neubauer) used cell counts. <br />  <br />  <br />  <br />                                           Labelling experiments sample preparation <br />                                           Two days prior extraction, 2 &#195;&#8212; 104 cells/ml seeded 30 ml HMI11 CMM. <br />                                           Growth media replaced media containing 50% 13C-labelled glucose (12.5 mM <br />                                           12C-glucose + 12.5 mM 13C-glucose HMI11, 5 mM 12C-glucose 5 mM of <br />                                           13 <br />                                              C-glucose CMM culture) 24 h 1 h prior extraction. All samples extracted at <br />                                           time atequivalent cell densities approximately 2 &#195;&#8212;106 cells/ml. To yield 5 &#195;&#8212;107 cells, <br />                                           appropriate volumes cell culture harvested quenched EtOH/dry ice bath pre- <br />                                           viously described [74]. Cells centrifuged 1250 &#195;&#8212;g 10 min spent media removed; <br />                                           cells washed pre-cooled phosphate buffered saline (PBS) 4&#194;&#176;C. After centrifugation <br />                                           1250 &#195;&#8212;g 5 min, supernatant removed metabolites extracted adding 100 &#206;&#188;l <br />                                           (ice cold) chloroform/methanol/water (1:3:1) mixed vigorously 1 h 4&#194;&#176;C. The ex- <br />                                           traction mixtures centrifuged 13,000 &#195;&#8212;g 10 min supernatants collected and <br />                                           stored -80&#194;&#176;C prior analysis. In order assess instrument performance, pooled quality <br />                                           control (QC) sample prepared mixing equal volume samples. Independent <br />                                           biological replicates prepared different days. <br />  <br />  <br />  <br />                                           LC-MS methodologies <br />                                           Hydrophilic interaction liquid chromatography (HILIC) carried Dionex UltiMate <br />                                           3000 RSLC (Thermo Fisher Scientific, Hemel Hempstead, UK) using ZIC-pHILIC <br />                                           column (150 mm &#195;&#8212; 4.6 mm, 5 &#206;&#188;m column, Merck Sequant) previously described [74]. Brief- <br />                                           ly, column maintained 30&#194;&#176;C samples eluted linear gradient (20 mM <br />                                           ammonium carbonate water, A acetonitrile, B) 46 min flow rate 0.3 ml/min <br />                                           follows: 80% B 20% B 30 min, 5% B 32 min held 39 min washing, to <br />                                           80% B 40 min held 46 min re-equilibration. The injection volume 10 &#206;&#188;l and <br />                                           samples maintained 4&#194;&#176;C prior injection. For MS analysis, Thermo Orbitrap <br />                                           Exactive (Thermo Fisher Scientific) operated polarity switching mode MS set- <br />                                           tings follows: resolution 50,000, AGC 106, m/z range 70&#226;&#8364;&#8220;1400, sheath gas 40, auxiliary <br />                                           gas 5, sweep gas 1, probe temperature 150&#194;&#176;C, capillary temperature 275&#194;&#176;C. For positive <br />                                           mode ionisation: source voltage +4.5 kV, capillary voltage +50 V, tube voltage +70 kV, skim- <br />                                           mer voltage +20 V. For negative mode ionisation: source voltage-3.5 kV, capillary voltage-50 <br />                                           V, tube voltage-70 V, skimmer voltage-20 V. Mass calibration performed polarity <br />                                           immediately prior analysis batch. The calibration mass range extended cover <br />                                           small metabolites inclusion low-mass contaminants standard Thermo calmix <br />                                           masses (below m/z 1400), C2H6NO2 positive ion electrospray ionisation (PIESI) mode <br />                                           (m/z 76.0393) C3H5O3 negative ion electospray ionisation (NIESI) mode (m/z <br />                                           89.0244). To enhance calibration stability, lock-mass correction applied ana- <br />                                           lytical run using ubiquitous low-mass contaminants. <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                      18 / 25 <br />                                                                                                              Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                           Data processing analysis <br />                                           Raw LC-MS data processed XCMS untargeted peak detection [75], and <br />                                           mzMatch.R [76] employed peak matching annotation related peaks. Tentative <br />                                           metabolite identification carried IDEOM using default parameters [77]. Metab- <br />                                           olite identification performed matching accurate masses retention times authen- <br />                                           tic standards (MSI confidence level 1, i.e. annotated compound matched standard with <br />                                           orthogonal approaches), standards available, predicted retention times <br />                                           calculated previously validated model [78] (MSI confidence level 2, i.e. putatively <br />                                           annotated compound based exact mass determination). Published literature path- <br />                                           way/genome databases considered improve annotation cases isomers could <br />                                           differentiated based accurate mass retention time. Metadata support iden- <br />                                           tification metabolite available IDEOM file study (S3 Table). Metabo- <br />                                           lites included manuscript manually annotated using authentic standards where <br />                                           available. However, note metabolite names given Ideom file generat- <br />                                           ed automatically software provides best match database entries given mass <br />                                           formula. In absence additional information considered putatively- <br />                                           annotated hits; confidence score column adjacent hit serves guide this. <br />                                           Clearly scope study provide authenticated annotations hun- <br />                                           dreds detected compounds, datasets included spirit open <br />                                           access data. <br />                                               The mzmatch-ISO software [39] used extract isotopologue abundances all <br />                                           identified putative annotated metabolites. Raw data available associated IDEOM <br />                                           files (S3 Table) chromatograms separate pdf files (S1&#226;&#8364;&#8220;S2 Figs). Labelled metabolites <br />                                           given S4 Table S5 Table. Data analysis interpretation based 24 hour <br />                                           time-point, approximate steady state, unless stated. <br />  <br />                                           NMR experiments <br />                                           T. brucei bloodstream form cells (2.5 x107) collected centrifugation 1,400 g 10 <br />                                           min, washed PBS containing 4 mM glucose incubated 5 h 37&#194;&#176;C 2.5 ml in- <br />                                           cubation buffer (PBS supplemented 5 g/l NaHCO3, pH 7.4) containing 4 mM glucose. <br />                                           The integrity cells incubation checked microscopic observation. The <br />                                           supernatant collected 50 &#206;&#188;l malate solution D2O (20 mM) added internal <br />                                           reference. 1H-NMR spectra performed 125.77 MHz Bruker DPX500 spectrometer <br />                                           equipped 5 mm broadband probe head. Measurements recorded 25&#194;&#176;C an <br />                                           ERETIC method. This method provides electronically synthesized reference signal [79]. Ac- <br />                                           quisition conditions follows: 90&#194;&#176; flip angle, 5,000 Hz spectral width, 32 K memory size, <br />                                           9.3 sec total recycle time. Measurements performed 256 scans total time <br />                                           close 40 min. Before experiment, phase ERETIC peak precisely adjusted. Reso- <br />                                           nances obtained spectra integrated results expressed relative ERETIC <br />                                           peak integration. <br />  <br />                                           Inhibition pepck gene expression RNAi <br />                                           Inhibition pepck gene expression RNAi 427 strain performed expression of <br />                                           stem-loop &#226;&#8364;&#339;sense/anti-sense&#226;&#8364;? RNA molecules targeted sequences introduced the <br />                                           pHD1334 expression vector. First, pLew-PEPCK-SAS plasmid constructed target a <br />                                           481 bp fragment pepck gene (from position 22 bp 503 bp). Briefly, PCR-amplified <br />                                           569 bp fragment, containing antisense pepck sequence restriction sites added the <br />                                           primers inserted HindIII BamHI restriction sites pLew100 plasmid. <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                     19 / 25 <br />                                                                                                             Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                           Then PCR-amplified fragment containing sense pepck sequence (505 bp) inserted <br />                                           upstream anti-sense sequence, using HindIII XhoI restriction sites (XhoI intro- <br />                                           duced 3&#226;&#8364;&#8482;-extremity antisense PCR fragment). The resulting plasmid (pLew- <br />                                           PEPCK-SAS) contains sense antisense version targeted gene fragment, separated <br />                                           52 bp fragment, control PARP promoter linked prokaryotic tetracy- <br />                                           cline (Tet) operator. Then PEPCK-SAS HindIII-BamHI cassette extracted pLew- <br />                                           PEPCK-SAS plasmid inserted HindIII-BamHI digested pHD1334 vector. The RNAi- <br />                                           PEPCK cell lines produced transfecting 427 &#226;&#8364;&#339;double marker&#226;&#8364;? (Hyg-Neo) mono- <br />                                           morphic cell line pHD-PEPCK-SAS plasmid. Transfected cells selected in <br />                                           IMDM medium containing, hygromycin (5 &#206;&#188;g/ml), neomycin (2.5 &#206;&#188;g/ml) phleomycin <br />                                           (2.5 &#206;&#188;g/ml). Induction double-stranded RNA expression performed addition 10 <br />                                           &#206;&#188;g/ml tetracycline. <br />  <br />  <br />                                           Cloning expression T. brucei transaldolase gene <br />                                           A gene annotated transaldolase T. brucei genome database Tb427.08.5600, am- <br />                                           plified following primers 5&#226;&#8364;&#8482;-GCCATATGAATCAACTGGAGAGCCT-3&#226;&#8364;&#8482; 5&#226;&#8364;&#8482;- <br />                                           GCCTCGAGTTACAAAAGTGTAGCGTGA-3&#226;&#8364;&#8482;. The PCR product cloned the <br />                                           vector pGEM-T easy transferred pET28a(+) expression vector using intro- <br />                                           duced Nde1 Xho1 restriction sites. The resultant plasmid sequenced confirm fideli- <br />                                           ty transformed E. coli strain BL21 (DE3) expression 37&#194;&#176;C 4 hours after <br />                                           induction IPTG. Expressed protein observed SDS protein gel electrophoresis and <br />                                           purified using Ni2+ chelate chromatography (Fig 2f). The ability transaldolase produce <br />                                           octulose 8-phosphate determined mixing ribose 5-phosphate (5 mM) fructose <br />                                           6-phosphate (5 mM) Tris-HCl (pH 7.2) adding transaldolase (1.2 mg/mL) incu- <br />                                           bating 20&#194;&#176;C 2 hrs. Alternatively reaction set enzyme. The presence <br />                                           octulose 8-phosphate determined direct infusion post-reaction mix the <br />                                           Exactive mass spectrometer (Fig 2f). <br />  <br />  <br />                                           Western blot analyses <br />                                           Total protein extracts wild-type mutant BSF T. brucei (5 x106 cells) size-fraction- <br />                                           ated SDS-PAGE (10%) immunoblotted Immobilon-P filters (Millipore) [80]. Immu- <br />                                           nodetection performed described [81,82] using primary antibodies, rat anti-T. brucei <br />                                           PEPCK (diluted 1:1000; gift T. Seebeck, Bern, Switzerland), rabbit anti- T. brucei enolase <br />                                           (diluted 1:1,000, gift P. Michels, Edinburgh, UK) mouse anti-HSP60 (diluted 1:10 <br />                                           000) [83], secondary antibodies, anti-rat, anti-rabbit anti-mouse IgG conjugated to <br />                                           horseradish peroxidase (BioRad, 1:5,000 dilution). Revelation performed using Super- <br />                                           Signal West Pico Chemiluminescent Substrate described manufacturer (Thermo Fish- <br />                                           er Scientific). Alternatively, quantitative analyses, revelation performed using the <br />                                           Luminata Crescendo Western HRP Substrate (Millipore). Images acquired analyzed <br />                                           KODAK Image Station 4000 MM quantitative analyses performed the <br />                                           KODAK MI application. <br />  <br />  <br />                                           Supporting Information <br />                                           S1 Text. PDF outlining use computational models glucose metabolism predict rela- <br />                                           tive contributions fructose 1,6 bis-phosphate pool. <br />                                           (PDF) <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                     20 / 25 <br />                                                                                                                       Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                           S1 Table. csv file relative isotopologue abundances BSF T. brucei grown 50%- <br />                                           U-13C-glucose HMI11. <br />                                           (CSV) <br />                                           S2 Table. csv file relative isotopologue abundances BSF T. brucei grown 50%- <br />                                           U-13C-glucose CMM. <br />                                           (CSV) <br />                                           S3 Table. IDEOM macro-enabled Excel file metabolomics data BSF T. brucei <br />                                           grown 50%-U-13C-glucose HMI11 CMM. <br />                                           (ZIP) <br />                                           S4 Table. csv file relative isotopologue abundances BSF T. brucei grown 50%- <br />                                           U-13C-glucose CMM HMI11. <br />                                           (ZIP) <br />                                           S5 Table. Table showing composition HMI11 CMM medium. <br />                                           (DOCX) <br />                                           S1 Fig. PDF chromatograms positive mode ESI isotopologues detected for <br />                                           identified putatively annotated metabolites BSF T. brucei grown 50%-U- <br />                                           13C-glucose. <br />                                           (PDF) <br />                                           S2 Fig. PDF chromatograms negative mode ESI isotopologues detected for <br />                                           identified putatively annotated metabolites BSF T. brucei grown 50%-U- <br />                                           13C-glucose. <br />                                           (PDF) <br />  <br />  <br />                                           Acknowledgments <br />                                           We grateful Amy M. Cattanach editorial assistance, Paul A. M. Michels and <br />                                           Thomas Seebeck providing anti-enolase anti-PEPCK immune <br />                                           sera, respectively. <br />  <br />                                           Author Contributions <br />                                           Conceived designed experiments: MPB DJC FA RB FB. Performed experiments: <br />                                           DJC MM JA FA EJK PM YM DHK RK KEVB SKW DGW MB. Analyzed data: MPB FB <br />                                           AC DJC FA RB. Wrote paper: DJC MPB FB RB FA. <br />  <br />                                           References <br />                                            1.   Brun R, Blum J, Chappuis F, Burri C (2010) Human African trypanosomiasis. Lancet 375:148&#226;&#8364;&#8220;159. doi: <br />                                                 10.1016/S0140-6736(09)60829-1 PMID: 19833383 <br />                                            2.   Barrett MP (2010) Potential new drugs human African trypanosomiasis: progress last. Curr <br />                                                 Opin Infect Dis 23:603&#226;&#8364;&#8220;608. doi: 10.1097/QCO.0b013e32833f9fd0 PMID: 20844428 <br />                                            3.   Besteiro S, Barrett MP, Rivi&#195;&#168;re L, Bringaud F (2005) Energy generation insect stages Trypano- <br />                                                 soma brucei: metabolism flux. Trends Parasitol 21:185&#226;&#8364;&#8220;191. PMID: 15780841 <br />                                            4.   Bringaud F, Rivi&#195;&#168;re L, Coustou V (2006) Energy metabolism trypanosomatids: adaptation avail- <br />                                                 able carbon sources. Mol Biochem Parasitol 149:1&#226;&#8364;&#8220;9. PMID: 16682088 <br />                                            5.   Opperdoes FR, Borst P (1977) Localization glycolytic enzymes microbody-like organelle in <br />                                                 Trypanosoma brucei: glycosome. FEBS Lett 80:360&#226;&#8364;&#8220;364. PMID: 142663 <br />                                            6.   Bursell E (1963) Aspects metabolism amino acids tsetse fly, Glossina (Diptera) J Insect <br />                                                 Physiol 9:439&#226;&#8364;&#8220;452. <br />  <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                                  21 / 25 <br />                                                                                                                        Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                            7.   Lamour N, Rivi&#195;&#168;re L, Coustou V, Coombs GH, Barrett MP, et al. (2005) Proline metabolism procyclic <br />                                                 Trypanosoma brucei down-regulated presence glucose. J Biol Chem 280:11902&#226;&#8364;&#8220;11910. <br />                                                 PMID: 15665328 <br />                                            8.   Coustou V, Biran M, Breton M, Guegan F, Rivi&#195;&#168;re L, et al. (2008) Glucose-induced remodeling inter- <br />                                                 mediary energy metabolism procyclic Trypanosoma brucei. J Biol Chem 283:16342&#226;&#8364;&#8220;16354. doi: <br />                                                 10.1074/jbc.M709592200 PMID: 18430732 <br />                                            9.   Bakker BM, Michels PA, Opperdoes FR, Westerhoff HV (1997) Glycolysis bloodstream form Trypa- <br />                                                 nosoma brucei understood terms kinetics glycolytic enzymes. J Biol Chem <br />                                                 272:3207&#226;&#8364;&#8220;3215. PMID: 9013556 <br />                                           10.   Haanstra JR, van Tuijl A, van Dam J, van Winden W, Tielens AG, et al. (2012) Proliferating blood- <br />                                                 stream-form Trypanosoma brucei use negligible consumed glucose anabolic processes. <br />                                                 Int J Parasitol 42:667&#226;&#8364;&#8220;673. doi: 10.1016/j.ijpara.2012.04.009 PMID: 22580731 <br />                                           11.   Opperdoes FR, Borst P, Bakker S, Leene W (1977) Localization glycerol-3-phosphate oxidase the <br />                                                 mitochondrion particulate NAD+-linked glycerol-3-phosphate dehydrogenase microbodies of <br />                                                 bloodstream form Trypanosoma brucei. Eur J Biochem 76:29&#226;&#8364;&#8220;39. PMID: 142010 <br />                                           12.   Ryley JF (1962) Studies metabolism protozoa. 9. Comparative metabolism blood- <br />                                                 stream culture forms Trypanosoma rhodesiense. Biochem J 85:211&#226;&#8364;&#8220;223. PMID: 13983265 <br />                                           13.   Grant PT, Fulton JD (1957) The catabolism glucose strains Trypanosoma rhodesiense. Bio- <br />                                                 chem J 66:242&#226;&#8364;&#8220;250. PMID: 13445679 <br />                                           14.   Fulton JD, Stevens TS (1945) The glucose metabolism vitro Trypanosoma rhodesiense. Biochem <br />                                                 J 39:317&#226;&#8364;&#8220;320. PMID: 16747912 <br />                                           15.   Rico E, Rojas F, Mony BM, Szoor B, Macgregor P, et al. (2013) Bloodstream form pre-adaptation the <br />                                                 tsetse fly Trypanosoma brucei. Front Cell Infect Microbiol 3:78. doi: 10.3389/fcimb.2013.00078 <br />                                                 PMID: 24294594 <br />                                           16.   Brown RC, Evans DA, Vickerman K (1972) Developmental changes ultrastructure physiology of <br />                                                 Trypanosoma brucei. Trans R Soc Trop Med Hyg 66:336&#226;&#8364;&#8220;337. PMID: 5048840 <br />                                           17.   Mackenzie NE, Hall JE, Seed JR, Scott AI (1982) Carbon-13 nuclear-magnetic-resonance studies of <br />                                                 glucose catabolism Trypanosoma brucei gambiense. Eur J Biochem 121:657&#226;&#8364;&#8220;661. PMID: 7056262 <br />                                           18.   Hall JE, Mackenzie NE, Mansfield JM, McCloskey DE, Scott AI (1988) 13C-NMR analysis alanine <br />                                                 metabolism isolated perfused livers C3HeB/FeJ mice infected African trypanosomes. <br />                                                 Comp Biochem Physiol B 89:679&#226;&#8364;&#8220;685. PMID: 3378406 <br />                                           19.   Spitznagel D, Ebikeme C, Biran M, Nic a&apos; Bh&#195;&#161;ird N, Bringaud F, et al. (2009) Alanine aminotransferase <br />                                                 Trypanosoma brucei&#226;&#8364;&#8221;a key role proline metabolism procyclic life forms. FEBS J 276:7187&#226;&#8364;&#8220; <br />                                                 7199. doi: 10.1111/j.1742-4658.2009.07432.x PMID: 19895576 <br />                                           20.   Cron&#195;&#173;n CN, Nolan DP, Voorheis HP (1989) The enzymes classical pentose phosphate pathway <br />                                                 display differential activities procyclic bloodstream forms Trypanosoma brucei. FEBS Lett <br />                                                 244:26&#226;&#8364;&#8220;30. PMID: 2924907 <br />                                           21.   Barrett MP, Le Page RW (1993) A 6-phosphogluconate dehydrogenase gene Trypanosoma bru- <br />                                                 cei. Mol Biochem Parasitol 57:89&#226;&#8364;&#8220;99. PMID: 8426618 <br />                                           22.   Kerkhoven EJ, Achcar F, Alibu VP, Burchmore RJ, Gilbert IH, et al. (2013) Handling uncertainty dy- <br />                                                 namic models: pentose phosphate pathway Trypanosoma brucei. PLoS Comput Biol 9: <br />                                                 e1003371. doi: 10.1371/journal.pcbi.1003371 PMID: 24339766 <br />                                           23.   Turnock DC, Ferguson MA. (2007) Sugar nucleotide pools Trypanosoma brucei, Trypanosoma cruzi, <br />                                                 Leishmania major. Eukaryot Cell 6:1450&#226;&#8364;&#8220;1463. PMID: 17557881 <br />                                           24.   Kuettel S, Wadum MC, G&#195;&#188;ther ML, Mari&#195;&#177;o K, Riemer C, et al. (2012) The novo salvage path- <br />                                                 ways GDP-mannose biosynthesis sufficient growth bloodstream-form Trypano- <br />                                                 soma brucei. Mol Microbiol 84:340&#226;&#8364;&#8220;351. doi: 10.1111/j.1365-2958.2012.08026.x PMID: 22375793 <br />                                           25.   Martin KL, Smith TK (2006) The glycosylphosphatidylinositol (GPI) biosynthetic pathway blood- <br />                                                 stream-form Trypanosoma brucei dependent novo synthesis inositol. Mol Microbiol <br />                                                 61:89&#226;&#8364;&#8220;105. PMID: 16824097 <br />                                           26.   Gonzalez-Salgado A, Steinmann ME, Greganova E, Rauch M, M&#195;&#164;ser P, et al. (2012) myo-Inositol up- <br />                                                 essential bulk inositol phospholipid glycosylphosphatidylinositol synthesis Trypano- <br />                                                 soma brucei. J Biol Chem 287:13313&#226;&#8364;&#8220;13323. doi: 10.1074/jbc.M112.344812 PMID: 22351763 <br />                                           27.   Mazet M, Morand P, Biran M, Bouyssou G, Courtois P, et al. (2013) Revisiting central metabolism <br />                                                 bloodstream forms Trypanosoma brucei: production acetate mitochondrion essen- <br />                                                 tial parasite viability. PLoS Negl Trop Dis 7:e2587. doi: 10.1371/journal.pntd.0002587 PMID: <br />                                                 24367711 <br />  <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                                   22 / 25 <br />                                                                                                                       Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                           28.   van Weelden SW, van Hellemond JJ, Opperdoes FR, Tielens AG (2005) New functions parts the <br />                                                 Krebs cycle procyclic Trypanosoma brucei, cycle operating cycle. J Biol Chem <br />                                                 280:12451&#226;&#8364;&#8220;12460. PMID: 15647263 <br />                                           29.   Coustou V, Biran M, Besteiro S, Rivi&#195;&#168;re L, Baltz T, et al.(2006) Fumarate essential intermediary <br />                                                 metabolite produced procyclic Trypanosoma brucei. J Biol Chem 281:26832&#226;&#8364;&#8220;26846. PMID: <br />                                                 16857679 <br />                                           30.   Millerioux Y, Morand P, Biran M, Mazet M, Moreau P, et al. (2012) ATP synthesis-coupled and-uncou- <br />                                                 pled acetate production acetyl-CoA mitochondrial acetate:succinate CoA-transferase ace- <br />                                                 tyl-CoA thioesterase Trypanosoma. J Biol Chem 287:17186&#226;&#8364;&#8220;17197. doi: 10.1074/jbc.M112.355404 <br />                                                 PMID: 22474284 <br />                                           31.   Rivi&#195;&#168;re L, Moreau P, Allmann S, Hahn M, Biran M, et al. (2009) Acetate produced mitochondrion <br />                                                 essential precursor lipid biosynthesis procyclic trypanosomes. Proc Natl Acad Sci U S A <br />                                                 106:12694&#226;&#8364;&#8220;12699. doi: 10.1073/pnas.0903355106 PMID: 19625628 <br />                                           32.   Creek DJ, Anderson J, McConville MJ, Barrett MP (2012) Metabolomic analysis trypanosomatid pro- <br />                                                 tozoa. Mol Biochem Parasitol 181:73&#226;&#8364;&#8220;84. doi: 10.1016/j.molbiopara.2011.10.003 PMID: 22027026 <br />                                           33.   Breitling R, Vitkup D, Barrett MP (2008) New surveyor tools charting microbial metabolic maps. Nat <br />                                                 Rev Microbiol 6:156&#226;&#8364;&#8220;161. PMID: 18026122 <br />                                           34.   Breitling R, Pitt AR, Barrett MP (2006) Precision mapping metabolome. Trends Biotechnol <br />                                                 24:543&#226;&#8364;&#8220;548. PMID: 17064801 <br />                                           35.   MacRae JI, Sheiner L, Nahid A, Tonkin C, Striepen B, et al. (2012) Mitochondrial metabolism glucose <br />                                                 glutamine required intracellular growth Toxoplasma gondii. Cell Host Microbe 12:682&#226;&#8364;&#8220;692. <br />                                                 doi: 10.1016/j.chom.2012.09.013 PMID: 23159057 <br />                                           36.   MacRae JI, Dixon MW, Dearnley MK, Chua HH, Chambers JM, et al. (2013) Mitochondrial metabolism <br />                                                 sexual asexual blood stages malaria parasite Plasmodium falciparum. BMC Biol 11:67. <br />                                                 doi: 10.1186/1741-7007-11-67 PMID: 23763941 <br />                                           37.   Storm J, Sethia S, Blackburn GJ, Chokkathukalam A, Watson DG, et al. (2014) Phosphoenolpyruvate <br />                                                 carboxylase identified key enzyme erythrocytic Plasmodium falciparum carbon metabolism. <br />                                                 PLoS Pathog 10:e1003876. doi: 10.1371/journal.ppat.1003876 PMID: 24453970 <br />                                           38.   Creek DJ, Chokkathukalam A, Jankevics A, Burgess KE, Breitling R, et al. (2012) Stable isotope-assis- <br />                                                 ted metabolomics network-wide metabolic pathway elucidation. Anal Chem 84:8442&#226;&#8364;&#8220;8447. doi: 10. <br />                                                 1021/ac3018795 PMID: 22946681 <br />                                           39.   Chokkathukalam A, Jankevics A, Creek DJ, Achcar F, Barrett MP, Breitling R (2013) mzMatch-ISO: an <br />                                                 R tool annotation relative quantification isotope-labelled mass spectrometry data. Bioin- <br />                                                 formatics. 29:281&#226;&#8364;&#8220;283. doi: 10.1093/bioinformatics/bts674 PMID: 23162054 <br />                                           40.   Achcar F, Barrett MP, Breitling R (2013) Explicit consideration topological parameter uncertainty <br />                                                 gives new insights well-established model glycolysis. FEBS J 280:4640&#226;&#8364;&#8220;4651. doi: 10.1111/ <br />                                                 febs.12436 PMID: 23865459 <br />                                           41.   Gualdron-L&#195;&#179;pez M, Vapola MH, Miinalainen IJ, Hiltunen JK, Michels PA, Antonenkov VD (2012) Chan- <br />                                                 nel-forming activities glycosomal fraction bloodstream form Trypanosoma brucei. <br />                                                 PLoS One 7:e34530. doi: 10.1371/journal.pone.0034530 PMID: 22506025 <br />                                           42.   Greig N, Wyllie S, Patterson S, Fairlamb AH (2009) A comparative study methylglyoxal metabolism <br />                                                 trypanosomatids. FEBS J 276:376&#226;&#8364;&#8220;386. doi: 10.1111/j.1742-4658.2008.06788.x PMID: 19076214 <br />                                           43.   Montgomery CM, Webb JL (1956) Metabolic studies heart mitochondria. II. The inhibitory action of <br />                                                 parapyruvate tricarboxylic acid cycle. J Biol Chem 221:359&#226;&#8364;&#8220;368. PMID: 13345825 <br />                                           44.   Stoffel SA, Alibu VP, Hubert J, Ebikeme C, Portais JC et al. (2011) Transketolase Trypanosoma bru- <br />                                                 cei. Mol Biochem Parasitol 179:1&#226;&#8364;&#8220;7. doi: 10.1016/j.molbiopara.2011.04.006 PMID: 21570429 <br />                                           45.   Fish WR, Looker DL, Marr JJ, Berens RL (1982) Purine metabolism bloodstream forms Trypa- <br />                                                 nosoma gambiense Trypanosoma rhodesiense. Biochim Biophys Acta 719:223&#226;&#8364;&#8220;231. PMID: <br />                                                 6817814 <br />                                           46.   Creek DJ, Nijagal B, Kim DH, Rojas F, Matthews KR, et al. (2013) Metabolomics guides rational devel- <br />                                                 opment simplified cell culture medium drug screening Trypanosoma brucei. Antimicrob <br />                                                 Agents Chemother 57:2768&#226;&#8364;&#8220;2779. doi: 10.1128/AAC.00044-13 PMID: 23571546 <br />                                           47.   Parkin DW (1996) Purine-specific nucleoside N-ribohydrolase Trypanosoma brucei brucei. Purifi- <br />                                                 cation, specificity, kinetic mechanism. J Biol Chem 271:21713&#226;&#8364;&#8220;21719. PMID: 8702965 <br />                                           48.   Davies MJ, Ross AM, Gutteridge WE (1983) The enzymes purine salvage Trypanosoma cruzi, Try- <br />                                                 panosoma brucei Leishmania mexicana. Parasitology 87:211&#226;&#8364;&#8220;217. PMID: 6316234 <br />                                           49.   Vodnala M, Fijolek A, Rofougaran R, Mosimann M, M&#195;&#164;ser P, Hofer A (2008) Adenosine kinase medi- <br />                                                 ates high affinity adenosine salvage Trypanosoma brucei J Biol Chem 283:5380&#226;&#8364;&#8220;5388. doi: 10. <br />                                                 1074/jbc.M705603200 PMID: 18167353 <br />  <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                                  23 / 25 <br />                                                                                                                        Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                           50.   Berger BJ, Dai WW, Wang H, Stark RE, Cerami A (1996) Aromatic amino acid transamination me- <br />                                                 thionine recycling trypanosomatids. Proc Natl Acad Sci USA 93: 4126&#226;&#8364;&#8220;4130. PMID: 8633027 <br />                                           51.   Hasne MP, Barrett MP (2000) Transport methionine Trypanosoma brucei brucei. Mol Biochem <br />                                                 Parasitol 111:299&#226;&#8364;&#8220;307. PMID: 11163438 <br />                                           52.   Durieux PO, Sch&#195;&#188;tz P, Brun R, K&#195;&#182;hler P (1991) Alterations Krebs cycle enzyme activities carbo- <br />                                                 hydrate catabolism strains Trypanosoma brucei vitro differentiation blood- <br />                                                 stream procyclic stages. Mol Biochem Parasitol 45:19&#226;&#8364;&#8220;27. PMID: 1904988 <br />                                           53.   Hart DT, Misset O, Edwards SW, Opperdoes FR (1984) A comparison glycosomes (microbodies) <br />                                                 isolated Trypanosoma brucei bloodstream form cultured procyclic trypomastigotes. Mol Bio- <br />                                                 chem Parasitol 12:25&#226;&#8364;&#8220;35. PMID: 6749187 <br />                                           54.   Hasne, MP (2001) Amino acid transporters Trypanosoma brucei brucei. University Glasgow PhD <br />                                                 thesis. <br />                                           55.   Aranda A, Maugeri D, Uttaro AD, Opperdoes F, Cazzulo JJ, et al.(2006) The malate dehydrogenase <br />                                                 isoforms Trypanosoma brucei: subcellular localization differential expression bloodstream <br />                                                 procyclic forms. Int J Parasitol 36:295&#226;&#8364;&#8220;307. PMID: 16321390 <br />                                           56.   Opperdoes FR, Borst P, Spits H (1977) Particle-bound enzymes bloodstream form Trypano- <br />                                                 soma brucei. Eur J Biochem 76:21&#226;&#8364;&#8220;28. PMID: 195809 <br />                                           57.   Opperdoes FR, Borst P, Bakker S, Leene W (1977) Localization glycerol-3-phosphate oxidase the <br />                                                 mitochondrion particulate NAD+-linked glycerol-3-phosphate dehydrogenase microbodies of <br />                                                 bloodstream form Trypanosoma brucei. Eur J Biochem 76:29&#226;&#8364;&#8220;39. PMID: 142010 <br />                                           58.   Ali JA, Tagoe DN, Munday JC, Donachie A, Morrison LJ, Koning HP (2013) Pyrimidine biosynthesis <br />                                                 essential function Trypanosoma brucei bloodstream forms. PLoS One 8:e58034. doi: 10. <br />                                                 1371/journal.pone.0058034 PMID: 23505454 <br />                                           59.   Ali JA, Creek DJ, Burgess K, Allison HC, Field MC, et al. (2013) Pyrimidine salvage Trypanosoma <br />                                                 brucei bloodstream forms trypanocidal action halogenated pyrimidines. Mol Pharmacol <br />                                                 83:439&#226;&#8364;&#8220;453. doi: 10.1124/mol.112.082321 PMID: 23188714 <br />                                           60.   Alsford S, Turner DJ, Obado SO, Sanchez-Flores A, Glover L, et al. (2011) High-throughput phenotyp- <br />                                                 ing using parallel sequencing <span id='am-398' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-399' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-400' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-401' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-402' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-404' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-405' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-406' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-407' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-408' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-409' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-410' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-411' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-412' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-413' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-414' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-415' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-417' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-418' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-421' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-422' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-424' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-426' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-427' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-429' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-430' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-431' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-434' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-436' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-438' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-439' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-441' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-442' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-444' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-445' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-447' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-449' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-453' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-454' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-455' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-456' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-457' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-458' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-459' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-460' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-461' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-462' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-463' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-464' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-466' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-467' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-469' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-470' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-471' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-472' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-473' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-474' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-475' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>RNA interference</span> targets African trypanosome. Genome Res <br />                                                 21:915&#226;&#8364;&#8220;924. doi: 10.1101/gr.115089.110 PMID: 21363968 <br />                                           61.   Coustou V, Besteiro S, Biran M, Diolez P, Bouchaud V et al. (2003) ATP generation Trypano- <br />                                                 soma brucei procyclic form: cytosolic substrate level essential, oxidative phosphorylation. <br />                                                 J Biol Chem 278:49625&#226;&#8364;&#8220;49635. PMID: 14506274 <br />                                           62.   Bowes AE, Samad AH, Jiang P, Weaver B, Mellors A (1993) The acquisition lysophosphatidylcholine <br />                                                 African trypanosomes. J Biol Chem 268:13885&#226;&#8364;&#8220;13892. PMID: 8314756 <br />                                           63.   Lee SH, Stephens JL, Paul KS, Englund PT (2006) Fatty acid synthesis elongases trypanosomes. <br />                                                 Cell 126:691&#226;&#8364;&#8220;699. PMID: 16923389 <br />                                           64.   Mari&#195;&#177;o K, G&#195;&#188;ther ML, Wernimont AK, Amani M, Hui R, et al. (2010) Identification, subcellular localiza- <br />                                                 tion, biochemical properties, high-resolution crystal structure Trypanosoma brucei UDP-glucose <br />                                                 pyrophosphorylase. Glycobiology 20:1619&#226;&#8364;&#8220;1630. doi: 10.1093/glycob/cwq115 PMID: 20724435 <br />                                           65.   Saunders EC, Ng WW, Chambers JM, Ng M, Naderer T, et al. (2011) Isotopomer profiling Leishman- <br />                                                 ia mexicana promastigotes reveals important roles succinate fermentation aspartate uptake in <br />                                                 tricarboxylic acid cycle (TCA) anaplerosis, glutamate synthesis, growth. J Biol Chem 286:27706&#226;&#8364;&#8220; <br />                                                 27717. doi: 10.1074/jbc.M110.213553 PMID: 21636575 <br />                                           66.   Achcar F, Kerkhoven EJ, SilicoTryp Consortium, Bakker BM, Barrett MP, et al.(2012) Dynamic model- <br />                                                 ling uncertainty: case Trypanosoma brucei energy metabolism. PLoS Comput Biol 8: <br />                                                 e1002352. doi: 10.1371/journal.pcbi.1002352 PMID: 22379410 <br />                                           67.   Hannaert V, Saavedra E, Duffieux F, Szikora JP, Rigden DJ et al.(2003) Plant-like traits associated <br />                                                 metabolism Trypanosoma parasites. Proc Natl Acad Sci USA 100:1067&#226;&#8364;&#8220;1071. PMID: <br />                                                 12552132 <br />            <br /> </body></html>