<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             Curr Biol. Author manuscript; available PMC 2015 March 17. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Curr Biol. 2014 March 17; 24(6): 621&#226;&#8364;&#8220;629. doi:10.1016/j.cub.2014.02.026. <br />  <br />  <br />  <br />                            SLEEPLESS bi-functional regulator excitability and <br />                            cholinergic synaptic transmission <br />                            Meilin Wu1, James E. Robinson1,2, William J. Joiner1,2,3,4 <br />                            1Department       Pharmacology, University California San Diego, La Jolla, CA, 92093-0636, <br />                            USA <br />                            2Biomedical   Sciences Graduate Program Molecular Pharmacology Training Grant, University <br />                            California San Diego, La Jolla, CA, 92093-0636, USA <br />                            3Neurosciences        Graduate Program, University California San Diego, La Jolla, CA, 92093-0636, <br />                            USA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            4Center    Chronobiology, University California San Diego, La Jolla, CA, 92093-0636, USA <br />  <br />                            Summary <br />                                 Background&#226;&#8364;&#8221;Although sleep conserved evolution, molecular basis its <br />                                 control largely mystery. We previously showed quiver/sleepless (qvr/sss) gene <br />                                 encodes membrane-tethered protein required normal sleep Drosophila. <br />                                 SLEEPLESS (SSS) protein functions, part, upregulating levels open <br />                                 probability Shaker (Sh) potassium channels suppress neuronal excitability enable sleep. <br />                                 Consistent proposed mechanism, loss-of-function mutations Sh phenocopy qvr/sss null <br />                                 mutants. However, sleep genetically modifiable Sh qvr/sss mutants, suggesting <br />                                 sss regulate additional molecules influence sleep. <br />  <br />                                 Results&#226;&#8364;&#8221;Here SSS antagonizes nicotinic acetylcholine receptors (nAChRs) <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                 reduce synaptic transmission promote sleep. Mimicking antagonism nAChR <br />                                 inhibitor mecamylamine RNAi knockdown specific nAChR subunits sufficient to <br />                                 restore sleep qvr/sss mutants. Regulation nAChR activity SSS occurs post- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 transcriptionally levels nAChR mRNAs unchanged qvr/sss mutants. Regulation <br />                                 nAChR activity SSS fact direct, SSS forms stable complex and <br />                                 antagonizes fly nAChR function transfected cells. Intriguingly, lynx1, mammalian homolog of <br />                                 SSS, partially restore normal sleep qvr/sss mutants, lynx1 form stable complexes <br />                                 Shaker-type channels nAChRs. <br />  <br />  <br />  <br />  <br />                            &#194;&#169; 2014 Elsevier Inc. All rights reserved <br />                            Address Correspondence: Dr. William Joiner 9500 Gilman Drive, BSB 3082 La Jolla, CA 92093-0636 wjoiner@ucsd.edu. <br />                            Publisher&apos;s Disclaimer: This PDF file unedited manuscript accepted publication. As service our <br />                            customers providing early version manuscript. The manuscript undergo copyediting, typesetting, review of <br />                            resulting proof published final citable form. Please note production process errors be <br />                            discovered affect content, legal disclaimers apply journal pertain. <br />                             Wu et al.                                                                                                      Page 2 <br />  <br />  <br />                                        Conclusions&#226;&#8364;&#8221;Together, data point evolutionarily conserved, bi-functional role for <br />                                        SSS homologs controlling excitability synaptic transmission fundamental <br />                                        processes nervous sleep. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Introduction <br />                                                  Sleep essential process highly conserved evolution, functions and <br />                                                  underlying mechanisms control poorly understood. One conserved <br />                                                  features sleep association large-scale changes brain activity relative the <br />                                                  waking state, suggesting modulation neuronal activity central the <br />                                                  regulation sleep. In support hypothesis, homologous genes mammals flies <br />                                                  encoding ion channels ionotropic receptors shown necessary normal <br />                                                  sleep [1&#226;&#8364;&#8220;4]. The requirement potassium (K) channels Shaker (Sh) family is <br />                                                  particularly notable. In mammals flies, loss-of-function mutations Sh its <br />                                                  orthologs cause reduced sleep [2, 4]. Although little known modulators of <br />                                                  Sh-type channels control sleep mammals, loss-of-function mutations of <br />                                                  genes upregulate Sh, hyperkinetic (Hk) quiver/sleepless (qvr/sss), result in <br />                                                  reduced sleep flies [5&#226;&#8364;&#8220;7]. Unlike Hk, cytosolic protein, SSS anchored by <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  glycosylphosphatidylinositol (GPI) outer leaflet plasma membrane, it <br />                                                  associate extracellular surfaces membrane proteins. Evidence suggests <br />                                                  protein Sh itself. For example, SSS Sh expressed the <br />                                                  regions fly brain; SSS form stable complex upregulate levels of <br />                                                  Sh protein; SSS increase activation kinetics decrease C-type inactivation <br />                                                  kinetics Sh channels excised patches transfected cells [6&#226;&#8364;&#8220;8]. <br />  <br />                                                  However, differences sleep phenotypes Sh qvr/sss mutants suggest that <br />                                                  genes act exactly signaling pathways. For example, several <br />                                                  generations selection pressure appears favor accumulation genetic modifiers in <br />                                                  populations Sh mutants compensate loss sleep [4]. In contrast, loss sleep in <br />                                                  qvr/sss <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>null mutants</span> lost time span, (unpublished data), <br />                                                  signifying qvr/sss <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>phenotype</span> easily overcome naturally occurring <br />                                                  alleles genome. Furthermore, Sh mutants exhibit homeostatic recovery sleep <br />                                                  following periods sleep deprivation [4], process impaired qvr/sss mutants [6]. <br />                                                  Collectively, differences suggest SSS downstream effectors <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  which, combination Sh, control sleep. <br />  <br />                                                  Clues identities effectors gleaned predicted structure of <br />                                                  SSS. Modeling tertiary structure reveals SSS member large family of <br />                                                  proteins, including snake &#206;&#177;-neurotoxins, possess &#226;&#8364;&#339;three-finger&#226;&#8364;? fold also <br />                                                  referred ly6 domain [7]. Intriguingly, targets &#206;&#177;-neurotoxins <br />                                                  identified ion channels acetylcholine signaling pathways. For example, FS-2 <br />                                                  blocks voltage-gated Ca channels [9]; fasciculin-I blocks acetylcholinesterase [10]; MT2 <br />                                                  acts muscarinic acetylcholine receptors [11]; &#206;&#177;-bungarotoxin inhibits nicotinic <br />                                                  acetylcholine receptors (nAChRs) [12]. Although ly6 domain-encoding homologs qvr /sss <br />                                                  identified genomes non-venomous animals [13, 14], cases <br />                                                  endogenous binding partners corresponding proteins identified. <br />  <br />  <br />                                                        Curr Biol. Author manuscript; available PMC 2015 March 17. <br />                             Wu et al.                                                                                             Page 3 <br />  <br />  <br />                                          In mammals, however, evidence strongly suggests proteins, lynx1 and <br />                                          lynx2, endogenous antagonists nAChRs. For example, lynx1 lynx2 complex <br />                                          accelerate desensitization rates &#206;&#177;4&#206;&#178;2 nAChRs [15, 16]. Consistent these <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          effects, lynx1 <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>knockout</span> mice extended critical periods ocular dominance plasticity <br />                                          requires nAChR activity [17], lynx2 knockout mice increased nicotine-evoked <br />                                          EPSPs prefrontal cortical pyramidal neurons [18]. <br />  <br />                                          As result studies, hypothesized ly6 domain-containing protein SSS may <br />                                          antagonize nAChRs reduce synaptic transmission. Here present evidence to <br />                                          support hypothesis demonstrate resulting receptor inhibition required for <br />                                          normal sleep Drosophila. Specifically, pharmacological genetic reduction <br />                                          nAChR signaling restores sleep qvr/sss mutants. We SSS regulates <br />                                          nAChRs post-transcriptionally, likely direct inhibition SSS co- <br />                                          immunoprecipitates fly nAChR D&#206;&#177;3 inhibits nAChR activity transfected <br />                                          cells. Finally, mammalian SSS homolog lynx1 restores sleep qvr/sss <br />                                          mutants co-immunoprecipitates murine homolog Sh, Kv1.2. Collectively our <br />                                          data suggest existence evolutionarily conserved, dual role SSS-like molecules <br />                                          reducing excitability synaptic transmission control essential nervous system <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          functions like sleep. <br />  <br />  <br />                               Results <br />                               SSS-expressing neurons required waking <br />                                          We previously demonstrated loss-of-function mutations qvr/sss Sh lead to <br />                                          increased mEPSP frequency reduced Sh current individual muscle fibers fly <br />                                          neuromuscular junction (NMJ). We concluded loss SSS Sh leads increased <br />                                          excitability. Since effect accompanied increased waking, hypothesized that <br />                                          loss sss Sh leads increased activity wake-promoting neurons central brain. <br />                                          Loss sleep qvr/sss Sh mutants attributed seizure activity associated <br />                                          hyperexcitability effect reduce locomotor activity, leading an <br />                                          apparent increase sleep, decrease actually observed animals. Loss <br />                                          sleep qvr/sss Sh mutants does appear caused hyperlocomotion <br />                                          associated hyperexcitability waking activity unchanged qvr/sss [6]. To <br />                                          determine role qvr/sss-expressing neurons control sleep/wake cycle more <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          directly, used Gal4/UAS [19] express temperature-sensitive TRPA1 <br />                                          channels [20] neurons normally express qvr/sss [7]. By activating neurons with <br />                                          temperature pulse 29&#194;&#176; C 6 hours zeitgeber time (ZT) 18&#226;&#8364;&#8220;24, sss-Gal4/UAS- <br />                                          TRPA1 animals deprived sleep relative pulsed controls (Figures 1a, S1a). Thus, <br />                                          activation qvr/sss-expressing neurons appears sufficient waking. <br />  <br />                                          In addition, expressed temperature-sensitive shibire (shits) qvr/sss neurons under <br />                                          Gal4/UAS control. shits encodes dominant mutant version dynamin ATPase blocks <br />                                          endocytosis exocytosis restrictive temperatures [21]. By raising temperature <br />                                          sss-Gal4/UAS-shits animals 28&#194;&#176;C 6 hours ZT0-6, sleep <br />                                          significantly increased experimental animals compared controls lacked sss-Gal4 or <br />                                          shits expression (Figure 1b, S1b). This change sleep sickness paralysis <br />  <br />                                                Curr Biol. Author manuscript; available PMC 2015 March 17. <br />                             Wu et al.                                                                                                Page 4 <br />  <br />  <br />                                          experimental animals able right 10 seconds being <br />                                          inverted temperature pulse (28/31, sss-Gal4&gt;UAS-shits; 32/32, sss-Gal4/+, 32/32, <br />                                          UAS-shits/+). Thus, activation qvr/sss-expressing neurons appears required for <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          waking. <br />  <br />                               Pharmacological antagonism nAChR Sh activities influences sleep <br />                                          We previously demonstrated reduction SSS Sh causes reciprocal <br />                                          reduction protein [6, 7]. This observation consistent idea two <br />                                          proteins form complex mutually stabilize other. These results led to <br />                                          hypothesize restoring expression Sh qvr/sss mutants vice-versa using the <br />                                          Gal4/UAS rescue severe sleep defects observed groups animals. <br />                                          However, expressing UAS-Sh sssP1 mutants UAS-sss Shmns mutants the <br />                                          control sss-Gal4 sufficient restore sleep normal levels (Figure S1c), <br />                                          suggesting additional factors required. <br />  <br />                                          Interestingly, SSS homologous endogenous nAChR antagonists lynx1 lynx2 as <br />                                          snake &#206;&#177;-neurotoxins, affect acetylcholine signaling. Thus, we <br />                                          hypothesized SSS normally reduce nAChR activity wake-promoting neurons to <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          enable animals sleep. According hypothesis, nAChR activity elevated in <br />                                          qvr/sss loss-of-function mutants Sh mutants well, SSS upregulates <br />                                          Sh. To test hypothesis, fed wild type, sssP1 Shmns flies increasing concentrations <br />                                          non-competitive nAChR antagonist mecamylamine (MCA) measured sleep 24&#226;&#8364;&#8220;48 <br />                                          hours later. We treatment wild-type animals drug caused small <br />                                          increase sleep. In contrast, treatment sssP1 Shmns mutants MCA caused a <br />                                          dose-dependent restoration sleep (Figures 1c 1d). We interpret changes true <br />                                          sleep sickness paralysis behavioral state acutely reversible <br />                                          brief mechanical agitation, waking activity drug treatment high as <br />                                          controls (Figure S2a,b). These findings strongly suggest ability SSS Sh to <br />                                          promote sleep normal conditions mediated suppression nAChR <br />                                          activity, loss suppression leads reduced sleep qvr/sss Sh mutants. <br />                                          This mechanism somewhat selective qvr/sss Sh mutants since <br />                                          mecamylamine able increase sleep Clkjrk mutants (Fig S3), been <br />                                          reported low sleep [22&#226;&#8364;&#8220;24]. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Although previously reported SSS enhances Sh protein levels gating <br />                                          kinetics, time did demonstrate direct role Sh mediating low sleep <br />                                          phenotype qvr/sss mutants. To examine relationship, tested acute <br />                                          pharmacological inhibition Sh activity wild-type animals reduce sleep, as <br />                                          observed sssP1 Shmns mutants. To accomplish task, fed animals voltage- <br />                                          gated potassium channel blocker 4-aminopyridine (4-AP). 4-AP treatment able dose- <br />                                          dependently reduce sleep wild-type background levels similar observed in <br />                                          Shmns mutants did affect waking activity (Figures 1e, 1f S2c). 4-AP appeared to <br />                                          act selectively Sh ion channels reduce sleep no <br />                                          effect drug Shmns mutants (Figures 1e 1f). Tellingly, trans-heterozygous <br />                                          sssP1/sssP2 mutants sleep reduced levels equivalent observed in <br />  <br />  <br />                                                Curr Biol. Author manuscript; available PMC 2015 March 17. <br />                             Wu et al.                                                                                              Page 5 <br />  <br />  <br />                                           Shmns mutants, 4-AP acute effect. These results suggest reduced sleep in <br />                                           qvr/sss mutants partially mediated reduction Sh activity, turn <br />                                           leads increased neuronal excitability arousal-promoting neurons, previously <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           hypothesized [6, 7]. <br />  <br />                               qvr/sss genetically interacts nAChR subunits control sleep <br />                                           After determining acute antagonism nAChR activity restore sleep sssP1 <br />                                           mutants, asked nAChR subunits aberrantly upregulated absence of <br />                                           SSS. To address question determined nAChR subunits expressed in <br />                                           adult fly brain. In mammals, nAChRs homopentamers alpha subunits or <br />                                           heteropentamers alpha beta subunits. In fly genome 7 genes encoding <br />                                           alpha subunits (D&#206;&#177;1&#226;&#8364;&#8220;7) 3 genes encoding beta subunits (D&#206;&#178;1&#226;&#8364;&#8220;3). The combinations of <br />                                           receptor subunits form functional channels flies unknown, largely it <br />                                           difficult measure activity cloned fly nAChRs heterologous expression <br />                                           systems [25]. Using quantitative PCR (qPCR) measure levels nAChR transcripts, <br />                                           subunit enriched adult fly brain relative body <br />                                           (Figure S4a). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           To determine subunits responsible increased waking time of <br />                                           qvr/sss mutants, reduced expression alpha beta subunit sss-expressing <br />                                           neurons RNAi knockdown. Consistent hypothesis nAChR activity is <br />                                           upregulated qvr/sss mutants, knockdown D&#206;&#177;3 particular lesser extent, D&#206;&#178;3, <br />                                           partly restored sleep sssP1 mutants (Figure 2a 2b) affecting waking activity <br />                                           (Figure S2d) effect sleep control animals normal levels qvr/sss <br />                                           expression (Figure S4b). qPCR analysis D&#206;&#177;3 transcripts heads pan-neuronal <br />                                           RNAi knockdown flies confirms ~65% reduction D&#206;&#177;3 expression levels (Figure S4c). <br />                                           These results suggest excessive nAChR activity partly responsible for <br />                                           increased waking time sssP1 mutants. These results indicate nAChR activity <br />                                           does normally need reduced withdrawal endogenous cholinergic signaling to <br />                                           allow wild-type animals sleep. <br />  <br />                                           To determine upregulation nAChR activity qvr/sss-expressing neurons could <br />                                           reduce sleep, coupled sss-Gal4 P-element (d08339) inserted ~40 bp upstream the <br />                                           predicted transcriptional start site D&#206;&#177;3 gene. Since P-element carries UAS <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           sequences pointing directions X chromosome, animals bearing in <br />                                           combination sss-Gal4 express elevated levels D&#206;&#177;3 transcript. When tested <br />                                           animals expressed high levels D&#206;&#177;3 mRNA slept less <br />                                           controls (Figures S4d, 2c). Thus, upregulation D&#206;&#177;3 nAChRs qvr/sss-expressing <br />                                           neurons sufficient account low-sleeping phenotype sssP1 <br />                                           mutants. <br />  <br />                               nAChRs post-transcriptionally regulated SSS co-localize SSS Sh in <br />                               mushroom bodies <br />                                           To determine SSS indirectly regulate nAChRs transcriptional feedback, we <br />                                           performed qPCR D&#206;&#177;3 D&#206;&#178;3, nAChR subunits genetically interacted <br />  <br />  <br />                                                 Curr Biol. Author manuscript; available PMC 2015 March 17. <br />                             Wu et al.                                                                                             Page 6 <br />  <br />  <br />                                          qvr/sss (Figure 2a 2b). Just previously showed regulation Sh transcript <br />                                          [7], levels fly brain nAChR transcripts unchanged sssP1 mutants. We found <br />                                          fly brain nAChR transcripts similarly unchanged Shmns mutants (Figure 3a). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Collectively, data support post-transcriptional role regulation nAChR activity <br />                                          SSS, exclude possibility upregulation nAChR transcript in <br />                                          qvr/sss mutants occurs small subset neurons brain preparations and <br />                                          undetectable assays. <br />  <br />                                          If, however, SSS acting directly K channels nAChRs suppress neuronal <br />                                          activity promote sleep, SSS effector proteins expressed the <br />                                          set neurons. To determine case, labeled qvr/sss-expressing <br />                                          neurons UAS-tdTomato expressed control sss-Gal4. In brains we <br />                                          stained presence Sh D&#206;&#177;3 using established antibodies molecule <br />                                          [7, 26]. We overlapping expression 3 molecules mushroom bodies (MBs) <br />                                          (Figure 3b), previously showed important sleep-regulatory loci <br />                                          [27, 28]. Our cell labeling likely specific intended antigens Sh <br />                                          labeling disappeared Sh mutants [7]; D&#206;&#177;3 labeling significantly reduced when <br />                                          antibody pre-incubated lysate COS cells transfected D&#206;&#177;3 cDNA (Figure <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          S5a&#226;&#8364;&#8220;c); D&#206;&#177;3 labeling increased brains animals sss-Gal4 combined <br />                                          UAS-D&#206;&#177;3 (i.e. P-element d08339) (Figure S5d). Thus, sleep-regulating proteins <br />                                          SSS, Sh D&#206;&#177;3 expressed known sleep-regulating locus fly brain. <br />  <br />                                          To determine MBs contribute sleep regulation qvr/sss, first <br />                                          coupled sss-Gal4 driver UAS-sss sssP1 mutant background. As previously <br />                                          demonstrated [7], qvr/sss transgene capable fully rescuing loss <br />                                          sleep sssP1 mutants. However, blocked expression qvr/sss transgene <br />                                          MB-Gal80, repressor Gal4 activity expresses MBs, restoration sleep <br />                                          genetically identical animals reduced (Figure 3c). These data suggest that <br />                                          qvr/sss utilizes MBs regulate sleep, exclude possibility that <br />                                          brain regions involved. <br />  <br />                               SSS forms stable complexes Sh D&#206;&#177;3 <br />                                          Our findings SSS upregulates Sh antagonizes nAChR activity vivo suggest that <br />                                          SSS mediate effects directly. We previously demonstrated SSS form a <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          complex Sh channels Xenopus oocytes, express K currents robustly <br />                                          following injection Sh cRNA. To determine SSS form complexes Sh channels <br />                                          nAChRs transfected cells, added MYC epitope near C-terminus of <br />                                          SSS, just attachment site glycosylphosphatidylinositol. We fused green <br />                                          fluorescent protein (GFP) hemagglutinin (HA) epitopes N- C-termini Sh <br />                                          D&#206;&#177;3, respectively. These constructs expressed HEK-tsa cells, complexes <br />                                          immunoprecipitated antibodies GFP (i.e. Sh) HA (i.e. D&#206;&#177;3). <br />                                          Subsequent western blot analyses revealed SSS co-immunoprecipitated Sh <br />                                          (Figure 4a) D&#206;&#177;3 (Figure 4b), absence channel receptor (Figure 4a <br />                                          4b). These results demonstrate SSS form stable complexes Sh and <br />                                          D&#206;&#177;3. <br />  <br />  <br />                                                Curr Biol. Author manuscript; available PMC 2015 March 17. <br />                             Wu et al.                                                                                               Page 7 <br />  <br />  <br />                               SSS reduces nAChR activity <br />                                          Regulation Sh kinetics SSS previously demonstrated multiple expression <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          systems [7, 8]. To determine complexes formed SSS nAChRs result altered <br />                                          receptor activity, examined contribution SSS nAChR function transfected <br />                                          HEK-tsa cells. For experiments used nAChRs mice fly nAChRs have <br />                                          recalcitrant functional expression own. In absence clear murine <br />                                          orthologs fly nAChRs chose &#206;&#177;4/&#206;&#178;2 assay receptors express well <br />                                          heterologous expression systems. In assay measured agonist-dependent influx of <br />                                          Ca2+ receptors using ratiometric FRETable cytosolic Ca2+ reporter TN-XXL <br />                                          [29]. In presence &#206;&#177;4/&#206;&#178;2 TN-XXL, nicotinic agonist epibatidine elicited an <br />                                          increase FRET ratio well-fit sigmoidal concentration-response curve a <br />                                          half-maximal effective concentration (EC50) similar previous reports [29]. In cells in <br />                                          qvr/sss cDNA included transfection mixture, however, maximal <br />                                          nAChR response reduced 75% (Figures 4c 4d). Thus, transfected cells, SSS <br />                                          suppresses nAChR function, just vivo. Taken data showing SSS <br />                                          fly nAChRs form stable complex, likely functional suppression of <br />                                          nAChR activity SSS direct. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               The molecular functions SSS homologs conserved evolution <br />                                          SSS structurally resembles snake &#206;&#177;-neurotoxins &#206;&#177;-bungarotoxin, blocks <br />                                          nAChRs, ly6-domain containing proteins endogenous mammalian <br />                                          nAChR antagonists lynx1 lynx2. While SSS lynx1 share 18% amino acid <br />                                          identity, contain series conserved cysteines thought form disulfide <br />                                          bonds maintain characteristic &#226;&#8364;&#339;three-finger&#226;&#8364;? tertiary structure ly6 domain [7]. <br />                                          Thus, possible tertiary structures SSS lynx1 similar the <br />                                          proteins function interchangeably. <br />  <br />                                          To test hypothesis, performed co-immunoprecipitation experiments MYC- <br />                                          tagged lynx1 HA-tagged D&#206;&#177;3 transiently transfected HEK-tsa cells. Lynx1 known <br />                                          associate mammalian nAChRs, like SSS, form stable <br />                                          complexes D&#206;&#177;3 (Figure 5a). We tested if, like SSS, lynx1 able <br />                                          associate voltage-gated K channels. We fused GFP N-terminus Sh well <br />                                          mammalian homolog, Kv1.2, co-transfected cells construct with <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          MYC-tagged lynx1 cDNA. In cases lynx1 co- <br />                                          immunoprecipitated K channel (Figure 5b). The formation complexes <br />                                          attributed tendency lynx1 associate proteins non-specifically <br />                                          lynx1could co-immunoprecipitated membrane protein GRID2 (Figure <br />                                          5b). These results suggest targeting voltage-gated K channels nAChRs ly6 <br />                                          proteins conserved evolution. <br />  <br />                                          Based conservation molecular function, tested lynx1 replace <br />                                          SSS allow flies sleep normally. We generated UAS-lynx1 transgenic flies and <br />                                          expressed lynx1 control sss-Gal4. Remarkably, lynx1 restore <br />                                          sleep sssP1 mutants altering waking activity (Figures 5c, S2e), demonstrating that <br />                                          lynx1 function sufficiently like SSS substitute vivo. <br />  <br />  <br />                                                Curr Biol. Author manuscript; available PMC 2015 March 17. <br />                             Wu et al.                                                                                             Page 8 <br />  <br />  <br />                               Discussion <br />                                        We previously demonstrated SSS couples Sh levels gating kinetics reduced <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        membrane excitability allow sleep Drosophila. Here SSS interacts with <br />                                        antagonizes nAChRs promote sleep well, activity sss-expressing <br />                                        neurons necessary sufficient process Drosophila. The molecular bi- <br />                                        functionality SSS unexpected K channels nAChRs functionally and <br />                                        structurally unrelated. For example, Sh-type channels gated voltage, 6 <br />                                        transmembrane domains multimerize form functional tetramers. In contrast, nAChRs <br />                                        gated synaptic release acetylcholine, 4 transmembrane domains and <br />                                        multimerize form functional pentamers. <br />  <br />                                        It unclear structural features SSS recognizes class membrane protein, <br />                                        especially case Sh, thought expose little surface area outside plasma <br />                                        membrane. X-ray structures &#206;&#177;-cobtratoxin (&#206;&#177;-Cbtx) bound Lymnaea stagnalis <br />                                        acetylcholine binding protein (LS-AChBP) interactions loops &#206;&#177;-Cbtx <br />                                        agonist binding pocket cis-loop AChBP [30]. Using information <br />                                        NMR structure water-soluble lynx1, Lyukmanova et. al, modeled possible <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        interaction lynx1 site, loops lynx1 shorter those <br />                                        &#206;&#177;-Cbtx, did form contacts AChBP [31]. <br />  <br />                                        It unclear interactions, translated SSS, result acute <br />                                        direct antagonism nAChR activity or, alternatively, reduction targeting receptors <br />                                        cell surface. Both potential mechanisms action account reduced &#206;&#177;4/&#206;&#178;2 <br />                                        activity observed cells transfected qvr/sss. In addition, unknown a <br />                                        single SSS molecule interact Sh nAChRs simultaneously. However, our <br />                                        data suggest proteins co-expressed regions brain, particularly <br />                                        mushroom bodies, shown play important role in <br />                                        controlling sleep [27, 28, 32]. We previously showed SSS enriched structures <br />                                        antennal nerves, superior protocerebrum lobula plate optic lobes <br />                                        [7], MB-Gal80 reduces ability sss-Gal4&gt;UAS-sss restore <br />                                        sleep sssP1 mutants. Nonetheless, contribution MBs relative brain loci <br />                                        regulating sleep SSS, Sh D&#206;&#177;3 needs determined. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        A particularly intriguing feature SSS ability reduce membrane excitability <br />                                        synaptic transmission, endows protein unusual gain control over <br />                                        neuronal activity. In present context propose SSS reduces activity wake- <br />                                        promoting neurons pathways permit sleep (Figure 6). In one, enhances Sh <br />                                        K channel protein levels open channel probability reduce neuronal excitability. In <br />                                        pathway, inhibits nAChR signaling reduce synaptic transmission. In qvr/sss <br />                                        mutants processes reversed wake-promoting neurons: K channel activity is <br />                                        reduced, leading increased excitability, nAChR activity increased, leading to <br />                                        increased synaptic transmission. In Sh mutants, reduction nAChR <br />                                        expression RNAi knockdown nAChR activity pharmacological antagonism <br />                                        reduces activity wake-promoting neurons range sleep once <br />                                        occur (Figure 6). <br />  <br />  <br />                                              Curr Biol. Author manuscript; available PMC 2015 March 17. <br />                             Wu et al.                                                                                               Page 9 <br />  <br />  <br />                                           Our finding molecular properties SSS shared mammalian homolog <br />                                           lynx1 suggests ly6 proteins general evolved multitude roles in <br />                                           regulating basic properties nervous system. It notable fundamental plastic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           processes perturbed loss-of-function mutants qvr/sss lynx1. For example, in <br />                                           sssP2 hypomorphs sleep homeostasis impaired [6], lynx1 knockouts critical <br />                                           period ocular dominance plasticity extended adulthood [17]. Although phenotypes <br />                                           described additional ly6 mutants flies mice, case of <br />                                           SSS, lynx1 lynx2 molecular effectors identified vivo, K <br />                                           channels nAChRs. Since 20 ly6 genes described Drosophila [14], and <br />                                           appear homologous genes mammals, additional roles these <br />                                           molecules remain discovered. <br />  <br />                                           In summary, identified role SSS regulating synaptic transmission, in <br />                                           addition established role regulating membrane excitability. Both functions appear to <br />                                           important ability SSS regulate sleep Drosophila. The ability lynx1 to <br />                                           substitute SSS form complexes effectors SSS, Sh-type K <br />                                           channels nAChRs, suggests lynx1 mammalian ly6 proteins <br />                                           possess similar, multifunctional roles controlling neuronal activity. Potassium channels, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           nAChRs modulators key regulators targets molecular intervention in <br />                                           various human disorders, including ataxias, congenital deafness, epilepsy, cardiac <br />                                           arrhythmias, type II diabetes, autoimmune diseases multiple sclerosis and <br />                                           rheumatoid arthritis, cognitive decline Alzheimer&apos;s Disease, loss motor coordination in <br />                                           Parkinson&apos;s Disease, nicotine addiction associated risk developing cancer and <br />                                           cardiovascular disease smoking [33&#226;&#8364;&#8220;39]. Understanding functions ly6 proteins <br />                                           provide insights disorders new screening strategies more <br />                                           selective efficacious pharmacotherapeutic regulators neuronal function. <br />  <br />  <br />                               Experimental Procedures <br />                               Fly stocks transgenic fly lines <br />                                           sssP1, sss-Gal4 UAS-sss flies described previously [6, 7]. Other lines were <br />                                           provided follows: Shmns (Chiara Cirelli), UAS-shits (Toshi Kitamoto), UAS-TRPA1 (Paul <br />                                           Garrity). UAS-D&#206;&#177;3 (stock d03389) Exelixis collection Harvard. Other fly <br />                                           lines obtained Bloomington Stock Center (UAS-dicer [24650]; UAS- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           tdTomato [32221]; UAS-nAChR RNAi&apos;s [28688, 27493, 27671, 31985, 25943, 25835, <br />                                           27251, 31883, 28038 25927]). UAS-lynx1 flies generated targeting UAS-lynx1 <br />                                           pUAST-attB attP site y1,w67c23;;attP2 flies (Rainbow Transgenics, Camarillo, <br />                                           CA) outcrossing transgenic animals w1118 iso31 background 2 generations. <br />  <br />                               Behavior assays <br />                                           1&#226;&#8364;&#8220;5 day old flies loaded glass tubes containing 5% sucrose 2% agarose and <br />                                           entrained 12hr:12hr light:dark (LD) cycle 2 days measuring sleep/wake <br />                                           patterns using Drosophila Activity Monitoring System (Trikinetics). Sleep defined <br />                                           5 minutes inactivity measured previously described [22]. Experiments were <br />  <br />  <br />  <br />  <br />                                                 Curr Biol. Author manuscript; available PMC 2015 March 17. <br />                             Wu et al.                                                                                            Page 10 <br />  <br />  <br />                                          carried 25&#194;&#176;C involving 6 hour heat pulses 27&#194;&#176;C activate TRPA1 or <br />                                          28&#194;&#176;C activate shits case baseline temperature maintained 22&#194;&#176;C. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Prior drug treatment, sucrose/agarose mixtures equilibrated 50&#194;&#176;C the <br />                                          appropriate amounts concentrated 4-aminopyridine (pH adjusted 7.0, Acros Organics) <br />                                          mecamylamine (R&amp;D, Tocris) added. Animals entrained 2 days on <br />                                          sucrose/agarose switched activity tubes containing drug mixture ZT0, <br />                                          sleep measured beginning day later period 24 hours. Animals died <br />                                          4 days drug intake excluded analysis. <br />  <br />                               Quantitative PCR <br />                                          For sample, 30&#226;&#8364;&#8220;50 brains 7&#226;&#8364;&#8220;12 heads 5&#226;&#8364;&#8220;9 day old flies lysed Trizol <br />                                          (Life Technologies), strand cDNA synthesized extracted RNA using High <br />                                          Capacity cDNA Reverse Transcription (ABI). Quantitative PCR performed each <br />                                          cDNA sample using primers listed Table 1 (Supplemental Experimental Procedures), <br />                                          results normalized levels measured RP49. Relative expression further <br />                                          normalized brain D&#206;&#177;1 levels (Figure S4a) levels D&#206;&#177;3 D&#206;&#178;3 transcript measured in <br />                                          w1118 controls (Figure 3). All primer pairs validated amplification efficiencies (R2) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          greater 0.98. <br />  <br />                               Molecular Biology <br />                                          To generate GFP-tagged Shaker, eGFP amplified eGFP-C3 (Clontech) PCR <br />                                          using Hind-eGFP-F eGFP-Xho-R primers (Table 2, Supplemental Experimental <br />                                          Procedures) subcloned frame 5&apos; end ShD pcDNA3 [7] using HindIII and <br />                                          XhoI. GFP-tagged Kv1.2 generated replacing ShD Kv1.2 been <br />                                          amplified PCR Xho-Kcna2-F Kcna2-Not-R2 primers (Table 2, Supplemental <br />                                          Experimental Procedures) cut XhoI-NotI. GFP-GRID2 similarly generated by <br />                                          amplifying eGFP EcoR1-eGFP-F eGFP-Kpn1-R primers (Table 2, Supplemental <br />                                          Experimental Procedures) subcloning fragment EcoRI KpnI sites of <br />                                          Flag-GRID2 (kind gift Z.Yue, Mt. Sinai). <br />  <br />                                          To generate Myc-tagged SSS, EcoRV Xho1 sites inserted sss pcDNA3 <br />                                          [7] PCR using SSS-Cmyc-S SSS-Cmyc-AS primers (Table 2, Supplemental <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Experimental Procedures). Cmyc-S Cmyc-AS oligos encoding Myc epitope (Table <br />                                          2, Supplemental Experimental Procedures) annealed subcloned newly <br />                                          engineered EcoRV Xho1 sites. To generate Myc-tagged lynx1, EST (I.M.A.G.E. ID <br />                                          5325123, Open Biosystems) encoding full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> mouse lynx1 amplified PCR <br />                                          primers EcoR1-Lynx1-F Lynx1-Not1-R (Table 2, Supplemental Experimental <br />                                          Procedures) subcloned EcoRI NotI sites pcDNA3. A fragment of <br />                                          Myc-tagged SSS amplified PCR primers Lynx1-Myc-F Lynx1- <br />                                          Myc-R (Table 2, Supplemental Experimental Procedures), resulting product was <br />                                          used primer extension using lynx1 pcDNA3 template. The product reaction <br />                                          amplified EcoR1-Lynx1-F Lynx1-Not1-R primers (Table 2, <br />                                          Supplemental Experimental Procedures) subcloned EcoR1 Not1 sites <br />                                          pcDNA3. <br />  <br />  <br />                                                  Curr Biol. Author manuscript; available PMC 2015 March 17. <br />                             Wu et al.                                                                                           Page 11 <br />  <br />  <br />                                          To generate hemagglutanin (HA)-tagged D&#206;&#177;3, subcloned oligos Not1-HA-Xba1-S <br />                                          Not1-HA-Xba1-AS (Table 2, Supplemental Experimental Procedures) NotI <br />                                          XbaI sites pcDNA3 make pcDNA3-HA. To fuse D&#206;&#177;3 HA epitope, D&#206;&#177;3 was <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          amplified PCR EST (DGRC clone number 12624) using primers D&#206;&#177;3-Kpn1-F <br />                                          D&#206;&#177;3-Not-R (Table 2, Supplemental Experimental Procedures) subcloned the <br />                                          KpnI NotI sites pcDNA3-HA. <br />  <br />                                          UAS-lynx1 generated subcloning previously described full-length mouse EST <br />                                          pUAST-attB EcoRI NotI sites. <br />  <br />                               Immunohistochemistry <br />                                          3&#226;&#8364;&#8220;7 day old female brains dissected ice cold PBS fixation 4% <br />                                          paraformaldehyde. Brains blocked PBST (PBS, 0.3% Triton X-100) containing 5% <br />                                          normal donkey serum (Jackson Laboratory) 5% normal goat serum (Life Technologies). <br />                                          Brains incubated 1:200 rat anti-Sh [40] pre-cleared ShDf lysate followed <br />                                          sequentially incubation 1:250 rabbit anti-D&#206;&#177;3 bleed 88 [26]. Brains washed <br />                                          PBST stained 1:1000 Alexa 633 anti-rabbit (Life Technologies) 1:1000 <br />                                          anti-rat Alexa 488 (Jackson ImmunoResearch). After additional washes, brains were <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          equilibrated mounted Vectashield (Vector Labs). Images taken 40&#195;&#8212; <br />                                          magnification Leica SP5 confocal microscope using 0.5&#206;&#188;m stack intervals. 5&#206;&#188;m Z- <br />                                          projection images generated, rotated brightness/contrast adjustments made <br />                                          entirety images using Fiji [41]. <br />  <br />                               Cell Culture Co-immunoprecipitations <br />                                          Cos-7 HEK-tsa cells maintained 25&#194;&#176;C 7.5% CO2 culture medium <br />                                          consisting 10% fetal bovine serum (Omega), 1% penicillin/streptomycin (Mediatech) and <br />                                          1% L-glutamine (Sigma) low glucose DMEM 2 mM L-Glutamine (Mediatech). <br />                                          Cells grown 50&#226;&#8364;&#8220;80% confluence transfection X-tremegene HP reagent <br />                                          (Roche) 2:1 ratio transfection reagent DNA Optimem (Life Technologies). <br />                                          Transfection mixture removed 24 hours transfection replaced normal <br />                                          growth medium. Cells lysed 48 hours transfection SDS lysis/IP buffer (10 mM <br />                                          Tris [pH 7.5], 100 mM NaCl, 5mM EDTA, 1% Triton X-100, 0.05% SDS) with <br />                                          Complete protease inhibitors (Roche). For co-immunoprecipitation assays, 500 &#206;&#188;g total <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          cell lysate incubated total volume 1 ml SDS lysis/IP buffer primary <br />                                          antibodies overnight 4&#194;&#176;C rotating platform. The following day, protein-G conjugated <br />                                          magnetic beads (NEB) added incubated 4&#194;&#176;C 2 &#226;&#8364;&#8220; 5 hours. Beads washed <br />                                          3 times 1 ml volume IP wash buffer (10 mM Tris [pH 7.5], 100 mM NaCl, 5 mM <br />                                          EDTA, 0.5% Triton X-100, 0.05% SDS) resuspended 2X LDS sample buffer <br />                                          (Life Technologies). Antibodies used rabbit anti-GFP (Life Technologies) rabbit <br />                                          anti-HA (Rockland). <br />  <br />                               Western Blot Analyses <br />                                          Western blot analyses Cos-7 cell lysates co-immunoprecipitation complexes were <br />                                          performed previously described [40]. For 20% input conditions, 100 &#206;&#188;g protein was <br />                                          loaded lane. Proteins detected using following antibodies: rabbit anti-GFP (Life <br />  <br />  <br />                                                Curr Biol. Author manuscript; available PMC 2015 March 17. <br />                             Wu et al.                                                                                                                 Page 12 <br />  <br />  <br />                                          Technologies), mouse anti-HA (Covance), mouse anti-Myc (Santa Cruz Biotechnologies), <br />                                          mouse anti-actin (Millipore EMD). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               FRET-based measurements nAChR activity <br />                                          Measurements &#206;&#177;4&#206;&#178;2 nAChR activity performed previously described [29] with <br />                                          following alterations: HEK-tsa cells transiently transfected &#206;&#177;4-pciNeo &#206;&#178;2- <br />                                          dm-pciNeo (gifts H. Lester) plus TN-XXL (gift P. Taylor) supplemented with <br />                                          sss-pcDNA3 pcDNA3 ratio 1:1:2:10, respectively. Cells pre- <br />                                          incubated growth media containing 1 &#206;&#188;M nicotine (Tocris, R&amp;D) 20 hours prior to <br />                                          assay artificial cerebral spinal fluid (ACSF: 121 mM NaCl, 5 mM KCl, 26 mM NaHCO3, <br />                                          1.2 mM NaH2PO4H2O, 10 mM Glucose, 5 mM HEPES, 2.4 mM Ca2+, 1.3 mM Mg2+, pH <br />                                          7.4). Maximum responses calculated concentration-response curves averaged <br />                                          4 experiments, performed triplicate, results normalized maximum <br />                                          response &#206;&#177;4&#206;&#178;2 nAChR sss. One-way ANOVA repeated measures analysis with <br />                                          Dunnett&apos;s multiple comparison post-test used statistical analysis. <br />  <br />  <br />                               Supplementary Material <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                               Acknowledgments <br />                                          We thank H. Lester mouse &#206;&#177;4 &#206;&#178;2 nAChR cDNAs, P. Taylor TN-XXL, Z. Yue Flag-GRID2 <br />                                          expression construct, U. Thomas E. Gundelfinger D&#206;&#177;3 antibody, A. Sehgal Sh antibody. We <br />                                          thank A. Sehgal M. Shi sharing data prior publication. We grateful technical assistance <br />                                          provided J. Yamauchi K. Gomez FRET assays B. Yip qPCR. We thank members the <br />                                          Joiner lab helpful discussions. This work supported grants NIH (NS072431), Whitehall <br />                                          Foundation (WF20110560), Pharma Foundation Research Starter Grant WJJ. This work also <br />                                          supported UCSD Neuroscience Microscopy Shared Facility Grant P30 NS047101. JER supported by <br />                                          funding National Science Foundation Graduate Research Fellowship Program (DGE-1144086) the <br />                                          UCSD Graduate Training Program Cellular Molecular Pharmacology (T32 GM007752). <br />  <br />  <br />                               References <br />                                          1. Agosto J, Choi JC, Parisky KM, Stilwell G, Rosbash M, Griffith LC. Modulation GABAA <br />                                             receptor desensitization uncouples sleep onset maintenance Drosophila. Nat Neurosci. 2008; <br />                                             11:354&#226;&#8364;&#8220;359. [PubMed: 18223647] <br />                                          2. Douglas CL, Vyazovskiy V, Southard T, Chiu S-Y, Messing A, Tononi G, Cirelli C. Sleep Kcna2 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             knockout mice. BMC Biol. 2007; 5:42. [PubMed: 17925011] <br />                                          3. Lancel M, Steiger A. Sleep Its Modulation Drugs That Affect GABA(A) Receptor Function. <br />                                             Angew Chem Int Ed Engl. 1999; 38:2852&#226;&#8364;&#8220;2864. [PubMed: 10540374] <br />                                          4. Cirelli C, Bushey D, Hill S, Huber R, Kreber R, Ganetzky B, Tononi G. Reduced sleep in <br />                                             Drosophila Shaker mutants. Nature. 2005; 434:1087&#226;&#8364;&#8220;1092. [PubMed: 15858564] <br />                                          5. Bushey D, Huber R, Tononi G, Cirelli C. Drosophila Hyperkinetic mutants reduced sleep and <br />                                             impaired memory. J Neurosci. 2007; 27:5384&#226;&#8364;&#8220;5393. [PubMed: 17507560] <br />                                          6. Koh K, Joiner WJ, Wu MN, Yue Z, Smith CJ, Sehgal A. Identification SLEEPLESS, sleep- <br />                                             promoting factor. Science. 2008; 321:372&#226;&#8364;&#8220;376. [PubMed: 18635795] <br />                                          7. Wu MN, Joiner WJ, Dean T, Yue Z, Smith CJ, Chen D, Hoshi T, Sehgal A, Koh K. SLEEPLESS, a <br />                                             Ly-6/neurotoxin family member, regulates levels, localization activity Shaker. Nat <br />                                             Neurosci. 2010; 13:69&#226;&#8364;&#8220;75. [PubMed: 20010822] <br />  <br />  <br />  <br />  <br />                                                 Curr Biol. Author manuscript; available PMC 2015 March 17. <br />                             Wu et al.                                                                                                 Page 13 <br />  <br />  <br />                                        8. Dean T, Xu R, Joiner W, Sehgal A, Hoshi T. Drosophila QVR/SSS Modulates Activation C- <br />                                           Type Inactivation Kinetics Shaker K+ Channels. J Neurosci. 2011; 31:11387&#226;&#8364;&#8220;11395. [PubMed: <br />                                           21813698] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        9. Yasuda O, Morimoto S, Jiang B, Kuroda H, Kimura T, Sakakibara S, Fukuo K, Chen S, Tamatani <br />                                           M, Ogihara T. FS2. mamba venom toxin, specific blocker L-type calcium channels. <br />                                           Artery. 1994; 21:287&#226;&#8364;&#8220;302. [PubMed: 8826098] <br />                                        10. Dajas F, Bolioli B, Castello ME, Silveira R. Rat striatal acetylcholinesterase inhibition by <br />                                            fasciculin (a polypeptide green mamba snake venom). Neurosci Lett. 1987; 77:87&#226;&#8364;&#8220;91. <br />                                            [PubMed: 3601220] <br />                                        11. Toomela T, Jolkkonen M, Rinken A, J&#195;&#164;rv J, Karlsson E. Two-step binding green mamba toxin <br />                                            muscarinic acetylcholine receptor. FEBS Lett. 1994; 352:95&#226;&#8364;&#8220;97. [PubMed: 7925953] <br />                                        12. Changeux JP, Kasai M, Lee CY. Use snake venom toxin characterize cholinergic <br />                                            receptor protein. Proc Natl Acad Sci USA. 1970; 67:1241&#226;&#8364;&#8220;1247. [PubMed: 5274453] <br />                                        13. Galat A. Common structural traits cystine knot domain TGF&#206;&#178; superfamily proteins <br />                                            three-fingered ectodomain cellular receptors. Cell. Mol. Life Sci. 2011; 68:3437&#226;&#8364;&#8220; <br />                                            3451. [PubMed: 21369710] <br />                                        14. Hijazi A, Masson W, Aug&#195;&#169; B, Waltzer L, Haenlin M, Roch F. boudin required septate <br />                                            junction organisation Drosophila codes diffusible protein Ly6 superfamily. <br />                                            Development. 2009; 136:2199&#226;&#8364;&#8220;2209. [PubMed: 19502482] <br />                                        15. Iba&#195;&#177;ez-Tallon I, Miwa JM, Wang HL, Adams NC, Crabtree GW, Sine SM, Heintz N. Novel <br />                                            modulation neuronal nicotinic acetylcholine receptors association endogenous <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            prototoxin lynx1. Neuron. 2002; 33:893&#226;&#8364;&#8220;903. [PubMed: 11906696] <br />                                        16. Miwa JM, Ibanez-Tallon I, Crabtree GW, S&#195;&#161;nchez R, Sali A, Role LW, Heintz N. lynx1, an <br />                                            endogenous toxin-like modulator nicotinic acetylcholine receptors mammalian CNS. <br />                                            Neuron. 1999; 23:105&#226;&#8364;&#8220;114. [PubMed: 10402197] <br />                                        17. Morishita H, Miwa JM, Heintz N, Hensch TK. Lynx1, cholinergic brake, limits plasticity adult <br />                                            visual cortex. Science. 2010; 330:1238&#226;&#8364;&#8220;1240. [PubMed: 21071629] <br />                                        18. Tekinay AB, Nong Y, Miwa JM, Lieberam I, Ibanez-Tallon I, Greengard P, Heintz N. A role for <br />                                            LYNX2 anxiety-related behavior. Proc Natl Acad Sci USA. 2009; 106:4477&#226;&#8364;&#8220;4482. [PubMed: <br />                                            19246390] <br />                                        19. Brand AH, Perrimon N. Targeted gene expression means altering cell fates generating <br />                                            dominant phenotypes. Development. 1993; 118:401&#226;&#8364;&#8220;415. [PubMed: 8223268] <br />                                        20. Rosenzweig M, Brennan KM, Tayler TD, Phelps PO, Patapoutian A, Garrity PA. The Drosophila <br />                                            ortholog vertebrate TRPA1 regulates thermotaxis. Genes &amp; Development. 2005; 19:419&#226;&#8364;&#8220;424. <br />                                            [PubMed: 15681611] <br />                                        21. Kitamoto T. Conditional modification behavior Drosophila targeted expression a <br />                                            temperature-sensitive shibire allele defined neurons. J Neurobiol. 2001; 47:81&#226;&#8364;&#8220;92. [PubMed: <br />                                            11291099] <br />                                        22. Shaw PJ, Cirelli C, Greenspan RJ, Tononi G. Correlates sleep waking Drosophila <br />                                            melanogaster. Science. 2000; 287:1834&#226;&#8364;&#8220;1837. [PubMed: 10710313] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        23. Hendricks JC, Lu S, Kume K, Yin JC, Yang Z, Sehgal A. Gender dimorphism role cycle <br />                                            (BMAL1) rest, rest regulation, longevity Drosophila melanogaster. J Biol Rhythms. <br />                                            2003; 18:12&#226;&#8364;&#8220;25. [PubMed: 12568241] <br />                                        24. Kumar S, Chen D, Sehgal A. Dopamine acts Cryptochrome promote acute arousal in <br />                                            Drosophila. Genes &amp; Development. 2012; 26:1224&#226;&#8364;&#8220;1234. [PubMed: 22581798] <br />                                        25. Lansdell SJ, Collins T, Goodchild J, Millar NS. The Drosophila nicotinic acetylcholine receptor <br />                                            subunits Dalpha5 Dalpha7 form functional homomeric heteromeric ion channels. BMC <br />                                            neuroscience. 2012; 13:73. [PubMed: 22727315] <br />                                        26. Chamaon K, Smalla K-H, Thomas U, Gundelfinger ED. Nicotinic acetylcholine receptors of <br />                                            Drosophila: subunits encoded genomically linked genes co-assemble same <br />                                            receptor complex. J Neurochem. 2002; 80:149&#226;&#8364;&#8220;157. [PubMed: 11796753] <br />                                        27. Joiner WJ, Crocker A, White BH, Sehgal A. Sleep Drosophila regulated adult mushroom <br />                                            bodies. Nature. 2006; 441:757&#226;&#8364;&#8220;760. [PubMed: 16760980] <br />  <br />  <br />  <br />                                              Curr Biol. Author manuscript; available PMC 2015 March 17. <br />                             Wu et al.                                                                                                  Page 14 <br />  <br />  <br />                                        28. Pitman JL, McGill JJ, Keegan KP, Allada R. A dynamic role mushroom bodies in <br />                                            promoting sleep Drosophila. Nature. 2006; 441:753&#226;&#8364;&#8220;756. [PubMed: 16760979] <br />                                        29. Yamauchi JG, Nemecz &#195;?, Nguyen QT, Muller A, Schroeder LF, Talley TT, Lindstrom J, Kleinfeld <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            D, Taylor P. Characterizing ligand-gated ion channel receptors genetically encoded Ca2++ <br />                                            sensors. PLoS ONE. 2011; 6:e16519. [PubMed: 21305050] <br />                                        30. Bourne Y, Talley TT, Hansen SB, Taylor P, Marchot P. Crystal structure Cbtx-AChBP <br />                                            complex reveals essential interactions snake alpha-neurotoxins nicotinic receptors. <br />                                            EMBO J. 2005; 24:1512&#226;&#8364;&#8220;1522. [PubMed: 15791209] <br />                                        31. Lyukmanova EN, Shenkarev ZO, Shulepko MA, Mineev KS, D&apos;Hoedt D, Kasheverov IE, Filkin <br />                                            SY, Krivolapova AP, Janickova H, Dolezal V, et al. NMR STRUCTURE AND ACTION ON <br />                                            NICOTINIC ACETYLCHOLINE RECEPTORS OF WATER-SOLUBLE DOMAIN OF <br />                                            HUMAN LYNX1. J Biol Chem. 2011 <br />                                        32. Yuan Q, Joiner WJ, Sehgal A. A sleep-promoting role Drosophila serotonin receptor 1A. <br />                                            Curr Biol. 2006; 16:1051&#226;&#8364;&#8220;1062. [PubMed: 16753559] <br />                                        33. Kass RS. The channelopathies: novel insights molecular genetic mechanisms human <br />                                            disease. Journal Clinical Investigation. 2005; 115:1986&#226;&#8364;&#8220;1989. [PubMed: 16075038] <br />                                        34. Ashcroft FM. <span id='am-1' about='protege:FROM' typeof='owl:Thing'>From</span> molecule malady. Nature. 2006; 440:440&#226;&#8364;&#8220;447. [PubMed: 16554803] <br />                                        35. Robbins CA, Tempel BL. Kv1.1 Kv1.2: similar channels, different seizure models. Epilepsia. <br />                                            2012; 53(Suppl 1):134&#226;&#8364;&#8220;141. [PubMed: 22612818] <br />                                        36. Lam J, Wulff H. The Lymphocyte Potassium Channels Kv1.3 KCa3.1 Targets for <br />                                            Immunosuppression. Drug Dev. Res. 2011; 72:573&#226;&#8364;&#8220;584. [PubMed: 22241939] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        37. Quik M, Wonnacott S. &#206;&#177;6&#206;&#178;2* &#206;&#177;4&#206;&#178;2* nicotinic acetylcholine receptors drug targets for <br />                                            Parkinson&apos;s disease. Pharmacological Reviews. 2011; 63:938&#226;&#8364;&#8220;966. [PubMed: 21969327] <br />                                        38. Picciotto MR, Kenny PJ. Molecular mechanisms underlying behaviors related nicotine <br />                                            addiction. Cold Spring Harbor Perspectives Medicine. 2013; 3:a012112. [PubMed: 23143843] <br />                                        39. Picciotto MR, Zoli M. Neuroprotection nAChRs: role nAChRs neurodegenerative <br />                                            disorders Alzheimer&apos;s Parkinson&apos;s disease. Front Biosci. 2008; 13:492&#226;&#8364;&#8220;504. [PubMed: <br />                                            17981563] <br />                                        40. Wu M, Li J, Engleka KA, Zhou B, Lu MM, Plotkin JB, Epstein JA. Persistent expression Pax3 <br />                                            neural crest causes cleft palate defective osteogenesis mice. J. Clin. Invest. 2008; <br />                                            118:2076&#226;&#8364;&#8220;2087. [PubMed: 18483623] <br />                                        41. Schindelin J, Arganda-Carreras I, Frise E, Kaynig V, Longair M, Pietzsch T, Preibisch S, Rueden <br />                                            C, Saalfeld S, Schmid B, et al. Fiji: open-source platform biological-image analysis. Nat <br />                                            Meth. 2012; 9:676&#226;&#8364;&#8220;682. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Curr Biol. Author manuscript; available PMC 2015 March 17. <br />                             Wu et al.                                                                                   Page 15 <br />  <br />  <br />  <br />                                        Highlights <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        &#226;&#8364;&#162;      Reduction nAChR activity restores normal sleep qvr/sss mutants. <br />  <br />                                        &#226;&#8364;&#162;      SSS complexes certain nAChRs K channels, are <br />                                               expressed mushroom bodies. <br />  <br />                                        &#226;&#8364;&#162;      SSS inhibits nAChR activity vitro. <br />  <br />                                        &#226;&#8364;&#162;      The mammalian SSS homolog lynx1 substitute SSS regulating sleep. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Curr Biol. Author manuscript; available PMC 2015 March 17. <br />                             Wu et al.                                                                                          Page 16 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Figure 1. <br />                                        sss neurons promote waking sensitive pharmacological inhibition nAChRs and <br />                                        K channels. (a) sss neurons sufficient waking. sss-Gal4&gt;UAS-TRPA1, UAS-TRPA1&gt; <br />                                        + sss-Gal4&gt;+ animals heat pulsed 27&#194;&#176;C ZT18-24 baseline <br />                                        temperature 22&#194;&#176;C. Change sleep calculated sleep heat pulse minus <br />                                        sleep equivalent time period 24 hrs before. N &gt; 13 animals group. (b) sss <br />                                        neurons necessary waking. sss-Gal4&gt;UAS-shits, UAS-shits&gt;+ sss-Gal4&gt;+ <br />                                        animals heat pulsed 28&#194;&#176;C ZT0-6 baseline temperature 22&#194;&#176;C. Change <br />                                        sleep calculated 12 hrs heat-dependent shift timing of <br />                                        daytime sleep. N &gt; 15 animals group. (c) The nAChR antagonist mecamylamine <br />                                        (MCA) dose-dependently restores sleep qvr/sss Sh mutants little effect on <br />                                        wild-type (WT) control animals (w1118, genetic background sssP1 Shmns mutants). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        N = 19 - 40 animals group. (d) 24 hr sleep profiles wild-type control (top), sssP1 <br />                                        (middle), Shmns (bottom) animals used collect data c. N = 19 &#226;&#8364;&#8220; 40 animals for <br />                                        plot. (e) The potassium channel antagonist 4-aminopyridine (4-AP) reduces sleep in <br />                                        wild-type controls effect sssP1/sssP2 Shmns mutants. N = 10 &#226;&#8364;&#8220; 69 animals <br />                                        group. (f) 24 hr sleep profiles wild-type control (top), sssP1/sssP2 (middle), and <br />                                        Shmns (bottom) animals used collect data e. N = 10 &#226;&#8364;&#8220; 69 animals plot. * <br />                                        p&lt; .01, *** p &lt; 0.001 unpaired t-test (a,b) one-way ANOVA (c,e) Bonferroni <br />                                        post-test. ns, significant. Error bars indicate s.e.m. See Figure S1. <br />  <br />  <br />  <br />  <br />                                              Curr Biol. Author manuscript; available PMC 2015 March 17. <br />                             Wu et al.                                                                                        Page 17 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Figure 2. <br />                                        RNAi knockdown nAChR subunits restores sleep qvr/sss mutants. (a) Sleep sss- <br />                                        Gal4/+;UAS-RNAi/UAS-dicer animals compared UAS-RNAi/+ controls (all sssP1 <br />                                        background). The low sleeping phenotype sssP1 mutants rescued knockdown of <br />                                        D&#206;&#177;3 lesser extent D&#206;&#178;3. N &gt; 26 group. (b) 24 hr sleep profiles sssP1 <br />                                        sssP1 animals D&#206;&#177;3 knocked sss neurons. (c) <br />                                        Overexpression D&#206;&#177;3 sss neurons reduces total daily sleep. N &gt; 15 group. *** <br />                                        p&lt;0.001 one-way ANOVA Bonferroni post-test. Error bars indicate s.e.m. See also <br />                                        Figure S2. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Curr Biol. Author manuscript; available PMC 2015 March 17. <br />                             Wu et al.                                                                                         Page 18 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Figure 3. <br />                                        D&#206;&#177;3 D&#206;&#178;3 transcriptionally regulated SSS Sh co-express in <br />                                        mushroom bodies. (a) Quantitative PCR analyses D&#206;&#177;3 D&#206;&#178;3 transcripts no <br />                                        dependence qvr/sss Sh normal expression. N = 3 group. ns, significant <br />                                        one-way ANOVA Bonferroni post-test. Error bars indicate s.e.m. (b) Coexpression <br />                                        sss, Sh D&#206;&#177;3 mushroom bodies. Representative immunostaining (N = 8) of <br />                                        tdTomato expressed control sss-Gal4 (upper panel), native Sh protein (center <br />                                        panel), native D&#206;&#177;3 protein (lower panel). Scale bar = 25&#206;&#188;m. (c) MB-Gal80 suppresses <br />                                        rescue sleep sss-Gal4&gt;UAS-sss sssP1 mutants. N &gt; 30 groups. *** p&lt;0.001 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        one-way ANOVA Bonferroni post-test. Error bars indicate s.e.m. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Curr Biol. Author manuscript; available PMC 2015 March 17. <br />                             Wu et al.                                                                                        Page 19 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Figure 4. <br />                                        SSS forms complexes nAChRs inhibits activity. Representative <br />                                        immunoprecipitations (a) GFP-tagged Sh (b) HA-tagged D&#206;&#177;3 transiently <br />                                        transfected Cos-7 cells. Co-transfected Myc-tagged SSS co-immunoprecipitates Sh and <br />                                        D&#206;&#177;3 (lane 3, blots) absence Sh (lane 1, panel a) D&#206;&#177;3 (b, lane 3). <br />                                        IP, immunoprecipitations. WB, western blots. <br /> </body></html>