<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>A Loss Function Screen Identified Genome-Wide <br /> Association Study Loci Reveals New Genes Controlling <br /> Hematopoiesis <br /> Ewa Bielczyk-Maczyn&#194;&#180;ska1,2,3, Jovana Serbanovic-Canic4,5, Lauren Ferreira1,2, Nicole Soranzo1,2, <br /> Derek L. Stemple2, Willem H. Ouwehand1,2,3, Ana Cvejic1,2* <br /> 1 Department Haematology, University Cambridge, Cambridge, United Kingdom, 2 Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Hinxton, <br /> Cambridge, United Kingdom, 3 NHS Blood Transplant, Cambridge, United Kingdom, 4 MRC Centre Developmental Biomedical Genetics, University Sheffield, <br /> Sheffield, United Kingdom, 5 Department Cardiovascular Sciences, University Sheffield, Sheffield, United Kingdom <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      The formation mature cells blood stem cells understood cellular level know key <br />      transcription factors control fate decisions. However, upstream signalling downstream effector processes are <br />      partially understood. Genome wide association studies (GWAS) particularly useful providing new <br />      directions dissect pathways. A GWAS meta-analysis identified 68 genetic loci controlling platelet size number. <br />      Only quarter genes, however, known regulators hematopoiesis. <span id='am-5' about='protege:TO' typeof='owl:Thing'>To</span> determine function remaining <br />      genes performed medium-throughput genetic screen zebrafish using antisense morpholino oligonucleotides (MOs) <br />      knock protein expression, followed histological analysis selected genes using wide panel different <br />      hematopoietic markers. The information generated initial knockdown used profile phenotypes to <br />      position candidate genes hierarchically hematopoiesis. Further analysis brd3a revealed essential role in <br />      differentiation maintenance <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>survival</span> thrombocytes. Using <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span>-GWAS-to-function strategy not <br />      identified series genes represent novel regulators thrombopoiesis hematopoiesis, work also <br />      represents, knowledge, example functional genetic screening strategy critical step toward <br />      obtaining biologically relevant functional data GWA study blood cell traits. <br />  <br />   Citation: Bielczyk-Maczyn&#194;&#180;ska E, Serbanovic-Canic J, Ferreira L, Soranzo N, Stemple DL, et al. (2014) A Loss Function Screen Identified Genome-Wide <br />   Association Study Loci Reveals New Genes Controlling Hematopoiesis. PLoS Genet 10(7): e1004450. doi:10.1371/journal.pgen.1004450 <br />   Editor: Leonard I. Zon, Children&#226;&#8364;&#8482;s Hospital, United States America <br />   Received December 26, 2013; Accepted May 2, 2014; Published July 10, 2014 <br />   Copyright: &#195;&#376; 2014 Bielczyk-Maczyn&#194;&#180;ska et al. This open-access article distributed terms Creative Commons Attribution License, which <br />   permits unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: The study supported Cancer Research UK (grant number C45041/A14953 AC LF), EHA-Jose&#194;&#180; Carreras Foundation Young Investigator <br />   Award (number DJCLS EHA 2012 F 12/02 AC), European Union NetSim training fellowship scheme (number 215820 JSC), Wellcome Trust grants (number <br />   084183/Z/07/Z EBM, number WT098051 NS, number WT091310 WHO number 098051 DLS), EU FP7 (EPIGENESYS Grant <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>Code</span> 257082 and <br />   BLUEPRINT Grant <span id='am-318' about='Thesaurus:code' typeof='owl:Thing'>Code</span> HEALTH-F5-2011-282510 NS), National Institute Health Research (grant number RP-PG-0310-1002 WHO) British Heart <br />   Fundation (grant number RG/09/12/28096 WHO). For providing zebrafish <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span>knockout</span> allele sa8062 thank Sanger Institute Zebrafish <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>Mutation</span> Project, <br />   sponsored Wellcome Trust (grant number 098051). The funders role study design, data collection analysis, decision publish, or <br />   preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * Email: as889@cam.ac.uk <br />  <br />  <br />  <br />  <br /> Introduction                                                                        silenced ak3, rnf145, arhgef3, tpm1, jmjd1c ehd3 zebrafish <br />                                                                                     MO injections. Profound effects thrombopoiesis were <br />    Erythrocytes platelets (thrombocytes zebrafish)                   observed ehd3 [3]. Furthermore, detailed studies of <br /> most abundant cells blood. In individual, number                      arhgef3 gene, encodes ,70 Rho guanine <br /> volume erythrocytes platelets highly heritable                  nucleotide exchange factors, showed important role iron <br /> tightly regulated narrow ranges, wide                         uptake transferrin receptor internalisation erythrocytes [5]. <br /> variation parameters population [1,2]. The 80%                      Based preliminary data, hypothesized the <br /> heritability blood cell indices provided foundation                  majority genes identified recent genomics efforts are <br /> recently completed GWAS meta-analysis ,68,000 healthy                            important rate-limiting regulators hematopoiesis and <br /> individuals cell types. We identified 68 genetic loci                 worthwhile investigation. <br /> control mass (volume x count) platelets [3] 75                      The zebrafish model distinct advantages animal <br /> for red cell indices [4]. About quarter genes proximal                  models screening large numbers genes. Zebrafish develop- <br /> the platelet GWAS association single nucleotide polymorphisms                       ment occurs rapidly course days with <br /> (SNPs) encode studied generally pivotal regulators                      thrombocytes, erythroid- myeloid- blood cells fully <br /> hematopoiesis function remaining ones unknown,                    formed functional 3 days post fertilisation (dpf). External <br /> demonstrating power GWAS identify novel regulators                     fertilisation transparency zebrafish embryos allow easy <br /> hematopoiesis. We recently reported functional validation                   visualisation early blood-related phenotypes giving the <br /> six genes sentinel SNP localized gene                     advantage mice, development occurs utero. <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                            1                              July 2014 | Volume 10 | Issue 7 | e1004450 <br />                                                                                                             Functional Genomic Screen Zebrafish <br />  <br />  <br />  <br />   Author Summary                                                             .10 kb. Five selected genes duplicated zebrafish, <br />                                                                              resulting total 24 genes investigated aiming to <br />   In manuscript report follow-up study                   define function blood cell formation MO knockdown <br />   GWAS loci associated platelet size number. A                  approach zebrafish. <br />   GWAS meta-analysis identified 68 genetic loci controlling                     We designed splice-blocking MOs gene validated <br />   platelet size number. Only quarter genes,                   efficacy RT-PCR sequencing. Of 24 MOs <br />   however, known regulators hematopoiesis. To                         tested MOs effect target RNA MOs <br />   determine function remaining genes performed                     excluded analysis (Figure S2). We assayed <br />   medium-throughput genetic screen zebrafish using                      remaining 19 functional MOs effect overall <br />   morpholinos (MOs) knock selected candidate                         development, morphology hematopoiesis first <br />   genes. Here, report major findings. First                     72 hours post fertilisation (hpf) selected optimal dose of <br />   identified 15 genes (corresponding 12 human genes)                      MO injected (Figure S3). Of note, based information <br />   required distinct stages specification differentia- <br />                                                                              available ZFIN time situ hybridization data <br />   tion HSCs zebrafish. A detailed review databases <br />   literature revealed limited knowledge                        (Figure S4) selected genes hematopoietic specific <br />   functional role Satb1, Rcor1 Brd3 hematopoiesis                  gene expression. For genes, grtp1a, optimal dose of <br />   remaining genes work represents                   MO selected resulted specific <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span>phenotype</span> without <br />   study putative role hematopoiesis. And                   gross lethality defects body shape size, vasculature, heart <br />   secondly, demonstrate brd3a critical                        circulation. We grtp1a MO injected embryos died <br />   establishing, maintaining thrombopoietic com-                      15 hpf injected 0.8 ng MO, we <br />   partment. Importantly, study introduces zebrafish                 excluded grtp1a analysis. <br />   model functional follow-up GWAS loci and <br />   generates valuable resource prioritization platelet               A reverse genetic screen identified genes essential in <br />   size number associated genes future in-depth                       hematopoiesis <br />   mechanistic analyses. Following route investigation                   Although morphological examination detect defects with <br />   new regulatory molecules hematopoiesis added <br />                                                                              great sensitivity, specific defects hematopoiesis be <br />   critical pathways. <br />                                                                              missed, result information obtained initial <br />                                                                              analysis limited. Thus, carried second level of <br /> Importantly, transcriptional mechanisms signalling pathways              analysis performing situ hybridisation hemato- <br /> in hematopoiesis conserved zebrafish                    poietic markers, specifically, c-myb, ae1 globin, mpeg rag1. These <br /> mammals [6].                                                                 complemented use Tg(cd41:EGFP) line and <br />    Herein, performed MO injection screen 15 genes                    histochemical stains, o-Dianisidine Sudan black. The <br /> identified GWAS platelet size number uncover novel             hematopoietic markers used phenotyping carefully chosen <br /> pathways essential thrombopoiesis hematopoiesis                    distinguish early late stages hematopoiesis as <br /> general. From screen, identified 12 new genes required               thrombocytes, erythrocytes, neutrophils, macrophages and <br /> for normal hematopoiesis ordered hematopoietic                 lymphocytes (Figure S5). <br /> lineage tree based presumed function hematopoi-                 We initiated screening taking advantage the <br /> esis. Further analysis hematopoietic lineage tree revealed          Tg(cd41:EGFP) reporter line, labels thrombocytes, to <br /> distinct <span id='am-319' about='xsp:pattern' typeof='owl:Thing'>pattern</span> gene distribution suggesting main gene               identify genes zebrafish that, disrupted, affect thrombo- <br /> clusters. One cluster genes appears work level HSCs          cyte number. We knock 15 18 genes <br /> affecting derived blood cell types second cluster appears        resulted 30&#226;&#8364;&#8220;95% reduction thrombocyte number (Figure 1, <br /> to limited controlling specification thrombocyte-           Figure S6). Importantly, functional second MO was <br /> erythroid progenitors. Additionally, novel           available, observed phenotype comparable one <br /> candidate genes, brd3a, essential differentiation thrombo-          observed MO (Figure S7, S8, Table S2). <br /> cytes HSCs dispensable maintenance.                    Furthermore, concurrent knock p53 gene specific <br />                                                                              MOs did attenuate thrombocyte phenotype induced by <br /> Results                                                                      gene specific MOs, confirming observed decrease the <br />                                                                              number thrombocytes induced p53 mediated off <br /> Gene selection                                                               target effects MO injection (Figure S9). For the <br />    To interrogate large number novel hematopoietic genes              &#226;&#8364;&#732;&#226;&#8364;&#732;phenotypic&#226;&#8364;&#8482;&#226;&#8364;&#8482; genes, brf1b, obtained mutant through <br /> identified GWAS platelet size number developed             ZMP. To test brf1b mutants decreased number of <br /> an vivo functional genomics screen zebrafish (Figure S1). The          thrombocytes, offspring subjected &#226;&#8364;&#732;&#226;&#8364;&#732;clotting time <br /> first step selection suitable candidate genes            assay&#226;&#8364;&#8482;&#226;&#8364;&#8482; 5 dpf [7]. Clotting time brf1b mutant larvae was <br /> initially selected single gene, closest sentinel SNP,       significantly longer wild type fish, suggesting defect in <br /> each GWAS locus [3] (Table S1). Distance nearest gene               thrombopoiesis (Figure S10 A). To confirm specificity <br /> was calculated absolute distance SNP                      phenotype observed MO injected embryos the <br /> transcription start site gene 39 end exon [3]              absence available mutants performed rescue rcor1 MO <br /> (Table S1). We eliminated genes known function             injected embryos using full-<span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> zebrafish rcor1 RNA (Figure S10 <br /> hematopoiesis identified putative zebrafish ortholog,             B, C). We focused analysis 15 &#226;&#8364;&#732;&#226;&#8364;&#732;phenotypic&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> over 38% identity, protein level, human                      genes paralogs. <br /> counterpart 40 genes. Finally, excluded genes                   To maintain population differentiated blood cells within <br /> would require use MOs, resulting list           normal ranges, HSCs need continuously maintain balance <br /> 33 genes. Nearly 80% genes sentinel SNPs localized            self-renewal differentiation. We reasoned that <br /> within 10 kb. In step, selected 19 genes 16         decreased number thrombocytes MO injected embryos <br /> a sentinel SNP 10 kb genes sentinel SNP               result reduced numbers HSCs altered <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                     2                           July 2014 | Volume 10 | Issue 7 | e1004450 <br />                                                                                                             Functional Genomic Screen Zebrafish <br />  <br />  <br />  <br />  <br /> Figure 1. In vivo morpholino screen zebrafish identifies 15 new regulators thrombopoiesis. MOs injected one-cell stage <br /> transgenic Tg(cd41:EGFP) zebrafish embryos assayed effect number thrombocytes (cd41high) 3 dpf. Representative confocal <br /> images taken CHT. For akap10, brd3a, brf1b, kalrn 1, kalrn 2, kif1b, mfn2, pdia5, psmd13 satb1 severe decrease number of <br /> cd41high positive cells observed. brf1a, rcor1, waspla, wasplb wdr66 depletion resulted mild phenotype, fen1, grtp1b tmcc2 MO <br /> injected embryos showed phenotype. All embryos oriented anterior left dorsal top. White arrow &#226;&#8364;&#8220; thrombocytes; white <br /> arrowhead &#226;&#8364;&#8220; HSCs. <br /> doi:10.1371/journal.pgen.1004450.g001 <br />  <br /> HSC differentiation. To assess stage hematopoiesis             Thus, screen effectively defined set genes required for <br /> affected MO injected embryo, performed situ                   differentiation HSCs thrombocytes possibly blood <br /> hybridization 3 dpf looked alterations expression          lineages. <br /> definitive hematopoiesis marker c-myb (Figure 2). Although                Hematopoiesis depicted hierarchical differentiation <br /> than half MOs tested effect number HSCs,            tree, HSCs root mature blood cells the <br /> depletion rcor1 resulted increased number HSCs              branches. One intermediate cellular states common <br /> depletion kalrn (1 2), mfn2, pdia5, psmd13 wasplb resulted       myeloid progenitor (CMP), proliferate differentiate <br /> in decreased numbers HSCs caudal hematopoietic tissue                 megakaryocyte-erythrocyte progenitors (MEP) granulo- <br /> (CHT) 3 dpf. This reduction number HSCs                cyte-monocyte (GM) progenitors, rise to <br /> evident 30 hpf kalrn1, mfn2, pdia5, psmd13 wasplb MO              megakaryocytes, erythrocytes, granulocytes, monocytes other <br /> injected embryos (Figure S11) suggesting number HSCs            cell types. To investigate lineage-specific effects candidate <br /> at 3 dpf probably adversely affected               genes, assessed status definitive erythropoiesis MO <br /> homing survival/proliferation CHT. However, kalrn2             injected embryos 4 dpf. As ae1-globin RNA reported be <br /> rcor1 depleted embryos marked decrease number               expressed definitive erythrocytes 4 dpf [8], detailed analysis <br /> HSCs 30 hpf, implying important role genes               expression pattern ae1-globin transcript exploited to <br /> specification HSCs aorta-gonad-mesonephros (AGM)                  reveal initiation definitive erythropoiesis gene silencing. <br /> (Figure S11). Importantly, analysis vascular development              Profound effects definitive erythropoiesis observed all <br /> injecting candidate gene MOs Tg(fli1:EGFP) embryos,              brd3a, brf1b, waspla wdr66 (Figure S13). However, silencing <br /> express EGFP endothelial cells, demonstrated major                    brf1a wasplb resulted diminished definitive erythropoiesis, <br /> abnormalities vascular morphogenesis remodeling                  reflecting functional divergence duplicated genes (brf1 waspl) <br /> would preclude circulation, indicating hematopoietic               (Figure S13). Furthermore, extensive analysis hemoglobin <br /> defects secondary vascular phenotype (Figure S12).            levels primitive erythroid cells 2 dpf showed that, the <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                    3                            July 2014 | Volume 10 | Issue 7 | e1004450 <br />                                                                                                            Functional Genomic Screen Zebrafish <br />  <br />  <br />  <br />  <br /> Figure 2. Characterization HSCs candidate gene depleted embryos. To assess stage hematopoiesis MO injected <br /> embryo defective, performed mount situ hybridization using c-myb probe 3 dpf. Although half MOs effect <br /> on number HSCs, depletion rcor1 resulted increased numbers HSCs depletion kalrn1, kalrn2, mfn2, pdia5, psmd13 wasplb <br /> resulted decreased numbers HSCs CHT 3 dpf. Representative images CHT region shown. All embryos oriented anterior the <br /> left dorsal top. <br /> doi:10.1371/journal.pgen.1004450.g002 <br />  <br /> exception brd3a, kalrn2 kif1b, primitive erythropoiesis          MO, negative impact number c-myb <br /> largely unaffected following MO knock candidate genes               positive cells, resulted significant decrease number T <br /> (Figure S14). These results consistent notion         lymphocytes. <br /> majority candidate genes dispensable specification and <br /> differentiation primitive erythrocytes fundamentally            Analysis expression patterns phenotype <br /> different molecular mechanisms regulate primitive definitive            classification <br /> erythropoiesis.                                                                The large number genes analyzed resultant volume <br />    To establish role candidate genes differentiation       data acquired present challenge understanding and <br /> myeloid lineage, neutrophils macrophages,                    interpreting results. Hence, used information gained <br /> performed Sudan Black staining (for neutrophils) situ                initial MO knockdown screen generate heat-map of <br /> hybridization using mpeg riboprobe (for macrophages) control             phenotype profiles (Figure 3) cluster genes similar <br /> and candidate gene depleted zebrafish embryos 3 dpf. Out 15           phenotypic profiles. We hierarchically positioned candidate <br /> genes tested, depletion resulted reduced numbers              genes hematopoietic lineage tree assigned them <br /> Sudan Black positive cells (kif1b, waspla) effect         potential role hematopoietic differentiation (Figure 4). <br /> the number macrophages (Figures S15, S16, S18).                         Our analysis hematopoietic lineage tree revealed a <br /> Reduction number Sudan Black positive cells                 distinct pattern gene distribution, suggesting main gene <br /> reflect absence granules neutrophils. We have,           clusters. The cluster represents set genes, kalrn1 <br /> therefore, performed situ hybridization using mpx riboprobe          -2, mfn2, pdia5, psmd13, rcor1 wasplb, depletion <br /> all genes knockdown resulted decrease             affect number HSCs. The second major cluster represents a <br /> number Sudan Black positive cells. For tested genes          set genes, akap10, brf1a, kif1b, satb1 wasplb, which <br /> observed phenotype comparable reported                    appear essential hematopoietic tree and <br /> following Sudan Black staining (Figure S17).                                affect differentiation definitive erythrocytes thrombo- <br />    Finally, analyzed impact loss candidate gene                cytes. These genes presumed role HSC fate decisions prior <br /> function lymphoid development. Differentiated thymic T-cells             specification thrombocyte erythrocyte progenitors. <br /> are exclusively derived definitive HSCs readily                Although frequency blood defects observed screen <br /> identified rag1 expression examined 4 dpf (Figure S19).          high, screen suited identification of <br /> Not surprisingly, significant decrease rag1 staining        knockdown phenotypes result subtle differences myeloid <br /> observed set genes observed decrease           lineage cell production skewing myeloid lineage differenti- <br /> in c-myb staining (a marker HSCs), kalrn1, kalrn2, mfn2,         ation. This mainly changes number neutrophils <br /> pdia5 psmd13. In addition, injection akap10 MO rcor1             macrophages arising HSCs undetectable using <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                    4                           July 2014 | Volume 10 | Issue 7 | e1004450 <br />                                                                                                            Functional Genomic Screen Zebrafish <br />  <br />  <br />  <br />  <br /> Figure 3. Heat map summarising hematopoietic phenotypes knock-down 19 candidate genes. Data obtained initial <br /> knock-down used generate heat map phenotype profiles. Each colored cell heat map shows severity observed <br /> phenotype relative control. The severe decrease number cells displayed red, moderate reduction displayed orange and <br /> green denotes cell number comparable control. A moderate increase cell number displayed blue grey indicates test was <br /> not performed. <br /> doi:10.1371/journal.pgen.1004450.g003 <br />  <br />  <br /> markers developmental time point outlined here. Previous            embryos resulted partial statistically significant rescue the <br /> studies shown erythroid-myeloid progenitor cells (EMPs)           number thrombocytes demonstrating brd3a MO used in <br /> are capable generating macrophages neutrophils              study exerted specific effect (Figure 5B, C). <br /> that blood cells appear mib zebrafish despite absence of <br /> HSCs [9]. However, caveats, believe                  brd3a essential differentiation maintenance <br /> screening procedure used proven effective extracting                thrombocytes <br /> functional information GWAS dataset medium-                        Morpholinos allow gene-specific perturbation be <br /> throughput manner.                                                          carried temporal resolution, disadvantage <br />                                                                             dissecting precise role selected gene hematopoi- <br /> brd3a rate limiting regulator thrombopoiesis                        esis. A number studies reported compounds targeting BET <br />    To gain additional insight mechanisms             proteins used manipulate hematopoietic development <br /> newly discovered genes affect thrombopoiesis, performed                exploratory therapeutic purposes [14&#226;&#8364;&#8220;16]. The BET family <br /> more extensive analysis function brd3a hematopoiesis.          inhibitor, thieno-triazolo-1,4-diazepine ((+)-JQ1 short) a <br /> The bromodomain extra terminal domain (BET) family                   potent, highly specific inhibitor displaces BET proteins from <br /> proteins, including BRD2, BRD3, BRD4, evolutionally                 chromatin competitively binding acetyl-lysine recogni- <br /> conserved play key role cellular processes                 tion pocket BET bromodomains [17,18]. Thus, evaluated <br /> controlling assembly histone acetylation-dependent chro-             pharmacological impact (+)-JQ1 zebrafish development <br /> matin complexes [10]. To confirm defects                   thrombopoiesis. Exposure zebrafish embryos (+)-JQ1 <br /> observed brd3a depleted embryos resulted loss                disrupted chromatin occupancy hBRD3-GFP confirming the <br /> brd3a, vitro&#226;&#8364;&#8220;transcribed RNA encoding human BRD3 (hBRD3)                 efficacy inhibitor (Figure S20 A&#226;&#8364;&#8220;C). <br /> was injected 1-cell stage embryos. Live confocal imaging               We incubated embryos 6 hpf various concentra- <br /> zebrafish embryos injected hBRD3-GFP confirmed                    tions (+)-JQ1 (2)-JQ1 (stereoisomer no <br /> hBRD3 binds mitotic chromosomes (Figure 5A), feature                   appreciable affinity BET bromodomains) [17] control <br /> previously reported BET family proteins i.e. BRD2, BRD3             (Figure S21). Exposure zebrafish embryos 1 mg/ml (+)-JQ1 <br /> BRD4 [11&#226;&#8364;&#8220;13]. Expression hBRD3 brd3a MO injected                      resulted complete mortality 24 hpf (2)-JQ1 <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                    5                           July 2014 | Volume 10 | Issue 7 | e1004450 <br />                                                                                                                Functional Genomic Screen Zebrafish <br />  <br />  <br />  <br />  <br /> Figure 4. A cartoon model hematopoietic tree showing stages likely affected candidate gene knockdown. Based <br /> on heat map phenotype profiles hierarchically positioned candidate genes hematopoietic tree assigned potential <br /> role hematopoietic differentiation. Analysis hematopoietic tree identified set genes enriched HSC cluster, namely: kalrn 1 and <br /> -2, mfn2, pdia5, psmd13, rcor1 wasplb set thrombocyte-erythroid progenitor cell cluster, namely: akap10, brf1a, kif1b, satb1, <br /> wasplb. MPP &#226;&#8364;&#8220; multipotent progenitor cell, CMP &#226;&#8364;&#8220; common myeloid progenitor, CLP &#226;&#8364;&#8220; common lymphoid progenitor, MEP &#226;&#8364;&#8220; thrombocyte-erythroid <br /> progenitor, GMP &#226;&#8364;&#8220; granulocyte-macrophage progenitor, L &#226;&#8364;&#8220; lymphocyte, E &#226;&#8364;&#8220; erythrocyte, T &#226;&#8364;&#8220; thrombocyte. <br /> doi:10.1371/journal.pgen.1004450.g004 <br />  <br /> enantiomer showed observable effect embryo development                   incubated Tg(cd41:EGFP) embryos (+)-JQ1 inhibitor starting <br /> (Figure S21). This early embryonic death zebrafish embryos              3 dpf, considerable number of <br /> not surprising considering knockout Brd2 Brd4 mice               thrombocytes CHT, assessed number 24 hours later, <br /> results embryonic lethality, indicating important role          4 dpf. We untreated (2)-JQ1 treated embryos <br /> two proteins embryonic development [19,20]. When treated,                   number thrombocytes markedly increased 3 and <br /> however, (+)-JQ1 24 hpf, embryos exhibited overall                   4 dpf. However, (+)-JQ1 treated embryos observed no <br /> normal development higher concentration (1 mg/ml)               change number thrombocytes (Figure 6C). Moreover we <br /> (+)-JQ1 (Figure S21). Although morphologically normal, throm-                  (+)-JQ1 adverse effect number HSCs <br /> bopoiesis completely abolished embryos (Figure 6 A).              24 h period treatment (Figure 6D). Taken together <br /> Interestingly, decrease number thrombocytes                      strongly suggests brd3a important differentiation of <br /> appeared prominent presence (+)-JQ1 inhibitor                   thrombocytes HSCs, however, differentiated, brd3a was <br /> compared brd3a MO knock down. This opened possibility                   dispensable maintenance survival. <br /> that members BET family contributing to <br /> the observed phenotype. To investigate further, performed              Discussion <br /> MO knock zebrafish brd2a, brd2b brd4 assessed <br /> the number thrombocytes 3 dpf. Single MO knock                      GWAS meta-analysis platelet size number been <br /> all genes resulted severe decrease number                 successful identifying SNPs associated mass (volume x <br /> thrombocytes. These data strongly suggested that, indeed,                count) platelets. In contrast results GWAS in <br /> members BET family proteins (i.e. brd2 brd4) play                 common diseases, 80% SNPs associated with <br /> important role thrombopoiesis (Figure S22 A&#226;&#8364;&#8220;C).                             hematological traits localized 10 Kb genes providing <br />    Both MO knock treatment (+)-JQ1 inhibitor                     sound argument infer biologically relevant candidate genes <br /> from 24 hpf resulted severe reduction number                    [3,4]. Canonical pathway analyses detected highly significant <br /> thrombocytes 3 dpf, suggesting essential role brd3a           over-representation &#226;&#8364;&#732;&#226;&#8364;&#732;core genes&#226;&#8364;&#8482;&#226;&#8364;&#8482; (the sentinel SNP the <br /> differentiation thrombocytes opposed requirement                gene 10 kb gene) relevant biological <br /> their maintenance survival. To address question                    functions hematological disease, cancer cell cycle [3]. <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                       6                            July 2014 | Volume 10 | Issue 7 | e1004450 <br />                                                                                                             Functional Genomic Screen Zebrafish <br />  <br />  <br />  <br />  <br /> Figure 5. brd3a important regulator thrombopoiesis. (A) Live confocal imaging zebrafish embryos injected hBRD3-GFP mRNA <br /> revealed nuclear localization hBRD3 binds mitotic chromosomes (white arrows). Expression hBRD3 brd3a MO injected <br /> embryos resulted partial rescue number thrombocytes shown (B). (C) A graph illustrate number thrombocytes control, <br /> splice brd3a MO splice brd3a MO plus hBRD3 mRNA injected embryos. Each dot represents number thrombocytes individual MO- <br /> injected embryos respect control. A blue horizontal line represents mean value number thrombocytes group of <br /> embryos. Student t test, * p = 0.016; n = 17. All embryos oriented anterior left dorsal top. <br /> doi:10.1371/journal.pgen.1004450.g005 <br />  <br /> However, quarters regions proximal platelet GWAS             tree assigned potential role hematopoietic <br /> SNPs harbor unfamiliar genes known genes previously                   differentiation. Interestingly, screen revealed that, although <br /> implicated hematopoiesis merit extensive follow-up                   initially selected based effect platelet size and/or <br /> analysis. Therefore, study set address need           number, candidates exerted thrombocyte specific <br /> a medium-throughput method zebrafish dissect functional            effect. These results interpreted context of <br /> roles assumed novel regulators hematopoiesis.                    major differences effect SNPs whole <br />    In total, screen identified 15 genes (corresponding 12             embryo MO knock hematological traits. <br /> human genes) required distinct stages specification                   First, majority associated SNPs identified platelet <br /> differentiation HSCs zebrafish. A detailed review               GWAS non-transcribed regions likely the <br /> content databases literature revealed limited knowledge               underlying mechanism linking phenotype regula- <br /> about functional role Satb1, Rcor1 Brd3 [21&#226;&#8364;&#8220;24]                tory. Thus, functional effects SNPs subtler compared to <br /> hematopoiesis remaining genes work                      knock transcripts achieved MOs screening. <br /> represents study putative role hematopoiesis.          Secondly, GWAS provided list SNPs associated with <br /> Importantly, results line findings          platelet size number, evidence the <br /> reported others. One example RCOR1 - lineage-restricted                biological processes link associated SNP phenotype. <br /> deployment RCOR1 LSD1 cofactors, interaction                  Indeed, shown cases reported SNP is <br /> with Gfi proteins, controls hematopoietic differentiation [23].              functional SNP itself linkage disequilibrium with <br /> Knock rcor1 zebrafish resulted completely blocked              SNP overlapping functional region [25]. Experimental <br /> differentiation erythroid, thrombocytic, myeloid lymphoid             evidence shows open chromatin profiles megakaryocytes <br /> lineages. These findings strongly support hypothesis            erythroblasts differ cell type-restricted regions of <br /> published platelet GWAS [3] enriched functional regulators            open chromatin influence penetrability functional <br /> the hematopoiesis support previous assumptions            SNP [26,27] lineage specific manner. In contrast, MO knock <br /> large proportion genes uncovered aforementioned                zebrafish spatially restricted offers the <br /> GWAS conserved role zebrafish.                                opportunity determine functional role candidate genes in <br />    In study, followed step screening approach:          blood lineages. <br /> first instance, used Tg(cd41:EGFP) line conjunction              To verify hematopoietic role genes identified in <br /> a panel hematopoietic situ hybridization probes                    GWAS, performed extensive evaluation effect of <br /> histochemical staining create heat map distinct                    brd3a thrombopoiesis. It shown BRD3 interacts <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;phenotype signatures&#226;&#8364;&#8482;&#226;&#8364;&#8482; gene knock-down. We                    acetylated GATA1 stabilizes chromatin occupancy <br /> positioned candidate genes hematopoietic cell lineage             [21]. A pharmacologic compound, JQ1, occupies the <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                     7                           July 2014 | Volume 10 | Issue 7 | e1004450 <br />                                                                                                            Functional Genomic Screen Zebrafish <br />  <br />  <br />  <br />  <br /> Figure 6. brd3a required differentiation maintenance thrombocytes. (A) Exposure zebrafish embryos 1 mg/ml (+)-JQ1 <br /> from 24 hpf resulted severe reduction number thrombocytes compared DMSO (2)-JQ1 treated embryos. (B) A graph illustrate <br /> the average number thrombocytes 3 dpf. Embryos exposed DMSO, (2)-JQ1 (+)-JQ1 starting 24 hpf. Each dot represents <br /> thrombocyte count single embryo. Horizontal line denotes average thrombocyte number. Student t test, ** p = 1.661028, n = 17. (C) To assess <br /> whether brd3a essential differentiation thrombocytes maintenance survival incubated Tg(cd41:EGFP) embryos DMSO or <br /> with (+)-JQ1 inhibitor starting 3 dpf checked thrombocyte number 24 hours later, 4 dpf. We untreated (DMSO) embryos <br /> the number thrombocytes markedly increased 3- 4 dpf; however, (+)-JQ1 treated embryos change number of <br /> thrombocytes. In contrast, inhibitor adverse effect number HSCs 24 h period treatment (D). Representative <br /> confocal images CHT region shown. All embryos oriented anterior left dorsal top. <br /> doi:10.1371/journal.pgen.1004450.g006 <br />  <br />  <br /> acetyl-lysine binding pockets Brd3 bromodomains disrupts             thrombocyte identity. However, work necessary <br /> BRD3-GATA1 interaction, diminishes chromatin occupancy                  identify precise molecular mechanisms brd3a exerts <br /> of proteins, inhibits erythroid maturation [21]. Although          effect thrombopoiesis. Taken together, study provides a <br /> GATA1 BRD3 co-occupancy GATA1 target genes                       paradigm usefulness zebrafish efficient translation of <br /> also observed megakaryocytic cell line [21], biological            GWAS findings relevant biological information an <br /> relevance binding confirmed. Here report            objective unbiased manner. GWAS mapped many <br /> the important role brd3a thrombopoiesis. Indeed, knock-down           novel, convincingly associated loci proximity genes where <br /> of brd3a independent MOs treatment                   functional significance expected. So far, functional validation of <br /> zebrafish embryos JQ1 inhibitor starting 24 hpf               genes remained confined single gene approaches. Here <br /> severely reduced number thrombocytes 3 days old                   utilized powerful genetics translucency zebrafish <br /> embryos. Interestingly, incubation embryos JQ1 inhibitor            larvae undertake medium-throughput screen genes <br /> between 3- 4 dpf, onset thrombopoiesis, did        implicated human hematopoietic variation. The results this <br /> not effect differentiated thrombocytes.             screen help tentatively place novel genes molecular <br /> However, number thrombocytes failed increase                      pathways close ever-increasing knowledge gap the <br /> compared control embryos 24-hour period. These               biological function gene candidates identified genomic <br /> results strongly support idea brd3a critical                technologies. <br /> establishing maintaining thrombopoietic compartment. <br /> Some previous studies suggest BRD3,              Materials Methods <br /> mitotically retained factors, functions molecular &#226;&#8364;&#732;&#226;&#8364;&#732;bookmark&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> by enabling post-mitotic transcription re-initiation target          Ethics statement <br /> genes [13,28]. It plausible assume similar mechanism          The maintenance, embryo collection staging wild <br /> employed thrombopoiesis. In scenario, retention              type (Tubingen Long Fin) transgenic zebrafish lines <br /> Brd3 chromatin mitosis thrombocyte precursors               (Tg(cd41:GFP), Tg(fli1:GFP), Tg(c-myb:EGFP) performed in <br /> erythroid-thrombocyte progenitor cells contribute              accordance EU regulations laboratory animals, as <br /> maintenance transcription patterns necessary establishment           previously described [29,30]. <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                    8                           July 2014 | Volume 10 | Issue 7 | e1004450 <br />                                                                                                            Functional Genomic Screen Zebrafish <br />  <br />  <br />  <br /> Embryo injections                                                            inflicting wound stop bleeding was <br />    Morpholinos (GeneTools, LLC) re-suspended sterile                 recorded. <br /> water diluted chosen concentration. Approximately 1 nl <br /> was injected embryos 1- 2-cell stage. MOs used                Genotyping <br /> summarized Table S2.                                                         Genomic DNA isolated 5 dpf larvae, were <br />    Plasmid full-length human hBRD3 cDNA purchased                   individually loaded wells 96-well plate. Fish were <br /> from Source Bioscience (Nottingham, UK). hBRD3 cloned                    incubated 25 ml lysis buffer (25 mM NaOH 0.2 mM <br /> into pCS2 expression vector using gene-specific primers: AAT-                EDTA) 95uC 30 min. Afterwards 25 ml neutralization <br /> TACATCGATACCATGTCCACCGCCACGACAGT (forward)                                   buffer (40 mM Tris-HCl) added. Genotyping carried out <br /> and CCCGAGTCTAGACTATTCTGAGTCACTGCTGTCA-                                      using KASP genotyping assays (KBioscience). Each reaction <br /> GA (reverse) AAATTAGAATTCACCATGTCCACCGC-                                 consisted 4 ml genomic DNA 5 ml PCR mix, according <br /> CACGACAGT (forward) ATGTTAACCGGTAGTTCT-                                  manufacturer&#226;&#8364;&#8482;s protocol (KBioscience). PCR products were <br /> GAGTCACTGCTGTCAGA (reverse) cloning pCS2-                       analyzed using PHERAstar plus (BMGlabtech) KlusterCaller <br /> EGFP vector. Restriction enzyme sites (ClaI/XbaI EcoRI/                  software (KBioscience). <br /> AgeI respectively) used cloning underlined. Zebrafish full- <br /> length rcor1 cDNA cloned pCS2 vector using gene-specific            Imaging <br /> primers: GTTATAGAATTCATGCCCGCAATGTTAGAGA-                                       Images captured use Leica TCS SP5 confocal <br /> AG (forward) AGGCGCCTCGAGTCAGGAAACCGAA-                                  microscope Leica LAS AF software (Leica Microsystems), <br /> GGGTTCTG (reverse). Restriction enzyme sites (EcoRI                      using 406 immersion lens Axio Zoom.V16 fluorescent <br /> XhoI, respectively) underlined.                                          microscope AxioCam MRm camera using 2606 magnifica- <br />    hBRD3, hBRD3-GFP rcor1 mRNA synthesized                      tion. <br /> mMESSAGE mMACHINE kit (Ambion), according the <br /> manufacturer&#226;&#8364;&#8482;s protocol. In rescue experiment, 100 pg                 BET protein inhibitor <br /> hBRD3 mRNA 125 pg rcor1 RNA injected cell                   Selective inhibitor human BET family bromodomain- <br /> stage control MO-injected Tg(cd41:EGFP) embryos. In                  containing proteins, thieno-triazolo-1,4-diazepine, named JQ1, <br /> hBRD3 localization experiment, 300 pg hBRD3-GFP mRNA                      kindly provided Dr Chas Bountra, Structural Genomics <br /> was injected Tubingen Long Fin embryos 1-cell stage.                 Consortium, University Oxford, Oxford, UK. Both active <br />                                                                              inhibitor, (+)-JQ1, inactive stereoisomer, (2)-JQ1, were <br /> Reverse Transcription-Polymerase Chain Reaction (RT-                         dissolved dimethylsulfoxide (DMSO) 10 mg/ml stored in <br /> PCR)                                                                         aliquots 220uC. For zebrafish embryo treatment, (+)-JQ1 and <br />     In order verify effectiveness MOs affecting           (2)-JQ1 diluted egg water desired concentration and <br /> target transcripts, RT-PCR performed. RNA subjected               added embryos ,6 hpf, 24 hpf 3 dpf afterwards <br /> reverse transcription using Superscript II Reverse Transcriptase             replaced daily. Control embryos incubated equal <br />                                                                              concentration DMSO egg water inhibitor-treated <br /> (Invitrogen). PCR performed using gene-specific primers <br />                                                                              embryos. <br /> (listed Table S2) KOD Hot Start DNA Polymerase <br /> (Novagen). <br />                                                                              Supporting Information <br /> Whole mount situ hybridisation                                            Figure S1 We selected genes unknown function in <br />    In situ hybridization performed riboprobes specific          hematopoietic biology reliable zebrafish ortholog identifica- <br /> c-myb, ae1-globin, mpx, mpeg rag1 previously described [31],       tion 38% identity protein level human <br /> well brd3a, brf1b, kalrn1, waspla wdr66. Primers used         counterpart. We designed MO candidate gene <br /> PCR amplification candidate genes probe synthesis listed          tested efficacy MOs RT-PCR sequencing, <br /> in Table S3. Photomicrographs taken Zeiss camera                 appropriate. For genes, optimal dose MO was <br /> AxioCam HRC attached LeicaMZ16 FA dissecting micro-                     selected results specific phenotype gross <br /> scope (Leica Microsystems, Germany) using AxioVision                     lethality defects body shape size, vasculature, heart and <br /> software.                                                                    circulation. Finally, performed phenotyping MO injected <br />                                                                              embryos using wide panel different hematopoietic markers. <br /> O-dianisidine staining                                                       (TIFF) <br />   O-dianisidine staining performed previously described               Figure S2 Splice modifications caused gene-specific MOs <br /> [32].                                                                        assayed RT-PCR, using gene-specific primers, are <br />                                                                              seen band shift gel electrophoresis RT-PCR products. <br /> Sudan Black staining                                                         The binding site MO pre-mRNA marked red <br />   Sudan Black staining performed previously described                 horizontal line. For gene schematic diagram illustrates the <br /> [33].                                                                        effect MOs pre-mRNA splicing. Nucleotide sequences <br />                                                                              denote frame shift caused aberrant splicing. Where band <br /> Clotting time assay                                                          obtained MO-injected group, RT-PCR b-actin <br />    Clotting time assay performed previously described [7].            used control equal loading cDNA. i-intron; e-exon. <br /> In short, 5 dpf larvae anaesthetized 0.02% tricaine                  (TIFF) <br /> solution embryonic water transferred Petri dish           Figure S3 In order determine optimal dose MOs to <br /> small drop liquid. Caudal veins larvae wounded           injected, dose-response experiment performed. For each <br /> the tip Microlance needle (0.4 mm613 mm, Becton                         gene concentration MO (shown lower right corner of <br /> Dickinson) anal area. For larva time passing                 image, ng) selected elicited specific phenotype <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                     9                          July 2014 | Volume 10 | Issue 7 | e1004450 <br />                                                                                                             Functional Genomic Screen Zebrafish <br />  <br />  <br /> without overt non-specific effect. Knock-down grtp1a resulted          gene-specific primers, seen band shift gel <br /> in early embryonic lethality injected 0.8 ng               electrophoresis RT-PCR products. Where band was <br /> MO. Thus, grtp1a excluded analysis. The optimal             obtained MO-injected group, RT-PCR b-actin was <br /> MO dose gene determined experiment used              used control equal loading cDNA. <br /> in subsequent experiments.                                               (TIFF) <br /> (TIFF) <br />                                                                              Figure S9 MO activities zebrafish embryos include both <br /> Figure S4 Whole-mount situ hybridization using brd3a, brf1b,              sequence-specific RNA binding effects associated <br /> waspla, kalrn1 wdr66 riboprobes shown. Panels lateral          loss function targeted locus i.e. &#226;&#8364;&#732;&#226;&#8364;&#732;off-target&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> view embryos 3 dpf. All embryos oriented anterior          effects. As p53 MO reduce common off-target effects, the <br /> the left dorsal top.                                              number thrombocytes assessed CHT 3 dpf in <br /> (TIFF)                                                                       embryos co-injected p53 MO gene-specific MOs. <br /> Figure S5 The phenotyping initiated using                         Concurrent knock p53 gene-specific MOs did not <br /> Tg(cd41:EGFP) reporter line labels thrombocytes identify             attenuate thrombocyte phenotype induced gene-specific <br /> genes zebrafish that, knocked down, affect thrombocyte               MOs, confirming observed decrease number of <br /> number. The genes MO knock-down did result                thrombocytes induced off-target effects MOs. All <br /> change number thrombocytes excluded                  embryos oriented anterior left dorsal the <br /> analysis. For &#226;&#8364;&#732;&#226;&#8364;&#732;phenotypic&#226;&#8364;&#8482;&#226;&#8364;&#8482; genes, carried second level        top. <br /> of analysis performing mount situ hybridisation              (TIFF) <br /> several hematopoietic markers, namely: c-myb, ae1 globin, mpeg           Figure S10 A) Fish heterozygous mutation brf1b (allele <br /> rag1, order assess number HSCs, definitive                      sa3097, ZMP) crossed progeny (n = 41) was <br /> erythrocytes, macrophages lymphocytes, respectively. These               subjected clotting time assay 5 dpf. The time clotting <br /> were complemented use histochemical stains: o-               caudal vein puncture recorded larvae were <br /> Dianisidine, assess number primitive erythrocytes 2 dpf         genotyped. Each bar represents average clotting time group, <br /> and Sudan Black, determine number neutrophils CHT               SEM shown. One-tailed Student t test, * p = 0.042. B) A graph <br /> at 3 dpf. Finally, MO knock-down resulted decreased              illustrate number thrombocytes control (n = 33), splice <br /> number HSCs 3 dpf, additional step introduced                rcor1 MO (n = 28) splice rcor1 MO plus rcor1 mRNA (n = 15) <br /> assess number 30 hpf AGM. Where MO knock-                    injected embryos. Each bar represents average number of <br /> down resulted decreased number Sudan Black positive cells, <br />                                                                              thrombocytes group, SEM shown. One-tailed Student t <br /> whole mount situ hybridisation mpx performed. <br />                                                                              test, * p = 0.024. C) Representative fluorescent images embryos <br /> (TIFF) <br />                                                                              rcor1 rescue experiment. All embryos oriented with <br /> Figure S6 The number thrombocytes (cd41bright) CHT                 anterior left dorsal top. <br /> was counted gene knock-down. Of 18 genes                        (TIFF) <br /> examined, knock 15 resulted 30&#226;&#8364;&#8220;95% reduction in <br />                                                                              Figure S11 To assess HSC emergence aorta-gonad- <br /> the number thrombocytes. One-tailed Student t test was <br />                                                                              mesonephros (AGM) region, injected gene-specific MOs in <br /> performed. Significant decrease number thrombocytes <br />                                                                              single-cell stage Tg(c-myb:GFP) transgenic embryos. For kalrn1, <br /> was observed in: akap10 (p = 4.4761025, n = 10), brd3a <br /> (p = 0.00021,    n = 11),  brf1a    (p = 0.005, n = 9),  brf1b               mfn2, pdia5, psmd13 wasplb MO injected embryos difference <br /> (p = 2.561027, n = 15), kalrn1 (p = 1.461026, n = 13), kalrn2                number HSCs observed compared the <br /> (p = 1.261029, n = 15), kif1b (p = 5.761026, n = 17), mfn2                   control 30 hpf. However, kalrn2 rcor1 depleted embryos had <br /> (p = 1.4961025, n = 15), pdia5 (p = 0.00018, n = 11), psmd13                 marked decrease number HSCs 30 hpf, implying an <br /> (p = 4.1261028, n = 9), rcor1 (p = 0.00042, n = 20), satb1                   important role genes specification HSCs the <br /> (p = 9.5161027, n = 15), waspla (p = 0.00157, n = 15), wasplb                AGM. Representative images AGM region shown. <br /> (p = 0.023, n = 20) wdr66 (p = 0.00027, n = 15) depleted                 All embryos oriented anterior left dorsal the <br /> embryos. Each dot represents number thrombocytes                top. <br /> individual MO-injected embryo respect control. A blue                (TIFF) <br /> horizontal line represents mean value number                   Figure S12 In order assess vascular development, MOs <br /> thrombocytes group embryos.                                      targeting candidate genes injected Tg(fli1:EGFP) <br /> (TIFF)                                                                       embryos. Vascular morphology embryos assessed at <br /> Figure S7 To verify MOs used study exerted                    3 dpf. No major abnormalities vascular morphogenesis were <br /> specific effect hematopoiesis designed second non-                   observed tested MOs, indicating the <br /> overlapping morpholino (MO2) &#226;&#8364;&#732;&#226;&#8364;&#732;phenotypic&#226;&#8364;&#8482;&#226;&#8364;&#8482; candidate                hematopoietic defects secondary vascular <br /> genes. Splice modifications caused gene-specific MOs                 phenotype. Representative images CHT region shown. <br /> assayed RT-PCR, using gene-specific primers,                     All embryos oriented anterior left dorsal to <br /> confirmed brf1a, kif1b wasplb MO2. These               top. <br /> MO2 excluded analysis. Injection              (TIFF) <br /> the remaining 12 MO2 resulted phenotype                      Figure S13     Whole mount situ hybridization performed <br /> comparable observed MO. Representative             using probe specific ae1-globin 4 dpf. As result candidate <br /> fluorescent images CHT shown. All embryos                     gene knock down, depletion ae1-globin staining observed for <br /> oriented anterior left dorsal top.                    genes brd3a, brf1b, waspla wdr66. Representative images <br /> (TIFF)                                                                       CHT shown. All embryos oriented anterior the <br /> Figure S8 Splice modifications caused gene-specific second                left dorsal top. <br /> non-overlapping MOs (MO2) assayed RT-PCR, using                      (TIFF) <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                    10                           July 2014 | Volume 10 | Issue 7 | e1004450 <br />                                                                                                               Functional Genomic Screen Zebrafish <br />  <br />  <br /> Figure S14 All erythrocytes present circulation 2 dpf                lymphocyte numbers. All embryos positioned anterior up <br /> zebrafish embryos derive primitive wave hematopoi-                 dorsal back. <br /> esis. In order assess primitive erythropoiesis, MO injected             (TIFF) <br /> embryos stained O-dianisidine 2 dpf (arrow). <br />                                                                                Figure S20    (+)-JQ1 highly specific inhibitor, displaces <br /> Knockdown 12 15 candidate genes, namely: akap10, <br />                                                                                BET proteins chromatin competitively binding the <br /> brf1a, brf1b, kalrn1, mfn2, pdia5, psmd13, rcor1, satb1, waspla, wasplb <br />                                                                                acetyl-lysine recognition pocket BET bromodomains. (A) <br /> and wdr66, resulted observable phenotype. Depletion of <br />                                                                                Treatment zebrafish embryos (+)-JQ1 disrupted the <br /> brd3a, kalrn2 kif1b resulted severe reduction number <br />                                                                                chromatin occupancy hBRD3-GFP demonstrated the <br /> of primitive erythrocytes. All embryos positioned anterior <br />                                                                                absence GFP-positive mitotic chromosomes embryos <br /> up dorsal back. <br />                                                                                (0/142 GFP-positive cells) compared 9/141 GFP-positive cells <br /> (TIFF) <br />                                                                                DMSO control group. (B&#226;&#8364;&#8220;C) To quantify the <br /> Figure S15 Individual neutrophils detected Sudan                   chromatin occupancy hBRD3-GFP mitosis, selected <br /> Black staining. The number neutrophils MO-injected                       $15 mitotic cells, shown DAPI staining, counted how <br /> embryos (nembryos = 15) CHT counted 3 dpf. Knock-                    cells GFP positive presence (+)-JQ1 or <br /> down akap10, brd3a, brf1b, kif1b, waspla wasplb caused                DMSO (control). Whereas (+)-JQ1 treated embryos the <br /> phenotype. A mild decrease (20&#226;&#8364;&#8220;50%) number                           DAPI positive mitotic chromosomes GFP positive, DMSO <br /> neutrophils observed brf1a, kalrn2, pdia5, rcor1, satb1            treated embryos 11 cells double DAPI/GFP positive. Asterisk <br /> wdr66 MO injected embryos compared control kalrn1,                      depicts GFP cytoplasm arrow shows mitotic <br /> mfn2 psmd13 MO injected embryos showed severe (over                      chromosomes. <br /> 50%) depletion neutrophils compared control.                        (TIFF) <br /> Representative images CHT region 3 dpf shown. <br /> All embryos oriented anterior left dorsal               Figure S21 To assess effect inhibitor embryo <br /> the top.                                                                       development, dose response experiments performed. Em- <br /> (TIFF)                                                                         bryos treated 1 mg/mL (+)-JQ1 6 hpf died 24 hpf. <br />                                                                                The concentration inactive enantiomer (2)-JQ1 did <br /> Figure S16 To identify neutrophils, Sudan Black staining                   affect development embryos. Incubation lower <br /> performed 3 dpf. The number neutrophils CHT                       concentration (+)-JQ1 (0.5 mg/ml) led aberrant development, <br /> counted group (nembryos = 15). One-tailed Student t test               i.e. tail malformation heart edema. The lowest tested dose, <br /> was performed. Significant decrease number neutro-                   0.25 mg/ml, did affect development embryos. To <br /> phils observed brf1a (p = 0.019), kalrn1 (p = 3.161029),                avoid early embryonic lethality (+)-JQ1 inhibitor added at <br /> kalrn2 (p = 0.004), mfn2 (p = 1.461026), pdia5 (p = 0.004), psmd13             24 hpf. The embryos exhibited overall normal development even <br /> (p = 4.2610211), rcor1 (p = 0.012), satb1 (p = 0.011) wdr66                higher concentration (1 mg/ml) (+)-JQ1. Representative <br /> (p = 9.261024) MO-injected embryos. Each dot represents                    images embryo morphology shown, taken 24 hpf, 48 hpf <br /> number neutrophils individual MO-injected embryo                      72 hpf. All embryos positioned anterior left <br /> with respect control. A blue horizontal line represents                 dorsal top. <br /> mean value number neutrophils group                      (TIFF) <br /> embryos. <br /> (TIFF)                                                                         Figure S22 To assess role BET family members in <br />                                                                                thrombopoiesis MO knock brd2a, brd2b brd4 was <br /> Figure S17 The expression mpx control brf1a,               performed. A) Although morphologically normal, embryos <br /> kalrn1, kalrn2, mfn2, pdia5, psmd13, rcor1, satb1 wdr66 depleted           injected brd2a, brd2b brd4 MOs severe decrease in <br /> embryos assessed situ hybridization. For genes tested            number thrombocytes 3 dpf (B). Representative pictures <br /> the reduced number mpx positive cells observed                     CHT shown. C) For splice-blocking MOs effect the <br /> compared control. All embryos oriented anterior             MOs confirmed RT-PCR. All embryos positioned <br /> the left dorsal top.                                                anterior left dorsal top. <br /> (TIFF)                                                                         (TIFF) <br /> Figure S18 In order detect macrophages, mount situ                 Table S1 Selection genes vivo functional screening. <br /> hybridization performed using probe specific mpeg1               (TIFF) <br /> 3 dpf. For 13 15 tested MOs (akap10, brd3a, brf1a, brf1b, <br /> kalrn1, kalrn2, mfn2, pdia5, psmd13, rcor1, satb1, wasplb wdr66)           Table S2 Sequences morpholinos primers used the <br /> there observable difference number macrophag-                 study. Morpholinos effect splicing target <br /> es. However, knock kif1b waspla resulted severe               transcript underlined. <br /> reduction number macrophages CHT 3 dpf. All                    (TIFF) <br /> embryos oriented anterior left dorsal               Table S3    Primers used PCR amplification candidate genes <br /> top.                                                                           probe synthesis. <br /> (TIFF)                                                                         (TIFF) <br /> Figure S19 Differentiated thymic T-cells readily <br /> identified rag1 expression examined 4 dpf. Whole                    Author Contributions <br /> mount situ hybridization rag1 riboprobe revealed severe            Conceived designed experiments: AC. Performed experiments: <br /> decrease number T lymphocytes kalrn2, mfn2, pdia5             AC EBM JSC LF. Analyzed data: AC WHO. Contributed reagents/ <br /> psmd13 MO injected embryos thymi (arrow) 4 dpf. Knock                    materials/analysis tools: WHO NS. Wrote paper: AC EBM. Edited the <br /> down akap10, kalrn1 rcor1 caused moderate decrease T               manuscript: DLS. <br />  <br />  <br />  <br />  <br /> PLOS Genetics | www.plosgenetics.org                                      11                           July 2014 | Volume 10 | Issue 7 | e1004450 <br />                                                                                                                                    Functional Genomic Screen Zebrafish <br />  <br />  <br />  <br />  <br /> References <br />  1. Evans DM, Frazer IH, Martin NG (1999) Genetic environmental causes                18. Filippakopoulos P, Picaud S, Fedorov O, Keller M, Wrobel M, et al. (2012) <br />     variation basal levels blood cells. Twin Res 2:250&#226;&#8364;&#8220;257.                                Benzodiazepines benzotriazepines protein interaction inhibitors targeting <br />  2. Garner C, Tatu T, Reittie JE, Littlewood T, Darley J, et al. (2000) Genetic                  bromodomains BET family. Bioorgan Med Chem 20: 1878&#226;&#8364;&#8220;1886. <br />     influences F cells hematologic variables: twin heritability study.        19. Houzelstein D, Bullock SL, Lynch DE, Grigorieva EF, Wilson VA, et al. (2002) <br />     Blood 95: 342&#226;&#8364;&#8220;346.                                                                           Growth Early Postimplantation Defects Mice Deficient the <br />  3. Gieger C, Radhakrishnan A, Cvejic A, Tang W, Porcu E, et al. (2011) New gene                 Bromodomain-Containing Protein Brd4. Mol Cell Biol 22: 3794&#226;&#8364;&#8220;3802. <br />     functions megakaryopoiesis platelet formation. Nature 480: 201&#226;&#8364;&#8220;208.               20. Shang E, Wang X, Wen D, Greenberg DA, Wolgemuth DJ (2009) Double <br />  4. van der Harst P, Zhang W, Mateo Leach I, Rendon A, Verweij N, et al. (2012)                  bromodomain-containing gene Brd2 essential embryonic development in <br />     Seventy-five genetic loci influencing human red blood cell. Nature 492: 369&#226;&#8364;&#8220;             mouse. Dev Dyn 2009 238: 908&#226;&#8364;&#8220;917. <br />     375.                                                                                     21. Lamonica JM, Deng W, Kadauke S, Campbell AE, Gamsjaeger R, et al. (2011) <br />  5. Serbanovic-Canic J, Cvejic A, Soranzo N, Stemple DL, Ouwehand WH, et al.                     Bromodomain protein Brd3 associates acetylated GATA1 promote its <br />     (2011) Silencing RhoA nucleotide exchange factor, ARHGEF3, reveals                    chromatin occupancy erythroid target genes. Proc Natl Acad Sci U S A 108: <br />     unexpected role iron uptake. Blood 118: 4967&#226;&#8364;&#8220;4976.                                        159&#226;&#8364;&#8220;168. <br />  6. Stachura DL, Traver D (2009) Cellular Dissection Zebrafish Hematopoiesis.             22. Notani D, Gottimukkala KP, Jayani RS, Limaye AS, Damle MV, et al. (2010) <br />     In Essential Zebrafish Methods: Cell Developmental Biology. Edited                    Global Regulator SATB1 Recruits b-Catenin Regulates TH2 Differenti- <br />     Westerfield M. Elsevier, Inc.                                                                ation Wnt-Dependent Manner. PLoS Biol 8: e1000296. <br />  7. Jagadeeswaran P, Liu Y (1997) A hemophilia model zebrafish: analysis               23. Saleque S, Kim J, Rooke HM, Orkin SH (2007) Epigenetic Regulation of <br />     hemostasis. Blood Cells Mol Dis 23(1):52&#226;&#8364;&#8220;7.                                                  Hematopoietic Differentiation Gfi-1 Gfi-1b Is Mediated Cofactors <br />  8. Brownlie A, Hersey C, Oates AC, Paw BH, Falick AM, et al. (2003)                             CoREST LSD1. Mol Cell 27: 562&#226;&#8364;&#8220;572. <br />     Characterization embryonic globin genes zebrafish. Dev Biol 255:               24. Yao H, Goldman DC, Nechiporuk T, Kawane S, McWeeney SK, et al. (2014) <br />     48&#226;&#8364;&#8220;61.                                                                                       The co-repressor Rcor1 essential murine erythropoiesis. Blood <br />  9. Bertrand JY, Cisson JL, Stachura DL, Traver D (2010) Notch signaling <br />                                                                                                  123(20):3175&#226;&#8364;&#8220;84 <br />     distinguishes 2 waves definitive hematopoiesis zebrafish embryo. Blood <br />                                                                                              25. Nurnberg ST, Rendon A, Smethurst PA, Paul DS, Voss K, et al. (2012) A <br />     115: 2777&#226;&#8364;&#8220;2783. <br />                                                                                                  GWAS sequence variant platelet volume marks alternative DNM3 <br /> 10. Zeng L, Zhou MM (2002) Bromodomain: acetyl-lysine binding domain. <br />                                                                                                  promoter megakaryocytes near MEIS1 binding site. Blood 120(24):4859&#226;&#8364;&#8220;68 <br />     FEBS Lett 513: 124&#226;&#8364;&#8220;128. <br />                                                                                              26. Paul DS, Nisbet JP, Yang T-P, Meacham S, Rendon A, et al. (2011) Maps of <br /> 11. Kanno T, Kanno Y, Siegel RM, Jang MK, Lenardo MJ, et al. (2004) Selective <br />     recognition acetylated histones bromodomain proteins visualized living              Open Chromatin Guide Functional Follow-Up Genome-Wide Associa- <br />     cells. Mol Cell 13: 33&#226;&#8364;&#8220;43.                                                                   tion Signals: Application Hematological Traits. PLoS Genet 7:e1002139. <br /> 12. Dey A (2003) The double bromodomain protein Brd4 binds acetylated                     27. Paul DS, Albers CA, Rendon A, Voss K, Stephens J, et al. (2013) Maps open <br />     chromatin interphase mitosis. Proc Natl Acad Sci U S A 100: 8758&#226;&#8364;&#8220;                 chromatin highlight cell type-restricted patterns regulatory sequence variation <br />     8763.                                                                                        hematological trait loci. Gen Res 23: 1130&#226;&#8364;&#8220;41. <br /> 13. Kadauke S, Blobel GA (2012) &#226;&#8364;&#732;&#226;&#8364;&#732;Remembering&#226;&#8364;&#8482;&#226;&#8364;&#8482; tissue-specific transcription                28. Kadauke S, Udugama MI, Pawlicki JM, Achtman JC, Jain DP, et al. (2012) <br />     patterns mitosis. Cell Cycle 11: 3911&#226;&#8364;&#8220;3912.                                          Tissue-Specific Mitotic Bookmarking Hematopoietic Transcription Factor <br /> 14. Belkina AC, Nikolajczyk BS, Denis GV (2013) BET Protein Function Is                          GATA1. Cell 150: 725&#226;&#8364;&#8220;737. <br />     Required Inflammation: Brd2 Genetic Disruption BET Inhibitor JQ1                 29. Kimmel CB, Ballard WW, Kimmel SR, Ullmann B, Schilling TF (1995) Stages <br />     Impair Mouse Macrophage Inflammatory Responses. J Immunol 190: 3670&#226;&#8364;&#8220;                         embryonic development zebrafish. Dev Dyn 203: 253&#226;&#8364;&#8220;310. <br />     3678.                                                                                    30. Westerfield M (2000) The zebrafish book. A guide laboratory use of <br /> 15. Chung C-W, Coste H, White JH, Mirguet O, Wilde J, et al. (2011) Discovery                    zebrafish (Danio rerio). Eugene: University Oregon Press. <br />     Characterization Small Molecule Inhibitors BET Family                    <br /> </body></html>