<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Accumulation Phosphatidic Acid Increases Vancomycin Resistance <br /> in Escherichia coli <br /> Holly A. Sutterlin,a Sisi Zhang,b Thomas J. Silhavya <br /> Department Molecular Biology, Princeton University, Princeton, New Jersey, USAa; Lewis-Sigler Institute Integrative Genomics, Princeton University, Princeton, New <br /> Jersey, USAb <br />  <br /> In Gram-negative bacteria, lipopolysaccharide (LPS) contributes robust permeability barrier outer membrane, pre- <br /> venting entry toxic molecules antibiotics. Mutations lptD, beta-barrel component LPS transport as- <br /> sembly machinery, compromise LPS assembly result increased antibiotic sensitivity. Here, report rare vancomycin- <br /> resistant suppressors improve barrier function subset lptD mutations. We seven suppressors analyzed <br /> mapped essential gene cdsA, responsible conversion phosphatidic acid CDP-diacylglycerol phos- <br /> pholipid biosynthesis. These cdsA mutations cause partial loss function and, expected, accumulate phosphatidic acid. We <br /> show suppression confined mutations cause defects outer membrane biogenesis these <br /> cdsA mutations confer general increase vancomycin resistance, wild-type cell. We use genetics quadrupole time <br /> of flight (Q-TOF) liquid chromatography-mass spectrometry (LC-MS) accumulation phosphatidic acid means <br /> other cdsA mutations increases resistance vancomycin. We suggest increased levels phosphatidic acid change <br /> the physical properties outer membrane impede entry vancomycin periplasm, hindering access target, <br /> an intermediate required synthesis peptidoglycan cell wall. <br />  <br />  <br />  <br /> T   outer membrane (OM) Gram-negative bacteria serves as <br />     robust permeability barrier (1). Composed lipopolysac- <br /> charide (LPS) molecules outer leaflet glycerophospho- <br />                                                                                        CDP-diacylglycerol step commitment either <br />                                                                                        zwitterionic GPL pathway acidic GPL pathway (6, 12). <br />                                                                                            In work, focus E. coli mutants defective LPS as- <br /> lipids (GPLs) inner leaflet, OM protects cell                      sembly exhibit increased sensitivity antibiotics. Mutations <br /> harmful agents, detergents antibiotics (1&#226;&#8364;&#8220;3). LPS                       compromise function LptD, beta-barrel compo- <br /> synthesized inner membrane (IM), transported                         nent Lpt machinery, sensitive vancomycin. In <br /> aqueous periplasm, finally assembled outer leaflet                     order act cell wall target, vancomycin traverse the <br /> the OM (4). The Lpt proteins responsible transport                        OM. Once periplasm, vancomycin binds termi- <br /> and assembly (5). Glycerophospholipids (GPLs) synthesized                          nal D-alanine residues cell wall precursor lipid II, pre- <br /> at IM, little known transport                        venting polymerization peptidoglycan cell wall (13, 14). <br /> periplasm assembly OM (6). The coordination LPS                        lptD mutants exhibit increased sensitivity vancomycin because <br /> and GPL assembly important maintain effective OM per-                         allow efficient entry drug periplasm <br /> meability barrier (1). <br />                                                                                        result compromised OM barrier. <br />     Because high density LPS outer leaflet, OM <br />                                                                                            We report isolation characterization vancomy- <br /> carries net negative charge (7). The human innate immune sys- <br />                                                                                        cin-resistant suppressors subset lptD mutations. These sup- <br /> tem takes advantage property OM producing <br />                                                                                        pressors map essential gene cdsA. Vancomycin-resistant <br /> cationic antimicrobial peptides (CAMPs) (8). Positively charged <br /> CAMPs associate negatively charged cell surface in-                       suppression specific lptD mutants; cdsA mutations <br /> sert OM, killing bacterial cell (8). Bacteria sys-                   actually increase vancomycin resistance wild- <br /> tems place, PhoPQ virulence Salmonella                        type cell. These cdsA mutations cause recessive, partial loss of <br /> enterica serovar Typhimurium, modify lipid A PL                           function, accumulate PA like cdsA mutants previ- <br /> content decrease negative charge OM, de-                         ously described (15, 16). We suggest increased PA alters the <br /> creasing affinity CAMPs OM (9&#226;&#8364;&#8220;11). This increases                       OM causes increased resistance vancomycin. <br /> bacterial resistance CAMPs enables evasion immune <br /> system. <br />     Like LPS, GPLs contribute electric charge the <br /> membrane. Escherichia coli membranes composed three <br /> major GPLs: phosphatidylethanolamine (PE), phosphatidylglyc-                              Received 20 May 2014 Accepted 16 June 2014 <br /> erol (PG), cardiolipin (CL). The relative abundances                          Published ahead print 23 June 2014 <br /> of GPL species similar IM OM (PE, 75%; PG,                           Address correspondence Thomas J. Silhavy, tsilhavy@exchange.Princeton.EDU. <br /> 20%; CL, 5%) (12). PE zwitterionic GPL neutral charge,                        Supplemental material article http://dx.doi.org/10.1128 <br /> while PG CL acidic GPLs carry negative charges (6).                          /JB.01876-14. <br /> Although GPLs different chemical properties                             Copyright &#194;&#169; 2014, American Society Microbiology. All Rights Reserved. <br /> their head groups, synthesized precursor                           doi:10.1128/JB.01876-14 <br /> molecule, phosphatidic acid (PA) (6, 12). The conversion PA <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br />  <br />  <br /> 3214   jb.asm.org                                   Journal Bacteriology   p. 3214 &#226;&#8364;&#8220;3220                                     September 2014 Volume 196 Number 18 <br />                                                                                                            Phosphatidic Acid Increases Vancomycin Resistance <br />  <br />  <br />  <br /> TABLE 1 Strains plasmids used study                               romycin, 30 &#226;?&#174;g moxalactam, 0.1 &#226;?&#174;g norfloxacin, 260 &#226;?&#174;g paraquat, 300 <br /> Strain                                                  Reference          units polymyxin B, 1 mg vancomycin. Isopropyl-&#226;?&#164;-D-1-thiogalactopy- <br /> plasmid            Relevant features                       and/or source      ranoside (IPTG; 0.1 mM) present LB agar indicated. <br />                                                                                   Growth rate determination. Overnight cultures diluted 1:100 in <br /> Strains                                                                       100 &#226;?&#174;l LB medium 96-well plates. One microliters min- <br />    MC4100          F&#226;&#171;&#186; araD139 &#226;&#338;&#172;(argF-lac)U169 rpsL150      17                 eral oil overlaid prevent evaporation. Optical density 600 nm <br />                      relA1 flbB5301 deoC1 ptsF25 rbsR                         (OD600) measured 20 min intermittent orbital shaking at <br />                      thi                                                      37&#194;&#176;C. These 4-hour OD600 growth curves used determine the <br />   NR754            MC4100 Ara&#226;&#171;&#185;                             18                 growth rate. OD600 plotted time logarithmic scale, and <br />   HC334            NR754 &#226;&#338;&#172;lptD pET2342::lptDI54E           21, 48, 49;   linear regression analysis used determine growth rate each <br />                                                               study           culture. For strain, technical replicates biological rep- <br />   HC338            NR754 &#226;&#338;&#172;lptD pET2342::lptDV52E-I54E      21, 48, 49;   licates averaged mean averages calculated and <br />                                                               study           reported standard error mean. <br />   HC340            NR754 &#226;&#338;&#172;lptD pET2342::lptD               21, 48, 49;       Cell size measurements. Overnight cultures diluted 1:100 and <br />                                                               study           grown OD600 approximately 0.4. Cells suspended 1% <br />   HC342            NR754 cdsA7 dgt::Tn10                   This study         agarose pad imaged using 100&#226;&#171;&#187; 1.4-numerical-aperture (NA) ob- <br />   HC351            HC334 &#226;&#338;&#172;clsA::kan                        34; study     jective Nikon 90i microscope equipped QImaging Rolera XR <br />   HC386            HC334 cdsA7 dgt::Tn10                   This study         camera NIS Elements software. Cell dimensions quantified with <br />   HC509            NR754 &#226;&#338;&#172;lptD pET2342::lptD&#226;&#338;&#172;D57&#226;&#338;&#172;K60       21, 48, 49;   MicrobeTracker version 0.936 (22), using MATLAB version R2013a. Ap- <br />                                                               study           proximately 100 cells examined independent exper- <br />   HC510            HC334 pgsA444 zed-3069::Tn10            39, 50, 51;   iments. P values calculated using Student&#226;&#8364;&#8482;s t test. <br />                                                               study               Bligh Dyer lipid extraction. Lipids extracted using acidic <br />   HC530            HC509 cdsA7 dgt::Tn10                   This study         Bligh Dyer method previously described (23, 24). One-hundred-milli- <br />   HC571            NR754 &#226;&#338;&#172;clsA::kan                        34; study     liter cultures grown OD600 1.5, cells were <br />   HC575            NR754 pgsA444 zed-3069::Tn10            39, 50, 51;   harvested centrifugation. Cell pellets washed phos- <br />                                                               study           phate-buffered saline (PBS) resuspended 1 ml 0.1 N HCl. <br />   HC630            HC509 pgsA444 zed-3069::Tn10            39, 50, 51;   Methanol (2.5 ml) chloroform (1.25 ml) added form single- <br />                                                               study           phase solution (chloroform-methanol-0.1 N HCl [1:2:0.8 {vol/vol/vol}]) <br />   HC960            HC334 cdsA9 dgt::Tn10                   This study         incubated 30 min room temperature sporadic vortexing. <br />   HC961            HC509 cdsA9 dgt::Tn10                   This study         Then, 0.1 N HCl (1.25 ml) chloroform (1.25 ml) added form <br />   HC962            NR754 cdsA9 dgt::Tn10                   This study         two-phase solution (chloroform-methanol-0.1 N HCl [2:2:1.8 {vol/vol/ <br />   HC1045           NR754 pASKA::plsC                       35; study     vol}]). These solutions centrifuged 3,000 &#226;&#171;&#187; g 25 min at <br />   HC1198           JAS193 cdsA7 dgt::Tn10                  This study         room temperature. The lower phase recovered dried nitro- <br />   HC1199           JAS193 cdsA9 dgt::Tn10                  This study         gen stream. <br />   HC1240           HC342 pASKA::cdsA                       35; study         Mass spectrometry. Samples reconstituted 500 &#226;?&#174;l 1:1:0.3 <br />   HC1241           HC962 pASKA::cdsA                       35; study     methanol-chloroform-water. Mass spectrometric analyses per- <br />   JAS193           MC4100 &#226;&#338;&#172;bamE::kan                       J. Schwalm,        formed 6550 Accurate-Mass quadrupole time flight (Q-TOF) mass <br />                                                               unpublished     spectrometer (Agilent Technologies, Inc., Santa Clara, CA), equipped <br />                                                               data            electrospray ionization (ESI) source operated negative- <br /> Plasmids                                                                      positive-ion mode. Liquid chromatography (LC) performed us- <br />   pET2342::lptD    lptD cloned pET2342                48                 ing 1290 Infinity HPLC (Agilent Technologies, Inc., Santa Clara, <br />   pASKA::cdsA      IPTG inducible, Camr                    35                 CA) coupled mass spectrometer, scans m/z 200 1,000 <br />   pASKA::plsC      IPTG inducible, Camr                    35                 2-GHz extended dynamic range. Nitrogen used drying gas, neb- <br />                                                                               ulizer gas, collision gas. Mass spectrometer operation conditions were <br />                                                                               follows: capillary voltage, 4,000 V; fragmentor voltage, 225 V; nebulizer <br />                                                                               gas, 35 lb/in2; drying gas, 13 liter/min; gas temperature, 275&#194;&#176;C. The in- <br /> MATERIALS AND METHODS                                                         strument control, data acquisition, data analysis performed by <br /> Bacterial strains, plasmids, media, chemicals. All strains used    Agilent Mass Hunter software (Agilent Technologies; version <br /> work isogenic derivatives NR754, araD&#226;&#171;&#185; revertant MC4100          B.05.00), controlled chromatography system. <br /> (17, 18). Strains constructed generalized P1 transduction trans-       The LC parameters follows: column, Agilent Poroshell-120 <br /> formation (19). Suppressor mutations mapped using pools ran-          EC-C18 (2.7 &#226;?&#174;m, 2.1 150 mm); autosampler temperature, 5&#194;&#176;C; injec- <br /> dom Tn10 insertions described previously (20). If necessary, kan cas-      tion volume, 10 &#226;?&#174;l; column temperature, 60&#194;&#176;C; solvent flow rate, 150 <br /> settes excised using Flp recombinase (21). Strains plasmids      &#226;?&#174;l/min. Solvent A 1 mM ammonium acetate plus 0.2% acetic acid in <br /> used work listed Table 1. All cultures grown Luria-    90:10 water-methanol, solvent B 1 mM ammonium acetate plus <br /> Bertani (LB) broth agar 37&#194;&#176;C. For mass spectrometry (MS), high-         0.2% acetic acid 2:98 methanol&#226;&#8364;&#8220;2-propanol. The gradient 0 min, <br /> pressure liquid chromatography (HPLC)-grade solvents (water, metha-           25% B; 2 min, 25% B; 4 min, 65% B; 16 min, 100% B; 20 min, 100% B; 21 <br /> nol, isopropanol), acetic acid, ammonium acetate obtained        min, 25% B; 22 min, 25% B. Postrun time 6 min. <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> Fisher Scientific (Pittsburgh, PA). <br />     Antibiotic sensitivity assays. Efficiency plating assays (EOPs)     RESULTS <br /> disk diffusion assays used determine antibiotic sensitivity. For <br />                                                                               Vancomycin-resistant suppressors lptD mutations map to <br /> EOPs, strains grown overnight, serially diluted, replica <br /> plated LB agar LB agar supplemented vancomycin (70 mg/liter,       cdsA. Escherichia coli lpt mutants exhibit defects transport <br /> 120 mg/liter, 140 mg/liter, 150 mg/liter, 200 mg/liter), 625 mg/liter      assembly LPS exhibit sensitivity detergents and <br /> bacitracin, 1 mg/liter gentamicin, 10 mg/liter rifampin. For disk diffu-   antibiotics (18, 25&#226;&#8364;&#8220;31). lptDI54E, lptDV52E-I54E, lptD&#226;&#338;&#172;D57&#226;&#338;&#172;K60 <br /> sion assays, disks contained following amounts antibiotics: 1.25   harbor denoted mutations alter soluble N terminus of <br /> mg ampicillin, 320 &#226;?&#174;g cetyltrimethylammonium bromide, 125 &#226;?&#174;g eryth-           OM beta-barrel protein LptD confer sensitivity vanco- <br />  <br />  <br /> September 2014 Volume 196 Number 18                                                                                                       jb.asm.org 3215 <br />  Sutterlin et al. <br />  <br />  <br />  <br />                                                                                         Because cdsA essential gene (16, 34), suppressors can- <br />                                                                                     null mutations. We used complementation analysis de- <br />                                                                                     termine cdsA7 cdsA9 mutations dominant <br />                                                                                     recessive. The cdsA clone &#226;&#8364;&#339;A Complete Set E. coli <br />                                                                                     K-12 ORF Archive&#226;&#8364;? (ASKA) collection introduced the <br />                                                                                     cdsA7 cdsA9 backgrounds (35). Basal-level (uninduced) ex- <br />                                                                                     pression wild-type copy cdsA ASKA plasmid de- <br />                                                                                     creased vancomycin resistance wild-type levels (see Fig. S1 <br />                                                                                     supplemental material). Because cdsA7 cdsA9 reces- <br />                                                                                     sive, conclude partial loss-of-function mutations. <br />                                                                                         The cdsA mutations alter cell size. Previously, has <br />                                                                                     shown mutations confer LPS transport defects can <br />                                                                                     suppressed mutations alter fatty acid biosynthesis (36), <br />                                                                                     wanted determine suppressors acted similar <br />                                                                                     manner. fabH accD mutations suppress certain lpt muta- <br />                                                                                     tions reducing cell size growth rate, turn decrease <br />                                                                                     rate growth cell envelope (36). The cdsA7 cdsA9 <br />                                                                                     mutants did exhibit decrease cell volume (see Fig. S2 the <br />                                                                                     supplemental material), growth rates did differ sig- <br />                                                                                     nificantly wild type (see Table S1). We conclude <br />                                                                                     results cdsA mutations suppress lptD <br />                                                                                     mutations decreasing rate growth cell envelope. <br />                                                                                         Partial loss-of-function cdsA mutants generally increase <br />                                                                                     vancomycin resistance E. coli strains. Because cdsA sup- <br />                                                                                     pressors isolated different lptD mutant backgrounds, we <br />                                                                                     asked mode suppression specific lptD <br />                                                                                     mutant. For simplicity, chose analyze cdsA mu- <br />                                                                                     tants (cdsA7 cdsA9) lptD mutant backgrounds <br />                                                                                     (lptDI54E lptD&#226;&#338;&#172;D57&#226;&#338;&#172;K60). cdsA7 cdsA9 able to <br />                                                                                     cross-suppress lptDI54E lptD&#226;&#338;&#172;D57&#226;&#338;&#172;K60, regardless lptD <br />                                                                                     background isolated (Fig. 2). This demonstrates that <br />                                                                                     cdsA suppressors allele specific increase <br />                                                                                     vancomycin resistance generalized manner. <br />                                                                                         Mutants beta-barrel assembly machinery (Bam) increase <br /> FIG 1 Vancomycin-resistant lptD suppressors map cdsA, gene respon- <br /> sible conversion phosphatidic acid CDP-diacylglycerol. (A) The        sensitivity vancomycin impaired assembly OM pro- <br /> inner membrane CdsA protein predicted seven transmembrane                teins (37, 38). The &#226;&#338;&#172;bamE::kan allele null <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>mutation</span> re- <br /> domains (33). Vancomycin-resistant cdsA suppressors mapped mainly            moves BamE, lipoproteins Bam complex. Strains <br /> these domains. Mutations V16G, L33P, &#226;&#338;&#172;56 &#226;&#8364;&#8220; 62, Y156D, L236Q                carrying &#226;&#338;&#172;bamE::kan allele exhibit mild OM protein assembly <br /> isolated lptDV52E-I54E background. I203S isolated lptDI54E back- <br />                                                                                     defects display increased sensitivity vancomycin (37). <br /> ground, R263L isolated lptD&#226;&#338;&#172;D57&#226;&#338;&#172;K60 background. The muta- <br /> tions enclosed circles, I203S (cdsA7) R263L (cdsA9), characterized       When cdsA7 cdsA9 introduced &#226;&#338;&#172;bamE::kan strain, <br /> further study. (B) Biosynthetic pathway major phospholipids          vancomycin resistance increased (Fig. 3A). This suggests that <br /> Escherichia coli. Protein names boxes, arrow   cdsA7 cdsA9 increase vancomycin resistance general man- <br /> signifies step act.                                               ner strain compromised OM. <br />                                                                                         To test cdsA7 cdsA9 increase resistance vanco- <br />                                                                                     mycin wild-type strains, evaluated mutants oth- <br /> mycin differing degrees. Spontaneously arising vancomycin-                       erwise wild-type background. On LB plates supplemented with <br /> resistant suppressors isolated LB medium supplemented                       200 mg/liter vancomycin, cdsA7 cdsA9 mutants able to <br /> with 120 mg/liter vancomycin.                                                       survive better wild type (Fig. 3B). This suggests the <br />     With strains carrying mutations, survivors arose                  cdsA7 cdsA9 strains altered way protects cell <br /> at frequency 10&#226;&#171;&#186;8, suppressors analyzed mapped                   vancomycin. Previously isolated cdsA mutants showed in- <br /> essential gene, cdsA. CdsA inner membrane protein respon-                     creased resistance erythromycin strain background <br /> sible conversion PA CDP-diacylglycerol GPL bio-                    tested, effect mutations sensitivity other <br /> synthesis (6, 15, 32). Figure 1A shows location different                antibiotics reported (16). Using disk diffusion assays, <br /> cdsA mutations predicted CdsA structure (33). Most muta-                       checked cdsA mutants battery other <br /> tions map transmembrane helices, affect                         antibiotics. We slight increase resistance eryth- <br /> protein stability membrane. Amino acid changes I203S                         romycin drug tested (norfloxacin, para- <br /> (cdsA7) R263L (cdsA9) chosen representative exam-                       quat, cetyltrimethylammonium bromide, bacitracin, poly- <br /> ples study located different parts              myxin B). Strikingly, however, cdsA mutations <br /> protein, unlike cdsA suppressors isolated,                    cause clear increase sensitivity ampicillin, moxalactam, genta- <br /> they exhibit growth defects.                                             micin, rifampin. <br />  <br />  <br /> 3216    jb.asm.org                                                                                                                     Journal Bacteriology <br />                                                                                                                 Phosphatidic Acid Increases Vancomycin Resistance <br />  <br />  <br />  <br />  <br /> FIG 2 Partial loss-of-function cdsA mutants pgsA444 suppress vancomycin sensitivity lptD mutants. (A) Efficiency plating assay (EOP) LB agar <br /> plates supplemented 70 mg/liter, 140 mg/liter, 150 mg/liter vancomycin grown 37&#194;&#176;C overnight. Suppression shown lptDI54E. Note the <br /> chromosomal lptD allele &#226;&#338;&#172;lptD, lptD allele listed EOP present pET2342::lptD, leaky T7 promoter. T7 RNA polymerase not <br /> present strains. (B) EOPs described panel A. Suppression shown lptD&#226;&#338;&#172;D57&#226;&#338;&#172;K60. <br />  <br />  <br />  <br />    Changes phospholipid content confer vancomycin resis-                      shown accumulate PA (15, 16). We expect a <br /> tance. The enzymes intermediates involved GPL biosyn-                     concomitant decrease CDP-diacylglycerol levels cdsA mu- <br /> thesis identified lab Eugene Kennedy (Fig. 1B).                   tants, result decrease acidic GPL biosynthesis, <br /> Strains harboring partial loss-of-function mutations cdsA                CDP-diacylglycerol higher affinity PssA for <br />                                                                                  PgsA (15, 16). <br />                                                                                      We hypothesized change phospholipid content in <br />                                                                                  OM result increased resistance vancomycin, <br />                                                                                  wondered accumulation PA or <br />                                                                                  decrease acidic GPLs. To distinguish two <br />                                                                                  possibilities, introduced pgsA444, partially inactivating mu- <br />                                                                                  tation phosphatidylglycerophosphate synthetase (39), as <br />                                                                                  &#226;&#338;&#172;clsA::kan Keio collection (34) lptD <br />                                                                                  mutant strains (note unable construct an <br />                                                                                  lptD&#226;&#338;&#172;D57&#226;&#338;&#172;K60 &#226;&#338;&#172;clsA::kan double mutant, data not <br />                                                                                  shown). ClsA responsible significant portion CL pro- <br />                                                                                  duced cell, cls mutants shown lower <br />                                                                                  levels CL (24, 40). We pgsA444 partially suppresses <br />                                                                                  vancomycin sensitivity lptD mutants, &#226;&#338;&#172;clsA::kan <br />                                                                                  does (Fig. 2). This suggests increase negatively <br />                                                                                  charged PA species membrane decrease acidic <br />                                                                                  GPLs responsible increase vancomycin resistance. It <br />                                                                                  likely pgsA444 partially suppresses vancomycin <br />                                                                                  sensitivity lptD mutants does accumulate as <br />                                                                                  PA cdsA mutants. We discovered pgsA444 <br />                                                                                  confers increased vancomycin resistance wild- <br />                                                                                  type background (Fig. 3B). The &#226;&#338;&#172;clsA::kan allele does confer <br />                                                                                  increased resistance fact renders cells sensitive to <br />                                                                                  vancomycin. <br />                                                                                      cdsA mutants accumulate PA. To confirm genetic results, <br /> FIG 3 cdsA mutants, pgsA444, plsC overexpression increase resistance      investigated GPL content strains harboring cdsA7, cdsA9, <br /> vancomycin. (A) Partial loss-of-function cdsA mutants suppress vancomy-      pgsA444 mutations. GPLs extracted cells <br /> cin sensitivity cell envelope mutants. EOPs strains supplemented     using acidic Bligh Dyer method (23, 24). Lipids sep- <br /> with 120 mg/liter vancomycin. (B) Partial loss-of-function cdsA mutants      arated quantified Agilent quadrupole time flight (Q- <br /> pgsA444 increase vancomycin resistance wild-type cells. EOPs strains <br /> supplemented 200 mg/liter vancomycin. (C) Overproduction PlsC <br />                                                                                  TOF) liquid chromatography-mass spectrometry (LC-MS). Dis- <br /> increases vancomycin resistance wild-type cells. EOPs strains supple-      tinct PA species summed, sum normalized to <br /> mented 200 mg/liter vancomycin 0.1 mM IPTG.                             total PS content strain. PS levels did change signifi- <br />  <br />  <br /> September 2014 Volume 196 Number 18                                                                                                            jb.asm.org 3217 <br />  Sutterlin et al. <br />  <br />  <br />  <br />  <br /> FIG 4 PA accumulates cdsA mutants pgsA444 plsC overexpressed. PA quantified Q-TOF LC-MS. The average fold differences all <br /> PA species &#226;&#171;&#190; standard errors experiments shown (top). The changes PA/PS reported &#226;&#171;&#190; standard errors experiments (bottom). <br /> (B) CL levels decrease cdsA mutants pgsA444. CL quantified Q-TOF LC-MS. The average fold differences CL species &#226;&#171;&#190; standard errors from <br /> three experiments shown (top). The changes CL/PS reported &#226;&#171;&#190; standard errors experiments (bottom). <br />  <br />  <br />  <br />  <br /> cantly strains tested. We PA accumulates cdsA7,           DISCUSSION <br /> cdsA9, pgsA444 strains (Fig. 4A). We hypothesize                  We OM permeability defects associated lptD <br /> cdsA9 mutation compromises CdsA function severely                    mutants suppressed partial loss-of-function mutations in <br /> does cdsA7 PA levels higher cdsA9 mutant.              essential gene, cdsA. The results selection increased <br /> This correlates suppression seen lptD mutants                vancomycin resistance striking: suppression rare the <br /> (Fig. 2); higher vancomycin concentrations, cdsA9 suppresses                seven suppressors analyzed altered seven different codons single <br /> vancomycin sensitivity better does cdsA7, suggesting                 essential gene. These cdsA mutations increase vancomycin resistance <br /> greater PA accumulation results better suppression. The                     lptD mutants, bam mutants, wild-type <br /> pgsA444 mutant accumulates PA cdsA7 or <br />                                                                                strain. However, markedly increase resistance any <br /> cdsA9 mutant does, correlates fact pgsA444 <br />                                                                                antibiotic tested. Thus, suppressors gene path- <br /> does suppress vancomycin sensitivity (Fig. 2). The <br />                                                                                way specific; rather, drug specific. Using classic genetic meth- <br /> cdsA7, cdsA9, pgsA444 mutants decreased CL levels <br />                                                                                ods Q-TOF LC-MS, increasing PA different <br /> compared wild type (Fig. 4B), likely PA is <br /> accumulating expense CL.                                             ways (using cdsA suppressors, pgsA444 mutation, overex- <br />    Overexpression plsC increases PA increases vancomy-                  pression plsC) increases vancomycin resistance E. coli. We con- <br /> cin resistance. We shown loss-of-function mutations               clude vancomycin resistance increases consequence PA <br /> the phospholipid biosynthesis pathway cause accumulation                  accumulation. <br /> of PA increase vancomycin resistance. We wondered                          It difficult speculate accumulation PA results in <br /> overproduction protein directly upstream PA synthesis,               increased vancomycin resistance primarily not <br /> PlsC, cause accumulation PA,                    know vancomycin enters periplasm bind target, <br /> increase vancomycin resistance. We used plsC ASKA clone                 lipid II (13, 14). When lpt compromised, LPS trans- <br /> overexpress plsC wild-type background (35).                    port assembly slow, allowing PLs flip outer <br /> When plsC expression induced 0.1 mM IPTG, increase                  leaflet OM (20). These PLs self-associate form rafts PL <br /> in vancomycin resistance observed (Fig. 3C). Q-TOF LC-MS                    bilayer OM allow diffusion hydrophobic mole- <br /> confirms level induction plsC expression results            cules periplasm (41). It obvious large, hydro- <br /> an accumulation PA (Fig. 4A). These results confirm accu-              philic molecule like vancomycin able cross mem- <br /> mulation PA increases vancomycin resistance E. coli.                     brane way. Instead, possible vancomycin enters the <br />  <br />  <br /> 3218    jb.asm.org                                                                                                                   Journal Bacteriology <br />                                                                                                         Phosphatidic Acid Increases Vancomycin Resistance <br />  <br />  <br />  <br /> cell &#226;&#8364;&#339;cracks&#226;&#8364;? bilayer disordered LPS-PL junctions       vated (9). Accumulation PA way OM <br /> (41). This predicts lpt mutant vancomycin           modified increase resistance antibiotics. <br /> sensitive increase LPS-PL junctions, i.e., &#226;&#8364;&#339;cracks&#226;&#8364;?          The emergence multidrug-resistant bacteria serious, life- <br /> the membrane.                                                             threatening problem (47). Owing barrier properties the <br />     Ganong et al. shown cdsA mutants accumulate PA           OM, problem especially acute Gram-negative patho- <br /> both IM OM (15). While true roughly       gens. Many antibiotics work infections caused by <br /> four times PA IM OM, PA concentrations           Gram-positive bacteria fail used Gram-negative bac- <br /> in OM 10-fold higher cdsA mutant          teria effectiveness OM barrier, not <br /> in wild type (15). Because vancomycin needs traverse          know modify agents reach their <br /> the OM act target periplasm, unlikely      targets. A better understanding permeability properties of <br /> PA accumulation IM contributing vancomycin resis-            OM modified help facilitate the <br /> tance. Rather, suggest PA accumulation OM im-              design new antibacterials. <br /> pedes efficient entry vancomycin periplasm reduc- <br /> ing size altering properties &#226;&#8364;&#339;cracks.&#226;&#8364;? Since PA         ACKNOWLEDGMENTS <br /> accumulation increases vancomycin resistance other-            This material based work supported National Science <br /> wise wild-type cell well, propose presence small      Foundation Graduate Research Fellowship Program grant no. <br /> amount PLs outer leaflet produce &#226;&#8364;&#339;cracks&#226;&#8364;? OM        DGE1148900 (H.A.S.). This work supported National <br />                                                                           Institute General Medical Sciences grant GM34821 (T.J.S.) NIH/ <br /> of wild-type strains. <br />                                                                           NIGMS grant P50 GM071508 (S.Z.). <br />     It possible accumulation PA OM reduces              Any opinions, findings, conclusions recommendations ex- <br /> the size &#226;&#8364;&#339;cracks&#226;&#8364;? increasing LPS assembly, increasing        pressed material author(s) necessarily <br /> activity Lpt system, decreasing PLs         reflect views National Science Foundation. <br /> the outer leaflet increasing activity PldA Mla trans-        We thank Silhavy lab, present past, helpful discussion and <br /> port (20). However, PA really intermediate PL       comments. We thank Mark Rose Zemer Gitai helpful discus- <br /> biosynthesis, present normally low levels,          sion critical analysis. We grateful Thomas Bartlett help with <br /> think unlikely.                                              imaging cell size analysis Jaclyn Schwalm strains. <br />     We propose accumulation PA increases vancomycin <br />                                                                           REFERENCES <br /> resistance altering biophysical properties OM. PA <br />                                                                            1. Silhavy TJ, Kahne D, Walker S. 2010. The bacterial cell envelope. Cold <br /> lacks head group increases membrane curvature, <br />                                                                               Spring Harb. Perspect. Biol. 2:a000414. http://dx.doi.org/10.1101 <br /> and affect size properties &#226;&#8364;&#339;cracks&#226;&#8364;? (42, 43).        /cshperspect.a000414. <br /> In addition increased membrane curvature, PA increases              2. Nikaido H. 2003. Molecular basis bacterial outer membrane permea- <br /> negative charge OM, increase vancomycin                   bility revisited. Microbiol. Mol. Biol. Rev. 67:593&#226;&#8364;&#8220; 656. http://dx.doi.org <br /> association OM (43). While counterintui-               /10.1128/MMBR.67.4.593-656.2003. <br />                                                                            3. Bos MP, Robert V, Tommassen J. 2007. Biogenesis gram-negative <br /> tive, studies shown CL microdomains interact                  bacterial outer membrane. Annu. Rev. Microbiol. 61:191&#226;&#8364;&#8220;214. http://dx <br /> with CAMPs prevent interacting targets,              .doi.org/10.1146/annurev.micro.61.080706.093245. <br /> thereby increasing resistance (44, 45).                                    4. Raetz CR, Whitfield C. 2002. Lipopolysaccharide endotoxins. Annu. Rev. <br />     Changes GPL composition previously shown                  Biochem. 71:635&#226;&#8364;&#8220;700. http://dx.doi.org/10.1146/annurev.biochem.71 <br />                                                                               .110601.135414. <br /> alter resistance susceptibility CAMPs antibiotics (9,           5. Ruiz N, Kahne D, Silhavy TJ. 2009. Transport lipopolysaccharide <br /> 46). In Staphylococcus aureus, fmtC lysC mutations,                 cell envelope: long road discovery. Nat. Rev. Microbiol. <br /> reduce lysyl-PG, increase resistance moenomycin vanco-                 7:677&#226;&#8364;&#8220; 683. http://dx.doi.org/10.1038/nrmicro2184. <br /> mycin (46). The increased net negative charge mutant              6. Dowhan W. 2013. A retrospective: use Escherichia coli vehicle to <br />                                                                               study phospholipid synthesis function. Biochim. Biophys. Acta 1831: <br /> membranes increase interaction drugs prevent               471&#226;&#8364;&#8220; 494. http://dx.doi.org/10.1016/j.bbalip.2012.08.007. <br /> them interacting targets. In Salmonella Typhimu-           7. Zhang YM, Rock CO. 2008. Membrane lipid homeostasis bacteria. <br /> rium, PhoPQ responds acidic pH CAMPs                     Nat. Rev. Microbiol. 6:222&#226;&#8364;&#8220;233. http://dx.doi.org/10.1038/nrmicro1839. <br /> increasing levels cardiolipin palmitoylated acyl-PG (9).            8. Wiesner J, Vilcinskas A. 2010. Antimicrobial peptides: ancient arm of <br />                                                                               human immune system. Virulence 1:440 &#226;&#8364;&#8220; 464. http://dx.doi.org/10 <br /> This increase GPLs correlates increase resis-             .4161/viru.1.5.12983. <br /> tance CAMPs helps evade host immune (9).                9. Dalebroux ZD, Matamouros S, Whittington D, Bishop RE, Miller SI. <br /> There speculation resistance increased             2014. PhoPQ regulates acidic glycerophospholipid content Salmo- <br /> membrane curvature CAMP association CL mi-                    nella Typhimurium outer membrane. Proc. Natl. Acad. Sci. U. S. A. 111: <br />                                                                               1963&#226;&#8364;&#8220;1968. http://dx.doi.org/10.1073/pnas.1316901111. <br /> crodomains, occluding CAMP interactions tar-           10. Needham BD, Trent MS. 2013. Fortifying barrier: impact lipid <br /> gets (9, 44, 45).                                                             A remodelling bacterial pathogenesis. Nat. Rev. Microbiol. 11:467&#226;&#8364;&#8220; <br />     The mode resistance shown inherently different            481. http://dx.doi.org/10.1038/nrmicro3047. <br /> that regulated PhoPQ system. When turned on,        11. Murata T, Tseng W, Guina T, Miller SI, Nikaido H. 2007. PhoPQ- <br />                                                                               mediated regulation produces robust permeability barrier the <br /> palmitoylation PG increases, results reduction polar-        outer membrane Salmonella enterica serovar Typhimurium. J. Bacte- <br /> ity increase hydrophobicity saturation mem-          riol. 189:7213&#226;&#8364;&#8220;7222. http://dx.doi.org/10.1128/JB.00973-07. <br /> brane (9, 10). In work, overall increase PA, regard-   12. Cronan JE. 2003. Bacterial membrane lipids: stand? Annu. <br /> less chain <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> saturation state (see Fig. S3              Rev. Microbiol. 57:203&#226;&#8364;&#8220;224. http://dx.doi.org/10.1146/annurev.micro.57 <br />                                                                               .030502.090851. <br /> supplemental material). As result accumulation PA,    13. Pootoolal J, Neu J, Wright GD. 2002. Glycopeptide antibiotic resistance. <br /> show reduction CL levels. This contrasts increase       Annu. Rev. Pharmacol. Toxicol. 42:381&#226;&#8364;&#8220; 408. http://dx.doi.org/10.1146 <br /> CL levels observed Salmonella PhoPQ acti-               /annurev.pharmtox.42.091601.142813. <br />  <br />  <br />  <br /> September 2014 Volume 196 Number 18                                                                                                     jb.asm.org 3219 <br />  Sutterlin et al. <br />  <br />  <br />  <br /> 14. Barna JC, Williams DH. 1984. The structure mode action                    coli K-12 in-frame, single-gene <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span>knockout</span> mutants: Keio collection. <br />     glycopeptide antibiotics vancomycin group. Annu. Rev. Microbiol.             Mol. Syst. Biol. 2:2006.0008. http://dx.doi.org/10.1038/msb4100050. <br />     38:339 &#226;&#8364;&#8220;357. http://dx.doi.org/10.1146/annurev.mi.38.100184.002011.           35.   Kitagawa M, Ara T, Arifuzzaman M, Ioka-Nakamichi T, Inamoto E, <br /> 15. Ganong BR, Leonard JM, Raetz CR. 1980. Phosphatidic acid accumula-                  Toyonaga H, Mori H. 2005. Complete set ORF clones Escherichia <br />     tion membranes Escherichia coli mutants defective CDP-                 coli ASKA library (a complete set E. coli K-12 ORF archive): unique <br />     diglyceride synthetase. J. Biol. Chem. 255:1623&#226;&#8364;&#8220;1629.                               resources biological research. DNA Res. 12:291&#226;&#8364;&#8220;299. http://dx.doi.org <br /> 16. Ganong BR, Raetz CR. 1982. Massive accumulation phosphatidic acid                /10.1093/dnares/dsi012. <br />     conditionally lethal CDP-diglyceride synthetase mutants cytidine       36.   Yao Z, Davis RM, Kishony R, Kahne D, Ruiz N. 2012. Regulation cell <br />     auxotrophs Escherichia coli. J. Biol. Chem. 257:389 &#226;&#8364;&#8220;394.                        size response <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> availability fatty acid biosynthesis Esch- <br /> 17. Casadaban MJ. 1976. Transposition fusion lac genes selected           erichia coli. Proc. Natl. Acad. Sci. U. S. A. 109:E2561&#226;&#8364;&#8220;E2568. http://dx.doi <br />     promoters Escherichia coli using bacteriophage lambda Mu. J. Mol.            .org/10.1073/pnas.1209742109. <br />     Biol. 104:541&#226;&#8364;&#8220;555. http://dx.doi.org/10.1016/0022-2836(76)90119-4.            37.   Rigel NW, Schwalm J, Ricci DP, Silhavy TJ. 2012. BamE modulates the <br /> 18. Ruiz N, Gronenberg LS, Kahne D, Silhavy TJ. 2008. Identification             Escherichia coli beta-barrel assembly machine component BamA. J. Bacte- <br />     inner-membrane proteins required transport lipopolysaccha-               riol. 194:1002&#226;&#8364;&#8220;1008. http://dx.doi.org/10.1128/JB.06426-11. <br />     ride outer membrane Escherichia coli. Proc. Natl. Acad. Sci.        38.   Ricci DP, Hagan CL, Kahne D, Silhavy TJ. 2012. Activation the <br />     U. S. A. 105:5537&#226;&#8364;&#8220;5542. http://dx.doi.org/10.1073/pnas.0801196105.                  Escherichia coli beta-barrel assembly machine (Bam) required essen- <br /> 19. Silhavy TJ, Berman ML, Enquist LW. 1984. Experiments gene                      tial components interact properly substrate. Proc. Natl. Acad. Sci. <br />     fusions. Cold Spring Harbor Laboratory Press, Plainview, NY.                        U. S. A. 109:3487&#226;&#8364;&#8220;3491. http://dx.doi.org/10.1073/pnas.1201362109. <br /> 20. Malinverni JC, Silhavy TJ. 2009. An ABC transport maintains       39.   Nishijima M, Raetz CR. 1979. Membrane lipid biogenesis Escherichia <br />     lipid asymmetry gram-negative outer membrane. Proc. Natl. Acad.              coli: identification genetic loci phosphatidylglycerophosphate syn- <br />     Sci. U. S. A. 106:8009 &#226;&#8364;&#8220; 8014. http://dx.doi.org/10.1073/pnas.0903229106.           thetase construction mutants lacking phosphatidylglycerol. J. Biol. <br /> 21. Cherepanov PP, Wackernagel W. 1995. Gene disruption Escherichia                  Chem. 254:7837&#226;&#8364;&#8220;7844. <br />     coli: TcR KmR cassettes option Flp-catalyzed excision      40.   Shibuya I, Miyazaki C, Ohta A. 1985. Alteration phospholipid com- <br />     antibiotic-resistance determinant. Gene 158:9 &#226;&#8364;&#8220;14. http://dx.doi.org            position combined defects phosphatidylserine cardiolipin syn- <br />     /10.1016/0378-1119(95)00193-A.                                                      thases physiological consequences Escherichia coli. J. Bacteriol. 161: <br /> 22. Sliusarenko O, Heinritz J, Emonet T, Jacobs-Wagner C. 2011. High-                   1086 &#226;&#8364;&#8220;1092. <br />     throughput, subpixel precision analysis bacterial morphogenesis        41.   Nikaido H. 2005. Restoring permeability barrier function outer mem- <br />     intracellular spatio-temporal dynamics. Mol. Microbiol. 80:612&#226;&#8364;&#8220; 627.                brane. Chem. Biol. 12:507&#226;&#8364;&#8220;509. http://dx.doi.org/10.1016/j.chembiol <br />     http://dx.doi.org/10.1111/j.1365-2958.2011.07579.x.                                 .2005.05.001. <br /> 23. Bligh EG, Dyer WJ. 1959. A rapid method total lipid extraction         42.   McMahon HT, Gallop JL. 2005. Membrane curvature mechanisms <br />                                                                                         dynamic cell membrane remodelling. Nature 438:590 &#226;&#8364;&#8220;596. http://dx <br />     purification. Can. J. Physiol. Pharmacol. 37:911&#226;&#8364;&#8220;917. http://dx.doi.org/10 <br />                                                                                         .doi.org/10.1038/nature04396. <br />     .1139/y59-099. <br />                                                                                   43.   Kooijman EE, Burger KN. 2009. Biophysics function phosphatidic <br /> 24. Tan BK, Bogdanov M, Zhao J, Dowhan W, Raetz CR, Guan Z. 2012. <br />                                                                                         acid: molecular perspective. Biochim. Biophys. Acta 1791:881&#226;&#8364;&#8220; 888. http: <br />     Discovery cardiolipin synthase utilizing phosphatidylethanolamine <br />                                                                                         //dx.doi.org/10.1016/j.bbalip.2009.04.001. <br />     phosphatidylglycerol substrates. Proc. Natl. Acad. Sci. U. S. A. <br />                                                                                   44.   Tran TT, Panesso D, Mishra NN, Mileykovskaya E, Guan Z, Munita <br />     109:16504 &#226;&#8364;&#8220;16509. http://dx.doi.org/10.1073/pnas.1212797109. <br />                                                                                         JM, Reyes J, Diaz L, Weinstock GM, Murray BE, Shamoo Y, Dowhan <br /> 25. Grabowicz M, Yeh J, Silhavy TJ. 2013. Dominant negative lptE mutation <br />                                                                                         W, Bayer AS, Arias CA. 2013. Daptomycin-resistant Enterococcus faecalis <br />     supports role LptE plug LptD barrel. J. Bacteriol.           diverts antibiotic molecule division septum remodels cell <br />     195:1327&#226;&#8364;&#8220;1334. http://dx.doi.org/10.1128/JB.02142-12.                               membrane phospholipids. mBio 4(4):e00281&#226;&#8364;&#8220;13. http://dx.doi.org/10 <br /> 26. Freinkman E, Chng SS, Kahne D. 2011. The complex inserts lipo-                 .1128/mBio.00281-13. <br />     polysaccharide bacterial outer membrane forms two-protein          45.   Hartmann M, Berditsch M, Hawecker J, Ardakani MF, Gerthsen D, <br />     plug-and-barrel. Proc. Natl. Acad. Sci. U. S. A. 108:2486 &#226;&#8364;&#8220;2491. http://dx          Ulrich AS. 2010. Damage bacterial cell envelope antimicrobial <br />     .doi.org/10.1073/pnas.1015617108.                                                   peptides gramicidin S PGLa revealed transmission scanning <br /> 27. Freinkman E, Okuda S, Ruiz N, Kahne D. 2012. Regulated assembly                  electron microscopy. Antimicrob. Agents Chemother. 54:3132&#226;&#8364;&#8220;3142. <br />     transenvelope protein complex required lipopolysaccharide ex-               http://dx.doi.org/10.1128/AAC.00124-10. <br />     port. Biochemistry 51:4800 &#226;&#8364;&#8220; 4806. http://dx.doi.org/10.1021/bi300592c.       46.   Nishi H, Komatsuzawa H, Fujiwara T, McCallum N, Sugai M. 2004. <br /> 28. Villa R, Martorana AM, Okuda S, Gourlay LJ, Nardini M, Sperandeo P,                 Reduced content lysyl-phosphatidylglycerol cytoplasmic mem- <br />     Deho G, Bolognesi M, Kahne D, Polissi A. 2013. The Escherichia coli Lpt             brane affects susceptibility moenomycin, vancomycin, genta- <br />     transenvelope protein complex lipopolysaccharide export assembled            micin, antimicrobial peptides, Staphylococcus aureus. Antimicrob. <br />     conserved structurally homologous domains. J. Bacteriol. 195:1100 &#226;&#8364;&#8220;             Agents Chemother. 48:4800 &#226;&#8364;&#8220; 4807. http://dx.doi.org/10.1128/AAC.48.12 <br />     1108. http://dx.doi.org/10.1128/JB.02057-12.                                        .4800-4807.2004. <br /> 29. Sampson BA, Misra R, Benson SA. 1989. Identification character-           47.   Dortet L, Poirel L, Nordmann P. 2014. Worldwide dissemination the <br />     ization new gene Escherichia coli K-12 involved outer membrane           NDM-type carbapenemases gram-negative bacteria. Biomed. Res. Int. <br />     permeability. Genetics 122:491&#226;&#8364;&#8220;501.                                                 2014:249856. http://dx.doi.org/10.1155/2014/249856. <br /> 30. Sperandeo P, Pozzi C, Deho G, Polissi A. 2006. Non-essential KDO              48.   Wu T, McCandlish AC, Gronenberg LS, Chng SS, Silhavy TJ, Kahne D. <br />      <br /> </body></html>