<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                          REVIEW ARTICLE <br />                                                                                                                             published: 29 April 2014 <br />                                                                                                                      doi: 10.3389/fphys.2014.00162 <br />  <br />  <br />  <br /> An approach exploring interaction two <br /> proteins vivo <br /> Hiroshi Qadota * Guy M. Benian * <br /> Department Pathology, Emory University, Atlanta, GA, USA <br />  <br />  <br />  <br /> Edited by:                             We strategy exploring function protein-protein interactions striated <br /> Aikaterini                             muscle vivo. We experience using strategy study interaction <br /> Kontrogianni-Konstantopoulos, <br /> University Maryland, USA <br />                                        UNC-112 (kindlin) PAT-4 (integrin linked kinase). Random mutagenesis used <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                        generate collection mutants screened lack binding gain binding <br /> Reviewed by: <br /> Marion L. Greaser, University       using yeast 2-hybrid assay. The mutant proteins expressed transgenic <br /> Wisconsin-Madison, USA                 C. elegans determine ability localize sarcomere. We emphasize two <br /> Jim Du, University Maryland, USA    advantages strategy: (1) studying interaction protein A protein B, <br /> *Correspondence:                       protein A interact multiple proteins, (2) explores function an <br /> Hiroshi Qadota, Department of <br />                                        interaction absence of, reduced level of, protein obtained <br /> Pathology, Emory University, <br /> Whitehead Biomedical Research          <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>null mutants</span> knockdown RNAi. We propose method generalized <br /> Building Room 165, Atlanta,            studying meaning protein-protein interaction muscle which <br /> GA 30322, USA                          transgenic animals generated muscles imaged. <br /> e-mail: hkadota@emory.edu; <br /> Guy M. Benian, Department           Keywords: protein-protein interactions, yeast 2-hybrid, C. elegans, transgenics, kindlin, ILK, protein localization <br /> Pathology, Emory University, <br /> Whitehead Biomedical Research <br /> Building Room 105E, Atlanta, <br /> GA 30322, USA <br /> e-mail: pathgb@emory.edu <br />  <br />  <br /> BACKGROUND&#226;&#8364;&#8221;UNC-112 (KINDLIN) AND PAT-4 (ILK)                                       Our laboratory helped discover interacting network of <br /> The model genetic C. elegans excellent platform                   proteins extend integrin muscle cell membrane <br /> for learning new information striated muscle, including                  inward myosin M-lines, actin dense bodies <br /> new components sarcomere, requirements assem-                       (Qadota Benian, 2010; Warner et al., 2011, 2013; Lecroisey <br /> bly maintenance sarcomere, mechanisms                    et al., 2013). One component attachment structures is <br /> regulation contraction/relaxation cycle (Waterston, 1988;               UNC-112 (kindlins mammals). UNC-112 con- <br /> Moerman Fire, 1997; Moerman Williams, 2006; Qadota                     served four-protein complex associates cytoplasmic <br /> and Benian, 2010). Because evolutionary conservation                 tail integrin. Although kindlin discovered humans <br /> sarcomere structure components, studies C. elegans                1994 (Wick et al., 1994), description of <br /> have relevance understanding inherited cardiomy-                 UNC-112 2000 (Rogalski et al., 2000) learned that <br /> opathies, mutations sarcomere                      kindlins role integrin adhesion complexes. In contrast <br /> proteins (Benian Epstein, 2011; Epstein Benian, 2012).                 C. elegans, kindlin (UNC-112), humans have <br /> In C. elegans myofibrils organized                 kindlins, encoded separate gene (Meves et al., <br /> filament organizing centers (M-lines) filament attach-                2009). Inherited mutations kindlin-1 result skin <br /> ment sites (dense bodies analogous Z-disks) anchored                 disease (Kindler syndrome) (Siegel et al., 2003), mutations in <br /> the muscle cell membrane. These integrin-based muscle adhe-                    kindlin-3 result severe dysfunction platelets leuko- <br /> sion sites ideal models costameres vertebrate striated              cytes (leukocyte adhesion deficiency type III) (Moser et al., 2009; <br /> muscle focal adhesions non-muscle cells. Many compo-                    Svensson et al., 2009). Kindlin-2 expressed heart and <br /> nents C. elegans myofibrils attachment structures                 localized intercalated disks costameres. Loss function <br /> have defined. Most identified <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>mutation</span>&#226;&#8364;&#8221;                kindlin-2 zebrafish results severe abnormalities heart <br /> most falling phenotypic classes. In class,             development function (Dowling et al., 2008). <br /> uncoordinated &#226;&#8364;&#339;Unc&#226;&#8364;? class, mutation                        In embryonic muscle, proper localization UNC-112 inte- <br /> &#226;&#710;&#188;40 genes, animals develop adults slow mov-                  grin adhesion sites requires PAT-4 (the nematode ILK ortholog) <br /> ing paralyzed (Waterston et al., 1980; Zengel Epstein,                  (Rogalski et al., 2000); converse true: proper local- <br /> 1980). The second class, called &#226;&#8364;&#339;Pats&#226;&#8364;? (paralyzed arrested two-             ization PAT-4, requires UNC-112 (Mackinnon et al., 2002). <br /> fold) result mutation 20 genes; embryos                      We sought understand molecular basis require- <br /> not eggshell elongation arrests two-fold                ment. We used cytoplasmic tail PAT-3 (&#206;&#178;-integrin) bait to <br /> embryonic stage (Williams Waterston, 1994; Meissner et al.,                screen yeast 2-hybrid library C. elegans cDNAs, recovered <br /> 2009).                                                                         interacting protein, UNC-112 (Qadota et al., 2012). <br />  <br />  <br />  <br /> www.frontiersin.org                                                                                                      April 2014 | Volume 5 | Article 162 | 1 <br />  Qadota Benian                                                                                                    Mutational analysis protein interactions <br />  <br />  <br />  <br />  <br /> This interaction confirmed ways: (1) glutathione                    et al., 2011) preliminary crystal structure (Lee et al., 2011) <br /> agarose beads coated GST-PAT-3, GST, able                    100 residue PH domain human kindlin-2, crystal <br /> to pull UNC-112 worm lysate; (2) similar beads                  structure PH domain mouse kindlin-1 (Yates et al., <br /> were able bind purified bacterially expressed MBP-UNC-                   2012b). Recently, SAXS analysis mouse kindlin-3 been <br /> 112-His. When C-terminal half UNC-112 used                       reported (Yates et al., 2012a). The proposed model suggests that <br /> screen 2-hybrid library, positive clones cDNAs encod-             halves FERM-M (aka F2 F3) split PH <br /> ing UNC-112. Additional 2-hybrid vitro binding data                     domain, physically other. Since halves <br /> indicate N-terminal C-terminal halves UNC-112                  N- C-terminal halves molecules, SAXS <br /> interact other. By 2-hybrid vitro bind-                 data compatible &#226;&#8364;&#339;closed conformation&#226;&#8364;? model. <br /> ing assays, PAT-4 does interact cytoplasmic tail of <br /> PAT-3. PAT-4 shown interact N-terminal half                 EVALUATION OF THE MEANING OF PROTEIN-PROTEIN <br /> UNC-112 (Mackinnon et al., 2002) (Figure 1). All data led                INTERACTION In vivo <br /> us hypothesize UNC-112 exists closed inactive,                  If model correct, reasoned UNC-112 mutant <br /> open active conformations; binding PAT-4 UNC-                   bind PAT-4, localize muscle adhesion <br /> 112 N-terminal half results opening UNC-112,                   sites vivo. We screened UNC-112 mutant protein that <br /> consequent binding UNC-112/PAT-4 complex cyto-                   bind PAT-4 yeast 2-hybrid assay, tested for <br /> plasmic tail PAT-3 (Qadota et al., 2012) (Figure 2). In support             localization generating analyzing transgenic worms. <br /> of model, demonstrate competition                    Error prone PCR (Cadwell Joyce, 1992) used introduce <br /> UNC-112 C-terminal half PAT-4 binding UNC-112                   mutations randomly N-terminal half UNC-112, the <br /> N-terminal half (Qadota et al., 2012).                                         portion UNC-112 known interact PAT-4. In order for <br />    Our model UNC-112 engages intramolecular inter-                     PCR introduce usual errors, fidelity Taq <br /> action form closed conformation compatible                    DNA polymerase decreased modifying reaction mixture <br /> phenomenon intramolecular masking ERM proteins                     contain Mn2+ (0.5 mM) increasing concentration of <br /> such ezrin, radixin moesin, like UNC-112 con-                     Mg2+ (e.g., 7.5 mM). Varying template DNA, <br /> tain FERM domain. In case ERM proteins, FERM                      empirically, controls extent mutagenesis. The concentration <br /> domain unable interact membrane proteins                 template inversely proportional level mutations; a <br /> bound &#226;&#710;&#188;100 residue C-terminal tail (Reczek Bretscher,               low level template results mutations. <br /> 1998). It pointed complete structure                   The PCR primers 30 nt sequence overlapped a <br /> of kindlin family protein reported. We                   linearized 2-hybrid prey plasmid containing wild type C- <br /> know structures small portions kindlins;             terminal half UNC-112. A mixture PCR fragment and <br /> NMR structure &#226;&#710;&#188;100 residue N-terminus (or Fo domain)                    linearized prey, transformed yeast harboring bait plas- <br /> of mouse kindlin-1 (Goult et al., 2009), NMR structure (Liu                 mid containing cytoplasmic tail PAT-3. Generation of <br />                                                                                full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> UNC-112 prey clones occurred recombination in <br />                                                                                yeast cells, 30 bp overlapping sequence (Takita <br />  <br />  <br />  <br />  <br />  FIGURE 1 | Schematic representation UNC-112 indicating predicted <br />  domains, mutation sites, interaction PAT-4. Domains each <br />  protein, indicated colored rectangles predicted PFAM, with <br />  boundaries indicated residue numbers. The black red brackets <br />  indicate minimal regions required interaction UNC-112 <br />  PAT-4, respectively. In UNC-112, D382V mutation prevents              FIGURE 2 | Conformation-based model interaction UNC-112 <br />  UNC-112 interacting PAT-4; T346A E349K mutations         integrin. Based yeast 2-hybrid, biochemical mutational <br />  prevent interaction N-terminal half UNC-112            analysis, hypothesized UNC-112 exists conformations, <br />  C-terminal half UNC-112. The blue bracket denotes 30 residue region     closed open, open form interact the <br />  (633&#226;&#8364;&#8220;663) containing 5 residues mutated singly permit            cytoplasmic tail integrin <span id='am-238' about='xsp:base' typeof='owl:Thing'>base</span> adhesion sites muscle. Further, <br />  N-terminal half UNC-112 containing T346A interact C-terminal    conversion open active form promoted binding UNC-112 <br />  half UNC-112.                                                               PAT-4. (modified Qadota et al., 2012). <br />  <br />  <br />  <br />  <br /> Frontiers Physiology | Striated Muscle Physiology                                                                      April 2014 | Volume 5 | Article 162 | 2 <br />  Qadota Benian                                                                                     Mutational analysis protein interactions <br />  <br />  <br />  <br />  <br /> et al., 1997). Transformed yeast cells placed selective     A SUPPRESSOR MUTATION APPROACH FOR FINDING <br /> media, grew indicated interaction mutag-       BINDING INTERFACES <br /> enized UNC-112 PAT-3. This step crucial eliminating       INTERMOLECULAR INTERACTION <br /> clones premature stop mutations trivial reasons.        Because structure UNC-112 (kindlin)/PAT-4 (ILK) has <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>From</span> 86 colonies, PAT-3 bait eliminated, UNC-         solved, decided suppressor mutation <br /> 112 prey plasmids isolated. These UNC-112 prey clones       approach revealing interaction surface pro- <br /> transformed separately yeast strains carrying PAT-    teins. As before, screened mutations yeast 2-hybrid <br /> 3 bait, PAT-4 bait. From 86 UNC-112 preys, 32 preys        system, tested striated muscle transgenic <br /> showed interaction PAT-3, showed interaction          nematodes. Beginning D382V UNC-112 mutant pro- <br /> with PAT-4. This clone mutations&#226;&#8364;&#8221;R73L D382V.              tein bind PAT-4, sought isolate &#226;&#8364;&#339;suppressor&#226;&#8364;? <br /> Prey clones UNC-112 harboring single amino           mutations PAT-4 allow binding D382V UNC- <br /> acid changes generated tested PAT-4 binding:        112. If successful, opportunity further <br /> D382V eliminated binding PAT-4 (Figure 1). The N-terminal          test conformational model requirement UNC-112 <br /> half UNC-112 containing D382V continued interact         bind PAT-4 order proper localization UNC-112 <br /> wild type version C-temrinal half UNC-112.                  muscle integrin adhesion sites: D382V UNC-112 can- <br />    We used heat shock promoter express wild type         localize, D382V UNC-112 presence suppressor <br /> C. elegans HA tagged versions wild type D382V UNC-112.         PAT-4 protein, localize (Figure 3B). <br /> By western blot using anti-HA, detect stable expression         Using general method described above, ran- <br /> of wild type D382V UNC-112. By immunofluorescence            domly mutagenized length PAT-4, screened PAT-4 <br /> microscopy using anti-HA, wild type HA-UNC-112           mutants bind D382V UNC-112 (Qadota et al., 2014). <br /> properly localized muscle adhesion sites, D382V HA-UNC-112         Upon screening 12,700 colonies, initially 62 positive this <br /> was localized, fulfilling expectation (Figure 3A).       interaction. Upon re-transformation, 40/62, remained positive. <br />    We reasoned, model open con-         After determining DNA sequences 37 PAT-4 clones, <br /> formation UNC-112 bind PAT-3, mutations           5 single amino acid changes, 2&#226;&#8364;&#8220;5 changes <br /> block interaction N- C-terminal halves         each. By comparing clones multiple changes, using <br /> UNC-112 (constitutively open), allow D382V UNC-112           vitro mutagenesis make clones single residue changes, <br /> properly localize (Figure 2). Using similar strategy described   total number clones missense mutations single <br /> above, random mutations introduced UNC-112 N-               unique residues totaled 9. It noteworthy entire <br /> terminal half, colonies screened yeast 2-hybrid assays    PAT-4 sequence mutagenized, mutations reside in <br /> for interacted PAT-4, failed interact     kinase domain, minimal region required interaction <br /> the UNC-112 C-terminal half. Ninety-three recominated plas-           UNC-112 (Figure 1). <br /> mids obtained, these, 87 showed interaction             Fukuda et al. (2009) reported crystal structure of <br /> PAT-4, 13/87 failed bind UNC-112 C-terminal half.       human ILK (PAT-4) kinase domain bound &#206;&#177;-parvin <br /> Two plasmids single residue changes&#226;&#8364;&#8221;T346A E349K               (PAT-6 nematodes; Lin et al., 2003). Based structure, <br /> (Figure 1). Single double mutants (with D382V) gen-          developed homology model PAT-4 kinase domain and <br /> erated length UNC-112, tested 2-hybrid: bound      located residues mutated PAT-4 suppressors. The sup- <br /> to PAT-3, T346A E349K bind PAT-4;          pressor mutations cluster 2 regions surface PAT-4: <br /> D382V added binding PAT-4 abolished. The single        6 cluster near edge beta sheet helix-1 N- <br /> double mutant UNC-112 proteins ectopically expressed        terminal subdomain, 3 cluster helices C-terminal <br /> from heat shock promoter, proteins detected            subdomain (Qadota et al., 2014). When PAT-4 substituted for <br /> anti-HA immunofluorescence. All single double mutants          ILK ILK/&#206;&#177;-parvin complex, &#206;&#177;-parvin, implication, <br /> localized muscle adhesions sites. Therefore, mutations pre-   PAT-6, did overlap cover cluster PAT-4 suppres- <br /> vent N- C-terminal interaction bypass requirement       sor residues. Therefore, cluster constitute <br /> UNC-112 bind PAT-4 proper localization UNC-112.          binding surfaces UNC-112. Moreover, PAT-4 sup- <br />    It pointed isolation specific            pressor mutations permit localization D382V UNC-112 <br /> non-binding mutants occurs high frequency prac-     integrin adhesion sites nematode muscle: co-expression of <br /> tical. In experience, average, 50 mutagenized       HA-UNC-112 D382V myc-PAT-4 wild type does show <br /> fragments premature stops, specific non-           localization, HA-UNC-112 D382V myc-PAT-4 P257L, <br /> binding mutants isolated. In cases presented here,        does localization (Qadota et al., 2014). <br /> isolatedonePAT-4non-bindingUNC-112mutantfrom32mutants <br /> that retained binding PAT-3, recovered 2 C-terminus         INTRAMOLECULAR INTERACTION <br /> non-binding mutants UNC-112 87 PAT-4 binding N-               Beginning T346A E349K UNC-112 (in N terminal <br /> terminal mutagenized fragments. A similar ratio reported     half UNC-112) bind wild type UNC-112 C ter- <br /> for types specific loss-of-function screens yeast. In     minal half, mutagenized C terminal half screened for <br /> the case yeast conditional lethals (for example, temperature       compensatory mutations permit T346A UNC-112 bind <br /> sensitive mutants) screening, genes (RHO1, MSS4, FKS1),     UNC-112 C terminal half. Nine UNC-112 C terminal half <br /> temperature sensitive mutants isolated similar ratio        mutants containing missense mutations isolated (Qadota <br /> (Homma et al., 1998; Saka et al., 2001; Okada et al., 2010).          et al. unpub. data). Three contained single missense <br />  <br />  <br /> www.frontiersin.org                                                                                        April 2014 | Volume 5 | Article 162 | 3 <br />  Qadota Benian                                                                                                        Mutational analysis protein interactions <br />  <br />  <br />  <br />  <br />                                                                                     FIGURE 3 | Continued <br />                                                                                     wild type protein A (blue), mutant protein A (purple), an <br />                                                                                     immunotag (e.g., HA, shown). These transgenic animals used two <br />                                                                                     types experiments. As indicated below, using antibody detects <br />                                                                                     immunotag, requirement interaction protein A protein <br />                                                                                     B localization protein A, assessed immunofluorescent detection <br />                                                                                     muscle. An example wild type mutant protein A UNC-112 <br />                                                                                     (kindlin) C. elegans muscle, shown. Wild type UNC-112 localizes <br />                                                                                     normally dense bodies (dots) M-lines (lines), D382V UNC-112 <br />                                                                                     does interact protein B (in case, PAT-4), fails localize <br />                                                                                     (modified Qadota et al., 2012). As indicated right, these <br />                                                                                     transgenic animals used assess mutant protein A (&#226;&#710;&#8212; and <br />                                                                                     purple) bind protein B vivo: protein extracts prepared from <br />                                                                                     type transgenic animal, immunotagged protein A (wild type, <br />                                                                                     blue, mutant, purple) immunoprecipitated (IP) anti-tag conjugated <br />                                                                                     beads, washed, eluted protein B detected immunoblot. An <br />                                                                                     example shown UNC-112/PAT-4 interaction C. elegans (modified <br />                                                                                     Qadota et al., 2012). (B) Isolation characterization mutations in <br />                                                                                     protein B restore interaction protein B mutant version of <br />                                                                                     protein A. From bottom: mutant form protein A (&#226;&#710;&#8212; purple) <br />                                                                                     does interact protein B. Random mutagenesis used generate <br />                                                                                     collection protein B mutants (indicated # ), tested by <br />                                                                                     yeast 2-hybrid assays interaction protein A&#226;&#710;&#8212; . One such <br />                                                                                     protein B mutants (# orange) interact protein A&#226;&#710;&#8212; <br />                                                                                     (denoted arrow). Transgenic animals generated express <br />                                                                                     mutant protein A&#226;&#710;&#8212; (purple), wild type protein B (yellow), or <br />                                                                                     mutant protein B# (orange). Note protein A protein B are <br />                                                                                     expressed different immunotags (e.g., HA myc, indicated). These <br />                                                                                     transgenic animals used types experiments. As indicated <br />                                                                                     below, using antibody detects immunotag protein A&#226;&#710;&#8212; , the <br />                                                                                     ability mutant protein B# restore ability protein A&#226;&#710;&#8212; localize in <br />                                                                                     muscle, assessed immunostaining. As indicated right, these <br />                                                                                     transgenic animals used assess mutant protein B (# and <br />                                                                                     orange) restore ability protein B interact mutant protein A <br />                                                                                     (&#226;&#710;&#8212; purple) vivo: protein extracts prepared type of <br />                                                                                     transgenic animal, immunotagged mutant protein A (&#226;&#710;&#8212; purple) is <br />                                                                                     immunoprecipitated (IP) anti-tag conjugated beads, washed, and <br />                                                                                     eluted protein B detected immunoblot. <br />  <br />  <br />  <br />                                                                                    mutations (S644C, N659D, R663Q). In addition, muta- <br />                                                                                    tions (R633Q I662T) occurred repeatedly multiply <br />                                                                                    mutant isolates. When mutations tested alone, they <br />                                                                                    suppressor activity. All residues <br />                                                                                    mutated lie 30-residue span (633&#226;&#8364;&#8220;663)(Figure 1). All five <br />                                                                                    allow binding T346A E349K UNC-112 N-terminal <br />                                                                                    half proteins. Our interpretation residues region <br />                                                                                    633&#226;&#8364;&#8220;663 contact residues 346&#226;&#8364;&#8220;349, closed conformation <br />                                                                                    wild type UNC-112. Two residues (R633 N659) <br />                                                                                    conserved human kindlins -1, -2, -3. Validation this <br />                                                                                    interpretation awaits determination structure UNC-112, <br />                                                                                    currently pursued. <br />                                                                                        This type &#226;&#8364;&#339;positive screening&#226;&#8364;? &#226;&#8364;&#339;gain-of-function screen- <br />                                                                                    ing&#226;&#8364;? easier previously described &#226;&#8364;&#339;negative screen- <br />                                                                                    ing&#226;&#8364;? (for non-binding mutants). To identify missense mutants <br />  FIGURE 3 | Generalized description method. (A) Isolation               non-binding, mutagenized fragments premature <br />  characterization mutant protein A does bind protein B. <br />                                                                                    stops need recovered. However, &#226;&#8364;&#339;positive screen- <br />  From bottom: protein A known interact protein B. Random <br />  mutagenesis used generate collection protein A mutants                 ing,&#226;&#8364;? step required, &#226;&#8364;&#339;positive reaction&#226;&#8364;? (i.e., <br />  (mutations indicated &#226;&#710;&#8212; various positions protein),     protein interaction) excludes premature stops other <br />  tested yeast 2-hybrid assays interaction protein B. One       severe mutations. &#226;&#8364;&#339;Gain-of-function&#226;&#8364;? suppressors useful for <br />  protein A mutants (&#226;&#710;&#8212; purple) interact protein       identifying interacting molecules (for classic example Adams <br />  B (denoted red X). Transgenic animals generated express either <br />                                                                      (Continued) <br />                                                                                    et al., 1989). In case, &#226;&#8364;&#339;gain-of-function&#226;&#8364;? suppressors PAT-4, <br />                                                                                    starting non-binding mutant single amino <br />  <br />  <br />  <br /> Frontiers Physiology | Striated Muscle Physiology                                                                           April 2014 | Volume 5 | Article 162 | 4 <br />  Qadota Benian                                                                                              Mutational analysis protein interactions <br />  <br />  <br />  <br />  <br /> change UNC-112, useful identifying interacting           expressed mouse heart, extensively studied cultured <br /> surface PAT-4 reasons. First, single amino acid         mouse cardiomyocytes. A yeast 2-hybrid screen revealed a <br /> change non-binding mutant UNC-112 likely           small segment obscurin interacts protein X. In addi- <br /> result large conformational change, prob-     tion, obscurin protein X localize regions the <br /> able single amino acid change interacting surface     sarcomere (M-lines Z-disks), proper localization of <br /> PAT-4 restore interaction. Second, random mutagene-          protein X depends presence obscurin (e.g., RNAi medi- <br /> sis possible identify suppressors. Indeed,        ated knockdown obscurin, results mis-loclization protein <br /> for UNC-112-PAT-4 interaction, identified 9 single amino        X). However, obscuring huge multi-domain pro- <br /> acid changes, finding residues map      tein known expected interactors, knockdown <br /> two clusters homology model PAT-4, inferred         disrupt localization interacting proteins. <br /> binding surfaces.                                                      Random mutatgenesis yeast 2-hybrid assays used iso- <br />                                                                        late <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span>missense mutation</span> protein X (&#226;&#710;&#8212; ), protein X&#226;&#710;&#8212; <br /> BROADER APPLICATIONS OF THE METHOD                                     does interact obscurin. Transient transfection used <br /> In model genetic organisms, S. cerevisiae, C. ele-         express cardiomyocytes, tagged versions wild type <br /> gans, Drosophila, genetic deletions available     &#226;&#710;&#8212; mutant protein X, tagged versions proteins <br /> systems, knock siRNA available. However, ask            detected anti-tag antibodies. Results that <br /> the meaning protein-protein interactions, deletions knock        wild type protein X localizes, &#226;&#710;&#8212; mutant protein X does not <br /> downs, resulting absence reduction level pro-      localize. This strategy demonstrates interaction specific <br /> tein vivo, ideal. One reason deletions knock      region obscurin required localization particu- <br /> downs cause pleiotropic effects. For example, case        lar protein. Such information obtained using simple <br /> of unc-112 pat-4, null mutants stop development             knockdown mutations result reduced levels or <br /> an embryonic stage, examine effect           elimination obscurin isoform. <br /> mutants adult stage muscle, structures well- <br /> defined amenable microscopic analysis. Expression        AUTHOR CONTRIBUTIONS <br /> non-binding mutants adult stage muscle allows examine         Hiroshi Qadota conceived developed method. Hiroshi <br /> the function protein adult stage, allows   Qadota Guy M. Benian obtained data, analyzed it, produced <br /> to examine essentiality protein interaction       figures, wrote manuscript together. <br /> the essentiality expression protein. Another reason is <br /> that proteins interact proteins, complete       ACKNOWLEDGMENTS <br /> absence protein reduced levels protein result      We thank Dr. Don Moerman, University British Columbia, <br /> multiple effects. For example, PAT-4 binds PAT-6 UNC-97,        encouraging work project. We thank Yating <br /> not just UNC-112. Thus, pat-4 null mutants, interactions         Luo, Emory University undergraduate conducting much <br /> with PAT-6 UNC-97 abolished, impossible        studies described Qadota et al. (2014); thank <br /> ask meaning absence UNC-112 interaction        Dr. Kim Gernert (previously Emory University) molecu- <br /> type mutant. In muscle, giant polypeptides        lar modeling, Dr. Yohei Matsunaga (Emory University) for <br /> (&gt;400,000 Da) consisting domains. For example, C.           generating transgenic animals helpful comments on <br /> elegans muscle, UNC-89 (the homolog obscurin mammals)            manuscript. Most studies reported sup- <br /> has reported interact number proteins (CPNA-         ported previous grant American Heart Association <br /> 1, MEL-26, LIM-9, SCPL-1) (Qadota et al., 2008; Xiong et al.,      (11GRNT7820000). <br /> 2009; Wilson et al., 2012; Warner et al., 2013). Severe loss-of- <br /> function alleles unc-89 abolish interaction, it <br />                                                                        REFERENCES <br /> would impossible determine meaning of          Adams, A. E., Botstein, D., Drubin, D. G. (1989). A yeast actin-binding protein <br /> individual protein-protein interactions. One strategy         encoded SAC6, gene suppression actin mutation. Science <br /> isolate unc-89 mutant bind specific interac-          243, 231&#226;&#8364;&#8220;233. doi: 10.1126/science.2643162 <br /> tor. Reciprocally, use methods described isolate      Benian, G. M., Epstein, H. F. (2011). Caenorhabditis elegans muscle: genetic <br />                                                                           molecular model protein interactions heart. Circ. Res. 109, <br /> mutants interactor bind UNC-89, after <br />                                                                           1082&#226;&#8364;&#8220;1095. doi: 10.1161/CIRCRESAHA.110.237685 <br /> expression transgenic worms, determine function         Cadwell, R. C., Joyce, G. F. (1992). Randomization genes PCR mutagen- <br /> interaction.                                                              esis. PCR Methods Appl. 2, 28&#226;&#8364;&#8220;33. doi: 10.1101/gr.2.1.28 <br />    Our strategy studying protein-protein interactions impor-       Dowling, J. J., Gibbs, E., Russell, M., Goldman, D., Minarcik, J., Golden, J. A., <br /> tant muscle sarcomere generalized, depicted             et al. (2008). Kindlin-2 essential component intercalated discs is <br />                                                                           required vertebrate cardiac structure function. Circ. Res. 102, 423&#226;&#8364;&#8220;431. <br /> in (Figure 3). This strategy utilized model sys-            doi: 10.1161/CIRCRESAHA.107.161489 <br /> tem transgenics easily generated muscle           Epstein, H. F., Benian, G. M. (2012). Paradigm shifts cardiovascular research <br /> be easily visualized, Drosophila, zebrafish,          Caenorhaditis elegans muscle. Trends. Cardiovasc. Med. 22, 201&#226;&#8364;&#8220;209. doi: <br /> Xenopus. Indeed, strategy extrapolated study            10.1016/j.tcm.2012.07.021 <br /> protein-protein interaction occurs large cellular struc-   Fukuda, K., Gupta, S., Chen, K., Wu, C., Qin, J. (2009). The pseudoactive <br />                                                                           site ILK essential binding alpha-Parvin localization focal <br /> ture imaged light microscopy, example, synaptic        adhesions. Mol. Cell 36, 819&#226;&#8364;&#8220;830. doi: 10.1016/j.molcel.2009.11.028 <br /> terminals neurons, desmosomes epithelial cells. Here       Goult, B. T., Bouaouina, M., Harburger, D. S., Bate, N., Patel, B., Anthis, N. <br /> one example vertebrate striated muscle: obscurin isoform,          J., et al. (2009). The structure N-terminus Kindlin-1: domain <br />  <br />  <br /> www.frontiersin.org                                                                                                 April 2014 | Volume 5 | Article 162 | 5 <br />  Qadota Benian                                                                                                                  Mutational analysis protein interactions <br />  <br />  <br />  <br />  <br />    important &#206;&#177;IIb&#206;&#178;3 integrin activation. J. Mol. Biol. 394, 944&#226;&#8364;&#8220;956. doi:             Siegel, D. H., Ashton, G. H., Penagos, H. G., Lee, J. V., Feiler, H. S., Wilhelmsen, <br />    10.1016/j.jmb.2009.09.061                                                                 K. C., et al. (2003). Loss kindlin-1, human homolog Caenorhabditis <br /> Homma, K., Terui, S., Minemura, M., Qadota, H., Anraku, Y., Kanaho, Y., et al.               elegans actin-extracellular-matrix linker protein UNC-112, causes Kindler syn- <br />    (1998). Phosphatidylinositol-4-phosphate 5-kinase localized plasma                 drome. Am. J. Hum. Genet. 73, 174&#226;&#8364;&#8220;187. doi: 10.1086/376609 <br />    membrane essential yeast cell morphogenesis. J. Biol. Chem. 273,                Svensson, L., Howarth, K., McDowall, A., Patzak, I., Evans, R., Ussar, S., <br />    15779&#226;&#8364;&#8220;15786. doi: 10.1074/jbc.273.25.15779                                                et al. (2009). Leukocyte adhesion deficiency-III caused mutations in <br /> Lecroisey, C., Brouilly, N., Qadota, H., Mariol, M. C., Rochette, N. C., Martin,             KINDLIN3 affecting integrin activation. Nat. Med. 15, 306&#226;&#8364;&#8220;312. doi: 10.1038/ <br />    E., et al. (2013). ZYX-1, unique zyxin protein C. elegans, involved          nm.1931 <br />    dystrophin-dependent muscle degeneration. Mol. Biol. Cell 24, 1232&#226;&#8364;&#8220;1249. doi:          Takita, Y., Takahara, M., Nogami, S., Anraku, Y., Ohya, Y. (1997). Applications <br />    10.1091/mbc.E12-09-0679                                                                   long accurate polymerase chain reaction method yeast molecular <br /> Lee, J. H., An, J. Y., Park, H., Kim, H. J., Eom, S. H. (2011). Crystallization          biology. Direct sequencing amplified DNA introduction yeast. <br />    preliminary X-ray crystallographic analysis human kindlin-2 PH                 Yeast 13, 763&#226;&#8364;&#8220;768. doi: 10.1002/(SICI)1097-0061(19970630)13:8&lt;763::AID- <br />    domain. Acta Crystallogr. Struct. Biol. Cryst. Commun. 67, 696&#226;&#8364;&#8220;699. doi:                  YEA135&gt;3.0.CO;2-0 <br />    10.1107/S1744309111013820                                                              Warner, A., Qadota, H., Benian, G. M., Vogl, A. W., Moerman, D. G. <br /> Lin, X., Qadota, H., Moerman, D. G., Williams, B. D. (2003). C. elegans PAT-6            (2011). The C. elegans paxillin orthologue, PXL-1, required pharyn- <br />    / Actopaxin plays critical role assembly integrin adhesion complexes          geal muscle contraction viability. Mol. Biol. Cell 22, 2551&#226;&#8364;&#8220;2563. doi: <br />    vivo. Curr. Biol. 13, 922&#226;&#8364;&#8220;932. doi: 10.1016/S0960-9822(03)00372-5                      10.1091/mbc.E10-12-0941 <br /> Liu, J., Fukuda, K., Xu, Z., Ma, Y. Q., Hirbawi, J., Mao, X., et al. (2011). Structural   Warner, A., Xiong, G., Qadota, H., Rogalski, T., Vogl, A. W., Moerman, D. G., <br />    basis phosphoinositide binding kindling-2 protein pleckstrin homologoy              et al. (2013). CPNA-1, copine domain protein, located integrin adhe- <br />    domain regulating integrin activation. J. Biol. Chem. 286, 43334&#226;&#8364;&#8220;43342. doi:           sion sites, required myofilament stability C. elegans. Mol. Biol. Cell <br />    10.1074/jbc.M111.295352                                                                   24, 601&#226;&#8364;&#8220;616. doi: 10.1091/mbc.E12-06-0478 <br /> Mackinnon, A. C., Qadota, H., Norman, K. R., Moerman, D. G., Williams,                Waterston, R. H. (1988). &#226;&#8364;&#339;Muscle,&#226;&#8364;? The Nematode Caenorhabditis Elegans, ed W. <br />    B. D. (2002). C. elegans PAT-4 / ILK functions adaptor protein               B. Wood (Plainview, NY: Cold Spring Harbor Laboratory Press), 281&#226;&#8364;&#8220;335. <br />    integrin adhesion complexes. Curr. Biol. 12, 787&#226;&#8364;&#8220;797. doi: 10.1016/S0960-              Waterston, R. H., Thomson, J. N., Brenner, S. (1980). Mutants altered <br />    9822(02)00810-2                                                                           muscle structure C. elegans. Dev. Biol. 77, 271&#226;&#8364;&#8220;302. doi: 10.1016/0012- <br /> Meissner, B., Warner, A., Wong, K., Dube, N., Lorch, A., McKay, S. J., et al. (2009).        1606(80)90475-3 <br />    An integrated strategy study muscle development myofilament structure           Wick, M., Burger, C., Brusselbach, S., Lucibello, F. C. Muller, R. (1994). <br />    C. elegans. PLoS Genet. 5:e1000537. doi: 10.1371/journal.pgen.1000537                  Identification serum-inducible genes: different patterns gene regulation <br /> Meves, A., Stremmel, C., Gottschalk, K., Fassler, R. (2009). The Kindlin protein         G0 S G1 S progression. J. Cell Sci. 107, 227&#226;&#8364;&#8220;239. <br />    family: new members club focal adhesion proteins. Trends Cell Biol. 19,      Williams, B. D., Waterston, R. H. (1994). Genes critical muscle development <br />    504&#226;&#8364;&#8220;513. doi: 10.1016/j.tcb.2009.07.006                                                   function C. elegans identified lethal mutations. J. Cell Biol. 124, <br /> Moerman, D. G., Fire, A. (1997). &#226;&#8364;&#339;Muscle structure, function develop-                475&#226;&#8364;&#8220;490. doi: 10.1083/jcb.124.4.475 <br />    ment,&#226;&#8364;? C. Elegans II, eds D. L. Riddle, T. Blumenthal, B. J. Meyer, J. R.       Wilson, K. J., Qadota, H., Mains, P. E., Benian, G. M. (2012). UNC-89 <br />    Priess (Plainview, NY: Cold Spring Harbor Laboratory Press), 417&#226;&#8364;&#8220;470.                     (obscurin) binds MEL-26, BTB domain protein, affects function of <br /> Moerman, D. G., Williams, B. D. (2006). &#226;&#8364;&#339;Sarcomere assembly C. ele-                   MEI-1 (katanin) striated muscle C. elegans. Mol. Biol. Cell 23, 2623&#226;&#8364;&#8220;2634. <br />    gans muscle,&#226;&#8364;? WormBook, ed The C.elegans Research community. doi:                      doi: 10.1091/mbc.E12-01-0055 <br />    10.1895/wormbook.1891.1881.1891                                                        Xiong, G., Qadota, H., Mercer, K. B., McGaha, L. A., Oberhauser, A. F., Benian, <br /> Moser, M., Bauer, M., Schmid, S., Ruppert, R., Schmidt, S., Sixt, M., et al. (2009).         G. M. (2009). A LIM-9 (FHL) / SCPL-1 (SCP) complex interacts C- <br />    Kindlin-3 required &#206;&#178;2 integrin mediated leukocyte adhesion endothelial          terminal protein kinase regions UNC-89 (obscurin) C. elegans muscle. <br />    cells. Nat. Med. 15, 300&#226;&#8364;&#8220;305. doi: 10.1038/nm.1921                                        J. Mol. Biol. 386, 976&#226;&#8364;&#8220;988. doi: 10.1016/j.jmb.2009.01.016 <br /> Okada, H., Abe, M., Asakawa-Minemura, M., Hirata, A., Qadota, H., Morishita, K.,          Yates, L. A., Fuzery, A. K., Bonet, R., Campbell, I. D., Gilbert, R. J. (2012a). <br />    et al. (2010). Multiple functional domains yeast 1,3-beta-glucan synthase          Biophysical analysis Kindlin-3 reveals elongated conformation maps <br />    subunit Fks1p revealed quantitative phenotypic analysis temperature-                integrin binding membrane-distal &#206;&#178;-subunit NPXY motif. J. Biol. Chem. <br />    sensitive mutants. Genetics 184, 1013&#226;&#8364;&#8220;1024. doi: 10.1534/genetics.109.109892              287, 37715&#226;&#8364;&#8220;37731. doi: 10.1074/jbc.M112.415208 <br /> Qadota, H., Benian, G. M. (2010). Molecular structure sarcomere mem-            Yates, L. A., Lumb, C. N., Brahme, N. N., Zalyte, R., Bird, L. E., De Colibus, <br />    brane attachment M-lines C. elegans muscle (review). J. Biomed. Biotech.            L., et al. (2012b). Structural functional characterization kindlin- <br />    2010:864749. doi: 10.1155/2010/864749                                                     1 pleckstrin homology domain. J. Biol. Chem. 287, 43246&#226;&#8364;&#8220;43261. doi: <br /> Qadota, H., Luo, Y., Matsunaga, Y., Park, A. S., Gernert, K. M., Benian,                 10.1074/jbc.M112.422089 <br />    G. M. (2014). Suppressor mutations suggest surface PAT-4 (integrin                Zengel, J. M., H. F. Epstein. (1980). Identification genetic el <br /> </body></html>