<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Schultz Qutub BMC Systems Biology (2015) 9:18 <br /> DOI 10.1186/s12918-015-0153-3 <br />  <br />  <br />  <br />  <br />  METHODOLOGY ARTICLE                                                                                                                        Open Access <br />  <br /> Predicting internal cell fluxes sub-optimal <br /> growth <br /> Andr&#195;&#169; Schultz* Amina A Qutub <br />  <br />  <br />   <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />   Background: Flux Balance Analysis (FBA) widely used tool model metabolic behavior cellular function. <br />   Applications FBA span breadth research synthetic engineering biofuels understanding evolutionary <br />   adaptations. FBA predicts metabolic reaction fluxes optimize given objective. This objective generally defined <br />   unicellular organisms theoretical reaction simulates biomass production. FBA extremely <br />   successful predicting E. coli growth rates different media gene essentiality, things. In <br />   order improve predictions, additional constraints coupled optimization biomass function. Studies <br />   suggested, however, unicellular organisms - like multicellular organisms - grow optimal rates. <span id='am-5' about='protege:TO' typeof='owl:Thing'>To</span> <br />   improve FBA predictions, particularly internal cell fluxes, new techniques explore sub-optimal <br />   solution space need developed. <br />   Results: We present innovative FBA method called corsoFBA based optimization protein cost at <br />   sub-optimal objective levels. Our method shows good agreement experimental data E. coli grown different <br />   dilution rates. Maintaining objective function close maximum value predicts metabolic states closely <br />   resemble low dilution rates; higher dilution rates mirrored lowering biomass production value. By <br />   using modified version Extreme Pathways, able quantify energy production overall protein <br />   cost possible pathways central carbon metabolism. <br />   Conclusion: Metabolic flux distributions optimal objective substantially different near-optimal <br />   distributions. Importantly, behavior E. coli central carbon metabolism better predicted exploring the <br />   sub-optimal FBA solution space. The corsoFBA method presented able predict behavior PEP <br />   Carboxylase, glyoxylate shunt Entner-Doudoroff pathway different glucose levels, behavior not <br />   predicted minimization metabolic steps FBA alone. This technique used better predict internal <br />   cell fluxes different conditions, corsoFBA great help study cells multicellular <br />   organisms using Flux Balance Analysis. <br />   Keywords: Flux balance analysis, COBRA, BiGG, Genome-wide metabolic reconstructions, Constraint based models, <br />   Extreme pathways, Sub-optimal growth, Protein cost <br />  <br />  <br /> Background                                                                          formulation reconstructions called Constraint <br /> Genome-wide metabolic reconstructions provide pow-                                Based Models. These models defined core a <br /> erful platform analysis metabolic pathways.                              sparse stoichiometric matrix, column repre- <br /> Reconstructions sixty-five different species                           sents reaction, row metabolite, entry the <br /> spanning fifty-one different genera, including humans                               corresponding stoichiometric coefficient. <br /> [1,2], available today [3]. Such reconstructions                             A vast array computational methods analysis <br /> been used successfully fields research,                               Constraint Based Models available, num- <br /> including metabolic engineering, evolutionary analysis                              ber continues grow [5,6]. Perhaps widely used <br /> and metabolic network properties [4]. The mathematical                              methods Flux Balance Analysis (FBA). FBA <br />                                                                                     returns flux distribution entire metabolism <br /> *Correspondence: as86@rice.edu                                                      optimizes (minimizes maximizes) given objec- <br /> Department Bioengineering, Rice University, 6500 Main Street, Houston, USA <br />                                                                                     tive function reaction, ATP biomass <br />  <br />                                         &#194;&#169; 2015 Schultz Qutub; licensee BioMed Central. This Open Access article distributed terms Creative <br />                                         Commons Attribution License (http://creativecommons.org/licenses/by/4.0), permits unrestricted use, distribution, and <br />                                         reproduction medium, provided original work properly credited. The Creative Commons Public Domain Dedication <br />                                         waiver (http://creativecommons.org/publicdomain/zero/1.0/) applies data available article, unless otherwise <br />                                         stated. <br />  Schultz Qutub BMC Systems Biology (2015) 9:18                                                                   Page 2 12 <br />  <br />  <br />  <br />  <br /> production. This flux distribution satisfies steady-state      thermodynamic cost, miss key biological <br /> assumption, net production con-         factors govern metabolism [16,28-33]. <br /> sumption metabolite. A pre-defined set con-               Significant efforts directed inte- <br /> straints, upper lower bounds reaction       grating omics data FBA [34-38], generally low <br /> and substrate availability, defined [7].                 correlation gene expression associated <br />   This technique commonly applied metabolic           reaction flux makes challenging field [34]. Further- <br /> reconstructions unicellular organisms opti-          more, predictive power integrated omics-FBA <br /> mization theoretical Biomass reaction. This reaction        models limited, experimental data needed to <br /> consumes resources amino acids, nucleotides          predict fluxes. <br /> ATP rate given organism need order              More recent studies relied theory E. <br /> to grow multiply [8,9]. FBA predicted           coli chooses pathways based minimization of <br /> E. coli uptake release rates certain metabolites      protein cost [39,40] number metabolic steps [41]. <br /> [10-12], cell growth rate different environmental          Several successful methods based assumptions <br /> conditions [10,11] gene essentiality [10] great         proposed, include minimization net <br /> success. However, prediction internal cell fluxes         metabolic flux [12], minimization number steps <br /> remains challenge [13], mainly reasons:            metabolism [28,42,43] enzymatic level con- <br />                                                                  straints [44-49]. One alternative method utilizes Elemen- <br />   1. The FBA solution unique. There           tary Modes pre-define directionality reactions, <br />      fluxes model defined,      reducing FBA solution space [50]. <br />      exist certain bounds objective            These methods successful improving <br />      function optimized [7]. These fluctuations         predictions growth rate, substrate usage internal <br />      define FBA solution space,            cell fluxes unicellular organisms. Several studies have <br />      single, unique output.                                      suggested, however, unicellular organisms reality <br />   2. Organisms operating maximum                 grow rates lower predicted FBA alone <br />      capacity [14-18]. In case, objective function      [14-17]. Although above-mentioned methods <br />      fully optimized, instead           predict growth rates lower standard FBA, they <br />      near-optimal sub-optimal state. Furthermore, Flux        rely implementing additional constraints opti- <br />      Variability Analysis shows FBA solution            mization objective function. Furthermore, although <br />      space increases drastically considering              approach successfully implemented the <br />      near-optimal optimal state [18], exacerbating        study cancer cells [49], objective function most <br />      possibility multiple FBA solutions.                      likely hold true analysis healthy multicellular <br />   3. The observed metabolic state unique. That       organisms, like mammals, healthy cells systems <br />      is, single flux distribution explain       complex objectives simply grow and <br />      flux states observed experiments, variation      multiply. <br />      occurs bacterial population. Wintermute             Additionally, one-step optimization methods with <br />      et al. [19] proposes cloud theory metabolic regu-     additional constraints return single flux distribution, <br />      lation, bacteria allowed fluctuate      unable explore near-optimal solution <br />      near-optimal solution space. The study demon-        space. This limitation significant given recent study <br />      strates variability fluxes region   suggested flux distribution E. coli vary freely <br />      matches observed variability data.           near-optimal space [19]. Therefore, order to <br />   4. Thermodynamically                                           improve FBA predictions, especially field <br />      infeasible loops appear FBA output. These        expands include multicellular organisms, new tech- <br />      sets pathways termed &#226;&#8364;&#339;type III pathways &#226;&#8364;?            niques explore sub-optimal solution space need <br />      [20], composed internal reactions              developed. <br />      combined yield steady-state              To address need, propose two-step optimiza- <br />      net input output. These cycles clutter        tion FBA method predicting internal fluxes, termed <br />      FBA output, hindering subsequent analysis [5].          COst Reduced Sub-Optimal FBA (corsoFBA), is <br />                                                                  suitable sub-optimal objectives. Rather impos- <br />   To better predict internal fluxes, studies           ing additional constraints optimizing objective <br /> relied additional thermodynamic constraints [21]. Sev-        function, fix objective predefined value. <br /> eral studies successfully reduce FBA solution       An estimated protein cost metabolism <br /> space, leave variety states that,        minimized order predict internal cell <br /> thermodynamically feasible, physiologically improba-         fluxes. By varying biomass production value, are <br /> ble [22-27]. Other studies rely optimization         able predict changes pathway usage depend <br />  Schultz Qutub BMC Systems Biology (2015) 9:18                                                                                   Page 3 12 <br />  <br />  <br />  <br />  <br /> on changes conditions, particularly glucose                            temperature. The molecular weight term included to <br /> availability. Although method suited                       represent resources needed order pro- <br /> predicting growth rates, able predict metabolic                       duce sufficient enzymes maintain asso- <br /> flux distributions near-optimal solution space.                        ciated reaction flux. The thermodynamic penalty, applied <br /> Furthermore, decomposing model using adapted                          reversible reactions, represents cost associ- <br /> version Extreme Pathway analysis, able                         ated change concentration metabolites <br /> break energy production pathways                           required reaction flow desired direc- <br /> understand model behavior glucose concentrations                         tion. That is, reaction positive  r G o require <br /> change. We validate methods using E. coli                            large disparity concentration associated <br /> model organism.                                                                 metabolites, cost term multiplied positive <br />                                                                                 number larger one. On hand,  r G o is <br /> Methods                                                                         negative, reaction favorable, cost <br /> Cost calculation implementation                                             term multiplied positive number zero and <br /> Some methods successful predicting inter-                        one, given little metabolic adjustment nec- <br /> nal cell fluxes based theory E. coli                      essary. It worth noting thermodynamic cost <br /> is subject protein cost constraint [44-49]. Five                        does directly represent change enzyme concentra- <br /> such methods available literature, cor-                       tion, increase decrease overall cost <br /> soFBA, summarized Table 1. Here explore                             associated reaction based standard Gibbs <br /> similar principle, calculate protein cost reac-                    free energy. Furthermore, term applied separately <br /> tion based net flux reaction (J),                        forward backward directions reversible reac- <br /> enzyme molecular weight (MW ), thermodynamic                              tions, favoring direction negative standard Gibbs <br /> penalty reversible reactions. The cost term used                         free energy. A similar thermodynamic cost, used with- <br /> corsoFBA defined as:                                                         molecular weight considerations, applied by <br />                                <br />                   &#206;&#177; &#194;&#183;  r G o                                                    Holzh&#195;&#188;tter [28], weighing fluxes directed the <br />     J&#194;&#183;MW &#194;&#183;exp                                                                   thermodynamically unfavorable direction. <br />                     R&#194;&#183;T                                                                                                                 mol <br />                                                                                   Finally, parameter &#206;&#177; chosen 0.02RT kcal      , <br />   Where  r G o standard Gibbs free energy                         making final thermodynamic cost simply: <br /> given reaction, R ideal gas constant T                                                        <br />                                                                                                        o mol <br />                                                                                     exp 0.02 &#194;&#183;  r G             . <br />                                                                                                          kcal <br /> Table 1 Optimization methods additional constraints <br /> Method                             Enzymatic cost                  Constraint <br />                                                                                   This value chosen order balance contribution <br />                                                                                 molecular weight thermodynamic penalty, yield- <br /> FBAwMC [44]                          ai Ji                         &#226;&#8240;&#164;1 <br />                                      <br />                                                                                 ing prediction considering costs (Additional file 1: <br /> MOMENT [46]                          gi &#194;&#183; MWi                      &#226;&#8240;&#164;C           Figure S1). Without parameter, exponen- <br />                                                 <br /> Tepper et al. [47]                   Mi + &#206;&#180; &#194;&#183; gi                   minimize     tial nature thermodynamic penalty, term would <br />                                      MWi Ji                                     quickly approach extremely large numbers (for positive <br /> Shlomi et al. [49]                                                 &#226;&#8240;&#164;C <br />                                                                                 values  r G o ) zero (for negative values  r G o ), and <br />                                         kcati <br />                                      <br /> Holzh&#195;&#188;tter [28]                        Ji++ keqi &#194;&#183; Ji&#226;&#710;&#8217;             minimize <br />                                                                                 thermodynamic cost essentially consid- <br />                                                       &#206;&#177;&#194;&#183; r G o <br /> corsoFBA                             Ji &#194;&#183;MWi &#194;&#183;exp       R&#194;&#183;T <br />                                                             i <br />                                                                    minimize     ered. In case, reaction directionality essentially <br />                                  Variables                                      predetermined based solely standard Gibbs free <br /> a - crowding coefficient                                                        energy, problematic. <br />                                                                                   The Gibbs free energy reactions estimated <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> J - flux reaction <br />                                                                                 MetaCyc database [51] change Gibbs energy <br /> g - enzyme concentration                                                        formation reaction compounds. Molecular <br /> MW - molecular weight associated enzyme                                      weight values obtained 506 523 (96.75%) <br /> M - metabolite levels                                                           enzyme catalyzed reactions E. coli iJR904 recon- <br /> &#206;&#180; - model parameter                                                             struction [52]. These values extracted the <br /> kcat - turnover number <br />                                                                                 BRENDA database [53] available. If E. coli mea- <br />                                                                                 surements available, extracted values <br /> keq - thermodynamic equilibrium constant <br />                                                                                 organism closely related E. coli. A small <br /> C - fraction dry weight mass associated proteins                        number molecular weights estimated the <br /> These methods calculate overall cost metabolism summing       EcoCyc database [54], remaining 17 values were <br /> individual costs associated reaction. While previous methods have <br /> been used constrain FBA solution, corsoFBA optimizes cost   defined median calculated values. Additional <br /> given growth rate.                                                              information concerning standard Gibbs free energy <br />  Schultz Qutub BMC Systems Biology (2015) 9:18                                                                 Page 4 12 <br />  <br />  <br />  <br />  <br /> of reactions, protein molecular weights protein cost         applied small networks. The number of <br /> calculations supplemental information        pathways obtained methods, as <br /> (Additional file 2, Additional file 3 Additional file 4:     computational cost analyses, increase expo- <br /> Table S1).                                                       nentially size reconstruction [5,59]. With <br />   Since protein cost term scales linearly           large-scale metabolic reconstructions, decomposi- <br /> flux associated reaction, incorporated        tions quickly impractical tools. Here propose <br /> molecular weight thermodynamic cost directly            modified version Extreme Pathways yields a <br /> the model. All enzyme associated reactions split       significantly lower number pathways decom- <br /> forward backward parts, cost added          position. Each pathways associated pro- <br /> produced metabolite:                                             tein cost ATP production potential, helps <br />                                                                  elucidate model behavior value objective <br />    Initial Reaction <br />                                                                  function varies. <br />    rxn : A + B &#226;&#8225;?&#226;&#8225;&#8217; C + D                                             We large number pathways stems par- <br />    Reactions cost                                           tially need balance pathway steady- <br />                                                        <br />                                        &#206;&#177; &#194;&#183;  r G orxn             state, metabolite net production or <br />    rxnf : A + B &#226;&#8225;&#8217; C + D + MWrxn &#194;&#183;exp                             consumption rate. Certain metabolites, however, known <br />                                            R&#194;&#183;T <br />                                                                  currency metabolites, responsible basic cell <br />                                        &#226;&#710;&#8217;&#206;&#177; &#194;&#183;  r G orxn <br />    rxnb : C + D &#226;&#8225;&#8217; A + B + MWrxn &#194;&#183;exp                             functions occur numerous reactions model. <br />                                             R&#194;&#183;T                  Standard reactions involving metabolites, as <br />    A reaction consumes produced metabolite            ATP hydrolysis, obtained different com- <br /> was added model. While original reac-          binations model reactions. These combinations, <br /> tion model produces cost rate relative   loops, used multiple times same <br /> absolute flux, added reaction       purpose, increasing number pathways have <br /> consume cost. Minimizing flux reac-        essentially core reactions. <br /> tion return flux distribution lowest         This phenomenon illustrated Extreme Path- <br /> enzyme associated cost achieve predefined biomass         ways analysis applied pathway set reac- <br /> production value. Reactions associated enzyme            tions presented Figure 1. The pathway depicted in <br /> were given cost zero.                                       Figure 1A production energy con- <br />    These methods applied E. coli iJR904 recon-       version glucose lactate, produces mols <br /> struction [52]. During simulations glucose uptake        ATP mol glucose. Reaction names are <br /> bound set arbitrary value. Uptake bounds           extracted E. coli reconstruction Orth et al. <br /> CO2 , Fe2 , H2 O, H, Na, NH4 , O2 , Phosphate, SO4      [60]. Figure 1B illustrates loops reactions present in <br /> set arbitrarily large values, larger glu-       reconstruction hydrolyze ATP, which <br /> cose uptake bound, considered present             combined reactions Figure 1A pro- <br /> in excess. All exchange reactions release bounds set        duce pathway satisfying ATP steady-state condition. <br /> to arbitrarily large values glucose, formate      Furthermore, ATPS4r introduces proton imbalance <br /> O2 , set zero. Predicted flux values          hydrolyzing ATP. This imbalance cor- <br /> then normalized flux glucose glucose-        rected proton transport loops presented <br /> 6-phosphate overall flux enzyme associated            Figure 1B. The combination pathway the <br /> reactions.                                                       given loops multiple Extreme Pathways with <br />    It worth noting protein cost term does       essentially core reactions; represent the <br /> account enzyme turnover rates substrate affin-        phenomenon - conversion glucose lactate <br /> ity. We opted cost term kinetic          glycolysis. <br /> parameters enzymes extremely difficult obtain             In order extract core reactions path- <br /> experimentally, vary greatly based pH temper-         way, remove selected set currency metabo- <br /> ature values. Furthermore, simplification constant      lites reconstruction. This removal reduces the <br /> turnover rate substrate affinity enzymes         number pathways obtain Extreme Pathway <br /> been shown yield significant results [39].                    analysis. This metabolite removal performed under <br />                                                                  certain considerations: <br /> Fundamental pathways analysis <br /> Different methods proposed characterize               1. Metabolites removed completely deleted <br /> and decompose FBA solutions space [5,55], includ-                 reactions compartments model. <br /> ing Elementary Modes [56,57], Extreme Pathways [20]                   Removing metabolite reaction not <br /> and Minimal Generators [58]. However, tools                block certain reactions, fewer <br />  Schultz Qutub BMC Systems Biology (2015) 9:18                                                                                         Page 5 12 <br />  <br />  <br />  <br />  <br />  Figure 1 Metabolic loops increase pathway multiplicity. (A) Pathway converting 1 mol glucose 2 mols lactate producing 2 mols of <br />  ATP. Reaction fluxes represented parentheses. Extracellular metabolites defined [e]. All metabolites cytosolic space. <br />  (B) Network loops reactions hydrolyze ATP transport protons cell membrane. These reactions loops used to <br />  balance reactions (A), yielding multiple balanced pathways set backbone reactions. <br />  <br />  <br />  <br />  <br />      options balance metabolite present                     removal currency metabolites reconstruc- <br />      reconstruction.                                                  tion. These reactions alone, however, yield ATP, <br />   2. No reaction sink source                           ADP, phosphate, proton water imbalances orig- <br />      metabolites. Allowing reactions sinks                      inal model. In Fundamental Pathways analysis, each <br />      sources introduce exchange reactions                          resulting fundamental pathway associated this <br />      certain metabolites, affecting model                                imbalance, used analyze each <br />      decomposition.                                                          pathway. In study, imbalance used analyze <br />   3. Reactions removed reconstruction                        ATP production capacity pathway obtained. <br />      contain metabolite been <br />      removed reconstruction. The removal                         Results discussion <br />      reactions remove pathway                          Comparison different cost functions <br />      uses reactions balance metabolite                 Cost-optimal flux simulations performed using four <br />      model.                                                              different types cost: (1) molecular weights alone, (2) <br />                                                                              thermodynamic penalties alone, (3) previously men- <br />   While aware considerations                      tioned combination molecular weights thermo- <br /> modifying model, breaking specifications                    dynamic penalties (4) uniform costs. The use of <br /> should invalidate Fundamental Pathway analy-                         uniform costs gives simple minimization over- <br /> sis. The model decomposition performed,                       flux enzyme associated reactions. This <br /> but limitations considered. We                         minimization strategy modified version widely <br /> have illustrated Fundamental Pathways method using                      regarded two-step optimization method called pFBA [12]. <br /> the E. coli model Orth et al. [60], demon-                       pFBA previously used predict unique flux <br /> strated original pathways recovered                       distributions predicted FBA optima. <br /> adding identified loops. This analysis                         Simulations comparing costs per- <br /> found supplemental information (Additional file 5                     formed growth rates ranging 50 100% the <br /> and Additional file 6: Table S2).                                            predicted optima. Flux distributions normalized by <br />   While obtain fewer pathways tailored recon-                     flux glucose glucose-6-phosphate, results <br /> struction, set reactions associated                       central carbon metabolism reactions be <br /> imbalance original model. For example, instead                        supplemental information (Additional file 7: <br /> of multiple Extreme Pathways, set reactions                        Figure S2). While simulations using molecular weights, <br /> Figure 1A yield single fundamental pathway                     thermodynamic costs, combination present <br />  Schultz Qutub BMC Systems Biology (2015) 9:18                                                                 Page 6 12 <br />  <br />  <br />  <br />  <br /> considerably different flux distributions different sub-      [16,63]. ATP production optimization lead to <br /> optimal values, simulations using uniform costs quickly          higher fluxes TCA cycle lower PPP fluxes. <br /> converge unique flux distribution objective          It worth noting that, fluxes the <br /> value lowered.                                                TCA cycle under-predicted according Figure 2, <br />    The set simulations compared sev-         MFA experimental results inconsistent, our <br /> eral data points Metabolic Flux Analysis (MFA)          predicted TCA fluxes good agreement two <br /> experiments: Ishii et al. [61] Yao et al. [62]. Correla-     MFA studies [64,65]. <br /> tion sum squared error simulated             These simulations mirror behavior known over- <br /> experimental flux distributions central carbon       flow metabolism [66], E. coli, high growth rates, <br /> metabolism reactions calculated. These results          moves away oxidation glucose through <br /> presented supplemental information (Additional            TCA cycle uses ATP efficient pathways, <br /> file 8: Figure S3). A similar method used             releasing acetate instead CO2 . Our simulations support <br /> Holzh&#195;&#188;tter [28] included Additional file 8:          hypothesis behavior stems trade- <br /> Figure S3. While pFBA simulations yield highest cor-         enzyme cost energy yield [40]. That is, <br /> relation values suboptimal space compared         glucose available, E. coli uses higher flux <br /> the Ishii et al. dataset, simulations yield      pathways enzymatically costly, but <br /> the highest sum squared error. Furthermore,          produce fewer ATP mmol glucose. <br /> objective function lowered 100% 65%             Predicted fluxes using molecular weights a <br /> optima, corsoFBA yields lowest sum squared error          combined cost excellent agreement the <br /> for data points (Ishii et al. dilution    experimental values reactions PPC (PEP Carboxy- <br /> 0.7h-1 ), highest correlation values higher      lase) ICL (Isocitrate Lyase)[61,62]. Both experiments <br /> dilution rates Yao et al. data (dilution = 0.4, 0.6   decrease ICL flux increase PPC <br /> 0.7h-1 ).                                                        flux, near-optimal conditions growth rate 0.5 <br />                                                                  h-1 . This result supported experimental <br /> Comparison metabolic flux analysis                   results Nanchen et al. [67], lower flux <br /> simulated fluxes                                                 ICL growth rate approximately 0.05 <br /> In order visualize trend different fluxes          h-1 compared growth rate 0.1 h-1 , suggest- <br /> change sub-optimal space, simulations utilizing           ing transient response pathway glucose <br /> molecular weight costs, thermodynamic costs com-           concentration increases. The transient response was <br /> bination both, normalized flux glucose        simulations. These results suggest E. <br /> to glucose-6-phosphate, plotted alongside exper-        coli utilizes anaplerotic reactions (reactions respon- <br /> imental values MFA experiments. Fluxes              sible forming intermediates metabolic pathways) <br /> were plotted starting 100% 75% pre-            relieve enzymatic cost, considering these <br /> dicted optima, results central car-          costs FBA increase internal flux predictions <br /> bon metabolism reactions presented Figure 2. The          comparing minimization metabolic steps <br /> threshold 75% chosen order match trend          alone. <br /> observed simulations experimental data                Simulation results suggest optimization of <br /> [61,62].                                                         biomass function yield fluxes fundamen- <br />   As objective function value decreases, corsoFBA            tally different near-optimal conditions. At <br /> flux distributions close agreement MFA data            optimal growth, predicted flux the <br /> for higher growth rates (Figure 2), particularly simu-       glyoxylate shunt, considerably lower fluxes the <br /> lations considering thermodynamic molecular             TCA cycle PPP, flux PPC reaction <br /> weight costs. Fluxes Pentose Phosphate Path-         present near-optimal conditions. Although com- <br /> way (PPP) Glycolysis remain relatively constant,       parisons experimental data simulated flux <br /> the flux TCA cycle gradually decreases. The          distributions highest correlation low- <br /> decrease TCA cycle usage leads higher levels            est error optimal growth, simulations <br /> acetate release. Quantitatively, fluxes glycol-      indicate implementation costs yield <br /> ysis TCA cycle slightly under-predicted           comparable results sub-optimal space (Additional <br /> our simulations, flux PPP gener-        file 8: Figure S3). Moreover, suboptimal corsoFBA <br /> ally over-predicted. Similar patterns observed         approach better predict fluxes certain reac- <br /> in FBA method approaches [33,45,48],              tions, PPC ICL. While previous studies have <br /> may arise optimization biomass func-           shown FBA solution space increases drastically <br /> tion alone. The optimization ATP production along-            considering objective function near-optimal <br /> side biomass shown yield better predictions          optimal values [18], optimization <br />  Schultz Qutub BMC Systems Biology (2015) 9:18                                                                                       Page 7 12 <br />  <br />  <br />  <br />  <br />  Figure 2 Comparison simulations MFA experiments. Comparison selected simulated fluxes using molecular weights, <br />  thermodynamic penalties, combination costs, Metabolic Flux Analysis (MFA) data. Fluxes normalized glucose to <br />  glucose-6-phosphate conversion rates. X-axes values kept constant plots. Reaction names taken Ecoli iJR904 model [52]. <br />  Yellow boxes indicate Pentose Phosphate Pathway (PPP), blue indicates glycolysis, green indicates TCA cycle red indicates reactions. <br />  <br />  <br />  <br /> of enzymatic cost near-optimal conditions yields                     (Additional file 9: Figure S4). Predictions using uniform <br /> results consistent experimental data                     costs quickly diverge experimental data. These sim- <br /> for certain reactions. This result strongly suggests                   ulations support &#226;&#8364;&#339;cloud theory&#226;&#8364;? proposed by <br /> exploring FBA solution space near-optimal opti-                   Wintermute et al. [19], suggest experimental <br /> mal objective values increase predictive accuracy                   data good agreement near-optimal flux <br /> of internal cell fluxes.                                                    distributions. <br />   The near optimal solution space analyzed for <br /> <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>knockout</span> strains. Flux distributions E. coli knock-                 Comparison gene expression data simulated <br /> out strains reported Ishii et al. [61] compared                  fluxes <br /> cost-optimal simulations using combined cost func-                      To validate analysis, simulations using the <br /> tion. Results cost-optimal flux distribution                      combined cost function compared gene <br /> from 95% optimal yield similar better predictions                     expression values reported MFA studies [61,62]. <br /> according correlation sum square error                       While overall correlation reaction flux and <br />  Schultz Qutub BMC Systems Biology (2015) 9:18                                                                                    Page 8 12 <br />  <br />  <br />  <br />  <br /> gene expression moderate best [34], increased                       decreased flux TCA cycle, supported by <br /> expression genes participating particular path-               relative gene expression associated path- <br /> way considered good indication increased flux                 ways [68]. <br /> [68]. Simulations performed con-                         Simulated values good agreement fluxes <br /> ditions before, utilizing combination molecular                 Entner-Doudoroff (ED) pathway. Genes edd <br /> weights thermodynamic penalty, time flux                 eda, associated pathway, long been <br /> distribution normalized overall flux                   known exist E. coli [69], activity their <br /> the enzyme associated reactions. Comparisons                      associated reactions observed mainly under <br /> simulated values gene expression data shown                    growth gluconate, glucuronate, methyl-beta-D- <br /> Figure 3.                                                                 glucuronide; phosphate limitation; carbon starvation <br />   Although gene expression data inconsistent                  [70]. Due fact, reactions generally not <br /> between experiments, reactions associated                    included MFA experiment networks. In contrast with <br /> with multiple genes, comparison supports                     Murray et al., simulations using molecular weight and <br /> the qualitative results presented previous section.                combined costs predict use ED pathway under <br /> When normalizing predicted fluxes overall flux                 excess glucose conditions, starvation (Additional <br /> through enzyme associated reactions, increased                     file 7: Figure S2). These predictions supported both <br /> relative uptake glucose observed objective                   genetic data presented MFA studies [61,62], <br /> value decreases. As result, increased relative flux                  increase edd eda expression high <br /> through glycolysis predicted. Furthermore,                    glucose concentrations, MFA experimental results by <br /> relative flux TCA cycle decreases                    Harcombe et al. [71], fluxes ED path- <br /> the growth rate increases. The increase relative glu-                  way measured bacteria growing growth rates <br /> cose uptake flux glycolysis,                   1 h-1 . <br />  <br />  <br />  <br />  <br />  Figure 3 Comparison simulations gene expression data. Comparison selected simulated fluxes, using combination of <br />  molecular weights thermodynamic penalties, associated gene expression. Plot axes the ones defined ptsG plot <br />  unless specified. Multiple genes associated reaction included box. Reaction names taken E. <br />  coli iJR904 reconstruction [52]. Yellow boxes indicate Pentose Phosphate Pathway, blue indicates glycolysis, green indicates TCA cycle red <br />  indicates reactions. <br />  Schultz Qutub BMC Systems Biology (2015) 9:18                                                                                       Page 9 12 <br />  <br />  <br />  <br />  <br /> Fundamental pathways analysis                                               total cost associated pathway. The final ATP <br /> To better understand transition metabolic                       production enzymatic cost compared the <br /> states value objective function decreased,                 potential ATP production mmol glucose (Figure 4). <br /> the energy producing pathways E. coli iJR904 recon-                    In accordance general E. coli metabolism knowl- <br /> struction decomposed using Fundamental Path-                       edge, fundamental pathway analysis opti- <br /> ways analysis described Methods section. Details                     mal pathways (OP), according ATP production both <br /> on reactions calculated                         mmol glucose protein cost. The energy pro- <br /> in supplemental information (Additional file 5                      ductive pathway oxidation glucose through <br /> Additional file 6: Table S2). Briefly, starting ATP                TCA cycle (OP2 ). The cost efficient pathway was <br /> imbalance associated fundamental pathway,                     production acetate glycoly- <br /> total ATP potential pathway estimated based                     sis (OP1 ). Although pathway potentially produces less <br /> on imbalance energy generating metabolites,                    half ATP OP2 , total enzymatic <br /> such NADH Ubiquinol-8. The lowest cost con-                       cost mmol ATP lower. Also agreement with <br /> verting metabolites ATP added                        general knowledge, analysis predicted release of <br />  <br />  <br />  <br />  <br />  Figure 4 Fundamental pathway analysis. ATP production potential energy producing fundamental pathways, compared associated <br />  protein cost mmol ATP. Optimal (OP) Near Optimal (NOP) Pathways highlighted scatter plot. OPs plotted blue and <br />  NOPs red central carbon metabolism diagrams. <br />  Schultz Qutub BMC Systems Biology (2015) 9:18                                                                                  Page 10 12 <br />  <br />  <br />  <br />  <br /> acetate efficient release lactate       Conclusion <br /> and ethanol.                                                        Here present corsoFBA, method analyzing and <br />    In simulations presented previous sections,             predicting metabolic flux distributions sub-optimal <br /> gradual transition OP2 OP1 observed               states based protein thermodynamic cost mini- <br /> objective function value decreases. This trend sup-            mization. CorsoFBA demonstrates optimization <br /> ports idea overflow metabolism takes place              protein cost near-optimal states produce sig- <br /> order alleviate enzymatic cost. Under low glucose con-           nificantly different results optimal <br /> ditions, E. coli need extract glucose             objective. Furthermore, correlation error <br /> largest possible energy order sustain               calculations indicate better predictions optimal <br /> growth. As glucose available, however,                 solution, protein-optimal results sub-optimal objec- <br /> this bacteria afford consume substrate              tives better agreement MFA experiments and <br /> produce energy cheaper pathways.                            gene expression profiles anaplerotic reactions. These <br />    One interesting observation analysis            results suggest important explore sub-optimal <br /> existence Near Optimal Pathways (NOP), com-                FBA space order better predict internal cell fluxes. <br /> bine previously described optimal pathways             Although method described suited pre- <br /> PPP ED pathways. NOP2 NOP3 produce                      dicting growth rates, provides platform analyzing <br /> more ATP mmol glucose OP1                 internal cell fluxes sub-optimal space. <br /> OP2 , remaining cost efficient OP1                We believe corsoFBA particularly useful given <br /> slightly OP2 . NOP2 NOP3 reduce                recent studies suggested E. coli exist freely within <br /> flux upper glycolysis partially re-routing           near-optimal space [19]. We believe method <br /> flux PPP, exhibit flux              useful predicting fluxes healthy, multicellu- <br /> Phosphoglucose isomerase (pgi). A Near Optimal                lar organisms, complex objectives than <br /> Pathway, termed NOP1 , skips upper glycolysis           production biomass. Future studies merited to <br /> the ED pathway. This pathway cost energy           explore predictive power methods which <br /> effective OP1 .                                                optimize objective function enzymatic level <br />    The use near optimal pathways explain             constraints, MOMENT [46] FBAwMC [44], <br /> the inconsistent PPP fluxes reported MFA experi-             benefit exploring sub-optimal solu- <br /> ments considered here. A short recovery simulated            tion space. Our results suggest significant conclusions <br /> PPP flux near growth rate 0.65 h&#226;&#710;&#8217;1 (Figure 2) demon-           drawn adapting methods optimize the <br /> strates pathway viable option certain dilu-       enzymatic cost implement near-optimal space, <br /> tion rates, coupling PPP biomass            using simply model constraint. <br /> production. It demonstrated  pfkA <br /> deficient E. coli strains reduce flux Phosphofruc- <br /> tokinase 1 diverting fluxes PPP [72],          Additional files <br /> using NOP2 NOP3 instead OP1 OP2 the <br /> cost upper glycolysis increased.                          Additional file 1: Figure S1. Sensitivity analysis parameter &#206;&#177;. Model <br />                                                                      simulations ranging 100 75% optima varying values &#206;&#177;. <br />    This Fundamental Pathway analysis elucidates               Simulations performed using thermodynamic costs and <br /> would expect model behave terms energy gen-             combined costs. <br /> eration optimize protein cost sub-optimal               Additional file 2: Gibbs free energy change distribution. File showing <br /> states. Based analysis, order fulfill catabolic    plot distribution Gibbs free energy change reactions <br />                                                                      considered. <br /> needs, model transitions OP2 OP1 move <br />                                                                      Additional file 3: Molecular weight cost calculation. Details on <br /> away optimal growth. Although fact               Molecular weight values obtained final costs calculated. <br /> general trend observe, anaplerotic needs need             Molecular weights final costs distribution plotted. <br /> be considered. These needs addressed efficiently            Additional file 4: Table S1. Thermodynamic, molecular weight final <br /> by reactions OP1 OP2 , PEP carboxy-                cost values. Table includes values used simulations: Gibbs energy <br />                                                                      formation metabolites, calculated Gibbs energy change all <br /> lase glyoxylate shunt. Interestingly, model              reactions based metabolite energy formation molecular weights <br /> also predicts use ED pathway anaplerotic             obtained databases. <br /> purposes high growth rates. While pathway                Additional file 5: Fundamental pathway analysis. Example <br /> been studied catabolic activity Z. mobilis         demonstrating recovery Extreme Pathways adding reactions loops <br />                                                                      Fundamental Pathways. Details fundamental <br /> and Pseudomonas species, simulations pre-               pathway analysis presented manuscript performed also <br /> dict pathway cheapest way glucose              available. <br /> shuttled TCA cycle production building             Additional file 6: Table S2. Model decomposition. Extreme Pathways and <br /> blocks, giving little efficiency energy pathways         Fundamental Pathways calculated analysis presented the <br />                                                                      manuscript pathway recovery analysis Additional file 5. <br /> through NOP1 . <br />  Schultz Qutub BMC Systems Biology (2015) 9:18                                                                                                      Page 11 12 <br />  <br />  <br />  <br />  <br />                                                                                      8.    Feist AM, Herrg&#195;&#165;rd MJ, Thiele I, Reed JL, Palsson B&#195;&#732;. Reconstruction of <br />  Additional file 7: Figure S2. Cost values comparison. Cost-optimal <br />                                                                                            biochemical networks microorganisms. Nat Rev Microbiol. 2009;7(2): <br />  simulations using molecular weights alone, thermodynamic penalties <br />                                                                                            129&#226;&#8364;&#8220;43. <br />  alone, combination uniform costs presented major <br />                                                                                      9.    Feist AM, Palsson BO. The biomass objective function. Curr Opin <br />  central carbon metabolism reactions range 100 50% of <br />                                                                                            Microbiol. 2010;13(3):344&#226;&#8364;&#8220;9. <br />  maximal growth. <br />                                                                                      10.   Edwards JS, Palsson BO. Metabolic flux balance analysis silico <br />  Additional file 8: Figure S3. Error correlation calculations.                         analysis escherichia coli k-12 gene deletions. BMC Bioinf. 2000;1:1. <br />  Correlation sum squared error calculated simulated           11.   Edwards JS, Ibarra RU, Palsson BO. In silico predictions escherichia coli <br />  experimental flux distributions major central carbon metabolism                   metabolic capabilities consistent experimental data. Nat <br />  reactions.                                                                                Biotechnol. 2001;19(2):125&#226;&#8364;&#8220;30. <br />  Additional file 9: Figure S4. Error correlation calculations                12.   Lewis NE, Hixson KK, Conrad TM, Lerman JA, Charusanti P, Polpitiya AD, <br />  knockout strains. Correlation sum squared error calculated                     et al. Omic data evolved e. coli consistent computed <br />  MFA experimental data simulated flux distributions using                  optimal growth genome-scale models. Mol Syst Biol. 2010;6:390. <br />  combined cost function knockout strains.                                    13.   Chen X, Alonso AP, Allen DK, Reed JL, Shachar-Hill Y. Synergy between <br />                                                                                            (13)c-metabolic flux analysis flux balance analysis understanding <br /> Abbreviations                                                                              metabolic adaptation anaerobiosis e. coli. Metab Eng. 2011;13(1): <br /> TCA: Tricarboxylic acid; FBA: Flux balance analysis; PPP: Pentose phosphate                38&#226;&#8364;&#8220;48. <br /> pathway; MFA: Metabolic flux analysis; ED: Entner-Doudoroff; glc: glucose; g6p:      14.   Tang YJ, Martin HG, Myers S, Rodriguez S, Baidoo EEK, Keasling JD. <br /> glucose-6-phosphate; f6p: fructose-6-phosphate; fdp: fructose-1,6-                         Advances analysis microbial metabolic fluxes (13)c isotopic <br /> biphosphate; dhap: Dihydroxyacetone phosphate; g3p: Glyceraldehyde                         labeling. Mass Spectrom Rev. 2009;28(2):362&#226;&#8364;&#8220;75. <br /> 3-phosphate; 13dpg: 3-Phospho-D-glyceroyl phosphate; 3pg: 3-Phospho-D-               15.   Fischer E, Sauer U. Large-scale vivo flux analysis shows rigidity and <br /> glycerate; 2pg: D-Glycerate 2-phosphate; pep: Phosphoenolpyruvate; pyr:                    suboptimal performance bacillus subtilis metabolism. Nat Genet. <br /> Pyruvate; accoa: Acetyl-CoA; actp: Acetyl phosphate; ac: Acetate; cit: Citrate;            2005;37(6):636&#226;&#8364;&#8220;40. <br /> icit: Isocitrate; akg: 2-Oxoglutarate; succoa: Succinyl-CoA; succ: Succinate; fum:   16.   Schuetz R, Kuepfer L, Sauer U. Systematic evaluation objective <br /> Fumarate; mal-L: L-Malate; oaa: Oxaloacetate; 6pgl:                                        functions predicting intracellular fluxes escherichia coli. Mol Syst <br /> 6-phospho-D-glucono-1,5-lactone; 6pgc: 6-Phospho-D-gluconate; ru5p-D:                      Biol. 2007;3:119. <br /> D-Ribulose 5-phosphate; r5p: alpha-D-Ribose 5-phosphate; xu5p-D:                     17.   Knoop H, Gr&#195;&#188;ndel M, Zilliges Y, Lehmann R, Hoffmann S, Lockau W, <br /> D-Xylulose 5-phosphate; s7p: Sedoheptulose 7-phosphate; e4p: D-Erythrose                   et al. Flux balance analysis cyanobacterial metabolism: metabolic <br /> 4-phosphate; 2ddg6p: 2-Dehydro-3-deoxy-D-gluconate 6-phosphate.                            network synechocystis sp. pcc 6803. PLoS Comput Biol. 2013;9(6): <br />                                                                                            1003081. <br />                                                                                      18.   Mahadevan R, Schilling CH. The effects alternate optimal solutions in <br /> Competing interests                                                                        constraint-based genome-scale metabolic models. Metab Eng. 2003;5(4): <br /> The authors declare competing interests.                                 264&#226;&#8364;&#8220;76. <br />                                                                                      19.   Wintermute EH, Lieberman TD, Silver PA. An objective function exploiting <br />                                                                                            suboptimal solutions metabolic networks. BMC Syst Biol. 2013;7:98. <br /> Authors&#226;&#8364;&#8482; contributions <br /> AS AQ conceived methods revised manuscript. AS drafted           20.   Schilling CH, Letscher D, Palsson BO. Theory systemic <span id='am-3' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> <br /> manuscript generated results. Both authors read manuscript                metabolic pathways use interpreting metabolic function <br /> and approved final version.                                                            pathway-oriented perspective. J Theor Biol. 2000;203(3): <br />                                                                                            229&#226;&#8364;&#8220;48. <br />                                                                                      21.   Soh KC, Hatzimanikatis V. Network thermodynamics post-genomic <br /> Acknowledgements                                                                           era. Curr Opin Microbiol. 2010;13(3):350&#226;&#8364;&#8220;7. <br /> This work funded NSF grants 1150645 1354390. We                22.   Yang F, Qian H, Beard DA. Ab initio prediction thermodynamically <br /> like thank K.W. Lin, B. Long, D. Noren, A. Mahadevan C.W. Hu helpful            feasible reaction directions biochemical network stoichiometry. <br /> discussions manuscripts. We like thank Dr. Jay Storz,            Metab Eng. 2005;7(4):251&#226;&#8364;&#8220;9. <br /> University Nebraska, Dr. Zac Cheviron, University Illinois, Dr.       23.   Beard DA, Babson E, Curtis E, Qian H. Thermodynamic constraints for <br /> Grant McClelland Dr. Graham Scott, McMaster University, helpful               biochemical networks. J Theor Biol. 2004;228(3):327&#226;&#8364;&#8220;33. <br /> discussions.                                                                         24.   Senger RS, Papoutsakis ET. Genome-scale model clostridium <br />                                                                                            acetobutylicum: Part i. metabolic network resolution analysis. <br /> Received: 29 September 2014 Accepted: 20 February 2015                                     Biotechnol Bioeng. 2008;101(5):1036&#226;&#8364;&#8220;52. <br />                                                                                      25.   Senger RS, Papoutsakis ET. Genome-scale model clostridium <br />                                                                                            acetobutylicum: Part ii. development specific proton flux states and <br /> References                                                                                 numerically determined sub-systems. Biotechnol Bioeng. 2008;101(5): <br /> 1. Duarte NC, Becker SA, Jamshidi N, Thiele I, Mo ML, Vo TD, et al. Global                 1053&#226;&#8364;&#8220;71. <br />     reconstruction human metabolic network based genomic               26.   Henry CS, Jankowski MD, Broadbelt LJ, Hatzimanikatis V. Genome-scale <br />     bibliomic data. Proc Natl Acad Sci USA. 2007;104(6):1777&#226;&#8364;&#8220;82.                           thermodynamic analysis escherichia coli metabolism. Biophys J. <br /> 2. Thiele I, Swainston N, Fleming RMT, Hoppe A, Sahoo S, Aurich MK, et al.                 2006;90(4):1453&#226;&#8364;&#8220;61. <br />     A community-driven global reconstruction human metabolism. Nat                27.   Hoppe A, Hoffmann S, Holzh&#195;&#188;tter H-G. Including metabolite <br />     Biotechnol. 2013;31(5):419&#226;&#8364;&#8220;25.                                                         concentrations flux balance analysis: thermodynamic realizability a <br /> 3. Monk JM. Available Predictive Genome-scale Metabolic Network                            constraint flux distributions metabolic networks. BMC Syst Biol. <br />     Reconstructions. http://systemsbiology.ucsd.edu/InSilicoOrganisms/                     2007;1:23. <br />     OtherOrganisms.                                                                  28.   Holzh&#195;&#188;tter H-G. The principle flux minimization application to <br /> 4. Oberhardt MA, Palsson B&#195;&#732;, Papin JA. Applications genome-scale                        estimate stationary fluxes metabolic networks. Eur J Biochem. <br />     metabolic reconstructions. Mol Syst Biol. 2009;5:320.                                  2004;271(14):2905&#226;&#8364;&#8220;22. <br /> 5. Lewis NE, Nagarajan H, Palsson BO. Constraining metabolic                     29.   Han B, Wang J. Least dissipation cost design principle robustness <br />     genotype-<span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>phenotype</span> relationship using phylogeny silico methods.                function cellular networks. Phys Rev E Stat Nonlin Soft Matter Phys. <br />     Nat Rev Microbiol. 2012;10(4):291&#226;&#8364;&#8220;305.                                                 2008;77(3 Pt 1):031922. <br /> 6. Bordbar A, Monk JM, King ZA, Palsson BO. Constraint-based models                  30.   Prigogine I, George C. The second law selection principle: The <br />     predict metabolic associated cellular functions. Nat Rev Genet.                    microscopic theory dissipative processes quantum systems. Proc <br />     2014;15(2):107&#226;&#8364;&#8220;20.                                                                     Natl Acad Sci USA. 1983;80(14):4590&#226;&#8364;&#8220;4. <br /> 7. Orth JD, Thiele I, Palsson B&#195;&#732;. What flux balance analysis? Nat                 31.   Henry CS, Broadbelt LJ, Hatzimanikatis V. Thermodynamics-based <br />     Biotechnol. 2010;28(3):245&#226;&#8364;&#8220;8.                                                          metabolic flux analysis. Biophys J. 2007;92(5):1792&#226;&#8364;&#8220;805. <br />  Schultz Qutub BMC Systems Biology (2015) 9:18                                                                                                    Page 12 12 <br />  <br />  <br />  <br />  <br /> 32. Hamilton JJ, Dwivedi V, Reed JL. Quantitative assessment                      54. Keseler IM, Collado-Vides J, Santos-Zavaleta A, Peralta-Gil M, <br />     thermodynamic constraints solution space genome-scale                      Gama-Castro S, Mu&#195;&#177;iz-Rascado L, et al. Ecocyc: comprehensive <br />     metabolic models. Biophys J. 2013;105(2):512&#226;&#8364;&#8220;22.                                     database escherichia coli biology. Nucleic Acids Res. 2011;39 <br /> 33. Fleming RMT, Thiele I, Provan G, Nasheuer HP. Integrated stoichiometric,             (Database issue):583&#226;&#8364;&#8220;90. <br />     thermodynamic kinetic modelling steady state metabolism.                  55. Llaneras F, Pic&#195;&#179; J. Which metabolic pathways generate characterize <br />     J Theor Biol. 2010;264(3):683&#226;&#8364;&#8220;92.                                                    flux space? comparison elementary modes, extreme <br /> 34. Blazier AS, Papin JA. Integration expression data genome-scale                 pathways minimal generators. J Biomed Biotechnol. 2010;2010: <br />     metabolic network reconstructions. Front Physiol. 2012;3:299.                        753904. <br /> 35. Chandrasekaran S, Price ND. Probabilistic integrative modeling                56. Schuster S, Dandekar T, Fell DA. Detection elementary flux modes in <br />     genome-scale metabolic regulatory networks escherichia coli               biochemical networks: promising tool pathway analysis and <br />     mycobacterium tuberculosis. Proc Natl Acad Sci USA. 2010;107(41):                    metabolic engineering. Trends Biotechnol. 1999;17(2):53&#226;&#8364;&#8220;60. <br />     17845&#226;&#8364;&#8220;50.                                                                        57. Schuster S, Fell DA, Dandekar T. A general <span id='am-4' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> metabolic <br /> 36. vanBerlo RJP, deRidder D, Daran J-M, Daran-Lapujade PAS, Teusink B,                  pathways useful systematic organization analysis complex <br />     Reinders MJT. Predicting metabolic fluxes usi <br /> </body></html>