<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Genetic Validation Aminoacyl-tRNA Synthetases Drug Targets in <br /> Trypanosoma brucei <br /> Savitha Kalidas,a Igor Cestari,b Severine Monnerat,b Qiong Li,a Sandesh Regmi,a Nicholas Hasle,b Mehdi Labaied,b Marilyn Parsons,b <br /> Kenneth Stuart,b Margaret A. Phillipsa <br /> Department Pharmacology, University Texas Southwestern Medical Center, Dallas, Texas, USAa; Seattle Biomedical Research Institute, Seattle, Washington, USAb <br />  <br /> Human African trypanosomiasis (HAT) important public health threat sub-Saharan Africa. Current drugs unsatis- <br /> factory, new drugs sought. Few validated enzyme targets available support drug discovery efforts, our <br /> goal obtain essentiality data genes proven utility drug targets. Aminoacyl-tRNA synthetases (aaRSs) known <br /> drug targets bacterial fungal pathogens required protein synthesis. Here survey essentiality eight <br /> Trypanosoma brucei aaRSs <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) gene expression knockdown, covering enzyme major aaRS <br /> class: valyl-tRNA synthetase (ValRS) (class Ia), tryptophanyl-tRNA synthetase (TrpRS-1) (class Ib), arginyl-tRNA synthetase <br /> (ArgRS) (class Ic), glutamyl-tRNA synthetase (GluRS) (class 1c), threonyl-tRNA synthetase (ThrRS) (class IIa), asparaginyl- <br /> tRNA synthetase (AsnRS) (class IIb), phenylalanyl-tRNA synthetase (&#226;?&#163; &#226;?&#164;) (PheRS) (class IIc). Knockdown mRNA <br /> encoding enzymes T. brucei mammalian stage parasites showed essential parasite growth <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span>survival</span> <br /> in vitro. The reduced expression resulted growth, morphological, cell cycle, DNA content abnormalities. ThrRS char- <br /> acterized greater detail, showing purified recombinant enzyme displayed ThrRS activity protein localized <br /> <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> cytosol mitochondrion. Borrelidin, known inhibitor ThrRS, inhibitor T. brucei ThrRS showed <br /> antitrypanosomal activity. The data aaRSs essential T. brucei survival likely excellent targets for <br /> drug discovery efforts. <br />  <br />  <br />  <br /> H     uman African trypanosomiasis (HAT), known African <br />       sleeping sickness, caused Trypanosoma brucei, proto- <br /> zoan parasite transmitted tsetse flies genus Glossina (1). <br />                                                                                       distinguished different structural folds site of <br />                                                                                       esterification (class I enzymes esterify 2=-hydroxyl aden- <br />                                                                                       osine ribose class II enzymes esterify 3=-hydroxyl of <br /> HAT endemic 36 countries threatens 60 million                     ribose). Inhibitors aaRSs antibacterial anti- <br /> people sub-Saharan Africa. Few drugs available treat                    fungal activity reported, mupirocin (used a <br /> HAT, use complicated limited efficacy depends                    topical antibiotic) currently aaRS inhibitor clinical <br /> on subspecies development stage parasite                      use, various stages discovery develop- <br /> (2, 3). Treatment late-stage disease particularly problematic,                  ment (11&#226;&#8364;&#8220;13) (see Table 1). Recent examples identification <br /> and recent introduction nifurtimox-eflornithine                          aaRS inhibitors targeting parasitic protozoa de- <br /> combination therapy (NECT) improved treatment outcomes                            scribed, including discovery cladosporin targets Plasmo- <br /> (4), more-effective drugs combat forms disease                    dium falciparum LysRS, providing potential lead malaria <br /> still badly needed.                                                                   drug discovery (19). The targeted aaRSs diverse, spanning <br />     Alliances academic industry partners emerged                       different amino acid specificities different enzyme <br /> with goal exploiting molecular-target approach antipa-                  classes. Thus, aaRSs appear broadly appealing targets for <br /> rasitic drug discovery (5, 6). An open-access resource (tdrtargets.org)               development new antimicrobial agents. <br /> was developed improve prioritization potential drug targets                     Trypanosomatids encode complement class 1 and <br /> major pathogens (7, 8). The premise undertaking idea                   class 2 aaRSs; bioinformatics analysis Leishmania major (20) <br /> that gene products exploited treatment hu-                  T. brucei (21) identified 25 24 genes encoding canonical <br /> man disease likely represent druggable targets                 aaRS homologs, respectively, covering key amino acids. Addi- <br /> for information available (9). Druggable targets                     tionally, encode multiple-aaRS (MARS) complex-asso- <br /> further defined bind small drug-like molecules high                ciated proteins (MCPs). The MCPs share sequence identity with <br /> potency, resulting disease-modifying outcomes. These reported                      aaRS noncatalytic domains aaRS editing domains do <br /> computational approaches utilized availability inhibitors for <br /> homologs gene drug-like properties inhibitors to <br /> assign targets druggability score.                                                        Received 15 January 2014 Accepted 13 February 2014 <br />     The translation apparatus, including ribosomes, specific elon-                          Published ahead print 21 February 2014 <br /> gation initiation factors, aminoacyl-tRNA synthetases                               Address correspondence Margaret A. Phillips, <br /> (aaRSs), represents major pathways targeted com-                              margaret.phillips@UTSouthwestern.edu. <br /> mercial antibiotics (10&#226;&#8364;&#8220;14). The aaRSs essential participants                        Supplemental material article http://dx.doi.org/10.1128 <br /> the protein translation mechanism, catalyzing esterification                         /EC.00017-14. <br /> specific amino acids corresponding tRNAs (15&#226;&#8364;&#8220;18) (Fig.                            Copyright &#194;&#169; 2014, American Society Microbiology. All Rights Reserved. <br /> 1). Two classes aaRSs described,                           doi:10.1128/EC.00017-14 <br /> turn divided additional subclasses. These classes are <br />  <br />  <br /> 504   ec.asm.org                                            Eukaryotic Cell   p. 504 &#226;&#8364;&#8220;516                                                  April 2014 Volume 13 Number 4 <br />                                                                                                                         T. brucei Aminoacyl tRNA Synthetases <br />  <br />  <br />  <br />                                                                              (GluRS) obtained particle counting, detects cell debris. <br />                                                                              All growth data collected biological triplicate, exception <br />                                                                              PheRS&#226;?&#163; GluRS data, collected duplicate. To <br /> FIG 1 Reactions catalyzed aaRR. AA, amino acid substrate; aaRS:AA-AMP,    determine 50% effective concentration (EC50) borrelidin, the <br /> enzyme-bound aminoacyl-adenylate intermediate; AA-tRNA, amino acyl <br />                                                                              dose-response data fitted log(inhibitor) versus response &#226;&#171;&#186; <br /> tRNA. <br />                                                                              variable slope (4-parameter) equation using GraphPad Prism. <br />                                                                                  Generation cell lines containing inducible RNAi constructs. RNA <br />                                                                              interference (RNAi) cell lines generated using gateway vector <br /> not aminoacylation activity (21). Both cytosolic translation            (TrypRNAiGate) following genes: AsnRS (Tb927.4.2310), PheRS&#226;?&#163; <br /> and mitochondrial translation require aaRSs. Trypanosomatids,                subunit (Tb927.11.14120), PheRS&#226;?&#164; subunit (Tb927.11.2360), ValRS <br /> unlike eukaryotes, <span id='am-2' about='Thesaurus:code' typeof='owl:Thing'>code</span> tRNA molecules             (Tb927.6.4480). A 400- 500-bp region gene selected using <br /> mitochondrial genome; instead, mitochondria import                     RNAit software (41). PCR products generated using High Fidelity <br /> tRNAs aaRSs cytosol (13, 22, 23). Almost aaRSs           Platinum Taq DNA polymerase (Invitrogen) following forward <br /> the T. brucei genome single copy serve dual roles          reverse primer pairs: AsnRS, 5=-(CGGGCTACATTCAAGTGGT <br /> in cytosol mitochondria; TrpRS, LysRS, AspRS                 T)-3= 5=-(CAAAATTGCACACCGCATAC)-3=; PheRS&#226;?&#163; subunit, <br />                                                                              5=-(TTTTAAGTCGCTGGAAGCAGGT)-3= 5=-(GGGGGCCCACGT <br /> three notable exceptions separate cytosolic mitochon- <br />                                                                              CTAACACTAA)-3=; PheRS&#226;?&#164; subunit, 5=-(AATGGGTACTGCGATG <br /> drial genes identified (24&#226;&#8364;&#8220;26). Several T. brucei aaRS             AAGG)-3= 5=-(TGATGCACGAGTAGTCCGAG)-3=; ValRS <br /> genes reported essential parasites          5=-(GGGTACGACAGGAAGGTTGA)-3= primer 5=-(AATGTCACC <br /> insect stage (procyclic) mammalian stage (21, 25&#226;&#8364;&#8220;30). Sev-            GGTCCTACTGC)-3=. Gel-purified PCR fragments incubated <br /> eral groups reported identification MetRS, IleRS,           pCR8/GW/TOPO vector (supplied ready use) 5 min room <br /> and LeuRS inhibitors antitrypanosome activity (30&#226;&#8364;&#8220;35).                  temperature, transformed Top10 Escherichia coli, plated spec- <br /> Crystal structures solved aaRSs             tinomycin (100 &#226;?&#174;g/ml)-containing LB plates described Gateway <br /> several species (15), including structures HisRS (36)             pENTR vectors instruction manual (Invitrogen Life Technologies). Se- <br /> TrpRS (37) T. brucei, aid understanding            quence-verified plasmids used LR recombination re- <br /> target enzymes HAT drug discovery. Finally, recently            action pTrypRNAiGate destination vector generate stem-loop <br /> showed number aaRSs T. brucei associate                constructs, parasite transfections performed previ- <br /> MARS complex (21). However, despite extensive efforts,               ously described (38, 39). Inducible RNAi plasmids ArgRS <br />                                                                              (Tb927.11.1990), ThrRS (Tb927.5.1090), TrpRS-1 (Tb927.3.5580), <br /> number T. brucei key aaRSs remain uncharacterized. <br />                                                                              GluRS (Tb927.6.4590) generated using pQuadra system <br />     Since aaRSs successful drug targets species,          (42). Briefly, &#226;&#172;&#402;400-bp fragment gene selected using RNAit <br /> we undertook comprehensive study identify essential aaRSs            software (41) amplified PCR, using following forward reverse <br /> T. brucei goal demonstrating potential utility        primer pairs, containing added BstXI sites 5= end: ArgRS, 5=-(A <br /> new class targets antitrypanosome drug discovery. The pres-            TACCAATGTGATGGGCTTGTGGACTTTTCTTCGC)-3= 5=-(ATAC <br /> ent study focused genetic validation uncharacter-            CATAGAGTTGGCTTTGCGTCCAGTGCAGATA)-3=; ThrRS, 5=-(ATA <br /> ized T. brucei aaRSs RNA interference (RNAi) gene knockdown               CCAATGTGATGGATGTTTATGACGTCCTCGGC)-3= 5=-(ATACCA <br /> in infectious mammalian bloodstream-form (BSF) parasites.                TAGAGTTGGGTGCAGCCTTCTCGTAGGAC)-3=; TrpRS-1, 5=-(ATA <br /> Genes selected high druggablity scores en-              CCAATGTGATGGCTGCTATACAGGCTGCACCA)-3= 5=-(ATACCA <br /> compass examples major aaRS classes. We                TAGAGTTGGCCAATTTCCGTTTCTCCTGA)-3=; GluRS, 5=-(AT <br /> here newly characterized genes essential mam-         ACCAATGTGATGGTTTACGGGCAAAGATATCCG)-3= 5=-(ATACC <br />                                                                              ATAGAGTTGGCAACAACACATCGGACCTTG)-3=. Ligation BstXI- <br /> malian BSF parasites. There partial correlation the <br />                                                                              digested PCR fragments pQuadra3 plasmids generated RNAi vectors, <br /> extent RNAi knockdown effects cell growth, mor- <br />                                                                              containing inverted repeats PCR product separated spacer region. <br /> phology, cell cycle, cellular DNA. A heterologous expression             Transfection NotI-linearized constructs SM cell line (43) se- <br /> system enzyme assay developed threonyl-tRNA                  lection transgenic cell lines (44) carried described previously. <br /> synthetase (ThrRS) ready target future work drug               Quantitative real-time PCR analysis. Quantitative real-time PCR was <br /> discovery program.                                                           performed using SYBR green Mastermix (Bio-Rad). Total RNA ex- <br />                                                                              tracted parasites using TRIzol reagent (Invitrogen) RNeasy <br /> MATERIALS AND METHODS                                                        minikit (Qiagen), treated DNase I (Worthington), reverse tran- <br /> TriTrypDB accession numbers genes analyzed study. Putative       scribed using High Capacity cDNA reverse transcription kit (Applied <br /> aaRS genes selected based annotation TriTrypDB,     Biosystems). An approximately 100-bp fragment gene ampli- <br /> following studied: ThrRS (T. brucei 927.5.1090 [Tb927.5.1090]),         fied using SYBR green Mastermix CFX96 real-time (Bio- <br /> arginyl-tRNA synthetase (ArgRS) (Tb927.11.1990), asparaginyl-tRNA            Rad). Reaction conditions maintained according manufacturer&#226;&#8364;&#8482;s <br /> synthetase (AsnRS) (Tb927.4.2310), phenylalanyl-tRNA synthetase alpha        instructions follows: 95&#194;&#176;C 1 min 40 cycles 95&#194;&#176;C 3 s <br /> (PheRS&#226;?&#163;) subunit (Tb927.11.14120), PheRS&#226;?&#164; subunit (Tb927.11.2360),           60&#194;&#176;C 30 s, followed continuous melting curve (55&#194;&#176;C 95&#194;&#176;C <br /> tryptophanyl-tRNA synthetase (TrpRS-1) (Tb927.3.5580), valyl-tRNA            0.5&#194;&#176;C increments). Data normalized constitutively <br /> synthetase (ValRS) (Tb927.6.4480), glutamyl-tRNA synthetase              expressed control mRNA, telomerase reverse transcriptase (TERT), <br /> (GluRS) (Tb927.6.4590).                                                      validated control quantitative PCR (qPCR) T. <br />     Growth T. brucei BSF vitro cultures. BSF 427 single-marker         brucei (45). Gene expression levels quantified comparative <br /> (SM) cells cultured HMI-9 media supplemented 10% fetal          threshold cycle (&#226;&#338;&#172;&#226;&#338;&#172;CT) method (46) TERT internal standard <br /> bovine serum described previously (38&#226;&#8364;&#8220;40). Cells maintained       normalization. Oligonucleotides follows: forward primer 5=- <br /> exponential growth (between 105 106 cells/ml), RNAi in-          (GAGCGTGTGACTTCCGAAGG)-3= reverse primer 5=-(AGGAAC <br /> duced cell density 2 &#226;&#171;&#187; 105/ml addition tetracycline (Tet) (1   TGTCACGGAGTTTGC)-3= TERT, forward primer 5=-(AATGCGAA <br /> &#226;?&#174;g/ml) media. Cell numbers monitored hemocytometer,           GGATTTGTCCAC)-3= reverse primer 5=-(TGCTCGGAGAGTTTC <br /> with exception data collected glutamyl-tRNA synthetase      ACCTT)-3= AsnRS, forward primer 5=-(TTATGGAGATGGGCTTCC <br />  <br />  <br /> April 2014 Volume 13 Number 4                                                                                                               ec.asm.org 505 <br />                                                                                                                                                                                                                                                             Eukaryotic Cell <br />                  TABLE 1 Classification druggability T. brucei aaRSsa <br />                                                                                                                                                                                                                           % RNA          Loss-of-fitness <br />                                                                                               No. BSF                                                                                            BSF RNAi growth       knockdown      BSF RNAi <br />                  Product     Gene ID             Class   Localization    MARS complex         RNA reads     Druggability    Chemical validation                      Structure(s) available         defect                (24 h)         screen (3) <br />                  AlaRS       Tb927.6.700         IIc     C/M             Y                    589.1         0.2                                                                                     Severe (21)           78             Yes <br />                  ArgRS       Tb927.11.1990       Ic      C/M             Y (Homo sapiens)     302.2         0.6                                                                                     Severe                79             Yes <br />                  AsnRS       Tb927.4.2310        IIb     C/M             Y                    566.3         No data                                                                                 Severe                60             Yes <br />                  AspRS-2     Tb927.10.1260       IIb     M                                    108.4         No data         Microcin C (63)                                                                                              No <br />                  AspRS-1     Tb927.6.1880        IIb     C                                    98.3          0.2             Microcin C (63)                          Y (Homo sapiens)                                                    No <br />                  CysRS       Tb927.6.950         Ia      C/M             Y                    517.0         No data                                                                                                                      No <br />                  GlnRS       Tb927.9.5210        Ic      C/M                                  345.0         0.8             Glu-KPA (E. coli) (64)                   Y (Saccharomyces cerevisiae)                                        Yes <br />                  GluRS       Tb927.6.4590        Ic      C/M             Y                    465.1         0.3                                                      Y (Saccharomyces               Moderate                             Yes <br />                                                                                                                                                                         cerevisiae) <br />                  GlyRS       Tb927.11.9640       IIc     C/M             Y                    302.4         No data                                                                                                                      Yes <br />                  HisRS       Tb927.6.2060        IIa     C(M?)                                100.0         0.6                                                      Y (T. brucei)                  Severe (36)           58             Yes <br />                  IleRS       Tb927.10.9190       Ia      C/M             Y                    1000.2        0.9             Reveromicin (S. aureus) (65);            Y (Homo sapiens)               Severe (21)           80             Yes <br />                                                                                                                                muporicin (S. enterica) (66); <br />                                                                                                                                NSC70422 (T. brucei) (21); PLD- <br />                                                                                                                                118 (icofungipen) (C. albicans) <br />                                                                                                                                (67); SB-203207 (Streptomyces) <br />                                                                                                                                (68, 69) <br />                  LeuRS       Tb927.11.3730       Ia      C/M                                  320.1         0.7             Granaticin (B. subtilis) (67);                                          Moderate (21)         64             No <br />                  LysRS-1     Tb927.8.1600        IIb     C/M                                  400.2         No data         Cladosporin (Plasmodium                  Y (Homo sapiens,               Severe (21)           73             No <br />                                                                                                                                falciparum) (19)                         Plasmodium falciparum) <br />                  LysRS-2     Tb927.6.1510        IIb     M                                    105.9         No data                                                                                                                      No <br />                  MetRS       Tb927.10.1500       Ia      C/M             Y                    326.4         0.7             Urea-based inhibitors (T. brucei) (33,   Y (T. brucei, Leishmania       Moderate (21)         77             No <br />                                                                                                                               35); REP8839 (S. aureus) (70)             major) <br />                  PheRS-&#226;?&#163;     Tb927.11.14120      IIc     C/M             Y (Saccharomyces     283.1         0.6             Ochratoxin A B (hepatoma                                            Moderate              57             No <br />                                                                             cerevisiae)                                       tissue culture) (71); <br />                                                                                                                               ethanolamine-based inhibitors <br />                                                                                                                               (Staphylococcus aureus) (72) <br />                  PheRS-&#226;?&#164;     Tb927.11.2360       IIc     C/M                                  248.2         No data                                                                                 Severe                73             Yes <br />                  ProRS       Tb927.10.12890      IIa     C/M             Y                    859.4         0.8             Halofuginone (Plasmodium) (73)           Y (Homo sapiens)               Severe (21)           53             Yes <br />                  SerRS       Tb927.11.7170       IIa     C/M                                  308.5         No data         SB-217452 (rat, S. aureus) (74)          Y (T. brucei, Homo sapiens)                                         Yes <br />                  ThrRS       Tb927.5.1090        IIa     C/M             Y (Saccharomyces     372.0         0.6             Borrelidin (fungi, bacteria) (53, 55,    Y (Saccharomyces               Severe                84             No <br />                                                                             cerevisiae)                                       56); T. brucei (this work)                cerevisiae, Homo <br />                                                                                                                                                                         sapiens, Escherichia <br />                                                                                                                                                                         coli) <br />                  TrpRS-1     Tb927.3.5580        Ib      C               Y                    168.4         0.2             Chuangxinmycin (bacteria) (73);          Y (T. brucei), Plasmodium      serere                59             No <br />                                                                                                                               indolmycin (Streptomyces) (73)            falciparum <br />                  TrpRS-2     Tb927.8.2240        Ib      M                                    117.4         0.2                                                      Y (Plasmodium falciparum)      Severe (conditional                  No <br />                                                                                                                                                                                                        null) (27) <br />                  TyrRS       Tb927.7.3620        Ib      C/M                                  200.0         0.6             3-Aryl-4-arylaminofuran-2(5H)-ones       Y (Leishmania major)                                                No <br />                                                                                                                               (bacteria) (75); SB-219383 <br />                                                                                                                               (Micromonospora sp.) (76) <br />                  ValRS       Tb927.6.4480        Ia      C/M                                  402.0         0.9             S-Adenosylhomocysteine (77)                                             Moderate              63             No <br />                  MCP1        Tb927.7.2400                                                     221.2         No data                                                                                 Unknown <br />                  MCP2        Tb927.8.5330                C                                    80.6                                                                                                  Moderate (21)         100 <br />                                                                                                                                                                                                       (conditional <br />                                                                                                                                                                                                       null) <br />                  MCP3        Tb927.10.1250                                                    68.7                                                                                                  Unknown <br />                  a <br />                    Gene names (http://tritrypdb.org/tritrypdb/) druggability scores (tdrtargets.org) taken indicated web sites. Enzymes evaluated study shown bold type. Loss-of-fitness data RNAi screen per- <br />                  formed Alsford et al. (3) summarized comparison. BSF RNA reads scaled read counts (median normalized; median 100) taken reference 78. MARS complex-associated proteins (MCP) 1 1, 2, 3 shown, <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                            ec.asm.org <br />                  aaRS activity. C, cytoplasm; M, mitochondrion; C/M, cytoplasm mitochondrion; Glu-KPA, glutamyl-beta-ketophosphonate-adenosine; Y, yes. Numbers parentheses refer publications cited. <br /> Kalidas et al. <br />  <br />  <br />  <br />                  ID, identification number. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                            506 <br />                                                                                                                          T. brucei Aminoacyl tRNA Synthetases <br />  <br />  <br />  <br /> AG)-3= reverse primer 5=-(GCGGGTTTTGAAATGAAGAA)-3=                         Aminoacylation spectrophotometric assays. Amino acylation assays <br /> PheRS&#226;?&#163; subunit, forward primer 5=-(GGAGGACCAATTCGAAGA                          performed previously described (50). Briefly, T. brucei tRNAThr <br /> CA)-3= reverse primer 5=-(AGAAAGCTCCTCTCCCTTGG)-3=                     synthesized vitro transcription PCR product template <br /> PheRS&#226;?&#164; subunit, forward primer 5=-(AAGAAGTATCACGGTAAGC)-3=                     contained T7 promoter followed tRNAThr sequence <br /> and reverse primer 5=-(TCACATCAACCAGAACAG)-3= ValRS, for-                  (Tb927.8.2858) CCA sequence. The tRNAThr template pro- <br /> ward primer 5=-(CGCTATGGGTCTTGTCAAGCT)-3= reverse                          duced PCR using primers overlap tRNAThr sequence: A forward <br /> primer 5=-(CAGTGCCGGGCTTGTCTTT)-3= ArgRS, forward                          [5=-(TAATACGACTCACTATAGGGGCCTCGTAGCACAGTGGC)-3=], B <br /> primer 5=-(CCCGCGGCAAGTTATTGT)-3= reverse primer 5=-(TG                    reverse [5=-(TGGAGGCCCCGCCGAG)-3=], C forward [5=-(GGCCTCGTA <br /> TCGCGCACCGTTTGT)-3= ThrRS, forward primer 5=-(ACGAG                    GCACAGTGGCAGTGCACTGGTCTTGTAAACC)-3=], D reverse [5=- <br /> TCCGGTCAACCTTTTT)-3= reverse primer 5=-(TCAAATGACCCA                       (AGGCCCCGCCGAGAATTGAACTCGGGACCCCTGGTTTACAAGAC <br /> TATGCATGGA)-3= TrpRS-1.                                                    CAGTGCAC)-3=]. The standard reaction ingredients, unless otherwise <br />     Cell morphology analysis. Log-phase cells collected centri-        stated, aminoacylation buffer (30 mM HEPES buffer [pH 7.4], 140 mM <br /> fuging 3,000 rpm 10 min 15-ml tube. Cells resuspended      NaCl, 30 mM KCl, 40 mM MgCl2), 1 mM dithiothreitol (DTT), 200 &#226;?&#174;M <br /> freshly 4% paraformaldehyde (PFA), fixed room temperature          ATP, 2 U/ml inorganic pyrophosphatase (Sigma-Aldrich), 10 mM L-Thr <br /> 45 min, washed phosphate-buffered saline (PBS; pH 7.4), resus-        (Sigma-Aldrich), 8 &#226;?&#174;M T. brucei tRNAThr, 0.4 &#226;?&#174;M recombinant <br /> pended 50 100 &#226;?&#174;l PBS. Cells (20 &#226;?&#174;l) placed poly-L-lysine-      ThrRS. The aminoacylation reactions (55 &#226;?&#174;l total volume each) per- <br /> coated slide, spread, dried 1 h. For nuclear (N) kinetoplast (K)   formed clear, flat-bottom, 96-well plates (Costar 96-well standard mi- <br /> DNA, cells applied coverslips Vectashield mounting me-        croplates), reaction mixtures incubated 30 min 37&#194;&#176;C. <br /> dium containing DAPI (4=,6-diamidino-2-phenylindole; hard set)             Liberated inorganic phosphate detected addition 100 &#226;?&#174;l mal- <br /> analyzed fluorescence confocal microscopy (Leica TCS SP5) mag-         achite green (Echelon Biosciences) incubated 30 min room <br /> nification &#226;&#171;&#187;63 excitation wavelength 350 nm bright-        temperature. Absorbance measured 620 nm using SpectraMax <br /> field differential interference contrast (DIC). Both uninduced Tet-        M2 microplate reader (Molecular Devices). Reactions enzyme or <br /> induced cultures examined multiple time intervals. The                 L-Thr performed background controls, data <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> percentage morphological anomalies (number nuclei, kinetoplasts,         reactions L-Thr subtracted measurements. For <br /> and phenotypic forms) determined counting 150 cells.                    time course experiments, aliquots 10 &#226;?&#174;l withdrawn 80-&#226;?&#174;l <br />     Flow cytometric analysis. For cell cycle analysis, BSF T. brucei       reaction mixture different time points (0, 2.5, 5, 10, 20, 30, 60 min) <br /> inoculated 105 parasites/ml cultures harvested 0, 2,    mixed 10 mM EDTA ice stop reaction. For ThrRS <br /> 3 days plating. Cells washed PBS (pH 7.4) fixed 70%         inhibition borrelidin, reaction mixture containing ThrRS (0.4 &#226;?&#174;M) <br /> ethanol resuspended PBS containing 10 &#226;?&#174;g ml&#226;&#171;&#186;1 RNase A 10            mixed 5 &#226;?&#174;l borrelidin 10 10,000 nM 50-&#226;?&#174;l volume <br /> &#226;?&#174;g ml&#226;&#171;&#186;1 propidium iodide (PI; Sigma). After incubation 30 min           incubated 30 min 37&#194;&#176;C. Reactions stopped described <br />                                                                                above. All data shown replicates means &#226;&#171;&#190; standard <br /> 37&#194;&#176;C, PI-stained cells analyzed flow cytometry described pre- <br />                                                                                errors. To determine 50% inhibitory concentration (IC50) borre- <br /> viously (47, 48). Histograms analyzed populations 2C, 4C, <br />                                                                                lidin, dose-response data fitted log(inhibitor) versus re- <br /> and &#226;&#172;&#381;4C measurement relative DNA content. <br />                                                                                sponse &#226;&#171;&#186; variable slope (4-parameter) equation using GraphPad Prism <br />     Immunofluorescence localization assay. The gene encoding ThrRS <br />                                                                                slope restrained 1.0. <br /> (Tb927.5.1090) amplified PCR genomic DNA T. brucei <br /> strain Lister 427 using specific primers forward 5=-(CCCAAGCTTATGG <br /> CCACGGCTGTTG)-3= reverse 5=-(CCCAGATCTGAAGGACGGA                           RESULTS <br /> TCATATTTATCGG)-3= cloned HindIII BamHI sites                   Gene selection. Enzymes selected analysis essentiality <br /> pLEW100 vector 3V5 tag C terminus. Plasmids digested        T. brucei based criteria: first, interested genes <br /> with NotI transfected T. brucei SM 427 BSF cells, resistant       high druggability scores; second, sought study repre- <br /> clones selected 5 &#226;?&#174;g/ml phleomycin (49). Expression        sentative major aaRS classes subclasses (Ta- <br /> 3V5-tagged protein induced 0.5 &#226;?&#174;g/ml Tet. Cells growing            ble 1). Of selected genes, ThrRS, ArgRS, AsnRS, ValRS, and <br /> mid-log phase fixed 4% paraformaldehyde&#226;&#8364;&#8220;PBS (pH 7.4), ad-            GluRS single-copy genes dual cytosolic mito- <br /> hered poly-L-lysine-treated 12-mm-diameter cover glass (Fisher),        chondrial localizations based reported proteomic analysis <br /> permeabilized 0.2% Triton X-100 &#226;&#8364;&#8220;PBS. Coverslips blocked <br />                                                                                (21, 51). In contrast, TrpRSs encoded T. brucei <br /> with 3% bovine serum albumin (BSA)&#226;&#8364;&#8220;PBS labeled monoclonal <br /> antibodies 3V5 tag (anti-V5-fluorescein isothiocyanate [FITC; <br />                                                                                genome, cytosolic enzyme analyzed study. Two <br /> Sigma] conjugated) diluted 1:500 PBS&#226;&#8364;&#8220;3% BSA. Mitochon-            subunits PheRS studied; alpha subunit consists the <br /> drial staining performed treating cells 0.01 &#226;?&#174;g/ml Mito-           catalytic core, beta subunit consists DNA ATP <br /> Tracker (Molecular Probes/Invitrogen) 30 min 37&#194;&#176;C 5% C02             binding sites. Druggability scores obtained tdrtarget- <br /> before fixation. DNA stained described above. Cells analyzed       s.org showed good predicted druggability ThrRS, PheRS, <br /> with Deltavision fluorescence microscope (Olympus IX70).                     ArgRS, ValRS (&#226;&#172;&#381;0.6 based range 0 1, 1 <br />     Cloning, expression, purification recombinant protein. The          representing druggable targets [7, 8]), suggesting that, if <br /> gene ThrRS (Tb927.5.1090) amplified PCR using specific prim-        essential, strong candidates drug discovery pro- <br /> ers forward 5=-(CCCGAATTCATGGCCACGGCTGTTGTT)-3= re-                        grams. Druggability data available AsnRS, TrpRS-1 <br /> verse 5=-(CCCGTCGACGAAGGACGGATCATATTT)-3= genomic                         GluRS score 0.6 threshold good druggability. <br /> DNA T. brucei strain SM 427 cloned pET28a&#226;&#171;&#185; vector (Nova-              Aminoacyl tRNA synthetases essential BSF T. brucei <br /> gen) using EcoRI XhoI restriction sites C-terminal 6&#226;&#171;&#187;His tag. <br />                                                                                growth. To test selected aaRSs essentiality T. brucei BSF <br /> The constructs transformed E. coli Rosetta 2(DE3) strain pLysS <br /> (Novagen), protein expression induced 0.1 mM IPTG (iso- <br />                                                                                parasites, established stable transgenic cell lines allow <br /> propyl-&#226;?&#164;-D-thiogalactopyranoside). E. coli grown LB media           Tet-inducible RNAi-mediated ablation enzyme expression. <br /> harvested, lysate prepared Bugbuster reagent (Novagen).         Stem-loop RNAi constructs generated using Gate- <br /> Proteins purified using Ni&#226;&#171;&#185;2 magnetic beads (Millipore), dialyzed         way pQuadra cloning enzymes ThrRS, AsnRS, <br /> against 50 mM Tris (pH 7.4)&#226;&#8364;&#8220;140 mM NaCl, kept 4&#194;&#176;C (with 0.05%           PheRS&#226;?&#163;, PheRS&#226;?&#164; belonging class II tRNA synthetases and <br /> sodium azide) use.                                                       ArgRS, TrpRS-1, ValRS, GluRS belonging class I aaRSs <br />  <br />  <br /> April 2014 Volume 13 Number 4                                                                                                               ec.asm.org 507 <br />  Kalidas et al. <br />  <br />  <br />  <br />  <br /> FIG 2 Effect RNAi-induced knockdown ThrRS BSF T. brucei. Independent RNAi cell lines generated BSF cells ThrRS analyzed growth and <br /> for cell cycle progression following RNAi induction. (A) Growth curves clonal lines Tet (data represent means, error bars <br /> represent ranges). Cell numbers monitored hemocytometer, cell number calculated product cell density total dilution. (B) <br /> Real-time qPCR analysis uninduced cells cells treated Tet 24 h 48 h relative TERT normalization standard. Analyses run in <br /> triplicate, error bars represent root mean square deviation (RMSD). (C) Genome analysis microscopy: distributions nuclei (N) kinetoplasts <br /> (K). Cells grown Tet 48 h, stained DAPI, examined fluorescence microscopy. A total 150 cells counted condition. <br /> (D E) Flow analysis cells minus (D) plus (E) Tet 48 h stained propidium iodide (PI). PI fluorescence (FL3-Area) plotted versus cell count. <br /> The tallest peak panel D represents G0/G1 (48%), span peaks represents S phase (4.59%), second tallest peak represents G0/M (27.6%). <br /> (F) Confocal imaging representative cells Tet induction (48 h). <br />  <br />  <br />  <br /> (Table 1). Figures 2, 3, 4, 5, 6, 7 characterization               trol wild-type SM cells stained PI DNA content was <br /> each aaRS knockdowns, Fig. 8 depicts character-                   analyzed flow cytometry. Cell morphology numbers of <br /> istics parental line comparative purposes. For 8             nuclei (N) kinetoplasts (K) additionally analyzed by <br /> targeted genes, induction RNAi led significant growth defects                confocal microscopy following staining DAPI. Cells were <br /> within 2 3 days, showing 8 genes essential parasite            scored &#226;&#8364;&#339;normal&#226;&#8364;? contained 1K1N 2K1N/2K2N <br /> viability (Fig. 2A, panels A Fig. 7, Table 2). These                   &#226;&#8364;&#339;abnormal&#226;&#8364;? contained different genome compo- <br /> growth defects correlated decreased mRNA levels ranging                       sition. The majority (90% 94%) control populations had <br /> from 55% 85% determined RT-PCR analysis using TERT                        normal <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span>phenotype</span>, contained 1K1N, lesser <br /> as control (Fig. 2B 7B Table 2) (RT-PCR analysis                     fraction dividing cells (2K1N 2K2N) observed (Fig. 8A <br /> not GluRS cell knockdown line). Knockdown                     B). Based flow cytometry, 53% cells G0/G1 <br /> genes (ThrRS, AsnRS, PheRS&#226;?&#164;, ArgRS) led rapid cell death,                   phase, 4% S phase, 26% G2/M phase. <br /> and evidence live cells detected past day 6. Knock-                     Dramatic morphological effects cell cycle changes were <br /> down remaining genes (PheRS&#226;?&#163;, TrpRS-1, ValRS,                           observed ThrRS, ArgRS, AsnRS RNAi cell lines and, to <br /> GluRS) stopped growth did completely lyse cells                   lesser extent, PheRS&#226;?&#164;, morphological abnormalities <br /> though level mRNA knockdown similar level seen                   observed early 16 h Tet induction (Fig. 2 5 and <br /> with genes showed robust cell death effect. Of                   Table 2). Flow cytometry suggested knockdown ThrRS led <br /> note, however, changes protein levels necessarily track                 accumulation cells low DNA content (Fig. 2D E). <br /> with mRNA levels, protein levels measured,                  This corresponded appearance 0N1K cells (4%) and <br /> it remains possible protein levels decreased significantly               1N0K cells (3%) (quantitated confocal microscopy) (Fig. 2C). <br /> in cell lines showed robust effects.                             There increase number 2N2K cells suggestive <br />     Morphology flow cytometric analysis T. brucei BSF                       block cytokinesis (Fig. 2C). Phenotypically, large percent- <br /> cells aaRS knockdown. To explore consequences gene                    age Tet-induced ThrRS RNAi cells rounded, detached <br /> knockdown observed growth arrest, RNAi cell                         flagella, lacked motility (Fig. 2F). Knockdown ArgRS led to <br /> lines Tet induced uninduced control cells con-                    rapid cell death increase number rounded cells <br />  <br />  <br /> 508   ec.asm.org                                                                                                                                     Eukaryotic Cell <br />                                                                                                                           T. brucei Aminoacyl tRNA Synthetases <br />  <br />  <br />  <br />  <br /> FIG 3 Effect RNAi-induced knockdown ArgRS BSF T. brucei. (A) Growth curves. Cell numbers monitored hemocytometer. (B) Real-time qPCR <br /> analysis. (C) Genome analysis microscopy. (D E) Fluorescence-activated cell sorter (FACS) analysis. (F) Confocal imaging. Details described for <br /> Fig. 2. <br />  <br />  <br /> with detached flagella (Fig. 3F), significant change DNA              creased populations 0N1K (up 20%), 1N0K, 1N2K cells <br /> content seen (Fig. 3D E). Knockdown AsnRS led                     (Fig. 4C E) suggestive block entering S phase. A signifi- <br /> increases abnormal complements nuclei kinetoplasts.                   percentage Tet-induced AsnRS RNAi cells either <br /> Both confocal microscopy flow cytometry identified in-                      spherical round detached flagella flagella <br />  <br />  <br />  <br />  <br /> FIG 4 Effect RNAi-induced knockdown AsnRS BSF T. brucei. (A) Growth curves. Cell numbers monitored hemocytometer. (B) Real-time qPCR <br /> analysis. (C) Genome analysis microscopy. (D E) Flow analysis. (F) Confocal imaging. Details described Fig. 2. <br />  <br />  <br />  <br /> April 2014 Volume 13 Number 4                                                                                                                ec.asm.org 509 <br />  Kalidas et al. <br />  <br />  <br />  <br />  <br /> FIG 5 Effect RNAi-induced knockdown PheRS BSF T. brucei. (Top panels) Knockdown PheRS&#226;?&#163;. (A) Growth curves. Cell numbers monitored <br /> by hemocytometer. (B) Real-time qPCR analysis. (C) Genome analysis microscopy. (D E) Flow analysis. (F) Confocal imaging. (Bottom panels) <br /> Knockdown PheRS&#226;?&#164;. (A) Growth curves. Cell numbers monitored hemocytometer. (B) Real-time qPCR analysis. (C) Genome analysis micros- <br /> copy. (D E) Flow analysis. (F) Confocal imaging. Details described Fig. 2. <br />  <br />  <br />  <br /> (Fig. 4F [arrow arrowhead]). After knockdown PheRS&#226;?&#164;,               8C DNA content suggesting block cytokinesis (Fig. 5D <br /> the DNA content normally distributed (Fig. 5C E,            E, panel), cell morphology looked fairly normal <br /> panel) cells somewhat enlarged (Fig. 5F, panel).          (Fig. 5F, panel). Knockdown TrpRS-1 ValRS did not <br />    Milder notable effects cell cycle cell morphology       alter cell morphology (data shown), knockdown of <br /> were observed PheRS&#226;?&#163; knockdown (Fig. 5). Knockdown                 genes reduced rate cell growth, remaining cells had <br /> PheRS&#226;?&#163; led strong increase populations cells, 4C          normal distribution DNA content (Fig. 6 7). Thus, RNAi <br />  <br />  <br /> 510   ec.asm.org                                                                                                                     Eukaryotic Cell <br />                                                                                                                    T. brucei Aminoacyl tRNA Synthetases <br />  <br />  <br />  <br />                                                                             84%) showing greater growth defects, exception that <br />                                                                             AsnRS cell lines showed decrease mRNA 60%, <br />                                                                             sufficient lead cell lysis (Table 2). <br />                                                                                 ThrRS enzymatic activity. Based strong cell death phe- <br />                                                                             notype observed ThrRS knockdown, decided character- <br />                                                                             ize protein greater detail. Recombinant ThrRS ex- <br />                                                                             pressed purified E. coli, used coupled-enzyme <br />                                                                             assay evaluate enzymatic activity following genera- <br />                                                                             tion pyrophosphate conversion inorganic phosphate, <br />                                                                             detected malachite green (50). ThrRS acylated <br />                                                                             tRNAThr time- enzyme concentration-dependent man- <br />                                                                             ner, demonstrating T. brucei ThrRS gene encodes func- <br />                                                                             tional enzyme (Fig. 9A B). The assay remained linear 30 <br />                                                                             min, taken representing steady-state phase the <br />                                                                             reaction, time point used subsequent studies. <br />                                                                             Enzyme concentrations 0.25 &#226;?&#174;M led loss linear <br />                                                                             dependence enzyme activity; thus, 0.25 &#226;?&#174;M enzyme used <br />                                                                             subsequent studies. The specific activity enzyme under <br />                                                                             conditions 1.4 min&#226;&#171;&#186;1, comparable published <br />                                                                             values yeast enzyme (kcat &#226;&#171;&#189; 2.8 min&#226;&#171;&#186;1 [52]). <br /> FIG 6 Effect RNAi-induced knockdown TrpRS-1 BSF T. brucei. (A)         To provide chemical validation ThrRS drug target T. <br /> Growth curves. Cell numbers monitored hemocytometer. (B) Real-      brucei, tested reported ThrRS inhibitor borrelidin against <br /> time qPCR analysis. (C D) FACS analysis. Details described for <br /> Fig. 2.                                                                     T. brucei ThrRS enzyme and, activity, BSF T. <br />                                                                             brucei cells vitro. Borrelidin natural-product polyketide that <br />                                                                             reported broad-spectrum antimicrobial anticancer <br /> knockdown aaRSs resulted morpholog-              activity (53&#226;&#8364;&#8220;57). Borrelidin potent inhibitor T. brucei <br /> ical and/or cell division abnormalities growth inhibition               ThrRS (IC50 &#226;&#171;&#189; 0.066 &#226;?&#174;M), result consistent the <br /> and/or death. There partial correlation degree            finding amino acids putative borrelidin binding <br /> of RNAi knockdown morphological growth defects,                 pocket conserved T. brucei bacterial enzymes <br /> with lines knockdown pronounced (73%                sensitive compound (53) (see Fig. S1 supple- <br />  <br />  <br />  <br />  <br /> FIG 7 (Top panels) Effect RNAi-induced knockdown ValRS BSF T. brucei. (A) Growth curves. Cell numbers monitored hemocytometer. (B) <br /> Real-time qPCR analysis. (C) Genome analysis microscopy. (Bottom panel) Effect RNAi-induced knockdown GluRS BSF T. brucei. (A) Growth <br /> curves. Cells counted particle counting, detects cell debris. Details described Fig. 2. <br />  <br />  <br />  <br /> April 2014 Volume 13 Number 4                                                                                                         ec.asm.org 511 <br />  Kalidas et al. <br />  <br />  <br />  <br />                                                                                tential family enzymes targets drug discovery T. <br />                                                                                brucei, assessed essentiality mammalian infective <br />                                                                                stage examples major aaRS class subclass (Table <br />                                                                                1) RNAi-based gene expression knockdown. RNAi resulted in <br />                                                                                growth inhibition them, evident be- <br />                                                                                tween days 1 4, depending cell line (Table 2). The cell <br />                                                                                lines exhibited differences morphology DNA content <br />                                                                                (revealed flow cytometry profiles) RNAi induction. Thus, <br />                                                                                aaRSs class essential normal growth mamma- <br />                                                                                lian stage T. brucei, data specifically provide genetic vali- <br />                                                                                dation seven key aaRSs potential drug targets parasite. <br />                                                                                Analysis recombinant T. brucei ThrRS demonstrated the <br />                                                                                enzyme catalytic efficiency similar enzymes from <br />                                                                                species inhibited borrelidin, natural- <br />                                                                                product inhibitor bacterial ThrRS. Borrelidin inhibited <br />                                                                                parasite growth, providing preliminary chemical validation for <br />                                                                                target. <br />                                                                                    To date, including data reported here, 16 24 aaRSs <br />                                                                                shown essential BSF T. brucei (Table 1). Addi- <br />                                                                                tionally, aaRSs examined genome-wide RNAi <br />                                                                                fitness screen BSF cells (58). Of genes studied here, <br />                                                                                AsnRS, PheRS &#226;?&#164;, ArgRS, GluRS important fitness in <br />                                                                                screen, knockdown PheRS&#226;?&#163;, ValRS, ThrRS, or <br />                                                                                TrpRS-1 did lead reduced fitness. The level RNA knock- <br />                                                                                quantified screen, illustrating importance <br />                                                                                detailed analysis obtain conclusive results. In genome- <br />                                                                                wide RNAi screen, highly essential genes, as <br />                                                                                aaRSs, leaky expression RNAi constructs absence <br />                                                                                Tet led selection parasites containing mutations the <br />                                                                                inducible T7 promoter. If occurred, RNAi longer be <br />                                                                                expressed, leading normal growth cells false read of <br />                                                                                nonessentiality. That study assessed essentiality the <br /> FIG 8 Cell morphology FACS analysis wild-type SM T. brucei cells. (A)   aaRSs localize mitochondrion, ap- <br /> Genome analysis microscopy. (B) Confocal imaging. (C D) Flow anal-      peared essential BSF parasites. However, RNAi showed that <br /> ysis. Details described Fig. 2.                                     AspRS TrpRS-1 essential insect (procyclic) <br />                                                                                stage (23, 24, 25) analysis conditional null mitochon- <br />                                                                                drial TrpRS-2 BSF parasites showed essential (27). Thus, <br /> mental material). T. brucei BSF cell growth inhibited                 mitochondrial aaRSs mitochondrial protein synthesis <br /> (EC50 &#226;&#171;&#189; 2.2 &#226;?&#174;M) (Fig. 9C), parasites 30-fold             essential mammalian stage parasites, im- <br /> sensitive compound purified recombinant enzyme.                portant drug discovery, mitochondrial aaRSs are <br /> This difference relate poor uptake borrelidin                 divergent eukaryotic enzymes (21). <br /> parasites, rule potential off-target effects                  While aaRSs appear essential, equal <br /> borrelidin parasites.                                                   potential drug targets implied differences responses <br />     Cell localization ThrRS. In order determine localiza-            RNAi expression knockdown. These included time course <br /> tion ThrRS enzyme, prepared transgenic cell lines allow- <br /> ing inducible expression ThrRS V5 tags C <br /> terminus. Cells stained DAPI (nuclei) MitoTracker                TABLE 2 Description T. brucei BSF RNAi knockdown phenotypes <br /> (mitochondria) mark compartments anti-V5                   analyzed studya <br /> antibody. A Tet-inducible diffuse staining tagged T. brucei                                             Time to <br />                                                                                                                growth <br /> ThrRS observed, consistent cytosolic localization (Fig.                                    % KD     defect     Cell morphology      DNA content FACS <br /> 10). However, merged image showed evidence mito-                  Figure     aaRS        (24 h)   (h)        (at 48 h)            microscopy <br /> chondrial staining marked appearance yellow                 2          ThrRS       84       48         Round lyse    Increase 0N, 0K, and <br /> the cell edges position mitochondria. This result                                                                                2K cells <br />                                                                                3          ArgRS       79       16         Round lyse    As wild type <br /> consistent T. brucei ThrRS gene encoding dual-purpose               4          AsnRS       60       16         Round lyse    Increase 0N, 0K, and <br /> aaRS functions cytoplasm mitochondria                                                                                    2K cells <br />                                                                                5          PheRS-&#226;?&#163;     57       48         Normal               As wild type <br /> is good agreement prior mass spectrometry data (21).               5          PheRS-&#226;?&#164;     73       24         Enlarge lyse     Increase 0N, 0K, and <br />                                                                                                                                                   2K cells <br /> DISCUSSION                                                                     6          TrpRS-1     59       24         Normal lyse      As wild type <br />                                                                                7          ValRS       63       48         Normal               As wild type <br /> The aaRSs validated antimicrobial drug targets high drug-             7          GluRS       ND       48         ND                   ND <br /> gability scores (11&#226;&#8364;&#8220;13). In order systematically evaluate po-           a <br />                                                                                    FACS, fluorescence-activated cell sorter analysis; KD, knockdown; ND, done. <br />  <br />  <br />  <br /> 512   ec.asm.org                                                                                                                                       Eukaryotic Cell <br />                                                                                                                              T. brucei Aminoacyl tRNA Synthetases <br />  <br />  <br />  <br />  <br /> FIG 9 Characterization ThrRS expression activity. (A) ThrRS activity assays recombinant protein purified E. coli. Data represent dependence of <br /> PO4-2 formation versus time. (B) Dependence PO4-2 formation ThrRS concentration. Two moles PO4-2 formed mole tRNAThr is <br /> acylated. (C) Dose response T. brucei BSF growth recombinant ThrRS activity versus concentration borrelidin, known ThrRS inhibitor. For inhibition <br /> of parasite growth, EC50 &#226;&#171;&#189; 2.2 &#226;?&#174;M (1.8 2.5); inhibition ThrRS activity, IC50 &#226;&#171;&#189; 0.066 &#226;?&#174;M (0.020 0.22), values parentheses represent <br /> the 95% confidence interval. <br />  <br />  <br /> of growth effects, effect cellular morphology, effect             zymes complex led more-severe growth pheno- <br /> on DNA flow cytometric profile. The data that,                   types knockdown not. Knockdown of <br /> eight genes studied here, knockdown ThrRS, ArgRS, AsnRS,                      ArgRS, PheRS&#226;?&#164;, ThrRS, complex, <br /> and PheRS&#226;?&#164; led rapid cell death phenotypes                    caused severe growth effects, observed TrpRS-1 and <br /> morphological cell cycle changes. The different responses                 aaRSs complex (21). Finally, reported that <br /> RNAi knockdown differences levels                    aaRSs alternative roles cell function, including the <br /> remaining mRNAs proteins reflect differ-                  production dinucleotide polyphosphates, act sec- <br /> ing cellular requirements enzyme, including relative cat-               ond messengers (59), amino acid sensing (60), transcriptional <br /> alytic efficiency levels relationship substrate concentra-            regulation (61). Thus, involvement enzyme cellular <br /> tions enzyme Km. In addition, cellular dependence                  process translation lead enhanced response <br /> each enzyme affected functions cytosolic                 gene knockdown respect growth cell cycle changes <br /> and/or mitochondrial compartments, integrated                      comparison genes involved translation. Over- <br /> within MARS complex, enzyme                       all, likely factors, including degree of <br /> any alternative functions cellular processes addition pro-              mRNA loss, cellular requirement certain aaRS, <br /> tein synthesis. While degree mRNA reduction ex-                   cellular location, potential functions the <br /> tent growth morphological abnormalities entirely                   aaRS, contribute phenotype considered when <br /> correlate, gene knockdowns showed severe                   choosing particular aaRS drug target based essentiality <br /> phenotypes (ThrRS, ArgRS, PheRS&#226;?&#164;, AsnRS), mRNA lev-                      data. <br /> els 24 h reduced 60% 84%,                    Finally, essentiality consideration evaluating <br /> moderate effects (PheRS&#226;?&#163;, TrpRS-1, ValRS), mRNA levels                       enzyme good drug target. Both the <br /> were reduced 57% 63% (Table 2). However, data                  druggability target likelihood achieving selectiv- <br /> all cases partial knockdown target required             ity versus human host need considered. Of genes <br /> cause growth reductions, suggesting partial reduction               targeted current study, (ThrRS, ArgRS, PheRS, and <br /> enzyme activity inhibitors affect parasite growth. This               ValRS) high (&#226;&#172;&#381;0.6) druggability scores (Table 1), among <br /> strengthens potential aaRSs successfully targeted             these, severe growth morphological changes observed <br /> for drug discovery.                                                              knockdown ThrRS ArgRS, suggesting latter <br />     In assessing potential effects MARS complex es-                enzymes strong potential successfully targeted. <br /> sentiality, knockdown enzyme MARS complex                    Furthermore, chemical validation strengthens case a <br /> potentially lead destabilization complex structure              particular target, data showing borrelidin inhibited <br /> changes abundance additional complex aaRSs. However,                   T. brucei ThrRS parasite growth provide preliminary chemical <br /> our analysis did indication knockdown en-                   validation target, stage, direct proof the <br />  <br />  <br />  <br />  <br /> FIG 10 Subcellular localization analysis T. brucei BSF expressing C-terminal V5-tagged ThrRS. Cells fixed 4% paraformaldehyde stained with <br /> FITC-conjugated anti-V5 monoclonal antibodies. Mitochondria stained MitoTracker (MT), DNA stained DAPI. Differential interfer- <br /> ence contrast (DIC) shown. Images merged indicated. <br />  <br />  <br />  <br /> April 2014 Volume 13 Number 4                                                                                                                   ec.asm.org 513 <br />  Kalidas et al. <br />  <br />  <br />  <br /> borrelidin growth effects consequence ThrRS inhibition        Science Education. I.C. awarded National Institutes Health <br /> lacking. Since ThrRS localize cytoplasm           postdoctoral fellowship (5T32AI007509-12). <br /> mitochondrion, possible inhibition cell growth be <br /> a consequence protein synthesis inhibition cellular            REFERENCES <br /> compartments.                                                              1. Kennedy PG. 2013. Clinical features, diagnosis, treatment human <br />     With regard selectivity, comparison T. brucei ThrRS             African trypanosomiasis (sleeping sickness). Lancet Neurol. 12:186 &#226;&#8364;&#8220;194. <br /> amino acid sequence human enzyme sequence shows                 http://dx.doi.org/10.1016/S1474-4422(12)70296-X. <br /> the enzymes highly conserved (see Fig. S1 supple-           2. Stuart K, Brun R, Croft S, Fairlamb A, Gurtler RE, McKerrow J, Reed <br />                                                                               S, Tarleton R. 2008. Kinetoplastids: related protozoan pathogens, differ- <br /> mental material). This case E. coli enzyme,           ent diseases. J. Clin. Invest. 118:1301&#226;&#8364;&#8220;1310. http://dx.doi.org/10.1172 <br /> yet recent study showed differences amino acid sequence             /JCI33945. <br /> in second-shell residues exploited identify bisubstrate        3. Jacobs RT, Nare B, Phillips MA. 2011. State art African trypano- <br /> analogs Thr adenine highly (&#226;&#172;&#381;300-fold) selective             drug discovery. Curr. Top. Med. Chem. 11:1255&#226;&#8364;&#8220;1274. http://dx.doi <br /> for bacterial enzyme (62). X-ray structure analysis showed           .org/10.2174/156802611795429167. <br />                                                                            4. Yun O, Priotto G, Tong J, Flevaud L, Chappuis F. 2010. NECT next: <br /> the replacement Leu361 Cys361 bacterial enzyme                  implementing new drug combination therapy Trypanosoma bru- <br /> created enlarged binding pocket allowed inhibitors                 cei gambiense sleeping sickness. PLoS Negl. Trop. Dis. 4:e720. http://dx <br /> adopt conformation bound E. coli enzyme                .doi.org/10.1371/journal.pntd.0000720. <br /> different seen bound human                 5. Frearson JA, Wyatt PG, Gilbert IH, Fairlamb AH. 2007. Target assess- <br /> enzyme. Examination putative Thr/ATP binding sites T.               ment antiparasitic drug discovery. Trends Parasitol. 23:589 &#226;&#8364;&#8220;595. http: <br />                                                                               //dx.doi.org/10.1016/j.pt.2007.08.019. <br /> brucei ThrRS shows T. brucei enzyme, like hu-           6. Brun R, Don R, Jacobs RT, Wang MZ, Barrett MP. 2011. Development <br /> man enzyme, Leu residue position 361 (TbThrRS Leu                    novel drugs human African trypanosomiasis. Future Microbiol. <br /> 409), amino acid differences second-shell residues               6:677&#226;&#8364;&#8220; 691. http://dx.doi.org/10.2217/fmb.11.44. <br /> present exploited identification selective        7. Ag&#195;&#188;ero F, Al-Lazikani B, Aslett M, Berriman M, Buckner FS, Campbell <br /> inhibitors (see Fig. S1). The borrelidin binding site, adja-         RK, Carmona S, Carruthers IM, Chan AW, Chen F, Crowther GJ, <br />                                                                               Doyle MA, Hertz-Fowler C, Hopkins AL, McAllister G, Nwaka S, <br /> cent substrate binding site, explored bacterial aaRSs           Overington JP, Pain A, Paolini GV, Pieper U, Ralph SA, Riechers A, <br /> by site-directed mutagenesis, identifying key residues           Roos DS, Sali A, Shanmugam D, Suzuki T, Van Voorhis WC, Verlinde <br /> conserved enzymes inhibited compound (53).                 CL. 2008. Genomic-scale prioritization drug targets: TDR Targets <br /> These residues conserved T. brucei human                  database. Nat. Rev. Drug Discov. 7:900 &#226;&#8364;&#8220;907. http://dx.doi.org/10.1038 <br /> enzymes, suggesting hard selectively target            /nrd2684. <br />                                                                            8. Magari&#195;&#177;os MP, Carmona SJ, Crowther GJ, Ralph SA, Roos DS, Shan- <br /> site. However, structural data available fully map             mugam D, Van Voorhis WC, Ag&#195;&#188;ero F. 2012. TDR Targets: chemog- <br /> borrelidin binding site, making difficult determine species-         enomics resource neglected diseases. Nucleic Acids Res. 40:D1118 &#226;&#8364;&#8220; <br /> selective binding possible site.                        D1127. http://dx.doi.org/10.1093/nar/gkr1053. <br />     In summary, demonstrate seven aaRSs,               9. Hopkins AL, Groom CR. 2002. The druggable genome. Nat. Rev. Drug <br />                                                                               Discov. 1:727&#226;&#8364;&#8220;730. http://dx.doi.org/10.1038/nrd892. <br /> which subunits, essential infectious mammalian <br />                                                                           10. Baquero F, Blazquez J. 1997. Evolution antibiotic resistance. Trends Ecol. Evol. <br /> form T. brucei previously showed AlaRs, ProRS,                 12:482&#226;&#8364;&#8220;487. http://dx.doi.org/10.1016/S0169-5347(97)01223-8. <br /> LeuRS, LysRS, IleRS essential mammalian in-           11. Gadakh B, Van Aerschot A. 2012. Aminoacyl-tRNA synthetase inhibitors <br /> fective stage parasite (21, 30). In addition genetic            antimicrobial agents: patent review 2006 till present. Expert <br /> validation 13 aaRS genes, shown additional            Opin. Ther. Pat. 22:1453&#226;&#8364;&#8220;1465. http://dx.doi.org/10.1517/13543776.2012 <br />                                                                               .732571. <br /> 5 essential relevant mammalian stage life cycle.     12. Vondenhoff GH, Van Aerschot A. 2011. Aminoacyl-tRNA synthetase <br /> That mRNA knockdown 60% 70% sufficient cell                inhibitors potential antibiotics. Eur. J. Med. Chem. 46:5227&#226;&#8364;&#8220;5236. http: <br /> growth arrest and, cases, cell death suggests aaRS         //dx.doi.org/10.1016/j.ejmech.2011.08.049. <br /> inhibitor effective enzyme activity re-             13. Niemann M, Schneider A, Cristodero M. 2011. Mitochondrial transla- <br /> mained treated parasites. In addition, enzymes an-             tion trypanosomatids: novel target chemotherapy? Trends Parasi- <br />                                                                               tol. 27:429 &#226;&#8364;&#8220; 433. http://dx.doi.org/10.1016/j.pt.2011.03.011. <br /> alyzed study (ValRS, PheRS, ArgRS, ThrRS)             14. Wilson DN. 2009. The A-Z bacterial translation inhibitors. Crit. Rev. Biochem. <br /> others genetic validation data high (i.e., &#226;&#172;&#381;0.6) drug-              Mol. Biol. 44:393&#226;&#8364;&#8220;433. http://dx.doi.org/10.3109/10409230903307311. <br /> gability scores (Table 1) essential likely bind       15. Perona JJ, Hadd A. 2012. Structural diversity protein engineering of <br /> small drug-like molecules good potency, making excel-               aminoacyl-tRNA synthetases. Biochemistry 51:8705&#226;&#8364;&#8220; 8729. http://dx <br /> lent candidates future drug discovery efforts. Finally,           .doi.org/10.1021/bi301180x. <br />                                                                           16. Delarue M. 1995. Partition aminoacyl-tRNA synthetases differ- <br /> provide preliminary chemical validation ThrRS showing                   ent structural classes dating early metabolism: implications the <br /> that borrelidin inhibitor T. brucei ThrRS T.             origin genetic <span id='am-238' about='Thesaurus:code' typeof='owl:Thing'>code</span> nature protein sequences. J. Mol. Evol. <br /> brucei cell growth. Other aaRSs T. brucei chem-             41:703&#226;&#8364;&#8220;711. <br /> ically validated (Table 1); thus, development recombinant          17. Nagel GM, Doolittle RF. 1991. Evolution relatedness amino- <br /> expression systems enzyme assays ThrRS                  acyl-tRNA synthetase families. Proc. Natl. Acad. Sci. U. S. A. 88:8121&#226;&#8364;&#8220; <br />                                                                               8125. http://dx.doi.org/10.1073/pnas.88.18.8121. <br /> aaRSs readies future drug discovery efforts, including           18. O&#226;&#8364;&#8482;Donoghue P, Luthey-Schulten Z. 2003. On evolution structure <br /> high-throughput screening novel inhibitors.                               aminoacyl-tRNA synthetases. Microbiol. Mol. Biol. Rev. 67:550 &#226;&#8364;&#8220;573. <br />                                                                               http://dx.doi.org/10.1128/MMBR.67.4.550-573.2003. <br /> ACKNOWLEDGMENTS                                                           19. Hoepfner D, McNamara CW, Lim CS, Studer C, Riedl R, Aust T, <br />                                                                               McCormack SL, Plouffe DM, Meister S, Schuierer S, Plikat U, Hart- <br /> This work supported National Institutes Health grants (R01          mann N, Staedtler F, Cotesta S, Schmitt EK, Petersen F, Supek F, <br /> AI078962) (to K.S. M.A.P.) Welch Foundation grant I-1257 (to          Glynne RJ, Tallarico JA, Porter JA, Fishman MC, Bodenreider C, <br /> M.A.P.). M.A.P. holds Beatrice Miguel Elias Distinguished Chair       Diagana TT, Movva NR, Winzeler EA. 2012. Selective specific inhi- <br /> in Biomedical Science Carolyn R. Bacon Professorship Medical       bition Plasmodium falciparum lysyl-tRNA synthetase fungal <br />  <br />  <br />  <br /> 514   ec.asm.org                                                                                                                                 Eukaryotic Cell <br />                                                                                                                                     T. brucei Aminoacyl tRNA Synthetases <br />  <br />  <br />  <br />       secondary metabolite cladosporin. Cell Host Microbe 11:654 &#226;&#8364;&#8220; 663. http:              prokaryotic homologs. J. Mol. Biol. 397:481&#226;&#8364;&#8220; 494. http://dx.doi.org/10 <br />       //dx.doi.org/10.1016/j.chom.2012.04.015.                                             .1016/j.jmb.2010.01.051. <br /> 20.   Gowri VS, Ghosh I, Sharma A, Madhubala R. 2012. Unusual domain architec-       37.   Merritt EA, Arakaki TL, Gillespie R, Napuli AJ, Kim JE, Buckner FS, <br />       ture aminoacyl tRNA synthetases paralogs Leishmania major.         Van Voorhis WC, Verlinde CL, Fan E, Zucker F, Hol WG. 2011. Crystal <br />       BMC Genomics 13:621. http://dx.doi.org/10.1186/1471-2164-13-621.                     structures protozoan homologs tryptophanyl-tRNA synthe- <br /> 21.   Cestari I, Kalidas S, Monnerat S, Anupama A, Phillips MA, Stuart K.                  tase. Mol. Biochem. Parasitol. 177:20 &#226;&#8364;&#8220;28. http://dx.doi.org/10.1016/j <br />       2013. A multiple aminoacyl-tRNA synthetase complex enhances                     .molbiopara.2011.01.003. <br />       tRNA-aminoacylation African trypanosomes. Mol. Cell. Biol. 33:4872&#226;&#8364;&#8220;         38.   Willert EK, Phillips MA. 2008. Regulated expression essential allo- <br />       4888. http://dx.doi.org/10.1128/MCB.00711-13.                                        steric activator polyamine biosynthesis African trypanosomes. PLoS <br /> 22.   Alfonzo JD, Soll D. 2009. Mitochondrial tRNA import&#226;&#8364;&#8221;the challenge                 Pathog. 4:e1000183. http://dx.doi.org/10.1371/journal.ppat.1000183. <br />       understand just begun. Biol. Chem. 390:717&#226;&#8364;&#8220;722. http://dx.doi.org/10       39.   Kalidas S, Li Q, Phillips MA. 2011. A Gateway((R)) compatible vector gene <br />       .1515/BC.2009.101.                                                                   silencing bloodstream form Trypanosoma brucei. Mol. Biochem. Parasitol. <br /> 23.   Tschopp F, Charriere F, Schneider A. 2011. In vivo study Trypano-                 178:51&#226;&#8364;&#8220;55. http://dx.doi.org/10.1016/j.molbiopara.2011.03.002. <br />       soma brucei links mitochondrial transfer RNA import mitochondrial           40.   Brun R, Schonenberger M. 1979. Cultivation vitro cloning or <br />       protein import. EMBO Rep. 12:825&#226;&#8364;&#8220; 832. http://dx.doi.org/10.1038                     procyclic culture forms Trypanosoma brucei semi-defined me- <br />       /embor.2011.111.                                                                     dium. Acta Trop. 36:289 &#226;&#8364;&#8220;292. <br /> 24.   Espa&#195;&#177;ol Y, Thut D, Schneider A, Ribas Pouplana L. 2009. A mech-             41.   Redmond S, Vadivelu J, Field MC. 2003. RNAit: automated web-based tool <br />       anism functional segregation mitochondrial cytosolic genetic              fortheselectionofRNAitargetsinTrypanosomabrucei.Mol.Biochem.Parasitol. <br />       codes. Proc. Natl. Acad. Sci. U. S. A. 106:19420 &#226;&#8364;&#8220;19425. http://dx.doi.org           128:115&#226;&#8364;&#8220;118. http://dx.doi.org/10.1016/S0166-6851(03)00045-8. <br />       /10.1073/pnas.0909937106.                                                      42.   Inoue M, Nakamura Y, Yasuda K, Yasaka N, Hara T, Schnaufer A, <br /> 25.   Charri&#195;&#168;re F, O&#226;&#8364;&#8482;Donoghue P, Helgad&#195;&#179;ttir S, Mar&#195;&#169;chal-Drouard L, Cris-                  Stuart K, Fukuma T. 2005. The 14-3-3 proteins Trypanosoma brucei <br />       todero M, Horn EK, S&#195;&#182;ll D, Schneider A 2009. Dual targeting                     function motility, cytokinesis, cell cycle. J. Biol. Chem. 280:14085&#226;&#8364;&#8220; <br />       tRNAAsp requires different aspartyl-tRNA synthetases Trypano-                 14096. http://dx.doi.org/10.1074/jbc.M412336200. <br />       soma brucei. J. Biol. Chem. 284:16210 &#226;&#8364;&#8220;16217. http://dx.doi.org/10.1074        43.   Wirtz E, Leal S, Ochatt C, Cross G. 1999. A tightly regulated inducible <br />       /jbc.M109.005348.                                                                    expression conditional gene knock-outs dominant- <br /> 26.   Charri&#195;&#168;re F, Helgad&#195;&#179;ttir S, Horn EK, S&#195;&#182;ll D, Schneider A. 2006. Dual                 negative genetics T. brucei. Mol. Biochem. Parasitol. 99:89 &#226;&#8364;&#8220;101. http: <br />       targeting single tRNA(Trp) requires different tryptophanyl-tRNA             //dx.doi.org/10.1016/S0166-6851(99)00002-X. <br />       synthetases Trypanosoma brucei. Proc. Natl. Acad. Sci. U. S. A. 103:        44.   Schnaufer A, Panigrahi AK, Panicucci B, Igo RP, Jr, Wirtz E, Salavati R, <br />       6847&#226;&#8364;&#8220; 6852. http://dx.doi.org/10.1073/pnas.0602362103.                               Stuart K. 2001. An RNA ligase essential RNA editing survival of <br /> 27.   Cristodero M, Seebeck T, Schneider A. 2010. Mitochondrial translation                bloodstream form Trypanosoma brucei. Science 291:2159 &#226;&#8364;&#8220;2162. <br />       essential bloodstream forms Trypanosoma brucei. Mol. Microbiol.             http://dx.doi.org/10.1126/science.1058655. <br />       78:757&#226;&#8364;&#8220;769. http://dx.doi.org/10.1111/j.1365-2958.2010.07368.x.                45.   Brennd&#195;&#182;rfer M, Boshart M. 2010. Selection reference genes mRNA <br /> 28.   Rinehart J, Horn EK, Wei D, Soll D, Schneider A. 2004. Non-canonical <br />                                                                                            quantification Trypanosoma brucei. Mol. Biochem. Parasitol. 172:52&#226;&#8364;&#8220; <br />       eukaryotic glutaminyl- glutamyl-tRNA synthetases form mitochon- <br />                                                                                            55. http://dx.doi.org/10.1016/j.molbiopara.2010.03.007. <br />       drial aminoacyl-tRNA Trypanosoma brucei. J. Biol. Chem. 279:1161&#226;&#8364;&#8220; <br />                                                                                      46.   Winer J, Jung CK, Shackel I, Williams PM. 1999. Development validation <br />       1166. http://dx.doi.org/10.1074/jbc.M310100200. <br />                                                                                            real-time quantitative reverse transcriptase-polymerase chain reaction for <br /> 29.   Rettig J, Wang Y, Schneider A, Ochsenreiter T. 2012. Dual targeting of <br />                                                                                            monitoring gene expression cardiac myocytes vitro. Anal. Biochem. 270:41&#226;&#8364;&#8220; <br />       isoleucyl-tRNA synthetase Trypanosoma brucei mediated through <br />                                                                                            49. http://dx.doi.org/10.1006/abio.1999.4085. <br />       alternative trans-splicing. Nucleic Acids Res. 40:1299 &#226;&#8364;&#8220;1306. http://dx.doi <br />                                                                                      47.   Tu X, Wang CC. 2004. The involvement cdc2-related kinases <br />       .org/10.1093/nar/gkr794. <br />                                                                                            (CRKs) Trypanosoma brucei cell cycle regulation distinctive <br /> 30.   Cestari I, Stuart K. 2013. Inhibition isoleucyl-tRNA synthetase a <br />                                                                                            stage-specific phenotypes caused CRK3 depletion. J. Biol. Chem. 279: <br />       potential treatment human African trypanosomiasis. J. Biol. Chem. <br />       288:14256 &#226;&#8364;&#8220;14263. http://dx.doi.org/10.1074/jbc.M112.447441.                         20519 &#226;&#8364;&#8220;20528. http://dx.doi.org/10.1074/jbc.M312862200. <br /> 31.   Zhao Y, Wang Q, Meng Q, Ding D, Yang H, Gao G, Li D, Zhu W, Zhou               48.   Tu X, Wang CC. 2005. Coupling posterior cytoskeletal morphogenesis <br />       H. 2012. Identification Trypanosoma brucei leucyl-tRNA synthetase                 G1/S transition Trypanosoma brucei cell cycle. Mol. Biol. Cell <br />       inhibitors pharmacophore- docking-based virtual screening                 16:97&#226;&#8364;&#8220;105. http://dx.doi.org/10.1091/mbc.E04-05-0368. <br />       synthesis. Bioorg. Med. Chem. 20:1240 &#226;&#8364;&#8220;1250. http://dx.doi.org/10.1016/j       49.   Carnes J, Schnaufer A, McDermott SM, Domingo G, Proff R, Steinberg <br />       .bmc.2011.12.035.                                                                    AG, Kurtz I, Stuart K. 2012. Mutational analysis Trypanosoma brucei <br /> 32.   Ding D, Meng Q, Gao G, Zhao Y, Wang Q, Nare B, Jacobs R, Rock F,                     editosome proteins KREPB4 KREPB5 reveals domains critical for <br />       Alley MR, Plattner JJ, Chen G, Li D, Zhou H. 2011. Design, synthesis,                function. RNA 18:1897&#226;&#8364;&#8220;1909. http://dx.doi.org/10.1261/rna.035048.112. <br />       structure-activity relationship Trypanosoma brucei leucyl-tRNA          50.   Cestari I, Stuart K. 2013. A spectrophotometric assay quantitative <br />       synthetase inhibitors antitrypanosomal agents. J. Med. Chem. 54:1276 &#226;&#8364;&#8220;            measurement aminoacyl-tRNA synthetase activity. J. Biomol. Screen. <br />       1287. http://dx.doi.org/10.1021/jm101225g.                                           18:490 &#226;&#8364;&#8220; 497. http://dx.doi.org/10.1177/1087057112465980. <br /> 33.   Shibata S, Gillespie JR, Kelley AM, Napuli AJ, Zhang Z, Kovzun KV,             51.   Panigrahi AK, Ogata Y, Zikova A, Anupama A, Dalley RA, Acestor N, <br />       Pefley RM, Lam J, Zucker FH, Van Voorhis WC, Merritt EA, Hol WG,                     Myler PJ, Stuart KD. 2009. A comprehensive analysis Trypanosoma <br />       Verlinde CL, Fan E, Buckner FS. 2011. Selective inhibitors methionyl-             brucei mitochondrial proteome. Proteomics 9:434 &#226;&#8364;&#8220; 450. http://dx.doi.org <br />       tRNA synthetase potent activity Trypanosoma brucei infec-               /10.1002/pmic.200800477. <br />       tion mice. Antimicrob. Agents Chemother. 55:1982&#226;&#8364;&#8220;1989. http://dx.doi        52.   Ling J, Peterson KM, Simonovic I, Cho C, Soll D, Simonovic M. 2012. <br />       .org/10.1128/AAC.01796-10.                                                           Yeast mitochondrial threonyl-tRNA synthetase recognizes tRNA isoac- <br /> 34.   Koh CY, Kim JE, Shibata S, Ranade RM, Yu M, Liu J, Gillespie JR,                     ceptors distinct mechanisms promotes CUN codon reassignment. <br />       Buckner FS, Verlinde CL, Fan E, Hol WG. 2012. Distinct states                     Proc. Natl. Acad. Sci. U. S. A. 109:3281&#226;&#8364;&#8220;3286. http://dx.doi.org/10.1073 <br />       methionyl-tRNA synthetase indicate inhibitor binding conformational               /pnas.1200109109. <br />       selection. Structure 20:1681&#226;&#8364;&#8220;1691. http://dx.doi.org/10.1016/j.str.2012        53.   Ruan B, Bovee ML, Sacher M, Stathopoulos C, Poralla K, Francklyn CS, <br />       .07.011.                                                                             Soll D. 2005. A unique hydrophobic cluster near active site contributes <br /> 35.   Shibata S, Gillespie JR, Ranade RM, Koh CY, Kim JE, Laydbak JU,                      differences borrelidin inhibition threonyl-tRNA synthetases. <br />       Zucker FH, Hol WG, Verlinde CL, Buckner FS, Fan E. 2012. Urea-based                  J. Biol. Chem. 280:571&#226;&#8364;&#8220;577. http://dx.doi.org/10.1074/jbc.M411039200. <br />       inhibitors Trypanosoma brucei methionyl-tRNA synthetase: selectivity        54.   Jackson KE, Pham JS, Kwek M, De Silva NS, Allen SM, Goodman CD, <br />       vivo characterization. J. Med. Chem. 55:6342&#226;&#8364;&#8220; 6351. http://dx.doi             McFadden GI, Pouplana LR, Ralph SA. 2012. Dual targeting of <br />       .org/10.1021/jm300303e.                                                              aminoacyl-tRNA synthetases apicoplast cytosol Plasmodium <br /> 36.   Merritt EA, Arakaki TL, Gillespie JR, Larson ET, Kelley A, Mueller N,                falciparum. Int. J. Parasitol. 42:177&#226;&#8364;&#8220;186. http://dx.doi.org/10.1016/j <br />       Napuli AJ, Kim J, Zhang L, Verlinde CL, Fan E, Zucker F, Buckner FS,                 .ijpara.2011.11.008. <br />       van Voorhis WC, Hol WG. 2010. Crystal structures trypanosomal               55.   Gao YM, Wang XJ, Zhang J, Li M, Liu CX, An J, Jiang L, Xiang WS. <br />       histidyl-tRNA synthetase illuminate differences eukaryotic               2012. Borrelidin, potent antifungal agent: insight antifungal <br />  <br />  <br /> April 2014 Volume 13 Number 4                                                                                                                            ec.asm.org 515 <br />  Kalidas et al. <br />  <br />  <br />  <br />       mechanism Phytophthora sojae. J. Agric. Food Chem. 60:9874 &#226;&#8364;&#8220;                        D, Bacher J, Groll AH, Walsh TJ. 2004. Efficacy PLD-118, novel <br />       9881. http://dx.doi.org/10.1021/jf302857x.                                                  inhibitor candida isoleucyl-tRNA synthetase, experimental <br /> 56.   Habibi D, Ogloff N, Jalili RB, Yost A, Weng AP, Ghahary A, Ong CJ.                          oropharyngeal esophageal candidiasis caused fluconazole-resistant <br />       2012. Borrelidin, small molecule nitrile-containing macrolide inhibitor                   C. albicans. Antimicrob. Agents Chemother. 48:3959 &#226;&#8364;&#8220;3967. http://dx.doi <br />       threonyl-tRNA synthetase, potent inducer apoptosis acute                      .org/10.1128/AAC.48.10.3959-3967.2004. <br />       lymphoblastic leukemia. Invest. New Drugs 30:1361&#226;&#8364;&#8220;1370. http://dx.doi                 68.   Houge-Frydrych CS, Gilpin ML, Skett PW, Tyler JW. 2000. SB-203207 <br />       .org/10.1007/s10637-011-97 <br /> </body></html>