<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Characterization Putative Spindle Assembly <br /> Checkpoint Kinase Mps1, Suggests Its Involvement in <br /> Cell Division, Morphogenesis Oxidative Stress <br /> Tolerance Candida albicans <br /> Mohan Kamthan1., Vijaya Kumar Nalla2., Deepa Ruhela1, Ayushi Kamthan1, Protiti Maiti1, Asis Datta1* <br /> 1 National Institute Plant Genome Research, New Delhi, India, 2 Department Microbiology, Bhaskaracharya college, University Delhi, New Delhi, India <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      In Saccharomyces cerevisiae MPS1 major protein kinase governs spindle checkpoint pathway. The S. <br />      cerevisiae structural homolog opportunistic pathogen Candida albicans CaMPS1, indispensable cell viability. The <br />      essentiality Mps1 confirmed Homozygote Trisome test. <span id='am-8' about='protege:TO' typeof='owl:Thing'>To</span> determine biological function pathogen <br />      conditional mutant generated regulatable MET3 promoter. Examination heterozygous conditional (+ <br />      Met/Cys) mps1 mutants revealed mitosis specific arrest <span id='am-87' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-88' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span>phenotype</span>, mutants showed large buds undivided <br />      nuclei. Flowcytometry analysis revealed abnormal ploidy levels mps1mutant. In presence anti-microtubule drug <br />      Nocodazole, mps1 mutant showed dramatic loss viability suggesting role Mps1 Spindle Assembly Checkpoint <br />      (SAC) activation. These mutants defective microtubule organization. Moreover, heterozygous mutant showed <br />      defective in-vitro yeast hyphae morphological transition. Growth defect heterozygous mutant suggest <br />      haploinsufficiency gene. qRT PCR analysis showed 3 fold upregulation MPS1 presence serum. This <br />      expression MPS1 dependent Efg1and independent hyphal regulators like Ras1 Tpk2. Furthermore, <br />      mps1 mutants sensitive oxidative stress. Heterozygous mps1 mutant did undergo morphological transition <br />      showed 5-Fold reduction colony forming units response macrophage. Thus, vital checkpoint kinase, Mps1 <br />      cell division role morphogenesis oxidative stress tolerance, pathogenic fungus. <br />  <br />   Citation: Kamthan M, Nalla VK, Ruhela D, Kamthan A, Maiti P, et al. (2014) Characterization Putative Spindle Assembly Checkpoint Kinase Mps1, Suggests Its <br />   Involvement Cell Division, Morphogenesis Oxidative Stress Tolerance Candida albicans. PLoS ONE 9(7): e101517. doi:10.1371/journal.pone.0101517 <br />   Editor: Yanchang Wang, Florida State University, United States America <br />   Received February 5, 2014; Accepted June 9, 2014; Published July 15, 2014 <br />   Copyright: &#195;&#376; 2014 Kamthan et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported Department Biotechnology, Ministry Science Technology, Government India. The funders role <br />   study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * Email: asis_datta@rediffmail.com <br />   . These authors contributed equally work. <br />  <br />  <br />  <br /> Introduction                                                                          centrosomes mammals) coordinated fashion response to <br />                                                                                       various cell division cues. In S. cerevisiae, seven genes been <br />    Candida albicans, close phylogenetic relative Saccharomyces                   identified function governing spindle checkpoint <br /> cerevisiae, major fungal pathogen humans systemic                       pathway; MAD1-3 [4], BUB1-3 [5], MPS1 [6]. Further <br /> infections fatal immunocompromised patients [1].                         studies Bub2 revealed prevents unusual multinucle- <br /> Pathogens like C. albicans evolved various counteractive                         ate cells misoriented spindles exiting mitosis. Thus, is <br /> cellular mechanisms evade host defenses. Despite,                            checkpoint, components that <br /> strategies damage key cellular components like DNA                         regulate exit mitosis [41]. Cells <span id='am-165' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-166' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span>mutation</span> of <br /> protein pathogen occurs repaired <span id='am-9' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-10' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span>survival</span>.                   genes fail arrest cell cycle response spindle defects <br /> Cell cycle checkpoints coordinate DNA integrity proper                        undergo aberrant, lethal mitosis. A study MAD2 showed <br /> chromosomal segregation cell division essential                   indispensable role gene morphogenesis viability of <br /> cell viability. Precise control chromosomal segregation carried                 C. albicans murine mouse model [2]. A non-essential gene, BUB2 <br /> out transient cytoskeletal structure termed mitotic                       controls pre-anaphase arrest polarization pseudohy- <br /> spindle. The checkpoint senses defects attachment chromo-                       phal-like cells [3]. The budding yeast homolog CENP-A, <br /> somes mitotic spindle preventing chromosomal loss                       CaCSE4, shown play role proper chromosome <br /> stopping chromosome segregation. The components checkpoint                         segregation growth C. albicans [7]. <br /> pathways drug targets, especially cancer research.                       One master protein kinase check point pathway is, <br /> In C. albicans, Spindle Assembly Checkpoint pathway plays                          MPS1 (Mono Polar Spindle Kinase) [6] (mutants gene form <br /> important role cell division [2,3]. During nuclear division,                       monopolar spindles), kinase activity required for <br /> mitotic spindle forms bipolar structure nucleation                      activating members checkpoint machinery. Homologs of <br /> spindle microtubules spindle pole body (equivalent                          MPS1 gene identified organisms play <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                        1                                 July 2014 | Volume 9 | Issue 7 | e101517 <br />                                                                                                                 Role MPS1 Candida albicans <br />  <br />  <br /> diverse roles checkpoint activation, spindle pole body duplica-         accession number obtained (AY632356.1). First allele <br /> tion, chromosome segregation, mitotic arrest, response           disruption MPS1 performed using URA blaster method <br /> hypoxic conditions. In budding yeast, MPS1 essential dual            [23] CAF3-1 strain (Methods S1). One Ura+ heterozygous <br /> specificity (serine/threonine tyrosine) protein kinase involved        mutant (MFD2) used Ura-curing process, SD+Uri agar <br /> in normal progression cell cycle [6,8]. In humans higher            plates supplemented 59-fluoroorotic acid (5-FOA) (Sigma <br /> expression MPS1 detected human                     Aldrich) 1 mg/ml concentration. Curing confirmed by <br /> neoplasms, including thyroid, breast lung cancers [9&#226;&#8364;&#8220;12].              southern blot analysis ura2 starin (MFD2-U1) was <br /> MPS1 considered promising drug target cancer              selected. A single transformation based gene function test [17] was <br /> cells. Due central role protein kinase cell division,        used confirm essentiality MPS1 (Methods S1). The <br /> several inhibitors MPS1 reported [13&#226;&#8364;&#8220;15]. In C.               heterozygous MFD2-U1 strain used construc- <br /> albicans, orf19.7293 MPS1 homolog S. cerevisiae. Although        tion MET3 regulated conditional mutant, MCM4. To create <br /> the inhibitor CaMps1 known [16] biological function       MET3-MPS1 cassette, 590 bp fragment MPS1 coding region, <br /> still unknown. In report characterized structural           comprising positions +1 +590 relation ATG start site, <br /> homologue budding yeast MPS1 gene proved                        PCR amplified using oligonucleotides, MpBamHI:59TT- <br /> indispensable role survival, cell division, morphogenesis           GACGGATCCATGCCACATGA TTTATTATC39 reverse <br /> oxidative stress tolerance human pathogenic yeast, C. albicans.         primer, MPSphI: 59GCATGCATCTTTCTAAC GACG39, having <br />                                                                            BamHI SphI sites (italicized bases) directional cloning. This <br /> Materials Methods                                                      fragment cloned pGEMT-easy vector resulted in <br />                                                                            plasmid pCaMPF (Table 1). This 590 bp fragment digested <br /> Mice                                                                       ligated BamHI SphI digested plasmid, pCaDis [24] <br />    Female Balb/c mouse weighing 18&#226;&#8364;&#8220;20 grams obtained               resulting 5,930 bp plasmid, pDMPS1 (Figure S2). This plasmid <br /> pathogen-free mice Animal House Jawaharlal Nehru               linearized BsgI digestion transformed Ura2/2 <br /> University (JNU) New Delhi, INDIA. The use Mice duly                MFD2-U1 strain lithium acetate method [25], isolate MET3 <br /> approved Institutional Animal Ethic committee (IAEC)             regulated strain; MCM4, SC agar plates lacking Uridine, <br /> Jawaharlal Nehru University (JNU), Registration No. 19/1999                Methionine Cysteine. Correct recombinants containing, <br /> (CPCSEA; Committee purpose control supervision              MET3 promoter confirmed southern blotting digesting <br /> of experiments animals). Approval <span id='am-5' about='Thesaurus:code' typeof='owl:Thing'>code</span> VO/AH/IAEC/                  genomic DNA strains XbaI DraII (Figure S2). <br /> 84/53. All housing experimental procedures conducted              Similarly, rescued strain MCM4R; MPS1 coding region <br /> under guidelines JNU Animal Care.                               expressed promoter, created reintroducing <br />                                                                            gene non-essential RPS1 locus [26] using pCaExP vector <br /> Media Strains                                                          (Generously gifted Peter E. Sudbery) [24], contains a <br />    C. albicans strains used study listed Table 1. C.        URA3 marker. For construction revertant cassette, MPS1 ORF <br /> albicans routinely grown 30uC YPD (1% yeast extract, 2%          500 bp promoter region (2573 bp) PCR amplified <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> Bacto Peptone, 2% glucose) Sabourad&#226;&#8364;&#8482;s Dextrose (SD) media               genomic DNA SC5314 using oligonucleotides MPSalF: <br /> (0.67% Yeast Nitrogen <span id='am-321' about='xsp:base' typeof='owl:Thing'>Base</span>, 2% Dextrose 2% agar (for solid             59GTCGACTTAAATTGGTTAAAATTTTC39 MPSalR: <br /> media)). Synthetic Complete (SC) medium used condi-                59GTCGACATACACTAGCTGTAGTTT AAG39 di- <br /> tional mutant (0.67% nitrogen base amino acids, 2%                 gested SalI enzyme. Next, pCaExP vector digested with <br /> glucose mixture amino acids) required 80 mg              SalI release 360 bp MET3 promoter fragment. The resultant <br /> ml21 Uridine supplemented (Uri+). Transformants HT             5,895 bp pCaEXP-MET3 vector backbone ligated with <br /> test grown synthetic defined media described Enloe           2,573 bp SalI digested MPS1 fragment creating 8,468 bp <br /> et al [17]. For conditional mutant studies, 2.5 mM L-Cystiene              plasmid, pCaExMP. After linearizing StuI digestion, this <br /> (Cys) L-Methionine (Met) externally added. For                    construct used transforming MCM4 strain. Transformants <br /> morphogenetic studies, Spider (adjusted pH 7.3) SLAD                screened SC agar plates supplemented 2.5 mM <br /> media prepared described Lin et al [18] Gimeno et           Met/Cys, inhibited growth untransformed MCM4 <br /> al [19] respectively. Cells induced GlcNAc Biswas         cells. Correct integration pCaExMP MCM4R confirmed <br /> et al [20]. For serum induction 10% bovine serum (Sigma)               Southern blot analysis SalI DraII digestion. Since <br /> added pre-warmed SD 37uC. For G1 cell synchronization,               ectopic expression URA3 cause phenotypic effects, for <br /> C.albicans cells cultured Carbon source deficient Emden-           morphological studies targeted reintegration URA3 was <br /> Meyeroff-Mininmal Media (EMM media) 25uC overnight                      performed heterozygous MFD2-U1 strain using pCaEXP <br /> [21,22]. Chlamydospore formation obtained growing               vector neutral RPS1 locus. <br /> cells coverslips 25uC 7 days cornmeal/Tween agar <br /> (BD) plates.                                                               Southern blotting <br />                                                                               For Southern blotting, genomic DNA extracted cells <br /> Cloning, Confirmation Essentiality generation                    grown YPD SD media. micrograms DNA was <br /> conditional mutants                                                        digested Restriction endonuclease resolved Agarose <br />    The entire 2073 base pairs (bp) gene amplified             gel electrophoresis 1% agarose gels, transferring [27] <br /> genomic DNA C.albicans (SC5314) 162 bp upstream              positively charged Nylon membranes (NEN Research <br /> and 250 bp downstream start stop codon respectively, using          Products Ltd.,) capillary transfer. For screening MFD2, <br /> oligonucleotides, MPS1F: 59CCTAGTGAGCACACACAT39and                         MFD2-U1,M-UAU M-AU transformants hybridizations were <br /> MPS1R: 59GTGTTGGCAGG GTGATGC39 cloned                           performed 32P radiolabelled DNA probes, prepared from <br /> pGEMT-easy vector (Promega) generate plasmid pGEM-                      NotI-digestion pGEM-MPS1 resulted ,2.5 Kb MPS1 <br /> MPS1. Following sequencing confirmation alignment              fragment. Whereas 0.597 Kbp fragment pDMPS1, was <br /> the Genome sequence information available Candida genome                digested BamHI &amp; SphI used screening MET3 <br /> database (CGD), <span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span> sequence deposited NCBI             transformants. <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             2                             July 2014 | Volume 9 | Issue 7 | e101517 <br />                                                                                                                           Role MPS1 Candida albicans <br />  <br />  <br />  <br />  Table 1. Strains plasmids used study. <br />  <br />  <br />  Strain/Plasmid             Genotype/description                                                                             Source/reference <br />  <br />  Strains <br />  SC5314 (wild-type)         URA3/URA3 <br />  CAF3-1                     Dura3::imm434/Dura3::imm434                                                                      W.A.Fonzi <br />  BWP17                      ura3D::limm434/ura3D::limm434his1::hisG/his1::hisG arg4::hisG/arg4::hisG                         Aaron P. Mitchell <br />  MFD2                       As CAF3-1, MPS1/Dmps1::hisG-URA3-hisG                                                        This Work <br />  MFD2-U1                    As CAF3-1, MPS1/Dmps1::hisG                                                                  This Work <br />  M-UAU 1&#226;&#8364;&#8220;30                 As BWP17, MPS1/DCamps1::Ura3-ARG4-Ura3                                                       This Work <br />  M-AU                       As BWP17, Dmps1::URA3/Dmps1::Ura3-ARG4-Ura3/MPS1                                             This Work <br />  MCM4,7                     As MFD2-U1, Dmps1/pMetMPS1                                                                   This Work <br />  MCM4R                      As MCM4, pMetMPS1/RP10::MPS1                                                                 This Work <br />  CAN52                      Dras1::hisG/Dras1::hph Dura3::ura3::imm434/Dura3::imm434                                         [42] <br />  HLC67                      Defg1::hisG/Defg1::hisG Dura3::imm434/Dura3::imm434                                              [43] <br />  AS1                        Dtpk2::hisG/Dtpk2::hisG Dura3::imm434/Dura3::imm434                                              [43] <br />  A-11-1-1-4 (Dcph1)         Dacpr1::hisG/Dacpr1::hisG Dura3::imm434/Dura3::imm434                                            Laboratory strain <br />  Plasmids <br />  pGEM-MPS1                  MPS1 ORF cloned pGEM-T Easy vector                                                            This work <br />  pMP1                       Carrying Dmps1::hisG-URA3-hisG disruption fragment                                               This work <br />  pUC19-CUB                  hisG-URA3-hisG disruption cassette cloned pUC19                                               [20] <br />  pBME101                    Carrying Ura39-ARG4-Ura39 cassette                                                               Aaron P. Mitcehll <br />  pCaDis                     Plasmid carrying MET3 promoter, URA3 marker                                                      P.E. Sudbery <br />  pCaEXP                     Plasmid carrying MET3 promoter URA3 marker RP10 gene                                         P.E. Sudbery <br />  pCaMPF                     0.5 Kb MPS1ORF BamHI SphI sites cloned pGEM-T                                        This work <br />  pDMPS1                     Same pMPF pDis vector                                                               This work <br />  pCaExMP                    pEXP 2.5 kb MPS1 ORF promoter                                                          This work <br />  pMUAU                      Carrying Dmps1::Ura39-ARG4-Ura39 disruption fragment                                             This work <br />  <br />  doi:10.1371/journal.pone.0101517.t001 <br />  <br />  <br />    qRT-PCR. Cells grown OD 0.8 SD media [28]                           Indirect Immunofluorescence <br /> and induced hrs synthetic media 2% carbon                               Cells grown up-to OD 0.6&#226;&#8364;&#8220;0.8 SC medium at <br /> source. Total RNA isolated using Tripure reagent (Roche).                        37uC conditional mutants induced 2.5 mM Met/ <br /> cDNA synthesized total RNA (5 mg) using oligo dT                         Cys 4 hrs. Cells fixed 4% formaldehyde 30 min <br /> primer (Invitrogen) Superscript I reverse transcriptase                          room temperature washed times PBS (150 mM <br /> (Invitrogen). The original mRNA template removed                            NaCl: 20 mM Na/K phosphate, pH 7.3). Cells resuspended <br /> the RNA: DNA heteroduplex RNase H. The cDNA samples                             PBS containing 1 M sorbitol (PBSS) 100 ml zymolyase <br /> were used templates qRT-PCR 7900 Fast Real                          incubated 37uC 1 hr. Cells washed twice with <br /> Time PCR (Applied Biosystems) using SDS 2.4 program.                          PBSS resuspended 100 ml PBS +1% BSA +0.1 M lysine <br /> The 10 ml reaction mix included 2X KAPA SYBR FAST (KAPA                              cell added slides allowed stand for <br /> BIOSYSTEMS), 0.5 ml cDNA 0.2 mM primers. The                                  30 min. Cells permeabilized chilled methanol <br /> amplification program included Stage I- 50uC 2 min, Stage 2-                     (9 min) acetone (6 min), air-dried. After rehydration in <br /> 95uC 10 min, Stage 3- 40 cycles 95uC 15 sec 60uC                      PBS, slides incubated anti-tubulin antibodies <br /> for 60 sec, Stage 4- 95uC 15 sec, 60uC 15 sec 95                     (Millipore, CBL270) diluted 1:200-1:400. Antibodies immu- <br /> 15 sec. Raw data analyzed using RQ manager 1.2.1 software                        nofluorescence diluted PBS, 1% BSA, antibody <br /> to determine relative gene expression. ACT1 mRNA level used                      incubations carried 1 h 37uC. The slides were <br /> as endogenous control. The results represent average                    washed 20 min changes PBS, second <br /> biological replicate experiments performed triplicates.                      antibody FITC conjugated goat anti Rat IgG used 1:200 <br /> Error bars, exist represent standard error                    dilution. Finally, slides washed 20 min four <br /> 3 separate biological replicates. The following primers designed                     changes PBS mounted viewed confocal microscope. <br /> with Primer 3 software used qPCR analysis. ACT1-F59 <br /> GACAATTTCTCTTTCAGCACTAGTAGTGA -39 ACT1-                                          Nocodazole Sensitivity Assay <br /> R59 GCTGGTAGAGA CTTGACCAACCA -39; MPS1-F59 CC-                                         Strains grown up-to OD 0.6&#226;&#8364;&#8220;0.8 liquid SD media <br /> AGGCAGCAAAACATT 39MPS1-R59 TTTGCCGAACCTTC-                                           induced Nocodazole (50 mM/ml) (Sigma aldrich). <br /> TTT 39.                                                                              Aliquots treated cultures collected 2, 4 hr intervals. <br />                                                                                      Conditional mutants induced Nocodazole in <br />                                                                                      presence absence 2.5 mM Met/Cys. For viability check, <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                       3                             July 2014 | Volume 9 | Issue 7 | e101517 <br />                                                                                                              Role MPS1 Candida albicans <br />  <br />  <br />  <br />  <br /> Figure 1. Multiple sequence alignment showing conserved regions Mps1p homologs. (A) Alignment N-terminal region Mps1, <br /> showing conserved Leucine residue. (B) Sequence alignment C-terminal homology CaMps1 (Orf19.7293/AY632356), Mps1 S. cerevisiae <br /> (YDL028C/P54199), Mph1 S. pombe (O94235), hMps1 Human (Q9BW51) mMps1 Mouse (P35761): Where ORF numbers obtained, <br /> accession numbers provided. Eleven subdomains shown Roman numerals (I&#226;&#8364;&#8220;XI). Subdomains kinase domain shown described <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                           4                            July 2014 | Volume 9 | Issue 7 | e101517 <br />                                                                                                                         Role MPS1 Candida albicans <br />  <br />  <br />  <br /> by [30]. The conserved amino acids serine/threonine kinases (SER/THR) shown described [31], conserved amino acids shown bold <br /> letters. The italicized E sub domain V SER/THR consensus, feature dual specificity kinases [31]. Valine (V) position <br /> 382 site insertional inactivation. Well conserved aa indicated double dot (:) perfectly conserved aa indicated an <br /> asterisk (*) <br /> doi:10.1371/journal.pone.0101517.g001 <br />  <br />  <br />  <br />  <br /> cultures serially diluted equal numbers spreaded               hours. Washed PBS stained 50 mg/ml Propidium <br /> SD Agar plates incubated 30uC 2&#226;&#8364;&#8220;3 days. For S-phase                  Iodide 4uC 8&#226;&#8364;&#8220;10 hrs. Stained cells diluted folds <br /> arrest, cultures treated 50 mM Hydroxy Urea (USB                      briefly sonicated disperse aggregates formed fixation <br /> biochemicals) 30uC hours addition                      process. Approximately 1&#226;&#8364;&#8220;26104 cells analyzed BD FACS <br /> nocodazole. Cells collected plated described earlier                Calibur machine gated settings linear scale. Forward <br /> [2].                                                                            Scatter analysis (FSC) performed similar way. Histograms <br />                                                                                 processed analyzed Morphid WinMDI softwares, <br /> Cell Synchronization Flowcytometry                                          respectively. <br />    C. albicans cells prepared FACS analysis Nitrogen <br /> starvation based synchronization described [21]. Cells                  Morphogenetic studies mouse peritoneal cavity and <br /> grown Edinburgh Minimal Medium (EMM) till OD 0.5.                      macrophages <br /> Cells inoculated EMM medium Ammonium                          Freund&#226;&#8364;&#8482;s complete adjuvant (0.4 ml) injected the <br /> Chloride grown saturation 23uC vigorous aeration.                peritoneal cavity female Balb/c mouse, weighing 18&#226;&#8364;&#8220;20 grams. <br /> At regular intervals, cells harvested inoculated SD              10 days injection, significant abdomen enlargement was <br /> SC medium supplemented nocodazole hydroxy urea.                         observed large exudates peritoneal cavity. <br /> Fixation 70% chilled ethanol incubated                  Further, 26108 late log phase cells strain 200 ml PBS <br /> hour 25uC. After fixation, cells harvested washed                   injected enlarged peritoneal cavity (three mice each <br /> twice 5X Tris-EDTA buffer (pH 8.0). Further, cells                    strain used). 24 hours injection, 100 ml the <br /> treated 1 mg/ml RNaseA (Invitrogen) 37uC                      peritoneal exudate retrieved using 23G hypodermic needle <br />  <br />  <br />  <br />  <br /> Figure 2. Construction MET3 promoter regulated conditional mps1 mutant. (A) Illustration representing conditional mutant preparation. <br /> Plasmid pDMPS1was digested BsgI linearized cassette used transform URA3-cured MFD2 (MPS1/mps1D:hisG) mutants. Genetic <br /> structure MCM4 locus homologous recombination represented. (B) Southern blot analysis confirm promoter replacement. Genomic DNA <br /> of strains digested DraII XbaI resulted 2.8 Kb 3 Kb bands MCM4 strain, 2.1 Kb band Wild type and <br /> MCM4R strain. MFD2-U1 URA-cured allele mutant. Complete ORF gene used probe. (C) Spot dilution assay Check the <br /> growth response conditional mutants absence (2) presence (+) Met/Cys containing Complete Synthetic medium (SC Agar). Mid log phase <br /> grown cultures serially diluted spotted. Plates incubated 30uC 2&#226;&#8364;&#8220;4 days. (D) qRT-PCR analysis compare expression MPS1 <br /> in wild-type (WT), heterozygous (MFD2) conditional mutant. <br /> doi:10.1371/journal.pone.0101517.g002 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                  5                               July 2014 | Volume 9 | Issue 7 | e101517 <br />                                                                                                                               Role MPS1 Candida albicans <br />  <br />  <br />  <br />  <br /> Figure 3. Fluorescence flowcytometry analysis wild type (WT) mps1 mutant strains. For studying defects nuclear <br /> division chromatin segregation mps1 mutants, cells stained DAPI 4 hours growth liquid SC medium 37uC. WT (A), <br /> heterozygous mutant (MFD2) (B), conditional mutant &#226;&#8364;&#8220;Met/Cys (C) conditional mutant +Met/Cys (D). Arrows indicate defective chromosome <br /> segregation. For flowcytometry analysis, chromatin stained Propidium Iodide, indicated time points. The X-axis indicates of <br /> DNA measured Y-axis represents number cells. Approximately, 104 cells strain analyzed FACS-Calibur machine FL2-A <br /> channel. Two imaginary lines drawn represent changes position peaks 2 N 4 N content DNA, respectively. Scale bar shown <br /> is10 mm applicable images. <br /> doi:10.1371/journal.pone.0101517.g003 <br />  <br /> attached 1 ml syringe. Peritoneal exudates smeared                     YPD agar incubated 37uC 3 days determine the <br /> microscopic glass slides stained described [2].                           C.albicans colony forming units (CFU). <br /> Photographs taken Leica digital camera attached to <br /> the microscope.10 ml peritoneal exudates added 190 ml                     H2O2 sensitivity <br /> sterile MQ lysing macrophages solution plated                      Cells grown upto OD 0.8 30uC SD media and <br />                                                                                     treated different concentrations H2O2 1 hour. Cells <br />  <br />  <br />  Table 2. Comparison wild type Dmps1 strains respect observed phenotypes DAPI staining. <br />  <br />  <br />                                                            Percentage                  Percentage dividing       Percentage mother <br />                                  Percentage non         dividing cells            cells normally          cell bud          No. cells <br />  Time       Strain               dividing cells            buds nuclei            dividing nucleus             nuclei                     counted <br />  <br />  2Hr        Wild Type            94                        1                              1                            4                          226 <br />             MFD2                 40                        38                             7                            15                         250 <br />             MCM+M/C              25                        55                             8                            12                         278 <br />             MCM-M/C              92                        2                              1                            5                          245 <br />  4Hr        Wild Type            83                        1                              2                            14                         291 <br />             MFD2                 27                        58                             6                            9                          282 <br />             MCM+M/C              21                        67                             8                            4                          274 <br />             MCM-M/C              76                        2                              3                            19                         265 <br />  <br />  Cells stained DAPI counted hemocytometer. Cells grown SD media 30uC indicated time points. Average percentages biological <br />  replicates shown table. <br />  doi:10.1371/journal.pone.0101517.t002 <br />  <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                      6                                 July 2014 | Volume 9 | Issue 7 | e101517 <br />                                                                                                                 Role MPS1 Candida albicans <br />  <br />  <br />  <br />  <br /> Figure 4. Immunostaining anti a- tubulin antibodies. Conditional mutant presence 2.5 mM Met/Cys showed formation mono polar <br /> spindles. Wild type (WT) conditional mutant absence Met/Cys showed proper bipolar spindles. White arrows indicate position spindles <br /> (Green fluorescence) strains. <br /> doi:10.1371/journal.pone.0101517.g004 <br />  <br /> were spotted H2O2 free SD agar plates incubated             function CaMPS1 gene C. albicans, created the <br /> at 30uC days determine effect H2O2.                      <span id='am-243' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-244' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-316' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-319' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>knockout</span>. Though MPS1 gene essential organisms <br />                                                                            including S.cerevisiae, MPS1 homolog (MPH1) Schizosacchar- <br /> Results                                                                    omyces pombe non-essential [33]. To confirm essentiality of <br />                                                                            MPS1 gene C. albicans, adopted specially designed single <br /> In silico analysis putative MPS1 C. albicans                         transformation based gene function test called Homozygote <br />    Genes MPS1 family highly conserved eukaryotes.             Trisome test. Southern blot analysis revealed independent <br /> Orthologs gene present majority organisms,           Arg+ Ura+ segregants mps1::UAU1/mps1::URA3/MPS1 <br /> with exception Caenorhabditis elegans. We cloned 2073 bp        triplication derivatives (Figure S2D). Thus, presence allelic <br /> putative C.albicans MPS1 homolog S. cerevisiae, using gene          triplication confirmed MPS1 essential gene. Further- <br /> sequence provided Candida Genome Database (CGD) [29].                   more, created conditional mutants replacing promoter <br /> CaMps1 encodes protein 690 amino acids estimated              region MPS1, heterozygous MFD2 strain (mps1/MPS1), <br /> Mr 77.69 kDa. A Clustal W multiple sequence alignment                   methionine (Met)/Cysteine (Cys) suppressible MET3 <br /> (http://www.ebi.ac.uk/) used determine homology              promoter (Figure 2A). Two positive strains, MCM4 MCM7 <br /> putative CaMps1, characterized Mps1 S. cerevisiae, S.        (Table 1) selected functional analysis promoter <br /> pombe, Mus musculas Human (Figure 1). CaMps1 shared                  replacement confirmed southern blot analysis (Figure 2B). <br /> significant homology 67% protein 54% DNA                      A serial dilution spot assay showed MCM4 strain (Dmps1/ <br /> sequence level S. cerevisiae Mps1. Multiple sequence              pMetMPS1) did grow presence 2.5 mM Met/Cys <br /> alignment showed C- terminal region CaMps1             suggesting conditional inhibition growth strain <br /> eleven conserved sub domains (from amino acids 367 637)                 (Figure 2C). No change observed growth MCM4 <br /> characteristic Serine/threonine kinase domain Mps1 [30]              strain absence Met/Cys (Figure 2C). The expression levels of <br /> (Figure S1A). CaMps1also tyrosine kinase specific                  MPS1 determined MET3 promoter <br /> signature, glutamine (E) residue sub domain V (at position          (Figure 2D). qRT PCR analysis showed comparison to <br /> 443) (Figure 1) [31]. The kinase function Mps1 family known          native promoter, MPS1 expression little higher the <br /> to associated processes chromosome segregation,            influence MET3 promoter absence Met/Cys. In presence <br /> genome stability vital cellular processes [32]. Phylogram        Met/Cys expression MPS1 observed. An additional <br /> revealed that, CaMps1 clustered D. hanseii&#226;&#8364;&#8482;s predicted Mps1           copy MPS1 promoter introduced MCM4 <br /> sequence closeness ScMps1 Candida glabrata Mps1             strain non essential RPS1 gene (Figure 2E). Correct <br /> (Figure S1B). ScMps2 ScMps3 formed out-group                    integration confirmed Southern blot analysis (Figure 2B). <br /> phylogram indicating evolutionary divergence compared            Growth rate studies liquid SD media showed conditional <br /> members Mps1 family kinases (Figure S1B). Apart                 mutant absence Met/Cys behaved like wild type strain, in <br /> CaMps1, Rad53 (cell cycle checkpoint) Orf19.3459 (an                   presence Met/Cys did growth. The heterozygous <br /> ortholog S. cerevisiae Mck1p) dual specificity kinases        mutant (MFD2) showed delayed growth compared of <br /> signatures C. albicans.                                                 wild type strain (Figure 2F). Growth revartant strain (MCM4R) <br />                                                                            having additional copy MPS1 neutral RPS1 locus also <br /> Disruption CaMPS1 gene                                                  analyzed. No significant change observed growth of <br />   Mps1 family protein kinase play role spindle checkpoint          MCM4R strain compared wild type. <br /> activation majority organisms studied. For determining <br />  <br />  <br /> PLOS ONE | www.plosone.org                                             7                             July 2014 | Volume 9 | Issue 7 | e101517 <br />                                                                                                                   Role MPS1 Candida albicans <br />  <br />  <br />  <br />  <br /> Figure 5. Sensitivity MPS1 mutants nocodazole. (A) Exponential phase yeast cells treated 50 mM Nocodazole 30uC and <br /> aliquots collected 2, 4 6 hours determine CFU Nocodazole free SD agar plates. (B) Cultures treated hydroxyurea <br /> for 2 hrs addition Nocodazole; aliquots collected counting CFU. (C) Cells treated Nocodazole (50 mM/ml) 6 hrs <br /> and plated Nocodazole free plates following washing serial dilution 2 days 30uC. D) Untreated control cells spotted SD agar <br /> plates 2 days 30uC. <br /> doi:10.1371/journal.pone.0101517.g005 <br />  <br />                                                                              Role MPS1 normal ploidy maintenance <br />                                                                                 Mps1 known master protein kinase, which <br />                                                                              phosphorylates downstream molecules spindle checkpoint <br />                                                                              pathway, like Mad1/Mad2 Cenp-E [32]. In budding yeast, <br />                                                                              mutants genes displayed defects chromosome segrega- <br />                                                                              tion, distribution nuclei ploidy levels [2,34]. Fluorescence <br />                                                                              microscopy 49, 69-diamidino-29-phenylindole (DAPI) stained <br />                                                                              mps1 mutant strains, revealed 58% heterozy- <br />                                                                              gous cells (MFD2) large buds nuclei (Figure 3B) <br />                                                                              (Table 2). MCM4 strain absence Met/Cys behaved wild <br />                                                                              type but, presence Met/Cys cells larger, average, <br />                                                                              cell nuclear size compared wild type around <br />                                                                              70% cells bud nuclei (Figure 3D) (Table 2).This <br />                                                                              suggested CaMps1 required proper chromosomal <br />                                                                              distribution mitosis. Growth defects single allele <br />                                                                              knockout indicates haploinsufficiency gene [35]. MAD2 <br />                                                                              gene C. albicans mammals haploinsufficiency, <br />                                                                              heterozygous mutants displayed premature anaphase and <br />                                                                              chromosome instability [2,36]. <br />                                                                                 To investigate role Mps1in chromosome segregation, we <br />                                                                              quantitatively analyzed DNA content mutants Propidium <br />                                                                              Iodide staining based flowcytometry assay. On inducing the <br />                                                                              MCM4 strain medium containing Met Cys, observed <br />                                                                              progressive shift DNA content, 2 N 4 N 8 N <br />                                                                              (Figure 3D) indicated accumulation DNA cell <br />                                                                              similarly treated control cells (Figure 3, A E). <br />                                                                              However, MCM4 strain absence Met/Cys, continued to <br />                                                                              grow 2 N content DNA cell (Figure 3C), after <br />                                                                              prolonged growth (8 hrs) cells 4 N observed. Thus, <br /> Figure 6. Polarized growth study nuclear dynamics                     fluorescence microscopy flowcytometery analysis MET3 <br /> nocodazole treated cells. Exponential phase cells indicated strains       conditional mutants, suggest role Mps1 normal mainte- <br /> were treated Nocodazole (50 mM/ml) liquid SD media 30uC           nance ploidy genomic stability C. albicans. Analysis of <br /> and harvested indicated time points stained DAPI. <br /> doi:10.1371/journal.pone.0101517.g006                                        microtubule organization performed immuno- <br />  <br />  <br /> PLOS ONE | www.plosone.org                                               8                             July 2014 | Volume 9 | Issue 7 | e101517 <br />                                                                                                                  Role MPS1 Candida albicans <br />  <br />  <br />  <br />  <br /> Figure 7. Expression analysis MPS1. A) qRT-PCR analysis determine expression MPS1 indicated hyphae inducing media. B) qRT-PCR <br /> to determine regulation MPS1 expression presence serum. C) Promoter analysis MPS1 gene: 500 bp upstream sequence MPS1 was <br /> analyzed using Regulatory Sequence Analysis tools (RSAT) server at, http://rsat.ulb.ac.be/rsat. Efg1p binding element 59CANNTG39 identified <br /> at locations. <br /> doi:10.1371/journal.pone.0101517.g007 <br />  <br />  <br />  <br />  <br /> Figure 8. Morphology mps1 mutant strains different hyphae inducing conditions. (A) Morphogenetic studies solid inducing <br /> media: Cultures study grown mid log phase, serially diluted 25 50 cells plate spreaded indicated hyphae inducing <br /> media. (B) Morphologies respective liquid inducing media; yeast cells overnight cultures inoculated indicated inducing media at <br /> 26105 cells/ml 3 hours. (C) Chlamydospore formation observed 7 days 25uC cornmeal/Tween agar plates coverslips. <br /> doi:10.1371/journal.pone.0101517.g008 <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              9                             July 2014 | Volume 9 | Issue 7 | e101517 <br />                                                                                                                         Role MPS1 Candida albicans <br />  <br />  <br />                                                                                    passage cell cycle absence SAC <br />                                                                                    function, caused cell death mps1 mutants response to <br />                                                                                    Nocodazole. <br />                                                                                       Additionally, serial dilution spot assay indicated loss of <br />                                                                                    growth presence Nocodazole (Figure 5C) compared to <br />                                                                                    untreated control plate (Figure 5D). Both heterozygous and <br />                                                                                    conditional mutants (+Met/Cys) showed sensitivity Nocodazole <br />                                                                                    treatment (6 hr). It observed absence of <br />                                                                                    Nocodazole Met/Cys treated conditional mutant showed loss <br />                                                                                    viability (Figure 5D). This essential role Mps1 <br />                                                                                    cell division. Thus, rapid fall viability Nocodazole <br />                                                                                    treated conditional mutant presence Met/Cys be <br />                                                                                    attributed dual action microtubule disruption and <br />                                                                                    suppression MPS1 expression. Conditional mutants absence <br />                                                                                    Met/Cys behaved wild-type. These results demonstrate role <br />                                                                                    Mps1 activation SAC C.albicans. <br />                                                                                       The morphogenetic response C. albicans checkpoint <br />                                                                                    inducing drugs like Nocodazole quite different of <br />                                                                                    budding yeast, S. cerevisiae yeast cells remain largely <br />                                                                                    budded multiple buds [37,38]. In C.albicans, conditions <br /> Figure 9. Oxidative stress tolerance mps1 mutants. A) Mutants                   arrest cell cycle progression result polarized growth <br /> were cultured Liquid SD medium till mid log phase resuspended               phenotype [38,39]. To determine morphological response, <br /> in PBS (pH 7.4) containing indicated concentrations H2O2 treated            DAPI staining fluorescence microscopy performed after <br /> for 1 hour 37uC. Cells spotted serial dilution H2O2 free <br /> SC-Agar plates incubated 30uC 2 days. (B) 10 ml peritoneal <br />                                                                                    treating cells 50 mM nocodazole. It observed after <br /> exudates mice injected indicated strain added 190 ml                2 hrs drug treatment nearly wild type cells formed elongated <br /> sterile MQ plated YPD agar plates determine CFU. Error bars              buds, nuclei indicating cell cycle arrest (Figure 6). After <br /> represents standard error biological replicates. (C)                 4 hr, wild type cells showed pseudo-hyphal like extensions, with <br /> Morphology wild type (WT) heterozygous mps1 mutant strains                  single nucleus present mother cell elongated bud. The <br /> injected peritoneal cavity mouse. Peritoneal exudates             heterozygous conditional mutant (MCM4) presence of <br /> retrieved 24 hrs injection. C. albicans cells indicated strains <br /> engulfed macrophages stained PAS stain. Arrows indicate               Met/Cys showed buds different sizes suggesting absence cell <br /> C. albicans cells engulfed macrophages.                                         cycle arrest (Figure 6). Conditional mutant absence Met/Cys <br /> doi:10.1371/journal.pone.0101517.g009                                              behaved like wild type (Figure 6). After 4 hours heterozygous and <br />                                                                                    conditional mutant cells showed irregular shape often <br /> staining a-tubulin. Wild type conditional mutant (-Met/                    multinucleate (Figure 6). Thus, Mps1 plays important role as <br /> Cys)) cells displayed normal bipolar spindle organization,                     cell cycle checkpoint, involved regulating the <br /> where spindles appear originating opposite end                  Nocodazole induced polarized growth. <br /> of cells (Figure 4). In contrast, microtubules conditional <br /> mutant (+Met/Cys) present end cells, indicative                 Role Mps1 morphogenesis <br /> of cells having monopolar spindles (Figure 4). Thus, defective                        C. albicans polymorphic organism exhibits variety of <br /> microtubule organization reason inaccurate chromo-                      cellular forms. The ability organism switch between <br /> somal segregation mps1 mutants determined DAPI                            distinct modes proliferation contributes virulence. <br /> staining.                                                                          We investigated role Mps1 yeast hyphae <br />                                                                                    morphological transition C. albicans. qRT-PCR analysis showed <br /> MPS1 required activation spindle checkpoint                          higher expression MPS1 gene major filamentation inducing <br />                                                                                    conditions like, GlcNAc, spider serum compared of <br /> pathway                                                                            Glucose containing media (Figure 7A). Maximum expression of <br />    Mps1 family protein kinases budding yeast                       MPS1 observed 10% Serum (3 folds) followed GlcNAc (2 <br /> organisms roles activation Spindle Assembly                         folds) compared glucose grown cells. <br /> Checkpoint (SAC) pathway spindle pole body duplication                            To identify signaling network required expression of <br /> [32]. To determine loss SAC function mps1 mutants,                  MPS1 gene, qRT PCR analysis performed mutants of <br /> heterozygous conditional mps1 mutant strains exposed                   major signal transduction pathways control morphogenesis. <br /> the microtubule toxin, Nocodazole. Cells treated 50 mM                   This analysis showed expression MPS1 unchanged <br /> Nocodazole 2, 4 6 hours harvested determine                         Dtpk2, Dcph1 Dras1 mutants, Defg1 mutant the <br /> colony forming units (CFUs) SC-Met/Cys agar plates (Material                    expression suppressed basal level (Figure 7B). Thus, the <br /> and methods). The results showed increase time,                       expression MPS1 dependent cAMP mediated signaling <br /> conditional mutants presence Met/Cys, readily lost viability                 pathway Efg1, independent Tpk2 Ras1. <br /> in presence Nocodazole compared untreated cells                           Computational analysis MPS1 promoter, showed Efg1 <br /> (Figure 5A). Moreover, treatment Nocodazole                            binding elements (CANNTG) 500 base pairs upstream of <br /> hydroxy-urea treatment given arrest cells S-                    start codon, positions 246, 2181 2417 (Figure 7C). <br /> phase cell cycle inhibiting ribonucleotide reductase.                       Higher expression MPS1 major hyphae inducing <br /> Under condition, cell mortality minimized, ,95%                     conditions suggests role morphogenesis. So, analyzed <br /> of wild type ,85% heterozygous mutant cells retained                    morphological transition mps1 mutants various solid and <br /> viability (Figure 5B). This showed microtubule disruption                  liquid filamentation inducing media like Spider, SLAD, GlcNAc <br /> Nocodazole responsible cell death,                  10% Bovine Serum. The heterozygous strain (MFD2) showed <br />  <br />  <br /> PLOS ONE | www.plosone.org                                                    10                             July 2014 | Volume 9 | Issue 7 | e101517 <br />                                                                                                                   Role MPS1 Candida albicans <br />  <br />  <br />  <br />  <br /> Figure 10. Model illustration possible regulatory mechanism activation MPS1 gene C. albicans, Efg1 dependent <br /> but Ras1 independent manner, leads morphogenesis represented lined arrows. A classical cAMP-PKA pathway included <br /> for comparison. Role Mps1 SAC activation ploidy maintenance shown dotted arrows. This model represents Efg1 conjunction <br /> point SAC machinery morphogenesis. <br /> doi:10.1371/journal.pone.0101517.g010 <br />  <br /> defective filamentation solid agar media tested             macrophages [2]. Since CaMps1is required activation of <br /> (Figure 8A). The conditional mutant (MCM4), absence Met/               spindle checkpoint pathway, analyzed involvement in <br /> Cys behaved wild type strain (Figure 8A). Like solid               oxidative stress tolerance. Thus, mps1 mutants treated with <br /> media, liquid inducing media also, heterozygous mutant cells              H2O2 (60 min) spotted H2O2 free SC agar plates, to <br /> have reduced response yeast hyphal transition            facilitate growth cells escaped free radical attack. <br /> prolonged incubation (Figure 8B). Morphologically, MFD2 cells                After 2 days incubation 30uC, mps1 mutant strains showed <br /> were elongated, swollen formed pseudo hyphal like growth                 sensitivity H2O2 compared wild type strain <br /> without true hyphae. The morphology conditional MCM4                  (Figure 9A). The conditional mutant MCM4 incubated H2O2 <br /> strain absence Met/Cys rescued strain (MCM4R)                   presence Met/Cys showed higher sensitivity as <br /> liquid culture comparable wild type media tested (data         compared heterozygous strain. This result demonstrates the <br /> not shown).                                                                  role Mps1 oxidative stress tolerance C. albicans. <br />    One distinctive morphological features C. albicans              Since MPS1 gene required oxidative stress response, is <br /> formation Chlamydospores oxygen limited environment.                particularly important check response macrophages, <br /> So, role Mps1in chlamydospore formation analyzed.                oxidative free radical attack hand defence. To <br /> Strains grown 10 days dark 25uC                         examine heterozygous mutant control wild type strains <br /> microaerophilic conditions. Large numbers chlamydospores                  injected peritoneal cavity mice. Cells were <br /> were observed wild type strain, situated end                   subsequently retrieved peritoneal exudates 24 hours <br /> branching filaments, heterozygous mutant showed               injection. Survival heterozygous mutant (MFD2) within <br /> sparse elongated branching filaments significantly               macrophages determined plating exudate YPD <br /> reduced chlamydospores end filaments (Figure 8C). No                   agar counting colony forming units (CFU) strains. A <br /> change chlamydospore formation observed                        5-fold decrease CFU observed MFD2 strain in <br /> heterozygous strain prolonged incubation 21 days.              comparison wild type (WT) strain (Figure 9B). This be <br /> Conditional mutant presence Met/Cys showed response                  attributed sensitivity MFD2 strain oxidative stress on <br /> similar wild type (Figure 8C).                                            exposure macrophages. Since, hyphae formation C. albicans is <br />                                                                              required rupturing macrophages facilitate escape from <br /> Sensitivity mps1 mutants oxidative stress                              hostile environment phagosome. A microscopic <br />   One important ways macrophages host                     examination peritoneal exudates performed examining <br /> immune defend endocytosed pathogens, release             morphological transition exposure macrophages in <br /> reactive oxygen species. So, pathogens evolved mechanisms               vivo conditions. After 24 hrs exposure macrophages wild <br /> required protecting free radical effects. It        type strain, engulfed macrophages showed significant filamen- <br /> was previously demonstrated that, checkpoint proteins                    tation (Figure 9C). On contrary, MFD2 strain, engulfed with <br /> required encountering free radical attacks caused                macrophages showed yeast form (Figure 9C). Thus, <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              11                             July 2014 | Volume 9 | Issue 7 | e101517 <br />                                                                                                                     Role MPS1 Candida albicans <br />  <br />  <br /> heterozygous mps1 mutant failed undergo morphological                     future challenge <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> signaling (like <br /> transition response macrophages.                                  GlcNAc) developmental processes C. albicans coordi- <br />                                                                              nated spindle assembly checkpoint machinery, shown <br /> Discussion                                                                   Caenorhabditis elegans [40]. <br />  <br />    In microbial pathogens like C. albicans spindle checkpoint            Supporting Information <br /> machinery play important role survival host. Because <br /> these organisms grow constant threat host defense               Figure S1    Computational characterization Mps1p. (A) <br /> mechanisms, damage cellular components like DNA                     Cartoon presentation Kinase domain C-terminal of <br /> inevitable. The checkpoint machinery ensures proper chromo-                  deduced amino acid sequence, identified using motif <br /> somal segregation. Deregulation checkpoint machinery                 finder Expasy server. Computationally predicted Ser/thr kinase <br /> leads aneuploidy chromosomal instability. In report,          domain located aa 367 637 Carboxy terminus. <br /> have characterized S. cerevisiae Mps1 homolog C. albicans.            (B) Phylogenic tree constructed studying evolutionary distances <br /> Essentiality MPS1 C. albicans confirmed Homozygote              known predicted MPS1 family protein kinases. <br /> Trisome test. To study function gene, created              The phylogram bootstrap analysis performed at <br /> conditional mutants replacing promoter MPS1                   Megatree server (http://www.megatree.com). Values start <br /> Methionine/Cysteine regulatable MET3 promoter. Analysis                   branch point indicate calculated distances peptide homology. <br /> mps1 mutants showed normal conditions gene                Organisms respective gene accession numbers <br /> required proper segregation chromosomes. When mutant                  provided. Sequences downloaded NCBI genome <br /> cells stained DAPI, displayed single, largely         sequence depository. <br /> stained nuclear region buds nucleus, suggesting            (TIF) <br /> mutant cells failed complete nuclear division. Flowcyto-             Figure S2 Mutant construction essentiality confir- <br /> metry analysis showed increase ploidy levels mps1             mation Mps1. (A) A schematic presentation construction of <br /> mutants time. This particular feature reminiscent                  cassette used disrupting MPS1gene, URA blaster <br /> behavior mps1-1 mutants budding yeast [6].                         technique. Disruption cassette, mps1: hisG:: URA3::hisG:mp s1 was <br />    In eukaryotes spindle assembly checkpoint highly                   employed inactivating allele, homologous <br /> conserved. It monitors attachment kinetochores               recombination shown. Restriction enzymes used preparing <br /> microtubules prometaphase. Under conditions                     cassette DNA fragment used probe screening the <br /> kinetocores lack proper tension lack microtubule occupancy             transformants southern analysis shown. (B) <br /> at kinetochore, halts metaphase anaphase transition.               Autoradiogram confirmation allele mutants through <br /> Several genes S. cerevisiae C. albicans share evolutionarily          PstI enzyme digestion. The indicated 6.1 kb 2 kb bands are <br /> conserved functions. In S. cerevisiae Mps1 play important role         integration hUh cassette MPS1, undisrupted <br /> SAC. So, analyzed SAC function Mps1 conserved               second allele respectively. Twelve positive transformants (MFD1- <br /> in C. albicans. We adopted Nocodazole sensitivity assay          12) shown support high efficiency recombination. (C) <br /> the role MPS1 gene spindle checkpoint activation. In               Confirmation essentiality Heterozygote Trisome test. <br /> presence Nocodazole, mutants readily lost viability,                Disruption cassette, mps1:Ura3&#226;&#8364;&#8482;:: ARG4::Ura3&#226;&#8364;&#8482;:mps1 used for <br /> suggesting that, gene required normal activation              inactivating allele schematically represented. An internal <br /> spindle assembly checkpoint (SAC) pathway. Moreover, Nocoda-                 cis-recombination help reconstruct active URA3 gene from <br /> zole treated mutants cells showed absence cell cycle arrest.         flanking Ura3&#226;&#8364;&#8482; fragments. (D) Southern blot confirmation first <br /> This suggests SAC function conserved C. albicans             allele disruptants triploids (mps1::UAU1/mps1::URA3/MPS1) <br /> Mps1.                                                                        obtained HT test. In autoradiogram, recurrent 2.3 kb band <br />    Further, attempt unravel functional link             observed colonies screened; indicates triploids the <br /> morphogenesis checkpoint signaling genes, studied                 locus MPS1 gene confirmed essentiality gene. <br /> MPS1 expression various filamentation inducing media                  (TIF) <br /> mutants major signal transduction pathway involved in <br />                                                                              Methods S1 Supplementary methods. <br /> morphogenesis. qRT-PCR analysis revealed upregulation of <br />                                                                              (DOC) <br /> MPS1 presence Serum N-acetyl Glucosamine in <br /> comparison glucose grown cells. qRT-PCR analysis also <br /> revealed MPS1 expression regulated Efg1, Ras1,               Acknowledgments <br /> Tpk2 independent pathway. Further, interpreted                Authors wish thank W.A.Fonzi, Aaron P.Mitchell P.E.Sudbery for <br /> the Efg1 dependent expression checkpoint kinase like MPS1               generously providing C.albicans strains plasmids. We grateful to <br /> suggests additional roles Efg1 regulating morphogenesis                John Carbon, Neil A. Gow D.Chattopadyaya suggestions in <br /> through spindle checkpoint signaling pathway (Figure 10). Mor-               cell synchronization flowcytomtery protocols S.S. Thakur for <br />                                                                              helping animal experiments. We thank Abira Chaudhuri for <br /> phological studies various vitro hyphae inducing conditions <br />                                                                              critical reading manuscript. VKN thanks ICMR research <br /> showed involvement Mps1 morphogenesis including chla-                  fellowship. MK AK thank CSIR SRA fellowship. AD thank <br /> mydospore formation C. albicans. Mps1 knockouts showed               Department Biotechnology National Institute Plant Genome <br /> sensitivity oxidative stress. One mechanisms C.          Research, Ministry Science Technology, Government India, for <br /> albicans uses evade macrophages morphological                  funding project. <br /> transition. Analysis heterozygous mps1 mutant (MFD2) engulfed <br /> by Macrophages showed presence yeast forms. Taken                    Author Contributions <br /> together, experiments MPS1 gene C. albicans             Conceived designed experiments: AD MK VKN AK. Performed <br /> obligatory functions survival, checkpoint activation, morpho-             experiments: MK VKN DR PM. Analyzed data: AD MK VKN <br /> genesis oxidative stress tolerance. The step elucidate        AK. Contributed reagents/materials/analysis tools: AD. Wrote paper: <br /> the interacting proteins assist functions Mps1. It           AD MK VKN AK. <br />  <br />  <br /> PLOS ONE | www.plosone.org                                              12                              July 2014 | Volume 9 | Issue 7 | e101517 <br />                                                                                                                                              Role MPS1 Candida albicans <br />  <br />  <br />  <br />  <br /> References <br />  1. Odds FC (1994) Candida albicans, life times pathogenic yeast. J Med         21. Barelle CJ, Bohula EA, Kron SJ, Wessels D, Soll DR, et al. (2003) Asynchronous <br />     Vet Mycol. 32: 1&#226;&#8364;&#8220;8.                                                                          cell cycle asymmetric vacuolar inheritance true hyphae Candida albicans. <br />  2. Bai C, Ramanan N, Wang YM, Wang Y (2002) Spindle assembly checkpoint                         Eukaryot Cell 2: 398&#226;&#8364;&#8220;410. <br />     component CaMad2p indispensable Candida albicans survival virulence           22. Chen CY, Rosamond J (1998) Candida albicans SSD1 suppress multiple <br />     mice. Mol Microbiol 45: 31&#226;&#8364;&#8220;44.                                                            mutations Saccharomyces cerevisiae. Microbiol 144: 2941&#226;&#8364;&#8220;2950. <br />  3. Finley KR, Bouchonville KJ, Quick A, Berman J (2008) Dynein-dependent                    23. Fonzi WA, Irwin MY (1993) Isogenic strain construction gene mapping in <br />     nuclear dynamics affect morphogenesis Candida albicans means                    Candida albicans. Genetics 134: 717&#226;&#8364;&#8220;728. <br />     Bub2p spindle checkpoint. J Cell Sci 121: 466&#226;&#8364;&#8220;476.                                       24. Care RS, Trevethick J, Binley KM, Sudbery PE (1999) The MET3 promoter: a <br />  4. Li R, Murray AW (1991) Feedback control mitosis budding yeast. Cell 66:                new tool Candida albicans molecular genetics. Mol Microbiol 34: 792&#226;&#8364;&#8220;798. <br />     519&#226;&#8364;&#8220;531.                                                                                 25. Gietz D, St Jean A, Woods RA, Schiestl RH (1992) Improved method high <br />  5. Hoyt MA, Totis L, Roberts BT (1991) S. cerevisiae genes required cell cycle              efficiency transformation intact yeast cells. Nucleic Acids Res 20: 1425. <br />     arrest response loss microtubule function. Cell 66: 507&#226;&#8364;&#8220;517.                    26. Murad AM, Lee PR, Broadbent ID, Barelle CJ, Brown AJ (2000) CIp10, an <br />  6. Winey M, Goetsch L, Baum P, Byers B (1991) MPS1 MPS2: novel yeast                        efficient convenient integrating vector Candida albicans. Yeast 16: 325&#226;&#8364;&#8220; <br />     genes defining distinct steps spindle pole body duplication. J Cell Biol 114:             327. <br />     745&#226;&#8364;&#8220;754.                                                                                 27. Sambrook J, Maniatis T, Fritsch EF (1989) Molecular cloning: laboratory manual. <br />  7. Sanyal K, Carbon J (2002) The CENP-A homolog CaCse4p pathogenic                       Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory Press. <br />     yeast Candida albicans centromere protein essential chromosome                  28. Sherman F (1991) Methods Enzymol 194: 3&#226;&#8364;&#8220;21 <br />     transmission. Proc Natl Acad Sci USA 99: 12969&#226;&#8364;&#8220;12974.                                    29. Arnaud MB, Costanzo MC, Skrzypek MS, Binkley G, Lane C, et al. (2005) The <br />  8. Lauze E, Stoelcker B, Luca FC, Weiss E, Schutz AR, et al. (1995) Yeast spindle               Candida Genome Database (CGD), community resource Candida albicans <br />     pole body duplication gene MPS1 encodes essential dual specificity protein                gene protein information. Nucleic Acids Res 33: D358&#226;&#8364;&#8220;363. <br />     kinase. Embo J 14: 1655&#226;&#8364;&#8220;1663.                                                            30. Hanks SK, Quinn AM, Hunter T (1988) The protein kinase family: conserved <br />  9. Salvatore G, Nappi TC, Salerno P, Jiang Y, Garbi C, et al. (2007) A cell                     features deduced phylogeny catalytic domains. Science 241: 42&#226;&#8364;&#8220;52. <br />     proliferation chromosomal instability signature anaplastic thyroid                31. Lindberg RA, Quinn AM, Hunter T (1992) Dual-specificity protein kinases: will <br />     carcinoma. Cancer Res 67: 10148&#226;&#8364;&#8220;10158.                                                       hydroxyl do? Trends Biochem Sci 17: 114&#226;&#8364;&#8220;119. <br /> 10. Yuan B, Xu Y, Woo JH, Wang Y, Bae YK, et al. (2006) Increased expression              32. Fisk HA, Mattison CP, Winey M (2004) A field guide Mps1 family of <br />     mitotic checkpoint genes breast cancer cells chromosomal instability.                protein kinases. Cell Cycle 3: 439&#226;&#8364;&#8220;442. <br />     Clin Cancer Res 12: 405&#226;&#8364;&#8220;410.                                                             33. He X, Jones MH, Winey M, Sazer S (1998) Mph1, member Mps1-like <br /> 11. Daniel J, Coulter J, Woo JH, Wilsbach K, Gabrielson E (2011) High levels              family dual specificity protein kinases, required spindle checkpoint in <br />     Mps1 checkpoint protein protective aneuploidy breast cancer cells. Proc            S. pombe. J Cell Sci 111: 1635&#226;&#8364;&#8220;1647. <br />     Natl Acad Sci USA 108: 5384&#226;&#8364;&#8220;5389.                                                        34. Weiss E, Winey M (1996) The Saccharomyces cerevisiae spindle pole body <br /> 12. Landi MT, Dracheva T, Rotunno M, Figueroa JD, Liu H, Dasgupta A, et al.                      duplication gene MPS1 mitotic checkpoint. J Cell Biol 132: 111&#226;&#8364;&#8220;123. <br />     (2008) Gene expression signature cigarette smoking role lung               35. Uhl MA, Biery M, Craig N, Johnson AD (2003) Haploin <br /> </body></html>