<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal vol.12 no.1 pp.285-294, 1993 <br />  <br />  <br /> Isolation characterization S.cerevisiae mutants <br /> defective somatostatin expression: cloning and <br /> functional role yeast gene encoding aspartyl <br /> protease precursor processing monobasic cleavage <br /> sites <br /> Yves Bourbonnais&apos; 3, Josee Ash1,                                      terminally 14-amino acid extended form SRIF-14 (somato- <br /> Mireille Daigle&apos; David Y.Thomas&apos; 2                                tropin releasing inhibitory factor). Both peptides display <br />                                                                       widespread effects inhibiting secretion numerous <br /> &apos;Eukaryotic Genetics Group, National Research Council Canada,      hormones including growth hormone, glucagon insulin. <br /> Biotechnology Research Institute, 6100 avenue Royalmount, Montreal, <br /> Quebec, H4P 2R2 2Department Biology, McGill University,        Although overlap activity SRIF-14 and <br /> Montreal, Quebec, H3A IBI, Canada                                     -28, distinctive biological effects. In mammals <br /> 3Corresponding author                                                 bioactive peptides synthesized common <br />                                                                       precursor released endoproteolytic activation; single <br /> Communicated Hugh R.B.Pelham                                       arginine residue (Arg - 15) constitutes cleavage site for <br />                                                                       release SRIF-28, proteolytic cleavage an <br /> The peptide somatostatin exists different molecular            internal dibasic sequence (Arg -2 Lys -1) generates <br /> species. In addition common form, somato-                 SRIF- 14. Differential processing cleavage sites has <br /> statin-14, fourteen amino acid N-terminally           shown tissue-specific (Patel et al., 1981; Noe <br /> extended form tetradecapeptide, somatostatin-28.               Spiess, 1983). However, species fish, <br /> Both peptides synthesized larger precursors                    SRIF-14 -28 encoded distinct related <br /> containing paired basic monobasic amino acids                  precursors, preproSRIF-I -II, expressed in <br /> their processing sites, cleavage generate                  different subpopulations pancreatic islet cells (McDonald <br /> either somatostatin-14 -28, respectively. In                  et al., 1987). Despite possessing potential cleavage <br /> species fish distinct, homologous, precursors              sites (i.e. Arg -5 Arg -2 Lys -1), proSRIF-I cleaved <br /> (prosomatostatin-I -II) rise somatostatin-14              Arg -Lys pair yield SRIF-14, SRIF-28 <br /> and -28, respectively. Whereas anglerfish prosomato-                  unique end product maturation proSRIF-II. <br /> statin-TI previously shown release exclusively                    In attempt understand molecular basis for <br /> somatostatin-28, yeast Saccharomyces cerevisiae                   differential processing anglerfish prosomatostatins, we <br /> proteolytically matures homologous prosomatostatin-I              initiated studies biosynthesis precursors in <br /> precursor somatostatin-28 -14 lysine-          yeast Saccharomyces cerevisiae. The choice yeast as <br /> extended form somatostatin-14. The Kex2 endo-                      model predicated observation time <br /> protease appears essential formation lysine          yeast unique eukaryote prohormone- <br /> somatostatin-14 involved directly                    converting enzyme unambiguously identified (Julius <br /> indirectly release mature somatostatin-14. The              et al., 1984). Furthermore, recent studies indicated the <br /> isolation yeast mutants defective somatostatin-28               yeast KEX2 gene product, member subtilisin <br /> expression (sex mutant) allowed cloning non-                 family serine proteinases, functionally and <br /> essential gene, encodes aspartyl protease,             structurally related mammalian counterparts (Thomas <br /> disruption severely affects cleavage mature                    et al., 1988; Seidah et al., 1990; Smeekens Steiner, <br /> somatostatin-28 somatostatin precursors. We                 1990; Smeekens et al., 1991). Our previous study revealed <br /> conclude distinct endoproteases, demon-                native somatostatin precursors faithfully <br /> strate cross specificity vivo, involved            transported yeast secretory pathway unless signal <br /> proteolytic maturation prosomatostatin mono-                peptides replaced leader sequence yeast <br /> dibasic processing sites yeast.                                    ca-factor precursor (Bourbonnais et al., 1991). Expression <br /> Key words: heterologous gene expression/prohormone                    ct-factor - proSRIF-IH chimera promoted efficient <br /> processing/somatostatin/yeast aspartyl protease/yeast mutants         transport led secretion mature SRIF-28, thus <br />                                                                       demonstrating accurate proteolytic cleavage monobasic <br />                                                                       processing site (Arg - 15). Most importantly, provided <br />                                                                       genetic evidence convertase involved the <br /> Introduction                                                          maturation proSRIF-II Arg -15 distinct the <br /> Most small peptide hormones neuropeptides                     paired basic-specific endoprotease encoded KEX2 gene <br /> synthesized larger polyproteins undergo series           (Julius et al., 1984). In present study investigated <br /> post-translational modifications generate bioactive            biosynthesis similar chimeric protein proSRIF-I <br /> molecule (Douglass et al., 1984). A common feature                 developed screening procedure isolation yeast <br /> many precursors pairs basic amino acids flank          mutants defective somatostatin-28 expression (sex <br /> active peptide sequences constitute endoproteolytic           mutants). Genetic complementation sex]-] mutant <br /> cleavage site. However, examples                 strain led cloning gene, YAP3, recently isolated <br /> prohormones bioactive peptide cleaved              group based ability partially suppress <br /> monobasic processing sites paired basic                <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>phenotype</span> kex2 null mutant (Egel-Mitani et al., <br /> residues (Schwartz, 1986; Devi, 1991). This cleavage usually           1990). We gene, encodes an <br /> occurs single arginine residues. Somatostatin-28 N-          aspartyl protease, required release SRIF-28 <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                                                                                  285 <br />  Y.Bourbonnais et al. <br />  <br />  SRIF precursors cleavage single arginine <br />  residue (Arg -15). Consistent role pro a-factor <br />  maturation absence functional KEX2 gene                                                     %pSI D                       14S aa <br />  (Egel-Mitani et al., 1990), data endo- <br />  proteases encoded KEX2 YAP3 demonstrate               Fig. 1. Schematic representation chimeric protein encoded by <br />                                                                    plasmid pSiD. The fusion protein encoded plasmid pSID 145 <br />  cross specificity vivo.                                        amino acids long composed 49 N-terminal residues <br />                                                                    prepro a-factor ( 1 ), including N-linked glycosylation site <br />                                                                    (?), fused frame anglerfish proSRIF-I. The SRIF-I precursor is <br />  Results                                                           composed 82 amino acid proregion ( g ) containing a <br />                                                                    mono- dibasic processing site (0). The basic residues the <br />   Mature SRIF- 14, mature SRIF-28 Lys-SRIF- 14             cleavage sites indicated single letter <span id='am-2' about='Thesaurus:code' typeof='owl:Thing'>code</span>. The mature <br />   end products maturation proSRIF-I                  SRIF-14 peptide (U) located carboxyl terminal end the <br />   chimera wild type yeast                                       precursor. <br />   The strategy used construction the <br />  proSRIF-IH (Bourbonnais et al., 1991) proSRIF-I <br />  chimeric genes. The plasmid carrying proSRIF-I fusion                   50   0)                                                   2000 <br />  gene called pSID gene expression the <br />  control ca-factor promoter. The hybrid protein is <br />  composed 49 amino terminal residues prepro                  &apos;1200 <br />  a-factor fused frame <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> proSRIF-I (preproSRIF- <br />  I 26-121) (Figure 1). The peptide SRIF-14 located              CL <br />                                                                              3000 <br />  carboxyl-terminal end precursor flanked            If <br />                                                                                                                                        1 000 <br />  pair basic amino acid residues (Arg -2 Lys -1). A             ulI <br />                                                                              2000 <br />  single arginine residue, cleavage generates SRIF-28, <br />  located 15 residues upstream SRIF-14 (Arg- 15). <br />  The hybrid precursor possess cysteine residues,                    &apos; 000 <br />  reside SRIF-14 moiety positions 3 and <br />   14 respectively. Consequently, [35S]cysteine used all <br />  metabolic labelling experiments. The labelled proteins from <br />  cell lysates media immunoprecipitated <br />  anti-SRIF-14 polyclonal antibody (RSSI B23;                                       FR ACT ION     NUMBEFR <br />  Materials methods). Analysis SRIF-14 immuno- <br />  reactive material secreted wild type yeast transformed       Fig. 2. Biosynthesis  a-factor-proSRIF-I fusion protein wild <br />  pSID (strain 20B-12) high performance liquid             type yeast. Strain 20B-12 transformed plasmid pSID labelled <br />                                                                   [35S]cysteine culture medium (U-U) the <br />  chromatography (HPLC) revealed presence major             intracellular labelled material (0-0) submitted <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br />  SRIF-14-like species eluting C18 column              immunoprecipitation anti-SRIF-14 antibodies (see Materials and <br />  fractions 15, 19 23, respectively (Figure 2). The labelled   methods). The SRIF-14 immunoreactive material loaded onto <br />  immunoreactive material fraction 23 co-eluted          C18 column, eluted gradient acetonitrile 1 min <br />  synthetic SRIF-14 used standard. The major            fractions collected radioactivity determined by <br />                                                                   scintillation counting described previously Bourbonnais et al. <br />  secreted peptides resolved high resolution    (1988). The arrow indicates position elution synthetic <br />  SDS -Tricine gel (Figure 2, inset). A calibration experiment     SRIF-14. Note ordinates secreted intracellular <br />  performed HPLC-purified peptides indicated         material left- right-side figure respectively. Inset, <br />  3, 2.2 1.8 kDa species correspond immuno-         analysis secreted SRIF-14 immunoreactive material 20B-12 <br />                                                                   transformed plasmid pSID high-resolution SDS-Tricine gel <br />  reactive material eluting fraction 15, 19 23, respect-    (see Materials methods details). Note lower <br />  ively (not shown). The 3 kDa peptide co-migrated            portion corresponding autoradiogram shown. hSRIF-28 and <br />  synthetic human SRIF-28, 1.8 kDa species             SRIF-14 indicate positions migration human synthetic <br /> co-migrated synthetic SRIF-14. The identity         SRIF-28 synthetic SRIF-14, respectively. <br /> peptides confirmed partial amino acid sequencing <br /> of HPLC-purified radiolabelled peptides (data              44 54, respectively. Based elution position and <br /> shown). As expected amino acid sequence mature         molecular masses, estimated SDS -Tricine gels <br /> SRIF-14, radioactivity 1.8 kDa peptide (fraction        (17 19 kDa), correspond pro prepro <br /> 23 HPLC) came cycles 3 14. Amino acid              forms hybrid protein. <br /> sequence determination 2.2 kDa species (fraction 19           Quantification radioactivity recovered five <br /> from HPLC) revealed presence cysteine residues          SRIF species intracellularly indicates that <br /> positions 4 15 respectively, identifying       processed forms accounted &gt;40% total SRIF- <br /> peptide lysine-extended form SRIF-14 (Lys-SRIF-14).       immunoreactive material. Overall (cell lysate plus medium) <br /> For material eluting fraction 15, possible       - 80% radioactivity recovered mature <br /> to sequence. This probably              peptides this, 86% culture medium <br /> relatively low radioactive material available       end labelling period (30 min). The precursor <br /> radiosequencing remote locations (at positions 17         forms secreted significant amounts (&lt;5 %). <br /> and 28) cysteine residues SRIF-28. In addition       These observations strongly argue chimeric protein <br /> these secreted peptides, HPLC analysis intracellular       transported efficiently secretory pathway that <br /> material previously treated endoglyco-         processing precursor did place culture <br /> sidase H identified SRIF species eluting fractions   medium. Maturation hybrid precursor SRIF-14 and <br /> 286 <br />                                                                                S.cerevisiae mutants defective somatostatin expression <br />  <br />                                                                          arginine residue (Arg -15), releases SRIF-28 from <br />                                 12                                       SRIF precursors, effected endoprotease distinct <br />                                                                          Kex2. <br />                                                                          Isolation partial characterization of <br />                                                                          somatostatin-28 expression mutants <br />                                                                          Problems inherent purification non-abundant <br />                                                                          proteinases prompted clone gene encoding novel <br />                                                                          activity direct genetic complementation mutant <br />                                                                          yeast unable recognize and/or cleave the <br />                                                                          monobasic site present proSRIFs. To facilitate the <br />                                                                          screening procedure involved isolation a <br />                                                                          mutant plasmid pS2B, encodes proSRIF-I1 <br />                                                                          fusion protein (Figure 4A), modified site-directed <br />                                                                          mutagenesis order replace SRIF-28 moiety a <br />                                                                          single copy a-factor peptide. The resulting gene was <br />                                                                          named pS3C encodes fusion protein 132 amino acids <br /> Fig. 3. High-resolution SDS-Tricine gel secreted SRIF-14          (see Figure 4A structure). Expression gene is <br /> immunoreactive material wild type kex2 mutants transformed      control a-factor promoter. Plasmid pS3C <br /> with pSID. Wild type (strain 20B-12, lane 1) kex2 yeast (strain      introduced ca cells carry disrupted copy <br /> DS20, lane 2) transformed plasmid pSID labelled           MFal MFa2 genes (strain DS7) secretion <br /> [35S]cysteine culture media immunoprecipitated anti-   bioactive ca-factor strain requires proper <br /> SRIF-14 antibodies described earlier (Figure 2). The labelled <br /> proteins resolved high resolution SDS-Tricine gel.        maturation hybrid precursor monobasic site. <br /> Arrowheads left indicate position migration         The rationale use assays based secretion of <br /> prestained low molecular weight standards (Gibco/BRL): ovalbumin         bioactive ca-factor (i.e. &apos;halo&apos; mating assays, Julius <br /> (43.0 kDa); carbonic anhydrase (29.0 kDa); f-lactoglobulin (18.4         et al., 1984). To distinguish mutants specific- <br /> kDa); lysozyme (14.3 kDa); bovine trypsin inhibitor (6.2 kDa) and; <br /> insulinf-  a-chains (3.0 kDa).                                       ally affect maturation hybrid precursor that <br />                                                                          perturb transcriptional regulation a-factor <br /> Lys-SRIF- 14, accounted approximately                 promoter, mutant primary defect does not <br /> two-thirds processed peptides, is, however, sharp              directly involve monobasic-specific endoproteolytic <br /> contrast processing proSRIF-I1 chimera. The             activity, plasmids constructed. Plasmid <br /> was matured exclusively SRIF-28                 pGS3C derivative pS3C chimeric gene is <br /> was evidence futher processing Arg -2 Lys            -      control GAL] promoter paFF a <br /> cleavage site (Bourbonnais et al., 1991).                                truncated version MFal gene encodes poly- <br />                                                                          peptide 98 amino acids retained single copy <br /> Proteolytic processing proSRlF-I chimera                       mature a-factor peptide immediately preceded the <br /> Lys-SRIF- 14 requires functional KEX2 gene                             Lys Arg dibasic processing site. We reasoned mutant <br /> The release SRIF-28 proSRIF-Il chimera               strain capable secreting bioactive a-factor <br /> cleavage monobasic processing site (Arg - 15)                   carrying paFF, pS3C, potential <br /> previously shown occur absence functional                 candidate defect gene encoding monobasic- <br /> KEX2 gene (Bourbonnais et al., 1991). This consistent                 specific endoprotease. <br /> with known vitro specificity Kex2 endoprotease                 Control &apos;halo&apos; mating assays showed DS7 <br /> for paired basic amino acids (Julius et al., 1984). To test              transformants expressing pS3C paFF, those <br /> whether SRIF-14-like peptides secreted yeast             expressing pS2B, elicit zone inhibition (&apos;halo&apos;) <br /> carrying pSID released action Kex2,                  lawn supersensitive cells (Figure 4B) were <br /> a kex2 mutant yeast (strain DS20) transformed                   competent mating (Figure 4C). Strain DS7, carrying <br /> plasmid pSl D. A comparison labelled SRIF-immuno-                 pS3C, subjected UV irradiation (see Materials <br /> reactive material secreted wild type kex2 yeast                 methods). From independent experiments total <br /> strains SDS - Tricine gels (Figure 3, compare lanes 1             &gt; 9000 colonies survived UV exposure were <br /> 2) showed release SRIF- 14 SRIF-28              screened mating competence MATa cells. <br /> not affected absence functional Kex2 endo-                   From mutated colonies, 39 mating <br /> protease. However, peptide Lys-SRIF-14 completely                defective (Table I). Halo assays performed these <br /> absent culture medium kex2 transformant.                 mutants identified class mutant (16 colonies, class I) <br /> The results obtained pSiD expressed                   mating defect associated reduced <br /> in strain carrying disrupted KEX2 gene (strain                       production and/or secretion bioactive a-factor (Table I). <br /> M213-1OC) (not shown). This indicates                  From remaining 23 mutants, correlation between <br /> differences kex2-1 kex2 null alleles             mating secretion biologically active a-factor was <br /> the release SRIF-14 SRIF-28. The likely                      observed, distinguish classes. The class <br /> explanation results Kex2                     II mutants (15 colonies) unable produce zone of <br /> endoprotease involved cleavage proSRIF-I                 inhibition lawn supersensitive strain <br /> chimera Lys-SRIF- 14. It extends previous                     M200-6C size &apos;halo&apos; produced the <br /> observations proSRIF-II hybrid (Bourbonnais et al.,             class III mutants (eight colonies) significantly reduced. <br />  1991) demonstrates cleavage single                   Expression plasmid pGS3C representative members <br />                                                                                                                                   287 <br />   Y.Bourbonnais et al. <br />  <br />  <br />  <br />    A <br />           ff <br />           l <br />  <br />  <br />  <br />  <br />   C4--..=_= <br />  <br />   rffl-- - d <br />                1-_ . {;j &apos; <br />                f <br />  <br />  <br />  <br />  <br />                _.d <br />                       &lt;X ;- <br />  <br />  <br />  <br />  <br /> Fig. 4. Strategy used isolation somatostatin expression mutants. (A) Schematic representation plasmids pS2B, pS3C paFF: ( <br /> ca-factor preproregion; (?) N-linked glycosylation site; ( g ) length proSRIF-II; (0) cleavage site; (-) mature SRIF-28; (O) mature <br />  ax-factor. Note basic residues cleavage sites indicated single letter <span id='am-161' about='Thesaurus:code' typeof='owl:Thing'>code</span>. (B) Halo assay (see Materials methods) <br /> performed indicated DS7 transformants lawn indicator MATa strain M200-6C (sstl sst2). (C) Prototroph selection diploid <br /> yeast resulting cross indicated DS7 transformants MATa his] tester strain. <br />  <br /> of classes showed contrast class III mutants, <br /> mutants   pertaining class II elicit wild type &apos;halo&apos; <br /> when hybrid gene expressed control of <br /> the GAL] promoter. This strongly suggests class <br /> II mutants defective expression hybrid gene <br />                                                                                 mutants) <br />                                                                                  Phenotypic <br />                                                                                 class <br />                                                                                                       ...F&lt;. <br />  <br />  <br />  <br />  <br />                                                                                                  No. of <br />                                                                                                  mutants <br />                                                                                                                Mating <br />                                                                                                                            pS3C     pGS3C <br />                                                                                                                                                      ) <br />  <br />  <br />  <br />  <br />                                                                                 Table I. Isolation mutants defective somatostatin expression (sex <br />  <br />                                                                                                                            Production a-factora with <br />                                                                                                                                                    pcaFF <br /> when control ct-factor promoter. To test <br /> whether class .111 mutants retained ability                       Wild type       NA            +           +++      +++            +++ <br />                                                                                  Class I         16            -           +++      ND             ND <br /> secrete bioactive ct-factor flanked normal                    Class II        15            -           _        +++ (4)b       ND <br /> processing site (Lys Arg) precursor, plasmid pceFF                        Class III        8            -           +        + (2)          +++ (2) <br /> was introduced members class III. One trans-                                                                            + (1)          + (1) <br /> formant (B51) secrete low levels the <br /> pheromone regardless nature cleavage site (Lys                    aProduction mature as-factor estimated &apos;halo&apos; assay       a <br /> Arg pcxFF; single Arg pS3C), suggesting                         lawn supersensitive strain (M200-6C; Table III). <br /> <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>mutation</span> disrupted cellular function directly                        &quot;Number mutants phenotypic class transformed <br />                                                                                indicated plasmid assayed production halo on <br /> involved maturation monobasic processing site.                   M200-6C. <br /> Two strains (B34 B22), however, secreted wild type                      Absence mating halo indicated (-). The size halo <br /> near wild type levels bioactive o-factor carrying                      produced wild type strain indicated (+++ +); reduction <br /> paFF estimated size &apos;halo&apos; lawn                       25-50% diameter halo indicated (+ +); a <br /> supersensitive cells (strain M200-6C). We                          reduction 50-75% diameter halo indicated (+). <br />                                                                                 NA,     applicable; ND,      determined. <br /> conclude strains B34 B22 carry mutations affecting <br /> either directly (the gene encoding endoprotease itself)                    copies MFct genes. From initial 7 positive clones, <br /> or indirectly (e.g. gene involved proper localiza-                  5 shown mate efficiently secrete wild type levels <br /> tion function endoprotease) putative monobasic                   ca-factor absence plasmid pS3C. Hybridiza- <br /> endoprotease activity. Without characterization, strain                tion plasmid DNA carried clones a <br /> B34 (sex]-]) used recipient cell cloning                   radiolabelled probe derived patFF confirmed a <br /> the putative monobasic-specific endoprotease gene.                             functional copy MFct genes introduced into <br />                                                                                strains (data shown). In positive clones, <br /> Cloning characterization multicopy                                    expression plasmids required restoration <br /> suppressor sex 1-1     mutation                                         wild type pheromone production mating <br /> A yeast genomic library constructed multicopy plasmid                   competence. The YEP24-based plasmids carried these <br /> YEP24 used transform strain B34 carrying pS3C.                          strains rescued transformation Escherichia coli <br /> From total &gt; 10 000 Ura+ Trp+ transformants,                           designated p9A242 p8B234. Partial restriction maps <br /> isolated seven clones mating defect                    p9A242 (9.8 kb insert) p8B234 (14 kb insert) <br /> the ability produce &apos;halo&apos; lawn sensitive cells                 suggested share common SalI -XbaI DNA <br /> (M200-6C) corrected (not shown). Given                            fragment -2.7 kb (Figure 5). This confirmed by <br /> phenotype parental strain (MFcel :: LEU2                                hybridizing plasmids probe derived the <br /> MFoc2:: LEU2) sex]-] mutation did                            Sall -XbaI fragment p9A242 (not shown). Upon <br /> significantly affect proteolytic processing native                      subcloning common fragment pTZ 1 8R the <br /> ct-factor precursor, anticipated cloning functional                    generation precise restriction map, became <br /> 288 <br />                                                                                        S.cerevisiae mutants defective somatostatin expression <br />  <br />                                              M              G <br />                  G                                              M                    A                                                               C <br />            3&apos;                                                            5&apos; <br />                                                                                                  n                                     ,::-            DU <br />                                                                                                                                                        *      sm&quot;d repaired VT <br />                                                                                                  * aso <br />                                                                                                       e <br />                                                                                                                                               .C      U     lasmid repaired sexl <br />  <br /> p9a242 <br /> (9.8 Kb)                                         (B/Sa) <br />                 (B/Sa) <br />  <br />  <br />                                                                                       -[ <br />                       XNK NG S      XC   X       G NCK BB <br /> p8b234               .I   Ia        V        I      IN <br /> (14 Kb)                                                         (B/Sa) <br />            (B/Sa) <br />  <br />  <br />                      - 2 Kb <br />  <br /> Fig. 5. Partial restriction map plasmids p9A242 p8B234. A <br /> partial restriction map inserts carried plasmids p9A242         Fig. 6. The gene carried plasmids p9A242 p8B234 acts a <br /> p8B234 established following restriction enzymes: BamHI          multicopy suppressor sexl-1 mutation. Quantitative mating <br /> (B); BgllI (G); KpnI (K); MluI (M); Ncol (N); Sacl (C); Sall (S);         assays (see Materials methods) performed indicated <br />  <br /> XbaI (X). The solid bar restriction maps indicates              MATo! transformants MATa his] tester strain. The efficiency of <br />  <br /> position common XbaI-SalI fragment inserts. The           mating expressed percentage diploid cells formed the <br /> enlarged portion restriction map shows location                indicated transformant relative mating efficiency parental <br />  <br /> orientation coding region cloned gene (U).                      strain DS7 (set 100%). All assays                        performed       duplicate   and <br />  <br />                                                                               average        mating efficiency           reported   standard deviation. <br />                                                                               In   (A) averages             obtained            independent <br /> obvious gene cloned genetic complementa-                        determinations. (A) Wild type (WT; strain DS7),                            sexl-]   (sexi; strain <br /> tion sexl mutant remarkably similar yeast                 B34)         strain    carrying      disrupted    copy cloned gene <br />  <br /> gene recently identified Egel-Mitani et al. (1990)                     (YAP3:: HIS3, strain YBADl)                        transformed indicated <br />  <br /> called YAP3. The YAP3 gene (Yeast Aspartyl Protease 3)                        plasmids       challenged mate. (B) Wild type (WT; strain DS7) and <br />                                                                                              and <br />                                                                               sexl-J  (sexi; strain B34) yeast co-transformed pS3C the <br /> was cloned ability partially suppress                   indicated multicopy centromere-based plasmids carrying cloned <br /> kex2 mutation present multicopy plasmid.                            gene (p9A242 pBA09, respectively) (YEP24 YCp5O, <br />  <br /> However, normal function cell                         respectively) challenged mate. (C) Yeast carrying disrupted <br />  <br /> investigated. Partial DNA sequencing confirmed                        copy cloned gene                (YAP3::     HIS3;    strain    YBADI)        were <br />  <br />  <br />                                                                               co-transformed             pS3C     indicated           gap-repaired plasmids. <br /> genes identical (data shown).                                         Plasmids       pBA12-13 pBA14-15 (see Materials methods) <br />    Quantitative mating assays quite sensitive provide                 contain      Bglll Mlul deletion YAP3 gene respectively. <br />                                                                                                               a <br />  <br />  <br /> a good estimate relative secreted a-factor.                  They              assayed    relative      mating efficiency. <br />  <br /> Quantitative mating assays performed strain YBAD1 <br /> carrying disrupted copy cloned gene indicated                   6C). As     expected                   gene allelic SEX], <br />  <br /> the null mutant slightly competent mating                           mating efficiency                                regardless      host <br />  <br /> (167%) wild type (set 100%) sex]-]                  (i.e. wild type                   sex]-] mutant)      plasmids <br /> mutant (67%) strain expressing plasmid paFF (Figure                      containing             deletions          repaired. We conclude <br /> 6A). In contrast, mating efficiency null mutant                    YAP3 gene acts                      multicopy suppressor sex]-] <br /> expressing plasmid pS3C greater 10-fold lower                        mutation. <br />  <br /> (7.3%) wild type strain similar that <br /> obtained strain carrying sex]-] mutation (12.4%).                 Functional role YAP3 gene                                     prosomatostatin <br /> However, cloned gene shown restore near wild                       processing <br /> type mating competence strain B34 present                        Deletion    YAP3 gene, encodes aspartyl <br />  <br /> multicopy plasmid (YEP24), unable                       protease,     profound effect mating competence <br />                                                                                                           a <br />  <br />  <br /> expressed centromere-based plasmid (YCp5O) (Figure                     strain expressing pS3C. Its role proSRIF proteolytic <br />  <br /> 6B). This provided indication gene cloned                  maturation investigated metabolic labelling <br />  <br /> is allelic SEX], acts multicopy                        appropriate yeast strain transformed with <br />  <br /> suppressor sex]-] mutation. Because phenotype                      pS2B pSlID, followed specific immunoprecipita- <br />  <br /> of sex]-] mutation observed cells,                     tion secreted SRIF species. Identical amounts of <br />  <br /> complementation tests strain carrying disrupted copy                 radiolabelled material secreted wild type mutant <br />                                                                                                                Tricine gels.- Strain YBAD1I <br /> of YAP3 gene performed. To circumvent                        yeastwere loaded SDS <br />  <br />  <br /> this problem decided clone YAP3 locus                      (AYAP3) transformed pS2B secrete <br /> sex]-] mutant using &apos;gap repair&apos; technique (Orr-Weaver                    considerably mature SRIF-28 ( &lt;24 % estimated by <br /> et al., 1981) test ability correct mating                      densitometric analysis) wild type strain (Figure 7A, <br />  <br /> deficiency strain YBAD1 (AYAP3) expressing pS3C.                           compare lanes                       2). This supports           hypothesis that <br />  Centromere-based plasmids (YCp5O) containing                            YAP3 gene                  required           proteolytic processing at <br /> BglH MluI deletion, covered                    monobasic              cleavage sites.            In absence, however,                  clearly <br />  coding region YAP3 gene (see Figure 5), used                     alternativeendoproteolytic activities generate some <br />  transform parental original sex]-] mutant               mature peptide. Although Km, reaction con- <br />  <br />  <br />  yeast allow repair homologous recombination. The                       siderably higher (&gt; 10-fold) single arginine residues <br />  resulting plasmids rescued transformation                  Lys Arg- Arg Arg-containing substrates, <br />  <br />  E. coli introduced strain YBAD 1 (AYAP3) (Figure                    previous studies specificity Kex.2 endoprotease <br />  <br />                                                                                                                                                                               289 <br />   Y.Bourbonnais et al. <br />  <br />  <br />             A                      B <br />                                                                             RHRDLE]                                       ,C     proSRIF-I <br />  <br />                    1 2                    1 2 34 <br />  <br />                                                                            GRMNLE          SUDSTNNLPPRE               RGC        proSR IF-I I <br />                                                      AA <br />  <br />  <br />  <br />  <br />                  OWb <br />                 Eo                                                         Fig. 8. Model proteolytic processing anglerfish <br />                                                                            somatostatin precursors yeast S.cerevisiae. The amino acid <br />                                                                            sequence proSRIF-I -II, comprising processing sites <br />                                                                            (grey boxes), represented using single letter code. Filled and <br />                                                                            open arrows represent KEX2- YAP3-dependent cleavage, <br />                         C                                                  respectively. Larger arrows emphasize prevalence indicated <br />                                                                            gene proteolytic cleavage. <br />  <br />  <br />                               1 234                                         expected preproSRIF-II 63-125, previously <br />                                                                            described SRIF-28 processing intermediate release <br />                                                                             requires cleavage Lys Arg doublet (Noe et al., 1986). <br />                                                                            Any decrease production SRIF-28 yield a <br />                                                                            concomitant increase precursor forms. Accumulation <br />                                                                            secreted, hyperglycosylated proSRIF-II hybrid <br />                                                                            precursor easily detected SDS -Tricine gels. <br />                                                                            That absence Kex2 Yap3 leads the <br />                                                                            secretion unprocessed precursor forms best demon- <br />                                                                            strated subjecting immunoreactive material to <br />  Fig. 7. Biosynthesis prepro ca-factor-proSRIF-I -II            endoglycosidase H treatment (Figure 7B, lanes 3 4). The <br />  chimeras wild type, single (AYAP3) double (AYAP3 AKEX2) <br />  mutant yeast. Yeast labelled [35S]cysteine the <br />                                                                            deglycosylated precursor (16 kDa band) detected <br />  radioactive material specifically immunoprecipitated anti-       culture media wild type double <br />  SRIF-28 (A B) anti-SRIF-14 (C) antisera resolved          mutant yeast. However, relative abundance respect <br />  high resolution SDS-Tricine gel described Figure 3. (A)             mature SRIF-28 indicated minor species <br />  Secreted immunoreactive material wild type (strain DS7; lane 1)      secreted wild type strain predominant secreted <br />  LYAP3 (strain YBADI; lane 2) yeast transformed plasmid <br />  pS2B. (B) Secreted immunoreactive material wild type (strain         peptide AYAP3 AKEX2 mutant. <br />  DS7; lanes I 3) AYAP3AKEX2 (strain YBAD2; lanes 2                 As shown above, expression proSRIF-I yeast leads <br />  4) yeast transformed plasmid pS2B. Samples lanes 3 4          secretion major peptides (SRIF-28, Lys- <br />  incubated endoglycosidase H prior electrophoresis.           SRIF-14 SRIF-14) genetic evidence suggested Kex2 <br />  (C) Secreted immunoreactive material wild type (strains DS7      directly involved release Lys-SRIF-14 through <br />  W303-lb; lanes 1 3, respectively), AYAP3 (strain YBADI; lane 2) <br />  AYAP3 AKEX2 (strain YBAD2; lane 4) yeast transformed             cleavage middle pair basic residues (Arg Lys). <br />  plasmid pSID. Note identical radioactivity          Consistent finding, expression pSlD the <br />  loaded pair lanes. Asterisks indicate migration position   absence functional YAP3 gene did affect conversion <br />  major glycosylated precursor form chimeras. Upward <br />  pointing arrowheads indicate position deglycosylated <br />                                                                            precursor Lys-SRIF-14 (Figure 7C, lanes 1 and <br />  proSRIF-II fusion protein (16 kDa). Arrowheads left indicate       2). However, nearly completely prevented accumulation <br />  position prestained protein standards described Figure   SRIF-28 significantly reduced secreted <br />  3.                                                                        SRIF-14. This accompanied appearance a <br />                                                                            diffuse band molecular mass (45 kDa) consistent <br /> indicated enzyme recognizes cleaves,                hyperglycosylation, contributes 30-50 kDa per <br /> in vitro, single arginine residue (Brenner Fuller,               chain (Julius et al., 1984) proSRIF-I hybrid precursor <br /> 1992). To test idea Kex2 could, absence                 (17 kDa). That mature SRIF-14 secreted kex2 <br /> YAP3, excise mature SRIF-28 hybrid precursor                  (Figure 3, lane 2) AYAP3 mutant yeast <br /> double mutant strain YBAD2 (AYAP3 AKEX2)                               (Figure 7C, lane 2) suggested gene products are <br /> constructed. Expression plasmid pS2B strain                     involved release peptide. Expression pS ID <br /> resulted decrease (&lt;105% wild type                     double mutant strain YBAD2 confirmed this <br /> production SRIF-28) proSRIF-I1 conversion                         hypothesis (Figure 7C, lanes 3 4). In mutant yeast <br /> SRIF-28. It totally prevented appearance 6 kDa               proteolytic maturation fusion protein was <br /> peptide, consistently observed culture                    completely abolished intact hyperglycosylated <br /> medium wild type A YAP3 yeast. Although nature                  precursor forms (45 kDa peptide) recovered from <br /> remains confirmed, mobility SDS -Tricine gels                 culture medium. <br />  <br />  290 <br />                                                                            S.cerevisiae mutants defective somatostatin expression <br />  <br />  Discussion                                                          corresponding SEX genes undoubtedly provide <br />  Our main objective define molecular basis the <br />                                                                     new    insights yeast cellular machinery involved in <br />                                                                      precursor processing. This work currently progress. <br />  differential processing anglerfish SRIF-I -II            Similarly, rapid sensitive biological assay used in <br />  precursors using yeast S. cerevisiae model system.         isolation sex mutants, combined &apos;cassette&apos;- <br />  As step goal recently expressed              directed mutagenesis, provide powerful tool for <br /> proSRIF-II yeast, fusion protein, identified           investigating role amino acid residues upstream <br /> novel endoproteolytic activity capable generating vivo         processing sites differential maturation of <br /> mature SRIF-28 (Bourbonnais et al., 1991). In present           anglerfish SRIF precursors. This particular interest <br /> study wanted investigate biosynthesis              despite possessing mono- dibasic cleavage <br /> analogous anglerfish somatostatin precursor (SRIF-I)             sites appear differentially processed identical <br /> characterize gene encoding novel yeast endoprotease.        cellular environment (see below). <br /> The recent cloning mammalian dibasic processing                  Expression proSRIF-I -11 fusion proteins single <br /> enzymes PC2, PC1/PC3 PC4 (Seidah et al., 1990;                  double mutant yeast enabled propose model for <br /> Smeekens Steiner, 1990; Smeekens et al., 1991;                  role KEX2 YAP3 genes proteolytic <br /> Nakayama et al., 1992) based sequence similarity           maturation somatostatin precursors yeast (Figure 8). <br /> with yeast Kex2 enzyme. A key finding cloning            In hybrid precursors efficient cleavage single <br /> of mammalian Kex2 homologues observation              arginine residue (Arg-15) requires functional YAP3gene. <br /> that Kex2 endoprotease, required               Although absence dramatic production <br /> maturation yeast a-factor (Julius et al., 1984),       SRIF-28 proSRIF-I, clearly enzymatic <br /> faithfully recognize cleave analogous dibasic processing        activities could, partially, process SRIF precursors <br /> sites (i.e. Lys Arg Arg Arg) heterologous precursors         generate peptide. Consistent previous vitro <br /> (Thomas et al., 1988; Germain et al., 1990; Zollinger et al.,       itt vivo studies substrate specificity Kex2 <br /> 1990). The cloning mammalian monobasic processing            endoprotease (Brenner Fuller, 1992; Zhu et al., 1992), <br /> enzyme benefit isolation gene       results implicate KEX2 gene alternative, less <br />  cognate enzyme yeast.                                 efficient processing. As reported pancreatic islets <br />     We shown expression anglerfish proSRIF-I           (Noe Spiess, 1983), proteolytic maturation proSRIF-IH <br />  yeast leads secretion mature SRIF-28, mature          yeast restricted single arginine residue (this study; <br />  SRIF-14 extended form SRIF-14 (Lys-SRIF-14).             Bourbonnais et al., 1991). In contrast, proSRIF-I further <br />  Accumulation SRIF-28 culture medium indicates            processed, presumably Arg -2 Lys -1 site, generate <br />  processing yeast internal Arg -2 Lys -1 basic pair       Lys-SRIF-14 mature SRIF-14. Since basic pair is <br />  does proceed completion. This contrast             present precursors supports hypothesis <br />  synthesis prohormone pancreatic islets,        (Bourbonnais et al., 1991) lack proSRIF-H cleavage <br />  exclusively yield mature SRIF-14. In higher eukaryotes,            Arg -2 Lys -1 results specific conformation <br />  however, secretion peptide hormones neuro-              rendering site inaccessible endoproteolytic processing. <br />  peptides regulated, yeast rapidly constitu-         The dibasic-specific Kex2 endoprotease essential the <br />  tively secretes proteins destined extracellular        release Lys-SRIF-14, suggesting endoproteolysis <br />  medium. The limited contact time pro-             occurs basic doublet. In favor idea, <br />  hormone convertase substrate yeast        previous studies specificity purified Kex2 <br /> account apparent discrepancy observed yeast         endoprotease peptidyl-MCA substrates revealed the <br /> and pancreatic islets. Expression proSRIF-I               P1 binding site enzyme highly specific Arg <br /> a kex2 yeast mutant extended previous observation               Arg P1 position, substitution P2 <br /> (Bourbonnais et al., 1991) provided genetic evidence            little effect overall catalytic rate (Brenner and <br /> that SRIF precursors processed mature SRIF-28           Fuller, 1992). However, additional experiments required <br /> by Kex2-independent cleavage. Genetic complementation             confirm hypothesis (see below). The Kex2 enzyme <br /> of yeast mutant (sexl-1), specifically affects cleavage     involved directly (i.e. cleavage Arg <br /> at monobasic processing site (Arg -15), resulted         Lys doublet)     indirectly (i.e.     limiting aminopeptidase <br /> cloning gene encoding aspartyl protease. This gene,         B-like activity remove extra lysine residue the <br /> YAP3, recently cloned group               Lys-SRIF-14 processing intermediate) release of <br /> (Egel-Mitani et al., 1990) based ability partially        mature SRIF-14. In support hypothesis, amino- <br /> suppress phenotype kex2 null mutant, shown             peptidase B-like enzyme purified rat brain cortex was <br /> to act multicopy suppressor sex]-] mutation. It         recently proposed required conversion of <br /> is unclear point primary defect sex]        processing intermediates mature SRIF-14 (Gluschankof <br /> mutant is. Because null phenotype YAP3 gene           et al., 1987; Gomez et al., 1988). Finally data showed <br /> (see below) is, however, unlikely Sexl directly          YAP3 gene, predominantly involved in <br /> involved proteolytic maturation proSRIFs. Most            processing monobasic cleavage site (Arg - 15), also <br /> probably, involved function (i.e. proteolytic   contributes extent release mature SRIF- 14. <br /> activation zymogen) intracellular localization       This consistent previous study proteolytic <br /> the YAP3 gene product. Hence, screening                maturation overexpressed pro a-factor-insulin <br /> procedure primarily designed facilitate cloning       chimera absence functional KEX2 gene <br /> gene encoding putative monobasic-specific endoprotease,         (Egel-Mitani et al., 1990). Under conditions 20% <br /> further characterization class III mutants cloning       proteolytic processing, ascribed action Yap3, <br />  <br />                                                                                                                                  291 <br />  Y.Bourbonnais et al. <br />  <br /> occurred pro a-factor Lys Arg cleavage site. Hence,                   argued proteolytic maturation occurs intracellularly. <br /> proteolytic maturation anglerfish SRIF precursors yeast                The presence mature SRIF peptides cell extracts <br /> implicates endoproteases distinct specificities                 prepared spheroplasts kex2-transformed yeast <br /> the mono- dibasic processing sites. The present model                    confirmed Yap3-dependent processing occurs the <br /> of proSRIF processing yeast based largely genetic                   secretory pathway periplasmic space (not <br /> arguments. The actual sites endoproteolytic cleavage need                 shown). In addition, previous observation the <br /> therefore confirmed vitro incubation SRIF                 processing proSRIF-II completely blocked when <br /> precursors purified endoproteases. For instance,                expressed sec,18 mutant grown restrictive <br /> data exclude possibility Lys-SRIF-14                     temperature, indicated proteolytic maturation <br /> peptide generated Kex2 endoproteolytic                         post-ER event (Bourbonnais et al., 1991). These results <br /> cleavage C-terminal Arg-4 (i.e. Pro-5 Arg-4                     good keeping a-factor maturation by <br /> putative monobasic site), instead Arg -2, followed                  Kex2 endoprotease which, probably, takes place <br /> by exoproteolytic cleavage aminopeptidase. This                  late-Golgi compartment (Julius et al., 1984; Redding <br /> be supported present finding implicating Kex2                  et al., 1991). Consistent physiological relevance <br /> release mature SRIF-28, recent                     Kex2 Yap3 proSRIF maturation, groups have <br /> report Kex2-dependent processing yeast K1                      documented action distinct somatostatin convertases <br /> killer toxin Pro43 Arg44 site (Zhu et al., 1992).                     production SRIF-28 SRIF-14, respectively, in <br />    Although precise site action YAP3 gene                      higher eukaryotes (Gluschankof et al., 1987; Mackin and <br /> product determined, preliminary studies                 Noe, 1987; Beinfeld et al., 1989). More recently Mackin <br /> localization Yap3-dependent cleavage proSRIFs                      et al. (199 la, 1991b) purified apparent homogeneity two <br /> support role precursor processing. Namely,                        candidate SRIF convertases anglerfish secretory <br /> accumulation significant various SRIF                       granules capable generating SRIF-28 -14 peptides <br /> peptides cell extracts pSlD- (this study) pS2B-                    respectively, vitro physiological precursor <br /> transformants (Bourbonnais et al., 1991) virtual                     (i.e. proSRIF-II -I). Partial N-terminal sequencing of <br /> absence precursor forms culture media strongly                     purified proteins revealed candidate SRIF-14 con- <br />  <br /> Table II. Plasmids used work <br />  Name             Description                                                                                        Reference <br />  pS2B             A 2 itm based vector derived p82-5 (Green et al., 1986) expressing prepro                 Bourbonnais   et   al. (1991) <br />                   a-factor-proSRIF-II chimeric gene control a-factor promoter <br /> pSID              Same pS2B, expressing prepro a-factor-proSRIF-I chimeric gene                           This study <br /> pS3C              Same pS2B, expressing prepro a-factor-proSRIF-II-mature a-factor chimeric gene          This study <br /> pGS3C             A centromere-based vector derived pJK6 (kindly provided J.Kurjan) prepro         This study <br />                   a-factor-proSRIF-II-mature a-factor chimeric gene control GALI promoter <br /> paFF              Same pS2B, expressing truncated version prepro a-factor single copy    This study <br />                   mature peptide retained <br /> YEP24G- <br /> B                 A yeast genomic library constructed plasmid YEP24                                               Rose Broach (1991) <br /> pBA09             Plasmid YCp5O 5.8 kb XbaI (filled-in) fragment p9A242 cloned       This study <br />                   Sall (filled-in) site <br /> pBA12             Derived YCp5O carrying 6.0 kb Sall p9A242, plasmid contained BglIl deletion     This study <br />                   (3.8 kb) removed seven codons Yap3 protease -2.1 kb 3&apos; <br />                   flanking nucleotides repaired vivo homologous recombination parental <br />                   strain DS7 <br /> pBA 13            Same pBA 12, repair plasmid mutant strain B34                     This study <br /> pBA14             Derived pBA09, plasmid contained MluI deletion (1.8 kb) removed 0.4 kb 5&apos;      This study <br />                   flanking sequence 459 codons (out 569) YAP3 gene, repaired <br />                   vivo homologous recombination parental strain DS7 <br /> pBA15             Same pBA14, repair plasmid mutant strain B34                      This study <br />  <br />  <br /> Table III. Yeast strains <br /> Name                               Genotype                                                                          Source reference <br /> 20-B12                             MATa pep4-3 trpl                                                                  Green et al. (1986) <br /> DS20                               MATa kex2-1 trpl ural                                                             Bourbonnais et al. (1991) <br /> DS7                                MATa MFaJ/a2 ::LEU2 ade2 his3 leu2 trpl ura3                                      J.Kurjan <br /> M200-6C                            MATa sstl sst2 adel ilv3 ura3                                                     Whiteway et al. (1988) <br /> B34                                MATa sexl-l MFaJ/a2 ::LEU2 ade2 his3 leu2 trpl ura3                               This study <br /> YBAD1                              MATa YAP3::HIS3 MFaJ/a2 ::LEU2 ade2 his3 leu2 trpl ura3                           This study <br /> M213-IOC                           MATa KEX2::HIS3 his3 leu2 trpl ura3                                               Dignard et al. (1991) <br /> W303-lb                            MATa ade2 his3 leu2 trpl ura3                                                     S.Henry <br /> YBAD2                              MATa YAP3::LEU2 KEX2::HIS3 his3 leu2 trpl ura3                                    This study <br />  <br /> 292 <br />                                                                                                S.cerevisiae mutants defective somatostatin expression <br />  <br /> vertase homologous human mouse PC2,                                        Gene disruption <br /> recently identified member Kex2-like mammalian                                   To disrupt yeast YAP3 locus YAP3 SphI-XbaI fragment first <br /> homologues (Seidah et al., 1990; Smeekens Steiner,                                  subcloned plasmid pTZ18R. Most YAP3 coding region then <br />                                                                                         deleted (544/569 467/569 codons, respectively) replaced the <br /> 1990). Both inhibition profile partial N-terminal                           HIS3 gene (1.8 kb BarnHI fragment plasmid pJJ215; Jones Prakash, <br /> sequencing protein indicated putative SRIF-28                                1990) LEU2 gene (2.0 kb BamHI-HindHI fragment pJJ283; <br /> convertase belongs aspartyl protease family. It                               Jones Prakash, 1990). One step disruptions (Rothstein, 1983) the <br /> therefore tempting speculate YAP3 gene                               HIS3 marker obtained use SphI-BamHI fragment <br />                                                                                         transformation yeast strains. The disruption LEU2 marker <br /> yeast functional homologue endoprotease. However,                               obtained using HindIII-BamHI fragment. Each disruption was <br /> it possible functionally related higher                           confirmed Southern analysis. <br /> cell convertase involved cleavage monobasic sites, the <br /> Yap3 endoprotease closely structurally related. This                             Metabolic labelling yeast, immunoprecipitation protein <br /> would analogous carboxypeptidase B-like activity                              analysis <br />                                                                                         Exponentially growing yeast labelled 250 ItCi [35S]cysteine <br /> required processing transient intermediates generated                            (Amersham) 30 min incubation 30&#194;&#176;C described previously <br /> by endoproteolysis pair basic residues. Hence,                             Bourbonnais et al. (1988, 1991). Cell extracts media prepared as <br /> while carboxypeptidases thought fulfil                             described Bourbonnais et al. (1988, 1991) subjected to <br /> function cell, sequence yeast KEX1 gene                               immunoprecipitation anti-SRIF-28 (R33 B5) anti-SRIF-14 (RSSI <br />                                                                                         B23) antibodies indicated. The antisera generous gifts Drs Shields <br /> product (Dmochowska et al., 1987) does resemble                                Danoff (Albert Einstein College Medicine, Bronx, NY). Produc- <br /> of mammalian carboxypeptidase E (Fricker, 1988). Future                                 tion specificity antibodies described previously by <br /> experiments addressing physiological function                              Bourbonnais et al. (1991) Danoff Shields (1991). Anti-SRIF-28 <br /> endoproteases, information complete amJno acid                               specifically recognizes SRIF-28 precursor form, does cross- <br /> sequence anglerfish aspartyl protease, need-                            react SRIF-14 proSRIF-I. Anti-SRIF-14 efficiently <br />                                                                                         immunoprecipitates SRIF-14 precursor; recognizes SRIF-28 <br /> ed clarify exact relationship enzymes.                           proSRIF-II, albeit lower affinity. Radiolabelled proteins were <br /> As step goal currently investigating                         analyzed high resolution Tricine-SDS gels reversed-phase HPLC <br /> the nature Yap3 substrate(s) yeast.                                           using C18 column (Vydac) described previously Bourbonnais et al. <br />                                                                                         (1991). Samples eluted column 0.1% trifluoroacetic <br />                                                                                         acid containing acetonitrile flow rate 1.5 ril/min according the <br /> Materials methods                                                                   following gradient: 5% acetonitrile 5 min; 5-20% 1 min; <br />                                                                                         20-35% 30 min; 35-50% 15 min; 50-80% 1 min; isocratic- <br /> Plasmids strains                                                                    ally 80% 13 min. <br /> Plasmids used study brief description genes carry <br /> are listed Table II. All DNA modifications subcloning performed             Partial amino acid sequencing <br /> according standard procedures. Site-directed mutagenesis performed               Peptides sequenced purified HPLC submitted directly <br /> on single-stranded DNA isolated dut-ung- bacteria using single                 Edman degradation gas phase sequenator (Applied Biosystem model <br /> primer annealing technique according procedure developed Kunkel               470A) described earlier Bourbonnais et al. (1991). <br /> (1985). Restriction endonucleases New England Biolabs T4 <br /> DNA polymerase Pharmacia LKB Biotechnology Inc. Yeast strains                  Endoglycosidase H treatment <br /> used study listed Table Ill. Yeast transformation carried            Treatment immunoprecipitates endoglycosidase H carried <br /> out using lithium acetate procedure described Ito et al.(1983).                  described previously Bourbonnais et al. (1991). <br /> Yeast mutagenesis <br /> Strain DS7, expresses plasmid pS3C, grown overnight plated                Acknowledgements <br /> on selective medium. The dishes immediately exposed short- <br /> wave UV light 35 s, wrapped aluminum foil placed 22&#194;&#176;C                 We grateful Susan Henry, Janet Kurjan Malcolm Whiteway who <br /> 3 days allow formation colonies. Control plates indicated treatment          kindly provided yeast strains, Mark Rose generous gift yeast <br /> resulted 90% killing. Colonies survived UV exposure               genomic library Dennis Shields Ann Danoff supplied with <br /> screened production bioactive a-factor described below.                       SRIF antibodies; France Dumas amino acid sequencing the <br />                                                                                         labelled peptides Caroline Kunze precious technical assistance <br /> Halo quantitative mating assays                                                     early work; Malcolm Whiteway Thierry Vernet for <br /> Halo assays performed strain M200-6C, carries mutant                critical reading manuscript. This NRCC publication no. 33655. <br /> alleles SSTJ SS72 genes supersensitive a-factor. <br /> Following overnight incubation 30&#194;&#176;C YPD medium, 5 ll of <br /> M200-6C mixed 10 ml YPD medium containing 0.7 % agar,                      References <br /> which prewarmed 42&apos;C, overlaid YPD plates. Colonies <br /> to tested spotted directly solidified agar            Beinfeld,M.C., Bourdais,J., Kuks,P., Morel.A. Cohen,P. (1989) J. <br /> plates incubated overnight 30&apos;C. For mating assay, haploid                    Biol. Chem., 264, 4460-4465. <br /> yeast grown overnight YPD medium (MATa tester strain)                  Bourbonnais,Y., Bolin,D. Shields,D. (1988) J. Biol. Chem., 263, <br /> the appropriate SC medium (AATa strains). The day cells diluted                15342-15347. <br /> 1/10 SC medium incubation resumed additional 2 h                  Bourbonnais,Y., Danoff,A., Thomas,D.Y. Shields,D. (1991) J. Biol. <br /> at 30&apos;C. The MATa strains mixed tester strain (MATa)                   Chem., 266, 13203 - 13209. <br /> in following ratio: 100 Al diluted MATa cells, 50 ,l MATa             Brenner,C. Fuller,R.S. (1992) Proc. Natl. Acad. Sci. USA, 89, <br /> cells (undiluted overnight culture) 100 A1 YPD medium. The mixture                 922-926. <br /> was gently filtered sterile nitrocellulose filters (Sartorius), filters   Danoff,A. Shields,D. (1991) J. Biol. Chem., 266, 10004-10010. <br /> were put, colonies up, YPD plates incubated                 Devi,L. (1991) FEBS Lett., 280, 189-194. <br /> 30&apos;C 4-6 h. Following incubation cells gently washed                   Dignard,D., Whiteway,M., Germain,D., Tessier,D. Thomas,D.Y. <br /> the filters 300 Al selective medium appropriate dilutions                (1991) Mol. Gen. Genet., 227, 127-136. <br /> plated SC plates allowing diploid cells grow prototroph selection.           Dmochowska,A., Dignard,D., Henning,D., Thomas,D.Y. Bussey,H. <br /> Parallel nitrocellulose filters containing YPD medium cells                 (1987) Cell, 50, 573-584. <br /> were plated YPD plates following appropriate dilutions. The percentage             Douglass,J., Civelli,O. Herbert,E. (1984) Annu. Rev. Biochem., 263, <br /> of mating determined according following equation: (No.                      665-715. <br /> colonies SC plate/No. colonies YPD plate) x 100. Alternatively,                 Egel-Mitani,M., Flygenring,H.P. Hansen,M.T. (1990) Yeast, 6, <br /> semi-quantitative mating assay performed directly replicating                   127-137. <br /> nitrocellulose filters SC plates.                                                  Fricker,L.D. (1988) Annu. Rev. Physiol., 50, 309-321. <br />                                                                                                                                                                293 <br />  Y.Bourbonnais et al. <br />  <br /> Germain,D. Zollinger,L., Racine,C., Gossard,F., Dignard,D., <br />    Thomas,D.Y., Crine,P. Boileau,G. (1990) Mol. Endocrinol., 4, <br />     1572-1579. <br /> Gluschankof,P., Gomez,S., Morel,A. Cohen,P. (1987) J. Biol. Chem., <br />    262, 9615-9620. <br /> Gomez,S., Gluschankof,P., Lepage,A. Cohen,P. (1988) Proc. Natl. <br />    Acad. Sci. USA, 85, 5468-5472. <br /> Green,R., Schaber,M.D., Shields,D. Kramer,R. (1986) J. Biol. Chem., <br />    261, 7558-7565. <br /> Ito,H., Fukada,Y., Murata,K. Kimura,A. (1983) J. Bacteriol., 153, <br />     163-168. <br /> Jones,J.S. Prakash,L. (1990) Yeast, 6, 363-366. <br /> Julius,D., Brake,A., Blair,L., Kunisawa,R. Thomer,J. (1984) Cell, <br />    37, 1075-1089. <br /> Kunkel,T.A. (1985) Proc. Natl. Acad. Sci. USA, 82, 488-492. <br /> Mackin,R.B. Noe,B.D. (1987) J. Biol. Chem., 262, 6453-6456. <br /> Mackin,R.B., Noe,B.D. Spiess,J. (1991a) Endocrinology, 129, <br />    1951-1957. <br /> Mackin,R.B., Noe,B.D. Spiess,J. (1991b) Endocrinology, 129, <br />    2263-2265. <br /> McDonald,J.K., Greiner,F., Bauer,G.E., Elde,R.P. Noe,B.D. (1987) <br />    J. Histochem. Cytochem., 35, 155-162. <br /> Nakayama,K., Kim,W.-S., Torii,S., Hosaka,M., Nakagawa,T., Ikemizu,J., <br />    Baba,T. Murakami,K. (1992) J. Biol. Chem., 267, 5897-5900. <br /> Noe,B.D. Spiess,J. (1983) J. Biol. Chem., 258, 1121-1128. <br /> Noe,B.D., Andrews,P.C., Dixon,J.E. Spiess,J. (1986) J. Cell Biol., <br />    103, 1205-1211. <br /> Orr-Weaver,T.C., Szostak,J. Rothstein,R.J. (1981) Proc. Natl. Acad. <br />    Sci. USA, 78, 6354-6358. <br /> Patel,Y.C., Wheatley,T. Ning,C. (1981) Endocrinology, 109, <br />    1943-1949. <br /> Redding,K., Holcomb,C. Fuller,R.S. (1991) J. Cell Biol., 113, <br />    527-538. <br /> Rose,M.D. Broach,J.R. (1991) Methods Enzymol., 194, 195-230. <br /> Rothstein,R.J. (1983) Methods Enzymol., 101, 202-211. <br /> Schwartz,T.W. (1986) FEBS Lett., 200, 1-10. <br /> Seidah,N.G., Gaspar,L., Mion,P., Marcinkiewicz,M., Mbikay,M. and <br />    Chretien,M. (1990) DNA Cell Biol., 9, 415-424. <br /> Smeekens,S.P. Steiner,D.F. (1990) J. Biol. Chem., 265, 2997-3000. <br /> Smeeke <br /> </body></html>