<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Inhibition Plasmepsin V Activity Demonstrates Its <br /> Essential Role Protein Export, PfEMP1 Display, and <br /> <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>Survival</span> Malaria Parasites <br /> Brad E. Sleebs1,2, Sash Lopaticki1, Danushka S. Marapana1,2, Matthew T. O&#226;&#8364;&#8482;Neill1, Pravin Rajasekaran1,2, <br /> Michelle Gazdik1,2, Svenja Gu&#194;&#168;nther1, Lachlan W. Whitehead1,2, Kym N. Lowes1,2, Lea Barfod3, Lars Hviid3, <br /> Philip J. Shaw4, Anthony N. Hodder1,2, Brian J. Smith5, Alan F. Cowman1,2, Justin A. Boddey1,2* <br /> 1 The Walter Eliza Hall Institute Medical Research, Parkville, Victoria, Australia, 2 Department Medical Biology, The University Melbourne, Parkville, Victoria, <br /> Australia, 3 University Copenhagen Copenhagen University Hospital (Rigshospitalet), Copenhagen, Denmark, 4 National Center Genetic Engineering and <br /> Biotechnology (BIOTEC), Pathum Thani, Thailand, 5 Department Chemistry, La Trobe University, Victoria, Australia <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      The malaria parasite Plasmodium falciparum exports proteins infected erythrocyte involved <br />      cellular remodeling severe virulence. The export mechanism involves Plasmodium export element (PEXEL), which <br />      cleavage site parasite protease, Plasmepsin V (PMV). The PMV gene refractory deletion, suggesting is <br />      essential, definitive proof lacking. Here, generated PEXEL-mimetic inhibitor potently blocks activity of <br />      PMV isolated P. falciparum Plasmodium vivax. Assessment PMV activity P. falciparum revealed PEXEL cleavage <br />      occurs cotranslationaly, similar signal peptidase. Treatment P. falciparum&#226;&#8364;&#8220;infected erythrocytes inhibitor <br />      caused dose-dependent inhibition PEXEL processing protein export, including impaired display major <br />      virulence adhesin, PfEMP1, erythrocyte surface, cytoadherence. The inhibitor killed parasites trophozoite <br />      stage knockdown PMV enhanced sensitivity inhibitor, overexpression PMV increased resistance. This <br />      provides direct evidence PMV activity essential protein export Plasmodium spp. parasite survival <br />      human erythrocytes validates PMV antimalarial drug target. <br />  <br />   Citation: Sleebs BE, Lopaticki S, Marapana DS, O&#226;&#8364;&#8482;Neill MT, Rajasekaran P, et al. (2014) Inhibition Plasmepsin V Activity Demonstrates Its Essential Role Protein <br />   Export, PfEMP1 Display, Survival Malaria Parasites. PLoS Biol 12(7): e1001897. doi:10.1371/journal.pbio.1001897 <br />   Academic Editor: Boris Striepen, University Georgia, United States America <br />   Received January 29, 2014; Accepted May 22, 2014; Published July 1, 2014 <br />   Copyright: &#195;&#376; 2014 Sleebs et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: This work supported NHMRC Project Grant (1010326 JAB), Human Frontiers Science Program (RGY0073/2012 JAB), Ramaciotti <br />   Foundation Establishment Grant (3197/2010 JAB), CASS Foundation Science Medicine grant (SM.12.4348 JAB), Australian Cancer Research <br />   Foundation, Victorian State Government Operational Infrastructure Support Australian Government NHMRC IRIISS. PJS acknowledges CPM/NSTDA <br />   grant P-11-00673. AFC Howard Hughes International Scholar JAB QEII Fellow Australian Research Council. The funders role study <br />   design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   Abbreviations: DAPI, 49-6-Diamidino-2-phenylindole; ER, endoplasmic reticulum; GFP, green fluorescent protein; GlcN, glucosamine; HA, hemagglutinin; KAHRP, <br />   knob-associated histidine-rich protein; MCs, Maurer&#226;&#8364;&#8482;s clefts; PEXEL, Plasmodium export element; PfEMP1, P. falciparum erythrocyte membrane protein 1; PMIX, <br />   Plasmepsin IX; PMV, Plasmepsin V; PvPMV, P. vivax PMV; SBP1, skeleton binding protein 1; Sta, statine; VTS, vacuolar transport signal. <br />   * Email: boddey@wehi.edu.au <br />  <br />  <br />  <br /> Introduction                                                                             PEXEL-containing proteins PMV conserved all <br />                                                                                          Plasmodium spp. [8,14&#226;&#8364;&#8220;16]. Repeated attempts disrupt PMV <br />    Each year malaria parasites cause million                             gene failed, suggesting essential [12,13,16], direct and <br /> infections 650,000 deaths [1]. Plasmodium falciparum causes                     decisive proof lacking. A functional survey P. falciparum <br /> the lethal malaria endemic Africa [2]. Plasmodium                         exported proteins indicated 25% essential for <br /> vivax causes malarial deaths outside Africa associated                       parasite survival human erythrocytes [17]. The current P. <br /> with liver-stage hypnozoites [3]. Although chloroquine                               falciparum PEXEL exportome predicted 463 proteins [18]; <br /> artemisinin effective antimalarials, decreasing                          thus, possibly 100 exported parasite proteins required <br /> efficacy [4,5] emphasizes need therapies novel                           development erythrocytes. <br /> targets shared Plasmodium spp.                                                      Some exported proteins lack PEXEL, example, skeleton <br />    Malaria parasites develop erythrocytes parasitophor-                      binding protein 1 (SBP1) major virulence adhesin family <br /> ous vacuole export 450 proteins cell (reviewed                        known P. falciparum erythrocyte membrane protein 1 (PfEMP1). <br /> [6,7]). Export utilizes N-terminal motif called Plasmodium                        PfEMP1 expressed erythrocyte surface mediates <br /> export element (PEXEL; RxLxE/Q/D) [8] Vacuolar transport                              cytoadherence microvascular endothelia, causing severe malaria <br /> signal (VTS) [9]. Exported proteins cleaved PEXEL                       [19]. PfEMP1 thought cleaved PMV [18], its <br /> Leu (RxLQ) endoplasmic reticulum (ER) [10],                                 transport <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span>, expression on, erythrocyte surface requires <br /> requires conserved Arg Leu residues [11]. PEXEL cleavage                         exported PEXEL PEXEL-negative proteins (reviewed in <br /> is performed aspartyl protease Plasmepsin V (PMV) [12,13].                        [7,20]). <br />  <br />  <br /> PLOS Biology | www.plosbiology.org                                                   1                                July 2014 | Volume 12 | Issue 7 | e1001897 <br />                                                                                                        Inhibition Plasmepsin V P. falciparum <br />  <br />  <br />  <br />   Author Summary                                                              To investigate PvPMVHA activity, affinity purified it, well <br />                                                                            PfPMVHA control, using anti-HA agarose, previously <br />   To survive human red blood cells, malaria parasites               described [13,18]. Immunoblot anti-HA anti-PfPMV <br />   export catalog proteins remodel host                  antibodies showed purified proteins species-specific <br />   cell surface. This enables parasites acquire                  (Figure S2A). The proteins incubated fluorogenic <br />   nutrients outside cell modify cell                   peptide amino acids contained PEXEL sequence <br />   surface order evade host defenses. Protein export                  knob-associated histidine-rich protein (KAHRP), and <br />   involves proteolytic cleavage Plasmodium Export                   efficient processing observed enzymes (Figure 1E). <br />   Element (PEXEL) aspartyl protease Plasmepsin V. We                Cleavage KAHRP PfPMVHA previously shown to <br />   report development small molecule inhibitor                occur Leu (RTLQ) mass spectrometry [13], this <br />   closely mimics natural PEXEL substrate blocks               position confirmed PvPMVHA (Figure S2B&#226;&#8364;&#8220;E). Km <br />   activity Plasmepsin V malarial parasites                 values 9.7 (63.0) 11.7 (61.8) mM calculated for <br />   Plasmodium falciparum Plasmodium vivax. The inhib- <br />                                                                            PvPMVHA PfPMVHA, respectively (Figure S2F). In contrast, <br />   itor impairs export cellular remodeling kills P. <br />   falciparum ring-trophozoite transition, providing                 processing observed PEXEL Arg Leu <br />   direct evidence Plasmepsin V activity essential              residues mutated Ala (ATAAQ), consistent the <br />   export PEXEL proteins parasite survival                substrate specificity PfPMV (Figure 1E). To verify that <br />   host. These findings validate Plasmepsin V highly                   processing HA-tagged PMV co- <br />   conserved antimalarial drug target.                                      precipitated proteases, expressed P. falciparum mutant of <br />                                                                            PvPMVHA PfPMVHA [13], catalytic Asp <br />                                                                            residues mutated Ala (see Figure 1A,B). Following affinity <br />    Aspartyl proteases inhibited transition-state isosteres       purification, mutant enzymes incubated KAHRP <br /> in scissile bond replaced noncleavable moiety.           PEXEL peptides, processing observed (Figure 1E; blue, <br /> Examples include statine (Sta)-containing inhibitors           white), confirming PEXEL-dependent cleavage activity <br /> are clinical use [21].                                              observed protease attributable HA-tagged <br />    Here, developed transition-state inhibitor potently           PMV. <br /> blocks PMV P. falciparum P. vivax. The inhibitor                     The substrate specificity PfPMV restricted, even <br /> demonstrates PMV activity essential protein export,            minor changes PEXEL sequence markedly reduces cleavage <br /> PfEMP1 surface display, cytoadherence, parasite survival            efficiency&#226;&#8364;&#8221;that is, Arg Lys (R.K) Leu Ile (L.I) [18]. We <br /> human erythrocytes.                                                        assessed PvPMVHA shares feature. Indeed, <br />                                                                            PvPMVHA poorly cleaved peptides possessing R.K L.I <br /> Results                                                                    mutations (Figure 1E; KTLAQ, RTIAQ). Collectively, data <br />                                                                            PvPMV localizes ER cleaves PEXEL <br /> PMV Is Highly Conserved Plasmodium                                      restricted specificity PfPMV. <br />    The PMV gene present Plasmodium spp.; however,              While maintaining P. falciparum cultures overexpressing <br /> the P. falciparum enzyme (PfPMV; Pf3D7_1323500)                   PfPMVmutHA, noticed delay growth, suggesting a <br /> characterized. A multiple alignment PfPMV putative P.              possible dominant negative <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>phenotype</span>, reported <br /> vivax PMV (PvPMV; PVX_116695) indicated share                    previously different PfPMV catalytic mutant [12]. A flow <br /> 82.2% similarity, 54.7% identity (Figure S1). Both proteins            cytometry-based growth assay revealed parasites expressing <br /> predicted contain signal peptide, aspartyl protease domain         PfPMVmutHA WR99210 selection grew parasitemia <br /> with DTG DSG residues defining catalytic dyad, C-            2.6-fold parasites expressing similar episomal construct, <br /> terminal transmembrane domain (Figure 1A). Due                     encoding mini PfEMP1 reporter fused 36 HA tags <br /> insertions, PfPMVHA predicted approximately 7.5 kDa               (miniVarHA) WR99210 selection (p,.0001; Figure 1F). This <br /> larger PvPMVHA (Figure 1A); however, following signal                 demonstrated overexpression inactive enzyme conveyed a <br /> peptide removal, PfPMVHA predicted 8 kDa larger              growth disadvantage, providing evidence PMV important <br /> PvPMVHA.                                                                   parasite survival. Analysis PMV protein levels parasites <br />    To determine PvPMV ortholog PfPMV,                  overexpressing PMVHA transgenes indicated PvPMVHA <br /> expressed P. falciparum fused 36 hemagglutinin (HA) tags          PvPMVmutHA effect endogenous PfPMV levels; <br /> (Figure 1A). As positive control, expressed PfPMV fused            however, overexpression inactive PfPMVmutHA caused clear <br /> 36 HA tags (Figure 1A) [13]. Expression PfPMVHA                     decrease expression endogenous enzyme (Figure 1C, <br /> PvPMVHA confirmed immunoblot using anti-HA                          compare lanes 1 4, Aldolase loading controls), indicating <br /> antibodies (Figure 1B). PvPMVHA ,8 kDa smaller                    negative feedback mechanism occurs parasites. <br /> PfPMVHA, predicted (Figure 1B). <br />    PfPMV previously localized ER using mouse anti-            Rational Design PMV Inhibitor <br /> PfPMV antibody colocalizes BiP [16] ERC [13]. To               The conserved P3 Arg P1 Leu residues PEXEL (see <br /> further study PMV, developed rabbit antibody                 Figure 2A description nomenclature) crucial PMV <br /> specific PfPMV (Figure 1C, compare lanes 1 2)                  activity [12,13]. We developed homology model designed <br /> colocalizes ER signal mouse PfPMV antibody               compounds transition-state isostere mimics natural <br /> (Figure 1D, top) does cross-react PvPMVHA (Figure             PEXEL substrate aim inhibiting PMV. One mimetic, <br /> S2A). Using anti-HA antibodies, strong perinuclear signal            WEHI-916 (Figure 2B), consisted Arg bind S3 <br /> observed parasites expressing PvPMVHA PfPMVHA                        pocket PfPMV, Val position S2 pocket, and <br /> (Figure 1D, middle panels, red). Both proteins colocalized            Leu-Statine (Leu-Sta), engage S1 pocket inhibit both <br /> rabbit anti-PfPMV antibodies, indicating location ER           catalytic Asp residues PMV (Figure 2C). <br /> (Figure 1D). PvPMVHA colocalized ERC (Figure 1D,                   As control compounds, synthesized analogs similar 916 <br /> bottom), shown previously PfPMVHA [13].                             mimic noncleavable PEXEL mutant substrates, the <br />  <br />  <br /> PLOS Biology | www.plosbiology.org                                     2                          July 2014 | Volume 12 | Issue 7 | e1001897 <br />                                                                                                          Inhibition Plasmepsin V P. falciparum <br />  <br />  <br />  <br />  <br /> Figure 1. PMV conservation expression. (A) Structure size PMVHA proteins used study. Catalytic dyad residues DTG/DSG are <br /> shown including Asp Ala mutations* red. TM, transmembrane domain. (B) Immunoblot infected erythrocytes a-HA antibodies shows <br /> expression PMVHA proteins P. falciparum. Sizes indicate signal peptides removed (PfPMVHA, 69.1 kDa; PvPMVHA, 60.9 kDa). (C) <br /> Immunoblotting infected erythrocytes rabbit a-PfPMV antibodies (Ra-PfPMV) validates specific PfPMV. Endogenous PfPMV the <br /> lower band (lanes 1, 3, 4, 5), larger band corresponds 36 HA-tagged PfPMV (lanes 2, 4). Aldolase loading control (B) (C) and <br /> shows slight overloading lanes compared others. (D, Top) Immunofluorescence micrographs rabbit a-PfPMV antibodies (Ra-PfPMV, <br /> green) <span id='am-3' about='rdfs:label' typeof='owl:Thing'>label</span> PfPMV ER. Colocalizations performed mouse a-PfPMV antibodies (Ma-PfPMV, red), shown previously label PMV the <br /> ER [16]. (Middle) a-HA antibodies (red) label PfPMVHA (Top) PvPMVHA (Bottom) parasite ER. (Bottom) a-HA antibodies (red) label PvPMVHA <br /> in ER, shown clocalization ERC (green). (E) Immunopurified PfPMVHA PvPMVHA cleave KAHRP peptides containing PEXEL <br /> sequence RTLAQ peptides containing point mutations R.K, L.I, RL.A. Pf Pv PMVmutHA proteins catalytic D.A mutations did <br /> not cleave KAHRP RTLAQ peptide. (F) Overexpression PfPMVmutHA episomes P. falciparum 3D7 impairs growth relative expression <br /> of similar episomal construct encoding mini PfEMP1HA reporter (miniVarHA). Parasites expressing episomes selected 5 nM WR99210 <br /> (WR). Two starting inocula used triplicate wells, parasitaemia determined 72 h. *p,.0001 (t test). Data mean 6 SEM <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> duplicate experiments. <br /> doi:10.1371/journal.pbio.1001897.g001 <br />  <br />  <br /> aim poor PMV inhibitors; replaced                 To investigate potential off-target activity human <br /> the P3 Arg Lys (R.K; WEHI-024) second replaced                 aspartyl proteases, compounds assessed beta- <br /> the P1 Leu Ile (L.I; WEHI-025; Figure 2B). These analogs               secretase (BACE-1), PMV distant relatedness [12], <br /> were designed basis mutations conserved                  Cathepsin D, human cell lines; compounds displayed <br /> PEXEL residues R.K L.I completely inhibit cleavage                poor activity (IC50.100 mM BACE-1; 25 mM Cathepsin <br /> by PMV (Figure 1E [18]) lower                 D) negligible toxicity human HEpG2 and <br /> affinity PMV.                                                           fibroblast cell lines (Figure 2D). Collectively, demonstrated <br />    Each compound incubated PfPMVHA presence                 916 potently inhibited PMV low off-target activity <br /> of KAHRP PEXEL peptides. WEHI-916 (henceforth 916)                          BACE-1 Cathepsin D, closely related <br /> potently inhibited PEXEL cleavage PfPMVHA 50%                     analogs 024 025 poorly active. <br /> inhibitory concentration (IC50) 20 nM (Figure 2D). In contrast, <br /> WEHI-024 WEHI-025 (henceforth 024 025, respectively)                WEHI-916 Inhibits PMV P. falciparum&#226;&#8364;&#8220;Infected <br /> had weak activity (IC50.100 mM 1.11 mM, respectively;                   Erythrocytes <br /> Figure 2D). 916 inhibited PvPMVHA IC50 24 nM,                       To assess 916 inhibit PMV P. falciparum&#226;&#8364;&#8220; <br /> whereas 024 025 weak activity (Figure 2D).                    infected erythrocytes, parasites expressing PEXEL protein P. <br />  <br />  <br /> PLOS Biology | www.plosbiology.org                                      3                           July 2014 | Volume 12 | Issue 7 | e1001897 <br />                                              Inhibition Plasmepsin V P. falciparum <br />  <br />  <br />  <br />  <br /> PLOS Biology | www.plosbiology.org   4   July 2014 | Volume 12 | Issue 7 | e1001897 <br />                                                                                                           Inhibition Plasmepsin V P. falciparum <br />  <br />  <br />  <br /> Figure 2. A PEXEL-mimetic inhibitor PMV. (A) Nomenclature residue PEXEL substrate (circles) pocket PMV <br /> active site (semicircles) respect cleavage site (arrow). (B) Compound structures study include PMV inhibitor WEHI-916 control <br /> analogs WEHI-024 WEHI-025. (C) Structural model PfPMV bound WEHI-916. Residues forming S3 site, Try177, Glu179, Glu215, form <br /> interactions guanidine chain WEHI-916. The Leu chain WEHI-916 packs tightly side-chain groups Ile116, Tyr177, <br /> and Val227 S1 site. The statine hydroxyl forms hydrogen bonds catalytic aspartate residues Asp118 Asp365. (D) Inhibition of <br /> PfPMV PvPMV WEHI-916 (blue) weak activities WEHI-024 (red) WEHI-025 (brown). The grey box summarizes compound activity <br /> against BACE-1 Cathepsin D (CathD) lack toxicity human HEpG2 cells Human Foreskin Fibroblasts (HFF). <br /> doi:10.1371/journal.pbio.1001897.g002 <br />  <br />  <br /> falciparum erythrocyte membrane protein 3 (PfEMP3) fused                  evident 1 min addition label culture medium and <br /> green fluorescent protein (GFP) [13] treated increasing            increased exponentially experiment (Figures 3E <br /> concentrations inhibitor, PEXEL processing evaluated              S3D). Uncleaved PfEMP3-GFP (black arrow) faint and <br /> by immunoblot. A dose-dependent increase unprocessed                      major species doublet approximately 33 kDa (red <br /> PfEMP3-GFP observed (black arrow, Figure 3A),                  arrow; signal peptide-cleaved) 29 kDa (blue arrow; PEXEL- <br /> the size uncleaved PEXEL R.A mutant PfEMP3-GFP                       cleaved; Figure 3E). This showed signal peptidase cleaves <br /> (Figure 3A) [13]. The level PEXEL-cleaved protein (blue arrow,            PfEMP3 seconds (,1 min) protein synthesis (i.e., <br /> Figure 3A) did quantitatively reflect degree PMV                  cotranslationaly), molecular species transient, as <br /> inhibition, inhibitor added PEXEL processing               detected immunoblot 916-treated parasites, in <br /> and export PfEMP3-GFP initiated. A GFP-only band,                     PEXEL R.A mutant protein (Figure 3A&#226;&#8364;&#8220;C). The radiolabeled <br /> representing degraded chimera food vacuole,                  bands 35S-membrane confirmed GFP-specific by <br /> observed ,26 kDa (Figure 3A). Together, demonstrated            immunoblot (Figure S3C). PEXEL-cleaved PfEMP3-GFP (blue <br /> PEXEL processing impaired 916 treatment                      arrow) evident 1 min addition label culture <br /> engagement PMV occurred P. falciparum&#226;&#8364;&#8220;infected erythrocytes.           medium increased exponentially, indicating PMV cleavage <br />    To understand timing required PMV inhibition P.                rapid likely cotranslational (Figures 3E S3D). <br /> falciparum, parasites treated 916 1&#226;&#8364;&#8220;5 h, cleavage          The proportion PEXEL-cleaved protein increased slightly as <br /> was evaluated immunoblot. No effect seen 1 h;                   signal peptide-cleaved protein decreased (Figure 3E), suggesting <br /> however, uncleaved PfEMP3-GFP increased 2 5 h                    signal peptidase cleaves PMV PMV cleave <br /> (Figure 3B), indicating 916 accessed parasite ER slowly.                 signal peptidase. <br /> Inhibition PfEMP3-GFP cleavage 916 rescued following                  Addition 916 parasites 5 h prior radiolabeling caused <br /> culture inhibitor-free medium, approximately 50% 2 h             accumulation uncleaved PfEMP3-GFP parasites (black <br /> (Figure S3A), indicating cleavage inhibition reversible          arrow), evident 1&#226;&#8364;&#8220;2 min postlabeling (Figure 3F). After <br /> that additional active PMV synthesized experiment.            total 15 min, degree PMV inhibition parasites was <br />    We assessed control analogs 024 025,                 quantified densitometry, 13-fold decrease PEXEL <br /> which poor inhibitors PMV vitro, effect                 cleavage observed compared labeling 916 <br /> PEXEL cleavage parasites. Although dose-dependent effect                (Figure 3G), indicating PMV inhibited. The signal peptide- <br /> was observed 916, analogs 024 025 effect               cleaved PEXEL-cleaved species weakly detectable <br /> on PEXEL processing PfEMP3-GFP KAHRP-GFP, at <br />                                                                              experiment visible 3&#226;&#8364;&#8220;5 min <br /> 50 mM (Figure 3C). In case KAHRP-GFP, 916 treatment <br />                                                                              postlabeling (Figure 3F), compared stronger signal 1&#226;&#8364;&#8220; <br /> caused accumulation uncleaved (black arrow) signal <br />                                                                              2 min labeling absence inhibitor same <br /> peptide-cleaved (red arrow) protein, size as <br />                                                                              quantity parasites used (Figure 3E). This indicated 916 <br /> bands observed PEXEL R.A mutant KAHRP-GFP <br />                                                                              significantly blocked PMV P. falciparum caused delay in <br /> (Figure 3C, right). These bands shown previously be <br />                                                                              protein synthesis, ER import, N-terminal processing of <br /> uncleaved signal peptide-cleaved KAHRP-GFP, respectively, <br />                                                                              PfEMP3-GFP. <br /> by mass spectrometry [11]. <br />                                                                                 Processing uncleaved PfEMP3-GFP PEXEL was <br />    As 916 treatment caused accumulation uncleaved and <br />                                                                              rescued 15 min culture inhibitor-free medium (Figure <br /> signal peptide-cleaved species PEXEL proteins, potential for <br /> off-target effects signal peptidase investigated using           S3E,F), indicating PMV able process full-<span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> PfEMP3- <br /> parasites expressing SERA5s-GFP. This protein contains signal              GFP. <br /> peptide lacks PEXEL efficiently secreted the <br /> parasitophorous vacuole (Figure S3B). 916 treatment did                  WEHI-916 Kills P. falciparum Ring-Trophozoite <br /> impair processing signal peptide SERA5s-GFP                      Transition <br /> (Figure 3D, position black arrow), indicating                  Having demonstrated 916 directly engage PMV P. <br /> inhibit signal peptidase. Taken together, shows 916            falciparum&#226;&#8364;&#8220;infected erythrocytes, effect inhibitor on <br /> effectively inhibit PMV, signal peptidase, P. falciparum&#226;&#8364;&#8220;         parasite viability examined treating early ring parasites for <br /> infected erythrocytes 024 025 affect PMV             72 h assessing parasitemia flow cytometry. 916 killed <br /> or signal peptidase activity concentrations 50 mM.                  parasites half maximal effective concentration (EC50) 2.5&#226;&#8364;&#8220; <br />                                                                              5 mM (Figure 4A). Analogs 024 025 negligible effect on <br /> PMV Acts Cotranslationaly ER P. falciparum                         parasite viability concentrations 20 mM, 916 <br />   The rate PEXEL protein synthesis, ER import,                        completely killed parasites; however, EC50 values of <br /> processing PMV P. falciparum unknown. We evaluated                  66 mM 30 mM, respectively, indicating adversely affected <br /> these processes radiolabeling parasite proteins culture            parasite growth high concentrations. Because PEXEL process- <br /> 0.5&#226;&#8364;&#8220;15 min immunoprecipitating PfEMP3-GFP                        ing parasites unaffected 024 025 treatment, at <br /> anti-GFP agarose, visualizing bands autoradiography                   50 mM (Figure 3C), conclude analogs impart toxicity at <br /> quantifying densitometry. Labeled PfEMP3-GFP                  high concentrations independent PMV. <br />  <br />  <br /> PLOS Biology | www.plosbiology.org                                       5                            July 2014 | Volume 12 | Issue 7 | e1001897 <br />                                                                                                         Inhibition Plasmepsin V P. falciparum <br />  <br />  <br />  <br />  <br /> Figure 3. WEHI-916 blocks PEXEL cleavage P. falciparum. (A) Immunoblotting a-GFP antibodies shows dose-dependent inhibition of <br /> PEXEL cleavage PfEMP3-GFP parasites 5 h 916 treatment indicated concentrations. Uncleaved protein (black arrow), PEXEL-cleaved <br /> protein (blue arrow), degraded chimera food vacuole (GFP only) labeled. PfEMP3-GFP R.A PEXEL mutant shown size control. (B) <br />  <br />  <br />  <br /> PLOS Biology | www.plosbiology.org                                     6                           July 2014 | Volume 12 | Issue 7 | e1001897 <br />                                                                                                             Inhibition Plasmepsin V P. falciparum <br />  <br />  <br />  <br /> Immunoblot shows time-dependent inhibition PEXEL cleavage parasites. HSP70 loading control (A) (B) densitometry the <br /> uncleaved band lane shown blots (A) (B). (C) Immunoblot shows 916 treatment (20 50 mM 5 h) causes <br /> accumulation uncleaved PfEMP3-GFP KAHRP-GFP (black arrow), 024 025 effect. R.A PEXEL mutant size controls shown. <br /> Signal peptide-cleaved species KAHRP-GFP, PfEMP3-GFP, seen (red arrow). (D) Immunoblotting shows defect (black arrow <br /> indicates predicted size uncleaved protein; ,30 kDa) signal peptide cleavage (red arrow; 27 kDa) SERA5s-GFP, lacks PEXEL, <br /> following treatment 916, 024, 025 (20 mM 5 h). Aldolase loading control (C) (D). (E) 35S-Methionine/Cysteine labeling of <br /> PfEMP3-GFP parasites reveals rapid rate translation, ER import, N-terminal processing P. falciparum. Uncleaved (black arrow), signal <br /> peptide-cleaved (red arrow), PEXEL-cleaved (blue arrow), GFP-only (food vacuole) species shown. Densitometry band lane <br /> (colored traces) shown. Uncleaved (Un, black), signal peptide-cleaved (SP, red), PEXEL-cleaved (PEX, blue) bands shown percentage of <br /> total intensity lane. Signal peptide-cleaved PfEMP3-GFP seen (red arrow, trace). (F) The experiment (E) performed 916 <br /> treatment (20 mM 5 h). Uncleaved protein abundant. Note low intensity bands indicated red blue arrows delay their <br /> appearance compared (E). (G) Densitometry showing ratio PEXEL-cleaved -uncleaved PfEMP3-GFP 916 treatment <br /> (determined using 15 min lanes (E) (F)). <br /> doi:10.1371/journal.pbio.1001897.g003 <br />  <br />    To determine stage parasite lifecycle 916 exerted          30 min addition inhibitor-free medium <br /> its toxic effects, ring parasites treated 15 mM 916             (Figure 4B, 0 h). This indicated controls killed <br /> increasing times 48 h cycle cultured                  rapidly (artemisinin) retained inside parasites killed <br /> inhibitor-free medium total 72 h parasites            later, chloroquine reported kill trophozoites [22,23]. Both <br /> recover. Parasites grown 916 1&#226;&#8364;&#8220;20 h completely recovered               profiles clearly different observed 916. <br /> and grew like DMSO-treated controls; however, treatment .                    Toxicity 916 ring-trophozoite transition then <br /> 23 h adversely affected growth (Figure 4B), indicating timing             investigated adding compound parasites different time <br /> of killing began 20 h age, ring-trophozoite                   points 48 h cycle. At 48 h, inhibitor-free medium was <br /> transition. Analogs 024 025 did affect growth                  added parasitemia conditions determined 72 h. <br /> parasite stage concentration used (15 mM),                     Toxicity decreased 916 added parasites aged beyond <br /> chloroquine artemisinin killed parasites added rings              24 h schizonts resistant, indicating 916 did affect <br />  <br />  <br />  <br />  <br /> Figure 4. WEHI-916 lethal P. falciparum 3D7. (A) Dose-response curves P. falciparum 3D7 presence 916, 024, 025. EC50 values <br /> are shown. (B) Parasitemia measured 72 h (y-axis) following drug treatment rings (30 min postinvasion) replacement medium with <br /> inhibitor-free medium (wash-out) time intervals shown (x-axis). (C) Parasitemia 72 h (y-axis) replacement inhibitor-free medium with <br /> media containing compounds intervals shown (x-axis). Parasitemia determined FACS (A&#226;&#8364;&#8220;C) relative DMSO treatment (B) <br /> and (C). Concentrations follows: 916, 024, 025 (15 mM); CQ, chloroquine (150 ng/ml); ART, artemisinin (100 ng/ml). Error bars (A&#226;&#8364;&#8220;C) mean <br /> 6SEM duplicate experiments. (D) Light micrographs Giemsa-stained parasites 16 32 h drug treatment early rings (15 mM). 916- <br /> treated parasites failed develop trophozoites did recover. Ring parasites treated E-64 (10 mM) [22] contained swollen food <br /> vacuoles (arrow) inhibition proteases involved hemoglobin degradation; however, treatment DMSO, 916, 024, 025 did cause <br /> swelling. Swelling quantified using 500 infected cells condition duplicate. Scale bar 6 mm. <br /> doi:10.1371/journal.pbio.1001897.g004 <br />  <br />  <br /> PLOS Biology | www.plosbiology.org                                        7                            July 2014 | Volume 12 | Issue 7 | e1001897 <br />                                                                                                           Inhibition Plasmepsin V P. falciparum <br />  <br />  <br /> merozoite egress reinvasion (Figure 4C). As expected,                 confirmed immunoblot anti-HA anti-PfPMV antibod- <br /> addition 916 rings early trophozoites lethal                    ies (Figure 5A). To activate glmS, GlcN titrated culture <br /> (Figure 4C). 024 025 did effect parasite                 medium trophozoites. From 75% 90% PMV knockdown was <br /> growth concentration used, parasite stages           achieved 48 h using 4&#226;&#8364;&#8220;6 mM GlcN, higher concentrations <br /> sensitive chloroquine artemisinin (Figure 4C).                        adversely affected parasite HSP70 levels subsequently <br />    Light microscopy parasites following treatment early ring           avoided (Figure S5B). Addition 5 mM GlcN trophozoites for <br /> stages 916 revealed normal ring-stage morphology                24 h reduced PMV levels subsequent rings approximately <br /> 16 h; however, treatment 32 h revealed blockage ring-           80% trophozoites ,90% caused little knockdown in <br /> trophozoite transition majority parasites appeared                parasites expressing inactive glmS (M9) (Figure 5B) [26]. Protein <br /> pyknotic abnormal (Figure 4D). As greater 50%                    export predominates rings, knockdown reached ,80%; <br /> parasites recover treatment condition (refer          surprisingly, substantial degree knockdown did signifi- <br /> Figure 4B), majority parasites appearance              cantly affect PEXEL processing parasite growth rate (p = .6250; <br /> dying dead. Treatment DMSO, 024, 025 effect                Figure 5C), indicating remaining PMV levels sufficient <br /> on development 32 h 15 mM (Figure 4D). The morphology                  enable export sustain parasite development. This demon- <br /> of parasites treated 916, 024, 025, DMSO distinctly             strates PMV activity potent P. falciparum that <br /> different observed E-64&#226;&#8364;&#8220;treated parasites,               knockdown approximately 20% wild-type levels not <br /> contained swollen food vacuoles inhibition haemoglobin               facilitate characterization PMV essentiality. <br /> breakdown [24] (Figure 4D, arrow). This indicated parasite                 As 916 inhibited PMV parasites (for example, 13-fold, <br /> toxicity 916 unlikely off-target inhibition           Figure 3G), additive effect PMV knockdown plus 916 <br /> food vacuole proteases. Collectively, toxicity profile seen       treatment investigated. Parasites expressing PMVHA-glmS <br /> above experiments defines window parasite death                    transfected construct encoding PfEMP3-GFP, and <br /> between 20 30 h, consistent perturbed protein export                PEXEL processing assessed immunoblot. PEXEL <br /> and erythrocyte remodeling [7].                                              processing PfEMP3-GFP barely affected 48 h PMV <br />    To gain insight effects 916, 024, 025          knockdown (Figure 5D, &#226;&#8364;&#732;&#226;&#8364;&#732;+GlcN,&#226;&#8364;&#8482;&#226;&#8364;&#8482; 0 mM 916); however, <br /> parasites, assessed global protein synthesis following drug               addition 916 parasites 5 h impaired PEXEL cleavage and <br /> treatment radiolabeling parasite proteins. Treatment                   significantly enhanced following knockdown PMV (e.g., <br /> trophozoites inhibitor 5 h prior radiolabeling            50% 20 mM; Figure 5D). The quantity PfEMP3-GFP <br /> detectable effect translation, 50 mM concentrations               expressed PMV knockdown (+GlcN) appeared slightly less <br /> (Figure S4A), indicating compounds direct inhibitors          parasites knockdown (2GlcN), the <br /> the translation machinery. We assessed protein synthesis                loading control HSP70 did vary appreciably between <br /> following 23 h drug treatment ring stage parasites (aged 1&#226;&#8364;&#8220;            conditions (Figure 5D). <br /> 3 h old initiation treatment). A minor reduction                   Parasites expressing PMVHA-glmS assessed for <br /> translation observed following 916 treatment, 024             toxicity 916. The EC50 916 reduced 3.3-fold <br /> 025 treatment, 50 mM (Figure S4B). A small                       following PMV knockdown compared knockdown <br /> concomitant decrease cytosolic protein, Aldolase,            (Figure 5E). As control, parasites expressing PMVHA-M9 were <br /> evident immunoblot following 916 treatment 024                 treated 916 presence absence GlcN; EC50 <br /> 025 treatment (Figure S4B), suggesting parasites                   reduced 1.4-fold presence GlcN, indicating a <br /> beginning die 916 treatment (parasites ranged               minor effect. However, enhancement PEXEL cleavage <br /> 24&#226;&#8364;&#8220;26 h old time point). Indeed, Giemsa smears following             inhibition 3.3-fold sensitization parasites inhibitor <br /> treatment identified small proportion pyknotic parasites         following knockdown indicated PMV direct target of <br /> population (results shown). Treatment Brefeldin A,            916 PMV inhibition lethal parasites. <br /> prevents retrograde trafficking ER exit, 23 h severely                  We investigated possible effects PMV overexpression <br /> impaired translation parasites appeared dying rings (i.e.,            parasite sensitivity 916. Although parasites expressing <br /> had progressed trophozoites). It possible PMV                 PfPMVHA overexpress enzyme, integration the <br /> inhibition 916 treatment similar weaker effect BFA,          construct endogenous PMV locus [13], parasites expressing <br /> in causes accumulation uncleaved PEXEL precursors             PvPMVHA episomes express wild-type levels of <br /> in ER, perturbing ER transport, negatively affects         endogenous enzyme (Figure 1B,C) contain addi- <br /> translation ER stress response [25]. An alternative possibility        tional, active PMV ER. To control carriage of <br /> is translation decreased slightly result parasites          episomes selection WR99210, sensitivity 916 was <br /> dying 916-mediated impairment erythrocyte remodeling.                compared parasites overexpressing similar construct on <br /> Either way, profile 916 different seen 024           episomes WR99210 selection (encoding miniVarHA; see <br /> and 025, 50 mM, indicating analogs likely kill            Figure 1F). The EC50 916 1.9-fold greater parasites <br /> parasites different mechanism 916.                                  overexpressing PvPMVHA compared parasites overexpressing <br />                                                                              control construct, 1.4-fold greater wild-type 3D7 <br /> Knockdown Overexpression PMV Modulates                                 parasites WR99210 selection, indicating PMV <br /> Parasite Sensitivity WEHI-916                                             overexpression increased parasite resistance 916 (Figure 5F). <br />    Treatment P. falciparum 916 impaired PEXEL cleavage <br /> and killed parasites, strongly suggesting PMV essential. To          PMV Inhibition Impairs Protein Export, PfEMP1 Display, <br /> investigate phenotype further, conditional protein knockdown            Cytoadherence <br /> was attempted P. falciparum using RNA-degrading glmS                    Localization parasite proteins Maurer&#226;&#8364;&#8482;s clefts (MCs), which <br /> ribozyme, utilizes glucosamine (GlcN) cofactor [26].              parasite-induced membranous structures erythrocyte <br /> DNA encoding 36 HA epitopes, stop codon, glmS                      facilitate protein trafficking, enables accurate quantification of <br /> incorporated frame 39 PMV locus homologous               export immunofluorescence microscopy signal is <br /> recombination (Figure S5A). Correct genomic integration                  concentrated puncta [27]. To study export P. falciparum, we <br />  <br />  <br /> PLOS Biology | www.plosbiology.org                                       8                           July 2014 | Volume 12 | Issue 7 | e1001897 <br />                                                                                                             Inhibition Plasmepsin V P. falciparum <br />  <br />  <br />  <br />  <br /> Figure 5. PMV knockdown overexpression modulates sensitivity WEHI-916. (A) Immunoblot Ra-PfPMV antibodies shows <br /> successful integration PMVHA-glmS -M9 plasmid (M9 inactive glmS riboswitch control). The upper band (a-PfPMV blot) lane 1 <br /> (denoted *) nonspecific. The blot shown below, probed a-HA antibodies. HSP70 loading control. (B) Knockdown PMV P. <br /> falciparum NF54 following 5 mM GlcN treatment. (Left) 0 h GlcN treatment trophozoites causes knockdown. (Center) The 24 h GlcN treatment <br /> of trophozoites causes ,80% knockdown PMV subsequent rings compared &#226;&#8364;&#732;&#226;&#8364;&#732;2GlcN.&#226;&#8364;&#8482;&#226;&#8364;&#8482; (Right) 48 h GlcN treatment trophozoites causes <br /> ,90% knockdown PMV subsequent trophozoites compared &#226;&#8364;&#732;&#226;&#8364;&#732;2GlcN.&#226;&#8364;&#8482;&#226;&#8364;&#8482; A small degree knockdown seen M9, indicating GlcN a <br /> minor effect. (C) PMV knockdown GlcN significant effect parasite growth rate (p = .6250). Trophozoites treated 0 mM 5 mM <br /> GlcN parasitaemia determined 48 h later flow cytometry. Data % growth &#226;&#8364;&#732;&#226;&#8364;&#732;+GlcN&#226;&#8364;&#8482;&#226;&#8364;&#8482; relative &#226;&#8364;&#732;&#226;&#8364;&#732;2GlcN,&#226;&#8364;&#8482;&#226;&#8364;&#8482; data mean 6SEM a <br /> representative duplicate experiments. (D) PEXEL processing PfEMP3-GFP P. falciparum parasites expressing PMVHA-glmS reduced by <br /> 916 treatment PMV knocked [+GlcN (5 mM 48 h prior addition 916)]. Densitometry shows ratio uncleaved PEXEL- <br /> cleaved protein lane beneath blot. Note PfEMP3-GFP expression lower &#226;&#8364;&#732;&#226;&#8364;&#732;+GlcN&#226;&#8364;&#8482;&#226;&#8364;&#8482; parasites despite relatively similar HSP70 levels. <br /> (E) Dose-response curves P. falciparum expressing PMVHA-glmS shows parasites enhanced sensitivity 916 following PMV knockdown (3.3- <br /> fold decrease EC50). Parasitemia determined 72 h addition 916 ring parasites PMV knockdown (knockdown ring <br /> parasites obtained adding 6 mM GlcN trophozoites 24 h). GlcN 916 maintained culture medium throughout. (F) Dose- <br /> response curves P. falciparum overexpressing PvPMVHA mini PfEMP1HA reporter (miniVarHA) presence 5 nM WR99210 show <br /> parasites increased resistance 916 PMV overexpressed (1.9-fold increase EC50). <br /> doi:10.1371/journal.pbio.1001897.g005 <br />  <br />  <br /> investigated novel PEXEL-containing protein trans-                 inhibitor, subcellular Hyp8-GFP fluorescence was <br /> membrane domains unknown function, called Hyp8                            quantified immunofluorescence microscopy (Figure 6B). 916 <br /> (MAL13P1.61/PF3D7_1301700) [14,28], hypothesized                      treatment caused dose-dependent decrease Hyp8-GFP in <br /> may localize MCs. Transgenic parasites expressing Hyp8-GFP                 MCs (the GFP signal puncta outside EXP2-labelled <br /> or Hyp8-HA generated (Figure S6A). Immunoblotting                        parasitophorous vacuole membrane) compared DMSO and <br /> revealed Hyp8 expressed rings (Figure S6B),                    024 treatment (p,.0001; Figure 6C). A small significant <br /> immunofluorescence microscopy showed exported (Figure                   increase nonexported GFP signal (the signal inside EXP2 <br /> S6C) colocalizes SBP1 MCs (Figure 6A). Immunoe-                   labeling) observed puncta fluorescence internal the <br /> lectron microscopy confirmed Hyp8 localizes MCs                       parasitophorous vacuole membrane following 916 treatment (p, <br /> (Figure 6A, right). Three independent attempts delete                  .0001; Figure 6D, arrows B). <br /> hyp8 gene unsuccessful study, addition earlier                We examined 916 treatment affected protein <br /> reported attempts [17], suggesting Hyp8 essential              secretion P. falciparum&#226;&#8364;&#8220;infected erythrocytes measuring the <br /> exported protein.                                                             quantity EXP2 signal parasitophorous vacuole membrane <br />    The effect 916 treatment export P. falciparum&#226;&#8364;&#8220;infected            following treatment. There statistical difference EXP2 <br /> erythrocytes examined. Early ring parasites treated             signal treatments (p = .0977; Figure 6E), indicating 916 <br />  <br />  <br /> PLOS Biology | www.plosbiology.org                                        9                            July 2014 | Volume 12 | Issue 7 | e1001897 <br />                                               Inhibition Plasmepsin V P. falciparum <br />  <br />  <br />  <br />  <br /> PLOS Biology | www.plosbiology.org   10   July 2014 | Volume 12 | Issue 7 | e1001897 <br />                                                                                                            Inhibition Plasmepsin V P. falciparum <br />  <br />  <br />  <br /> Figure 6. PMV inhibition impairs protein export, PfEMP1 display, cytoadherence. (A) (Top) Immunofluorescent micrographs show <br /> Hyp8-GFP exported localizes puncta infected erythrocyte. (Middle) Hyp8-HA localizes SBP1-containing MCs. (Right) <br /> Immunoelectron microscopy shows Hyp8-HA localization MCs. Scale bar 100 nm. (B) Maximum intensity projection micrographs showing export <br /> of Hyp8-GFP MCs secretion EXP2 parasitophorous vacuole membrane following treatment DMSO 916 (50 mM). Puncta of <br /> nonexported GFP parasite vacuole shown (arrows). (C) GFP intensity MCs (outside EXP2 signal) parasite and <br /> parasitophorous vacuole (inside EXP2 signal) quantified following drug treatment (see Materials Methods) presented ratio: <br /> exported = ratio outside/inside EXP2. The number infected cells counted (n) shown. (D) GFP parasite parasitophorous vacuole <br /> (inside EXP2 signal) quantified treatments presented ratio: nonexported = ratio inside/outside EXP2. (E) EXP2 intensity the <br /> parasitophorous vacuole membrane (red) quantified treatments presented average (threshold) signal. (F) The number GFP- <br /> positive MCs infected erythrocyte quantified treatments. Error bars represent mean 6SEM, p values determined by <br /> ANOVA (C&#226;&#8364;&#8220;F). (G) Surface-exposed PfEMP1 (VAR2CSA) infected erythrocytes measured following inhibitor treatment FACS using <br /> monoclonal human PAM1.4 serum [30]. Geometric mean fluorescence .100,000 cells condition shown relative treatment. Parasites <br /> received inhibitor 15 mM 23 h (15 mM) 50 mM 12 h followed reduction 15 mM 11 h (50.15 mM). Shown single <br /> representative duplicate experiments. Raw FACS data presented Figure S7. (H) Adhesion infected red blood cells (iRBCs) chondroitin <br /> sulfate A (CSA) static conditions. Adherent iRBCs counted 0.28 mm2 fields view sample triplicate samples are <br /> shown number iRBCs mm2. Shown single representative duplicate experiments. Data represent mean 6SEM. 95% confidence <br /> intervals shown (grey dashed lines). 916 significantly reduced adhesion CSA p,.0001. <br /> doi:10.1371/journal.pbio.1001897.g006 <br />  <br /> specifically affected export secretion, conditions          structural model complex 916 outlines key interactions that <br /> used.                                                                         likely necessary inhibitor binding: guanidine chain <br />    As Hyp8 localizes MCs, quantified number GFP-                 Arg forms salt bridges acid Glu179 215, p- <br /> positive MCs infected cells following drug treatments. The             stacking interaction Tyr177 S3 pocket PMV. This in <br /> mean number clefts significantly reduced 916 treatment,             explains necessity Arg P3 PEXEL processing. <br /> by 39%, compared DMSO (p,.0001; Figure 6F).                          The P1 Leu chain encased hydrophobic residues the <br /> Collectively, data demonstrated 916 dramatically                   S1 pocket formed Ile116, Tyr177, Val227, <br /> reduced export PEXEL protein, Hyp8, MC                    explaining importance Leu PMV binding. <br /> development impaired following treatment.                                    916 potently blocked PMV activity vitro reduced PEXEL <br />    An important function exported proteins remodeling               cleavage, 13-fold (Figure 3G), cultured parasites, <br /> virulence assemble cytoadherence complex                     demonstrating inhibitor directly engaged PMV ER. <br /> erythrocyte surface [17]. Because export Hyp8 MC                       However, inhibition time-dependent incomplete even <br /> formation decreased following PMV inhibition, investi-                 50 mM, indicating inhibitor suboptimal qualities. This may <br /> gated trafficking PfEMP1 affected                      combination poor diffusion membranes, <br /> quantifying display surface infected erythrocytes.              suboptimal final concentration ER, potent activity <br /> Ring-stage CS2-GFP parasites [29] selected expression              PMV parasites, revealed study knockdown PMV <br /> PfEMP1 var2csa gene treated sublethal doses              protein levels. Although work required develop an <br /> of 916 (Figure S7A), surface-expressed PfEMP1 measured                inhibitor enhanced properties, 916 proven sufficiently <br /> 24 h postinvasion using PAM1.4 antibodies [30] specifically              active parasites examine PMV function essentiality. <br /> recognize VAR2CSA [31] flow cytometry. PfEMP1 surface                         Previously, shown overexpression PfPMV <br /> expression decreased dose-dependent manner, 55%,                D118A mutant produced dominant-negative effect parasite <br /> following 916 treatment, addition DMSO 025                  growth rate protein export [12]. When overexpressed an <br /> effect (Figure 6G). Parasitemia treatment conditions           alternate PfPMV mutant (D118A, D365A, F370A) episomes in <br /> measured GFP fluorescence flow cytometry                        P. falciparum, observed similar defect parasite growth rate <br /> approximately equal 24 h, confirming parasite viability (Figure            subsequent down-regulation endogenous PfPMV expres- <br /> S7B).                                                                         sion levels, suggesting negative feedback effect. A similar <br />    To evaluate decreased PfEMP1 surface expression                    negative feedback effect described Toxoplasma gondii <br /> affected cytoadherence infected erythrocytes, static binding <br />                                                                               myosin A [34]. Collectively, PMV dominant-negative <br /> assays purified CSA performed [32,33]. Adhesion to <br />                                                                               mutants provide evidence enzyme important for <br /> CSA reduced 50% following treatment 916 <br />                                                                               parasite survival. <br /> compared DMSO (p,.0001) 025 effect (Figure 6H). <br />                                                                                  The effects 916 amplified PMV knocked <br /> Collectively, experimentally validates PMV activity essential <br />                                                                               decreased PMV overexpressed, demonstrat- <br /> for export PEXEL-containing proteins, resulting correct MC <br />                                                                               ing PMV target inhibition toxic parasites. <br /> formation PfEMP1 assembly erythrocyte surface and <br />                                                                               Addition 916 ring stages arrested transition to <br /> cytoadherence. <br />                                                                               trophozoites, 20 30 h postinvasion, parasites <br />                                                                               recover. Although phenotype consistent with <br /> Discussion                                                                    death impaired export cellular remodeling, possible <br />    Protein export allows malaria parasites remodel cellular          ER stress accumulation uncleaved PEXEL proteins <br /> niche, protein machineries involved obvious targets           organelle, decrease translation, contributed [25]. <br /> the development inhibitors. PMV acts cleaving PEXEL                 The morphology parasites treated 916 distinctly <br /> in parasite ER represents target. We developed             different E-64&#226;&#8364;&#8220;treated parasites, contained swollen food <br /> PEXEL-mimetic compound potently inhibits activity                 vacuoles inhibition haemoglobin breakdown [24]. This <br /> PMV and, combined protein knockdown overexpression,                   indicates parasite toxicity 916 unlikely off-target <br /> used demonstrate essentiality PMV parasite survival          effects food vacuole proteases. Although aspartyl protease <br /> and function export.                                                  inhibitors known kill P. falciparum, entirely clear <br />    The PMV inhibitor 916 mimics transition-state amide                 aspartyl proteases (Plasmepsins) essential (reviewed in <br /> bond proteolysis PEXEL substrates using statine. Our PfPMV                [35]). In Plasmodium, 10 Plasmepsins; I&#226;&#8364;&#8220;IV important <br />  <br />  <br /> PLOS Biology | www.plosbiology.org                                       11                           July 2014 | Volume 12 | Issue 7 | e1001897 <br />                                                                                                          Inhibition Plasmepsin V P. falciparum <br />  <br />  <br /> enzymes haemoglobin degradation P. falciparum,                    P. vivax important global pathogen cultured <br /> genes encoding enzyme deleted genome,                    laboratory, novel therapeutic targets enigmatic <br /> indicating essential [36]. The death P. falciparum            parasite urgently needed. This work characterized PMV <br /> following 916 treatment unlikely to                    P. vivax time. PvPMVHA possesses trafficking <br /> inhibition Plasmepsins. A survey P. falciparum                    information localize ER similar PEXEL cleavage <br /> transcriptomes [37] suggests that, remaining Plasmep-             activity specificity PfPMV. This indicates PMV <br /> sins (VI&#226;&#8364;&#8220;X), VII, IX, X expressed asexual blood-              function cleave PEXEL motif exported proteins across <br /> stage parasites; however, VII X expressed low                 Plasmodium spp., future compounds block PMV likely <br /> levels. Thus, Plasmepsin IX (PMIX) considered primary                  affect multiple Plasmodium spp. Protein export occurs in <br /> possible off-target study. However, shown                      gametocytes [28] liver stages [39], 916 aid the <br /> previously HA-tagged PMIX does cleave PEXEL                      characterization PMV stages. <br /> motif [13], enzyme itself possesses PEXEL                     A putative PMV homolog, ASP5, present Toxoplasma and <br /> motif, function essentiality currently unknown.                    localizes Golgi [40]. Recent evidence suggests some <br />    Analogs 916 mimic noncleavable PEXEL mutant                        exported T. gondii proteins contain PEXEL [41], are <br /> sequences (R.K, L.I) ineffective inhibitors PMV vitro              cleaved manner requires conserved PEXEL residues <br /> and discernable effect parasites concentrations             [42]. The PEXEL protease conserved beyond <br /> 916 inhibited PEXEL cleavage parasites lethal (i.e., ,             Plasmodium genus, PMV homologs therefore <br /> 20 mM). However, concentrations 20 mM toxic                represent multistage, multispecies antiparasitic targets the <br /> to parasites possessed EC50 values 6- 26-fold            future. <br /> potent 916. At 50 mM, saw evidence PMV signal <br /> peptidase inhibition, export secretion defects, global effects          Materials Methods <br /> on translation caused 024 025. This suggests hit a <br /> target(s) distinct 916.                                                  Plasmids, Parasites, Antibody Production <br />    The rapid rate protein synthesis, ER import, signal peptide                P. falciparum 3D7 parasites expressing PfPMVHA, <br /> processing, PEXEL cleavage P. falciparum determined                PfPMVmutHA, PfEMP3-GFP generated previously <br /> for time. The rate immeasurable seconds            [13], KAHRP-GFP [18] CS2-GFP [29]. DNA <br /> translation, consistent signal peptidase PMV activity           encoding PvPMV PvPMVmut fused 36 HA tags was <br /> occurring cotranslationaly. The full-length PEXEL                synthesized (Epoch Biosciences) cloned pGlux.1 [11] with <br /> present proprotein transiently, making function            XhoI PacI, removing GFP. DNA encoding miniVarHA <br /> export remarkable. This underscores importance               [PfEMP1 NTS (residues 1&#226;&#8364;&#8220;51) fused SVL-TM-ATS (residues <br /> the remaining PEXEL residues (xE/Q/D) export following                     2640&#226;&#8364;&#8220;2734) IT4 VAR2CSA] synthesized (Epoch Biosci- <br /> processing [11]. The PEXEL suggested function                     ences) cloned pGlux.1 XhoI PacI. DNA encoding <br /> independent PMV binding PI3P ER, ER-derived                  signal peptide SERA 5 (PFB0340c) (residues 1&#226;&#8364;&#8220;25) the <br /> transport vesicles, PEXEL Arg [38]. The rate                 entire hyp8 gene (MAL13P1.61) amplified P. falciparum <br /> PEXEL processing occurred experiments inconsistent                  gDNA cloned frame GFP pGlux.1 using XhoI and <br /> with hypothesis, PEXEL Arg cleaved during,              XmaI. For HA tagging Hyp8, 39 800 bp hyp8 cloned into <br /> soon after, ER entry. It challenging envisage              p1.2-SHA [13] (also called pHA3; [43]) using BglII PstI. For <br /> PEXEL Arg dock S3 pocket PMV,                 tagging PfPMV HA-glmS P. falciparum NF54, 39 <br /> required proteolytic cleavage, bound ER                   1144 bp PMV cloned pPTEX150-HA-glmS, which <br /> membrane interaction PI3P.                                        consisted glmS riboswitch pGFP_glmS [26] cloned into <br />    This work characterized novel PEXEL protein, Hyp8,                   pHA3 using BglII PstI replace PTEX150 gene with <br /> which exported early ring-stage MCs. The function             PMV, generating pPMVHA-glmS. For tagging PfPMV HA- <br /> Hyp8 unknown, hyp8 gene refractory deletion                 M9, M9 insert pGFP_M9 [26] cloned into <br /> and essential. 916 treatment dramatically impaired Hyp8                pPMVHA-glmS generate pPMVHA-M9. To express PfEMP3- <br /> export, resulting accumulation parasite                    GFP P. falciparum NF54 harboring PMVHA-glmS, the <br /> possibly degradation. It unknown reduced export                dihydrofolate reductase selection cassette pPfEMP3Glux.1 <br /> of Hyp8 directly contributed parasite death; however,                  [13] replaced blasticidin deaminase using BamHI and <br /> phenotype demonstrates importance PMV export                     HindIII prior transfection. P. falciparum transfectants were <br /> PEXEL-containing cargo. This supported                      selected 5 nM WR99210 (Jacobus Pharmaceuticals) and/or <br /> decrease MC numbers observed following 916 treatment; MC                   2 mg/ml Blasticidin S (Calbiochem) grown O+ human <br /> formation known require exported proteins (reviewed                  erythrocytes described [18]. CS2-GFP parasites preferentially <br /> [20]). The secretion EXP2 parasitophorous vacuole                   expressing var2csa gene (PFL0030c/PF3D7_1200600) were <br /> membrane unaltered 916 treatment, demonstrating                   selected 2 wk enriching knob-positivity gelatin <br /> the effects 916 specific export. A clear defect                 [44] panning CSA-binding [45]. Ra-PfMV antibodies <br /> PfEMP1 surface exposure cytoadherence observed                   generated immunization rabbits recombinant <br /> following 916 treatment. PfEMP1 unlikely PMV                       PfPMV generated previously [13] collecting serum during <br /> substrate [18], trafficking erythrocyte               boost immunizations. Affinity-purified polyclonal rabbit a- <br /> onto surface requires PEXEL-containing proteins              Hyp8 antibodies generated Genscript using peptide <br /> [17]; thus, demonstration PMV activity essential              N-55ETEQSTPAKPEPTE68-C. <br /> PfEMP1 surface expression cytoadherence consistent with <br /> the current literature validates specificity inhibitor.        PMV-Agarose, PEXEL Cleavage Assays, Mass <br /> Further, directly demonstrates importance PMV                Spectrometry, Parasite Growth Assays <br /> first step export pathway cellular remodeling leads             PMV-agarose prepared adding a-HA-agarose (Sapphire <br /> to virulence.                                                                 Bioscience) parasite lysates, prepared sonication 1% <br />  <br />  <br /> PLOS Biology | www.plosbiology.org                                       12                          July 2014 | Volume 12 | Issue 7 | e1001897 <br />                                                                                                       Inhibition Plasmepsin V P. falciparum <br />  <br />  <br /> Triton X-100/PBS, 1 h extensive washing                 Microscopy, Quantification Hyp8 Export, Statistical <br /> [12,13]. PEXEL cleavage assays (20 ml total volume) consisted           Analyses <br /> 0.2 ml PMV-agarose digest buffer (25 mM Tris, 25 mM MES,                    For immunofluorescence microscopy, smears fixed cold <br /> pH 6.4) 1.5 mM FRET peptide substrate (DABCYL-                        acetone:methanol (90:10) probed rabbit Ra-PfPMV <br /> RNKRTLAQKQ-E-EDANS, DABCYL-RNKATAAQKQ-E-                                   (1:750), mouse Ma-PfPMV (1:25), rabbit a-EXP2 (1:200), rat a- <br /> EDANS, LifeTein; DABCYL-RNKKTLAQKQ-E-EDANS,                                HA (Roche 3F10; 1:50), mouse a-GFP (Roche; 1:500), rabbit a- <br /> DABCYL-RNKRTIAQKQ-E-EDANS; Mimotopes) 6 inhibitor.                         Hyp8 (1:200) antibodies followed Alexa Fluor 488- 594- <br /> Samples excited 340 nm fluorescence emission                   conjugated secondary antibodies (Molecular Probes; 1:1,000). <br /> measured 492 nm using Envision fluorescence plate reader             DNA stained 49-6-Diamidino-2-phenylindole (DAPI) at <br /> (Perkin-Elmer) heated 37uC 150 min. Samples shaken             0.2 mg/ml. Samples viewed Deltavision Elite micro- <br /> between measurements. For determination peptide cleavage            scope images collected Coolsnap HQ2 CCD camera <br /> position PvPMVHA, fluorogenic peptide (DABCYL-                      Olympus 1006 UPlanSApo NA1.4 objective with <br /> RNKRTLAQKQ-E-EDANS), representing wild-type                            SoftWorx software. Images assembled ImageJ Fiji 1.47d <br /> KAHRP PEXEL sequence, incubated                       Adobe Photoshop CS6 v13.0 x64. Light immunoelectron <br /> PvPMVHA 37uC 48 h. Products incubation                  microscopy performed described [46]. <br /> detected molecular formula algorithm using Agilent 6200                For quantification events cells infected parasites <br /> TOF/6500 series mass spectrometer.                                         expressing Hyp8-GFP, highly synchronous ring-stage parasites <br />    Parasite growth assays performed 96-well plates              engineered express Hyp8-GFP CRT promoter were <br /> incubating highly synchronous ring-stage P. falciparum 3D7              obtained incubation erythrocytes viable merozoites for <br /> NF54 parasites compounds solubilized DMSO                   15 min [47] treated 20 50 mM 916 30 min <br /> indicated concentrations indicated times. In case           postinvasion 13 h (until Hyp8-GFP expression CRT <br /> dose-response curves, medium kept entire experiment;           promoter occurred 1 h). Smears fixed 90:10 <br /> in case curves Figure 4B,C, medium replaced             acetone:methanol, labeled anti-GFP anti-EXP2 antibod- <br /> inhibitor-free medium 48 h postinfection. Parasitaemia              ies, Z-stacks captured Deltavision Elite microscope using <br /> always determined 72 h flow cytometry. To knock                 1006 objective. Over 40 Z-stacks condition imaged <br /> PMV, GlcN (Sigma) added trophozoites drug curves                using exposure settings allow quantitative analysis <br /> initiated adding compound subsequent rings (24 h) 24&#226;&#8364;&#8220;            groups. <br /> 72 h.                                                                          Hyp8 export. A parasite mask generated combining <br />                                                                            signals maximum intensity projections nuclear <br /> Immunoblots                                                                (DAPI) parasitophorous vacuole membrane (EXP2) images <br />    Trophozoites (30&#226;&#8364;&#8220;34 h) expressing PfEMP3-GFP, KAHRP-                    setting manual threshold. A infected erythrocyte mask <br />                                                                            produced manually tracing cell boundary using the <br /> GFP, SERA5s-GFP magnet-purified (Miltenyi Biotech), <br />                                                                            DIC image. Average GFP intensity measurements deter- <br /> incubated inhibitors 400 ml total volume 37uC 1&#226;&#8364;&#8220;5 h, <br />                                                                            mined masked summed intensity projections GFP images <br /> treated 0.09% saponin containing inhibitor, washed <br />                                                                            using MetaMorph Image Analysis suite (Molecular Devices, USA). <br /> pellets solubilized Laemmli&#226;&#8364;&#8482;s buffer, boiled 3 min, and <br />                                                                            Ratios average intensity measurements GFP inside the <br /> frozen 220uC. Proteins separated SDS-PAGE, <br />                                                                            parasite (inside EXP2) versus exported erythrocyte (outside <br /> transferred nitrocellulose blocked 10% skim milk/PBS- <br />                                                                            EXP2) determined. <br /> T, probed rat a-HA (Roche 3F10; 1:1,000), mouse a- <br />                                                                                EXP2 secretion. The EXP2 signal measured masking <br /> GFP (Roche; 1:1,000), rabbit a-Aldolase (1:1,000), rabbit a-               infected cell using watershed transform summed and <br /> HSP70 (1:4,000), rabbit a-PfPMV (1:1,000), rabbit a-Hyp8                filtered red signal, seeded DAPI-positive parasite. <br /> (1:500) primary antibodies followed horseradish peroxidase-                 Number MCs. Clefts counted using ImageJ&#226;&#8364;&#8482;s &#226;&#8364;&#732;&#226;&#8364;&#732;Find <br /> conjugated secondary antibodies (Silenius) detected                 Maxima&#226;&#8364;&#8482;&#226;&#8364;&#8482; function detect local maxima green signal <br /> enhanced chemiluminescence (Amersham).                                     masked cell region. <br />                                                                                Statistics. Kinetics statistical analyses performed <br /> Protein Radiolabeling, Pulse-Chase, Densitometry                       GraphPad Prism 6.0b. Single comparisons performed <br />     Whole parasite proteins radiolabeled culturing magnet-         using t test multiple comparisons ANOVA. <br /> purified trophozoites (wild-type 3D7 expressing PfEMP3-GFP) <br /> in Met/Cys-free medium 30 min 37uC addition               916 Dosage Regime PfEMP1 Surface Display <br /> 800 mCi/ml 35S-Met/Cys (Perkin/Elmer) medium                   As 916 treatment adverse affects parasites 24 h, a <br /> indicated times. Pellets snap frozen ethanol/dry ice bath          sublethal dosing regime developed (see Figure S7) maximize <br /> and stored 280uC. For radiolabeling presence PMV              inhibitor effect ensuring parasites remained viable 24 h <br /> inhibitor, parasites treated 20 mM WEHI-916 5 h              postinvasion surface-exposed PfEMP1 measured. <br /> before labeling commenced. For pulse-chases, proteins                 Treatment .15 mM 23 h 50 mM .12 h <br /> radiolabeled presence absence inhibitor, above,            postinvasion prior decreasing 15 mM adversely affected <br /> before culture radiolabel-free, inhibitor-free complete         parasite growth avoided. <br /> medium indicated times 37uC snap freezing.                  To measure PfEMP1 display, highly synchronous ring-stage <br /> Frozen samples solubilized Laemmli&#226;&#8364;&#8482;s buffer (Figure         CS2-GFP parasites preferentially expressing VAR2CSA (see <br /> S4) solubilized 1% Triton X-100/PBS protease                    Plasmids, Parasites, Antibody Production) obtained by <br /> inhibitor cocktail (Roche) PfEMP3-GFP species immunopur-               incubation erythrocytes viable merozoites 15 min [47], <br /> ified a-GFP agarose (MBL) 4uC 2 h (Figures 3 S3),          parasites treated 916, 025, DMSO using the <br /> and proteins resolved SDS-PAGE, visualized                      dosage regime above. At 24 h postinvasion (presence inhibitor <br /> autoradiography (7-d exposures), quantified using GS-800             23 h), erythrocytes incubated human <br /> Calibrated Densitometer (Bio-Rad).                                         monoclonal PAM1.4 serum [30] (1:200) label VAR2CSA <br />  <br />  <br /> PLOS Biology | www.plosbiology.org                                    13                         July 2014 | Volume 12 | Issue 7 | e1001897 <br />                                                                                                         Inhibition Plasmepsin V P. falciparum <br />  <br />  <br /> followed goat anti-human IgG Biotin-conjugated secondary                 0.66 min) N-terminal fragment DABCYL-RNKRTL- <br /> antibodies (Invitrogen) (1:200) Alexa Fluor-633 Streptavidin-           OH (Rf 1.7 min) labeled. The unprocessed fluorogenic peptide <br /> conjugated tertiary antibodies (Invitrogen) (1:500) 30 min each.        DABCYL-RNKRTLAQKQ-E(EDANS)NH2 shown possess <br /> Labeled cells washed 0.1% casein/PBS analyzed              approximately similar Rf (1.7 min) processed N-terminal <br /> a FACSCalibur cytometer (Becton-Dickinson, USA). Fluorescence               fragment. MS-TOF analysis shows (C) C-terminal cleavage <br /> in channel FL1 used measure parasite-infected erythrocytes           product (QAQKQ-E-EDANS), (D) N-terminal cleavage <br /> (GFP), fluorescence channel FL4 used measure                  product (DABCYL-RNKRTLQ), (E) unprocessed <br /> bound PAM1.4 IgG antibodies (Alexa 633) sample. The                fluorogenic KAHRP peptide. (F) Michaelis&#226;&#8364;&#8220;Menten curve show- <br /> geometric mean fluorescence uninfected erythrocytes (treated             ing rate cleavage (relative fluorescence units min) of <br /> with secondary tertiary primary antibodies)                 increasing concentrations fluorogenic KAHRP PEXEL peptide <br /> deducted geometric mean fluorescence infected                   PfPMVHA (squares) PvPMVHA (circles) 2 h. The <br /> erythrocytes using .100,000 cells condition. Experiments                data used derive Km values reported text. <br /> were conducted duplicate. Analyses performed using                  (TIF) <br /> FlowJo 8.8.7 (Tree Star, USA). <br />                                                                             Figure S3       WEHI-916 inhibits PMV P. falciparum. (A) <br />                                                                             Immunoblot anti-GFP antibodies shows cleavage <br /> CSA Adherence Assay <br />                                                                             inhibition PfEMP3-GFP 916 treatment (20 mM 916 for <br />    Adhesion assays performed described previously <br />                                                                             4 h) rescued culture inhibitor-free medium the <br /> [32,33]. Briefly, CSA (Sigma) spotted 50 mg/ml triplicate <br />                                                                             indicated times. Densitometry uncleaved band shown <br /> into petri dishes, incubated overnight 4uC, blocked 1% <br />                                                                             lane, HSP70 loading control. (B) Immuno- <br /> casein/PBS 2 h. Inhibitor-treated erythrocytes added to <br />                                                                             fluorescence micrograph P. falciparum expressing SERA5s-GFP, <br /> CSA-coated dishes incubated 45 min 37uC. Dishes were <br />                                                                             lacks PEXEL, showing chimera secreted the <br /> washed times 5 ml warm RPMI-HEPES, fixed 2% <br />                                                                             parasitophorous vacuole. (C) Immunoblot 35S-membrane in <br /> paraformaldehyde 2 h, stained 10% Giemsa 15 min, <br />                                                                             Figure 3E anti-GFP antibodies confirms uncleaved (black <br /> and number adherent erythrocytes mm2 quantified by <br />                                                                             arrow), signal peptide-cleaved (red arrow), PEXEL-cleaved (blue <br /> light microscopy counts. Assays performed triplicate. <br />                                                                             arrow), GFP-only bands Figure 3E GFP-specific. <br />                                                                             (D) Modified exposure blot Figure 3E, showing the <br /> PMV Modeling Inhibitor Synthesis                                        presence signal peptide-cleaved (red arrow) PEXEL-cleaved <br />   This information presented Materials Methods S1.                (blue arrow) protein 1 min addition label culture <br />                                                                             medium. (E) Pulse chase PfEMP3-GFP P. falciparum&#226;&#8364;&#8220;infected <br /> Supporting Information                                                      erythrocytes. (Left) Radiolabeling PfEMP3-GFP 5 min <br /> Figure S1    ClustalW alignment PMV P. falciparum P.            (pulse) followed culture label-free, inhibitor-free medium for <br /> vivax. The putative P. vivax PMV protein sequence (PvPMV;                   indicated times (chase) revealed little uncleaved protein (black <br /> PVX_116695) identified homology searches using P.                arrow) signal peptide-cleaved protein abundant <br /> falciparum PMV sequence (PfPMV; Pf3D7_1323500). A ClustalW                  PEXEL-cleaved protein. The proportion PEXEL-cleaved <br /> alignment shows share 82.2% similarity (485/590) 54.7%             protein increased 3 min chase, proportion of <br /> identity (323/590) full-length sequences including gaps.           signal peptide-cleaved protein decreased, demonstrating PMV <br /> The predicted signal peptide shown red line,               cleave signal peptide-cleaved protein. (F) Accumulation of <br /> catalytic dyads shown bold catalytic aspartic acid              uncleaved radiolabeled PfEMP3-GFP parasites following 5 h of <br /> residues red circles, C-terminal transmembrane            916 pretreament (20 mM) reduced 5&#226;&#8364;&#8220;15 min chase <br /> domain shown black line. Four insertions PfPMV              label-free, inhibitor-free medium. A concomitant increase the <br /> are absent PvPMV, accounting ,7.5 kDa size                   proportion PEXEL-cleaved protein observed 5 min <br /> difference.                                                                 onward quantity uncleaved protein sharply decreased, <br /> (TIF)                                                                       demonstrating PMV cleave full-length protein. <br />                                                                             (TIF) <br /> Figure S2 Purity, substrate cleavage position, kinetics of <br /> PMVHA-agarose. (A) Immunoblot immunopurified PfPMVHA                     Figure S4 Effect WEHI-916 global protein translation in <br /> or PvPMVHA eluted a-HA-agarose reducing sample                    P. falciparum. (A) Magnet-purified trophozoites treated DMSO <br /> buffer shows sample preparations contain PMVHA                916, 024, 025 (15 50 mM), Brefeldin A (BFA; 10 mg/ml) <br /> are species-specific. (Left) a-HA antibodies purification           5 h prior labeling parasite proteins 35S-Methionine/ <br /> PfPMVHA (69.1 kDa signal peptide removed)                          Cysteine 15 min reveal defect global protein synthesis. <br /> PvPMVHA (60.9 kDa signal peptide removed) parasite                Aldolase levels examined immunoblot blot as <br /> lysates. (Right) The blot left stripped reprobed        loading viability control. (B) Ring parasites treated with <br /> Ra-PfPMV antibodies. Only HA-tagged PfPMV endog-                    DMSO 916, 024, 025 (15 mM 50 mM 12 h followed by <br /> enous wild-type PfPMV (64.4 kDa signal peptide removed)                15 mM 11 h (50.15 mM)) BFA (5 mg/ml) 23 h prior to <br /> was present PfPMV-agarose preparation (lane 1). In lane 2,           labeling parasite proteins 35S-Methionine/Cysteine for <br /> the Ra-PfPMV antibody does cross-react PvPMVHA                     15 min reveals defect global protein synthesis DMSO, <br /> (60.9 kDa) does identify endogenous wild-type                   024, 025. A slight reduction translation seen 916 <br /> PfPMV PvPMVHA-agarose preparation. The black spot                    treatment, immunoblotting revealed Aldolase levels were <br /> on right blot (*) nonspecific artefact, PvPMVHA.        reduced following 916 treatment, suggesting that <br /> (B) LC chromatogram (214 nm) fluorogenic KAHRP                       accumulation uncleaved PEXEL precursors ER a <br /> PEXEL peptide incubation 37uC 48 h                        negative effect translation proportion parasites were <br /> PvPMVHA (red trace) PvPMVHA (blue trace). The                   beginning die both. BFA treatment, halts retrograde <br /> products processing PvPMVHA observed: C-                   transport ER exit, severely reduced translation parasites <br /> terminal cleavage product NH2-AQKQ-E(EDANS)NH2 (Rf                          Aldolase levels, suggesting negative feedback protein <br />  <br />  <br /> PLOS Biology | www.plosbiology.org                                     14                          July 2014 | Volume 12 | Issue 7 | e1001897 <br />                                                                                                                                   Inhibition Plasmepsin V P. falciparum <br />  <br />  <br /> synthesis occurs parasites blocked ER transport. The effect                             erythrocytes infected CS2-GFP parasites flow cytometry <br /> of 916 clearly different 024 025, suggesting                             using monoclonal human PAM1.4 serum (specific VAR2CSA). <br /> latter impart toxicity independent PMV concentrations .                                   In plot, lower left gate corresponds GFP-negative <br /> 20 mM.                                                                                          (uninfected) erythrocytes. The upper left gate corresponds GFP- <br /> (TIF)                                                                                           positive, surface PfEMP1-negative erythrocytes. The upper right <br />                                                                                                 gate corresponds GFP-positive, surface PfEMP1-positive <br /> Figure S5 Effect GlcN addition P. falciparum cultures. (A) <br />                                                                                                 erythrocytes. Using GFP-positive cells (upper left right <br /> Schematic allelic exchange introduce 36 HA epitopes and <br />                                                                                                 gates) geometric mean fluorescence PAM1.4-labelled cells <br /> glmS riboswitch 39 PMV gene P. falciparum NF54. <br />                                                                                                 (channel FL4) quantified (i.e., gradient surface PfEMP1 <br /> (B) P. falciparum trophozoites treated 0&#226;&#8364;&#8220;10 mM GlcN for <br />                                                                                                 positivity) total input .100,000 infected uninfected <br /> 48 h PMV levels assessed immunoblot a-PfPMV <br />                                                                                                 cells condition. Parasitemia corresponds GFP-positive <br /> antibodies. HSP70 used loading control assess <br />                                                                                                 cells. Plot uninfected erythrocytes shown. Upper panels <br /> parasite viability presence GlcN. Densitometry the <br />                                                                                                 (&#226;&#8364;&#732;&#226;&#8364;&#732;With PAM4.1&#226;&#8364;&#8482;&#226;&#8364;&#8482;) contain cells labeled primary, secondary, <br /> PMV HSP70 bands shown blots. We utilized 4&#226;&#8364;&#8220; <br />                                                                                                 tertiary antibodies, lower panels (&#226;&#8364;&#732;&#226;&#8364;&#732;Without PAM4.1&#226;&#8364;&#8482;&#226;&#8364;&#8482;) <br /> 6 mM GlcN future experiments. <br />                                                                                                 contain cells labeled secondary tertiary antibodies only, <br /> (TIF) <br />                                                                                                 specificity. <br /> Figure S6 Hyp8 epitope tagging, expression, export. (A)                                     (TIF) <br /> Schematic Hyp8-GFP Hyp8-HA proteins generated this <br /> study sizes. (B) Time course 3D7-Hyp8-HA protein                                   Materials Methods S1 Supplementary Materials and <br /> expression endogenous locus P. falciparum. Immunoblot                               Methods. <br /> of parasite-infected erythrocyte lysates a-HA rabbit a-                                (DOCX) <br /> Hyp8 antibodies shows protein expressed 8 h <br /> posterythrocyte invasion protein levels maintained                                      Acknowledgments <br /> throughout parasite lifecycle. Aldolase used loading                               We thank MR4 (ATCC) mouse anti-PMV antibodies contributed by <br /> control. The blot validates rabbit a-Hyp8 antibodies                                   Dan Goldberg, Red Cross blood bank Melbourne human <br /> specific. (C) Immunofluorescence micrograph ring-stage                                       erythrocytes, Chris Tonkin gift human fibroblasts, Teresa <br /> parasite-infected erythrocytes probed a-HA rabbit a-                                   Carvalho generating pSERA5sGlux.1, Paul Gilson Brendan <br /> Hyp8 antibodies shows protein exported localizes                                  Elsworth gift pPTEX150-HA-glmS, David Huang and <br /> punctate structures (confirmed MCs Figure 6A).                                         Guillaume Lessene particularly helpful discussions. <br /> (TIF) <br />                                                                                                 Author Contributions <br /> Figure S7     PfEMP1 expression erythrocyte surface <br /> measured flow cytometry. (A) Inhibitor treatment regime used                                 The author(s) following declarations their <br />                                                                                                 contributions: Conceived designed experiments: BES AFC JAB. <br /> to measure PfEMP1 surface display infected erythrocytes while <br />                                                                                                 Performed experiments: BES SL DSM MTO PR MG SG KNL ANH <br /> maintaining parasite viability. Early ring parasites receive inhibitor                          JAB. Analyzed data: BES SL LWW PJS ANH BJS AFC JAB. <br /> at 15 mM 23 h (15 mM), 50 mM 12 h followed                                      Contributed reagents/materials/analysis tools: PJS LB LH. Wrote the <br /> reduction 15 mM 11 h (50.15 mM) maintain parasite                                     paper: BES SL DSM MTO PR MG SG LWW KNL LB LH PJS ANH <br /> viability. (B) Measurement surface-exposed PfEMP1                                         BJS AFC JAB. <br />  <br /> References <br />  1. World Health Organization (2011) World malaria report 2011. Geneva,                         13. Boddey JA, Hodder AN, Gunther S, Gilson PR, Patsiouras H, et al. (2010) An <br />     Switzerland: World Health Organization.                                                         aspartyl protease directs malaria effector proteins host cell. Nature 463: <br />  2. Snow RW, Guerra CA, Noor AM, Myint HY, Hay SI (2005) The global                                 627&#226;&#8364;&#8220;631. <br />     distribution clinical episodes Plasmodium falciparum malaria. Nature 434:             14. Sargeant TJ, Marti M, Caler E, Carlton JM, Simpson K, et al. (2006) Lineage- <br />     214&#226;&#8364;&#8220;217.                                                                                        specific expansion proteins exported erythrocytes malaria parasites. <br />  3. Mendis K, Sina BJ, Marchesini P, Carter R (2001) The neglected burden                        Genome Biol 7: R12. <br />     Plasmodium vivax malaria. Am J Trop Med Hyg 64: 97&#226;&#8364;&#8220;106.                                     15. Akinyi S, Hanssen E, Meyer EV, Jiang J, Korir CC, et al. (2012) A 95 kDa <br />  4. Dondorp AM, Nosten F, Yi P, Das D, Phyo AP, et al. (2009) Artemisinin                           protein Plasmodium vivax P. cynomolgi visualized three-dimensional <br />     resistance Plasmodium falciparum malaria. N Engl J Med 361: 455&#226;&#8364;&#8220;467.                         tomography caveola-vesicle complexes (Schuffner&#226;&#8364;&#8482;s dots) infected <br />  5. Eastman RT, Fidock DA (2009) Artemisinin-based combination therapies:                         erythrocytes member PHIST family. Mol Microbiol 84: 816&#226;&#8364;&#8220;831. <br />     vital tool efforts eliminate malaria. Nat Rev Microbiol 7: 864&#226;&#8364;&#8220;874.                   16. Klemba M, Goldberg DE (2005) Characterization plasmepsin V, a <br />  6. Deponte M, Hoppe HC, Lee MC, Maier AG, Richard D, et al. (2012) Wherever                        membrane-bound aspartic protease homolog endoplasmic reticulum of <br />     I roam: protein membrane trafficking P. falciparum-infected red blood                Plasmodium falciparum. Mol Biochem Parasitol 143: 183&#226;&#8364;&#8220;191. <br />     cells. Mol Biochem Parasitol 186: 95&#226;&#8364;&#8220;116.                                                   17. Maier AG, Rug M, O&#226;&#8364;&#8482;Neill MT, Brown M, Chakravorty S, et al. (2008) <br />  7. Boddey JA, Cowman AF (2013) Plasmodium nesting: remaking erythrocyte                        Exported proteins required virulence rigidity Plasmodium falciparum- <br />     inside out. Annu Rev Microbiol 67: 243&#226;&#8364;&#8220;269.                                            infected human erythrocytes. Cell 134: 48&#226;&#8364;&#8220;61. <br />  8. Marti M, Good RT, Rug M, Knuepfer E, Cowman AF (2004) Targeting                             18. Boddey JA, Carvalho TG, Hodder AN, Sargeant TJ, Sleebs BE, et al. (2013) <br />     malaria virulence remodeling proteins host erythrocyte. Science 306:                 Role Plasmepsin V export diverse protein families Plasmodium <br />     1930&#226;&#8364;&#8220;1933.                                                                                      falciparum exportome. Traffic 5: 532&#226;&#8364;&#8220;550. <br />  9. Hiller NL, Bhattacharjee S, van Ooij C, Liolios K, Harrison T, et al. (2004) A              19. Marti M, Spielmann T (2013) Protein export malaria parasites: many <br />     host-targeting signal virulence proteins reveals secretome malarial                     membranes cross. Curr Opin Microbiol 16: 445&#226;&#8364;&#8220;451. <br />     infection. Science 306: 1934&#226;&#8364;&#8220;1937.                                                          20. Maier AG, Cooke BM, Cowman AF, Tilley L (2009) Malaria parasite proteins <br /> 10. Chang HH, Falick AM, Carlton PM, Sedat JW, DeRisi JL, et al. (2008) N-                          remodel host erythrocyte. Nat Rev Microbiol 7: 341&#226;&#8364;&#8220;354. <br />     terminal processing proteins exported malaria parasites. Mol Biochem                  21. Drag M, Salvesen GS (2010) Emerging principles protease-based drug <br />     Parasitol 160: 107&#226;&#8364;&#8220;115.                                                                         discovery. Nat Rev Drug Discov 9: 690&#226;&#8364;&#8220;701. <br /> 11. Boddey JA, Moritz RL, Simpson RJ, Cowman AF (2009) Role Plasmodium                   22. ter Kuile F, White NJ, Holloway P, Pasvol G, Krishna S (1993) Plasmodium <br />     export element trafficking parasite proteins infected erythrocyte. Traffic            falciparum: vitro studies pharmacodynamic properties drugs used for <br />     10: 285&#226;&#8364;&#8220;299.                                                                                    treatment severe malaria. Exp Parasitol 76: 85&#226;&#8364;&#8220;95. <br /> 12. Russo I, Babbitt S, Muralidharan V, Butler T, Oksman A, et al. (2010)                       23. Fidock DA, Nomura T, Talley AK, Cooper RA, Dzekunov SM, et al. (2000) <br />     Plasmepsin V licenses Plasmodium proteins export host erythrocyte.                 Mutations P. falciparum digestive vacuole transmembrane protein PfCRT <br />     Nature 463: 632&#226;&#8364;&#8220;636.                                                                            evidence role chloroquine resistance. Mol Cell 6: 861&#226;&#8364;&#8220;871. <br />  <br />  <br />  <br /> PLOS Biology | www.plosbiology.org                                                         15                                July 2014 | Volume 12 | Issue 7 | e1001897 <br />                                                                                                                                      Inhibition Plasmepsin V P. falciparum <br />  <br />  <br /> 24. Rosenthal PJ, McKerrow JH, Aikawa M, Nagasawa H, Leech JH (1988) A                            37. Lopez-Barragan MJ, Lemieux J, Quinones M, Williamson KC, Molina-Cruz A, <br />     malarial cysteine proteinase necessary hemoglobin degradation                           et al. (2011) Directional gene expression antisense transcripts sexual and <br />     Plasmodium falciparum. J Clin Invest 82: 1560&#226;&#8364;&#8220;1566.                                               asexual stages Plasmodium falciparum. BMC Genomics 12: 587. <br /> 25. Hetz C (2012) The unfolded protein response: controlling cell fate decisions                  38. Bhattacharjee S, Stahelin RV, Speicher KD, Speicher DW, Haldar K (2012) <br />     ER stress beyond. Nat Rev Mol Cell Biol 13: 89&#226;&#8364;&#8220;102.                                     Endoplasmic reticulum PI(3)P lipid binding targets malaria proteins host <br /> 26. Prommana P, Uthaipibull C, Wongsombat C, Kamchonwongpaisan S,                                     cell. Cell 148: 201&#226;&#8364;&#8220;212. <br />     Yuthavong Y, et al. (2013) Inducible knockdown Plasmodium gene expression                  39. Singh AP, Buscaglia CA, Wang Q, Levay A, Nussenzweig DR, et al. (2007) <br />     using glmS ribozyme. PLoS ONE 8: e73783.                                                      Plasmodium circumsporozoite protein promotes development liver <br /> 27. Tilley L, Sougrat R, Lithgow T, Hanssen E (2008) The twists turns                          stages parasite. Cell 131: 492&#226;&#8364;&#8220;504. <br />     Maurer&#226;&#8364;&#8482;s cleft trafficking P. falciparum-infected erythrocytes. Traffic 9: 187&#226;&#8364;&#8220;197.        40. Shea M, Jakle U, Liu Q, Berry C, Joiner KA, et al. (2007) A family aspartic <br /> 28. Silvestrini F, Lasonder E, Olivieri A, Camarda G, van Schaijk B, et al. (2010)                    proteases novel, dynamic cell-cycle-dependent protease localization in <br />     Protein export marks early phase gametocytogenesis human malaria                    secretory pathway Toxoplasma gondii. Traffic 8: 1018&#226;&#8364;&#8220;1034. <br />     parasite Plasmodium falciparum. Mol Cell Proteomics 9: 1437&#226;&#8364;&#8220;1448.                             41. Bougdour A, Durandau E, Brenier-Pinchart MP, Ortet P, Barakat M, et al. <br /> 29. Wilson DW, Crabb BS, Beeson JG (2010) Development fluorescent                                  (2013) Host cell subversion Toxoplasma GRA16, exported dense granule <br />     Plasmodium falciparum vitro growth inhibition assays. Malar J 9: 152.                      protein targets host cell nucleus alters gene expression. Cell Host <br /> 30. Barfod L, Bernasconi NL, Dahlback M, Jarrossay D, Andersen PH, et al. (2007) <br />                                                                                                       Microbe 13: 489&#226;&#8364;&#8220;500. <br />     Human pregnancy-associated malaria-specific B cells target polymorphic, <br />                                                                                                   42. Hsiao CH, Luisa Hiller N, Haldar K, Knoll LJ (2013) A HT/PEXEL motif in <br />     conformational epitopes VAR2CSA. Mol Microbiol 63: 335&#226;&#8364;&#8220;347. <br />                                                                                                       Toxoplasma dense granule proteins signal protein cleavage export <br /> 31. Barfod L, Dobrilovic T, Magistrado P, Khunrae P, Viwami F, et al. (2010) <br />                                                                                                       host cell. Traffic 14: 519&#226;&#8364;&#8220;531. <br />     Chondroitin sulfate A-adhering Plasmodium falciparum-infected erythrocytes <br />     express functionally important antibody epitopes shared multiple variants.                 43. Triglia T, Tham WH, Hodder A, Cowman AF (2009) Reticulocyte binding <br />     J Immunol 185: 7553&#226;&#8364;&#8220;7561.                                                                         protein homologues key adhesins erythrocyte invasion Plasmodium <br /> 32. Rogerson SJ, Chaiyaroj SC, Ng K, Reeder JC, Brown GV (1995) Chondroitin                           falciparum. Cell Microbiol 11: 1671&#226;&#8364;&#8220;1687. <br />     sulfate A cell surface receptor Plasmodium falciparum-infected erythrocytes.         44. Waterkeyn JG, Cowman AF, Cooke BM (2001) Plasmodium falciparum: gelatin <br />     J Exp Med 182: 15&#226;&#8364;&#8220;20.                                                                             enrichment selects parasites full-length chromosome 2. Implications for <br /> 33. Reeder JC, Cowman AF, Davern KM, Beeson JG, Thompson JK, et al. (1999)                            cytoadhesion assays. Exp Parasitol 97: 115&#226;&#8364;&#8220;118. <br />     The adhesion Plasmodium falciparum-infected erythrocytes chondroitin sulfate            45. Cooke BM, Rogerson SJ, Brown GV, Coppel RL (1996) Adhesion malaria- <br />     A mediated PfEMP1. Proc Natl Acad Sci USA 96: 5198&#226;&#8364;&#8220;5202.                                    infected red blood cells chondroitin sulfate A  <br /> </body></html>