<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Methods <br />  <br />  <br /> Chromosomal Deletion Formation System Based <br /> on Tn5 Double Transposition: <span id='am-1' about='Thesaurus:Use_For' typeof='owl:Thing'>Use For</span> Making <br /> Minimal Genomes Essential Gene Analysis <br /> Igor Y. Goryshin,1 Todd A. Naumann,1 Jennifer Apodaca, William S. Reznikoff2 <br /> Department Biochemistry, University Wisconsin, Madison, Wisconsin 53706, USA <br />  <br />  <br />        In communication, use specialized transposons (Tn5 derivatives) create deletions the <br />        Escherichia coli K-12 chromosome. These transposons essentially rearranged composite transposons have <br />        assembled promote use internal transposon ends, resulting intramolecular transposition <br />        events. Two similar transposons developed. The deletion transposon utilized create a <br />        consecutive set deletions E. coli chromosome. The deletion procedure repeated 20 serial times <br />        reduce genome average 200 kb (averaging 10 kb deletion). The second deletion transposon <br />        contains conditional origin replication allows deleted chromosomal DNA captured a <br />        complementary plasmid. By plating cells media support plasmid replication, deleted <br />        chromosomal material lost essential, cells survive. This methodology used analyze <br />        15 chromosomal regions 100 open reading frames (ORFs). This provides robust technology for <br />        identifying essential dispensable genes. <br />        [Supplemental material available online www.genome.org supplied extended table enumerating <br />        genes lost multiple round deletion strains (&#226;&#338;&#172;20-1 &#226;&#338;&#172;20-4). These data summarized Table 1.] <br />  <br />  <br /> Transposons powerful tools performing genomic struc-               chromosomal transposon insertions, followed sequencing <br /> ture/function studies. They long used gen-                PCR analysis inserts surviving cells, assumed <br /> eration <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>knockout</span> mutations. With advent vitro                 gene contain transposon in- <br /> transposition systems (Devine Boeke 1994; Gwinn et al.                 sertion essential. In essence, methods catalog ob- <br /> 1997; Akerley et al. 1998; Goryshin Reznikoff 1998; Grif-              served nonessential genes assume genes must <br /> fin IV et al. 1999; Haapa et al. 1999), use transposons          essential. An important exception devel- <br /> genome analysis greatly expanded include,                  opment method using transposon regulated out- <br /> instance, applied mobile primer binding sites            ward-facing promoters. Insertion transposon the <br /> high-throughput sequencing efforts (Butterfield et al. 2002;               promoter region essential gene recovered under <br /> Shevchenko et al. 2002). In communication, de-                conditions transposon promoters turned on <br /> scribe powerful application DNA transposition                   support expression essential gene. Insertions can <br /> that combines vitro vivo Tn5-based technologies               screened medium does turn trans- <br /> generate random deletions Escherichia coli K12 genome.              poson promoter, resulting loss viability cells <br /> As described, Tn5 deletion formation                depend conditional transposon promoter for <br /> be used essentially bacterial species define essential           growth. This results ability positively identify essen- <br /> genes, generate minimal essential genomes, clone                 tial genes used purpose (Judson and <br /> genes.                                                                     Mekalanos 2000b). However, technique hindered the <br />      The entire DNA sequences bacterial genomes                    fact transposon inserts population <br /> have determined. However, experimentalist                inserted gene promoter regions. <br /> faced challenge determining role various                      In addition insertions, transposons capable of <br /> putative genes. Transposition-based strategies de-               promoting types DNA rearrangements. Deletion (and <br /> veloped recently identifying essential genes (for review,              inversion) formation natural feature composite trans- <br /> see Judson Mekalanos 2000a; Hamer et al. 2001; Gerdes et               posons. Composite transposons create deletions inver- <br /> al. 2002). Conceptually, methods based fact               sions intramolecular transposition mechanism (Fig. 1). <br /> that transposon insertion gene causes loss gene func-            Internal transposon ends oriented way that <br /> tion (<span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-317' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span>gene knockout</span>), insertion essential gene              DNA ends considered donor DNA, <br /> lethal organism observed (Akerley et al.              released transposase-catalyzed cleavage. The rest <br /> 1998; Hare et al. 2001). By generating large libraries                DNA (a plasmid chromosome) recognized <br />                                                                            transposon undergo self-integration, leading <br />                                                                            deletion inversion event. During process, de- <br /> 1 <br />  These authors contributed equally work.                           letions formed, removal internal trans- <br /> 2 <br />  Corresponding author. <br /> E-MAIL Reznikoff@biochem.wisc.edu; FAX (608) 265-2603. <br />                                                                            poson double-ended cleavage transposon ends and <br /> Article publication http://www.genome.org/cgi/doi/10.1101/      deletion portion chromosome integration <br /> gr.611403. Article published online print March 2003.            event. <br />  <br />  <br />  <br /> 644     Genome Research                        13:644&#226;&#8364;&#8220;653 &#194;&#169;2003 Cold Spring Harbor Laboratory Press ISSN 1088-9051/03 $5.00; www.genome.org <br />            www.genome.org <br />                                                                           Tn5 Double Transposition Chromosomal Deletions <br />  <br />  <br />  <br />  <br /> Figure 1 Strategy recursive deletion coupled deletion/plasmid formation systems. The strategy deletion formation used after <br /> integration transposon host&#226;&#8364;&#8482;s genome. The internal transposon ends (MEs) used second transposition event. Two deletions <br /> result transposition event vivo, leading removal internal transposon, second resulting the <br /> deletion portion chromosome. TnpEK/LP binds MEs, resulting blunt-end cleavage loss donor DNA. Tnp-EK/LP then <br /> facilitates intra molecular strand transfer chromosome. (A) Not events strand transfer result deletion. Inversions also <br /> result mechanism. Deletions happen left right, defining loss corresponding transposon. Deletions <br /> <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> right result loss transposon DNA, exception neutral linker. This event detected replica plating (under <br /> conditions high transposition frequency). Traditional transposition use external pair transposon ends (IEs) does occur because <br /> the expressed Tnp does interact external ends. (B) The addition conditional origin replication allows capture the <br /> deleted chromosomal material complementary self-replicating plasmid. The presence IPTG medium results capture these <br /> circular DNAs plasmids complementary chromosome. <br />  <br />  <br />  <br />  <br />      The makes use types                  RESULTS <br /> of deletions associated intramolecular transposition (Fig. <br /> 1A). Double-ended cleavage eliminates transposase gene                     Deletion Modules: Composition Strategy <br /> and selectable marker used prior transposon inser-                 The transposons used exploit observation com- <br /> tion selection. The self-integration reaction results forma-                posite transposons make deletions use internal transpo- <br /> tion second deletion starts point trans-                 son ends. The structures transposons used study are <br /> poson location extends point chromosome                      shown Figure 2. The transposon Tn5Del7 designed for <br /> or plasmid defined second transposition event. This                     immediate deletion chromosomal DNA does con- <br /> deletion deletion interest. An important feature                  tain origin replication. The Tn5Del8 transposon con- <br /> this scenario protocol eliminates selectable                   tains conditional origin replication allows cap- <br /> markers transposase gene and, thus, performed                   ture deleted chromosomal DNA cell comple- <br /> repetitively accumulation deletions.                                 mentary plasmid. Elimination plasmid triggered by <br />      In communication, use                    removal IPTG medium. <br /> transposon-based deletion strategy tool defining                      The particular features transposons used these <br /> essential genes removing nonessential genes                   experiments follows. Both transposons flanked by <br /> chromosome.                                                                    19-bp inside ends (IEs) separated donor <br />  <br />  <br />  <br />                                                                                                                     Genome Research              645 <br />                                                                                                                        www.genome.org <br />  Goryshin et al. <br />  <br />  <br />  <br />  <br /> Figure 2 Structure features transposons used work. The structure Tn5Del7 Tn5Del8 conceptually same. Both <br /> transposons defined IE sequences CTGTCTCTTGATCAGATCT (open triangle indicates IE sequence). Two ME sequences CTGTCTCTTATA- <br /> CACATCT (filled triangle indicates ME sequence) faced other, defining donor DNA transposition using ends. Distance <br /> between tips IE ME ends left 64 bp. This size linker remains chromosome deletion. Donor DNA <br /> encodes Tnp gene ME end-mediated transposition control arabinose inducible promoter KmR gene (Kan). <br /> Between right pair transposon ends, transposons selectable marker (Cam). The difference Tn5Del8 conditional <br /> origin replication. Lac repressor encoded lacI gene controls origin. Moderate plasmid (after formation) copy number ensured <br /> by Rop function. <br />  <br />  <br /> DNA component vector digestion PshAI. The                     located 362,522 bp E. coli chromosome the <br /> Tn5 IE ends capable participating synaptic complex                 lacZ gene. <br /> formation presence mutant Tnp protein Tnp sC7v2.0                       Transposition events stimulated inducing Tnp <br /> (Naumann Reznikoff 2002). The synaptic complexes                     EK/LP synthesis. The resulting colonies tested trans- <br /> delivered cells electroporation (Goryshin et. al. 2000).             position-associated events replica plating described <br /> The second set 19-bp end sequences deletion mod-                   Methods section. As typical experi- <br /> ules artificially constructed mosaic ends (MEs) (Zhou et. al.            ments using Tn5Del7, transposition events occurred in <br /> 1998). We use MEs combination Tnp EK/LP                      &#226;&#710;&#188;50% surviving cells 1/4 events being <br /> second transposition step, efficient Tn-                deletions desired direction, indicated loss re- <br /> p:DNA end combination available experiments                       sistance chloramphenicol. The large fraction cells that <br /> were performed. A high efficiency vivo transposition                   undergone transposition-associated events presumably <br /> events absolutely critical identification screen-            hyperactive nature transposase end- <br /> ing.                                                                         sequence combination used. In addition, Tnp expres- <br />      The deletion formation protocol follows (also                 sion known relatively toxic (Weinreich et al. 1994), <br /> Fig. 1). Cells electroporated preformed transposome                 and, therefore, cells undergo transposition loose the <br /> complexes, transposon inserts selected plating                Tnp gene selective advantage. <br /> on LB medium containing kanamycin chloram-                               A total independent deletions selected and <br /> phenicol. Then, single colony collection colo-              analyzed DNA sequencing directly chromo- <br /> nies used start liquid culture. At early exponential                 somal DNA use inverse PCR. Two deletions ended <br /> phase, arabinose added induce Tnp EK/LP synthesis. In                  Cam gene. The seven deletions described <br /> the case Tn5Del8, IPTG chloramphenicol added                   Figure 3. The deletion sizes varied 4&#226;&#8364;&#8220;23 kb, most <br /> activate transposon origin replication select              deletion sizes 20 kb. This large deletion size <br /> the presence excised plasmid, respectively. After               makes feasible create minimal genome reasonable <br /> hours, cells subcultured medium 500-                  time using method recursively. <br /> fold dilution grown overnight shaking. Cells are <br /> then diluted plated agar medium containing                   Recursive Deletion Formation <br /> components obtain single colonies replica plating. In                 An important feature transposition/deletion is <br /> the case Tn5Del7, colonies sensitive kana-               deletion generated, components the <br /> mycin chloramphenicol picked deletions. In                    transposon lost short linker (64 bp). The loss <br /> case Tn5Del8, cells checked sensitivity kanamycin              transposase gene ensures stability chromo- <br /> on plates contain IPTG. Plasmid DNA isolated                (in terms transposition). The loss selectable <br /> further analysis.                                                            markers provides opportunity recursive deletion forma- <br />                                                                              tion using transposon insertion transposase-mediated de- <br /> Deletions Lactose Operon Region                                       letion formation, repetitively. <br /> The maximum size chromosomal DNA deleted                           We performed 20 rounds recursive deletion formation <br /> in single transposition-mediated event limited               strain MG1655. Rather focusing deletion a <br /> relative location essential DNA. To analyze distribution              time, isolated mixed 10 colonies following <br /> of deletion sizes created Tnp limitations imposed                 round. This used starting material the <br /> by presence essential genes, chose isolate                    subsequent rounds. It assumed strains debili- <br /> Tn5Del7 insert lactose operon, known de-                tating deletions lost deletion strains <br /> letions exceeding 100 kb isolated region              unimpaired growth compete during <br /> negative impact cell growth rich medium (Bachmann                      growth mixed culture. We obtained 10 final strains <br /> 1996). After electroporation deletion module                     20 rounds growth rate equal the <br /> MG1655 plating Lactose-MacConkey agar kana-                      parental strain (data shown). <br /> mycin chloramphenicol, selected white colonies                       Pulsed-field gel electrophoresis analysis NotI-digested <br /> among thousands red colonies. One colony isolated,                    chromosomal DNA used analyze diversity among <br /> and transposon insert site sequenced. The insert                 10 deletion strains estimate average deletion <br />  <br />  <br />  <br /> 646     Genome Research <br />            www.genome.org <br />                                                                         Tn5 Double Transposition Chromosomal Deletions <br />  <br />  <br />  <br />  <br /> Figure 3 Deletions characterized lactose operon region. Genes deleted E. coli MG1655 deletion transposon Tn5Del7 are <br /> shown gray line seven deletions lac operon. The insertion site located 362,522 bp E. coli chromosome within <br /> the lacZ gene. <br />  <br />  <br /> size occurred. An example gel shown Figure             genome genes known paralogs ge- <br /> 4. The differences similarities samples evi-                nome. Another surprising result fewer appar- <br /> dent inspection gel. In cases, bands disappear               ent deletions strains expected <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> original position (in MG1655 DNA digestion)                  20 cycles (9 11 deletions two <br /> and substituted shorter fragments. In cases,                   strains). It possible deletions ended <br /> longer fragments appear, presumably loss NotI                 transposon lac insert-deletion <br /> site(s), results formation large band coupled             analysis and, thus, detected. Alternatively, <br /> with loss smaller bands. We calculated               deletions small picked by <br /> total DNA deleted 4 strains undergone                microarray hybridization technique. In cases, what <br /> 20 rounds deletion estimating size chromo-                looks like deletion array data really represent <br /> somal band compared marker DNAs. From gels                     adjacent deletions. Finally, strains <br /> that run various conditions resolve different                 extremely large deletion (&#226;&#710;&#188;125,000 and <br /> regions DNA <span id='am-319' about='xsp:pattern' typeof='owl:Thing'>pattern</span>, calculated strains             146,000-bp long) NotI fragment G, near chromosome <br /> whose DNAs analyzed Figure 4 contain deletions                   replication terminus. These deletions end in <br /> 250, 262, 100, 247 kb. This indicates average                  common (near NotI I/G fragment boundary), sug- <br /> deletion size round 11 kb, reasonable agree-             gests arose insertion event, had <br /> ment results obtained lactose operon region                separate deletion events. Thus, likely strains <br /> in previous experiment.                                                 diverged formation deletions. The observa- <br />                                                                             tion deletions relatively large indicates <br /> Microarray Mapping Deletions                                             transposition/deletion capable generat- <br /> The locations deletions strains &#226;&#338;&#172;20-1 (as judged             ing extremely large deletions, essential genes <br /> the pulsed-field gel electrophoresis missing &#226;&#710;&#188;250 kb)             region, chromosome condensation this <br /> &#226;&#338;&#172;20-4 (missing &#226;&#710;&#188;262 kb) mapped microarray hybrid-                   region chromosome favor large deletion forma- <br /> ization experiments versus progenitor strain. This tech-                tion. <br /> nique provided excellent means determine genes <br /> were removed transposition/deletion procedure. Figure <br /> 5 presents results schematically Table 1 describes              Coupled Chromosomal Deletion/Plasmid <br /> results detail. The strains apparently                 Formation System <br /> lacking common sets genes. &#226;&#338;&#172;20-1 seven unique                   The obvious limitation deletion formation de- <br /> deletions &#226;&#338;&#172;20-4 unique deletions. The data sug-                scribed inability introduce deletions involving <br /> gests strains diverged fourth common                 essential genes. To address this, developed technique to <br /> deletion generated. The strains contain wide range                conditionally save deleted DNA cell at- <br /> of deletion sizes. The deletion sizes presumably reflect                tempt elimination later. In manner, essentiality of <br /> locations essential genes nearest different transpo-             various sections chromosome directly tested by <br /> son inserts and/or differences localized DNA condensation                removal deleted DNA cell testing for <br /> near various inserts (see below). The mapped deletions                  viability. Tn5Del7 modified inserting conditional <br /> consistent pulsed-field gel electrophoresis pat-              origin replication generate Tn5Del8 (Fig. 2). When a <br /> tern total DNA missing, determined                  deletion formed, deleted material excised circle <br /> the pulsed-field gel electrophoresis patterns.                              (Fig. 1B). Because circle contains origin replication, <br />      There surprises. First, positive hybrid-            deletion formed replicating plasmid. <br /> ization results interrupting appear                       The protocol used Tn5Del8 experiments de- <br /> be contiguous set deleted genes. This observation likely            scribed above, Methods section, Figure 1B. The <br /> due phenomenon observed Richmond et al. (1999)                      critical feature IPTG present steps, com- <br /> that cross hybridization occurs IS paralog-                      mencing induction transposase order sup- <br /> containing sequences. The majority positive hybrid-                port replication plasmids formed deletion forma- <br /> ization signals represent IS sequences                tion. In addition supporting replication newly <br />  <br />  <br />  <br />                                                                                                                 Genome Research             647 <br />                                                                                                                     www.genome.org <br />  Goryshin et al. <br />  <br />  <br />                                                                            depresses cell growth creates selection cells that <br />                                                                            undergone excision transposon-encoded origin <br />                                                                            replication, experiments, &gt;95% resulting <br />                                                                            colonies contained desired class deletions. <br />                                                                                  We used coupled deletion/plasmid the <br />                                                                            search essential genes. A total 15 insertion events were <br />                                                                            treated independently isolate deletion events. Fi- <br />                                                                            nal induced cultures diluted plated agar <br />                                                                            medium containing chloramphenicol IPTG. For in- <br />                                                                            sert, total 20 colonies analyzed isolation plas- <br />                                                                            mid DNA, followed estimation plasmid size by <br />                                                                            agarose gel electrophoresis. In case, largest plasmid <br />                                                                            isolated analyzed DNA sequencing trans- <br />                                                                            poson/chromosomal DNA junctions determine portion <br />                                                                            chromosome deleted. Cells colony <br />                                                                            streaked plates containing LB (no IPTG) <br />                                                                            determine cells survive deleted <br />                                                                            chromosomal material. The colonies streaked LB <br />                                                                            plates chloramphenicol IPTG confirm loss of <br />                                                                            plasmid integration chromosome. Growth of <br />                                                                            cells medium detected case, indicating <br />                                                                            reintegration plasmid did occur. <br />                                                                                  Results analysis shown Table 2. Of 15 in- <br />                                                                            sertion strains, 11 yielded viable cells following plasmid loss <br />                                                                            largest isolated deletion. This indicated these <br />                                                                            regions DNA dispensable. In cases, cells <br />                                                                            unable grow deleted plasmid DNA, indi- <br />                                                                            cating presence essential gene deleted <br />                                                                            material. An example deletion 4.9. In case, a <br />                                                                            smaller deletion, 4.6, analyzed survive in <br />                                                                            absence plasmid (Fig. 6). In case, result <br />                                                                            indicates genes glyS and/or glyQ essential. <br />                                                                                  In case insert 8, short chromosomal DNA <br />                                                                            deletion isolated. This transposon insert resides between <br />                                                                            open reading frames (ORFs), deletion does con- <br />                                                                            tain predicted ORFs. The inability isolate larger dele- <br />                                                                            tions insert fact area <br />                                                                            contains large region essential DNA by- <br />                                                                            passed typical deletion lengths. Alternatively, localized <br />                                                                            condensation DNA amenable forma- <br />                                                                            tion large deletions. A final possibility capture of <br />                                                                            DNA immediately insert, transferred the <br />                                                                            moderate copy number plasmid, lethal cell. We <br />                                                                            distinguish options, area con- <br />                                                                            tains stretch essential genes, including gyrB dnaA. <br />                                                                                  The data shown Table 2 compared known <br /> Figure 4 Pulsed-field gel electrophoretic analysis MG1655 dele-         information essentiality E. coli ORFs using the <br /> tion derivatives 20 rounds transposition deletion. Trans- <br />                                                                            PEC database (www.shigen.nig.ac.jp/ecoli/pec/). Of four <br /> positional deletion chromosomal DNA performed recursively <br /> for total 20 cycles. After 20 cycles, chromosomal DNA       deletions determined contain essential DNA, all <br /> prepared 4 deletion strains initial strain MG1655.     contained ORF(s) reported previ- <br /> The chromosomal DNA digested completion NotI restric-          ously essential. In remaining case, deletion oc- <br /> tion endonuclease analyzed pulsed-field gel electrophoresis.        curs single ORF, dnaK. Mutations dnaK rise a <br /> Visualization ethidium bromide staining UV irradiation re-          conditionally essential <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> 37&#194;&#176;C (Bukau Walker <br /> vealed pattern DNA bands. The pattern MG1655 matches <br /> that reported previously (Heath et al. 1992), identification <br />                                                                            1989; Wild et al. 1992). Of nonessential cases, ORFs <br /> letter fragment shown right. For deletion       reported nonessential unknown. The <br /> strains, pattern altered indicates loss DNA. The size   deletions fall category indicate 40 ORFs of <br /> each new band deletion strains estimated com-           unknown essentiality considered nonessential. <br /> parison DNA markers. The estimation total deleted <br /> DNA strains shown bottom. <br />                                                                            DISCUSSION <br />                                                                            In communication, simple reliable sys- <br /> formed plasmids, addition IPTG activates origins repli-              tem making chromosomal deletions E. coli K-12. This <br /> cation remain chromosome cells                use bacterial species for <br /> undergone deletion formation. This activation presumably                   Tn5 transposome/electroporation strategy be <br />  <br />  <br />  <br /> 648      Genome Research <br />            www.genome.org <br />                                                                         Tn5 Double Transposition Chromosomal Deletions <br />  <br />  <br />  <br />  <br /> Figure 5 Microarray deletion map deletions strains &#226;&#338;&#172;20-1 &#226;&#338;&#172;20-4. Each dot represents log ratio normalized signal intensities <br /> for particular ORF deletion strain control strain counterpart. Deletion regions determined consecutive signal ratios E. <br /> coli ORFs 1 P-values &lt;0.05. These hybridization results plotted <span id='am-318' about='xsp:base' typeof='owl:Thing'>base</span> pair location comparison NotI restriction map <br /> (Heath et al. 1992). <br />  <br />  <br /> utilized. We used transposition driven Tn5 transposase de-                considered nonessential, genes contain <br /> rivatives way separate transposition               knockouts labeled essential. It assumed a <br /> events, using different Tnp:transposon end combina-                     chromosome containing essential genes be <br /> tions (Naumann Reznikoff 2002). The transposition                  sufficient <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span>survival</span>. However, analysis assumes that <br /> event delivers components second transposition event                 genes contain transposon knockouts are <br /> into chromosome transposase&#226;&#8364;&#8220;DNA complex codeliv-                     essential may, instead, poor targets the <br /> ery (Goryshin et al. 2000). The second transposition event                   transposon and, importantly, removal gene <br /> generates deletion inversion event. By simple screening                 effect essentiality remaining genes. This sec- <br /> for loss antibiotic resistance, deletions small               ond assumption likely results artificially low estimate of <br /> portion transposon left chromosome cho-                 number genes needed form viable organ- <br /> sen. As variation technique, deleted material               ism. In event, construction living cell a <br /> saved cell conditional plasmid.                             minimal genome desirable. <br />       Determining minimal genome content topic                        E. coli K-12 ideal subject minimal genome <br /> interest laboratories. One approach used deter-                 construction. It short generation time, extensive genetic <br /> mining minimal genome assemble theoretical                       tools, wealth knowledge genome physiol- <br /> minimal genome silico comparison variety                   ogy. It attractive try generate minimal <br /> different microbial genomes. This method assumes                    significantly reduced E. coli K-12 genome. In addition to <br /> essential DNA exists homologous form genomes,                  knowledge gained creation strain, the <br /> that nonessential DNA absent cases.               strain itself useful. Two examples usefulness of <br /> Alternatively, smallest genome existing genomes                    strain (1) overproduction recombinant proteins <br /> (mycoplasma) analyzed transposon knockout                        fewer E. coli proteins need removed, (2) as <br /> mutagenesis large-scale sequencing (Hutchison III et al.                 simplified metabolic model used math- <br /> 1999). Any genes knockout mutants                      ematical modeling cell metabolism. We currently <br />  <br />  <br />  <br />                                                                                                                   Genome Research             649 <br />                                                                                                                      www.genome.org <br />  Goryshin et al. <br />  <br />  <br />                                                                            chromosomal DNA random coil, com- <br />  Table 1. Genes Deleted Strains &#226;&#338;&#172;20-1 &#226;&#338;&#172;20-4 <br />                                                                            pact nucleoid body supercoiled domain structure that <br />          Deletion                                                          brings distant points close proximity (Staczek Hig- <br />                                              Map          Estimated        gins 1998). The isolation large deletions (&gt;100 kb) <br />  &#226;&#338;&#172;20-1        &#226;&#338;&#172;20-4      B-Numbera         Positionb      Sizec (bp)       near replicon terminus suggests region the <br />                                                                            chromosome unusually condensed. <br />  1.1                       b0116              2.8            1,424               Recently, strategies repetitive reduction the <br />  1.2            4.1     b0288&#226;&#8364;&#8220;b0315         6.5&#226;&#8364;&#8220;7.2         29,606 <br />                                                                            E. coli chromosome described (Kolisnychenko et al. <br />  1.3                    b1188&#226;&#8364;&#8220;b1197        26.6&#226;&#8364;&#8220;26.9        11,667 <br />  1.4                    b1335&#226;&#8364;&#8220;b1452        30.1&#226;&#8364;&#8220;32.8       124,647         2002; Yu et al. 2002). In communication, 12 planned <br />                 4.2     b1336&#226;&#8364;&#8220;b1471        30.1&#226;&#8364;&#8220;33.3       145,467         deletions introduced use efficient &#226;?&#173; red <br />  1.5            4.3     b1878&#226;&#8364;&#8220;b1892        42.3&#226;&#8364;&#8220;42.6        15,625         recombination system. This approach different in <br />                 4.4        b2379             53.8            1,238         fundamental ways. First, approach described by <br />  1.6                    b2649&#226;&#8364;&#8220;b2655          60.0            3,604         Kolisnychenko et al. (2002) generates planned deletions and <br />                 4.5     b2648&#226;&#8364;&#8220;b2655          60.0            4,117 <br />                 4.6     b2853&#226;&#8364;&#8220;b2861        64.5&#226;&#8364;&#8220;64.6         4,293 <br />                                                                            uses prior knowledge genes dispensable, <br />                 4.7        b3027             68.3              332         requires knowledge genome sequence. Our system <br />  1.7                       b3647             82.3            1,688         utilizes random approach used prior <br />  1.8            4.8     b3708&#226;&#8364;&#8220;b3715        83.8&#226;&#8364;&#8220;84.0         8,724         knowledge genes dispensable prior sequence <br />  1.9                       b4216             95.6              554         information. Secondly, Kolisnychenko et al. (2002) ap- <br />  1.10           4.9     b4308&#226;&#8364;&#8220;b4315        97.7&#226;&#8364;&#8220;97.9         9,252         proach requires use &#226;?&#173; red recombination system. <br />  1.11                   b4349&#226;&#8364;&#8220;b4354        98.7&#226;&#8364;&#8220;98.9         2,165 <br />                                                                            We know limited host range. <br />                                                                          The strategy described communication uses Tn5 <br />    Deletion span based b-number (Blattner et al. 1997). <br />  b <br />    Map position genes lost deletion span.                        transposase, active tested <br />  c <br />   Size estimated left end deleted ORF right end     bacterial species (Goryshin al. 2000). Third, strategy <br />  detected ORF.                                                        described communication allows impose se- <br />                                                                            lection fast growing cells possible deletion <br />                                                                            strains. Finally, system, deleted material be <br /> going 20 rounds presented effort                  saved conditional plasmid. Yu colleges used Cre/ <br /> create strain. The average size deletions observed            loxP delivered Tn5 derivatives P1 transduction <br /> our experiments lac operon area random deletions               locate Tn5 inserts cell (Yu et al. 2002). <br /> gives hope able reduce size E.            Importantly, depends E. coli genetic tools (P1 <br /> coli chromosome significantly reasonable time frame.                  transduction) involves multiple steps formation of <br />       The large average deletion size striking,          deletion. Furthermore, fails save deleted material as <br /> genomic DNA random coil, intramolecular trans-                  complementary plasmid. <br /> position site selection strongly biased dis-                     In communication, started list essential/ <br /> tances base pairs transposon end               nonessential genes E. coli using Tn5Del8 system. In <br /> sequences. We observed bias vitro, case,              future, technically possible create representative <br /> DNA free packing (York al. 1998). One explana-                 library deletions complementary plasmids would <br /> tion observed large vivo deletion size              cover entire E. coli genome multiple times. Systematic <br />  <br />  <br />  <br />  Table 2. The Summary Analyzed Deletion/Plasmid Combinations Different Insertions <br />  <br />  Insert                Insert                                                                                                          Grow w/o <br />  No.         &#226;&#338;&#172; &#226;&#338;&#181;&#226;?&#177;.      End        &#226;&#338;&#172; End        Size      Partial ORFs                       Complete ORFs                               IPTG <br />  <br />      1         1.3    1073109     1094823      21714     b1028; b1012    b1013; putA; putP; b1016&#226;&#8364;&#8220;1018; ycdB; phoH; <br />                                                                             b1021&#226;&#8364;&#8220;1025; tra5&#226;&#338;&#172;3; b1027                                    Y <br />      2         2.8    2776930     2770104       6826     b2638; b2647    b2639&#226;&#8364;&#8220;b2646                                                     Y <br />      3         3.1    4407731     4403605       4126     yjfl            yjeB; vacB; yjfH                                                Y <br />      4         4.2    3735047     3725227       9821     yiaB            xylB; xylA; xylF; xylG; xylH; xylR; b3570                       Y <br />      4         4.6    3735047     3723521      11526                yiaH; yiaA; yiaB; xylB; xylA; xylF; xylG; xylH; xylR; b3570     Y <br />      4         4.9    3735047     3719490      15557     yi5b            glyS; glyQ; yiaH; yiaA; yiaB; xylB; xylA; xylF; xylG;xylH; <br />                                                                             xylR; b3570                                                  N <br />   5           5.1     1224154     1210042      14112     mcrA; minD      b1160-b1173; minE                                               Y <br />   6           6.2     3045069     3042244       2825     bglA; gcvP      ygfF                                                            Y <br />   7           7.1     2779269     2767042      12227     b2633; b2647    b2634&#226;&#8364;&#8220;B2646                                                     Y <br />   8           8.4     3882592     3882500         92                                                                           Y <br />   9           9.7     4589240     4595093       5853     mdoB; tsr       yjjN; yjjM; b4356                                               Y <br />  10          10.3     4141018     4117696      23322     menA; frwC      hsIU; hsIV; ftsN; cytR; priA; rpmE; yiiX; metJ; metB; metL; <br />                                                                             metF; katG; yijE; yijF; gldA; talC; ptsA; yijl               N <br />  11          11.3     2793099     2795902       2803     gabP; b2668     ygaE; b2665&#226;&#8364;&#8220;b2667                                               Y <br />  12          12.17      14166       12088       2078                dnaK                                                            N <br />  13          13.19    4542104     4546611       4507     fimC; fimH      fimD; fimF; fimG                                                Y <br />  14          14.17    3821524     3818998       2526     spoT            rpoZ; gmk                                                       N <br />  15          15.17    1570686     1544492      26194     yddC; yddG      fdnG; fdnH; fdnl; b1477; b1478; sfcA; rpsV; b1481; <br />                                                                             osmC; b1483&#226;&#8364;&#8220;1491; xasA; gadB                                 Y <br />  <br />  <br />  <br />  <br /> 650        Genome Research <br />              www.genome.org <br />                                                                     Tn5 Double Transposition Chromosomal Deletions <br />  <br />  <br />  <br />  <br /> Figure 6 An example plasmid dependent strains. (A) The gel showing plasmids generated insertion deletion module <br /> Tn5Del8. (B) Strains 4.6 4.9 streaked TYE medium IPTG. The difference plasmids 4.6 4.9 lies the <br /> absence glyS glyQ plasmid 4.6. The elimination plasmids indicate glyS glysQ, both, essential. <br />  <br />  <br /> sequencing deletion borders coupled survival tests              EcoRI site pGT6. The resulting plasmid contains the <br /> could used determine essentiality of genes         Tn5Del7 transposon structure shown Figure 2. <br /> in entire chromosome. This type analysis                 Plasmid pGT8 source transposon Tn5Del8 <br /> adapted, use appropriate origin replica-           (Fig. 2). The KpnI&#226;&#8364;&#8220;BssSI fragments pGT7 ligated <br />                                                                         AatII&#226;&#8364;&#8220;BamHI fragment pAM34 (ATCC 77185, Hare <br /> tion, analyzing chromosomes bacteria. <br />                                                                         et al. 2001). Tn5Del8 similar structure Tn5Del7, except <br />                                                                         contains regulated origin replication. <br /> METHODS                                                                 Transposase Purification <br /> Medium                                                                  Transposase protein TnpsC7v2.0 purified using IMPACT <br />                                                                         T7 (NEB) described previously (Naumann and <br /> For experiments, used Luria-Bertanii (LB) broth liquid           Reznikoff 2000). <br /> or agar medium Lactose MacConkey agar medium (Sam- <br /> brook et. al. 1989) modified contain following antibi-           Transposome Complex Preparation <br /> otics indicated: chloramphenicol (20 mg/L), kanamycin <br /> (40 mg/L), ampicillin (100mg/L).                                        Transposons Tn5Del7 Tn5Del8 cut donor <br />                                                                         plasmid DNA digestion PshAI, followed purifica- <br /> Bacterial Strains                                                       tion agarose gel using QIAquick Gel Extraction <br />                                                                         Kit (QIAGEN). <br /> For making deletions, used E. coli K12 strain MG1655                      Transposome complexes assembled incubation <br /> (Blattner et al. 1997). For DNA manipulations, used E. coli          precut transposon TnpsC7v2.0 20 mM Tris Acetate <br /> K12 strain DH5&#226;?&#163; (Sambrook et al. 1989).                                 (pH 7.5), 100 mM K Glutamate 1 h 37&#194;&#176;C. The DNA:pro- <br />                                                                         tein molar ratio 1:5, DNA concentration of <br /> Plasmids                                                                0.1&#194;&#181;g/&#194;&#181;l. <br /> Plasmid pGT7 source transposon Tn5Del7 (Fig. 2). <br /> Plasmid pGT7 constructed inserting different compo-              Selection Initial Transposition Events <br /> nents pGT4 (Nauman Reznikoff 2002). In                    Electroporation complexes (Goryshin et al. 2000) done <br /> cloning steps, blunt ends generated using T4 DNA poly-             using standard recommended conditions (2.5 Kv, 5mS). <br /> merase (Promega). Two ME sequences taken                           After electroporation, cells recovered incubation <br /> pPDM-2 (Epicentre) KmR gene HindIII&#226;&#8364;&#8220;                1 h LB broth, plated LB agar containing <br /> EaeI fragment ligated large HindIII&#226;&#8364;&#8220;SmaI frag-          kanamycin chloramphenicol. <br /> ment pGT4, giving rise pGT5. The Tnp EK/LP gene and <br /> the AraC gene portion KmR gene taken                 Induction Deletions <br /> from pGRARAK (I. Goryshin, unpubl.) BclI&#226;&#8364;&#8220;ClaI frag- <br /> ment ligated ClaI&#226;&#8364;&#8220;NaeI large fragment pGT5             (Secondary Transposition Events) <br /> give pGT6. Finally, CmR gene pACYC184 taken                Individual pooled colonies initial selection were <br /> by isolating BsaAI&#226;&#8364;&#8220;PshAI fragment ligating            inoculated LB medium arabinose concentration <br />  <br />  <br />  <br />                                                                                                           Genome Research            651 <br />                                                                                                              www.genome.org <br />  Goryshin et al. <br />  <br />  <br /> of 0.4%, grown 8 h 37&#194;&#176;C. In case Tn5Del8,        ACKNOWLEDGMENTS <br /> IPTG included concentration 1 mM. Cells were <br />                                                                    We thank Barb Schriver providing medium and <br /> subcultured 500-fold dilution LB medium (plus <br />                                                                    reagent preparations work. We thank Laura Vander- <br /> chloramphenicol IPTG Tn5Del8), grown overnight, <br />                                                                    ploeg Department Biochemistry Media Lab for <br /> and plated appropriate dilution LB agar (plus <br />                                                                    valuable assistance figures tables. We also <br /> chloramphenicol IPTG Tn5Del8) obtain individual <br />                                                                    thank Kelly Winterberg helpful discussion the <br /> colonies analysis. <br />                                                                    manuscript members research laboratory <br />      Screening deletions Tn5Del7 rep- <br />                                                                    support. This research supported NSF <br /> lica plating colonies LB agar agar containing LB, <br />                                                                    grant MCB-0084089, NIH grant GM50692, grant from <br /> LB-kanamycin, LB-chloramphenicol. Cells sensitive to <br />                                                                    Robert Draper Technology Fund, Wisconsin Alumni Re- <br /> kanamycin considered undergone transposition, <br />                                                                    search Foundation. <br /> and cells sensitive drugs considered a <br />                                                                         The publication costs article defrayed part <br /> deletion adjacent DNA correct portion trans- <br />                                                                    payment page charges. This article be <br /> poson small transposon linker left the <br />                                                                    marked &#226;&#8364;&#339;advertisement&#226;&#8364;? accordance 18 USC <br /> chromosome. In case Tn5Del8, cells were <br />                                                                    section 1734 solely indicate fact. <br /> kanamycin sensitive chloramphenicol resistant, the <br /> replication excised DNA circles supported pres- <br /> ence 1 mM IPTG medium. <br />                                                                    REFERENCES <br /> Microarray Analysis                                                Akerley, B., Rubin, E., Camilli, A., Lampe, D., Robertson, H., and <br />                                                                        Mekalanos, J. 1998. Systematic identification essential genes <br /> We used spotted DNA microarrays containing 95%                  vitro mariner mutagenesis. Proc. Natl. Acad. Sci. <br /> 4290 ORFs E. coli K-12 genome. These spotted microar-             95: 8927&#226;&#8364;&#8220;8932. <br /> rays supplied Gene Expression Center, University       Bachmann, B.J. 1996. Derivations genotypes mutant <br /> of Wisconsin. Genomic DNAs E. coli strain MG1655,                 derivatives Escherichia coli K-12. In Escherichia coli and <br /> &#226;&#338;&#172;20-1 &#226;&#338;&#172;20-4 prepared using Master Pure DNA Pu-              Salmonella, pp. 2460&#226;&#8364;&#8220;2488. ASM Press, Washington, D.C. <br />                                                                    Blattner, F.R., Plunkett III, G., Bloch, C.A., Perna, N.T., Burland, V., <br /> rification Kit (Epicentre) digested AluI (Promega)            Rilley, M., Collado-Vides, J., Glasner, J.D., Rode, C.K., Mayhew, <br /> further purified phenol/chloroform extraction ethanol           G.F., et al. 1997. The complete genome sequence Escherichia <br /> precipitation. Genomic DNA labeling Cy3 Cy5                   coli K12. Science 277: 1453&#226;&#8364;&#8220;1462. <br /> dUTP (NEN, Life Science) hybridizations performed         Bukau, B. Walker, G.C. 1989. Cellular defects caused deletion <br /> according protocols provided Gene Expression Cen-            Escherichia coli dnaK gene indicate roles heat shock <br /> ter, University Wisconsin (http://www.gcow.wisc.edu/Gec/            protein normal metabolism. J. Bacteriol. 171: 2337&#226;&#8364;&#8220;2346. <br /> index.htm). Microarray images scanned using Packard           Butterfield, Y.S.N., Marra, M.A., Chan, S.T., Guin, R., Kryzwinski, <br /> BioChip SA5000 quantitated using Scanalyze 2.1. Nor-               M.I., Lee, S.S., MacDonald, K.W.K., Mathewson, D.A., Olson, <br />                                                                        T.E., Pandoh, P.K., et al. 2002. An efficient strategy large-scale <br /> malization microarray signal intensities involved median            high-throughput transposon-mediated sequencing cDNA <br /> background subtraction calculation percent signal in-           clones. Nucleic Acids Res. 30: 2460&#226;&#8364;&#8220;2468. <br /> tensities spot (Richmond et al. 1999). The percent        Devine, S.E. Boeke, J.D. 1994. Efficient integration artificial <br /> signal intensities used determine ratio deletion        transposons plasmid targets vitro: A useful tool DNA <br /> strain control E. coli MG1655 signals. Values percent       mapping, sequencing genetic analysis. Nucleic Acids Res. <br /> signal intensity log transformed order combine              22: 3765&#226;&#8364;&#8220;3772. <br /> data paired slide (dye swap) experiments. The ratios        Gerdes, S.Y., Scholle, M.D., D&#226;&#8364;&#8482;Souza, M., Bernal, A., Baev, M.V., <br /> deletion strain control values used subsequent             Farrell, M., Kurnasov, O.V., Daugherty, M.D., Mseeh, F., <br />                                                                        Polanuyer, B.M., et al. 2002. From genetic footprinting to <br /> Z-score calculations determine deletion sites. We estab-            antimicrobial drug targets: Examples cofactor biosynthetic <br /> lished null hypothesis presence DNA,          pathways. J. Bacteriol. 184: 4555&#226;&#8364;&#8220;4572. <br /> indicated ratios values close 1 rejected   Goryshin, I.Y. Reznikoff, W.S. 1998. Tn5 vitro transposition. <br /> null ratios P  <br /> </body></html>