<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Proc. Natl. Acad. Sci. USA <br /> Vol. 91, pp. 12696-12700, December 1994 <br /> Genetics <br />  <br />  <br /> Allele-specific quantification Drosophila engrailed and <br /> invected transcripts <br /> ANDREW S. GOLDSBOROUGH AND THOMAS B. KORNBERG <br /> Department Biochemistry Biophysics, University California, San Francisco, CA 94143 <br /> Communicated Edward B. Lewis, September 26, 1994 (receivedfor review August 2, 1994) <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>           Changes levels transcription                                                                    Tissue dissection _                        mRNA  purification <br /> difficult gauge animals lethal mutations. For exam-                                                                                                             cDNA synthesis _ <br /> ple, mutations regulatory region essential gene                                                                 PCR amplification                         _       Sequencing and <br /> have secondary consequences complicate attempts                                                                                                                                       PhosphorImager analysis <br /> quantify transcripts produced mutant gene. We                             A                                                                        B <br /> describe method circumvents problem revealing                          ... ....:......... <br />                                                                                        ddG ddG <br />                                                                                         ..................... <br />                                                                                       .....................                        ddG............... <br />                                                                                                                                             .....             .ddA ddG                                   ..................... <br />                                                                                                                                                                                                                                          dG                     ddA ddG <br /> the relative transcript produced allele                                                ... . <br />                                                                                           . . . . ........... . <br />                                                                                                               .   .                           _....                ...    ......... <br />                                                                                                                                                                           ::.:.:... <br />                                                                                                                                                                                                     .... .............. <br />                                                                                                                                                                                                                                        ...... <br />                                                                                                                                                                                                                                            _.... <br />  <br />  <br />  <br />  <br /> a heterozygote. With method, recessive lethal mutations                                           :......                                          ... <br />                                                                                                                                                                                                                                         ........ <br />  <br />                                                                                                                                                                                                                                    .....:::. <br /> can analyzed animals phenotypically wild type.                               .... ........                                                                ... <br />                                                                                                                                                                            ..                     ....                    .... .                      ....... <br />                                                                                                                                ......................                                                                              .... <br />                                                                                                                                                                                                                                      ........ <br />                                                                                                                                                                                                                                      ........ . <br />                                                                                             .....    ......        ...   ...                  ...       .:...........                         .    ..            .                 .    ... <br />  <br />                                                                                                                                                                           ..................................                                       ......... <br /> We used technique analyze mutations regulatory                                                                  .................. <br />                                                                                                               ..                                              ..                                                                                                  .. <br />  <br />  <br />  <br />  <br />                                                                                       :.:           ::.::....... :..            ......... <br /> region Drosophila engrailed gene trun-                                           :::.. <br />                                                                                                                                                                                       ..... <br />  <br />  <br />  <br /> cations reduce transcription levels depend the <br /> tissue location chromosomal break. We also <br /> found mutations affect expression linked                                                                      ........... <br />                                                                                                                                                                                                                                                                        .....   .......... <br />  <br />  <br />  <br />  <br /> invected gene, suggesting engrailed invected share a <br /> complex set regulatory elements operate 85                            Uk OR Uk/OR                                                                  B3razil                                             OR                                Brazil/OR <br /> kb. We suggest technique general utility for <br /> the quantitation allele-specific transcripts,                              FIG. 1. Procedure allele-specific quantification transcripts. <br /> amounts tissue limiting.                                                      Tissue dissected mRNA purified first-strand cDNA <br />                                                                                      synthesis PCR amplification engrailed invected exons. The <br />                                                                                      coamplification cDNA derived mutant (M) wild-type <br /> Genes present copies diploid organisms                        (WT) alleles removes need carefully calibrate PCR amplifica- <br /> assumed contribute equally RNA production. However,                            tions reference transcripts. PCR primers span polymor- <br /> exceptions arise course normal development                             phism intron allow discrimination transcripts <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br /> [e.g., allelic exclusion maturation immune (1)                     avoid genomic DNA amplification. Polymorphisms revealed by <br /> and olfactory (2) systems parental imprinting (3)]                       sequencing PCR product representing transcript are <br /> mutations alter promoter utilization RNA stability. Al-                           quantified following gel electrophoresis PhosphorImager analy- <br />                                                                                      sis. (A) Hypothetical representation gel bands obtained a <br /> though increasing documenting                        single sequencing reaction chain termiinator dideoxygua- <br /> variations, available methods accurate sensitive                         nosine triphosphate (ddG) engrailed cDNA derived Ukraine <br /> quantitation detect subtle changes. Northern blotting,                        (Uk) Oregon-R (OR) homozygotes heterozygotes (Uk/OR). <br /> RNase protection, quantitative PCR measure total                             The insertion polymorphism Ukraine cDNA produces shift <br /> transcript levels, crude comparisons provide                            indicated arrows. (B) Hypothetical representation the <br /> depend normalizing data unrelated reference tran-                            invected cDNA sequence Brazil Oregon-R homozygotes or <br /> script abundance (4). However, reference transcripts vary                        heterozygote (Brazil/OR) containing single <span id='am-163' about='xsp:base' typeof='owl:Thing'>base</span> change poly- <br /> in different tissues changing growth conditions. Fur-                        morphism (arrow). Two sequencing lanes (ddG) (ddA) each <br /> thermore, Northern blotting RNase protection can-                          cDNA preparation illustrate polymorphism. <br /> not, general, distinguish transcripts contributed each <br /> gene heterozygote, use methods limited                        gous genes standardization reference transcript. <br /> mostly studies homozygous mutants. As consequence,                           The rationale ofthe method follows. In cell two <br /> lethal mutations largely precluded analysis.                          different copies actively transcribed gene, RNA <br />    Interest measuring relative abundance transcripts                       pool contain transcripts copy proportion <br /> in heterozygous human tissues cells increased                        relative rate synthesis half-life. If tran- <br /> mutant genes identified isolated. An example                        scripts different copies distinguished their <br /> the myotonin-protein kinase gene, expansions                           size sequence, resolving differences provides a <br /> repetitive trinucleotide sequence associated myotonic                       measure relative abundance. Our method, termed <br /> muscular dystrophy (5-7). Efforts determine                           allele-specific amplification (ASA), uses simple sequence <br /> level myotonin-protein kinase gene transcripts altered                      polymorphisms monitor relative amounts transcript <br /> heterozygotes carrying autosomal dominant <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>mutation</span>                           produced alleles heterozygote. mRNA <br /> have frustrated inadequacy available                              isolated small amounts tissue used template <br /> methods analysis. Indeed, use quantitative reverse                        reverse transcription provided template PCR <br /> transcription-PCR compare expression levels                                amplification sequences chosen include relevant <br /> groups led contradictory conclusions (8-11).                                      polymorphism span intron. Sequence analysis of <br />    In report, new method yields a <br /> relative measure transcript accumulation heterozy-                           classes DNA product yielded quantitative <br />                                                                                      measure relative transcript abundance (Fig. 1). <br /> The publication costs article defrayed page charge <br /> payment. This article marked &quot;advertisement&quot;                Abbreviations: ASA, allele-specific amplification; M, mutant; WT, <br /> in accordance 18 U.S.C. &#194;&#167;1734 solely indicate fact.                     wild type; ddN, dideoxynucleoside triphosphate. <br />  <br />                                                                              12696 <br />           Genetics:   Goldsborough Kornberg                                  Proc. Natl. Acad. Sci. USA 91 (1994)         12697 <br />  <br />              MATERIALS AND METHODS                                Brazil WT flies sequenced primer 5&apos;-ACCGGAT- <br />                                                                   GCTCCTGTGGCACC-3&apos; (nt 1486-1506). <br />    Genomic DNA Amplification. Individual flies homog-           Transcript Quantification. To quantify relative abun- <br /> enized 100 ,l 6 M NaI (Bio 101) 1.5-ml Teflon        dance transcripts, amplified products M WT mRNA <br /> pestle, DNA purified binding glass particles        preparations fractionated sequenced standard <br /> (Geneclean, Bio 101). DNA eluted glass 20       sequencing gels. Gels fixed exposed overnight <br /> ,l water, 1 ,l4 used template PCR        Molecular Dynamics Phosphor screen. Data ob- <br /> amplification exon-specific oligonucleotides.                tained scanning exposed screen Molecular <br />    Drosophila Crosses, mRNA Purification, cDNA Synthe-        Dynamics Phosphorlmager equipped IMAGEQUANT <br /> sis. Heterozygote male flies engrailed (en) break-       Version 3.3 software. <br /> point/Black cells (Bc) genotype crossed Ukrainian            The ddG-terminated sequence position 2017 engrailed <br /> WT females breakpoint/Ukraine larvae identified          quantified follows: The position nucleotide 2017 <br /> by absence Bc <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. Approximately 1-3 ,ug         derived M Ukraine (WT) mRNA separated six <br /> of tissue dissected late instar larvae lysed   bases sequenced 3&apos; cDNA to <br /> on poly(U) paper (2 mm2; Hybond-mAP, Amersham)             presence additional nucleotides WT tran- <br /> presence 4 M guanidinium isothiocyanate/10 mM                  script. The relative abundance transcripts be <br /> Tris HCl, pH 7.5/1 mM EDTA/0.5 M LiCl. mRNA stable             determined directly specific bands DNA- <br /> on paper 2 days 4&#194;&#176;C. After       sequencing gel calculating average pixel value the <br /> washed times 200 /, 0.5 M LiCl times          bands, subtracting background, determining area of <br /> with 70% EtOH, poly(U) paper briefly air dried,           bands integration. Expression M transcript was <br /> added 10 Al water, heated 10 min 70&#194;&#176;C. cDNA      calculated percentage WT ratio M and <br /> was prepared using Superscript II according manufac-       WT bands. Since values M WT reflect the <br /> turer&apos;s (BRL) instructions. Reverse-transcription reactions       synthetic products single sequencing reaction after <br /> were performed addition oligo(dT)           amplification RNA preparation, controlling for <br /> poly(U) bound Hybond-mAP paper served                variations reaction conditions loading efficiency was <br /> effective primer. Incubations 30 min 42&#194;&#176;C, 30 min     unnecessary. <br /> at 45&#194;&#176;C, 10 min 75&#194;&#176;C.                                         For analysis invected transcripts, assay in- <br />    PCR Amplification. Amplified products engrailed       volved sequencing lanes, necessary com- <br /> invected cDNAs prepared follows. Due lower         pensate variations loading follows. Corrections to <br /> efficiency engrailed cDNA amplified,         differences loading normalizing values <br /> second amplification step (semi-nested PCR) performed.        experimental bands lane selected reference <br /> engrailed cDNA (1 /4) amplified 10-,ul PCR reaction      bands. For crosses en49, en63, en59, en2, poly- <br /> mixture containing final concentrations 15 mM Tris HCl      morphisms caused C -- T transitions positions 826 and <br /> pH 8.8, 2.5 mM MgCl2, 60 mM KCl, 0.4 mM dNTPs, 0.4 mM             834 gave rise reciprocal changes DNA in <br /> primer oligonucledotides [5&apos;-ATCGTCCCAGCTCAGGAC-                  ddG ddA lanes positions. Reference bands <br /> 3&apos; 5&apos;-GTTGCTCGTTAACATTTCGC-3&apos; (nt 1151-1528                   821 823 ddG lane 820 829 ddA <br /> and 2332-2351, respectively)], 0.4 <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> Taq DNA            lane. For crosses en52, en61, en67, polymor- <br /> polymerase. Conditions PCR follows: 30 cycles         phisms caused C -- T transitions positions 1341 and <br /> of 30 sec 94&#194;&#176;C, 30 sec 65&#194;&#176;C, 60 sec 72&#194;&#176;C. A 1-,ul    1347 scored ddG ddA sequencing <br /> aliquot primary amplification product added       reactions. Reference bands nt 1348 1349 ddG <br /> 25-,ul PCR reaction mixture primers 5&apos;-TTGGC-            lane 1355 ddA lane. Normalized pixel values for <br /> CCGCCTCAAGCGG-3&apos; 5&apos;-GTTGCTCGTTAACATT-                         polymorphic bands expressed percentage WT. <br /> TCGC-3&apos; (nt 1609-1626 2332-2351, respectively). Both <br /> forward PCR primers designed span introns avoid                                    RESULTS <br /> genomic DNA amplification. <br />    invected cDNA amplified 25-,ul reaction mixture       To demonstrate sensitivity validity ASA (see Fig. 1 <br /> containing final concentrations 20 mM Tris HCl pH 8.3,      Materials Methods), studied expression of <br /> 1 mM MgCl2, 50 mM KCl, 0.4 mM dNTPs, 0.4 mM                engrailed invected transcripts genetic contexts. <br /> primer oligonucleotides [5&apos;-CAACTGCCTCCCTCCAC-                    Fig. 2 depicts linked genes, opposing transcrip- <br /> TGG-3&apos; 5&apos;-TGGACGTCGCCGGCTTCTTGG-3&apos; (nt                        tion units, positions 11 mutations chromo- <br /> 772-791 1613-1633, respectively)]. PCR cycles 1          somal rearrangements affect engrailed function various ex- <br /> min 94&#194;&#176;C, 1 min 55&#194;&#176;C, 2 min 72&#194;&#176;C. All invected       tents. Polymorphic changes engrailed invected se- <br /> PCR products spanned intron 2 sequenced             quences used distinguish allelic <br /> ddG ddA reveal polymorphisms allow quanti-          transcripts sought laboratory strains in <br /> tative analysis transcripts.                               seven WT strains isolated geographically <br />    DNA Sequencing. The engrailed PCR product purified         distant regions (e.g., Oregon-R, Canton-S, Brazil, Ukraine, <br /> (Geneclean) following agarose gel electrophoresis directly    Siberia, Australia, Serbia). Sequencing engrailed in- <br /> sequenced. Template DNA (200 ng) primer [200 ng; 5&apos;-          vected exons amplified strains revealed differences <br /> TACTTTCGGCTATCCAGG-3&apos; (nt 2094-2111)] mixed               engrailed 3 &apos;-untranslated regions CyO chromosome <br /> a total volume 4.5 ,ul H20 containing 10%o (vol/vol)        [C -* A (2236), +A (2262), -A (2265), +T (2268)], the <br /> dimethyl sulfoxide. DNA denatured 94&#194;&#176;C 2 min       Ukraine chromosome [+TA (2015), +ATCCTA (2032), C <br /> then quickly frozen dry ice. Labeling reactions per-      -* T (2041)], en1 chromosome [+TA (2015) C -&gt; A <br /> formed Sequenase according manufacturer&apos;s (<span id='am-3' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span>-      (2236)]. No sequence differences engrailed 3&apos; untranslated <br /> ed States Biochemical) instructions following modifi-    region WT strains. No sequence <br /> cations. Labeling reactions included 10% dimethyl sulfoxide       differences 200-residue 3&apos; untranslated region <br /> and terminated 2 min 5 ,ul ofddC containing 10o   invected strains. However, comparisons se- <br /> dimethyl sulfoxide. The invected PCR product crosses      quences invected exon 1 revealed differences Brazil <br /> en49, en63, en59, en2 WT sequenced     chromosome [T -- C (1161), C -* T (1341), C -) T (1347)] <br /> primer 5&apos;-TCCCAGACCCAATGCCGTGGC-3&apos; (nt 1088-                      CyO, Bc, en49, en63, en59, en2 chromosomes [C <br /> 1108). PCR products crosses en-52, en61, en67    -- T (826 834)]. <br />  12698     Genetics: Goldsborough Kornberg                                                     Proc. Natl. Acad. Sci. USA 91              (1994) <br />                                                                                   c                                         r- <br />                                                                                                                             &quot;I <br />                                                                                    Ul~                                         c <br />                                                                                   ON      CI              I <br />  <br />                                                              CIA <br />                                                                                           r_                                   I <br />                                                           -&apos;1 <br />           -60 kb                 -40                     -20                      0                           20                             40 <br />  <br />                                                          I <br />  <br />                                 invected                                  engrailed <br />    FIG. 2. Map engrailed/invected region. Genomic map (horizontal line) engrailed invected transcription units indicated <br /> by arrows. Intron/exon organization engrailed invected depicted, approximate locations 11 engrailed mutations (23). Scale <br /> is kilobases start site engrailed transcription <span id='am-164' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> designated 0. Sequence polymorphisms engrailed invected transcripts: Ukraine <br /> DNA additional base pairs (ATCCTA) 3&apos;-untranslated region nt 2032 2033 engrailed transcript. All M <br /> chromosomes sequence Canton-S region. Two single base pair polymorphisms (C -- T) exon invected <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> en19, en63, en9, en2 positions 826 834 allowed identification transcripts derived chromosomes. Transcripts from <br /> the Brazil chromosome identified single polymorphisms (C -* T) nt 1341 1347. <br />     To gauge proportion input mRNA correlates               genes heterozygous animals. engrailed mRNA isolated <br /> with sequenced product ASA, separate preparations              tissues instar larva engrailed is <br /> RNA isolated Oregon-R Ukraine flies copied                    expressed, including wing imaginal discs brains. In tissue <br /> into cDNA, amplified, spectrophotometrically quantified.                M/Ukraine (WT) heterozygous larvae, proportion <br /> These DNA preparations mixed various propor-                   product derived M chromosome varied consid- <br /> tions, amplified, sequenced. The ratio DNA ddG                erably manner dependent the <br /> bands position 2017, distinguishes Oregon-R-                   particular mutant tissue type (Fig. 4). In wing discs, <br /> Ukraine-derived transcripts, revealed excellent correlation              mutants lethal breakpoints 20 kb immediately <br /> between ratio input cDNA ratio amplified                  upstream engrailed transcription start (en42, en49, en63, <br /> product (Fig. 3). Furthermore, genomic DNA isolated from <br /> Oregon-R/Ukraine heterozygotes yielded equivalent amounts                                                      1        2            3   4       5 <br /> of sequenced products PCR amplification (not shown). <br /> Additional controls exploited polymorphic differences be-                                            Uk    .   _             _ <br /> tween 3&apos; untranslated regions CyO, Oregon-R,                                             O-R    _                     _   -   - <br /> Ukraine engrailed transcripts demonstrate propor-                                                      *O~ .:III <br />  <br />  <br />  <br /> tion amplified product equivalent type heterozy-                           1.5 <br /> gote fly (for Oregon-R/Ukraine, Fig. 4; shown). <br /> We conclude WT engrailed genes Oregon-R, CyO,                           U <br /> and Ukraine flies produced equivalent amounts transcript                                1.0 <br /> and ASA method accurately assesses relative                              - <br />  <br />  <br />  <br />  <br /> abundance initial template.                                                     z <br />    To demonstrate utility ASA quantifying relative                              0.5 <br /> levels transcripts, various mutations regulatory                        v <br />                                                                                   ce <br /> region engrailed analyzed. Although engrailed                         0) <br />                                                                                   u <br /> transcription <span id='am-165' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> relatively small (approximately 3.7 kb),                     _g. <br />                                                                                              0 <br /> it contained large regulatory region                     54 <br /> defined mutations truncate gene far 15 kb                       C4 <br /> downstream 45 kb upstream transcription unit (12).                               0.5 <br /> These mutants recessive and, exception en61, <br /> en67, en1, en3, embryonic lethal hemizygous <br /> or homozygous. Since developmental program en-                                     -1.0 *- <br /> grailed expression includes spatially restricted domains                                  -1.0    -0.5         0          0.5                 -1 .0 <br /> developmental stages, know                                                   log (ORIUK) imput template <br /> various gene truncations alter spatially temporally reg- <br /> ulated expression. The early developmental lethality associ-                  FIG. 3. Quantitative ASA different ratios Ukraine and <br /> ated mutants, however, precluded anal-                 Oregon-R engrailed cDNAs. cDNA prepared homozygous <br /> ysis later stages. ASA provides convenient way study                Ukraine Oregon-R larval tissue amplified, quantified by <br /> these lethal mutations, M transcripts distin-                absorbance 260 nm, mnixed various proportions, reamplified, and <br /> guished tissue obtained phenotypically normal, viable               sequenced. (Inset) The region sequencing gel includes ddG <br /> heterozygotes.                                                              2017 reveals 6-bp polymorphic difference two <br />    A representative set engrailed mutants (en52, en42, en49,             strains (see legend Fig. 2). Lane 1, 10%o Ukraine:90%o Oregon-R; <br />                                                                             lane 2, 30% Ukraine:70%o Oregon-R; lane 3, 50% Ukraine:50% <br /> en63, en50, en&apos;, en2, en24, en61, en67, en3) selected           Oregon-R; lane 4, 70%6 Ukraine:30%o Oregon-R; lane 5, 90%o <br /> analysis. These mutants lack 6-bp insertion present              Ukraine:10% Oregon-R. Graphical representation data shows <br /> Ukraine WT engrailed gene, ASA used compare                   linear relationship ratio input cDNA (OR/UK) to <br /> relative proportion engrailed RNA produced M                      ratio OR/UK product. <br />                Genetics: Goldsborough Komberg                                                      Proc. Natl. Acad. Sci. USA 91 (1994)          12699 <br />  <br />   A WT                                                                      B     WT             -.m. <br />  <br />          M      };   -                                                                 M              am*                                           *=::.qft <br />  <br />  <br />  <br />  <br />         250                                                                       150 <br />  <br />  <br />   (1) 200 <br />   a <br />                                                                                   100 <br />   -o <br />   .D 150 <br />  <br />         100 <br />   0)                                                                               50 <br />   c) <br />    a) <br />   0-    50 <br />  <br />          0                                                                             0 <br />                en52 en42 en49 en63 en5O en2 en24 en67 en3          O-R                          en52 en42 en49 en63 en5O en2 en24 en67 en3 O-R <br />                               Chromosome                                                                         Chromosome <br />   FIG. 4. Quantitative analysis engrailed expression heterozygous larval tissues. The relevant portion sequencing gel pictured <br /> above graphic representation data. The lanes arranged left right proximal distal order positions mutant <br /> breakpoints. Relative levels expression shown percentage WT (100%). (A) Wing imaginal discs (approximately discs per <br /> sample). (B) Larval brain (approximately brains sample). Each bar represents average independent preparations. The <br /> values obtained multiple preparations highly reproducible. Expression engrailed WT Oregon-R (O-R) chromosome is <br /> shown equal Ukraine (100%o 6%). The horizontal broken lines graph represent WT levels engrailed expression. <br />                                                  &#194;&#177; <br />  <br />  <br />  <br /> en50, en2, en24) reduced expression M chromo-                      changing number engrailed invected transcripts <br /> some 30% WT. In contrast, nonlethal                         produced mutant chromosomes illustrates its <br /> breakpoint mutation, en3, located 45 kb upstream,                    utility sensitivity. Unlike quantitative PCR meth- <br /> reduced expression wing disc expression                      ods, ASA requires careful calibration PCR <br /> 10%. Curiously, en52, lethal breakpoint 15 kb                     amplification reaction quantitative restriction enzyme <br /> downstream engrailed transcription unit, 2-fold                    digestion (14, 15). Rather, method, WT transcripts <br /> greater expression M chromosome. Values obtained                     serve internal reference expression M <br /> from multiple preparations highly reproducible.                          gene. Since ASA amplifies transcripts WT and <br />    Preparations larval brain (Fig. 4B) yielded quantita-                 M chromosomes together, using set primers, no <br /> tively different results. The alleles breakpoints                 calibration standard necessary. This simplifies and <br /> region immediately upstream engrailed transcription                    improves reproducibility accuracy analysis. In <br /> start site (en42, en49, en63) marginal effects            practice, necessary calibrate sequencing <br /> brain. In contrast, distal breakpoint mutations en50, <br /> en2, en24 increasingly severe effects expression                                100 - <br /> levels. This inverse relationship <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> 5&apos; <br /> regulatory region level expression suggests the <br /> sequence elements regulating transcription larval brain                     0L)      80    - <br />  <br /> are numerous varied function. <br />   Located approximately 20 kb downstream engrailed <br /> transcription unit gene, invected, developmental                        CL <br />                                                                                             60    - <br />                                                                                    30 <br /> program expression virtually identical en- <br /> grailed (Fig. 2A). invected encodes protein closely <br /> related engrailed protein (13). We analyzed invected                                 40 - <br />                                                                                             j <br />                                                                                    cJ <br /> transcripts heterozygous engrailed mutants determine <br /> whether gene truncations engrailed regulatory region <br />                                                                                   0~ <br />                                                                                    a) <br />                                                                                             0 <br />                                                                                             9 <br />                                                                                                              T <br /> also affect invected expression. RNA isolated hetero- <br /> zygous larval wing imaginal discs used ASA taking <br /> advantage polymorphisms exon invected to <br /> distinguish WT M chromosome transcripts (Fig. 2C).                                               en52        en63   en59   en2   en6l   en67 <br /> Quantitative sequence analysis revealed engrailed                                                          Chromosome <br /> mutants substantially alter invected expression (Fig. 5). All <br /> the mutants reduced expression, 50% WT levels en52                      FIG. 5. Quantitative analysis invected expression hetero- <br /> and en67 heterozygotes, 15% WT en63                    zygous   wing imaginal discs. Graphic representation ASA from <br /> and en59 heterozygotes.                                                       heterozygous engrailed Ms reveals invected expression depends <br />                                                                               integrity regulatory region surrounding engrailed <br />                                                                               gene. The exon invected amplified PCR primers <br />                           DISCUSSION                                          flank sequence polymorphisms. The lanes arranged left <br />                                                                               right proximal distal order positions mutant <br /> We shown ASA used accurately quantify                     breakpoints. Relative levels expression shown percentage <br /> the relative abundance particular transcripts derived                 WT (100%). Expression invected WT Oregon-R <br /> multiple gene copies diploid nucleus. Application ASA                 chromo <br /> </body></html>