<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                    PubMed Central CANADA <br /> PMC Canada Author Manuscript <br />  <br />                                     Author Manuscript / Manuscrit d&apos;auteur <br />                                     Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                Published final edited form as: <br />                                 Mol Microbiol. 2010 March ; 75(5): 1182&#226;&#8364;&#8220;1198. doi:10.1111/j.1365-2958.2009.07038.x. <br />  <br />  <br />  <br />                                Forward genetics Candida albicans reveals Arp2/3 <br />                                complex required hyphal formation, endocytosis <br />                                Elias Epp1,2, Andrea Walther3, L&#195;&#169;pine Guylaine4, Zully Leon1, Alaka Mullick1, Martine <br />                                Raymond4, J&#195;&#188;rgen Wendland3, Malcolm Whiteway1,2,* <br />                                1Biotechnology       Research Institute, National Research Council Canada, Montr&#195;&#169;al, QC H4P <br />                                2R2, Canada <br />                                2Department       Biology, McGill University, Montr&#195;&#169;al, QC H3A 1B1, Canada <br />                                3Yeast    Biology, Carlsberg Laboratory, Gamle Carlsberg Vej 10, Valby Copenhagen, Denmark <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                4Institut <br />                                       Recherche en Immunologie et en Canc&#195;&#169;rologie (IRIC), Universit&#195;&#169; Montr&#195;&#169;al, <br />                                Montr&#195;&#169;al, QC H3C 3J7, Canada <br />  <br />                                Summary <br />                                     Candida albicans diploid fungal pathogen lacking defined complete sexual cycle, thus <br />                                     refractory standard forward genetic analysis. Instead, transcription profiling and <br />                                     reverse genetic strategies based Saccharomyces cerevisiae typically used link <br />                                     genes functions. <span id='am-9' about='protege:TO' typeof='owl:Thing'>To</span> overcome restrictions inherent indirect approaches, have <br />                                     investigated forward genetic mutagenesis strategy based UAU1 technology. We screened <br />                                     4700 random insertion mutants defects hyphal development linked new genes <br />                                     (ARP2 VPS52) hyphal growth. Deleting ARP2 abolished hyphal formation, generated round <br />                                     swollen yeast phase cells, disrupted cortical actin patches blocked virulence mice. The <br />                                     mutants showed global lack induction hyphae-specific genes yeast-to-hyphae <br />                                     switch. Surprisingly, arp2&#206;&#8221;/&#206;&#8221; arp2&#206;&#8221;/&#206;&#8221;arp3&#206;&#8221;/&#206;&#8221; mutants able endocytose <br />                                     FM4-64 Lucifer Yellow, shown time-lapse movies internalization FM4-64 <br />                                     somewhat delayed mutant cells. Thus non-essential role Arp2/3 complex <br />                                     discovered forward genetic screening C. albicans showed uptake membrane <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                     components plasma membrane vacuolar structures dependent actin <br />                                     nucleating machinery. <br />  <br />  <br />                                Introduction <br />                                                  Candida albicans leading causes hospital-acquired mycosis an <br />                                                  estimated mortality rate 38&#226;&#8364;&#8220;49% (Pfaller Diekema, 2007; Koh et al., 2008; Leroy et <br />  <br />  <br />  <br />                                &#194;&#169; 2010 Crown right Canada <br />                                * <br />                                For correspondence. malcolm.whiteway@cnrc-nrc.gc.ca; Tel. (+1) 514 496 6146; Fax (+1) 514 496 6213. <br />                                Supporting information <br />                                Additional supporting information online version article: <br />                                Please note: Wiley-Blackwell responsible content functionality supporting materials supplied authors. <br />                                Any queries (other missing material) directed corresponding author article. <br />                                 Epp et al.                                                                                          Page 2 <br /> PMC Canada Author Manuscript <br />  <br />  <br />                                             al., 2009). Although normal commensal gastrointestinal tracts mouths <br />                                             70% healthy human population, C. albicans life-threatening the <br />                                             increasing population immunocompromized individuals result conditions such <br />                                             HIV infections organ transplatation patients undergoing broad-spectrum- <br />                                             antibiotic chemotherapy treatments (Ruhnke Maschmeyer, 2002; Pfaller and <br />                                             Diekema, 2007). Identifying new genes involved virulence fungus remains a <br />                                             challenge, especially genetic manipulation functional characterization studies in <br />                                             C. albicans limited diploidy, absence true sexual cycle and <br />                                             pathogen&#226;&#8364;&#8482;s use non-canonical genetic <span id='am-6' about='Thesaurus:code' typeof='owl:Thing'>code</span> translates CUG serine instead <br />                                             leucine (Kurtz et al., 1988). <br />  <br />                                             The well-studied yeast Saccharomyces cerevisiae frequently used model to <br />                                             identify functions C. albicans. For instance, understanding morphogenesis, signal <br />                                             transduction, mating drug resistance S. cerevisiae led successful gene <br />                                             discovery subsequently functional analysis fungal pathogen (Berman and <br />                                             Sudbery, 2002; Casamayor Snyder, 2002; Schwartz Madhani, 2004; Bennett and <br />                                             Johnson, 2005; Berman, 2006; Whiteway Bachewich, 2007; Cowen, 2008). There are, <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             however, evident limitations using yeast define C. albicans processes. For example, <br />                                             clear homologues S. cerevisiae proteins identified C. albicans <br />                                             (Weig et al., 2004; Bennett Johnson, 2005; Braun et al., 2005). In cases a <br />                                             homologue S. cerevisiae gene identified C. albicans, functional <br />                                             conservation observed (Nicholls et al., 2004; Santos et al., 2004). Moreover, <br />                                             determining function based homology particularly challenging cases <br />                                             yeast model lacks process investigation. <br />  <br />                                             One example hyphal morphogenesis C. albicans. Although years <br />                                             researchers relied principles bud emergence S. cerevisiae attempt to <br />                                             understand hyphal formation filamentous fungi C. albicans, features of <br />                                             hyphal growth explained extrapolating findings model yeast (Harris <br />                                             Momany, 2004). Overall, reversible yeast-to-hyphal switch intensively <br />                                             studied polymorphic fungus (Liu, 2001; Whiteway Bachewich, 2007). Well- <br />                                             known environmental signals trigger morphological transition involve high <br />                                             temperature (37&#194;&#176;C), serum, neutral pH, starvation, CO2, adherence N-acetylglucosamine <br />                                             (GlcNAc) (Gow, 1997; Sudbery et al., 2004). Multiple pathways, instance cAMP <br />                                             protein kinase, MAP kinase pH-responsive pathways transmit signals to <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             activate expression hyphal-specific genes (Biswas et al., 2007). The importance the <br />                                             yeast-to-hyphae transition C. albicans highlighted implication virulence; C. <br />                                             albicans mutants unable switch yeast hyphal growth mode greatly <br />                                             reduced virulence mouse infection models (Lo et al., 1997; Laprade et al., 2002; <br />                                             Kumamoto Vinces, 2005). It proposed yeast form contributes to <br />                                             dissemination infection host, hyphal form facilitates penetrating tissue <br />                                             surfaces escaping host cell internalization (Gow et al., 2002; Whiteway Oberholzer, <br />                                             2004). <br />  <br />                                             With goal identifying new genes involved yeast-to-hyphae transition C. <br />                                             albicans, used unbiased approach randomly generating homozygous null <br />  <br />  <br />                                                  Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                               Page 3 <br /> PMC Canada Author Manuscript <br />  <br />  <br />                                               mutants directly pathogen. Screening 4700 random transposon insertion mutants <br />                                               identified new genes, ARP2 VPS52, required hyphal formation. <br />                                               Surprisingly, contrast organisms, highly conserved Arp2/3 complex <br />                                               essential viability endocytosis C. albicans, structural differences in <br />                                               actin organization support importance functional cytoskeletal architecture in <br />                                               permitting morphological switching. These findings underline potential forward <br />                                               genetics pathogen itself link functions genes. <br />  <br />  <br />                                   Results <br />                                   Strategy vivo random forward mutagenesis screen C. albicans <br />                                               The UAU1 marker cassette (Enloe et al., 2000) allows selection homozygosed <br />                                               mutants diploid organism C. albicans. This insertional mutagenesis strategy employs <br />                                               single transformation arg4/arg4, ura3/ura3 double auxotrophic background selecting <br />                                               initially Arg+/Ura&#226;&#710;&#8217; colonies (genotype orf::UAU1/ORF), allowing mitotic <br />                                               recombination homozygose insertion (genotype orf::UAU1/orf::UAU1), and <br />                                               following second round selection yielding Arg+/Ura+ segregants (genotype <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                               orf::UAU1/orf::URA3) potentially carry homozygous disruption site the <br />                                               initial insertion. Mitchell et al. (Davis et al., 2002) used UAU1 marker cassette to <br />                                               randomly mutagenize vitro genome C. albicans. This vitro random mutagenesis <br />                                               resulted pool plasmids carrying UAU1 marker cassette flanked specific <br />                                               C. albicans genomic DNA sequence. We used entire pool, chosen set of <br />                                               characterized UAU1 plasmids, randomly mutagenize genome C. albicans vivo, <br />                                               screened potentially homozygous orf::UAU1/orf::URA3 inserts <span id='am-88' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-89' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> <br />                                               interest. <br />  <br />                                               To assess possible extract desired phenotype using entire UAU1 <br />                                               plasmid pool, created preliminary set 300 orf::UAU1/orf::URA3 random insertion <br />                                               mutants screened specific auxotrophic mutants. We mutant was <br />                                               unable grow SD-Trp media (Fig. 1A). To map <span id='am-166' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-167' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>mutation</span> performed inverse <br />                                               PCR (Ochman et al., 1988) transposon insertion TRP1 (ORF19.6096), the <br />                                               phosphoribosylanthranilate isomerase gene. TRP1 essential enzyme tryptophan <br />                                               biosynthesis, mutating gene results tryptophan auxotrophy (Ostrander and <br />                                               Gorman, 1994). <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                               In attempt link genes challenging conserved phenotypes between <br />                                               fungal species, carbon source utilization (Martchenko et al., 2007a,b; Askew et al., <br />                                               2009), expanded number orf::UAU1/ORF insertions 5000. <span id='am-5' about='protege:FROM' typeof='owl:Thing'>From</span> we <br />                                               obtained 4700 (94%) generated Arg+Ura+ segregants represent potential <br />                                               orf::UAU1/orf::URA3 derivatives. Among insertion mutant (Tn- <br />                                               orf19.875) grow glycerol (Fig. 1B). A deletion mutant revertant strain <br />                                               confirmed link growth defects glycerol uncharacterized C. albicans <br />                                               gene obvious homologues budding fission yeast (Arnaud et al., 2007). <br />                                               Thus, preliminary screen auxotrophs mutants grow media <br />                                               containing glycerol sole carbon source showed possible directly link <br />                                               functions gene using pool UAU1 plasmids. <br />  <br />                                                    Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                             Page 4 <br /> PMC Canada Author Manuscript <br />  <br />  <br />                                   Screening random mutants involved hyphal formation <br />                                               In order link genes phenotype absent S. cerevisiae, phenotypically analysed <br />                                               4700 candidate homozygotes hyphae formation defects. No convincingly hyper- <br />                                               filamentous colonies identified. By contrast, isolated total 20 mutants a <br />                                               strong consistent hypo-filamentous phenotype confirmed phenotypes by <br />                                               retesting single colony mutant. Mapping 20 hits inverse PCR showed that <br />                                               11 insertions directly ORF, insertion mapped promoter of <br />                                               ORF19.860. In remaining events map insertion of <br />                                               repeated sequences (six events) fell intergenic region (two events). Mutants <br />                                               transposon insertion intergenic region excluded analysis <br />                                               unclear observed phenotype linked genes adjacent the <br />                                               transposon insertion. Figures 1C S1 summarize hypo-filamentation phenotypes 11 <br />                                               ORF promoter insertion mutant hits determined plate assays. To distinguish <br />                                               hypo-filamentation non-filamenting phenotypes, analysed 11 ORF and <br />                                               promoter hypo-filamentation insertion mutants liquid hyphal inducing conditions <br />                                               [Yeast extract, peptone, dextrose (YPD) media + 10% fetal bovine serum (FBS) 37&#194;&#176;C for <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                               3 h] insertions CDC39, VPS52 ARP2 failed form true hyphae in <br />                                               liquid. <br />  <br />                                   VPS52 ARP2 deletion mutants form hyphae <br />                                               CDC39 previously linked hypo-filamentation phenotype (Uhl et al., 2003), while <br />                                               ARP2 VPS52 newly identified mutants form hyphae liquid- <br />                                               inducing conditions. We validated transposon insertion hits both <br />                                               verifying absence WT band transposon mutant constructing deletion <br />                                               mutants (Fig. S2). To confirm ARP2 VPS52 mutants unable form true <br />                                               hyphae, re-analysed polarized morphogenesis hyphal inducing conditions liquid <br />                                               culture (Fig. 2). Tn::arp2 (the transposon mutant ARP2), arp2&#206;&#8221;/&#206;&#8221; (the deletion mutant <br />                                               ARP2), Tn::vps52 vps52&#206;&#8221;/&#206;&#8221; did form true hyphae, BWP17 (WT strain for <br />                                               transposon mutants) SN95 (WT strain deletion mutants), heterozygous <br />                                               mutants (arp2/ARP2, vps52/VPS52) ARP2 VPS52 revertants (arp2&#206;&#8221;/&#206;&#8221; + <br />                                               ARP2, vps52&#206;&#8221;/&#206;&#8221; + VPS52) did. <br />  <br />                                               To analyse polarized morphogenesis detail, stained strains calcofluor white <br />                                               (CFW) assess percentage cells growing yeast, pseudohyphae true <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                               hyphae (Fig. 2B). Cells septum/septin ring located growing bud tube <br />                                               showed constrictions septal junctions considered true hyphae (Sudbery et al., <br />                                               2004). Under conditions tested, 96% WT strain heterozygotes <br />                                               revertants growing true hyphae. On hand, true hyphal formation <br />                                               observed arp2&#206;&#8221;/&#206;&#8221; vps52&#206;&#8221;/&#206;&#8221; cells, 60% arp2&#206;&#8221;/&#206;&#8221; 77% of <br />                                               vps52&#206;&#8221;/&#206;&#8221; cells grow pseudohyphae 40% arp2&#206;&#8221;/&#206;&#8221; 23% of <br />                                               vps52&#206;&#8221;/&#206;&#8221; cells yeast state. <br />  <br />                                               Taken together, hyphal-deficient transposon insertion phenotypes ARP2 VPS52 <br />                                               confirmed deletion mutants reverting phenotypes reintroduction the <br />                                               WT genes. Comparative genome hybridization analysis showed aneuploidies <br />  <br />  <br />                                                    Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                             Page 5 <br /> PMC Canada Author Manuscript <br />  <br />  <br />                                               arp2&#206;&#8221;/&#206;&#8221; vps52&#206;&#8221;/&#206;&#8221; strains (Fig. S3). Because C. albicans vacuolar <br />                                               protein sorting (VPS)-type mutants vps11&#206;&#8221;/&#206;&#8221;, vps28&#206;&#8221;/&#206;&#8221;, vps32&#206;&#8221;/&#206;&#8221; the <br />                                               conditional vps1&#206;&#8221;/VPS1-tetR mutant showed hyphal formation defects similar vps52&#206;&#8221;/&#206;&#8221; <br />                                               cells (Palmer et al., 2003; Cornet et al., 2005; Bernardo et al., 2008), decided focus <br />                                               investigation Arp2/3 complex. <br />  <br />                                   C. albicans Arp2/3 complex mutants dramatic actin cytoskeleton defects, still <br />                                   able endocytose <br />                                               ARP2 encodes seven evolutionary conserved subunits Arp2/3 complex, which <br />                                               nucleates actin filaments (F-actin) branched networks (Machesky et al., 1994; Welch et <br />                                               al., 1997). Given conserved association actin regulation, asked deletion of <br />                                               ARP2 C. albicans affects actin cytoskeleton staining logarithmically growing yeast <br />                                               cells rhodamine/phalloidin (Fig. 3). In contrast WT stained cells actin patches <br />                                               observed bright dots localizing sites polarized growth, C. albicans arp2&#206;&#8221;/&#206;&#8221; <br />                                               mutant did distinct, bright actin patches. Instead, prominent actin <br />                                               structures formed arp2&#206;&#8221;/&#206;&#8221; cells large filamentous aggregates similar those <br />                                               observed conditional arp2 arp3 mutants S. cerevisiae bee1/las17 yeast <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                               mutants, lack actin patches (Li, 1997; Winter et al., 1997; Martin et al., 2005). <br />                                               On hand, actin-based structures actin cables cytokinetic rings <br />                                               observed arp2&#206;&#8221;/&#206;&#8221; cells (Fig. 3). When copy ARP2 gene was <br />                                               present (arp2&#206;&#8221;/&#206;&#8221; + ARP2), distinct actin patches observed hyphal formation was <br />                                               restored, patches appeared frequent compared WT. A double KO <br />                                               mutant, copies ARP2 ARP3 deleted (arp2&#206;&#8221;/&#206;&#8221;arp3&#206;&#8221;/&#206;&#8221;), <br />                                               showed similar morphological phenotypes arp2&#206;&#8221;/&#206;&#8221; single mutant. These mutants also <br />                                               exhibited hyphal formation defects, cells clumping grown logarithmic <br />                                               phase, individual cells appearing phenotypically round swollen wider bud <br />                                               neck similar actin cytoskeleton defects (Figs 2 3). <br />  <br />                                               Because assembly actin filaments mediated Arp2/3 complex shown be <br />                                               essential endocytosis wide range organisms (for reviews see: Goley and <br />                                               Welch, 2006; Kaksonen et al., 2006; Galletta Cooper, 2009), stained C. albicans <br />                                               Arp2/3 complex mutants lipophilic dye FM4-64, commonly used to <br />                                               visualize membrane internalization endocytotic delivery vacuole (Vida Emr, <br />                                               1995). Both arp2&#206;&#8221;/&#206;&#8221; arp2&#206;&#8221;/&#206;&#8221;arp3&#206;&#8221;/&#206;&#8221; mutants clearly able deliver FM4-64 to <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                               vacuole observed intracellular appearance dye 45 min chase <br />                                               period (Fig. 4A). In Arp2/3 mutant cells, however, vacuolar morphology appeared <br />                                               fragmented, resulted staining vacuole. While in <br />                                               WT cells typically vacuoles apparent, smaller vacuoles could <br />                                               observed Arp2/3 complex mutant cells. This fragmented vacuolar morphology also <br />                                               described C. albicans wal1&#206;&#8221;/&#206;&#8221; (LAS17 S. cerevisiae) mutants, activator the <br />                                               Arp2/3 complex (Walther Wendland, 2004). <br />  <br />                                               As S. cerevisiae conditional Arp2/3 complex mutants partially able endocytose <br />                                               FM4-64, LY (Lucifer Yellow dye taken fluid phase endocytosis) <br />                                               (Riezman, 1985; Moreau et al., 1996; 1997; Martin et al., 2005; Daugherty Goode, <br />  <br />  <br />                                                    Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                            Page 6 <br /> PMC Canada Author Manuscript <br />  <br />  <br />                                               2008), assessed LY uptake C. albicans Arp2/3 complex mutants. No difference LY <br />                                               uptake observed Arp2/3 mutants compared WT 90 min incubation <br />                                               (Fig. 4B). <br />  <br />                                               To quantitatively assess endocytosis, performed time-lapse microscopy included C. <br />                                               albicans myo5&#206;&#8221;/&#206;&#8221; mutant shown endocytosis-defective (Oberholzer et <br />                                               al., 2004). Co-incubation GFP-labelled WT C. albicans arp2&#206;&#8221;/&#206;&#8221;, arp2&#206;&#8221;/ <br />                                               &#206;&#8221;arp3&#206;&#8221;/&#206;&#8221; myo5&#206;&#8221;/&#206;&#8221; cells confirmed endocytosis occurred arp2&#206;&#8221;/&#206;&#8221; arp2&#206;&#8221;/ <br />                                               &#206;&#8221;arp3&#206;&#8221;/&#206;&#8221; mutants, myo5&#206;&#8221;/&#206;&#8221; cells did endocytose FM4-64 3 h (see movies <br />                                               1, 2, 3 4 Fig. S4, Supporting information). <br />                                               However, vacuolar staining somewhat delayed arp2&#206;&#8221;/&#206;&#8221; arp2&#206;&#8221;/&#206;&#8221;arp3&#206;&#8221;/&#206;&#8221; cells; <br />                                               FM4-64 staining appeared 20&#226;&#8364;&#8220;30 min WT cells, took 70 80 min <br />                                               dye reached vacuole arp2&#206;&#8221;/&#206;&#8221; arp2&#206;&#8221;/&#206;&#8221;arp3&#206;&#8221;/&#206;&#8221; mutants. In both <br />                                               arp2&#206;&#8221;/&#206;&#8221;, arp2&#206;&#8221;/&#206;&#8221;arp3&#206;&#8221;/&#206;&#8221; myo5&#206;&#8221;/&#206;&#8221; cells, dye initially appeared punctate-like <br />                                               dots apparently stuck plasma membrane. As dye-internalization occurred arp2&#206;&#8221;/&#206;&#8221; <br />                                               arp2&#206;&#8221;/&#206;&#8221;arp3&#206;&#8221;/&#206;&#8221; cells time, bright dots membrane slowly faded the <br />                                               time vacuolar staining began emerge. In myo5&#206;&#8221;/&#206;&#8221; cells, hand, bright <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                               dot-like structures remained cell membrane end observations no <br />                                               distinct vacuolar structures appearing. Taken together, results suggest Arp2/3 <br />                                               complex plays crucial role actin cytoskeleton organization C. albicans, while <br />                                               clearly delayed endocytosis, Arp2/3 complex mutants endocytose assessed by <br />                                               FM4-64 LY uptake. <br />  <br />                                   Transcriptional analysis Arp2/3 complex mutants <br />                                               To gain insights cellular processes affected disrupting Arp2/3 complex functions, we <br />                                               performed transcriptional profiling yeast growth conditions (YPD 30&#194;&#176;C 3 h) or <br />                                               hyphal induction (YPD + 10% FBS 37&#194;&#176;C 3 h) compared transcriptional <br />                                               consequences deleting ARP2 MYO5 SLA2 microarray data sets (Oberholzer et al., <br />                                               2006). MYO5 Arp2/3 complex activator, SLA2 actin binding protein that <br />                                               couples actin vesicle coat endocytosis (Robertson et al., 2009a). Both sla2&#206;&#8221;/&#206;&#8221; <br />                                               myo5&#206;&#8221;/&#206;&#8221; C. albicans mutants suffer similar related complications, hyphal <br />                                               formation, delocalized actin patches endocytosis defects (Asleson et al., 2001; <br />                                               Oberholzer et al., 2002; 2004). Despite using different chip platforms different WT <br />                                               strains, good overall correlation ARP2 microaray data set MYO5 <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                               SLA2 data sets (compare &#226;&#8364;&#732;HY Epp&#226;&#8364;&#8482; &#226;&#8364;&#732;HY Oberholzer&#226;&#8364;&#8482; Fig. 5A, Table S1 shows <br />                                               numerical values correlation coefficient). When significantly regulated genes (more <br />                                               twofold, P &lt; 0.05) compared, deleting ARP2, MYO5 SLA2 resulted similar <br />                                               cellular responses, overlap regulated genes significant under <br />                                               yeast compared hyphal growth conditions (Fig. 5B C). To compare the <br />                                               transcriptional response arp2&#206;&#8221;/&#206;&#8221; mutants hyphal induction, hyphae-specific genes, <br />                                               i.e. genes significantly regulated (more twofold, P &lt; 0.05) WT-HY <br />                                               comparison, clustered Fig. 6. Many genes list known become <br />                                               highly induced activation hyphal programme, example ECE1, ALS3, HYR1, <br />                                               SAP5, SAP6, HWP1, RBT8, IHD1, PST1, CIP1, DCK1 ORF19.1691 (Nantel et al., <br />                                               2002; Garcia-Sanchez et al., 2005; Kadosh Johnson, 2005) (Table S2). Deleting ARP2 <br />  <br />  <br />                                                    Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                            Page 7 <br /> PMC Canada Author Manuscript <br />  <br />  <br />                                             resulted global lack hyphal-specific gene induction (arp2-H Fig. 6). When arp2-Y <br />                                             compared directly arp2-H, observation improper hyphae-specific gene <br />                                             induction confirmed &gt; 92% 35 38 highly upregulated (more <br />                                             fourfold) hyphae-specific genes twofold induced absence of <br />                                             ARP2 &gt; 95% 75 79 remaining upregulated hyphae-specific genes were <br />                                             induced arp2&#206;&#8221;/&#206;&#8221; mutants compared WT cells (Fig. 6, Table S2). Deleting myo5&#206;&#8221;/&#206;&#8221; <br />                                             resulted comparable response hyphae-specific genes properly <br />                                             induced (Oberholzer et al., 2006). However, lack proper gene induction much <br />                                             pronounced absence ARP2 absence MYO5 (compare arp2-H vs. <br />                                             myo5-H Fig. 6, Table S3). Together, results suggest ARP2 profile <br />                                             showed significant similarities MYO5 SLA2 profiles yeast growth <br />                                             conditions, correlation hyphal growth conditions possibly of <br />                                             pronounced lack proper hyphal-specific gene induction absence ARP2. <br />  <br />                                             These observations suggest failure hyphal growth Arp2/3 complex mutants could <br />                                             result impaired endocytosis, problems actin cytoskeleton, failure to <br />                                             activate hyphal-specific genes combination defects. If hyphal defect <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             primarily failure activate gene expression, derepressing hyphal-specific gene <br />                                             expression deleting NRG1 repressor potentially suppress defect, deletion <br />                                             NRG1 leads constitutive filamentous growth absence hyphal <br />                                             induction signals (Garcia-Sanchez et al., 2005; Kadosh Johnson, 2005). We created an <br />                                             nrg1&#206;&#8221;/&#206;&#8221;arp2&#206;&#8221;/&#206;&#8221; mutant, exhibited doubling time twice long WT <br />                                             (Fig. S5). When grown non-inducing conditions, nrg1&#206;&#8221;/&#206;&#8221;arp2&#206;&#8221;/&#206;&#8221; cells showed the <br />                                             arp2&#206;&#8221;/&#206;&#8221; mutant morphology round swollen cells. When induced hyphal growth, <br />                                             nrg1&#206;&#8221;/&#206;&#8221;arp2&#206;&#8221;/&#206;&#8221; cells exhibited arp2&#206;&#8221;/&#206;&#8221; cell morphology did form hyphae <br />                                             extended overnight incubation times (Fig. 7). We attempted create a <br />                                             tup1&#206;&#8221;/&#206;&#8221;arp2&#206;&#8221;/&#206;&#8221; mutant, successful. <br />  <br />                                             To determine hyphal-specific genes derepressed nrg1&#206;&#8221;/&#206;&#8221;arp2&#206;&#8221;/&#206;&#8221; mutant, <br />                                             performed transcript profiling. We compared nrg1&#206;&#8221;/&#206;&#8221;arp2&#206;&#8221;/&#206;&#8221; mutant grown under <br />                                             hyphal conditions arp2&#206;&#8221;/&#206;&#8221; mutant grown conditions (YPD + 10% <br />                                             serum, 37&#194;&#176;C, 3 h) significant number hyphal-specific genes are <br />                                             normally induced WT cells undergoing yeast-to-hyphae switch (WT-HY) <br />                                             showed greater expression nrg1&#206;&#8221;/&#206;&#8221;arp2&#206;&#8221;/&#206;&#8221; mutant compared arp2&#206;&#8221;/&#206;&#8221; cells (P- <br />                                             value 4.9 &#195;&#8212; 10&#226;&#710;&#8217;9). When examined set NRG1-dependent hyphal-specific genes <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             previously identified (Kadosh Johnson, 2005), seven 28 genes (HYR1, <br />                                             SAP5, SAP4, KIP4, ORF19.6079, ALS3 UME6) showed significantly increased <br />                                             expression (twofold more) nrg1&#206;&#8221;/&#206;&#8221;arp2&#206;&#8221;/&#206;&#8221; cells compared arp2&#206;&#8221;/&#206;&#8221; cells, a <br />                                             genes (IHD1, CBP1, ORF19.6705 ALS10) showed moderately increased <br />                                             expression 1.5- 2-fold (Table S4). Thus, deleting transcriptional <br />                                             repressor filamentation programme leads derepression hyphal genes, the <br />                                             entire regulated gene set derepressed; presumably reflects complex interplay <br />                                             different transcriptional (co-)repressors exert yeast-to-hyphae transition (Garcia- <br />                                             Sanchez et al., 2005; Kadosh Johnson, 2005). We despite the <br />                                             increased induction hyphal genes nrg1&#206;&#8221;/&#206;&#8221;arp2&#206;&#8221;/&#206;&#8221; mutant, those <br />  <br />  <br />  <br />                                                  Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                             Page 8 <br /> PMC Canada Author Manuscript <br />  <br />  <br />                                               genes highly induced WT cells (Table S4). One gene induced in <br />                                               &#226;&#8364;&#732;nrg1&#206;&#8221;/&#206;&#8221;arp2&#206;&#8221;/&#206;&#8221; vs arp2&#206;&#8221;/&#206;&#8221;&#226;&#8364;&#8482; &#226;&#8364;&#732;nrg1&#206;&#8221;/&#206;&#8221;arp2&#206;&#8221;/&#206;&#8221; vs WT&#226;&#8364;&#8482; comparisons is <br />                                               UME6, recently identified key regulator hyphal programme (Banerjee et al., 2008; <br />                                               Zeidler et al., 2009). Interestingly, constitutive overexpression UME6 WT <br />                                               cells resulted constitutive filamentous growth absence hyphae signals <br />                                               (Carlisle et al., 2009), increased expression level UME6 nrg1&#206;&#8221;/&#206;&#8221;arp2&#206;&#8221;/&#206;&#8221; <br />                                               mutant sufficient restore filamentation absence functional Arp2/3 <br />                                               complex. Thus despite partial derepression hyphal programme, hyphae form, <br />                                               making likely roles Arp2/3 complex, function actin patch <br />                                               formation actin branching, required hyphal development. <br />  <br />                                   Confirming &#226;&#8364;&#732;actin-patch&#226;&#8364;&#8482; phenotypes <br />                                               Besides defects filamentous growth, phenotypes myo5&#206;&#8221;/&#206;&#8221; sla2&#206;&#8221;/&#206;&#8221; <br />                                               mutants linked Arp2/3 complex include cell membrane cell wall <br />                                               defects salt sensitivity (Oberholzer et al., 2006). We tested actin <br />                                               patch-associated phenotypes observed Arp2/3 complex mutants. arp2&#206;&#8221;/&#206;&#8221; and <br />                                               arp2&#206;&#8221;/&#206;&#8221;arp3&#206;&#8221;/&#206;&#8221; cells showed typical actin-patch phenotypes salt sensitivity well <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                               cell wall cell membrane defects, illustrated increased sensitivity congo red, <br />                                               CFW, SDS hygromycin B. Arp2/3 complex mutants showed abnormal cell wall <br />                                               patterning aberrant, relatively random chitin deposition (Fig. 8). <br />  <br />                                               Transcriptional analysis showed ergosterol genes (e.g. ERG1, ERG5, ERG6, <br />                                               ERG10, ERG11, ERG27 ERG252) downregulated arp2&#206;&#8221;/&#206;&#8221; cells were <br />                                               compared WT cells yeast hyphae condition (Table S5). We reasoned that <br />                                               Arp2/3 complex mutants sensitive drugs targeting important <br />                                               component fungal cell membranes. Figure 9 illustrates arp2&#206;&#8221;/&#206;&#8221; arp2&#206;&#8221;/ <br />                                               &#206;&#8221;arp3&#206;&#8221;/&#206;&#8221; mutants showed increased sensitivity ergosterol targeting drug fluconazole, <br />                                               phenotype described mob2&#206;&#8221;/&#206;&#8221; cells, key component RAM <br />                                               pathway showed additional related actin-patch phenotypes (Song et al., 2008). Taken <br />                                               together, actin patch-associated phenotypes previously described myo5&#206;&#8221;/&#206;&#8221;, <br />                                               sla2&#206;&#8221;/&#206;&#8221;, wal1&#206;&#8221;/&#206;&#8221; mob2&#206;&#8221;/&#206;&#8221; cells confirmed Arp2/3 complex mutants. <br />  <br />                                   A functional Arp2/3 complex required virulence <br />                                               Because yeast-to-hyphae switch important virulence attribute, tested the <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                               arp2&#206;&#8221;/&#206;&#8221; mutant fungal replication complement-5 (C5)-deficient mouse model of <br />                                               disseminated candidiasis (Mullick et al., 2004; 2006; Tuite et al., 2005). Mice infected with <br />                                               WT strain SN95 moribund 24 h post infection, mice infected with <br />                                               arp2&#206;&#8221;/&#206;&#8221; cells did clinical signs lethargy, ruffled fur hunched back <br />                                               day 4 post infection (Fig. 10A). This observation confirmed measuring <br />                                               fungal load kidney, site highest fungal replication A/J mouse model <br />                                               (Mullick et al., 2004). WT-infected mice significantly higher fungal burden 24 h <br />                                               post infection compared arp2&#206;&#8221;/&#206;&#8221;-infected mice sacrificed time 4 <br />                                               days post infection hardly fungal cells recovered kidneys mice <br />                                               infected arp2&#206;&#8221;/&#206;&#8221; mutants (Fig. 10B). <br />  <br />  <br />  <br />                                                    Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                              Page 9 <br /> PMC Canada Author Manuscript <br />  <br />  <br />                                               To gain insights host&#226;&#8364;&#8220;pathogen interaction arp2&#206;&#8221;/&#206;&#8221; cells trigger <br />                                               host response despite reduced capacity replicate A/J mice background, we <br />                                               biochemically analysed host blood collected heart puncture 24 h post infection. We <br />                                               focused metabolic markers typically specific response infection <br />                                               C. albicans (Mullick et al., 2006): levels interleukin 6 (IL-6), key inflammatory <br />                                               cytokine, creatine kinase (CK), cardiac protein, highly <br />                                               upregulated encounter C. albicans. The blood arp2&#206;&#8221;/&#206;&#8221; mutant-infected mice <br />                                               significantly lower levels IL-6 compared WT-infected mice (Fig. 10C). <br />                                               Likewise, significantly lower amounts CK blood collected mice <br />                                               infected arp2&#206;&#8221;/&#206;&#8221; compared WT-infected mice (Fig. 10D). These results <br />                                               indicate arp2&#206;&#8221;/&#206;&#8221; cells trigger normal host response determined IL-6 and <br />                                               CK levels. Taken together, based <span id='am-10' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-11' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>survival</span> time, fungal burden biochemical <br />                                               analysis host blood, conclude C. albicans arp2&#206;&#8221;/&#206;&#8221; mutants avirulent in <br />                                               mouse model infection. <br />  <br />  <br />                                   Discussion <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                               While complete sequence C. albicans genome (assembly 20 available in <br />                                               2006), combination molecular tools epitope-tagging, inducible promoters, <br />                                               reporter genes, auxotrophic dominant markers, tremendously facilitated functional <br />                                               analysis C. albicans (Care et al., 1999; Reuss et al., 2004; Nantel, 2006; Schaub et al., <br />                                               2006; Lavoie et al., 2008), comprehensive genome-wide collection C. albicans mutants <br />                                               available, clearly desirable functional genomics. The inherent difficulty <br />                                               genetic manipulation C. albicans resulted large-scale mutagenesis <br />                                               efforts capable directly linking gene cellular function (Roemer et al., 2003; Uhl et <br />                                               al., 2003; Nobile Mitchell, 2005; Bruno et al., 2006; Shen et al., 2008). Here, have <br />                                               investigated forward genetics approach C. albicans. Screening 4700 insertion <br />                                               mutants showed VPS52 ARP2 essential yeast-to-hyphae transition, <br />                                               demonstrating approach successfully link genes phenotype C. <br />                                               albicans. <br />  <br />                                   Advantages forward genetics C. albicans <br />                                               Given mutating seven Arp2/3 complex subunits resulted severe growth <br />                                               defects lethality S. cerevisiae, Schizosaccharomyces pombe, Drosophila melanogaster <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Caenorhbditis elegans (Lees-Miller et al., 1992; Schwob Martin, 1992; <br />                                               Balasubramanian et al., 1996; Mccollum et al., 1996; Winter et al., 1997; 1999; Hudson and <br />                                               Cooley, 2002; Zallen et al., 2002; Sawa et al., 2003), surprised C. albicans <br />                                               loss Arp2/3 complex subunits does severely compromise viability (Fig. S5). <br />                                               Using reverse genetics relying yeast models led incorrect <br />                                               assumptions Arp2/3 complex essential C. albicans have <br />                                               escaped functional characterization. Functional analysis based alternate models would <br />                                               proven challenging linking ORF19.875, uncharacterized gene C. albicans <br />                                               obvious homologues S. cerevisiae S. pombe (Arnaud et al., 2007), glycerol <br />                                               growth defect phenotype (Fig. 1B). These examples demonstrate utility unbiased <br />                                               large-scale mutagenesis directly C. albicans. <br />  <br />  <br />                                                    Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                            Page 10 <br /> PMC Canada Author Manuscript <br />  <br />  <br />                                   Limitations forward genetics C. albicans <br />                                               Why did new robust filamentation phenotypes result 4700 potentially <br />                                               homozygosed orf::UAU1/orf::URA3 mutants? First, 50% insertions were <br />                                               directly ORF putative promoter region (see Supporting information). For our <br />                                               analysis, restricted in-depth characterization deletion mutant construction of <br />                                               orf::UAU1/orf::URA3 mutants ORF insertions, showed consistent <br />                                               pronounced phenotypes. Second, insertion mutants aneuploid, consistent <br />                                               previous findings directed UAU1 plasmids used (Enloe et al., 2000), or <br />                                               showed major chromosomal rearrangements chromosomes initial <br />                                               UAU1 marker inserted (Figs S2 S3). Finally, exclude multiple insertions <br />                                               transformant, explain map insertions. As well, <br />                                               insertions repeated regions difficult map. For instance, several <br />                                               interesting phenotypes mapped genes TLO (TeLOmere-associated gene) family, <br />                                               genes known contain 80% similarity (van het Hoog et al., <br />                                               2007) conveniently map insertions single gene. <br />  <br />                                               Taken together, based random selection verification WT band, speculate <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                               4700 potentially homozygosed orf::UAU1/orf::URA3 mutants, 5% or <br />                                               200 simple homozygous gene insertion mutants aneuploidy site of <br />                                               insertion. This make overall isolation rate hyphal defects close 100 <br />                                               genes inactivated. <br />  <br />                                   Exploiting non-essential function Arp2/3 complex C. albicans <br />                                               The actin cytoskeleton plays crucial role cell polarity eukaryotes hyphal <br />                                               growth fungal species particular (Pruyne Bretscher, 2000; Steinberg, 2007). <br />                                               In C. albicans, instance, importance actin cytoskeleton structure dynamics to <br />                                               morphogenesis shown experiments C. albicans cells treated Cytochalasin <br />                                               A, actin monomer sequestering drug, led hyphal formation defects preserving <br />                                               cellular growth processes (Akashi et al., 1994). Subsequent work demonstrated link <br />                                               actin cytoskeleton stability hyphae-specific gene expression, observations that <br />                                               corroborate findings disrupting Arp2/3 complex prevented morphological <br />                                               switching hyphal-specific genes properly induced. This consistent <br />                                               idea complex transcriptional regulatory actin cytoskeleton <br />                                               itself influence hyphal gene activation (Hazan Liu, 2002; Wolyniak and <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Sundstrom, 2007), potentially cAMP signalling (Zou et al., 2009). However, our <br />                                               data partial derepression hyphal programme does restore hyphae <br />                                               formation absence functional Arp2/3 complex, suggesting Arp2/3 complex <br />                                               mutants, additional factors involved. <br />  <br />                                               Several actin-patch phenotypes previously described C. albicans myo5&#206;&#8221;/&#206;&#8221;, wal1&#206;&#8221;/&#206;&#8221;, <br />                                               sla2&#206;&#8221;/&#206;&#8221; RAM pathway mutants confirmed arp2&#206;&#8221;/&#206;&#8221; arp2&#206;&#8221;/&#206;&#8221;arp3&#206;&#8221;/&#206;&#8221; <br />                                               mutants. While link Arp2/3 complex Arp2/3 complex activators, <br />                                               WAL1 MYO5, SLA2, actin binding protein, obvious, remains <br />                                               speculative RAM pathway influence actin dynamics general the <br />                                               Arp2/3 complex particular. Of phenotypes displayed Arp2/3 complex mutants, the <br />  <br />  <br />                                                    Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                          Page 11 <br /> PMC Canada Author Manuscript <br />  <br />  <br />                                             surprising ability endocytose widely accepted actin patches <br />                                             major sites endocytosis yeast mammals (Kaksonen et al., 2006; Moseley <br />                                             Goode, 2006; Smythe Ayscough, 2006; Galletta Cooper, 2009). In general, <br />                                             delayed, arp2&#206;&#8221;/&#206;&#8221; arp2&#206;&#8221;/&#206;&#8221;arp3&#206;&#8221;/&#206;&#8221; cells clearly able endocytose LY <br />                                             FM4-64, case Arp2/3 complex activator Myo5p (compare <br />                                             movie 2 movie 3, Supporting information), suggesting myo5&#206;&#8221;/&#206;&#8221; cells Arp2/3 <br />                                             complex-independent, MYO5-dependent pathways exist contribute plasma membrane <br />                                             component uptake. Together microarray studies showed degree of <br />                                             similarity transcriptional response arp2&#206;&#8221;/&#206;&#8221; myo5&#206;&#8221;/&#206;&#8221; mutants condition- <br />                                             dependent, supports previous conclusions (Oberholzer et al., 2006) myo5&#206;&#8221;/&#206;&#8221; <br />                                             phenotypes Arp2/3 complex-independent. <br />  <br />                                             The process endocytosis divided phases (Robertson et al., 2009a). During <br />                                             earliest phase, assembly endocytic coat complex takes place involves cargo <br />                                             recruitment, initial membrane curvature facilitate invagination process, as <br />                                             recruitment proteins trigger actin polarization. This initial process seems <br />                                             unaffected Arp2/3 complex mutants bright FM4-64 dots accumulate plasma <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             membrane prior membrane internalization WT arp2&#206;&#8221;/&#206;&#8221; arp2&#206;&#8221;/&#206;&#8221;arp3&#206;&#8221;/&#206;&#8221; <br />                                             deletion mutants. These observations consistent model step of <br />                                             endocytosis actin-independent (Robertson et al., 2009a). During second step of <br />                                             endocytosis, WT cells invaginate FM4-64-loaded vesicles, Arp2/3 complex <br />                                             mutants distinct invagination observed, good agreement current <br />                                             models Arp2/3 complex-mediated actin polymerization central driving force <br />                                             step membrane invagination. Instead, arp2&#206;&#8221;/&#206;&#8221; arp2&#206;&#8221;/&#206;&#8221;arp3&#206;&#8221;/&#206;&#8221; <br />                                             mutants bright distinct dots appeared stuck membrane little movement <br />                                             membrane appearance invagination 3 h. Finally, later <br />                                             steps endocytosis, vesicle scission movement away plasma membrane was <br />                                             readily detected WT cells showing internalization rapidly moving vesicles with <br />                                             subsequent dye accumulation vacuole. While vacuolar dye accumulation was <br />                                             observed somewhat delayed appeared overall weaker Arp2/3 complex mutants, <br />                                             vacuolar accumulation FM4-64 gradually occurred concomitantly fading the <br />                                             plasma membrane staining. Curiously, vacuolar staining appeared independent of <br />                                             vesicle movement, clear, distinct vesicles emanating plasma membrane could <br />                                             detected. This observation lending support recent reports S. cerevisiae that <br />                                             Arp2/3-independent routes actin-driven polymerization involved endocytosis <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             (Robertson et al., 2009b), idea particularly attractive distantly related <br />                                             eukaryotes like red algae BLAST searches did detect Arp2/3 complex subunits <br />                                             (Galletta Cooper, 2009). <br />  <br />                                             While work needed refine interplay various endocytotic machinery <br />                                             components Arp2/3 complex order explore spatiotemporal involvement in <br />                                             processes endocytosis filamentous growth C. albicans, Arp2/3 complex <br />                                             mutants molecularly manipulatible model organism provide powerful tool probe <br />                                             process greater depth. <br />  <br />  <br />  <br />  <br />                                                  Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                           Page 12 <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />                                   Experimental procedures <br />                                   Strains, plasmids, primers media <br />                                               Strains, plasmids primers (oligonucleotides) used study listed Tables S6&#226;&#8364;&#8220; <br />                                               S8. YPD media consisted 1% yeast extract, 2% peptone, 2% dextrose 2% agar <br />                                               supplemented 50 mg l&#226;&#710;&#8217;1 uridine. YPD + 10% FBS Spider media (1% Difco <br />                                               <span id='am-3' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> broth, 1% mannitol, 0.2% K2HPO4, 1.35% agar, pH 7.2), media known to <br />                                               potently induce hyphal formation, used hyphae screen (Liu et al., 1994; Uhl et <br />                                               al., 2003). SD media contained 2% dextrose, 6.7% yeast nitrogen <span id='am-322' about='xsp:base' typeof='owl:Thing'>base</span> amino acids, <br />                                               2% agar supplemented appropriate amino acids. <br />  <br />                                   In vivo random mutagenesis mapping mutation <br />                                               The Escherichia coli-based UAU1 plasmid pool (Davis et al., 2002) amplified on <br />                                               Luria broth (LB) plates supplemented 50 &#206;&#188;g ml&#226;&#710;&#8217;1 kanamycin 50 &#206;&#188;g ml&#226;&#710;&#8217;1 ampicillin <br />                                               37&#194;&#176;C 2 days followed 2 h liquid incubation 37&#194;&#176;C. For transformation <br />                                               round C. albicans, roughly 5 &#206;&#188;g maxi-kit (Quiagen) extracted plasmid mix was <br />                                               NotI-digested transformed (Chen et al., 1992) BWP17 (relevant genotype: ura3/ <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                               ura3 his1/his1 arg4/arg4) (Wilson et al., 1999). The orf::UAU1/ORF heterozygotes were <br />                                               selected SD-Arg + Uri plates, patched YPD media, grown 30&#194;&#176;C 1&#226;&#8364;&#8220;2 days and <br />                                               serially replica-plated times SD-Arg-Ura plates selecting orf::UAU1/ <br />                                               orf::URA3 homozygotes. Generally, ~94% orf::UAU1/ORF heterozygotes gave rise <br />                                               orf::UAU1/orf::URA3 homozygote colonies replica-plated SD-Arg- <br />                                               Ura plates YPD plates compared ~30% heterozygotes generated <br />                                               homozygotes replica-plating transformation plates directly SD-Arg-Ura plates. <br />                                               For phenotypic analysis, potential orf::UAU1/orf::URA3 insertion mutants screened <br />                                               desired phenotype simultaneously Arg+/Ura+ prototrophy. If desired <br />                                               phenotype detected, single colony mutant selected, retested growth on <br />                                               SD-Arg SD-Ura media verify segregation markers time <br />                                               retested robustness desired phenotype. Only colony second <br />                                               testing showed consistent phenotype, i.e. growth SD-Arg, SD-Ura media with <br />                                               stable phenotype, insertion mapped. If insertion mapped directly an <br />                                               ORF WT band PCR analysis, deletion mutant constructed this <br />                                               gene. <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                               To map transposon insertions inverse PCR performed (Ochman et al., 1988). For all <br />                                               mutants used inverse PCR PCR verify absence WT band in <br />                                               orf::UAU1/orf::URA3 mutants, single colony isolated genomic DNA was <br />                                               extracted. Briefly, 20 &#206;&#188;g extracted DNA (Rose et al., 1990) purified using ice-cold <br />                                               2.5 M NH4AcO solution (Maniatis et al., 1982). MboI (New England Biolabs, NEB) was <br />                                               added 2 U &#206;&#188;g&#226;&#710;&#8217;1 DNA incubated 37 C 2 h. After enzyme inactivation and <br />                                               dilution DNA, T4 DNA ligase (NEB) added circularize fragments that <br />                                               PCR-amplified Taq polymerase (NEB) oligonucleotides (primers) oEE5/ <br />                                               oEE6. PCR products sequenced primers oEE9 oEE23 Genome <br />                                               Sequencing Centre McGill (http://www.genomequebecplatforms.com/mcgill). Sequences <br />  <br />  <br />  <br />  <br />                                                    Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                          Page 13 <br /> PMC Canada Author Manuscript <br />  <br />  <br />                                               obtained inverse PCR mapped using BLAST tools CDG homepage <br />                                               (http://www.candidagenome.org/). <br />  <br />                                   Deletion mutant construction phenotypic analysis <br />                                               Strain SN95 (Noble Johnson, 2005) used deletion mutant constructions. For <br />                                               deletion mutant, homozygotes derived different heterozygotes <br />                                               constructed. One hundred-mer 120-mer oligos flanking coding sequence of <br />                                               genes ARP2 VPS52 used amplify Arginine Histidine auxotrophic marker <br />                                               cassettes (Wilson et al., 1999) sequential disruption alleles. For arp2&#206;&#8221;/ <br />                                               &#206;&#8221;arp3&#206;&#8221;/&#206;&#8221; double mutant, fusion PCR approach strain SN148 used (Noble and <br />                                               Johnson, 2005). Correct integration marker cassettes absence WT gene <br />                                               verified mutant standard PCR analysis (Schaub et al., 2006) (Fig. S2). In <br />                                               order reintegrate WT gene mutant WT locus, nourseothricin marker <br />                                               used (Reuss et al., 2004). Briefly, WT gene amplified primers oEE166/ <br />                                               oEE228, oEE174/oEE232, oEE170/oEE229 genes ARP2, VPS52 ORF19.875, <br />                                               respectively, cloned KpnI/XhoI-digested pSFS2A plasmid (Reuss et al., 2004). <br />                                               The resulting plasmids, designated pEE22, pEE18 pEE23, sequenced. <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Downstream flanking sequences ARP2, VPS52 ORF19.875 amplified with <br />                                               primers oEE167/oEE169, oEE233/oEE234 oEE230/oEE231 cloned NotI/ <br />                                               SacII-digested plasmids pEE22, pEE18 pEE23 respectively. This resulted plasmids <br />                                               pEE33, pEE20 pEE30, KpnI/SacII-digested directly transformed <br />                                               corresponding deletion mutant. Selection 200 &#206;&#188;g ml&#226;&#710;&#8217;1 of <br />                                               nourseothricin, correct integration verified PCR analysis. Finally, before <br />                                               revertant phenotypes compared mutant WT phenotypes, nourseothricin <br />                                               marker cassette looped incubating revertants 6 h YP media <br />                                               supplemented 2% maltose. Loop SAT1 flipper cassette confirmed by <br />                                               nourseothricin sensitivity, Histidine Arginine auxotrophy finally PCR analysis <br />                                               (not shown). <br />  <br />                                               The nrg1&#206;&#8221;/&#206;&#8221;arp2&#206;&#8221;/&#206;&#8221; mutant created nrg1&#206;&#8221;/&#206;&#8221; deletion strain (Murad et al., <br />                                               2001). Briefly, upstream flanking sequence ARP2 amplified primers <br />                                               oEE166/oEE382 cloned KpnI/NotI-digested plasmid pEE33, resulted in <br />                                               plasmid pEE59. This plasmid used sequentially delete alleles ARP2 <br />                                               gene nrg1&#206;&#8221;/&#206;&#8221; mutant. <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                   Microscopy time-lapse <br />                                               For microscopic analysis, upright Leitz Aristoplan inverted Leica DMIRE2 <br />                                               microscope 100&#195;&#8212; immersion oil objective, 10&#195;&#8212; projection lense used. <br />                                               Characterization Arp2/3 complex mutant phenotypes previously described <br />                                               CFW, rhodamine/phalloidin FM4-64 staining (Vida Emr, 1995; Oberholzer et <br />                                               al., 2002) LY visualization (Baggett et al., 2003). <br />  <br />                                               Time-lapse movies (Supporting information) performed TEF1-GFP labelled WT <br />                                               cells co-incubated mutant cells one-to-one ratio. Cells grown to <br />                                               exponential phase washed SD media. Agar slides prepared follows: 0.75 <br />  <br />  <br />                                                    Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                          Page 14 <br /> PMC Canada Author Manuscript <br />  <br />  <br />                                               ml half strength YPD media mixed 0.75 ml 3.4% Agarose (preheated) after <br />                                               1 &#206;&#188;l FM4-64 (200 &#206;&#188;g ml&#226;&#710;&#8217;1, dimethylsulphoxide) added. A total 100 &#206;&#188;l of <br />                                               mixture transferred deep slides covered coverslip <br />                                               sealing. After medium solidified, coverslip removed; 1 &#206;&#188;l mixed <br />                                               WT/mutant culture applied covered fresh coverslip top. Differential <br />                                               interference contrast (DIC) GFP images taken prior time-lapse microscopy to <br />                                               distinguish WT mutant cells (Fig. S4). Time-lapse image acquisition started 5 <br />                                               min application cells followed 3 h image minute. <br />                                               Images taken Zeiss Axio Imager M1 microscope Photometrics Coolsnap <br />                                               HQ camera. Image acquisition movie file assembly performed Metamorph 7 <br />                                               software (Molecular Devices). <br />  <br />                                   Comparative genome hybridization microarray studies <br />                                               Comparative genome hybridization analysis previously described (Znaidi et al., <br />                                               2007) following modifications. Genomic DNA extracted C. albicans <br />                                               culture grown saturation Qiagen Genomic DNA Extraction kit according to <br />                                               manufacturer&#226;&#8364;&#8482;s instructions. DNA hybridization Advalytix SlideBooster <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                               16 h 42&#194;&#176;C according manufacturer&#226;&#8364;&#8482;s instructions. <br />  <br />                                               For transcriptional profiling experiment, total RNA extracted using RNeasy <br />                                               Mini kit (Qiagen). Total RNA quantification measured nanodrop (ND-1000 <br />                                               spectrophotometer, NanoDrop Technologies). The quality mRNA verified a <br />                                               RNA lab-on-chip assay using 2100 expert mRNA nano BIO analyser (Agilent). cDNA <br />                                               labelling, DNA microarray hybridization, washing scanning adapted a <br />                                               standardized protocol (Nantel et al., 2006). Each condition covered minimum of <br />                                               DNA microarrays, results analysed using Gene-Spring software (Silicon <br />                                               Genetics, Redwood City, CA). In order compare significantly overlapping gene lists, the <br />                                               P-value calculated using hypergeometric distribution described GO Term <br />                                               Finder Tool web site (http://www.candidagenome.org/cgi-bin/GO/goTermFinder). Gene <br />                                               lists Tables S2&#226;&#8364;&#8220;S5. The entire data set microarray experiments has <br />                                               deposited GEO accession number GSE19583. (http:// <br />                                               www.ncbi.nlm.nih.gov/geo). For calculating Pearson correlation, GraphPad Prism 5 was <br />                                               used. <br />  <br />                                   Virulence studies <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Virulence testing C. albicans, including survival experiments, tissue fungal burden <br />                                               counting biochemical analysis host blood, previously described <br />                                               (Mullick et al., 2004; 2006). Briefly, 8- 12-week-old A/J mice (Jackson Laboratories, Bar <br />                                               Harbor, ME) inoculated tail vein 200 &#206;&#188;l suspension containing 3 &#195;&#8212; 105 <br />                                               C. albicans PBS. Three male female mice used experimental <br />                                               group. Mice closely monitored period maximal 4 days clinical signs of <br />                                               disease lethargy, ruffled fur hunched back. Mice showing extreme lethargy were <br />                                               considered moribund euthanized. All experimental procedures involving animals <br />                                               approved Biotechnology Research Institute Animal Care Committee, which <br />                                               operates guidelines Canadian Council Animal Care. <br />  <br />  <br />                                                    Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                                           Page 15 <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />                                   Supplementary Material <br />                                             Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                                   Acknowledgments <br />                                             We thank Aaron P. Mitchell providing UAU1 plasmid library, Cynthia H&#195;&#169;lie Mario Mercier for <br />                                             excellent technical assistance animal handling, Jean-S&#195;&#169;bastien Deneault Andr&#195;&#169; Nantel help with <br />                                             microarray studies Lichun Liang (a summer student John Abbott College, Montr&#195;&#169;al) help screening <br />                                             random orf::UAU1/orf::URA3 mutants. This work supported CIHR team grant fungal pathogenesis to <br />                                             A.M., M.R. M.W. (CTP-79843). This NRC publication number 49596. <br />  <br />  <br />                                   References <br />                                             Akashi T, Kanbe T, Tanaka K. The role cytoskeleton polarized growth germ tube in <br />                                               Candida albicans. Microbiology. 1994; 140:271&#226;&#8364;&#8220;280. [PubMed: 8180692] <br />                                             Arnaud MB, Costanzo MC, Skrzypek MS, Shah P, Binkley G, Lane C, et al. Sequence resources the <br />                                               Candida Genome Database. Nucleic Acids Res. 2007; 35:D452&#226;&#8364;&#8220;D456. [PubMed: 17090582] <br />                                             Askew C, Sellam A, Epp E, Hogues H, Mullick A, Nantel A, Whiteway M. Transcriptional regulation <br />                                               carbohydrate metabolism human pathogen Candida albicans. PLoS Pathog. 2009; <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                               5:e1000612. [PubMed: 19816560] <br />                                             Asleson CM, Bensen ES, Gale CA, Melms AS, Kurischko C, Berman J. Candida albicans INT1- <br />                                               induced filamentation Saccharomyces cerevisiae depends Sla2p. Mol Cell Biol. 2001; <br />                                               21:1272&#226;&#8364;&#8220;1284. [PubMed: 11158313] <br />                                             Baggett JJ, Shaw JD, Sciambi CJ, Watson HA, Wendland B. Fluorescent labeling yeast. Curr Protoc <br />                                               Cell Biol. 2003; Chapter 4(<span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>Unit</span> 4):13. [PubMed: 18228435] <br />                                             Balasubramanian MK, Feoktistova A, McCollum D, Gould KL. Fission yeast Sop2p: novel and <br />                                               evolutionarily conserved protein interacts Arp3p modulates profilin function. EMBO <br />                                               J. 1996; 15:6426&#226;&#8364;&#8220;6437. [PubMed: 8978670] <br />                                             Banerjee M, Thompson DS, Lazzell A, Carlisle PL, Pierce C, Monteagudo C, et al. UME6, novel <br />                                               filament-specific regulator Candida albicans hyphal extension virulence. Mol Biol Cell. <br />                                               2008; 19:1354&#226;&#8364;&#8220;1365. [PubMed: 18216277] <br />                                             Bennett RJ, Johnson AD. Mating Candida albicans search sexual cycle. Annu Rev <br />                                               Microbiol. 2005; 59:233&#226;&#8364;&#8220;255. [PubMed: 15910278] <br />                                             Berman J. Morphogenesis cell cycle progression Candida albicans. Curr Opin Microbiol. 2006; <br />                                               9:595&#226;&#8364;&#8220;601. [PubMed: 17055773] <br />                                             Berman J, Sudbery PE. Candida albicans: molecular revolution built lessons budding yeast. <br />                                                 Nat Rev Genet. 2002; 3:918&#226;&#8364;&#8220;930. [PubMed: 12459722] <br />                                             Bernardo SM, Khalique Z, Kot J, Jones JK, Lee SA. Candida albicans VPS1 contributes protease <br />                                                 secretion, filamentation, biofilm formation. Fungal Genet Biol Mycopathologia. 2008; <br />                                                 167:55&#226;&#8364;&#8220;63. <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Biswas S, Van Dijck P, Datta A. Environmental sensing signal transduction pathways regulating <br />                                                 morphopathogenic determinants Candida albicans. Microbiol Mol Biol Rev. 2007; 71:348&#226;&#8364;&#8220;376. <br />                                                 [PubMed: 17554048] <br />                                             Braun BR, van Het Hoog M, d&#226;&#8364;&#8482;Enfert C, Martchenko M, Dungan J, Kuo A, et al. A human-curated <br />                                                 annotation Candida albicans genome. PLoS Genet. 2005; 1:36&#226;&#8364;&#8220;57. [PubMed: 16103911] <br />                                             Bruno VM, Kalachikov S, Subaran R, Nobile CJ, Kyratsous C, Mitchell AP. Control C. albicans <br />                                                 cell wall damage response transcriptional regulator Cas5. PLoS Pathog. 2006; 2:e21. [PubMed: <br />                                                 16552442] <br />                                             Care RS, Trevethick J, Binley KM, Sudbery PE. The MET3 promoter: new tool Candida <br />                                                 albicans molecular genetics. Mol Microbiol. 1999; 34:792&#226;&#8364;&#8220;798. [PubMed: 10564518] <br />                                             Carlisle PL, Banerjee M, Lazzell A, Monteagudo C, Lopez-Ribot JL, Kadosh D. Expression levels a <br />                                                 filament-specific transcriptional regulator sufficient determine Candida albicans <br />                                                 morphology virulence. Proc Natl Acad Sci USA. 2009; 106:599&#226;&#8364;&#8220;604. [PubMed: 19116272] <br />  <br />  <br />                                                   Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                                 Page 16 <br /> PMC Canada Author Manuscript <br />  <br />  <br />                                             Casamayor A, Snyder M. Bud-site selection cell polarity budding yeast. Curr Opin Microbiol. <br />                                                 2002; 5:179&#226;&#8364;&#8220;186. [PubMed: 11934615] <br />                                             Chen DC, Yang BC, Kuo TT. One-step transformation yeast stationary phase. Curr Genet. 1992; <br />                                                 21:83&#226;&#8364;&#8220;84. [PubMed: 1735128] <br />                                             Cornet M, Bidard F, Schwarz P, Da Costa G, Blanchin-Roland S, Dromer F, Gaillardin C. Deletions of <br />                                                 endocytic components VPS28 VPS32 affect growth alkaline pH virulence both <br />                                                 RIM101-dependent RIM101-independent pathways Candida albicans. Infect Immun. 2005; <br />                                                 73:7977&#226;&#8364;&#8220;7987. [PubMed: 16299290] <br />                                             Cowen L. The evolution fungal drug resistance: modulating trajectory genotype to <br />                                                 phenotype. Nat Rev Microbiol. 2008; 6:187&#226;&#8364;&#8220;198. [PubMed: 18246082] <br />                                             Daugherty KM, Goode BL. Functional surfaces p35/ARPC2 subunit Arp2/3 complex <br />                                                 required cell growth, actin nucleation, endocytosis. J Biol Chem. 2008; 283:16950&#226;&#8364;&#8220;16959. <br />                                                 [PubMed: 18381280] <br />                                             Davis DA, Bruno VM, Loza L, Filler SG, Mitchell AP. Candida albicans Mds3p, conserved <br />                                                 regulator pH responses virulence identified insertional mutagenesis. Genet. 2002; <br />                                                 162:1573&#226;&#8364;&#8220;1581. <br />                                             Enloe B, Diamond A, Mitchell AP. A single-transformation gene function test diploid Candida <br />                                                 albicans. J Bacteriol. 2000; 182:5730&#226;&#8364;&#8220;5736. [PubMed: 11004171] <br />                                             Galletta BJ, Cooper JA. Actin endocytosis: mechanisms phylogeny. Curr Opin Cell Biol. <br />                                                 2009; 21:20&#226;&#8364;&#8220;27. [PubMed: 19186047] <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Garcia-Sanchez S, Mavor AL, Russell CL, Argimon S, Dennison P, Enjalbert B, Brown AJ. Global <br />                                                 roles Ssn6 Tup1- Nrg1-dependent gene regulation fungal pathogen, Candida <br />                                                 albicans. Mol Biol Cell. 2005; 16:2913&#226;&#8364;&#8220;2925. [PubMed: 15814841] <br />                                             Goley ED, Welch MD. The ARP2/3 complex: actin nucleator comes age. Nat Rev Mol Cell Biol. <br />                                                 2006; 7:713&#226;&#8364;&#8220;726. [PubMed: 16990851] <br />                                             Gow NA. Germ tube growth Candida albicans. Curr Top Med Mycol. 1997; 8:43&#226;&#8364;&#8220;55. [PubMed: <br />                                                 9504066] <br />                                             Gow NA, Brown AJ, Odds FC. Fungal morphogenesis host invasion. Curr Opin Microbiol. 2002; <br />                                                 5:366&#226;&#8364;&#8220;371. [PubMed: 12160854] <br />                                             Harris SD, Momany M. Polarity filamentous fungi: moving yeast paradigm. Fungal <br />                                                 Genet Biol. 2004; 41:391&#226;&#8364;&#8220;400. [PubMed: 14998522] <br />                                             Hazan I, Liu H. Hyphal tip-associated localization Cdc42 F-actin dependent Candida albicans. <br />                                                 Eukaryot Cell. 2002; 1:856&#226;&#8364;&#8220;864. [PubMed: 12477786] <br />                                             van het Hoog M, Rast TJ, Martchenko M, Grindle S, Dignard D, Hogues H, et al. Assembly the <br />                                                 Candida albicans genome sixteen supercontigs aligned chromosomes. Genome <br />                                                 Biol. 2007; 8:R52. [PubMed: 17419877] <br />                                             Hudson AM, Cooley L. A subset dynamic actin rearrangements Drosophila requires Arp2/3 <br />                                                 complex. J Cell Biol. 2002; 156:677&#226;&#8364;&#8220;687. [PubMed: 11854308] <br />                                             Kadosh D, Johnson AD. Induction Candida albicans filamentous growth program relief of <br />                                                 transcriptional repression: genome-wide analysis. Mol Biol Cell. 2005; 16:2903&#226;&#8364;&#8220;2912. [PubMed: <br />                                                 15814840] <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Kaksonen M, Toret CP, Drubin DG. Harnessing actin dynamics clathrin-mediated endocytosis. Nat <br />                                                 Rev Mol Cell Biol. 2006; 7:404&#226;&#8364;&#8220;414. [PubMed: 16723976] <br />                                             Koh AY, K&#195;&#182;hler JR, Coggshall KT, Van Rooijen N, Pier GB. Mucosal damage neutropenia are <br />                                                 required Candida albicans dissemination. PLoS Pathog. 2008; 4:e35. [PubMed: 18282097] <br />                                             Kumamoto CA, Vinces MD. Contributions hyphae hypha-co-regulated genes Candida <br />                                                 albicans virulence. Cell Microbiol. 2005; 7:1546&#226;&#8364;&#8220;1554. [PubMed: 16207242] <br />                                             Kurtz MB, Kirsch DR, Kelly R. The molecular genetics Candida albicans. Microbiol Sci. 1988; <br />                                                 5:58&#226;&#8364;&#8220;63. [PubMed: 3079219] <br />                                             Laprade L, Boyartchuk VL, Dietrich WF, Winston F. Spt3 plays opposite roles filamentous growth <br />                                                 Saccharomyces cerevisiae Candida albicans required C. albicans virulence. <br />                                                 Genet. 2002; 161:509&#226;&#8364;&#8220;519. <br />  <br />  <br />  <br />  <br />                                                  Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                                      Page 17 <br /> PMC Canada Author Manuscript <br />  <br />  <br />                                             Lavoie H, Sellam A, Askew C, Nantel A, Whiteway M. A toolbox epitope-tagging genome- <br />                                                 wide location analysis Candida albicans. BMC Genomics. 2008; 9:578. [PubMed: 19055720] <br />                                             Lees-Miller JP, Henry G, Helfman DM. Identification act2, essential gene fission yeast <br />                                                 Schizosaccharomyces pombe encodes protein related actin. Proc Natl Acad Sci USA. <br />                                                 1992; 89:80&#226;&#8364;&#8220;83. [PubMed: 1729722] <br />                                             Leroy O, Gangneux JP, Montravers P, Mira JP, Gouin F, Sollet JP, et al. Epidemiology, management, <br />                                                 risk factors death invasive Candida infections critical care: multicenter, prospective, <br />                                                 observational study France (2005&#226;&#8364;&#8220;2006). Crit Care Med. 2009; 37:1612&#226;&#8364;&#8220;1618. [PubMed: <br />                                                 19325476] <br />                                             Li R. Bee1, yeast protein homology Wiscott-Aldrich syndrome protein, critical the <br />                                                 assembly cortical actin cytoskeleton. J Cell Biol. 1997; 136:649&#226;&#8364;&#8220;658. [PubMed: 9024694] <br />                                             Liu H. Transcriptional control dimorphism Candida albicans. Curr Opin Microbiol. 2001; 4:728&#226;&#8364;&#8220; <br />                                                 735. [PubMed: 11731326] <br />                                             Liu H, Kohler J, Fink GR. Suppression hyphal formation Candida albicans mutation a <br />                                                 STE12 homolog. Science. 1994; 266:1723&#226;&#8364;&#8220;1726. [PubMed: 7992058] <br />                                             Lo HJ, K&#195;&#182;hler JR, DiDomenico B, Loebenberg D, Cacciapuoti A, Fink GR. Nonfilamentous C. <br />                                                 albicans mutants avirulent. Cell. 1997; 90:939&#226;&#8364;&#8220;949. [PubMed: 9298905] <br />                                             McCollum D, Feoktistova A, Morphew M, Balasubramanian M, Gould KL. The Schizosaccharomyces <br />                                                 pombe actin-related protein, Arp3, component cortical actin cytoskeleton interacts <br />                                                 profilin. EMBO J. 1996; 15:6438&#226;&#8364;&#8220;6446. [PubMed: 8978671] <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Machesky LM, Atkinson SJ, Ampe C, Vandekerckhove J, Pollard TD. Purification cortical <br />                                                 complex containing unconventional actins Acanthamoeba affinity chromatography on <br />                                                 profilin-agarose. J Cell Biol. 1994; 127:107&#226;&#8364;&#8220;115. [PubMed: 7929556] <br />                                             Maniatis, T.; Fritsch, EF.; Sambrook, J. Molecular Cloning: A Laboratory Manual. New York: Cold <br />                                                 Spring Harbor Laboratory Press, Cold Spring Harbor Laboratory, New York; 1982. <br />                                             Martchenko M, Levitin A, Hogues H, Nantel A, Whiteway M. Transcriptional rewiring fungal <br />                                                 galactose-metabolism circuitry. Curr Biol. 2007a; 17:1007&#226;&#8364;&#8220;1013. [PubMed: 17540568] <br />                                             Martchenko M, Levitin A, Whiteway M. Transcriptional activation domains Candida albicans <br />                                                 Gcn4p Gal4p homologs. Eukaryot Cell. 2007b; 6:291&#226;&#8364;&#8220;301. [PubMed: 17158732] <br />                                             Martin AC, Xu XP, Rouiller I, Kaksonen M, Sun Y, Belmont L, et al. Effects Arp2 Arp3 <br />                                                 nucleotide-binding pocket mutations Arp2/3 complex function. J Cell Biol. 2005; 168:315&#226;&#8364;&#8220;328. <br />                                                 [PubMed: 15657399] <br />                                             Moreau V, Madania A, Martin RP, Winson B. The Saccharomyces cerevisiae actin-related protein <br />                                                 Arp2 involved actin cytoskeleton. J Cell Biol. 1996; 134:117&#226;&#8364;&#8220;132. [PubMed: 8698808] <br />                                             Moreau V, Galan JM, Devilliers G, Haguenauer-Tsapis R, Winsor B. The yeast actin-related protein <br />                                                 Arp2p required internalization step endocytosis. Mol Biol Cell. 1997; 8:1361&#226;&#8364;&#8220;1375. <br />                                                 [PubMed: 9243513] <br />                                             Moseley JB, Goode BL. The yeast actin cytoskeleton: cellular function biochemical <br />                                                 mechanism. Microbiol Mol Biol Rev. 2006; 70:605&#226;&#8364;&#8220;645. [PubMed: 16959963] <br />                                             Mullick A, Elias M, Picard S, Bourget L, Jovcevski O, Gauthier S, et al. Dysregulated inflammatory <br />                                                 response Candida albicans C5-deficient mouse strain. Infect Immun. 2004; 72:5868&#226;&#8364;&#8220;5876. <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 [PubMed: 15385488] <br />                                             Mullick A, Leon Z, Min-Oo G, Berghout J, Lo R, Daniels E, Gros P. Cardiac failure C5-deficient <br />                                                 A/J mice Candida albicans infection. Infect Immun. 2006; 74:4439&#226;&#8364;&#8220;4451. [PubMed: <br />                                                 16861630] <br />                                             Murad AM, Leng P, Straffon M, Wishart J, Macaskill S, MacCallum D, et al. NRG1 represses yeast- <br />                                                 hypha morphogenesis hypha-specific gene expression Candida albicans. EMBO J. 2001; <br />                                                 20:4742&#226;&#8364;&#8220;4752. [PubMed: 11532938] <br />                                             Nantel A. The long hard road completed Candida albicans genome. Fungal Genet Biol. 2006; <br />                                                 43:311&#226;&#8364;&#8220;315. [PubMed: 16517185] <br />                                             Nantel A, Dignard D, Bachewich C, Harcus D, Marcil A, Bouin AP, et al. Transcription profiling of <br />                                                 Candida albicans cells undergoing yeast-to-hyphal transition. Mol Biol Cell. 2002; 13:3452&#226;&#8364;&#8220; <br />                                                 3465. [PubMed: 12388749] <br />  <br />  <br />  <br />                                                  Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                                   Page 18 <br /> PMC Canada Author Manuscript <br />  <br />  <br />                                             Nantel, A.; Rigby, T.; Hogues, H.; Whiteway, M. Microarrays Studying Pathogenicity Candida <br />                                                 albicans. In: Kavanaugh, K., editor. Medical Mycology: Cellular Molecular Techniques. New <br />                                                 York, NJ: Wiley Press; 2006. <br />                                             Nicholls S, Straffon M, Enjalbert B, Nantel A, Macaskill S, Whiteway M, Brown AJ. Msn2- and <br />                                                 Msn4-like transcription factors play obvious roles stress responses fungal pathogen <br />                                                 Candida albicans. Eukaryot Cell. 2004; 3:1111&#226;&#8364;&#8220;1123. [PubMed: 15470239] <br />                                             Nobile CJ, Mitchell AP. Regulation cell-surface genes biofilm formation C. albicans <br />                                                 transcription factor Bcr1p. Curr Biol. 2005; 15:1150&#226;&#8364;&#8220;1155. [PubMed: 15964282] <br />                                             Noble SM, Johnson AD. Strains strategies large-scale gene deletion studies diploid <br />                                                 human fungal pathogen Candida albicans. Eukaryot Cell. 2005; 4:298&#226;&#8364;&#8220;309. [PubMed: 15701792] <br />                                             Oberholzer U, Marcil A, Leberer E, Thomas DY, Whiteway M. Myosin I required hypha <br />                                                 formation Candida albicans. Eukaryot Cell. 2002; 1:213&#226;&#8364;&#8220;228. [PubMed: 12455956] <br />                                             Oberholzer U, Iouk TL, Thomas DY, Whiteway M. Functional characterization myosin I tail <br />                                                 regions Candida albicans. Eukaryot Cell. 2004; 3:1272&#226;&#8364;&#8220;1286. [PubMed: 15470256] <br />                                             Oberholzer U, Nantel A, Berman J, Whiteway M. Transcript profiles Candida albicans cortical <br />                                                 actin patch mutants reflect cellular defects: contribution Hog1p Mkc1p signaling <br />                                                 pathways. Eukaryot Cell. 2006; 5:1252&#226;&#8364;&#8220;1265. [PubMed: 16896210] <br />                                             Ochman H, Gerber AS, Hartl DL. Genetic applications inverse polymerase chain reaction. Genet. <br />                                                 1988; 120:621&#226;&#8364;&#8220;623. <br />                                             Ostrander DB, Gorman JA. Characterization Candida albicans TRP1 gene construction a <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 homozygous trp1 mutant sequential co-transformation. Gene. 1994; 148:179&#226;&#8364;&#8220;185. [PubMed: <br />                                                 7958943] <br />                                             Palmer GE, Cashmore A, Sturtevant J. Candida albicans VPS11 required vacuole biogenesis and <br />                                                 germ tube formation. Eukaryot Cell. 2003; 2:411&#226;&#8364;&#8220;421. [PubMed: 12796286] <br />                                             Pfaller MA, Diekema DJ. Epidemiology invasive candidiasis: persistent public health problem. <br />                                                 Clin Microbiol Rev. 2007; 20:133&#226;&#8364;&#8220;163. [PubMed: 17223626] <br />                                             Pruyne D, Bretscher A. Polarization cell growth yeast. J Cell Sci. 2000; 113:571&#226;&#8364;&#8220;585. [PubMed: <br />                                                 10652251] <br />                                             Reuss O, Vik A, Kolter R, Morschh&#195;&#164;user J. The SAT1 flipper, optimized tool gene disruption in <br />                                                 Candida albicans. Gene. 2004; 341:119&#226;&#8364;&#8220;127. [PubMed: 15474295] <br />                                             Riezman H. Endocytosis yeast: yeast secretory mutants defective endocytosis. <br />                                                 Cell. 1985; 40:1001&#226;&#8364;&#8220;1009. [PubMed: 3886157] <br />                                             Robertson AS, Smythe E, Ayscough KR. Functions actin endocytosis. Cell Mol Life Sci. 2009a; <br />                                                 66:2049&#226;&#8364;&#8220;2065. [PubMed: 19290477] <br />                                             Robertson AS, Allwood EG, Smith AP, Gardiner FC, Costa R, Winder SJ, Ayscough KR. The WASP <br />                                                 homologue Las17 activates novel actin-regulatory activity Ysc84 promote endocytosis in <br />                                                 yeast. Mol Biol Cell. 2009b; 20:1618&#226;&#8364;&#8220;1628. [PubMed: 19158382] <br />                                             Roemer T, Jiang B, Davison J, Ketela T, Veillette K, Breton A, et al. Large-scale essential gene <br />                                                 identification Candida albicans applications antifungal drug discovery. Mol Microbiol. <br />                                                 2003; 50:167&#226;&#8364;&#8220;181. [PubMed: 14507372] <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Rose, MD.; Winston, F.; Hieter, P. Methods Yeast Genetics: A Laboratory Course Manual. New <br />                                                 York: Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York; 1990. <br />                                             Ruhnke M, Maschmeyer G. Management mycoses patients hematologic disease cancer <br />                                                 &#226;&#8364;&#8220; review literature. Eur J Med Res. 2002; 7:227&#226;&#8364;&#8220;235. [PubMed: 12069913] <br />                                             Santos R, Buisson N, Knight SA, Dancis A, Camadro JM, Lesuisse E. Candida albicans lacking the <br />                                                 frataxin homologue: relevant yeast model studying role frataxin. Mol Microbiol. 2004; <br />                                                 54:507&#226;&#8364;&#8220;519. [PubMed: 15469520] <br />                                             Sawa M, Suetsugu S, Sugimoto A, Miki H, Yamamoto M, Takenawa T. Essential role C. <br />                                                 elegans Arp2/3 complex cell migration ventral enclosure. J Cell Sci. 2003; 116:1505&#226;&#8364;&#8220; <br />                                                 1518. [PubMed: 12640035] <br />                                             Schaub Y, Dunkler A, Walther A, Wendland J. New pFA-cassettes PCR-based gene manipulation <br />                                                 Candida albicans. J Basic Microbiol. 2006; 46:416&#226;&#8364;&#8220;429. [PubMed: 17009297] <br />  <br />  <br />  <br />  <br />                                                  Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                                  Page 19 <br /> PMC Canada Author Manuscript <br />  <br />  <br />                                             Schwartz MA, Madhani HD. Principles MAP kinase signaling specificity Saccharomyces <br />                                                 cerevisiae. Annu Rev Genet. 2004; 38:725&#226;&#8364;&#8220;748. [PubMed: 15568991] <br />                                             Schwob E, Martin RP. New yeast actin-like gene required late cell cycle. Nature. 1992; <br />                                                 355:179&#226;&#8364;&#8220;182. [PubMed: 1729653] <br />                                             Shen J, Cowen LE, Griffin AM, Chan L, K&#195;&#182;hler JR. The Candida albicans pescadillo homolog is <br />                                                 required normal hypha-to-yeast morphogenesis yeast proliferation. Proc Natl Acad Sci <br />                                                 USA. 2008; 105:20918&#226;&#8364;&#8220;20923. [PubMed: 19075239] <br />                                             Smythe E, Ayscough KR. Actin regulation endocytosis. J Cell Sci. 2006; 119:4589&#226;&#8364;&#8220;4598. [PubMed: <br />                                                 17093263] <br />                                             Song Y, Cheon SA, Lee KE, Lee SY, Lee BK, Oh DB, et al. Role RAM network cell polarity <br />                                                 hyphal morphogenesis Candida albicans. Mol Biol Cell. 2008; 19:5456&#226;&#8364;&#8220;5477. [PubMed: <br />                                                 18843050] <br />                                             Steinberg G. Hyphal growth: tale motors, lipids, Spitzenkorper. Eukaryot Cell. 2007; <br />                                                 6:351&#226;&#8364;&#8220;360. [PubMed: 17259546] <br />                                             Sudbery P, Gow N, Berman J. The distinct morphogenic states Candida albicans. Trends Microbiol. <br />                                                 2004; 12:317&#226;&#8364;&#8220;324. [PubMed: 15223059] <br />                                             Tuite A, Elias M, Picard S, Mullick A, Gros P. Genetic control susceptibility Candida albicans in <br />                                                 susceptible A/J resistant C57BL/6J mice. Genes Immun. 2005; 6:672&#226;&#8364;&#8220;682. [PubMed: <br />                                                 16079897] <br />                                             Uhl MA, Biery M, Craig N, Johnson AD. Haploinsufficiency-based large-scale forward genetic <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 analysis filamentous growth diploid human fungal pathogen C. albicans. EMBO J. 2003; <br />                                                 22:2668&#226;&#8364;&#8220;2678. [PubMed: 12773383] <br />                                             Vida TA, Emr SD. A new vital stain visualizing vacuolar membrane dynamics endocytosis in <br />                                                 yeast. J Cell Biol. 1995; 128:779&#226;&#8364;&#8220;792. [PubMed: 7533169] <br />                                             Walther A, Wendland J. Polarized hyphal growth Candida albicans requires Wiskott-Aldrich <br />                                                 Syndrome protein homolog Wal1p. Eukaryot Cell. 2004; 3:471&#226;&#8364;&#8220;482. [PubMed: 15075276] <br />                                             Weig M, Jansch L, Gross U, De Koster CG, Klis FM, De Groot PW. Systematic identification silico <br />                                                 covalently bound cell wall proteins analysis protein-polysaccharide linkages the <br />                                                 human pathogen Candida glabrata. Microbiology. 2004; 150:3129&#226;&#8364;&#8220;3144. [PubMed: 15470094] <br />                                             Welch MD, Iwamatsu A, Mitchison TJ. Actin polymerization induced Arp2/3 protein complex at <br />                                                 surface Listeria monocytogenes. Nature. 1997; 385:265&#226;&#8364;&#8220;269. [PubMed: 9000076] <br />                                             Whiteway M, Oberholzer U. Candida morphogenesis host-pathogen interactions. Curr Opin <br />                                                 Microbiol. 2004; 7:350&#226;&#8364;&#8220;357. [PubMed: 15358253] <br />                                             Whiteway M, Bachewich C. Morphogenesis Candida albicans. Annu Rev Microbiol. 2007; 61:529&#226;&#8364;&#8220; <br />                                                 553. [PubMed: 17506678] <br />                                             Wilson RB, Davis D, Mitchell AP. Rapid hypothesis testing Candida albicans gene <br />                                                 disruption short homology regions. J Bacteriol. 1999; 181:1868&#226;&#8364;&#8220;1874. [PubMed: 10074081] <br />                                             Winter D, Podtelejnikov AV, Mann M, Li R. The complex containing actin-related proteins Arp2 and <br />                                                 Arp3 required motility integrity yeast actin patches. Curr Biol. 1997; 7:519&#226;&#8364;&#8220;529. <br />                                                 [PubMed: 9210376] <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Winter DC, Choe EY, Li R. Genetic dissection budding yeast Arp2/3 complex: comparison of <br />                                                 vivo structural roles individual subunits. Proc Natl Acad Sci USA. 1999; 96:7288&#226;&#8364;&#8220; <br />                                                 7293. [PubMed: 10377407] <br />                                             Wolyniak MJ, Sundstrom P. Role actin cytoskeletal dynamics activation cyclic AMP <br />                                                   pathway HWP1 gene expression Candida albicans. Eukaryot Cell. 2007; 6:1824&#226;&#8364;&#8220;1840. <br />                                                   [PubMed: 17715368] <br />                                             Zallen JA, Cohen Y, Hudson AM, Cooley L, Wieschaus E, Schejter ED. SCAR primary regulator <br />                                                   Arp2/3-dependent morphological events Drosophila. J Cell Biol. 2002; 156:689&#226;&#8364;&#8220;701. <br />                                                   [PubMed: 11854309] <br />                                             Zeidler U, Lettner T, Lassnig C, Muller M, Lajko R, Hintner H, et al. UME6 crucial downstream <br />                                                   target transcriptional regulators true hyphal development Candida albicans. FEMS <br />                                                   Yeast Res. 2009; 9:126&#226;&#8364;&#8220;142. [PubMed: 19054126] <br />  <br />  <br />  <br />  <br />                                                  Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                              Page 20 <br /> PMC Canada Author Manuscript <br />  <br />  <br />                                             Znaidi S, De Deken X, Weber S, Rigby T, Nantel A, Raymond M. The zinc cluster transcription factor <br />                                                  Tac1p regulates PDR16 expression Candida albicans. Mol Microbiol. 2007; 66:440&#226;&#8364;&#8220;452. <br />                                                  [PubMed: 17897373] <br />                                             Zou H, Fang HM, Zhu Y, Wang Y. Candida albicans Cyr1, Cap1 G-actin form sensor/effector <br />                                                  apparatus activating cAMP synthesis hyphal growth. Mol Microbiol. 200910.1111/j. <br />                                                  1365-2958.2009.06980.x <br /> PMC Canada Author Manuscript <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                          Page 21 <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Fig. 1. Phenotypic hits resulting forward genetics screening <br />                                             A. Screening auxotrophs identified TRP1. The picture shows replica-plating WT <br />                                             (BWP17) insertion TRP1 (Tn::trp1) SD-complete (SD-c) versus SD-c minus <br />                                             tryptophan (SD-trp) media. <br />                                             B. Screening glycerol-sensitive mutants identified ORF19.875. An night culture <br />                                             adjusted OD600 0.1 10-fold serially diluted. Two microlitres then <br />                                             spotted, plates incubated 2 days photographed. <br />                                             C. Examples phenotypic hits involved hyphal formation. Hyphae formation was <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             assessed spotting 10 &#206;&#188;l night culture diluted OD600 0.1 different hyphae <br />                                             inducting media (10% FBS Spider) YPD media. Plates incubated 37&#194;&#176;C 2&#226;&#8364;&#8220; <br />                                             3 days photographed. Transposon insertions CDC39 (Tn::cdc39), VPS52 (Tn::vps52) <br />                                             ARP2 (Tn::arp2) blocked wrinkle formation cell surface inducing media. <br />                                             The column indicates site insertion, relative start codon, entire <br />                                             <span id='am-7' about='xsp:length' typeof='owl:Thing'>length</span> ORF. See Fig. S1 morphology phenotypes associated the <br />                                             remaining potential insertion mutant hits. <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                      Page 22 <br /> PMC Canada Author Manuscript <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Fig. 2. VSP52 Arp2/3 complex mutants form hyphae <br />                                             A. Overnight cultures grown YPD media diluted 1:200 YPD (yeast conditions) or <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             YPD + 10% FBS (hyphae conditions) placed 3 h 30&#194;&#176;C (yeast) 37&#194;&#176;C <br />                                             (hyphae). Both transposon deletion mutants VSP52 ARP2 form hyphae <br />                                             conditions. Heterozgygous mutants revertants behaved like WT. An arp2&#206;&#8221;/ <br />                                             &#206;&#8221;arp3&#206;&#8221;/&#206;&#8221; double <span id='am-244' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-245' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-320' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>knockout</span> showed identical behaviour compared arp2&#206;&#8221;/&#206;&#8221; single <br />                                             knockout. Bar = 10 &#206;&#188;m. <br />                                             B. Quantifying polarized morphogenesis. Hyphal induction described (A). <br />                                             While &gt; 96% WT, heterozygous revertant strains formed true hyphae, majority of <br />                                             vps52&#206;&#8221;/&#206;&#8221;, arp2&#206;&#8221;/&#206;&#8221; arp2&#206;&#8221;/&#206;&#8221;arp3&#206;&#8221;/&#206;&#8221; mutants did form hyphae grew as <br />                                             pseudohyphae instead. n &gt; 200 strain. <br />  <br />  <br />  <br />  <br />                                                  Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                            Page 23 <br /> PMC Canada Author Manuscript <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Fig. 3. <br />                                             Assaying actin cytoskeleton structures Arp2/3 complex mutants. Rhodamine/phalloidin <br />                                             staining used visualize actin cytoskeleton logarithmically growing yeast cells. <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             In WT (SN95) cells, actin patches localize sites polarized growth phases the <br />                                             cell cycle. Independent cell cycle, distinctive actin patch structures observed <br />                                             arp2&#206;&#8221;/&#206;&#8221; arp2&#206;&#8221;/&#206;&#8221;arp3&#206;&#8221;/&#206;&#8221; cells. Actin cables unaffected mutant cells. Bar = <br />                                             5 &#206;&#188;m. <br />  <br />  <br />  <br />  <br />                                                   Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                     Page 24 <br /> PMC Canada Author Manuscript <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Fig. 4. Arp2/3 complex mutants endocytose FM4-64 LY <br />                                             A. Cells logarithmic phase incubated 5 min 20 &#206;&#188;M FM4-64, washed twice, <br />                                             chased 45 min visualized epifluorescence microscopy. Under conditions, <br />                                             dye endocytosed reliably visualized vacuolar membrane staining WT <br />                                             (SN95) mutant cells. <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             B. Logarithmically growing cells incubated 4 mg ml&#226;&#710;&#8217;1 Lucifer Yellow (LY), <br />                                             incubated 90 min, washed twice visualized. LY endocytosed WT and <br />                                             Arp2/3 complex mutants. Bars = 10 &#206;&#188;m. <br />  <br />  <br />  <br />  <br />                                                  Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                            Page 25 <br /> PMC Canada Author Manuscript <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Fig. 5. Transcript similarities ARP2 microaray data set MYO5/SLA2 data sets <br />                                             A. Pearson analysis showing overall correlation ARP2 MYO5/SLA2 microarray <br />                                             data sets. Note analysis selection significantly regulated genes made. <br />                                             The conditions include comparing WT mutant strains grown hyphal growth <br />                                             conditions strains grown yeast growth conditions (HY) comparing <br />                                             arp2&#206;&#8221;/&#206;&#8221;, myo5&#206;&#8221;/&#206;&#8221; sla2&#206;&#8221;/&#206;&#8221; mutants WT cells yeast (Y) hyphal (H) <br />                                             growth conditions. &#226;&#8364;&#732;WT-HY (Epp)&#226;&#8364;&#8482; &#226;&#8364;&#732;WT-HY (Oberholzer)&#226;&#8364;&#8482; correspond data <br />                                             set obtained Oberholzer et al. respectively. A value (dark blue) corresponds to <br />                                             perfect correlation &#226;&#710;&#8217;1 (dark yellow) inverse correlation. <br />                                             B C. Venn diagrams showing deleting ARP2, MYO5 SLA2 resulted similar <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             transcriptional consequences significantly regulated genes compared. P- <br />                                             values overlap given Venn diagram. B shows data yeast form cells, C <br />                                             shows data hyphal form cells. <br />  <br />  <br />  <br />  <br />                                                  Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                         Page 26 <br /> PMC Canada Author Manuscript <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Fig. 6. <br />                                             Cluster tree hyphae-specific genes. While majority hyphae-specific genes not <br />                                             properly induced absence ARP2, MYO5 SLA2, lack hyphal gene induction <br />                                             prominent arp2&#206;&#8221;/&#206;&#8221; cells. Table S2 reports values used create cluster tree. <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                         Page 27 <br /> PMC Canada Author Manuscript <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Fig. 7. <br />                                             The nrg1&#206;&#8221;/&#206;&#8221;arp2&#206;&#8221;/&#206;&#8221; mutant phenocopies arp2&#206;&#8221;/&#206;&#8221; cell morphologies. Hyphal formation <br />                                             assayed described Fig. 2A. Deletion transcriptional repressor hyphal- <br />                                             specific genes, NRG1, leads constitutive filamentous growth absence any <br />                                             hyphal signals (yeast). Deleting NRG1 arp2&#206;&#8221;/&#206;&#8221; mutant (nrg1&#206;&#8221;/&#206;&#8221;arp2&#206;&#8221;/&#206;&#8221;) did not <br />                                             restore filamentous growth presence hyphal signals (hyphae). Bar = 10 &#206;&#188;m. <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                       Page 28 <br /> PMC Canada Author Manuscript <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Fig. 8. Arp2/3 complex mutants typical actin patch-associated phenotypes <br />                                             A. Plate spotting assays described Fig. 1 legend. Arp2/3 complex mutants <br />                                             similar cell wall cell membrane defects salt sensitivity previously <br />                                             described myo5&#206;&#8221;/&#206;&#8221; mutants. <br />                                             B. Arp2/3 complex mutants showed aberrant cell wall deposition, indicating defects cell <br />                                             separation (arrows). See movie 2 (Supporting information) arp2&#206;&#8221;/&#206;&#8221; cells that <br />                                             cell separation defects. Logarithmically growing cells stained CFW directly <br />                                             YPD media 5 min, washed visualized. Bar = 10 &#206;&#188;m. <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                                 Epp et al.                                                                                       Page 29 <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Fig. 9. <br />                                             Arp2/3 complex mutants increased fluconazole sensitivity. arp2&#206;&#8221;/&#206;&#8221; arp2&#206;&#8221;/ <br />                                             &#206;&#8221;arp3&#206;&#8221;/&#206;&#8221; mutants sensitive ergosterol targeting drug fluconazole compared <br />                                             WT revertant strains. Serial dilution spotting Fig. 1 <br />                                             legend. <br /> PMC Canada Author Manuscript <br /> PMC Canada Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Mol Microbiol. Author manuscript; available PMC 2014 July 10. <br />                 <br /> </body></html>