<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Dominant lethal mutations in the plasma membrane H+-ATPase <br /> gene of Saccharomyces cerevisiae <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>            The plasma membrane H+-ATPase of Sac-                            occurred after the D378N <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>mutation</span> was transformed into <br /> charomyces cerevisiae is an essential protein that is required <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span>                    yeast. <br /> establish cellular membrane potential and maintain a normal                            We have investigated the fate of thepmal-D378N mutation <br /> internal pH. An Asp-378 to Asn substitution at the residue                           after its introduction into yeast and have discovered that it is <br /> phosphorylated during catalysis is dominant lethal when the                          invariably lost by gene conversion with the wild-type allele <br /> pmal-D378N mutation is expressed along with a wild-type                              during sequence shuffling if the mutant gene is constitutively <br /> plasma membrane H+-ATPase (PMAI) gene. Several muta-                                 expressed. We demonstrate, using a conditionally expressed <br /> tions in the first two putative transmembrane domains are also                       pmal-D378N gene, that the mutation is dominant lethal and <br /> dominant lethal. However, these dominant lethal mutants often                        causes an arrest of cell growth with the accumulation of <br /> appear to be innocuous, because they are frequently lost by                          mutant and wild-type Pmal proteins in novel cytoplasmic <br /> gene conversion to the wild-type sequence during the process of                      structures that also contain the endoplasmic reticulum pro- <br /> introducing the mutant sequence and subsequently removing                            tein Kar2. We show further that several other site-directed <br /> the wild-type gene. Loss of the mutation by gene conversion                          mutations are also dominant lethal and cause a similar <br /> does not occur while introducing recessive lethal mutations.                         cellular arrest. <br /> Cells carrying the wild-type PMAI gene on the chromosome <br /> and a dominant lethal mutation under the control of a GALI                                         MATERIALS AND METHODS <br /> promoter on a centromere-containing plasiid exhibit a galac- <br /> tose-dependent lethaity. Indirect Imunofluorescence staining                            Media and Growth Conditions. The media and growth <br /> using anti-Pnal antibodies shows that induction of domint                            conditions for analyzing pmal mutants have been described <br /> lethal PAl mutations leads to the accumulation of a number                           (7, 8). Synthetic media containing dextrose (SD), lactic acid <br /> of intensely saining cytoplasmic structurs that are not coin-                        (SL), or galactose (SG) were used (9). Yeast cells were <br /> cident with the nucleus and its immediately surrounding en-                          transformed by the lithium acetate method of Schiestl and <br /> doplasinc reticulum. These structures also accumulate the                            Gietz (10). <br /> endoplasmic reticulum protein Kar2. Expression of the dom-                              Strain Construction. Strains SH122 and SH129, derived <br /> inant lethal protein also prevents transport of the wild-type                        <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> Y55 (HO gal3 MALI SUCI), are heterozygous for a <br /> ATPase to the plasma membrane.                                                       LEU2 marked deletion of PMAI in which the 4.1-kb Xho <br />                                                                                      I-Rgl II PMAI coding region was replaced with a 2.3-kb <br />                                                                                      LEU2 fragment (6) (see Fig. 1). Strains XZ611 and XZ634 are <br /> The plasma membrane H+-ATPase gene (PMAJ) of the yeast                               Gal+ ho leu2 ura3 trpl segregants derived from segregants of <br /> Saccharomyces cerevisiae plays an essential role in the                              a cross of strain Y55-296 (HO leu2 ura3 gal3) with strain <br /> maintenance of intracellular pH and in the establishment of                          JKM30 (ho MATa leu2 ura3 trpl GAL? hmr::LEU2). The <br /> a large electrochemical gradient that is required for the                            diploid XZ635 was obtained by crossing strains XZ611 with <br /> transport of many nutrients into the cell [reviewed by Serrano                       XZ634. <br /> (1)]. The study of the structure and function of this major                             Plasmids. Plasmid pSH30 was constructed by deleting a <br /> plasma membrane protein is interesting not only in its own                           2.1-kb Asp718 PMAI fragment from a pSH29 (7). pmal- <br /> right but also because of its extensive structural similarity to                     D378N was made by in vitro mutagenesis as described (7, 11, <br /> the large family of cation ATPases in mammalian cells,                               12) and subsequently subcloned as an Asp718 fragment into <br /> including the Na+,K+-, Ca2+-, and H+,K+-ATPases (1, 2).                              pSH30. All site-directed mutations of the first two transmem- <br />    One residue in PMAI that has been studied in detail is                            brane segments were made in pSN54 and subsequently <br /> Asp378, which undergoes phosphorylation and dephosphor-                              subcloned into the EcoRV and BstEII sites of pSN57 (11). <br /> ylation during catalysis (3, 4). The analogous residue in the                           To place the pmal genes containing dominant lethal mu- <br /> mammalian Ca2+-ATPase has been mutated and shown to be                               tations under the control of galactose-inducible promoter, all <br /> essential for catalytic activity (5). Surprisingly, Portillo and                     mutations were subcloned into the centromere-containing <br /> Serrano (6) reported that substitution of Asn for Asp-378                            plasmids pSN107 and pXZ03. An 11-kb Aat II-Pvu II frag- <br /> (D378N) in the yeast H+-ATPase was nearly wild type in its                           ment of pSH14 (7) and a 1.9-kb Aat II-Pvu II fragment of <br /> behavior and formed a normally phosphorylated intermedi-                             pRS61 (6) were ligated together to create pXZ03, containing <br /> ate. Given the high degree of conservation of this Asp and its                       the entire PMAI gene joined to a galactose-inducible pro- <br /> surrounding amino acids in all P-type ATPases, the difference                        moter, in the URA3 single-copy plasmid YCp5O. pXZ09 <br /> in results between the yeast and mammalian ATPases was <br /> unexpected. It seemed possible to us that this surprising                            Abbreviations: HA, hemagglutinin; ER, endoplasmic reticulum; <br /> result could be explained by some secondary event that                               DAPI, 4&apos;,6-diamidino-2-phenylindole. <br />                                                                                      *S.L.H., S.N., and X.Z. contributed equally to this work. <br />                                                                                      &#194;&#167;Present address: Department of Molecular Biology, Princeton Uni- <br /> The publication costs of this article were defrayed in part by page charge            versity, Princeton, NJ 08544. <br /> payment. This article must therefore be hereby marked &quot;advertisement&quot;                 Present address: Pfizer Corp., Groton, CT 06340. <br /> in accordance with 18 U.S.C. &#194;&#167;1734 solely to indicate this fact.                     ttTo whom reprint requests should be addressed. <br />                                                                              10531 <br />  10532    Biochemistry: Harris et al.                                           Proc. Natl. Acad. Sci. USA 91 (1994) <br /> contains the D378N mutation. pSN107 is identical to pXZ03                                 URA3 <br /> except that the URA3 gene on the original vector was deleted                      pmaISmut2 <br /> and replaced by inserting a URA3 gene lacking its EcoRV site <br /> into the 3&apos; end of GALI::PMAJ. This modification made it <br /> easier to replace the EcoRV-BstEII fragment with others <br /> containing site-directed mutations in the first two transmem- <br />                                                                            A pmal1a:: LEU2            X <br /> brane domains. <br />    Plasmid pXZ28 is identical to pXZ03 except that it carries <br /> a 33-nucleotide insert specifying the amino acids A, S, and <br />                                                                                      PMA I <br /> the 9-amino acid hemagglutinin (HA) epitope (YPYDVP- <br /> DYA). The epitope was introduced after the second amino                                                        selection of Leu- Urat <br /> acid of PMAI (X.Z. and J.E.H., unpublished data). This                                                         transformants <br /> epitope insertion has no effect on the function ofPMAI (X.Z.      A                                 RA- <br /> and J.E.H., unpublished data). Plasmid pXZ29 is identical to                      pma I -mut Ie <br /> pXZ28 except that it carries the D378N mutation, inserted as <br /> described above for pXZ09. pXZ33 is identical to pXZ28                                   :-: <br /> except the URA3 gene was replaced with a LEU2-containing <br /> fragment.                                                                            PMA   I <br />  <br />  <br />  <br />  <br />    DNA Analysis. DNA was extracted from yeast by standard <br /> methods (7, 11). Genomic DNA containing portions of PMAI                                                       loss of pma l-MLrt <br /> was amplified by the polymerase chain reaction (PCR) (13)                                                  i   by   gene conversion <br /> using Pyrostase DNA polymerase (Molecular Genetic Re-              B                                URA3 <br /> sources, Tampa, FL). The amplified DNA was sequenced                                 PMA 1 <br /> with Sequenase (United States Biochemical) using the di-                         _E= <br />                                                                                    -=- <br /> methyl sulfoxide method of Winship (14) or fmol-DNA se- <br /> quencing system (Promega). <br />    Cytological Analysis and Fluorescence Microscopy. Staining                        PMA 1 <br /> of fixed yeast cells by indirect immunofluorescence was <br /> carried out essentially as described by Davis and Fink (15).       FIG. 1. Introduction of mutant pmal sequences and the gene <br /> XZ635 was transformed with pXZ09 or with pSN107 deriv-          conversion of dominant lethal mutations. (A) PMAI can be replaced <br /> atives containing different mutations in the first two trans-   with mutant pmal genes by using a modification of the one-step gene <br /> membrane domains. The transformed strains were pregrown         replacement technique (18). Because recombination must take place <br /> in 5 ml of SD-ura medium for 4 hr at 300C. The cultures were    within regions of homology, all Ura+ Leu- transformants should <br />                                                                 contain the desired mutation and Ura+ haploid segregants, obtained <br /> then collected by centrifugation, resuspended, and grown in     after meiosis, will reveal the mutant <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span>phenotype</span>. (B) If the transform- <br /> 20 ml of SL-ura medium overnight at 30&#194;&#176;C. These cells were      ing fragment contains a dominant lethal mutation, this mutation can <br /> then collected and resuspended in SG-ura medium and grown       be eliminated by gene conversion between the incoming fragment <br /> for 4 hr at 300C for induction of GALJ::pmal expression.        and the resident, wild-type PMAI * gene (in this case marked with 12 <br />    The cellular localization of Pmal was determined first       silent <span id='am-238' about='xsp:base' typeof='owl:Thing'>base</span>-pair substitutions). When Ura+ haploid segregants are <br /> using a monoclonal antibody, F10, directed against an un-       analyzed, they will appear to be wild type, as the lethal mutation has <br /> specified epitope in Pmal protein by indirect immunofluo-       been lost. <br /> rescence (ref. 16; J.H.T., unpublished data). Localization of <br /> HA-tagged, galactose-induced Pmal proteins, either wild         in four tetrads, one each from four different transformants, <br /> type or mutant, was determined by indirect immunofluores-       none of the 16 segregants carried the D378N mutation (data <br /> cence staining using an anti-HA antibody (Berkeley Anti-        not shown). Apparently, the mutation was lost either by gene <br /> body, Richmond, CA). Similar indirect immunofluorescent         conversion involving either PMAI or PMA2 (7, 17, 19) or by <br /> staining was carried out using an anti-Kar2 antibody provided   base-pair substitution. <br /> by M. Rose (17).                                                   To determine how the mutation was being lost, we re- <br />                                                                 peated the transplacement experiment using diploid strain <br />                           RESULTS                               Table 1. Introduction of in vitro-generated mutations into <br />    Introduction of Mutant pmal Genes into Yeast. Strain         pmalA::LEU2/PMA1 diploids SH122 and SH129 <br /> SH122 was transformed with the HindIII PMAI::URA3               Transforming        % Ura+ Leu-                      Phenotype of <br /> fragments containing an in vitro-generated mutation (Fig.          DNA              transformants*                  Ura+ segregants <br /> 1A). All Ura+ Leu- transformants must have obtained the         PMAI                           14         Wild type <br /> mutation. The pmal allele containing mutations can then be      pmal-S368F                     11         Recessive mutant <br /> isolated away from the wild-type PMAI gene by sporulation       pmal-AJ35V                     11         Recessive mutant <br /> of the diploid and tetrad dissection. The two Ura+ segregants   pmal-A135D                      8         Recessive lethal <br /> should therefore contain the mutation carried on the trans-     pmal-AJ35W                     10         Apparently wild type; <br /> forming fragment. The results for a number of different in                                                  actually dominant lethal <br /> vitro-generated mutations are presented in Table 1. Reces-      pmal-D378N                  7             Apparently wild type; <br /> sive mutations such as A1351 are viable, but show reduced                                                   actually dominant lethal <br /> growth, while A135R and A135D are recessive lethal, yield- <br /> ing only two viable Ura- segregants per tetrad. The results        The transforming DNA carried the designated mutation linked to <br /> for A135W and D378N appear to suggest that these alter-         the URA3 gene inserted at the 3&apos; end of the PMAI gene, as illustrated <br /> ations are inconsequential, as four wild-type spores are        in Fig. 1. Phenotypes of haploid, Ura+ meiotic segregants of tetrads <br />                                                                 from Ura+ Leu- diploids were analyzed. For each transforming <br /> produced (Table 1).                                             DNA at least five tetrads from two independent Ura+ Leu- diploid <br />    Loss of pmal-D378N by Gene Conversion. The tetrad data       transformants were analyzed. <br /> for D378N seemed to confirm a previous report that D378N        *At least 80 Ura+ transformants were scored for Leu+. Percentages <br /> is apparently a viable, wild-type substitution (6). However,     for PMAJ and pmal-D378N DNA represent combined data for <br /> when we sequenced the DNA prepared from each segregant           transformations into SH122 and SH129. <br />            Biochemistry: Harris et al.                                                          Proc. Natl. Acad. Sci. USA 91 (1994)      10533 <br /> SH129 in which the wild-type PMAi gene was ireplaced by                           had lost the pmal-D378N allele. This result is also seen in <br /> the PMAI * allele. The PMAI * allele in this stral n ([pmalA::                    liquid-grown cultures (data not shown). These results dem- <br /> LEU2]IPMAI*) contains 12 silent mutations itn the 66 bp                           onstrate directly that the D378N mutation is dominant lethal. <br /> surrounding the GAC (D378) codon, resulting f5rom a gene                          Diploid cells carrying the wild-type PMAI gene are not as <br /> conversion event between PMA1 and its unexipressed ho-                            strongly arrested, which suggests that the D378N dominant <br /> mologous gene, PMA2 (19). When the HindIII pn:al-D378N,                           lethal allele is partially suppressed by a 2-fold change in <br /> URA3 fragment was transplaced into strain SHi:29, all Ura+                        PMAI gene dosage. pmal-D378N still exhibits dominant <br /> segregants from each of 10 independent Ura+ Le-u- diploids                        lethalilty when the wild-type protein is also expressed from <br /> were wild type. DNA samples from one Ura+ s;egregant of                           the same GAL) promoter (see below). <br /> each of 10 transformants were amplified by PCR and se-                               Expression of the Dominant Lethal D378N Mutation Causes <br /> quenced. Every one of these new Pma+ alleles clontained all                       Accumulation of Pmal Protein in Discrete Cytoplasmic Struc- <br /> 12 silent mutations from the PMAJ* allele (Figg. 1B). This                        tures. There was no characteristic cell-cycle arrest phenotype <br /> clearly demonstrates that the loss of the D378 IN mutation                        after galactose induction of pmal-D378N, as cells with all <br /> occurred by gene conversion and not by a simp]le reversion                        sizes of buds were obtained. However, there was a distinc- <br /> of a base-pair substitution.                                                      tive phenotype associated with this arrest. With cells grown <br />   Transient Expression of pmal-D378N Causes dbe Arrest of                         under permissive conditions (glucose or lactate medium), the <br /> Cell Growth. A more direct demonstration of ti ie dominant                        anti-Pmal antibody F10 decorates the cell periphery, con- <br /> lethal effect of pmal-D378N was obtained by tiransforming                         sistent with the location of the Pmal protein in the plasma <br /> both wild-type haploid and diploid strains with a crentromere-                    membrane (data not shown). However, cells shifted to ga- <br /> containing plasmid, pXZO9, that carries the GiALJ::pmal-                          lactose medium show a significant accumulation of Pmal in <br /> D378N gene (Fig. 2A; see also Materials andI Methods).                            a set of discrete cytoplasmic structures (Fig. 3B). Such <br /> When these strains were transferred from glucosse- to galac-                      structures are not seen in galactose-grown cells carrying the <br /> tose-containing medium lacking uracil, the cells Ewere unable                     control plasmid pXZ03, which contains an inducible wild- <br /> to grow (Fig. 2B). A small number of papillae werre observed,                     type PMAI gene (Fig. 3A). The location of these subplasma <br /> most likely representing revertants or gene convrertants that                     membrane structures is distinct from the location of the <br />                                                                                   DAPI-stained nucleus. These structures are highly reminis- <br /> A                            Eco RI   1 <br />                                                                                   cent of those recently observed after overexpression of the <br />                                                                                   normally unexpressed homologue, Pma2 (20). These same <br />                                                                                   immunofluorescent dots are seen in all of the other dominant <br />                                                    BstEF     si    1 520          lethal mutations we describe below. One other example, for <br />                                                                                   pmal-AJ35W, is shown in Fig. 3C. <br />                                                                                      Because these strains express wild-type Pmal protein <br />                                                                   BamHI    2420   (already in the plasma membrane before induction) and the <br />                                                                                   galactose-inducible gene, it is not possible with immunoflu- <br />                                                                                   orescent straining of all Pmal protein to determine if the <br />                                                                                   dominant lethal protein is accumulating only in the unusual <br />                                                                                   submembrane bodies or if it is also capable of being trans- <br />                                                                                   ported to the cell surface. To answer these questions we <br />                                                                                   added a 9-amino acid HA epitope to the N terminus of the <br />                                                                                   Pmal protein (plasmid pXZ28). The D378N mutation was <br />                                                                                   introduced into pXZ28, as described above, to create pXZ29. <br />                                                                                   When the wild-type, HA-tagged gene was expressed for 4 hr <br />                                                                                   and stained with anti-HA antibody, the tagged wild-type <br />                                                    HincAll        4835            protein was located only at the surface of the cells, as <br />                                                                                   expected (Fig. 3D). In contrast, when the HA-tagged, <br />                                                              8                    D378N-containing protein was expressed for 4 hr, it was not <br />                                                                                   visible at the plasma membrane surface, but only in the novel <br />                 Eco   RV   6647 <br />                                                                                   cytoplasmic dots (Fig. 3E). <br />                                                                                      The trapping of the mutant Pmal protein could reflect its <br />          B       Galactose                  Glucose                               inability to exit some part of the secretory apparatus during <br />                                                                                   its normally slow (1 hr) transit to the plasma membrane (21). <br />                                                                                   However, we find that its effect is more global, in that <br />                                                                                   wild-type PMA1 protein also becomes trapped in the cyto- <br />                                                                                   plasm and fails to appear at the cell surface. This was <br />                                                                                   demonstrated by placing two centromere plasmids into yeast: <br />                                                                                    URA3-marked pXZ09 carrying the galactose induciblepmal- <br />                                                                                   D378N gene and LEU2-marked pXZ33 with a galactose- <br />                                                                                   inducible wild-type PMA1 gene carrying the HA epitope. <br />                                                                                   When HA-tagged wild-type protein was induced at the same <br />                                                                                   time as the dominant lethal protein, the anti-HA antibody <br />                                                                                   only stained the cytoplasmic dots characteristic of the mu- <br />   FiG. 2. Demonstration of dominant lethality by expressing a                     tants (Fig. 3F). <br /> galactose-inducible pmal gene carrying a dominant let hal mutation.                  It appears these cytoplasmic bodies derive from-or at <br /> (B) Growth of haploid strains transformed with plasm idS cafrying a <br /> galactose-inducible wild-type or dominant lethal PM WA gene and <br />                                                                                   least contain protes that <br />                                                                                   latcnanpoen                 are normay associated <br />                                                                                                            htaenral          soitdwt-h  wit-the <br /> grown on glucose (right) or galactose (left). The patch of ceils at the           endoplasmic reticulum (ER). This is evident from the staining <br /> top is transformed with the wild-type PMAI gene in p)(Z03 (A). The                of cells with an antibody directed against the chaperone <br /> other cells, reading clockwise, were transformed with plasmids                    protein, Kar2. As reported previously (17), Kar2 is normally <br /> carrying A135F F144A, A135F F144S, A135W, and D2378N, respec-                     localized in ER surrounding the (DAPI-stained) nucleus (Fig. <br /> tively.                                                                            3G). In contrast, when pmal-D378N is expressed, much of <br />  10534     Biochemistry: Harris et al.                                                    Proc. Natl. Acad. Sci. USA 91 (1994) <br />                  A                                      B                                     C <br />  <br />  <br />  <br />  <br />                           GAL::PMA1-HA                           GAL.::pmral-D378N-HFA                  GAL ::mal-A 135W -HA <br />                           AntLi-PMAI AL                          Anti-PMAI Ab                           Anti-PMA1 Ab <br />  <br />                  D                                     E                                      FEUK <br />  <br />  <br />  <br />                                                                  GAL::pma I -D378N-HA                  GAL:: PMA !-HA <br />                                                                  Anti-HA Ab                            GAL.:pm77al-D378N&apos; <br />                                                                                                        Ar.&apos;-HA A h <br />  <br />                  G                                     H <br />  <br />  <br />  <br />  <br />                           GAL:.PMA l-HA                         GAL::pmal-D37 8N-HA <br />                           Ant-KAR2 Ab                           Anfi-KAR2 An <br />   FIG. 3. Accumulation of cytoplasmic structures in cells expressing dominant lethal pmal- mutations. Cells carrying either pXZ03 (PMAJ), <br /> pXZO9 (pmal-D378N), or pSN122 (pmal-A135W) were induced with galactose for 4 hr. In each panel, indirect immunofluorescence using one <br /> of three antibodies is shown. The lower panel shows 4&apos;,6-diamidino-2-phenylindole (DAPI) staining of the nucleus. In cells expressing only <br /> wild-type, HA-tagged PMAI, immunofluorescence using an anti-Pmal antibody is found only around the cell periphery, as expected for a plasma <br /> membrane protein (A). When similar cells are stained with anti-HA antibody (D), the same <span id='am-239' about='xsp:pattern' typeof='owl:Thing'>pattern</span> is observed. When these cells are stained <br /> with an anti-Kar2 antibody, bright perinuclear staining is observed (G). In cells expressing the dominant lethal mutation, there is staining of <br /> the periphery and of several large cytoplasmic structures (B). When the similar cells are stained with anti-HA antibody, only the cytoplasmic <br /> structures are stained (E), indicating that the mutant protein does not get transported to the plasma membrane. When these cells are stained <br /> with anti-Kar2 antibody, the immunofluorescence is seen around the nucleus and in highly concentrated &quot;dots&quot; (H) similar to that observed <br /> with anti-HA antibody. Similar staining of cytoplasmic dots is seen when the dominant lethal mutation pmal-A135W is expressed and stained <br /> with anti-PMA1 antibody (C). <br /> the anti-Kar2 staining is found in dots similar to those                  four wild-type segregants in which the mutation was lost or <br /> observed for the mutant pmal-D378N protein (Fig. 3H).                     else only yielded two Ura- wild-type segregants and two <br />    Dominant Negative Mutations Are Also Found in the Region               slow-growing Ura+ segregants characteristic of a recessive <br /> of the First Two Transnembrane D             ns. The first two            defect. This suggests that the loss of the mutation occurred <br /> transmembrane helices of PMAI (H1/H2) comprise a domain                   before the initially transformed cell had divided and may <br /> that appears to be tightly conformationally coupled to the                suggest that the selective pressure for loss of the mutation by <br /> active site (7, 17). Several site-directed mutations in this              gene conversion is especially acute when cells are recovering <br /> region are also dominant lethal. As with D378N, when the                  from transformation. This also indicates that the basis of <br /> pmal mutation A135W was transformed into SH122 all four                   dominant lethality for alterations in the first two transmem- <br /> segregants were wild type and all Ura+ segregants had lost                brane helices may not be the same as for a <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span>knockout</span> of the <br /> the original mutations (data not shown). pmal-AJ35W also                  essential D378 residue, which was lost in 25 of25 independent <br /> proved to be dominant lethal when expressed from the                      transformants. <br /> galactose-inducible promoter (Fig. 2). Dominant lethality has <br /> also been demonstrated for several double mutant pairs in the                                    DISCUSSION <br /> H1/H2 region: A135V F144W, A135V F144Y, A135V <br /> F144V, A135F F144G, and A135V F144A (Fig. 2 and data not                  We have discovered that a number of different mutations of <br /> shown). In these cases, the single mutations ofA135 and F144              the yeast H+-ATPase gene are dominant lethal. These mu- <br /> are recessive, and viable, but the double mutants are domi-               tations are lost through gene conversion with a wild-type <br /> nant lethal. Several of the double mutations were not always              PMAI gene during the time that both genes are present and <br /> gene converted away when Leu+ Ura- transformants were                     expressed in the same cell. We show that these mutations <br /> sporulated and dissected. The A135F F144A mutation, which                 were indeed dominant lethal by using a conditionally ex- <br /> is clearly dominant lethal (Fig. 2), was lost in only 12 of 25            pressed promoter. Cells expressing dominant lethal alleles <br /> transformants. However, the loss was always all-or-none-                  lead to the accumulation of mutant and wild-type Pmal <br /> that is, all cells of a particular transformant gave rise only to         protein in a small number of cytoplasmic structures that stain <br />           Biochemistry: Harris et al.                                             Proc. Natl. Acad. Sci. USA 91 (1994)            10535 <br /> intensely with anti-Pmal antibodies. These structures appear         Our findings have important implications for studies of the <br /> to be derived from the ER as they also stain with antibodies      PMAI gene and for analogous in vitro mutagenesis studies of <br /> directed against the ER protein Kar2.                             other essential genes in yeast. All sequence shuffling para- <br />    Finding that D378N is dominant lethal agrees with studies      digms require that the mutant gene be coresident in the cell <br /> ofthe mammalian Ca2+-ATPase, where the cognate mutation           with a wild-type gene for some time before the wild-type gene <br /> is essential (5). Whether the mammalian mutant is also            on a plasmid is evicted from the cell or a wild-type gene on <br /> dominant lethal is not known. We have also found that             a chromosome is either turned off or segregated away (24- <br /> several changes in the putative first two transmembrane           27). During this time, a dominant lethal mutation can be <br /> helices of the protein are dominant lethal. All seven of the      replaced by gene conversion with the wild-type allele. Thus, <br /> dominant lethal mutations we have examined cytologically          it is essential to confirm that any apparently harmless muta- <br /> share the same arrest phenotype, with the accumulation of         tion is actually present after sequence shuffling. <br /> Pmal in a few intensely staining cytoplasmic structures.            <br />    A mutant protein might be dominant negative by competing       <br /> with the wild type for substrate molecules. With multimeric        <br /> proteins, the mutant protein may associate with wild type to      <br /> produce an inactive oligomeric complex. A third possibility        <br /> is that the mutant Pmal protein might be transported to the <br /> plasma membrane but then create unregulated ion channels           <br /> that could collapse the cell&apos;s membrane potential. Alterna-            <br /> tively, the mutant ATPase might enter the secretory appa-             <br /> ratus and fail to be properly transported, thus &quot;clogging&quot; the      <br /> secretion pathway and arresting cell growth. This could occur         <br /> either because the mutant protein was misfolded or if the           <br /> mutant protein depolarized the ER or other compartments of         <br /> the secretory apparatus.                                              <br />    The fact that mutant Pmal protein is not transported to the     <br /> surface makes unlikely the idea that the dominant lethality is      <br /> due to some failure of the ATPase on the plasma membrane.           <br /> Moreover, since pmal-D378N is still dominant lethal when               <br /> two other copies of PMAI are expressed (one from the same           <br /> galactose-driven promoter), it is also unlikely that the mutant        <br /> proteins are poisoning the function of a multimeric plasma         <br /> membrane H+-ATPase. These results seem most consistent                <br /> with the idea that the dominant mutant proteins interfere with     <br /> transport of wild-type H+-ATPase and possibly other pro-           <br /> teins. The fact that Kar2 is found in such cytoplasmic                <br /> structures suggests that these structures are derived from the   <br /> ER, though the possibility that the Kar2 (BiP) chaperone              <br /> protein might have been carried into another compartment         <br /> cannot be discounted.                                                 <br />    Recently, similar unusual cytoplasmic structures that ap-      <br /> pear to derive from the ER were observed when the normally         <br /> unexpressed Saccharomyces Pma2 protein is transcribed             <br /> from a PMAI promoter on a high-copy plasmid (20). Expres-              <br /> sion of the Arabidopsis H+-ATPase homologue also leads to         <br /> its accumulation in ER-like membranes, but not in the plasma          <br /> membrane (22). However, the accumulation of these heter-          <br /> ologous Pmal proteins in the cytoplasm is apparently not          <br /> dominant lethal. Mutations preventing H+-ATPase transport         <br /> to the plasma membrane have also been observed by Rao and            <br /> Slayman (23). They recently showed that six different glycine     <br /> insertions at the phosphorylation domain (amino acids 378-         <br /> 383) all failed to be accumulated in the secretory vesicles          <br /> even though the 100-kDa polypeptide was synthesized and           <br /> was relatively stable. We do not know ifthese mutants would           <br /> show the same sort of cytoplasmic structures we have              <br /> observed by immunofluorescence. Coupled with our dem-                <br />                                                                    <br /> onstration, and that of the Goffeau laboratory (an unpub-             <br /> lished result mentioned in ref. 20), that the Kar2 protein also   <br /> accumulates in these unusual structures, it seems likely that         <br /> they derive from the ER rather than from later stages of          <br /> secretion.                                                            <br /> </body></html>