<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             DNA Repair (Amst). 2008 October 1; 7(10): 1731&#226;&#8364;&#8220;1745. doi:10.1016/j.dnarep.2008.06.019. <br />  <br />  <br />  <br />                            Substrate Binding Pocket Residues Human Alkyladenine-DNA <br />                            Glycosylase Critical Methylating Agent <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>Survival</span> <br />                            Cheng-Yao Chena, Haiwei H. Guoa, Dharini Shahb, A. Blanka, Leona D. Samsonb, and <br />                            Lawrence A. Loeba,* <br />                            Lawrence A. Loeb: laloeb@u.washington.edu <br />                            aJosephGottstein Memorial Cancer Research Laboratory, Department Pathology, University of <br />                            Washington, Seattle, WA 98195-7705, USA <br />                            bDepartmentof Biological Engineering Center Environmental Health Sciences, <br />                            Massachusetts Institute Technology, Cambridge, Massachusetts 02139, USA <br />  <br />                            <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  Human alkyladenine-DNA glycosylase (AAG) initiates <span id='am-241' about='xsp:base' typeof='owl:Thing'>base</span> excision repair (BER) alkylated <br />                                  deaminated bases DNA. Here, assessed mutability AAG substrate binding <br />                                  pocket, essentiality individual binding pocket amino acids survival methylation <br />                                  damage. We used oligonucleotide-directed mutagenesis randomize 19 amino acids, 8 which <br />                                  interact substrate bases, created 4.5 million variants. We expressed mutant <br />                                  AAG&apos;s repair-deficient E. coli selected protection cytotoxicity either <br />                                  methylmethane sulfonate (MMS) methyl-lexitropsin (Me-lex), agent produces 3- <br />                                  methyladenine predominant base lesion. Sequence analysis 116 methylation-resistant <br />                                  mutants revealed substitutions highly conserved Tyr127and His136. In contrast, one <br />                                  <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>mutation</span>, L180F, greatly enriched MMS- Me-lex-resistant libraries. <br />                                  Expression L180F single mutant conferred 4.4-fold enhanced survival high dose of <br />                                  MMS used selection. The homogeneous L180F mutant enzyme exhibited 2.2-fold reduced <br />                                  excision 3-methyladenine 7.3-fold reduced excision 7-methylguanine methylated <br />                                  calf thymus DNA. Decreased excision methylated bases mutant glycosylase could <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  promote survival high MMS concentrations, capacity downstream enzymes <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                  process toxic BER intermediates saturated. The mutant displayed 6.6-, 3.0-fold <br />                                  reduced excision 1,N6-ethenoadenine hypoxanthine oligonucleotide substrates, <br />                                  respectively, 1.7-fold increase binding abasic site-containing DNA. Our work provides <br />                                  vivo evidence substrate binding mechanism deduced crystal structures, illuminates <br />                                  function Leu180 wild-type human AAG, consistent role balanced <br />                                  expression BER enzymes damage survival. <br />  <br />  <br />  <br />  <br />                            &#194;&#169; 2008 Elsevier B.V. All rights reserved. <br />                            * <br />                             Corresponding author. Tel.: 206 543 6015; fax: 206 543 3967. <br />                            Publisher&apos;s Disclaimer: This PDF file unedited manuscript accepted publication. As service our <br />                            customers providing early version manuscript. The manuscript undergo copyediting, typesetting, review of <br />                            resulting proof published final citable form. Please note production process errors be <br />                            discovered affect content, legal disclaimers apply journal pertain. <br />                             Chen et al.                                                                                                Page 2 <br />  <br />  <br />  <br />                               Keywords <br />                                     Alkylating agents; base excision repair; 3-methyladenine; 7-methylguanine; methyl-lexitropsin; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                     random mutagenesis <br />  <br />  <br />                               1. Introduction <br />                                               Alkylating agents widely dispersed environment, occur endogenously, are <br />                                               frequently used treatment cancer [1,2]. If repaired, alkylated bases DNA can <br />                                               mutagenic cytotoxic [3]. For example, modification N3 position adenine can <br />                                               disrupt interaction DNA polymerases template DNA strand [4,5], apparently <br />                                               leading arrest replication machinery cell death [6-8]. Base excision repair <br />                                               (BER) primary mechanism cells remove damaged bases genome, <br />                                               including alkylated bases, conserved evolution. BER pathways initiated <br />                                               DNA glycosylase recognizes adducted base hydrolyzes N-glycosylic <br />                                               bond linking damaged base sugar-phosphate backbone [9]. The resulting abasic <br />                                               site processed AP endonuclease Ap lyase, DNA polymerase DNA ligase <br />                                               activities [10]. Importantly, abasic sites downstream interrupted-strand intermediates <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               lead double-strand breaks potentially cytotoxic. DNA glycosylase <br />                                               action commits cells pathway step produces different possibly <br />                                               lethal form damaged DNA, repair complete. As consequence, balanced <br />                                               expression [11,12] coordinated action [10] enzymes BER pathways important <br />                                               survival DNA damage. <br />  <br />                                               Alkyladenine DNA glycosylases organisms ranging bacteria plants to <br />                                               humans [2,13]. The alkyladenine-DNA glycosylase AAG, [also known N-methylpurine <br />                                               DNA glycosylase (MPG), ANPG] known human DNA glycosylase that <br />                                               removes alkylated bases [1,9]. Human AAG active broad range chemically and <br />                                               structurally diverse substrates. In addition 3-methyladenine (3-meA), AAG excises <br />                                               alkylated purines 7-methylguanine (7-meG) 3-methylguanine (3-meG) [13]; the <br />                                               oxidative purine deamination products hypoxanthine, xanthine, oxanine [14,15]; the <br />                                               lipid peroxidation-derived 1,N6-ethenoadenine (&#206;&#181;A) 1,N2-ethenoguanine (&#206;&#181;G); the <br />                                               oxidative lesion 8-hydroxyguanine [16]; and, inefficiently, normal guanine [13,17-19]. <br />                                               AAG reported excise hypoxanthine efficiently, followed &#206;&#181;A 3-meA, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               leading suggestion AAG primarily evolved hypoxanthine-DNA <br />                                               glycosylase [20,21]. <br />  <br />                                               Crystallographic studies molecular modeling [22,23], site-specific <br />                                               mutagenesis [24], provided substantial information concerning mechanism of <br />                                               action human AAG structural basis binding adducted purines. Co-crystal <br />                                               structures AAG complexed lesion-containing DNA reveal AAG, like other <br />                                               DNA glycosylases, uses nucleotide flipping mechanism recognize target bases an <br />                                               extrahelical conformation [22,23]. AAG binds minor groove DNA, single <br />                                               tyrosine (Tyr165) inserts minor groove, intercalating bases adjacent to, <br />                                               occupying space left by, flipped-out nucleotide. The flipped-out nucleotide is <br />                                               captured substrate binding pocket, and, case &#206;&#181;A, stacks side-chains <br />  <br />  <br />                                                  DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                              Page 3 <br />  <br />  <br />                                           Tyr127 His136 Tyr159 [23]. Cleavage glycosylic <br />                                           bond catalyzed nucleophilic water molecule activated essential Glu125 <br />                                           [20,23]. Many evolutionarily conserved amino acids form substrate binding pocket <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           [22,24], site-directed mutagenesis begun illuminate functions. For example, <br />                                           Asn169 proposed play important role substrate specificity. When normal <br />                                           guanine modeled substrate-binding pocket, exocyclic amino group clashes <br />                                           Asn169 chain [23]; accord, replacements Asn169 enhance excision of <br />                                           guanine mispairs [25]. Other substrate binding pocket residues include Leu180, which <br />                                           lies adjacent N3 position &#206;&#181;A, expected pack closely methyl <br />                                           adduct 3-meA [22]. <br />  <br />                                           Our present understanding roles individual substrate binding pocket residues in <br />                                           substrate discrimination AAG, importance damage survival, incomplete. <br />                                           Here, used oligonucleotide-directed random mutagenesis (e.g., 26) selection for <br />                                           methylating agent survival examine replaceability amino acids line AAG <br />                                           substrate binding pocket. Analysis methylation-resistant mutants identified mutable and <br />                                           immutable amino acids, yielding vivo evidence essential role Tyr127 His136 <br />                                           survival methylation damage. Our results provide functional validation of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           crystallographically observed interactions, suggest functions Leu180 substrate <br />                                           discrimination wild-type human AAG, provide presumptive example the <br />                                           importance damage survival coordinated processing intermediates BER <br />                                           pathways. <br />  <br />                               2. Materials methods <br />                               2.1. Strains materials <br />                                           The alkylation repair-deficient strain E. coli MV1932 (alkA1 tag) derivative strain <br />                                           AB1157 [see [12]]. E. coli BL21(DE3) purchased Agilent technologies (Santa <br />                                           Clara, CA). Enzymes NEB (Beverly, MA), oligonucleotides IDT <br />                                           (Coralville, IA). E. coli endonuclease IV kindly provided Dr. S. Bennett (Oregon <br />                                           State University). Anti-AAG (MPG) antibody obtained Imgenex (San Diego, CA). <br />  <br />                               2.2. Creation pUC-AAG-SK KX mutant libraries <br />                                           The starting vector encoded <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> human AAG cDNA pUC19-derived plasmid. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           A schematic diagram construction pUC-AAG-SK pUC-AAG-KX random <br />                                           mutant libraries shown Supplementary Fig. S1. The library designations SK KX <br />                                           refer SacII KpnI, KpnI XbaI silent restriction sites, respectively, were <br />                                           introduced AAG coding sequence allow substitution oligonucleotide cassettes <br />                                           containing randomized codons. The SK library encodes 9 randomized amino acids (Ala126, <br />                                           Tyr127, Leu128, Gly129; Glu133, Ala134, Ala135, His136, Ser137) KX library 10 <br />                                           randomized amino acids (Val158, Tyr159, Ile160; Cys167, Met168, Asn169, Ile170; Val179, <br />                                           Leu180, Leu181). To permit library construction, pre-existing SacII XbaI sites in <br />                                           pUC19 multiple cloning region eliminated restriction digestion, followed by <br />                                           mung bean exonuclease digestion remove non-complementary termini, filling with <br />                                           Klenow fragment, religation. To generate silent restriction sites AAG reading <br />  <br />  <br />  <br />                                              DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                           Page 4 <br />  <br />  <br />                                          frame, site-directed mutagenesis using QuickChange kit (Stratagene) performed, <br />                                          introducing silent mutations A354C create SacII site; G462T create KpnI site; <br />                                          A549T, G552A create XbaI site (numbers refer wild type AAG cDNA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          sequence.) <br />  <br />                                          A hexahistidine tag factor Xa cleavage sequence inserted N-terminus of <br />                                          AAG PCR amplification forward primer 5&#226;&#8364;&#178;-GCA CGC GGA TCC ATG GGT <br />                                          GGT TCT CAT CAC CAT CAC CAT CAC GGT GGG ATC GAA GGT CGT GCT GTC <br />                                          ACC CCC GCT TTG CAG ATG AAG AAA CCA AAG CAG-3&#226;&#8364;&#178; reverse primer 5&#226;&#8364;&#178;- <br />                                          GAT GCG GCC GCG GAG TTC TGT GCC ATT AGG AAG TCG CCG-3&#226;&#8364;&#178;. The amplified <br />                                          product gel purified, cut BamHI SacII religated correspondingly <br />                                          cut vector create pUC-AAG (Fig. S1A). <br />  <br />                                          To avoid contamination libraries re-circularized vector containing wild-type <br />                                          AAG, inactive constructs (&#226;&#8364;&#339;dummy vectors&#226;&#8364;?) internal deletions AAG cDNA <br />                                          prior library creation (Fig. S1A). To generate internal deletion inserts for <br />                                          inactive SK KX &#226;&#8364;&#339;dummy&#226;&#8364;? constructs, sense oligonucleotide 5&#226;&#8364;&#178;-CCG CGG CCT <br />                                          GCA CGT GGT CGG TAC CTG TGC TAG CAA GTA TCT AGA-3&#226;&#8364;&#178; annealed the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          antisense oligonucleotide 5&#226;&#8364;&#178;-CTT AGA TAC TTG CTA GCA CAG GTA CCG ACC ACG <br />                                          TGC AGG CCG C-3&#226;&#8364;&#178; generate DNA fragment SacII XbaI sticky ends and <br />                                          internal PmlI, KpnI, NheI restriction sites. The fragment digested KpnI. To <br />                                          generate pUC-AAG-SK pUC-AAG-SK dummy vectors, digestion products were <br />                                          ligated (T4 DNA ligase, Invitrogen) pUC-AAG pre-cut SacII KpnI, or <br />                                          KpnI XbaI, respectively. The constructs confirmed sequencing. The SK <br />                                          random oligonucleotide cassette (Fig. S1B) constructed annealing single- <br />                                          stranded oligomers SK sense: 5&#226;&#8364;&#178;-ACA GAA CTC CGC GGC CGC ATC GTG GAG ACC <br />                                          GAG GCA TAC CTG GGT CCA GAG GAT GAA GCC GCC CAC TCA CGC GGT GGC <br />                                          CGG CAG ACG-3&#226;&#8364;&#178; SK antisense: 5&#226;&#8364;&#178;-CAC GTA CAG GGT ACC CGG CTT CAT GAA <br />                                          CAT GCC TCG GTT GCG AGG CGT CTG CCG GCC ACC GCG-3&#226;&#8364;&#178;. The KX random <br />                                          oligonucleotide cassette (Fig. S1B) constructed annealing KX sense: 5&#226;&#8364;&#178;-G AAG <br />                                          CCG GGT ACC CTG TAC GTG TAC ATC ATT TAC GGC ATG TAC TTC TGC ATG <br />                                          AAC ATC TCC AGC CAG GGC GAC GGG-3&#226;&#8364;&#178; KX antisense: 5&#226;&#8364;&#178;-CAG CGG CTC <br />                                          TAG AGC TCG CAG CAA GAC GCA AGC CCC GTC GCC CTG GCT GGA-3&#226;&#8364;&#178;. <br />                                          Oligomers synthesized contain 88% wild-type nucleotide 4% of the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          nucleotides underlined position; 18 bp regions hybridization are <br />                                          italicized. Annealed oligomers extended Klenow fragment, fully double- <br />                                          stranded SK random oligonucleotides incubated SacII KpnI, KX <br />                                          random oligonucleotides KpnI XbaI. The resulting SK KX random <br />                                          oligonucleotide cassettes ligated correspondingly pre-cut pUC-AAG-SK and <br />                                          pUC-AAG-KX dummy constructs. Ligation products cut restriction enzymes <br />                                          (PmlI SK dummy vector NheI KX dummy vector) linearize <br />                                          recircularized dummy constructs lacking randomized oligonucleotide inserts, not <br />                                          mutant AAG constructs inserted cassettes; subsequent sequencing unselected and <br />                                          selected mutants revealed carry-over dummy constructs. To estimate library size, i.e., the <br />                                          number variants subsequently screened MMS survival, aliquots plasmid DNA <br />  <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                            Page 5 <br />  <br />  <br />                                           harboring randomized oligonucleotide inserts electroporated E. coli XL1-Blue <br />                                           cells number transformants determined plating LB-carbenicillin agar. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               2.3. Construction YH library <br />                                           Tyr127 His136 targeted mutagenesis generation YH library <br />                                           containing fully randomized nucleotides codons amino acids. Procedures <br />                                           followed construction SK KX libraries. The YH sense oligonucleotide <br />                                           [5&#226;&#8364;&#178;-ACA GAA CTC CGC GGC CGC ATC GTG GAG ACC GAG GCA NNN CTG GGT <br />                                           CCA GAG GAT GAA GCC GCC NNN TCA CGC GGT GGC CGG CAG ACG-3&#226;&#8364;&#178;] <br />                                           contained 25% nucleotide positions underlined. The YH anti-sense <br />                                           oligonucleotide 5&#226;&#8364;&#178;-CAC GTA CAG GGT ACG CGG CTT CAT GAA CAT GCC TCG <br />                                           GTT GCG AGG CGT CTG CCG GCC ACC GCG-3&#226;&#8364;&#178;. The annealed oligomers cut <br />                                           SacII KpnI ligated correspondingly cut SK dummy vector. <br />  <br />                               2.4. Selection characterization active AAG mutant libraries <br />                                           MV1932 cells transformed pUC-AAG-SK, pUC-AAG-KX pUC-AAG-YH <br />                                           mutant libraries dummy vectors, grown mid-log phase LB-carbenicillin <br />                                           medium. Aliquots cultures plated LB-carbenicillin agar, clones were <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           subsequently isolated order characterize unselected libraries. Mid-log phase cells <br />                                           incubated 0.2% (w/v) MMS l hr 37 &#194;&#176;C, serially diluted M9 minimal <br />                                           medium spread LB-carbenicillin plates. Survival estimated counting <br />                                           surviving colonies incubation 37 &#194;&#176;C 24 hr. Plasmids total 72 unselected <br />                                           118 selected clones SK KX libraries, 15 unselected 27 selected <br />                                           clones YH libraries isolated. The 198 bp SacII-XbaI region AAG was <br />                                           sequenced using primer 5&#226;&#8364;&#178;-ACTGGG GTTGGAGTTCTTCG-3&#226;&#8364;&#178; BigDye 3.0 (ABI). <br />                                           Mutant sequences aligned identified Sequencher 3.0 (Gene Codes <br />                                           Corporation, Ann Arbor, MI). <br />  <br />                                           Mutants resistant Me-lex isolated growing cells mid-logarithmic phase LB- <br />                                           carbenicillin medium. Prior Me-lex selection, library aliquots plated LB- <br />                                           carbenicillin, unselected clones isolated sequenced order characterize the <br />                                           composition unselected SK KX libraries. MV2157 cells (1 mL) expressing wild- <br />                                           type pUC-AAG, SK KX library, SK dummy washed phosphate-buffered <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           saline, resuspended citrate buffer, treated 200 &#206;&#188;M Me-lex 1 hr. Cells were <br />                                           diluted M9 minimal media, spread LB-carbenicillin plates incubated 24 <br />                                           hr 37 &#194;&#176;C. Plasmids surviving colonies isolated sequenced. <br />  <br />                               2.5. Purification wild-type L180F AAG <br />                                           Full length wild-type AAG mutant L180F cloned expression vector <br />                                           pET101/D-TOPO directional TOPO cloning (Invitrogen). The resulting AAGs fused <br />                                           C-terminal hexahistidine tag V5 epitope site expressed control the <br />                                           T7 promoter. The pET-AAGs transformed E. coli BL21(DE3) grown 30 &#194;&#176;C <br />                                           6 L 2&#195;&#8212;YT medium containing 0.5 % glucose 50 &#206;&#188;g/ml carbenicillin; IPTG (1 mM <br />                                           final concentration) added OD600 = &#226;&#710;&#188;0.6. After incubation 16 h 30 &#194;&#176;C, <br />                                           cells harvested centrifugation resuspended 120 ml buffer SB [50 mM Tris- <br />  <br />  <br />                                              DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                          Page 6 <br />  <br />  <br />                                          HCl (pH 8.0), 10 mM NaCl, 1 mM PMSF, 1 mM DTT 10 % (w/v) glycerol], and <br />                                          adjusted 500 &#206;&#188;g/ml lysozyme prior storage -80 &#194;&#176;C. The cell suspension thawed, <br />                                          immediately supplemented 1&#195;&#8212; protease inhibitor cocktail (EMD Biosciences) and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          sonicated ice. The following procedures carried 4&#194;&#176;C. After centrifugation at <br />                                          20,000 &#195;&#8212;g 30 min, supernatant fraction reserved, cell pellet resuspended <br />                                          30 ml buffer SB; sonification centrifugation steps repeated the <br />                                          supernatant fractions combined (Fraction I). Fraction I loaded DEAE <br />                                          cellulose (Sigma) column (4.9 cm2 &#195;&#8212; 4 cm) equilibrated buffer A [30 mM Tris-HCl (pH <br />                                          8.0), 10 mM NaCl, 1 mM DTT, 1 mM EDTA, 2 mM benzamidine (EMD Biosciences) and <br />                                          10 % (w/v) glycerol] column washed 60 ml equilibration buffer. <br />                                          Fractions containing AAG activity pooled dialyzed buffer A (Fraction II). <br />                                          Fraction II applied SP-Sepharose column (1.8 cm2 &#195;&#8212; 4.2 cm) equilibrated buffer <br />                                          A. The column washed 40 ml buffer A, followed 45 ml linear gradient of <br />                                          10-500 mM NaCl buffer A. The fractions analyzed 12.5% SDS-polyacrylamide <br />                                          gels, fractions containing AAG pooled dialyzed buffer B [50 mM <br />                                          sodium phosphate (pH 8.0), 300 mM NaCl, 10 mM imidazole, 1 mM &#206;&#178;-mercatoethanol, 2 <br />                                          mM benzamidine 5 % (w/v) glycerol] (Fraction III). Fraction III loaded a <br />                                          Qiagen nickel-nitrilotriacetic acid (Ni-NTA) affinity column (1.77 cm2 &#195;&#8212; 3 cm) equilibrated <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          buffer B. The column washed 20 ml buffer B 30 ml linear <br />                                          gradient 10-300 mM imidazole buffer SB. Fractions containing AAG, identified by <br />                                          electrophoresis 12.5 % SDS-polyacrylamide gels, pooled (fraction IV). After <br />                                          dialysis storage buffer [30 mM Tris-HCl (pH 7.4), 1 mM DTT, 1 mM EDTA, 2 mM <br />                                          benzamidine 10 % (w/v) glycerol], preparation concentrated using Amicon <br />                                          filter <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> (MW cut-off 30,000), aliquoted stored -80 &#194;&#176;C. Protein concentration was <br />                                          determined Bradford reaction [26] using Bio-Rad Protein Assay bovine <br />                                          serum albumin standard. Equivalent amounts wild-type mutant enzymes, as <br />                                          assessed Bradford assay, exhibited equivalent absorbance A280. <br />  <br />                               2.6. DNA glycosylase assays <br />                                          DNA glycosylase activity measured quantifying release [3H]-labeled N- <br />                                          methylpurines calf thymus DNA methylated [3H]-methylnitrosourea (1 mCi, GE <br />                                          Healthcare) previously described [27]. Reaction mixtures (100 &#206;&#188;l) contained [3H]-calf <br />                                          thymus DNA (20,000 cpm), 50 nM 500 nM AAG L180F mutant enzyme specified, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          20 mM Tris-HCl (pH 7.6), 100 mM KCl, 5 mM EDTA, 2 mM EGTA, 5 mM &#206;&#178;- <br />                                          mercaptoethanol; protein concentrations equalized addition bovine serum <br />                                          albumin. Following incubation 37 &#194;&#176;C 0, 1, 2, 4, 8, 16 min, reactions stopped <br />                                          addition 0.1 volumes 3 M sodium acetate (pH 5.2) 3 volumes ethanol <br />                                          followed incubation &#226;&#710;&#8217;20 &#194;&#176;C 1 hr precipitate DNA. Non-radioactive 3-meA and <br />                                          7-meG standards added supernatant containing released 3H-methylated bases, <br />                                          solution taken dryness vacuum desiccation. Samples dissolved 15 <br />                                          ml 0.1 M HCl spotted Whatman 3MM paper. Descending paper chromatography was <br />                                          performed 18 hr 70% isopropanol, 20% water 10% concentrated ammonium <br />                                          hydroxide separate 3-meA 7-meG. Standards visualized UV light, the <br />                                          paper cut 1 cm strips. Radioactivity measured scintillation counting. <br />  <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                           Page 7 <br />  <br />  <br />                                           DNA glycosylase activity measured using double-stranded 5&#226;&#8364;&#178;-32P end-labeled <br />                                           36-mer DNA substrate containing &#206;&#181;A&#226;&#8364;&#162;T Hx&#226;&#8364;&#162;T base pair [15]. The lesion- <br />                                           containing strand 5&#226;&#8364;&#178; TTGGCAGCAGAATATTGCTXGCGGG-AATTCGGCGCG-3&#226;&#8364;&#178; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           X= &#206;&#181;A Hx. Reaction mixtures (10 &#206;&#188;l) contained 1 pmol 32P-labeled DNA substrate, <br />                                           62.5 fmol 1.6 pmol AAG L180F mutant enzyme 30 mM Tris-Hcl (pH 7.4), 1 mM <br />                                           EDTA, 1 mM DTT, 100 mM KCl. Following incubation 37 &#194;&#176;C 30 min, reactions were <br />                                           stopped heating 70 &#194;&#176;C 5 min, brought 50 mM KCl addition 1.2 &#206;&#188;l 0.5 M <br />                                           KCl, incubated E. coli endonuclease IV (1 <span id='am-3' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>) 37 &#194;&#176;C 30 min. Reactions <br />                                           terminated heating 70 &#194;&#176;C 5 min placed ice 3 min. Samples were <br />                                           combined equal volume 2X denaturing buffer containing 95% de-ionized <br />                                           formamide, 1 mM EDTA, 0.05% bromophenol blue, heated 97 &#194;&#176;C 3 min, and <br />                                           resolved electrophoresis 12% polyacrylamide, 8.3 M urea gels. Gels dried and <br />                                           visualized phosphor imaging, reaction products analyzed ImageQuant <br />                                           software (GE Healthcare). <br />  <br />                               2.7. DNA binding assay <br />                                           The relative binding affinity lesion-containing oligonucleotides evaluated using <br />                                           electrophoretic mobility shift assay previously described [25]. The 5&#226;&#8364;&#178;-32P end-labeled <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           36-mer double-stranded DNA ligand contained lesion opposite T. The lesion-containing <br />                                           strand 5&#226;&#8364;&#178;TTGGCAGCAGAATA-TTGCTXGCGGGAATTCGGCGCG-3&#226;&#8364;&#178; X was <br />                                           &#206;&#181;A, Hx tetrahydrofuran abasic site analog. Briefly, reaction mixtures (10 &#206;&#188;l) <br />                                           contained 1 pmol DNA ligand 1-15 pmol AAG 50 mM Tris-HCl (pH 7.4), 1 mM <br />                                           EDTA, 1 mM DTT, 100 &#206;&#188;g/ml acetylated bovine serum albumin. Following incubation <br />                                           ice 30 min, samples analyzed electrophoresis 7.5% polyacrylamide gels. <br />                                           Gels dried, visualized phosphor imaging, reaction products analyzed <br />                                           ImageQuant software (GE Healthcare). <br />  <br />                               2.8. Fluorescence polarization binding assay <br />                                           Steady-state fluorescence polarization assays performed compare dissociation <br />                                           constants binding wild-type AAG vs. L180F mutant enzyme double-stranded <br />                                           DNA containing single abasic (AP) site. The AP site-containing strand in <br />                                           DNA binding assay, DNA 5&#226;&#8364;&#178;-end labeled 6-carboxyfluorescein <br />                                           (FAM). The fluorescence polarization FAM measured 25 &#194;&#176;C SpectraMax <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           M5 Microplate Reader (Molecular Devices Co., Sunnyvale, CA) exciting 495 nm and <br />                                           monitoring emission 518 nm. Binding reactions (150 &#206;&#188;l) carried binding <br />                                           buffer (30 mM Tris-HCl [pH 7.4], 1 mM EDTA, 1 mM DTT, 25 mg/ml acetylated bovine <br />                                           serum albumin) contained 100 nM FAM-labeled DNA increasing concentrations of <br />                                           enzyme. Reaction mixtures equilibrated 25 &#194;&#176;C 15 min prior fluorescence <br />                                           measurements. Assays performed triplicate. After subtracting background DNA <br />                                           fluorescence polarization, measurements fit following equation determine the <br />                                           apparent dissociation constant Kd (41). <br />  <br />  <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                            Page 8 <br />  <br />  <br />  <br />  <br />                                                                                                                        Eq. (1) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            In Eq. (1), P represents intensity fluorescence polarization enhancement, c the <br />                                            maximum value fluorescence polarization, Lo Ro total concentrations DNA <br />                                            substrate wild-type AAG L180F mutant protein, respectively, Kd overall <br />                                            dissociation constant binding wild-type mutant protein FAM-AP&#194;&#183;T-36-mer. <br />  <br />                               3. Results <br />                               3.1. Creation AAG substrate binding pocket mutant libraries <br />                                            Crystal structures molecular modeling AAG complexed lesion-containing <br />                                            oligonucleotides revealed active site pocket binds substrate bases while <br />                                            discriminating normal purines. A model AAG bound DNA oligomer <br />                                            containing &#206;&#181;A shown Fig. 1A, detailed view substrate binding pocket <br />                                            illustrated Fig. 1B. The proposed pocket formed mainly aromatic hydrophobic <br />                                            residues, believed bind substrate bases stacking interactions involving <br />                                            conjugated bases Tyr127, His136, Tyr159 [22,23]. In order provide evidence the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            vivo function binding pocket residues, assess essentiality methylating <br />                                            agent resistance, created libraries mutants containing random substitutions 19 <br />                                            wild-type amino acids. We mutated residues contact &#206;&#181;A substrate co- <br />                                            crystal complex [Tyr127, Ala134, Ala135, His136, Tyr159, Cys167, Asn169, Leu180 [23]] (Fig. <br />                                            1B), flanking residues primary sequence. We used random <br />                                            oligonucleotide replacement mutagenesis create libraries, designated SK KX. <br />                                            The SK library targets residues half model Fig. 1B, including <br />                                            Tyr127, Ala134, Ala135, His136; KX library targets residues half <br />                                            including Tyr159, Cys167, Asn169, Leu180. Construction mutant libraries is <br />                                            described Experimental Procedures represented schematically Supplementary <br />                                            Fig. S1. Oligonucleotide mixtures contained 12% random nucleotides position <br />                                            encoding targeted amino acid replaced wild-type codon libraries; i.e., each <br />                                            targeted position contained 88% wild-type nucleotide 3% three <br />                                            nucleotides. The unselected SK KX libraries contained 2.8 &#195;&#8212; 106 1.7 &#195;&#8212; 106 mutant <br />                                            AAGs, respectively. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            We sequenced randomized regions 37 unselected clones SK library 35 <br />                                            KX library. The majority clones contained multiple nucleotide substitutions, <br />                                            yielding average 2.5 amino acid changes SK mutant cDNA 2.3 amino acid <br />                                            changes KX mutant cDNA (Fig. 2A,B). Multiple nucleotide substitutions observed <br />                                            targeted codon libraries (Fig. 3B). Approximately 15% 4% the <br />                                            unselected SK KX libraries harbored frameshift mutations, respectively, presumably <br />                                            derived incomplete extension random oligonucleotide synthesis. No mutant <br />                                            cDNA observed once. <br />  <br />  <br />  <br />  <br />                                              DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                           Page 9 <br />  <br />  <br />                               3.2. Genetic complementation yields mutant libraries confer MMS resistance <br />                                           The well-characterized DNA repair-deficient E. coli strain MV1932 lacks 3-meA DNA <br />                                           glycosylase AlkA O6-methylguanine-DNA methyltransferase Ada [12], is <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           highly sensitive methylating agent MMS. Expression human AAG rescues <br />                                           MV1932 cells MMS cytoxicity [6], basis functional complementation <br />                                           isolation active AAG mutants. AAG excises 3-meA 7-meG <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                           double-stranded DNA [13]. 3-meA comprises 10% MMS-induced base lesions, is <br />                                           highly cytotoxic, efficiently cleaved; 7-meG comprises 70% base lesions, <br />                                           appears relatively innocuous, efficiently cleaved. We incubation <br />                                           mid-logarithmic phase MV1932 cells 0.2% MMS 1 hr 37&#194;&#176; C yielded 2.5 % <br />                                           survival cells expressing wild-type AAG, 4.6 &#195;&#8212; 10-5 % survival cells expressing an <br />                                           inactive construct internal deletion AAG cDNA, 54,000-fold difference in <br />                                           resistance (Fig. 2A). To select active AAG mutants, MV1932 cells transformed <br />                                           SK KX libraries, incubated 0.2 % MMS 1 hr plated on <br />                                           medium lacking MMS. The surviving fraction 3.6 &#195;&#8212; 10-3 % 2.2 &#195;&#8212; 10-2 % SK <br />                                           KX libraries, respectively, (Fig. 2A), indicating small proportion mutants <br />                                           able confer MMS resistance. Surviving colonies picked, plasmids harboring <br />                                           mutant AAG genes sequenced. The independent libraries comparable in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           respects (Fig. 2A,B). Selected mutants contained 3- 4-fold fewer mutations than <br />                                           unselected mutants. The selected SK KX libraries averaged 0.6 0.7 amino acid <br />                                           substitutions mutant, respectively, compared 2.5 2.3 unselected <br />                                           libraries. No frameshifts, nonsense mutations deletions observed the <br />                                           survivors. AAG zero amino acid changes enriched libraries, initial <br />                                           5% 11% unselected SK KX libraries, respectively, 49% 44% the <br />                                           selected libraries. The large majority mutants wild-type amino acid level <br />                                           harbored different silent mutations nucleotide level. The post-selection shift toward <br />                                           lower mutational load, noted previous studies enzymes, e.g., [28], observed <br />                                           absence frameshifts, nonsense mutations deletions, enrichment wild-type <br />                                           amino acid sequence indicative strong selective pressure active AAG. <br />  <br />                               3.3. Mutability AAG substrate binding pocket residues <br />                                           Selection MMS resistance cross-species functional complementation system <br />                                           revealed mutability substrate binding pocket showed striking parallels with <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           natural evolution (Fig. 3A), differences. DNA sequencing 116 mutants the <br />                                           active SK KX libraries yielded spectrum amino acid substitutions shown Fig. <br />                                           3C,D. On average, strictly conserved residues showed fewer replacements relatively <br />                                           conserved residues, and, turn, relatively conserved residues showed fewer replacements <br />                                           non-conserved residues. This finding holds number different replacements, <br />                                           total replacements, single position. Most notably, replacements found <br />                                           strictly conserved residues Tyr127, His136, Asn169. The absence of <br />                                           substitutions single mutants double mutants (which potential for <br />                                           compensatory substitutions) emphasizes functional importance respective side <br />                                           chains implicit evolutionary conservation (Fig. 3A) X-ray structures [[22,23] <br />                                           Fig. 1]. These results, proposed key roles Tyr127 His136 in <br />  <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                            Page 10 <br />  <br />  <br />                                           substrate binding [23], led examine residues greater depth. The partially <br />                                           random oligonucleotide mutagenesis described represents incomplete search for <br />                                           amino acid substitutions. In order test possible substitutions Tyr127 His136, we <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           constructed YH library, wild-type nucleotide encoding residues was <br />                                           synthesized mixture 25% A, T, G, C. Over 16,000 mutants this <br />                                           totally random library transformed MV1932 cells subjected MMS selection. <br />                                           Sequencing surviving colonies revealed wild-type amino acid <br />                                           level, harbored silent mutations (data shown). These results further <br />                                           document critical importance Tyr127 His136 MMS resistance. <br />  <br />                                           We noted interesting potentially informative exceptions general evolutionary <br />                                           parallels noted above. The striking Leu180, strictly conserved residue whose <br />                                           chain lies adjacent N3 position &#206;&#181;A substrate [23] expected pack <br />                                           closely methyl adduct 3-meA [22]. Leu180 Phe substitutions in <br />                                           fully 33% sequenced mutants selected KX library, representing 6-fold <br />                                           enrichment 3% sequenced clones unselected library 19% selected <br />                                           library. In comparison, enrichment wild-type AAG sequence 4-fold, 11% <br />                                           sequenced clones unselected library 44% selected library (Fig. 2B). The <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           observed selection L180F AAG suggests Leu Phe replacement confers as <br />                                           great greater MMS resistance wild-type complementation system. <br />                                           Another, contrasting, exception evolutionary conservation parallel the <br />                                           results Cys167. We observed replacements amino acid single double <br />                                           mutants, conserved natural evolution (Fig. 3A). <br />  <br />                               3.4. Me-lex MMS survivors share similar spectrum substitutions <br />                                           We subjected SK KX libraries selection resistance site-specific <br />                                           methylating agent Me-lex. Me-lex selectively binds AT-rich sequences preferentially <br />                                           generates 3-meA lesions (93% base adducts) [29-31]. Selection Me-lex survival <br />                                           enriched wild-type AAG amino acid sequence, comprised 63% 54% the <br />                                           selected SK KX libraries, respectively. Fig. 3E shows sequence 15 SK 27 KX <br />                                           library mutants. The overall <span id='am-242' about='xsp:pattern' typeof='owl:Thing'>pattern</span> comparable MMS selection (Fig. <br />                                           3C,D). Again, substitutions Tyr127 His136, Cys167. One <br />                                           substitution observed Asn169, I160S, N169D, V179I triple mutant. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Interestingly, Leu180 Phe substitutions highly enriched, appearing 4 of <br />                                           19 mutants selected KX library. The similarity spectrum substitutions <br />                                           produced MMS Me-lex selection suggests protection lethality 3- <br />                                           meA key factor governing MMS resistance complementation system. <br />  <br />                               3.5. Effect L180F substitution MMS survival <br />                                           In accord notable enrichment L180F mutant MMS- Me-lex- <br />                                           selected libraries (Fig. 3B-E), expression L180F mutant conferred <br />                                           improved MMS protection relative wild-type AAG (Fig. 4A). Although LD50 <br />                                           essentially unchanged, observed statistically significant 4.4-fold increase in <br />                                           survival (P &lt; 0.005) high concentration MMS used selection active mutants <br />                                           (0.2% 1 hr 37 &#194;&#176;C); overall survival reduced orders magnitude <br />  <br />  <br />                                              DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                          Page 11 <br />  <br />  <br />                                           stringent conditions. The difference mutant wild-type viability was <br />                                           greatest high MMS concentrations, presumably compensatory survival <br />                                           mechanisms, translesion DNA synthesis recombination, saturated or <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           approaching saturation. Immunoblotting showed expression L180F mutant <br />                                           protein comparable wild-type start MMS exposure, 4 hr <br />                                           (Fig. 4B,C). <br />  <br />                               3.6. Effect L180F substitution excision methyl adducts <br />                                           To examine effect L180F substitution removal methylated bases, purified <br />                                           wild-type L180F AAG proteins. The final preparations essentially <br />                                           homogeneous (&gt;98% pure) SDS-polyacrylamide gels exhibited mobility consistent <br />                                           predicted molecular weight (Fig. 5A). Both wild-type mutant preparations <br />                                           free contaminating nuclease activity, measured assays utilizing 5&#226;&#8364;&#178;-end 32P-labeled <br />                                           single- double-stranded oligonucleotides (data shown). DNA glycosylase activity <br />                                           assayed quantifying release chromatographically separated radiolabelled N- <br />                                           methylpurines calf thymus DNA methylated [3H]-methylnitrosourea [27]. <br />                                           Previous measurements shown potentially lethal lesion 3-meA preferred <br />                                           substrate wild-type human AAG, presumptively benign lesion 7-meG is <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           removed slower rate [32]. As illustrated Fig. 5B,C, L180F mutant enzyme <br />                                           displayed reduced activity, rate release [3H]-labeled 3-meA 7-meG lower <br />                                           observed wild type AAG. The rate excision 3-meA mutant enzyme <br />                                           reduced 2.2-fold relative wild-type AAG (0.82 vs. 0.38 fmol excised/min/ <br />                                           pmol enzyme, respectively), rate excision 7-meG reduced 7.3-fold <br />                                           (0.049 vs. 0.0068 fmol excised/min/pmol enzyme, respectively). Thus, mutant enzyme <br />                                           exhibited modestly increased discrimination 7-meG. Decreased generation of <br />                                           cytotoxic abasic sites resulting reduced excision 3-meA 7-meG factor <br />                                           promoting survival high MMS concentrations (Fig. 4A), downstream enzymes in <br />                                           BER pathways saturated; factor particularly relevant case 7- <br />                                           meG, believed toxic abasic site reaction product. Possible <br />                                           structural determinants altered substrate preference presented Discussion. <br />  <br />                               3.7. Effect L180F substitution excision binding neutral adducted bases <br />                                           The methylated bases derived MMS Me-lex electron deficient, carry a <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           delocalized positive charge presumably important binding excision AAG <br />                                           [23]. However adducted bases net charge efficiently cleaved. To <br />                                           examine effect L180F substitution activity neutral adducted bases, we <br />                                           measured excision &#206;&#181;A oligonucleotide substrate coupled assay which <br />                                           excision damaged base followed endonuclease-mediated cleavage the <br />                                           abasic site product (Fig. 6A). Quantification resulting, cleaved oligonucleotide a <br />                                           denaturing polyacrylamide gel (Fig. 6B) revealed 6.6-fold decrease activity the <br />                                           L180F mutant enzyme, calculated using regression analysis 5 points initial, <br />                                           linear portion curves. The specific activity wild-type AAG 68 fmol &#206;&#181;A <br />                                           excised/min/fmol enzyme L180F mutant 10 fmol &#206;&#181;A excised/min/fmol <br />                                           enzyme. Similarly, observed 3.0-fold decrease excision Hx mutant enzyme <br />                                           (Fig. 6C,D); oligomer the &#206;&#181;A assays, lesion. The <br />  <br />  <br />                                              DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                            Page 12 <br />  <br />  <br />                                           specific activity wild-type AAG 111 fmol Hx excised/min/fmol enzyme of <br />                                           L180F mutant 37 fmol Hx excised/min/fmol enzyme. We examined binding the <br />                                           wild-type L180F mutant enzymes lesion-containing oligonucleotides in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           electrophoretic mobility shift assays (Fig. S2). In case &#206;&#181;A (Fig. S2A), quantification <br />                                           bound enzyme (Fig. S2B) revealed 1.2-fold increase relative binding affinity the <br />                                           mutant enzyme. Relative binding affinity Hx-containing oligonucleotide (Figs. S2C,D) <br />                                           enzymes. <br />  <br />                               3.8. Effect L180F substitution binding abasic site-containing DNA <br />                                           The abasic site product AAG cleavage highly cytotoxic. Therefore enhanced <br />                                           survival methylation damage mediated L180F mutant tighter <br />                                           binding retarded release potentially lethal intermediate. To compare relative <br />                                           binding affinity wild-type mutant proteins AP-DNA, performed <br />                                           electrophoretic mobility shift assays using double-stranded 5&#226;&#8364;&#178;-32P end-labeled 36 base-pair <br />                                           oligonucleotide containing abasic site analog opposite thymine; oligomer the <br />                                           &#206;&#181;A Hx assays, lesion. We observed affinity the <br />                                           L180F mutant AP-DNA slightly (&#226;&#710;&#188;1.5 fold) higher wild-type AAG <br />                                           (data shown). In order compare difference wild-type and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           mutant proteins, performed sensitive fluorescence polarization assays determine the <br />                                           apparent steady-state equilibrilium dissociation constant binding duplex 36-mer <br />                                           containing abasic site opposite T 5&#226;&#8364;&#178;-end labeled fluorescent dye FAM. When <br />                                           protein binds fluorescently-labeled DNA, rotational freedom DNA usually <br />                                           decreased, leading increased polarization. As shown Fig. 7A, duplex, FAM- <br />                                           labeled AP&#194;&#183;T-36-mer titrated increasing amounts wild-type AAG, a <br />                                           concentration-dependent increase fluorescence polarization observed saturated at <br />                                           approximately 1 &#206;&#188;M AAG. The net polarization enhancement concentration fit <br />                                           Eq. (1), yielding apparent Kd 824 &#194;&#177; 104 nM binding assay conditions. <br />                                           The magnitude polarization enhancement saturation observed F180L mutant <br />                                           enzyme (Fig. 7B), comparable wild-type AAG, yielding apparent Kd of <br />                                           479&#194;&#177; 38 nM. Hence, binding affinity L180F duplex AP-DNA, measured by <br />                                           1/Kd, &#226;&#710;&#188; 1.7 fold greater wild-type AAG. <br />  <br />                                           To explore possible structural basis enhanced abasic site binding, modeled the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Leu180 Phe substitution co-crystal structure wild-type human AAG complexed to <br />                                           pyrrolidine abasic nucleotide-containing DNA [22]. The model derived replacing <br />                                           Leu180 phenylalanine energy-minimizing, using Amber parameters chiral <br />                                           restraints Molecular Operating Environment package programs <br />                                           [Chemical Computing Group, Montreal (www.chemcorp.com)]. A conformation be <br />                                           adopted Phe chain simply replaced Leu chain; total potential <br />                                           energy calculated conformer -7197 kcal mol-1. A n energetically equivalent <br />                                           conformation (-7287 kcal mol-1) adopted choosing alternate rotamer, which <br />                                           resulted Phe chain projecting abasic site. This model, superposed with <br />                                           un-mutated protein, illustrated Fig. 7C. As shown, aromatic chain of <br />                                           Phe180 rotates abasic residue position Leu180 wild-type AAG, and <br />                                           partially fills space occupied substrate base. Greater proximity Phe180 side <br />  <br />  <br />                                              DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                           Page 13 <br />  <br />  <br />                                          chain baseless site provide favorable geometry interaction, e.g., van der <br />                                          Waals contact epsilon zeta carbons Phe deoxyribose, thereby <br />                                          increase binding affinity L180F protein AP-DNA. Modeling shows the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          abasic residue displaced outward mutant, causing loss possible, weak H bond <br />                                          O2P 5&#226;&#8364;&#178; phosphate hydroxyl Tyr159 (not shown). <br />  <br />                               4. Discussion <br />                                          Human AAG broad substrate specificity, acting variously modified purines arising <br />                                          methylation, oxidation lipid peroxidation. Substrate discrimination wild- <br />                                          type enzyme likely represents compromise promotes protection differing <br />                                          sources DNA damage, governed, part, detailed architecture the <br />                                          substrate binding pocket. Here, used oligonucleotide-directed mutagenesis to <br />                                          simultaneously randomize residues contribute substrate binding pocket and <br />                                          shown crystal structures contact substrate bases. Our mutagenesis strategy <br />                                          affords experimental opportunity remodel active site, optimized during <br />                                          natural evolution defense diverse damaging agents optimized defense <br />                                          specific agent. We used bacterial complementation select variant <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          AAGs protect killing methylating agent MMS. Our selection for <br />                                          methylation-resistant mutants recapitulated conservation certain key amino acids in <br />                                          natural evolution, providing vivo evidence support active site model deduced <br />                                          crystal structures [22,23]. We observed departures natural evolution may <br />                                          reflect optimization methylation survival complementation system. <br />  <br />                                          Crystallographic data indicated requirement strictly conserved aromatic <br />                                          residues Tyr127, His136, Tyr159 appear provide crucial stacking interactions of <br />                                          AAG &#206;&#181;A substrate [23]. Consistent data, replacements for <br />                                          Tyr127 single double MMS- Me-lex-resistant mutants, suggesting Tyr127 <br />                                          essential interactions methylated bases well. We replacements for <br />                                          His136, suggested His136 dispensable excision 3- <br />                                          meA, based model methylated base does reach far into <br />                                          substrate binding cavity stack His136 [22]. Our data indicate His136 is <br />                                          required survival methylation damage complementation system, modeling <br />                                          suggests functions impede accommodation 7-meG binding pocket, thus <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          limiting conversion relatively innocuous base lesion potentially lethal abasic site. <br />                                          Superimposing 7-meG crystal structures AAG complexed abasic site- or <br />                                          &#206;&#181;A-containing oligonucleotide [22,23] reveals steric interference His136 imidazole <br />                                          chain exocyclic N7 methyl group (Fig. 8C). The position His136 fixed by <br />                                          numerous H bonds main chain chain atoms (e.g., OH group the <br />                                          Tyr157 chain phosphate upstream baseless deoxyribose), limiting the <br />                                          size binding pocket disfavoring accommodation 7-meG. Interestingly, an <br />                                          H136Q mutant allow survival, albeit diminished relative wild-type AAG, in <br />                                          yeast complementation [42]; difference result likely <br />                                          reflects differences species employed expression levels AAG relative to <br />                                          downstream BER enzymes. We observed non-conservative (non-planar) substitutions <br />  <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                             Page 14 <br />  <br />  <br />                                          Tyr159, Asn Cys, MMS-resistant single mutants, indicating the <br />                                          function(s) Tyr159 essential selection conditions. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          The substitution highly conserved Asn169 single double mutants <br />                                          Asp, replacement observed Me-lex survivors. Preservation Asn is <br />                                          concordant proposed role chain excluding normal guanine the <br />                                          active site steric clash exocyclic N2 amino group [23]. Interestingly, we <br />                                          observed substitutions Cys167 single double methylation-resistant mutants, <br />                                          indicating required excision methylated bases complementation <br />                                          system. <br />  <br />                                          An unexpected finding work remarkable selection Phe place strictly <br />                                          conserved Leu180. The L180F mutant enriched random library &#226;&#710;&#188;50% <br />                                          greater extent wild-type AAG, dominating spectrum MMS-resistant mutants. <br />                                          Expression L180F mutant yielded survival equivalent wild-type AAG low <br />                                          doses MMS, 4.4-fold better survival high doses viability severely <br />                                          compromised. Purification L180F mutant protein examination catalytic <br />                                          properties support conclusion increased survival mediated reduction in <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          excision activity, change substrate discrimination, slight increase binding <br />                                          affinity abasic sites. These alterations consistent idea that, relative human <br />                                          cells, bacterial host cells reduced capacity repair potentially lethal BER <br />                                          intermediates arising glycosylase action, better protected methylators a <br />                                          variant AAG generates abasic sites slower-than-wild-type rate. We observed that <br />                                          overall rate removal adducted bases methylated DNA homogenous <br />                                          L180F protein reduced relative wild type AAG. The rate excision 3-meA <br />                                          calf thymus DNA decreased 2.2-fold, rate 7-meG decreased 7.3- <br />                                          fold. Because 7-meG represents &#226;&#710;&#188;70% - 80% base lesions generated MMS, <br />                                          increased discrimination 7-meG contribute enhanced survival high levels <br />                                          methylation abasic sites potentially cytotoxic BER intermediates might <br />                                          saturate downstream enzymes, leading lethal double strand breaks cell death [33]. <br />                                          Also possible importance survival, apparent binding affinity the <br />                                          L180F mutant abasic-site-containing DNA increased 70%, assessed by <br />                                          fluorescence polarization; reduced rate release relieve toxicity associated with <br />                                          unrepaired abasic sites. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Several previous studies suggested imbalanced BER caused excess DNA <br />                                          glycosylase activity increase sensitivity alkylating agents. A genetic study yeast <br />                                          [11] demonstrated spontaneous mutation rate apurinic endonuclease-deficient <br />                                          apn1 cells increased 600-fold simple imbalance 3-methyladenine <br />                                          DNA glycosylase Mag1 apurinic endonuclease. The increased spontaneous mutation is <br />                                          generation abasic sites resulting mainly Mag1 glycosylase activity, and <br />                                          exogenous Apn1 expression dramatically suppresses elevation mutation. Another <br />                                          report [34] suggested repair damaged lesions regulated DNA <br />                                          glycosylase activity glycosylase reaction rate-limiting step BER. Importantly, <br />                                          recent work suggests that, human cells, coordination activities AAG APE1 <br />                                          depends, part, slow excision AAG rapid incision AP sites the <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                            Page 15 <br />  <br />  <br />                                          abundant APE1 (34). This balance BER activities, reached natural <br />                                          evolution, lost cross-species complementation systems ours, presumably <br />                                          leading selection variant AAGs better matched capacity host cells <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          repair potentially lethal BER intermediates arising glycosylase action. <br />  <br />                                          In addition alteration rates hydrolysis methylated bases L180F mutant, <br />                                          observed 3- 7-fold decrease rate excision neutral adducted bases <br />                                          ethenoA, arises lipid peroxidation, hypoxanthine, arises oxidative <br />                                          deamination adenine. Overall, data indicate L180F substitution alters <br />                                          discrimination methylated bases better protect MMS complementation <br />                                          system, concomitant reduction activity damage sources. <br />  <br />                                          How does L180F substitution mediate altered substrate specificity observed? <br />                                          Introduction bulky aromatic Phe chain decreases space available the <br />                                          substrate binding pocket, limit accommodation modified bases. Fig. 8 <br />                                          shows model wild-type mutant substrate binding pocket, occupied 3- <br />                                          meA 7-meG. In case 3-meA, wild-type Leu180 chain makes hydrophobic <br />                                          contact methyl adduct (Fig. 8A, left panel). In L180F mutant (Fig. 8B, top <br />                                          right panel), modeled Phe180 benzyl ring located 4.2-4.6 &#195;&#8230; methyl adduct, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          favorable range &#207;&#8364;-cation stacking interaction [35,36]. In addition, orientation <br />                                          Glu125 chain perpendicular wild-type enzyme. Glu125 may <br />                                          function deprotonate water molecule (not shown) nucleophilic attack the <br />                                          glycosydic bond [22], re-orientation residue suggests possible mechanism for <br />                                          slight reduction rate excision 3-meA. In important case 7-meG, the <br />                                          mutant Phe180 decrease excision mechanisms. The re-orientation of <br />                                          Glu125 chain, described (Fig. 8B, compare panels A B). Second, <br />                                          Phe180 decrease accommodation, adducted substrate tightly boxed into <br />                                          mutant binding pocket, His136 imidazole planar benzyl ring <br />                                          other. Yet pronounced steric effects described excluding 7-meG <br />                                          substrate binding pocket highly specific 3-methyladenine DNA glycosylase <br />                                          H. pylori, glycosylase belongs different superfamily different <br />                                          overall architecture [37]. Third, substitution Phe Leu180 increases surface area <br />                                          van der Waal&apos;s interactions, apparently resulting increased contact deoxyribose <br />                                          mediate tighter binding slower release abasic site product. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          By illustrating consequences replacement Phe, work illuminates roles of <br />                                          Leu180 wild-type human AAG. As expected residue lines substrate <br />                                          binding pocket contacts substrate bases, Leu contributes, albeit modestly, wild-type <br />                                          excision activity, pattern substrate discrimination, overall compromise <br />                                          protection diverse sources DNA damage. Of particular interest, Leu180 appears <br />                                          modulate excision 7-meG, contributing balance BER pathways that <br />                                          remove methylation damage [33,38], raising question wild-type rate <br />                                          excision 7-meG, relatively benign lesion, necessary structural consequence of <br />                                          preserving activity deleterious adducts. <br />  <br />  <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                                         Page 16 <br />  <br />  <br />                               Supplementary Material <br />                                          Refer Web version PubMed Central supplementary material. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Acknowledgments <br />                                          We thank Dr. Elinor Adman molecular modeling valuable discussions Yunny Yunita, K. Marinov <br />                                          Georgi Dr. Eric Althoff technical assistance. This work supported NIH grants CA78885, CA77852, <br />                                          ES07033 L.A.L. CA55042 L.D.S. H.H.G supported Medical Scientist Training Program. <br />                                          L.D.S. American Cancer Society Research Professor. <br />  <br />  <br />                               References <br />                                          1. Sedgwick B. Repairing DNA-methylation damage. Nat Rev Mol Cell Biol. 2004; 5:148&#226;&#8364;&#8220;157. <br />                                             [PubMed: 15040447] <br />                                          2. Wyatt MD, Pittman DL. Methylating agents DNA repair responses: Methylated bases and <br />                                             sources strand breaks. Chem Res Toxicol. 2006; 19:1580&#226;&#8364;&#8220;1594. [PubMed: 17173371] <br />                                          3. Friedberg, EC.; Walker, GC.; Siede, W.; Wood, RD.; Schultz, RA.; Ellenberger, T. DNA Repair and <br />                                             Mutagenesis. American Society Microbiology; Washington, D.C.: 2005. <br />                                          4. Doublie S, Tabor S, Long AM, Richardson CC, Ellenberger T. Crystal structure bacteriophage <br />                                             T7 DNA replication complex 2.2 A resolution. Nature. 1998; 391:251&#226;&#8364;&#8220;258. [PubMed: 9440688] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          5. Beard WA, Osheroff WP, Prasad R, Sawaya MR, Jaju M, Wood TG, Kraut J, Kunkel TA, Wilson <br />                                             SH. Enzyme-DNA interactions required efficient nucleotide incorporation discrimination in <br />                                             human DNA polymerase beta. J Biol Chem. 1996; 271:12141&#226;&#8364;&#8220;12144. [PubMed: 8647805] <br />                                          6. Samson LD, Derfler B, Boosalis M, Call K. Cloning characterization 3-methyladenine <br />                                             DNA glycosylase cDNA human cells gene maps chromosome 16. Proc Natl Acad <br />                                             Sci U S A. 1991; 88:9127&#226;&#8364;&#8220;9131. [PubMed: 1924375] <br />                                          7. Engelward BP, Dreslin A, Christensen J, Huszar D, Kurahara C, Samson LD. Repair-deficient 3- <br />                                             methyladenine DNA glycosylase homozygous mutant mouse cells increased sensitivity to <br />                                             alkylation-induced chromosome damage cell killing. EMBO J. 1996; 15:945&#226;&#8364;&#8220;952. [PubMed: <br />                                             8631315] <br />                                          8. Engelward BP, Allan JM, Dreslin AJ, Kelly JD, Wu MM, Gold B, Samson LD. A chemical and <br />                                             genetic approach define biological consequences 3-methyladenine lesions the <br />                                             mammalian genome. J Biol Chem. 1998; 273:5412&#226;&#8364;&#8220;5418. [PubMed: 9479003] <br />                                          9. Scharer OD, Jiricny J. Recent progress biology, chemistry structural biology DNA <br />                                             glycosylases. Bioessays. 2001; 23:270&#226;&#8364;&#8220;281. [PubMed: 11223884] <br />                                          10. Hosfield DJ, Daniels DS, Mol CD, Putnam CD, Parikh SS, Tainer JA. DNA damage recognition <br />                                              repair pathway coordination revealed structural biochemistry DNA repair enzymes. <br />                                              Prog Nucleic Acid Res Mol Biol. 2001; 68:315&#226;&#8364;&#8220;347. [PubMed: 11554309] <br />                                          11. Glassner BJ, Rasmussen LJ, Najarian MT, Posnick LM, Samson LD. Generation strong <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              mutator <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> yeast imbalanced base excision repair. Proc Natl Acad Sci U S A. 1998; <br />                                              95:9997&#226;&#8364;&#8220;10002. [PubMed: 9707589] <br />                                          12. Posnick LM, Samson LD. Imbalanced base excision repair increases spontaneous mutation and <br />                                              alkylation sensitivity Escherichia coli. J Bacteriol. 1999; 181:6763&#226;&#8364;&#8220;6771. [PubMed: 10542179] <br />                                          13. Wyatt MD, Allan JM, Lau AY, Ellenberger TE, Samson LD. 3-methyladenine DNA glycosylases: <br />                                              structure, function, biological importance. Bioessays. 1999; 21:668&#226;&#8364;&#8220;676. [PubMed: 10440863] <br />                                          14. Wuenschell GE, O&apos;Connor TR, Termini J. Stability, miscoding potential, repair 2&#226;&#8364;&#178;- <br />                                              deoxyxanthosine DNA: implications nitric oxide-induced mutagenesis. Biochemistry. 2003; <br />                                              42:3608&#226;&#8364;&#8220;3616. [PubMed: 12653565] <br />                                          15. Hitchcock TM, Dong L, Connor EE, Meira LB, Samson LD, Wyatt MD, Cao W. Oxanine DNA <br />                                              glycosylase activity Mammalian alkyladenine glycosylase. J Biol Chem. 2004; 279:38177&#226;&#8364;&#8220; <br />                                              38183. [PubMed: 15247209] <br />  <br />  <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                                 Page 17 <br />  <br />  <br />                                          16. Bessho T, Roy R, Yamamoto K, Kasai H, Nishimura S, Tano K, Mitra S. Repair 8- <br />                                              hydroxyguanine DNA mammalian N-methylpurine-DNA glycosylase. Proc Natl Acad Sci U <br />                                              S A. 1993; 90:8901&#226;&#8364;&#8220;8904. [PubMed: 8415629] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          17. Dosanjh MK, Roy R, Mitra S, Singer B. 1,N6-ethenoadenine preferred 3-methyladenine as <br />                                              substrate cloned human N-methylpurine-DNA glycosylase (3-methyladenine-DNA <br />                                              glycosylase). Biochemistry. 1994; 33:1624&#226;&#8364;&#8220;1628. [PubMed: 8110764] <br />                                          18. Saparbaev M, Langouet S, Privezentzev CV, Guengerich FP, Cai H, Elder RH, Laval J. 1,N(2)- <br />                                              ethenoguanine, mutagenic DNA adduct, primary substrate Escherichia coli mismatch- <br />                                              specific uracil-DNA glycosylase human alkylpurine-DNA-N-glycosylase. J Biol Chem. 2002; <br />                                              277:26987&#226;&#8364;&#8220;26993. [PubMed: 12016206] <br />                                          19. Berdal KG, Johansen RF, Seeberg E. Release normal bases intact DNA native DNA <br />                                              repair enzyme. EMBO J. 1998; 17:363&#226;&#8364;&#8220;367. [PubMed: 9430628] <br />                                          20. O&apos;Brien PJ, Ellenberger T. Dissecting broad substrate specificity human 3-methyladenine- <br />                                              DNA glycosylase. J Biol Chem. 2004; 279:9750&#226;&#8364;&#8220;9757. [PubMed: 14688248] <br />                                          21. Aamodt RM, Falnes PO, Johansen RF, Seeberg E, Bjoras M. The Bacillus subtilis counterpart of <br />                                              mammalian 3-methyladenine DNA glycosylase hypoxanthine 1,N6-ethenoadenine as <br />                                              preferred substrates. J Biol Chem. 2004; 279:13601&#226;&#8364;&#8220;13606. [PubMed: 14729667] <br />                                          22. Lau AY, Scharer OD, Samson L, Verdine GL, Ellenberger T. Crystal structure human <br />                                              alkylbase-DNA repair enzyme complexed DNA: mechanisms nucleotide flipping base <br />                                              excision. Cell. 1998; 95:249&#226;&#8364;&#8220;258. [PubMed: 9790531] <br />                                          23. Lau AY, Wyatt MD, Glassner BJ, Samson LD, Ellenberger T. Molecular basis discriminating <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              normal damaged bases human alkyladenine glycosylase, AAG. Proc Natl <br />                                              Acad Sci U S A. 2000; 97:13573&#226;&#8364;&#8220;13578. [PubMed: 11106395] <br />                                          24. Guo HH, Choe J, Loeb LA. Protein tolerance random amino acid change. Proc Natl Acad Sci U <br />                                              S A. 2004; 101:9205&#226;&#8364;&#8220;9210. [PubMed: 15197260] <br />                                          25. Connor EE, Wyatt MD. Active-site clashes prevent human 3-methyladenine DNA glycosylase <br />                                              improperly removing bases. Chem Biol. 2002; 9:1033&#226;&#8364;&#8220;1041. [PubMed: 12323378] <br />                                          26. Bradford MM. A rapid sensitive method quantitation microgram quantities protein <br />                                              utilizing principle protein-dye binding. Anal Biochem. 1976; 72:248&#226;&#8364;&#8220;254. [PubMed: <br />                                              942051] <br />                                          27. Samson LD, Linn S. DNA alkylation repair induction cell death sister chromatid <br />                                              exchange human cells. Carcinogenesis. 1987; 8:227&#226;&#8364;&#8220;230. [PubMed: 3802405] <br />                                          28. Encell LP, Landis DM, Loeb LA. Improving enzymes cancer gene therapy. Nat Biotechnol. <br />                                              1999; 17:143&#226;&#8364;&#8220;147. [PubMed: 10052349] <br />                                          29. Fronza G, Gold B. The biological effects N3-methyladenine. J Cell Biochem. 2004; 91:250&#226;&#8364;&#8220; <br />                                              257. [PubMed: 14743385] <br />                                          30. Encell L, Shuker DE, Foiles PG, Gold B. The vitro methylation DNA minor groove <br />                                              binding methyl sulfonate ester. Chem Res Toxicol. 1996; 9:563&#226;&#8364;&#8220;567. [PubMed: 8728498] <br />                                          31. Kelly JD, Inga A, Chen FX, Dande P, Shah D, Monti P, Aprile A, Burns PA, Scott G, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Abbondandolo A, Gold B, Fronza G. Relationship DNA methylation mutational <br />                                              patterns induced sequence selective minor groove methylating agent. J Biol Chem. 1999; <br />                                              274:18327&#226;&#8364;&#8220;18334. [PubMed: 10373436] <br />                                          32. O&apos;Connor TR. Purification characterization human 3-methyladenine-DNA glycosylase. <br />                                              Nucleic Acids Res. 1993; 21:5561&#226;&#8364;&#8220;5569. [PubMed: 8284199] <br />                                          33. Rinne ML, He Y, Pachkowski BF, Nakamura J, Kelley MR. N-methylpurine DNA glycosylase <br />                                              overexpression increases alkylation sensitivity rapidly removing non-toxic 7-methylguanine <br />                                              adducts. Nucleic Acids Res. 2005; 33:2859&#226;&#8364;&#8220;2867. [PubMed: 15905475] <br />                                          34. Allinson SL, Sleeth KM, Matthewman GE, Dianov GL. Orchestration base excision repair by <br />                                              controlling rates enzymatic activities. DNA Repair (Amst). 2004; 3:23&#226;&#8364;&#8220;31. [PubMed: <br />                                              14697756] <br />                                          35. Gallivan JP, Dougherty DA. Cation-pi interactions structural biology. Proc Natl Acad Sci U S <br />                                              A. 1999; 96:9459&#226;&#8364;&#8220;9464. [PubMed: 10449714] <br />  <br />  <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                                  Page 18 <br />  <br />  <br />                                          36. Hu G, Gershon PD, Hodel AE, Quiocho FA. mRNA cap recognition: dominant role enhanced <br />                                              stacking interactions methylated bases protein aromatic chains. Proc Natl Acad <br />                                              Sci U S A. 1999; 96:7149&#226;&#8364;&#8220;7154. [PubMed: 10377383] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          37. Eichman BF, O&apos;Rourke EJ, Radicella JP, Ellenberger T. Crystal structures 3-methyladenine <br />                                              DNA glycosylase MagIII recognition alkylated bases. EMBO J. 2003; 22:4898&#226;&#8364;&#8220;4909. <br />                                              [PubMed: 14517230] <br />                                          38. Maher RL, Vallur AC, Feller JA, Bloom LB. Slow base excision human alkyladenine DNA <br />                                              glycosylase limits rate formation AP sites AP endonuclease 1 does stimulate base <br />                                              excision. DNA Repair (Amst). 2007; 6:71&#226;&#8364;&#8220;81. [PubMed: 17018265] <br />                                          39. Fishel ML, Seo YR, Smith ML, Kelley MR. Imbalancing DNA base excision repair pathway in <br />                                              mitochondria; targeting overexpressing N-methylpurine DNA glycosylase mitochondria <br />                                              leads enhanced cell killing. Cancer Res. 2003; 63:608&#226;&#8364;&#8220;615. [PubMed: 12566303] <br />                                          40. Berman HM, Westbrook J, Feng Z, Gilliland G, Bhat TN, Weissig H, Shindyalov IN, Bourne PE. <br />                                              The Protein Data Bank. Nucleic Acids Res. 2000; 28:235&#226;&#8364;&#8220;242. [PubMed: 10592235] <br />                                          41. Yang Y, Sass LE, Du C, Hsieh P, Erie DA. Determination protein-DNA binding constants and <br />                                              specificities statistical analyses single molecules: MutS-DNA interactions. Nucleic Acids <br />                                              Res. 2005; 33:4322&#226;&#8364;&#8220;4334. [PubMed: 16061937] <br />                                          42. O&apos;Brien PJ, Ellenberger T. Human alkyladenine DNA glycosylase uses acid-base catalysis for <br />                                              selective excision damaged purines. Biochemistry. 2003; 42:12418&#226;&#8364;&#8220;12429. [PubMed: <br />                                              14567703] <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                          Page 19 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 1. Modelled AAG bound eA-containing DNA close-up view substrate <br />                                          binding pocket randomized amino acids <br />                                          A, ribbon drawing AAG illustrating context residues mutated this <br />                                          work surround flipped-out base [coordinate set 1F4R (Lau et al (23) the <br />                                          Protein Data Bank (40)]. The flipped-out base (&#206;&#181;A, 1F4R) depicted space-filling <br />                                          (colored red), His136 Tyr127 (colored yellow), Leu180 (colored green). The <br />                                          flipped-out base fits closely binding pocket stacked Tyr127 side, <br />                                          His136 Tyr159 other. B, substrate binding pocket randomized <br />                                          residues. We randomized residues contacting &#206;&#181;A substrate (Tyr127, Ala134, Ala135, <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                         Page 20 <br />  <br />  <br />                                          His136, Tyr159, Cys167, Asn169, Leu180), flanking amino acids primary <br />                                          sequence, using oligonucleotide-directed mutagenesis. Residues randomized SK <br />                                          library shown yellow, residues randomized KX library green. Ala134 not <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          shown. The figure drawn using PyMOL (DeLano, W.L. The PyMOL Molecular <br />                                          Graphics System (2002) DeLano Scientific, Palo Alto, CA, USA. http://www.pymol.org). <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                       Page 21 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 2. Characterization unselected selected AAG mutant libraries <br />                                          A, unselected SK KX libraries. Library size measured plating LB- <br />                                          carbenicillin medium counting colonies; nucleotide amino acid changes were <br />                                          determined sequencing 72 individual mutants. For assessment MMS resistance, <br />                                          cultures log phase MV1932 cells expressing wild type AAG, SK KX library, <br />                                          inactive AAG (SK dummy vector) treated 0.2% MMS 1 hr 37 &#194;&#176;C, and <br />                                          surviving colonies counted. B, The distribution amino acid changes SK and <br />                                          KX libraries assessed sequences 73 unselected mutants 118 selected <br />                                          mutants survived MMS treatment described above. The distribution shows post- <br />                                          selection enrichment wild-type AAG sequence shift fewer substitutions <br />                                          clone. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                        Page 22 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 3. Amino acid substitutions unselected selected AAG mutants <br />                                          A, Conservation AAG primary sequence natural evolution. Aligned sequences of <br />                                          human AAG homologs shown; numbering refers human sequence. Absolutely <br />                                          conserved residues indicated dark gray numbers boxed. Residues showing <br />                                          conservative substitutions indicated light gray numbers underlined. <br />                                          B-E, Plasmid libraries harboring mutant AAGs created random mutagenesis were <br />                                          expressed glycosylase-deficient E. coli MV1932. Mutants complemented the <br />                                          methylating agent sensitivity MV1932 cells selected plasmid sequences <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                      Page 23 <br />  <br />  <br />                                          determined. Amino acid substitutions shown observed 72 mutants the <br />                                          unselected AAG SK KX libraries (B), 89 single mutants selected MMS resistance <br />                                          (C), 25 double mutants selected MMS resistance (D), 42 mutants selected Me-lex <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          resistance (E). Mutations 4 MMS-resistant triple mutants shown. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                        Page 24 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 4. MMS resistance conferred wild-type L180F mutant AAG <br />                                          A, MMS survival repair-proficienct E. coli AB1157, glycosylase-deficient MV1932, and <br />                                          MV1932 expressing wild-type L180F mutant AAG. The AAG proteins were <br />                                          expressed vector pTrcHis2 (Invitrogen) control T7 promoter. Cells <br />                                          grown OD600 0.6-0.85, adjusted &#226;&#710;&#188;108 cells/ml LB-carbenicillin medium and <br />                                          treated various concentrations MMS 1 hr 37 &#194;&#176;C. Cells diluted plated <br />                                          triplicate, colonies counted 24 hr. The results average two <br />                                          different experiments, carried different clone wild-type L180F AAG, <br />  <br />  <br />                                            DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                         Page 25 <br />  <br />  <br />                                          yielded similar results. Error bars shown standard deviations <br />                                          small indicated y-axis ranges 5 orders magnitude. At 0.2% <br />                                          MMS, concentration used selection, difference surviving fraction wild- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          type strain (0.0064% &#194;&#177; 0.0003%; mean standard deviation) mutant strain <br />                                          (0.0283% &#194;&#177; 0.0011%) highly significant (P &lt; 0.005, sample t-test, calculated using <br />                                          OriginPro software, OriginLab Corporation, Northampton, MA). B, Western blot. Aliquots <br />                                          cells taken various times MMS treatment subjected immunoblotting <br />                                          using anti-AAG polyclonal antibody previously described [39]. C, Expression wild- <br />                                          type AAG (closed circles) L180F mutant protein (open circles) quantified by <br />                                          phosphor imaging using ImageQuant software (GE Healthcare). <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                      Page 26 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 5. Excision 3-meA 7-meG DNA homogeneous wild-type L180F AAG <br />                                          A, wild-type AAG L180F mutant protein purified described Experimental <br />                                          Procedures. Samples (4 &#206;&#188;g) wild-type fractions I-IV L180F fraction IV were <br />                                          subjected electrophoresis 12.5% SDS-polyacrylamide gel protein visualized <br />                                          staining Coomassie Brilliant Blue G-250. The mobility molecular weight <br />                                          standards tracking dye indicated arrows. B,C, DNA glycosylase activity wild- <br />                                          type (closed circles) L180F (open circles). AAG assayed function time by <br />                                          measuring release [3H]-labelled 3-meA 7-meG methylated calf thymus <br />  <br />  <br />                                            DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                   Page 27 <br />  <br />  <br />                                          DNA, described Experimental Procedures. Excision 3-meA 7-meG was <br />                                          measured reactions containing 50 500 nM enzyme, respectively. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                        <br /> </body></html>