<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Plant Cell, Vol. 16, 1091&#226;&#8364;&#8220;1104, May 2004, www.plantcell.org &#194;&#170; 2004 American Society Plant Biologists <br />  <br />  <br />  <br />  <br /> ATR Regulates G2-Phase Cell-Cycle Checkpoint <br /> in Arabidopsis thaliana <br />  <br /> Kevin Culligan,a,1 Alain Tissier,b,2 Anne Britta <br /> a Sectionof Plant Biology, University California, Davis, California 95616 <br /> b Commissariat  a` l&#226;&#8364;&#8482;Energie Atomique Cadarache, Direction des Sciences du Vivant De&#194;&#180;partement d&#226;&#8364;&#8482;Ecophysiologie Ve&#194;&#180;ge&#194;&#180;tale et de <br /> Microbiologie, Laboratoire Radiobiologie Ve&#194;&#180;ge&#194;&#180;tale, 13108 St. Paul-lez-Durance Cedex, France <br />  <br />  <br />  <br /> Ataxia telangiectasia-mutated Rad3-related (ATR) plays central role cell-cycle regulation, transmitting DNA damage <br /> signals downstream effectors cell-cycle progression. In animals, ATR essential gene. Here, that <br /> Arabidopsis (Arabidopsis thaliana) atr&#195;&#191;/&#195;&#191; mutants viable, fertile, phenotypically wild-type absence of <br /> exogenous DNA damaging agents exhibit altered expression AtRNR1 (ribonucleotide reductase large subunit) and <br /> alteration damage-induced cell-cycle checkpoints. atr mutants hypersensitive hydroxyurea (HU), <br /> aphidicolin, UV-B light mildly sensitive g-radiation. G2 arrest observed response g-irradiation in <br /> both wild-type atr plants, albeit slightly different kinetics, suggesting ATR plays secondary role response <br /> <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> double-strand breaks. G2 arrest observed wild-type plants response aphidicolin defective atr <br /> mutants, resulting compaction nuclei subsequent cell death. By contrast, HU-treated wild-type atr plants <br /> arrested G1 showed obvious signs cell death. We propose that, plants, HU invokes novel checkpoint <br /> responsive low levels deoxynucleotide triphosphates. These results demonstrate important role cell-cycle <br /> checkpoints ability plant cells sense cope problems associated DNA replication. <br />  <br />  <br />  <br /> INTRODUCTION                                                                   (RNR), catalyzes final reduction step production <br />                                                                                dNTPs. Genetic studies Saccharomyces cerevisiae suggest <br /> The detection DNA damage important aspect damage                   elevation dNTP levels (through transcriptional and/or <br /> resistance. Cells routinely express DNA repair                      posttranslational regulation RNR) response DNA damage <br /> mechanisms, instead upregulate functions required,                result efficient translesion DNA synthesis, enabling <br /> concomitantly slowing cell cycle permit time repair                 cell complete replication spite presence of <br /> before S-phase (as replication past damaged sites                       persisting lesions (Tanaka et al., 2000; Chabes et al., 2003; Yao <br /> mutagenic) M-phase (after point double-strand                  et al., 2003). <br /> breaks lesions daughter strand gaps far                         The proteins sense damaged single-stranded DNAs <br /> more problematic repair). Damage generally detected                   trigger elaborate responses extensively <br /> protein complexes form single-stranded DNA (Crowley                  characterized organisms ranging bacteria human <br /> and Courcelle, 2002; Zou Elledge, 2003), created                  (Crowley Courcelle, 2002; Melo Toczyski, 2002). In <br /> replication fork physically blocked DNA lesions                plants, however, regulation DNA replication repair and <br /> the process DNA repair (e.g., homologous recombination,                     cell cycle response DNA damage beginning be <br /> nucleotide excision repair, mismatch repair). These damage                 understood. <br /> sensors activate downstream checkpoint mechanisms                         The Schizosaccharomyces pombe Rad3 homologs (ataxia <br /> prevent progression phase cell cycle              telangiectasia-mutated Rad3-related [ATR] ataxia <br /> as upregulate DNA repair (Nyberg et al., 2002).                                telangiectasia-mutated [ATM] mammals, Mec1 S. cerevi- <br />    Some damage sensors involved regulation                  siae) essential regulators cell-cycle checkpoints, sensing <br /> deoxynucleotide triphosphate (dNTP) pools, turn                   DNA damage and/or single-stranded DNA activating <br /> influence mode DNA replication. One primary targets              downstream effectors cell-cycle progression DNA repair. <br /> for dNTP pool regulation enzyme ribonucleotide reductase                ATM activated primarily DNA double-strand breaks. ATM- <br />                                                                                deficient plants animals hypersensitive g-irradiation <br />                                                                                replication-blocking agents UV-B light, <br /> 1 To correspondence addressed. E-mail kmculligan@ <br />                                                                                hydroxyurea (HU), aphidicolin (Abraham, 2001; Garcia et al., <br /> ucdavis.edu; fax 530-752-5410. <br /> 2 Current address: Librophyt SAS, Centre Cadarache, Ba      &#203;&#8224; timent 185,   2003; K. Culligan A. Britt, unpublished data). Furthermore, <br /> 13108 St. Paul-lez-Durance Cedex, France.                                      Arabidopsis (Arabidopsis thaliana) ATM homolog required <br /> The author responsible distribution materials integral           transcriptional induction repair genes response to <br /> findings presented article accordance policy described     g-irradiation efficient progression meiosis <br /> in Instructions Authors (www.plantcell.org) is: Kevin M. Culligan <br /> (kmculligan@ucdavis.edu). <br />                                                                                (Garcia et al., 2003), role plant cell-cycle responses <br /> Article, publication date, citation information            DNA damage determined. In animals, <br /> www.plantcell.org/cgi/doi/10.1105/tpc.018903.                                  activated ATM phoshorylates downstream components of <br />  1092      The Plant Cell <br />  <br />  <br />  <br /> checkpoint pathways include p53, BRCA1, NBS1,                     Arabidopsis genome homologs ATR, using S. pombe <br /> CHK2, initiating G1-, S-, G2-phase arrest and/or apoptosis.             Rad3 C-terminal domain query sequence, found <br />    By contrast, ATR activated primarily agents block            candidate genes. Phylogenetic analysis corre- <br /> the progression replication forks. ATR dominant-negative                sponding predicted full-<span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> protein sequences indicated that <br /> and conditional <span id='am-319' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-320' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-326' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-332' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-341' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-350' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-364' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-379' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-380' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-392' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-393' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-394' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-395' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>knockout</span> mammalian cell lines display hyper-               ortholog Target Rapamycin (TOR), an <br /> sensitivity UV-B light, HU, aphidicolin sen-           ortholog ATM, ortholog ATR (Figure 1A). <br /> sitive g-radiation. Thus, animal cells, ATR thought            No candidate genes fell broad categories. The <br /> play generalized role response DNA damage                ATR-orthologous sequence (previously reported GenBank as <br /> ATM. Activated ATR phosphorylates CHK1 initiate G2-phase                mRNA AtRad3 [AB040133], designate AtATR) <br /> arrest act points cell cycle,              revealed high similarity entire length ATR <br /> evidenced ATR-dependent phosphorylation BRCA1                        proteins, similarity ATM TOR sequences was <br /> (Tibbetts et al., 2000; Melo Toczyski, 2002). However,             limited highly conserved middle C-terminal <br /> determination extent ATR&#226;&#8364;&#8482;s role genome main-             regions. We confirmed GenBank sequence intron/exon <br /> tenance complicated fact ATR homozygous               boundaries isolating sequencing cDNAs AtATR <br /> knockouts die early embryogenesis (Brown Baltimore,                 ( 8.1 kb, ecotype Wassilewskija [Ws]) using RT-PCR, the <br /> 2000; Klein et al., 2000); cre/lox-induced nulls survive           resulting sequence predicted 17 exons encoding protein of <br /> a rounds cell division, dominant-negative re-               2702 amino acids (Figure 1B). <br /> duction-of-function cell lines yielded conflicting results <br /> (Cliby et al., 1998; Wright et al., 1998). Thus, tractable genetic <br /> system <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>null allele</span> ATR tolerated         Identification T-DNA Insertion-<span id='am-241' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-242' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span>Mutation</span> Alleles <br /> the development higher organism particularly                 AtATR <br /> desirable. <br />                                                                            To study function ATR Arabidopsis, searched for <br />    Here, investigate effects ATR T-DNA insertion <br />                                                                            lines T-DNA insertions ATR. Three independent lines <br /> allele plant Arabidopsis. The allele 6-kb insertion <br />                                                                            identified different collections termed atr-1 <br /> within kinase domain presumably null. We that <br />                                                                            (Arabidopsis Knockout Facility, ecotype Ws), atr-2 (SIGnAL data- <br /> the mutant plant (atr) viable developmentally normal in <br />                                                                            <span id='am-397' about='xsp:base' typeof='owl:Thing'>base</span>, ecotype Columbia), atr-3 (FLAG database, ecotype <br /> the absence exogenous DNA-damaging treatments is <br />                                                                            Ws). atr-1 atr-2 contain insertions middle ATR, in <br /> hypersensitive replication-blocking agents UV-B light, <br />                                                                            exons 7 10, respectively (Figure 1B). atr-3 contains single, <br /> HU, aphidicolin. Thus, AtATR does appear act a <br />                                                                             6-kb T-DNA insertion highly conserved C-terminal <br /> functional homolog mammalian counterpart. However, the <br />                                                                            kinase (PI3Kc) domain (Figures 1B 1D), causes 30-bp <br /> Arabidopsis atr mutants differ mammalian mutants not <br />                                                                            deletion adjacent catalytic Asp required kinase function. <br /> only terms viability degree sensitivity <br />                                                                            RT-PCR analysis C-terminal region ATR confirms <br /> to g-radiation. <br />                                                                            transcript absent atr-3 (Figure 7). Thus, believe <br />    In course experiments, observed <br />                                                                            atr-3 likely null mutation focused major- <br /> a differential response HU versus aphidicolin wild-type <br />                                                                            ity characterization line (termed atr below). <br /> cells, HU primarily inducing G1 arrest, aphidicolin <br />                                                                            Nonetheless, lines exhibit similar recessive <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>phenotype</span> <br /> induced arrest G2. This stands contrast G2 arrest <br />                                                                            (see below) cosegregates respective T-DNA <br /> induced agents yeast mammals (at concen- <br />                                                                            insertions ATR. <br /> trations allow S-phase progression) suggests plants <br /> may possess novel G1 checkpoint response low dNTP <br /> levels. The G2 arrest response aphidicolin abrogated          Neither AtATR AtATM Is Required Normal <br /> atr mutants, suggesting ATR regulates G2 checkpoint               Somatic Development <br /> response replication blocks, does mammals. Moreover, <br /> programmed cell death observed atr mutants response              All atr homozygous alleles phenotypically wild-type in <br /> to aphidicolin, HU, experimental conditions.             growth (of root shoot) development (leaf flower <br /> We propose model suggests HU aphidicolin               development, seed set viability) standard conditions. <br /> hypersensitivities atr mutants result defect      Thus, ATR essential normal somatic development in <br /> G2 checkpoint response blocked replication forks.                       Arabidopsis. By contrast, ATR essential gene animals; <br />                                                                            ATR knockout mice die early embryogenesis (Brown and <br /> RESULTS                                                                    Baltimore, 2000; Klein et al., 2000), conditional <br />                                                                            knockout human cell lines divide times dying <br /> Identification ATR Ortholog Arabidopsis                          (Cortez et al., 2001; Brown Baltimore, 2003). Because ATR <br />                                                                            ATM paralogs known play partially overlapping roles <br /> ATR orthologs, including Mec1 S. cerevisiae Rad3 S.              animal cells (Abraham, 2001), possible ATM could <br /> pombe, members large gene family eukaryotes               compensate ATR deficiency Arabidopsis. To test this <br /> encode Ser-Thr kinases, C-terminal catalytic domains                 hypothesis, established atr atm double mutants found <br /> share similarity yeast mammalian phosphoinositide                   obvious phenotypic differences (in vegetative growth and <br /> 3-kinases (Tibbetts Abraham, 2000). We searched                    development) compared wild-type single-mutant <br />                                                                                                           AtATR Regulates G2 Checkpoint            1093 <br />  <br />  <br />  <br />  <br /> Figure 1. Phylogenetic Analysis, Structure, T-DNA Insertions AtATR. <br />  <br /> (A) The neighbor-joining tree Rad3-like protein sequences performed described (Culligan et al., 2000). Numbers branch represent <br /> the number times branch 1000 bootstrap replicas. Arabidopsis sequences bold. No DNA-PKcs (DNA&#226;&#8364;&#8220;protein kinase catalytic <br /> subunit)-like sequences identified Arabidopsis genome. <br /> (B) Overview AtATR gene, positions atr-1, atr-2, atr-3 T-DNA insertions, corresponding cDNA structure. Numbered gray boxes <br /> indicate exons AtATR; positions T-DNA insertions denoted triangles. The cDNA structure (bottom) shows corresponding highly <br /> conserved domains ATM, ATR, TOR, FKBP12-rapamycin-associated protein sequences, including kinase domain denoted black <br /> rectangle (PI3Kc). The cDNA sequence confirmed isolating sequencing independent cDNAs AtATR (ecotype Ws) using RT-PCR. The <br /> resulting sequence predicts 17 exons encode protein 2702 amino acids. <br /> (C) Structure atr-3 T-DNA insertion. Capital letters denote T-DNA insertion sequence. Corresponding amino acids shown DNA <br /> sequence. The left-border sequence encodes 10 new amino acids AtATR reading frame premature stop (boxed TGA). The <br /> approximate size entire insertion 6 kb. <br /> (D) Partial alignment kinase domain atr-3 inserted: 1 denotes 30-bp deletion (equivalent 10 amino acids) caused insertion, and <br /> 2 denotes position catalytic Asp required kinase function. <br />  1094     The Plant Cell <br />  <br />  <br />  <br />  <br /> Figure 2. Siliques Harvested Mature Wild-Type, atr, atm-1 (Ecotype Ws) Mutants. <br /> Wild-type, atr&#195;&#191;/&#195;&#191;, atr&#195;&#191;/&#195;&#191; atm1/&#195;&#191; average produce  50 seeds silique. atm plants, partially sterile, produce range silique <br /> sizes fewer (&lt;10 average) seeds. The siliques double atr atm line produced seeds unable outcross males or <br /> females, suggesting complete sterility. <br />  <br />  <br /> plants, indicating ATR ATM play essential role            We wanted determine RNR transcription is <br /> during normal vegetative (somatic) growth.                                 regulated AtATR-dependent manner challenged and/or <br />    The double mutants, however, completely sterile (Figure            unchallenged plants. We searched GenBank identified <br /> 2). atr mutant plants fertile produce normal gametes;              Arabidopsis homolog RNR1, large subunit RNR <br /> pollen staining viable spores revealed differences versus           (accession number AF092841). Transcription RNR1 has <br /> the wild type, siliques (normal seed set). atm           shown regulated cell-cycle dependent <br /> mutant plants, however, partially sterile abundant          manner tobacco (Nicotiana tabacum) (Chaboute&#194;&#180; et al., 1998, <br /> chromosomal fragmentation meiosis, resulting re-               2002), transcription induced DNA damage plant <br /> duction viable pollen reduced seed set (Garcia et al.,              (M.E. Chaboute&#194;&#180;, personal communication). We probed RNA gel <br /> 2003). We suggest AtATM AtATR play partially redun-               blots AtRNR1 AtRNR1 transcript reduced <br /> dant roles meiosis Arabidopsis.                                  atr mutants,  40% versus wild type under <br />                                                                            standard conditions (Figure 3), suggesting ATR required <br /> AtATR Is Required Full Transcriptional Induction                support normal expression RNR1. Plants treated with <br /> Large Subunit RNR, AtRNR1                                               chronic doses UV-B light, sufficient inhibit eliminate <br />                                                                            growth, display level RNR1 transcript similar that <br /> In budding yeast S. cerevisiae, Mec1 (the ATR ortholog)                observed standard (UV-free) conditions, atr <br /> regulates expression RNR transcriptional                plants exhibit reduced levels RNR1 transcript. Interestingly, <br /> posttranscriptional levels (Zhao et al., 2001). Null alleles mec1       plants treated acute doses UV-B light displayed <br /> are lethal, lethality suppressed mutations genes         transient inhibition AtRNR1 transcript (at 2 h postirradia- <br /> that negatively regulate RNR (Huang et al., 1998; Zhao et al.,             tion), followed return standard condition levels 6 h <br /> 1998), transcriptional repressor Crt1 (Huang et al.,           postirradiation. This UV-induced suppression ATR de- <br /> 1998) inhibitor RNR activity Sml1 (Zhao et al., 1998).           pendent, again, overall level RNR1 transcript is <br /> Derepression RNR transcription response DNA damage                reduced (Figure 3). <br /> is dependent Mec1 downstream kinase Dun1,                    Because used HU (an inhibitor RNR activity) DNA <br /> indirectly induces inhibitory phosphorylation Crt1 (Elledge          replication-blocking agent (see below) characterize pheno- <br /> et al., 1992; Zhao Rothstein, 2002). Whether Mec1 ATR               typic effects atr mutant, wanted determine the <br /> also play role normal cell-cycle specific regulation RNR       effects HU treatment RNR expression plants. <br /> transcription unchallenged cells remains unclear.                       Surprisingly, presence HU, RNR1 expression same <br />                                                                                                        AtATR Regulates G2 Checkpoint         1095 <br />  <br />  <br />  <br />                                                                                   To determine role Arabidopsis ATR cell-cycle <br />                                                                                checkpoint regulator response g-irradiation, established <br />                                                                                homozygous wild-type atr mutant lines carrying cyclinB1 <br />                                                                                promoter coupled b-glucuronidase (PcyclinB1:GUS) reporter <br />                                                                                construct. This construct harbors GUS gene fused mitotic <br />                                                                                destruction sequence cyclinB1 promoter (Colon-Carmona <br />                                                                                et al., 1999). The GUS fusion gene expressed entry into <br />                                                                                G2 (via cyclinB1 promoter), fusion protein is <br />                                                                                degraded just anaphase mitosis. GUS-positive <br />                                                                                cells stain blue G2/early M, allowing visually monitor the <br />                                                                                progression cell cycle root meristems (Preuss Britt, <br />                                                                                2003). To determine G2-specific arrest (versus post-G2 mitotic <br />                                                                                arrest), stained root tips 49,6-diamidino-2- <br />                                                                                phenylindole (DAPI) visualize mitotic nuclei. In response to <br />                                                                                400 Gy g-radiation, atr mutants showed slightly altered <br />                                                                                cell-cycle response compared wild type; both <br /> Figure 3. RNA Gel Blot Analysis Wild Type atr.                      wild-type atr meristematic cells arrest G2 (DAPI staining <br /> RNA samples prepared 7-d-old wild-type atr plants                revealed accumulation mitotic figures) differ slightly in <br /> (whole seedlings) sown MS agar plates plus 1 mM HU, presence         timing response. For instance, 8 h postirradiation, atr <br /> of UV-B (chronic dose), left untreated (n.t.). For acute doses UV-B      root tips accumulated G2 cells versus wild type, at <br /> light, wild type atr grown 7 d MS agar plates          1 d postirradiation root tips accumulated G2 cells ver- <br /> irradiated UV-B light (10 kJ/m2) dark harvested 2 h    sus wild type (Figure 4B). This suggests Arabidopsis <br /> 6 h UV irradiation. Approximately 30 mg (lanes 1 6) 25 mg (lanes <br />                                                                                ATR absolutely required G2 checkpoint response to <br /> 7 10) total RNA sample blotted probed with <br />                                                                                acute doses g-radiation does influence rate induction <br /> a putative AtRNR1 (GenBank AF092841) mRNA sequence. The AtRNR1 <br /> mRNA sequence 2.65 kb. The bands corresponding 4.4 2.4 kb <br />                                                                                persistence. <br /> of RNA ladder (Ambion) shown left denoted arrows. <br /> The ethidium bromide&#226;&#8364;&#8220;stained gel shown (bottom), relative           Arabidopsis atr Mutants Are Hypersensitive to <br /> rates expression (Rel. exp.) determined respect wild-type      Replication-Blocking Agents <br /> no-treatment control. This experiment repeated independently <br /> isolated RNA samples similar results.                                hsATR, scMec1, spRad3 deficiencies associated <br />                                                                                hypersensitivity replication blocking agents. For exam- <br /> in wild type atr (Figure 3) levels similar            ple, yeast mec1 mutants fail undergo G2 arrest normally <br /> wild type standard conditions. This suggests                  induced stalled replication instead proceed directly into <br /> are additional ATR-independent levels RNR1 regulation                    mitosis. Consequently, mutant cells accumulate single-strand <br /> Arabidopsis consistent previous studies showing               gaps double-strand breaks, leading cell death (Weinert <br /> HU induces RNR expression plants (Chaboute&#194;&#180; et al., 1998).                  et al., 1994). In animals, failure arrest induces apoptosis <br />                                                                                (Brown Baltimore, 2000; Cortez et al., 2001). To determine <br />                                                                                ATR required plant cell-cycle responses to <br /> Arabidopsis atr Mutants Are Mildly Sensitive                                replication blocks, challenged wild-type homozygous atr <br /> Acute Doses g-Radiation Proficient                                      mutant plants replication-blocking agents HU (1 mM), <br /> in g-Induced G2 Arrest                                                         aphidicolin (12 mg/mL), UV-B light. All agents <br />                                                                                inhibit replication fundamentally different ways. HU inhibits <br /> Conditional mutants ATR/Mec1 mammalian cells S.                      RNR, depleting available dNTPs DNA polymerases. <br /> cerevisiae, Rad3 mutants S. pombe,                           Aphidicolin inhibitor replicative DNA polymerases d <br /> hypersensitive g-radiation (Jimenez et al., 1992; Cliby et al.,             e, UV-B light creates physical replication blocks (e.g., <br /> 1998; Wright et al., 1998). Mutants ATM/scTel1, including                   cyclobutane pyrimidine dimers) DNA. We chose focus our <br /> those Arabidopsis, hypersensitive g-radiation                   analysis root meristems (Wiedemeier <br /> (Sedgwick Boder, 1991; Garcia et al., 2003). This sensitivity              et al., 2002) HU aphidicolin inhibit root <br /> is generally interpreted failure checkpoint responses               growth wild-type plants, suggesting agents are <br /> double-strand breaks (Abraham, 2001) involve                      absorbed root. We germinated wild-type homozy- <br /> a failure induce DNA repair genes (Garcia et al., 2003). To                 gous atr seeds control MS agar plates plates <br /> determine ATR plays role g-resistance Arabi-                   containing HU aphidicolin, selecting dose each <br /> dopsis, challenged wild-type, atr, atm lines sublethal             agent mild perceptible effects growth the <br /> doses g-radiation. In contrast atm mutants, atr lines                  wild-type root. Both wild type atr grew similar <br /> exhibited mild sensitivity g-radiation. The root length atr         rates control plates. On plates containing 1 mM HU, wild- <br /> was reduced  30% versus wild type 7 d seeds                  type root growth reduced  40%, atr root <br /> g-irradiated 200 Gray (Gy), atm root length                     growth reduced &gt;95% (Figures 5A 5D) produced <br /> reduced &gt;95% (Figure 4A).                                                   abnormally long root hairs densely clustered the <br />  1096      The Plant Cell <br />  <br />  <br />  <br />  <br /> Figure 4. Root Growth Cell-Cycle Arrest Phenotypes g-Irradiated Wild-Type atr Plants. <br /> (A) Wild-type, atm, atr seeds g-irradiated dose 200 Gy. The seeds immediately germinated MS agar plates, root growth <br /> was measured 2 weeks. No-irradation controls grew described (Garcia et al., 2003) Figure 5D. <br /> (B) Five-day-old wild-type atr seedlings containing PcyclinB1:GUS fusion construct (as described text Figure 6) g-irradiated with <br /> 400 Gy, individual wild-type atr plants harvested 8 h, 1 d, 2 d time points GUS staining (n &gt; 12). Two representative examples <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> treatment shown. <br />  <br />  <br />  <br />  <br /> root tip. Moreover, lateral root growth initiated earlier              AtATR Regulates G2 Checkpoint Response to <br /> atr HU plates compared control plates. The shoot                   Replication Blocks <br /> reduced size produced anthocyanins (stress-induced <br /> pigments), indirect effect lack root                   Mec1 ATR required G2 checkpoint response to <br /> growth. The differences wild type atr                     replication blocks fungi mammals, respectively (Melo and <br /> (40% versus &gt;95% growth inhibition, overproduction root                     Toczyski, 2002). To determine ATR performs similar <br /> hairs lateral roots) observed germinated                      function plants, GUS DAPI stained homozygous wild- <br /> plates containing 12 mg/mL aphidicolin 5-d-old seed-                type atr mutant lines carrying PcyclinB1:GUS reporter <br /> lings transferred plates containing HU aphidicolin,                 construct (described above) growth HU aphidicolin. <br /> starting  2 d transfer (Figures 5C 5D). Furthermore,                 As shown Figure 6, wild type atr displayed typical <br /> chronic doses UV-B light similar phenotypic effect                    patterns PcyclinB1:GUS expression control plates ( 10 20 <br /> on atr roots compared wild type (Figure 5B);                      blue cells root tip), representing fraction G2/early <br /> mutant retarded (to lesser extent dose)                  M cells present normal growth. By contrast, meri- <br /> its root growth formed root hairs tip root.                  stematic cells aphidicolin treated wild-type plants displayed <br /> Thus, treatments block replication differ                substantial increase number PcyclinB1:GUS-expressing <br /> mechanism inhibition produced similar gross phenotypic                      cells density staining (Figure 6). Using <br /> effects atr roots.                                                          DAPI staining visualize mitotic nuclei, observed no <br />                                                                                                           AtATR Regulates G2 Checkpoint             1097 <br />  <br />  <br />  <br />  <br /> Figure 5. Phenotypes atr Challenged HU, Aphidicolin, UV-B Light. <br /> (A) Wild-type atr seeds germinated MS plates MS plus 1 mM HU grown 3 weeks. <br /> (B) Wild-type atr seeds germinated MS plates grown 3 weeks presence absence UV-B light (see Methods). <br /> (C) Wild-type atr seeds germinated MS plates grown 5 d transferred plates containing 12 mg/mL aphidicolin (Aph) 5 d. <br /> (D) Five-day-old wild-type atr seedlings grown presence absence (n.t., treatment) HU aphidicolin (Aph). However, to <br /> make direct comparison effects HU versus aphidicolin, employed concentrations HU aphidicolin inhibited wild-type root <br /> growth  40% treatments, case, 1 mM HU 12 mg/mL aphidicolin. We measured root growth (n &gt; 30) treatment for <br /> 8 d transfer. Standard deviations shown error bars. <br />  <br />  <br />  <br /> significant aphidicolin-induced accumulation mitotic figures              occurred atr root meristems appeared slightly <br /> either wild-type atr meristematic cells, ruling post-G2             accelerated. <br /> mitotic arrest likely cause PcyclinB1:GUS accumula-                   To confirm PcyclinB1:GUS analysis, employed semi- <br /> tion. This indicates dividing cells wild-type plants                quantitative RT-PCR, primers specific cyclinB1 gene <br /> experiencing aphidicolin-induced replicational stress spend                     (which does amplify PcyclinB1:GUS transcript), un- <br /> a greater fraction cell cycle G2 atr plants            treated, HU-, aphidicolin-treated wild-type atr root tips. As <br /> defective G2 arrest (Figure 6).                                         shown Figure 7, cyclinB1 gene expression reduced  20% <br />    In contrast aphidicolin response, HU did induce                 (relative untreated control root tips) wild type when <br /> G2-phase arrest wild-type roots. In fact, decline                   grown presence HU elevated nearly threefold in <br /> fraction G2 cells observed day 4 6 (as measured                  presence aphidicolin, consistent PcyclinB1:GUS <br /> by lack PcyclinB1:GUS staining; Figure 6). The cells                 experimental conclusion HU induces G1- S-phase arrest <br /> presumably arresting G1- S-phase increase                   aphidicolin induces G2 arrest. Furthermore, aphidicolin- <br /> mitotic figures observed DAPI staining. This response                   treated atr root tips display enhancement cyclinB1 <br />  1098      The Plant Cell <br />  <br />  <br />  <br />  <br /> Figure 6. GUS Staining Root Tips. <br /> Five-day-old wild-type atr-3 seedlings (carrying PcyclinB1:GUS) transferred aphidicolin (Aph), HU, control (n.t.) plates, root growth <br /> was measured day transfer 8 d (Figure 5D). Individual wild-type atr plants harvested time point GUS staining of <br /> root tips. Wild-type atr grown 5 d MS plates transferred control plates plates containing 1 mM HU 12 mg/mL of <br /> aphidicolin. These experiments repeated times similar results; thus, representative pictures shown. GUS stained root tips <br /> from 2, 4, 6 d transfer shown. At 12 individual plants harvested stained treatment time point. Two representative <br /> examples shown treatment. <br />  <br />  <br /> transcript response aphidicolin, suggesting lack             organized cells enlarged, nuclei similar in <br /> G2 arrest atr.                                                               appearance wild type (round slightly diffuse). All cells <br />    To distinguish G1- S-phase arrest wild-type HU-                root cap defined nucleus (Figure 8A). The <br /> treated roots, employed RT-PCR analysis S-phase                       meristems atr grown aphidicolin disorganized, <br /> specific gene Histone H4. In plants, expression Histone H4               appearance nuclei differed observed in <br /> has shown strongly induced entry S-phase                   wild type (all conditions) atr grown HU. Starting on <br /> and maintained G2 (Reichheld et al., 1995; Menges                   day 4, meristematic nuclei near tip condensed, <br /> Murray, 2002). If HU induces S-phase arrest (e.g., wild-type                 DAPI staining reduced relative atr grown HU the <br /> root tips), predict concomitant increase S-phase&#226;&#8364;&#8220;                 wild type (all conditions). By day 6, cells meristematic <br /> specific gene expression. Likewise, G1-phase arrest                       zone aphidicolin-treated roots completely lost nuclei, <br /> predict concomitant decrease S-phase&#226;&#8364;&#8220;specific gene ex-                     leaving blue haze (Figures 8A 8B). This indicates that <br /> pression. As shown Figure 7, Histone H4 expression reduced                atr mutant cells undergo nuclear degradation (cell death) when <br /> in HU-treated wild-type root tips elevated aphidicolin-                  treated aphidicolin, HU, dose levels agents <br /> treated wild-type root tips, suggesting G1- G2-phase arrest,                inhibit wild-type growth  40%. Although cell death <br /> respectively. In atr mutant, Histone H4 expression elevated              seen higher doses HU (data shown), these <br /> in untreated root tips drastically reduced HU-            higher doses inhibited wild-type growth &gt;95%. This suggests that <br /> aphidicolin-treated root tips. This suggests atr mutant,               (1) aphidicolin-induced AtATR-dependent G2 arrest plays <br /> although phenotypically normal root growth, induces S-                  critical role protecting genome, (2) additional <br /> phase arrest (cyclinB1 induced) standard growth                    factors, G1 arrest induced HU, rescue atr <br /> conditions presence replication blocking agents,                  meristems massive genomic instability induced pro- <br /> arrests and/or exits cell cycle G1.                                      gression M-phase absence complete replication. <br />    Altogether, data suggest independent check- <br /> points responsible arrest phenotypes observed the <br />                                                                                 DISCUSSION <br /> wild type versus atr: aphidicolin-induced G2 checkpoint that <br /> is AtATR dependent HU-induced G1 checkpoint.                              ATR plays central role cell&#226;&#8364;&#8482;s response DNA damage by <br />                                                                                 activating cell-cycle checkpoints allow time DNA repair. <br /> Aphidicolin, Not HU, Induces Cell Death atr                              We shown Arabidopsis atr mutants viable and <br />                                                                                 normal standard growth conditions hypersensi- <br /> To characterize different cellular responses HU              tive replication blocks mildly sensitive g-radiation. <br /> aphidicolin, DAPI stained wild-type atr root tips (time                  Furthermore, shown AtATR required the <br /> points treatments above) observe integrity                 G2-phase checkpoint response replication blocks is <br /> nuclei. As seen Figure 8A, wild-type roots organized                absolutely required G2-arrest response double- <br /> meristems round slightly diffuse nuclei,                      strand breaks (using acute doses g-radiation). The differences <br /> retained morphology HU aphidicolin. However, atr                    seen response HU, aphidicolin, g-irradiation suggest <br /> exhibited different cellular morphologies HU aphidi-                additional checkpoints play important roles cell- <br /> colin. The cell files atr meristems grown HU dis-                    cycle cell-death phenotypes observed. <br />                                                                                                       AtATR Regulates G2 Checkpoint          1099 <br />  <br />  <br />  <br />                                                                                 The cell-cycle arrest phenotypes atr comparison the <br />                                                                              wild type suggest mild g-sensitivity atr probably not <br />                                                                              absolute effect abrogated checkpoint G2; wild <br />                                                                              type atr display accumulation GUS meristem <br />                                                                              consistent G2 arrest, slightly different kinet- <br />                                                                              ics. Although know AtATM contributes the <br />                                                                              g-induced G2 checkpoint, possible AtATR AtATM <br />                                                                              play partially redundant roles response. In case, AtATM <br />                                                                              partially compensate AtATR deficiency or <br />                                                                              cell cycle. In mammalian cells, experiments employing <br />                                                                              conditional knockout (null) alleles ATR demonstrated that <br />                                                                              G2 arrest abrogated response g-radiation, indicating that <br />                                                                              ATR necessary G2 checkpoint response double- <br />                                                                              strand breaks (Brown Baltimore, 2003). However, these <br />                                                                              experiments used synchronized cells, irradiating cells S/ <br />                                                                              G2-phase assaying progression M-phase. <br />                                                                              Because acute doses g-radiation used create double- <br />                                                                              strand breaks phases cell cycle, possible ATR <br />                                                                              primarily activated specific phase cell cycle (e.g., <br />                                                                              S-phase) response double-strand breaks (for review, see <br />                                                                              Abraham, 2001). In scenario, fraction the <br />                                                                              meristematic cells (most G1) DNA damage would <br />                                                                              invoke ATR-dependent G2 arrest, cells progressing <br />                                                                              S-phase. G1 cells acquire damage be <br />                                                                              activated G2 arrest ATR-independent mechanism, <br />                                                                              progress S-phase, arrest G2-phase. It be <br />                                                                              interesting determine cell-cycle responses acute and <br />                                                                              chronic doses g-radiation atm atr atm double mutant <br /> Figure 7. Semiquantitative RT-PCR Analysis G2-Phase S-Phase&#226;&#8364;&#8220;          plants comparison atr. Nonetheless, preliminary results <br /> Specific Genes.                                                              suggest plants regulate G2-phase checkpoint in <br /> Wild-type atr plants grown described Figure 6. Root tips      response g-radiation differently animals. <br /> (n &gt; 200) excised (1 2 mm root cap) wild-type and <br /> atr plants 4-d transfer RNA extraction, cDNA preparation, and <br /> RT-PCR analysis. Twenty-five microliters RT-PCR sample was <br />                                                                              AtATR Responses Replication Blocks <br /> loaded ethidium bromide&#226;&#8364;&#8220;stained agarose gel, representative <br /> gels shown. All RT-PCR products 400 bp 850 bp           It clear yeast animal studies ATR plays <br /> in size. The RT-PCR product eIF4A (eukaryotic initiation factor)      critical role G2 checkpoint response replication blocks. <br /> employed standard RT-PCR amplification. Numbers each <br />                                                                              Many studies employed HU and/or aphidicolin as <br /> gel band denote relative expression (first normalized eIF4A) the <br />                                                                              model agents replication blocks. HU aphidicolin treatment of <br /> wild-type untreated (n.t.) sample. Standard deviations replicates <br /> are shown parentheses. Aph, aphidicolin.                                  wild-type yeast animals cells (at concentrations allow S- <br />                                                                              phase progression) results G2 arrest. In ATR-deficient mam- <br />                                                                              malian cells, progression mitosis (without G2 delay) leads to <br />                                                                              caspase-independent chromosomal fragmentation, turn <br /> AtATR Plays Relatively Minor Role g-Resistance                          induces apoptotic response (Brown Baltimore, 2000). We <br />                                                                              shown atr plants hypersensitive growth- <br /> At 400 Gy g-radiation, atr mutant similar wild type      inhibitory effects HU aphidicolin. The G2 arrest (as mea- <br /> root growth ability recover g-radiation treatments.          sured PcyclinB1:GUS) wild-type root meristems aphidicolin <br /> By contrast, atm able recover g-radiation doses            concomitant lack G2 arrest atr root meristems indi- <br /> low 80 Gy (Garcia et al., 2003). This suggests AtATR plays           cates ATR required G2-phase checkpoint response <br /> a relatively minor role g-resistance versus AtATM plants.              replication blocks. The lack wild-type G2 arrest response to <br /> Human cell lines expressing dominant-negative kinase mutant                HU, however, surprising. Because fundamental difference <br /> of ATR used characterize relative contributions             HU aphidicolin blocking S-phase replication that <br /> of ATR ATM g-resistance (Cliby et al., 1998; Wright et al.,           HU-treated cells reduced dNTPs, implies G1 <br /> 1998). These experiments clearly demonstrated ATR               arrest activated novel checkpoint responsive low dNTPs. <br /> involved g-resistance, independent studies employing                  Previous studies human cells demonstrated p53-dependent <br /> these approaches yielded inconclusive results relative           G1 arrest response low ribonucleotide levels, not <br /> contributions ATR ATM g-resistance (Jimenez et al.,                deoxynucleotide levels (Linke et al., 1996). The response low <br /> 1992; Cliby et al., 1998; Wright et al., 1998).                              dNTPs help prevent plant cells initiating S-phase DNA <br />  1100      The Plant Cell <br />  <br />  <br />  <br />  <br /> Figure 8. DAPI-Stained Root Tips Wild-Type atr Grown Figure 6. <br />  <br /> (A) Shown root tips day 4 day 6 transfer. <br /> (B) Close-up 6 d transfer aphidicolin wild-type (left) atr (right) DAPI-stained root tips. <br />                                                                                                    AtATR Regulates G2 Checkpoint        1101 <br />  <br />  <br />  <br /> replication absence sufficient dNTP pools. Because               AtATR-Dependent AtATR-Independent Regulation <br /> phosphate relatively immobile limiting <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> soil (Abel     AtRNR1 Transcription <br /> et al., 2002), checkpoint represent plant-specific <br /> response helps direct root growth away phosphate-                The regulation RNR yeast defined involves <br /> depleted regions soil, eliminating growth root meristems          transcriptional posttranslational control (i.e., inhibitory <br /> phosphate-depleted regions, promoting formation               protein&#226;&#8364;&#8220;protein interactions, modification protein localization, <br /> other lateral root meristems root.                     allosteric regulation). For instance, yeast RNR1 (large <br />     The cell death observed atr response aphidicolin,         subunit) gene expressed vegetative growth, upregu- <br /> not HU, suggests G1 (dNTP-dependent) checkpoint               lated late G1 early S, upregulated response <br /> rescue atr cell death. However, atr mutants hypersen-             DNA damage (Elledge et al., 1992). The expression patterns of <br /> sitive effects HU growth root, fact         RNR genes characterized tobacco, <br /> may appear paradoxical given hypothesis line              including RNR1-like gene (Chaboute&#194;&#180; et al., 1998, 2002). The <br /> proficient G1 arrest induced agent. For reason,        reduced level RNR1 transcript (Figure 3) observed atr <br /> we remind reader HU-treated wild-type roots               versus wild type, absence DNA damaging agents or <br /> growing, slightly reduced rate. In words,            chronic UV-B light, suggests RNR1 transcriptionally <br /> cells progressing entire cell cycle,         maintained Arabidopsis AtATR-dependent and <br /> are spending relatively longer fraction cycle G1. We          AtATR-independent mechanisms. Our analysis employing HU <br /> thus propose following model Arabidopsis cell-cycle                seedlings (Figure 3) indicate RNR1 is <br /> checkpoint regulation response HU. HU treatment induces              regulated transcriptionally, independently AtATR, via <br /> two independent checkpoints Arabidopsis: G1                   feedback-type mechanism boosts RNR1 transcripts when <br /> response low dNTPs (AtATR independent) G2                 dNTPs low. <br /> response replication blocks (AtATR dependent). HU-treated                  The rapid reduction AtRNR1 transcript response acute <br /> wild-type atr meristematic cells arrest G1 response           doses UV-B wild-type atr plants suggests additional <br /> low dNTP pools, retarding entry S-phase. The G1            mode transcriptional regulation AtRNR1 response DNA <br /> checkpoint eventually relieved dNTPs accumu-               damage independent AtATR. In yeast, RNR upregulation <br /> late permit entry S-phase, fraction cells leak         response DNA damage rapid, occurring 30 min after <br /> through S-phase. The resulting S-phase&#226;&#8364;&#8220;induced depletion              treatment (Elledge Davis, 1990; Elledge, 1993). It thus <br /> of dNTPs presence HU results requirement           surprising experimental conditions, acute dose <br /> AtATR-dependent G2 checkpoint activation. The inability                 UV-B light caused rapid reduction AtRNR1 transcript <br /> atr activate G2 checkpoint results defective DNA                2 h UV-B treatment, followed return no- <br /> replication meristematic failure.                                      treatment control levels 6 h UV-B treatment. This <br />     By contrast, aphidicolin-treated atr cells invoke G1        response represent process inhibition AtRNR <br /> checkpoint normal levels dNTPs enter              expression leads low dNTP levels concomitant arrest of <br /> S-phase delay. atr cells fail arrest G2             cell cycle DNA damage, possibly G1 <br /> proceed mitosis incompletely replicated genomes,                 checkpoint described above. A decrease RNR expression <br /> perhaps larger degree HU-treated atr cells. This            expected inhibit potentially mutagenic <br /> results majority daughter G1 cells having intolerable       actions bypass polymerases, require high levels of <br /> amount single-strand gaps double-strand breaks, leading             dNTPs activity (Johnson et al., 2001; Chabes et al., 2003). One <br /> to meristematic failure cell death.                                    plausible model mode regulation that <br />     The cell death seen atr unexpected. Mutants DNA              immediately experiencing acute DNA damage, cells initially <br /> repair/maintenance genes Arabidopsis, DNA ligase IV,            try eliminate damage possible error-free <br /> ERCC1, telomerase, exhibit obvious cell death,             pathways. Initial downregulation dNTPs inhibit the <br /> in response treatment DNA damaging agents (Riha et al.,            inherently mutagenic bypass polymerases contribute the <br /> 2001; Friesner Britt, 2003; Hefner et al., 2003). A prevailing         arrest cell cycle. Return normal levels dNTPs then <br /> hypothesis plants, p53 homolog, lack DNA             allows cell cycle progress enables cell utilize <br /> damage&#226;&#8364;&#8220;induced cell death (apoptosis). We believe atr                  bypass polymerases needed complete replication. This <br /> response aphidicolin example programmed cell             model suggestive SOS response Escherichia coli, <br /> death, indicated fact cells root tip die,        error-free repair mechanisms induced early, and <br /> including normally nondividing cells (e.g., epidermal cells), sug-         error-prone translesion synthesis induced later (Goodman and <br /> gesting signal cell death diffusible. Alternatively,   Woodgate, 2000; Rangarajan et al., 2002) response acute <br /> it possible cells surrounding dying meristem           induction DNA damage. <br /> recruited reconstitute missing meristem, induction <br /> of cell cycle induces death. In case, death           AtATR Is Required Completion Meiosis the <br /> simply direct result chromosomal fragmentation               Absence AtATM <br /> induced progression M-phase incompletely <br /> replicated genome. In event, clear replication-             Given partial sterility atm mutant plants nonsterility of <br /> induced checkpoints play important role protecting cells             atr, observation complete sterility atr atm double <br /> from consequences replicational stress.                             mutant suggests ATR partially complement ATM <br />  1102      The Plant Cell <br />  <br />  <br />  <br /> function meiosis. Although clear ATR            using aphidicolin, control HU plates contained equivalent <br /> a distinct meiotic function ATR simply filling lack   (0.05%) DMSO allow direct comparison. <br /> ATM (or vice versa), experiments suggest <br /> that ATR present functional meiosis Arabidopsis.            UV-B g-Irradiation Arabidopsis <br /> In mammalian cells, situ localization studies shown that <br /> both ATM ATR localized meiotic chromosomes                        For chronic doses UV-B light, plants sown MS plates and <br />                                                                              grown 24-h light cycle cool-white lamps, supplemented with <br /> (Keegan et al., 1996; Moens et al., 1999). Further studies using atr <br />                                                                              UV-B light using medium wave UV-B lamp (Spectronics, Westbury, NY; <br /> and atm single double mutant plants allow better <br />                                                                              peak intensity 312 nm) filtered sheets cellulose acetate <br /> understand role ATR plant meiosis, possibly leading             (Golden State Plastics, Sacramento, CA; thickness 0.12 mm) eliminate <br /> additional insights ATR function animal meiosis.                 UV-C (&lt;280 nm) fraction emitted light. The cellulose acetate was <br />                                                                              replaced 3 d. The dose rate UV-B 2.5 J/s/m2. For acute <br /> Conclusions                                                                  doses UV-B, 5-d-old seedlings irradiated UV trans- <br />                                                                              illuminator (Fisher, Houston, TX), peak output 305 nm, filtered through <br /> We shown plants encode ortholog ATR                 single cellulose acetate sheet. UV-B intensities measured with <br /> its role G2 checkpoint regulation similar respects          UV-B radiometer peak response 310 nm 50% response <br /> its yeast mammalian counterparts. The presence ATM                    280 340 nm (UV Products, San Gabriel, CA). For g-radiation <br /> and downstream effectors cell cycle DNA repair,             experiments, Arabidopsis seeds plants irradiated using Cs137 <br /> including BRCA1 (Lafarge Montane, 2003) putative                   source (Institute Toxicology Environmental Health, University of <br /> CHK1 (S.B. Preuss, K. Culligan, A. Britt, unpublished data)              California, Davis). Seeds imbibed water 48C 3 d, irradiated, <br />                                                                              immediately placed MS agar plates germination and <br /> for example, suggest G2 cell-cycle checkpoint <br />                                                                              placed growth chamber; plants grown MS plates (for 5 d), <br /> responses DNA damage replication blocks regulated <br />                                                                              irradiated, immediately returned growth chamber. <br /> in similar fashion plants animals. This say that <br /> plant responses DNA damage replication blocks are <br /> identical animals; significant                   Preparation RNA RNA Gel Blotting <br /> difference plants animals lack p53-                  Total RNA prepared 7-d-old seedlings using RNeasy plant <br /> dependent programmed cell death response persisting DNA                   mini kit (Qiagen, Valencia, CA). RNA blotted positively charged <br /> damage. Plants encode analogous pathway                       membranes (BriteStar-Plus; Ambion, Austin, TX) probed a <br /> degradation hopelessly compromised cells, evidenced                 32P radioactively labeled AtRNR1 (GenBank AF092841) cDNA fragment. <br />  <br /> the cell death observed atr, probably response           This 761-bp fragment generated PCR primers RNR1 (59-AAG- <br /> massive disruption DNA replication coupled uninhibited                 AATGGAGTGAGGAACTCTC-39) RNR3 (59-CTCAAGCAACATTAA- <br /> progression cell cycle. Perhaps plants                GCATTAG-39). The resulting bands visualized Storm 860 <br />                                                                              PhosphorImager quantified using ImageQuant software (Molecular <br /> not die cancer, cell death response minor levels DNA <br />                                                                              Dynamics, Sunnyvale, CA). <br /> damage (such occur spontaneously persist in <br /> repair defective mutants) counteradaptive. Because of <br /> this insensitivity (versus animals) persisting DNA damage,                Semiquantitative RT-PCR <br /> plants ultimately prove robust model the <br />                                                                              Wild-type atr Arabidopsis plants sown MS agar plates, <br /> study fundamental processes cell-cycle regulation                 grown 5 d, transferred MS plates (n.t.), MS plates plus 1 mM <br /> and DNA repair pathways eukaryotes (for review, Hays,                 HU (1HU), MS plates plus 12 mg/mL aphidicolin (1Aph) 4 d of <br /> 2002).                                                                       additional growth. All plants verified respective pheno- <br />                                                                              types described above. Approximately 1 2 mm root tips, &gt;200 <br /> METHODS                                                                      individual plants treatment, excised pooled together <br />                                                                              RNA extraction (RNeasy mini-prep; Qiagen). RNA samples were <br />                                                                              DNase treated (DNase set; Qiagen) quantified. To produce cDNA <br /> Isolation Mutants <br />                                                                              RT-PCR, 2 mg total RNA condition described above <br /> The atr-1 allele (ecotype Ws) isolated Arabidopsis Knockout     reverse transcribed, employing random hexamer primers, with <br /> Facility described (Krysan et al., 1999). The atr-2 allele (ecotype       Superscript first-strand synthesis kit (Stratagene, La Jolla, CA) 40- <br /> Columbia; SALK_032841) identified using Salk SIGnAL Web site         mL reaction according manufacturers protocol. RT-PCR was <br /> (http://signal.salk.edu), seeds obtained ABRC. The atr-3    performed 50 mL reactions using Ex-Taq polymerase (Takara Bio, <br /> allele (ecotype Ws) identified using FLAG database (Genoplante;      Otsu, Shiga, Japan), 2 mL cDNA, following gene-specific <br /> http://genoplante-info.infobiogen.fr/ ).                                     primers: eIF4A-1 (59-CTCTCGCAATCTTCGCTCTTCTCTTT-39), eIF4A-5 <br />                                                                              (59-TCATAGATCTGGTCCTTGAAAC-39), CycB1-2 (59-GTCGCTTTCT- <br />                                                                              TCTTAGTAGCCTTCT-39), CycB1-6 (59-GGCCTCCATTCACTCTCAA- <br /> Growth Arabidopsis <br />                                                                              CAG-39), H4-5 (59-ATGTCTGGTCGTGGAAAGGGAG-39), H4-8 (59-AC- <br /> For standard growth conditions, seeds sown 13 MS (GIBCO,             CAAATTGCGTGTTTCCATTG-39), cATR-1 (59-CAGCGCCCAAAGAAGA- <br /> Cleveland, OH) phytagel (Sigma, St. Louis, MO) agar, pH 5.8, grown       TCATTC-39), cATR-3 (59-GGCCCGCTGAGCATGTGGGTTC-39). All <br /> under cool-white lamps filtered Mylar intensity 100      primers (except Histone H4) spanned intron-exon boundaries to <br /> 150 mmol/m2/s 24-h day light cycle 218C. HU (Sigma; 500 mM         eliminate possibility amplifying contaminating genomic DNA. <br /> stock water) aphidicolin (ICN Biomedicals, Basingstoke, UK;           PCR-cycle parameters primer pairs 20 s 948C, 20 s at <br /> 20 mg/mL stock DMSO) added MS plates (as above) con-           628C, 45 s 728C. Primer pairs tested log-linear <br /> centrations described text figure legends. For experiments        amplification using 18, 21, 24, 27, 30 cycles PCR. The optimized <br />                                                                                                          AtATR Regulates G2 Checkpoint              1103 <br />  <br />  <br />  <br /> number cycles primer pair was: eIF4A, 23 cycles; cycB1, 27           (1999). Spatio-temporal analysis mitotic activity labile cyclin- <br /> cycles; Histone H4, 29 cycles; ATR, 27 cycles. PCR products               GUS fusion protein. Plant J. 20, 503&#226;&#8364;&#8220;508. <br /> ethidium bromide&#226;&#8364;&#8220;stained gels quantified using ImageQuant                 Cortez, D., Guntuku, S., Qin, J., Elledge, S.J. (2001). ATR and <br /> software above.                                                               ATRIP: Partners checkpoint signaling. Science 294, 1713&#226;&#8364;&#8220;1716. <br />                                                                                Crowley, D.J., Courcelle, J. (2002). Answering call: Coping <br /> GUS DAPI Staining                                                            DNA damage inopportune time. J. Biomed. <br />                                                                                  Biotechnol. 2, 66&#226;&#8364;&#8220;74. <br /> GUS staining performed described (Jefferson et al., 1987; Preuss <br />                                                                                Culligan, K.M., Meyer-Gauen, G., Lyons-Weiler, J., Hays, J.B. <br /> and Britt, 2003). DAPI staining performed fixing roots solution <br />                                                                                  (2000). Evolutionary origin, diversification specialization of <br /> of ethanol acetic acid (3:1). The roots washed times in <br />                                                                                  eukaryotic MutS homolog mismatch repair proteins. Nucleic Acids <br /> PBS, pH 7.4, stained PBS containing 2.5 mg/mL DAPI (Molecular <br />                                                                                  Res. 28, 463&#226;&#8364;&#8220;471. <br /> Probes, Eugene, OR) 20 min. The roots washed PBS two <br />                                                                                Klein, A., Muijtjens, M., van Os, R., Verhoeven, Y., Smit, B., Carr, <br /> times mounted slides 50% glycerol. Slides visualized on <br />                                                                                  A., Lehmann, A., Hoeijmakers, J. (2000). Targeted disruption of <br /> a Zeiss Axiophot microscope, photographs taken Zeiss <br />                                                                                  cell-cycle checkpoint gene ATR leads early embryonic lethality <br /> MC100 camera. DAPI staining visualized standard UV <br /> fluorescence filter set.                                                         mice. Curr. Biol. 10, 479&#226;&#8364;&#8220;482. <br />                                                                                Elledge, S., Zhou, Z., Allen, J.B. (1992). Ribonucleotide re- <br />   Sequence data article deposited EMBL/             ductase: Regulation, regulation, regulation. Trends Biochem. Sci. 17, <br /> GenBank data libraries accession numbers AB040133                      119&#226;&#8364;&#8220;123. <br /> AF092841.                                                                      Elledge, S.J. (1993). DNA damage cell cycle regulation of <br />                                                                                  ribonucleotide reductase. Bioessays 15, 333&#226;&#8364;&#8220;339. <br />                                                                                Elledge, S.J., Davis, R.W. (1990). Two genes differentially <br />                                                                                  regulated cell cycle DNA-damaging agents encode <br /> ACKNOWLEDGMENTS                                                                  alternative regulatory subunits ribonucleotide reductase. Genes <br />                                                                                  Dev. 4, 740&#226;&#8364;&#8220;751. <br /> We thank Ying Peng assistance isolating atr atm double mutant <br />                                                                                Friesner, J.D., Britt, A.B. (2003). Ku80- DNA ligase IV-deficient <br /> and Cheryl Whistler, Joanna Friesner, Eli Hefner critical reading of <br />                                                                                  plants sensitive ionizing radiation defective T-DNA <br /> the manuscript. We thank Arabidopsis Knockout Facility <br />                                                                                  integration. Plant J. 34, 427&#226;&#8364;&#8220;440. <br /> (University Wisconsin, Madison) SIGnAL (Salk Institute) and <br />                                                                                Garcia, V., Bruchet, H., Camescasse, D., Fabienne, G., Bouchez, <br /> FLAG databases providing T-DNA knockout resources. This work was <br />                                                                                  D.L., Tissier, A. (2003). AtATM essential meiosis <br /> supported USDA postdoctoral fellowship (2001-01860) K.M.C. <br />                                                                                  somatic response DNA damage plants. Plant Cell 15, <br /> and National Science Foundation Grant (MCB-9983142) A.B.B. <br />                                                                                  119&#226;&#8364;&#8220;132. <br />                                                                                Goodman, M.F., Woodgate, R. (2000). The biochemical basis and <br />                                                                                  vivo regulation SOS-induced mutagenesis promoted by <br /> Received October 30, 2003; accepted February 13, 2004. <br />                                                                                  Escherichia coli DNA polymerase V (UmuD&#226;&#8364;&#8482;2C). Cold Spring Harb. <br />                                                                                  Symp. Quant. Biol. 65, 31&#226;&#8364;&#8220;40. <br />                                                                                Hays, J. (2002). Arabidopsis thaliana, versatile model study <br /> REFERENCES <br />                                                                                  eukaryotic genome-maintenance functions. DNA Repair 1, <br /> Abel, S., Ticconi, C., Delatorre, C. (2002). Phosphate sensing            579&#226;&#8364;&#8220;600. <br />   higher plants. Physiol. Plant 115, 1&#226;&#8364;&#8220;8.                                      Hefner, E.A., Preuss, S.B., Britt, A.B. (2003). Arabidopsis mutants <br /> Abraham, R. (2001). Cell cycle checkpoint signaling ATM              sensitive gamma radiation include homolog human repair <br />   ATR kinases. Genes Dev. 15, 2177&#226;&#8364;&#8220;2196.                                     gene ERCC1. J. Exp. Bot. 54, 669&#226;&#8364;&#8220;680. <br /> Brown, E., Baltimore, D. (2003). Essential dispensable roles        Huang, M., Zhou, Z., Elledge, S. (1998). The DNA replication and <br />   ATR cell cycle arrest genome maintenance. Genes Dev. 17,                damage checkpoint pathways induce transcription inhibition the <br />   615&#226;&#8364;&#8220;628.                                                                       Crt1 repressor. Cell 94, 595&#226;&#8364;&#8220;605. <br /> Brown, E.J., Baltimore, D. (2000). ATR disruption leads                 Jefferson, R., Kavanagh, T.A., Bevan, M.W. (1987). GUS fusions: <br />   chromosome fragmentation early embryonic lethality. Genes                  Beta-glucuronidase sensitive versatile gene fusion marker in <br />   Dev. 15, 397&#226;&#8364;&#8220;402.                                                              higher plants. EMBO J. 6, 3901&#226;&#8364;&#8220;3907. <br /> Chabes, A., Clement, B., Domkin, V., Zhao, X., Rothstein, R.,              Jimenez, G., Yucel, J., Rowley, R., Subramani, S. (1992). The <br />   Thelander, L. (2003). <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span>Survival</span> DNA damage yeast directly                 rad31 gene Schizosaccharomyces pombe involved multiple <br />   depends increased dNTP levels allowed relaxed feedback                   checkpoint functions DNA repair. Proc. Natl. Acad. Sci. USA <br />   inhibition ribonucleotide reductase. Cell 112, 391&#226;&#8364;&#8220;401.                     89, 4952&#226;&#8364;&#8220;4956. <br /> Chaboute&#194;&#180;, M., Combettes, B., Clement, B., Gigot, C., Philipps,            Johnson, R., Haracska, L., Prakash, S., Prakash, L. (2001). Role <br />   G. (1998). Molecular characterization tobacco ribonucleotide                DNA polymerase zeta bypass (6-4) TT photoproduct. <br />   reductase RNR1 RNR2. Plant Mol. Biol. 38, 797&#226;&#8364;&#8220;806.                         Mol. Cell. Biol. 21, 3558&#226;&#8364;&#8220;3563. <br /> Chaboute&#194;&#180;, M.E., Clement, B., Philipps, G. (2002). S phase             Keegan, K.S., et al. (1996). The Atr Atm protein kinases associate <br />   meristem-specific expression tobacco RNR1b gene medi-                different sites meiotically pairing chromosomes. Genes <br />   ated E2F element located 59 leader sequence. J. Biol.             Dev. 10, 2423&#226;&#8364;&#8220;2437. <br />   Chem. 277, 17845&#226;&#8364;&#8220;17851.                                                      Krys <br /> </body></html>