<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal vol.7 no.13 pp.4367-4378, 1988 <br />  <br />  <br /> Pilin expression Neisseria gonorrhoeae                                               both <br /> positive negative transcriptional control <br />  <br />  <br /> Muhamed K.Taha, Magdalene So&apos;,                                        DNA transformation (Seifert et al., 1988). There two <br /> H.Steven Seifert&apos; 3, Elizabeth Billyard2                          loci strain MS 1 (pilE1 pil E2) mapping 20 kb <br />                                                                                                                              - <br />  <br />  <br />  <br /> Christian Marchal                                                     apart (Meyer et al., 1984), strains derivatives <br /> Unite des Antigenes Bacteriens, Institut Pasteur, 75724 Paris Cedex   MS 11 (Segal et al., 1985; Swanson et al., 1986) contain <br /> 15, France &apos;Department Molecular Biology, MB4, Scripps         one. Many regions gonococcal chromosome <br /> Clinic Research Foundation, 10666 N. Torrey Pines Rd, La Jolla,   contain partial pilin sequences; expression sites are <br /> CA 92037, USA                                                         5&apos; pilin coding sequences pilin promoter <br /> 2Present address: Agouron Pharmaceuticals, 505 Coast Boulevard        sequences (Haas Meyer, 1986; Segal et al., 1986). Two <br /> South, La Jolla, CA 92037, USA                                        mechanisms, involving sequence changes pilE <br /> 3Present address: Department Microbiology Immunology, North    locus, shown affect gonococcal piliation status. <br /> Western University, Chicago, IL 60611, USA <br />                                                                       Deletions pilin structural gene pilE rise <br /> Communicated M.Hofnung                                             P- cells (Segal et al., 1985). Some P- variants an <br />                                                                       intact expression site, produce pilin mRNA. Their P- <br /> We identified closely linked genes, pilA                 status results nonsense missense mutations the <br /> pilB, act trans piUn promoter. pilA-pilB              pilin gene pilE (Bergstrom et al., 1986; Haas et al., 1987). <br /> map downstream expression loci pilEl opaEl                  P- variants additional class intact expression <br /> the gonococcal chromosome. Subcloning data indicate                   sites pilE loci encode apparently normal pilins <br /> that pilB acts negatively pilin promoter,                  (Hagblom et al., 1985). <br /> insertional inactivation pilB results hyperpiliated                Although pilin gene pilE -10 sequence, it <br /> gonococci. A pilA clone activates pilin promoter                  lacks consensus -35 sequence (Meyer et al., 1984). The <br /> in Escherichia coli, pilA-/pilA+ heterodiploid                  absence -35 region shown characteristic <br /> gonococcus exhibits P- <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span>phenotype</span>. Our inability                  positively controlled bacterial promoters (Raibaud and <br /> obtain simple pilA- mutants strongly suggests pilA               Schwartz, 1984). This data led the <br /> is essential gene gonococcus. In vitro                proposal P- variants arise trans regulation <br /> coupled transcription/translation system, inserts spanning            (Hagblom et al., 1985; Segal et al., 1985). In this <br /> the pilA pilB region direct synthesis                  manuscript, experiments designed study trans <br /> proteins 40 58 kd. DNA sequence analysis shows                 regulation pilin expression N.gonorrhoeae. <br /> that pilA pilB loci encode proteins 38.6 kd <br /> (with putative DNA binding domain) 57.9 kd <br /> respectively. The pilA pi1B genes opposite                 Results <br /> orientation relative other, 5&apos; ends            Construction reporter plasmids <br /> two genes overlap. We discuss loci                  In order screen gonococcal gene bank Escherichia <br /> interact control pilin expression gonococcus.               coli inserts act trans pilin <br /> Key words: activation/gonococcal pili/repression                   promoter, constructed &apos;reporter&apos; plasmids the <br /> transcription                                                         gene encoding chloramphenicol resistance fused the <br />                                                                       gonococcal pilin promoter. We reported previously on <br />                                                                       cloning pil E2 locus strain MS 11 (Meyer et <br />                                                                       al., 1984; Segal et al., 1985). One clone, pNG1749, <br /> Introduction                                                          multicopied plasmid recombinant derived MS 11 <br /> Pili Neisseria gonorrhoeae important virulence             variant 9B, P- variant pilE2 locus did undergo <br /> in mediate adhesion gonococcus human          obvious sequence rearrangement (Segal et al., 1985). <br /> host (Swanson, 1973; Pearce Buchanan, 1980). They                 pNG 1749 used create fusion pilin <br /> are composed major protein (pilin), undergoes              promoter structural gene chloramphenicol <br /> antigenic variation vitro vivo (Lambden et al., 1979;       acetyltransferase (CAT cartridge) (Close Rodriguez, <br /> Hagblom et al., 1985). In vitro, pilus expression                 1982). In construction, CAT cartridge, also <br /> undergoes phase variation high frequencies. The biological         containing ribosome binding site promoter, was <br /> relevance P- state known,            inserted downstream pilin -10 sequence, and <br /> imagined adherent (P-) cells able            upstream pilin ribosome binding site structural <br /> spread host host, different anatomical sites        gene (Figure 1). This, effect, creates operon which <br /> the host, selective advantage                 CAT gene placed pilin promoter control. That <br /> intracellular location.                                               construction resulted operon confirmed by <br />    An expression locus (pilE) gonococcal chromosome            following evidence: (i) recombinants tested (12/12) <br /> controls pilin expression (Meyer et al., 1984). In vitro,             expected restriction map CAT inserted in <br /> strains gonococci multiple expression loci arise          desired orientation; (ii) Northern blot analysis, pilin- <br />  <br />  CIRI Prpss Limited, Oxford, England                                                                                             4367 <br />  M.K.Taha et al. <br />  <br />                                               pil El                                                           pil E2 <br />                                                 i:: Mstil RBS,,.w                                               -l0 Mstil RBS% <br />                                                                                  i,&lt;C~~~~~~~~-- <br />                                                                                          --     r-                                ; <br />                                                            PILIN <br />                                                                              Y <br />                                                                                      0 <br />                                                                                      CJi~. <br />                                                                          -10 MstIl RBSp <br />                           pNG 1749                      *:~~             ~           ~ ~ 1 <br />                                                                                      _ <br />                                                                                                                  o_ <br />                                                                                              PILIN              --- <br />                           pNG 1721                                                       . <br />  <br />  <br />  <br />                                                   C-,                    .7 CAT}                                 &apos; <br />                                                                         RBSc a <br />                                                                                  p <br />  <br />  <br />                                                                     -      kRBSc               H1    .:) <br />                        pNG         749-COAT <br />                         DNG 1 721 -CAIT                                          CAT                   PILIN <br />  <br />                         pNG    I   749-CATmn-                                                                             .1 Kb <br /> Fig. 1. Schematic representation pilE2 -CAT transcriptional fusion. The insert pNG1749 5-kb EcoRI fragment MSlP1- variant 9B <br /> (Segal et al., 1985) containing pilE2, cloned EcoRI site pBA (Meyer et al., 1982). The insert pNG1721 10-kb BclI fragment <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> MS1 1A, P+, containing pil E2, cloned pBR322 (Meyer et al., 1984). The MstII site pNG1749 pNG1721 located 55 bp <br /> downstream start pilin transcript 16 bp upstream ribosomal binding site (Meyer et al., 1984). The 0.7-kb CAT cartridge <br /> from pCM7 (Close Rodriguez, 1982) blunt end ligated MSTII site of inserts. Black boxes represent pilin structural <br /> gene; white boxes represent regions homology flanking pilEl pilE2; striped boxes represent CAT cartridge. The pilin promoter <br /> region denoted (-10) symbol. RBSp RBSc are, respectively, pilin gene CAT gene ribosome binding sites. pNG1749-CATm <br /> was generated cloning EcoRI partial digest pNG1749-CAT EcoRI site monocopy plasmid pRPZ1 11 (Seifert, 1984). Only <br /> DNA fragments represented boxes drawn scale. <br />  <br />  <br /> gene-specific mRNA increased size CAT                                  Isolation clones negative effect trans on <br /> cartridge transcript (-- 0.6 1.4 kb, data shown). The <br />                                     - <br />                                                                                          piIE2-CAT expression <br /> pilE2 -CAT-operon, pNG 1749-CAT, subsequently                                        Gene products act trans promoters be <br /> recloned pBR322-compatible, kanamycin-resistant                                   inducers repressors. <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> examine possibility that <br /> mini-RI plasmid pRPZ1          (Seifert, 1984). The resultant                            pilin expression regulated repressor, gene <br /> recombinant named pNG1749-CATm (&apos;m&apos;                                              bank initially variant 9B (P-). Chromosomal DNA <br /> monocopy), like vector single copied.                                         9B partially digested MboI ligated into <br />    The pilE2 promoter pNG1749-CAT pNG1749-                                        dephosphorylated BamHI site pBR322. The ligation <br /> CATm derived variant 9B original MS lA                                    mixture transformed E.coli GC6(pNG1749- <br /> strain (Segal et al., 1985). Variant 9B P- phenotype,                              CATm), transformants selected plates containing <br /> and pilE2 locus apparently intact does                             Kan30 Ap5 Of 1500 transformants screened for <br />                                                                                                                       . <br />  <br />  <br />  <br />  <br /> any obvious sequence rearrangements. To ascertain                               sensitivity Cm&apos;o, Kanr, Apr Cm&apos;. Two <br /> pilin promoter 9B fully functional, created                              exhibited sensitivity Cm&quot; more <br /> pilin promoter-CAT fusion using pilE2 pilin promoter                                 pronounced others, contained same <br /> from P+ strain. pNG1721 pilE2 clone derived                                  4-kb insert (pNG 11 pNG56). The restriction maps of <br /> the original MS11 P+ strain (Meyer et al., 1984). An                                     clones identical. Furthermore, restriction <br /> operon fusion pNG1721 CAT                                  analysis strongly suggested insert derived from <br /> cartridge placed downstream pilin -10 sequence                                region downstream pilE1 opaEl, expression <br /> and upstream pilin structural gene (pNG1721-CAT)                                  site variant Protein II (P.11) gene (Meyer et al., 1984) <br /> (Figure 1). As vectors pNG1721-CAT pNG1749-                                  (Figure 2). Southern hybridizations pNG56 probe <br /> CAT derived pBR322, copy number                                   showed hybridized high stringency conditions <br /> be equivalent, CAT produced cells                                   clones containing region (pNG171 1) (Segal et al., <br /> containing recombinant plasmids accurate                                1985; data shown). <br /> reflection pilin-promoter activity. CAT assays indicate                                  The gene products encoded pNG1 1 pNG56 appear <br /> that E. coli GC6(pNG172 1-CAT) GC6(pNG1749-CAT)                                      negative effect trans pilin promoter. This <br /> produced CAT (80 + /- 10 units). This                                 effect opaE1 locus: pNG57, subclone of <br /> result indicates P- phenotype 9B                               pNG56, 5&apos; portion opaEl gene deleted, <br /> sequence changes pilin promoter region pilE2. In                               conferred Cm sensitivity E. coli host (Figure 2). <br /> contrast, E. coli containing monocopied pNG 1749-                                    Thus, appears region 3 kb downstream pilE1 <br /> CATm produced 10 units CAT. These data indicate                                  encodes products affect pilin promoter trans. <br /> pNG1749-CATm used reporter plasmid                                        As pNGI pNG56 derived P- variant, <br /> screening gene bank genes act trans                               examined region P+ <br /> pilin promoter.                                                                          variant affect pilin promoter. pNG1711 a <br /> 4368 <br />                                                                                                                    Trans-regulation gonococcal pilin expression <br />  <br />  <br />              A                 pil El opa El                                pil A        b       pil B                                  pil E2 <br />                                1:.=:-) .. I&gt; <br />                                      5 <br />                                          -0                                                                                        -as <br />                                                                                                                                   AT&gt; <br />               B                               6        cj c-)               -2   Dv <br />                                                                                  ZZ-     -c&gt;-&lt;.;; <br />                                                                                              X <br />                                                                                                                         C <br />                                                                                                                               ...AT assay !&gt;W <br />             pNG 17                                                                                                             5&#194;&#177; 1 <br />                                                                        1t-- <br />             pNG 11- pNG 56                                                                            i <br />                                                                                                                                1 &#194;&#177;1 <br />                                                        I I <br />             pNG 57                                                     &apos;A <br />                                                                                                                                1 &#194;&#177;1I <br />  <br />             pNG 58                                                                                                             6 &#194;&#177;1 <br />  <br />             pNG 59                                                                                                             5 &#194;&#177;1 <br />  <br />             pNG 60                                              1...                             p lc Z                      50 &#194;&#177;5 <br />                                                                                      p <br />  <br />  <br />             pNG 62                                                                                                           54&#194;&#177;5 <br />             pNG 63                                                                                                           25 &#194;&#177; 5 <br />  <br />              pNG 61                  p    Iac Z   -- <br />                                                          -F&quot; <br />                                                           I-                     r                                           11   &#194;&#177;2 <br />  <br />                                                                                                            control           102 <br />                                  I       1 Kb <br />  <br /> Fig. 2. Subcloning pilA-pilB region effects subclones trans pilE2-CAT expression E.coli. (A) Schematic <br /> representation pilA pilB region relative pilEl opaEl loci MS1 lA. ORFs pilin pilEl (Meyer et al., 1984) P.II <br /> in opaEl (Stem et al., 1986) reported. ORFs pilA pilB region determined DNA sequence analysis, included here <br /> for reference. Direction boxes indicate relative orientation reading frames. Black triangles indicate positions mTn3Cm-3 <br /> insertions. (B) Subcloning pilA pilB region: pNGl pNG56 siblings containing 4-kb insert derived MS1 1A variant <br /> 9B (P-), cloned pBR322. pNG17 pNG58 subclones pNG1711 [a pilEl clone derived MSl lA (P+) (Meyer et al., 1984)] in <br /> pUC9 pUC18 respectively. Other inserts derived pNGl pNG56, cloned pUC9 (pNG57), pUC18 (pNG59, pNG60, pNG62 and <br /> pNG63) pUCl9 (pNG61). placZ- indicates orientation lacZ promoter. (C) CAT assays Ecoli harboring pNG1749-CATm <br /> plus plasmids. Controls strain harboring pNG1749-CATm pBR322 pUC18. Cultures extracts were <br /> made according procedure Close Rodriguez (1982), CAT assays described Shaw (1975). The rates increase in <br /> absorption measured room temperature, numbers indicate units CAT sp. act./mg protein extract (1 <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> CAT = 1 nmol of <br /> chloramphenicol acetylated/min). Standard deviations determined results separate experiments. <br /> clone region original MSIlA P+ strain                                           B-galactosidase promoter pUC 18 (see Figure 4A). <br /> (Segal et al., 1985). pNG 17, pUC9 subclone pNG171 1                                        When pNG60 insert recloned pUC19 the <br /> in pilE 5&apos; portion opa El deleted,                                         opposite orientation (pNG6 1), ability clone to <br /> also negative effect pilin promoter (Figure 2).                                     stimulate CAT expression lost. These differences in <br />    To map region pNG 11 pNG56 encodes                                           CAT expression copy number effects, as <br /> the &apos;repressor&apos; activity, subclones insert,                                    apparent ratios monocopy reporter plasmid and <br /> and tested effect trans pilE2-CAT expression                                     subclones constant. These results suggest that <br /> in E. coli. Surprisingly, subclones fell classes:                                   region downstream pilEl controls expression pilin <br /> those retained negative effect CAT expression,                                     trans, clones pNGl 1 pNG56 encode two <br /> and stimulated (Figure 2). Fragments rightwards                                    genes act antagonistically pilin promoter. The <br /> of XAoI site cloned pUC18 exerted negative                                      region leftwards XhoI stimulates pilin promoter <br /> effect pilin promoter, lesser degree                                   named pilA, region rightwards XhoI with <br /> pNGl pNG56. Three subclones leftwards XhoI                                                repressor activity named pilB. <br /> activated CAT expression 2.5- 5-fold. pNG62 contains <br /> an additional 210 bp rightwards XhoI site,                                          Cell-free protein synthesis assays pilA pilB <br /> pNG63 contains additional 584 bp rightwards XhoI.                                          clones <br /> The construction pNG60 subsequently                                         In order identify gene products encoded gonococcal <br /> placed leftwards open reading frame (ORF) region                                     inserts act trans pilin promoter, have <br />  <br />                                                                                                                                                           4369 <br />  M.K.Taha et al. <br />  <br />                                                                                           used subclones previous experiments program <br />                                                                                           Zubay-type cell-free protein synthesis (Figure 3). <br />                                                                                           The pNG17 insert, encoding pilA pilB, directs the <br />                                                                                           synthesis proteins apparent mol. wts 40 and <br />                                                                                           58 kd (Figure 3A, lane 4). The 58-kd protein encoded <br />                                                                                           DNA rightwards XhoI site subclone containing <br />                                                                                           region directs synthesis protein (pNG58, <br />                                                                                           Figure 3A, lane 6), subclones missing this <br />                                                                                           region (pNG57, lane 1; pNG59, lane 2). pNG57 may <br />                                                                                           expected direct synthesis -30-kd truncated <br />                                                                                           derivative 58-kd protein. However, product <br />                                                                                           apparent gel system, would <br />                                                                                           comigrate vector-encoded products. <br />                                                                                              The gene encoding 40-kd protein spans XhoI site, <br />                                                                                           majority coding sequence located leftwards <br />                                                                                           site. The 40-kd protein detected assays of <br />                                                                                           pNG57 pNG62, subclones containing sequences left- <br />                                                                                           wards XhoI plus 1013 210 bp respectively rightwards <br /> Fig. 3. Acrylamide gel electrophoresis autoradiography                             XhoI (Figure 3A, lane 1; B, lanes 1 2). A shorter <br /> [35S]methionine-labeled PilA PilB proteins synthesized vitro.                      hybrid protein observed assays pNG60 (Figure <br /> Reactions programmed 2.5 /g CsCl-purified recombinant                        3A, lanes 3 5), SalI-XhoI subclone pUC18 which <br /> plasmids 50 4d final volume. Each lane contained 10 141 vitro <br /> synthesis mixture. Amersham mol. wt markers gel were:                            pilA activity (Figure 2). Subsequent sequence analysis <br /> bovine serum albumin (69 kd), ovalbumin (46 kd), carbonic anhydrase                       indicates construction created gene fusion bet- <br /> (30 kd), soybean trypsin inhibitor (21.5 kd) lysozyme (14 kd).                        ween gene encoding 40-kd protein (see Figure 4A) <br /> (A) Autoradiograph protein separated 15% acrylamide gel. The                      5&apos; end lacZ gene pUC 18. The 40-kd pro- <br /> reaction mixes programmed with: pNG57 (lane 1); pNG59 (2);                           tein detected assays pNG61, subclone which <br /> pNG60 (3); pNG17 (4); pNG60 (5); pNG58 (6); pUC18 (7); pNG56 <br /> (8). (B) Autoradiograph proteins separated 12% polyacrylamide                     insert pNG60 recloned opposite orientation <br /> gel. The reaction mixes programmed with: pNG57 (lane 1);                             lacZ pUC19 (data shown). This hybrid protein is <br /> pNG62 (2); pUC18 (3).                                                                     presumably responsible activation pilin promoter, <br />  <br />      A <br />  AAACATAAAGGACACCAGCTCGCCCAAAATTTAATCAGCGTCGGTTTGTCTTTTTTCAAATAAACACTGGCGGGGCGGTTGTCCGCGGTTTTTAACGTGGATLAAGTGTGCGGCACGGTC <br />  *                 .             .           .         .             .       -300                       .        -35           .          .-10     .   T <br />  GCGGTCCCGGCATCGACGATTTTGGGCGAACAAGCGCCCAGCGCAAGCAGGCAGCCGAACTTGGCGCAAAGGGAAAAGAAAGTACGGTGTTTCATTTTGATGTTTCCTGTGTGGACGGTT <br />  *                 .             .           .   -200 <br />  TGCATGATTAGACGTTTGAGATGCCGAAACCTTACGGCCCGGATTTTCAGACAACCTTGCCGCGCAAAATACGCTACAATACGCCCTATTTCAAGTTTCTAAAATTAAAAaGGAAAATTCA <br />  *         .      -100                       .         .             .                .                                                           SD        -1 <br /> MetPheSerPhePheArgArgLysLysLysGlnGluThrProAlaLeuGluGluAlaGlnValGlnGluThrAlaAlaLysvalGluserGluValAlaGlnIleValGlyAJnI1.Lys <br /> ATGTTCAGCTTCTTCCGTCGCAAGAAAAAACAGGAAACGCCGGCTLCLCAGGAGGCCCAAGTTCAGGAAACCGCAGCAAAAGTAGAATCTGAAGTTGCTCAAATAGTTGGAAATATTAAA <br />  1             .             .                             XhoI                   .                 .            .                  100 <br />  GluAapValGluSerLruAlaGluSerValLysGlyArgAlaGluSerAlaValGluThrValSerGlyAlaValGluGlnValLysGluThrValAlaGluMetProSerGluAlaGly <br /> GAAGATGTCGAATCTTTAGCAGAAAGCGTCAAAGGGCGGGCCGAATCTGCCGTTGAAACCGTCAGCGGTGCGGTTGAACAGGTAAAGGAAACCGTTGCCGAGATGCCGTCTGAAGCAGGG <br />                *       *             *                           .       *        .                 .       200            . <br /> GluAlaAlaGluArgValGluSerAlaLysGluAlaValAlaGluThrValGlyGluAlaValGlyGlnValGlnGluAlaValAlaThrThrGluGluHisLysLeuGlyTrpAlaAla <br /> GAAGCGGCGGAACGCGTTGAATCCGCAAAAGAAGCTGTTGCCGAAACCGTCGGCGAGGCTGTCGGGCAAGTTCAAGAAGCCGTTGCGACAACTGAAGAACACAAGCTCGGTTGGGCTGCG <br />          *          *         .         .         .       300         .         . <br /> ArgLeuLysGlnGlyLeuAlaLysSerArgAspLysMetAlaLysSerLeuAlaGlyValPheGlyGlyGlyGlnIleGlyGluAspLeuTyrGluGluLeuGluThrValLeuIleThr <br /> CGTTTGAAACAAGGCTTGGCCAAATCACGCGACAAAATGGCGAAATCGCTGGCCGGCGTGTTCGGCGGCGGACAAATCGGCGAGGATTTGTACGAAGAGCTGGAAACCGTGCTGATTACC <br />                            BalI                  400             .                .                 .            . <br /> GlyAspMetGlyMetGluAlaThrGluTyrLeuMetLysAspValArgGlyArgValSerLeuLysGlyLeuLysAspGlyAsnGluLeuArgGlyAlaLeuLysGluAlaLeuTyrAsp <br /> GGCGATATGGGCATGGAGGCCACCGAATACCTGATGAAAGACGTGCGCGGCCGCGTCAGCCTCAAAGGGCTGAAAGACGGCAACGAATTGCGCGGCGCGTTGAAAGAAGCCTTGTACGAC <br />          *                 500                              NotI                  .                 .            .         .                               600 <br /> LeuIleLysProLeuGluLysProLeuValLeuProGluThrLysGluProPheValIleMetLeuAlaGlyIleAsnGlyAlaGlyLysThrThrSerIleGlyLysLeuAlaLysTyr <br /> CTGATTAAGCCGTTGGAAAAACCGCTGGTCTTGCCCGAAACTAAAGAGCCTTTCGTGATTATGCTTGCCGGTATCAACGGCGCGGGCAAAACCACGTCTATCGGCAAACTCGCCAAATAT <br />                    * <br />                    *                                *       *         .          .        .       700 <br /> PheGlnAlaGlnGlyLysSerValLeuLeuAlaAlaGlyAspThrPheArgAlaAlaAlaArgGluGlnLeuGlnAlaTrpGlyGlyArgAsnAsnValThrValIleSerGlnThrThr <br /> TTCCAAGCGCAGGGCAAATCCGTATTGCTGGCGGCGGGCGATACCTTCCGCGCCGCCGCCCGTGAGCAGCTTCAGGCTTGGGGCGGGCGCAACAATGTAACCGTCATTTCACAAACCACG <br />          *          *         *         *         *         .         .       800         . <br /> GlyAspSerAlaAlaValCysPheAspAlaValGlnAlaAlaLysArgAlaAspArgHisArgAlaCysArgHisArgArgProProAlaHisAlaAlaSerPheAspGlyArgAsnGln <br /> GGCGATTCCGCCGCCGTGTGCTTCGATGCCGTCCAAGCCGCCAAGCGCGCGGATGACATCGTGCTTGCCGACACCGCCGGCCGCCTGCCCACGCAGCTTCATTTGATGGAAGAAATCAA <br />          *          .         .          .         . MboI 900         .         . <br /> LysSerGluAlaArgAlaAlaLysSerHisSerArgArgAlaAlaArgAsnTyrArgArgThrArgCysGlnTyrArgAlaLysArgArgGlnProSerGlnSerLeu*** <br /> AAAAGTGAAGCGCGTGCTGCAAAAAGCCATTCCCGGCGCGCCGCACGAAATTATCGTCGTACTCGATGCCAATATCGGGCAAAACGCCGTCAACCAAGTCAAAGCCTTTGACGACGCATT <br />                                      1000 <br /> GGGGCTGACGGGGCTTATCGTTACCAAACTCGACGGCACGGCAAAAGGCGGCATCCTCGCCGCGCTTGCTTCCGACCGCCCCGTCCCCGTCCGCTACATCGGCGTGGGCGAAGGCATAGA <br />          *      1100          .         .         .                           .                 .                                                       1200 <br /> CGACCTGCGCCCGTTTGACGCGCGCGCGTTTGTGGACCGACTGCTGGATTGAGCCGAAATGCCGTCCGAAAACGGCAGACCGAACCGTCkZf=G <br />            *      *                      *                   *                *                              .         . <br />                                                                                                                                           GTGGGA CTAGGACGCGG <br />                                                                                                *                                   1300 <br /> GGTTTGGGCAACCGTTTTATCCGATAAGTTTCCGTGCGGACAGGTCCGGk <br />           . <br />                    .CTG*1                          C                   *GGCGGGTTTTAGGGTTACGGTGTATCGGCAATGACGGTTCGGGTATTTTAC <br />                      ..                  .         .          .         .      ~~1400... <br />                                                                               ~~~~    ~    ~     ~ ~       ~ ~ <br /> TGCGCCCGCCCCGCGCCTGTAAACGGCGGGCGCATCAAAAASTGCCLGTCTGAGGTTCAGACGGC-A CGGTATCGGGGAATCAGAAGCGGTAGCGCACGCCCAATGAGGCTTCGTGGGTTT <br />            ..                 .          .         .      ~~1500....- <br />                                                          ~~~~     ~    ~ <br /> 4370 <br />                                                                                                                   Trans-regulation gonococcal pilin expression <br />  <br />      B <br />  ACTTCAGATTCTACTTTTGCTGCGGTTTCCTGAACTTGGGCCTCCTC,GAQAGCCGGCGTTTCCTGTTTTTTCTTGCGACGGAAGAAGCTGAACATDGAATTTTCCTTTTAATTTTAGAAA <br />  *         .                          -200    XhoI.          ..            -35                    -10. <br />  CTTGAAATAGGGCGTATTGTAGCG7hTTTTGCGCGGCAAGGTTGTCTGAAAATCCGGGCCGTAAGGTTTCGGCATCTCAAACGTCTAATCATGCAAACCGTCCACACASCAA.ACATCAAA <br />  . -35            -100    -10 .          .          .         .         .         .                           SD.       -1 <br />  MetLysHisArgThrPhePheSerLeuCysAlaLysPheGlyCysLeuLeuAlaLeuGlyAlaCysSerProLysIleValAspAlaGlyThrAlaThrValProHisThrLeuSerThr <br />  ATGAAACACCGTACTTTCTTTTCCCTTTGCGCCAAGTTCGGCTGCCTGCTTGCGCTGGGCGCTTGTTCGCCCAAAATCGTCGATGCCGGGACCGCGACCGTGCCGCACACTTTATCCACG <br />  1                 ..                         .              .           .              .                 .              .               .    100 <br />  LeuLysThrAlaAspAsnArgProAlaSerValTyrLeuLysLysAspLysProThrLeuIleLysPheTrpAlaSerTrpCysProLeuCysLeuSerGluLeuGlyGlnAlaGluLys <br />  TTAAAAACCGCGGACAACCGCCCCGCCAGTGTTTATTTGAAAAAAGACAAACCGACGCTGATTAAATTTTGGGCGAGCTGGTGTCCTTTATGTTTGTCCGAATTGGGACAGGCCGAGAAA <br />                    *              .           .              .           .              .                 .    200                   . <br />  TrpAlaGlnAspAlaLysPheSerSerAlaAsnLeuIleThrValAlaSerProGlyPheLeuHisGluLysLysAspGlyGluPheGlnLysTrpTyrAlaGlyLeuAsnTyrProLys <br />  TGGGCGCAAGATGCAAAATTCAGCTCCGCCAACCTGATTACCGTCGCCTCCCCCGGCTTTTTGCACGAGAAAAAAGACGGCGAGTTTCAAAAATGGTATGCCGGTTTGAACTACCCCAAG <br />                    *          *           *                  *           .    300                         .          .               . <br />  LeuProValValThrAspAsnGlyGlyThrIleAlaGlnAsnLeuAsnIleSerValTyrProSerTrpAlaLeuIleGlyLysAspGlyAspValGlnArgIleValLysGlySerIle <br />  CTGCCCGTCGTTACCGACAACGGCGGCAZG&amp;Tf..CCCAAAACCTGAATATCAGCGTTTATCCTTCTTGGGCGTTAATCGGTAAAGACGGCGACGTGCAGCGCATCGTCAAAGGCAGCATC <br />                *              .           PvuI     400               .              .                  .             .               . <br />  AsnGluAlaGlnAlaLeuAlaLeuIleArgAsnProAsnAlaAspLeuGlySerLeuLysHisSerPheTyrLysProAspThrGlnLysLysAspSerAlaIleMetAsnThrArgThr <br />  AACGAAGCGCAGGCATTGGCGTTAATCCGCAACCCGAATGCCGATTTGGGCAGTTTGAAACATTCGTTCTACAAACCCGACACTCAGAAAAAGGATTCAGCAATCATGAACACGCGCACC <br />                *        500               *              .           .              .                  .             .               .              .       .    600 <br />  IleTyrLeuAlaAlaAlaAlaSerGlyAlaTrpLysProIleSerAsnAlaSerThrAlaTrpLeuThrArgTyrArgTyrAlaAsnGlyAsnThrGluAsnProSerTyrGluAspVa1 <br />  ATCTACCTCGCGGCGGCTGCTTCTGGGGCTTGGAAGCCTATTTCCAACGCATCGACGGCGTGGTTGACGCGGTATCGCTACGCCAACGGCAACACGGAAAACCCGAGCTACGAAGACGTG <br />                        * <br />                        . <br />                        .. <br />                         .                                                           .                 .              .           .            700 <br />  SerTyrArgHisThrGlyHisAlaGluThrValLysvalThrTyrAspAlaAspLysLeuSerLeuAspAspIleLeuGlnTyrTyrPheArgValValAspProThrSerLeuAsnLys <br />  TCCTACCGCCATACGGGCCATGCCGAGACCGTCAAAGTGACCTACGATGCCGACAAACTCAGCCTGGACGACATCCTGCAATATTATTTCCGCGTCGTTGATCGACCAGCCTCAACAAA <br />                *              .           .              .           .              .                 .        800               .              MboI <br />  GlnGlyAsnAspThrGlyThrGlnTyrArgSerGlyValTyrTyrThrAspProAlaGluLysAlaValIleAlaAlaAlaLeuLysArgGluGlnGlnLysTyrGlnLeuProLeuVal <br />  CAGGGTAACGACACCGGCACGCAATACCGCAGCGGCGTGTACTACACCGACCCCGCCGAAAAAGCCGTCATCGCCGCCGCCCTCAAACGCGAGCAGCAAAAATACCAACTGCCCCTCGTT <br />                *              *           *              *           .        900 <br /> ValGluAsnGluProLeuLysAsnPheTyrAspAlaGluGluTyrHisGlnAspTyrLeuIleLysAsnProAsnGlyTyrCysHisIleAspIleArgLysAlaAspGluProLeuPro <br /> GTTGAAAACGAGCCGCTGAAAAACTTCTACGACGCCGAGGAATACCATCAGGACTACCTGATTAAAAACCCCAACGGCTACTGCCACATCGACATCCGCAAAGCCGACGAACCGCTGCCG <br />                                                   1000 <br /> GlyLysThrLysAlaAlaProGlnGlyGlnArgLeuArgArgGlyGlnArgIleLysAsnArgValThrProAsnSerAsnAlaProAspArgArgAlaIleProSerAspGlnAsnSer <br /> GGCAAAACCAAAGCCGCACCGCAAGGCCAAAGGCTTCGACGCGGCCAACGTATAAAAAACCGAGTGACGCCGAACTCAAACGCACCTGACCGAAGAGCAATACCAAGTGACCAAAACAGC <br />            *           1100           .              .           .              .                  .             .               .              .       .       1200 <br /> AlaThrGluTyrAlaPheSerHisGluTyrAspHisLeuPheLysProGlyIleTyrValAspValValSerGlyGluProLeuPheSerSerAlaAspLysTyrAspSerGlyCysGly <br /> GCGACCGAATACGCCTTCAGCCACGAATACGACCATTTGTTCAAACCCGGCATTTATGTGGACGTTGTCAGCGGCGAACCCCTGTTCAGCTCCGCCGACAAATATGATTCCGGCTGCGGC <br />            *              .           .              .           .              .                  .             .           .               1300 <br /> TrpProSerPheThrArgProIleAspAlaLysSerValThrGluHisAspAspPheSerPheAsnMetArgArgThrGluValArgSerArgAlaAlaAspSerHisLeuGlyHisVa1 <br /> TGGCCGAGCTTCACGCGCCCGATTGATGCAAAATCCGTTACCGAACACGATGATTTCAGCTTCAATATGCGCCGCACCGAAGTCAGAAGCCGCGCCGCCGATTCGCACTTGGGACACGTC <br />            *              .           .              .           .              .                 .           1400           . <br /> PheProAspGlyProArgAspLysGlyGlyLeuArgTyrCysIleAsnGlyAlaSerLeuLysPheIleProLeuGluGlnMetAspAlaAlaGlyTyrGlyAlaLeuLysGlyGluVal <br /> TTCCCCGACGGCCCCCGCGACAAAGGCGGACTGCGCTACTGCATCAACGGCGCGAGCTTGAAATTCATCCCGCTGGAACAAATGGACGCGGCAGGCTACGGCGCGTTGAAGGGCGAAGTG <br />            *              .           .              .           .           1500                 .              .           . <br /> Lys*** <br /> AAATAAGCCGCACCGCCGCCTAACCCGGCNAAATGCCGTCTGAAACCTGCAACGTTTCAGACGGCATTT;TTATCCGGCGGGGATTTGTTCAGACAGCATCGCCGCCGTTTTCAACCAGC <br />            *              .           .           1600           .              .                 .              .           . <br /> CCGGCCAACCGTTCCAACGCGAAGGCGACCGCCTGCGCGCGGACGGATTCGCGGTTGCCGTCAAAACGGCGCATTGCTTCGCAACTTCCGCCCGGAAAGGCAAACCCGAACCAAACCGTG <br />            *           1700           .              .           .              .                 .              .           . <br /> CCGACGGGTTTGCTTTCGCCGCCGCCGCCCGGG <br />            *              .           SnmaI <br />  <br />  <br /> Fig. 4. DNA sequence pilA pilB. The deduced amino acid sequences pilA pilB appear coding sequence starting position <br /> + 1. (-35) (-10) indicate hypothetical RNA polymerase binding sites, indicated bold letters. The arrow position -248 (A) <br /> indicates start pilA mRNA synthesis. &apos;SD&apos; denotes Shine-Dalgarno ribosome binding sequence. The amino acid residues bold letters in <br /> (A) constitute possible DNA binding site PilA protein. Underlined bases correspond key restriction sites. Divergent arrows indicate <br /> possible pairing sequences. TCA underlined position 1520 (A) represents reverse strand sequence opaEl stop codon (Stem et al., <br /> 1986). (A) DNA deduced amino acid sequence pilA pilA-opa El intergenic region. (B) DNA deduced amino acid sequence pilB. <br />  <br />  <br /> as pNG61 activate pilin promoter (Figure 2). Two                                 observation indicates that, MS1 1 variants, <br /> minor polypeptides expressed pilA pilB, smaller                                    modulation pilin expression pilA -pilB does not <br /> than 40- 58-kd products, likely represent pro-                                 occur sequence changes loci. <br /> ducts incomplete synthesis commonly observed                                     The pNG17 insert contains ORFs opposite <br /> vitro system.                                                                               orientation. One frame 1068 bp begins ATG 45 <br />                                                                                             bases rightwards XhoI site, continues leftwards <br /> Sequence analysis pNG57 pNG 17                                                       XhoI site SalI site (Figures 2A and <br /> pNG57 pNG17 subcloned M13 mplO                                             4A). The predicted mol. wt protein 38 641 daltons, <br /> mpll, series overlapping deletions generated                                       corresponding 40-kd protein observed Zubay assays <br /> in inserts technique Dale (1985). Both                                      subclones region. This ORF corresponds the <br /> strands resultant inserts sequenced Sanger                               region defined subcloning subsequent transposon <br /> chain termination technique (Sanger, 1977). The insert                                   mutagenesis studies pilA gene. Thus, 38.6-kd <br /> pNG57 derived P- variant MS1 1,                                      protein pilA gene product (PilA). Protection studies <br /> pNG17 insert P+ derivative. The sequences                                      identify transcriptional start site pilA. <br /> both inserts identical pilA-pilB region. This                                    We used 587 bases 5&apos; XhoI-3&apos; PvuI pilA-pilB <br />                                                                                                                                                                 4371 <br />  M.K.Taha et al. <br />  <br /> fragment single-stranded radiolabelled probe (see             N. gonorrhoeae replacing j3-lactamase gene an <br /> Materials methods Figure 2). This experiment placed        E. coli Cmr gene (Seifert et al., 1986a). A minitransposon <br /> the start pilA transcription guanosine position   <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>mutation</span> Cm promoter generated by <br /> -248, bases upstream putative promoter. Thus         chemical mutagenesis. This minitransposon, mTn3Cm-3 <br /> pilA coding sequence preceded putative promoter          (1.6 kb), transferred N.gonorrhoeae strain MS1 1, <br /> ribosome binding sequences:                                        allowed bacterium grow 10 Rg/m1 Cm (H.S.Seifert, <br />                                                                    M.Vito, R.S.Ajioka M.So, preparation). <br /> TTGTCC-(N)-17-TAAAGT-(N)-242-AGGAA-(N)6-ATG                           pNG17, contains pilA, pilB 3&apos; half of <br />                                                                    opa E1, recloned SalI-SmnaI fragment pHSS6, <br /> A stretch 20 amino acids PilA (positions 29-48)          minitransposon insertions isolated. After restriction <br /> strong sequence homology consensus DNA binding            mapping, plasmids transposon insertions were <br /> domain described Pabo Sauer (1984) (Figure 4A):             selected studies; pilA::mTn3Cm-3a; pilA::- <br /> within 20-amino-acid region, 5/6 residues match           mTn3Cm-3b; pilB::mTn3Cm-3; opaEl::mTn3Cm-3. In <br /> consensus sequence: Ala, Gly Val positions 5, 9         plasmids, minitransposon in- <br /> 15 respectively, hydrophobic residues positions 4       serted 550 100, 250 600 bp downstream the <br />                                                                            - <br />  <br />  <br />  <br />  <br /> 8. Furthermore, secondary structure prediction analysis            pilA, pilB opaEl start codons respectively (Figure 2A). <br /> (Garnier et al., 1978) region suggests domain    Each plasmids linearized digestion with <br /> is organized alpha helix-turn-alpha helix motif.            SmaaI, total digested plasmid DNA used transform <br /> These observations strongly suggest 38.6-kd pilA          piliated cells MS llA (pilE1 +, pilE2+), piliated <br /> gene product interacts DNA.                                   derivative 4. 1C (pilE 1 +, ApilE2) (Hagblom et al., 1985; <br />    The second reading frame opposite orientation         Segal et al., 1985). <br /> with respect pilA reading frame (Figure 4B). It begins <br /> with ATG 191 <span id='am-163' about='xsp:base' typeof='owl:Thing'>base</span> rightwards ofXoI contains 1563            Generation pilB- mutants <br /> bases. The ORF preceded putative ribosome                The opaEl::mTn3Cm-3 transformants appeared the <br /> binding site possible promoters:                           plates 24 h, frequency 2 x 10-5/c.f.u. When <br />                                                                    transformed pilB: :mTn3Cm-3 DNA, Cm-resistant <br /> a: TTGCGA-(N) 17-TGAATT-(N) 126-AGGAA-(N)8-ATG                     colonies MSl lA 4. 1C appeared 24 h, at <br /> b: TTGAAA-(N)17-TATTTT-(N)77-AGGAA-(N)8-ATG                        frequencies 7 x 10-5 1 x 10-4/c.f.u. pilB- and <br />                                                                    opaE1- mutants grew equally Cm&apos;I, colonies <br />   This reading frame encodes protein 57 880 daltons,          types mutants P+ phenotype. Interestingly, <br /> corresponding 58-kd protein synthesized vitro         dark rings circumscribing colonies pilB- mutants <br /> subclones region DNA. We shown           pronounced surrounding colonies <br /> region, named pilB, acts negatively pilin promoter.         wild-type cells colonies opaEl - cells. The edges <br /> Thus, second reading frame encodes pilB gene.             colonies appeared sharper. The pilB- <br /> Downstream (23 bp) TAA stop codon pilB              piliation phenotype appeared stable Cm <br /> gene sequences indicative transcriptional stop signal     selection. <br /> (Platt, 1986): exact inverted repeats 17 bp flanking 7 bp          Five pilB- MSllA transformants, pilB- 4.1C <br /> of non-homology, followed series thymines.                 transformants opaEl- MSl1 4.1C trans- <br />   The pilA pilB genes divergently transcribed          formants selected Southern blot analysis (Figure <br /> the 5&apos; promoter regions overlap. Moreover, pilA tran-          SB). Chromosomal DNA transformants was <br /> script expected overlap transcript pilB,        digested Cla I, filters hybridized pilB <br /> whether starts promoter &apos;a&apos; &apos;b&apos; (Figure 4). The         probe (a 0.4-kb PvuI-MboI pilB internal fragment, see <br /> deduced amino acid sequences PilA PilB com-            Figure 2A). A 5.8-kb ClaI fragment MSl lA and <br /> pared protein sequences NBRF bank (release         4.1C contains pilA, pilB opaE1 (Figure 5A-C, <br /> 15.0) using FASTP program (Lipman Pearson,                 lanes b). This fragment pilB- opaEl- <br /> 1985), significant homology found.                      transformants increased size 1.6 kb (Figure SB, <br />                                                                                                            - <br />  <br />  <br />  <br />                                                                    lanes 1-12), expected simple insertion <br /> Transposon inactivation pilA pilB                           mTn3Cm-3 element homologous <br /> The effects pilA pilB pilin promoter          chromosomal DNA. The CAT gene probe hybridized the <br /> observed E. coli. In order assess role locus        7.4-kb ClaI fragment transformants, not <br /> plays regulating pilin expression, pilA-            wild-type MSllA 4.1C DNA (Figure 5B&apos;). <br /> pilB- mutants N.gonorrhoeae strain MS 1. A method <br /> of direct transposon mutagenesis does exist N.             Generation pilA - mutants <br /> gonorrhoeae. We used shuttle mutagenesis (Seifert        Major differences observed pilA- mutants were <br /> et al., 1986a,b) obtain pilA- pilB- mutants. Cloned         generated.pilA::mTn3Cm-3a contains minitransposon at <br /> inserts transposon mutated E. coli mutated          position 550 bp pilA. The pilA: :mTn3Cm-3a transform- <br /> inserts introduced N. gonorrhoeae DNA transform-           ants MSl lA 4. 1C (pilA-a) appeared 48 h <br /> ation. This takes advantage fact piliated       (cf. 24 h pilB- opaEl -). pilA::mTn3Cm-3b <br /> cells N.gonorrhoeae transformed homologous               contains minitransposon position 100 bp pilA. The <br /> DNA high frequency, linearized transforming            pilA::mTn3m-3b transformants MS1 lA 4.1C <br /> DNA recombine replace homologue. The             (pilA-h) appeared 24 h. In cases (for pilA-a and <br /> defective minitransposon mTn3 adapted use               pilA-b transformants) transformation frequency was <br />  <br /> 4372 <br />                                                                                                                     Trans-regulation gonococcal pilin expression <br />  <br />  <br />                                                                                                                                     4   9.4 <br />                                                                                     _       _ <br />  <br />  <br />  <br />                                                                                                                                     46.6 <br />                            A         e           _.                                 e <br />                                                                                                                                     44.4 <br />  <br />                                        b       1        2   4     5       6       7       8       9       10 11        12 <br />                                                                                                                                     4 9.4 <br />  <br />                                                  o            a.                                                              - <br />                                                                                                                                     4 6.6 <br />                            A&apos; <br />                                                                                                                                     44.4 <br />  <br />  <br />  <br />                                                                                                                                     49.4 <br />  <br />                                                                   e@                    -       e <br />                                     B        _        _ <br />                                                                                                                                     46.6 <br />  <br />  <br />  <br />                                                                                                                                     44.4 <br />                                                     b       2 3       4       5       6       7       8    9 10             /2` <br />                                                                                                                                     49.4 <br />  <br />                                                                                                               fo   &quot; <br />                                     B&apos;                                                                                              46.6 <br />  <br />  <br />  <br />                                                                                                                                     * 4.4 <br />  <br />                                                                                                                                     49.4 <br />  <br />  <br />                                                                                                                                     46.6 <br />                                                                                         C                                     e_- <br />  <br />                                                                                                                                     44.4 <br />  <br />  <br />                                                                                                             b      1 <br />  <br /> Fig. 5. Southern blot analysis pilA- pilB- mutants. Cla I-digested chromosomal DNA mutant hybridized nick-translated <br /> intragenic segments pilA (A C), pilB (B), nick-translated pCM7 containing CAT gene (A&apos; B&apos;). The pilA pilB probes <br /> hybridize 5.8-kb Clal fragment strains MS1lA P+ variant 4. IC (a b respectively photograph). mTn3Cm-3 <br /> (1.6 kb) (Seifert et al., 1986b) does contain ClaI site. (A A&apos;) pilA::mTn3Cm-3a-transformed MS1 derivatives (lanes 1-3) 4. 1C <br /> derivatives (lanes 4-13). Two derivatives grown nonselective medium DNA extraction (lanes 6 7), cultures presumably <br /> contained mixture pilA-lpilA+ heterodiploids pilA+ revertants. DNA lane 2 (A&apos;) lost restriction digestion. (B B&apos;) <br /> pilB::mTn3Cm-3-transformed MSl lA derivatives (lanes 2-6) 4. 1C derivatives (lanes 9-12) opaEl ::mTn3Cm-3-transformed MS1 lA <br /> derivative (lane 1) 4. 1C derivatives (lanes 7 8). (C) pilA::mTn3Cm-3a-transformed 4. 1C derivative shown A5, lane 1, P+ <br /> revertants grown nonselective medium (lanes 2 3). <br />  <br /> low (- 7 x 10-7). The pilA-a transformants (distal                                  pilA-a colonies convex P- <br /> insertion pilA) grew slowly pilB-,                                 colonies. pilA-b colonies exhibited usual P+ phenotype. <br /> opaEl- pilA-b transformants                                     Chromosomal DNA MS1lA pilA-a trans- <br /> passed selective nonselective media. The pilA-b                               formants, 4. IC pilA-a transformants (Figure <br /> transformants (proximal insertion pilA) grew                             SA), MS 1A pilA-b sixteen 4. 1C pilA-b <br /> as pilB- opaEl- transformants. pilA-a colonies                                  transformants (data shown) digested Cla I, and <br /> exhibited morphology similar P- colonies, i.e.                         analyzed Southern blotting using pilA probe (a 0.85-kb <br /> no dark rings surround edges colonies. However,                          MboI -XhoI pilA internal fragment cloned pUC 18) <br />  <br />  <br />                                                                                                                                                            4373 <br />  M.K.Taha et al. <br />  <br /> (Figure 2A). Surprisingly, 13 pilA-a 18                A                         C <br /> pilA-b transformants, pilA probe hybridized two <br /> ClaI fragments, 7.4 kb 5.8 kb (Figure           23SPI <br /> 5A, lanes 1-13). Only 7.4-kb fragment hybridized                                                              4 23S <br /> the CAT probe, indicating 5.8-kb fragment contains       16S &apos; <br /> the wild-type pilA gene (Figure 5A&apos;). Both 5.8-                          .a.                                   *416S <br /> 7.4-kb fragments hybridized pilB probe (data not <br /> shown). Further mapping 31 transformants indicated <br /> that pilA-pilB duplication containing minitransposon <br /> had occurred chromosomal locations outside <br /> of pilE1 (data shown). <br />    Because unexpected Southern blotting data,                    1 2 3                   1 2 3 b <br /> repeated transformation experiment. DNAs                B <br /> of pilA-alpilA+ transformants used                                            D <br /> retransform MS 1 IA 4. IC. As incoming gonococcal <br /> DNA did overcome restriction barrier,                       *R&apos;                                   4pilin mRNA <br /> transformation frequency higher (10-5/c.f.u.), and <br /> Cmr colonies appeared 24 h incubation. The trans- <br /> formants exhibited P- like phenotype (convex colonies <br /> without dark edges), growth transformants upon <br /> repassage slow. Southern blot analysis trans- <br /> formants experiments using pilA probe         Fig. 6. Level pilin transcription pilA- pilB- mutants. Total <br /> gave <span id='am-164' about='xsp:pattern' typeof='owl:Thing'>pattern</span> seen Figure SA.                       gonococcal RNA separated formaldehyde agarose gels and <br />    These data strongly suggest pilA gene, addition    transferred N-hybond filter. The filters hybridized nick- <br /> to regulating pilin gene expression, essential function    translated pNGl 100 probe (B D) contains pilin gene of <br />                                                                   MS1 1 A (Meyer et al., 1984). After autoradiography, filters were <br /> in gonococcus, transposon insertion pilA            washed extensively remove probe rehybridized with <br /> appears lethal. The presence pil A hybridizing       nick-translated pBT18-88 (A C) containing Bacillus <br /> ClaI fragments indicates transformants               thuringiensis 23S 16S ribosomal genes (Klier et al., 1979). The <br /> heterodiploids contain copy pilA+-pilB+              ratio pilin 16S ribosomal signal variant determined <br /> and pilA--pilB+. It curious ClaI               densitometer scanning. 16S rRNA transfer observed be <br />                                                                   efficient 23S separate experiments. Hybridizations with <br /> fragment containing pilA --pilB+ 7.4 kb,        total RNA mutants pilus phase variants shown: <br /> size wild-type locus containing minitransposon           4.1C pil A-alpil A+ (lane a), 4.1C pil B- (lane b), 4.1C (lane 2), <br /> insertion, transforming DNA contains          4. 1B (lane 1) 2B (lane 3). 2B P- variant pilin <br />                                                                   gene deleted expression sites 4. 1B P- variant in <br /> Cla I site (see Figure 2). There possible             pilEl intact reduced pilin gene transcription <br /> explanations blotting result. One incoming   (Segal et al., 1985; Hagblom et al., 1985; M.K.Taha, M.So and <br /> DNA containing minitransposon simply recombined           C.Marchal preparation). <br /> with loci shares partial homology, <br /> fortuitously generating new 7.4-kb ClaI fragment        media. However, pil A -alpil A+ transformants were <br /> case. Such coincidence extremely unlikely. A        passaged plates Cm, gave rise high <br /> likely explanation incoming DNA recombined        frequency (10-50%) P + colonies normal size. These <br /> initially wild-type pilA -pilB locus, generating       P+ colonies Cms, parental P colonies are <br /> non-viable cell undergoes autolysis. DNA          Cmr. Southern blotting analysis DNA these <br /> region chromosome, containing pilA--pilB+              revertants using pilA probe shows 7.4-kb ClaI <br /> region flanking ClaI sites, transform          fragment disappeared (Figure SC). Further blots XhoII <br /> a neighboring wild-type cell, recombine            EcoRI-digested DNA indicated revertants are <br /> partially homologous loci cell. Such transformant    pilA+-pilB+ homodiploids (data shown), suggesting <br /> would contain wild-type pilA-pilB region (in          probably resulted homogenotization. <br /> 5.8-kb ClaI fragment) mutated pilA- -pilB+ <br /> region plus flanking sequences (in 7.4-kb ClaI fragment).       Pilin transcription pilA - pilB- mutants <br /> Supporting hypothesis observations          The level pilin transcription mutants 4.1C pilA-a/ <br /> gonococcus undergoes autolysis readily (Heebeler              pilA+ 4. IC pilB- determined Northern blot <br /> Young, 1975), transformed DNA high       analysis compared P+ wild-type strain <br /> frequency (Sparling, 1966), DNA lysed cells         4. IC (Figure 6, compare Ba B2 Db D2). After <br /> are active DNA transformation (Sarubbi Sparling,           normalization rRNA concentration detected same <br />  1974). Indeed, observed recently DNA trans-         filters (Figure 6A C), level pilin transcription was <br /> formation primary means pilin expression      observed reduced -3- 4-fold pi1A-aapilA+ <br /> sites acquire variant pilin gene sequences (H.S.Seifert et al.,   mutant increased -3- 5-fold pilB- mutant <br /> 1988).                                                            (results independent experiments). <br /> Deriving pilA + revertants                                        Electron microscopy pilA - pilB- mutants <br /> The slow growth rate pilA-alpilA+ mutants              We examined pilA-a/pilA+ pilB- mutants, pilA+ <br /> not presence Cm medium,               revertants, electron microscopy, compared their <br /> mutants grow slowly Cm&apos;o nonselective                 piliation status P+ P- variants. In all, <br /> 4374 <br />                                                                                                  Trans-regulation gonococcal pilin expression <br />    4t <br />  <br />  <br />  <br />  <br /> Fig. 7. Piliation status wild-type gonococci mutants observed electron microscopy. Grids examined x21 000 magnification. <br /> Platinum-paladium shadowed cells strain MS1I A P+ (a), 4. 1B, P- phase variant (b) 4. 1C, P+ revertant 4. 1B <br /> (c). Representative samples mutants shown MS1 1A pil B - (d) 4. 1C pilA-alpilA+ heterodiploid (e). The 4. 1C pilA+ revertant is <br /> shown (f). Three mutants class (d, e f) examined. Each mutant class piliation phenotype. <br /> Furthermore, mutation resulted piliation phenotype recipient strain MS1 1A 4. IC. No pili observed on <br /> cells 2B, P- variant MS1 A pilin sequences deleted pilE 1 pilE2 (Segal et al., 1985; data shown). <br />  <br />  <br />                                                                                                                                            4375 <br />  M.K.Taha et al. <br />  <br /> three mutants class studied,             transcription involves activity PilA complex. In a <br /> representatives class shown Figure 7. The        pilA-alpilA+ heterodiploid, aberrant complexes be <br /> pilA-alpilA+ mutants observed pili          formed consisting wild-type truncated PilA proteins. <br /> (Figure 7e). The number pili seen pilA- mutant          Supporting hypothesis observation wild-type <br /> is comparable observed 4. 1B, P- variant          mutant recA1 proteins capable forming mixed <br /> MS1 1 pilE1 intact (Segal et al., 1985),    tetramers ATPase activity dramatically reduced <br /> makes reduced pilin mRNA (Hagblom et al.,               (Ogawa et al., 1978). In addition, negative dominance <br /> 1985; M.K.Taha, M.So C.Marchal, preparation). We           observed missence lamB mutations <br /> examined 50- 100 cells, did observe whose <br />             - <br />                                                                   (Marchal Hofnung, 1983). These alternatives would <br /> piliation status comparable MSl lA (P+). On         explain slow growth rate pilA-alpilA+ <br /> the hand, presumptive pilA+ revertants produce          heterodiploids. If pilA essential regulatory gene, a <br /> the pili MS 1 lA 4. IC examined        negative dominance pilA-a pilA+ mutants <br /> by electron microscopy (Figure 7f). These data indicate      result decreased cell viability. By isolating early <br /> pilA positive effect pilin expression, mutation     insertion pilA (pilA-b), obtained mutants <br /> in pilA results cells producing pili. This effect    heterodiploid pilA. This confirms the <br /> is transcriptional level pilA clones observed   essential role pilA. But P+ normal <br /> to stimulate pilin promoter activity E. coli pilin    viability, expect insertion early <br /> mRNA pilA-alpilA+ gonococci.                           pilA completely inactivate pilA. In case, <br />    Compared wild-type P+ cells (Figure 7a c),        pilA- allele negative dominance wild-type <br /> pilB- mutant hyperpiliated (Figure 7d). The pili appear        pilA+. <br /> in clumps, detached cells. In all,             We observed pilB, conjunction pilA, a <br /> five pilB- mutants MSl lA 4. IC examined              stronger negative effect pilB pilin promoter <br /> and exhibited hyperpiliation phenotype. These       activity. pilB expression need sequence located the <br /> observations agree earlier observations             pilA gene pilA positive control. Finally, <br /> wild-type pilB inhibitory effect pilin expression       PilB activity need PilA protein co-effector. Precisely <br /> by repressing pilin gene transcription.                           pilA pilB interact regulate pilin expression is <br />                                                                   unclear. This control involve heteropolymer composed <br /> Discussion                                                        pilB pilA gene products. Alternatively, pilA <br />                                                                   pilB proteins compete binding sites pilin <br /> We identified loci mapping downstream pilEl,          promoter. The pilB ORF extends minimal <br /> pilA pilB, act trans regulate pilin expression.   fragment shown negative activity pilin <br /> pilB conjunction pilA, decreases pilin promoter           promoter E. coli. That 5&apos; half pilB to <br /> activity E. coli pilB- mutants gonococcus      exert effect activity truncated <br /> increased pilin gene transcription hyperpiliated. The     PilB 5&apos; half pilB transcript. Our sequence data <br /> inhibitory activity pilB gene pilin expression       reveal existence substantial antisense homology <br /> similar effect E. coli Pseudomonas              5&apos; ends pilin pilB message. Our <br /> aeruginosa hyp genes piliation (Orndorff Falkow,           sequence data possible <br />  1984; Johnson Lory, 1987). pilA stimulates pilin       promoters pilB overlap pilA transcrip- <br /> promoter activity E. coli pilA-alpilA+ heterodiploid       tion. Indeed, transcriptional initiation pilB promoter <br /> gonococci produce small pili. Our failure        result pilB transcript overlaps pilA <br /> to obtain single pilA- mutant strongly suggests pilA       transcript. The significance findings awaits detailed <br /> has essential regulatory function gonococcus. The       transcription studies. <br /> reduced pilin gene transcription piliation pilA-a/            Our data strongly suggest pilA essential regulator <br /> pilA+ heterodiploids, stimulatory activity      gonococcus. We favor model regulation <br /> pilA pilin promoter, indicate pilA plays       pilA gene product plays role central <br /> role pilin transcriptional regulation. Sequence analysis       activator. Indeed recent data suggest piliation and <br /> shows pilA pilB genes opposite orientation    envelope components coordinately regulated <br /> relative other, 5&apos; genes overlap.     (Klimpel Clark, 1988). <br /> The ORFs pilA pilB genes <span id='am-4' about='Thesaurus:code' typeof='owl:Thing'>code</span> 38.6-              Transcriptional regulation shown control pilus <br /> 57.9-kd proteins respectively. These sizes correspond        expression systems. Examples pap <br /> with proteins detected Zubay assays subclones      pilus E. coli urinary tract isolates (Baga et al., 1985), the <br /> of genes.                                                   piltfs Vibrio cholerae (Miller et al., 1987) E. coli <br />    It interesting piliation affected pilA-alpilA+   Type 1 pilus (Eisenstein, 1981). Sequence homology the <br /> but pilA-blpilA+ heterodiploids. One explanation           5&apos; promoter regions pilin genes N. gonorrhoeae and <br /> for observation minitransposon insertion    P. aeruginosa (Johnson et al., 1986) strongly suggests that <br /> the distal pilA (pilA-a) leads production       systems common regulatory components. A <br /> a truncated protein able bind pilin promoter,     central activator shown control expression of <br /> but unable activate transcription. Indeed,        virulence factors V. cholerae (Miller et al., <br /> sequence data indicate putative DNA binding domain         1987), Bordetella pertussis (Weiss Falkow, 1984), and <br /> exists near N-terminal end PilA. The truncated PilA        Agrobacterium tumefaciens (Winans et al., 1986). Moreover, <br /> protein compete native protein pilin           environmental factors play major role expression <br /> promoter binding site, decreasing pilin transcription.    genes. <br /> A variation theme activation pilin                As pilA pilB effect pilin expression, they <br />  <br /> 4376 <br />                                                                                                   Trans-regulation gonococcal pilin expression <br />  <br /> are candidates factors control class &apos;trans-                    fixed UV (254 nm) 4 min. <br /> regulated&apos; pilus phase variants. If pilA pilB participate                    Total RNA isolated N.gonorrhoeae cells grown <br /> in pilus phase switch, expression pilA pilB                    18-20 h G medium supplement (Diagnostics Pasteur, France), <br />                                                                               harvested lysis buffer (10 mM Na-acetate, 50 mM NaCl, 0.5% <br /> would finely balanced, shift level                    SDS). RNAs extracted hot acidic phenol diluted 3 <br /> of expression gene relative                         vol. ice-cold 4 M guanidinium isothiocyanate, 10 mM Na-acetate, 50 mM <br /> determine piliation status cell. This balance                NaCl 1 M 3-mercaptoethanol. 2.5 ml solution layered on <br /> be brought ways. One possibility                1.5 ml CsCl cushion (6 M CsCl 10 mM Tris-HCI, pH 7.5, 1 mM <br />                                                                               EDTA, filtered 0.45-ltm membrane) centrifuged 38 000 <br /> differential balance determined high frequency                          r.p.m. 20 h 20&#194;&#176;C SW 50.1 rotor. The RNA pellet was <br /> revertable mutations/rearrangements pilA                     resuspended water, extracted hot acidic phenol, acidic phenol/ <br /> pilB gene. In case studied sequence changes                       chloroform, chloroform/isoamyl alcohol, precipitated ethanol. <br /> detected pilA pilB genes MS 11                       Pellets vacuum dried resuspended 20 mM phosphate buffer <br /> of P- variants. An alternate possibility expression               (pH 7). <br />                                                                                 RNA (10    jg) resuspended formamide/formaldehyde separated in <br /> of pilA pilB genes control locus.                 1.8% agarose gel 20 mM phosphate buffer (pH      7), 6% formaldehyde, <br /> Finally, pilA pilB genes regulate                      blotted overnight N-hybond (Amersham) filters 20 x SSPE. <br /> by undefined way. A detailed characterization                    The filter dried exposed UV (254 nm) 4 min. The filter <br />                                                                               washed 4 x SSPE, prehybridized 0.5 M phosphate buffer (pH <br /> these genes shed light matter.                               7), 7% SDS, 1 mM EDTA 1% BSA, hybridized 32P-labeled <br />    Finally, pilA -pilB control pilin expression                   nick-translated probe buffer 100 Ag/ml salmon sperm DNA <br /> phenomenon separate pilus phase switch. The                          65&#194;&#176;C. Filters washed 40 mM phosphate buffer (pH 7), 1 % SDS, <br /> coexistence modes piliation control                        1 mM EDTA, times, 15 min 65&#194;&#176;C. Kodak XAR-5 film <br /> described E. coli Type I fimbriae, &apos;on                   exposed -70&#194;&#176;C intensifying screen. <br /> off&apos; control &apos;shares&apos; regulation pilus expression                   Recombinant DNA techniques <br /> with transcriptional control system. The drastic metastable                 All recombinant DNA protocols               Maniatis   et   al.   (1982). <br /> regulation inversion DNA fragment                          pNG1749-CAT pNG1721-CAT generated flush ending the <br /> containing pilin gene promoter (Abraham et al., 1985;                     Hin dHm fragment containing CAT gene pCM7 (Close Rodriguez, <br /> Klemm, 1986). In addition, transcriptional regulation                      1982) inserting fragment single blunt-ended MstII site <br />                                                                               pNG1749 pNG 1721. pNG 1749-CATm partial digestion <br /> effected hyp gene product pilin gene                    pNG1749-CAT EcoRl, ligation EcoRI linearized pRPZ1 11 <br /> the &apos;on&apos; mode (Ormdorff et al., 1985). Two modes control                   (Seifert, 1984). Transformants selected agar containing Kan30 and <br /> of expression virulence factors described                   Cm&apos;5. <br /> in B.pertussis (Weiss Falkow, 1984). In system,                        pNG17 4-kb pNG1711 Sall-SnaI fragment cloned pUC9 <br />                                                                               linearized enzymes. pNG57 3-kb Sall fragment from <br /> modulation expression virulence genes sensitive                  pNG56 cloned SalI site pUC9. pNG58 2-kb XhooI-EcoRI <br /> to environmental changes. To knowledge, detailed                          fragment pNG17 cloned SalI-Eco RI site pUC 18. pNG59 <br /> studies carried examine influence                    pNG60 are, respectively, 1.3- 1.9-kb XNoI -SalI fragment from <br /> of environment gonococcal pilus expression,                   pNG57 cloned SalI site pUC18. pNG61 1.9-kb HindmI-XWoI <br />                                                                               fragment pNG57 cloned Hindlll-Sall site pUC19. To <br /> gonococcal piliation virulence change culture                    generate pNG63, 2.5-kb PvuI-HindIII fragment pNG57 gel <br /> conditions (Keevil et al., 1986). Of different pathways                   purified ligated SmaI-HindIll site pUC 18. PvuI ends were <br /> which control pilus expression gonococcus,                         blunted second ligation carried circularize recombinant <br /> pilA-pilB pathway likely respond                          DNA. To generate pNG62, 2.5-kb EcoRI-HindIH fragment from <br /> environmental factors. The identification pilA pilB                pNG63 cloned Eco RI- Hin dIll-cut M13 mpl 1 vector, and <br />                                                                               370-bp deletion generated Eco RI Xho I site using <br /> loci help define parameters                       technique Dale   et   al.   (1985). <br /> virulence factor expressed. <br />                                                                               Cell-free protein synthesis assays <br /> Materials methods                                                         The described Zubay et al. (1970) used minor <br />                                                                               modifications (Marchal et al., 1980). DNAs used program vitro <br />                                                                               purified CsCI density gradients. <br /> Strains media <br /> All strains used N.gonorrhoeae strain MS1I A (pilEl +, pilE2+)       DNA sequencing <br /> its derivatives, described Segal et al. (1985) Hagblom   The 1.9-kb SalI-XhoI 1.3-kb XhooI-SalI fragments pNG56 <br /> et al. (1985). Cultures passed 18-22 h G medium plates          subcloned M13 DNA mplO mpl 1 orientations, well <br /> with G supplement (Diagnostics Pasteur, France) GCB plates (Difco)      1.9-kb SalI-XhoI 2-kb         X7hoI-SmnaI <br />                                                                                                                                      fragments pNG17. <br /> with supplements (Kellogg et al., 1963) previously described (Segal et     Sequential deletions inserts sequenced technique <br /> al., 1985). For recipients recombinant plasmids, E.coli RDP145 (Buzby      Dale et al. (1985). Oligonucleotides RD29 RD22 purchased <br /> et al., 1983) used isolation pNG1749-CAT pNG172 1-CAT;         IBI. Briefly, unidirectional digestions single-stranded inserts <br /> E.coli GC6 [a P1 transductant GC1 (Meyer et al., 1982) brings        varying times (2-20 min), using T4 DNA polymerase, after <br /> in locus delta (srl-recA)306::TnlO (H.S.Seifert, unpublished data)]       annealing specific oligomer create double-stranded EcoRI site (for <br /> for isolation pNG1749-CATm, pNGl1 pNG56; Ecoli JM109                   mp 11 recombinants) HindIll site (for mplO recombinants), and <br /> (Yanisch-Perron et al., 1985) subcloning pNG17, pNG57; E.coli      linearization cutting site. Recircularization transformation of <br /> TGI(Gibson, 1984) pNG58 63 M13 recombinant clones              E.coli strain TG1 made, sizing deletions, DNA was <br /> that sequenced. Plasmids isolated JM109 TG1            sequenced subcloning step. Sequencing reactions carried <br /> transferred Ecoli DH5 (Hanahan, 1983). Transformation N.                using dideoxy chain termination procedure Sanger et al. (1977). <br /> gonorrhoeae described Sparling (1966) gonococci <br /> were plated directly antibiotic-containing plates allowing     S1 mapping <br /> expression antibiotic resistance liquid GCB medium (H.S.Seifert,        The start pilA mRNA synthesis determined SI mapping as <br /> unpublished data).                                                            described Maniatis et al. (1982). DNA probe single-stranded <br />                                                                               fragment 5&apos; Xhol-3&apos; PvuI, complementary pilA transcript. Itas wasthe <br />  Nucleic acid extraction, purification hybridization                      synthesized homogeneously labeled using radioactive dTTP, and <br />  DNA isolation restriction digestion described Segal et al.    complement 3&apos; XhoI -5&apos; PvuI fragment clonedwasin M13 mplO, <br />  (1985), DNA transferred agarose gels           gel purified. The <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> protected fragment      analyzed 7 % <br />  N-hybond (Amersham) 1 M ammonium acetate, filter dried           polyacrylamide   8 M  urea gel. <br />                                                                                                                                                           4377 <br />   M.K.Taha et al. <br />  <br />  Shuttle mutagenesis gonococcal inserts E.coli                             Klier,A., Kunst,F. Rapoport,G. (1979) Nucleic Acids Res., 7, <br />  Techniques, strains plasmids used described Seifert et al.          997-1010. <br />  (1986a,b). The 4-kb EcoRl-HindIH fragment pNG17 subcloned              Klimpel,W.K. Clark,V.L. (1988) Infect. Immun., 56, 808-814. <br />  shuttle vector pHSS6, recombinant plasmid transferred     Lambden,P.R., Heckels,J.E., James,L.T. Watt,P.J. (1979) J. Gen. <br />  Tn3 transposase +E.coli strain RDPl45(pTCA). The resultant               Microbiol., 144, 305-312. <br />  strain conjugated RDPI45(pOX38::mTn3Cm-3) allow                Lipman,D.Z. Pearson,W.R. (1985) Science, 277, 1435-1440. <br />  transposition occur. The plasmids conjugated NS2114Sm         Maniatis,T., Fritsch,E.F. Sambrook,J. (1982) Molecular Cloning. A <br />  allow resolution cointegrates. Transconjugants mTn3Cm-3                Laboratory Manual. Cold Spring Harbor Laboratory Press, Cold Spring <br />  insertions plasmid selected plating agar containing           Harbor, NY. <br />  Cm30, insertion sites minitransposon mapped restriction      Marchal,C. Hofnung,M. (1983) EMBO J., 2, 81-86. <br />  analysis plasmids.                                                           Marchal,C., Perrin,D., Hedgpeth,J. Hofnung,M. (1980) Proc. Natl. <br />                                                          <br /> </body></html>