<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Khuri Wuchty BMC Bioinformatics (2015) 16:109 <br /> DOI 10.1186/s12859-015-0536-x <br />  <br />  <br />  <br />  <br />  RESEARCH ARTICLE                                                                                                                              Open Access <br />  <br /> Essentiality centrality protein interaction <br /> networks revisited <br /> Sawsan Khuri1,2 Stefan Wuchty1,2* <br />  <br />  <br />   <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />   Background: Minimum dominating sets (MDSet) protein interaction networks allow control underlying <br />   protein interaction networks topological placement. While essential proteins enriched MDSets, <br />   hypothesize statistical properties biological functions essential genes enhanced we <br />   focus essential MDSet proteins (e-MDSet). <br />   Results: Here, determined minimum dominating sets proteins (MDSet) interaction networks E. coli, S. <br />   cerevisiae H. sapiens, defined subsets proteins remaining protein reached single <br />   interaction. We compared topological functional parameters essential, MDSet, essential MDSet <br />   (e-MDSet) proteins. In particular, observed topological placement allowed e-MDSet proteins provide <br />   positive correlation degree lethality, connect protein complexes, stronger impact <br />   network resilience essential proteins alone. In comparison essential proteins interactions <br />   e-MDSet proteins appeared frequently complexes, interactions e-MDSet proteins between <br />   complexes depleted. Finally, e-MDSet proteins classified functional groupings play central role in <br />   <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span>survival</span> adaptability. <br />   Conclusions: The determination e-MDSet organism highlights set proteins enhances enrichment <br />   signals biological functions essential proteins. As consequence, surmise e-MDSets provide new <br />   method evaluating core proteins organism. <br />   Keywords: PPI, Essential proteins, Protein network, Protein function <br />  <br />  <br /> Background                                                                           prompting Nacher Akutsu determine minimum <br /> The biological importance protein frequently con-                            dominating sets (MDSet) nodes, defined set cen- <br /> sidered question number interactions given                             trally located nodes provide control undirected net- <br /> protein involved [1-3], suggesting high topo-                             works [16]. Answering question nodes that <br /> logical centrality indicator protein&#226;&#8364;&#8482;s importance                         predicted important control interaction <br /> [4-9]. In addition, proteins involved                            networks translate directly functional sets proteins, <br /> large number protein complexes [10], signifying                              minimum dominating sets enriched dis- <br /> their essentiality consequence complex in-                             ease related essential genes protein interaction net- <br /> volvement [5,9,11-14].                                                               works [17,18]. <br />   Focusing determination nodes control                               Here, hypothesize essential proteins appear <br /> entire network, Liu et al. introduced maximum matching                             MDSets (e-MDSet) enhance enrichment sig- <br /> approach predict nodes allowed control vari-                          nals biological functions compared essential proteins <br /> ous technical, social biological networks [15]. However,                         alone. Specifically, considered high-quality protein in- <br /> their approach applicable directed networks,                             teractions S. cerevisiae H. sapiens de- <br />                                                                                      termined large-scale yeast two-hybrid approaches, as <br /> * Correspondence: wuchtys@cs.miami.edu                                               recently released high-throughput binary inter- <br /> 1 <br />  Department Computer Science, University Miami, Coral Gables, FL               action set E. coli [19]. While highly connected proteins <br /> 33146, USA                                                                           showed weak enrichment essential proteins recov- <br /> 2 <br />  Center Computational Science, University Miami, Coral Gables, FL <br /> 33146, USA                                                                           ered strong correlation protein&#226;&#8364;&#8482;s degree its <br />  <br />                                         &#194;&#169; 2015 Khuri Wuchty; licensee BioMed Central. This Open Access article distributed terms Creative <br />                                         Commons Attribution License (http://creativecommons.org/licenses/by/4.0), permits unrestricted use, distribution, and <br />                                         reproduction medium, provided original work properly credited. The Creative Commons Public Domain <br />                                         Dedication waiver (http://creativecommons.org/publicdomain/zero/1.0/) applies data available article, <br />                                         unless stated. <br />  Khuri Wuchty BMC Bioinformatics (2015) 16:109                                                                                                                                                 Page 2 8 <br />  <br />  <br />  <br />  <br /> tendency essential focused essential pro-                                                                 1,203 proteins experimentally obtained <br /> teins appeared corresponding MDSet                                                                     yeast-two-hybrid approach [19]. As source infor- <br /> (e-MDSet). The impact combination essentiality                                                                   mation essential genes, collected 712 essential <br /> of protein presence corresponding MDSet                                                                   genes E. coli 1,110 essential genes S. cerevisiae <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> was evidenced observation e-MDSet                                                                      DEG database [21], obtained 2,708 essential genes in <br /> proteins likely connect protein complexes                                                                 H. sapiens online gene essentiality database (OGEE) <br /> essential proteins organisms. Further-                                                                  [22]. We defined MDSet optimized subset proteins <br /> more, e-MDSet proteins predominantly oc-                                                                     interaction network remaining (i.e. <br /> cupied central positions networks connected                                                                    non-MDSet) protein reached interaction. <br /> more protein complexes. As corollary, e-MDSet proteins                                                                   Therefore, non-MDSet protein connected least <br /> were enriched interactions occured com-                                                                     MDSet protein (Figure 1A). In protein interaction <br /> plexes, vice versa. Taking closer look functional                                                                 networks determined corresponding minimum dominat- <br /> classifications e-MDSet proteins enhance                                                                     ing sets solving integer-based linear programming <br /> the enrichment signals functional groups play role                                                               problem (see Materials Methods). In particular, we <br /> in responses external stimuli physiological con-                                                                569 MDSet proteins yeast interactions network <br /> dition cells.                                                                                                           (21.26%), 352 proteins MDSet E. <br />                                                                                                                            coli interactions (29.2%) 940 MDSet proteins (15.9%) in <br /> Results                                                                                                                    human interaction network. In comparison pro- <br /> We utilized network 6,225 high-quality interactions be-                                                               teins (&lt;kall &gt; = 3.9), mean degree MDSet proteins in <br /> tween 2,640 proteins S. cerevisiae 17,523 high-quality                                                              yeast interaction network significantly increased <br /> links 5,926 human proteins entirely deter-                                                               (&lt;kMDSet &gt; = 8.5), observation held E. coli <br /> mined yeast-two hybrid approaches [20]. As E. coli,                                                              (&lt;kall &gt; = 3.2, &lt;kMDSet &gt; = 5.6) human networks well <br /> used recently released, map 1,938 interactions                                                                  (&lt;kall &gt; = 5.3, &lt;kMDSet &gt; = 15.5). In turn, observed that <br />  <br />  <br />                       A                                                                                    B <br />                                                                                                                                        1.5 <br />                                                                                                            lg2 [fess. MDSet /fess.] <br />  <br />  <br />  <br />  <br />                                                                                                                                               1 <br />  <br />                                                                                                                                        0.5 <br />  <br />                                                                                                                                               0 <br />                                                                                                                                                                             E. coli <br />                                                                                                                                       -0.5                                  S. cerevisiae <br />                                                                                                                                                                             H. sapiens <br />                                                                                                                                               -1 <br />                                                                                                   MDS                                              1               10                       100 <br />                                                                                                                                                                degree <br />                       C7                                                                                   D <br />                                                                  7                                                                            7 <br />                                                  enr.essential <br />  <br />  <br />  <br />  <br />                                                                  6 <br />                                                                                                                           enrichmente-MDSet <br />  <br />  <br />  <br />  <br />                                          6                       5 <br />                       enrichment MDSet <br />  <br />  <br />  <br />  <br />                                                                  4                                                                            6 <br />                                                                  3 <br />                                          5                       2                                                                            5 <br />                                                                  1 <br />                                          4                       0 <br />                                                                      1   3   5   7   9 11 13 15 <br />                                                                                                                                               4 <br />                                          3                                                                                                    3 <br />                                          2                                                                                                    2 <br />                                          1                                                                                                    1 <br />                                          0                                                                                                    0 <br />                                              1                                       6              11                                             1   3   5   7        9    11    13       15 <br />                                                                                                          degree k <br />  Figure 1 e-MDSet proteins follow centrality-lethality rule. (A) In toy network defined minimum dominating set (MDSet) an <br />  optimized subset nodes (red square symbol) remaining (i.e. non-MDSet) node (gray circle symbol) immediately <br />  reached step. Therefore, non-MDSet protein connected MDSet protein. (B) After calculated corresponding <br />  MDSets binary interaction networks E. coli, S. cerevisiae H. sapiens, grouped proteins bins logarithmically increasing degree. In <br />  bin determined fraction essential proteins participated underlying MDSet well, allowing observe essential <br />  MDSet proteins (e-MDSet) preferably enriched highly connected essential proteins organisms. In inset (C) calculated <br />  enrichment essential proteins function degree binary protein interaction networks E. coli, S. cerevisiae H. sapiens. <br />  Generally, essential proteins binary interactions failed produce significant trend organism. Focusing enrichment MDSet <br />  proteins, however, recovered strong increasing trend (D) reinforced focusing e-MDSet proteins. <br />  Khuri Wuchty BMC Bioinformatics (2015) 16:109                                                                                                                Page 3 8 <br />  <br />  <br />  <br />  <br /> essential genes slightly increased mean degree (E. coli                                  H. sapiens (Figure 1C). Notably, trends rein- <br /> &lt; kess. &gt; = 3.4, S. cerevisiae &lt; kess. &gt; = 4.8, H. sapiens &lt; kess. &gt; =                         forced focused e-MDSet proteins, confirming <br /> 6.7) compared corresponding values proteins                                   hypothesis (Figure 1D). <br /> the underlying networks. In particular, 91 essential                                     As different measure central placement essen- <br /> proteins participated underlying MDSet proteins                                    tial proteins, calculated betweeness centrality all <br /> in E. coli (e-MDSet), number statistically significant                               proteins underlying networks. We defined set of <br /> applying Fisher&#226;&#8364;&#8482;s exact test (P = 9.2 &#195;&#8212; 10&#226;&#710;&#8217;4). Notably,                                bottleneck proteins 20% proteins highest <br /> enirchment <span id='am-82' about='xsp:pattern' typeof='owl:Thing'>pattern</span> applied remaining organisms                                       centrality [23]. As null-model, randomly picked essen- <br /> well, 179 e-MDSet proteins S. cerevisiae                                     tial proteins 10,000 times determined enrichment of <br /> (P = 2.3 &#195;&#8212; 10&#226;&#710;&#8217;7) 209 H. sapiens (P = 5.3 &#195;&#8212; 10&#226;&#710;&#8217;3).                                       essential proteins corresponding sets bottleneck <br />                                                                                                proteins. Figure 2A indicates essential proteins were <br /> Centrality lethality                                                                       weakly enriched bottleneck proteins. While MDSet <br /> As direct comparison, grouped proteins                                      proteins strongly enriched bottleneck proteins <br /> bins logarithmically increasing degree calculated                                   organisms, observed reinforcement these <br /> fraction essential genes group. Determining                                     trends considered e-MDSet proteins, suggesting <br /> fraction e-MDSet proteins bin, Figure 1B clearly                                    topological placement MDSet proteins enhanced <br /> shows e-MDSet proteins predominantly enriched                                        enrichment signals essential proteins (Figure 2A). <br /> in groups essential proteins increased num-                                        <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> measure protein&#226;&#8364;&#8482;s impact interaction net- <br /> ber interaction partners. Considering enrichment                                     work&#226;&#8364;&#8482;s resilience, performed robustness analysis. We <br /> essential genes function degree under-                                      sorted e-MDSet proteins according degree in <br /> lying interaction networks, grouped proteins                                        interaction networks organisms. Starting with <br /> least certain number interactions counted                                         connected protein gradually deleted proteins <br /> number essential proteins bin. To provide                                         calculated number connected components after <br /> control, randomly sampled sets essential genes                                      deletion step. In comparison, considered sets of <br /> null-model defined ratio observed ex-                                       equal size connected, essential proteins. Figure 2B <br /> pected number enrichment essential genes                                          indicates successive deletion e-MDSet proteins <br /> each group. The inset Figure 1C indicates binary                                       higher impact network topology producing <br /> interaction networks organisms failed produce                                      connected components removing fewer inter- <br /> viable trend. Hypothesizing proteins                                          actions connected essential proteins E. <br /> enriched highly connected proteins repeated                                           coli, S. cerevisiae H. sapiens. <br /> our initial enrichment analysis considering MDSet and <br /> e-MDSet proteins. Indeed, increasing correl-                                       Protein complexes <br /> ation elevated degree presence                                            Moving higher level cellular organization, calcu- <br /> MDSets interaction networks E. coli, S. cerevisiae                                   lated complex participation coefficients proteins, a <br />  <br />  <br />                      A                                  essential <br />                                                                                        B <br />                                           3                              ** P &lt; 10-4                   700 <br />                                                         MDSet                                                    essential <br />                                                                           * P &lt; 10-2                   600 <br />                                          2.5            E-MDSet                                                  e-MDSet <br />                                                                              ** <br />                      lg2 [fobs./fexp.] <br />  <br />  <br />  <br />  <br />                                                                                        N conn. comp. <br />  <br />  <br />  <br />  <br />                                                                     **                                 500                                       yeast     500 <br />                                           2        ** <br />                                                                                                        400                                                 400 <br />                                          1.5                                                                                                               300 <br />                                                                                                        300 <br />                                                                                                                                             E. coli        200 <br />                                           1 <br />                                                                                                        200                                                 100 <br />                                                * <br />                                          0.5                                                           100                                                  0 <br />                                                                                                                          0     200   400   600   800     1000 <br />                                           0                                                             0 <br />                                                E. coli      S. cervisiae H. sapiens                          0   1000   2000     3000       4000         5000 <br />                                                                                                                         N deleted PPI <br />  <br />  Figure 2 Centrality e-MDSet proteins. (A) We determined betweeness centrality protein interaction network E. <br />  coli, S. cerevisiae H. sapiens chose 20% proteins highest betweeness centrality sets bottleneck proteins. <br />  Randomly sampling sets 10,000 times, observed bottleneck proteins weakly enriched essential proteins all <br />  organisms. In turn, MDSet e-MDSet proteins significantly enriched bottleneck proteins. (B) Utilizing subset e-MDSet <br />  proteins E. coli, S. cerevisiae (both inset) H. sapiens, sorted proteins according degree. Starting highest <br />  connected protein, successively deleted proteins calculated number connected components. To compare, applied this <br />  procedure set highest connected essential proteins size organism. Our results suggest removal of <br />  e-MDSet proteins led lower number deleted interactions higher number connected components. <br />  Khuri Wuchty BMC Bioinformatics (2015) 16:109                                                                                                              Page 4 8 <br />  <br />  <br />  <br />  <br /> value indicates protein&#226;&#8364;&#8482;s tendency interact                         Functional classes <br /> different complexes interactions. The com-                           These striking network patterns consistent proteins <br /> plex participation coefficient tends 1 given                         high level regulatory functions. Therefore, took <br /> protein predominantly interacts proteins                          logical step grouping protein sets according <br /> complex vice versa. In particular, utilized set                        broad functional classes defined clusters of <br /> 517 protein complexes E. coli [24], 430 protein com-                            orthologous groups (COGs) [27,28]. Specifically, counted <br /> plexes S. cerevisiae [25] 1,843 protein complexes                        occurrence essential, MDSet, e-MDSet pro- <br /> H. sapiens [26]. Since essential proteins tend connect                          teins groups (Figure 5) randomly assigned <br /> more complexes non-essential proteins, hypothe-                            number functional classes protein <br /> sized topological placement MDSet proteins                        10,000 times null model. On organism specific level, <br /> enhance statistical characteristic essential proteins.                     essential, MDSet, e-MDSets proteins roughly appeared <br /> The comparison frequency distributions                               enriched/depleted functional groups or- <br /> corresponding participation coefficients Figure 3A                              ganisms, apart certain striking cases enrichment <br /> clearly confirmed assumption. Indeed, e-MDSet                                  patterns different organisms largely incongruent. <br /> proteins interaction networks largely reached                               Notably, enrichment signals essential proteins were <br /> into higher number different protein complexes                                involved RNA processing (group A) cell cycle control <br /> compared essential proteins alone. On basis                             (D), transcription (K), post-translational modification (O), <br /> of finding, wondered interactions be-                              intracellular transport (U) cytoskeleton (Z) en- <br /> tween essential proteins enriched single                                hanced considering e-MDSet proteins S. cerevisiae. <br /> complexes complexes (Figure 4A). Specif-                                Conversely, e-MDSet proteins exceedingly depleted in <br /> ically, counted number inter- intra-                                 aminoacid (A), carbohydrate (G) lipid (I) transport and <br /> complex interactions randomly assigned                                metabolism chaperone turnover activities (P) <br /> number proteins corresponding complex                                   compared essential proteins alone. While enrich- <br /> 10,000 times random null model. Generally,                                 ment patterns largely differ E. coli H. sapiens, we <br /> observed interactions essential proteins                              amino acid transporations metabolism (E) <br /> connecting complexes appeared frequently                                 chaperones turnover functions (P) depleted in <br /> expected organisms (P &lt; 10&#226;&#710;&#8217;4, Figure 4B). In                                organisms. In comparison essential proteins and <br /> turn, interactions proteins                                MDSet proteins, e-MDSet proteins E. coli enhanced the <br /> complex occurred frequently expected                                  enrichment signal proteins play role translation <br /> all organisms (P &lt; 10&#226;&#710;&#8217;4, Figure 4B). Focusing inter-                            (J). In turn, cell cycle control proteins enriched with <br /> actions MDSet proteins, similar,                                essential proteins (D), signal mitigated e- <br /> albeit slightly weaker, signal. In turn, trends                         MDSet proteins. In H. sapiens, e-MDSet pro- <br /> were largely enhanced considered e-MDSet                                   teins enhance enrichment signals essential genes with <br /> proteins organisms (P &lt; 10&#226;&#710;&#8217;4, Figure 4B).                                   transcription (K), replication (L) signal transduction <br />  <br />  <br />                                                        1                                  1                                      1 <br />                                                        0.9                                0.9                                    0.9 <br />                                       E. coli          0.8         S. cerevisiae          0.8                H. sapiens          0.8 <br />                                                                                                                                        complex participation <br />  <br />  <br />  <br />  <br />                                                        0.7                                0.7                                    0.7 <br />                                                        0.6                                0.6                                    0.6 <br />                                                        0.5                                0.5                                    0.5 <br />                                                                                                     essential <br />                                                        0.4                                0.4       e-MDSet                      0.4 <br />                                                        0.3                                0.3                                    0.3 <br />                                                        0.2                                0.2                                    0.2 <br />                                                        0.1                                0.1                                    0.1 <br />                                                        0                                  0                                      0 <br />  <br />                       0.6      0.4      0.2        0         0.6    0.4      0.2      0          0.6      0.4      0.2       0 <br />                                                                    frequency <br />  Figure 3 Essential MDSet proteins protein complexes. We calculated complex participation coefficients proteins binary <br />  interaction networks E. coli, S. cerevisiae H. sapiens. Specifically, e-MDSet proteins reached complexes compared essential proteins in <br />  E. coli (P = 1.9 &#195;&#8212; 10&#226;&#710;&#8217;4, Wilcoxon test), S. cerevisiae (P = 3.5 &#195;&#8212; 10&#226;&#710;&#8217;6,) H. sapiens (P = 9.6 &#195;&#8212; 10&#226;&#710;&#8217;11). <br />  Khuri Wuchty BMC Bioinformatics (2015) 16:109                                                                                           Page 5 8 <br />  <br />  <br />  <br />  <br />                     A                                 B <br />                                                                          2 <br />  <br />  <br />  <br />  <br />                                                                                                                             E. coli <br />                                                                          1 <br />                               complex <br />                                   A                                      0 <br />                                                                          -1 <br />  <br />  <br />  <br />  <br />                                                       lg2 [Nobs/Nexp] <br />                                                                          2 <br />  <br />  <br />  <br />  <br />                                                                                                                             S. cerevisiae <br />                                                                          1 <br />                                                                          0 <br />                                         B <br />                                                                          -1 <br />  <br />                               C                                          1 <br />  <br />  <br />  <br />  <br />                                                                                                                             H. sapiens <br />                                                                         0.5 <br />                                 within <br />                                                                  0 <br />                                                                   -0.5 <br />                           complexes                                           MDSet        essential         e-MDSet <br />  Figure 4 Enrichment interactions protein complexes. (A) Schematic illustration interactions appear <br />  complexes. In (B) determined number interactions MDSet, essential e-MDSet proteins same <br />  complex complexes E. coli, S. cerevisiae H. sapiens. As random null model, resampled proteins complexes <br />  10,000 times. Generally, interactions complexes appear diluted interactions complexes to <br />  enriched. While appeared significant sets interactions essential proteins observed signals enhanced <br />  interaction sets e-MDSet proteins (P &lt; 10&#226;&#710;&#8217;4). <br />  <br />  <br />  <br /> functions, transportation (E,H,P,U) translation (J)                      The ability MDSet &#226;&#8364;&#732;cover&#226;&#8364;&#8482; proteins network <br /> functions depleted.                                                        just question finding connected pro- <br />                                                                                 teins, necessitates determination lowest num- <br /> Discussion conclusions                                                      ber strategically placed proteins [17]. Such concept <br /> Here, determined minimum domainting sets (MDSets)                            highlights idea MDSets capture proteins <br /> of proteins underlying binary interaction networks                       involved adaptability lethality. While <br /> of E. coli, S. cerevisiae H. sapiens en-                     essential MDset proteins appear enriched bottleneck <br /> tirely determined yeast two-hybrid approaches. We de-                        protein sets, respectively, topological <br /> fined MDSets smallest group strategically placed                      placement e-MDSet enhanced initial trends. Further- <br /> proteins remaining protein (i.e. non-                           more, e-MDSet proteins predominantly connect more <br /> MDSet protein) immediately reached                             protein complexes essential MDSet proteins alone, <br /> single interaction. Therefore, non-MDSet protein in-                       confirming hypothesis. As functional aspects, our <br /> teracts MDSet protein. In words,                      analysis revealed e-MDSet proteins broadly fell into <br /> MDSet smallest possible set proteins allows                      functional classes vital survival reproduction <br /> us &#226;&#8364;&#732;cover&#226;&#8364;&#8482; remaining proteins underlying net-                     cell cycle control, trafficking, cytoskeletal, transla- <br /> works. We hypothesized topological placement                        tional posttranslational modifiying functions. These <br /> MDSet proteins correlate presence                          functional groupings highly dynamic responses <br /> essential proteins underlying protein interactions                       external stimuli physiological conditions cells. <br /> networks. Indeed, MDSet proteins                             As such, MDSet provides optimized set topologic- <br /> enriched essential proteins interaction net-                        ally central proteins contributing essenti- <br /> works. Despite absence correlation                          ality genes, including necessary continued <br /> number interaction protein&#226;&#8364;&#8482;s tendency es-                        survival changing environment. <br /> sential, anticipated ability cover pro-                       Notably, results organism-independent and <br /> teins allow highly connected MDSet                          strongly suggested MDSet proteins provide statis- <br /> proteins increasingly essential survival                     tical enhancement topological functional charac- <br /> organism adaptability [17]. Indeed, recovered                      teristics essential genes. We think MDSet proteins <br /> strongly ascending correlation protein&#226;&#8364;&#8482;s lethality                    &#226;&#8364;&#339;vital essential&#226;&#8364;? set, scoring significantly higher es- <br /> and number interactions organisms                         sential proteins alone, enhancing topological parameters <br /> considered e-MDSet proteins.                                                    underlying networks. Furthermore, network <br />  Khuri Wuchty BMC Bioinformatics (2015) 16:109                                                                                               Page 6 8 <br />  <br />  <br />  <br />  <br />                     4 <br />  <br />                     2 <br />  <br />  <br />  <br />  <br />                                                                                                                                                        E. coli <br />                     0 <br />  <br />                     -2 <br />  <br />                     -4 <br />                          A    B    C    D    E    F    G     H   I    J    K    L    M    N    O     P    Q RS T           U    V    W     Y     Z <br />                     4 <br />   lg2 [Nobs/Nexp] <br />  <br />  <br />  <br />  <br />                                                                                                                                                        S. cerevisiae <br />                     2 <br />  <br />                     0 <br />                                                                                                                                     MDSet <br />                     -2                                                                                                              essential <br />                                                                                                                                     e-MDSet <br />                     -4 <br />                          A    B    C    D    E    F    G     H   I    J    K    L    M    N    O     P    Q RS T           U    V    W     Y     Z <br />                     4 <br />  <br />                     2 <br />  <br />  <br />  <br />  <br />                                                                                                                                                        H. sapiens <br />                     0 <br />  <br />                     -2 <br />  <br />                     -4 <br />                          A    B    C    D    E    F    G     H   I    J    K    L    M    N    O     P    Q RS T           U    V     W     Y    Z <br />               A      RNA processing modification         J Translation, ribosomal structure biogenesis   RS General Function/unknown <br />               B      Chromatin structure dynamics        K Transcription                                     T  Signal transduction mechanisms <br />               C      Energy production conversion        L Replication, recombination repair             U  Intracellular trafficking, secretion &amp; <br />               D      Cell cycle control, cell division,      M Cell wall/membrane/envelope biogenesis               vesicular transport <br />                      chromosome partitioning                 N Cell motility                                     V  Defense mechanisms <br />               E      Amino acid transport metabolism     O Posttranslational modification, protein           W Extracellular structures <br />               F      Nucleotide transport metabolism     P turnover, chaperones                              Y  Nuclear structure <br />               G      Carbohydrate transport metabolism     Inorganic ion transport metabolism            Z  Cytoskeleton <br />               H      Coenzyme transport metabolism       Q Sec. metabolites biosynthesis, transport <br />               I      Lipid transport metabolism            catabolism <br />  Figure 5 Enrichment functional classes. We determined number proteins MDSet, essential e-MDSet sets E. coli, S. cerevisiae <br />  H. sapiens belong underlying functional classes. As random null model resampled proteins classes 10,000 times. <br />  <br />  <br /> resilience aspect, directly compared e-MDSets sets                          context crucial elucidating possible roles <br /> of essential protein hubs equal size. Notably,                     certain genes pathway caus- <br /> the deletion e-MDSet proteins higher disruptive ef-                        ing slight perturbations. <br /> fect underlying networks organisms the <br /> deletion essential hub proteins alone. Taken our <br /> results demonstrate topological central relevance                        Methods <br /> proteins involved MDSets essen-                        Determination minimum dominating set (MDSet) <br /> tial protein interaction networks, characteristc                   We defined set S &#226;&#352;&#8224; V nodes network G = (V, E) <br /> just matter highest connectivity.                                              minimum dominating set node v &#226;&#710;&#710; V ei- <br />    Most research protein interaction networks seeks                            ther element S adjacent element S <br /> to better understand disease mechanism evolution-                              (inset, Figure 1A). In binary integer-programming <br /> ary development question. As data collecting tech-                              problem assigned binary variable xv =1 a <br /> niques sophisticated, ask                                   protein v &#226;&#710;&#710; V participates interactions E pro- <br /> intricate questions data collected. We con-                            tein interaction network G = (V, E) element the <br /> clude identification MDSet proteins                             MDSet, xv =0 otherwise. The smallest set MDSet <br />  Khuri Wuchty BMC Bioinformatics (2015) 16:109                                                                                  Page 7 8 <br />  <br />  <br />  <br />  <br />                               X <br /> nodes obtained min v&#226;&#710;&#710;V xv , subject con-           enrichment essential proteins Ni,&#226;&#8240;&#165; k was <br />              X <br /> straint xv &#195;&#190; w&#226;&#710;&#710;&#206;&#8220; &#195;&#176;v&#195;&#382; xw &#226;&#8240;&#165;1 &#206;&#8220;(v) set            corresponding random number essential pro- <br />                                                                 teins N&#226;&#8240;&#165; k proteins corresponding bin. <br /> interaction partners protein v. Since domination <br />                                                                 After averaging Ei 10,000 randomizations Ei &gt;1 <br /> problem graphs NP-complete algorithm ne- <br />                                                                 pointed enrichment vice versa, Ei ~ 1 <br /> cessarily exists allows determination a <br />                                                                 indicated random process [40]. <br /> minimum dominating set arbitrary graphs poly- <br /> nomial time [29]. We utilized branch-and-bound al- <br /> gorithm [30] implemented library lpSolve           Bottleneck proteins <br /> R programming language solve binary integer-             As global measure nodes centrality, calculated <br /> programming problem.                                            betweenness centrality, indicating proteins appear- <br />                                                                 ance shortest paths network. In <br />                                                                 particular, defined betweeness centrality cB pro- <br /> Protein-protein interactions E. coli, S. cerevisiae H.                      P <br /> sapeins                                                         tein v cB &#195;&#176;v&#195;&#382; &#194;&#188; s&#226;&#8240;&#160;t&#226;&#8240;&#160;v&#226;&#710;&#710;V &#207;&#402; st&#207;&#402; &#195;&#176;stv&#195;&#382; , &#207;&#402;st num- <br /> We collected 1,938 interactions 1,203 proteins          ber shortest paths proteins s t &#207;&#402;st <br /> that experimentally determined using yeast-two-          (v) number shortest paths running through <br /> hybrid approach E. coli [19]. As S. cerevisiae        node v. We defined set bottleneck nodes top <br /> collected 6,225 high-quality interactions 2,640         20% interactions highest node betweeness cen- <br /> proteins entirely determined large-scale           trality [23]. <br /> yeast-two hybrid approaches HINT database <br /> [20], including [2,31,32]. Furthermore, assembled          Functional classes <br /> network 17,523 high-quality interactions H. sapiens       Proteins grouped according broad functional <br /> between 5,926 proteins HINT database [20].             classes defined clusters orthologous <br /> Specifically, set protein interactions en-     groups (COGs) [27,28]. COGs provide consistent clas- <br /> tirely determined large-scale yeast-two hybrid ap-           sification bacterial eukaryotic species based on <br /> proaches, including [33-36].                                    orthologous groups. <br />  <br />                                                                 Competing interests <br /> Protein complexes E. coli, S. cerevisiae H. sapiens      The authors declare competing interests. <br /> We utilized set 517 protein complexes E. coli from <br /> a co-affinity purification study followed mass      Authors&#226;&#8364;&#8482; contributions <br />                                                                 SW designed concept, SK SW analyzed data wrote the <br /> spectrometry analyses [24]. As S. cerevisiae, utilized   manuscript. All authors read approved final manuscript. <br /> 430 protein complexes compiled [25], including the <br /> SGD Macromolecular Complex GO standard [37],                Acknowledgements <br /> CYC2008 protein complex catalogue [38] set             This study supported start-up funds University Miami. <br /> manually curated protein complexes. Furthermore, uti-        Received: 18 November 2014 Accepted: 13 March 2015 <br /> lized 1,843 protein complexes H. sapiens the <br /> CORUM database [26]. <br />                                                                 References <br />                                                                 1. Jeong H, Mason SP, Barabasi AL, Oltvai ZN. Lethality centrality in <br /> Protein complex participation coefficient                           protein networks. Nature. 2001;411(6833):41&#226;&#8364;&#8220;2. <br /> For protein protein com-      2. Yu H, Braun P, Yildirim MA, Lemmens I, Venkatesan K, Sahalie J, et al. <br />                                                                     High-quality binary protein interaction map yeast interactome <br /> plex, defined protein complex participation coef- <br />                                       &quot;      #2                     network. Science. 2008;322(5898):104&#226;&#8364;&#8220;10. <br />                                PN       ni;s                    3. Coulomb S, Bauer M, Bernard D, Marsolier-Kergoat MC. Gene essentiality <br /> ficient protein P &#194;&#188; s&#194;&#188;1 N           ni,s       topology protein interaction networks. Proc Biol Sci/Roy Soc. <br />                                        &#226;&#710;&#8216;ni;s                        2005;272(1573):1721&#226;&#8364;&#8220;5. <br />                                            s&#194;&#188;1 <br />                                                                 4. Chin CS, Samanta MP. Global snapshot protein interaction network-a <br /> the number links protein proteins complex            percolation based approach. Bioinformatics. 2003;19(18):2413&#226;&#8364;&#8220;9. <br /> s N total complexes. If protein predominantly          5. Li M, Zheng R, Zhang H, Wang J, Pan Y. Effective identification essential <br /> interacts partners complex, P tends             proteins based priori knowledge, network topology gene <br />                                                                     expressions. Methods. 2014;67(3):325&#226;&#8364;&#8220;33. <br /> 1 vice versa [39].                                          6. Estrada E. Virtual identification essential proteins protein <br />                                                                     interaction network yeast. Proteomics. 2006;6(1):35&#226;&#8364;&#8220;40. <br />                                                                 7. Li M, Wang J, Chen X, Wang H, Pan Y. A local average connectivity-based <br /> Enrichment analysis function degree <br />                                                                     method identifying essential proteins network level. Comput <br /> We binned proteins groups N&#226;&#8240;&#165; k protein                Biol Chem. 2011;35(3):143&#226;&#8364;&#8220;50. <br /> had k interactions calculated corre-           8. Missiuro PV, Liu K, Zou L, Ross BC, Zhao G, Liu JS, et al. Information flow <br /> sponding number essential proteins i, Ni,&#226;&#8240;&#165; k. Ran-               analysis interactome networks. PLoS Comput Biol. 2009;5(4):e1000350. <br />                                                                 9. Li M, Wang X, Wu F-X, Pan Y. A topology based method identifying essential <br />                                                   N <br /> domly picking essential genes defined E i;&#226;&#8240;&#165;k &#194;&#188; N i;&#226;&#8240;&#165;k <br />                                                     r            proteins PPI networks. IEEE/ACM Trans Comput Biol Bioinform. 2014;1:1. <br />                                                          i;&#226;&#8240;&#165;k <br />  Khuri Wuchty BMC Bioinformatics (2015) 16:109                                                                                                           Page 8 8 <br />  <br />  <br />  <br />  <br /> 10. Wuchty S, Almaas E. Peeling yeast protein network. Proteomics.                   36. Yu H, Tardivo L, Tam S, Weiner E, Gebreab F, Fan C, et al. Next-generation <br />     2005;5(2):444&#226;&#8364;&#8220;9.                                                                         sequencing generate interactome datasets. Nat Methods. 2011;8(6):478&#226;&#8364;&#8220;80. <br /> 11. Zotenko E, Mestre J, O&#226;&#8364;&#8482;Leary DP, Przytycka TM. Why hubs yeast              37. Cherry JM, Hong EL, Amundsen C, Balakrishnan R, Binkley G, Chan ET, et al. <br />     protein interaction network tend essential: reexamining                        Saccharomyces Genome Database: genomics resource budding <br />     connection network topology essentiality. PLoS Comput                    yeast. Nucleic Acids Res. 2012;40(Database issue):D700&#226;&#8364;&#8220;5. <br />     Biol. 2008;4(8):e1000140.                                                            38. Pu S, Wong J, Turner B, Cho E, Wodak SJ. Up-to-date catalogues yeast <br /> 12. Song J, Singh M. From hub proteins hub modules: relationship                      protein complexes. Nucleic Acids Res. 2009;37(3):825&#226;&#8364;&#8220;31. <br />     essentiality centrality yeast interactome different            39. Wuchty S, Siwo G, Ferdig MT. Viral organization human proteins. PLoS <br />     scales organization. PLoS Comput Biol. 2013;9(2):e1002910.                            One. 2010;5(8):e11796. <br /> 13. Li A, Li J: Characters patterns communities networks. In.                  40. Wuchty S. Evolution topology yeast protein interaction network. <br />     arxiv:1301.2957v1; 2013. http://arxiv.org/abs/1301.2957                                  Genome Res. 2004;14:1310&#226;&#8364;&#8220;4. <br /> 14. Ren J, Wang J, Li M, Wang H, Liu B. Prediction Essential Proteins by <br />     Integration PPI Network Topology Protein Complexes Information, <br />     vol. 6674. 2011. <br /> 15. Liu YY, Slotine JJ, Barabasi AL. Controllability complex networks. Nature. <br />     2011;473(7346):167&#226;&#8364;&#8220;73. <br /> 16. Nacher J, Akutsu T. Dominating scale-free networks variable scaling <br />     exponent: heterogeneous networks difficult control. New J <br />     Phys. 2012;14. doi:10.1088/1367-2630/14/7/073005. <br /> 17. Wuchty S. Controllability protein interaction networks. Proc Natl Acad Sci <br />     USA. 2014;111(19):7156&#226;&#8364;&#8220;60. <br /> 18. Milenkovic T, Memisevic V, Bonato A, Przulj N. Dominating biological <br />     networks. PLoS One. 2011;6(8):e23016. <br /> 19. Rajagopala SV, Sikorski P, Kumar A, Mosca R, Vlasblom J, Arnold R, et al. The <br />     binary protein-protein interaction landscape Escherichia coli. Nat Biotechnol. <br />     2014;32(3):285&#226;&#8364;&#8220;90. <br /> 20. Das J, Yu H. HINT: High-quality protein interactomes applications <br />     understanding human disease. BMC Syst Biol. 2012;6:92. <br /> 21. Luo H, Lin Y, Gao F, Zhang CT, Zhang R. DEG 10, update database <br />     essential genes includes protein-coding genes noncoding <br />     genomic elements. Nucleic Acids Res. 2014;42(Database issue):D574&#226;&#8364;&#8220;80. <br /> 22. Chen WH, Minguez P, Lercher MJ, Bork P. OGEE: online gene essentiality <br />     database. Nucleic Acids Res. 2012;40(Database issue):D901&#226;&#8364;&#8220;6. <br /> 23. Yu H, Kim PM, Sprecher E, Trifonov V, Gerstein M. The importance of <br />     bottlenecks protein networks: correlation gene essentiality and <br />     expression dynamics. PLoS Comput Biol. 2007;3(4):e59. <br /> 24. Hu P, Janga SC, Babu M, Diaz-Mejia JJ, Butland G, Yang W, et al. Global functional <br />    <br /> </body></html>