<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Proc. Natl. Acad. Sci. USA <br /> Vol. 92, pp. 9786-9790, October 1995 <br /> Biochemistry <br />  <br />  <br /> Cloning, expression, function TFC5, gene encoding the <br /> B&quot; component Saccharomyces cerevisiae RNA polymerase <br /> III transcription factor TFIIIB <br />      (promoter complexes/initiation factors/protein-DNA interactions/multiprotein complex assembly/DNA footprinting) <br />  <br /> GEORGE A. KASSAVETIS*, STEPHANIE T. NGUYEN*, RYuJi KOBAYASHIt, ASHOK KuMAR*, E. PETER GEIDUSCHEK*, <br /> AND MARINA PISANO*t <br /> *Department Biology Center Molecular Genetics, University California San Diego, La Jolla, CA 92093-0634;   tCold Spring Harbor <br /> Laboratory, P.O. Box 100, Cold Spring Harbor, NY 11724 <br />  <br /> Contributed E. Peter Geiduschek, July 19, 1995 <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>         TFC5, unique essential gene encod-                       pCS6 (containing SNR6 gene coding U6 snRNA) have <br /> ing B&quot; component Saccharomyces cerevisiae RNA                          described (13, 14). The p539H6-derived PU6R de- <br /> polymerase III transcription factor (TF)IIIB cloned.                     scribed (ref. 14; S. K. Whitehall, G. A. K., and <br /> It encodes 594-amino acid protein (67,688 Da). Escherichia                      E.P.G., unpublished data). TFIIIC affinity-purified box <br /> coli-produced B&quot; used reconstitute entirely recom-                    B-DNA, Mono Q-purified pol III, recombinant yeast TBP <br /> binant TFIIIB fully functional TFIIIC-directed,                    prepared described (1, 7). His6-tagged Brf (10) was <br /> well TATA box-dependent, DNA binding transcription.                        purified Ni2+-chelating agarose cited purification <br /> The DNase I footprints entirely recombinant TFIIIB,                            recombinant B&quot; below. Greater purification achieved by <br /> composed B&quot;, 67-kDa Brf, TATA box-binding                              eliminating previously used step Mono S chromatography <br /> protein, TEIIIB reconstituted natural B&quot; indis-                      (7) and, instead, allowing Brf precipitate direct dialysis <br /> tinguishable. A truncated form B&quot; lacking 39 N-terminal                        Ni2+-chelating agarose Brf fraction 7 M urea into <br /> and 107 C-terminal amino acids functional tran-                       40 mM Tris-HCl, pH 8/100 mM NaCl/10% (vol/vol) glycerol <br /> scription.                                                                        MPLP (10 mM 2-mercaptoethanol/0.5 mM phenylmeth- <br />                                                                                   ylsulfonyl fluoride/1 ,ug leupeptin ml/1 ,g pepstatin <br /> The central transcription factor (TF) RNA polymerase (pol)                     ml). The Brf precipitate resuspended 6 M guanidine <br /> III TFIIIB: TFIIIB suffices recruit pol III                       hydrochloride renatured (15), using bovine serum albumin <br /> multiple rounds accurately initiated transcription (1). Yeast                  (BSA) diluent (16) 500 mM NaCl dilution buffer. <br /> TFIIIB binds avidly greatly varying DNA sequence up-                           The preparation Brf (Fig. 1, lane d), contains the <br /> stream transcriptional start sites pol 111-transcribed                  67-kDa full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> protein 49-kDa fragment (resulting <br /> genes requires assembly factor, TFIIIC, proper                         proteolysis internal translational initiation) ex- <br /> positioning (reviewed refs. 2-4). An exception                         tends amino acid V165 C-terminal His6 tag, was <br /> general rule provided yeast U6 snRNA&apos; gene (SNR6),                      15% active formation heparin-resistant TFIIIB-DNA <br /> <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> TATA box TFIIIB bind own. In fact,                            complexes (12). <br /> accurately initiating transcription SNR6 gene                          Conditions analysis transcription, DNase I protection, <br /> executed vitro TFIIIB pol III (5-7).                              gel mobility shift SUP4 SNR6 genes been <br />    Yeast TFIIIB comprises components (8): TATA                          described (7, 16). Quantities protein specified fmol of <br /> box-binding protein, TBP; TBP-interacting 67-kDa TFIIB-                       DNA-binding activity TFIIIC, fmol protein capable of <br /> related protein, Brf (gene BRFI/PCF4/TDS4) (9-11);                          forming heparin-resistant TFIIIB-DNA complexes Brf, <br /> chromatographically separable component named B&quot; (12).                            TBP, B&quot;, fmol pol III active specific transcription <br /> Active B&quot; previously isolated 90-kDa &quot;band&quot;                       (12, 13). <br /> denaturing (SDS) gels, renatured, shown reconstitute                          Purification B&quot; Component TFIIIB Micro- <br /> active TFIIIB. In experiments described below,                    Sequence Analysis. B&quot; extracted 6 M urea S-100 <br /> complete proof constitution TFIIIB cloning                       pellets derived 1600 g Saccharomyces cerevisiae BJ926 <br /> gene encoding B&quot; protein, expressing Escherichia coli,                  subjected chromatography Bio-Rex 70 described <br /> and using resulting protein reconstitute transcriptionally                 (8). Ninety percent material (70 mg protein) was <br /> active TFIIIB entirely recombinant constituents.                   loaded 60-ml hydroxylapatite column (Bio-Gel HTP; <br /> Wholly recombinant TFIIIB suffices precise binding                     Bio-Rad) equilibrated 100 mM sodium phosphate, pH <br /> upstream promoter yeast SNR6 gene, accurate                            7.8/20% (vol/vol) glycerol/0.01% (vol/vol) Tween 20 and <br /> recruitment highly purified TFIIIC yeast tRNA gene                        MPLP, washed 60 ml buffer followed by <br /> promoter, accurate initiation U6 snRNA tRNA                        sequential 120-ml washes buffer 135, 160, 185, <br /> synthesis yeast pol III.&#194;&#167;                                                      215, 500 mM place 100 mM sodium phosphate (pH <br />                                                                                   7.8). B&quot; TP activity eluted end 160 mM and <br />                                                                                   185 mM sodium phosphate step. Eighty per- <br />              MATERIAL AND METHODS                                                 cent material (3.6 mg protein) concentrated <br />   DNA Proteins. Plasmids pTZ2 (containing SUP4                            5-fold (Centriprep 30; Amicon), diluted buffer D [40 mM <br /> tRNATyr G62 -&gt; C promoter-up <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span>mutation</span>), pLNG56 (the same <br /> gene C56 -&gt; G promoter-down mutation), pLNWT                             Abbreviations: TF, transcription factor; pol, RNA polymerase; BSA, <br /> (containing wild-type SUP4 gene; similar pTZ1),                        bovine serum albumin; RACE, rapid amplification cDNA ends; <br />                                                                                   TBP, TATA box-binding protein. <br />                                                                                   tPresent address: Consiglio Nazionale delle Ricerche, Istituto Di <br /> The publication costs article defrayed page charge         Genetica Molecolare, Alghero (Sassari), Italy. <br /> payment. This article marked &quot;advertisement&quot; inL         &#194;&#167;The sequence reported paper deposited the <br /> accordance 18 U.S.C. &#194;&#167;1734 solely indicate fact.                      GenBank data <span id='am-83' about='xsp:base' typeof='owl:Thing'>base</span> (accession no. U31819). <br />                                                                            9786 <br />            Biochemistry: Kassavetis et al.                                                     Proc.   Natli. Acad.   Sci. USA 92   (1995)   9787 <br />  <br />  <br />  <br />                               b   c   d                                    bands estimated Coomassie stain <br />                                                                              density, using dilution series BSA gel a <br />                                                                              standard curve. Since 31 Zg 90-kDa polypeptide (340 pmol) <br />                                              -109                            yielded 170 pmol active B&quot;, concluded this <br />                                                                              material desired B&quot; polypeptide(s). <br />                                             &apos;&apos; Brf                              The protein major &apos;90-kDa band weaker <br />                                              -70 <br />                   AB&quot;-&apos;                                                      (shadow) band separately isolated microse- <br />                                                (ABtf)                        quencing: 406 Zg protein Mono S B&quot; fraction was <br />                                                                              precipitated acetone, resuspended lx sample buffer <br />                                              -44                             (18), separated SDS/PAGE, stained 0.05% Coo- <br />                                                                              massie brilliant blue G. Bands corresponding major and <br />                                                                              minor 90-kDa proteins precisely excised, soaked in <br />                                                                              HPLC-grade water, stored -70&#194;&#176;C. An in-gel digestion <br />                                                                              method (19) used modification obtain internal <br />                                              -28                             amino acid sequence B&quot;. Individual gel slices digested <br />    FIG. 1. Polypeptide composition B&quot; fractions Brf. Eleven        Achromobacter protease I (lysylendopeptidase), the <br /> micrograms Mono S-purified B&quot; (lane a; dot right        resulting peptides separated reverse-phase HPLC <br /> lane identifies material subjected microsequencing; slower        (Vydac C18 column: 1.0 x 250 mm, 10 p-im, 300 A) analysis of <br /> migrating minor shadow band resolved major B&quot;                amino acid sequence (Applied Biosystems sequencer, models <br /> polypeptide gel), 0.9 ,jg His6-tagged recombinant B&quot; (lane        470 477, on-line HPLC analysis phenylthiohydan- <br /> b), 1.2 Atg His6-tagged truncated B&quot; (AB&apos;; lane c), -0.5 ,ug       toin-amino acids). The reverse-phase chromatographic pro- <br /> purified His6-tagged Brf (lane d; full-length Brf fragment           files peptides generated digestion major minor <br /> referred text identified right) analyzed      90-kDa protein lysylendopeptidase nearly identical <br /> 10% polyacrylamide/SDS gel stained Coomassie brilliant <br /> blue R.                                                                      (one additional peptides possibly present in <br />                                                                              minor 90-kDa protein). <br /> sodium Hepes, pH 7.8/7 mM MgCl2/20% (vol/vol) glycerol/                         Identification Cloning TFC5 Gene. Degenerate <br /> 0.01% (vol/vol) Tween 20 MPLP] conductivity                    inosine-containing oligodeoxynudt6otides based peptide <br /> buffer D plus 150 mM NaCl, loaded Mono S (Pharmacia,                    sequence synthesized (Operon Technologies, Alameda, <br /> HR 5/5 column) equilibrated buffer D plus 150 mM NaCl,                    CA) sense antisense orientation amplification of <br /> and eluted 25-ml 150-600 mM NaCl gradient buffer                   genomic DNA fragments coding portions B&quot; gene by <br /> D. B&quot; activity eluted 290 350 mM NaCl; 2 mg                   PCR (Table 1). Sense antisense primer pairs (100 pmol <br /> total protein contained &quot;170 pmol B&quot; measured forma-                   each) combined 10 ng yeast genomic DNA and <br /> tion heparin-resistant TFIIIB-DNA complexes excess                   20 AM dNTPs touchdown PCR amplification (20). Single- <br /> recombinant Brf TBP gel retardation assays (8).                       primer control reactions (21) proved useful eliminating <br />    Substantial purification Mono S-purified B&quot; (Fig. 1, lane              incorrect products. Each sense antisense primer pair <br /> a) achieved chromatography poly(dA-dT)-poly(dA-                    capable correctly amplifying portion B&quot; gene did so. <br /> dT) coupled CNBr-activated Sepharose (17)                   The resulting map PCR products confirmed gener- <br /> polypeptide band 67- 100-kDa range (data shown                 ating predicted internal PCR products fromh larger, gel-isolated <br /> but marked Fig. 1, lane a) coeluting B&quot; activity. The                PCR products (22) identifying additional already- <br /> identity &quot;90-kDa polypeptide B&quot; confirmed                  determined peptide sequence DNA sequence. <br /> subjecting 7 ,ug protein Mono S-derived B&quot; fraction                 Confirmed PCR products derived primer combina- <br /> to SDS/PAGE. The material stained Coomassie brilliant                   tions 2 + 7, 2 + 8, 4 + 8 wete labeled nick-translation <br /> blue G precisely excised, eluted, renatured de-                   (23) (4-6 x 108 cpm/yg DNA) sequentially used to <br /> scribed (8) slices similar size taken                probe mapping membranes- S. cerevisiae (prime clone grid <br /> below presumptive band B&quot; protein. B&quot; activity                    filters; ATCC 77284) colony blots S. cerevisiae <br /> observed gel slices: presumptive B&quot; polypeptide               genomic plasmid library [ATCC 77162, kindly provided by <br /> a weak (shadow) band directly it. The quantity protein              J. L. Cereghino S. Emr, University California San <br />                      Table 1. Oligonucleotides <br />                                               Peptide <br />                      No. Location*           sequence         Dir.t                      5&apos; -&gt;3&apos; sequence* <br />                        1       33-39        TPQLFIP             S      ACICCICARYT-ITTYATHCC <br />                        2     218-224       PPTAMTD              S      CCICCIACIGCNATGACNGA <br />                        3     218-224       PPTAMTD              A      TCIGTCATIGCIGTNGGNGG <br />                        4     396-403 KVDENPFA                   S      AAIGTIGAIGARAAYCCNTTYGC <br />                        5     396-402        KVDENPF             A      AAIGGITTYTCRTCNACYTT <br />                        6     406-412       YNYGSYG              S      TAIAAYTAYGGITCNTAYGG <br />                        7     406-413 YNYGSYGR                   A      CTICCRTAIGAICCRTARTTRTA <br />                        8     490-496        KNIGTVA             S      GCIACIGTDCCDATRAAYTT&#194;&#167; <br />                        9         N               -S                    GGTATCCATGGGTAGTATTGTTAATAAAAGTG <br />                       10         C-                             A      AATGTCT7CGAGTTGATCAATCTCAGG <br />                       11      N (40)             -S                    TTGTCCATGGCCGAAAGTAAAGAAATAGA <br />                       12      C (487)            -A                    TCTTCITCGAGTTCGCAGCAATACTCAT <br />                      *Amino~ acid positions Fig. 2; oligonucleotides 9-12 used generate expression plasmids the <br />                       complete truncated B&quot; proteins. <br />                      t(S)ense (A)ntisense orientation oligonucleotide. <br />                      tI: inosine; R: purine; Y: pyrimidine; H: A, C, T; D: A, G, T; N: A, G, C, T. <br />                      &#194;&#167;The. underlined A residues T residues, error specification peptide <br />                       microsequencing. <br />  9788      Biochemistry: Kassavetis et al.                                              Proc. Natl. Acad. Sci. USA 92    (1995) <br />                                                                         rose (Qiagen) 25 g wet weight cells denaturing <br /> MSSIVNKSGTRFAPKVRQRRAATGGTPTPKPRTPQLFIPESKEIEEDNSD-50                   conditions, yielding -2.6 mg -6 mg full-length and <br /> NDKGVDENETAIVEKPSLVGERSLEGFTLTGTNGHDNEIGDEGPIDASTQ-100                  truncated B&quot;, respectively (Fig. 1, lanes b c). B&quot; was <br />                                                                         renatured Ni2+-chelating agarose elution buffer <br /> NPKADVIEDNVTLKPAPLQTHRDQKVPRSSRLASLSKDNESRPSFKPSFL-150                  containing 7 M urea (adjusted pH 8 Tris base soon after <br />                                                                         elution) rapid addition 100 volumes BSA diluent (16). <br /> DSSSNSNGTARRLSTISNKLPKKIRLGSITSNDMNLKTFiRHRVLGKPSS-200                  The activity recombinant, full-length B&quot; determined by <br />                                                                         double-reciprocal plots formation heparin-resistant <br /> AKKPAGAHRISIVSKISPPTArMTDSLDRNEFSSETSTSREADENENYVIS-250                 TFIIIB-tDNA complexes fixed quantity B&quot; (estimat- <br />                                                                         ed relative BSA standard) increasing concentrations <br /> KVKDIPKKVRDGESAKYFIDEENFTMAELCKPNFPIGQISENFEKSKMAK-3 00                 TFIIIC-Brf-TBP-tDNA complexes. Extrapolation the <br /> KAKLEKRHLRELRMRARQEFKPLHSLTKEEQEEEEEKRKEERDKLLtAD-350                   least-squares best-fit linear plot (r2 = 0.9996) infinite DNA <br />            *                         II <br />                                                                         substrate indicated =100% active molecules. The specific <br /> IPESDRKAHTAIQLKLNPDGTMAIDEETMVVDRHKNASIENEYKEKVDEN-400                  activity truncated version B&quot;, quantified, <br />                                                                         appears comparable. <br /> PFANLYNYGSYGRGSYTDPWTVEEMIKFYKALSMWGTDFNLISQLYPYRS-450 <br />                                              A <br /> RKQVKAKFVNEEKKRPILIELALRSKLPPNFDEYCCEIKKNIGTVADFNE-500                                             RESULTS <br />            .1 <br /> KLIELQNEHKHHMKEIEEAKNTAKEEDQTAQRLNDANLNKKGSGGIMTND-550 <br />                                                                            Identification Isolation TFC5 Gene. The B&quot; <br />                                                                         component TFIIIB purified 6 M urea-extracted <br /> LKVYlTEVVIGTIDDLKRKKLKERNNDDNEDNEGSEEEPEIDQl )-594                      S-100 cellular debris basis ability reconstitute <br />                                                                         active TFIIIB combination recombinant TBP Brf. <br />    FIG. 2. Amino acid sequence B&quot; component TRIIIB.           DNA-protein photocrosslinking (28) renaturation B&quot; <br /> Peptide sequences obtained microsequencing Mono S-purified B&quot;        TF activity denaturing gels (8) identified B&quot; as <br /> are underlined (dots placed unassigned uncertain and,   consisting =90-kDa polypeptide(s). The pu- <br /> it turned out, incorrectly assigned residues). The N C termini   rification B&quot; precise identification corresponding <br /> B&quot; sequence remaining truncated form B&quot; (residues 40      polypeptide band (Fig. 1, lane a) described Material and <br /> 487) designated line. <br />                          A <br />                                                                         Methods. In-gel digestion lysylendopeptidase micro- <br /> Diego (UCSD)] specified (24). Hybridization             sequencing generated subset peptide sequences (under- <br /> probes prime clone grid filters identified cosmid c8487          lined Fig. 2) used design degenerate oligonu- <br /> (ATCC 71191) containing gene coding B&quot;. Adjacent             cleotides (Table 1, oligonucleotides 1-8) PCR amplifica- <br /> and partially overlapping A clones 7151 2713 failed              tion genomic DNA fragments encoding B&quot;. Three <br /> hybridize probes, limiting location B&quot;        confirmed amplification products (see Material Methods) <br /> gene kbp 534.8 553.1 physical map of <br />                                                                         sequentially used probe phage A cosmid prime <br /> chromosome XIV. Three clones B&quot; gene                   clone grid filters covering 95% S. cerevisiae genome. All <br /> obtained screening colony blots genomic plasmid               probes identified presence gene encoding B&quot;, <br /> library. The presence gene cosmid c8487 the <br />                                                                         TFCS [the fifth transcription factor gene class C (pol III) <br /> three library-derived plasmids confirmed PCR ampli- <br />                                                                         transcription], cosmid c8487, containing centromere- <br /> fication B&quot; gene sequence sources. The gene <br />                                                                         proximal sequence chromosome XIV. The sequence of <br />                                                                         2445-bp region cosmid c8487 contains open reading <br /> coding B&quot; sequenced strands                  frame deduced 594-amino acid sequence (67,688 Da) <br /> upstream (507 bp) downstream (157 bp) flanking se-                  contains 14 peptide sequences derived B&quot;. <br /> quence, continuing strand 220 bp upstream               The disparity apparent sequence-deduced <br /> and 450 bp downstream. Mapping 5&apos; 3&apos; ends                 size B&quot; (90 vs. 68 kDa) led examine possibility that <br /> mRNA coding B&quot; utilized 5&apos; 3&apos; RACE systems (RACE:               gene coding B&quot; contains intron. However, analysis <br /> rapid atnplification cDNA ends; GIBCO/BRL) se-                   3&apos; 5&apos; ends mRNA coding B&quot; RACE and <br /> quencing amplified products using total yeast RNA gen-           sequencing products placed poly(A) sites -52 =106 <br /> erously provided G. Cereghino I. E. Scheffler (UCSD).            nt downstream ochre stop codon TFCS open <br />    Disruption B&quot; gene followed standard procedures (25)          reading frame 5&apos; end -39 nt upstream initiation <br /> with HindlIl fragment (<span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> pFL1, ref. 26) containing             codon, evidence splice junctions (data shown). <br /> URA3 gene replacing HindIII fragment TFCS encoding                 Comparison amino acid sequence B&quot; the <br /> amino acids 170-366 (Fig. 2) pET21d expression clone             nonredundant combined data base National Center for <br /> described introduction Xba I-Xho I fragment              Biotechnology Information (BLASTP, May 1995) (29) revealed <br /> of disrupted B&quot; gene diploid S. cerevisiae strain             significant homologies outside glutamic acid-rich, <br /> MY4298 (ura3, leu2, his3) (27). MY4298 pFL1 kindly             highly charged region residues 329-357, homology <br /> provided M. Nickas M. Yaffe (UCSD).                              nucleolin ORF YKL202W S. cerevisiae <br />    TFCS Gene Expression E. coli Purification Re-              chromosome XIV noted. We did discern known <br /> combinant B&quot;. The B&quot; coding sequence PCR-amplified                  sequence motifs B&quot; (PROsmrE, May 1995) (30) the <br /> from cosmid c8487 high-fidelity Pfu DNA polymerase             exception potential signals glycosylation, phosphoryla- <br /> (Stratagene), using oligonucleotides 9 10 (Table 1)              tion, nuclear localization. TFCS essential cell via- <br /> generate Nco I site N-terminal methionine (mutating           bility: replacement B&quot; amino acid residues 170-366 the <br /> the adjacent S G) Xho I site ochre stop codon.         URA3 gene transformation ura- diploid yeast, fol- <br /> The amplified product cut Nco I Xho I                  lowed sporulation, produced asci, 110 dissected, <br /> inserted pET2ld, extending C-terminal amino acid               containing viable spores. <br /> sequence LEHHHHHH. A similar strategy, oligo-                    Reconstitution Entirely Recombinant TFIIIB. To confirm <br /> nucleotides 11 12 (Table 1), used clone truncated          TFCS gene coding B&quot;, B&quot; consists <br /> version B&quot;, comprising amino acids 40-487, MA            single 68-kDa polypeptide migrates anomalously on <br /> N terminus LEHHHHHH C terminus, pET2ld                  denaturing electrophoresis (apparent size, =90 kDa), made <br /> for expression E. coli BL21(DE3)pLysS. The resulting                 His6-tagged B&quot; E. coli purified Ni2+-chelating <br /> His-tagged B&quot; proteins purified Ni2+-chelating aga-             agarose denaturing conditions (see Material Meth- <br />            Biochemistry: Kassavetis et al.                                                         Proc. Natl. Acad. Sci. USA 92 (1995)            9789 <br />                                 b c d e f g h                                relatively impure yeast B&quot; [such TFIIIE (32)] affect these <br />                                                                                properties. Fig. 4 compares electrophoretic mobilities of <br />                                   nI   -    * <br />                                                       * rm                     TFIIIB-DNA complexes formed recombinant B&quot;, recom- <br />                                                                                binant TBP, recombinant Brf TFIIIC-dependent assem- <br />                                                                                bly SUP4 tRNA gene (lanes a-i) TFIIIC-independent <br />                                                                                assembly SNR6 gene (lanes j-n). On SUP4 gene, <br />                                                                                addition Brf TBP TFIIIC-DNA complexes (lane b) <br />                  SUP4-o-*                             *U6                      generated slightly slower migrating B&quot;-TFIIIC-DNA com- <br />                                                                                plex (lane c). Inclusion recombinant B&quot; generated ftu-ther <br />                                                                                retardation expected TFIIIB-TFIIIC-DNA complexes (lane <br />                      B&quot;: -y r rA- y r rA                                       d). Treatment heparin stripped TFIIIC generated a <br />                           tRNA     U6                                          TFIIIB-DNA complex (lane e) mobility corre- <br />   FIG. 3. Full-length truncated recombinant B&quot; competent               sponding complexes formed Mono S-purified B&quot; (lane g). <br /> for vitro transcription SUP4 tRNA SNR6 U6 small nuclear          Formation complexes SUP4 gene completely <br /> RNA (snRNA) genes. Two nanograms pTZ2 encoding                  TFIIIC-dependent (lane f). Truncated B&quot; generated slightly <br /> SUP4 tRNATYr gene (lanes a-d) pCS6 encoding SNR6 U6                     faster migrating TFIIIB complex stable heparin <br /> snRNA gene (lanes e-h) incubated 100 fmol recombinant              (lane h; appearance discrete heparin-resistant bands <br /> TBP, 27 fmol recombinant Brf, 10 fmol purified pol III, 29 fmol          reproducible resulted adventitious <br /> of purified TFIIIC (lanes a-d only), 5 finol Mono S-purified            proteolysis). Recombinant natural B&quot; likewise generated <br /> yeast B&quot; (lanes b f; designated &quot;y&quot; figure), 46 fmol of <br /> recombinant His6-tagged B&quot; (lanes c g; designated &quot;r&quot;), 4 ng         heparin-resistant complexes SNR6 gene identical <br /> truncated B&quot; (lanes d h; designated &quot;rA&quot;) 60 min 20&#194;&#176;C.              mobilities (lanes m k). The absence component <br /> Labeled ribonucleotides added 30 min, generating                 derived yeast reaction proves heparin <br /> multiple rounds transcription. The SUP4 U6 transcripts              resistance TFIIIB-DNA complex solely property <br /> identified; r.m., labeled DNA recovery marker.                                 constituent polypeptides. The presence hep- <br />                                                                                arin routinely diminishes electrophoretic mobility the <br /> ods). Recombinant B&quot; migrated anomalously &quot;90-kDa                        TFIIIB-DNA complex (compare lanes 1 m), possibly due <br /> protein SDS/PAGE (Fig. 1, lane b), indistinguishable                     interaction TFIIIB (see ref. 7, Fig. 3). <br /> mobility natural B&quot; (Fig. 1, lane a). A truncated variant <br /> of B&quot;, lacking 39 N-terminal 107 C-terminal amino acids                                          : h   r d   A   f   n   h   i <br /> (AB&quot;; Fig. 2; molecular mass, 52 kDa), purified E. coli as <br /> a C-terminally His6-tagged protein, migrated anomalously <br /> as -61-kDa protein SDS/PAGE (Fig. 1, lane c). <br />    Recombinant full-length B&quot; truncated AB&quot; able <br /> to replace natural yeast B&quot; TFIIIC-dependent tRNA gene <br /> transcription (Fig. 3, compare lanes c d lane b)                               +1   - <br />                                                                                                                                       I -40 <br /> for U6 snRNA gene transcription (Fig. 3, compare lanes g                                -7 <br /> h lane f); nonrecombinant protein U6 assay <br /> was highly purified pol III (31). <br />    The stability structure TFIIIB-DNA complexes formed <br /> with recombinant natural B&quot; compared deter- <br /> mine missing modification recombinant <br /> protein lack uncharacterized protein                                                                              -10 <br />                                                                                            -39- <br />                   b c d e f g h          j k        m n <br />  B-C-tDNA-, <br /> B&apos;-C-tDNA <br />                     i-4   .!-                                                                                                        -   +1 <br />    C-tDNA--:r <br />    B-tDN A-                         *0                       --c-   B-U6 DNA <br />  <br />                                                                                               B&quot;:     - y rrA - - y rrA - <br />                                                                                             gene:       SUP4       SNR6 <br />      tDNA- -                                                        U61DNA        FIG. 5. TFIIIB reconstituted natural recombinant B&quot; <br />                    -c cc c-cc                -                                 generates identical DNase I footprints SUP4 tRNAtYr SNR6 <br />                   - - B&apos; B&apos; B&apos;B&apos;BP Bl B&apos;BBB&apos; B&apos;                                (U6 snRNA) genes. Six femtomoles 298-bp fragment pLNWT <br />            B&quot;:    ---r r r y rA rA - y r r rA                                  containing wild-type SUP4 gene, 5&apos; end-labeled Xba I site <br />      heparin:                    +   + +    +         + +                      (in nontranscribed strand), incubated (lanes and <br />                                 SUP4             U6                            e) TFIIIC (57 fmol), TBP (100 fmol), Brf (34 fmol), either <br />                                                                                Mono S-purified yeast B&quot; (5 fmol, lane b), recombinant B&quot; (36 fmol, <br />   FIG. 4. Fully recombinant TFIIIB (Bn + Brf + TBP) forms                      lane c), truncated, recombinant AB&quot; (6 ng, lane d) 15 ,ul buffer <br /> heparin-resistant complexes SUP4 SNR6 genes. The                  60 min 20&#194;&#176;C. Heparin added (to 125 ,ug/ml) 8 min, <br /> components incubated 60 min 21&#194;&#176;C 15-,ul volume included 5.5          followed DNase I digestion 30 sec, 21&#194;&#176;C, reactions <br /> fmol &quot;300 bp end-labeled fragment pTZ2 containing                 terminated adding EDTA 20 mM. Free DNA heparin- <br /> SUP4 tRNAtyr gene (lanes a-i) 5.8 fmol 201-bp end-labeled              resistant TFIIIB-DNA complexes separated 4% native <br /> fragment pU6R containing SNR6 gene (lanes j-n)                  polyacrylamide gel partial DNase I digestion patterns were <br /> following proteins designated: TFIIIC (C; 58 fmol), recombinant             analyzed 10% polyacrylamide/8 M urea gel, shown. The <br /> yeast Brf TBP (B&apos;; 42 100 fmol, respectively), Mono S-purified         samples displayed lanes f-j identical lanes a-e, <br /> yeast B&quot; (y; 5 fmol), recombinant His6-tagged B&quot; (r; 46 fmol),             respectively, TFIIIC omitted, 4.4 fmol SNR6 <br /> truncated, recombinant AB&apos; (rA; 4 ng). Heparin added 125                gene probe (3&apos; end-labeled transcribed strand) noted Fig. <br /> ,ug/ml 5 min prior loading 4% native polyacrylamide gel,           4 replaced SUP4 tRNA gene probe. The start sites transcription <br /> as indicated. DNA complexes containing TFIIIC, TFIIIB, B&apos;              (+ 1) extent footprints (determined Phosphorlmager <br /> component TFIIIB (i.e., Brf + TBP) identified sides.             scan) indicated sides. <br />  9790      Biochemistry: Kassavetis et al.                                                   Proc. Natl. Acad. Sci. USA 92         (1995) <br />   The DNase I footprints SUP4 (Fig. 5, lanes b-d)                  known (35). We combinations candidate sites <br /> SNR6 complexes (lanes g-i) formed TFIIIB reconstituted                 deviating nucleotides. The sequence 5&apos; end <br /> with natural (Mono S-purified) B&quot; (lanes b g) recom-              5&apos; RACE product shows indication splicing the <br /> binant full-length B&quot; (lanes c h) indistinguishable. A           N-proximal end gene, 3&apos; RACE indicates poly(A) <br /> subtle diminution enhancement near start                    addition sites relatively close downstream end the <br /> site-proximal border noted complex recombi-                translational open reading frame. <br /> nant truncated AB&quot; (lanes d i) SUP4 (lane d). The <br /> absence yeast-derived component TFIIIB-SNR6                   We grateful C. Bowman S. Beardsley experimental <br /> complex shown lane h proves extent TFIIIB              assistance, P. Scott generous help tetrad dissections, <br /> footprint solely B&quot;, Brf, TBP.                              S. Emr generous hospitality,&quot; C. A. P. Joazeiro, S. K. <br />                                                                           Whitehall, G. Cereghino, J. Kadonaga discussions and <br />                                                                           advice, J. Kadonaga S. K. Whitehall critical reading of <br />                           DISCUSSION                                      manuscript. M. P. graduate fellow University of <br />                                                                           Sassari. Our research supported National Institute <br /> These experiments complete pr <br /> </body></html>