<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Proc. Natl. Acad. Sci. USA <br /> Vol. 91, pp. 4249-4253, May 1994 <br /> Biochemistry <br />  <br />  <br /> Ribosome recycling factor (ribosome releasing factor) essential <br /> for bacterial growth <br /> LASZLO JANOSI, ICHIRO SHIMIZU*, AND AKIRA KAJIt <br /> Department Microbiology, School Medicine, University Pennsylvania, Philadelphia, PA 19104 <br /> Communicated W. D. McElroy, January 18, 1994 <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>           Ribosome releasing factor, product thefn&apos;                     immediately adjacent translational termination signal <br /> gene Escherichia col, responsible dissociation                         (8, 13, 14). Despite compelling data, question re- <br /> ribosomes mRNA termination translation. It                        mained essentiality ofthis factor. For example, the <br /> functions &quot;recycle&quot; ribosomes renamed ribosome                            function RRF-i.e., release ribosomes mRNA <br /> recycling factor paper. An E. coil strain constructed                   nascent peptide released-can accom- <br /> (MC1061-2), carried frame-shifted firr chromo-                         plished vitro lowering Mg2+ concentration 3.5 <br /> some wild-type frr temperature-sensitive plasmid.                          mM (2). Although physiological Mg2+ concentration 4 <br /> MC1061-2 temperature-sensitive growth does                        mM (15), local lowering Mg2+ lead nonenzymatic <br /> segregate fir-carrying plasmid plasmid incom-                         dissociation ribosomes mRNA. Furthermore, vitro <br /> patibility pressure. In contrast, isogenic E. coil carrying wild-                   translation mRNA ribosomes without <br /> type frr chromosome mutated firr tempera-                         RRF gives amino acid incorporation, novo enzyme <br /> ture-sensitive plasmid temperature-sensitive growth                   synthesis. Only ribosomes recycled can <br /> and segregates plasmid incompatible plasmids. AU                           RRF effect observed. Thus, impossible <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br /> spontaneously formed thermoresistant colonies derived                          demonstrate absolute requirement factor vitro. <br /> MC1061-2 carried wild-type frr resided                              In paper exchanged chromosomal wild-typefrr <br /> bacterial chromosome re-exchange plasmid,                        frame-shiftedfrr carried temperature-sensitive plas- <br /> became temperature-resistant. These observations establish                          mid (16). The resulting E. coli strain temperature- <br /> that frr essential gene cell growth.                                      sensitive growth <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span>phenotype</span> indicating thatfrr essential <br />                                                                                     gene E. coli growth. This conclusion sup- <br /> Ribosome releasing factor (RRF) Escherichia coli                              ported absence wild-type frr-less segregants the <br /> protein responsible dissociation ribosomes                      plasmid incompatibility assays. <br /> mRNA termination translation (1). In final steps <br /> of protein biosynthesis termination-codon specific pep-                                       MATERIALS AND METHODS <br /> tide-chain release factor releases nascent complete peptide <br /> chain peptidyl tRNA. The resulting complex ribo-                               Bacterial Strains Plasmids. Vector plasmid pUC19 (17) <br /> some, mRNA, tRNA recycled                              GIBCO, tetracycline-resistance plasmid <br /> round protein synthesis. Thus, RRF working                          pSC101 (18) American Type Culture Collection, <br /> with GTP elongation factor (EF) G dissociates                              pMW119, replicon pSC101 ampicillin- <br /> complex tRNA, ribosome, mRNA (2-5). Therefore,                             resistance gene pBR322, Nippon Gene (Toyama, <br /> RRF distinct release (termination) factors, function-                       Japan). Plasmids containing frr, pRR1 (10), pRR2 (11) <br /> ally chemically. Although factors bear                          described. The chloramphenicol (CA) resis- <br /> name &quot;release,&quot; release (termination) factor releases peptide                       tance plasmid pMAK705, carrying temperature-sensitive <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> tRNA, RRF releases ribosomes mRNA.                                pSC101 replicon (16), E. coli MC1061 (19) provided <br /> In fact, RRF called &quot;ribosome recycling factor&quot;                              S. R. Kushner, University Georgia. <br /> because real function recycle ribosomes                        Media Chemical. L-broth (20) glucose was <br /> round protein synthesis. Therefore, RRF stands                         used liquid (LB) solid (LA) agar media. In cases <br /> for ribosome recycling factor. Other laboratories                         LA medium supplemented ampicillin 60 <br /> confirmed factor E. coli (6, 7). Although 70S                           pg/ml, tetracycline 20 ,ug/ml, CA 20 pg/ml obtain <br /> ribosomes released mRNA dissociated                           selective medium. Chemicals antibiotics from <br /> their subunits initiation factor (IF) 3 (8), RRF                         Sigma. Restriction endonucleases, T4-DNA ligase, and <br /> clearly distinct IF-3. The gene RRF,frr, located                       5-bromo-4-chloro-3-indolyl 13-D-galactoside (X-Gal) were <br /> in 4-min region E. coli chromosome, 1.1 kb                               Boehringer Mannheim. Single-stranded oligonucleo- <br /> downstream translational termination codon                          tides Wistar Institute (Philadelphia). <br /> tsf gene codes EF-Ts (9). Thefrr cloned,                         DNA Genetic Techniques. Plasmid DNAs pre- <br /> nucleotide sequence determined (10). The gene pre-                           pared described (21, 22). Agarose gel electrophoresis was <br /> ceded strong promoter (11) followed                                described (23, 24). Nucleotide sequence determina- <br /> rho-independent transcriptional termination sequence (10)A                          tions (25) carried using Sequenase V2.0 kit <br />    The importance RRF protein synthesis process                           (United States Biochemical). PCRs GeneAmp kit <br /> was indicated following vitro experiments: (i)                         (Perkin-Elmer/Cetus) presence Perfect <br /> caused 7-fold stimulation ofthe synthesis ofboth phage R17 <br /> coat protein phage T4 lysozyme (8), (ii) necessary                       Abbreviations: RRF, ribosome recycling (releasing) factor; frr (fac- <br /> for maximum synthesis ,-galactosidase (6, 12), (iii)                     tor ribosome release), gene RRF; CA, chloramphenicol; <br /> its omission led unwanted initiation protein synthesis                        EF, elongation factor; IF, initiation factor. <br />                                                                                     *Present address: Second Department Internal Medicine, School <br />                                                                                      Medicine, University Tokushima, Tokushima 770, Japan. <br /> The publication costs article defrayed page charge          tTo reprint requests addressed. <br /> payment. This article marked &quot;advertisement&quot;               tThe sequence reported paper deposited the <br /> in accordance 18 U.S.C. &#194;&#167;1734 solely indicate fact.                     GenBank data <span id='am-161' about='xsp:base' typeof='owl:Thing'>base</span> (accession no. L20141). <br />                                                                              4249 <br />                                   ~653 <br /> 4250     Biochemistry: Janosi et A                                                                                                     Proc. Natl. Acad. Sci. USA 91 (1994) <br />  <br /> Match (Stratagene) follows: 940C 2 min, 550C 1 min,                                     higher frequency control, MC1061 <br /> extension 740C 3 min, duration extension period                                      carrying pMAK705 frr insert, formed cointegrate <br /> was increased 1 sec cycle total 35 cycles.                                     colonies. Several cointegrates grown LB 32&#194;&#176;C <br /> As template, 5 ,ul intact E. coli cells (cells 1-ml                                        resolve plasmids bacterial chromosome. <br /> overnight cultures resuspended 50 ul sterile distilled                                       Two colonies isolated tested for <br /> water) used (26). Transformation CaCl2                                              growth presence absence CA 32&#194;&#176;C 43&#194;&#176;C. <br /> method (27). The plasmid incompatibility test                                          Among these, 105 colonies exhibited temperature-sensitive <br /> described (28). Exchange chromosomal wild-typefrr                                        phenotype presence CA, indicating plas- <br /> strain MC1061 mutagenizedfrr temperature-                                          mid resolved. Nine 105 resolved colonies were <br /> sensitive pMAK705 plasmid described (16).                                              sensitive elevated temperature, absence of <br />   In Vitro Mutagenesis thefnf Gene. Insertion cytidine                                     CA. In control experiment, resolved colonies <br /> base frr coding frame position 259 (counting                                      MC1061 harboring pMAK705 (the vector frr) <br /> the base translational initiation coding triplet)                                 showed CA-independent temperature-sensitive pheno- <br /> done described (29), using T7-Gen In Vitro Mutagen-                                         type. All plasmids 48 resolved MC1061-1 isolates <br /> esis Kit (United States Biochemical) 22-mer oligonu-                                         temperature-sensitive presence CA <br /> cleotide annealed nt 250-271 frr coding se-                                         identical parental pMR2M (with mutated frr). <br /> quence (10). The 893-bp Sma I-EcoRI insert pRR2 (11),                                           Plasmids resolved isolates were <br /> cloned vector supplied kit, used                                      temperature-sensitive, absence CA, showed <br /> the target DNA. The cytidine base insertion caused                                        loss Sty I recognition site. This result suggested that <br /> a + 1 frame-shift (behind thymidine 258-bp location)                                      isolates, designated MC1061-2, contained the <br /> also created Sty I restriction endonuclease recognition                                          mutated frr chromosome wild-type frr the <br /> sequence (5&apos;-CCTTGG-3&apos;) wild-typefrr (10).                                        plasmid gene replacement. To confirm loss one <br /> The base insertion confirmed Sty I digestion                                         Sly I recognition sites plasmid MC1061-2 <br /> nucleotide sequence determination. The mutagenized Sma                                             strains result gene replacement, PCR experiments <br /> I-EcoRI fragment finally inserted single HincII                                       illustrated Fig. 1 done. The nucleotide sequence in <br /> site temperature-sensitive pMAK705 plasmid (16),                                            plasmids upstream Sma I site (in -138-bp location) <br /> resulting pMR2M.                                                                                known (16), sequence chromosome <br />                                                                                                    upstream frr gene not. This sequence then <br />                                          RESULTS                                                   determined using pRR1, carries =1.1-kb stretch <br />                                                                                                    DNA upstream offrr chromosome (10). Fig. 1A <br />   Isolation Mutant Strains Carrying Frame-Shifted frr                                           shows primer A (devised basis newly <br /> Cistron Bacterial Chromosome. E. coli MC1061 cells                                          determined sequence) anneals specifically chromosomal <br /> were transformed temperature-sensitive pMR2M,                                             sequences, primer B (devised basis the <br /> carrying mutagenized frr, CA-resistant transformants                                           known vector plasmid sequence) anneals specifically plas- <br /> were isolated 32&#194;&#176;C. From these, designated                                            mid sequences. Primer C anneals DNA strand <br /> MC1061-1, plasmid-chromosome cointegrate colonies                                             downstream Sty I site created mutagenesis within <br /> isolated plating 43&#194;&#176;C presence CA. The                                             frr coding sequence common chromosomal <br /> frequency appearance cointegrates 29-fold                                          plasmidfrr. It clear Fig. 1B that, gene <br />                                                                                                                                                                   II <br />                                                                                                              I   C <br />                                                                                                              %4l II <br />                                                                                                              N        -                                  C., lI0 l.i                   I&apos;), i <br />                                                                                                                                                                                    II L1A <br />                                                                                                    V <br />  <br />                                                                                                    &apos;I            2 3 4 ;                           )      ,            YLl 1 12 &apos;3 14 <br />  A <br />                                          776 <br />                        424                                        5M2 <br />                        575                         v              201 <br />    1;                  650                        Xr      k 126 <br />                                                           I-            - <br />                    Sma I                 BstX I   (Sy I) B II1                          77:6 -&gt; <br />                                                                                         ;-71   &gt; <br />                                                                                                                                    &lt;      *7 50i <br />                                                                                                                                          f3 <br />                                                   RRF frame                                                                        1&lt;-   432 <br />                                                                              C                                                                                             4j 24                &lt;   .&apos;i <br />   -316   -193 -138               1        108     259    334                460   555 <br />  <br />                                                   RRF frame                       1 1 <br />                    Sma I                 Bs XI    (Sty I) Bgl U <br />            -               527                           -        126                                                                                  NiC f)(1:I-1 <br />            -               452                       _    r                                                  M( <br />                                                                                                              h -1 ol - 1 <br />                _           301       _   3                                                              1e       A    I        4 <br />  <br />  <br />                            _                              ;                 o                            C&apos; hz            Il <br />  <br />   FIG. 1. Evidence chromosomal frr mutated plasmid frr wild-type gene-exchanged E. coli MC1061-2. (A) <br /> Restriction enzyme map frr cistron upstream regions located bacterial chromosome (Upper) temperature-sensitive <br /> pMAK705 plasmid (Lower), deduced nucleotide sequence data (refs. 10 11, study). The broken line plasmid map <br /> indicates nucleotide sequence upstream Sma I site differs ofchromosomal DNA region. Numbers between <br /> the maps (vertical arrows) indicate locations restriction endonuclease sites, ends RRF coding region, primer annealing. <br /> Nucleotide 1 nucleotide initiation codon frr cistron. Numbers horizontal lines arrow ends denote distance <br /> between arrows base pairs. A (5&apos;-AGAAGGGACTTTAATCACGG-3&apos;), B (5&apos;-GTAAAACGACGGCCAGT-3&apos;), C (5&apos;- <br /> GATCGTCGTCTTCG-CTG-3&apos;) indicate single-stranded oligonucleotide primers PCR. The arrows primer boxes indicate <br /> the direction primer-extension Taq DNA polymerase. The Sty I site infrr parentheses site appears mutated, <br /> but wild-type, frr. (B) Restriction endonuclease analysis PCR products. Lanes: 3 12, molecular weight markers; 1, 6, 10, <br /> chromosomal DNA MC1061-1 (non-gene-exchanged); 4, 8, 13, plasmid DNA MC1061-1; 2, 7, 11, chromosomal DNA MC1061-2 <br /> (gene-exchanged); 5, 9, 14, plasmid DNA MC1061-2. Ch, chromosome; PI, plasmid. <br />              Biochemistry: Janosi et al.                                                                     Proc. Natl. Acad. Sci. USA 91 (1994)               4251 <br />  exchange occurred (MC1061-1), PCR product chro-                                            1 010                                         2.0 <br />  mosomalfrr digested Sty I, the <br />  plasmidfrr was. The reverse true pair the <br />  exchange (MC1061-2). Other restriction endonuclease sites <br />  remained same. Nucleotide sequence determination (data <br />  shown), using PCR products MC1061-1                                              109 <br />  MC1061-2, confirmed extra cytosine base <br />  259th bp location chromosomal                                                                                              0 <br /> frr MC1061-2 plasmidfrr MC1061-1.                                                                                                          Lfl <br />     Conditional Growth Inhibition Strains Carrying Frame-                                   E                                                         ao <br />  Shifted frr Cistron Bacterial Chromosome. Fig. 2B                                                                                                6 <br />  shows viable counts MC1061-2 (mutated frr the <br />  chromosome, wild-typefrr temperature-sensitive plas- <br />  mid) stop increasing 2-hr lag following temperature <br />  shift, CA (E). In contrast, MC1061-1 (wild-type                                        Il07 <br /> frr chromosome, mutated frr temperature- <br />  sensitive plasmid) clearly shows temperature- <br />  sensitive growth (Fig. 2A). Furthermore, MC1061-1, which <br /> grew 430C liquid culture form colonies in <br /> the presence CA (U). This result indicates plasmid                                                      Time (hours) <br /> loss 430C liquid culture occurred equally                                 FIG. 3. Continued increase cell mass MC1061-2, after <br /> MC1061-1 MC1061-2. Therefore, stoppage growth                                temperature shift up. The experimental conditions identical <br /> 430C MC1061-2 clearly presence mutated                           described Fig. 2. Growth frr-exchanged E. coli <br /> frr chromosome. This result establishes frr                             MC1061-2 strain followed measuring OD 540 nm. At the <br /> essential bacterial growth. In control experiments                         point indicated vertical arrow temperature shifted <br /> at 320C, strains behaved identically (V,V).                                       320C 430C (e) kept 320C (o). The viable cell count data <br />    Fig. 3 shows cell mass MC1061-2 did stop                            MC1061-2 Fig. 2 shown. Cfu, colony-forming units. <br /> increasing temperature shift up. However, mass                            vated temperature followed longer period time <br /> increase 430C 32TC. This result consis-                         starting cultivation various dilutions an <br /> tent assumption culture contains viable frr                          overnight starter culture. The gene-exchanged MC1061-2 <br /> diploid cells. Upon shifting temperature 430C, plasmid                          strain started growing prolonged time allow gene <br /> number increase, plasmids synthesized                              exchange back. As expected, dilute initial cell <br /> 320C remain intact (16, 30). The frr diploid cells mul-                           concentration, longer took emergence cells <br /> tiplying elevated temperature, producing plasmidless prog-                          capable growing 430C. <br /> eny, nonviable appear                                      Studies Temperature-Resistant Population That <br /> increase cell mass (OD increase).                                               Emerged Prolonged Cultivation Gene-Exchanged <br />    Because strain MC1061-2 arose result gene                             MC1061-2 Elevated Temperature. As indicated the <br /> exchange, expect mutated frr exchange                                 preceding section, prolonged cultivation MC1061-2 430C <br /> back plasmid. One emergence                                   resulted emergence temperature-resistant (Heatr) bac- <br /> re-exchange higher number cells                               teria. It -60% resistant to <br /> involved, shown Fig. 4. In experiment, growth                            CA (CArHeatr). The CA-sensitive bacteria did the <br /> MC1061-1 gene-exchanged MC1061-2 strains ele-                               plasmid. PCR analysis strains revealed they <br />            1 010                                              1 v -r ---~~~~~1 <br />                                                                                        carried chromosomal frr wild-type. As <br />                                            1      B           A ,I                                 2.0 <br />                                                                                  10 <br />  <br />  <br />  <br />  <br />                                            1j             ij <br />  <br />        \-- 1108 <br />                                                                                08           0 <br />  <br />  <br />                                                                                  07 <br />                                                                                               0 <br />  <br />            106                     -                                              06 <br />                    0   2   4   6       8   10 0       4   6      8       10 <br />  <br />                                Time (hours) <br />                                                                                                               5c                5   10   1E   2C   25 <br />    FIG. 2. Temperature-sensitive growth E. coli containing mu- <br /> tated frr chromosome wild-type frr temperature- <br /> sensitive plasmid. (A) E. coli MC1061-1 carrying wild-typefrr the <br /> chromosome mutated frr temperature-sensitive plasmid.                          FIG. 4. Effect inoculum size growth E. coli strains <br /> (B) E. coli MC1061-2 carrying wild-type frr temperature-                        intact frame-shiftedfrr chromosomes 43TC. E. coli <br /> sensitive plasmid mutated frr bacterial chromosome.                         MC1061-1, carrying frame-shiftedfrr plasmid wild-typefrr <br /> Bacteria grown LB antibiotics plated                      chromosomal location, MC1061-2, carrying frame-shiftedfrr <br /> LA plates CA 320C. v, Growth 320C plating                    chromosome wild-type frr plasmid, grown <br /> onto nonselective agar; v, growth 320C plating CA plates;                  overnight 32TC, diluted 1:100, 1:1000, 1:10,000, 1:100,000 <br /> a, temperature shift 320C 430C 1 hr plating                        fresh LB incubated 43TC. (A) MC1061-1. (B) <br /> nonselective agar; *, temperature shift plating CA                   MC1061-2. o, *, v, v, Growth started 1:100, 1:1000, <br /> plates. Cfu, colony-forming units.                                                     1:10,000, 1:100,000 dilutions, respectively. <br />  4252      Biochemistry: Janosi et al.                                                     Proc. Natl. Acad. Sci. USA 91 (1994) <br /> Table 1. Essentiality offrr overcomes incompatibility <br />                                                                                  Colonies incoming resident plasmids,t% <br />                                     Resident              Incoming            Selection             Selection for <br /> Line              Host*              plasmidt             plasmidt          resident plasmid          incoming plasmid            No selection <br />  1             MC1061            pMAK705                  pMW119                    0                          0                    0 (i50/r50) <br />  2                                                        pSC101                    0                          0                    0 (i94/r6) <br />  3             MC1061-2          pMR2                     pMW119                    0                        100                    0 (rWOS) <br />  4                                                        pSC101                    0                        100                    0 (rlOO) <br />  5             MC1061-1          pMR2M                    pMW119                    0                          0                    0 (i81/r19) <br />  6                                                        pSC101                    0                           0                   0 (i94/r6) <br />  7              MC1061           pMR2                     pMW119                    0                           0                   0 (i56/r44) <br />  8                                                        pSC101                    0                           0                   0 (i94/r6) <br />  9             MC1061-2           pMR2;pRR2               pSC101                    0                           0                   0 (i56/r44) <br /> 10             MC1061-2           pMR2;pRR2M              pSC101                    0                        100                    0 (rWO0) <br /> *All strains MC1061 derivatives. MC1061 MC1061-1 carry wild-typefrr chromosomes, MC1061-1 contains pMR2M <br />  plasmid mutated frr. MC1061-2 carries mutated frr chromosome wild-type frr pMR2 plasmid. <br /> tpMAK705, pMW119, pSC101 carry pSC101 replicon <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span> CA, ampicillin, tetracycline resistance, respectively. pMR2M <br />  pMAK705 containing 893-bp Sma I-EcoRI insert E. coli chromosome carrying promoter, frame-shifted coding sequence, <br />  rho-independent transcription terminator frr. pMR2 identical pMR2M, carries wild-type frr. pMR2 arose <br />  result gene exchange pMR2M chromosome. pRR2 pRR2M pUC19 derivatives coding ampicillin resistance <br />  carrying insert pMR2 pMR2M, respectively. <br /> tSixteen colonies tested experiment successive single-colony isolations. Bold-faced characters indicate alteration <br />  expected incompatibility results. Numbers parentheses indicate percentage colonies carrying incoming (i) resident <br />  (r) plasmids single-colony isolations absence selection. <br />  <br /> to CArHeatr strains emerged MC1061-2, plasmid                                               DISCUSSION <br /> DNA detected 32 strains examined. PCR                      On basis vitro biochemical studies, &gt;10 soluble <br /> analysis 16 strains revealed 8 carried            protein factors thought essential protein biosynthesis <br /> the mutated site plasmid wild-typefrr allele             aminoacyl tRNAs E. coli. Convincing genetic data <br /> the chromosome. The reverse true 8                      available factors, proving partic- <br /> strains examined. We conclude studies (i)              ipation initiation, elongation, termination steps of <br /> Heatr strains formed exchange mutated                  translational process indispensable E. coli growth. <br /> chromosomal frr wild-type plasmid frr,                  Thus, Laalami et al. (31) E. coli cells carrying null <br /> followed loss plasmid, (ii) CArHeatr             <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>mutation</span> infB determinant, coding IF-2, only <br /> is loss temperature-sensitive character               grow functional copy gene existed a <br /> replication plasmids. In words, strain                plasmid chromosome. Isolation thermosensitive <br /> emerged prolonged cultivation MC1061-2 43&#194;&#176;C                     E. coli altered infC (the gene coding IF-3), and <br /> contained wild-type frr.                                                   conversion sensitive phenotype thermostable one <br />    Additional Evidence That frr Is Essential Growth-Use                wild-type infC carrying F&apos;-factor (32) specialized <br /> of Incompatible Plasnids. In experiments described                  transducing recombinant A phage (33) proved IF-3 is <br /> lines 1 2 Table 1, CA resistance plasmid pMAK705                necessary growth. As EFs, insertional inactivation <br /> (the vector forfrr study) shown incompatible              tufA, tufB, (genes EF-Tu) bacterio- <br /> with ampicillin-resistance plasmid pMW119 (the                    phage Mu lethal complemented plasmid-born tufA <br /> replicon pSC101) tetracycline-resistance plasmid             (34). The thermosensitive phenotype altered EF-G (35, <br /> pSC101. From experiments shown lines 3 4,                       36) converted thermoresistant transducing <br /> despite incompatibility, strains clearly retained          appropriate bacteriophages harboring wild-type EF-G <br /> the resident CA-resistance plasmid pMR2 (contains wild-                    determinant (37). Finally, DNA insertion prfA, the <br /> type frr) host chromosome mutated frr                         gene coding release (termination) factor 1, pro- <br /> (MC1061-2). The control MC1061-1, carrying wild-typefrr                 moter appeared deprive E. coli capacity grow (38). <br /> its chromosome, expressed regular incompatibility, shown                The temperature-sensitive growth phenotype prfA <br /> in lines 5 6. The incompatibility masked MC1061-2               mutants E. coli converted thermoresistant the <br /> under selective condition incoming plasmid. This               wild-type prfA allele plasmid complemented defi- <br /> result abnormal nature pMR2                    cient function mutated gene (39). <br /> this plasmid readily segregated incoming incompat-                   We took advantage recently published method <br /> ible plasmids MC1061 background carried wild-type                  proving essentiality genes based allele exchange <br /> frr chromosome (lines 7 8). In addition,             chromosomal wild-type gene mutated <br /> due possible change MC1061-2 host functions,                 gene temperature-sensitive plasmid (16). Using this <br /> are essential replication pSC101 derivatives. This <br />                                                                            method, concluded RRF essential growth of <br />                                                                            E. coli. This conclusion based following findings: (i) <br /> conclusion because, shown line 9,                  Strain MC1061-2, carries frame-shift mutation chro- <br /> strain, expected incompatibility observed strain                mosomal frr, exist wild-type frr gene is <br /> carried additional compatible plasmid, pRR2, wild-                 present resident plasmid. We unable, any <br /> typefrr insert. Under conditions, compatible               means, isolate monoploid cells mutated frr the <br /> plasmid carried mutated frr (pRR2M), pMR2                     diploid strain E. coli carrying mutated wild-type frr. <br /> eliminated (line 10). These results establish absolute            (ii) Analysis spontaneous thermoresistant mutants that <br /> requirement offrr E. coli growth. An isolation tempera-                emerged prolonged incubation MC1061-2 43&#194;&#176;C <br /> ture-sensitive mutant frr enable perform                    indicated emerged re-exchanging the <br /> analysis type.                                                     chromosomal mutated frr wild-type frr plasmid <br />            Biochemistry: Janosi et A                                                 Proc. Natl. Acad. Sci. USA 91 (1994)          4253 <br /> or temperature-independent presence resident                 15. Noll, M. &amp; Noll, H. (1976) J. Mol. Biol. 105, 111-130. <br /> plasmid carrying wild-type frr. None thermoresistant            16. Hamilton, C. M., Aldea, M., Washburn, B. K., Babitzke, P. &amp; <br /> revertants contained monoploid mutatedfrr. When                   Kushner, S. R. (1989) J. Bacteriol. 171, 4617-4622. <br /> monoploids forfrr, gene wild-type. (iii)         17. Yanish-Perron, C., Vieira, J. &amp; Messing, J. (1985) Gene 33, <br /> We unable remove resident plasmid, carrying                 103-119. <br />                                                                        18. Cohen, S. N. &amp; Chang, A. C. Y. (1977) J. Bacteriol. 132, <br /> wild-type frr gene-exchanged E. coli, incompati-                 734-737. <br /> bility plasmid (40).                                                19. Casadaban, M. J. &amp; Cohen, S. N. (1980) J. Mol. Biol. 138, <br />    Among steps protein synthesis (initiation, elonga-               179-207. <br /> tion, termination, ribosome recycling), recycling step        20. Bertani, G. (1951) J. Bacteriol. 62, 293-300. <br /> appears closely related step-i.e., initiation. It is,     21. Birnboim, H. C. &amp; Doly, J. (1979) Nucleic Acids Res. 7, <br /> therefore, tempting speculate happens initia-                1513-1523. <br /> tion protein synthesis E. coli RRF suddenly         22. Humphreys, G. O., Willshaw, G. A. &amp; Anderson, E. S. (1975) <br />         <br /> </body></html>