<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal vol.8 no.6 pp. 1685 - 1693, 1989 <br />  <br />  <br /> Mutational analysis SEC4 suggests cyclical <br /> mechanism regulation vesicular traffic <br />  <br />  <br /> Nancy C.Walworth, Bruno Goud1, Alisa                                   form, protein resting inactive state. Exchange <br /> Kastan Kabcenell Peter J.Novick                                    GTP GDP results activation protein until <br />                                                                        GTP hydrolyzed. The intrinsic hydrolysis activity as <br /> Department Cell Biology, Yale University School Medicine,        exchange bound GDP new <br /> 333 Cedar Street, New Haven, CT 06510, USA <br />                                                                        molecule GTP processes affected by <br /> &apos;Present address: Department Immunology, Institut Pasteur, Paris,   interaction proteins (Trahey McCormick, <br /> France                                                                 1987; Neer Clapham, 1988). <br /> Communicated P.de Carnilli                                             Two basic mechanisms GTP-binding proteins <br />                                                                        perform functions described. The mammalian <br /> Mutant alleles SEC4, essential gene required                 G proteins, heterotrimeric protein complexes responsible for <br /> the final stage secretion yeast, generated             transduction extracellular signals, couple cell surface <br /> by vitro mutagenesis. Deletion cysteine                  receptors intracellular enzymes produce regulatory <br /> residues C terminus protein results                 molecules cell. In GTP bound state, the <br /> soluble non-functional protein, indicating              activated G protein stimulates effector molecule <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br /> residues required normal localization Sec4p                 increase intracellular level second messengers <br /> to secretory vesicles plasma membrane. A mutant                cAMP, Ca2+ inositol lipid products. Thus, G <br /> allele SEC4 generated mimic activated, trans-                 protein initiates amplification signal, beginning <br /> forming allele H-ras, predicted, does bind GTP.              cascade events leads cellular response <br /> The presence allele cells containing wild-type              (Gilman, 1987). <br /> SEC4 causes secretory defect accumulation                    An alternative mechanism action GTP-binding <br /> secretory vesicles. The results genetic studies indicate            protein ensure reaction occurs unidirectional <br /> that allele behaves dominant loss function                fashion. The bacterial protein synthesis factor EF-Tu <br /> mutant prevents wild-type protein                     mediates attachment subsequent release a <br /> functioning properly. We propose model Sec4p                macromolecular complex ribosome. The process <br /> cycles active inactive state order                protein synthesis stopped vitro presence of <br /> to mediate fusion vesicles plasma membrane.              non-hydrolyzable GTP analog, GTP--y-S. EF-Tu, trapped <br /> Key words: GTP binding protein/secretion/Sec4p/ras                     GTP bound state, change conformation and <br /> homolog/vesicular traffic                                              come ribosome allow round delivery <br />                                                                        occur. Thus, single round nucleotide binding and <br />                                                                        hydrolysis, single event (delivery) takes place (Kaziro, <br /> Introduction                                                           1978). <br />                                                                           We taken genetic approach determine either <br /> The SEC4 gene Saccharomyces cerevisiae encodes 24 kd              general mechanisms account the <br /> GTP-binding protein plays essential role final         regulation secretion Sec4p. Since large of <br /> stage secretion (Salminen Novick, 1987). Post-Golgi             information available structure GTP-binding <br /> secretory vesicles transported specific region         proteins function specific amino acids that <br /> plasma membrane called bud subsequently fuse              structure, undertaken mutational analysis Sec4p <br /> this membrane (Novick et al., 1980; Salminen Novick,               based well-characterized mutations proteins <br /> 1987). A recessive temperature-sensitive <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span>mutation</span> sec4              (Jurnak, 1985; LaCour et al., 1985; Vos et al., 1988). <br /> causes cells accumulate secretory vesicles restric-          Point mutations alter ability p21 H-ras bind <br /> tive temperature (Novick et al., 1980). The localization            hydrolyze GTP known confer <br /> the wild-type protein, Sec4p, cytoplasmic surface            protein capacity induce transformation of <br /> secretory vesicles consistent participating          mammalian cells expressed (Barbacid, <br /> regulation transport recognition secretory               1987). <br /> vesicles plasma membrane (Goud et al., 1988).                     <span id='am-1' about='protege:FROM' typeof='owl:Thing'>From</span> mutational studies yeast RAS proteins that <br />    GTP-binding proteins shown mediate diverse             regulate entry cells mitotic cell cycle GI, <br /> cellular processes. Members family proteins include         clear intact carboxy terminus required for <br /> factors involved protein synthesis elongation factor        proper localization normal function proteins <br /> Tu (EF-Tu), G proteins involved signal transduction,            (Fujiyama Tamanoi, 1986; Deschenes Broach, <br /> and product ras oncogene (see Kaziro, 1978;                 1987). The true oncogenic mammalian ras <br /> Barbacid, 1987; Gilman, 1987, reviews). These proteins             proteins, transforming ability attached <br /> share ability assume different conformations depending          inner surface plasma membrane, usual <br /> on GTP GDP bound                  location normal cellular version protein <br /> act regulatory switch. Generally, GDP bound                (Willumsen et al., 1984a,b). Yptlp, yeast ras-like protein <br />  <br />                                                                                                                                1685 <br />  N.C.Walworth et al. <br />  <br />  <br />  <br />  <br />                                                                           Fig. 2. Deletion C-terminal cysteines results soluble Sec4p. <br />                                                                           Lysates exponentially growing cells prepared osmotic lysis <br />                                                                           spheroplasts 0.8 M sorbitol followed 450 g centrifugation to <br />                                                                           remove unlysed cells (SI). SI spun 100 000 g yield <br />                                                                           membrane (P2) cytosolic fractions (S2). (A) NY678 sec4-8 <br />                                                                           strain (NY405) transformed pNB254 expresses the <br />                                                                           sec4-CCA gene. NY682 sec4-8 strain (NY405) transformed with <br />                                                                           pNBl70 expresses wild-type SEC4 gene. In cases the <br />                                                                           plasmid encoded protein distinguished endogenous <br />                                                                           protein, Sec4-8p. (B) NY637 SEC&apos; strain (NY15) transformed <br />                                                                           pNB286 expresses Sec4-11e133,CCA gene. NY639 a <br />                                                                           SEC+ strain (NY15) transformed pNBI41, integrating version <br />                                                                           pNBl39, expressing wild-type Sec4p. <br />  <br />                                                                           model membrane-associated Sec4p functions a <br />                                                                           cyclical fashion direct fusion post-Golgi secretory <br />                                                                           vesicles plasma membrane. <br /> Fig. 1. Documentation Sec4p expression levels various <br /> plasmid constructions. The Sec4p plasmids         Results <br /> shown varies depending (i) genomic DNA 5&apos; the <br /> initiator ATG included insert (ii) orientation      Deletion C-terminal cysteines prevents <br /> insert plasmid respect URA3 gene. On right, <br /> total yeast lysates prepared glass bead lysis method           membrane attachment <br /> transferred SDS-polyacrylamide gel nitrocellulose filter     To assess possible role membrane attachment the <br /> were incubated anti-Sec4p antibody alkaline phosphatase          C-terminal cysteine residues Sec4p, deleted <br /> conjugated secondary antibody. NY405, sec4-8 mutant strain,         site-specific mutagenesis (sec4-CCA, Materials <br /> transformed plasmid expressing wild-type SEC4 gene.         methods). Plasmid pNB254 (Figure 1), allows <br /> The protein synthesized plasmid (Sec4p) distinguished <br /> from endogenous protein (Sec4-8p) different      expression Sec4-CCAp higher wild-type <br /> mobilities. NY13 wild-type strain. In plasmid pNB227 SEC4        levels, introduced temperature-sensitive sec4-8 <br /> gene oriented 3&apos; end near URA3 - 150 bp 5&apos; non-        strain. This strain grow 37&#194;&#176;C unless rescued <br /> coding sequence present insert. Sec4p expressed low        functional copy SEC4. The sec4-CCA gene on <br /> level pNB227. Plasmid pNB170 size insert 227, <br /> but orientation insert reversed, protein         pNB254 rescue strain, suggesting encodes <br /> expressed high levels. Plasmid pNB139 expresses Sec4p level       non-functional protein. The plasmid wild-type cells <br /> which approximately equivalent Sec4p     effect cell growth, indicating allele is <br /> untransformed wild-type strain, NY13. pNB139               recessive wild-type allele. Overproduction the <br /> orientation pNBl70 -400 bp non-coding region         Sec4-CCAp GAL] promoter does comple- <br /> SEC4 5&apos; end. Plasmids 227, 170 139 centromere- <br /> containing plasmids. Various plasmids used report include   ment temperature-sensitive growth defect sec4-8, <br /> which integrating forms plasmids              GAL+ strain 37&#194;&#176;C (data shown). <br /> centromere integrating plasmids expressing mutant forms SEC4.         To confirm hypothesis Sec4-CCAp a <br />                                                                           functional protein, performed gene replacement <br /> that implicated function earlier stage               experiment makes sec4-CCA sole copy SEC4 in <br /> the secretory pathway (Segev et al., 1988), terminates                 cell. An integrating plasmid containing sec4-CCA was <br /> two cysteine residues essential proper                      introduced diploid strain chromosome <br /> functioning protein (Molenaar et al., 1988). Sec4p                complete wild-type copy gene other <br /> also terminates cysteines.                                       chromosome complete copy sec4-CCA <br />    We specific amino acid changes GTP-                   gene marked URA3 selectable marker. Upon <br /> binding domains Sec4p, C terminus,                sporulation dissection resulting tetrads viable <br /> an attempt alter biochemical activities study              spores Ura+ (data shown). This <br /> effects changes function protein vivo.           result demonstrates sec4-CCA gene sufficient <br /> Using mutants, hoped distinguish                  support growth copy sec4 cell. <br /> two mechanisms described. Like signal transducing G                      To determine intracellular location Sec4-CCAp, <br /> proteins, Sec4p serve amplify signal required              isogenic strains constructed plasmid pNB254 <br /> secretion regulating level second messenger                   (sec4-CCt, NY678) pNB170 (SEC4, NY682) was <br /> within cell. Alternatively, like EF-Tu, Sec4p                   introduced NY405. Lysates prepared osmotic <br /> function direct macromolecular structure specific               lysis spheroplasts, membrane soluble fractions <br /> location cell. The phenotypes mutants described                 generated centrifugation 100 000 g (see Materials <br /> in paper suggest mechanism action Sec4p               methods). On Western blot, proteins expressed <br /> may closely resemble EF-Tu. We propose                     plasmids distinguished endogenous <br />  <br /> 1686 <br />                                                                                                                                   Regulation vesicular traffic Sec4p <br />  <br />  <br />                                                                                                           expressed mammalian cells (Barbacid, 1987). <br />  SEC4      NHz-GDSGVGKS--OTAGOER--LVGNKSDM-ESSAKND-o                                                         We changed asparagine position 133 Sec4p <br />  YPTS       -GNSGVGKS--DTAGOERLVGNKCDL-ESALD. <br />                                                                                                           isoleucine. The analogous mutation position 116 in <br />                  *                      *          *                                                      H-ras results oncogenic transformation despite inability <br />  H-m.       -GAGGVGKS--DTAGOEE-LVGNKCDL                            .ETSAKTR                               protein detectably bind GTP (Walter et al., 1986). <br />  EF-Tu        -GHVDHGKT---DCPGHADVPLNKCDM                                                                 To vary level Sec4-1lel33p expressed, mutant gene <br />               -GAGESGKi B <br />                                                                                                           cloned various plasmids (see Figure 1 Materials <br />  C%a                             DVGGORE <br />                                  S LtCKan t <br />                                                                                                           methods). The efficiency plasmid pNB256 <br />                                                                                                           (which synthesize mutant protein low level) <br />                Ph_phry      BirNn SHe              Guw        Ring Contwt Sho <br />                                                                                                           transformed wild-type cells equivalent to <br />  B. <br />                                                                                                           plasmid pNB227 (containing normal gene). <br />                                         L79       133          D147                sop                    However, plasmid pNB260, synthesize wild- <br />  SEC4      12-GDSGVGKSC-DTAGOER-LVGNKSDM-G-ESSAKNDKND <br />                                                                                    II <br />                                                                                                           type levels Sec4-Ile133p used, transformation <br />   Allele         Mutation           GTP-binding         Memrane        <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>Phenotype</span>      Cnplnemnetatio   efficiency reduced 10-fold compared transformation <br />                                                                                                                                  - <br />  <br />  <br />  <br />                                                         Anmacent       Presence of <br />                                                                        Wild-type SEC4 <br />                                                                                          sec4-8        control plasmid, pNB139. No transformants are <br />  SEC4 <br />                                                                                                           obtained pNB225, synthesize high levels <br />                  Wild4ype <br />  sec4-8          GseDI47                                  +            Recessive <br />                                                                                                           mutant protein, analogous wild-type <br />                                                                        ts Lethal                          plasmid, pNB170, does yield colonies. However, pNB225 <br />  sec4-32         CC=sA214                                 -            Recessive                          digested restriction enzymes create 70 bp <br />  sec4-20, 21.22 Ne1t33                                                 Dose Dependent                     gap Ilel33 mutation, transformation does occur <br />                                                                        Dminan Lethal <br />  sec4-21,31      Nes1I33+ <br />                                                                                                           gap repaired using endogenous wild-type <br />                                                                        Recessive <br />                   CC.oA2&apos;4                                                                                gene template (Orr-Weaver et al., 1981). Thus, the <br />  sec441          QeL79                                                 Recessive           +              inability obtain transformants pNB225 construc- <br />                                                                                                           tion sequence located deleted 70 bp rather <br /> Fig. 3. (A) Comparison regions involved GTP binding. The                                            plasmid. The change the <br /> amino acid sequence portions Sec4p make                                            wild-type sequence region encodes Ile133 <br /> putative GTP binding site compared GTP binding regions of <br /> other known guanine nucleotide binding proteins. Based crystal                                     mutation. <br /> structure mammalian H-ras protein (de Vos et al., 1988)                                           In order study phenotype induced mutant <br /> two conserved domains amino terminal end interact                                         protein constructed strain which, addition wild- <br /> phosphoryl group. The conserved domains carboxy end                                       type SEC4 gene present chromosomal locus, has <br /> the protein form pocket interaction guanine ring. <br /> Positions single amino acid substitutions alter GTP binding                                     SEC4Ile133 (pNB276) integrated URA3 locus <br /> or hydrolysis activity H-ras p21 result oncogenic                                               (NY640). This plasmid synthesize mutant protein <br /> transformation mammalian cells indicated * (G 12, Q 61,                                         levels approximately equivalent endogenous <br /> N 116). Sequences obtained from: SEC4 (Salminen Novick,                                               Sec4p expression levels. As mentioned above, transfor- <br />  1987); YPTI (Gallwitz et al., 1983); H-ras (Capon et al., 1983); EF-                                     mation frequency wild-type cells mutant plasmid <br /> Tu (Arai et al., 1980); Gsay (Robishaw et al., 1986). (B) Summary of <br /> the behavior SEC4 alleles. The regions Sec4p make                                       10-fold lower equivalent wild-type plasmid <br />                                                                                                                - <br />  <br />  <br />  <br />  <br /> GTP binding site shown, sites sec4-8                                            (pNB141). In addition, growth rate cells expressing <br /> sec4-CCA mutations. GTP binding assayed nitrocellulose                                         Sec4-Ilel33p reduced. The doubling time wild-type is <br /> filter binding assay and/or solution. Membrane attachment                                           - <br />                                                                                                              2 h doubling time NY640, cells expressing <br /> determined centrifugation 100 000 g. The phenotypes listed                                          Sec4-11el33p addition wild-type Sec4p, 4 h <br /> reflect behavior various alleles SEC&apos; background. <br /> &apos;Dose-dependent dominant lethal&apos; reflects fact low levels                                     (data shown). <br /> this protein tolerated, normal levels cause severe growth                                       We examined efficiency secretion cells by <br /> secretion defect wild-type cells high levels lethal SEC+                                  assaying release secreted enzyme invertase the <br /> background. Complementation sec4-8 tests                                            cell surface. Approximately 40% invertase secreted <br /> encoded protein functional. <br />                                                                                                           NY640, cells expressing mutant protein, com- <br /> Sec4-8 mobility slower                                             pared wild-type cells (data shown). NY640 an <br /> of Sec4p Sec4-CCAp. As shown Figure 2A, contrast                                                 analogous strain bearing extra copy SEC4 also <br /> to wild-type protein primarily high-speed                                             examined electron microscopy. A dramatic increase in <br /> pellet (P2), Sec4-CCA protein soluble                                            number secretory vesicles seen NY640 (Figure <br /> fraction (S2), suggests carboxy-terminal                                               4), despite presence wild-type Sec4p cells. <br /> cysteines necessary membrane attachment. Sec4-                                                    Thus, SEC4-Ie133 induces dominant secretory defect. The <br /> CCAp does retain ability bind GTP (data shown).                                                vesicles strain similar size appearance to <br />                                                                                                           vesicles present late secretory mutants that <br /> SEC441e 133 dominant lethal mutation                                                                 accumulate post-Golgi vesicles incubated the <br /> The crystal structures EF-Tu mammalian H-ras                                                restrictive temperature (Novick et al., 1980). <br /> protein allowed conclusive identification                                                    To confirm phenotype conferred presence of <br /> regions proteins interact bound guanine                                            Sec4-1le133p, gene cloned inducible <br /> nucleotide (Jurnak, 1985; LaCour et al., 1985; Vos et                                                  GAL] promoter integrated URA3 locus in <br /> al., 1988). Four distinct domains higly conserved                                               SEC+GAL+ cells (NY519). The control strain, NY496, <br /> among large variety GTP-binding proteins (Figure 3A,                                                 wild-type SEC4 gene cloned GAL] and <br /> Dever et al., 1987) form pocket nucleotide                                                 integrated URA3. After 0, 6 12 h induction on <br /> rests. Particular mutations domains H-ras result                                              galactose, cells harvested preparation lysates <br /> in alleles lead oncogenic transformation                                                    measure Sec4p expression examination electron <br />  <br />                                                                                                                                                                   1687 <br />  N.C.Walworth et al. <br />  <br />  <br />  <br />  <br />     ves <br />  <br />  <br />                                                                                             , <br />                                                                                             ...      --    .- <br />                                                                                                                  0. <br />  <br />  <br />  <br />  <br />                                                                          Fig. 5. GTP binding ability mutant Sec4 proteins. Gal+ strains with <br />                                                                          Sec4 genes control inducible GALl promoter were <br />                                                                          induced galactose 12 h. Aliquots cells harvested 0, <br />                                                                          6 12 h induction, lysed glass beads boiled sample <br />                                                                          buffer. Samples separated SDS-polyacrylamide (15%) gel <br />                                                                          transferred nitrocellulose. The filters shown panel <br />                                                                          incubated anti-Sec4p antibody followed [1251]protein A <br />                                                                          exposed X-ray film. In panel, filters were <br />                                                                          incubated [cs-32P]GTP exposed X-ray film. Strains are <br />                                                                          follows (transformed strains plasmid integrated the <br />                                                                          URA3 locus): NY451 untransformed Gal + strain. NY496 is <br />                                                                          NY451 SEC4 control GAL] promoter (Sec4p). <br />                                                                          NY519 NY451 SEC4-lle133 control GAL] <br />                                                                          (Sec4-23p). NY643 NY451 sec4-Leu79 control GAL] <br />                                                                          (Sec4-43p). NY456 sec4-8, Gal&apos; strain. NY644 NY456 with <br /> Fig. 4. Cells expressing Sec4-Ilel33p contain large number          sec4-Leu79 control GAL]. <br /> secretory vesicles despite presence wild-type Sec4p. Cells were <br /> harvested growth exponential phase fixed electron      Cells containing times wild-type protein this <br /> microscopy. (A) NY640 wild-type SEC4 gene                 time accumulate secretory vesicles (Figure 6B). After <br /> chromosomal locus SEC4-41el33 gene integrated URA3        12 h growth galactose, overexpression mutant <br /> locus. The integrated plasmid, pNB276, synthesize normal          protein causes accumulation large number of <br /> levels mutant protein. Bar = 0.5 ttm. (B) Wild-type strain <br /> containing extra copy wild-type SEC4 gene URA3          secretory vesicles, overexpression wild-type Sec4p <br /> locus shown magnification cell (A). Typical     results slight accumulation membranes including a <br /> wild-type cells, secretory vesicles seen section.         secretory vesicles (not shown). The growth rate of <br />                                                                          NY519 begins drop 4 h inducing expression <br /> microscopy. The Western blot shown Figure 5 indicates                 mutant protein (Figure 6C), overexpression <br /> that level Sec4p NY496 increases induction                wild-type protein NY496 12 h no <br /> with galactose, compared NY451, parental strain                effect doubling time. Thus, production Sec4-11el33p <br /> which does contain plasmid. Similarly, NY519 total              results secretory defect correlates decreased <br /> Sec4p overexpressed extent                growth rate, despite presence wild-type protein <br /> in NY496. This greater instability                cells. <br /> mutant protein. Sec4-Ile133p slightly faster mobility <br /> than wild-type protein accompanying GTP                      Sec4-lle 133p dominant loss function mutant <br /> binding blot Figure 5 indicates mutant protein               When SEC4-41e133 gene introduced sec4-8 strain <br /> does bind significant amounts radiolabeled nucleotide             pNB256, plasmid produces low levels the <br /> by assay. This observation confirmed                    protein, transformed cells grow 25&#194;&#176;C able <br /> measuring GTP-&apos;y-S binding activity solution lysates               grow 37&#194;&#176;C. As control shown sec4-8 <br /> of cells protein overexpressed (Table              cells transformed pNB227 expressing wild-type <br /> I). Sec4-Ilel33p 10 000 g                   allele able grow 37&#194;&#176;C. Thus, Sec4-11el33p does <br /> 100 000 g pellet fractions following differential centrifuga-            appear wild-type function. Genetic analysis of <br /> tion lysed spheroplasts indicating protein does              interaction SEC4 subset late acting <br /> attach membranes (data shown).                                    secretory mutants allows distinction between <br />    By thin-section electron microscopy, dramatic accumula-             gain loss Sec4p function. A gain Sec4p function <br /> tion secretory vesicles seen NY519 early              achieved duplicating wild-type SEC4 gene, which <br /> 6 h induction twice mutant              results suppression temperature-sensitive growth <br /> protein wild-type protein present cell (Figure 6A).            defect sec2-41, sec8-9 seciS-1 strains. A loss Sec4p <br /> 1688 <br />                                                                                                           Regulation vesicular traffic Sec4p <br />  <br />  <br /> Table I. GTP--y-S binding Sec4 proteins overexpressed sec4-8 background <br />  Induced protein (strain)               Specific                        Relative               Binding normalized               % wild-type <br />                                         GTP-y-S bindinga                Sec4 proteinb                    Sec4p level                   Sec4p <br />                                         (pmol/mg x 10)                                                   (pmol/mg x 10)                   binding <br />  Membrane fractionc <br />    Sec4p (NY500)                         14                             1                                14                               100 <br />    Sec4-Ile133p (NY520)                   0.5                           0.72                              0.7                               5 <br />    Sec4-Leu79p (NY644)                    8.0                           1.1                               7.3                              52 <br />  Soluble fractiond <br />    Sec4p (NY500)                         14                             1                                14                               100 <br />    Sec4-Ile133p (NY520)e                      -                         0.25 <br />    Sec4-Leu79p (NY644)                   11                             1.2                               9.2                              66 <br />  <br /> aCells grown presence galactose 24 h induce expression indicated proteins. Spheroplasts lysed osmotically and <br /> centrifuged 100 000 g 1 h. The assay measures binding [35S]GTP-y-S solution using rapid filtration technique. Specific binding was <br /> calculated difference binding presence 1 mM AppNHp binding presence 1 mM GppNHp. Binding specific to <br /> the induced protein calculated subtracting detected equivalent fraction strain NY456, GAL+, sec4-8 strain. Values <br /> represent average binding experiments duplicate samples assayed. <br /> bThe levels Sec4 protein fraction determined quantitative Western blot analysis. Counts normalized number counts <br /> detected equivalent fraction NY500, strain overproducing Sec4p. <br /> cPellet fraction 100 000 g centrifugation. Specific binding values represent total binding minus detected equivalent fraction of <br /> NY456: 5.5 pmol/mg x 10. <br /> dSupematant fraction 100 000 g centrifugation. Specific binding values represent total binding minus detected equivalent <br /> fraction NY456: 2.2 pmol/mg x 10. <br /> eThe Sec4-Ilel33p present fraction low assess binding. <br /> function tolerated particular mutants                    C-terminal portion SEC4-41el33 gene <br /> strains bearing sec4-8 mutation sec2-41,                   replaced portion sec4-CCA gene <br /> sec8-9 secl5-J constructed (Salminen                          bearing cysteine deletion. When gene introduced <br /> Novick, 1987).                                                                 wild-type SEC&apos; background obvious effect on <br />    Analysis behavior SEC4-lle133 allele                       growth observed gene expressed a <br /> crosses sec strains indicates behaves                  normal level (pNB286) high level (pNB285). Western <br /> loss function allele. This behavior dominant,                 blot analysis cell fractionation experiments indicates that <br /> case wild-type SEC4 gene present. Strains                          protein mol. wt Sec4-Ilel33p that <br /> containing SEC4-Ile133 (marked Ura+) strains                            soluble (Figure 2B). Thus, removal cysteines, <br /> bearing sec mutations (marked temperature                             makes Sec4-1lel33p soluble wild-type cells, relieves <br /> sensitivity) crossed resulting diploids                           dominant lethality mutant protein. <br /> sporulated. The SEC441e133 mutant gene synthesized at <br /> wild-type levels lethal combined subset                  Sec4-Leu79p binds GTP efficiently, still <br /> of sec mutants recessive, temperature-                          functional <br /> sensitive sec4-8 mutant lethal (Salminen Novick, 1987)                  A mutation domain GTP binding site responsible <br /> even functional copy SEC4 present (Tables II                      interacting phosphoryl group GTP causes ras <br /> and Ill). In crosses SEC4-Ile133 and, example,                     transforming activity (see Figure 3A). This mutation, <br /> seciS-l, tetrads inviable spores.                          changes glutamine position 61 leucine, causes <br /> Phenotypic analysis remaining viable spores indicated                   protein hydrolyze GTP 10-fold slower rate. If <br /> that inviable spore contained seciS-]                  glutamine changed amino acids besides <br /> and Ura+ marked SEC4-11e33 allele. Thus,                               leucine, hydrolysis rate decreased protein <br /> combination seclS-] SEC4-Ilel33 lethal event.                      various degrees transforming ability depending the <br /> The results obtained sec2-41 sec8-9. This                    specific amino acid substituted (Der et al., 1986). We have <br /> phenotype specific, crosses vesicle accumulating                    changed analogous glutamine position 79 Sec4p to <br /> mutants secl-Ji sec6-4, crosses Golgi                  leucine. <br /> accumulating mutant sec7-1 endoplasmic reticulum                          Sec4-Leu79p able bind GTP filter assay <br /> accumulating mutant secl8-1, yielded viable temperature-                   (Figure 5) quantitative solution assay the <br /> sensitive, ura+ spores. If SEC41el33 present low                     mutant protein binds nucleotide 50-60% wild-type <br /> expression plasmid, viable ura+, temperature-sensitive                    levels (Table I). Neither assays rule the <br /> spores obtained indicating low levels                       possibility rate rate GTP binding the <br /> mutant protein tolerated sec mutants                  mutant protein different rates wild-type <br /> a functional copy SEC4 present (tetrad data                     protein. Introduction protein wild-type cells has <br /> shown, summarized Table III).                                           obvious effect cell growth. When introduced into <br />                                                                                temperature-sensitive sec4-8 cells expressed normal <br /> Analysis Sec4-lle 133, CCA                                                  levels (pNB287), protein does rescue temperature- <br /> Since deletion cysteine residues causes Sec4p                        sensitive growth defect cells. Thus, although <br /> become soluble non-functional, tested                      Sec4-Leu79p binds GTP altered characteristics as <br /> their removal dominant Sec4-1lel33p prevent                     compared wild-type protein, functional. However, <br /> this protein detrimental. A gene constructed                    protein expressed higher normal levels in <br />                                                                                                                                                 1689 <br />  N.C.Walworth et al. <br />  <br />                                                                            Discussion <br />                                                                            The behaviors various Sec4 mutant proteins are <br />                                                                            summarized Figure 3B. The proteins examined <br />                                                                            capacity bind GTP attach membranes. <br />                                                                            Complementation temperature-sensitive growth defect <br />                                                                            sec4-8 used determine they <br />                                                                            functional. The phenotypes induced mutant <br />                                                                            proteins wild-type cells cells containing other <br />                                                                            secretory defects documented. The behavior of <br />                                                                            mutants consistent Sec4p playing cyclical role <br />                                                                            mediating transport secretory vesicles plasma <br />                                                                            membrane yeast. <br />                                                                            Relationship Sec4p localization function <br />                                                                            We shown previously Sec4p cell <br />                                                                            stably associated plasma membrane secretory <br />                                                                            vesicles despite hydrophilic nature protein <br />                                                                            sequence. A soluble pool 10-15% total Sec4p is <br />                                                                            maintained evidence indicates protein can <br />                                                                            recycle plasma membrane secretory vesicles most <br />                                                                            probably soluble intermediate (Goud et al., 1988). <br />                                                                            The structurally related ras family GTP-binding proteins, <br />                                                                            including yeast Yptlp, attach cytoplasmic surfaces <br />                                                                            cellular membranes lipid moiety linked a <br />                                                                            C-terminal cysteine (Chen et al., 1985; Buss Sefton, <br />                                                                            1986; Fujiyama Tamanoi, 1986; Molenaar et al., 1988). <br />                                                                            Yptlp shown palmitoylated deletion of <br />                                                                            cysteines results soluble, unmodified protein <br />                                                                            (Molenaar et al., 1988). Like Yptlp, Sec4p terminates with <br />                                                                            cysteines (Salminen Novick, 1987). Deletion of <br />                                                                            amino acid residues results majority the <br />                                                                            protein remaining soluble. The protein functional even <br />                                                                            retains ability bind GTP. This suggests that <br />                                                                            cysteines necessary mediating stabilizing the <br />                                                                            attachment Sec4p membranes association <br />                                                                            required normal function. As modification such <br />                                                                            palmitoylation detected. <br />  <br />                                                                            SEC4-lle 133 dominant secretory mutant <br />                                                                            SEC4-4e133     constructed based behavior a <br />                                                                            transforming mutant H-ras dominant lethal mutant <br />                                                                            YPTI (Schmitt et al., 1986; Walter et al., 1986). <br />                                                                            Mutations p21 H-ras position 116 119 third <br />                                                                            domain GTP-binding site decrease protein&apos;s affinity <br />                                                                            GTP GDP. Despite lowered affinity for <br />                                                                            nucleotide, particular substitutions 116 119 result in <br />                                                                            transformation competent ras proteins, assumed <br /> Fig. 6. Induced expression Sec4-Ilel33p causes accumulation of <br />                                                                            constitutively active (Sigal et al., 1986; Walter et al., <br /> secretory vesicles decrease growth rate. The growth rate       1986). A mutation domain yeast YPTI <br /> cells expressing wild-type Sec4p Sec4-Ile133p control          gene position 121 changes asparagine isoleucine and <br /> GAL] followed induction galactose cells            prevents protein binding GTP. Schmitt et al. (1986) <br /> harvested electron microscopy. (A) After 6 h induction         proposed dominant <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> induced <br /> galactose, cells expressing Sec4-Ilel33p (NY519) accumulate large <br /> number secretory vesicles. Bar = 0.5 ,tm. (B) At time          mutant protein locked active <br /> point, cells expressing Sec4p (NY496) display normal morphology.         conformation. The SEC4-lle133 mutation constructed <br /> (C) Up 12 h induction galactose growth rate cells        mimic situation. As expected, Sec4-Ilel33p is <br /> expressing wild-type Sec4p normal 6 h growth rate        defective GTP binding. When protein expressed <br /> of cells expressing mutant Sec4-Ilel33p slowed. Open squares    = <br />  <br /> NY451. Closed diamonds = NY496. Open triangles       =  NY519.             normal levels cells expressing wild-type <br />                                                                            Sec4p, cells grow slowly. In addition, cells <br />                                                                            secrete lower levels invertase accumulate excess <br /> sec4-8 (on   pNB293)     cells grow          slower rate   secretory vesicles cytoplasm. We postulate there <br /> at 37&#194;&#176;C cells expressing level wild-type                  reversible interaction Sec4p effector <br /> Sec4p (data shown). This phenotype reflect                     molecule. This interaction irreversible case <br /> altered binding activity protein.                                  mutant protein which, therefore, prevents wild-type <br /> 1690 <br />                                                                                                             Regulation vesicular traffic Sec4p <br />  <br />  <br /> Table 11. Tetrad analysis URA3:: SEC4-41el33 crossed ts- sec mutants <br />                                               Categories tetrads: distribution crosses <br />                                               2 ts+ura&apos;               I ts+ura&apos;                                      1 ts+ura+ <br />                                               2 ts-ura-               1 ts-ura-                                      I ts-ura- <br />                                                                       1 tstura-              2 ts+ura-               1 ts+ura-                2 ts+ura- <br />                                                                       1 ts-ura&apos;              2 ts-ura&apos; <br />                                                                                                                      I inviablea              2 inviablea <br />  URA3:: SEC4-11e133   x se(4-8                3                                                                      1                        8 <br />  NY64()                 NY405 <br />  URA3:: SEC4-11e133   x se(l5-1                                                                                      9                        2 <br />  NY640                   NY64b <br />  URA3:: SEC4-11e133   x sec2-41               3                                                                      8                        1 <br />  NY640                  NY130 <br />  URA3:: SEC4-11e133   x sec8-9                 1                                                                     8                        2 <br />  NY640                  NY410&apos; <br />  URA3:: SEC4-4le133   x se(6-4                2                      8                      1 <br />  NY640                   NY17d <br />  URA3:: SEC4-11e133   x seel-II                1                     6                      3 <br />  NY640                  NY24&apos; <br />  URA3:: SEC4-41el33   x sec7-1                                       8 <br />  NY640                  NY 176&apos; <br />  URA3::SEC4-41el33    x seel8-1                1                     9                      1 <br />  NY640                  NY43I <br />  <br /> aThe absence ts ura+ spores indicates combination ura+ marked SEC4-11eI33 SEC4-I1e133,CCA allele indicated ts- seC <br /> allele lethal event. <br /> Additional tetrads <br /> bi tetrad, I viable spore: ts-ura-. <br /> cl tetrad; 2 viable spores: ts-ura-; ts ura-. <br /> dl tetrad; 4 viable spores: ts-ura&apos;: 2 ts-ura-; ts+ura-. <br /> &apos;2 tetrads, 4 viable spores: ts+ura+; 2 ts-ura-, ts+ura-. <br />  11 tetrad, 4 viable spores: ts+ura+; ts+ura-; 2 tsura&apos;, 1 tetrad; 4 viable spores: 2 ts+ura+; ts--ura-; ts-ura&apos;, 1 tetrad; 4 viable spores: ts+ura+; <br /> ts ura7 ts ura+; ts+ura: 1 tetrad. 3 viable spores: ts+ura+; ts+ura-; ts-ura&apos;. <br /> &quot;I tetrad; 3 viable spores: ts ura ; ts-ura-; ts-ura&apos;. <br />  <br /> protein function responsible dominant nature <br /> of mutation.                                                              Table III. Summary genetic interactions: double mutant viability at <br />                                                                               25 &#194;&#176;C <br />    Evidence SEC4-41el33 dominant loss function <br /> mutant comes genetic experiments. Previous studies                        ts- strain       Site block    NY405             NY640          NY617 <br /> (Salminen Novick, 1987) shown gain                                           37&#194;&#176;C          sec4-8            sec4-2IP       sec4-20P <br />                                                                                                                                    SEC4           SEC4 <br /> Sec4p function case SEC4 duplication can <br /> suppress temperature-sensitive growth defects set                     sec4-8           vesicle                            -              + <br /> of vesicle-accumulating sec mutants including seclS-]                      sec2-41          vesicle          -                 -              + <br /> sec2-41. A loss Sec4p function, exemplifed                        sec 8-9          vesicle          -                 -              + <br /> presence sec4-8 sole copy SEC4, lethal                      secl5-1          vesicle          -                 -              + <br /> strains bearing mutations. The SEC4-41el33 gene,                   secl-1l          vesicle          +                 + <br /> when introduced strains level permits                     sec6-4           vesicle          +                 + <br />  <br /> viability wild-type background, behaves loss                      sec7-1           Golgi            +                 + <br />                                                                                sec l8-1         ER               +                 + <br /> function allele, i.e. combination SEC4-41el33 with <br /> seciS-1 sec2-41 lethal. Importantly, inviability                 asec4-21: SEC4-lle133 normal level. <br /> seen specifically strains known interact                bsec 4-20: SEC4-Ile133 low level. <br /> at genetic level SEC4. Introduction SEC4-41el33 <br /> into sec6-4, secl-11, sec7-J sec]8-1 lethal               conformation Sec4p recognizes attachment site newly <br /> are mutations suppressed duplication                   formed secretory vesicle. This complex recognized a <br /> SEC4 inviable combined sec4-8 (Salminen                         hypothetical effector cytoplasmic surface plasma <br /> and Novick, 1987).                                                            membrane triggering membrane fusion exocytosis. By <br />                                                                               analogy, release Sec4p effector coupled to <br /> Sec4p functions cycle                                            dependent hydrolysis bound GTP leading to <br /> Two models mechanism action GTP-binding                         conformational change. <br /> proteins discussed (see Introduction). The model                      A failure bind nucleotide efficiently leave Sec4p <br /> shown Figure 7 based loosely mechanism                        inactive active conformation. The <br /> which EF-Tu functions deliver aminoacyl-tRNA                     phenotype cells expressing Sec4-lle l 33p consistent with <br /> ribosome (Kaziro, 1978; Bourne, 1988). At steady state,                       hypothesis trapped active conformation. <br /> Sec4p cell soluble form, bound                Thus, competent attach vesicles enter <br /> to secretory vesicles bound plasma membrane.                       cycle, relax inactive <br /> Sec4p binds GTP postulate GTP-bound                        conformation blocks cycle. Vesicles formed but <br />                                                                                                                                                       1691 <br />   N.C.Walworth et al. <br />  <br />  <br />  <br />  <br />              bF      ,       i.l <br />                                          .~~~~~        / <br />                                                         ,~ ~ i. <br />                                                        I, <br />                                                                 i/ <br />                                                                r:. <br />                                                                         Table IV. Strains used <br />                                                                          Strain <br />                                                                          NY13 <br />                                                                          NY15 <br />                                                                          NY17 <br />                                                                          NY24 <br />                                                                          NY64 <br />                                                                          NY 130 <br />                                                                          NY176 <br />                                                                          NY405 <br />                                                                          NY410 <br />                                                                          NY451 <br />                                                                                       Genotype <br />                                                                                       MATa, ura3-52 <br />                                                                                       MATa, ura3-52, his4-619 <br />                                                                                       MATa, ura3-52, sec6-4 <br />                                                                                       MATa, ura3-52, secl-Ji <br />                                                                                       MATa, ura3-52, secl5-1 <br />                                                                                       MATa, ura3-52, sec2-41 <br />                                                                                       MA Ta, ura3-52, sec7-1 <br />                                                                                       MATa, ura3-52, sec4-8 <br />                                                                                       MATa, ura3-52, sec8-9 <br />                                                                                       MATa, ura3-52, Gal&apos; <br />                                                                          NY456        MATa, ura3-52, Gal&apos;, sec4-8 <br />                                                                          NY496        MATa, ura3-52:: (URA3, GALJ-SEC4), Gal+ <br />                                                                          NY500        MATa, ura3-52:: (URA3, GAL1-SEC4), sec4-8, Gal+ <br />                                                                          NY519        MATa, ura3-52:: (URA3, GALI-SEC4-23), Gal+ <br />                                                                          NY520        MATa, ura3-52:: (URA3, GALJ-SEC4-23), sec4-8, Gal+ <br />                                                                          NY617        MATa, ura3-52:: (URA3, sec4-20), his4-619 <br />                                                                          NY637        MATTa, ura3-52:: (URA3, sec4-21,31), his4-619 <br />                                                                          NY639       MAToe, ura3-52:: (URA3, SEC4), his4-619 <br />                                                                          NY640       MAToe, ura 3-52::(URA3, SEC4-21), his4-619 <br />                                                                          NY643       MATa, ura3-52:: (URA3, GALI-sec4-43), Gal&apos; <br />                                                                          NY644       MATa, ura3-52:: (URA3, GALJ-sec4-43), Gal+ <br />                                                                          NY678       MATa, ura3-52, pNB254 (URA3, sec4-32), sec4-8 <br />                                                                          NY682       MATa, ura3-52, pNBJ70 (URA3, SEC4), sec4-8 <br />                                                                          sec4 <br /> Fig. 7. Model Sec4p function. The GDP bound Sec4p is <br /> exchanged GTP. In GTP-bound conformation Sec4p recognizes        alleles <br /> an attachment site newly formed secretory vesicle. This complex     sec4-8 <br /> is recognized effector cytoplasmic surface plasma <br />                                                                                      Sec4-Asp147 <br /> membrane-triggering membrane fusion exocytosis. Release of <br />                                                                          sec4-20     Sec4-IIeJ33 (low) <br /> Sec4p effector coupled hydrolysis bound GTP. <br />                                                                          sec4-21     Sec4-IIel33p (normal) <br />                                                                         sec4-21,31 Sec4-Ile133p,CCA (normal) <br />                                                                         sec4-23    Gal/Sec4-Ilel33 <br /> an   effector necessary vesicle delivery place      sec4-32    Sec4-CCA (high) <br />  unavailable bound activated Sec4p              sec4-43    Gal/Sec4-Leu79 <br />  previous round. As dominant mutant, Sec4-Ilel33p <br />  prevents wild-type protein functioning manner that <br />  suggests Sec4p interacts directly specific              shown interact SEC4 genetic level. The <br />  component(s) cycle. Lowering level mutant                dominant SEC4-Ile133 allele provides tool with <br />  protein allows wild-type protein compete function               probe interacting proteins genetic <br />  effectively. A higher level mutant protein does             biochemical level. Furthermore, studies the <br />  permit wild-type protein function all. Significantly,         biochemical properties Sec4p identification of <br />  normal levels synthesis secretory defect             factors influence properties promise be <br />  induced presence Sec4-Ilel33p seen.                       revealing. <br />     In yeast Ras/adenylate cyclase number of <br />  components regulatory pathway identified                   Materials methods <br />  suppressor analysis (Toda et al., 1985; Cannon et <br /> al., 1986). We attempted identify suppressors loss               Yeast genetic techniques <br />                                                                         Yeast strains used study listed Table IV. Cells grown <br /> of Sec4p function isolating revertants sec4-8 strain.           rich medium (YPD) containing 1 % Bacto-yeast extract, 2 % Bacto-Peptone <br /> However, intragenic suppressors obtained (data                (Difco Laboratories, Inc., Detroit, MI) 2 % glucose carbon source <br /> not shown). Our failure obtain extragenic suppressors            indicated. When selective conditions required cells were <br /> reflect fundamental difference mechanism               grown minimal medium containing 0.7% yeast nitrogen <span id='am-237' about='xsp:base' typeof='owl:Thing'>base</span> without <br /> which Ras2p regulates cell cycle mechanism               amino acids (Difco) 2% glucose, supplemented auxotrophic <br /> which Sec4p regulates vesicular traffic. That is,                requirements, described Sherman et al. (1974), necessary. <br />                                                                            Yeast transformation method alkali cation treatment <br /> adenylate cyclase involves amplification             (Ito et al., 1983). Transformants selected minimal medium 25&#194;&#176;C. <br /> a second messenger molecule, constitutive activation                 Crosses, sporulation diploids dissection tetrads as <br /> downstream components pathway suppresses loss                described Sherman et al. (1974). <br /> of function Ras protein, upstream regulatory                 Nucleic acid techniques <br /> molecule. Loss Sec4p function suppressible                Escherichia coli strain DH 1 used cloning experiments. Plasmid <br /> in similar manner operate cyclical fashion.           preparations essentially described (Silhavy et al., <br />    Future studies role Sec4p directed              1984). Centromere-containing plasmids pNB 139 pNB170 encoding the <br /> the identification components interact               SEC4 genes described Salminen Novick (1987) plasmid <br />                                                                         pNB227 constructed sub-cloning BamHI -EcoRI fragment of <br /> Sec4p function cycle. Candidates proteins          pNB170 BanHI-SphI site YCp5O, reverses orientation <br /> that involved products genes             SEC4 gene respect URA3 gene plasmid. Mutant <br />  <br /> 1692 <br />                                                                                                             Regulation vesicular traffic Sec4p <br />  <br /> SEC4 genes cloned plasmids integrating versions     Barbacid,M. (1987) Annu. Rev. Biochem., 56, 779-827. <br /> plasmids (YIpS derivatives) M 13 vectors mutants          Boume,H.R. (1988) Cell, 53, 669-671. <br /> were constructed (see below). The parent integrating plasmid containing         Buss,J. Sefton,B. (1986) Mol. Cell. Bio., 6, 116-122. <br /> the GALl -SEC4 construct (pNB219) described Goud et al. (1988)            Cannon,J.F., Gibbs,J.B. Tatchell,K. (1986) Genetics, 113, 247-264. <br /> and mutant derivatives constructed direct mutagenesis portion      Capon,D.J., Chen,E.Y., Levinson,A.D., Seeburg,P.H. Goeddel,D.V. <br /> of plasmid sub-cloning mutagenized M13 pNB219.               (1983) Nature, 302, 33-37. <br />    Mutagenesis carried using Amersham oligonucleotide directed      Chen,Z., Ulsh,L.S., DuBois,G. Shih,T.Y. (1985) J. Virol., 56, <br /> in vitro mutagenesis kit according manufacturer&apos;s instructions              607-612. <br /> (Amersham, Arlington Heights, IL). Oligonucleotides synthesized         Der,C., Finkel,T. Cooper,G.M. (1986) Cell, 44, 167-176. <br /> the Yale University DNA Synthesizing Service. All sequences mutant       Deschenes,R.I. Broach,J.R. (1987) Mol. Cell. Bio., 7, 2344-2351. <br /> genes confirmed dideoxy chain termination method (Sanger            Dever,T.E., Glynias,M.J. Merrick,W.C. (1987) Proc. Natl. Acad. Sci. <br /> et al., 1977) presence [cs-35S]dATP (650 Ci/mmol; Amersham)              USA, 84, 1814-1818. <br /> (Williams et al., 1986). To construct 1le33 mutation EcoRI-HindIlI      Vos,A.M., Tong,L., Milburn,M.V., Matias,P.M., Jancarik,J., <br /> fragment pNB219 subcloned mpl8, M13 derivative                 Noguchi,S., Nishimura,S., Miura,K., Ohtsuka,E. Kim,S.-H. (1988) <br /> mutagenized using 25 base oligonucleotide encoding Ile instead Asn:           Science, 239, 888-893. <br /> 5&apos;CATATCGCTCTTAATACCAACCAAC3&apos;. To construct CCA                             Fujiyama,A. Tamanoi,F. (1986) Proc. Natl. Acad. Sci. USA, 83, <br /> mutant Hindmll-EcoRI fragment pNB170 cloned M13               1266-1270. <br /> derivative mpl9 mutagenized using 30 base oligonucleotide        Gallwitz,D., Donrath,C. Sander,C. (1983) Nature, 306, 704-709. <br /> two codons cysteines C terminus deleted: 5&apos;AGTTCTAA-             Gilman,A. (1987) Annu. Rev. Biochem., 56, 615-649. <br /> ATCAAATATGAAGAAAAGAAGAT3&apos;. To construct Leu 79 mutant                       Goud,B., Salminen,A., Walworth,N.C. Novick,P.J. (1988) Cell, 53, <br /> the BamHI-HindIII fragment pNB139 EcoRl-HindIll                       753-768. <br /> fragment pNB190 (the YCpSO version pNB219) cloned                   Holm,C., Meeks-Wagner,D., Fangman,W. Botstein,D. (1986) Gene, <br /> independently mpl 8 mutagenized using 24-base oligonucleotide           42, 169-173. <br /> encoding Leu instead Gln: 5&apos;GGAAACGTTCCAAACCAGCGGTAT3&apos;.                      Ito,H., Fukuda,Y., Murata,K. Kimura,A. (1983) J. Bacteriol., 153, <br /> M13 plasmids propagated E.coli strain TG1 (for mutagenesis)              163-168. <br /> JM101 (for sequencing).                                                         Jurnak,F. (1985) Science, 230, 32-36. <br />                                                                                 Kaziro,Y. (1978) Biochim. Biophys. Acta, 505, 95-127. <br /> Expressing SEC4 GAL 1 promoter                                         LaCour,T.F.M., Nyborg,J., Thirup,S. Clark,B.F.C. (1985) EMBO <br /> Cells grown YP medium containing 2% lactate. Expression               J., 4, 2385-2388. <br /> the GALl promoter induced using 0.5% galactose, 0. 1 % glucose,          Lapetina,E. Reep,B. (1987) Proc. Natl. Acad. Sci. USA, 84, <br /> included maintain moderate level expression (Adams, 1972).                  2261 -2265. <br />                                                                                 Molenaar,C., Pranger,R. Gallwitz,D. (1988) EMBO J., 7, 971-976. <br /> Other methods                                                                   Neer,E.J. Clapham,D.E. (1988) Nature, 33, 129-134. <br /> Electron microscopy carried described Walworth Novick         Northup,J.K., Smigel,M.D. Gilman,A.G. (1982) J. Biol. Chem., 257, <br /> (1987). Electrophoresis immunoblotting described Goud et al.             11416-11423. <br /> (1988). Glass bead lysates prepared harvesting 5 x 107 cells,           Novick,P., Field,C. Schekman,R. (1980) Cell, 21, 205-215. <br /> washing 1 ml cold 10 mM Tris, pH 7.5, 10 mM NaN3, resuspending            Orr-Weaver,T.,Szostak,J. Rothstein,R. (1981) Proc. Natl. Acad. Sci. <br /> in 75 Al buffer, adding glass beads reach meniscus.          USA, 78, 6354-6358. <br /> Lysis achieved mixing 3 min Vortex mixer (VWR                 Robishaw,J.D., Russell,D.W., Harris,B.A., Smigel,M.D. Gilman,A.G. <br /> Scientific) 4&#194;&#176;C cold room followed immediately addition SDS             (1986) Proc. Natl. Acad. Sci. USA, 83, 1251-1255. <br /> sample buffer boiling. Osmotic lysis spheroplasts essentially   Salminen,A. Novick,P. (1987) Cell, 49, 527-538. <br /> as Goud et al. (1988). Following homogenization, cell lysates spun      Sanger,F., Nicklen,S. Coulson,A.R. (1977) Proc. Nat!. Acad. Sci. USA, <br /> at 450 g, pellets washed pooled supernatants spun              74, 5463-5467. <br /> 100 000 g SW50. 1 rotor (Beckman Instruments) 33 000 r.p.m.         Schmitt,H., Wagner,P., Pfaff,E. Gallwitz,D. (1986) Cell, 47,401-412. <br /> 1 h generate membrane soluble fractions. GTP binding                  Segev,N., Mulholland,J. Botstein,D. (1988) Cell, 52, 915-924. <br /> nitrocellulose filters according Lapetina Reep (1987).          Sherman,F., Fink,G. Lawrence,? (1974) Methods Yeast Genetics. <br /> GTP-y-S binding solution measured rapid filtration technique         Cold Spring Harbor Laboratory, Cold Spring Harbor, NY. <br /> utilizing HA MF-type membrane filters (Millipore, Bedford, MA)               Sigal,T., Gribbs,J., D&apos;Alonzo,J., Temeles,G., Wolanski,B., Socher,S. and <br /> described Northup et al. (1982). From soluble membrane fractions,           Scolnick,E. (1986) Proc. Natl. Acad. Sci. USA, 83, 952-956. <br /> 7.5-10 I g protein assayed 25 i1 50 mM Hepes, pH 8.0,              Silhavy,T., Berman,M. Enquist,L. (1984) Experiments Gene <br /> 50 mM MgCl2, 200 mM NaCl, 1 mM EDTA, 1 mM dithiothreitol, 0.1%                     Fusions. Cold Spring Harbor Laboratory, Cold Spring Harbor, NY. <br /> Lubrol 2 ttM [355]GTP--y-S (Amersham, 5 x 105 c.p.m.) 30&#194;&#176;C           Toda,T., Uno,j., Ishikawa,T., Powers,S., Kataoka,T., Broek,D., <br /> 60 min. Where appropriate, guanosine-5&apos;-(3,-y-imino)triphosphate                   Cameron,S., Broach,J., Matsumoto,K. Wigler,M. (1985) Cell, 40, <br /> [Gpp(NH)p] adenine-5&apos;-(f,,y-imino)triphosphate [App(NH)p] added             27-36. <br /> to 1 mM (Boehringer Mannheim).                                                  Trahey,M. McCormick,F. (1987) Science, 238, 542-545. <br />                                                                                 Walter,M., Clark,S. Levinson,A. (1986) Science, 233, 649-652. <br />                                                                                 Walworth,N. Novick,P. (1987) J. Cell Biol., 105, 163-174. <br /> Acknowledgements                                                                Williams,S., Slatko,B., Moran,L. deSimone,S. (1986) BioTechniques, <br />                                                                                    4, 139-148. <br /> We thank Drs P.DeCamilli, I.Mellman, H.Boume, S.Powers, A.Tamir             Willumsen,B., Christensen,A., Hubbert,N., Papageorge,A. Lowry,D. <br /> J.Northup helpful discussions. We thank M.Talmor synthesis              (1984a) Nature, 310, 583-586. <br /> oligonucleotides, H.Stukenbroek R.Dowin assistance section      Willumsen,B.M., Norris,K., Papageorge,A.G., Hubbert,N.L. and <br /> electron microscopy A.Curley-Whitehouse assistance photography.         Lowry,D.L. (1984b) EMBO J., 3, 2581-2585. <br /> This work supported grants GM35370 CA46218 the <br /> National Institutes Health. N.C.W. supported training grant         Received 27th January, 1989; revised 20th March, 1989 <br /> from National Institutes Health, B.G. supported grant from <br /> the Swebilius Foundation A.K.K. supported grant th <br /> </body></html>