<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Proteomic Analysis Acidocalcisome, Organelle <br /> Conserved Bacteria Human Cells <br /> Guozhong Huang1, Paul N. Ulrich2, Melissa Storey1, Darryl Johnson3, Julie Tischer1, Javier A. Tovar2, <br /> Silvia N. J. Moreno1, Ron Orlando3, Roberto Docampo1* <br /> 1 Center Tropical Emerging Global Diseases Department Cellular Biology, University Georgia, Athens, Georgia, United States America, 2 Department of <br /> Biology, Georgia State University, Atlanta, Georgia, United States America, 3 Complex Carbohydrate Research Center, University Georgia, Athens, Georgia, United <br /> States America <br />  <br />  <br />  <br />      <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />      Acidocalcisomes acidic organelles present diverse range organisms bacteria human cells. In study <br />      acidocalcisomes purified model organism Trypanosoma brucei, protein composition was <br />      determined mass spectrometry. The results, previously reported, acidocalcisomes are <br />      rich pumps transporters, involved phosphate cation homeostasis, calcium signaling. We validated the <br />      acidocalcisome localization seven new, putative, acidocalcisome proteins (phosphate transporter, vacuolar H+-ATPase <br />      subunits d, vacuolar iron transporter, zinc transporter, polyamine transporter, acid phosphatase), confirmed the <br />      presence previously characterized acidocalcisome proteins, validated localization novel proteins <span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span> <br />      different subcellular compartments expressing fused epitope tags endogenous loci by <br />      immunofluorescence microscopy specific antibodies. Knockdown newly identified acidocalcisome proteins <br />      <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) revealed essential <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span>survival</span> parasites. These results provide a <br />      comprehensive insight unique composition acidocalcisomes T. brucei, important eukaryotic pathogen, and <br />      direct evidence acidocalcisomes especially adapted accumulation polyphosphate. <br />  <br />   Citation: Huang G, Ulrich PN, Storey M, Johnson D, Tischer J, et al. (2014) Proteomic Analysis Acidocalcisome, Organelle Conserved Bacteria to <br />   Human Cells. PLoS Pathog 10(12): e1004555. doi:10.1371/journal.ppat.1004555 <br />   Editor: David Horn, University Dundee, United Kingdom <br />   Received August 15, 2014; Accepted November 5, 2014; Published December 11, 2014 <br />   Copyright: &#195;&#376; 2014 Huang et al. This open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Data Availability: The authors confirm data underlying findings fully available restriction. All relevant data paper its <br />   Supporting Information files. <br />   Funding: This work supported U.S. National Institutes Health (grant AI077538 RD). The funders role study design, data collection and <br />   analysis, decision publish, preparation manuscript. <br />   Competing Interests: The authors declared competing interests exist. <br />   * Email: rdocampo@uga.edu <br />  <br />  <br />  <br />  <br /> Introduction                                                                               Trypanosoma brucei belongs group organisms responsible <br />                                                                                         human African trypanosomiasis (sleeping sickness), and <br />    Acidocalcisomes originally observed bacteria                             nagana, cattle disease Africa. The best-studied life stages <br /> unicellular eukaryotes named metachromatic [1] volutin                           T. brucei procyclic forms (PCF), grow the <br /> [2] granules. Later, polymers orthophosphate called                             intestine tse tse fly vector, bloodstream forms (BSF), <br /> polyphosphate (polyP) identified high levels                       replicate blood mammalian host. Both stages <br /> organelles, acidocalcisomes called polyphosphate gran-                        grown laboratory possess acidocalcisomes, <br /> ules [3]. The <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> polyP varies                    abundant PCF [24]. Knowledge of <br /> as thousands residues [4]. The discovery diverse array                       protein composition acidocalcisomes facilitate under- <br /> transporters established acidocalcisomes real organelles                       standing physiological roles organelles. Among the <br /> present bacteria human cells [5]. Acidocalcisomes                          proteins localized acidocalcisomes T. brucei far the <br /> been described species bacteria [6,7], trypanoso-                       vacuolar proton pyrophosphatase (TbVP1), used <br /> matids [8&#226;&#8364;&#8220;10], apicomplexan parasites [11&#226;&#8364;&#8220;13], fungi [14,15],                           acidocalcisome marker subcellular fractionation studies <br /> algae [16,17], insect eggs [18,19], sea urchin eggs [20],                           [24]. In work, used iodixanol gradient centrifugation to <br /> chicken eggs [21]. Additionally, organelles present                   obtain TbVP1-enriched fractions examine acidocalcisome <br /> mammalian cells human platelets [22] mast cells                         proteome. We validated localization essentiality selected <br /> basophils [23], belong group organelles                            group proteins situ epitope tagging immunofluores- <br /> known lysosome-related organelles (LROs). However,                          cence assays specific antibodies, RNA interference <br /> acidocalcisome used organelles                           (RNAi) experiments, respectively. The results support impor- <br /> trypanosomatids [8,9], acidocalcisomes                               tant role organelles phosphate cation homeostasis, <br /> extensively studied organisms.                                                 calcium signaling. <br />  <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                                              1                       December 2014 | Volume 10 | Issue 12 | e1004555 <br />                                                                                                                         Acidocalcisome Proteome <br />  <br />  <br />  <br />   Author Summary                                                            purification, respectively. The organelles were <br />                                                                             enriched extent acidocalcisome preparation the <br />   Acidocalcisomes acidic organelles conserved                      iodixanol gradient glycosomes lysosomes, as <br />   bacteria human cells rich polyphosphate,                 evidenced 3- 2-fold purification hexokinase a- <br />   polymer orthophosphate units linked high-energy                     mannosidase, respectively. However, purification greatly <br />   phospho-anyhidride bonds. We acidocal-                    reduced second gradient isolation. Mitochondria (marked <br />   cisomes Trypanosoma brucei, belonging group                   succinate cytochrome c reductase) enriched these <br />   organisms produces African sleeping sickness                  fractions. The acidocalcisomes obtained iodixanol <br />   nagana, rich pumps, channels, transporters                     gradients enriched technique .60-fold <br />   involved phosphate cation homeostasis,                         cell compartments. <br />   calcium signaling. Proteomic analysis acidocalcisome                      Electron microscopy PCF acidocalcisome fraction (fraction 5) <br />   fractions expression genes epitope tags <br />                                                                             (S1B Figure) showed round organelles various sizes to <br />   validated presence number novel transporters, <br />                                                                             200 nm diameter, cases containing electron-dense <br />   RNA interference demonstrated essentiality of <br />   organelles.                                                         material (arrows arrowheads) appearance <br />                                                                             acidocalcisomes isolated using Percoll gradients [24]. When <br />                                                                             fixed, acidocalcisomes lose electron-dense content a <br /> Results                                                                     variable extent, resulting heterogeneous appearance. In <br />                                                                             contrast purity fraction 5, electron microscopy the <br />     We isolated acidocalcisomes modification isolation              15,0006g pellet used load gradient showed presence <br /> procedures described previously [17,25]. After grinding                mitochondria, glycosomes, flagella (S2C Figure), that <br /> silicon carbide break cells, lysates fractionated        pellet gradient showed contamination with <br /> differential centrifugation followed density-gradient ultracen-          glycosomes (S2D Figure). <br /> trifugation using high-density solutions iodixanol                After SDS-PAGE different pellets gradient centrifugation <br /> specially prepared condensing commercial iodixanol                   bands, enzymatic digestion trypsin, peptides were <br /> solution [17] (S1A Figure). Fractions collected               analyzed LC-MS/MS (see Materials Methods). Fig. 1G <br /> upper layers gradients. Composition fraction             shows typical Coomassie brilliant blue-stained gel proteins <br /> confirmed using enzymatic western blot analyses                     present fraction 5 approximate positions gel that <br /> organellar markers microscopic observation.                             putative acidocalcisome proteins studied work <br />     We analyzed proteome using acidocalcisomes obtained             have. A similar <span id='am-163' about='xsp:pattern' typeof='owl:Thing'>pattern</span> bands obtained other <br /> two different strategies. First, utilized pellet fraction       fractionations (S3 Figure). S3 Figure shows western blot <br /> the iodixanol gradient containing acidocalcisomes [25] (S1A           analyses preparations antibodies known <br /> Figure). Second, used acidocalcisome samples obtained               acidocalcisome proteins, TbVP1 [24] (S3A Figure, <br /> fraction 5 second ultracentrifugation step iodixanol          arrowheads), inositol-1,4,5-trisphosphate receptor (TbIP3R) [27] <br /> gradient protocol (Fig. 1A). Similar enzyme activity profiles          (S3B Figure, arrowhead), vacuolar soluble pyrophosphatase <br /> obtained independent fractionations. Since           (TbVSP) [28] (S3C Figure, arrow). Antibodies TbVP1 <br /> vacuolar pyrophosphatase (TbVP1) activity (measured amino-               reveal presence bands previously reported [29], <br /> methylenediphosphonate (AMDP)-sensitive pyrophosphatase ac-                 antibodies T. cruzi VSP extra cross-reacting bands, <br /> tivity [24,26] highly enriched fraction 5 second              (arrowhead) probably corresponds soluble <br /> iodixanol gradient ultracentrifugation, reporting                inorganic pyrophosphatase (Tb927.3.2840; MW 28.7 kDa). Anti- <br /> proteomic results purified fraction                 bodies TbIP3R lower molecular mass bands are <br /> experiments, acidocalcisome proteins described                probably hydrolysis products high molecular weight <br /> here detected acidocalcisome pellet obtained         protein. <br /> the iodixanol gradient centrifugation (results shown). <br />     Fig. 1 illustrates protein abundance (Fig. 1B)               Protein identification <br /> distribution markers acidocalcisomes (TbVP1) (Fig. 1C),                 We identified total 580 proteins (1% false discovery rate, <br /> mitochondria (succinate cytochrome c reductase) (Fig. 1D),                  protein probabilities .0.95) fraction 5 (ACCS1) <br /> glycosomes (hexokinase) (Fig. 1E), lysosomes (a-mannosidase)            second (ACCS2) experiments. The ACCS1 ACCS2 <br /> (Fig. 1F) percentage total recovered activity         datasets included 520 340 protein identifications, respectively <br /> three independent experiments using double iodixanol gradient               (proteins reported S1 Table; peptides S2 Table). When <br /> centrifugation PCF lysates. Fractions 3 5 PCF showed              variants similar proteins indistinguishable peptide data, <br /> the highest TbVP1 activity (Fig. 1C) contaminated             ProteinProphet [30] algorithm utilized ProteoIQ <br /> with glycosome (Fig. 1E) mitochondrial (Fig. 1D) markers            software treats identifications single protein (a protein <br /> the fractions. Very similar results enzymatic activities       &#226;&#8364;&#732;&#226;&#8364;&#732;group&#226;&#8364;&#8482;&#226;&#8364;&#8482;). For example, virtually identical isoforms <br /> were obtained western blot analyses different fractions           (Tb927.4.4380 Tb927.8.7980) vacuolar-H+-pyrophospha- <br /> (F1&#226;&#8364;&#8220;F7) (S2A&#226;&#8364;&#8220;B Figure) using antibodies proteins localized          tase (TbVP1) present T. brucei vary 6 826 <br /> to acidocalcisomes (TbVP1), mitochondria (voltage-dependent                 residues. Peptides proteins unequivocally identi- <br /> anion channel, TbVDAC), glycosomes (pyruvate, phosphate                     fied acidocalcisome datasets, report single <br /> dikinase, TbPPDK), lysosomes (Tbp67) (S2B Figure).                      identification. In instances, proteins may <br />     We evaluated purification method comparing                  present. Two nineteen annotated &#226;&#8364;&#732;&#226;&#8364;&#732;hypothet- <br /> marker enzymes activities 15,0006g fraction applied           ical&#226;&#8364;&#8482;&#226;&#8364;&#8482; T. brucei genome, represented in <br /> the iodixanol gradient acidocalcisome fraction             proteomic data available TriTrypDB.org (downloaded May 28, <br /> first second iodixanol gradients (S1A Figure Table 1).              2014). Of prior mass spectrometry evidence, three <br /> The pyrophosphatase yield ,10 ,5.0%,                    annotated hypothetical. The remaining proteins for <br /> yield protein 0.14 0.05%, 70 99-fold                  provide novel expression evidence annotated as <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                                  2                   December 2014 | Volume 10 | Issue 12 | e1004555 <br />                                                                                                                                        Acidocalcisome Proteome <br />  <br />  <br />  <br />  <br /> Fig. 1. Distribution iodixanol gradients organellar markers PCF trypanosomes. (A) Photograph showing bands obtained after <br /> the second iodixanol gradient centrifugation. Fraction 5 corresponds purified acidocalcisomes. (B) Protein distribution. (C) TbVP1 activity <br /> (measured AMDP-sensitive Pi release) concentrated fractions 3 5. (D) Mitochondrial marker distribution, succinate cytochrome c <br /> reductase. (E) Glycosomal marker distribution, hexokinase. (F) Lysosomal marker distribution, a-mannosidase. In (B&#226;&#8364;&#8220;F) y-axis indicates relative <br /> distribution; x-axis indicates fraction number; bars means 6 SD (as percentage total recovered activity) three <br /> independent experiments. (G) SDS-PAGE Fraction 5 representative acidocalcisome (ACCS) fractionation stained Coomassie brilliant <br /> blue. The relative intensities bands obtained bitmap file gel image shown right. Background subtracted. <br /> The approximate localization acidocalcisome proteins identified Table 2 shown. BenchMark protein markers shown left. <br /> doi:10.1371/journal.ppat.1004555.g001 <br />  <br />  <br /> frame-shift pseudogenes retrotransposon hot spot protein                        Subcellular localizations protein predicted (S4 <br /> a variant surface glycoprotein. Approximately 21% (120)                         Table) using series algorithms (pTARGET, targetP, WoLF- <br /> 580 proteins predicted transmembrane domains (S3 Table),                          PSORT, SLP-LOCAL). Both plant non-plant-optimized <br /> consistent estimates representation organisms [31].                   predictions performed means comparison, we <br /> Of 40 identifications (6.9% total), predicted signal peptides,                 report non-plant, targeting predictions. Approximately 20% <br /> 22 possessed putative transmembrane domains.                                      identifications nuclear, 17% cytosolic, ,9% are <br />    Annotated proteins proteomic dataset span broad range                      mitochondrial. Plasma membrane secretory predictions <br /> of metabolic groups. Transport-related proteins accounted                          represent ,5% 1%, respectively. Table 2 shows proteins <br /> ,15%. Among transporters pumps, vacuolar-H+-                            known localization acidocalcisomes T. brucei those <br /> pyrophosphatase, acidocalcisomal marker, identified                      established work (see below) proteins we <br /> dataset. Other well-represented metabolic groups dataset                        selected localization studies. Table 2 indicates these <br /> were energy metabolism (,14%), protein, lipid, carbohydrate,                       markers present proteomic datasets (labeled with <br /> nucleic acid metabolism (,36%), cell structure organi-                         asterisks). Of proteins identified proteomic analysis the <br /> zation (,18%).                                                                         subcellular fractions, selected proteins, previously <br />  <br />  Table 1. Purification acidocalcisomes iodixanol step gradients. <br />  <br />  <br />                                                                              Yield (%)                                    Purification-fold <br />  <br />                                                                              1st                    2nd                   1st                 2nd <br />  <br />  Protein (mg)                                                                0.14 (3)               0.05 (3) <br />  Pyrophosphatase*                                                            9.81 (3)               4.96 (3)              70                  99 <br />  Succinate cytochrome c reductase                                            0.12 (3)               0.02 (3)              1                   0.2 <br />  Hexokinase                                                                  0.46 (3)               0 (3)                 3                   0 <br />  a-mannosidase                                                               0.27 (3)               0.08 (3)              2                   1.6 <br />  <br />  Yield values percentages relative 15,0006g pellet fraction represent averages number preparations parentheses. <br />  *Pyrophosphatase activities 15,000 g pellet, 1st 2nd gradient acidocalcisome preparations 0.2260.09, 15.663.2, and <br />  22.662.1 mmol min21 mg21 protein, respectively (mean 6 SD). <br />  doi:10.1371/journal.ppat.1004555.t001 <br />  <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                                             3                   December 2014 | Volume 10 | Issue 12 | e1004555 <br />                                                    Table 2. Identification acidocalcisome protein candidates T. brucei, showing localization essentiality BSF PCF. <br />  <br />  <br />                                                                                                                                                                                                     Required growth   Confidence <br />                                                   TriTrypDB Gene ID                      Annotation (protein name)                 MW (kDa)         TMD             Localization                    BSF PCF         (peptides)   Ref. <br />                                                                                                     + <br />                                                   Tb927.4.4380 Tb927.8.7980              Vacuolar H -PPase (TbVP1)                 86               14              Ac                              BSF, PCF              0.99 (8)     [29] <br />                                                   Tb927.8.1180                           Vacuolar-Ca2+-ATPase (TbPMC1)             121              8               Ac                              BSF, PCF              0.96 (8)     [36] <br />                                                   Tb927.8.2770                           IP3 receptor (TbIP3R)                     345              5               Ac                              BSF, PCF              1 (22)       [27] study <br />                                                   Tb927.7.3900                           Vacuolar transporter chaperone 1          20               3               Ac                              PCF                   -            [40] study <br />                                                                                          (TbVtc1)* <br />                                                   Tb927.11.12220                         Vacuolar transporter chaperone 4          91               3               Ac                              BSF, PCF              1 (8)        [38,39] study <br />                                                                                          (TbVtc4) <br />  <br />  <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org <br />                                                   Tb927.11.7060 Tb927.11.7080            Vacuolar soluble PPase (TbVSP)*           47               0               Ac                              BSF, PCF              -            [28] <br />                                                   Tb927.11.10650                         Adaptor protein 3 subunit beta            100              0               Ac, Golgi, Endosomes            BSF, PCF              -            [63] <br />                                                                                          (TbAP-3b)* <br />                                                   Tb927.5.3610                           Adaptor protein 3 subunit delta           125              1               Ac, Golgi, Endosomes            BSF, PCF              -            [63] <br />                                                                                          (TbAP-3d)* <br />                                                   Tb927.5.1300                           Vacuolar H+-ATPase subunit (TbVAa)      89               6               Ac, Lysosome, Golgi             BSF, PCF              1 (5)        This study <br />                                                                                                     + <br />                                                   Tb927.5.550                            Vacuolar H -ATPase subunit d (TbVAd)      42               0               Ac, Lysosome, Golgi             BSF, PCF              1 (1)        This study <br />                                                   Tb927.3.800                            Vacuolar iron transporter (TbVIT1)*       30               3               Ac                              BSF, PCF              -            This study <br />                                                   Tb927.4.4960                           Zinc transporter (TbZnT)*                 50               5               Ac                              -                     -            This study <br />  <br />  <br />  <br />  <br /> 4 <br />                                                   Tb927.11.11160                         Phosphate transporter (TbPho91)           81               10              Ac                              -                     1 (1)        This study <br />                                                   Tb927.10.7020                          Acid phosphatase (TbAP)                   49               0               Ac                              -                     1 (9)        This study <br />                                                   Tb927.9.10340                          Polyamine transporter 1 (TbPOT1)*         54               11              Ac, Lysosome, Endosome          -                     -            This study <br />                                                   Tb927.11.6680                          Polyamine transporter 2 (TbPOT2)*         56               10              Lysosome                        -                     -            This study <br />                                                   Tb927.8.1870                           Golgi/lysosome glycoprotein 1             68               1               Lysosome, Golgi                 -                     1 (1)        This study <br />                                                                                          (TbGLP1) <br />                                                   Tb927.11.540                           ABC transporter (TbABCT)                  76               6               Mitochondria                    -                     1 (1)        This study <br />                                                   Tb927.10.3640                          Nuclear protein (TbNP)                    31               6               Nucleus                         -                     1 (1)        This study <br />                                                   Tb927.11.840.1                         Cation/proton antiporter (TbFTP)          81               15              Flagellar tip                   -                     -            This study <br />  <br />                                                   MW, molecular weight; TMD, transmembrane domains; Ac, acidocalcisome; IP3, inositol 1,4,5-trisphosphate; PPase, pyrophosphatase; -, tested. <br />                                                   *Proteins peptides acidocalcisome proteome. <br />                                                   doi:10.1371/journal.ppat.1004555.t002 <br />                                                                                                                                                                                                                                                             Acidocalcisome Proteome <br />  <br />  <br />  <br />  <br /> December 2014 | Volume 10 | Issue 12 | e1004555 <br />                                                                                                                           Acidocalcisome Proteome <br />  <br />  <br /> tested, validation (Table 2). Additionally, selected           ER, ER localization attributed artifact of <br /> other targets validation based properties justify           protein overexpression [40]. Although did detect <br /> acidocalcisome localization (Table 2).                                        peptides protein acidocalcisome proteome, we <br />                                                                               re-examined localization avoided pitfalls overexpres- <br /> Proteins involved Ca2+ signaling                                           sion abnormal distribution expressing 36 HA-tagged <br />    The acidocalcisomes trypanosomatids serve large acidic               TbVtc1 endogenous locus wild-type regulation. <br /> calcium stores [5,32], number proteins organelles           TbVtc1 perfectly co-localized TbVP1 acidocalcisomes <br /> can mediate Ca2+ signaling cell. The localization               (Fig. 3A). TbVtc4, positively identified acido- <br /> inositol 1,4,5-trisphosphate receptor (IP3R) trypanosomatids           calcisome proteome (S1 Table), co-localized acidocalci- <br /> been controversial, endogenous tagging IP3R T.                  somes TbVP1 (Fig. 3B), reported previously [38]. <br /> brucei 36HA epitope tag demonstrated specific localization             Western blot analyzes confirmed expression tagged <br /> to acidocalcisomes species [27]. The IP3R-HA did              proteins (Fig. 3E 3F). <br /> co-localize TbBiP, ER marker [33] clear reticular                 A putative phosphate transporter (TcPho1, TcCLB.508831.60) <br /> labeling. Proteomic analysis acidocalcisome fractions (unpub-              T. cruzi, originally annotated sodium/sulphate <br /> lished) contractile vacuole complex fractions [34] T. cruzi            symporter, localizes contractile vacuole intracellular <br /> also supported presence IP3R organelles. These                membranes epimastigotes T. cruzi [34]. The product the <br /> results corroborate punctate vacuolar localization T. cruzi            T. brucei homologue (TbPho91, Tb927.11.11160) co-localized <br /> reported TcIP3R authors [35]. These authors                      TbVP1 acidocalcisomes (Fig. 3C). Expression tagged <br /> suggested endoplasmic reticulum (ER) localization TcIP3R,               protein confirmed western blot analysis (Fig. 3G). <br /> but clear co-localization TbBiP antibodies presented                 Previous work [28] indicated presence vacuolar <br /> [35]. To confirm acidocalcisome localization TbIP3R,                soluble pyrophosphatase acidocalcisomes T. brucei (TbVSP, <br /> generated antibody IP3 binding region TbIP3R.               Tb927.11.7060 Tb927.11.7080). Although peptides corre- <br /> Immunofluorescence analysis using antibody confirmed                 sponding protein identified proteome, <br /> acidocalcisome localization, determined co-localization            antibodies protein reacted band ,50 kDa <br /> antibodies TbVP1 T. brucei (Fig. 2A). Western blot                 corresponding apparent molecular mass protein the <br /> analysis confirmed specificity antibodies (Fig. 2B S3B           acidocalcisome fraction (S3C Figure, arrow). <br /> Figure).                                                                         We investigated localization putative acid <br />    The acidocalcisome localization vacuolar Ca2+-ATPase                phosphatase (Tb927.10.7020; TbAP), present our <br /> (TbPMC1, Tb927.8.1180) [36] confirmed                         acidocalcisome fractions (S1 Table). The presence acid <br /> proteomic analysis (Table 2). Peptides Ca2+-ATPases                phosphatase activity T. rangeli acidocalcisomes detected <br /> (Tb927.3.3400, annotated sarcoplasmic-endoplasmic reticulum-               cytochemical methods [41], early work T. brucei <br /> type Ca2+-ATPase; Tb927.8.1160, annotated vacuolar-type                rhodesiense localized acid phosphatase activity lysosome- <br /> Ca2+-ATPase) detected (S3 Table),                     like vesicles probably correspond acidocalcisomes [42]. We <br /> probably indicate similarity peptides different ATPases               TbAP co-localized TbVP1 acidocalcisomes <br /> or contamination subcellular membrane fractions.                   (Fig. 3D). Western blot analysis confirmed expression the <br />                                                                               tagged protein (Fig. 3H). <br /> Proteins involved phosphate polyP metabolism <br />    The vacuolar transporter chaperone complex (VTC complex)                Proton pumps <br /> involved polyP synthesis yeast [37] trypanosomes [38,39].              Proton pumps maintain low pH inside acidocalcisomes. We <br /> Homologues yeast proteins (Vtc1p Vtc4p) present              identified TbVP1 vacuolar proton ATPase (V-H+- <br /> the genomes trypanosomatids, apicomplexan, fungi, algae                ATPase) proteomic analysis (Table 2). Early physiological <br /> but absent mammalian cells. GFP-tagged T. brucei vacuolar                  studies using bafilomycin A1, specific inhibitor V-H+-ATPase <br /> transporter chaperone 1 (TbVtc1) localized acidocalcisomes             [43], demonstrated V-H+-ATPase activity permeabilized T. <br />                                                                               brucei PCF trypanosomes [8]. This finding later confirmed in <br />                                                                               experiments intact cells [44] isolated acidocalcisomes <br />                                                                               [24]. All putative subunits pump present T. brucei <br />                                                                               genome (TriTrypDB.org, S5 Table), subunits, the <br />                                                                               putative H+-translocating subunit (TbVAa) putative H+ <br />                                                                               transporting subunit d (TbVAd), acidocalcisome <br />                                                                               proteomic analysis (Table 2). We tagged subunits a, d a <br />                                                                               36 HA tag excellent co-localization TbVP1 <br />                                                                               (Fig. 4A, S4A Figure). Additional punctate staining a <br />                                                                               d subunits did co-localize TbVP1 could <br />                                                                               correspond labeling Golgi complex endocytic <br />                                                                               pathway, V-H+-ATPase localizes eukary- <br /> Fig. 2. Immunofluorescence microscopy analysis TbIP3R. (A)                 otic cells. In agreement additional localization, found <br /> TbIP3R co-localized TbVP1 acidocalcisomes PCF trypano-             antibody reaction subunits co- <br /> somes (Pearson&#226;&#8364;&#8482;s correlation coefficient 0.8399). Yellow merge          localizes Golgi marker Golgi reassembly stacking <br /> images indicates co-localization. Scale bars = 10 mm. (B) Western blot        protein (TbGRASP) [45] (Fig. 4B S4B Figure) the <br /> analysis TbIP3R expressed PCF trypanosomes using polyclonal anti-       lysosomal markers cathepsin L (TbCATL), luminal lysosomal <br /> TbIP3R antibody. Lysate containing 30 mg protein PCF <br /> trypanosomes subjected SDS/PAGE 4&#226;&#8364;&#8220;15% polyacrylamide                cysteine peptidase, p67, lysosomal membrane glycoprotein <br /> gel, transferred nitrocellulose membrane. Molecular weight           [46] (Figs. 4C 4D, S4C Figure S4D Figure, <br /> markers left arrow shows band corresponding TbIP3R.             respectively). Western blot analyses confirmed expression of <br /> doi:10.1371/journal.ppat.1004555.g002                                         proteins (Fig. 4E S4E Figure). <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                                    5                   December 2014 | Volume 10 | Issue 12 | e1004555 <br />                                                                                                                                Acidocalcisome Proteome <br />  <br />  <br />  <br />  <br /> Fig. 3. Immunofluorescence microscopy western blot analysis proteins involved phosphorus metabolism. TbVtc1 (A), <br /> TbVtc4 (B), TbPho91 (C), TbAP (D) 36HA epitope-tagged situ co-localized TbVP1 acidocalcisomes PCF trypanosomes <br /> (Pearson&#226;&#8364;&#8482;s correlation coefficients 0.873, 0.734, 0.728, 0.680, respectively). Yellow merge images indicate co-localization. Scale bars (A&#226;&#8364;&#8220; <br /> D) = 10 mm. Western blot analyses monoclonal anti-HA showing labeling TbVtc1 (E), TbVtc4 (F), TbPho91 (G), TbAP (H) PCF <br /> trypanosomes. Molecular weight markers left, arrows corresponding bands identified. Tubulin (Tub) used loading control. <br /> doi:10.1371/journal.ppat.1004555.g003 <br />  <br />  <br /> Other transporters                                                             homologues present acidocalcisomes species. For <br />    Several acidocalcisome proteins trypanosomatids                 example, Tb927.10.3640 predicted TMD annotated <br /> with potential localization acidocalcisomes investigat-           TriTrypDB.org hypothetical protein. The C terminus was <br /> ed. Since iron detected acidocalcisomes T. cruzi                tagged 36 HA tag using homologous recombination with <br /> [47], Phytomonas spp. [48,49], Leishmania amazonensis [50],                endogenous locus. Surprisingly, protein showed nuclear <br /> we tagged hypothetical protein (Tb927.3.800) similarity              membrane localization (S5A Figure), western blot analysis <br /> vacuolar iron transporters (VIT). This protein co-localized               identified single band ,35 kDa (predicted molecular mass, <br /> TbVP1 (Fig. 5A), western blot analysis PCF trypanosome                  32 kDa, S5C Figure). Interestingly, protein previously <br /> lysates showed single band using anti-HA antibodies (Fig. 5C).               identified nuclear proteome T. brucei [52] <br /> We tagged putative metal-ion (zinc) transporter                            An ABC transporter identified acidocalcisomes <br /> (Tb927.4.4960) homologue T. cruzi (TcCLB.511439.50)                   Cyanidoschyzon merolae [17] Tb927.11.540, listed a <br /> occurs acidocalcisomes [51]. Fig. 5B shows HA-tagged                   putative ABC transporter predicted TMD, enriched <br /> Tb927.4.4960 co-localized TbVP1, western blot analyses                T. brucei acidocalcisome proteome (S1 Table). However, <br /> (Fig. 5D) confirmed expression.                                            antibodies HA co-localized MitoTracker the <br />                                                                                mitochondrion PCF (S5B Figure), western blot analysis <br /> Proteins proteome localize                               showed strong band ,75 kDa compatible predicted <br /> acidocalcisomes                                                                molecular mass 76 kDa. A second band ,60 kDa, be <br />   Several proteins enriched acidocalcisome proteome                     cleavage mitochondrial targeting signal 97 amino <br /> possess transmembrane domains (TMD),                             acids (S5D Figure). <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                                     6                    December 2014 | Volume 10 | Issue 12 | e1004555 <br />                                                                                                                             Acidocalcisome Proteome <br />  <br />  <br />  <br />  <br /> Fig. 4. Immunofluorescence microscopy western blot analysis V-H+-ATPase subunit PCF trypanosomes. Epitope-tagged V- <br /> H+-ATPase subunit co-localizes TbVP1 acidocalcisomes (A), TbGRASP Golgi complex (B) TbCATL (C) p67 (D) <br /> and lysosomes (Pearson&#226;&#8364;&#8482;s correlation coefficients 0.631, 0.539, 0.804, 0.754, respectively). Yellow merge images indicate co-localization <br /> (also shown arrows (B&#226;&#8364;&#8220;D)). Scale bars (A&#226;&#8364;&#8220;D) = 10 mm. (E) Confirmation tagging western blot analyses monoclonal anti-HA PCF <br /> trypanosomes. HRP-conjugated goat anti-mouse used secondary antibody. Magic Mark XP (Invitrogen) used molecular weight <br /> marker arrow shows band corresponding TbVAa. Tubulin (Tub) used loading control (bottom panel). <br /> doi:10.1371/journal.ppat.1004555.g004 <br />  <br />    Tb927.8.1870 Golgi/lysosome glycoprotein 1 (TbGLP1)                   TriTrypDB.org, potential Na+/H+ exchanger in <br /> reported localize Golgi complex, multivesicular                     TransportDB. Interestingly, HA-tagged TbFTP localizes the <br /> lysosomes, unidentified small vesicles [53]. As detected            distal tip flagellum PCF, does co-localize with <br /> localization acidocalcisome proteins Golgi                    acidocalcisomes (S7D Figure). Western blot analysis identified one <br /> lysosomes (Fig. 4 S4 Figure) tagged C terminus                  band absent wild type cells (S7F Figure). Few proteins, among <br /> TbGLP1 36 HA. The small vesicles previously described                    adenylyl cyclases [58], calpain-like protein TbCALP.1.3 <br /> [53] apparently acidocalcisomes TbGLP1 does                [59], kinesin motor Kif13-2 [60], unknown antigen, the <br /> co-localize TbVP1 (S6A Figure). Consistent this,                    flagellar protein FLAM8 [61], previously reported to <br /> antibodies HA co-localized TbGRASP (S6B Figure),                 exhibit localization flagellar tip T. brucei. In addition, a <br /> TbCATL (S6C Figure) p67 (S6D Figure). Western blot                        cation channel does occur distal tip flagellum T. cruzi <br /> analysis showed band ,90 kDa, close apparent                      [62] presence channels exchangers this <br /> molecular mass native protein [53] (S6E Figure).                       localization compatible proposed role the <br />                                                                               flagellum environmental sensor. <br /> Proteins potential acidocalcisome localization <br />    Acidocalcisomes rich basic amino acids, potentially             Requirement newly discovered acidocalcisome <br /> polyamines balance anionic charges polyphosphate, occurs             proteins normal growth <br /> in yeast vacuole [54]. We investigated localization HA-               We reported number genes encoding <br /> tagged putative polyamine transporters TbPOT1 (Tb927.9.10340)                 acidocalcisome proteins TbVP1 [29], TbPMC1 [36], <br /> and TbPOT2 (Tb927.11.6680). TbPOT1 partially co-localizes                     TbIP3R [27], TbVtc1 [40], TbVtc4 [38,39], TbVSP [28], and <br /> with acidocalcisomes (S7A Figure), lysosomes (S7B&#226;&#8364;&#8220;C                  AP-3 b d subunits [63] essential growth BSF <br /> Figure). TbPOT2, contrast, did co-localize Golgi                  and/or PCF trypanosomes (Table 2). We selected some <br /> complex (S8A Figure) showed exclusive lysosomal                        newly identified acidocalcisome proteins investigate their <br /> localization (S8B&#226;&#8364;&#8220;C Figure). Western blot analyses confirmed              requirement growth. Knockdown TbVAa TbVAd by <br /> expression tagged proteins (S7E Figure S8E Figure,                 induction double-stranded RNA resulted growth defects in <br /> respectively).                                                                BSF PCF trypanosomes (Fig. 6A, 6B, 6D, 6E, <br />    Biochemical evidence presence Na+/H+ exchanger                respectively), 8164% 6963% reduction the <br /> in acidocalcisomes different trypanosomatids [55] including T.             number cells, respectively. Northern blots (analysis performed <br /> brucei PCF [56,57] presented. We investigated              ImageJ software) showed mRNA down-regulated by <br /> the localization Tb927.11.840.1, 15 predicted TMD                73&#226;&#8364;&#8220;96% 2 4 d RNAi BSF PCF trypanosomes, <br /> and annotated putative cation/proton antiporter                    respectively (Fig. 6C 6F). <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                                    7                    December 2014 | Volume 10 | Issue 12 | e1004555 <br />                                                                                                                            Acidocalcisome Proteome <br />  <br />  <br />                                                                                brucei (data TriTrypDB.org, accessed May 28, 2014). This <br />                                                                                confirms relevance subcellular proteomics method of <br />                                                                                choice identification larger numbers proteins than <br />                                                                                cell proteomics [51]. <br />                                                                                   Subcellular fractionation partially purifies cellular compo- <br />                                                                                nents contaminants. This contamination the <br />                                                                                abundance proteins, adhesive properties others, and <br />                                                                                junctions connect organelles each <br />                                                                                [65]. In regard previously discussed [66] close <br />                                                                                association acidocalcisomes mitochondria trypanosomes <br />                                                                                [67], association important Ca2+ signaling, could <br />                                                                                explain contamination fractions mitochondrial <br />                                                                                membrane proteins. It essential mass spectromet- <br />                                                                                ric analysis validated vivo expression tagged proteins. <br />                                                                                Only studies date [34,51,61,68] implemented a <br />                                                                                method verify proteomes trypanosomatid parasites. To <br />                                                                                validate dataset, expressed number proteins the <br />                                                                                acidocalcisome proteome HA-fusion proteins. We complement- <br />                                                                                ed set proteins selected proteins known <br />                                                                                localizations acidocalcisomes species, with <br />                                                                                proteins potentially present acidocalcisomes <br />                                                                                basis knowledge organelle. Interestingly, <br />                                                                                proteins previously localized acidocalcisomes were <br />                                                                                absent dataset. These notable absences dataset <br />                                                                                suggest low expression levels. <br />                                                                                   The proteins localized acidocalcisomes (Fig. 7) belong <br />                                                                                groups: proteins involved Ca2+ signaling, phosphate <br /> Fig. 5. Immunofluorescence microscopy western blot                         homeostasis, membrane transport. The acidocalcisome <br /> analyses metal ion transporters. Epitope-tagged TbVIT1 (A) <br /> and TbZnT (B) co-localize TbVP1 acidocalcisomes <br />                                                                                localization IP3R [27] confirmed using antibodies <br /> (Pearson&#226;&#8364;&#8482;s correlation coefficients 0.6879 0.7604, respectively).       IP3 binding region receptor, recognized a <br /> Yellow merge images indicate co-localization. Scale bars A&#226;&#8364;&#8220;             band 345 kDa corresponds apparent molecular mass <br /> B = 10 mm. Tagging HA confirmed western blot analyses           receptor (343 kDa). The antibody marked additional <br /> TbVIT1 (C) TbZnT (D) using anti-HA antibodies. Markers          band ,80 kDa likely corresponds hydrolysis product, <br /> left side, arrows corresponding bands. Tubulin (Tub)          band weak immunoblots total cell lysates. <br /> used loading control (bottom panel). <br /> doi:10.1371/journal.ppat.1004555.g005                                          Although TbIP3R T. cruzi suggested localize ER <br />                                                                                [35], IFA results T. cruzi convincing given that <br />                                                                                endogenously tagged T. brucei IP3R localizes acidocalcisomes <br />    Knockdown TbVIT1 BSF PCF trypanosomes                        [27]. Further work necessary confirm localization other <br /> (Fig. 6G 6H) resulted growth defects 4466%                   trypanosomatids. The identification mechanism Ca2+ <br /> 4163% reduction number cells 2 4 d                      uptake (TbPMC1) Ca2+ release (TbIP3R) acidocalcisomes <br /> tetracycline addition BSF PCF trypanosomes, respectively.               underscore relevance organelles Ca2+ signaling. <br /> Knockdown TbZnT weakly affected growth PCF                         The acidocalcisome localization components VTC <br /> trypanosomes (Fig. 6J 6K). Northern blot analyses showed                   complex involved synthesis polyP [38,40] confirmed, and <br /> that mRNA downregulated cases (Fig. 6I 6L).                 excellent co-localization TbVtc1 TbVtc4 TbVP1 <br />                                                                                acidocalcisomes suggest previously reported localization of <br /> Discussion                                                                     TbVtc1 ER [40] consequence overexpression <br />                                                                                exogenous locus. <br />    We report proteomic analysis subcellular fractions                 A phosphate transporter (TbPho91) annotated sulfate/ <br /> enriched acidocalcisomes T. brucei. These fractions                sodium symporter, encoding putative Saccharomyces <br /> enriched proteins previously demonstrated localize                    cerevisiae Pho91p orthologue (S9A Figure) localized the <br /> acidocalcisomes like TbVP1 [24], TbPMC1 [36], TbVtc4 [38],                     acidocalcisomes. Pho91p, localized vacuole proposed <br /> and TbIP3R [27]. Our protocol yields fractions resolved              involved exporting Pi vacuole cytosol [69]. <br /> organelle markers mitochondria (succinate cytochrome c                     The orthologue identified T. cruzi (TcCLB.508831.60) shares <br /> reductase, TbVDAC), glycosomes (hexokinase, TbPPDK)                        65% amino acid identity TbPho91, localized to <br /> lysosomes (a-mannosidase, Tbp67). We 580 identifications                  contractile vacuole membranes parasite [34]. <br /> in fractions highly enriched TbVP1 activity. Membrane                       The ORF TbPho91 encodes predicted, 728 amino acid <br /> proteins challenging proteomic analysis, dataset               protein apparent molecular weight 81 kDa, nine <br /> includes relatively high representation membrane proteins                 transmembrane domains, N-terminal regulatory SPX domain <br /> (21% fraction 5). A published plasma membrane proteome T.                anion-permease domain present anion <br /> brucei contains lower proportion membrane proteins (16.1%                 transporters. The recognized polypeptide apparent <br /> of 1,536 proteins, [64], suggesting fractionation success-            molecular mass ,70 kDa and, T. brucei Pho91 possesses <br /> fully enriched proteins potential, membrane-related func-                 transmembrane domains, size discrepancy the <br /> tions. Additionally, proteomic analysis confirmed expression            expected (99 kDa) observed molecular mass be <br /> five proteins previously undetected cell analyses T.               attributed usual anomalous migration hydrophobic <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                                     8                   December 2014 | Volume 10 | Issue 12 | e1004555 <br />                                                                                                                                 Acidocalcisome Proteome <br />  <br />  <br />  <br />  <br /> Fig. 6. Effect inhibition expression acidocalcisome genes tetracycline-induced RNAi cell growth. (A&#226;&#8364;&#8220;B), (D&#226;&#8364;&#8220;E), (G&#226;&#8364;&#8220;I), <br /> and (J&#226;&#8364;&#8220;L) growth BSF (left panels) PCF (right panels) trypanosomes TbVAa, TbVAd, TbVIT1, TbZnT RNAi absence (2Tet, black <br /> lines) presence (+Tet, red lines) 1 mg/ml tetracycline indicated number days, respectively. Values means 6 SD (n = 3&#226;&#8364;&#8220;4). (C), (F), (I) <br /> and (L) northern blot analyses TbVAa, TbVAd, TbVIT1, TbZnT RNAi absence (0) presence (2 4 days) tetracycline, respectively. <br /> Tubulin shown loading control. Markers shown right. <br /> doi:10.1371/journal.ppat.1004555.g006 <br />  <br />  <br /> proteins SDS gels [70]. If TbPho91 functions orthologue               signal peptide belongs histidine phosphatase <br /> in S. cerevisiae [69], involved release Pi           superfamily (TriTrypDB.org). Catalytic activity superfamily <br /> the acidocalcisomes.                                                            centers phosphorylation dephosphorylation histidine <br />    The acid phosphatase (TbAP) soluble enzyme                      residue follows b-strand protein. A conserved <br /> identified molecular level acidocalcisomes trypanoso-              Arg-His-Gly (RHG) triad proposed contain the <br /> matids. The gene (Tb927.10.7020) encodes 50 kDa protein                  phosphorylated histidine [71] conserved TbAP. The <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                                      9                     December 2014 | Volume 10 | Issue 12 | e1004555 <br />                                                                                                                            Acidocalcisome Proteome <br />  <br />  <br />  <br />  <br /> Fig. 7. Schematic representation acidocalcisome T. brucei. The identified acidocalcisome proteins corresponding pumps, <br /> exchangers, transporters protein complexes Table 2 shown model. The newly identified acidocalcisome proteins study are <br /> marked blue. <br /> doi:10.1371/journal.ppat.1004555.g007 <br />  <br /> LTXXG motif region b1 b2 conserved                   described Arabidopsis thaliana [76] yeast Ca2+- <br /> [71]. It interesting note acid phosphatases, like           sensitive cross-complementer 1 (CCC1) [77] (S9C Figure). These <br /> tartrate-resistant purple acid phosphatase (S9B Figure)                 transporters localized plant yeast vacuole, <br /> exopolyphosphatase activity [72] work needed                respectively, involved iron manganese <br /> to investigate exopolyphosphatase activity detected              sequestration vacuoles. The present iron transporter <br /> acidocalcisomes [73] enzyme.                                     agreement detection iron acidocalcisomes of <br />    The presence V-H+-ATPase activity defining               different species [78]. <br /> properties led identification acidocalcisomes                    Tb9274.4960 member cation diffusion facilitator <br /> trypanosomes [8,9]. The enzyme activity later localized                  (CDF) family [79], includes mammalian zinc transporters <br /> acidocalcisomes different unicellular eukaryotes [5],            ZnT4 [80], S. cerevisiae ZRC1 [81], A. thaliana metal <br /> the work studying localization enzyme using                    tolerance protein 1 (AtMTP1) [82], Escherichia coli YiiP <br /> epitope-tagged subunits. V-H+-ATPases multisubunit proton                   (EcYiiP) [83] (S10A Figure). These transporters function as <br /> pumps composed subcomplexes. The peripheral V1                           antiporters Zn, Cd, Co and/or Ni protons. All known <br /> complex consists subunits (A H) responsible              CDF domains proteins contain 6 TMD share characteristic <br /> ATP hydrolysis, membrane-integral V0 complex (a, c,                 motifs, CDF family-specific signature sequence the <br /> c9, c0, d, e subunits) responsible proton translocation              start second membrane-spanning helix (TM2), long <br /> <span id='am-4' about='protege:FROM' typeof='owl:Thing'>from</span> cytosol lumen endomembrane compartments                    C-terminus [82]. The presence zinc transporter in <br /> [74]. Epitope tagging membrane integral V0 complex                       agreement abundant presence zinc acidocalci- <br /> subunits (a d) identified localization multisubunit             somes, detected X-ray microanalyses different prokaryotes <br /> complex acidocalcisomes, lysosomes, Golgi complex. This                  eukaryotes [5,78]. <br /> is contrast T. cruzi P-type H+-ATPase                        We report localization proteins previously <br /> involved acidification endocytic pathway [75]. As occurs              investigated, mitochondrial ABC transporter (Tb <br /> with organisms studied date, enzyme essential                927.11.540) (TbABCT), flagellar cation/proton antiporter <br /> parasite growth survival. It quite interesting        (Tb927.11.840.1) flagellar tip protein (TbFTP), nuclear <br /> some heterogeneity TbVP1 stain compared                     periphery protein (Tb927.10.3640) (TbNP), lysosome/acidocal- <br /> markers, suggest class               cisome putative polyamine transporter (Tb927.9.10340) (Tb <br /> of compartment differential compositions. This                POT1) (S10B Figure), lysosomal putative polyamine <br /> indicate functional differences maturation/degradation             transporter (Tb927.11.6680) (TbPOT2). We confirmed the <br /> these compartments.                                                             Golgi lysosomal localization TbGLP1 [53]. <br />    Two new metal ion transporters identified. Tb927.3.800                  Finally, report requirement growth subunits of <br /> an orthologue vacuolar iron transporter (VIT1) originally                V-H+-ATPase (TbVAa TbVAd), orthologue of <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                                     10                   December 2014 | Volume 10 | Issue 12 | e1004555 <br />                                                                                                                          Acidocalcisome Proteome <br />  <br />  <br /> a vacuolar iron transporter (TbVIT1) PCF BSF                     (Middleton, WI). [a-32P]dCTP (3,000 Ci mmol21) from <br /> trypanosomes, supporting role acidocalcisomes parasite             Perkin Elmer (Waltham, Massachusetts). Rabbit antibodies against <br /> growth survival.                                                         T. brucei vacuolar H+-pyrophosphatase (TbVP1) [29] gift <br />    The identification novel acidocalcisome proteins provides              Dr. Norbert Bakalara (Ecole Nationale Supe&#194;&#180;rieure de <br /> useful insights biogenesis organelles. A common            Chimie Montpellier, Montpellier, France). Mouse monoclonal <br /> feature acidocalcisome proteins validated endogenous           antibody HA (purified HA.11 clone 16B12) purchased <br /> expression HA-tags study presence              Covance Inc. (Princeton, NJ). Rat monoclonal antibody <br /> more tyrosine-based, sorting signals YXX&#195;&#732; (&#195;&#732; corre-                 HA (clone 3F10) Complete, EDTA-free protease <br /> sponds hydrophobic amino acid) consensus motif (see S7                 inhibitor cocktail tablets purchased Roche Applied <br /> Table). The m subunits of adaptor protein (AP)          Science (Indianapolis, IN). The pMOTag4H vector [89] gift <br /> complexes bind motif [84]. In regard, AP-3 required          Dr. Thomas Seebeck (University Bern, Bern, Switzerland). <br /> for biogenesis acidocalcisomes [63]. All proteins          The p2T7Ti vector [90] gift Dr. John Donelson <br /> we validated expression possess generic N-glycosylation              (University Iowa, Iowa City, IA). Antibody GRASP [45] <br /> motifs, phosphothreonine modules binding FHA domains                    gift Dr. Graham Warren (Max F. Perutz Laboratories, <br /> large aliphatic amino acids pT+3 position casein           Vienna, Austria), antibodies p67 TbCATL [46] <br /> kinase 2 (CK2), glycogen synthase kinase b (GSK3b) NEK2                  gift Dr. James Bangs (University Wisconsin, <br /> (never mitosis (NimA)-related kinases 2) phosphorylation sites            Madison, WI). Rabbit polyclonal antibody TbVDAC was <br /> (see S7 Table). A variety kinases GSK3b localize           gift Dr. Minu Chadhuri (Meharry Medical College, TN). <br /> Golgi regulate post-Golgi membrane trafficking [85]. These               Anti T. brucei pyruvate, phosphate dikinase (PPDK)-producing <br /> findings help guiding future studies biogenesis         mouse hybridoma culture supernatant gift Dr. Fre&#194;&#180;de&#194;&#180;ric <br /> organelles.                                                                  Bringaud (University Bordeaux, France). The enhanced <br />    In summary, addition validate expression protein         chemiluminescence (ECL) detection kit purchased from <br /> level number important genes identify localization           Amersham Biosciences (GE Healthcare Life Sciences, Piscataway, <br /> of proteins previously studied, identified new                NJ), Pierce ECL Western blotting substrate Thermo <br /> acidocalcisome proteins using strategy complementing subcellu-             Fisher Scientific Inc. (Rockford, IL). The Bradford protein assay <br /> lar proteomics bioinformatics localization using           reagent, Precision Plus Protein WesternC pack, 4&#226;&#8364;&#8220;15% polyacryl- <br /> situ epitope-tagged proteins specific antibodies, RNAi            amide Ready gels, Zeta-Probe GT Genomic Testing blotting and <br /> functional validation. Four proteins newly identified           nitrocellulose membranes Bio-Rad (Hercules, CA). <br /> acidocalcisome proteins, identification facilitate            AMAXA Human T-cell Nucleofector kit purchased from <br /> further studies elucidate roles organelle T. brucei        Lonza (Koln, Germany). Prime-a Gene Labeling System from <br /> physiology.                                                                  Promega (Madison, WI). QIAquick gel extraction kit and <br />                                                                              MinElute PCR purification kit, Ni-NTA agarose, Protein G <br /> Materials Methods                                                        Agarose Resins Qiagen (Valencia, CA). The primers <br />                                                                              purchased Integrated DNA Technologies (Coralville, <br /> Ethics statement                                                             IA). All reagents analytical grade Sigma (St. <br />   Mice experiments work followed reviewed                      Louis, MO). <br /> approved protocol Institutional Animal Care Use <br /> Committee (IACUC). Animal protocols followed US Govern-                  Subcellular fractionation acidocalcisomes 1-D gel <br /> ment principles Utilization Care Vertebrate                   electrophoresis <br /> animals. The University Georgia IACUC approved animal                    Fractions enriched acidocalcisomes isolated purified <br /> protocol (Protocol number A2012-3-010).                                      using iodixanol gradient centrifugations (S1 Figure). PCF <br />                                                                              trypanosomes (3&#226;&#8364;&#8220;4 g wet weight) washed twice Buffer A <br /> Cell culture                                                                 (116 mM NaCl, 5.4 mM KCl, 0.8 mM MgSO4, 50 mM Hepes, <br />    T. brucei PCF trypanosomes (wild type 29-13 strains)              pH 7.2) 5.5 mM glucose. The parasites washed in <br /> BSF (single marker (SM) strains) used. PCF 29-13 (T7RNAP                cold isolation buffer (125 mM sucrose, 50 mM KCl, 4 mM <br /> NEO TETR HYG) co-expressing T7 RNA polymerase Tet                        MgCl2, 0.5 mM EDTA, 20 mM Hepes, 3 mM dithiothreitol <br /> repressor gift Dr. George A. M. Cross (Rockefeller               (DTT) supplied Complete, EDTA-free, protease inhibitor <br /> University, NY) grown SDM-79 medium [86],                        cocktail (Roche) prior lysis silicon carbide isolation <br /> supplemented hemin (7.5 mg/mL) 10% heat-inactivated                 buffer. Silicon carbide cell debris eliminated series <br /> fetal bovine serum, 27uC presence G418 (15 mg/              low speed centrifugations (100 g 5 min, 300 g 10 min, <br /> ml) hygromycin (50 mg/ml) maintain integrated genes               1,200 g 10 min). The supernatant centrifuged at <br /> for T7 RNA polymerase tetracycline repressor, respectively               15,000 g 10 min, pellet resuspended 1 ml <br /> [87]. BSF trypanosomes (single marker strain) gift          isolation buffer applied 34% step discontinuous <br /> Dr. G.A.M. Cross grown 37uC HMI-9 medium                      gradient 4 ml steps 20, 24, 28, 34, 37 40% iodixanol <br /> [88] supplemented 10% fetal bovine serum (FBS), 10% serum               (diluted isolation buffer). The gradient centrifuged at <br /> plus (JRH Biosciences, Inc.), 2.5 mg/ml G418.                            50,000 g Beckman JS-24.38 rotor 60 min 4uC, and <br />                                                                              fractions collected top. The pellet resuspended <br /> Chemicals reagents                                                       700 ml isolation buffer applied 27% step another <br />    TRIzol reagent, Taq polymerase, Magic Marker protein                      discontinuous gradient iodixanol, 1.4 ml isolation buffer <br /> standards, BenchMark protein ladder, Mito-Tracker Red,                   containing 10% w/v sucrose over-layered 1 ml <br /> Alexa-conjugated secondary antibodies purchased Life               steps 27, 62 80% iodixanol, diluted 90% <br /> Technologies (Carlsbad, CA). The expression vector pET32 EK/                 w/v iodixanol isolation buffer. To prepare 90% w/v <br /> Lic purchased Novagen (Madison, WI). E. coli                        iodixanol, 60% w/v iodixanol solution (Optiprep) dried <br /> OverExpression C43 (DE3) strain purchased Lucigen                   completely 70uC resuspended isolation buffer. After <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                                  11                   December 2014 | Volume 10 | Issue 12 | e1004555 <br />                                                                                                                       Acidocalcisome Proteome <br />  <br />  <br /> the second gradient centrifugation 50,000 g 60 min 4uC,         algorithm utilized ProteoIQ software combines hit proteins <br /> fractions collected top, washed twice isolation        degenerate peptide fingerprints single identification (a <br /> buffer centrifugation 20,000 g 15 min 4uC,               protein &#226;&#8364;&#732;&#226;&#8364;&#732;group&#226;&#8364;&#8482;&#226;&#8364;&#8482;) generates group probability. In cases, <br /> analyzed various organelle marker enzyme assays. The pro-              individual proteins actually present in <br /> tein concentration quantified Bradford assay using               sample. <br /> SpectraMax Microplate Reader. After washing fraction 5, <br /> containing highest vacuolar-H+-pyrophosphatase (PPase)                Bioinformatic analysis mass spectrometry results <br /> activity (Fig. 1A), resuspended 200-ml isolation buffer.           Subcellular fractionation protocols enrich samples target <br /> Aliquots purified acidocalcisome suspension separated         organelles produce somewhat heterogeneous preparations <br /> on 4&#226;&#8364;&#8220;15% SDS-PAGE gels stained Coomassie brilliant               containing material cell compartments readily <br /> blue, immunoblotted acidocalcisome markers,                  detected exquisitely sensitive tools mass spectrometry. <br /> precipitated electron microscopy, used proteomic               To identify likely contaminants non-acidocalcisomal com- <br /> analysis. Chromatograms protein bands SDS-PAGE                  partments proteomic dataset, used series subcellular <br /> gels obtained background subtraction using ImageJ              prediction algorithms: TargetP 1.1 [91], pTARGET [92], SLP- <br /> (National Institute Health, Bethesda, MD).                             LOCAL [93], WoLF-PSORT [94]. Data these <br />                                                                           algorithms processed using Perl scripts MySQL <br /> In-gel tryptic digestion                                                  database screen proteins prediction confidence <br />    Gel lanes washed twice ddH2O 15 min cut           thresholds 80%. Final consensus predictions subcellular <br /> 10 equal slices. Proteins reduced 10 mM dithiothreitol          localization individual protein hits assigned or <br /> (DTT)/100 mM ammonium bicarbonate solution 65uC 1 h                algorithms agreed. In event mass spectrometry <br /> and carboxyamidomethylated 55 mM iodoacetamide/                      data identified protein group member, <br /> 100 mM ammonium bicarbonate 1 h room temperature                   consensus predictions individual proteins combined a <br /> in dark. Enzymatic digestion performed porcine               group consensus prediction predictions two <br /> trypsin (1:50, Promega, Madison, WI) 37uC overnight. Tryptic           individual proteins agreed. The membrane topology presence <br /> peptides extracted times 100 ml 50% acetonitrile/        signal peptides predicted using following tools: <br /> 0.1% formic acid. Combined extracts evaporated dryness            SignalP3 [95], TMHMM2.0c [96], HMMTOP2.1 [97] and <br /> and stored 220uC mass spectrometry analysis.                     PolyPhobius [98,99] (accessed May 28, 2014). In addition, we <br />                                                                           used published data annotated proteins validate our <br /> Mass spectrometry                                                         data. <br />    Peptides resuspended 20 ml 2% acetonitrile/0.1% <br /> formic acid. Data acquired using Agilent 1100 Capillary            Enzyme assays <br /> LC (Palo Alto, CA) 0.26150 mm Halo Peptide ES-                 Pyrophosphatase (PPase) activity (acidocalcisome marker) was <br /> C18 capillary column packed 2.7 mm diameter superficially            assayed measuring phosphate (Pi) release using malachite <br /> porous particles (Advanced Materials Technology, Inc., Wilming-           green assay [100] modifications. Briefly, reactions <br /> ton, DE). On-line MS detection used Thermo-Fisher LTQ ion             contained 130 mM KCl, 2 mM MgCl2, 10 mM Hepes, pH 7.2, <br /> trap (San Jose, CA) Michrom (Michrom Bioresources,                 100 mM PPi, 0.5 mg gradient fraction 40 mM <br /> Auburn, CA) captive spray interface. Sample analysis utilized         aminomethylenediphosphonate (AMDP). After incubation at <br /> LTQ divert valve fitted EXP Stem Trap 2.6 mL cartridge            30uC 10 min, reaction stopped addition an <br /> packed Halo Peptide ES-C18 2.7 mm diameter superficially             equal volume freshly prepared mixture parts 0.045% <br /> porous particles (Optimize Technologies, Oregon City, OR).                malachite 4.2% ammonium molybdate. The <br /> Sample injection volume 8 ml. Gradient conditions increased           absorbance (A) 660 nm read using microplate reader. <br /> the concentration mobile phase B 6% 75% B                 The Pi released determined comparison a <br /> 90 min. Mobile phase A consisted 99.9% water, 0.1% formic              standard curve. AMDP used distinguish vacuolar <br /> acid 10 mM ammonium formate. Mobile phase B contained                 (sensitive) soluble (insensitive) PPase activities. The specific <br /> 80% acetonitrile, 0.1% formic acid 10 mM ammonium                     activity TbVP1 defined mmol Pi released/min6mg of <br /> formate. Mobile phases used formic acid, ammonium formate             protein. <br /> acetonitrile Sigma-Aldrich (St. Louis, MO).                             Succinate-cytochrome C reductase activity (mitochondria <br />    Raw tandem mass spectra converted mzXML files,            marker) assayed described previously [101], using 3 mM <br /> into mascot generic files (MGF) Trans-Proteomic Pipeline          succinate (pH 7.2) substrate following reaction <br /> (Seattle Proteome Center, Seattle, WA). MGF files searched           containing 0.1 mM cytochrome C (Cyt C), 0.3 mM KCN, <br /> using Mascot (Matrix Scientific Inc, Boston, MA) separate         40 mM Hepes pH 7.5, 10 ml gradient fraction 30uC at <br /> target decoy databases obtained National Center          550-540 nm microplate reader. Hexokinase (glycosome <br /> Biotechnology Information (NCBI). The target database contained           marker) assayed described previously [102]. The reaction <br /> all T. brucei protein sequences decoy database contained          mixtures (100 ml) contained 10 mM D-glucose, 0.6 mM ATP, <br /> the reversed sequences target database. Mascot settings          0.6 mM NADP+, 10 mM MgCl2, 2.5 units/ml glucose-6-phos- <br /> were follows: tryptic enzymatic cleavages allowing 2         phate dehydrogenase, 50 mM potassium Hepes, pH 7.8. The <br /> missed cleavages, peptide tolerance 1000 parts-per-million,            oxidation NADP monitored 30uC microplate <br /> fragment ion tolerance 0.6 Da, fixed modification               reader 340&#226;&#8364;&#8220;430 nm. <br /> carboxyamidomethylation cysteine (+57 Da), variable                   Alpha-mannosidase activity (lysosome marker) assayed <br /> modifications oxidation methionine (+16 Da) deamida-            using p-nitrophenyl-a-D-mannopyranoside (pNP-Man) sub- <br /> tion asparagine glutamine (+0.98 Da). Mascot files             strate described previously [103]. The reaction mixtures <br /> loaded ProteoIQ (NuSep, Bogart, GA), 1% false                contained 200 mM sodium acetate buffer (pH 4.6), 0.6 mM <br /> discovery rate 0.9 peptide probability applied             pNP-Man 10 ml gradient fraction total volume of <br /> confirmation protein identifications. The ProteinProphet               100 ml. The mixture incubated 30uC 30 min, the <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                               12                   December 2014 | Volume 10 | Issue 12 | e1004555 <br />                                                                                                                          Acidocalcisome Proteome <br />  <br />  <br /> reaction terminated addition 160 ml 1 M Na2CO3.                 For BSF, 10 mg NotI-linearized plasmid DNA (,10 ml) <br /> Two microliter final mixture transferred              used 46107 mid-log phase cells 100 ml AMAXA <br /> microtitre plate read 405 nm using micro plate reader.             Human T-cell Nucleofector solution. Electroporation per- <br /> 1 <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> activity corresponds hydrolysis 1 mmol                 formed using 2 mm gap cuvettes program X-001 the <br /> substrate/min 30uC. The a-mannosidase activity expressed               AMAXA Nucleofector. Following transfection, stable trans- <br /> as mmol/min6mg protein.                                                       formants selected cloned limiting dilution HMI-9 <br />                                                                               medium containing 15% FBS appropriate antibiotics <br /> Electron microscopy                                                           (2.5 mg/ml phleomycin 2.5 mg/ml G418) 24-well plates. <br />    Aliquots (25 ml) 15,0006g pellet fraction, pellet        Antibiotic-resistant clones characterized described <br /> first gradient fraction 5 second gradient (Fig. 1 S1           below. The correct epitope-tagging target genes was <br /> Figure) precipitated centrifugation 20,000 g 15 min            confirmed PCR followed sequencing Western blot <br /> at 4uC. The pellets fixed 2.5% glutaraldehyde 4%                  analyses. RNAi induced 1 mg/ml fresh tetracycline when <br /> paraformaldehyde 0.1 M sodium cacodylate buffer (pH 7.4)                cells density 26106 PCF 16105 BSF/ml. <br /> room temperature 1 h. The supernatants carefully <br /> replaced fresh fixative disturbing pellets               Anti-TbIP3R antibodies <br /> then stored 4uC. Samples processed transmission electron                The cDNA fragment TbIP3R encoding putative IP3 <br /> microscopy Electron Microscopy Laboratory                       binding domain (amino acids 329&#226;&#8364;&#8220;804) [27] amplified by <br /> University Georgia College Veterinary Medicine.                         PCR using primers TbIP3BD-F TbIP3BD-R (S6 Table) and <br />                                                                               cloned frame expression vector pET32 EK/Lic <br /> Generation epitope tagging cassettes RNAi                              (Novagen) generate pET32(TbIP3R-BD). The correct plasmid <br />                                                                               pET32(TbIP3R-BD) confirmed sequencing then <br /> constructs                                                                    transformed E. coli OverExpress C43 (DE3) strain (Lucigen, <br />    The one-step epitope-tagging protocol reported Oberholzer               WI). His-tagged TbIP3R-BD fusion protein affinity purified <br /> et al. [89] used produce 14 C-terminal HA-tagging cassettes            Ni-NTA agarose (Qiagen) based manufacturer&#226;&#8364;&#8482;s <br /> (TriTrypDB gene ID numbers listed Table 2) transfection             protocol. The purified protein used immunize mice and <br /> T. brucei PCF trypanosomes. In brief, PCR forward                     polyclonal antibodies purified anti-serum Protein <br /> reverse primers included terminal 100&#226;&#8364;&#8220;120 nucleotides                 G Agarose Resins (Qiagen). <br /> ORF stop codon reverse complement first <br /> 100&#226;&#8364;&#8220;120 nucleotides 39UTR, respectively, followed frame <br />                                                                               Immunofluorescence microscopy <br /> by 21&#226;&#8364;&#8220;26 nucleotides backbone sequences pMOTag <br />                                                                                  When Mitotracker Red CMXRos (Invitrogen) used, live <br /> vector series [89]. The HA-tagging cassettes containing a <br />                                                                               cells labeled 30 min red-fluorescent dye at <br /> hygromycin resistant gene selection marker generated <br />                                                                               50 nM trypanosome culture medium. PCF trypanosomes were <br /> for cell transfection PCR using pMOTag4H template                  washed PBS fixed 4% paraformaldehyde in <br /> the corresponding PCR primers gene.                                    PBS room temperature 1 h. The fixed parasites washed <br />    To knockdown expression TbVAa, TbVAd, TbVIT,                 twice PBS, allowed adhere poly-L-lysine-coated <br /> TbZnT genes (TriTrypDB gene ID numbers listed Table 2)                  coverslips, permeabilized 0.3% Triton X-100/PBS for <br /> double-stranded RNA expression, inducible T7 RNA poly-                    3 min PCF. After blocking PBS containing 3% BSA, 1% <br /> merase-based protein expression p2T7Ti vector                  fish gelatin, 50 mM NH4Cl 5% goat serum 1 h, <br /> with dual-inducible T7 promoters employed. A cDNA                        trypanosomes stained 3% BSA/PBS polyclonal <br /> fragment (ranging 566 757 bp) genes targeted                rabbit antibody TbVP1 (1:500), mouse polyclonal antibody <br /> nucleotides (TbVAa: 310&#226;&#8364;&#8220;876, TbVAd: 364&#226;&#8364;&#8220;1121, TbVIT: 125&#226;&#8364;&#8220;                     TbIP3R-BD (1:100), purified HA.11 clone 16B12 mouse <br /> 755, TbZnT: 620&#226;&#8364;&#8220;1241) open reading frames (ORFs)                   monoclonal antibody HA (1:50), rat monoclonal antibody <br /> amplified using forward reverse primers listed S6 Table,           HA (1:100) (Roche), rabbit anti-GRASP antibody (1:100), <br /> digested restriction enzymes (BamHI HindIII),                    mouse anti-p67 monoclonal antibody (1:200), rabbit anti-trypa- <br /> cloned p2T7Ti vector. The recombinant constructs                    nopain (TbCATL) antibody (1:600) 1 h. After thoroughly <br /> confirmed sequencing DNA Analysis Facility Yale                  washing PBS containing 3% BSA, cells incubated with <br /> University (New Heaven, CT), NotI-linearized, purified               Alexa 488-conjugated goat anti-mouse anti-rat antibody, and <br /> QIAGEN&#226;&#8364;&#8482;s DNA purification kit cell transfections.                         Alexa 546-conjugated goat anti-rabbit anti-mouse antibody at <br />                                                                               1:1,000 1 h. The cells counterstained DAPI before <br /> Cell transfection                                                             mounting Gold ProLong Gold antifade reagent (Molecular <br />    Mid-log phase PCF (,56106 cells/ml) harvested                      Probes). Differential interference contrast (DIC) fluorescent <br /> centrifugation 1,000 g 7 min, washed Cytomix buffer               optical images captured using Olympus IX-71 inverted <br /> (2 mM EGTA, 3 mM MgCl2, 120 mM KCl, 0.5% glucose,                             fluorescence microscope Photometrix CoolSnapHQ CCD <br /> 0.15 mM CaCl2, 0.1 mg/ml BSA, 10 mM K2HPO4/KH2PO4,                            camera driven DeltaVision software (Applied Precision, Seattle, <br /> 1 mM hypoxanthine, 25 mM Hepes, pH 7.6) resuspended                       WA). Images deconvolved 15 cycles using Softwarx <br /> in 0.45 ml buffer cell density 2.56107 cells/             deconvolution software. Pearson&#226;&#8364;&#8482;s correlation coefficients (PCC) <br /> ml. The washed cells mixed 50 ml NotI-linearized                 calculated using Softwarx software measuring the <br /> plasmid DNA purified PCR products (10 mg) 0.4-cm                      images cells specific cell-staining regions. <br /> electroporation cuvette subjected pulses Bio- <br /> Rad Gene Pulser electroporator set 1.5 kV 25 mF. The                   Western blot analyses <br /> stable transformants obtained SDM-79 medium                              The cells harvested washed twice PBS. The washed <br /> supplemented 15% FBS plus appropriate antibiotics                        cells aliquots purified acidocalcisome suspension lysed <br /> (5 mg/ml phleomycin, 50 mg/ml hygromycin 15 mg/ml                         RIPA buffer (150 mM NaCl, 20 mM Tris/HCl, pH 7.5, <br /> G418).                                                                        1 mM EDTA, 1% SDS, 0.1% Triton X-100) containing <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                                   13                   December 2014 | Volume 10 | Issue 12 | e1004555 <br />                                                                                                                          Acidocalcisome Proteome <br />  <br />  <br /> protease inhibitor tablet (Roche) ice 1 h. The protein                (M) shown left. Western blot analyses (B) done <br /> concentration determined using Pierce BCA protein assay               using antibodies acidocalcisome marker TbVP1, mito- <br /> kit microplate reader. The total cell lysates mixed            chondrial marker voltage-dependent anion channel (TbVDAC), <br /> with 26 Laemmli sample buffer (BioRad) 1:1 ratio (volume/                 glycosomal marker pyruvate, phosphate dikinase (TbPPDK), and <br /> volume) directly loaded. The separated proteins                     lysosome marker Tbp67. M, Magic Marker protein standards. (C&#226;&#8364;&#8220; <br /> transferred nitrocellulose membranes using Bio-Rad                    D) Electron microscopy 15,0006g pellet P1 (C) the <br /> transblot apparatus. The membranes blocked 10%                     pellet obtained gradient centrifugation P2 <br /> non-fat milk PBS containing 0.5% Tween-20 (PBS-T) 4uC                  (D). Arrows indicate electron-dense acidocalcisomes, other <br /> overnight. The blots incubated rabbit antibodies           organelles. M, mitochondria; G, glycosome, Ac, acidocacisome <br /> TbVP1 (1:5,000), rabbit antibodies TbVDAC (1:2,000),                 (note electron-dense material them). Scale bar = <br /> mouse antibodies TbPPDK (1:200), mouse antibodies                    0.5 mm. <br /> against Tbp67 (1:3,000), rabbit antibodies TcVSP                     (TIF) <br /> (1:5,000), mouse antibodies TbIP3R (1:1,000), mouse <br />                                                                              S3 Figure Proteins present fraction 5. SDS-PAGE (left <br /> antibodies HA (1:1,000), mouse antibodies against <br /> tubulin (1:20,000) 1 h. After washings PBS-T,              panels) western blot analyses (right panels) fraction 5 from <br /> blots incubated horseradish peroxidase conjugated anti-            representative fractionations. The SDS-PAGE gels were <br /> mouse anti-rabbit IgG (H+L) antibody dilution 1:20,000            stained Coomassie brilliant blue. BenchMark protein <br /> for 1 h. After washing times PBS-T, immunoblots                molecular markers shown left gels. Western blot <br /> were visualized using Pierce ECL Western blotting substrate                  analyses using antibodies TbVP1 (A), TbIP3R <br /> according manufacturer&#226;&#8364;&#8482;s instructions.                                (B), TcVSP (C), described Materials Methods. <br />                                                                              Arrowheads A B, arrow C reactions of <br />                                                                              antibodies bands expected size. Arrowhead C <br /> Northern blot analysis                                                       probably corresponds reaction soluble pyrophospha- <br />    Total RNA isolated TRIzol reagent treated with <br />                                                                              tase. <br /> DNA-free following manufacturer&#226;&#8364;&#8482;s instructions. RNA samples <br />                                                                              (TIF) <br /> (10 mg/lane) fractionated 1% agarose/formaldehyde gels, <br /> transferred Zeta-Probe nylon membranes capillary action,               S4 Figure Immunofluorescence microscopy west- <br /> and fixed membranes baking 80uC 1 h. The                  ern blot analysis V-H+-ATPase subunit d (TbVA d) in <br /> probes TbVAa, TbVAd, TbVIT TbZnT generated                   PCF trypanosomes. V-H+-ATPase subunit d co-localize with <br /> PCR using set primers (S6 Table)                        TbVP1 acidocalcisomes (A), TbGRASP Golgi <br /> corresponding RNAi constructs p2T7Ti described                   complex (B), TbCATL (C) p67 (D) lysosomes <br /> and labeled [a-32P]-dCTP using Prime-a-Gene Labeling                  (Pearson&#226;&#8364;&#8482;s correlation coefficients 0.625, 0.561, 0.785, and <br /> System according manufacturer&#226;&#8364;&#8482;s protocol. The [a-32P]-                0.796 respectively). Yellow merge images indicate co-localization <br /> dCTP-labeled probe Tb-b-tubulin gene (GeneDB                              (also shown arrows (B&#226;&#8364;&#8220;D)). Scale bars A&#226;&#8364;&#8220;D = 10 mm. (E) <br /> Tb927.1.2390) generated T. brucei genomic DNA                    Confirmation tagging western blot analyses monoclonal <br /> PCR using gene-specific primers TbTubb-F TbTubb-R (S6                    anti-HA PCF trypanosomes. HRP-conjugated goat anti-mouse <br /> Table). RNA-bound membranes hybridized 32P-                    used secondary antibody. Precision Plus Protein <br /> labeled probes 0.5 M Na2HPO4, pH 7.4 7% SDS 65uC                   WesternC marker (Bio-Rad) used molecular weight <br /> overnight agitation. After hybridization, membranes            markers. Arrow indicates band corresponding TbVA d. Tubulin <br /> washed twice 10 min 68uC 16 SSC 0.1%                    (Tub) used loading control (bottom panel). <br /> SDS twice 30 min 65uC 0.16SSC 0.1% SDS.                  (TIF) <br /> Northern blots visualized autoradiography, quantified <br />                                                                              S5 Figure Localization proteins. (A) Epitope- <br /> by using ImageJ (National Institute Health, Bethesda, MD). <br />                                                                              tagged TbNP localizes nuclear membrane. (B) TbABCT co- <br />                                                                              localizes MitoTracker (Pearson&#226;&#8364;&#8482;s correlation coefficient of <br /> Supporting Information                                                       0.688). Yellow merge images indicate co-localization. Scale bars <br /> S1 Figure Subcellular fractionation acidocalcisomes.                      A&#226;&#8364;&#8220;B = 10 mm. (C&#226;&#8364;&#8220;D) Tagging HA confirmed by <br /> (A) Trypanosome lysates obtained grinding silicon               western blot analyses using anti-HA antibodies. Markers the <br /> carbide, decanted low speed centrifugation eliminate debris            left arrows indicate corresponding bands. Equivalent <br /> and silicon carbide, centrifuged 15,000 g 10 min               amounts wild type cell (WT) proteins loaded evidenced <br /> isolate organellar fraction applied 34% step        similar background test lanes. <br /> discontinuous iodixanol gradient. After centrifugation 50,000 g           (TIF) <br /> for 1 h, pellet resuspended applied 27% step           S6 Figure Localization proteins. Epitope-tagged <br /> a second iodixanol gradient centrifuged 50,000 g 1 h.             TbGLP1 does co-localize TbVP1 (A) co-localizes <br /> Aliquots fraction used enzymatic assays. (B)              TbGRASP Golgi complex (B), TbCATL (C) <br /> Electron microscopy acidocalcisome fraction prepared               p67 (D) lysosome (Pearson&#226;&#8364;&#8482;s correlation coefficients of <br /> iodixanol procedure (fraction 5). Scale bar = 0.2 mm. Arrows             0.5369, 0.8050 0.8426, respectively). Yellow merge images <br /> arrowheads electron-dense material inside acidocalcisomes. <br />                                                                              indicate co-localization (also shown arrows (B&#226;&#8364;&#8220;D)). Scale <br /> (TIF)                                                                        bars A&#226;&#8364;&#8220;D = 10 mm. (E) Tagging HA confirmed by <br /> S2 Figure SDS-PAGE, immunoblots, electron mi-                            western blot analyses using anti-HA antibodies. Markers at <br /> crographs subcellular fractions. (A&#226;&#8364;&#8220;B) SDS-PAGE                       left arrow shows band corresponding to <br /> immunoblot analyses 15,0006g pellet (P1, 30 mg),            TbGLP1. Equivalent amounts wild type cell (WT) proteins <br /> gradient pellet (P2, 2 mg), second gradient fractions (F1         loaded evidenced similar background test <br /> F7, 2 mg each). The SDS-PAGE gel (A) stained                        lanes. <br /> Coomassie brilliant blue. BenchMark protein molecular markers                (TIF) <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                                  14                   December 2014 | Volume 10 | Issue 12 | e1004555 <br />                                                                                                                            Acidocalcisome Proteome <br />  <br />  <br /> S7 Figure Immunofluorescence microscopy western                           S2 Table    Peptide list high confidence identifica- <br /> blot analysis polyamine transporters TbPOT1. Epitope-                      tions (1% false discovery rate, protein probability <br /> tagged TbPOT1 partially co-localizes TbVP1 acidocalci-            $0.95) acidocalcisome data sets (ACCS1 and <br /> somes (A) co-localizes TbCATL (B) p67 (C),                ACCS2). When high-confidence identifications similar proteins <br /> lysosomes (Pearson&#226;&#8364;&#8482;s correlation coefficients 0.4064, 07191,           identified peptide degeneracy limited discrimination, <br /> 0.6710, respectively). Arrrows merge images co-                   peptides matched indistinguishable proteins repre- <br /> localization. (D) A putative cation/proton antiporter localizes        sented single protein &#226;&#8364;&#732;&#226;&#8364;&#732;group&#226;&#8364;&#8482;&#226;&#8364;&#8482; described Methods. <br /> falgellar tip (white arrow) named flagellar tip protein               (PDF) <br /> (TbFTP). (E, F) Tagging HA confirmed western blot                 S3 Table Signal peptide (SP) transmembrane <br /> analyses using anti-HA antibodies. Markers left           domain (TMD) predictions high confidence protein <br /> arrows shows band corresponding TgGLP1, TbFTP,                     identifications (1% false discovery rate, protein proba- <br /> respectively. Equivalent amounts wild type cell (WT) proteins         bility $0.95) T. brucei acidocalcisome datasets <br /> loaded evidenced similar background test lanes.              (ACCS1 ACCS2). Predictions TMD SP for <br /> (TIF)                                                                         individual proteins based consensus more <br /> S8 Figure Immunofluorescence microscopy west-                             algorithms. If predicted number TMD varied the <br /> ern blot analysis polyamine transporters TbPOT2.                           different predictions packages, report median number of <br /> Epitope-tagged TbPOT2 does co-localize TbGRASP                    TMD. If protein group contained protein hit (see <br /> the Golgi complex (A) co-localizes TbCATL (B)                 Methods explanation protein grouping based degenerate <br /> p67 lysosomes (C) (Pearson&#226;&#8364;&#8482;s correlation coefficients               peptide fingerprints), number TMD presence SP <br /> 0.8806, 0.8404, respectively). Scale bars A&#226;&#8364;&#8220;C = 10 mm. (D)                assigned predicted members. <br /> Tagging HA confirmed western blot analyses using                  (PDF) <br /> anti-HA antibodies. Markers left side. Arrow indicate              S4 Table Predicted subcellular locations high- <br /> band corresponding TbPOT2. Equivalent amounts wild type                 confidence protein groups (protein probability: 1% false <br /> cell (WT) proteins loaded evidenced similar                    discovery rate, p$0.95,) identified T. brucei <br /> background test lanes.                                                 acidocalcisome fractions (ACCS1 ACCS2) our <br /> (TIF)                                                                         prediction servers using non-plant based algorithms. <br /> S9 Figure     Comparison newly identified acid-                      Individual protein predictions based agreement between <br /> ocalcisome proteins homologues known                           analysis packages. Consensus identifications with <br /> functions organisms. Multiple protein sequence                     multiple protein hits (see Methods explanation protein <br /> alignments (A) phosphate transporters S. cerevisiae                   grouping based degenerate peptide fingerprints) given when <br /> (ScPho91p, accession number CAY82206), T. cruzi (TcPho91,                     prediction compartment agrees members <br /> TcCLB.508831.60), T. brucei Pho91 (TbPho91, Tb927.                        group. S, secreted. C, cytosol. M, mitochondrion. N, <br /> 11.11160). (B) Acid phosphatases Homo sapiens (HsTRAP,                   nucleus. PM, plasma membrane. G, Golgi complex. ER, <br /> P13686) T. brucei (TbAP, Tb927.10.7020). (C) Vacuolar iron                endoplasmic reticulum. P, peroxisome. L, lysosome. CYKS, <br /> transporters A. thaliana (AtVIT1, NP_178286), S. cerevisiae              cytoskeleton. Threshold probabilities confidences used to <br /> (ScVIT1, DDA09536), T. brucei (TbVIT1, Tb927.3.800).                      screen predictions poor reliability: targetP (RC = 1). <br /> The protein sequences analyzed ClustalW2                      pTarget $80%, SLP-LOCAL $2, WoLfPsort $,80%. WoLFP- <br /> EMBL-EBI          website     (http://www.ebi.ac.uk/Tools/msa/                SORT thresholds subcellular location derived from <br /> clustalw2/). The symbols &#226;&#8364;&#732;&#226;&#8364;&#732;*&#226;&#8364;&#8482;&#226;&#8364;&#8482;, &#226;&#8364;&#732;&#226;&#8364;&#732;:&#226;&#8364;&#8482;&#226;&#8364;&#8482;, &#226;&#8364;&#732;&#226;&#8364;&#732;.&#226;&#8364;&#8482;&#226;&#8364;&#8482; represent identical,         empirical prediction confidence statistics (wolfpsort.org/empiri- <br /> conserved, semi-conserved amino acid (aa) substitutions,                   calConfidenceByNumNeighbors/index.html, updated August 15, <br /> respectively. Red: small hydrophobic aa (AVFPMILW); blue:                 2007). <br /> acidic aa (DE); magenta: basic aa (RK); green: hydroxyl,                  (PDF) <br /> amine, basic aa (STYHCNGQ).                                               S5 Table Subunits vacuolar H+-ATPase present in <br /> (TIF)                                                                         genome T. brucei. <br /> S10 Figure     Comparison newly identified acid-                       (PDF) <br /> ocalcisome proteins homologues known                           S6 Table Primers used generate probes blotting <br /> functions organisms. Multiple protein sequence                     constructs antibody production RNAi (the <br /> alignments (A) Zinc transporters Mus musculus                         underlined nucleotides indicate primer extension <br /> (MmZnT4, AAB82593), S. cerevisiae (ScZRC1, CAA88653.1),                       sequences ligation independent cloning the <br /> A. thaliana (AtMTP1, NP_850459), E. coli (EcYiiP, P69380.1), T.               introduced HindIII BamHI sites). <br /> cruzi (TcZnT, TcCLB.511439.50), T. brucei (TbZnT,                         (PDF) <br /> Tb927.4.4960). (B) Polyamine transporters S. cerevisiae <br />                                                                               S7 Table Common features confirmed acidocalci- <br /> (ScTPO1, Q07824) T. brucei (TbPOT1, Tb927.9.10340). <br />                                                                               proteins identified ELM server. <br /> Analysis Fig. S9. <br />                                                                               (PDF) <br /> (TIF) <br /> S1 Table     T. brucei proteins identified high confi-                   Acknowledgements <br /> dence (1% false discovery rate, protein probability <br /> $0.95) fraction 5 datasets (ACCS1 ACCS2).                            We thank George A.M. Cross providing strain 29-13 (PCF) single <br />                                                                               marker (BSF), John Donelson p2T7Ti vector, Thomas Seebeck for <br /> Proteins degenerate peptide (peptides shared                   pMOTag4H vector, Norbert Bakalara anti-TbVP1 antibody, <br /> members protein cluster) fingerprints reported single         Graham Warren anti-TbGRASP antibody, Minu Chaudhuri anti- <br /> protein &#226;&#8364;&#732;&#226;&#8364;&#732;group&#226;&#8364;&#8482;&#226;&#8364;&#8482; described Methods.                                TbVDAC antibody, Fre&#194;&#180;de&#194;&#180;ric Bringaud anti-TbPPDK antibody, James <br /> (PDF)                                                                         Bangs antibodies T. brucei p67 TbCATL. <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                                   15                   December 2014 | Volume 10 | Issue 12 | e1004555 <br />                                                                                                                                                    Acidocalcisome Proteome <br />  <br />  <br />  <br /> Author Contributions                                                                          GH PNU DJ JAT RO RD. Contributed reagents/materials/analysis tools: <br />                                                                                               PNU JAT SNJM RO. Wrote paper: GH PNU DJ RD. <br /> Conceived designed experiments: GH DJ SNJM RO RD. <br /> Performed experiments: GH PNU MS DJ JT JAT. Analyzed data: <br />  <br /> References <br />  1. Babes V (1895) Beobachtungen u&#194;&#168;ber die metachromatischen ko&#194;&#168;rperchen,                     27. Huang G, Bartlett PJ, Thomas AP, Moreno SN, Docampo R (2013) <br />     sporenbildung, verzwiegung, kolben- und kapsel-bildung pathogener bakterien.                  Acidocalcisomes Trypanosoma brucei inositol 1,4,5-trisphosphate <br />     Zentralbl Bakteriol Parasitenkd Infektionskr Hyg 20: 412&#226;&#8364;&#8220;420.                                 receptor required growth infectivity. Proc Natl Acad Sci U S A <br />  2. Meyer A (1904) Orientirende untersuchungen uber verbreitung, morphologie                      110: 1887&#226;&#8364;&#8220;1892. <br />     und chemie des volutins. Bot Zeit 62: 113&#226;&#8364;&#8220;152.                                            28. Lemercier G, Espiau B, Ruiz FA, Vieira M, Luo S, et al. (2004) A <br />  3. Wiame JM (1947) Etude d&#226;&#8364;&#8482;une substance polyphosphoree, basophile et                            pyrophosphatase regulating polyphosphate metabolism acidocalcisomes is <br />     metachromatique chez les levures. Biochim Biophys acta 1: 234&#226;&#8364;&#8220;255.                            essential Trypanosoma brucei virulence mice. J Biol Chem 279: 3420&#226;&#8364;&#8220; <br />  4. Rao NN, Gomez-Garcia MR, Kornberg A (2009) Inorganic polyphosphate:                           3425. <br />     essential growth survival. Annu Rev Biochem 78: 605&#226;&#8364;&#8220;647.                          29. Lemercier G, Dutoya S, Luo S, Ruiz FA, Rodrigues CO, et al. (2002) A <br />  5. Docampo R, Souza W, Miranda K, Rohloff P, Moreno SN (2005)                                 vacuolar-type H+-pyrophosphatase governs maintenance functional acid- <br />     Acidocalcisomes - conserved bacteria man. Nat Rev Microbiol 3: 251&#226;&#8364;&#8220;                   ocalcisomes growth insect mammalian forms Trypanosoma <br />     261.                                                                                          brucei. J Biol Chem 277: 37369&#226;&#8364;&#8220;37376. <br />  6. Seufferheld M, Lea CR, Vieira M, Oldfield E, Docampo R (2004) The H+-                     30. Nesvizhskii AI, Keller A, Kolker E, Aebersold R (2003) A statistical model for <br />     pyrophosphatase Rhodospirillum rubrum predominantly located                          identifying proteins tandem mass spectrometry. Anal Chem 75: 4646&#226;&#8364;&#8220;4658. <br />     polyphosphate-rich acidocalcisomes. J Biol Chem 279: 51193&#226;&#8364;&#8220;51202.                         31. Wallin E, von Heijne G (1998) Genome-wide analysis integral membrane <br />  7. Seufferheld M, Vieira MC, Ruiz FA, Rodrigues CO, Moreno SN, et al. (2003)                     proteins eubacterial, archaean, eukaryotic organisms. Protein Sci 7: <br />     Identification organelles bacteria similar acidocalcisomes unicellular            1029&#226;&#8364;&#8220;1038. <br />     eukaryotes. J Biol Chem 278: 29971&#226;&#8364;&#8220;29978.                                                 32. Patel S, Docampo R (2010) Acidic calcium stores open business: expanding <br />  8. Vercesi AE, Moreno SN, Docampo R (1994) Ca2+/H+ exchange acidic                            potential intracellular Ca2+ signaling. Trends Cell Biol 20: 277&#226;&#8364;&#8220;286. <br />     vacuoles Trypanosoma brucei. Biochem J 304: 227&#226;&#8364;&#8220;233.                                   33. Bangs JD, Uyetake L, Brickman MJ, Balber AE, Boothroyd JC (1993) <br />  9. Docampo R, Scott DA, Vercesi AE, Moreno SN (1995) Intracellular Ca2+                          Molecular cloning cellular localization BiP homologue Trypanosoma <br />     storage acidocalcisomes Trypanosoma cruzi. Biochem J 310: 1005&#226;&#8364;&#8220;1012.                    brucei. Divergent ER retention signals lower eukaryote. J Cell Sci 105: <br /> 10. Rodrigues CO, Scott DA, Docampo R (1999) Presence vacuolar H+-                           1101&#226;&#8364;&#8220;1113. <br />     pyrophosphatase promastigotes Leishmania donovani localization              34. Ulrich PN, Jimenez V, Park M, Martins VP, Atwood J, 3rd, et al. (2011) <br />     different compartment vacuolar H+-ATPase. Biochem J 340:                        Identification contractile vacuole proteins Trypanosoma cruzi. PLoS One <br />     759&#226;&#8364;&#8220;766.                                                                                      6: e18013. <br /> 11. Moreno SN, Zhong L (1996) Acidocalcisomes Toxoplasma gondii                            35. Hashimoto M, Enomoto M, Morales J, Kurebayashi N, Sakurai T, et al. (2013) <br />     tachyzoites. Biochem J 313: 655&#226;&#8364;&#8220;659.                                                          Inositol 1,4,5-trisphosphate receptor regulates replication, differentiation, <br /> 12. Marchesini N, Luo S, Rodrigues CO, Moreno SN, Docampo R (2000)                                infectivity virulence parasitic protist Trypanosoma cruzi. Mol <br />     Acidocalcisomes vacuolar H+-pyrophosphatase malaria parasites.                       Microbiol 87: 1133&#226;&#8364;&#8220;1150. <br />     Biochem J 347: 243&#226;&#8364;&#8220;253.                                                                   36. Luo S, Rohloff P, Cox J, Uyemura SA, Docampo R (2004) Trypanosoma <br /> 13. Soares Medeiros LC, Gomes F, Maciel LR, Seabra SH, Docampo R, et al.                          brucei plasma membrane-type Ca2+-ATPase 1 (TbPMC1) 2 (TbPMC2) <br />     (2011) Volutin granules Eimeria parasites acidic compartments                 genes encode functional Ca2+-ATPases localized acidocalcisomes and <br />     physiological structural characteristics similar acidocalcisomes. J Eukar-             plasma membrane, essential Ca2+ homeostasis growth. J Biol <br />     yot Microbiol 58: 416&#226;&#8364;&#8220;423.                                                                    Chem 279: 14427&#226;&#8364;&#8220;14439. <br /> 14. Marchesini N, Ruiz FA, Vieira M, Docampo R (2002) Acidocalcisomes                     37. Hothorn M, Neumann H, Lenherr ED, Wehner M, Rybin V, et al. (2009) <br />     functionally linked contractile vacuole Dictyostelium discoideum. J Biol <br />                                                                                                   Catalytic core membrane-associated eukaryotic polyphosphate polymerase. <br />     Chem 277: 8146&#226;&#8364;&#8220;8153. <br />                                                                                                   Science 324: 513&#226;&#8364;&#8220;516. <br /> 15. Franzen AJ, Cunha MM, Miranda K, Hentschel J, Plattner H, et al. (2008) <br />                                                                                               38. Lander N, Ulrich PN, Docampo R (2013) Trypanosoma brucei vacuolar <br />     Ultrastructural characterization melanosomes human pathogenic <br />                                                                                                   transporter chaperone 4 (TbVtc4) acidocalcisome polyphosphate kinase <br />     fungus Fonsecaea pedrosoi. J Struct Biol 162: 75&#226;&#8364;&#8220;84. <br />                                                                                                   required vivo infection. J Biol Chem 288: 34205&#226;&#8364;&#8220;34216. <br /> 16. Ruiz FA, Marchesini N, Seufferheld M, Govindjee, Docampo R (2001) The <br />                                                                                               39. Ulrich PN, Lander N, Kurup S, Reiss L, Brewer J, et al. (2014) The <br />     polyphosphate bodies Chlamydomonas reinhardtii possess proton-pumping <br />                                                                                                   acidccalcisome vacuolar transporter chaperone 4 catalyzes synthesis of <br />     pyrophosphatase similar acidocalcisomes. J Biol Chem 276: 46196&#226;&#8364;&#8220; <br />                                                                                                   polyphosphate insect-stages Trypanosoma brucei T. cruzi. J Eukaryot <br />     46203. <br />                                                                                                   Microbiol 61: 155&#226;&#8364;&#8220;165. <br /> 17. Yagisawa F, Nishida K, Yoshida M, Ohnuma M, Shimada T, et al. (2009) <br />     Identification novel proteins isolated polyphosphate vacuoles                40. Fang J, Rohloff P, Miranda K, Docampo R (2007) Ablation small <br />     primitive red alga Cyanidioschyzon merolae. Plant J 60: 882&#226;&#8364;&#8220;893.                              transmembrane protein Trypanosoma brucei (TbVTC1) involved the <br /> 18. Motta LS, Ramos IB, Gomes FM, Souza W, Champagne DE, et al. (2009)                         synthesis polyphosphate alters acidocalcisome biogenesis function, and <br />     Proton-pyrophosphatase polyphosphate acidocalcisome-like vesicles                      leads cytokinesis defect. Biochem J 407: 161&#226;&#8364;&#8220;170. <br />     oocytes eggs Periplaneta americana. Insect Biochem Mol Biol 39:               41. Gomes SA, Fonseca Souza AL, Silva BA, Kiffer-Moreira T, Santos-Mallet <br />     198&#226;&#8364;&#8220;206.                                                                                      JR, et al. (2006) Trypanosoma rangeli: Differential expression cell surface <br /> 19. Ramos I, Gomes F, Koeller CM, Saito K, Heise N, et al. (2011)                                 polypeptides ecto-phosphatase activity short long epimastigote <br />     Acidocalcisomes calcium- polyphosphate-storage compartments                     forms. Exp Parasitol 112: 253&#226;&#8364;&#8220;262. <br />     embryogenesis insect Rhodnius prolixus Stahl. PLoS One 6: e27276.                  42. Williamson J, McLaren DJ (1981) Localization phosphatases Trypanosoma <br /> 20. Ramos IB, Miranda K, Pace DA, Verbist KC, Lin FY, et al. (2010) Calcium-                      rhodesiense. J Protozool 28: 460&#226;&#8364;&#8220;467. <br />     polyphosphate-containing acidic granules sea urchin eggs similar            43. Bowman EJ, Siebers A, Altendorf K (1988) Bafilomycins: class inhibitors of <br />     acidocalcisomes, targets NAADP. Biochem J 429: 485&#226;&#8364;&#8220;495.                   membrane ATPases microorganisms, animal cells, plant cells. Proc <br /> 21. Ramos IB, Miranda K, Ulrich P, Ingram P, LeFurgey A, et al. (2010) Calcium-                   Natl Acad Sci U S A 85: 7972&#226;&#8364;&#8220;7976. <br />     polyphosphate-containing acidocalcisomes chicken egg yolk. Biol Cell               44. Scott DA, Moreno SN, Docampo R (1995) Ca2+ storage Trypanosoma <br />     102: 421&#226;&#8364;&#8220;434.                                                                                 brucei: influence cytoplasmic pH importance vacuolar acidity. <br /> 22. Ruiz FA, Lea CR, Oldfield E, Docampo R (2004) Human platelet dense                            Biochem J 310: 789&#226;&#8364;&#8220;794. <br />     granules contain polyphosphate similar acidocalcisomes bacteria             45. He CY, Ho HH, Malsam J, Chalouni C, West CM, et al. (2004) Golgi <br />     unicellular eukaryotes. J Biol Chem 279: 44250&#226;&#8364;&#8220;44257.                                     duplication Trypanosoma brucei. J Cell Biol 165: 313&#226;&#8364;&#8220;321. <br /> 23. Moreno-Sanchez D, Hernandez-Ruiz L, Ruiz FA, Docampo R (2012)                             46. Tazeh NN, Silverman JS, Schwartz KJ, Sevova ES, Sutterwala SS, et al. (2009) <br />     Polyphosphate novel pro-inflammatory regulator mast cells                      Role AP-1 developmentally regulated lysosomal trafficking Trypano- <br />     located acidocalcisomes. J Biol Chem 287: 28435&#226;&#8364;&#8220;28444.                                     soma brucei. Eukaryot Cell 8: 1352&#226;&#8364;&#8220;1361. <br /> 24. Rodrigues CO, Scott DA, Docampo R (1999) Characterization vacuolar                   47. Correa AF, Andrade LR, Soares MJ (2002) Elemental composition of <br />     pyrophosphatase Trypanosoma brucei localization acidocalcisomes.                acidocalcisomes Trypanosoma cruzi bloodstream trypomastigote forms. <br />     Mol Cell Biol 19: 7712&#226;&#8364;&#8220;7723.                                                                  Parasitol Res 88: 875&#226;&#8364;&#8220;880. <br /> 25. Salto ML, Kuhlenschmidt T, Kuhlenschmidt M, Lederkremer RM,                            48. Miranda K, Rodrigues CO, Hentchel J, Vercesi A, Plattner H, et al. (2004) <br />     Docampo R (2008) Phospholipid glycolipid composition acidocalcisomes                   Acidocalcisomes Phytomonas francai possess distinct morphological <br />     Trypanosoma cruzi. Mol Biochem Parasitol 158: 120&#226;&#8364;&#8220;130.                                     characteristics contain iron. Microsc Microanal 10: 647&#226;&#8364;&#8220;655. <br /> 26. Scott DA, Souza W, Benchimol M, Zhong L, Lu HG, et al. (1998) Presence                 49. Miranda K, Docampo R, Grillo O, Souza W (2004) Acidocalcisomes of <br />     plant-like proton-pumping pyrophosphatase acidocalcisomes                          trypanosomatids species-specific elemental composition. Protist 155: 395&#226;&#8364;&#8220; <br />     Trypanosoma cruzi. J Biol Chem 273: 22151&#226;&#8364;&#8220;22158.                                              405. <br />  <br />  <br />  <br /> PLOS Pathogens | www.plospathogens.org                                                   16                     December 2014 | Volume 10 | Issue 12 | e1004555 <br />                                                                                                                                                     Acidocalcisome Proteome <br />  <br />  <br /> 50. Miranda K, Docampo R, Grillo O, Franzen A, Attias M, et al. (2004)                         75. Vieira M, Rohloff P, Luo S, Cunha-e-Silva NL, Souza W, et al. (2005) Role <br />     Dynamics polymorphism acidocalcisomes Leishmania parasites.                           P-type H+-ATPase acidification endocytic pathway of <br />     Histochem Cell Biol 121: 407&#226;&#8364;&#8220;418.                                                              Trypanosoma cruzi. Biochem J 392: 467&#226;&#8364;&#8220;474. <br /> 51. Ferella M, Nilsson D, Darban H, Rodrigues C, Bontempi EJ, et al. (2008)                    76. Kim SA, Punshon T, Lanzirotti A, Li L, Alonso JM, et al. (2006) Localization <br />     Proteomics Trypanosoma cruzi&#226;&#8364;&#8220;localization novel proteins various                      iron Arabidopsis seed requires vacuolar membrane transporter VIT1. <br />     organelles. Proteomics 8: 2735&#226;&#8364;&#8220;2749.                                                           Science 314: 1295&#226;&#8364;&#8220;1298. <br /> 52. DeGrasse JA, DuBois KN, Devos D, Siegel TN, Sali A, et al. (2009) Evidence                 77. Li L, Chen OS, McVey Ward D, Kaplan J (2001) CCC1 transporter that <br />     shared nuclear pore complex architecture conserved                 mediates vacuolar iron storage yeast. J Biol Chem 276: 29515&#226;&#8364;&#8220;29519. <br />     common eukaryotic ancestor. Mol Cell Proteomics 8: 2119&#226;&#8364;&#8220;2130.                              78. Docampo R, Moreno SN (2011) Acidocalcisomes. Cell Calcium 50: 113&#226;&#8364;&#8220;119. <br /> 53. Lingnau A, Zufferey R, Lingnau M, Russell DG (1999) Characterization                    79. Nies DH, Silver S (1995) Ion efflux systems involved bacterial metal <br />     tGLP-1, Golgi lysosome-associated, transmembrane glycoprotein                         resistances. J Ind Microbiol 14: 186&#226;&#8364;&#8220;199. <br />     African trypanosomes. J Cell Sci 112 Pt 18: 3061&#226;&#8364;&#8220;3070.                                     80. Wang ZY, Li JY, Danscher G, Dahlstrom A (2002) Localization zinc- <br /> 54. Tomitori H, Kashiwagi K, Asakawa T, Kakinuma Y, Michael AJ, et al. (2001)                      enriched neurons mouse peripheral sympathetic system. Brain Res 928: <br />     Multiple polyamine transport systems vacuolar membrane yeast.                        165&#226;&#8364;&#8220;174. <br />     Biochem J 353: 681&#226;&#8364;&#8220;688.                                                                    81. MacDiarmid CW, Gaither LA, Eide D (2000) Zinc transporters regulate <br /> 55. Vercesi AE, Rodrigues CO, Catisti R, Docampo R (2000) Presence Na+/                       vacuolar zinc storage Saccharomyces cerevisiae. EMBO J 19: 2845&#226;&#8364;&#8220;2855. <br />     H+ exchanger acidocalcisomes Leishmania donovani alkalization              82. Kawachi M, Kobae Y, Kogawa S, Mimura T, Kramer U, et al. (2012) Amino <br />     anti-leishmanial drugs. FEBS Lett 473: 203&#226;&#8364;&#8220;206.                                             acid screening based structural modeling identifies critical residues the <br /> 56. Vercesi AE, Docampo R (1996) Sodium-proton exchange stimulates Ca2+                            function, ion selectivity structure Arabidopsis MTP1. FEBS J 279: 2339&#226;&#8364;&#8220; <br />     release acidocalcisomes Trypanosoma brucei. Biochem J 315: 265&#226;&#8364;&#8220;270.                    2356. <br /> 57. Vercesi AE, Grijalba MT, Docampo R (1997) Inhibition Ca2+ release                  83. Chao Y, Fu D (2004) Kinetic study antiport mechanism Escherichia <br />     Trypanosoma brucei acidocalcisomes 3,5-dibutyl-4-hydroxytoluene: role                    coli zinc transporter, ZitB. J Biol Chem 279: 12043&#226;&#8364;&#8220;12050. <br />     Na+/H+ exchanger. Biochem J 328: 479&#226;&#8364;&#8220;482.                                              84. Bonifacino JS, Traub LM (2003) Signals sorting transmembrane proteins <br /> 58. Saada EA, Kabututu ZP, Lopez M, Shimogawa MM, Langousis G, et al.                              endosomes lysosomes. Annu Rev Biochem 72: 395&#226;&#8364;&#8220;447. <br />     (2014) Insect stage-specific receptor adenylate cyclases localized distinct         85. Adachi A, Kano F, Tsuboi T, Fujita M, Maeda Y, et al. (2010) Golgi-associated <br />     subdomains Trypanosoma brucei flagellar membrane. Eukaryot Cell 13:                     GSK3beta regulates sorting process post-Golgi membrane trafficking. <br />     1064&#226;&#8364;&#8220;1076.                                                                                     J Cell Sci 123: 3215&#226;&#8364;&#8220;3225. <br /> 59. Liu W, Apagyi K, McLeavy L, Ersfeld K (2010) Expression cellular                       86. Cunningham I (1977) New culture medium maintenance tsetse tissues <br />     localisation calpain-like proteins Trypanosoma brucei. Mol Biochem                       growth trypanosomatids. J Protozool 24: 325&#226;&#8364;&#8220;329. <br />     Parasitol 169: 20&#226;&#8364;&#8220;26. <br />                                                                                                87. Wirtz E, Leal S, Ochatt C, Cross GA (1999) A tightly regulated inducible <br /> 60. Chan KY, Ersfeld K (2010) The role Kinesin-13 family protein TbKif13-                   expression conditional gene knock-outs dominant-negative <br />     2 flagellar length control Trypanosoma brucei. Mol Biochem Parasitol 174: <br />                                                                                                    genetics Trypanosoma brucei. Mol Biochem Parasitol 99: 89&#226;&#8364;&#8220;101. <br />     137&#226;&#8364;&#8220;140. <br />                                                                                                88. Hirumi H, Hirumi K (1989) Continuous cultivation Trypanosoma brucei <br /> 61. Subota I, Julkowska D, Vincensini L, Reeg N, Buisson J, et al. (2014) Proteomic <br />                                                                                                    bloodstream forms medium containing low concentration serum <br />     analysis intact flagella procyclic Trypanosoma brucei cells identifies novel <br />                                                                                                    protein feeder cell layers. J Parasitol 75: 985&#226;&#8364;&#8220;989. <br />     flagellar proteins unique sub-localization dynamics. Mol Cell <br />                                                                                                89. Oberholzer M, Morand S, Kunz S, Seebeck T (2006) A vector series rapid <br />     Proteomics 13: 1769&#226;&#8364;&#8220;1786. <br />                                                                                                    PCR-mediated C-terminal situ tagging Trypanosoma brucei genes. Mol <br /> 62. Jimenez V, Docampo R (2012) Molecular electrophysiological character- <br />                                                                                                    Biochem Parasitol 145: 117&#226;&#8364;&#8220;120. <br />     ization novel cation channel Trypanosoma cruzi. PLoS Pathog 8: <br />                                                                                                90. LaCount DJ, Barrett B, Donelson JE (2002) Trypanosoma brucei FLA1 is <br />     e1002750. <br />                                                                                                    required flagellum attachment cytokinesis. J Biol Chem 277: 17580&#226;&#8364;&#8220; <br /> 63. Huang G, Fang J, Sant&#226;&#8364;&#8482;Anna C, Li ZH, Wellems DL, et al. (2011) Adaptor <br />                                                                                                    17588. <br />     protein-3 (AP-3) complex mediates biogenesis acidocalcisomes is <br />     essential growth virulence Trypanosoma brucei. J Biol Chem 286:                 91. Emanuelsson O, Nielsen H, Brunak S, von Heijne G (2000) Predicting <br />     36619&#226;&#8364;&#8220;36630.                                                                                   subcellular localization proteins based N-terminal amino acid <br /> 64. Bridges DJ, Pitt AR, Hanrahan O, Brennan K, Voorheis HP, et al. (2008)                         sequence. J Mol Biol 300: 1005&#226;&#8364;&#8220;1016. <br />     Characterisation plasma membrane subproteome bloodstream form                    92. Guda C, Subramaniam S (2005) pTARGET [corrected] new method for <br />     Trypanosoma brucei. Proteomics 8: 83&#226;&#8364;&#8220;99.                                                       predicting protein subcellular localization eukaryotes. Bioinformatics 21: <br /> 65. Lam AK, Galione A (2013) The endoplasmic reticulum junctional                              3963&#226;&#8364;&#8220;3969. <br />     membrane communication calcium signaling. Biochim Biophys Acta                      93. Matsuda S, Vert JP, Saigo H, Ueda N, Toh H, et al. (2005) A novel <br />     1833: 2542&#226;&#8364;&#8220;2559.                                                                               representation protein sequences prediction subcellular location using <br /> 66. Huang G, Vercesi AE, Docampo R (2013) Essential regulation cell                             support vector machin <br /> </body></html>