<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Inducible Knockdown of <br /> MONOGALACTOSYLDIACYLGLYCEROL SYNTHASE1 <br /> Reveals Roles Galactolipids Organelle <br /> Differentiation Arabidopsis Cotyledons1[W][OPEN] <br /> Sho Fujii 2, Koichi Kobayashi 2*, Yuki Nakamura*, Hajime Wada <br /> Department Life Sciences, Graduate School Arts Sciences, University Tokyo, Meguro-ku, Tokyo <br /> 153&#226;&#8364;&#8220;8902, Japan (S.F., K.K., H.W.); PRESTO (Y.N.) CREST (H.W.), JST, Kawaguchi, Saitama 332&#226;&#8364;&#8220;0012, <br /> Japan; Institute Plant Microbial Biology, Academia Sinica, Nankang, Tapei 11529, Taiwan (Y.N.) <br />  <br />  <br /> Monogalactosyldiacylglycerol (MGDG) major lipid constituent thylakoid membranes essential chloroplast <br /> biogenesis plants. In Arabidopsis (Arabidopsis thaliana), MGDG predominantly synthesized inner envelope-localized <br /> MONOGALACTOSYLDIACYLGLYCEROL SYNTHASE1 (MGD1); <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>knockout</span> causes albino seedlings. Because lethal <br /> <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> null MGD1 mutant, functional details MGDG synthesis seedling development remained elusive. <br /> In study, used inducible gene-suppression investigate impact MGDG synthesis cotyledon <br /> development. We created transgenic Arabidopsis lines express arti&#239;&#172;?cial microRNA targeting MGD1 (amiR-MGD1) <br /> under control dexamethasone-inducible promoter. The induction amiR-MGD1 resulted 75% suppression <br /> of MGD1 expression, resulting phenotypes related chloroplast development diverse, line. <br /> The strong MGD1 suppression continuous dexamethasone treatment caused substantial decreases galactolipid content in <br /> cotyledons, leading severe defects formation thylakoid membranes impaired photosynthetic electron transport. <br /> Time-course analyses MGD1 suppression seedling germination revealed MGDG synthesis early <br /> germination stage particularly important chloroplast biogenesis. The MGD1 suppression down-regulated genes associated <br /> with photorespiratory pathway peroxisomes mitochondria responsible photosynthesis in <br /> chloroplasts caused high expression genes glyoxylate cycle. MGD1 function link galactolipid synthesis <br /> with coordinated transcriptional regulation chloroplasts organelles cotyledon greening. <br />  <br />  <br />  <br />   In dicotyledonous plants, cotyledons,                             growth germinated seedlings, metabolic activities <br /> formed embryogenesis, initially serve storage                       change greatly cotyledon cells. Before devel- <br /> organs seed germination mainly function                         opment photosynthetic capacity cotyledons, the <br /> photosynthesis seedling establishment. During                       seedlings oilseed plants Arabidopsis (Arabi- <br /> developmental switch heterotrophic autotrophic                        dopsis thaliana) grow heterotrophically depending on <br />                                                                               triacylglycerol (TAG) stored oil bodies cotyle- <br />                                                                               don cells. In stage, peroxisomes function the <br />    1 <br />      This work supported Grants-in-Aid Scienti&#239;&#172;?c Re-               glyoxysome, converts fatty acids bound TAG <br /> search Priority Areas (grant no. 24770055) JST CREST              succinate b-oxidation glyoxylate cycle, <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br /> the Ministry Education, Sports, Science, Culture Japan;          provide carbon sources energy growth. After <br /> JST PRESTO (to Y.N.); Institute Plant Microbial             chloroplast development, plants rely photosynthesis, <br /> Biology, Academia Sinica (to Y.N.). <br />    2                                                                          converts solar energy chemical energy and <br />      These authors contributed equally article. <br />    * Address correspondence kkobayashi@bio.c.u-tokyo.ac.jp             &#239;&#172;?xes carbon dioxide carbohydrates. Concomitant <br /> nakamura@gate.sinica.edu.tw.                                                  photosynthesis, photorespiration, performed by <br />    S.F. performed experiments, analyzed data, wrote           cooperation chloroplasts, peroxisomes, mi- <br /> article; K.K. conceived project, designed study, performed         tochondria, activated recycle 2-phosphoglycolate, <br /> the experiments, analyzed data, wrote majority             product oxygenation reaction instead carbox- <br /> article; Y.N. designed study, created materials, performed            ylation Rubisco (Peterhansel et al., 2010). <br /> the experiments, wrote article; H.W. supervised comple-              Chloroplast biogenesis involves remarkable devel- <br /> mented writing. <br />                                                                               opment thylakoid membranes consisting photo- <br />    The author responsible distribution materials integral the <br /> &#239;&#172;?ndings presented article accordance policy de- <br />                                                                               synthetic protein-pigment complexes membrane <br /> scribed Instructions Authors (www.plantphysiol.org) is:            lipid bilayer. In chloroplasts, monogalactosyldiacylglycerol <br /> Yuki Nakamura (nakamura@gate.sinica.edu.tw).                                  (MGDG) digalactosyldiacylglycerol (DGDG) account <br />    [W] <br />        The online version article contains Web-only data.             approximately 50 25 mol % thylakoid mem- <br />    [OPEN] <br />            Articles viewed online subscription.              brane lipids, respectively (Block et al., 1983). In addition to <br />    www.plantphysiol.org/cgi/doi/10.1104/pp.114.250050                         providing lipid matrix thylakoid membranes, these <br /> 1436 Plant Physiology&#195;&#8217;, November 2014, Vol. 166, pp. 1436&#226;&#8364;&#8220;1449, www.plantphysiol.org &#195;&#8220; 2014 American Society Plant Biologists. All Rights Reserved. <br />                                                                                               Role Galactolipids Organelle Differentiation <br />  <br />  <br />                                                                                galactolipids required structure func- <br />                                                                                tion photosynthetic complexes (Mizusawa Wada, <br />                                                                                2012). <br />                                                                                   The step MGDG synthesis catalyzed by <br />                                                                                MONOGALACTOSYLDIACYLGLYCEROL SYNTHASE <br />                                                                                (MGD), transfers Gal moiety UDP-Gal di- <br />                                                                                acylglycerol plastid envelopes (Kobayashi et al., 2009b). <br />                                                                                Because DGDG synthesized galactosylation of <br />                                                                                MGDG (D&#195;&#182;rmann Benning, 2002), MGD key <br />                                                                                enzyme biosynthesis of galactolipids <br />                                                                                and, therefore, biogenesis thylakoid membranes. <br />                                                                                Three MGD isoforms identi&#239;&#172;?ed Arabidopsis are <br />                                                                                MGD1, MGD2, MGD3 (Kobayashi et al., 2009b). The <br />                                                                                contribution MGD2 MGD3 galactolipid syn- <br />                                                                                thesis limited; isoforms localized outer <br />                                                                                envelope membrane chloroplasts (Awai et al., 2001) <br />                                                                                affect galactolipid synthesis phosphate- <br />                                                                                limited conditions (Kobayashi et al., 2009a). By contrast, <br />                                                                                MGD1 targeted chloroplast inner envelope <br />                                                                                membrane expressed actively photosynthetic <br />                                                                                tissues (Awai et al., 2001). The signi&#239;&#172;?cance MGD1 in <br />                                                                                galactolipid biosynthesis thylakoid membrane bio- <br />                                                                                genesis demonstrated study Arabi- <br />                                                                                dopsis mgd1 mutants: knockdown mutant mgd1-1 <br />                                                                                (Jarvis et al., 2000) knockout mutant mgd1-2 <br />                                                                                (Kobayashi et al., 2007). The mgd1-1 mutant reduces <br />                                                                                MGDG 58% wild-type plants chlo- <br />                                                                                roplasts fewer thylakoid membranes (Jarvis et al., <br />                                                                                2000). These defects result impaired thylakoid mem- <br />                                                                                brane energization decreased photoprotective capac- <br />                                                                                ity (Aronsson et al., 2008). Similar results recently <br />                                                                                reported MGD1 knockdown mutant (M18) to- <br />                                                                                bacco (Nicotiana tabacum), revealed in- <br />                                                                                volvement MGD1-mediated MGDG biosynthesis in <br />                                                                                photosynthetic electron transport PSII PSI <br />                                                                                (Wu et al., 2013). However, mgd1-2 knockout mutant <br />                                                                                accumulated negligible amounts MGDG and <br />                                                                                DGDG, resulting absence thylakoid mem- <br />                                                                                branes leaf plastids complete dysfunction of <br />                                                                                photosynthetic activities (Kobayashi et al., 2007). <br /> Figure 1. Characterization amiR-MGD1 seedlings. A, Schematic               Recently, revealed loss galactolipids in <br /> representation inducible amiR-MGD1 construction. The artificial          mgd1-2 caused strong down-regulation plastid- <br /> microRNA precursor fragment designed target MGD1 cloned           nucleus-encoded photosynthesis-associated genes, <br /> six times repeat pOp promoter. The rat glucocorticoid receptor (GR) <br />                                                                                partial complementation <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>mutation</span> by <br /> fused transcription activator LhG4, consists high-affinity <br /> DNA-binding mutant lac repressor (lacIHis-17) fused transcription- <br />                                                                                alternative MGD2/MGD3 pathway phos- <br /> activation domain II GAL4 Saccharomyces cerevisiae (Gal4-II),      phate starvation attenuated down-regulation of <br /> cloned control cauliflower mosaic virus (CaMV) 35S            genes (Kobayashi et al., 2013). Our &#239;&#172;?ndings sug- <br /> promoter. amiR-MGD1 designed target 20 nucleotides           gest galactolipid biosynthesis plays crucial role in <br /> stop codon MGD1 mRNA. LB, Left border; RB, right border; ter, tran-         coordinating formation photosynthetic protein- <br /> scriptional terminator; UTR, untranslated region. B, Photograph wild-type   pigment complexes development thylakoid <br /> (WT) amiR-MGD1 (L2, L4, L5, L7) seedlings grown DEX-free            membranes chloroplast biogenesis. However, <br /> (2DEX) 10 mM DEX-containing (+DEX) medium. C, Chlorophyll content           genes processes associated photo- <br /> per seedling amiR-MGD1 line grown 2DEX +DEX medium.              synthesis intertwined development of <br /> The horizontal line box represents median value distri- <br />                                                                                thylakoid membrane bilayers remains largely unknown <br /> bution. The box represent upper lower <br /> quartiles, respectively. The whiskers represent range (n = 24). D, Ex- <br />                                                                                strong pleiotropic effects consti- <br /> pression MGD1 wild type amiR-MGD1 line +DEX           tutive loss galactolipids plant growth devel- <br /> and 2DEX conditions. E, Expression MGD genes phosphate-            opment. Furthermore, detailed analysis been <br /> deficient L4 seedlings grown +DEX 2DEX conditions. Data          reported effect galactolipid biosynthesis and <br /> fold differences +DEX wild type D 2DEX control        consequent thylakoid membrane biogenesis dif- <br /> in E (means 6 SE, n = 3). Plants used B E 5-d-old seedlings.    ferentiation organelles peroxisomes and <br /> Plant Physiol. Vol. 166, 2014                                                                                                             1437 <br />  Fujii et al. <br />  <br />  <br /> mitochondria leaf development, tight        DEX treatment. By contrast, presence of <br /> metabolic coordination occurs chloroplasts        10 mM DEX, MGD1 expression decreased 70% 80% <br /> these organelles transition heterotrophic     L2, L4, L5 seedlings compared wild <br /> to photoautotrophic growth leaf cells (Peterhansel         type. Even L7 seedlings, did reduced <br /> et al., 2010; Nunes-Nesi et al., 2011).                       chlorophyll content DEX treatment, MGD1 expres- <br />     To gain insight role galactolipid biosyn-     sion decreased half wild-type level with <br /> thesis MGD1 coordinated formation           DEX treatment (Fig. 1D). <br /> photosynthetic complexes thylakoid membranes,                Because Arabidopsis possesses paralogs MGD1, <br /> we constructed arti&#239;&#172;?cial microRNA gene (amiR-MGD1)         MGD2 MGD3 (Kobayashi et al., 2009b), we <br /> targeting Arabidopsis MGD1 control           examined amiR-MGD1 transgene targets <br /> dexamethasone (DEX)-inducible promoter introduced         expression MGD2 MGD3 L4 seedlings. <br /> it Arabidopsis Landsberg erecta (Ler) ecotype. The   Under phosphate-suf&#239;&#172;?cient normal growth conditions, <br /> resulting plants showed wide phenotypic variation        expression MGD2 MGD3 low to <br /> cotyledon development strongly reduced MGD1 ex-          quantify wild type amiR-MGD1 lines. We <br /> pression DEX treatment, allowed analyze      grew seedlings phosphate-de&#239;&#172;?cient con- <br /> the function MGD1-mediated galactolipid bi-            dition up-regulate MGD2 MGD3, described <br /> osynthesis initial stage chloroplast bio-       previously (Awai et al., 2001; Kobayashi et al., 2004). In <br /> genesis. Moreover, investigated linkage          phosphate-de&#239;&#172;?cient L4 seedlings, DEX treatment sup- <br /> MGD1 function gene expression associated per-        pressed expression MGD1 MGD2 MGD3 <br /> oxisomal mitochondrial functions early seed-       (Fig. 1E). Therefore, amiR-MGD1 construct speci&#239;&#172;?cally <br /> ling development.                                             targeted MGD1 transcript suppression no <br />                                                               effect expression MGD2 MGD3. We con- <br />                                                               &#239;&#172;?rmed DEX responsiveness target speci&#239;&#172;?city of <br />                                                               amiR-MGD1 system. <br /> RESULTS AND DISCUSSION <br /> Speci&#239;&#172;?c Suppression MGD1 amiR-MGD1 <br /> Transgene Is DEX Dependent                                    Tight Link Chlorophyll Accumulation and <br />                                                               Galactolipid Biosynthesis amiR-MGD1 Seedlings <br />    To control timing MGD1 expression during <br /> chloroplast development, used microRNA-mediated             To analyze phenotypes L2 L4, we <br /> gene suppression technique (Schwab et al., 2006) com-         classi&#239;&#172;?ed DEX-treated seedlings lines three <br /> bined glucocorticoid-inducible (Craft et al.,   groups, white, pale green, green, using a <br /> 2005; Fig. 1A). We obtained seven DEX-responsive lines        chlorophyll-de&#239;&#172;?cient hemA1 mutant (Kobayashi et al., <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> 48 individual transgenic Arabidopsis lines          2013) color reference. The white seedlings paler <br /> harbor amiR-MGD1. We chose lines 2, 4, 5, 7 (L2, L4,      cotyledons hemA1 mutants, the <br /> L5, L7) subsequent analyses. First, examined       cotyledon color green seedlings similar of <br /> the effect continuous DEX treatment early           DEX-untreated seedlings (Fig. 2A). The remaining seed- <br /> seedling growth amiR-MGD1 transgenic lines. The            lings, intermediate phenotypes white and <br /> phenotypes amiR-MGD1 lines indistinguish-         green seedlings, de&#239;&#172;?ned pale green, were <br /> able wild type absence DEX (Fig. 1B). In   majority L2 L4 (Fig. 1B). This classi&#239;&#172;?ca- <br /> the presence 10 mM DEX, cotyledon greening im-         tion supported quantifying chlorophyll content in <br /> paired L2, L4, L5, wide variety color    class DEX-treated seedlings (Fig. 2B). <br /> phenotypes, albino-like wild-type-like, ob-         We compared expression MGD1 white and <br /> served lines. Meanwhile, L7 showed remark-        green seedlings grown 5 d presence DEX <br /> able difference wild type visible phenotype       DEX-untreated controls (Fig. 2C). In L2 and <br /> even presence 10 mM DEX. Chlorophyll mea-           L4 lines, MGD1 expression lower white seedlings <br /> surements 5-d-old seedlings supported results        green seedlings. Together results in <br /> (Fig. 1C). Although chlorophyll content deviated          Figure 1D, data link MGD1 <br /> considerably line, median values sub-        suppression impaired cotyledon greening sup- <br /> stantially lower DEX-treated L2, L4, L5 seedlings      port previous conclusion galactolipids nec- <br /> than DEX-untreated control line. However,      essary chloroplast development (Kobayashi et al., <br /> chlorophyll content DEX-treated        2007). Why wide variation cotyledon phenotypes is <br /> L7 seedlings DEX-untreated control.                   L2, L4, L5 remains elusive; amiR-MGD1 <br />    To address relationship amiR-MGD1          activity differ individually single line, <br /> phenotypes suppression MGD1 expression,            small differences amiR-MGD1 activity may <br /> we examined expression MGD1 5-d-old amiR-           lead considerable disparity chlorophyll accu- <br /> MGD1 seedlings quantitative reverse transcription          mulation chloroplast development cotyledons. <br /> (qRT)-PCR (Fig. 1D). In absence DEX, lines             Next analyzed galactolipid content color <br /> showed decreased MGD1 expression, con&#239;&#172;?rms               class 5-d-old DEX-treated L4 seedlings the <br /> that amiR-MGD1 transgene does suppress MGD1           untreated control (Fig. 2D). The proportion MGDG in <br /> 1438                                                                                            Plant Physiol. Vol. 166, 2014 <br />                                                                                        Role Galactolipids Organelle Differentiation <br />  <br />  <br />                                                                         reduced distribution MGDG membrane bilayers. <br />                                                                         In pale-green white seedlings, proportion of <br />                                                                         DGDG decreased MGDG, reduced <br />                                                                         proportion total galactolipids 80% white <br />                                                                         seedlings. The ratio non-bilayer-forming MGDG to <br />                                                                         bilayer-forming DGDG affect properties chlo- <br />                                                                         roplast membranes tightly regulated through <br />                                                                         unknown mechanism (D&#195;&#182;rmann Benning, <br />                                                                         2002). The inhibition MGD1 expression induce <br />                                                                         coordinated down-regulation DGDG biosyn- <br />                                                                         thesis paler amiR-MGD1 seedlings the <br />                                                                         MGDG-DGDG ratio maintained photosynthetic <br />                                                                         membranes. <br />                                                                            In addition examining membrane lipids, ana- <br />                                                                         lyzed levels TAG seedlings. The propor- <br />                                                                         tion TAG 1.5-fold higher white L4 seedlings <br />                                                                         DEX-untreated control, proportion <br />                                                                         green pale-green seedlings unchanged <br />                                                                         (Fig. 2D). As wild type, TAG white seedlings <br />                                                                         contained eicosenoic acid (20:1) major constituent <br />                                                                         (Supplemental Fig. S1), re&#239;&#172;&#8218;ect high reten- <br />                                                                         tion seed TAG white seedlings germination, <br />                                                                         eicosenoic acid speci&#239;&#172;?cally accumulated in <br />                                                                         TAG seed maturation (Li et al., 2006). The strong <br />                                                                         MGD1 suppression white seedlings decrease the <br />                                                                         demand diacylglycerol galactolipid biosynthesis, <br />                                                                         in&#239;&#172;&#8218;uencing TAG metabolism. However, unlike <br />                                                                         white seedlings, pale-green seedlings showed change <br />                                                                         TAG content, despite substantial decrease gal- <br /> Figure 2. Diverse phenotypes amiR-MGD1. A, Classification         actolipid content. Therefore, changes metabolic <br /> DEX-treated L4 seedlings color groups (green, pale green,    processes caused severe defects chloroplast bio- <br /> and white). B, Chlorophyll content color class DEX-treated   genesis affect TAG metabolism white seedlings. <br /> L2 L4 seedlings DEX-untreated (2DEX) control. FW, Fresh <br /> weight. C, MGD1 expression green white seedlings DEX- <br /> treated L2 L4. Data fold differences 2DEX control of <br /> each line. D, Glycerolipid composition L4 seedlings grown      Impaired Chloroplast Biogenesis Cotyledon Cell <br /> +DEX 2DEX conditions. In B D, data means 6 SE     Organization MGD1 Suppression <br /> independent experiments. Plants used experiments 5-d-old <br /> seedlings. <br />                                                                            Partial de&#239;&#172;?ciency MGDG MGD1 knockdown <br />                                                                         leads decreased amounts thylakoid membranes <br />                                                                         altered architecture (Jarvis et al., 2000; Wu et al., <br /> total glycerolipids decreased 25%, 77%, 85%                   2013), crucial lack galactolipids by <br /> green, pale-green, white seedlings, respectively,                MGD1 knockout results severely underdevel- <br /> compared DEX-untreated control. The MGDG                       oped internal membrane structures leaf plastids <br /> level seedlings associated chlo-                (Kobayashi et al., 2007). We examined effect MGD1 <br /> rophyll content (Fig. 2B), suggests MGDG                     suppression chloroplast development green and <br /> biosynthesis strongly affects chlorophyll accumulation                  white cotyledons DEX-treated L4 seedlings by <br /> and formation photosynthetic machinery,                     transmission electron microscopy. Consistent pre- <br /> presumably varying size and/or quality thylakoid              vious reports (Jarvis et al., 2000; Wu et al., 2013), chlo- <br /> membrane bilayers. In green seedlings, proportion                   roplasts green cotyledons, slightly reduced <br /> of DGDG did change, MGDG-DGDG                          MGDG content (Fig. 2D), appeared little mature <br /> ratio decreased. These &#239;&#172;?ndings agree previous                  DEX-untreated controls (Fig. 3, A B; <br /> reports Arabidopsis mgd1-1 mutant to-                    Supplemental Fig. S2, A&#226;&#8364;&#8220;D). However, white cotyledons <br /> bacco M18 line showing decrease MGD1                        severely undeveloped plastids internal <br /> activity primarily results loss MGDG (Jarvis                    membrane structures (Fig. 3C; Supplemental Fig. S2, <br /> et al., 2000; Wu et al., 2013). A possible explanation              E F). In addition, immature plastids white cotyle- <br /> the decrease MGDG-DGDG ratio partial defects                   dons oval &#239;&#172;&#8218;attened shapes lacked lens-like <br /> the MGD1 function decreased MGD1 activity                   structures typically observed mature chloroplasts in <br /> results overbalance DGDG biosynthesis relative                 green DEX-untreated seedlings. The defective phe- <br /> to MGDG biosynthesis consequent consumption                      notype chloroplasts white L4 cotyledons, is <br /> MGDG formation DGDG, leading                        milder mgd1-2 knockout mutant (Kobayashi <br /> Plant Physiol. Vol. 166, 2014                                                                                                      1439 <br />  Fujii et al. <br />  <br />  <br /> et al., 2007) severe mgd1-1 knockdown   affect transition leaf cells proliferation <br /> mutant (Jarvis et al., 2000), agrees phenotype    stage expansion stage, resulting decreased <br /> in galactolipid content, indicates direct impact     cell size young true leaves (Andriankaja et al., 2012). <br /> of MGD1-derived MGDG synthesis devel-             Our data reveal inhibition cell expansion also <br /> opment thylakoid membrane networks. In addition           occurred cotyledons impaired MGDG biosyn- <br /> to abnormal chloroplasts, oil bodies frequently     thesis postgerminative growth, suggests <br /> observed close contact glyoxysomes coty-      thylakoid membrane biogenesis conse- <br /> ledon cells white seedlings (Supplemental Fig. S2,    quent chloroplast development play role regulating <br /> G H), consistent high TAG level     cell development cotyledons basic struc- <br /> the seedlings.                                               ture established embryogenesis. <br />    The MGD1 suppression in&#239;&#172;&#8218;uenced structure <br /> and organization cotyledon cells (Fig. 3, D&#226;&#8364;&#8220;F). Both      Requirement Galactolipids Photosynthetic <br /> epidermal mesophyll cells L4 white cotyle-        Electron Transport <br /> dons irregular shapes, indented cell <br /> outlines, green cotyledons ap-             A partial loss MGDG (approximately 50% wild- <br /> peared similar DEX-untreated control. In          type levels) results decreased intersystem elec- <br /> white cotyledons, alignment mesophyll cells       tron transport PSII PSI (Wu et al., 2013) <br /> disordered, irregular organization palisade cells    increased conductivity thylakoid membranes <br /> and spongy cells. Moreover, size mesophyll cells      (Aronsson et al., 2008), absence MGDG <br /> in white cotyledons smaller green        (approximately 5% wild-type levels) causes severe <br /> cotyledons DEX-treated DEX-untreated seed-            disorder photosystem complexes complete de- <br /> lings. Because seeds amiR-MGD1 lines col-        &#239;&#172;?ciency photosynthetic electron transport (Kobayashi <br /> lected parents grown absence DEX,         et al., 2013). Because amiR-MGD1 L4 seedlings <br /> therefore embryonic development progressed nor-          showed wide variation MGDG content, 15% to <br /> mally lines, morphological disorder      75% wild-type levels DEX treatment (Fig. 2D), <br /> the white cotyledons attributed impaired         used line similar L2 line investigate <br /> postgerminative development. Considering cell           relationship MGDG biosynthesis the <br /> proliferation cotyledon formation nearly com-         functionality photosynthetic electron transport <br /> pleted embryogenesis cell expansion           chain. The maximum quantum ef&#239;&#172;?ciency PSII the <br /> mainly contributes postgerminative cotyledon          dark (Fv/Fm), represents intrinsic photosyn- <br /> growth light (Tsukaya et al., 1994; Stoynova-         thetic ef&#239;&#172;?ciency PSII, actual PSII ef&#239;&#172;?ciency <br /> Bakalova et al., 2004), morphological disorder       (FII) light (36 mmol photons m22 s21) were <br /> the dwarf phenotype white L4 cotyledons           determined individual cotyledons using chlo- <br /> be distorted mesophyll cell expansion          rophyll &#239;&#172;&#8218;uorescence imaging based pulse- <br /> postgerminative growth. The distorted organization        amplitude modulation techniques. Maximal chlorophyll <br /> mesophyll cells observed mutants         &#239;&#172;&#8218;uorescence dark adaptation (Fm) used a <br /> with defective chloroplast development (Wang et al.,         measure chlorophyll accumulation cotyledons to <br /> 2000; Wycliffe et al., 2005; Garcion et al., 2006; Sulmon    indicate severity amiR-MGD1-mediated sup- <br /> et al., 2006), including mgd1-2 mutant (Kobayashi        pression galactolipid biosynthesis. Unlike wild <br /> et al., 2007), showing tight developmental link       type, Fv/Fm FII decreased L2 L4, <br /> between chloroplasts host mesophyll cells. Fur-          correlated decrease Fm values (Fig. 4, <br /> thermore, application nor&#239;&#172;&#8218;urazon, inhibits          A B). These result suggests galactolipid bio- <br /> carotenoid biosynthesis impairs chloroplast devel-       synthesis MGD1 closely associated the <br /> opment, Arabidopsis seedlings recently       ef&#239;&#172;?ciency light utilization PSII. Very similar scatter <br />  <br />  <br /> Figure 3. Plastid ultrastructure and <br /> cotyledon morphology amiR-MGD1. <br /> Electron micrographs plastids (A&#226;&#8364;&#8220;C) and <br /> light micrographs cross sections (D&#226;&#8364;&#8220;F) <br /> from 5-d-old L4 seedlings shown. <br /> A D DEX-untreated (2DEX) <br /> control. Other images green (B and <br /> E) white (C F) cotyledons DEX- <br /> treated seedlings. Bars = 1 mm A C <br /> and 50 mm D F. <br />  <br />  <br />  <br />  <br /> 1440                                                                                            Plant Physiol. Vol. 166, 2014 <br />                                                                           Role Galactolipids Organelle Differentiation <br />  <br />  <br /> diagrams L2 L4 indicate lines       ef&#239;&#172;?ciency. Thus, white amiR-MGD1 cotyledons, <br /> are equivalent.                                     de&#239;&#172;?ciency MGDG primarily responsible <br />    To elucidate in&#239;&#172;&#8218;uence MGD1 de&#239;&#172;?ciency          substantial decrease maximal PSII ef&#239;&#172;?ciency. <br /> photosynthetic electron transport cotyledons       Next, analyzed quantum yields PSII low <br /> detail, photosynthetic parameters charac-     photosynthetically active radiation (45 mmol photons <br /> terized color classes 5-d-old DEX-treated L2   m22 s21). Whereas FII wild-type green seedlings <br /> seedlings. Fv/Fm decreased white seedlings,    increased 2 min actinic illumination started, <br /> followed pale-green green seedlings (Fig. 4C).      white pale-green seedlings remained very <br /> A dgd1 mutant showed slight decrease Fv/Fm            low measurement (Fig. 4D). A similar pat- <br /> (approximately 0.74) compared wild type        tern observed coef&#239;&#172;?cient photochemical <br /> (approximately 0.81), mutant contains         quenching (qP; Fig. 4E). Because qP represents re- <br /> only approximately 10% wild-type level           dox state primary electron-accepting plastoquinone <br /> DGDG (H&#195;&#182;lzl et al., 2009), indicates loss   PSII (QA) openness PSII, low qP in <br /> of DGDG minor impact maximal PSII        white pale-green seedlings suggests acceptor-side <br />  <br />  <br />                                                                          Figure 4. Photosynthesis characteristics 5-d-old <br />                                                                          amiR-MGD1. A B, Scatter diagrams Fm <br />                                                                          (x axis) Fv/Fm (A) FII (B; y axis) wild type <br />                                                                          (WT), L2, L4 DEX-treated conditions. Each <br />                                                                          point represents fluorescence data coty- <br />                                                                          ledon (n = 75&#226;&#8364;&#8220;100). C, Fv/Fm wild type and <br />                                                                          color class DEX-treated L2 seedlings. D H, <br />                                                                          Induction curves FII (D), qP (E), Fv9/Fm9 (F), FNO <br />                                                                          (G), FNPQ (H). I J, Transient fluorescence <br />                                                                          induction kinetics chlorophyll wild-type and <br />                                                                          white seedlings L2 L4 absence (I) or <br />                                                                          presence (J) 40 mM DCMU. Two inflections, la- <br />                                                                          beled J I, observed levels O <br />                                                                          (origin) P (peak) 2DCMU samples. <br />  <br />  <br />  <br />  <br /> Plant Physiol. Vol. 166, 2014                                                                                                 1441 <br />  Fujii et al. <br />  <br />  <br /> limitation PSII resulting defective electron trans-    increase presence DCMU faster L2 <br /> fer downstream PSII. MGDG de&#239;&#172;?ciency decreased               L4 wild type, suggests decreased <br /> levels cytochrome b6 f complex blocked          electron-accepting capacity QA transgenic <br /> intersystem electron transport tobacco MGD1             lines. The limitation electron-accepting capacity within <br /> mutant (Wu et al., 2013). Therefore, PSII acceptor-        PSII decreases utilization light energy photo- <br /> side limitation amiR-MGD1 cotyledons             synthesis increases excess energy, would <br /> attributed inhibition intersystem   dissipated nonregulated forms heat &#239;&#172;&#8218;uo- <br /> electron transport cytochrome b6f complex.          rescence DEX-treated amiR-MGD1 cotyledons, in- <br /> In addition, maximum quantum ef&#239;&#172;?ciency open             dicated high FNO value (Fig. 4G). Crystallographic <br /> PSII (Fv9/Fm9) constantly lower white          studies cyanobacteria revealed gal- <br /> pale-green seedlings wild-type green          actolipid molecules reside PSII complex that <br /> seedlings (Fig. 4F). The low Fv9/Fm9 indicates dysfunc-        present vicinity reaction center <br /> tional photochemical reactions open PSII          (Guskov et al., 2009; Umena et al., 2011). Therefore, de- <br /> line low Fv/Fm seedlings (Fig. 4C). In       &#239;&#172;?cient galactolipids PSII reaction center be <br /> the white pale-green seedlings, defective electron         involved dysfunction PSII photochem- <br /> transfer downstream PSII              ical reaction DEX-treated amiR-MGD1 cotyledons, <br /> cause decreased light utilization photosynthesis rep-      proposed mgd1-2 mutant (Kobayashi <br /> resented low FII.                                       et al., 2013). <br />    We evaluated quantum yield light-induced <br /> nonphotochemical quenching (FNPQ) quantum <br /> yield non-light-induced nonphotochemical quench-            Role MGD1 Formation Photosystems <br /> ing (FNO), represent proportion regulated <br /> and nonregulated dissipation light energy PSII,             Previously, revealed chlorophyll &#239;&#172;&#8218;uorescence <br /> respectively (Kramer et al., 2004). In white          analysis 77 K lack MGDG Arabi- <br /> pale-green seedlings, low FII inversely related        dopsis mgd1-2 knockout mutant caused severely dis- <br /> to high FNO (Fig. 4G), FNPQ stayed low          ordered formation PSII PSI complexes <br /> levels (Fig. 4H), indicates severe galactolipid     (Kobayashi et al., 2013). Meanwhile, knockdown of <br /> de&#239;&#172;?ciency causes nonregulated dissipation excess            MGD1 Arabidopsis tobacco slight <br /> light energy PSII low light. The high          impact chlorophyll &#239;&#172;&#8218;uorescence spectra 77 K <br /> value FNO shows limited photoprotective capac-          (Aronsson et al., 2008; Wu et al., 2013). To evaluate the <br /> ity DEX-treated amiR-MGD1 cotyledons, consistent            importance galactolipids formation photo- <br /> with observation accelerated photodamage         complexes, determined chlorophyll &#239;&#172;&#8218;uores- <br /> tobacco MGDG-de&#239;&#172;?cient mutant high-light con-             cence spectra 77 K white cotyledons L2 L4, <br /> ditions (Wu et al., 2013). Although pale-green seedlings       phenotypes intermediate mgd1-2 <br /> retained certain DGDG compared            knockout mutant (Kobayashi et al., 2007) the <br /> strong reduction MGDG, seedlings showed se-             knockdown mutants (Aronsson et al., 2008; Wu et al., <br /> vere photosynthetic defects similar white seedlings.        2013). In wild type, major emission peaks were <br /> Thus, de&#239;&#172;?ciency MGDG major cause              detected 682, 690, 729 nm (Fig. 5A). The peak at <br /> photosynthetic dysfunction seedlings.                 729 nm attributed PSI associated light- <br />    To dissect functionality PSII white coty-     harvesting complex I (LHCI), peaks at <br /> ledons amiR-MGD1 lines, analyzed transient           682 690 nm primarily originate CP43 and <br /> kinetics chlorophyll &#239;&#172;&#8218;uorescence logarithmic           CP47 PSII, respectively (Govindjee, 1995). In white <br /> timing series (Fig. 4I). Wild-type cotyledons showed         cotyledons L2 L4, peak wavelengths did not <br /> typical polyphasic &#239;&#172;&#8218;uorescence increase called             differ wild type, slight shift of <br /> origin-in&#239;&#172;&#8218;ection-intermediary peak-peak (OJIP) tran-           peak 732 nm, suggests global <br /> sient (Govindjee, 1995). The O-J phase J-I phase       formation photosystem complexes largely <br /> re&#239;&#172;&#8218;ect photochemical process PSII re-           perturbed L2 L4 white cotyledons. When <br /> duction plastoquinone pool, respectively,       &#239;&#172;&#8218;uorescence intensities normalized emission <br /> the I-P phase related process PSI. White          maximum 682 nm samples, emission <br /> cotyledons L2 L4 showed fast increase         peaks 729 nm lower L2 L4 for <br /> chlorophyll &#239;&#172;&#8218;uorescence photochemical               wild type. However, &#239;&#172;&#8218;uorescence intensity from <br /> phase (the O-J phase), indicates impaired electron       PSII vary chlorophyll concentration samples <br /> transfer excited chlorophylls downstream           self-absorbance PSII emission the <br /> the PSII. In presence 3-(3,4-dichlorophenyl)-1,1-       PSI complex. Therefore, compared 77 K chlo- <br /> dimethylurea (DCMU), rapid reduction total QA             rophyll &#239;&#172;&#8218;uorescence spectra membrane fractions <br /> occurs illumination inhibition electron       color class DEX-treated L2 seedlings the <br /> transfer QA secondary electron-accepting           chlorophyll concentration (0.8 mg mL21; Fig. 5B). <br /> plastoquinone PSII, resulting fast increase       Fluorescence PSI largely decreased white and <br /> in chlorophyll &#239;&#172;&#8218;uorescence in&#239;&#172;&#8218;ection, ob-           pale-green seedlings compared wild type <br /> served wild-type cotyledons (Fig. 4J). The &#239;&#172;&#8218;uorescence      (Fig. 5B), suggests decrease PSI proteins or <br /> 1442                                                                                              Plant Physiol. Vol. 166, 2014 <br />                                                                                         Role Galactolipids Organelle Differentiation <br />  <br />  <br /> reduced energy transfer PSI reaction center                  abundance PSI PSII maintained. Thus, <br /> antenna complexes. Even green seedlings,                    reduced PSI &#239;&#172;&#8218;uorescence 77 K (Fig. 5B) due <br /> reduced MGDG wild-type level DGDG,                          decrease PSI protein levels presumably to <br /> emission PSI decreased, suggests speci&#239;&#172;?c                    reduced energy transfer PSI reaction center from <br /> requirement MGDG formation PSI-                        LHCI PSII antenna system, which <br /> LHC complexes.                                                           dissociated reaction centers mem- <br />    Moreover, emission peaks PSII PSI                       brane fraction pale-green seedlings (Fig. 5B). <br /> both blue shifted L2 seedlings correlation                   The immunoblot analysis revealed core <br /> extent MGD1 suppression. This result suggests                     proteins (PsaA/B, D1, D2) decreased <br /> that LHCs membrane fractions L2 seed-                        antenna proteins (LHCA1, LHCA3, LHCB1, and <br /> lings largely dissociated core complexes                     LHCB3) pale-green seedlings. Similar results were <br /> both photosystems. Considering emission peaks                       observed phosphate-de&#239;&#172;?cient mgd1-2 mutant <br /> were blue shifted intact white seedlings (Fig. 5A),               (Kobayashi et al., 2013). Thus, MGDG required <br /> LHCs interact weakly photosystems                       accumulation maintenance photosystem <br /> vivo dissociated core complexes                       core complexes. <br /> membrane fractionation. Indeed, vitro analysis <br /> revealed MGDG intensi&#239;&#172;?es physical interac- <br /> tions LHCII PSII core complexes                          Involvement Galactolipid Biosynthesis Coordinated <br /> increases energy coupling (Zhou et al., 2009).                     Regulation Genes Associated Photosynthesis, <br /> Thus, galactolipids, particularly MGDG,                       Photorespiration, Glyoxylate Cycle during <br /> essential strong interactions photosystem                    Cotyledon Development <br /> core complexes LHCs. <br />    To address MGD1 suppression affects                      We recently reported mgd1-2 mutant that <br /> abundance membrane photosynthetic proteins,                        galactolipid biosynthesis subsequent membrane <br /> performed immunoblot analysis DEX-treated L2 pale-                    biogenesis inside plastid strongly in&#239;&#172;&#8218;uence the <br /> green seedlings. Total membrane proteins (20 mg)                    expression plastid- nucleus-encoded <br /> pale-green seedlings analyzed di-                   photosynthetic genes independent photosynthesis <br /> lution series (1, 5, 20 mg) membrane proteins                (Kobayashi et al., 2013). To investigate effect the <br /> the wild type. Although PSI (PsaA, PsaB, LHCA1,                     MGD1 suppression photosynthetic gene expression <br /> and LHCA3) PSII (D1, D2, LHCB1, LHCB3)                           cotyledon development, examined ex- <br /> proteins largely decreased pale-green seedlings                       pression psaA psbA, plastid-encoded <br /> compared wild type, balance protein                      genes PsaA D1 PSI PSII core com- <br />                                                                          plexes, respectively. Consistent substantial <br />                                                                          decrease PsaA D1 proteins (Fig. 5C), psaA <br />                                                                          psbA down-regulated white cotyledons and <br />                                                                          green cotyledons DEX-treated L4 seedlings (Fig. 6A). <br />                                                                          In developing plastids, large DNA-protein complex <br />                                                                          named nucleoids exists close contact envelope <br />                                                                          thylakoid membranes (Sato, 2001). Considering <br />                                                                          plastid transcription regulated activity of <br />                                                                          RNA polymerase complexes structural <br />                                                                          organization plastid DNA (Sekine et al., 2002), the <br />                                                                          galactolipid biosynthesis plastid envelope and <br />                                                                          subsequent thylakoid membrane biogenesis affect <br />                                                                          transcriptional activities nucleoid. In fact, we <br />                                                                          previously showed galactolipid biosynthesis mod- <br />                                                                          i&#239;&#172;?es morphology nucleoids leaf plastids, which <br />                                                                          suggests association galactolipid biosyn- <br />                                                                          thesis nucleoid activity (Kobayashi et al., 2013). <br />                                                                             We examined expression LHCB6 and <br />                                                                          CHLH, nuclear genes encoding LHCII subunit <br />                                                                          6 H subunit magnesium chelatase involved in <br /> Figure 5. Role MGD1 formation photosystem complexes.        chlorophyll biosynthesis, respectively. Coordinately with <br /> A B, The 77 K chlorophyll fluorescence spectra wild-type (WT)     plastid-encoded genes (psaA psbA), nucleus-encoded <br /> and white seedlings L2 L4 presence DEX (A) the <br />                                                                          photosynthetic genes (LHCB6 CHLH) down- <br /> membrane fraction (0.8 mg mL21 chlorophyll) wild type and <br /> each color class DEX-treated L2 seedlings (B). C, Immunoblot <br />                                                                          regulated white green cotyledons DEX- <br /> analysis membrane photosynthetic proteins wild-type L2       treated L4 seedlings (Fig. 6A). Consistent their <br /> pale-green seedlings presence DEX. A 20-mg aliquot total    galactolipid phenotypes, suppression plastid- <br /> membrane protein L2 seedlings compared dilution      nucleus-encoded genes stronger white than <br /> series (20, 5, 1 mg) membrane proteins wild type.        green cotyledons, suggests MGDG <br /> Plant Physiol. Vol. 166, 2014                                                                                                       1443 <br />  Fujii et al. <br />  <br />  <br />                                                                          et al., 2000; Cornah et al., 2004). Our qRT-PCR analysis <br />                                                                          revealed photorespiratory genes GOX1, HPR1, <br />                                                                          GLDP1, SHM1 down-regulated concomitantly <br />                                                                          photosynthetic genes DEX treatment, particu- <br />                                                                          larly white seedlings (Fig. 6B). The photorespiratory <br />                                                                          metabolism associated photosynthetic functional- <br />                                                                          ities (Peterhansel et al., 2010); therefore, co-down-regulation <br />                                                                          photorespiratory genes photosynthesis- <br />                                                                          associated genes reasonable. <br />                                                                              Because photorespiratory genes down- <br />                                                                          regulated photosynthesis-associated nu- <br />                                                                          clear genes nor&#239;&#172;&#8218;urazon-treated seedlings through <br />                                                                          plastid signaling pathways (Strand et al., 2003), the <br />                                                                          MGD1 suppression down-regulate genes co- <br />                                                                          ordinately nuclear photosynthetic genes acti- <br /> Figure 6. Expression genes associated organelle functions in <br />                                                                          vating plastid signaling. By contrast, expression of <br /> 5-d-old cotyledons. qRT-PCR analysis expression genes as- <br /> sociated photosynthesis (A), photorespiration (B), glyox- <br />                                                                          glyoxysomal genes ICL MLS remained higher <br /> ylate cycle (C) DEX-untreated (2DEX) DEX-treated green white    levels DEX-treated green white seedlings <br /> L4 seedlings shown. Data fold differences 2DEX control   untreated control (Fig. 6C). The low expres- <br /> (means 6 SE, n = 3).                                                     sion photorespiratory genes high expression <br />                                                                          glyoxylate cycle-associated genes suggest the <br /> biosynthesis tightly linked photosynthetic gene ex-                differentiation peroxisomes mitochondria leaf- <br /> pression chloroplast biogenesis. Inhibition                    speci&#239;&#172;?c types cotyledon development inhibited <br /> chloroplast protein translation results down-regulated                amiR-MGD1-mediated suppression galactolipid <br /> nuclear photosynthetic gene expression early                      biosynthesis. Considering glyoxysomes were <br /> seedling growth, suggests involvement plastid                observed close oil bodies 5-d-old white L4 cotyle- <br /> translation plastid signaling down-regulates nuclear             don cells (Supplemental Fig. S2, G H), is <br /> photosynthetic gene expression response chloroplast                typically observed early seed germination <br /> dysfunction (Nott et al., 2006). Furthermore, plastid gene               stage case wild type (Hayashi et al., 2001), <br /> expression mediated nucleus-encoded s factors plays                   white plants b-oxidation the <br /> a role plastid signaling (Woodson et al., 2013). There-               glyoxylate cycle activities high maintain heterotrophic <br /> fore, possibility decreased expression                   growth response disrupted chloroplast biogenesis. <br /> of plastid-encoded genes DEX-treated amiR-MGD1                        Of note, observed high expression glyoxysomal <br /> seedlings triggers down-regulation photosyn-                      ICL MLS genes green DEX-treated seedlings <br /> thetic genes nucleus plastid signaling                    (Fig. 6C), did extra TAG pos- <br /> pathways.                                                                session cotyledons (Fig. 2D), implies the <br />    During cotyledon greening, peroxisomes mito-                      glyoxysomal gene expression simply associated <br /> chondria transform functions concert chlo-                 TAG levels appears linked lipid <br /> roplast development (Hayashi Nishimura, 2006). To                    metabolism plastids. Furthermore, high expression <br /> examine MGD1 suppression affects organ-                      ICL MLS strong down-regulation of <br /> elle differentiation early seedling development,                  GOX1 HPR1 green seedlings indicates the <br /> we analyzed expression genes associated                      expression glyoxylate cycle genes regulated <br /> peroxisomal mitochondrial functions 5-d-old L4                    independently photorespiratory genes per- <br /> seedlings (Fig. 6, B C). GOX1 HPR1,                        oxisome differentiation cotyledons. <br /> encode glycolate oxidase hydroxypyruvate reduc- <br /> tase photorespiratory pathway, respectively, were <br /> used leaf peroxisome markers. In addition, GLDP1                      Requirement Galactolipids Initiation of <br /> and SHM1, encode P protein Gly de-                      Chloroplast Biogenesis <br /> carboxylase complex Ser hydroxymethyltransferase, <br /> respectively, used markers functionality                    To elucidate role MGD1 expression galac- <br /> of photorespiration mitochondria. All photo-                 tolipid biosynthesis initial stage chloroplast <br /> respiratory genes reported speci&#239;&#172;?cally ex-                     development seed germination, suppressed <br /> pressed photosynthetic tissues (Kamada et al., 2003;                  MGD1 expression early chloroplast biogenesis by <br /> Voll et al., 2006; Engel et al., 2007; Timm et al., 2008).               adding 10 mM DEX lapse certain period after <br /> Meanwhile, ICL MLS, encode isocitrate lyase                    seeding. Application DEX 36 h seeding <br /> and malate synthase glyoxylate cycle, respectively,               strongly suppressed chlorophyll accumulation 5-d-old <br /> were used markers glyoxysomes. ICL MLS                        seedlings, DEX application 48 h seeding <br /> are transiently expressed earliest post-                 decreased inhibition chlorophyll accumulation <br /> germinative growth stages, peroxisomes actively                     (Fig. 7A). In 5-d-old seedlings DEX added 72 h after <br /> operate glyoxylate cycle glyoxysomes (Eastmond                    seeding, chlorophyll content equal in <br /> 1444                                                                                                          Plant Physiol. Vol. 166, 2014 <br />                                                                                                Role Galactolipids Organelle Differentiation <br />  <br />  <br />                                                                            severe retardation MGD1 expression inhibited <br />                                                                            afterward. MGD1 expression crucial particularly <br />                                                                            initiation chloroplast biogenesis cotyledons <br />                                                                            required seedlings grow. <br />  <br />  <br />                                                                            Effect MGD1 Suppression Development of <br />                                                                            True Leaves <br />  <br />                                                                              The continuous induction amiR-MGD1 affected <br />                                                                            development true leaves L2 L4 seedlings, <br />  <br />  <br />  <br />  <br /> Figure 7. Involvement galactolipids initial organelle development. <br /> A, Inhibition chlorophyll accumulation 5-d-old L2 L4 seedlings <br /> by DEX treatment different times 0 72 h seeding. Chlo- <br /> rophyll content 5-d-old seedlings grown continuously absence <br /> of DEX shown untreated control (2DEX). The horizontal line in <br /> each box represents median value distribution. The and <br /> bottom box represent upper lower quartiles, respec- <br /> tively. The whiskers represent range (n = 24). B, MGD1 expression in <br /> 5-d-old L2 L4 seedlings DEX added 72 h seeding (72 h) or <br /> grown continuously absence DEX (2DEX). C, Expression of <br /> genes associated photosynthesis, photorespiration, glyox- <br /> ylate cycle 5-d-old L4 seedlings DEX added 72 h seeding or <br /> grown continuously absence DEX. In B C, data fold <br /> differences 2DEX control (means 6 SE, n = 3). <br />  <br />  <br />  <br /> DEX-untreated controls, MGD1 expression was <br /> suppressed nearly minimal levels amiR- <br /> MGD1 constructs (Fig. 7B). Even seedlings <br />                                                                            Figure 8. Characteristics amiR-MGD1 true leaves. A, Eight-day-old <br /> were treated DEX 5 d 72 h seeding, <br />                                                                            seedlings wild type (WT), L2, L4 grown continuously DEX- <br /> chlorophyll content did differ substantially that <br />                                                                            containing medium. B, Chlorophyll content true leaves 14-d-old L4 <br /> in untreated controls (Supplemental Fig. S3A). Consis-                     seedlings grown DEX-free (2DEX) DEX-containing (+DEX) medium. <br /> tent chlorophyll levels, expression                   FW, Fresh weight. C, Changes Fv/Fm true leaves (left) cotyledons <br /> nucleus- plastid-encoded genes impaired                     (right) DEX-treated wild-type, L2, L4 seedlings day 7 day 10 <br /> 5- 8-d-old L4 seedlings treated DEX 72 h                    seeding. Data means 6 SE (n = 10). D, MGD1 expression true <br /> seeding (Fig. 7C; Supplemental Fig. S3B), suggests                   leaves 10-d-old L4 seedlings grown 2DEX +DEX medium. <br /> that galactolipid biosynthesis presumably thylakoid                    E, Proportion galactolipids total glycerolipids true leaves 14-d-old <br /> membrane biogenesis initiation chloroplast                       L4 seedlings grown 2DEX +DEX medium. F, Relative radioactivity in <br /> biogenesis suf&#239;&#172;?cient ensure induction                        MGDG total radioactivity lipid fractions extracted [14C]acetate- <br />                                                                            labeled leaves L4 seedlings grown 2DEX +DEX medium. True <br /> photosynthesis-associated genes. The expression genes <br />                                                                            leaves 10-d-old seedlings cotyledons 5-d-old seedlings were <br /> associated peroxisomal mitochondrial func- <br />                                                                            used. G, Expression genes associated photosynthesis, photorespir- <br /> tions unchanged DEX treatment                        ation, glyoxylate cycle true leaves 10-d-old L4 seedlings <br /> initial seedling growth (Fig. 7C). Therefore, MGDG                    grown 2DEX +DEX medium. *Not detected. In D G, data are <br /> is properly synthesized MGD1 beginning                        fold differences DEX-untreated control. In B D G, data are <br /> chloroplast biogenesis, development chloroplasts                    means 6 SE independent experiments. All +DEX amiR- <br /> and organelles cotyledons proceeds                        MGD1 seedlings used experiments white cotyledons. <br />  <br /> Plant Physiol. Vol. 166, 2014                                                                                                                         1445 <br />  Fujii et al. <br />  <br />  <br /> although effect milder cotyledons. In    germination (Mans&#239;&#172;?eld Briarty, 1996), true <br /> L2 L4 seedlings white cotyledons, &#239;&#172;?rst          leaves, proplastids shoot apical meristem gradually <br /> second true leaves appeared smaller paler     differentiate mature chloroplasts leaf biogene- <br /> in wild type presence DEX (Fig. 8A). Con-      sis (Charuvi et al., 2012). Thus, requirement MGD1 <br /> sistent color, chlorophyll content L4 true leaves    gene expression suf&#239;&#172;?cient galactolipid accumulation <br /> was decreased DEX treatment (Fig. 8B). When         vary organs. <br /> PSII activity monitored measuring Fv/Fm           To address reduced MGDG biosynthesis <br /> presence DEX, consistently detected lower values       modi&#239;&#172;?es gene expression true leaves observed in <br /> in true leaves L2 L4 white seedlings    cotyledons (Fig. 6), examined expression genes <br /> in wild type (Fig. 8C). However, lines,     associated organelle functions true leaves the <br /> PSII impairment true leaves considerably          10-d-old white L4 seedlings (Fig. 8G). The expression of <br /> weaker cotyledons, shows dimin-          genes associated photosynthesis photorespira- <br /> ished impact amiR-MGD1 true leaves.                    tion decreased DEX treatment, psbA <br />    To evaluate suppression effect amiR-MGD1       expression appeared increased true leaves. <br /> transgene MGD1 expression true leaves, grew     In addition, ICL expression undetectable true <br /> L4 seedlings white cotyledons DEX-containing         leaves regardless DEX treatment, implies no <br /> medium 10 d compared MGD1 expression              prominent expression glyoxysomal genes response <br /> the true leaves DEX-untreated control.      MGD1 suppression true leaves. Our data indicate <br /> DEX treatment suppressed MGD1 expression              MGD1 suppression differently affects gene expres- <br /> true leaves (Fig. 8D) level white      sion cotyledons true leaves. Impaired <br /> cotyledons 5-d-old seedlings (Fig. 2C), weak       MGDG biosynthesis mgd1-2 mutant strongly <br /> effect DEX treatment true leaves       down-regulates nucleus- plastid-encoded <br /> due release MGD1 suppression. To investigate       photosynthetic genes true leaves, indicates a <br /> whether reduced MGD1 expression suf&#239;&#172;?cient            requirement galactolipid biosynthesis photosyn- <br /> for galactolipid biosynthesis true leaves, analyzed    thetic gene expression true leaves (Kobayashi et al., <br /> galactolipid content true leaves white L4 class    2013). Therefore, 75% reduction MGD1 ex- <br /> seedlings grown 14 d presence DEX (Fig. 8E).   pression suf&#239;&#172;?cient down-regulate photo- <br /> The proportion MGDG total membrane glycero-            synthetic genes cotyledons true leaves. <br /> lipids decreased 25% DEX-treated true leaves       Distinct regulation chloroplast differentiation between <br /> as compared DEX-untreated control,          cotyledons true leaves evident identi&#239;&#172;?- <br /> the proportion DGDG unchanged. Consistent             cation mutants cotyledon-speci&#239;&#172;?c defects <br /> with lipid data, MGDG synthesis rate estimated       chloroplast biogenesis (Pogson Albrecht, 2011). <br /> by [14C]acetate incorporation partially decreased     Most cotyledon-speci&#239;&#172;?c mutants impaired <br /> true leaves DEX treatment, cotyledons     transcription translation plastid-encoded genes <br /> was low compared DEX-untreated control         (Shirano et al., 2000; Yamamoto et al., 2000; Ishizaki <br /> (Fig. 8F). These data indicate MGDG biosynthesis        et al., 2005; Albrecht et al., 2006; Ruppel Hangarter, <br /> was decreased true leaves amiR-MGD1, al-          2007; Chi et al., 2010; Woodson et al., 2013) ho- <br /> though suppression effect weaker    meostasis plastid proteins (Shimada et al., 2007; <br /> white cotyledons. The reason MGDG synthesis          Albrecht et al., 2008; Tanz et al., 2012), suggests <br /> activity largely differs MGD1 transcriptional       cotyledons possess particular set components <br /> level true leaves remains unclear. MGD2           essential photosynthetic gene expression early <br /> MGD3 expression appeared decreased true leaves       seedling growth, galactolipid biosynthesis by <br /> of amiR-MGD1, contribution leaf MGDG            MGD1 involved. <br /> synthesis negligible (Supplemental Fig. S4; <br /> Kobayashi et al., 2009a, 2013). The MGD1 enzyme re- <br /> duced activated vitro chloroplast thioredoxins,    MATERIALS AND METHODS <br /> which coupled photosynthetic activity in <br /> vivo (Yamaryo et al., 2006). Thus, difference pho-    Plant Materials Growth Conditions <br /> tosynthetic activity (Fig. 8C) explain discrepancy       All plants used study Ler ecotype Arabidopsis <br /> in MGDG synthesis rate true leaves coty-         (Arabidopsis thaliana). Seeds surface sterilized cold treated 4&#194;&#176;C <br /> ledons DEX-treated amiR-MGD1. In addition,            4 d dark seeding. Plants grown Murashige Skoog <br /> MGDG-synthesizing activity regulated anionic           (MS) medium (adjusted pH 5.7 KOH) containing 1% (w/v) Suc so- <br />                                                              lidi&#239;&#172;?ed 0.8% (w/v) agar experiments Figures 1C 7, <br /> membrane lipids (Dubots et al., 2010) be <br />                                                              A B, liquid form medium used. All plants were <br /> in&#239;&#172;&#8218;uenced substrate supply. These factors        grown 23&#194;&#176;C continuous white light (30 mmol photons m22 s21) a <br /> affect MGDG synthetic activity differently       growth chamber. For DEX treatment, DEX (Wako; http://www.wako-chem. <br /> true leaves cotyledons. Processes chloroplast dif-    co.jp/english/) added &#239;&#172;?nal concentration 10 mM medium <br /> ferentiation differ considerably cotyledons      50 mM stock dimethyl sulfoxide. Unless stated otherwise, DEX was <br />                                                              added plants start cold treatment seed stage. <br /> true leaves (Pogson Albrecht, 2011); cotyledons,          For analyses Figures 1C 7, A B, seeds cold treated deionized <br /> plastids partially developed embryogenesis differ-    water dark seeded liquid MS medium grown 3 d the <br /> entiate simultaneously chloroplasts light-induced    chamber gentle rotation (90 rpm). Only germinated seedlings grown <br />  <br /> 1446                                                                                                         Plant Physiol. Vol. 166, 2014 <br />                                                                                                              Role Galactolipids Organelle Differentiation <br />  <br />  <br /> on solidi&#239;&#172;?ed MS medium 2 d synchronize germination time           Chlorophyll Determination <br /> the experiments. For experiment Figure 7, A B, DEX added the <br /> medium lapse time start light growth.                       For chlorophyll determination Figures 2B 8B, seedlings crushed in <br />     For phosphate-de&#239;&#172;?cient condition Figure 1E, concentration             liquid nitrogen homogenized 80% (v/v) acetone, debris re- <br /> KH2PO4 reduced 10 mM MS medium. Seedlings DEX-                    moved centrifugation 15,000 rpm 5 min. The chlorophyll content of <br /> untreated control grown phosphate-de&#239;&#172;?cient medium DEX                   samples determined spectrophotometrically measuring absorbance <br /> for 5 d germination. For DEX treatment, seedlings &#239;&#172;?rst grown              supernatant 720, 663, 645 nm Ultrospec 2100 pro spec- <br /> on phosphate-de&#239;&#172;?cient medium DEX 3 d phos-                      trophotometer (GE Healthcare) according previous report (Melis et al., <br /> phate-de&#239;&#172;?cient medium containing 10 mM DEX 2 d.                             1987). For analyses Figures 1C 7A, chlorophyll extracted from <br />                                                                                         single intact seedlings incubating seedling 1 mL 80% (v/v) ac- <br />                                                                                         etone 4&#194;&#176;C 3 d. Chlorophyll content single seedlings determined <br /> Construction Transgenic Lines                                                        measuring &#239;&#172;&#8218;uorescence emission 666 nm 440-nm excitation with <br />                                                                                         FP-6200 spectro&#239;&#172;&#8218;uorometer (JASCO; http://www.jascoinc.com/) a <br />     amiR-MGD1 designed (miR-sense, 59-TAGATTATTAGGCAGTG-                            chlorophyll sample known concentration used standard. <br /> CAAC-39) according WMD2 software instructions Web site <br /> (http://wmd2.weigelworld.org/cgi-bin/mirnatools.pl.). The assembled arti- <br /> &#239;&#172;?cial microRNA precursor fragment cloned pENTR entry vector                    Photosynthetic Chlorophyll Fluorescence Analysis <br /> (Invitrogen; http://www.lifetechnologies.com/), pOp6 inserted <br /> at NotI site. We cloned 35S::GRLhG4 fragment AscI site                  Photosynthetic activity determined chlorophyll &#239;&#172;&#8218;uorescence <br /> the pENTR vector selected orientation promoters                     imaging (IMAGING-PAM MAXI version; Walz; http://www.walz. <br /> outgoing (Fig. 1A). Then, 35S::GRLhG4/pOp6::amiR-                   com/; Figs. 4, A B, 8C) chlorophyll &#239;&#172;&#8218;uorometer (JUNIOR-PAM; <br /> MGD1 fragment pENTR transferred pBGW destination vector                   Walz; Fig. 4, C&#226;&#8364;&#8220;H). Seedlings MS agar plates (Figs. 4, A B, 8C) or <br /> by LR recombination. Wild-type plants (Ler ecotype) transformed               detached cotyledons (Fig. 4, C&#226;&#8364;&#8220;H) dark incubated 15 min before <br /> the resulting plasmid transfection Agrobacterium tumefaciens strain            measurement. After measuring minimum chlorophyll &#239;&#172;&#8218;uorescence (Fo) the <br /> GV3101.                                                                                 dark, Fm determined saturating pulse. After treatment actinic <br />     Forty-eight transgenic plant lines isolated 0.1% (w/v) Basta              light (36 45 mmol photons m22 s21 IMAGING-PAM JUNIOR- <br /> spray T1 generation. At T2 generation, Basta-resistant seedlings             PAM, respectively) 10 min, stationary &#239;&#172;&#8218;uorescence (F) maximum <br /> germinated soil sprayed 1 mM DEX solution (pH 7) daily                &#239;&#172;&#8218;uorescence actinic light (Fm9) determined. Minimal &#239;&#172;&#8218;uores- <br /> until true leaves appeared. Seven lines showing color change true leaves             cence illuminated samples (Fo9) computed approximation of <br /> with DEX treatment isolated candidates functional amiR-MGD1                  Oxborough Baker (1997): Fo9 = Fo/(Fv/Fm + Fo/Fm9). From &#239;&#172;&#8218;uo- <br /> transgenic lines. Seeds harvested individually seven lines showed            rescence yields, photosynthetic parameters calculated follows (van <br /> approximately 75% Basta resistance, indicating single-copy transgene in-              Kooten Snel, 1990; Maxwell Johnson, 2000): Fv/Fm = (Fm 2 Fo)/Fm, <br /> sertion. We harvested independent T3 seeds line                  Fv9/Fm9 = (Fm9 2 Fo9)/Fm9, FII = (Fm9 2 F)/Fm9, qP = (Fm9 2 F)/(Fm9 2 Fo9). The <br /> (56 lines all) selected showing 100% Basta resistance                actual FII transformed product qP Fv9/Fm9: FII = (Fm9 2 <br /> homozygous transgene. In study, lines 2, 4, 5, 7 used          F)/Fm9 = qP 3 Fv9/Fm9. FNPQ FNO determined according the <br /> detailed analyses.                                                                      method Kramer et al. (2004). Measurement parameters IMAGING-PAM <br />                                                                                         follows: measuring light intensity = 2, measuring light frequency = 4, <br />                                                                                         damping = 4, gain = 1, saturation pulse intensity = 10. For quanti&#239;&#172;?cation, <br /> qRT-PCR Analysis                                                                        averaged &#239;&#172;&#8218;uorescence values circular area collected cotyle- <br />                                                                                         don true leaf seedling using software ImagingWin (Walz). For <br />     Total RNA extracted using RNeasy Plant Mini kit (Qiagen;                 analysis JUNIOR-PAM, automated induction program provided <br /> http://www.qiagen.com/). Genomic DNA digestion reverse tran-                        WinControl-3 software (Walz) used default settings for <br /> scription involved PrimeScript RT reagent kit gDNA Eraser                      measuring light saturation pulse. <br /> (TaKaRa Bio; http://www.takara-bio.com/). Complementary DNA am-                             For chlorophyll &#239;&#172;&#8218;uorescence induction experiments (Fig. 4, I J), 5-d-old <br /> pli&#239;&#172;?cation involved Thunderbird PreMix kit (Toyobo; http://www.                     seedlings dark incubated 5 min experiments. Eight seedlings <br /> toyobo-global.com/) 200 n M gene-speci&#239;&#172;?c primers (Supplemental                      used experiment batch. When required, seedlings were <br /> Table S1). Thermal cycling consisted initial denaturation step 95&#194;&#176;C            in&#239;&#172;?ltrated 40 mM DCMU 150 mM sorbitol depression dark <br /> for 10 s followed 40 cycles 5 s 95&#194;&#176;C 30 s 60&#194;&#176;C. Signal detection        incubation. Chlorophyll &#239;&#172;&#8218;uorescence transients measured loga- <br /> and quanti&#239;&#172;?cation performed duplicate use MiniOpticon (Bio-               rithmic time series 30 ms 10 s onset strong actinic light <br /> Rad; http://www.bio-rad.com/). The relative abundance transcripts                (1,650 mmol photons m22 s21) light-emitting diode pump-probe spec- <br /> ampli&#239;&#172;?ed normalized constitutive expression level ACTIN8                  trometer (JTS-10; BioLogic; http://www.bio-logic.info/). <br /> (Pfaf&#239;&#172;&#8218;, 2001). Three independent biological experiments performed                  Fluorescence emission spectra chlorophyll proteins 77 K obtained <br /> each sample.                                                                            directly plant tissues (Fig. 5A) membrane fractions (Fig. 5B) in <br />                                                                                         liquid nitrogen using RF-5300PC spectro&#239;&#172;&#8218;uorometer (Shimadzu) under <br />                                                                                         435-nm excitation. To prepare membrane fractions, seedlings pulverized <br /> Lipid Analysis                                                                          liquid nitrogen homogenized cold buffer (0.33 M sorbitol, 5 mM <br />                                                                                         MgCl2, 5 mM EDTA, 50 mM HEPES-KOH, pH 7.6). The homogenate was <br />     Total lipids extracted seedlings crushed powder liquid <br />                                                                                         &#239;&#172;?ltered single layer Miracloth (Merck Millipore; http://www. <br /> nitrogen separated thin-layer chromatography solvent <br />                                                                                         merckmillipore.com/) gentle hand pressure. After centrifugation at <br /> system acetone:toluene:water (136:45:12, v/v/v) described (Kobayashi <br />                                                                                         5,000g 10 min 4&#194;&#176;C, supernatant discarded pellet was <br /> et al., 2006). Lipids visualized 0.01% (w/v) primuline 80% (v/v) <br />                                                                                         resuspended cold buffer obtain 0.8 mg mL21 chlorophyll-containing <br /> acetone UV light. MGDG, DGDG, TAG, mixture glycerolipids <br />                                                                                         membrane fractions. <br /> consisting phosphatidylcholine, phosphatidylethanolamine, phosphatidylinositol, <br /> phosphatidylglycerol, sulfoquinovosyldiacylglycerol isolated silica <br /> gel plates. Fatty acids lipid fraction methyl esteri&#239;&#172;?ed incubation   Microscopy Analysis <br /> 1 M HCl methanol 85&#194;&#176;C 2 h quanti&#239;&#172;?ed gas chromatography <br /> (GC-17A; Shimadzu; http://www.shimadzu.com/) myristic acid                       Arabidopsis seedlings &#239;&#172;?xed 2.5% (v/v) glutaraldehyde 4% (v/v) <br /> internal standard.                                                                      paraformaldehyde 0.1 M sodium phosphate buffer, pH 7, room temperature <br />     For [14C]acetate labeling assay, cotyledons 5-d-old seedlings true leaves   4 h. After rinses phosphate buffer (three times 20 min each), samples <br /> from 10-d-old seedlings incubated 30 min sodium [1-14C]acetate (1.9       post&#239;&#172;?xed 1% (w/v) OsO4 buffer 4 h room temperature <br /> MBq mL21). Lipids extracted separated thin-layer chromatography             rinsed phosphate buffer (three times 20 min each). Samples were <br /> as described above. Radioactivity detected Imaging Plate (Fuji&#239;&#172;?lm; http://           dehydrated ethanol series propylene oxide, embedded Spurr&#226;&#8364;&#8482;s resin, <br /> www.fuji&#239;&#172;?lm.com/) imaging analyzer (BAS-2500; Fuji&#239;&#172;?lm) quanti&#239;&#172;?ed             sectioned use ultramicrotome (Reichert Ultracut S EM <br /> by ImageQuant (GE Healthcare; http://www3.gehealthcare.com/).                           UC6; Leica; http://www.leica-microsystems.com/). For light microscopy, <br />  <br /> Plant Physiol. Vol. 166, 2014                                                                                                                                       1447 <br />  Fujii et al. <br />  <br />  <br /> sections (0.8 mm) stained 1% (w/v) toluidine blue sodium borate         thylakoid membrane energization photoprotection leaves. Plant <br /> and analyzed light microscope (BX60; Olympus; http://www.olympus-               Physiol 148: 580&#226;&#8364;&#8220;592 <br /> global.com/en/). For electron microscopy observation, ultrathin sections (70&#226;&#8364;&#8220;90     Awai K, Mar&#195;&#169;chal E, Block MA, Brun D, Masuda T, Shimada H, <br /> nm) stained 5% (w/v) uranyl acetate 50% (v/v) methanol 0.4%           Takamiya K, Ohta H, Joyard J (2001) Two types MGDG synthase <br /> (w/v) lead citrate 0.1 N NaOH. Sections observed transmission               genes, widely 16:3 18:3 plants, differentially mediate <br /> electron microscopy (CM 100; Philips; http://www.fei.com/) 80 kV,               galactolipid syntheses photosynthetic nonphotosynthetic tissues <br /> images obtained use Gatan Orius CCD camera (http://www.gatan.             Arabidopsis thaliana. Proc Natl Acad Sci USA 98: 10960&#226;&#8364;&#8220;10965 <br /> com/).                                                                              Block MA, Dorne AJ, Joyard J, Douce R (1983) Preparation charac- <br />                                                                                        terization membrane fractions enriched outer inner envelope <br />                                                                                        membranes spinach chloroplasts. II. Biochemical characterization. <br /> Immunoblot Analysis <br />                                                                                        J Biol Chem 258: 13281&#226;&#8364;&#8220;13286 <br />     The membrane protein fraction prepared described previously              Charuvi D, Kiss V, Nevo R, Shimoni E, Adam Z, Reich Z (2012) Gain and <br /> (Kobayashi et al., 2013) 5-d-old wild-type L2 pale-green seedlings            loss photosynthetic membranes plastid differentiation the <br /> treated 10 mM DEX. Twenty micrograms total membrane protein            shoot apex Arabidopsis. Plant Cell 24: 1143&#226;&#8364;&#8220;1157 <br /> L2 sample electrophoresed dilution series (0.2&#226;&#8364;&#8220;5 mg) total   Chi W, Mao J, Li Q, Ji D, Zou M, Lu C, Zhang L (2010) Interaction the <br /> membrane protein wild type electrotransferred nitrocellulose         pentatricopeptide-repeat protein DELAYED GREENING 1 sigma <br /> membranes (Amersham Protran Premium 0.2 NC; GE Healthcare) described                factor SIG6 regulation chloroplast gene expression Arabi- <br /> (Kobayashi et al., 2007). Protein bands reacted primary antibodies      dopsis cotyledons. Plant J 64: 14&#226;&#8364;&#8220;25 <br /> secondarily labeled goat anti-rabbit IgG secondary antibody conjugated    Cornah JE, Germain V, Ward JL, Beale MH, Smith SM (2004) Lipid utilization, <br /> horseradish peroxidase (Thermo Scienti&#239;&#172;?c; http://www.thermoscienti&#239;&#172;?c.com/)             gluconeogenesis, seedling growth Arabidopsis mutants lacking the <br /> and detected using chemiluminescence reagent (Pierce Western Blotting Sub-           glyoxylate cycle enzyme malate synthase. J Biol Chem 279: 42916&#226;&#8364;&#8220;42923 <br /> strate Plus; Thermo Scienti&#239;&#172;?c) imager (ImageQuant LAS 4000 mini; GE          Craft J, Samalova M, Baroux C, Townley H, Martinez A, Jepson I, Tsiantis M, <br /> Healthcare). Antibodies PsaA/PsaB kindly provided Ryouichi             Moore I (2005) New pOp/LhG4 vectors stringent glucocorticoid-dependent <br /> Tanaka (Hokkaido University) D1 D2 kindly pro-              transgene expression Arabidopsis. Plant J 41: 899&#226;&#8364;&#8220;918 <br /> vided Masahiko Ikeuchi (University Tokyo). Antibodies LHCA1,          D&#195;&#182;rmann P, Benning C (2002) Galactolipids rule seed plants. Trends <br /> LHCA3, LHCB1, LHCB3 AgriSera (http://www.agrisera.com/).                 Plant Sci 7: 112&#226;&#8364;&#8220;118 <br />                                                                                     Dubots E, Audry M, Yamaryo Y, Bastien O, Ohta H, Breton C, Mar&#195;&#169;chal E, <br />                                                                                        Block MA (2010) Activation chloroplast monogalactosyldiacylglycerol <br /> Supplemental Data                                                                      synthase MGD1 phosphatidic acid phosphatidylglycerol. J Biol Chem <br />    The following materials available online version article.        285: 6003&#226;&#8364;&#8220;6011 <br />                                                                                     Eastmond PJ, Germain V, Lange PR, Bryce JH, Smith SM, Graham IA <br />    Supplemental Figure S1. Fatty acid composition TAG.                              (2000) Postgerminative growth lipid catabolism oilseeds lacking <br />    Supplemental Figure S2. Organelle ultrastructure 5-d-old amiR-MGD1               glyoxylate cycle. Proc Natl Acad Sci USA 97: 5669&#226;&#8364;&#8220;5674 <br />      L4 cotyledons.                                                                 Engel N, van den Daele K, Kolukisaoglu U, Morgenthal K, Weckwerth <br />                                                                                        W, P&#195;&#164;rnik T, Keerberg O, Bauwe H (2007) Deletion glycine decar- <br />    Supplemental Figure S3. Effect 5-d DEX treatment 72 h                 boxylase Arabidopsis lethal nonphotorespiratory conditions. <br />      seeding amiR-MGD1 L4 line.                                                 Plant Physiol 144: 1328&#226;&#8364;&#8220;1335 <br />    Supplemental Figure S4. Expression MGD2 MGD3 true leaves            Garcion C, Guilleminot J, Kroj T, Parcy F, Giraudat J, Devic M (2006) <br />      10-d-old amiR-MGD1 L4 seedlings grown DEX-free DEX-containing               AKRP EMB506 ankyrin repeat proteins essential plastid <br />      medium.                                                                           differentiation plant development Arabidopsis. Plant J 48: 895&#226;&#8364;&#8220;906 <br />                                                                                     Govindjee (1995) Sixty-three years Kautsky: chlorophyll &#239;&#172;&#8218;uores- <br />    Supplemental Table S1. Oligonucleoide primers used qRT-PCR                      cence. Aust J Plant Physiol 22: 131&#226;&#8364;&#8220;160 <br />      analysis.                                                                      Guskov A, Kern J, Gabdulkhakov A, Broser M, Zouni A, Saenger W <br />                                                                                        (2009) Cyanobacterial photosystem II 2.9-A resolution role of <br />                                                                                        quinones, lipids, channels chloride. Nat Struct Mol Biol 16: 334&#226;&#8364;&#8220;342 <br />                                                                                     Hayashi M, Nishimura M (2006) Arabidopsis thaliana: model organism to <br /> ACKNOWLEDGMENTS <br />                                                                                        study plant peroxisomes. Biochim Biophys Acta 1763: 1382&#226;&#8364;&#8220;1391 <br />    We thank Dr. Wann-Neng Jane (Plant Cell Biology Core Laboratory,                 Hayashi Y, Hayashi M, Hayashi H, Hara-Nishimura I, Nishimura M <br /> Institute Plant Microbial Biology, Academia Sinica) technical support       (2001) Direct interaction glyoxysomes lipid bodies cot- <br /> in electron microscopy Dr. Krishna K. Niyogi (Department Plant              yledons Arabidopsis thaliana ped1 mutant. Protoplasma 218: 83&#226;&#8364;&#8220;94 <br /> Microbial Biology, University California, Berkeley) providing access      H&#195;&#182;lzl G, Witt S, Gaude N, Melzer M, Sch&#195;&#182;ttler MA, D&#195;&#182;rmann P (2009) <br /> equipment photosynthetic measurements valuable comments.                       The role diglycosyl lipids photosynthesis membrane lipid <br /> Received September 7, 2014; accepted September 23, 2014; published Septem-             homeostasis Arabidopsis. Plant Physiol 150: 1147&#226;&#8364;&#8220;1159 <br /> ber 24, 2014.                                                                       Ishizaki Y, Tsunoyama Y, Hatano K, Ando K, Kato K, Shinmyo A, Kobori M, <br />                                                                                        Takeba G, Nakahira Y, Shiina T (2005) A nuclear-encoded sigma <br />                                                                                        factor, Arabidopsis SIG6, recognizes sigma-70 type chloroplast pro- <br />                                                                                        moters regulates early chloroplast development cotyledons. <br /> LITERATURE CITED <br />                                                                                        Plant J 42: 133&#226;&#8364;&#8220;144 <br /> Albrecht V, Ingenfeld A, Apel K (2006) Characterization snowy                Jarvis P, D&#195;&#182;rmann P, Peto CA, Lutes J, Benning C, Chory J (2000) Galactolipid <br />   cotyledon 1 mutant Arabidopsis thaliana: impact chloroplast                de&#239;&#172;?ciency abnormal chloroplast development Arabidopsis MGD <br />   elongation factor G chloroplast development plant vitality. Plant             synthase 1 mutant. Proc Natl Acad Sci USA 97: 8175&#226;&#8364;&#8220;8179 <br />   Mol Biol 60: 507&#226;&#8364;&#8220;518                                                              Kamada T, Nito K, Hayashi H, Mano S, Hayashi M, Nishimura M (2003) <br /> Albrecht V, Ingenfeld A, Apel K (2008) Snowy cotyledon 2: identi&#239;&#172;?-                 Functional differentiation peroxisomes revealed expression pro&#239;&#172;?les of <br />   cation zinc &#239;&#172;?nger domain protein essential chloroplast devel-               peroxisomal genes Arabidopsis thaliana. Plant Cell Physiol 44: 1275&#226;&#8364;&#8220;1289 <br />   opment cotyledons true leaves. Plant Mol Biol 66: 599&#226;&#8364;&#8220;608           Kobayashi K, Awai K, Nakamura M, Nagatani A, Masuda T, Ohta H <br /> Andriankaja M, Dhondt S, De Bodt S, Vanhaeren H, Coppens F,                            (2009a) Type-B monogalactosyldiacylglycerol synthases involved in <br />   De Milde L, M&#195;&#188;hlenbock P, Skirycz A, Gonzalez N, Beemster GTS,                       phosphate starvation-induced lipid remodeling, crucial low- <br />   et al (2012) Exit proliferation leaf development Arab-                phosphate adaptation. Plant J 57: 322&#226;&#8364;&#8220;331 <br />   idopsis thaliana: not-so-gradual process. Dev Cell 22: 64&#226;&#8364;&#8220;78                    Kobayashi K, Awai K, Takamiya K, Ohta H (2004) Arabidopsis type B <br /> Aronsson H, Sch&#195;&#182;ttler MA, Kelly AA, Sundqvist C, D&#195;&#182;rmann P, Karim S,                   monogalactosyldiacylglycerol synthase genes expressed during <br />   Jarvis P (2008) Monogalactosyldiacylglycerol de&#239;&#172;?ciency Arabidopsis                pollen tube growth induced phosphate starvation. Plant Physiol <br />   affects pigment composition prolamellar body impairs                      134: 640&#226;&#8364;&#8220;648 <br />  <br /> 1448                                                                                                                              Plant Physiol. Vol. 166, 2014 <br />                                                                                                    Role Galactolipids Organelle Differentiation <br />  <br />  <br /> Kobayashi K, Kondo M, Fukuda H, Nishimura M, Ohta H (2007) Gal-                Shimada H, Mochizuki M, Ogura K, Froehlich JE, Osteryoung KW, <br />    actolipid synthesis chloroplast inner envelope essential proper      Shirano Y, Shibata D, Masuda S, Mori K, Takamiya K (2007) Arabi- <br />    thylakoid biogenesis, photosynthesis, embryogenesis. Proc Natl             dopsis cotyledon-speci&#239;&#172;?c chloroplast biogenesis factor CYO1 protein <br />    Acad Sci USA 104: 17216&#226;&#8364;&#8220;17221                                                  disul&#239;&#172;?de isomerase. Plant Cell 19: 3157&#226;&#8364;&#8220;3169 <br /> Kobayashi K, Masuda T, Takamiya K, Ohta H (2006) Membrane lipid                Shirano Y, Shimada H, Kanamaru K, Fujiwara M, Tanaka K, Takahashi H, <br />    alteration phosphate starvation regulated phosphate sig-          Unno K, Sato S, Tabata S, Hayashi H, et al (2000) Chloroplast develop- <br />    naling auxin/cytokinin cross-talk. Plant J 47: 238&#226;&#8364;&#8220;248                     ment Arabidopsis thaliana requires nuclear-encoded transcription <br /> Kobayashi K, Nakamura Y, Ohta H (2009b) Type A type B mo-                     factor sigma B. FEBS Lett 485: 178&#226;&#8364;&#8220;182 <br />    nogalactosyldiacylglycerol synthases spatially functionally         Stoynova-Bakalova E, Karanov E, Petrov P, Hall MA (2004) Cell division cell <br />    separated plastids higher plants. Plant Physiol Biochem 47:          expansion cotyledons Arabidopsis seedlings. New Phytol 162: 471&#226;&#8364;&#8220;479 <br />    518&#226;&#8364;&#8220;525                                                                     Strand A, Asami T, Alonso J, Ecker JR, Chory J (2003) Chloroplast to <br /> Kobayashi K, Narise T, Sonoike K, Hashimoto H, Sato N, Kondo M,                   nucleus communication triggered accumulation Mg-protoporphyrinIX. <br />    Nishimura M, Sato M, Toyooka K, Sugimoto K, et al (2013) Role               Nature 421: 79&#226;&#8364;&#8220;83 <br />    galactolipid biosynthesis coordinated development photosynthetic      Sulmon C, Gouesbet G, Cou&#195;&#169;e I, Cabello-Hurtado F, Cavalier A, Penno C, <br />    complexes thylakoid membranes chloroplast biogenesis             Zaka R, Bechtold N, Thomas D, El Amrani A (2006) The pleiotropic <br />    Arabidopsis. Plant J 73: 250&#226;&#8364;&#8220;261                                               Arabidopsis frd mutation altered coordination chloroplast biogenesis, <br /> Kramer DM, Johnson G, Kiirats O, Edwards GE (2004) New &#239;&#172;&#8218;uorescence                cell size differentiation, organ size number. Gene 382: 88&#226;&#8364;&#8220;99 <br />    parameters determination qa redox state excitation energy    Tanz SK, Kilian J, Johnsson C, Apel K, Small I, Harter K, Wanke D, <br />    &#239;&#172;&#8218;uxes. Photosynth Res 79: 209&#226;&#8364;&#8220;218                                              Pogson B, Albrecht V (2012) The SCO2 protein disulphide isomerase is <br /> Li Y, Beisson F, Pollard M, Ohlrogge J (2006) Oil content Arabidopsis          required thylakoid biogenesis interacts LHCB1 chlorophyll <br />    seeds: in&#239;&#172;&#8218;uence seed anatomy, light plant-to-plant variation.       a/b binding proteins affects chlorophyll biosynthesis Arabi- <br />    Phytochemistry 67: 904&#226;&#8364;&#8220;915                                                     dopsis seedlings. Plant J 69: 743&#226;&#8364;&#8220;754 <br /> Mans&#239;&#172;?eld SG, Briarty LG (1996) The dynamics seedling cotyledon          Timm S, Nunes-Nesi A, P&#195;&#164;rnik T, Morgenthal K, Wienkoop S, Keerberg O, <br />    cell development Arabidopsis thaliana reserve mobilization. Int      Weckwerth W, Kleczkowski LA, Fernie AR, Bauwe H (2008) A cytosolic <br />    J Plant Sci 157: 280&#226;&#8364;&#8220;295                                                       pathway conversion hydroxypyruvate glycerate pho- <br /> Maxwell K, Johnson GN (2000) Chlorophyll &#239;&#172;&#8218;uorescence: practical                 torespiration Arabidopsis. Plant Cell 20: 2848&#226;&#8364;&#8220;2859 <br />    guide. J Exp Bot 51: 659&#226;&#8364;&#8220;668                                                Tsukaya H, Tsuge T, Uchimiya H (1994) The cotyledon: superior system <br /> Melis A, Spangfort M, Andersson B (1987) Light-absorption electron-           studies leaf development. Planta 195: 309&#226;&#8364;&#8220;312 <br />    transport balance photosystem II photosystem I spinach       Umena Y, Kawakami K, Shen JR, Kamiya N (2011) Crystal structure oxygen- <br />    chloroplasts. Photochem Photobiol 45: 129&#226;&#8364;&#8220;136                                  evolving photosystem II resolution 1.9 &#195;&#8230;. Nature 473: 55&#226;&#8364;&#8220;60 <br /> Mizusawa N, Wada H (2012) The role lipids photosystem II. Biochim        van Kooten O, Snel JFH (1990) The use chlorophyll &#239;&#172;&#8218;uorescence no- <br />    Biophys Acta 1817: 194&#226;&#8364;&#8220;208                                                     menclature plant stress physiology. Photosynth Res 25: 147&#226;&#8364;&#8220;150 <br /> Nott A, Jung HS, Koussevitzky S, Chory J (2006) Plastid-to-nucleus ret-        Voll LM, Jamai A, Renn&#195;&#169; P, Voll H, McClung CR, Weber AP (2006) The <br />    rograde signaling. Annu Rev Plant Biol 57: 739&#226;&#8364;&#8220;759                             photorespiratory Arabidopsis shm1 mutant de&#239;&#172;?cient SHM1. Plant <br /> Nunes-Nesi A, Ara&#195;&#186;jo WL, Fernie AR (2011) Targeting mitochondrial                 Physiol 140: 59&#226;&#8364;&#8220;66 <br />    metabolism machinery means enhance photosynthesis. Plant        Wang Y, Duby G, Purnelle B, Boutry M (2000) Tobacco VDL gene encodes <br />    Physiol 155: 101&#226;&#8364;&#8220;107                                                           plastid DEAD box RNA helicase involved chloroplast dif- <br /> Oxborough K, Baker NR (1997) Resolving chlorophyll &#239;&#172;&#8218;uorescence images        ferentiation plant morphogenesis. Plant Cell 12: 2129&#226;&#8364;&#8220;2142 <br />    photosynthetic ef&#239;&#172;?ciency photochemical non-photochemical com-      Woodson JD, Perez-Ruiz JM, Schmitz RJ, Ecker JR, Chory J (2013)  <br /> </body></html>