<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                          RESEARCH ARTICLE <br />  <br />  <br />                                                           Protein Complexes Bacteria <br />                                                           J. Harry Caufield, Marco Abreu, Christopher Wimble, Peter Uetz* <br />                                                           Center Study Biological Complexity, Virginia Commonwealth University, Richmond, Virginia, United <br />                                                           States America <br />  <br />                                                           * uetz@vcu.edu <br />  <br />  <br />  <br />  <br />                                                           <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                                                           Large-scale analyses protein complexes recently available Escherichia <br />                                                           coli Mycoplasma pneumoniae, yielding 443 116 heteromultimeric soluble protein <br />                                                           complexes, respectively. We coupled results mass spectrometry- <br />                                                           characterized protein complexes 285 &#226;&#8364;&#339;gold standard&#226;&#8364;? protein complexes identified <br />                                                           EcoCyc. A comparison databases gene orthology, conservation, essentiality <br />                                                           identified proteins conserved lost complexes species. For instance, 285 <br />                                                           &#226;&#8364;&#339;gold standard&#226;&#8364;? protein complexes E. coli, 10% fully conserved set <br />                                                           7 distantly-related bacterial &#226;&#8364;&#339;model&#226;&#8364;? species. Complex conservation follows three <br />                                                           models: well-conserved complexes, complexes conserved core, complexes with <br />     OPEN ACCESS                                           partial conservation conserved core. Expanding comparison 894 distinct bac- <br /> Citation: Caufield JH, Abreu M, Wimble C, Uetz P <br />                                                           terial genomes illustrates fractional conservation limits co-conservation among <br /> (2015) Protein Complexes Bacteria. PLoS Comput         components protein complexes: just 14 285 model protein complexes perfectly <br /> Biol 11(2): e1004107. doi:10.1371/journal.                conserved 95% genomes used, predict 180 partially <br /> pcbi.1004107 <br />                                                           conserved half genomes. No clear relationship gene essenti- <br /> Editor: Christine A. Orengo, University College           ality protein complex conservation observed, poorly conserved complexes <br /> London, United Kingdom <br />                                                           contain significant number essential proteins. Finally, identify 183 complexes con- <br /> Received: July 31, 2014                                   taining well-conserved components uncharacterized proteins interesting <br /> Accepted: January 2, 2015                                 targets future experimental studies. <br /> Published: February 27, 2015 <br />  <br /> Copyright: &#194;&#169; 2015 Caufield et al. This open <br /> access article distributed terms the <br /> Creative Commons Attribution License, permits         Author Summary <br /> unrestricted use, distribution, reproduction any <br />                                                             Though 20,000 binary protein-protein interactions published a <br /> medium, provided original author source are <br /> credited. <br />                                                             well-studied bacterial species, results rarely capture extent pro- <br />                                                             teins complexes. Here, use experimentally-observed protein complexes <br /> Data Availability Statement: All relevant data are <br />                                                             E. coli Mycoplasma pneumoniae, gene orthology, predict protein com- <br /> within paper Supporting Information files. <br />                                                             plexes species bacteria. Surprisingly, majority protein complexes is <br /> Funding: This project supported US               conserved, demonstrating unexpected evolutionary flexibility. We observe <br /> National Institutes Health, grant R01GM109895. <br />                                                             broader trends protein complex conservation, especially genome-reduced species <br /> The funders role study design, data <br /> collection analysis, decision publish, or <br />                                                             minimal sets protein complexes. <br /> preparation manuscript. <br />  <br /> Competing Interests: The authors declared <br /> that competing interests exist. <br />  <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                                                 1 / 23 <br />                                                                                                                  Protein Complexes Bacteria <br />  <br />  <br />  <br />                                            Introduction <br />                                            Abundant genome sequencing revealed astounding diversity bacterial genomes. <br />                                            Even species inhabit environment share fraction genes. This <br />                                            raises question organisms adapted environments using limit- <br />                                            ed number genes. Here, investigate protein complements bacterial genomes, <br />                                            proteins combined protein complexes species, com- <br />                                            plexes conserved diverse branches prokaryotic tree life. <br />                                                Other studies compared interaction networks S. cerevisiae, S. pombe E. coli <br />                                            possible systematic screens genetic interactions notable differences <br />                                            structure content [1,2]. Studies comparing baker&#226;&#8364;&#8482;s yeast fission yeast found <br />                                            essentiality varies species [3]. This explained functional redun- <br />                                            dancy importance mechanism structure. The extent differences be <br />                                            unexpected make sense seen light evolutionary flexibility [1]. <br />                                                Numerous studies protein-protein interactions revealed organization prote- <br />                                            omes networks interactions protein complexes. Systematic surveys protein <br />                                            complexes exist bacterial species, E. coli [4,5] Mycoplasma pneumo- <br />                                            niae [6]. The list binary protein-protein interactomes clearly larger con- <br />                                            sidered study. Based limited dataset, investigated complexes <br />                                            model organisms sufficient reconstruct homologous protein complexes <br />                                            species. This particular challenge prokaryotes genomes species <br />                                            highly divergent model species used here. However, E. coli Mycoplasma <br />                                            provide important paradigms: E. coli generalist live variety condi- <br />                                            tions Mycoplasma specialized parasite requires host cells grow. With ~4,300 <br />                                            ~700 genes, respectively, represent medium-sized minimal genomes and <br />                                            medium minimal diversity protein complexes. <br />                                                Few studies investigated evolution diversity protein complexes wide <br />                                            range taxa. This surprising given large-scale experimental data become <br />                                            available recent years. In combination large number completed genome sequences <br />                                            use data evaluate extent protein complexes likely con- <br />                                            served microbial species. Furthermore, evaluate biological role proteins <br />                                            complexes unknown function species. <br />                                                Existing studies comparing sets interactome data, including pure bioinformatics ap- <br />                                            proaches [7] generally limited comparisons well-characterized protein- <br />                                            protein interaction networks, comparisons S. cerevisiae, S. pombe E. coli [2,7,8]. <br />                                            Methodological frameworks predicting co-evolution basis gene presence/absence <br />                                            [1,9] employed predict novel interactions species. In study, we <br />                                            focus distinct bacterial species, seven subject essentiality <br />                                            screens comprehensive protein complex surveys available. We ex- <br />                                            pand focus set 894 bacterial genomes. <br />                                                In order compare genomes protein complexes species, couple results of <br />                                            mass spectrometry-characterized protein complexes [5,6] databases gene orthology <br />                                            [10] essentiality [11] characterize interaction conservation protein complexes. <br />                                            Furthermore, use perspective genome reduction evaluate patterns levels of <br />                                            protein conservation. Comparing sets protein complexes divergent bacterial species <br />                                            (in case, E. coli M. pneumoniae) alleviates bias inherent using single <br />                                            species universal model. Rather, observing protein complexes components <br />                                            present distinct species allows draw conclusions critical <br />                                            components microbial life. <br />  <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                        2 / 23 <br />                                                                                                                   Protein Complexes Bacteria <br />  <br />  <br />  <br />                                            Results <br />                                            Conservation proteins bacterial genomes <br />                                            In general, small microbial genomes enriched proteins conserved bac- <br />                                            teria (Fig. 1; Table 1). This trend noticeable paralogy eliminated, re- <br />                                            moving paralogous group (PG) natural effects genome reduction, <br />                                            seen smallest bacterial genomes. In cases, average protein conserva- <br />                                            tion decreases genome size increases. Genes larger genomes, Pseudomonas <br />                                            aeruginosa, conserved 20 30 percent bacteria, average. The <br />                                            minimal genomes, including Mycoplasma species, share orthologous <br />                                            groups (OGs) 60 80 percent bacterial species, average. These results rea- <br />                                            sonable expected: reduced genomes, <span id='am-4' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span>, lost sequence space not <br />                                            lost loci crucial bacterial life itself. Furthermore, genomes in- <br />                                            creasing fraction paralogs conserved size increases (Fig. 1), most <br />                                            reduced genomes actually greater average conservation potential paralogs re- <br />                                            moved. The paralogous protein-coding loci reduced genomes enriched accessory <br />                                            genes broadly-conserved core genes. <br />                                                The presence multiple members single orthologous group effect aver- <br />                                            age gene conservation. Here, display effect difference average locus con- <br />                                            servation average OG conservation. (Fig. 1, orange vs. blue dots). The difference between <br />                                            values approximation level paralogy genome; larger genomes ap- <br />                                            pear contain paralogs smaller genomes, especially genome size falls 1 <br />                                            Mb. The effect average gene conservation expected, using orthology-based compari- <br />                                            sons compresses paralogs single OGs. Within data set, 20 genomes (within 10 unique <br />                                            genera) 3 Mb greater average conservation OGs individually- <br />                                            considered loci. The smallest genome set, cicada endosymbiont Hodgkinia <br />                                            cicadicola [12] demonstrates difference average conservation OGs in- <br />                                            dividual loci. All genomes greater 3 Mb higher average conservation individu- <br />                                            ally-considered loci OGs. <br />  <br />  <br />                                            The protein complexomes E. coli Mycoplasma pneumoniae <br />                                            In study, used literature-curated set EcoCyc E. coli protein complexes the <br />                                            protein complexes isolated Hu et al. [5] set experimentally-determined complexes for <br />                                            E. coli (Fig. 2A). The set experimentally-determined Mycoplasma pneumoniae complexes <br />                                            identified K&#195;&#188;hner et al. [6] included comparison distantly-related, mini- <br />                                            mal set. Though datasets differ content approach, E. coli data sets contain <br />                                            300 complexes. Most complexes EcoCyc set contain 2 4 unique proteins <br />                                            Hu set contains comparatively higher number complexes (more 30) con- <br />                                            taining 5 unique protein components (i.e, unique proteins mapping different ortho- <br />                                            logous groups). Note Hu et al. complexes appear represent subsets full <br />                                            complexes (i.e., ribosome constitutes single complex EcoCyc represented by <br />                                            complexes Hu et al.). Also, EcoCyc set partially redundant (i.e., RNA poly- <br />                                            merase holoenzyme represented different protein complex, F1 F0 subre- <br />                                            gions ATP synthase). <br />                                                The size complexes data set produced K&#195;&#188;hner et al. appears differ in <br />                                            distribution characterized Hu et al. (Fig. 2A). Specifically, M. pneumoniae <br />                                            complexes unique members contain just unique proteins. The <br />                                            cross-species discrepancy result methodology, K&#195;&#188;hner et al. suggest is <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                         3 / 23 <br />                                                                                                                                 Protein Complexes Bacteria <br />  <br />  <br />  <br />  <br /> Fig 1. Protein complexes enriched highly conserved components. Each point indicates single genome average conservation loci <br /> or orthologous groups (OGs) measured presence 898 bacterial genomes. Representative genomes 8 species focused study <br /> are indicated vertical lines following labels: M. g., Mycoplasma genitalium; M. p., Mycoplasma pneumoniae; H. p., Helicobacter pylori; S. s., <br /> Streptococcus sanguinis; C. c., Caulobacter crescentus; B. s., Bacillus subtilis; E. c., E. coli; P. a., Pseudomonas aeruginosa. See Materials Methods for <br /> specific genome identities. Average gene conservation specified percentage. Average gene conservation values reduced fraction their <br /> predicted protein-coding genes present eggNOG v.3 account genes predicted orthology. <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> produce OG conservation instead locus <br /> conservation, locus set potential paralogs (in case, genes sharing OG) removed prior calculating averages. A <br /> logarithmic regression fitted sets values. Average OG conservation values shown subsets protein-coding genes present within <br /> protein complexes E. coli [5] M. pneumoniae [6]. For species, values representative members complexomes for <br /> other species predicted complexomes using data sets models. <br /> doi:10.1371/journal.pcbi.1004107.g001 <br />  <br />  <br />  <br />                                                 representative authentic biological differences species. M. pneumoniae con- <br />                                                 tains fewer unique proteins E. coli does difference limits number unique <br />                                                 proteins seen single complex. <br />                                                    The exact protein complexes defined data set differ. Pairwise comparison pres- <br />                                                 ence absence proteins complex improved mapping components ortholo- <br />                                                 gous groups complexes appear present identical form data <br />                                                 sets. Fig. 2B provides examples types complex matches seen data sets. <br />                                                 For instance, DNA polymerase III holoenzyme (EcoCyc: CPLX0&#226;&#8364;&#8220;3803) contains 9 unique <br />                                                 proteins EcoCyc closest match Hu set contains 7, including proteins <br />                                                 EcoCyc complex. The &#226;&#8364;&#339;missing&#226;&#8364;? proteins EcoCyc complex found <br />                                                 Hu complexes. The Hsp70 chaperone complex (EcoCyc: HSP70-CPLX) provides an- <br />                                                 example: The M. pneumoniae complexes provide better match EcoCyc complex <br />                                                 Hu set does. Topoisomerase IV (EcoCyc: CPLX0&#226;&#8364;&#8220;2424) good match three <br />                                                 data sets representative Hu complex contains additional protein. Lastly, RecBCD <br />                                                 serves example good E. coli-specific match components present the <br />                                                 M. pneumoniae complexes. <br />                                                    In aggregate, EcoCyc complexes reliable matches experi- <br />                                                 mental sets (Fig. 2C). Using 285 EcoCyc complexes guide, best matches the <br />                                                 sets classified &#226;&#8364;&#339;good&#226;&#8364;? contain half unique proteins <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                                          4 / 23 <br />                                                                                                                             Protein Complexes Bacteria <br />  <br />  <br />  <br />  <br /> Table 1. Protein conservation 8 species. <br />  <br />                      M. pneumoniae      M. genitalium   B. subtilis     S. sanguinis    H. pylori        C. crescentus     P. aeruginosa    E. coli <br /> M. pneumoniae        601                517 (86.02%)    430 (71.55%)    427 (71.05%)    336 (55.91%)     379 (63.06%)      395 (65.72%)     408 (67.89%) <br /> M. genitalium        466 (96.68%)       482             388 (80.50%)    381 (79.05%)    290 (60.17%)     333 (69.09%)      353 (73.23%)     355 (73.65%) <br /> B. subtilis          644 (15.88%)       627 (15.46%)    4056            1974 (48.67%)   1374 (33.88%)    2101 (51.80%)     2465 (60.77%)    2428 (59.86%) <br /> S. sanguinis         538 (26.37%)       519 (25.44%)    1514 (74.22%)   2040            804 (39.41%)     1186 (58.14%)     1341 (65.74%)    1379 (67.60%) <br /> H. pylori            314 (21.43%)       304 (20.75%)    844 (57.61%)    631 (41.21%)    1465             916 (62.53%)      979 (66.83%)     992 (67.71%) <br /> C. crescentus        454 (12.58%)       440 (12.20%)    2064 (57.21%)   1487 (41.21%)   1401 (38.83%)    3608              2618 (72.56%)    2414 (66.91%) <br /> P. aeruginosa        678 (11.74%)       655 (11.34%)    3275 (56.72%)   2353 (40.75%)   1963 (34.00%)    3615 (62.61%)     5774             4007 (69.40%) <br /> E. coli              594 (14.33%)       567 (13.68%)    2418 (58.34%)   1791 (43.21%)   1414 (34.11%)    2409 (58.12%)     3027 (73.03%)    4145 <br />  <br /> On leftmost column organism basis comparison row organism compared to. An organism <br /> compared itself shows total number proteins organism dataset used. For example, M. pneumoniae shares 86.02% proteins with <br /> M. genitalium M. genitalium shares 96.68% proteins M. pneumoniae. Data species collected Uniprot and <br /> proteins mapped using common COGs, NOGs, bactNOGs [10] <br />  <br /> doi:10.1371/journal.pcbi.1004107.t001 <br />  <br />  <br />  <br />  <br />                                                 (as members orthologous groups) &#226;&#8364;&#339;poor&#226;&#8364;? contain match half the <br />                                                 EcoCyc complex&#226;&#8364;&#8482;s components. No complex size greater 4 unique proteins a <br />                                                 good match Hu et al. K&#195;&#188;hner et al. complex sets. 28 complexes (9.8%) the <br />                                                 complexes size 4 good matches sets. The majority complexes in <br />                                                 size class (153 246) contained matching component Hu E. coli <br />                                                 complexes match K&#195;&#188;hner et al. M. pneumoniae complexes. <br />                                                    The set M. pneumoniae complexes serves rough model complexes com- <br />                                                 monly bacterial species (see S9 Table S10 Table) predicted conserva- <br />                                                 tion complex). It imperfect model: 116 complexes, 28 fully <br />                                                 conserved (that is, components present orthologs) 7 model spe- <br />                                                 cies study. 39 M. pneumoniae complexes appear share 2/3 components <br />                                                 species, 75 complexes share half. Just complex contains <br />                                                 components entirely specific M. pneumoniae (complex 87, composed uncharacterized <br />                                                 proteins Mpn036 Mpn676, respective UniProt entries P75078 P75116). <br />  <br />  <br />  <br />                                                 Using protein complexomes predict complexes conserved other <br />                                                 species <br />                                                 The variability EcoCyc Hu datasets direct impact usefulness of <br />                                                 complexomes models bacterial species. In case, EcoCyc Hu com- <br />                                                 plex sets provide comprehensive complex set currently available E. coli. The inter- <br />                                                 section sets (Fig. 3A) limited: just 576 unique orthologous groups are <br />                                                 shared sets just 132 complexes appear &#226;&#8364;&#339;good&#226;&#8364;? matches sets. <br />                                                 Using 132 complexes model P. aeruginosa shows 120 the <br />                                                 complexes conserved based orthologous components present P. aeruginosa <br />                                                 genome. If yet-uncharacterized P. aeruginosa complexome contains roughly same <br />                                                 number complexes E. coli prediction method misses half <br />                                                 (that is, 150) potential complexes unless use unique complexes of <br />                                                 set. We used results evidence data sets used independent <br />                                                 models intersecting set: losing half potential model <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                                        5 / 23 <br />                                                                                                                            Protein Complexes Bacteria <br />  <br />  <br />  <br />  <br /> Fig 2. Protein complex data sets vary composition. (A) Count complexes E. coli complex datasets ([5]; EcoCyc [13]) M. pneumoniae <br /> dataset [6], size (in number unique protein components). Multimers single proteins (i.e., homodimers) included. (B) Examples complex <br /> matching data sets. Once mapped orthologous group (OG), components complex directly comparable complex sets <br /> yet perfect matches rare. In instances, OG complex present best matching complex OG present <br /> elsewhere different complex. In cases, matched complex contain components (OGs) seen query complex (as case with <br /> topoisomerase IV). (C) Summary matching complex quality data sets. EcoCyc complexes used set query complexes two <br /> experimental data sets used search space. Here, poor match requires just matching component, good match requires half <br /> of components query complex present matching complex. The number complexes category shown; complex size in <br /> part A. <br /> doi:10.1371/journal.pcbi.1004107.g002 <br />  <br />                                               complexes simply inconsistencies data sets limiting broad cross- <br />                                               species comparision. <br />                                                   Fig. 3B displays distributions protein complex conservation bacterial species <br />                                               E. coli. (M. pneumoniae complexes used comparison.) These plots <br />                                               provide median interquartile range protein complex conservation fractions each <br />                                               species, using EcoCyc Hu et al. complexes model complex set. A compre- <br />                                               hensive set protein complexes identified species yet. Fol- <br />                                               lowing results shown Fig. 1, however, predict bacterial protein <br />                                               complex component sets share half OGs bacterial ge- <br />                                               nomes, average. Basic biology plays role here: expect subset crucial protein <br />                                               complexes like polymerases well-conserved species. The set EcoCyc com- <br />                                               plexes, appears highly-conserved P. aeruginosa (the entire interquartile range lies be- <br />                                               tween 75% complex conservation, showing average EcoCyc complex well- <br />                                               represented P. aeruginosa) shows greater range conservation other <br />                                               species. The Hu complexes lower complex conservation median values EcoCyc for <br />                                               H. pylori lower variability P. aeruginosa. Here, median values not <br />                                               useful conservation ranges: distance highest lowest values in- <br />                                               cludes possibility 0 100% conservation using model E. coli complexes. <br />                                               We species closely related E. coli set&#226;&#8364;&#8221;P. aeruginosa C. cres- <br />                                               centus&#226;&#8364;&#8221;produce different median values interquartile ranges sets all <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                                   6 / 23 <br />                                                                                                                                  Protein Complexes Bacteria <br />  <br />  <br />  <br />  <br /> Fig 3. Protein complex sets vary conservation bacteria. (A) Overlap literature-curated (EcoCyc) experimentally-observed (Hu <br /> et al.) E. coli complex sets limited. Each data set contains unique proteins, mapped orthologous groups (far left). Each complex one <br /> of E. coli complex sets appear shared complex set (middle; potentially shared complex half of <br /> its components complex sets). Using just set complexes shared E. coli sets model predicting <br /> complexomes species (far right; case, P. aeruginosa used example) limiting. 12 complexes shared set appear be <br /> conserved P. aeruginosa roughly additional 150 complexes expected based seen E. coli. (B) Each box plot displays range of <br /> conservation fractions E. coli protein complexes literature curated (EcoCyc) experimental (Hu et al.) sets respect species than <br /> E. coli. The upper lower edges box correspond quartile conservation fraction values, respectively. The upper whisker <br /> corresponds highest value 1.5 times inter-quartile range (IQR) lower whisker corresponds lowest value same <br /> range. Data points outside 1.5 times IQR represented single data points. <br /> doi:10.1371/journal.pcbi.1004107.g003 <br />  <br />  <br />                                                 protein complexes. Components complexes E. coli sets, used models, clearly <br />                                                 conserved differently bacterial species. A higher-resolution comparison necessary to <br />                                                 determine complexes highly-conserved. <br />  <br />  <br />                                                 Protein complexes essentiality poorly conserved bacteria <br />                                                 Although size distribution different E. coli Mycoplasma, hypothesized ho- <br />                                                 mologous complexes similar, size composition. However, not <br />                                                 true: complexes share half components data sets (Fig. 2C). The ma- <br />                                                 jority complexes shows 50% overlap EcoCyc Hu, Hu <br />                                                 Mycoplasma. This suggests technical (E. coli vs E. coli) biologi- <br />                                                 cal reasons (E. coli vs. Mycoplasma) differences. <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                                           7 / 23 <br />                                                                                                                            Protein Complexes Bacteria <br />  <br />  <br />  <br />  <br /> Fig 4. Examples protein complex conservation bacteria. Complexes identified using common EcoCyc ID. Each complex <br /> subunit assigned COG ID. Grey areas indicate OG presence, white areas indicate OG absence, blue areas indicate essentiality species- <br /> specific screen (see Materials Methods references). Values spaces indicate presence potential paralogs corresponding <br /> species; proteins specified values paralogs. Complex structures available PDB: ATP synthase F1, 3OAA; succinate dehydrogenase, <br /> 1NEN; Outer membrane protein assembly complex, (2KM7, 3TGO, 3TGO, 4K3C, 2YH3). Species arranged taxonomy (see Materials and <br /> Methods details) E. coli Mycoplasma serving distant species. <br /> doi:10.1371/journal.pcbi.1004107.g004 <br />  <br />  <br />                                                  To global detailed picture protein complex conservation, com- <br />                                               pared conservation 8 bacterial species, including species protein <br />                                               complex sets exist. The EcoCyc complex set used standard species <br />                                               compared. Fig. 4 provides examples ways protein complexes not <br />                                               conserved species. Conservation protein complexes roughly grouped into <br />                                               categories: well-conserved complexes, complexes core set proteins conserved, <br />                                               core set appears consistently conserved. As conservation es- <br />                                               sentiality related paralogy, compared components complexes on <br />                                               presence absence paralogs. <br />                                                  It commonly assumed highly conserved proteins important should <br />                                               essential cases. Interestingly, true (Fig. 4). For example, well- <br />                                               conserved succinate dehydrogenase components essential 3 species shown. <br />                                               The components complex (as defined default structure E. coli) present <br />                                               Pseudomonas aeruginosa Caulobacter crescentus. Helicobacter pylori B. subtilis <br />                                               encode 3 4 components 3 species appear lost entire complex. <br />                                               Similarly, Bam outer membrane protein assembly complex (EcoCyc: CPLX0&#226;&#8364;&#8220;3933) shows <br />                                               partial essentiality complex 4 species components conserved in <br />                                               3 species. This complex similarly patchy <span id='am-9' about='xsp:pattern' typeof='owl:Thing'>pattern</span> conservation, number <br />                                               zero 5 components conserved. In case H. pylori Bam complex, initially <br />                                               like lack conservation result component replacement functionally <br />                                               similar proteins [14]. By contrast, F1 ATP synthase conserved species examined. These <br />                                               examples complexes conserved important functions <br />                                               indicate (as measured presence essential proteins complexes). <br />                                                  Fig. 5A displays EcoCyc E. coli complexes component present M. <br />                                               pneumoniae. In case, fraction essentiality (the number protein components to <br />                                               essential protein components present) shown. Fig. 5B displays conservation <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                                    8 / 23 <br />                                                                                                                                  Protein Complexes Bacteria <br />  <br />  <br />  <br />  <br /> Fig 5. Fractional essentiality conservation protein complexes species. (A) Each column represents protein complex (as defined in <br /> EcoCyc E. coli) fraction essentiality species shown left. This subset complexes component is <br /> predicted conserved M. pneumoniae. Black stars complex IDs indicate complexes shown Fig. 3. Two example complexes predicted be <br /> present M. pneumoniae shown far right complex list. See S1 Table key complexes. For species E. coli, complexes <br /> are predicted using orthologous groups (OGs). Colors indicate fraction essentiality: blue&#226;&#8364;&#8221;conserved components essential fraction specified <br /> at right, grey&#226;&#8364;&#8221;no components conserved conserved components essential. (B) Conservation complexes shown (A). Colors indicate <br /> the fraction conservation ranging dark green (all proteins present) red (no protein present). General functional group assignments were <br /> manually assigned based EcoCyc annotations. Columns panels A B correspond complexes. (C) As A, set of <br /> EcoCyc E. coli complexes; column single complex. An extended version heat map provided S1 Fig. (D) As B, set of <br /> EcoCyc E. coli complexes; column single complex. The order complexes identical (C). An extended version heat map is <br /> provided S2 Fig. Columns panels C D correspond complexes. <br /> doi:10.1371/journal.pcbi.1004107.g005 <br />  <br />  <br />  <br />  <br />                                                 fractions E. coli complexes protein conserved M. pneumoniae, though <br />                                                 necessarily present M. pneumoniae complex. A complete survey EcoCyc com- <br />                                                 plexes species terms conservation essentiality provided S1 Fig. and <br />                                                 S2 Fig., respectively. Conservation fraction established fraction unique proteins in <br />                                                 defined complex present target species. Notably, proteins 21 complete EcoCyc <br />                                                 complexes fully conserved 8 species, just 15 complexes subunits al- <br />                                                 ternate forms (i.e., RNA polymerase different sigma factors) removed. An additional <br />                                                 19 complexes fully conserved species Mycoplasma species. The re- <br />                                                 maining complexes vary extensively degree extent conservation. A number of <br />                                                 complexes conserved E. coli, P. aeruginosa, C., crescentus, H. pylori, B. subti- <br />                                                 lis S. sanguinis Mycoplasma (e.g. succinate dehydrogenase, EcoCyc: SUCC- <br />                                                 DEHASE). Overall, 176 EcoCyc complexes 3 unique proteins (S1 Fig.), 128 <br />                                                 appear lost unique protein component species. This demon- <br />                                                 strates protein complexes far flexible evolutionary terms than <br />                                                 previously assumed. <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                                           9 / 23 <br />                                                                                                                   Protein Complexes Bacteria <br />  <br />  <br />  <br />                                                Protein complex function varies similar way conservation (Fig. 5B). As expected, <br />                                            highly conserved complexes directly involved DNA replication, tran- <br />                                            scription, translation. Many protein complexes varying conservation fractions trans- <br />                                            port complexes&#226;&#8364;&#8221;as bacterial membrane structures vary species, degree of <br />                                            transporter component evolution expected. At distinct complexes involved in <br />                                            DNA modification repair demonstrate perfect conservation. <br />                                                E. coli complexes serve &#226;&#8364;&#339;gold standard&#226;&#8364;? protein complexes bacteria in <br />                                            cases components complex broadly conserved. This property <br />                                            true just small fraction complexes. Fig. 5D displays conservation fractions 285 <br />                                            E. coli complexes EcoCyc set, clustered similarity conservation patterns across <br />                                            7 species used study. Just 21 complexes appear fully conserved (that is, <br />                                            orthologs of components present) species. This broad taxo- <br />                                            nomic range, relaxed cutoff appropriate predict complex conserved; <br />                                            so, 28 complexes contain 2/3 E. coli components species. <br />                                            Lowering cutoff half E. coli components yields 34 complexes. The <br />                                            lack broad conservation not, however, matter complex presence absence across <br />                                            species. Rather, complex components appear conserved independently other <br />                                            members complex. Similarly patchy conservation seen essentiality (Fig. 5C), <br />                                            broadly well-conserved complexes (far left) generally retain essentiality spe- <br />                                            cies consistently-conserved complexes not, retain essentiality <br />                                            appearing lose complex components. <br />  <br />  <br />                                            The E. coli protein complexome model species <br />                                            E. coli frequently used model organism bacteria general. Using literature- <br />                                            curated set protein complexes EcoCyc, sought determine protein <br />                                            complexome serves model complexes bacterial species. A comparison the <br />                                            fractional conservation EcoCyc complex 894 different bacterial genomes the <br />                                            result (Fig. 6; S5 Fig. expanded version). The genomes comparison ar- <br />                                            ranged NCBI taxonomy definitions, revealing patterns complex conservation closely <br />                                            corresponding numerous taxonomic boundaries. Hierarchical clustering E. coli <br />                                            model complex (specifically, UPGMA) basis fractional conservation all <br />                                            species reveals groups complexes similar patterns predicted conservation. <br />                                                The species overall conservation E. coli complexes are, unsurprisingly, <br />                                            closely related E. coli. Roughly complex set conserved all <br />                                            species minimal Rickettsia Mycoplasma genomes, others, serving nota- <br />                                            ble exceptions. The middle shows difference conservation Pro- <br />                                            teobacteria species. The Lactobacillales difference conservation <br />                                            complexes, degree resemble Cyanobacteria closely among <br />                                            subset. The (far left Fig. 6) complexes demonstrate variable <br />                                            conservation species. Many complexes missing partially conserved <br />                                            Proteobacteria fully present Firmicutes species extrem- <br />                                            ophiles like Thermus Thermotoga species. Overall, 285 EcoCyc complexes, 12 (~4%) <br />                                            component present 894 bacterial genomes set. None perfectly <br />                                            conserved genomes 14 complexes appear conserved 95% of <br />                                            genomes. If potential complex conservation generously defined conservation at <br />                                            half complex components, 3 EcoCyc complexes potentially conserved all <br />                                            894 genomes, 25 potentially conserved 95% genomes, 186 potentially <br />                                            conserved half genomes. Variance set complex <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                        10 / 23 <br />                                                                                                                                Protein Complexes Bacteria <br />  <br />  <br />  <br />  <br /> Fig 6. E. coli complex conservation Bacteria corresponds taxonomic boundaries. The heat map displays fractional conservation 285 <br /> EcoCyc protein complexes Fig. 5, case 894 different bacterial genomes indicated tree left. See Materials Methods <br /> for taxonomic details. Specific complex names species/strain names S5 Fig. Complexes (columns) clustered basis of <br /> the distance average fractional conservations (average linkage). <br /> doi:10.1371/journal.pcbi.1004107.g006 <br />  <br />                                                conservation fractions 0.189. Because conservation complexes follows existing <br />                                                taxonomy well, generally well-conserved complexes like RNA polymerases miss- <br />                                                ing entire genera. <br />                                                    The experimentally-determined protein complexes identified Hu et al. used as <br />                                                model E. coli complexome (S6 Fig.). Most complexes appear partial conserva- <br />                                                tion nearly species using model. Distinctions seen minimal ge- <br />                                                nomes Rickettsiales Mycoplasma genomes related species. Out <br />                                                310 Hu et al. complexes, 16 (~5%) component present 894 bacterial <br />                                                genomes set. As EcoCyc complexes, perfectly conserved ge- <br />                                                nomes single complex (complex 271) appears conserved 95% the <br />                                                genomes. Using 50% cutoff potential complex conservation used above, Hu <br />                                                complexes appear conserved 894 genomes, 10 potentially conserved 95% <br />                                                genomes, 182 potentially conserved half genomes. Though <br />                                                Hu et al. complex values appear similar Ecocyc complexes, variance <br />                                                set Hu complex conservation fractions 0.097, indicating variability <br />                                                values seen EcoCyc complexes. This lesser variance be <br />                                                seen surprising consistency taxonomic lines (S6 Fig.). <br />                                                    Both literature-curated EcoCyc model Hu et al.-based experimental model were <br />                                                evaluated comparision randomized version respective components. For lit- <br />                                                erature-curated model, Pearson correlation 0.185, experimental model, <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                                       11 / 23 <br />                                                                                                                                Protein Complexes Bacteria <br />  <br />  <br />  <br />  <br /> Fig 7. Conserved complex components enriched essential proteins. This correlation pronounced Mycoplasma (blue). Protein <br /> complexes E. coli [5] compared complexes M. pneumoniae [6] vice versa. Fraction conservation fraction essentiality calculated <br /> as described Materials Methods. Each node represents single protein complex relative size corresponding size complex number <br /> of components. K&#195;&#188;hner complex 50 corresponding Hu complex 77 indicated example complex match. <br /> doi:10.1371/journal.pcbi.1004107.g007 <br />  <br />                                                Pearson correlation 0.293. The higher correlation value experimental model indi- <br />                                                cates closer random distribution complex correlation fractions species <br />                                                set. We expect complexes conserved random pattern indicate the <br />                                                Hu et al. complex set useful EcoCyc complex set prediction wide <br />                                                range genomes. <br />  <br />  <br />                                                Essentiality proteins complexes impact paralogy <br />                                                Mycoplasma species highly reduced genomes generally assumed have <br />                                                retained essential proteins. In fact, fraction conserved essential proteins much <br />                                                higher comparing Mycoplasma pneumoniae E. coli vice-versa (Fig. 7). In these <br />                                                comparisons, complex components searched genomes essentiality as- <br />                                                signed based target species. Among set Hu et al. E. coli complexes, complexes <br />                                                average conservation fraction 0.198&#194;&#177;0.230 average essentiality fraction of <br />                                                0.122&#194;&#177;0.196 M. pneumoniae. High variability conservation complexes expected <br />                                                complex components, like single proteins, subject broad variety evolutionary pres- <br />                                                sures. Among 53% complexes component present M. pneumoniae, <br />                                                average fractions increase 0.375&#194;&#177;0.184 0.231&#194;&#177;0.218, respectively. Among set <br />                                                K&#195;&#188;hner et al. M. pneumoniae complexes, complexes average conservation fraction <br />                                                0.716&#194;&#177;0.292 average essentiality fraction 0.32&#194;&#177;0.332 E. coli. Among 95% of <br />                                                complexes component present E. coli, average fractions increase to <br />                                                0.755&#194;&#177;0.245 0.337&#194;&#177;0.332, respectively. Overall, Mycoplasma protein complex components <br />                                                likely present essential E. coli E. coli protein complex components <br />                                                Mycoplasma. <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                                       12 / 23 <br />                                                                                                                    Protein Complexes Bacteria <br />  <br />  <br />  <br />                                                One possible explanation lower fraction conserved essential proteins E. coli is <br />                                            presence paralogs renders duplicate genes non-essential, given presence an <br />                                            additional copy redundant function. We performed comparisons fraction con- <br />                                            servation complex sum paralogy (that is, total number copies all <br />                                            genes coding complex components target species). As number paralogs for <br />                                            gene broadly defined using orthologous groups, numbers considered maxi- <br />                                            mum possible values specific counts known paralogous regions. We observed an <br />                                            inverse trend E. coli complexes vs. M. pneumoniae (S3A Fig.) vice versa (S3B Fig.): <br />                                            paralogs E. coli conserved proteins Mycoplasma <br />                                            vice versa. More specifically, E. coli complexes conservation fraction greater 0.6 <br />                                            M. pneumoniae total paralogy sums lower 40 poorly-conserved <br />                                            complexes paralogy sums 2 100. M. pneumoniae complexes con- <br />                                            servation fraction greater 0.6 E. coli range sums paralogy 2 and <br />                                            nearly 80. The poorly-conserved complexes paralogy sums 60 less. Calculat- <br />                                            ed Pearson anti-correlation E. coli complexes vs. M. pneumoniae (S3A Fig.) -0.04 and <br />                                            Pearson correlation M. pneumoniae complexes model E. coli (S3B Fig.) 0.05, in- <br />                                            dicating limited overall correlation comparison. As case conser- <br />                                            vation complexes species (Fig. 6), correlation case-specific. <br />                                                The fraction essential components protein complexes non-random be <br />                                            greater expected, depending complexes compared (Fig. 8). When compared to <br />                                            random assortment, Hu et al. E. coli complexes essential proteins expected by <br />                                            chance (Fig. 8A). A Spearman anti-correlation -0.25 found. E. coli complexes Eco- <br />                                            Cyc (Fig. 8B) demonstrate similar trends, Spearman anti-correlation -0.22. M. pneu- <br />                                            moniae complexes K&#195;&#188;hner et al. (Fig. 8C) trend declining essentiality <br />                                            compared randomized essentiality fractions 0.6&#226;&#8364;&#8220;08. A Spearman anti-correlation -0.03 <br />                                            M. pneumoniae complex set. Both E. coli anti-correlations a <br />                                            weak relationship. <br />                                                Paralogy examined function essentiality (S4 Fig.). Here, average paralogy <br />                                            values determined complex minimize impact complex size, especially as <br />                                            components complex essential. E. coli complexes Hu et al. <br />                                            (S4A Fig.) decrease average paralogy fraction essentiality decreases. In total, com- <br />                                            pared random assortment, far E. coli complexes expected appear essenti- <br />                                            ality fractions 0.4 more. E. coli complexes EcoCyc (S4B Fig.) demonstrate similar <br />                                            trends. M. pneumoniae complexes K&#195;&#188;hner et al. (S4C Fig.), however, appear re- <br />                                            tain relationship essentiality average paralogy. Additionally, M. <br />                                            pneumoniae complexes expected essentiality fractions 0.2 less <br />                                            fewer expected essentiality fractions greater 0.6. Spearman anti-correlation <br />                                            statistically significant -0.03. Overall, essentiality average paralogy appear be <br />                                            related E. coli M. pneumoniae complexes, probably M. pneumoniae con- <br />                                            tains relatively paralogs. Example complexes sets OGs shared <br />                                            provided S4D Fig.; example, majority complex com- <br />                                            ponents essential representative genomes contain paralogs coding redun- <br />                                            dant complex components. <br />  <br />  <br />                                            Proteins unknown function <br />                                            Protein complexes attractive targets functional analysis, given proteins embed- <br />                                            ded functional context. This especially true proteins unknown function are <br />                                            complex (Fig. 9A, B). Here, conservation defined greater 0.5 conservation <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                         13 / 23 <br />                                                                                                                            Protein Complexes Bacteria <br />  <br />  <br />  <br />  <br />                                            Fig 8. Essentiality proteins complexes. Distribution essential genes E. coli (Hu <br />                                            et al. (A) EcoCyc (B), respectively) M. pneumoniae (C). The fraction essential genes protein <br />                                            complexes determined complex set. In E. coli, essential protein complexes enriched for <br />                                            essential proteins. By contrast, complexes non-essential proteins over-represented genome- <br />                                            reduced Mycoplasma pneumoniae. Each distribution expressed binned log2 ratios observed over <br />                                            expected frequency. Values indicate observed frequency random results (= 1), respectively. <br />                                            doi:10.1371/journal.pcbi.1004107.g008 <br />  <br />  <br />  <br />  <br />                                            fraction essential complexes essential component target <br />                                            species. Among highly conserved components, essential 4 8 spe- <br />                                            cies. Using species reduces effect noise inconsistency essential- <br />                                            ity screens. Starting 39 EcoCyc-defined complexes contain unknown proteins, least <br />                                            15 appear conserved species study Mycoplasma. Fig. 9C displays <br />                                            example complexes Hu (E. coli) K&#195;&#188;hner (Mycoplasma pneumoniae) complexes, <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                                   14 / 23 <br />                                                                                                                                Protein Complexes Bacteria <br />  <br />  <br />  <br />  <br /> Fig 9. Protein complexes rich highly-conserved proteins unknown function. (A) The list EcoCyc E. coli protein complexes compared <br /> on basis component presence vs. absence seven species study. Conserved complexes, figure, least <br /> one orthologous component present target species. Similarly, essential complexes include component essential E. <br /> coli target species. (B) As (A), subset EcoCyc complexes containing protein unknown unclear function. In <br /> these instances, complex itself known function roles components remain unclear. (C) Examples experimentally- <br /> observed protein complexes containing proteins unknown function. E. coli complex examples Hu et al. shown left, M. pneumoniae complexes <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> K&#195;&#188;hner et al. shown right. Complexes labeled identifier used corresponding study. <br /> doi:10.1371/journal.pcbi.1004107.g009 <br />  <br />  <br />  <br />                                                respectively. Unlike parts A B, complexes shown experimental results rather <br />                                                literature-defined complexes. Each complex contains component unknown <br />                                                unclear function, context protein complex broader cellular function. <br />                                                For instance, complex 66 Hu et al. (Fig. 9C) consists 6 unique proteins 3 are <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                                       15 / 23 <br />                                                                                                                   Protein Complexes Bacteria <br />  <br />  <br />  <br />                                            unknown function (or remain annotation). Of 6 proteins, 3 highly conserved <br />                                            1 frequently essential. The E. coli protein MraZ, present Hu complex <br />                                            149, shown protein unknown function recently transcrip- <br />                                            tional regulator involved multiple pathways [15]. More 149 Hu et al. E. coli complexes <br />                                            34 K&#195;&#188;hner et al. Mycoplasma pneumoniae (183 total) complexes contain one <br />                                            component unknown function. Of these, 109 Hu et al. E. coli complexes 19 K&#195;&#188;hner et al. <br />                                            M. pneumoniae complexes contain components highly conserved essential proteins. The full <br />                                            list experimental complexes unknown components available S11 Table. <br />  <br />  <br />                                            Discussion <br />                                            The substantial variation protein complexes species supports notion that <br />                                            complexes malleable previously thought. A possible explanation of <br />                                            function complex important content. Complexes serve <br />                                            role contain different proteins function lost, the <br />                                            gap. Other studies functional redundancy lead variation there <br />                                            little overlap terms protein interaction species [2,3]. While mutational change in <br />                                            protein complex catastrophic potential, complexes immutable. In fact, sev- <br />                                            eral complexes essential species varying composition species. <br />                                            For instance, 5 9 components E. coli Sec translocation complex (EcoCyc: SEC-SE- <br />                                            CRETION-CPLX) well-conserved species P. aeruginosa M. genitalium. One <br />                                            components, SecA, essential species focused work <br />                                            exception S. sanguinis; orthologs protein present 894 bacterial ge- <br />                                            nomes examined. The remaining 4 E. coli components variable conservation across <br />                                            species. For instance, YajC present 727 894 genomes. Strong selection <br />                                            pressure avoid mutations render entire complex ineffectual. This explain <br />                                            observed higher level conservation protein complex components for <br />                                            proteins general (Fig. 1). <br />                                                The essential &#226;&#8364;&#339;core&#226;&#8364;? components protein complexes conserved taxonomic <br />                                            levels &#226;&#8364;&#339;accessory&#226;&#8364;? components [1]. Given multiple interactions, proteins <br />                                            protein complexes highly conserved &#226;&#8364;&#339;un-complexed&#226;&#8364;? <br />                                            ones, retain essential roles fellow complex members present <br />                                            [16,17]. <br />                                                Components protein complexes are, average, likely present bacte- <br />                                            rial species proteins complexes [1]. This result high conservation sets <br />                                            large, essential complexes. 128 285 literature-verified E. coli protein complexes are <br />                                            fully present B. subtilis, 30 completely present M. genitalium. For in- <br />                                            stance, components ATP synthase complex (EcoCyc: ATPSYN-CPLX) present in <br />                                            species examined, vary essentiality. B. subtilis essentiality screens found <br />                                            essential genes ATP synthase, M. pneumoniae compo- <br />                                            nent essential. Other complexes&#226;&#8364;&#8221;predominantly transmembrane domains <br />                                            and/or transporter functions&#226;&#8364;&#8221;are variable conservation essentiality, though <br />                                            provide examples dispensable accessory proteins be. <br />                                                Some protein complexes essential functions E. coli present spe- <br />                                            cies. The lipopolysaccharide transport complex (EcoCyc: CPLX0&#226;&#8364;&#8220;7992) serves excellent <br />                                            example: seven Lpt proteins complex essential E. coli <br />                                            conservation limited Gram-negative species including C. crescentus <br />                                            P. aeruginosa. We transmembrane protein complexes follow pattern. <br />                                            Interestingly, species partial complex component conservation vs. E. coli highlight <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                        16 / 23 <br />                                                                                                                     Protein Complexes Bacteria <br />  <br />  <br />  <br />                                            situations core elements complex conserved modified carry <br />                                            functions adapted special physiological circumstances. For example, 3 4 <br />                                            succinate dehydrogenase complex (EcoCyc: SUCC-DEHASE) components E. coli are <br />                                            present B. subtilis S. sanguinis. This especially interesting example <br />                                            components, SdhC SdhD, inner membrane proteins, SdhC is <br />                                            present three-component B. subtilis succinate dehydrogenase. We conclude mem- <br />                                            brane proteins complexes particularly malleable, given role signaling and <br />                                            transport reflects adaptations specific environments nutrients present <br />                                            them. <br />                                                Smaller reduced bacterial genomes (that is, relative E. coli) appear <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span> a <br />                                            greater fraction highly-conserved protein complexes. This conservation evident com- <br />                                            parisons Mycoplasma pneumoniae protein complexes. In examination pro- <br />                                            tein complex components 800 bacterial genomes, species such <br />                                            M. pneumoniae offers better model protein complexes critical bacterial life. <br />                                            Protein complexes observed M. pneumoniae retained core set func- <br />                                            tions utilized higher degree multifunctionality metabolic enzymes <br />                                            [18,19]. <br />                                                Surprisingly, essential proteins poorly conserved essentiality itself not <br />                                            conserved species (Figs. 4 6). This suggests functions replaced <br />                                            non-homologous displacement [20] genomes malleable evolutionary <br />                                            terms previously expected. Clearly, evolutionary flexibility contributed to <br />                                            success microbes populate possible environments planet. Variability com- <br />                                            plex conservation highlights limitation study: unavoidably limited the <br />                                            availability sequenced bacterial genomes. Newly-characterized genomes reveal addi- <br />                                            tional variation consistency protein complexes highly reduced in <br />                                            respects. <br />                                                As protein components, individual complexes reveal underlying evolutionary <br />                                            processes (Fig. 6 S5 Fig.). The highly-conserved complexes functions <br />                                            critical microbial life, including transcription, translation, transcript degradation. <br />                                            Though different RNA polymerase (RNAP) holoenzymes (that is, RNA polymerases dif- <br />                                            ferent sigma factors) considered distinct complexes study, bacterial species <br />                                            unsurprisingly retained type RNAP. The ribosome (EcoCyc: CPLX0&#226;&#8364;&#8220;3964) is <br />                                            well-conserved size high level conservation obscure cross- <br />                                            species differences. <br />                                                Variable conservation complexes visible Escherichia genomes. <br />                                            CPLX0&#226;&#8364;&#8220;7909 (the RnlA-RnlB toxin-antitoxin complex) appears present K-12 <br />                                            E. coli single species Shewanella Photobacterium. This toxin-antitoxin system <br />                                            role bacteriophage resistance E. coli [21] unclear function re- <br />                                            tained distantly related bacteria. CPLX0&#226;&#8364;&#8220;2001 (the ferric dicitrate transport system) provides <br />                                            example gradual change. This complex spans membrane, suggesting conser- <br />                                            vation membrane-dependent. This appears case conserved <br />                                            Proteobacteria (except Rickettsiales Buchnera species) poorly- <br />                                            conserved species traditionally considered Gram positive. A subset com- <br />                                            plexes, including CPLX0&#226;&#8364;&#8220;1163 (HslVU protease) ABC-56-CPLX (aliphatic sulfonate ABC <br />                                            transporter), fit strict co-conservation model: complexes present in <br />                                            form fraction E. coli model complex. These complexes ex- <br />                                            ceptions rule. Using E. coli model, complexes conserved perfectly <br />                                            wide range species; fact, complexes fractionally conserved. <br />  <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                          17 / 23 <br />                                                                                                                     Protein Complexes Bacteria <br />  <br />  <br />  <br />                                            Materials Methods <br />                                            All data management performed using in-house Python scripts (SPICEDNOG; available at <br />                                            http://github.com/caufieldjh/spicednog). Statistical analysis clustering performed <br />                                            using R package vegan [22]. <br />  <br />                                            Data sources <br />                                            The set protein complexes Escherichia coli K-12 W3110 defined Hu et al. [5] <br />                                            assigned membership orthologous groups (OGs) version 3 eggNOG database <br />                                            [10] protein complex assigned single OG. The remaining loci were <br />                                            referred using original locus identifiers (in case, b-codes) retained <br />                                            analysis. The process repeated protein complexes isolated K&#195;&#188;hner <br />                                            et al. [6] Mycoplasma pneumoniae M129 E. coli protein complexes defined the <br />                                            EcoCyc database [13]. A representative set species (Bacillus subtilis 168, Caulobacter <br />                                            crescentus, Helicobacter pylori 26695, Mycoplasma genitalium G37, Pseudomonas aeruginosa <br />                                            UCBPP-PA14, Streptococcus sanguinis SK36) whole-genome gene essentiality <br />                                            data selected in-depth analysis. This species set referred focused set. Lists of <br />                                            protein-coding loci species obtained using respective proteome sets <br />                                            UniProt (see S4 Table taxonomy IDs corresponding genomes used). Essentiality <br />                                            data collected Database Essential Genes [11]. Protein structures obtained <br />                                            Protein Data Bank (www.rcsb.org, [23]) referenced used. <br />                                                A set 894 species, referred large set, prepared using bacterial spe- <br />                                            cies present eggNOG v.3 NCBI Taxonomy database [24]. The trees shown Figs. <br />                                            4, 5, 6 cladograms intended general relationship species within <br />                                            context consensus taxonomy. <br />  <br />                                            Orthologous groups <br />                                            Each locus genome assigned single orthologous group (OG) eggNOG v.3 <br />                                            [10], loci assigned COG, NOG, bactNOG, depending the <br />                                            widely-conserved group assignment available (see Powell et al. [10] details regarding <br />                                            OG levels). Next, presence locus determined entire set bacterial <br />                                            species; locus seen half bacterial species assigned conservation value of <br />                                            0.5. This presence averaged loci generate value average locus conservation <br />                                            genome. This value adjusted based locus coverage eggNOG (i.e., 70 <br />                                            percent loci genome mapped eggNOG OGs, average value reduced 30 <br />                                            percent.) An identical set comparisons performed loci predicted paralogs <br />                                            (that is, loci OG assignment) removed prior comparison. Subsets selected <br />                                            species prepared included loci orthologous groups <br />                                            seen Hu et al., EcoCyc, K&#195;&#188;hner et al. protein complex sets. Genome sizes <br />                                            retrieved NCBI GenBank KEGG GENOME [25]. <br />  <br />                                            Comparing complex composition random model observe the <br />                                            distribution essentiality <br />                                            The observed distributions essential genes coding protein complex compo- <br />                                            nents obtained using protein complex sets [5,6,13], eggNOG (v. 3) [10], Database <br />                                            Essential Genes [11] defined above. For single protein complex, essentiality fraction <br />                                            defined fraction genes complex essential, set all <br />                                            unique protein-coding genes complex. The conservation scores used judge <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                          18 / 23 <br />                                                                                                                     Protein Complexes Bacteria <br />  <br />  <br />  <br />                                            participation complex dataset, establishing maximum species data- <br />                                            set combination. Second, essentiality fraction linking essential protein an <br />                                            OG. In instances multiple proteins shared OGs essentiality, essentiality con- <br />                                            sidered primary case OG counted essential. <br />                                                A random model created purpose comparing data set background <br />                                            noise. The random model, meant represent collection randomly sized complexes, was <br />                                            populated proteins randomly assigned essential status. The complex sizes <br />                                            randomly assigned value ten. Each complex assigned protein val- <br />                                            ues essential non-essential status. The probability essential determined <br />                                            overall percent essential genes organism, random model size is <br />                                            equal maximum species dataset compared. This random model was <br />                                            binning process observed data. The mean bin ob- <br />                                            tained 10,000 replications. This results s bins size longer equal ac- <br />                                            tual data set demonstrates appropriate background noise level comparison purposes. <br />                                            The log2(Observed/Expected) values plotted Fig. 8 significant difference be- <br />                                            tween observed essentiality expected. <br />  <br />  <br />                                            Comparative proteome complexome analysis <br />                                            The general scheme data analysis follows: (1) A list orthologous groups (OG) <br />                                            produced 894 bacterial species large set defined above. (2) Presence <br />                                            absence OG determined species. (3) Repeated OGs removed from <br />                                            list step 2 repeated. (4) The list step 1 used map OGs compo- <br />                                            nents sets protein complexes. The complexes compared search cross- <br />                                            data set complex matches. Gene essentiality mapped OG species- <br />                                            dependent basis. (5) A list 8 taxonomically-divergent species selected used define <br />                                            fractional conservation fractional essentiality protein complex. <br />                                                OGs used basis comparison similarity data sets. Complex size <br />                                            defined number unique proteins isolated complex; i.e. complex con- <br />                                            tain 3 unique OGs 4 distinct protein components, yielding complex size 4. For each <br />                                            complex, presence OG complex assayed proteome sets of <br />                                            seven representative species. The resulting binary presence/absence values com- <br />                                            bined produce value percent complex conservation. This value intentionally disre- <br />                                            gards gene context similarity (that is, OG present genomes if <br />                                            neighboring genes differ genomes) simply expresses fraction complex <br />                                            components specific genome <span id='am-8' about='Thesaurus:code' typeof='owl:Thing'>code</span> for. When target proteome did contain a <br />                                            specified complex component, number paralogs component-coding gene de- <br />                                            termined number proteins list mapping OG. While verifica- <br />                                            tion, necessary define protein-coding genes true paralogs, simply <br />                                            used OGs (including paralogs) determined eggNOG. <br />                                                All protein complex components assigned binary essentiality values using pub- <br />                                            lished assays specific species listed above. These values used define essentiali- <br />                                            ty fraction potentially conserved complex, i.e. E. coli complex 80% the <br />                                            components appear conserved M. pneumoniae 60% components may <br />                                            essential species. <br />                                                A broader comparison prepared using list 894 species defined above. Genome <br />                                            sizes species retrieved KEGG GENOME Database (http://www.genome. <br />                                            jp/kegg/genome.html, [25]). For species, total number OG-mapped protein-coding <br />                                            loci divided total number loci produce value percentage mapped. Using <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                          19 / 23 <br />                                                                                                                    Protein Complexes Bacteria <br />  <br />  <br />  <br />                                            list OGs species, OG compared species determine its <br />                                            conservation Bacteria. Adjusted average locus conservation particular genome, <br />                                            CAAL(g), calculated as: <br />                                                                                               P        <br />                                                                                                     CL &#195;&#176;g&#195;&#382; <br />                                                                                                    L&#195;&#176;g&#195;&#382; <br />                                                                                   CAAL &#195;&#176; g &#195;&#382; &#194;&#188; m <br />                                                                                                    N <br />                                            CL number genomes locus present, L(g) number loci in <br />                                            genome, N total number genomes, m percentage loci mapped egg- <br />                                            NOG v.3. Values adjusted using fraction loci actually mapped unmapped loci <br />                                            lower effective conservation. <br />                                               An identical list values, repeated OGs reduced single occurrence, aver- <br />                                            aged produce average OG conservation. This modification removes effect counting <br />                                            loci share OGs, happen loci paralo- <br />                                            gous. Adjusted average OG conservation particular genome, CAAO(g), calculated as: <br />                                                                                                P        <br />                                                                                                     CL &#195;&#176;g&#195;&#382; <br />                                                                                                    O&#195;&#176;g&#195;&#382; <br />                                                                                   CAAO &#195;&#176; g &#195;&#382; &#194;&#188; m <br />                                                                                                    N <br />                                            CL number genomes locus present, O(g) number unique <br />                                            OGs genome, N total number genomes, m percentage loci mapped <br />                                            eggNOG v.3. <br />                                               Species/strains sorted genome size compared average conservation val- <br />                                            ues. For set bacterial genomes, N = 943, Fig. 1 presents results remov- <br />                                            ing 45 genomes similar size sequence. For subset species, Average Locus <br />                                            Average OG Conservation values calculated using OGs published pro- <br />                                            tein complex data sets. <br />                                               Mapping fractional complex conservation species performed follows for <br />                                            focused set (8 species) large set. A cladogram species set pre- <br />                                            pared using Interactive Tree Life (iTOL, [26]) project NCBI taxonomy. All protein <br />                                            components mapped eggNOG v.3 OGs complex size determined defined <br />                                            above. Conservation fraction complex species defined number of <br />                                            complex component OGs shared model (an E. coli complex) target ge- <br />                                            nome size model complex. Heatmaps prepared using R heatmap.2 <br />                                            function gplots package. Randomized models large set heatmaps (Fig. 5, S5 S6 <br />                                            Figs.) retaining species order randomized distribution conservation frac- <br />                                            tions prepared using R function randomizeMatrix (in picante package [27]) and <br />                                            &#226;&#8364;&#732;richness&#226;&#8364;&#8482; null model respect overall conservation levels. <br />  <br />  <br />                                            Supporting Information <br />                                            S1 Fig. All EcoCyc complexes fractional conservation selected bacterial species. <br />                                            An extended version Fig. 5. Names blue stars indicate example complexes shown in <br />                                            Fig. 4. <br />                                            (PDF) <br />                                            S2 Fig. All EcoCyc complexes fractional essentiality selected bacterial species. <br />                                            An extended version Fig. 5. Names blue stars indicate example complexes shown in <br />                                            Fig. 4. <br />                                            (PDF) <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                         20 / 23 <br />                                                                                                                  Protein Complexes Bacteria <br />  <br />  <br />  <br />                                            S3 Fig. Cross-species conservation experimentally-observed protein complexes the <br />                                            sums counts potential paralogs components. (A) Proteins E. coli com- <br />                                            plexes [5] tend paralogs complexes conserved. (B) By contrast, in <br />                                            M. pneumoniae complexes [6] conserved complexes tend paralogous pro- <br />                                            teins. Fraction conservation sum paralogy calculated described Materials <br />                                            Methods. Each node represents single protein complex relative size corresponding <br />                                            size complex number components. E. coli complexes defined Hu et al. <br />                                            compared M. pneumoniae proteome M. pneumoniae complexes were <br />                                            compared E. coli proteome; cross-species comparison using predicted <br />                                            orthologs described Materials Methods. <br />                                            (PDF) <br />                                            S4 Fig. Essentiality protein complexes average paralogy. More essential protein <br />                                            complexes tend fewer components paralogs, E. coli literature-curated <br />                                            Ecocyc (A) experimentally-observed Hu et al. (B) complexes. However, true in <br />                                            reduced genomes Mycoplasma pneumoniae (C). Each node represents single <br />                                            protein complex relative size corresponding size complex number com- <br />                                            ponents. Fraction essentiality average paralogy calculated described Materials <br />                                            Methods. Data Hu et al. [5] (A), EcoCyc [13] (B) [6] (C). (D) An example com- <br />                                            plex data sets shown. These complexes identical composi- <br />                                            tion similar components. Each complex, defined single data set, offer an <br />                                            incomplete set protein components overlook impact paralogy. <br />                                            (PDF) <br />                                            S5 Fig. E. coli literature-curated complex conservation bacteria corresponds taxo- <br />                                            nomic boundaries. This figure expands Fig. 6. Using EcoCyc set protein <br />                                            complexes model, column single complex set row distinct <br />                                            bacterial genome. 285 complexes 894 genomes shown total. Genome order corre- <br />                                            sponds cladogram produced using NCBI taxonomy Interactive Tree Life (iTOL, <br />                                            [26]) tree. Color gradients correspond fractional conservation. Complexes (columns) have <br />                                            clustered basis distance average fractional conservations (aver- <br />                                            age linkage). <br />                                            (PDF) <br />                                            S6 Fig. E. coli experimentally-observed complex conservation bacteria corresponds <br />                                            taxonomic boundaries. Using Hu et al. set protein complexes model, col- <br />                                            umn single complex set row distinct bacterial genome. 310 com- <br />                                            plexes 894 genomes shown total. Genome order corresponds cladogram <br />                                            produced using NCBI taxonomy Interactive Tree Life (iTOL, [26]) tree. Color gradi- <br />                                            ents correspond fractional conservation. Complexes (columns) clustered the <br />                                            basis distance average fractional conservations (average linkage). <br />                                            (PDF) <br />                                            S1 Text. Guide content provided supporting tables. <br />                                            (DOCX) <br />                                            S1 Table. Key protein complex IDs shown Fig. 5. <br />                                            (XLSX) <br />                                            S2 Table. Protein complex conservation bacteria. <br />                                            (XLSX) <br />  <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                       21 / 23 <br />                                                                                                                              Protein Complexes Bacteria <br />  <br />  <br />  <br />                                            S3 Table. Conservation orthologous groups species pairs. <br />                                            (XLSX) <br />                                            S4 Table. Conservation numerous bacterial species. <br />                                            (XLSX) <br />                                            S5 Table. Conservation E. coli complexes Hu et al. (2009). <br />                                            (XLSX) <br />                                            S6 Table. Essentiality E. coli complexes Hu et al. (2009). <br />                                            (XLSX) <br />                                            S7 Table. Conservation E. coli complexes EcoCyc. <br />                                            (XLSX) <br />                                            S8 Table. Essentiality E. coli complexes EcoCyc. <br />                                            (XLSX) <br />                                            S9 Table. Conservation Mycoplasma pneumoniae complexes K&#195;&#188;hner et al. (2009). <br />                                            (XLSX) <br />                                            S10 Table. Essentiality Mycoplasma pneumoniae complexes K&#195;&#188;hner et al. (2009). <br />                                            (XLSX) <br />                                            S11 Table. Experimental protein complexes containing uncharacterized components. <br />                                            (XLSX) <br />  <br />  <br />                                            Acknowledgments <br />                                            We thank Stefan Wuchty (Miami) critically reading manuscript. <br />  <br />                                            Author Contributions <br />                                            Conceived designed experiments: PU JHC. Performed experiments: JHC MA CW. <br />                                            Analyzed data: PU JHC MA CW. Wrote paper: PU JHC MA CW. <br />  <br />                                            References <br />                                             1.   Ryan CJ, Roguev A, Patrick K, Xu J, Jahari H, Tong Z, et al. Hierarchical modularity evolution <br />                                                  genetic interactomes species. Molecular Cell. 2012; 46(5):691&#226;&#8364;&#8220;704. doi: 10.1016/j.molcel. <br />                                                  2012.05.028 PMID: 22681890 <br />                                             2.   Dixon SJ, Costanzo M, Baryshnikova A, Andrews B, Boone C. Systematic mapping genetic interac- <br />                                                  tion networks. Annual Review Genetics. 2009; 43:601&#226;&#8364;&#8220;25. doi: 10.1146/annurev.genet.39.073003. <br />                                                  114751 PMID: 19712041 <br />                                             3.   Ryan CJ, Krogan NJ, Cunningham P, Cagney G. All nothing: protein complexes flip essentiality be- <br />                                                  tween distantly related eukaryotes. Genome Biology Evolution. 2013; 5(6):1049&#226;&#8364;&#8220;59. doi: 10.1093/ <br />                                                  gbe/evt074 PMID: 23661563 <br />                                             4.   Arifuzzaman M, Maeda M, Itoh A, Nishikata K, Takita C, Saito R, et al. Large-scale identification pro- <br />                                                  tein-protein interaction Escherichia coli K-12. Genome Research. 2006; 16:686&#226;&#8364;&#8220;91. PMID: 16606699 <br />                                             5.   Hu P, Janga SC, Babu M, D&#195;&#173;az-Mej&#195;&#173;a JJ, Butland G, Yang W, et al. Global functional atlas Escheri- <br />                                                  chia coli encompassing previously uncharacterized proteins. PLoS Biology. 2009; 7:e96. doi: 10.1371/ <br />                                                  journal.pbio.1000096 PMID: 19402753 <br />                                             6.   K&#195;&#188;hner S, van Noort V, Betts MJ, Leo-Macias A, Batisse C, Rode M, et al. Proteome organization a <br />                                                  genome-reduced bacterium. Science (New York, NY). 2009; 326:1235&#226;&#8364;&#8220;40. doi: 10.1126/science. <br />                                                  1176343 PMID: 19965468 <br />                                             7.   Sharan R, Ideker T, Kelley B, Shamir R, Karp RM. Identification protein complexes comparative <br />                                                  analysis yeast bacterial protein interaction data. Journal Computational Biology: journal of <br />                                                  computational molecular cell biology. 2005; 12:835&#226;&#8364;&#8220;46. PMID: 16108720 <br />  <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                                     22 / 23 <br />                                                                                                                              Protein Complexes Bacteria <br />  <br />  <br />  <br />                                             8.   Matos Simoes R, Dehmer M, Emmert-Streib F. Interfacing cellular networks S. cerevisiae and <br />                                                  E. coli: connecting dynamic genetic information. BMC genomics. 2013; 14:324. doi: 10.1186/1471- <br />                                                  2164-14-324 PMID: 23663484 <br />                                             9.   Cohen O, Ashkenazy H, Levy Karin E, Burstein D, Pupko T. CoPAP: Coevolution presence-absence <br />                                                  patterns. Nucleic acids research. 2013;41(Web Server issue: ):W232&#226;&#8364;&#8220;7. <br />                                            10.   Powell S, Szklarczyk D, Trachana K, Roth A, Kuhn M, Muller J, et al. eggNOG v3.0: orthologous groups <br />                                                  covering 1133 organisms 41 different taxonomic ranges. Nucleic acids research. 2012; 40:D284&#226;&#8364;&#8220;9. <br />                                                  doi: 10.1093/nar/gkr1060 PMID: 22096231 <br />                                            11.   Luo H, Lin Y, Gao F, Zhang CT, Zhang R. DEG 10, update database essential genes in- <br />                                                  cludes protein-coding genes noncoding genomic elements. Nucleic acids research. 2014; 42 <br />                                                  (Database issue):D574&#226;&#8364;&#8220;80. doi: 10.1093/nar/gkt1131 PMID: 24243843 <br />                                            12.   McCutcheon JP, McDonald BR, Moran NA. Origin alternative genetic code extremely small <br />                                                  GC-rich genome bacterial symbiont. PLoS genetics. 2009; 5(7):e1000565. doi: 10.1371/ <br />                                                  journal.pgen.1000565 PMID: 19609354 <br />                                            13.   Keseler IM, Mackie A, Peralta-Gil M, Santos-Zavaleta A, Gama-Castro S, Bonavides-Mart&#195;&#173;nez C, et al. <br />                                                  EcoCyc: fusing model organism databases systems biology. Nucleic acids research. 2013; 41: <br />                                                  D605&#226;&#8364;&#8220;12. doi: 10.1093/nar/gks1027 PMID: 23143106 <br />                                            14.   Liechti G, Goldberg JB. Outer membrane biogenesis Escherichia coli, Neisseria meningitidis, and <br />                                                  Helicobacter pylori: paradigm deviations H. pylori. Frontiers cellular infection microbiology. <br />                                                  2012; 2:29. doi: 10.3389/fcimb.2012.00029 PMID: 22919621 <br />                                            15.   Eraso JM, Markillie LM, Mitchell HD, Taylor RC, Orr G, Margolin W. The highly conserved MraZ protein <br />                                                  transcriptional regulator Escherichia coli. Journal bacteriology. 2014; 196(11):2053&#226;&#8364;&#8220;66. doi: <br />                                          <br /> </body></html>