<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Proc. Natl. Acad. Sci. USA <br /> Vol. 75, No. 2, pp. 774-778, February 1978 <br /> Biochemistry <br />  <br />  <br /> Construction characterization Escherichia coli plasmid <br /> bearing functional gene G bacteriophage 4X174* <br />     (in vitro recombination/transformation/gene-specific suppressor/spike protein) <br /> M. ZAFRI HUMAYUN AND ROBERT W. CHAMBERS <br /> New York University School Medicine, Department Biochemistry, 550 First Avenue, New York, New York 10016 <br /> Communicated Severo Ochoa, December 1, 1977 <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>           In order study mutagenic effects                   gene G product, virus spike protein, constitutively the <br /> site-specific, covalent modifications biologically active DNA,               host cell. In communication, construction <br /> we need host cells permissive type <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>mutation</span>                 characterization ColEl type plasmid containing a <br /> that produced vivo modified DNA. Spe-                      segment kX174 replicative form (RF) DNA carrying gene <br /> cifically, require general, vivo complementation                 G. We plasmid, p4XG, renders previously <br /> for bacteriophage OX174 gene G, essential gene we <br /> have chosen initial studies chemical mutagenesis.                    nonpermissive Eschertchia coli strains (H514, HF4704, and <br /> Toward end, constructed plasmid (pOXG)                      HF4740) permissive OX174 amn9, phage mutant defective <br /> carries functional copy gbX174 gene G. Three different                     gene G (5). <br /> bacterial strains nonpermissive am9, gene G <br /> amber mutant, transformed pOXG. The trans- <br /> formants permissive gene G mutant,                               METHODS AND MATERIALS <br /> the gene A E mutants tested. This paper de- <br /> scribes construction biochemical characterization                      jpX174  Strains.  Wild type R. C. Warner; am9 (a <br /> of plasmid, p$XG, describes biological                     gene G mutant), am3 (a gene E mutant), am86 (a gene A <br /> properties exhibited pOXG-bearing strains.                               mutant) R. L. Sinsheimer (5). <br />                                                                                    E. coli Strains. E. coli C (wild type) R. C. Warner. <br /> A large class mutagens, including ultimate carcinogens,                 The C/K12 hybrids, H514 Su-, 4pXs, end, uvrA, thyA, arg, rK, <br /> are reactive electrophiles produce different kinds                    mK (6), HF4704 F-, OXs, Su-, thy, uvrA (7), HF4714 F-, <br /> of covalent modifications react DNA (1). At                      Su+, OX%, arg, his, leu, thr, pro (5, 7) obtained J. <br /> present, direct method determining                     Hurwitz. HF4740 F-, Su-, OXs, recA13,uvrA103, tlr ob- <br /> these modifications produce mutations, direct                  tained P. Howard Flanders. HB129 Srm, rg, mB, gal, laC, <br /> way showing type mutation (<span id='am-83' about='xsp:base' typeof='owl:Thing'>base</span> substitution,                        leu, pro, B1, end carrying pMB9 (8, 9) M. Ptashne&apos;s <br /> frameshift, large deletion) produced different kinds                laboratory. <br /> premutational lesions DNA. Our laboratory begun                          Growth Bacteria Virus. Luria broth L plates <br /> intensive effort develop methodology enable                  used rich media (10). L-tet medium plates were <br /> explore problem directly.                                                  Luria broth (or L plates) containing 15 mg tetracycline per <br />    The availability extensive information regarding                      liter. Virus prepared single plaques grown HF4714 <br /> structure biology bacteriophage OX174 (2-4) makes                   Su+ using defined medium, modified 3XD (11). Titers were <br /> virus especially attractive model initial studies.                   measured soft agar techniques R plates (10). <br /> Briefly, approach follows: First, well-characterized                   Enzymes. Restriction endonuclease EcoRI Miles. <br /> modification introduced DNA single, preselected                    Hae III Hpa II gifts R. Roberts; Hha I <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> site combination chemical enzymatic synthesis.                      A. Jeffrey. Lambda exonuclease gift C. Radding&apos;s <br /> Second, biological response site-altered DNA ex-                 laboratory. Deoxynucleotidyltransferase PL Bio- <br /> amined isolating mutant phage produced transfection                       chemicals. RNase A Worthington Biochemicals. <br /> of permissive cells. Third, precise change produced                     DNA Preparations. OX174 RF DNA prepared a <br /> site-specific modification determined determining                     modification Godson Boyer&apos;s method (12). Plasmid DNA <br /> sequence region DNA isolated                     amplified presence chloramphenicol (200 ,ug/ml) <br /> mutant phage. In principle, covalent modifications DNA,                  isolated minor modifications &quot;cleared lysate&quot; <br /> including produced carcinogens, examined                     method Clewell (13). For electrophoretic identification of <br /> this procedure, molecular expression mutagenic                   recombinant clones, plasmid DNA partially purified as <br /> activity displayed premutational lesion de-                       follows: Lysates prepared 20 ml chlorampheni- <br /> fined.                                                                          col-treated cultures essentially according Clewell that <br />    Towards goal, need develop bacterial strains ca-                  20 ,ug RNase added detergent solution. <br /> pable suppressing base substitutions,                      After chromosomal material pelleted, clear lysates <br /> frameshifts large deletions gene G, essential gene                 extracted twice phenol times with <br /> OX174 chosen initial studies. In principle,                ether. The DNA recovered ethanol precipitation. Some <br /> complementation provides general solution problem,                    detergent coprecipitated DNA, caused prob- <br /> but technical difficulties using helper DNA              lems electrophoretic assay. <br /> in transfection process. Therefore, investigated the <br /> possibility using plasmid carrying OX174 gene G produce                 Abbreviations: RF, replicative form; TE buffer, 10 mM Tris-HCI, pH <br />                                                                                 8.0/1 mM EDTA; bp, base pairs; restriction fragments named ac- <br /> The costs publication article defrayed           cording number base pairs contain. <br /> payment page charges. This article marked           * This paper no. 1 series: &quot;Molecular mechanisms mutation <br /> &quot;advertisement&quot; accordance 18 U. S. C. &#194;&#167;1734 solely indicate           carcinogens.&quot; It presented &quot;Single Stranded DNA <br /> this fact.                                                                        Phage&quot; meeting Cold Spring Harbor, NY, August 29, 1977. <br />                                                                           774 <br />                Biochemistry: Humay-un Chambers                                                      Proc. Natl. Acad. Sci. USA 75(1978)           775 <br />  <br />                                Hae III <br />  <br />                                           1        1 Hae III           N.T <br />                                  G <br />                                                   2. Isolate          dTTP <br />                                                                       Co2 + <br />                                Hae.11 <br />  <br />  <br />  <br />  <br />                                                                               N.T <br />                                                                           dATP <br />                                                                           Co2+ <br />     FIG. 1. Experimental protocol biochemical construction hybrid molecules pMB9 DNA segment OX174 RF DNA. X <br />  Exo X exonuclease; N.T. terminal deoxynucleotidyltransferase (20). Heavy dots symbolize 5&apos; ends. &quot;A&quot; &quot;B&quot; possible orientations <br />  inserted viral DNA fragment relative HindIII restriction site shown open triangle (9). The 5&apos; 3&apos; direction viral plus <br />  strand (+) indicated arrowhead. <br />      Gel Electrophoresis Electron Microscopy. Gel elec-                          pmol) annealed incubation 200 Ml buffer (100 mM <br />   trophoretic techniques used described (14). Electron                     CaCl2 TE buffer) 500 30 min followed slow cooling <br />   microscopy formamide (50%) aqueous                              220 hours. In control experiments, tailed <br />   techniques described Davis et al. (15). The restriction                         pMB9 DNA tailed viral DNA fragment alone <br />   fragment Hae 1342f OX RF DNA used internal                          identical annealing procedures. Electron <br />   size marker.                                                                       microscopy revealed types DNA present <br />      Biochemical Construction Plasmid-Viral DNA Hybrid                            annealing, 10% linear DNA converted <br />   Molecules. The general procedure Jackson et al. (16)                        circular form. No circles detectable control ex- <br />  used DNA ligase step omitted (17). The                          periments. <br />  pMB9 DNA digested EcoRI buffer (80 mM Tris-                               Transformation. Calcium-shocked cells prepared from <br />  HCl, pH 8.0/50 mM NaCI/10 mM MgCI2/1 mM dithiothreitol)                            exponentially growing cultures according Cohen et al. (21), <br />  conditions 95% circles                        CaCI2 concentration increased 50 mM. An- <br />  converted linear forms determined electron mi-                          nealed DNA (0.15 pmol) 200 Ml buffer (50 mM CaCl2 and <br />  croscopy. The linear DNA (20 ,g) taken 200 ,l                         10 mM MgCl2 TE buffer) 00 mixed 400,Ml the <br />  buffer (67 mM glycine-KOH, pH 9.4/4 mM MgCl2)                              calcium-shocked cell suspension following <br />  digested 65 units X exonuclease (18) 1 hr 0&#194;&#176;. DNA                   sequential steps (22): incubation 00 15 min, incubation <br />  recovered following steps sequence: phenol                       420 2 min, incubation 220 10 min, addition 0.3 <br />  extraction, ether extractions, ethanol precipitation.                    ml 3 X concentrated L broth, incubation 370 30 min, <br />  Exonuclease-treated linear pMB9 DNA (10.ug) dissolved                          plating 0.1 ml L-tet plates. Tetracycline-resistant <br />  100 MAl transferase buffer (140 mM Tris-cacodylate adjusted                  colonies visible 12 hr incubation 370. <br />  pH 6.9 KOH/1 mM CoCl2/1 mM 2-mercaptoethanol)                                 The recombinant DNA experiments P 1 <br />  (19) containing 10 nmol [3H]dATP (1.1 Ci/mmol)                          containment outlined &quot;National Institutes Health <br />  incubated 66 units deoxynucleotidyltransferase (20)                    Recombinant DNA Research Guidelines, Part II.&quot; <br />  160 min 370. The reaction terminated phenol and <br /> three ether extractions, DNA solution dialyzed <br /> overnight changes (2 liters each) TE buffer (10                                                   RESULTS <br /> mM Tris*HCl, pH 8.0/1 mM EDTA). Deoxyadenylate incor- <br /> poration estimated trichloroacetic acid-precipitable                          Cloning Segment joX174 DNA Bearing Gene G. Hae <br /> radioactivity 100 residues 3&apos; end pMB9. The                       1342t largest fragment produced Hae III digestion of <br /> viral fragment, Hae 1342, prepared digesting qX174                            OX174 RF DNA contains entire structural gene (gene <br /> RF DNA Hae III fractionating products 35%                          G) spike protein parts genes F H (23, 24). <br /> preparative polyacrylamide slab gel procedures similar                        By experimental procedures schematized Fig. 1, have <br /> those described (14). Four picomoles Hae 1342                          cloned viral fragment E. coli plasmid, pMB9, a <br /> were incubated (no X exonuclease) 55 units transferase                      convenient vehicle carrying ColEl linear DNA EcoRI. <br /> in 100 Mul transferase buffer containing 10 nmol [3H]dTTP                     Then poly(dA) tails added. Poly(dT) tails added <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> <br /> (1.8 Ci/mmol) 370 60 min. Poly(dT)-tailed DNA                            viral fragment. Equimolar amounts molecular <br /> isolated above. Incorporation thymidylate estimated                       species annealed, resulting mixture used to <br /> to 80 residues 3&apos; end Hae 1342.                                     transform E. coli host, H514. Transformants selected <br />    One microgram poly(dA)-tailed, linear pMB9 DNA                                plating presence tetracycline. DNA 11 the <br /> (&apos;-0.28 pmol) 0.25 ,g poly(dT)-tailed Hae 1342 (&apos;-0.3                        346 tetracycline-resistant colonies obtained isolated and <br />                                                                                     examined agarose gel electrophoresis. Out 11 transfor- <br /> t This fragment Z1   according widely used, descriptive,         mants examined, plasmid DNA mobility <br />  nomenclature (2).                                                                  identical parental plasmid, pMB9; slowly <br />  776         Biochemistry: Humayun Chambers                                                                        Proc. Natl. Acad. Sci. USA 75 (1978) <br />                                                                                                                                   y <br />         a <br />                     708.*        (dA)n       (dT)nflJ                                                                                        560   +    (dA)n (dT)n <br />                                                                                                                                                                           i <br />              H               Z                              H                 H              H                                H                                           H   Z <br />  <br /> A            1   270         1         350           msl                201   1        289                614                 1150                          410           19ol1 <br />                                                                                                                        I          T- <br />                                                                                                                   Y               Y <br />                                                                                   Hae III    2192 * (dA)n * (dT)n <br />                   C          I <br />              H               Z                                      H                                H                H           H                                       H   Z <br />  <br /> B            1 <br />              I   270         1           350            1501 <br />                                                      --.;           I                 614            1I     289       1 201           II 8 <br />                                                                                                                                         --                  410           1901 <br />                                                                                                                                                                       A <br />                                                                                                                                               0 0 0 0 <br />                                                      -          I                 I <br />                                                             I                     I <br />                                                             y                     y <br />                        770       *   (dA)n * (dT)n                                                                                            4968 t (dA)         (dT)n <br />         b    F <br />                                                             Y <br />  <br />    FIG. 2. A partial restriction cleavage map pOXG105. (A) p4XG105 &quot;A&quot; orientation (see Fig. 1). (B) pOXG105 &quot;B&quot; orientation. <br /> (a b) Predicted Hha I bridge fragments orientations &quot;A&quot; &quot;B,&quot; respectively. (c) Predicted Hae III fragment pkXG. H, Y, and <br /> Z refer cleavage sites restriction endonucleases Hha I, Hpa II, Hae III, respectively; v marks HindIII site used orient <br /> pMB9 relative EcoRI site (9). The arrowhead gives orientation (5&apos;         3&apos;) OX174 (+) strand; dots represent (dA)n.(dT)n <br />                                                                                                     - <br />  <br />  <br />  <br /> bridges. <br />  <br /> moving DNA, suggesting larger molecule. All slowly                                       The data Fig. 3 b OX Hae 1342 bearing <br /> moving plasmids similar mobilities, indicating similar                                       gene   G inserted EcoRI site pMB9. Two <br /> size.                                                                                            different orientations (see Fig. 1) possible, giving <br />    In order gene G functional                                      slightly different bridge fragments (left: 708 + tail 770 + tail; <br /> transformants, screened plaque formation                                        right: 560 + tail 498 + tail; Fig. 2). Since values are <br /> OX174 am9, gene G mutant incapable growth                                            similar different orientations <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> <br /> parent bacterial strain (H514) transformants                                    tails known precision, difficult deter- <br /> were derived. The results showed tetracycline-                                      orientation estimated size bridge <br /> resistant transformants plasmid DNA                                           fragments Hha I digest. However, map predicts that <br /> mobility pMB9 marker did form plaques. The                                        left Hha I bridge fragment change double di- <br /> remaining transformants (harboring larger, apparently                                       gest pOXG Hha I Hpa II OX Hae 1342 the <br /> hybrid, plasmids) gave plaques,                                     &quot;A&quot; orientation shown Fig. 1, right Hha I bridge <br /> the plating efficiency. Two transformants                                     fragment shortened 40 bp. The reverse true <br /> (named H514/pOXG105 H514/pOXG109) selected                                              opposite orientation (&quot;B&quot;, Fig. 1). The cleavage <span id='am-84' about='xsp:pattern' typeof='owl:Thing'>pattern</span> <br /> for analysis.                                                                            Fig. Sc shows left Hha I bridge fragment remains <br />    Characterization poXG105 p4XG109. In addition                                          constant 830 bp. The right bridge (600 bp) disappears and <br /> to electrophoretic mobilities mentioned above, pOXG105                                       new fragment 540 bp appears. Hha I 613 disappears, it <br /> and poXG109 characterized electron microscopy                                        (in orientation) contains Hpa II site (2). <br /> restriction mapping. Electron microscopy gave average contour                                    These data suggest strongly OX Hae 1342 DNA the <br /> lengths of: (a) poly(dA)-tailed, linear, pMB9 DNA, 1.68 ,um;                                     orientation shown &quot;A&quot; Fig. 1. Similar results (not shown) <br /> (b) poly(dT)-tailed OX Hae 1342, 0.35 ,um; (c) circular mole-                                    obtained pOXG109. <br /> cules produced annealing (a) (b), 1.98 ,um; (d)                                          Biological Properties Bacterial Hosts Carrying <br /> pOXG105 p4XG109, 2.07 Aim.                                                                   p4$XG105 p4XG109. Table 1 summarizes results ex- <br />    Fig. 2 shows partial restriction map p4XG constructed                                 periments tested ability nonpermissive <br /> from available information (2, 9). Fig. 3a shows Hae III                                     host, H514, form plaques kX174 amber mutants de- <br /> cleavage pattern purified pOXGl05 DNA OX                                      fective genes A, E, G transformation <br /> RF DNA pMB9 DNA. The map predicts insertion                                          pOXG105 p4XG109. Within limits data, <br /> OX Hae 1342 single EcoRI site pMB9 cause                                        suppression specific gene G mutants. <br /> pMB9 Hae 850 (9) replaced new fragment, pOXG                                             When gene G nonsense mutant, am9, grown the <br /> Hae 2192 + (dA)n-(dT)n. Lanes 2 3 Fig. 3a shows this.                                     plasmid-bearing strain, H514/pbXG105, soft agar, small <br /> The new fragment estimated 2400 base pairs (bp)                                   turbid plaques barely visible 3 hr 370 compared to <br /> from mobility.                                                                               large (-2 mm), clear plaques produced wild-type phage. <br />    The map predicts Hha I digest pOXG                                    Several individual plaques picked examined lawns <br /> show OX fragments (201, 289, 614 bp)                                    permissive nonpermissive test strains (HF4714 Su+ and <br /> new bridge fragments join pMB9 OX Hae 1342 DNA.                                         C Su-), carry plasmid. In typical ex- <br /> The exact size bridge fragments depend                                      periment, 1/3 virus single plaque H514/ <br /> orientation OX Hae 1342 relative pMB9 length                                       p4XG105 wild type. Plaques picked longer incubation <br /> of (dA)n- (dT)n tails. The map predicts pMB9 Hha                                   greater wild-type content. This peculiar to <br /> I 1030, containing EcoRI site (9), disappear. Fig. 3b                                 H514/pOXG105 lawns am9 stock used ex- <br /> verifies predictions. From mobility, larger bridge                                 periments gives plaques containing &lt;10-5 wild-type virus on <br /> fragment estimated 830 bp pOXG105                                           permissive host, HF4714. Results similar for <br /> pOXG109; fragment appears pMB9 OX                                       H514/pOXG105 obtained H514/pOXG109. <br /> RF DNA digests themselves. The smaller bridge fragment                                          To investigate production wild-type phage plas- <br /> is estimated 600 bp pOXG105 625                                          mid-bearing cells infected am9, recombination was <br /> pOXG109.                                                                                         measured infective center assay. H514/pOXG105 cells <br />               Biochemistry: Humayun Chambers                                                 Proc. Natl. Acad. Sci. USA 75 (1978)            777 <br />                             1   2 3                       1 2 34                            1 2 3 4 <br />  <br />  <br />                                          -   2400 <br />                                                                                                              -   1030(1,4) <br />                                          - 850(2)                                                            -   830(2,3) <br />                                                                                                                  614(2) <br />                                                                                                              ={600(2) <br />                                                                                                              =~580(1,2) <br />                                                                                                                  560(3) <br />  <br />  <br />  <br />  <br />                                                            b                                   c <br />    FIG. 3. Fractionation restriction endonuclease cleavage products pkXG105 pOXG109. DNA (2-10 Mg) digested the <br /> appropriate enzyme(s) 100 ml buffer (60 mM Tris.HCl, pH 7.5/6 mM MgCl2/6 mM 2-mercaptoethanol) 3 hr 37&#194;&#176;. Reaction terminated <br /> by phenol ether extractions. Ten microliters dye mix (0.5% bromophenol blue 0.5% xylene cyanol 50% glycerol) were <br /> added. Electrophoresis vertical 5% polyacrylamide slab gels (20 X 40 X 0.15 cm) 90 mM Tris-borate/2.5 mM EDTA (pH 8.3) 400 <br /> V bromophenol blue marker reached just ran (10-12 hr). Gels stained 30 min 0.5-1 ,ug ethidium bromide <br /> per ml photographed UV light. Sizes certain relevant restriction fragments indicated. For easier identification certain bands, <br /> the slots occur given parentheses. (a) Hae III cleavage patterns OX174 RF DNA (slot 1), pMB9 DNA (slot 2), pOXG105 <br /> DNA (slot 3). (b) Hha I cleavage patterns OX174 RF DNA (slot 1), pMB9 DNA (slot 2), pOXG105 (slot 3), pOXG109 (slot 4). (c) Hha <br /> I Hpa II double-digestion patterns pMB9 pOXG105 DNA: pMB9 cleaved Hha I (slot 1), pOXG105 cleaved Hha I (slot <br /> 2), pOXG105 cleaved Hha I Hpa II (slot 3), pMB9 cleaved Hha I Hpa II (slot 4). <br />  infected am9 diluted plated lawns                  cycles infection lysis occur plaque becomes <br />  permissive nonpermissive test strains. Wild-type phage             visible. During period, probability recombination <br />  0.03% infected cells. Clearly recombination              plasmid DNA mutant viral DNA give <br />  itself explain plaques examined lawns             wild-type viral DNA high. Once occurs, wild-type phage <br /> of HF514/pOXG105 mixtures mutant wild-type                      produced rapidly burst size 30 times am9 <br /> phage.                                                                      plasmid-bearing cell. To test hypothesis, lowered <br />     An important clue plaque purity puzzle came                 recombination frequency p0XG am9 DNA by <br /> single cell, single burst experiments (25) carried according            transforming recombination-defective, nonpermissive host <br /> to Benbow et al. (7). In 200-tube experiment, average burst            (HF4740 recA13) pOXG105. A tetracycline-resistant <br /> size 10 obtained H514/p4XG105 infected                      transformant, permissive am9, obtained dif- <br /> with am9 compared 300 plasmid-                       ficulty. Two types plaques observed infection of <br /> bearing host infected wild-type phage. Thus,                  recA-, poXG-bearing host am9. Most plaques <br /> recombination small burst size involved             small regular edge. These plaques than <br /> production mixed plaques am9 grown H514/                      0.1% wild-type phage. About 2% plaques large with <br /> poXG105.                                                                   irregular edge. These plaques contained large of <br />   This suggested small burst size,                wild-type phage. Thus, reduction recombination fre- <br />                                                                            quency recA- mutation makes possible isolate <br />     Table 1. Plaque formation OX174 mutants E. coli                  plaques essentially pure am9. Why 2% plaques <br />                   containing pOXG plasmids                                 mixed clear us. <br />                                     Plaque formation                          These biological effects peculiar H514/p4XG since <br />                Location              H514/       H514/                  similar effects observed bacterial <br />    4X174        mutation       H514 poXG105 pOXG109                        strain, HF4704, transformed pOXGl05 poXGl09. <br />   Wild type                     +         +           + <br />   am9            GeneG          -         +           +                                            DISCUSSION <br />   am3            Gene E                                                    Gene G, essential gene bacteriophage kX174, required <br />   am86           GeneA          -         -           -                    synthesis single-stranded viral DNA RF. Its gene <br />  778       Biochemistry: Humayun Chambers                                               Proc. Natl. Acad. Sci. USA 75 (1978) <br />  <br /> product spike protein virion (3, 4). If wishes         initial isolation pMB9- DNA gift EcoRI enzyme; R. <br /> isolate propagate conditional unconditional lethal           Roberts A. Jeffrey restriction enzymes; W. Holloman X <br /> mutants arise covalent modification              exonuclease; P. Howard-Flanders, J. Hurwitz, R. Sinsheimer, R. <br /> DNA virulant phage fX174,             Warner bacterial phage strains; M. Schneider techrical <br /> that permissive base substitutions,          assistance; H. Annus photography. This work supported <br /> frameshifts large deletions. Gene-specific suppression             Grant 2 ROI CA 16319-03 BIO VO 2 (M77) National In- <br />                                                                           stitutes Health. <br /> complementation using plasmid vehicle de- <br /> scribed appear provide general system.                1. Miller, J. A. &amp; Miller, E. (1974) Chemical Carcinogenesis, eds. <br />    We anticipated number potential difficulties con-                  Ts&apos;o, P. &amp; Dipaulo, J. A. (Marcel Dekker, Inc., New York), Part <br /> structing functional strain carrying pOXG. For example,                 A, pp. 61-63. <br /> restriction fragment OX Hae III 1342 does contain known              2. Sanger, F., Air, G. M., Barrell, B. G., Brown, N. L., Coulson, A. <br /> promoter (2, 26, 27). Also, production viral spike protein              R., Fiddes, J. C., Hutchison, C. A., III, Slocombe, P. M. &amp; Smith, <br /> an uninfected cell lethal. None materialized,               M. (1977) Nature 265,687-695. <br /> but problem did arise. The burst size am9, gene C                 3. Sinsheimer, R. L. (1968) Progress Nucleic Acid Research <br /> nonsense mutant, pOXG-bearing host, H514,                      Molecular Biology, eds. Davidson, J. N. &amp; Cohn, W. E. <br /> 5-10% wild-type phage H514/pOXG. The small burst                   (Academic Press, New York), Vol. 8, pp. 115-169. <br /> size apparently known polar effect am9                4. Denhardt, D. (1975) Crit. Rev. Microbiol. 4,161-223. <br /> mutation gene H (28). Our recent experiments (un-                  5. Benbow, R. M., Hutchison, C. A., III, Fabricant, J. D. &amp; Sin- <br /> published) shown minicells carrying p4XG produce                    sheimer, R. L. (1971) J. Virol. 7,549-558. <br />                                                                            6. Vosberg, H. P. &amp; Hoffmann-Berling, H. (1971) J. Mol. Biol. 58, <br /> a large gene G spike protein. The gene H product,                   739-753. <br /> another spike protein, furnished infecting mu-              7. Benbow, R. M., Zuccarelli, A. J., Davis, G. C. &amp; Sinsheimer, R. <br /> tant phage. Because severe polarity am9 mutation,               L. (1974) J. Virol. 13, 898-907. <br /> only small H protein formed (28),              8. Rodriguez, R. L., Bolivar, F., Goodman, H. M., Boyer, H. W. &amp; <br /> this limits burst size. Further support comes               Betlach, M. (1976) Molecular Mechanisms Control of <br /> experiments (unpublished) transformed                     Gene Expression, eds. Nierlich, D. P., Rutter, W. J. &amp; Fox, C. F. <br /> amber suppressor strain nonpermissive am9                    (Academic Press, New York), Vol. V, pp. 471-477. <br /> pOXG. The transformant permissive am9,                9. Maniatis, T., Kee, S. G., Efstratiadis, A. &amp; Kafatos, F. C. (1976) <br /> but burst size increases 7-fold compared transfor-                 Cell 8, 163-182. <br />                                                                           10. Miller, J. H. (1972) Experiments Molecular Genetics (Cold <br /> mants described paper carry suppressor                  Spring Harbor Laboratory, Cold Spring Harbor, NY), pp. <br /> tRNA.                                                                         433-434. <br />    The small burst size combined recombination leads              11. Guthrie, G. D. &amp; Sinsheimer, R. L. (1960) J. Mol. Biol. 2,297- <br /> plaques containing mixture wild-type am9 phage. Pure                 305. <br /> am9 isolated plaques grown recombination-                12. Godson, G. N. &amp; Boyer, H. (1974) Virology 62,270-275. <br /> deficient, poXG-bearing host HF4740/pOXG                     13. Clewell, D. B. (1972) J. Bacteriol. 110,667-676. <br /> though burst size small. This generally useful        14. Humayun, Z., Jeffrey, A. &amp; Ptashne, M. (1977) J. Mol. Biol. 112, <br /> for proposed recA function                   265-277. <br /> required interesting, carcinogen-induced mutations               15. Davis, R. W., Simon, M. &amp; Davidson, N. (1971) Methods in <br /> (29, 30). This does appear propose              Enzymology, eds. Grossman, L. &amp; Moldave, K. (Academic Press, <br />                                                                               New York), Vol. XXI, pp. 413-418. <br /> produce mutants transfection spheroplasts site-              16. Jackson, D. A., Symons, R. H. &amp; Berg, P. (1972) Proc. Natl. Acad. <br /> specific, covalently modified DNA. Transfection spheroplasts               Sci. USA 69,2904-2909. <br /> derived cells resistant infection intact          17. Wensink, P. C., Finnegan, D. J., Donelson, J. E. &amp; Hogness, D. <br /> virus produces single burst virus DNA. Since                S. (1974) Cell 3,315-325. <br /> only 0.03% wild-type virus single cells H514/pOXG                 18. Little, J. W., Lehman, I. R. &amp; Kaiser, A. D. (1967) J. Biol. Chem. <br /> infected am9 plated, expect                242,672-678. <br /> a single burst transfected spheroplasts.                               19. Roychoudhury, R., Jay, E. &amp; Wu, R. (1976) Nucleic Acids Res. <br />    In terms developing general detecting                  3, 101-116. <br /> propagating type mu.tation gene G, polar effects                20. Bollum, F. J. (1969) J. Biol. Chem. 234,2733-2734 <br /> </body></html>