<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal vol.10 no.4 pp.1017- 1023, 1991 <br />  <br />  <br /> Identification sequence analysis gene <br /> mutated conditionally lethal outer membrane <br /> permeability mutant SS-C Salmonella typhimurium <br />  <br /> Laura Hirvas, Pertti Koski Martti Vaara                          lipid A essential LPS bacterial <br />                                                                      cell, enzymes involved lipid A biosynthesis <br /> Department Bacteriology Immunology, University Helsinki,   characterized molecular terms their <br /> Haartmaninkatu 3, 00290 Helsinki, Finland <br />                                                                      genes identified (Raetz, 1990; Raetz and <br /> Communicated B.A.D.Stocker                                        Dowhan, 1990). However, biosynthesis lipid <br />                                                                      A -KDO region LPS novel, attractive target <br /> The biosynthesis structure-function relationships                antibacterial therapy (Hogenauer Waisetschlager, <br /> of enterobacterial outer membrane subjects                1981; Allen, 1985; Goldman et al., 1987; Hammond <br /> current intensive research. We previously described             et al., 1987; Galloway Raetz, 1990; Raetz, 1990), <br /> the antibiotic supersensitive SS-C mutant (SH7622)                the research lipid A biosynthesis has <br /> Salmonella typhimurium shown outer                      increased considerably years. We also <br /> membrane permeability barrier hydrophobic                    lack detailed knowledge molecular biology of <br /> antibiotics severely defective. In study,            structure -function relationships relative <br /> that mutant heat-sensitive, conditionally lethal,        impermeability OM hydrophobic compounds <br /> carries missense <span id='am-240' about='xsp:base' typeof='owl:Thing'>base</span>-pair substitution novel gene            (Nikaido Vaara, 1987). Yet functions been <br /> which recently reported named ssc                implicated main functions OM, such <br /> gene. ssc encodes earlier uncharacterized 36 kd                   functions inhibited appropriately tailored <br /> protein (the Ssc protein) mutant expresses Ssc               selected antibacterial agents (Vaara Vaara, 1983a,b). <br /> which valine 291 changed methionine                         Mutants defective OM structure and/or function <br /> methionine-rich region Ssc. A plasmid containing              invaluable tools OM research very <br /> wild-type ssc allele completely reverts antibiotic-              beginning, elucidation biosynthetic pathways of <br /> and heat-sensitive <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span>phenotype</span> SS-C mutant. Cor-                OM components studies structure-function <br /> responding plasmids carrying mutant allele,                relationships (reviewed Makela Stocker, 1984; <br /> identical mutant allele prepared loalized mutagenesis,            Lehmann Hansen-Hagge, 1984; Nikaido Vaara, <br /> are inactive. The ssc gene probably analogous               1985; Sukupolvi Vaara, 1989). Novel mutants be <br /> firA locus Escherichia coli earlier                expected shed light current puzzles OM research. <br /> implicated totally different function, mRNA                     It relatively easy select, example, mutants <br /> synthesis. Furthermore, ssc apparently lies close            decreased OM permeability barrier. On the <br /> the Ipx genes involved far known steps lipid          hand, spite intensive research, those <br /> A biosynthesis (the distance, -560 bp). <span id='am-5' about='protege:TO' typeof='owl:Thing'>To</span> conclude,                 permeability mutants remained mysterious, basis <br /> our findings define new essential gene involved             <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span>mutation</span> determined DNA level. <br /> generation outer membrane.                                    Such mutants include abs, acrA, acrB, envC other <br /> Key words: Escherichia coli/lipopolysaccharide/outer                 mutants E.coli SS-B SS-C mutants of <br /> membrane permeability/Salmonella typhimuriumIssc                     S.typhimurium (reviewed Nikaido Vaara, 1985; <br /> mutation                                                             Sukupolvi Vaara, 1989). <br />                                                                          We described SS-C mutant S.typhimurium 1984. <br />                                                                      Its supersensitivity number unrelated hydrophobic <br />                                                                      antibiotics (such rifampicin, erythromycin, clindamycin, <br /> Introduction                                                          fusidic acid, vancomycin novobiocin) to <br /> The outer membrane (OM) Gram-negative enteric bacteria             hydrophobic drugs detergents indicated a <br /> is extensively studied comprehensively            defective permeability barrier. Unlike hyper- <br /> characterized biological membranes (for review,                 permeable OM mutants, SS-C mutant detectable <br /> Nikaido Vaara, 1987). The structure elaborate              alterations LPS. Neither obvious changes in <br /> lipopolysaccharide (LPS) constituent determined              phospholipids OM proteins. Furthermore, growth rate <br /> in Escherichia coli Salmonella typhimurium              cell division morphology unaltered. No <br /> model strains (Rietschel et al., 1984; Rick, 1987; Qureshi            molecular basis supersensitivity that <br /> et al., 1988) primary structures of OM            time (Sukupolvi et al., 1984, 1985). <br /> proteins E. coli and/or S. typhimurium known.                     In paper, serendipitously <br />    However, lack complete understanding                 discovered gene mutated SS-C mutant while <br /> the biosynthesis OM. The translocation process OM           studying gene, ompH, recently found, <br /> proteins synthesis site cytoplasmic mem-            cloned, sequenced (Koski et al., 1989, 1990). The <br /> brane elucidated (Bolla et al., 1988;      mutation causes Val - Met change previously <br /> Lill et al., 1989; Randall Hardy, 1989). Practically              uncharacterized 36 kd protein, named Ssc protein. <br /> nothing known export LPS outer surface         Furthermore, mutation conditionally <br /> of OM (review Raetz, 1990). Furthermore,                  lethal; mutant does grow 42&#194;&#176;C. The corresponding <br />                                                                                                                              101 7 <br />    Oxford University Press <br />  L.Hirvas, P.Koski M.Vaara <br />  <br /> gene, ssc, probably analogous firA gene E. coli           recently found, cloned, sequenced (Koski et <br /> which long linked cellular function quite                al., 1989, 1990). Accordingly, felt important to <br /> different related OM, i.e. transcription          investigate possible relationship ompH gene <br /> (Lathe et al., 1980), firA mutants E. coli rpoB,            SS-C mutation. We employed plasmid pUCHS16 <br /> which carry rifampicin-resistant RNA polymerase, rifam-            (Figure 1) carries, ligated vector pUC 19, the <br /> picin sensitive. Furthermore, ssc gene Ipx             entire ompH gene 2.3 kb genomic BamHI-SalI <br /> genes involved known steps lipid A biosynthesis              fragment S.typhimurium SH5014. The fragment also <br /> are separated open reading frame.                          contains open reading frame, ORF-36 (Hirvas et al., <br />                                                                        1990b). It encodes far uncharacterized 36 kd protein. <br /> Results                                                                No complete ORFs present fragment. <br />                                                                           When pUCHS 16 introduced SH7622, the <br /> The antibiotic supersensitive thermosensitive                      antibiotic sensitive phenotype SH7622 completely reverted <br /> phenotype SS-C mutant                                           identical parent (Table II). The ability <br /> As reported earlier (Sukupolvi et al., 1984), SH7622             grow 42&#194;&#176;C resumed. Separate controls <br /> (the SS-C mutant) remarkably sensitive hydrophobic               indicated plasmid vector (pUC 19) did itself <br /> antibiotics. This especially evident sensitivity          effect phenotype SH7622 (Table II). <br /> was measured determining minimum inhibitory                     Furthermore, pUCHS 16 did revert phenotype an <br /> concentrations (MICs) antibiotics (Table I). The MICs           unrelated supersensitive S. typhimurium mutant SH7616 <br /> of hydrophobic drugs tested 16- 256-fold              mutation SS-B. <br /> lower SH7622 parent SH5014. The observed                Further studies, utilizing pUCHS16 derivatives <br /> MICs low comparable corresponding                  pUCHS16A (lacks ORF-36, Figure 1) pUCHS 14-TI <br /> MICs enterobacterial OM permeability mutants             (lacks 5&apos; terminus ompH, Figure 1), indicated that <br /> such heptoseless LPS mutants (Vaara Vaara,                  plasmid carrying ORF-36 required complement the <br /> 1983b) SS-B mutant (Vaara, 1990).                              phenotype SH7622 presence ompH the <br />    Furthermore, temperature             plasmid essential (Table III). Accordingly, the <br /> susceptibility tests performed greatly affects degree          ORF-36 gene tentatively named ssc gene. <br /> of sensitivity SH7622. Thus, drugs MICs <br /> significantly lower (4- 16-fold) 39&#194;&#176;C 33&#194;&#176;C               Complementation ssc mutant alleles <br /> (Table I); 36&#194;&#176;C sensitivity intermediate (data          As shown above, pUCHS14-ll, carrying wild-type ssc <br /> shown). This temperature-induced susceptibility increase               gene S.typhimurium SH5014 1.6 kb PstI-PstI <br /> is common property permeability mutants                   fragment, effective complementation the <br /> suggested SH7622 possibly regarded                  phenotype SS-C mutant SH7622. When cor- <br /> temperature-sensitive mutant. Further studies indicated           responding 1.6 kb PstI-PstI fragment isolated <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> SH7622 conditionally lethal mutant. It does grow              SH7622, ligated pUC19 (to constitute plasmid <br /> at 42&#194;&#176;C. However, growth rate 370C indiscernible             pUCHS214, Figure 1), introduced SH7622, no <br /> from parent strain.                                        complementation SS-C phenotype observed (Table <br />                                                                        III). Accordingly, ssc gene SH7622 designated <br /> Complementation SS-C phenotype                                  sscl mutant allele. <br /> We shown earlier SS-C mutation lies between <br /> map positions 5 7 S.typhimurium chromosome                                               P             Q.                             IL <br /> (Sukupolvi et al., 1984). Alignment studies (Riley                                                                                           c: <br />                                                                                                04 <br />                                                                                                1        cI        0 <br />                                                                                                                        (DO4 <br />                                                                                                                        a) X        1C0tn) <br />                                                                                                                                    Cl            0 <br /> Krawiec, 1987) indicate 5&apos; region S. typhi-                                              I,       o-                        I) <br /> murium genome corresponds 4&apos; region E. coli                                                                     , <br />                                                                                                                                                    0( <br /> genome. The E. coli genes 4&apos; include hlpA gene (see                                     Cl 0lCOl <br />                                                                          pUCHS16 <br /> Bachmann, 1990). As shown (Hirvas et al., 1990a), <br />                                                                                                                               cc <br /> it analogue ompH gene S. typhimurium                                                   g                             Q.        0~ <br />  <br />                                                                         pUCHS16A <br /> Table I. The antibiotic supersensitive phenotype SS-C mutant                        B             P                E <br /> SH7622 <br /> Antibiotic     MIC (/ig/ml)                           Ratioa            pUCHS1 4-Il   IJ <br />                                                                         pUCHS14 <br />                SH5014            SH7622     SH7622    33&#194;&#176;C 39&#194;&#176;C         pUCHS214                         P                E                 p <br />                33&#194;&#176;CC 39&apos;C    33&apos;C       39&#194;&#176;C <br />                                                                        Fig. 1. Genomic fragments S.typhimurium pUCHS16 related <br /> Fusidic acid    2 512            16          2        -32      -256    plasmids. pUCHS16 carries 2.3 kb BamHI-SalI fragment of <br /> Rifampicin          8             0.5        0.06     2 16     2 135   S.typhimurium SH5014 genome described previously <br /> Erythromycin       64             4          1          16        64   (Koski et al., 1990; Hirvas et al., 1990a,b). This fragment contains <br />                                                                        putative promoter region ompH (-35 region starting nt 142), <br /> Clindamycin        64            32          2           2        32   structural ompH gene (nt 311 -796), structural ssc (= ORF-36) <br /> Vancomycin        256            64         16           4        16   gene (nt 800-1825) 5&apos; terminus open reading frame <br />                                                                        ORF-II (from nt 1931). The nucleotide positions relevant restriction <br /> The values minimum inhibitory concentrations (MICs) various     sites indicated. pUCHS16A pUCHS14-II derivatives of <br /> hydrophobic antibiotics SH7622 parent strain SH5014     pUCHS16. pUCHS14 contains independently isolated 1.6 kb <br /> 33&#194;&#176;C 39&#194;&#176;C, determined L broth.                               PstI-PstI genomic fragment SH5014 (Koski et al., 1989). <br /> aThe approximate ratio MIC SH5014         pUCHS214 contains corresponding 1.6 kb fragment SS-C <br /> SH7622.                                                                mutant strain SH7622. <br /> 1018 <br />                                                                                           Lethal outer membrane mutant allele novel    ssc gene <br />  <br />  <br />  <br />    To surprise, spontaneous ssc mutant                           pUCHS 14, did not. The latter, however, produced new <br /> allele (ssc5l). This discovered recognized                            protein having apparent molecular mass 14 kd. <br /> the plasmid pUCHS 14 (Koski et al., 1989; Figure <br /> 1), carrying independently isolated (not pUCHS16-                                 Nucleotide sequences mutant ssc genes <br /> derived) 1.6 kb PstI-PstI fragment S.typhimurium                                We sequenced SH7622-derived, ompH- sscl- <br /> SH5014, did complement SS-C mutant phenotype                                 containing 1496 bp region pUCHS214 using the <br /> (Table III). This unexpected result                             sequencing strategy primers identical used <br /> be contradictory obtained pUCHS 16-derived                          sequencing wild-type ompH ssc genes of <br /> pUCHS14-ll plasmid. However, results shown                                     S. typhimurium SH5014. Sequencing revealed mutation, <br /> indicate ssc gene pUCHS14 mutation,                            G - A transition, sscl gene (codon 291, position <br /> this explains discrepancy.                                                       1670 Figure 3). This results change amino <br />                                                                                      acid sequence valine methionine. Such missense <br /> Expression ssc gene mutant alleles                                    mutation unexpected, ssc sscl <br /> We previously shown ORF-36 gene (the ssc                               <span id='am-4' about='Thesaurus:code' typeof='owl:Thing'>code</span> proteins mol. wt. No changes were <br /> gene) expressed minicells encodes protein                           observed calculated hydrophobicity secondary <br /> an apparent molecular mass 36 kd (Hirvas et al., 1990b).                          structure Ssc SscMet291 <br /> This corresponds molecular mass 35 929                                   Sequencing ssc5l-containing region pUCHS14 <br /> daltons, calculated ssc gene product. Furthermore,                           revealed <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span>frameshift mutation</span>, deletion adenine codon <br /> and expected basis deduced amino acid se-                           107 (position 1119 Figure 3). It resulted formation <br /> quence data, detected preprotein forms                            TGA stop codon 17 codons downstream. The calculated <br /> the Ssc protein.                                                                     molecular mass truncated Ssc protein encoded ssc5l <br />    In Figure 2, expression ssc mutant alleles                        13 397 daltons (37% wild-type Ssc) and <br /> been compared wild-type ssc. The mutant                             corresponds apparent molecular mass determined <br /> allele sscl, present SS-C mutant SH7622, did encode                           using minicells described above. <br /> a 36 kd protein, mutant allele ssc5l, present <br />                                                                                      Reconstruction ssc- 1 allele site-directed <br />                                                                                      mutagenesis <br /> Table II. The effect plasmid pUCHS16 antibiotic <br />                                                                                      To verify single change Val291 Met the <br /> sensitive phenotype SS-C mutant SH7622                                        Ssc protein responsible phenotype sscl <br />                                                                                      mutant SH7622, reconstructed G       -A transition by <br /> Antibiotic             MIC (pug/ml)                                                  mutagenizing wild-type ssc gene pUCHS16 a <br />                        SH7622                SH7622                  Ratioa          17-mer mutagenic oligonucleotide bears transition <br />                        pUCHS16          pUC19                              (see Materials methods). Sequencing mutagenized <br />                         -512                   4                      2 128          ssc revealed desired mutation (sscl) achieved. <br /> Fusidic acid <br /> Rifampicin               - 16                  0.06                   2270           We vitro-prepared sscl sscl&apos;. When <br /> Erythromycin               64                  1                           64        sscl&apos;-carrying plasmid pUCHS14II-1 introduced in <br /> Clindamycin                64                  4                           16        SH7622, complementation sscl phenotype was <br /> Vancomycin               256                  16                           16        accomplished (data presented connection other <br />                                                                                      complementation data, Table III). This finding confirms <br /> The values MICs hydrophobic antibiotics SH7622 carrying                   sequence data. In vitro-prepared missense mutations to <br /> pUCHS16 control plasmid pUC19, determined 39&#194;&#176;C L                     locations ssc pUCHS16 did reduce <br /> broth containing ampicillin (100 yg/ml). <br /> aThe approximate ratio MIC SH7622/pUCHS16                        ability complement sscl mutation (data be <br /> that SH7622/pUC19.                                                               presented elsewhere). <br />  <br /> Table III. Complementation antibiotic- heat-sensitive phenotype SS-C mutant SH7622 various plasmids containing ompH <br /> and/or 3&apos; flanking open reading frame (ORF-36) <br />  Strain      Plasmid        Brief description insert&apos;     Sensitivity antibioticsb                                 Heat sensitivity <br />                             Complete chromosomal Gene origin Fusidic acid Rifampicin Erythromycin Clindamycin Vancomycin Loglo reduction <br />                             genes present                                                                                c.f.u.s 42OCC <br />  <br />  SH7622      pUC19                               -            23           26          19          18           6           &gt;5 <br />  SH7622      pUCHS16        ompH, ORF-36           SH5014   2                   11           6           0           0           &lt;0.3 <br />  SH7622      pUCHS16A       ompH                   pUCHS16 23                   24          20          12          10           &gt;5 <br />  SH7622      pUCHS14-II     ORF-36                 pUCHS16 2                     9           5           0           0           &lt;0.3 <br />  SH7622      pUCHS14        ORF-36                 SH5014  26                   33          21          19          10             4.2 <br />  SH7622      pUCHS214       ORF-36                 SH7622  21                   26          17          14           7           &gt;5 <br />  SH7622      pUCHS14-II-1   ORF-36                 d                19          24          18          11           8           &gt;5 <br />  SH5014                -                       -                4          11           8           0           0           &lt;0.3 <br />  <br />  aSee Figure    1 Materials methods      detailed data. <br />  bMeasured diameter (in millimeters) inhibition zone antibiotic disc. The diameter disc itself subtracted. The <br />  tests performed 39&#194;&#176;C L agar containing ampicillin (100 Ag/ml; SH5014 ampicillin omitted). <br />  cCalculated logl0 (c.f.u.s 36&#194;&#176;C/c.f.u.s 42&#194;&#176;C). c.f.u., colony formingA <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span>. <br />                                                                                   transition nucleotide codon 291 ORF-36. Its <br />  dpUCHS14-I1-1   obtained   site-directedmutagenesis carries G      - <br />  <br />  <br />  properties results obtained described later section Results. <br />                                                                                                                                                 1019 <br />   L.Hirvas, P.Koski M.Vaara <br />  <br />  <br />       kDa <br />                                                                          pUCHS16 produce Ssc major protein (P.Koski, <br />                                                                          L.Hirvas M.Vaara, unpublished). Ssc overproducer <br />                                                                          strains obtained placing ssc control <br />                                                                          strong, regulated promoter. Such constructions are, <br />        36                                                                however, necessarily stable. As shown Figure 3, the <br />        32 <br />                                                                          ssc   gene pUCHS 14, plasmids used the <br />        30-                                                                present study, base deletion resulted the <br />                                                                           formation truncated Ssc protein. The reason the <br />                                                                           spontaneous appearance inactive plasmid ssc not <br />                                                                           studied. <br />                                                                             The ssc gene located immediately downstream ompH, <br />                                                                          gene encoding new OM protein recently described by <br />                                                                          (Koski et al., 1989, 1990). There three <br />                                                                          nucleotides stop codon ompH the <br />                                                                          initiation codon ssc. Our results present paper <br />                                                             _ 14         indicate ssc expressed absence ompH <br />                                                                          promoter, i.e. plasmid pUCHS14-ll lacks 5&apos; <br />                  1           2                       2 <br />                                                                          terminus ompH. Furthermore, orientation <br />                                                                          ssc pUCHS14-II (see Materials methods), its <br />                                                                          expression promoter plasmid vector <br />                       A                        B                         either. These results strongly suggest ssc promoter <br />                                                                          own. A putative promoter region found <br />  Fig. 2. Proteins synthesized Ecoli minicells harboring pUCHS14-II     159 nucleotides upstream, overlapping protein- <br />  (panel A, lane 1), control plasmid pUC19 (panel A, lane 2),         encoding region ompH (Hirvas et al., 1990b). <br />  pUCHS214 (panel B, lane 1) pUCHS14 (panel B, lane 2). Both             The gene organization ompH-ssc region S. typhi- <br />  pUCHS14-II pUCHS214 encode 36 kd protein                  murium corresponding region E. coli are <br />  pUCHS14 encodes 14 kd truncated protein. <br />                                                                          shown Figure 4. ssc probably analogous E. coli <br /> Discussion                                                               genefirA, sequence published. <br />                                                                          According recent data (see Thome et al., 1990; <br />  As reviewed Introduction, current OM research              Bachmann, 1990) contrast previous reports <br />  greatly benefit discovery novel mutants                (Aasland et al., 1988; Kroger et al., 1990), firA <br />  characterized mutations affecting function OM.               gene distinct hlpA (also named skp) does not <br />  We shown mysterious SS-C mutant (strain              <span id='am-241' about='Thesaurus:code' typeof='owl:Thing'>code</span> 17 kd HlpI (Skp) protein E. coli <br />  SH7622) S. typhimurium severe OM                         analogue OmpH protein S.typhimurium (Hirvas <br />  permeability barrier defect, carries missense base-pair               et al., 1990a). firA known located hipA <br />  substitution mutation novel gene recently            open reading frame encoding 17.4 kd protein <br />  sequenced encode previously uncharacterized              (ORF-17.4) (see Bachmann, 1990). The distance between <br />  36 kd protein. On basis, suggest gene            hlpA ORF-17.4 estimated 1.1 kb (Aasland <br /> be called ssc, product, Ssc protein.                         et al., 1988; Hirvas et al., 1990a) the <br />     The mutated Ssc protein Val291 changed                        analogous S. typhimurium genes ompH ORF-II is <br /> methionine. The hydropathy indices methionine valine              1125 bp (Hirvas et al., 1990b). Ninety-one cent <br /> are reasonably close (Kyte Doolittle, 1982),           (1020 bp) space S. typhimurium occupied the <br /> as a-helix, fl-sheet coil conformational             ssc structural gene. Because hlpA ompH 87% <br /> potentials helix-boundary f-sheet-boundary               homologous (Hirvas et al., 1990a) ORF-17.4 <br /> forming potentials (Chou Fasman, 1978). Consequently,               ORF-II 90% homologous (Hirvas et al., 1990b), <br /> the calculated hydropathicity profiles secondary                    expected firA ssc genes be <br /> structures Ssc SscMet291 similar. On basis              considerably homologous, too. Indeed, sequence a <br /> of data, tempting speculate Ssc            169 bp fragment immediately downstream E. coli hlpA is <br /> an enzymatic function Val291 impor-                 known (Holck Kleppe, 1988). It 86% homologous <br /> tant catalytic activity. Interestingly, region span-        corresponding 5&apos; terminus structural ssc gene <br /> ning residues 288-292 unusually rich                     (Hirvas et al., 1990b). <br /> methionine contains methionine Ssc                   firA mutants described Lathe et al. (1980). They <br /> in SscMea9l. We currently modifying region             selected rifampicin-sensitive phenotypic revertants <br /> as regions Ssc localized mutagenesis order               (hencefir versus rif) E. coli rpoB strain carries <br /> to gain information amino acid residues essential            rifampicin-resistant RNA polymerase. While E. coli rpoB is <br /> for function Ssc. Similarly, analysis set            resistant rifampicin target mutation (MIC <br /> spontaneous temperature-resistant revertants SH7622               - <br />                                                                             300 ytg/ml), MIC rifampicin rpoB, firA <br /> be expected valuable data functionally                strain 3-fold lower (Lathe et al., 1980). Furthermore, <br />                                                                                     - <br />  <br />  <br />  <br /> important domains Ssc, possible bypass                 thefirA, rpoB strain heat-sensitive RNA synthesis <br /> mutations, well. This analysis started               halted 42&#194;&#176;C. Based preliminary evidence, <br /> laboratory.                                                             firA mutation thought affect RNA transcription <br />    Further work involves purification Ssc protein.          (Lathe et al., 1980). Whether sensitization firA <br /> JM109 derivatives carrying ssc gene pUCHS 14-I1               mutant rifampicin decreased OM permeability <br /> 1020 <br />                                                                                     Lethal outer membrane mutant allele novel ssc gene <br />  <br />                                        782   AAACAGGTTAAATAAGTAATGCCTTCAATTCGACTGGCTGACTTAGCAGAACAGTTGGAT <br />                                              LysGlnValLys***       MetProSerIleArgLeuAlaAspLeuAlaGluGlnLeuAsp <br />                                        842   GCAGAATTACACGGTGATGGCGATATCGTCATCACCGGCGTTGCGTCCATGCAATCTGCA <br />                                              AlaGluLeuHisGlyAspGlyAspIleValIleThrGlyValAlaSerMetGlnSerAla <br />                                        902   ACAACAGGCCACATTACGTTTATGGTGAATCCTAAGTACCGTGAACACTTAGGTTTATGC <br />                                              ThrThrGlyHisIleThrPheMetValAsnProLysTyrArgGluHisLeuGlyLeuCys <br />                                        962   CAGGCTTCTGCGGTTGTCATGACGCAGGACGATCTTCCTTTTGCTAAGAGTGCGGCGCTG <br />                                              GlnAlaSerAlaValValMetThrGlnAspAspLeuProPheAlaLysSerAlaAlaLeu <br />                                       1022   GTAGTTAAAAATCCCTACCTGACCTACGCGCGCATGGCGCAAATTTTAGATACTACGCCG <br />                                              ValValLysAsnProTyrLeuThrTyrAlaArgMetAlaGlnIleLeuAspThrThrPro <br />                              ssc5l                   .                            - <br />                                       1082   CAGCCCGCGCAGAATATCGCGCCAAGCGCCGTGATTGATGCGACGGCAACGCTGGGTAGC <br />                                              GlnProAlaGlnAsnIleAlaProSerAlaValIleAspAlaThrAlaThrLeuGlySer <br />                              Ssc51                                                        ValArgArgGlnArgTrpValAl <br />                                       1142   AATGTTTCAGTCGGCGCGAATGCGGTGATTGAATCTGGCGTACAACTGGGCGATAACGTG <br />                                              AsnValSerValGlyAlaAsnAlaValIleGluSerGlyValGlnLeuGlyAspAsnVal <br />                                              aMetPheGlnSerAlaArgMetArg*** <br />                                       1202   GTTATCGGCGCAGGCTGTTTCGTCGGAAAAAATAGCAAAATCGGGGCGGGTTCACGCTTG <br />                                              ValIleGlyAlaGlyCysPheValGlyLysAsnSerLysIleGlyAlaGlySerArgLeu <br />                                      1262    TGGGCGAACGTAACGATTTACCACGACATTCAGATCGGTGAGAATTGCCTGATCCAGTCC <br />                                              TrpAlaAsnValThrIleTyrHisAspIleGlnIleGlyGluAsnCysLeuIleGlnSer <br />                                      1322    AGTACGGTGATCGGCGCGGACGGTTTTGGCTACGCTAACGATCGTGGCAACTGGGTGAAG <br />                                              SerThrValIleGlyAlaAspGlyPheGlyTyrAlaAsnAspArgGlyAsnTrpvalLys <br />                                      1382    ATCCCACAACTGGGCCGGGTCATTATTGGCGATCGTGTCGAGATCGGCGCTTGTACCACC <br />                                              IleProGlnLeuGlyArgValIleIleGlyAspArgValGluIleGlyAlaCysThrThr <br />                                      1442    ATTGACCGTGGCGCGTTGGATGATACTGTTATTGGCAATGGCGTGATTATTGATAATCAG. <br />                                              IleAspArgGlyAlaLeuAspAspThrValIleGlyAsnGlyValIleIleAspAsnGln <br />                                      1502    TGCCAGATTGCACATAACGTCGTGATTGGCGACAATACGGCAGTTGCCGGTGGCGTCATT <br />                                              CysGlnIleAlaHisAsnValValIleGlyAspAsnThrAlaValAlaGlyGlyvalIle <br />                                      1562    ATGGCGGGTAGCCTGAAGATTGGCCGTTACTGCATGATTGGCGGCGCCAGCGTGATCAAT <br />                                              MetAlaGlySerLeuLysIleGlyArgTyrCysMetIleGlyGlyAlaSerValIleAsn <br />                             sscl                                                                       A <br />                                      1622    GGGCATATGGAAATATGCGACAAAGTCACGGTAACTGGCATGGGTATGGTGATGCGTCCC <br />                                              GlyHisMetGluIleCysAspLysValThrValThrGlyMetGlyMetValMetArgPro <br />                            Sscl                                                                        Met <br />  <br />                                      1682    ATCACGGAACCGGGCGTCTACTCCTCAGGCATTCCGCTGCAACCCAACAAAGTATGGCGT <br />                                              IleThrGluProGlyValTyrSerSerGlyIleProLeuGlnProAsnLysValTrpArg <br />                                      1742    AAAACTGCTGCACTGGTGATGAACATTGATGATATGAGCAAGCGTCTCAAAGCGATTGAG <br />                                              LysThrAlaAlaLeuValMetAsnIleAspAspMetSerLysArgLeuLysAlaIleGlu <br />                                      1802    CGCAAGGTTAATCAACAAGACTAACGTTCCGCCTTGTAGTTGCCATTCTTTTCCGGCCTG <br />                                              ArgLysValAsnGlnGlnAsp*** <br />  <br /> Fig. 3. Nucleotide sequences ssc gene (Hirvas et al., 1990b) sscl ssc5l mutant alleles. The mutated nucleotides ssc5l sscl <br /> have indicated wild-type nucleotide sequence. Also shown deduced amino acid sequences Ssc protein mutant <br /> proteins. The putative Shine-Dalgarno sequence initiation codon underlined. The numbering consistent gene map <br /> shown Figure 1. The sequence data presented available GenBank database accession number M35193. <br />  <br /> barrier function studied. Furthermore,                       Raetz, 1981) lpxA discovered course <br /> sensitivities firA mutant hydrophobic                        sequencing lpxB (Crowell et al., 1986). However, other <br /> antibiotics published. Whether ssc firA                    enzymes involved lipid A synthesis been <br /> genes analogous functions, remains studied. Even                  purified, genes mapped or <br /> if have, probable, phenotypes sscl allele              sequenced. These include UDP-3-monoacyl-N-glucosamine <br /> and firA, rpoB + alleles need necessarily identical                   acyltransferase (Raetz Dowhan, 1990), lipid A 4&apos; kinase <br /> because mutations differ extent                        (Ray Raetz, 1987) KDO2-lipidIVA lauroyl- <br /> functional lesion produced.                                                  myristoyltransferases (Brozek Raetz, 1990). The <br />    Figure 4 shows ssc located close                 lack mutants conditionally defective those <br /> Ipx cluster. lpxA lpxB genes encode UDP-N-acetyl-                        enzymes greatly hampered research. <br /> glucosamine acetyltransferase lipid A disaccharide                          Our previous studies did reveal detectable <br /> synthase, respectively; enzymes lipid A                    alterations LPS sscl mutant (Sukupolvi et al., <br /> biosynthesis (Crowell et al., 1987; Coleman Raetz,                       1985). However, time, did know the <br /> 1988). The distance ssc lpxA - 560 bp.                   mutant thermosensitive, studied LPS <br /> This clustering intimate ssc lipid A                 produced 37&#194;&#176;C. Even OM sscl mutant <br /> synthesis machinery. The lipid A -KDO                       obviously defective 37&#194;&#176;C evidenced the <br /> essential LPS bacterial viability growth. Its                mutant&apos;s sensitivity antibiotics 37&#194;&#176;C 33&#194;&#176;C <br /> biosynthesis systematically extensively studied,                (Table I), relative proportion defective LPS molecules <br /> mainly Raetz collaborators, considerable break-                   abnormal precursors (if exist) larger <br /> throughs years. The lpxA                  42&apos;C, lethal temperature, 37&#194;&#176;C. Accordingly, <br /> and lpxB genes products (see above)                      currently studying lipid A -KDO precursors <br /> characterized. lpxB discovered aid E. coli                   synthesized sscl mutant temperature shift <br /> mutants accumulated novel phospholipids (Nishijima                     42&#194;&#176;C. Furthermore, enzymes <br />                                                                                                                                              1021 <br />   L.Hirvas, P.Koski M.Vaara <br />  <br />                                                                                           BamHI-EcoRV ligating ompH-containing 863 bp fragment to <br />                                                                                           BamHI -Hincll digested pUC 19. It contains entire ompH gene only <br />                                           I&apos;                                              22 codons ssc. <br />                          U). UU) <br />                                                                                              pUCHS14-II lacks ompH (the promoter Shine-Dalgarno <br />                 i.       8.           . <br />                                                                  S1 <br />                                                                  0.                lu <br />                                                                                    i&amp;a    regions seven codons) contains entire ssc gene. <br />    A                 I                    ,    1. <br />  <br />  <br />  <br />                                I      - <br />                                                                                           It prepared digesting pUCHS16 PstI ligating released <br />                                                                                           1.6 kb PstI fragment PstI-opened pUCHS19. A hybrid plasmid which <br />                                                                                           contained Salmonella-derived DNA orientation in <br />                               topA                       firA    ORF <br />                                                                  17.4 <br />                                                                             IpxA          pUCHS16 (i.e. produced 0.55 kb fragment BamHI-EcoRV diges- <br />                                                                                           tion) selected. <br />                                                                                              pUCHS14 (Koski et al., 1989) carries independently cloned 1.6 kb <br />                                                                                           PstI fragment S.typhimurium SH5014 chromosome, ligated pUC18. <br />                                                                                           This fragment corresponds 1.6 kb PstI fragment present pUCHS16 <br />                                itSI           a,WI <br />                                               ccili             I 0, <br />    B <br />                      ,         0. <br />  <br />  <br />                                I <br />                                                 I   1. <br />                                                     I <br />                                                                 IIF <br />                                                                  . <br />                                                                         S <br />                                                                       (I) <br />                                                                                           derivative pUCHS 14-I1 differs having lac <br />                                                                                           promoter proximal ssc gene orientation ssc. <br />                                                                                              Plasmids pUCHS214 pUCHS14-II-l described below. <br />                                II <br />                                .npH <br />                               Ci <br />                                Ii <br />                                                     I <br />                                                     I ssc <br />                                                     I <br />                                                     i3 <br />                                                                 IORF <br />                                                                 ,II911 <br />                                                                                           Cloning ssc gene SH7622 <br />                                                                                           DNA isolated S. typhimurium SH7622 exactly method we <br />                                                                               0.5 kb      employed earlier (Koski et al., 1989). It digested PstI, run on <br />                                                     1 <br />                                1                                                          0.7% agarose gel, analysed Southern hybridization using an <br />                                I                    I <br />  <br />                                                                                           ompH specific oligonucleotide probe (probe I Koski et al., 1989). The <br />                                                   I <br />                                                   I <br />                                                                                           1.6 kb fragment, detected probe, ligated pUC19, the <br />                                                     I <br />    C                           I 87%                1.   ND      90%                      orientation pUCHS16 pUCHS14-II (see above), named <br />                                                                                           pUCHS214. <br />                                                                                          Site-directed mutagenesis <br />  Fig. 4. (A, B) Alignment hlpA-firA-ORF-17.4-lpxA region                       To produce site-directed mutagenesis ssc allele bears a <br />  Ecoli (panel A) corresponding region Styphimurium. The                      G - A transition identical sscl, mutagenic oligonucleotide <br />  sequence firA published. The conserved restriction                      5&apos;-GACGCATCATCATACCC-3&apos; prepared BamHI-SalI <br />  sites indicated asterisk (*). (C) Nucleotide identity               fragment pUCHS16 cloned corresponding sites M13mp8. <br />  gene areas shown panels (A) (B). ND, determined.                         Site-directed mutagenesis performed method Nakamaye and <br />                                                                                          Eckstein (1986) using oligonucleotide mutagenesis kit version 2 of <br /> involved lipid A biosynthesis lack characterization,                            Amersham conditions recommended manufacturer. The <br /> as reviewed above, enzymatic activity suitably                                  presence desired mutation verified sequencing primer <br /> prepared cell fractions E.coli assayed using                                5&apos;-CAATGTTCATCACCAGT-3&apos;. The 1.4 kb PstI fragment mutant <br /> appropriate lipid A precursors substrates (Raetz, 1990).                              M13 DNA ligated PstI-opened pUC19, orientation <br />                                                                                          pUC19 recombinant plasmids described above. The resultant <br /> Consequently, sscl mutant lacks activities,                           plasmid named pUCHS14-II-1. <br /> that lack detected assays. <br />    Thus, mechanism defective Ssc protein                                Antibiotic susceptibility determinations <br /> mediates OM permeability increase growth-permissive                               The MICs antibiotics measured L broth containing (per litre): <br /> temperature growth inhibition 42&#194;&#176;C, remains                                   tryptone (Difco Laboratories, Detroit, MI), 10 g; yeast extract (Oxoid, <br />                                                                                          Columbia, MD), 5 g; NaCl, 5 g (pH 7.2). The inoculum L agar-grown <br /> to elucidated. This elucidation certainly yield                                  cells, inoculum size 104 cells/nil. The incubation 18 h <br /> considerable new data function OM                                   37&#194;&#176;C. The lowest concentration completely inhibited visual growth <br /> might form useful basis future development                                  recorded interpreted MIC. The following antibiotics were <br /> pharmacological inhibitors OM biosynthesis.                                           used: rifampicin fusidic acid (Na), (Sigma Chemical Co., St Louis, <br />                                                                                          MO); erythromycin ethylsuccinate, Orion, Helsinki, Finland; clindamycin <br />                                                                                          hydrochloride, Upjohn, Kalamazoo, MI; vancomycin hydrochloride, <br />  Materials methods                                                                   Eli Lilly Co., Indianapolis. IN. <br />                                                                                             When sensitivity antibiotics measured agar diffusion <br /> Bacterial strains                                                                        method, antibiotic discs (Neo-Sensitabs) A/S Rosco (Copenhagen, <br /> S.typhimurium LT2 strain SH5014 (ilv.1178, thr914, his6116, metA22,                      Denmark), L agar containing ampicillin (100 fg/ml) stated, <br /> metE551, trpB2, xy1404, Hi-b, H2-e,nx, flaA66, rpsL120, rfaJ4041 causing                 Kirby-Bayer methodology (as described disc manufacturer) <br /> LPS chemotype Rb2) typical well-characterized representative                 used. The discs rifampicin (30 pg), fusidic acid (400 Mg), <br /> rough S.typhimurium (Stocker et al., 1979). SH7622 antibiotic                     erythromycin (78 JLg), clindamycin (25 M,g) vancomycin (70 Mg). <br /> supersusceptible class SS-C mutant (Sukupolvi et al., 1984). The antibiotic <br /> supersusceptible phenotype SH7622 repeatedly rechecked using                   Testing heat sensitivity <br /> antibiotic discs agar diffusion method (see below). SH7616                 Serial 10-fold dilutions bacterial suspensions (109 cells/ml) made <br /> antibiotic supersusceptible class SS-B mutant SH5014 (Sukupolvi et al.,               0.9% NaCl 50 f1 dilution plated L agar plates. After <br /> 1984; Vaara, 1990). The bacterial strains preserved -70&#194;&#176;C 20%                 15 h incubation, colonies (c.f.u.s) developed counted. The <br /> skimmed milk.                                                                            reduction viability 42&#194;&#176;C expressed logl0 ratio <br />   The primary host plasmids Ecoli JM109. When plasmids                       colony count 36&#194;&#176;C 42&#194;&#176;C. <br /> were introduced S.typhimurium, restriction mutant LB5010 of <br /> S.typhimurium (Bullas Ryu, 1983) used intermediate host                    Expression minicells <br /> for plasmids.                                                                      The plasmid-encoded proteins studied using Ecoli PK251 minicells <br />                                                                                          described earlier (Koski et al., 1990). Briefly, minicells isolated <br />  Plasmids                                                                                method Dougan Kehoe (1984), labeled [35S]methionine <br />  pUCHS16 (Koski et al., 1989; Figure 1) carries cloned 2.3 kb               (Amersham; specific activity, 1 Ci/mol; final concentration, 150 ftCi/ml) <br /> BamHI-SalI fragment S.typhimurium SH5014 chromosome, ligated                       boiled 3.5% SDS, 10% mercaptoethanol. SDS-PAGE (15% <br /> to cloning vector pUC19 (Pharmacia, Uppsala, Sweden), contains                   polyacrylamide) followed soaking AmplifyTM (Amersham) before <br /> the entire ompH ssc genes, 506 bp downstream region.                    drying autoradiography. <br /> The complete nucleotide sequence pUCHS16 determined (Koski <br /> et al., 1989, 1990; Hirvas et al., 1990b). In plasmid, lac promoter             DNA sequencing <br /> of vector distal ompH ssc genes opposite orien-             Sequencing plasmid DNA denatured plasmids (Hattori <br /> tation. pUCHS16A (Figure 1) prepared digesting pUCHS16                       Sakaki, 1986) dideoxyribonucleotide chain termination method <br />  <br /> 1022 <br />                                                                                          Lethal outer membrane mutant allele novel ssc gene <br />  <br /> (Sanger et al., 1977) SequenaseTM (United States Biochemical,                 Mizusawa,S., Nishimura,S. Freela,F. (1986) Nucleic Acids Res., 14, <br /> Cleveland, OH) [a-35S]dATP. Sequencing strategy primers used                   1319-1324. <br /> in sequencing ompH ssc genes S. typhimurium SH7622              Nakamaye,K. Eckstein,F. (1986) Nucleic Acids Res., 14, 9679-9698. <br /> ssc gene pUCHS 14 identical used earlier sequencing         Nikaido,H. Vaara,M. (1985) Microbiol. Rev., 49, 1-32. <br /> the wild-type ompH ssc genes S.typhimurium SH5014 (Koski et al.,            Nikaido,H. Vaara,M. (1987) In Neidhardt,F.C., Ingraham,J.L., <br /> 1989, 1990; Hirvas et al., 1990b). Occasional problems arising                   Low,K.B., Magasanik,B., Schaechter,M. Umbarger,H.E. (eds), <br /> compressions gel overcome substituting 7-deazaguanosine                Escherichia coli Salmonella typhimurium-Cellular Molecular <br /> 5&apos;-triphosphate (Mizusawa et al., 1986) dGTP.                                     Biology. ASM Publications, Washington DC, pp. 7-22. <br />                                                                                    Nishijima, <br /> </body></html>