<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Letter <br />  <br />  <br /> Evolution Topology Yeast Protein <br /> Interaction Network <br /> Stefan Wuchty <br /> Department Physics, University Notre Dame, Notre Dame, Indiana 46556, USA <br />  <br />  <br />        The integrity yeast protein&#226;&#8364;&#8220;protein interaction network maintained highly connected proteins, or <br />        hubs, hold numerous less-connected proteins together. The structural importance increased <br />        essentiality proteins suggest likely conserved evolution, implying strong relationship <br />        number interactions evolutionary distance orthologs organisms. The <br />        existence coherence recently reported strongly depend quality protein interaction and <br />        orthologs data. Here, introduce novel method, evolutionary excess retention (ER), allowing uncover a <br />        robust strong correlation conservation, essentiality, connectivity yeast protein. We <br />        conclude relevance hubs network integrity simultaneously reflected considerable <br />        probability simultaneously evolutionarily conserved essential, observation does an <br />        equivalent nonessential proteins. Providing thorough assessment impact noisy incomplete data have <br />        findings, conclude results largely insensitive quality utilized data. <br />        [Supplemental material available online www.genome.org.] <br />  <br /> The recently uncovered scale-free topology protein&#226;&#8364;&#8220;protein            served strongly correlated, trend does not <br /> interaction networks focused attention important          equivalent nonessential proteins. In addition, we <br /> role small subset highly linked proteins, hubs,          provide thorough assessment impact quality of <br /> guarantee functional structural integrity network         used data sets ability method determine <br /> (Jeong et al. 2001; Park et al. 2001; Wagner 2001). The observed         trends, conclude findings largely in- <br /> correlations essentiality connectivity pro-         sensitive incomplete noisy protein interaction as <br /> tein (Jeong et al. 2001; Wuchty 2002) suggest hubs likely       ortholog data. <br /> maintained evolution, implying emergence correla- <br /> tions number interactions protein            RESULTS AND DISCUSSION <br /> evolutionary conservation (Hurst Smith 1999). Such trend           <span id='am-1' about='protege:TO' typeof='owl:Thing'>To</span> build source ortholog data, browsed protein clusters <br /> has reported Saccharomyces cerevisiae, concluding          compiled InParanoid database (Remm et al. 2001), which <br /> proteins higher connectivity smaller evolutionary          provides sequence information orthologous protein pairs be- <br /> distance orthologs Caenorhabditis elegans (Fraser et al.     tween S. cerevisiae higher eukaryotes: Homo sapiens, Mus <br /> 2002). Yet, recent reanalyses (Jordan et al. 2002, 2003a,b)          musculus, C. elegans, Drosophila melanogaster, Arabidopsis <br /> alternative approaches (Hahn et al. 2002) rigorously questioned          thaliana. The connectivity 3677 yeast proteins inferred <br /> the strength available evidence, concluding absence        Web 11,249 interactions assembled DIP da- <br /> distinctive correlations connectivity evolution-         tabase (Xenarios et al. 2002). Although quality interaction <br /> ary conservation proteins. A recent reply objections         ortholog data matter debate (Koski Golding <br /> (Fraser et al. 2003) argued utilization incomplete       2001, Goldberg Roth 2003; Wall et al. 2003), did not <br /> noisy sets protein interactions, inaccurately deter-       perform quality checks, allowing assess ability of <br /> mined orthologs, reasons asserted absence       method uncover assumed trends putatively flawed <br /> trends. After carefully compiling interactions known            data. Using data sets, scatterplot Figure 1A, depicting <br /> high-throughput screens proteins S. cerevisiae (Uetz et al.        evolutionary distance D (see Methods) yeast protein as <br /> 2000; Ito et al. 2001; Gavin et al. 2002; Ho et al. 2002)            function connectivity k, fails indicate convincing corre- <br /> determining evolutionary distances orthologs Schizo-         lations. This indicated low average values the <br /> saccharomyces pombe Candida candidans novel phyloge-            Pearson&#226;&#8364;&#8482;s correlation coefficient, &#194;&#175;r = &#226;&#179;&#174;0.045, P&#194;&#175; = 0.137, and <br /> netic estimation method (Wall et al. 2003), highly disputed          Spearman&#226;&#8364;&#8482;s rank coefficient &#226;?&#179;&#194;&#175; = &#226;&#179;&#174;0.055, P    &#194;&#175; = 0.092 (for detailed <br /> correlations ultimately found.                                      values, Supplemental material). The small average slope &#226;?&#163;&#194;&#175; <br />       Although results statistically significant, sensi-     = &#226;&#179;&#174;0.0072 best linear fit D &#226;&#710;&#188; &#226;?&#163;k supports previous con- <br /> tivity data sources remains considerable weakness        clusions dependence evolutionary distance D on <br /> introduced method. Here, contribute current debate          protein connectivity k negligible (Jordan et al. 2003a,b; for <br /> presenting alternative approach, allowing uncover sig-        detailed values, Supplemental material). However, ob- <br /> nificant strong correlation highly interacting proteins      served correlations determined enhanced <br /> S. cerevisiae far higher probability evolutionarily         level accounting scale-free nature protein net- <br /> conserved higher eukaryotes. Observing propensity         work. Indeed, frequency highly interacting proteins decays <br /> essential, highly connected proteins evolutionarily con-           power-law (Jeong et al. 2001;Wagner 2001) P(k) &#226;&#710;&#188; k&#226;&#179;&#174;&#226;?&#165;, indi- <br />                                                                          cating sparsely connected proteins significantly outnumber <br />                                                                          highly linked counterparts. The uniform sampling used in <br /> E-MAIL swuchty@nd.edu; FAX (574) 631-5952. <br /> Article publication http://www.genome.org/cgi/doi/10.1101/    Figure 1A does account differences number <br /> gr.2300204.                                                              proteins different connectivity groups. To guarantee <br />  <br /> 1310     Genome Research                             14:1310&#226;&#8364;&#8220;1314 &#194;&#169;2004 Cold Spring Harbor Laboratory Press ISSN 1088-9051/04; www.genome.org <br />             www.genome.org <br />                                                                               Evolution Topology Yeast Protein Network <br />  <br />  <br />  <br />  <br /> Figure 1 (A) Scatterplot showing yeast protein evolutionary distance D orthologs higher eukaryote function proteins&#226;&#8364;&#8482; <br /> number interactions k. The black line corresponds D &#226;&#710;&#188; &#226;?&#163;k &#226;?&#163;&#194;&#175; = &#226;&#179;&#174;0.0072. (B) By applying logarithmic binning horizontal k-axis, we <br /> obtain improved correlations excess sequence conservation ESCk k (&#226;&#338;&#169;ESCk&#226;&#338;&#170; &#226;&#710;&#188; &#226;?&#164;log k, &#226;?&#164;        &#194;&#175; = &#226;&#179;&#174;0.152). (C) The orthologous excess retention <br /> parameter ERk shown function logarithmically binned ks indicates statistically significant monotonic trend conservation more <br /> connected proteins (ERk &#226;&#710;&#188; &#226;?&#165;log k, &#226;?&#165;&#194;&#175; = 0.813). In plot, symbols correspond orthologs S. cerevisiae identified higher organisms <br /> listed legends box C. <br />  <br />  <br />  <br /> balanced sampling k-values, use logarithmic binning              teins groups logarithmically binning connectivity k, <br /> the k-axis, procedure corrects skewed nature             determine respective fraction number proteins <br /> scale-free distribution (see Methods; Albert et al. 2000; Goldstein            having orthologs reference organisms total num- <br /> et al. 2004). Providing comparable numbers proteins                 ber proteins bin. As null hypothesis, assume a <br /> bin, determine mean excess sequence conservation, &#226;&#338;&#169;ESCk&#226;&#338;&#170;                random distribution orthologs quantified frac- <br /> (see Methods). The distributions obtained (Fig. 1B) indicate              tion total number proteins orthologs a <br /> statistically significant correlations &#226;&#338;&#169;ESCk&#226;&#338;&#170; k (average           reference eukaryote total number proteins present in <br /> Pearson&#226;&#8364;&#8482;s &#194;&#175;r = &#226;&#179;&#174;0.775, P&#194;&#175; = 2.6 &#226;&#179;&#175; 10 &#226;&#179;&#174;3 ; average Spearman&#226;&#8364;&#8482;s                   interaction network (see Methods). Defined orthologous <br /> &#226;?&#179;&#194;&#175; = &#226;&#179;&#174;0.720, P&#194;&#175; = 0.022), best fit logarithmic curve,                  excess retention ERk, ratio k-dependent random <br /> &#226;&#338;&#169;ESCk&#226;&#338;&#170; &#226;&#710;&#188; &#226;?&#164;log k, &#226;?&#164;&#194;&#175; = &#226;&#179;&#174;0.152. Although fits indicate stronger            fractions orthologous proteins shows visually striking correla- <br /> dependence, result agree earlier conclu-              tions k (Fig. 1C). These results supported by <br /> sions weak interdependence connectivity                  distinct statistically significant average Pearson&#226;&#8364;&#8482;s correlation <br /> protein sequence conservation (Hahn et al. 2002; Jordan                coefficients &#194;&#175;r = 0.892 (P&#194;&#175; = 2 &#226;&#179;&#175; 10&#226;&#179;&#174;3) Spearman&#226;&#8364;&#8482;s rank corre- <br /> et al, 2003a).                                                                 lation coefficients &#226;?&#179;&#194;&#175; = 0.993 (P &#194;&#175; = 4 &#226;&#179;&#175; 10&#226;&#179;&#174;3). The linear depen- <br />       However, apparently weak correlation sequence                 dence log-linear plot indicates ERk &#226;&#710;&#188; &#226;?&#165;log k with <br /> conservation connectivity does necessarily invalidate                  &#226;?&#165;&#194;&#175; = 0.813, representing clear trend evolutionary <br /> the original hypothesis, assumed reflect data depen-             conservation connected proteins (for detailed values, <br /> dence applied analytical framework (Fraser et al. 2003).                Supplemental material). <br /> To probe initial assumption connected proteins                         Similarly, observe trend excess retention <br /> have higher tendency evolutionarily conserved, intro-               essential proteins collected Bioknowledge <br /> duce alternative approach separately identifying frac-               library (Fig. 2A; Costanzo et al. 2001). As expected, observe a <br /> tions orthologous proteins k interactions. Dividing pro-               dilution process nonessential proteins increasing levels of <br />  <br />                                                                                                                      Genome Research              1311 <br />                                                                                                                         www.genome.org <br />  Wuchty <br />  <br />  <br />  <br />  <br /> Figure 2 (A) The essential nonessential excess retention function logarithmically binned connectivities proteins shows statistically <br /> significant monotonic trends (ERk &#226;&#710;&#188; &#226;?&#165;log k, essential: &#226;?&#165; = 1.386, nonessential: &#226;?&#165; = &#226;&#179;&#174;0.371). (B) The trend excess retention essential proteins <br /> is enhanced focus proteins simultaneously orthologs higher eukaryotes (&#226;?&#165;      &#194;&#175; = 2.128). (C) However, proteins have <br /> orthologs reference eukaryotes dispensable cell&#226;&#8364;&#8482;s <span id='am-2' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-3' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-4' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span>survival</span> fail significant excess retention signal. <br />  <br />  <br />  interactions. In cases, application logarithmic bin-             comparable correlation proteins nonessential <br />  ning uncovers sharp logarithmic trends, ERk &#226;&#710;&#188; &#226;?&#165;log k (for detailed           evolutionarily conserved (Fig. 2C). <br />  values, Supplemental material), supported dis-                   To estimate potential influence incomplete noisy <br />  tinctive statistically significant Pearson&#226;&#8364;&#8482;s correlation coeffi-         protein interaction data findings, added (removed) up <br />  cients (essential proteins: r = 0.868, P = 4.5 &#226;&#179;&#175; 10&#226;&#179;&#174;4, nonessential          20% interactions randomly selected protein pairs, <br />  proteins: r = &#226;&#179;&#174;0.848, P = 9.5 &#226;&#179;&#175; 10&#226;&#179;&#174;4) Spearman&#226;&#8364;&#8482;s rank coef-              mimicking false positives (negatives). Similarly, simu- <br />  ficients (essential proteins: &#226;?&#179; = 0.976, P = 9.8 &#226;&#179;&#175; 10&#226;&#179;&#174;3, nonessen-           lating presence false-positive (negative) ortholog and/or <br />  tial proteins: &#226;?&#179; = 0.868, P = 4.5 &#226;&#179;&#175; 10&#226;&#179;&#174;3; detailed values,           essential (nonessential) signals, randomly increased (de- <br />  Supplemental material). Furthermore, excess                 creased) original sets respective proteins 20%. In <br />  retention orthologous essential proteins indepen-             case, generated 1000 different realizations removal <br />  dent: Figure 2B shows strong logarithmic, ER k &#226;&#710;&#188; &#226;?&#165;log k,                   (addition) repeated assessment essential/nonessential <br />  &#226;?&#165;&#194;&#175; = 2.128, statistically significant trend (average Pearson&#226;&#8364;&#8482;s           and/or orthologous excess retention. We basic <br /> &#194;&#175;r = 0.909, P&#194;&#175; = 7.1 &#226;&#179;&#175; 10 &#226;&#179;&#174;4 , average Spearman&#226;&#8364;&#8482;s &#226;?&#179;&#194;&#175; = 0.971,                 trends remain qualitatively unaltered, albeit slopes the <br />  P&#194;&#175; = 8.8 &#226;&#179;&#175; 10&#226;&#179;&#174;3) highly linked proteins simul-              actual curves gradually change, allowing conclude the <br />  taneously essential evolutionarily conserved. In particular,             uncovered correlations largely unaffected data incom- <br />  proteins interactions excess retention            pleteness (for details, Supplemental material). <br />  smaller (ERk &lt; 1), indicating high fraction                In summary, results clearly indicate highly con- <br />  numerous weakly connected proteins discounted                    nected proteins far likely essential (simulta- <br />  evolution. Compared respective numbers simple or-                neously) conserved orthologs higher eukaryotes are <br />  thologous excess retention, significantly higher excess reten-           less-connected counterparts. Focusing D-inde- <br />  tion proteins &gt;10 links (ERk&gt;10 &gt; 1) suggests            pendent measure orthologous excess retention ERk and <br />  proteins preferentially conserved higher eukaryotes               correcting scale-free statistics applying logarithmic <br />  essential survival cell. However,            binning, reduced noise level uncovered significant <br />  <br /> 1312      Genome Research <br />             www.genome.org <br />                                                                         Evolution Topology Yeast Protein Network <br />  <br />  <br /> trend connectivity evolutionary conservation             assumption sequences species more <br /> the underlying data sets. Although earlier approaches determin-         similar main ortholog sequence other <br /> ing trends severely impaired used data,          species in-paralogs belong group or- <br /> our results suggest method widely insensitive        thologs. The quality resulting orthologous clusters ex- <br />                                                                         amined increased final bootstrap analysis. Furthermore, <br /> quality data sources, observation holds for <br />                                                                         InParanoid provides comprehensive pairwise comparative or- <br /> data inconsistencies noise.                                         thologous information S. cerevisiae H. sapiens, D. melanogas- <br />       Our observations allow reappraisal conclusion       ter, C. elegans, M. musculus A. thaliana. In study, used <br /> that dependence evolutionary distance D protein           core pairs cluster provide confidence level of <br /> connectivity k negligible (Hahn et al. 2002; Jordan et al.           100%. Thus, underlying interaction network, found <br /> 2003a). Because logarithmic dependency ex-            1997 proteins orthologs H. sapiens, 1757 D. me- <br /> cess retention proteins&#226;&#8364;&#8482; connectivity, ERk &#226;&#710;&#188; log k, as-      lanogaster, 1754 M. musculus, 1489 C. elegans 1898 A. <br /> sumption effect evolutionary distance D           thaliana. <br /> also depend logarithmically k, appears reasonable. Provided <br /> that effect connectivity k evolutionary reten-        Assignment (non-)Essentiality <br /> tion ERk deemed large, effect evolutionary distance <br />                                                                         The Bioknowledge library (Costanzo et al. 2001) compiles scans <br /> D exemplified mean excess sequence conservation &#226;&#338;&#169;ESC&#226;&#338;&#170;         experimental literature provide comprehensive list es- <br /> between orthologous sequences appears stronger previously          sential nonessential proteins. Of S. cerevisiae proteins <br /> described. Therefore, absence distinctive correlations be-       appearing interaction network, 810 assigned es- <br /> tween connectivity protein sequence conserva-          sential, 2704 considered nonessential. <br /> tion result incorrect assumption these <br /> distributions follow linear instead logarithmic trend.              Evolutionary Distance <br />       Most importantly, uncovered correlations indicate that <br />                                                                         Assessing number substitutions site, define the <br /> the evolutionary conservation protein affected by <br />                                                                         evolutionary distance (Grishin 1997) D yeast protein <br /> the protein&#226;&#8364;&#8482;s individual functional role topological    orthologous sequences reference eukaryote as <br /> and contextual placement cellular network. These trends          q = [ln(1 + 2D)]/2D, q fraction unchanged sites in <br /> are enhanced adding essential information sets        sequence alignment protein pairs (Fraser et al. 2002, 2003; <br /> of orthologous proteins. The fact proteins essential       Jordan et al. 2003a,b). <br /> for survivability cell evolutionarily con- <br /> served contradiction. However, protein core <br />                                                                         Excess Sequence Conservation <br /> that ensures cells survival strongly privileged by <br /> evolution.                                                              To guarantee balanced sampling k-values, grouped all <br />                                                                         proteins bins logarithmically increasing connectivity k. In <br />                                                                         bin, determine mean excess sequence conservation, <br /> METHODS                                                                                                  1 <br />                                                                                                               Nk <br />                                                                                                                       &#229;&#8230;&#183;D&#229;&#8230;&#184; &#226;&#710;&#8217; Di <br /> Protein Interactions <br />                                                                                               &#229;&#8230;&#183;ESCk&#229;&#8230;&#184; = <br />                                                                                                          Nk   &#229;&#8230;&#186; <br />                                                                                                                       &#229;&#8230;&#183;D&#229;&#8230;&#184; <br />                                                                                                                               , <br />  <br /> Large-scale two-hybrid screens allow identification         Nk number proteins respective bin, and <br /> potential protein&#226;&#8364;&#8220;protein interactions open reading <br /> frames S. cerevisiae genomic sequence (Uetz et al. 2000;                                                 N <br />                                                                                                           1 <br /> Ito et al. 2000, 2001) integral proteomics research. <br /> Yet, quality two-hybrid data significantly affected by <br />                                                                                                   &#229;&#8230;&#183;D&#229;&#8230;&#184; = <br />                                                                                                           N   &#229;&#8230;&#186;D  i <br />                                                                                                                          i <br />  <br />  <br /> high rates false positives false negatives (von Mering et al.    mean evolutionary distance total number N pro- <br /> 2003), indicated fact results ob-        teins. <br /> tained different groups limited overlap (Hazbun and <br /> Fields 2001). Moreover, identified interactions merely rely <br /> on positive signals single technique result indi-       Excess Retention <br /> rect observations. In study, used database interact-      According degree k underlying yeast protein in- <br /> ing proteins (DIP), based extensive literature searches          teraction network, grouped proteins bins logarithmi- <br /> aims, provide curated collection functional link-      cally increasing connectivity k. In bin, ratio eA      A <br />                                                                                                                                   k = nk /Nk <br /> ages proteins obtained experimental methods. The majority         represents certain feature A, nAk number proteins <br /> of protein&#226;&#8364;&#8220;protein interaction data relies yeast two-hybrid          A (e.g., essential orthologous reference <br /> and coimmunoprecipitation experiments. Eighty-four percent              organism), Nk total number proteins. In absence <br /> of interactions detected single experiment,         correlation A position network, eA k has <br /> whereas 16% confirmed experimental                 general k-independent value e = n/N, n = &#226;&#338;&#186;knA     k the <br /> method. DIP currently records 3677 proteins involved        total number yeast proteins having feature A, N = &#226;&#338;&#186;k Nk is <br /> 11,249 interactions (Xenarios et al. 2002).                             total number yeast proteins underlying network. <br />                                                                         Thus, bin k, define evolutionary excess retention <br /> Assignment Orthology                                                 feature A ERA    A <br />                                                                                             k = ek /e, k-independent <br /> The determination orthologous pairs sequences em-           value ERk = 1 random assignment A. <br /> ploys pairwise BLAST comparisons proteomes. Each <br /> protein represents query entire proteome the <br /> other species. Reciprocal best hits BLAST searches, em-        Logarithmic Binning <br /> phasizing expectation values &lt;10&#226;&#179;&#174;3, considered or-            To guarantee balanced sampling k-values, use logarith- <br /> thologous. Our choice orthologous protein sequence informa-          mic binning k-axis, procedure curve estimation that <br /> tion InParanoid database (Remm et al. 2001), runs          corrects skewed nature scale-free distribution. On a <br /> all-versus-all BLAST searches sets sequences. Se-           logarithmic scale, define bin size <br />  <br />                                                                                                                       &#229;&#8224;&#8240;&#229;&#8224;&#352; <br /> quence pairs mutual two-way best hits detected and <br /> serve central core ortholog pairs or-                                            1     b <br />                                                                                                   &#226;&#338;&#172;=       log   , <br /> thologs species clustered later steps. The initial                                      N     a <br />  <br />                                                                                                                Genome Research          1313 <br />                                                                                                                        www.genome.org <br />  Wuchty <br />  <br />  <br /> where N corresponds selected number bins. Values              Hazbun, T. Fields, S. 2001. Networking proteins yeast. Proc. Natl. <br /> b refer minimal maximal value connectivity ki,                  Acad. Sci. 98: 4277&#226;&#8364;&#8220;4278. <br /> b = maxi (ki) = mini (ki). Thus,                                        Ho, Y., Gruhler, A., Heilbut, A., Bader, G.D., Moore, L., Adams, S.L., <br />                                                                                   Millar, A., Taylor, P., Bennett, K., Boutilier, K., et al. 2002. <br />  <br />                     ni = log   &#229;&#8224;&#8240;&#229;&#8224;&#352; <br />                                ki <br />                                a <br />                                   &#226;&#178;? &#226;&#338;&#172;, ni &#226;&#710;&#710; &#229;&#8230;&#179;0, N &#226;&#710;&#8217; 1&#229;&#8230;&#180; <br />                                                                                   Systematic identification protein complexes Saccharomyces <br />                                                                                   cerevisiae mass spectrometry. Nature 415: 180&#226;&#8364;&#8220;183. <br />                                                                               Hurst, L. Smith, N. 1999. Do essential genes evolve slowly? Curr. <br />                                                                                   Biol. 9: 747&#226;&#8364;&#8220;750. <br /> reflects number bin assign protein ki inter-             Ito, T., Tashiro, K., Muta, S., Ozawa, R., Chiba, T., Nishizawa, M., <br /> actions. Representing nith bin k-axis, place kni                 Yamamoto, K., Kuhara, S., Sakaki, Y. 2000. Towards a <br /> the end bin using kni = e&#226;&#338;&#172;(ni+1). The advantage loga-                protein&#226;&#8364;&#8220;protein interaction map budding yeast: A <br /> rithmic binning elevated degree noise reduction                  comprehensive examine two-hybrid interactions all <br /> dependent bin size (Albert et al. 2000; Goldstein et al.                   possible combinations yeast proteins. Proc. Natl. Acad. <br /> 2004). Although procedure causes loss accuracy,                Sci. 97: 1143&#226;&#8364;&#8220;1147. <br /> uncover buried trends satisfying extent applying              Ito, T., Chiba, T., Ozawa, R., Yoshida, M., Hattori, M., Sakaki, Y. <br />                                                                                   2001. A comprehensive two-hybrid analysis explore yeast <br /> statistical methods binned data. <br />                                                                                   protein interactome. Proc. Natl. Acad. Sci. 98: 4569&#226;&#8364;&#8220;4574. <br />                                                                               Jeong, H., Mason, S., Barab&#195;&#161;si, A.-L., Ol <br /> </body></html>