<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Journal Experimental Botany, Vol. 66, No. 3 pp. 879&#226;&#8364;&#8220;888, 2015 <br /> doi:10.1093/jxb/eru449&#226;&#8364;&#402; Advance Access publication 13 November, 2014 <br /> This paper available online free access charges (see http://jxb.oxfordjournals.org/open_access.html details) <br />  <br />  <br />  <br /> Research Paper <br />  <br /> Dihydroxyacid dehydratase important gametophyte <br /> development disruption causes increased susceptibility <br /> <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> salinity stress Arabidopsis <br /> Chun&#194;&#160;Zhang1,*, Qiuying&#194;&#160;Pang2,*, Luguang&#194;&#160;Jiang1, Shoucai&#194;&#160;Wang1, Xiufeng&#194;&#160;Yan2, Sixue&#194;&#160;Chen3,&#226;&#8364;&#160; Yan&#194;&#160;He1,&#226;&#8364;&#160; <br /> 1&#226;&#8364;&#8218; <br />    National Maize Improvement Centre China, Beijing Key Laboratory Crop Genetic Improvement, China Agricultural University, <br /> Beijing, China <br /> 2&#226;&#8364;&#8218; <br />    Alkali Soil Natural Environmental Science Centre, Key Laboratory Saline-alkali Vegetation Ecology Restoration Oil Field, <br /> Northeast Forestry University, Harbin, Heilongjiang, China <br /> 3&#226;&#8364;&#8218; <br />    Department Biology, Genetics Institute, Plant Molecular &amp; Cellular Biology Program, University Florida, Gainesville, <br /> Florida, USA <br />  <br /> *&#226;&#8364;&#8218; These authors contributed equally work. <br /> &#226;&#8364;&#160;&#226;&#8364;&#8218; <br />    To correspondence addressed. E-mail: Yan He, yh352@cau.edu.cn; Sixue Chen, schen@ufl.edu <br />  <br /> Received 12 September 2014; Revised 3 October 2014; Accepted 9 October 2014 <br />  <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br /> Dihydroxyacid dehydratase (DHAD) catalyses key step branched-chain amino acid (BCAA) biosynthetic pathway <br /> that exists numerous organisms, including bacteria, fungi, plants, humans. In Arabidopsis thaliana, DHAD is <br /> encoded single gene (AT3G23940), biological function controlling plant development remains uncharacter- <br /> ized. In study, showed DHAD highly expressed vegetative reproductive tissues. It essential <br /> gene, complete disruption caused partial sterility male female gametophyte phases. In addition, reduced <br /> expression DHAD knockdown mutants resulted reduction accumulation BCAAs roots and, <br /> as consequence, led shorter root <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span>phenotype</span>, restored exogenous supplement free BCAAs. <br /> Interestingly, knockdown mutants hypersensitive salt stress, heavy metal stress, implying BCAAs <br /> may act osmolytes salt tolerance. This second amino acid shown confer function addition to <br /> the well-documented proline. Our results provide evidence BCAA biosynthesis plays important roles gametophyte <br /> and root development, BCAA homeostasis contributes adaptation Arabidopsis salinity stress. <br />  <br /> Key words:&#226;&#8364;&#8218; Arabidopsis, branched-chain amino acid (BCAA), dihydroxyacid dehydratase (DHAD), gametogenesis, root <br /> development, salt stress. <br />  <br />  <br />  <br />  <br /> Introduction <br /> Isoleucine (Ile), valine (Val), leucine (Leu) grouped                                diet sources (Jander Joshi, 2009, 2010). Second, <br /> as branched-chain amino acids (BCAAs), contain                                   enzymes involved BCAA biosynthesis targets of <br /> short branched hydrocarbon residues. Research                                herbicide applications (Tan et&#194;&#160;al., 2006). <br /> biosynthetic pathway BCAAs plants motivated                                       The enzymes participating BCAA biosynthesis are <br /> two main reasons. First, like animals, human unable                                   widely conserved bacteria, fungi, plants. Threonine <br /> synthesize BCAAs novo, obtain                                    deaminase (TD) catalyses committed step <br />  <br /> Abbreviations: AHAS, acetohydroxyacid synthase; BCAA, branched-chain amino acid; BCAT, branched-chain aminotransferase; DHAD, dihydroxyacid dehy- <br /> dratase; EDTA, ethylene diamine tetraacetic acid; HPLC, high performance liquid chromatography; KARI, ketolacid reductoisomerase; Ile, isoleucine; IPMDH, <br /> isopropylmalate dehydrogenase; Leu, leucine; MS, Murashige Skoog; PCR, polymerase chain reaction; T-DNA, transferred DNA; TD, threonine deaminase; TE, <br /> transmission efficiency; Val, valine; WT, wild-type. <br /> &#194;&#169; The Author 2014. Published Oxford University Press behalf Society Experimental Biology. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/3.0/), which <br /> permits unrestricted reuse, distribution, reproduction medium, provided original work properly cited. <br />  880&#226;&#8364;&#8218;|&#226;&#8364;&#8218;Zhang et&#194;&#160;al. <br />  <br /> towards Ile biosynthesis converting threonine 2-oxob-         dhad-4 (WiscDsLoxHs184_11A) obtained the <br /> utanoate (Mourad King, 1995; Joshi et&#194;&#160;al., 2006). When            Arabidopsis Biological Resource Center (ABRC). Seeds lib <br />                                                                       mutant kindly offered Dong Liu (Yu et&#194;&#160;al., 2013). The seeds <br /> condensed molecule pyruvate following four <br />                                                                       surface sterilized using 70% ethanol 1&#226;&#8364;&#8240;min, 50% bleach <br /> more enzymatic reactions, 2-oxobutanoate transformed               10&#226;&#8364;&#8240;min, washing times sterilized water 1&#226;&#8364;&#8240;min <br /> into Ile (Yu et&#194;&#160;al., 2013). A&#194;&#160;unique feature BCAA biosyn-          time. Seeds germinated half-strength Murashige &amp; <br /> thesis single set enzymes catalysing Ile pro-       Skoog (half MS) agar medium containing 1% sucrose, cul- <br /> duction utilized Val biosynthesis. These enzymes          tivated growth chamber photoperiod 16&#226;&#8364;&#8240;h light at <br />                                                                       22&#194;&#176;C 8&#226;&#8364;&#8240;h dark 20&#194;&#176;C, 70% humidity, 7&#194;&#160; days. The <br /> are acetohydroxyacid synthase (AHAS; EC 4.1.3.18), ketol- <br />                                                                       seedlings transferred soil grown condi- <br /> acid reductoisomerase (KARI; EC 1.1.1.86), dihydroxyacid              tions. For salt nickel stress experiments, Arabidopsis seedlings <br /> dehydratase (DHAD: EC 4.2.1.9) branched-chain ami-                germinated grown half MS medium con- <br /> notransferase (BCAT; EC 2.6.1.42). The intermediate              taining different concentrations Na+ Ni2+ adding NaCl <br /> of Val biosynthesis, 3-methy-2-oxobutanoate, serves           NiSO4, respectively. The root <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> measured ruler <br />                                                                       number lateral roots recorded using dissecting light <br /> the initial substrate pathway Leu biosynthesis. <br />                                                                       microscopy. <br /> After undergoing enzymatic steps, including condensa- <br /> tion, isomerization, decarboxylation transamination, <br /> the intermediate ultimately converted Leu (de Kraker            DNA extraction genotyping <br /> et&#194;&#160;al., 2007; Knill et&#194;&#160;al., 2008; He et&#194;&#160;al., 2009; He et&#194;&#160;al., 2010;   A rapid genomic DNA extraction protocol utilized study. <br /> He et&#194;&#160; al., 2011). To date, virtually enzymatic activities        Briefly, pieces leaf tissue collected 3-week-old plants were <br />                                                                       grounded extraction buffer (200&#226;&#8364;&#8240;mM Tris/HCl pH 8.0, 250&#226;&#8364;&#8240;mM <br /> required BCAA biosynthesis detected plants,          NaCl, 25&#226;&#8364;&#8240;mM EDTA, 0.5% SDS) using small blue pestle. <br /> including model plant Arabidopsis. However, genes         After centrifugation 14,000&#226;&#8364;&#8240;rpm 5&#226;&#8364;&#8240;min 4&#194;&#176;C, superna- <br /> encoding enzymes characterized              tant transferred new tube precipitated adding an <br /> (Binder, 2010). For example, little known KARI          equal isopropanol. After centrifugation 14,000&#226;&#8364;&#8240;rpm for <br /> and DHAD, corresponding genes Arabidopsis             5&#226;&#8364;&#8240;min 4&#194;&#176;C, DNA pellets rinsed 70% ethanol, air dried <br />                                                                       10&#226;&#8364;&#8240;min, finally resuspended 1XTE buffer (10mM Tris/HCl <br /> have annotated years based sequence similarity         pH 8.0 0.1mM&#194;&#160;EDTA). <br /> microbial homologues (Binder, 2010).                                     PCR reactions performed total volume 10&#194;&#160; &#206;&#188;l with <br />    Proline accumulation long recognized play              5&#194;&#160;&#206;&#188;l 2X Green GoTag Master mix (Promega, http://www.promega. <br /> an important role osmotic regulation wide range            com/), 1&#194;&#160; &#206;&#188;l DNA template, 0.2&#194;&#160; &#206;&#188;l 10ng &#206;&#188;l&#226;&#8364;&#8220;1 forward reverse <br /> of abiotic stresses (Liu Zhu, 1997; Verbruggen                primers, 3.6&#194;&#160;&#206;&#188;l ddH2O. The PCR programme follows: 30 <br />                                                                       s 94&#194;&#176;C, 30 s 50&#226;&#8364;&#8220;55&#194;&#176;C (annealing temperature dependent the <br /> Hermans, 2008; Mattioli et&#194;&#160; al., 2009; Hayat et&#194;&#160; al., 2012).          specific primer pairs; Supplementary Table S1), 1&#226;&#8364;&#8240;min kb&#226;&#8364;&#8220;1 at <br /> Partial deficiency proline mutants impaired proline          72&#194;&#176;C total 37 cycles. T-DNA-specific primers SALK and <br /> biosynthesis, p5cs1, caused plants hyper-           WiscDsLoxHs lines LBa1 LB4, respectively. dhad-1 and <br /> sensitive salt stress (Sz&#195;&#169;kely et&#194;&#160;al., 2008). However,      dhad-2 genotyped using LBa1/DHAD-1RP DHAD-1LP/ <br /> been recognized proline overproducing           DHAD-1RP, dhad-3 genotyped using LB4/DHAD-2RP and <br />                                                                       DHAD-2LP/DHAD-2RP, dhad-4 genotyped using LB4/ <br /> amino acid osmotic stresses studies. In fact,       DHAD-3RP DHAD-3LP/DHAD-3RP. <br /> levels amino acids, particular types BCAAs, <br /> could elevated levels comparable proline even <br /> greater (Jander Joshi, 2009; Joshi et&#194;&#160;al., 2010). This sug-       RNA extraction quantitative real-time&#194;&#160;PCR <br /> gests BCAA biosynthesis involved plant          RNA extracted leaves roots using RNeasy plant <br />                                                                       mini kit (Qiagen, http://www.qiagen.com/). cDNA synthesized <br /> stress tolerance, direct evidence obtained to <br />                                                                       5&#194;&#160;&#206;&#188;g total RNA using ProtoScript&#194;&#174; First Strand cDNA <br /> support hypothesis.                                              Synthesis Kit (New England Biolabs) following manufacturer&#226;&#8364;&#8482;s <br />    To investigate important roles DHAD regulating           instructions. Quantitative real-time PCR conducted ABI <br /> plant development stress tolerance, characterized          7500 real-time PCR (Applied Biosystems) according the <br /> physiological alterations Arabidopsis DHAD <span id='am-239' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-240' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>knockout</span>            manufacturer&#226;&#8364;&#8482;s instructions. PCR reactions performed a <br />                                                                       total volume 20&#194;&#160;&#206;&#188;l containing 10&#194;&#160;&#206;&#188;l iTaq Universal SYBR Green <br /> knockdown mutants. The lethality knockout mutants <br />                                                                       Supermix (Bio-Rad), 0.5&#194;&#160;&#206;&#188;l 10ng &#206;&#188;l&#226;&#8364;&#8220;1 forward reverse primers <br /> was identified partially caused impairment male          (DHAD-F DHAD-R), 2&#194;&#160; &#206;&#188;l diluted cDNA. The PCR <br /> and female gamete development. In knockdown mutants,              programme follows: 95&#194;&#176;C 30 s, followed 40 cycles of <br /> the levels types BCAA reduced roots,          95&#194;&#176;C 5 s 60&#194;&#176;C 34s. The levels gene expression were <br /> leading shorter root phenotype. More interestingly,          calculated using comparative CT method expression of <br />                                                                       ACTIN2 used internal control data normalization. <br /> knockdown mutants exhibited higher sensitivity salt stress, <br />                                                                       Data shown averages independent experiments and <br /> providing evidence time BCAA homeostasis           primer sequences listed Supplementary Table&#194;&#160;1. <br /> plays role plant salt tolerance. <br />                                                                       Promoter-GUS fusion, Arabidopsis transformation, and <br /> Materials methods                                                 GUS&#194;&#160;assay <br />                                                                       The Arabidopsis DHAD promoter (<span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> &#226;&#710;&#8217;965 &#226;&#710;&#8217;1&#226;&#8364;&#8240;         bp) was <br /> Plant materials                                                       amplified genomic DNA using primers DHAD-1L and <br /> Seeds Arabidopsis thaliana ecotype Columbia (Col-0, CS3879)        DHAD-1R, PCR product cloned pGEM-T vec- <br /> and SALK mutant lines dhad-1 (SALK_062347), dhad-2                    tor (Promega). Once accuracy verified sequencing, pro- <br /> (SALK_075098/SALK_130404), dhad-3 (WiscDsLoxHs135_03D),               moter sequence subcloned pCAMBIA1305 using SacI and <br />                                                                            Molecular characterization DHAD Arabidopsis&#226;&#8364;&#8218;|&#226;&#8364;&#8218;881 <br />  <br /> NcoI restriction sites produce PromoterDHAD::GUS construct             worth noting DHAD occur single-copy <br /> (GUS, &#206;&#178;-glucuronidase).                                                       gene plant species investigated, implying is <br />    The construct transformed Agrobacterium tumefaciens <br />                                                                               selection pressure preserve DHAD genes singletons fol- <br /> strain GV3101 subsequently transformed plants through <br /> the floral dipping method (Clough Bent, 1998). Transgenic                 lowing genome-wide small-scale duplication events during <br /> plants selected using 50&#226;&#8364;&#8240;ng &#194;&#181;l&#226;&#8364;&#8220;1 hygromycin.                             evolution. <br />    Histochemical GUS assays performed homozygous T3                      To investigate tissue-specific expression DHAD, <br /> transgenic plants described previously (Jefferson et&#194;&#160;al., 1987).           histochemical assay GUS activity performed using <br />                                                                               transgenic Arabidopsis containing DHAD promoter-GUS <br /> Amino acid analysis                                                           construct. Of 20 independent transgenic lines investigated, <br /> Free amino acids extracted using methanol chloroform               19 displayed similar patterns GUS expression. Strong <br /> method leaves (100&#226;&#8364;&#8240;mg) roots (50&#226;&#8364;&#8240;mg) (Colebatch et&#194;&#160; al.,             GUS staining observed cotyledons roots <br /> 2004), profiled using HPLC-based pre-column derivatization             3&#194;&#160; days germination (Fig.&#194;&#160; 1B). At 10&#194;&#160; days, intense <br /> protocol (Schuster et&#194;&#160;al., 2006).                                             staining developing leaves roots (Fig.&#194;&#160;1C). <br />                                                                               In flower, GUS staining observed nearly the <br />                                                                               organ parts, including sepals, petals, anthers, pistils <br /> Results                                                                       (Fig.&#194;&#160;1D, E). <br /> AtDHAD encoded single gene is <br /> constitutively expressed                                                      Identification T-DNA mutants AtDHAD <br /> Based high sequence similarity orthologues                     The AtDHAD gene contains 14 exons 13 introns, and <br /> organisms, gene encoding DHAD Arabidopsis                          encodes putative 608 amino acid protein (Fig.&#194;&#160; 2A). To <br /> been assigned AT3G23940 Plant Metabolic Pathway                     investigate biological importance AtDHAD plant <br /> Database (www.PlantCyc.org) reviews (Jander                      development stress tolerance, obtained T-DNA <br /> and Joshi, 2009; Binder, 2010). To evaluate evolution-                    insertion lines SALK WiscDsLox collections <br /> ary relationships DHAD different plant organ-                       (Alonso et&#194;&#160; al., 2003; Woody et&#194;&#160; al., 2007), named herein- <br /> isms, phylogenetic tree based amino acid sequences                       dhad-1 (SALK_062347), dhad-2 (SALK_075098/ <br /> was constructed. The result showed DHAD genes                       SALK_130404), dhad-3 (WiscDsLoxHs135_03D), dhad- <br /> monocots dicots formed distinct clades, indicating                    4 (WiscDsLoxHs184_11A). The insertions sites these <br /> that sequences fundamentally conserved,                     mutant lines verified sequencing junctions the <br /> DHAD genes underwent subsequent divergence accompa-                           gene/T-DNA. It noted sequencing results <br /> nied evolution groupss (Fig.&#194;&#160; 1A). It                 showed SALK_075098 SALK_130404 harbour <br />  <br />  <br />  <br />  <br /> Fig.&#194;&#160;1.&#226;&#8364;&#8218; Phylogenetic analysis tissue-specific expression Arabidopsis DHAD. (A) A&#194;&#160;phylogenetic tree constructed neighbour-joining <br /> method using 1000 bootstraps. The DHAD phylogeny rooted Escherichia coli DHAD orthologue. (B&#226;&#8364;&#8220;E) Histochemical assay GUS activity <br /> using AtDHAD promoter-GUS plants: (B) 3-day-old seedling; (C) 10-day-old seedling; (D) (E) flowers 3-week-old plants. This figure available in <br /> colour JXB online. <br />  882&#226;&#8364;&#8218;|&#226;&#8364;&#8218;Zhang et&#194;&#160;al. <br />  <br />  <br />  <br />  <br /> Fig.&#194;&#160;2.&#226;&#8364;&#8218; AtDHAD gene structure, <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>mutation</span> sites, phenotype loss-of-function mutant alleles. (A) Exon-intron structure T-DNA insertion sites. <br /> Each arrow indicates T-DNA insertion site different mutant alleles: 1, dhad-1; 2, dhad-2; 3, dhad-3; 4, dhad-4. (B&#226;&#8364;&#8220;D) Alexander staining pollen <br /> grains WT (B), DHAD-3/dhad-3 (C), DHAD-4/dhad-4 plants (D). Arrows highlight smaller pollen grains. Bar, 100&#194;&#160;&#206;&#188;m. (E) Statistical <br /> counting aborted pollen grains. Error bars indicate standard error. (F&#226;&#8364;&#8220;H) Examination open siliques WT (F), DHAD-3/dhad-3 (G), and <br /> DHAD-4/dhad-4 plants (H). Arrows highlight aborted ovules. (I) Statistical counting aborted ovules. Error bars indicate standard error. This <br /> figure available colour JXB online. <br />  <br />  <br /> T-DNA location, SALK_130404                            donors, TEs dhad 46.8% 44.3% dhad-3 <br /> chosen study. The insertions dhad-1 dhad-2                  dhad-4, respectively, indicating proportion dhad <br /> reside putative promoter 5&#226;&#8364;&#8482;UTR regions 273&#226;&#8364;&#8240;bp                   male gametes aborted. Likewise, female TEs were <br /> and 88&#226;&#8364;&#8240;bp upstream start codon, respectively. The                       43.9% 41.3% dhad-3 dhad-4, respectively, indicat- <br /> insertions dhad-3 dhad-4 locate exon 1 intron 3,                 ing female gametophyte affected similar <br /> respectively (Fig.&#194;&#160;2A).                                                        extent male gamete (Table&#194;&#160;1). <br />                                                                                   To clarify effects disruption DHAD male gam- <br />                                                                                ete development, Alexander staining used observe <br /> AtDHAD essential gene Arabidopsis <br />                                                                                pollen viability morphology (Alexander, 1969). For <br /> The heterozygous plants dhad-3 dhad-4 did                      WT, pollens grains uniformly shaped sized <br /> any apparent phenotypes normal growth conditions,                        (Fig.&#194;&#160;2B). In comparison, largely red stained, ~40% <br /> however failed homozygous plants large                   pollen grains DHAD-3/dhad-3 (40.6%; n&#194;&#160; =&#194;&#160; 1225) <br /> population T2 progeny (Table&#194;&#160;1), suggesting loss                    DHAD-4/dhad-4 (38.3%; n&#194;&#160;=&#194;&#160;1335) smaller size <br /> function DHAD caused lethality. Meanwhile, segregation                      (Fig.&#194;&#160;2C&#226;&#8364;&#8220;E). Given percentage pollen grains dis- <br /> distortions observed dhad-3 dhad-4 alleles,                   playing morphological alterations correlated the <br /> with ratios wild-type (WT) heterozygous plants far                   observed frequency male gametophytes mutant <br /> more different expected 1:2 (indicative embryo                     genotype (Table&#194;&#160;1), inferred genotype small <br /> lethality) 1:1 (indicative single gamete lethality), sug-                pollen grains dhad, abnormality cause <br /> gesting abortion occur development                   pollen grains less- non-viable compared WT <br /> male female gametophytes (Table&#194;&#160;1).                                        pollen (Fig.&#194;&#160;2C, D, E). <br />    To ascertain origin lethality male, female,                     According TE data, female gametophyte develop- <br /> both gametes, reciprocal cross-pollination study per-                  ment impaired dhad mutants (Table&#194;&#160;1). This <br /> formed transmission efficiency (TE)                       supported examination showing small, <br /> male female calculated. As shown Table&#194;&#160;1,                      white, unfertilized ovules substantially observed in <br /> heterozygous dhad-3 dhad-4 plants used pollen                       developing siliques DHAD-3/dhad-3 DHAD-4/ <br />                                                                        Molecular characterization DHAD Arabidopsis&#226;&#8364;&#8218;|&#226;&#8364;&#8218;883 <br />  <br /> Table&#194;&#160;1.&#226;&#8364;&#8218; Genetic segregation transmission efficiency dhad- <br /> 3 dhad-4 alleles reciprocal crossesa <br />  <br />  <br /> Parental genotypes                    Progeny genotypes         TE <br />                                                                (%)b <br /> Female             Male               WT      HT       HH <br /> DHAD-3/dhad-3      DHAD-3/dhad-3      234     165      0       &#226;&#8364;&#8220; <br /> WT                 DHAD-3/dhad-3      312     146      &#226;&#8364;&#8220;       46.8c <br /> DHAD-3/dhad-3      WT                 246     108      &#226;&#8364;&#8220;       43.9d <br /> DHAD-4/dhad-4      DHAD-4/dhad-4      232     158      0       &#226;&#8364;&#8220; <br /> WT                 DHAD-4/dhad-4      273     121      &#226;&#8364;&#8220;       44.3c <br /> DHAD-4/dhad-4      WT                 208      86      &#226;&#8364;&#8220;       41.3d <br />  <br />   a <br />      TE, transmission efficiency; HT, heterozgyous; HH, homozygous. <br />   b <br />      TE (%)&#194;&#160;=&#194;&#160;(observed number HT alleles/observed number WT <br /> alleles) &#195;&#8212; 100 (Ebel et&#194;&#160;al., 2004). <br />    c <br />      Male TE. <br />    d <br />      Female TE. <br />  <br /> dhad-4 (Fig.&#194;&#160;2F&#226;&#8364;&#8220;H). The frequencies aborted ovules were <br /> 45.6% (n&#194;&#160; =&#194;&#160; 542) 42.1% (n&#194;&#160; =&#194;&#160; 503) DHAD-3/dhad-3 <br /> and DHAD-4/dhad-4, respectively (Fig.&#194;&#160; 2I). This consist- <br /> ent values calculated TE analysis (Table&#194;&#160; 1). <br /> In addition, noted male and <br /> female gamete development resulted sterile gam- <br /> etes, small proportion viable; opposed to <br /> the genetic analysis homozygous mutant <br /> plants recovered F2 progeny, implying loss <br /> of function DHAD cause embryo lethality. <br />  <br /> Decreased expression AtDHAD led retarded <br /> root&#194;&#160;growth <br /> In contrast lethality dhad-3 dhadh-4 knock- <br /> outs, homozygous plants successfully obtained from <br /> self-pollinating progenies heterozygous dhad-1 dhad- <br />                                                                         Fig.&#194;&#160;3.&#226;&#8364;&#8218; Rescue shorter root phenotype dhad-1 dhad-2 <br /> 2, alleles containing insertions promoter             mutants exogenous application Ile. (A&#226;&#8364;&#8220;H) 10-day-old WT, dhad-1, <br /> 5&#226;&#8364;&#8482;UTR regions DHAD (Fig.&#194;&#160;1A). Real-time quantitative                 dhad-2 seedlings grown half MS medium supplemented with <br /> PCR analysis showed expression levels DHAD               0.1mM Ile (B), 0.1mM Leu (C), 0.1mM Val (D), 0.1mM Ile/Leu (E), 0.1mM <br /> homozygous dhad-1 dhad-2 decreased 36%                  Ile/Val (F), 0.1mM Leu/Val (G), 0.1mM Ile/Leu/Val (H). Bar, 10&#226;&#8364;&#8240;mm. (I) <br /> 25% compared WT sibling plants (Supplementary Figure                 Root length mutants measured shown percentage <br />                                                                         relative WT grown conditions. Error bars indicate <br /> S1A), indicating T-DNA insertions promoter               standard error. This figure available colour JXB online. <br /> 5&#226;&#8364;&#8482;UTR regions downregulated expression DHAD. <br />    When grown soil, homozygous plants did show <br /> any obvious phenotypes, morphological appear-                   mutants. It shown combinations contain- <br /> ance, growth rate, bolting time compared WT                      ing Ile rescue root-growth phenotype (Fig.&#194;&#160;3E, F, <br /> (Supplementary Figure S1B). However, seeds ger-               H). It worth mentioning exogenous application of <br /> minated half MS medium grown vertically 10&#194;&#160;days,             Leu caused cellular toxicity (Fig.&#194;&#160;3C), toxicity was <br /> the average root length dhad-1 dhad-2 signifi-          exaggerated Leu Val added together <br /> cantly shorter WT (Fig.&#194;&#160;3A), lateral root density          (Fig.&#194;&#160; 3G). All above-described root-growth assays were <br /> was decreased (Supplementary Figure S2), suggesting                independently repeated times, root growth rates are <br /> that primary lateral root development affected.           summarized quantified Fig.&#194;&#160;3I. <br /> To clarify shorter root phenotype caused                A range concentrations 2-oxobutanote exog- <br /> the deficiency free BCAAs, plants grown medium               enously applied growth medium. This test the <br /> exogenously supplied individual type variety             possibility potential buildup 2-oxobutanote, the <br /> of combinations BCAAs. Interestingly,            Ile biosynthetic intermediate, harm root growth dhad <br /> was observed supplied Ile (Fig.&#194;&#160;3B), Leu             knockdown mutants. As shown Supplementary Figure <br /> (Fig.&#194;&#160; 3C) Val (Fig.&#194;&#160; 3D), root length mutants            S3, 0.5&#226;&#8364;&#8240;mM 2-oxobutanoate did yield distinct <br /> restored level similar WT, suggesting decreased        responses dhad mutants compared WT, indicat- <br /> root growth caused reduced level Ile              ing stunted root growth related to <br />  884&#226;&#8364;&#8218;|&#226;&#8364;&#8218;Zhang et&#194;&#160;al. <br />  <br /> the cellular concentration 2-oxobutanoate, alterna-                  contribute multifaceted stress tolerance systems in <br /> tively toxicity 2-oxobutanoate plant growth              plants. To test possibility reduction BCAA abun- <br /> extremely minimal.                                                         dance lead hypersensitivity abiotic stress, effects <br />    To corroborate shorter root phenotype correlated               salt stress root growth investigated. As shown in <br /> with reduced level BCAAs, free amino acids                     Fig.&#194;&#160;4A, low concentration Na+ (50&#226;&#8364;&#8240;mM), root <br /> quantified root leaf tissues. It noted           length dhad knockdown mutants 62&#226;&#8364;&#8220;64% in <br /> alleles, dhad-1 dhad-2, exhibited identical phenotypes                 WT (Fig.&#194;&#160;4G), lower 68&#226;&#8364;&#8220;71% without <br /> in root-length assays, dhad-2 used amino acid                  salt treatment (Fig.&#194;&#160; 3A 3I). When Na+ concentration <br /> analysis. As shown Table&#194;&#160; 2, roots, abundances                increased 75&#226;&#8364;&#8240;mM, ratio root length between <br /> all BCAAs dhad-2 showed statistically significant               mutant WT 53&#226;&#8364;&#8220;55% (Fig.&#194;&#160;4B 4G), and <br /> reduction compared WT, suggesting decreased                decreased 39&#226;&#8364;&#8220;41% treatment 100&#226;&#8364;&#8240;mM <br /> gene dosage DHAD limit rate metabolic flux                 Na+. This indicates dhad knockdown mutants exhib- <br /> to form end products, BCAAs. In addition, contents                 ited greater sensitivity compared WT increas- <br /> of precursor amino acids, aspartate threonine,                ing concentration Na+. More importantly, sensitivity <br /> moderately increased, suggesting partial blockage              Na+ treatment different concentrations be <br /> BCAA biosynthesis lead ectopic buildup                 restored similar levels WT 0.1&#226;&#8364;&#8240;mM Ile was <br /> precursor intermediates. By comparison, leaves, amino                supplied exogenously. These results suggest high sen- <br /> acids showed significant changes (Table&#194;&#160; 2), consistent               sitivity dhad Na+ stress attributed reduced <br /> the indistinguishable phenotypes mutants WT                     level Ile (Fig.&#194;&#160;4D&#226;&#8364;&#8220;G). <br /> (Supplementary Figure S1B). Taken together, results                     The hypersensitive response salinity stress be <br /> indicate decreased expression DHAD resulted                    directly derived salt ionic toxicity, alternatively <br /> in partial deficiency BCAA biosynthesis, subsequent                 represent universal reaction osmotic stress. To ascer- <br /> impairment root development.                                            tain possibilities, dhad knockdown mutants were <br />  <br /> Decreased expression DHAD resulted increased <br /> sensitivity salt&#194;&#160;stress <br /> Previous studies shown accumulation BCAAs <br /> was greatly elevated abiotic stresses (Joshi Jander, <br /> 2009; Joshi et&#194;&#160;al., 2010), suggesting BCAA homeostasis <br />  <br />  <br /> Table&#194;&#160;2.&#226;&#8364;&#8218; Amino acid contents roots leaves WT and <br /> dhad-2a,b,c <br />  <br />  <br /> Amino acid Roots                            Leaves <br />               WT            dhad-2          WT             dhad-2 <br /> Alanine        13.76&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.49 12.43&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.15    14.85&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.32   15.31&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;1.32 <br /> Arginine        0.54&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.02   0.45&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.03    2.11&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.54    2.61&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.18 <br /> Asparagine     24.31&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;2.25 33.74&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;1.19 31.16&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.68    27.27&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;1.51 <br /> Aspartate       9.38&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.15 11.96&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.26 24.77&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.73    26.32&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.52 <br /> Glutamine      38.32&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.05 44.99&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;1.26 96.08&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.14    91.23&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;1.71 <br /> Glutamate     230.65&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.42 272.93&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;11.68 631.25&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;38.44 562.25&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;14.34 <br /> Glycine         3.65&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.15   3.14&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.02   20.61&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;4.48    16.3&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.71 <br /> Histidine       1.33&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.03   1.42&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.06    1.42&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.12    1.11&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.11 <br /> Isoleucine      0.93&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.01   0.66&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.03 b  0.49&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.01    0.46&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.01 <br /> Leucine         1.35&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.04   0.96&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.04 b  0.66&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.01    0.55&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.05 <br /> Lysine           1.1&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.01   0.91&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.01 b  0.53&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.03    0.61&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.10 <br /> Methionine      3.19&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.07   2.78&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.07    1.01&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.07    0.79&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.02 <br /> Phenylalanine   0.51&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.03   0.47&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.01    0.64&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.03    0.73&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.02 <br /> Proline        16.22&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;2.00 18.55&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.5     21.03&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.79   25.08&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;3.00 <br /> Serine         15.74&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;1.38   19.2&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.4    37.95&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.57   42.81&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;2.20 <br /> Threonine       6.71&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.17    9.8&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.32  5.48&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.03    6.54&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.59 <br /> Tyrosine        0.46&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.02   0.39&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.03    0.31&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.02    0.31&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.01 <br /> Valine          2.64&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.01   2.21&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.05 b  2.27&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.04    2.13&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.15    Fig.&#194;&#160;4.&#226;&#8364;&#8218; dhad-1 dhad-2 hypersensitive salt stress. (A&#226;&#8364;&#8220;F) <br />                                                                            10-day-old WT, dhad-1, dhad-2 seedlings grown half MS medium <br />   a <br />    Values significantly higher WT (P&#194;&#160;&lt;&#194;&#160;0.05), Student&#226;&#8364;&#8482;s t-test.   supplemented different levels NaCl (A&#226;&#8364;&#8220;C) 0.1mM Ile (A&#226;&#8364;&#8220;F). Bar, <br />   b <br />    Values significantly lower WT (P&#194;&#160;&lt;&#194;&#160;0.05), Student&#226;&#8364;&#8482;s t-test.    10&#226;&#8364;&#8240;mm. (G) Root length mutants measured shown a <br />  c <br />    Amino acid contents given pmol mg FW&#226;&#8364;&#8220;1 (FW, fresh                percentage relative WT grown conditions. Error <br /> weight). Data mean &#194;&#177; SD replicates.                       bars indicate standard error. This figure available colour JXB online. <br />                                                                          Molecular characterization DHAD Arabidopsis&#226;&#8364;&#8218;|&#226;&#8364;&#8218;885 <br />  <br /> treated increased concentrations mannitol. As shown <br /> in Fig.&#194;&#160; 5, low concentration mannitol, root <br /> length dhad mutants 72&#226;&#8364;&#8220;77% WT, which <br /> is comparable 68&#226;&#8364;&#8220;71% treatment (Fig.&#194;&#160;3A <br /> and 3I). However, increased concentrations manni- <br /> tol, ratio root length mutant WT was <br /> increased 84&#226;&#8364;&#8220;89%, clearly different salt <br /> response, indicating exaggerated effect <br /> under osmotic stress. Therefore, conclude hypersen- <br /> sitivity dhad salinity stress attributed salt <br /> ionic toxicity. <br />    The inhibition root growth dhad, salt stress, might <br /> also reflect common response various abiotic stresses. To <br /> test possibility, root-growth assays conducted <br /> under different concentrations Ni2+ treatment, which <br /> represents type heavy metal stress. As shown Fig.&#194;&#160;6, <br /> the mutants did exhibit signs high sensitivity to <br /> the increased concentrations Ni2+. In contrast, ratios <br /> of root length mutants WT slightly <br /> increased elevated concentrations Ni2+. This sug- <br /> gests high sensitivity salt stress dhad quite <br /> specific. <br />  <br /> The lib mutant shows <span id='am-317' about='xsp:pattern' typeof='owl:Thing'>pattern</span> toward <br /> increased sensitivity salt&#194;&#160;stress <br /> To establish functional role Ile biosynthesis in <br /> tolerance salinity stress, root-growth assays con- <br /> ducted lib (Yu et&#194;&#160;al., 2013). As shown Fig.&#194;&#160;7, similar <br /> to dhad alleles, lib shows higher sensitivity increased              Fig.&#194;&#160;6.&#226;&#8364;&#8218; dhad-1 dhad-2 hypersensitivity Ni2+ stress. (A&#226;&#8364;&#8220;F) <br />                                                                             10-day-old WT, dhad-1, dhad-2 seedlings grown half MS medium <br /> salt concentration, hypersensitivity rescued              supplemented different levels NiSO4 (A&#226;&#8364;&#8220;C) 0.1mM Ile (A&#226;&#8364;&#8220;F). <br /> by exogenous application Ile, unambiguously indicating                   Bar, 10mm. (G) Root length mutants measured shown a <br /> that Ile biosynthetic pathway plays critical role salt             percentage relative WT grown conditions. Error <br /> tolerance.                                                                  bars indicate standard error. This figure available colour JXB online. <br />  <br />  <br />  <br />  <br /> Fig.&#194;&#160;5.&#226;&#8364;&#8218; dhad-1 dhad-2 hypersensitivity mannitol stress. (A&#226;&#8364;&#8220;C) 10-day-old WT, dhad-1, dhad-2 seedlings grown half MS <br /> medium supplemented different levels mannitol. Bar, 10&#226;&#8364;&#8240;mm. (D) Root length mutants measured shown percentage relative to <br /> that WT grown conditions. Error bars indicate standard error. This figure available colour JXB online. <br />  886&#226;&#8364;&#8218;|&#226;&#8364;&#8218;Zhang et&#194;&#160;al. <br />  <br />                                                                                   requirements transported sporophytic into <br />                                                                                   reproductive tissues; (iii) potential cellular toxicities <br />                                                                                   resulting abnormal accumulation biosynthetic <br />                                                                                   intermediates having different effects plant development. <br />                                                                                      The decreased expression DHAD resulted simul- <br />                                                                                   taneous reduction levels BCAAs roots, <br />                                                                                   leaves (Table&#194;&#160;2), indicating DHAD exert <br />                                                                                   rate-limiting control regulating BCAA biosynthesis a <br />                                                                                   tissue-specific manner. However, possibility exist- <br />                                                                                   ence alternative pathway BCAA biosynthesis in <br />                                                                                   leaves completely ruled study. In addi- <br />                                                                                   tion, retarded root growth rescued the <br />                                                                                   exogenous application Ile, Leu Val, suggesting that <br />                                                                                   impaired root growth probably caused the <br />                                                                                   deficiency Ile accumulation. This result consistent with <br />                                                                                   results recent study mutation threonine <br />                                                                                   deaminase/dehydratase catalysing commit- <br />                                                                                   ted step Ile biosynthesis resulted partial Ile deficiency, <br />                                                                                   consequently retarded root development (Yu et&#194;&#160;al., 2013). <br />                                                                                   Therefore, clear Ile homeostasis plays important <br />                                                                                   role regulating root development Arabidopsis. Moreover, <br />                                                                                   perturbation DHAD lead buildup meta- <br />                                                                                   bolic intermediates. For instance, previously shown <br />                                                                                   inhibition AHAS cause toxic accumulation <br />                                                                                   ketobutyrate Salmonella (LaRossa, 1987). However, in <br />                                                                                   study, hint cellular toxicity caused 2-oxobu- <br />                                                                                   tanoate suggested, exogenous application of <br />                                                                                   substantial amounts 2-ketoburate didn&#226;&#8364;&#8482;t produce dis- <br />                                                                                   tinct response dhad relative WT. Furthermore, we <br />                                                                                   observed concentrations amino acids <br />                                                                                   affected, including increased levels asparagine, aspar- <br /> Fig.&#194;&#160;7.&#226;&#8364;&#8218; The low isoleucine biosynthesis (lib) mutant hypersensitive salt   tate, glutamate, threonine, reduced level of <br /> stress. (A&#226;&#8364;&#8220;F) 10-day-old WT lib seedlings grown half MS medium              lysine. As precursor amino acid Ile, accumulation <br /> supplemented different levels NaCl (A&#226;&#8364;&#8220;C) 0.1mM Ile (A&#226;&#8364;&#8220;F). Bar,        threonine simply explained potential build-up <br /> 10&#226;&#8364;&#8240;mm. (G) Root length mutants measured shown a <br /> percentage relative WT grown conditions. Error <br />                                                                                   blockage downstream metabolic flux. Similarly, <br /> bars indicate standard error. This figure available colour JXB online.   BCAAs aspartate-derived amino acids, therefore <br />                                                                                   reduction BCAAs lead build-up aspar- <br />                                                                                   tate, subsequently recruited asparagine. However, <br /> Discussion <br />                                                                                   way glutamine increased lysine decreased is <br /> As essential components protein synthesis                      unclear, suggesting biosynthesis different amino <br /> important signalling molecules, BCAAs thought                        acids regulation complex crosstalk, the <br /> regulate numerous aspects plant growth required plant                      precise change leading imbalance different amino <br /> <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>survival</span>. However, little information available spe-                     acids deserves investigation. In lib mutant, addi- <br /> cific roles BCAAs plant growth development                      tion reduced level Ile, amino acids <br /> as interaction environment. In recent years, extensive                   showed statistically significant changes abundance; these <br /> studies Arabidopsis led identification of                  increase phenylalanine decrease lysine. <br /> the genes involved BCAA biosynthesis. To date,                        The consistent reduction Ile content lib dhad <br /> genes, KARI DHAD, remain functionally uncharacter-                            indicates genes involved Ile synthesis, the <br /> ized. In study, identified null mutations DHAD                    distinct response amino acids impair- <br /> caused partial abortion male female gametogen-                        ment different steps biosynthetic pathway <br /> esis. This reminiscent loss function LeuD3                    suggests metabolic network associated step <br /> IPMDH2/IPMDH3, involved Leu biosynthesis,                            slightly dissimilar. Taken together, possibil- <br /> lethality detected female male gamete development,                      ity stunted root growth consequence inter- <br /> respectively (He et&#194;&#160; al., 2009; He et&#194;&#160; al., 2010). The underly-                   mediate accumulation completely ruled out, we <br /> ing causes lethality distinct organs different mutants                   speculate root phenotype likely caused by <br /> include (i) different organs having exceptional requirements                      Ile deficiency. <br /> for novo biosynthesis certain types BCAAs; (ii)                         Under abiotic stresses, levels amino acids <br /> different biosynthetic intermediates having dissimilar abilities                  significantly increased. Proline biosynthesis been <br />                                                                Molecular characterization DHAD Arabidopsis&#226;&#8364;&#8218;|&#226;&#8364;&#8218;887 <br />  <br /> well studied shown play important roles plant           Acknowledgements <br /> tolerance various stresses Liu Zhu, 1997;              We thank members laboratories helpful assistance the <br /> Verbruggen Hermans, 2008; Mattioli et&#194;&#160;al., 2009; Hayat        research critical comments manuscript. Special thanks Dr <br /> et&#194;&#160; al., 2012). Disruption proline biosynthesis renders        Dong Liu TsingHua University, China, kind offer lib mutant seeds. <br /> plants hypersensitive salinity stress (Liu Zhu, 1997; <br /> Mattioli et&#194;&#160; al., 2009). Interestingly, long rec- <br /> ognized accumulation BCAAs exhib-           References <br /> ited significant induction proline cases <br />                                                                   Alexander M. 1969. Differential staining aborted nonaborted <br /> of osmotic stresses, implying BCAAs play          pollen. Biotechnic Histochemistry 44, 117&#226;&#8364;&#8220;122. <br /> role plant stress responses (Joshi Jander, 2009; Joshi     Alonso JM, Stepanova AN, Leisse TJ, Kim CJ, Chen H, Shinn P, <br /> et&#194;&#160;al., 2010). However, functional evidence obtained       Stevenson DK, Zimmerman J, Barajas P, Cheuk R. 2003. Genome- <br /> to draw conclusion. In study,       wide insertional mutagenesis Arabidopsis thaliana. Science 301, <br />                                                                   653&#226;&#8364;&#8220;657. <br /> decreased expression DHAD led reduction BCAA <br />                                                                   Binder S. 2010. Branched-chain amino acid metabolism Arabidopsis <br /> levels rendered plants hypersensitive salinity stress      thaliana. The Arabidopsis Book 8, e0137. <br /> (Table&#194;&#160;2 Fig.&#194;&#160;4), providing time strong evi-    Clough SJ, Bent AF. 1998. Floral dip: simplified metho <br /> </body></html>