<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                      Meta Gene 2 (2014) 819&#226;&#8364;&#8220;830 <br />  <br />  <br />                                              Contents lists available ScienceDirect <br />  <br />                                                           Meta Gene <br />  <br />                                                          journal homepage: <br />  <br />  <br />  <br />  <br /> let-65 cytoplasmic methionyl tRNA synthetase <br /> in C. elegans <br /> Maha Z. Alriyami a,c,&#226;?&#381;, Martin R. Jones b, Robert C. Johnsen a, <br /> Yajnavalka Banerjee c, David L. Baillie a <br /> a <br />     Department Molecular Biology Biochemistry, Simon Fraser University Burnaby, 8888 University Drive, Burnaby, BC V5A 1S6 Canada <br /> b <br />     Genome Sciences Centre, British Columbia Cancer Research Centre, 600 West 10th Avenue, Vancouver, BC V5Z 4E6, Canada <br /> c <br />     Department Biochemistry, College Medicine Health Sciences, Sultan Qaboos University, 35, Al-Khod 123, Oman <br />  <br />  <br />  <br />  <br /> a r t c l e            n f o                         b s t r c t <br />  <br /> Article history:                                         Cytoplasmic methionyl tRNA synthetase (MetRS) 20 <br /> Received 13 September 2013                               cytoplasmic aminoacyl tRNA synthetase enzymes (ARS). This family of <br /> 8 August 2014                                            enzymes catalyzes process fundamental protein translation. <br /> Accepted 11 August 2014 <br />                                                          Using combination genetic mapping, oligonucleotide array compar- <br /> Available online xxxx <br />                                                          ative genomic hybridization, phenotypic correlation, that <br />                                                          mutations essential gene, let-65, reside predicted <br /> Keywords: <br />                                                          Caenorhabditis elegans homologue MetRS, named <br /> Caenorhabditis elegans <br /> let-65 <br />                                                          mars-1. We demonstrate lethality associated alleles of <br /> Aminoacyl tRNA synthetase, methionyl-tRNA                let-65 fully rescued transgenic array spans mars-1 geno- <br /> synthetase                                               mic region. Furthermore, sequence analysis reveals let-65 alleles <br /> Oligonucleotide array comparative genomic                lead alteration highly conserved amino acids. <br /> hybridization                                              &#194;&#169; 2014 The Authors. Published Elsevier B.V. This open access <br /> Essential genes                                                                              article CC BY-NC-ND license <br />                                                                              (http://creativecommons.org/licenses/by-nc-nd/4.0/). <br />  <br />  <br /> Introduction <br />  <br />    Analysis mutations affecting genes essential <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span>survival</span> model organisms, the <br /> nematode Caenorhabditis elegans, instrumental developing understanding fundamental <br /> biological processes, example cell division morphology (Green et al., 2011). Many of <br /> these processes direct relevance understanding human diseases metabolic disorders <br /> and obesity (Hashmi et al., 2013). The generation mutations essential genes correlation a <br /> speci&#239;&#172;?c genetic locus is, therefore, &#239;&#172;?rst step understanding nature gene&apos;s function promoting <br /> survival. <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> easily identify genes essential survival model organism C. elegans, we <br /> created physical de&#239;&#172;?ciency map encompasses portion Chromosome IV, allowing correlation <br />  <br />  &#226;?&#381; Corresponding author at: Department Biochemistry, College Medicine Health Sciences, Sultan Qaboos University, Oman, 35, <br /> Al-Khod 123, Oman. Tel.: +968 24141152; fax: +968 2441 3880. <br />     E-mail address: mahazr@squ.edu.om (M.Z. Alriyami) <br />  <br />  <br /> http://dx.doi.org/10.1016/j.mgene.2014.08.006 <br /> 2214-5400 &#194;&#169; 2014 The Authors. Published Elsevier B.V. This open access article CC BY-NC-ND license <br /> (http://creativecommons.org/licenses/by-nc-nd/4.0/). <br />  820                                  M.Z. Alriyami et al. / Meta Gene 2 (2014) 819&#226;&#8364;&#8220;830 <br />  <br />  <br /> of genetically de&#239;&#172;?ned mutations physically de&#239;&#172;?ned candidate gene lists. In study, the <br /> use physical de&#239;&#172;?ciency map identify essential gene (let-65), previously isolated ethyl <br /> methanesulfonate (EMS) genetic screens (Clark Baillie, 1992; Clark et al., 1988; Moerman Baillie, <br /> 1981; Rogalski Baillie, 1985; Rogalski et al., 1982). <span id='am-1' about='protege:FROM' typeof='owl:Thing'>From</span> analysis determined mutations in <br /> let-65 correspond methionyl tRNA synthetase gene, mars-1. <br />     Organisms utilize key components translate genetic information proteins: ribosomes, messenger <br /> RNAs (mRNA), transfer RNAs (tRNA), aminoacyl-tRNA synthetases (ARS), reviewed Brown et al. (2010). <br /> Ribosomes two-subunit organelles direct synthesis proteins translating nucleic acid triplets, carried <br /> by mRNAs, amino acids (Crick et al., 1961). mRNAs, generated DNA transcription, act trans- <br /> lation templates, tRNAs act interfaces mRNA strands amino acids added poly- <br /> peptide chain transferring required amino acids growing chain (Crick et al., 1961). <br />     The ARS family proteins encompasses 20 multi-domain enzymes. Each proteins <br /> speci&#239;&#172;?cally recognizes amino acid corresponding subset cognate tRNA molecules catalyze <br /> the aminoacylation two-step reaction. In &#239;&#172;?rst step speci&#239;&#172;?c amino acid <br /> is activated ATP produce aminoacyl adenylate; second step amino acid transferred onto <br /> the 3&#226;&#8364;&#178;-end tRNA (Jakubowski, 2001). ARS categorized classes (I II) based their <br /> distinct core domains (Wolf et al., 1999). The catalytic domain Class I synthetases exhibit typical Rossman <br /> fold, three-layer topology (&#206;&#177;/&#206;&#178;/&#206;&#177;) inner core &#239;&#172;?ve parallel beta strands. However, Class <br /> II synthetases generally consist &#206;&#178; strands (Berg et al., 2001). Class I II synthetases recognize <br /> different faces tRNA molecules (Berg et al., 2001). The tRNA CCA terminus (containing amino acid <br /> attachment site) adopts different conformations tRNA-synthetase complexes class. The CCA termini <br /> of Class I enzymes hairpin conformations, whereas, class II enzymes helical con&#239;&#172;?rmations <br /> observed free tRNAs (Berg et al., 2001). An intriguing enzyme belongs family Class I <br /> synthetase, methionyl tRNA synthetase (MetRS), recognizes initiator tRNA the <br /> methionine carrying tRNA functions elongation peptide chains (Deniziak Barciszewski, <br /> 2001). <br />     In study, use oligonucleotide array Comparative Genomic Hybridization (aCGH) de&#239;&#172;?ne phys- <br /> ical de&#239;&#172;?ciency map region Chromosome IV (LGIV) nematode C. elegans. We use map to <br /> assign precisely de&#239;&#172;?ned lists candidate genes previously generated genetically mapped mutations, <br /> which confer lethal <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>phenotype</span> organism. Using approach, demonstrate lethality <br /> associated alleles let-65 fully rescued transgenic array spans mars-1 genomic region. <br />  <br /> Materials methods <br />  <br /> Growth handling C. elegans <br />  <br />    Nematode strains cultured previously described (Brenner, 1974). Strains used study: N2 <br /> Bristol (BC00049), dpy-5(e907) (CB00907), let-65(s254); nT1(IV) (BC00962), let-65(s694); nT1(IV) <br /> (BC1121), let-65(s1154); nT1(IV) (BC2028), let-65(s1222); nT1(IV) (BC2116), let-65(s1083); nT1(IV) <br /> (BC1909), let-65(s1084); nT1(IV) (BC1910), let-65(s1730); nT1(IV) (BC3258), let-65(s1777); nT1(IV) <br /> (BC3305), let-65(s1083), unc-22(s7)(IV) [WRM0615dH10 + myo-2::GFP + pCeh361] (BC8656), let-65(s1222), <br /> unc-22(s7) (IV) [WRM0615dH10 + myo-2::GFP + pCeh361] (BC8696). <br />  <br /> Oligonucleotide array comparative genomic hybridization (aCGH) data analysis <br />  <br />       aCGH experiments analysis performed previously described (Jones et al., 2007). <br />  <br /> DNA sequence analysis <br />  <br />    Genomic DNA extracted let-65 homozygous animals alleles (s1730, s1083, s694, s1084, <br /> s254, s1154, s1777 s1222). PCR fragments spanning mars-1 ORF ampli&#239;&#172;?ed genomic DNA <br /> templates primers designed using Primer 3 (Rozen Skaletsky, 2000) (primer sequences available <br /> upon request). The ampli&#239;&#172;?ed templates subsequently sequenced (Macrogen, Korea) appropriate <br /> sequencing primers (Supplementary Table 1). <br />                                            M.Z. Alriyami et al. / Meta Gene 2 (2014) 819&#226;&#8364;&#8220;830                                  821 <br />  <br />  <br /> Transgenic rescue <br />  <br />    The fosmid WRM0615dH10, spans mars-1 genomic region (11,631,182 11,665,626 bp on <br /> LGIV), injected syncytial gonads dpy-5 (e907) animals &#239;&#172;?nal concentration 3 ng/&#206;&#188;l, <br /> along pharyngeal targeted green &#239;&#172;&#8218;uorescent protein (GFP) reporter myo-2::GFP 10 ng/&#206;&#188;l and <br /> the plasmid pCeh361,which contains wild-type dpy-5 rescues dpy-5 phenotype, 100 ng/&#206;&#188;l <br /> (Thacker et al., 2006). Injected worms plated (&#239;&#172;?ve P0s plate) phenotypically wild-type animals <br /> from F1s generation expressed pharyngeal GFP reporter individually isolated propagated. <br /> GFP positive wild-type F2 animals transmitting transgenic array selected PCR speci&#239;&#172;?c the <br /> fosmid backbone performed con&#239;&#172;?rm presence fosmid (Supplementary Table 1). A single, stably <br /> transmitting, animal selected create transgenic lines. To conduct transgenic rescue experiments indi- <br /> vidual hermaphrodites carrying WRM0615dH10 + myo-2::GFP + pCeh361 crossed +/+ N2 males to <br /> obtain dpy-5 (e907)/+[WRM0615dH10 + myo-2::GFP + pCeh361] males. These GFP-expressing males were <br /> then crossed let-65 (s1222); unc-22 (s7) let-65 (s1803); unc-22 (s7) heterozygously maintained <br /> hermaphrodites. F1 progeny screened presence males ensure cross successful <br /> and F1 wild-type hermaphrodites plated individually. F2 animals individually plated screened for <br /> unconditional twitchers, indicative viable homozygous animals genotype let-65 (s1083 or <br /> s1222) unc-22 (s7)/let-65 (s1083 s1222) unc-22 (s7)(IV)[WRM0615dH10 + myo2::GFP + pCeh361]. <br /> Unconditional twitchers individually plated incubated 20 &#194;&#176;C assess rescue. <br />  <br /> Microscopy <br />  <br />    Screening transgenic animals performed using Zeiss Axioscope (Quorum Technologies) set a <br /> QImaging Camera appropriate &#239;&#172;?lter sets used GFP expression analysis. Animals were <br /> immobilized 100 mM sodium azide (in water) immediately prior imaging. <br />  <br /> Bioinformatics studies <br />  <br />    All reference DNA sequences derived WormBase release WS208. Analysis sequence data and <br /> image processing performed using BioEdit (http://www.mbio.ncsu.edu/BioEdit/BioEdit.html), <br /> WolFPSORT (Horton et al., 2007) (http://wolfpsort.org/), BLASTP. <br />  <br /> Results <br />  <br /> aCGH mapped de&#239;&#172;?ciencies physically de&#239;&#172;?ne seven zones unc-22 region LGIV <br />  <br />     In effort characterize genes essential survival C. elegans previously generated and <br /> mapped mutations confer <span id='am-394' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-395' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-397' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-398' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-399' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-400' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-401' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-402' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-403' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-404' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-405' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-406' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-407' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-408' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-409' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-410' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-411' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-412' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-413' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-414' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-415' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-416' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-417' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-418' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-419' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-420' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-421' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-422' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-423' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-424' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-425' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-426' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-427' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-428' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-429' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-430' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-431' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-432' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-433' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-434' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-435' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-436' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-437' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-438' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-439' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-440' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-441' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-442' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-443' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-444' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-445' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-446' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-447' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-448' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-449' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-450' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-451' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-452' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-453' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-454' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-455' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-456' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-457' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-458' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-459' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-460' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-461' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-462' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-463' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-464' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-465' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-466' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-467' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-468' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-469' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-470' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-471' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> unc-22 region C. elegans genome (Rogalski <br /> et al., 1982). The unc-22 region (from unc-43 unc-31) represents approximately map units (mu) on <br /> chromosome IV, spanned 35 well-de&#239;&#172;?ned genetic de&#239;&#172;?ciencies (Clark Baillie, 1992; Clark <br />  <br /> Table 1 <br /> aCGH de&#239;&#172;?ned de&#239;&#172;?ciency breakpoints gene complement. <br />  <br />   Zone        Left breakpoint                      Right breakpoint                     Genes zones         Size kb <br />  <br />               Left (bp)         Right (bp)         Left (bp)          Right (bp)        Leftmost       Rightmost <br />  <br />   1           10,942,926        10,947,094         11,070,201         11,076,154        Y69E1A.1       Y5F2A.3     123&#226;&#8364;&#8220;134 <br />   2           11,070,201        11,076,154         11,629,649         11,629,799        Y5F2A.4        lys-6       553&#226;&#8364;&#8220;560 <br />   3           11,629,649        11,629,799         11,639,511         11,639,570        F58B3.4        F58B3.7a     10&#226;&#8364;&#8220;11 <br />   4           11,639,511        11,639,570         11,983,621         11,983,655        F58B3.7a       unc-22a     343&#226;&#8364;&#8220;344 <br />   5           11,983,621        11,983,655         12,243,159         12,251,414        unc-22a        sre-16      252&#226;&#8364;&#8220;253 <br />   6           12,243,159        12,251,414         12,551,447         12,553,668        sre-19         ZK795.1a    315&#226;&#8364;&#8220;317 <br />   7           12,551,447        12,553,668         12,757,572         12,776,111        ZK795.1a       inx-6       196&#226;&#8364;&#8220;204 <br />  <br /> Regions annotated based probe sequence falling adjacent breakpoint position. <br />  a <br />     Breakpoints falling single gene. <br />  822                                  M.Z. Alriyami et al. / Meta Gene 2 (2014) 819&#226;&#8364;&#8220;830 <br />  <br />  <br /> et al., 1988; Marra Baillie, 1994; Moerman Baillie, 1981; Rogalski Baillie, 1985; Rogalski et al., <br /> 1982; Schein et al., 1993). Previously, genetic analysis used identify position 31 lethal gene loci <br /> within genetic de&#239;&#172;?ciency map (Clark et al., 1988). However, precise molecular extent of <br /> the de&#239;&#172;?ciencies known construction de&#239;&#172;?nitive candidate gene lists corresponded to <br /> each mutated loci possible. <br />     To physically de&#239;&#172;?ne de&#239;&#172;?ciency map unc-22 region analyzed informative <br /> de&#239;&#172;?ciencies using array comparative genomic hybridization (aCGH). aCGH method used compare <br /> the DNA ratio individual samples organism order determine copy number var- <br /> iations chromosomal genome-wide scale (Dhami et al., 2005). This method used determine <br /> the precise physical extent genetic de&#239;&#172;?ciencies C. elegans (Jones et al., 2007). In cases, array data <br /> was suf&#239;&#172;?cient position de&#239;&#172;?ciency deletion breakpoints single-gene resolution approximately <br /> 6 kb (Table 1). Deletion breakpoints fell single genes genes. Using data a <br /> seven-zone physical de&#239;&#172;?ciency map constructed spanning approximately 800 kb unc-22 region <br /> (Fig. 1A Table 1). Furthermore, incorporating previously generated genetic mapping data mo- <br /> lecularly unidenti&#239;&#172;?ed lethal loci known map region, able assign locus precisely <br /> de&#239;&#172;?ned list candidate genes. The largest number lethal loci map zone de&#239;&#172;?ciency map <br /> (nine genes). Zone contains seven lethal loci, zones seven contain lethal loci each. <br /> Zones &#239;&#172;?ve contain lethal loci each. Finally, single lethal locus positioned zone <br /> three (Fig. 1A). <br />  <br /> Sequence analysis let-65 <br />  <br />     let-65 single lethal loci mapped zone de&#239;&#172;?ciency map (Fig. 1A). Based the <br /> physical extent zone annotated genes, F58B3.4, F58B3.5, F58B3.6 F58B3.7 candi- <br /> dates let-65. let-65 represented alleles, isolated EMS screens larval lethal- <br /> ity (Clark Baillie, 1992; Clark et al., 1988; Rogalski Baillie, 1985; Rogalski et al., 1982). Since mutations <br /> in let-65 confer lethal phenotype correlated available RNAi data order rank candidate genes <br /> based severity reported RNAi phenotype (Kamath et al., 2003; Maeda et al., 2001; Piano <br /> et al., 2002; Simmer et al., 2003; Sonnichsen et al., 2005) (Fig. 1B). Using approach identi&#239;&#172;?ed <br /> F58B3.5 F58B3.4 strongest candidate genes. To identify mutations F58B3.5 F58B3.4 <br /> their genomic regions ampli&#239;&#172;?ed PCR let-65 (s254) homozygous animals sequenced. A <br /> comparison sequence data reference C. elegans genome (WormBase release WS208) revealed no <br /> mutations associated F58B3.4. However, <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span>point mutation</span> identi&#239;&#172;?ed coding region <br /> of F58B3.5. This <span id='am-316' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-317' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-320' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-321' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-323' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-327' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-330' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-333' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-335' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-336' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-338' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-356' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-357' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-362' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-370' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-371' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-372' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-374' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-376' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-377' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-378' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-381' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-382' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-383' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-384' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-386' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-387' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-388' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-390' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span>mutation</span> resulted glutamic acid lysine substitution (Glu364Lys) highly conserved <br /> amino acid (Fig. 2 Table 2). To provide additional con&#239;&#172;?rmation mutation F58B3.5 associated <br /> with let-65 sequenced F58B3.5 ORF remaining let-65 alleles, detecting presence additional <br /> mutations seven alleles tested. All mutations identi&#239;&#172;?ed G &#226;&#8224;&#8217; C A &#226;&#8224;&#8217; T transitions, con- <br /> sistent known mutational spectrum ethane methylsulfonate (EMS) mutagenesis (Fig. 2, Table 2) <br /> (Maple Moller, 2007). In case, exception s1777 s1222, identi&#239;&#172;?ed mutations were <br /> substitutions highly conserved amino acids predicted MetRS domain F58B3.5 (Figs. 1C 2). The <br /> lesion s1777 resulted 508 amino acid truncation, s1222 contained mutation &#239;&#172;?rst <span id='am-472' about='xsp:base' typeof='owl:Thing'>base</span> the <br /> &#239;&#172;?rst intron. s1222 potentially prevents splicing &#239;&#172;?rst intron, result premature trunca- <br /> tion protein amino acid 28 (Fig. 1C). Together data demonstrate let-65 F58B3.5 repre- <br /> sent gene. F58B3.5 encodes cytoplasmic Class I synthetase, methionyl tRNA synthetase (MetRS) <br /> that evolutionary conserved, 58% identity 71% similarity human homologue (BlastP <br /> alignment Altschul et al., 1997, 2005) (Fig. 2). We renamed F58B3.5 mars-1 (Methionyl Amino-acyl <br /> tRNA Synthetase-1). <br />  <br /> Rescue let-65 fosmid containing mars-1 <br />  <br />    We shown mutations mars-1 present let-65 animals. To provide evidence that <br /> these mutations caused lethality associated let-65, performed transgenic rescue experiments. <br /> The fosmid clone WRM0615dH10 spans entire mars-1 genomic region, excludes F58B3.4 ORF <br /> (Fig. 1B). We transgenic animals carrying WRM0615dH10 extra-chromosomal array <br />                                              M.Z. Alriyami et al. / Meta Gene 2 (2014) 819&#226;&#8364;&#8220;830                                         823 <br />  <br />  <br /> A <br />  <br />  <br />  <br />  <br /> B <br />  <br />  <br />  <br />  <br /> C <br />  <br />  <br />  <br />  <br /> Fig. 1. Physical map unc-22 region LGIV including aCGH de&#239;&#172;?ciency mapping data. A. Schematic LGIV showing regions <br /> uncovered de&#239;&#172;?ciencies analyzed study shown &#239;&#172;?gure (dark gray). aCGH data region four <br /> analyzed de&#239;&#172;?ciencies shown. Commonly used genetic markers physical positions de&#239;&#172;?ciency breakpoints (estimated <br /> to 0.5 kb) shown array data. A schematic describing seven zones de&#239;&#172;?ned breakpoints de&#239;&#172;?ciencies is <br /> shown array data. let-56 zone &#239;&#172;?ve zone six. B. An expansion zone showing candidate genes for <br /> let-65. The region covered fosmid used complementation tests depicted gene models. C. Schematic showing protein <br /> structures human MetRS C. elegans MARS-1. Conserved domains locations indicated. The positions identi&#239;&#172;?ed <br /> mars-1 mutations shown. Figure scale. <br />  824                                        M.Z. Alriyami et al. / Meta Gene 2 (2014) 819&#226;&#8364;&#8220;830 <br />  <br />  <br />  <br />  <br /> Fig. 2. C. elegans MARS-1 alignment human (GenBank accession no. NP_004981), D. melanogaster (GenBank accession no. <br /> NP_611382), S. cerevisiae (GenBank accession no. CAA97293) orthologs showing conserved amino acids identi&#239;&#172;?ed let-65 <br /> mutations. MARS-1 evolutionarily conserved 58% identity 71% similarity human ortholog. Red lines indicate the <br /> positions amino acid changes let-65 alleles. s1222 prevents splicing &#239;&#172;?rst intron. s1777 induces stop codon prema- <br /> turely truncates protein. <br />                                             M.Z. Alriyami et al. / Meta Gene 2 (2014) 819&#226;&#8364;&#8220;830                        825 <br />  <br />  <br /> Table 2 <br /> Mutations identi&#239;&#172;?ed let-65 sequencing BioEdit analysis. <br />  <br />   Allele                       Nucleotide change                         Codon mutation            Amino acid change <br />  <br />   s174                         ND                                        ND                        NDa <br />   s254                         G1188A                                    gag &#226;&#8224;&#8217; aag                 Glu-364 &#226;&#8224;&#8217; Lys <br />   s694                         G655A                                     gga &#226;&#8224;&#8217; gaa                 Gly-186 &#226;&#8224;&#8217; Glu <br />   s1083                        G429A                                     gga &#226;&#8224;&#8217; gaa                 Gly-126 &#226;&#8224;&#8217; Glu <br />   s1084                        G921A                                     gga &#226;&#8224;&#8217; aga                 Gly-275 &#226;&#8224;&#8217; Arg <br />   s1154                        C1297T                                    ccc &#226;&#8224;&#8217; ctc                 Pro-400 &#226;&#8224;&#8217; Leu <br />   s1222                        G81A                                      (&#226;&#710;&#8217;) &#226;&#8224;&#8217; aga                 (&#226;&#710;&#8217;)27 &#226;&#8224;&#8217; Arg <br />   s1730                        C174T                                     gct &#226;&#8224;&#8217; gtt                 Ala-41 &#226;&#8224;&#8217; Val <br />   s1777                        G1324A                                    tgg &#226;&#8224;&#8217; tag                 Trp-409 &#226;&#8224;&#8217; Stop <br />  a <br />      Not determined &#226;&#8364;&#8221; potentially complex lesion. <br />  <br />  <br /> successfully speci&#239;&#172;?cally rescued lethality associated alleles let-65 tested (s1083 s1222). <br /> Furthermore transgenic animals super&#239;&#172;?cially wild type appearance twitching pheno- <br /> type presence unc-22 (s7) homozygous mutation, used visible phenotypic marker. <br /> Individual viable animals isolated start strains BC8656 BC8696 alleles s1083 s1222 respec- <br /> tively, cultured multiple generations con&#239;&#172;?rm continued rescue lethal phenotype. Together the <br /> sequence rescue data con&#239;&#172;?rm mutation mars-1 gives rise lethal phenotype let-65 animals. <br />  <br /> Mitochondrial methionyl tRNA synthetase candidate <br />  <br />     MARS-1 predicted encode cytoplasmic form MetRS (Havrylenko et al., 2010). Computa- <br /> tional analysis predicted MARS-1 protein (WormBase WS209) demonstrated mars-1 encodes a <br /> protein localizes subcellularly cytoplasm. BLASTP analysis Y105E8A.20 (Altschul et al., 1997), <br /> a second gene predicted encode methionyl tRNA synthetase C. elegans (WormBase WS211), implies <br /> mitochondrial localization encoded protein. <br />  <br /> Discussion <br />  <br />     In study used aCGH de&#239;&#172;?ne physical de&#239;&#172;?ciency map 2 Mb portion LGIV C. elegans. <br /> Using de&#239;&#172;?ciency map correlated physical candidate gene lists mutations number previously <br /> identi&#239;&#172;?ed essential genes (Clark, 1990; Clark Lyckegaard, 1988; Moerman, 1980; Rogalski Baillie, <br /> 1985; Rogalski et al., 1982). From analysis able determine genetically identi&#239;&#172;?ed <br /> locus let-65 corresponds mars-1, encodes C. elegans homologue MetRS. let-65 number <br /> of lethal genes previously identi&#239;&#172;?ed EMS screens essential genes right end LGIV. These screens <br /> generated seven alleles let-65 resulted lethality (s253, s1777, s1083, s1154, s1222 s1730), <br /> and additional allele causing sterility (s694). Previously reported inactivation MARS-1 by <br /> RNAi leads embryonic lethality (Kamath et al., 2003; Sonnichsen et al., 2005). Moreover, loss MARS-1 <br /> function lead sterility (Havrylenko et al., 2011). Furthermore, correlation available phenotypic in- <br /> formation ARS family members deleted targeted mutations demonstrates complete correlation <br /> of lethal sterile phenotypes, highlighting essential nature family enzymes (Table 3). Together <br /> these data indicate mars-1 essential processes require large scale protein biosynthesis, as <br /> embryogenesis (Havrylenko et al., 2011). <br />     In C. elegans cytoplasmic MetRS (MARS-1) 917 amino acid protein multiple predicted func- <br /> tional domains. The minimal core enzyme, bacteria Aquifex aeolicus, consists core catalytic <br /> domain (CAT), distinguished Rossmann fold characteristic domain Class I aminoacyl <br /> tRNA synthetase enzymes (catalyzes aminoacylation reaction), anticodon-binding domain (ABD) <br /> (promotes tRNA&#226;&#8364;&#8220;protein association) (Nakanishi et al., 2005). Additional domains, eukaryotes, are <br /> appended core domain enzyme believed increase stability of <br /> tRNA&#226;&#8364;&#8220;protein complexes (Havrylenko et al., 2010). In humans, MARS carries C-terminal domain causes <br /> a slow release aminoacyl-tRNA provides aminoacylation reaction limiting step (Havrylenko <br /> et al., 2010). In C. elegans, MARS-1 protein model contains t-RNA binding domain (tRBD) 170 <br />  826                                        M.Z. Alriyami et al. / Meta Gene 2 (2014) 819&#226;&#8364;&#8220;830 <br />  <br />  <br /> Table 3 <br /> Correlation known viability phenotypes ARS-family genes. <br /> Data source: WormBase, http://www.wormbase.org, release WS243, May 2014. <br />  <br />   Gene                                   RNAi phenotype                                    Mutation phenotype [allele] <br />  <br />   aars-1                                      ste <br />   aars-2                                      emb; lvl; ste <br />   cars-1                                      emb; let; lvl; ste <br />   cars-2                                      ste <br />   dars-1                                      emb; lvl; ste <br />   dars-2                                      emb; lvl; ste                                     let; ste [tm3799] <br />   ears-1                                      emb; lvl <br />   ears-2                                      emb; lvl; ste <br />   fars-1                                      emb; lvl; ste <br />   fars-2                                      ND <br />   fars-3                                      emb; lvl; ste <br />   gars-1                                      emb; lvl; ste <br />   hars-1                                      emb; let; lvl; ste                                let; ste [tm4074] (Pierce et al., 2011) <br />   iars-1                                      emb; ste <br />   iars-1                                      emb; let <br />   kars-1                                      emb; lvl; ste <br />   lars-1                                      emb; let; lvl; ste <br />   lars-2                                      emb                                               let; ste [tm5774] <br />   mars-1                                      emb; lvl <br />   nars-1                                      emb; let; lvl; ste                                let; ste [tm524] <br />   nars-2                                      ND <br />   pars-1                                      emb; let; lvl; ste <br />   pars-2                                      ND <br />   qars-1                                      emb; let; lvl; ste <br />   rars-1                                      emb; lvl; ste                                     emb; lvl; ste [gc47] <br />   rars-2                                      emb; ste <br />   sars-1                                      emb; lvl; ste <br />   sars-2                                      emb <br />   tars-1                                      emb; let; lvl; ste <br />   vars-1                                      emb; lvl; ste <br />   vars-2                                      emb; let; lvl; ste                                let; ste [tm3947] <br />   wars-1                                      emb; lvl; ste <br />   yars-1                                      emb; lvl; ste <br />  <br /> emb = embryonic lethality, lvl = larval lethality, let = lethal, ste = sterile, ND = determined. <br />  <br />  <br /> residues appended cis C-terminus enzyme, homologous tRBD domain appended <br /> at C-terminus MetRS Oriza sativa (rice) (Havrylenko et al., 2010). However, C. elegans, domain <br /> is separated ABD domain putative protein-binding domain approximately 150 residues <br /> (Havrylenko et al., 2010). This domain similar human P43 protein, non-synthetase compo- <br /> nent aminoacyl tRNA synthetase complex required assembly protein complex <br /> (Havrylenko et al., 2011). The predicted C. elegans MARS-1 catalytic core domain comprises amino acids 36 to <br /> 407 (Marchler-Bauer et al., 2009). In let-65 mutations reside domain (s1730, s1083, <br /> s694, s1084, s254 s1154) suggesting disrupt function catalytic core. The EMS induced <br /> mutation let-65(s1777) creates truncated protein lacks anticodon-binding domain (AA416 to <br /> AA545) (Marchler-Bauer et al., 2009) tRNA binding domain (AA755 AA857) (Marchler-Bauer <br /> et al., 2009). let-65(s1222) contains mutation &#239;&#172;?rst base &#239;&#172;?rst intron putatively preventing splic- <br /> ing &#239;&#172;?rst intron resulting putative premature truncation amino acid 28. <br />     mars-1 highly conserved encodes protein unique signi&#239;&#172;?cance translational control. Struc- <br /> tures MARS-1 Escherichia coli (Mechulam et al., 1999) Thermus thermophiles (Sugiura et al., 2000) <br /> have reported. MARS-1 E. coli greater identity T. thermophiles protein C. elegans. <br /> Amino acids altered C. elegans missense mutations located conserved regions both <br /> E. coli T. thermophiles: let-65(s1083): &#206;&#178;C; let-65(s694): turn &#206;&#178;2 &#206;&#178;3 (zinc-binding domain); <br /> let-65(s254): turn &#206;&#178;7 &#206;&#177;6; let-65(s1154): turn &#206;&#177;7 &#206;&#178;8. let-65(s1730) turn <br /> between &#206;&#178;A &#206;&#177;A let-65(s1084) turn &#206;&#178;5 &#206;&#178;6. All identi&#239;&#172;?ed let-65 changes located <br /> in E. coli Rossman-like fold. <br />                                      M.Z. Alriyami et al. / Meta Gene 2 (2014) 819&#226;&#8364;&#8220;830                         827 <br />  <br />  <br />     let-65(s1777) contains <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-242' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-246' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span>nonsense mutation</span> tryptophan codon amino acid 408 an <br /> amber stop codon. let-65(s1222) mutation putatively prevents splicing &#239;&#172;?rst intron resulting in <br /> premature truncation, truncated protein lacks majority synthetase including catalytic <br /> domain, binding domains, anticodon-binding domain, tRNA binding domain. s1777 and <br /> s1222 are, therefore, putatively null alleles. The phenotypes (mid-larval arrest) (Clark Baillie, 1992; <br /> Clark et al., 1988) mutants indistinguishable let-65 alleles. Based obser- <br /> vation predict alleles null MARS-1 function, reduce MARS-1 activity threshold <br /> required survival. Moreover, survival nematodes, containing alleles mid-larva, suggests that <br /> for early developmental stages, mother supplies MARS-1 eggs oogenesis. <br />     let-65(s694) let-65(s1083) contain mutations change glycine glutamic acid amino acids 186 <br /> and 126 respectively. These amino acids sequence blocks conserved human yeast. <br /> let-65(s1084) contains mutation changes glycine arginine amino acid 275. Generally, change <br /> from small non-charged non-hydrophilic glycine large positively charged hydrophilic arginine could <br /> disrupt structure protein render non-functional, case let-65 would <br /> cause lethality. let-65 (s1730) contains mutation changes alanine valine position 41. In humans, <br /> Drosophila melanogaster Saccharomyces cerevisiae, serine codon located position within <br /> a highly conserved sequence block. Because change polar amino acid (serine) non-polar <br /> amino acid (valine) disrupt protein&apos;s structure replacing alanine valine analyzed <br /> the available sequences con&#239;&#172;?rm C. elegans wild type reference sequence position 41. After analyzing <br /> the sequence stretch let-65 alleles concluded de&#239;&#172;?nitively alanine present at <br /> position 41. The mutation let-65(s254) replaces conserved acidic amino acid (glutamic acid) a <br /> basic amino acid (lysine) position 364. This position catalytic domain MARS-1. let- <br /> 65(s1154) polar amino acid (proline) instead non-polar amino acid (leucine) position 400 <br /> that catalytic domain. Additionally, alterations proline &#239;&#172;&#8218;anking regions active <br /> sites known change activity protein (Proline bracket hypothesis) (Kini et al., 1998), this <br /> mutation exhibit drastic effects MARS-1 activity. <br />     Typically, eukaryotic cells forms ARS: cytoplasmic mitochondrial (Walker et al., 1983). <br /> MARS-1 localizes cytoplasm (Havrylenko et al., 2010). Therefore, attempted &#239;&#172;?nd candidate <br /> gene coding mitochondrial form enzyme. The C. elegans genome contains second ORF, <br /> Y105E8A.20 predicted encode methionyl tRNA synthetase (WormBase WS211). Y105E8A.20 a <br /> 1221 bp gene encodes 406 amino acid protein. BlastP used search Swiss-Prot database <br /> with MARS-1 amino acid sequences queries (Altschul et al., 1997, 2005). In BlastP searches <br /> S. cerevisiae used canonical genome identify protein localizations C. elegans. S. cerevisiae <br /> was used it&apos;s simplest regard identifying open reading frames (ORFs), the <br /> most highly characterized eukaryotic genome (Fisk et al., 2006). BlastP localization, based S. cerevisiae <br /> matches, predicted mitochondrial localization protein encoded Y105E8A.20. <br />     The C. elegans million mutations project (MMP) comprises library 2007 mutagenized ho- <br /> mozygous C. elegans strains sequenced identify resulting mutations. The aim this <br /> project generate mutant alleles gene C. elegans genome. However, limitation <br /> of approach severely detrimental mutations genes required viability select- <br /> ed strains viable fertile. Of 14 MMP mutations recorded for <br /> mars-1, missense mutations single exception mutation predicted affect <br /> splicing intron C-terminus protein. This intron conserved species <br /> (Table 4). Out remaining 13 alleles 10 affect residues conserved. Interestingly <br /> three mutations reside catalytic core enzyme. Since strains viable is, how- <br /> ever, unlikely lesions severely affect function protein. Alternatively secondary <br /> mutations strains present compensate detrimental effect on <br /> enzyme function. Further studies required ascertain case. <br />     In study isolated identi&#239;&#172;?ed mars-1 alleles lead loss-of-function pheno- <br /> type. Furthermore, number alleles isolated mars-1 greater average number alleles <br /> isolated genes screens type (Berg et al., 2001; Brenner, 1974; Deniziak Barciszewski, <br /> 2001; Horton et al., 2007; Jones et al., 2007; Rozen Skaletsky, 2000; Thacker et al., 2006). This indicates <br /> that let-65 (mars-1) large mutagenic target EMS. The availability loss function alleles allows <br /> for exploration gene&apos;s structure function tractable model system. Moreover, will <br />  828                                        M.Z. Alriyami et al. / Meta Gene 2 (2014) 819&#226;&#8364;&#8220;830 <br />  <br />  <br /> Table 4 <br /> mars-1 MMP mutations. <br />  <br />   Allele            Mutation           Effect                  Domain affected            Type                    Conserved residue <br />  <br />   gk213073          C&#226;&#8224;&#8217;T                A887T                   None                       Missense                No <br />   gk410589          G&#226;&#8224;&#8217;A                P865S                   None                       Missense                No <br />   gk445673          C&#226;&#8224;&#8217;T                Affects splicing        None                       Intron, splicing        No <br />   gk485164          C&#226;&#8224;&#8217;T                A453T                   Anticodon_1                Missense                No <br />   gk509072          C&#226;&#8224;&#8217;T                D190N                   tRNA-synt_1g               Missense                Yes <br />   gk515000          C&#226;&#8224;&#8217;T                G668E                   coiled_coil_region         Missense                No <br />   gk541772          G&#226;&#8224;&#8217;A                P265L                   tRNA-synt_1g               Missense                Yes <br />   gk550543          C&#226;&#8224;&#8217;T                A613T                   Anticodon_1                Missense                No <br />   gk636296          G&#226;&#8224;&#8217;A                S837L                   tRNA_bind                  Missense                No <br />   gk656240          T&#226;&#8224;&#8217;C                M844V                   tRNA_bind                  Missense                No <br />   gk719138          C&#226;&#8224;&#8217;T                A431T                   tRNA-synt_1g               Missense                Yes <br />   gk780068          C&#226;&#8224;&#8217;T                D752N                   None                       Missense                No <br />   gk793812          A&#226;&#8224;&#8217;T                F412L                   tRNA-synt_1g               Missense                No <br />   gk889082          G&#226;&#8224;&#8217;A                A520V                   Anticodon_1                Missense                No <br />  <br />  <br /> facilitate better understanding ARS, lead deeper understanding complete protein <br /> synthesis machinery organisms including humans. <br />     Supplementary data article online http://dx.doi.org/10.1016/j.mgene.2014.08.006. <br />  <br />  <br /> Abbreviations <br /> mRNA      messenger RNAs <br /> tRNA      transfer RNAs <br /> ARS       aminoacyl-tRNA synthetases <br /> METRS methionyl tRNA synthetase <br /> aCGH      oligonucleotide array comparative genomic hybridization <br /> EMS       ethyl methanesulfonate <br /> mars-1 C. elegans methionyl tRNA synthetase encoding gene <br /> GFP       green &#239;&#172;&#8218;uorescent protein <br /> NLS       nuclear localization sequence <br /> MARS-1 C. elegans methionyl tRNA synthetase <br />  <br />  <br />  <br /> Funding <br />  <br />     This project funded Natural Sciences Engineering Research Council Canada (NSERC) <br /> [RGPIN9975] Canada. Maha Z. AlRiyami supported fellowship Sultan Qaboos University, Sultanate <br /> of Oman. <br />  <br />  <br /> Acknowledgments <br />  <br />    We thank Domena Tu help injecting constructs study. We thank Shu Yi <br /> Chua designing fosmid PCR primers. <br />  <br />  <br /> References <br />  <br /> Altschul, S.F., Madden, T.L., Schaffer, A.A., Zhang, J., Zhang, Z., Miller, W., Lipman, D.J., 1997. Gapped BLAST PSI-BLAST: new <br />     generation protein database search programs. Nucleic Acids Res. 25, 3389&#226;&#8364;&#8220;3402. <br /> Altschul, S.F., Wootton, J.C., Gertz, E.M., Agarwala, R., Morgulis, A., Schaffer, A.A., Yu, Y.K., 2005. Protein database searches using <br />     compositionally adjusted substitution matrices. FEBS J. 272, 5101&#226;&#8364;&#8220;5109. <br /> Berg, J.M., Tymoczko, J.L., Stryer, L., 2001. Biochemistry. Biochemistry. W.H. Freeman Company, New York, p. 822. <br /> Brenner, S., 1974. The genetics Caenorhabditis elegans. Genetics 77, 71&#226;&#8364;&#8220;94. <br /> Brown, M.V., Reader, J.S., Tzima, E., 2010. Mammalian aminoacyl-tRNA synthetases: cell signaling functions protein translation <br />     machinery. Vasc. Pharmacol. 52, 21&#226;&#8364;&#8220;26. <br />                                                   M.Z. Alriyami et al. / Meta Gene 2 (2014) 819&#226;&#8364;&#8220;830                                                    829 <br />  <br /> Clark, D.V., 1990. The unc-22 (IV) region Caenorhabditis elegans: genetic analysis molecular mapping(PhD Thesis) Simon Fraser <br />     University. <br /> Clark, D.V., Baillie, D.L., 1992. Genetic analysis complementation germ-line transformation lethal mutations unc-22 IV <br />     region Caenorhabditis elegans. Mol. Gen. Genet. 232, 97&#226;&#8364;&#8220;105. <br /> Clark, A.G., Lyckegaard, E.M., 1988. Natural selection nuclear cytoplasmic transmission. III. Joint analysis segregation and <br />     mtDNA Drosophila melanogaster. Genetics 118, 471&#226;&#8364;&#8220;481. <br /> Clark, D.V., Rogalski, T.M., Donati, L.M., Baillie, D.L., 1988. The unc-22(IV) region Caenorhabditis elegans: genetic analysis lethal <br />     mutations. Genetics 119, 345&#226;&#8364;&#8220;353. <br /> Crick, F.H., Barnett, L., Brenner, S., Watts-Tobin, R.J., 1961. General nature genetic <span id='am-2' about='Thesaurus:code' typeof='owl:Thing'>code</span> proteins. Nature 192, 1227&#226;&#8364;&#8220;1232. <br /> Deniziak, M.A., Barciszewski, J., 2001. Methionyl-tRNA synthetase. Acta Biochim. Pol. 48, 337&#226;&#8364;&#8220;350. <br /> Dhami, P., Coffey, A.J., Abbs, S., Vermeesch, J.R., Dumanski, J.P., Woodward, K.J., Andrews, R.M., Langford, C., Vetrie, D., 2005. Exon array <br />     CGH: detection copy-number changes resolution individual exons human genome. Am. J. Hum. Genet. 76, <br />     750&#226;&#8364;&#8220;762. <br /> Fisk, D.G., Ball, C.A., Dolinski, K., Engel, S.R., Hong, E.L., Issel-Tarver, L., Schwartz, K., Sethuraman, A., Botstein, D., Cherry, J.M., 2006. <br />     Saccharomyces cerevisiae S288C genome annotation: working hypothesis. Yeast 23, 857&#226;&#8364;&#8220;865. <br /> Green, R.A., Kao, H.L., Audhya, A., Arur, S., Mayers, J.R., Fridolfsson, H.N., Schulman, M., Schloissnig, S., Niessen, S., Laband, K., Wang, S., <br />     Starr, D.A., Hyman, A.A., Schedl, T., Desai, A., Piano, F., Gunsalus, K.C., Oegema, K., 2011. A high-resolution C. elegans essential gene <br />     network based phenotypic pro&#239;&#172;?ling complex tissue. Cell 145, 470&#226;&#8364;&#8220;482. <br /> Hashmi, S., Wang, Y., Parhar, R.S., Collison, K.S., Conca, W., Al-Mohanna, F., Gaugler, R., 2013. A C. elegans model study human metabolic <br />     regulation. Nutr. Metab. 10, 31. <br /> Havrylenko, S., Legouis, R., Negrutskii, B., Mirande, M., 2010. Methionyl-tRNA synthetase Caenorhabditis elegans: speci&#239;&#172;?c <br />     multidomain organization convergent functional evolution. Protein Sci. 19, 2475&#226;&#8364;&#8220;2484. <br /> Havrylenko, S., Legouis, R., Negrutskii, B., Mirande, M., 2011. Caenorhabditis elegans evolves new architecture multi-aminoacyl- <br />     tRNA synthetase complex. J. Biol. Chem. 286, 28476&#226;&#8364;&#8220;28487. <br /> Horton, P., Park, K.J., Obayashi, T., Fujita, N., Harada, H., Adams-Collier, C.J., Nakai, K., 2007. WoLF PSORT: protein localization predictor. <br />     Nucleic Acids Res. 35, W585&#226;&#8364;&#8220;W587. <br /> Jakubowski, H., 2001. Translational accuracy aminoacyl-tRNA synthetases: implications atherosclerosis. J. Nutr. 131, 2983S&#226;&#8364;&#8220;2987S. <br /> Jones, M.R., Maydan, J.S., Flibotte, S., Moerman, D.G., Baillie, D.L., 2007. Oligonucleotide array comparative genomic hybridization <br />     (oaCGH) based characterization genetic de&#239;&#172;?ciencies aid gene mapping Caenorhabditis elegans. BMC Genomics 8, 402. <br /> Kamath, R.S., Fraser, A.G., Dong, Y., Poulin, G., Durbin, R., Gotta, M., Kanapin, A., Le Bot, N., Moreno, S., Sohrmann, M., Welchman, D.P., <br />     Zipperlen, P., Ahringer, J., 2003. Systematic functional analysis Caenorhabditis elegans genome using RNAi. Nature 421, <br />     231&#226;&#8364;&#8220;237. <br /> Kini, R.M., Caldwell, R.A., Wu, Q.Y., Baumgarten, C.M., Feher, J.J., Evans, H.J., 1998. Flanking proline residues identify L-type Ca2+ <br />     channel binding site calciseptine FS2. Biochemistry 37, 9058&#226;&#8364;&#8220;9063. <br /> Maeda, I., Kohara, Y., Yamamoto, M., Sugimoto, A., 2001. Large-scale analysis gene function Caenorhabditis elegans high- <br />     throughput RNAi. Curr. Biol. 11, 171&#226;&#8364;&#8220;176. <br /> Maple, J., Moller, S.G., 2007. Mutagenesis Arabidopsis. Methods Mol. Biol. (Clifton, N.J.) 362, 197&#226;&#8364;&#8220;206. <br /> Marchler-Bauer, A., Anderson, J.B., Chitsaz, F., Derbyshire, M.K., DeWeese-Scott, C., Fong, J.H., Geer, L.Y., Geer, R.C., Gonzales, N.R., Gwadz, <br />     M., He, S., Hurwitz, D.I., Jackson, J.D., Ke, Z., Lanczycki, C.J., Liebert, C.A., Liu, C., Lu, F., Lu, S., Marchler, G.H., Mullokandov, M., Song, J.S., <br />     Tasneem, A., Thanki, N., Yamashita, R.A., Zhang, D., Zhang, N., Bryant, S.H., 2009. CDD: speci&#239;&#172;?c functional annotation the <br />     Conserved Domain Database. Nucleic Acids Res. 37, D205&#226;&#8364;&#8220;D210. <br /> Marra, M.A., Baillie, D.L., 1994. Recovery duplications drug resistance selection Caenorhabditis elegans. Genome 37, 701&#226;&#8364;&#8220;705. <br /> Mechulam, Y., Schmitt, E., Maveyraud, L., Zelwer, C., Nureki, O., Yokoyama, S., Konno, M., Blanquet, S., 1999. Crystal structure of <br />     Escherichia coli methionyl-tRNA synthetase highlights species-speci&#239;&#172;?c features. J. Mol. Biol. 294, 1287&#226;&#8364;&#8220;1297. <br /> Moerman, D.G., 1980. A genetic analysis unc-22 region Caenorhabditis elegans(PhD Thesis) Simon Fraser University. <br /> Moerman, D.G., Baillie, D.L., 1981. Formaldehyde mutagenesis nematode Caenorhabditis elegans. Mutat. Res. 80, 273&#226;&#8364;&#8220;279. <br /> Nakanishi, K., Ogiso, Y., Nakama, T., Fukai, S., Nureki, O., 2005. Structural basis anticodon recognition methionyl-tRNA synthetase. <br />     Nat. Struct. Mol. Biol. 12, 931&#226;&#8364;&#8220;932. <br /> Piano, F., Schetter, A.J., Morton, D.G., Gunsalus, K.C., Reinke, V., Kim, S.K., Kemphues, K.J., 2002. Gene clustering based RNAi phenotypes <br />     ovary-enriched genes C. elegans. Curr. Biol. 12, 1959&#226;&#8364;&#8220;1964. <br /> Pierce, S.B., Chisholm, K.M., Lynch, E.D., Lee, M.K., Walsh, T., Opitz, J.M., Li, W., Klevit, R.E., King, M.C., 2011. Mutations mitochondrial <br />     histidyl tRNA synthetase HARS2 cause ovarian dysgenesis sensorineural hearing loss Perrault syndrome. Proc. Natl. Acad. <br />     Sci. U. S. A. 108, 6543&#226;&#8364;&#8220;6548. <br /> Rogalski, T.M., Baillie, D.L., 1985. Genetic organization unc-22 IV gene adjacent region Caenorhabditis elegans. Mol. Gen. <br />     Genet. 201, 409&#226;&#8364;&#8220;414. <br /> Rogalski, T.M., Moerman, D.G., Baillie, D.L., 1982. Essential genes de&#239;&#172;?ciencies unc-22 IV region Caenorhabditis elegans. <br />     Genetics 102, 725&#226;&#8364;&#8220;736. <br /> Rozen, S., Skaletsky, H.J., 2000. Primer3 WWW general users biologist programmers. In: Krawetz, S., M.S.e (Eds.), <br />     Methods Molecular Biology. Humana Press, Totowa, NJ, pp. 365&#226;&#8364;&#8220;386. <br /> Schein, J.E., Marra, M.A., Benian, G.M., Fields, C., Baillie, D.L., 1993. The use de&#239;&#172;?ciencies determine essential gene content the <br />     let-56-unc-22 region Caenorhabditis elegans. Genome 36, 1148&#226;&#8364;&#8220;1156. <br /> Simmer, F., Moorman, C., van der Linden, A.M., Kuijk, E., van den Berghe, P.V. <br /> </body></html>