<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMB <br />    Reports <br />  <br />  <br />  <br /> Partial AUC maximization essential gene prediction using <br /> genetic algorithms <br />                                                                   &amp; Sangsoo Kim * <br />                      1                     1                1,&#226;&#8364;&#160;                2, <br /> Kyu-Baek Hwang , Beom-Yong Ha , Sanghun Ju <br /> 1 <br />  School Computer Science Engineering, Soongsil University, 2School Systems Biomedical Science, Soongsil University, Seoul <br /> 156-743, Korea <br />  <br />  <br />  <br /> Identifying genes indispensable organism&#226;&#8364;&#732;s life                 screening. For example, computational prediction essential <br /> characteristics central questions current                       genes pathogenic microbes implication prioritizing <br /> biological research, helpful develop                    antimicrobial drug targets (2). In addition candidate list, <br /> computational approaches prediction essential                     computational predictions identify sequence features and <br /> genes. The performance predictor usually measured                     related properties relevant essentiality. <br /> the area receiver operating characteristic curve                       &#227;&#8364;&#8364;Computationally, predicting essential genes amounts a <br /> (AUC). We propose novel method implementing genetic                         problem classifying genes binary classes essen- <br /> algorithms maximize partial AUC restricted                   tial nonessential ones based number features that <br /> specific interval lower false positive rate (FPR), region                 calculated gene. These features some <br /> relevant follow-up experimental validation. Our predictor                     relevance essentiality. Previous studies used various <br /> uses various features based sequence information, protein-                    types features, comparative genomic properties, se- <br /> protein interaction network topology, gene expression                        quence-based features, functional genomic properties. It is <br /> profiles. A feature selection wrapper developed                           established essential genes conserved than <br /> alleviate over-fitting problem weigh feature&#226;&#8364;&#8482;s                   nonessential ones (3). Phyletic retention, number organ- <br /> relevance prediction. We evaluated method using                       isms ortholog conserved, used widely <br /> proteome budding yeast. Our implementation genetic                         prediction essential genes (4). Besides comparative ge- <br /> algorithms maximizing partial AUC 0.05 0.10                      nomic features, number sequence-based features be <br /> FPR outperformed popular classification methods. [BMB                      directly calculated sequenced genome em- <br /> Reports 2013; 46(1): 41-46]                                                      ployed predicting essential genes (4, 5). For newly se- <br />                                                                                  quenced genome, sequence-based features typi- <br />                                                                                  cally available, thus, essentiality prediction solely based on <br /> INTRODUCTION                                                                     features important applications. <br />                                                                                  &#227;&#8364;&#8364;Proliferation high-throughput functional genomics data <br /> Identifying genes indispensable organism&#226;&#8364;&#732;s life,               provided opportunities explore properties relation <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br /> identifying characteristics genes, cen-               essentiality. For example, protein-protein interaction network <br /> tral questions biology. The essential genes number or-                crucial biological processes. Essential genes tend to <br /> ganisms identified catalogued ge-                          play special roles, like hubs network, thus, topo- <br /> nome-wide <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>knock-out</span> experiments (1). However, identification                     logical properties used predict gene essentiality <br /> of essential genes wet experiments time-consuming                     (6-8). In fact, Hwang et al. combined protein-protein inter- <br /> and labor-intensive. Hence, helpful develop com-                  action network properties sequence features predict- <br /> putational approaches essential gene prediction candidate                 ing essential genes, demonstrated essentiality pre- <br />                                                                                  diction improved adding network properties (9). Gene ex- <br />                                                                                  pression fundamental process tightly regulated sys- <br /> *Corresponding author. Tel: +82-2-820-0457; FAX: +82-2-820-0816, <br />                                                                                  tem level, related gene essentiality. It shown <br /> E-mail: sskimb@ssu.ac.kr <br /> &#226;&#8364;&#160; <br />  Present address: Platform R&amp;D Team, Technical Group, Daum                       gene expression variation related gene essentiality (7, <br /> Commnucations, Seoul 140-894, Korea.                                             10, 11). <br /> http://dx.doi.org/10.5483/BMBRep.2013.46.1.159                                   &#227;&#8364;&#8364;We integrated features, based sequence information <br />                                                                                  protein-protein interaction network topology, have <br /> Received 2 August 2012, Revised 8 August 2012,                                   used methods. In addition, included gene ex- <br /> Accepted 8 August 2012                                                           pression properties transcriptional activity variation, <br /> Keywords: AUC, Classification, Essential genes, Genetic algo- <br />                                                                                  phyletic retention. While inclusion fea- <br /> rithms, Partial AUC                                                              tures improve prediction power, irrelevant or <br />  <br />  <br /> ISSN: 1976-6696 (print edition) / 1976-670X (electronic edition) <br /> Copyright &#226;&#8220;&#8217; 2013 The Korean Society Biochemistry Molecular Biology <br />    This open-access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/li- <br /> censes/by-nc/3.0) permits unrestricted non-commercial use, distribution, reproduction medium, provided original work properly cited. <br />     PAUC maximization using genetic algorithms&#226;&#8364;&#402; <br />    Kyu-Baek Hwang, et al. <br />  <br />  <br />  <br />  <br />    distracting features confuse machine learning         ture selection wrapper training set fold. For each <br />    cause over-fitting. Pruning features improve         fold, feature selection procedure repeated 50 times re- <br />    performance predictor, help assess rela-       duce variability (see MATERIALS METHODS). In doing so, <br />    tive importance feature essentiality prediction. For        identify features consistently selected. <br />    this, various techniques used. For example, corre-       These features relevant to <br />    lation coefficient feature essentiality, n&#195;&#164;ive    essentiality. Table 1 summarizes number times fea- <br />    Bayes technique used assess relative importance         ture selected fold. <br />    feature essentiality predictor (4, 12). Wilcoxon rank       &#227;&#8364;&#8364;Among 31 features, following selected <br />    sum tests used evaluate statistical difference essen-   50 times fold: nucleus, phyletic retention, essentiality <br />    tial nonessential genes feature (9). The receiver op-      index (EI), clique level (KL), variance gene expression. <br />    erating characteristic (ROC) curves evaluated in-        Our findings highly important features essentiality <br />    dividual features assess performances (9). We employed        concordant previous studies follows. First, essential <br />    rigorous robust method avoids over-fitting.            genes tend localized nucleus, essential bio- <br />    Using backward greedy search elimination, features se-            logical processes information storage processing <br />    lected based ROC evaluation, followed 10-fold                place (4, 16). Second, essential genes known more <br />    cross-validation.                                                      evolutionarily-conserved nonessential genes bacteria <br />    &#227;&#8364;&#8364;The area ROC curve (a.k.a., AUC) common per-               (3) eukaryotes (17). In regard, Gustafson et <br />    formance measure evaluate classification method. When             al. (5) showed phyletic retention, surrogate evolu- <br />    applied binary classification problem, conventional         tionary conservation, highly correlated gene essentiality <br />    AUC maximization concerns range true pos-            yeast E. coli. Third, essential genes likely in- <br />    itive rate false positive rate (FPR). For practical purpose    teract essential genes nonessential genes. For ex- <br />    prioritizing candidates undergo experimental vali-       ample, Hwang et al. (9) showed essential genes higher <br />    dation, important produce short list depleted    values essentiality index (the proportion essential genes <br />    false positives. Seringhaus et al. (4) approached problem      interacting partners) nonessential genes. Fourth, <br />    assigning relatively higher costs false positives false     clique level (the size largest cliques protein belongs <br />    negatives. Here, propose partial AUC maximization fo-          protein-protein interaction network) related con- <br />    cuses scoring candidates, entire       nectedness gene, known related gene es- <br />    set. Partial AUC used medical diagnostic screening,        sentiality (18). It outperformed network-based features in <br />    high true-positive rate fixed lower FPR pref-        Hwang et al.&#226;&#8364;&#8482;s study (9). Finally, variance gene expression is <br />    erable (13). We developed method maximized partial          known negatively correlated gene essentiality and <br />    AUC directly given FPR 0.05 0.1 using genetic      evolutionary conservation rate eukaryotes, including yeast (7, <br />    algorithm. It heuristic search techniques      10). <br />    inspired evolutionary biology, used wide-         &#227;&#8364;&#8364;Besides transcriptional variability, transcriptional activity or <br />    ly solve complicated problems various fields. We evaluated       gene expression level observed related es- <br />    method using proteome Saccharomyces cerevisiae,             sentiality, highly expressed genes evolve slowly bac- <br />    comprehensive set experimentally validated es-          teria mammals (10). In study, mean gene expression was <br />    sential genes available. In addition, high throughput pro-          selected essentiality features time <br />    tein-protein interaction data gene expression profiles     (&#239;&#188;&#382;90% average). Three sequence-based features, i.e., <br />    model organism readily available DIP (14) GEO             A3s, Gravy, CLOSE STOP RATIO, chosen very <br />    (15), respectively. We comparisons machine             often. Gravy, general average hydropathicity, and <br />    learning algorithms.                                                   TM_HELIX, number transmembrane helices, been <br />                                                                           shown negatively correlated essentiality (4). They are <br />    RESULTS                                                                correlated hydropathy im- <br />                                                                           portant factors contributing prediction transmembrane <br />    We compiled total 31 features 5825 S. cerevisiae genes        proteins. Many transmembrane proteins function transporters <br />    various sources. After removing genes missing val-       participate metabolic nonessential roles. In yeast, high- <br />    ues, final dataset comprised 3979 genes (including 940 es-      ly expressed (more likely essential nonessential) genes pre- <br />    sential genes). Ten-fold cross validation used dataset     fer pyrimidine bases (C T) synonymous codon po- <br />    performance comparison.                                            sition prefer codons A+T richness, a <br />                                                                           characteristics yeast genome (38% G+C content) (19). <br />    Features relevant gene essentiality                                 The fact A3s, frequency A synonymous po- <br />    Pruning irrelevant distracting features important step   sition codons, relatively frequently chosen feature <br />    machine learning system. By maintaining highly-relevant           selection experiments high frequency in <br />    features, simple fast model built. We applied fea-      nonessential genes. If essential genes yeast prefer pyr- <br />  <br />  <br /> 42 BMB Reports                                                                                                             http://bmbreports.org <br />                                                                                                             PAUC maximization using genetic algorithms <br />                                                                                                                               Kyu-Baek Hwang, et al. <br />  <br />  <br />  <br />  <br /> imidine bases synonymous codon position,                   G3s showed negative correlation (C.C. = &#226;&#710;&#8217;0.78) <br /> would expect C3s T3s, frequencies C T, re-              fold feature selection experiments (Table 1). <br /> spectively, synonymous codon position, similar <br /> prediction powers. Indeed, number times C3s se-                  Prediction performance gene essentiality <br /> lected showed high correlation T3s (C.C. = 0.88),            Genetic algorithms different objective functions: In each <br />  <br />  <br /> Table 1. The number timesa feature selected fold <br />  <br />                                                                             Fold number <br />                                 Features                                                                                              Average <br />                                                         1    2    3    4     5       6        7       8           9       10 <br />                             1 <br />        Cytosol                                          17   20   23   3     23      6        7       13         14        5            13.1 <br />                                         1 <br />        Extracellular                                    0    2    0    0     0       0        1       0           0        0             0.3 <br />                                                 1 <br />        Plasma membrane                                  2    5    6    1     14      4       21       3          16       12             8.4 <br />                                             1 <br />        Mitochondria                                     0    0    0    0     0       1        0       0           0        0             0.1 <br />                             1 <br />        Nucleus                                          50   50   50   50    50      50      50       50         50       50             50 <br />                                     2 <br />        TM_HELIX                                         44   44   43   16    47      43      38       45         50       18            37.8 <br />                 3 <br />        T3s                                              38   21   23   28    38      10       7       19         16       25            22.5 <br />        C3s3                                             29   14   14   24    23      14       7       13         15       23            17.6 <br />                 3 <br />        A3s                                              50   50   50   50    50      50      50       50         50       49            49.9 <br />                  3 <br />        G3s                                              19   29   32   22    21      49      43       41         33       49            33.8 <br />        CAI3                                             2    0    1    6     1       0        3       2           1        2             1.8 <br />                 3 <br />        CBI                                              34   10   24   27    43      19      38       22         18       30            26.5 <br />                 3 <br />        Fop                                              12   42   18   22    10      32      11       25         29       19             22 <br />              3 <br />        Nc                                               15   0    5    1     8       2        7       3           1        1             4.3 <br />                         3 <br />        GC3s                                             39   24   19   27    36      12       8       16         17       18            21.6 <br />              3 <br />        GC                                               2    2    17   0     11      2        2       10          7        9             6.2 <br />                         3 <br />        L_sym                                            4    6    16   9     2       9        4       9          17        7             8.3 <br />                     3 <br />        L_aa                                             4    6    15   9     2       9        4       9          17        6             8.1 <br />                         3 <br />        Gravy                                            50   49   49   49    50      50      48       49         48       47            48.9 <br />        Aromo3                                           28   3    7    0     6       34       5       16          8        2            10.9 <br />                                                     4 <br />        RAREAMINO_ACID                                   9    4    2    0     6       6        0       5           2        0             3.4 <br />                                                     4 <br />        CLOSE STOP RAIO                                  46   44   46   50    50      48      50       45         46       47            47.2 <br />        Phyletic retention5                              50   50   50   50    50      50      50       50         50       50             50 <br />                                 6 <br />        DegreeK                                          2    5    0    6     3       7        2       29          8        1             6.3 <br />                     6 <br />        CCo                                              9    0    0    0     0       0        0       3           3        2             1.7 <br />              6 <br />        BC                                               5    4    3    43    4       3        0       5           0        2             6.9 <br />              6 <br />        CC                                               33   26   34   38    25      37      16       50         41       25            32.5 <br />             6 <br />        KL                                               50   50   50   50    50      50      50       50         50       50             50 <br />          6 <br />        EI                                               50   50   50   50    50      50      50       50         50       50             50 <br />                         7 <br />        Mean                                             49   48   50   47    45      41      47       50         49       47            47.3 <br />                                 7 <br />        Variance                                         50   50   50   50    50      50      50       50         50       50             50 <br /> a <br />  The number times feature selected. The colored background highlights cells selected time (dark blue), more <br />                                                                1                                                              2 <br /> than 90% (puple), 80% (cyan) 50 trials fold. subcellular localization probabilities calculated ConLoc. Number trans- <br /> membrane helices calculated TMHMM. 3Condon usage freqeuncies calculated CondonW. 4Ratios derived results CondonW. <br /> 5                                        6                                             7 <br />  Number organisms having ortholog. Protein-protein interaction network features. Gene expression features. <br />  <br /> http://bmbreports.org                                                                                                                   BMB Reports     43 <br />     PAUC maximization using genetic algorithms&#226;&#8364;&#402; <br />    Kyu-Baek Hwang, et al. <br />  <br />  <br />  <br />  <br />    fold, applied criteria feature selection: filtering   genetic algorithm-based methods generally useful than <br />    (Entire), &#226;&#8240;&#165;80% (FS_80), &#226;&#8240;&#165;90% (FS_90) 50 trials (see        classification methods task performance <br />    Table 1 MATERIALS METHODS). For selected fea-            measure partial AUC. Other methods ob- <br />    ture subset, linear discriminant function trained ge-      jective functions. Multilayer perceptrons usually trained by <br />    netic algorithm. As objective functions genetic algorithm,     minimizing squared error maximizing cross entropy <br />    employed performance measures suitable task         function. Logistic regression performed maximizing the <br />    essential gene prediction, i.e., partial AUC 5% FPR (GA_           likelihood. Support vector machines trained maximizing <br />    PAUC_0.05) partial AUC 10% FPR (GA_PAUC_0.10)               distance decision boundary closest data <br />    AUC (GA_ AUC). Supplementary Fig. 1 shows perform-            point. All objective functions calculated train- <br />    ance comparisons different objective functions fea-         ing data examples. However, partial AUC calculated a <br />    ture selection criteria. Supplementary Figs. 1A 1B compare        subset data examples, i.e., examples classified positive <br />    performance, measured partial AUC 5% 10% FPR,           specific value FPR. Thus, optimal classifier re- <br />    respectively. Regardless feature selection criteria,           spect objective functions achieve suboptimal partial <br />    GA_PAUC_ 0.05 GA_PAUC_0.10 performed significantly                AUC values. On contrary, methods trained di- <br />    better GA_AUC terms partial AUC 5% 10%              rectly maximizing performance measure, i.e., partial AUC <br />    FPR, respectively. Supplementary Fig. 1C shows GA_AUC            likely achieve better performance terms <br />    achieved higher AUC values GA_ PAUC_0.05                measure. <br />    GA_PAUC_0.10. <span id='am-1' about='protege:FROM' typeof='owl:Thing'>From</span> results, conclude            &#227;&#8364;&#8364;Feature selection different impact classification <br />    objective function used genetic algorithms highly influen-     method. Our feature selection methods greatly improved the <br />    tial performance. Thus, objective function     performance multilayer perceptrons, regardless per- <br />    matched performance measure.                                   formance measure feature selection criteria (Supple- <br />    &#227;&#8364;&#8364;Our feature selection method improved prediction perform-        mentary Fig. 2). However, feature selection did strong <br />    ance general, performance GA_PAUC_0.05 sig-         influence performance classification methods. In <br />    nificantly enhanced feature selection (Supplementary Figs. 1A      experiments, multilayer perceptrons severely overfitted the <br />    1C). GA_PAUC_0.10 showed performance enhancement               training dataset fold (see Supplementary Tables 4, 5, and <br />    feature selection terms AUC (Supplementary Fig. 1C).            6). Overfitting reduced eliminating relevant <br />    However, performance measured partial AUC 10% FPR           features. This explains feature selection improved per- <br />    degraded feature selection (Supplementary Fig. 1B). The        formance multilayer perceptrons, methods. <br />    performance GA_AUC improved feature selection <br />    measured partial AUC 10% FPR (Supplementary Fig.           DISCUSSION <br />    1B). When measured partial AUC 5% FPR AUC, per- <br />    formance did significant difference feature se-       The ultimate goal developing computational methods the <br />    lection criteria (Supplementary Figs. 1A 1C). We ob-         prediction essential genes predictability under-stud- <br />    served variance performance reduced feature        ied organisms based universal set features devel- <br />    selection (Entire vs. FS_80 FS_90; Supplementary Tables       oped studied model organisms. The prediction system <br />    1, 2 3). By reducing number features, feature se-      robust applicable distantly re- <br />    lection method effectively enhances generalization perform-       lated organisms. Deng et al. investigated issue dis- <br />    ance genetic algorithm cases.                          tantly related bacteria (12). In report, did address the <br />    &#227;&#8364;&#8364;Comparison classification methods: We compared            transferability issue, focused core issues, feature se- <br />    genetic algorithm-based methods (GA_PAUC_0.05, GA_                lection classification algorithms. <br />    PAUC_0.10, GA_AUC) popular classification              &#227;&#8364;&#8364;For newly sequenced genome, sequence-based fea- <br />    methods multilayer perceptrons (multiLayer), logistic re-     tures usually available. However, advent of <br />    gression (logistic), support vector machines RBF (radial     next-generation sequencing technology, gene expression profil- <br />    basis function) polynomial kernels (smoRBF smoPoly,           ing using RNA-seq technology performed concomitantly <br />    respectively) (Supplementary Fig. 2). All parameters        independent genome sequencing. Considering that <br />    methods set default Weka (see MATERIALS                gene expression properties closely related evolu- <br />    METHODS). Our genetic algorithm-based methods outperfor-              tionary conservation essentiality, appropriate include <br />    med case, i.e., performance meas-      features prediction models. Unlike gene expression <br />    ured partial AUC 5% FPR using entire feature set,       profiles, high throughput protein-protein interaction data usu- <br />    GA_PAUC_0.05 achieved second highest perform-               ally require lengthy experiments. As gene coexpression has <br />    ance (Supplementary Fig. 2A). Furthermore, methods                observed permanent, transient, protein com- <br />    showed lower variance performance others, regardless          plex partners (20), gene coexpression network properties <br />    performance measure feature selection criteria (see        compensate protein-protein interaction network properties. <br />    Supplementary Tables 1, 2, 3). Thus, conclude         &#227;&#8364;&#8364;The power predict essential genes varies various <br />  <br />  <br /> 44 BMB Reports                                                                                                            http://bmbreports.org <br />                                                                                                          PAUC maximization using genetic algorithms <br />                                                                                                                            Kyu-Baek Hwang, et al. <br />  <br />  <br />  <br />  <br /> types features. Unlike previous studies meas-        DEG (16) used calculating EI 4662 genes. <br /> ured feature&#226;&#8364;&#8482;s power independently, using statistical meas-         &#227;&#8364;&#8364;Gene expression features: We downloaded gene expression <br /> ures, employed feature selection wrapper technique,               profiles yeast obtained Affymetrix Yeast Genome S98 <br /> which evaluated feature based classification models in-          Array (9335 probes) GEO (http://www.ncbi.nlm.nih.gov/ <br /> volving features. As measure relevance es-       geo/) (15). Total number samples 2015. From raw CEL <br /> sentiality, used number times feature selected.          files, expression level probe calculated RMA <br /> The majority consistently selected features           (21). Then, calculated mean variance expression of <br /> study previously reported powerful pre-             5885 genes (10). When multiple probes mapped to <br /> dictors studies. It possible rationalize se-   gene, used average values. <br /> lection terms biological knowledge. As such, feature se-           &#227;&#8364;&#8364;Phyletic retention features: We obtained phyletic retention <br /> lection essential gene prediction essen-       data 4728 genes S. cerevisiae (5), phyletic <br /> tial steps machine learning strong biological          retention defined number organisms having an <br /> foundation.                                                              ortholog. <br /> &#227;&#8364;&#8364;While computational prediction biological effects            &#227;&#8364;&#8364;Final dataset: Finally, obtained dataset consisting of <br /> shed insight features play major roles, results        3979 genes 31 features excluding genes having missing <br /> have bigger impact experimental validations. It          value. There 940 essential genes. In experiments, <br /> then better suggest candidate list short de-         10-fold cross validation used. <br /> pleted false positives order minimize labor-in- <br /> tensive time-consuming wet experiments. In cases,               Feature selection <br /> partial AUC, measured fixed lower FPR,          We employed backward search-based wrapper feature se- <br /> important AUC. We implemented genetic algorithms            lection (22). Feature selection wrappers known produce <br /> maximize directly partial AUC. While classification            better results filter methods, information gain, be- <br /> methods objective functions, genetic algorithms           cause consider classification models involved. AUC (Area <br /> allow flexibility formulating one&#226;&#8364;&#8482;s objective                 ROC Curve) 3-fold cross validation adopted <br /> functions. To knowledge, implementation         performance measure wrapper. As classification <br /> genetic algorithms maximizing partial AUC biomedical               model wrapper, used logistic regression. We applied <br /> applications. We believe flexibility robustness         wrapper 50 times selected features based number <br /> stimulate wide acceptance.                                               times selected. According result the <br />                                                                          50 trials, different feature groups: filtering, <br /> MATERIALS AND METHODS                                                    80% selected, 90% selected. <br />  <br /> Data preparation                                                         Partial AUC maximization genetic algorithms <br /> Sequence-based features: We downloaded 5885 ORF se-                      The partial AUC (area ROC curve) sub-area of <br /> quences S. cerevisiae ftp://genome-ftp.stanford.edu.             AUC measured range false positive rates (see <br /> CodonW (http://codonw.sourceforge.net/) used calculate            Supplementary Fig. 1). In order maximize partial AUC a <br /> 14 features related codon usage (see Supplementary Table          training dataset, used genetic algorithms. A linear discrim- <br /> 7). ConLoc (http://sbi.postech.ac.kr/conloc/) used predict        inant function used scoring essential gene candi- <br /> the subcellular localizations: cytosol, extracellular matrix, plas-      date, c, follows. <br /> ma membrane, mitochondria, nucleus. The probabilities for <br /> these locations estimated used features,                                         score(c) = w&#227;&#8224;? f, <br /> respectively. TMHMM Server (http://www.cbs.dtu.dk/ services/ <br /> TMHMM/) predicted number transmembrane helices                 w weight vector f denotes set feature values <br /> each 5825 amino acid sequences. Finally, ratios rare           c. Each feature value fi normalized [0, 1] follows: <br /> amino acids amino acids close stop codons calcu- <br /> lated suggested (4).                                                              fi = (fi - min(fi)) / (max(fi) - min(fi)). <br /> &#227;&#8364;&#8364;Protein-protein interaction network features: We followed the <br /> protocol established Hwang et al. (9). Briefly, protein-pro-        &#227;&#8364;&#8364;A genetic algorithm used finding w maximizes <br /> tein interaction data S. cerevisiae downloaded DIP           partial AUC. See Supplementary Table 8 parameter setting of <br /> (version Scere20091230), contained 5033 proteins               genetic algorithm. In experiments, used partial AUCs <br /> 22,118 interactions. From yeast protein-protein interaction         FPRs (false positive rates) 5% 10% examples ac- <br /> network, calculated following topological properties           ceptable FPRs subsequent validation. <br /> suggested (9): degree (degreeK), clustering coefficient (CCo), <br /> betweenness centrality (BC), closeness centrality (CC), clique           Other classification methods <br /> level (KL), essentiality index (EI). A list essential genes       We used Weka package (http://www.cs.waikato.ac.nz/ml/ <br />  <br />  <br /> http://bmbreports.org                                                                                                                BMB Reports     45 <br />     PAUC maximization using genetic algorithms&#226;&#8364;&#402; <br />    Kyu-Baek Hwang, et al. <br />  <br />  <br />  <br />  <br />    weka/) comparing performance GA-based methods                network sequence analysis. Mol. Biosyst. 5, 1672-1678. <br />    following popular classification methods: multilayer           10. Choi, J. K., Kim, S. C., Seo, J., Kim, S. Bhak, J. (2007) Impact <br />    perceptrons, logistic regression, support vector machines.              transcriptional properties essentiality evolutionary <br />    Default settings models Weka used. For multi-             rate. Genetics. 175, 199-206. <br />    layer perceptrons, number hidden layers one,         11. Zhou, L., Ma, X. Sun, F. (2008) The effects protein inter- <br />                                                                                actions, gene essentiality regulatory regions ex- <br />    number hidden nodes 16. The backpropagation algo- <br />                                                                                pression variation. BMC Syst. Biol. 2, 54. <br />    rithm used training multilayer perceptrons. A ridge esti-       12. Deng, J., Deng, L., Su, S., Zhang, M., Lin, X., Wei, L., Minai, <br />    mator used l <br /> </body></html>