<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal vol. 12 no. 1 pp. 177 - 186, 1993 <br />  <br />  <br /> CDC39, essential nuclear protein negatively <br /> regulates transcription differentially affects the <br /> constitutive inducible HIS3 promoters <br />  <br />  <br /> Martine A.Collart Kevin Struhl                                  HIS3 TATA region (Struhl, 1984). These transcriptional <br />                                                                     initiation patterns arise TR respond <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br /> Department Biological Chemistry Molecular Pharmacology,      transcriptional stimulation GCN4 GAL4, whereas <br /> Harvard Medical School, Boston, MA 02115, USA <br /> Communicated R.Treisman <br />                                                                     Tc cannot. <br />                                                                        Besides distinct responses acidic activator proteins, <br />                                                                     TR Tc differ ways. The TR element defined <br /> The yeast HIS3 promoter region contains functionally            primarily canonical TATA sequence, TATAAA <br /> distinct TATA elements, Tc TR, responsible             (Chen Struhl, 1988; Harbury Struhl, 1989), its <br /> respectively initiation + 1 + 13 sites.            functional activity strongly correlates ability bind <br /> Both Tc TR support basal HIS3 transcription                 TFIID (Strubin Struhl, 1992) support TFIID- <br /> require TATA binding protein TFIID, TR                 dependent transcription vitro (Ponticelli Struhl, 1990; <br /> responds transcriptional activation GCN4                  Wobbe Struhl, 1990). In contrast, Tc does contain <br /> GAL4. By selecting yeast strains increase                  sequence fits TATAAA consensus, surprisingly <br /> transcription GCN4 derivative defective                 tolerant mutations (Mahadevan Struhl, 1990), does <br /> activation domain, isolated temperature-                  support transcription yeast nuclear extracts (Ponticelli <br /> sensitive <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span>mutation</span> CDC39, previously defined gene              Struhl, 1990), does yield DNase I footprints <br /> implicated cell-cycle control pheromone                  TFIID (A.S.Ponticelli K.Struhl, unpublished <br /> response. This cdc39-2 mutation causes increased basal              results). Nevertheless, Tc-dependent transcription requires <br /> transcription many, genes,                TFIID temperature proteolytic inactivation of <br /> increased transcriptional activation GCN4 GAL4.              TFIID vivo results equally rapid decline + 1 <br /> Surprisingly, basal HIS3 transcription + 1                 + 13 transcripts (Cormack Struhl, 1992). Finally, <br /> initiation site strongly increased, initiation        micrococcal nuclease sensitivity his3 TATA region <br /> the + 13 site barely affected. Thus, unlike acidic               associated Tc- TR-dependent transcription <br /> activator proteins function TR, CDC39                  (Oettinger Struhl, 1985; Struhl, 1986). <br /> preferentially affects transcription mediated Tc.                   We begun genetic approach investigate the <br /> CDC39 essential gene encodes large                mechanism GCN4 activation distinction between <br /> nuclear protein (2108 amino acids) containing                   TR- Tc-mediated transcription. Starting strain <br /> glutamine-rich regions. These observations suggest             containing GCN4 derivative partially defective acidic <br /> CDC39 negatively regulates transcription                  activation domain, isolated unlinked suppressor <br /> affecting general RNA polymerase II machinery                mutations increase his3 transcription. In study, we <br /> by altering chromatin structure.                                    mutation CDC39 causes novel <br /> Key words: eukaryotic promoters/GCN4 activation/Sac-                <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span>phenotype</span> increasing his3 transcription + 1 site <br /> charomyces cerevisiaelTATA element/transcriptional                  hardly affecting initiation + 13 site. CDC39 <br /> repression                                                          originally identified temperature-sensitive mutation <br />                                                                     arrests cells GI inappropriately activates the <br />                                                                     pheromone response pathway (Reed, 1980; Barros Lopes <br />                                                                     et al., 1990). However, molecular genetic analyses <br /> Introduction                                                        presented suggest CDC39 directly involved <br /> Constitutive GCN4-inducible expression yeast              cell-cycle control, general negative regulator <br /> HIS3 gene involves TATA elements, Tc TR,               transcription differentially affects constitutive and <br /> promote transcription different molecular mechanisms              inducible his3 promoters. <br /> (Struhl, 1986; Chen Struhl, 1988; Harbury Struhl, <br /> 1989; Mahadevan Struhl, 1990; Ponticelli Struhl,            Results <br /> 1990). Tc responsible transcription initiated the <br /> +1 site, TR-dependent transcription initiated            Isolation temperature-sensitive suppressor <br /> almost exclusively +13 site. In absence             mutation <br /> GCN4, HIS3 transcription initiated equal efficiency         To obtain suppressor mutations increase activation by <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> + 1 + 13 Tc TR act                     GCN4 derivatives partially defective activation <br /> conjunction upstream poly(dA)-poly(dT) sequence             domain, utilized genetic approach based fact <br /> (Struhl, 1985). However, GCN4-activated                      degree GCN4 function correlated cell <br /> transcription, initiation + 13 site increased 5-fold,   growth medium containing aminotriazole (AT), a <br /> whereas initiation + 1 remains essentially unaffected.         competitive inhibitor his3 gene product (Hope and <br /> Preferential initiation + 13 observed GAL4          Struhl, 1986). The starting strain, KY1600, contains a <br /> activates transcription hybrid promoters          chromosomal deletion GCN4, plasmid expressing the <br /> GAL enhancer fused various positions upstream          gcn4-C163 derivative (amino acids 119-281) the <br />    Oxford University Press                                                                                            177 <br />  M.A.Collart K.Struhl <br />  <br />  <br />           dA-dT <br />  <br />  <br />  <br />  <br />              HIS3 <br />            gcn74- 1I <br />                          &lt;~ ~ ~ ~ ~ ~ <br />                         GCN4 <br />  <br />                         GCN4 <br />  <br />  <br />  <br />  <br />                                -    _ <br />                                      r&quot; C.. <br />                                      Tc <br />  <br />  <br />  <br />  <br />                                acti vatio&apos; r <br />                                                i. <br />  <br />  <br />  <br />  <br />                                                     --_ <br />                                                           HIS3 <br />  <br />                                                           2 <br />                                                               v <br />  <br />  <br />  <br />  <br />                                                eIec v r2C n!N A <br />                                                                   -W- <br />  <br />  <br />  <br />  <br />                                                                    b^ <br />                                                                         HIS3 <br />  <br />  <br />  <br />  <br />                                                               Tiabi r ri;r <br />                                                                              lacZ <br />  <br />                                                                               2 <br />  <br />  <br />  <br />  <br />                                                                               / <br />                                                                                     O <br />  <br />  <br />  <br />                                                                                     gcnv <br />                                                                                         i- <br />                                                                                      H;IS.&apos; <br />                                                                                                   c <br />  <br />  <br />                                                                                                   A <br />                                                                                                       86   ;; <br />                                                                                                                      S EN <br />                                                                                                                        11 <br />                                                                                                                         &lt; <br />                                                                                                                       Sc3861 <br />                                                                                                                       Sc3862 <br />                                                                                                                       Sc3863 <br />                                                                                                                       Sc3864 <br />  <br />                                                                                                                       Sc3865 <br />                                                                                                                                 \ <br />                                                                                                                                     RR G HB H ScH <br />                                                                                                                                     II 1 <br />  <br />  <br />  <br />  <br />                                                                                                                                     RRGH <br />                                                                                                                                       _ <br />                                                                                                                                        I       ~oRF-o <br />  <br />  <br />                                                                                                                                                N <br />                                                                                                                                                 iw <br />                                                                                                                                                       H <br />                                                                                                                                                               G <br />                                                                                                                                                               I+ <br />  <br />  <br />  <br />  <br />                                                                                                                                                               G <br />                                                                                                                                                                         B <br />  <br />  <br />  <br />  <br />                                                                                                                                                                         B <br />                                                                                                                                                                             Z <br />  <br />  <br />  <br />                                                                                                                                                                             +- <br />                                                                                                                                                                             + <br />  <br />                                                                                                                                                                             + <br />                                                                                                                                                                                  AIs <br />  <br />  <br />  <br />                                                                                                                                                                                  + <br />  <br />                                                                                                                                                                                  + <br />  <br />  <br />  <br />  <br />                                                                                                                 Fig. 2. The CDC39 locus. The structures phenotypes (determined <br />                                                                                                                 complementation) indicated DNA fragments shown below <br />                                                                                                                 restriction map containing CDC39 gene region (broken lines <br />           (-&#194;&#167;c3866 )                          ;Si Cernlotj?I X g                    Sc3AI83L6                   shown Sc3860 indicate uncertainty lengths). The location <br />               1 6_ <br />                           ~~~~~~~~I                                                 ,Ct1-67-                    transcriptional direction CDC39 indicated open box <br />                                                                                                                 arrow. S, Sall; B, BamHI; N, NcoI; R, EcoRI; G, BglII; H, <br />            faint blue                                                                        re <br />                                                                                                                 HindIII; Sc, SaM. <br />            5 niM A <br />  <br />                                                                                                                 protein shows    significant similarities proteins the <br /> Fig. 1. Genetic selection. The structures HIS3 his3-lacZ                                              NBRF database. However, sequence regions <br /> reporter genes (promoter elements transcriptional initiation sites <br /> indicated) GCN4 gcn4-C163 proteins (transcriptional                                                  amino acids 1009-1046 1300-1329 <br /> activation DNA binding domains indicated) shown. The                                                     respectively contain 63 52% glutamine residues. <br /> parental strain (relevant chromosomal alleles indicated lines                                                Genomic hybridization indicates suppressor gene <br /> plasmid alleles indicated circles) subjected two-step genetic                                        present single copy patterns observed the <br /> selection screen obtain unlinked suppressor mutations (X). <br />                                                                                                                  wild-type suppressor strains indistinguishable <br />                                                                                                                  (Figure 4A). Hybridization probe collection <br /> constitutive DED] promoter (Hope Struhl, 1986),                                                          X bacteriophages covering yeast genome <br /> a reporter plasmid his3-lacZ fusion controlled                                                     (Olson et al., 1986) reveals suppressor gene maps <br /> by promoter optimal GCN4 binding site                                                                 close genetically defined CDC39 locus right <br /> (Figure 1). KY 1600 grows 5 mM AT faintly blue                                                         arm chromosome III (L.Riles, personal communication). <br /> on X-gal indicator plates. By selecting derivatives                                                       CDC39 initially identified temperature-sensitive <br /> KY 1600 grow 20 mM AT darker blue,                                                            mutation (cdc39-1) causes cell-cycle arrest GI, <br /> isolated 13 independent suppressor mutations.                                                                   pachytene arrest effects meiosis, and <br />    Genetic analysis indicated suppressor mutations                                                     constitutively activates pheromone response pathway <br /> are recessive define complementation groups. One                                                       (Reed, 1980; Shuster Byers, 1989; Barros Lopes <br /> of complementation groups defined single                                                          et al., 1990). Sequence comparisons our <br /> mutation prevents growth 37&#194;&#176;C rich medium.                                                       suppressor gene CDC39 identical (S.I.Reed, personal <br /> Diploids mutant wild-type strains grow                                                       communication) correspond YCR93W chromo- <br /> 37&#194;&#176;C, indicating temperature-sensitive growth                                                          III (Oliver, 1992). Moreover, cdc39-l1cdc39-2 <br /> phenotype recessive. The temperature-sensitive                                                      heterozygotes grow high temperature indicating <br /> AT-resistant phenotypes cosegregate 2:2 tetrads, indicating                                                  alleles fail complement. Thus, refer to <br /> that single suppressor mutation confers phenotypes.                                                      suppressor mutation cdc39-2 corresponding <br /> The suppressor mutation does confer                                                             gene CDC39. <br /> phenotype absence GCN4, mutant                                                                 In parental cdc39-2 strains, CDC39 expressed at <br /> parental strains grow indistinguishably 2 mM AT.                                                             equivalent levels single 7 kb RNA (Figure 4B). This <br />                                                                                                                 transcript long encompass putative CDC39 <br /> Cloning molecular analysis CDC39                                                                         protein 2108 residues. The 5&apos; end RNA was <br /> The gene corresponding suppressor mutation                                                           mapped primer extension analysis initiation sites, <br /> cloned complementation temperature-sensitive                                                      major 30 minor 34 nucleotides upstream <br /> AT-resistant phenotypes. A plasmid containing 20 kb yeast                                                    ATG putative CDC39 protein (Figure 4C). <br /> DNA (Sc3860) recovered transformant                                                           An AT-rich region containing sequence GATAAA is <br /> and shown complement phenotypes conferred                                                        centered 51 bp upstream RNA start sites likely <br /> suppressor mutation. Genomic integration genetic                                                            serve TATA element CDC39 transcription. <br /> mapping indicate Sc3860 tightly linked                                                           About 50 bp upstream putative TATA element a <br /> suppressor mutation. By phenotypically testing restriction                                                      poly(dA) poly(dT) stretch, yeast <br /> fragments, complementing gene delimited                                                            promoters utilized upstream promoter element <br /> 7.4 kb region (Figure 2).                                                                                       (Struhl, 1985). Interestingly, Sc3861 (Figure 2), DNA <br />   The nucleotide sequence EcoRI-BglIl fragment                                                           fragment contains putative TATA element entire <br /> (Sc3864) encompassing suppressor gene reveals single                                                      protein coding region lacks poly(dA) * poly(dT) stretch <br /> significant open reading frame 2108 amino acids                                                              sequences upstream, complements the <br /> (Figure 3). Replacement 3276 bp coding sequence                                                        temperature-sensitive AT-resistant phenotype. <br /> the URA3 gene (Figure 2) eliminates complementation                                                          This suggests elements required CDC39 <br /> both phenotypes, indicating functional importance                                                    expression lower levels CDC39 sufficient <br /> putative protein. Despite large size, putative                                                     viability suppression. <br /> 178 <br />                                                                                                                       Role CDC39 transcription regulation <br />  <br />     1 gaattcttggaagaaacactgcctttgagtggaaagctatcattagt tactgttgtgcttgcaacagaagataaattgocctcttcatcagtgtcaaggc 100 <br />   101 agtgtccacaccatcacccatcttctgcttatgcgtcagctctgat tttaccgccttcttgaaaaacctgcttattgt 200 <br />   201 ggtgcccattgtatgtcaaataacatgatctttggctctcaaaagtaccagtttaggttcattacctaocacc 300 <br />   301   atgtcgtatactgaataggtgattcgtagggtatccaatttttttcggcac                                                                                            400 <br />   401   aataactgatgaagtacggcagtatgggacca                                                        gtaaaagfttaaagcccgg                                    500 <br />     501 caattgattac ATO crA TCG GCC ACA TAC CGT OAT TIC MC ACA GCA TCT MAT TTA GMA ACA TCA MAG GMA MAA CAG                                              577 <br />       1               M   L    S   A   T  Y   R    D    L  N   T    A   S   N    L  E  T   S   K    E   K   Q                                          22 <br />     578 GCC OCT CMA ATC OTC ATT GCA CMA ATT AGT TTA TTA TTC ACG ACT CTT MAC MAC GAC MAT TTT GMA TCC GTG GMA                                            652 <br />      23 A A      Q    I   V    I  A    Q   I  S    L    L  F   T   T   L    N   N   D N    F   E    S   V   E                                          47- <br />     653 AGA GM. ATT AGA CAT ATT&apos; TTA GAC AGG TCG TCC GTA OAT ATT TAC ATA MAA GTT TOO GMA CGA T!&apos;A T!&apos;A ACC TTA                                         727 <br />      48 R    E   I    R   H    I   L  D   R   S    S   V    D   I  Y   I    K   V   W  E  R    L    L   T   L                                          72 <br />     728 MGT TCT CGG OAT ATT TTA CMA 003 GGA AMA TT!&apos; TTA CTT CAA GMA MAT CTA CTA CAC AGA CTA CTA &apos;ITA GMA TTT                                          802 <br />      73 s    S   R   D    I    L  Q   A   0   K    F   L   L   Q   E   N    L1 L N R L L L E F                                                         97 <br />     803 GCO MOG OAT TTA CCG MOG MAA AGC ACA GAC CTT ATT GAG CTT TTG MAA GMA CGA ACC TTC MAT MAC CAG GAG TTT                                            877 <br />      98 A    K   D   L    P   K   K   S   T   D    L   I   E   L   1L K E R T F N N Q E F                                                              122 <br />    878 CMA MAA CMA ACA GGA AT!&apos; ACA TTA TCA CTT TTC Aw! OAT CTA TTT OAT MAA TCT GCA MAC AMG OAC ATT ATA GAG                                            952 <br />    123 Q     K   Q   T   0     I  T   L   S   L    F   I   D   L. F D K S A N K D I I E                                                                147 <br />    953 TCA CTT GAC CGC TCC TCT CAG ATr MAC OAT TTC AMG ACA AT!&apos; MG ATG MAT CAT ACA MAT TAT TTA AGG MAT TTT                                             1027 <br />    148 S     L  D    R   S    S   Q   I   N   D    F   K   T   I   K   M   N    N   T  N  Y    L   R   N    F                                          172 <br />  1028 TTT CTT CMA ACC ACA CCA GMA ACA CTA GAG TCC MAT CTA CGC GAC TTA rTG CAT TCC TrIO GMA GOT GMA AGT CTA                                             1102 <br />    173 F     L  Q    T   T     P  E   T   L   E    S   N   L   R   D   L   L    N   S  L  E    G   E    S   L                                          197 <br />  1103 MAT GAC TTA TTA OCT CTT&apos; TTA CTG TCC GMA ATA COT TCA CCT GGG TCT CMG MAT TTA CMA MAT OAT CCC ACA CCGG                                            1177 <br />    198 N     D  L    L   A    L   L   L   S   E    I   L   S   P   G   S   Q    N   L  Q  N    D   P   T   R                                           222 <br />  1178 AIT TOGG T`T0 ACA CCT CCG ATG OTT TTA GAC GCA ACO MAC COT 000 MAC OTT ATA GCA AGA TCT ATA MGT TCT CTG                                            1252 <br />    223 S    W   L    T   P    P   M   V   L   D    A   T   N   R 0     N   V    I   A  R  S    I   S   S   L                                           247 <br />  1253 CMA 0CC MAC CM ATA MAT TOG MAT COT GTG TOT MAT TTA ATG TCA ACA AMG TAT TTC TTG AGC GCA CCA ITOG ATC                                              1327 <br />    248 Q    A   N    Q   I    N   W N&apos; R V         F   N   L   M   S  T    K    Y   F  L  S    A   P   L   M                                           272 <br />  1328 CCT ACT ACA GCA TC TTG AGT TiGC TTA TTT GCA GCA TTGI CAC OAT GGT CCA OTT ATT OAT GMA lO TTC AGT TGC                                              1402 <br />    273 P    T   T    A   S    L   S   C   L   F    A   A   L   H   D  G    P    V   I  D  E    F   F   S   C                                           297 <br />  1403 GAC TOOG MAA OTT ATT TTC MAA CTA OAT TTo 0CC AlIT CMA COT CAT MAG TGG TCG GTA CAG MAT GGT CCC TTT GAC                                            1477 <br />    298 D W      K    V   I    F   K   L   D   L    A   I   Q   L   H  K    W    S   V  Q N    0   C    F   D                                           322 <br />  1478 ITA TTA MAT GCA GMA GOT ACC AGO MAA OTT TCT GMA ACC ATC CCA MAC ACA MAG CMA TCT TTA CTC TAC TTA ITA                                              1552 <br />    323 L    L   N    A   E    0   T  R    K  V     S   E   T   I   P  N    T    K   Q  S  L    L  Y    L   L                                           347 <br />  1553 TCCC ATT GCA WCA TTG MAT TTA GMA TTG TTC CTA CMA MOG GAG GMA 10G TOT OAT GOT CCT ATG CTA OCT TAT lT7                                              1627 <br />    348 S    I   A    S   L   N   L    E   L  F     L  Q    R   E   E  L    S   DOG     P M    L   A    Y   F                                           372 <br />  1628 CMA GAG TGC TTC IT!&apos; GMA OAT TIC MAC TAC GCC CCT&apos; GMA TAT CTT ATT TTA GCA TTA OTC MAA GMA ATG MOG COG                                             1702 <br />    373 Q    E   C    F   F    E  D   F   N   Y    A    P   E  Y    L  I    L   A   L  V  K    E   M    K   R                                           3 97 <br />  1703 TTC OTT TTA rIO ATA GMA MAC AGO ACA GTC ATA GAC GMA ATA CTT ATT ACC TTA TTG ATT CMA OTG CAT MAT MAA                                              1777 <br />    398 F    V   L   L    I    E N    R   T V      I   D   E    I   L  I   T    L   L  I  Q V      N    N   K                                           422 <br />  1778 TCA CCO TCA TCG TT&apos;C MAG GAC OTT ATT TCT ACA ATA ACC OAT OAT TCT AMA ATC OTA OAT GCA GCA MAA ATC ATA                                             1852 <br />    423 S    P   S    S   F   K   D   V    I  S    T    I  T   D    D  S   K     I  V  D  A    A   K    I   I                                           447 <br />  1853 ATC MAC TCG OAT GAC GCA COT ATIT 0CC MAC TT!&apos; TTA AMA TCG&apos; rio TTA OAT AGO GOA AGA TTrA OAT ACG GTC ATT                                          1927 <br />    4481I N      S   D   D    A   P    I  A N      F   L   K   S   L   L   D    T   0  R  L    D   T    V   I                                           472 <br />  1928 MAT MAA CTT ccT TTc MAT GMA GOT TV!&apos; AMA ATT Tio CCA TCC OCA AGA CMA ATT GOT TOO GMG 000 TTC OAT ACT                                             2002 <br />   473 N     K   L   P   F    N   E   A   F   K    I   L    P  C   A   R   Q    I   0  W  E    0   F    D   T                                           497 <br /> 2003 TTC TTA AMA ACA MAA GTT TOT CCA TOT MAT OWC OAT OTA GTG CTG GMA WCA CTA GAO OTT CMA ACO MAA ACC ACT                                               2077 <br />   498 F     L   K   T   K    V   S   P   S   N    V   D   V   V   L   E   S    L   E  V  Q    T   K    M   T                                           522 <br /> 2078 OAT ACA MAC ACT CCA TV!&apos; AGO WCA TTA MOG ACA nTT GAC rIA TTc OCT OT7 CAT WCA TTA ATT GMA OTA CCC MAC                                              2152 <br />   523 D    T   N    T    P   F   R   S   L   K    T   F   D   L   F   A   F    N   S  L  I    E   V    L  N                                            547 <br /> 2153 MAA CCGC CCA CTA OAT OTT CTC CMA TTA CMA AGG Tyr GMA TCC TCC GMA rTr CCC TTA TTA ATT GCA TTT COT AGA                                              2227 <br />   548 K    C    P   L   D    V   L   Q   L   Q    R   F   E   S   L   E   F    S   L  L  I    A   F    P  R                                            572 <br /> 2228 rio ATC MAT TOT GOT TIT! GOA CAC OAT GMA OCT ATT TTA GCC MAT GGT GAC ATC GCA 000 ATT MAT MAT OAT ATT                                              2302 <br />   573 L     I  N    F   0    F   0   N   D   E    A   I   L   A   N   0   D    I   A G   I    N   N    D   I                                           597 <br /> 2303 GMA MOG GMG ACC CAG MAC TAT TTA CAG MAA ACC TAT AIT GOT GAO TTA 0CC ATT AMA OAT OTA AWC GMA CTT CCC                                               2377 <br />   5985E K E M Q N Y L Q K M Y S 0 E L A I K D V I E L L                                                                                                622 <br /> 2378 AGA AGO TTA AGA OAT AGC GAC TIO CCA AGO GAC CMG GMA OTC TTC ACA TOGT ATT ACC CAT 0CC GOT ATA GCA GMA                                              2452 <br />   623KR R      L    R   D    S   D   L   P R      D   Q   E   V   F   T   C    I   T  N  A    V   I   A    E                                           647 <br /> 2453 TCG ACA TTC TTC CAA OAT TAT CCA TM GOAT GCA rTo OCT ACT ACA TOT OTT CTT! rr GOA TCC ACC ATTcr    CTC rr                                           2527 <br />   648 S    T   F    F   Q    D   Y   P   L   D    A   L   A   T   T   S   V    L   F  0  S    M   I    L  F                                            672 <br /> 2528 CMA CCC TTA COT GGA TTC GTA TTA OAC OWC GCA TTT MOG ATA AWC ACC AGO TIT! 0CC MOG GAG COT CCA GAO TCC                                              2602 <br />   673 Q    L   L    K   0    F   V   L   D V      A   F   K   I   I   M   R    F   A  K  E    P   P   E   5                                            697 <br /> 2603 MOG ACCG r     AMG TIT! GOT GTA CMA OCT AT`T TAT GCA T MGG ATA COT Tio 0CC GMA TAT CCA CAG TAT CCT AAG                                            2677 <br />   698 K    M   F    K   F    A   V   Q   A   I    Y   A   F   K   I   R   L    A   E  Y  P    Q   Y   C   K                                            722 <br /> 2678 OAC CTC rTG AOA OAT OTT CCG GOT rio MOG TOT CMG OCT CMA OTT TAC CMA TOT ATC OTC GMA GOT OCT ACC CTA                                               2752 <br />   723 D    L   L    R   D    V   P   A   L   K    S   Q   A   Q V     Y   Q S      I  V  E    A   A   T   L                                            747 <br /> 2753 GCA MAT GOT CCA MOG GMA AGO WCA MGA CCC OWC CMG GMA ACC ATc CCA TTA AMA TTT TT!&apos; GOT OTA OAT GMA OTT                                              2827 <br />   748 A N      A    P   K    E   R   S   R   P    V   Q   E   M   I   P   L    K   F  F  A V      D   E   V                                            772 <br /> 2828 WCA TOGT CMG ATC MAT CMA GMA GOT GOT COT AMA OAT OTC OTA GMA AMA OTT cT&apos;r rTTT OTT CTC MAC MAC OTT ACT                                            2902 <br />   773 S    C   Q    I   N    Q   E   0   A   P    K   D V     V   E   K   V    L   F  V  L N      N   V   T                                            797 <br /> 2903 CTOG OCT MAC TT MAT MAT MOG OTT OAT GMA Tio MAA MAA NIT TCCG ACA CCA MAT TAT TTT TOT TG0 TTT &#194;&#182;1PCC ACA                                            2977 <br />   798 L    A   N    L   N    N   K   V   D   E    L   K   K   S   L   T   P    N   Y  F  S    W   F   S   T                                            822 <br /> 2978 TAT TTA OTT ACGO CMA AGO GOT AMA ACA GMA COT MAC TAT CAT OAT COT TAT MGC MOG OTT ATA GTT GOT ACC 000                                              3052 <br />   823 Y    L   V   T    Q    R   A   K   T   E    P   N   Y   H   D   L   Y    S   K  V  I    V   A   M   0                                            847 <br /> 3053 WCA 000 rio OTA CAT CAG TTC ATG OTC MAC OTT ACT rio NIA CMA TTA TTT OTC OTA CTA TOT ACA MAA GAC GMG                                               3127 <br />   848 S 0      L   L    N    Q   F   M   V N      V   T   L   R   Q   L   F    V   L  L  S    T   K   D   K                                            872 <br /> 3128 CMA 0CC ATC OAT MAA MOG CAC OTA MOG MAT TTG GOT WCA TOG TTA GGA TOGT ATC ACA TTA GOT Tio MAT MAA CCA                                              3202 <br />   873 Q    A   I   D    K    K   N   L   K N      L   A   S   W   L   0   C    I   T  L  A    L   N   K   P                                            897 <br /> 3203 ATT MAA CAC AMG MAT AWC GCA TTC MGG GMA ACC TTA ATC GMA OCT TAT AMG GMA MAT AGA CTT GMA ATA OTT G0CC                                              3277 <br />   8981I    K   N   K    N    I   A   F   R   K    M   L   I   E   A   Y   K    E   N  R  L    E   I   V   V                                            922 <br /> 3278 COT TOT GTA ACA MOG ATT TTA CMA MOG OCT TOT GMA WCA MAA ATT TTC AAO COT CCA MAT CCC &#194;&#182;100 ACT 0GTT GGOC                                            3352 <br />   923 P    F   V   T    K    I   L   Q   R   A    S   E   S   K   I   F   K    P   P  N  P    W   T   V   G                                            947 <br /> 3353 ATA TTA AAG CCC rio ATT GAG Tio MAC GMA MAA GCA MAC T00 MAA TTA AOT TTo ACT TTC GMA (GTT GAG 0G02 TTA                                             3427 <br />   9481I L K        L    L    I   E   L   N   E    K   A   N   W   K   L   S    L   T  F  K    V   E   V   L                                            972 <br /> 3428 TrA MAA TOT TTT MAT Tio ACC ACC MAA TOT CTC AMG CCC TCG MAT TTC ACC MAT ACT CCO GMA OTT ATA GMA ACTr                                              3502 <br />   973 L    K   S   F    N    L   T   T   K   S    L   K   P   S   N   F   I    N   T  P  E    V   I   E   T                                            997 <br />  103 TIA TCC GGT CCL` CCC GGA TCA ATC ACT CTG GAG CACACAAAGG                                        C.AAG&apos;&amp;AVATCIACPAA37 <br />  998 L  S G A L G S I   T   L E(Q                                         )()T            E                         0  1022, <br />   359~ACAC AG_QGAGCA  T   A  P  AG                                    A       PAkA                     -AAA; C VA T 3652 <br /> 3 653 &apos;CTATT <br /> 1048 H     I <br />                 ~&apos;AT GCP&apos;0~T    ACA ATC GCA GAC &apos;VAA&apos;cAA          GCACATTTAC <br />                                                                         3TCTCTC1727 <br />                                                                                 AC <br />                 S   A     N     T   I   A   D    Q   Q   A                A    DF     G   G     E     G      S   I     S       H           N     P     1072 <br /> 3 728 TIT MAC   MAC TTA CIT GGT TCT ACT ATT TTT GTA ACC CAC CCT GAC TTG P.AG AGG GTA TTT CPAI ATG GCT TTA GCC                                          3902 <br /> 1073 F    N     N   L     L     0   S     T     I     F     V     T       H     P     D   L     K     R V        F     Q       H     A     L     A     1097 <br /> 3803 MAG TCA    GTT COG   GMA   ATT TTG   TCC GMA OTA GTC GMA MOG WCA                 TCA GGA   ATT   GCT <br />                                                                                                         GTT      GTT   ACI&apos;,   ACG   ACA   MAA   ATA   3877 <br /> 1098 K    S     V   K     E     I   L     L     E     V   E V             K   s       s   G     I     A V        v     T       T     T     K     I     1122 <br /> 3878 ATA CIT    MAA GAC   TTT   0CC ACT   GMA OTT OAT GAO TCT             MOG TTG     MOG ACG   OCT     ATC <br />                                                                                                       GCA        ATT   A7N7    GTA   AGG   CAT   TCC   39 52 <br /> 1123 I    L     K   D     F     A   T     E   V   D   81  5               K   L       K   T     A     A I        I     H       V     R     H     L     1147 <br /> 3 953 OCA CMA   AGT TTA   OCT   CGA OCT   ACT WCA ATT GMA CCA             rio MAA     GMA GGC   ATA     TCI <br />                                                                                                        CGT       ACT   ATW     CM.   WCA   CTA   GCA   4027 <br /> 1148 A    Q     S   L     A     R   A     T   S   I   K   P               L   K       E   0     I     R S        T     H       Q     S     L     A     1172 <br /> 4028 CCG MAT    TTA ACC   WCT   CIT TCT   TCT WCA CCI GCA GAO GAOGI                   GAC ACG   GCA     MAT <br />                                                                                                       ATA        GMA   MAT     ATT   GGC   ATT   OCT   4102 <br /> 1 1 73 P1 NI    L   H     S     L   I <br />                                           S                                               T     A     I N0       S     N0  T <br />                                                                                                                                      0     T     A     1   1907 <br /> 4103 CTA GTT    TTG ATT   GAG   AAA GCA   TCT   ATG   GAC   AAG TCT       ACT   CAA   GAT TTA   GCA     CAA <br />                                                                                                       GAC        TTG   ATG CAA       GCG   ATT GCT     4177 <br /> 1198 L V        L   I     E     K   A     S     M     D     K   S         T     Q     D   L     A     D Q        L     M       Q     A     I   A       1222 <br /> 4178 ATT CGT    CGT TAT   CAC   AAG GAA   AGA   AGG   GCA   GAC CAA       CCA   TTT   ATT ACG   CAA AAT ACC      AAT   CCA     TAT   TCA   CTG TCT     4252 <br /> 1223 I    R     R   Y     H     K   E     R     R     A     D   Q         P     F     I   T     Q   N   T        N     P       Y     S     L   S       1247 <br /> 4253 TTA CCA    GAA CCT   CTT   GGT TTG   AAA   AAC   ACT   GGT GTT       ACT   CCT   CAA CAA   TWC AGG GTA      TAC   GAA     GAA   TTT   GGT AAG     43 27 <br /> 1248 L    P     E   P     L     G   L     K     N     T     G   V         T     P     Q   Q     F   R   V        Y     E       E     F     G   K       1272 <br /> 4328 AAT ATT    CCA AAC   TTG   GAT GTT   ATT   CCG   TTT   GCA GGA       TTG   CCC   GCT CAC   GCT CCA CCG      ATG   ACT     CAA   AAT   GTG GGT     44 02 <br /> 1273 N    I     P   N     L     D   V     I     P     F     A   G         L     P     A   H     A   P   P        M     T       Q     N     V   G       1297 <br />  <br />                                                                                                                                                                   179 <br />  M.A.Collart K.Struhl <br />  <br />                 4403 TCA ACT fCAG CCT CAG c  AACAA GCG CAA ATG CCT ACC,CAA ATC CTA ACC TCC GAA,CxAA ATA AGA GCT,CAA         4477 <br />                 1298 S T (Q) P (Q) (Q) (Q)(Q) A (Q) M             P  T (Q)I       L T     SEE Q) I R A (Q) (Q)              1322 <br />                 4478 CAA aAG&#194;&#167;*G 4A:tA G&gt;AAGC CGT TTG AAT $:G CCA &apos;C AG TCG GCTr A CCT CA GGA GTG AATGTC tCA                 4552 <br />                 1323(Q) (Q)(Q)(Q)     L (Q)K     S   R    L  N (Q) P      S (Q) S     A (Q) P     P   G   V   N   V   P     1347 <br />                 4553 (LAT (TDTiA <br />                              (Q     T GGG TT GCT GCA GTT CAA TCA AT TTG GAAAG AAT CAA &apos;T GTT CTC GTT CAC CTC ATG GAC        4627 <br />                 1348 N    P  Q    G   G   I  A   A   V   Q    S   D   L   E   Q N     Q R V       L   V   H   L   M   D     1372 <br />                 4628 ATT TTA GTT TCT CAA ATT AAA GAA AAT GCT ACG AAG AAT AAC TTA GCT GAA TA GGC GAT CAA AAC CAA ATT AAA     4702 <br />                 1373 I    L  V    S   Q   I  K   E   N   A   T    K  N    N   L   A   E   L G     D   Q N     Q   I   K     1397 <br />                 4703 ACC ATC ATT TTT CAA ATT TTG ACA TTC ATT GCA AAA AGC GCA CAA AAG GAT CAA TTA GCT TTA AAG GTA TCC CAA    4777 <br />                 1398 T I     I    F   Q   I  L   T    F   I   A   K   S   A   Q   K   D   Q   L   A   L   K   V   S   Q     1422 <br />                 4778 GCT GTC GTT AAT AGC CTT TTT GCC ACT AGT GAG AGT CCT CTC TGC AGA GAA GTT TTG TCC CTA CTT TTG GAA AAG    4852 <br />                 1423 A V V N          S   L  F   A   T    S   E   S   P   L   C   R E V L         S   L   L   L   E   K     1447 <br />                 4853 TTA TGT TCT TTA TCC CTC GTT GCT AGA AAA GAC GTT GTC TGG TGG TTA GTT TAT GCC TTG GAC AGT AGG AAA TTC    4927 <br />                 1448 L C     S    L   S   L  V   A   R   K    D  V V      W   W   L V Y A         L   D   S   R   K   F     1472 <br />                 4928 AAT GTT CCC GTT ATC AGA TCC CTT CTA GAA GTT AAT TTA ATT GAT GCT ACA GAA TTA GAT AAC GTT TTA GTT ACT    5002 <br />                 1473 N V     P    V   I   R  S   L    L   E  V   N    L   I   D   A   T   E   L   D   N   V   L   V   T     1497 <br />                 5003 GCA ATG AAA AAT AAA ATG GAG AAC TCA ACT GAA TTT GCT ATG AAA TTA ATT CAG AAT ACT GTC TTG TCT GAT GAT    5077 <br />                 1498 A M K N K M             E   N    S   T   E   F   A   M   K   L   I   Q   N   T   V   L   S   D   D     1522 <br />                 5078 CCA ATT TTG AMG AGA ATG GAC TTC ATT AAA ACC TTA GAA CAC TTG GCC TCT TCG GAA GAT GAA AAT GTA AAG AAA    5152 <br />                 1523 P    I  L    M   R   M  D   F    I   K  T    L   E   H   L   A   S   S E     D   E N V K         K     1547 <br />                 5153 TTC ATC AAA GAG TTC GAA GAT ACT AAG ATA ATG CCA GTG AGG AAA GGT ACC AAA ACC ACA AGA ACA GAA AAG CTT    5227 <br />                 1548 F I K        E   F   E  D   T   K    I  M    P  V    R   K   G   T   K   T   T   R   T   E   K   L     1572 <br />                 5228 TAC TTA GTA TmT ACG GAA TGG GTA AAA TTA CTT CAA AGA GTT GAG AAT AAC GAC GTA ATC ACA ACT GTT TTT ATC    53 02 <br />                 1573 Y L V F T E W V K                    L   L  Q    R   V   E   N   N   D V     I   T   T   V   F   I     1597 <br />                 5303 AAG CAA TTA GTC GAA AAG GGT GTT ATC AGC GAT ACT GAT AAT TTA CTT ACA TTT GTC AAA AGT TCT CTT GAG CTA    5377 <br />                 1598 K    Q  L    V   E   K  G   V    I   S  D   T   D    N   L   L   T   F   V   K   S   S   L   E   L     1622 <br />                 5378 TCA GTT TCT TCA TTC AAA GAA AGT GAC CCG ACT GAT GAG GTT TTC ATC GCT ATT GAT GCT CTA GGA TCG CTA ATT    5452 <br />                 1623 S V S S F K E S D P T D E V F I A I D A L G S L I                                                      1647 <br />                 5453 ATA AAA TTG TTG ATT TTA CAG GGT TTC AAA GAT GAT ACA AGA AGA GAT TAC ATA AAT GCA ATA TTT TCT GTG ATC    5527 <br />                 1648 I    K  L    L   I   L  Q   G   F   K   D   D   T    R   R   D   Y   I   N   A   I   F   S   V   I     1672 <br />                 5528 GTT TTA GTG TTT GCT AAG GAT CAT AGC CAA GAG GGT ACC ACA TTC AAT GAA CGA CCA TAT TTC AGA CTA TTT TCT    5602 <br />                 1673 V L V        F   A   K  D   H    S  Q    E  G   T    T   F   N   E   R   P   Y   F   R   L   F   S     1697 <br />                 5603 AAC ATC TTA TAC GAA TGG GCT ACC ATC AGG ACG CAC AAT TTT GTT AGA ATA TCT GAT TCC AGC ACT AGG CAG GAG    5677 <br />                 1698 N I L Y E W A T                  I  R   T    H  N    F   V   R   I   S   D   S   S   T   R   Q   E     1722 <br />                 5678 CTG ATC GAA TTT GAT TCT GTA TTT TAC AAC ACT TTC TCA GGA TAT TTG CAC GCT CTG CAA CCA TTT GCC TTC CCT    5752 <br />                 1723 L    I  E    F   D   S  V   F   Y   N   T    F   S   G   Y   L   H   A   L   Q   P   F   A   F   P     1747 <br />                 5753 GGA TTC TCA TTT GCA TGG GTG ACA CTA TTA TCA CAC AGA ATG TTA TTA CCA ATT ATG CTA AGA ITA CCC AAT AAA    5827 <br />                 1748 G F     S    F   A W    V   T   L    L   S   H   R   M   L   L   P   I   M   L   R   L   P   N   K     1772 <br />                 5828 ATA GGT TGG GAA AAG TTA ATG CTT TTG ATT ATC GAT TTIG mTT AAA TTT TTG GAC CAA TAC ACA AGT AAA CAT GCA   5902 <br />                 1773 I G W E K L M L L                    I   I  D    L   F   K   F   L   D   Q   Y   T   S   K   H   A     1797 <br />                 5903 GTC TCT GAC GCT GTT TCG GTT GTT TAT AAG GGA ACA CTG CGT GTT ATT TTA GGC ATT TCG AAT GAT ATG CCA TCC    5977 <br />                 1798 V S D A V S V V Y K G T L R V I                                  L   G   I   S   N   D   M   P   S     1822 <br />                 5978 TTT TTG ATT GAA AAT CAC TAT GAA TTA ATG AAC AAT CTA CCT CCA ACA TAT TTC CAA CTA AAG AAT GTT ATT TTA    6052 <br />                 1823 F L     I    E   N   H  Y   E   L   M   N   N    L   P   P   T   Y   F   Q   L   K   N   V   I   L     1847 <br />                 6053 TCT GCT ATT CCT AAG AAT ATG ACC GTT CCC AAC CCA TAT GAC GTG GAT CTT AAT ATG GAG GAT ATT CCA GCA TGT    6127 <br />                 1848 S A I        P   K   N  M   T   V    P  N    P  Y    D   V   D   L   N   M   E   D   I   P   A   C     1872 <br />                 6128 AAA GAA CTA CCT GAA GTC TTC TTT GAT CCT GTA ATT GAT TTA CAC TCA TTG AAA AAG CCA GTT GAC AAC TAC CTA    6202 <br />                 1873 K E L P E V F F D P V I D L H S L K K P V D N Y L                                                      1897 <br />                 6203 CGT ATT CCC TCA AAT TCA TTA TTA AGA ACA ATA CTA AGC GCT ATT TAC AAG GAT ACC TAT GAC ATA AAA AAG GGC    6277 <br />                 1898 R I     P    S   N   S  L   L   R   T    I   L   S   A   I   Y   K   D   T   Y   D   I   K   K   G     1922 <br />                 6278 GTA GGC TAC GAC ITT TTA TCT GTT GAT AGT AAA TTA ATT CGC GCT ATT GTA TTA CAT GTG GGC ATT GAA GCT GGA    6352 <br />                 1923 V G Y D F L             S   V   D   S   K   L    I   R   A   I   V   L   H   V   G   I   E   A   G     1947 <br />                 6353 ATA GAG TAT AAG AGA ACT TCT TCA AAT GCG GTA TTT AAT ACG AAG TCT TCT TAT TAT ACT TTA ITG TTC AAT CTG    6427 <br />                 1948 I    E  Y    K   R   T  S   S   N   A   V   F   N    T   K   S   S   Y   Y   T   L   L   F   N   L     1972 <br />                 6428 ATT CAA AAT GGT AGC ATC GAA ATG AAA TAT CAA ATT ATT CTG TCT ATT GTG GAA CAA TTG CGG TAT CCA AAC ATC    6502 <br />                 1973 1    Q  N    G   S   I  E   M   K   Y   Q    I   I   L   S   I   V   E   Q   L   R   Y   P   N   I     1997 <br />                 6503 CAC ACC TAT TGG TTC AGC TTT GTG TTA ATG AAT ATG TTC AAA AGT GAC GAA TGG AAT GAT CAA AAA CTT GAA GTC    6577 <br />                 1998 H T Y W F S             F   V   L   M   N   M   F    K   S   D   E   W N     D   Q   K   L   E   V     2022 <br />                 6578 CAA GAA ATT AIT TTA AGA AAC TTr TTA AAA AGA ATT ATT GTT AAC AAA CCA CAT ACC TGG GGT GTT TCA GTT TTC    6652 <br />                 2023 Q E I        I   L   R  N   F   L   K   R    I   I   V   N   K   P   H   T   W   G   V   S   V   F     2047 <br />                 6653 ITT ACT CAG TTG ATA AAC AAT AAC GAT ATT AAT CTT TTA GAC CTG CCC ITT GTA CAA AGT GTT CCC GAA ATT AAA    6727 <br />                 2048 F T Q L          I   N  N   N   D   I   N   L   L    D   L   P   F   V   Q   S   V   P   E   I   K     2072 <br />                 6728 CTA ATT TTA CAA CAA TTA GTA AAA TAT TCC AAA AAA TAC ACA ACC AGT GAA CAA GAT GAC CAA TCC GCC ACC ATC    6802 <br />                 2073 L I     L    Q   Q   L  V   K   Y   S   K   K   Y    T   T   S   E   Q   D   D   Q   S   A   T   I     2097 <br />                 6803 AAT AGA AGG CAA ACC CCT CTA CAA TCC AAC GCA TAA aaaaactgcatttatttatatacgaaaatcatgatttcagaaaaaaaaatat   6890 <br />                 2098 N    R  R    Q   T   P  L   Q   S   N   A                                                              2109 <br />                 6891 aacttgtataatactatatttcagagaatactattttgaataagaatagaagcaatttcgattaacttttcttactgtatatgggtcgctttcacttttg   6990 <br />                 6991 aactactgaaaattagtgttacccgtattattgttttggagaataataataataatgagcatcaatttatatcaaacaagagctgcaattaccttatgtc   7090 <br />  <br />  <br /> Fig. 3. CDC39 sequence. The nucleotide sequence Sc3864 putative amino acid sequence CDC39 shown major RNA <br /> start sites (arrows), potential promoter elements (underlined) glutamine-rich region (glutamine residues circled). <br />  <br />  <br /> CDC39 essential viability                                          Suppression independent mating pheromone <br /> Strains containing original (cdc39-1) present               response pathway <br /> (cdc39-2) allele unable grow 37&#194;&#176;C, suggesting              The initial cdc39-1 allele constitutively activates mating <br /> CDC39 essential gene. To determine phenotype                    pheromone response pathway acting level the <br /> of cdc39 null mutation, disrupted CDC39 locus                 transducing G protein (de Barros Lopes et al., 1990). It was <br /> a diploid strain (KY805) cdc39:: URA3 substitutution             deduced cell-cycle defect consequence rather <br /> allele described above. Genomic hybridization confirms               cause pheromone pathway induction. Although <br /> the resulting strain (KY1640) heterozygous wild-               cdc39-2 mutant strain does exhibit GI arrest <br /> type disrupted CDC39 alleles (Figure 4A). RNA analysis                phenotype restrictive temperature arrests <br /> of heterozygous diploid shows disrupted copy                 asynchronously, remained possible cdc39-2 <br /> of CDC39 expressed small transcripts ending                  mutation affected pheromone response pathway and <br /> the URA3 inserted gene (Figure 4B). Sporulation                       indirectly caused suppression phenotype. <br /> dissection KY1640 resulted tetrads viable                To address issue, constructed ste4A::URA3 and <br /> two inviable progeny. All viable progeny Ura-                        stel2A:: URA3 derivatives KY 1603 lack functional <br /> indicating presence CDC39 allele absence               pheromone response pathway. These strains mate <br /> of cdc39:: URA3 allele. Microscopic examination                       STE4, : subunit transducing G protein <br /> indicated inviable spores germinated yielded             (Whiteway et al., 1989) STEJ2, pheromone-responsive <br /> microcolonies consisting 10-20 normal looking cells.                   transcription factor (Dolan et al., 1989; Errede and <br /> Thus, CDC39 encodes essential gene required cell                   Ammerer, 1989), essential components pheromone <br /> growth.                                                                   response pathway. However, strains grow on <br />  <br /> 180 <br />                                                                                                    Role CDC39 transcription regulation <br />  <br />    A                                       B                                            control       352     \34     A44 AC163 GCN4 <br />                                                                                          Wt 39      Vt39 Wt 39 Wt 39 Wt39 Wt 39 <br />               I.; ca      .c z a. <br />                                                                           +1 --*_ <br />                                                            4-       : <br />     .*. <br />      -N.&apos; <br />                                                                                                                                          -pw <br />                                                                          +13 -l- <br />  <br />                                                                         DED11 -*                                      *        * <br />  <br />                                                            4&quot;I- . <br />  <br />  <br />  <br />                                                                         Fig. 5. Transcriptional activation indicated GCN4 derivatives in <br />         C                                                               wild-type cdc39-2 strains grown permissive temperature. <br />                                                                         Total RNA samples (25 Ag) hybridized completion an <br />                                   G   A    T-C- <br />                                            av&gt; <br />                                             s                           excess HIS3 DED] oligonucleotide probes digested SI <br />                                                                         nuclease. The positions DEDI H1S3 + 1 + 13 <br />                                                                         transcripts indicated. As described previously (Hope Struhl, <br />                                                                         1986; Hope et al., 1988), gcn4-52 contains residues 107-121 fused to <br />                                                   / <br />                                                                         179-281, gcn4-A34 contains residues 1-95 fused 144-281, <br />                                                                         gcn4-zA44 contains residues 107-137 fused 179-183, and <br />                                                                         gcn4-C163 residues 119-281. Hybridization reactions carried out <br />                                                                         parallel, products analyzed gel; however, <br />                                                                         autoradiographic exposure times detecting various <br />                                                                         transcripts equivalent. <br />  <br />  <br />                                                                         Table I. Effect cdc39-2 AT resistance HIS3 transcript levels <br />                                                                         strains containing various GCN4 derivatives <br />  <br /> Fig. 4. Genomic transcriptional analysis CDC39, cdc39-2                                               HIS3 transcriptionc <br /> CDC39/cdc39::URA3 strains. (A) Genomic DNAs wild-type <br /> KY803 (a), mutant KY1617 (b), heterozygous disruption KY1640        Genotypea                     Growthb    +1        +13     ratio +1/+13 <br /> (c) strains hybridized radiolabeled CDC39 probe. The <br /> positions molecular weight markers shown left. The        gcn4-A 1       CDC39             &lt;2      20       21      0.95 <br /> CDC39 cdc39-2 alleles restriction fragments      gcn4-A I       cdc39-2           &lt;2      21       22      0.95 <br /> following sizes: BamHI, 6 kb; Sall, 20 kb; NcoI, 20 kb; EcoRI, 10       gcn4-A52d      CDC39             &lt;2      22       23      0.95 <br /> kb. The fragment sizes cdc39::URA3 allele           gcn4-A52d      cdc39-2           &lt;2      19       20      0.95 <br /> follows: BamHI, 11 kb site lost; SaII, 18 kb; NcoI, 4 kb   gcn4-A34d      CDC39              2      19       27      0.70 <br /> and 17 kb site URA3; EcoRI, 8 kb. (B) Total RNAs <br /> from strains electrophoretically separated, and <br />                                                                         gcn4-A34d      cdc39-2           10      23       29      0.79 <br /> hybridized CDC39 probe. The positions 26S, 17S rRNAs      gcn4-A44d      CDC39              5      20       26      0.77 <br /> and 7 kb CDC39 RNA indicated right. (C) Primer               gcn4-44d       cdc39-2           40      25       29      0.86 <br /> extension product (PE), generated extending 5&apos;-end-labeled CDC39     gcn4-C 163     CDC39              5      19       28      0.68 <br /> oligonucleotide primer RNA KY803, compared              gcn4-C 163     cdc39-2           40      26       32      0.81 <br /> products DNA sequencing using primer. Arrows indicate       GCN4           CDC39          &gt; 120      19       60      0.32 <br /> the 5&apos; end CDC39 RNA.                                            GCN4           cdc39-2        &gt; 120      30       63      0.48 <br />  <br /> 40 mM AT KY 1603, indicating cdc39-2 suppresses                 aStrains contain chromosomal gcn4-A1 allele (lacks GCN4 coding <br /> the gcn4-C 163 activation mutant independently                   sequences) plasmid expressing indicated (or no) gcn4 <br />                                                                         derivative. <br /> pheromone response pathway. Conversely, parental                    bMaximal AT concentration (mM) supports cell growth. <br /> CDC39 strain stel2A:: URA3 ste4A:: URA3                      CLevels + I + 13 transcripts (arbitrary units) normalized the <br /> derivatives indistinguishable AT-resistance phenotypes             level DEDI internal control hybridization reaction. <br /> indicating elimination mating response pathway              dDescribed Hope et al. (1988). <br /> does lead suppression. <br />                                                                         gcn4-A52, derivative lacks functional transcriptional <br />                                                                         activation region. <br /> Effect cdc39-2 transcriptional activation                         To examine directly effect cdc39-2 GCN4- <br /> varous GCN4 dervatives                                                  activated HIS3 transcription, levels + 1 + 13 <br /> In order test suppression cdc39-2 specific            transcripts quantitated respect internal DEDJ <br /> to gcn4-C 163 derivative used isolation,                 control (Figure 5; Table I). As expected, transcriptional <br /> introduced number different GCN4 derivatives wild-            activation GCN4 partially functional derivatives in <br /> type cdc39-2 strains tested AT resistance                wild-type strains increases transcription + 13 site <br /> resulting transformants. As shown Table I, cdc39-2                   + 1 site. In accord growth phenotypes, <br /> increases AT resistance strains containing gcn4-A34              cdc39-2 slightly increases overall HIS3 transcription in <br /> gcn4-A44, partially functional derivatives different               strains containing GCN4 derivatives capable partial <br /> portions activation domain. In contrast, cdc39-2 does            activation (A34, A44, C 163), strains containing <br /> not appear increase transcription strain containing           non-activating derivative gcn4-A52. Surprisingly, the <br />  <br />                                                                                                                                            181 <br />  M.A.Collart K.Struhl <br />  <br />  <br />          _cn4 <br />           6W3 <br />           1 <br />                                   uCfl <br />                                    1 63 <br />          Wt     y3(         Wth%c <br />  <br />  <br />  <br />  <br />                                                                                                DED    .r&apos; <br />  <br />  <br />  <br />  <br />               :....   ::: <br />  <br />                             *w                      .4---            &quot;C.. <br />  <br />                 .40- <br />                             .* ;&apos;10                                                                               cdt- <br />                                                                                              I&quot;tk.v         cdc <br />                                                                                                             39     26 <br />                                                                                          ;:     1. <br />                                                                                        rm.                    3   0 3        4--    hrs 38.5 <br />                                                     .4--    r4J. <br />                                                                                                                                                10V <br />                                                                                                                                       tRNA <br />                             ,:i     :..,:i   .... <br />  <br />  <br />  <br />  <br />                                    A                .4---   H h&apos;.v                                                              ... <br />                                                                                                                                   HIS4 <br />                              .ii. <br />                                                               t                                                              .4--     UEDIE <br />  <br />  <br /> Fig. 6. Transcription various genes wild-type cdc39-2 strains <br /> grown permissive temperature. Total RNA samples (25 itg)                                  :FI.....-D <br />                                                                                                  .- <br />                                                                                                   S. <br />                                                                                                      &apos; <br />  <br />  <br />  <br /> were hybridized completion excess indicated probes <br /> and digested SI nuclease. Hybridization reactions carried <br /> out parallel, products analyzed gel;                                                         *             ilRP3 <br /> however, autoradiographic exposure times detecting various <br /> transcripts equivalent. <br /> effect cdc39-2 GCN4-dependent activation                      Fig. 7. Transcription various genes wild-type, cdc39-2 cdc28 <br /> pronounced + 1 initiation site. Indeed, cdc39-2 causes               strains shifting restrictive temperature. Total RNA samples <br /> stronger activators produce + 1 transcripts, unlike                 (25 /%g) hybridized completion excess indicated <br />                                                                             probes digested SI nuclease. Hybridization reactions were <br /> the situation wild-type strains + 1 transcription               carried parallel, products analyzed same <br /> unaffected GCN4. Thus, suppression cdc39-2 appears                    gel; however, autoradiographic exposure times detecting the <br /> to require acidic activation region, specific             various transcripts equivalent. All strains deleted the <br /> particular GCN4 proteins.                                                   chromosomal gcn4 locus. <br />  <br /> Effect cdc39-2 transcription genes                            stimulated different activator proteins, lacZ reporter <br /> The small effects cdc39-2 his3 transcription                      plasmids control GAL] PH05 promoter <br /> unlikely account fully significant differences              region introduced mutant wild-type strains <br /> in AT resistance. However, AT resistance determined                  (Table II). Transcriptional induction GAL] promoter <br />                                                                             galactose medium, mediated GAL4 <br /> only his3 enzyme levels flux                     activator, increased -3-fold cdc39-2 strain. In <br /> histidine biosynthetic pathway ATRJ (Kanazawa <br /> et al., 1988), suggesting cdc39-2 increase <br />                                                                             glycerol/ethanol medium, cdc39-2 appears minimal <br />                                                                             effect low activity GAL] promoter <br /> transcription GCN4-activated genes. We therefore <br /> examined RNA levels genes cdc39-2 wild-                   makes determination difficult. In contrast, PH05 <br /> type strains containing control gcn4-C 163 expressing                transcription unaffected cdc39-2 normal <br /> plasmid (Figure 6). In presence gcn4-C 163, GCN4-                    conditions medium depleted inorganic phosphate, <br /> regulated genes (HIS3, TRP3 probably HIS4)                         results activation PH02 PHO4. Taken <br /> increased transcription cdc39-2 background. In                       together, results suggest cdc39-2 does affect <br /> contrast, cdc39-2 does affect basal level these <br />                                                                             basal transcription, increase transcriptional <br /> genes, does affect transcription unrelated genes <br />                                                                             activation some, genes. <br /> such DED], ribosomal RNA tryptophan tRNA. These                      Transcription cdc39-2 strains shifted the <br /> results suggest cdc39-2 increases activation                        restrictive temperature <br /> gcn4-C163 does affect basal transcription.                          The results presented far transcriptional <br />   To determine cdc39-2 increases transcription                 alterations growing cdc39-2 cells, conditions where <br />  <br /> 182 <br />                                                                                                          Role CDC39 transcription regulation <br />  <br />  <br /> Table II. Effect cdc39-2 expression GAL] -lacZ                       wild-type strain, presumably cell growth <br /> PHO5-lacZa                                                                        inhibited. These results confirm preferential effects <br />                                                                                   initiation HIS3 +1 site, demonstrate <br />                                  GAL] - lacZP                   PH05-lacZc        CDC39 negatively regulates basal transcription HIS3 <br />                                                                                   genes. However, appears GCN4-activated <br /> Genotype            Glu          Gly +Eth Gly +Eth+Gal          +Pi         -P.   transcription sensitive CDC39 function basal <br />                                                                                   transcription preferentially affected when <br /> CDC39               &lt;0.01        0.1             80             8       95        cdc39-2 cells grown permissive temperature. <br /> cdc39-2             &lt;0.01        0.15           270             5       97 <br />  <br /> a,3-galactosidase   activities (Miller units). <br />                                                                                   Effect cdc39-2 various his3 promoter <br /> bCells grown 2% indicated carbon sources (glucose, glycerol,            derivatives <br /> ethanol, galactose).                                                              To determine element(s) HIS3 promoter region <br /> CCells grown phosphate-depleted medium presence                  responsible negative regulation CDC39, assayed <br /> absence inorganic phosphate.                                                   AT-resistance (not shown) basal HIS3 transcription in <br />                                                                                   cdc39-2 wild-type strains containing his3 <br />              dA-d T         + <br />                                                                                   promoter alleles (Figure 8). In cases, AT resistance and <br />              GCN4           -      -                                              HIS3 transcription higher cdc39-2 strain, although <br />              TC                                                                   magnitude increase varies depending allele. <br />              TR                                                                   Deletion poly(dA) * poly(dT) sequence and/or GCN4 <br />                           :; <br />                            1. .,           .;                                     binding site does significantly reduce magnitude of <br />      -          I                                           . <br />                                                             &apos; <br />                                                                                   cdc39-2 effect, indicating HIS3 promoter sequences <br />                                                                                   upstream TATA region required negative <br />          1 <br />                 o <br />                                                                                   regulation CDC39. The cdc39-2 effect appears somewhat <br />      -   j <br />  <br />  <br />  <br />  <br />                                                 : <br />                                                                                   reduced strains containing deletion Tc a <br />                                                                                   <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>point mutation</span> TR (TGTAAA); expected, increase <br />   DEDI                      j@                                                    + 1 transcription depends Tc increase in <br />                                                                                   + 13 transcription depends TR. This suggests that <br />                           HIS3     A90      A84       \74   93        205 <br />                                                                                   CDC39 does affect inherent properties + 1 <br />                                                                                   + 13 initiator elements, negatively regulates <br /> Fig. 8. Effect loss CDC39 function transcription various            transcription mediated Tc TR. However, the <br /> HIS3 promoters. Wild-type cdc39-2 strains (deleted GCN4) <br /> containing indicated his3 alleles shifted 2 h                 wild-type HIS3 promoter, loss CDC39 function leads to <br /> restrictive temperature, total RNAs hybridized completion             preferential increase + 1 transcription. This suggests that <br /> with excess HIS3 DEDI probes digested SI                       HIS3 TATA elements present, CDC39 affects <br /> nuclease. As described previously (Struhl Hill, 1987; Harbury             Tc-dependent transcription strongly TR- <br /> Struhl, 1989; Mahadevan Struhl, 1990), his3-A90 deletes                   dependent transcription. <br /> GCN4 binding site (-109 -80), his3-A84 deletes sequences <br /> upstream Tc (-447 -94), his3-A-74 retains known <br /> promoter elements lacks region -447 -156,                     CDC39 nuclear protein <br /> his3-A93 deletes Tc (-83 -46) his3-205 point mutation                To determine cellular localization CDC39 protein, we <br /> (TGTAAA) TR.                                                                   expressed CDC39, N-terminal tag (the <br />                                                                                   flu epitope), constitutive DED] promoter. <br /> CDC39 function completely eliminated. We                             Expression protein complemented the <br /> investigated role wild-type CDC39                                 temperature-sensitive AT-resistant phenotypes the <br /> definitively analyzing transcription genes                    cdc39-2 mutation. Proteins total cellular, cytoplasmic <br /> shifting cdc39-2 cells restrictive temperature (38.5&#194;&#176;C).                   nuclear extracts electrophoretically separated and <br /> To distinguish direct effects CDC39 inactivation                          probed antibodies flu epitope (Figure 9). <br /> indirect effects growth arrest, analyzed strain                      A protein migrating apparent molecular weight of <br /> with temperature-sensitive mutation CDC28, gene                             -200 kDa detected total nuclear extracts of <br /> encoding cyclin-associated protein kinase mediates                       transformants containing flu-tagged CDC39 only, <br /> entry cell cycle. These temperature-shift experiments                    indicating CDC39 nuclear protein. <br /> were carried strains deleted GCN4, hence <br /> measure basal (i.e. non-GCN4 activated) transcription. <br />    A strong increase HIS3 transcription, predominantly                         Discussion <br /> from + 1 initiation site, observed cdc39-2 strain                   CDC39 transcriptional regulatory protein <br /> but wild-type cdc28 strain (Figure 7). This effect                  In work, isolated temperature-sensitive mutation <br /> occurs 30 min 38.5&#194;&#176;C (or 1-2 h 37&#194;&#176;C)                          CDC39 selecting yeast strains increase HIS3 <br /> and maintained 4 h (not shown). Similarly,                        transcription GCN4 derivative defective <br /> basal transcription HIS4, TRP3 TFIID increased                          activation domain. CDC39 originally identified a <br /> specifically temperature-shifted cdc39-2 strain. In                        temperature-sensitive mutation causes cell-cycle arrest <br /> contrast, DEDJ transcript levels strains                         GI pachytene arrest meiosis, inappropriately <br /> indistinguishable time points indicating CDC39                        activates mating pheromone response (Reed, 1980; <br /> is essential transcription RNA polymerase II. The                      Shuster Byers, 1989; Barros Lopes et al., 1990). <br /> levels rRNA (not shown) tryptophan tRNA similar                        Many phenotypes conferred original cdc39-1 <br /> in cdc39 cdc28 strains; decreased relative                       mutation, including transcriptional induction pheromone <br />  <br />                                                                                                                                             183 <br />  M.A.Collart K.Struhl <br />  <br />  <br />                                        A <br />  <br />                                         Nuc <br />  <br />  <br />  <br />  <br />                                         .... <br />                                                =*.. <br />  <br />  <br />  <br />  <br />                                                ... <br />                                                      . _. . <br />                                                         +-     u <br />                                                                     B <br />  <br />                                                                       Nuc <br />  <br />                                                                           + <br />                                                                                Cyet   Uot <br />                                                                                       -   _    M <br />  <br />  <br />  <br />  <br />                                                                                                ,   - <br />                                                                                                          OOK <br />                                                                                                        -97.4K <br />                                                                                                        -69K <br />                                                                                                          46K <br />  <br />  <br />  <br />  <br /> Fig. 9. Subcellular localization CDC39. Protein (50 yg) total (Tot), nuclear (Nuc) cytoplasmic (Cyt) extracts prepared strains <br /> expressing CDC39 (+) (-) flu epitope tag analyzed SDS-PAGE assayed (A) protein Ponceau staining and <br /> (B) epitope-tagged CDC39 Western blotting using specific antibodies epitope. The positions corresponding molecular weight markers <br /> and tagged CDC39 protein (arrow) indicated. <br />  <br />  <br /> responsive genes (FUS], GPAI, STE4), interpreted                      tryptophan tRNA RNA polymerase III rRNA by <br /> occurring consequence activating pheromone                        RNA polymerase I (not shown) significantly affected. <br /> response (de Barros Lopes et al., 1990). It hypothesized                     At permissive temperature, cdc39-2 increases GCN4- <br /> that CDC39, concert genes (CDC36, CDC72,                        activated, basal, transcription HIS3, HIS4 and <br /> CDC73 SRMJ) mutated confer                             TRP3. Moreover, cdc39-2 affects activation GAL4 and <br /> phenotypes (Reed et al., 1988; Clark Sprague, 1989),                      GCN4 derivatives different portions acidic <br /> might regulate activity G proteins mating                       activation region; however, does increase <br /> pheromone response signaling pathways (de Barros                    PH02/PH04-mediated activation PHO5. Under these <br /> Lopes et al., 1990).                                                          conditions, likely cdc39-2 protein retains <br />    Several lines evidence presented strongly argue                    sufficient function repress basal transcription not <br /> that CDC39 primarily transcriptional regulatory protein.                 activation GCN4 derivatives GAL4. In regard, <br /> First, cdc39-2 mutation described increases                          Sc3861, fragment lacks sequences upstream the <br /> transcription unrelated genes involved                   putative TATA element presumably expresses less <br /> in pheromone response pathway. Although                               CDC39 protein, permits growth high temperature does <br /> quantitative effects individual genes modest,                negatively regulate GCN4-activated transcription <br /> phenotypic consequences, depend activity                      (assayed AT resistance). Although exclude the <br /> many genes, marked. Second, cdc39-2 strains arrest                        possibility repression basal activated transcription <br /> asynchronously cell cycle, transcriptional                   involves distinct functions, simplest interpretation these <br /> effects (assayed AT resistance) occur                        results negative regulation GCN4 (and GALA) <br /> pheromone response pathway inactivated. In regard,                    activation requires common CDC39 function than <br /> it previously noted CDC39 function essential                 negative regulation basal transcription. <br /> when pheromone response pathway inactivated                            Yeast cells contain number negative regulatory <br /> mutation a/a diploids (de Barros Lopes et al., 1990).                   proteins affect transcription genes the <br /> Third, loss CDC39 function affects Tc-dependent                            HMG1-like protein SINI (Kruger Herskowitz, 1991), <br /> transcription strongly TR-dependent transcription                   SIN3 (Wang et al., 1990; Vidal et al., 1991), putative <br /> of wild-type HIS3 gene, leading preferential                    helicase MOTI (Davis et al., 1992) SSN6-TUP1 <br /> increase initiation +1 site. Fourth, CDC39                        complex (Schultz et al., 1990; Williams Trumbly, 1990; <br /> encodes nuclear protein. For reasons, suggest                      Williams et al., 1991). These proteins function by <br /> that cell-cycle pheromone response phenotypes                         distinct molecular mechanisms. It suggested that <br /> caused original cdc39-1 mutation arise indirectly                   SINI functions C-terminal tail largest <br /> a consequence increased transcription genes responsible                 subunit RNA polymerase II (Peterson et al., 1991) and <br /> for functions.                                                          SSN6-TUP1 complex functions directly <br />                                                                               interacting specific DNA binding proteins (Keleher <br /> CDC39 negatively regulates basal activated                                et al., 1992). CDC39 functionally related MOTI <br /> transcription genes                                                   proteins essential cell growth and <br /> The basic observation cdc39-2, recessive                     constitutively activate pheromone response pathway. <br /> presumably loss-of-function mutation, causes increased RNA                    Given similarities GI-arrest mating response <br /> levels formally defines CDC39 transcriptional inhibitory                 phenotypes mutant strains, possible CDC36, <br /> protein. At restrictive temperature, cdc39-2 increases                CDC72, CDC73 SRM1 (Reed et al., 1988; Clark and <br /> basal (i.e. non-GCN4-activated) transcription HIS3, HIS4,                  Sprague, 1989; Barros Lopes et al., 1990) also <br /> TRP3 TFIID. We presume, tested,                         negative regulators transcription. <br /> GCN4-activated transcription HIS3, HIS4, TRP3 also <br /> increased absence CDC39 function. However,                          Potential molecular mechanisms <br /> DED] transcription appears unaffected, suggesting                        There general classes model explain how <br /> negative regulation CDC39 universal genes                    CDC39 negatively regulates basal acidic activator- <br /> transcribed RNA polymerase II. In addition, synthesis                      mediated transcription unrelated genes. In one <br /> 184 <br />                                                                                                Role CDC39 transcription regulation <br />  <br /> model, CDC39 involved maintaining normal                 1992). However, CDC39 affect general <br /> chromatin structure highly repressive transcription.   transcription factors (e.g. TFIIA, TFIIB, RNA polymerase <br /> Indeed, mutations reduce eliminate function         II) influence chromatin structure manner subtle <br /> histones result increased transcription genes           nucleosome disruption. <br /> (Clark-Adams et al., 1988; Han Grunstein, 1988; Han <br /> et al., 1988; Durrin et al., 1992). In alternative model, <br /> CDC39 inhibits activity general component the <br /> RNA polymerase II transcription machinery TFIID,           Materials methods <br /> TFIIB, TFIIA enzyme itself. Such inhibition           Yeast strains <br /> be mediated direct interaction               The starting strain used isolate suppressor mutations KY1600, <br /> component(s) indirectly intermediary proteins.          derivative KY803 (a ura3-52 trpl-A 1leu2-PET56 gal2 gcn4-A 1; Hope <br /> In regard, human Dr, NC 1 NC2 factors interact        Struhl, 1986) carries YCp88-Sc4363, URA3 centromeric vector <br />                                                                    expresses gcn4-C163 derivative constitutive DEDI promoter <br /> with TFIID repress basal activated transcription        (Hope Struhl, 1986), YCp87-Sc3866, LEU2 centromeric vector <br /> vitro (Meisterernst Roeder, 1991; Inostroza et al.,            containing his3-lacZ fusion protein control promoter <br /> 1992). Either classes explanation              containing optimal AP-1 site fused upstream HIS3 TATA region <br /> compatible non-equivalent effects CDC39                 (position -86). KY 1603, initial strain containing cdc39-2, generated <br />                                                                    spontaneous revertant KY 1600 grew glucose minimal plates <br /> different genes distinction basal          containing 20 mM aminotriazole (AT) blue X-gal indicator plates. <br /> activated transcription observed permissive                 To test effect cdc39-2 various GCN4 proteins, plasmids were <br /> temperature.                                                       cured KY1603 generate strain KY1617 transformed <br />    The striking mechanistically informative               YCp88 plasmids expressing various GCN4 derivatives (Hope and <br /> observation CDC39 preferential repression          Struhl, 1986; Hope et al., 1988). To determine phenotype loss of <br />                                                                    CDC39 function, cdc39:: URA3 substitution allele (Sc3865) was <br /> Tc-mediated transcription wild-type HIS3                  introduced one-step gene replacement (Rothstein, 1983) KY805, <br /> promoter. The stronger increase + 1 + 13                   a/ci diploid strain generated HO-mediated diploidization KY803 <br /> transcription novel, having observed          (Herskowitz Jensen, 1991). To eliminate mating pheromone response <br /> wide variety his3 promoter derivatives analyzed        pathway, ste4:: URA3 (Whiteway et al., 1989) stel2:: URA3 (Errede and <br /> past decade. The CDC39 effect clearly distinct             Ammerer, 1989) substitution alleles introduced KY803 and <br />                                                                    KY 1617 one-step gene replacement. The resulting strains then <br /> transcriptional induction acidic activator proteins       transformed pRS314-Sc4363 (a TRPI centromeric vector expressing <br /> is mediated exclusively TR results           gcn4-C 163 DED1 promoter). Isogenic derivatives KY1600 and <br /> preferential initiation + 13 (Struhl, 1986); loss       KY 1603 containing various his3 promoter alleles obtained two <br /> of CDC39 function lead increased transcription           successive gene replacement events (Struhl, 1987). <br /> by unmasking potential acidic activator proteins.           DNA manipulations <br /> Consistent view, GCN4 derivatives                   YCp87-Sc3866 generated substituting EcoRI-HindIlI fragment <br /> increasingly strong activation regions stimulate increasingly      Sc4099 (Hill et al., 1986) corresponding region Sc3802 (Singer <br /> more transcription + 13, cdc39-2 strains         et al., 1990) YCp87 vector (a LEU2 derivative YCp86 constructed <br /> GCN4-dependent activation + 1 site. Although Tc           Joan Sellers). pRS314-Sc4363 contains SmaI-EcoRI fragment of <br />                                                                    YCp88-Sc4363 (Hope Struhl, 1986) cloned pRS314 (Sikorski and <br /> does normally respond transcriptional activators            Hieter, 1989). <br /> (Struhl, 1986; Ponticelli Struhl, 1990), present results      CDC39 cloned complementation cdc39-2 allele. Specifically, <br /> suggest reflect inherent defect Tc       YCp5O library containing partial Sau3A fragments yeast genomic DNA <br /> but inhibition proteins CDC39.                   (obtained Mark Rose) introduced KY1617 containing <br />    In considering classes model CDC39               pRS314-Sc4363 screened AT-sensitive colonies. A plasmid <br />                                                                    containing 20 kb yeast DNA (Sc3860) recovered such <br /> function, disfavor view CDC39 involved           transformant shown complement phenotypes conferred the <br /> maintaining chromatin normally repressive state.            suppressor mutation. YCp50-Sc3861 generated religating largest <br /> Although TR Tc support comparable levels basal              EcoRI fragment YCp5O-Sc3860. YCp5O-Sc3862 obtained partially <br /> HIS3 TFIID-dependent transcription vivo (Cormack            digesting YCp5O-Sc3860 EcoRI digestion religating. Sc3863 was <br />                                                                    generated ligating partial EcoRI-SacI fragment YCp5O-Sc3862 into <br /> Struhl, 1992), TR efficiently binds TFHID supports        pRS316 (Sikorski Hieter, 1989). Sc3864 obtained ligating a <br /> TFIID-dependent transcription vitro (Ponticelli Struhl,     SacI-BglII fragment Sc3861 Sall-SacI fragment Sc3863 into <br /> 1990). Thus, loss CDC39 activity alleviated nucleosome       pRS316. For sequencing CDC39, deleted derivatives pRS316-Sc3864 <br />                                                                    generated subcloning restriction fragments digestion <br /> repression, expect observe transcriptional            exonuclease III S1 nuclease. Sc3865 obtained replacing <br /> patterns resembling obtained naked DNA templates          HindlIl fragments Sc3862 1.1 kb HindIII URA3 fragment. <br /> in vitro. Indeed, nucleosome loss vivo, obtained                To construct plasmid expressing flu-tagged CDC39 derivative, <br /> repressing histone H4 mRNA synthesis, leads increased           GCCATCGATAATGCTATCGGCCACACATACCG CGGGGTAC- <br /> HIS3 transcription exclusively mediated             CTTCTGCATTTAATAAGTCAAAGC used primers amplify <br />                                                                    CDC39 sequences ATG initiation codon proximal KpnI <br /> through TR (Durrin et al., 1992). In contrast, CDC39               site. The amplified DNA digested ClaI KpnI cloned into <br /> preferentially mediates repressive effects Tc.         pBS29 (constructed Alice Vincent), molecule containing NdeI site <br /> We favor idea CDC39 directly indirectly      upstream flu epitope. An NdeI blunted KpnI fragment the <br /> inhibits component(s) general transcription               resulting molecule KpnI-SacI fragment clone Sc3863 were <br />                                                                    cloned HindIII blunted SacI cleaved YCp92, derivative of <br /> machinery. TFIID attractive candidate             YCp88-GCN4 (Hope Struhl, 1986) constructed Brendan Cormack <br /> component functional distinctions           coding region replaced oligonucleotide containing <br /> TR Tc (Struhl, 1986; Mahadevan Struhl, 1990;               Sacl site stop codons frames. A SacI-EcoRI fragment <br /> Ponticelli Struhl, 1990) numerous           containing 3&apos; region CDC39 substituted 3&apos; region <br />                                                                    GCN4 complete gene encoding desired protein. A control <br /> proteins TFIID physically interacts (Dynlacht           construct flu tag N-terminus cloning EcoRT <br /> et al., 1991; Lee et al., 1991; Lieberman Berk, 1991;          blunted Sacl fragment Sc3861 SacI-EcoRI fragment from <br /> Meisterernst Roeder, 1991; Timmers Sharp, 1991;            CDC39 derivative containing flu tag HindUI blunted EcoRI cleaved <br /> Comai et al., 1992; Eisenmann et al., 1992; Inostroza et al.,      YCp92. <br />                                                                                                                                           185 <br />  M.A.Collart K.Struhl <br />  <br /> Phenotypic analyses                                                                 References <br /> GCN4 function assayed ability cells grow glucose minimal <br /> medium containing various concentrations AT described previously             Chen,W. Struhl,K. (1988) Proc. Natl. Acad. Sci. USA, 85, 2691-2695. <br /> (Hope Struhl, 1986; Hope et al., 1988). The degree AT resistance            Chen,W., Tabor,S. Struhl,K. (1987) Cell, 50, 1047-1055. <br /> is related ability GCN4 activate transcription HIS3 (which         Clark,K.L. Sprague,G.F.J. (1989) Mol. Cell. Biol., 9, 2682-2694. <br /> encodes enzyme competitively inhibited AT), histidine         Clark-Adams,C.D., Norris,D., Osley,M.A., Fassler,J.S. Winston,F. <br /> biosynthetic genes (which affect level substrate utilized           (1988) Genes Dev., 2, 150-159. <br /> HIS3 enzyme), ATRJ (which encodes membrane associated protein                Comai,L., Tanese,N. Tjian,R. (1992) Cell, 68, 965-976. <br /> that lowers intracellular AT levels) (Kanazawa et al., 1988). For strains          Cormack,B.P. Struhl,K. (1992) Cell, 69, 685-696. <br /> containing particular GCN4 derivative, degree AT resistance             Davis,J.L., Kunisawa,R. Thorner,J. (1992) Mol. Cell. Biol., 12, <br /> is directly related HIS3 RNA levels (Hill et al., 1986; Chen Struhl,           1879-1892. <br /> 1988). To assay GAL] expression, strains transformed 131 (Ma             Barros Lopes,M., Ho,J.-Y. Reed,S.I. (1990) Mol. Cell. Biol., 10, <br /> and Ptashne, 1987), centromeric plasmid containing GAL] -lacZ fusion              2966-2972. <br /> and grown supplemented minimal medium containing 2 % glucose 2 %             Dolan,J.W., Kirkman,C. Fields,S. (1989) Proc. Natl. Acad. Sci. USA, <br /> each glycerol ethanol + 2 % galactose. PH05 expression assayed                86, 5703-5707. <br /> by transforming cells pMH313 (Han Grunstein, 1988), high copy           Durrin,L.K., Mann,R.K. Grunstein,M. (1992) Mol. Cell. Biol., 12, <br /> PH05-lacZ fusion plasmid, growing resulting strains                        1621-1629. <br /> supplemented glucose minimal medium was depleted               Dynlacht,B.D., Hoey,T. Tjian,R. (1991) Cell, 66, 563-576. <br /> inorganic phosphate.                                                               Eisenmann,D.M., Arndt,K.M., Ricupero,S.L., Rooney,J.W. and <br />                                                                                      Winston,F. (1992) Genes Dev., 6, 1319-1331. <br />                                                                                    Errede,B. Ammerer,G. (1989) Genes Dev., 3, 1349-1361. <br /> RNA analysis                                                                       Han,M. Grunstein,M. (1988) Cell, 55, 1137-1145. <br /> For experiments carried permissive growth conditions, strains            Han,M., Kim,U.J., Kayne,P. Grunstein,M. (1988) EMBO J., 7, <br /> were grown overnight glucose minimal medium containing 1 % casamino               2221 -2228. <br /> acids selective plasmid maintenance, diluted A6 = 0.2 YPD,           Harbury,P.A.B. Struhl,K. (1989) Mol. Cell. Biol., 9, 5298-5304. <br /> and grown 4 h. Temperature-shift experiments performed                 Herskowitz,I. Jensen,R.E. (1991) Methods Enzymol., 194, 132-146. <br /> described previously (Cormack Struhl, 1992). CeHls grown YPD           Hill,D.E., Hope,I.A., Macke,J.P. Struhl,K. (1986) Science, 234, <br /> at 30&#194;&#176;C A6W = 0.3, shifted 37&#194;&#176;C 20 min, returned 30&#194;&#176;C                451 -457. <br /> 1 h, shifted restrictive temperature (typically 38.5&#194;&#176;C).          Hope,I.A. Struhl,K. (1986) Cell, 46, 885-894. <br /> Aliquots (20 ml) taken shift, second shift,       Hope,I.A., Mahadevan,S. Struhl,K. (1988) Nature, 333, 635-640. <br /> and various times second shift, cells quickly pelleted   Inostroza,J.A., Mermelstein,F.H., Ha,I., Lane,W.S. Reinberg,D. <br /> and frozen -80&#194;&#176;C. This protocol avoids looking consequences                (1992) Cel <br /> </body></html>