<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Wu et al. BMC Cancer 2014, 14:107 <br /> http://www.biomedcentral.com/1471-2407/14/107 <br />  <br />  <br />  <br />  <br />  RESEARCH ARTICLE                                                                                                                               Open Access <br />  <br /> miR-372 down-regulates oncogene ATAD2 <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> influence hepatocellular carcinoma proliferation <br /> and metastasis <br /> Gang Wu1*, Haiyang Liu1, Hui He1, Yawei Wang1, Xiaojun Lu1, Yanqiu Yu2, Shuguan Xia1, Xiangyu Meng1 <br /> and Yongfeng Liu1 <br />  <br />  <br />   <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />   Background: ATAD2 associated cellular processes, cell growth, migration invasion. However, <br />   studies conducted molecular biological function ATAD2 gene hepatocellular <br />   carcinoma (HCC). <br />   Methods: The protein mRNA level expression ATAD2 examined tissues cell lines. Prognostic <br />   significance analyzed Kaplan-Meier <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>survival</span> method Cox regression. ATAD2 knockdown used <br />   analyze cell proliferation invasion. The upstream downstream ATAD2 analyzed RT 2 Profiler&#226;&#8222;&#162; <br />   PCR array luciferasex fluorescence system. <br />   Results: ATAD2 highly expressed liver cancer samples correlated poor survival. High ATAD2 <br />   expression positively correlated metastasis (P = 0.005) independent prognostic factor in <br />   HCC (P = 0.001). ATAD2 depletion <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> reduced capacity invasion proliferation and <br />   led G1 phase arrest vitro. Further study revealed miR-372 upstream target ATAD2 miR-372 was <br />   bound directly 3&#226;&#8364;&#178; untranslated region (3&#226;&#8364;&#178; UTR). In addition, ATAD2 knockdown extremely up-regulate <br />   APC expression down-regulate CTNNA1 mRNA level. <br />   Conclusions: The findings demonstrated miR-372 suppressed expression ATAD2, highly <br />   expressed HCC exerted proto-oncogene effect hepatic carcinogenesis. In conclusion, ATAD2 may <br />   promote HCC progression. <br />  <br />  <br /> Background                                                                            reliable prognostic biomarker, help <br /> HCC fifth common malignant tumor                                          clinicians predict characteristics malignancy <br /> worldwide, incidence approximately 626,000                                 decrease rate unfavorable outcomes a <br /> cases year [1,2]. In China Southeast Asia, HCC                               high-risk population. <br /> is highly associated viral hepatitis B cirrhosis [3].                          ATAD2 (ATPase family AAA domain-containing protein <br /> The prognosis patients HCC largely improved                               2), member AAA + ATPase family proteins, <br /> due extensive advances surgical techniques diag-                            identified microarray analysis [6], contains both <br /> nostic methods recent years. However, long-term                                bromodomain ATPase domain, maps to <br /> prognosis unsatisfactory, largely high                            chromosome 8q24 region frequently amplified <br /> recurrence invasion rates resection (50-70%                            cancer [7]. The structure ATAD2 suggests has <br /> at years) [4,5]. Unfortunately, little known                             functions related genome regulation, including cell <br /> respect molecular mechanisms underlying                                   proliferation, differentiation apoptosis. Studies have <br /> aggressive behavior. Therefore, great demand                              revealed ATAD2 highly expressed types <br />                                                                                       tumors breast cancer, lung cancer, large <br /> * Correspondence: wugang@mail.cmu.edu.cn                                              B-cell lymphoma [8-13]. And recently Huang Q et al. has <br /> 1 <br />  Department General Surgery, First Affiliated Hospital China Medical <br /> University, Shenyang, Liaoning 110001, China <br />                                                                                       reported novel, highly up-regulated exon-exon junc- <br /> Full list author information available end article                tion detected ATAD2 gene RNA-seq the <br />                                          &#194;&#169; 2014 Wu et al.; licensee BioMed Central Ltd. This Open Access article distributed terms Creative <br />                                          Commons Attribution License (http://creativecommons.org/licenses/by/2.0), permits unrestricted use, distribution, and <br />                                          reproduction medium, provided original work properly credited. The Creative Commons Public Domain <br />                                          Dedication waiver (http://creativecommons.org/publicdomain/zero/1.0/) applies data available article, <br />                                          unless stated. <br />  Wu et al. BMC Cancer 2014, 14:107                                                                                     Page 2 11 <br /> http://www.biomedcentral.com/1471-2407/14/107 <br />  <br />  <br />  <br />  <br /> gene highly expressed HCC tissues [14]. However,          Table 1 Distribution ATAD2 Clinicopathological <br /> there studies gene function              Characteristics HCC patients <br /> prognosis associated ATAD2 HCC.                          Characteristics Number      ATAD2          Negative     P <br />    In present study, detected expression                           patients overexpression weak expression <br /> ATAD2 HCC matched adjacent non-cancerous                  Total cases       129     83             46 <br /> liver tissues. Different methods used determine          Age (years) <br /> the relationships expression ATAD2,             &#226;&#8240;&#165;60             54      34(63.0%)      20(37.0%)          0.782 <br /> clinical relevance, overall survival (OS) resec-     &lt;60             75      49(65.3%)      26(34.7%) <br /> tion. In addition, effects ATAD2 expression cell       Gender <br /> invasion metastasis investigated SMMC7721,             Male            106     69(65.1%)      37(34.9%)          0.701 <br /> QGY-7701, Bel-7402, PLC5, Huh7, HCCLM3, HepG2, <br />                                                                    Female          23      14(60.9%)      9(39.1%) <br /> and LO2 cell lines using small interfering RNAs. Also the <br />                                                                  Tumor size <br /> miR-372 identified direct functional target <br />                                                                    &#226;&#8240;&#165; 5cm           83      53(63.9%)      30(36.1%)          0.877 <br /> for ATAD2 hepatic carcinogenesis. Therefore, both <br /> ATAD2 miR-372 appear good targets study               &lt; 5cm           46      30(65.2%)      16(34.8%) <br /> in research.                                                 Metastasis <br />                                                                    Yes             72      54(75%)        18(25%)            0.005 <br /> Methods                                                            No              57      29(50.9%)      28(49.1%) <br /> Patient tissue samples                                           HBsAg status <br /> Primary tumor samples obtained 129 patients              Positive        73      43(58.9%)      30(41.1%)          0.141 <br /> (106 males 23 females) diagnosed                 Negative        56      40(71.4%)      16(28.6%) <br /> HCC undergone routine hepatic resection           Tumor <br /> First Affiliated Hospital China Medical University            differentiation <br /> between January 2006 December 2009. The follow-up              WD              54      34(63.0%)      20(37.0%)          0.955 <br /> period survivors 5 years. None patients         MD              60      39(65.0%)      21(35.0%) <br /> received preoperative radiotherapy chemotherapy prior <br />                                                                    PD              15      10(66.7%)      5(33.3%) <br /> to surgical resection. The histological diagnosis differ- <br />                                                                  Cirrhosis <br /> entiation evaluated independently pathologists <br />                                                                    Yes             86      58(67.4%)      28(32.6%)          0.298 <br /> according WHO classification [15]. The <br /> clinicopathological features shown Table 1. Fresh           No              43      25(58.1%)      18(41.9%) <br /> specimens snap-frozen liquid nitrogen stored         Serum AFP <br /> at &#226;&#710;&#8217;70&#194;&#176;C immediately resection extraction         &lt;200 ng/dl      41      29(70.7%)      12(29.3%)          0.301 <br /> RNA protein. The project protocol approved              &#226;&#8240;&#165;200 ng/dl      88      54(61.4%)      34(38.6%) <br /> the Institutional Ethics Committee China Medical              Tumor stage <br /> University prior initiation study. All patients      TNM I+II        42      25(59.5%)      17(40.5%)          0.427 <br /> provided written informed consent use               TNM III+IV      87      58(66.7%)      29(33.3%) <br /> tumor tissues clinical research. <br />  <br /> Liver cancer cell lines cell cultures                        followed two-step PCR 40 cycles 95&#194;&#176;C 5 s <br /> The liver cancer cell lines, SMMC7721, QGY-7701,                 64&#194;&#176;C 30 s. Each reaction performed we <br /> Bel-7402, PLC5, Huh7, HCCLM3, HepG2, normal              previously described [3]. The primer sequences as <br /> liver cell line, LO2, obtained Shanghai Cell       follows: ATAD2 forward, 5&#226;&#8364;&#178;-GGAATCCCAAACCACT <br /> Bank (Shanghai, China). SMMC7721, QGY-7701, Bel-7402             GGACA-3&#226;&#8364;&#178;; <br /> and PLC5 cells cultured RPMI-1640 (Invitrogen,             ATAD2 reverse, 5&#226;&#8364;&#178;-GGTAGCGTCGTCGTAAAGCAC <br /> Carlsbad, CA). Huh7, HCCLM3, HepG2 LO2 cells                 A-3&#226;&#8364;&#178;; <br /> were grown DMEM (Invitrogen). All media                    GAPDH forward, 5&#226;&#8364;&#178;-ATAGCACAGCCTGGATAGCA <br /> supplemented 10% fetal calf serum (Invitrogen)          ACGTAC-3&#226;&#8364;&#178;; <br /> 100 IU/ml penicillin (Sigma, St. Louis, MO).                       GAPDH reverse, 5&#226;&#8364;&#178;-CACCTTCTACAATGAGCTGC <br />                                                                  GTGTG-3&#226;&#8364;&#178;. <br /> RNA preparation quantitative real-time PCR <br /> Total RNA extracted according manufacturer&#226;&#8364;&#8482;s          Western blots <br /> instructions. RT-qPCR performed using SYBR Pre-            Total protein extracted tumor tissues, non-tumor <br /> mix Ex Taq (TaKaRa) Thermal Cycler Dice Real Time           adjacent tissues liver cancer cell lines using Nuclear <br /> System (TaKaRa) following protocol: 30 s 95&#194;&#176;C        Cytoplasmic Protein Extraction Kit (Beyotime, China). <br />  Wu et al. BMC Cancer 2014, 14:107                                                                                   Page 3 11 <br /> http://www.biomedcentral.com/1471-2407/14/107 <br />  <br />  <br />  <br />  <br /> Fifty micrograms total nuclear protein separated           transfections, 24 h cells seeded 6-well <br /> by SDS-PAGE electrotransferred PVDF                   plate, transfected ATAD2 siRNA control <br /> membranes (Millipore, Billerica, MA, USA). Milk (5%)          siRNA using DharmaFECT 1 according manu- <br /> used block membranes 2 hours room temperature.          facturer&#226;&#8364;&#8482;s protocol. The mRNA protein levels were <br /> After blocking, primary antibodies, including ATAD2,              detected 48 h later. <br /> APC, CTNNA1 rabbit polyclonal antibody GAPDH <br /> mouse polyclonal antibody (both diluted 1:1000, Sigma, <br /> St. Louis, MO, USA), incubated membranes            Cell cycle analysis <br /> overnight 4&#194;&#176;C. The secondary antibodies              Huh7 HCCLM3 cells 6-well plates transfected <br /> incubated 2 hours room temperature. Protein bands          ATAD2 siRNA negative control siRNA. After 48 h <br /> were identified using ECL (Millipore, Bedford,          transfection, cells seeded density 5 &#195;&#8212; 105 well <br /> MA, USA)                                                          trypsinized, fixed 70% ethanol 4&#194;&#176;C, washed <br />                                                                   PBS. A quantity 100 &#206;&#188;L RNase A added, and <br /> Immunohistochemistry                                              mixture incubated 37&#194;&#176;C water bath 30 min. <br /> ATAD2 expression analyzed paraffin-embedded                An additional 400 &#206;&#188;L PI staining solution added and <br /> specimens 129 patients. Tissue sections 4-&#206;&#188;m              incubated 4&#194;&#176;C dark 30 min; was <br /> thickness deparaffinized xylene dehydrated            used detect record red fluorescence upon <br /> before antigen retrieval 5 min autoclave.             excitation wavelength 488 nm. <br /> Hydrogen peroxide (0.3%) used block endogenous <br /> peroxidase activity; nonspecific immunoglobulin binding           CCK8 Colony formation assay <br /> sites blocked normal goat serum 30 min             Cells plated 96-well plates media containing <br /> 37&#194;&#176;C. Tissue sections incubated anti-ATAD2              10% FBS approximately 2,000 cells well, 24 h after <br /> (1:200, Sigma) overnight 4&#194;&#176;C. Biotinylated goat anti-          transfection. Then, 10 &#206;&#188;l CCK8 (Thiazolyl Blue) solu- <br /> rabbit immunoglobulin G used secondary antibody          tion added incubated 1 h 37&#194;&#176;C. <br /> (1:300, Sigma). After washing, sections incubated        The results quantified spectrophotometrically using <br /> with streptavidin-biotin conjugated HRP 30 min,          test wavelength 450 nm. <br /> and peroxidase reaction developed 3, 3&#226;&#8364;&#178;-di-             After transfection, logarithmic growth phase cells in <br /> aminobenzidine tetrahydrochloride (DAB).                          monolayer culture prepared colony formation <br />                                                                   assay. Cells plated 6-well plates media contain- <br /> Semi-quantitative assessment scoring                          ing 10% FBS approximately 200 cells well. Colony <br /> ATAD2 expression levels scored semiquantitatively            formation allowed proceed weeks. <br /> according percent positively stained cells com-         Cells washed 1 ml PBS, fixed, stained with <br /> bined staining intensity. Samples considered        500 &#206;&#188;l 0.1% crystal violet solution 20 min, and <br /> positive ATAD2 nucleus cytoplasm sam-        finally washed times 1 ml water. The fixed <br /> ple cells presented positive staining. The percent positivity   cell colonies allowed air dry. The clone formation <br /> was defined &#226;&#8364;&#339;0&#226;&#8364;? 0%, &#226;&#8364;&#339;1&#226;&#8364;? 1-10%, &#226;&#8364;&#339;2&#226;&#8364;? 11-50%, &#226;&#8364;&#339;3&#226;&#8364;?     rate calculated. <br /> 51-80%, &#226;&#8364;&#339;4&#226;&#8364;? &gt;80%. The staining intensity scored <br /> as &#226;&#8364;&#339;0&#226;&#8364;? (no staining), &#226;&#8364;&#339;1&#226;&#8364;? (weakly stained), &#226;&#8364;&#339;2&#226;&#8364;? (moderately <br /> stained) &#226;&#8364;&#339;3&#226;&#8364;? (strongly stained). Both percent posi-       Cell invasion migration assay <br /> tivity staining intensity assessed            Huh7 HCCLM3 cells infected ATAD2 <br /> doubly blinded investigators. The ATAD2 expression                siRNA 48 h. Cells seeded synthetic <br /> score calculated value percent positivity         basement membrane present inset 24-well <br /> score &#195;&#8212; staining intensity score. This value ranged      culture plate. In invasion assay, polycarbonate filters <br /> <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> 0 12, tumors classified             coated 50 &#206;&#188;L Matrigel (1:9, BD Bioscience) were <br /> following: negative (&#226;&#710;&#8217;), score 0; lower expression (1+),          placed Transwell chamber (Costar). In migration <br /> score 1&#226;&#8364;&#8220;4; moderate expression (2+), score 5&#226;&#8364;&#8220;8;               assay, Matrigel placed chambers. Fetal <br /> strong expression (3+), score 9&#226;&#8364;&#8220;12. The immunohisto-              bovine serum added lower chamber a <br /> chemical ATAD2 staining grouped categories:          chemoattractant. Cells incubated 37&#194;&#176;C <br /> low expression (0/1+) high expression (2+/3+).                allowed invade Matrigel barrier for <br />                                                                   hours. After incubation, filters fixed stained <br /> Depletion ATAD2 small interfering RNAs                      0.1% crystal violet solution. Non-invading cells <br /> On-TargetPlus SMARTpool siRNA ATAD2 (No.                      removed using cotton swab, invading cells on <br /> LU-017603-00-0002) On-TargetPlus siControl (D-                underside filter counted inverted <br /> 001810-01-20) purchased Dharmacon. For                  microscope. <br />  Wu et al. BMC Cancer 2014, 14:107                                                                                Page 4 11 <br /> http://www.biomedcentral.com/1471-2407/14/107 <br />  <br />  <br />  <br />  <br /> Real-time polymerase chain reaction gene array                  Association ATAD2 expression HCC patient <br /> Forty-eight hours siRNA knockdown, RNA                clinical outcome <br /> extracted cells using Trizol (Invitrogen)          Patients high ATAD2 expression metastases and <br /> cleaned RNeasy_MinEluteTM Cleanup Kit (Qiagen,         poorer prognosis (Table 1). The overall survival sig- <br /> Valencia, CA). Subsequently, total RNA reverse              nificantly lower patients high ATAD2 expression <br /> transcribed using SuperScript III Reverse Transcriptase         patients low ATAD2 expression (P &lt; 0.05, <br /> (Qiagen), complementary DNA amplified                Figure 2). In addition, multivariate analysis demonstrated <br /> the polymerase chain reaction (PCR) using 2_Super Array         ATAD2 status, tumor size, metastasis AFP <br /> PCR master mix (Qiagen). Real-time PCR per-            status significant prognostic factors HCC patients <br /> formed sample Human Tumor Metastasis           (Table 2). <br /> RT2 ProfilerTM PCR Array (SuperArray Bioscience) a <br /> Thermal Cycler Dice Real Time System (Takara TP800,             Depletion ATAD2 inhibits tumor cell growth liver <br /> Japan) according manufacturer&#226;&#8364;&#8482;s instructions. Data       cancer cell lines <br /> were normalized GAPDH levels 2-&#206;&#8221;&#206;&#8221;Ct method.           A significant reduction proliferation rate was <br />                                                                 observed CCK8 assay 2 days transfection <br /> Luciferase assays                                               ATAD2 siRNA compared negative control <br /> 293 T cells plated 24-well plates 80% confluence   siRNA (Figure 3a). Consistent CCK8 assay, the <br /> 24 hours transfection. A mixture 200 ng pGL3-3&#226;&#8364;&#178;       depletion ATAD2 Huh7 (Neg. siRNA vs. ATAD2 <br /> UTR, 700 ng pGV214-miR-372, 100 ng Renilla lu-              siRNA: 124 &#194;&#177; 16 vs. 36 &#194;&#177; 10, P &lt; 0.001) HCCLM3 (Neg. <br /> ciferase plasmid transfected 293 T cells using        siRNA vs. ATAD2 siRNA: 102 &#194;&#177; 22 vs. 39 &#194;&#177; 13, P &lt; 0.001) <br /> Lipofectamine 2000. Firefly Renilla luciferase activities   cells led significant reduction number size <br /> were calculated using dual-luciferase reporter         foci (Figure 3b). The DNA content determined using <br /> (Promega, Madison, WI).                                         flow cytometry demonstrated ATAD2 siRNA transfec- <br />                                                                 tion increased percentage cells G1 phase de- <br />                                                                 creased S phase cell lines (Figure 4). <br /> Statistical analysis <br /> SPSS 17.0 software Windows used. The association        Invasive migratory capacity liver cancer cells is <br /> between ATAD2 expression HCC patients&#226;&#8364;&#8482; clinico-             decreased ATAD2 knockdown <br /> pathological features evaluated &#207;&#8225;2 test. The         Cell invasion migration assays demonstrated that <br /> Wilcoxon test performed compare data            Huh7 HCCLM3 liver cancer cell lines were <br /> densitometry analysis mRNA protein expression.           transfected ATAD2 siRNA displayed attenu- <br /> The Kaplan-Meier method used calculate pa-           ated invasive migratory capacities the <br /> tients&#226;&#8364;&#8482; survival log-rank test. A Cox repression model     negative controls (Figure 5). <br /> was performed univariate multivariate analysis        The depletion ATAD2 Huh7 (control vs. ATAD2 <br /> of prognostic variables. All P values reported two-sided,   siRNA: 63 &#194;&#177; 11 vs. 33 &#194;&#177; 7, *P &lt; 0.05) HCCLM3 <br /> and P &lt; 0.05 considered statistically significant.           (control vs. ATAD2 siRNA: 78 &#194;&#177; 14 vs. 32 &#194;&#177; 9, *P &lt; 0.05) <br />                                                                 cells led significant reduction invasive cells <br /> Results                                                         (Figure 5a-b). The migration Huh7 (control vs. ATAD2 <br /> Expression Clinical significance ATAD2 protein           siRNA: 80 &#194;&#177; 11 vs. 51 &#194;&#177; 6,*P &lt; 0.05) HCCLM3 <br /> expression HCC                                               (control vs. ATAD2 siRNA: 70 &#194;&#177; 12 vs. 31 &#194;&#177; 8, *P &lt; 0.05) <br /> The median ATAD2 expression level primary HCCs           cells significantly reduced (Figure 5c-d). <br /> nearly two-fold higher non-tumorous livers (median <br /> expression = 0.158 0.083, respectively), (P &lt; 0.05          ATAD2 regulated APC CTNNA1 expression HCC cells <br /> [Wilcoxon signed-rank test]) (Figure 1a). ATAD2             To obtain insight functions ATAD2 in <br /> overexpressed 83 129 tumor samples (83/129, 64.3%)        HCC cell invasion metastasis, mRNA expression <br /> according IHC. The ATAD2 protein appeared              profile siATAD2-transfected Huh7 cells compared <br /> expressed nucleus cytoplasm tumor        si-negative transfected cells using Human <br /> cells, stronger expression nucleus (Figure 1b).     Tumor Metastasis RT2 Profiler&#226;&#8222;&#162; PCR Array containing 84 <br /> Compared normal cell line LO2, Huh7                  cell metastasis-related genes. Six upregulated genes and <br /> HCCLM3 cells showed relatively higher levels ATAD2           downregulated genes (at 2-fold) identified <br /> expression seven liver cancer cell lines Western          siATAD2-transfected Huh7 cells compared si-negative <br /> blotting (Figure 1c). At protein level, 6/8 (75%) HCC       transfected controls (Additional file 1: Table S1). Subse- <br /> samples showed upregulated ATAD2 expression compared            quently, APC CTNNA1, showed greatest <br /> to adjacent normal liver tissues (Figure 1d).             fold change (3-fold more) ATAD2 knockdown, <br />  Wu et al. BMC Cancer 2014, 14:107                                                                                                     Page 5 11 <br /> http://www.biomedcentral.com/1471-2407/14/107 <br />  <br />  <br />  <br />  <br />  Figure 1 The ATAD2 expression level higher HCC samples liver cancer cell lines. (a) The ATAD2 mRNA expression up-regulated <br />  tumors compared paired nontumour patient samples. (b) IHC staining ATAD2 HCC. (c) Huh7 HCCLM3 cells showed relatively higher <br />  levels ATAD2 expression liver cancer cell lines. (d) ATAD2 protein expression tumor paired non-tumor samples was <br />  determined Western blotting. <br />  <br />  <br />  <br /> were selected analyzed ATAD2 45 HCC                            CTNNA1 HCC tissues (R2 = 0.327, P = 0.01). At <br /> tissues real time PCR using linear correlation statistics               protein level, trend observed Western <br /> (Figure 6a b). Consistent results                    blotting (Figure 6c). <br /> real-time PCR array, expression levels ATAD2 <br /> and APC mRNA negatively correlated (R2 = 0.429, <br /> P &lt; 0.01), ATAD2 expression positively correlated                  Oncogenic ATAD2 direct downstream target for <br />                                                                            miR-372 <br />                                                                            We miR-372 expression level lower <br />                                                                            HCC adjacent normal liver tissues (Figure 7a) <br />                                                                            negatively correlated ATAD2 ex- <br />                                                                            pression (r = -0.5237, P = 0.0002; Figure 7b) RT-qPCR. <br />                                                                            According prediction websites (TargetScan, PicTar and <br />                                                                            miRanda), searched target genes miR-372 <br />                                                                            ATAD2 contains potential miR-372 <br />                                                                            binding site 3&#226;&#8364;&#178; UTR (Figure 7c). This sequence is <br />                                                                            highly conserved different species (human, chim- <br />                                                                            panzee, mouse, rat, pig, rabbit) (Figure 7d). There- <br />                                                                            fore, constructed vectors containing wild-type or <br />                                                                            mutant 3&#226;&#8364;&#178; UTR ATAD2 fused downstream the <br />                                                                            firefly luciferase gene. The wild-type mutant vector <br />                                                                            co-transfected 293 T cells miR-372 expres- <br />                                                                            sion construct, transfection efficiency normal- <br />                                                                            ized co-transfection Renilla luciferase vector. As <br />                                                                            shown Figure 7e, miR-372 significantly decreased the <br />                                                                            relative luciferase activity wild-type ATAD2 3&#226;&#8364;&#178; UTR <br />                                                                            (40%) compared mutant 3&#226;&#8364;&#178; UTR, indicating <br />  Figure 2 Overall survival HCC patients relation ATAD2            miR-372 directly bind 3&#226;&#8364;&#178; UTR of <br />  status. Survival HCC patients low ATAD2 expression versus <br />                                                                            ATAD2. Thus, suggest ATAD2 direct target of <br />  high expression. <br />                                                                            miR-372. <br />  Wu et al. BMC Cancer 2014, 14:107                                                                                                 Page 6 11 <br /> http://www.biomedcentral.com/1471-2407/14/107 <br />  <br />  <br />  <br />  <br /> Table 2 Univariate multivariate analyses individual parameters correlations overall survival rate: Cox <br /> proportional hazards model <br /> Variables                                                   Univariate                                          Multivariate <br />                                         HR                  CI (95%)     P value               HR               CI (95%)               P value <br /> ATAD2                                 1.843                1.220-2.784    0.004a              2.221            1.444-3.416             0.001a <br /> Age                                   1.253                0.856-1.542     0.407 <br /> Gender                                1.176                0.601-1.583     0.921 <br /> Tumor stage                           1.122                0.920-1.367     0.255 <br /> HBsAg                                 1.150                0.792-1.670     0.464 <br /> Tumor differentiation                 1.028                0.784-1.347     0.843 <br /> Metastasis                            1.816                1.232-2.678    0.003a              1.822            1.221-2.718             0.003a <br />                                                                                a <br /> Tumor size                            1.573                1.059-2.336    0.025               1.909            1.276-2.855             0.002a <br /> Serum AFP                             1.670                1.110-2.512    0.014a              1.992            1.306-3.038             0.001a <br /> Liver cirrhosis                       1.056                0.640-1.429     0.827 <br /> Abbreviation: HR, Hazard Ratio; CI, Confidence Interval. <br /> a. <br />   Statistically significant (p &lt; 0.05). <br />  <br />  <br />  <br /> Discussion                                                               HCC studied far. IHC showed that <br /> It demonstrated ATAD2 novel candidate                      ATAD2 expression HCC tissues significantly <br /> oncogene possibly therapeutic target                   higher adjacent non-tumor tissues. In addition, <br /> types human cancer [5,7,12,16]. However, abnormal                 upregulation ATAD2 observed majority <br /> expression ATAD2 possible carcinogenesis                   HCCs compared adjacent normal liver tissues <br />  <br />  <br />  <br />  <br />   Figure 3 The CCK8 assay performed ATAD2 siRNA treatment. (a) A reduction absorbance observed (P &lt; 0.05). (b) <br />   Clonogenic assays performed ATAD2-depleted cancer cells. The number colonies formed cells treated ATAD2 siRNA far <br />   fewer control siRNA-treated cells (P &lt; 0.05). <br />  Wu et al. BMC Cancer 2014, 14:107                                                                                      Page 7 11 <br /> http://www.biomedcentral.com/1471-2407/14/107 <br />  <br />  <br />  <br />  <br />  Figure 4 siRNA-mediated ATAD2 knockdown reduced Huh7 HCCLM3 cell proliferation led G1 phase cell cycle arrest. <br />  <br />  <br /> by Western blotting. These findings provide evidence         closely involved key regulatory mechanisms <br /> the upregulation ATAD2 play important role           control cell proliferation tumor metastasis through <br /> HCC tumorigenesis.                                                structure domain [18,19]. Taken together, data <br />   Further correlation analyses indicated high            provide evidence ATAD2 important in <br /> expression ATAD2 HCC tissues positively             HCC cell proliferation involved carcinoma <br /> correlated tumor metastasis. These results demon-            cell migration invasion. <br /> strated upregulation ATAD2 HCC                  However, molecular mechanisms ATAD2 <br /> play important role promoting malignant tumors.             regulates cancer cell proliferation invasion remain <br /> Similar results observed human malig-          unclear. From Human Tumor Metastasis RT-PCR array <br /> nancies, esophageal, gastric, colon, breast can-      ATAD2 siRNA-treated cells, identified APC, <br /> cers [12,16], overexpression ATAD2            ITGB3, FXYD5, ITGA7 CTNNA1 (&#206;&#177;-catenin), which <br /> gene observed aggressive tumor sub-             related cell adhesion; these, expression <br /> groups provided diagnostic value. Furthermore,             APC CTNNA1 changed 3-fold. <br /> found high expression ATAD2 HCC                 APC, negatively regulates &#206;&#178;-catenin, plays an <br /> strong independent predictor shortened overall             important role development HCC. In mice <br /> survival.                                                         lacking APC liver, &#206;&#178;-catenin stabilized, which <br />   With regard gene function, bromodomain               significantly increased incidence HCC [20]. Colnot <br /> encoded ATAD2 contributes high-affinity recog-          et al. [20] 67% APC(lox/lox) mice developed <br /> nition acetylated lysine [16], frequently       HCC Cre adenovirus injected. The &#206;&#178;-catenin <br /> in chromatin remodeling histone acetylation [17]. The         signaling mice strongly activated, indicating <br /> bromodomain play important role cell growth,            loss APC function led Wnt signaling <br /> proliferation, invasion apoptosis. In present study,      pathway activation caused occurrence tumors. <br /> ATAD2 protein detectable Western blotting              CTNNA1 functions complex CTNNB1, where <br /> seven cell types studied. The Huh7 HCCLM3 cells               acts tether cytoplasmic domain E-cadherin to <br /> had relatively higher levels endogenous ATAD2,             cytoskeleton [21,22]. Vasioukhin et al. inhi- <br /> the depletion ATAD2 transfection siRNA              biting CTNNA1 destablizes adherens junctions, which <br /> these cell lines led G1 phase cell cycle arrest      weakens interaction cells makes them <br /> reduced cell growth/proliferation. In addition, siRNA-            sensitive contact-mediated inhibition [23]. Previous <br /> mediated ATAD2 knockdown significantly inhibit              studies showed reduced CTNNA1 expression some <br /> cell migration cell invasion. This result agree-       solid tumors associated increased malignant <br /> ment previous studies showed ATAD2             behavior [24-26]. However, studies showed that <br />  Wu et al. BMC Cancer 2014, 14:107                                                                                                        Page 8 11 <br /> http://www.biomedcentral.com/1471-2407/14/107 <br />  <br />  <br />  <br />  <br />  Figure 5 Transwell assays Huh7 HCCLM3 cells transfected control ATAD2 siRNA. (a). ATAD2 depletion measurable <br />  inhibitory effect cell invasion cell lines. (b). The numbers invading cells counted, significant difference observed <br />  (*P &lt; 0.05). (c). A significant difference observed cell lines (d) numbers migrating cells counted (*P &lt; 0.05). <br />  <br />  <br />  <br />  <br />  Figure 6 APC CTNNA1 expression associated ATAD2 45 HCC tissues. (a) Real-time PCR results showed ATAD2 APC ex- <br />  pression negatively correlated HCC tissues (P &lt; 0.01). (b) ATAD2 expression positively correlated CTNNA1 (P &lt; 0.01). (c) At pro- <br />  tein level, trend observed Western blotting. <br />  Wu et al. BMC Cancer 2014, 14:107                                                                                                      Page 9 11 <br /> http://www.biomedcentral.com/1471-2407/14/107 <br />  <br />  <br />  <br />  <br />  Figure 7 The relationship miR-372 ATAD2. (a) Significant down-regulation miR-372 paired HCC/NT samples defined <br />  log2-fold change &lt; &#226;&#710;&#8217;1. (b) miR-372 expression negatively correlated ATAD2 expression 45 paired samples. (c) Luciferase assay <br />  wild-type miR-372 binding site-mutated ATAD2-3&#226;&#8364;&#178; UTR presence miR-372 expression plasmid. Mutations underlined and <br />  italicized. (d) miR-372 binding site ATAD2 3&#226;&#8364;&#178; UTR different species. The seed sequence white box. (e) Relative luciferase <br />  activity analyses. The relative luciferase activity pGV214 group set 1. <br />  <br />  <br />  <br />  <br /> CTNNA1 highly expressed poorly differ-                          ATAD2 significantly negatively corre- <br /> entiated HCC well-differentiated HCC [27,28].                       lated APC CTNNA1 expression. <br />   CTNNA1 CTNNB1 link cadherins cytoskel-                            Moreover, studied upstream mechanism <br /> eton epithelial cell-cell adherens junction complexes                    regulating ATAD2 expression HCC. miR-372 belongs <br /> [29]. As known, addition formation                 miR-371-373 gene cluster includes miR-93 <br /> E-cadherin-catenin complex, CTNNB1 associated                       miR-302a [30]. These microRNAs play important <br /> the Wnt/Wingless signal transduction pathway binds                      role development types human malignant <br /> to APC, EGFR c-erbB2 proto-oncogene. Thus,                          tumors. One report noted miR-372 oncogene in <br /> whether ATAD2 affect APC CTNNA1 foster                         human glioma; highly expressed glioma cells <br /> tumorigenesis activating WNT pathway goal                       normal brain cells closely related the <br /> for research.                                                       prognosis glioma [31]. In gastric cancer cells, Cho <br />   In present study, detected changes ex-                      et al. miR-372 promote cell proliferation <br /> pression genes&#226;&#8364;&#8221;upregulated APC down-                             inhibit apoptosis; biological roles caused by <br /> regulated CTNNA1&#226;&#8364;&#8221;after knocking ATAD2                               acting downstream LATS2 gene [32]. LATS2, which <br /> Huh7 cells, validated changes protein                   important tumor suppressor, plays important role <br /> level Western blotting. It appears HCC                       inhibiting cell proliferation gastric cancers. In the <br /> cells, ATAD2 regulates cell migration/invasion                      present study, miR-372 negatively cor- <br /> regulation APC and/or CTNNA1 expression. Through                         related ATAD2. Through luciferase reporter gene <br /> real time PCR detection 45 paired tissue samples,                     assay, confirmed ATAD2 mRNA 3&#226;&#8364;&#178; <br />  Wu et al. BMC Cancer 2014, 14:107                                                                                                                       Page 10 11 <br /> http://www.biomedcentral.com/1471-2407/14/107 <br />  <br />  <br />  <br />  <br /> non-coding region (3&#226;&#8364;&#178; UTR) binding site                                  8.    Alizadeh AA, Eisen MB, Davis RE, Ma C, Lossos IS, Rosenwald A, Boldrick JC, <br /> miR-372. More work necessary understand                                        Sabet H, Tran T, Yu X, Powell JI, Yang L, Marti GE, Moore T, Hudson J Jr, Lu <br />                                                                                          L, Lewis DB, Tibshirani R, Sherlock G, Chan WC, Greiner TC, Weisenburger <br /> miR-372 regulates ATAD2 expression.                                                      DD, Armitage JO, Warnke R, Levy R, Wilson W, Grever MR, Byrd JC, Botstein <br />                                                                                          D, Brown PO, et al: Distinct types diffuse large B-cell lymphoma <br />                                                                                          identified gene expression profiling. Nature 2000, 403:503&#226;&#8364;&#8220;511. <br /> Conclusions                                                                        9.    Chen X, Cheung ST, So S, Fan ST, Barry C, Higgins J, Lai KM, Ji J, Dudoit S, <br /> In summary, provided basis concept                                 Ng IO, Van De Rijn M, Botstein D, Brown PO: Gene expression patterns in <br /> high expression ATAD2 HCC important                                      human liver cancers. Mol Biol Cell 2002, 13:1929&#226;&#8364;&#8220;1939. <br />                                                                                    10.   Ma XJ, Salunga R, Tuggle JT, Gaudet J, Enright E, McQuary P, Payette T, <br /> the acquisition aggressive <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> indicate                                Pistone M, Stecker K, Zhang BM, Zhou YX, Varnholt H, Smith B, Gadd M, <br /> poor prognosis HCC patients. Furthermore, func-                                  Chatfield E, Kessler J, Baer TM, Erlander MG, Sgroi DC: Gene expression <br /> tional studies ATAD2 report suggest potential                               profiles human breast cancer progression. Proc Natl Acad Sci U S A <br />                                                                                          2003, 100:5974&#226;&#8364;&#8220;5979. <br /> role ATAD2 affecting cell proliferation, invasion,                       11.   Ramaswamy S, Tamayo P, Rifkin R, Mukherjee S, Yeang CH, Angelo M, Ladd <br /> migration make ATAD2 attractive target future                                 C, Reich M, Latulippe E, Mesirov JP, Poggio T, Gerald W, Loda M, Lander ES, <br /> cancer therapeutics.                                                                     Golub TR: Multiclass cancer diagnosis using tumor gene expression <br />                                                                                          signatures. Proc Natl Acad Sci U S A 2001, 98:15149&#226;&#8364;&#8220;15154. <br />                                                                                    12.   van&#226;&#8364;&#8482;t Veer LJ, Dai H, van Vijver MJ, He YD, Hart AA, Mao M, Peterse HL, <br /> Additional file                                                                          van der Kooy K, Marton MJ, Witteveen AT, Schreiber GJ, Kerkhoven RM, <br />                                                                                          Roberts C, Linsley PS, Bernards R, Friend SH: Gene expression profiling <br />                                                                                          predicts clinical outcome breast cancer. Nature 2002, 415:530&#226;&#8364;&#8220;536. <br />  Additional file 1: Table S1. Genes differentially expressed HCC after <br />                                                                                    13.   Caron C, Lestrat C, Marsal S, Escoffier E, Curtet S, Virolle V, Barbry P, <br />  ATAD2 knockdown according Human Tumor Metastasis Real-time                         Debernardi A, Brambilla C, Brambilla E, Rousseaux S, Khochbin S: Functional <br />  PCR Array.                                                                              characterization ATAD2 new cancer/testis factor predictor <br />                                                                                          poor prognosis breast lung cancers. Oncogene 2010, <br />                                                                                          29:5171&#226;&#8364;&#8220;5181. <br /> Competing interests <br />                                                                                    14.   Huang Q, Lin B, Liu H, Ma X, Mo F, Yu W, Li L, Li H, Tian T, Wu D, Shen F, <br /> The authors declare competing interest. <br />                                                                                          Xing J, Chen ZN: RNA-Seq analyses generate comprehensive <br />                                                                                          transcriptomic landscape reveal complex transcript patterns in <br /> Authors&#226;&#8364;&#8482; contributions                                                                   hepatocellular carcinoma. PLoS One 2011, 6:e26168. <br /> HYL, HH, YYW, XJL, SGX XXM carried experimental work, YQY              15.   Zhou L, Rui JA, Ye DX, Wang SB, Chen SG, Qu Q: Edmondson-Steiner <br /> provided data analysis histopathological analysis, YFL provided tumor                grading increases predictive efficiency TNM staging long-term <br /> samples, clinical information, GW designed study participated                 survival patients hepatocellular carcinoma curative <br /> writing paper. All authors read approved manuscript.                         resection. World J Surg 2008, 32:1748&#226;&#8364;&#8220;1756. <br />                                                                                    16.   Zeng L, Zhou MM: Bromodomain: acetyl-lysine binding domain. FEBS <br /> Acknowledgements                                                                         Lett 2002, 513:124&#226;&#8364;&#8220;128. <br /> This work supported Liaoning Provincial Committee Education,         17.   Marmorstein R, Berger SL: Structure function bromodomains in <br /> Science Technology Research (grant 20060903 Gang Wu).                             chromatin-regulating complexes. Gene 2001, 272:1&#226;&#8364;&#8220;9. <br />                                                                                    18.   Kalashnikova EV, Revenko AS, Gemo AT, Andrews NP, Tepper CG, Zou JX, <br /> Author details                                                                           Cardiff RD, Borowsky AD, Chen HW: ANCCA/ATAD2 overexpression <br /> 1 <br />  Department General Surgery, First Affiliated Hospital China Medical           identifies breast cancer patients poor prognosis, acting drive <br /> University, Shenyang, Liaoning 110001, China. 2Department                             <br /> </body></html>