<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                           Redox Biology 2 (2014) 673&#226;&#8364;&#8220;678 <br />  <br />  <br />  <br />                                                                  Contents lists available ScienceDirect <br />  <br />  <br />                                                                             Redox Biology <br />                                                   journal homepage: www.elsevier.com/locate/redox <br />  <br />  <br />  <br /> Research Paper <br />  <br /> Zebra&#239;&#172;?sh heart development regulated glutaredoxin 2 <br /> dependent migration <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span>survival</span> neural crest cells <br /> Carsten Berndta , d , * , Gereon Poschmannc , Kai Stuhler <br />                                                     &#194;&#168;     c <br />                                                             , Arne Holmgrena , Lars Brautigam <br />                                                                                       &#194;&#168;       a,b,* <br /> a <br />   Department Medical Biochemistry Biophysics, Karolinska Institutet, Stockholm, Sweden <br /> b <br />   Science Life Laboratory, Division Translational Medicine Chemical Biology, Department Medical Biochemistry Biophysics, Karolinska Institutet, Stockholm, Sweden <br /> c <br />   Molecular Proteomics Laboratory, Heinrich-Heine-University, BMFZ, Du   &#194;&#168; sseldorf, Germany <br /> d <br />   Department Neurology, Medical Faculty, Heinrich-Heine-University Du    &#194;&#168; sseldorf, Life Science Center, Merowinger Platz 1, Du <br />                                                                                                                                   &#194;&#168; sseldorf, Germany <br />  <br />  <br />  <br /> a r t c l e          n f o                            b s t r c t <br />  <br /> Article history:                                          Glutaredoxin 2 vertebrate speci&#239;&#172;?c oxidoreductase thioredoxin family proteins modulating the <br /> Received 9 April 2014                                     intracellular thiol pool. Thereby, glutaredoxin 2 important speci&#239;&#172;?c redox signaling regulates em- <br /> Received revised form 28 April 2014                    bryonic development brain vasculature reversible oxidative posttranslational thiol modi&#239;&#172;?cations. <br /> Accepted 29 April 2014 <br />                                                           Here, glutaredoxin 2 required successful heart formation. Knock-down glutare- <br />                                                           doxin 2 zebra&#239;&#172;?sh embryos inhibits invasion cardiac neural crest cells primary heart &#239;&#172;?eld. <br /> Keywords: <br /> S-glutathionylation                                       This leads impaired heart looping subsequent obstructed blood &#239;&#172;&#8218;ow. Glutaredoxin 2 speci&#239;&#172;?city of <br /> Zebra&#239;&#172;?sh                                                  observed <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span>phenotype</span> con&#239;&#172;?rmed rescue experiments. Active site variants glutaredoxin 2 re- <br /> Glutaredoxin                                              vealed (de)-glutathionylation activity required proper heart formation. Our data suggest that <br /> Cardiac development                                       actin target glutaredoxin 2 regulated cardiac neural crest cell migration embryonic <br /> Migration                                                 heart development. In summary, work represents evidence general importance redox <br />                                                           signaling embryonic development highlights additionally importance glutaredoxin 2 during <br />                                                           embryogenesis. <br />                                                                                                                                         c 2014 Published Elsevier B.V. <br />                                                                                                            This open access article CC BY-NC-ND license <br />                                                                                                                 (http://creativecommons.org/licenses/by-nc-nd/3.0/). <br />  <br />  <br />  <br />  <br /> Introduction                                                                                       The redox state cysteine residues regulated mem- <br />                                                                                                bers thioredoxin family proteins including glutaredoxins <br />     About 2.5 billion years ago, photosynthesis evolved rais-                          (Grxs) [5]. Grxs subdivided dithiol Grxs harboring Cys- <br /> ing oxygen levels changed radically composition Earth&#226;&#8364;&#8482;s at-                         X-X-Cys active site, monothiol Grxs characterized a <br /> mosphere. Rapidly, organisms evolved strategies exploit new                            Cys-X-X-Ser motif. Whereas catalytic activity yet <br /> resource, defend unavoidable toxic                          characterized, established dithiol Grxs reduce pro- <br /> side products aerobic metabolism: reactive oxygen species (ROS).                            tein disul&#239;&#172;?des dithiol mechanism dependent both <br /> ROS long considered deleterious organisms,                             active site cysteines [6]. Additionally, dithiol Grxs able reduce <br /> damaging DNA, proteins, lipids, decades                               protein-GSH-mixed disul&#239;&#172;?des (de-glutathionylation) solely <br /> of extensive research established fundamental role ROS                            N-terminal cysteine required [6]. After reducing disul&#239;&#172;?des or <br /> second messengers [1]. Today, redox signaling known essen-                            glutathionylated cysteine residues, Grxs recycled expense <br /> tial cellular processes [2] pivotal proper embryonic                          GSH NADPH &#239;&#172;?nal electron donor [6]. <br /> development [3]. The major targets redox signaling events                                  Grxs proteins able reversible (de)- <br /> protein thiol groups, regulate protein activity re-                          glutathionylate substrates likely prime regulators <br /> versible oxidative modi&#239;&#172;?cations [4].                                                           redox signaling protein S-glutathionylation [7]. Grx2 char- <br />                                                                                                acterized vertebrate speci&#239;&#172;?c oxidoreductase conserved ad- <br />                                                                                                ditional cysteine residues forming intramolecular disul&#239;&#172;?de. Iso- <br />                                                                                                forms localized mitochondria (Grx2a) cytosol/nucleus <br />      Abbreviations: A, atrium; CNC, cardiac neural crest; CCV, common cardinal vein;           (Grx2b/c) [8]. Our group unraveled vertebrate Grx2 essen- <br /> GSH, glutathione; Grx, glutaredoxin; NC, neural crest; V, ventricle; zf, zebra&#239;&#172;?sh.             tial successful embryogenesis. Using zebra&#239;&#172;?sh model sys- <br />   * Corresponding authors at: Department Medical Biochemistry Biophysics,               tem, Grx2 regulates vertebrate neuronal survival and <br /> Karolinska Institutet, Stockholm, Sweden Department Neurology, Heinrich- <br />                      &#194;&#168; <br /> Heine-University Dusseldorf,     &#194;&#168; <br />                                Dusseldorf, Germany. , <br />                                                                                                axon growth thiol&#226;&#8364;&#8220;disul&#239;&#172;?de mechanism [9] vertebrate <br />      E-mail addresses: carsten.berndt@med.uni-duesseldorf.de (C. Berndt)                       angiogenesis S-glutathionylation histone-deacetylase <br />                              &#194;&#168; <br /> lars.braeutigam@ki.se (L. Brautigam). <br /> 2213-2317/$ - matter     c 2014 Published Elsevier B.V. This open access article CC BY-NC-ND license (http://creativecommons.org/licenses/by-nc-nd/ <br /> 3.0/). <br /> http://dx.doi.org/10.1016/j.redox.2014.04.012 <br />  674                                                   C. Berndt et al. / Redox Biology 2 (2014) 673&#226;&#8364;&#8220;678 <br />  <br />  <br /> sirtuin1 [10]. The zebra&#239;&#172;?sh powerful model offers multi-               incubated 10 mM NEM subsequently, actin protein sep- <br /> ple advantages vivo studies played fundamental               arated SDS polyacrylamide gels, stained coomassie brilliant <br /> role re&#239;&#172;?ning knowledge vertebrate embryonic development                blue, destained, alkylated iodoacetamide (55 mM aqueous <br /> including formation heart [11].                                       solution containing 50 mM ammonium bicarbonate) digested <br />     Here, report zfGrx2 required formation             chymotrypsin. Peptides extracted gel &#239;&#172;?nally <br /> zebra&#239;&#172;?sh heart. Loss zfGrx2 leads migratory defects cardiac         resuspended 0.1% tri&#239;&#172;&#8218;uoroacetic acid subjected liquid chro- <br /> neural crest (CNC) results heart looping defects causes            matography. Mass spectrometry parameters otherwise <br /> impaired heart functionality.                                                    stated described earlier [18]. <br />                                                                                      Brie&#239;&#172;&#8218;y, peptides separated 54 min LC-gradient using <br /> Material methods                                                             Ultimate 3000 Rapid Separation liquid chromatography system <br />                                                                                  (Dionex/Thermo Scienti&#239;&#172;?c) equipped Acclaim PepMapRSLC <br /> Zebra&#239;&#172;?sh husbandry                                                               100 C18 column (2 &#206;&#188; m C18 particle size, 100 A&#203;&#353; pore size, 75 &#206;&#188; m in- <br />                                                                                  ner diameter, 25 cm <span id='am-1' about='xsp:length' typeof='owl:Thing'>length</span>, Dionex/Thermo Scienti&#239;&#172;?c, Dreieich, Ger- <br />    Zebra&#239;&#172;?sh kept standard conditions, obtained mass mat-              many). Mass spectrometry carried Orbitrap Elite high <br /> ing raised E3 medium. For precise age-matching dif-             resolution instrument (Thermo Scienti&#239;&#172;?c, Bremen, Germany) oper- <br /> ferent embryo groups, early embryos staged counting                  ated positive mode equipped nanoelectrospray ioniza- <br /> number somites embryos older 24 h hours post fertilization             tion source. Survey scans carried Orbitrap analyzer <br /> (hpf) following protocol Kimmel [12]. The 12 somite stage cor-            mass range 350 1700 m/z resolution 60,000 <br /> responds &#226;&#710;&#188;15 hpf, 16 somite stage &#226;&#710;&#188;17 hpf 19 somite                (at 400 m/z). The target value automatic gain control was <br /> stage &#226;&#710;&#188;18.5 hpf. <span id='am-2' about='protege:TO' typeof='owl:Thing'>To</span> prevent pigmentation stages older 24              1,000,000 maximum &#239;&#172;?ll time 200 ms. The 15 &#226;&#8240;&#165; 2 + charged <br /> (hpf), Phenyl-2-thiourea (Sigma) added E3 medium. All ex-             peptide ions (minimal signal intensity 500) isolated, transferred <br /> periments reviewed granted North Stockholm Ethical               linear ion trap (LTQ) instrument fragmented us- <br /> Council.                                                                         ing collision induced dissociation (CID) runs fragmented <br />                                                                                  higher-energy collisional dissociation (HCD) HCD cell of <br />                                                                                  mass spectrometer transferred orbitrap analyzer. <br /> Morpholino mRNA injections <br />                                                                                  Already fragmented ions excluded fragmentation 30 s. <br />                                                                                      Raw &#239;&#172;?les processed protein peptide iden- <br />     The morpholino knocking zfGrx2 designed ob- <br />                                                                                  ti&#239;&#172;?cation MaxQuant software suite version 1.4.1.2 (Max <br /> tained Genetools (http://www.gene-tools.com) injected <br />                                                                                  Planck Institute Biochemistry, Planegg, Germany) otherwise <br /> as described [9,10]. Capped mRNA synthesized the <br />                                                                                  stated default parameters. Within software suite database <br /> mMessage/Machine Kit (Ambion) using constructs described else- <br />                                                                                  searches carried 888 Oryctolagus cuniculus se- <br /> where [9]. Morpholino mRNA injected 1 cell embryos <br />                                                                                  quences UniProtKB/Swiss-Prot database (downloaded on <br /> to ensure ubiquitous distribution. <br />                                                                                  2014-01-30) using following parameters: mass tolerance Fourier <br />                                                                                  transformed mass spectra (Orbitrap) &#239;&#172;?rst/second search: 20 ppm/ <br /> In situ hybridization acridine orange staining <br />                                                                                  4.5 ppm, mass tolerance fragment spectra (linear ion trap): 0.5 Da, <br />                                                                                  variable modi&#239;&#172;?cation: glutathione, carbamidomethyl (C), methionine <br />     The generation riboprobes situ hybridization carried <br />                                                                                  oxidation, acetylation protein N-termini, chymotryptic (FWYML) <br /> out according standard protocols [13]. As marker genes, used <br />                                                                                  digestion. Quanti&#239;&#172;?cation 2 + 3 + charged peptide ions (VCGlu- <br /> cmlc2 (demarks cardiac mesoderm [14]), foxD3 crestin <br />                                                                                  tathioneDNGSGL: m/z 535.1999, RCGlutathionePETLF: m/z 390.8355 <br /> (both demarking neural crest [15,16]). To detect cell death living <br />                                                                                  m/z 585.7495, VGDEAQSKRGIL: m/z 636.8494, ESAGIHETTY: m/z <br /> embryos, immersed 0.002% acridine orange (Sigma) so- <br />                                                                                  554.2513, QKEITAL: m/z 401.7371) carried inte- <br /> lution 45 min followed washing steps PBS and <br />                                                                                  gration extracted ion chromatogram (10 ppm mass window, 3 min <br /> imaging directly afterwards. <br />                                                                                  time window experiment) areas using Xcalibur 2.2 SP1.48 <br />                                                                                  Qual Browser (Thermo Scienti&#239;&#172;?c, Bremen, Germany). Summed signals <br /> Microscopy, image processing, statistics                                     detectable charge states used quantitative correlate for <br />                                                                                  relative peptide amounts. <br />      Fixed specimens mounted glycerol, life embryos were <br /> mounted low-melting agarose bright &#239;&#172;?eld pictures taken <br /> with Leica MZ16 microscope equipped Leica DFC300FX cam-                 Results <br /> era. Images processed Gimp (http://www.gimp.org) with- <br /> out obstructing original data. Movies captured Zeiss             Knock-down zfGrx2 impairs cardiac functionality <br /> Axiovert 40 equipped Zeiss Axiocam ICM1 angiograms <br /> were calculated ImageJ using previously published proto-                     Using antisense morpholinos targeting translation-initiation <br /> col [17]. To avoid unspeci&#239;&#172;?c effects, tricaine added an-             codon zfGrx2, able knock-down speci&#239;&#172;?cally zfGrx2 pro- <br /> giogram experiments. Data expressed mean &#194;&#177; SD. Statistical                tein synthesis zebra&#239;&#172;?sh embryos 75% described veri&#239;&#172;?ed <br /> signi&#239;&#172;?cance revealed using two-tailed Student&#226;&#8364;&#8482;s T-test. * p &lt; 0.05;          [9,10]. Examining embryos 48 hpf, observed reduced <br /> ** p &lt; 0.005; *** p &lt; 0.0005. <br />                                                                                  blood &#239;&#172;&#8218;ow dorsal aorta common cardinal vein <br />                                                                                  93% (n = 249) embryos lacking zfGrx2, obstructed <br /> Mass spectrometry mass spectrometric data analysis                           blood &#239;&#172;&#8218;ow observed age-matched controls (n = 150) (Fig. 1, <br />                                                                                  supplementary movies). Performing rescue experiments, i.e. injecting <br />    Rabbit muscle actin (Sigma) (0.2 &#206;&#188; g/&#206;&#188; l) incubated 5 mM             anti-zfGrx2 morpholino capped mRNA encoding <br /> GSSG 1 h. After removal GSSG (Zeba Spin Desalting column,                 zfGrx2, able reduce phenotypic penetrance 35% <br /> Thermo Scienti&#239;&#172;?c) actin solution divided parts. First             (n = 57), verifying speci&#239;&#172;?city observed phenotype. We addi- <br /> part incubated 10 mM NEM (glutathionylated sample), 2nd                 tionally observed reduced pumping ef&#239;&#172;?ciency hearts zfGrx2 <br /> part incubated 10 &#206;&#188; M zfGrx2/100 &#206;&#188; M GSH, 3rd                  knock-down embryos, likely caused reduced perfusion in <br /> was incubated 5 mM DTT. After 45 min 2nd 3rd parts                 main axial vessels (Fig. 1A, Movie S2). <br />                                                                   C. Berndt et al. / Redox Biology 2 (2014) 673&#226;&#8364;&#8220;678                                               675 <br />  <br />  <br />                                                                                             thinner &#239;&#172;?sh lacking Grx2 (98 &#194;&#177; 20 &#206;&#188; m, n = 18) compared con- <br />                                                                                             trols (147 &#194;&#177; 17 &#206;&#188; m, n = 10) middle stream almost <br />                                                                                             undetectable 83% embryos lacking zfGrx2 (n = 18). <br />  <br />  <br />                                                                                             Monothiol mechanism Grx2 important heart formation <br />  <br />                                                                                                 To gain insight molecular mechanism zfGrx2 <br />                                                                                             involved cardiac formation, generated variants zfGrx2 <br />                                                                                             enzymatically inactive (zfGrx2C37S) able <br />                                                                                             de-glutathionylate substrates monothiol mechanism (zf- <br />                                                                                             Grx2C40S). The mRNA variants injected mor- <br />                                                                                             pholino rescue ef&#239;&#172;?ciencies compared injection of <br />                                                                                             mRNA coding unmodi&#239;&#172;?ed zfGrx2 (Fig. 4A). The enzymatically inactive <br /> Fig. 1. Knock-down zfGrx2 impairs cardiac functionality perfusion. (A) An-           variant zfGrx2 able rescue cardiac looping phenotype <br /> giograms based supplementary movies demonstrate lower heart pumping capabil- <br />                                                                                             (rescue ef&#239;&#172;?cacy 22 &#194;&#177; 1.4% compared wildtype mRNA, n = 12). <br /> ity (left panel, Movie S2) subsequently reduced blood &#239;&#172;&#8218;ow pre-cardiac region <br /> of common cardinal vein (CCV) (left panel) main axial vessels (right panel,     However, zfGrx2C40S rescued heart looping defect ef&#239;&#172;?ciently <br /> Movie S1) loss zfGrx2 compared wildtype embryos 48 hpf (V: ventricle, A:         unmodi&#239;&#172;?ed zfGrx2 (rescue ef&#239;&#172;?cacy 106 &#194;&#177; 2.7% compared to <br /> atrium). (B) Quanti&#239;&#172;?cation A, right panel shows 150 examined wild-          wildtype mRNA, n = 19), indicating biochemical mechanism <br /> type embryos displayed reduced blood &#239;&#172;&#8218;ow, 93% embryos lacking zfGrx2             involves (de)-glutathionylation target proteins. It estab- <br /> (n = 249, black bar) showed reduced blood &#239;&#172;&#8218;ow. Phenotypic penetrance reduced <br /> to 35% (n = 57, gray bar) capped zfGrx2 mRNA injected parallel the <br />                                                                                             lished posttranslational modi&#239;&#172;?cations components cy- <br /> morpholino.                                                                                 toskeleton tightly controlled prerequisite cellular migra- <br />                                                                                             tion [23]. Actin crucial protein required cytoskeletal reorgani- <br />                                                                                             zation migration polymerization regulated reversible <br /> Knock-down zfGrx2 leads cardiac looping defects                                       S-glutathionylation [23,24]. Therefore, tested ability zfGrx2 <br />                                                                                             modulate glutathionylation state actin vitro. Recombi- <br />     Since reduced pumping ef&#239;&#172;?ciency likely cause                          nant muscle rabbit actin glutathionylated incu- <br /> obstructed blood &#239;&#172;&#8218;ow trunk zfGrx2 morpholino injected em-                         bated zfGrx2/GSH DTT analyzed mass spectrometry. <br /> bryos, examined heart morphology 48 hpf detail. In situ                      Two peptides harboring cysteines 12 (VCDNGSGL) 259 (RCPETLF) <br /> hybridization detecting cmlc2 transcripts, cardiomyocyte marker                           identi&#239;&#172;?ed targets S-glutathionylation (Figs. S1 S2 ). <br /> gene [14], revealed hearts embryos lacking zfGrx2                        To determine relative glutathionylated peptides, summed <br /> undergo proper heart looping contrast controls (Fig. 2). In con-                      signals detectable charge states non-modi&#239;&#172;?ed pep- <br /> trol embryos measured angle atrium ventricle                           tides (VGDEAQSKRGIL, ESAGIHETTY, QKEITAL) (Fig. S1) used as <br /> 54 &#194;&#177; 17&#226;&#8212;&#166; (n = 29), age-matched morpholino injected em-                              quantitative correlates (Fig. 4B C). Compared glutathionylated <br /> bryos signi&#239;&#172;?cantly stretched angle 112 &#194;&#177; 24&#226;&#8212;&#166; (n = 35).                        samples (VCDNGSGL: 100 &#194;&#177; 30%, RCPETLF: 100 &#194;&#177; 14%), treatment <br /> As reduced perfusion itself, defects heart looping spe-                        zfGrx2 (VCDNGSGL: 19 &#194;&#177; 11%, RCPETLF: 9 &#194;&#177; 0.7%) and <br /> ci&#239;&#172;?c lack zfGrx2, rescue experiments restored angle                       DTT (VCDNGSGL: 0.7 &#194;&#177; 0.7%, RCPETLF: 2 &#194;&#177; 1%) signi&#239;&#172;?cantly reduced <br /> between atrium ventricle 89 &#194;&#177; 21&#226;&#8212;&#166; (n = 27).                                          glutathionylated peptides indicating actin po- <br />                                                                                             tential target zfGrx2 mediated de-glutathionylation CNC <br />                                                                                             migration heart formation. <br /> Knock-down zfGrx2 impairs migration cardiac neural crest cells <br />  <br />     To unravel processes cardiac development                           Knock-down zfGrx2 leads cell death region neural crest <br /> affected loss zfGrx2, dissected early heart formation                       cells <br /> detail. Cardiomyocytes speci&#239;&#172;?ed lateral mesoderm de- <br /> tectable bilateral, cmlc2 positive stripes 16 somite stage [19].                      It reported NC cells fail migrate <br /> Determining size shape structures, did &#239;&#172;?nd                          undergo cell death [25], suggested experimen- <br /> signi&#239;&#172;?cant difference embryos lacking zfGrx2 controls                           tal setup reduced staining crestin positive cells zfGrx2 <br /> (n = 25 26 respectively, Fig. 3). However, 19 somite stage,                      lacking embryos (Fig. 3). By acridine orange staining, con&#239;&#172;?rmed <br /> we cardiac &#239;&#172;?elds embryos reduced levels zfGrx2                         loss zfGrx2 induces massive cell death time in <br /> to signi&#239;&#172;?cantly smaller (81 &#194;&#177; 11 &#206;&#188; m, n = 37) compared age-                           region migrating NC cells 83% embryos analyzed (n = 37) <br /> matched controls (100 &#194;&#177; 9 &#206;&#188; m, n = 28) (Fig. 3). Cardiac neural crest                       compared acridine orange positive cells controls <br /> (CNC) cells migrate primary heart &#239;&#172;?eld 16 19                         (n = 20) (Fig. 5). This observation supported detecting <br /> somite stage contribute proper cardiac looping zebra&#239;&#172;?sh                       apoptotic cells using TUNEL staining 24 hpf (Fig. 5) loss <br /> mammals [20&#226;&#8364;&#8220;22]. We followed movement neural crest (NC)                              foxD3 demarked NC cells morpholino injected 24 hpf embryos <br /> cells early embryonic development making use                             (Fig. 5). <br /> speci&#239;&#172;?c crestin expression [16]. At 12 somites, observed shorter <br /> distance lateral edges NC cells, indicating slower                         Discussion <br /> lateral movement cell population. We distance be- <br /> tween lateral NC &#239;&#172;?elds embryos lacking zfGrx2 78 &#194;&#177; 15 &#206;&#188; m                               Glutaredoxins oxidoreductases control cellular thiol <br /> (n = 69) compared 109 &#194;&#177; 10 &#206;&#188; m (n = 34) age-matched controls                          redox pool [6]. Our previous research unraveled Grx2 is <br /> (Fig. 3). Later development, 16 somite stage, width                        crucial mediator redox signaling embryonic develop- <br /> the caudal NC stream signi&#239;&#172;?cantly smaller embryos lacking                            ment [9,10]. Members thioredoxin family proteins including <br /> zfGrx2 (62 &#194;&#177; 12 &#206;&#188; m, n = 17) compared controls (98 &#194;&#177; 16 &#206;&#188; m,                             glutaredoxins known expressed mammalian heart <br /> n = 22) additionally middle NC stream markedly reduced                          involved cardiovascular physiology pathology [26]. How- <br /> in 78% Grx2 knock-down embryos (n = 22) (Fig. 3). This phenotype                         ever, report presents &#239;&#172;?rst evidence Grx2 required for <br /> remained 19 somite stage caudal NC stream                             development heart itself. Importantly, heart &#239;&#172;?rst <br />  676                                                             C. Berndt et al. / Redox Biology 2 (2014) 673&#226;&#8364;&#8220;678 <br />  <br />  <br />  <br />  <br /> Fig. 2. Knock-down zfGrx2 leads cardiac looping failure. In situ hybridization detecting transcripts cardiac-myocyte speci&#239;&#172;?c gene cmlc2 revealed embryos lacking <br /> zfGrx2 suffer heart looping defects 48 hpf. Whereas angle atrium ventricle wildtype embryos 54 &#194;&#177; 17&#226;&#8212;&#166; (n = 29), measured angle 112 &#194;&#177; 24&#226;&#8212;&#166; <br /> in zfGrx2 morpholino injected embryos (n = 35). Cardiac looping restored angle 89 &#194;&#177; 21&#226;&#8212;&#166; zfGrx2 capped mRNA injected simultaneously the <br /> morpholino (n = 27). A: Atrium, V: ventricle. Presented mean &#194;&#177; SD (*** p &lt; 0.0005). <br />  <br />  <br />  <br />  <br /> Fig. 3. Knock-down zfGrx2 impairs migration neural crest cells. Using situ hybridization detecting cmlc2 transcripts, width primary cardiac &#239;&#172;?eld measured <br /> in wildtype morpholino injected embryos 16, 19 somite stage. Impaired migration neural crest cells demonstrated anti-crestin situ hybridization at <br /> 12, 16, 19 somites. Quanti&#239;&#172;?cation shows width bilateral edges 12 somites width caudal stream later stages. Asterisks mark middle stream. <br /> Presented mean &#194;&#177; SD (** p &lt; 0.005, *** p &lt; 0.0005). <br />  <br />  <br />  <br /> organ develop function vertebrate embryonic develop-                         embryos lacking zfGrx2. Modulating levels zfGrx2 reduced the <br /> ment prerequisite successful formation                        size cardiac &#239;&#172;?elds presumptive arrival CNC cells <br /> a new organism. Knock-down zfGrx2 zebra&#239;&#172;?sh embryos leads                             [30], indicating reduced migration speed cell population if <br /> to reduced pumping ef&#239;&#172;?cacy hearts obstructed blood &#239;&#172;&#8218;ow                            zfGrx2 absent. Moreover, early development lateral mi- <br /> in main axial vessel disrupted heart looping. Looping                        gration NC cells inhibited course development <br /> the cardiac tube &#239;&#172;?rst manifestation embryonic lat-                      number NC cells severely reduced, prominently the <br /> erality poorly understood, essential process                     caudal middle streams. Interestingly, NC streams rec- <br /> during cardiac development [27]. In mammals lower vertebrates                          ognized origin CSC cells zebra&#239;&#172;?sh [31]. If NC cells are <br /> cardiac looping depends, like out&#239;&#172;&#8218;ow tract septation cushion for-                      disturbed migration terminal differentiation, un- <br /> mation, tightly orchestrated migration invasion CNC                          dergo cell death [25,32]. In line previous studies, have <br /> cells cardiac &#239;&#172;?eld [20,22,28]. Those migratory cells represent                    observed massive cell death region NC cells located. <br /> a specialized subgroup NC cells originate,                       Cell migration, including migration NC cells, based continuous <br /> other NC cells, neuroepithelium [29].In line re- <br /> ports, migration CNC cells severely impaired in <br />                                                                    C. Berndt et al. / Redox Biology 2 (2014) 673&#226;&#8364;&#8220;678                                                             677 <br />  <br />  <br />  <br />  <br /> Fig. 4. Heart development depends reversible S-glutathionylation. (A) Rescue ef&#239;&#172;?cacy heart looping defect using active site cysteine variants zfGrx2 (white bar: zfGrx2C37S, <br /> black bar: zfGrx2C40S) related ef&#239;&#172;?cacy rescue unmodi&#239;&#172;?ed zfGrx2 mRNA. (B,C) S-glutathionylation state recombinant rabbit actin determined mass spectrometry <br /> after incubation GSSG (white bars) following incubation zfGrx2 (black bars) DTT (gray bars). Two cysteines identi&#239;&#172;?ed S-glutathionylated (Cys12 (B) Cys259 <br /> (C)). Relative signal intensity calculated relation 3 peptides cysteines compared GSSG treated samples. <br />  <br />  <br />  <br />  <br /> Fig. 5. Knock-down glutaredoxin 2 induces cell death areas formation migration neural crest cells. Using acridine orange (AO) staining 19 somite stage <br /> embryos, detected cell death area speci&#239;&#172;?cation migration NC cells zfGrx2 knock-down embryos, wildtype controls showed acridine orange <br /> positive cells. Apoptotic cell death con&#239;&#172;?rmed TUNEL staining 24 hpf. Detecting neural crest cells using speci&#239;&#172;?c marker foxD3 shows staining rostral otic <br /> vesicle rostral brain embryos lacking zfGrx2 compared wildtypes, indicating loss NC cells. Asterisks mark otic vesicle. <br />  <br />  <br />  <br /> remodeling cytoskeleton [33&#226;&#8364;&#8220;35]. Posttranslational modi&#239;&#172;?ca-                           cardiac development independent CRMP2 <br /> tions, including reversible redox modi&#239;&#172;?cations actin pro-                       [41].Here, described zfGrx2 crucial successful heart <br /> teins cytoskeleton, involved cytoskeleton remodeling                       formation controlling invasion cardiac neural crest cells into <br /> cell migration [24,36&#226;&#8364;&#8220;39]. Several proteins cytoskeleton                         primary heart &#239;&#172;?eld (Fig. S4). Together recent work the <br /> been recently identi&#239;&#172;?ed potential interaction partners Grx2 [40].                      role glutaredoxin 2 brain [9] vascular development [10], this <br /> Using vitro approach combination mass spectrometry                             work represents evidence general importance redox <br /> analysis, present evidence observed phenotype                           signaling embryonic development highlights additionally the <br /> connected reversible S-glutathionylation actin zfGrx2. Nev-                         importance Grx2 embryogenesis. <br /> ertheless, zfGrx2 regulate additional proteins pathways <br /> which elucidated future.We previously identi&#239;&#172;?ed                           Acknowledgments <br /> collapsin-response-mediator protein 2 (CRMP2) substrate Grx2 <br /> [9]. CRMP2 involved semaphorin signaling im-                                We thank Lena Ringden  &#194;&#180; administrative assistance, Karolin- <br /> plicated cardiogenesis CNC migration [41,42]. However,                            ska Institutet zebra&#239;&#172;?sh core facility excellent service, Ulrich <br /> Grx2-CRMP2 signaling axis regulate cardiac looping                            &#194;&#168; <br />                                                                                              Ruther  (Heinrich Heine University) helpful discussions. Templates <br /> CNC migration setting (i) unable rescue                             foxD3 crestin riboprobes generously provided the <br /> phenotype overexpression CRMP2 (data shown), (ii)                               Yost laboratory (University Utah, USA) Bo Zhang (Peking Uni- <br /> previously Grx2 regulates CRMP2 activity reduction                         versity, P.R. China). This work supported Karolinska Institutet <br /> a disul&#239;&#172;?de, (iii) unable detect CRMP2 tran-                        (postdoctoral fellowships CB, PhD fellowship (KID) Dnr. 2379/07- <br /> scripts heart [43] (Fig. S3 ). Moreover, semaphorin signaling                         225 LB), German Research Foundation (DFG priority program 1710, <br />  678                                                                C. Berndt et al. / Redox Biology 2 (2014) 673&#226;&#8364;&#8220;678 <br />  <br />  <br /> BE 3259/5-1 CB), Swedish Society Medical Research (fellow-                                222 (2001) 552&#226;&#8364;&#8220;563. <br /> ship L.B.) K.A. Wallenberg Foundation (KAW 2006.0192 AH),                             [20] Y.-X. Li, et al. Cardiac neural crest zebra&#239;&#172;?sh embryos contributes myocar- <br />                                                                                                      dial cell lineage early heart function, Developmental Dynamics: An Of&#239;&#172;?cial <br /> the Swedish Cancer Society (961 AH), Swedish Research                                     Publication American Association Anatomists 226 (2003) 540&#226;&#8364;&#8220;550, <br /> Council (K2012-68X-03529-41-3 AH).                                                                12619138. <br />                                                                                                 [21] T.L. Creazzo, R.E. Godt, L. Leatherbury, S.J. Conway, M.L. Kirby, Role car- <br />                                                                                                      diac neural crest cells cardiovascular development, Annual Review Phys- <br /> Appendix A. Supplemental Materials                                                                   iology 60 (1998) 267&#226;&#8364;&#8220;286. http://dx.doi.org/10.1146/annurev.physiol.60.1.267, <br />                                                                                                      9558464. <br />     Supplementary material associated article                                  [22] T.M. Yelbuz, et al. Shortened out&#239;&#172;&#8218;ow tract leads altered cardiac looping af- <br /> found, online version, http://dx.doi.org/10.1016/                                          ter neural crest ablation, Circulation 106 (2002) 504&#226;&#8364;&#8220;510. http://dx.doi.org/10. <br />                                                                                                      1161/01.CIR.0000023044.44974.8A, 12135953. <br /> j.redox.2014.04.012                                                                             [23] J. Sakai, et al. Reactive oxygen species-induced actin glutathionylation controls <br />                                                                                                      actin dynamics neutrophils, Immunity 37 (2012) 1037&#226;&#8364;&#8220;1049. http://dx.doi. <br />                                                                                                      org/10.1016/j.immuni.2012.08.017, 23159440. <br />                                                                                                 [24] J. Wang, et al. Reversible glutathionylation regulates actin polymerization in <br /> References                                                                                           A431 cells, Journal Biological Chemistry 276 (2001) 47763&#226;&#8364;&#8220;47766, 11684673. <br />                                                                                                 [25] A. Barrallo-Gimeno, J. Holzschuh, W. Driever, E.W. Knapik, Neural crest survival <br />                                                                                                      differentiation zebra&#239;&#172;?sh depends Mont Blanc/tfap2a gene function, <br />   [1] C.C. Winterbourn, Reconciling chemistry biology reactive oxygen                     Development (Cambridge, England) 131 (2004) 1463&#226;&#8364;&#8220;1477, 14985255. <br />       species. Nature Chemical Biology 4 (2008) 278&#226;&#8364;&#8220;286.                                        [26] C. Berndt, C.H. Lillig, A. Holmgren, Thiol-based mechanisms thioredoxin <br />   [2] R. Brigelius-Flohe, &#194;&#180; L. Flohe, <br />                                     &#194;&#180; Basic principles emerging concepts redox            glutaredoxin systems: implications diseases cardiovascular sys- <br />       control transcription factors, Antioxidants Redox Signaling 15 (2011)                   tem, American Journal Physiology: Heart Circulatory Physiology 292 <br />       2335&#226;&#8364;&#8220;2381. http://dx.doi.org/10.1089/ars.2010.3534, 21194351.                                  (2007) H1227&#226;&#8364;&#8220;H1236, 17172268. <br />   [3] C. Ufer, C.C. Wang, A. Borchert, D. Heydeck, H. Kuhn, Redox control mam-               [27] L. Miquerol, R.G. Kelly, Organogenesis vertebrate heart, Wiley Interdisci- <br />       malian embryo development, Antioxidants Redox Signaling 13 (2010) 833&#226;&#8364;&#8220;                     plinary Reviews: Developmental Biology 2 (2013) 17&#226;&#8364;&#8220;29. http://dx.doi.org/10. <br />       875. http://dx.doi.org/10.1089/ars.2009.3044, 20367257.                                        1002/wdev.68, 23799628. <br />   [4] C.C. Winterbourn, M.B. Hampton, Thiol chemistry speci&#239;&#172;?city redox sig-              [28] K.L. Waldo, et al. Cardiac neural crest necessary normal addition the <br />       naling, Free Radical Biology Medicine 45 (2008) 549&#226;&#8364;&#8220;561. http://dx.doi.org/                myocardium arterial pole secondary heart &#239;&#172;?eld, Developmen- <br />       10.1016/j.freeradbiomed.2008.05.004, 18544350.                                                 tal Biology 281 (2005) 66&#226;&#8364;&#8220;77. http://dx.doi.org/10.1016/j.ydbio.2005.02.011, <br />   [5] A. Holmgren, et al. Thiol redox control thioredoxin glutaredoxin systems,              15848389. <br />       Biochemical Society Transactions 33 (2005) 1375&#226;&#8364;&#8220;1377. http://dx.doi.org/10.               [29] P. Snider, M. Olaopa, A.B. Firulli, S.J. Conway, Cardiovascular development and <br />       1042/BST20051375, 16246122.                                                                    colonizing cardiac neural crest lineage, The Scienti&#239;&#172;?c World Journal 7 (2007) <br />   [6] C.H. Lillig, C. Berndt, Glutaredoxins thiol/disul&#239;&#172;?de exchange, Antioxidants                 1090&#226;&#8364;&#8220;1113. http://dx.doi.org/10.1100/tsw.2007.189, 17619792. <br />       Redox Signaling 18(13) (2013) 1654&#226;&#8364;&#8220;1665. http://dx.doi.org/10.1089/ars.               [30] M. Sato, H.-J. Tsai, H.J. Yost, Semaphorin3D regulates invasion cardiac neural <br />       2012.5007, 23231445.                                                                           crest cells primary heart &#239;&#172;?eld, Developmental Biology 298 (2006) 12&#226;&#8364;&#8220; <br />   [7] J.J. Mieyal, B. Chock, Posttranslational modi&#239;&#172;?cation cysteine redox signaling            21. http://dx.doi.org/10.1016/j.ydbio.2006.05.033, 16860789. <br />       oxidative stress: focus S-glutathionylation, Antioxidants Redox Sig-           [31] M. Sato, H.J. Yost, Cardiac neural crest contributes cardiomyogenesis ze- <br />       naling 16 (2012) 471&#226;&#8364;&#8220;475. http://dx.doi.org/10.1089/ars.2011.4454, 22136616.                   bra&#239;&#172;?sh, Developmental Biology 257 (2003) 127&#226;&#8364;&#8220;139. http://dx.doi.org/10.1016/ <br />   [8] C.H. Lillig, C. Berndt, A. Holmgren, Glutaredoxin systems, Biochimica et Bio-                  S0012- 1606(03)00037- X, 12710962. <br />       physica Acta 1780 (2008) 1304&#226;&#8364;&#8220;1317. http://dx.doi.org/10.1016/j.bbagen.2008.              [32] A. Hinoue, et al. Disruption actin cytoskeleton anchorage-dependent cell <br />       06.003, 18621099.                                                                              spreading induces apoptotic death mouse neural crest cells cultured vitro, <br />             &#194;&#168; <br />   [9] L. Brautigam,    et al. Vertebrate-speci&#239;&#172;?c glutaredoxin essential brain de-             Anatomical Record Part A: Discoveries Molecular, Cellular, Evolutionary <br />       velopment, Proceedings National Academy Sciences United                       Biology 282 (2005) 130&#226;&#8364;&#8220;137, 15627983. <br />       States America 108 (2011) 20532&#226;&#8364;&#8220;20537. http://dx.doi.org/10.1073/pnas.                 [33] T.D. Pollard, G.G. Borisy, Cellular motility driven assembly disassem- <br />       1110085108, 22139372.                                                                          bly actin &#239;&#172;?laments, Cell 112 (2003) 453&#226;&#8364;&#8220;465. http://dx.doi.org/10.1016/ <br />             &#194;&#168; <br />  [10] L. Brautigam,    et al. Glutaredoxin regulates vascular development reversible              S0092- 8674(03)00120- X, 12600310. <br />       glutathionylation sirtuin 1, Proceedings National Academy Sciences           [34] J.R. Terman, A. Kashina, Post-translational modi&#239;&#172;?cation regulation actin, <br />       United States America 110 (2013) 20057&#226;&#8364;&#8220;20062. http://dx.doi.org/10.                  Current Opinion Cell Biology 25 (2013) 30&#226;&#8364;&#8220;38. http://dx.doi.org/10.1016/j. <br />       1073/pnas.1313753110, 24277839.                                                                ceb.2012.10.009, 23195437. <br />  [11] D. Staudt, D. Stainier, Uncovering molecular cellular mechanisms                  [35] T.P. Stossel, On crawling animal cells, Science (New York, N.Y.) 260 (1993) <br />       heart development using zebra&#239;&#172;?sh, Annual Review Genetics                             1086&#226;&#8364;&#8220;1094. http://dx.doi.org/10.1126/science.8493552, 8493552. <br />       46 (2012) 397&#226;&#8364;&#8220;418. http://dx.doi.org/10.1146/annurev- genet- 110711- 155646,              [36] X. Hou, et al. Coactosin accelerates cell dynamism promoting actin polymer- <br />       22974299.                                          <br /> </body></html>