<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Liu et al. BMC Evolutionary Biology 2014, 14:241 <br /> http://www.biomedcentral.com/1471-2148/14/241 <br />  <br />  <br />  <br />  <br />  RESEARCH ARTICLE                                                                                                                              Open Access <br />  <br /> Integrative analysis young genes, positively <br /> selected genes lncRNAs development <br /> of Drosophila melanogaster <br /> He-Qun Liu1,2&#226;&#8364;&#160;, Yan Li1,2&#226;&#8364;&#160;, David M Irwin1,3,4, Ya-Ping Zhang1,2* Dong-Dong Wu1,2* <br />  <br />  <br />   <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />   Background: Young genes genes positive selection commonly contribute adaptive phenotypic <br />   evolution. Early developmental stages important establishing phenotypes, helpful for <br />   studying evolutionary patterns rapidly evolving genes. <br />   Results: Here, performed weighted gene co-expression network analysis identify modules co-expressed <br />   genes different stages Drosophila melanogaster development. We young genes, including duplicated, <br />   orphan, young lncRNA genes, significantly enriched modules associated specific developmental <br />   stages. In addition, genes undergoing rapid amino acid sequence evolution driven positive selection showed a <br />   similar proportion essentiality genes, enrichment modules specific developmental stages. <br />   Conclusions: Our integrative analysis revealed important roles origin new genes rapid amino acid <br />   sequence evolution development account specific <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>phenotype</span> evolution Drosophila melanogaster. <br />  <br />  <br /> Background                                                                           evolution protein coding sequences involved in <br /> Evolutionary biologists long sought understand                               phenotypic evolution known underes- <br /> the molecular genetic basis phenotypic evolution.                                 timated. For example, coding sequences genes <br /> Britten Davidson proposed regulatory muta-                                  involved development human brain have <br /> tions play crucial role phenotypic evolution                               shown evolved positive selection, <br /> their observations substantial proportion repeti-                          account rapid evolution human <br /> tive sequences regulate expression genomes [1]. A                                 brain [7-13]. Adaptive changes amino acid sequence <br /> pioneering study King Wilson de-                               MC1R account pigment adaptation including varia- <br /> gree divergence protein sequences account                               tions skin hair color species [14]. Genome- <br /> for phenotypic differences human chim-                               wide surveys polymorphism humans estimated <br /> panzee, proposed evolution anatomy                               &gt;10% amino acid substitutions sequences <br /> influenced greater extent changes gene regula-                            occurred humans chimpanzee have <br /> tion changes protein sequence [2]. Recently,                              adaptive [15,16]. During short-term evolution within <br /> Carroll forward proposal evolution                           species, mutations cause morphological variation <br /> animal morphology mainly attributable mutations                             protein-coding regions [17]. In con- <br /> the cis-regulatory regions genes, instead mutations                         trast, long-term evolution species most <br /> their protein-coding regions [3-6]. With advent                               mutations cause morphological differences been <br /> large-scale genomic data, role positive selection                          cis-regulatory regions. It proposed that <br />                                                                                      difference abundance coding regulatory mu- <br /> * Correspondence: zhangyp@mail.kiz.ac.cn; wudongdong@mail.kiz.ac.cn                  tation morphological characters coding muta- <br /> &#226;&#8364;&#160; <br />  Equal contributors                                                                  tions species able spread through <br /> 1 <br />  State Key Laboratory Genetic Resources Evolution, Kunming Institute <br /> of Zoology, Chinese Academy Sciences, Kunming, China                              populations pleiotropic deleterious effects [17]. <br /> 2 <br />  Kunming College Life Science, University Chinese Academy                      New genes provide crucial material evolution- <br /> Sciences, Kunming, Yunnan, China                                                     ary innovations. Accumulating evidence supports the <br /> Full list author information available end article <br />  <br />                                         &#194;&#169; 2014 Liu et al.; licensee BioMed Central Ltd. This Open Access article distributed terms Creative <br />                                         Commons Attribution License (http://creativecommons.org/licenses/by/4.0), permits unrestricted use, distribution, and <br />                                         reproduction medium, provided original work properly credited. The Creative Commons Public Domain <br />                                         Dedication waiver (http://creativecommons.org/publicdomain/zero/1.0/) applies data available article, <br />                                         unless stated. <br />  Liu et al. BMC Evolutionary Biology 2014, 14:241                                                                Page 2 10 <br /> http://www.biomedcentral.com/1471-2148/14/241 <br />  <br />  <br />  <br />  <br /> remarkable phenotypic effects new genes develop-        constitutive RNAi). This database contained 20,262 trans- <br /> ment [18-20]. For example, pioneering study Chen         genic RNAi lines predicted target 11,619 the <br /> et al (2010) new genes rapidly evolve      14,139 protein-coding genes (82.2%) release 5.7 the <br /> gain essential functions Drosophila, knockdown       Drosophila genome. Lethality described database <br /> experiments of new genes RNA interfer-       included categories: completely lethal (no adults of <br /> ence (RNAi) leads lethality various development         desired genotype, 1803 lines); lethality (adult &#226;&#8364;&#339;es- <br /> stages. It new genes        capers&#226;&#8364;? obvious phenotype, 506 lines); lines <br /> have important functions process organogenesis      showed phenotype accompanying le- <br /> in Drosophila [21]. A functional association new      thality (1,230 lines, 1,069 genes). Of 15,379 Drosophila <br /> genes newly evolved brain structure observed       reference genes present modules, 10,753 in- <br /> humans [22,23] Drosophila [24]. In human gen-         cluded 18,663 lines knock-down data the <br /> ome new genes appear recruited           Bristle Screen database, 1,237 genes (in 1,655 lines) <br /> function development brain [18,22,23,25].       showing completely lethality (~8.868%). The analysis was <br />   Evolution phenotypes mainly determined changes     expanded include genes (i.e., just ref- <br /> that occur developmental processes. Examination         erence Drosophila genes) represented modules. A <br /> evolutionary changes occurring genes expressed      total 11,632 genes, represented 20,262 RNAi lines, <br /> development helpful understanding roles     Bristle Screen database, represented our <br /> of genetic innovation phenotypic evolution. Here, based    modules, 1,339 genes (<span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> 1,803 lines) show- <br /> on analysis gene co-expression networks different    ing complete lethality (~8.898%). From analysis, three <br /> developmental stages, important roles new     modules, M17, M6 M28, identified being <br /> genes, rapid amino acid sequence evolution genes,      significantly enriched lethal genes (P = 1.84E-22, <br /> in development responsible specific as-     P = 3.61E-62 P = 1.03E-22, respectively, &#207;&#8225;2 test, <br /> pects phenotypic evolution Drosophila melanogaster.     FDR correction) (Figure 1A). Expression genes in <br />                                                               module M17 significantly associated white <br /> Results                                                       prepupae (WPP) stages (P = 6e &#226;&#710;&#8217; 04, Figure 1B, Additional <br /> Analysis gene co-expression networks derived          file 4), genes module M6 significantly correlated <br /> transcriptomes different developmental stages           0-2 hr embryo stage (P = 4e &#226;&#710;&#8217; 04, Figure 1C, <br /> drosophila melanogaster                                       Additional file 5) genes module M28 signifi- <br /> Transcriptomes early developmental stages di-        cantly associated 10-12 hr embryo development <br /> verse adult tissues Drosophila melanogaster studied     stage (P = 0.01, Figure 1D, Additional file 6), revealing es- <br /> [26] used identifying new transcripts calcu-     sential roles highly expressed genes de- <br /> lating expression values genes (see Materials     velopmental stages. The 0-2 hr embryo stage, i.e. the <br /> Methods, Additional file 1). Genes involved       specific stage module M6, maternal zygotic <br /> biological process tend expression levels    transition period Drosophila [31]. Consistent this <br /> highly correlated samples. Identification groups    period development, gene ontology analysis revealed <br /> of co-regulated genes, &#226;&#8364;&#339;modules&#226;&#8364;?, inform-        genes module M6 enriched biological cat- <br /> ative identifying specific features developmental      egories associated chromatin organization (P = 2.52e- <br /> stages/tissues/cells. We implemented weighted gene          37, FDR correction), transcription (P = 7.40E-32, after <br /> co-expression network analysis (WGCNA) [27] identify       FDR correction), cell cycle (P = 1.07E-29, FDR cor- <br /> modules co-expressed genes different stages      rection), development (P = 2.88e-04, FDR correction) <br /> development Drosophila melanogaster. This approach         morphogenesis (P = 8.79e-04, FDR correction) <br /> has successfully employed deduce regulatory net-      (Additional file 7). These data suggest essential func- <br /> works developing human brain [28,29]. Here, total    tion genes maternal zygotic transition <br /> of 46 coexpression modules (labeled numerically, e.g.,        period. <br /> M29, color, e.g., black, Additional file 2) were <br /> obtained Drosophila development. Many mod-         Developmental stage-specific enrichment new genes <br /> ules developmental stage-specific (Additional file 3),   revealed gene co-expression networks Drosophila <br /> for example, M2 correlated, high significance,   melanogaster <br /> the 2-4 hr embryonic developmental state (P = 5e-42),     We identified total 221 new genes including newly <br /> M39 strongly correlated, high significance,      duplicated young orphan genes (detailed Material <br /> adult females 30 days eclosion (P = 2e-10).             Methods), specific Drosophila melanogaster. We <br />   The Bristle Screen online database [30] used         identified modules, developmental <br /> search essential genes (here, gene deemed        stages, enriched new genes Drosophila <br /> essential <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span>survival</span> completely lethal       melanogaster. Significant enrichment new genes was <br />  Liu et al. BMC Evolutionary Biology 2014, 14:241                                                                                          Page 3 10 <br /> http://www.biomedcentral.com/1471-2148/14/241 <br />  <br />  <br />  <br />  <br />      A                                                  B <br />                                                         0.08 <br />        yellowgreen <br />                 yellow <br />                                                         0.06 <br />                  white <br />                  violet <br />                                                 * <br />            turquoise <br />                     tan <br />                                   *                     0.04 <br />  <br />             steelblue <br />                                                         0.02 <br />             skyblue3 <br />               skyblue <br />               sienna3                                       0 <br />                salmon <br />       saddlebrown <br />             royalblue                                   C <br />                    red <br />                 purple <br />                                       *                 0.12 <br />  <br />                 plum1                                    0.1 <br />                   pink                                  0.08 <br />      paleturquoise <br />        orangered4                                       0.06 <br />                orange <br />                                                         0.04 <br />       midnightblue <br />   mediumpurple3                                         0.02 <br />             magenta <br />                                                             0 <br />          lightyellow <br />    lightsteelblue1 <br />           lightgreen                                    D <br />          lightcyan1                                     0.08 <br />             lightcyan <br />                  ivory <br />                grey60                                   0.06 <br />        greenyellow <br />                 green                                   0.04 <br />          floralwhite <br />      darkturquoise                                      0.02 <br />               darkred <br />       darkorange2 <br />         darkorange                                          0 <br />  <br />  <br />  <br />  <br />                                                                 Male,4days,AccessoryGlands <br />  <br />  <br />  <br />  <br />                                                                     20days,DigestiveSystem <br />                                                                       4days,DigestiveSystem <br />                                                                        Female,4days,Ovaries <br />  <br />  <br />  <br />                                                                        Female,4days,Ovaries <br />  <br />  <br />  <br />  <br />                                                                        1day,DigestiveSystem <br />                                                                         L3,lightbluegutPS3~6 <br />  <br />  <br />  <br />                                                                           L3,DigestiveSystem <br />  <br />  <br />  <br />  <br />                                                                                 WPP,FatBody <br />                                                                          WPP,SalivaryGlands <br />                                                                            L3,SalivaryGlands <br />  <br />  <br />  <br />  <br />                                                                                  Female,1day <br />  <br />  <br />  <br />  <br />                                                                         Female,4days,Heads <br />  <br />                                                                        Female,20days,Heads <br />                                                                              L3,12hrpost~molt <br />  <br />  <br />  <br />  <br />                                                                         Female,4days,Heads <br />  <br />                                                                        Female,20days,Heads <br />  <br />  <br />  <br />                                                                            Male,4days,Testes <br />                                                                         L3,darkbluegutPS1~2 <br />  <br />  <br />  <br />  <br />                                                                                    L3,Fatbody <br />  <br />  <br />  <br />  <br />                                                                                     Male,1day <br />  <br />  <br />                                                                          Female,1day,Heads <br />  <br />  <br />  <br />                                                                          Female,1day,Heads <br />  <br />  <br />  <br />  <br />                                                                           Male,20days,Heads <br />                                                                              L3,ImaginalDiscs <br />  <br />  <br />  <br />  <br />                                                                               WPP+2daysFat <br />  <br />  <br />  <br />  <br />                                                                              Male,1day,Heads <br />  <br />                                                                            Male,4days,Heads <br />                                                                             L3,cleargutPS7~9 <br />  <br />  <br />  <br />  <br />                                                                              WPP+2daysCNS <br />  <br />  <br />  <br />  <br />                                                                                 Female,5days <br />                                                                                Female,30days <br />  <br />                                                                                    Male,5days <br />                                                                                  Male,30days <br />  <br />  <br />  <br />  <br />                                                                               20days,Carcass <br />                                                                                 1day,Carcass <br />  <br />                                                                                4days,Carcass <br />                                                                                    L3,Carcass <br />                                                                                        8~10hr <br />                                                                                       10~12hr <br />                                                                                       12~14hr <br />                                                                                       14~16hr <br />                                                                                       16~18hr <br />                                                                                       18~20hr <br />                                                                                       20~22hr <br />                                                                                       22~24hr <br />  <br />  <br />  <br />  <br />                                                                                       L3,CNS <br />                                                                                         0~2hr <br />                                                                                         2~4hr <br />                                                                                         4~6hr <br />  <br />  <br />  <br />  <br />                                                                                          12hr <br />                                                                                         24hrs <br />                                                                                         2days <br />                                                                                         3days <br />                                                                                         4days <br />                                                                                         6~8hr <br />  <br />  <br />  <br />  <br />                                                                                          WPP <br />                                                                                            L1 <br />                                                                                            L2 <br />  <br />  <br />  <br />  <br />   darkolivegreen <br />      darkmagenta <br />              darkgrey <br />          darkgreen <br />                  cyan <br />               brown4 <br />                 brown                                            Stage         Stage    Stage   Stage   Stage   Stage    Stage   Stage of <br />                   blue                                           embryos          larvae                           adult      adult <br />                  black                                                                        white   pup     adult   adult    mated      post <br />                                                                                               pup     -ae       virgin   post       eclosion , <br />                           0   1   2   3     4       5                                         -aria     eclos   post     eclosion   mixed <br />                                                                                                       WPP     -ion    eclos               male <br />                                                                                                                       -ion                and <br />                                                                                                                                           female <br />  Figure 1 Expression profile completely lethal genes module. (A) Enrichment completely lethal genes gene modules. Stars <br />  identify modules significant enrichment lethal genes. (B-D) Histogram expression genes modules M17 (B), M6 (C) M28 <br />  (D) heatmaps expression genes modules seeing Additional files 4 5. Expression genes developmental <br />  stage calculated expression stage divided sum stages. Mean expression standard deviation (SD) genes in <br />  stage shown. <br />  <br />  <br />  <br />  <br /> found 6 modules, M41, M24, M37, M13, M36                             stage L3 larvae (P = 4e-27) 20-22 hr embryo stages <br /> M23 (P = 7.34E-14, P = 1.56E-11, P = 3.52E-09, P = 5.56E-                   (P = 1e-05), respectively (Additional file 3). Gene expres- <br /> 08, P = 8.18E-07, P = 2.39E-04, respectively, &#207;&#8225;2                 sion analysis new genes enriched six <br /> test, FDR correction) (Additional file 8),                modules showed consistent specific high levels ex- <br /> modules significantly correlated WPP + 2 days                      pression associated developmental stages, sup- <br /> Fat (P = 9e-37), stage L1 larvae (P = 1e-06, Figure 2B,                     porting potential roles new genes these <br /> Additional file 9), dark blue gut PS(1-2) stage L3 larvae                tissues/developmental stages (Additional file 11). PCA <br /> (P = 3e-15, Figure 2C, Additional file 10), 4 day post-                     analysis (Principal Component Analysis) an <br /> eclosion testes adult males (P = 3e-15), 12 hr post-molt                 inseparable relationship new genes other <br />  Liu et al. BMC Evolutionary Biology 2014, 14:241                                                                                                 Page 4 10 <br /> http://www.biomedcentral.com/1471-2148/14/241 <br />  <br />  <br />  <br />  <br />         A                                                         B <br />        yellowgreen                 #                              0.35 <br />                 yellow <br />                                                                    0.3 <br />                  white <br />                  violet                              #            0.25 <br />            turquoise                                               0.2 <br />                     tan <br />                                                                   0.15 <br />             steelblue <br />             skyblue3                                               0.1 <br />               skyblue                                             0.05 <br />               sienna3 <br />                                                                       0 <br />                salmon <br />       saddlebrown                  # <br />             royalblue              * <br />                    red <br />                 purple                                            C <br />                 plum1                                             0.6 <br />                   pink <br />      paleturquoise                                                0.5 <br />        orangered4                                                 0.4 <br />                orange <br />       midnightblue <br />                                *                                  0.3 <br />   mediumpurple3 <br />                                                                   0.2 <br />             magenta <br />          lightyellow <br />                           *                                       0.1 <br />    lightsteelblue1 <br />                                                                    0 <br />           lightgreen                   # <br />  <br />                                                                                              L3,Fatbody <br />  <br />  <br />  <br />                                                                                           WPP,FatBody <br />  <br />  <br />  <br />  <br />                                                                                            Female,1day <br />  <br />  <br />                                                                                               Male,1day <br />                                                                                                    WPP <br />  <br />  <br />  <br />                                                                                         WPP+2daysFat <br />                                                                                                   0~2hr <br />                                                                                                   2~4hr <br />                                                                                                   4~6hr <br />                                                                                                   6~8hr <br />                                                                                                  8~10hr <br />  <br />  <br />  <br />  <br />                                                                                        L3,12hrpost~molt <br />  <br />  <br />  <br />  <br />                                                                                                    12hr <br />                                                                                                 L3,CNS <br />  <br />  <br />  <br />  <br />                                                                                                   2days <br />                                                                                                   3days <br />                                                                                                   4days <br />                                                                                                 10~12hr <br />                                                                                                 12~14hr <br />                                                                                                 14~16hr <br />                                                                                                 16~18hr <br />                                                                                                 18~20hr <br />                                                                                                 20~22hr <br />                                                                                                 22~24hr <br />                                                                                                      L1 <br />                                                                                                      L2 <br />  <br />  <br />  <br />  <br />                                                                                              L3,Carcass <br />  <br />  <br />  <br />  <br />                                                                                    WPP,SalivaryGlands <br />                                                                                        WPP+2daysCNS <br />  <br />  <br />                                                                                                   24hrs <br />  <br />  <br />  <br />  <br />                                                                                              Male,5days <br />                                                                                             Male,30days <br />  <br />  <br />  <br />  <br />                                                                                      Male,4days,Testes <br />  <br />  <br />  <br />  <br />                                                                                         20days,Carcass <br />                                                                                     L3,DigestiveSystem <br />  <br />  <br />  <br />  <br />                                                                                           Female,5days <br />                                                                                        L3,ImaginalDiscs <br />                                                                                      L3,SalivaryGlands <br />  <br />  <br />  <br />  <br />                                                                                          Female,30days <br />  <br />  <br />  <br />                                                                                    Female,1day,Heads <br />                                                                                   Female,4days,Heads <br />                                                                                  Female,4days,Ovaries <br />  <br />                                                                                    Female,1day,Heads <br />                                                                                   Female,4days,Heads <br />                                                                                  Female,4days,Ovaries <br />  <br />                                                                                        Male,1day,Heads <br />  <br />                                                                                      Male,4days,Heads <br />  <br />  <br />                                                                                           1day,Carcass <br />                                                                                  1day,DigestiveSystem <br />                                                                                          4days,Carcass <br />                                                                                 4days,DigestiveSystem <br />  <br />                                                                               20days,DigestiveSystem <br />                                                                                   L3,lightbluegutPS3~6 <br />  <br />  <br />  <br />  <br />                                                                                  Female,20days,Heads <br />  <br />  <br />  <br />                                                                                  Female,20days,Heads <br />                                                                                       L3,cleargutPS7~9 <br />  <br />  <br />  <br />  <br />                                                                                     Male,20days,Heads <br />                                                                                   L3,darkbluegutPS1~2 <br />  <br />  <br />  <br />  <br />                                                                           Male,4days,AccessoryGlands <br />          lightcyan1            * <br />             lightcyan <br />                  ivory <br />                grey60 <br />        greenyellow <br />                 green <br />                            * <br />                                                                            Stage       Stage   Stage   Stage   Stage   Stage    Stage   Stage of <br />          floralwhite <br />                                                                            embryos        larvae                          adult      adult <br />      darkturquoise <br />                                                                                                      white   pup     adult   adult    mated      post <br />               darkred <br />       darkorange2 <br />                                                                                                      pup     -ae       virgin   post       eclosion , <br />         darkorange                                                                                   -aria     eclos   post     eclosion   mixed <br />   darkolivegreen                   #                                                                         WPP     -ion    eclos               male <br />      darkmagenta                       *                                                                                     -ion                and <br />             darkgrey                                                                                                                             female <br />          darkgreen <br />                  cyan <br />               brown4                       orphan genes # <br />                 brown <br />                                            duplicated genes <br />                   blue    *#                                  * <br />                  black <br />  <br />                  0      5      10     15     20 <br />  Figure 2 Expression profile new genes module. (A) Enrichment new genes (including duplicated genes orphan genes) in <br />  gene module. Different symbols used modules significant enrichment duplicated orphan genes. (B-C) Histogram <br />  expression genes modules M24 (B) M37 (C) heatmaps expression genes modules seeing Additional <br />  files 9 10. <br />  <br />  <br />  <br /> genes modules, supporting potential gene                                 Additional file 9), 12 hr post-molt stage L3 larvae (P = 4e- <br /> co-expression network (Additional file 12).                                           27), 20-22 hr embryos (P = 1e-05), 4 days post-eclosion <br />   When considered young duplicated genes,                                 accessory glands adult males (P = 4E-32), 18-20 hr <br /> that young duplicated genes significantly enriched 7                           embryo stages (P = 2e-17), respectively (Additional file 3). <br /> modules, M37, M13, M24, M36, M23, M12 M26                                         These observations support previously reported testis- <br /> (P = 2.91E-06, P = 7.70E-06, P = 5.17E-05, P = 3.04E-04,                              bias expression new genes, the <br /> P = 0.0013, P = 0.030 P = 0.031, respectively, &#207;&#8225;2                          function new genes evolution pheno- <br /> test, FDR correction) (Figure 2A) (Additional file 13),                         types determined expression specific devel- <br /> which significantly correlated dark blue gut                             opmental stages. However, tissues (59 <br /> PS(1-2) stage L3 larvae (P = 3e-15, Figure 2C,                                     development stages tissues) unique highly ex- <br /> Additional file 10), 4 day post-eclosion testes adult                              pressed genes taken account, 40 genes <br /> males (P = 3e-15), stage L1 larvae (P = 1e-06, Figure 2B,                             137 young genes enriched M13 <br />  Liu et al. BMC Evolutionary Biology 2014, 14:241                                                                 Page 5 10 <br /> http://www.biomedcentral.com/1471-2148/14/241 <br />  <br />  <br />  <br />  <br /> module correlated significantly testes. In addition,   Roles specific lncRNAs development revealed from <br /> the significant clustering young genes different    gene co-expression network analysis <br /> co-expression modules correlate different tis-        We assembled 18,385-26,178 new transcripts the <br /> sues/developmental stages suggests young          transcriptome data 7 Drosophila species (Additional <br /> genes operate different functional classes,       file 16). In Drosophila melanogaster, 145 newly as- <br /> and testis. For example, gene ontology ana-     sembled transcripts, 133 genes, identified as <br /> lysis 5 modules young genes revealed         linage-specific lncRNAs homology tran- <br /> genes module M23 enriched functions          scripts species. Among new lncRNA <br /> are integral component membranes (intrinsic mem-          genes, 129 significantly associated with <br /> brane) (P = 0.0038, FDR correction) genes     25 modules, 6 modules, M14, M20, M42, M18, <br /> the module M24 enriched aromatic amino acid          M3, M46, significantly enriched Drosophila <br /> family metabolic process (P = 0.035, FDR correction)      melanogaster specific lncRNAs (P = 6.10E-31, P = 4.57E- <br /> and oxidation reduction process (P = 2.24E-07,        04, P = 8.15E-12, P = 8.75E-04, P = 3.25E-07 P = 3.82E- <br /> FDR correction), revealing potential functions        06, respectively, &#207;&#8225;2 test, FDR correction) <br /> new genes processes (Additional file 14).              (Figure 3A, Additional file 17). Of modules, M14 <br />    Our searches identified 84 young orphan genes           correlated adult male 30 days eclosion <br /> show homology proteins species (and            stage (Figure 3D, Additional file 18), M20 correlated <br /> were duplicated D. melanogaster lineage). 83 or-     pupae 12 hr WPP (P = 1e-06, Figure 3C, s <br /> phan genes significantly enriched 6 modules, M13,       Additional file 19), M18 correlated pupae 2 days <br /> M37, M24, M36, M41 M40 (P = 0.021, P = 0.0077,              WPP (P = 3e-24), M42 corrected stage L3 <br /> P = 3.02E-07, P = 0.0096, P = 2.88E-22 P = 0.0052, re-      CNS (P = 9e-20), M3 corrected stage L3 Imaginal <br /> spectively, &#207;&#8225;2 test, FDR correction) (Additional   Discs (P = 3e-28), M46 corrected heads of <br /> file 15, Figure 2A). Among modules, M13            mated adult males 20 days post-eclosion (P = 8e-04, <br /> correlated 4 day post-eclosion testes adult males       Figure 3B, Additional files 3 20). We used the <br /> (P = 3e-15), M37 correlated dark blue gut PS(1-2)       genes modules lncRNAs enriched <br /> in stage L3 larvae (P = 3e-15, Figure 2C, Additional            conduct gene ontology analysis, suggest <br /> file 10), M24 correlated stage L1 larvae (P = 1e-06,    potential functions lncRNAs. For example, <br /> Figure 2B, Additional file 9), M36 correlated          genes module M14 enriched odorant binding <br /> 12 hr post-molt stage L3 larvae (P = 4e-27), M41 cor-        (P = 2.98e-24, FDR correction), sensory perception <br /> rected WPP + 2 days Fat (P = 9e-37), M40            smell (P = 1.03E-17, FDR correction), olfactory re- <br /> corrected Mixed Adult Male Female Carcass              ceptor activity (P = 7.73E-15, FDR correction) and <br /> 4 days Post-eclosion (P = 2e-06) (Additional file 3), sug-      neurological process (P = 3.57E-06, FDR cor- <br /> gesting new genes roles stages      rection), suggesting potential function the <br /> development.                                                    lncRNAs regulating expression sensory perception <br />    The young genes queried Bristle        genes. In addition, genes module M20 enriched <br /> Screen online database [30]           defense response (P = 0.047, FDR correc- <br /> the 111 young duplicated genes (from 181 lines) having          tion), genes module M18 enriched proteolysis <br /> knock-down data, 4 genes (from 6 lines) com-               (P = 0.035, FDR correction), genes module <br /> pletely lethal (Additional file 13). The proportion          M46 enriched photo transduction (P = 1.06e-23, <br /> young genes showing lethality (3.3%) slightly lower         FDR correction) (Additional file 21), suggesting that <br /> than genome wide value (8.9%, http://bristlescreen.         lncRNAs regulatory roles processes. <br /> imba.oeaw.ac.at/data_summary.php). Among orphan             Although functions lncRNAs remain unclear, <br /> genes, 52 (from 84 lines) knock-down data           co-expression data provides information ex- <br /> Bristle Screen database, 1 gene (from 1 line)         amined future studies. <br /> being completely lethal (Additional file 15). Our results, <br /> of low proportion lethality young genes, contrasts      Essentiality positively selected genes Drosophila <br /> with previous conclusion young genes              9,746 one-<span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span>-one orthologs Drosophila melanoga- <br /> similar proportion lethality old genes [21], result     ster, D. annanassae, D. sechellia, D. yakuba D. erecta <br /> that attributable target effects     retrieved EnsemblMetazoa database (http:// <br /> of RNAi duplicated genes, producing false positives          metazoa.ensembl.org). After alignment trimming, <br /> for essentiality young duplicated genes. More        9,531 orthologs remained use detection posi- <br /> accurate experimental methods knocking down/out             tive selection employing branch site model PAML <br /> genes Drosophila necessary evaluate essen-        [32]. A total 263 383 Positive Selected Genes <br /> tiality young genes.                                         (PSGs) identified Drosophila melanogaster and <br />  Liu et al. BMC Evolutionary Biology 2014, 14:241                                                                                   Page 6 10 <br /> http://www.biomedcentral.com/1471-2148/14/241 <br />  <br />  <br />  <br />  <br />        A                                      B <br />                                                 0.2 <br />        yellowgreen <br />                 yellow                        0.15 <br />                  white <br />                  violet <br />                                                 0.1 <br />            turquoise <br />                     tan <br />             steelblue                         0.05 <br />             skyblue3 <br />               skyblue                              0 <br />               sienna3 <br />                salmon <br />       saddlebrown                             C <br />             royalblue                         0.25 <br />                    red <br />                 purple                          0.2 <br />                 plum1 <br />                                               0.15 <br />                   pink    * <br />      paleturquoise <br />        orangered4                               0.1 <br />                orange <br />       midnightblue                            0.05 <br />   mediumpurple3 <br />                                                    0 <br />             magenta <br />          lightyellow <br />    lightsteelblue1 <br />           lightgreen <br />                                               D <br />                                               0.25 <br />          lightcyan1 <br />             lightcyan <br />                                   * <br />                           *                     0.2 <br />                  ivory                * <br />                grey60                         0.15 <br />        greenyellow <br />                 green                           0.1 <br />          floralwhite <br />      darkturquoise                            0.05 <br />               darkred <br />       darkorange2                                  0 <br />                                                                           L3,Fatbody <br />  <br />  <br />  <br />                                                                        WPP,FatBody <br />  <br />  <br />  <br />  <br />                                                                         Female,1day <br />  <br />  <br />                                                                            Male,1day <br />                                                                                 WPP <br />  <br />  <br />  <br />                                                                      WPP+2daysFat <br />                                                                                0~2hr <br />                                                                                2~4hr <br />                                                                                4~6hr <br />                                                                                6~8hr <br />  <br />  <br />  <br />  <br />                                                                                 12hr <br />                                                                               8~10hr <br />  <br />  <br />  <br />  <br />                                                                                24hrs <br />                                                                                2days <br />                                                                                3days <br />                                                                                4days <br />                                                                              10~12hr <br />                                                                              12~14hr <br />                                                                              14~16hr <br />                                                                              16~18hr <br />                                                                              18~20hr <br />                                                                              20~22hr <br />                                                                              22~24hr <br />                                                                                   L1 <br />                                                                                   L2 <br />                                                                     L3,12hrpost~molt <br />  <br />  <br />  <br />                                                                           L3,Carcass <br />                                                                              L3,CNS <br />  <br />  <br />  <br />  <br />                                                                     WPP+2daysCNS <br />  <br />  <br />  <br />  <br />                                                                        Female,5days <br />  <br />  <br />                                                                           Male,5days <br />                                                                  L3,DigestiveSystem <br />  <br />                                                                     L3,ImaginalDiscs <br />  <br />  <br />  <br />                                                                 WPP,SalivaryGlands <br />  <br />  <br />  <br />  <br />                                                                       Female,30days <br />  <br />  <br />                                                                          Male,30days <br />  <br />  <br />                                                               Female,4days,Ovaries <br />  <br />  <br />  <br />                                                               Female,4days,Ovaries <br />  <br />  <br />  <br />  <br />                                                                   Male,4days,Testes <br />  <br />                                                                        1day,Carcass <br />  <br />  <br />  <br />                                                                      20days,Carcass <br />                                                                    L3,cleargutPS7~9 <br />  <br />  <br />  <br />  <br />                                                                   L3,SalivaryGlands <br />  <br />  <br />  <br />  <br />                                                                   Male,4days,Heads <br />  <br />  <br />  <br />                                                               1day,DigestiveSystem <br />                                                                       4days,Carcass <br />                                                                 Female,1day,Heads <br />                                                                Female,4days,Heads <br />  <br />  <br />                                                                 Female,1day,Heads <br />                                                                Female,4days,Heads <br />  <br />  <br />  <br />  <br />                                                              4days,DigestiveSystem <br />  <br />                                                            20days,DigestiveSystem <br />                                                                L3,darkbluegutPS1~2 <br />  <br />  <br />  <br />  <br />                                                               Female,20days,Heads <br />  <br />  <br />  <br />                                                               Female,20days,Heads <br />                                                                     Male,1day,Heads <br />                                                        Male,4days,AccessoryGlands <br />  <br />  <br />                                                                  Male,20days,Heads <br />                                                                L3,lightbluegutPS3~6 <br />  <br />  <br />  <br />  <br />         darkorange <br />   darkolivegreen <br />                               * <br />      darkmagenta <br />              darkgrey <br />          darkgreen <br />                  cyan <br />               brown4                                    Stage          Stage     Stage   Stage   Stage   Stage    Stage Stage of <br />                 brown         *                         embryos           larvae                            adult    adult <br />                   blue <br />                                                                                        white   pup     adult   adult    mated    post <br />                  black <br />                                                                                        pup     -ae       virgin   post     eclosion , <br />                    0     5      10     15                                              -aria     eclos   post     eclosion mixed <br />                                                                                                WPP     -ion    eclos             male <br />                                                                                                                -ion              and <br />                                                                                                                                  female <br />  Figure 3 Expression profile lncRNAs module. (A) Enrichment lncRNAs gene module. Stars modules with <br />  significant enrichment lncRNAs. (B-D) Histogram expression genes modules M46 (B), M20 (C) M14 (D) heatmaps the <br />  expression genes modules seeing Additional files 18, 19 20. <br />  <br />  <br />  <br /> D. sechellia lineages, respectively. Of one-to-one ortho-           sense module. The remaining 381 genes distributed <br /> logs, 7,939 (from 13,784 lines) knock-down data                 38 modules, 40 (from 49 lines) 304 <br /> the Bristle Screen database, 929 (~11%) genes (from                PSGs (from 553 lines) completely lethal (Additional <br /> 1,291 lines) completely lethal. Among 263 PSGs,               file 23). <br /> 216 genes (from 413 lines) knock-down data Dros-                   Excluding gene detectable expression <br /> ophila melanogaster, 36 genes (from 54 lines)             transcriptome datasets, 262 PSGs Drosophila <br /> being completely lethal (~13.08%) (Additional file 22). In              melanogaster distributed 36 modules were <br /> Drosophila sechellia, 383 PSGs, 1 gene did                significantly enriched M6 (P = 0.002196, &#207;&#8225;2 test, <br /> have detectable expression Drosophila melanogaster               FDR correction) (Additional file 22). Module M6 is <br /> transcriptome datasets 1 gene placed non-                significantly correlated 0-2 hr embryo stage <br />  Liu et al. BMC Evolutionary Biology 2014, 14:241                                                               Page 7 10 <br /> http://www.biomedcentral.com/1471-2148/14/241 <br />  <br />  <br />  <br />  <br /> (P = 4e-04), module significantly enriched   Roles positively selected genes development <br /> with lethal genes (Figure 1C, Additional file 5). Enrich-     With advent large-scale genomic data, large num- <br /> ment lethal genes points importance         bers positively selected adaptive mutations cod- <br /> module M6. Among 36 PSGs completely lethality        ing sequences genes involved evolution of <br /> in Drosophila melanogaster, 18 distributed module      phenotypes, including morphological physiological <br /> M6. These data demonstrate PSGs essential           traits, identified [14]. Here, positively selected <br /> functions, rapid evolution amino      genes showing rapid amino acid sequence change spe- <br /> acid sequence changes driven positive selection.           cific Drosophila melanogaster lineage identi- <br />                                                               fied, genes essential as <br /> Discussion                                                    knock-down leads lethality. The correlation between <br /> Roles new genes development revealed gene           evolutionary rate essentiality genes weak [34], <br /> co-expression network analysis                                gene essentiality tends associated adap- <br /> Early developmental stages important estab-      tive evolution protein sequence. Although, func- <br /> lishing phenotypes later stages. Based gene         tional shifts caused amino acid changes PSGs are <br /> co-expression network analysis, observed          unclear, mutations driven positive selection ap- <br /> non-random distribution essential genes early de-       pear contribute significantly evolution the <br /> velopmental stages Drosophila development leads         phenotypes generated PSGs. Enrichment PSGs <br /> different developmental stages showing varying levels      gene co-expression modules correlated early <br /> essentiality. For example, 0-2 hr embryonic stage,        developmental stages supports hypothesis changes <br /> which maternal zygotic transition period         amino acid sequences PSGs leads phenotypic <br /> Drosophila, enriched expression essential genes,       evolution. <br /> suggesting contribution gene expression of <br /> these genes period developmental time im-       Conclusions <br /> portant phenotype evolution Drosophila. Similarly,     Young genes, including duplicated, orphan, young <br /> we expression new genes, including          lncRNA genes, significantly enriched mod- <br /> young protein coding lncRNA genes,           ules associated specific developmental stages. In <br /> randomly distributed early development,         addition, genes undergoing rapid amino acid sequence <br /> significantly enriched modules correlate spe-    evolution driven positive selection showed similar <br /> cific developmental stages. This suggests     proportion essentiality genes, enrich- <br /> new genes play roles evolution specific        ment module specific developmental stages. Our <br /> phenotypes. New genes provide crucial material evo-       integrative analysis revealed important roles origin <br /> lutionary innovation, evidence accumulat-        new genes rapid amino acid sequence evolution in <br /> ing supporting remarkable phenotypic effects new       development account specific phenotype evo- <br /> genes development. A pioneering study Chen et al        lution Drosophila melanogaster extent. <br /> (2010) new genes rapidly evolve essen- <br /> tial functions Drosophila, knockdown ex-         Methods <br /> pression new genes <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span>          RNA-seq transcriptome data Drosophila melanogaster <br /> (RNAi) leads lethality various stages Drosophila     Drosophila species <br /> development [21]. Enrichment new genes early de-        Transcriptome data Drosophila melanogaster, includ- <br /> velopmental stages supports role adaptive function      ing 27 distinct developmental stages 12 tissues from <br /> by new genes early developmental stages. For ex-           different development stages [26], data six <br /> ample, human, recruitment young genes          Drosophila species (D. ananassae, D. mojavensis, <br /> the early developmental stages neocortex         D. pseudoobscura, D. simulans, D. virilis, D. yakuba) <br /> to account potentially evolution cognitive abil-   downloaded SRA database (http://www. <br /> ity [23]. Many studies remarkable diver-         ncbi.nlm.nih.gov/sra/) (Additional file 1). <br /> gence related species early and <br /> late development, consistent hourglass      Analysis transcript expression Tophat and <br /> model development, proposed        Cufflinks <br /> others genes expressed early stages develop-       Transcriptome raw read data, sra format, con- <br /> ment highly conserved [33]. Recruitment          verted fastq using SRA Toolkit (available http:// <br /> newly originated genes early stages developmental       www.ncbi.nlm.nih.gov/Traces/sra/?view=software), and <br /> might allow evolution specific phenotypes, con-      trimmed Btrim [35]. Tophat [36] (v2.0.4) im- <br /> served genes retain function conserved core        plemented align reads, default, Cufflinks <br /> development.                                                  (v 2.0.2) used assemble transcripts &#226;&#8364;&#8220;g <br />  Liu et al. BMC Evolutionary Biology 2014, 14:241                                                                               Page 8 10 <br /> http://www.biomedcentral.com/1471-2148/14/241 <br />  <br />  <br />  <br />  <br /> parameter. Cuffcompare used generate inte-              gene ages making use different ancestral family re- <br /> grated combined gtf annotation file                construction algorithms pre-existing protein family <br /> Ensembl annotation file. New transcripts class codes         database phylogenetic trees. Duplicated genes in <br /> &#226;&#8364;&#339;c, j, e, i, o, u, x&#226;&#8364;?, having 2 exons, lengths longer   Drosophila melanogaster retrieved Ensembl <br /> than 200 bp, expression levels FPKM values               using Biomart [40]. However, Ensembl database dates <br /> higher 1.0 sample, retrieved,       duplication occurred, does provide in- <br /> merged reference annotated gtf file. The merged          formation gene source copy which <br /> gtf file used input Cufflinks calculate expres-     derived copy. Here, defined young duplicated <br /> sion values gene &#226;&#8364;&#8220;G parameter.                  genes (derived copies) duplicated genes an <br />                                                                   age 0 induced ProteinHistorian. Protein sequences <br /> Detection lncRNAs specific Drosophila melanogaster          Drosophila melanogaster used BLASTP against <br /> The coding potential transcripts newly assem-         protein database Drosophila e-value <br /> bled loci evaluated coding Potential Calculator        1e-10 [38] (1e-3 1e-4 used results, seeing <br /> (CPC) [37], values lower -1 considered           Additional files 24 25, similar 1e- <br /> be noncoding. All transcripts Drosophila spe-        10). The genes Drosophila melanogaster having no <br /> cies (D. ananassae, D. mojavensis, D. pseudoobscura, D.           homologous hits species treated or- <br /> simulans, D. virilis, D. yakuba), including annotated         phan genes. <br /> cDNA newly assembled transcripts, merged. <br /> Drosophila melanogaster lncRNAs used quer-            Detection positive selection protein coding genes <br /> ies BLASTN searches merged cDNA              Orthologous_one2one genes melanogaster group: <br /> a cutoff e-value 1e-10 [38]. Drosophila melanogaster              Drosophila sechellia, D. menlanogaster, D. yakuba, D. <br /> lncRNAs homologous hits species              erecta, D. ananassae retrieved using Biomart from <br /> were treated Drosophila melanogaster specific.                 EnsemblMetazoa (http://metazoa.ensembl.org). PRANK <br />                                                                   (available http://www.ebi.ac.uk/goldman-srv/prank/ <br /> WGCNA (weighted gene co-expression network analysis)              prank/) used multiple sequence alignments the <br /> WGCNA (weighted gene co-expression network ana-                   codon level. To obtain high quality sequence alignments, <br /> lysis) package implemented R [27] used con-             longest transcript species chosen when <br /> struct weighted gene network based expression            genes transcript sequencing errors <br /> values calculated above. The power 8,            non-orthologous sequences removed using an <br /> scale-free topology fit index curves flatten roughly       in-house script. Only alignments lengths longer than <br /> 0.9, interpreted soft-threshold adja-           30 codons retained. The accepted phylogenetic rela- <br /> cency matrix. In total, 46 modules identified. Un-           tionship melanogaster group used (((Drosophila <br /> assigned genes, 37genes, placed &#226;&#8364;&#339;grey&#226;&#8364;?              schellia, D. melanogaster), (D. yakuba, D. erecta)), D. ana- <br /> module (module M47). Module membership, defined                nassae). The CODEML algorithm PAML4 pack- <br /> the Pearson correlation expression level           age [41] used detect positive selection utilizing the <br /> a given gene given module eigengene calcu-              branch-site model [32]. <br /> lated identify intramodular hub genes. Modules whose <br /> eigengenes (referred principal component) <br />                                                                   Gene ontology analysis <br /> were highly correlated different biological stages <br />                                                                   The Database Annotation visualization Inte- <br /> were merged. <br />                                                                   grated Discovery (DAVID) used investigate the <br />    Each module summarized single representa- <br />                                                                   enrichment Gene Ontology (GO) terms KEGG <br /> tive expression profile, referred the <br />                                                                   pathways [42]. In order avoid selection bias, also <br /> module eigengene. The module eigengene more <br />                                                                   performed GO analysis using goseq [43] package <br /> complete representation gene co-expression relation- <br />                                                                   (http://www.bioconductor.org/packages/release/bioc/html/ <br /> ship network. We correlated eigengenes with <br />                                                                   goseq.html) R (v-2.15.3). The analysis corrected for <br /> various developmental stages access association <br />                                                                   <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> Drosophila melanogaster reference genes. FDR <br /> between module developmental stage. For ex- <br />                                                                   correction performed using package &#226;&#8364;&#339;multtest&#226;&#8364;? <br /> ample, correlation, like 0.97, indicates high associ- <br />                                                                   (available http://www.bioconductor.org/packages/release/ <br /> ation gene co-expression network one <br />                                                                   bioc/html/multtest.html) parameter &#226;&#8364;&#339;proc = BH&#226;&#8364;?. <br /> developmental stage. <br />  <br /> Identification new genes Drosophila melanogaster            Additional files <br /> Ages genes Drosophila melanogaster retrieved <br />                                                                    Additional file 1: Transcriptome data used study. <br /> from ProteinHistorian [39]. ProteinHistorian estimated <br />  Liu et al. BMC Evolutionary Biology 2014, 14:241                                                                                          Page 9 10 <br /> http://www.biomedcentral.com/1471-2148/14/241 <br />  <br />  <br />  <br />  <br />  Additional file 2: Coexpression modules identified study        Additional file 25: Young orphan genes (BLASTP e-value 1e-4), <br />  labeled numerically color.                                       results Bristle Screen online database and <br />  Additional file 3: Table presenting correlations (and               membership modules co-expressed genes. <br />  corresponding p-values) modules developmental <br />  stages adult tissues. <br />                                                                         Competing interests <br />  Additional file 4: Heatmap expression genes module        The authors declare competing interests. <br />  M17, expression genes significantly associated with <br />  white prepupae (WPP) stages.                                       Authors&#226;&#8364;&#8482; contributions <br />  Additional file 5: Heatmap expression genes module        DDW YPZ designed study. HQL YL performed analysis. HQL, <br />  M6, expression genes significantly associated     DDW wrote manuscript, DMI substantial intellectual contributions <br />  0-2 hr embryo stage.                                               project. All authors read approved final manuscript. <br />  Additional file 6: Heatmap expression genes mouse <br />  M28, expression genes significantly associated    Acknowledgements <br />  10-12 hr embryo development stage.                                 This work supported grants National Natural Science <br />                                                                         Foundation China (31271339 31321002). <br />  Additional file 7: Gene ontology analysis genes M6 <br />  module, results DAVID GOseq. <br />                                                                         Author details <br />  Additional file 8: New genes, results Bristle        1 <br />                                                                          State Key Laboratory Genetic Resources Evolution, Kunming Institute <br />  Screen online database membership modules                Zoology, Chinese Academy Sciences, Kunming, China. 2Kunming <br />  co-expressed genes.                                                    College Life Science, University Chinese Academy Sciences, Kunming, <br />  Additional file 9: Heatmap expression genes module        Yunnan, China. 3Department Laboratory Medicine Pathobiology, <br />  M24, expression genes significantly associated    University Toronto, Toronto, Canada. 4Banting Best Diabetes Centre, <br />  stage L1 larvae.                                                   University Toronto, Toronto, Canada. <br />  Additional file 10: Heatmap expression genes module <br />                                                                         Received: 23 July 2014 Accepted: 10 November 2014 <br />  M37, expression genes significantly associated with <br />  dark blue gut PS(1-2) stage L3 larvae. <br />  Additional file 11: Histogram expression new genes        References <br />  modules M13 (A), M37 (B), M23 (C), M24 (D), M36 (E) M41 (F).       1. Britten RJ, Davidson EH: Repetitive non-repetitive DNA sequences <br />  Additional file 12: The PCA (Principal Component Analysis)              speculation origins evolutionary novelty. Q Rev Biol 1971, <br />  modules M13 (A), M37 (B), M23 (C), M24 (D), M36 (E) M41 (F).           46:111&#226;&#8364;&#8220;138. <br />  Additional file 13: Newly duplicated genes, results      2. King M-C, Wilson AC: Evolution levels humans chimpanzees. <br />  Bristle Screen online database membership modules                Science 1975, 188(4184):107&#226;&#8364;&#8220;116. <br />  co-expressed genes.                                                    3. Carroll SB: The Making Fittest: DNA Ultimate Forensic Record of <br />                                                                             Evolution. New York: WW Norton &amp; Company; 2007. <br />  Additional file 14: Gene ontology analysis genes M23     4. Carroll SB: Evolution levels: genes form. PLoS Biol 2005, <br />  M24 modules, results DAVID GOseq.                            3(7):e245. <br />  Additional file 15: Young orphan genes (BLASTP e-value            5. Carroll SB: Endless Forms Most Beautiful: The New Science Evo Devo the <br />  1e-10), results Bristle Screen online database       Making Animal Kingdom. New York: WW Norton &amp; Company; 2005. <br />  membership modules co-expressed genes.                       6. Carroll SB: Evo-devo expanding evolutionary synthesis: genetic <br />  Additional file 16: Total newly assembled transcripts 7           theory morphological evolution. Cell 2008, 134(1):25&#226;&#8364;&#8220;36. <br />  Drosophila species.                                                    7. Enard W, Gehre S, Hammerschmidt K, H&#195;&#182;lter SM, Blass T, Somel M, Br&#195;&#188;ckner <br />                                                                             MK, Schreiweis C, Winter C, Sohr R, Becker L, Wiebe V, Nickel B, Giger T, <br />  Additional file 17: Detection lncRNAs detected                   M&#195;&#188;ller U, Groszer M, Adler T, Aguilar A, Bolle I, Calzada-Wack J, Dalke C, <br />  membership modules co-expressed genes.                           Ehrhardt N, Favor J, Fuchs H, Gailus-Durner V, Hans W, H&#195;&#182;lzlwimmer G, <br />  Additional file 18: Heatmap expression genes module           Javaheri A, Kalaydjiev S, Kallnik M, et al: A humanized version Foxp2 <br />  M14, expression genes significantly associated        affects cortico-basal ganglia circuits mice. Cell 2009, 137(5):961&#226;&#8364;&#8220;971. <br />  adult male 30 days eclosion stage.                           8. Zhang J: Evolution human ASPM gene, major determinant of <br />  Additional file 19: Heatmap expression genes module           brain size. Genetics 2003, 165(4):2063&#226;&#8364;&#8220;2070. <br />  M20, expression genes significantly associated    9. Vallender EJ, Lahn BT: Positive selection human genome. Hum Mol <br />  pupae 12 hr WPP.                                                 Genet 2004, 13(suppl 2):R245&#226;&#8364;&#8220;R254. <br />                                                                         10. Evans PD, Anderson JR, Vallender EJ, Gilbert SL, Malcom CM, Dorus S, Lahn <br />  Additional file 20: Heatmap expression genes module           BT: Adaptive evolution ASPM, major determinant cerebral cortical <br />  M46, expression genes significantly associated        size humans. Hum Mol Genet 2004, 13(5):489&#226;&#8364;&#8220;494. <br />  heads mated adult males 20 days post-eclosion.                  11. Evans PD, Anderson JR, Vallender EJ, Choi SS, Lahn BT: Reconstructing the <br />  Additional file 21: Gene ontology analysis genes M14,            evolutionary history microcephalin, gene controlling human brain <br />  M20, M18, M41 M45 modules, results DAVID                 size. Hum Mol Genet 2004, 13(11):1139&#226;&#8364;&#8220;1145. <br />  GOseq.                                                                 12. Y-q W, Su B: Molecular evolution microcephalin, gene determining <br />  Additional file 22: PSGs Drosophila melanogaster,              human brain size. Hum Mol Genet 2004, 13(11):1131&#226;&#8364;&#8220;1137. <br />  results Bristle Screen online database membership      13. Enard W, Przeworski M, Fisher SE, Lai CSL, Wiebe V, Kitano T, Monaco AP, <br />  modules co-expressed genes.                                         Paabo S: Molecular evolution FOXP2, gene involved speech and <br />                          <br /> </body></html>