<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                HHS Public Access <br />                                 Author manuscript <br />                                 J Biomol Screen. Author manuscript; available PMC 2015 March 30. <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                     Published final edited form as: <br />                      J Biomol Screen. 2015 January ; 20(1): 122&#226;&#8364;&#8220;130. doi:10.1177/1087057114548832. <br />  <br />  <br />  <br />                     Identification Potent Inhibitors Trypanosoma brucei <br />                     Methionyl-tRNA Synthetase High Throughput Orthogonal <br />                     Screening <br />                     Laura Pedr&#195;&#179;-Rosa1,&#226;&#8364;&#160;, Frederick S. Buckner2,&#226;&#8364;&#160;, Ranae M. Ranade2,&#226;&#8364;&#160;, Christina Eberhart1, <br />                     Franck Madoux1, J. Robert Gillespie2, Cho Yeow Koh3, Steven Brown4, Jacqueline Lohse4, <br />                     Christophe L. M. Verlinde3, Erkang Fan3, Thomas Bannister1, Louis Scampavia1, Wim G. J. <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                     Hol3, Timothy Spicer1,*, Peter Hodder1,&#194;&#167; <br />                     1The   Scripps Research Institute Molecular Screening Center, Scripps Florida, Jupiter, Florida <br />                     2Department      Medicine, University Washington, Seattle, Washington <br />                     3Department      Biochemistry, University Washington, Seattle, Washington <br />                     4Department      Chemical Physiology, The Scripps Research Institute, La Jolla, California <br />  <br />                     <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                          Improved therapies treatment Trypanosoma brucei (T. brucei), etiological agent of <br />                          neglected tropical disease human African trypanosomiasis, urgently needed. We targeted T. <br />                          brucei methionyl-tRNA synthetase (MetRS), aminoacyl-tRNA synthase (aaRS), is <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                          considered important drug target role protein synthesis, cell <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>survival</span> its <br />                          significant differences structure mammalian ortholog. Previous work using RNA <br />                          interference MetRS demonstrated growth inhibition T. brucei, validating an <br />                          attractive target. We report development implementation orthogonal high <br />                          throughput screening assays identify inhibitors T. brucei MetRS. First, chemiluminescence <br />                          assay implemented 1536-well plate format used monitor ATP depletion the <br />                          aminoacylation reaction. Hit confirmation used counterscreen AMP production <br />                          assessed using fluorescence polarization technology. In addition, miniaturized cell viability <br />                          assay used triage cytotoxic compounds. Finally, lower throughput assays involving whole <br />                          parasite growth inhibition human parasite MetRS used analyze compound <br />                          selectivity efficacy. The outcome HTS campaign led discovery nineteen <br />                          potent selective T. brucei MetRS inhibitors. <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                     Keywords <br />                          aminoacyl-tRNA synthetases; high-throughput screening; human African trypanosomiasis; <br />                          orthogonal screening; Trypanosoma brucei <br />  <br />  <br />  <br />  <br />                     * <br />                      Address correspondence <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span>: Timothy Spicer Scripps Florida 130 Scripps Way #1A1 Jupiter, FL 33458 U.S.A. spicert@scripps.edu. <br />                     &#194;&#167;Current affiliation: Amgen Inc., Thousand Oaks, California. <br />                     &#226;&#8364;&#160;These authors equal contributions. <br />                      Pedr&#195;&#179;-Rosa et al.                                                                                            Page 2 <br />  <br />  <br />                        Introduction <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Aminoacyl-tRNA synthases (aaRS) essential protein synthesis, catalyzing the <br />                                         synthesis aminoacyl-tRNAs. The catalyzed reaction aaRS usually proceeds two <br />                                         steps: recognition specific amino acid ATP form aminoacyl- <br />                                         adenylate intermediate secondly, recognition associated tRNA transfer of <br />                                         aminoacyl group terminal adenosine tRNA. Inhibition step results <br />                                         disruption protein biosynthesis, attenuating cell growth compromising viability. <br />                                         aaRS excellent drug targets inhibition, delivering selective cytotoxicity vs. <br />                                         pathogens. This demonstrated development drug mupirocin, which <br />                                         selectively inhibits IleRS archaea eubacteria but, does inhibit eukaryotic <br />                                         IleRS.1 Other aaRSs targeted high throughput screening (HTS) efforts, as <br />                                         PheRS MetRS S. aureus.2 Herein HTS program targeting MetRS <br />                                         protozoan parasite Trypanosoma brucei, including orthogonal counterscreen. <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         T. brucei causative agent neglected tropical disease human African <br />                                         trypanosomiasis (HAT) threatens ~60 million people sub-Saharan Africa.3 When <br />                                         left untreated HAT fatal current treatment outcomes unsatisfactory high <br />                                         toxicity, difficult administration regimes poor efficacy existing drugs.4 Hence, new <br />                                         drugs effective, non-toxic affordable needed. Early studies using RNAi <br />                                         demonstrated MetRS T. brucei essential parasite survival, making MetRS <br />                                         inhibition attractive strategy treating HAT.5,6,7 Potent aminoquinolone-based <br />                                         compounds, like 2-((3-((3,5-dichlorobenzyl)amino)propyl)amino)quinolin-4(1H)-one (CID <br />                                         18353708, Figure 2), identified present poor bioavailability.8 Moderately <br />                                         potent urea-based compounds oxazolone&#226;&#8364;&#8220;dipeptides reported.9,10,11 The goal <br />                                         HTS campaign presented identify selective, potent drug like small <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         molecule T. brucei MetRS inhibitors, ultimate potential novel <br />                                         therapeutics HAT. <br />  <br />                                         To minimize false leads, key aspect effort use orthogonal screens early <br />                                         program, using different assay technologies, identify novel inhibitors MetRS <br />                                         catalyzed reaction vitro. This reaction monitored measuring residual <br />                                         ATP AMP MetRS reaction luminescence fluorescence polarization (FP), <br />                                         respectively. Excellent performance assays 1536-well plate format achieved <br />                                         screen MLSMR collection 364,131 compounds. More dozen <br />                                         tractable compounds sub micromolar potency identified using assays. <br />  <br />  <br />                        Materials Methods <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                        Protein expression purification <br />                                         The full-<span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> gene T. brucei MetRS (Tb927.10.1500) amplified T. brucei <br />                                         brucei strain 427, expressed E. coli, purified previously described.8 The N-terminal <br />                                         6-His tag retained. The human mitochondrial MetRS enzyme expressed, as <br />                                         previously reported, plasmid provided Dr. L. Spremulli.12 <br />  <br />  <br />  <br />  <br />                                            J Biomol Screen. Author manuscript; available PMC 2015 March 30. <br />                      Pedr&#195;&#179;-Rosa et al.                                                                                           Page 3 <br />  <br />  <br />                        Optimization Miniaturization MetRS inhibition assays <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         The assay high-throughput screening required robust, non-radiometric methodology. <br />                                         An ATP depletion assay based chemiluminescence developed initial <br />                                         screen.13,14 Optimization Kinase-Glo&#194;&#174; reagent (Promega) initially 96- <br />                                         format using buffer conditions previously utilized radiometric aminoacylation <br />                                         assay:8 10 mM MgCl2, 25 mM HEPES-KOH pH 7.9, 50 mM KCl, 2.5 mM dithiothreitol, <br />                                         0.1 mg/mL bovine serum albumin, 0.2 mM spermine, 10 U/mL pyrophosphatase. To <br />                                         achieve separation high controls (wells containing cold L-methionine and <br />                                         inhibitors) low controls (wells containing inhibitors) different variables <br />                                         (incubation time, substrate concentration, enzyme concentration) manipulated <br />                                         independently conditions held constant. Optimized substrate <br />                                         concentrations follows: L-methionine 32 &#194;&#181;M, bulk E. coli tRNA (Sigma-Aldrich) <br />                                         200 &#194;&#181;g/mL, ATP 100 nM. The L-methionine range reported Km of <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         54&#194;&#177;42 &#194;&#181;M15 concentration ATP lower Km reported related <br />                                         MetRS enzyme E. coli (20 &#194;&#181;M).12 It necessary use low concentration ATP to <br />                                         achieve optimal separation high low controls. Once substrate <br />                                         concentrations established, enzyme reaction progress curves (Supplement Figure 1A) <br />                                         completed determine optimal enzyme concentration incubation time, 50 nM <br />                                         120 minutes respectively. Under conditions ATP consumed was <br />                                         85% resulted excellent assay reproducibility statistics average Z&#226;&#8364;&#8482;- <br />                                         factor 0.84 (Supplement Figure 1B). DMSO tolerance tested significant <br />                                         decrease ATP consumption observed 2% DMSO. For HTS, <br />                                         pyrophosphatase decreased 10 0.1 U/mL significant decrease ATP <br />                                         consumption. The assay miniaturized 384-well format Scripps Research <br />                                         Institute (La Jolla, CA) pilot screen 5,120 Maybridge compounds tested 1 &#194;&#181;M <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         completed. The pilot screen excellent Z&#226;&#8364;&#8482;-factors 0.75 &#226;&#8364;&#8220; 0.89. Finally, assay <br />                                         miniaturized 1536-well format discussed results section. <br />  <br />                                         A stepwise protocol depicted Supplement Table 1. During primary luminescence- <br />                                         based screen, test compounds analyzed singlicate final nominal concentration of <br />                                         12 &#194;&#181;M using automated Kalypsys/GNF platform (GNF Systems, San Diego, CA) the <br />                                         Scripps Research Institute Molecular Screening Center (SRIMSC, Jupiter, FL). The cutoff <br />                                         used qualify active compounds calculated average percent inhibition the <br />                                         compounds tested plus times standard deviation.16 The confirmation assay and <br />                                         orthogonal FP-based counterscreen tested compound triplicate single <br />                                         concentration (~12 &#194;&#181;M). The primary assay hit-cutoff applied confirmation assay. <br />                                         The counterscreen employed algorithm determine hit cut off. Actives were <br />                                         progressed forward determine concentration response curves (CRC) (Table 1). CRC <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         experiments performed above, testing compounds triplicate 10-point <br />                                         serial dilution starting 120 &#194;&#181;M. ATP levels measured luminescence using <br />                                         Kinase-Glo&#194;&#174; detection reagent (Promega) ViewLux microplate reader (PerkinElmer). <br />                                         In orthogonal assay, AMP levels measured FP using Transcreener&#194;&#174; <br />                                         AMP2/GMP2 reagent (BellBrook Labs) EnVision microplate reader (PerkinElmer) <br />                                         using Cy5 FP filter set Cy5 dichroic mirror (excitation = 620 nm, emission = 688 <br />  <br />  <br />  <br />                                            J Biomol Screen. Author manuscript; available PMC 2015 March 30. <br />                      Pedr&#195;&#179;-Rosa et al.                                                                                           Page 4 <br />  <br />  <br />                                         nm). FP read using 50 flashes well. Further details assays be <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         PubChem website using unique assay identifiers listed Table 1. <br />  <br />                        Cytotoxicity counterscreen <br />                                         Jurkat cells (clone E6.1, ATCC- TIB-152) cultured suspension growth media <br />                                         containing RPMI-1640, 10% dialyzed fetal bovine serum, 0.1 mM NEAA, 1 mM sodium <br />                                         pyruvate, 25 mM HEPES, 5 mM L-glutamine 1&#195;&#8212; Anti-Anti (Life Technologies, <br />                                         Carlsbad, CA.) 37&#194;&#176;C 95% relative humidity 5% CO2. Five cells well <br />                                         added 1536-well plates incubated compounds DMSO 18hrs <br />                                         37&#194;&#176;C 95% relative humidity 5% CO2, time 5&#194;&#181;l CellTiter-Glo <br />                                         (Promega, Madison, WI) added wells. Plates centrifuged incubated for <br />                                         10 minutes room temperature. Luminescence measured ViewLux microplate <br />                                         reader (PerkinElmer, Waltham, MA). Doxorubicin used positive control. For more <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         information, Table 1 PubChem AID 364. <br />  <br />                        Aminoacylation assay (T. brucei human mitochondrial MetRS) <br />                                         To confirm specificity hits HTS, follow-up screen performed using <br />                                         aminoacylation assay quantifies incorporation [3H]L-methionine tRNA.8 <br />                                         Reactions performed 96-well filter plates Durapore membranes <br />                                         (MSHVN4B10; Millipore, Billerica, MA) volumes 75 &#194;&#181;l. The reaction buffer 25 <br />                                         mM HEPES-KOH pH 7.9, 10 mM MgCl2, 50 mM KCl, 0.2 mM spermine, 0.1 mg/ml <br />                                         bovine serum albumin, 2.5 mM dithiothreitol, 0.1 mM ATP, 240 nM [3H]L-methionine (83 <br />                                         Ci/mmol), 2% DMSO, 0.1 U/ml pyrophosphatase (I1643; Sigma-Aldrich, St. Louis, <br />                                         MO). Recombinant enzyme (10 nM) test compounds (10 &#194;&#181;M 1 &#194;&#181;M) mixed with <br />                                         buffer preincubated 15 min. To start reaction, 400 &#194;&#181;g/ml bulk Escherichia <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         coli tRNA (R4251; Sigma-Aldrich, St. Louis, MO) added plate incubated <br />                                         shaking room temperature 120 min. The reactions stopped the <br />                                         addition 100 &#194;&#181;l/well cold 10% trichloroacetic acid. The reaction components were <br />                                         separated tRNA filtration vacuum manifold washed times with <br />                                         300 &#194;&#181;l/well cold 10% trichloroacetic acid. The filter plates dried, 25 &#194;&#181;l/well <br />                                         scintillation fluid added, counts plates determined using a <br />                                         scintillation plate counter. Samples run triplicate, average activity of <br />                                         inhibitors compared control wells inhibitors. Percent inhibition was <br />                                         calculated concentrations (10 &#194;&#181;M 1 &#194;&#181;M). <br />  <br />                                         As selectivity counterscreen, hit compounds assayed human <br />                                         mitochondrial MetRS enzyme using aminoacylation methodology described above <br />                                         differences. The assay performed reaction buffer containing 50 mM <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Tris-HCl pH 8.0, 6 mM MgCl2, 2.5 mM KCl, 0.2 mM spermine, 0.2 mg/ml bovine serum <br />                                         albumin, 2.5 mM dithiothreitol, 2.5 mM ATP, 240 nM [3H]L-methionine (83 Ci/mmol), 2% <br />                                         DMSO, 0.1 U/ml pyrophosphatase. Recombinant enzyme (13 nM) inhibitors (10 <br />                                         &#194;&#181;M 1 &#194;&#181;M) mixed buffer pre-incubated 15 min. To start reaction, <br />                                         200 &#194;&#181;g/ml bulk Escherichia coli tRNA added. The plate incubated shaking <br />                                         room temperature 60 min processed described above. <br />  <br />  <br />  <br />                                            J Biomol Screen. Author manuscript; available PMC 2015 March 30. <br />                      Pedr&#195;&#179;-Rosa et al.                                                                                              Page 5 <br />  <br />  <br />                        T. brucei growth inhibition assay <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         T. brucei (bloodstream form strain 427 K. Stuart, Seattle Biomedical Research <br />                                         Institute, Seattle, WA) cultured HMI-9 medium17 containing 10% fetal bovine <br />                                         serum, penicillin, streptomycin 37&#194;&#176;C 5% CO2. Drug sensitivity T. brucei <br />                                         strain determined 96-well microtiter plates triplicate initial inoculum of <br />                                         1&#195;&#8212;104 trypomastigotes well. Compound stock solutions prepared DMSO 20 <br />                                         mM and/or 10 mM added serial dilutions final volume 200 &#194;&#181;l/well. Parasite <br />                                         growth quantified 48 h addition AlamarBlue (Alamar Biosciences, <br />                                         Sacramento, CA).18 Pentamidine isethionate (Aventis, Dagenham, United Kingdom) was <br />                                         included assay positive control. Compounds tested triplicate using 4- <br />                                         point serial fold dilution starting test concentration 10 &#194;&#181;M. Inhibition at <br />                                         various concentrations determined EC50 value calculated online <br />                                         service CDD (www.collaborativedrug.com) using equation sigmoidal <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         dose-response curve. <br />  <br />                        Screening data acquisition, normalization, representation analysis <br />                                         Raw luminescence data FP values uploaded SRIMSC institutional HTS <br />                                         database (Symyx Technologies Inc., Santa Clara, CA). Raw luminescence data was <br />                                         imported directly FP values calculated using following equation prior to <br />                                         upload: <br />  <br />  <br />  <br />  <br />                                         Where Raw1 defined polarized fluorescence P channel Raw2 defined as <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         polarized fluorescence S channel. <br />  <br />                                         In case, response data normalized using following equation: <br />  <br />  <br />  <br />  <br />                                         High Control represents wells plate containing 1.5 &#194;&#181;M control <br />                                         compound CID 18353708, case cytotoxicity assay, 8 &#194;&#181;M doxorubicin, and <br />                                         Low Control represents wells treated DMSO only. <br />  <br />                                         Data normalized plate basis assay plate underwent quality control <br />                                         check; Z&#226;&#8364;&#178; value greater 0.5 required acceptance data.19 CRC data were <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         analyzed plotting average percent inhibition triplicate results the <br />                                         compound concentration. A four-parameter equation describing sigmoidal concentration- <br />                                         response curve fitted adjustable baseline using Assay Explorer software <br />                                         (Symyx). Similarly, graphs shown manuscript, CRCs pIC50 values were <br />                                         generated Prism (GraphPad Software, San Diego, CA). All results the <br />                                         screens NIH&#226;&#8364;&#8482;s PubChem website (http://pubchem.ncbi.nlm.nih.gov/) <br />                                         utilizing AID listed Table 1. <br />  <br />  <br />                                            J Biomol Screen. Author manuscript; available PMC 2015 March 30. <br />                      Pedr&#195;&#179;-Rosa et al.                                                                                          Page 6 <br />  <br />  <br />                        Cheminformatics <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Shared scaffolds active compound families confirmation screen CRC <br />                                         experiments identified using Maximum Common Substructure hierarchical clustering <br />                                         (ChemAxon LibraryMCS 5.10.2). The physical properties (i.e. molecular mass, topological <br />                                         polar surface area, chiral atoms, H-bond acceptors/donors, ring count rotatable bonds) of <br />                                         compounds tested dose response format calculated (ChemAxon Instant JChem <br />                                         6.2.2). <br />  <br />                        Chemicals <br />                                         The MLSMR library provided BioFocus DPI (South San Francisco, CA) the <br />                                         NIH&#226;&#8364;&#8482;s Roadmap Molecular Libraries Initiative. Details regarding compound selection for <br />                                         library online http://mli.nih.gov/mli/compound-repository/mlsmr- <br />                                         compounds/. Briefly, library highly diversified collection small molecules (more <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         50% molecular weight range 350&#226;&#8364;&#8220;410 g/mol) comprised synthetic <br />                                         natural products, commercial academic sources, grouped into <br />                                         4 following categories: (1) specialty sets known bioactive compounds drugs <br />                                         toxins (0.65%), (2) focused libraries aimed specific target classes (2.85%), (3) non- <br />                                         commercial sources (7.4%) (4) diversity sets covering large area chemical space <br />                                         (89.1%). <br />  <br />                                         The positive control compound CID18353708 synthesized described elsewhere.8 <br />  <br />  <br />                        Results <br />                        Implementation orthogonal HTS assays <br />                                         A luminescence based assay designed run primary screen identify inhibitors <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         T. brucei MetRS measuring ATP depletion aminoacylation reaction. <br />                                         Residual ATP measured luciferase catalyzed reaction mono- <br />                                         oxygenation luciferin substrate (Kinase-Glo&#194;&#174;) luminescence signal is <br />                                         inversely proportional MetRS activity (Figure 1). <br />  <br />                                         To reduce cost increase throughput HTS campaign, assay was <br />                                         miniaturized implemented 1536-well plate format. Optimization assay was <br />                                         performed assessing effect critical variables affect aminoacylation <br />                                         reaction; (i.e.; MetRS concentration, ATP concentration, DMSO tolerance; see <br />                                         Supplement). Three sets controls, N=24 set, placed assay plate: the <br />                                         high control (1.5 &#194;&#181;M CID18353708 final), low control containing DMSO (0.9% final) <br />                                         50% inhibition control containing CID18353708 IC50 concentration (40 nM). <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         These controls used (1) ensure T. brucei MetRS activity, (2) normalize data, and <br />                                         (3) monitor data quality measuring Z&#226;&#8364;&#8482; S:B. Two additional controls placed <br />                                         plate: positive control containing ATP tRNA L-methionine, a <br />                                         background control ATP ensure proper compound reagent dispensing. Good <br />                                         separation controls observed shown plot raw luminescence values <br />                                         measured primary screen (Figure 2). All primary HTS data normalized the <br />                                         inhibition compound CID18353708 vs. DMSO wells, used produce a <br />  <br />  <br />                                            J Biomol Screen. Author manuscript; available PMC 2015 March 30. <br />                      Pedr&#195;&#179;-Rosa et al.                                                                                              Page 7 <br />  <br />  <br />                                         scatterplot aid visualization activity HTS campaign (Figure 3). <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Notably positive control compound shown produce sub-micromolar <br />                                         inhibition T. brucei MetRS.18 Under final conditions listed Supplement Table 1, <br />                                         excellent assay performance observed average Z&#226;&#8364;&#8482; factor 0.93 &#194;&#177; 0.02 a <br />                                         pIC50=7.62&#194;&#177;0.05 (IC50=24.20 &#194;&#177; 2.74 nM) Hill slope 4.41 &#194;&#177; 0.56 for <br />                                         CID18353708 (N=11). <br />  <br />                                         The orthogonal FP counterscreen implemented help prioritize hit selection by <br />                                         removing potential artifacts luminescent detection (quenchers, luciferase <br />                                         inhibitors, etc.). This assay measures AMP production T. brucei MetRS catalyzed <br />                                         aminoacylation reaction function proportional MetRS activity. Optimal assay <br />                                         conditions accord manufacturer&#226;&#8364;&#8482;s recommendations (BellBrook Labs, <br />                                         Madison, WI) subsequent experiments performed using conditions listed in <br />                                         Supplement Table 1. In miniaturized format, Z&#226;&#8364;&#8482; factor 0.71 &#194;&#177; 0.02 a <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         pIC50=7.52&#194;&#177;0.07 (IC50 = 30.34 &#194;&#177; 4.81) nM Hill slope 4.34 &#194;&#177; 0.17 for <br />                                         CID18353708 (N=3). Inhibition MetRS CID18353708 nearly identical both <br />                                         assays, indication good correlation sensitivity assay formats <br />                                         (Figure 2). <br />  <br />                        T. brucei MetRS inhibition screening campaign results <br />                                         The T. brucei MetRS inhibition screening campaign summarized Table 1. A total of <br />                                         364,131 distinct samples MLSMR tested primary screen a <br />                                         nominal concentration 12 &#194;&#181;M. While Z&#226;&#8364;&#8482; data indicate robustness step <br />                                         HTS campaign, did observe high degree variation signal to <br />                                         basal ratio (S/B) MetRS luminescence assay. This variability batches was <br />                                         basal signal levels approaching zero terms raw data relative light units. Hence, <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         small shift raw data, resulted fairly significant shift normalized <br />                                         S/B. <span id='am-2' about='protege:FROM' typeof='owl:Thing'>From</span> primary screen 1,456 compounds (i.e. 0.4% total library) were <br />                                         identified actives, exhibiting percent inhibition greater nominal hit cutoff of <br />                                         5.61% (average background + 3SD). Of these, 1,370 available MLSMR for <br />                                         testing confirmation counterscreen stages. <br />  <br />                                         In order confirm inhibitory activity, eliminate luminescent artifacts, eliminate <br />                                         broadly cytotoxic compounds, 1,370 samples retested triplicate different <br />                                         assays: (1) luminescence MetRS ATP depletion assay, (2) fluorescence polarization <br />                                         AMP production assay, (3) luminescence-based Jurkat cell cytotoxicity assay. The <br />                                         161 compounds active cytotoxicity assay eliminated further <br />                                         follow-up. Non-cytotoxic compounds activity ATP depletion AMP <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         formation assays prioritized CRC studies (522 compounds). To facilitate compound <br />                                         selection titration studies, 522 compounds classified 86 clusters based on <br />                                         common substructure. The compounds CRC studies selected based on <br />                                         main criteria: (1) potent compounds cluster selected ensure <br />                                         chemical diversity (2) compounds exhibited 1:1 &#194;&#177; 0.6 correlation the <br />                                         IC50s target-based assays selected proceed. Under criteria, 266 <br />  <br />  <br />  <br />  <br />                                            J Biomol Screen. Author manuscript; available PMC 2015 March 30. <br />                      Pedr&#195;&#179;-Rosa et al.                                                                                              Page 8 <br />  <br />  <br />                                         compounds selected IC50 titration, 249 compounds available from <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         MLSMR. <br />  <br />                                         The CRCs 249 compounds determined testing triplicate 10 point three <br />                                         fold serial dilution format assays. Individual CRCs offered better <br />                                         insight selectivity compounds inhibition profiles the <br />                                         different assays, facilitating triage cytotoxic compounds CID16312830 and <br />                                         luminescent artifacts CID2810697 (Figure 4 Supplement Table 2). From these <br />                                         results, compounds, selected follow-up based criteria: (1) IC50 <br />                                         10 &#194;&#181;M luminescence FP assays, (2) overlapping dose response curves in <br />                                         luminescence FP assays (maximum percent response 10 percent other) <br />                                         (3) relatively high cytotoxicity index (i.e.; Jurkat cytotoxicity assay resulting cytotoxic <br />                                         concentration (CC50) greater 10 times IC50 luminescence assay). A total 54 <br />                                         compounds met criteria subjected characterization, including <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         secondary assays structure activity relationship (SAR) studies (Supplement Table 2). <br />  <br />                        Low Throughput Secondary assays <br />                                         The 54 confirmed active compounds tested activity aminoacylation assay vs. <br />                                         T. brucei cells grown culture. The data shown Supplement Table 2 compounds <br />                                         ranked pIC50 luminescence dose-response assay 1536-well format. <br />                                         Nineteen compounds sub-micromolar IC50 values ATP depletion assay <br />                                         potent compound (CID 24793614) having IC50 44 nM. Results the <br />                                         orthogonal FP assay showed strong correlation luminescent assay an <br />                                         R2=0.895. The activity compounds determined low-throughput <br />                                         aminoacylation assay measures incorporation [3H]L-methionine tRNA. All <br />                                         compounds demonstrated enzyme inhibition secondary assay single <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         compound (CID 4656759) having somewhat discordant result (28% inhibition 10 M <br />                                         compared IC50 value 2.2 M ATP depletion assay). <br />  <br />                                         To assess parasite vs. human selectivity, 54 confirmed active compounds assayed <br />                                         human mitochondrial MetRS using aminoacylation methodology. Forty four <br />                                         compounds (81%) showed weak inhibition human mitochondrial MetRS <br />                                         (i.e.; &lt;20% 10 M). Five compounds &gt;30% inhibition human enzyme, <br />                                         indicating screen successful identifying numerous active compounds are <br />                                         specific trypanosome mammalian MetRS. Only compound (CID <br />                                         46902334) moderately potent human mitochondrial MetRS (80.3% <br />                                         inhibition 10 M, 14.3% inhibition 1 M). As noted earlier, selection criteria <br />                                         compound progression low cytotoxicity; CC50 Jurkat cells &gt;10 M. <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Supplement Table 2, shows Jurkat CC50 data; majority compounds (66.7%) no <br />                                         inhibitory activity highest dilution (83.3 M) single compound (CID <br />                                         3718852) CC50 &lt;20 M. <br />  <br />                                         Finally, 54 compounds tested growth inhibition activity (EC50) T. brucei <br />                                         cultures. Twelve compounds (22.2%) showed degree inhibition 10 M. The most <br />                                         potent compounds apparent EC50s 2 M range. These compounds screening <br />                                         hits penetrate cells well, important criterion drug discovery efforts. <br />  <br />  <br />                                            J Biomol Screen. Author manuscript; available PMC 2015 March 30. <br />                      Pedr&#195;&#179;-Rosa et al.                                                                                             Page 9 <br />  <br />  <br />                        Discussion <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Orthogonal screens successfully developed implemented identify potent small <br />                                         molecule inhibitors T. brucei MetRS, using ATP AMP levels reporters MetRS <br />                                         activity luminescence fluorescence polarization detection methods, respectively. <br />                                         Both assays technologies demonstrated robust performance Kalypsys GNF &#194;&#181;HTS <br />                                         platform. As noted, batch batch variability observed overall signal basal <br />                                         ratio (S/B) primary screen. This finding little concern given (i) data <br />                                         normalized plate plate basis, (ii) relative shift raw data minor yet <br />                                         resulted substantial shifts S/B (iii) pharmacological control behaved well, <br />                                         giving expected IC50. We maintained Z&#226;&#8364;&#8482; values &gt; 0.8 average throughput of <br />                                         approximately 15,000 compounds hour HTS campaign (Supplement <br />                                         Figure 4). The use substrate (ATP) depletion assay primary screen necessitated <br />                                         high fractional conversion ~85% substrate robust signal (Supplement Figure <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         1A). A high fractional conversion substrate associated lower sensitivity to <br />                                         identifying inhibitors.20 Notably, enzyme ATP concentration nearly <br />                                         same, 50 nM 100nM respectively, assay run conditions enabling <br />                                         limited turnover which, minimize sensitivity slow binding inhibitors. Assay <br />                                         operating conditions HTS predicated working substrate concentrations <br />                                         significantly lower Km; condition prevents identification ATP <br />                                         uncompetitive inhibitors. However, given large number confirmed hits (1,270) <br />                                         identified initial high-throughput screen, felt assay methodology more <br />                                         sufficient provide ample numbers hits forward downstream <br />                                         analysis. <br />  <br />                                         Use orthogonal FP-based counterscreen eliminate detection format-specific hits <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         (such CID2810697) eliminated nuisance compounds early process allowed a <br />                                         high confirmation rate subsequent secondary assays. In addition, cytotoxic compounds <br />                                         (i.e.; CID16312830) eliminated advancement based results Jurkat <br />                                         assay. This battery complementary assays proved effective way quickly <br />                                         identify inhibitors aaRS. Initial SAR analysis CID2973195, indole oxalamide <br />                                         shown Figure 4C, demonstrated excellent &#226;&#8364;&#339;out box&#226;&#8364;? potency primary FP <br />                                         assays relatively cytotoxicity; pIC50 values respective order are: 6.77, 6.53, &lt;4.08 <br />                                         (IC50 values 171 nM, 292 nM, &gt;83 &#194;&#181;M respectively). <br />  <br />                                         Virtually compounds selected low-throughput screening (52/54) confirmed be <br />                                         active T. brucei aminoacylation assay. This methodology measures MetRS <br />                                         enzyme reaction quantifying incorporation [3H]L-methionine tRNA. These <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         confirmatory results reassuring showed HTS assay cascade used was <br />                                         effective. <br />  <br />                                         Since human mitochondrial MetRS potential target hits, tested same <br />                                         set compounds human mitochondrial enzyme. It noted human <br />                                         cells contain cytoplasmic MetRS, homology T. brucei MetRS the <br />                                         active site relatively low (46% identical residues), homology human <br />                                         mitochondrial MetRS active site T. brucei MetRS higher (79% identical <br />  <br />  <br />                                            J Biomol Screen. Author manuscript; available PMC 2015 March 30. <br />                      Pedr&#195;&#179;-Rosa et al.                                                                                                        Page 10 <br />  <br />  <br />                                         residues). The human selectivity screen showed 81% percent compounds &#226;&#8240;&#164;20% <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         inhibition human mitochondrial MetRS, meaning developed drugs, relatively <br />                                         compounds likely cause host toxicity, inhibiting human enzyme HAT <br />                                         patients. This selectivity hit set likely enhanced earlier implementation the <br />                                         Jurkat cytotoxicity screen. It concluded differences human <br />                                         mitochondrial trypanosomal MetRS (21% amino acid sequence difference ATP- <br />                                         methionine binding sites) sufficient make selective inhibition small molecules <br />                                         possible. Structural studies compounds bound T. brucei MetRS aid <br />                                         understanding guide enhancements potency selectivity hit <br />                                         compounds. <br />  <br />                                         Finally, 12 54 HTS hit compounds (22%) inhibited growth T. brucei grown in <br />                                         culture. It assumed case growth inhibition observed inhibition of <br />                                         MetRS target, work necessary confirm hypothesis. We <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         shown previous work aminoquinolone MetRS inhibitors fact block protein <br />                                         synthesis cultured T. brucei cells.10 It likely 42 compounds inactive vs. <br />                                         cultured T. brucei poor membrane permeability reach target. Also, <br />                                         weaker MetRS inhibitors sufficiently bind target affect growth the <br />                                         concentrations tested, enter cells. <br />  <br />                                         The screening approach used campaign exhibited great robustness ability <br />                                         rapidly identify triage potent, non-cytotoxic compounds. In combination with <br />                                         secondary assays, promising selective compounds (with pIC50 values high 7.36 <br />                                         IC50 low 44 nM) identified pursued future studies further <br />                                         development novel potent molecular probes, ultimately leading drug to <br />                                         treat HAT. This orthogonal screening approach, using automated platform, proven <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         excellent tactic screen large compound libraries certainly extended to <br />                                         aaRSs. <br />  <br />  <br />                        Supplementary Material <br />                                         Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                        Acknowledgements <br />                                         This work supported National Institutes Health&#226;&#8364;&#8482;s Roadmap Initiative grants U54MH084512 <br />                                         (LPR, CE, FM, SB, JL, TB, LS, TS, PH), PO1AI067921 [Medical Structural Genomics Pathogenic Protozoa <br />                                         (MSGPP)] RO1AI084004 WGJH, RO1 AI097177 FB EF). We thank Pierre Baillargeon Lina <br />                                         DeLuca (Lead Identification, Scripps Florida) compound management MSGPP co-workers for <br />                                         contributions early stages project. <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                        References <br />                                         1. Nakama T, Nureki O, Yokoyama S. Structural basis recognition isoleucyl-adenylate and <br />                                            antibiotic, mupirocin, isoleucyl-tRNA synthetase. The Journal biological chemistry. 2001; <br />                                            276(50):47387&#226;&#8364;&#8220;47393. [PubMed: 11584022] <br />                                         2. Payne DJ, Gwynn MN, Holmes DJ, et al. Drugs bad bugs: confronting challenges of <br />                                            antibacterial discovery. Nat Rev Drug Discov. 2007; 6(1):29&#226;&#8364;&#8220;40. [PubMed: 17159923] <br />  <br />  <br />  <br />  <br />                                             J Biomol Screen. Author manuscript; available PMC 2015 March 30. <br />                      Pedr&#195;&#179;-Rosa et al.                                                                                                  Page 11 <br />  <br />  <br />                                         3. Hotez PJ, Molyneux DH, Fenwick A, et al. Control neglected tropical diseases. N Engl J Med. <br />                                            2007; 357(10):1018&#226;&#8364;&#8220;1027. [PubMed: 17804846] <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         4. Brun R, Blum J, Chappuis F, et al. Human African trypanosomiasis. Lancet. 2010; 375(9709):148&#226;&#8364;&#8220; <br />                                            159. [PubMed: 19833383] <br />                                         5. Sheppard K, Yuan J, Hohn MJ, et al. From amino acid another: tRNA-dependent amino acid <br />                                            biosynthesis. Nucleic Acids Res. 2008; 36(6):1813&#226;&#8364;&#8220;1825. [PubMed: 18252769] <br />                                         6. Hurdle JG, O&apos;Neill AJ, Chopra I. Prospects aminoacyl-tRNA synthetase inhibitors new <br />                                            antimicrobial agents. Antimicrobial agents chemotherapy. 2005; 49(12):4821&#226;&#8364;&#8220;4833. [PubMed: <br />                                            16304142] <br />                                         7. Raczniak G, Ibba M, Soll D. Genomics-based identification targets pathogenic bacteria for <br />                                            potential therapeutic diagnostic use. Toxicology. 2001; 160(1&#226;&#8364;&#8220;3):181&#226;&#8364;&#8220;189. [PubMed: <br />                                            11246138] <br />                                         8. Shibata S, Gillespie JR, Kelley AM, et al. Selective inhibitors methionyl-tRNA synthetase have <br />                                            potent activity Trypanosoma brucei Infection Mice. Antimicrobial agents and <br />                                            chemotherapy. 2011; 55(5):1982&#226;&#8364;&#8220;1989. [PubMed: 21282428] <br />                                         9. Koh CY, Kim JE, Wetzel AB, et al. Structures Trypanosoma brucei Methionyl-tRNA Synthetase <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Urea-Based Inhibitors Provide Guidance Drug Design Sleeping Sickness. PLoS <br />                                            neglected tropical diseases. 2014; 8(4):e2775. [PubMed: 24743796] <br />                                         10. Shibata S, Gillespie JR, Ranade RM, et al. Urea-based inhibitors Trypanosoma brucei <br />                                             methionyl-tRNA synthetase: selectivity vivo characterization. J Med Chem. 2012; 55(14): <br />                                             6342&#226;&#8364;&#8220;6351. [PubMed: 22720744] <br />                                         11. Tandon M, Coffen DL, Gallant P, et al. Potent selective inhibitors bacterial methionyl tRNA <br />                                             synthetase derived oxazolone-dipeptide scaffold. Bioorganic &amp; medicinal chemistry <br />                                             letters. 2004; 14(8):1909&#226;&#8364;&#8220;1911. [PubMed: 15050625] <br />                                         12. Spencer AC, Heck A, Takeuchi N, et al. Characterization human mitochondrial methionyl- <br />                                             tRNA synthetase. Biochemistry. 2004; 43(30):9743&#226;&#8364;&#8220;9754. [PubMed: 15274629] <br />                                         13. Godeau JM, Charlier J. Adenosine triphosphate consumption bacterial arginyl-transfer <br />                                             ribonucleic acid synthetases. The Biochemical journal. 1979; 179(2):407&#226;&#8364;&#8220;412. [PubMed: 384995] <br />                                         14. Wu Y, Yu K, Xu B, et al. Potent selective inhibitors Staphylococcus epidermidis <br />                                             tryptophanyl-tRNA synthetase. The Journal antimicrobial chemotherapy. 2007; 60(3):502&#226;&#8364;&#8220;509. <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                             [PubMed: 17606484] <br />                                         15. Cestari I, Kalidas S, Monnerat S, et al. multiple aminoacyl-tRNA synthetase complex that <br />                                             enhances tRNA-aminoacylation African trypanosomes. Molecular cellular biology. 2013; <br />                                             33(24):4872&#226;&#8364;&#8220;4888. [PubMed: 24126051] <br />                                         16. Madoux F, Simanski S, Chase P, et al. An ultra-high throughput cell-based screen wee1 <br />                                             degradation inhibitors. Journal biomolecular screening. 2010; 15(8):907&#226;&#8364;&#8220;917. [PubMed: <br />                                             20660794] <br />                                         17. Hirumi H, Hirumi K. Continuous cultivation Trypanosoma brucei blood stream forms a <br />                                             medium containing low concentration serum protein feeder cell layers. The Journal of <br />                                             parasitology. 1989; 75(6):985&#226;&#8364;&#8220;989. [PubMed: 2614608] <br />                                         18. Koh CY, Kim JE, Shibata S, et al. Distinct states methionyl-tRNA synthetase indicate inhibitor <br />                                             binding conformational selection. Structure. 2012; 20(10):1681&#226;&#8364;&#8220;1691. [PubMed: 22902861] <br />                                         19. Zhang JH, Chung TD, Oldenburg KR. A Simple Statistical Parameter Use Evaluation and <br />                                             Validation High Throughput Screening Assays. Journal biomolecular screening. 1999; 4(2): <br />                                             67&#226;&#8364;&#8220;73. [PubMed: 10838414] <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         20. Wu G, Yuan Y, Hodge CN. Determining appropriate substrate conversion enzymatic assays in <br />                                             high-throughput screening. Journal biomolecular screening. 2003; 8(6):694&#226;&#8364;&#8220;700. [PubMed: <br />                                             14711395] <br />  <br />  <br />  <br />  <br />                                            J Biomol Screen. Author manuscript; available PMC 2015 March 30. <br />                      Pedr&#195;&#179;-Rosa et al.                                                                                       Page 12 <br /> Author Manuscript <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 1. <br />                                         T. brucei MetRS catalyzed reaction. The primary screen assay focuses luminescence <br />                                         signal generated luciferase catalyzed reaction using residual ATP. The orthogonal <br />                                         counterscreen focuses fluorescence polarization signal generated the <br />                                         displacement labeled AMP molecule selective antibody AMP generated <br />                                         T. brucei MetRS catalyzed reaction. <br /> Author Manuscript <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                            J Biomol Screen. Author manuscript; available PMC 2015 March 30. <br />                      Pedr&#195;&#179;-Rosa et al.                                                                                       Page 13 <br /> Author Manuscript <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 2. Assay controls <br />                                         A) Compound CID 18353708 used assay pharmacological control. B) Average <br />                                         raw luminescence units controls used primary screen assess assay quality. 100% <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         inhibition control wells contain compound CID 18353708 1.5&#194;&#181;M. 50% inhibition wells <br />                                         contain CID 18353708 concentration corresponding IC50 0% inhibition control <br />                                         wells contain assay components DMSO, ATP positive control correspond wells not <br />                                         containing L-methionine, background control refers wells containing ATP. C) <br />                                         CRC results reference control CID 18353708 orthogonal assays. (&#226;&#8212;?) Luminescence <br />                                         curve (&#226;&#8212;&#8249;) fluorescence polarization assay curve. Reference compound tested 20- <br />                                         point, 2:1 serial dilution starting 15&#194;&#181;M. <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                            J Biomol Screen. Author manuscript; available PMC 2015 March 30. <br />                      Pedr&#195;&#179;-Rosa et al.                                                                                      Page 14 <br /> Author Manuscript <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 3. Primary screen scatter plot <br />                                         (&#226;&#710;&#8224;) Represent low control wells containing DMSO only, (&#226;&#8212;?) represent data wells containing <br />                                         compounds (&#226;&#8211;&#160;) represent high control wells containing compound CID18353708. The <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         dashed line represents hit cut-off. Compounds inhibition line were <br />                                         considered primary actives hits. <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                            J Biomol Screen. Author manuscript; available PMC 2015 March 30. <br />                      Pedr&#195;&#179;-Rosa et al.                                                                                         Page 15 <br /> Author Manuscript <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 4. Compound dose response curves <br />                                         (&#226;&#8212;?) Luminescence curve (&#226;&#8212;&#8249;) fluorescence polarization assay curve (&#226;&#8211;&#178;) cytotoxicity assay <br />                                         curve. A) Compound CID 2810697 flagged luminescent artifact inactivity in <br />                                         fluorescence polarization assay. B) Compound CID 16312830 flagged cytotoxic <br />                                         compound activity Jurkat cytotoxicity assay. C) Compound CID 2973195 <br />                                         identified potent hit high potency T. brucei MetRS assay modes <br />                                         low cytotoxic effect Jurkat cells. <br /> Author Manuscript <br /> Author Manuscript <br />  <br />  <br />  <br />  <br />                                            J Biomol Screen. Author manuscript; available PMC 2015 March 30. <br />                                                                        Author Manuscript                               Author Manuscript                                   Author Manuscript                                    Author Manuscript <br />  <br />  <br />                                                                                                                                                                         Table 1 <br />  <br />                                                                             Ultra-High-Throughput Screening Campaign Summary Results <br />  <br />                                                                                                                                                                                                                         Assay Statistics <br />  <br />                                                                                 Step   Screen type      Target      Number        Number            Selection       Number        *PubChem          Z&#226;&#8364;&#8482;              S/B <br />                                                                                                                                                                                                                                                    Pedr&#195;&#179;-Rosa et al. <br />  <br />  <br />  <br />  <br />                                                                                                                     compounds        replicates/          Criteria          selected          AID <br />                                                                                                                       tested       concentrations                         compounds <br />                                                                                                                                                                              (Hits) <br />                                                                                  1       Primary        MetRS         364,131            1/1            %inh&gt; 5.61%          1,456          624268       0.90 &#194;&#177; 0.03    16.33 &#194;&#177; 13.13 <br />                                                                                 2a     Confirmation     MetRS          1,370             3/1            %inh&gt; 5.61%     1,270 (522, 266)    624412       0.92 &#194;&#177; 0.02     7.10 &#194;&#177; 0.20 <br />                                                                                 2b     Cytotoxicity   Cell growth      1,370             3/1         %inh&gt; 14.97% 161       624413         0.83 <br /> </body></html>