<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Identification of 315 genes essential for early <br /> zebrafish development <br />  <br /> We completed a large insertional mutagenesis screen in zebrafish                 reciprocal best BLASTP hits, but also if no other gene in either <br /> <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> identify genes essential for embryonic and early larval devel-                species&#226;&#8364;&#8482; genome finds the gene as its top hit. A worm or yeast <br /> opment. We isolated 525 mutants, representing lesions in &#226;&#172;&#8225;390                    gene was defined as an &#226;&#8364;&#732;&#226;&#8364;&#732;ancestor&#226;&#8364;&#8482;&#226;&#8364;&#8482; of a human&#229;&#8230;&#190;fish gene if <br /> different genes, and we cloned the majority of these. Here we                    several human genes found the worm or yeast gene as their top <br /> describe 315 mutants and the corresponding genes. Our data                       hit, but no other worm or yeast gene found any of those human <br /> suggest that there are roughly 1,400 embryonic-essential genes in                genes as their top hit. <br /> the fish. Thus, we have mutations in &#226;&#172;&#8225;25% of these genes and have <br /> cloned &#226;&#172;&#8225;22% of them. Re-screens of our collection to identify                    Results <br /> mutants with specific developmental defects suggest that &#226;&#172;&#8225;50                     Identification of 25% of the Genes Essential for Early Zebrafish <br /> genes are essential for the development of some individual organs                Development. We isolated 525 insertional mutants that have <br /> or cell types. Seventy-two percent of the embryonic-essential fish               visible phenotypes by 5 dpf, a time when the embryo has <br /> genes have homologues in yeast, 93% have homologues in inver-                    developed <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> a fertilized egg to a free swimming larva that has <br /> tebrates (fly or worm), and 99% have homologues in human. Yeast                  begun to feed. Most of these mutations result in lethality, and <br /> and worm orthologues of genes that are essential for early ze-                   many mutants die by 5 dpf. Almost all mutants that have not died <br /> brafish development have a strong tendency to be essential for                   by 5 days fail to inflate their swim bladder, a <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-82' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span>phenotype</span> <br /> viability in yeast and for embryonic development in the worm.                    associated with certain death by 2 weeks of age. Thus, we refer <br /> Thus, the trait of being a genetically essential gene is conserved in            to the mutants and mutations as &#226;&#8364;&#732;&#226;&#8364;&#732;embryonic lethal(s)&#226;&#8364;&#8482;&#226;&#8364;&#8482; and the <br /> evolution. This mutant collection should be a valuable resource for              mutated genes as &#226;&#8364;&#732;&#226;&#8364;&#732;embryonic-essential&#226;&#8364;&#8482;&#226;&#8364;&#8482; genes. <br /> diverse studies of cell and developmental biology.                                  The insertional mutagenesis procedure and the methodology <br />                                                                                  for identifying mutagenic inserts and cloning their flanking <br />                                                                                  DNA have been described (1&#226;&#8364;&#8220;3). A summary of the numbers of <br /> T   o identify a significant fraction of the genes essential for early <br />     vertebrate development, we developed a method of inser- <br /> tional mutagenesis for the zebrafish using mouse retroviral <br />                                                                                  mutants isolated and flanking sequences and genes cloned is <br />                                                                                  shown in Table 1. The 486 mutants for which we have obtained <br /> vectors (1, 2) and applied the method in a large-scale screen. We                DNA sequence at the site of the mutagenic insertion probably <br /> identified mutants by visual inspection of embryos at 1, 2, and 5                represent 362 different loci; we have identified the mutated gene <br /> days postfertilization (dpf), by which time they have developed                  for 315 of these, 86 of which we have reported (1, 2). As discussed <br /> into free-swimming and feeding larvae. Mutants that result in a                  next, our findings suggest that these 362 loci represent &#226;&#172;&#8225;25% of <br /> visible defect by 5 dpf are almost invariably lethal. Here, we                   the genes whose <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span>mutation</span> leads to an embryonic lethal pheno- <br /> describe 315 mutants and their genes and present an analysis of                  type. Thus, there are only &#226;&#172;&#8225;1,400 such genes. The following <br /> the evolutionary conservation of these genes. The results argue                  evidence supports this conclusion. <br /> that the mutant collection contains mutations in at least 25% of                    Our collection includes mutations in 5 of 20 (25%) tRNA <br /> the genes essential for the development of many different                        synthase genes and 26 of the 79 (33%) ribosomal protein genes <br /> embryonic organs and structures. Re-screens of the collection in                 in the zebrafish genome. We also have mutations in 23 of 97 <br /> our lab to identify mutations in specific developmental processes                genes (24%) for which a chemically induced mutant has also <br /> support this conclusion. This collection should be a valuable                    been identified and the mutated gene cloned by positional or <br /> resource for diverse studies of cell and developmental biology in                candidate gene cloning (as of February 2004). Most of the latter <br /> this vertebrate.                                                                 genes encode transcription factors, receptors, and ligands. These <br />                                                                                  data argue that we have screened &#226;&#172;&#8225;25% of the genes in the fish <br /> Materials and Methods                                                            genome, whether extrapolated from housekeeping-type genes or <br /> Mutagenesis and Gene Cloning. Retroviral-mediated insertional <br />                                                                                  genes with more specific developmental functions. The data in <br />                                                                                  Table 2 also show that viral integrations do not occur prefer- <br /> mutagenesis and the cloning of the mutated genes were carried <br />                                                                                  entially into housekeeping-type genes because the allele fre- <br /> out as described (1&#226;&#8364;&#8220;3). <br />                                                                                  quency for housekeeping genes is no greater than that for other <br />                                                                                  types of genes. <br /> Comparative Genomic Analysis. The amino acid sequence of each <br />                                                                                     In theory, a second way to estimate the number of embryonic- <br /> fish gene was compared by BLASTP (4) to the reference genomes <br />                                                                                  essential genes from these data is to apply the Poisson distribu- <br /> of Saccharomyces cerevisiae, Schizosaccharomyces pombe, Cae- <br /> norhabditis elegans, Drosophila melanogaster, and Homo sapiens, <br /> as well as the nonredundant database of all organisms to find                    Abbreviations: SCE, single-celled eukaryotes; IFT, intraflagellar transport; RNAi, RNA in- <br /> genes absent in the two yeast species but present in other                       terference; dpf, days postfertilization. <br /> unicellular eukaryotes. Comparative orthologous group (COG)                      Data deposition: The sequences reported in this paper have been deposited in the GenBank <br /> analysis (5) to determine whether homologues were 1:1 ortho-                     database (accession numbers can be found in Table 3, which is published as supporting <br /> logues or had other orthology relationships was done by itera-                   information on the PNAS web site). <br />  <br /> tively blasting the top hits from each organism against the                      See accompanying Biography on page 12789. <br /> genomes of the others. We consider two genes from different                      *To whom correspondence should be addressed. E-mail: nhopkins@mit.edu. <br /> species to be 1:1 orthologues if they are not only each other&#226;&#8364;&#8482;s                  &#194;&#169; 2004 by The National Academy of Sciences of the USA <br />  <br />  <br />  <br /> 12792&#226;&#8364;&#8220;12797 &#229;&#8230;&#169; PNAS &#229;&#8230;&#169; August 31, 2004 &#229;&#8230;&#169; vol. 101 &#229;&#8230;&#169; no. 35                                                            www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0403929101 <br />                                                                                                                                                      INAUGURAL ARTICLE <br /> Table 1. Insertional mutants recovered                                        (SCEs) (e.g., Giardia, Plasmodium, Trypanosoma, Chlamydomo- <br />                                                    Mutants            Loci    nas), worm, fly, and human genomes. Analysis of the completed <br />                                                                               human and mouse genomes has already indicated that 50% of <br /> Insertional mutants                                  525             &#226;&#172;&#8225;390     their genes are present in SCEs and &#226;&#172;&#8225;80% in flies and&#229;&#8230;&#190;or worms <br /> Mutants with cloned junction DNA                     486              362     (10, 11), (with homology defined as a BLASTP E value of &#226;&#172;&#65533;10&#226;&#171;&#186;5), <br /> Mutants with mutated gene identified                 438              315     and our analysis of 50 fish genes selected at random from the July <br />                                                                               2003 annotation of the zebrafish genome found a similar pro- <br />                                                                               portion of conserved genes (data not shown). Applying the same <br /> tion to the numbers of single and multiple hits in Table 2 to                 homology <span id='am-2' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> to our list of 315 embryonic-essential <br /> calculate the number of loci that have not yet been mutated and               zebrafish genes, however, we found that a higher fraction, 74% <br /> hence the total number of loci that can give rise to an embryonic-            of them, have homologues in yeast or other SCEs, and 93% are <br /> visible phenotype. However, as is almost always seen in mu-                   present in fly and&#229;&#8230;&#190;or worm. By using a more conservative cutoff <br /> tagenesis screens, the data do not fit the Poisson distribution               requiring homology over 80% of the <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> of the protein and <br /> precisely, making this approach unreliable. Attempting to apply               E &#226;&#172;&#65533; 10&#226;&#171;&#186;20, 64% of the fish genes have homologues in yeast or <br /> a Poisson distribution under these circumstances usually results              some other SCE whereas 91% are present in flies or worms. <br /> in an underestimate of the number of mutable loci (6, 7). For                 Ninety-nine percent of the genes (all but 2 of the 315) have <br /> example, by using the number of loci hit twice in our study (62)              homologues in mammals. Thus, as found in other species (12, <br /> and keeping either the number of loci hit (362) or the number                 13), the fish genes with essential functions are more likely than <br /> of mutants obtained (486) fixed, solving the Poisson equation                 randomly selected genes to be conserved across species. The <br /> predicts that the total number of mutable loci is either 1,035 or             conservation of the 315 embryonic-essential fish genes in yeast <br /> 1,310 mutable loci. This finding is somewhat lower than (but not              and other SCEs, in worm, in fly, and in human is represented in <br /> inconsistent with) the estimate of 1,400 genes obtained above.                Fig. 3 by using the more liberal <span id='am-3' about='Thesaurus:DEFINITION' typeof='owl:Thing'>definition</span> of homology. The <br />    The identities of the 315 mutated genes we have identified to              identity of the human, fly, worm, and yeast homologues of each <br /> date are provided in Fig. 1 (additional details about these genes             essential fish gene can be found in Table 3. <br />  <br />  <br />  <br />  <br />                                                                                                                                                     GENETICS <br /> are described in Table 3). A preliminary phenotypic description <br /> of the corresponding 315 mutants is provided in Table 3 and                   Mutant Phenotypes and the Evolutionary Conservation of Genes <br /> images are available at http:&#229;&#8230;&#190;&#229;&#8230;&#190;web.mit.edu&#229;&#8230;&#190;ccr&#229;&#8230;&#190;pnas&#227;&#8250;&#173;zebrafish&#227;&#8250;&#173;                Required for Specific Developmental Processes. About 30% of <br /> mutant&#227;&#8250;&#173;images&#229;&#8230;&#190;index.html. More detailed phenotypic descrip-                   embryonic zebrafish mutants identified in gross morphological <br /> tions of the defects in particular organs and structures will come            screens have developmentally &#226;&#8364;&#732;&#226;&#8364;&#732;specific&#226;&#8364;&#8482;&#226;&#8364;&#8482; and unique phenotypes <br /> from careful re-screens of the collection (see below).                        whereas 70% display relatively &#226;&#8364;&#732;&#226;&#8364;&#732;nonspecific&#226;&#8364;&#8482;&#226;&#8364;&#8482; or common syn- <br />    If we take 1,400 as the number of zygotic genes whose                      dromes (7, 14). The latter more frequently result from mutations <br /> mutation can lead to an embryonic visible&#229;&#8230;&#190;<span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>lethal phenotype</span> in                 in cell-essential genes. Many mutants can be placed into either <br /> the zebrafish, the 315 genes listed here represent 22% of the                 of the two broad phenotypic categories by superficial visual <br /> total. Because mutated genes were cloned without regard to                    inspection. However, to identify mutants with phenotypic de- <br /> phenotype, these genes should be representative of the entire set             fects in specific organs or processes with certainty requires <br /> of protein-coding genes that are genetically essential for early              considerable effort. To accomplish this goal, we are re-screening <br /> zebrafish development.                                                        the insertional mutant collection, a process called &#226;&#8364;&#732;&#226;&#8364;&#732;shelf- <br />    Fig. 2 provides a summary of the types of proteins encoded by              screening.&#226;&#8364;&#8482;&#226;&#8364;&#8482; Summaries of three such screens from our lab are <br /> the 315 embryonic essential genes based on their biochemical                  shown in Fig. 3 Right. <br /> function. Many of the genes are probably essential for cell                      Black boxes in the three columns in Fig. 3 Right identify genes <br /> viability whereas others are likely to be required for more                   specifically required (i) to prevent cystic kidney, probably be- <br /> specific developmental processes. Mutations in cell-essential                 cause these genes are required for the normal development of <br /> genes can survive for 1 to at least several days due to maternal              kidney epithelial tubes (15), (ii) to form cartilage that appears <br /> supplies in the egg (8). Mutation of some cell-essential genes may            normal after staining with Alcian blue (R.M.N., A.A., and N.H., <br /> not have been detected in our screen because maternal supplies                unpublished observations) or (iii) for melanocyte pigmentation <br /> of some genes are sufficient to sustain the embryo beyond 5 dpf,              (E. Maldonado, A.A., and N.H., unpublished observations). <br /> e.g., Dicer1 (9). About 20% of all of the genes we identified                 Twelve genes when mutated resulted in kidney cysts, 8 in <br /> encode proteins of unknown biochemical function.                              abnormal cartilage condensation as revealed by appearance <br />                                                                               after Alcian staining, and 11 affect melanocyte pigmentation. <br /> Evolutionary Conservation of Embryonic-Essential Zebrafish Genes.             Because we have cloned &#226;&#172;&#8225;22% of the genes essential for early <br /> To better understand the genetic basis of animal development                  fish development, these results predict that mutations in &#226;&#172;&#8225;55 <br /> and its evolution, we asked whether the embryonic-essential fish              genes can give rise to cystic kidney, 36 to abnormal cartilage, and <br /> genes have homologues in yeast, other single-celled eukaryotes                50 to defects in melanocyte pigmentation in the zebrafish. As <br />                                                                               discussed later, many genes identified in each screen are com- <br />                                                                               ponents of a common pathway, with different pathways or <br /> Table 2. Allele frequencies                                                   processes emerging for each screen. <br />                                                                                  Fig. 3 shows the conservation of the genes identified in each <br /> Loci with one allele                                                275 <br />                                                                               of the three screens in the genomes of yeast, other SCEs, <br /> Loci with two alleles                                                62 <br />                                                                               invertebrates (fly or worm), and human. Of the 12 genes that can <br /> Loci with three allele                                               17 <br />                                                                               give rise to cystic kidney when mutated, 6 are shared with SCEs <br /> Loci with four alleles                                                6 <br />                                                                               but not with yeast. Three of these are homologues of genes <br /> Loci with five allele                                                 1 <br />                                                                               identified in Chlamydomonas that encode intraflagellar trans- <br /> Loci with seven alleles                                               1 <br />                                                                               port (IFT) proteins required for flagellum formation (16) (see <br /> Average allele frequency                                              1.34 <br />                                                                               Discussion). <br /> Average allele freq, RP and tRS                                       1.10 <br />                                                                                  Formation of cartilaginous structures requires the deposition <br /> Average allele freq, ENU cloned                                       1.95 <br />                                                                               of proteoglycans in the extracellular matrix (17). Among the <br />    RP, ribosomal protein genes; tRS, tRNA synthase genes; ENU cloned, genes   eight cartilage mutants identified, four have lesions in genes <br /> identified as the cause of ENU-induced mutants.                               required for proteoglycan synthesis, predicting that &#226;&#172;&#8225;20 such <br />  <br /> Amsterdam et al.                                                                               PNAS &#229;&#8230;&#169; August 31, 2004 &#229;&#8230;&#169; vol. 101 &#229;&#8230;&#169; no. 35 &#229;&#8230;&#169; 12793 <br />  Fig. 1. Genes essential for zebrafish embryonic development identified by insertional mutagenesis. Genes are listed by mutant number and sorted by <br /> evolutionary conservation and gene function. Phenotypic descriptions are available in Table 3, and images are available at http:&#229;&#8230;&#190;&#229;&#8230;&#190;web.mit.edu&#229;&#8230;&#190;ccr&#229;&#8230;&#190; <br /> pnas&#227;&#8250;&#173;zebrafish&#227;&#8250;&#173;mutant&#227;&#8250;&#173;images&#229;&#8230;&#190;index.html. <br />  <br />  <br /> 12794 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0403929101                                                                               Amsterdam et al. <br />                                                                                                                                                                   INAUGURAL ARTICLE <br /> Fig. 2. Types of genes whose mutation in zebrafish leads to an embryonic <br /> visible phenotype. The genes are assigned to the same categories as in Fig. 1 <br /> although some categories have been combined. <br />  <br />  <br />  <br /> genes would be found in a saturation screen in the fish. Three of <br /> these four genes are animal specific. In C. elegans, mutations in <br /> homologues of these four genes led to a &#226;&#8364;&#732;&#226;&#8364;&#732;squashed vulva&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> phenotype (18, 19). Thus, whereas the genes are presumably <br />  <br />  <br />  <br />  <br />                                                                                                                                                                  GENETICS <br /> required for proteoglycan synthesis in both worm and fish, they <br /> are involved in the formation and structural integrity of very <br /> different body parts in the two organisms. <br />    Extending our earlier observations (2), 9 of 11 genes we <br /> identified as required for normal melanocyte pigmentation in <br /> fish encode v-ATPase subunits or associated proteins or proteins <br /> otherwise involved in intracellular vesicles. Whereas the main <br /> v-ATPase subunits are found in yeast, zebrafish required several <br /> animal-specific v-ATPase-associated proteins in addition. <br />  <br /> The Trait of Being a Genetically Essential Gene Is Conserved in <br /> Evolution. As Fig. 3 above reveals, and has been observed in other <br /> species (12, 13), genes that are essential in zebrafish are highly <br /> conserved in evolution. We next asked whether genes that are <br /> genetically essential in zebrafish are also genetically essential in <br /> other species. It is convenient to ask this question of genes that <br /> have counterparts in yeast and worm because a large fraction of <br /> the annotated genes in these species have been deleted [yeast <br /> (12, 20)] or knocked down by using <span id='am-240' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-241' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-315' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> (RNAi) <br /> [worm (13, 21)]. To make functional comparisons of genes across <br /> these species, however, it is necessary first to identify the yeast <br /> and worm genes that are not merely homologues of the essential <br /> fish genes, but that have orthologous relationships to them, <br /> because orthologues are more likely to perform the same <br /> function in their respective species. We determined which ze- <br /> brafish genes have clear orthologous relationships with their <br /> homologues in S. cerevisiae and C. elegans by comparative <br /> orthologous group (COG) analysis (5) (see Materials and Meth- <br /> ods). Because the zebrafish genome is not yet fully sequenced <br /> and annotated, it cannot be used for this purpose, so we used the <br /> human genome as a &#226;&#8364;&#732;&#226;&#8364;&#732;surrogate vertebrate&#226;&#8364;&#8482;&#226;&#8364;&#8482; genome. We con-                      Fig. 3. Evolutionary conservation of essential zebrafish genes. Horizonal <br /> sidered both genes that were 1:1 orthologues and genes for which                lines on the Left represent 315 different genes. The genes are listed in the <br /> a single yeast or worm gene is an &#226;&#8364;&#732;&#226;&#8364;&#732;ancestor&#226;&#8364;&#8482;&#226;&#8364;&#8482; of two or more                   same order as in Fig. 1. In the first four colored columns, the presence of a <br /> human paralogues. The results of the analysis are shown in Fig.                 colored box indicates the presence of one or more homologous genes (BLASTP <br /> 4 for yeast (Left) and for worm (Right).                                        E value of &#226;&#172;&#65533;10&#226;&#171;&#186;5) in yeast (either S. cerevisiae or S. pombe), C. elegans, D. <br />    The left-most bars in Fig. 4 Left and Right represent the list of            melanogaster, or H. sapiens. Green boxes indicate genes without homologues <br /> 315 essential fish genes, the red columns next to them show the                 in yeast but with homologues in other SCEs such as Giardia, Plasmodium, <br /> number of these genes that have homologues in yeast or worm,                    Trypanosoma, and&#229;&#8230;&#190;or Chlamydomonas. Thus, yellow boxes represent genes <br />                                                                                 conserved through yeast, green are those found in SCEs other than yeast, red <br /> and the blue columns show the fraction of homologues that have                  are those found in invertebrates but not SCEs, and blue are vertebrate- <br /> an orthologous relationship to the vertebrate genes. Having                     specific. The last two genes (with no colored boxes) seem to be fish-specific. <br /> identified the yeast and worm orthologues, we then asked which                  The black boxes in the last three columns indicate genes whose mutation leads <br /> of these genes are essential in their respective species. The results           to one of three phenotypes: cystic kidney, chondrogenesis defects, or reduc- <br /> are shown by the dark green columns in Fig. 4.                                  tion or lack of melanocyte pigmentation. <br />  <br />  <br /> Amsterdam et al.                                                                                    PNAS &#229;&#8230;&#169; August 31, 2004 &#229;&#8230;&#169; vol. 101 &#229;&#8230;&#169; no. 35 &#229;&#8230;&#169; 12795 <br />                                                                                       In C. elegans, RNAi analysis of protein-coding genes reveals <br />                                                                                   that only 7% are required for embryo viability, an additional <br />                                                                                   1.5% for other developmental and&#229;&#8230;&#190;or physiological processes, <br />                                                                                   and another 1.5% for wild-type growth (21). Yet among the <br />                                                                                   worm genes that are orthologous to the vertebrate genes we <br />                                                                                   identified and for which RNAi data have been reported, 72% are <br />                                                                                   required for embryonic viability and a further 6% are associated <br />                                                                                   with postembryonic developmental phenotypes, a total of 78%. <br />                                                                                   This is an enrichment of nearly 10-fold over the worm genome <br />                                                                                   at large. In the large RNAi screens in C. elegans that have been <br />                                                                                   published to date, it was estimated that only 78% of embryonic- <br />                                                                                   essential genes were detected, largely due to inefficiencies in the <br />                                                                                   RNAi technology (21). Correcting for this failure rate, we <br />                                                                                   conclude that nearly all of the worm orthologues of essential fish <br />                                                                                   genes might in fact be essential for embryonic development (see <br />                                                                                   lighter colored extensions on the green columns in Fig. 4 Right). <br />                                                                                   In summary, this analysis reveals that genetically essential genes <br />                                                                                   have a strong tendency to retain this special status through <br />                                                                                   evolution from yeast to vertebrates. <br />  <br />                                                                                   Discussion <br />                                                                                   We have described the completion of a large genetic screen in <br />                                                                                   zebrafish, the isolation of insertional mutations in &#226;&#172;&#8225;25% of the <br />                                                                                   embryonic-essential genes of the fish, and the molecular cloning <br />                                                                                   of &#226;&#172;&#8225;22% of all such genes. This collection of mutants will be a <br />                                                                                   valuable resource for the study of many cellular and develop- <br />                                                                                   mental processes in a vertebrate. Many of the genes we identified <br />                                                                                   are probably required for cell viability, others for more specific <br />                                                                                   developmental processes including patterning, differentiation or <br />                                                                                   physiology. Twenty percent of the genes encode proteins that <br />                                                                                   have no known or sufficiently clearly identifiable biochemical <br />                                                                                   function. About three quarters of the genes we cloned have <br />                                                                                   homologues in yeast or other single-celled organisms, and &#226;&#172;&#8225;25% <br />                                                                                   are animal-specific with 7% overall being vertebrate-specific. <br />                                                                                      Our results imply that there are only &#226;&#172;&#8225;1,400 genes that when <br />                                                                                   mutated result in a visible, usually lethal phenotype in the <br />                                                                                   zebrafish embryo and 5-day-old larva. This number is fewer than <br />                                                                                   the 2,400 such genes proposed by Haffter et al. (7) from data <br />                                                                                   obtained in a large chemical mutagenesis screen by using ENU <br /> Fig. 4.    Essentialness of genes is evolutionarily conserved. Shown is an        as the mutagen. We do not think the discrepancy is due to the <br /> analysis of S. cerevisiae (Left) and C. elegans (Right) genes that are homolo-    inability of retroviral vectors to target genes than can be mutated <br /> gous to the essential fish genes. In each case, the leftmost columns represent    by ENU because genes mutated by ENU and cloned by posi- <br /> 315 essential fish genes, the red columns show how many of these have <br />                                                                                   tional or candidate gene cloning were mutated at the same <br /> homologues in yeast (214) or worm (272), and the blue columns show which <br /> have a 1:1 orthologue or &#226;&#8364;&#732;&#226;&#8364;&#732;ancestor&#226;&#8364;&#8482;&#226;&#8364;&#8482; gene in yeast (176) or worm (235). The <br />                                                                                   efficiency as genes encoding ribosomal proteins or tRNA syn- <br /> dark green columns represent the number of these yeast or worm orthologues        thetases in our screen. Rather, it seems likely that the discrep- <br /> that are essential in their respective species, 135 of the 176 yeast genes, and   ancy reflects inaccuracies in both calculations. The failure to <br /> 155 of the 235 worm genes. The pea green columns show the number that             achieve saturation in either screen and the fact that the data for <br /> would be predicted to be essential at random, 33 of the 176 yeast genes, and      single and multiple hits do not fit the Poisson distribution <br /> 15 of the 235 worm genes. Thus, the difference between the dark green and         precisely in either screen make highly accurate calculations <br /> pea green columns is the enrichment. In the case of the worm, the pale green      impossible. <br /> extensions to the two green columns represent projections of how many of             The phenotypic descriptions of most of our mutants remain <br /> the orthologues would be found to be essential if 100% of the worm genes <br />                                                                                   preliminary, and re-screens of the collection are needed to <br /> had been successfully knocked down by RNAi; this estimate prorates both for <br /> the reported failure rate of RNAi and the number of genes for which no RNAi <br />                                                                                   identify the specific defects in most mutants. The first three such <br /> data have been reported. This calculation estimates that 216 of the 235 worm      re-screens to be completed, which we summarized here, are <br /> genes are likely to be essential whereas only 22 would be expected to be at       revealing in this respect. The screen for cystic kidney in partic- <br /> random. Note that the percentage of worm orthologues that are essential           ular supports our conclusion that our mutant collection contains <br /> that is stated in the text does not include the genes for which no RNAi data      25% of the genes essential for diverse developmental processes <br /> have been reported.                                                               in the embryo. Of 12 genes identified in the kidney cyst screen, <br />                                                                                   7 seemed to be novel when first cloned, whereas another is <br />                                                                                   PKD2, a gene known to be mutated in human polycystic kidney <br />    In yeast, only 19% of protein-coding genes are essential for cell              disease (22). In humans, cystic kidney disease results from a <br /> viability under optimal growth conditions, despite the fact that                  failure of epithelial cells in kidney tubes and ducts to differen- <br /> most &#226;&#8364;&#732;&#226;&#8364;&#732;nonessential&#226;&#8364;&#8482;&#226;&#8364;&#8482; yeast genes seem to be single-copy (12). In                 tiate properly and to cease dividing. This defect can result from <br /> contrast, among the yeast genes that are 1:1 orthologues or                       defects in primary cilia located on the epithelial cells of kidney <br /> ancestors of the vertebrate genes in our study, 77% are essential                 tubules and ducts. When we gained access to the mostly unpub- <br /> in yeast. This is a 4-fold enrichment relative to the yeast genome                lished sequences of 13 genes that encode IFT proteins in <br /> at large (compare the number observed in dark green with the                      Chlamydomonas, and which are required for flagellum forma- <br /> number predicted if random in pea green).                                         tion or function in that species, we found that 3 of our 7 &#226;&#8364;&#732;&#226;&#8364;&#732;novel&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br />  <br /> 12796 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0403929101                                                                                   Amsterdam et al. <br />                                                                                                                                                                                   INAUGURAL ARTICLE <br /> genes were in fact fish homologues of 3 of the 13 IFT genes. It                          genetically essential genes have a unique status in biological <br /> is possible that others of the &#226;&#8364;&#732;&#226;&#8364;&#732;novel&#226;&#8364;&#8482;&#226;&#8364;&#8482; genes identified in the fish                    processes. Consistent with this possibility, we found that the <br /> kidney cyst screen will prove to be yet unidentified IFT genes, or                       yeast or worm orthologues of genes that are essential in fish have <br /> other genes involved in a pathway linking cilia to cell differen-                        a high probability of also being essential in these species. Thus <br /> tiation and cell division in vertebrates. The finding that we have                       genes that can be detected in genetic screens, and in particular <br /> mutations in 3 of 13 IFT genes whose sequences are available is                          those that are essential for early viability, seem to have retained <br /> consistent with the notion that our mutant collection includes                           this special status through evolution. The implications of this <br /> 25% of the genes essential for processes involved in embryonic                           observation are not known, but we suggest that these genes may <br /> development. The results of the kidney cyst screen also suggest                          be all or most of the genes that are absolutely required for many <br /> that genetic screens in vertebrate animals can reveal many of the                        biological processes whereas most other genes may serve to assist <br /> genes in a pathway and argue that redundancy may be no more                              these critical genes in making biological processes more robust. <br /> of a problem in genetic screens in vertebrates than it has been in                       Evolution may have required that the number of genetically <br /> genetic screens in invertebrate animals.                                                 essential genes be small and that they remain the same genes. <br />    The data from the three shelf screens described here, along                              By identifying about a quarter of the embryonic lethal genes <br /> with the data showing the conservation of the fish genes in the                          in the zebrafish, our studies provide strong evidence that genetic <br /> genomes of single-celled eukaryotes and invertebrate animals,                            screens in vertebrate animals, as in invertebrates, can reveal <br /> show how many of the same genes used in SCEs and inverte-                                molecular pathways of development. Whereas redundancy may <br /> brates are used for different biological tasks in vertebrate                             hide some genes from this analysis, the fact that the genes that <br /> animals although their cellular functions are similar in the                             we do find for particular phenotypes can define a biochemical <br /> different species. Thus, IFT genes used by Chlamydomonas to                              complex or pathway suggests that forward genetic screens can <br /> make motile flagella are used to make cilia with different sensory                       indeed identify many if not most of the genes that are key players <br /> roles in different organisms, for example, on chemosensory                               in specifying complex developmental processes in vertebrates. <br /> neurons sensing salt concentrations in the worm (23) and on <br /> kidney epithelial cells in vertebrates (24). Similarly, the same                          <br /> proteoglycan synthesis enzymes that the worm uses to make a                               <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                  GENETICS <br /> properly shaped vulva (18, 19) are used in the fish to make a                            <br /> vertebrate-specific structure, cartilage. And the production of                          <br />                                                                                         <br /> melanin pigment in the fish, which takes place in acidic subcel- <br />                                                                                           <br /> lular compartments (25), requires the same genes to acidify these                         <br /> compartments that yeast cells use to acidify vacuoles (26).                              <br />    The fact that there is such a small number of embryonic-                               <br /> essential genes and that they include genes that comprise                                 <br /> coherent genetic pathways of development suggests that the                               <br />  <br /> </body></html>