<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>&#194;&#169; 2001 Oxford University Press                                                  Nucleic Acids Research, 2001, Vol. 29, No. 17 3621&#226;&#8364;&#8220;3630 <br />  <br />  <br />  <br /> ERB1, yeast homolog mammalian Bop1, an <br /> essential gene required maturation 25S and <br /> 5.8S ribosomal RNAs <br /> Dimitri G. Pestov, Michael G. Stockelman, Zaklina Strezoska Lester F. Lau* <br /> Department Molecular Genetics, M/C 669, University Illinois Chicago College Medicine, 900 South Ashland <br /> Avenue, Chicago, IL 60607-7170, USA <br />  <br /> Received April 27, 2001; Revised Accepted July 11, 2001 <br />  <br />  <br /> <span id='am-1' about='protege:abstract' typeof='owl:Thing'>ABSTRACT</span>                                                                    ribosome, flanking regions designated 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; external tran- <br />                                                                             scribed sequences (5&#226;&#8364;&#178;-ETS 3&#226;&#8364;&#178;-ETS) intervening <br /> We recently shown mammalian nucleolar <br />                                                                             sequences termed internal transcribed spacers 1 2 (ITS1 <br /> protein Bop1 involved synthesis 28S                        ITS2). The primary transcript extensively modified by <br /> 5.8S ribosomal RNAs (rRNAs) large ribosome                              methylation pseudouridylation processed series <br /> subunits mouse cells. Here investigated                          endo- exonucleolytic reactions. This process ultimately <br /> the functions Saccharomyces cerevisiae                               generates mature 18S, 5.8S 25S rRNAs appears <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br /> homolog Bop1, Erb1p, encoded previously                           occur parallel assembly ribosomal particles. A large <br /> uncharacterized open reading frame YMR049C. Gene                            number trans-acting factors required ribosome <br /> disruption showed ERB1 essential                                biogenesis identified date, including than <br /> viability. Depletion Erb1p resulted loss 25S                     150 small nucleolar RNAs (snoRNAs) multiple non- <br /> and 5.8S rRNAs synthesis, causing                                ribosomal proteins (for reviews 1,2). <br /> a moderate reduction complete block                               Much current knowledge rRNA processing comes <br /> 18S rRNA formation. Processing analysis showed                              studies Saccharomyces cerevisiae. Ribosome synthesis <br /> that Erb1p required synthesis 7S pre-rRNA                         organisms characterized largely the <br /> and mature 25S rRNA 27SB pre-rRNA. In                                  lack comparably amenable systems genetic manipula- <br /> Erb1p-depleted cells products 27SB                                 tion. However, insights obtained studies yeast <br /> processing largely absent 27SB pre-rRNA                          ribosome biogenesis relevance eukaryotes. <br />                                                                             rRNA structure processing generally similar a <br /> under-accumulated, apparently degradation. <br />                                                                             wide range eukaryotes (3) recent sequencing efforts <br /> In addition, depletion Erb1p caused delayed <br />                                                                             revealed known factors yeast ribosome <br /> processing 35S pre-rRNA. These findings                              biogenesis counterparts eukaryotic <br /> demonstrate Erb1p, like mammalian counter-                         kingdom. <br /> part Bop1, required formation rRNA compo-                            In order characterize rRNA processing mammalian <br /> nents large ribosome particles. The similarities                     cells, employed strategy inducible expression of <br /> in processing defects caused functional disrup-                          dominant negative mutants inactivate function critical <br /> tion Erb1p Bop1 suggest late steps                           proteins involved ribosome biogenesis. Using strategy, <br /> maturation large ribosome subunit rRNAs                              recently identified characterized novel mouse <br /> employ mechanisms evolutionarily                                   nucleolar protein, Bop1, involved rRNA processing <br /> conserved eukaryotes.                                            ribosome assembly (4). Bop1&#226;&#710;&#8224;, N-terminally truncated <br />                                                                             form Bop1, acts dominant negative inhibitor Bop1 <br />                                                                             function mouse cells, causing block synthesis mature <br /> INTRODUCTION                                                                28S 5.8S rRNAs deficiency 60S ribosomal <br /> In eukaryotes, ribosome biogenesis highly complex                      subunit assembly (4). Analysis pre-rRNA processing <br /> process involves assembly ribosomal RNAs                   revealed Bop1&#226;&#710;&#8224;-expressing cells reduced <br /> (rRNAs) 80 ribosomal proteins 60S 40S                    conversion 36S precursor 32S pre-rRNA, in <br /> ribosome subunits specialized nuclear compartment,                 turn blocked processed 28S 12S/5.8S <br /> the nucleolus. Our current understanding ribosome                        rRNAs, instead degraded. Expression Bop1&#226;&#710;&#8224; thus <br /> synthesis largely organized observed rRNA                         completely blocks formation mature 28S 5.8S rRNAs, <br /> processing steps. In yeast, rRNAs (18S, 5.8S 25S)                 minor effect synthesis 18S rRNA <br /> are derived single 35S precursor rRNA (pre-rRNA)                     production 40S subunits (4). Bop1 contains WD <br /> transcribed RNA polymerase I. The primary transcript                     repeats, structural motif involved intra- intermolecular <br /> contains, addition sequences present mature                    protein interactions (5,6). <br />  <br /> *To correspondence addressed. Tel: +1 312 996 6978; Fax: +1 312 996 7034; Email: lflau@uic.edu <br />  3622 Nucleic Acids Research, 2001, Vol. 29, No. 17 <br />  <br />  <br />    To understand function Bop1 further, sought            SstII-cut pRS316GAL1 (11; gift Dr S. Liebman). The <br /> establish role homolog yeast. In work identify   inducible expression cassette resulting construct was <br /> YMR049C, S.cerevisiae homolog mammalian Bop1,             excised ApaI FspI ligated ApaI&#226;&#8364;&#8220;SmaI <br /> essential gene required synthesis 25S 5.8S rRNAs.           fragment pRS413 (12) make pGAL1yBOPfl-HIS/CEN. <br /> Together previous results, findings demonstrate         The diploid strains containing erb1::URA3 allele were <br /> that Bop1 essential factor large ribosomal subunit            transformed plasmid, sporulated tetrads were <br /> biogenesis, conserved yeast mammals structurally          dissected YPGal plates. <span id='am-2' about='protege:FROM' typeof='owl:Thing'>From</span> resulting <br /> and functionally. In reference important role           erb1::URA3[pGAL1::ERB1] haploids, showed identical <br /> protein eukaryotic ribosome biogenesis, named yeast          phenotypes, strain, YMS0086, selected subsequent <br /> gene ERB1 (yeast BOP1 unrelated gene). Bop1/Erb1p             experiments. <br /> the protein detailed analysis pre-rRNA               To obtain HA-tagged Erb1p, synthetic oligonucleotide <br /> processing effects performed yeast                encoding HA tag inserted upstream ERB1 start <br /> mammalian cells. The functional disruption ERB1 yeast            codon pGAL1yBOPfl-HIS/CEN Bsp120I&#226;&#8364;&#8220;EagI <br /> and Bop1 mouse cells provides unique opportunity              fragment containing pGAL1::HA-ERB1 cassette cloned <br /> parallel analysis pre-rRNA processing species,         Bsp120I-cut pRS319a (13). The resulting expression plasmid <br /> thereby contributing understanding mechanisms             pDP943-33 used replace pGAL1yBOPfl-HIS/CEN in <br /> ribosome formation higher eukaryotes.                               YMS0086 plasmid shuffling His&#226;&#8364;&#8220;Leu+ <br />                                                                        clones, YDP0135, used experiments. The <br />                                                                        phenotypes erb1::URA3 complementation wild-type <br /> MATERIALS AND METHODS                                                  ERB1 HA&#226;&#8364;&#8220;ERB1 similar, indicating HA-tagged <br /> Strains, media genetic methods                                     Erb1p wild-type Erb1p functionally equivalent. <br />  <br /> Saccharomyces cerevisiae strains used work deriva-         Depletion Erb1p protein analysis <br /> tives diploid strain resulting crossing BY4741        Cells grown YPGal mid log phase depleted Erb1p <br /> BY4742 (7). YMS0086 YDP0135 isogenic strains                   shifting YPD medium. Protein lysates prepared as <br /> (MATa ura3&#226;&#710;&#8224;0 his3&#226;&#710;&#8224;1 leu2&#226;&#710;&#8224;0 lys2&#226;&#710;&#8224;0 met15&#226;&#710;&#8224;0 erb1::URA3)                  described (8), lysis buffer additionally <br /> carrying plasmids pGAL1yBOPfl-HIS/CEN (CEN, HIS3,                      supplemented 10% (v/v) protease inhibitor cocktail <br /> pGAL1::ERB1) pDP943-33 (CEN, LEU2, pGAL1::HA-                      (Sigma) 1 mM EDTA. Protein concentration lysates <br /> ERB1), respectively. YMS0092 endogenous wild-type              determined using RC DC assay (Bio-Rad) and <br /> ERB1 allele carries plasmid pRS413 (CEN, HIS3),             samples normalized total protein separated 10% <br /> otherwise isogenic YMS0086. Growth genetic                    SDS&#226;&#8364;&#8220;polyacrylamide gel. Proteins transferred nitro- <br /> manipulation yeast cells performed established              cellulose membrane, stained Ponceau S verify equal <br /> methods (8).                                                           transfer probed HA.11 antibody (Babco), followed by <br />                                                                        detection ECL reagents (NEN). <br /> Cloning, disruption rescue ERB1 <br /> The high fidelity Platinum Pfx DNA polymerase (Gibco BRL)              Pulse&#226;&#8364;&#8220;chase labeling RNA <br /> was used PCR cloning. A fragment ERB1 locus             For pulse&#226;&#8364;&#8220;chase analysis, 25&#226;&#8364;&#8220;30 ml culture grown an <br /> was cloned PCR yeast genomic DNA                      OD600 0.6&#226;&#8364;&#8220;0.8 precipitated, washed with <br /> PCR-Script vector (Stratagene), using primers GTCTCTAG-                medium lacking methionine uracil resuspended 1 ml <br /> CAAAGGGAG GGTGTGCTAGCCAAATCC (&#226;&#8364;&#8220;408                             medium 30&#194;&#176;C. L-[Methyl-3H]methionine or <br /> +2126 relative start codon ERB1, respectively). A            [5,6-3H]uracil (NEN) added 100 &#194;&#181;Ci/ml 2 min <br /> fragment containing 197 bp 5&#226;&#8364;&#178;-untranslated sequence             labeling mixture chased 4-fold dilution with <br /> 1167 bp ERB1 coding sequence removed NruI              medium containing 5 mM methionine 2 mM uracil. Cells <br /> BglII replaced SmaI&#226;&#8364;&#8220;BamHI fragment YCp50               collected aliquots removed 0, 2, 8 16 min of <br /> (9) containing URA3. The resulting plasmid cut XmnI           chase brief centrifugation immediately frozen dry <br /> to generate fragment disrupting ERB1 diploid strain         ice/ethanol mixture. RNA isolated frozen cells by <br /> (BY4741 crossed BY4742). Gene disruption carried              acid phenol extraction (14). <br /> out using standard method (10). Correct integration was <br /> confirmed Southern blot analysis.                                   Northern primer extension analyses steady-state <br />   A galactose-inducible ERB1 expression plasmid                    RNA levels <br /> constructed allow rescue haploids inheriting                 The sequence oligonucleotide yU3-1 used detection of <br /> erb1::URA3 allele. Primers CGGGATCCGGTCAATGAT-                         U3 snoRNA ACCAAGTTGGATTCAGTGGC; oligo- <br /> GGCTAAG (BamHI site underlined, start codon ERB1                 nucleotides used northern hybridizations identical to <br /> bold) GGTGTGCTAGCCAAATCC (+2126 relative                    previously described (15). Primer extensions done <br /> start codon ERB1) used amplify ERB1                1 &#194;&#181;g total RNA 2 pmol labeled primer 20 &#194;&#181;l <br /> cosmid SC9796 (ATCC). A 625 bp BamHI&#226;&#8364;&#8220;EcoRI fragment                 reaction mixture containing 200 U MMLV reverse tran- <br /> this PCR product containing 5&#226;&#8364;&#178;-portion ERB1 open            scriptase (Gibco BRL), 1&#195;&#8212; strand buffer supplied the <br /> reading frame ligated EcoRI&#226;&#8364;&#8220;SstII fragment            enzyme, 10 mM DTT, 1 mM dNTPs, 10 U RNasin (Promega) <br /> SC9796 contained remaining 1797 bp ERB1                10% formamide 40 min 42&#194;&#176;C. Aliquots 1/15 the <br /> coding sequence 617 bp 3&#226;&#8364;&#178; flanking sequence. The                reaction products separated 7.5 M urea&#226;&#8364;&#8220;6% poly- <br /> complete ERB1 coding fragment cloned BamHI-               acrylamide gel alongside dideoxy sequencing reaction to <br />                                                                                 Nucleic Acids Research, 2001, Vol. 29, No. 17 3623 <br />  <br />  <br /> determine positions primer extension stops. All hybrid-        medium led significant impediment growth 8&#226;&#8364;&#8220;12 h after <br /> izations primer extensions analyzed Storm 860           medium shift (Fig. 3A). We analyzed expression HA&#226;&#8364;&#8220;Erb1p in <br /> phosphorimager quantitated using ImageQuant 5.2 soft-             strain immunoblotting using antibodies HA <br /> ware (Molecular Dynamics).                                            tag. In cells grown galactose-containing medium we <br />                                                                       detected bands resulting expression HA-tagged <br />                                                                       Erb1p absent control strain containing non- <br /> RESULTS                                                               tagged Erb1p (Fig. 3B). The largest form HA&#226;&#8364;&#8220;Erb1p <br /> Highly conserved homologs Bop1 exist                migrated SDS&#226;&#8364;&#8220;polyacrylamide gels &#226;&#710;&#188;140 kDa, is <br /> Eukaryota                                                             larger calculated molecular mass Erb1p (93 kDa). <br />                                                                       This anomalous mobility observed mouse Bop1, <br /> A database search revealed genes homologous mouse Bop1             migrates &gt;100 kDa calculated molecular <br /> in various eukaryotic species display high levels             mass 83 kDa (4). Several HA-tagged polypeptides that <br /> sequence similarity well-conserved domain structure            detected HA&#226;&#8364;&#8220;Erb1p strain, including prom- <br /> (Fig. 1). All proteins contain WD repeats              inent bands migrating 80&#226;&#8364;&#8220;85 kDa, represent proteolyti- <br /> similarly positioned C-terminal portion protein         cally processed forms Erb1p (Fig. 3B). The amounts all <br /> sequence. Based analysis multiple aligned sequences,         proteins significantly reduced &#226;&#710;&#188;2 h shift from <br /> we recognize distinct WD repeats (Fig. 1),           galactose- glucose-containing medium, indicating that <br /> only WD repeats initially described mouse            Erb1p depleted (Fig. 3B). <br /> Bop1 sequence (16). Individual repeats various levels of <br /> divergence canonical structure (5). Two additional           Depletion Erb1p impairs synthesis 25S 5.8S <br /> WD-like segments identified organisms          rRNA <br /> gap WD repeats 1 2, scores fall           Since Bop1 involved processing pre-rRNA mouse <br /> significance level. The absence critical amino acid residues       cells (4), tested inhibition cell growth caused by <br /> and lack evolutionary conservation preclude assign-          depletion Erb1p related defects pre-rRNA <br /> ment true WD repeats. However, leads interesting        synthesis. Analysis total RNA prepared YDP0135 <br /> speculation &#226;&#8364;&#732;primordial&#226;&#8364;&#8482; Bop1 entire C-terminus       cells repression ERB1 showed decrease rRNA, <br /> (starting WD1, shown Fig. 1) consisted            progressively declining ratio 25S 18S rRNA (Fig. 3C), <br /> entirely WD repeats. In addition well-conserved core,         indicating 25S rRNA preferentially affected. This result <br /> all proteins contain N-terminal domain 50&#226;&#8364;&#8220;150             suggested that, like mouse homolog, Erb1p plays role in <br /> amino acid residues conserved sequence level,      biogenesis large ribosome subunit rRNA. <br /> but similar composition predominated polar residues            Expression Bop1&#226;&#710;&#8224;, dominant negative form Bop1, in <br /> such serine, threonine, aspartate glutamate.                   mouse cells affects processing steps 28S rRNA <br />   These data Bop1 structurally conserved                 formation (4). We tested depletion its <br /> throughout Eukaryota. All eukaryotic organisms               yeast homolog affect pre-rRNA processing (Fig. 4). First, <br /> sequenced genomes appear contain single copy             analyzed pre-rRNA processing pulse&#226;&#8364;&#8220;chase labeling of <br /> gene. No homologs Bop1 Archaea Eubacteria.        RNA [methyl-3H]methionine (Fig. 5, upper panels). To <br />                                                                       observed effects altered methy- <br /> ERB1, yeast homolog mammalian Bop1, an <br />                                                                       lation, performed similar analysis cells labeled with <br /> essential gene <br />                                                                       [3H]uracil (Fig. 5, lower panels). <br /> The mouse Bop1 protein previously shown participate         The kinetics pre-rRNA processing erb1::URA3 cells <br /> in regulation ribosome biogenesis (4) cell cycle           grown galactose-containing medium essentially <br /> control (16,17), based analysis dominant-negative       similar wild-type strain (Fig. 5, compare left and <br /> mutant protein, Bop1&#226;&#710;&#8224;. To investigate yeast homolog            right panels). Depletion Erb1p glucose-containing <br /> Bop1, replaced ERB1 allele (the previously uncharac-           medium produced strong defects pre-rRNA processing. <br /> terized open reading frame YMR049C) URA3 marker                In pulse&#226;&#8364;&#8220;chase labeling non-depleted cells, 27SA 20S <br /> cassette (see Materials Methods). Tetrad analysis          pre-rRNAs, products processing primary 35S <br /> resulting diploid strain showed 2:2 segregation viable          pre-rRNA sites A0, A1 A2 (see Fig. 4), visible <br /> non-viable spores (Fig. 2). All viable spores Ura&#226;&#8364;&#8220;, indicating   2 min labeling (Fig. 5, left, 0 min chase). In contrast, <br /> that disruption ERB1 incompatible vegetative               highly labeled 27SA 20S pre-rRNAs apparent at <br /> growth. We constructed centromeric plasmid conditional          point Erb1p-depleted cells, 35S pre-rRNA takes <br /> expression ERB1 glucose-repressible GAL1                  longer process remains detectable 8 min chase <br /> promoter (see Materials Methods). Expression Erb1p             (Fig. 5; center panel demonstrates point better <br /> from plasmid supported growth erb1::URA3 strains           efficient chase methionine labeling). <br /> YPGal plates, repression Erb1p glucose             Although processing 35S pre-rRNA delayed Erb1p <br /> YPD plates prevented growth (Fig. 2).                                 depletion, completely blocked: 8 min chase most <br />    Effects ERB1 disruption growth studied          material labeled 35S pre-rRNA cells is <br /> strain YDP0135, GAL1 promoter drove expression           processed A2 step, judging accumulation of <br /> of HA-tagged Erb1p. The growth rate strain galactose-      labeled 18S rRNA. <br /> containing medium exponential similar            Whereas transfer <span id='am-3' about='rdfs:label' typeof='owl:Thing'>label</span> 18S rRNA occurs in <br /> wild-type strain (Fig. 3A data shown). Repression          Erb1p-depleted cells, virtually labeled 25S rRNA be <br /> the promoter shifting cells glucose-containing YPD              detected 16 min chase cells (Fig. 5, center <br />  3624 Nucleic Acids Research, 2001, Vol. 29, No. 17 <br />  <br />  <br />  <br />  <br /> Figure 1. Homology Erb1/Bop1 primary structure eukaryotes. Organisms sequence accession nos are: Sce, S.cerevisiae (NP_013764); Spo, <br /> Schizosaccharomyces pombe (S54549); Ath, Arabidopsis thaliana (AAD25679); Cel, Caenorhabditis elegans (T26995); Dme, Drosophila melanogaster <br /> (T13579); Mmu, Mus musculus (NP_038509); Hsa, Homo sapiens (BAA09473, incomplete sequence). Initial alignment obtained ClustalX (34) then <br /> manually edited using GeneDoc (35). Black shading indicates identical residues conservative substitutions sequences; grey shading indicates resi- <br /> dues &gt;50% sequences. WD repeats 1&#226;&#8364;&#8220;5 identified search Pfam database (36) seven sequences. The WD1 repeat closest the <br /> canonical WD40 structure (5). Numbers parentheses sequence indicate lengths small unique amino acid insertions N-terminal domains, <br /> which conserved primary structure level (see text). <br />                                                                                               Nucleic Acids Research, 2001, Vol. 29, No. 17 3625 <br />  <br />  <br />  <br />  <br /> Figure 2. Disruption rescue ERB1. (A) Tetrad analysis ERB1/ <br /> erb1::URA3 diploid strain shows 2:2 segregation viable spores, indicating <br /> that ERB1 essential viability. (B) Four haploid products tetrad <br /> obtained sporulation ERB1/erb1::URA3 strain carrying CEN, HIS3, <br /> pGAL1::ERB1 plasmid streaked YPGal (top) YPD (bottom) plates. <br /> The Ura His genotypes clones indicated the <br /> figure, showing growth erb1::URA3 haploids restored expression <br /> of plasmid-borne, galactose-inducible ERB1. <br />  <br />  <br />  <br />  <br /> panels). In control cells expressing Erb1p galactose medium <br /> formation 25S 18S rRNAs essentially complete this <br /> time (Fig. 5, left), wild-type cells (Fig. 5, right). The <br /> disappearance label 35S precursor absence <br /> of label processing intermediates 25S rRNA in <br /> Erb1p-depleted cells indicates precursors are <br /> degraded, extremely slowly converted                         Figure 3. Effects Erb1p depletion growth rate rRNA. (A) Increase in <br /> 25S rRNA. Analysis low molecular weight RNAs labeled                           OD600 measured parallel YDP0135 cultures, was <br />                                                                                   grown YPGal medium transferred YPD time 0. Cultures <br /> with [3H]uracil showed loss synthesis 5.8S rRNA                        periodically diluted maintain OD600 &lt; 1.0 experiment. <br /> after Erb1p depletion (data shown). Thus, Erb1p depletion                     (B) Protein lysates prepared YDP0135 cells (pGAL1::HA-ERB1) at <br /> causes virtually complete processing block branch                   indicated times transfer YPD control YMS0086 strain <br /> leads formation 25S 5.8S rRNAs (Fig. 4) does                    (pGAL1::ERB1) (lane c), normalized protein content, analyzed by <br /> prevent 18S rRNA synthesis, consistent decreasing                        immunoblotting anti-HA tag antibody. (C) RNA extracted from <br />                                                                                   cultures used protein analysis. Equal amounts total RNA, as <br /> 25S/18S rRNA ratio cells (Fig. 3C).                                      determined absorbance 260 nm, separated formaldehyde- <br />                                                                                   containing 1.2% agarose gel, transferred nylon membrane stained with <br /> Pre-rRNA processing anomalies Erb1p-depleted cells                             methylene blue. <br /> To determine processing steps affected Erb1p <br /> depletion, isolated RNA depleted, non-depleted and <br /> wild-type cells, hybridized oligonucleotide probes                      site A2. As noted above, pulse&#226;&#8364;&#8220;chase analysis shows accumu- <br /> designed detect various pre-rRNA intermediates (Fig. 6)                    lation significant amounts labeled 18S rRNA 8&#226;&#8364;&#8220;16 min <br /> quantified results phosphorimager. Consistent                   chase (Fig. 5), implying A1 A2 processing steps <br /> results pulse&#226;&#8364;&#8220;chase experiments, hybridization                         occur relatively efficiently. However, steady-state <br /> probes complementary pre-rRNA revealed increase                          level 20S precursor 18S rRNA, revealed hybridiza- <br /> steady-state levels 35S pre-rRNA, indicating effi-                    tion probe y002, undergoes 8&#226;&#8364;&#8220;10-fold decrease after <br /> ciency cleavage site A0 5&#226;&#8364;&#178;-ETS diminished. The                    18&#226;&#8364;&#8220;24 h Erb1p depletion (Fig. 6). Although results <br /> product A0 processing, 33S pre-rRNA, rapidly converted                      contradictory, explained the <br /> by cleavage site A1 32S pre-rRNA, subsequent                          lack inhibition processing 20S pre-rRNA in <br /> processing site A2 generates 20S precursor 18S rRNA                     Erb1p-depleted cells. Thus, combination lowered influx <br /> and 27SA2 precursor 25S 5.8S rRNAs (Fig. 4). The                       delayed processing 35S pre-rRNA rapid subse- <br /> decrease levels 32S pre-rRNA (Figs 5 6) suggests                   quent conversion 18S RNA account low steady- <br /> depletion Erb1p does significantly affect processing                    state levels 20S precursor observed northern <br />  3626 Nucleic Acids Research, 2001, Vol. 29, No. 17 <br />  <br />  <br />  <br />  <br /> Figure 4. Pre-rRNA processing S.cerevisiae. The structure major sites processing primary 35S pre-rRNA shown top. The external tran- <br /> scribed sequences 5&#226;&#8364;&#178;-ETS 3&#226;&#8364;&#178;-ETS flank ends mature 18S, 5.8S 25S rRNA sequences, separated internal transcribed spacers ITS1 <br /> and ITS2. Initially, 35S pre-rRNA cleaved succession sites A0 A1 generate 32S pre-rRNA. The cleavage 32S pre-rRNA site A2 separates two <br /> processing branches lead formation mature rRNAs small large ribosome subunits. The 20S precursor undergoes cleavage site D export <br /> to cytoplasm, yielding mature 18S rRNA. The 27SA2 precursor processed alternative pathways form long short forms 5.8S rRNA and <br /> 25S rRNA. The major pathway, generates 5.8SS form, proceeds cleavage 27SA2 site A3 ITS1 processing site B2, giving rise <br /> to 27SA3, followed exonucleolytic processing A3 site B1S, generates 27SBS pre-rRNA. In minor pathway leads 5.8SL formation, 27SBL <br /> pre-rRNA generated processing site B1L, occurs unidentified mechanism, processing B2. The subsequent processing 27SB <br /> precursors sites C1 C2 probably identical gives rise mature 25S rRNA 7S pre-rRNAs. The trimmed exonucleases 3&#226;&#8364;&#178;-end <br /> to site E, yielding mature 5.8S rRNA species. <br />  <br />  <br />  <br /> hybridizations (Fig. 6) absence strongly labeled                      increased Erb1p-depleted cells (1.3&#226;&#8364;&#8220;2-fold), suggesting that <br /> 20S band pulse&#226;&#8364;&#8220;chase labeling (Fig. 5).                                         processing 27SA2 pre-rRNA fact occur with <br />    Hybridizations probes y002, y005 y033 detected                      reduced efficiency. The end products processing branch <br /> &#226;&#710;&#188;2-fold accumulation 23S RNA Erb1p depletion (Fig. 6).                    starts 27SA pre-rRNA (see Fig. 4) synthesized <br /> This RNA considered aberrant product formed                               depleted cells: pulse&#226;&#8364;&#8220;chase analysis shows virtually <br /> cleavage 35S pre-rRNA site A3, bypassing processing                       complete absence 25S 5.8S rRNAs (see above) and <br /> sites A0&#226;&#8364;&#8220;A2, observed pre-rRNA processing                            hybridization probe y013 reveals disappearance 7S <br /> perturbed (1). The increase 23S RNA indicates                    pre-rRNA, precursor 5.8S rRNA (Fig. 6). <br /> cleavage site A0 delayed. However, detect accumu-                         We examined levels processing intermediates <br /> lation aberrant RNA species, 21S 22S                           formed 27SA pre-rRNA. 27SA2 pre-rRNA predomi- <br /> RNAs (1,15), arguing A0 processing completed, <br />                                                                                    nantly processed site A3 rise 27SA3 pre-rRNA and <br /> subsequent processing sites A1 A2 significantly <br />                                                                                    processed site B1L generate 27SBL pre-rRNA. The <br /> inhibited. <br />                                                                                    subsequent processing 27SA3 RNA involves exonucleolytic <br />    The steady-state levels 27SA2 pre-rRNA detected with <br />                                                                                    trimming site A3 site B1S, generating 27SBS pre-rRNA <br /> probe y005 4&#226;&#8364;&#8220;7-fold decrease combined levels <br /> of 27SA2 27SA3 pre-rRNAs detected probe y001                          (Fig. 4). To determine steady-state levels 27SA and <br /> undergo 3&#226;&#8364;&#8220;5-fold decrease 18&#226;&#8364;&#8220;24 h shifting                              27SB pre-rRNAs affected Erb1p depletion, we <br /> pGAL1::ERB1 cells glucose-containing medium,                                 performed primer extension oligo y013 (Fig. 7). In good <br /> compared non-depleted cells (Fig. 6). A slowed synthesis                      agreement hybridization results, intensity of <br /> due delayed processing A0 likely contributes                          primer extension stop site A2 reduced (5-fold 24 h), <br /> decrease 27SA pre-rRNA steady-state levels. It clear                     signal A3 modestly (1.4-fold) increased, indicating <br /> whether 27SA pre-rRNA destabilized. Interest-                          increase steady-state levels 27SA3 pre-rRNA. The <br /> ingly, ratio levels products A2                      altered A3/A2 ratio Erb1p-depleted cells be, in <br /> cleavage, 27SA2 20S pre-rRNAs (Fig. 4), slightly                            part, preferential cleavage slowly processed 35S <br />                                                                                          Nucleic Acids Research, 2001, Vol. 29, No. 17 3627 <br />  <br />  <br />                                                                                pathway pre-rRNA degradation Erb1p-depleted cells, <br />                                                                                however, remains determined. <br />                                                                                  In summary, significant effect Erb1p depletion in <br />                                                                                yeast cells absence synthesis 25S rRNA 7S <br />                                                                                pre-rRNA 27SB pre-rRNAs. This accompanied by <br />                                                                                under-accumulation 27SB pre-rRNAs, suggesting that <br />                                                                                pre-rRNAs immediate products are <br />                                                                                destabilized result Erb1p deficiency. No steps in <br />                                                                                pre-rRNA processing completely blocked, although <br />                                                                                processing 35S pre-rRNA, possibly 27SA pre-rRNA, is <br />                                                                                delayed. <br />  <br />                                                                                DISCUSSION <br />                                                                                In study examined role Erb1p, S.cerevisiae <br />                                                                                homolog mammalian Bop1, processing rRNA. Gene <br />                                                                                disruption ERB1 yeast showed gene essential <br />                                                                                cell viability. In conditional expression system, depletion <br />                                                                                Erb1p caused virtually complete inhibition synthesis of <br />                                                                                25S 5.8S rRNA. Synthesis 18S rRNA cells was <br />                                                                                blocked, rRNA formed reduced rate, <br /> Figure 5. Pulse&#226;&#8364;&#8220;chase analysis pre-rRNA processing. YMS0086 cells      agreement observation defects 25S/5.8S <br /> grown galactose-containing synthetic SGal medium (left) transferred   rRNA synthesis inhibit extent 18S rRNA <br /> glucose-containing SD medium 20 h deplete Erb1p (center). RNA from <br /> the control strain YMS0092 containing wild-type ERB1 shown right.    synthesis (1). The levels 5S rRNA significantly <br /> Cells labeled L-[methyl-3H]methionine (upper) [5,6-3H]uracil      affected Erb1p depletion (data shown). These findings <br /> (lower) 2 min chased non-radioactive methionine uracil,        consistent previous observations showing that <br /> respectively, 2, 8 16 min described Materials Methods. The   interfering mouse Bop1 function results inhibition of <br /> same total RNA loaded lane methyl methionine- <br /> labeled samples c.p.m. lane loaded uracil-labeled <br />                                                                                28S 5.8S rRNA maturation (4) suggest Erb1p and <br /> samples. RNA separated formaldehyde&#226;&#8364;&#8220;agarose gel electrophoresis,        Bop1 serve parallel functions yeast mammalian cells. <br /> transferred nylon membrane visualized fluorography.                   Analysis pre-rRNA processing hybridization and <br />                                                                                primer extension demonstrates depletion Erb1p yeast <br />                                                                                affects multiple steps synthesis 25S 5.8S rRNAs. <br />                                                                                The affected step appears cleavage 35S pre-rRNA <br /> pre-rRNA site A3, generates 23S RNA (see                         site A0. Both 20S 27SA pre-rRNA levels decreased <br /> above).                                                                        partially delayed A0 processing. The subse- <br />   The intensity primer extension stop corresponding                  quent fate precursors different: 18S rRNA is <br /> processing sites B1S B1L decreased Erb1p depletion                   synthesized, albeit reduced efficiency, mature <br /> (Fig. 7). The effect especially striking B1S stop,                 rRNAs formed 27SA pre-rRNA. The steady-state <br /> which greatly diminished (13&#226;&#8364;&#8220;33-fold reduction),                     levels different 27S pre-rRNA species decrease varying <br /> the stop corresponding site B1L decreased lesser extent                degrees, reflecting complex balance synthesis, <br /> (2&#226;&#8364;&#8220;3-fold). One possible interpretation decrease                 processing degradation rates. The striking deple- <br /> B1S signal relative B1L processing 27SA3 27SBS                tion 27SB pre-rRNA, occurs largely under- <br /> pre-rRNA inhibited 27SA2 27SBL                     accumulation 27SBS. Further products 27SB pre-rRNA <br /> pre-rRNA. However, strong decrease B1S signal                    processing virtually absent Erb1p-depleted cells, apparently <br /> disproportional relatively small increase A3, arguing                accelerated degradation 27SB pre-rRNAs or <br /> that effect unlikely blocked 27SA3 27SBS                     derivatives. <br /> processing. Since precursors formed Erb1p-depleted                      A number yeast proteins previously shown to <br /> cells 27SB pre-rRNAs 27SB pre-rRNAs them-                           involved 25S/5.8S rRNA synthesis (for reviews 1,2). <br /> selves accumulate, bulk RNA precursors                  Notably, depletion proteins leads accumu- <br /> appears destined degradation. This raises possi-                 lation 27SB pre-rRNA, apparently delay its <br />                                                                                processing. Examples factors include Nip7p <br /> bility stronger decrease B1S signal be <br />                                                                                (18), Nop56p (19), putative methyltransferases Nop2p (20) <br /> affected rapid degradation newly formed <br />                                                                                Spb1p (21) helicases Spb4p (22), Dbp3p (23) and <br /> 27SBS pre-rRNA compared 27SBL.                                         Dbp10p (24). Deficiencies factors appear to <br />   The pulse&#226;&#8364;&#8220;chase data (Fig. 5) consistent inter-                 increase 27S pre-rRNA turnover. For example, reduced accumu- <br /> pretation 27SB pre-rRNA does accumulate                        lation 27S pre-rRNA, presumably increased degra- <br /> degraded Erb1p-depleted cells. With types labeling                  dation, reported depletion Nop8p (25); not, <br /> (Fig. 5, center) minute 27S bands                      however, shown 27SB pre-rRNA preferentially <br /> detected, control samples non-depleted cells                 affected, Erb1p-depleted cells. Under-accumulation of <br /> 27SB products clearly visible starting 2 min chase               27SB pre-rRNA observed cells depleted Dbp6p <br /> are later converted 25S rRNAs (Fig. 5, left). The precise                   (26), Dbp7p (27) Nop4p/Nop77p (28,29). In cells, <br />  3628 Nucleic Acids Research, 2001, Vol. 29, No. 17 <br />  <br />  <br />  <br />  <br /> Figure 6. Northern analysis pre-rRNA processing effects produced Erb1p depletion. YDP0135 cells grown YPGal transferred YPD time <br /> indicated. Equal amounts total RNA separated agarose gel electrophoresis, blotted hybridized labeled oligonucleotide probes complementary <br /> to different regions pre-rRNA transcript (shown top). Hybridization membrane probe detecting U3 snoRNA, affected <br /> by Erb1p depletion, performed loading control. Lane wt, RNA isogenic wild-type ERB1 strain YMS0092. <br />  <br />  <br />  <br /> however, B1S B1L pathways similarly                            stop detectable depletion Erb1p (Fig. 7). <br /> affected 27SA3 pre-rRNA appeared depleted well.                     Moreover, strains carrying mutations Rat1p Xrn1p that <br /> The pre-rRNA processing defects produced Erb1p depletion                      unable perform normal processing site A3 site <br /> most closely resemble defects observed depletion                        B1S display ladder aberrant RNA forms 5&#226;&#8364;&#178;-ends <br /> Rlp7p (30). Deficiency Rlp7p similarly causes increase                     extending B1S A3 (31). Similar RNAs accumulate <br /> in steady-state levels 27SA3 strong depletion                        temperature-sensitive mutant nucleolar protein <br /> 27SBS, little effect 27SBL. Rlp7p proposed                   Ebp2p, involved 27SA3 processing (32). No <br /> to function recruiting exonucleases 27SA3 precursor.                   extended forms observed primer extension reac- <br /> Initiation exonucleolytic processing site A3                 tions RNA Erb1p-depleted cells (Fig. 7). We there- <br /> negatively affected depletion Erb1p, rela-                    fore favor model strong decrease B1S <br /> tively small increase 27SA3 cells difficult inter-             signal occurs primarily destabilization 27SBS pre- <br /> pret unequivocally A3/A2 ratio altered                    rRNA. One possible scenario Erb1p depletion lead <br /> preferential cleavage site A3 slowly processed 35S                         misassembly pre-ribosomal particles, 5&#226;&#8364;&#178;&#226;&#8224;&#8217;3&#226;&#8364;&#178; <br /> pre-rRNA.                                                                        exonucleases enter site A3 proceed degrade pre-rRNA <br />    While Erb1p depletion potentially affect processing                  normal stop site B1S. Exonucleolytic digestion <br /> 27SA3 27SBS, Erb1p absolutely required step.                  initiating ITS1 region previously suggested <br /> The intensity primer extension stop site B1S reduced,               function quality control mechanism degrades <br />                                                                                             Nucleic Acids Research, 2001, Vol. 29, No. 17 3629 <br />  <br />  <br />                                                                                   transdominant mutants Bop1 mouse cells (4). Interference <br />                                                                                   function genes similarly blocks formation two <br />                                                                                   components large ribosome subunits, 25S 5.8S rRNAs <br />                                                                                   yeast 28S 5.8S rRNAs mammalian cells. The <br />                                                                                   functional, structural (Fig. 1), conservation between <br />                                                                                   mouse Bop1 yeast Erb1p strongly suggests these <br />                                                                                   proteins perform similar functions pre-rRNA processing and <br />                                                                                   likely orthologous. Unfortunately, efforts to <br />                                                                                   complement Erb1p deficiency mammalian Bop1 have <br />                                                                                   unsuccessful far. Despite use different vectors, <br />                                                                                   detect expression full-<span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> mouse Bop1 <br />                                                                                   yeast cells, suggesting presently unidentified <br />                                                                                   feature Bop1 cDNA prevents efficient expression or <br />                                                                                   mammalian protein rapidly degraded yeast cells. <br />                                                                                      Disruption Bop1 Erb1p functions presents unique <br />                                                                                   opportunity comparative analysis pre-rRNA processing <br />                                                                                   mechanisms yeast mammalian cells. Like yeast counter- <br />                                                                                   part, Bop1 previously implicated processing within <br />                                                                                   ITS1 mouse cells (4). In mammalian cells Bop1 func- <br />                                                                                   tions analyzed using inducible expression of <br /> Figure 7. Primer extension analysis processing ITS1. Total RNA          transdominant mutant Bop1&#226;&#710;&#8224; (4). This analysis showed that <br /> isolated wild-type ERB1 strain YMS0092 (lane wt) the <br />                                                                                   inhibition Bop1 function slows early steps pre-rRNA <br /> pGAL1::HA-ERB1 strain YDP0135 grown YPGal medium transferred to <br /> YPD medium indicated times deplete Erb1p. Primer extensions       processing, severely affects processing 36S and <br /> performed equal amounts RNA using oligonucleotide y013 (see Fig. 6)         32S pre-rRNAs, structurally homologous 27SA <br /> as described Materials Methods. Positions primer extension stops        27SB pre-rRNAs yeast. In striking similarity the <br /> (indicated right) match previously determined major processing sites   effects Erb1p depletion yeast, Bop1&#226;&#710;&#8224; causes accumula- <br /> within ITS1 (31); asterisk indicates additional stop unidentified <br /> nature observed strain. <br />                                                                                   tion 36S pre-rRNA absence products of <br />                                                                                   processing 32S pre-rRNA (4). These studies demon- <br />                                                                                   strate key role Erb1p/Bop1 processing events <br /> pre-rRNA incorrectly assembled ribosomes (31). The                             lead synthesis 25S/28S 5.8S rRNA diverse <br /> different mechanism 5&#226;&#8364;&#178;-end formation B1L processing                         eukaryotic organisms. <br /> (1) render 27SBL pre-rRNAs succeptible                                 We initially identified N-terminally truncated form of <br /> exonucleolytic degradation, making levels                         Bop1, Bop1&#226;&#710;&#8224;, genetic selection reversible inhibitors of <br /> affected Erb1p-depleted cells.                                                 cell cycle mammalian cells (16). Remarkably, induction <br />                                                                                   Bop1&#226;&#710;&#8224; causes cell cycle arrest asynchronously growing <br />   The exact biochemical function Erb1p unknown at <br />                                                                                   cells cell cycle, does decrease cell <br /> present. The mammalian homolog Bop1 co-fractionates with <br />                                                                                   viability (17). In contrast, yeast cells depleted Erb1p <br /> 32S pre-rRNA-containing complexes (4) forms tight, salt- <br />                                                                                   continue growing unaltered rate generations, <br /> resistant bonds nucleolar RNPs (D.G.Pestov, unpublished <br />                                                                                   gradually slowing &#226;&#710;&#188;8 h depletion (Fig. 3). <br /> data). Erb1p recently identified component 66S                    These kinetics growth inhibition similar those <br /> RNP complexes isolated purification tagged Nop7p,                           observed number yeast proteins depletion <br /> which contain 40 proteins appear represent                       blocks rRNA synthesis. <br /> intermediates 60S ribosome subunit assembly yeast                              Does Bop1 additional function mammalian cells <br /> (P.Harnpicharnchai, J.Jakovljevic, E.Horsey, T.Miles,                             related cell cycle control lacking Erb1p? While we <br /> J.Roman, M.Rout J.Woolford, personal communication).                          rule possibility, notable Bop1 <br /> Remarkably, depletion components                        sequence does exhibit obvious additional domains that <br /> complexes, including Nop7p, Drs1p (33) Rlp7p (30),                       account function (Fig. 1). It possible the <br /> inhibits processing 27S precursors, suggesting cooperation                difference fundamentally different ways <br /> between proteins critical proper maturation 27S                   mammalian yeast cells respond defects ribosome <br /> pre-rRNA. Erb1p/Bop1 does display known                                   biosynthesis. The slow decline proliferation rate yeast <br /> RNA-binding motifs, presence WD motifs                            cells likely results gradual ribosome depletion <br /> suggests participation protein&#226;&#8364;&#8220;protein interactions.                       caused distribution pre-existing ribosomes between <br /> Erb1p/Bop1 serve assembly factor                         dividing cells naturally occuring degradation. In <br /> coordinates protein interactions RNPs                             contrast, mammalian cells evolved special system <br /> processing 27S pre-rRNA (36S/32S pre-rRNA mammalian                         monitors defects ribosome biogenesis blocks cell <br /> cells) takes place. Detailed analysis interactions                     cycle progression detected. We recently <br /> Erb1p proteins Nop7p-containing complexes                        proposed response &#226;&#8364;&#732;nucleolar stress&#226;&#8364;&#8482; built- <br /> may provide important insights molecular mechanisms                      protective mechanism prevents DNA synthesis under <br /> of formation 60S ribosome particle.                                        sub-optimal conditions, reducing chances of <br />   There substantial similarity effects pre-rRNA                     acquiring mutations deleterious context of <br /> processing caused depletion Erb1p yeast cells                        multicellular organism (17). This hypothesis supported by <br />  3630 Nucleic Acids Research, 2001, Vol. 29, No. 17 <br />  <br />  <br /> the findings inactivation p53 abolishes stringent cell                 15. Allmang,C., Mitchell,P., Petfalski,E. Tollervey,D. (2000) <br /> cycle block caused Bop1&#226;&#710;&#8224; expression allows slow                             Degradation ribosomal RNA precursors exosome. Nucleic Acids <br />                                                                                        Res., 28, 1684&#226;&#8364;&#8220;1691. <br /> proliferation cells despite pre-rRNA processing block                         16. Pestov,D.G., Grzeszkiewicz,T.M. Lau,L.F. (1998) Isolation growth <br /> (17). These data suggest mammalian cells possess special                        suppressors cDNA expression library. Oncogene, 17, 3187&#226;&#8364;&#8220;3197. <br /> checkpoint mechanism linked ribosome biosynthesis                       17. Pestov,D.G., Strezoska,Z. Lau,L.F. (2001) Evidence p53- <br /> at partially depends p53. It extremely inter-                         dependent cross-talk ribosome biogenesis cell cycle: <br />                                                                                        effects nucleolar protein Bop1 G1/S transition. Mol. Cell. Biol., 21, <br /> esting determine functional disruption                             4246&#226;&#8364;&#8220;4255. <br /> proteins participate ribosome biogenesis lead                       18. Zanchin,N.I., Roberts,P., DeSilva,A., Sherman,F. Goldfarb,D.S. <br /> similar cell cycle effects mammalian cells. These studies                           (1997) Saccharomyces cerevisiae Nip7p required efficient 60S <br /> will require broad application knowledge ribosome                            ribosome subunit biogenesis. Mol. Cell. Biol., 17, 5001&#226;&#8364;&#8220;5015. <br />                                                                                    19. Gautier,T., Berges,T., Tollervey,D. Hurt,E. (1997) Nucleolar KKE/D <br /> biogenesis gained yeast system. The conservation                             repeat proteins Nop56p Nop58p interact Nop1p required <br /> function proteins involved, study Erb1p/Bop1                         ribosome biogenesis. Mol. Cell. Biol., 17, 7088&#226;&#8364;&#8220;7098. <br /> illustrates, facilitate exploration.                                   20. Hong,B., Brockenbrough,J.S., Wu,P. Aris,J.P. (1997) Nop2p is <br />                                                                                        required pre-rRNA processing 60S ribosome subunit synthesis in <br />                                                                                        yeast. Mol. Cell. Biol., 17, 378&#226;&#8364;&#8220;388. <br /> ACKNOWLEDGEMENTS                                                                   21. Kressler,D., Rojo,M., Linder,P. la Cruz,J. (1999) Spb1p a <br />                                                                                        putative methyltransferase required 60S ribosomal subunit biogenesis <br /> We thank Dr Sue Liebman invaluable advice help                             Saccharomyces cerevisiae. Nucleic Acids Res., 27, 4598&#226;&#8364;&#8220;4608. <br /> Dr John Woolford permission mention data prior                       22. la Cruz,J., Kressler,D., Rojo,M., Tollervey,D. Linder,P. (1998) <br /> publication. This work supported NIH grant CA52220             <br /> </body></html>