<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>EUKARYOTIC CELL, Oct. 2004, p. 1206&#226;&#8364;&#8220;1216                                                                                         Vol. 3, No. 5 <br /> 1535-9778/04/$08.00&#226;&#171;&#185;0 DOI: 10.1128/EC.3.5.1206&#226;&#8364;&#8220;1216.2004 <br /> Copyright &#194;&#169; 2004, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />  TbDSS-1, Essential Trypanosoma brucei Exoribonuclease Homolog <br />    That Has Pleiotropic Effects Mitochondrial RNA Metabolism <br />             Jonelle L. Penschow,&#226;&#8364;&#160; Daniel A. Sleve,&#226;&#8364;&#160; Christopher M. Ryan, Laurie K. Read* <br />                  Department Microbiology Immunology Witebsky Center Microbial Pathogenesis and <br />                                Immunology, SUNY Buffalo School Medicine, Buffalo, New York <br />                                               Received 13 April 2004/Accepted 21 June 2004 <br />  <br />              Mitochondrial gene expression trypanosomes controlled primarily levels RNA processing and <br />           RNA stability. This regulation undoubtedly involves numerous ribonucleases. Here characterize the <br />           Trypanosoma brucei homolog yeast DSS-1 mitochondrial exoribonuclease, term TbDSS-1. <br />           Biochemical fractionation indicates TbDSS-1 mitochondrially localized, predicted N-terminal <br />           sequence. In contrast yeast homolog, TbDSS-1 does appear associated mitochondrial <br />           ribosomes. Targeted downregulation TbDSS-1 <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-253' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-281' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-290' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> procyclic-form T. brucei results a <br />           severe growth defect. In addition, TbDSS-1 depletion leads decrease levels edited cyto- <br />           chrome oxidase subunit I (COI) mRNA unedited edited COIII mRNAs, indicating enzyme <br />           functions control mitochondrial RNA abundance. We observe considerable reduction the <br />           level edited apocytochrome b (CYb) mRNA corresponding increase unedited CYb mRNA, suggest- <br />           ing TbDSS-1 functions, directly indirectly, control RNA editing. The abundance both <br />           gCYb[560] gA6[149] guide RNAs reduced TbDSS-1 depletion, reduction in <br />           gCYb[560] dramatic. The significant reduction gCYb levels potentially account the <br />           observed decrease CYb RNA editing. Western blot analyses mitochondrial RNA editing stability <br />           factors indicate perturbations RNA levels observed TbDSS-1 knock-downs result <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br />           secondary effects mitochondrial proteins. In all, data demonstrate TbDSS-1 essential <br />           protein plays role mitochondrial RNA stability RNA editing. <br />  <br />  <br />    In mitochondria protozoan parasite Trypanosoma                 RNA processing, regulation RNA decay rates also <br /> brucei, posttranscriptional mechanisms gene regulation                likely major factor controlling abundance mature <br /> of major importance. Transcription maxicircle (30,               RNAs trypanosome mitochondria (37, 38, 51). Despite the <br /> 48) minicircle (22) genomes polycistronic. This suggests              extensive requirement ribonucleases trypanosome mito- <br /> that regulation processing stability rRNA-, mRNA-,                 chondrial gene expression, nucleases carry the <br /> and guide RNA (gRNA)-containing transcripts likely                 majority processes remain unidentified. <br /> critical steps gene expression. For example, endonu-                    Several RNase activities potential mediate mito- <br /> clease cleavage exonuclease trimming presumably re-                  chondrial RNA processing decay trypanosomes have <br /> quired production mature RNAs polycistronic                  described. gRNA-directed endonuclease U-specific <br /> precursors. Processing particularly critical step for-           exonuclease activities associated RNA editing com- <br /> mation mRNAs genes overlap                    plexes (42, 46, 50), distributive U-specific exonuclease <br /> impossible produce mature monocistronic RNAs from <br />                                                                              purified 4,000-fold Leishmania tarentolae mito- <br /> the precursor molecule (30, 48). Moreover, overlap- <br />                                                                              chondria (3). In addition, purified editing complexes both <br /> ping gene arrangement coupled polycistronic transcrip- <br />                                                                              T. brucei L. tarentolae contain proteins exo/endo/phos <br /> tion indicates exist rapid degrada- <br />                                                                              RNase III motifs predicted possess RNase <br /> tion nonfunctional by-products processing reactions. The <br />                                                                              activity (4, 43). A T. brucei mitochondrial RNase P-like activity <br /> majority mitochondrial RNAs T. brucei require an <br /> extensive RNA editing process involving uridine insertion                presumably functions tRNA maturation been <br /> deletion form translatable RNAs (55, 56). The steady-state                reported (52). Furthermore, two, possibly three, T. <br /> RNA pool contains large amounts improperly edited RNAs                    brucei mitochondrial endonucleases distinct in- <br /> (12, 29). These RNAs intermediates destined be-                    volved editing tRNA processing partially <br /> come properly edited represent aberrantly pro-                   purified characterized (45, 53). An endoribonuclease <br /> cessed RNAs need removed. The scenario                     termed MAR1, mitochondrial associated ribonuclease, was <br /> would require presence RNA surveillance                  purified gene cloned Leishmania (1). <br /> identify degrade improperly edited RNAs. In addition                  Whether relationship MAR1 protein <br />                                                                              endonuclease activities described T. brucei un- <br />                                                                              known. Two exoribonuclease activities addition U- <br />   * Corresponding author. Mailing address: Department Microbi-            specific nuclease detected mitochondria L. taren- <br /> ology Immunology, 138 Farber Hall, SUNY Buffalo School of <br />                                                                              tolae (3). One processive hydrolytic enzyme, while <br /> Medicine, Buffalo, NY 14214. Phone: (716) 829-3307. Fax: (716) 829- <br /> 2158. E-mail: lread@acsu.buffalo.edu.                                        exhibits preference 3&#226;&#172;&#732; phosphate ends. Finally, <br />   &#226;&#8364;&#160; J.L.P. D.A.S. contributed equally work.                      recently described hydrolytic exoribonuclease activity <br />  <br />                                                                       1206 <br />  VOL. 3, 2004                                                                          TbDSS-1 AFFECTS T. BRUCEI MITOCHONDRIAL RNAs                                   1207 <br />  <br />  <br /> from T. brucei mitochondrial membranes preferentially                              ACGCGTCGACCTACTCGAGCGAGTCCAGAGATGGAAGAAGGCA <br /> degrades polyadenylated RNAs (51).                                                      CT 3&#226;&#172;&#732;), RNase1-I-5&#226;&#172;&#732; (5&#226;&#172;&#732; CCGCTCGAGTGGAGCGCGACGGTAGCGAC 3&#226;&#172;&#732;), <br />                                                                                         RNase1-I-3&#226;&#172;&#732; (5&#226;&#172;&#732; CCCAAGCTTCCCACCTGTGCCTCCTCATC 3&#226;&#172;&#732;), DSS1-9 (5&#226;&#172;&#732; <br />    The yeast mitochondrial degradosome (originally termed                               GAAAATCATCTTTCTATACCATTTCGCG 3&#226;&#172;&#732;), DSS1-10 (5&#226;&#172;&#732; TCGTAAGGAT <br /> mtEXO) purified Saccharomyces cerevisiae mi-                             GTTTTGTTTAATGTGTTTC 3&#226;&#172;&#732;), p22i-5&#226;&#172;&#732; (5&#226;&#172;&#732; CCGCTCGAGCGCAAATCCCA <br /> tochondria decade ago (39) shown play role                          TGGGGACGAGGA 3&#226;&#172;&#732;), p22i-3&#226;&#172;&#732; (5&#226;&#172;&#732; CCCAAGCTTACGAAACAAATTTGTTA <br /> in multiple aspects mitochondrial RNA turnover pro-                              ATGCTGCTC 3&#226;&#172;&#732;), 12S-1 (5&#226;&#172;&#732; GCTTGTTAACCTGCTCGAAC 3&#226;&#172;&#732;), 9S-1 (5&#226;&#172;&#732; CCG <br />                                                                                         CAACGGCTGGCATCCAT 3&#226;&#172;&#732;), Tub-RT (5&#226;&#172;&#732; GGGGGTCGCACTTTGTC 3&#226;&#172;&#732;), <br /> cessing organism. The degradosome comprises pro- <br />                                                                                         CYb-RT (5&#226;&#172;&#732; CAACCTGACATTAAAAGAC 3&#226;&#172;&#732;), COI-RT (5&#226;&#172;&#732; GTAATGAGTAC <br /> teins: DSS-1 exoribonuclease SUV3 RNA helicase                              GTTGTAAAACTG 3&#226;&#172;&#732;), COIII-3&#226;&#172;&#732;NE (5&#226;&#172;&#732; ACTTCCTACAAACTAC 3&#226;&#172;&#732;), A6-3&#226;&#172;&#732;NE <br /> (17, 33). It exhibits 3&#226;&#172;&#732;-<span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span>-5&#226;&#172;&#732; exonuclease activity de-                       (5&#226;&#172;&#732; ATTTGATCTTATTCTATAACTCC 3&#226;&#172;&#732;), gCYb[560]A (5&#226;&#172;&#732; TCCCTAGAGAG <br /> pendent standard ribonucleosides (NTPs)                            TAGTTATCCTCCCCATTACTCAG 3&#226;&#172;&#732;), gA6[149] (5&#226;&#172;&#732; ATAATTTCACA <br /> deoxyribonucleoside triphosphates (dNTPs) (39), RNA-stimu-                              GATATCTTTTC 3&#226;&#172;&#732;). <br />                                                                                            Cloning TbDSS-1 cDNA. A BLAST search T. brucei genomic data- <br /> lated NTPase activity (39), RNA helicase activity (17).                             <span id='am-316' about='xsp:base' typeof='owl:Thing'>base</span> The Institute Genomic Research revealed gene fragment <br /> Isolation yeast degradosome using tandem affinity                             (GATGG96TV) encoding portion protein homology S. cerevisiae <br /> purification (TAP)-tagged DSS-1 SUV3 revealed                               DSS-1. Based sequence genomic fragment, oligonucleotides were <br /> complex exclusively associated mitochondrial ribo-                              designed cloning 5&#226;&#172;&#732; 3&#226;&#172;&#732; ends TbDSS-1 cDNA RXS-dT17- <br /> somes. The functions degradosome mitochondrial                                primed cDNA nested rapid amplification cDNA ends (RACE) strategies. <br />                                                                                         To clone 5&#226;&#172;&#732; end, cDNA amplified primers ESL-22, corre- <br /> gene expression explored genetic approaches                             sponds T. brucei spliced leader sequence, RNase 1-4. Five microliters <br /> yeast. One function appears related splicing sta-                          resulting product amplified ESL-22 RNase 1-3, the <br /> bility group I intron-containing transcripts. Cells                      product cloned BamHI/EcoRI site pBluescript II SK&#226;&#171;&#186;. To obtain <br /> express SUV3-1 suppressor allele DSS-1                            3&#226;&#172;&#732; end sequence, cDNA amplified primers RXS-dT17 RNase 1-1. <br />                                                                                         Five microliters resulting product amplified RXS-dT17 and <br /> gene deleted defective splicing group <br />                                                                                         RNase 1-2, product cloned BamHI/EcoRI site pBlue- <br /> I introns undergo massive accumulation excised group                             script II SK&#226;&#171;&#186;. Two clones sequenced directions 5&#226;&#172;&#732; 3&#226;&#172;&#732; <br /> I introns (9, 16, 21). Non-splicing-related degradosome func-                           ends. Based 5&#226;&#172;&#732; 3&#226;&#172;&#732; end sequences, entire TbDSS-1 open reading <br /> tions specifically examined yeast containing intron-                         frame (ORF) amplified cDNA primers RNase 1-5&#226;&#172;&#732;exp2 and <br /> less mitochondrial genome (16, 17). Deletion SUV3                             RNase 1-3&#226;&#172;&#732;exp2. The resulting product cloned NheI/XhoI site of <br />                                                                                         pET21a, sequence complete ORF verified. The TbDSS-1 <br /> or DSS-1 genes cells resulted dramatic effects                       sequence compared sequences GenBank database, using default <br /> abundance mitochondrial RNAs. Unexpectedly, lev-                             BLAST algorithm. Multiple alignments performed CLUSTAL W. <br /> els mRNAs encoding cytochrome b (COB) 16S                                      Production recombinant TbDSS-1 antibody production. TbDSS-1 was <br /> rRNA dramatically decreased SUV3- DSS1-                                expressed six-His (His6) fusion protein follows. The entire TbDSS-1 ORF <br />                                                                                         PCR amplified oligonucleotides RNase 1-5&#226;&#172;&#732;exp2 RNase 1-3&#226;&#172;&#732;exp2, <br /> <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>null mutants</span>. This suggests RNAs directly <br />                                                                                         amplified product digested ligated NheI XhoI sites <br /> or indirectly stabilized mitochondrial degradosome. In                           pET21a. Escherichia coli cells carrying pET21a-TbDSS-1 plasmid were <br /> addition, Northern blot S1 nuclease analyses revealed                             grown Luria-Bertani (LB) medium 100-&#226;?&#174;g/ml ampicillin 37&#194;&#176;C an <br /> large accumulation mitochondrial mRNA rRNA pre-                                  A600 0.7. Protein production induced 0.3 mM isopropyl &#226;?&#164;-D-thioga- <br /> cursors improperly processed 5&#226;&#172;&#732; 3&#226;&#172;&#732; ends                        lactopyranoside (IPTG) 2 h, cells harvested centrifugation at <br />                                                                                         5,000 &#226;&#171;&#187; g 10 min 4&#194;&#176;C. Cells resuspended lysis buffer (6 M <br /> cells SUV3 DSS-1 gene deleted (16, 17).                              guanidine-HCl, 20 mM Tris-Cl [pH 7.9], 500 mM NaCl, 10% glycerol) lysed <br /> Accumulation precursors direct effect                         sonication ice (4 pulses 30 s each). The lysed cells rocked 60 <br /> on RNA processing and/or effect degradation                                min 4&#194;&#176;C, centrifuged 15,000 &#226;&#171;&#187; g 15 min 4&#194;&#176;C. The supernatant <br /> unprocessed transcripts. Regarding possibility,                       mixed Ni2&#226;&#171;&#185;-nitrilotriacetic acid (NTA) resin 3 h 4&#194;&#176;C, the <br />                                                                                         mixture poured column. The column washed 10 column <br /> been suggested function degradosome be <br />                                                                                         volumes wash buffer A (6 M guanidine-HCl, 20 mM Tris-Cl [pH 7.9], 500 mM <br /> as RNA surveillance targets improperly pro-                              NaCl, 10% glycerol, 20 mM imidazole). To renature recombinant protein, <br /> cessed RNAs.                                                                            column washed sequentially 10 column volumes 1:1, 3:1, and <br />    Toward goal functionally characterizing RNases                            7:1 (vol/vol) wash buffer B (wash buffer A lacking guanidine-HCl)-wash buffer A. <br /> that mediate trypanosome mitochondrial gene expression,                              The column finally washed 15 column volumes wash buffer B con- <br />                                                                                         taining 1 mM phenylmethonylsulfonyl fluoride eluted 10 column vol- <br /> searched homologs known exoribonucleases T. <br />                                                                                         umes wash buffer B containing gradient 50 500 mM imidazole. The <br /> brucei genomic databases. We report characterization                            purified protein dialyzed overnight 4&#194;&#176;C buffer containing 20 mM <br /> the T. brucei homolog yeast degradosome component                                Tris [pH 8.0] 25 mM KCl. The His6-TbDSS-1 protein yield 6.5 mg per <br /> DSS-1, term TbDSS-1. Western blot analysis con-                                liter E. coli cells. E. coli cells transformed pET21a vector were <br /> firms predicted mitochondrial localization TbDSS-1.                              processed identically provide negative control enzymatic assays. To <br />                                                                                         generate anti-TBDSS-1 antibodies, His6-TbDSS1 used antigen poly- <br /> Targeted gene depletion using RNA interference (RNAi) pro-                              clonal antibody production rabbits (Bethyl Laboratories, Inc., Mongomery, <br /> vides evidence TbDSS-1 affects stability editing                        Tex.). <br /> specific mitochondrial RNAs.                                                               Trypanosome growth, mitochondrial isolation, transfection, induction of <br />                                                                                         RNAi. The procyclic form T. brucei brucei clone IsTaR1 stock EATRO 164 was <br />                                                                                         grown described previously (7). Mitochondria isolated procedure <br />                         MATERIALS AND METHODS <br />                                                                                         Harris et al. (24). Procyclic T. brucei brucei strain 29-13 (kindly provided by <br />    Oligonucleotides used study. The oligonucleotides used study     George Cross), contains integrated genes T7 RNA polymerase and <br /> listed follows restriction sites underlined: RXS-dT17 (5&#226;&#172;&#732; GAGAATTCTCG           tetracycline repressor, grown SDM-79 supplemented 15% fetal <br /> AGTCGACTTTTTTTTTTTTTTTTT 3&#226;&#172;&#732;), ESL-22 (5&#226;&#172;&#732; GCGAATTCGCTATTAT                               bovine serum described previously (7, 58) presence G418 (15 &#226;?&#174;g/ml) <br /> TAGAACAGAGTTTCTG 3&#226;&#172;&#732;), RNase 1-1 (5&#226;&#172;&#732; GCGGATCCTAGTGGAGGCGG                                hygromycin (50 &#226;?&#174;g/ml). To construct pTbDSS-1i vector RNAi, a <br /> TGTGTTAC 3&#226;&#172;&#732;), RNase 1-2 (5&#226;&#172;&#732; GCGGATCCACAAAGAGCTCCCCTGTG 3&#226;&#172;&#732;),                             450-bp fragment TbDSS-1 gene (nucleotides [nt] 511 961 start <br /> RNase 1-3 (5&#226;&#172;&#732; GCGGATCCGTGCCGCGAAATGGTATAG 3&#226;&#172;&#732;), RNase 1-4 (5&#226;&#172;&#732;                            codon) amplified PCR oligonucleotides RNase1-i-5&#226;&#172;&#732; RNase1-i- <br /> GCGGATCCTGCAACATGGTCTAAAGACAAC 3&#226;&#172;&#732;), RNase 1-5&#226;&#172;&#732;exp2 (5&#226;&#172;&#732; GCG                              3&#226;&#172;&#732;. The fragment digested inserted XhoI/HindIII sites pZJM <br /> GATCCGCTAGCATGACCCCTCGGCGCGTCGCAAA 3&#226;&#172;&#732;), RNase 1-3&#226;&#172;&#732;exp2 (5&#226;&#172;&#732;                              (kindly provided Paul Englund) (57). For transfection, 1 &#226;&#171;&#187; 109 cells were <br />  1208       PENSCHOW ET AL.                                                                                                          EUKARYOT. CELL <br />  <br /> washed 100 ml ice-cold Cytomix resuspended fresh Cytomix                                  RESULTS <br /> a concentration 2.5 &#226;&#171;&#187; 107 cells/ml. Twenty micrograms pTbDSS1-i linear- <br /> ized NotI added 0.45 ml cells. Transfections carried out <br />                                                                                         Identification TbDSS-1. To identify RNases function <br /> on ice 2-mm cuvettes using Bio-Rad electroporator pulses        trypanosome mitochondrial RNA turnover processing, <br /> following settings: 800 V, 25 &#226;?&#174;F, 400 &#226;?&#8364;. Following transfection, cells      utilized BLAST algorithm search available T. <br /> transferred 10 ml SDM-79 supplemented G418 hygromycin           brucei databases homologs known exoribonucleases. Us- <br /> allowed recuperate 20 h. Selection applied addition        ing strategy, detected strong match the <br /> 2.5-&#226;?&#174;g/ml phleomycin, cells grown 4 weeks obtain stable <br />                                                                                      virtual translation product T. brucei genomic fragment <br /> transfectants. For induction double-stranded RNA (dsRNA), cells cul- <br /> tured presence 1-&#226;?&#174;g/ml tetracycline. Growth curves obtained        (TIGR GATGG96TV) segment S. cerevisiae <br /> plotting total cell number (the product cell number total         DSS-1 mitochondrial exoribonuclease (14). We obtained the <br /> dilution) period 11 days. Two separate inductions (including monitoring    complete sequence corresponding T. brucei cDNA using <br /> of growth RNA isolation) performed, analyses repeated         nested 5&#226;&#172;&#732; 3&#226;&#172;&#732; RACE strategies. Five prime RACE analysis <br /> with protein RNA inductions.                                           using 5&#226;&#172;&#732; primer corresponding T. brucei spliced leader <br />    RNA analysis. Total RNA purified 1.3 &#226;&#171;&#187; 109 4.8 &#226;&#171;&#187; 109 cells <br />                                                                                      sequence indicated presence single splice site a <br /> (Purescript RNA isolation kit; Gentra Systems). For PCR analysis, cDNA was <br /> synthesized 10 &#226;?&#174;g total RNA using oligonucleotide RXS-dT17. Ten           14-nt 5&#226;&#172;&#732;-untranslated region. Two polyadenylation sites were <br /> percent resulting cDNA using template amplification       identified 3&#226;&#172;&#732; RACE, resulting 146 148 nt 3&#226;&#172;&#732; <br /> full <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> TbDSS-1 ORF (oligonucleotides DSS1-9 DSS1-10)                   untranslated regions. The complete predicted ORF sub- <br /> 500-bp fragment p22 cDNA (oligonucleotides p22i-5&#226;&#172;&#732; p22i-3&#226;&#172;&#732;).              sequently amplified procyclic-form oligo(dT)-primed <br /> cDNA titrated ensure PCRs performed linear range <br />                                                                                      cDNA, cloned pET21a vector, sequenced both <br /> (27). For Northern blot analysis, 10 &#226;?&#174;g total RNA electrophoresed on <br /> 1.5% formaldehyde-agarose gels transferred nylon membrane. Blots <br />                                                                                      directions. The cDNA sequence contained 2,229-nt ORF <br /> were probed kinase-labeled oligonucleotide probes 12S-1 9S-1            predicts protein molecular mass 83.5 kDa and <br /> detection 12S 9S rRNA, respectively, described (10). For Northern          pI 6.43. Comparison predicted ORF nonre- <br /> blot analysis ND4 mRNA, antisense riboprobe generated vitro          dundant databases revealed significant homology fungal <br /> transcription incorporation [&#226;?&#163;-32P]UTP, hybridization per-           mitochondrial exoribonucleases S. cerevisiae DSS-1 (14) N. <br /> formed described previously (10). Primer extensions using oligonucleotides <br />                                                                                      crassa cyt-4 (15) wide variety prokaryotic <br /> Tub-RT, CYb-RT, COI-RT, COIII-3&#226;&#172;&#732;NE, A6-3&#226;&#172;&#732;NE performed with <br /> 10 15 &#226;?&#174;g total RNA described previously (44). For primer extension <br />                                                                                      hydrolytic 3&#226;&#172;&#732;-5&#226;&#172;&#732; exoribonucleases RNase II/RNase R <br /> of gRNAs, 25 35 &#226;?&#174;g total RNA used. Gels analyzed            (RNR) family (13, 60). In view homology DSS-1 <br /> autoradiography followed densitometry nonsaturated autoradiographs             protein mitochondrial localization (see below), we <br /> or phosphoimager analysis Bio-Rad Personal FX Phosphoimager                  termed T. brucei gene TbDSS-1. <br /> using Quantity One software.                                                            TbDSS-1 identified RNR exoribonuclease family <br />    Glycerol gradient fractionation. Glycerol gradient fractionation mitochon- <br />                                                                                      member PROSITE BLOCKS algorithms. All <br /> drial lysates performed previously described (11). Mitochondrial lysates <br /> from 1010 procyclic-form T. brucei cells loaded 12-ml gradient, <br />                                                                                      members RNR family, including prokaryotic enzymes <br /> and 500-&#226;?&#174;l fractions collected centrifugation. Fifteen microliters     mitochondrial DSS-1 cyt-4 proteins, possess a <br /> each fraction subjected sodium dodecyl sulfate-polyacrylamide gel elec-       conserved central region variable N-terminal extension. <br /> trophoresis (SDS-PAGE) (7.5% polyacrylamide) anti-TbDSS-1 Western                TbDSS-1 23% identical 42% similar S. cerevisiae <br /> blot analysis. Standards (bovine serum albumin, 4S; yeast alcohol dehydrogenase,     DSS-1 430 amino acids encompassing conserved cen- <br /> 7.4S; thyroglobulin, 19S) fractionated parallel gradient ana-      tral region (amino acids 191 621 TbDSS-1) (Fig. 1A). <br /> lyzed SDS-PAGE Coomassie blue staining. The 40S region gradi- <br />                                                                                      Comparable levels homology observed the <br /> ent defined location REAP-1 protein (32) determined by <br /> Western blot antibodies generously provided Steve Hajduk. To deter- <br />                                                                                      TbDSS-1 amino acid sequence cyt-4 the <br /> mine sedimentation mitochondrial ribosomes, RNA isolated 150         prokaryotic Thermatoga maritima RNase R. Four conserved <br /> &#226;?&#174;l glycerol gradient fractionation follows. Glycerol gradient fractions   sequence motifs define RNR exoribonuclease family <br /> were incubated 15 mins 37&#194;&#176;C 0.5% SDS, 50-&#226;?&#174;g/ml proteinase K,         contained central region (60). Motifs I IV of <br /> 2 U RNaseOUT (Invitrogen). Samples extracted twice phenol-              TbDSS-1 protein depicted Fig. 1A, se- <br /> chloroform-isoamylalcohol (25:24:1), RNA precipitated ethanol           quences aligned corresponding sequences S. <br /> and 10 &#226;?&#174;g glycogen carrier. RNAs fractionated 1.5% agarose- <br />                                                                                      cerevisiae DSS-1, N. crassa cyt-4, T. maritima RNase R <br /> formaldehyde gel transferred nylon membrane. Sedimentation 9S <br /> rRNA detected Northern blot analysis using kinase-labeled 9S-1 <br />                                                                                      proteins Fig. 1B. Within sequence blocks, TbDSS-1 <br /> oligonucleotide.                                                                     possesses 21 29 amino acids conserved 80% <br />    Western blot analysis RNAi cells. Aliquots (5 &#226;&#171;&#187; 106 cells) uninduced    RNR family members (60). Motif IV conserved <br /> induced cells suspended SDS-PAGE sample buffer immediately             blocks considered RNase II signature <br /> harvesting, boiled 10 min, stored &#226;&#171;&#186;80&#194;&#176;C use. For Western blot       (36, 60). TbDSS-1 contains 10 11 highly conserved amino <br /> analysis, 5 &#226;&#171;&#187; 106 cells electrophoresed SDS-PAGE (10% polyacryl- <br />                                                                                      acids present motif IV (underlined Fig. 1B). The pre- <br /> amide) (TbMP52 blot) 1.66 &#226;&#171;&#187; 106 cells electrophoresed SDS-PAGE <br /> (15% polyacrylamide) (all blots) transferred nitrocellulose mem- <br />                                                                                      dicted TbDSS-1 protein contains features common both <br /> brane. Blots probed anti-RBP16 (26), anti-p22 (27), anti-TbMP52 (54)       prokaryotic mitochondrial RNR family members. Most <br /> (generously provided Ken Stuart), anti-gBP21 anti-gBP25 (both gen-         strikingly, majority RNR family members, including the <br /> erously provided Julius Lukes). Primary antibodies detected incuba-       prokaryotic RNase R proteins, possess S1 RNA binding <br /> tion horseradish peroxidase-conjugated goat anti-mouse antibodies (Pierce;      motif C terminus (Fig. 1A) (60). However, TbDSS-1 is <br /> 1:10,000 dilution) TbMP52, horseradish peroxidase-conjugated goat anti- <br />                                                                                      similar mitochondrial RNR family enzymes, DSS-1 and <br /> rabbit antibodies (Pierce; 1:10,000 dilution) followed development the <br /> SuperSignal West Pico chemiluminescent (Pierce).                              cyt-4, lacking S1 motif. TbDSS-1 similar to <br />    Nucleotide sequence accession number. The TbDSS-1 sequence               mitochondrial RNR members conserved arginine at <br /> submitted GenBank database accession no. AY233297.                      extreme C terminus motif IV absent (asterisk Fig. <br />  VOL. 3, 2004                                                             TbDSS-1 AFFECTS T. BRUCEI MITOCHONDRIAL RNAs                           1209 <br />  <br />  <br />  <br />  <br />   FIG. 1. Alignment TbDSS-1 (Tb; accession no. AY233297) T. maritima RNase R (Tm; accession no. Q9WZI1), S. cerevisiae DSS-1 (Sc; <br /> accession no. AAC49144), Neurospora crassa cyt-4 (Nc; accession no. P47950). (A) Schematic representation structure TbDSS-1 and <br /> other RNR family members. Black boxes indicate conserved motifs I IV. The checked box indicates S1 RNA binding motif present <br /> in RNR family exoribonucleases, exception mitochondrial members family. The bar defines conserved <br /> central region encompassing amino acids 191 621 TbDSS-1, amino acid conservation region listed right. (B) Alignment <br /> of motif I IV sequences. Amino acids identical proteins indicated white letters black background. <br /> Conservative substitutions indicated white letters gray background. Residues underlined motif IV highly conserved <br /> RNase II signature residues. The C-terminal residue indicated asterisk highly conserved arginine typically absent in <br /> mitochondrial RNR family members. <br />  <br />  <br />  <br /> 1B). Conversely, TbDSS-1 bears greater resemblance pro-                   recombinant protein required denaturation/renaturation, is <br /> karyotic mitochondrial RNR family members possess-                   possible small fraction protein folded into <br /> ing relatively short N-terminal variable region (Fig. 1A).                 conformation supports enzymatic activity. A likely <br />   Based predicted function, attempts               explanation properly folded TbDSS-1 lacks enzy- <br /> demonstrate exoribonuclease activity bacterially expressed                matic activity. This consistent observation <br /> His6-tagged TbDSS-1 (Fig. 2B). However, able                   S. cerevisiae DSS-1 completely dependent association <br /> to demonstrate enzymatic activity levels vec-              SUV3 helicase exoribonuclease activity (17). It <br /> tor control nickel column eluates. Because solubilization             thought inability DSS-1 act exonuclease <br />  1210     PENSCHOW ET AL.                                                                                                        EUKARYOT. CELL <br />  <br />  <br />  <br />  <br />    FIG. 2. Mitochondrial localization TbDSS-1. (A) A putative mitochondrial targeting sequence present TbDSS-1. The putative <br /> N-terminal mitochondrial import sequence TbDSS-1 predicted Target P v1.0. This sequence aligned known predicted <br /> targeting sequences T. brucei mitochondrial proteins TBmtRNAP, TBMP48, ISP, ATPase, RBP16, gBP21 (8, 23, 26, 28, 35, 47). The <br /> characteristic N-terminal arginine residues shown white letters black background. Serine threonine residues (white <br /> letters gray background) flank hydrophobic residues shown light gray. (B) Production bacterially expressed His6-TbDSS-1 <br /> (6XHis-TbDSS-1). Proteins electrophoresed SDS-PAGE stained Coomassie blue. M, molecular mass markers; Un, extracts from <br /> uninduced cells; In, extracts induced cells; P, nickel column-purified His6-TbDSS-1. (C) TbDSS-1 enriched mitochondria. Anti-TbDSS-1 <br /> antibodies used probe blots containing 10 25 &#226;?&#174;g procyclic-form T. brucei whole-cell mitochondrial extracts. Anti-RBP16 antibodies <br /> (26) used positive control known mitochondrial protein. <br />  <br />  <br />  <br /> on lack S1 RNA binding domain              exhibits characteristics sequences, including <br /> that SUV3 required substrate interaction. TbDSS-1              tandem arginine residues followed multiple hydrophobic <br /> lacks S1 domain, SUV3 RNA helicase homolog                    amino acids interspersed flanking serine threo- <br /> present T. brucei genome database.                                  residues (Fig. 2A). Furthermore, TargetP and <br />    Southern blot analysis indicates TbDSS-1 gene               PSORTII programs predict mitochondrial localization for <br /> present single copy T. brucei genome (data                 TbDSS-1. To biochemically define subcellular localization <br /> shown). The genome related trypanosomatid, Leishma-                 TbDSS-1, expressed His6-tagged TbDSS-1 E. coli and <br /> nia major, encodes predicted protein 50% amino acid                 purified protein denaturing nickel chelate chromatog- <br /> identity TbDSS-1 (GenBank accession no. AL389894.4),                    raphy (Fig. 2B). Antibodies generated renatured <br /> indicating mitochondrial exoribonuclease homolog              protein used analyze TbDSS-1 protein T. <br /> highly conserved kinetoplastid species.                              brucei whole-cell mitochondrial extracts Western blot <br />    Mitochondrial localization TbDSS-1. The high degree               (Fig. 2C). Anti-TbDSS-1 antibodies recognized protein with <br /> homology TbDSS-1 known mitochondrial exori-                    apparent molecular mass approximately 90 kDa whole- <br /> bonucleases suggested TbDSS-1 localized               cell mitochondrial extracts, agreement pre- <br /> mitochondria T. brucei. To address issue,               dicted mass 83.5 kDa. Comparison TbDSS-1 levels in <br /> examined TbDSS-1 amino acid sequence regions                    whole-cell mitochondrial extracts reveals TbDSS-1 is <br /> homology known trypanosome mitochondrial import se-                     enriched approximately 10-fold mitochondrial lysates. This <br /> quences (25, 35). The amino terminus TbDSS-1 ORF                    similar degree enrichment observed mito- <br />  VOL. 3, 2004                                                        TbDSS-1 AFFECTS T. BRUCEI MITOCHONDRIAL RNAs                        1211 <br />  <br />  <br />  <br />  <br />   FIG. 3. Glycerol gradient analysis TbDSS-1. Mitochondrial extract procyclic-form T. brucei fractionated 10 40% glycerol <br /> gradients. The sedimentation TbDSS-1 protein determined Western blot. The position mitochondrial ribosomes identified by <br /> Northern blot analysis 9S rRNA. Fraction numbers positions size standards indicated figure. C, His6-TbDSS-1. <br />  <br />  <br />  <br /> chondrial RNA binding protein, RBP16, probed              cells escaped RNAi growth resumed, commonly ob- <br /> the blot. Based sequence biochemical evi-              served T. brucei. To confirm downregulation TbDSS-1 <br /> dence, conclude TbDSS-1 protein nuclearly             mRNA, steady-state TbDSS-1 mRNA levels analyzed by <br /> encoded, mitochondrially localized protein.                            reverse transcription (RT)-PCR days 2, 4, 6 following <br />    TbDSS-1 does appear associated mitochon-          induction RNAi compared uninduced cells. <br /> drial ribosomes. Sucrose gradient analysis yeast mitochon-          TbDSS-1 mRNA levels reduced 43 23% unin- <br /> dria indicated degradosomes, including DSS-1 com-           duced levels days 2 4, respectively. By 6 days RNAi <br /> ponent, entirely associated mitochondrial ribosomes           induction, TbDSS-1 mRNA undetectable. mRNA encod- <br /> (17). To determine TbDSS-1 similarly ribosome associated <br /> in trypanosomes assess potential association with <br /> other mitochondrial components, fractionated T. brucei <br /> mitochondrial extracts 10 40% glycerol gradients (11) <br /> (Fig. 3). Gradient fractions analyzed Western blot to <br /> determine sedimentation TbDSS-1. RNA isolated <br /> from gradient fractions, 9S rRNA analyzed North- <br /> ern blot determine sedimentation mitochondrial ri- <br /> bosomes. TbDSS-1 protein exhibited broad distribution in <br /> fractions 3 12, correspond S values approxi- <br /> mately 5 30S. These data suggest fraction <br /> of TbDSS-1 present multicomponent complexes, ex- <br /> pected constituent mitochondrial degradosome. <br /> In contrast sedimentation TbDSS-1, 9S rRNA was <br /> detected half gradient, primarily frac- <br /> tions corresponding S values 35S greater. We observed <br /> almost overlap distribution TbDSS-1 mito- <br /> chondrial ribosomes. Thus, initial experiments suggest that <br /> TbDSS-1 complexed mitochondrial proteins, but <br /> is stably associated ribosomes. <br />    TbDSS-1 essential procyclic T. brucei. To begin to <br /> assess function TbDSS-1 vivo, used RNAi to <br /> selectively down-regulate TbDSS-1 mRNA levels. A 450-bp <br /> fragment TbDSS-1 coding region corresponding nt <br /> 511 961 start codon cloned pZJM <br /> vector opposing T7 promoters (57). The resulting <br /> vector transformed T. brucei strain 29-13, ex- <br /> presses T7 polymerase tetracycline repressor pro- <br /> tein (58). Two cultures integrated TbDSS-1 frag- <br /> ment rDNA spacer selected phleomycin <br /> treatment, cultures used subsequent             FIG. 4. Effect TbDSS-1 RNAi cell growth. (A) Growth of <br /> detailed analyses. RNAi induced addition 1-&#226;?&#174;g/ml             procyclic-form T. brucei TbDSS-1 RNAi cells uninduced (open <br /> tetracyline, cell growth monitored induced versus           squares) induced 1 &#226;?&#174;g tetracycline (closed squares). Growth <br /> uninduced cells (Fig. 4A). Cells grew normally 4         curves obtained plotting total cell number product <br />                                                                        cell density total dilution. (B) TbDSS-1 mRNA levels <br /> days induction. However, cell growth ceased day 6          monitored PCR amplification full-length TbDSS-1 RNA in <br /> tetracycline addition, indicating TbDSS-1 essential        induced uninduced cells days 2, 4, 6 tetracycline <br /> growth procyclic-form T. brucei (Fig. 4A). Beyond day 9,            addition. p22 mRNA levels monitored control. <br />  1212     PENSCHOW ET AL.                                                                                                   EUKARYOT. CELL <br />  <br />  <br />  <br />  <br />    FIG. 5. Analysis effect TbDSS-1 RNAi rRNA and <br /> never edited mRNA levels. (A) Ten micrograms RNA isolated from <br /> uninduced (&#226;&#171;&#186;Tet) induced (&#226;&#171;&#185;Tet) cells days 2, 4, 6 after <br /> tetracycline addition subjected Northern blot analysis. rRNAs <br /> and ND4 mRNA detected radiolabeled oligonucleotide <br /> probes riboprobe, respectively. Ethidium bromide staining is <br /> shown indicate loading. (B) COI mRNA levels detected by <br /> poisoned primer extension. Tubulin (Tub) levels similarly ana-       FIG. 6. Analysis effect TbDSS-1 RNAi edited RNAs. <br /> lyzed control.                                                    Ten 15 &#226;?&#174;g RNA isolated uninduced (&#226;&#171;&#186;Tet) induced <br />                                                                        (&#226;&#171;&#185;Tet) cells days 4 6 tetracycline addition subjected <br />                                                                        poisoned primer extension analysis. Unedited (UE) edited <br />                                                                        (E) A6, COIII, CYb RNAs indicated arrows. The small <br />                                                                        boxes primer extension corresponding tubulin con- <br /> ing p22 protein (27) analyzed control RNA             trol poisoned primer extensions. <br /> recovery integrity. Together, results indicate that <br /> TbDSS-1 essential gene procyclic-form T. brucei. <br />    Effect TbDSS-1 downregulation rRNAs ed-             sor transcripts detected Northern blot TbDSS-1 <br /> ited mRNAs. To assess role TbDSS-1 mitochondrial             RNAi cells (Fig. 5A). In contrast, COI mRNA levels, de- <br /> RNA metabolism, asked steady-state levels spe-           tected poisoned primer extension, dramatically de- <br /> cific mitochondrial RNAs perturbed TbDSS-1 RNAi                creased TbDSS-1 downregulation (Fig. 5B). Taking into <br /> cells. We began looking RNAs undergo RNA             account levels tubulin control RNA, COI mRNA in <br /> editing, including rRNAs edited mRNAs. In               induced cells reduced 26% levels uninduced <br /> this way, determine role TbDSS-1 RNA                cells day 4. These results similar observed in <br /> stability uncomplicated potential effects editing        DSS-1-null mutants yeast, COB mRNA found <br /> process. When abundance mitochondrial rRNAs                 unstable (17). The significantly reduced COI mRNA <br /> examined Northern blot analysis, significant changes          levels observed TbDSS-1 downregulation indicate that <br /> the levels mature 9S 12S rRNA observed               protein plays role regulating mitochondrial RNA <br /> (Fig. 5A). In DSS-1-null mutants S. cerevisiae, mature mito-        abundance T. brucei. <br /> chondrial 15S rRNAs significantly decreased, accu-               Effect TbDSS-1 downregulation edited RNAs. We next <br /> mulation unprocessed precursor RNAs evident                  wanted determine TbDSS-1 RNAi affected <br /> Northern blot (17). However, long exposures 9S            mRNAs require editing maturation. We used a <br /> and 12S Northern blots TbDSS-1 RNAi cells, did             poisoned primer extension assay allows visualization of <br /> observe larger transcripts indicative precursor RNAs.           edited unedited versions given RNA same <br /> Thus, TbDSS-1 does appear involved rRNA pro-              assay (41). Interestingly, observed different pheno- <br /> cessing T. brucei mitochondria.                                     types regarding edited RNAs TbDSS-1 RNAi cells shown <br />    We examined effect TbDSS-1 downregulation            Fig. 6. ATPase subunit 6 (A6) COIII mRNAs are <br /> two edited mRNAs, NADH dehydrogenase subunit 4                   extensively edited insect (procyclic) mamma- <br /> (ND4), cytochrome oxidase subunit I (COI). As             lian life cycle stages T. brucei (5, 19). Neither edited nor <br /> rRNAs, ND4 levels essentially unaffected precur-           unedited A6 mRNA abundance significantly affected by <br />  VOL. 3, 2004                                                         TbDSS-1 AFFECTS T. BRUCEI MITOCHONDRIAL RNAs                      1213 <br />  <br />  <br /> downregulation TbDSS-1. On hand, observed <br /> that unedited edited versions COIII mRNAs <br /> were dramatically reduced days 4 6 TbDSS-1 <br /> RNAi induction. Accounting RNA recovery determined <br /> by tubulin primer extension standard, edited and <br /> unedited COIII mRNA levels reduced 20% in <br /> uninduced cells day 4. Similar results observed RNA <br /> preparations separate TbDSS-1 RNAi inductions. <br />                                                                          FIG. 7. Analysis effect TbDSS-1 RNAi gRNA levels. <br /> The corresponding decrease edited unedited <br />                                                                        RNA isolated uninduced (&#226;&#171;&#186;Tet) induced (&#226;&#171;&#185;Tet) cells <br /> mRNA indicate TbDSS-1 downregulation directly                 day 4 tetracycline addition. Twenty-five 35 &#226;?&#174;g RNA was <br /> affects abundance unedited COIII mRNA. The decrease             subjected poisoned primer extension analysis gA6[149] and <br /> in edited RNA levels indirect effect de-             gCYb[560] levels, respectively. <br /> creased editing substrate. It possible edited <br /> and unedited COIII mRNAs directly destabilized a <br /> similar degree TbDSS-1 depletion. We currently               unable detect gCYb[558] poisoned primer extension, <br /> distinguish possibilities.                           Northern blot, PCR determine whether <br />    Accumulation edited apocytochrome b (CYb) mRNA                gRNA affected TbDSS-1 depletion. The decrease in <br /> strictly developmentally regulated, edited mRNA be-           gCYb[560] similar extent decrease observed in <br /> ing present procyclic-form cells (18). When levels      CYb mRNA editing day tetracycline induc- <br /> CYb mRNA examined TbDSS-1 RNAi cells, ob-                   tion (compare Fig. 6 7). These results suggest CYb <br /> served decrease edited RNA levels increase         gRNAs limiting depletion leads a <br /> unedited CYb mRNA (Fig. 6). Taking account RNA re-                downregulation CYb RNA editing. gA6[149] levels in- <br /> covery, edited CYb mRNA levels reduced 59 47%              duced cells decreased 82.0% &#226;&#171;&#190; 2.8% un- <br /> of uninduced levels days 4 6 induction, respec-           induced cells. The modest reduction observe in <br /> tively. At time, unedited CYb mRNA levels increased           gA6[149] compared gCYb[560] consistent ab- <br /> by 2.3- 1.8-fold, days 4 6. We observed similar             sence A6 editing defect TbDSS-1 depletion. How- <br /> results separate primer extensions:           ever, poisoned primer extension analysis A6 mRNA <br /> two individual RNAi inductions. Two different scenarios          does specifically monitor region editing di- <br /> account reduction edited CYb mRNA increase              rected gA6[149], rule modest re- <br /> in unedited CYb mRNA TbDSS-1 downregulation. First,               duction editing observed region mRNA in <br /> TbDSS-1 opposite effects stabilization              TbDSS-1 knock-down cells. In all, results indicate that <br /> edited unedited CYb mRNAs. Edited CYb mRNA                  TbDSS-1 downregulation leads decrease abundance <br /> stabilized TbDSS-1 manner similar observed         gRNAs, magnitude decrease is <br /> COI COIII mRNAs. On hand, TbDSS-1                    gRNA specific. <br /> normally degrade percentage unedited CYb mRNA,                    TbDSS-1 does act indirectly modulation of <br /> leading accumulation RNA TbDSS-1 levels             known mitochondrial editing stability factors. The phe- <br /> are reduced. A second possibility simultaneously            notype TbDSS-1 RNAi cells reminiscent of, not <br /> explain decrease edited RNA increase un-            identical to, <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span>phenotype</span> reported RNAi mi- <br /> edited RNA TbDSS-1 plays role facilitating CYb            tochondrial RNA binding proteins. One protein is <br /> mRNA editing.                                                          RBP16, downregulation procyclic-form T. brucei <br />    Effect TbDSS-1 downregulation gRNA levels. One                leads significant reduction levels COI ND4 <br /> potential mechanism TbDSS-1 affect CYb                  RNAs, decrease edited CYb correspond- <br /> mRNA editing regulation gRNA abundance.                  ing increase unedited CYb mRNA (44). Based these <br /> That is, decrease CYb mRNA editing TbDSS-1                   observations, wanted determine TbDSS-1 acting, <br /> knock-down cells (Fig. 6) direct result gCYb               part, indirectly decrease RBP16. We <br /> depletion. To determine gRNA levels affected            performed Western blot analysis TbDSS-1 RNAi cells both <br /> TbDSS-1 RNAi cells, used poisoned primer extension               uninduced induced tetracycline 2, 4, 6 days <br /> quantify levels gRNAs specify editing CYb A6           (Fig. 8). We observed significant perturbation RBP16 <br /> mRNA cells uninduced induced               levels TbDSS-1 downregulation. p22 T. brucei mito- <br /> tetracycline 4 days (Fig. 7). gCYb[560] directs editing     chondrial protein interacts RBP16 dramat- <br /> 3&#226;&#172;&#732; end CYb editing domain, region             ically increase RNA binding capacity vitro (27). Thus, it <br /> whose editing monitored experiment presented             possible RBP16 function compromised in <br /> Fig. 6. The region editing specified gA6[149] 5&#226;&#172;&#732;       vivo decreased p22 levels. However, Western blot analysis <br /> edited region monitored poisoned primer extension Fig.           indicated p22 levels unchanged TbDSS-1 <br /> 6. We gCYb[560] gA6[149] levels               RNAi cells. gBP21 gBP25 RNA binding proteins <br /> decreased tetracycline-induced cells, different extents.     T. brucei bind gRNA high affinity, gBP21 <br /> In duplicate experiments, gCYb[560] levels            interact transiently editosome (2, 6, 28). Blood- <br /> induced cells 48.9% &#226;&#171;&#190; 3.1% uninduced cells.           stream-form gBP21 knock-outs display greatly decreased levels <br /> A second gRNA, gCYb[558], specify editing 3&#226;&#172;&#732;           COI mRNA unedited RNAs (31). In <br /> region CYb edited domain (49). However,                 addition, simultaneous elimination gBP21 gBP25 in <br />  1214      PENSCHOW ET AL.                                                                                                   EUKARYOT. CELL <br />  <br />  <br />                                                                           ganisms. Thus, study marks report DSS-1 exori- <br />                                                                           bonuclease homolog outside fungal kingdom. A protein <br />                                                                           high homology S. cerevisiae SUV3 RNA helicase is <br />                                                                           present T. brucei database. However, yet <br />                                                                           know protein associated TbDSS-1 degradosome <br />                                                                           complex. It regard unable to <br />                                                                           demonstrate exoribonuclease activity recombinant TbDSS-1. <br />                                                                           Yeast DSS-1 strictly dependent association SUV3 for <br />                                                                           exoribonuclease activity, probably unable bind <br />                                                                           RNA high affinity. Mitochondrial RNR exoribonuclease <br />                                                                           family members, including TbDSS-1, differ prokaryotic <br />                                                                           counterparts lack C-terminal S1 RNA binding <br />                                                                           domain. It thought SUV3 helicase required both <br />                                                                           substrate recognition unwinding yeast degradosome <br />                                                                           complex. Studies way determine protein compo- <br />                                                                           sition enzymatic activities T. brucei mitochondrial de- <br />                                                                           gradosome. <br />    FIG. 8. Western blot analysis known mitochondrial RNA stabil-          Targeted disruption TbDSS-1 using RNAi demonstrates <br /> ity editing factors TbDSS-1 RNAi cells. Extracts TbDSS-1      protein essential growth procyclic-form T. <br /> RNAi cells uninduced induced tetracycline 2, 4, 6   brucei. Western blot analyses known mitochondrial <br /> days electrophoresed SDS-PAGE (10% polyacrylamide)             RNA stability editing factors indicate an <br /> gel (TbMP52 blot) SDS-PAGE (15% polyacrylamide) gel (all <br />                                                                           indirect effect alterations abundance these <br /> other blots) transferred nitrocellulose membrane. Blots were <br /> probed antibodies RBP16, p22, gBP21, gBP25,              proteins. The essential nature TbDSS-1 contrast the <br /> TbMP52.                                                                   phenotype S. cerevisiae DSS-1-null mutants viable, <br />                                                                           albeit respiratory incompetent (14). It to <br />                                                                           determine phenotype TbDSS-1 disruption blood- <br /> procyclic forms results decreased COI mRNA edited                  stream-form trypanosomes, rely primarily glycolysis <br /> CYb mRNA levels, alterations mitochondrial                 energy generation. Because bloodstream forms de- <br /> RNA abundance (J. Lukes K. Stuart, personal communi-                  pend cytochrome-mediated respiration, TbDSS-1 not <br /> cation). However, Western blot analysis gBP21 gBP25                critical life cycle stage. <br /> indicated levels proteins unaffected             In procyclic trypanosomes, TbDSS-1 depletion results ab- <br /> by TbDSS-1 downregulation. Finally, alterations               errant levels mitochondrial RNA species, including <br /> of CYb mRNA levels TbDSS-1 RNAi cells suggested                      edited, unedited, edited mRNAs gRNAs. <br /> possible editing defect, determined levels               The diversity phenotypes associated TbDSS-1 down- <br /> editosome component altered. Analysis TbMP52 RNA                  regulation suggests participation protein multiple <br /> ligase abundance Western blot TbDSS-1 RNAi cells                    aspects mitochondrial RNA metabolism. We showed that <br /> revealed changes levels core editosome com-             edited COI unedited edited COIII <br /> ponent. Taken together, Western blot data indicate               mRNAs significantly reduced abundance TbDSS-1 <br /> the perturbations mitochondrial RNA metabolism observed                RNAi. It paradoxical depletion putative <br /> upon downregulation TbDSS-1 secondary effects                  exoribonuclease lead decreased RNA levels. How- <br /> attributable known mitochondrial RNA processing factors.               ever, similar observed DSS-1- and <br /> Thus, TbDSS-1 appears directly affect abundance                 SUV3-null mutants yeast, mature 15S rRNA and <br /> several classes mitochondrial RNAs procyclic-form T.                COB mRNAs significantly reduced (17). There sev- <br /> brucei.                                                                   eral potential explanations observation decreased <br />                                                                           RNA levels TbDSS-1 RNAi cells. First, trypanosome <br />                                                                           mitochondrial RNAs polycistronically transcribed re- <br />                            DISCUSSION <br />                                                                           quire 5&#226;&#172;&#732; 3&#226;&#172;&#732; processing maturation, possible that <br />   In study, present characterization putative            TbDSS-1 involved processing events. If precursor <br /> exoribonuclease T. brucei mitochondria, termed TbDSS-1.              transcripts improperly processed, result lower <br /> TbDSS-1 identified BLAST search T. brucei            levels mature mRNA species. Future experiments be <br /> genome databases RNR exoribonuclease family member                  aimed characterization precursor RNAs TbDSS-1- <br /> and homolog S. cerevisiae DSS-1 N. crassa cyt-4              depleted cells. A second possibility TbDSS-1 and/or a <br /> mitochondrial 3&#226;&#172;&#732;-to-5&#226;&#172;&#732; exoribonucleases. TbDSS-1 possesses              degradosome complex component acts sta- <br /> predicted N-terminal mitochondrial import sequence, West-             bilize mRNAs, directly indirectly. In E. coli, depletion <br /> ern blot analysis whole-cell mitochondrial extracts indi-          RNase II leads destabilization mRNAs ending in <br /> cates protein highly enriched mitochondria. In yeast,       hairpins, RNase II overproduction stabilizes some <br /> DSS-1 exoribonuclease associates SUV3 RNA helicase               RNA species (34). RNase II preferentially degrades poly(A) <br /> to form mitochondrial degradosome (17). Although SUV3                 tails, depletion leads increase polyadenylated <br /> homologs present human (40) Arabidopsis (20) mito-             mRNAs. Because poly(A) tails destabilizing elements in <br /> chondria, DSS-1 homologs identified or-            prokaryotes, expanded population polyadenylated <br />  VOL. 3, 2004                                                        TbDSS-1 AFFECTS T. BRUCEI MITOCHONDRIAL RNAs                                  1215 <br />  <br />  <br /> RNAs subject increased rates degradation           cassette, cassettes encoding gRNAs. This <br /> second exonuclease (34). We recently reported polyade-           suggests processing differ somewhat of <br /> nylation destabilizes unedited RNAs partially purified T.          gRNAs. In support hypothesis, gCYb[560]-con- <br /> brucei mitochondrial extracts (51). If TbDSS-1 normally de-           taining chimeras 19- 20-nt 3&#226;&#172;&#732; extensions that <br /> grades poly(A) tails mitochondrial RNA species,           predicted guide accurate editing reported (49). <br /> its downregulation result increased polyadenylation          Thus, gCYbs inherently subject inaccurate 3&#226;&#172;&#732; <br /> and subsequent RNA destabilization, analogous E. coli              processing gRNAs. Future experiments be <br /> RNase II. A potential scenario TbDSS-1             aimed determining TbDSS-1 plays role gRNA mat- <br /> levels depleted RNAi, levels mitochondrial        uration. <br /> ribonucleases increased effort compensate. This             In summary, results presented indicate that <br /> would reminiscent reported situation E. coli,      TbDSS-1, trypanosome homolog yeast mitochon- <br /> polynucleotide phosphorylase (PNPase) levels increased            drial DSS-1 exoribonuclease, mitochondrially localized and <br /> about twofold RNase II-deficient strains, RNase II             effects steady-state levels mature RNAs, poten- <br /> activity increased twofold PNPase gene           tially levels RNA stability RNA editing. The <br /> deleted (59). Testing hypothesis awaits identification     presence DSS-1 exoribonuclease SUV3 helicase <br /> additional mitochondrial exoribonuclease genes produc-            homologs T. brucei suggests trypanosomes possess <br /> tion antibodies encoded proteins. Finally,         mitochondrial degradosome similar structure de- <br /> mitochondrial degradosome comprised multiple ex-            scribed S. cerevisiae (17). This contrast plants and <br /> oribonucleases, similar cytoplasmic nuclear exo-           mammals, apparently lack DSS-1 homologs. Neverthe- <br /> some, deregulated depletion              less, results suggest structural functional differ- <br /> TbDSS-1. Identification mitochondrial degradosome com-             ences exist yeast T. brucei DSS-1 homologs. <br /> ponents begin address possibility.                       TbDSS-1 does appear stably associated ribo- <br />    The effect TbDSS-1 downregulation edited mRNAs               somes does participate rRNA maturation like its <br /> differs, depending RNA analyzed. Whereas edited           yeast counterpart. Moreover, role TbDSS-1 modulat- <br /> and unedited COIII mRNAs decreased, case CYb            ing levels specific edited RNAs, direct indirect, <br /> only edited mRNA decreased. Coincident de-            constitutes novel function family enzymes. <br /> crease edited CYb mRNA, unedited CYb mRNA levels are <br /> increased TbDSS-1-depleted cells. The CYb phenotype                                          ACKNOWLEDGMENTS <br /> could result opposite effects edited unedited <br />                                                                         This work supported National Institutes Health grant <br /> mRNA stabilization. That is, abundance edited CYb              AI47329 L.K.R. C.M.R. supported National Institutes <br /> mRNA decreased mech-                Health training grant AI07614. <br /> anisms described above. Conversely, TbDSS-1 normally <br /> regulate unedited CYb mRNA levels degrading transcript                                              REFERENCES <br /> in excess required, leading accumulation          1. Alfonzo, J. D., O. H. Thiemann, L. Simpson. 1998. Purification and <br />                                                                           characterization MAR1, mitochondrial associated ribonuclease from <br /> unedited transcript enzyme depleted. Alterna-                 Leishmania tarentolae. J. Biol. Chem. 273:30003&#226;&#8364;&#8220;30011. <br /> tively, TbDSS-1 act level CYb RNA editing. We            2. Allen, T. E., S. Heidmann, R. Reed, P. J. Myler, H. U. Go&#194;&#168;ringer, K. D. <br /> favor model, analysis gRNA levels              Stuart. 1998. Association guide RNA binding protein gBP21 active <br />                                                                           RNA editing complexes Trypanosoma brucei. Mol. Cell. Biol. 18:6014&#226;&#8364;&#8220; <br /> TbDSS-1 knock-down cells reveals corresponding decrease              6022. <br /> gCYb[560] edited CYb mRNA. Interestingly, regula-              3. Aphasizhev, R., L. Simpson. 2001. Isolation characterization a <br />                                                                           U-specific 3&#226;&#172;&#732;-5&#226;&#172;&#732;-exonuclease mitochondria Leishmania tarentolae. <br /> tion gRNA levels TbDSS-1 appears somewhat                     J. Biol. Chem. 276:21280&#226;&#8364;&#8220;21284. <br /> gRNA specific. While gCYb[560] levels induced TbDSS-1               4. Aphasizhev, R., I. Aphasizheva, R. E. Nelson, G. Gao, A. M. Simpson, X. <br /> knock-down cells 50% uninduced                 Kang, A. M. Falick, S. Sbicego, L. Simpson. 2003. Isolation U- <br />                                                                           insertion/deletion editing complex Leishmania tarentolae mitochondria. <br /> cells, gA6[149] present greater 80% unin-            EMBO J. 22:913&#226;&#8364;&#8220;924. <br /> duced levels. The absence dramatic decrease gA4[149]           5. Bhat, G. J., D. J. Koslowsky, J. E. Feagin, B. L. Smiley, K. Stuart. 1988. <br /> levels consistent unaltered A6 editing TbDSS-1                 An extensively edited mitochondrial transcript kinetoplastids encodes a <br />                                                                           protein homologous ATPase subunit 6. Cell 61:885&#226;&#8364;&#8220;894. <br /> knock-down cells.                                                      6. Blom, D., J. V. Burg, C. K. Breek, D. Speijer, A. O. Muijsers, R. Benne. <br />    How TbDSS-1 specificity regulating certain                   2001. Cloning characterization guide RNA-binding proteins from <br />                                                                           mitochondria Crithidia fasciculata: gBP27, novel protein, gBP29, the <br /> gRNAs, case gCYb[560], achieved? A potential point             orthologue Trypanosoma brucei gBP21. Nucleic Acids Res. 29:2950&#226;&#8364;&#8220;2962. <br /> of regulation gRNA processing. gRNAs polycistronically          7. Brun, R., M. Schonenberg. 1979. Cultivation vitro cloning of <br /> transcribed minicircle-encoded genes (22). Results             procyclic culture forms Trypanosoma brucei semi-defined medium. <br />                                                                           Acta Trop. 36:289&#226;&#8364;&#8220;292. <br /> based vitro processing suggest mature             8. Chi, T. B., S. V. Brown, N. Williams. 1998. Subunit 9 mitochon- <br /> gRNAs produced accurate transcription initiation            drial ATP synthase Trypanosoma brucei nuclearly encoded devel- <br /> 5&#226;&#172;&#732; end ribonucleolytic processing 3&#226;&#172;&#732; end (22). If              opmentally regulated. Mol. Biochem. Parasitol. 92:29&#226;&#8364;&#8220;38. <br />                                                                        9. Conrad-Webb, H., P. S. Perlman, H. Zhu, R. A. Butow. 1990. The <br /> TbDSS-1 involved gRNA 3&#226;&#172;&#732; end processing,                 nuclear SUV3&#226;&#8364;&#8220;1 <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>mutation</span> affects variety post-transcriptional processes <br /> result decrease mature gRNAs subsequent de-                 yeast mitochondria. Nucleic Acids Res. 18:1369&#226;&#8364;&#8220;1376. <br />                                                                       10. Corell, R. A., P. Myler, K. Stuart. 1994. Trypanosoma brucei mitochon- <br /> crease RNA editing. Intriguingly, 3&#226;&#172;&#732;-most CYb gRNAs,               drial CR4 gene encodes extensively edited mRNA completely edited <br /> gCYb[558] gCYb[560], present unusual mi-                    sequence bloodstream forms. Mol. Biochem. Parasitol. 64:65&#226;&#8364;&#8220;74. <br /> nicircle location (49). While vast majority gRNAs          11. Corell, R. A., L. K. Read, G. R. Riley, J. K. Nellissery, T. E. Allen, M. L. <br />                                                                           Kable, M. D. Wachal, S. D. Seiwert, P. J. Myler, K. Stuart. 1996. <br /> encoded cassettes defined conserved inverted 18-bp re-              Complexes Trypanosoma brucei exhibit deletion editing other <br /> peats, known CYb gRNA genes present                  editing-associated properties. Mol. Cell. Biol. 16:1410&#226;&#8364;&#8220;1418. <br />  1216       PENSCHOW ET AL.                                                                                                                          EUKARYOT. CELL <br />  <br /> 12. Decker, C. J., B. Sollner-Webb. 1990. RNA editing involves indiscrim-          36. Mian, I. S. 1997. Comparative sequence analysis ribonucleases HII, III, II, <br />     inate U changes precisely defined editing domains. Cell 61:1001&#226;&#8364;&#8220;            PH, D. Nucleic Acids Res. 25:3187&#226;&#8364;&#8220;3195. <br />     1011.                                                                              37. Michelotti, E. F., M. E. Harris, B. Adler, A. F. Torri, S. L. Hajduk. 1992. <br /> 13. Deutscher, M. P., Z. Li. 2001. Exoribonucleases multiple roles           Trypanosoma brucei mitochondrial ribosomal RNA synthesis, processsing <br />     RNA metabolism. Prog. Nucleic Acid Res. Mol. Biol. 66:67&#226;&#8364;&#8220;105.                       developmentally regulated expression. Mol. Biochem. Parasitol. 54:31&#226;&#8364;&#8220; <br /> 14. Dmochowska, A., P. Golik, P. P. Stepien. 1995. A novel nuclear gene                42. <br />     DSS-1 Saccharomyces cerevisiae necessary mitochondrial biogenesis.       38. Militello, K. T., L. K. Read. 2000. UTP-dependent -independent <br />     Curr. Genet. 28:108&#226;&#8364;&#8220;112.                                                               pathways mRNA turnover Trypanosoma brucei mitochondria. Mol. <br /> 15. Dobinson, K. F., M. Henderson, R. L. Kelley, R. A. Collins, A. M.                  Cell. Biol. 20:2308&#226;&#8364;&#8220;2316. <br />     Lambowitz. 1989. Mutations nuclear gene cyt-4 Neurospora crassa result       39. Min, J., R. M. Heuertz, H. P. Zassenhaus. 1993. Isolation charac- <br />     pleiotropic defects processing splicing mitochondrial RNAs.               terization NTP-dependent 3&#226;&#172;&#732;-exoribonuclease mitochondria of <br />     Genetics 123:97&#226;&#8364;&#8220;108.                                                                   Saccharomyces cerevisiae. J. Biol. Chem. 268:7350&#226;&#8364;&#8220;7357. <br /> 16. Dziembowski, A., M. Malewicz, M. Minczuk, P. Golik, A. Dmochowska,             40. Minczuk, M., J. Piwowarski, M. A. Papworth, K. Awiszus, S. Schalinski, A. <br />     P. P. Stepien. 1998. The yeast nuclear gene DSS1, codes putative           Dziembowski, A. Dmochowska, E. Bartnik, K. Tokatlidis, P. P. Stepien, and <br />     RNase II, necessary function mitochondrial degradosome            P. Borowski. 2002. Localisation human hSuv3p helicase mito- <br />     processing turnover RNA. Mol. Gen. Genet. 260:108&#226;&#8364;&#8220;114.                          chondrial matrix preferential unwinding dsDNA. Nucleic Acids <br /> 17. Dziembowski, A., J. Piwowarski, R. Hoser, M. Minczuk, A. Dmoshowska, M.                Res. 30:5074&#226;&#8364;&#8220;5086. <br />     Siep, H. van der Spek, L. Grivell, P. P. Stepien. 2003. The yeast mito-        41. Missel, A., A. E. Souza, G. No&#194;&#168;rskau, H. U. Go&#194;&#168;ringer. 1997. Disruption of <br />     chondrial degradosome, composition, interplay RNA helicase                 gene encoding novel mitochondrial DEAD-box protein Trypanosoma <br />     RNase activities role mitochondrial RNA metabolism. J. Biol.            brucei affects edited mRNAs. Mol. Cell. Biol. 17:4895&#226;&#8364;&#8220;4903. <br />     Chem. 278:1603&#226;&#8364;&#8220;1611.                                                               42. Panigrahi, A. K., S. P. Gygi, N. L. Ernst, R. P. Igo, Jr., S. S. Palazzo, A. <br /> 18. Feagin, J. E., D. P. Jasmer, K. Stuart. 1987. Developmentally regulated            Schnaufer, D. S. Weston, N. Carmean, R. Salavati, R. Aebersold, K. D. <br />     addition nucleotides apocytochrome b transcripts Trypanosoma              Stuart. 2001. Association novel proteins, TbMP52 TbMP48, with <br />     brucei. Cell 49:337&#226;&#8364;&#8220;345.                                                               Trypanosoma brucei RNA editing complex. Mol. Cell. Biol. 21:380&#226;&#8364;&#8220;389. <br /> 19. Feagin, J. E., J. M. Abraham, K. Stuart. 1988. Extensive editing        43. Panigrahi, A. K., A. Schnaufer, N. L. Ernst, B. Wang, N. Carmean, R. <br />     cytochrome c oxidase III Trypanosoma brucei. Cell 53:413&#226;&#8364;&#8220;422.                       Salavati, K. Stuart. 2003. Identification novel components of <br /> 20. Gagliardi, D., J. Kuhn, U. Spadinger, A. Brennicke, C. J. Leaver, S.               Trypanosoma brucei editosomes. RNA 9:484&#226;&#8364;&#8220;492. <br />     Binder. 1999. An RNA helicase (AtSUV3) present Arabidopsis thaliana          44. Pelletier, M., L. K. Read. 2003. RBP16 multifunctional gene regu- <br />     mitochondria. FEBS Lett. 458:337&#226;&#8364;&#8220;342.                                                  latory protein involved editing stabilization specific mitochondrial <br /> 21. Golik, P., T. Szczcepancek, E. Bartnik, P. P. Stepien, J. Lasowska. 1995.          RNAs Trypanosoma brucei. RNA 9:457&#226;&#8364;&#8220;468. <br />     The S. cer <br /> </body></html>