<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />       The F-Plasmid TraI Protein Contains Three Functional Domains <br />              Required for Conjugative DNA Strand Transfer <br />  <br />            The F-plasmid-encoded TraI protein, also known as DNA helicase I, is a bifunctional protein required for <br />          conjugative DNA transfer. The enzyme catalyzes two distinct but functionally related reactions required for the <br />          DNA processing events associated with conjugation: the site- and strand-specific transesterification (relaxase) <br />          reaction that provides the nick required <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> initiate strand transfer and a processive 5&#226;&#180;&#8226;-to-3&#226;&#180;&#8226; helicase reaction <br />          that provides the motive force for strand transfer. Previous studies have identified the relaxase domain, which <br />          encompasses the first &#226;&#172;&#402;310 amino acids of the protein. The helicase-associated motifs lie between amino acids <br />          990 and 1450. The function of the region between amino acids 310 and 990 and the region <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> amino acid 1450 <br />          to the C-terminal end is unknown. A protein lacking the C-terminal 252 amino acids (TraI&#226;&#338;&#172;252) was <br />          constructed and shown to have essentially wild-type levels of transesterase and helicase activity. In addition, <br />          the protein was capable of a functional interaction with other components of the minimal relaxosome. However, <br />          TraI&#226;&#338;&#172;252 was not able to support conjugative DNA transfer in genetic complementation experiments. We <br />          conclude that TraI&#226;&#338;&#172;252 lacks an essential C-terminal domain that is required for DNA transfer. We speculate <br />          this domain may be involved in essential protein-protein interactions with other components of the DNA <br />          transfer machinery. <br />  <br />  <br />    Conjugative DNA strand transfer is a highly conserved                   phodiester backbone at nic via a transesterification reaction <br /> mechanism for unidirectional transfer of genetic information               that leaves the initiator protein covalently bound on the 5&#226;&#172;&#732; end <br /> among bacteria of the same species, from one species to an-                of the cleaved DNA strand via a phosphotyrosyl linkage. The <br /> other and, in some instances, between kingdoms (for reviews,               DNA is subsequently unwound, through the action of a heli- <br /> see references 15, 22, 25, 30, 37, and 44). Importantly, this is           case or as a function of DNA replication extending from the <br /> the mechanism used by most plasmids and conjugative trans-                 newly formed 3&#226;&#172;&#732;-OH, to produce the ssDNA molecule that can <br /> posons to facilitate their spread throughout a bacterial popu-             be transferred to the recipient cell. In the case of plasmid DNA <br /> lation, and the same underlying mechanism is used to transfer              transfer, the 3&#226;&#172;&#732;-OH subsequently acts as a nucleophile to re- <br /> T-DNA from Agrobacterium tumefaciens to plant cells (23, 45).              verse the original transesterification. This restores the integrity <br />    The process of conjugative DNA transfer (CDT) is initiated              of the ssDNA transferred to the recipient and releases the <br /> with the formation of a stable mating pair between donor and               bound initiator protein. The action of the initiator protein at <br /> recipient to establish the close cell-cell contact required for            oriT is governed by several auxiliary proteins that act to alter <br /> physical transfer of single-stranded DNA (ssDNA) from one                  the architecture of the DNA molecule in the vicinity of nic to <br /> cell to another. This is followed by the production of a site- and         allow binding of the initiator protein. These auxiliary proteins <br /> strand-specific nick at a locus called nic within the origin of            are both host encoded and plasmid encoded in origin (7, 22, <br /> transfer (oriT) or at the ends of T-DNA and the unwinding/                 29). <br /> replication of the duplex DNA molecule to provide the ssDNA <br />                                                                               A small subset of the conjugative initiator proteins contain a <br /> that enters the recipient. Within the recipient cell, host en- <br />                                                                            second domain harboring a functional DNA helicase (16, 19, <br /> zymes (primarily) convert the transferred ssDNA into double- <br />                                                                            39). These proteins include, among others, the TrwC protein <br /> stranded DNA that either circularizes to form a plasmid or is <br />                                                                            from plasmid R388 (16, 24) and the TraI protein from the F <br /> recombined into the recipient chromosome (for reviews, see <br />                                                                            plasmid (6, 10, 39). Both of these proteins have been shown to <br /> references 15 and 44). <br />                                                                            be bifunctional proteins with a transesterase (relaxase) activity <br />    Although reasonably well understood at this &#226;&#8364;&#339;macroscopic&#226;&#8364;&#65533; <br />                                                                            encoded in the first &#226;&#172;&#402;300 amino acids of the protein and a <br /> level, the molecular details of the process by which DNA is <br /> transferred from donor to recipient are still being resolved.              helicase residing in the C-terminal portion of the protein (6, <br /> Key players are the proteins that initiate the physical transfer           24). Moreover, in both cases, both biochemical activities have <br /> of ssDNA, the conjugative initiator proteins (7, 22, 30). These            been shown to be required for DNA transfer. In the case of <br /> proteins introduce a site- and strand-specific break in the phos-          F-plasmid-encoded TraI, in addition to these two defined func- <br />                                                                            tional domains, there are additional regions of the protein with <br />                                                                            no known function (Fig. 1). For example, there is a region <br />                                                                            between the C-terminal end of the active transesterase domain <br />   * Corresponding author. Mailing address: Department of Biology, <br />                                                                            (amino acid 306) and the beginning of the helicase domain <br /> CB#3280, Coker Hall, University of North Carolina, Chapel Hill, NC <br /> 27599-3280. Phone: (919) 962-0005. Fax: (919) 962-1625. E-mail:            (amino acid 990 as defined by conserved helicase-associated <br /> smatson@bio.unc.edu.                                                       motifs) that has no known function. However, previous studies <br />  <br />                                                                      697 <br />  698      MATSON AND RAGONESE                                                                                                                        J. BACTERIOL. <br />  <br />  <br />  <br />  <br />   FIG. 1. Diagram of TraI and ClustalW alignments. (A) Schematic diagram of TraI. The relaxase domain is wholly contained within the region <br /> spanned by residues 1 to 310 (6, 11). The helicase-associated motifs are located within the region spanned by residues 990 to 1450. The functional <br /> role of the central region (residues 310 to 990) and the C-terminal region (residues 1450 to 1756) is unknown. (B) ClustalW alignment of selected <br /> sequences obtained from a BLAST search using TraI (NP_862951.1) as the query sequence. The relaxase domain is shown in red, the central region <br /> with unknown function is shown in green, and the region containing the helicase-associated motifs is in blue. The sequences used in the alignment <br /> shown are TraI (F plasmid), TraI (plasmid R100) (NP_052981.1), TraI (Salmonella enterica serovar Typhimurium) (AAM90727.1), oriT nicking/ <br /> unwinding protein fragment (Shigella flexneri) (NP_085415.1), putative conjugative transfer protein TraI (Vibrio vulnificus YJ016) (NP_932226.1), <br /> putative DNA helicase (TraI) (Photobacterium profundum SS9) (YP_015476.1), putative TraC protein (Pseudomonas putida) (AAP57243.1), TrwC <br /> (Xanthomonas citri) (NP_942625.1), TrwC protein (IncW plasmid R388) (S43878), and TraI [IncN plasmid R46] (NP_511201.1). <br />  <br />  <br /> have suggested that the active helicase begins before amino                      (DE3) (Rifr)] with a pET-derived expression plasmid. Cells were grown at 37&#194;&#176;C <br /> acid 348 (6), so this region may play a role in the processive                   in Luria-Bertani (LB) medium to an optical density at 600 nm of &#226;&#172;&#402;1.0, and <br />                                                                                  protein expression was induced by the addition of isopropyl-&#226;&#65533;&#164;-D-thiogalactopy- <br /> helicase reaction catalyzed by TraI. In addition, there are ap- <br />                                                                                  ranoside (IPTG) to a final concentration of 0.5 mM. Growth was continued for <br /> proximately 300 amino acids at the C-terminal end of the                         3 h at 37&#194;&#176;C, and cells were harvested by centrifugation. After cell lysis, a cleared <br /> protein with no known function.                                                  lysate was prepared, nucleic acids were precipitated with Polymin P, and, after a <br />    In this report, we have investigated the role of the C-termi-                 series of ammonium sulfate backwashes, the proteins were purified with phos- <br /> nal end of F-TraI, using a truncation mutant lacking the last                    phocellulose, MonoQ, and Superose 6 chromatography columns. The TraI con- <br />                                                                                  centration was determined by measuring A280 with an extinction coefficient for <br /> 252 amino acid residues. This protein (TraI&#226;&#338;&#172;252) is an active <br />                                                                                  TraI of 129,795 M&#226;&#171;&#186;1 cm&#226;&#171;&#186;1. The TraI&#226;&#338;&#172;252 concentration was determined with a <br /> transesterase/relaxase that binds and catalyzes a transesterifi-                 Bio-Rad protein assay with purified TraI protein as the standard. Protein con- <br /> cation reaction at the F plasmid oriT as well as an active DNA                   centrations are expressed as equivalents of protein monomers. IHF and TraY <br /> helicase. In addition, the protein interacts with the other com-                 were purified as described previously (18, 29). <br /> ponents of the minimal relaxosome, integration host factor                          DNA substrates. Plasmid pBSoriT has been previously described (26) and <br />                                                                                  contains the F-plasmid oriT region from the BglII site to the SalI site cloned in <br /> (IHF), and the F-plasmid-encoded TraY protein. Thus, the <br />                                                                                  compatible sites in pBluescript. Plasmid DNA was purified by double banding on <br /> protein is fully competent for the two known biochemical ac-                     CsCl gradients in the presence of ethidium bromide (17). This was necessary to <br /> tivities of TraI. However, genetic complementation studies re-                   reduce the background of nicked DNA molecules to an acceptable level. Partial <br /> veal the inability of this protein to support DNA transfer. We                   duplex unwinding substrates were prepared essentially as described previously <br /> conclude that TraI contains a third functional domain at the                     (21). Briefly, a 91-nucleotide oligonucleotide or the complementary strand of an <br />                                                                                  849-bp M13mp18 replicative form DNA HaeIII restriction fragment was an- <br /> C-terminal end of the protein that is required for DNA strand <br />                                                                                  nealed to its complementary sequence on purified M13mp18 ssDNA at a molar <br /> transfer. Further, we hypothesize that the C-terminal end of                     ratio of 1:1. The 3&#226;&#172;&#732; end of the annealed DNA strand was extended with E. coli <br /> TraI may play a critical role in the next step in the process of                 DNA polymerase I (large fragment) and [&#226;&#65533;&#163;-32P]dCTP. The final <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> of the <br /> DNA transfer&#226;&#8364;&#8221;interaction with the export machinery at the                        oligonucleotide was 93 or 851 nucleotides. The preparation was phenol-chloro- <br /> interior surface of the donor cell.                                              form extracted and further purified with a Biogel A5 M column equilibrated with <br />                                                                                  10 mM Tris-HCl (pH 7.5), 1 mM EDTA, and 100 mM NaCl. The void volume <br />                         MATERIALS AND METHODS                                    fractions were pooled and used directly in DNA unwinding reaction mixtures. <br />   Protein purification. Wild-type (full-length) TraI protein and TraI&#226;&#338;&#172;252 were   The DNA concentration of the pooled fractions is estimated to be 20 &#226;&#65533;&#174;M <br /> expressed and purified as described previously (28). Briefly, each protein was   (expressed as DNA-Pi). <br /> expressed in Escherichia coli HMS174(DE3) [F&#226;&#171;&#186; recA1 hsdR (rK12&#226;&#171;&#186; mK12&#226;&#171;&#185;)              The gene expressing TraI&#226;&#338;&#172;252 was constructed by using PCR to amplify the <br />  VOL. 187, 2005                                                                                              FUNCTIONAL DOMAINS IN F-TraI                       699 <br />  <br /> traI gene on pET11d-traI (28). The upstream primer 5&#226;&#172;&#732;-TTTTTCATATGATG               The mating mixtures were then vigorously vortexed to disrupt mating pairs, and <br /> AGTATTGCGC-3&#226;&#172;&#732; contained an NdeI site (underlined), and the downstream              10-fold serial dilutions were prepared in phosphate-buffered saline for analysis. <br /> primer 5&#226;&#172;&#732;-TTTGGATCCTCACTACGCCTGACGGAGAACAGC-3&#226;&#172;&#732; contained                           Appropriate dilutions were plated onto LB plates containing streptomycin and <br /> a BamHI site (underlined). In addition, the downstream PCR primer contained        tetracycline to counterselect donors and unmated recipients while selecting for <br /> two stop codons immediately following codon 1504 in the traI sequence to           transconjugants. Aliquots of the unmated donor and recipient cultures were <br /> produce the C-terminal truncation of the traI gene. The amplified fragment of      subjected to 10-fold serial dilution and plated onto LB plates containing the <br /> traI was digested with NdeI and BamHI and inserted into the expression vector      appropriate antibiotics to determine viable donor cell count and viable recipient <br /> pET3c, digested with the same two enzymes. The gene was sequenced in its           cell count. Mating frequency was calculated as the number of transconjugants <br /> entirety to ensure the absence of unintended mutations. The protein was ex-        per 100 viable donor cells. <br /> pressed and purified as indicated above. Oligonucleotides were synthesized by <br /> IDT Inc. Each oligonucleotide was purified on a 20% denaturing polyacrylamide <br /> gel. Plasmids containing point mutations in traI (pET11d-traIK998M and <br />                                                                                                                     RESULTS <br /> pET11d-traIY16V) were constructed by the QuikChange mutagenesis procedure             The F-plasmid-encoded TraI protein, also known as DNA <br /> (Stratagene) with the appropriate oligonucleotide primers. The construction of <br /> pET11d-traIK998M has been described previously (28), and the construction of <br />                                                                                    helicase I (3), is essential for CDT (41). Recently, we and <br /> pET11d-traIY16V will be described elsewhere.                                       others have shown this protein to be capable of catalyzing two <br />    DNA helicase assays. DNA unwinding reaction mixtures (typically 20 &#226;&#65533;&#174;l)          distinct but functionally related biochemical reactions (6, 10, <br /> contained 25 mM Tris-HCl (pH 7.5), 20 mM NaCl, 3 mM MgCl2, 5 mM &#226;&#65533;&#164;-mer-             39). The protein is a site- and strand-specific transesterase that <br /> captoethanol, approximately 2 &#226;&#65533;&#174;M (DNA-Pi) partial duplex DNA substrate, and        makes the site- and strand-specific strand break required to <br /> 2 mM ATP. Reaction mixtures were assembled at room temperature, and the <br /> reaction was initiated by the addition of enzyme. Incubation was at 37&#194;&#176;C for 5 <br />                                                                                    initiate CDT (26, 33, 38). This reaction is also referred to as a <br /> min. The reactions were quenched with 10 &#226;&#65533;&#174;l of stop solution (37.5% glycerol, 50   relaxation reaction since the product of the reaction (after <br /> mM EDTA, 0.05% xylene cyanol, 0.05% bromophenol blue, 0.3% sodium do-              protein denaturation) is a relaxed plasmid (7). As a result of <br /> decyl sulfate [SDS]), and the reaction products were resolved on a nondenatur-     the transesterification, TraI is covalently bound to the 5&#226;&#172;&#732; end of <br /> ing polyacrylamide gel (10% for the 93-bp substrate and 8% for the 851-bp <br />                                                                                    the nicked DNA strand that will ultimately be transferred into <br /> substrate; 20:1 cross-linking ratio) run at 200 V for 2 h (93-bp substrate) or <br /> overnight (851-bp substrate). The results were visualized with a PhosphorImager    the recipient cell (27). TraI is also a processive 5&#226;&#172;&#732;-to-3&#226;&#172;&#732; DNA <br /> and analyzed with ImageQuant software (Molecular Dynamics).                        helicase (1, 2, 21). Both of these catalytic activities are essential <br />    Duplex DNA relaxation assays. Assays were done in a manner slightly modi-       for CDT (6, 28). It is known that TraI binds the F-plasmid oriT <br /> fied from those previously described (26). A typical reaction mixture (16 &#226;&#65533;&#174;l)      sequence, with the aid of the F-plasmid-encoded TraY protein <br /> contained 7 nM supercoiled pBSoriT (or pBS) DNA, 40 mM Tris-HCl (pH, 7.5), <br /> 4 mM MgCl2, 15% glycerol, and the indicated concentration of either TraI or <br />                                                                                    and the host-encoded IHF, to form the relaxosome that pro- <br /> TraI&#226;&#338;&#172;252. Reaction mixtures were assembled at room temperature, the reaction       duces the site- and strand-specific nick at oriT (18, 29). It is <br /> was initiated by the addition of enzyme, and the mixtures were incubated at 37&#194;&#176;C   presumed that the helicase activity of the protein then unwinds <br /> for 20 min. Reactions were stopped by the addition of proteinase K (Boehringer-    the plasmid for transfer into the recipient. This assumption is <br /> Mannheim Biochemicals) and SDS to final concentrations of 0.2 mg/ml and <br />                                                                                    based on two facts: (i) a single strand of the F plasmid is <br /> 0.1%, respectively, and the mixtures were allowed to incubate at 37&#194;&#176;C for an <br /> additional 20 min. The products were resolved on 0.8% agarose gels and visu-       transferred to the recipient, and (ii) the helicase activity of <br /> alized by ethidium bromide staining (0.5 &#226;&#65533;&#174;g/ml). For the duplex DNA relaxation     TraI is essential for CDT (28). <br /> assays involving the addition of IHF and TraY, the reaction mixture was altered       Interestingly, TraI contains significant regions that do not <br /> to include NaCl at a final concentration of 75 mM as previously described (29).    appear to correlate with the known biochemical activities of <br />    Oligonucleotide cleavage assays. Oligonucleotide cleavage assays were similar <br /> to those used previously to analyze the cleavage reaction catalyzed by TraI from <br />                                                                                    the protein (Fig. 1). Biochemical studies have demonstrated <br /> the IncP&#226;&#65533;&#163; plasmid RP4 (31). The complete reaction mixture (10 &#226;&#65533;&#174;l) contained 50     the transesterase domain to be wholly contained within the <br /> mM Tris-HCl (pH 7.5), 150 mM NaCl, 6 mM MgCl2, 20% glycerol, 1 pmol of             first 310 amino acids of the protein (6, 38). Recent structural <br /> 5&#226;&#172;&#732;-end-labeled 22-mer (5&#226;&#172;&#732;-TTTGCGTGGGGTGTGGTGCTTT-3&#226;&#172;&#732;), and 1 pmol                   studies are consistent with this conclusion (11). The conserved <br /> of TraI or TraI&#226;&#338;&#172;252 (unless otherwise stated). Reaction mixtures were assem- <br />                                                                                    motifs associated with superfamily I DNA helicases are found <br /> bled at room temperature and incubated at 37&#194;&#176;C for 20 min. Reactions were <br /> stopped by the addition of SDS to 0.2%, and incubation was continued at 37&#194;&#176;C       in the region of the protein from amino acid 990 to amino acid <br /> for 10 min. Ten microliters of 85% formamide&#226;&#8364;&#8220;50 mM EDTA&#226;&#8364;&#8220;0.1% dyes was              1450. The large region between amino acids 310 and 990 and <br /> added to each reaction mixture, and the products were denatured at 100&#194;&#176;C for 3     the region beyond amino acid 1450 are not defined in terms of <br /> min and analyzed on a 16% polyacrylamide&#226;&#8364;&#8220;8 M urea denaturing gel. The gels         function within this large protein. However, it should be noted <br /> were electrophoresed at 25 W in 1 &#226;&#171;&#187; TBE (90 mM Tris-borate, 2 mM EDTA) <br />                                                                                    that previous studies suggested that the residues between <br /> and visualized with a PhosphorImager (Molecular Dynamics). Markers were <br /> prepared as described previously (36).                                             amino acids 310 and 990, in addition to the region encompass- <br />    Strand transfer reactions. Strand transfer reactions (31) were performed in a   ing the helicase-associated motifs, were important for helicase <br /> manner similar to the oligonucleotide cleavage assay, except after the 20-min      activity (6). More recent work (E. G. Gibbons and S. W. <br /> incubation, 1 pmol of a second unlabeled oligonucleotide (5&#226;&#172;&#732;-CTTGTTTTTG            Matson, unpublished data) indicates that the region of the <br /> CGTGGGGTGTGGTGCTTTTG&#226;&#8364;&#8220;3&#226;&#172;&#732;) of different length containing the F-plas- <br /> mid nic site was added to the reaction mixture and incubation was continued at <br />                                                                                    protein from amino acid 903 to amino acid 1756 does exhibit a <br /> 37&#194;&#176;C for 1 h. The reaction was stopped and analyzed by the procedure described     very weak DNA helicase activity and a poor ATPase activity <br /> above.                                                                             that went undetected in the earlier studies. It is apparent that <br />    Genetic assays. The liquid mating assay protocol was followed as previously     the region from amino acid 310 to amino acid 990 is critical for <br /> described (28). Briefly, the donor strain (JS10) was HMS174 [F&#226;&#171;&#186; recA1 hsdR <br />                                                                                    wild-type levels of helicase and ATPase activity. However, the <br /> (rK12&#226;&#171;&#186; mK12&#226;&#171;&#185;) (Rifr)] containing pOX38T, a mini-F plasmid fully competent for <br /> conjugative DNA transfer, or HMS174 containing pOX38T&#226;&#338;&#172;traI (JS11), a de-           region harboring the helicase-associated motifs can support a <br /> rivative of pOX38T lacking the traI gene (28), and the appropriate complement-     basal helicase activity. Additional functional roles for the re- <br /> ing plasmid. The recipient strain was HB101 or JS4 (Strr). Donor and recipient     gion of the protein from residue 310 to residue 903 remain <br /> strains were diluted 1:50 into LB medium from saturated overnight cultures         undefined. <br /> grown under antibiotic selection and allowed to grow to mid-log phase in the <br /> absence of selection at 37&#194;&#176;C. Donors and recipients were mixed at a volume ratio <br />                                                                                       The aim of these studies was to examine the functional role <br /> of 1 donor to 9 recipients and incubated at 37&#194;&#176;C for 5 min. The mating mixtures    of the C-terminal &#226;&#172;&#402;250 amino acids of TraI. This region of the <br /> were then diluted 1:10 in prewarmed medium and incubated for 60 min at 37&#194;&#176;C.       protein is conserved in several, but not all, conjugative initiator <br />  700     MATSON AND RAGONESE                                                                                                                   J. BACTERIOL. <br />  <br />  <br />                                                                                     TABLE 1. Conjugative DNA transfer complementationa <br />                                                                                                                                        Transfer frequency <br />                                                                            Strain                Relevant genotype                   (no. of transconjugants/ <br />                                                                                                                                            100 donors) <br />  <br />                                                                            JS10 HMS174/pOX38T                                                   0.2 <br />  <br />                                                                            JS11 HMS174/pOX38T&#226;&#338;&#172;traI                                         &#226;&#172;&#65533;2 &#226;&#171;&#187; 10&#226;&#171;&#186;6b <br />                                                                                 HMS174/pOX38T&#226;&#338;&#172;traI/pET11c-traI                                0.03 <br />                                                                                 HMS174/pOX38T&#226;&#338;&#172;traI/pET11d-traIY16V                         &#226;&#172;&#65533;1 &#226;&#171;&#187; 10&#226;&#171;&#186;5b <br />                                                                                 HMS174/pOX38T&#226;&#338;&#172;traI/pET11d-traIK998M                        &#226;&#172;&#65533;7 &#226;&#171;&#187; 10&#226;&#171;&#186;4b <br />                                                                                 HMS174/pOX38T&#226;&#338;&#172;traI/pET3c-traI&#226;&#338;&#172;252                          &#226;&#172;&#65533;8 &#226;&#171;&#187; 10&#226;&#171;&#186;6b <br />                                                                              a <br />                                                                                Complementation assays were performed as described in Materials and <br />                                                                            Methods, using the indicated donor cell strain and JS4 as the recipient cell strain. <br />                                                                            All donor strains are HMS174 [F&#226;&#171;&#186;recA1 hsdR (rK12&#226;&#171;&#186; mK12&#226;&#171;&#185;) (Rifr)] containing <br />    FIG. 2. SDS-polyacrylamide gel analysis of the purification and <br />                                                                            the indicated plasmid(s). <br /> expression of TraI and TraI&#226;&#338;&#172;252. (A) One microgram each of purified          b <br />                                                                                No transconjugants were detected in any experiment. This value represents <br /> TraI (lane 2) and purified TraI&#226;&#338;&#172;252 (lane 3) was resolved on an 8%         the sensitivity of the complementation assay and was determined by assuming <br /> polyacrylamide gel run in the presence of SDS and stained with Coo-        there was a single transconjugant in the experiment. The actual transfer fre- <br /> massie brilliant blue. High-range molecular mass standards (Bio-Rad)       quency may be much lower. <br /> with sizes in kilodaltons indicated to the left are shown in lane 1. <br /> (B) Whole-cell lysates of HMS174/pOX38T (lane 1), HMS174/ <br /> pOX38T&#226;&#338;&#172;traI (lane 2), HMS174/pET11d-traI (lane 3), and HMS174/ <br /> pET3c-traI&#226;&#338;&#172;252 (lane 4) were resolved on an 8% polyacrylamide gel             The level of expression of TraI and TraI&#226;&#338;&#172;252 determined <br /> run in the presence of SDS, transferred to nitrocellulose, and probed <br />                                                                            with expression plasmids in HMS174 was evaluated and com- <br /> with anti-TraI antisera. Prestained molecular mass standards (Bio- <br /> Rad) with sizes in kilodaltons are shown on the right. The slight          pared with the level of TraI expression from its native pro- <br /> difference in migration between TraI in lane 1 and TraI in lane 3 is due   moter on plasmid pOX38T (mini-F plasmid that supports <br /> to a gel artifact.                                                         CDT). Expression of both proteins in HMS174 results from the <br />                                                                            adventitious use of an E. coli promoter on the pET expression <br />                                                                            plasmid since this strain lacks the lambda lysogen (DE3) re- <br /> proteins that harbor both a relaxase domain and a helicase                 quired for expression of T7 RNA polymerase and IPTG-de- <br /> domain (Fig. 1B). It is interesting to note that this region is not        pendent induction of TraI&#226;&#338;&#172;252. The expression of TraI and <br /> present in the IncW plasmid initiator protein TrwC, which, in              TraI&#226;&#338;&#172;252 in HMS174 (Fig. 2B) was evaluated by Western <br /> addition, to F-TraI, has both relaxase and helicase activity and           blotting (i) to ensure the protein was stably expressed in strains <br /> has been extensively characterized. To examine the functional              to be used for genetic complementation studies (see below) <br /> role of the last 252 amino acids, a truncated protein lacking the          and (ii) to ensure the protein was not significantly overex- <br /> C-terminal 252 amino acids (TraI&#226;&#338;&#172;252) was expressed, puri-                 pressed under these conditions. It is clear that both TraI and <br /> fied, and characterized in both genetic and biochemical assays.            TraI&#226;&#338;&#172;252 are stably expressed in HMS174 (Fig. 2B, lanes 3 and <br />    TraI&#226;&#338;&#172;252 expression and purification. The traI&#226;&#338;&#172;252 gene                 4) and that expression is elevated only slightly relative to ex- <br /> was constructed as described in Materials and Methods and                  pression of TraI from pOX38T (Fig. 2B, lane 1). It is also clear <br /> lacks the C-terminal 252 amino acid residues associated with               that TraI is not expressed in the pOX38T&#226;&#338;&#172;traI derivative that <br /> wild-type TraI. The protein contains the relaxase domain                   lacks the traI gene (Fig. 2B, lane 2). <br /> (amino acids 1 to 310) as well as all seven of the helicase motifs            The C-terminal end of TraI is required for conjugative DNA <br /> associated with superfamily I DNA helicases. The protein also              transfer. Using genetic complementation assays, we have <br /> contains approximately 60 residues beyond helicase motif VI.               shown that both the transesterase activity and the helicase <br /> The truncation site was chosen based on the distance between               activity of TraI are essential for DNA transfer (6, 28). These <br /> other superfamily I DNA helicases (i.e., UvrD, Rep, and he-                experiments utilized a mini-F plasmid (pOX38T) that ex- <br /> licase IV) and the C terminus of the protein and a prediction              presses all the genes required for CDT and its derivative <br /> of secondary structure that indicated the possibility of a loop in         (pOX38T&#226;&#338;&#172;traI) that contains a traI deletion/insertion. Since <br /> this region.                                                               TraI is required for CDT, pOX38T&#226;&#338;&#172;traI cannot be transferred <br />    TraI and TraI&#226;&#338;&#172;252 were expressed in both HMS174 and                     to a recipient cell unless a complementing plasmid expressing <br /> HMS174(DE3) (Fig. 2). Expression in HMS174(DE3) was the                    TraI (pET11d-TraI) is also provided (Table 1, compare lines 2 <br /> result of IPTG-dependent induction, and these cells were used              and 3). The complemented transfer frequency is somewhat <br /> for purification of the protein as described in Materials and              lower than the transfer frequency obtained with cells that con- <br /> Methods. The TraI&#226;&#338;&#172;252 protein was purified by the same pro-                tain pOX38T (Table 1, line 1) because the recipient in a <br /> cedure used for purification of the full-length protein and is             pOX38T transfer event becomes a donor and can continue to <br /> shown in Fig. 2A (lane 3). The purified protein was truncated              transfer DNA to other recipients, while a recipient in a com- <br /> relative to the wild-type protein, as expected, and was approx-            plemented transfer event does not become a donor since the <br /> imately 90% homogeneous and was fully soluble. The rela-                   plasmid it receives (pOX38T&#226;&#338;&#172;traI) cannot support further <br /> tively minor contaminant that migrates faster than TraI&#226;&#338;&#172;252 is             transfer. <br /> a degradation product, as evidenced by its reaction with anti-                As expected, the wild-type TraI protein, expressed from the <br /> sera directed against TraI protein on Western blots (data not              pET11d plasmid (Table 1, line 3) complements DNA transfer. <br /> shown). This protein was used for the biochemical studies                  As indicated above, expression in this case is due to an adven- <br /> described below.                                                           titious promoter on the plasmid and is not due to induction of <br />  VOL. 187, 2005                                                                             FUNCTIONAL DOMAINS IN F-TraI                  701 <br />  <br />  <br /> T7 RNA polymerase. Therefore, expression is relatively low <br /> and we estimate less than threefold greater than expression <br /> from the mini-F plasmid (Fig. 2B). Thus, high-level expression <br /> of TraI is not required for genetic complementation. A point <br /> <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>mutation</span> in the active site tyrosine of the transesterase domain <br /> (TraIY16V) or a <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>point mutation</span> in helicase motif I of the <br /> helicase domain (TraIK998M) abolishes genetic complemen- <br /> tation as reported previously (6, 28; unpublished results). In- <br /> terestingly, deletion of the C-terminal 252 amino acids also <br /> abolishes genetic complementation, suggesting that this region <br /> of the protein is essential for some aspect of the DNA transfer <br /> event. It is important to note that this truncation leaves all <br /> helicase-associated motifs intact. <br />    TraI&#226;&#338;&#172;252 maintains essentially wild-type helicase and <br /> transesterase activity. To eliminate the possibility that <br /> TraI&#226;&#338;&#172;252 was unable to support CDT due to an impact on <br /> either helicase activity or relaxase activity, the purified protein <br /> was evaluated in biochemical assays for both activities. The <br /> helicase activity of the protein was tested with a 93-bp partial <br /> duplex DNA substrate and an 851-bp partial duplex DNA <br /> substrate (Fig. 3). Helicase activity measured with a 93-bp <br /> partial duplex substrate was reduced slightly compared to the <br /> activity of the full-length protein at low protein concentrations. <br /> However, at higher protein concentrations, the activity of <br /> TraI&#226;&#338;&#172;252 was virtually indistinguishable from that of full- <br /> length protein. Similarly, TraI&#226;&#338;&#172;252-catalzyed helicase activity <br /> on the longer partial duplex substrate (851 bp) was reduced <br /> relative to that of full-length protein at low protein concentra- <br /> tions but was only slightly lower than that of full-length protein <br /> at higher protein concentrations. Importantly, the protein ex- <br /> hibits robust helicase activity on both substrates, suggesting the <br /> failure to complement in genetic assays is not due to a loss of <br /> DNA helicase activity. <br />    The transesterase activity of the protein was also evaluated <br /> in a series of biochemical activity assays. Previous studies have <br /> shown that TraI catalyzes the &#226;&#8364;&#339;nicking&#226;&#8364;&#65533; of a plasmid containing          FIG. 3. DNA helicase activity of TraI and TraI&#226;&#338;&#172;252. DNA unwind- <br /> its cognate oriT sequence (7, 26, 33). This reaction is not a         ing assays using a 93-bp partial duplex substrate (A) or an 851-bp <br /> classical endonuclease cleavage reaction but rather a transes-        partial duplex substrate (B) were performed as described in Materials <br />                                                                       and Methods, using the indicated amount of TraI (F) or TraI&#226;&#338;&#172;252 (E). <br /> terification in which TraI becomes covalently attached to the 5&#226;&#172;&#732;      Reaction mixtures were incubated for 10 min at 37&#194;&#176;C, and the products <br /> end of the cleaved DNA strand via a phosphotyrosyl linkage.           were resolved on a native polyacrylamide gel. The fraction of the <br /> This reaction does not require the participation of the other         substrate unwound was calculated by the formula [(U &#226;&#171;&#186; Bu)/(S &#226;&#171;&#186; Bs &#226;&#171;&#185; <br /> components of the relaxosome (F-plasmid-encoded TraY pro-             U &#226;&#171;&#186; Bu)] &#226;&#171;&#187; 100, where U represents the DNA in the product band, S <br /> tein and IHF), provided the reaction is done at low ionic             represents DNA in the substrate band and Bu and Bs represent back- <br />                                                                       ground levels of DNA at the position of the product (Bu) or the <br /> strength. Under these conditions, the plasmid must be nega-           substrate (Bs) determined from reaction mixtures containing no pro- <br /> tively supercoiled (26), with negative supercoiling introducing       tein that were either incubated as described above or incubated at 95&#194;&#176;C <br /> sufficient ssDNA character to the DNA to allow binding and            for 5 min to denature the substrate. <br /> transesterification by TraI. <br />    TraI&#226;&#338;&#172;252 was compared with full-length TraI protein in a <br /> plasmid nicking assay. A titration of TraI (Fig. 4A upper) and        tration. The transesterase reaction catalyzed by TraI is isoen- <br /> TraI&#226;&#338;&#172;252 (Fig. 4A, lower) clearly demonstrated that both pro-         ergetic and reversible (7). One reason the nicking reaction fails <br /> teins were capable of nicking a plasmid containing oriT at            to go to completion is the fact that the reverse reaction (liga- <br /> similar protein concentrations. The reaction was specific for         tion of the nicked strand) is in equilibrium with the forward <br /> DNA containing the F-plasmid oriT, since a plasmid lacking            (nicking) reaction. It is possible that this equilibrium is altered <br /> this region was not nicked by either protein (data not shown).        somewhat in reaction mixtures containing the TraI&#226;&#338;&#172;252 pro- <br /> Quantitation of the nicking activity (Fig. 4B) indicated that         tein, resulting in a lower fraction of the plasmid in the nicked <br /> TraI&#226;&#338;&#172;252 was as active as the full-length protein at low con-         form at higher protein concentrations. Nonetheless, it is clear <br /> centrations and only slightly less active at higher concentra-        the truncated protein retained essentially wild-type levels of <br /> tions. The reduction in activity at higher protein concentra-         relaxase activity and this activity is specific for the F-plasmid <br /> tions may reflect a shift in the equilibrium of the transesterase     oriT. <br /> reaction with the truncated protein at a high protein concen-            As indicated above, the nicking reaction is, in fact, a trans- <br />  702     MATSON AND RAGONESE                                                                                                      J. BACTERIOL. <br />  <br />  <br />                                                                            nucleotide fragment. It is evident that both full-length TraI <br />                                                                            protein and TraI&#226;&#338;&#172;252 cleave the oligonucleotide at the same <br />                                                                            location and with nearly equal efficiency. The 8-mer is co- <br />                                                                            valently bound to TraI (TraI&#226;&#338;&#172;252) as a result of the transes- <br />                                                                            terification and is not observed since it is not radioactively <br />                                                                            labeled. Protein titrations suggest that the full-length protein is <br />                                                                            somewhat more active than TraI&#226;&#338;&#172;252 at high protein concen- <br />                                                                            trations (Fig. 5C). Again, this may be a manifestation of a shift <br />                                                                            in the equilibrium of the transesterase reaction. <br />                                                                               As indicated above, the transesterase reaction is reversible <br />                                                                            and a strand transfer reaction has been used to demonstrate <br />                                                                            this reversibility (31, 36). In this reaction, TraI protein is incu- <br />                                                                            bated with a 5&#226;&#172;&#732;-end-labeled oligonucleotide containing the nic <br />                                                                            site (Fig. 6A; 22-mer) to allow cleavage to occur. Subsequently, <br />                                                                            an unlabeled oligonucleotide of different length that also con- <br />                                                                            tains the nic site (Fig. 6A; 30-mer) is added. The reversibility of <br />                                                                            the transesterase reaction is demonstrated by the formation of <br />                                                                            &#226;&#8364;&#339;recombinant&#226;&#8364;&#65533; products when the labeled DNA fragment re- <br />                                                                            sulting from cleavage of the 22-mer (14-mer) is ligated to the <br />                                                                            10-mer that is part of the covalent protein-DNA complex re- <br />                                                                            sulting from cleavage of the 30-mer. In the experiment shown <br />                                                                            in Fig. 6, the labeled oligonucleotide was 22 nucleotides in <br />                                                                            length, the unlabeled oligonucleotide was 30 nucleotides in <br />                                                                            length, and the expected recombinant product is 24 nucleo- <br />                                                                            tides in length. It is clear that both TraI protein and TraI&#226;&#338;&#172;252 <br />                                                                            catalyze an efficient exchange reaction, as evidenced by the <br />                                                                            appearance of the expected recombinant product. We con- <br />                                                                            clude that TraI&#226;&#338;&#172;252 is a fully functional transesterase capable <br />                                                                            of both cleavage and ligation at the nic sequence. <br />                                                                               Finally, we tested the ability of TraI&#226;&#338;&#172;252 to interact with the <br />                                                                            other components of the relaxosome in a relaxosome recon- <br />                                                                            stitution experiment. Previous studies have shown that TraI <br />                                                                            assembles with the host-encoded IHF and the F-plasmid-en- <br />                                                                            coded TraY to form a relaxosome at oriT (18, 29). Importantly, <br />                                                                            the addition of both IHF and TraY is required, and together <br />                                                                            these two proteins markedly stimulate the nicking reaction <br />                                                                            catalyzed by TraI. We considered the possibility that the defect <br />                                                                            in CDT observed with TraI&#226;&#338;&#172;252 might be due to its inability to <br />                                                                            form a fully functional relaxosome with IHF and TraY. To test <br />   FIG. 4. Relaxase activity of TraI and TraI&#226;&#338;&#172;252. (A) The relaxase <br /> activity assays using pBSoriT DNA were performed as described in <br />                                                                            this directly, relaxosomes were assembled with TraI and <br /> Materials and Methods. A representative experiment is shown. The           TraI&#226;&#338;&#172;252 and tested for nicking activity (Fig. 7). As expected, <br /> upper half of the gel represents a titration of TraI, and the lower half   neither TraI nor TraI&#226;&#338;&#172;252 exhibited significant nicking activity <br /> represents a titration of TraI&#226;&#338;&#172;252. Lane 1, no-enzyme control; lane 2,     under the conditions of this assay (Fig. 7, lanes 2 and 3), which <br /> 3.9 nM TraI (TraI&#226;&#338;&#172;252); lane 3, 7.8 nM TraI (TraI&#226;&#338;&#172;252); lane 4, 15.6       includes a significantly higher NaCl concentration than the <br /> nM TraI (TraI&#226;&#338;&#172;252); lane 5, 31.3 nM TraI (TraI&#226;&#338;&#172;252); lane 6, 62.5 nM <br /> TraI (TraI&#226;&#338;&#172;252); and lane 7, 125 nM TraI (TraI&#226;&#338;&#172;252). (B) The gel           relaxase assay shown in Fig. 4. The addition of both IHF and <br /> shown in panel A was quantified as described previously (29) for TraI      TraY markedly stimulated the TraI-catalyzed nicking reaction <br /> (F) and TraI&#226;&#338;&#172;252 (E), and the data were plotted as a fraction of total     under these conditions (Fig. 7, lane 5). The addition of both <br /> plasmid in the relaxed species.                                            IHF and TraY also stimulated the nicking reaction catalyzed <br />                                                                            by TraI&#226;&#338;&#172;252 (Fig. 7, lane 6). We conclude that TraI&#226;&#338;&#172;252 is <br />                                                                            fully capable of participating in the formation of an active <br /> esterification reaction that utilizes the &#226;&#8364;&#8220;OH group on a specific          relaxosome at the F-plasmid oriT. <br /> tyrosine in the relaxase active site as a nucleophile to disrupt a <br /> specific scissile phosphodiester bond in oriT (7). To ensure that                                    DISCUSSION <br /> the nicking reaction observed with TraI&#226;&#338;&#172;252 was the result of <br /> transesterification, two experiments were done. First, a cleav-               Previous studies (6, 10, 33, 39) have shown that TraI is a <br /> age assay (31) using a specific 22-nucleotide oligonucleotide              bifunctional protein capable of catalyzing both a processive <br /> that contains the 11-nucleotide sequence recognized by the                 helicase reaction and a site- and strand-specific transesterase <br /> transesterase active site of TraI was performed (Fig. 5A and               reaction. Both of these biochemical activities are critical for <br /> B). The oligonucleotide was labeled on the 5&#226;&#172;&#732; end, and cleav-              the process of DNA transfer, and the protein is fully functional <br /> age at the specific scissile phosphodiester bond releases a 14-            only when both catalytic domains reside on the same polypep- <br />  VOL. 187, 2005                                                                                   FUNCTIONAL DOMAINS IN F-TraI                 703 <br />  <br />  <br />  <br />  <br />    FIG. 5. Cleavage activity of TraI and TraI&#226;&#338;&#172;252. (A) Schematic illustration of the oriT cleavage reaction catalyzed by TraI. The 22-nucleotide <br /> substrate was 5&#226;&#172;&#732;-end labeled and contained the binding site recognized by TraI and the scissile phosphodiester bond (nic). The reaction requires <br /> the addition of MgCl2 and results in the release of a 5&#226;&#172;&#732;-end-labeled 14-mer and an 8-mer that is covalently bound to TraI (TraI&#226;&#338;&#172;252). (B) Cleavage <br /> assays were performed as described in Materials and Methods and were incubated for 20 min at 37&#194;&#176;C prior to denaturation and resolution of <br /> products on a 16% denaturing polyacrylamide gel. Lane 1, no-protein control; lane 2, 1.7 nM TraI; lane 3, 4.1 nM TraI; lane 4, 12.3 nM TraI; lane <br /> 5, 37 nM TraI; lane 6, 111 nM TraI; lane 7, 333.3 nM TraI; lane 8, 1,000 nM TraI; lane 9, 1.7 nM TraI&#226;&#338;&#172;252; lane 10, 4.1 nM TraI&#226;&#338;&#172;252; lane 11, <br /> 12.3 nM TraI&#226;&#338;&#172;252; lane 12, 37 nM TraI&#226;&#338;&#172;252; lane 13, 111 nM TraI&#226;&#338;&#172;252; lane 14, 333.3 nM TraI&#226;&#338;&#172;252; and lane 15, 1,000 nM TraI&#226;&#338;&#172;252. This is <br /> a representative experiment. (C) Quantitation of multiple (three to five) oligonucleotide cleavage assays using TraI (F) and TraI&#226;&#338;&#172;252 (E) by <br /> PhosphorImager analysis using ImageQuant software. <br />  <br />  <br />  <br /> tide (6). The relaxase domain is wholly contained within the               C-terminal domain, TraI&#226;&#338;&#172;252, is properly folded but lacks a <br /> first &#226;&#172;&#402;310 amino acids of the protein, and the recognized                  region of the protein that is critical for some aspect of DNA <br /> helicase-associated motifs are found in the region from amino              transfer. Thus, this region of the protein represents a third <br /> acid 990 to amino acid 1450. Together, these two activities                functional domain that is essential for CDT but whose specific <br /> account for the major DNA processing events known to be                    role in the events leading to DNA transfer remains to be fully <br /> required in CDT. However, while necessary for the DNA pro-                 elucidated. <br /> cessing events associated with DNA transfer, these two activ-                 The process of CDT is highly regulated, even in the dere- <br /> ities are not sufficient and several other proteins, including             pressed F plasmid. Since the F plasmid is derepressed, due to <br /> IHF, TraY, TraM, and TraD, are known to have essential roles               an insertion in the finOP regulatory region (9, 43), all the <br /> in the process (15, 44).                                                   components of the relaxosome are available in the cell at all <br />    In addition to the two functional domains mentioned above,              times and it seems likely that the relaxosome is assembled at <br /> TraI has two regions that are not shared by all CDT initiator              oriT in advance of the formation of a stable mating pair. It has <br /> proteins and that may have a significant and specific functional           been suggested that a &#226;&#8364;&#339;mating signal&#226;&#8364;&#65533; triggers the relaxation of <br /> role in DNA transfer. The central region of the protein, en-               the conjugative plasmid in the cell upon formation of a stable <br /> compassing residues 310 to 990, has no well-characterized                  mating pair (42). The nature of this mating signal and its <br /> function but is known to be important for wild-type levels of              propagation in the cell is unknown. The TraM protein, which <br /> DNA helicase activity (6). The functional role of the C-termi-             is known from genetic studies to be required for DNA transfer <br /> nal 250 amino acids was explored in this study, and the results            yet has a poorly defined role in the process (5), may be a <br /> clearly indicate this region of TraI is essential for CDT. Dele-           component of this mating signal. It is known that the tet- <br /> tion of the C-terminal domain eliminates DNA transfer while                rameric TraM binds DNA at multiple sites within oriT (4, 12, <br /> having little or no effect on either the relaxase or the helicase          14, 32, 35, 40). The two highest-affinity binding sites overlap <br /> activity of TraI. We conclude that the protein lacking the                 the traM promoter and act to regulate the expression of TraM. <br />  704     MATSON AND RAGONESE                                                                                                       J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 6. Cleavage/ligation activity of TraI and TraI&#226;&#338;&#172;252. (A) Schematic representation of the strand exchange assay. See text for details. <br /> (B) Strand exchange assays were performed as described in Materials and Methods. The substrate (22-mer) is indicated on the left, and the <br /> cleavage product (14-mer) and expected recombinant product (24-mer) are indicated on the right. Lane 1, no-protein control; lane 2, 1.7 nM TraI; <br /> lane 3, 4.1 nM TraI; lane 4, 12.3 nM TraI; lane 5, 37 nM TraI; lane 6, 111 nM TraI; lane 7, 333.3 nM TraI; lane 8, 1,000 nM TraI; lane 9, 1.7 nM <br /> TraI&#226;&#338;&#172;252; lane 10, 4.1 nM TraI&#226;&#338;&#172;252; lane 11, 12.3 nM TraI&#226;&#338;&#172;252; lane 12, 37 nM TraI&#226;&#338;&#172;252; lane 13, 111 nM TraI&#226;&#338;&#172;252; lane 14, 333.3 nM TraI&#226;&#338;&#172;252; <br /> and lane 15, 1,000 nM TraI&#226;&#338;&#172;252. This is a representative experiment. <br />  <br />  <br />  <br /> The third, lower-affinity binding site is located upstream of the         are thought to mediate an interaction between the relaxosome <br /> traM promoter, and TraM binding at this site is presumed to be            formed on oriT and the membrane-spanning protein complex <br /> important for the DNA transfer event. In addition, TraM has               responsible for translocation of DNA into the recipient cell. <br /> been demonstrated to interact with TraD, a protein involved in            The F-plasmid-encoded TraD protein contains a C-terminal <br /> coupling the relaxosome to the secretion apparatus at the                 extension that is not present in several other members of the <br /> membrane (10). It is possible that TraM, bound at the low-                TraG family and that appears to be important for this inter- <br /> affinity site within oriT, interacts specifically with TraI to trig-      action (34). The C-terminal extension on TraD renders the <br /> ger relaxation of the plasmid. Such an interaction would be               protein highly specific in its interaction with the F-plasmid <br /> essential for DNA transfer and could be mediated by the                   relaxosome, and the full-length protein fails to support the <br /> C-terminal domain of TraI. At this point, it is not known if              transfer or mobilization of other conjugative plasmids. When <br /> F-TraI interacts with F-TraM. However, experiments using the              the C-terminal extension is removed, the specificity is reduced <br /> F-plasmid-like plasmid R1 have demonstrated involvement of                and the truncated TraD is able to promote the transfer of other <br /> TraM in the formation of an active relaxosome (20).                       conjugative and mobilizable plasmids (34). It is tempting to <br />    An alternative, and not mutually exclusive, possibility envi-          speculate that there is an interaction between the C-terminal <br /> sions an interaction between TraI and TraD that involves the              extension of TraI and the C-terminal extension on TraD that <br /> C-terminal domain of TraI. TraD is the F-plasmid-encoded                  is, at least in part, responsible for the coupling of the F-plasmid <br /> representative of the TraG family of proteins that have been              relaxosome to the membrane-spanning complex in the cell <br /> defined as conjugative coupling proteins (34). These proteins             envelop. If this were the case, then loss of either C-terminal <br />  VOL. 187, 2005                                                                                                 FUNCTIONAL DOMAINS IN F-TraI                        705 <br />  <br />  <br />   FIG. 7. TraI&#226;&#338;&#172;252 is capable of assembling into a functional relaxo-                     <br /> some. The relaxosome reconstitution experiments were performed as                      <br />                                                                                   <br /> described in Materials and Methods with the indicated proteins at the                  <br /> following concentrations: TraI (TraI&#226;&#338;&#172;252), 62.5 nM; IHF, 49.4 nM;                  <br /> and TraY, 170 nM. The positions of relaxed and supercoiled DNA are           <br /> indicated on the left. Lane 7 contains a linear DNA marker that                          <br /> migrates just slightly faster than nicked DNA on an agarose gel.                         <br /> here that loss of the C-terminal extension on TraI does indeed                    <br />                                                                                       <br /> abate DNA transfer. Previous studies (34) have shown that loss                   <br /> of the C-terminal extension on TraD also significantly reduces                     <br /> F-plasmid DNA transfer. Further studies will be required to                           <br />                                                                           <br /> determine if TraD and TraI do, in fact, interact and if the                      <br /> C-terminal extensions on each protein are responsible for that                   <br />                                                                                  <br /> interaction.                                                                            <br />    In summary, the C-terminal region on F-plasmid-encoded                        <br /> TraI represents a third functional domain of the protein that is         <br /> required for CDT. The relaxase domain and the helicase do-                        <br /> main are necessary but not sufficient to support the DNA                     <br /> transfer event. We speculate that the C-terminal domain is a                  <br /> protein-protein interaction domain and that either TraM or                 <br /> TraD (or both) interacts with TraI to relax the plasmid in                   <br /> preparation for transfer of ssDNA and to mediate the inter-                          <br /> action of the active relaxosome with the secretion apparatus at                     <br /> the cell envelop.                                                                        <br /> </body></html>