<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal vol. 1 1 no. 1 0 pp.3619 - 3628, 1992 <br />  <br />  <br /> MPI1, essential gene encoding mitochondrial <br /> membrane protein, possibly involved protein import <br /> into yeast mitochondria <br />  <br />  <br /> Ammy C.Maarse, Jolanda Blom,                                          GIP, outer membrane (Pfaller et al., 1988). ISP42 <br /> Leslie A.Grivell Michiel Meijer                                   yeast (Vestweber et al., 1989; Baker et al., 1990) and <br />                                                                       N. crassa homologue MOM38 (Kiebler et al., 1990) are <br /> Section Molecular Biology, Department Molecular Cell Biology,   GIP. Recent crosslinking studies showed MOM7, <br /> University Amsterdam, Kruislaan 318, 1098 SM Amsterdam,            MOM8 MOM30 components GIP (Soliner <br /> The Netherlands <br />                                                                       et al., 1992). Translocation proteins inner <br /> Communicated L.A.Grivell                                           membrane dependent membrane potential also <br />                                                                       involves matrix-localized heat shock protein mhsp7O <br /> <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> identify components mitochondrial protein                   (Kang et al., 1990; Scherer et al., 1990; Manning-Krieg <br /> import pathway yeast, adopted positive                   et al., 1991). Both mhsp7O mitochondrial heat shock <br /> selection procedure isolating mutants disturbed                protein hsp60 (Cheng et al., 1989; Ostermann et al., 1989; <br /> protein import. We cloned sequenced gene,                  Reading et al., 1989) participate refolding of <br /> termed MPIl, rescue genetic defect                translocated proteins matrix, proteolytic <br /> group mutants. MPII encodes hydrophilic                    maturation accomplished mitochondrial processing <br /> 48.8 kDa protein essential cell viability. Mpilp          peptidase MPP processing enhancing protein PEP <br /> is low abundance constitutively expressed                       (Hawlitschek et al., 1988; Jensen Yaffe, 1988; Pollock <br /> mitochondrial protein. Mpilp synthesized                    et al., 1988; Witte et al., 1988; Yang et al., 1988, 1991; <br /> characteristic mitochondrial targeting sequence                Schneider et al., 1990). Other proteases, including IMPI, <br /> amino-terminus, probably proteolytically                removes sorting sequence cytochrome b2 <br /> removed import. It membrane protein,                      (Behrens et al., 1991; Schneider et al., 1991), required <br /> oriented carboxy-terminus facing inter-                  deliver imported proteins correct mito- <br /> membrane space. In cells depleted Mpilp activity,                  chondrial location. While proteins functionally <br /> import precursor proteins tested far,             redundant, participate key steps of <br /> is arrested. We speculate Mpil protein                  mitochondrial protein import. ISP42, MPP, PEP the <br /> component proteinaceous import channel trans-                stress proteins hsp60 mhsp7O, essential for <br /> location precursor proteins mitochondrial               viability yeast, belong category. As yet, no <br /> inner membrane.                                                       essential proteins identified constitute protein <br /> Key words: gene cloning/membrane proteins/mitochondria/               transport channel inner membrane. <br /> protein import/yeast mutants                                             Here report isolation characterization MPIl, <br />                                                                       encoding mitochondrial protein involved import <br />                                                                       proteins mitochondria. Mpilp mitochondrial <br /> Introduction                                                          membrane protein probably essential <br />                                                                       component mitochondrial import machinery. <br /> Most mitochondrial proteins synthesized cytoplasmic <br /> ribosomes post-translationally imported mito-                Results <br /> chondria. Targeting information proteins typically <br /> specified stretch amino acids amino-terminus.        Isolation mutants disturbed mitochondrial protein <br /> This targeting signal, carries net positive charge        import <br /> often ability form amphiphilic structure         To identify components mitochondrial protein import <br /> surface mitochondrial outer membrane, usually               machinery Saccharomyces cerevisiae used positive <br /> removed metalloprotease entry mitochon-              screening procedure mutants (partial) defects in <br /> drial matrix (reviewed Hartl et al., 1989; Pfanner             mitochondrial protein import. This genetic approach been <br /> Neupert, 1990). A rapidly increasing number protein                successfully applied isolate mutants yeast secretory <br /> factors appears mediate specific import proteins            pathway (Deshaies Schekman, 1987) based on <br /> by mitochondria routing submitochondrial compart-              mislocalization chimeric protein efficiently <br /> ments. Components participate import pathway             imported cellullar organelle wild-type strain and <br /> are located cytosol inside mitochondria. In           enzyme activity normally residing cytosol <br /> the cytosol, 70 kDa heat shock proteins accelerate import             (Figure 1). To isolate mitochondrial import mutants, we <br /> of precursor proteins preventing misfolding                 constructed centromeric test plasmid carries hybrid <br /> polypeptide chains (Deshaies et al., 1988; Murakami et al.,           gene, encoding SOD-URA fusion protein. This chimeric <br /> 1988). Import-competent precursor proteins bound               protein contains 100 amino acids precursor <br /> receptors mitochondrial surface [MAS70 yeast                form manganese superoxide dismutase (SOD; mito- <br /> (Hines et al., 1990; Steger et al., 1990); MOM72                  chondrial matrix enzyme) amino-terminus, URA3 <br /> MOM 19 Neurospora crassa (Sollner et al., 1989, 1990)]             gene product (a cytosolic enzyme OMP decarboxylase <br /> and subsequently transferred common insertion site,              activity) carboxy-terminus HIS4C bridge in <br />                                                                                                                                 3619 <br />  A.C.Maarse et at. <br />  <br /> A.         wt (AURA3)                      B. mutant (AURA3) <br />  <br />  <br />  <br />  <br />                                  EMS <br />                               mutagenesis <br />  <br />  <br />  <br />  <br />                                                                            Fig. 2. Import SOD-URA fusion protein mitochondria of <br />      ura   -   <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>                      ura   -   phenotype          mutant MB3-52 impaired. The wild-type strain MB3 (panels and <br />                                                                            c) mutant MB3-52 (panels b d), harbouring the <br />                                                                            SOD-URA test plasmid (panels b) ATG-URA construct <br />                                            curing followed              (panels c d) plated selective medium supplemented with <br />                                            retransformation           adenine, leucine uracil (left panel) adenine leucine <br />                                            SOD-URA test plasmid            (right panel). <br />  <br />  <br />                                                                            SOD- URA test plasmid subjected ethylmethanesul- <br />                  nuclear mutants                                      fonate mutagenesis mutants Ura+ phenotype <br />                       +                                                    selected minimal medium containing glucose a <br />                  ura - phenotype                                           carbon source. Potentially, screening procedure selects <br />                                                                            mutations SOD targeting signal fusion <br />                                                                            protein mutations components mitochondrial <br />                                                                            protein import machinery. Both classes mutations be <br />                19 recessive nuclear mutants                                distinguished curing test plasmid mutant cells <br />                                                                            re-transforming plasmid-free cells original test <br />                                                                            plasmid. Twenty-seven mutants carry nuclear <br /> Fig. 1. Selection recessive nuclear yeast mutants defects       mutations. Genetic crosses revealed 19 mutants <br /> mitochondrial protein import. (A) Strain MB3 harbouring test             harbour recessive mutations fall minimally three <br /> plasmid encoding SOD-URA fusion protein displays Ura- <br /> phenotype efficient import chimeric protein         maximally 12 different complementation groups. <br /> mitochondria. (B) After ethylmethanesulfonate (EMS) mutagenesis            Figure 2 illustrates phenotype MB3-52, mutant <br /> mutants Ura+ phenotype selected. Mutants carrying              report paper. The cytosolic OMP <br /> nuclear mutations selected curing test plasmid Ura+       decarboxylase deficiency MB3-52 complemented <br /> cells re-transformation plasmid-free cells original test <br /> plasmid. Nuclear mutants bearing recessive mutations selected      ATG - URA construct (Figure 2, panel d) and <br /> testing Ura-phenotype cross MB5.                            SOD- URA test plasmid (Figure 2, panel b). This <br />                                                                            restoration pyrimidine biosynthetic pathway the <br /> between. After transformation test plasmid                 SOD -URA fusion protein simply explained an <br /> yeast strain MB3, complete chromosomal URA3                   impaired import fusion protein mitochondria. <br /> coding region deleted, OMP decarboxylase activity <br /> was determined detergent treated cellular extracts. As                  Isolation structural analysis MP11 gene <br /> considerable enzyme activity detected lysates                    To clone wild-type copy gene responsible the <br /> transformed cells, concluded fusion protein                 protein import defect MB3-52, mutant cells harbouring <br /> synthesized carries enzymically active OMP decarboxy-                  SOD- URA test plasmid transformed library <br /> lase (data shown). However, transformed MB3 cells                yeast genomic DNA fragments plasmid vector <br /> required exogenously supplied uracil growth (Figure 2).                YEpl3 (Nasmyth Tatchell, 1980). Double transformants <br /> The likely explanation observations                 selected plating cells minimal medium supple- <br /> the matrix targeting signal superoxide dismutase efficiently            mented uracil 5-fluoro-orotic acid (Boeke et al., <br /> targets enzymically active fusion protein mito-                   1984). This permitted identification complemented <br /> chondria, preventing participation cytosolic                    clone acquired uracil auxotrophy. We obtained <br /> pyrimidine biosynthesis. As control, MB3 cells                      YEp13 derivative complements Ura+ <br /> transformed similar construct SOD coding                   phenotype MB3-52 cells containing test plasmid <br /> sequences replaced ATG initiation codon. Thus,                  (Figure 3, construct 1). We clone also <br /> this plasmid-encoded ATG-URA fusion protein                         rescue genetic defect mutants MB3-68 MB3-75, <br /> mitochondrial targeting information. As expected, OMP                      mutant MB3-67. Several subclones this <br /> decarboxylase activity present extracts                     primary clone tested complementing ability. <br /> transformed cells cells grow selective medium                 The 2.7 kb HindIlI fragment, cloned high copy <br /> lacking uracil (Figure 2).                                                 number vector (Figure 3, construct 4) low copy <br />    Mutations block import SOD URA chimeric <br />                                                     -                      number vector (Figure 3, construct 8), contains informa- <br /> protein mitochondria allow cells grow                tion complementation. Partial complementation of <br /> absence uracil. Therefore MB3 cells harbouring                      Ura+ phenotype achieved HindIII-XbaI <br /> 3620 <br />                                                                                                                           Protein import yeast mitochondria <br />  <br />                                                                                                                           Complementation <br />                                                                                                                           mutant MB3-52: <br />                    CH <br />                   R   B/Sa                   P            X H S                         C HHC P               H           P     Sa/B <br />                 1 I I                             .. <br />                                                           I      ------------------------ <br />                                                                                        .-    Iv   V <br />                                                                                                   11 <br />                                                                                                                            \   ( Sp,S,R <br />                                                                                                                                   .. ;01 r   + <br />                                         I                  I <br />                                                                                                                    lkb <br />  <br />                 2                                                    R/Sp <br />                 3                           H/X                                                                                              +1 <br />  <br />  <br />  <br />  <br />                 4                           H/H <br />                                                                                                                                              + <br />  <br />  <br />  <br />  <br />                 5                                 C/C                                                                                        + <br />  <br />  <br />  <br />  <br />                 6                                              P/P <br />  <br />                 7                                                                                      S/S <br />  <br />                 8                           H/H <br />                                                                                                                                              + <br />  <br />  <br />  <br />  <br /> Fig. 3. Demarcation complementing activity deletion analysis. The indicated fragments YEpl3 derivative (1) subcloned either <br /> into high copy number vector YEplacl81 (2-7) low copy number vector YCplacl1l (8). Full, partial complementation the <br /> Ura+ phenotype mutant MB3-52 harbouring test plasmid indicated respectively +, +/-        The single headed black arrow indicates <br />                                                                                                                      -. <br />  <br /> the MPII reading frame determined DNA sequencing grey arrow indicates RPB3 reading frame. B, BamHI; C, ClaI; H, HindIll; <br /> P, PstI; R, EcoRI; S, Sall; Sa, Sau3A; Sp, SphI; X, XbaI. <br />  <br /> subclone (Figure 3, construct 3), complementa-                                   chondrial fraction (marked f-subunit F1-ATPase), <br /> tion observed PstI subclone (Figure 3,                                         Mpilp exclusively cofractionates mitochondria <br /> construct 6). This indicates PstI -XbaI segment                                    (Figure 5A, lane 2). This result confirmed immuno- <br /> within HindlIl fragment essential insufficient                               electron microscopy, supports mitochondrial <br /> full complementation. DNA sequence analysis com-                                     localization Mpilp (not shown). <br /> plete HindIH fragment revealed PstI XbaI                                    Subfractionation mitochondria revealed Mpilp <br /> sites located inside reading frame, corresponding                                  cofractionates ISP42 (an outer membrane protein) and <br /> a protein 431 amino acids molecular weight 48.8                                 F1-ATPase $-subunit (an inner membrane protein), but <br /> kDa (black arrow Figure 3). The nucleotide sequence                                   superoxide dismutase (a soluble matrix protein) <br /> the gene, termed MPII (Mitochondrial Protein Import 1),                                     (Figure SB, lanes 3 4). The loose association of <br /> and derived amino sequence shown Figure 4. As                                    Flo subunit inner face inner membrane <br /> nearly half amino acids Mpilp acidic,                                  explains protein matrix <br /> basic polar, protein hydrophilic. Moreover,                                fraction (Figure SB, lane 3). However, Mpilp found <br /> Mpilp obvious membrane spanning domains. The                                         exclusively membrane fraction (Figure SB, cf. <br /> amino-terminal 55 amino acids completely devoid                                      lanes 3 4), indicates tight association with <br /> acidic residues, 10 basic residues (including                                  mitochondrial membranes. With intact mitochondria the <br /> arginines) 17 polar residues (serine threonine)                                 protein completely resistant proteinase K (Figure SB, <br /> contained area. These features characteristic                               lane 5). In mitoplasts, however, c-myc tail fusion <br /> for mitochondrial targeting sequences suggest Mpilp                                protein degraded (Figure SB, compare lanes 2 6) <br /> is imported mitochondrial protein. The low codon                                  matrix enzyme superoxide dismutase and <br /> adaptation index MPI1 (0.14; Sharp Li, 1987) sug-                                    Flo subunit inner membrane (Figure SB, lane 6) <br /> gests vivo expression gene low.                                       remained intact. Mpilp mitochondrial membrane <br />    A search revealed homology similarity                                     protein carboxy-terminus facing intermembrane <br /> of Mpilp protein sequences EMBL data <span id='am-241' about='xsp:base' typeof='owl:Thing'>base</span>                                         space. <br /> (release 28.0). However, did presence gene                                   The mitochondrial location Mpilp determined with <br /> encoding RPB3 subunit RNA polymerase II, down-                                       fusion protein Mpil c-myc. The MPI -c-myc <br /> stream MPI] transcribed strand. This                                   construct encoding fusion protein contains identical MPII <br /> physical linkage means MPIl located chromosome                                   coding sequences HindII-XbaI subclone shown in <br /> IX, determined RPB3 (Kolodziej Young, 1989).                                     Figure 3 (construct 3). We MPI -c-myc <br />                                                                                             construct partially complement Ura+ <br /> Mpilp mitochondrial membrane protein                                                   phenotype MB3-52 cells harbouring SOD- URA test <br /> To determine location Mpilp yeast cell,                                    plasmid (not shown). In addition, introduction <br /> have DNA construct encoding fusion protein                                      low copy number plasmids carrying MPIl -c-myc <br /> which carboxy-terminal amino acid residues                                 construct strain disrupted nuclear MPI] gene, <br /> Mpilp replaced human c-myc epitope tag (Munro                                     rescue lethal effect gene inactivation (see <br /> and Pelham, 1987). This plasmid encoded Mpil -c-myc                                      later Results section). These experimental results <br /> fusion protein detected yeast cells                                   prove fusion protein functional, implying the <br /> commercially available antibodies recognizing c-myc                                     presence c-myc epitope chimeric protein does <br /> epitope. When transformed cells fractionated                                    interfere targeting authentic submitochondrial <br /> cytosolic fraction (marked hexokinase) mito-                                       location Mpilp. <br />                                                                                                                                                      3621 <br />  A.C.Maarse et al. <br />  <br />                           10                        20                 30                  40               50                     60                70                  80                 90             100             110              120 <br />        GAkTCTCGATT ATGATGTAC-T CCAkTGGGACA CCACTAGGGG TGCAATGCTT                                                     AGCAAATAAG AGCAAATCTG CGAkCGCCTTC                                                           TTCATTCTTA <br />                                                                                                                                                                               ATGCTTTTTC TGCTTCTGTT CTCCTTCTAT AAGTMAGAAT <br />        CTAGAGCTMA TACTACATGAk GGTACC~CTGT GGTGAT~CCCC ACGTTACGAA                                                     TCGTTTATTC TCGTTTAGAkC GCTGCGGAAG                        TACGAAAAAG ACGAAGACAkA GAkGGAAGATA <br />                130         140         150        160                                                  170                  180         190         200                               210        220        230         240 <br />        TTTTCTTc,r TCCTTTTCCA CCTTTTTCCT AAGACGT1TTT                                             TTTTTTTTTC           AGTTTACTGT TCTATGGAAA AGGAkATGAkAT                       AATAATCCGG AAAACTATAT ATTGGAAGMA CTGGCACAGT <br />        AAAAGAAGAAk AGGAAAkAGGT GGAAAAAGGAk TTCTGCAAAA                                           AAAAAAAAAG           TCAAATGACA AGATACCTTT TCCTTACTTA                         TTATTAGGCC TTTTGATATA TAACCTTCTT GAkCCG;TGTCA <br />                250         260         270        280                                                  290                  300         310         320                               330        340        350         360 <br />        TTGG,rCTCC GCAGAAkATTC TACTMAATTA CGTATAAATT                                             GTGTATTAGAk          AGAAATTCAG AAACGAAATA TCTGAACAGA                         CCTACTTTGG TATTTTTATA TGAkTAACTVT TTGAGTTGTG <br />        AAccMAGAGG cGTcrrTAAG ATGATTTAAT GCATATTTAA                                              CACATAATCT TCTTTAAGTC TTGTTTAGACTTGTCT                                        GGATGAAAkCC ATAMAAATAT ACTATTGAAA AACTCAACAC <br />                370         380         390        400                                                  410        420         430        440                                          450        460        470         480 <br />        GTCCTTTTCT TCATTTATTT TTAGAAACTT GCTTTTTTTG                                              GCCATAACAT MAAATrTAAA TAAcOAkCACT GAAACAAAAA                                  GAAAAGAAkCA CCACGACTAA TAAAACAAAA rrACMACATT <br />        CAGGAAAAGA AGrAAATAMA AATCTTTGMA CGAAAAAAAC                                              CGGTATTGTA TTTAAArrT ATTGCTGTGA CTTTGTTTTT                                    CTTTTCTTGT GGTGCTGATT ATTTTGTTTT AATGTTGTAA <br />  <br />                490        500        510          520         530         540         550        560                                                                                  570         580                    590        600 <br />        ATGATTAGGT ACTTCTTCAG TACAkACTTTT CMAGCTACTT CTTACGATMA GTAGACTTCT CACCTOAAA TAOThZAAC                                                                                  GTrAGCAACC ATACGGAAAA              CCATGTTCAG TTACCCGGCC <br />        TACTAATCCAk TGAAGAAGTC ATGTTGAAAA GTTCGATGMA   GAATGCTATT  CATCTGAAGA  GTGGACATTT  ATGATCATTG                                                                          CCATCGTTGG TATOCCTTTT               OGTACAAGTC AATGGGCCGG <br />                610        620         630         640         650         660         670        680                                                                                  690         700                      710              720 <br />        GGGTTTfTTTi TTCTCTTTCT TTTTCOCCCA CTGGAGAGCC   TCAOGTCATAT ATGTCAAAGAk AGAkCTOOTT AGTAGCATGG                                                                           GCTTCGGAkAC CAGGVTTTGAA             AGTACCAAkGT ATAATAAGCC <br />        CCCAAAAAAA AAGAGOAAAO AAAAGCGGGT GAkCCTCTCGG AGTCAGTATA TACAOTTTC TCTGAkCCAAC TCATCGrACC                                                                               CGAAGCCTTG GTCCAAACTT               TCATGGTTCA TATTATTCOGG <br />                730        740         750         760         770         780         790        800                                                                                  810         820                           830 <br />        CGCAAGTAAG CAAAAGTTGG GAkTAAGAkGGG CAGGAkCcGAG CTTATAGTGT TTCGTTTTTT TATTC~CTGAA TTTTTCACCA                                                                            CCGCAGCAAT TGCTATTTCA                GVTC ATG CAC AGAk TCC ACT <br />        GCGTTCATTC GTTTTCAACC CTATTCTCCC GTCCTGGCTC GAATATCACA AAGCAAAAAA ATMAGGAkCTT AAAAAGTGGT                                                                               GGCGTCGTTA ACGATAAAGT                CAAG TAC GTG TCT AGG TGA <br />                                                                                                                                                                                                                            M      H   R     S       T <br />       840          850          860           870           880          890           900           910          920            930           940 <br />         TTT ATC AGG ACG TCC GGC ACG AGC TCT AGG ACA CTA ACC GCA AGO TAC AGAk TCG CAG TAC ACA GGA TTA CTC GTT GCC CGA  GTA TTA TTC TCT ACC TCT ACG <br />         AMA TAG TCC TGC AGG CCG TGC TCG AGA TCC TGT OAT TGG  CGT TCC ATG TCT AGC GTC ATG TGT CCT MAT  GAG CAA COG GCT CAT MAT MAG  AGA TGG AGA  TGC <br />                             R          T        S        G         T   S      S      R      T     L    L V <br />                                                                                                        T      AA   R   V <br />                                                                                                                        R   L   F <br />                                                                                                                                Y     S  TRA S S  T 0        Y      T     G       L <br />          F        I <br />                            950                               960                   970                980                          990                 1000                   1010                  1020               1030                 1040 <br />         ACC      COT WCG CMA GOT GGA MAC <br />                                     CCT                                     CGAk TCA CCA CTC CAG ATT TTC CGC OAT ACA TTC MOG MG GMk TOO GAG MAG TCT CAG GMA CTG CAG GAG MAC ATA MAG <br />         TOG GCA CGC GTT CCA CCT TTG GGA                                     OCT AGT GGT GAG GTC TMA MAG GCG CTA TGT MAG TTC TTC CTT ACC CTC TTC AGA GTC CTT GAC GTC CTC TTG TAT TTC <br />          T R A Q G G N P                                                     RA S P L         Q   I   F   R   D   T   F   K   K  E   W   E   K S Q ES L Q E N I K <br />                   1050               1060         1070          1080                                                        1090                  1100                    1110                   1120               1130                  1140 <br />         ACG CTG            CMA OAT OCT TCG GGA                         0CC TAC AMA MG OCT AGO GMA GCA TAT TTG AMA OCT <br />                                                MAG TTA GGC GAO TCT GAO CGO                                                                                                                         CAG AGA GOC TCC ACA ATT GTG               GOT AMA <br />         TGC GAC            OTT CTA CGA AGC CCT TTC MAT CCG CTC AOA CTC     ATG TTT TTC CGA TCC CTT COT ATA MAC TTT COA                                                                             GTC TCT CCG AGO TOT TMA CAC               CCA TTT <br />          T L                Q0 D A S 0G K L G0 E S E A Y K K A R ES A Y L K A                                                                                                                       0 R G0 S T I V                            G0 K <br />           1150          1160          1170         1180                                                               1190                    1200                     1210                 1220            1230         1240 <br />         ACT TTo AMA MAG ACC GOT GAG ACC ATG GMA CAC ATA                                               0CC ACT MOG GCC TGO GAG TCC GAkG CTC GOT MAG MC ACA                                          AOA MOG GCG GCT 0CC 0CC ACG               GCG MAG <br />         TGA MAC TTT TTC TGO CCA CTC TGO TAC CTT GTG TAT                                               CGO TOA TTC CGO ACC CTC AGO CTC GAG CCA TTC TTG TOT                                          TCT TTC CGC COA CGO CGO TGC               CGC TTC <br />          T L K K T G0 E T MH E H I                                                                     A T K A W E S ES L G0 K N T                                                                  R K A A A A T                             A K <br />        1250          1260          1270         1280                                                             1290                      1300         1310          1320                               1330         1340 <br />         MOG CTG OAT GAG  AGT TTT GAG CCA GTG AOA CAG ACG                                              MOG ATT TAC MAG GMA                    GTC TCA GMA OTC ATT GAC OAT 000                       GAG AGT TCC CGA TAC GOT 000               TTT ATC <br />         TTC GAC CTA CTC TCA AMA CTC GOT CAC TCT OTC TGC                                               TTC TMA ATG TTC CTT                    CAG AOT CTT CAG TMA CTG CTA CCC                       CTC TCA AGO OCT ATG CCA CCC               AMA TAO <br />          K L D E S F E P V R 0 T                                                                       K         I      Y      K         ES V S ES V I D D 0                                        E S S R Y G0 G0                           F I <br />      1350                          1360                       1370                   1380                  1390                     1400                    1410                     1420               1430                   1440               1450 <br />         ACG      AMA GAOG CMk AGO                  AGA CTT AMA COT GAOG                    AGA OAC CTG GCC TCT 000 AMA AGA                       CAC AGO GCA GTA MOG AGC MAT GMA OAT GCA GGA ACA GCA GTO onr GCG <br />         TOC TTT      CTC      TCC     GrT          TCT GMk TTT GCA CTC                     TCT CTG GAC CGO AGAk CCC TTT TCT                      GTG TCC COT CAT TTC TCG TTA CTT CTA CGT CCT TOT COT CAC CMA CGC <br />             T     K            E       0        R RA L K R E                                R   D L A 5 0G K R                                    H RA A V K S N E D A 0G T A V V A <br />                1460                                 1470         1480                             1490           1500                                1510         1520           1530             1540          1550 <br />         ACA MAT ATC GAG                        TCT AMA GMA TCG TTT GOT                     MAG AMA GTG GAO OAT TTC MOG GAO                       AMA ACC GTG onT GGC COT TCT ATA CMA TCT TTA MOG MAC AMA         TTO TGO <br />         TOT TTA TAO CTC                        AGA TTT CTT AOC AMA CCA                     TTC TTT CAC CTC CTA MOG TTC CTC                       TTT TGO CAC CMA CCG GCA AGA TAT OTT AGA MAT TTC TTO TTT MAC ACC <br />          T N I E                                S K E S F G0                                K K V E D F K E                                        K    T <br />                                                                                                                                                         T V     V    0G R   S    I    0    S    L    K   N    K    LW <br />                1560                                   1570                  1580     1590          1600                                         1610            1620          1630           1640             1650 <br />          OAkT GMA AOT GMA MC CCC TTA ATT OTT                               GTC ATG AGO AMA ATA ACC MAC                                  AMA GTG GGC 000 TTC TTT GCA GMA ACA GMk TCC TCC COT         OTT TAC AOT CMA TTT <br />          CTA  CTT TCA CTT noG ooo MAT TMA CMA                              CAG TAC TCC TTT TAT TGG noG                                  nTT CAC CCG CCC MAG AMA COT CTT TOT CTT AGO       AGO GCA CMA ATO TCA onT AMA <br />           D ES ES N P L I V                                                 V NM R K I T N                                               K V 0G 0 F F A E T E S S AR V Y S 0 F <br />                 1660                           1670                 1680          1690          1700                                          1710           1720         1730             1740            1750 <br />          MOG CTA           ATG        GAkC CCA ACT             TTT TCG MAC GAG AGC TTC ACT AGAk CAC TTA                                 AGA GMA TAC ATT OTT CCC GAO ATT CTC GMA GCG TAT GTG AMA GGC OAT GTC AMA <br />          TTC OAT           TAC        CTG GOT TGA              AMA AGC TTG E S F T R GTG <br />                                                                            CTC  TCG MAG TGA TCT      MAT                                TCT CTT ATG TMA CMA GGG CTC TMA GAkG CTT CGC ATA CAC TTT CCG CTA CAG TTT <br />             K         L     M D                   P      T      F S N                             H L                                    RA E Y I V P E I L E A Y V K G0 D V K <br />         1760                               1770                  1780          1790          1800                                          1810           1820          1830            1840           1850 <br />          OTT      CTC AMA AMA TGG TTC                          AGC GAG GCG CCA TTC MAT GTn TAC GCC GCC                                  CMA CAG AMA ATC TTC AMA GMA CAG OAT OTT TAC GCC OAT         GGC COT ATC CTA OAkT <br />          CMA      GAG TTT TTT ACC MOG                          TCG CTC CGC GOT MAG TTA CMA ATG CGO CGO                                  onT GTC TTT TAG MAG TTT CTT GTC CTA CMA ATG CGO       CTA CCG GCA TAO OAT CTA <br />           V        L K K W F                                    S E A P F N V Y A A                                                      0 0 K I F K EQ                      D V Y A D G0 RK I L D <br />       1860                           1870                      1880                      1890               1900                     1910                       1920                 1930               1940                   1950                1960 <br />          ATC AGO GGC OTT GMA ATA OTG AGT GCA MOk TTA TTG OCT CCC CMA GAC ATC CCA GTC TTG OTG GTC GGG TOT AGA GCA CMA GMA ATC MAC CnT TAC AGO AMA <br />          TAG TCC CCG CMA CTT TAT CAC TCA COT TTC MAT MAC CGA GGG OTT CTG TAO GGT CAG MAC CAC CAG CCC ACA TCT COT GTT CTT TAG TTG GMA ATG TCC TTT <br />           I R G0 V E I V S A K L L A P Q D I P V L V V 0G C R A 0 E I N L Y R K <br />                            1970                              1980                  1990               2000                         2010                2020                    2030                 2040                2050                2060 <br />          MAG AMA ACT GGC GAG ATT GCG                                   GCT GGT GAC GMk OCT MAT ATC TTG ATG AGC TCT TAT 0CC ATG GnT TTC ACC AGA GAkC CCA GAO CMA ATC GAC OAT GAC GMA <br />          nTC TTT TOA CCG CTC TMA CGC                                   CGA CCA CTG CTT CGA TTA TAG MAC TAC TCG AGA ATA CGG TAC CMA MAG TGG TCT CTG GOT CTC onT TAG CTO CTA CTO CTT <br />           K K T G0 E5 I A                                               A G0 D E A N I L NM S S Y A M V F T R D P E5 0 I D DD0 0 E <br />                                                                                                               XbaI <br />               2070         2080          2090          2100         2110          2120       2130       2140                                                                                               2150            2160            2170 <br />          ACA                         GAkG TTT OTG CGC 000 GOrTT AQ OTT <br />              GMA 000 TOG MAG ATC noG CTC                              CMA TTC ACC TOA TTGOTTTCGA TOTGCTCTTT TOTTTTCTTT AOGMMOAGA <br />                                                                                                                         TCCTTTTCCT TCCTATTCCG <br />                                                                                                                                    AGGATMAGGC <br />          TOT CTT CCC ACC TTC TAG MAC      AMA CAC GCG CCC CCA AGA TCT     MAG TGO  ACT MJCCAAAGCT ACACGAGMAA ACAAAAGAMA <br />              T        E   K I L E F V R G0 G S R 0 F T <br />                                 G0         W <br />                           2180                      2190               2200             2210                2220                   2230              2240               2250                2260            2270               2280          2290 <br />          CTTGTAMATA ACTCCGTTAT GTCTCACCTT                                         CGATCCCCAC      TTTTTCTTTT GGMATACTOC ATCAMATAGO MOAGAkGAGM TMAGGMAATC ATATCCCMiT ATTTTTGMAC ATACGACACG <br />          GMACATTTAT TGOAGGCAATA CAGAGTOGMA                                        GCTAGGGGTG      AMAAAOAMA  CCTTATGACG TAGTTTATCC TTCTCTCTTT ATTCCTTTAG TATAGGGTTA TAAAMACTTG TATGCTGTGC <br />                 2300                                2310               2320                2330             2340                   2350 <br />          ATAT&apos;ACCTTATATAGTATCAk TCTGTACTAAk TAGTACTATA ACTMATACTT CTTGACOGOkC MOGCTT <br />          TATATGGMAT ATATCATAOT AOAkCATGATT ATC-ATGOATAT TGOATTATGMA GMACTGCCTG TTCGMA <br />  Fig. 4. Nucleotide sequence MPh] deduced amino acid sequence. The unique restriction                                                                                  enzyme recognition sites SpeI Xbal are <br />  underlined. <br />  Mpil protein proteolytically processed vivo                                                                                            mitochondrial targeting signal amino-terminus. We <br />  Imported mitochondrial proteins contain targeting sequences                                                                                  explored possibility targeting signal is <br />  cases located amino-terminus                                                                                      proteolytically removed vivo. Labelled Mpil-c-myc <br />  cleavable presequences. Mpilp carries characteristic                                                                                  fusion protein synthesized vitro, mixed protein <br />  3622 <br />                                                                                                                Protein import yeast mitochondria <br />  <br />  <br />  <br />  <br />                      NI&apos;l1 <br />                                                                    l <br />                                                                     bA F   i-,I        -1     .1 <br />  <br />  <br />  <br />  <br />                                                                                                                             -.1 <br />                                                                                                                               A&apos;. <br />  <br />  <br />  <br />  <br /> Fig. 5. Mpilp mitochondrial membrane protein. (A) YPl02 cells harbouring MPII-c-myc DNA construct, fractionated cytosolic <br /> fraction mitochondrial fraction, analysed SDS -PAGE, Western blotting subsequent immunodecoration antibodies <br /> raised hexokinase, 3-subunit FI-ATPase c-myc. Lane 1, cytosolic fraction; lane 2, mitochondrial fraction. (B) Mitoplasts, obtained <br /> by subjecting mitochondria osmotic shock, subsequently fractionated membrane fraction matrix fraction. Intact mitochondria <br /> and mitoplasts treated proteinase K (0.2 mg/ml, 30 min ice) indicated. Samples, equivalent 100 Ag mitochondrial protein were <br /> analyzed described above. In addition ,3-subunit FI-ATPase Mpil-c-myc superoxide dismutase (SOD) ISP42 were <br /> immunodecorated. Lane 1, mitochondria (MIT); lane 2, mitoplasts (MP); lane 3, matrix fraction (MAT); lane 4, membrane fraction (MEM); lane 5, <br /> mitochondria treated proteinase K (PK); lane 6, mitoplasts treated proteinase K; lane 7, mitoplasts treated proteinase K the <br /> presence 0.2% SDS (D). <br />  <br /> lysate yeast cells harbouring MPI -c-myc DNA <br /> construct run SDS -polyacrylamide gel. After <br /> blotting proteins nitrocellulose, vivo synthesized <br /> fusion protein decorated antibodies raised against <br /> the c-myc epitope, vitro synthesized protein <br /> was detected autoradiography. As shown Figure 6, <br /> the vivo synthesized fusion protein migrated significantly <br /> faster vitro synthesized protein. We conclude                    Fig. 6. In vivo processing Mpil protein. In vitro synthesized <br /> the Mpil protein proteolytically processed vivo. We                         Mpil -c-myc fusion, labelled [35S]methionine, mixed a <br />                                                                                   protein lysate yeast cells containing MPII-c-myc DNA <br /> assume proteolytic cleavage occurs action                        construct; proteins separated SDS-polyacrylamide gel and <br /> of specific processing peptidase inside mitochondria.                           subsequently blotted nitrocellulose. Lane 1, immunodecoration of <br />                                                                                   vivo synthesized fusion protein antibodies raised the <br />                                                                                   c-myc epitope; lane 2, autoradiographic detection vitro <br /> MP11 essential viability yeast                                          synthesized chimeric protein blotting. <br /> To determine MPIl encodes essential mito- <br /> chondrial protein, constructed null <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>mutation</span> MPI]                         To surprise, cells survived medium <br /> by inserting yeast LYS2 gene MPI]                         supplemented galactose medium supple- <br /> alleles homozygous lys2 diploid MB2. The presence                          mented glucose. Apparently, residual amount <br /> of wild-type disrupted MPI] allele resulting                       Mpilp, synthesized glucose-repressed growth <br /> strain MB2-22 verified Southern blot analysis                         conditions, sufficient viability yeast cells. This <br /> probes complementary MPI] sequences LYS2                             observation suggests Mpilp low abundance <br /> sequences. When strain MB2-22 sporulated                                  mitochondrial protein, feature agreement low <br /> ascospores dissected, tetrad maximally yielded                          codon adaptation index gene, indicating poor <br /> two viable spores, medium containing fermentable                        translational efficiency MPII mRNA. Moreover, <br /> carbon source (data shown). All viable spores                            Northern blot analysis revealed low steady state level and <br /> Lys- did carry <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span>null allele</span> MPI].                         constitutive expression endogenous MPI] mRNA in <br /> When MB2-22, harbouring plasmid containing intact                           wild-type cells, grown glucose- galactose- <br /> MPI] gene, sporulated, viable spores re-                          containing medium (Figure 7, lanes 1 2). The transcrip- <br /> quire lysine obtained. All Lys+ spores carried                      tion level MPI] comparable residual level of <br /> plasmid marker. These results prove MPI] essential                     transcription QCR8 (encoding 11 kDa subunit of <br /> gene yeast.                                                                    ubiquinol cytochrome c oxidoreductase; Maarse Grivell, <br />                                                                                   1987; Figure 7, lane 1) HIS3 (Hill et al., 1986; data <br /> The involvement Mpil protein import proteins                             shown) cells expression genes is <br /> into mitochondria vivo                                                         repressed. <br /> To explore function Mpilp, expression MPI]                             To able test effect depletion Mpilp activity <br /> placed control GAL] promoter. This promoter                          import proteins mitochondria, GAL] promoter <br /> is induced galactose repressed glucose used                        placed modified gene encoding <br /> as carbon source. A haploid mpil null mutant harbouring                       crippled Mpil protein. Deletion analysis MPh] <br /> this plasmid (pGAL-MPI1) obtained transformation                           (Figure 3) shown MPI] coding sequences running <br /> and subsequent sporulation diploid strain MB2-22.                          XbaI site partially complement Ura+ <br />                                                                                                                                                  3623 <br />  A.C.Maarse et at. <br />  <br />  <br />  <br />  <br /> Fig. 7. Steady state MPII mRNA level constitutively low yeast <br /> cells. RNA isolated YPI02 cells, grown early stationary <br /> phase rich medium containing 2% glucose 3% galactose, size- <br /> fractionated 1.2% agarose gel, blotted Hybond-N and <br /> hybridized probes specific for: MPI1 (1325 bp PstI-HindIII <br /> fragment), QCR8 (840 bp HindIII-Sall fragment; Maarse Grivell, <br /> 1987) ACT] (1600 bp BamHI-KpnI fragment; Ng Abelson, <br /> 1980). The actin probe used assess comparable          Fig. 8. The effect Mpilp depletion steady state levels nuclear- <br /> amounts RNA loaded slot. Lane 1, RNA cells       encoded mitochondrial proteins. Wild-type cells cells the <br /> grown presence glucose; lane 2, RNA cells grown     mpil null mutant, containing plasmid-borne gene fusion the <br /> the presence galactose.                                            GAL] promoter truncated MPII, grown overnight on <br />                                                                       medium containing 3% galactose, collected divided equal <br /> phenotype mutant MB3-52. Indeed, growth                   portions. One portion shifted medium containing 2% glucose <br /> haploid mpil null mutant containing plasmid-borne gene              (d) half resuspended fresh galactose- <br />                                                                       containing medium (g). At different time points steady state levels <br /> fusion GAL] promoter truncated MPII coding            hexokinase (HK) mitochondrial proteins superoxide <br />  sequence severely impaired glucose-containing                 dismutase (SOD), subunit f F1-ATPase (Fl-3), citrate synthetase <br /> medium compared growth galactose-containing                   (CS) internal NADH dehydrogenase (NDH) determined by <br /> medium. A <span id='am-1' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> shift experiment performed                size-fractionation, Western blotting immunodecoration. (A) Steady <br />                                                                       state levels mature superoxide dismutase (mSOD) precursor <br /> wild-type cells cells mpil null mutant,                superoxide dismutase (pSOD) wild-type cells (wt) cells the <br /> harbouring DNA construct (Figure 8). After preculturing          mpil null mutant (AMPII) harbouring fusion gene. The levels <br /> in galactose-containing medium, cells collected              determined 1, 6, 12 24 h <span id='am-2' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> shift. (B) Steady <br /> divided equal parts. One half shifted glucose-        state levels hexokinase mitochondrial proteins, <br />                                                                       visualized 1 24 h <span id='am-4' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> shift. <br /> containing medium, half resuspended <br /> in fresh galactose-containing medium. At different time               synthetase, subunit (3 FI-ATPase internal NADH <br /> points samples taken steady state levels          dehydrogenase. After 24 h incubation glucose- <br /> cytosolic protein hexokinase mitochondrial proteins           containing medium, proteins virtually absent in <br /> superoxide dismutase, citrate synthetase, subunit 3                mpil null mutant cells harbouring GAL] -MPI] DNA <br /> FI-ATPase internal NADH dehydrogenase                        construct (Figure 8B, lane 7). Part reduction these <br /> determined. Figure 8A shows steady state levels                protein levels explained glucose-repressed <br /> superoxide dismutase time course experiment. After            expression nuclear genes (Figure 8B, cf. lanes 3 and <br /> 6 h incubation glucose-containing medium, level             4), superimposed reduction (Figure 8B, cf. <br /> of mature superoxide dismutase mpil null mutant                lanes 3 7) caused depletion Mpi lp activity. <br /> diminished 24 h protein disappeared         This effect specific mitochondrial proteins, level <br /> (Figure 8A, panel, cf. lanes 3, 5 7). This                 cytosolic protein hexokinase significantly <br /> decrease accompanied gradual increase                 affected <span id='am-5' about='Thesaurus:Nutrient' typeof='owl:Thing'>nutrient</span> shift galactose glucose <br /> amount precursor superoxide dismutase. Turning                 (Figure 8B, panel, cf. lanes 3 7). These results <br /> expression GAL] -MPH] fusion gene responsible              strongly imply involvement Mpilp in <br /> for effect level superoxide dismutase               mitochondrial protein import. <br /> remained constant mutant cells, grown galactose-                  After depletion Mpilp activity, decrease the <br /> containing medium (Figure 8A, panel, cf. lanes 2,              mature forms imported mitochondrial proteins was <br /> 4, 6 8). In addition, level superoxide dismutase           compensated comparable increase their <br /> remained unchanged wild-type cells, containing            precursors. This suggests proteins probably <br /> authentic chromosomal copy MPII, grown                   accumulate unfolded state import stage makes <br /> galactose- glucose-containing medium (Figure 8A,               sensitive proteolytic degradation. <br /> panel). Figure 8B shows comparable results                    To provide additional experimental evidence role of <br /> obtained imported mitochondrial proteins citrate              Mpilp mitochondrial protein import, pulse-chase <br /> 3624 <br />                                                                                                     Protein import yeast mitochondria <br />  <br />                                                                           sensitivity MPI 1 -c-myc fusion protein proteinase <br />                                                                           K suggests carboxy-terminus Mpilp faces in- <br />                                                                           termembrane space. The fact Mpilp proteolytically <br />                                                                           processed vivo suggests inner membrane <br />                                                                           protein, imported outer membrane proteins not <br />                                                                           usually carry cleavable presequence (Hartl et al., 1989). <br />                                                                           Although amino acid sequence Mpilp predicts a <br />                                                                           hydrophilic protein obvious transmembrane <br />                                                                           domains, Mpilp behaves like membrane protein. In <br />                                                                           respect Mpilp resembles import site protein ISP42, <br />                                                                           located yeast mitochondrial outer membrane <br />                                                                           (Baker et al., 1990). Stabilization hydrophilic <br />                                                                           proteins lipid environment achieved inter- <br />                                                                           actions membrane proteins. Crosslinking studies <br /> Fig. 9. Rate maturation precursor superoxide dismutase           revealed MOM38, N. crassa homologue ISP42, <br /> Mpilp-depleted cells. A nutrient shift experiment performed      associated import receptors MOM 19 and <br /> the mpil null mutant containing GAL-MPII construct (see legend <br /> of Figure 8). Cells grown 6 12 h presence    MOM72, outer membrane proteins MOM7, <br /> glucose (glu) galactose (gal) OD6wynm Of 1.5, collected       MOM8 MOM30 (Kiebler et al., 1990; Sollner et al., <br /> centrifugation, resuspended 40 mM KP, pH 6.0, 2% glucose            1992). Whereas proteins constitute multi- <br /> galactose OD60jnm 20 pulse-labelled [35S]methionine     subunit hydrophilic transmembrane channel translocation <br /> (75 /ACi/ml) 2.5 min 28&#194;&#176;C. Labelling stopped adding <br /> 10 mM unlabelled L-methionine 0.1 mg/ml cycloheximide. The            precursor proteins outer membrane, Mpi Ip may <br /> cells chased 28&#194;&#176;C times indicated, aliquots      component proteinaceous import channel through <br /> removed, cells spun cell pellet frozen liquid   inner membrane. <br /> nitrogen. Cellular proteins extracted subjected immune           So far components mitochondrial protein import <br /> precipitation rabbit antisera superoxide dismutase.         apparatus yeast essential cell viability. One them <br /> Immunoprecipitates collected protein A-sepharose beads and <br /> analysed SDS-PAGE autoradiography.                                outer membrane protein ISP42 (Baker et al., 1990). <br />                                                                          The matrix proteins: subunits the <br /> experiment performed mpil null mutant                       matrix-processing protease (encoded MASI MAS2; <br /> containing GAL -MPII DNA construct. After 6 12 h                 (Jensen Yaffe, 1988; Witte et al., 1988; Yang et al., <br /> growth glucose- galactose-containing medium, cells                  1988), mitochondrial hsp60 (encoded MIF4; Cheng et al., <br /> were pulse-labelled 2.5 min [35S]methionine and <br />                                                                           1989) mitochondrial mhsp70 (encoded SSCI; Craig <br />                                                                          et al., 1989; Scherer et al., 1990). <br /> subsequently chased indicated time periods. In cells                 In contrast essential components import <br /> grown presence galactose, labelled superoxide <br /> dismutase exclusively mature size (Figure 9,                      machinery, dispensable. These include the <br /> lanes 9-16). However, 6 h growth glucose-                       mitochondrial import receptor MAS70 (Hines et al., 1990; <br /> containing medium, significant labelled                      Steger et al., 1990), cytochrome c heme lyase, is <br /> precursor superoxide dismutase present, was <br />                                                                          involved import apocytochrome c (Dumont et al., <br /> slowly converted mature form (Figure 9, lanes 1-9).              1991) inner membrane protease I, removes <br /> Twelve hours nutrient shift glucose,              cytochrome b2 sorting sequence (Behrens et al., 1991; <br /> precursor superoxide dismutase present rate of <br />                                                                          Schneider et al., 1991). Mpilp sixth essential <br /> maturation reduced (Figure 9, lanes 5-8).                    component localized inner <br />                                                                          membrane. This fits picture emerging the <br />                                                                          import route proteins translocated mitochondrial <br /> Discussion                                                               matrix (Pfanner Neupert, 1990). After recognition by <br />                                                                          functionally redundant receptors surface outer <br /> To identify components mitochondrial protein import               membrane, precursor proteins enter common import <br /> pathway yeast performed new screening                   pathway outer membrane essential <br /> mutants disturbed protein import. Positive selection               components ISP42 participate. Subsequently, <br /> such mutants based mislocalization plasmid-                  proteins transferred unique transport channel across <br /> encoded fusion protein efficiently imported                 inner membrane essential import catalysts the <br /> mitochondria OMP decarboxylase activity,                    matrix space. It tempting speculate Mpilp part <br /> normally residing cytoplasm. Mutants isolated                protein import inner membrane and <br /> with trans-acting recessive nuclear mutations belong             functions before, membrane <br /> at complementation groups. By transformation                 potential dependent translocation targeting sequences <br /> with yeast nuclear DNA bank identified new                   inner membrane. According hypothesis, <br /> gene, MPII, rescue genetic defect mutant                mitochondrial protein import specifically <br /> MB3-52. MPI] encodes mitochondrial protein                     inhibited Mpilp-depleted cells illustrated by: (i) a <br /> essential cell viability import precursor proteins            strongly reduced rate maturation precursor of <br /> into mitochondria.                                                       superoxide dismutase (ii) virtually unaffected level of <br />   The intramitochondrial location Mpilp determined                cytosolic enzyme hexokinase drastically reduced <br /> with functional Mpi lp derivative, furnished c-myc              steady state levels mature forms imported <br /> epitope tag extreme carboxy-terminus. The fractiona-              proteins tested far. As decrease <br /> tion behaviour Mpilp consistent protein               compensated parallel increase corres- <br /> of inner outer mitochondrial membrane. The                        ponding precursor proteins, imported proteins arrested an <br />                                                                                                                                    3625 <br />  A.C.Maarse et al. <br />  <br />  Mpilp dependent translocation step probably                                    DNA manipulations <br />  susceptible proteolytic degradation.                                                 To generate pUC18ura3::LYS2, pUC18 (Sambrook et al., 1989; Yanisch- <br />                                                                                          Perron et al., 1985) restricted PstI, treated T4 DNA <br />     All mitochondrial proteins known essential                                 polymerase, digested SmiaI ligated. This plasmid, pUC18APS, <br />  viability yeast cells key components mito-                                linearized HindIII ligated 1.1 kb HindIlI URA3 <br />  chondrial protein import machinery (Baker Schatz,                                   fragment construct pUC18URA3. Finally, 0.9 kb PstI-SmaI URA3 <br />  1991). The results presented paper Mpilp                              fragment replaced 4.9 kb PstI-HindIII (blunt) LYS2 fragment <br />  new essential mitochondrial protein,                                    liberated pDP6 (Fleig et al., 1986). <br />                                                                                             pBPHura3: :HIS3 obtained pBPHl (Van Loon et al., 1987) by <br />  participates protein import. Future experiments                                  replacing EcoRV fragment containing URA3 coding region <br />  necessary establish Mpilp plays direct                                  1.8 kb blunt-ended BamHI HIS3 fragment, isolated YEp6 (Struhl <br />  indirect role mitochondrial protein import.                                          et al., 1979). The SOD- URA test plasmid generated steps. <br />                                                                                          Firstly, 369 bp EcoRI-BamHI fragment, liberated SOD deletion <br />                                                                                          construct encoding 100 amino acids precursor of <br />                                                                                          superoxide dismutase, cloned pBPHlura3::HIS3. Secondly, this <br /> Materials methods                                                                    construct linearized BamHI, treated Klenow fragment and <br />                                                                                          ligated 1.2 kb blunt-ended BamHI fragment isolated pNKY48 <br /> Strains growth media                                                                 (Alani Kleckner, 1987). The ATG- URA plasmid constructed by <br /> S.cerevisiae strains used study listed Table I. MB3 derivative       cloning successively synthetic 10 bp EcoRI-BamHI fragment containing <br /> of YP102 (kindly provided P.Philippsen) coding sequences                 ATG codon BamHI fragment pNKY48 into <br /> of mutant ura3-52 allele YP102 replaced LYS2 sequences.                    pBPH lura3: :HIS3. The nucleotide sequences frame fusions the <br /> MB3 constructed transformation YP102 5.1 kb HindIll                   SOD-HIS4 ATG-HIS4 junctions constructs indicated in <br /> fragment pUC18ura3::LYS2 selection Lys+ transformant                       Table II. <br /> with predicted URA3 deletion Southern blot analysis. MB4,                          pUC18mpil::LYS2 constructed cloning 3.0 kb EcoRI-SalI <br /> spontaneous Lys2- derivative S150-2B (Cold Spring Harbor Laboratory                   MPI) fragment original YEp 13 derivative pUC 18 derivative <br /> collection), selected according method described Chattoo et al.              lacking PstI site replacing 1.6 kb SpeI-XbaI MPI) fragment <br /> (1979). MB5 derived MB4 similar way MB3 YP102.                     4.9 kb XbaI LYS2 fragment pDP6. In construct LYS2 <br /> MB1 selected random tetrad analysis sporulated diploid obtained              transcribed strand MPI] replacement. <br /> by crossing YP102 S150-2B. The diploid strain MB2 constructed                       To construct MPI) -c-myc fusion gene, synthetic SmnaI -EcoRI <br /> by crossing MB1 YP102. MB2-22, harbouring inactivated MPII                       fragment encoding c-myc epitope cloned YEplac181 and <br /> allele, obtained transforming MB2 5.5 kb Ball -SphI fragment               YCplacl 11 (Geitz Sugino, 1988). A stop codon created both <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> pUC18mpil::LYS2 selecting Lys-positive transformant                      constructs filling EcoRI site downstream c-myc coding sequences <br /> one disrupted MPII allele Southern blot analysis.                                     Klenow fragment E. coli DNA polymerase, generating the <br />    Selective minimal medium growth yeast contained 0.67% yeast                     plasmids YEpmycl81 YCpmycl 1 1. The complete MPI) coding <br /> nitrogen base, 2% glucose 3% galactose supplemented                      sequence, liberated HindIll-XbaI subclone (Figure 3, construct 3) <br /> appropriate amino acids, uracil adenine (each 20-50 /Ag/mI). Rich                 restriction XbaI, treatment Klenow fragment restriction <br /> broth contained 2 % bactopeptone, 1 % yeast extract 2 % glucose 3 %               HindIl, cloned YEpmyc 181 YCpmycl 1 1, digested <br /> galactose.                                                                               Xnal SmaI site, treated Klenow fragment restricted <br />    Recombinant DNA constructs M 13 vectors propagated                         HindHI, creating YEpmycMPIl YCpmycMPI1. The nucleotide <br /> Escherichia coli JM1O1 (Yanisch-Perron et al., 1985) grown 2YT                        sequence running MPI) -c-myc junction filled EcoRI <br /> medium (1.6% bactotryptone, 1% yeast extract, 0.5% NaCl). E.coli HBIO1                   site plasmids shown Table II. <br /> (Boyer Dussoix, 1969) JF1754 (lac, gal, metB, leuB, hisB, hsdR)                      To construct pGAL-MPI1 introduced PCR techniques BamHI <br /> was used propagation DNA constructs plasmid vectors. E. coli                   site immediately ATG start codon MPII the <br /> transformants selected 2YT medium containing 100 itg/ml                          low copy number HindIll subclones. A 1.2 kb EcoRI-BamHI fragment <br /> ampicillin, JF1754 used transformation plasmid DNA isolated                   containing MPIh promoter replaced 0.8 kb <br /> from double transformed yeast cells. This permitted direct selection                EcoRJ-BamHI GAL) promoter fragment isolated pNN265 (kindly <br /> transformants carrying plasmids containing yeast LEU2 gene                  provided M.Johnston). The nucleotide sequence GAL) -MPI] <br /> complement leuB mutation.                                                            junction indicated Table II. The truncated MPIh gene regulated by <br />  <br />  <br /> Table I. Yeast strains used study <br /> Strain                                                                               Genotype <br /> YP102                                                                                MATai, ade2-101Jchre, his3-A200, leu2-A1, lys28OJamber, ura3-52 <br /> MB3                                                                                  MATa, ade2J101ochre, his3-A200, leu2-Al, tys2_801amrbe, ura3::LYS2 <br /> S150-2B                                                                              MATa, his3-AJ, leu2,3,112, trpl-289, ura3-52 <br /> MB4                                                                                  MATa, his3-AJ, leu2-3, 112, lys2, trpl-289, ura3-52 <br /> MB5                                                                                  MATa, his3-AJ, leu2-3, 112, Iys2, trpl-289, ura3::LYS2 <br /> MB1                                                                                  MATa, his3, leu2,   lys2-801amber,   trpl-289, ura3-52 <br /> MB2                                                                                  MATalo!, ADE21ade2-JOPochre, his3/his3-A200, 1eu211eu2-A1, lys2-801amber/ <br />                                                                                      Iys2_801amber, trpl-289/TRPI, ura3-52/ura3-52 <br /> MB2-22                                                                               MATal/, ADE2/ade2J101Ochre, his3/his3-/i200, leu2lleu2-A1, lys2-801amber/ <br />                                                                                      Iys2_801amber, trpl-289/TRPI, ura3-52/ura3-52, MPII/mpil::LYS2 <br />  <br /> Table H. Nucleotide sequence junctions hybrid genes <br /> SOD-HIS4                                                                C CAT Ggg gga tcg atc cAA GCT <br /> ATG-HIS4                                                                GAATTC ATG ggg atc cAA GCT <br /> MPI -c-myc                                                              TCT AGC CGG GAA CAA AAA CTT ATT TCT GM GM GAT CTG AAT TM 7TC <br /> GAL] -MPII                                                              GGAGAAAAAACcccggatcc ATG CAC <br />  <br /> Sequences introduced constructing hybrid genes shown lower case. HIS4, c-myc MPI) coding sequences flanking junctions are <br /> shown upper case italics. The CAT codon SOD sequence corresponds histidine residue position 100 wild-type precursor of <br /> superoxide dismutase (Marres et al., 1985). Initiation termination codons underlined. <br /> 3626 <br />                                                                                                                  Protein import yeast mitochondria <br />  <br /> the GAL] promoter derived pGAL-MPIl digestion XbaI               Deshaies,R.J., Koch,B.D., Werner-Washburne,M., Craig,E.A. and <br /> and HindIII recircularization plasmid filling ends          Schekman,R. (1988) Nature, 332, 800-805. <br /> with Klenow fragment.                                                             Dumont,M.E., Cardillo,T.S., Hayes,M.K. Sherman,F. (1991) Mol. <br />   To synthesize MPI -c-myc fusion protein vitro, fusion gene                Cell. Biol., 11, 5487-5496. <br /> cloned pEP30 (Laird, 1988) two-step procedure. A 0.2 kb                 Fleig,U.N., Pridmore,R.D. Philippsen,P. (1986) Gene, 46, 237-245. <br /> BamHI-PstI fragment pGAL-MPII, encoding amino-terminus                   Gietz,R.D. Sugino,A. (1988) Gene, 74, 527-534. <br /> of MPII, cloned pEP30. The resulting construct linearized            Hartl,F.-U., Pfanner,N., Nicholson,D. Neupert,W. (1989) Biochim. <br /> with HindIll, treated Klenow, digested PstI ligated             Biophys. Acta, 988, 1-45. <br /> 1.6 kb PstI-RsaI fragment YCpmycMPIl encoding carboxyl-                  Hawlitschek,G., Schneider,H., Schmidt,B., Tropschug,M., Hartl,F.-U. and <br /> terminus fusion protein.                                                      Neupert,W. (1988) Cell, 53, 795-806. <br />                                                                                   Hill,D.E., Hope,I.A., Macke,J.P. Struhl,K. (1986) Science, 234, <br /> Miscellaneous                                                                        451-457. <br /> The following procedures performed essentially according published        Hines,V., Brandt,A., Griffiths,G., Horstmann,H., Britsch,H. Schatz,G. <br /> methods: manipulation nucleic acids (Sambrook et al., 1989); isolation            (1990) EMBO J., 9, 3191-3200. <br /> of RNA yeast (Losson Lacroute, 1979); isolation total yeast           Jensen,R.E. Yaffe,M. (1988) EMBO J., 7, 3863-3871. <br /> cell extracts (Needleman Tzagoloff, 1975); pulse-labelling immune         Kang,P.-J., Ostermann,J., Shilling,J., Neupert,W. Pfanner,N. (1990) <br /> precipitation (Brandt, 1991); protein electrophoresis (Laemmli, 1970);               Nature, 348, 137-143. <br /> immunoblotting using HRP-conjugated secondary antibodies described             Kiebler,M., Pfaller,R., S6llner,T., Griffiths,G., Horstmann,H., Pfanner,N. <br /> in Promega Protocols Applications Guide, second edition, 1991;               Neupert,W. (1990) Nature, 348, 610-616. <br /> DNA sequence analysis chain elongation inhibitors (Sanger et al., 1977);     Klebe,R.J., Harriss,J.V., Sharp,Z.D. Douglas,M.G. (1983) Gene, 25, <br /> EMS mutagenesis sporulation diploid yeast cells (Rose et al., 1988);          333-341. <br /> transformation yeast cells (Klebe et al., 1983).                               Kolodziej,P. Young,R.A. (1989) Mol. Cell. Biol., 9, 5387-5394. <br />    For vitro synthesis MPI -c-myc fusion protein, MPI          -    Laemmli,U.K. (1970) Nature, 227, 680-685. <br /> c-myc DNA construct pEP30 linearized PvuII, run-off transcript      Laird,P.W. (1988) Ph.D. thesis. University Amsterdam. <br /> was T7 RNA polymerase according standard methods (Melton             Losson,R. Lacroute,F. (1979) Proc. Natl. Acad. Sci. USA, 76, <br /> et al., 1984) transcript translated vitro presence          5134-5137. <br /> [35S]methionine recommended supplier rabbit reticulocyte             Maarse,A.C. Grivell,L.A. (1987) Eur. J. Biochem., 165, 419-425. <br /> lysate (GIBCO BRL).                                                               Manning-Krieg,U.C., Scherer,P.E. Schatz,G. (1991) EMBO J., 10, <br />    Fractionation yeast cells performed essentially described            3273 -3280. <br /> Daum et al. (1982) isolated mitochondria resuspended          Marres,C.A.M., Van Loon,A.P.G.M., Oudshoom,P., Van Steeg,H., <br /> 0.65 M sorbitol, 20 mM KP,, 1 mM MgC12, 1 mM EDTA. Mitoplasts                        Grivell,L.A. Slater,E.C. (1985) Eur. J. Biochem., 147, 153-161. <br /> were obtained diluting mitochondrial suspension 3.25-fold 20 mM         Melton,D.A., Krieg,P.A., Rebagliati,M.R., Maniatis,T., Zinn,K. and <br /> Tris-HCI pH 7.4, incubating suspension 10 min ice, sedimenting                Green,M.R. (1984) Nucleic Acids Res., 12, 7035-7056. <br /> the mitoplasts 165 000 g 20 min Beckman Type SW50. 1 rotor            Munro,S. Pelham,H.R.B. (1987) Cell, 48, 899-907. <br /> and finally resuspending 10 mM Tris-HCI pH 7.4, 1 mM EDTA.                Murakami,H., Pain,D. Blobel,G. (1988) J. Cell Biol., 107, 2051-2057. <br /> After sonicating mitoplasts 6 x 5 s 0&#194;&#176;C sedimentation 165 000 g     Nasmyth,K.A. Tatchell,K. (1980) Cell, 19, 753-764. <br /> during 60 min Beckman Type SW50. 1 rotor, pellet supernatant           Needleman,R.B. Tzagoloff,A. (1975) Anal. Biochem., 64, 545-549. <br /> were obtained containing respectively mitochondrial membranes matrix          Ng,R. Abelson,J. (1980) Proc. Natl. Acad. Sci. USA, 77, 3912-3916. <br /> proteins.                                                                         Ostermann,J., Horwich,A.L., Neupert,W. Hartl,F.-U. (1989) Nature, <br />                                                                                      341, 125-130. <br />                                                                                   Pfaller,R., Steger,H.F., Rassow,J., Pfanner,N. Neupert,W. (1988) J. <br />                                                                                      Cell Biol., 107, 2483-2490. <br /> Acknowledgements                                                                  Pfanner,N. Neupert,W. (1990) Annu. Rev. Biochem., 59, 331-353. <br />                                                                                   Pollock,R.A., Hartl,F.-U., Cheng,M.Y., Ostermann,J., Horwich,A. and <br /> We grateful Marian Jong expert technical assistance. We wish           Neupert,W. (1988) EMBO J., 7, 3493-3500. <br /> to thank Bertrand Daignan Fornier, Nelleke Corbett Gyuri Fenivesi             Reading,D.S., Hallberg,R.L. Myers,A.M. (1989) Nature, 337, <br /> for contribution experiments. The electron microscopic          655-659. <br /> analysis Evert Jan van Tuinen gratefully acknowledged. We thank             Rose,M.D., Winston,F. Hieter,P. (1988) Methods Yeast Genetics: <br /> Dr G.Schatz (Basel) supplying antisera raised yeast subunit              A Laboratory Course Manual, Cold Spring Harbor Laboratory Press, <br /> of FI-ATPase, citrate synthetase ISP42 Dr A.Author (Vancouver)               Cold Spring Harbor, NY. <br /> for supply antiserum raised yeast manganese superoxide          Sambrook,J., Fritsch,E.F. Maniatis,T. (1989) Molecular Cloning, A <br /> dismutase. This work supported Biotechnology Centre                Laboratory Manual, second edition. Cold Spring Harbor Laboratory Press, <br /> University Amersterdam.                                                           Cold Spring Harbor, NY. <br />                                                                                   Sanger,F., Nicklen,S. Coulson,A.R. (1977) Proc. Natl. Acad. Sci. USA, <br />                                                                                      74, 5463-5467. <br />                                    <br /> </body></html>