<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>      Composition and Architecture of the Schizosaccharomyces pombe <br />                        Rad18 (Smc5-6) Complex <br />  <br />  <br />              The rad18 gene of Schizosaccharomyces pombe is an essential gene that is involved in several different DNA <br />           repair processes. Rad18 (Smc6) is a member of the structural maintenance of chromosomes (SMC) family and, <br />           together with its SMC partner Spr18 (Smc5), forms the core of a high-molecular-weight complex. We show here <br />           that both S. pombe and human Smc5 and -6 interact through their hinge domains and that four independent <br />           temperature-sensitive mutants of Rad18 (Smc6) are all mutated at the same glycine residue in the hinge region. <br />           This <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-206' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>mutation</span> abolishes the interactions between the hinge regions of Rad18 (Smc6) and Spr18 (Smc5), as does <br />           mutation of a conserved glycine in the hinge region of Spr18 (Smc5). We purified the Smc5-6 complex <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> S. <br />           pombe and identified four non-SMC components, Nse1, Nse2, Nse3, and Rad62. Nse3 is a novel protein which <br />           is related <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> the mammalian MAGE protein family, many members of which are specifically expressed in <br />           cancer tissue. In initial steps to understand the architecture of the complex, we identified two subcomplexes <br />           containing Rad18-Spr18-Nse2 and Nse1-Nse3-Rad62. The subcomplexes are probably bridged by a weaker <br />           interaction between Nse2 and Nse3. <br />  <br />  <br />    Structural maintenance of chromosomes (SMC) proteins                      (5). This is particularly true of the hinge region. While the <br /> are found in eubacteria, archaea, and eukaryotes and appear to               hinges of Smc1 to -4 share considerable sequence similarity, <br /> be present in all eukaryotes examined so far (5). In eukaryotes,             which also extends to the prokaryotic SMCs, the Smc5 and <br /> six classes of SMC proteins have been identified, and they form              Smc6 hinges are markedly different. <br /> the heterodimeric cores of three conserved protein complexes                    The prototype member of the Smc5-6 complex is the Rad18 <br /> (15). Smc1 and -3 comprise the core of the cohesins, which                   (Smc6) protein of Schizosaccharomyces pombe. (Note that S. <br /> prevent sister chromatid separation following DNA replication                pombe rad18 and Saccharomyces cerevisiae RAD18 are com- <br /> by holding the sister chromatids together. Smc2 and -4 form                  pletely unrelated genes.) rad18.X (encoding an R706C muta- <br /> the core of condensins, which play an important role in con-                 tion in the second coiled coil region close to the hinge) and <br /> densing chromosomes during mitosis. Smc5 and -6 are the core                 rad18-74 (encoding an A151T mutation in the N-terminal glob- <br /> of a complex involved in DNA repair (7). <br />                                                                              ular domain) cells are sensitive to both UV and &#226;&#65533;&#165;-irradiation <br />    SMC proteins have a common structure. The N- and C- <br />                                                                              and to methyl methanesulfonate (MMS) treatment (19, 26) <br /> terminal domains are globular and are separated by two long <br />                                                                              and are deficient in the repair of double strand breaks in DNA <br /> coiled coils interrupted by a flexible hinge. The N- and C- <br /> terminal domains contain Walker A and B motifs, respectively,                (26). Epistasis analysis suggests that Rad18 (Smc6) is involved <br /> which are brought together by the molecule folding back on                   in several recombinational repair processes, including a path- <br /> itself via the hinge such that the two coiled coils interact in an           way for removing UV damage that is distinct from nucleotide <br /> antiparallel structure. For cohesin and condensin, it has been               excision repair, a UV DNA damage tolerance pathway, and a <br /> shown that the Smc1-Smc3 and Smc2-Smc4 heterodimers are                      double-strand-break repair pathway (19, 23). After the treat- <br /> formed by intermolecular interactions of the hinge domains                   ment of cells with &#226;&#65533;&#165;-irradiation, rad18.X and rad18-74 cells <br /> (10, 14). In addition, it has been proposed that the globular                arrest the cell cycle and then, after a delay, return to the cell <br /> domains of the two SMC monomers of cohesin can also inter-                   cycle with the same kinetics as wild-type cells, even though the <br /> act to form a ring around the chromosomal DNA, encompass-                    DNA damage is not repaired in the mutant cells (1, 26). It has <br /> ing the two sister chromatids (10). The mean evolutionary                    therefore been suggested that Rad18 (Smc6) is not required to <br /> distance of the Smc5 and -6 families from the root of the                    establish a checkpoint but is needed to maintain it (11, 26). <br /> evolutionary tree is about twice that of other eukaryotic SMC                Rad18 (Smc6) also has an essential function (19). The lethality <br /> proteins, indicating a higher degree of sequence divergence                  of a rad18 deletion strain may result either from an accumu- <br />                                                                              lation of unrepaired DNA damage or from a separate function <br />                                                                              that is unrelated to the protein&#226;&#8364;&#8482;s involvement in repair. Several <br />   * Corresponding author. Mailing address: Genome Damage and <br /> Stability Centre, University of Sussex, Falmer, Brighton BN1 9RQ,            viable hypomorphic mutations have been reported which con- <br /> United Kingdom. Phone: 44-1273 678120. Fax: 44-1273 678120. E-               fer sensitivity to DNA damage while retaining viability. These <br /> mail: a.r.lehmann@sussex.ac.uk.                                              include the mutation of serine 1045 to alanine (7). By extrap- <br />   &#226;&#8364;&#160; J.S., E.T., and J.P. contributed equally to this work. <br />   &#226;&#8364;&#161; Present address: Department of Toxicogenetics, Leiden University         olation from the SMC-like structure of the Rad50 and Smc1 <br /> Medical Centre, 2333-AL Leiden, The Netherlands.                             proteins, Ser1045 is likely to be involved in binding of the <br />  <br />                                                                        172 <br />  VOL. 25, 2005                                                                                      ARCHITECTURE OF THE Smc5-6 COMPLEX                                173 <br />  <br />  <br /> &#226;&#65533;&#165;-phosphate of ATP and required for ATP hydrolysis (2, 17,                                Yeast two-hybrid analysis. The ProQuest two-hybrid system (Invitrogen) was <br /> 27).                                                                                   used to analyze interactions between human Smc5 (hSmc5) and hSmc6 hinge <br />                                                                                        regions. Transformation of the plasmids pPC86-hSmc5 hinge, pDBLeu-hSMC6 <br />    Purification of Rad18 (Smc6) from S. pombe cells showed it                          hinge, and pDBLeu-hSmc6gly506 into S. cerevisiae MaV203, growth on selective <br /> to be part of a high-molecular-weight complex with several                             media, and X-Gal (5-bromo-4-chloro-3-indolyl-&#226;&#65533;&#164;-D-galactopyranoside) assays <br /> components, one of which is its SMC protein partner, Spr18                             were performed according to the manufacturer&#226;&#8364;&#8482;s instructions. <br /> (Smc5). Two other proteins that interact with Rad18 (Smc6)                                The Matchmaker Gal4 two-hybrid system 3 (Clontech) was used to analyze <br />                                                                                        interactions between the individual components of the S. pombe complex. Trans- <br /> were recently identified by mass spectroscopy and were desig-                          formation of the constructs cloned in pGBKT7, pGADT7, and/or pACT2 (de- <br /> nated Nse1 and Nse2 (20). Like Rad18 (Smc6), Spr18 (Smc5),                             tails available on request) into S. cerevisiae Y190, growth on selective media, and <br /> Nse1, and Nse2 are essential genes (7, 20).                                            X-Gal assays were performed according to the manufacturer&#226;&#8364;&#8482;s instructions. <br />    In the present work, we show that four randomly isolated                               Antibody generation. A cDNA fragment corresponding to Rad18 (Smc6) <br />                                                                                        amino acids (aa) 152 to 684 was cloned into the vector pQE-30 (Qiagen) and <br /> temperature-sensitive mutants of Rad18 (Smc6) have muta- <br />                                                                                        expressed in Escherichia coli as an N-terminal hexahistidine-tagged fusion pro- <br /> tions at an identical glycine residue in the hinge region, that                        tein. The protein was purified to near homogeneity under denaturing conditions <br /> Rad18 (Smc6) and Spr18 (Smc5) interact via their hinge re-                             by Ni2&#226;&#171;&#185;-nitrilotriacetic acid affinity chromatography and then was used to inoc- <br /> gions, and that this interaction is destroyed in the temperature-                      ulate two rabbits. Antibodies were affinity purified by use of an antigen immo- <br /> sensitive rad18 mutants or if a conserved glycine in the hinge                         bilized with Aminolink Plus coupling gel (Pierce) according to the manufactur- <br />                                                                                        er&#226;&#8364;&#8482;s instructions. <br /> region of Spr18 (Smc5) is mutated. Aside from the Rad18                                   Polyclonal antibodies to Spr18 (Smc5), raised in sheep, and to Nse2, raised in <br /> (Smc6) and Spr18 (Smc5) core proteins, we have identified                              rabbits, are described elsewhere (7) and in the accompanying paper (1a). Anti- <br /> four other polypeptides in the affinity-purified Rad18 (Smc6)                          bodies to the Nse1 and Nse3 proteins were raised in rabbits against a mix of two <br /> complex. These include Nse1, Nse2, and two new components.                             peptides each (for Nse1, EKERQDGLSDKHKFIL and DFKIKRVQDQLDG <br />                                                                                        RL; for Nse3, LRRPATSNANSSNL and TEYRQEYQNQSSSSAA). Nse3 <br /> We have begun to delineate the architecture of the complex by                          antibodies were purified against the same peptides linked to N-hydroxysuccin- <br /> determining which of the individual subunits interact in vitro.                        imide-activated Sepharose (Pharmacia). <br />                                                                                           These antibodies identified the corresponding in vitro-translated proteins. <br />                                                                                        They also detected proteins of the appropriate sizes in cell extracts which were <br />                          MATERIALS AND METHODS                                         not detected by the corresponding preimmune sera. <br />                                                                                           Purification of the Rad18 (Smc6) complex. Forty liters of S. pombe Myc-Rad18 <br />    Isolation of temperature-sensitive rad18 mutants. A rad18 deletion construct <br />                                                                                        cells in which the genomic rad18 gene was N-terminally tagged with the c-myc <br /> in which the 3.7-kb AccI fragment spanning the entire open reading frame <br />                                                                                        epitope were grown to mid-log phase, harvested, and washed, and the cell pellet <br /> (ORF) was replaced with the arg3 gene was generated essentially as described by <br />                                                                                        was resuspended in an equal volume of lysis buffer A (45 mM HEPES [pH 7.8], <br /> Lehmann et al. (19). A diploid strain was constructed with the genotype ura4- <br />                                                                                        300 mM KCl, 10% glycerol, 5 mM EDTA, 5 mM EGTA, 80 mM &#226;&#65533;&#164;-glycerophos- <br /> D18/ura4-D18 leu1-32/leu1-32 arg3-D4/arg3-D4 h&#226;&#171;&#185;/h&#226;&#171;&#185;, and the deletion plasmid <br />                                                                                        phate, 12 mM NaF, 0.1 mM sodium orthovanadate, 1 mM dithiothreitol [DTT], <br /> was used to transform this strain to arginine prototrophy. Colonies were <br />                                                                                        1 mM phenylmethylsulfonyl fluoride [PMSF], and a protease inhibitor cocktail <br /> screened for the replacement event by Southern blotting, and an h&#226;&#171;&#185;/h90 deriv- <br />                                                                                        consisting of 5 &#226;&#65533;&#174;g each of trypsin inhibitor, pepstatin, leupeptin, and aprotinin/ <br /> ative was isolated. This strain was transformed with the MH42-rad18&#226;&#171;&#185; plasmid <br />                                                                                        ml, 10 &#226;&#65533;&#174;g each of bestatin and E-64/ml, and 50 &#226;&#65533;&#174;g of chymostatin/ml). The cells <br /> (ura4&#226;&#171;&#185;) and sporulated, and haploid progeny with the genotype rad18::arg3&#226;&#171;&#185; <br />                                                                                        were snap-frozen as droplets in liquid nitrogen and lysed with a Retsch grinder, <br /> ura4-D18 leu1-32 arg3-D4 h&#226;&#171;&#185; (MH42-rad18&#226;&#171;&#185;) were selected. A randomly mu- <br />                                                                                        with frequent additions of liquid nitrogen. The lysate was clarified by centrifu- <br /> tagenized pool of rad18 genes was created by treating the MH41-rad18 plasmid           gation and subjected to ammonium sulfate precipitation. The 13-to-50%-cut <br /> (leu&#226;&#171;&#185;) with hydroxylamine (18). Mutagenized plasmids were transformed into             precipitate containing Rad18 (Smc6) was resuspended in 1 volume of binding <br /> the rad18::arg3 (MH42-rad18&#226;&#171;&#185;) strain, and transformants were selected on me-           buffer B (25 mM HEPES [pH 7.8], 300 mM KCl, 10% glycerol, 2 mM EDTA, 3 <br /> dium lacking leucine and arginine and then replica plated onto medium contain-         mM EGTA, 3 mM MgCl2, 0.2 mM DTT, 0.5 mM PMSF, and the protease <br /> ing fluoroorotic acid (FOA) to select for the loss of the MH42-rad18&#226;&#171;&#185; plasmid.         inhibitor cocktail described above) and extensively dialyzed against the same <br /> For the identification of temperature-sensitive alleles of rad18, the resulting        buffer. The dialyzed lysate was diluted onefold with binding buffer B and incu- <br /> 15,000 colonies were screened by replica plating onto plates that were incubated       bated overnight at 4&#194;&#176;C with 5 ml of protein G-Sepharose beads that had been <br /> at 27 and 36&#194;&#176;C. Four clones that could grow at 27&#194;&#176;C but not at 36&#194;&#176;C were isolated.     previously cross-linked to a monoclonal anti-Myc antibody (9E10). The beads <br /> Plasmids were recovered from these temperature-sensitive strains, and the mu-          were washed once with 10 volumes of binding buffer B, four times with 10 <br /> tation sites were identified by sequencing. For genomic integration of tempera-        volumes of wash buffer 1 (25 mM HEPES [pH 7.8], 500 mM KCl, 8% glycerol, <br /> ture-sensitive alleles (T1 to T3), the mutations were incorporated into the 7.8-kb     2 mM EDTA, 2 mM MgCl2, 0.5 mM PMSF, 0.1% NP-40), and twice with wash <br /> genomic clone in plasmid SpF (19), and rad18::arg3 haploid strains harboring           buffer 2 (25 mM HEPES [pH 7.8], 100 mM KCl, 8% glycerol, 2 mM EDTA, 2 <br /> each of these plasmids (SpF.T1 to SpF.T3) were generated. These strains were           mM MgCl2, 0.5 mM PMSF, 0.01% NP-40). The beads were transferred to a <br /> each transformed with the appropriate 7.8-kb linear fragment of rad18 DNA              column, incubated with 2 volumes of wash buffer 2 containing 5 mM ATP for 10 <br /> containing the T1, T2, or T3 mutation, and integration events were screened by         min at 4&#194;&#176;C, and then washed once more with 10 volumes of wash buffer 2. The <br /> the use of FOA to select for the loss of the SpF plasmid. Southern blotting was        Rad18 (Smc6) complex was eluted at 30&#194;&#176;C for 1 h, with 3 mg of peptide/ml <br /> used to confirm the correct replacement of the arg3 deletion with the rad18            corresponding to the Myc epitope (AEEQKLISEEDL). <br /> temperature-sensitive mutant alleles.                                                     Mass spectrometric analysis of the Rad18 (Smc6) complex. Purified Myc- <br />    DNA damage sensitivity tests. For UV irradiation, cells were grown to mid-log       Rad18 (Smc6) complex components were precipitated with 4 volumes of ace- <br /> phase at the appropriate temperature (25 or 30&#194;&#176;C), plated on yeast extract (YE)        tone, separated by sodium dodecyl sulfate&#226;&#8364;&#8220;4 to 20% polyacrylamide gradient gel <br /> plates, and irradiated at 254 nm by use of a Stratalinker chamber (Stratagene).        electrophoresis (SDS-PAGE), and stained with colloidal Coomassie blue after <br /> For &#226;&#65533;&#165;-irradiation, exponentially growing cells were irradiated in suspension by a      fixation with 40% methanol&#226;&#8364;&#8220;10% acetic acid for 1 h. Individual bands were <br /> 137 <br />     Cs source (dose rate, 8.5 Gy min&#226;&#171;&#186;1) prior to plating on YE plates. All plates      excised and identified by trypsin digestion and mass fingerprinting by mass <br /> were incubated at 25 or 30&#194;&#176;C as appropriate, and colonies were counted after 4         spectrometry at the COGEME Proteome Service Facility 1, University of Ab- <br /> to 5 days. <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>Survival</span> was expressed as a percentage of the number of colonies that       erdeen. Proteins were identified by searching databases with the MASCOT <br /> were formed in the absence of irradiation.                                             search engine (Matrix Science). <br />    Cell cycle synchronization. Cells were synchronized by lactose gradient cen-           In vitro transcription-translation. ORFs corresponding to proteins or protein <br /> trifugation as described by Barbet and Carr (3). G2 cells were resuspended in          fragments were amplified from S. pombe cDNAs and singly or coexpressed in <br /> fresh prewarmed YE and incubated at 36&#194;&#176;C for the duration of the experiment.           vitro as Myc-, hemagglutinin (HA)-, S- or untagged proteins from the pEPEX, <br /> Aliquots were removed at 20-min intervals, plated on YE plates, and incubated          pET-Duet, pTriEx, or pGem-T-Easy vector by use of the TNT Quick coupled <br /> at 25&#194;&#176;C for 4 to 5 days to determine cell viability. At the same time, aliquots were   transcription-translation system (Promega) in 50-&#226;&#65533;&#174;l reactions according to the <br /> fixed in methanol and stained with DAPI (4&#226;&#172;&#732;,6&#226;&#172;&#732;-diamidino-2-phenylindole) and           manufacturer&#226;&#8364;&#8482;s instructions. <br /> calcofluor in order to determine the percentage of cells that passed mitosis.             The following constructs were used: for Rad18, pEPEX-myc-Rad18, pEPEX- <br />  174       SERGEANT ET AL.                                                                                                              MOL. CELL. BIOL. <br />  <br /> myc-Rad18-hinge (aa 504 to 719), pEPEX-HA-Rad18-hinge (aa 504 to 719), and            mutations at Gly551 (T2 was used for the Gly-to-Arg muta- <br /> pEPEX-myc-Rad18-gly551-hinge; for Spr18, pEPEX-Spr18, pEPEX-Spr18-                    tion) was then introduced into the genome of wild-type S. <br /> gly529, pEPEX-myc-Spr18-hinge (aa 393 to 620), pEPEX-HA-Spr18-hinge (aa <br /> 393 to 620), pEPEX-myc-Spr18-gly529-hinge, pEPEX-HA-Spr18-gly529-hinge, <br />                                                                                       pombe cells. The resulting strains were analyzed for growth and <br /> pET-Duet-Spr18 &#226;&#8364;&#339;headless&#226;&#8364;&#65533; (aa 171 to 910), pGem-Spr18 (aa 226 to 836),                viability at 27 and 36&#194;&#176;C. All three strains grew normally and <br /> pGem-Spr18 (aa 324 to 731), and pET-Duet-Spr18 &#226;&#8364;&#339;heads&#226;&#8364;&#65533; (aa 2 to 174 plus aa           were viable at 27&#194;&#176;C (Fig. 1A and C). At 36&#194;&#176;C, growth was <br /> 897 to 1065); for Nse1, pEPEX-Nse1 and pGem-T-Easy-Nse1(aa 1 to 116); for             severely impaired and cells began to lose viability after 6 h (Fig. <br /> Nse2, pET-Duet-Nse2 (aa 1 to 178) and pET-Duet-Nse2 (aa 114 to 250); for <br />                                                                                       1B and D). At 27&#194;&#176;C, the morphology of the rad18 (smc6) <br /> Nse3, pEPEX-myc-Nse3 and pEPEX-HA-Nse3; and for Rad62, pTriEx-4- <br /> Rad62. Details of the construction of the plasmids are available on request.          mutant cells was indistinguishable from that of wild-type cells, <br />    Preparation of recombinant proteins. The Nse1, Nse2, and Nse3 proteins were        whereas within 6 h of shifting the temperature to 36&#194;&#176;C, the <br /> expressed as glutathione S-transferase (GST) fusions. The expression of pGEX-         cells showed a variety of abnormal phenotypes, including ab- <br /> Nse1 and pGEX-Nse2 constructs was carried out in BL21 cells, while pGEX-              errant nuclear segregation, cut phenotypes, abnormal septa, <br /> Nse3 and pTriEx-Rad62 constructs were expressed from Rosetta-gami B cells <br /> (Novagen). The cells were induced with 0.5 mM IPTG (isopropyl-&#226;&#65533;&#164;-D-thiogalac- <br />                                                                                       and elongated cells (Fig. 1E). These phenotypes are charac- <br /> topyranoside) and harvested after 4 h of induction. The bacteria were lysed with      teristic of terminal phenotypes that were observed previously <br /> HEPES buffer C (25 mM HEPES [pH 7.5], 200 mM KCl, 3 mM MgCl2, 10%                     for rad18 (smc6) deletion mutants (19). Using synchronized <br /> glycerol, 1 mM DTT, 1 mM PMSF, 0.2% NP-40, and protease inhibitors).                  cells, we found that at 36&#194;&#176;C, the cells were able to pass through <br />    GST pull-down experiments. For GST pull-down experiments, 50 &#226;&#65533;&#174;l of GST <br />                                                                                       at least two rounds of mitosis (Fig. 1F), and viability was <br /> fusion proteins immobilized on glutathione-Sepharose beads (Amersham Phar- <br /> macia Biotech) was incubated with 5 to 10 &#226;&#65533;&#174;l of in vitro-expressed proteins in a      gradually lost after each round of mitosis (Fig. 1G). To deter- <br /> total volume of 200 &#226;&#65533;&#174;l of HEPES buffer C containing 1 mM ATP, 50 mM                   mine if this ability to maintain viability following the shift to <br /> creatine phosphate, and 1 U of creatine phosphokinase. After 2 h at 4&#194;&#176;C, the          the restrictive temperature reflected a relatively long half-life <br /> beads were washed twice with HEPES buffer C containing 400 mM KCl and                 of the Rad18 (Smc6) protein, we incubated the rad18.T2 mu- <br /> 0.4% NP-40 and once with HEPES buffer C containing 100 mM KCl and 0.1% <br /> NP-40. Input, unbound, and bound fractions were separated by SDS-PAGE, <br />                                                                                       tant at 36&#194;&#176;C for different times and analyzed the amount of <br /> transferred to nitrocellulose membranes, and analyzed by phosphorimaging and          Rad18 (Smc6) protein by immunoblotting. Figure 1H shows <br /> immunoblotting with anti-GST (Amersham Pharmacia Biotech).                            that there was a substantial reduction in the amount of Rad18 <br />    Myc pull-down experiments. For Myc pull-down experiments, 5 to 10 &#226;&#65533;&#174;l of in         (Smc6) protein 1 h after the temperature shift and that this <br /> vitro-expressed proteins in a total volume of 200 &#226;&#65533;&#174;l of HEPES buffer C contain- <br />                                                                                       reduced level remained fairly constant for several hours. In- <br /> ing 1 mM ATP, 50 mM creatine phosphate, and 1 U of creatine phosphokinase <br /> was mixed with 10 &#226;&#65533;&#174;l of an anti-Myc monoclonal antibody and incubated over-           terestingly, the amounts of Spr18 (Smc5) and Nse2 remained <br /> night at 4&#194;&#176;C. Fifty microliters of protein G-Sepharose beads (Amersham Phar-          unchanged despite the loss of the Rad18 (Smc6) protein. <br /> macia Biotech) was then added, and the mixture was incubated for 1 h. The                We also examined the sensitivity of the rad18.T2 cells to <br /> beads were washed twice with HEPES buffer C containing 400 mM KCl and                 DNA damage at the permissive and semipermissive tempera- <br /> 0.4% NP-40 and once with HEPES buffer C containing 100 mM KCl and 0.1% <br /> NP-40. Alternatively, 20 &#226;&#65533;&#174;l of the reaction was added to 180 &#226;&#65533;&#174;l of binding buffer <br />                                                                                       tures. Figure 2A shows that the mutant cells were mildly sen- <br /> B (described above) and incubated overnight at 4&#194;&#176;C with 100 &#226;&#65533;&#174;l of protein             sitive to UV irradiation at 25&#194;&#176;C but that this sensitivity was <br /> G-Sepharose beads that had been previously cross-linked with a monoclonal             dramatically increased at 30&#194;&#176;C (Fig. 2B) and far exceeded the <br /> anti-Myc antibody (9E10). The beads were washed twice with 10 volumes of wash         sensitivity of the rad18.X mutant. Sensitivity to &#226;&#65533;&#165;-irradiation <br /> buffer 1 (described above) and twice with 10 volumes of wash buffer 2. Input, <br />                                                                                       was barely affected at 25&#194;&#176;C (Fig. 2C) but was more marked at <br /> unbound, and bound fractions were separated by SDS-PAGE and analyzed by <br /> phosphorimaging and/or immunoblotting with respective antibodies.                     30&#194;&#176;C, although the cells were less sensitive than the rad18.X <br />    S tag pull-down experiments. For S tag pull-down experiments, 50 &#226;&#65533;&#174;l of S tag       mutant (Fig. 2D). <br /> fusion proteins immobilized on protein S-agarose beads (Novagen) was incu-               To determine if the temperature sensitivity of rad18.T2 cells <br /> bated with 5 to 10 &#226;&#65533;&#174;l of in vitro-expressed proteins in a total volume of 200 &#226;&#65533;&#174;l of   could be suppressed by overexpression of other genes involved <br /> HEPES buffer MB (25 mM HEPES [pH 7.2], 100 mM NaCl, 5 mM EDTA, 5% <br /> glycerol, 1 mM DTT, 1 mM PMSF, 0.1% NP-40, and protease inhibitors). After <br />                                                                                       in either the DNA damage checkpoint, DNA repair, or DNA <br /> 2 h at 4&#194;&#176;C, the beads were washed three times with HEPES buffer MB. Input,            replication, we transformed cells with plasmids expressing a <br /> unbound, and bound fractions were separated by SDS-PAGE, transferred to               variety of genes involved in these processes under the control <br /> nitrocellulose membranes, and analyzed by phosphorimaging and immunoblot-             of the thiamine-repressible nmt1 promoter. In no case did the <br /> ting with protein S conjugated with horseradish peroxidase (Novagen). <br />                                                                                       overexpression of any of these genes suppress the temperature <br />                                                                                       sensitivity of rad18.T2 (data not shown). <br />                                                                                          Interaction between the hinge regions of Smc5 and Smc6. <br />                                   RESULTS <br />                                                                                       Our finding that four independent temperature-sensitive <br />    Temperature-sensitive mutants of rad18 (smc6) with muta-                           clones were all mutated at the same glycine residue in the <br /> tions in the hinge region. Since rad18 (smc6) is an essential                         hinge region of Rad18 (Smc6) points to the importance of this <br /> gene in S. pombe, we were interested in isolating conditionally                       region in maintaining the function of the Smc5-6 complex. The <br /> lethal mutants. To this end, we randomly mutagenized a rad18                          hinge region is highly conserved within the Smc6 family (Fig. <br /> (smc6)-containing plasmid by using hydroxylamine and then                             2E), and Gly551 is conserved in all species. Although the hinge <br /> used the mutagenized population to isolate plasmids that were                         region mediates the heterodimerization of Smc1 with Smc3 in <br /> temperature sensitive in their ability to rescue the lethality of                     cohesin and of Smc2 with Smc4 in condensin, the hinge regions <br /> rad18 (smc6) deletion mutants. Four independent tempera-                              of Smc6 and Smc5 have minimal sequence similarity to the <br /> ture-sensitive plasmid-containing clones (T1 to T4) were ob-                          hinges of Smc1 to -4 (compare Fig. 2E with Fig. 2F). In par- <br /> tained, from which the plasmids were recovered and se-                                ticular, there is a series of glycine residues that have been <br /> quenced. Strikingly, all of the plasmids were mutated at Gly551                       shown to be vital for hinge-mediated dimerization and that are <br /> in the hinge region of the protein. In plasmids T2 and T4,                            conserved not only in the Smc1 to -4 families (Fig. 2F) but also <br /> Gly551 was mutated to Arg, whereas in T1 and T3 it was                                in bacterial SMC proteins (13, 14). These glycine residues are <br /> mutated to Glu and Lys, respectively. Each of these three                             not found in the Smc6 or Smc5 family. <br />  VOL. 25, 2005                                                                        ARCHITECTURE OF THE Smc5-6 COMPLEX                       175 <br />  <br />  <br />  <br />  <br />   FIG. 1. Temperature-sensitive mutants of rad18 (smc6). The wild type and three temperature-sensitive mutants of rad18 (smc6) were incubated <br /> at 27&#194;&#176;C (A and C) or 36&#194;&#176;C (B and D), and the growth (A and B) and viability (C and D) of each strain were measured at different times. <br /> (E) Wild-type and rad18.T2 cells were grown at 36&#194;&#176;C, and at different times the cells were fixed in methanol and stained with DAPI and calcofluor. <br /> Cells that were synchronized by lactose gradient centrifugation were incubated at 36&#194;&#176;C, and at various times the numbers of cells passing mitosis <br /> (F) and the cell viability (G) were measured. (H) Cell lysates from wild-type and rad18.T2 cells grown at 36&#194;&#176;C for 0 to 6 h were analyzed by <br /> SDS-PAGE and immunoblotting with antibodies to Rad18 (Smc6), Spr18 (Smc5), Nse2, and tubulin. <br />  176     SERGEANT ET AL.                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 2. Mutations in the hinge region cause sensitivity to DNA damage. Wild-type, rad18.X, and rad18-T2 cells grown at 25&#194;&#176;C (A and C) or <br /> 30&#194;&#176;C (B and D) were exposed to different doses of UV (A and B) or &#226;&#65533;&#165;-rays (C and D), and their viabilities were determined at 25&#194;&#176;C (A and C) <br /> or 30&#194;&#176;C (B and D). (E) Alignment of the hinge regions of SMC6 proteins from humans, mice, Caenorhabditis elegans, D. melanogaster, A. thaliana, <br /> S. pombe, and S. cerevisiae. The conserved residue corresponding to Gly551 in Rad18 (Smc6) is indicated with an arrow. (F) Alignment of the hinge <br /> regions of the Smc1 (Psm1), Smc2 (Cut3), Smc3 (Psm3), and Smc4 (Cut14) proteins of S. pombe. Conserved glycine residues that were shown to <br /> be important for hinge-mediated interactions (13) are indicated with asterisks. (G) Alignment of part of the hinge regions of SMC5 proteins <br /> corresponding to aa 505 to 585 of the S. pombe protein (the hinge region spans aa 400 to 620). The conserved Gly529 residue is indicated. <br />  <br />  <br />    In order to determine if the hinge region also mediates the             tween them. We were not able to detect any interaction of <br /> interaction between Rad18 (Smc6) and Spr18 (Smc5), we gen-                 His-tagged N-terminal globular domains of Rad18 (Smc6) or <br /> erated different tagged fragments of the proteins by in vitro              Spr18 (Smc5) with the C-terminal globular domain of Rad18 <br /> transcription-translation and analyzed the interactions be-                (Smc6) (data not shown). However when the hinge region of <br />  VOL. 25, 2005                                                                     ARCHITECTURE OF THE Smc5-6 COMPLEX                  177 <br />  <br />  <br />                                                                          abolished (Fig. 3A, lane 9, bottom panel). Singly expressed <br />                                                                          HA-tagged hinge constructs were not precipitated by the anti- <br />                                                                          Myc beads alone (Fig. 3A, lanes 12 and 15). These data provide <br />                                                                          strong evidence that Rad18 (Smc6) and Spr18 (Smc5), like the <br />                                                                          other SMC proteins, interact through their hinge regions and <br />                                                                          that the mutation of glycine 551 confers temperature sensitivity <br />                                                                          by weakening this interaction. <br />                                                                             The hinge regions of Spr18 (Smc5) orthologs are well con- <br />                                                                          served between species but differ from those of other SMC <br />                                                                          proteins. As shown in Fig. 2G, there is a conserved glycine at <br />                                                                          aa 529 in the hinge region of the S. pombe Spr18 (Smc5) pro- <br />                                                                          tein. We mutated this glycine to arginine and examined the <br />                                                                          effect of this mutation on the interaction with the Rad18 <br />                                                                          (Smc6) hinge. Figure 3B shows that, as with the Gly551 muta- <br />                                                                          tion in Rad18 (Smc6), a Gly529Arg mutation in Spr18 (Smc5) <br />                                                                          abolishes the interaction between the hinges (compare lanes 3 <br />                                                                          and 6 of Fig. 3B with the corresponding lanes of Fig. 3A). <br />                                                                             Spr18 (Smc5) and Rad18 (Smc6) have human orthologs <br />                                                                          (25). We used a yeast two-hybrid system to analyze interactions <br />                                                                          between the hinge regions of hSmc5 and hSmc6. Figure 3C <br />                                                                          shows the results of this analysis. As with S. pombe, our results <br />                                                                          show that the hinge regions of the two proteins interact. In- <br />                                                                          teractions were measured between the hinge regions of hSmc5 <br />                                                                          and either wild-type hSmc6 or hSmc6 with a mutation to Arg <br />                                                                          at Gly506 (corresponding to Gly551 of the S. pombe protein) <br />                                                                          by the use of three different reporter genes (HIS3, URA3, and <br />                                                                          lacZ). Interaction-dependent transcriptional activation of the <br />                                                                          HIS3 and URA3 genes allowed growth on plates lacking histi- <br />                                                                          dine (Fig. 3C, panel 2) but inhibited growth on medium con- <br />    FIG. 3. SMC6 hinge domain interacts with SMC5 hinge domain, <br /> and mutations at conserved glycine residues abolish this interaction. <br />                                                                          taining 5-FOA (Fig. 3C, panel 3). Induction of the lacZ gene <br /> (A) Tagged S. pombe hinge constructs (Myc-Rad18 [Smc6], Myc-Spr18        resulted in a blue color when assayed with X-Gal (Fig. 3C, <br /> [Smc5], HA-Rad18 [Smc6], HA-Spr18 [Smc5], and Myc-Rad18gly551)           panel 4). Positive controls in the assay were the weakly inter- <br /> were expressed in vitro as indicated and were incubated overnight at     acting proteins Rb and E2F1 (control B) and the strongly <br /> 4&#194;&#176;C with an anti-Myc antibody that was previously cross-linked to        interacting Drosophila proteins DP and E2F1 (control C). The <br /> protein G-Sepharose beads to precipitate Myc-tagged constructs. The <br /> beads were washed extensively, and the input (I), unbound (UB), and      negative control was empty vectors (control A). Our results <br /> bound (B) proteins were analyzed by SDS-PAGE and immunoblotting          clearly demonstrate that the hinge regions of hSmc5 and <br /> with either anti-Myc or anti-HA antibodies. Lanes 10 to 15 are nega-     hSmc6 interact (Fig. 3C, top row). The strength of the inter- <br /> tive controls without any Myc-tagged protein. (B) Immunoprecipita-       action was intermediate between the weakly and strongly in- <br /> tions were performed as described for panel A, but with constructs <br /> containing the Spr18 Gly529 mutation. (C) Yeast two-hybrid analysis <br />                                                                          teracting controls. The mutation of Gly506 in hSmc6 com- <br /> of hSmc5 and hSmc6. Yeast strains harboring the plasmids GAL4-DB         pletely abolished its interaction with hSmc5 (Fig. 3C, second <br /> hSmc6 hinge/GAL4-AD hSmc5 hinge and GAl4-DB hSmc6 Gly506                 row). Thus, the interaction between the hinge regions and the <br /> hinge/GAL4-AD hSmc5 hinge along with the control strains A (no           importance of Gly551 in this interaction are conserved in yeast <br /> interaction), B (weak interaction), and C (strong interaction) were      and humans. <br /> patched onto an SC-Leu-Trp master plate and incubated at 30&#194;&#176;C for <br /> 18 h (panel 1). Cells from this plate were replica plated onto SC-Leu-      Composition of the SMC5-6 complex. In an earlier study, it <br /> Trp-His plus 25 mM 3-amino-2,4,6-triazole (panel 2), SC-Leu-Trp plus     was reported that Rad18 (Smc6) and Spr18 (Smc5) are part of <br /> 0.2% 5-FOA (panel 3), and enriched yeast extract medium containing       a high-molecular-weight complex which contains several other <br /> a nylon membrane (for X-Gal assay) (panel 4) before growth at 30&#194;&#176;C       components (7). Using cells in which the genomic rad18 gene <br /> for an additional 2 days. <br />                                                                          was N-terminally tagged with the c-Myc epitope, we were able <br />                                                                          to purify the complex partially by immuno-affinity chromatog- <br /> Rad18 (Smc6) tagged with the c-Myc epitope was mixed with                raphy with anti-Myc beads but were hindered from analyzing <br /> the HA-tagged Spr18 (Smc5) hinge region and immunopre-                   the components because of a strong contaminating 70-kDa <br /> cipitated with an anti-Myc antibody, both Myc-Rad18 (Smc6)               protein in all of our purifications (see Fig. 3C of reference 7) <br /> and HA-Spr18 (Smc5) were quantitatively recovered in the                 (Fig. 4A). Even though it was shown previously that this pro- <br /> immune precipitate (Fig. 3A, lane 3). In a converse experiment           tein bound nonspecifically to anti-Myc beads (7), we were <br /> with Myc-Spr18 (Smc5) mixed with HA-Rad18 (Smc6) and                     unable to remove it from the complex by use of a variety of <br /> immunoprecipitated with anti-Myc, both proteins were again               different chromatographic procedures. Using mass spectros- <br /> found in the immune complex (Fig. 3A, lane 6). When com-                 copy, we identified the contaminating protein as Hsp70 and <br /> parable experiments were performed with the Rad18 (Smc6)-                ultimately were able to remove it by washing the anti-Myc <br /> Gly551 mutant construct, the interaction between the Myc-                beads with ATP (Fig. 4B). The complex, which remained at- <br /> Spr18 (Smc6) and HA-Spr18 (Smc5) hinge fragments was                     tached to the beads, was eluted with the c-Myc peptide, elec- <br />  178     SERGEANT ET AL.                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 4. Purification and identification of Rad18 (Smc6)/Spr18 (Smc5) complex components. A lysate prepared from a Myc-tagged Rad18 <br /> (Smc6) strain was incubated overnight at 4&#194;&#176;C with an anti-Myc antibody that was previously cross-linked to protein G-Sepharose beads. The beads <br /> were washed extensively, and bound proteins were eluted by incubation with an excess peptide corresponding to the Myc epitope, separated by <br /> SDS-PAGE, and stained with colloidal Coomassie blue (A). (B) The intense contaminating band at 70 kDa was identified as Hsp70 by mass <br /> spectrometry and was effectively removed by washing the beads with 5 mM ATP. (C) The remaining proteins were eluted as before and separated <br /> by SDS-PAGE, and individual bands were excised and identified by trypsin digestion and mass fingerprinting. (D) Antibodies were raised against <br /> Nse1, Nse2, Nse3, and Rad62 and were used, along with Rad18 (Smc6) and Spr18 (Smc5) antibodies, to confirm the presence of the identified <br /> proteins in an independent preparation of the complex by immunoblotting. (E) Lysates prepared from Myc-tagged Rad18 (Smc6) (lanes 1 to 3), <br /> HA-tagged Nse3 (lanes 7 to 9), or untagged (lanes 4 to 6 and 10 to 12) strains were incubated overnight at 4&#194;&#176;C with either anti-Myc or anti-HA <br /> antibodies that were previously cross-linked to protein G-Sepharose beads. The beads were washed extensively, and the input (I), bound (B), and <br /> unbound (U) proteins were analyzed by SDS-PAGE and immunoblotting with anti-Rad18 and anti-Nse3 antibodies. (F) Alignment of Nse3, <br /> S. cerevisiae YDR288W, and the MAGE consensus sequence KOG4562. <br />  <br />  <br />  <br /> trophoresed in an SDS-PAGE gel, and stained with colloidal                (8, 11, 20) and Nse2 (20) proteins. One of the other two we <br /> Coomassie blue (Fig. 4C). The bands were excised and ana-                 designated Nse3, and the other is orthologous to S. cerevisiae <br /> lyzed by mass spectroscopy. Apart from Rad18 (Smc6) and                   Qri2. The ortholog of Qri2 was recently shown to be identical <br /> Spr18 (Smc5), four other proteins were identified (Table 1).              to the S. pombe Rad62 protein (21). We raised antibodies to <br /> Two of them corresponded to the previously identified Nse1                Nse1, Nse2, Nse3, and Rad62 and used these to probe immu- <br />  VOL. 25, 2005                                                                 ARCHITECTURE OF THE Smc5-6 COMPLEX                 179 <br />  <br />  <br />        TABLE 1. Identification of Rad18 (Smc5-6) complex             were also synthesized singly or together by in vitro transcrip- <br />               components by mass spectrometry                        tion-translation (IVT), and their interactions were analyzed in <br />                                       No. of peptides                several different ways. <br />                            Protein                      % Sequence <br />         Protein <br />                           size (aa) <br />                                        matched/total <br />                                                          coverage       Interaction of Nse2 with Spr18 (SMC5). GST-Nse2 pull- <br />                                        no. searched <br />                                                                      down assays were performed with 35S-labeled Rad18 (Smc6) or <br /> Rad18                      1,140          10/13             9        Spr18 (Smc5). As shown in Fig. 5A (bottom panels), Nse2 was <br /> Spr18                      1,065          15/24            16        bound to the beads. Rad18 (Smc6) did not bind to the GST- <br /> Nse3 (spcc645.04)            328           9/19            23        Nse2 beads (lane 2, top panel), whereas Spr18 (Smc5) bound <br /> Rad62 (spbc20F10.04c)        300           6/25            26 <br /> Nse2                         250           5/14            27        strongly (lane 4, top panel). When Rad18 (Smc6) and Spr18 <br /> Nse1                         232           8/13            26        (Smc5) were either synthesized together in vitro or synthesized <br />                                                                      separately and then mixed together, significant proportions of <br />                                                                      both Rad18 (Smc6) and Spr18 (Smc5) were bound (Fig. 5A, <br /> noblots of the complex after electrophoresis in SDS-PAGE             lane 7, top panel) compared to binding to GST beads alone <br /> gels. Figure 4D confirms that these four proteins are indeed         (i.e., without GST-Nse2) (lane 10). Similar results were ob- <br /> present in the complex. The presence of Nse1, Nse2, and Nse3         tained irrespective of whether the Rad18 (Smc6) and Spr18 <br /> as members of the complex and as free proteins was also              (Smc5) proteins were coexpressed or not (not shown). We <br /> analyzed by gel filtration (see the accompanying paper [1a]).        conclude that Nse2 binds to the Rad18 (Smc6)-Spr18 (Smc5) <br />    Nse3. To further confirm that Nse3 is part of the complex,        heterodimer via the Spr18 (Smc5) component. <br /> we immunoprecipitated Rad18 (Smc6) from cells in which it               Nse2 contains a RING finger motif that is characteristic of <br /> was tagged with Myc in the genome and showed by immuno-              SUMO ligases in its C-terminal half. We have shown, in the <br /> blotting that Nse3 was present in the immunoprecipitates             accompanying paper, that Nse2 has SUMO ligase activity that <br /> (Fig. 4E, lane 2). No Nse3 was present in immunoprecipi-             is dependent on this RING finger motif (1a). In order to see if <br /> tates from untagged cells (lane 5). In converse experiments,         this motif was involved in binding to Spr18 (Smc5), we gener- <br /> we HA-tagged Nse3 in the genome and showed that Rad18                ated two fragments of Nse2, from aa 1 to 178 and from aa 114 <br /> (Smc6) was present in anti-HA immunoprecipitates (Fig. 4E,           to 250. Each of these fragments was C-terminally tagged with <br /> lane 8; note that the immunoprecipitation of HA-Nse3 with            S protein and synthesized in vitro, incubated with IVT Spr18 <br /> anti-HA [lane 8] was less efficient than that of Myc-Rad18           (Smc5), and then bound to protein S beads. Figure 5B (bottom <br /> (Smc6) with anti-Myc [lane 2]). Again, no Rad18 was precip-          panels) shows that both S-tagged Nse2 fragments bound to the <br /> itated from extracts of untagged cells (lane 11). These data         beads (lanes 3 and 6) but that Spr18 (Smc5) bound only to the <br /> confirm that Nse3 is in the same protein complex as Rad18            N-terminal fragment (lane 3, top panel), not the C-terminal <br /> (Smc6). Deletion of most of the nse3 ORF showed that it is an        fragment (lane 6, top panel) or protein beads alone (lane 9). <br /> essential gene, like all of the other components of the complex.     Similar results were obtained with bacterially expressed Nse2 <br /> The terminal morphology of the deletion mutant was a heter-          fragments (not shown). Thus, the RING finger motif of Nse2 is <br /> ogeneous mixture of abnormal cells, as has been found previ-         not required for binding to Spr18 (Smc5). <br /> ously with rad18, spr18, and rad62 mutants (H. Morikawa, I.             To investigate which part of Spr18 (Smc5) is involved in the <br /> Miyabe, T. Morishita, and H. Shinagawa, unpublished results).        interaction with Nse2, we made several different constructs of <br />    Nse3 is a 328-aa protein and is designated SPCC645.04 in          Spr18 (Smc5). The first, Spr18 (Smc5) heads, contained the <br /> the S. pombe sequencing project. BLAST searching of the pro-         globular N- and C-terminal domains joined by a short linker. <br /> tein databases identified a putative ortholog in the S. cerevisiae   The other, Spr18 (Smc5) headless constructs, lacked the glob- <br /> database, an ORF encoding a 303-aa protein of unknown func-          ular heads and contained different lengths of the central coiled <br /> tion designated YDR288W. An alignment of the two proteins            coil together with the hinge domain. In GST pull-down assays <br /> showed a relatively low level of identity of 15.5% (Fig. 4F).        with GST-Nse2 and IVT Spr18 (Smc5) heads, only a very small <br /> Further iterative BLAST searching revealed sequence similar-         amount of the Spr18 heads bound to the beads (Fig. 5C, lane <br /> ity to a melanoma antigen-encoding gene (MAGE) domain.               3, top panel) compared to the control (lane 6). The Spr18 <br /> The MAGE family is a large family of proteins, most of which         (Smc5) headless construct with about 250 aa of coiled coil <br /> are expressed in a wide variety of tumors but not in normal          bound strongly to the beads (Fig. 5D, lane 3, top panel) com- <br /> cells, with the exception of the male germ cells, the placenta,      pared to the control (lane 12). Strong binding remained when <br /> and possibly cells of the developing embryo. The cellular func-      we reduced the size of the coiled coil by a further 50 aa (Fig. <br /> tion of this family is unknown. Within the conserved domain          5D, lane 6) but was abolished when we deleted a further 100 aa <br /> database, both Nse3 and its S. cerevisiae homolog could be           from each arm (Fig. 5D, lane 9). We were not able to detect <br /> aligned with a MAGE family consensus sequence, KOG4562               binding between Nse2 and Nse1, Nse3, or Rad62 under the <br /> (Fig. 4F), with 17 to 20% sequence identity. The sequence            conditions used for our experiments (not shown). We conclude <br /> identity between Nse3 and both YDR288W and the MAGE                  that there is an interaction between Nse2 and Spr18 (Smc5) <br /> consensus is highest for the C-terminal half of the protein,         which is mediated by the N-terminal half of Nse2 and the <br /> which is also the region with the most sequence similarity           coiled-coil region of Spr18 (Smc5). The regions of Spr18 <br /> between different members of the MAGE family.                        (Smc5) between aa 226 and 324 of the N-terminal arm and aa <br />    Interactions between components. In order to dissect the          731 and 836 of the C-terminal arm are essential for this inter- <br /> interactions between the individual components of the com-           action. <br /> plex, we expressed them as GST-tagged proteins in E. coli.              The experiments depicted in Fig. 3B showed that the G529R <br /> GST-tagged Nse1, Nse2, and Nse3 were soluble. The proteins           mutation in the hinge region of Spr18 (Smc5) destroys the <br />  180     SERGEANT ET AL.                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 5. Interaction between Nse2 and Spr18 (Smc5). (A) A soluble protein extract prepared from E. coli strain BL21 expressing GST-Nse2 was <br /> preincubated with glutathione beads, and either IVT Rad18 (Smc6) (lanes 1 and 2), Spr18 (Smc5) (lanes 3 and 4), or a mixture of IVT Rad18 <br /> (Smc6) and Spr18 (Smc5) protein extracts (lanes 5 to 7) was added. The control experiment (lanes 8 to 10) was identical to that of lanes 5 to 7 <br /> except that GST was used instead of GST-Nse2. Input (I), unbound (U), and bound (B) fractions were collected and run in SDS&#226;&#8364;&#8220;10% PAGE gels. <br /> The in vitro-expressed proteins were visualized with a phosphorimager. Bacterially expressed proteins were analyzed by immunoblotting with an <br /> anti-Nse2 rabbit antiserum. (B) IVT Spr18 (Smc5) was mixed with an IVT N-terminal (aa 1 to 178; lanes 1 to 3) and/or C-terminal fragment (aa <br /> 114 to 250; lanes 4 to 6) of Nse2-S tag and incubated with protein S beads. In the control experiment, the Spr18 (Smc5) protein was incubated <br /> with protein S beads in the absence of Nse2-S tag (lanes 7 to 9). (C) The IVT Spr18 (Smc5) head construct (aa 2 to 174 plus aa 897 to 1065) was <br /> mixed with either the GST-Nse2 (full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span>) protein (lanes 1 to 3) or GST alone (lanes 4 to 6) and incubated with glutathione beads. (D) IVT <br /> Spr18 (Smc5) headless fragments containing aa 170 to 910 (lanes 1 to 3 and 10 to 12), aa 226 to 836 (lanes 4 to 6), or aa 324 to 731 (lanes 7 to <br /> 9) were mixed with either GST-Nse2 (full-length) prebound beads (lanes 1 to 9) or GST-only beads (lanes 10 to 12). (E) GST-Nse2 beads (lanes <br /> 1 to 6) or GST beads alone (lanes 7 to 10) were mixed with IVT Rad18 and Spr18 (lanes 1 to 3, 7, and 8) or Rad18-Spr18-gly529 (lanes 4 to 6). <br /> Note that in Fig. 5 to 7, the unbound and input fractions can be compared directly and represent 1/20 of the total reaction mix, whereas the bound <br /> fraction represents approximately 1/3 of the reaction. <br />  VOL. 25, 2005                                                                          ARCHITECTURE OF THE Smc5-6 COMPLEX                 181 <br />  <br />  <br />                                                                               confirm our pull-down data, we used the yeast two-hybrid <br />                                                                               system. All Nse genes were fused to either Gal4(BD) or <br />                                                                               Gal4(AD) and cotransformed into the Y190 budding yeast <br />                                                                               strain. Figure 6C shows a strong Nse1-Nse3 interaction-depen- <br />                                                                               dent transcriptional activation of the HIS3 gene (panel 2). <br />                                                                               Induction of the lacZ gene resulted in a clear blue color when <br />                                                                               assayed with X-Gal (Fig. 6C, panel 3). Nse1 has a RING finger <br />                                                                               motif that is typical of E3 ubiquitin ligases close to its C <br />                                                                               terminus (8, 20). In further experiments, we mixed IVT Myc- <br />                                                                               tagged Nse3 with either the full length (232 aa) or the N- <br />                                                                               terminal half (aa 1 to 116) of Nse1 lacking the RING finger <br />                                                                               motif, followed by immunoprecipitation with anti-Myc anti- <br />                                                                               bodies. Although a small amount of Nse1 bound to the beads <br />                                                                               in the absence of Myc-Nse3 (Fig. 6B, lanes 5 and 10), there was <br />                                                                               much stronger binding in its presence (Fig. 6B, lanes 3 and 8, <br />                                                                               top panels). In contrast, the C-terminal half of Nse1 did not <br />                                                                               interact with Nse3 (data not shown). Thus, as with Nse2 and <br />                                                                               Spr18 (Smc5), the interaction of Nse1 with Nse3 is mediated by <br />                                                                               the N-terminal half of the protein and does not require the <br />                                                                               RING finger motif. <br />                                                                                  Interaction between Rad62 and Nse3. In similar experiments <br />    FIG. 6. Interaction between Nse3 and Nse1 or Nse2. (A) Soluble <br /> protein extracts prepared from E. coli expressing either GST-Nse1             to those described above, we examined the interaction of <br /> (lanes 1 to 3) or GST-Nse3 (lanes 7 to 9) were preincubated with              Rad62 with all of the other components of the complex, which <br /> glutathione beads, and in vitro-expressed Nse3 (lanes 1 to 3) or Nse1         were synthesized in vitro either together or separately. The <br /> (lanes 7 to 9), respectively, was added (I). In the control experiments       only interaction we detected in this system was between S- <br /> (lanes 4 to 6 and 10 to 12), GST was used instead of GST-Nse1 and <br />                                                                               tagged Rad62 and Nse3 (Fig. 7A, lane 2, middle panel). This <br /> GST-Nse3. GST, GST-Nse1, and GST-Nse3 were detected by immu- <br /> noblotting with anti-GST. (B) IVT Myc-Nse3 (lanes 1 to 3 and 6 to 8)          interaction was rather weak compared to those described <br /> was mixed with either IVT full-length Nse1 (lanes 1 to 3) or the              above. Only a small proportion of the Nse3 protein bound to <br /> N-terminal part of Nse1 (aa 1 to 116; lanes 6 to 8). Myc-Nse3 was             the beads. We obtained further evidence for this interaction by <br /> immunoprecipitated with an anti-Myc monoclonal antibody and pro-              yeast two-hybrid analysis (Fig. 7B, panel 2). Rad62 did not bind <br /> tein G beads. In the control experiments, Myc-Nse3 was not added <br /> (lanes 4, 5, 9, and 10). The proteins were visualized with a phospho-         detectably to Nse1 either by pull-down assays (Fig. 7A, lane 4, <br /> rimager. (C) Yeast two-hybrid analysis of interactions between Nse1           bottom panel) or by yeast two-hybrid analysis (Fig. 7B, panel <br /> and Nse3 (panels 1 to 3) and Nse3 and Nse2 (panels 4 to 6). In each           4). However, when we mixed S-tagged Rad62 with both Nse3 <br /> set of three panels, the first panel is a positive control of cells growing   and Nse1, all three proteins bound strongly to the beads (Fig. <br /> in nonselective medium, the second panel shows growth in a selective <br />                                                                               7A, lane 7). This suggests that the interaction between Nse1 <br /> medium, and the third panel shows &#226;&#65533;&#164;-galactosidase activity. Results for <br /> two concentrations of cells differing by a factor of 10 are shown. In         and Nse3 strengthens the binding of Nse3 to Rad62. <br /> each case, controls with appropriate empty vectors are provided above            Interaction between Nse3 and Nse2. The analysis described <br /> and below the test pairs.                                                     so far delineated two subcomplexes, made up of Rad18-Spr18- <br />                                                                               Nse2 and Nse1-Nse3-Rad62. Using pull-down assays, we were <br />                                                                               not able to identify any interactions between the components <br /> interaction between Rad18 (Smc6) and Spr18 (Smc5). Figure                     of the subcomplexes. However, by using yeast two-hybrid anal- <br /> 5E shows that this mutation does not affect the interaction                   ysis, we detected a weak interaction between Nse3 and Nse2 <br /> between Spr18 (Smc5) and Nse2. As described above, GST-                       (Fig. 6C, panels 5 and 6). Based on this observation, we ten- <br /> Nse2 pulled down both Spr18 and Rad18 when the two Smc                        tatively propose that this may represent the link between the <br /> proteins were mixed together, as indicated by the doublet in                  two subcomplexes. <br /> Fig. 5E, lane 3. However when Rad18 (Smc6) and Spr18- <br /> G529R were incubated with GST-Nse2, Spr18-G529R was still                                             DISCUSSION <br /> pulled down, as indicated by the strong single band in lane 6, <br /> whereas the upper band corresponding to Rad18 (Smc6) was                        In previous work, it was shown that rad18 (smc6) is an <br /> no longer bound to the beads. These data are consistent with                  important DNA repair gene that is involved in several different <br /> binding of Spr18 (Smc5) and Rad18 (Smc6) via their hinges,                    repair pathways (7, 19, 23). Other genes, such as rad60 (22) and <br /> whereas binding of Spr18 (Smc5) to Nse2 is mediated via the                   brc1 (26), have phenotypes which are very similar to those of <br /> coiled coil region and does not involve the hinge.                            rad18 (smc6) as well as being synthetically lethal with rad18 <br />    Interaction between Nse1 and Nse3. In GST pull-down as-                    (smc6), suggesting the presence of a complex multiprotein <br /> says using GST-Nse1 and Rad18 (Smc6), Spr18 (Smc5), Nse2,                     network, whose mechanism of action remains to be elucidated. <br /> Rad62, or mixtures of these proteins that were synthesized by                 In this paper, we have begun to define the molecular architec- <br /> IVT, we were unable to detect any interactions (data not                      ture of the core Smc5-6 protein complex. <br /> shown). In contrast, we were able to detect an interaction                      Our results show that the Smc5-6 complex is similar to the <br /> between GST-Nse1 and Nse3 (Fig. 6A, lane 3) and, conversely,                  previously characterized cohesin and condensin complexes in <br /> between GST-Nse3 and Nse1 (Fig. 6A, lane 9). In order to                      that it has a SMC heterodimeric protein core associated with a <br />  182     SERGEANT ET AL.                                                                                                     MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 7. Interaction between Rad62 and Nse3. (A) A soluble protein extract prepared from E. coli strain Rosetta-gami B expressing S tag-Rad62 <br /> was preincubated with protein S beads and mixed with IVT Nse3 (lanes 1 and 2), Nse1 (lanes 3 and 4), or both Nse1 and Nse3 (lanes 5 to 7). In <br /> the control experiment, Nse3 and Nse1 were incubated with protein S beads in the absence of S tag-Rad62 (lanes 8 to 10). (B) Yeast two-hybrid <br /> analysis of interactions between Rad62 and Nse3 or Nse1. The details are the same as those for Fig. 6C. (C) Model for interactions between <br /> components. The two subcomplexes are indicated. The single arrows indicate components that are sumoylated by Nse2 in vitro (1a). The <br /> double-headed arrow indicates a tentative interaction between the subcomplexes, based on the data shown in Fig. 6C. <br />  <br />  <br />  <br /> series of non-SMC proteins. Although there is little sequence            at glycine 551 in the hinge region. Mutation of this conserved <br /> identity between the hinge regions of Smc6 and those of the              residue destabilized the interaction with the Smc5 hinge for <br /> other SMC proteins (Figs. 2E and F), as with the other com-              both the S. pombe and human proteins, as did mutation of the <br /> plexes, Smc5 and -6 interact through their hinge regions. This           conserved glycine 529 residue in Spr18 (Smc5). Temperature- <br /> suggests that SMC and SMC-like proteins may interact                     sensitive mutations in S. cerevisiae SMC6 have also been iso- <br /> through their hinges in three different ways, firstly by using the       lated (24), but none of these mutations was in the hinge region. <br /> structures determined for the Smc1-3 hinge (10), secondly via            The cause of the temperature sensitivity of these S. cerevisiae <br /> a presumably different structure in the Smc5-6 complex, and              mutants was not established. The <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>phenotype</span> of our tempera- <br /> thirdly via a zinc hook, as found for the SMC-like protein               ture-sensitive mutants, namely, continued growth and prolifer- <br /> Rad50 (16).                                                              ation for several hours despite a substantial decrease in the <br />    We generated four independent temperature-sensitive mu-               amount of Rad18 (Smc6) protein within 2 h at the restrictive <br /> tants of rad18 (smc6). Remarkably, they all carried a mutation           temperature, suggests that cells can survive and remain viable <br />  VOL. 25, 2005                                                                  ARCHITECTURE OF THE Smc5-6 COMPLEX                              183 <br />  <br />  <br /> for a significant period with low levels of Rad18 (Smc6) pro-       with Spr18 (Smc5) via the N-terminal half of Nse2 and a <br /> tein. This is consistent with previous observations of the phe-     central section of the coiled coil region of Spr18 (Smc5). (iii) <br /> notype of a dominant-negative mutation and the terminal phe-        Nse1 interacts with Nse3 via the N-terminal half of Nse1. (iv) <br /> notype of a rad18 (smc6) deletion mutant (7, 19) as well as with    Nse3 interacts with Rad62, and this interaction is strengthened <br /> recent findings by Harvey et al. (11). These authors used an S.     substantially upon binding of Nse1. Furthermore, our prelim- <br /> pombe strain, rad18-so, in which Rad18 (Smc6) could be con-         inary data suggest that Rad62 also binds directly to Nse1 when <br /> ditionally depleted. When thiamine was added to this strain to      Nse1 is bound to Nse3 (unpublished observations). Further <br /> deplete Rad18 (Smc6), the protein became undetectable after         interactions may not have been detected in our in vitro studies <br /> 12 to 18 h, but cells continued to grow at normal rates for 24 h.   for several reasons, such as an interference of interactions by <br />    We identified a total of six components in the Smc5-6 com-       the epitope tags or a requirement for posttranslational modi- <br /> plex, namely, Rad18 (Smc6), Spr18 (Smc5), Nse1, Nse2, Nse3,         fications. Inside the cell, the Nse1-Nse3-Rad62 subcomplex <br /> and Rad62. In a recent paper, Hazbun et al. analyzed a series       must bind to the Rad18 (Smc6)-Spr18 (Smc5)-Nse2 subcom- <br /> of anonymous protein complexes from S. cerevisiae by tagging        plex. Our yeast two-hybrid analysis suggests that the two sub- <br /> essential genes and analyzing copurifying proteins by mass          complexes may be bridged by an interaction between Nse2 and <br /> spectrometry and multidimensional protein identification tech-      Nse3. This enabled us to propose a model for the complex, as <br /> nology (12). One complex was associated with ORF YDR288W            indicated in Fig. 7C. Further work is required to understand <br /> and contained Rhc18, Smc5, Nse1, Mms21, and Qri2. This ap-          the nature of the interactions between the two subcomplexes in <br /> pears to be an analogous complex to the one that we have char-      more detail. Our work has so far not revealed any proteins that <br /> acterized from S. pombe, since these proteins are orthologs of      bridge the head domains of Rad18 (Smc6) and Spr18 (Smc5), <br /> Rad18 (Smc6), Spr18 (Smc5), Nse1, Nse2, and Rad62, respec-          as is the case for S. cerevisiae cohesin, in which Scc1p bridges <br /> tively. As described above, Nse3 is related to and is probably an   the head domains of Smc1p and Smc3p (9). It may well be that <br /> ortholog of YDR288W. These authors also suggested that Smc5         the Smc5-6 complex is not, like cohesin, capable of forming a <br /> and -6 are associated with a second complex containing two          ring structure. Indeed, a model for the function of the complex <br /> different proteins, YML023C (64 kDa) and Kre29 (54 kDa).            that was proposed earlier, in which broken DNA ends are held <br /> We have not detected this putative second complex in our            together to enable the repair of double strand breaks to take <br /> experiments with S. pombe. No proteins corresponding to these       place (7), would not be readily compatible with a ring struc- <br /> molecular masses were evident in our affinity-purified Rad18        ture. Different SMC complexes may have different overall <br /> (Smc6) complex. However, in studies using gel filtration de-        structures, as discussed for the SMC-like Rad50 proteins (6, <br /> scribed in the accompanying paper (1a), we obtained evidence        16). These may depend on the nature of the hinge interaction, <br /> for a possible lower-molecular-weight complex containing            the flexibility of the coiled coils, the non-SMC proteins, and <br /> Rad18 (Smc6) and Spr18 (Smc5) but lacking Nse1 and Nse2.            posttranslational modifications. <br /> The nature of this complex remains to be determined.                   The sequences of Nse1 and Nse2 suggest that they are E3 <br />    Nse3 shows sequence similarity to the MAGE family, many          ubiquitin and SUMO ligases, respectively. In a companion <br /> members of which are expressed in cancer tissues but not in         paper (1a), we show that Nse2 does indeed have SUMO ligase <br /> normal cells and thus provide potential targets for tumor im-       activity, which is capable of sumoylating Rad18 (Smc6), Nse3, <br /> munotherapy. There are 55 human MAGE genes or pseudo-               and Rad62 in vitro (as indicated in Fig. 7C), and that within a <br /> genes, and they have a conserved MAGE homology domain               Smc5-6 complex that was purified from cells, Rad18 (Smc6) <br /> (4). No MAGE genes have been identified previously in yeasts.       was sumoylated. Furthermore, this sumoylating activity is in- <br /> However, our analysis using iterative BLAST searches clearly        volved in the response to DNA damage and the inhibition of <br /> demonstrated that both Nse3 and the S. cerevisiae ORF               DNA replication. Our future work will be directed toward <br /> YDR288W are related to the MAGE family. There are also              uncovering further interactions within the complex and the <br /> single related genes in Aspergillus nidulans, Neurospora crassa,    effect of posttranslational modifications on these interactions <br /> Drosophila melanogaster, and Arabidopsis thaliana. Despite          and on the functions of the complex. <br /> many publications about the MAGE family, little is known <br /> about their function. The implications of the relationship of                                 <br /> Nse3 to the MAGE family are not clear at present. Many of the         <br /> conserved residues in the MAGE family are not found in Nse3,       <br /> so it is distantly related to the family. There is no obvious       <br /> connection between what little is known at present of the       <br /> functions of the MAGE family proteins and the role of Nse3 in <br />                                                                     <br /> the Smc5-6 complex. Our results raise the possibility that         <br /> MAGE proteins are involved in DNA damage responses                     <br /> and/or the maintenance of chromatin structure.                       <br />    We have initiated a study to understand the molecular ar-      <br /> chitecture of the Smc5-6 complex. Our in vitro interaction                               <br /> studies have highlighted the following interactions between          <br /> different components of the complex. (i) Rad18 (Smc6) and             <br /> Spr18 (Smc5) interact via their hinge regions, and this inter-          <br /> action is dependent on conserved glycine residues in both               <br /> Rad18 (Smc6) and Spr18 (Smc5). (ii) Nse2 interacts strongly            <br />  BIOL. <br />  <br /> </body></html>