<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>ANTIMICROBIAL AGENTS AND CHEMOTHERAPY, Sept. 2004, p. 3425&#226;&#8364;&#8220;3435                                                                  Vol. 48, No. 9 <br /> 0066-4804/04/$08.00&#226;&#171;&#185;0 DOI: 10.1128/AAC.48.9.3425&#226;&#8364;&#8220;3435.2004 <br /> Copyright &#194;&#169; 2004, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />   Disruption Candida albicans CYB5 Gene Results Increased <br />                           Azole Sensitivity&#226;&#8364;&#160; <br />             K. M. Rogers,1 C. A. Pierson,1 N. T. Culbertson,1 C. Mo,1 A. M. Sturm,1 J. Eckstein,2 <br />                                  R. Barbuch,2 N. D. Lees,1 M. Bard1* <br />                  Department Biology, Indiana University Purdue University Indianapolis,1 Department Drug <br />                             Disposition, Eli Lilly &amp; Co., Lilly Corporate Center,2 Indianapolis, Indiana <br />                        Received 31 December 2003/Returned modification 29 January 2004/Accepted 18 May 2004 <br />  <br />              Sterol synthesis fungi aerobic process requiring molecular oxygen and, cytochrome- <br />           mediated reactions, aerobically synthesized heme. Cytochrome b5 required sterol C5-6 desaturation and <br />           encoding gene, CYB5, nonessential Saccharomyces cerevisiae. Cyb5p Ncp1p (cytochrome P-450 <br />           reductase) appear overlapping functions organism, disruptions viable. <br />           The cytochrome P-450 reductase <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span>phenotype</span> shown demonstrate increased sensitivity azole <br />           antifungals. Based phenotype, CYB5 gene human pathogen Candida albicans investigated <br />           determine cyb5 genotype viable demonstrate azole sensitivity. Sequential <br />           disruption CYB5 alleles direct transformation resulted viability, presumably conferred the <br />           presence copy CYB5 gene. Subsequent disruption procedures pMAL2-CYB5 rescue <br />           cassette CYB5-URA3 blaster cassette resulted viable cyb5 strains copy. The C. albicans <br />           CYB5 gene concluded nonessential. Thus, essentiality gene observed or <br />           alleles dependent gene disruption protocol. The C. albicans cyb5 strains produced sterol <br />           profile containing low ergosterol levels sterol intermediates similar reported S. cerevisiae <br />           cyb5. The C. albicans cyb5 shows increased sensitivity azoles terbinafine, inhibitor squalene <br />           epoxidase, and, unexpectedly, increased resistance morpholines, inhibit ERG2 ERG24 gene <br />           products. These results indicate inhibitor Cyb5p lethal make cell <br />           significantly sensitive azole treatment. <br />  <br />  <br />   The fungal end product sterol, ergosterol, biosyn-                   coded CYP51, referred ERG11 ERG16), the <br /> thetic pathway targets majority antifungal               cytochrome P-450-dependent desaturation step C-22 (en- <br /> compounds currently use human infections agricul-                   coded CYP61, referred ERG5), cyto- <br /> tural applications. The primary class compounds used                     chrome b5 requiring C-5 desaturation (encoded ERG3). In <br /> human infections azoles, drugs inhibit C-14                    case cytochrome P-450 steps, distinct enzymes <br /> demethylation pathway intermediate lanosterol. Over-                    involved (11, 12), requires electron <br /> use azoles, especially immunocompromised patients,                   donor, NADPH cytochrome P-450 oxidoreductase (en- <br /> has led increases incidence antifungal resistance                 coded NCP1). Ncp1p donates electrons pre- <br /> (5). As resistance levels compounds rise, need                 sterol, oxygen-requiring reaction catalyzed squalene epoxi- <br /> identify new targets antifungal therapy expands. Several                   dase (32). The cytochrome b5 step pathway utilizes <br /> reactions ergosterol biosynthetic pathway patho-                 NADH-cytochrome b5 reductase NADPH-cytochrome c re- <br /> genic fungus Candida albicans identified (1, 18, 26)                 ductase electron carriers creation double bond <br /> as required viability and, thus, subject explo-                C5-6 (25). <br /> ration new antifungal sites. Inhibition steps                   Sutter Loper (27) reported disruption Sac- <br /> prove effective antifungal interventions or-                charomyces cerevisiae NCP1 gene lethal despite fact <br /> ganism import exogenous sterol (19, 29), making sterol                  steps pathway utilize electron donor and <br /> biosynthesis obligatory.                                                       steps, C-14 demethylation lanosterol, has <br />   Sterol biosynthesis fungi exclusively aerobic process               proven essential based fact disruption the <br /> because steps ergosterol biosynthetic pathway                   cytochrome P-450 demethylase gene (CYP51) results non- <br /> require molecular oxygen heme, synthesized                   viability (17). The presence alternative electron carrier, <br /> only aerobic conditions used primarily cyto-                      cytochrome b5, postulated explain viability <br /> chrome-mediated reactions. Among the cyto-                      ncp1 mutant (13). This substitution function cyto- <br /> chrome P-450-dependent step lanosterol demethylation (en-                   chrome b5 confirmed follow-up study (20). Similarly, <br />                                                                                gene encoding cytochrome b5 S. cerevisiae <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br />                                                                                nonessential (28). Disruption CYB5 generates growth <br />   * Corresponding author. Mailing address: Department Biology,              phenotype wild-type background results lethality <br /> Indiana University Purdue University Indianapolis, West Michigan St.,          present ncp1 background (28). This suggests that <br /> Indianapolis, IN 46202-5132. Phone: (723) 274-0588. Fax: (317) 274- <br /> 2846. E-mail: nlees@iupui.edu. <br />                                                                                NCP1 provide reciprocal function missing cy- <br />   &#226;&#8364;&#160; M.B. N.D.L. dedicate manuscript memory            tochrome b5 protein. <br /> mothers, Tillie Bard Adeline Lees.                                            Ncp1p Cyb5p appear overlapping functions S. <br />  <br />                                                                         3425 <br />  3426       ROGERS ET AL.                                                                                                   ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br />                                        TABLE 1. C. albicans strains derived BWP17 generated study <br />  Strain                              Disruption protocol                                                               Genotype <br />  <br /> KRC2                              Direct transformation                            CYB5/cyb5::ARG4 <br /> KRC3                              Direct transformation                            cyb5::URA3/cyb5::ARG4/CYB5 <br /> KRC6                              pMAL-CYB5 cassette                               CYB5/cyb5::ARG4 ADE2/ade2::pMAL-CYB5 <br /> KRC7                              pMAL-CYB5 cassette                               cyb5::URA3/cyb5::ARG4 ADE2/ade2::pMAL-CYB5 <br /> KRC9                              URA3-CYB5 blaster                                CYB5/cyb5::HIS1 <br /> KRC10                             URA3-CYB5 blaster                                CYB5/cyb5::HIS1 ERG25/erg25::hisG-URA3-CYB5-hisG <br /> KRC11                             URA3-CYB5 blaster                                cyb5::ARG4/cyb5::HIS1 ERG25/erg25::hisG-URA3-CYB5-hisG <br /> KRC12                             URA3-CYB5 blaster                                cyb5::ARG4/cyb5::HIS1 ERG25/erg25::hisG <br /> KRC13                             URA3-CYB5 blaster                                cyb5::ARG4/cyb5::HIS1 ERG25/erg25::hisG ADE2/ade2::pMAL-CYB5 <br />  <br />  <br />  <br />  <br /> cerevisiae. One phenotypes ncp1 increased sen-                        Transformations. E. coli DH5&#226;?&#163; transformations accomplished stan- <br /> sitivity azole antifungals (27). This postulated to                dard protocols (2). C. albicans transformations carried previously <br />                                                                                    described (31). <br /> the increased sensitivity substitute electron carrier                   DNA sequencing. DNA sequencing performed Biochemistry Bio- <br /> C-14 demethylation reaction suggests mechanism ac-                        technology Facility Indiana University School Medicine. Initial se- <br /> tion azoles interacting Cyp51p. A recent                      quence data obtained T3 T7 primers used construct new <br /> study (28) confirmed azole sensitivity ncp1 phe-                    primers. Primers obtained Sigma-Genosys, Ltd. (The Woodlands, <br /> notype demonstrated ncp1 cells produce                          Tex.). <br />                                                                                       PCR. The standard PCR composed template DNA (5 10 ng of <br /> about 25% ergosterol produced NCP1 strains. In                           plasmid DNA 10 50 ng genomic DNA), sequence-specific oligonucleo- <br /> addition, elevated levels lanosterol detected, indi-                   tide primers (10 60 pmol primer), 0.2 mM deoxynucleoside triphos- <br /> cating cells able complete C-14 demeth-                       phates (Sigma, St. Louis, Mo.), appropriate buffer, sterile water, thermo- <br /> ylation reaction reactions Ncp1p                        stable DNA polymerase. <br />                                                                                       The kinds thermostable DNA polymerases used study Taq <br /> normally functions. Based vitro assay (13) on <br />                                                                                    (Promega) Expand High Fidelity Taq polymerase (EHF Taq) (Roche, In- <br /> suppression ncp1 phenotype CYB5 (28), Cyb5p                        dianapolis, Ind.). Taq polymerase largely used screening genetic con- <br /> be replacement electron donor.                                                 structions genetic insertions PCRs high DNA fidelity not <br />    Based increased azole sensitivity ncp1 strains cou-                   critical. The reaction buffer contained 10&#226;&#171;&#187; buffer MgCl2 (50 mM KCl, 10 <br /> pled fact required reactions sterol biosynthesis                  mM Tris-HCl [pH 9.0], 1% Triton X-100), 1.5 2.0 mM MgCl2 1.25 U of <br />                                                                                    enzyme. The polymerase used study EHF Taq (Roche). This <br /> in S. cerevisiae C. albicans differ (16), worthwhile <br />                                                                                    polymerase used PCRs DNA accuracy crucial. The <br /> to investigate characteristics cyb5 ncp1                        reaction buffer contained 10&#226;&#171;&#187; buffer 15 mM MgCl2 (20 mM Tris-HCl [pH <br /> phenotypes pathogenic fungus. This report investigated                     7.5], 100 mM KCl, 1 mM dithiothreitol, 0.1 mM EDTA, 0.5% [vol/vol] Tween 20, <br /> whether CYB5 essential gene Cyb5p                         0.5% [vol/vol] Nonidet P40, 50% [vol/vol] glycerol), 2.6 U polymerase was <br /> a potential drug target. This report employed various                         used reaction. <br />                                                                                       PCRs carried Perkin Elmer GeneAmp2400 (Perkin Elmer, Inc., <br /> methods gene disruption difficulties isolating ho-                    Foster City, Calif.) PTC-0200 DNA Engine Thermal Cycler Chassis (M.J. <br /> mozygous mutants determination gene essentiality.                       Research, Waltham, Mass.) standard methods described Jia et al. (16). <br />                                                                                    The PCR primers used study listed Table 2. <br />                                                                                       Sterol analysis. Sterols isolated previously described (22) analyzed <br />                        MATERIALS AND METHODS                                       gas chromatography. An HP5890 Series II equipped Hewlett-Packard <br />    Strains plasmids. Escherichia coli strain DH5-&#226;?&#163; [&#226;?&#190;80dlacZ&#226;&#338;&#172;M15 recA1         Chemstation software package used analyze sterols. The capillary <br /> endA1 gyrA96 thi-1 hsdR17 supE44 relA1 deoR &#226;&#338;&#172;(lacZYA-argF)U169] used           column (DB-5) 15 m 0.25 mm 0.25 &#226;?&#174;m (J&amp;W Scientific, Folsom, <br /> for routine bacterial transformations. C. albicans strain BWP17 (ura3&#226;&#338;&#172;::           Calif.). The gas chromatograph programmed 195&#194;&#176;C (1 min 195&#194;&#176;C, <br /> &#226;?&#173;imm-434/ura3&#226;&#338;&#172;::&#226;?&#173;imm-434 his1::hisG/his1::hisG arg4::hisG/arg4::hisG) ob-      increase 20&#194;&#176;C/min 240&#194;&#176;C 240 280&#194;&#176;C 2&#194;&#176;C/min). The <br /> tained A. Mitchell (31). C. albicans strain CAI-4 (ura3&#226;&#338;&#172;::&#226;?&#173;imm-434/           linear velocity 30 cm/s, nitrogen carrier gas. All injections were <br /> ura3&#226;&#338;&#172;::&#226;?&#173;imm-434) obtained W. Fonzi (9). S. cerevisiae strain Wb5&#226;&#338;&#172;     run splitless mode. <br /> obtained G. Truan (28). The C. albicans strains generated study      Gas chromatography/mass spectrometry analyses sterols a <br /> listed Table 1. Plasmids pGEM-URA3, pGEM-HIS1, pRS-ARG4                Thermoquest Trace 2000 gas chromatograph interfaced Thermoquest Voy- <br /> obtained A. Mitchell (31). Plasmid pDBI52 obtained C. Kum-           ager mass spectrometer. The gas chromatography separations a <br /> amoto (6).                                                                         fused silica column, DB-5MS, 20 m 0.18 mm 0.18 &#226;?&#174;m film thickness (J&amp;W <br />    Media growth. E. coli strains grown 37&#194;&#176;C Luria broth (LB)        Scientific, Folsom, Calif.). The injector temperature 190&#194;&#176;C. The oven tem- <br /> (Bio101) containing 10 g tryptone, 5 g yeast extract, 10 g NaCl   perature programmed remain 100&#194;&#176;C 1 min, followed temper- <br /> liter. For solid media, 20 g agar (Difco) liter added. Ampicillin       ature ramp 6.0&#194;&#176;C/min final temperature 300&#194;&#176;C. The final temperature <br /> (Sigma) added autoclaving concentration 60 &#226;?&#174;g/ml plasmid     held 25 min. Helium carrier gas linear velocity 50 cm/s <br /> selection.                                                                         splitless mode. The mass spectrometer electron impact ioniza- <br />    S. cerevisiae C. albicans strains grown complete synthetic medium   tion mode electron energy 70 eV, ion source temperature 150&#194;&#176;C, <br /> (CSM) YPD medium. CSM medium contained 0.17% yeast nitrogen <span id='am-237' about='xsp:base' typeof='owl:Thing'>base</span>                scanning 40 850 atomic mass units 0.6-s intervals. <br /> without amino acids (Difco), 0.5% ammonium sulfate, 2% D-glucose,               Southern analysis. The probes used amplified C. albicans wild- <br /> mixture amino acids (Bio 101). CSM lacking specific nutrients used       type strain (BWP17) PCR. Probe 1 used cytb5 fragments disrupted by <br /> selection. YPD comprised 1% Bacto yeast extract, 2% Bacto peptone,      selectable markers. The primers used, b5-5probeI b5-3probeI (Table 2), were <br /> 2% D-glucose. Adenine added 360 mg/liter C. albicans cultures         located 1,539 bp 862 bp, respectively, upstream start codon and <br /> ensure optimal growth (YPAD). Uridine added 80 mg/liter media used       amplified 677-bp DNA fragment. Probe 2 designed hybridize the <br /> to cultivate C. albicans ura3 strains. Maltose used 2% replace glucose   CYTB5 open reading frame. The primers used, b5-5ProbeX b5-3ProbeX <br /> in media (YPAM). S. cerevisiae cultures grown 30&#194;&#176;C, C.          (Table 2), located 46 bp downstream start codon 129 bp up- <br /> albicans cultures grown 30&#194;&#176;C 37&#194;&#176;C.                                     stream stop codon, respectively, amplified 200-bp DNA fragment. <br />  VOL. 48, 2004                                                                                          STEROL SYNTHESIS IN C. ALBICANS                       3427 <br />  <br />  <br />                                                          TABLE 2. Primers used study <br />        Primer                                                   Sequence (5&#226;&#172;&#732;&#226;&#8364;&#8220;3&#226;&#172;&#732;)a                                                             Application <br />  <br /> b5-5 drI                 CCATCCTTTCGTTACATACATTTATACAATCATGTCAGAAGAAACCGTTACTATT                                                      Disruption CYB5 with <br />                           TACGAGTTTTCCCAGTCACGACGTT                                                                                     ARG4 <br /> b5-3 drI                 GCAAAGTTAGTTTTGTAATAATAAGCACCAAATGCAAGTAAAAAGACACCAAC                                                        Disruption CYB5 with <br />                           AGCAATCTGTGGAATTGTGAGCGGATA                                                                                   ARG4 <br /> b5-5 chI                 AATTCAGCCCCTTTCAACCC                                                                                         Check ARG4 URA3 <br />                                                                                                                                         CYB5 disruptions <br /> b5-3 chI                 GTGGTAAATTGGGATTCTCC                                                                                         Check ARG4 URA3 <br />                                                                                                                                         CYB5 disruptions <br /> b5-5 drIII               CCATCCTTTCGTTACATACATTTATACAATCATGTCAGAAGAAACCGTTACTATT                                                      Disruption second <br />                           TACGATTATGAAGAAGTTTCCAAACGTTTTCCCAGTCACGACGTT                                                                 CYB5 allele with <br />                                                                                                                                         URA3 ARG4 <br /> b5-3 drIII               CCAACAGCAATCAATGGGAAATTAATACCACTATCTTCAGTAGCATAAGATTG                                                        Disruption second <br />                           AGCATGTTTAGCTTCAACAGGTTTAGCGTGGAATTGTGAGCGGATA                                                                CYB5 allele with <br />                                                                                                                                         URA3 ARG4 <br /> b5-5EcoRV                GGGATATCATGTCAGAAGAAACCGTTAC                                                                                 Amplify CYB5 in <br />                                                                                                                                         BWP17 <br /> b5-3XbaI                 GGTCTAGAAATTATCTAAGTTCTATTACCCTCTC                                                                           Amplify CYB5 in <br />                                                                                                                                         BWP17 <br /> HIS1-5                   GGGGATCCGACTCGACAGGTACCTGG                                                                                   Amplify HIS1 <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br />                                                                                                                                         CAI4 <br /> HIS1-3                   GGTCTAGAAAACCGTACCAGGT                                                                                       Amplify HIS1 from <br />                                                                                                                                         CAI4 <br /> ADE-3ck2                 AGTATTCACGGATAGATCTGTTAGAG                                                                                   ADE2 integration <br /> pMAL-5                   GCAGTTGAGAATGTTAGTTTTTG                                                                                      ADE2 integration <br /> b5-5 drII                ATACATATATACTACAAAAAGAAAATAGAAGCACAATAATCGAAGGGAGAGA                                                         Disruption CYB5 with <br />                           GGGAGAGGGTTTTCCCAGTCACGACGTT                                                                                  HIS1 <br /> b5-3 drII                GTAAGAATATCAAATTGCTTGCAGATGGAAATATTCTCTACTACTCTCTTTCAA                                                       Disruption CYB5 with <br />                           ATTAATTGTGGAATTGTGAGCGGATA                                                                                    HIS1 <br /> b5-5 chII                GTACTGTTGTTGGAGTCTCG                                                                                         Check CYB5 disrup- <br />                                                                                                                                         tions ARG4 and <br />                                                                                                                                         HIS1 <br /> b5-3 chII                GATAATGTCAACGTGAGTCG                                                                                         Check CYB5 disrup- <br />                                                                                                                                         tions ARG4 and <br />                                                                                                                                         HIS1 <br /> b5-5 promoter            GGTCTAGATCTTAAACCTTTAGTGTGGG                                                                                 Amplify CYB5 in <br />                                                                                                                                         BWP17 <br /> b5-3XbaI                 GGTCTAGAAATTATCTAAGTTCTATTACCCTCTC                                                                           Amplify CYB5 in <br />                                                                                                                                         BWP17 <br /> hisG-3                   GATAATACCGAGATCGAC                                                                                           ERG25 integration <br /> E25-b5-3chI              CGTTCAGCCATAGCCCAACC                                                                                         ERG25 integration <br /> E25-5chII                AGCACAACAGGATGACGCCC                                                                                         ERG25 integration <br /> b5-5probeI               CTTCTGAAGCTACTACCACC                                                                                         Southern analysis for <br />                                                                                                                                         CYB5 disruptions <br /> b5-3probeI               CCACTATGCTTTATCATGGG                                                                                         Southern analysis for <br />                                                                                                                                         CYB5 disruptions <br /> b5-5ProbeX               ATAGAACTCATGATGATTTATGGGTCG                                                                                  Southern analysis for <br />                                                                                                                                         CYB5 <br /> b5-3ProbeX               ACCTTTCAAATTACCAATATATAATTTTTG                                                                               Southern analysis for <br />                                                                                                                                         CYB5 <br /> ARG4-5                   CAACCTATTCGCTGGGCCC                                                                                          Detection ARG4 <br />                                                                                                                                         disruption Southern <br />                                                                                                                                         analysis <br /> ARG4-5R                  AGCAGTACCACCAAGAGC                                                                                           Detection ARG4 <br />                                                                                                                                         disruption Southern <br />                                                                                                                                         analysis <br /> NJ-ura3-5                TCATGCCTCACCAGTAGC                                                                                           Detection URA3 <br />                                                                                                                                         disruption Southern <br />                                                                                                                                         analysis 3 <br /> KR-ura3-5                GGTACCTGATTTCTAGAAGGACC                                                                                      Detection URA3 <br />                                                                                                                                         disruption Southern <br />                                                                                                                                         analysis <br />   a <br />       Plasmid sequences bold. <br />  <br />  <br />  <br />  <br /> The probes designed sequences present cyb5 remnants         membrane (Schleicher Schuell, Keene, N.H.) according manufactur- <br /> disrupted alleles. Probes labeled Rediprime II random prime           er&#226;&#8364;&#8482;s protocol. <br /> labeling (Amersham, Piscataway, N.J.) according manufacturer&#226;&#8364;&#8482;s          Growth drug susceptibility testing. Cells grown overnight 30&#194;&#176;C in <br /> protocol. Blotting accomplished Nytran Supercharge nylon transfer        50-ml cultures. Cell concentration determined hemacytometer di- <br />  3428       ROGERS ET AL.                                                                                          ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br />  <br />  <br />   FIG. 1. Disruption methods used study. (A) Direct transformation disruption method. (B) Disruption method inducible rescue <br /> cassette. (C) Disruption method URA3 blaster system. <br />  <br />  <br />  <br />  <br /> luted yield concentrations 2 &#226;&#171;&#187; 107, 2 &#226;&#171;&#187; 106, 2 &#226;&#171;&#187; 105, 2 &#226;&#171;&#187; 104 cells/ml;   disruption carried assuming gene also <br /> 5 &#226;?&#174;l dilution spotted medium tested. Cerulenin,            nonessential. <br /> clotrimazole, cycloheximide, nystatin obtained Sigma. Flucon- <br /> azole ketoconazole obtained ICN Pharmaceuticals Inc. (Cosa               Method I, disruption CYB5 direct transformation; <br /> Mesa, Calif.), tridemorph fenpropimoph obtained Sigma             disruption allele. Method I diagrammatically <br /> Aldrich. Terbinafine gift I. Hapala (Slovak Academy Science),     represented Fig. 1A. PCR primers b5-5drI b5-3drI (Ta- <br /> itraconazole gift Janssen Pharmaceuticals. Stock solutions ceru-      ble 2) containing 60 bp CYB5 gene 20 bp plasmid <br /> lenin, terbinafine, tridemorph 95% ethanol, stock solu- <br /> tions cycloheximide fluconazole distilled H2O. Dimethyl         sequence flanking selectable marker used amplify the <br /> sulfoxide used solvent clotrimazole, itraconazole, ketocon-      ARG4 gene plasmid pRS-ARG4. The resulting 2.1-kb DNA <br /> azole. All stocks stored &#226;&#171;&#186;20&#194;&#176;C.                                             fragment used transform C. albicans strain BWP17, and <br />                                                                                     transformants selected CSM-Arg&#226;&#171;&#185;Ade medium. PCR <br />                                                                                     used verify disruption correct. Checking <br />                                  RESULTS <br />                                                                                     primers b5-5chI b5-3chI (Table 2) designed amplify <br />    The C. albicans CYTB5 gene identified C.                            0.5-kb fragment CYB5 allele 2.4-kb fragment in <br /> albicans genomic DNA sequence (19X contig) obtained                            ARG4-disrupted allele. Figure 2, lanes A B, the <br /> the Stanford University DNA Sequencing Project. The CYB5                            predicted bands wild type heterozygote KRC2, <br /> gene 381-bp open reading frame encoding 126-amino-                           respectively. Sterol analysis (not shown) KRC2 showed a <br /> acid protein. C. albicans Cyb5p shows 43% identity                           wild-type profile. <br /> amino acid level S. cerevisiae Cyb5p.                                             Disruption second allele. PCR primers b5-5drIII and <br />    In order determine Cyb5p essential growth                      b5-3drIII (Table 2) containing 60 bp CYB5 sequence not <br /> in C. albicans, copies CYB5 gene                       represented ARG4-disrupted CYB5 allele 20 bp of <br /> sequentially disrupted. Since organism does import                         plasmid sequence used amplify URA3 gene on <br /> exogenous sterol (19, 29), second disruption                      pGEM-URA3. The PCR products use transform <br /> take place presence copy gene Cyb5p                     KRC2, transformants selected CSM-Ura- <br /> is essential. Since CYB5 gene S. cerevisiae shown                      Arg&#226;&#171;&#185;Ade. Confirmation disruption accomplished with <br /> to nonessential (28), attempt C. albicans CYB5                      primers b5-5chI b5-3chI (Table 2), amplify a <br />  VOL. 48, 2004                                                                             STEROL SYNTHESIS IN C. ALBICANS                 3429 <br />  <br />  <br />  <br />  <br />                                                                              FIG. 3. PCR confirmation disruption C. albicans CYB5 <br />                                                                           gene inducible rescue cassette. Lane A shows 3.4-kb frag- <br />                                                                           ment representing integration pKR02 containing pMAL- <br />                                                                           CYB5 cassette ADE2 locus KCR6. Lane B shows 2.4-kb <br />                                                                           fragment ARG4-disrupted cyb5 allele 1.7-kb fragment of <br />   FIG. 2. PCR confirmation disruption C. albicans CYB5          HIS1-disrupted cyb5 allele KCR7. Lane C confirms presence <br /> gene sequential transformation. Lane A shows 0.5-kb fragment       (3.4-kb fragment) pMAL-CYB5 cassette KCR7. <br /> of CYB5 wild-type allele; lane B shows 0.5-kb CYB5 allele and <br /> the 2.4-kb fragment ARG4-disruptped cyb5 allele hetero- <br /> zygote KCR2. Lane C shows 1.7-kb fragment HIS1-disrupted       strain CAI-4. HIS1 ligated replace URA3 generate <br /> cyb5 allele, 2.4-kb fragment ARG4-disrupted cyb5 allele, and <br /> the 0.5-kb fragment copy CYB5 wild-type allele KCR3.      pKR02. This plasmid linearized ade2 sequence <br />                                                                           BamHI used transform CYB5 heterozygote <br />                                                                           KRC2. Transformants selected CSM-Arg-His&#226;&#171;&#185;Ade. <br />                                                                           Randomly selected colonies checked PCR the <br /> 1.7-kb fragment URA3-disrupted allele. Figure 2, lane C,           proper integration ADE2 locus. Primers pMAL-5 and <br /> shows results confirmation isolate (KRC3)             ADE-3ck2 designed amplify 3.4-kb fragment repre- <br /> of cyb5 homozygote. The 1.7-kb fragment representing              senting proper integration. Figure 3, lane A, shows the <br /> URA3-disrupted allele 2.4-kb fragment representing                proper integration isolate, KRC6. <br /> the ARG4-disrupted allele present. However, 0.5-kb frag-               Disruption second CYB5 allele. The remaining wild- <br /> ment reflecting wild-type allele present. Sequencing           type CYB5 allele KRC6 disrupted transformation <br /> of 0.5-kb fragment isolated gel indicated 100%               1.7-kb URA3 fragment. Selection CSM- <br /> identity CYB5 sequence, indicating                Arg-His-Ura maltose carbon energy source. <br /> authentic copy wild-type allele. The results             Several colonies analyzed PCR determine na- <br /> obtained isolates KRC3. Sterol analysis (not                ture disruptions integration. Primers b5-5chI and <br /> shown) strain KRC3 indicated wild-type sterol profile,               b5-3chI (Table 2) used check ARG4 URA3 <br /> indicating allele functional. Despite success-         disruptions CYB5 alleles. Figure 3, lane B, shows the <br /> ful disruption CYB5 alleles, presence            2.4-kb fragment representing ARG4-disrupted allele and <br /> wild-type allele imply gene essential. To con-           1.7-kb fragment representing URA3-disrupted allele in <br /> firm essentiality gene, employed alternative            isolate (KRC7). With primers pMAL-5 ADE-3ck2 <br /> method disruption.                                                     (Table 2), plasmid integration ADE2 locus shown <br />    Method II, disruption CYB5 inducible rescue                 correct virtue presence 3.4-kb fragment <br /> cassette; preparation rescue cassette. Method II                (Fig. 3, lane C). Identical results obtained four <br /> shown Fig. 1B. The vector pKR01, containing copy              additional isolates KRC7. <br /> CYB5 gene control pMAL2 promoter,                       The cyb5 homozygote (KRC7) spot plated maltose <br /> constructed inserting CYB5 open reading frame                   glucose media determine CYB5 gene was <br /> pDB152 (6). Primers b5-5EcoRV b5-3XbaI (Table 2)                 essential. The results shown Fig. 4 indicate growth was <br /> used amplify CYB5 open reading frame strain                     equivalent media. If CYB5 gene essential, normal <br /> BWP17, creating 5&#226;&#172;&#732; EcoRV site 3&#226;&#172;&#732; XbaI site                 growth maltose expected, growth glucose <br /> 0.6-kb fragment. The plasmid confirmed cor-               minimal lack induction wild-type <br /> rect insertion generated 6.4-kb fragment linear-               CYB5 allele control pMAL2. While leakage <br /> ized.                                                                     pMAL2 promoter expected (1, 3), the <br />    Since disruption place used ARG4 URA3                product formed allow normal growth if <br /> as selectable markers CYB5 disruptions, URA3                  gene essential. Since growth normal glucose, the <br /> marker pKR01 replaced HIS1. Primers HIS1-5                CYB5 gene concluded nonessential. <br /> HIS1-3 (Table 2) designed 5&#226;&#172;&#732; BamHI site 3&#226;&#172;&#732;                 Sterol accumulation strain KRC7 grown maltose and <br /> XbaI site amplify 1.6-kb HIS1 fragment C. albicans              glucose determined. Figure 5 shows sterol profiles <br />  3430     ROGERS ET AL.                                                                                  ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br />  <br />  <br />          FIG. 4. Growth C. albicans cyb5 pMAL-CYB5 cassette (KRC7) YPA CSM media glucose maltose. <br />  <br />  <br />  <br /> along BWP17 Wb5&#226;&#338;&#172; (an S. cerevisiae cyb5                CYB5 essential rescue cassette method indicated <br /> mutant). The sterol profile maltose-grown KRC7 resembles              gene essential, method (shown Fig. <br /> that wild-type strain, ergosterol constituting          1C) employed final confirmation. In method, the <br /> 85% total sterol. When grown glucose, however, KRC7                CYB5 gene integrated adjacent URA3 gene and <br /> produced profile similar Wb5&#226;&#338;&#172;, decreased               flanked Salmonella hisG repeats (URA3 blaster cassette) <br /> ergosterol accumulation pathway intermediates er-             (9). This copy serve rescue copy plating <br /> gosta-7,22-dien-ol, episterol, fecosterol. The accumulation          5-fluoroorotic acid, selects URA3 cells <br /> of substantial amounts ergosta-7,22-dien-ol expected              hisG recombination excises CYB5-URA3 frag- <br /> because C-5 desaturation Cyb5p-dependent reaction.                  ment. <br />   Method III, disruption CYB5 URA3 blaster sys-                 HIS1 disruption CYB5 allele. Primers b5-5drII <br /> tem. Since direct transformation method suggested               b5-3drII (Table 2) designed contain 60 bp the <br />                                                                          CYB5-flanking sequence located 769 bp upstream the <br />                                                                          start codon 289 bp downstream stop codon, respec- <br />                                                                          tively, 20 bp plasmid sequence. They used to <br />                                                                          amplify HIS1 marker pGEM-HIS1. The resulting 2.7-kb <br />                                                                          fragment containing HIS1 gene flanked adjacent CYB5 <br />                                                                          sequence used transform strain BWP17, transfor- <br />                                                                          mants selected CSM-His&#226;&#171;&#185;Ade. Ten resulting <br />                                                                          93 transformants tested PCR proper configu- <br />                                                                          ration. Checking primers b5-5chII b5-3chII (Table 2) were <br />                                                                          designed amplify 1.9-kb fragment representing wild- <br />                                                                          type CYB5 allele 3.1-kb fragment representing HIS1- <br />                                                                          disrupted allele. Figure 6, lane A, shows presence both <br />                                                                          fragments heterozygote KRC9. <br />                                                                             Construction integration CYB5-URA3 blaster cas- <br />                                                                          sette. The vector pIU1300 used basic structure for <br />                                                                          URA3 blaster cassette. pIU1300 pBluescript which <br />                                                                          C. albicans ERG25 gene (encoding sterol C-4 methyl- <br />                                                                          oxidase) integrated. The URA3 blaster cassette has <br />                                                                          integrated ERG25 sequence. <br />                                                                             With primers b5-5promoter b5-3XbaI (Table 2), the <br />                                                                          CYB5 sequence including 654 bp promoter 187 bp of <br />                                                                          terminal sequence amplified BWP17. The resulting <br />                                                                          1.3-kb fragment contained XbaI sites ends. After re- <br />                                                                          moval XbaI sites pIU1300, resulting <br />                                                                          plasmid, pKR04, CYB5 fragment digested with <br />                                                                          XbaI, ligated, used transform E. coli DH5&#226;?&#163;. Forty <br />                                                                          resulting colonies 1,450 screened appropri- <br />                                                                          ate integration CYB5 sequence URA3 blaster <br />   FIG. 5. Sterol profiles C. albicans wild-type (BWP17), S.   adjacent URA3 gene. Transformants yielded the <br /> cerevisiae cyb5 mutant (Wb5&#226;&#338;&#172;), C. albicans cyb5 mutant (KCR7) <br /> grown maltose glucose, C. albicans cyb5 mutant <br />                                                                          proper orientation designated pKR05. <br /> (KCR12). Peaks: A, ergosterol; B, fecosterol; C, epsiterol; D, lanos-       XhoI ClaI used extract 9.1-kb fragment from <br /> terol; E, ergosta-7,22-dien-ol. RT, retention time.                      pKR05 containing erg25-hisG-CYB5-URA3-hisG-erg25 <br />  VOL. 48, 2004                                                                              STEROL SYNTHESIS IN C. ALBICANS                 3431 <br />  <br />  <br />  <br />  <br />    FIG. 6. PCR confirmation disruption C. albicans CYB5 gene URA3 blaster cassette. Lane A shows 3.1-kb fragment the <br /> HIS1-disrupted cyb5 allele 1.9-kb fragment wild-type CYB5 allele KCR9. Lane B shows 1.7-kb fragment indicating integration <br /> of URA3-CYB5 cassette ERG25 locus KCR10. Lane C confirms CYB5 heterozygosity KCR10. Lane D shows 3.7-kb fragment <br /> of ARG4-disrupted cyb5 allele 3.1-kb fragment HIS1-disrupted cyb5 allele KCR11. Lane E confirms (1.7-kb fragment) the <br /> continued presence URA3-CYB5 cassette ERG25 locus KCR11. Lane F shows presence ARG4- HIS1-disrupted cyb5 <br /> alleles KCR12. Lane G shows confirmation hisG recombination eliminating URA3-CYB5 construct (2.9-kb fragment) wild-type <br /> ERG25 allele second chromosome. <br />  <br />  <br /> construct. This fragment used transform KRC9,                  KRC12 isolates yielded 1.9-kb 2.9-kb fragments, indicating <br /> transformants (KRC10) selected CSM-His-Ura&#226;&#171;&#185;Ade.                   presence wild-type ERG25 allele hisG recom- <br /> To confirm integration ERG25 chromosomal locus,                    bination ERG25 locus, respectively (Fig. 6, lane <br /> primers hisG-3 E25-b5-3chI (Table 2) designed                 G). Sterol analysis KCR12 indicated predominant <br /> amplify sequence hisG repeat chromosomal                sterol ergosta-7,22-dien-ol, consistent sterol pro- <br /> region flanking ERG25, respectively. The expected fragment                file authentic S. cerevisiae cyb5 mutant Wb5&#226;&#338;&#172; (26). Other <br /> would 1.7 kb. Figure 6, lane B, shows presence              sterols lesser amounts, fecosterol, ergosterol, <br /> 1.7-kb fragment KRC10. KRC10 rechecked                episterol, lanosterol, C. albicans <br /> continued presence chromosomal heterozygosity (Fig. 6,             S. cerevisiae cyb5 mutants (Fig. 5). Cyb5p nonessential by <br /> lane C) CYB5.                                                          procedure. <br />    ARG4 disruption second CYB5 allele. Primers b5-                    Confirmation disruptions Southern analysis. Southern <br /> 5drIII b5-3drIII (Table 2) used amplify ARG4              analysis confirm results generated PCR for <br /> gene plasmid pRS-ARG4, described disruption               CYB5 heterozygotes cyb5 nulls created study. <br /> the CYB5 allele direct transformation above. The re-             Two probes employed. The located upstream of <br /> sulting 2.1-kb fragment used transform strain KRC10,               CYB5 chromosomal alleles designed detect wild- <br /> and transformants selected CSM-Arg-His-Ura.                       type disrupted versions CYB5 gene (primers b5- <br /> Twelve 240 transformants analyzed PCR                  5probeI b5-3probeI Table 2). The second probe was <br /> proper configuration. Primers b5-5chII b5-3chII (Table 2)             designed detect CYB5 gene integrated position <br /> were used check disrupted CYB5 alleles, primers                C. albicans genome. (primers b5-5probeX b5- <br /> hisG-3 E25-b5-3chI (Table 2) used confirm                 3probeX Table 2). DNA prepared probes with <br /> CYB5-URA3 blaster cassette integration ERG25 locus.                EcoRI. <br /> Figure 6, lane D, shows 3.1-kb HIS1-disrupted allele                 Figure 7A shows results probe 1. The wild-type <br /> the 3.7-kb ARG4-disrupted allele fragments, lane E shows              (BWP17) shows predicted 5.6-kb fragment representing <br /> the 1.7-kb fragment indicating cassette integration ERG25              wild-type CYB5 allele. Strain KRC3 containing ARG4- <br /> for KRC11.                                                                URA3-disrupted alleles shows heavy band approxi- <br />    Recombination hisG fragments cyb5 null. KCR11               mately 4.5 kb presence wild-type allele (5.6 kb). <br /> was freshly grown plated CSM containing 5-fluoroo-                 Further experiments URA3 ARG4 probes (prim- <br /> rotic acid. After 3 days incubation, total 40 colonies            ers Table 2) confirmed predicted 4.5-kb ARG4- <br /> emerged. The colonies patched CSM-Ura con-                   4.3-kb URA3-disrupted bands present heavy <br /> firm loss URA3 gene. Twenty colonies               band shown, confirming presence copy the <br /> confirmed Ura&#226;&#171;&#186;. Five isolates chosen confirmation             CYB5 gene KRC3. KRC7 shows doublet 4.5 kb, <br /> by PCR continuing CYB5 disruptions, hisG recombi-                 ARG4- URA3-disrupted alleles were <br /> nation expected resulting strain, KRC12. The ARG4-                 present. KRC9, heterozygote, shows pres- <br /> and HIS1-disrupted alleles confirmed present (Fig.          ence 5.6-kb wild-type allele 6.9-kb fragment <br /> 6, lane F) KRC12 isolates chosen. The fifth           representing HIS1-disrupted allele. KRC12 (and KRC11- <br /> showed HIS1-disrupted allele wild-type allele              shown) shows presence HIS1-disrupted allele <br /> and set aside. Primers E25-b5-3chI e25-5chII (Table 2)            (6.9 kb) ARG4-disrupted allele (4.5 kb). The results <br /> were used amplify 1.9-kb fragment representing wild-             confirm obtained PCR. <br /> type ERG25 allele 2.9-kb fragment representing hisG                Probe 2 results shown Fig. 7B. The chromosomal <br /> recombination product second allele                 wild-type CYB5 allele displayed 5.6-kb band BWP17. <br /> CYB5-URA3 blaster cassette inserted. All                  KRC12, expected, produced bands, indicating no <br />  3432     ROGERS ET AL.                                                                                           ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br />                                                                            TABLE 3. Effects antifungal drugs cellular inhibitors on <br />                                                                               wild-type (CYB5) cyb5 <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>null mutants</span> C. albicans <br />                                                                                                                         Inhibitory concna (&#226;?&#174;g/ml) <br />                                                                               Inhibitor <br />                                                                                                                     CYB5                            cyb5 <br />  <br />                                                                          Ketoconazole                              &#226;&#172;&#381;1.5                             0.5 <br />                                                                          Fluconazole                              &#226;&#172;&#381;10                               0.5 <br />                                                                          Itraconazole                               1.5                            &#226;&#172;?0.5 <br />                                                                          Tridemorph                                 0.025                          &#226;&#172;&#381;5 <br />                                                                          Fenpropimorph                              0.025                           0.1 <br />                                                                          Terbinafine                                3                               1 <br />                                                                          Cycloheximide                           &#226;&#172;&#381;140                            &#226;&#172;&#381;140 <br />                                                                          Cerulenin                                 &#226;&#172;&#381;1.5                             1.5 <br />                                                                          Fluphenazine                             &#226;&#172;&#381;10                             &#226;&#172;&#381;10 <br />                                                                           a <br />                                                                               Concentration visible growth 72 h incubation. <br />  <br />  <br />  <br />                                                                          glucose resulted increased azole sensitivity, expected with <br />                                                                          functional Cyb5p present. Induction CYB5 gene on <br />                                                                          maltose resulted azole sensitivity approaching levels <br />    FIG. 7. Confirmation CYB5 disruption strategies South-      seen wild type (Fig. 8). <br /> ern analysis. The upper panel shows results probe 1 detect <br /> wild-type disrupted versions CYB5. The wild-type band 5.6 kb <br /> is shown wild-type strain (BWP17) copy KCR3.                                   DISCUSSION <br /> KCR3 KCR7 presence band approximately <br /> 4.4 kb, comprising 4.3-kb URA3-disrupted 4.5-kb ARG4-dis-           The CYB5 gene C. albicans identified database <br /> rupted cyb5 alleles. The 6.9-kb band representing HIS1-disrupted <br />                                                                          based homology CYB5 gene S. cerevisiae. In <br /> cyb5 allele shown KCR9 KCR12. The lower panel shows the <br /> results probe 2. BWP17 shows wild-type allele (5.6-kb band),    order determine Cyb5p essential growth in <br /> while KCR12 shows bands confirming CYB5 gene          C. albicans, copies gene sequentially disrupted <br /> found location genome.                                     CYB5 sequences flanking selectable markers. The direct <br />                                                                          transformation disruption protocol resulted successful <br />                                                                          disruption alleles presence copy the <br /> copies CYB5 gene remain hisG recombination                  wild-type CYB5 allele. The third-copy phenomenon been <br /> eliminated cassette copy CYB5.                                    previously reported C. albicans thought occur <br />    Drug susceptibilities C. albicans cyb5 mutant KRC12.           essential genes (8). Based assumption, we <br /> The viable C. albicans cyb5 mutant KCR12 generates novel               conclude CYB5 gene essential. However, <br /> sterol profile, array sterol    gene essential S. cerevisiae, confirmation <br /> alters cell&#226;&#8364;&#8482;s sensitivity variety drugs. Of particular       approaches sought. <br /> interest sensitivities commonly used antifungal             With second protocol, copy CYB5 gene under <br /> agents. Figure 8 shows growth patterns C. albicans            control pMAL2, maltose-inducible promoter, was <br /> wild-type, BWP17 (CYB5), KRC12, cyb5 null,              inserted ADE2 locus CYB5 heterozygote. The sec- <br /> azoles (ketoconazole, fluconazole, itraconazole), inhibi-            ond allele CYB5 gene disrupted. This done <br /> tors Erg11p (30), morpholines (tridemorph fen-                presence maltose rescue cyb5 homozygotes should <br /> propimorph), inhibitors Erg24p Erg2p (4), terbin-             gene essential. When transferred glucose, cyb5 <br /> afine, inhibitor squalene epoxidase (Erg1p) (14). In               null (KRC7) grew normally, indicating gene nones- <br /> addition, cycloheximide, protein synthesis inhibitor, cerule-          sential C. albicans. Low-level leakage pMAL2 <br /> nin, fatty acid synthase inhibitor (23), fluphenazine,           promoter absence inducer anticipated, but <br /> calmodulin antagonist (10), tested.                                 previous studies (1, 3) promoter slow <br />    KCR12 shown significantly sensitive             growth results gene question essential. Sterol <br /> azoles wild type. Terbinafine sensitivity increased,       analysis KRC7 indicated profile included low levels of <br /> but cyb5 strain considerably resistant morpho-           ergosterol array intermediates closely resembled <br /> lines CYB5 strain. There changes sensitiv-           sterol accumulation cyb5 (Wb5&#226;&#338;&#172;) mutant S. cer- <br /> ities general metabolic inhibitors. These patterns shown          evisiae (28). This approach did reveal copy the <br /> in Table 3 terms drug concentration                CYB5 gene indicated gene essential. <br /> growth evident 72 h incubation.                                A final disruption procedure used confirm results <br />    In order confirm KRC12 azole-sensitive pheno-             pMAL2 inducible rescue cassette. This approach used <br /> type loss Cyb5p, copy CYB5 gene           URA3 blaster cassette copy CYB5 <br /> reintroduced KCR12. Plasmid pKR02 linearized                    gene integrated adjacent URA3 marker. The <br /> within ade2 sequence AvaI used transform                 cassette integrated unrelated chromosomal site in <br /> KRC12. Transformants (KRC13) selected CSM-Arg-                   genome CYB5 heterozygote. Following disruption of <br /> His-Ura. Figure 9 shows growth KRC13 glucose               second allele, resulting chromosomal cyb5 homozygote <br /> maltose presence tested azoles. Growth            plated 5-fluoroorotic acid eliminate URA3-CYB5 <br />  VOL. 48, 2004                                                                           STEROL SYNTHESIS IN C. ALBICANS                 3433 <br />  <br />  <br />  <br />  <br />      FIG. 8. Sensitivity C. albicans BWP17 (CYB5) cyb5 null (KRC12) variety antifungal agents cellular inhibitors. <br />  <br />  <br />  <br /> fragment hisG recombination. The resulting strain                   YAL36, CPP1, CPH1, FKS1 triplicated but <br /> (KRC12) viable, indicating, again, CYB5               nonessential. However, results suggest trisomic <br /> gene essential. This procedure advantage              CYB5 performing transformation CYB5/ <br /> having remaining copies CYB5 gene, eliminating           cyb5 heterozygote introduced rescue cas- <br /> the possibility recombination CYB5 allele, pos-         sette disruption second allele we <br /> sible pMAL2 (1, 3), sites          selected deletion CYB5 allele URA3 blaster <br /> control pMAL2.                                                       approach. Thus, direct transformation determine essential- <br />    The issue gene disruption complicated third-        ity reliable rescue deletion approaches. Im- <br /> allele phenomenon occurs C. albicans           portantly, pMAL2 inducible CYB5 cassette URA3 <br /> possible trisomies occur organism. Whereas se-             blaster-CYB5 deletion cassettes associated the <br /> quential gene disruption essential genes results dis-         CYB5 locus avoid allelic chromosomal ploidy interfering <br /> rupted alleles, allele observed. The origin         final test gene essentiality. <br /> trisomies unclear various C. albicans strains               Sterol analysis KRC12 indicated synthesis some <br /> have trisomic regions entire chromosomes. A recent report            ergosterol sterol intermediate accumulation similar that <br /> by Chen et al. (7) finds isolates CAI-4, chro-          S. cerevisiae Wb5&#226;&#338;&#172;. This indicates electron <br /> mosome 1 trisomic, explaining genes GLY1,                donor able partially substitute Cyb5p allows the <br />  <br />  <br />  <br />  <br />   FIG. 9. Azole sensitivity C. albicans KCR13, cyb5 null containing CYB5 gene pMAL2 control, grown glucose maltose. <br />  3434     ROGERS ET AL.                                                                                       ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br /> synthesis ergosterol. Based data, accompa-                                           REFERENCES <br /> nied PCR Southern analysis confirmations, conclude          1. Aaron, K. E., C. A. Pierson, N. D. Lees, M. Bard. 2001. The Candida <br /> that CYB5 gene essential C. albicans.                      albicans ERG26 gene encoding C-3 sterol dehydrogenase (C-4 decar- <br />                                                                          boxylase) essential growth. FEMS Yeast Res. 1:93&#226;&#8364;&#8220;101. <br />    The results obtained raise question regarding           2. Ausubel, F., R. Brent, R. E. Kingston, D. D. Moore, J. G. Seidman, J. A. <br /> third gene copy phenomenon. Chromosome duplication                   Smith, K. Struhl (ed.). 1995. Short protocols molecular biology, 3rd <br />                                                                          ed. John Wiley &amp; Sons, Inc., New York, N.Y. <br /> duplication smaller genomic elements (8) sug- <br />                                                                       3. Backen, A. C., I. D. Broadbent, R. W. Fetherston, J. D. C. Rosamond, N. F. <br /> gested mechanisms copies genes                 Schnell, M. J. R. Stark. 2000. Evaluation CaMAL2 promoter for <br /> arise. To point, copies shown occur         regulated expression genes Candida albicans. Yeast 16:1121&#226;&#8364;&#8220;1129. <br />                                                                       4. Baloch, R. I., E. I. Mercer. 1987. Inhibition sterol 8&#226;&#8364;&#8220;7 isomerase <br /> with essential genes, recently use UAU1 dis-              and14-reductase fenpropimorph, tridemorph, fenpropidin cell-free <br /> ruption procedure suggested rapid test determin-            enzyme systems Saccharomyces cerevisiae. Phytochemistry. 26:663&#226;&#8364;&#8220;668. <br /> ing given gene essential (8). Unfortunately,       5. Boschman, C. R., U. R. Bodnar, M. A. Tornatore, A. A. Obias, G. A. Noskin, <br />                                                                          K. Englund, M. A. Postelnick, T. Suriano, L. R. Peterson. 1998. Thir- <br /> procedure successfully implemented case,            teen-year evolution azole resistant yeast isolates prevalence resis- <br /> possibly CYB5 gene small. Since           tant strains carried cancer patients large medical center. Antimicrob. <br /> evidence copy disruption protocols,         Agents Chemother. 42:734&#226;&#8364;&#8220;738. <br />                                                                       6. Brown, D. H., I. V. Slobodkin, C. Kumamoto. 1996. Stable transforma- <br /> the results indicate possibility third-copy phe-       tion regulated expression inducible promoter construct Candida <br /> nomenon occur spontaneous event certain cases                albcians using restiction enzyme-mediated integration. Mol. Gen. Genet. <br />                                                                          251:75&#226;&#8364;&#8220;80. <br /> when chromosomal alleles inactivated. Alterna- <br />                                                                       7. Chen, X., B. B. Magee, D. Dawson, P. T. Magee, C. Kumamoto. 2004. <br /> tively, successful direct disruption procedure             Chromosome 1 trisomy compromises virulence Candida albicans. Mol. <br /> third copy occurred small sub-              Microbiol. 51:551&#226;&#8364;&#8220;565. <br />                                                                       8. Enloe, B., A. Diamond, A. P. Mitchell. 2000. A single-transformation <br /> population CYB5 allele existed triplicate.               gene function test diploid Candida albicans. J. Bacteriol. 182:5730&#226;&#8364;&#8220;5736. <br />    Since inhibition Cyb5p result viable cell       9. Fonzi, W. A., M. Y. Irwin. 1992. Isogenic strain construction gene <br /> altered sterol composition, determine              mapping Candida albicans. Genetics 134:717&#226;&#8364;&#8220;728. <br />                                                                      10. Hait, W. N., J. F. Gesmonde, J. S. Lazo. 1993. Effect anti-calmodulin <br /> whether cell altered responses common               drugs growth sensitivity C6 rat glioma cells bleomycin. <br /> inhibitors antifungal drugs. Surprisingly, sig-           Anticancer Res. 14:1711&#226;&#8364;&#8220;1721. <br /> nificant increase azole sensitivity KCR12. In order        11. Hata, S., T. Nishino, H. Katsuki, Y. Aoyama, Y. Yoshida. 1983. Two <br />                                                                          species cytochrome P-450 involved ergosterol biosynthesis yeast. <br /> confirm linkage azole sensitivity lack Cyb5p,          Biochem. Biophys. Res. Commun. 116:162&#226;&#8364;&#8220;166. <br /> the azole experiment repeated KRC13, derivative        12. Hata, S., T. Nishino, M. Konori, H. Katsuki. 1981. Involvement of <br /> KCR12. KCR13 KRC12 pMAL2-CYB5 cas-                     cytochrome P-450 &#226;&#338;&#172;22-desaturation ergosterol biosynthesis yeast. <br />                                                                          Biochem. Biophys. Res. Commun. 103:272&#226;&#8364;&#8220;277. <br /> sette integrated ADE2 locus. When grown           13. Imai, Y. 1981. The roles cytochrome-b5 reconstituted mono-oxygenase <br /> glucose, KRC13 shows <span id='am-238' about='xsp:pattern' typeof='owl:Thing'>pattern</span> increased inhibition                systems containing various forms hepatic microsomal cytochrome P-450. <br />                                                                          J. Biochem. 89:351&#226;&#8364;&#8220;362. <br /> azoles similar seen KCR12. The increased sensi-         14. Jandrositz, A., F. Turnowski, G. Hogenauer. 1991. The gene encoding <br /> tivity quite great, probably low-level leakage          squalene epoxidase Saccharomyces cerevisiae: cloning characteriza- <br /> through pMAL2 promoter (1, 3). Nearly normal azole                   tion. Gene 107:155&#226;&#8364;&#8220;160. <br />                                                                      15. Jensen-Perkages, K. L., M. A. Kennedy, N. D. Lees, R. Barbuch, C. Koegel, <br /> sensitivity restored KCR13 grown maltose, indi-            M. Bard. 1998. Sequencing, disruption, characterization the <br /> cating loss Cyb5p responsible increased           Candida albicans sterol methyltransferasae (ERG6) gene: drug susceptibility <br /> azole sensitivity. Other C. albicans sterol mutants            studies erg6 mutants. Antimicrob. Agents Chemother. 42:1160&#226;&#8364;&#8220;1167. <br />                                                                      16. Jia, N., B. Artington-Skaggs, W. Lee, C. A. Pierson, N. D. Lees, J. Eckstein, <br /> produced altered sterol profiles showed increases             R. Barbuch, M. Bard. 2002. Candida albicans sterol C-14 reductase, <br /> azole sensitivity (15, 16). Coupled changes inhibition        encoded ERG24 gene, potential antifungal target site. Antimi- <br /> levels cellular inhibitors, indicates increased           crob. Agents Chemother. 46:947&#226;&#8364;&#8220;957. <br />                                                                      17. Kalb, V. F., C. W. Woods, T. G. Turi, C. R. Drey, T. R. Sutter, J. C. <br /> azole sensitivity likely increased permeability         Loper. 1987. Primary structure P450 lanosterol demethylase gene from <br /> these compounds. The cyb5 null sensitive                Saccharomyces cerevisiae. DNA. 6:529&#226;&#8364;&#8220;537. <br />                                                                      18. Kennedy, M. A., T. A. Johnson, N. D. Lees, R. Barbuch, J. A. Eckstein, and <br /> terbinafine, inhibitor squalene epoxidase.                         M. Bard. 2000. Cloning sequencing Candida albicans C-4 sterol <br />    These results azoles consistent drugs&#226;&#8364;&#8482;        methyl oxidase gene (ERG25) expression ERG25 conditional <br /> having effects substitute electron donors reactions         lethal <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>mutation</span> Saccharomyces cerevisiae. Lipids 35:257&#226;&#8364;&#8220;262. <br />                                                                      19. Kurtz, M. B., J. Marrinan. 1989. Isolation hem3 mutants from <br /> the sterol biosynthetic pathway. Both squalene epoxidase             Candida albicans sequential disruptions. Mol. Gen. Genet. 217:47&#226;&#8364;&#8220;52. <br /> and C-14 demethylase steps dependent NCP1            20. Lamb, D. C., D. E. Kelly, N. J. Manning, M. A. Kaderbhai, S. L. Kelly. <br /> an electron carrier. The ncp1 genotype shown            1999. Biodiversity P450 catalytic cycle: yeast cytochrome b5/NADH <br />                                                                          cytochrome b5 reductase complex efficiently drives entire sterol 14- <br /> be sensitive azoles, indicating possibility             demthylation (CYP51) reaction. FEBS Lett. 462:283&#226;&#8364;&#8220;288. <br /> Cyb5p Cprp significant functional overlap            21. Marcireau, C., M. Guilloton, F. Karst. 1990. In vivo effects fenpropi- <br /> that sensitive azole inhibition. Further investi-         morph yeast Saccharomyces cerevisiae determination mo- <br />                                                                          lecular basis antifungal property. Antimicrob. Agents Chemother. <br /> gations needed understand cyb5 mutants               34:989&#226;&#8364;&#8220;993. <br /> resistant morpholine class sterol biosynthesis inhibi-     22. Molzhan, S. W., R. A. Woods. 1972. Polyene resistance isolation <br />                                                                          sterol mutants Saccharomyces cerevisiae. J. Gen. Microbiol. 72:339&#226;&#8364;&#8220;348. <br /> tors. However, S. cerevisiae elo2 elo3 sphingolipid mutants <br />                                                                      23. Morisaki, N., H. Funabashi, R. Shimazawa, J. Furukawa, A. Kawaguchi, S. <br /> (24) commonly morpholine resistant,            Okuda, S. Iwasaki. 1993. Effect side-chain structure inhibition of <br /> mutations known suppressors erg2 erg24 (21).            yeast fatty acid synthase cerulenin analogues. Eur. J. Biochem. 211:111&#226;&#8364;&#8220; <br />                                                                          115. <br />                                                                      24. Oh, C. S., D. A. Toke, S. Mandela, C. E. Martin. 1997. ELO2 ELO3 <br />                                                                          homologues Saccharomyces cerevisiae ELO1 gene function fatty acid <br />                      ACKNOWLEDGMENT                                      elongation required sphingolipid formation. J. Biol. Chem. 272: <br />                                                                          17376&#226;&#8364;&#8220;17384. <br />  M.B. recipient Burroughs Wellcome Mycology Scholar        25. Osumi, T., T. Nishino, H. Katsuki. 1979. Studies &#226;&#338;&#172;5-desaturation of <br /> Award.                                                                   ergosterol biosynthesis yeast. J. Biochem. 85:819&#226;&#8364;&#8220;826. <br />  VOL. 48, 2004                                                                                        STEROL SYNTHESIS IN C. ALBICANS                          3435 <br />  <br /> 26. Pierson, C. A., N. Jia, C. Mo, N. D. Lees, A. M. Sturm, J. Eckstein, R.      29. Tsai, H.-F., M. Bard, K. Izumikawa, A. A. Krol, A. M. Sturm, N. T. Culbert- <br />     Barbuch, M. Bard. Isolation, characterization, regulation         son, C. A. Pierson, J. E. Bennett. 2004. A Candida glabrata erg1 mutant <br />     Candida albicans ERG27 gene encoding sterol 3-keto reductase. J. Med.        increased sensitivity azoles low oxygen tension. Antimicrob. <br />     Mycol., press.                                                                Agents Chemother. 48:2483&#226;&#8364;&#8220;2489. <br /> 27. Sutter, T. R., J. C. Loper. 1989. Disruption Saccharomyces cer-   30. Vanden Bossche, H., G. Willemssens, P. Marichal. 1987. Anti-Candida <br />     evisiae gene NADH-cytochrome P450 reductase causes increased sensi-          drugs&#226;&#8364;&#8220;the biochemical basis activity. Crit. Rev. Microbiol. 15:57&#226;&#8364;&#8220;72. <br />     tivity ketoconazole. Biochem. Biophys. Res. Commun. 160:1257&#226;&#8364;&#8220;1266.        31. Wilson, R. B., D. Davis, A. P. Mitchell. 1999. Rapid hypothesis testing <br /> 28. Truan, G., J.-C. Epinat, C. Rougeulle, C. Cullin, D. Pompon. 1994.           Candida albicans gene disruption short homology regions. <br />     Cloning characterization yeast cytochrome b5-encoding gene        J. Bacteriol. 181:1868&#226;&#8364;&#8220;1874. <br />     suppresses ketoconazole hypersensitivity NADPH-P-450 reductase-de-      32. Yoshida, Y. 1988. Cytochrome P450 fungi: primary target azole anti- <br />     ficient strain. Gene 149:123&#226;&#8364;&#8220;127.                                                fungal agents. Curr. Top. M <br /> </body></html>