<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Analysis Sequence, Map Position, Gene <br /> Expression Reveals Conserved Essential Genes <br /> for Iron Uptake Arabidopsis Tomato1[w] <br />  <br /> Petra Bauer*, Thomas Thiel, Marco Klatte, Zsolt Bereczky, Tzvetina Brumbarova, <br /> Ru&#194;&#168;diger Hell2, Ivo Grosse <br /> Institute Plant Genetics Crop Plant Research, D&#226;&#8364;&#8220;06466 Gatersleben, Germany <br />  <br />  <br /> Arabidopsis (Arabidopsis thaliana) tomato (Lycopersicon esculentum) similar physiological responses iron deficiency, <br /> suggesting homologous genes involved. Essential gene functions generally considered carried by <br /> orthologs remained conserved sequence map position evolutionarily related species. This assumption has <br /> not proven plant genomes underwent large genome rearrangements. We addressed question an <br /> attempt deduce functional gene pairs iron reduction, iron transport, iron regulation Arabidopsis and <br /> tomato. Iron uptake processes essential plant growth. We investigated iron uptake gene pairs tomato and <br /> Arabidopsis, sequence, conserved gene content regions containing iron uptake homologs based conserved <br /> orthologous set marker analysis, gene expression patterns, and, cases, genetic data. Compared tomato, the <br /> Arabidopsis genome revealed larger gene families coding iron uptake functions. The number possible <br /> homologous pairs reduced functional expression data taken account addition sequence map <br /> position. We predict novel homologous partially redundant functions ferric reductase-like iron-regulated <br /> transporter-like genes Arabidopsis tomato. Arabidopsis nicotianamine synthase genes encode partially redundant <br /> family. In study, Arabidopsis gene redundancy generally reflected presumed genome duplication structure. In some <br /> cases, statistical analysis conserved gene regions tomato Arabidopsis suggested common evolutionary origin. <br /> Although involvement conserved genes iron uptake found, essential genes paralogous than <br /> orthologous origin tomato Arabidopsis. <br />  <br />  <br />   Arabidopsis (Arabidopsis thaliana) serves ref-                  Yeast (Saccharomyces cerevisiae) 1,000 <br /> erence dicot genome analysis regarding gene                          essential genes (Winzeler et al., 1999; Giaever et al., <br /> sequence, gene number, gene function. In mini-                   2002). In plants, number conserved essential <br /> mal genome concept, assumed orthologous                       plant genes remains experimentally deter- <br /> gene functions transmitted lineage-dependent                        mined. It unclear searching conserved <br /> vertical manner related organisms con-                      sequence homologs expressed sequence tag (EST) <br /> served sequence function essential                    genomic databases diverse plant species may <br /> (Mushegian, 1999). Using systematic gene <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span>knockout</span>                        provide clues essential gene functions. A potential <br /> approaches genome sequence comparisons,                       obstacle homolog searches plants fact <br /> calculated Bacillus subtilis 250                  plant genomes underwent large evolutionary <br /> 300 essential genes (Kobayashi et al., 2003). Ninety-six                 genomic rearrangements involving polyploidization, <br /> percent essential genes conserved bacteria                     chromosome rearrangement, partial gene loss <br /> and encode proteins known functions cell                         (Gaut et al., 2000; Simillion et al., 2002). The analysis <br /> metabolism energetics, information processing,                       genome sequence suggested Arabi- <br /> and cell growth division (Kobayashi et al., 2003).                   dopsis genome contained duplicated blocks corre- <br />                                                                          sponding ancient recent genome duplication <br />                                                                          events (Arabidopsis Genome Initiative, 2000; Blanc <br />    1 <br />      This work supported Deutsche Forschungsgemeinschaft          et al., 2000, 2003; Vision et al., 2000; Simillion et al., <br /> grants Emmy Noether program (Ba1610/3&#226;&#8364;&#8220;1 P.B.)       2002; Bowers et al., 2003; Ermolaeva et al., 2003). These <br /> Arabidopsis Functional Genomics Network program (Ba1610/4&#226;&#8364;&#8220;1           genome duplication events led horizontal multipli- <br /> P.B. R.H.), Bundesministerium fu&#194;&#168;r Bildung und For-       cation homologous genes known paralogs. The <br /> schung Bioinformatics Center Gatersleben-Halle (grant I.G.).   presence paralogous genes hampers straightfor- <br />    2 <br />      Present address: Heidelberg Institute Plant Sciences, Uni-      ward genome colinearity studies diverged <br /> versity Heidelberg, Im Neuenheimer Feld 360, D&#226;&#8364;&#8220;69120 Heidel- <br />                                                                          species different families. <br /> berg, Germany. <br />    * Corresponding author; e-mail bauer@ipk-gatersleben.de; fax <br />                                                                            Full genomic sequence Arabidopsis <br /> 49&#226;&#8364;&#8220;39482&#226;&#8364;&#8220;5139.                                                           chromosomes skeletons genomic EST mark- <br />    [w] <br />        The on-line version article contains Web-only data.       ers mapped chromosomes dicot <br />    Article, publication date, citation information   species currently available assessing homolo- <br /> www.plantphysiol.org/cgi/doi/10.1104/pp.104.047233.                      gous gene functions (Fulton et al., 2002; Gebhardt et al., <br /> Plant Physiology, December 2004, Vol. 136, pp. 4169&#226;&#8364;&#8220;4183, www.plantphysiol.org &#195;&#8220; 2004 American Society Plant Biologists     4169 <br />  Bauer et al. <br />  <br />  <br /> 2003; Zhu et al., 2003). Tomato (Lycopersicon esculentum)    LeIRT1, LeNRAMP1, LeFRO1 response iron <br /> belongs plant species detailed     availability tomato roots (Ling et al., 2002; Bereczky <br /> genetic map 1,000 mapped EST sequences            et al., 2003). <br /> is available (Tanksley et al., 1992; Fulton et al., 2002).      In study, term homologous genes proteins <br /> Tomato EST markers identify unique genes         refers genes proteins similar sequences <br /> Arabidopsis genome termed conserved ortholo-            qualify share common properties, as <br /> gous set (COS) markers utilization comparative        specific transporters enzymes. The term homolo- <br /> mapping (Fulton et al., 2002). Comparisons map            gous function specifies homologous genes and <br /> positions COS markers tomato Arabidopsis           proteins act similar biological context, as <br /> revealed evidence small regions fairly con-          metal transporter root iron mobilization. We made <br /> served gene order (microsyntenies)            use essential characters genes FRO2, IRT1, <br /> disrupted genes fit apparent         NRAMP, NAS, FER analyze homology between <br /> syntenic regions (Fulton et al., 2002). Microsynteny     gene functions Arabidopsis tomato. By <br /> be observed comparing order predicted            investigating sequence similarity, map position, and <br /> genes tomato Arabidopsis bacterial artificial         functional expression data, identified gene pairs <br /> chromosome clones (Ku et al., 2000, 2001; Mao et al.,        represent homologous functions two <br /> 2001; Rossberg et al., 2001; Oh et al., 2002; Van der        species. We discuss conservation gene function with <br /> Hoeven et al., 2002). In general,              respect orthologous paralogous origin the <br /> Arabidopsis region identified potentially          genes. <br /> colinear, reflect ancient polyploidization <br /> events species. <br />    Since iron required basic enzymatic <br /> reactions biological processes organisms,         RESULTS <br /> iron uptake strictly essential mechanism growth     Identification Sequence Comparison of <br /> of organism. In plants, iron deficiency caused        Tomato-Arabidopsis Iron Uptake Homologs <br /> when iron available low solubility, is <br /> the case alkaline calcareous soils. Plants           <span id='am-6' about='protege:TO' typeof='owl:Thing'>To</span> identify homologs iron uptake proteins <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> able cope iron deficiency mobilize           tomato Arabidopsis, screened databases <br /> sufficient iron environment (Hell             using amino acid sequences AtIRT1, LeIRT1, <br /> Stephan, 2003). Lack appropriate iron mobilization        LeNRAMP1, LeNRAMP3, AtFRO2, LeNAS, and <br /> results leaf chlorosis severe growth retardation.     LeFER (&#226;&#8364;&#732;&#226;&#8364;&#732;Materials Methods;&#226;&#8364;&#8482;&#226;&#8364;&#8482; Table I). We only <br /> Both tomato Arabidopsis mobilize iron re-             retained sequences analysis were <br /> duction increased uptake Fe II (for review,       related according E-values (&#226;&#8364;&#732;&#226;&#8364;&#732;Materials and <br /> Bauer Bereczky, 2003; Curie Briat, 2003).            Methods&#226;&#8364;&#8482;&#226;&#8364;&#8482;). Partial sequences named according <br /> Essential gene functions involved iron assimilation       transcript <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> number database (e.g. <br /> in Arabidopsis demonstrated         LeFRO-TC129233). The available peptide sequences <br /> the ferric chelate reductase (FRO) gene AtFRO2        aligned represented phylogenetic trees <br /> the root plasma membrane iron-regulated transporter          (Fig. 1, shown IRT NAS sequences). <br /> (IRT) gene AtIRT1 (Robinson et al., 1999; Varotto et al.,       LeIRT1 LeIRT2 showed highest sequence sim- <br /> 2002; Vert et al., 2002). AtIRT1 AtFRO2 genes be-        ilarity (Fig. 1A). Among 15 <br /> long gene families (Eng et al., 1998; Robinson et al.,    Arabidopsis zinc iron-regulated (ZIP) transporter <br /> 1999). Natural resistance-associated macrophage pro-         sequences (Ma&#194;&#168;ser et al., 2001), be <br /> tein-like metal transporter genes (NRAMP1                grouped LeIRT1 LeIRT2 (Fig. 1A). <br /> NRAMP3/NRAMP4) up-regulated response               One these, AtIRT1, shown essential for <br /> iron deficiency tomato Arabidopsis in-            iron uptake, AtIRT2 iron regulated but <br /> fluence metal homeostasis Arabidopsis (Curie et al.,      essential iron uptake (Vert et al. 2001, 2002; <br /> 2000; Thomine et al., 2000, 2003; Bereczky et al., 2003).    Varotto et al., 2002). In sequence comparisons, <br /> In mammalian yeast systems, shown            AtIRT1 AtIRT2 did appear closest <br /> NRAMP homologs essential iron uptake                homologs. In fact, AtIRT1 similar AtZIP8, <br /> (Fleming et al., 1997; Liu Culotta, 1999). Nicotian-     LeIRT1 LeIRT2 similar AtZIP10. <br /> amine synthase (NAS) encoded chloronerva gene         AtZIP7 distantly related IRT proteins. All other <br /> (LeNAS) required synthesis metal chelator      ZIP sequences database more <br /> nicotianamine (Herbik et al., 1999; Higuchi et al., 1999;    distant IRT AtZIP7. <br /> Ling et al., 1999). Nicotianamine essential iron         LeNRAMP1, LeNRAMP3, AtNRAMP1, AtNRAMP3, <br /> distribution plants (Scholz et al., 1992; Ling et al.,    AtNRAMP4 encoded iron-regulated genes <br /> 1999, 2002; Takahashi et al., 2003). The regulatory basic    (Curie et al., 2000; Thomine et al., 2000; Bereczky et al., <br /> helix-loop-helix (bHLH) domain protein LeFER              2003). As shown previously, LeNRAMP1 be <br /> essential iron uptake presumably acts             aligned AtNRAMP1 AtNRAMP6, LeN- <br /> a transcription factor (Ling et al., 2002). LeFER controls   RAMP3 aligned AtNRAMP3 AtN- <br /> directly indirectly iron reduction expression      RAMP4 (Bereczky et al., 2003). LeNRAMP-AI778139 <br /> 4170                                                                                        Plant Physiol. Vol. 136, 2004 <br />  Table I. Amino acid sequence comparisons tomato Arabidopsis homologs alignment results using BLASTP http://www.arabidopsis. <br /> org/Blast <br />    Arabidopsis iron uptake homologs represented considered analysis showed E-values similar best hits. The only <br /> exceptions FRO proteins marked **, retained map position. For purpose highlighting tomato-Arabidopsis <br /> conserved gene clusters Figure 3, iron uptake genes marked color <span id='am-4' about='Thesaurus:code' typeof='owl:Thing'>code</span> (compare Table I Fig. 3 Supplemental Table I). Arabidopsis <br /> iron uptake homologs located distance 6400 genes (about 62 Mb) marked color (e.g. At4g19680-IRT2 At4g19690-IRT1). <br /> Homologous Arabidopsis iron uptake genes located different regions genome (at distance 400 genes) highlighted by <br /> different colors. The best hits highlighted dark blue, light blue, dark green, light green, brown-orange, light orange. <br />  <br />  <br />  <br />  <br /> Plant Physiol. Vol. 136, 2004                                                                                                                       4171 <br />  Bauer et al. <br />  <br />  <br />  <br />  <br />           Figure 1. Phylogenetic trees Arabidopsis tomato iron uptake homologs. Sequences aligned using ClustalX. N-J Trees <br />           generated; relative branch lengths indicated. Arabidopsis sequences retrieved gene locus number from <br />           http://mips.gsf.de/proj/thal/db/search/search_frame.html. Tomato sequences retrieved using TC numbers http:// <br />           www.tigr.org/tdb/tgi/lgi/searching/reports.html. The comparisons include homologs IRT metal transporter (A) NAS (B). <br />  <br />  <br />  <br />  <br /> could aligned AtNRAMP2 (phylogenetic tree                          genomes. For map position comparison, neces- <br /> not shown).                                                                sary map tomato genes. <br />    AtFRO2 required iron reduction low iron                        LeNAS LeFER map positions known be <br /> supply (Robinson et al., 1999). Three LeFRO sequences                      tomato chromosomes 1 6, respectively (Ling <br /> and Arabidopsis FRO sequences significant                         et al., 1999, 2002). Here, mapped IRT1, FRO, and <br /> similarity AtFRO2. The                             NRAMP homologs restriction fragment <span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> <br /> LeFRO-TC124302 sequence similar Ara-                           polymorphism analysis tomato genome <br /> bidopsis FRO homologs At5g23980 At5g23990.                             (&#226;&#8364;&#732;&#226;&#8364;&#732;Materials Methods;&#226;&#8364;&#8482;&#226;&#8364;&#8482; Fig. 2). We that <br /> LeFRO1 LeFRO-TC129233 related                                LeIRT1 LeIRT2 mapped chromosome 2 <br /> to AtFRO2 AtFRO1. The FRO-like pro-                              CT255 TG554. LeNRAMP1 localized <br /> teins&#226;&#8364;&#8221;At5g50160, At5g49730, At5g49740&#226;&#8364;&#8221;were                             chromosome 11 TG47 TG400. LeN- <br /> not related sequence. Their genes                          RAMP3 localized chromosome 2 between <br /> retained mapping location,                     CT59 TG154. LeFRO-TC124302 mapped <br /> shown later.                                                               chromosome 3 CT171 TG525. LeFRO- <br />    NAS essential iron homeostasis tomato.                        TC129233 LeFRO1 localized between <br /> Four different Arabidopsis NAS sequences                       TG460 CT165 chromosome 1. All genes were <br /> database (see Suzuki et al., 2001; Becher et al.,                     mapped codominant markers exception of <br /> 2004), related                        Lefro-TC124302, mapped dominant <br /> than tomato LeNAS (CHLORONERVA; Fig. 1B).                           marker L. esculentum background. Since homo- <br /> Arabidopsis NAS sequences divided                        log Lefro-TC124302 the <br /> subgroups, encoded genes chromosomes 1                           potato EST database, hypothesize gene is <br /> 5, respectively.                                                           lacking potato Lycopersicon pennellii <br />    A single Arabidopsis protein named FER-                         genomes. <br /> like regulator iron uptake (AtFRU;At2g28160, <br /> bHLH029; Heim et al., 2003) corresponded LeFER <br /> (see Jakoby et al., 2004). All predicted bHLH                   Analysis Conserved Gene Content Arabidopsis <br /> domain protein sequences Arabidopsis                        Tomato Chromosomal Regions Containing Iron <br /> tomato differed significantly outside bHLH do-                         Uptake Genes <br /> main (data shown).                                                        Chromosomal regions harboring iron uptake genes <br />    In summary, possible predict unique                         Arabidopsis tomato genomes analyzed for <br /> homologous Arabidopsis-tomato gene pair                            level conserved genes. For analysis, we <br /> a single case, LeFER-AtFRU. For                         selected COS markers mapped Fulton <br /> other gene functions Arabidopsis                         et al. (2002) approximately 10-cM distance of <br /> larger gene families encoding functions                         tomato iron uptake genes could <br /> tomato.                                                                    identify corresponding Arabidopsis homologs by <br />                                                                            sequence similarity searches (&#226;&#8364;&#732;&#226;&#8364;&#732;Materials Methods;&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> Mapping Iron Uptake Genes Tomato                                     Supplemental Table I, available www.plantphysiol. <br />                                                                            org). Among Arabidopsis COS markers, iden- <br />   A criterion gene homology                        tified located 400 genes up- <br /> species location colinear region                        stream downstream respective iron uptake <br /> 4172                                                                                                             Plant Physiol. Vol. 136, 2004 <br />                                                                                              Functional Homology Iron Uptake Genes <br />  <br />  <br />  <br />  <br />          Figure 2. Mapping tomato iron uptake genes. A, LeIRT1, LeIRT2, LeNRAMP3 located chromosome 2; B, <br />          LeNRAMP1 chromosome 11; C, LeFRO-TC124302 chromosome 3; D, LeFRO1 LeFRO-TC129233 chromosome 1. <br />          LeNAS (chloronerva) previously fine mapped chromosome 1 (Ling et al., 1996, 1999), LeFER fine mapped to <br />          chromosome 6 (Ling et al., 1996, 2002). <br />  <br />  <br /> genes (approximately 62 Mb) defined                        FRO-like homologs At5g23990/At5g23980 <br /> being located similar region (&#226;&#8364;&#732;&#226;&#8364;&#732;Materials                     (4 COS markers 35), AtFRO2/AtFRO1 (one COS <br /> Methods;&#226;&#8364;&#8482;&#226;&#8364;&#8482; Supplemental Table I; Fig. 3). For illustra-                marker), At5g50160/At5g49730/At5g49740 (seven <br /> tion, COS markers iron uptake genes                       COS markers; Fig. 3D). The LeFRO1/LeFRO-TC129233 <br /> homologous colocalize tomato Arabi-                     region showed similarity regions AtFRO2/ <br /> dopsis genomes highlighted color                      AtFRO1 (3 COS markers 21) as <br /> (Table I; Supplemental Table I; Fig. 3).                               At5g23980/At5g23990 (one COS marker; Fig. 3E). No <br />    We 8 tomato COS markers 24                   indication conserved genes the <br /> located region LeIRT1/LeIRT2 recognized                  region Arabidopsis FRO3. <br /> at Arabidopsis region containing                         For LeNAS regions identified corre- <br /> AtIRT1/AtIRT2, AtZIP8, AtZIP10 (Fig. 3A). Two                    sponding regions Arabidopsis (twice 2 twice 3 <br /> these tomato COS markers homologous mul-                       COS markers 29; Fig. 3F), indicating four <br /> tiple Arabidopsis COS markers located                     Arabidopsis NAS regions related. <br /> two IRT/ZIP regions, respectively. There-                       Only single COS marker 21 recognized the <br /> fore, identify Arabidopsis regions                      LeFER AtFRU regions (Fig. 3G). <br /> harboring IRT/ZIP genes shared con-                          With exception LeFER/AtFRU regions, we <br /> served gene sequences LeIRT1/LeIRT2 region                    determine 20% 50% analyzed <br /> in tomato.                                                             tomato COS markers recognized Arabidopsis genes <br />    Although 10 COS markers available                     located clusters vicinity respective iron <br /> the LeNRAMP1 region, identify 2                     uptake genes. <br /> located near AtNRAMP1 (Fig. 3B). Single LeNRAMP1-                         We realized clusters conserved COS markers <br /> neighboring COS markers identified regions AtN-                     regions did not <br /> RAMP4, AtNRAMP5, AtNRAMP6. Three COS                               appear related presence homologous iron <br /> markers located near AtNRAMP3. The LeN-                           uptake genes. To analyze point further, we <br /> RAMP3 region showed similarity region                        searched matches COS markers 6 un- <br /> AtNRAMP3 (3 COS markers 26), and, lesser                   related iron uptake regions (42 comparisons non- <br /> extent, AtNRAMP4 (two COS markers)                      homologous regions 7 comparisons of <br /> AtNRAMP5 (one COS marker; Fig. 3C).                                    homologous regions control; Table II). We found <br />    The tomato region LeFRO-TC124302 showed                          number matching COS markers highest <br /> similar gene sequences Arabidopsis regions                      comparing Arabidopsis tomato regions with <br /> Plant Physiol. Vol. 136, 2004                                                                                                     4173 <br />  Bauer et al. <br />  <br />  <br />  <br />  <br />           Figure 3. COS marker analysis tomato Arabidopsis chromosomal regions containing iron uptake genes. Represented were <br />           tomato chromosome linkage maps modified high density molecular marker map (on left side: <br />           Tanksley et al., 1992) COS marker map (on right side: Fulton et al., 2002). The original maps downloaded from <br />           http://www.sgn.cornell.edu/maps/tomato_arabidopsis/synteny_map.html. Gene locus names corresponding Arabidopsis <br />           iron uptake genes indicated. Arabidopsis gene locus names reflect chromosomal position. In cases, multiple <br />           Arabidopsis iron uptake homologs corresponding tomato homolog. Such multiple homologs boxed. COS <br />           markers located 610-cM distance tomato iron uptake genes corresponded Arabidopsis COS markers <br />           located range 400 genes upstream downstream iron uptake homologs (about 62 Mb) indicated. <br />           Multiple Arabidopsis COS markers homologs single tomato COS marker boxed. To better illustrate the <br />           presence conserved gene clusters tomato Arabidopsis iron uptake gene regions, color <span id='am-323' about='Thesaurus:code' typeof='owl:Thing'>code</span> used to <br />           indicate clustered Arabidopsis genes (compare Table I Supplemental Table I). Homologous iron uptake genes 6400 <br />           genes (about 62 Mb) denoted color (e.g. At4g19680-IRT2 At4g19690-IRT1). Homologous Arabidopsis iron <br />           uptake genes located different regions genome (at distance 400 genes) highlighted by <br />           different colors. Similarly, Arabidopsis COS markers received colors neighboring iron uptake genes located <br />           6400 genes (about 62 Mb; e.g. COS marker At4g20410 color At4g19690-IRT1). Therefore, iron uptake <br />           genes COS markers represented color map form cluster chromosome. The tomato regions <br />           analyzed contain LeIRT1, LeIRT2 chromosome 2 (A); LeNRAMP1 chromosome 11 (B); LeNRAMP3 chromosome 2 (C); <br />           LeFRO-TC124302 chromosome 3 (D); LeFRO1 LeFRO-TC129233 chromosome 1 (E); LeNAS chromosome 1 (F); <br />           LeFER chromosome 6 (G). With exception LeFER-AtFRU, tomato-Arabidopsis regions harboring homologous iron <br />           uptake genes showed levels conserved gene content. <br />  <br />  <br />  <br /> corresponding iron uptake genes (three con-                           (FRO-like), LeNRAMP1-AtFRU. In 19 comparisons <br /> served genes; exception LeFER/AtFRU). Only                           nonhomologous iron uptake gene regions, no <br /> in case LeFER highest level gene                            matching COS marker detected. In 12 cases, single <br /> conservation observed different region. Only 4                         matching COS marker found. In cases, two <br /> out 42 comparisons regions nonhomologous                           matching COS markers found. These observa- <br /> iron uptake genes showed similar levels conserved                          tions indicate level conserved gene se- <br /> gene content regions corresponding iron                           quences homologous iron uptake gene <br /> uptake genes (three conserved COS markers),                          regions generally tends higher homol- <br /> namely      Lefro-TC124302-AtIRT1/AtIRT2,     LeFRO-                          ogous iron uptake gene regions regions <br /> TC124302-At1g09240 (AtNAS3), LeFER-At5g23990                                  unrelated iron uptake genes. <br /> 4174                                                                                                                 Plant Physiol. Vol. 136, 2004 <br />                                                                                               Functional Homology Iron Uptake Genes <br />  <br />  <br />  <br /> Table II. Number COS marker matches seven Arabidopsis seven tomato regions containing iron uptake homologs, resulting in <br /> 42 comparisons regions nonhomologous iron uptake genes seven control comparisons regions homologous iron uptake <br /> genes (in bold) <br />   Iron uptake homologs counted 11 controls. &#226;&#8364;&#8220;, No COS marker matches found. <br />                                   At4g19690      At1g80830      At2g23150       At5g23990      At1g01580      At1g09240     At2g28160 <br />                                     AtIRT        AtNRAMP1       AtNRAMP3        AtFRO-Like      AtFRO2         AtNAS3         AtFRU <br />    LeIRT1/LeIRT2                   411              2              1               &#226;&#8364;&#8220;              &#226;&#8364;&#8220;             2              2 <br />    LeNRAMP1                         1              211             2               2              &#226;&#8364;&#8220;             &#226;&#8364;&#8220;              3 <br />    LeNRAMP3                         1               &#226;&#8364;&#8220;             311              &#226;&#8364;&#8220;              &#226;&#8364;&#8220;             1              2 <br />    LeFRO-TC124302                   4               &#226;&#8364;&#8220;              &#226;&#8364;&#8220;              311             2             4              &#226;&#8364;&#8220; <br />    LeFRO1/LeFRO-TC129233            1               1              &#226;&#8364;&#8220;               &#226;&#8364;&#8220;             311            1              &#226;&#8364;&#8220; <br />    LeNAS                            &#226;&#8364;&#8220;               1              1               1              &#226;&#8364;&#8220;            311             &#226;&#8364;&#8220; <br />    LeFER                            &#226;&#8364;&#8220;               &#226;&#8364;&#8220;              1               3              1             &#226;&#8364;&#8220;             111 <br />  <br />  <br />  <br />    To analyze significance findings,                    500 genes (52/26,404) genes randomly <br /> determined probability occurrence                         distributed. The significantly similar gene content of <br /> tomato-Arabidopsis homolog clusters random                    IRT/ZIP regions tomato Arabidopsis sug- <br /> 2-Mb intervals Arabidopsis genome. Similar                       gests common evolutionary origin. An equally <br /> results obtained 4-Mb (62 Mb) windows                          significant conservation clustered genes was <br /> (data shown). We calculated P-values                        LeFRO-TC124302/At5g47930/At5g49740/ <br /> occurrence clustering selected Arabidopsis                    At5g50160 (Table III). Significant clustering ob- <br /> gene homologs 61-Mb windows taking                       tained LeFRO1/LeFRO-TC129233/At1g01580/ <br /> account number neighbors recognizing tomato                      At1g01590 LeNAS/At1g09240 (Table III). <br /> homologs iron uptake region, number                       P-values 0.05 0.075 the <br /> homologous genes tomato region                       regions LeNRAMP1/At2g23150 LeFRO- <br /> entire Arabidopsis genome, number                        TC124302/At1g01580/At1g01590 (Table III). All other <br /> genes Arabidopsis genome (&#226;&#8364;&#732;&#226;&#8364;&#732;Materials                        comparisons iron uptake regions tomato <br /> Methods&#226;&#8364;&#8482;&#226;&#8364;&#8482;). Small P-values (P , 0.05) indicate                     Arabidopsis showed significant clustering, <br /> the number homologous Arabidopsis neighbors                          P-values higher 0.1 (Table III). However, <br /> recognized corresponding tomato region ex-                       speculate that, NRAMP, FRO, and <br /> ceeded significantly number neighbors accord-                    NAS gene regions, P-values decrease more <br /> ing random clustering 61-Mb window.                       COS markers available analysis these <br /> We 7 31 Arabidopsis iron uptake                       regions. <br /> genes analyzed (22.6%) P-values 0.05,                        In summary, Arabidopsis tomato genome re- <br /> and 43 279 noniron uptake genes analyzed                         gions homologous iron uptake genes gen- <br /> (15%) showed clustering P-values 0.05                    erally characterized presence multiple <br /> (Supplemental Table II). The cumulative distribution                    conserved genes. Since Arabidopsis genome fre- <br /> of P-values iron uptake genes indicated overall                quently contained region con- <br /> iron uptake genes significantly clustered                     served gene content studied tomato iron <br /> than noniron uptake genes (data shown). Three                       uptake regions, did appear sufficient to <br /> Arabidopsis homologs LeIRT1/LeIRT2 P-values                      determine functional homology based map posi- <br /> less 0.015, At1g31260, At5g45105,                       tion sequence alone. <br /> At4g19690/At4g19680 (Table III). Moreover, 9 11 <br /> COS homologs suspected located                   Functional Analysis Tomato-Arabidopsis <br /> regions (compare Supplemental Table I Fig. 3)                   Iron Uptake Genes <br /> P-values 0.02 (Table III). The 2 11 <br /> remaining markers P-values 0.06, just                         Within gene families, specific biological functions of <br /> slightly cutoff value 0.05                  gene family members conferred specific <br /> considered results statistically significant (Table               expression patterns. Iron uptake genes be <br /> III). Thus, LeIRT1 maps                        expressed root and/or induced iron de- <br /> tomato markers 610-cM region surrounding                       ficiency, indicating function iron mobilization of <br /> the LeIRT1 locus single 61-Mb window                        external internal iron. Here, investigated which <br /> Arabidopsis genome region containing At1g31260                          iron uptake homologs fulfilled expression <br /> together 419 genes. This means                        <span id='am-324' about='xsp:pattern' typeof='owl:Thing'>pattern</span> criteria tomato Arabidopsis. Gene <br /> Arabidopsis region homologs tomato markers                       expression surveyed analyzing EST expression <br /> of LeIRT1/LeIRT2 region highly enriched,                        data available http://www.tigr.org tomato <br /> which reflected average density                      genes (Supplemental Table III), experimental <br /> one markers 83 genes (5/419). We                         gene expression studies tomato Arabidopsis <br /> would expect markers                      (&#226;&#8364;&#732;&#226;&#8364;&#732;Materials Methods;&#226;&#8364;&#8482;&#226;&#8364;&#8482; Figs. 4 5). <br /> Plant Physiol. Vol. 136, 2004                                                                                                       4175 <br />  Bauer et al. <br />  <br />  <br />  <br />                Table III. P-values indicating occurrence tomato homolog clusters random 61-Mb <br />                intervals Arabidopsis genome <br />                  P-values calculated Fisher&#226;&#8364;&#8482;s exact test (&#226;&#8364;&#732;&#226;&#8364;&#732;Materials Methods&#226;&#8364;&#8482;&#226;&#8364;&#8482;). P-values 0.05 are <br />                considered significant nonrandom clustering. Tandemly repeated genes counted one. <br />                P-values iron uptake genes bold. <br />                       Tomato Region              Tomato Gene/Marker               Arabidopsis Gene            P-Value <br />                   LeIRT1/LeIRT2                 LeIRT1, LeIRT2                At4g19680/At4g19690           0.014 <br />                                                 T1668                         At4g20410                     0.016 <br />                                                 T1698                         At4g19880                     0.015 <br />                                                 CLEC27M9                      At4g16250                     0.607 <br />                                                                               At4g18130                     0.002 <br />                                                 CLPT1E8                       At4g18970                     0.014 <br />                                                 LeIRT1, LeIRT2                At1g31260                     0.001 <br />                                                 T1654                         At1g29520                     0.002 <br />                                                 CLPT2K20                      At1g31160                     0.001 <br />                                                 CLPT1E8                       At1g29660/At1g29670           0.002 <br />                                                                               At1g33811                     1.000 <br />                                                 T984                          At1g30580                     0.001 <br />                                                 LeIRT1, LeIRT2                At5g45105                     0.012 <br />                                                 T1105                         At5g43280                     0.062 <br />                                                 T1698                         At5g45020/At5g44990           0.012 <br />                                                                               At5g4400                      0.012 <br />                                                 CLPT1E8                       At5g45670                     0.064 <br />                   LeNRAMP1                      LeNRAMP1                      At1g80830                     1.000 <br />                                                 T1161                         At1g77470                     0.353 <br />                                                 T675                          At1g77670                     0.353 <br />                                                 LeNRAMP1                      At5g67330                     0.229 <br />                                                 T1071                         At5g65760                     0.328 <br />                                                 LeNRAMP1                      At2g23150                     0.334 <br />                                                 T1071                         At2g24280                     0.320 <br />                                                 T1787                         At2g27920                     0.056 <br />                                                 CLET10O11                     At2g27290                     0.055 <br />                                                 LeNRAMP1                      At4g18790                     0.337 <br />                                                 T1014                         At4g21110                     0.342 <br />                                                 LeNRAMP1                      At1g15960                     0.377 <br />                                                 T1787                         At1g1500                      0.377 <br />                   LeNRAMP3                      LeNRAMP3                      At2g23150                     0.280 <br />                                                 T1480                         At2g25140                     0.073 <br />                                                 CTOB9L18                      At2g23780                     0.262 <br />                                                 T1744                         At2g26560                     0.626 <br />                                                 LeNRAMP3                      At5g67330                     0.144 <br />                                                 T634                          At5g67370                     0.140 <br />                                                 CLER17N11                     At5g65700                     0.276 <br />                                                 LeNRAMP3                      At4g18790                     0.655 <br />                                                 T1003                         At4g18370                     0.659 <br />                   LeFRO-TC124302                LeFRO-TC124302                At5g23990/At5g23980           0.072 <br />                                                 T1540                         At5g23940                     0.074 <br />                                                 T1424                         At5g24460                     0.069 <br />                                                 T1429                         At5g23670                     0.075 <br />                                                 LeFRO-TC124302                At1g01580/At1g01590           0.526 <br />                                                 CLEC37C6                      At1g01780                     0.542 <br />                                                 T1659                         At1g05350                     1.000 <br />                                                 LeFRO-TC124302                At5g49730/At5g49740           0.005 <br />                                                                               At5g50160                     0.001 <br />                                                 T1119                         At5g53180                     0.030 <br />                                                 T1321                         At5g52050                     0.000 <br />                                                 T1155                         At5g51150                     0.000 <br />                                                 T648                          At5g51540                     0.000 <br />                                                 T753                          At5g52390                     0.001 <br />                                                 T1278                         At5g50320                     0.001 <br />                   Lefro1/Lefro-129233           Lefro1/Lefro-129233           At1g01580/At1g01590           0.018 <br />                                                 CLES5J1                       At1g01820                     0.021 <br />                                                 T1409                         At1g01610                     0.019 <br />                                                                                   (Table continues following page.) <br />  <br /> 4176                                                                                                         Plant Physiol. Vol. 136, 2004 <br />                                                                                               Functional Homology Iron Uptake Genes <br />  <br />  <br />  <br />                     Table III. (Continued previous page.) <br />                        Tomato Region               Tomato Gene/Marker           Arabidopsis Gene          P-Value <br />                                                  T1327                          At1g01300                  0.015 <br />                                                  Lefro1/Lefro-129233            At1g23020                  1.000 <br />                                                  T1173                          At1g20200                  1.000 <br />                        LeNAS                     LeNAS                          At1g09240                  0.015 <br />                                                  T1118                          At1g09830                  0.069 <br />                                                  T1687                          At1g09430                  0.016 <br />                                                                                 At1g10670                  0.065 <br />                                                  CLET4E21                       At1g06680                  0.254 <br />                                                  LeNAS                          At1g56430                  0.190 <br />                                                  T1176                          At1g56700                  0.183 <br />                                                  T1485                          At1g54780                  0.226 <br />                                                  T1687                          At1g60810                  1.000 <br />                                                  LeNAS                          At5g04950                  0.262 <br />                                                  T141                           At5g05010                  0.261 <br />                                                  T852                           At5g07020                  0.260 <br />                                                  LeNAS                          At5g56080                  0.629 <br />                                                  T1084                          At5g27190                  0.643 <br />                                                  CTOC14I18                      At5g60910                  1.000 <br />  <br />  <br />    First, analyzed expression IRT/ZIP genes (Fig.                 expressed roots expression induced upon <br /> 4A). LeIRT1 LeIRT2 previously shown                      iron deficiency, expected (Eide et al., 1996; Vert et al., <br /> expressed root, LeIRT1 iron reg-                     2001). AtIRT2 appeared induced higher level <br /> ulated (Bereczky et al., 2003). LeIRT2 EST sequences                    AtIRT1. The AtZIP8 expression pattern was <br /> were available EST database, data                    similar AtIRT2. AtZIP10 transcripts were <br /> available LeIRT1. LeIRT2 EST sequences                         detected roots sufficient iron supply. AtZIP7 <br /> all derived root tissue (data shown). In                       mainly expressed leaves induced low <br /> Arabidopsis (Fig. 4A), AtIRT1 AtIRT2 mainly                    iron supply. <br />                                                                            Expression NRAMP genes ana- <br />                                                                         lyzed here. LeNRAMP1 AtNRAMP1 pre- <br />                                                                         viously root specific iron regulated <br />                                                                         (Curie et al., 2000; Bereczky et al., 2003). LeNRAMP3, <br />                                                                         AtNRAMP3, AtNRAMP4 iron regulated and <br />                                                                         expressed leaf (Curie et al., 2000; Thomine <br />                                                                         et al., 2000; Bereczky et al., 2003). EST sequences were <br />                                                                         available LeNRAMP3 derived from <br />                                                                         different tissues (data shown). <br />                                                                            EST sequences LeFRO-TC124302 in <br />                                                                         roots callus (data shown). Experimental ex- <br />                                                                         pression analysis showed expression LeFRO- <br />                                                                         TC124302 root specific, slightly <br />                                                                         iron regulated dependent functional LeFER <br />                                                                         gene tomato (Fig. 5). On hand, transcripts <br />                                                                         LeFRO-TC129233 detected experimentally <br />                                                                         leaves roots (data shown). All EST sequences <br />                                                                         derived flower libraries, indicating that <br />                                                                         LeFRO-TC129233 involved root iron uptake. <br />                                                                         As analyzed previously, LeFRO1 expressed in <br />                                                                         roots leaves, up-regulated iron deficiency, and <br />                                                                         dependent functional LeFER gene (Li et al., 2004). <br />                                                                         Since LeFRO-TC124302 appeared specific the <br />                                                                         L. esculentum genome, tested LeFRO- <br />                                                                         TC124302 essential iron reduction L. escu- <br /> Figure 4. Expression analysis Arabidopsis iron uptake homologs in <br />                                                                         lentum M82 introgression lines containing the <br /> roots leaves plants grown low (2Fe) sufficient iron     expected LeFRO-TC124302 locus introgressed <br /> supply (1Fe). A, IRT/ZIP genes; B, FRO genes; C, NAS genes; D, FRU      L. pennellii chromosomal fragment devoid LeFRO- <br /> gene. Expression analysis performed semiquantitative RT-PCR      TC124302 (Eshed et al., 1992). We these <br /> and Southern-blot analysis. Amplification elongation factor cDNA     introgression lines similar levels iron reductase <br /> EF1b-a (At5g19510) served constitutive control.                      activity control line M82 grew very <br /> Plant Physiol. Vol. 136, 2004                                                                                                   4177 <br />  Bauer et al. <br />  <br />  <br />                                                                           induced iron deficiency (Fig. 4D). The expres- <br />                                                                           sion pattern AtFRU suggests gene <br />                                                                           involved iron deficiency responses in <br />                                                                           Arabidopsis. AtBHLH021 expressed flow- <br />                                                                           ering tissues (data shown). Spliced transcripts for <br />                                                                           BHLH genes AtBHLH022, AtBHLH090, AtBHLH035, <br />                                                                           AtBHLH027 detected (data not <br />                                                                           shown). <br /> Figure 5. Expression analysis tomato LeFRO-TC124302 roots and <br />                                                                              Overall, expression data contributed assigning <br /> leaves wild-type fer mutant tomato plants, grown sufficient <br /> iron supply iron deficiency. Expression analysis performed by <br />                                                                           homologous functions. In general, gene <br /> semiquantitative RT-PCR Southern-blot analysis. Amplification      family members tomato differed signifi- <br /> elongation factor cDNA LeEF1a served constitutive control. No          cantly expression patterns did ho- <br /> expression detected LeFRO-TC129233 (data shown).              mologous Arabidopsis gene family members, which <br />                                                                           tended retain similar expression patterns. <br /> similar way (data shown). LeFRO-TC124302 was <br /> therefore essential iron uptake iron re- <br /> duction L. esculentum, indicating presumably                  DISCUSSION <br /> function performed redundant LeFRO1 <br /> gene. Arabidopsis FRO-like genes expressed                          In study, assigned functionally struc- <br /> in roots leaves tended induced                         turally <span id='am-319' about='Thesaurus:Homologous_Gene' typeof='owl:Thing'>homologous gene</span> functions involved iron <br /> iron deficiency (Fig. 4B). AtFRO1, AtFRO2,                            uptake tomato Arabidopsis. For this <br /> At5g23990 shared similar expression pattern. Ex-                        purpose, based studies sequence <br /> pression genes induced iron deficiency                    comparisons took account map position <br /> in roots leaves. Upon sufficient iron supply,                         functional expression data. Analyzing, addi- <br /> AtFRO1, AtFRO2, At5g23990 expressed                           tion, levels conserved gene content adjacent the <br /> a higher level roots leaves. At5g23980                     genes allowed predict gene <br /> AtFRO3 expressed leaves sufficient                      family members redundant genes resulting from <br /> iron supply. FRO3 expressed roots                       internal genome duplication events nonredundant <br /> sufficient iron. Upon low iron supply, genes                      single-acting genes distinct biological functions. <br /> were induced roots. FRO3 induced                           We easier determine functional <br /> leaves iron deficiency, expression                        homologs tomato using Arabidopsis informa- <br /> At5g23980 similar level leaves                      tion vice versa. Due lower gene complexity <br /> sufficient iron supply. Thus, Arabidopsis FRO                     tomato, hypothesize generally <br /> genes, expression patterns suggest function                     convenient utilize Arabidopsis genome information <br /> in iron deficiency responses.                                             predict tomato homologs. <br />    We expression Arabidopsis NAS genes in <br /> leaves roots sufficient low iron supply                      Gene Homology Map Position Reflect Genome <br /> (Fig. 4C), supporting tomato data NAS activity                   Evolution Tomato Arabidopsis <br /> was required constantly (Ling et al., 1999). <br /> However, observed AtNAS1 AtNAS2                             Most comparative genome programs based on <br /> expressed roots stronger leaves. Upon low                      finding orthologs using criterion sequence <br /> iron supply, AtNAS1 AtNAS2 induced                        similarity (e.g. http://www.tigr.org/tdb/tgi/lgi/ <br /> root. AtNAS4 induced iron deficiency roots                      GO.html). In study, sequence similarity alone <br /> and leaves. At sufficient iron supply, AtNAS4                         sufficient criterion determining func- <br /> mainly expressed leaves. AtNAS3 mainly ex-                         tional homologs iron uptake Arabidopsis <br /> pressed leaf sufficient iron supply. Taking                   tomato. For given gene, frequently identified <br /> our expression data indicates Arabidop-                     related gene species. The <br /> sis NAS genes iron regulated. We identified Ara-                      Arabidopsis genome particular higher num- <br /> bidopsis lines T-DNA insertions coding                        ber homologs compared tomato. Extrapolating <br /> regions AtNAS1, AtNAS2, AtNAS4 (data                           data suggest Arabidopsis may <br /> shown). None lines segregated dis-                       larger gene families tomato. <br /> cernible phenotypes, suggesting NAS genes                     Taking number related FER-like, NAS, <br /> least partially redundant.                                                IRT, NRAMP, FRO genes indicate <br />    LeFER expressed root leaves                   Arabidopsis 21 genes (114151615), <br /> and cotyledons (Ling et al., 2002). The tomato BHLH                       tomato 10 genes functions <br /> genes weak sequence similarity tomato                         (111121313). Van der Hoeven et al. (2002) calculated <br /> LeFER gene did expressed roots                    50% tomato genes uncovered from <br /> all EST data derived flower tissue callus                    EST analysis. In study, 7 10 tomato genes <br /> (data shown). AtFRU mainly expressed                           uncovered EST analysis. Correcting our <br /> roots hardly leaves, root expression                       numbers estimated 14 tomato genes versus <br /> 4178                                                                                                    Plant Physiol. Vol. 136, 2004 <br />                                                                               Functional Homology Iron Uptake Genes <br />  <br />  <br /> 21 Arabidopsis genes suggest              LeFRO1 LeFRO-TC129233 present in <br /> Arabidopsis 33% genes tomato           tandem duplication tomato genome. <br /> for families analyzed. This result supported by <br /> the similar findings Van der Hoeven et al. (2002)    Importance Functional Expression Data for <br /> the comparison multigene family copy numbers            Assignment Functional Homology and <br /> between tomato EST data Arabidopsis genome             Redundancy Iron Uptake <br /> data. <br />    The comparative mapping studies presented             This study dealt mainly essential gene func- <br /> showed 20% 50% analyzed COS markers             tions required root and/or iron regulation. <br /> were located vicinity corresponding iron         We expression data greatly contributed to <br /> uptake genes tomato Arabidopsis (except      assigning appropriate unique <span id='am-320' about='Thesaurus:Homologous_Gene' typeof='owl:Thing'>homologous gene</span> <br /> those LeFER region). In contrast, 0% 14%    pairs. For IRT FRO sequences, indispens- <br /> of COS markers matched Arabidopsis              able consider sequence, map position, gene <br /> regions containing nonhomologous iron uptake genes.        expression aspects together. For example, the <br /> The clustering conserved COS marker positions        15 Arabidopsis ZIP genes (Ma&#194;&#168;ser et al., 2001), AtIRT1 <br /> between tomato Arabidopsis calculated        AtIRT2 LeIRT1 LeIRT2 most <br /> significant them,          similar pairs, respectively, genome. AtIRT1 and <br /> excluded clusters occurred random. For         LeIRT1 likely encoding essential iron- <br /> some compared Arabidopsis-tomato regions            regulated iron transporters species (Varotto <br /> with P-values greater 0.05, speculate      et al., 2002; Vert et al., 2002; Bereczky et al., 2003). <br /> increase number analyzed markers           AtIRT2 iron regulated essential (Varotto et al., <br /> decrease P-values make clustering statis-      2002). In contrast, LeIRT2 iron regulated nor <br /> tically significant. We avoid utilizing term colin-    dependent regulator FER likely <br /> earity observations actual gene        involved iron mobilization (Eckhardt et al., 2001; <br /> order conserved cases. Similar gene         Bereczky et al., 2003). In addition, LeIRT1 LeIRT2 <br /> content suggests common evolutionary origin       closely related sequence, suggesting that <br /> corresponding Arabidopsis tomato genome re-            genes recent tandem duplications. In <br /> gions. The analysis showed internally duplicated      contrast, AtIRT1 AtIRT2, located <br /> regions Arabidopsis genome. It likely    tandem, diverged sequence if <br /> the reason higher multigene family copy numbers        tandem duplication event older than <br /> in Arabidopsis genome structure. Some du-          tomato. We AtZIP8 shared similar <br /> plications uncovered pre-          expression pattern AtIRT2. AtZIP8 most <br /> vious studies internally duplicated regions      related sequence AtIRT1 AtIRT2. Hence, it <br /> Arabidopsis genome; example, duplication           possibility AtIRT2 function overlaps with <br /> events involving chromosome 1, chromosome 4,           redundant function provided AtZIP8. It seems, <br /> chromosome 5 containing IRT/ZIP gene regions            however, unlikely that, despite tandem location <br /> well chromosome 1 duplication involving NAS         IRT1, AtIRT2 LeIRT2 functioning as <br /> genes (Blanc et al., 2003). We detected signs      orthologs. Similarly, study suggests LeIRT2 <br /> duplications previously detected         AtZIP10 share common function. <br /> and reflect hidden duplications (Simillion et al.,      The drastic example functional gene <br /> 2002), chromosome 5 duplication involving      analysis data needed investigating homologous <br /> NAS genes. Due similar gene content sequence        gene functions provided FRO genes. <br /> similarity, predict NAS genes reveal older      Among Arabidopsis FRO-like genes, AtFRO2 <br /> duplication event responsible duplication       unequivocally identified FRO gene in <br /> NAS regions chromosomes 1 5       Arabidopsis genetic experiments (Robinson <br /> two recent duplication events involving chro-         et al., 1999). From expression studies, suggest that <br /> mosomes 1 5. The tomato genome presum-             Arabidopsis FRO gene homologs could <br /> ably partially duplicated, lower     share additional functions iron mobilization. In <br /> degree. LeIRT1/LeIRT2 appear recent duplicates          tomato, LeFRO1 LeFRO-TC124302 be <br /> based high sequence similarity. Both LeFRO1/Le-         root FRO genes acting iron mobilization. Both <br /> FRO-TC129233 LeFRO-TC124302 regions showed             genes iron regulated roots dependent on <br /> levels gene conservation AtFRO2 region      FER (see Li et al., 2004) The analysis introgres- <br /> well region containing related FRO        sion L. esculentum lines devoid LeFRO-TC124302 <br /> genes At5g50160/At5g49730/At5g49740. The            suggested LeFRO-TC124302 essential and <br /> three Arabidopsis genes related tomato FRO         redundant iron reductase gene exist the <br /> genes terms gene content despite          Lycopersicon genome. This redundant function could <br /> distantly related sequence. These observations          provided LeFRO1. In contrast, LeFRO- <br /> might suggest genomic region           TC129233 similar sequence AtFRO2 <br /> also duplicated tomato duplication          genes located regions conserved <br /> occurred early dicot evolution. We hypothesize          gene content. However, LeFRO-TC129233 not <br /> Plant Physiol. Vol. 136, 2004                                                                                    4179 <br />  Bauer et al. <br />  <br />  <br /> found expressed roots leaves, iron                    genes did apparent conservation of <br /> deficiency, suggesting LeFRO-TC129233                          gene clusters. LeFER AtFRU expressed <br /> involved putative iron reductase different bi-                     roots, AtFRU expressed low <br /> ological process LeFRO1 LeFRO-TC124302.                           level leaves. AtFRU induced iron <br />    The chloronerva (nas) tomato mutant characterized                    deficiency. Recent genetic experiments suggest that <br /> by distinct leaf chlorosis root <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>                       LeFER AtFRU serve conserved biological <br /> low iron supply (Scholz et al., 1992). A similar mutant                    function iron regulation (Jakoby et al., 2004). <br /> has described Arabidopsis. Our analysis                     Perhaps genes encoding regulatory components <br /> the NAS genes Arabidopsis indicated                           lower evolutionary constraints sequence <br /> NAS genes duplicated twice. Interest-                        conservation genes encoding transporters and <br /> ingly, despite strong sequence conservation,                       enzymes. <br /> found differential regulation NAS genes iron                            Arabidopsis tomato similar physiological <br /> deficiency responses, NAS                        responses iron deficiency (strategy I), most <br /> genes up-regulated low iron supply. We                             likely iron regulation iron uptake conserved <br /> established previously NAS required                           processes species involve conserved <br /> induction LeFER-mediated LeIRT1 LeNRAMP1                            gene functions. If gene functions not <br /> induction iron deficiency (Bereczky et al., 2003).                    performed orthologs, expected <br /> In accordance observations, iron regulation                     compensated paralogs. We predict our <br /> of NAS genes Arabidopsis suggest                             analysis essential <span id='am-321' about='Thesaurus:Homologous_Gene' typeof='owl:Thing'>homologous gene</span> functions of <br /> nicotianamine function iron mobiliza-                      iron uptake involved conserved be- <br /> tion signaling. Due high sequence conserva-                      tween Arabidopsis tomato. However, detailed <br /> tion, NAS genes form partially redundant gene                            sequence map position analysis indicated that <br /> family. Absence single functional genes did                         conserved genes likely paralogous <br /> show obvious phenotypes. However, up-regulation                         origin orthologous origin. Despite the <br /> individual gene family members observed                            similarities genes proteins involved iron <br /> in metallophyte zinc hyperaccumulator                              mobilization, tomato induces root morphological al- <br /> Arabidopsis halleri lead discrete dominant                          terations, root hair proliferation, transfer cell <br /> phenotypes&#226;&#8364;&#8221;for instance A. halleri zinc tolerance                       development response iron deficiency <br /> (Becher et al., 2004).                                                     (Schmidt, 1999; Schikora Schmidt, 2001, 2002). <br />    The Arabidopsis genome contains 162 predicted                           These morphological alterations supposed aid <br /> BHLH genes (Bailey et al., 2003). Sequence compar-                         iron uptake. In Arabidopsis, morphological alterations <br /> isons outside conserved bHLH                                 iron deficiency pronounced, is <br /> domain, AtFRU protein quite distinct                           possible Arabidopsis relies instead dupli- <br /> all bHLH proteins. The genome regions                         cated gene functions. <br />  <br />  <br /> Table IV. Oligonucleotide primers used RT-PCR experiments <br />        Gene Name                Number                         5# Primer 5#&#226;&#8364;&#8220;3#                            3# Primer 5#&#226;&#8364;&#8220;3# <br />    LeFRO-TC129233             TC129233             atgggggttatgggtgcatcagag                   tctatgcctgcattatgcttcctggt <br />    LeFRO-TC124302             TC124302             tgtgaacgtgccaagtgtatcca                    ggcccacaaacaacaactcca <br />    LeEF-1a                    TC123773             cctcttgggctcgttaatctggct                   ctggtggttttgaagctggtatct <br />    AtZIP7                     At2g04032            gtgcgtatgcgaggagaact                       atcaaagtgaaggacagaagtaaaga <br />    AtZIP8                     At5g45105            atgcgagaccgattcaacag                       tttcataaaagtcgagaggataatgt <br />    AtZIP10                    At1g31260            tacagcttctgcggtatcgtat                     catctatttgtaagctccgctct <br />    AtIRT1                     At4g19690            gcatgggtcttggcggttgt                       atccacatgatttcaatcccgcaat <br />    AtIRT2                     At4g19680            ggtaagaactcagtcggaccagt                    gtcgccttgaataataaatcaaataa <br />    AtFRO-like                 At5g23990            gataaggactccaagaagcaggta                   caaacatatatagttagaacatggaataga <br />    AtFRO-like                 At5g23980            ggataaggagttcaagaatcaggta                  aacacacatagttagaacatggaataca <br />    AtFRO1                     At1g01590            cgacaacttatctccggtgatt                     ttgtaacccaaacatctatgataaaa <br />    AtFRO2                     At1g01580            tctccaacatcttctcctacctcatcat               caacacatagtgaaaacagagttatatacgcaa <br />    AtNAS3                     At1g09240            caactccgtggtgatctctaaa                     atagagaattaggaaacaagaaacg <br />    AtNAS4                     At1g56430            aggtgaagatgctaatggtgtt                     acacagcatttttctaggtaagtt <br />    AtNAS1                     At5g04950            ggggttaatggtactcgtgg                       agacatgaaatgaaaagagcagtt <br />    AtNAS2                     At5g56080            gccagatcggacggtgt                          ctcgatcaaattcttctccatac <br />    AtFRU BHLH029              At2g28160            atggaaggaagagtcaacgc                       tcaatatagtgcagaaccgg <br />    AtBHLH 090                 At1g10610            atgatgatgatgagaggtggtgagagagtg             ttacgttactaccttgacattgagaacggtcat <br />    AtBHLH 035                 At5g57150            atggaggatatcgtcgaccaagaattaagc             ttagagagacaagagagatagagagagaaagatgctga <br />    AtBHLH 027                 At4g29930            atggaagatctcgaccatgagtacaagaattac          tcaaaccaaaacaagacacgtacagtattctct <br />    AtBHLH 022                 At4g21330            atgggtggaggaagcagatttcaag                  ttatggattgcttctcataacttccaaaaga <br />    AtBHLH 021                 At2g16910            atgagcctgaacgggacagtggtc                   ttattggttgtggtaatggttgatgtgttgg <br />    AtEF1b-a                   At5g19510            aggagagggaggctgctaag                       aatcttgttgaaagcgacaatg <br />  <br /> 4180                                                                                                     Plant Physiol. Vol. 136, 2004 <br />                                                                                                            Functional Homology Iron Uptake Genes <br />  <br />  <br /> MATERIALS AND METHODS                                                            total, 26,405 Arabidopsis genes, including At1g31260 (M 5 26,404). <br />                                                                                  Out 26,404 genes, 51 homologs COS markers the <br /> Sequence Analysis Database Searches                                          610-cM region LeIRT1 (n 5 51). The density 5 genes 419 is <br />                                                                                  6-fold higher density 51 genes 26,404, P-value <br />     Amino acid sequences AtFRO2 (Robinson et al., 1999), AtIRT1 (Eide         0.001. <br /> et al., 1996), LeIRT1 (Eckhardt et al., 2001), AtNRAMP1 (Curie et al., 2000), <br /> AtNRAMP3 (Thomine et al., 2000), LeNRAMP1 LeNRAMP3 (Bereczky <br /> et al., 2003), LeNAS (CHLORONERVA; Ling et al., 1999), LeFER (Ling <br /> et al., 2002) used search homologous sequences tomato               Plant Growth Plant Material <br /> (Lycopersicon esculentum) Arabidopsis (Arabidopsis thaliana) BLASTP <br /> and BLASTX. Arabidopsis amino acid sequence homologs identified                 Tomato plants used RNA expression analysis derived the <br /> using BLAST program (http://www.Arabidopsis.org/Blast) re-               lines L. esculentum T3238fer (fer mutant phenotype) L. esculentum Money- <br /> trieved http://mips.gsf.de/proj/thal/db/search/search_frame.html. To-         maker (wild type). Homozygous plants first-generation introgression <br /> mato amino acid sequences identified retrieved using BLAST          lines L. pennellii (LA 716) genetic background processing <br /> program (http://tigrblast.tigr.org/tgi). Sequences retained analysis    tomato variety M82 propagated utilized study (Eshed et al., <br /> if significantly related comparable E-values. Sequences        1992). <br /> with drastically increased E-values taken account              For tomato iron uptake studies, 12-d-old plants grown hydro- <br /> (exception FRO). A general E-value cutoff applied different        ponic Hoagland medium containing 0.1 (low iron) 10 mM (sufficient iron) <br /> protein classes showed different E-values. Multiple amino acid sequences         FeNaEDTA 1 week according Bereczky et al. (2003). Iron reductase <br /> were aligned using ClustalX N-J Trees generated http://www-          assays performed roots described Stephan Prochazka <br /> igbmc.u-strasbg.fr/BioInfo/ClustalX/Top.html. N-J Trees represented         (1989). <br /> by NJPLOT (Perrie`re Gouy, 1996). Functional expression data                For expression studies Arabidopsis, 2-week-old Arabidopsis Columbia <br /> available http://www.tigr.org tomato genes.                               plants grown solid Hoagland medium presence 10 mM FeNaEDTA <br />                                                                                  transferred Hoagland medium containing 10 mM FeNaEDTA (suffi- <br />                                                                                  cient iron) iron 200 mM bathophenanthroline disulfonic acid (low <br />                                                                                  iron) 5 d. <br /> Mapping Tomato Genes Analysis                                                 nas1, nas2, nas4 T-DNA insertion lines identified database <br />                                                                                  searches SALK Institute Web site (http://signal.salk.edu) ordered <br /> of Conserved Gene Regions <br />                                                                                  Arabidopsis Biological Resource Center. <br />     Genomic tomato DNA fragments mapped restriction fragment <br /> length polymorphism analysis using 43 F2 individuals L. esculentum/ <br /> Lycopersicon pennellii mapping population according Tanksley et al. (1992). <br />                                                                                  Gene Expression Analysis <br /> Genetic maps generated using MAPMAKER program (Lander et al., <br /> 1987).                                                                               Semiquantitative reverse transcription (RT)-PCR analysis performed <br />     Tomato COS markers mapped distance 610 cM iron           according Bereczky et al. (2003). Total RNA extracted tomato <br /> uptake genes selected analysis conserved gene regions http://     using Purescript RNA Isolation kit (Gentra Systems, Minneapolis) and <br /> www.sgn.cornell.edu/maps/tomato_Arabidopsis/synteny_map.html. The                Arabidopsis using Invisorb Spin Plant RNA Mini kit (Invitek, Berlin). <br /> encoded amino acid sequences selected COS markers used            From 0.1 2 mg RNA reverse transcribed cDNA using oligo(dT) <br /> BLAST corresponding Arabidopsis amino acid sequences http://              primer RevertAid First Strand cDNA synthesis kit (Fermentas GmbH, <br /> www.Arabidopsis.org/Blast. Arabidopsis genes used COS markers            St. Leon-Rot, Germany). cDNA amplified using specific primers accord- <br /> if encoded peptides gave expected values e-20 alignments     ing standard procedures (Table IV). The number amplification cycles was <br /> encoded peptides tomato COS markers (Supplemental Table I). Arabidopsis       determined experimentally reaction analyzed exponen- <br /> COS markers located 400 genes iron uptake genes (approximately        tial phase. PCR fragments separated agarose gel electrophoresis, <br /> 62 Mb) considered chromosomal region. For example,       blotted, hybridized according standard procedures. Amplification of <br /> genes region At4g19690 expected gene locus            elongation factor cDNA used constitutive control normalize all <br /> numbers At4g15690 At4g23690.                                         samples. <br />     Statistical analysis calculation P-values random clustering of <br /> homologous genes Arabidopsis tomato performed as <br /> follows: For Arabidopsis genes interest, interval 61 Mb <br /> was considered gene center. If K     ACKNOWLEDGMENTS <br /> total number Arabidopsis genes 61-Mb interval, L number of <br /> Arabidopsis genes located interval homologous genes         Tomato mapping filters provided Dr. M. Ganal. Help the <br /> corresponding 610-cM tomato region minus tandem duplicates, M            MAPMAKER program Dr. X. Huang Dr. M. Ro&#194;&#168;der greatly <br /> total number genes Arabidopsis genome, N number         acknowledged. <br /> genes Arabidopsis genome homologous genes 610-cM              Received June 2, 2004; returned revision July 7, 2004; accepted July 7, 2004. <br /> tomato region minus tandem duplicates, probability pL finding <br /> L genes chance given K, M, N computed Fisher&#226;&#8364;&#8482;s exact test as <br /> the right tail hypergeometric distribution: <br />                                                        <br />                                                                                  LITERATURE CITED <br />                                          K     M2K <br />                                              N2i                               Arabidopsis Genome Initiative (2000) Analysis genome sequence of <br />                            pL 5 +                        :                          flowering plant Arabidopsis thaliana. Nature 408: 796&#226;&#8364;&#8220;815 <br />                                i$L            M <br />                                               N                                  Bailey PC, Martin C, Toledo-Ortiz G, Quail PH, Huq E, Heim MA, Jakoby <br />                                                                                     M, Weisshaar B (2003) Update basic helix-loop-helix transcrip- <br />    When determining K, L, M, N, gene center 61-Mb             tion factor gene family Arabidopsis thaliana. Plant Cell 15: 2497&#226;&#8364;&#8220;2501 <br /> interval counted. The statistical significance computed       Bauer P, Bereczky Z (2003) Gene networks involved iron acquisition <br /> probability L L genes occur chance 61-Mb          strategies plants. Agronomie 23: 1&#226;&#8364;&#8220;8 <br /> region numbers K, M, N fixed. The P-values assigned      Becher M, Talke IN, Krall L, Kramer U (2004) Cross-species microarray <br /> the genes center 61-Mb intervals, P-values 0.05         transcript profiling reveals high constitutive expression metal ho- <br /> were considered significant nonrandom clustering. For example,                  meostasis genes shoots zinc hyperaccumulator Arabidopsis <br /> At1g31260 maps position 11,175,540 bp Arabidopsis chromosome 1.               halleri. Plant J 37: 251&#226;&#8364;&#8220;268 <br /> The 61-Mb interval centered position 11,175,540 contains 420 genes,           Bereczky Z, Wang HY, Schubert V, Ganal M, Bauer P (2003) Differential <br /> including At1g31260 (K 5 419). Out 419 genes, homologs         regulation nramp irt metal transporter genes wild type iron <br /> COS markers 610-cM tomato region containing LeIRT1 (L 5 5). In             uptake mutants tomato. J Biol Chem 278: 24697&#226;&#8364;&#8220;24704 <br />  <br /> Plant Physiol. Vol. 136, 2004                                                                                                                               4181 <br />  Bauer et al. <br />  <br />  <br /> Blanc G, Barakat A, Guyot R, Cooke R, Delseny M (2000) Extensive                  constructing primary genetic linkage maps experimental natural <br />    duplication reshuffling Arabidopsis genome. Plant Cell 12:          populations. Genomics 1: 174&#226;&#8364;&#8220;181 <br />    1093&#226;&#8364;&#8220;1101                                                                   Li L, Cheng X, Ling HQ (2004) Isolation characterization of <br /> Blanc G, Hokamp K, Wolfe KH (2003) A recent polyploidy superimposed               Fe(III)-chelate reductase gene LeFRO1 tomato. Plant Mol Biol 54: <br />    older large-scale duplications Arabidopsis genome. Genome Res        125&#226;&#8364;&#8220;136 <br />    13: 137&#226;&#8364;&#8220;144                                                                 Ling HQ, Bauer P, Bereczky Z, Keller B, Ganal M (2002) The tomato fer <br /> Bowers JE, Chapman BA, Rong J, Paterson AH (2003) Unravelling                     gene encoding bHLH protein controls iron- uptake responses roots. <br />    angiosperm genome evolution phylogenetic analysis chromo-                Proc Natl Acad Sci USA 99: 13938&#226;&#8364;&#8220;13943 <br />    somal duplication events. Nature 422: 383&#226;&#8364;&#8220;384                               Ling HQ, Koch G, Baumlein H, Ganal MW (1999) Map-based cloning of <br /> Curie C, Alonso JM, Le Jean M, Ecker JR, Briat JF (2000) Involvement              chloronerva, gene involved iron uptake higher plants encoding <br />    NRAMP1 Arabidopsis thaliana iron transport. Biochem J 347:          nicotianamine synthase. Proc Natl Acad Sci USA 96: 7098&#226;&#8364;&#8220;7103 <br />    749&#226;&#8364;&#8220;755                                                                     Ling HQ, Pich A, Scholz G, Ganal MW (1996) Genetic analysis two <br /> Curie C, Briat J-F (2003) Iron transport signaling plants. Annu Rev        tomato mutants affected regulation iron metabolism. Mol Gen <br />    Plant Biol 54: 183&#226;&#8364;&#8220;206                                                         Genet 252: 87&#226;&#8364;&#8220;92 <br /> Eckhardt U, Marques AM, Buckhout TJ (2001) Two iron-regulated cation           Liu XF, Culotta VC (1999) Mutational analysis Saccharomyces cerevisiae <br />    transporters tomato complement metal uptake-deficient yeast               Smf1p, member Nramp family metal transporters. J Mol Biol <br />    mutants. Plant Mol Biol 45: 437&#226;&#8364;&#8220;448                                            289: 885&#226;&#8364;&#8220;891 <br /> Eide D, Broderius M, Fett J, Guerinot ML (1996) A novel iron-regulated         Mao L, Begum D, Goff SA, Wing RA (2001) Sequence analysis the <br />    metal transporter plants identified functional expression           tomato JOINTLESS locus. Plant Physiol 126: 1331&#226;&#8364;&#8220;1340 <br />    yeast. Proc Natl Acad Sci USA 93: 5624&#226;&#8364;&#8220;5628                                 Ma&#194;&#168;ser P, Thomine S, Schroeder JI, Ward JM, Hirschi K, Sze H, Talke IN, <br /> Eng BH, Guerinot ML, Eide D, Saier MH (1998) Sequence analyses                Amtmann A, Maathuis FJ, Sanders D, et al (2001) Phylogenetic <br />    phylogenetic characterization ZIP family metal ion transport         relationships cation transporter families Arabidopsis. Plant <br />    proteins. J Membr Biol 166: 1&#226;&#8364;&#8220;7                                                Physiol 126: 1646&#226;&#8364;&#8220;1667 <br /> Ermolaeva MD, Wu M, Eisen JA, Salzberg SL (2003) The age                Mushegian A (1999) The minimal genome concept. Curr Opin Genet Dev 9: <br />    Arabidopsis thaliana genome duplication. Plant Mol Biol 51: 859&#226;&#8364;&#8220;866            709&#226;&#8364;&#8220;714 <br /> Eshed Y, Abu-Abied M, Saranga Y, Zamir D (1992) Lycopersicon esculentum        Oh K, Hardeman K, Ivanchenko MG, Ellard-Ivey M, Nebenfuhr A, White <br />    lines containing small overlapping introgressions L. pennellii.           TJ, Lomax TL (2002) Fine mapping tomato using microsynteny with <br />    Theor Appl Genet 83: 1027&#226;&#8364;&#8220;1034                                                 Arabidopsis genome: Diageotropica (Dgt) locus. Genome Biol 3: <br /> Fleming MD, Trenor CC, Su MA, Foernzler D, Beier DR, Dietrich WF,                 0049.0041 <br />                                                                                Perrie`re G, Gouy M (1996) WWW-query: on-line retrieval for <br />    Andrews NC (1997) Microcytic anaemia mice <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span>mutation</span> in <br />                                                                                   biological sequence banks. Biochimie 78: 364&#226;&#8364;&#8220;369 <br />    Nramp2, candidate iron transporter gene. Nat Genet 16: 383&#226;&#8364;&#8220;386 <br />                                                                                Robinson NJ, Procter CM, Connolly EL, Guerinot ML (1999) A ferric- <br /> Fulton TM, Van der Hoeven R, Eannetta NT, Tanksley SD (2002) <br />                                                                                   chelate reductase iron uptake soils. Nature 397: 694&#226;&#8364;&#8220;697 <br />    Identification, analysis, utilization conserved ortholog set mark- <br />                                                                                Rossberg M, Theres K, Acarkan A, Herrero R, Schmitt T, Schumacher K, <br />    ers comparative genomics higher plants. Plant Cell 14: 1457&#226;&#8364;&#8220;1467 <br />                                                                                   Schmitz G, Schmidt R (2001) Comparative sequence analysis <br /> Gaut BS, Le Thierry d&#226;&#8364;&#8482;Ennequin M, Peek AS, Sawkins MC (2000) Maize <br />                                                                                   reveals extensive microcolinearity lateral suppressor regions <br />    model evolution plant nuclear genomes. Proc Natl Acad <br />                                                                                   tomato, Arabidopsis, Capsella genomes. Plant Cell 13: <br />    Sci USA 97: 7008&#226;&#8364;&#8220;7015 <br />                                                                                   979&#226;&#8364;&#8220;988 <br /> Gebhardt C, Walkemeier B, Henselewski H, Barakat A, Delseny M, <br />                                                                                Schikora A, Schmidt W (2001) Iron stress-induced changes root <br />    Stuber K (2003) Comparative mapping potato (Solanum tuber- <br />                                                                                   epidermal cell fate regulated independently physiological <br />    osum) Arabidopsis thaliana reveals structurally conserved domains <br />                                                                                   responses low iron availability. Plant Physiol 125: 1679&#226;&#8364;&#8220;1687 <br />    ancient duplications potato genome. Plant J 34: 529&#226;&#8364;&#8220;541 <br />                                                                                Schikora A, Schmidt W (2002) Formation transfer cells H(1)- <br /> Giaever G, Chu AM, Ni L, Connelly C, Riles L, Veronneau S, Dow S, <br />                                                                                   ATPase expression tomato roots P Fe deficiency. Planta <br />    Lucau-Danila A, Anderson K, Andre B, et al (2002) Functional profiling <br />                                                                                   215: 304&#226;&#8364;&#8220;311 <br />    Saccharomyces cerevisiae genome. Nature 418: 387&#226;&#8364;&#8220;391 <br />                                                                                Schmidt W (1999) Mechanisms regulation reduction-based iron <br /> Heim MA, Jakoby M, Werber M, Martin C, Weisshaar B, Bailey PC (2003) <br />                                                                                   uptake plants. New Phytol 141: 1&#226;&#8364;&#8220;26 <br />    The basic helix-loop-helix transcription factor family plants: A         Scholz G, Becker R, Pich A, Stephan UW (1992) Nicotianamine&#226;&#8364;&#8221;a <br />    genome-wide study protein structure functional diversity. Mol           common constituent strategy-I strategy-II iron acquisition by <br />    Biol Evol 20: 735&#226;&#8364;&#8220;747                                                          plants&#226;&#8364;&#8221;a review. J Plant Nutr 15: 1647&#226;&#8364;&#8220;1665 <br /> Hell R, Stephan UW (2003) Iron uptake, trafficking homeostasis          Simillion C, Vandepoele K, Van Montagu MC, Zabeau M, Van Peer Y <br />    plants. Planta 216: 541&#226;&#8364;&#8220;551                                                    (2002) The hidden duplication past Arabidopsis thaliana. Proc Natl <br /> Herbik A, Koch G, Mock HP, Dushkov D, Czihal A, Thielmann J,                      Acad Sci USA 99: 13627&#226;&#8364;&#8220;13632 <br />    Stephan UW, Baumlein H (1999) Isolation, characterization cDNA          Stephan UW, Prochazka Z (1989) Physiological disorders nicotian- <br />    cloning nicotianamine synthase barley&#226;&#8364;&#8221;a key enzyme iron            amine-auxotroph tomato mutant chloronerva different levels of <br />    homeostasis plants. Eur J Biochem 265: 231&#226;&#8364;&#8220;239                              iron nutrition. I. Growth characteristics physiological abnormal- <br /> Higuchi K, Suzuki K, Nakanishi H, Yamaguchi H, Nishizawa NK, Mori S               ities related iron nicotianamine supply. Acta Bot Neerl 38: <br />    (1999) Cloning nicotianamine synthase genes, novel genes involved           147&#226;&#8364;&#8220;153 <br />    biosynthesis phytosiderophores. Plant Physiol 119: 471&#226;&#8364;&#8220;479        Suzuki K, Nakanishi H, Nishizawa NK, Mori S (2001) Analysis of <br /> Jakoby M, Wang HY, Reidt W, Weisshaar B, Bauer P (2004) FRU                       upstream region nicotianamine synthase gene Arabidopsis <br />    (BHLH029) required induction iron mobilization genes              thaliana: presence putative ERE-like sequence. Biosci Biotechnol <br />    Arabidopsis thaliana. FEBS Lett (in press)                                     Biochem 65: 2794&#226;&#8364;&#8220;2797 <br /> Kobayashi K, Ehrlich SD, Albertini A, Amati G, Andersen KK, Arnaud             Takahashi M, Terada Y, Nakai I, Nakanishi H, Yoshimura E, Mori S, <br />    M, Asai K, Ashikaga S, Aymerich S, Bessieres P, et al (2003) Essential         Nishizawa NK (2003) Role nicotianamine intracellular <br />    Bacillus subtilis genes. Proc Natl Acad Sci USA 100: 4678&#226;&#8364;&#8220;4683                 delivery metals plant reproductive development. Plant Cell 15: <br /> Ku HM, Liu J, Doganlar S, Tanksley SD (2001) Exploitation Arabidopsis-         1263&#226;&#8364;&#8220;1280 <br />    tomato synteny construct high-resolution map ovate-             Tanksley SD, Ganal MW, Prince JP, Vicente MC, Bonierbale MW, <br />    containing region tomato chromosome 2. Genome 44: 470&#226;&#8364;&#8220;475                   Broun P, Fulton TM, Giovannoni JJ, Grandillo S, Martin GB, et al <br /> Ku HM, Vision TJ, Liu J, Tanksley SD (2000) Comparing sequenced                   (1992) High density molecular linkage maps tomato potato <br />    segments tomato Arabidopsis genomes: large-scale duplica-           genomes. Genetics 132: 1141&#226;&#8364;&#8220;1160 <br />    tion followed selective gene loss creates network synteny. Proc     Thomine S, Lelievre F, Debarbieux E, Schroeder JI, Barbier-Brygoo H <br />    Natl Acad Sci USA 97: 9121&#226;&#8364;&#8220;9126                                                (2003) AtNRAMP3, multispecific vacuolar metal transporter involved <br /> Lander ES, Green P, Abrahamson J, Barlow A, Daly MJ, Lincoln SE,                  plant responses iron deficiency. Plant J 34: 685&#226;&#8364;&#8220;695 <br />    Newburg L (1987) MAPMAKER: interactive package              Thomine S, Wang RC, Ward JM, Crawford NM, Schroeder JI (2000) <br />  <br /> 4182                                                                      <br /> </body></html>