<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Divergence of the mRNA targets for the Ssb proteins of <br />  <br /> Ssb protein, gp32, RNA-binding proteins, DNA-binding proteinstranslational control, DNA replication <br />  <br />  <br />                   <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                   The single-strand binding (Ssb) protein of phage T4 (T4 gp32, product of gene 32) is a mRNA- <br />                   specific autogenous translational repressor, in addition <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> being a sequence-independent ssDNA- <br />                   binding protein that participates in phage DNA replication, repair and recombination. It is not clear <br />                   how this physiologically essential protein distinguishes between specific RNA and nonspecific <br />                   nucleic acid targets. Here, we present phylogenetic evidence suggesting that ssDNA and specific <br />                   RNA bind the same gp32 domain and that plasticity of this domain underlies its ability to configure <br />                   certain RNA structures for specific binding. We have cloned and characterized gene 32 of phage <br />                   RB69, a relative of T4 We observed that RB69 gp32 and T4 gp32 have nearly identical ssDNA <br />                   binding domains, but diverge in their C-terminal domains. In T4 gp32, it is known that the C- <br />                   terminal domain interacts with the ssDNA-binding domain and with other phage-induced proteins. <br />                   In translation assays, we show that RB69 gp32 is, like T4 gp32, an autogenous translational <br />                   repressor. We also show that the natural mRNA targets (translational operators) for the 2 <br />                   proteins are diverged in sequence <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> each other and yet can be repressed by either gp32. Results <br />                   of chemical and RNase sensitivity assays indicate that the gp32 mRNA targets from the 2 related <br />                   phages have similar structures, but differ in their patterns of contact with the 2 repressors. These <br />                   and other observations suggest that a range of gp32-RNA binding specificities may evolve in nature <br />                   due to plasticity of the protein-nucleic acid interaction and its response to modulation by the C- <br />                   terminal domain of this translational repressor. <br />  <br />  <br />  <br />  <br /> Introduction                                                                  repair and recombination [2,3]. Like other Ssb proteins, <br /> T4 gp32, the single-strand binding (Ssb) protein of bacte-                    T4 gp32 facilitates transactions at the replication fork, <br /> riophage T4, is a well studied member of the Ssb protein                      especially along the lagging strand, through its binding to <br /> family, and was the first such ssDNA-binding replication                      the unwound DNA template and its specific interactions <br /> protein to be discovered [1]. The protein, product of T4                      with other protein components of the DNA replisome. T4 <br /> gene 32, is an essential component of the phage DNA rep-                      gp32 is known to stimulate the phage induced DNA <br /> lication complex and also plays essential roles in DNA                        polymerase (T4 gp43) and to play a role in the dynamics <br />  <br />  <br />                                                                                                                                         Page 1 of 14 <br />                                                                                                                   (page number not for citation purposes) <br />  Virology Journal 2004, 1:4                                                              http://www.virologyj.com/content/1/1/4 <br />  <br />  <br />  <br /> of primosome (T4 gp61-gp41 complex) recruitment by                by the protein to selectively bind its own message in the <br /> the primase-helicase assembly protein T4 gp59 [4-6]. In           phage-induced mRNA pool. The translational operator for <br /> general, Ssb proteins lack specificity to the ssDNA               T4 gp32 has been mapped by RNA footprinting assays and <br /> sequence and this property allows them to perform their           determined to consist of two contiguous components, a 5&apos; <br /> physiological roles at all genomic locations undergoing           terminal ~28-nucleotide component that forms a folded <br /> replication, repair or recombination. The presence of a           structure (RNA pseudoknot) and an adjacent, less struc- <br /> Ssb protein in the right place at the right time may              tured, &gt;40-nucleotide component that lies 3&apos; to the pseu- <br /> depend, in large measure, on specificity of its interactions      doknot [15,16]. The 3&apos; terminal component includes <br /> with other proteins from the same biological source.              several repeats of UUAAA or UAAA sequences, in addition <br />                                                                   to harboring typical prokaryotic nucleotide determinants <br /> T4 gp32 has the interesting property of being able to con-        for translation initiation by ribosomes [7,16,17]. The <br /> trol its own biosynthesis at the translational level in vivo.     RNA pseudoknot and UUAAA/UAAA elements are both <br /> The protein binds to a specific target (translational opera-      essential for autogenous repression of the mRNA by T4 <br /> tor) in the 5&apos; leader segment of the mRNA from gene 32,           gp32 [15,16,18]. In vitro studies suggest that the pseudo- <br /> and represses translation of this RNA [7]. Another Ssb pro-       knot serves as the initial recognition (nucleation) site for <br /> tein, gp5 of the M13 ssDNA phage family, has also been            the protein and that this gp32-RNA interaction leads to <br /> shown to act as a mRNA-specific translational repressor,          cooperative binding of additional gp32 monomers to the <br /> although in this case, the RNA target is located in the mes-      less structured downstream sequence containing the <br /> sage for another essential M13 replication protein, gp2           UUAAA/UAAA elements and ribosome-binding site (RBS) <br /> (an endonuclease) [8,9]. It is not known if other Ssb pro-        [16]. Cooperative binding to the mRNA is envisaged to be <br /> teins, especially those for cellular DNA replication and          analogous to gp32-ssDNA interactions, except that the <br /> maintenance, also possess RNA binding functions that              UUAAA/UAAA sequence elements probably contribute to <br /> regulate specific translation or other physiologically            specificity of the mRNA interaction to the protein. <br /> important RNA-dependent processes. In T4, the physio- <br /> logical link between the sequence-independent ssDNA               The 3-dimensional structure of intact T4 gp32 has not <br /> and specific RNA binding functions of gp32 has been               been solved, although a number of biochemical and phys- <br /> explained by a model based on in vitro measurements of            iological observations have provided clues that the pro- <br /> the protein&apos;s binding affinities to different nucleic acid lig-   tein is modularly organized into 3 distinct domains [19]. <br /> ands. It has been observed that ssDNA is favored over             In particular, studies with proteolytic fragments of puri- <br /> translational operator RNA as a ligand for T4 gp32 and            fied T4 gp32, including the analysis of a crystal structure <br /> that RNA of nonspecific sequence is the least preferred           for one of these fragments [20], have assigned the ssDNA <br /> nucleic-acid ligand for this Ssb protein [10-12]. In vivo, T4     binding function to a module formed by an internal seg- <br /> encoded mRNA for gp32 is intrinsically more metaboli-             ment of the 301-residue protein. It is presumed that this <br /> cally stable than the typical prokaryotic mRNA and is             domain is responsible for binding specific RNA as well, <br /> thought to have opportunities to undergo many cycles of           although no direct evidence exists for this notion. In the <br /> gp32-mediated repression and depression during the rep-           studies described here, we show that the ssDNA-binding <br /> lication and other processing of phage DNA. The potential         domain is highly conserved between T4 gp32 and the phy- <br /> for translation of this mRNA in the T4 infected E coli host       logenetic variant of this protein from the T4-like phage <br /> is thought to be determined by availability of ssDNA in           RB69. Yet, we also show that sequences of the mRNA tar- <br /> the metabolic pool [10,13,14]. DNA damage or unwind-              gets for the two Ssb proteins are different and that the two <br /> ing transactions are thought to draw gp32 away from its           repressors differ in their patterns of interaction with these <br /> mRNA target to the exposed ssDNA, thus causing dere-              targets. We present results suggesting that specificity of <br /> pression of translation and upward adjustments in gp32.           gp32 to RNA has co-evolved with specificity of this Ssb <br /> Repression of the mRNA would then be reestablished if             protein to other phage induced proteins of DNA metabo- <br /> the amount of gp32 exceeded the number of exposed                 lism that interact with gp32&apos;s C-terminal domain. Our <br /> ssDNA sites for the protein. This model is consistent with        studies suggest that the ability of a diverging regulatory <br /> many in vivo observations relating to levels of T4 gp32 bio-      RNA to make alternate contacts with a mutually plastic, <br /> synthesis under conditions of DNA damage or abnormal              but highly conserved, RNA-binding protein site may allow <br /> accumulation of ssDNA in the phage infected bacterial             the RNA to tolerate mutational changes without loss of <br /> host [7].                                                         the regulatory function. Such plasticity of the interacting <br />                                                                   partners could allow for the evolution of a broad spec- <br /> It is not clear how T4 gp32 distinguishes between specific        trum of gp32-RNA binding specificities despite selective <br /> RNA and the non-specific nucleic acid sequence of ssDNA           pressures that conserve the amino acid sequence of the <br /> or ssRNA ligands. It appears that single-strandedness of          protein&apos;s nucleic acid-binding domain. <br /> the nucleic acid is not the most important criterion used <br />  <br />  <br />                                                                                                                       Page 2 of 14 <br />                                                                                                 (page number not for citation purposes) <br />  Virology Journal 2004, 1:4                                                           http://www.virologyj.com/content/1/1/4 <br />  <br />  <br />  <br />  <br /> Methods                                                         cloned and amplified RB69 genomic segments yielded <br /> Bacterial and phage strains used                                sufficient information for designing new primers to <br /> The E coli K-12 strain K802 (hsdR, hsdM+, gal, met, supE)       amplify, from genomic DNA, the entire wild-type RB69 <br /> was used as host in cloning experiments and the E coli B        gene 32, as well as shorter segments of this gene and its <br /> strain NapIV (hsdRk+, hsdMk+, hsdSk+, thi, supo) was the host   putative control region in the untranslated RB69 IC59-32 <br /> for plasmid-mediated gene expression studies that uti-          region (Fig 1). DNA sequence information obtained from <br /> lized lambda pL control. E coli B strain BL21(DE3), which       these analyses was also used for another study, which was <br /> harbors a T7 RNA polymerase gene under cellular lac pro-        aimed at determining the sequence of the entire RB69 <br /> moter control [21], was used as the host for T7 &#206;&#166;10-pro-        genome (GenBank NC_004928). <br /> moter plasmids in pilot experiments that assessed toxicity <br /> of cloned RB69 gene 32 to bacterial cells.                      Assays for plasmid directed gene 32 expression <br />                                                                 We used the lambda pL plasmid vector pLY965 [24] to <br /> Cloning and nucleotide sequence determination of RB69           clone RB69 gene 32 sequences that were designated for in <br /> gene 32                                                         vivo expression studies. This vector expresses cloned DNA <br /> In preliminary experiments, we used Southern blot analy-        under control of the heat-inducible &#206;&#187;cI857pL element, <br /> sis of AseI-digested RB69 genomic DNA to identify and           which produces sufficient cI857 repressor under unin- <br /> retrieve an ~35-kb DNA fragment that hybridized to a T4         duced conditions (&#226;&#8240;&#164;30&#194;&#176;C) as to maintain pL-mediated <br /> gene 32-specific riboprobe under stringent conditions.          expression at undetectable levels. Minimizing plasmid- <br /> The riboprobe was prepared by methods described previ-          driven transcription from pL contributed to stable mainte- <br /> ously [22,23] using the T4 gene 32 clone pYS69 [15],            nance of the cloned wild-type RB69 gene 32, the product <br /> which was generously provided by Y Shamoo. We were              of which is highly toxic to bacterial cells. RB69 gene 32 <br /> unable to clone this AseI fragment in AseI-compatible Eco       mutants still emerged when such clones were grown at <br /> R1-generated ends of plasmid vectors. However, further          &#226;&#8240;&#164;30&#194;&#176;C. Some of these mutants were archived for use as <br /> digestion of the AseI fragment with ApoI (which generates       controls in certain studies (eg, PL2 and PL8, Fig 4). With <br /> Nde1-compatible ends) yielded a shorter, ~15-kb, frag-          the T7 &#206;&#166;10-promoter expression vector pSP72 (Promega) <br /> ment that could be cloned in the NdeI-EcoRI interval of         as the cloning vehicle, clones containing the wild-type <br /> vector pNEB193 (cat# N3051S, New England Biolabs,               RB69 gene 32 were not viable when introduced into E coli <br /> Beverly, MA). The cloned fragment was sequenced and             BL21(DE3), probably because of residual (constitutive) <br /> found to be very similar to the T4 genetic segment extend-      lac-promoter activity in this bacterial host. To circumvent <br /> ing from gene 59 through the 5&apos; terminal ~2/3 of gene 32,       potential toxicity, pSP72-based recombinants were prop- <br /> except that the RB69-derived DNA appeared to lack a             agated in hosts lacking a T7 RNA polymerase gene. The <br /> homologue of the T4 ORF 32.1 (see below). Comparisons           purified plasmid DNA from these hosts was used for in <br /> between the T4 and RB69 gene 59-32 regions are dia-             vitro transcription and translation assays. Methods for the <br /> grammed in Fig 1. We retrieved the remainder (3&apos; terminal       radiolabeling of plasmid encoded proteins and their sub- <br /> segment) of RB69 gene 32 from RB69 genomic DNA,                 sequent analysis by SDS-PAGE have been described else- <br /> through PCR amplification using Taq DNA polymerase.             where [24,25], and conditions pertaining to specific <br /> For this purpose, we utilized two primers, one perfectly        experiments are given in figure legends. <br /> matching a sequence in the cloned AseI-ApoI RB69 frag- <br /> ment             (ie,          upstream             primer:     Purification of gp32 from clones of the structural gene <br /> 5&apos;GCTGCTAAGAAATTGTTCATAG3&apos;) and the other (the                  RB69 gp32 and T4 gp32 were purified from the overpro- <br /> downstream primer), an 18-mer bearing the sequence              ducing clones pRBg32&#226;&#710;&#8224;op (RB69 gp32) and pYS69 (T4 <br /> 5&apos;CAGCAGCAGTGAAACCTTTA3&apos;, was chosen from a                     gp32), respectively. We used the gp32 purification proto- <br /> PCR screen of an RB69 primer library. DNA amplification         col outlined by Bittner et al, [26] with minor modifica- <br /> was carried out under low-stringency conditions for             tions. The preparation of crude extracts, from 6-liter <br /> primer annealing (30 sec at 25&#194;&#176;C), which allowed activity       batches of heat-inducible E coli NapIV clones of phage <br /> from the imperfectly matched downstream primer. We              genes, was as described previously for T4 RegA protein <br /> obtained several products that we resolved by agarose gel       [27]. Anionic-exchange chromatography (using Q-Sepha- <br /> electrophoresis Only one of these products, an ~35-kb           rose; Cat# 17-0510-01; Pharmacia) was as described for <br /> DNA fragment, hybridized, although poorly, to the T4            purification of plasmid-generated RB69 gp43 [28]. Under <br /> gene 32-specific riboprobe initially used for the Southern      the conditions used, gp32 eluted at 0.3&#226;&#8364;&#8220;0.4 M NaCl. In <br /> blot analysis of Ase1-digested RB69 genomic DNA. This           the subsequent chromatographic step, utilizing Phenyl- <br /> fragment was sequenced, using the PCR, and found to             Sepharose (Cat#17-0965-05; Pharmacia), we tested col- <br /> contain the 3&apos; terminal segment of RB69 gene 32 as well         umn fractions for nuclease contamination by incubating <br /> as some of the region distal to RB69 gene 32 (relative to       4 &#194;&#181;l samples with plasmid DNA (~1 &#194;&#181;g) overnight at <br /> the T4 genetic map). Collectively, sequence analysis of the     room temperature and then analyzing the mixtures by <br />  <br />  <br />                                                                                                                    Page 3 of 14 <br />                                                                                              (page number not for citation purposes) <br />  Virology Journal 2004, 1:4                                                              http://www.virologyj.com/content/1/1/4 <br />  <br />  <br />  <br />  <br /> A <br /> Figure <br />    comparison <br />         1     between the genetic maps of the Ssb protein (gp32) encoding regions of phages T4 and RB69 <br /> A comparison between the genetic maps of the Ssb protein (gp32) encoding regions of phages T4 and RB69. <br /> Note the presence of an open-reading frame (ORF) for a homing endonuclease (SegG protein; [45]) between T4 genes 59 <br /> (gp59; primase-helicase loader) and 32 (gp32; Ssb protein). The restriction sites we used for cloning RB69 gene 32 are marked, <br /> and compared to the locations of analogous sites in T4. GenBank Accession numbers for the genetic regions of interest are <br /> also noted. <br />  <br />  <br />  <br />  <br /> agarose gel electrophoresis. The gp32-containing fractions       elsewhere [23,29]. Sequencing primers were annealed to <br /> that exhibited no hydrolysis of the plasmid DNA were             codons 12 to 20 of the transcripts and the sequenced seg- <br /> pooled and the protein was purified further by chroma-           ments of the RNA spanned nucleotide positions +36 <br /> tography on ssDNA-agarose (Cat #15906-019; Invitro-              through about -100 relative to the initiator AUG. For in <br /> gen). Pooled fractions from the ssDNA chromatography             vitro translation assays, the RNA preparations included <br /> were dialyzed against a gp32 storage buffer containing 0.1       full <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> and truncated versions of the gene 32 open- <br /> M NaCl, 20 mM Tris-HCl pH 8.0, 1 mM EDTA, 0.5 mM                 reading frame from each of the 2 phage sources. <br /> DTT and 50% glycerol Protein stocks (at 4&#226;&#8364;&#8220;8 mg gp32/ <br /> ml) were stored at -20&#194;&#176;C until used.                             Assays for gp32-mediated in vitro translational repression <br />                                                                  We used E coli S30 cell-free extracts (Cat#L1020; <br /> Preparation of RNA for in vitro studies                          Promega) with purified pSP72-based gene 32 recom- <br /> RNA preparations used for footprinting and other in vitro        binant DNA (coupled transcription-translation assays) or <br /> studies originated from in vitro transcription of pSP72          purified RNA (DNA-free translation assays) to assess <br /> clones of the desired gene 32 sequences Methods have             repressor activities of purified RB69 gp32 and T4 gp32. <br /> been described elsewhere [29]. Phage-specific RNA                With plasmid-directed gene 32 expression, it was possible <br /> sequences of the purified transcription products used for        to use expression of the plasmid borne bla gene (&#206;&#178;-lacta- <br /> footprinting included nucleotide positions -102 to +161          mase) as an internal control. Each 50 &#194;&#181;l in vitro assay reac- <br /> (relative to the initiator AUG) in case of the RB69 gene 32      tion mixture (placed in a 15-ml conical tube) contained 1 <br /> transcripts and positions -96 to +161 in case of the T4          &#194;&#181;g of plasmid DNA template or 4 &#194;&#181;g RNA, 5 &#194;&#181;l of a mix- <br /> gene 32 transcripts. These products also included a 10-nt        ture of all amino acids (1 mM each) except L-methionine, <br /> sequence from the plasmid&apos;s T7 promoter region RNA               1 &#194;&#181;l of an S30-premix cocktail (containing rNTPs, tRNAs, <br /> sequencing was carried out by using the RVT-catalyzed            an ATP generating system and required salts), 15 &#194;&#181;l S30 <br /> primer-extension (cDNA synthesis) method described               extract and the balance of volume in nuclease-free water <br />  <br />  <br />                                                                                                                      Page 4 of 14 <br />                                                                                                (page number not for citation purposes) <br />  Virology Journal 2004, 1:4                                                              http://www.virologyj.com/content/1/1/4 <br />  <br />  <br />  <br />  <br /> Results <br /> Figure of4 experiments showing that RB69 gp32 is an autogenous translational repressor <br /> Results of experiments showing that RB69 gp32 is an autogenous translational repressor. For Panel A, <br /> &#206;&#187;CI857PLN-bearing plasmid clones of the diagrammed DNA segments were heat-induced (42&#194;&#176;C) and assayed for gp32 synthe- <br /> sis as described in other work [24,27]. RBG32 is a DNA segment that carries the wild-type sequence from -120 through +900 <br /> relative to the first <span id='am-84' about='xsp:base' typeof='owl:Thing'>base</span> of the initiator AUG of RB69 gene 32. RBG32&#226;&#710;&#8224;op is a truncated derivative of RBG32 that lacks ele- <br /> ments of the putative RNA pseudoknot of RB69 gene 32 (Figs 3 &amp; 6). PL8 is identical to RBG32 except that it carries a single- <br /> base substitution (marked with an asterisk) in codon 173, leading to a F173S substitution in RB69 gp32. PL2 is similar to RBG32 <br /> and PL8, except that it carries several point mutations (map positions marked with asterisks). Panel B shows results of an <br /> experiment in which purified RB69 gp32 was shown to inhibit in vitro translation of purified mRNA from the cloned RBG32 <br /> fragment, as well as mRNA from in vitro expressed plasmid clone (coupled transcription/translation). Conditions for these <br /> assays are described in METHODS. <br />  <br />  <br />  <br />  <br />                                                                                                                       Page 5 of 14 <br />                                                                                                 (page number not for citation purposes) <br />  Virology Journal 2004, 1:4                                                           http://www.virologyj.com/content/1/1/4 <br />  <br />  <br />  <br /> Reaction mixtures, including any added gp32, were con-         ~85% of amino-acid positions (92% overall similarity), <br /> stituted in an ice bath before transferring to 37&#194;&#176;C for        with most of the differences being clustered in 2 short <br /> incubations (30 or 60 min). Reactions were stopped by          blocks of amino-acid sequence in the highly charged C- <br /> rechilling in the ice bath. Proteins from 5 &#194;&#181;l samples were    terminal segment of the protein, D264(RB69)/A264(T4) <br /> precipitated with 20 &#194;&#181;l acetone, collected by centrifuga-      to L299(RB69)/L301(T4). Both C-terminal segments are <br /> tion, dried and suspended in SDS extraction buffer for         rich in serines and aspartates; however, they differ in their <br /> analysis by SDS-PAGE and autoradiography. Analysis of          arrangements of these residues and the serine-rich cluster <br /> plasmid encoded (N-terminal) gp32 fragments was car-           is 5 residues longer in T4 gp32 (S282-S286). In contrast to <br /> ried out in SDS-PAGE (10% gels) using Tricine as the elec-     their conspicuous differences in the C-terminal domain, <br /> trophoresis buffer. This buffer system allows for effective    T4 gp32 and RB69 gp32 are closely similar in segments <br /> resolution of small polypeptides [30]. When used, puri-        that, in T4 gp32, have been implicated in cooperative <br /> fied gp32 was added at concentrations ranging between 5        gp32-gp32 interactions (95% identity/100% similarity for <br /> and 20 &#194;&#181;M.                                                     the N-terminal 21 residues) and ssDNA binding (residues <br />                                                                21 to 254; ~92% identity/~95% similarity). We note that <br /> Treatments of RNA with RNases and chemical agents              all T4 gp32 residues that have been implicated in ssDNA <br /> The RNA-modifying chemical reagents Dimethylsulfate            binding are conserved in RB69 gp32 (Fig 2). However, <br /> (DMS; Cat# D18,630-9; Aldrich) and Diethylpyrocar-             interestingly, codon sequences for the two aligned N-ter- <br /> bonate (DEPC; Cat# D5758; Sigma) and the ribonucle-            minal gp32 segments differ at many third nucleotide posi- <br /> ases (RNases A1, T1 and V1 respectively) were used to          tions between T4 and RB69, suggesting that there has <br /> probe RB69- and T4-derived operator RNAs for intrinsi-         been natural selection for amino acid identity (and not <br /> cally structured regions. The RNases were also used for        merely chemical or side-chain similarity) in the N-termi- <br /> RNA footprinting (protection by gp32) studies.                 nal two-thirds of the phage Ssb protein. We also note that <br />                                                                both proteins contain 2 &quot;LAST&quot; (3KRKST7 or <br /> DMS was diluted in absolute ethanol at ratios of 1:2, 1:4,     110KRKTS114) sequence motifs, which in the T4 system <br /> and 1:5 ratio v/v and its effects were analyzed at the three   have been implicated in interactions with the negatively <br /> concentrations. The reaction buffer contained 30 mM            charged surfaces of DNA as well as with the C-terminal <br /> HEPES pH 7.5, 10 mM MgCl2. Reactions were stopped in           domain of gp32 [31]. One of these motifs (K3-T7) lies <br /> 0.5 M &#206;&#178;-mercaptoethanol and 0.75 M sodium acetate. The         near the extreme N-terminus of the protein and the sec- <br /> protocol for DEPC treatment was identical to that for          ond (K110-S114) is adjacent to a short sequence (residues <br /> DMS, except that we used 1 &#194;&#181;l of DEPC per 100 &#194;&#181;l of reac-      102&#226;&#8364;&#8220;108) that diverges between T4 and RB69 (~50% sim- <br /> tion mix and incubated the reactions at room temperature       ilarity), but that also contains 3 conserved charged resi- <br /> for 10 min.                                                    dues including the DNA-binding tyrosine Y106 of T4 <br />                                                                gp32 [20]. <br /> For the RNase-sensitivity assays, including gp32-mediated <br /> RNA footprinting, digestions with RNases A1 and T1 were        The RB69 IC59-32 region <br /> carried out in 30 &#194;&#181;l buffer containing 60 mM NH4Cl, 10         Figure 3 shows an alignment of the RB69 IC59-32 region <br /> mM Mg acetate, 10 mM Tris-HCl pH 7.4, and 6 mM &#206;&#178;-              with its counterpart (the IC32.1-32 region) from T4 The <br /> Mercaptoethanol. The buffer for digestions with RNase V1       T4 region (GenBank NC_00866) has been experimentally <br /> contained 25 mM Tris-HCl pH 7.2, 10 mM MgCl2, and 0.2          documented to harbor the translational operator for gene <br /> M NaCl Incubations were at 37&#194;&#176;C in 30 &#194;&#181;l buffer in all         32 expression [6]. The RB69 counterpart (GenBank <br /> cases. RNase treatments were halted with an equal volume       NC_004928) is 7 nucleotides longer and ~70% identical <br /> of buffer containing 0.4 M Na acetate pH 5.2, 20 mM            in sequence. By comparison, the gp32 encoding portions <br /> EDTA, and 30 &#194;&#181;g E coli tRNA. When used for RNA foot-           of the T4 and RB69 genes are ~80% identical in the overall <br /> printing, RB69 gp32 or T4 gp32 was added at concentra-         nucleotide sequence (see Fig 1 for GenBank accession <br /> tions in the range between 1 &#194;&#181;M and 5 &#194;&#181;M.                      numbers) and their predicted protein products are &gt;90% <br />                                                                similar in amino acid sequence. There is an additional 40- <br /> Results                                                        nt untranslated sequence in the RB69 IC59-32 region that <br /> A sequence comparison between T4 gp32 and RB69 gp32            appears to have no T4 counterpart (Fig 3), and ORF321 is <br /> The amino acid sequence of RB69 gp32 was deduced from          missing altogether in RB69 (Fig 1). So, it appears that the <br /> the determined nucleotide sequence of the gene. An align-      regions between genes 59 and 32 of T4 and RB69 have <br /> ment between the predicted primary structures of this pro-     undergone more evolutionary divergence from each other <br /> tein and its T4 homologue is shown in Fig 2, which also        than their gp32-encoding regions. However, despite their <br /> highlights the main differences between the 2 proteins         differences in nucleotide sequence, the translational oper- <br /> and points out certain functionally important landmarks        ator sequence of T4 gene 32 and its putative RB69 coun- <br /> on the T4 gp32 sequence. The two proteins are identical at     terpart are predicted, by computer programs, to form <br />  <br />  <br />                                                                                                                    Page 6 of 14 <br />                                                                                              (page number not for citation purposes) <br />  Virology Journal 2004, 1:4                                                                 http://www.virologyj.com/content/1/1/4 <br />  <br />  <br />  <br />  <br /> Figure 2 sequence alignments between the Ssb proteins (gp32s) of T4 and RB69 <br /> Amino-acid <br /> Amino-acid sequence alignments between the Ssb proteins (gp32s) of T4 and RB69. Residues and segments of the <br /> T4 gp32 sequence that have been implicated in specific biological functions of the protein are marked as follows: Db [DNA <br /> binding residue]; Zb (residues that coordinate Zn++ in the zinc-binding domain; [20,46]); gp32-gp32 [residues involved in coop- <br /> erative gp32 binding to ssDNA]; XLgp59 (residue that cross-links to gp59; [42]); LAST (sequence motifs, (Lys/Arg)3 (Ser/ <br /> Thr)2, that have been proposed to directly bind nucleic-acids or mediate gp32-gp32 interactions [31]). The shaded C-terminal <br /> portion of T4 gp32 has been implicated in interactions with other phage induced proteins [38]. The small deletion (&#226;&#710;&#8224;32PR201) <br /> alters specificity of T4 gp32 in phage replication without affecting autogenous translational repression [39]. The largest vertical <br /> arrows denote trypsin-hypersensitive sites (19) The G-to-A <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span>mutation</span> marked &quot;(ts)&quot; was isolated in this laboratory as a mis- <br /> sense (temperature-sensitive) suppressor of a defective gp43 function (unpublished). In the RB69 gp32 sequence, residues <br /> whose codons differ from their conserved T4 counterpart at the third nucleotide are underscored with a single dot; those dif- <br /> fering by 2 nucleotides are marked by 2 dots. <br />  <br />  <br />  <br />                                                                                                                         Page 7 of 14 <br />                                                                                                   (page number not for citation purposes) <br />  Virology Journal 2004, 1:4                                                             http://www.virologyj.com/content/1/1/4 <br />  <br />  <br />  <br />  <br /> A <br /> Figure <br />    comparison <br />         3      between the nucleotide sequences of the T4 IC321-32 and RB69 IC59-32 regions <br /> A comparison between the nucleotide sequences of the T4 IC32.1-32 and RB69 IC59-32 regions. These 2 regions <br /> contain determinants for translation initiation of the respective phage-induced mRNAs for gp32. The chart emphasizes <br /> sequence differences (entered as lettered residues in the RB69 sequence) between the 2 regions. The dashes indicate identity <br /> between RB69 and T4 residues. Sequence elements contributing to RNA pseudoknot formation in the T4 gene 32-specific <br /> mRNA are marked by horizontal arrows. Note the sequence overlap between elements of the pseudoknot and ORF32.1 (segG) <br /> of the T4 sequence. Also, see Fig 6 for a summary of properties of the RB69 sequence. <br />  <br />  <br />  <br />  <br /> similar structures. We address this prediction below and        that the cloned RB69 wild-type gene 32 supported effi- <br /> present experimental evidence for the RNA structure and         cient growth of T4 gene 32 mutants (bursts of ~100) By <br /> its role in translational control of RB69 gp32 synthesis.       these criteria, the T4 and RB69 proteins appeared to be <br />                                                                 similarly functional in each other&apos;s physiological systems. <br /> RB69 gp32 and T4 gp32 are functionally similar                  Yet, the natural targets for the 2 proteins are clearly differ- <br /> Figure 4 shows results from experiments that measured           ent from each other in topography (Fig 3) and as we <br /> the effects of RB69 gp32 on its own synthesis in vivo (Fig      describe later, RNA-binding specificity differences <br /> 4A) and in vitro (Fig 4B). The in vivo experiments meas-        between the 2 proteins could be detected through in vitro <br /> ured plasmid-directed RB69 gene 32 expression by E coli         RNA-footprinting assays, which utilized lower concentra- <br /> clones carrying wild-type and mutant versions of the RB69       tions of gp32 than is usually required to detect gp32- <br /> gene. As shown in Fig 4A, induced expression of the gene        mediated repression by in vitro translational assays. <br /> was lower (by ~4-fold) with the wild-type construct than <br /> with deletion mutants of the untranslated 5&apos; leader of the      RNA structure in the RB69 gene 32 translational initiation <br /> mRNA (RBG32&#226;&#710;&#8224;op, Fig 4A) or missense mutants in the              region (TIR) <br /> structural gene from this phage (PL2 and PL8 constructs;        As discussed above for Fig 3, computer-assisted and visual <br /> Fig 4A). These observations are consistent with the expla-      examinations of the RB69 IC59-32 nucleotide sequence <br /> nation that RB69 gp32, like T4 gp32, is able to bind and        predicted an RNA topology that was similar to the T4 gene <br /> repress its own mRNA. The results shown in Fig 4B con-          32 translational operator, particularly with regards to <br /> firm that purified RB69 gp32 is a potent repressor of trans-    presence of a putative RNA pseudoknot structure to the 5&apos; <br /> lation of purified mRNA for this protein.                       side of the Shine-Dalgarno and UUAAA/UUAA sequence <br />                                                                 elements of the mRNA. We used 3 RNA modifying agents <br /> We have used similar experiments to those for Fig 4 to          to test directly for intrinsic secondary or higher-order <br /> compare repressor activities of T4 gp32 and RB69 gp32 on        structure in the RB69-derived RNA: DMS, DEPC and <br /> identical RNA targets, and observed that either protein can     RNase V1, respectively. Results are shown in Fig 5. We <br /> repress gene 32-specific mRNA from either source (results       observed that the RB69-derived sequence from nucleotide <br /> not shown). However, such experiments, which require            position A(-1) through A(-45), relative to the initiator <br /> 10&#226;&#8364;&#8220;30 &#194;&#181;M purified protein to demonstrate repression (Fig        AUG, was hypersensitive to cleavage following DMS or <br /> 4B), did not unambiguously distinguish between the              DEPC treatment (Fig 5A) and relatively insensitive to <br /> RNA-binding specificities of the 2 proteins. Also, in           cleavage by the dsRNA-specific RNase V1 (Fig 5B). These <br /> phage-plasmid complementation assays, we observed               observations, which are summarized in Fig 6A, are <br />  <br />  <br />                                                                                                                      Page 8 of 14 <br />                                                                                                (page number not for citation purposes) <br />  Virology Journal 2004, 1:4                                                              http://www.virologyj.com/content/1/1/4 <br />  <br />  <br />  <br />                                                                   to be insensitive to DEPC modification (Fig 5A). Below, <br />                                                                   we show that another position in this segment, G(-10), is <br />                                                                   relatively insensitive to the ssRNA-specific RNase T1. Pos- <br />                                                                   sibly, cleavage at U(-20) and G(-10) by RNA modifying <br />                                                                   agents is affected by RNA hairpin formation in the U(-8) <br />                                                                   to A(-21) sequence. The location of this putative hairpin, <br />                                                                   which is not predicted in the T4 RNA counterpart, is dia- <br />                                                                   grammed in Fig 6A. In summary, the T4 gene 32 transla- <br />                                                                   tional operator region and its putative counterpart from <br />                                                                   RB69 exhibit several topographical differences from each <br />                                                                   other, including an additional 6-nt sequence in RB69 that <br />                                                                   may contribute to RNA secondary structure formation in <br />                                                                   the RBS. Below, we show that the 2 regions also differ in <br />                                                                   their interactions with translational repressors. <br />  <br />                                                                   The footprints of T4 gp32 and RB69 gp32 on gene 32- <br />                                                                   specific RNA targets from T4 and RB69 <br />                                                                   We used the ssRNA-specific RNases A1 and T1 to deter- <br />                                                                   mine the abilities of gp32 from the 2 phage systems to <br />                                                                   protect RNA targets from cleavage with these enzymes. <br />                                                                   These RNA footprinting studies also extended the infor- <br />                                                                   mation we obtained from treatments with DMS and <br />                                                                   DEPC about intrinsic structure of the RNA targets. Results <br />                                                                   are shown in Fig 7 for the RB69-derived RNA target and <br />                                                                   Fig 8 for the T4-derived target. Also, a summary of our <br />                                                                   observations from these experiments is presented on the <br /> showing <br />    (Panel <br /> lowing <br /> V1 <br /> Portions <br /> Figure   5sites of cleavage <br />         treatments <br />           ofB)autoradiograms <br />                      with DMS   and DEPC <br />                             in RB69 <br />                                from RNA   (Panel A) and <br />                                     gene 32-derived <br />                                          sequencing  RNA   fol- <br />                                                      gelsRNase    RNA sequence charts in Fig 6B In the aggregate, our stud- <br /> Portions of autoradiograms from RNA sequencing                    ies showed that T4 gp32 and RB69 gp32 contact RNA tar- <br /> gels showing sites of cleavage in RB69 gene 32-                   gets differently from each other, although the two <br /> derived RNA following treatments with DMS and                     proteins overlap in their RNA-binding properties. We <br /> DEPC (Panel A) and RNase V1 (Panel B). These exper-               highlight the following specific observations. <br /> iments probed the RB69 RNA for secondary and higher- <br /> order structure. The lanes marked &quot;RNA seq&quot; show results          1. At the protein concentrations used (1&#226;&#8364;&#8220;5 &#194;&#181;M), the RB69 <br /> from sequencing untreated RNA by the RVT-catalyzed chain          gp32 footprint on the RNA target from RB69 was 5 resi- <br /> termination method [23,35]. In Panel A the lane marked with       dues longer than the footprint of this protein on the T4- <br /> a &quot;minus&quot; sign shows the positions of RVT chain termination <br />                                                                   derived RNA target; however, the positions of the 2 foot- <br /> caused by RNA structure in the untreated RNA. The DMS <br /> and DEPC lanes show sites of hypersensitivity (cleavage) of       prints relative to the respective initiator AUG and 5&apos; termi- <br /> the same RNA to treatment with these chemical agents. In          nal boundary of the pseudoknot structure appeared to be <br /> Panel B, the V1 lanes denote the amount of RNase V1 (&#195;&#8212;10-5        identical (Fig 6) <br /> units) used to digest the RNA substrate. <br />                                                                   2. As can be seen in Figs 7A and 7B, RB69 gp32 protected <br />                                                                   its own mRNA target strongly within the nucleotide seg- <br />                                                                   ment between U(-14) and G(-61), and weakly in the <br />                                                                   segment from U(-2) to G(-9) In contrast, as seen in Figs <br /> consistent with the prediction that the A(-1) to A(-45) seg-      7C and 7D, T4 gp32 protected this RNA strongly only in <br /> ment of the RB69 IC59-32 RNA region is intrinsically              the segment from C(-42) to G(-61) <br /> unstructured. In contrast, the segment of this RNA corre- <br /> sponding to the putative pseudoknot structure can                 3. As can be seen in Fig 8C and 8D, T4 gp32 protected the <br /> accommodate a range of /RNA sequences. The interaction            T4-derived RNA strongly in the G(+3) to U(-70) segment <br /> may also be subject to is hypersensitive to RNase V1 (Fig         In contrast, RB69 gp32 protected this RNA target best in <br /> 5B) and less sensitive than the A(-1) to A(-45) segment to        the U(-16) to U(-70) segment (Fig 7A and 7B) <br /> the 2 chemical agents used (Fig 5A). There was one unex- <br /> pected observation in these experiments RB69 nucleotide           It should be noted that the gp32 footprint sizes reported <br /> position U(-20), which is located in the putatively               here are shorter than has been reported in studies that uti- <br /> unstructured portion of the RNA target (Fig 6A), appeared         lized higher concentrations of T4 gp32 with T4-specific <br />  <br />  <br />                                                                                                                       Page 9 of 14 <br />                                                                                                 (page number not for citation purposes) <br />  Virology Journal 2004, 1:4                                                            http://www.virologyj.com/content/1/1/4 <br />  <br />  <br />  <br />  <br /> Figure <br /> Summaries6 of results from the chemical and RNase sensitivity and RNA footprinting studies reported here <br /> Summaries of results from the chemical and RNase sensitivity and RNA footprinting studies reported here. <br /> Panel A shows our interpretation of experiments that probed the existence of RNA structure in RB69 gene 32-specific RNA <br /> (Fig 5). The T4-derived RNA counterpart is shown for comparison The &quot;caret&quot; symbol denotes sensitivity to cleavage after <br /> DMS treatment; asterisks denote sensitivity to cleavage after DEPC treatment. The darker symbols denote greater sensitivity. <br /> Positions that are not marked by any symbols were resistant to the modifying agents under the conditions used. Vertical <br /> arrows mark positions that were sensitive to RNase V1. Panel B shows our interpretation of the RNA footprinting studies <br /> described in Figs 7 and 8. Positions of protection from RNaseA1 by gp32 are marked by the triangles and protection from <br /> RNase T1 by the pentagonal symbols. The darker symbols denote stronger protection. Unmarked positions were not pro- <br /> tected by either gp32 from phage source under the experimental conditions used. <br />  <br />  <br />  <br />  <br /> RNA targets [16,32]. As stated earlier in this report (Fig 4),   overlaps in function, we have commonly observed specif- <br /> the higher gp32 concentrations (&gt;5 &#194;&#181;M) mask specificity          icity differences between protein homologues from the 2 <br /> differences between the T4 and RB69 proteins.                    phage systems. For example, in plasmid-phage comple- <br />                                                                  mentation assays, RB69 DNA polymerase (gp43) was <br /> Discussion                                                       observed to be just as effective as T4 gp43 in T4 DNA rep- <br /> Phages T4 and RB69 are phylogenetically related to each          lication in vivo, whereas the T4 enzyme was less effective <br /> other and encode homologous sets of DNA replication              than its RB69 counterpart for RB69 DNA replication <br /> proteins that exhibit a significant degree of compatibility      [22,33]. Also, the 2 DNA polymerases, like the 2 Ssb pro- <br /> with each other&apos;s biological systems [22,24]. Despite such       teins compared here, are RNA-binding autogenous <br />  <br />  <br />                                                                                                                    Page 10 of 14 <br />                                                                                               (page number not for citation purposes) <br />  Virology Journal 2004, 1:4                                                           http://www.virologyj.com/content/1/1/4 <br />  <br />  <br />  <br />  <br /> In <br /> Figure <br />    vitro footprinting <br />          7            of RB69 gene 32-specific RNA with purified RB69 gp32 (Panels A and B) and T4 gp32 (Panels C and D) <br /> In vitro footprinting of RB69 gene 32-specific RNA with purified RB69 gp32 (Panels A and B) and T4 gp32 (Pan- <br /> els C and D). Preparation of RNA and proteins and experimental conditions for footprinting are described in METHODS. <br /> Horizontal arrows mark nucleotide positions (Fig 6B) that exhibited gp32-mediated protection from RNaseA (panels A and C) <br /> and RNase T1 (panels B and D). Darker arrows denote stronger protection. The results are summarized in Fig 6B. <br />  <br />  <br />  <br />  <br /> translational repressors that differ in RNA binding specif-     observed that the same RNA target could exhibit different <br /> icity and RNA target sequence. Studies with the T4 ver-         patterns of protection depending on source of the Ssb pro- <br /> sions of gp43 and gp32 clearly show that the binding of         tein. This observation suggests that the RNA-protein inter- <br /> these proteins to specific RNA is mutually exclusive with       action is intrinsically flexible and can accommodate a <br /> their binding to DNA [7,34]. So, conservation of the trans-     range of RNA sequences as long as these sequences can be <br /> lational functions of these proteins may be related to con-     made to assume a certain configuration. In addition, the <br /> servation of their replication functions. Based on previous     interaction could be subject to modulation by intra- and <br /> studies with RB69 gp43 [35], as well as the current study       intermolecular protein-protein interactions of the <br /> with RB69 gp32, we surmise that neither of these transla-       repressor. In this regard, it is known that the extreme N- <br /> tional repressors possesses a domain that binds RNA             terminal segment (~20 residues) and C-terminal segment <br /> exclusively. Rather, in both cases, the RNA binding site        (~100 residues) of T4 gp32 have profound effects on the <br /> seems to be contained within the region of the protein          ssDNA binding activity, which is housed in the region <br /> that binds DNA. Thus, it is possible that in phage infected     bracketed by these 2 segments of the protein [19,36,37]. <br /> cells, specific RNA serves as a regulator of both the biosyn-   The N-terminal segment determines cooperative binding <br /> thesis and replicative activities of these proteins.            to ssDNA (through gp32-gp32 interactions) and the C-ter- <br />                                                                 minal segment has been implicated in interactions of <br /> In the purified system we have used to compare RNA foot-        gp32 with other phage induced proteins [38]. Possibly, <br /> prints for gp32 from T4 and RB69 (Figs 6, 7, 8), we             the observed sequence divergence between the C-terminal <br />  <br />  <br />                                                                                                                   Page 11 of 14 <br />                                                                                              (page number not for citation purposes) <br />  Virology Journal 2004, 1:4                                                            http://www.virologyj.com/content/1/1/4 <br />  <br />  <br />  <br />  <br /> D; <br /> In vitro <br /> Figure <br />     RNasefootprinting <br />          8 T1)        of T4 gene32-specific RNA with purified RB69 gp32 (Panels A and C; RNase A) and T4 gp32 (Panels B and <br /> In vitro footprinting of T4 gene32-specific RNA with purified RB69 gp32 (Panels A and C; RNase A) and T4 <br /> gp32 (Panels B and D; RNase T1). Conditions for these experiments were identical to those described in Fig 7, except <br /> that the RNA substrate used for footprinting was derived from clones of T4 gene 32 rather than RB69 gene 32. See also Fig 6C <br /> for a summary. <br />  <br />  <br />  <br />  <br /> domains of T4 gp32 and RB69 gp32 (Fig 2) was in part            to gp59, the phage-induced primase-helicase loading <br /> coupled to divergence of the mRNA targets during evolu-         protein [41,42]. Such observations suggest that the 2 <br /> tion of the 2 related translational repressors.                 nucleic-acid binding functions of gp32 may be subject to <br />                                                                 regulation by a combination of intra- and intermolecular <br /> Although we cannot rule out the possibility that the C-ter-     protein-protein interactions involving the divergence- <br /> minal domain of gp32 influences specificity to RNA by           prone C-terminal domain. It would be particularly <br /> interacting directly with this ligand, there are indications    interesting to find out if the gp32 sequence divergence <br /> that this negatively charged segment of gp32 is a modula-       near the DNA binding residue Y106 (Fig 2) is important <br /> tor of gp32 interactions with nucleic acids rather than a       for RNA recognition. X-ray crystallographic studies [20] <br /> carrier of nucleic acid binding determinants. In particular,    suggest that T4 gp32 residues T101-K110 constitute part <br /> a small deletion that maps within this protein segment          of the ssDNA-binding surface of the protein, which <br /> (&#226;&#710;&#8224;32PR201; Fig 2) exhibits altered specificity to other pro-    includes Y84, Y99, Y106 and the nearby &quot;LAST&quot; motif <br /> teins but has no effects on autogenous control of gp32          (residues 110&#226;&#8364;&#8220;114; 31). Also, as suggested by the 3D <br /> synthesis in vivo [39]. Also, recent studies with purified      structure, these residues are located within or very close to <br /> RB69 gp32 implicated the C-terminal domain of this              the Zn-binding domain of the protein; ie, the putative <br /> protein in regulating access of gp43 from the same phage        &quot;zinc-finger&quot; sequence Cys77-X3-His-X5-Cys-X2-Cys90, <br /> to binding sites in the ssDNA-binding module of the Ssb         which has counterparts in a number of RNA-binding pro- <br /> protein [40]. It has also been shown that in T4, the            teins [40]. The construction and analysis of RB69-T4 gp32 <br /> ssDNA-binding module of gp32 forms specific crosslinks          chimeras could help to establish if the divergence near <br />  <br />  <br />                                                                                                                    Page 12 of 14 <br />                                                                                               (page number not for citation purposes) <br />  Virology Journal 2004, 1:4                                                                   http://www.virologyj.com/content/1/1/4 <br />  <br />  <br />  <br /> Y106 is responsible for the observed differences in RNA           doknot sequence. Such lateral transfer events and <br /> footprints between T4gp32 and RB69 gp32 (Figs 6, 7, 8).           subsequent mutation may have profound influences on <br />                                                                   evolution of the RNA binding functions of proteins that <br /> In summary, we envisage that as a mediator of gp32&apos;s              have relaxed sequence but stringent structural require- <br /> interactions with other phage induced proteins, the C-ter-        ments for their RNA target. <br /> minal domain of gp32 may co-diverge with its protein tar- <br /> gets to maintain mutual recognition, and that structural          Competing interests <br /> plasticity of a conserved ssDNA-binding domain may                None declared. <br /> allow an also diverging RNA target to establish rearranged <br /> contacts within a relatively conserved protein pocket. It is      Authors&apos; contributions <br /> unclear if the 2 sets of divergence are interconnected, but       Jamilah Borjac-Natour: Conducted most of the experi- <br /> together, they could facilitate the evolution of a high           mental work and initial data analysis and prepared sum- <br /> degree of diversity in how the synthesis and/or replication       maries; wrote the first draft and participated in <br /> activity of this Ssb protein is regulated among phyloge-          subsequent revisions of the manuscript. Vasiliy Petrov: <br /> netic relatives of T4. It will be important to find out if this   Conducted independent analysis of data and generated <br /> diversity includes RNA ligands for gp32 that control the          summaries and composite figures for presentation in the <br /> DNA-binding activity but not synthesis of gp32, or if             manuscript. Participated in revision of the manuscript <br /> autogenous translational repression has been replaced by          during later stages of preparation. Jim Karam: Directed the <br /> other mechanisms for control of gene 32 in some T4 rela-          study, evaluated results on an ongoing basis, worked <br /> tives. There is at least one reported example where evolu-        closely with the coauthors during preparation of Figures, <br /> tion resulted in lack of RNA binding function in the Ssb          played a major role during revision of manuscript drafts <br /> protein of an M13-like phage [43]. Also, a scan of availa-        and communicated the manuscript to the journal. <br /> ble genomic sequences for T4-like phages http:// <br /> phage.bioc.tulane.edu/ reveals a high degree of sequence           <br /> divergence in the putative translational operator regions        <br /> of the corresponding gene 32 regions. In one case, phage           <br /> RB49 (GenBank NC_005066), it has been reported that         <br /> there are no indications that an RNA pseudoknot structure <br />                                                                   <br /> exists in the putative TIR for gene 32, although the UUAA/      <br /> UUAAA sequence units are conserved in the RB49 IC59-32             <br /> region [44]. It remains to be seen if gp32 from this and      <br /> other T4 like phages that appear to lack the RNA <br /> pseudoknot do bind their respective TIR regions or repress         <br /> their own translation.                                  <br /> Finally, we should <span id='am-3' about='rdfs:comment' typeof='owl:Thing'>comment</span> about ORF32.1 (Fig 1) and           <br /> its possible relevance to evolution of the mRNA target for     <br /> gp32. This ORF is present in some T4-like genomes (eg T4            <br /> and GenBank Ac No AF033323) and absent in others (eg,             <br /> RB69 and GenBank Ac No AY310907). Recently, it was           <br /> shown that T4 ORF 32.1 encodes a Seg-type (G1Y-YIG                <br /> family) homing endonuclease (now named SegG) that         <br /> mediates its own transfer, along with T4 gene 32, to the             <br /> ORF32.1-less genome of phage T2 in T4 &#195;&#8212; T2 genetic                  <br /> crosses. We note that the 5&apos; terminal sequence of the RNA        <br /> pseudoknot for T4 gp32 translational control overlaps the           <br /> reading frame of the segG gene, in addition to being very    <br /> similar (~83% identity) to the corresponding segment of                 <br /> the pseudoknot sequence of RB69, which lacks a segG       <br /> gene (Fig 3). Possibly, this portion of the RNA pseudo-           <br /> knot preexisted the entry of an ORF32.1-like sequence ele-        <br /> ment into the gene 59-32 intercistronic region of a T4                 <br /> progenitor and that the modern day segG gene (ORF32.1)       <br /> may be a chimera consisting of an extension of the paren-               <br /> tal segG reading frame into the recipient genome&apos;s pseu-                 <br />  <br />  <br /> </body></html>