<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>EUKARYOTIC CELL, Feb. 2005, p. 262&#226;&#8364;&#8220;273                                                                                               Vol. 4, No. 2 <br /> 1535-9778/05/$08.00&#226;&#171;&#185;0 doi:10.1128/EC.4.2.262&#226;&#8364;&#8220;273.2005 <br /> Copyright &#194;&#169; 2005, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />       Functional Specialization Chlamydomonas reinhardtii Cytosolic <br />           Thioredoxin h1 Response Alkylation-Induced <br />                                DNA Damage <br />         Nandita Sarkar,1 Ste&#194;&#180;phane Lemaire,2 Danxia Wu-Scharf,1 Emmanuelle Issakidis-Bourguet,2 <br />                                         Heriberto Cerutti1* <br />                       School Biological Sciences Plant Science Initiative, University Nebraska&#226;&#8364;&#8221;Lincoln, <br />                                  Lincoln, Nebraska,1 Institut Biotechnologie des Plantes, UMR <br />                                            8618 CNRS, Universite&#194;&#180; Paris-Sud, Orsay, France2 <br />                                               Received 25 July 2004/Accepted 8 November 2004 <br />  <br />              DNA damage occurs by-product intrinsic cellular processes, like DNA replication, conse- <br />           quence exposure genotoxic agents. Organisms evolved multiple mechanisms avoid, tolerate, or <br />           repair DNA lesions. <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> gain insight processes, isolated mutants hypersensitive DNA- <br />           damaging agents green alga Chlamydomonas reinhardtii. One mutant, Ble-1, showed decreased <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>survival</span> <br />           treated methyl methanesulfonate (MMS), bleomycin, hydrogen peroxide (H2O2) but <br />           behaved like wild type exposed UVC irradiation. Ble-1 carries extensive chromosomal <br />           deletion includes gene encoding cytosolic thioredoxin h1 (Trxh1). Transformation Ble-1 a <br />           wild-type copy Trxh1 fully corrected MMS hypersensitivity partly restored tolerance bleo- <br />           mycin. Trxh1 complemented defect repair MMS-induced DNA strand breaks alkali-labile <br />           sites. In addition, Trxh1&#226;&#8364;&#8220;&#226;?&#164;-glucuronidase fusion protein translocated nucleus response treatment <br />           MMS. However, somewhat surprisingly, Trxh1 failed correct Ble-1 hypersensitivity H2O2. <br />           Moreover, Trxh1 suppression <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-241' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-247' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-251' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-265' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>RNA interference</span> wild-type strain resulted enhanced sensitivity to <br />           MMS DNA repair defects increased cytotoxicity H2O2. Thioredoxins implicated in <br />           oxidative-stress responses organisms. Yet results indicate specific role Chlamydomonas Trxh1 <br />           repair MMS-induced DNA damage, dispensable response H2O2. These <br />           observations suggest functional specialization cytosolic thioredoxins Chlamydomonas <br />           isoform (Trxh2) does compensate lack Trxh1. <br />  <br />  <br />    Genome integrity stability, key components sur-                  helix-distorting DNA lesions (58). However, despite pref- <br /> vival organism, constantly threatened DNA dam-                    erential role certain systems repair specific lesions, <br /> age. Endogenous sources DNA lesions include, oth-                     DNA repair pathways partly redundant (4, 22, 58). In <br /> ers, replication errors, spontaneous depurination,                         addition, cells evolved DNA damage tolerance mecha- <br /> alterations caused reactive oxygen species (ROS) generated                  nisms allow replicative bypass <span id='am-316' about='xsp:base' typeof='owl:Thing'>base</span> damage, pro- <br /> during normal metabolism (83). DNA damage result                      cess called postreplication repair (22, 50, 65). <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> exposure environmental agents UV light, ion-                      Organisms developed scavenging mechanisms to <br /> izing radiation, chemical mutagens, including methyl                       detoxify genotoxic agents. For instance, ROS produced <br /> methanesulfonate (MMS), bleomycin, H2O2 (70). Many                         continuously by-products metabolic pathways but <br /> DNA lesions, left unrepaired, lead mutations, chro-                  toxicity minimized variety antioxidant systems, <br /> mosomal aberrations, aneuploidy, cell death (22, 58).                       depending glutathione thioredoxins (Trxs) re- <br />    A complex cellular composed intricate network                  ducing power (44). Trxs contain redox-active cysteine res- <br /> of surveillance repair pathways evolved monitor                 idues display main functions: (i) substrate for <br /> mend DNA damage. DNA lesions detected molecular                         catalytic enzymes like involved reduction ribo- <br /> sensors signal delay arrest cell cycle progression              nucleotides, methionine sulfoxide, peroxides (ii) reg- <br /> as array transcriptional DNA repair responses                ulators modulate activity functional proper- <br /> (83). DNA repair responses include direct repair, base excision                ties interacting proteins, including variety signaling and <br /> repair (BER), nucleotide excision repair (NER), mismatch                       transcription factors (2, 3, 51). Through activities, Trxs <br /> repair, DNA double-strand break (DSB) repair (22, 58).                     implicated ROS detoxification, redox-sensitive <br /> Base excision repairs oxidized, alkylated (usually methylated),                signal transduction, transcriptional activation stress re- <br /> or deaminated bases single-strand breaks (SSBs) (48),                      sponse genes, apoptosis (14, 35, 37). Trxs modu- <br /> whereas NER major repair removing bulky,                     late activity mammalian apurinic/apyrimidinic endo- <br />                                                                                nuclease 1 (APE-1)/Redox factor 1, multifunctional protein <br />                                                                                involved BER (26, 34). However, effect thioredoxins <br />   * Corresponding author. Mailing address: School Biological Sci- <br /> ences Plant Science Initiative, University Of Nebraska&#226;&#8364;&#8221;Lincoln, <br />                                                                                repair DNA damage remained unexplored. <br /> E211 Beadle Center, P.O. Box 880666, Lincoln, NE 68588-0666.                      Our understanding cellular responses DNA damage is <br /> Phone: (402) 472-0247. Fax: (402) 472-8722. E-mail: hcerutti1@unl.edu.         largely derived genetic biochemical studies animal, <br />  <br />                                                                          262 <br />  VOL. 4, 2005                                                             Trxh1 AND REPAIR OF ALKYLATION-INDUCED DNA DAMAGE                                           263 <br />  <br />  <br /> fungal, prokaryotic systems (22, 30, 58). By comparison,                           mapped restriction enzyme analysis. The longest (cosmid 1) was <br /> relatively little known DNA repair pathways plant                          cotransformed Ble-1 plasmid pJK7, containing genetically <br />                                                                                        engineered acetolactate synthase gene conferring resistance herbicide <br /> and algal systems, analysis completely se-                          sulfometuron methyl (36). However, herbicide-resistant transfor- <br /> quenced Arabidopsis thaliana genome revealed numerous ho-                              mants showed complementation Ble-1&#226;&#8364;&#8482;s hypersensitivity genotoxic agents. <br /> mologs yeast mammalian DNA repair genes (1, 5).                                 Moreover, Southern hybridization 1.4-kb EcoRI-XhoI fragment the <br /> Photosynthetic organisms evolved novel DNA                               3&#226;&#172;&#732; end cosmid 1 (distal cloned rs-3 flanking sequence) revealed large <br />                                                                                        chromosomal deletion Ble-1. Thus, combination genome walking and <br /> damage repair sensing transduction mechanisms since <br />                                                                                        Southern blot analyses (57, 78) used construct contig partly overlap- <br /> they face distinct challenges, recurring exposure                           ping cosmid clones spanned deleted region. A subset clones, as <br /> solar UV radiation DNA-damaging by-products photo-                              subclones, used complementation assays described <br /> synthetic metabolism (5). To gain insight pathways,                         above. Some subclones partially sequenced, putative transcrip- <br /> we used unicellular green alga Chlamydomonas rein-                            tional units identified searching National Center Biotechnology <br />                                                                                        Information (http://www.ncbi.nlm.nih.gov) Chlamydomonas genome <br /> hardtii model isolate insertional mutants sensi-                        (http://genome.jgi-psf.org/chlre2/chlre2.home.html) databases. <br /> tive DNA-damaging agents. We report character-                                DNA RNA analyses. Standard techniques used isolate nucleic <br /> ization mutant, named Ble-1 hypersensitivity                       acids Chlamydomonas cells (32, 57). Electrophoretic fractionation nu- <br /> to bleomycin.                                                                          cleic acids, transfer nylon membranes, hybridization 32P-labeled <br />    Ble-1 survival severely compromised exposure                              probes carried previously described (32, 57). <br />                                                                                           Immunoblot analysis. Cells resuspended sample buffer (30 mM Tris-HCl <br /> MMS bleomycin, behaved like wild type                            [pH 7.9], 1 mM phenylmethylsulfonyl fluoride) lysed French press cell <br /> was irradiated UVC light. Integration mutagenic                            1,380 lb/in2. After cellular debris pelleted consecutive centrifugations <br /> plasmid resulted deletion nearly 60 kb Ble-1                            20,000 &#226;&#171;&#187; g 20 min 45,000 &#226;&#171;&#187; g 45 min, soluble polypeptides were <br /> genome. Complementation Ble-1 cosmid subclones                                 recovered supernatant. Samples standardized total protein <br />                                                                                        concentration bicinchoninic acid assay (62). Next, 100-&#226;?&#174;g aliquots of <br /> identified gene encoding cytosolic isoforms of <br />                                                                                        proteins boiled 3 min gel loading buffer (10% glycerol, 1.4% sodium <br /> thioredoxin (Trxh1) responsible sensitivity MMS.                         dodecyl sulfate, 100 mM dithiothreitol, 30 mM Tris-HCl [pH 6.8]) fraction- <br /> Ble-1 deficient repair MMS-induced strand breaks                          ated sodium dodecyl sulfate&#226;&#8364;&#8220;15% polyacrylamide gels (57). Electrophoreti- <br /> and alkali-labile abasic sites, <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>phenotype</span> partly                     cally separated samples blotted nitrocellulose filters (Amersham) and <br /> corrected introduction genomic copy Trxh1. More-                            blocked Tris-buffered saline&#226;&#8364;&#8220;Tween 20 (TBS-T) buffer (10 mM Tris-HCl <br />                                                                                        [pH 7.4], 150 mM NaCl, 0.01% Tween 20) containing 5% nonfat dry milk. <br /> over, strains Trxh1 expression suppressed RNA                             Further steps performed TBS-T buffer containing 1% milk. Membranes <br /> interference (RNAi) showed MMS hypersensitivity de-                                incubated overnight 4&#194;&#176;C primary antisera Chlamydomonas <br /> fects DNA damage repair. Consistent role Trxh1                         Trxh1 reacted 1 h room temperature anti-rabbit immuno- <br /> the response DNA damage, fusion protein Trxh1                             globulin G conjugated horseradish peroxidase (Amersham). Signals were <br />                                                                                        visualized enhanced chemiluminescence (17). <br /> and Escherichia coli &#226;?&#164;-glucuronidase (GUS) localized predom- <br />                                                                                           Phenotypic characterization Ble-1, complemented strains, transgenic <br /> inantly cytosol normal conditions redistrib-                          lines Trxh1 expression suppressed RNAi. RNAi epi-mutants of <br /> uted nucleus following exposure genotoxic                            Trxh1 generated previously described (56). <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>Cell survival</span> exposure <br /> agents. Further, hypersensitivity MMS Saccharomy-                          MMS UVC irradiation examined reported (32). To test for <br /> ces cerevisiae trx1 trx2 double mutant complemented                           hypersensitivity H2O2 bleomycin, Chlamydomonas cells grown to <br />                                                                                        logarithmic phase, serially diluted, spotted TAP plates containing 1 mM <br /> by ectopic expression Chlamydomonas Trxh1, provided                            H2O2 1.5 &#226;?&#174;g bleomycin/ml. Cell growth evaluated 7 10 days of <br /> it contained intact redox catalytic site. Our findings indicate                     incubation moderate light. <br /> (i) role Chlamydomonas Trxh1 DNA repair pathways                                 DNA damage repair analysis. Cells logarithmic phase collected by <br /> coping MMS-induced abasic sites and/or SSBs (ii)                              centrifugation, resuspended TAP medium, treated 25 mM MMS for <br />                                                                                        30 min dark (8). Untreated control cells incubated TAP medium <br /> functional specialization Chlamydomonas cytosolic thiore- <br />                                                                                        MMS (8). Immediately afterwards, aliquots control MMS-treated <br /> doxins, Trxh2 does compensate deficiency                          cells frozen isolation DNA corresponding zero time points. <br /> Trxh1.                                                                                 After washes TAP medium remove unreacted MMS, remaining <br />                                                                                        cells allowed recover dark, moderate shaking, different <br />                                                                                        time periods aliquots frozen described (8). Genomic DNA <br />                          MATERIALS AND METHODS <br />                                                                                        isolated frozen samples, separated denaturing gel electrophore- <br />    Culture conditions, Ble-1 isolation, genetic analysis. Unless noted other-      sis (8), hybridized 32P-labeled probe corresponding TOC1 <br /> wise, C. reinhardtii cells grown moderate light Tris-acetate-phos-       retroelement (32) present 15 20 copies haploid nuclear genome. <br /> phate (TAP) medium (28). To isolate insertional mutants hypersensitive              The distribution radioactivity lane quantified phosphorim- <br /> DNA-damaging agents, wild-type strain CC-124 transformed glass          ager Quantity One software (Amersham). For analysis DNA repair by <br /> bead procedure plasmid containing mutant form protoporphyrinogen           PCR (33), cells treated 5 10 mM MMS incubated described <br /> oxidase (rs-3 marker) conferring resistance diphenyl ether herbicides (32).         above. Thirty nanograms genomic DNA used amplification a <br /> Herbicide-resistant transformants tested ability survive      2.1-kb fragment corresponding Chlamydomonas Lsm5 (Like Sm 5) gene <br /> presence 1.5 &#226;?&#174;g bleomycin (Invitrogen)/ml 2.5 mM MMS (Sigma). By              primers Mut3-1 (5&#226;&#172;&#732;-AGAGCTAGGGACCGTGGAGT-3&#226;&#172;&#732;) Mut3-2 (5&#226;&#172;&#732;- <br /> using approach, recovered mutant strain, Ble-1, sensitive geno-      TGTTCTCTGTTGCTTGTCTGACG-3&#226;&#172;&#732;). The number cycles showing lin- <br /> toxic agents. For genetic analysis, Ble-1 crossed wild-type strain   ear relationship input DNA final product determined in <br /> opposite mating type (CC-125) tetrads dissected previously de-             preliminary experiments. The PCR conditions consisted 30 cycles 93&#194;&#176;C for <br /> scribed (27). The phenotype meiotic tetrad products evaluated spot       30 s, 55&#194;&#176;C 30 s, 71&#194;&#176;C 180 s. Five-microliter aliquots PCR <br /> tests TAP medium containing 2.5 mM MMS (32). Five-microliter aliquots            resolved 1% agarose gels visualized ethidium bromide staining, <br /> appropriately diluted cells pipetted plates incubated pre-        signal intensities quantified Quantity One software (Bio-Rad). <br /> viously reported (32).                                                                 DNA lesion frequency calculated, assuming random Poisson distribution, <br />    Plasmid rescue, cosmid library screening, deletion mapping, sequence            previously described (11, 33). Under experimental conditions used (i.e., <br /> analyses. The genomic sequence flanking end integrated rs-3 marker          incubation concentrated cells dark) MMS treatment, DNA <br /> in Ble-1 recovered plasmid rescue E. coli (32). A 1.5-kb BamHI-NotI          replication associated cell cycle progression asynchronous Chlamy- <br /> fragment flanking DNA used probe screen Chlamydo-              domonas cultures increased DNA content 1.5-fold 24 h <br /> monas genomic library (57, 78). Twelve hybridizing cosmid clones isolated         (data shown). <br />  264       SARKAR ET AL.                                                                                                                  EUKARYOT. CELL <br />  <br />    Subcellular localization GUS-Trxh1 fusion protein transiently trans- <br /> formed onion epidermal cells. The coding sequence Trxh1 amplified by <br /> PCR full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> cDNA plasmid CrTRXh1 (64) primers Trx-cod-3&#226;&#172;&#732; <br /> (5&#226;&#172;&#732;-CATGCCATGGCCGCGGCGGCGTGCTTGGC-3&#226;&#172;&#732;, adding NcoI site) <br /> and Trx-cod-5&#226;&#172;&#732; (5&#226;&#172;&#732;-CATGCCATGGGCGGTTCTGTTATTGTG-3&#226;&#172;&#732;, adding an <br /> NcoI site). The Trxh1 PCR fragment cloned frame GUS <br /> start codon plasmid pPTN134 (82). The transgenes GUS-Trxh1 GUS <br /> alone transformed onion epidermal cells microprojectile bombard- <br /> ment DNA-coated tungsten particles (82). After bombardment, cells were <br /> allowed recover 20 h regular Murashige Skoog (MS) medium on <br /> MS medium containing 1 mM MMS, 0.2 mM H2O2, 0.5 &#226;?&#174;g bleomycin/ml. <br /> Some epidermal peels, 20 h recovery MS medium, incubated for <br /> 1 h liquid MS medium containing 8 mM MMS. GUS activity detected by <br /> staining X-glucuronide, nuclei identified propidium <br /> iodide (PI) (82). Stained cells observed bright-field microscopy for <br /> distribution GUS activity epifluorescent microscopy PI labeling of <br /> the nucleus (71, 82). <br />    Plasmid construction expression Chlamydomonas Trxh1 S. cerevi- <br /> siae. EMY63 (MATa ade2-1 ade3-100 his3-11 leu2-3 lys2-801 trp1-1 ura3-1 <br /> trx1::TRP1 trx2::LEU2), yeast strain genes encoding cytosolic Trxs <br /> deleted, previously described (47). EMY63 cells transformed by <br /> the lithium chloride method (24) constructs conditional expression S. <br /> cerevisiae Trx1 C. reinhardtii Trxh1 (CrTrxh1) wild-type redox- <br /> inactive C36S forms (25). Each thioredoxin sequence amplified the <br /> corresponding cDNA PCR. The upstream primer allowed introduction an <br /> MluI site start codon position corresponding N terminus of <br /> each protein, downstream primer allowed introduction BamHI <br /> restriction site stop codon. The PCR products cloned the <br /> control yeast Gal1 promoter centromere plasmid YCpGal2 <br /> containing URA3 selectable marker (12). Sequence-verified recombinant <br /> plasmids transformed EMY63, clones selected main- <br /> tained standard synthetic minimal medium (lacking uracil) supplemented <br /> with 2% glucose carbon source. For induction recombinant Trx expres- <br /> sion, 2% galactose substituted glucose. <br />    Examination tolerance yeast strains MMS H2O2. Transformed <br /> EMY63 cells grown mid-log phase glucose medium 30&#194;&#176;C              FIG. 1. Hypersensitivity MMS cosegregates tagging rs-3 <br /> diluted transferred inducing galactose medium. Growth continued        marker. (A) Growth survival cells TAP medium TAP <br /> 30&#194;&#176;C, cells mid-log phase used tolerance tests. After dilution    medium containing 2.5 mM MMS (TAP &#226;&#171;&#185; MMS). The parental strains <br /> an optical density 600 nm 0.2, cell growth survival presence     meiotic products tetrads cross the <br /> genotoxic agents evaluated halo assay (31). Cells mixed         wild-type (CC-125) mutant Ble-1 strains shown. Strains grown <br /> galactose agar spread plates obtain uniform lawn. Disks con-         logarithmic phase diluted TAP medium 5 &#226;&#171;&#187; 104 cells per <br /> taining 10 &#226;?&#174;l MMS (1.36 M) H2O2 (500 mM) placed center          5 &#226;?&#174;l, spotted plates, incubated described Materials and <br /> the yeast lawns, cell growth monitored 3 5 days incubation    Methods. CTRL, control. (B) Southern blot analysis indicated <br /> 30&#194;&#176;C.                                                                                strains. Genomic DNA digested HindIII probed the <br />    Flow cytometry analysis. Yeast cells grown synthetic minimal medium       pBluescript sequence, vector backbone rs-3 tagging plasmid. <br /> (lacking uracil) supplemented 2% galactose optical density 600 nm <br /> of 0.5, centrifuged, washed 2 ml 50 mM Tris-HCl (pH 8). Cells were <br /> then fixed 70% ethanol 1 h room temperature, centrifuged, resus- <br /> pended 1 ml Tris-HCl buffer containing 1 mg RNase A/ml. After <br />                                                                                      containing DNA-damaging agents. Since Chlamydomonas is <br /> incubation 2 h 37&#194;&#176;C, cells pelleted (12,000 &#226;&#171;&#187; g, 1 min), resuspended     haploid, nonlethal mutations genes required DNA dam- <br /> in 1 ml Tris-HCl buffer containing PI (50 mg/ml), allowed stain     age repair tolerance result reduced survival the <br /> dark 4&#194;&#176;C overnight mild agitation. Analysis performed fluo-        presence genotoxic agents. By using approach, iso- <br /> rescence-activated cell sorter (Vantage; Becton Dickinson, Le pont Claix, <br />                                                                                      lated mutant strain (Ble-1) sensitive bleomycin <br /> France). Nuclei excited 488 nm argon laser (Spectra-Physics, <br /> Mountain View, Calif.), FL1-Height FL1-Area collected           MMS. Ble-1 contained single, partly rear- <br /> a band-pass filter allowing light 620 630 nm reach detector.      ranged, copy rs-3 plasmid integrated nuclear <br /> Ten thousand nuclei analyzed sample. Data collected               genome (Fig. 1 data shown). <br /> Cellquest software (Becton Dickinson, Mansfield, Mass.) analyzed               To test mutant phenotypes cosegregated with <br /> MODFIT (Verity Software House, Inc., Topsham, Maine). <br />                                                                                      rs-3 marker, Ble-1 crossed wild-type strain of <br />                                                                                      opposite mating type, CC-125. The meiotic tetrad products <br />                                                                                      examined survival medium containing MMS or <br />                                   RESULTS <br />                                                                                      bleomycin (Fig. 1A data shown). Only tetrad products <br />   Isolation genetic analysis Ble-1. To identify C. rein-                      containing rs-3 gene, detected Southern blot hybrid- <br /> hardtii genes involved cellular response DNA damage,                       ization (Fig. 1B), hypersensitive MMS (Fig. 1A). In <br /> we carried random insertional mutagenesis wild-                           contrast, tetrad products did carry integrated <br /> type strain CC-124 (28). Cells CC-124 transformed                          mutagenic plasmid behaved like wild type (Fig. 1). The <br /> with rs-3 gene, encodes mutated form proto-                           analysis 10 complete tetrads indicated hypersensitivity <br /> porphyrinogen oxidase, conferring resistance diphenyl ether                       MMS bleomycin segregated single Mendelian locus <br /> herbicides (32). Herbicide-resistant transformants                         genetically linked (within map units) integrated rs-3 <br /> tested replica plating ability grow media                         marker. <br />  VOL. 4, 2005                                            Trxh1 AND REPAIR OF ALKYLATION-INDUCED DNA DAMAGE                        265 <br />  <br />  <br />    Cloning identification disrupted gene conferring      hypothesis, transformed wild-type Chlamydomonas <br /> MMS hypersensitivity Ble-1. The chromosomal sequence             cells inverted repeat constructs designed produce dou- <br /> flanking end introduced rs-3 gene obtained        ble-stranded RNA homologous Trxh1 (56). In inde- <br /> plasmid rescue (66) used probe screen Chlamy-         pendent transformants, Trxh1 transcript levels specifically <br /> domonas genomic library. Although partly overlapping        down-regulated, mRNA amounts closely re- <br /> cosmid clones isolated, complemented Ble-1            lated cytosolic thioredoxin h2 gene (Trxh2) (40) remained un- <br /> mutant phenotype (Fig. 2A, e.g., Cosmid 1). These clones       perturbed (Fig. 2D, Trxh1-IR-3 Trxh1-IR-4). Like Ble-1, <br /> mapped restriction enzyme digestion, end          RNAi strains (Trxh1 epi-mutants) hypersensitive to <br /> longest (distal cloned rs-3 flanking sequence)       MMS treatment (Fig. 2E). However, showed mild <br /> used probe Southern hybridization analysis Ble-1        survival defects grown bleomycin-contain- <br /> and CC-124. Lack hybridization sequence Ble-1         ing medium (data shown). These results, taken together, <br /> DNA revealed integration rs-3 marker caused           suggested role cytosolic Trxh1 isoform cellular <br /> large deletion nuclear genome mutant strain (Fig.     response certain genotoxic agents MMS. Yet, Trxh1 <br /> 2A data shown). By using genome walking approach,         did fully complement defect survival Ble-1 <br /> we isolated overlapping set cosmids encom-          exposed bleomycin H2O2 (see below). There- <br /> passed Ble-1 chromosomal deletion (Fig. 2A). The left end       fore, hypothesize disruption yet-to-be- <br /> of deletion precisely defined sequencing         identified gene(s) 60-kb deletion responsible for <br /> chromosome-plasmid junction. The right junction se-         phenotypes. <br /> quenced, restriction enzyme mapping cosmid contig           Effect genotoxic agents cell survival DNA damage <br /> together Southern blot analyses wild-type mu-       repair Ble-1, Trxh1-complemented strains, the <br /> tant strains indicated deleted chromosomal region          Trxh1 RNAi-induced epi-mutants. To gain insight mo- <br /> spans approximately 60 kb (Fig. 2A data shown).             lecular role Trxh1, exposed cells variety genotoxic <br />    To identify gene(s) responsible hypersensitivity     agents causing different kinds DNA lesions requiring <br /> of Ble-1 DNA-damaging agents, individual cosmid clones           distinct pathways repair. In cases, compared the <br /> were tested ability complement mutant pheno-       survival wild-type CC-124 strain, Ble-1 mutant, a <br /> types. Ble-1 cells cotransformed cosmid clone        strain complemented cosmid 4 [Ble-1(Cos4)-20], a <br /> and plasmid pJK7, encoding resistance herbicide         strain complemented Trxh1-containing 3-kb PstI frag- <br /> sulfometuron methyl (36). Herbicide resistance transformants        ment [Ble-1(Trxh1)-33]. All strains behaved similarly the <br /> were examined survival medium containing          wild type irradiated UVC light al- <br /> MMS bleomycin. This analysis showed cosmid 4 com-           lowed recover nonphotoreactivating conditions (Fig. <br /> plemented hypersensitivity Ble-1 MMS         3A). In contrast, Ble-1 sensitive treatment with <br /> a partial effect restoring bleomycin tolerance (Fig. 2A      MMS phenotype nearly fully complemented by <br /> 3). To define precisely gene required phenotypic       ectopic expression Trxh1 (Fig. 3A). In addition, already <br /> correction, cosmid 4 subclones cotransformed           discussed, strains Trxh1 suppressed RNAi dis- <br /> into Ble-1. In addition, partial sequence analysis cosmid 4      played hypersensitivity MMS (Fig. 2E). Ble-1 survival was <br /> revealed included Trxh1 gene, encodes cyto-       compromised exposure H2O2 bleomycin (Fig. 2E <br /> solic thioredoxin h1. Therefore, tested complemen-      3B). However, bleomycin sensitivity partly re- <br /> tation capability 3-kb PstI fragment exclusively containing    versed transformation wild-type copy Trxh1 (Fig. <br /> Trxh1 (64). In cases, transformation Ble-1 fragments    3B), defect H2O2 tolerance corrected <br /> that included full-length Trxh1 gene reversed MMS hy-         (Fig. 2E). Conversely, down-regulation Trxh1 expression by <br /> persensitivity, survival defect exposure bleo-      RNAi caused mild hypersensitivity bleomycin (data not <br /> mycin partly corrected (Fig. 2A 3).                    shown) did affect survival presence H2O2 <br />    To evaluate Trxh1 required toler-        (Fig. 2E). Somewhat surprisingly, given known role of <br /> ance genotoxic agents, examined expression         thioredoxins oxidative-stress response (14, 35, 38), these <br /> wild-type strain mutant Ble-1 strain,        findings indicated Trxh1 plays key role cellular pro- <br /> strains complemented 3-kb PstI fragment, Ble-              tection MMS and, extent, bleomycin has <br /> 1(Trxh1)-10 Ble-1(Trxh1)-33. Northern blot analysis to-      apparent effect tolerance H2O2. <br /> tal RNA revealed detectable Trxh1 transcripts Ble-1,             To test Trxh1 required repair MMS- <br /> whereas complemented strain, Ble-1(Trxh1)-33, RNA           induced DNA damage, cells briefly treated this <br /> levels similar wild type (Fig. 2B). The       chemical allowed recover different periods of <br /> complemented strain, Ble-1(Trxh1)-10, slightly reduced          time. The extent induced residual DNA damage was <br /> amounts Trxh1 mRNA comparison CC-124 (Fig.               evaluated alkaline gel electrophoresis (8). Under con- <br /> 2B). Corresponding variations Trxh1 protein levels          ditions, SSBs DSBs alkali-labile sites (i.e., <br /> observed strains immunoblot assay total           abasic sites) arising BER (21) detectable en- <br /> protein extracts probed anti-Trxh1 antibody (Fig. 2C).      hanced electrophoretic mobility fragmented DNA. Exami- <br /> Interestingly, Trxh1 expression level independently com-     nation DNA isolated immediately treatment re- <br /> plemented strains negatively correlated sensitivity   vealed similar extents MMS-induced nuclear DNA damage <br /> MMS (data shown). Moreover, Trxh1 required            wild-type, Ble-1, Ble-1(Trxh1)-33 strains (Fig. 4). <br /> tolerance MMS exposure, Trxh1 suppression RNAi                However, incubation drug-free medium, Ble-1 cells <br /> should result phenotype similar Ble-1. To test      repaired DNA strand breaks slowly comparison with <br />  266     SARKAR ET AL.                                                                                                       EUKARYOT. CELL <br />  <br />  <br />  <br />  <br />   FIG. 2. Genomic structure region affected integration rs-3 marker Ble-1 phenotypic defects associated lack of <br /> cytosolic Trxh1. (A) The open-box diagram represents chromosomal DNA, dashed line immediately indicates deletion &#226;&#172;&#402;60 <br /> kb Ble-1. The 3&#226;&#172;&#732; end deletion (dotted line) precisely defined. Genes encoding hybrid-cluster protein (HCP) Trxh1 <br /> (Trxh1) depicted solid boxes. Cosmids cosmid subclones used complementation Ble-1&#226;&#8364;&#8482;s hypersensitivity MMS also <br /> indicated. Southern blots Ble-1 wild-type CC-124 strains shown diagram. Genomic DNA digested indicated <br /> enzymes hybridized different sequences (depicted solid bars) examine deleted chromosomal region. Restriction enzyme sites: <br /> B, BamHI; E, EcoRI; P, PstI; X, XhoI. (B) Northern blot analysis CC-124, Ble-1, independent transformants Ble-1 complemented <br /> with 3-kb PstI fragment containing Trxh1 [Ble-1(Trxh1)-10 Ble-1(Trxh1)-33]. Total cell RNA separated denaturing agarose gel <br /> electrophoresis sequentially probed coding sequence Trxh1 (top panel) coding sequence small subunit the <br /> Rubisco gene (RbcS2, panel) control equal loadings lanes. (C) Immunoblot analysis total soluble proteins, the <br /> indicated strains, probed polyclonal antibodies raised Chlamydomonas Trxh1 (top panel) Rubisco holoenzyme (bottom panel). <br /> LS, large subunit Rubisco. (D) RNA gel blot analysis CC-124 independent strains Trxh1 expression suppressed RNAi <br /> (Trxh1-IR-3 Trxh1-IR-4). Total cell RNA fractionated sequentially hybridized Trxh1 probe (top panel), coding sequence <br /> of cytosolic thioredoxin h2 gene (Trxh2) (middle panel), fragment 25S rRNA gene (bottom panel) control comparable <br /> sample loadings. (E) Growth survival indicated strains TAP medium TAP medium containing 2.5 mM MMS (TAP &#226;&#171;&#185; MMS) <br /> or 1 mM hydrogen peroxide (TAP &#226;&#171;&#185; H2O2). <br />  VOL. 4, 2005                                                 Trxh1 AND REPAIR OF ALKYLATION-INDUCED DNA DAMAGE                             267 <br />  <br />  <br />  <br />  <br />    FIG. 3. Effect genotoxic agents survival mutant <br /> (Ble-1), wild-type strain (CC-124), transformants Ble-1 <br /> containing 3-kb Trxh1 fragment [Ble-1(Trxh1)-33] cosmid <br /> 4 [Ble-1(Cos4)-20]. (A) The panels survival cells grown on <br /> TAP medium containing increasing concentrations MMS ex- <br /> posed increasing levels UVC irradiation. Each graph point rep- <br /> resents mean (&#226;&#171;&#190; standard deviation) results replicates <br /> (three independent experiments). Where error bars visible, <br /> they smaller symbols [&#228;&#352;?, CC-124; F, Ble-1; &#197;&#8217;, Ble- <br /> 1(Cos4)-20; &#227;&#8364;&#171;, Ble-1(Trxh1)-33]. (B) Growth survival indi- <br /> cated strains TAP medium (TAP &#226;&#171;&#185; Ble) (TAP) 1.5 <br /> &#226;?&#174;g bleomycin/ml. <br />  <br />  <br />  <br /> the wild type, indicated smaller average molecular <br /> mass DNA molecules time point (Fig. 4). The <br /> complemented strain, Ble-1(Trxh1)-33, showed intermedi- <br /> ate phenotype (Fig. 4). This partial correction DNA <br /> repair capacity differs nearly reversal sur- <br /> vival defect strain exposed MMS (Fig. <br /> 3A).                                                                        FIG. 4. Analysis nuclear DNA repair MMS-induced dam- <br />    MMS induces high levels N-methylpurines secondary              age wild-type, Ble-1, Ble-1(Trxh1)-33 strains. (A) Southern <br /> lesions resulting DNA damage processing replica-                blot showing DNA repair exposure 25 mM MMS 30 min. <br />                                                                          Genomic DNA isolated untreated control cells (CTRL) and <br /> tion, SSBs DSBs (21). We speculate that,              MMS-treated cells immediately treatment (0 h) or <br /> higher MMS concentration (25 mM) used cause DNA dam-                  recovery absence MMS 6, 12, 24 h. The DNA was <br /> age repair experiments (in comparison concen-            separated alkaline gel electrophoresis hybridized se- <br /> trations employed test cell survival), greater proportion       quence corresponding TOC1 transposable element. (B) In the <br />                                                                          kind experiment described above, relative distribution of <br /> of MMS-induced lesions corresponds secondary DSBs. In-                radioactivity lane (indicative DNA mass distribution) <br /> deed, relatively high concentrations, MMS behaves                analyzed phosphorimager plotted function of <br /> radiomimetic agent manner similar DSB-induc-             migration distance. Graphs represent average results two <br /> ing bleomycin (43). Hence, considering Ble-1 shows             independent experiments. CC-124 indicated dotted lines, Ble-1 is <br /> hypersensitivity bleomycin phenotype            indicated solid lines, Ble-1(Trxh1)-33 indicated dashed <br />                                                                          lines. The vertical solid line indicates average molecular mass of <br /> partly corrected expression Trxh1, Ble-1 likely             damaged DNA (similar strains) immediately MMS treat- <br /> defective repair DSBs, deficiency            ment. <br /> consequence lack Trxh1. <br />    We tested role Trxh1 repair DNA <br />  268     SARKAR ET AL.                                                                                                      EUKARYOT. CELL <br />  <br />  <br />                                                                          repair alkali-labile abasic sites and/or SSBs induced treat- <br />                                                                          ment alkylating agents. <br />                                                                             Relocalization Trxh1-GUS fusion protein nucleus <br />                                                                          cells exposed genotoxic agents. The Chlamydomonas <br />                                                                          Trxh1 isoform does contain canonical nuclear localiza- <br />                                                                          tion signal, assumed localize cytosol <br />                                                                          (64). However, Trxh1 directly involved DNA repair, <br />                                                                          expected localize nucleus and/or re- <br />                                                                          distribute nucleus response treatment geno- <br />                                                                          toxic agents. To test hypothesis, examined subcel- <br />                                                                          lular partitioning fusion polypeptide consisting the <br />                                                                          Trxh1 coding sequence linked N terminus E. coli <br />                                                                          GUS protein. A transgene expressing GUS used as <br />                                                                          control, 68-kDa polypeptide largely excluded <br />                                                                          nucleus (71). Both constructs placed the <br />                                                                          control Cauliflower mosaic virus 35S promoter in- <br />                                                                          troduced onion epidermal cells particle bombardment. <br />                                                                          Onion epidermal peels used transient gene expres- <br />                                                                          sion large, transparent (chlorophyll-less) <br />                                                                          cells facilitate imaging subcellular structures (71). In <br />                                                                          addition, previously demonstrated cor- <br />                                                                          rect subcellular localization Chlamydomonas fusion proteins <br />    FIG. 5. Repair MMS-induced DNA lesions wild type            (79, 82). <br /> (CC-124) Trxh1 RNAi epi-mutant (Trxh1-IR-3) examined            After particle bombardment, onion peels incubated <br /> semiquantitative PCR. (A) Amplification 2.1-kb genomic DNA <br /> fragment (Lsm5) using, template, DNA isolated cells       18 20 h MS medium MMS, bleomy- <br /> immediately treatment 10 mM MMS (0 h) allowing       cin, H2O2. In cases, epidermal peels incubated <br /> the cells recover 6 12 h absence MMS. Untreated      overnight MS transferred short period to <br /> control cells (CTRL) analyzed. The amplified products     medium containing genotoxic agents. The subcellular distribu- <br /> resolved agarose gel electrophoresis stained ethidium <br /> bromide. (B) Relative amplification 2.1-kb fragment calculated    tion expressed proteins determined histochemically <br /> by dividing amplification damaged samples (AD)        X-glucuronide assay (71). Regardless treatment, the <br /> by amplification nondamaged controls (AC). Each       GUS polypeptide predominantly localized cyto- <br /> graph point represents mean (&#226;&#171;&#190; standard error) results   plasm onion cells (Fig. 6B D). Likewise, absence <br /> independent experiments. Symbols: &#197;&#8217;, CC-124; &#226;&#8211;&#160;, Trxh1-IR-3. <br />                                                                          genotoxic agents treatment H2O2, Trxh1- <br />                                                                          GUS protein largely cytosol (Fig. 6A C). <br />                                                                          In contrast, treatment MMS bleomycin, fusion <br /> lesions induced lower concentration MMS, com-                polypeptide showed dual localization, nucleus and <br /> parable employed survival experiments (see Ma-            cytoplasm, majority examined cells (Fig. 6A <br /> terial Methods), using semiquantitative PCR ap-                 C data shown). Thus, consistent role of <br /> proach (11, 33). The ability DNA fragment support                Trxh1 DNA repair, data indicated protein <br /> PCR amplification indicator vivo intactness,       relocalizes nucleus response certain DNA-damag- <br /> DNA sequences containing DNA polymerase-blocking -ter-                ing agents. <br /> minating lesions amplified assay (11, 33).              Functional complementation S. cerevisiae trx1 trx2 dou- <br /> However, removal DNA lesions DNA repair                  ble mutant expression C. reinhardtii Trxh1. Budding <br /> enhance template integrity genomic DNA, enabling                  yeast contains genes encoding cytoplasmic thioredoxins <br /> more PCR amplification. The MMS treatment generated sim-                 (Trx1 Trx2), dispensable normal growth <br /> ilar levels DNA lesions wild-type Trxh1 RNAi               conditions (47). However, deletion Trx genes results in <br /> epi-mutants (Fig. 5), suggesting Trxh1 does play              hypersensitivity oxidative agents defects cell cycle, <br /> major role detoxification MMS prevention                particularly prolonged S phase (47, 69). Trx functions a <br /> alkylation DNA damage. In contrast, strains Trxh1              reductant ribonucleotide reductase, essential enzyme in <br /> expression suppressed RNAi showed lower rate                 deoxyribonucleotide biosynthesis (69), alterations in <br /> DNA repair (measured recovery amplification signal             cell cycle result reduced levels deoxyribo- <br /> at 6 12 h treatment) comparison wild type          nucleotides (46). In fact, glutaredoxin function an <br /> (Fig. 5). It unlikely increase DNA amplification          electron donor ribonucleotide reductase (54) thi- <br /> over time results predominantly replication intact           oredoxin glutathione-glutaredoxin systems appear have <br /> DNA molecules since, conditions used,              overlapping functions, required S. cerevi- <br /> absence treatment genotoxic agents, measured DNA                 siae viability (16, 68). The actual role Trxs yeast cell <br /> replication 1.5-fold 24 h (data shown).          cycle remained elusive, hypothesize lack of <br /> Therefore, results suggest Chlamydomonas Trxh1               cytosolic Trxs result increased DNA damage endo- <br /> required, directly indirectly (see discussion),        genously produced ROS and/or slower repair spontaneous <br />  VOL. 4, 2005                                                 Trxh1 AND REPAIR OF ALKYLATION-INDUCED DNA DAMAGE                                269 <br />  <br />  <br />  <br />  <br />   FIG. 6. Subcellular distribution GUS GUS-Trxh1 fusion proteins transiently transformed onion epidermal cells. Polypeptides were <br /> localized histochemically X-glucuronide assay. (A) Representative cellular staining patterns corresponding GUS-Trxh1. Onion epidermal <br /> peels bombarded DNA-coated tungsten particles incubated MS medium (No treatment) indicated concentrations <br /> of MMS, bleomycin (Ble), hydrogen peroxide (H2O2). Tissues simultaneously analyzed X-glucuronide staining (left panels, blue color) <br /> and nucleus-specific PI staining (right panels, orange color). Nuclei indicated arrowheads. (B) Representative staining patterns onion <br /> epidermal cells transiently transformed GUS treated described panel A. Panels stained X-glucuronide (left) PI (right) are <br /> shown. (C) Frequency analysis GUS-Trxh1 subcellular distribution indicated treatments. Transformed cells classified showing <br /> exclusive cytoplasmic localization (black bars) dual, nuclear, cytoplasmic localization (red bars) GUS activity. The results the <br /> averages (&#226;&#171;&#190; standard deviations) results independent experiments (300 cells analyzed treatment). (D) Frequency analysis GUS <br /> subcellular localization indicated treatments described panel C. <br />  <br />  <br />  <br /> DNA lesions activate S-phase DNA damage check-                    monas Trxh1 yeast-replicating vector, control <br /> point, preventing entry mitosis (42, 67).                             galactose-inducible promoter, resulted reversion of <br />    Little known potential function yeast Trxs             MMS hypersensitivity. However, mutant form Chlamydo- <br /> response monofunctional alkylating agents, Trx2                monas Trxh1 cysteine residue redox-active site <br /> and thioredoxin reductase 1 gene transcriptionally ac-             replaced serine (C36S) (25) failed complement this <br /> tivated MMS treatments (10, 35). Therefore, examined                 phenotype (Fig. 7A). In flow cytometry analysis asynchro- <br /> whether S. cerevisiae trx1 trx2 double mutant hypersen-            nous cultures, yeast trx1 trx2 mutant displays consid- <br /> sitive variety genotoxic agents Chlamydo-              erably lengthened S phase apparent marked <br /> monas Trxh1 complement phenotypic deficiencies,                  decrease proportion cells having G1 (1N) or, a <br /> suggestive evolutionary conservation function. S. cerevi-            lower degree, G2 (2N) DNA content (47). Expression of <br /> siae cells Trx1 Trx2 deleted showed defects               Chlamydomonas Trxh1 partly corrected deficiency cell <br /> survival (although different degrees) ex-                cycle progression, provided protein contained the <br /> posed MMS H2O2 (Fig. 7A). Expression Chlamydo-                    wild-type cysteine residues catalytic site (Fig. 7B). Thus, <br />  270     SARKAR ET AL.                                                                                                     EUKARYOT. CELL <br />  <br />  <br />                                                                           unknown (23, 38). In mammals yeast, Trxs <br />                                                                           clearly involved response oxidative stress via <br />                                                                           ROS-scavenging mechanisms modulation activity of <br />                                                                           signaling transcription factors (14, 35, 37). Plant and <br />                                                                           Chlamydomonas cytosolic Trxs interact ROS-detoxi- <br />                                                                           fying enzymes, ascorbate peroxidase, catalase, gluta- <br />                                                                           thione peroxidase, peroxiredoxins, superoxide dismutase <br />                                                                           (3, 23, 41, 80). However, Arabidopsis Trxh5 been <br />                                                                           implicated vivo response oxidative stress, whereas <br />                                                                           Trxh genes induced oxidative agents (38, 53). <br />                                                                           Moreover, expression studies Arabidopsis differential <br />                                                                           complementation thioredoxin-deficient phenotypes yeast <br />                                                                           suggested functional specialization plant cytosolic <br />                                                                           Trxs (38, 45, 53). Yet, issue difficult address <br />                                                                           experimentally mutants deficient individual Trxs <br />                                                                           obvious phenotype, effect attributed <br />                                                                           compensation thioredoxins and/or glutaredoxins <br />                                                                           (38). <br />                                                                              While conducting screen Chlamydomonas mutants hy- <br />                                                                           persensitive DNA-damaging agents, isolated strain <br />                                                                           (Ble-1) large genomic deletion included Trxh1. <br />                                                                           Ble-1 showed hypersensitivity treatment MMS, bleo- <br />                                                                           mycin, H2O2. The strain&#226;&#8364;&#8482;s poor tolerance MMS and, a <br />    FIG. 7. Phenotypic complementation S. cerevisiae trx1 trx2       lower degree, deficient survival presence bleomycin <br /> double mutant (EMY63) expression Chlamydomonas Trxh1.               complemented transformation wild-type copy of <br /> (A) Expression Trxh1, control galactose-inducible       Trxh1. In contrast, hypersensitivity H2O2 res- <br /> promoter centromeric plasmid restored tolerance mu- <br /> tant MMS extent transformation S. cerevisiae     cued ectopic expression Trxh1. Conversely, wild- <br /> Trx1 did (left panel). In contrast, Trxh1 did complement hy-      type cells, suppression Trxh1 expression RNAi resulted in <br /> persensitivity H2O2 (right panel). Yeast cells transformed   reduced survival exposure MMS and, lower ex- <br /> the plasmid (YCpGal2), S. cerevisiae Trx1 (Yeast Trx1),        tent, bleomycin enhanced sensitivity H2O2. These <br /> with Chlamydomonas Trxh1 (CrTrxh1), mutant form Trxh1 <br /> lacking catalytically active cysteine (CrTrxh1 C36S). Cells were <br />                                                                           findings indicated (i) unidentified gene(s) the <br /> plated, forming lawn, exposed gradient MMS H2O2          deleted chromosomal region Ble-1 likely responsible for <br /> concentrations, established diffusion centrally placed disk     bleomycin virtually H2O2 hypersen- <br /> containing 1.36 M MMS 500 mM H2O2. The size halo          sitivity (ii) Trxh1 essential cellular response to <br /> cell growth suppressed used estimation sensitivity   certain DNA-damaging agents, including monofunctional al- <br /> to genotoxic agents. The results averages (&#226;&#171;&#190; standard devi- <br /> ations) independent experiments. (B) Relative DNA        kylating chemicals MMS, required for <br /> content asynchronous cultures transformed trx1 trx2 mutant cells.   coping H2O2-induced oxidative stress. Consistent with <br /> Yeast cells indicated transformants grown logarithmic      observations, C. reinhardtii Trxh1 expression en- <br /> phase, stained PI, analyzed flow cytometry. Relative flu-     hanced exposure H2O2 diamide (39). Moreover, <br /> orescence intensity (DNA content) plotted number of <br /> counted events (Nuclei counts). Relative DNA contents corresponding <br />                                                                           Chlamydomonas Trxh1 correct hypersensitivity to <br /> to 1N (G1) 2N (G2) indicated.                                     H2O2 yeast trx1 trx2 double mutant. <br />                                                                              To gain insight molecular mechanism(s) uniquely <br />                                                                           dependent Trxh1 (Fig. 8), examined phenotypic <br /> these results suggest functional conservation budding             defects associated lack protein exposure to <br /> yeast Chlamydomonas cytosolic thioredoxins key com-                array DNA-damaging agents. UVC irradiation leads pri- <br /> ponents cellular response alkylating DNA-damaging               marily formation bulky pyrimidine dimers (61). In <br /> agents. Moreover, redox-active Cys-X-X-Cys site necessary            contrast, MMS generates mainly N7-methylguanine N3- <br /> for function.                                                        methyladenine, DNA lesion processing replication of <br />                                                                           damaged templates produce SSBs DSBs secondary <br />                                                                           lesions (15, 59). Bleomycin consists mixture glycopep- <br />                            DISCUSSION <br />                                                                           tides intercalate DNA bases generate an <br />    Thioredoxins small (&#226;&#172;&#402;12 kDa), ubiquitous proteins             activated oxygen species (most likely hydroxyl radical) that <br /> thiol:disulfide oxidoreductase activity consensus WC(G/             causes SSBs DSBs (77). We Ble-1 nor <br /> P)PC active site. In reduced state, Trxs reduce disulfide           Trxh1-suppressed RNAi strains hypersensitive UVC <br /> bridges target polypeptides modulate activ-            irradiation. Thus, UV-induced pyrimidine dimers are <br /> ity proteins involved variety cellular processes (7, 23,       mended direct repair and/or NER (5, 22, 58), Trxh1 does <br /> 37). Eukaryotes contain multiple cytosolic thioredoxin              appear required correct function these <br /> isoforms (38, 41, 53)&#226;&#8364;&#8221;for instance, S. cerevisiae uni-            pathways (Fig. 8). Similarly, Trxh1 does nec- <br /> cellular green alga C. reinhardtii contain (2, 51)&#226;&#8364;&#8221;but           essary homologous recombination nonhomologous <br /> the functional specificity redundancy proteins         end-joining pathways (19, 58) because, Ble-1 appears <br />  VOL. 4, 2005                                                  Trxh1 AND REPAIR OF ALKYLATION-INDUCED DNA DAMAGE                        271 <br />  <br />  <br />                                                                           directly, BER (Fig. 8). Yet role Trxh1 BER might <br />                                                                           counterintuitive pathway expected to <br />                                                                           participate repair H2O2-generated oxidized bases (18, <br />                                                                           21, 58), lack Trxh1 does result hypersensitivity <br />                                                                           H2O2 Chlamydomonas. However, mammals, BER is <br />                                                                           carried distinct subpathways. BER commonly <br />                                                                           initiated DNA N-glycosylase removes damaged base <br />                                                                           form apurinic/apyrimidinic (AP) site (48). This fol- <br />                                                                           lowed strand scission APEs, DNA resynthesis, liga- <br />                                                                           tion (4, 13, 58). In mammalian cells, short-patch BER de- <br />                                                                           pendent DNA polymerase &#226;?&#164; (Pol&#226;?&#164;) long-patch <br />                                                                           BER requires Pol&#226;?&#164; DNA Pol&#226;?&#166;/Pol&#206;&#181; (13, 58). The choice of <br />                                                                           repair subpathway determined nature <br />                                                                           DNA N-glycosylase nature resultant AP <br />                                                                           site (13, 58). Interestingly, short-patch BER favored <br />                                                                           pathway repair MMS-induced N-methylpurines, <br />                                                                           long-patch BER preferentially repairs oxidation-medi- <br />                                                                           ated base loss (21). Mouse cells deficient Pol&#226;?&#164; hyper- <br />                                                                           sensitive variety monofunctional alkylating chemicals <br />                                                                           sensitive DNA-damaging agents, includ- <br />                                                                           ing H2O2 (21, 63). These phenotypes remarkably similar to <br />    FIG. 8. Proposed role(s) Chlamydomonas Trxh1 response <br /> to DNA-damaging agents. A function Trxh1 DNA damage avoid-          Trxh1-defective Chlamydomonas. Moreover, short- <br /> ance (detoxification genotoxic agents) transcriptional acti-   patch BER likely operative C. reinhardtii ho- <br /> vation stress response genes inferred well-documented      mologs key enzymes pathway, mammalian APE 1/Re- <br /> role cytosolic thioredoxins eukaryotes. Our results suggest   dox factor 1 (APE-1/Ref-1) Pol&#226;?&#164;, encoded the <br /> a specialized function Chlamydomonas Trxh1 repair alky- <br /> lation-induced DNA damage (see text details). DSBR, DNA           Chlamydomonas genome (http://genome.jgi-psf.org/chlre2/chlre2 <br /> DSB repair; HR, homologous recombination; MMR, mismatch repair;           .home.html). <br /> TLS, translesion DNA synthesis.                                              Consistent role Trxh1 DNA damage repair, in <br />                                                                           transient-expression assays, Trxh1-GUS fusion protein redis- <br />                                                                           tributed nucleus treatment certain genotoxic <br /> to defective repair DSBs generated bleomycin              agents. Similarly, wheat Trxh proteins localized predominantly <br /> as exposure high concentrations MMS, Trxh1 does             nuclei aleurone scutellum cells maturating <br /> not complement defect. This interpretation sup-              seeds, feature correlated oxidative stress <br /> ported mild hypersensitivity bleomycin Trxh1-            tissues (60). Exposure UV light, ionizing radiation, <br /> suppressed RNAi strains. An unidentified gene(s)               alkylhydroxyperoxides increases translocation of <br /> deleted chromosomal region Ble-1 likely responsible             Trx cytoplasm nucleus mammalian cells (14, <br /> the putative defect DSB repair greater sen-               72, 74), potential role Trx repair DNA lesions <br /> sitivity bleomycin mutant.                                     examined. Mammalian Trx implicated <br />    Both Ble-1 RNAi strains showed significant defects             modulating transcriptional activity number of <br /> in survival exposed MMS, transforma-                factors, cases association APE-1/Ref-1, in <br /> tion Ble-1 wild-type copy Trxh1 complemented             activation stress response genes (20, 29, 52, 74). APE- <br /> hypersensitivity MMS partly restored capacity               1/Ref-1 multifunctional protein possesses DNA <br /> repair MMS-induced DNA damage. Moreover, based het-                    repair redox-regulatory activities (20), control <br /> erologous complementation S. cerevisiae, wild-type Trx               BER regulating assembly, function, and/or expression of <br /> redox site appears necessary function. Enhanced            enzymes pathway (52, 76). Some APE-1/Ref-1 <br /> MMS-induced cytotoxicity Chlamydomonas lacking Trxh1                   activities modulated posttranslationally direct inter- <br /> may defect detoxification genotoxic            action thioredoxin (26, 34). <br /> agent. However, vitro, MMS preferentially modifies               Thioredoxins implicated genotoxic stress re- <br /> lysine histidine protein residues cysteines (the          sponses role damage prevention, provid- <br /> critical amino acids Trx active site) (49). Further, glu-          ing reducing power detoxifying enzymes and/or regulat- <br /> tathione, present millimolar concentration cells,        ing expression (14, 51, 81). Although entirely <br /> has implicated detoxification alkylating electro-          rule scavenging function Chlamydomonas Trxh1 in <br /> philes eukaryotes (6, 9, 73, 75). In addition, Chlamydomonas           response MMS treatment, results indicate a <br /> Trxh1 RNAi epi-mutants enhanced level                   lack Trxh1 results defective repair alkylation- <br /> MMS-induced DNA lesions (Fig. 5), expected                induced DNA lesions. We propose Chlamydomonas Trxh1 <br /> strains defect scavenging function.                           two, mutually exclusive, roles DNA <br />    In number organisms, predominant pathway in-                  damage repair (Fig. 8). Trxh1 modulate DNA repair ac- <br /> volved mending MMS-induced DNA lesions BER (4, 21,                  tivities direct interaction BER components, the <br /> 55). Therefore, phenotypes mutant RNAi                     Chlamydomonas APE-1/Ref-1 homolog. Alternatively, Trxh1 <br /> strains strongly suggest Trxh1 required, directly in-          regulate expression BER enzymes controlling <br />  272       SARKAR ET AL.                                                                                                                             EUKARYOT. CELL <br />  <br />  <br /> the activity redox-dependent transcription factors. Although                            nucleases functions Ref-1. Am. J. Respir. Cell Mol. Biol. <br />                                                                                            25:664&#226;&#8364;&#8220;667. <br /> we emphasized potential role Trxh1 BER,                         21.   Fortini, P., B. Pascucci, F. Belisario, E. Dogliotti. 2000. DNA polymer- <br /> this primary mechanism removal DNA lesions                              ase &#226;?&#164; required efficient DNA strand break repair induced methyl <br /> cause minor helix distortion (4, 13, 58), Trxh1 func-                             methanesulfonate hydrogen peroxide. Nucleic Acids Res. 28: <br />                                                                                            3040&#226;&#8364;&#8220;3046. <br /> tion mismatch repair and/or translesion DNA synthesis (Fig.                       22.   Friedberg, E. C., L. D. McDaniel, R. A. Schultz. 2004. The role of <br /> 8). Nonetheless, findings demonstrate essential require-                            endogenous exogenous DNA damage mutagenesis. Curr. Opin. <br /> ment Chlamydomonas Trxh1 DNA damage repair.                                         Genet. Dev. 14:5&#226;&#8364;&#8220;10. <br />                                                                                      23.   Gelhaye, E., N. Rouhier, J. P. Jacquot. 2004. The thioredoxin h system <br /> Moreover, indicate functional specialization                               higher plants. Plant Physiol. Biochem. 42:265&#226;&#8364;&#8220;271. <br /> Chlamydomonas cytosolic thioredoxins Trxh2 does                            24.   Gietz, D., A. St. Jean, R. A. Woods, R. H. Schiestl. 1992. Improved <br />                                                                                            method high efficiency transformation intact yeast cells. Nucleic Acids <br /> compensate role.                                                                  Res. 20:1425. <br />                                                                                      25.   Goyer, A., P. Decottignies, S. Lemaire, E. Ruelland, E. Issakidis-Bourguet, <br />                           ACKNOWLEDGMENTS                                                  J. P. Jacquot, M. Miginiac-Maslow. 1999. The internal Cys-207 of <br />                                                                                            sorghum leaf NADP-malate dehydrogenase form mixed disulphides with <br />   We thank Byeong-ryool Jeong Karin Van Dijk thoughtful                            thioredoxin. FEBS Lett. 444:165&#226;&#8364;&#8220;169. <br /> discussions. We thank Sarbani Chakraborty, M. P. Doutriaux,                 26.   Gros, L., A. A. Ishchenko, H. Ide, R. H. Elder, M. K. Saparbaev. 2004. <br /> G. Noctor, members lab, critical reading                      The major human AP endonuclease (Ape1) involved nucleotide <br /> manuscript Michel Hours technical assistance flow                           incision repair pathway. Nucleic Acids Res. 32:73&#226;&#8364;&#8220;81. <br /> cytometry experiments.                                                               27.   Harris, E. H. 2001. Chlamydomonas model organism. Annu. Rev. Plant <br />                                                                                            Physiol. Plant Mol. Biol. 52:363&#226;&#8364;&#8220;406. <br />   This work supported grant GM62915 National                         28.   Harris, E. H. 1989. The Chlamydomonas sourcebook: comprehensive guide <br /> Institutes Health.                                                                      biology laboratory. Academic Press, San Diego, Calif. <br />                                                                                      29.   Hirota, K., M. Matsui, S. Iwata, A. Nishiyama, K. Mori, J. Yodoi. 1997. <br />                                  REFERENCES                                                AP-1 transcriptional activity regulated direct association between <br />  1. Arabidopsis Genome Initiative 2000. Analysis genome sequence             thioredoxin Ref-1. Proc. Natl. Acad. Sci. USA 94:3633&#226;&#8364;&#8220;3638. <br />     flowering plant Arabidopsis thaliana. Nature 408:796&#226;&#8364;&#8220;815.                        30.   Hochegger, H., E. Sonoda, S. Takeda. 2004. Post-replication repair in <br />     &#203;&#353; slund, F., J. Beckwith. 1999. The thioredoxin superfamily: redundancy, <br />  2. A                                                                                      DT40 cells: translesion polymerases versus recombinases. Bioessays 26:151&#226;&#8364;&#8220; <br />     specificity, gray-area genomics. J. Bacteriol. 181:1375&#226;&#8364;&#8220;1379.                      158. <br />  3. Balmer, Y., W. H. Vensel, C. K. Tanaka, W. J. Hurkman, E. Gelhaye, N.            31.   Issakidis-Bourguet, E., N. Mouaheb, Y. Meyer, M. Miginiac-Maslow. <br />     Rouhier, J. P. Jacquot, W. Manieri, P. Schurmann, M. Droux, B. B.                  2001. Heterologous complementation yeast reveals new putative func- <br />     Buchanan. 2004. Thioredoxin links redox regulation fundamental               tion chloroplast m-type thioredoxin. Plant J. 25:127&#226;&#8364;&#8220;135. <br />     processes plant mitochondria. Proc. Natl. Acad. Sci. USA 101:2642&#226;&#8364;&#8220;2647.       32.   Jeong B.-R., D. Wu-Scharf, C. Zhang, H. Cerutti. 2002. Suppressors of <br />  4. Boiteux, S., M. Guillet. 2004. Abasic sites DNA: repair biological          transcriptional transgenic silencing Chlamydomonas sensitive DNA- <br />     consequences Saccharomyces cerevisiae. DNA Repair (Amsterdam) 3:1&#226;&#8364;&#8220;                  damaging agents reactivate transposable elements. Proc. Natl. Acad. Sci. <br />     12.                                                                                    USA 99:1076&#226;&#8364;&#8220;1081. <br />  5. Britt, A. B. 1999. Molecular genetics DNA repair higher plants. Trends     33.   Kalinowski, D. P., S. Illenye, B. Van Houten. 1992. Analysis DNA <br />     Plant Sci. 4:20&#226;&#8364;&#8220;25.                                                                    damage repair murine leukemia L1210 cells using quantitative <br />  6. Britten, R. A., J. A. Green, H. M. Warenius. 1992. Cellular glutathione            polymerase chain reaction assay. Nucleic Acids Res. 20:3485&#226;&#8364;&#8220;3494. <br />     (GSH) glutathione-S-transferases (GST) activity human ovarian tu-         34.   Kelley, M. R., S. H. Parsons. 2001. Redox regulation DNA repair <br />     mor biopsies following exposure alkylating agents. Int. J. Radiat. Oncol.           function human AP endonuclease Ape1/ref-1. Antioxid. Redox. Sig- <br />     Biol. Phys. 24:527&#226;&#8364;&#8220;531.                                                                nal. 3:671&#226;&#8364;&#8220;683. <br />  7. Buchanan, B. B., P. Schurmann, P. Decottignies, R. M. Lozano. 1994.          35.   Koerkamp, M. G., M. Rep, H. J. Bussemaker, G. P. Hardy, A. Mul, K. <br />     Thioredoxin: multifunctional regulatory protein bright future              Piekarska, C. A. Szigyarto, J. M. De Mattos, H. F. Tabak. 2002. Dis- <br />     technology medicine. Arch. Biochem. Biophys. 314:257&#226;&#8364;&#8220;260.                          section transient oxidative stress response Saccharomyces cerevisiae by <br />  8. Cerutti, H., A. M. Johnson, J. E. Boynton, N. W. Gillham. 1995. Inhi-              using DNA microarrays. Mol. Biol. Cell 13:2783&#226;&#8364;&#8220;2794. <br />     bition chloroplast DNA recombination repair dominant negative          36.   Kovar, J. L., J. Zhang, R. P. Funke, D. P. Weeks. 2002. Molecular <br />     mutants Escherichia coli RecA. Mol. Cell. Biol. 15:3003&#226;&#8364;&#8220;3011.                       analysis acetolactate synthase gene Chlamydomonas reinhardtii and <br />  9. Chasseaud, L. F. 1979. The role glutathione glutathione S-trans-                development genetically engineered gene dominant selectable <br />     ferases metabolism chemical carcinogens electrophilic              marker genetic transformation. Plant J. 29:109&#226;&#8364;&#8220;117. <br />     agents. Adv. Cancer Res. 29:175&#226;&#8364;&#8220;274.                                             37.   Laloi, C., K. Apel, A. Danon. 2004. Reactive oxygen signalling: latest <br /> 10. Chen, D., W. M. Toone, J. Mata, R. Lyne, G. Burns, K. Kivinen, A. Brazma,              news. Curr. Opin. Plant Biol. 7:323&#226;&#8364;&#8220;328. <br />     N. Jones, J. Bahler. 2003. Global transcriptional responses fission       38.   Laloi, C., D. Mestres-Ortega, Y. Marco, Y. Meyer, J. P. Reichheld. 2004. <br />     yeast environmental stress. Mol. Biol. Cell 14:214&#226;&#8364;&#8220;229.                             The Arabidopsis cytosolic thioredoxin h5 gene induction oxidative stress <br /> 11. Chen, K. H., F. M. Yakes, D. K. Srivastava, R. K. Singhal, R. W. Sobol, J. K.          W-box-mediated response pathogen elicitor. Plant Physiol. 134: <br />     Horton, B. Van Houten, S. H. Wilson. 1998. Up-regulation base                   1006&#226;&#8364;&#8220;1016. <br />     excision repair correlates enhanced protection DNA damaging       39.   Lemaire, S., E. Keryer, M. Stein, I. Schepens, E. Issakidis-Bourguet, C. <br />     agent mouse cell lines. Nucleic Acids Res. 26:2001&#226;&#8364;&#8220;2007.                            Ge&#194;&#180;rard-Hirne, M. Miginiac-Maslow, J.-P. Jacquot. 1999. Heavy-metal <br /> 12. Cole, G. M., D. E. Stone, S. I. Reed. 1990. Stoichiometry G protein             regulation thioredoxin gene expression Chlamydomonas reinhardtii. <br />     subunits affects Saccharomyces cerevisiae mating pheromone signal trans-           Plant Physiol. 120:773&#226;&#8364;&#8220;778. <br />     duction pathway. Mol. Cell. Biol. 10:510&#226;&#8364;&#8220;517.                                    40.   Lemaire, S. D., V. Collin, E. Keryer, A. Quesada, M. Miginiac-Maslow. <br /> 13. Dianov, G. L., K. M. Sleeth, I. I. Dianova, S. L. Allinson. 2003. Repair           2003. Characterization thioredoxin y, new type thioredoxin identified <br />     abasic sites DNA. Mutat. Res. 531:157&#226;&#8364;&#8220;163.                                       genome Chlamydomonas reinhardtii. FEBS Lett. 543:87&#226;&#8364;&#8220;92. <br /> 14. Didier, C., I. Kerblat, C. Drouet, A. Favier, J. C. Beani, M. J. Richard.    41.   Lemaire, S. D., B. Guillon, P. Le Marechal, E. Keryer, M. Miginiac-Maslow, <br />     2001. Induction thioredoxin ultraviolet-A radiation prevents oxidative-          P. Decottignies. 2004. New thioredoxin targets unicellular pho- <br />     mediated cell death human skin fibroblasts. Free Radic. Biol. Med. 31:              tosynthetic eukaryote Chlamydomonas reinhardtii. Proc. Natl. Acad. Sci. <br />     585&#226;&#8364;&#8220;598.                                                                               USA 101:7475&#226;&#8364;&#8220;7480. <br /> 15. Doetsch, P. W., N. J. Morey, R. L. Swanson, S. Jinks-Robertson. 2001.        42.   Longhese, M. P., M. Clerici, G. Lucchini. 2003. The S-phase checkpoint <br />     Yeast base excision repair: interconnections networks. Prog. Nucleic               regulation Saccharomyces cerevisiae. Mutat. Res. 532:41&#226;&#8364;&#8220;58. <br />     Acid Res. Mol. Biol. 68:29&#226;&#8364;&#8220;39.                                                   43.   Mirzayans, R., M. Liuzzi, M. C. Paterson. 1988. Methylmethanesulfon- <br /> 16. Draculic, T., I. W. Dawes, C. M. Grant. 2000. A single glutaredoxin             ate-induced DNA damage repair cultured human fibroblasts: <br />     thioredoxin gene essential viability yeast Saccharomyces cerevi-         normal rates induction removal alkali-labile sites xeroderma <br />     siae. Mol. Microbiol. 36:1167&#226;&#8364;&#8220;1174.                                                    pigmentosum (group A) cells. Carcinogenesis 9:2257&#226;&#8364;&#8220;2263. <br /> 17. Durrant, I. 1990. Light-based detection biomolecules. Nature 346:297&#226;&#8364;&#8220;         44.   Mittler, R. 2002. Oxidative stress, antioxidants stress tolerance. Trends <br />     298.                                                                                   Plant Sci. 7:405&#226;&#8364;&#8220;410. <br /> 18. Evans, M. D., M. S. Cooke. 2004. Factors contributing outcome      45.   Mouaheb, N., D. Thomas, L. Verdoucq, P. Monfort, Y. Meyer. 1998. In <br />     oxidative damage nucleic acids. Bioessays 26:533&#226;&#8364;&#8220;542.                               vivo functional discrimination plant thioredoxins heterologous <br /> 19. Featherstone, C., S. P. Jackson. 1999. DNA double-strand break repair.             expression yeast Saccharomyces cerevisiae. Proc. Natl. Acad. Sci. USA <br />     Curr. Biol. 9:R759&#226;&#8364;&#8220;R761.                                                               95:3312&#226;&#8364;&#8220;3317. <br /> 20. Flaherty, D. M., M. M. Monick, G. W. Hunninghake. 2001. AP endo-             46.   Muller, E. G. 1995. A redox-dependent function thioredoxin necessary <br />  VOL. 4, 2005                                                              Trxh1 AND REPAIR OF ALKYLATION-INDUCED DNA DAMAGE                                            273 <br />  <br />       sustain rapid rate DNA synthesis yeast. Arch. Biochem. Biophys.        65. Sutton, M. D., G. C. Walker. 2001. Managing DNA polymerases: coor- <br />       318:356&#226;&#8364;&#8220;361.                                                                          dinating DNA replication, DNA repair, DNA recombination. Proc. <br /> 47.   Muller, E. G. 1991. Thioredoxin deficiency yeast prolongs S phase              Natl. Acad. Sci. USA 98:8342&#226;&#8364;&#8220;8349. <br />       shortens G1 interval cell cycle. J. Biol. Chem. 266:9194&#226;&#8364;&#8220;9202.         66. Tam, L. W., P. A. Lefebvre. 1995. Insertional mutagenesis isolation <br /> 48.   Nilsen, H., H. E. Krokan. 2001. Base excision repair network              tagged genes Chlamydomonas. Methods Cell. Biol. 47:519&#226;&#8364;&#8220;523. <br />       defence tolerance. Carcinogenesis 22:987&#226;&#8364;&#8220;998.                                 67. Tercero, J. A., M. P. Longhese, J. F. Diffley. 2003. A central role for <br /> 49.   Paik, W. K., P. DiMaria, S. Kim, P. N. Magee, P. D. Lotlikar. 1984.               DNA replication forks checkpoint activation response. Mol. Cell <br />       Alkylation protein methyl methanesulfonate 1-methyl-1-                      11:1323&#226;&#8364;&#8220;1336. <br />       nitrosourea vitro. Cancer Lett. 23:9&#226;&#8364;&#8220;17.                                       68. Trotter, E. W., C. M. Grant. 2003. Non-reciprocal regulation the <br /> 50.   Plosky, B. S., R. Woodgate. 2004. Switching high-fidelity replicases         redox state glutathione-glutaredoxin thioredoxin systems. EMBO <br />       low-fidelity lesion-bypass polymerases. Curr. Opin. Genet. Dev. 14:113&#226;&#8364;&#8220;            Rep. 4:184&#226;&#8364;&#8220;188. <br />       119.                                                                              69. Trotter, E. W., C. M. Grant. 2002. Thioredoxins required pro- <br /> 51.   Powis, G., W. R. Montfort. 2001. Properties biological activities          tection reductive stress yeast Saccharomyces cerevisiae. Mol. <br />       thioredoxins. Annu. Rev. Pharmacol. Toxicol. 41:261&#226;&#8364;&#8220;295.                              Microbiol. 46:869&#226;&#8364;&#8220;878. <br /> 52.   Raffoul, J. J., D. C. Cabelof, J. Nakamura, L. B. Meira, E. C. Friedberg,     70. Tuteja, N., M. B. Singh, M. K. Misra, P. L <br /> </body></html>