<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Conditional Mutations in &#226;&#65533;&#165;-Tubulin Reveal Its <br /> Involvement in Chromosome Segregation and <br /> Cytokinesis <br />  <br />           &#226;&#65533;&#165;-Tubulin is a conserved essential protein required for assembly and function of the mitotic <br />           spindle in humans and yeast. For example, human &#226;&#65533;&#165;-tubulin can replace the &#226;&#65533;&#165;-tubulin gene in <br />           Schizosaccharomyces pombe. <span id='am-2' about='protege:TO' typeof='owl:Thing'>To</span> understand the structural/functional domains of &#226;&#65533;&#165;-tubulin, we <br />           performed a systematic alanine-scanning mutagenesis of human &#226;&#65533;&#165;-tubulin (TUBG1) and studied <br />           phenotypes of each mutant allele in S. pombe. Our screen, both in the presence and absence of the <br />           endogenous S. pombe &#226;&#65533;&#165;-tubulin, resulted in 11 lethal mutations and 12 cold-sensitive mutations. <br />           Based on structural mapping onto a homology model of human &#226;&#65533;&#165;-tubulin generated by free <br />           energy minimization, all deleterious mutations are found in residues predicted to be located on <br />           the surface, some in positions to interact with &#226;&#65533;&#163;- and/or &#226;&#65533;&#164;-tubulins in the microtubule lattice. As <br />           expected, one class of tubg1 mutations has either an abnormal assembly or loss of the mitotic <br />           spindle. Surprisingly, a subset of mutants with abnormal spindles does not arrest in M phase but <br />           proceeds through anaphase followed by abnormal cytokinesis. These studies reveal that in <br />           addition to its previously appreciated role in spindle microtubule nucleation, &#226;&#65533;&#165;-tubulin is in- <br />           volved in the coordination of postmetaphase events, anaphase, and cytokinesis. <br />  <br />  <br /> INTRODUCTION                                                           Leguy et al., 2000) and is required for the assembly of the <br />                                                                        mitotic spindle microtubules both in vivo and in vitro (Oak- <br /> The microtubule lattice is composed of a head-to-tail assem-           ley et al., 1990; Horio et al., 1991; Stearns et al., 1991; Joshi et <br /> bly of heterodimeric protein subunits, &#226;&#65533;&#163;-tubulin and &#226;&#65533;&#164;-tu-             al., 1992; Stearns and Kirschner, 1994; Felix et al., 1994; <br /> bulin (Amos and Klug, 1974). The two ends of a microtubule             Sunkel et al., 1995, Sobel and Snyder, 1995; Spang et al., 1996, <br /> are different in composition, and this difference is reflected         Marschall et al., 1996, reviewed in Oakley, 2000; Wiese and <br /> in the assembly and disassembly kinetics at the two ends.              Zheng, 2000). <br /> For example, the microtubule plus end is much faster at                   The mitotic spindle, a bipolar assembly of microtubules, is <br /> addition and dissociation of tubulin subunits during its               the macromolecular machine that is responsible for chromo- <br /> growth and shrinkage compared with the minus end. In the               some segregation and their faithful transmission into daugh- <br /> mitotic spindle, minus ends are anchored to the spindle                ter cells during mitosis. To accomplish this task, paired <br /> poles, whereas the plus ends are captured by the kineto-               duplicated sister chromatids must obtain a bipolar orienta- <br /> chores of chromosomes (Euteneuer and McIntosh, 1981).                  tion at the mid plate of the metaphase spindle, via the <br /> The protein that binds and anchors microtubules to the                 attachment of their kinetochores to the spindle microtubules <br /> spindle pole is another protein of the tubulin family, &#226;&#65533;&#165;-tu-           <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> two opposite spindle poles. On successful attachment <br /> bulin, which was originally discovered in a suppressor                 of kinetochore microtubules, and perhaps upon the applica- <br /> screen of a &#226;&#65533;&#164;-tubulin mutant in the filamentous fungus                 tion of the resulting poleward tension between the sister <br /> Aspergillus nidulans (Oakley and Oakley, 1989). &#226;&#65533;&#165;-Tubulin is           kinetochores generated by pulling forces, a biochemical sig- <br /> an essential protein that is highly conserved (reviewed in             naling cascade is activated. The pathway culminates in the <br /> Oakley, 2000). Furthermore, &#226;&#65533;&#165;-tubulin binds with high affin-           proteolytic cleavage of the sister chromatid cohesin subunit, <br /> ity to microtubule minus ends in vitro (Li and Joshi, 1995;            thus triggering a decisive and irreversible metaphase-an- <br />                                                                        aphase transition (Uhlmann et al. 1999; reviewed in Amon, <br />   &#226;&#8364;&#160; <br />       These authors contributed equally to this work.                  1999; Burke, 2000; Nasmyth et al., 2000). Subsequent to the <br />   &#194;&#167; <br />       Corresponding author. E-mail address: joshi@cellbio.emory.edu.   completion of anaphase, another signal transduction path- <br />  <br />  <br /> &#194;&#169; 2001 by The American Society for Cell Biology                                                                                       2469 <br />  T.W. Hendrickson et al. <br />  <br />  <br /> way involving the components of the septation initiation                18&#194;&#176;C and temperature-sensitive mutants at 36&#194;&#176;C. Strains were main- <br /> network is activated by a spindle pole-associated GTP bind-             tained at the permissive temperatures of 30 and 26&#194;&#176;C, respectively. <br /> ing protein Spg1p (reviewed in Gould and Simanis, 1997). <br /> This pathway culminates in the events that lead to the <br /> physical constriction of the actomyosin cytokinetic ring that           Alanine-scanning Mutagenesis of TUBG1 <br /> pinches the plasma membrane at the center followed by the               TUBG1 cDNA was subcloned into the NdeI site of the pALTER-EX1 <br /> deposition of septal components and fission. The precise                vector (Promega, Madison, WI) downstream from the SP6 promoter <br /> coordination of this sequence of events is achieved by feed-            to create pTWH101. The amino acid sequence of TUBG1p was <br /> back loops that ensure the completion of the previous steps             scanned for clusters of charged residues. Clusters were defined as a <br />                                                                         group of charged amino acids occurring within two to five residues <br /> before the initiation of the next steps (Balasubramanian et al.,        of each other. Twenty-seven charged clusters were identified. The <br /> 2000).                                                                  charged amino acids (Arg [R], Asp [D], Glu [E], Lys [K]) in these <br />   After successful bipolar attachment of kinetochores to                clusters were changed to alanine one cluster at a time. The mutagen- <br /> microtubules, one feedback loop monitors the connection                 esis was carried out with the use of oligonucleotide-directed mu- <br /> between the kinetochore and the plus ends of the microtu-               tagenesis, with the use of pTWH101 as the template. pALTER-EX1 <br /> bules before triggering anaphase (reviewed in Shah and                  contains a tetracycline resistance gene and an inactivated ampicillin <br /> Cleveland, 2000). At the end of anaphase another feedback               resistance gene. The ampicillin-repair oligonucleotide restores the <br /> loop triggers subsequent cytokinesis (reviewed in Amon,                 activity of the inactivated AmpR gene, whereas the tetracycline- <br /> 1999; Burke, 2000; Hoyt 2000). The role of minus end attach-            <span id='am-237' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-238' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-267' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-278' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-282' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-283' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-300' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-309' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>knockout</span> oligonucleotide inactivates the TetR gene. The inactivation <br />                                                                         of the TetR gene and the activation of the AmpR gene provided an <br /> ment to the spindle pole in coordinated triggering of these             effective method for selecting potential alanine-scanning mutants. <br /> mitotic steps is not known. To investigate whether the mi-              Oligonucleotides synthesized by Integrated DNA Technologies <br /> crotubule minus end binding protein of the spindle pole,                (Coralville, IA) were used to change each cluster independently. <br /> &#226;&#65533;&#165;-tubulin, plays any role in mitotic events, we carried out a           Because contiguous alanine codons can be used to create a PstI <br /> systematic alanine-scanning mutagenesis of the human &#226;&#65533;&#165;-tu-              recognition site, we were able to use PstI digestion to further screen <br /> bulin TUBG1 and generated 27 mutations, 12 of which con-                for potential alanine-scanning mutants. Each of the 27 mutations <br /> fer cold-sensitive (cs&#226;&#171;&#186;) growth defects in Schizosaccharomyces          was sequenced to verify that the intended <span id='am-159' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-160' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-217' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>mutation</span> was present and <br /> pombe, whereas 11 were lethal. Based on structural mapping              that it was the only difference between the mutant (tubg1) and the <br /> onto a homology model of human &#226;&#65533;&#165;-tubulin, all deleterious               wild type (TUBG1). Additionally, we could produce stable protein <br />                                                                         of the predicted size from all of the mutations generated in vitro. <br /> mutations are found in residues predicted to be located on <br /> the surface, often in a position to interact with &#226;&#65533;&#163;-tubulins <br /> and/or &#226;&#65533;&#164;-tubulins in the microtubule lattice. Although some             Generation of &#226;&#65533;&#165;-Tubulin Homology Model <br /> of the tubg1 cs&#226;&#171;&#186; phenotypes at the restrictive temperature <br />                                                                         The human &#226;&#65533;&#165;-tubulin homology model was generated with the use <br /> showed an expected loss of mitotic spindle, many tubg1                  of the Swiss-Model homology algorithm (www.expasy.ch/swiss- <br /> mutants could assemble a mitotic spindle, although it often             mod/SWISS-MODEL.html). This modeling algorithm requires a <br /> appeared abnormal. Despite abnormal spindle assembly,                   target sequence that shares at least 25% identity to a template with <br /> four of these mutants accomplished aberrant completion of               a known three-dimensional structure (Peitsch et al., 1995; Peitsch, <br /> mitosis and septation despite spindle defects. These results            1996; Guex and Peitsch, 1997). The TUBG1p sequence was first <br /> suggest a role for &#226;&#65533;&#165;-tubulin in monitoring proper spindle               processed by Swiss-Model, which BLASTs (Altschul et al., 1990) it <br /> assembly before anaphase and cytokinesis (see DISCUS-                   against other sequences derived from the Brookhaven Protein Data <br /> SION).                                                                  Bank. In this process, porcine &#226;&#65533;&#163;-tubulin and &#226;&#65533;&#164;-tubulin (structures <br />                                                                         solved by Nogales et al., 1998, 1999) were selected as templates for <br />                                                                         the homology model construction. The TUBG1p sequence was then <br /> MATERIALS AND METHODS                                                   aligned to the template sequences with the use of a structurally <br />                                                                         corrected multiple sequence alignment. This was done with the use <br /> Yeast Strains, Media, and Genetic Manipulations                         of the best-scoring diagonals as determined by SIM (Huang et al., <br />                                                                         1991). A three-dimensional match was then performed by superim- <br /> The following yeast strains were used in the characterization and       posing C&#226;&#65533;&#163; atom pairs from the highest scoring local sequence <br /> genetic manipulation of the &#226;&#65533;&#165;-tubulin mutants. NC377 (h&#226;&#171;&#185;/h&#226;&#171;&#186;, leu1-      alignment to construct a model. The structure was then optimized <br /> 32/leu1-32, his3D1/&#226;&#171;&#185;, ura4-D18/ura4-D18, ade6-M210/ade6-M216,           by maximizing the number of C&#226;&#65533;&#163; atom pairs in the common core <br /> &#226;&#171;&#185;/gtb1::ura4&#226;&#171;&#185;) (Horio and Oakley, 1994)-derived haploids were           and minimizing their relative mean square deviation. The second- <br /> used to identify mutations that are conditional in the absence of       ary structures and loops were further modified with the use of <br /> endogenous &#226;&#65533;&#165;-tubulin, whereas JLP201 (h&#226;&#171;&#185;, leu1-32, ura4-D18, ade6-      energy considerations such as van der Waals radii, hydrogen bonds, <br /> 210) (Paluh and Clayton, 1996) was used to identify mutations that      hydrophobic, polar, and ionic interactions to generate a minimal <br /> are conditional in the presence of endogenous &#226;&#65533;&#165;-tubulin. PY321 (h&#226;&#171;&#185;,     free energy conformation. <br /> leu1-32, ura4-D18, lys1&#226;&#171;&#185;::lacO, GFP-lacI-NLS) (Watanabe and Nurse, <br /> 1999) was used to monitor sister chromatid separation in &#226;&#65533;&#165;-tubulin <br /> mutants. PY321 is a yeast strain that encodes a green fluorescent       Flow Cytometry Analysis of Alanine-scanning <br /> protein (GFP)-LacI fusion protein at the his1&#226;&#171;&#185; locus. This fusion <br /> protein binds to a tandem repeat of LacO sequences, which is            Mutants <br /> inserted near the centromere of chromosome I at the lys1&#226;&#171;&#185; locus         Wild-type cells, gtb1&#226;&#338;&#172; cells with wild-type TUBG1, wild-type cells <br /> (Nabeshima et al., 1998; Watanabe and Nurse, 1999). Binding of the      with wild-type TUBG1, wild-type cells with dominant tubg1 alleles, <br /> GFP-LacI fusion protein to LacO at the his1&#226;&#171;&#185; locus provides an          and gtb1&#226;&#338;&#172; cells with recessive tubg1 alleles were grown at 30&#194;&#176;C until <br /> efficient tool for marking the centromeric region (Nabeshima et al.,    mid-log phase and then shifted to 18&#194;&#176;C for 10 h. Cells were then <br /> 1998). Cells were grown in either rich medium (YPD), or minimal         fixed with 70% cold ethanol. Cells were then processed with 0.1 <br /> medium (MM) with the indicated supplements (Alfa et al., 1993). All     mg/ml RNase A in 50 mM M Na citrate for 2 h at 37&#194;&#176;C. For DNA <br /> yeast transformations were performed by the lithium acetate             staining, cells were suspended in 1 &#226;&#65533;&#174;M SYTOX Green (Molecular <br /> method (Alfa et al., 1993). We screened for cold-sensitive mutants at   Probes, Eugene, OR) in 50 mM Na citrate. Cells were then immedi- <br />  <br />  <br /> 2470                                                                                                           Molecular Biology of the Cell <br />                                                                                                      &#226;&#65533;&#165;-Tubulin in Anaphase and Cytokinesis <br />  <br />  <br /> ately analyzed with the use of fluorescence-activated cell sorter          RESULTS <br /> (FACS). <br />                                                                            Alanine-scanning Mutants Map to Surface of &#226;&#65533;&#165;- <br />                                                                            Tubulin Homology Model <br /> Analysis of Alanine-scanning Mutants of TUBG1 in                           The logic for with the use of alanine-scanning mutagenesis is <br /> Fission Yeast                                                              that by changing charged clusters of amino acids, which are <br /> The plasmids used to transform yeast strains were constructed by <br />                                                                            hydrophilic and thus usually on the surface of the protein, <br /> subcloning each of the tubg1 alleles into pREP1 at the NdeI site           subtle changes can be created in the protein structure that <br /> downstream of the nmt1&#226;&#171;&#185; promoter (Maundrell, 1990, 1993). Wild-            could prove to be useful in studying structure&#226;&#8364;&#8220;function <br /> type JLP201 (Paluh and Clayton, 1996) cells were transformed with          relationships. The amino acid sequence of &#226;&#65533;&#165;-tubulin is as <br /> either one of the tubg1-pREP1 plasmids or TUBG1-pREP1 and                  similar to &#226;&#65533;&#163;-tubulin and &#226;&#65533;&#164;-tubulin as the sequence of &#226;&#65533;&#163;-tu- <br /> grown on minimal media supplemented with adenine, histidine,               bulin is to that of &#226;&#65533;&#164;-tubulin (Oakley and Oakley, 1989). <br /> and uracil. Transformants were then screened at 18 and 36&#194;&#176;C to             Although the atomic structures of &#226;&#65533;&#163;-tubulin and &#226;&#65533;&#164;-tubulin <br /> identify conditional mutants in the presence of endogenous &#226;&#65533;&#165;-tubu-         are very similar to one another, there must exist some subtle <br /> lin. The strains were maintained at 30 and 26&#194;&#176;C, respectively. A           differences in the structure of &#226;&#65533;&#165;-tubulin. This is because <br /> diploid strain, NC377 (Horio et al., 1991; Horio and Oakley, 1994), <br /> bearing one endogenous wild-type copy of S. pombe &#226;&#65533;&#165;-tubulin, gtb1&#226;&#171;&#185; <br />                                                                            &#226;&#65533;&#165;-tubulin is not embedded in the microtubule lattice, but <br /> and one disrupted copy, gtb1::ura4&#226;&#171;&#185;, was also transformed with the         instead binds the component(s) of the microtubule-organiz- <br /> mutant plasmids. The resulting transformants were then randomly            ing center, in addition to the tubulin subunit(s) at microtu- <br /> sporulated, selected for ura&#226;&#171;&#185;, leu&#226;&#171;&#185;, and the spores were tested for        bule minus ends. Thus, a simple substitution of &#226;&#65533;&#165;-tubulin <br /> conditional growth.                                                        residues onto the &#226;&#65533;&#163;-tubulin or &#226;&#65533;&#164;-tubulin structural model <br />                                                                            was not sufficient for our mapping studies. Therefore, we <br />                                                                            generated a free energy, minimized homology model of <br /> Immunofluorescence Microscopy                                              &#226;&#65533;&#165;-tubulin taking into consideration the allowable van der <br />                                                                            Waals radii of atoms, hydrogen bonds, and hydrophobic, <br /> To obtain cultures enriched for mitotic cells, cells were grown to         polar, and ionic interactions. Figure 1 shows the locations of <br /> mid-log phase and then synchronized in S phase for 3 h by the <br /> addition of 20 mM hydroxyurea (HU) (Sigma Chemical Co., St. <br />                                                                            the charged amino acids that were changed to alanine on the <br /> Louis, MO). The HU was then removed and the cells were shifted to          &#226;&#65533;&#165;-tubulin amino acid sequence (Figure 1A), and mutations <br /> 18&#194;&#176;C for 8 &#226;&#8364;&#8220;12 h. Cells were then fixed and processed for immuno-          that confer deleterious growth phenotypes are shown on the <br /> fluorescence microscopy.                                                   stereo image of the homology model of human &#226;&#65533;&#165;-tubulin <br />    To prepare for microtubule staining, cells were grown to a density      (Figure 1B). For orientation, we placed the equivalent struc- <br /> of 2&#226;&#8364;&#8220; 4 &#226;&#171;&#187; 106 cells/ml at the permissive temperature and then shifted to   tural model of the most likely tubulin subunit exposed at the <br /> the restrictive temperature for 8&#226;&#8364;&#8220;12 h. Cells were fixed with formalde-    minus end of a microtubule, &#226;&#65533;&#163;-tubulin (Fan et al., 1996). As <br /> hyde and gluteraldehyde according to Alfa et al. (1993). After fixation    expected, most of the alanine-scanning mutations are on the <br /> and preincubation with PEMBAL [100 mM piperazine-N,N&#226;&#172;&#732;-bis(2-eth-           external surface of the protein where it is exposed to the <br /> anesulfonic acid), 1 mM EGTA, 1 mM MgSO4, pH 6.9, 1% bovine <br />                                                                            aqueous environment of the cytoplasm. Many of the muta- <br /> serum albumin, 0.1% NaN3, 100 mM l-lysine hydrochloride] for 30 <br /> min at room temperature, the cells were incubated overnight at 26&#194;&#176;C        tions that are deleterious to S. pombe growth are interestingly <br /> with the anti-&#226;&#65533;&#163;-tubulin monoclonal antibody TAT1 (1:25) in PEMBAL          located at sites that might be involved in lateral (tubg1-8, <br /> (Woods et al., 1989). The cells were then washed and incubated over-       tubg1-9, tubg1-11, tubg1-12, tubg1-15, and tubg1-18) or longi- <br /> night at 26&#194;&#176;C with the secondary antibody Cy3-conjugated goat anti-        tudinal (tubg1-4, tubg1-9, tubg1-13, and tubg1-22) interactions <br /> mouse (1:200) in PEMBAL (Jackson Laboratories, Bar Harbor, ME). The        with the &#226;&#65533;&#163;-tubulin and/or &#226;&#65533;&#164;-tubulin subunits of the micro- <br /> cells were then washed with PEMBAL and phosphate buffered saline           tubule lattice (Incla&#194;&#180; n and Nogales, 2001). These observations <br /> (PBS). To visualize the DNA, cells were incubated in 1 &#226;&#65533;&#174;g/ml 4&#226;&#172;&#732;,           suggest that at least some of the lateral and longitudinal <br /> 6&#226;&#172;&#732;-diamidino-2-phenylindole (DAPI) in PBS at 26&#194;&#176;C for 30 min or            contact sites of &#226;&#65533;&#165;-tubulin must be equivalent to those of <br /> incubated with 10 &#226;&#65533;&#174;g/ml Hoechst in PBS for 1 h. Cells were viewed <br />                                                                            &#226;&#65533;&#163;-tubulin and &#226;&#65533;&#164;-tubulin within the microtubule lattice. The <br /> under a Ziess Axiovert 135 microscope. <br />    To determine &#226;&#65533;&#165;-tubulin localization, cells were fixed with meth-        mutations, such as tubg1-2, tubg1-7, tubg1-24, and tubg1-26, <br /> anol. Briefly, cells were filtered onto a membrane disk and then           that are not located at either the putative lateral or longitu- <br /> fixed in cold methanol at &#226;&#171;&#186;80&#194;&#176;C for 10 min or longer. After fixation       dinal interaction sites are localized to regions that may be <br /> the cells were washed with PEM [100 mM piperazine-N,N&#226;&#172;&#732;-bis(2-              involved in interactions with other nontubulin proteins. This <br /> ethanesulfonic acid), 1 mM EGTA, 1 mM MgSO4, pH 6.9] then                  homology model of human &#226;&#65533;&#165;-tubulin can be used to further <br /> permeabilized with 100T Zymolase 1.3 mg/ml (Seikagu, Japan) in             investigate the structure&#226;&#8364;&#8220;function relationship of &#226;&#65533;&#165;-tubulin. <br /> PEMS (PEM &#226;&#171;&#185; 1.2 M sorbitol). The cells were then blocked with              The alanine-scanning mutants will be useful to further study <br /> PEMBALG (PEMBAL &#226;&#171;&#185; glysine) for 30 min and then incubated                   the interactions of &#226;&#65533;&#165;-tubulin with other tubulins or other <br /> overnight with the anti-&#226;&#65533;&#165;-tubulin antibody, GTU-88 (1:200) (Sigma <br />                                                                            components of the microtubule-organizing center. <br /> Chemical Co.) in PEMBALG. The after day, cells were washed and <br /> incubated overnight with the secondary antibody goat anti-mouse- <br /> Cy3 (1:1000) (Jackson Laboratories) in PEMBALG. DNA was visu- <br /> alized by either DAPI or Hoechst staining as previously described.         Growth of Alanine-scanning Mutants in Fission <br /> Cells were viewed under a Ziess Axiovert 135 microscope.                   Yeast <br />    To monitor separation of sister chromatids, PY321 cells were <br /> transformed with tubg1 mutations. The transformants were grown             It has been previously shown that human &#226;&#65533;&#165;-tubulin (TUBG1) <br /> to mid-log phase and then incubated with 10 &#226;&#65533;&#174;g/ml Hoechst for 1 h.         can functionally replace the S. pombe &#226;&#65533;&#165;-tubulin gene gtb1&#226;&#171;&#185; <br /> The cells were visualized with the use of an Olympus BX60 Epiflu-          (Horio and Oakley, 1994). Because we were interested in <br /> orescence microscope equipped with a Photometrics Quantix digital          identifying evolutionarily conserved features of &#226;&#65533;&#165;-tubulin, <br /> camera and IP-Lab Spectrum software.                                       we have examined the functional consequences of human <br />  <br />  <br /> Vol. 12, August 2001                                                                                                                 2471 <br />  T.W. Hendrickson et al. <br />  <br />  <br />  <br />  <br /> Figure 1. Locations of the charged clusters of human &#226;&#65533;&#165;-tubulin. (A) Amino acid sequence of human &#226;&#65533;&#165;-tubulin is shown with the charged <br /> clusters underlined and the residues that were changed to alanine in bold letters. A cluster was defined as two or more charged residues <br /> within five residues of each other. Twenty-seven such charged clusters were altered by site-directed mutagenesis, replacing the residues with <br /> the neutral amino acid with a small side chain, alanine. The color of each cluster represents the phenotypic outcome after its expression in <br /> either wild-type S. pombe (hence a dominant <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span>phenotype</span>), or in haploid S. pombe cells in which the endogenous &#226;&#65533;&#165;-tubulin gene is disrupted <br /> (hence the recessive phenotype). Green, recessive lethal; purple, recessive cold sensitive; red, dominant cold sensitive; yellow, recessive lethal <br /> and dominant cold sensitive. (B) Stereo pair of the minimal free energy &#226;&#65533;&#165;-tubulin homology model shown in longitudinal contact with the <br /> &#226;&#65533;&#163;-tubulin subunit of the microtubule minus end. The model is oriented to reveal a view from inside the lumen of a microtubule. The left and <br /> right sides thus represent putative lateral contact sites of &#226;&#65533;&#165;-tubulin with other tubulin subunits. The colors of these clusters correspond to the color <br /> scheme showed in A. <br />  <br />  <br /> TUBG1 alanine-scanning mutants in fission yeast. All exper-                     The amino acid changes and growth phenotypes of the <br /> iments were performed with the use of either haploid cells                    alanine-scanning mutants of TUBG1 are summarized in <br /> carrying a disrupted &#226;&#65533;&#165;-tubulin allele or wild-type haploid                    Table 1. All of the conditional mutants identified were <br /> cells each transformed with one of the alanine-scanning                       cold sensitive (cs&#226;&#171;&#186;) to varying degrees, six in the presence <br /> mutant human &#226;&#65533;&#165;-tubulin alleles. In both cases, the wild-type                  of endogenous gtb1&#226;&#171;&#185; and six in the absence of gtb1&#226;&#171;&#185; <br /> TUBG1 transformants served as controls.                                       indicated as recessive cs&#226;&#171;&#186; (Table 1). There were also 11 <br />  <br />  <br /> 2472                                                                                                                   Molecular Biology of the Cell <br />                                                                                             &#226;&#65533;&#165;-Tubulin in Anaphase and Cytokinesis <br />  <br />  <br />  <br /> Table 1. Growth phenotypes of alanine-scanning mutants of TUBG1 in S. pombe <br />  <br /> Mutant                                Amino Acid Change                              with gtb1&#226;&#171;&#185;                   without gtb1&#226;&#171;&#185; <br />  <br /> tubg1-1                   R3A, E4A                                                       wt                      wt <br /> tubg1-2                   E20A, K23A                                                     wt                      Recessive   lethal <br /> tubg1-3                   E38A, E39A, E43A                                               wt                      wt <br /> tubg1-4                   D46A, R47A, K48A, D49A                                         wt                      cs&#226;&#171;&#186; <br /> tubg1-5                   D56A, D57A, E58A                                               cs&#226;&#171;&#186;                     wt <br /> tubg1-6                   D68A, E70A, P71A, R72A                                         wt                      cs&#226;&#171;&#186; <br /> tubg1-7                   K84A, L85A, Y86A, N87A, P88A, E89A                             wt                      cs&#226;&#171;&#186; <br /> tubg1-8                   D120A, D123A, R124A, E125A, D127A                              wt                      Recessive   lethal <br /> tubg1-9                   S129A, D130A, S131A, E133A                                     wt                      Recessive   lethal <br /> tubg1-10                  E155A, R156A, L157A, D159A, R160A                              wt                      cs&#226;&#171;&#186; <br /> tubg1-11                  K163A, K164A, L165V                                            wt                      Recessive   lethal <br /> tubg1-12                  D159A, R160A, K163A, K164A                                     wt                      Recessive   lethal <br /> tubg1-13                  D176A, E177A, S179A, D180A                                     cs&#226;&#171;&#186;                     Recessive   lethal <br /> tubg1-14                  K193A, R194A, L195A                                            wt                      cs&#226;&#171;&#186; <br /> tubg1-15                  R286A, K287A, T288A                                            wt                      cs&#226;&#171;&#186; <br /> tubg1-16                  D292A, R295A, R296A, L297A                                     wt                      wt <br /> tubg1-17                  R309A, D310A, R311A                                            wt                      wt <br /> tubg1-18                  R339A, R341A, E342A, R343A, K344A                              cs&#226;&#171;&#186;                     Recessive   lethal <br /> tubg1-19                  R362A, K363A, S364A                                            wt                      wt <br /> tubg1-20                  E389A, R390A, T391A, R393A                                     wt                      wt <br /> tubg1-21                  D396A, K397A, R399A, K400A, R401A, E402A                       wt                      wt <br /> tubg1-22                  E406A, R409A, K410A, E411A, D412A                              cs&#226;&#171;&#186;                     wt <br /> tubg1-23                  K415A, D416A, N417A                                            wt                      Recessive   lethal <br /> tubg1-24                  D419A, E420A, D422A                                            wt                      Recessive   lethal <br /> tubg1-25                  R425A, E426A, I427A                                            wt                      wt <br /> tubg1-26                  D433A, E434A, Y435A                                            cs&#226;&#171;&#186;                     Recessive   lethal <br /> tubg1-27                  R440, P441A, D442A                                             cs&#226;&#171;&#186;                     Recessive   lethal <br />  <br /> wt, wild type. <br />  <br />  <br />  <br />  <br /> mutants that were unable to support growth in the ab-             cells (Figure 3). It is therefore likely that mutations that do <br /> sence of endogenous gtb1&#226;&#171;&#185; (referred to as recessive lethal        disrupt &#226;&#65533;&#165;-tubulin localization confer lethality. <br /> in Table 1). Representative growth phenotypes at the <br /> permissive and restrictive temperatures are shown in Fig-         Mutant &#226;&#65533;&#165;-Tubulins Display Aberrant Cell Cycle <br /> ure 2. Because the alanine-scanning mutations are pre-            Profiles <br /> dicted to affect microtubule assembly, we hypothesized <br /> that some of these mutations could be sensitive to micro-         To gain a general insight into the progression of the cell <br /> tubule-depolymerizing drugs, such as thiabenzadole                cycle in the cs&#226;&#171;&#186; alleles, we performed a flow cytometric <br /> (TBZ). To test this hypothesis, serial dilutions of all cs&#226;&#171;&#186;       analysis (FACS) of DNA content (Figure 4). Wild-type cells <br /> mutants were plated on media containing 10 &#226;&#65533;&#174;g/ml TBZ.             (Figure 4A), gtb1&#226;&#338;&#172; cells expressing TUBG1 (Figure 4B), and <br /> All six recessive cold-sensitive mutants showed increased         wild-type cells expressing TUBG1, and the dominant (Figure <br /> sensitivity to TBZ. A few representative examples of these        4, D&#226;&#8364;&#8220;I) and the recessive (Figure 4, J&#226;&#8364;&#8220;O) cs&#226;&#171;&#186; cells were grown <br /> data of the cold-sensitive growth phenotypes and TBZ              to mid-log phase at 30&#194;&#176;C and then shifted to 18&#194;&#176;C for 10 h. <br /> sensitivity are shown in Figure 2.                                Cells were prepared for FACS analysis by staining with the <br />                                                                   fluorescent DNA dye SYTOX Green. For each strain, 10,000 <br />                                                                   events were analyzed. The profiles reveal a striking differ- <br /> Mutant &#226;&#65533;&#165;-Tubulin Localization Is Comparable to                    ence between wild-type controls and tubg1 mutants. As <br /> Wild-Type &#226;&#65533;&#165;-Tubulin Localization                                  shown in Figure 4A, wild-type cells seem to accumulate in <br /> By creating a mutation on the surface of &#226;&#65533;&#165;-tubulin that may       the G1 phase of the cell cycle with an unduplicated, 1N DNA <br /> disrupt protein-protein interactions, one might also disrupt      content. gtb1&#226;&#338;&#172; cells expressing wild-type TUBG1 seem to <br /> the interactions that are responsible for the intracellular       retard the cell cycle with 2N DNA content (Figure 4B). <br /> localization of &#226;&#65533;&#165;-tubulin. To determine whether any of the        Wild-type cells expressing wild-type TUBG1 also display <br /> cs&#226;&#171;&#186; alanine-scanning mutants differ in the &#226;&#65533;&#165;-tubulin local-       this delay as judged by an increased number of cells with 2N <br /> ization from that of the wild-type protein, we performed          DNA content (Figure 4C). Although human &#226;&#65533;&#165;-tubulin can <br /> immunolocalization of &#226;&#65533;&#165;-tubulin in strains transformed with       functionally complement gtb1&#226;&#338;&#172; cells (Horio and Oakley, <br /> either the mutant alleles or the wild-type &#226;&#65533;&#165;-tubulin. We did      1994), there are some subtle abnormalities in the cell cycle <br /> not detect any differences in the intracellular localization of   timing of these cells (Figure 4, B and C). Also, it should be <br /> &#226;&#65533;&#165;-tubulin in the cs&#226;&#171;&#186; mutants compared with the wild-type          noted that a very small fraction of the cell population in all <br />  <br />  <br /> Vol. 12, August 2001                                                                                                           2473 <br />  T.W. Hendrickson et al. <br />  <br />  <br />  <br />  <br />                                                                      Figure 3. Normal &#226;&#65533;&#165;-tubulin localization is not perturbed in the <br />                                                                      conditional tubg1 mutants. Exponentially growing cells at 30&#194;&#176;C were <br />                                                                      either shifted at 18&#194;&#176;C for 8 h, fixed in methanol, and then processed <br />                                                                      for immunolocalization with the use of the mouse monoclonal <br />                                                                      GTU-88 followed by Cy3-conjugated goat anti-mouse antibodies. <br />                                                                      DNA was stained with 10 &#226;&#65533;&#174;g/ml Hoechst for 1 h. Arrows indicate <br />                                                                      dividing cells. <br />  <br />  <br /> Figure 2. tubg1 alanine-scanning mutants that show conditional       ble 2; Figure 5). These defects include asymmetrically local- <br /> growth. A 10-fold dilution series containing 105&#226;&#8364;&#8220;102 cells of (A)    ized nuclei (asym nuc), lagging DNA during anaphase <br /> dominant cold-sensitive (cs&#226;&#171;&#186;) or TUBG1, (B) recessive cs&#226;&#171;&#186; mutants <br />                                                                      (DNA seg), cells with more than one nucleus (&#226;&#172;&#381;1 nucleus), <br /> or TUBG1 in the background of a &#226;&#65533;&#165;-tubulin null mutant (C) reces- <br /> sive cs&#226;&#171;&#186; mutants that were sensitive to the microtubule-depolymer-   and anucleate cells (0 nuc). Other defects include altered <br /> izing drug TBZ or TUBG1 in the background of a &#226;&#65533;&#165;-tubulin null        cellular morphologies: pear, banana, and oval shapes. We <br /> mutant were plated on appropriate selective media with or without    also observed defects in septation and cell separation. Sep- <br /> 10 &#226;&#65533;&#174;g/ml TBZ. The plates were then incubated at a permissive         tation defects include asymmetric placement of septa (asym <br /> temperature of 30&#194;&#176;C or a restrictive temperature of 18&#194;&#176;C for 7 d.    sept), multiple septa (mult sept), and/or obliquely placed <br /> Panels shown in C were all incubated at the permissive temperature   septa (obliq sept). Defects in cell separation are consistent <br /> of 30&#194;&#176;C.                                                             with the FACS profiles with an accumulation of more than <br />                                                                      2N DNA content. The percentages for each of the pheno- <br />                                                                      types observed for each of the 12 mutants is tabulated in <br /> the wild-type controls show cells with 4N and 8N DNA                 Table 2 and the most striking DNA segregation defects are <br /> content. FACS profiles of some mutants display striking              shown in Figure 5 along with cellular microtubules. <br /> deviations from the FACS profiles of their appropriate con-             To our surprise, none of our mutants showed a complete <br /> trol counterparts. For example tubg1-5, tubg1-14, and                loss of microtubules. However, tubg1-7, tubg1-10, tubg1-13, <br /> tubg1-15 (8N) and tubg1-7 and tubg1-10 (4N) have a pro-              and tubg1-18 showed an overall reduction in total tubulin <br /> nounced increased in the population of cells with more than          polymer. The most pronounced defects in microtubules in- <br /> 2N DNA content. Additionally, tubg1-6, tubg1-7, tubg1-18,            cluded curved and S-shaped spindles (tubg1-6 and tubg1-15, <br /> and tubg1-22 have cells with less than 1N DNA content. The           Figure 5). The number of cells showing microtubule defects <br /> remaining mutants, like their respective wild-type controls,         varied from experiment to experiment but in general, 5&#226;&#8364;&#8220;10% <br /> have a predominant population of cells with 2N DNA con-              of the cells expressed this defect. <br /> tent with the exception of tubg1-14 and tubg1-15.                       To determine how many of the observed phenotypes rep- <br />                                                                      resented terminal events, we wanted to enrich for cells that <br />                                                                      synchronously enter mitosis. To do this, we enriched for <br /> Penetrance and Expressivitiy of Mutant Phenotypes                    synchronous cells with the use of hydroxyurea, to induce an <br /> The strikingly aberrant cell cycle profiles of most tubg1 mu-        S phase block, and then followed cells 4 &#226;&#8364;&#8220; 6 h after their <br /> tants persuaded us to examine the cytological phenotypes of          release from arrest. This strategy yielded cells that entered <br /> each of the strains. To do this, both dominant and recessive         mitosis in a partially synchronous manner as shown in <br /> cs&#226;&#171;&#186; mutants were grown at the permissive temperature of              Figure 6. These studies further confirmed that the defects <br /> 30&#194;&#176;C and then shifted to the restrictive temperature of 18&#194;&#176;C         observed appear as early as 4 &#226;&#8364;&#8220; 6 h after the temperature shift <br /> for 8 &#226;&#8364;&#8220;12 h. Cells were then fixed and stained to visualize          and allowed further detailed observations of the phenotypes <br /> chromosomes and microtubules. All of the mutants shared              that were not apparent from studying nonsynchronous cell <br /> defects in chromosome segregation to various degrees (Ta-            populations. Multiple septa were visible in cells that had <br />  <br />  <br /> 2474                                                                                                       Molecular Biology of the Cell <br />                                                                                          &#226;&#65533;&#165;-Tubulin in Anaphase and Cytokinesis <br />  <br />  <br />  <br />  <br /> Figure 4. Flow cytometric anal- <br /> ysis of DNA content of the cold <br /> sensitive tubg1 mutants. Wild- <br /> type cells (A), gtb1&#226;&#338;&#172; cells express- <br /> ing TUBG1 (B), and wild-type <br /> cells expressing TUBG1, and the <br /> dominant (D&#226;&#8364;&#8220;I) and the recessive <br /> (J&#226;&#8364;&#8220;O) cold-sensitive cells were <br /> grown to mid-log phase at 30&#194;&#176;C <br /> and then shifted to 18&#194;&#176;C for 10 h. <br /> Cells were fixed with ethanol and <br /> stained with the fluorescent DNA <br /> dye SYTOX Green. For each <br /> strain, 10,000 events were ana- <br /> lyzed. The relative fluorescence <br /> corresponding to 1N, 2N, 4N, and <br /> 8N is indicated on the x-axis. The <br /> profiles reveal a striking differ- <br /> ence between wild-type controls <br /> and tubg1 mutants. <br />  <br />  <br />  <br /> failed to undergo fission but the presence of DNA spots         the nature of the chromosome segregation defects. The ap- <br /> suggested that the cells progressed through another round       pearance of elongated spindles and improper chromosome <br /> of DNA replication and segregation (Figure 6). Additionally,    segregation might arise due to simple mechanical defects in <br /> the daughter cells that remained attached to each other often   the microtubule attachment at the spindle pole and thus lead <br /> lost synchrony as revealed by the presence of the elongated     to an accidental elongation of the spindle. Alternatively, it <br /> anaphase spindle in one cell and the interphase microtu-        might represent a premature anaphase spindle elongation <br /> bules in the other (Figure 6, tubg1-13).                        before the proper biorientation of chromosomes. To distin- <br />                                                                 guish between these possibilities, we monitored sister chro- <br />                                                                 matid separation by transforming PY321 with the alanine- <br /> Metaphase-Anaphase Transition Defects in tubg1                  scanning mutants of tubg1. PY321 is an S. pombe strain that <br /> Mutants                                                         contains a GFP-LacI marker that binds to a region proximal <br /> A striking observation in chromosome segregation was dis-       to the centromere of chromosome I, which contains a tan- <br /> persed chromosomes that were left behind at the metaphase       dem, repeat of LacI-binding LacO sequences (Nabeshima et <br /> plate despite elongated anaphase B spindles (Figure 5A,         al., 1998). By monitoring the movement of the GFP signal, <br /> tubg1-5 and tubg1-22). Furthermore as shown in Table 2,         one can monitor sister chromatid segregation. If the first <br /> many cells showed the formation of multiple septa, indica-      possibility is correct, the missegregated chromosomes de- <br /> tive of exit from mitosis. These observations combined with     spite elongated spindles might simply represent unsepa- <br /> perturbed FACS profiles compelled us to investigate further     rated metaphase chromatid pairs. If, however, a premature <br />  <br />  <br /> Vol. 12, August 2001                                                                                                    2475 <br />  T.W. Hendrickson et al. <br />  <br />  <br />  <br /> Table 2. Summary of the expressivity of various phenotypes of the tubg1 cold-sensitive mutants <br />  <br />              Pear   Banana     Oval    Asym nuc     DNA seg     &#226;&#172;&#381;1 Nucleus     0 Nucleus    Asym sept     Mult sept   Obliq sept   Total <br /> Mutant       (%)      (%)      (%)        (%)         (%)          (%)            (%)          (%)          (%)          (%)        (%) <br />  <br /> TUBG1        0.49                                      1.48                       2.96         0.49                                 5.42 <br /> tubg1-4                        24.77      1.80         4.50                       0.90         1.80                                33.78 <br /> tubg1-5      1.01     3.54                3.03        20.71         2.02         15.15         3.54         11.62        3.03      63.64 <br /> tubg1-6      0.48              31.03      0.48         8.57                       3.81         0.43                                45.24 <br /> tubg1-7      2.00     0.50      6.00      6.50         3.50                       4.00         1.00          1.00                  24.50 <br /> tubg1-10     0.83     7.50     10.00      4.17         3.33                       4.17                                             30.00 <br /> tubg1-13     0.49     4.37                4.00        15.05         1.00          1.94         2.43         17.48        8.74      54.58 <br /> tubg1-14     2.04     4.08      7.65      1.02        27.04         0.51          4.08         2.04                                48.47 <br /> tubg1-15              1.65      4.13                   4.55                       0.41                                             10.74 <br /> tubg1-18     0.51     2.02                1.52        21.21                       4.55                       3.54        1.01      34.34 <br /> tubg1-22              2.00                2.00        18.41                       4.48         0.50          4.48                  31.84 <br /> tubg1-26                                               9.47                      12.63         6.32         10.00        2.63      41.05 <br /> tubg1-27              3.57                            17.34         1.53          9.69         5.10          7.14        2.55      46.94 <br />  <br /> pear, pear shaped; banana, curved shaped; oval, oval shaped; asym nuc, asymmetric localization of the nucleus; DNA seg, abnormal DNA <br /> segregation; &#226;&#172;&#381;1 nuc, reduced nuclear material; 0 nucleus, absence of nucleus; asym sept, misplaced septum; mult sept, multiple septa; and <br /> obliq sept, formation of septum that is not perpendicular to the long axis of the cell. <br />  <br />  <br />  <br />  <br /> anaphase is triggered, the centromeres must separate even             are located at the putative lateral contact sites of &#226;&#65533;&#165;-tubulin <br /> though the chromosomes fail to segregate to the poles. For            with other tubulins of the microtubule lattice. The recessive <br /> tubg1-13 and tubg1-18, we found that the sister chromatids            cs&#226;&#171;&#186; mutant tubg1-10 is also located at the putative lateral <br /> clearly separated while the chromatids failed to segregate to         tubulin contact site. Collectively, the results from these anal- <br /> the poles (Figure 7). These data combined with the occur-             yses provide strong support to the hypothesis that at least <br /> rence of abnormal septation suggests a bypass of the                  some of the &#226;&#65533;&#165;-tubulin interactions with the microtubule <br /> spindle assembly checkpoint and a possible role for &#226;&#65533;&#165;-tu-             lattice are equivalent to those of &#226;&#65533;&#163;-tubulin and &#226;&#65533;&#164;-tubulin. <br /> bulin in the coordination of sister chromatid segregation                There are also other deleterious mutations such as tubg1-2, <br /> and cytokinesis.                                                      tubg1-4, tubg1-5, tubg1-6, tubg1-7, tubg1-14, tubg1-15, tubg1- <br />                                                                       24, tubg1-26, and tubg1-27 that map to sites that do not lie at <br />                                                                       either the lateral or the longitudinal contact sites of &#226;&#65533;&#165;-tubulin <br /> DISCUSSION                                                            with other tubulin subunits. These mutant proteins can be <br /> We have taken advantage of the genetically amenable model             produced as soluble and stable proteins in an in vitro rabbit <br /> system, S. pombe, in an attempt to answer the question &#226;&#8364;&#339;Does          reticulocyte transcription/translation system. Thus, muta- <br /> &#226;&#65533;&#165;-tubulin have other functions besides microtubule nucle-             tions at these sites might not denature or destabilize the <br /> ation?&#226;&#8364;&#65533; This analysis yielded 12 cold-sensitive mutants of            protein. It is possible that these mutations either affect &#226;&#65533;&#165;-tu- <br /> TUBG1, thus providing powerful means to analyze &#226;&#65533;&#165;-tubu-               bulin interaction with other tubulin subunits by subtle long- <br /> lin function during the fission yeast cell cycle. Six of these        range perturbations in its structure, or these sites on &#226;&#65533;&#165;-tu- <br /> mutants were cold sensitive in the presence of the endoge-            bulin might be in contact with other nontubulin proteins. <br /> nous S. pombe gtb1&#226;&#171;&#185;, and thus were dominant. The other six            Hence, these mutations offer a possible means to further <br /> mutants conferred cold sensitivity in the absence of wild-            explore the interactions of &#226;&#65533;&#165;-tubulin with other heterologous <br /> type gtb1&#226;&#171;&#185;, and thus were recessive. Of the dominant cs&#226;&#171;&#186;              subunits of the &#226;&#65533;&#165;-tubulin complexes or other spindle pole <br /> mutants, tubg1-13, tubg1-18, tubg1-26, tubg1-27 were unable           body components. <br /> to support growth of S. pombe cells in the absence of endog-             Aside from the deleterious mutations, there are eight mu- <br /> enous &#226;&#65533;&#165;-tubulin, and thus conferred recessive lethality. Of           tations that do not appear to affect &#226;&#65533;&#165;-tubulin function. Sur- <br /> the 27 alanine-scanning mutants of &#226;&#65533;&#165;-tubulin, eight were              prisingly, some of these mutations such as tubg1-1, tubg1-3, <br /> able and 19 were unable to sustain growth of S. pombe cells.          tubg1-16, tubg1-19, and tubg1-21 lie in regions that are highly <br />    To understand the nature of &#226;&#65533;&#165;-tubulin interactions that are        evolutionarily conserved in &#226;&#65533;&#165;-tubulin between fission yeast <br /> perturbed in these deleterious mutations, we mapped each              and humans. How can cells tolerate these changes in the <br /> mutation on a minimal free energy homology model of                   regions of &#226;&#65533;&#165;-tubulin that were not allowed to change in 1 <br /> human &#226;&#65533;&#165;-tubulin constructed by with the use of the known              billion years due to the naturally ticking evolutionary clock? <br /> core structures of &#226;&#65533;&#163;-tubulin and &#226;&#65533;&#164;-tubulin and then mini-             It is possible that these conserved regions might provide <br /> mizing the free energy. Of the mutants that were unable to            some selective advantage to the growth of S. pombe cells in <br /> support the growth of S. pombe cells, tubg1-9, tubg1-11, tubg1-       the wild, but not in the amenities of a laboratory Petri dish. <br /> 13, tubg1-22, are located at the putative longitudinal contact        It is also possible that the conservation of these sites is due <br /> sites of &#226;&#65533;&#165;-tubulin with other tubulin subunits. Two muta-             to the coevolution of a &#226;&#65533;&#165;-tubulin binding protein that im- <br /> tions, tubg1-8 and tubg1-18, that conferred recessive lethality       posed constraints on the mutation of these conserved sites. <br />  <br />  <br /> 2476                                                                                                       Molecular Biology of the Cell <br />                                                       &#226;&#65533;&#165;-Tubulin in Anaphase and Cytokinesis <br />  <br />  <br />  <br />  <br /> Figure 5. Cold-sensitive tubg1 mutants undergo <br /> abnormal mitosis. Dominant cs&#226;&#171;&#186; cells (A and B), <br /> recessive cs&#226;&#171;&#186; cells (C and D), and wild-type cells <br /> expressing TUBG1 were grown to mid-log phase at <br /> 30&#194;&#176;C and then shifted to 18&#194;&#176;C (A and C) or left at <br /> 30&#194;&#176;C for 10 h (B and D). Cells were then fixed and <br /> microtubules were stained with the anti-&#226;&#65533;&#163;-tubulin, <br /> monoclonal antibody TAT-1 (1:50) followed by Cy3- <br /> conjugated goat anti-mouse secondary antibody. <br /> DNA was stained with 1 &#226;&#65533;&#174;g/ml DAPI. Arrows in- <br /> dicate mitotic cells. <br />  <br />  <br /> Vol. 12, August 2001                                                                 2477 <br />  T.W. Hendrickson et al. <br />  <br />  <br />  <br />  <br />                                                                                  Figure 6. Early phenotypes in partially synchro- <br />                                                                                  nized cells (20 mM HU, 3 h) reveal disrupted coor- <br />                                                                                  dination of mitosis and cytokinesis. After partial <br />                                                                                  synchronization, HU was removed and cells were <br />                                                                                  shifted to 18&#194;&#176;C for 4 &#226;&#8364;&#8220; 6 h. Cells were then fixed and <br />                                                                                  stained for microtubules and DNA as previously <br />                                                                                  described. <br />  <br />  <br />  <br />    Because &#226;&#65533;&#165;-tubulin is a component of the microtubule-            data suggest that in addition to its previously identified role <br /> organizing center that is required for the proper orientation      in the nucleation of spindle microtubules, &#226;&#65533;&#165;-tubulin may <br /> and organization of microtubules, we expected that these           also play a crucial role in the coordination of sister chroma- <br /> alanine-scanning &#226;&#65533;&#165;-tubulin mutants would have microtu-             tid separation in anaphase followed by cytokinesis. Real- <br /> bule organization defects (Stearns et al., 1991; Zheng et al.,     time analysis will be necessary to determine the precise <br /> 1991; Joshi et al., 1992). Similarly, because it had been deter-   extent of the loss of the coordination between different mi- <br /> mined that microtubule organization affects cell morphol-          totic stages. The defects in the coordination of mitosis are <br /> ogy and position of the interphase nucleus, we also expected       reflected in the gross perturbations of the cell cycle profiles <br /> those defects in the &#226;&#65533;&#165;-tubulin mutants (Walker, 1982; Verde        in the &#226;&#65533;&#165;-tubulin mutants compared with their wild-type <br /> et al., 1995; Hagan and Yanagida, 1997; Mata and Nurse,            control counterparts (Figure 4). <br /> 1997; Sawin and Nurse, 1998). Finally, we expected to find            Our hypothesis that &#226;&#65533;&#165;-tubulin is involved in chromosome <br /> mutations that affected the spindle would also block the           separation, septation, and cytokinesis is consistent with <br /> subsequent M phase events such as sister chromatid sepa-           some curious unexplained observations in the literature. For <br /> ration and cytokinesis because a damaged or abnormal spin-         example, Oakley et al. (1990) disrupted &#226;&#65533;&#165;-tubulin in A. nidu- <br /> dle activates spindle assembly checkpoint mechanisms re-           lans to generate a recessive lethal mutant that blocked nu- <br /> sponsible for delaying these subsequent postmetaphase              clear division and the spindle formation. However, there <br /> events. Surprisingly, in some mutants, despite the presence        was an increase in DNA staining while the number of cells <br /> of abnormal spindles, anaphase and cytokinesis progressed,         in mitosis did not increase. This indicated that although the <br /> leading to the generation of aneuploid or aploid cells. These      mutation resulted in a lack of a mitotic spindle, the cells did <br />  <br />  <br />  <br />  <br />                                                                                                     Figure 7. Sister chromatid sepa- <br />                                                                                                     ration in tubg1 mutants. GFP- <br />                                                                                                     marked centromere containing <br />                                                                                                     cells (PY321) were transformed <br />                                                                                                     with tubg1 alleles and the cultures <br />                                                                                                     were grown at 30&#194;&#176;C and then ei- <br />                                                                                                     ther left at 30&#194;&#176;C (B) or shifted to <br />                                                                                                     18&#194;&#176;C for 10 h (A). DNA was <br />                                                                                                     stained with DAPI. <br />  <br />  <br /> 2478                                                                                                    Molecular Biology of the Cell <br />                                                                                                   &#226;&#65533;&#165;-Tubulin in Anaphase and Cytokinesis <br />  <br />  <br /> not arrest in mitosis. This observation suggested that deple-        Marschall et al., 1996). Therefore, we predicted that the cs&#226;&#171;&#186; <br /> tion of &#226;&#65533;&#165;-tubulin not only inhibited spindle microtubule             &#226;&#65533;&#165;-tubulin mutant alleles might display defects in microtu- <br /> assembly but also allowed exit from mitosis in the absence of        bule assembly and organization at the restrictive tempera- <br /> a mitotic spindle. This scenario is different from the mitotic       ture. Like other mutations that affect microtubule assembly, <br /> arrest that results from microtubule disassembly induced             tubg1 mutant alleles might block the progression of an- <br /> either by treatment with microtubule-depolymerizing drugs            aphase and cytokinesis that would normally arrest cells in M <br /> (Hoyt et al., 1991; Li and Murray, 1991) or by mutations in          phase. To our surprise, none of our mutants showed a <br /> &#226;&#65533;&#164;-tubulin (Hiraoka et al., 1984). In addition, Paluh et al.          complete loss of microtubules although some mutants had <br /> (2000) have reported that in the presence of a mutant &#226;&#65533;&#165;-tu-          fewer microtubules. The most pronounced defects in micro- <br /> bulin allele in S. pombe, proper chromosome segregation is           tubules were curved spindles (Figure 6). One surprising <br /> dependent on the presence of a checkpoint gene mad2&#226;&#171;&#185;.                result that emerged from our observations was that four <br /> Furthermore, a mutant allele of the S. cerevisiae homolog of         mutants, tubg1-7, tubg1-13, tubg1-18, and tubg1-26, did not <br /> &#226;&#65533;&#165;-tubulin, TUB4, requires a mitotic checkpoint control gene,         seem to arrest in mitosis but seemed to segregate chromo- <br /> BUB2, for cell cycle arrest and <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>survival</span> (Spang et al., 1996). In    somes and accomplish cytokinesis, albeit abnormally, pro- <br /> addition, a C-terminal deletion in Tub4p causes mitotic de-          ducing aneuploid or aploid cells with misplaced septa. Al- <br /> fects, apparently unrelated to microtubule nucleation (Vogel         though three of these, tubg1-13, tubg1-18, and tubg1-26, arrest <br /> and Snyder, 2000). Together, these data suggest a possible           after the completion of mitosis, presumably in the G1 phase <br /> role for &#226;&#65533;&#165;-tubulin in regulating not only spindle microtubule        of the next cell cycle, tubg1-7 seemed to accumulate 4N DNA <br /> assembly but also the coordination of anaphase and exit              content (Figure 4). Collectively, our data suggest that &#226;&#65533;&#165;-tu- <br /> from mitosis.                                                        bulin might be involved not only in microtubule nucleation <br />    The cell cycle is closely monitored to verify that DNA is         and organization but also, surprisingly, in the coordination <br /> properly replicated before the cell enters mitosis as well as to     of postmetaphase events such as chromosome segregation <br /> ensure that once the cell has entered mitosis, sister chroma-        and septation during cytokinesis. <br /> tids are attached to microtubules from both poles before <br /> anaphase and subsequent cytokinesis are allowed to <br /> progress. These surveillance mechanisms, which monitor <br /> the timely and proper progression of the cell cycle are called        <br /> checkpoints (Hartwell and Weinert, 1989). Two prominent <br /> checkpoints that monitor mitosis are the spindle assembly            <br /> checkpoint (reviewed in Amon, 1999) and the exit from                <br /> mitosis (reviewed in Cerutti and Simanis, 1999; Balasubra-           <br /> manian et al., 2000; Hoyt, 2000). This system of mitotic             <br /> checkpoints has been well elucidated in the budding yeast             <br /> and homologs have been identified in other systems, includ-          <br /> ing fission yeast, Xenopus, and mammalian systems (re-               <br /> viewed in Amon, 1999). The spindle checkpoint monitors               <br /> attachment of the sister chromatids to properly assembled            <br /> spindle microtubules, thereby preventing premature an-                <br /> aphase from occurring. One of the major players involved in           <br /> this system of checkpoints is a spindle pole protein, Cdc16p         <br /> (the fission yeast homolog of Bub2p). Cdc16p is a negative           <br /> regulator of the septation initiation pathway involved in <br /> ensuring that sister chromatids are properly segregated to <br /> opposite poles before the cells undergo septation, signaling <br /> exit from mitosis (Fankhauser et al., 1993). Cdc16p is also <br /> required to prevent septation from occurring during inter-          <br /> phase (Cerutti and Simanis, 1999). Mutations in &#226;&#65533;&#165;-tubulin, <br />                                                                      <br /> particularly tubg1-13, are reminiscent of mutations in the <br />                                                                      <br /> genes, such as cdc16&#226;&#171;&#185;, involved in the septation initiation          <br /> pathway (reviewed in Gould and Simanis, 1997). The <br /> tubg1-13 phenotype of septum formation without cytokine-              <br /> sis is also similar to that of ppb1 (Yoshida et al., 1994) and        <br /> sep1 (Sipiczki et al., 1993), both of which are also defective in     <br /> cytokinesis. Thus, these data suggest that &#226;&#65533;&#165;-tubulin may be           <br /> involved in the spindle assembly checkpoint that prevents <br />                                                                     <br /> sister chromatid separation and cytokinesis in response to            <br /> spindle damage. <br />    &#226;&#65533;&#165;-Tubulin is required for the nucleation of microtubule           <br /> assembly in vivo in mammalian cells (Joshi et al., 1992) and          <br /> for spindle assembly and function in S. pombe, S. cerevisiae, A.      <br /> nidulans, and Drosophila (Oakley et al., 1990; Horio et al., 1991;    <br /> Sobel and Snyder, 1995; Sunkel et al., 1995; Spang et al., 1996; <br /> </body></html>