<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Allele specific synthetic lethality between priC and dnaAts alleles at <br /> the permissive temperature of 30&#194;&#176;C in E. coli K-12 <br />  <br />  <br />                  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                  Background: DnaA is an essential protein in the regulation and initiation of DNA replication in <br />                  many bacteria. It forms a protein-DNA complex at oriC <span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span> which DnaC loads DnaB. DNA <br />                  replication forks initiated at oriC by DnaA can collapse on route to the terminus for a variety of <br />                  reasons. PriA, PriB, PriC, DnaT, Rep and DnaC form multiple pathways to restart repaired <br />                  replication forks. DnaC809 and dnaC809,820 are suppressors of priA2::kan mutant phenotypes. The <br />                  former requires PriC and Rep while the latter is independent of them. RnhA339::cat mutations allow <br />                  DnaA-independent initiation of DNA replication. <br />                  Results: It is shown herein that a priC303::kan <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-164' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-209' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-224' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-237' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>mutation</span> is synthetically lethal with either a dnaA46 <br />                  or dnaA508 temperature sensitive mutation at the permissive temperature of 30&#194;&#176;C. The priC-dnaA <br />                  lethality is specific for the dnaA allele. The priC303::kan mutant was viable when placed in <br />                  combination with either dnaA5, dnaA167, dnaA204 or dnaA602. The priC-dnaA508 and priC-dnaA46 <br />                  lethality could be suppressed by rnhA339::cat. The priC-dnaA508 lethality could be suppressed by a <br />                  dnaC809,820 mutation, but not dnaC809. Neither of the dnaC mutations could suppress the priC- <br />                  dnaA46 lethality. <br />                  Conclusions: A hitherto unknown function for either DnaA in replication restart or PriC in <br />                  initiation of DNA replication that occurs in certain dnaA temperature sensitive mutant strains at <br />                  the permissive temperature of 30&#194;&#176;C has been documented. Models considering roles for PriC <br />                  during initiation of DNA replication and roles for DnaA in replication restart were tested and found <br />                  not to decisively explain the data. Other roles of dnaA in transcription and nucleoid structure are <br />                  additionally considered. <br />  <br />  <br />  <br />  <br /> Background                                                                    dependent on the DnaA and DnaC proteins (reviewed in <br /> The loading of the DnaB replicative helicase at the E. coli                   [2-4]). In vitro, in the presence of several other accessory <br /> origin of DNA replication (oriC) is a highly regulated and                    proteins (i.e. RNA polymerase, DNA gyrase, HU protein), <br /> is thought to be a key step in the assembly of the repli-                     multiple DnaA proteins bind to four asymmetric 9 bp <br /> some. DnaB makes important contacts with the &#207;&#8222;-subunit                        DnaA binding sites in the 225 bp oriC region allowing for- <br /> of DNA Polymerase III Holoenzyme and DNA primase                              mation of a protein/DNA complex [5-7]. This in turn <br /> [1]. DnaB loading at oriC during initiation of DNA repli-                     causes melting of a nearby AT rich sequence. A complex of <br /> cation is a sequence specific, cell cycle regulated event <br />  <br />                                                                                                                                          Page 1 of 9 <br />                                                                                                                  (page number not for citation purposes) <br />  BMC Microbiology 2004, 4:47                                                     http://www.biomedcentral.com/1471-2180/4/47 <br />  <br />  <br />  <br /> Table 1: dnaA alleles used in this work and some phenotypes <br />  <br />               Allele                Amino Acid Change    Domain Affecteda       Ability to replicate P+          Ability to grow in <br />                                                                                  plasmids at 30&#194;&#176;Cb              priC303::kan at 30&#194;&#176;C <br />  <br />              dnaA46                        A184V                 III                     No                              No <br />                                            H252Y                 III <br />             dnaA508                         P28L                  I                      No                              No <br />                                             T80I                  I <br />              dnaA5                         A184V                 III                     Yes                             Yes <br />                                            G426S                 IV <br />             dnaA167                        V157E                 III                     Yes                             Yes <br />             dnaA204                        I389N                 IV                      No                              Yes <br />             dnaA602                        A184V                 III                     Yes                             Yes <br />                                            A347V              linker <br />  <br />  aDomains    are defined according to [25, 35, 36]. <br />  b Data   <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span> [34] <br />  <br />  <br />  <br />  <br /> DnaB6-DnaC6 then interacts with the DnaA/oriC complex              mutants suggested the availability of alternate pathways. <br /> to load DnaB at the AT rich sequence.                              The PriA-independent pathway depends on PriC and Rep <br />                                                                    [17]. Two types of priA suppressor mutations have been <br /> It is thought that DnaA may have other roles in the cells          found and both map to the dnaC gene. The first typified <br /> in addition to initiation. These additional roles stem from        by dnaC809 (E176G) and is dependent on the genes in the <br /> the fact that there are many DnaA binding sites in the             PriA-independent pathway of replication restart, rep and <br /> chromosome outside the oriC region [8,9] and that DnaA             priC [17]. The second type of priA suppressor has an addi- <br /> binding to these asymmetric 9 bp sites, can bend the DNA           tional mutation (K178N) in dnaC relative to dnaC809 and <br /> [10]. It can be hypothesized based on large number of              makes this protein&apos;s suppression of priA mutant pheno- <br /> potential DnaA binding sites in the chromosome and the             types independent of priC and rep. This dnaC allele is <br /> ability of DnaA to bend DNA that it may influence the              called dnaC809,820 [17,18]. The multiplicity of replica- <br /> structure of nucleoid. It has been shown that if a DnaA            tion restart pathways may be a general property in Bacteria <br /> binding site falls within a promoter region that mutations         since Bacillus subtilis also has a similar arrangement of <br /> in dnaA can affect the level of transcription from that pro-       PriA-dependent and PriA-independent pathways [19,20]. <br /> moter [11-14]. Thus, mutations in dnaA may have global <br /> effects in gene expression and nucleoid structure as well as       As mentioned above, DnaA-dependent initiation of DNA <br /> affecting initiation of DNA replication at oriC.                   replication at oriC and replication restart share several <br />                                                                    properties. The most important of these is that both strive <br /> In E. coli, dnaA is an essential gene. Several different dnaA      to make protein-DNA complexes that are recognized by <br /> temperature sensitive mutant alleles have been isolated.           the DnaC protein so that DnaB can be loaded. Previous <br /> Many of these share the property that they are double              work by Kogoma and colleagues showed that oriC-DnaA- <br /> mutants (dnaA5, dnaA46, dnaA508 and dnaA602 &#226;&#8364;&#8220; Table 1              independent initiation of DNA replication could take <br /> and [15]). Several of the double mutants share a muta-             place in an rnhA mutant strain [16,21]. This type of initi- <br /> tion: a change at codon 184 that replaces an alanine with          ation of DNA replication was termed Constitutive Stable <br /> a valine. Additionally, a dnaA850::Tn10 mutant has been            DNA Replication (cSDR) and is dependent on both <br /> isolated. This is only viable in strains that have an alter-       recombination and replication restart functions (reviewed <br /> nate, oriC-independent method of initiation of DNA rep-            in [22] and Sandler, submitted). <br /> lication [16]. <br />                                                                    To begin testing the roles of priB and priC in cSDR <br /> The loading of DnaB by DnaC in E. coli can occur away              (reviewed in [22]) required the construction of dnaAtsrnhA <br /> from oriC at a repaired replication fork. This is governed         priB and dnaAtsrnhA priC triple mutant strains. However <br /> by the Replication Restart Proteins (RRPs): PriA, PriB,            when trying to construct these strains, we found that priC <br /> PriC, DnaT and Rep. The genes coding for these products            was required for growth in two different dnaAts strains, <br /> form multiple pathways to identify the proper substrate            dnaA46 and dnaA508, at the permissive temperature of <br /> and then help DnaC load DnaB (Figure 1 and [17]). Since            30&#194;&#176;C. When priC303::kan was tested with other dnaAts <br /> replication restart is thought to be an essential process, the     alleles, the synthetic lethality was found to be allele spe- <br /> poor viability (versus complete inviability) of priA               cific. Two different mutations were found to suppress the <br />  <br />  <br />                                                                                                                             Page 2 of 9 <br />                                                                                                     (page number not for citation purposes) <br />  BMC Microbiology 2004, 4:47                                                       http://www.biomedcentral.com/1471-2180/4/47 <br />  <br />  <br />  <br />  <br /> Figure <br /> This <br /> in E. diagram <br />       coli 1     compares the ways in which the replicative helicase can be loaded either from oriC or a repaired replication fork <br /> This diagram compares the ways in which the replicative helicase can be loaded either from oriC or a repaired replication fork <br /> in E. coli. Left side of the diagram indicates the starting substrate to which the replisome is to be loaded. The horizontal arrows <br /> indicate the way in which the proteins may interact to load the replisome. The dotted lines represent suppressor pathways. <br />  <br />  <br />  <br />  <br /> priC-dnaA lethality. One was rnhA339::cat, a non-allele             and dnaA508 mutation may be synthetically lethal at the <br /> specific suppressor of dnaA mutants. The other was                  permissive temperature of 30&#194;&#176;C. It is also formally possi- <br /> dnaC809,820, a suppressor of priC and priA mutations.               ble that priC303::kan suppressed the temperature sensitive <br /> The first type suppressed both the dnaA508-priC and                 nature of dnaA508. These two possibilities are tested <br /> dnaA46-priC lethality while the latter only suppressed the          below. <br /> dnaA508-priC lethality. These studies suggest that priC <br /> may have a role in initiation of DNA replication at oriC            Since it is known that the absence of other gene products <br /> with certain dnaA alleles and or that dnaA may have an              (i.e., rnhA [23] and trxA [24]) can suppress the essentiality <br /> additional role in the cell important for replication restart.      of dnaA, it is possible that the absence of priC might also <br />                                                                     suppress the temperature sensitivity of the dnaAts allele. If <br /> Results                                                             so, then one should be able to detect the presence of the <br /> PriC, but not priB, is required for growth in a dnaA508             dnaA mutation on the chromosome of the 42&#194;&#176;C resistant <br /> mutant at the permissive temperature of 30&#194;&#176;C                        transductants. To test this possibility, TetR transductants <br /> We began this study by asking if priB and priC are required         selected at 30&#194;&#176;C were screened for a 42&#194;&#176;C R <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. <br /> for cSDR. During this study it was found that certain               These were then further screened for the presence of a <br /> dnaAts could be introduced into a strain containing a priB          restriction site polymorphism (a Eco NI site) created by <br /> mutation, but not into a strain containing a priC muta-             the dnaA508 mutation. To do this, the dnaA region from <br /> tion. The standard P1 transductional cross used for the             the 42&#194;&#176;C R TetR transductants was amplified by standard <br /> introduction of the dnaAts mutant alleles is shown in Fig-          Polymerase Chain Reaction methods using the primers, <br /> ure 2. Table 2 shows that the co-transduction frequency             prSJS480 and prSJS481 (Table 3). The amplified DNA was <br /> between tnaA300::Tn10 and dnaA508 in a wild type strain             then restricted with Eco NI. Examination of eight inde- <br /> is 92% (49/53). Using a priB mutant strain as the recipi-           pendent 42&#194;&#176;CR TetR transductants, constructed using <br /> ent,    the     co-transduction      frequency     between          JC12390 as the donor, revealed no case in which a restric- <br /> tnaA300::Tn10 and dnaA508 was approximately the same                tion <span id='am-239' about='xsp:pattern' typeof='owl:Thing'>pattern</span> was consistent with the presence of the tem- <br /> as when the wild type strain was used as the recipient              perature sensitive allele (data not shown). From these <br /> (data not shown). Surprisingly, when a priC mutant was              results, it is concluded that the priC303::kan mutation <br /> used as a recipient, the co-transduction frequency was 0/           does not suppress the absence of dnaA508 and is synthet- <br /> 72 or 0% (Table 2). This suggested that the priC303::kan            ically lethal with it. <br />  <br />  <br />                                                                                                                            Page 3 of 9 <br />                                                                                                    (page number not for citation purposes) <br />  BMC Microbiology 2004, 4:47                                                                   http://www.biomedcentral.com/1471-2180/4/47 <br />  <br />  <br />  <br />  <br /> The <br /> Figure <br />      tnaA- <br />         2 dnaA region of the E. coli chromosome is diagramed on the lower line <br /> The tnaA- dnaA region of the E. coli chromosome is diagramed on the lower line. The upper line is symbolic of the DNA intro- <br /> duced by the P1 transduction in the standard cross described in this paper where a tnaA300::Tn10 dnaAts donor is introduced <br /> to a dnaA+ recipient. Potential crossover events between the two markers are shown. <br />  <br />  <br />  <br />  <br /> Table 2: P1 crosses using dnaAts strains as the donors and isogenic priC303::kan and dnaC mutants as the recipientsa <br />  <br />                    Donor                    Recipient Strain Number                      Recipient Genotype                         Ts/total <br />  <br />    Strain Number             dnaA                                                 priC                    dnaC <br />       JC12390                508                    JC13509                        +                        +                        49/53 <br />        AL0454                 46                    JC13509                        +                        +                        60/69 <br />       JC12390                508                      SS145                       303                       +                         0/72 <br />        AL0454                 46                      SS145                       303                       +                         0/72 <br />        SS1750                  5                      SS145                       303                       +                         7/12 <br />        SS1751                167                      SS145                       303                       +                        11/11 <br />       WM433                  204                      SS145                       303                       +                          4/5 <br />        SS1752                602                      SS145                       303                       +                        10/12 <br />       JC12390                508                     SS1099                       303                      809                        0/63 <br />       JC12390                508                     SS1100                       303                    809,820                     41/48 <br />        AL0454                 46                     SS1099                       303                      809                        0/32 <br />        AL0454                 46                     SS1100                       303                    809,820                      0/24 <br />  <br />  aThese are the results of either single or multiple transductions. The numbers show the number of TetR transductants that were also sensitive to <br />  growth at 42&#194;&#176;C. If the transductions yielded temperature sensitive transductants then only a few transductants from one transduction is reported. <br />  The recipient cells were grown at 37&#194;&#176;C in Luria broth. They were then treated with donor lysate made on the tnaA300::Tn10 dnaAts strain <br />  indicated. The transductions were then spread on Luria plates containing 10 &#194;&#181;gm/ml tetracycline and incubated 24&#226;&#8364;&#8220;48 hours at 30&#194;&#176;C. Transductants <br />  were then picked, purified and tested for growth at 42&#194;&#176;C by replica plating. <br />  <br />  <br />  <br />  <br /> PriC303::kan is synthetically lethal with dnaA46 and                        thetic lethality at the permissive temperature of 30&#194;&#176;C is <br /> dnaA508, but not with dnaA5, dnaA167, dnaA204 or                            allele specific or occurs with all dnaAts alleles. To <br /> dnaA602                                                                     distinguish between these two possibilities, several other <br /> It is possible that either the dnaA508-priC303::kan syn-                    dnaAts alleles were tested. Selection of a diverse collection <br />  <br />  <br />                                                                                                                                        Page 4 of 9 <br />                                                                                                                (page number not for citation purposes) <br />  BMC Microbiology 2004, 4:47                                                        http://www.biomedcentral.com/1471-2180/4/47 <br />  <br />  <br />  <br /> Table 3: List of Oligonucleotide Primers <br />  <br />  Name                                      5&apos; to 3&apos; oligonucleotide sequence             Position <br />  <br />  prSJS283                                  ATATTGAGTGTTGTCAGC                            Upstream of priC <br />  prSJS284                                  TCCTCCAGCAGCACAATC                            Downstream of priC <br />  prSJS480                                  CCGCGGTCCCGATCGTTTTG                          dnaA upstream primer <br />  prSJS481                                  GCAGGGCGTTGAAGGTGTGG                          dnaA downstream primer <br />  <br />  <br />  <br />  <br /> of mutations to test was aided by an already large reper-            wild type co-transduction frequency of 90% (data not <br /> toire of characterized dnaAts alleles [15] and the recent elu-       shown). <br /> cidation of the crystal structure of DnaA from Aquifex <br /> aeolicus [25]. This allowed the selection of several temper-         It was then tested if the synthetic lethality was due to polar <br /> ature sensitive dnaA alleles that had different amino acids          effects of dnaA508 or dnaA46 on downstream dnaN-recF <br /> substitutions in different parts of the protein (Table 1).           expression. This was tested in a similar way. A plasmid, <br /> Hence, it was attempted to introduce dnaA5, dnaA46,                  pAB3 [27], that expresses the dnaA gene in trans was intro- <br /> dnaA167, dnaA204 and dnaA602 into the priC303::kan                   duced into the priC303::kan mutant strain (SS145). This <br /> strain (SS145) using the selectable marker tnaA300::Tn10             strain was then used as a recipient in a cross with either <br /> as before. Table 2 shows that the synthetic lethality only           ALO450 (dnaA46 tnaA300::Tn10) or JC12390 (dnaA508 <br /> occurred additionally with dnaA46, but not with dnaA5,               tnaA300::Tn10). TetR transductants were selected at 30&#194;&#176;C. <br /> dnaA167, dnaA204 or dnaA602. It is concluded that the                In each case, several transductants were selected and <br /> synthetic lethality between priC303::kan and dnaA508 and             screened for the presence of the dnaAts mutation by <br /> dnaA46 at 30&#194;&#176;C is allele specific.                                   backcrosses to JC13509. The temperature sensitive pheno- <br />                                                                      type associated with dnaA508 and dnaA46 was detected in <br /> The priC303::kan dnaAts synthetic lethality is solely due            each case (data not shown). <br /> to the absence of only priC and the presence of the dnaAts <br /> mutation                                                             It is concluded that the synthetic lethality seen between <br /> Since both the dnaA and priC genes are in operons, it is             priC303::kan and dnaA508 or dnaA46 is due to solely the <br /> possible that the synthetic lethality seen above is due to           absence of priC and the presence of the dnaAts mutation. <br /> not just the mutation in priC or dnaA, but also due to that <br /> mutation and or polar effects on downstream genes                    RnhA mutations suppress the priC-dnaA synthetic <br /> within their respective operons. While the potential polar           lethality <br /> effects of a priC303::kan <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>insertion mutation</span> are easily              It has been shown that rnhA mutations are non-allele spe- <br /> envisioned, the potential polar effects of a dnaA missense           cific suppressors of both dnaAts and dnaA insertion muta- <br /> mutation are less obvious. This is tested here as well               tions [16]. The mechanism of suppression is thought to be <br /> because, as introduced above and discussed below, dnaA               the stabilization of R-loops on the chromosome [22]. To <br /> mutations can have effects on the level of transcription of          determine if the priC-dnaA synthetic lethality is sup- <br /> promoters in which there are DnaA binding sites. Since               pressed by a mutation in rnhA, it was attempted to intro- <br /> dnaA binds to its own promoter and autoregulates its own             duce dnaA508 and dnaA46 into a priC303::kan <br /> expression [12,26], it is possible that dnaA mutation may            rnhA339::cat (SS1531) double mutant strain. It was found <br /> effect transcription from its own promoter and subse-                that this dnaAts rnhA priC triple mutant combination was <br /> quently effect dnaN and recF expression.                             viable in each case (see SS1543 and SS3032 in Table 4). <br />                                                                      This suggested that the cause of the priC-dnaA lethality was <br /> It was first tested if the synthetic lethality between the           a defect in the mutant dnaA protein&apos;s ability to initiation <br /> dnaA508 and priC303::kan was dependent solely on the                 of DNA replication and that priC has some role in <br /> priC gene. This was necessary to determine because                   initiation of DNA replication in the dnaA508 and dnaA46 <br /> priC303::kan is an insertion mutation and could be polar             mutant strains. <br /> on the downstream gene, ybaM. This was tested by cloning <br /> priC into a plasmid (pTH1, see Methods) and seeing if the            DnaC809,820, but not dnaC809, suppresses the absence <br /> priC plasmid could complement the synthetically lethal               of priC in the dnaA508 mutant at 30&#194;&#176;C <br /> phenotype. pTH1, containing just the priC promoter and               The above experiment suggested that PriC has a role in ini- <br /> gene, in the priC303::kan mutant strain (SS145), allowed             tiation of DNA replication in certain dnaA mutants. If so, <br /> the dnaA508 allele to be introduced into that strain at the          then suppressors of priC&apos;s role in replication restart should <br />  <br />  <br />                                                                                                                             Page 5 of 9 <br />                                                                                                     (page number not for citation purposes) <br />  BMC Microbiology 2004, 4:47                                                                   http://www.biomedcentral.com/1471-2180/4/47 <br />  <br />  <br />  <br /> Table 4: Strain List <br />  <br />  Strain                          priC                   dnaA                   dnaC            Relevant Genotype           Source or Derivation <br />  <br />  ALO454                           +                     46                      +              tnaA300::Tn10               Kirsten Skarstad <br />  AQ12251                          +                      +                      +              rnhA339::cat                T. Kogoma <br />  CAG18442                         +                      +                      +              thr-34::Tn10                [38] <br />  CM740                            +                      5                      +                                          Kirsten Skarstad <br />  CM2556                           +                     167                     +                                          Kirsten Skarstad <br />  CM2733                           +                     602                     +                                          Kirsten Skarstad <br />  JC12390                          +                     508                     +              tnaA300::Tn10               Lab stock <br />  JC13509a                         +                      +                      +                                          [18] <br />  JC19008                          +                      +                     809             priA2::kan                  [28] <br />  JC19165                         303                     +                      +                                          [18] <br />  JC19257                          +                      +                   809,820           priA2::kan                  [18] <br />  SS145                           303                     +                      +                                          JC19165 -&gt; JC13509b <br />  SS1091                           +                      +                     809                                         JC19008 -&gt; SS1213c <br />  SS1092                           +                      +                   809,820                                       JC19257 -&gt; SS1213c <br />  SS1099                          303                     +                     809                                         JC19165 -&gt; SS1091b <br />  SS1100                          303                     +                   809,820                                       JC19165 -&gt; SS1092b <br />  SS1201                           +                     508                     +              tnaA300::Tn10               JC12390 -&gt; JC13509e <br />  SS1213                           +                      +                      +              thr-34::Tn10                CAG18442 -&gt; JC13509d <br />  SS1504                          303                    508                  809,820           tnaA300::Tn10               JC12390 -&gt; SS1100e <br />  SS1531                          303                     +                      +              rnhA339::cat                AQ12251 -&gt; SS145 <br />  SS1543                          303                    508                     +              rnhA339::cat                JC12390 -&gt; SS1531f <br />  SS1738                           +                      +                      +              tnaA300::Tn10               Lab Stock <br />  SS1750                           +                      5                      +              tnaA300::Tn10               SS1738 -&gt; CM740e <br />  SS1751                           +                     167                     +              tnaA300::Tn10               SS1738 -&gt; CM2556e <br />  SS1752                           +                     602                     +              tnaA300::Tn10               SS1738 -&gt; CM2733e <br />  SS1793                          303                    167                     +              tnaA300::Tn10               SS1751 -&gt; SS145e <br />  SS1796                          303                    602                     +              tnaA300::Tn10               SS1752 -&gt; SS145e <br />  SS1797                          303                     5                      +              tnaA300::Tn10               SS1750 -&gt; SS145e <br />  SS1798                          303                    204                     +              tnaA300::Tn10               WM433 -&gt; SS145e <br />  SS3032                          303                    46                      +              rnhA339::cat                ALO454 -&gt; SS1531f <br />  WM433                            +                     204                     +              tnaA300::Tn10               Mark Sutton <br />  <br />  a JC13509   has the following genotype: sulB103 lacMS286 &#206;&#166; 80dIIlacBK1 argE3 his-4 thi-1 xyl-5 mtl-1 SmR T6R. The lacMS286 &#206;&#166; 80dIIlacBK1 <span id='am-3' about='Thesaurus:code' typeof='owl:Thing'>code</span> for <br />  two partial copies of the lac operon [39, 40]. <br />  bSelect for kanamycin resistance on minimal media. <br />  cSelect for threonine prototrophy on minimal media. Screen for the presence of the dnaC mutation by the method described in [18]. <br />  dSelect for tetracycline resistance on rich media and screen for threonine auxotrophy. <br />  eSelect for tetracycline resistance on rich media at 30&#194;&#176;C and then screen for temperature sensitive growth at 42&#194;&#176;C. <br />  fSelect for tetracycline resistance on rich media at 30&#194;&#176;C and then screen for restriction site polymorphism and or backcross to test the presence of <br />  the temperature sensitive allele. <br />  <br />  <br />  <br />  <br /> not suppress the priC-dnaA synthetic lethality. Two types                    either JC12390 (dnaA508) or AL0450 (dnaA46). Table 2 <br /> of replication restart suppressor mutations are known and                    shows that when dnaC809,820 was used as the recipient <br /> were tested [18,28]. DnaC809 suppresses the phenotypes                       and JC12390 as the donor that 41/48 or 83% of the TetR <br /> of priA2::kan and dnaT822 [28,29]. In vitro, DnaC809 can                     transductants were also temperature sensitive (they inher- <br /> suppress the absence of all RRPs on several different sub-                   ited the dnaA508 allele). However, when only dnaC809 <br /> strates [30]. In vivo however, priA2::kan suppression                        was used as the recipient, 0/63 TetR transductants inher- <br /> requires priC and rep [17]. DnaC809 can be additionally                      ited the temperature sensitive phenotype. It is concluded <br /> mutated to make the priA suppression both priC and rep                       that dnaC809,820 can suppress the absence of priC in the <br /> independent [17]. This additionally mutated dnaC allele                      dnaA508 mutant and dnaC809 cannot. The dnaA46 allele <br /> is called dnaC809,820 [18]. To test the above hypothesis,                    was additionally tested and was not suppressed by either <br /> priC303::kan dnaC809 (SS1099) and priC303::kan                               dnaC809 or dnaC809,820 (Table 3). From this it is con- <br /> dnaC809,820 (SS1100) strains were constructed (Table 4)                      cluded that dnaC809,820 is able to suppress the absence <br /> and used as recipients in crosses with the donor P1 from                     of some dnaAts allele. In contradiction to the suggestion of <br />  <br />  <br />                                                                                                                                          Page 6 of 9 <br />                                                                                                                  (page number not for citation purposes) <br />  BMC Microbiology 2004, 4:47                                                       http://www.biomedcentral.com/1471-2180/4/47 <br />  <br />  <br />  <br /> the above rnhA experiment, this result suggests that dnaA           Domain III near the ATP binding region. Their second <br /> may have some role in replication restart necessary in a            mutations cause amino acid replacements in other <br /> priC mutant.                                                        domains. Unfortunately, the positions of the second <br />                                                                     changes yield no clues about what might make dnaA508 <br /> Discussion                                                          and dnaA46 mutants require priC for growth at 30&#194;&#176;C and <br /> This paper shows that priC, a gene involved in both the             why the other four dnaA mutants do not. <br /> PriA-dependent and PriA-independent pathways for repli- <br /> cation restart, is also required for cell viability in two of six   What might PriC be doing to help initiation of DNA rep- <br /> dnaAts mutants at the permissive growth temperature of              lication in a dnaAts strain? One idea is that the dnaAts pro- <br /> 30&#194;&#176;C. These results were surprising on at least two                 tein is defective in its ability to create a region of ssDNA at <br /> accounts. The first is that in vitro systems for either replica-    oriC. Since the RRPs are also thought to help create regions <br /> tion restart or initiation of DNA replication at oriC posed         of ssDNA (away from oriC) so that DnaC binds and loads <br /> no requirement for the DnaA or PriC protein respectively.           DnaB, it is possible that PriC may help the DnaA mutants <br /> The second is that priC has no known role in vivo in initi-         in this endeavor. Another type of model that is formally <br /> ation of DNA replication (the only reported role is in rep-         possible is that PriC may somehow stabilize the dnaAts <br /> lication restart [17]) and that dnaA has no known role in           protein. This seems unlikely, however, given that <br /> replication restart.                                                dnaC809,820 can rescue the synthetic lethality of <br />                                                                     priC303::kan and dnaA508. Other models may also be <br /> One way to answer the question of why the mutations are             possible. <br /> synthetically lethal is to see what types of mutations may <br /> suppress the lethality. RnhA339::cat, a non-allele specific         One needs to consider if DnaA may be involved in repli- <br /> suppressor of dnaA mutants role in initiation of DNA rep-           cation restart. In considering this, one needs to remember <br /> lication, could suppress the priC-dnaA synthetic lethality          that DnaA has the ability to bind DNA at specific sites and <br /> for both dnaAts mutant alleles. Such suppression is strong          bend it. It has been shown that different dnaAts allele can <br /> evidence that priC and dnaA may both be missing a func-             differentially influence the rate of initiation of transcrip- <br /> tion needed during initiation of DNA replication. It was            tion in some promoters with DnaA binding sites (see <br /> further observed, however, that dnaC809,820 (but not                below). This in turn can influence replication restart in <br /> dnaC809) could suppress the absence of priC in the                  two ways. First changes in the level of gene expression of <br /> dnaA508 mutant. Neither dnaC809 nor dnaC809,820                     a single gene (or groups of genes) may indirectly influence <br /> could suppress the dnaA46-priC synthetic lethality.                 the replication restart process. Second, the ability of DnaA <br /> DnaC809,820 is a PriC-independent suppressor of several             to bind to many sites on the chromosome may influence <br /> genes required in replication restart. This suppression             structure of the nucleoid and the sites at which replication <br /> implicates dnaA in replication restart. Thus the inferences         restart may occur. <br /> from the two types of suppressors seem to contradict one <br /> another.                                                            There are many examples where several phenotypes had <br />                                                                     been tested systematically for several dnaA alleles (Table 1 <br /> What function(s) in dnaAts strains are missing for initia-          and [31-33]). The only other system that seems to have <br /> tion of DNA replication that make the strain dependent              some similarity to the data here is one in which the ability <br /> on priC at 30&#194;&#176;C? A structure-function analysis of DnaA              to replicate &#206;&#187; P+ plasmids was investigated [27,34]. Table <br /> would help to answer this question. Briefly, based on               1 shows that dnaA46, dnaA204 and dnaA508 all fail to rep- <br /> alignments of DnaA proteins, the X-ray crystal structure of         licate these plasmids while dnaA5, dnaA167 and dna602 <br /> the DnaA protein from Aquifex aeolicus and much research            can. The model proposed to explain this phenomenon <br /> on the genetics and biochemistry of DnaA, the DnaA pro-             suggests that DnaA is required to activate transcription at <br /> tein can be divided into four domains with four proposed            the &#206;&#187; PR promoter and that the dnaA46, dnaA204 and <br /> functions: Domain I) DnaB recruitment, Domain II)                   dnaA508 mutations decrease this ability[27]. With the <br /> Linker region, Domain III) ATP binding and Domain IV)               exception of dnaA204, the inability to replicate these plas- <br /> DNA binding [25,35,36]. Table 1 shows that the six muta-            mids mirrors the ability of the dnaAts mutants to grow in <br /> tions tested substitute amino acids spread throughout               the absence of priC. <br /> DnaA. The two mutants that show a requirement for priC <br /> have mutations in Domains I (DnaB recruitment) and III              The results presented in this paper do not allow one to <br /> (ATP hydrolysis). However, several of the mutations not             definitively know whether the synthetic lethality studied <br /> requiring priC also affect Domain III. An interesting aspect        here is due to a failure in initiation of replication at oriC <br /> to dnaA genetics is that many temperature sensitive                 or is it due to a failure in replication restart based on the <br /> mutants have two mutations (Table 1 and [2]). The                   study of the suppressors. Since dnaA mutations can affect <br /> dnaA5, dnaA46 and dnaA602 all have mutations in                     more than just initiation of DNA replication, it is <br />  <br />  <br />  <br />  <br /> tempting to speculate that some other dnaA function:                           <br /> transcription of a particular gene or set of genes or the            <br /> shape of nucleoid in the priC mutant may contribute or be                     <br /> the cause of the synthetic lethality. Understanding the                  <br /> molecular mechanism underlying the dnaAts-priC syn-               <br /> thetic lethality may require appreciation of these other             <br /> aspects of dnaA biology.                                   <br /> Conclusions                                                         <br /> A hitherto unknown function for either DnaA in replica-               <br /> tion restart or PriC in initiation of DNA replication that                  <br /> occurs in certain dnaA temperature sensitive strains at the                     <br /> permissive temperature of 30&#194;&#176;C has been documented.                    <br /> Models considering roles for PriC during initiation of                           <br /> DNA replication and roles for DnaA in replication restart                <br /> were tested and found not to decisively explain the data.                   <br /> Other roles of dnaA in transcription and nucleoid struc-                <br /> ture are additionally considered.                                 <br /> Methods                                                               <br /> Bacterial strains                                              <br /> All bacterial strains used in this work are derivatives of E.          <br /> coli K-12 and are described in Table 4. The protocol for P1      <br /> transduction has been described elsewhere [37]. All P1               <br /> transduction were selected on 2% agar plates containing               <br /> either minimal or rich media and either tetracycline 10                <br /> &#194;&#181;g/ml or kanamycin 50 &#194;&#181;g/ml final concentration. All              <br /> transductants were first purified on the same type of                 <br /> media on which they were selected. Tests for temperature           <br /> sensitivity were then done by replica plating patches of the           <br /> purified transductants at 30&#194;&#176;C and 42&#194;&#176;C on solid rich    <br /> media without any antibiotics. Growth was scored by                            <br /> either the presence or absence of a patch after 24 hours.                 <br />                                                         <br /> Cloning of the priC gene                                               <br /> Wildtype chromosome DNA was used as the template in              <br /> a standard PCR reaction using prSJS283 and prSJS284                     <br /> (Table 3) as the priming oligonulceotides. The amplified                   <br /> PCR fragment (that includes the putative promoter) was          <br /> purified by gel electrophoresis and cloned into the pCR                   <br /> 2.1 using the TOPO-TA cloning system from Invitrogen.                 <br /> The priC containing plasmid was called pTH1.                  <br /> Authors contributions                                             <br /> TH carried out the initial part of the molecular genetic                          <br /> studies. These were completed by SJS. SJS conceived of the                       <br /> study and wrote the manuscript.                                             <br /> </body></html>