<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Genomics                                                                                                                             BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access <br /> Functional characterization Caenorhabditis elegans of <br /> transmembrane worm-human orthologs <br /> Anna Henricson1, Erik LL Sonnhammer1, David L Baillie2 and <br /> Ana Vaz Gomes*1 <br />  <br /> Address: 1Center Genomics Bioinformatics, Karolinska Institutet, Stockholm, Sweden 2Department Molecular Biology and <br /> Biochemistry, Simon Fraser University, Burnaby, Canada <br /> Email: Anna Henricson - anna.henricson@cgb.ki.se; Erik LL Sonnhammer - erik.sonnhammer@cgb.ki.se; David L Baillie - baillie@sfu.ca; <br /> Ana Vaz Gomes* - ana.vaz.gomes@cgb.ki.se <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 08 November 2004                                                   Received: 06 April 2004 <br />                                                                               Accepted: 08 November 2004 <br /> BMC Genomics 2004, 5:85    doi:10.1186/1471-2164-5-85 <br /> This article available <span id='am-2' about='protege:FROM' typeof='owl:Thing'>from</span>: http://www.biomedcentral.com/1471-2164/5/85 <br /> &#194;&#169; 2004 Henricson et al; licensee BioMed Central Ltd. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), <br /> which permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  <span id='am-1' about='protege:abstract' typeof='owl:Thing'>Abstract</span> <br />                  Background: The complete genome sequences human nematode Caenorhabditis <br />                  elegans offer opportunity learn human gene function functional <br />                  characterization orthologs worm. Based previous genome-wide analysis worm- <br />                  human orthologous transmembrane proteins, selected seventeen genes explore <br />                  experimentally C. elegans. These genes selected basis high <br />                  confidence candidate human orthologs function unknown. We analyzed their <br />                  phylogeny, membrane topology domain organization. Then gene functions studied <br />                  experimentally worm using <span id='am-161' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-162' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-177' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-185' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-196' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span>RNA interference</span> transcriptional gfp reporter gene <br />                  fusions. <br />                  Results: The experiments gave functional insights genes studied. For example, <br />                  C36B1.12, worm ortholog presenilin-like genes, exclusively expressed in <br />                  head neurons, suggesting ancient conserved role important neuronal function. We propose <br />                  new transmembrane topology presenilin-like protein family. sft-4, worm ortholog of <br />                  surfeit locus gene Surf-4, proved essential gene required development larval <br />                  stages worm. R155.1, human ortholog entirely uncharacterized, implicated in <br />                  body size control developmental processes. <br />                  Conclusions: By combining bioinformatics C. elegans experiments orthologs, provide <br />                  functional insights previously uncharacterized human genes. <br />  <br />  <br />  <br />  <br /> Background                                                                    model organism. A majority human genes have <br /> The nematode Caenorhabditis elegans used                        homologs C. elegans. In comparative proteomics <br /> simple model understanding animal biology                             study, 83% worm proteome have <br /> nearly decades. After sequencing entire                           human homologous genes [1]. Only 11% contains <br /> genomes metazoans, excel-                       nematode specific genes. This makes worm suitable <br /> lent position gene-centric approach worm <br />  <br />  <br />                                                                                                                                        Page 1 18 <br />                                                                                                                  (page number citation purposes) <br />  BMC Genomics 2004, 5:85                                                      http://www.biomedcentral.com/1471-2164/5/85 <br />  <br />  <br />  <br /> model organism delineating human gene function [2-          Here present initial functional characterization in <br /> 4].                                                             C. elegans seventeen genes. The criteria selecting <br />                                                                 genes high confidence candidate <br /> In previous study, transmembrane protein families      orthologs human genes [5] function is <br /> the C. elegans genome classified human             unknown. They predicted encode transmem- <br /> orthologs identified [5]. Predicted proteins        brane proteins, imply constitute as <br /> more membrane domains clustered               unknown receptors, channels transporters playing <br /> cluster multiple alignment created. From align-       important roles various biological processes multi- <br /> ments, HMMs (Hidden Markov Models) built               cellular organisms. We particularly interested stud- <br /> subsequently used search mammalian homologs.             ying genes neuronal function. The <br /> The consensus different phylogenetic methods            phylogeny, membrane topology domain organiza- <br /> and BLAST used assign orthology. This resulted       tion analyzed. Gene function explored experi- <br /> a total 174 worm-human orthology assignments          mentally worm means RNA interference <br /> high confidence.                                                induced knock-down phenotypes gene expression <br />                                                                 patterns. <br /> Orthologs sequences arose common <br /> ancestor gene separated speciation event          Results <br /> [6]. Identification orthologs important,       Membrane topology predictions <br /> might share functionality. In closely related species,     The consensus different methods used pre- <br /> as human mouse, orthologs normally trivial           dict membrane topologies putative C. elegans pro- <br /> find. However, comparing distantly related species,        teins, different methods used predict <br /> e.g. human worm, longer case         signal peptides (see Methods details). Each predicting <br /> the similarity levels overall low. Instead, needs    method margin error; consensus <br /> rely sophisticated phylogenetic reconstruction tech-         different predictors likely a <br /> niques infer genes stem node         better estimate true topology. Results viewed <br /> corresponds speciation split duplication event     using SFINX tool [8,9], example output be <br /> within lineage. Close orthologs likely        seen Fig. 1. The number transmembrane (TM) <br /> same biological role organisms. Distant              regions ranges (except the <br /> orthologs hand, likely        splice variants R155.1), majority proteins hav- <br /> same phenotypical role, role       ing seven TM regions (see Table 1). Such proteins <br /> corresponding pathway. Consequently, studying true           likely receptors, channels, transporters. One <br /> C. elegans orthologs human genes experimentally       case, however, (C36B1.12) likely intramem- <br /> worm, potentially learn gene             brane protease. <br /> function humans. Depending duplica- <br /> tion(s) occurred lineages         Phylogenetic analysis <br /> speciation event, orthologs form one-<span id='am-4' about='protege:TO' typeof='owl:Thing'>to</span>-one, one-to-        The results phylogenetic analysis presented <br /> many many-to-many relationships.                             Table 1. The previous orthology assignments still <br />                                                                 valid [5], C. elegans genes, additional <br /> Paralogs arise duplication event. A common sce-          human orthologs emerged sequencing <br /> nario genes duplicated gene         efforts. At present, 29% (5 17) worm genes have <br /> copies negative selective pressure       one-to-many ortholog relationship human genes, <br /> retains function ancestor. The copy      means probably expansion <br /> then free evolve new function different       human lineage. This case C36B1.12 <br /> the ancestral function. This reason paralogs      (Q93346) ZK721.1 (Q9GYF0) (see Fig. 2A 2D, <br /> different species likely share functionality com-   respectively). 53% (9 17) showed one-to-one rela- <br /> pared orthologs. Paralogs divided sub-       tionship human ortholog, example sft-4 <br /> types &#226;&#8364;&#8220; outparalogs inparalogs [7]. Outparalogs         (Q18864) (see Fig. 2C). Two worm genes, F40F9.1 <br /> paralogs evolved gene duplications happened        (Q8MQ55, Q8MQ56) F40F9.2 (Q20241), to <br /> before speciation event form      many-to-many relationship (see Fig. 2B). The phy- <br /> orthologous relationships. Inparalogs, hand,       logenetic tree Fig. 2B somewhat inconclusive sup- <br /> form co-orthologous relationships, paralogs      port speciation event taken <br /> that evolved gene duplications happened       place. It possible human gene Q8IVW7 also <br /> speciation event.                                               ortholog worm genes. To investigate the <br />                                                                 ortholog relationship further, Orthostrapper used <br />                                                                 [10]. Orthostrapper analyzes set bootstrap trees <br />  <br />  <br />                                                                                                                    Page 2 18 <br />                                                                                              (page number citation purposes) <br />  BMC Genomics 2004, 5:85                                                        http://www.biomedcentral.com/1471-2164/5/85 <br />  <br />  <br />  <br />  <br />   A <br />       Phobius                                                                           *              * <br />     TMHMM2.0 <br />     TMHMM1.0 <br />       PHDhtm <br />    HMMTOP2.1 <br />    HMMTOP1.0 <br />      MEMSAT <br />       TOPPRED <br />   Kyte-Doolittle <br />  <br />                                100             200              300               400            500                 600 <br />   B <br />       Phobius                                                                           *              * <br />     TMHMM2.0 <br />     TMHMM1.0 <br />       PHDhtm <br />    HMMTOP2.1 <br />    HMMTOP1.0 <br />      MEMSAT <br />       TOPPRED <br />   Kyte-Doolittle <br />  <br />                                      100               200                  300                  400                       500 <br />  <br /> Output <br /> Figure from <br />          1     SFINX (A) C. elegans protein C36B1.12 (B) assigned human orthologs Q8TCT8 <br /> Output SFINX (A) C. elegans protein C36B1.12 (B) assigned human orthologs Q8TCT8. The overall <br /> membrane topology proteins similar. The consensus different topology predictors trans- <br /> membrane (TM) regions, N-terminal signal peptide, &gt;150 amino acids non-cytoplasmic N-terminal region. Conserved <br /> aspartic acid residues marked asterisk (residues 433 516 C36B1.12; residues 351 412 Q8TCT8). <br /> Phobius program used predicts TM regions N-terminal signal peptides. The programs are <br /> designed detect TM regions, therefore, commonly mistake signal peptide TM region. Numbers on <br /> the horizontal axis indicate amino acid positions. Color coding: black = TM region, white = non-cytoplasmic, gray = cytoplas- <br /> mic, striped = N-terminal signal peptide predicted Phobius. <br />  <br />  <br />  <br />  <br /> instead optimal tree orthologs. The algorithm         Fig. 2B using Orthostrapper, results showed stronger <br /> detects orthologous relations (groups of)            support human genes Q9BWQ8 Q969X1 be <br /> species. The frequency orthology assignments           orthologs compared Q8IVW7 (65% vs. 23%). It <br /> bootstrap trees interpreted confidence value         orthologous relationship particular <br /> for possible orthology proteins. Orthology            case complicated elucidate. Still, Q9BWQ8 and <br /> assignments optimal phylogenetic tree          Q969X1 best candidate human orthologs the <br /> be incorrect identified low ortholog boot-       C. elegans genes F40F9.1 F40F9.2. <br /> strap value. This makes possible resolve complicated <br /> many-to-many orthologous relationships. When analyz-             H13N06.5 T28F3.3 sequence similarity to <br /> ing multiple alignment phylogenetic tree          human gene Q92504. However, phylogenetic tree <br />  <br />  <br />                                                                                                                      Page 3 18 <br />                                                                                                (page number citation purposes) <br />  BMC Genomics 2004, 5:85                                                                                         http://www.biomedcentral.com/1471-2164/5/85 <br />  <br />  <br />  <br /> Table 1: Description C. elegans &#226;&#8364;&#8220; human orthologs. The Swiss-Prot accession numbers given worm sequences human orthologs. TM: <br /> the number transmembrane regions predicted using consensus different methods SFINX tool. Bootstrap support (%) given the <br /> inferred speciation node phylogenetic tree constructed using PHYLOWIN PAM distances. Identity (%): sequence percentage identity the <br /> Blastp output C. elegans gene nearest human ortholog. Four worm genes predicted splice variants (a b). <br /> However, small differences protein sequences variants, R155.1. Splice variant R155.1b predicted have <br /> a truncation amino acids N-terminus compared R155.1a. The PF03062 domain (MBOAT) present splice <br /> variants. The protein sequence R155.1a used phylogenetic analysis. <br />  <br />  C. elegans orthologs          TM         Pfam-A domains           Human orthologs             Bootstrap        Identity (%)      Nearest human ortholog putative <br />                                                                                               support (%)                                      function <br />  <br />  <br />  <br />  C30H6.2 (Q9XVR4)                   6    PF02535 (ZIP)           Q9H6T8, Q9NXC4,                    57               50         Q9H6T8: SLC39A4, involved intestinal <br />                                                                  Q96NN4                                                         absorption zinc. <br />  <br />  <br />  T11F9.2a (Q8I4G0),                 7    PF02535 (ZIP)           Q15043, Q96SM9,                    91               30         Q9C0K1: BIGM103, involved intracellular <br />  T11F9.2b (Q22395)                                               Q9C0K1, Q96BB3                                                 zinc retention accumulation. <br />  <br />  <br />  H13N06.5 (Q9XTQ7)                  9    PF02535 (ZIP)           Q92504                             98               52         Transport zinc ER2 other <br />                                                                                                                                 intracellular stores. <br />  <br />  <br />  T28F3.31 (Q9XUC4)              7*       PF02535 (ZIP)           Q92504                             79               38         Transport zinc ER2 other <br />                                                                                                                                 intracellular stores. <br />  <br />  <br />  T01D3.5 (Q9XVJ5)                   8    PF02535 (ZIP)           Q9NUM3                             99               38         Function unknown. <br />  <br />  <br />  F40F9.1a (Q8MQ56),                 7    PF01027 (UPF0005)       Q9BWQ8, Q969X1                     70               39         Q9BWQ8: Lifeguard protein, protects cells <br />  F40F9.1b (Q8MQ55)                                                                                                              Fas-mediated cell death. <br />  <br />  <br />  F40F9.2 (Q20241)                   7    PF01027 (UPF0005)       Q9BWQ8, Q969X1                     70               40         Q9BWQ8: Lifeguard protein, protects cells <br />                                                                                                                                 Fas-mediated cell death. <br />  <br />  <br />  F08F1.7 (O17388)               9*       PF02990 (EMP70)         Q99805                            100               62         Endosomal integral membrane protein. <br />  <br />  <br />  ZK858.6a (Q94422)              9        PF02990 (EMP70)         Q92544                            100               53         Function unknown. <br />  ZK858.6b (Q7YTF9)              9* <br />  <br />  <br />  F14F3.3 (Q19468)                   9    PF03062 (MBOAT)         Q96N66, Q99908                    100               27         Q99908: BB1 protein, malignant cell expression- <br />                                                                                                                                 enhanced gene. <br />  <br />  <br />  R155.1a (O01925)                   8    PF03062 (MBOAT)         Q92980                            100               32         Function unknown. <br />  R155.1b (Q86DC4)                   4 <br />  <br />  <br />  C36B1.12 (Q93346)              9*       PF04258                 Q8TCT7, Q8TCT8,                   100               30         Q8TCT7: Presenilin-like protein, act as <br />                                          (Peptidase_A22B)        Q8IUH8                                                         intra-membrane protease. <br />  <br />  <br />  sft-4 (Q18864)                     7    PF02077 (SURF4)         O15260                            100               55         Surfeit locus protein 4, probable ER2 integral <br />                                                                                                                                 membrane protein. <br />  <br />  <br />  D2013.10 (O62126)                  6    None                    Q15055                             98               53         Function unknown. <br />  <br />  <br />  T04A8.12 (Q22141)                  6    None                    Q9UHJ9                             50               35         FRAG1 (FGFR (fibroblast growth factor <br />                                                                                                                                 receptor) activating gene 1). <br />  <br />  <br />  Y6B3B.10 (Q9XWE9)                  6    PF03798 (LAG1)          P27544                             99               37         Function unknown. <br />  <br />  <br />  ZK721.1 (Q9GYF0)               10*      None                    Q9NXL6, Q9Y357                     97               34         Function unknown. <br />  <br />  1 T28F3.3 initially included   strong similarity Q92504; however, phylogenetic analysis showed probably outparalog human gene. <br />  2 ER = endoplasmic reticulum. <br />  * predicted N-terminal signal peptide. <br />  <br />  <br />  <br /> and results Orthostrapper (data shown), suggest                                     The bootstrap support speciation node between <br /> that H13N06.5 putative ortholog Q92504,                                            T04A8.12 human ortholog Q9UHJ9 barely made <br /> whereas T28F3.3 outparalog.                                                        cutoff 50% PAM distances used. With <br />                                                                                              observed divergence Poisson correction distance <br />  <br />  <br />  <br />                                                                                                                                                                 Page 4 18 <br />                                                                                                                                       (page number citation purposes) <br />  BMC Genomics 2004, 5:85                                                                     http://www.biomedcentral.com/1471-2164/5/85 <br />  <br />  <br />  <br />  <br />       A                                                                       B <br />  <br />                                                               P34248 (SC)                                             P48558 (SC) <br />  <br />                                                                                                                     100               Q94A20 (AT) <br />                                                     Q93Z32 (AT) <br />                                     100                                               77                                          Q94GN3 (OS) <br />  <br />                                                 Q947T8 (OS)                                 93                       Q9VFM3 (DM) <br />                                                                                                               Q9V6H7 (DM) <br />                                                          Q93346 (CE) * <br />                                                                                                             Q8AVK4 (XL) <br />                                                                                             100 <br />                                                      Q9JJF9 (MM)                                            Q63863 (Rattus sp) <br />              100                          100 <br />                                                                                                      100    Q9ESF4 (MM) <br />                                                     Q8TCT8 (HS)                                        97    Q8IVW7 (HS) <br />                         99 <br />                                             Q8BXJ5 (MM)                                                 100 O88407 (RN) <br />                                       100                                         61                   100  Q8K097 (MM) <br />                                                   Q8TCT7 (HS)                              96                 Q9BWQ8 (HS) <br />                                98                                                                                  Q8BJZ3 (MM) <br />                                                           Q8BHP0 (MM)                                   100 <br />                                             100 <br />                                                                                                                    Q969X1 (HS) <br />           0.250                                                                       70 <br />                                                         Q8IUH8 (HS)                                                 Q20241 (CE) * <br />                                                                                                 92 <br />                                                                                                                      100 Q8MQ55 (CE) ** <br />                                                                               0.219                                         Q8MQ56 (CE) ** <br />  <br />  <br />  <br />  <br />        C                                                                      D <br />                                                       O45731 (CE)                                                               Q9XXF4 (CE) <br />  <br />  <br />                                                           P53337 (SC) <br />                                                                                                                                     Q9GZC8 (CE) <br />                    97 <br />  <br />                                                                 O74559 (SP) <br />                                                                                                                      Q9GYF0 (CE) * <br />  <br />                                             Q18864 (CE) * <br />                                                                                            97                                XP_148505 (MM) <br />                     100                                                                                               100 <br />                                                  O18405 (DM) <br />                                                                                                                               Q9NXL6 (HS) <br />                                                   O57590 (FR)                                                100 <br />                           69 <br />                                      100                                                                                     Q8CIF6 (MM) <br />                                                  Q64310 (MM)                                                          100 <br />  <br />                                           100                                  0.344                                          Q9Y357 (HS) <br />           0.195                                   O15260 (HS) <br />  <br />  <br />  <br /> Phylogenetic <br /> (C) <br /> Figure <br />      sft-42(Q18864), <br />               trees genes <br />                       (D) (A) <br />                               ZK721.1 <br />                                   C36B1.12 <br />                                         (Q9GYF0) <br />                                              (Q93346), (B) F40F9.1a (Q8MQ56), F40F9.1b (Q8MQ55) F40F9.2 (Q20241), <br /> Phylogenetic trees genes (A) C36B1.12 (Q93346), (B) F40F9.1a (Q8MQ56), F40F9.1b (Q8MQ55) F40F9.2 (Q20241), <br /> (C) sft-4 (Q18864), (D) ZK721.1 (Q9GYF0). The trees constructed using PHYLOWIN neighbor-joining <br /> method PAM distances. 500 bootstrap replicates run. All gene identifiers Swiss-Prot accession numbers, except <br /> in (D) XP_148505 NCBI accession number. The C. elegans genes studied marked asterisks. F40F9.1 <br /> is predicted splice variants; however, putative proteins <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> differ C- <br /> terminal amino acids. Species abbreviations: Arabidopsis thaliana (AT), Caenorhabditis elegans (CE), Drosophila melanogaster (DM), <br /> Fugu rubripes (FR), Homo sapiens (HS), Mus musculus (MM), Oryza sativa (OS), Rattus norvegicus (RN), Saccharomyces cerevisiae <br /> (SC), Schizosaccharomyces pombe (SP), Xenopus laevis (XL). <br />  <br />  <br />  <br />                                                                                                                                              Page 5 18 <br />                                                                                                                      (page number citation purposes) <br />  BMC Genomics 2004, 5:85                                                   http://www.biomedcentral.com/1471-2164/5/85 <br />  <br />  <br />  <br /> methods, bootstrap support improved 99%           C-terminal half protein N-terminus there <br /> 67%, respectively. When analyzing phylogenetic tree      substantial variation. This suggests C-terminal <br /> using Orthostrappper, strong support    constitute proteolytic subdomain, whereas <br /> this orthology assignment (94%). Therefore, conclude      N-terminus defines specific function respec- <br /> that T04A8.12 probably ortholog Q9UHJ9.            tive proteins. <br />  <br /> C30H6.2 potential human orthologs, Q9H6T8,         F40F9.1 F40F9.2 PF01027 domain <br /> Q9NXC4 Q96NN4. The bootstrap support             (UPF0005), uncharacterized protein family. <br /> speciation node PAM distances 57%. This             Both F08F1.7 ZK858.6 belong PF02990 <br /> improved 87% 88% observed divergence         domain family (EMP70). Proteins family be <br /> Poisson correction, respectively. Orthostrapper results      located endosomal membranes [16]. F14F3.3 and <br /> showed strong support Q9H6T8 Q9NXC4             R155.1 predicted PF03062 domain, which <br /> orthologs C30H6.2 (84%), support          annotated MBOAT (Membrane bound O-acyl trans- <br /> Q96NN4 weaker (57%). Considering results,          ferases) domain. Biochemically characterized proteins of <br /> we believe human genes orthologs   group encode enzymes transfer organic acids <br /> worm gene; however, ortholog relationship       hydroxyl groups membrane-embedded targets <br /> be weaker Q96NN4 C30H6.2.                        [17]. SFT-4 likely PF02077 (SURF4) domain. <br />                                                              Members family believed encode integral <br /> Putative domain assignments                                  membrane proteins located endoplasmic reticulum <br /> The domain organization predicted proteins        [18]. A PF03798 domain (LAG1) Y6B3B.10. <br /> analyzed using Pfam database [11,12] (see Table 1).      This domain associated longevity yeast (Jiang et <br /> Conclusions possible functions drawn         al. 1998). Three seventeen putative proteins <br /> from mere presence putative domain,     (D2013.10, T04A8.12 ZK721.1) match any <br /> can indication.                                    Pfam-A domain. <br />  <br /> Five proteins (C30H6.2, T11F9.2, H13N06.5,            RNA interference studies <br /> T28F3.3 T01D3.5) PF02535 domain,              Out seventeen genes studied, sft-4 R155.1 <br /> which annotated ZIP domain. The ZIP family     exhibited phenotypes N2 (wildtype) and <br /> is believed include zinc metal transporters.    RNAi sensitive rrf-3(pk1426) II [19,20] strains were <br /> The ZIP proteins classified groups       subjected RNAi feeding (see Table 2). The pheno- <br /> based sequence conservation [13]; ZIP subfamily I     types enhanced strain rrf-3, Dpy <br /> and II, gufA subfamily LIV-1 subfamily (also     (dumpy) <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> seen R155.1 low pene- <br /> called LZT subfamily). The ZIP I subfamily contains      trant relatively weak. The Lva (larval arrest) observed <br /> mostly plant yeast sequences; however,           sft-4 occurred larval stages L2&#226;&#8364;&#8220;L3 an <br /> includes T01D3.5 putative orthologs Drosophila    complete penetrance sensitive strain. The <br /> melanogaster, mouse human (Q9V4C6, Q8BFU1            RNAi phenotypes genes detected tem- <br /> Q9NUM3, respectively). The worm genes             peratures, although, sft-4 severe at <br /> appear belong LIV-1 subfamily. This subfamily      higher temperatures. The positive results verified <br /> has unique metalloprotease motif raises possi-    RNAi injection strain N2. <br /> bility protease activity [14]. Within <br /> the LIV-1 subfamily subgroup called KE4       Analysis gene expression <br /> group, H13N06.5 human ortholog hKE4         Transcriptional fusions gfp established four- <br /> (Q92504) belong.                                             teen genes resulting gene expression ana- <br />                                                              lyzed. The results presented Table 2. Because the <br /> C36B1.12 predicted PF04258 domain,           arrays extrachromosomal integrated; mosaic <br /> probable signal peptide peptidase (SPP) domain. SPP cat-     patterns expression observed. Also, germ line <br /> alyzes intramembrane proteolysis signal peptides     expression analyzed, germ line silenc- <br /> after cleaved preprotein. This         ing. For 18% (3 17) genes transgenic lines <br /> processing SPP related protein cleavage prese-   established despite attempts, of <br /> nilins. Homologs SPP divided subfamilies    lines established, 14% (2 14) showed expres- <br /> based phylogenetic analysis (subfamily SPP            sion. This reasons (see <br /> subfamilies SPPL1-4, SPP like) [15]. C36B1.12    Discussion). In half transgenic lines established, <br /> putative human orthologs Q8TCT7, Q8TCT8                  expression different tissues. <br /> Q8IUH8 belong SPPL2 subfamily. The members         The prevalent major tissues expression hypo- <br /> subfamilies SPPL1-4 homology SPP the <br />  <br />  <br />                                                                                                                 Page 6 18 <br />                                                                                           (page number citation purposes) <br />  BMC Genomics 2004, 5:85                                                               http://www.biomedcentral.com/1471-2164/5/85 <br />  <br />  <br />  <br /> Table 2: RNAi phenotype major tissues gene expression C. elegans orthologs. Abbreviations: RNAi phenotype: clear (Clr), <br /> dumpy (Dpy), larval arrest (Lva), ruptured (Rup), sterile (Ste), wildtype (WT). Gene expression: body wall muscle (bwm), commissures <br /> (c), excretory (exc), gonad (g), hypodermis (h), hypodermal seam cells (hs), intestine (i), neuronal (n), pharyngeal muscle (phm), <br /> rectal epithelial cells (re), spermatheca (s), vulva (v), ventral nerve cord (vnc). A limitation extrachromosomal array transgenes <br /> are used expression germ line possible evaluate. No transgenic lines generated T11F9.2, H13N06.5 <br /> and T04A8.12. Possible reasons injected DNA concentration low sequence toxic. In <br /> either case, extrachromosomal array formed sufficiently large inheritable [46]. The transgenic lines for <br /> ZK858.6 F14F3.3 showed expression gfp. This caused conditional gene expression, germline silencing absence <br /> of promoter::gfp fusion inheritable extrachromosomal array [46]. <br />  <br />  C. elegans orthologs                                       RNAi phenotype                       Major tissues gene expression <br />  <br />  C30H6.2                                      WT                                            h, phm <br />  T11F9.2a, T11F9.2b                           WT                                            No transgenic line <br />  H13N06.5                                     WT                                            No transgenic line <br />  T28F3.3                                      WT                                            h, i, n, v, vnc <br />  T01D3.5                                      WT                                            hs <br />  F40F9.1a, F40F9.1b                           WT                                            bwm, c, h, n, phm, vnc <br />  F40F9.2                                      WT                                            exc, n, phm <br />  F08F1.7                                      WT                                            h, n, phm, re, s, v <br />  ZK858.6a, ZK858.6b                           WT                                            No expression <br />  F14F3.3                                      WT                                            No expression <br />  R155.1a, R155.1b                             Dpy                                           bwm, h, i, phm <br />  C36B1.12                                     WT                                            i, n <br />  sft-4                                        Clr, Lva, Rup, Ste                            bwm, h, i, n, phm, v <br />  D2013.10                                     WT                                            bwm, h, i, n, s, v <br />  T04A8.12                                     WT                                            No transgenic line <br />  Y6B3B.10                                     WT                                            phm <br />  ZK721.1                                      WT                                            bwm, g, h, i, n, phm, s, v <br />  <br />  <br />  <br />  <br /> dermis (9 14 transgenic lines), nervous pha-             Putative function assignments <br /> ryngeal muscle (8 14) intestine (6 14).                      C36B1.12 <br />                                                                        The putative human orthologs (Q8TCT7, Q8TCT8 <br /> Examples gene expression patterns observed pre-                 Q8IUH8) C. elegans gene C36B1.12 are <br /> sented Fig. 3, 4, 5, 6. C36B1.12 shows expression                   thought presenilin-like (PSL) proteins (also called <br /> restricted head neurons intestine (see Fig. 3). The             PSH proteins presenilin homologs). Presenilins an <br /> intestinal expression stronger larval stages                important group proteases acting nervous sys- <br /> compared adult stage, predominantly                  tem. Abnormal proteolytic cleavage result accumu- <br /> located posterior intestinal nuclei. F40F9.1                    lation pathogenic insoluble proteins, implied e.g. <br /> F40F9.2 demonstrate overlapping expression                     Alzheimer&apos;s disease. We shown C36B1.12 is <br /> nervous pharyngeal muscle (see Fig. 4A,4G);                 probably expressed head neurons intestine; how- <br /> however, F40F9.1 appear widely expressed                 ever, intestinal expression ectopic (see Dis- <br /> the nervous expression neuronal cell               cussion details), imply gene is <br /> bodies commissures ventral nerve cord (see                  exclusively expressed neurons. This suggests the <br /> Fig. 4C). Expression F40F9.1 located body                human orthologs encode neuronal <br /> wall muscle hypodermal cells tail (see Fig. 4E).            functions. <br /> F40F9.2 expressed excretory system, <br /> although weaker compared tissues (see               The membrane topology human proteins belonging to <br /> Fig. 4G). Widespread expression patterns observed                 presenilin-like family analyzed previously. <br /> for sft-4 ZK721.1 larval adult stages              Q8IUH8 predicted seven transmembrane <br /> (see Fig. 5 6, respectively). For sft-4 highly              (TM) segments cytoplasmic C-terminus [21], and <br /> mosaic pharyngeal muscle consistent tis-              predicted HM13_HUMAN (Swissprot: <br /> sue expression.                                                     Q8TCT9) [15]. However, noted although <br />                                                                        number TM segments predictions the <br />                                                                        same, topologies fact different. The fourth seg- <br />                                                                        ment Q8IUH8 prediction missing the <br />  <br />  <br />                                                                                                                                Page 7 18 <br />                                                                                                          (page number citation purposes) <br />  BMC Genomics 2004, 5:85                                                        http://www.biomedcentral.com/1471-2164/5/85 <br />  <br />  <br />  <br />  <br /> Figuretissues <br /> Major    3     expression C36B1.12 <br /> Major tissues expression C36B1.12. (A) Fluorescence micrograph L4 larvae hermaphrodite carrying transcrip- <br /> tional fusion gfp putative promoter C36B1.12 expressed neurons head. (C) Fluorescence micro- <br /> graph young adult hermaphrodite carrying construct expressed intestine. The observed intestinal expression is <br /> mostly located posterior intestinal nuclei prominent younger worms. (B D) Corresponding DIC images. <br /> Scale bars, 20 &#194;&#181;m. <br />  <br />  <br />  <br />  <br /> HM13_HUMAN prediction, C-terminal segment                Merging proposed topologies accepting all <br /> in HM13_HUMAN prediction missing                 TM segments predicted study would <br /> Q8IUH8 prediction. This means C-terminal           yield topology TM segments. Our analy- <br /> TM segments register predictions,         sis proteins question using SFINX tool [8,9] <br /> and consequently loops opposite sides. This           provides strong support topology, C-ter- <br /> includes loop putatively catalytic aspartic      minus cytoplasm (data shown). We therefore <br /> acid residues present presenilins predicted     propose previous TM topologies incorrectly <br /> cytoplasmic Ponting et al., non-cytoplasmic            left TM segment, correspond seg- <br /> Weihofen et al. Because TM segment disagreement,          ments 4 9 9-TM segment model. <br /> these aspartic acid residues predicted located in <br /> TM5 TM6 Ponting et al. prediction, TM4         We analyzed members family <br /> and TM5 Weihofen et al. prediction.                       SFINX [8,9], consistently 9-TM topology <br />  <br />  <br />                                                                                                                      Page 8 18 <br />                                                                                                (page number citation purposes) <br />  BMC Genomics 2004, 5:85                                                       http://www.biomedcentral.com/1471-2164/5/85 <br />  <br />  <br />  <br />  <br /> Figuretissues <br /> Major    4     expression F40F9.1 F40F9.2 <br /> Major tissues expression F40F9.1 F40F9.2. Fluorescence micrographs adult hermaphrodite carrying transcrip- <br /> tional fusion gfp putative promoter F40F9.1 expressed (A) neurons pharyngeal muscle, (C) commis- <br /> sures (c) ventral nerve cord (vnc), (E) body wall muscle (bwm) hypodermis (h). (G) Fluorescence micrograph <br /> of L4 hermaphrodite carrying transcriptional fusion gfp putative promoter F40F9.2 expressed the <br /> excretory (exc), neurons, pharyngeal muscle. (B, D, F, H) Corresponding DIC images. Scale bars, 20 &#194;&#181;m. <br />  <br />  <br />  <br />                                                                                                                     Page 9 18 <br />                                                                                               (page number citation purposes) <br />  BMC Genomics 2004, 5:85                                                         http://www.biomedcentral.com/1471-2164/5/85 <br />  <br />  <br />  <br />  <br /> Figuretissues <br /> Major    5    expression sft-4 <br /> Major tissues expression sft-4. Fluorescence micrographs adult hermaphrodite carrying transcriptional fusion <br /> between gfp putative promoter sft-4 expressed (A) pharyngeal muscle, (C) vulva (E) intestinal nuclei (i). The <br /> intestine shows unspecific autofluorescence, specific expression intestinal nuclei. Expression in <br /> body wall muscle, hypodermis neurons shown. (B, D, F) Corresponding DIC images. Scale bars, 20 &#194;&#181;m. <br />  <br />  <br />  <br />                                                                                                                       Page 10 18 <br />                                                                                                  (page number citation purposes) <br />  BMC Genomics 2004, 5:85                                                       http://www.biomedcentral.com/1471-2164/5/85 <br />  <br />  <br />  <br />  <br /> Figuretissues <br /> Major   6     expression ZK721.1 <br /> Major tissues expression ZK721.1. Fluorescence micrographs adult hermaphrodite carrying transcriptional fusion <br /> between gfp putative promoter ZK721.1 expressed (A) body wall muscle, (C) hypodermis (E) gonad. Gene <br /> expression hypodermis weaker compared expression tissues. Expression intestine, neurons, pharyngeal <br /> muscle, spermatheca, vulva shown. (B, D F) Corresponding DIC images. Scale bars, 20 &#194;&#181;m. <br />  <br />  <br />  <br />                                                                                                                    Page 11 18 <br />                                                                                               (page number citation purposes) <br />  BMC Genomics 2004, 5:85                                                      http://www.biomedcentral.com/1471-2164/5/85 <br />  <br />  <br />  <br /> model C-terminus cytoplasm. The conserved           protein required systemic RNAi [25,26]. It was <br /> aspartic acid residues located TM6 TM7.         predicted transmembrane (TM) regions <br /> As example, SFINX output C36B1.12         experimentally verified [27]. <br /> its human orthologs Q8TCT8 shown Fig. 1. The over-        The high number TM regions suggests SID-1 forms <br /> all topology similar putative worm          channel. Double stranded RNA thought diffuse <br /> protein orthologs human mouse,       channel, leading spreading RNA and <br /> as presenilin-like proteins. One              hence, systemic RNAi effect. This idea supported <br /> difference, however, C36B1.12            fact homolog sid-1 in <br /> human (Q8TCT8 Q8IUH8) mouse orthologs                   Drosophila, explain observed absence sys- <br /> (Q9JJF9 Q8BHP0) predicted N-terminal          temic RNAi organism [28,29]. Our analysis of <br /> signal peptide, feature missing     ZK721.1 predicts TM regions, makes <br /> other presenilin-like proteins.                                 likely candidate forming channel. The phyloge- <br />                                                                 netic tree indicates ZK721.1 best candidate <br /> F40F9.1 F40F9.2                                             ortholog human genes Q9NXL6 Q9Y357, whereas <br /> F40F9.1 F40F9.2 48% identical          sid-1 probable outparalog human genes (see Fig. <br /> the protein sequence level appear         2D). The tree supports previous finding there <br /> similar membrane topologies. They close              homolog ZK721.1 sid-1 Drosophila <br /> genome (&lt;100 bp apart), opposite strands. It         genome. We observed wildtype RNAi phenotype <br /> been shown genes closer 500 bp opposite            ZK721.1, consistent results other <br /> strands likely shared control region [22],        studies [30,31]. Further analysis reveal ZK721.1 <br /> which means genes coexpressed.          involved systemic RNAi other <br /> The expression patterns observed overlapping,        function. Four additional genes required systemic <br /> although extent (see Table 2). One         RNAi reported (rsd-2, -3, -4 -6) [26], but <br /> human orthologs (Q9BWQ8) identified shown              map locus ZK721.1. <br /> to protect cells Fas-mediated cell death [23], suggest- <br /> ing F40F9.1 F40F9.2 involved               C30H6.2, H13N06.5, T01D3.5, T11F9.2 T28F3.3 <br /> apoptosis.                                                      SLC39A4 (Q9H6T8) identified possi- <br />                                                                 ble human orthologs C30H6.2. The human gene has <br /> sft-4                                                           implicated rare inherited condition acroder- <br /> The sft-4 gene (C54H2.5) highly conserved         matitis enteropathica, results defect the <br /> evolution orthologs vertebrates non-ver-       absorption zinc [32]. It believed SLC39A4 might <br /> tebrates. All orthologous relationships one-to-one          encode zinc transporter responsible intestinal <br /> with high bootstrap support. The tree Fig. 2C indi-        absorption zinc. Therefore, C30H6.2 a <br /> cates exists worm homolog (O45731) sft-4.       zinc/metal transporter. <br /> O45731 (T02E1.7) 33% identical sft-4 <br /> on protein sequence level predicted      The predicted human ortholog (Q92504) H13N06.5 in <br /> have PF02077 (SURF4) domain. However, RNAi                C. elegans shown zinc transporter local- <br /> screen indicates little functional redun-   ized intracellular membranes [33]. Q92504 probably <br /> dancy genes, sft-4 strong      transports zinc endoplasmic reticulum and <br /> RNAi phenotype, showing complete larval               intracellular stores. The Drosophila ortholog to <br /> arrest stages L2&#226;&#8364;&#8220;L3. The RNAi phenotype T02E1.7       H13N06.5 Catsup (Catecholamines up, Q9V3A4), which <br /> wildtype according previous studies [24]. Our gene           encodes negative regulator tyrosine hydroxylase (TH) <br /> expression analysis revealed wide spread expression        activity [34]. TH rate-limiting enzyme production <br /> the reporter construct sft-4 (see Table 2). Taken           dopamine brain. The Arabidopsis thaliana gene <br /> together, data suggests sft-4 play essen-     IAR1 (Q9M647) ortholog H13N06.5. It is <br /> tial role development acting tissues.            proposed involved auxin metabolism response <br />                                                                 [35]. Interestingly, mouse ortholog (Q31125) to <br /> ZK721.1                                                         H13N06.5 IAR1 shown functionally substitute <br /> ZK721.1 probably orthologous human               Arabidopsis gene. These data indicate is <br /> genes Q9NXL6 Q9Y357 (CGI-40 protein). The CGI-              functional conservation orthologs is <br /> 40 protein screen novel human              likely H13N06.5, possibly T28F3.3, could <br /> genes evolutionary conserved C. elegans identified      play similar roles corresponding pathways the <br /> [1]. The function CGI-40 Q9NXL6                  worm. <br /> unknown. ZK721.1 worm homologs, of <br /> which sid-1 (Q9GZC8). SID-1 identified a <br />  <br />  <br />                                                                                                                   Page 12 18 <br />                                                                                              (page number citation purposes) <br />  BMC Genomics 2004, 5:85                                                    http://www.biomedcentral.com/1471-2164/5/85 <br />  <br />  <br />  <br /> BIGM103 (Q9C0K1) identified candi-         elegans, suggesting important biological role. Previous <br /> date human orthologs T11F9.2. The human gene           studies shown human ortholog p76 <br /> found induced infection                  (Q99805) ubiquitously expressed [37]. There no <br /> inflammatory response. It shown play role    expression observed ZK858.6 gene T04A8.12, <br /> intracellular zinc ion accumulation retention [36].       failed generate transgenic line. All genes <br /> Consequently, possible T11F9.2 inte-   exhibited wildtype RNAi phenotype. For F08F1.7 and <br /> gral membrane zinc/metal transporter.                         ZK858.6, wildtype phenotype possibly be <br />                                                               explained functional redundancy two <br /> The human ortholog T01D3.5 known putative           genes C. elegans EMP70 domain containing <br /> function. We observed wildtype RNAi phenotype           protein Y41D4A.4 (Q95Y24). We Y41D4A.4 is <br /> T01D3.5 PF02535 (ZIP)           homologous F08F1.7 ZK858.6, likely an <br /> domain containing putative proteins (C30H6.2,                 ortholog human gene Q9HD45. Taken together, <br /> H13N06.5, T11F9.2 T28F3.3). This indicates           findings point F08F1.7, ZK858.6 and <br /> there functional redundancy             T04A8.12 play fundamental biological roles, and <br /> these genes. Considering information available,     involved processes cellular <br /> conceivable T01D3.5 zinc/metal             organization (F08F1.7) translation (ZK858.6 and <br /> transporter.                                                  T04A8.12). <br />  <br /> F08F1.7, T04A8.12 ZK858.6                                 F14F3.3 R155.1 <br /> F08F1.7 ZK858.6 47% identity protein          The human genes Q99908 (BB1) Q96N66 proba- <br /> sequence level similar membrane topolo-         ble orthologs F14F3.3. BB1 shown over- <br /> gies large N-terminal non-cytoplasmic region       expressed breast bladder carcinoma [40], <br /> nine transmembrane regions C-terminal part.            suggesting role tumor progression. <br /> F08F1.7 splice variants ZK858.6        The function Q96N66 unknown. <br /> predicted N-terminal signal peptide. The phylo- <br /> genetic analysis indicated human gene p76            The likely Drosophila ortholog R155.1 Nessy <br /> (Q99805) ortholog F08F1.7 appears         (Q9XYV9), putative Hox gene target [41], indicating a <br /> localize endosomes [37]. The function probable      possible developmental role. The Dpy (dumpy) RNAi <br /> human ortholog (Q92544) ZK858.6 unknown.                phenotype detected R155.1 some <br />                                                               developmental/body size regulatory error possibly the <br /> Both F08F1.7 ZK858.6 predicted operons,      hypodermis and/or body wall muscle; tissues the <br /> as T04A8.12 [38]. F08F1.7 probably operon         gene expressed according analysis. The best <br /> with tth-1 (F08F1.8, O17389). TTH-1 likely belong    human ortholog candidate (Q92980) R155.1 not <br /> the PF01290 domain family (thymosin beta-4),            functionally characterized. <br /> includes actin-binding proteins, implicating possible <br /> role cytoskeleton organization. ZK858.6 predicted    Y6B3B.10 <br /> be operon ZK858.5 (Q94421) ZK858.7             Y6B3B.10 probably orthologous human <br /> (Q94416). A PF05154 (TM2) domain unknown                 gene P27544 belong the <br /> function likely present ZK858.5. ZK858.7    PF03798 (LAG1) domain family. LAG1 longevity gene <br /> have PF04189 domain (eIF3gamma; eukaryotic initia-          cloned yeast [42]. Members LAG1 <br /> tion factor 3, gamma subunit), suggesting    family thought involved determining lifespan. <br /> involved translation. T04A8.12 operon         However, phylogenetic tree revealed Y6B3B.10 <br /> with T04A8.11 (Q22140) T04A8.13 (Q22142).                 human mouse ortholog (P27545) form tight <br /> T04A8.11 ribosomal protein, appears       cluster tree, separated LAG1 domain <br /> to PF00252 (ribosomal L16) domain. T04A8.13            containing proteins, indicating have <br /> was predicted PF00646 (F-box) domain,         evolved slightly different function. Y6B3B.10 showed a <br /> is known forming structural complexes pro-     wildtype RNAi phenotype appears an <br /> teins. There matching Pfam-A domains               expression restricted pharyngeal muscle. <br /> T04A8.12. The best candidate human ortholog to <br /> T04A8.12 FRAG1 (fibroblast growth factor receptor acti-    D2013.10 <br /> vating gene 1, Q9UHJ9). FRAG1 ubiquitously        D2013.10 orthologous Q15055 (human), Q8K1A5 <br /> expressed adult human tissues         (mouse) Q9VX39 (Drosophila). Neither genes <br /> detected human tumor cell lines [39]. Our          putative function assigned have <br /> results gene expression analysis demonstrated        matching Pfam-A domains. D2013.10 expressed in <br /> that F08F1.7 probably expressed tissues C. <br />  <br />  <br />                                                                                                                Page 13 18 <br />                                                                                           (page number citation purposes) <br />  BMC Genomics 2004, 5:85                                                      http://www.biomedcentral.com/1471-2164/5/85 <br />  <br />  <br />  <br /> several tissues C. elegans exhibits wildtype RNAi   weak missed. Furthermore, differ- <br /> phenotype.                                                      ences results RNAi screens shown to <br />                                                                 exist. A 10&#226;&#8364;&#8220;30% difference experiments in <br /> Discussion                                                      different laboratories been <br /> This study illustrates bioinformatic experimental       reported [44]. <br /> analysis combined elucidate putative gene func- <br /> tion. We predicted worm-human orthologs per-           When analyzing expression patterns using transcriptional <br /> formed initial functional characterization worm       reporter fusions, issue concern pat- <br /> genes. Since orthologs likely              tern observed expression <span id='am-239' about='xsp:pattern' typeof='owl:Thing'>pattern</span> native gene <br /> biological function, better understanding func-        not. Ectopic lack expression occur the <br /> tion human genes accomplished             putative promoter used does include regulatory <br /> analysis C. elegans. The genes explored study        elements. Expression different cell types the <br /> were selected previous study [5]        pharynx posterior intestinal cells young ani- <br /> predicted encode transmembrane proteins. Such pro-           mals attributed use incomplete promot- <br /> teins attractive study interesting recep-     ers    [46].     Another     limitation          using <br /> tors, channels, transporters signaling proteins         extrachromosomal arrays analysis expression in <br /> found them, making likely involved          germ line possible, germ line silencing. <br /> important regulatory processes multicellular <br /> organisms.                                                      The putative promoter used transcriptional fusion <br />                                                                 C36B1.12 1 kb, presence an <br /> The number transmembrane (TM) regions predicted              upstream gene. Therefore, possible intestinal <br /> for protein, similar number predicted        expression seen predominantly young worms and <br /> each cluster putative TM proteins study      located posterior intestinal nuclei, artifact <br /> (&#194;&#177; 1 TM region) [5]. For putative proteins H13N06.5,        use incomplete promoter region [46]. If this <br /> F14F3.3 ZK721.1, however, difference larger          case, C36B1.12 expressed exclusively in <br /> (+2-3 TM regions). This divergence Remm         neurons head (see Fig. 3). This finding provides <br /> and Sonnhammer having performed predictions                support idea C36B1.12 human <br /> cluster individual genes. In addition, used     orthologs encode neuronal functions. A possible conse- <br /> only program TMHMM [43] analyzing membrane              quence act fashion analo- <br /> topology. TMHMM, using default settings,               gous presenilin, involved in <br /> miss weak TM regions, leading possible underestima-        &#206;&#178;-amyloid precursor protein processing. It inter- <br /> tion number TM segments. A better estimate         esting study role nervous development <br /> the true topology achieved use sev-       function, linked neurological <br /> eral different prediction programs. In study, used      disorders <br /> the consensus different methods provided the <br /> SFINX tool [8,9] assign membrane topology.                   The transcriptional fusion T28F3.3 showed sim- <br />                                                                 ilar intestinal expression, apart specific expres- <br /> We observed RNAi phenotype 11.8% (2 17)            sion neurons head, ventral nerve cord, vulva and <br /> the genes using strains N2 (wildtype) rrf-        weak expression hypodermis. The putative promoter <br /> 3 (RNAi sensitive), respectively. This agreement     region used 0.8 kb, presence an <br /> previous experiments, 10.3% (N2) 12.8% (rrf-          upstream gene. Thus, intestinal expression seen for <br /> 3) phenotypes detected [30,44]. The RNAi phe-         T28F3.3 related use incom- <br /> notypes sft-4 consistent previous results          plete promoter region [46]. <br /> [30,31]. However, groups reported non- <br /> overlapping phenotypes, screen              Two transcriptional fusions (for genes F14F3.3 <br /> observed them. The Lva (larval arrest) phenotype         ZK858.6) showed expression reporter gene. <br /> has reported genome wide screen              This unlikely use incomplete promoter <br /> using strain rrf-3 [44]. The Dpy (dumpy) phenotype          region, upstream region included 2.9 kb and <br /> R155.1 reported before. The gene down-         3.0 kb, respectively. Instead, caused condi- <br /> regulated using RNAi injection screen chromo-        tional gene expression, germline silencing absence of <br /> some III [45] phenotype                 promoter::gfp fusion inheritable extrachro- <br /> wildtype. However, focus analysis iden-      mosomal array [46]. For genes study we <br /> tify genes involved cell division       failed establish transgenic lines. Possible reasons for <br /> few post-embryonic phenotypes scored. The Dpy              injected DNA concentration too <br /> phenotype observed low penetrant relatively         low sequence toxic. In case, the <br />  <br />  <br />                                                                                                                    Page 14 18 <br />                                                                                               (page number citation purposes) <br />  BMC Genomics 2004, 5:85                                                      http://www.biomedcentral.com/1471-2164/5/85 <br />  <br />  <br />  <br /> extrachromosomal array formed suffi-          Databases <br /> ciently large inheritable [46].                           The Pfamseq database version 10.0 [56] used for <br />                                                                 searching homologous sequences. It based the <br /> Out seventeen genes study, pre-        Swiss-Prot 41.10 SP-TrEMBL 23.15 databases. The <br /> dicted operons (18%). This equivalent        Pfam database [11] version 11.0 [12] used domain <br /> number genes C. elegans genome               assignments. <br /> believed operons (15%). Whether C. elegans oper- <br /> ons contain genes related function           Phylogenetic analysis <br /> unknown. There are, however, indications              The Pfamseq database [56] searched homologs <br /> genes encoding proteins fundamental biological               using Blastp 2.2.5 program [57] default settings <br /> importance clustered operons. For example,        putative worm proteins query. Multiple <br /> genes mitochondrial proteins strong tendency         alignments full-length sequences created using <br /> to operons genes encoding splic-        POA [58] default settings. Gappy sequences col- <br /> ing proteins [38].                                              umns (&gt;50% gaps) redundant sequences (&gt;99% <br />                                                                 identical) removed. The program PHYLOWIN [59] <br /> Conclusions                                                     tree building method neighbor-joining [60] and <br /> This study shed light putative function       PAM distance used constructing phylogenetic <br /> of predicted worm-human orthologs. Our aim            trees. Trees built observed divergence and <br /> to identify genes play role nervous sys-    Poisson correction distance methods, however, the <br /> tem able genes       results analysis discussed genes <br /> appear expressed neurons. C. elegans excel-      major differences bootstrap support. <br /> lent model organism pursuing functional charac-         If available, yeast sequence used outgroup. A <br /> terization genes, considering mapped      total 500 bootstrap tests run trees assess the <br /> relatively sophisticated nervous system. Investigating      significance branching order. Only bootstrap values <br /> function orthologous proteins using simple multicel-       &#226;&#8240;&#165; 50% considered positive. <br /> lular organism suitable approach possibility of <br /> learning function gene          Domain assignments <br /> one species hopefully several. This approach        Pfam-A domains assigned using Pfam database <br /> becomes valid genomes            [11,12]. Pfam-B domains considered, they <br /> sequenced moment additional ones         automatically generated non-curated there- <br /> the pipeline. With enormous data             fore lower quality. <br /> these sequencing efforts generating, useful to <br /> be able start delineating gene function based         Nematode strains culture conditions <br /> functional characterization ortholog spe-     Maintenance handling C. elegans strains as <br /> cies, initiating studies complex organisms.      previously described [61]. Strains used CGC N2 <br />                                                                 (wildtype) CGC NL2099 (rrf-3(pk1426) II) [19] <br /> Methods                                                         (Caenorhabditis Genetics Center [62]). The rrf-3 mutant <br /> Membrane topology predictions                                   strain increased sensitivity RNAi, neuro- <br /> The membrane topology predicted different         nal genes [20], refractory <br /> methods, SFINX tool [8,9], used display          RNAi compared tissue types. Strain <br /> the results. Eight membrane topology predictors            CB00907 (dpy-5(e907) I) used generating the <br /> used: Phobius [47], TMHMM2.0 [48], TMHMM1.0 [43],               transgenic lines [63]. <br /> PHDhtm [49], HMMTOP2.1 [50], HMMTOP1.0 [51], <br /> MEMSAT [52] TOPPRED [53]. In addition, Kyte-              RNAi screening <br /> Doolittle hydrophobicity curve [54] constructed         Generation cloning PCR products <br /> each putative protein sequence. Transmembrane regions           Total RNA extracted purified C. elegans using <br /> were considered positive predicted            TRIzol (Invitrogen Life Technologies, Carlsbad, CA) was <br /> majority methods, methods having          reverse transcribed using Reverse Transcription System <br /> a supporting Kyte-Doolittle hydrophobicity curve. Pho-          (Promega, Madison, WI) PCR products were <br /> bius predicts N-terminal signal peptides. The signal       generated using Advantage&#226;&#8222;&#162; 2 PCR Enzyme System <br /> peptides predicted Phobius verified                (Clontech, Palo Alto, CA) gene specific primers as <br /> SignalP1.1 [55]. Each program used default             primers spliced leader 1 (SL1) SL2 (Invit- <br /> settings.                                                       rogen Life Technologies): 95&#194;&#176;C 60 s, 35 cycles (95&#194;&#176;C <br />                                                                 30 s, 55&#194;&#176;C 30 s, 68&#194;&#176;C 4 min) followed additional <br />                                                                 extension 68&#194;&#176;C 4 min. See additional data file 1 the <br />  <br />  <br />                                                                                                                   Page 15 18 <br />                                                                                              (page number citation purposes) <br />  BMC Genomics 2004, 5:85                                                      http://www.biomedcentral.com/1471-2164/5/85 <br />  <br />  <br />  <br /> primer sequences used RNAi studies. Products            pPD34.09 [64]). Empty L4440 vector used negative <br /> were ligated linearized (XmaI) (New England                control. <br /> Biolabs, Frankfurt Main, Germany) dephosphor- <br /> ylated L4440 vector (Fire Laboratory [64]) using Rapid          RNAi injections <br /> DNA Ligation Kit (Roche, Mannheim, Germany)                 The L4440 vector (Fire Laboratory [64]) containing the <br /> transformed JM109 E. coli bacterial strain (Promega).      cloned gene fragment linearized separate reac- <br /> Colonies screened using XmaI, correct colonies        tions using restriction enzymes NcoI XhoI (New Eng- <br /> grown overnight cultures DNA extracted using         land Biolabs, Frankfurt Main, Germany), respectively. <br /> QIAfilter Plasmid Kit (QIAGEN, Hilden, Germany). The            The reactions purified single stranded RNA was <br /> vector insert sequenced using ABI PRISM&#194;&#174; Big       synthesized using T7 RNA polymerase (Promega, Madi- <br /> Dye&#226;&#8222;&#162; Terminator Cycle Sequencing Ready Reaction Kits            son, WI). The reactions mixed annealing <br /> (Applied Biosystems, Foster City, CA).                          performed produce double stranded (ds) RNA, <br />                                                                 subsequently purified. The dsRNA injected <br /> RNAi feeding                                                 undiluted young adult N2 hermaphrodites for <br /> Strains N2 rrf-3 used RNAi screening feed-      gene. The injected worms individual <br /> ing [30,65]. CGC [62] bacterial strain E. coli HT115(DE3)       plates split different incubation tem- <br /> was transformed L4440 vector (Fire Laboratory          peratures (15, 20 25&#194;&#176;C). Phenotypes scored for <br /> [64]) containing cloned gene fragment using standard        injected worms subsequent generations <br /> methods. The vector contains ampicillin (Amp) resist-        daily basis. Phenotypes scored criteria scoring <br /> ance strain HT115 tetracycline (Tet) resistant,       the RNAi feeding strain N2. <br /> bacteria selected Amp (75 &#194;&#181;g/ml) Tet (12.5 <br /> &#194;&#181;g/ml) plates. Single colonies picked grown         Generation transgenic lines <br /> cultures LB Amp (60 &#194;&#181;g/ml) Tet (12.5 &#194;&#181;g/ml)         The transgenic lines constructed Baille Labora- <br /> for 14&#226;&#8364;&#8220;17 h. The bacterial solution seeded NGM         tory, Simon Fraser University, Canada [66]. Transcrip- <br /> plates containing 1 mM IPTG 25 &#194;&#181;g/ml carbenicillin.         tional expression constructs gonadal injection were <br /> Seeded plates allowed dry room temperature.          generated using fusion PCR, known &quot;PCR-stitch- <br /> Eggs prepared standard bleaching method           ing&quot; [67]. Typically, 3 kb genomic DNA sequence <br /> transferred plates. N2 strain incubated 15, 20    immediately upstream predicted ATG initiator site, <br /> and 25&#194;&#176;C. rrf-3 incubated 15&#194;&#176;C 20&#194;&#176;C,            used putative promoter (see supplementary <br /> since strain temperature-sensitive decrease        material primer sequences). When upstream gene <br /> broodsize [20]. The hatched worms progeny             3 kb, size putative promoter was <br /> were scored number different phenotypes [30,44].       adjusted downwards. For genes operons, sequence <br /> The phenotypes assayed were: Adl (adult lethal), Bli (blis-     upstream gene operon used. The <br /> tering cuticle), Bmd (body morphology defect), Brd           putative promoter fused DNA fragment <br /> (low broodsize), Clr (clear), Dpy (dumpy), Egl (egg laying      containing gfp (green fluorescent protein) unc-54 <br /> defect), Emb (embryonic lethal), Gro (slow post-embry-          3&apos;UTR amplified vector pPD95.67 (Fire Laboratory <br /> onic growth), Him (high incidence males), Lon (long          [64]). See additional data file 1 primer sequences <br /> body), Lva (larval arrest), Lvl (larval lethal), Mlt (molt      used generating fusion PCR products. The resulting <br /> defect), Muv (multivulva), Prz (paralyzed), Pvl (protrud-       fusion PCR product injected purification into <br /> ing vulva), Rol (roller), Rup (ruptured), Sck (sick), Sma       gonad young adult hermaphrodites strain <br /> (small), Ste (sterile), Stp (sterile progeny), Unc (uncoordi-   CB00907 concentration 10 ng/&#194;&#181;l 100 <br /> nated). Emb defined greater 10% dead                ng/&#194;&#181;l dpy-5(+) plasmid (pCeh361) 1xTE buffer gen- <br /> embryos N2 greater 30% dead embryos            erate extrachromosomal array. Analysis expres- <br /> rrf-3. Ste Stp required brood size fewer      sion patterns different transgenic lines was <br /> for N2 fewer rrf-3. Each postembryonic        performed Vaz Gomes Laboratory, Karolinska Insti- <br /> phenotype required present 10%         tutet, Sweden. <br /> of analyzed worms. The experiment ongoing for <br /> about 4 generations phenotypes scored         Author&apos;s contributions <br /> daily basis. A constant supply transformed HT115 bac-        AH carried membrane topology predictions, phy- <br /> teria ensured. The experiments performed            logenetic analysis, domain assignments, RNAi studies and <br /> duplicates temperature gene worm           analysis gene expression transgenic worm strains. <br /> strain. For postembryonic phenotypes typically     ES participated membrane topology predictions, <br /> 20 worms plate scored. As positive control       phylogenetic analysis domain assignments. DB con- <br /> gene unc-22 (&quot;twitchin&quot;) used (Fire Laboratory vector       tributed making transgenic worm strains. AVG par- <br />                                                                 ticipated RNAi studies analysis gene <br />  <br />  <br />                                                                                                                   Page 16 18 <br />                                                                                              (page number citation purposes) <br />  BMC Genomics 2004, 5:85                                                                          http://www.biomedcentral.com/1471-2164/5/85 <br />  <br />  <br />  <br /> expression. All authors read approved final                             16.   Chluba-de Tapia J, Tapia M, Jaggin V, Eberle AN: Cloning a <br />                                                                                       human multispanning membrane protein cDNA: evidence <br /> manuscript.                                                                           new protein family. Gene 1997, 197:195-204. <br />                                                                                 17.   Hofmann K: A superfamily membrane-bound O-acyltrans- <br /> Additional material                                                                   ferases implications wnt signaling. Trends Biochem Sci <br />                                                                                       2000, 25:111-2. <br />                                                                                 18.   Reeves JE, Fried M: The surf-4 gene encodes novel 30 kDa <br />                                                                                       integral membrane protein. Mol Membr Biol 1995, 12:201-8. <br />      Additional File 1                                                          19.   Sijen T, Fleenor J, Simmer F, Thijssen KL, Parrish S, Timmons L, Plas- <br />      Primer sequences used RNAi gene expression studies.                       terk RH, Fire A: On role RNA amplification dsRNA- <br />      Click file                                                              triggered gene silencing. Cell 2001, 107:465-76. <br />                                                                                 20.   Simmer F, Tijsterman M, Parrish S, Koushika SP, Nonet ML, Fire A, <br />      [http://www.biomedcentral.com/content/supplementary/1471- <br />                                                                                       Ahringer J, Plasterk RH: Loss putative RNA-directed <br />      2164-5-85-S1.xls]                                                                RNA polymerase RRF-3 makes C. elegans hypersensitive to <br />                                                                                       RNAi. Curr Biol 2002, 12:1317-9. <br />                                                                                 21.   Ponting CP, Hutton M, Nyborg A, Baker M, Jansen K, Golde TE: Iden- <br />                                                                                       tification novel family presenilin homologues. Hum Mol <br />                                                                                       Genet 2002, 11:1037-44. <br /> Acknowledgements                                                                22.   Lercher MJ, Blumenthal T, Hurst LD: Coexpression neighbor- <br /> We thank Robert Johnsen, Domena Tu, Allan Mah Lily Fang                  ing genes Caenorhabditis elegans operons <br />                                                                                       duplicate genes. Genome Res 2003, 13:238-43. <br /> David Baillie Laboratory making gene fusion constructs injecting    23.   Somia NV, Schmitt MJ, Vetter DE, Van Antwerp D, Heinemann SF, <br /> them make transgenic worm strains genes study. Other               Verma IM: LFG: anti-apoptotic gene provides protec- <br /> strains kindly provided Caenorhabditis Genetics Center,             tion Fas-mediated cell death. Proc Natl Acad Sci U S A 1999, <br /> is funded NIH National Center Research Resources. We thank                 96:12667-72. <br />                                                                                 24.   Fraser AG, Kamath RS, Zipperlen P, Martinez-Campos M, Sohrmann <br /> the Fire Laboratory plasmids, Elizabeth Valenzuela technical assist- <br />                                                                                       M, Ahringer J: Functional genomic analysis C. elegans chro- <br /> ance Ivica Tamas critical reading manuscript. This work            mosome I systematic RNA interference. Nature 2000, <br /> supported grant Swedish Knowledge Foundation                    408:325-30. <br /> Research School Medical Bioinformatics Pfizer Corporation A.H.,       25.   Winston WM, Molodowitch C, Hunter CP: Systemic RNAi C. <br />                                                                                       elegans requires putative transmembrane protein SID-1. <br /> by grants Pfizer Corporation E.S A.V.G., grants from <br />                                                                                       Science 2002, 295:2456-9. <br /> Genome Canada Genome BC D.B.                                             26.   Tijsterman M, May RC, Simmer F, Okihara KL, Plasterk RH: Genes <br />                                                                                       Required Systemic RNA Interference Caenorhabditis <br /> References                                                                            elegans. Curr Biol 2004, 14:111-6. <br />                                                                                 27.   Feinberg EH, Hunter CP: Transport dsRNA cells the <br /> 1.      Lai CH, Chou CY, Ch&apos;ang LY, Liu CS, Lin W: Identification novel <br />                                                                                       transmembrane protein SID-1. Science 2003, 301:1545-7. <br />         human genes evolutionarily conserved Caenorhabditis <br />                                                                                 28.   Piccin A, Salameh A, Benna C, Sandrelli F, Mazzotta G, Zordan M, <br />         elegans comparative proteomics. Genome Res 2000, <br />                                                                                       Rosato E, Kyriacou CP, Costa R: Efficient heritable func- <br />         10:703-13. <br />                                                                                       tional <span id='am-83' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-84' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span>knock-out</span> adult phenotype Drosophila using a <br /> 2.      Kenyon C: A conserved regulatory aging. Cell 2001, <br />                                                                                       GAL4-driven hairpin RNA incorporating heterologous <br />         105:165-8. <br />                                                                                       spacer. Nucleic Acids Res 2001, 29:E55-5. <br /> 3.      O&apos;Kane CJ: Modelling human diseases Drosophila and <br />                                                                                 29.   Fortier E, Belote JM: Temperature-dependent gene silencing by <br />         Caenorhabditis. Semin Cell Dev Biol 2003, 14:3-10. <br />                                                                                       expressed inverted repeat Drosophila. Genesis 2000, <br /> 4.      Schulenburg H, Kurz CL, Ewbank JJ: Evolution innate <br />                                                                                       26:240-4. <br />         immune system: worm perspective. Immunol Rev 2004, <br />                                                                                 30.   Kamath RS, Fraser AG, Dong Y, Poulin G, Durbin R, Gotta M, Kana- <br />         198:36-58. <br />                                                                                       pin A, Le Bot N, Moreno S, Sohrmann M, Welchman DP, Zipperlen <br /> 5.      Remm M, Sonnhammer E: Classification transmembrane pro- <br />                                                                                       P, Ahringer J: Systematic functional analysis Caenorhab- <br />         tein families Caenorhabditis elegans genome iden- <br />                                                                                       ditis elegans genome using RNAi. Nature 2003, 421:231-7. <br />         tification human orthologs. Genome Res 2000, 10:1679-89. <br />                                                                                 31.   Maeda I, Kohara Y, Yamamoto M, Sugimoto A: Large-scale analysis <br /> 6.      Fitch WM: Distinguishing homologous analogous <br />                                                                                       gene function Caenorhabditis elegans high-through- <br />         proteins. Syst Zool 1970, 19:99-113. <br />                                                                                       RNAi. Curr Biol 2001, 11:171-6. <br /> 7.      Sonnhammer EL, Koonin EV: Orthology, paralogy proposed <br />                                                                                 32.   Wang K, Zhou B, Kuo YM, Zemansky J, Gitschier J: A novel mem- <br />         classification paralog subtypes. Trends Genet 2002, <br />                                                                                       ber zinc transporter family defective acrodermatitis <br />         18:619-20. <br />                                                                                       enteropathica. Am J Hum Genet 2002, 71:66-73. <br /> 8.      Sonnhammer EL, Wootton JC: Integrated graphical analysis of <br />                                                                                 33.   Taylor KM, Morgan HE, Johnson A, Nicholson RI: Structure-func- <br />         protein sequence features predicted sequence <br />                                                                                       tion analysis HKE4, member new LIV-1 subfamily <br />         composition. Proteins 2001, 45:262-73. <br />                                                                                       zinc transporters. Biochem J 2004, 377:131-9. <br /> 9.      SFINX [http://sfinx.cgb.ki.se/] <br />                                                                                 34.   Stathakis DG, Burton DY, McIvor WE, Krishnakumar S, Wright TR, <br /> 10.     Storm CE, Sonnhammer EL: Automated ortholog inference <br />                                                                                       O&apos;Donnell JM: The catecholamines (Catsup) protein of <br />         phylogenetic trees calculation orthology <br />                                                                                       Drosophila melanogaster functions negative regulator <br />         reliability. Bioinformatics 2002, 18:92-9. <br />                                                                                       tyrosine hydroxylase activity. Genetics 1999, 153:361-82. <br /> 11.     Bateman A, Birney E, Cerruti L, Durbin R, Etwiller L, Eddy SR, Grif- <br />                                                                                 35.   Lasswell J, Rogg LE, Nelson DC, Rongey C, Bartel B: Cloning and <br />         fiths-Jones S, Howe KL, Marshall M, Sonnhammer EL: The Pfam <br />                                                                                       characterization IAR1, gene required auxin conju- <br />         protein families database. Nucleic Acids Res 2002, 30:276-80. <br />                                                                                       gate sensitivity Arabidopsis. Plant Cell 2000, 12:2395-2408. <br /> 12.     Pfam Sanger Institute, version 11.0                    [ftp:// <br />                                                                                 36.   Begum NA, Kobayashi M, Moriwaki Y, Matsumoto M, Toyoshima K, <br />         ftp.sanger.ac.uk/pub/databases/Pfam/old_releases/Pfam11.0/] <br />                                                                                       Seya T: Mycobacterium bovis BCG cell wall lipopolysac- <br /> 13.     Gaither LA, Eide DJ: Eukaryotic zinc transporters their <br />                                                                                       charide induce novel gene, BIGM103, encoding 7-TM pro- <br />         regulation. Biometals 2001, 14:251-70. <br />                                                                                       tein: identification new protein family having Zn- <br /> 14.     Taylor KM, Nicholson RI: The LZT proteins; LIV-1 sub- <br />                                                                                       transporter Zn-metalloprotease signatures. Genomics <br />         family zinc transporters. Biochim Biophys Acta 2003, <br />                                                                                       2002, 80:630-45. <br />         1611:16-30. <br />                                                                                 37.   Schimmoller F, Diaz E, Muhlbauer B, Pfeffer SR: Characterization <br /> 15.     Weihofen A, Binns K, Lemberg MK, Ashman K, Martoglio B: Identi- <br />                                                                                       76 kDa endosomal, multispanning membrane protein <br />         fication signal peptide peptidase, presenilin-type aspartic <br />                                                                                       highly conserved evolution. Gene 1998, <br />         protease. Science 2002, 296:2215-8. <br />                                                                                       216:311-8. <br />  <br />  <br />  <br />  <br />                                                                                                                                           Page 17 18 <br />                                                                                                                      (page number citation purposes) <br />  BMC Genomics 2004, 5:85                                                                             http://www.biomedcentral.com/1471-2164/5/85 <br />  <br />  <br />  <br /> 38.   Blumenthal T, Evans D, Link CD, Guffanti A, Lawson D, Thierry-Mieg         61.   Brenner S: The genetics Caenorhabditis elegans. Genetics <br />       J, Thierry-Mieg D, Chiu WL, Duke K, Kiraly M, Kim SK: A global                   1974, 77:71-94. <br />       analysis Caenorhabditis elegans operons. Nature 2002,                   62.   Caenorhabditis Genetics Center [http://biosci.umn.edu/CGC/ <br />       417:851-4.                                                                       CGChomepage.htm] <br /> 39.   Lorenzi MV, Castagnino P, Aaronson DC, Lieb DC, Lee CC, Keck CL,           63.   Hodgkin J, Edgley M, Riddle DL, Albertson DG: Appendix 4: Genet- <br />       Popescu NC, Miki T: Human FRAG1 encodes novel mem-                             ics. In In The nematode Caenorhabditis elegans Edited by: Wood WB. <br />       brane-spanning protein localizes chromosome                              Cold Spring Harbor: Cold Spring Harbor Laboratory Press; <br />       11p15.5, region frequent loss heterozygosity                          1988:491-584. <br />       cancer. Genomics 1999, 62:59-66.                                           64.   Fire Laboratory [ftp://www.ciwemb.edu/pub/FireLabInfo/] <br /> 40.   Fukunaga-Johnson N, Lee SW, Liebert M, Grossman HB: Molecular              65.   Timmons L, Fire A: Specific interference ingested dsRNA. <br />       analysis gene, BB1, overexpressed bladder breast                     Nature 1998, 395:854. <br />       carcinoma. Anticancer Res 1996, 16:1085-90.                                66.   McKay SJ, Johnsen R, Khattra J, Asano J, Baillie DL, Chan S, Dube N, <br /> 41.   Maurel-Zaffran C, Chauvet S, Jullien N, Miassod R, Pradel J, Aragnol             Fang L, Goszczynski B, Ha E, Halfnight E, Hollebakken R, Huang P, <br />       D: nessy, evolutionary conserved gene controll <br /> </body></html>