<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal vol.6 no.3 pp.737-742, 1987 <br />  <br />  <br /> Yeast DNA polymerase -DNA primase complex: cloning PRI 1, <br /> a single essential gene related DNA primase activity <br />  <br />  <br />  <br />  <br /> Giovanna Lucchini, Stefania Francesconi, Marco Foiani,                     tivity correlates p58 p48 polypeptides (Plevani et <br /> Gianfranco Badaracco Paolo Plevani                                     al., 1985). Interestingly, DNA primases mouse <br /> Dipartimento di Genetica e di Biologia dei Microorganismi, Universita di <br />                                                                            myeloma cells Drosophila embryos related two <br /> Milano, Via Celoria 26, 20133 Milano, Italy                                polypeptides 58 kd 48 kd, DNA polymerase ac- <br />                                                                                              - <br />  <br />  <br />  <br />                                                                            tivity associated higher mol. wt protein species, when <br /> Communicated A.Falaschi                                                 proteolysis purification procedure prevented (Kaguni <br /> The immunopurified yeast DNA polymerase-DNA primase                        et al., 1983; Tseng Ahlem, 1984). <br /> complex constituted DNA polymerase I polypeptides                   The biochemical properties DNA polymerase-DNA <br /> by protein species, called p74, p58 p48,             primase complex extensively characterized (Plevani <br /> we immunologically unrelated. The gene encoding                 et al., 1984; Singh Dumas, 1984; Badaracco et al., 1985; <br /> the p48 polypeptide identified immunological                   Plevani et al., 1985), DNA polymerase I gene been <br /> screening Xgtll yeast genomic DNA library. Antiserum                  cloned (Johnson et al., 1985; Lucchini et al., 1985), in- <br /> specific p48 inhibits DNA primase, immunoreactive,                 formation available possible structural rela- <br /> inhibitory antibodies affinity-purified clone-                  tionships components complex. Furthermore, <br /> encoded protein, relating p48 polypeptide DNA                  p74 polypeptide copurifies DNA polymerase I, <br /> primase activity. The entire gene cloned,                 p58 p48 polypeptides associated non- <br /> 1.45-kb p48 mRNA overproduced cells containing                   denaturing conditions. It therfore difficult identify role <br /> gene high copy number. Gene disruption Southern                     p74 complex clarify p48 <br /> hybridization experiments demonstrate p48 protein                 p58 necessary primase activity. <br /> is encoded single gene performs essential                      Cloning genes encoding lower mol. wt polypeptides <br /> function.                                                                  essential step solving problems, final goals <br /> Key words: cloning/DNA primase/DNA replication/yeast                       reconstitution vitro DNA replication <br />                                                                            purified components, knowledge the <br />                                                                            genetic control complex. In paper, provide im- <br /> Introduction                                                               munological data showing polypeptides p74, p58 p48 <br /> Eukaryotic DNA synthesis occurs S phase cell             structurally different DNA <br /> cycle requires macromolecular interactions involving DNA               polymerase I, p48 polypeptide involved DNA <br /> and proteins. RNA synthesis necessary initia-            primase activity. <br /> tion step short RNA primers covalently link-              We used mixture antisera raised p74, p58 <br /> ed 5&apos; end nascent DNA chains isolated                   p48 protein species isolated SDS -polyacrylamide gel <br /> prokaryotic eukaryotic organisms (Komberg, 1980). A                    electrophoresis, screen yeast DNA library constructed in <br /> specialized enzymatic activity, devoted initiation DNA               Xgtl 1 expression vector (Young Davis, 1983a,b) we <br /> replication RNA primer synthesis, called DNA primase,              isolated positive recombinant phages grouped three <br /> first identified prokaryotes product dnaG gene           expected immunological classes. Here report cloning and <br /> is prototype enzyme. Recently, DNA primase                characterization gene encoding p48 polypeptide, which <br /> activity associated replicative DNA                shown perform essential function yeast cells. <br /> polymerase purified variety eukaryotic organisms (Con-           Results <br /> away Lehman, 1982; Yagura et al., 1982; Schioda et al., <br /> 1982; Tseng Ahlem, 1983; Wang et al., 1984; Chang et                   Specificity antisera raised single polypeptides of <br /> al., 1984; Plevani et al., 1984).                                          complex <br />    A DNA polymerase-DNA primase complex purified                    The major polypeptides revealed electrophoresis in <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> cells yeast Saccharomyces cerevisiae (Plevani et al.,          SDS -polyacrylamide gel yeast DNA polymerase -DNA <br /> 1984, 1985). This complex constituted polypeptides,             primase complex (Figure 1) completely different <br /> called respectively p180, p140, p74, p58 p48, according             proteins, lower mol. wt protein species derive from <br /> their mol. wt. The DNA polymerase present complex                higher mol. wt polypeptides proteolysis, even <br /> DNA polymerase I, accounts 90 % total activity            different proteins sharing structural charac- <br /> in yeast cells (Wintersberger Wintersberger, 1970; Chang,              teristics. In order discriminate possibilities, we <br /> 1977) biochemical properties similar             prepared specific antisera injecting rabbits single poly- <br /> mammalian DNA polymerase ca. The DNA polymerase activity                   peptides, p140, p74, p58 p48, purified electrophoresis <br /> is associated polypeptides p180 p140 (Plevani et al.,             SDS -polyacrylamide gels, used test whole <br /> 1985), proteolytic derivative p180,                complex immunoblotting. As shown Figure 1, antibodies <br /> demonstrated biochemical immunological studies (Plevani             raised p140 (which powerfully inhibit DNA <br /> et al., 1985; Lucchini et al., 1985; paper). DNA primase              polymerase I activity) recognize p140 itself p180, <br /> can dissociated DNA polymerase, primase ac-               ladder polypeptides intermediate mol. wt. <br />                                                                                                                                            737 <br />  G.Lucchini et al. <br />  <br />  <br />  <br />  <br /> Fig. 1. Specificity immunological reagents. The immunopurified <br /> DNA polymerase-DNA primase complex, electrophoresis 10% <br /> SDS-polyacrylamide gel, stained Coomassie (lane 1) or <br /> transferred nitrocellulose filters, probed antisera <br /> raised p140 (lane 2), p74 (lane 3), p58 (lane 4) p48 (lane 5), <br /> respectively, purified electrophoresis SDS-polyacrylamide gel. <br />  <br /> This result confirms previous data suggesting p140 is <br /> a proteolytic derivative p1 80 bands repre-            Fig. 2. Immunological classification positive recombinant phages. For <br /> sent intermediate degradative products (Plevani et al., 1985). On            phage identified immunoscreening Xgtl 1 library, l-y1 <br />                                                                              drop containing 100 plaque forming units (PFU) spotted lawn of <br /> the contrary, remaining antisera reacts           E. coli Y1090 cells. Four identical independent plates prepared, which <br /> with specific polypeptide. These findings clearly indicate          treated screening procedure. The corresponding filters <br /> DNA polymerase I polypeptides p74, p58 p48                   incubated antisera raised purified p48 (Ab 48), <br /> immunologically related likely represent struc-          purified p58 (Ab 58), purified p74 (Ab 74) mixture the <br />                                                                              specific antisera (Mix Ab) used initial screening, <br /> turally independent proteins.                                                respectively. Spots IA 6F correspond Xgtl vector. <br /> Immunoscreening expression library <br /> The high specificity immunological reagents described                 Table I. Inhibition DNA primase activity Ab 48 X105 affinity- <br /> above allowed use immunological approach iden-             purified antibodiesa <br /> tification genes encoding polypeptides p47, p58 p48. <br /> As previously DNA polymerase I gene                 Antiserum dilution                         % DNA primase activity b <br /> (Johnson et al., 1985; Lucchini et al., 1985), used                                                        Ab 48            X105-Ab 48 <br /> purpose yeast DNA library constructed expression vec-               1:128                                     100                 100 <br /> tor Xgtl (Young Davis, 1983a,b).                                         1:64                                      103                  87 <br />    We screened 5 x 105 recombinant phages mixture                  1:32                                       98                  81 <br /> the antisera specific respectively p74, p58 p48,              1:16                                       75                  70 <br /> identified total 32 positives, purified             1:8                                        65                  48 <br /> retested mixture single antisera. As              1:4                                        45                  38 <br /> shown Figure 2, specific antiserum identifies distinct             1:2                                        28                  27 <br /> class recombinant phages, overlapping             aTen nanograms purified DNA primase (Plevani et al., 1985) were <br /> classes. This consistent previous statement             incubated serial dilutions antiserum raised purified p48 (Ab <br /> three polypeptides analysis immunologically cor-               48) X105 affinity-purified antibodies (X105-Ab 48). Dilutions made <br /> related.                                                                     50 mM Tris-HCI pH 8.0, 100 mM NaCl, 0.1 mg/ml bovine serum <br />                                                                              albumin, 10% pre-immune serum, samples incubated 0&#194;&#176;C for <br /> Characterization class p48 recombinants                                   2.5 h microtiter plates pre-coated 1 mg/ml bovine serum albumin. <br /> Polypeptide p48 present p58 DNA               The DNA primase activity remaining antiserum treatment was <br /> primase fraction, purified DNA                         assayed described Materials methods. <br /> polymerase -DNA primase complex (Pelvani et al., 1985). Fur-                 b100% DNA primase activity corresponds 33 pmol [a-32P]dAMP <br />                                                                              incorporated 30 min 37&#194;&#176;C. <br /> thermore, shown Table I, antiserum raised the <br /> purified p48 protein inhibits primase activity, tested               reason, focused efforts cloning of <br /> purified yeast DNA primase. The inhibitory effect specific            p48 gene. <br /> DNA primase, effect observed yeast                         The 2.2-kb yeast DNA EcoRI fragment (Figure 6) isolated <br /> Escherichia coli DNA polymerase I activities, pre-                   p48 phages (X105) used probe test <br /> immune serum totally inactive. These data support                    remaining phages plaque hybridization. Eight the <br /> hypothesis p48 involved primase activity. For                p48 clones, belong X105 cross-hybridization <br /> 738 <br />                                                                                                          A yeast essential gene related DNA   primase <br />  <br />  <br />  <br />                &apos;.9- <br />  <br />  <br />  <br />  <br /> Fig. 3. Analysis proteins encoded X105 genome. E. coli Y1090 <br /> cells infected X105 (lanes 1 2) Xgt 1l (lanes 1 and <br /> 4) respectively, eventually treated (lanes 2 4) 10 mM IPTG. <br /> Lysates prepared described Materials methods               Fig. 5. Northern blot analysis poly(A)+ yeast RNA. Poly(A)+toRNA, <br /> electrophoresed SDS-polyacrylamide gels. (A) Coomassie-stained gel;        prepared strain S288c, electrophoresed transferred nitro- <br /> (B) blot analogous gel probed antiserum raised     cellulose filters. Filters probed 2.2-kb EcoRI DNA fragment <br /> purified p48 polypeptide.                                                     purified X105 (lane 1), internal 0.7-kb EcoRI-BgllI fragment <br />                                                                               (lane 2), 1.2-kb HinduI URA3 probe (lane 3), respectively. <br />                                                                                       did contain intact ,(-galactosidase, IPTG-inducible <br />                                                                                       p48 -j3-galactosidase hybrid protein, strongly reacts with <br />                                                                                       antiserum raised purified p48 polypeptide. The <br />                                                                                       mol. wt hybrid polypeptide 160 kd; the <br />                                                                                                                               - <br />  <br />  <br />  <br />  <br />                                                                                                                               114 kd, coding <br />                                                                                       fl-galactosidase portion accounts forfused <br />                                                                                       region    p48 gene          frame lacZ. <br />                                                                                          <span id='am-3' about='protege:TO' typeof='owl:Thing'>To</span> confirm identity yeast gene represented <br />                                                                                       X105, verified encoded yeast protein fragment <br />                                                                                       affinity-purify antibodies specifically reacting p48 <br />                                                                                       polypeptide. As shown Figure 4, X105 affinity-purified <br />                                                                                       antibodies (see Materials methods) strongly react with <br />                                                                                       p48 polypeptide, confirming foreign protein en- <br />                                                                                       coded X105 polypeptide correlated yeast DNA <br />                                                                                       primase activity share epitopes anti- <br />                                                                                       bodies. Moreover, shown Table I, affinity-purified <br />                                                                                        antibodies inhibit DNA primase, confirming p48 <br />                                                                                        polypeptide correlated primase activity. <br />                                                                                        mRNA analysis <br />                                                                                        The size p48 mRNA expected 1.2 kb, <br />                                                                                        theoretical <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> sequence encoding 48-kd polypep- <br />  Fig. 4. Immunoblot analysis using X105 affinity-purified antibodies. The              tide. In order verify sequences homologous yeast <br />  immunopurified   yeast  DNA    polymerase-DNA        primase   complex   was <br />                                                                                        mRNA appropriate size present 2.2-kb EcoRI <br />  electrophoresed SDS-polyacrylamide gel stained Coomassie               DNA insert X105 (see Figure 6), used probe <br />  (lane 1) transferred nitrocellulose filter, reacted with <br />  X105 affinity-purified antibodies (lane 2). Identical results     obtained       Northern blot analysis yeast poly(A)+ RNA. This probe <br />  using X105  affinity-purified  antibodies    anti-p48  serum     identified major 1.45-kb mRNA species additional <br />  mixture antisera used    initial screening.                                minor species higher mol. wt (Figure 5). The 2.2-kb EcoRI <br />                                                                                        fragment contain additional sequences homologous to <br />  group (data shown), considered <br />                                                                                   the <br />                                                                                        p48 gene, 0.7-kb EcoRl-Bgml portion should <br />   major candidates encoding              p48  polypeptide.     internal p48 sequence. As shown Figure 5, last <br />                                                                                  blot <br />     This hypothesis          supported        results   Western           fragment does hybridize 1.45-kb mRNA species, <br />   analysis   lysates   E.   coli   cells  infected     recombinant         identified p48 message. <br />   phages grown induced uninduced conditions                           Cloning characterization entire p48 gene <br />   lacZ expression. The <span id='am-82' about='xsp:pattern' typeof='owl:Thing'>pattern</span>, shown Figure 3 phage                        The coding region p48 gene fused frame lacZ <br />    &gt;105, observed lysates derived                         coding region X105 genome, con- <br />   phages belonging X105 cross-hybridization group. They <br />                                                                                                                                                        739 <br />  G.Lucchini et al. <br />  <br />                                                  . <br />                                                       1 kb      .~~~~~~~~~~~~~~~ <br />  <br />  <br />  <br />  <br />                     Ir _ <br />                     E _ <br />                      cu   cm <br />                                - <br />                                X..                                                            4--   cc     0                   E <br />                                                                                    m           0-                              cu <br />                                    a.       z       W.&lt;                   m          cn &lt;         w <br />                                                                                                                 0 <br />                                                                                                                Za <br />                                                                                                  .                         m Genomic <br />                                                                                                                                   Locus <br />                                             mRNA - <br />                                                       A 105 *- <br />  <br /> Fig. 6. Restriction map p48 locus. The map determined restriction analysis YEpB2 plasmid confirmed Southern blot analysis of <br /> total yeast DNA digested appropriate enzymes probed 2.2-kb EcoRI fragment. The direction transcription indicated (the exact 5&apos; <br /> and 3&apos; end points determined). The position size X105 insert indicated horizontal bar map, asterisk <br /> denoting position amino-terminal portion lacZ gene. <br />  <br />  tain complete p48 sequence. To isolate entire gene, we <br />  used 0.7-kb EcoRI-Bgll internal fragment probe screen <br />  yeast DNA library carrying 10-20-kb inserts high copy <br />  number vector YEp24 (Carlson Botstein, 1982). We isolated <br />  recombinant clones, shown carry same <br />  10-kb insert. An internal 5.5-kb BamHI restriction fragment (see <br />  Figure 6), cloned BamHI site plasmid YEp24. <br />  The derivative plasmid YEpB2 transformed haploid <br />  strain TD28, transformed cells seen overproduce <br />  1.45-kb p48 mRNA (data shown), indicating that <br />  complete gene contained 5.5-kb BamHI fragment. <br />    The direction transcription gene (Figure 6) should <br />  NruI site SstI site, according orientation <br />  yeast DNA insert lacZ gene X)105. Neither <br /> the BamHI-NruI fragment left-hand map, nor <br /> the SstI-BamHI fragment right contain sequences <br /> homologous p48 mRNA (data shown), locating <br /> both 5&apos; 3&apos; end points message inside the <br />  1.9-kb NruI-SstI region. <br />    The NruI -SstI fragment ligated SmaI SstI sites                      Fig. 7. In vitro translation p48 transcript. A wheat germ lysate <br />                                                                                        supplemented [35S]methionine incubated wtih transcripts BglIl <br /> of plasmid pGEMTM-2 (Promega Biotec), control                             cut pPPI (lane 3), SstI cut pPPl (lane 4) transcript (lane <br /> the T7 phage promoter. The derivative plasmid, pPP1, cut                           2). Translation products analysed electrophoresis 15% <br /> either internal Bglll site, SstI end point                     SDS-polyacrylamide gel autoradiography. Lane 1 shows Western <br /> insert. The linearized plasmids used vitro transcrip-                      blot DNA polymerase-DNA primase complex electrophoresed the <br />                                                                                        gel probed anti-p48 specific serum. [14C]methylated <br /> tion T7 RNA polymerase purified transcripts                           protein standards (Amersham) used mol. wt makers. <br /> were translated vitro. As shown Figure 7, major transla- <br /> tional product transcript derived SstI-digested <br /> plasmid polypeptide, mobility SDS -polyacryl-                         division cycle gene, CDC29, maps chromosome <br /> amide gel indistinguishable p48 polypeptide.                       (Pringle Hartwell, 1981). We verified were <br /> Therefore, cloned fragment confirmed contain en-                         gene, transforming uracil prototrophy ura3-52, <br /> tire p48 coding sequence, transcribed expected                         cdc 29 strain plasmid YEpB2. The presence cells of <br /> direction. The Bgll site certainly internal coding region,                   p48 gene high copy number does change thermo- <br /> because truncated polypeptides obtained vitro                       sensitive <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> (data shown), demonstrating that <br /> translation transcript derived BglII-digested                          gene cloned CDC29. Further experiments were <br />                                                                                        necessary test p48 function essential. <br /> plasmid, confirming translation termination signal                          We carried gene-disruption experiment, using method <br /> of gene located 0.75-kb BglII-SstI region. <br />                                                                                        Shortle et al. (1982). A plasmid containing deletions the <br /> p48 encoded single essential gene                                              5&apos; 3&apos; ends gene (pSF48) constructed replacing <br /> Blot hybridization BamHI, Sall, KpnI XhoI digests total                      0.375-kb EcoRI-BamHI region plasmid YIp5 [a pBR322 <br /> yeast DNA 2.2-kb EcoRI fragment probe showed                               derivative containing yeast URA3 gene (Scherer Davis, <br /> single bands (data shown), indicating p48 gene                       1979)] 0.7-kb EcoRI-BglIl fragment p48 gene. <br /> occurs single copy haploid genome. Since showed                           Plasmid pSF48 cut HindIll, direct integration <br /> that p48 polypeptide involved DNA primase activity,                          p48 locus, SmaI, direct integration ura3; <br /> we expect essential yeast cells.                                    cut plasmids used transform diploid strain DES5 <br />    Blot hybridization intact yeast chromosomes fractionated                      uracil prototrophy. One transformant carrying single copy <br /> orthogonal field gel electrophoresis (Carle Olson, 1985)                           plasmid integrated ura3 p48 locus <br /> localizes p48 gene chromosome IX. A little studied cell                         identified blot hybridization EcoRI digests total yeast <br />  <br />  <br /> 740 <br />                                                                                                                   A yeast essential gene related DNA       primase <br />  <br />  Table II. Tetrad analysis ura 3-52/ura 3-52 diploid (DES5) transformed    ming identity gene, correlation <br />  plasmid pSF48                                                               DNA primase. <br />                                                                                      The gene transcribed 1.45-kb mRNA, an <br />  Location     Total number Viable spores Number Genotype            appropriate size encoding 48-kd polypeptide. This mRNA <br />  integrated pSF48    tetrads   tetrad    tetrads   spores <br />                                                 observed ura+ ura- <br />                                                                                   species overproduced yeast cells containing high copy <br />                                                                                  number plasmid carrying 5.5-kb yeast BamHI DNA fragment. <br />  At p48 locus    21             4                0                               Subcloning 1.9-kb NruI -SstI region, control <br />                                     3                0                           T7 phage promoter, allowed confirm direc- <br />                                     2               19            0     38       tion transcription region includes the <br />                                     1                2            0      2       entire p48 coding sequence; fact vitro transcribed <br />                                     o                0                           mRNA translated vitro polypeptide, mobility <br />  At ura 3 locus 21              4               21           42    42        SDS -polyacrylamide gel indistinguishable of <br />                                     3                0                           p48 polypeptide. <br />                                     2                0                              The gene occurs single copy yeast genome has <br />                                     1                0                           locaized chromosome IX. Disruption gene lethal <br />                                     o                0                           cell, p48 polypeptide performs essential <br />                                                                                  function yeast. <br /> DNA EcoRI-Bglll fragment probe. Integration                          Altogether data demonstrate identified cloned <br /> the HindIl cut plasmid result disruption copy               yeast essential gene, propose PRII, because <br /> of p48 gene diploid. As shown Table II,                  cloned eukaryotic gene related DNA primase ac- <br /> a diploid sporulated, tetrads exhibited 2:2 lethality,                 tivity. However, unknown p48 polypeptide <br /> that linked URA3 gene plasmid. All un-                     itself sufficient enzymatic activity, interactions <br /> viable germinated spores, 16 h complete medium, ap-                     p58 polypeptide and/or identified proteins <br /> peared undivided elongated cells, eventually                  necessary. The characterization isolated gene of <br /> buds, morphology typical cell division cycle mutants im-                    vitro translated vivo overexpressed product, well <br /> paired DNA replication (Hartwell, 1971). No lethality                     isolation conditional lethal mutants vitro <br /> observed spores containing copy pSF48 integrated                     mutagenesis, enable clarify points. Cloning <br /> to ura3 (Table II), showing plasmid does contain                    p74 p58 genes, progress, useful <br /> sequences harmful se cells. Therefore, spore lethality                purpose understanding activities interactions <br /> in case integration p48 locus                   DNA polymerase-DNA primase complex. <br /> disruption gene resulting lack p48 <br /> polypeptide.                                                                     Materials methods <br />                                                                                  Strains plasmids <br /> Discussion                                                                       Saccharomyces cerevisiae strains S288c, a, prototroph, TD28, a, ura 3.52, <br />  A DNA primase activity associated                       inosl, can], Dr G.R.Fink MIT; strain ts 17048, a, cdc 29, ade 1, <br />                                                                                  ade 2, ura 1, 7, lys 2, tyr 1, gal 1, ade 7, Yeast Genetic Stock <br />  replicative DNA polymerase eukaryotic organisms (for                 Center, Berkeley, CA; strain DGL23/la, a, ura 3-52, cdc 29, inos 1, 1, <br /> review, Campbell, 1986). It suggested, com-                      strain DES5 a/lr, ura 3-52/ura 3-52, constructed laboratory. <br /> parison well-defined prokaryotic systems,                        Escherichia coli strain Y1090 (AlacU169 proA+ Alon araD139 strA supF <br /> eukaryotic DNA polymerase -DNA primase complexes                           trpC22:TnlO pMC9) Xgtl 1 library kindly provided Drs R.Young <br />                                                                                  R.Davis Stanford University. The yeast genomic library YEp24 was <br /> be involved elongation nascent DNA chains,                kindly provided Dr D.Botstein, MIT. Plasmid pGEMTM-2 from <br /> but formation initiation complex replica-              Promega-Biotec, Cambridge, MA. <br /> tion fork.                                                                       Enzyme antibody preparation <br />    We characterized polypeptides compose                  The isolation DNA polymerase I-DNA primase complex immuno- <br /> yeast DNA polymerase-DNA primase complex using specific                       affinity chromatography physical separation enzyme activities <br /> immunological reagents. In complexes, high mol. wt                     described (Plevani et al., 1985). <br /> polypeptides, p180 p140, associated DNA poly-                          DNA primase activity assayed poly(dT) template measuring the <br /> merase activity (Plevani et al., 1985). Our immunological data                   incorporation [U-32P]dAMP acid-insoluble materials presence of <br />                                                                                  200 units/ml E. coli DNA polymerase I (Plevani et al., 1985). <br /> confirm p140 degradative product p180,                  Polyvalent rabbit antisera pl40, p74, p58 p48 polypeptides of <br /> is immunological cross-reaction p140                     DNA polymerase-DNA primase complex obtained separating <br />                                                                                   - 100 jig complex 10% SDS-polyacrylamide gel. After brief stain- <br /> three polypeptides complex, im- <br /> munologically unrelated other. Moreover, antiserum                   ing Coomassie Blue, single protein bands sliced gel, <br />                                                                                  homogenized TBS (50 mM Tris-HCI, pH 8, 150 mM NaCl), the <br /> raised purified p48 polypeptide specifically inhibits                polyacrylamide suspension directly used imunize rabbits multiple intra- <br /> the activity purified yeast DNA primase, providing                   dermal injection (Vaitukaitis, 1981). <br /> first direct evidence protein involved primase ac-               Plaque screening <br /> tivity.                                                                          The screening Xgt 1 I yeast DNA library carried following pro- <br />    The genes encoding p74, p58 p48 polypeptides                     cedure used isolation DNA polymerase I gene (Lucchini et al., 1985). <br /> been identified using mixture specific antisera screen                The anti-p48, anti-p58 anti-p74 sera used 1:50 dilution. <br /> a yeast genomic Xgtl 1 library. Recombinant phages carrying                 1251-labeled protein A (0.5 ACi 90-mm-diameter filter; sp. act. 30 ptCi/jlg, <br />                                                                                  Amersham) used visualize positive plaques. <br /> of p48 gene isolated positive reaction <br /> with p48 specific antiserum. Furthermore, lysates E. coli                 Preparation E. coli lysates <br />                                                                                  Approximately 2 x 103 PFU plated lawn E. coli strain Y1090. <br /> cells infected phages affinity-purify p48 antibodies,                 One 90-mm-diameter plate prepared lysate. After 2.5 h incuba- <br /> that specifically inhibit DNA primase activity, confir-                tion 42&#194;&#176;C, 1 ml 250 mM isopropyl thio-1-D-galactopyranoside (IPTG) was <br />  <br />                                                                                                                                                               741 <br />  G.Lucchini et al. <br />  <br /> eventually added plates plates incubated additional        Kornberg,A. (1980) DNA Replication. W.H.Freeman Co., San Francisco. <br /> 5 h 37&#194;&#176;C. Lysates recovered 8 ml 50 mM Tris-HCl (pH 8),               Laemmli,U.K. (1970) Nature, 227, 680-685. <br /> 200 mM NaCl, 20 mM 2-mercaptoethanol, 20 mM EDTA, concentrated                 Lucchini,G., Brandazza,A., Badaracco,G., Bianchi,M. Plevani,P. (1985) <br /> adding (NH4)2SO4 70% saturation. After 30 min centrifugation 12 000    <br /> </body></html>