<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Aug. 2000, p. 5939&#226;&#8364;&#8220;5946                                                                                 Vol. 20, No. 16 <br /> 0270-7306/00/$04.00&#226;&#171;&#185;0 <br /> Copyright &#194;&#169; 2000, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />          The Two Proteins Pat1p (Mrt1p) Spb8p Interact In Vivo, <br />                  Are Required mRNA Decay, Are <br />                        Functionally Linked Pab1p <br />                             CLAIRE BONNEROT,1 RONALD BOECK,2                          AND   BRUNO LAPEYRE1* <br />                   Centre Recherche Biochimie Macromole&#194;&#180;culaire du CNRS, 34293 Montpellier, France,1 and <br />                          Centre Me&#194;&#180;dical Universitaire, Universite&#194;&#180; Gene`ve, CH-1211 Geneva, Switzerland2 <br />                         Received 27 January 2000/Returned modification 28 March 2000/Accepted 15 May 2000 <br />  <br />              We report characterization bypass suppressor pab1&#226;&#338;&#172; leads fourfold stabilization <br />           unstable MFA2 mRNA. Cloning wild-type gene suppressor reveals identical <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br />           PAT1 (YCR077c), gene product reported interact Top2p. PAT1 essential gene, but <br />           deletion leads thermosensitive <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. Further analysis shown PAT1 allelic mrt1-3, <br />           <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span>mutation</span> previously reported affect decapping bypass suppress pab1&#226;&#338;&#172;, case dcp1, <br />           spb8, mrt3. Coimmunoprecipitation experiments Pat1p associated Spb8p. On sucrose <br />           gradients, proteins cosediment fractions containing polysomes. In absence Pat1p, <br />           however, Spb8p longer cofractionates polysomes, removal Spb8p leads sharp <br />           decrease level Pat1p. Our results suggest factors involved mRNA degradation could <br />           associated mRNA translated, awaiting specific signal commit mRNA to <br />           degradation pathway. <br />  <br />  <br />    A general pathway mRNA degradation yeast Sac-                   established structures present ends of <br /> charomyces cerevisiae described mRNAs                  mRNA, cap 5&#226;&#172;&#732; end poly(A) tail 3&#226;&#172;&#732; <br /> deadenylated prior decapping degraded                     end, act synergistically enhance mRNA translation (17, 41). <br /> 5&#226;&#172;&#732;-to-3&#226;&#172;&#732; direction exonuclease Xrn1p (4, 12). The de-                 The cap structure poly(A) tail play major role in <br /> capping enzyme identified yeast named Dcp1p                  mRNA stability, target steps of <br /> (5). In yeast strains, depletion Dcp1p leads slow-              mRNA degradation. The poly(A)-binding protein Pab1p es- <br /> growth phenotype stabilization mRNAs                      tablishes bridge poly(A) tail cap to <br /> accumulated capped oligoadenylated form (5). How-                   interaction initiation factor eIF4G. This interaction <br /> ever, functional, Dcp1p requires activity                   facilitates recruitment 40S ribosomal subunit onto <br /> Dcp2p, putative pyrophosphatase (14). Recently, polypep-                 mRNA, allowing translation initiation proceed (41&#226;&#8364;&#8220; <br /> tide (Mr 70,000) copurifying Dcp1p identified               43). Pab1p plays role mRNA stability, its <br /> Ssa1p Ssa2p, Hsp70 family members (50). Interestingly,                absence mRNA decapped deadenylated (9). <br /> mutations inhibit decapping, vps16 mrt1, enhance                Relationships translation degradation further <br /> interaction Ssa1p Ssa2p Dcp1p. This observation                   supported observation mutations 5&#226;&#172;&#732; region of <br /> suggests Ssa1p Ssa2p involved regulating                 PGK1 mRNA inhibit translation stimulate decay <br /> the activity Dcp1p.                                                       (26, 30). Moreover, mutations genes coding <br />    In addition Dcp1p Dcp2p, factors regu-                 translation initiation factors lead increased rates of <br /> late decapping activity, mutations               deadenylation decapping (39). <br /> isolated lead stabilization capped, oligoadenylated                  A search suppressors deletion PAB1 gene <br /> mRNAs. The best characterized factors Spb8p (6),                 (spb) yeast proven powerful tool isolate factors <br /> a protein containing Sm-like domain                    involved ribosome synthesis mRNA turnover (6, <br /> referred Lsm1p (40). The Sm motif set                 35). Several mutations alter mRNA decapping turned out <br /> proteins interact small nuclear RNAs involved               suppress pab1&#226;&#338;&#172; mutation, case dcp1, mrt1, and <br /> mRNA splicing. However, Spb8p distinguished                    mrt3 (21). Recently, reported isolation spb8-2 by <br /> Sm Lsm proteins associated known                    using transposon insertion mutagenesis yeast genome <br /> small nuclear RNA (29, 38). Mutations                     isolate new spb mutants (6). During course anal- <br /> loci named mrt1, mrt3, vps16 lead accumulation                  ysis, spb mutants linked the <br /> capped, oligoadenylated mRNAs (21, 50). While VPS16                      transposon insertion linked secondary phenotype <br /> been characterized (22, 50), genes mrt1 mrt3                used subsequent cloning. Here re- <br /> not cloned.                                                         port characterization spb mutant turned to <br />    Several lines evidence indicate existence link                identical mrt1 previously characterized gene <br /> between mRNA translation degradation (23). It                     PAT1. We Pat1p associated Spb8p the <br />                                                                              proteins cofractionate polysomes sucrose gradients. <br />  <br />   * Corresponding author. Mailing address: Centre Recherche                                   MATERIALS AND METHODS <br /> Biochimie Macromole&#194;&#180;culaire du CNRS, 1919 Route Mende, 34293                 Microbiological methods recombinant DNA work. The strains used this <br /> Montpellier Cedex 5, France. Phone: 33 467 61 36 80. Fax: 33 467 04          study handled standard techniques (20). They constructed <br /> 02 31. E-mail: lapeyre@crbm.cnrs-mop.fr bonnerot@crbm.cnrs                study (Table 1) derivative W303 (3) obtained R. <br /> -mop.fr.                                                                     Parker: yRP1066 yRP1067 (21), yRP1070 yRP1062 (5), yRP1070 <br />  <br />                                                                       5939 <br />  5940       BONNEROT ET AL.                                                                                                                       MOL. CELL. BIOL. <br />  <br />  <br />            TABLE 1. Yeast strains constructed study                         GCGAATTCTTAGTACATGTCAGATTTATG-3&#226;&#172;&#732;) cloned BamHI- <br />                                                                                      EcoRI fragment pGEX-2T expression vector (Pharmacia) plas- <br />   Strain                     Relevant genotypea                        Plasmid       mid pRB48. <br />                                                                                         RNA work. RNA extraction performed previously described (11). Tran- <br /> YCB461            MATa trp1&#226;&#338;&#172;                                        pRP485           scriptional shutoff experiments described previously (13) with <br /> YBL4145           MATa spb10-1 pab1&#226;&#338;&#172;::HIS3                                           strains containing plasmid pRP485. RNA samples separated electro- <br /> YCB557            MAT&#226;?&#163; spb10-1                                                       phoresis 1.5% agarose slab gels. Capillary transfer positively charged nylon <br /> YCB451            MAT&#226;?&#163; spb10-1                                      pFL44-A1         performed using 40 mM NaOH transfer solution. Hybridization was <br /> YCB452            MAT&#226;?&#163; spb10-1                                      pFL44-A1K        described previously (10) oRP140 (9) probe reveal specif- <br /> YCB571            MATa/&#226;?&#163; pat1&#226;&#338;&#172;::TRP1/PAT1                                            ically MFA2pG species PCR fragment corresponding scR1 RNA <br /> YCB499            MATa pat1&#226;&#338;&#172;::TRP1                                                   served internal loading standard (15). This fragment PCR ampli- <br /> YCB467            MATa pat1::TRP1 pab1&#226;&#338;&#172;::HIS3                       pAS77            fied scR1-U (5&#226;&#172;&#732;-GGCTGTAATGGCTTTCTG-3&#226;&#172;&#732;) scR1-L (5&#226;&#172;&#732;-CCTTT <br /> YCB573            MATa/&#226;?&#163; pat1&#226;&#338;&#172;::TRP1/spb10-1                                         GCTGACGCTGGAT-3&#226;&#172;&#732;) genomic DNA template. Quantification was <br /> YCB572            MATa/&#226;?&#163; pat1&#226;&#338;&#172;::TRP1/pat1&#226;&#338;&#172;::TRP1                                     performed PhosphorImager (Molecular Dynamics). <br />                                                                                         Preparation rabbit antibody raised recombinant Spb8p. Plasmid <br /> YCB482            MAT&#226;?&#163; pat1&#226;&#338;&#172;::TRP1                                  pRP 485 <br />                                                                                      pRB48 transformed BL21 bacterial strain express fusion protein <br /> YCB574            MATa/&#226;?&#163; mrt1-3 pat1&#226;&#338;&#172;::TRP1 <br />                                                                                      consisting glutathione S-transferase followed thrombin cleavage site <br /> YCB575            MATa/&#226;?&#163; mrt3-1 pat1&#226;&#338;&#172;::TRP1                                          Spb8p sequence. A bacterial lysate obtained incubated with <br /> YCB562            MATa dcp1&#226;&#338;&#172;::URA3 pat1&#226;&#338;&#172;::TRP1                                       glutathione-Sepharose 4B resin (Pharmacia). After washes, beads were <br /> YCB563            MAT&#226;?&#163; xrn1&#226;&#338;&#172;::URA3 pat1&#226;&#338;&#172;::TRP1                                       treated thrombin release recombinant Spb8p. A 1-liter volume of <br /> YBL4452           MATa spb8&#226;&#338;&#172;::LEU2                                                   bacterial culture gave ca. 1.8 mg recombinant protein. Rabbits injected <br /> YCB568            MATa spb8&#226;&#338;&#172;::LEU2 pat1&#226;&#338;&#172;::TRP1                                       times 100 &#226;?&#174;g recombinant protein injection 4-month <br /> YCB536            MAT&#226;?&#163; PAT1-protA::TRP1, pep4::URA3                                  period bleeding (Berkeley Antibody Co., Berkeley, Calif.). <br />   a <br />                                                                                         Protein extraction yeast cells polysome analysis. Protein extracts <br />     All strains ade2-1 his3-11 leu2-3,112 trp1-1 trp1&#226;&#338;&#172; ura3-1 can1-100,       obtained mid-log-phase culture trichloroacetic acid (TCA) extrac- <br /> unless stated.                                                             tion. TCA directly added culture medium reach final concentra- <br />                                                                                      tion 20%. Yeast cells harvested centrifugation, washed 10% TCA, <br />                                                                                      frozen liquid nitrogen. Cells broken 10% TCA zirconium <br /> transformed pRP801 (26). 5-Fluoroorotic acid (5-FOA) bought            beads (Biospec Products, Bartlesville, Okla.) 10 min multimixer (VWR, <br /> Toronto Research Chemicals (Toronto, Ontario, Canada). 5-FOA used             South Plainfield, N.J.). Extracts collected centrifuged, pellets <br /> counterselect URA3 marker, i.e., case, select loss      resuspended Laemmli loading buffer (25). <br /> URA3 PAB1 plasmid. Yeast transformation performed using lithium              Native protein extracts prepared mid-log-phase cells were <br /> acetate method (18).                                                                 harvested centrifugation, washed ice-cold buffer (0.9% NaCl, 1 mM <br />    PCR performed Taq DNA polymerase (Gibco-BRL), prod-              NaN3, 10 mM EDTA, 50 mM NaF), frozen liquid nitrogen. Zirconium <br /> ucts purified Sepharose CL6B spin columns (Pharmacia) prior use.          beads added ice-cold breaking buffer (50 mM Tris-Cl [pH 7.5], 15 mM <br /> The yeast genomic DNA library prepared 2&#226;?&#174;m-based plasmid containing             MgCl2, 150 mM NaCl, 1% NP-40, 1 mM EDTA, 1 mM dithiothreitol) supple- <br /> the URA3 marker (pFL44) (7) gift F. Lacroute. Two clones isolated      mented complete protease inhibitor cocktail (Boehringer Mannheim) plus <br /> that able complement growth defect spb10 mutants 37&#194;&#176;C.         0.1 mM phenylmethylsulfonyl fluoride, cells lysed vortexing 4&#194;&#176;C <br /> Plasmids extracted, analyzed, reintroduced different alleles    5 min multimixer. Extracts clarified times centrifugation <br /> spb10. Both plasmids able complement thermosensitive phenotype        4&#194;&#176;C 10 min 16,000 &#226;&#171;&#187; g. <br /> the alleles. The smaller plasmids (pFL44-A1) ana-            Polysome analysis performed described previously (24), that <br /> lyzed, sequence extremities genomic fragment           extracts prepared 20 mM HEPES (pH 7.4)&#226;&#8364;&#8220;2 mM magnesium acetate&#226;&#8364;&#8220; <br /> determined. To destroy PAT1 gene complementing plasmid, frame-          100 mM potassium acetate&#226;&#8364;&#8220;0.5 mM dithiothreitol&#226;&#8364;&#8220;100 &#226;?&#174;g cycloheximide per <br /> shift introduced digesting plasmid pFL44-A1 restriction enzyme       ml. The cells broken alternate vortexing speed 30 s cooling <br /> EcoNI, filling ends Klenow enzyme, religating plasmid       ice 30 s total 6 min. <br /> pFL44-A1K. To construct CEN-based plasmid pFL36-A1, BamHI-SphI                    Protein analysis, Western blotting, coimmunoprecipitation. Sodium do- <br /> fragment containing genomic insert prepared pFL44-A1                decyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) performed as <br /> cloned pFL36 (7).                                                               described earlier (25). Gels electroblotted nitrocellulose membrane <br />    Disruption PAT1 performed using PCR-based strategy           (Protran; Schleicher &amp; Schuell) semidry blotter (Owl Scientific, Woburn, <br /> diploid strain BMA64 (3) oligonucleotides &#226;&#338;&#172;pat1-U (5&#226;&#172;&#732;-AAGGAAGC              Mass.) 2 h 1.5 mA/cm2 30 mM glycine&#226;&#8364;&#8220;48 mM Tris <span id='am-163' about='xsp:base' typeof='owl:Thing'>base</span>&#226;&#8364;&#8220;0.037% <br /> AAAGGTTTTAACCGGAAGTAAGAGCAGCAAGAAGCACTAGCACTG                                        SDS&#226;&#8364;&#8220;20% ethanol. The membranes saturated 25 mM Tris-Cl (pH 8)&#226;&#8364;&#8220;2.6 <br /> ATGCGGTATTTTCTCCT-3&#226;&#172;&#732;) &#226;&#338;&#172;pat1-L (5&#226;&#172;&#732;-GGGAGAAAAAAAAATAC                              mM KCl&#226;&#8364;&#8220;13.8 mM NaCl&#226;&#8364;&#8220;0.05% Tween&#226;&#8364;&#8220;3% nonfat milk. Antibodies incu- <br /> ATGCGTAAGTACATTAAAATTACAGGAAAAATCCGGGTGTTGGCG                                        bated medium 1 h room temperature. Primary antibodies were <br /> GGTGTC-3&#226;&#172;&#732;) amplify TRP1 cassette. Disruption confirmed Southern          added following dilutions: anti-Pat1p (33), 1/1,000; anti-Pab1p (2), <br /> hybridization. Sporulation Trp&#226;&#171;&#185; diploid cells led viable spores   1/10,000; rabbit anti-Spb8p (this work), 1/750; anti-Qsr1p (47), 1/1,000; and <br /> each tetrad analyzed, slow-growing fast-growing spores. The         anti-Flag (Kodak), 1/1,000. Anti-mouse anti-rabbit antibodies coupled to <br /> two slow-growing spores failed grow 37&#194;&#176;C, TRP1 marker segregated       horseradish peroxidase obtained Sigma. Spb8p immunoprecipi- <br /> with slow-growing thermosensitive phenotype successive            tated steps. First, 2-mg samples native protein extracts incubated <br /> backcrosses. Tagging Pat1p immunoglobulin G-binding domains           ice 1 h 4 &#226;?&#174;l preimmune rabbit serum presence 5 mg of <br /> Staphylococcus aureus protein A (Pat1-protAp) performed PCR-based           bovine serum albumin (BSA) ml. Then 20 &#226;?&#174;l 50% slurry protein <br /> strategy oligonucleotides PAT1-A-U (5&#226;&#172;&#732;-TAAACGTTATGGGGTTGGTG                     A-Sepharose (Pharmacia) added, mixture incubated 1 h at <br /> TATCGCGATGGTGAAATATCAGAACTAAAGAAGCTGGAGCTCAAA                                        4&#194;&#176;C wheel. Supernatant recovered centrifugation, treated 4 &#226;?&#174;l <br /> AC-3&#226;&#172;&#732;) PAT1-A-L (5&#226;&#172;&#732;-AGAAAAAAAAATACATGCGTAAGTACATTA                               anti-Spb8p, incubated described above. Beads recovered by <br /> AAATTACAGGAAAAATCTTATACGACTCACTATAGGG-3&#226;&#172;&#732;) plasmid                                centrifugation 1 min 400 &#226;&#171;&#187; g washed times 0.3 ml breaking <br /> pBS1173 template (32). A pep4&#226;&#338;&#172;::URA3 mutation introduced genetic         buffer. To analyze proteins, Laemmli loading buffer added the <br /> crossing reduce degradation tagged protein native extracts.         samples denatured 90&#194;&#176;C 10 min. To precipitate Pat1-protAp, native <br />    To disrupt SPB8, plasmid constructed contains LEU2 gene            extracts incubated 20 &#226;?&#174;l 50% slurry immunoglobulin G- <br /> flanked 5&#226;&#172;&#732; (450 bp) 3&#226;&#172;&#732; (245 bp) region SPB8 gene. These    Sepharose beads (Pharmacia) 2 h 4&#194;&#176;C. Then beads washed and <br /> regions amplified PCR oligonucleotides oAS319 (6) plus oRB44            processed described above. <br /> (5&#226;&#172;&#732;-CGGGATCCCATATGTTTTGGTGAATTAATTCGATTCG-3&#226;&#172;&#732;) and <br /> oRB45 (5&#226;&#172;&#732;-CGGGATCCTAAGAATTCGAAAGAAAAACACAATACTAC-                                                                      RESULTS <br /> 3&#226;&#172;&#732;) plus oAS320 (6), respectively, cloned SalI-EcoRI sites pBlue- <br /> script-SK (Stratagene), creating plasmid pRB35. The LEU2 gene amplified             Identification spb10, bypass suppressor pab1&#226;&#338;&#172;. We <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> YIplac128 (19) oligonucleotides oRB49 (5&#226;&#172;&#732;-GGAATTCCATATGA <br /> ATAGGCGTATCACCAGCGG-3&#226;&#172;&#732;) oRB50 (5&#226;&#172;&#732;-CGGAATTCCACCGAA <br />                                                                                      previously reported isolation new bypass suppressors of <br /> ACGCGCGAGACGAAAGGG-3&#226;&#172;&#732;) cloned NdeI-EcoRI fragment                      deletion PAB1 gene (spb) using transposon mu- <br /> pRB35 yield pRB36. A SalI-XbaI fragment containing LEU2 gene flanked          tagenesis strategy (6). However, genetic analysis numer- <br /> by upstream downstream regions SPB8 prepared                ous spb mutations isolated revealed them <br /> plasmid used transform W303 haploid strain, gene not <br /> essential cell growth (6). To produce recombinant Spb8 protein, SPB8 <br />                                                                                      linked insertion transposon. Three of <br /> open reading frame (ORF) amplified PCR yeast genomic DNA            spontaneous spb mutations thermosen- <br /> oRB46 (5&#226;&#172;&#732;-CGGGATCCTCTGCAAATAGCAAGGACA-3&#226;&#172;&#732; oRB47 (5&#226;&#172;&#732;-                              sitive characterized. After genetic anal- <br />  VOL. 20, 2000                                                            THE Pat1p-Spb8p COMPLEX IS REQUIRED FOR mRNA DECAY                                       5941 <br />  <br />  <br />                                                                                        37&#194;&#176;C, Trp&#226;&#171;&#185; (pat1&#226;&#338;&#172;::TRP1) unable grow at <br />                                                                                        nonpermissive temperature, expected. The pat1&#226;&#338;&#172;::TRP1 <br />                                                                                        clones able form colonies 5-FOA-containing <br />                                                                                        plates, demonstrating ability lose plasmid contain- <br />                                                                                        ing PAB1 gene (Fig. 1B, row). Finally, spb10-1/ <br />                                                                                        pat1&#226;&#338;&#172; diploid strain constructed genetic crossing and <br />                                                                                        thermosensitive. This strain sporulated and <br />                                                                                        dissected, progeny spores tetrad were <br />                                                                                        thermosensitive (data shown). This result <br />                                                                                        demonstrates pat1&#226;&#338;&#172;::TRP1 allelic spb10-1. We con- <br />                                                                                        clude experiments PAT1 wild-type gene <br />                                                                                        corresponding spb10 mutants deletion leads <br />                                                                                        thermosensitivity spb phenotype. <br />                                                                                           As previously reported, search typical patterns Pat1p <br />                                                                                        reveals presence single EF hand, calcium- <br />    FIG. 1. Characterization spb10 mutation cloning PAT1              binding motif, residues 648 660 (33). Another <br /> gene. (A) Complementation testing thermosensitivity spb10-1 allele <br /> with various plasmids. Yeast cultures spotted serial 10-fold dilution     characteristic Pat1p presence region enriched in <br /> yeast-peptone-dextrose (YPD) plates grown 3 days permis-         proline histidine residues 114 200. Such re- <br /> sive (25&#194;&#176;C, right panel) restrictive (37&#194;&#176;C, left panel) temperature. First   gions present numerous proteins databases, al- <br /> row, spb10-1 mutant strain; second row, strain transformed the <br /> pFL44-A1 plasmid contains PAT1 gene; row, spb10-1 strain <br />                                                                                        function assigned elements. <br /> transformed pFL44-A1K plasmid contains mutated PAT1 gene.            Searching current databases potential homologues of <br /> (B) Deletion PAT1 leads thermosensitivity spb phenotype. A             Pat1p S. cerevisiae revealed existence ORF from <br /> diploid strain, pat1&#226;&#338;&#172;::TRP1/PAT1 pab1&#226;&#338;&#172;::HIS3/PAB1 (pAS77 [pPAB1 URA3                   Schizosaccharomyces pombe encode 744-residue <br /> CEN]), constructed genetic crossing sporulated, tetrads were <br /> dissected analyzed. His&#226;&#171;&#185; spores selected, growth tested <br />                                                                                        protein, compared 797-residue Pat1p S. cerevisiae. <br /> under different conditions indicated.                                               The proteins 25% similarity N and <br />                                                                                        C termini. However, similarity reaches 45% when <br />                                                                                        central domain 440 residues considered. Five motifs <br /> ysis, mutations recessive allelic                       particularly conserved yeast <br /> for thermosensitive phenotype spb phenotype                            proteins distinguished (Fig. 2). <br /> were named spb10-1, spb10-2, spb10-3 (Fig. 1A, row,                             Deletion PAT1 does affect ribosome biogenesis but <br /> and data shown). A yeast genomic DNA fragment borne                                stabilizes MFA2 mRNA. The majority bypass suppres- <br /> on URA3 2&#226;?&#174;m-based multicopy plasmid isolated                            sors pab1&#226;&#338;&#172; impaired 60S ribosomal subunit synthesis <br /> able complement growth defect spb10 mutants                           (31, 35, 36, 51). To determine Pat1p involved in <br /> 37&#194;&#176;C (Fig. 1A, second row). The sequence insert reveals                         ribosome biogenesis, monitored accumulation pre- <br /> that contains characterized gene, PAT1, trun-                           rRNA processing intermediates spb10 mutant strain. <br /> cated ORFs. Inactivation PAT1 introducing frameshift                           Northern blot analysis performed oligonucleotide <br /> mutation (pFL44-A1K) results loss complemen-                             probes complementary various spacer sequences pre- <br /> tation activity (Fig. 1A, row), providing strong support                         rRNA. No difference observed pat1&#226;&#338;&#172; strain compared <br /> that PAT1 responsible complementation spb10                          isogenic wild-type strain (data shown). As different <br /> mutation. When transferred CEN-based vector (pFL36-                               way study ribosome biogenesis, ribosome subunit profiles <br /> A1), PAT1 able support growth spb10-1 <br />                                                                                        examined pat1&#226;&#338;&#172; strain similar to <br /> strain 37&#194;&#176;C suppress spb phenotype (data not <br />                                                                                        wild-type strain (data shown). These observa- <br /> shown). This result demonstrates PAT1 been <br /> isolated multicopy suppressor spb10 capable of <br /> complementing spb10 mutation. <br />    pat1&#226;&#338;&#172; allelic spb10-1 bypass suppresses pab1&#226;&#338;&#172;. <br /> PAT1 previously identified yeast genome- <br /> sequencing project (33). Antibodies raised N-termi- <br /> nal peptide corresponding protein (anti-Ycr77c) de- <br /> tected cytosolic protein (Mr 97,000). A partial deletion this <br /> ORF led slightly reduced growth rate 37&#194;&#176;C. Later, the <br /> same gene identified two-hybrid screen proteins <br /> interacting topoisomerase II named PAT1 (for &#226;&#8364;&#339;pro- <br /> tein associated topoisomerase II&#226;&#8364;?) (49). In study, the <br /> authors deletion half gene was <br /> associated thermosensitive phenotype. We decided to <br /> replace entire PAT1 gene TRP1 marker using a <br /> PCR-based strategy (3) deletion PAT1 leads <br /> to slow-growth thermosensitive phenotype. Then, to <br /> determine pat1&#226;&#338;&#172; mutation able bypass                               FIG. 2. Sequence comparison Pat1p budding fission <br /> PAB1 gene, diploid strain heterozygous mutations                         yeasts. The sequence Pat1p S. cerevisiae (Sc) translated ORF from <br /> pat1&#226;&#338;&#172;::TRP1 pab1&#226;&#338;&#172;::HIS3 containing pAS77 plasmid                               S. pombe (Sp) (accession number CAA17064) aligned (Clustal W [46]). <br /> (pPAB1 URA3 CEN) (37) constructed standard genetic                              Five regions higher similarity chosen named motifs I V <br />                                                                                        (shown Roman numerals right). Residues identical or <br /> crossing. After sporulation tetrad dissection, growth                       chemically equivalent boxed. Numbers refer Pat1p sequence <br /> the His&#226;&#171;&#185; spores (i.e., PAB1 deleted) tested various                        S. cerevisiae. The dash represents gap inserted motif I optimize the <br /> media (Fig. 1B). The Trp&#226;&#171;&#186; clones (PAT1) able grow                           alignment. <br />  5942       BONNEROT ET AL.                                                                                                                   MOL. CELL. BIOL. <br />  <br />  <br />                                                                                  hypothesis, plasmid containing wild-type PAT1 gene <br />                                                                                  (pFL44-A1) transformed mrt1-3 strain. This plas- <br />                                                                                  mid able support growth mutant strain high <br />                                                                                  temperature, demonstrating PAT1 complement the <br />                                                                                  thermosensitivity mrt1-3 mutation (Fig. 4A). To confirm <br />                                                                                  result, protein extracts prepared various mutant <br />                                                                                  strains analyzed Western blotting antibody that <br />                                                                                  recognizes Pat1 protein (33). As expected, Pat1p not <br />                                                                                  detected haploid pat1&#226;&#338;&#172; strain (Fig. 4B, lane 2) in <br />                                                                                  homozygous pat1&#226;&#338;&#172;/pat1&#226;&#338;&#172; diploid strain (lane 6). We then <br />                                                                                  examined mrt1-1, mrt1-2, mrt1-3 haploid strains and <br />                                                                                  mrt1-3 pat1&#226;&#338;&#172; double-mutant diploid strain that <br />                                                                                  contain detectable Pat1p (Fig. 4B, lanes 3 and <br />                                                                                  7, data shown). In contrast, mrt3-1 pat1&#226;&#338;&#172; double- <br />                                                                                  mutant diploid strain possesses normal levels Pat1p (lane 8). <br />                                                                                  Taken together, data strongly suggest PAT1 iden- <br />                                                                                  tical MRT1. <br />                                                                                     Pat1p Spb8p associated vivo shown coim- <br />                                                                                  munoprecipitation experiments. Mutations affecting PAT1 <br />                                                                                  gene lead phenotype reminiscent spb8-2 <br />                                                                                  mutant strain (6), suggesting Pat1p Spb8p be <br />                                                                                  involved pathway. To investigate possibility, <br />                                                                                  decided search potential interactions between <br />                                                                                  Pat1p Spb8p. A strain expressing Pat1-protA fusion <br />                                                                                  protein normal locus constructed (see Materials and <br />                                                                                  Methods). The growth rate strain identical that <br />                                                                                  isogenic untagged strain, level Pat1-protAp <br />    FIG. 3. MFA2pG mRNA stabilized pat1&#226;&#338;&#172; strain. (A B) Deg- <br />                                                                                  similar level natural protein wild-type <br /> radation MFA2pG mRNA wild-type (WT) (A) pat1&#226;&#338;&#172; (B) strains. Cells       strain (data shown). <br /> carrying GAL::MFA2pG reporter gene shifted galactose- glu-         To detect Spb8p, polyclonal antibody prepared by <br /> cose-containing medium repress transcription. RNA recovered sam-     injecting rabbits recombinant Spb8p. The specificity of <br /> ples taken indicated times repression analyzed Northern <br /> blotting. The panel shows hybridization oligonucleotide (oRP140) <br />                                                                                  antibody tested Western blotting, extract <br /> recognizing full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> MFA2pG mRNA pG degradation interme-          wild-type yeast strain (Fig. 5A, lane 1). A unique band <br /> diate. The panel shows hybridization signal scR1 probe,      detected migrated Mr 20,000 polypeptide, size <br /> which used internal control. (C) The level full-length MFA2pG   good agreement predicted size Spb8p. The spec- <br /> mRNA time point quantitated PhosphorImager (Molecular <br /> Dynamics) analysis, normalized scR1, plotted log scale a <br />                                                                                  ificity reaction illustrated absence signal <br /> function time.                                                                extract prepared strain SPB8 <br />                                                                                  gene deleted (see Materials Methods) (lane 2). <br />  <br /> tions led conclusion PAT1 required for <br /> ribosome biogenesis. <br />    A second group bypass suppressors pab1&#226;&#338;&#172;, affect <br /> mRNA decay stabilizing cap structure, iden- <br /> tified (6, 21). To determine absence Pat1p <br /> affects mRNA decay, measured half-life reporter <br /> MFA2pG mRNA wild-type pat1&#226;&#338;&#172; strains. Insertion a <br /> poly(G) tract 3&#226;&#172;&#732; untranslated region MFA2 mRNA <br /> leads formation stable secondary structure that <br /> inhibits exonucleases, allowing detection intermedi- <br /> ate degradation product (called pG hereafter) (13). In these <br /> experiments, MFA2pG reporter gene con- <br /> trol GAL1 promoter, allows rapid transcrip- <br /> tional repression. Northern blot analysis shows MFA2- <br /> pG mRNA rapid turnover wild-type strain a <br /> half-life 3.5 min, half-life increased about <br /> fourfold pat1&#226;&#338;&#172; strain (Fig. 3). In addition, pG frag- <br /> ment barely detected pat1&#226;&#338;&#172; mutant. These obser-                          FIG. 4. PAT1 identical MRT1. (A) A plasmid containing wild-type <br /> vations demonstrate that, like DCP1, MRT1, MRT3, SPB8,                       (WT) PAT1 gene support growth mrt1-3 strain 37&#194;&#176;C. Tenfold <br /> the PAT1 gene product required normal rates mRNA                       serial dilutions indicated strains spotted yeast minimal medium <br />                                                                                  plates lacking uracil incubated 25 37&#194;&#176;C shown bottom. <br /> turnover.                                                                        WT, W303 derivative containing plasmid pRP485; &#226;&#171;&#185;pFL44, yRP1066 (18) <br />    PAT1 identical MRT1. The pat1&#226;&#338;&#172; mutant strain                        transformed pFL44 vector; &#226;&#171;&#185;pFL44-A1, strain trans- <br /> certain similarities mrt1-3 mutant strain: unstable                       formed plasmid containing PAT1 gene. (B) Pat1p detected in <br /> mRNAs stabilized fourfold mutants,                      mrt1-3 haploid strain diploid pat1&#226;&#338;&#172; mrt1-3 double-mutant strain. <br />                                                                                  Protein extracts (50 &#226;?&#174;g) fractionated SDS-PAGE (10% polyacrylamide), <br /> two mutations bypass deletion PAB1 gene,                    electroblotted nitrocellulose, probed antibodies raised against <br /> two mutant strains thermosensitive. These similarities sug-                  Pat1p. Lanes: 1 3, haploid strains; 4 8, diploid strains. Relevant genotypes <br /> gested MRT1 identical PAT1. To test                        indicated lane. Mr, molecular weight markers thousands. <br />  VOL. 20, 2000                                                         THE Pat1p-Spb8p COMPLEX IS REQUIRED FOR mRNA DECAY                                       5943 <br />  <br />  <br />                                                                                       Pat1p Spb8p cofractionate polysomes. Several <br />                                                                                    lines evidence led proposal mRNA degra- <br />                                                                                    dation mRNA translation interlaced processes in <br />                                                                                    cell (23). Recently, view received support, <br />                                                                                    finding 5&#226;&#172;&#732;-to-3&#226;&#172;&#732; exoribonuclease Xrn1p, which <br />                                                                                    major component mRNA degradation pathway, was <br />                                                                                    associated polysomes (28). Pab1p associated with <br />                                                                                    polysomes plays role mRNA stability. Since spb10 and <br />                                                                                    spb8 suppressors pab1&#226;&#338;&#172;, examined Pat1p <br />                                                                                    Spb8p associated polysomes. These exper- <br />                                                                                    iments performed strain Dcp1p was <br />                                                                                    tagged Flag epitope allows detection (27). <br />                                                                                       Polysomes fractionated sucrose gradients, the <br />                                                                                    fractions analyzed Western blotting antibodies <br />                                                                                    raised Pat1p, Spb8p, Pab1p, Flag-Dcp1p, Xrn1p. <br />                                                                                    Qsr1p, 60S ribosomal subunit protein, used control <br />                                                                                    integrity polysomes (47). Expression Flag- <br />  <br />  <br />    FIG. 5. The absence Spb8p leads sharp decrease level Pat1p. <br /> (A) Protein extracts (50 &#226;?&#174;g) prepared indicated strains tested by <br /> Western blotting rabbit anti-Spb8p antibody. (B) Detection Pat1p in <br /> various mutant strains. Protein extract (50 &#226;?&#174;g) prepared strain (5 &#226;?&#174;g <br /> for lane 3) subjected SDS-PAGE (10% polyacrylamide). The filters were <br /> probed anti-Pat1p antibody. Relevant genotypes indicated above <br /> each lane. Mr, molecular weight markers thousands. WT, wild type. <br />  <br />  <br />  <br /> To assess level protein tested, amounts of <br /> protein loaded lane, confirmed staining the <br /> membrane amido black probing membrane <br /> with control antibody (anti-Pab1p [data shown]). The <br /> level Spb8p unchanged strain PAT1 <br /> gene deleted (lane 3). We tested presence of <br /> Pat1p various mutant strains. The level Pat1p unaf- <br /> fected dcp1&#226;&#338;&#172;, xrn1&#226;&#338;&#172;, mrt3-1 strains (Fig. 5B, lanes 4 6). <br /> In contrast, level Pat1p reduced 10-fold the <br /> spb8&#226;&#338;&#172; strain (lanes 1 3). This result suggests Pat1p and <br /> Spb8p interact, physically genetically. We con- <br /> structed series double-mutant strains (spb8&#226;&#338;&#172; pat1&#226;&#338;&#172;, dcp1&#226;&#338;&#172; <br /> pat1&#226;&#338;&#172;, xrn1&#226;&#338;&#172; pat1&#226;&#338;&#172;) able <br /> to grow permissive temperature, demonstrating spb8&#226;&#338;&#172;, <br /> dcp1&#226;&#338;&#172;, xrn1&#226;&#338;&#172; synthetically lethal pat1&#226;&#338;&#172;. <br />    To test Pat1p interacts physically Spb8p, ex- <br /> tracts containing Pat1-protAp incubated immuno- <br /> globulin G-Sepharose beads precipitate protA-tagged <br /> protein associated proteins. Western blot analysis was <br /> performed rabbit immunoglobulins coupled horserad-                            FIG. 6. Spb8p Pat1p coimmunoprecipitated. (A) Immunoprecipita- <br />                                                                                    tion Pat1-protAp. Native extracts prepared wild-type strain or <br /> ish peroxidase reveal Pat1-protAp (Fig. 6A, upper panel)                        PAT1-protA tagged strain incubated immunoglobulin G-Sepha- <br /> and anti-Spb8p antibody reveal Spb8p (lower pan-                       rose beads. After extensive washes, proteins eluted beads and <br /> el). Spb8p detected total-cell extracts (lanes 1 2)                     separated SDS-PAGE (10% polyacrylamide). Western blotting per- <br /> and pellet fraction immunoprecipitation per-                       formed rabbit immunoglobulin G coupled horseradish peroxidase <br />                                                                                    detect protA-tagged protein (upper panel) anti-Spb8p antibody <br /> formed strain expressing Pat1-protAp (lanes 7 8).                     (lower panel). Lanes: 1, 3, 6, wild-type strain (tag &#226;&#171;&#186;); 2, 4, 5, 7, 8, <br /> No Spb8p detected experiment per-                            PAT1-protA strain (tag &#226;&#171;&#185;). Lanes 1 5 contain 1/40 protein <br /> formed strain expressing untagged Pat1p (lane                      extract used immunoprecipitation experiment. T, total extract; S, super- <br /> 6). Interaction Spb8p Pat1p unaffected adding                        natant; P, pellet. Lanes 5 8 contain supernatant pellet, respectively, <br />                                                                                    immunoprecipitation, washed buffer containing 0.5 M NaCl. (B) <br /> RNase A immunoprecipitation procedure (data                         Immunoprecipitation Spb8p. Native extracts prepared strain yRP1070 <br /> shown) washing pellet buffer containing 0.5 M                     transformed pRP801 (see Materials Methods) treated a <br /> NaCl (lane 8). The reverse experiment carried                      preimmune serum incubated protein A-Sepharose remove non- <br /> first precipitating Spb8p anti-Spb8p antibody, using                    specific complexes. Then supernatants treated anti-Spb8p <br />                                                                                    antibody left untreated; followed precipitation protein A- <br /> a strain Pat1p tagged. Western blot analyses                      Sepharose. Proteins pellets supernatants analyzed by <br /> of pellet anti-Pat1p antibody revealed Pat1p                     Western blotting anti-Pat1p (upper panel) anti-Spb8p <br /> was coimmunoprecipitated Spb8p, confirming physi-                         (lower panel) antibodies. Lane 1 contains 1/40 protein used the <br />                                                                                    immunoprecipitation (T); lanes 2 3 contain pellets immunoprecipita- <br /> cal association proteins vivo (Fig. 6B, lane 3). No                  tion (P); lanes 4 5 contain 1/40 supernatants (S). Lanes 2 4, <br /> Pat1p precipitated control experiment anti-                        control immunoprecipitation performed serum (&#226;&#171;&#186;); lanes 3 5, im- <br /> bodies omitted (lane 2).                                                      munoprecipitation presence anti-Spb8p antibodies (&#226;&#171;&#185;). <br />  5944       BONNEROT ET AL.                                                                                                                       MOL. CELL. BIOL. <br />  <br />  <br />                                                                                        Since Pat1p interacts physically Spb8p, searched <br />                                                                                     presence proteins fractions <br />                                                                                     containing polysomes depend presence the <br />                                                                                     other. Cellular extracts prepared different mutant <br />                                                                                     strains fractionated sucrose gradients. The fractions <br />                                                                                     analyzed Western blotting described above. In <br />                                                                                     pat1&#226;&#338;&#172; strain, Spb8p detected the <br />                                                                                     gradient (Fig. 7B). In contrast, spb8&#226;&#338;&#172; strain, level of <br />                                                                                     Pat1p sharply reduced (Fig. 5B) rest Pat1p was <br />                                                                                     detected fractions containing polysomes (Fig. 7C). <br />                                                                                        To assess cofractionation Spb8p Pat1p with <br />                                                                                     polysomes fortuitous physical interactions of <br />                                                                                     proteins polysomes, performed sucrose gradient <br />                                                                                     analysis conditions disrupt free monosomes <br />                                                                                     polysome structure. We knew presence <br />                                                                                     proteins fractions containing high-molecular- <br />                                                                                     weight structures labile, shifted to <br />                                                                                     gradient polysomes prepared and <br />                                                                                     analyzed standard methods (reference 16 data not <br />                                                                                     shown). Therefore, gradients run high-salt con- <br />                                                                                     taining buffer (28), Spb8p Pat1p detected the <br />                                                                                     upper gradient, expected (data shown). <br />                                                                                     We assayed association factors pres- <br />                                                                                     ence EDTA, disrupts polysomes leads ac- <br />                                                                                     cumulation free 40S 60S subunits. This shift ribo- <br />                                                                                     somal subunits accompanied shift sedimentation <br />                                                                                     <span id='am-164' about='xsp:pattern' typeof='owl:Thing'>pattern</span> Spb8p Pat1p (Fig. 8). This result suggests that <br />                                                                                     proteins associated translated mRNAs. <br />  <br />                                                                                                                    DISCUSSION <br />  <br />                                                                                       We report characterization spb10, suppressor <br />                                                                                     lethality pab1&#226;&#338;&#172; lies PAT1 gene. Se- <br />                                                                                     quence alignment Pat1p S. cerevisiae putative <br />                                                                                     homologue S. pombe reveals motifs con- <br />                                                                                     served putative proteins metazoans. It worth <br />    FIG. 7. Spb8p Pat1p cosediment polysomes sucrose gradients.          noticing single EF-hand motif detected the <br /> (A) A total-cell extract prepared strain yRP1070 transformed pRP801, <br /> which expresses tagged version Dcp1p, fractionated 7 50% linear <br /> sucrose gradient. (Top) A continuous record absorbance 254 nm <br /> (A254nm) gradient presented, gradient left. <br /> The arrows indicate peaks 40S, 60S, 80S subunits. (Bottom) <br /> Western blot analysis 15 fractions collected sucrose gradient was <br /> performed antibodies raised indicated proteins. T, total-cell <br /> extract. (B) A total-cell extract prepared strain PAT1 deleted <br /> and fractionated sucrose gradient described panel A. The 15 fractions <br /> were analyzed Western blotting anti-Qsr1p (upper row) the <br /> anti-Spb8p (bottom row) antibody. (C) As panel B, extracts were <br /> prepared strain SPB8 deleted tested anti-Pat1p (upper <br /> row) anti-Qsr1p (bottom row) antibody. <br />  <br />  <br />  <br /> Dcp1p chimera does alter polysome profile, is <br /> similar wild type (Fig. 7A). Pab1p associated <br /> with polysomes lighter fractions corresponding to <br /> the 60S, 40S, lighter structures. As expected, Western <br /> blot analysis anti-Qsr1p antibody detected protein <br /> in fractions containing 60S subunits fractions con- <br /> taining polysomes. Under experimental conditions used <br /> here, Flag-Dcp1p located exclusively upper of <br /> the gradient (see Discussion). The antibodies raised against <br /> Spb8p detect protein gradient, the <br /> case Pab1p. In contrast, Pat1p detected frac- <br /> tions containing mRNPs, monosomes, polysomes                               FIG. 8. Addition EDTA results shift Spb8p Pat1p upper <br /> with lightest fractions. Therefore, large frac-                   sucrose gradient. Extracts prepared analyzed described <br /> tion Spb8p associated Pat1p, distributions                       legend Fig. 7, 40 mM EDTA added breaking <br />                                                                                     buffer sucrose gradient solutions. (Top) Relative profile ab- <br /> the proteins slightly different. Interestingly, Xrn1p                      sorbance 254 nm (A254nm) gradient left. Arrows <br /> has distribution Spb8p through-                        indicate positions 40S 60S subunits. (Bottom) Western blot <br /> out gradient (reference 28 data shown).                           analysis fractions using anti-Pat1p anti-Spb8p antibodies. <br />  VOL. 20, 2000                                            THE Pat1p-Spb8p COMPLEX IS REQUIRED FOR mRNA DECAY                             5945 <br />  <br />  <br /> S. cerevisiae sequence conserved S. pombe se-         metabolic pathway. Coimmunoprecipitation experiments indi- <br /> quence, weakening potential role. Interestingly, pu-     cate interact vivo. This association <br /> tative homologue Xenopus laevis, P100 protein,       does depend presence RNA, not <br /> has localized exclusively cytoplasm oocyte,   sensitive RNase treatment. Moreover, proteins <br /> able bind single-stranded DNA vitro (see below) (34).         strong affinity other, coimmunoprecipi- <br />    Pat1p previously interact topoisomerase II     tation affected high-salt washes. In contrast, their <br /> (49). A strain PAT1 partially deleted exhibits alterations     presence fractions containing polysomes labile <br /> of chromosome segregation rDNA recombination (48, 49).          (data shown), suggesting intrinsic constit- <br /> Our data previous workers support role         uents polysomes. However, Spb8p detected in <br /> for protein. The report Ycr77c shown     upper gradient, fractions Pat1p is <br /> protein associated cytosolic fraction (33),      absent present small amounts. This observa- <br /> and Pat1p cosediments sucrose gradients            tion suggests Spb8p likely involved different <br /> with polysomes, observations favor idea Pat1p     structures, contain Pat1p not. <br /> is located cytoplasm. We exclude      Functional interactions Spb8p Pat1p further <br /> a subset protein localized nucleus         supported observation level Pat1p sharply <br /> interacts Top2p. Alternatively, Pat1p as-       reduced absence Spb8p. Moreover, absence of <br /> sociated subset Top2p cytoplasm,          Pat1p, Spb8p detected upper gra- <br /> could regulate function. It conceivable chromo-    dient. This suggests Pat1p bind poly- <br /> some segregation alterations seen pat1&#226;&#338;&#172; strain indirect    somes establish link Spb8p Spb8p bind <br /> consequences inhibition mRNA decay,              directly polysomes require Pat1p interact <br /> preferentially affect certain mRNAs changed expression        steadily. It worth noticing protein X. laevis <br /> would lead phenotype.                                       presents certain degree similarity Pat1p. This pro- <br />    We PAT1 identical MRT1,          tein, P100, shown bind preferentially vitro single- <br /> mutations reported affect mRNA decay stabi-          stranded DNA-cellulose double-stranded DNA-cellulose <br /> lizing cap structure (21). The mrt1 mutations isolated     (34). However, protein localized cytoplasm <br /> from collection temperature-sensitive strains stabilize   oocyte, result opens possibility P100 in <br /> a reporter MFA2pG mRNA. Other mutations isolated            fact RNA-binding protein. Following view, con- <br /> XRN1, DCP1, MRT3 genes, mutants            ceivable function Pat1p bind mRNAs, <br /> actually thermosensitive se. In contrast, decapping         bringing polysomes complex containing Spb8p. <br /> defect associated mrt1 mutations linked            Spb8p possesses Sm-like motif, signature of <br /> thermosensitive phenotype. This result agreement         group proteins involved snRNA metabolism (40). Two <br /> our observations pat1&#226;&#338;&#172; mutant thermosensitive       recent reports demonstrated Spb8p (or Lsm1p) is <br /> leads mRNA stabilization. The mrt1 mutations            associated group Lsm proteins (Lsm2 to <br /> been directly linked PAT1 gene sequencing DNA         Lsm7) (8, 44) likely form doughnut-shaped struc- <br /> corresponding PAT1 ORF mrt1 mutants (44).       ture (1). Mutations LSM genes alter <br />    Decapping activated distinct mechanisms. In        mRNA decapping. Two-hybrid analyses shown Lsm <br /> the general mRNA decay pathway, decapping activated af-          proteins interact Dcp1p, Xrn1p, Dcp2p, Pat1p, num- <br /> ter deadenylation mRNA. Mutations MRT1             ber uncharacterized ORF products (M. Fromont- <br /> (PAT1), SPB8, MRT3 prevent deadenylation-dependent          Racine P. Legrain, personal communication). Pat1p was <br /> decapping activity vivo (6, 21). Extensive mutagenesis    interact physically Xrn1p (8) Dcp1p <br /> DCP1 gene revealed mutations abolish deadeny-          (44). Our data indicate Spb8p-Pat1p complex cosedi- <br /> lation-dependent decapping activity vivo changing        ments polysomes, like exonuclease Xrn1p. Using our <br /> the activity vitro (45). These results suggest dead-    experimental conditions, detected Dcp1p of <br /> enylation-dependent decapping activity vivo requires par-      gradient, suggesting Dcp1p associated <br /> ticular domain Dcp1p interact factors      polysomes. However, experiments used Flag- <br /> such Pat1p, Spb8p, Mrt3p. In contrast, nonsense-      Dcp1p construction overexpressed 50- 100-fold com- <br /> mediated decay (NMD) pathway, decapping occurs prior             pared natural levels (R. Parker, personal communica- <br /> deadenylation does require Pat1p, Spb8p, Mrt3p.          tion). It possible Dcp1p seen of <br /> Also, mutations DCP1 alter deadenylation-dependent          gradient reflects extra Dcp1p functional <br /> decapping affect NMD, suggesting          Dcp1p associates polysomes detection <br /> different ways activate Dcp1p: requires Pat1p, Spb8p,   level experiments. Alternatively, Spb8p-Pat1p com- <br /> or Mrt3p responding activation deadenylation       plex, associated polysomes, await specific <br /> and activated directly NMD pathway.          signal, probably mediated Pab1p, recruit Dcp1p-Dcp2p <br />    Another mutation, vps16, inhibits decapping normal            complex mRNA activate decapping. Future work <br /> mRNAs prevents NMD (50), mutations DCP1           focus characterizing factors involved decap- <br /> or DCP2. Ssa1p Ssa2p, Hsp70 family member, copurifies         ping associated components Dcp2p, Ssa1p, or <br /> with Dcp1p. Interestingly, interaction enhanced mrt1     Ssa2p, factors recently discovered two-hybrid anal- <br /> or vps16 strains, observation explain re-      ysis. This serve basis mechanisms <br /> duced decapping activity mutants (50). Moreover,           control switch mRNA translation degradation. <br /> Sis1p, required translation initiation, belongs to <br /> the DnaJ protein family, members modulate activity                               ACKNOWLEDGMENTS <br /> of Hsp70 proteins. SIS1 genetically linked             We indebted A. Sachs, initiated work. We thank F. <br /> PAB1, establishing link heat shock proteins,        Lacroute gift yeast genomic library, thank R. Lill, <br /> initiation translation, Pab1 protein (51).               M. Swanson, B. Trumpower providing antibodies used this <br />    Our results demonstrate Pat1p Spb8p share       study. We grateful J. Morrissey D. Tollervey comments <br /> properties, supporting view operate       manuscript F. Martin chromatography experi- <br />  5946       BONNEROT ET AL.                                                                                                                            MOL. CELL. BIOL. <br />  <br />  <br /> ments M. Doe`re technical assistance polysome analysis.                        conditional dbp6 alleles identifies Rsa1p, nucleoplasmic protein involved in <br /> We thank L. Pintard M. Brengues help stimulat-                        assembly 60S ribosomal subunits. Mol. Cell. Biol. 19:8633&#226;&#8364;&#8220;8645. <br /> ing discussions, thank E. Schwob, G. Lutfalla, G. Uze&#194;&#180;                 25.   Laemmli, U. K. 1970. Cleavage structural proteins assembly of <br /> generously sharing equipment supplies.                                                  head bacteriophage T4. Nature 227:680&#226;&#8364;&#8220;685. <br />   This work supported Centre National la Recherche                      26.   LaGrandeur, T., R. Parker. 1999. The cis acting sequences responsible <br />                                                                                             differential decay unstable MFA2 stable PGK1 transcripts <br /> Scientifique grants Ligue contre le Cancer from <br />                                                                                             yeast include context translational start codon. RNA 5:420&#226;&#8364;&#8220;433. <br /> the Fondation pour la Recherche Me&#194;&#180;dicale. R.B. supported                    27.   LaGrandeur, T. E., R. Parker. 1998. Isolation characterization of <br /> grant Ernst et Lucie Schmidheiny foundation, C.B.                      Dcp1p, yeast mRNA decapping enzyme. EMBO J. 17:1487&#226;&#8364;&#8220;1496. <br /> of Institut National la Sante&#194;&#180; et la Recherche Me&#194;&#180;dicale.                   28.   Mangus, D. A., A. Jacobson. 1999. Linking mRNA turnover trans- <br />                                                                                             lation: assessing polyribosomal association mRNA decay factors and <br />                                  REFERENCES                                                 degradative intermediates. Methods 17:28&#226;&#8364;&#8220;37. <br />  1. Achsel, T., H. Brahms, B. Kastner, A. Bachi, M. Wilm, R. Luhrmann.            29.   Mayes, A. E., L. Verdone, P. Legrain, J. D. Beggs. 1999. Characterization <br />     1999. A doughnut-shaped heteromer human Sm-like proteins binds                Sm-like proteins yeast association U6 snRNA. EMBO J. <br />     3&#226;&#172;&#732;-end U6 snRNA, facilitating U4/U6 duplex formation vitro.               18:4321&#226;&#8364;&#8220;4331. <br />     EMBO J. 18:5789&#226;&#8364;&#8220;5802.                                                             30.   Muhlrad, D., C. J. Decker, R. Parker. 1995. Turnover mechanisms the <br />  2. Adam, S. A., T. Nakagawa, M. S. Swanson, T. K. Woodruff, G. Dreyfuss.               stable yeast PGK1 mRNA. Mol. Cell. Biol. 15:2145&#226;&#8364;&#8220;2156. <br />     1986. mRNA polyadenylate-binding protein: gene isolation sequencing           31.   Pintard, L., D. Kressler, B. Lapeyre. 2000. Spb1p yeast nucleolar <br />     identification ribonucleoprotein consensus sequence. Mol. Cell.                protein associated Nop1p Nop58p able bind S-adenosyl- <br />     Biol. 6:2932&#226;&#8364;&#8220;2943.                                                                      L-methionine vitro. Mol. Cell. Biol. 20:1370&#226;&#8364;&#8220;1381. <br />  3. Baudin, A., O. Ozier-Kalogeropoulos, A. Denouel, F. Lacroute, C. Cul-         32.   Puig, O., B. Rutz, B. G. Luukkonen, S. Kandels-Lewis, E. Bragado-Nilsson, <br />     lin. 1993. A simple efficient method direct gene deletion Saccha-            B. Se&#194;&#180;raphin. 1998. New constructs strategies efficient PCR- <br />     romyces cerevisiae. Nucleic Acids Res. 21:3329&#226;&#8364;&#8220;3330.                                    based gene manipulations yeast. Yeast 14:1139&#226;&#8364;&#8220;1146. <br />  4. Beelman, C. A., R. Parker. 1995. Degradation mRNA eukaryotes.           33.   Rodriguez-Cousino, N., R. Lill, W. Neupert, D. A. Court. 1995. Identi- <br />     Cell 81:179&#226;&#8364;&#8220;183.                                                                        fication initial characterization cytosolic protein Ycr77p. Yeast 11: <br />  5. Beelman, C. A., A. Stevens, G. Caponigro, T. E. LaGrandeur, L. Hatfield,                581&#226;&#8364;&#8220;585. <br />     D. M. Fortner, R. Parker. 1996. An essential component decapping       34.   Rother, R. P., M. B. Frank, P. S. Thomas. 1992. Purification, primary <br />     enzyme required normal rates mRNA turnover. Nature 382:642&#226;&#8364;&#8220;646.                  structure, bacterial expression subcellular distribution oocyte- <br />  6. Boeck, R., B. Lapeyre, C. E. Brown, A. B. Sachs. 1998. Capped mRNA                  specific protein Xenopus. Eur. J. Biochem. 206:673&#226;&#8364;&#8220;683. <br />     degradation intermediates accumulate yeast spb8-2 mutant. Mol. Cell.       35.   Sachs, A. B., R. W. Davis. 1989. The poly(A) binding protein required <br />     Biol. 18:5062&#226;&#8364;&#8220;5072.                                                                     poly(A) shortening 60S ribosomal subunit-dependent translation <br />  7. Bonneaud, N., O. Ozier-Kalogeropoulos, G. Y. Li, M. Labouesse, L. Min-                  initiation. Cell 58:857&#226;&#8364;&#8220;867. <br />     vielle-Sebastia, F. Lacroute. 1991. A family low high copy repli-      36.   Sachs, A. B., R. W. Davis. 1990. Translation initiation ribosomal <br />     cative, integrative single-stranded S. cerevisiae/E. coli shuttle vectors.          biogenesis: involvement putative rRNA helicase RPL46. Science <br />     Yeast 7:609&#226;&#8364;&#8220;615.                                                                        247:1077&#226;&#8364;&#8220;1079. <br />  8. Bouveret, E., G. Rigaut, A. Shevchenko, M. Wilm, B. Se&#194;&#180;raphin. 2000. A        37.   Sachs, A. B., R. W. Davis, R. D. Kornberg. 1987. A single domain of <br />     Sm-like protein complex participates mRNA degradation. EMBO J.                  yeast poly(A)-binding protein necessary sufficient RNA binding <br />     19:1661&#226;&#8364;&#8220;1671.                                                                           cell viability. Mol. Cell. Biol. 7:3268&#226;&#8364;&#8220;3276. <br />  9. Caponigro, G., R. Parker. 1995. Multiple functions poly(A)-           38.   Salgado-Garrido, J., E. Bragado-Nilsson, S. Kandels-Lewis, B. Se&#194;&#180;ra- <br />     binding protein mRNA decapping deadenylation yeast. Genes Dev.                phin. 1999. Sm Sm-like proteins assemble related complexes of <br />     9:2421&#226;&#8364;&#8220;2432.                                                                            deep evolutionary origin. EMBO J. 18:3451&#226;&#8364;&#8220;3462. <br /> 10. Church, G. M., W. Gilbert. 1984. Genomic sequencing. Proc. Natl. Acad.        39.   Schwartz, D. C., R. Parker. 1999. Mutations translation initiation <br />     Sci. USA 81:1991&#226;&#8364;&#8220;1995.                                                                  factors lead increased rates deadenylation decapping mRNAs in <br /> 11. Cross, F. R., A. H. Tinkelenberg. 1991. A potential positive feedback               Saccharomyces cerevisiae. Mol. Cell. Biol. 19:5247&#226;&#8364;&#8220;5256. <br />     loop controlling CLN1 CLN2 gene expression start yeast          40.   Se&#194;&#180;raphin, B. 1995. Sm Sm-like proteins belong large family: iden- <br />     cell cycle. Cell 65:875&#226;&#8364;&#8220;883.                                                            tification proteins U6 U1, U2, U4 U5 snRNPs. <br /> 12. Decker, C. J., R. Parker. 1994. Mechanisms mRNA degradation                   EMBO J. 14:2089&#226;&#8364;&#8220;2098. <br />     eukaryotes. Trends Biochem. Sci. 19:336&#226;&#8364;&#8220;340.                                      41.   Tarun, S., A. B. Sachs. 1995. A common function mRNA 5&#226;&#172;&#732; 3&#226;&#172;&#732; <br /> 13. Decker, C. J., R. Parker. 1993. A turnover pathway stable              ends translation initiation yeast. Genes Dev. 9:2997&#226;&#8364;&#8220;3007. <br />     unstable mRNAs yeast: evidence requirement deadenylation.            42.   Tarun, S. Z., Jr., S. E. Wells, J. A. Deardorff, A. B. Sachs. 1997. <br />     Genes Dev. 7:1632&#226;&#8364;&#8220;1643.                                                                 Translation initiation factor eIF4G mediates vitro poly(A) tail-dependent <br /> 14. Dunckley, T., R. Parker. 1999. The Dcp2 protein required mRNA <br />                                                                                             translation. Proc. Natl. Acad. Sci. USA 94:9046&#226;&#8364;&#8220;9051. <br />     decapping Saccharomyces cerevisiae contains functional MutT motif. <br />                                                                                       43.   Tarun, S. Z. J., A. B. Sachs. 1996. Association yeast poly(A) tail <br />     EMBO J. 18:5411&#226;&#8364;&#8220;5422. <br />                                                                                             binding protein translation initiation factor eIF-4G. EMBO J. 15:7168&#226;&#8364;&#8220; <br /> 15. Felici, F., G. Cesareni, J. M. Hughes. 1989. The abundant small <br />                                                                                             7177. <br />     cytoplasmic RNA Saccharomyces cerevisiae important function <br />                                                                                       44.   Tharun, S., W. He, A. E. Mayes, P. Lennertz, J. D. Beggs, R. Parker. <br />     required normal cell growth. Mol. Cell. Biol. 9:3260&#226;&#8364;&#8220;3268. <br />                                                                                             2000. Yeast Sm-like proteins function mRNA decapping decay. Na- <br /> 16. Foiani, M., A. M. Cigan, C. J. Paddon, S. Harashima, A. G. Hinnebusch. <br />     1991. GCD2, translational repressor GCN4 gene, general                   ture 404:515&#226;&#8364;&#8220;518. <br />     function initiation protein synthesis Saccharomyces cerevisiae.      45.   Tharun, S., R. Parker. 1999. Analysis mutations yeast mRNA <br />     Mol. Cell. Biol. 11:3203&#226;&#8364;&#8220;3216.                                                          decapping enzyme. Genetics 151:1273&#226;&#8364;&#8220;1285. <br /> 17. Gallie, D. R. 1991. The cap poly(A) tail function synergistically          46.   Thompson, J. D., D. G. Higgins, T. J. Gibson. 1994. CLUSTAL W: <br />     regulate mRNA translational efficiency. Genes Dev. 5:2108&#226;&#8364;&#8220;2116.                         improving sensitivity progressive multiple sequence alignment through <br /> 18. Gietz, D., A. St. Jean, R. A. Woods, R. H. Schiestl. 1992. Improved                 sequence weighting, position-specific gap penalties weight matrix choice. <br />     method high efficiency transformation intact yeast cells. Nucleic Acids          Nucleic Acids Res. 22:4673&#226;&#8364;&#8220;4680. <br />     Res. 20:1425.                                                                     47.   Tron, T., M. Yang, F. A. Dick, M. E. Schmitt, B. L. Trumpower. 1995. <br /> 19. Gietz, R. D., A. Sugino. 1988. New yeast-Escherichia coli shuttle vectors           QSR1, essential yeast gene genetic relationship subunit the <br />     constructed vitro mutagenized yeast genes lacking six-base pair re-             mitochondrial cytochrome bc1 complex, <span id='am-160' about='Thesaurus:Homologous_Gene' typeof='owl:Thing'>homologous gene</span> implicated <br />     striction sites. Gene 74:527&#226;&#8364;&#8220;534.                                                       eukaryotic cell differentiation. J. Biol. Chem. 270:9961&#226;&#8364;&#8220;9970. <br /> 20. Guthrie, C., R. G. Fink. 1991. Guide yeast genetics molecular          48.   Wang, X., P. M. Watt, R. H. Borts, E. J. Louis, I. D. Hickson. 1999. The <br />     biology. Methods Enzymol. 194:1&#226;&#8364;&#8220;863.                                                    topoisomerase II-associated protein, Pat1p, required maintenance of <br /> 21. Hatfield, L., C. A. Beelman, A. Stevens, R. Parker. 1996. Mutations              rDNA locus stability Saccharomyces cerevisiae. Mol. Gen. Genet. 261: <br />     trans-acting factors affecting mRNA decapping Saccharomyces cerevisiae.              831&#226;&#8364;&#8220;840. <br />     Mol. Cell. Biol. 16:5830&#226;&#8364;&#8220;5838.                                                    49.   Wang, X., P. M. Watt, E. J. Louis, R. H. Borts, I. D. Hickson. 1996. Pat1: <br /> 22. Horazdovsky, B. F., S. D. Emr. 1993. The VPS16 gene product associates              topoisomerase II-associated protein required faithful chromosome <br />     sedimentable protein complex essential vacuolar protein               transmission Saccharomyces cerevisiae. Nucleic Acids Res. 24:4791&#226;&#8364;&#8220;4797. <br />     sorting yeast. J. Biol. Chem. 268:4953&#226;&#8364;&#8220;4962.                                   50.   Zhang, S., C. J. Williams, K. Hagan, S. W. Peltz. 1999. Mutations in <br /> 23. Jacobson, A., S. W. Peltz. 1996. Interrelationships pathways              VPS16 MRT1 stabilize mRNAs activating inhibitor decap- <br />     mRNA decay translation eukaryotic cells. Annu. Rev. Biochem. 65:                 ping enzyme. Mol. Cell. Biol. 19:7568&#226;&#8364;&#8220;7576. <br />     693&#226;&#8364;&#8220;739.                                                                          51.   Zhong, T., K. T. Arndt. 1993. The yeast SIS1 protein, DnaJ homolog, <br /> 24. Kressler, D., M. Doe`re, M. Rojo, P. Linder. 1999. Synthetic lethality         required initiation translation. Cell 73:11 <br /> </body></html>