<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Jan. 1999, p. 342&#226;&#8364;&#8220;352                                                                          Vol. 19, No. 1 <br /> 0270-7306/99/$04.0010 <br /> Copyright &#194;&#169; 1999, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />             Functional Domains Rsp5 Ubiquitin-Protein Ligase <br />                                GUANGLI WANG, JOYCE YANG,                     AND   JON M. HUIBREGTSE* <br />              Department Molecular Biology Biochemistry, Rutgers University, Piscataway, New Jersey 08855-1059 <br />                     Received 18 August 1998/Returned modification 16 September 1998/Accepted 23 September 1998 <br />  <br />              RSP5, essential gene Saccharomyces cerevisiae, encodes hect domain E3 ubiquitin-protein ligase. Hect <br />           E3 proteins proposed consist broad functional domains: conserved catalytic carboxyl- <br />           terminal domain approximately 350 amino acids (the hect domain) large, nonconserved amino- <br />           terminal domain containing determinants substrate specificity. We report mapping minimal <br />           region Rsp5 necessary essential vivo function, minimal region necessary stably interact with <br />           substrate Rsp5 (Rpb1, large subunit RNA polymerase II), finding hect domain, by <br />           itself, sufficient formation ubiquitin-thioester intermediate. Mutations hect domain that <br />           affect ability form ubiquitin-thioester catalyze substrate ubiquitination abrogate vivo <br />           function, strongly suggesting ubiquitin-protein ligase activity Rsp5 intrinsically linked its <br />           essential function. The amino-terminal region Rsp5 contains WW domains C2 calcium-binding <br />           domain. Two WW domains required essential vivo function, C2 domain is <br />           not, requirements Rpb1 binding ubiquitination lie region required vivo function. <br />           Together, results support two-domain model hect E3 function indicate WW domains <br />           play role recognition substrates Rsp5, including related essential <br />           function. In addition, haploid yeast strains bearing complete disruptions of other <br />           hect E3 genes yeast, designated HUL4 (YJR036C) HUL5 (YGL141W), viable. <br />  <br />  <br />    The number proteins recognized substrates modi-                yl-terminal hect domain (homologous E6-AP carboxyl <br /> fication ubiquitination increased dramatically recent             terminus) approximately 350 amino acids (14, 37). Hect E3s <br /> years, relatively poor understanding               appear present eukaryotes, exactly 5 encoded <br /> components ubiquitin function defining                 Saccharomyces cerevisiae genome 30 far <br /> controlling substrate specificity. At classes pro-           identified mammalian species. Based biochemical char- <br /> teins recognized cooperate catalyzing protein ubiq-             acterization E6-AP, proposed obligatory <br /> uitination, which, E1 ubiquitin-activating enzyme                intermediate ubiquitination reaction catalyzed hect <br /> and family E2 ubiquitin-conjugating enzymes,                E3s ubiquitin-thioester formed thiol an <br /> characterized (37). The group activities, E3 ubiq-             absolutely conserved cysteine hect domain the <br /> uitin-protein ligases, characterized, diver-          terminal carboxyl group ubiquitin (36). The E3 becomes <br /> sity activities, terms composition mechanisms            &#226;&#8364;&#339;charged&#226;&#8364;? ubiquitin cascade ubiquitin-thioester <br /> of action, unknown. It clear, however, E3 activities             transfers, ubiquitin transferred active-site <br /> play major role defining substrate specificity, E1             cysteine E1 enzyme active-site cysteine E2 <br /> and E2 activities, alone, generally sufficient direct            and, finally, hect E3, catalyzes isopeptide bond <br /> ubiquitination biologically relevant substrates.                         formation ubiquitin substrate. The E3 is <br />    The largest known family E3 proteins, hect E3s,              thought recharged ubiquitin bound the <br /> discovered characterization human E6-AP (15, 35,                 substrate catalyze ligation multiple ubiq- <br /> 36). The interaction E6-AP human papillomavirus                 uitin moieties substrate conjugation <span id='am-2' about='protege:TO' typeof='owl:Thing'>to</span> <br /> (HPV) E6 protein cervical cancer-associated HPV types                lysines substrate lysine residues previ- <br /> causes E6-AP associate ubiquitinate p53 tumor               ously conjugated ubiquitin molecules. The resulting multiubiq- <br /> suppressor, lines evidence suggest           uitinated substrate recognized degraded 26S <br /> important component cell-immortalizing activity               proteasome. <br /> cancer-associated HPVs (13). While natural (HPV E6-                        The hect E3s range mass 92 500 kDa and <br /> independent) substrates E6-AP known,                 generally hect domain common. This <span id='am-159' about='xsp:pattern' typeof='owl:Thing'>pattern</span> <br /> proposed lack expression maternal allele                  similarity, structure-function analyses human <br /> E6-AP (also known UBE3A) human brain likely                E6-AP (16), suggested large highly variable ami- <br /> cause Angelman syndrome, severe neurologic disorder                    no-terminal domains proteins function defining <br /> (20, 26). This suggests E6-AP-dependent ubiquitination              substrate specificity. Therefore, simple model func- <br /> one proteins brain, particular, hip-               tion hect E3s consist broad functional <br /> pocampal Purkinje neurons (1), critical normal brain             domains: large amino-terminal domain contains deter- <br /> function.                                                                   minants substrate specificity carboxyl-terminal hect <br />    The family E6-AP-related proteins defined carbox-             domain, catalyzes multiubiquitination associated sub- <br />                                                                             strates. To test refine model, explored <br />                                                                             function substrate specificity hect E3 encoded by <br />   * Corresponding author. Mailing address: Department Molecular          RSP5 gene S. cerevisiae. <br /> Biology Biochemistry, Rutgers University, Piscataway, NJ 08855-            RSP5 (also known NPI1 MDP1) essential gene <br /> 1059. Phone: (732) 445-0938. Fax: (732) 445-4213. E-mail: huibregt          isolated types genetic screens, <br /> @waksman.rutgers.edu.                                                       including suppressor mutations SPT3 (46b). SPT3 <br />  <br />                                                                       342 <br />  VOL. 19, 1999                                                                                   FUNCTIONAL DOMAINS OF Rsp5                         343 <br />  <br />  <br /> encodes component SAGA (Spt/Ada/Gcn5/acetyl-                                         MATERIALS AND METHODS <br /> transferase) complex (8, 34), proposed affect <br /> a minimum aspects RNA polymerase II (pol II)                   Yeast strains plasmids. Yeast strain FY56 (MATa his4-912 DR5 lys2-128D <br />                                                                       ura3-52) isogenic rsp5-1 mutant strain FW1808 obtained Fred <br /> transcription: function TATA binding protein               Winston (Harvard Medical School). The RSP5 rsp5-1 open reading frames <br /> (Spt15) activity Gcn5 histone acetyltransferase.       (ORFs) amplified PCR genomic DNAs FY56 FW1808, <br /> We previously reported largest subunit RNA polII,         respectively, cloned pGEM-1 vector. Three independent clones of <br />                                                                       sequenced compared. The rsp5-1 ORF subcloned the <br /> Rpb1, substrate Rsp5 (17). While biological rele-         pYES2 pGEX-6p-1 vectors, previously described wild-type <br /> vance Rpb1 ubiquitination clear, related       (wt) RSP5 ORF, active-site Cys-to-Ala (C-A) mutant (amino acid 777), and <br /> the SPT3/RSP5 genetic interaction. In addition, study         CD6 mutant. Additional mutated RSP5 genes (see Fig. 3) generated by <br />                                                                       PCR using gene-specific primers. 59 PCR primers, cases, encoded the <br /> showed large subunit human polII (polII-LS)            hemagglutinin (HA) epitope, cases, PCR products subcloned <br /> subject ubiquitination degradation response UV           version pYES2 initiating ATG HA epitope cloned <br /> irradiation, suggesting ubiquitin-mediated degradation        upstream multicloning sites (pYES-HA). Mutants M, N, O utilized <br />                                                                       primers encoding indicated mutations, introduced SalI restriction <br /> the polII-LS involved response DNA damage (4,            site. Multiple clones construct analyzed in vitro vivo <br /> 32).                                                                  assays, clones partially sequenced confirm introduction of <br />    RSP5 shown genetically affect activity     mutations cloning junctions. All constructs shown Fig. 3, the <br />                                                                       pYES2 vector, transformed FY56 FW1808 strains. Plasmids <br /> several plasma membrane-associated permeases, including               encoding Rpb1 glutathione S-transferase (GST)-carboxy-terminal do- <br /> Gap1 (general amino acid permease), Fur4 (uracil permease),           main (CTD) fusion described previously (17). <br /> maltose permease, plasma membrane H1-ATPase (6,                  The rsp5D::LEU2 mutant strains derived W303 diploid strain <br /> 11, 23), ubiquitination Fur4 Gap1 results         (leu2-3112 ade2-1 his3-11 trp1-1 can1-100). A 4.5-kbp fragment chromosome <br />                                                                       V spanning RSP5 locus amplified PCR cloned pBlue- <br /> down-regulation (9, 42). Mutations RSP5             script II KS1 vector. Plasmid pBlueDR5 generated replacing com- <br /> shown alter mitochondrial-cytoplasmic protein distribu-        plete RSP5 ORF plasmid LEU2 gene. A linear 3.5-kbp disrupting <br /> tion Mod5 protein, tRNA modification enzyme             fragment isolated pBlueDR5 digestion HindIII NotI. This <br />                                                                       DNA fragment transformed diploid W303 cells, Leu1 transfor- <br /> functions cellular compartments (49). Al-            mants isolated. The deletion chromosomal copy RSP5 gene <br /> though effect Rsp5 pathways            introduction LEU2 RSP5 locus confirmed PCR. <br /> indirect, broad range cellular processes affected RSP5      Sporulation heterozygous diploid (GW001) gave expected 2:2 ratio of <br />                                                                       viable-to-inviable spores essential gene. GW001 transformed a <br /> suggests Rsp5 protein multiple diverse substrates.       centromere- URA3-based plasmid, pRS416-RSP5, harboring 4.5-kbp <br />    The primary structure Rsp5 reveals types domains         genomic RSP5 fragment, selecting growth plates lacking uracil leu- <br /> within amino-terminal region protein (see Fig. 1).         cine. Haploid spores bearing rsp5D::LEU2 allele pRS416-RSP5 <br />                                                                       plasmid isolated yield strains GW003 (MATa rsp5D::LEU2 ade2-1 his3-11 <br /> The C2 domain (amino acids 3 140),       trp1-1 can1-100[pRS416-RSP5]) GW004, equivalent MATa strain. <br /> in variety proteins, including protein kinase C, synapto-           Centromere- TRP1-based plasmids pRS414-Gal-A, -B, -D gen- <br /> tagmin, phospholipase C, p120ras-GAP (33). C2 domains             erated subcloning NgoM1-NotI fragment pYES2-HA-Rsp5 (construct <br /> have shown interact membrane phospholipids,              A; Fig. 3), pYES2-HA-Rsp5C-A (construct B), pYES2-HA-Rsp5DC2 <br />                                                                       (construct D) sites pRS414, yielding pRS414-Gal-A, -B, -D. <br /> inositol polyphosphates, proteins. The interaction C2          Expression respective Rsp5 proteins plasmids control <br /> domains ligands is, cases, dependent         GAL1 promoter. The pRS414-RSP5 plasmid contained 4.5-kbp <br /> regulated Ca21. Since membrane-associated sub-             genomic RSP5 fragment, expression Rsp5 control of <br />                                                                       natural RSP5 promoter. Plasmids pRS414-Gal-A, pRS414-Gal-B, pRS414- <br /> strates Rsp5 identified, conceivable      Gal-D, pRS414-RSP5 transformed yeast strains GW003 and <br /> C2 domain involved targeting proteins,          GW004, followed 5-fluoroorotic acid (5-FOA) counterselection the <br /> localizing Rsp5 plasma membrane directly medi-           pRS416-RSP5 plasmid. GW015 GW016 MATa -a, respectively, <br />                                                                       containing pRS414-Gal-A plasmid. GW017 GW018 MATa -a, <br /> ating interaction membrane-associated substrates,            respectively, containing pRS414-Gal-D plasmid, GW013 GW014 are <br /> such Fur4 Gap1.                                                MATa -a, respectively, containing pRS414-RSP5 plasmid. 5-FOA-resis- <br />    Rsp5 contains WW domains, spanning amino                tant colonies obtained pRS414-Gal-B (the C-A mutant) <br />                                                                       transformant. <br /> acids 231 418. WW domains protein-protein interaction             The HUL4 (YJR036C) HUL5 (YGL141W) ORFs, located chromo- <br /> modules affinity proline-rich sequences,        somes X VII, respectively, isolated PCR FY56 genomic DNA <br /> the consensus binding site containing PPxY sequence (PY             cloned pBluescript. Heterozygous chromosomal disruptions both <br /> motif) (7, 45). WW domains consist 38 40 amino acids,           ORFs individually W303 diploid background, described <br />                                                                       RSP5. Haploid spores containing hul4D::LEU2 hul5D:: <br /> have absolutely conserved tryptophan residues, form         LEU2 allele fully viable. <br /> three-stranded antiparallel b-sheet hydrophobic binding           Protein expression vitro biochemical assays. GST fusion proteins were <br /> pocket PPxY ligand (24). Functionally, WW do-             expressed Escherichia coli standard methods affinity purified gluta- <br />                                                                       thione-Sepharose (Pharmacia). Rsp5 proteins expressed pGEX-6p-1 <br /> mains somewhat analogous SH3 domains              vector cleaved GST using PreScission protease (Pharmacia) under <br /> both affinities proline-rich ligands, cases,     manufacturer-recommended conditions, proteins used ubiq- <br /> WW SH3 domains compete binding                uitin-thioester ubiquitination reactions. Ubiquitin-thioester reactions were <br />                                                                       performed using vitro-translated proteins (see Fig. 7), described <br /> ligands vitro (3, 44). Given proposed function pro-       previously (14). E1 ubiquitin-activating enzyme Arabidopsis thaliana Ubc8 <br /> tein-protein interaction modules, WW domains clearly          protein prepared described previously (35). In vitro ubiquitin-thioester <br /> strong candidates sites Rsp5 substrate interaction.             assays utilized 32P-labeled ubiquitin, prepared pGEX-2TK-ubiquitin (35) <br />                                                                       labeled vitro [g-32P]ATP heart muscle kinase (Sigma). The <br />    We performed structure-function analysis Rsp5         ubiquitin cleaved GST thrombin, thrombin heat <br /> order understand domain structure vitro             inactivated incubation 65&#194;&#176;C 10 min. In vitro translation reactions were <br /> biochemical activities Rsp5 related essential vivo   carried coupled transcription-translation reactions using Promega TNT <br />                                                                       reagents [35S]methionine. Rpb1-DCTD coupled vitro tran- <br /> function. We determined minimal region do-               scription-translation using template pYES2-Rbp1 plasmid DNA that <br /> mains Rsp5 necessary complementation rsp5-1             treated BsiWI restriction endonuclease described previously <br /> temperature-sensitive mutant vivo regions necessary        (17). <br /> for interaction ubiquitination Rpb1 vitro. In addition,        Whole-cell yeast cell extracts immunoblotting analyses (see Fig. 4A B) <br />                                                                       prepared method Silver et al. (41). Primary antibodies either <br /> we hect domain, itself, sufficient forma-     anti-HA rabbit polyclonal antibody (Santa Cruz Biotechnology) anti- <br /> tion ubiquitin-thioester intermediate.                         Rsp5 mouse monoclonal antibody (17). Horseradish peroxidase-linked second- <br />  344       WANG ET AL.                                                                                                                           MOL. CELL. BIOL. <br />  <br />  <br /> ary antibodies chemiluminescent reagents obtained Du Pont <br /> NEN. <br />   Rsp5/Rpb1 binding assays (see Fig. 6) conducted previously described <br /> (17). Briefly, reaction mixture contained 125 ml 25 mM Tris (pH 8.0), 125 <br /> mM NaCl, 0.1% Nonidet P-40, 100 ng GST fusion protein bound 10 ml <br /> of glutathione-Sepharose. Five microliters vitro-translated protein was <br /> added reaction mixture. Reaction mixtures rotated 2 h 4&#194;&#176;C, <br /> and Sepharose beads washed times 500 ml buffer contain- <br /> ing 100 mM Tris, 100 mM NaCl, 1% Nonidet P-40. Sodium dodecyl sulfate- <br /> polyacrylamide gel electrophoresis (SDS-PAGE) loading buffer added di- <br /> rectly beads heated 95&#194;&#176;C 5 min, proteins analyzed by <br /> SDS-PAGE autoradiography. <br />  <br />  <br />                                  RESULTS <br />  <br />    The protein encoded rsp5-1 allele impaired in <br /> ubiquitin-thioester formation catalysis substrate ubiq- <br /> uitination. The S. cerevisiae rsp5-1 mutant (strain FW1808),                           FIG. 1. Schematic Rsp5 protein C2 domain, WW domains 1 <br /> isolated F. Winston coworkers (cited references 14                        3, hect domain indicated. The sequence carboxyl-terminal 100 <br /> and 17), grows 30&#194;&#176;C rich media doubling time                      amino acid residues Rsp5 (amino acids 710 809) shown. Amino acids <br />                                                                                     marked asterisks residues highly conserved hect domain <br /> slightly longer isogenic RSP5 mutant strain                        family proteins. The rsp5-1 <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span>mutation</span> (L S, amino acid 733), active-site <br /> (FY56) severely impaired growth elevated tem-                         mutation (C A, amino acid 777), residues deleted CD6 mutation <br /> perature. Characterization rsp5-1 allele protein                     indicated. <br /> product undertaken order understand biochem- <br /> ical basis mutant <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-5' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span>phenotype</span>. The rsp5-1 RSP5 ORFs <br /> were amplified PCR genomic DNAs respec- <br /> tive strains, multiple independent clones isolated                     abrogates ubiquitination p53 (14). The rsp5-1-encoded <br /> sequenced. The difference rsp5-1 sequence                          protein impaired formation ubiquitin-thioester <br /> and wild-type RSP5 sequence T-to-C transition                          intermediate relative CD6 mutant. A small of <br /> nucleotide 2198 rsp5-1 ORF, resulting Leu-to-Ser                        monoubiquitinated rsp5-1-encoded protein detected after <br /> alteration amino acid 733 (Fig. 1). Leu733 hect                    DTT heat treatment, suggesting retains ca- <br /> domain, amino terminal active-site cysteine residue                       pacity catalyze isopeptide-linked ubiquitination. A temper- <br /> 777. Leu733 represents conserved hydrophobic residue                              ature dependence thioester formation observed for <br /> among hect domains, leucine phenylalanine                       rsp5-1-encoded protein vitro, impaired of <br /> in nearly cases.                                                                temperatures tested, 4 37&#194;&#176;C (data shown). <br />    The protein encoded rsp5-1 allele expressed                         We previously reported largest subunit RNA <br /> GST fusion protein pGEX-6p vector, allows                          polII, Rpb1, substrate Rsp5 (17). The purified Rsp5 <br /> cleavage GST portion molecule highly                           proteins assayed ability catalyze ubiq- <br /> specific protease (PreScission Protease; Pharmacia). Wt Rsp5,                       uitination Rpb1 vitro (Fig. 2B). The wt Rsp5 protein very <br /> the active-site Cys-to-Ala (C-A) mutant, previously                         efficiently ubiquitinated wt Rpb1 did ubiquitinate Rpb1 <br /> described CD6 mutant, carboxyl-terminal                            CTD, binding site Rsp5, deleted. The <br /> amino acids truncated, expressed manner.                       C-A protein, CD6 protein, rsp5-1-encoded protein <br /> Figure 2A shows result vitro ubiquitin-thioester                       inactive catalyzing ubiquitination Rpb1. To de- <br /> assay utilizing purified Rsp5 proteins partially                   termine inability mutant protein ubiquitinate <br /> purified recombinant E1 (human) E2 (A. thaliana Ubc8)                           Rpb1 inability bind Rpb1, GST-Rsp5 fusion <br /> proteins 32P-labeled ubiquitin. The basis assay                       proteins assayed ability bind Rpb1 present in <br /> that thioester adducts formed radiolabeled ubiquitin                        crude yeast cell extracts. Figure 2C shows GST <br /> and enzyme E1, E2, E3 stable analyzed                            fusion proteins (wt, C-A, CD6, Rsp5-1) equally ca- <br /> SDS-PAGE performed absence reducing agent,                          pable binding Rpb1. Together, results suggest that <br /> these adducts disrupted analyzed standard                        biochemical basis rsp5-1-encoded phenotype a <br /> SDS-PAGE conditions (high reducing agent concentration).                            defect ability enzyme form ubiquitin-thioester <br /> As reported previously (14), difficult detect ubiquitin-                 intermediate catalyze substrate ubiquitination, rather <br /> thioester intermediate wild-type Rsp5 protein                      general defect recognition binding sub- <br /> of propensity protein catalyze isopeptide-linked                     strates. <br /> (dithiothreitol [DTT] insensitive) ubiquitination itself                     The hect domain WW domains 2 3 required for <br /> intramolecular transfer cysteine lysine residues                        essential vivo function Rsp5. In order determine <br /> elsewhere protein. While unclear under-                    minimal domain requirements complementation the <br /> lying biological relevance vitro self-ubiquitination                      rsp5-1 allele, set mutated Rsp5 proteins expressed in <br /> reaction, dependent active-site cysteine and,                       rsp5-1 mutant yeast strain FW1808, isogenic RSP5 <br /> shown below, correlates ability catalyze intermo-                       mutant strain FY56, control GAL1 promoter. <br /> lecular ubiquitination substrates.                                               Figure 3 illustrates set truncated mutated Rsp5 <br />    The CD6 mutant forms DTT-sensitive ubiquitin-thioester                         proteins generated. All constructs encoded HA <br /> but, contrast wild-type protein, does catalyze                        antibody epitope 59 end, construct E. Mutations <br /> isopeptide-linked self-ubiquitination, suggesting ex-                      disrupted individual WW domains altered second con- <br /> treme carboxyl terminus protein important                         served tryptophan nearby conserved proline residue <br /> ability catalyze isopeptide-linked ubiquitin conjugates. This                    WxxP FVDA, shown Fig. 3. These substitutions <br /> is supported observation analogous mutation                      originally suggested based nuclear magnetic res- <br /> human E6-AP does affect ubiquitin-thioester formation                           onance structure YAP WW domain, substitu- <br />  VOL. 19, 1999                                                                                                    FUNCTIONAL DOMAINS OF Rsp5                        345 <br />  <br />  <br />  <br />  <br />    FIG. 2. (A) Ubiquitin-thioester assay using 32P-labeled ubiquitin (ub) purified Rsp5 proteins (the wt, C-A mutant protein; rsp5-1-encoded [ts] mutant <br /> protein, CD6 mutant protein). The presence absence recombinant E1 E2 proteins indicated, migration positions ubiquitin adducts <br /> are indicated. The samples left (lanes 1 7) analyzed SDS-PAGE using loading buffer lacked DTT, samples right analyzed <br /> with DTT-containing loading buffer (lanes 8 14). Migration positions molecular mass standards (in kilodaltons) indicated right. (B) Rpb1 <br /> ubiquitination assay using purified Rsp5 proteins vitro-translated, 35S-labeled Rpb1 (lanes 1 5) Rpb1 CTD deleted (Rpb1-DCTD, lanes 6 10). <br /> (C) Binding Rpb1, present total yeast cell extract (prepared glass bead lysis), GST-Rsp5 fusion proteins (lanes 2 5) GST&#226;&#8364;&#8220;E6-AP negative control <br /> (lane 6). Lane 1 represents 20% input used binding reaction mixture. Rpb1 detected immunoblotting anti-CTD antibody. <br />  <br />  <br />  <br />  <br /> tions, individually, shown render domain                             medium, total cell extracts analyzed protein ex- <br /> completely inactive terms ligand (5, 24).                                       pression immunoblotting using anti-HA anti- <br />    To confirm vivo expression altered Rsp5 proteins,                        Rsp5 antibodies (Fig. 4A B). All proteins expressed <br /> the FY56 transformants grown galactose-containing                             detectable levels, level expression exogenous <br />  346     WANG ET AL.                                                                                                          MOL. CELL. BIOL. <br />  <br />  <br /> HA-tagged wt Rsp5 roughly equivalent level the <br /> endogenous Rsp5 protein (protein C, CD6 mutant form, is <br /> not shown results experiment expressed <br /> at level similar wt Rsp5). Expression protein E, <br /> which does contain HA epitope, confirmed by <br /> using anti-Rsp5 antibody (Fig. 4B). Proteins K L were <br /> also detected anti-Rsp5 antibody, though <br /> the HA epitope encoded ORF, suggesting the <br /> epitope removed proteolysis posttranslation- <br /> ally. Protein I did react anti-Rsp5 antibody, con- <br /> sistent epitope mapping experiments indicated that <br /> this monoclonal antibody recognizes epitope spanning the <br /> region containing WW domains 2 3. The level expres- <br /> sion WW domain mutants M, N, O similar to <br /> that wt Rsp5; however, proteins N O consistently <br /> showed pattern higher-molecular-weight species which <br /> might represent ubiquitinated intermediates proteins. <br /> The expression pattern mutant proteins rsp5-1 <br /> transformants identical seen RSP5 back- <br /> ground (data shown). <br />    When expression exogenous RSP5 genes re- <br /> pressed (by growth dextrose), FY56 (RSP5) and <br /> FW1808 (rsp5-1) transformants grew similarly respec- <br /> tive untransformed strains; is, FY56 transformants <br /> grew 30 37&#194;&#176;C, FW1808 transformants <br /> grew 30&#194;&#176;C 37&#194;&#176;C. When expression induced (by <br /> growth galactose-containing medium), FY56 trans- <br /> formants, K L (Discussed below), grew similarly to <br /> the control strain (vector alone) 30 37&#194;&#176;C. The <br /> ability various constructs complement rsp5-1 tem- <br /> perature-sensitive growth defect tested assaying for <br /> growth rsp5-1 transformants 37&#194;&#176;C galactose-con- <br /> taining agar plates. Expression wild-type HA-tagged             FIG. 3. Schematic mutated Rsp5 proteins (A O; numbering corre- <br /> Rsp5 protein (A) fully complemented temperature-sensi-          sponds amino acids retained mutated proteins) summary of <br /> tive growth defect, vector did not. Neither       abilities complement rsp5-1-encoded growth defect 37&#194;&#176;C ga- <br /> the active-site C-A mutant (B) CD6 mutant (C)           lactose. All proteins E contained HA epitope amino- <br />                                                                     terminal end. The WW consensus sequence (7) shown relative se- <br /> able complement growth defect. These results strongly        quences WW domains 1, 2, 3. The lowercase h consensus sequence <br /> suggest ubiquitin-protein ligase activity Rsp5       represents hydrophobic residues, lowercase t represents turn-like or <br /> intrinsically linked essential vivo function.             polar position (45). Each WW domain mutant proteins (M, N, O) <br />    Constructs D, E, F, G, deletions increasing          contained sequence FVDA place WxxP sequence, indicated. <br /> portions amino-terminal domain, complemented the <br /> rsp5-1 mutation, variable efficiency. Protein D, <br /> with entire C2 domain deleted, complemented growth          L, share characteristics: disrupted hect do- <br /> defect efficiency similar full-<span id='am-1' about='xsp:length' typeof='owl:Thing'>length</span> Rsp5;      main intact WW domains. Expression proteins K L, <br /> however, protein E, sequences to, including,        contain amino-terminal domain sequences, inhib- <br /> WW domain 1 deleted grew significantly slower 37&#194;&#176;C. Trun-        ited growth severely inhibited growth the <br /> cation WW domain 1 (F) restored nearly comple-          RSP5 strain. We speculate growth-inhibitory effect of <br /> mentation activity, truncation WW domain 1        mutant proteins titration substrates away <br /> of spacer region domains 1 2 supported              endogenous rsp5-1-encoded protein, resulting a <br /> growth reduced efficiency (G). Proteins amino-       lack degradation substrate(s) related essential <br /> terminal truncations included WW domain 2 (proteins           function Rsp5. The fact region spanning WW <br /> H I) did support growth 37&#194;&#176;C, did         domains required rsp5-1 complementation the <br /> carboxyl-terminally truncated proteins (C, J, K, L). In ad-      dominant-inhibitory effect consistent hypothesis. <br /> dition, WW domain 1 mutant fully complemented                  The C2 domain required essential vivo <br /> rsp5-1 growth defect (protein M), mutation        function Rsp5. It somewhat surprising C2 do- <br /> WW domain 2 3 completely abrogated complementation               main dispensable complementation rsp5-1 mu- <br /> activity (proteins N O, respectively). These results indicate   tation, plasma membrane-associated substrates of <br /> that region amino-terminal domain includes          Rsp5 identified. This led question the <br /> WW domains 2 3, complete functional            protein encoded rsp5-1 allele playing role in <br /> carboxyl-terminal hect domain, required essential       supporting function proteins C2 deleted (proteins <br /> function Rsp5.                                                   D, E, F, G) complementation assay described above. <br />    Interestingly, subgroup noncomplementing mutant         For example, conceivable 37&#194;&#176;C temperature- <br /> proteins actually inhibited growth rsp5-1 transformants      inactivated rsp5-1 mutant protein, heterodimerization <br /> on galactose-containing medium permissive temperature        protein C2 deleted, serve localize the <br /> (data shown) dominant-negative             protein C2 deleted plasma membrane, it <br /> mutant proteins. This set included mutant proteins B, C, J, K,      contact ubiquitinate substrates. This idea was <br />  VOL. 19, 1999                                                                                           FUNCTIONAL DOMAINS OF Rsp5                     347 <br />  <br />  <br />  <br />  <br />    FIG. 4. (A) Anti-HA immunoblot total cell extracts indicated FY56 transformants grown galactose-containing medium. vec., vector. (B) <br /> Anti-Rsp5 immunoblot extracts. The migration position endogenous Rsp5 protein indicated. (C) Growth FW1808 transformants on <br /> galactose-containing agar plates 37&#194;&#176;C. <br />  <br />  <br />  <br /> suggested experiments indicate Rsp5,                    tion results, 5-FOA-resistant transformants obtained on <br /> human E6-AP, form homodimers certain condi-                           galactose-containing plates plasmids expressing protein A <br /> tions (2a). We determined Rsp5 protein                     D, wild-type RSP5 genomic fragment, but <br /> with C2 deleted (protein D) function sole source                   construct B, C-A mutant (Fig. 5A). As expected, <br /> of Rsp5 protein cell.                                                    A D transformants mating type grow <br />    A chromosomal deletion complete RSP5 ORF                          galactose glucose carbon source (Fig. 5B). <br /> made diploid strain (W303). Tetrad dissections con-                   Immunoblotting confirmed transformant D rsp5D <br /> sistent RSP5 essential gene, previously re-                    background expresses form Rsp5 C2 do- <br /> ported (48, 49). Viable rsp5D::LEU2 haploids obtained                   main deleted (Fig. 5C). We conclude C2 <br /> sporulation heterozygous diploid transformation                    domain required essential Rsp5 function under <br /> with URA3-based centromere-containing plasmid expressing                      standard growth conditions. <br /> wild-type RSP5 promoter. This strain                         WW domain 2 essential mediating interaction of <br /> transformed TRP1-based plasmid expressing con-                    Rsp5 Rpb1. We previously showed Rsp5 binds and <br /> struct A (wt HA-Rsp5), B (the C-A mutant), D (with C2                    ubiquitinates Rpb1, largest subunit RNA polII that <br /> domain deleted) control galactose-inducible pro-                     CTD Rpb1 necessary sufficient stable binding <br /> moter wild-type RSP5 expressed pro-                        Rsp5 (17). To determine regions Rsp5 critical <br /> moter. The URA3-based RSP5 plasmid selected                            binding Rpb1, subset Rsp5 mutant proteins (A, <br /> against 5-FOA. In accord rsp5-1 complementa-                      D, E, G I, L O) translated vitro rabbit <br />  348      WANG ET AL.                                                                                                                            MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 5. (A) Drsp5::LEU2 haploid cells transformed URA3-based plasmid expressing wt RSP5 TRP1-based plasmid expressing HA-tagged <br /> wt Rsp5 (Gal-A), C-A mutant protein (Gal-B), protein C2 domain deleted (Gal-D) control GAL1 promoter wt Rsp5 control <br /> of promoter (RSP5). The transformants streaked galactose-containing agar plates containing uracil 5-FOA select loss URA3 plasmid. <br /> All TRP1-based plasmids, expressing C-A mutant protein (B), support continued growth cells conditions. (B) Galactose <br /> and dextrose plates RSP5 Gal-A Gal-D transformants panel A. The upper lower sectors mating types, respectively. (C) Total cell <br /> extracts genomic RSP5 Gal-A Gal-D transformants, grown galactose, analyzed immunoblotting using anti-Rsp5 (top) an <br /> anti-HA (bottom) antibody. Both mating types analyzed. <br />  <br />  <br />  <br />  <br /> reticulocyte lysate (Fig. 6A) assayed ability                     deletion hect domain (L) resulted binding GST- <br /> to bind GST-CTD (Fig. 6B). Paralleling importance                        CTD efficiency similar wild-type Rsp5 (A). <br /> the vivo rsp5-1 complementation assay, deletion C2                            Binding Rpb1 Rsp5 analyzed expressing the <br /> domain WW domain 1 did affect CTD binding (D, E, G,                           subset Rsp5 mutants GST fusion proteins (Fig. 6C) <br /> and L), deletion WW domain 2 (H I)                            assaying binding vitro-translated Rpb1 (Fig. 6D). <br /> abolished CTD binding. Likewise, disruption WW domain 1                           The results similar described above, that <br /> by mutation did affect CTD binding (M), disruption                         assay, disruption WW domain 3 did significantly <br /> of WW domain 2 completely abolished binding (N). Mutation                            affect Rpb1 binding. The results same, regardless of <br /> of WW domain 3 (protein O) consistently reduced binding                           Rpb1 vitro translated present crude yeast <br /> approximately 50%. As shown previously (17), binding Rsp5                         cell extracts (data shown), consistent previous <br /> to CTD independent hect domain, complete                           findings Rsp5 bind Rpb1 free or <br />  VOL. 19, 1999                                                                                             FUNCTIONAL DOMAINS OF Rsp5                        349 <br />  <br />  <br />  <br />  <br />                                                                                     FIG. 7. Ubiquitin-thioester assay. The wt C-A mutant hect domains of <br />                                                                                  Rsp5 (amino acids 456 809) vitro translated rabbit reticulocyte <br />                                                                                  lysate incubated partially purified recombinant E1 (human) E2 (A. <br />                                                                                  thaliana UBC8) proteins. Products subjected SDS&#226;&#8364;&#8220;10% PAGE using <br />                                                                                  loading buffer lacked contained DTT. <br />  <br />  <br />  <br />                                                                                  full-length protein (Fig. 2A) lie outside hect <br />                                                                                  domain. <br />                                                                                     Two yeast hect E3 genes, HUL4 HUL5, non- <br />                                                                                  essential genes. Database searches indicate S. cerevisiae <br />                                                                                  genes encode apparent hect E3 proteins. These <br />                                                                                  RSP5, UFD4, TOM1, HUL4, HUL5 (HUL stands for <br />                                                                                  hect ubiquitin ligase). RSP5 essential (49), UFD4 not <br />                                                                                  essential (19), TOM1 (Trigger Mitosis) presumed to <br />                                                                                  essential (required G2/M cell cycle transition; un- <br />                                                                                  published data; reference 46). To determine phenotype <br />                                                                                  haploid strains bearing disruption of previous- <br />                                                                                  ly uncharacterized hect E3 genes, designated HUL4 <br />                                                                                  (YJR036C) HUL5 (YGL141W), completely deleted <br />                                                                                  ORF individually W303 diploid background, <br />                                                                                  creating HUL4/hul4D::LEU2 HUL5/hul5D::LEU2 <br />                                                                                  heterozygous strains. We created RSP5/rsp5D::LEU2 <br />    FIG. 6. (A) 35S-labeled vitro translation products Rsp5 mutants,   diploid manner. Tetrad analysis the <br /> designations corresponding proteins depicted Fig. 2, human      RSP5/rsp5D::LEU2 diploid showed expected 2:2 ratio of <br /> E6-AP. (B) Translation products panel A assayed binding         viable-to-inviable spores essential gene, vi- <br /> GST-CTD immobilized glutathione-Sepharose. (C) Coomassie-stained SDS- <br /> PAGE GST-Rsp5 fusion proteins, designations corresponding proteins    able spores leucine auxotrophs. In contrast, 90% <br /> depicted Fig. 2. (D) GST-Rsp5 fusion proteins (100 ng lane)      tetrads HUL4/hul4D::LEU2 and <br /> panel C assayed binding 35S-labeled, vitro-translated Rpb1. GST   HUL5/hul5D::LEU2 diploids yielded viable spores <br /> (no fusion) GST&#226;&#8364;&#8220;E6-AP served negative controls binding. Twenty        (Fig. 8), 2:2 segregation LEU2 allele. Both hap- <br /> percent input translation reaction mixture used each <br /> binding reaction shown.                                                       loid disruptants grew similarly parental strain 37&#194;&#176;C, as <br />                                                                                  well. Therefore, HUL4 HUL5 essential genes un- <br />                                                                                  der standard growth conditions. <br />  <br /> present multisubunit polII enzyme. Together, re-                                                  DISCUSSION <br /> sults indicate C2 domain, WW domain 1, hect <br /> domain required Rpb1 binding, WW domain                           The hect E3 proteins proposed consist two <br /> 2 absolutely required. WW domain 3 contribute                     functional domains: large amino-terminal domain func- <br /> Rpb1 binding required.                                                tions defining substrate specificity approximately <br />    The hect domain, itself, form ubiquitin-thioester                    40-kDa carboxyl-terminal domain catalyzes ubiquitination <br /> intermediate. To determine hect domain, itself,                     bound substrates. The results Rsp5 structure-function <br /> sufficient recognition activation E1 E2                       analysis support two-domain model point those <br /> enzymes ubiquitin system, isolated hect domain                     determinants amino-terminal domain crit- <br /> Rsp5 (amino acids 456 809) assayed ubiquitin-                         ical essential vivo function Rsp5. In addition, our <br /> thioester formation, equivalent active-site (C-A)                 results strongly suggest ubiquitin-protein ligase activity <br /> mutant. Figure 7 shows hect domain sufficient                    Rsp5 intrinsically linked essential vivo function. <br /> ubiquitin-thioester formation mutation con-                      This based finding mutations hect <br /> served cysteine hect domain abolishes activity.                      domain disrupt impair formation ubiquitin-thio- <br /> These results indicate hect domain separable                       ester and/or ability protein catalyze transfer of <br /> functional domain determinants necessary inter-                     ubiquitin substrates (the C777-to-A, rsp5-1, CD6 muta- <br /> action upstream components ubiquitin-thio-                       tions) abrogate vivo function Rsp5. Several rsp5 <br /> ester cascade (the E1 E2 enzymes) lie completely                      temperature-sensitive alleles isolated Zolladek et al. <br /> the hect domain. The hect domain, unlike full-length pro-                    (49), like rsp5-1, mutations lie hect <br /> tein, did catalyze isopeptide-linked self-ubiquitination, sug-               domain disrupt conserved amino acids. We speculate that <br /> gesting lysines sites self-ubiquitination              mutations result defect Rsp5 ubiquitination <br />  350     WANG ET AL.                                                                                                    MOL. CELL. BIOL. <br />  <br />  <br />                                                                        tion clear; however, human polII-LS ubiquitinated in <br />                                                                        response UV irradiation (32), suggesting possible role in <br />                                                                        response DNA damage. Preliminary results shown <br />                                                                        DNA damage induces degradation Rpb1 yeast, as <br />                                                                        well, dependent Rsp5 (2a). Apparent ho- <br />                                                                        mologs Rsp5 exist mammalian cells (see below), is <br />                                                                        possible mediate UV- <br />                                                                        dependent ubiquitination human polII. <br />                                                                           While majority hect E3s unique amino-terminal <br />                                                                        domain sequences recognizable characterized se- <br />                                                                        quence motifs, hect E3s mamma- <br />                                                                        lian nonmammalian species which, like Rsp5, contain mul- <br />                                                                        tiple (two four) WW domains (18, 21, 29, 30, 47). Although <br />                                                                        cDNAs encoding proteins remain be <br />                                                                        sequenced entirety, proteins appear also <br />                                                                        contain C2 domain extreme amino terminus. Schizo- <br />                                                                        saccharomyces pombe encodes hect E3 genes three <br />                                                                        WW domains C2 domain, pub1, which <br />   FIG. 8. Tetrad analysis RSP5/rsp5D::LEU2, HUL4/hul4D::LEU2, and <br />                                                                        targets cdc25 protein phosphatase (27). The mouse-human <br /> HUL5/hul5D::LEU2 W303 diploids leucine-containing agar plates.      Nedd4 protein (human Rpf1) C2/WW-hect E3 protein that <br />                                                                        proposed down-regulate epithelial Na1 chan- <br />                                                                        nel ubiquitinating b and/or g subunits (31, 43). Thus, the <br />                                                                        WW- C2/WW-hect E3s appear form distinct subgroup <br /> activity, general defect binding associa-       hect E3s. The diversity functions substrates the <br /> tion substrates.                                                  WW hect E3s suggests proteins perform their <br />    With respect specific determinants amino-         functions diverse intracellular locations. Studies the <br /> terminal region Rsp5, results clearly indicate C2      mammalian C2/WW-hect E3, Nedd4, shown pri- <br /> domain required essential function Rsp5.             marily cytoplasmic perinuclear calcium influx <br /> Several membrane-associated permeases shown               mediates localization plasma membrane (10, 21, 31). <br /> be regulated ubiquitination Rsp5-dependent manner,            Preliminary localization studies yeast suggest Rsp5 <br /> including Fur4 (uracil permease) Gap1 (general amino               cytoplasmic nuclear (46a). <br /> acid permease) (9, 42). Based fact C2 domains            Of yeast hect E3s, Rsp5 contains WW domains, <br /> been shown associate membrane phospholipids                 Tom1, Ufd4, Hul4, Hul5 contains previ- <br /> membrane-associated proteins, C2 domain in-               ously identified protein motifs amino-terminal re- <br /> volved targeting group substrates, affecting      gion. We speculate individual WW domains of <br /> intracellular localization Rsp5 directly contacting sub-      Rsp5 involved mediating interaction more <br /> strates. If case, targeting membrane-       specific substrates. Five yeast genes encode proteins with <br /> associated substrates essential function            apparent WW domains (ESS1, PRP40, TIN1, YPR152C, and <br /> Rsp5. Further experiments necessary determine               YFL010C), contain single WW domain, <br /> effect specific Rsp5 mutations ubiquitination plasma          PRP40, contains two. A subject inves- <br /> membrane-associated substrates.                                        tigation WW domain proteins are <br />    In contrast C2 domain, subset WW domains            hect E3s compete Rsp5 binding sub- <br /> are essential Rsp5 function. Disruption WW domain 2          strates, basis regulated ubiquitination. <br /> 3 abrogates Rsp5 function vivo, WW domain 1               The hect domain Rsp5, itself, sufficient <br /> disrupted significant effect rsp5-1 complementation         formation ubiquitin-thioester intermediate, indicat- <br /> activity. WW domains proposed protein-pro-             ing hect domain biochemical activity independent <br /> tein interaction modules affinity proline-rich se-         amino-terminal domain. Schwarz et al. recently reported <br /> quences containing PPxY motif (45). WW domains 2 3               similar results human E6-AP (38). The hect domains of <br /> are likely involved mediating interaction       E3s shown interact specific, some <br /> with substrate(s) Rsp5 related essential function.       cases multiple, E2 enzymes (10, 22, 28). The specific yeast Ubc <br /> WW domain 2 absolutely required Rpb1 binding,             proteins capable activating Rsp5 known, by <br /> WW domain 3 contribute, C2 domain                 homology Arabidopsis Ubc8, efficiently activates <br /> WW domain 1 disrupted affecting Rpb1 bind-              Rsp5 vitro, yeast Ubc4 Ub5 best candidates, <br /> ing. Thus, determinants Rpb1 binding ubiquitina-           followed Ubc1 Ubc13. UBC1, UBC4, UBC5 con- <br /> tion correspond required rsp5-1 complementation,          stitute essential subset UBC genes (39, 40). The fact that <br /> and Rpb1 candidate        hect domain possesses determinants ubiq- <br /> substrates related essential function Rsp5.                  uitin-thioester formation supports two-domain model of <br />    The CTD Rpb1, consisting 26 copies heptapeptide          hect E3 function suggests hect domains, which, on <br /> repeat, binding site Rsp5 (17). The CTD essential        average, share approximately 40% similarity, inter- <br /> for Rpb1 activity vivo site interaction      changeable hect E3 proteins. This gener- <br /> components transcriptional machinery. The consensus             ally case, however, Schwarz et al. showed an <br /> binding site WW domains PPxY sequence,               E6-AP/hectH6 chimera, capable forming ubiq- <br /> certain substitutions proline permissible (5). The    uitin-thioester binding HPV E6 protein p53, did <br /> CTD heptapeptide sequence SPTSPSY                 ubiquitinate p53 (38). In addition, structure-function anal- <br /> direct recognition sequence WW domains Rsp5. The            yses E6-AP demonstrated determinants E6-de- <br /> function Rpb1 ubiquitination respect polII transcrip-       pendent p53 binding lie partially hect domain (16). <br />  VOL. 19, 1999                                                                                                  FUNCTIONAL DOMAINS OF Rsp5                          351 <br />  <br />  <br /> Thus, results obtained Rsp5 support simple                              uitin-protein ligase. Proc. Natl. Acad. Sci. USA 92:2563&#226;&#8364;&#8220;2567. <br /> two-domain model hect E3 function, results obtained                         15. Huibregtse, J. M., M. Scheffner, P. M. Howley. 1993. Cloning and <br />                                                                                          expression cDNA E6-AP, protein mediates interaction <br /> E6-AP suggest complex picture substrate recogni-                               human papillomavirus E6 oncoprotein p53. Mol. Cell. Biol. <br /> tion. Combined genetic biochemical analyses                                 13:775&#226;&#8364;&#8220;784. <br /> yeast hect E3s provide opportunities address                          16. Huibregtse, J. M., M. Scheffner, P. M. Howley. 1993. Localization the <br /> the question substrate recognition specificity.                                   E6-AP regions direct human papillomavirus E6 binding, association <br />                                                                                          p53, ubiquitination associated proteins. Mol. Cell. Biol. 13:4918&#226;&#8364;&#8220; <br />    The results presented support idea ubiq-                            4927. <br /> uitin-thioester intermediate requisite intermediate ubiq-                    17. Huibregtse, J. M., J. C. Yang, S. L. Beaudenon. 1997. The large subunit <br /> uitination reactions catalyzed hect E3 proteins. Three                          RNA polymerase II substrate Rsp5 ubiquitin-protein ligase. <br /> classes E3 activities identified, repre-                          Proc. Natl. Acad. Sci. USA 94:3656&#226;&#8364;&#8220;3661. <br />                                                                                      18. Imhof, M. O., D. P. McDonnell. 1996. Yeast RSP5 human <br /> sented Ubr1 yeast (2, 25), multimeric anaphase-                                homolog hRPF1 potentiate hormone-dependent activation transcription <br /> promoting complex (or cyclosome) (12), Cdc53/Cdc4/                               human progesterone glucocorticoid receptors. Mol. Cell. Biol. 16: <br /> Skp1 complex (12). None types E3 activities                             2594&#226;&#8364;&#8220;2605. <br /> been shown function ubiquitin-thioester intermediate,                       19. Johnson, E. S., P. C. M. Ma, I. M. Ota, A. Varshavsky. 1995. A proteo- <br />                                                                                          lytic pathway recognizes ubiquitin degradation signal. J. Biol. Chem. <br /> and presumed, therefore, E2 directly catalyzes sub-                           270:17442&#226;&#8364;&#8220;17456. <br /> strate ubiquitination. These E3 activities func-                       20. Kishino, T., M. Lalande, J. Wagstaff. 1997. UBE3A/E6-AP mutations <br /> tion providing surface simultaneous interaction E2                           cause Angelman syndrome. Nat. Genet. 15:70&#226;&#8364;&#8220;73. <br /> and substrate(s). Whether mechanism action                            21. Kumar, S., K. F. Harvey, M. Kinoshita, N. G. Copeland, M. Noda, N. A. <br />                                                                                          Jenkins. 1997. cDNA cloning, expression analysis, mapping mouse <br /> hect E3s rule exception E3 activities                             Nedd4 gene. Genomics 40:435&#226;&#8364;&#8220;443. <br /> awaits characterization potentially                       22. Kumar, S., W. H. Kao, P. M. Howley. 1997. Physical interaction between <br /> types E3 activities substrates.                                             specific E2 Hect E3 enzymes determines functional cooperativity. J. <br />                                                                                          Biol. Chem. 272:13548&#226;&#8364;&#8220;13554. <br />                                                                                      23. Lucero, P., R. Lagunas. 1997. Catabolite inactivation yeast mal- <br />                           ACKNOWLEDGMENTS                                                tose transporter requires ubiquitin-ligase npi1/rsp5 ubiquitin-hydrolase <br />   This work supported Public Health Service grant                                 npi2/doa4. FEMS Microbiol. Lett. 147:273&#226;&#8364;&#8220;277. <br /> CA72943-01 National Cancer Institute.                                       24. Macias, M. J., M. Hyvonen, E. Baralk, J. Schultz, M. Sudol, M. Saraste, and <br />                                                                                          H. Oschkinat. 1996. Structure WW domain kinase-associated <br />   We thank Sylvie Beaudenon helpful discussions, Mike                           protein complexed proline-rich peptide. Nature 382:646&#226;&#8364;&#8220;649. <br /> Kiledjian critical reading manuscript, Xin Li assis-              25. Madura, K., A. Varshavsky. 1994. Degradation Ga N-end rule <br /> tance tetrad dissections.                                                           pathway. Science 265:1454&#226;&#8364;&#8220;1458. <br />                                                                                      26. Matsuura, T., J. S. Sutcliffe, P. Fang, R.-J. Galjaard, Y.-H. Jiang, C. S. <br />                                    REFERENCES                                            Benton, J. M. Rommens, A. L. Beaudet. 1997. De novo truncating <br />  1. Albrecht, U., J. S. Sutcliffe, B. M. Cattanach, C. V. Beechey, D. Armstrong,         mutations E6-AP ubiquitin-protein ligase gene (UBE3A) Angelman <br />     G. Eichele, A. L. Beaudet. 1997. Imprinted expression murine              syndrome. Nat. Genet. 15:74&#226;&#8364;&#8220;77. <br />     Angelman syndrome gene, Ube3a, hippocampal Purkinje neurons.              27. Nefsky, B., D. Beach. 1996. Pub1 acts E6-AP-like protein ubiquitin <br />     Nat. Genet. 17:75&#226;&#8364;&#8220;78.                                                                ligase degradation cdc25. EMBO J. 15:1301&#226;&#8364;&#8220;1312. <br />  2. Bartel, B., I. Wu <br />                     &#194;&#168;nning, A. Varshavsky. 1990. The recognition component       28. Nuber, U., S. Schwarz, P. Kaiser, R. Schneider, M. Scheffner. 1996. <br />     N-end rule pathway. EMBO J. 9:3179&#226;&#8364;&#8220;3189.                                      Cloning human ubiquitin-conjugating enzymes UbcH6 UbcH7 (E2- <br />  2a.Beaudenon, S., J. M. Huibregtse. Unpublished data.                               F1) characterization interaction E6-AP RSP5. J. Biol. <br />  3. Bedford, M. T., D. C. Chan, P. Leder. 1997. FBP WW domains               Chem. 271:2795&#226;&#8364;&#8220;2800. <br />     Abl SH3 domain bind specific class proline-rich ligands. EMBO J.         29. Perry, W. L., C. M. Hustad, D. A. Swing, T. N. O&#226;&#8364;&#8482;Sullivan, N. A. Jenkins, and <br />     16:2376&#226;&#8364;&#8220;2383.                                                                        N. G. Copeland. 1998. The itchy locus encodes novel ubiquitin protein <br />  4. Bregman, D. B., R. Halaban, A. J. van Gool, K. A. Henning, E. C. Friedberg,          ligase disrupted a18H mice. Nat. Genet. 18:143&#226;&#8364;&#8220;146. <br />     S. L. Warren. 1996. UV-induced ubiquitination RNA polymerase II:          30. Pirozzi, G., S. J. McConnell, A. J. Uveges, J. M. Carter, A. B. Sparks, B. K. <br />     novel modification deficient Cockayne syndrome cells. Proc. Natl. Acad.         Kay, D. M. Fowlkes. 1997. Identification novel human WW domain- <br />     Sci. USA 93:11586&#226;&#8364;&#8220;11590.                                                             containing proteins cloning ligand targets. J. Biol. Chem. 272:14611&#226;&#8364;&#8220; <br />  5. Chen, H. I., A. Einbond, S.-J. Kwak, H. Linn, E. Koepf, S. Peterson, J. W.           14666. <br />     Kelly, M. Sudol. 1997. Characterization WW domain human            31. Plant, P. J., H. Yeger, O. Staub, P. Howard, D. Rotin. 1997. The C2 <br />     Yes-associated protein polyproline-containing ligands. J. Biol. Chem.        domain ubiquitin protein ligase Nedd4 mediates Ca21-dependent <br />     272:17070&#226;&#8364;&#8220;17077.                                                                     plasma membrane localization. J. Biol. Chem. 272:32329&#226;&#8364;&#8220;32336. <br />  6. la Fuente, N., A. M. Maldonado, F. Portillo. 1997. Glucose activation     32. Ratner, J. N., B. Balasubramanian, J. Corden, S. L. Warren, D. B. <br />     yeast plasma membrane H1-ATPase requires ubiquitin-protea-                Bregman. 1998. Ultraviolet radiation-induced ubiquitination proteaso- <br />     proteolytic pathway. FEBS Lett. 411:308&#226;&#8364;&#8220;312.                                    mal degradation large subunit RNA polymerase II. Implications for <br />  7. Einbond, A., M. Sudol. 1996. Towards prediction cognate complexes             transcription-coupled DNA repair. J. Biol. Chem. 273:5184&#226;&#8364;&#8220;5189. <br />     WW domain proline-rich ligands. FEBS Lett. 384:1&#226;&#8364;&#8220;8.              33. Rizo, J., T. C. Sudhof. 1998. C2-d <br /> </body></html>