<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Mar. 1999, p. 2389&#226;&#8364;&#8220;2399                                                                          Vol. 19, No. 3 <br /> 0270-7306/99/$04.0010 <br /> Copyright &#194;&#169; 1999, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />        NMD3 Encodes Essential Cytoplasmic Protein Required for <br />         Stable 60S Ribosomal Subunits Saccharomyces cerevisiae <br />                                    JENNIFER HEI-NGAM HO                AND    ARLEN W. JOHNSON* <br />                             Department Microbiology Institute Cellular Molecular Biology, <br />                                          The University Texas Austin, Austin, Texas <br />                     Received 8 September 1998/Returned modification 13 October 1998/Accepted 4 November 1998 <br />  <br />             A <span id='am-163' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-164' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-208' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span>mutation</span> NMD3 lethal absence XRN1, encodes major cytoplasmic <br />          exoribonuclease responsible mRNA turnover. Molecular genetic analysis NMD3 revealed an <br />          essential gene required stable 60S ribosomal subunits. Cells bearing temperature-sensitive allele of <br />          NMD3 decreased levels 60S subunits nonpermissive temperature resulted formation <br />          half-mer polysomes. Pulse-chase analysis rRNA biogenesis indicated 25S rRNA and <br />          processed kinetics similar wild-type kinetics. However, mature RNA rapidly degraded, a <br />          half-life 4 min. Nmd3p fractionated cytoplasmic protein sedimented position free 60S <br />          subunits sucrose gradients. These results suggest Nmd3p cytoplasmic factor required late <br />          cytoplasmic assembly step 60S subunit ribosomal protein. Putative orthologs Nmd3p exist <br />          Drosophila, nematodes, archaebacteria eubacteria. The Nmd3 protein sequence does not <br />          contain readily recognizable motifs known function. However, proteins amino-terminal <br />          domain containing repeats Cx2C, reminiscent zinc-binding proteins, implicated nucleic acid <br />          binding protein oligomerization. <br />  <br />  <br />    A screen mutations lethal yeast cells lacking            indicating yeast pathway dispensable normal <br /> the major cytoplasmic exoribonuclease, Xrn1p, identified mu-                growth. <br /> tations SKI2 SKI3 additional complementation                    Dominant mutants NMD3 (also referred SRC5) (30) <br /> group (27). ski6 ski8 mutations synthetic lethal               identified suppressors growth defect of <br /> with xrn1D (1, 9). Although superkiller (SKI) genes                temperature-sensitive mutation QSR1/GRC5 (31), encod- <br /> initially described host antiviral genes repress expres-            ing large ribosomal subunit protein identified L10 the <br /> sion killer toxin encoded endogenous yeast RNA virus               current ribosomal protein nomenclature Mager et al. (40) or <br /> (58; reviewed reference 63), recently                L7 nomenclature Zinker Warner (68). L10 is <br /> shown play general role cell (27), repress trans-            thought exchangeable ribosomal protein (34) may <br /> lation poly(A)2 RNA (7, 43, 64), required                  added large subunit late cytoplasmic assembly <br /> normal mRNA 39-exonucleolytic degradation (1). SKI6 en-                     step (13). The genetic interaction NMD3 QSR1 <br /> codes essential 39359 exonuclease component                 suggests role NMD3 ribosome function synthesis. <br /> the exosome (45), ski6 mutants defects assembly                 Eukaryotic ribosome biogenesis complex process occur- <br /> 60S ribosomal subunits (7). We cloned wild-type                ring largely nucleolus, rRNA transcribed, mod- <br /> gene complementation group identified                   ified, processed assembly approximately 80 <br /> previous XRN1 synthetic lethal screen (27) shown                   ribosomal proteins mature ribosomal subunits. In Saccha- <br /> that NMD3.                                                            romyces cerevisiae, (25S, 5.8S, 18S) four <br />    NMD3 (nonsense-mediated decay) previously identified                 rRNAs transcribed RNA polymerase I single 35S <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> two-hybrid screen proteins interact                precursor nucleolus, fourth rRNA, 5S <br /> nonsense-mediated decay factor Upf1p (22). Nonsense-medi-                   rRNA, transcribed independently RNA polymerase III <br /> ated decay yeast cytoplasmic pathway rapid                  (66). The pre-rRNA undergoes sequential cleavages and <br /> elimination aberrant transcripts containing premature stop               trimming generate mature rRNAs. Most ribo- <br /> codons (reviewed reference 25). Homologous pathways                  somal proteins assemble subunits nucleolus, <br /> found nematodes (49) mammalian cells (5; reviewed                 proteins, including L10, exchangeable and <br /> reference 41). Nonsense-mediated decay involves recogni-                added transport subunits cytoplasm. <br /> tion premature stop codon translating ribosome                 rRNA processing assembly ribosomal proteins onto <br /> is thought activate scanning complex recognizes              rRNAs intimately linked processes. Thus, mutations in <br /> downstream sequence elements, leading rapid decapping                 ribosomal proteins factors required assembly typically <br /> the transcript (12, 50). In yeast, pathway depends UPF1,             lead production defective subunits potentially <br /> UPF2/NMD2, UPF3 (10, 22, 36&#226;&#8364;&#8220;38). Deletion of                     alter specificity translation. Such defects lead the <br /> these genes prevents nonsense-mediated decay, stabilizing              production unstable subunits, resulting imbalance be- <br /> otherwise unstable mRNA containing premature nonsense                       tween 40S 60S (51, 67), directly affect ribosome function <br /> codons. However, mutants display growth defects,                   resulting subunit imbalance (14). In paper, we <br />                                                                             identification functional analysis NMD3, an <br />    * Corresponding author. Mailing address: Department Microbi-          essential gene S. cerevisiae. While initial identification <br /> ology, The University Texas Austin, Austin, TX 78712-1095.            nmd3 mutant synthetic lethal screen xrn1 <br /> Phone: (512) 475-6350. Fax: (512) 471-7088. E-mail: arlen@mail              mutant previous implication nonsense-mediated de- <br /> .utexas.edu.                                                                cay suggested role mRNA turnover, results suggest <br />  <br />                                                                      2389 <br />  2390        HO AND JOHNSON                                                                                                                   MOL. CELL. BIOL. <br />  <br />  <br />                                                       TABLE 1. S. cerevisiae strains used study <br />    Strain                                                     Genotype                                                 Plasmid               Reference source <br />  <br /> AJY362                    MATa/a leu2/leu2 ura3/ura3 trp1/trp1 L-A-o/L-A-o <br /> AJY384                    MATa/a leu2/leu2 ura3/ura3 trp1/trp1 L-A-o/L-A-o NMD3/nmd3::TRP1 <br /> AJY404                    MATa ade2 ade3 leu2 his3 ura3-52 xrn1D NMD3-LEU2-NMD3 <br /> AJY483                    MATa ade2 ade3 his3D leu2 lys2-801 trp1D63 ura3-52 <br /> AJY529                    MATa his3D leu2 trp1 ura3-52 nmd3::TRP1                                                    pAJ112 <br /> AJY543                    MATa ade2 ade3 leu2 lys2-801 ura3-52 nmd3-1                                                                             27 <br /> AJY590                    MATa his3 leu2 trp1 ura3-52 nmd3::TRP1                                                     pAJ124 <br /> AJY592                    MATa his3 leu2 trp1 ura3-52 nmd3::TRP1                                                     pAJ123 <br /> AJY596                    MATa his3 leu2 trp1 ura3-52 nmd3::TRP1                                                     pAJ129 <br /> AJY717                    MATa ade2 ade3 leu2 lys2-801 ura3-52 nmd3-4 r2 <br /> AJY734                    MATa ade2 ade3 leu2 lys2-801 ura3-52 nmd3-4 <br /> AJY735                    MATa ade2 ade3 leu2 lys2-801 ura3-52 <br /> CH1305                    MATa ade2 ade3 leu2 lys2-801 ura3-52                                                                                    32 <br /> RDKY1977                  MATa ade2 ade3 leu2 lys2-801 ura3-52 xrn1D <br /> RDKY1978                  MATa ade2 ade3 leu2 his3 ura3-52 xrn1::URA3                                                                             27 <br /> RDKY1979                  MATa ade2 ade3 leu2 his3 ura3-52 xrn1D                                                                                  27 <br /> RDKY1997                  MATa leu2 ura3 trp1 L-A-o                                                                                               27 <br /> RDKY2037                  MATa leu2 ura3 trp1 L-A-o <br /> RDKY2050                  MATa ade2 ade3 leu2 lys2-801 ura3-52 nmd3-1                                                pRDK297                      27 <br /> YAS398                    MATa ura3 trp1 his3 leu2 spb2::LEU2                                                                                     A. Sachs <br />  <br />  <br />  <br /> that NMD3 required late cytoplasmic assembly step                      pAJ78 pAJ113. The NMD3-containing NcoI-<span id='am-6' about='protege:TO' typeof='owl:Thing'>to</span>-HindIII fragment <br /> 60S ribosomal subunit biogenesis.                                                  pAJ113 ligated corresponding sites pRDK306 (47) give <br />                                                                                    pAJ118. Finally, oligonucleotide AJO148 (59-CATGGAACAAAAGCTTATTT <br />                                                                                    CTGAAGAAGACTTGAA), encoding c-Myc epitope, complemen- <br />                        MATERIALS AND METHODS                                       tary oligonucleotide ligated NcoI site start codon NMD3 <br />                                                                                    pAJ118 pAJ234. pAJ153, encoding c-Myc-tagged NMD3 expressed <br />    Strains, media, genetic methods. Strains used study listed   promoter, constructed follows. The wild-type NMD3 pro- <br /> Table 1. Rich medium (YPD), 5-fluoroorotic acid (5FOA), synthetic complete         moter 59 sequences amplified pAJ78 using M13 reverse <br /> (SC) dropout media, standard yeast manipulations described else-       sequencing primer oligonucleotide AJO168 (59-GGTAACGGTACCATGG <br /> where (28). Yeast transformations carried lithium acetate          TTTTGTCAAATTCCTCAACG). The resulting PCR product digested with <br /> method previously described (19) cells transformed       NcoI SmaI ligated c-Myc-NMD3-containing NcoI-to-BglII frag- <br /> grown light lawns plates.                                                    ment pAJ234 BglII-to-SmaI vector pAJ123. The resulting construct <br />    Cloning NMD3 DNA manipulation. Strain RDKY2050, identified          complemented temperature sensitivity nmd3-4 mutant. <br /> previous screen (27), transformed centromeric LEU2 library contain-        nmd3 mutants. The genomic nmd3-1 mutation rescued plasmid pAJ123 <br /> ing 9- 12-kbp inserts. Leu1 transformants replica plated 5FOA plates,   vivo recombination transforming strain AJY543 pAJ123 digested <br /> and 5FOA-resistant colonies restruck 5FOA plates. To identify          SnaBI HindIII, resulting plasmid sequenced. nmd3-1 <br /> eliminate wild-type XRN1 clones, 5FOA-resistant clones analyzed PCR        sequence confirmed directly sequencing genomic locus amplified by <br /> using primers distinguish wild-type XRN1 genomic xrn1 deletion         PCR. <br /> (xrn1D). Plasmid DNA complementing clone XRN1               PCR mutagenesis used create random temperature-sensitive mutations <br /> subcloned Sau3AI partial digest YEp351 described previously (27).     NMD3. Oligonucleotide primers AJO108 AJO109 (see above) used <br /> The insert pAJ78, containing smallest complementing subclone, se-       PCR amplification NMD3. The PCR product digested NcoI and <br /> quenced (Sequenase dideoxy sequencing kit; Amersham) contain          cotransformed AJY529 pAJ123 digested MscI BglII to <br /> one single complete open reading frame encoding NMD3.                              remove NMD3 coding sequence nt 245 1143. Leu1 transformants were <br />    nmd3::TRP1 disruption. The TRP1-containing BstUI fragment pRS424           replica plated 5FOA plates 37 26&#194;&#176;C. From approximately 2,000 <br /> was ligated pAJ78 digested MscI BglII blunt        transformants, temperature-sensitive mutations, nmd3-2, nmd3-3, and <br /> ended T4 DNA polymerase. This gave TRP1 disruption NMD3             nmd3-4, obtained. All nmd3 temperature-sensitive mutants behaved <br /> nucleotides (nt) 248 1146 create pAJ81. The L-A double-stranded RNA          similarly, nmd3-4 used temperature shift experiments. pAJ124 <br /> virus-deficient diploid, AJY362, mating RDKY1997 and <br /> RDKY2037. The nmd3::TRP1-containing NheI-to-SpeI fragment pAJ81 was <br /> transformed AJY362. Trp1 transformants analyzed PCR het- <br /> erozygotes containing correct integration nmd3::TRP1 allele the <br /> NMD3 locus AJY384.                                                                          TABLE 2. Plasmids used study <br />    Genomic NMD3-LEU2. An NMD3-containing SmaI-to-NheI fragment from <br /> pAJ78 ligated LEU2-containing yeast integrating plasmid pRS305                                                                            Source or <br />                                                                                     Plasmids                     Relevant markers <br /> digested SmaI XbaI. The resulting plasmid (pAJ92) digested                                                                           reference <br /> BamHI, cuts NMD3 coding sequence, integrated the <br /> genomic NMD3 locus RDKY1979 (xrn1D) transformation strain <br />                                                                                    pAJ78               2mm LEU2 NMD3 <br /> AJY404. The resulting NMD3 duplication intervening LEU2 gene           pAJ81               2mm LEU2 nmd3::TRP1 <br /> confirmed Southern blotting.                                                    pAJ92               LEU2 NMD3 integrating vector <br />    Plasmids. Table 2 describes plasmids used study. All nucleotide     pAJ112              2mm His6-NMD3(PCR) URA3 <br /> numbering relative translation start, defined 11 respective    pAJ113              2mm His6-NMD3(WT) URA3 <br /> gene. pAJ123 constructed ligating NMD3-containing SmaI-to-SalI          pAJ118              2mm LEU2 GAL10::NMD3 <br /> fragment pAJ78 respective sites pRS315. pAJ112, expressing        pAJ123              CEN LEU2 NMD3 <br /> His6-tagged NMD3, constructed follows. Oligonucleotide primers AJO108       pAJ124              CEN LEU2 nmd3-2 <br /> (59-CTAGTCTAGACTCGAGAAAATGCATCATCATCATCATCATTCCAT <br /> GGAATTCACACCTATAGATCC), encoding His6 tag containing an <br />                                                                                    pAJ129              CEN LEU2 nmd3-4 <br /> XhoI site, AJO109 (59-GCGCAAGCTTGAGTATATACTACTCTCC),                           pAJ150              URA3 nmd3-4 integrating vector <br /> containing HindIII site, used PCR amplify NMD3. The XhoI-            pAJ153              CEN LEU2 c-Myc-NMD3 <br /> HindIII-digested PCR product ligated XhoI- HindIII-digested      pAJ234              2mm LEU2 GAL10::c-Myc-NMD3 <br /> pVT102U pAJ112.                                                            pRDK306             YEp351 GAL10::XRN1 <br />    Epitope-tagged NMD3. pAJ234, containing galactose-inducible c-Myc-tagged      pRP485              GAL1::MFA2 URA3                                 R. Parker <br /> NMD3, created follows. The PCR-amplified NMD3 coding sequence            pVT102U             2mm URA3                                        60 <br /> pAJ112 replaced NMD3-containing EcoRI-to-HindIII fragment <br />  VOL. 19, 1999                                                                         NMD3 IS REQUIRED FOR STABLE 60S SUBUNITS                                2391 <br />  <br />  <br /> (bearing nmd3-2) pAJ129 (bearing nmd3-4) transformed AJY529,           extracts prepared buffer C containing 30 mM EDTA lacking MgCl2 <br /> replacing pAJ112 strains AJY590 AJY596 respectively. pAJ150,             cycloheximide. In addition, gradients prepared 30 mM EDTA and <br /> containing nmd3-4 URA3 integrating plasmid, created ligating         MgCl2, centrifugation 5 h. Sucrose gradients analyzed by <br /> nmd3-4-containing NcoI-to-SalI fragment pAJ129 respective sites        continuous monitoring A254 Pharmacia Uvicord monitor. <br /> of pRS406. pAJ150 digested BglII transformed wild-type            Yeast cell fractionation Western blotting. Yeast cells fractionated <br /> strain CH1305 integrate pAJ150 NMD3 genomic locus. Ura1 trans-           nuclear cytoplasmic fractions described previously (65). Proteins <br /> formants patched 5FOA plates select loss            cytoplasmic nuclear fractions sucrose density gradients were <br /> intervening URA3 sequence copy NMD3 gene. 5FOA-resistant              analyzed Western blotting described previously (26), using anti-c-Myc <br /> clones scored temperature sensitivity, loss integrated      monoclonal antibody 1:5,000 primary antibody detect c-Myc-tagged <br /> plasmid confirmed PCR genomic NMD3 locus. Since temper-            Nmd3p. Rabbit anti-topoisomerase II (Topo II) rabbit anti-glucose-6-phos- <br /> ature-sensitive isolates petite, strain (AJY717) bearing nmd3-4        phate dehydrogenase (G6PDH) polyclonal antibodies diluted 1:7,500 and <br /> crossed wild-type grande (AJY483). The resulting heterozygous diploid       used detect nuclear cytoplasmic protein markers, respectively. Rabbit <br /> sporulated, resulting tetrads dissected. All spore clones      polyclonal anti-Topo II antibody gift J. Lindsley J. Wang, <br /> cross grandes. AJY734 (nmd3-4) AJY735 (wild type)             anti-G6PDH antibody obtained Sigma, mouse monoclonal anti-c- <br /> tetrad used study.                                                  Myc derived 9E10.2 cells (American Type Culture Collection, Rock- <br />    Northern blot analysis. For CYH2 steady-state mRNA analysis, 10-ml cultures       ville, Md.). <br /> were grown SC dropout medium density approximately 107 cells/ml at <br /> 26&#194;&#176;C shifted 37&#194;&#176;C 2 h. Cells harvested RNA was <br /> prepared analyzed Northern blotting previously described (26). For                                         RESULTS <br /> time course experiments thiolutin, 40-ml cultures grown YPD at <br /> 26&#194;&#176;C shifted 37&#194;&#176;C. After 1.75 h 37&#194;&#176;C, cells collected            Identification nmd3-1 synthetically lethal xrn1 <br /> centrifugation resuspended 10 ml YPD 37&#194;&#176;C. After 30 min         deletion. A previous screen mutations synthetically lethal <br /> at 37&#194;&#176;C, thiolutin added 3 mg/ml (final concentration). Immediately after <br /> addition thiolutin (time zero) various times afterward, 1.5-ml aliquots <br />                                                                                      xrn1D identified recessive mutations complemen- <br /> were removed, cells collected centrifugation, cell pellets placed   tation groups. The wild-type genes were <br /> on dry ice. RNA prepared analyzed Northern blotting. For MFA2             cloned identified SKI2 SKI3 (27). The wild-type <br /> Northern analysis, 25-ml cultures grown 26&#194;&#176;C SC Ura2 medium               gene complementation group cloned a <br /> containing 2% raffinose density approximately 107 cells/ml then <br /> shifted 37&#194;&#176;C 2 h. Transcription induced adding galactose 2%. <br />                                                                                      centromeric LEU2 plasmid library complementation and <br /> After 10 min, glucose added 2% repress transcription. At various times,    shown NMD3. NMD3 previously two- <br /> cells collected centrifugation cell pellets placed dry ice.      hybrid screen proteins interact Upf1p (22). Mu- <br /> RNA prepared analyzed Northern blotting using oligonucleotide             tants UPF1 gene, originally isolated basis their <br /> AJO101 (59-GATCAGGAATTCCCCCCCCCCCCCCCCCCAAATTCCTA), <br /> specific poly(G) insertion MFA2 mRNA. <br />                                                                                      ability enhance suppression <span id='am-85' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-86' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-156' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span>frameshift mutation</span> <br />    For Northern analysis pre-rRNA steady-state levels, total RNAs            led premature translational termination (11), selectively <br /> strains AJY734 AJY735 prepared shift 37&#194;&#176;C 1, 3, 6 h,     stabilize mRNAs containing early nonsense mutations. <br /> then separated formaldehyde-agarose gel, transferred Zeta-probe           To confirm mutation complementation <br /> blotting membrane (Bio-Rad) described above. Membranes probed with <br />                                                                                      group allelic NMD3, genetic linkage assayed. <br /> the following 32P-radiolabeled oligonucleotide probes: AJO215 (59-GGTCTCT <br /> CTGCTGCCGGAAATG), specific 5&#226;&#8364;&#8482; external transcribed spacer                     RDKY2050 (xrn1D putatively nmd3-1) containing <br /> (59ETS); AJO130 (59-TCTTGCCCAGTAAAAGCTCTCATG), specific in-                       pRDK297 (XRN1 URA3) mated AJY404 (xrn1D <br /> ternal transcribed spacer 1 (ITS1); AJO214 (59-GTTCGCCTAGACGCTCTCT                   NMD3::LEU2::NMD3), diploids sporulated, re- <br /> TC), specific ITS2; AJO213 (59-CCACTTAGAAAGAAATAAAAAACAAA                         sulting tetrads dissected. All tetrads containing via- <br /> TCAG), complementary 9 nt 39 end 25S plus 22 nt 39ETS; and <br /> AJO216 (59-CCCGGATCATAGAATTCTTAAG), complementary 39ETS,                          ble spores (17 tetrads) showed 2:2 segregation synthetic <br /> downstream RNase III cleavage site.                                           lethality, scored 5FOA sensitive (data shown). No <br />    Pulse-chase analysis pre-rRNA processing. Labeling [methyl-3H]me-         5FOA-sensitive colonies Leu1, indicating recombina- <br /> thionine carried described previously (67). Briefly, 5-ml cultures   tion putative nmd3-1 allele LEU2-marked <br /> grown SC Met2 medium density approximately 0.6 3 107 cells/ml at <br /> room temperature. Cells harvested centrifugation, resuspended 1.5 ml      NMD3 locus, confirming original mutation in <br /> of SC Met2 medium prewarmed 37&#194;&#176;C, grown 37&#194;&#176;C 2 h. Cells           NMD3. <br /> then labeled addition 150 mCi [methyl-3H]methionine. After labeling        NMD3 essential gene. A heterozygous diploid contain- <br /> for 2 min, 150 mg unlabeled methionine added chase. At various           ing nmd3::TRP1 allele constructed genetic back- <br /> times, samples taken quickly frozen dry ice. <br />    A similar procedure followed pulse-chase experiments using [3H]uracil     ground lacking double-stranded RNA virus, L-A. Upon <br /> except strains used transformed pRS316 allow growth         sporulation tetrad dissection, Trp1 spore clones were <br /> SC Ura2 medium. Cells shifted 37&#194;&#176;C 2 h 3 ml SC Ura2 medium,        obtained 37 tetrads ascus contained two <br /> pulse-labeled 150 mCi [3H]uracil 3 min, chased 900 mg        dead spores (data shown), indicating NMD3 essen- <br /> unlabeled uracil. RNAs prepared analyzed electrophoresis through <br /> formaldehyde-agarose gels (6% formaldehyde, 1.2% agarose) described else-         tial. To confirm essentiality NMD3, LEU2 <br /> where (28) electrophoresis urea-polyacrylamide gels (6% polyacryl-         NMD3 plasmid transformed NMD3 heterozygous <br /> amide, 8.2 M urea). To detect labeled RNAs, gels soaked Enlightning      diploid sporulation. This plasmid rescued inviability <br /> (New England Nuclear), dried, subjected autoradiography.                      Trp1 spore clones allowed recovery tetrads con- <br />    Drug sensitivity tests. Fresh single colonies plates resuspended in <br /> sterile distilled water plated YPD plates. After cell suspension      taining viable spores (data shown). All Trp1 spore <br /> soaked plate, 0.25-in. sterile filters (Schleicher &amp; Schuell catalog no.    clones Leu1. Because NMD3 essential, original <br /> 740-E) placed plates, indicated antibiotic       recessive allele, nmd3-1, identified synthetically lethal with <br /> spotted filter. Plates incubated room temperature 33&#194;&#176;C.       xrn1D, hypomorphic, partially functional, allele. <br />    Sucrose density gradients. For polysome preparation, yeast cultures were <br /> grown density 0.6 3 107 cells/ml 500 ml YPD 26&#194;&#176;C. One half of <br />                                                                                         Sequence analysis wild-type mutant NMD3. NMD3 <br /> the culture removed shifted 37&#194;&#176;C 2 h. Just cells      encodes 59.1-kDa protein homologous hypothetical <br /> harvested, cycloheximide added final concentration 100 mg/ml. The        proteins Caenorhabditis elegans Drosophila melano- <br /> cells harvested, washed 25 ml ice-cold buffer C (10 mM             gaster (Fig. 1) Schizosaccharomyces pombe (data not <br /> Tris-HCl [pH 7.4], 100 mM NaCl, 30 mM MgCl2, cycloheximide [50 mg/ml], <br /> heparin [200 mg/ml]), resuspended 1 ml ice-cold buffer C; 300 ml cold <br />                                                                                      shown), suggesting conservation NMD3 function throughout <br /> glass beads (400-mm diameter; Sigma Chemical Co.) added, sus-            eukaryotes. In addition, apparent homologs, corresponding to <br /> pension vortexed times, consisting 30 s vortexing followed     amino-terminal half Nmd3p, predicted genome <br /> 1 min cooling ice. Extracts clarified centrifugation, 30 A260      sequencing archaebacteria Methanococcus jannaschii <br /> units cell lysate loaded 12-ml linear 7 47% sucrose gradient <br /> and centrifuged 2.5 h 40,000 rpm Beckman SW40 rotor 4&#194;&#176;C as <br />                                                                                      Methanobacterium thermoautotrophicum. Although this <br /> described (2).                                                             protein family does contain readily identifiable protein <br />    Analysis dissociated ribosomal subunits described     motifs related proteins known function, does contain a <br />  2392       HO AND JOHNSON                                                                                                                         MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 1. Sequence analysis Nmd3p. (A) Expected domain structure Nmd3p based sequence alignment amino acid composition. Numbers indicate amino <br /> acid residues Nmd3p. The acidic C terminus (residues 470 518) lacking nmd3-1 mutant protein. (B) Multiple sequence alignment highly <br /> conserved region amino-terminal domain Nmd3p eukaryotic archaeal homologs. Positions $80% identity shaded black, positions $80% <br /> similarity shaded gray. Bars indicate conserved Cx2C repeats. Numbers parentheses indicate sizes full-<span id='am-5' about='xsp:length' typeof='owl:Thing'>length</span> proteins. Proteins accession numbers: <br /> ScNmd3p, Nmd3p S. cerevisiae; Ce, hypothetical protein C. elegans (GenBank 1301731); Dm, hypothetical protein D. melanogaster (GenBank 2661560); <br /> Mj, hypothetical protein M. jannaschii (GenBank 150051); Mt, hypothetical protein M. thermoautotrophicum (GenBank 2622898). Alignments were <br /> performed using CLUSTALW http://transfac.gbf-braunschweig.de/dresearch/clustalw.html. Shading MacBoxshade, obtained http://www.isre- <br /> c.isb-sib.ch/sib-isrec/boxshade/. <br />  <br />  <br />  <br /> highly conserved cysteine repeat motif indicated Fig. 1.                           tein, informative protein structure func- <br /> These cysteine repeats reminiscent zinc-binding motifs                         tion. With exception synthetic lethality xrn1D, the <br /> of RING fingers, LIM domains (39), type IV fingers (57).                          nmd3-1 temperature-sensitive mutants behaved similarly <br /> Determination motif novel zinc-binding                           gave rise reduced levels 60S subunits resulting <br /> motif require biochemical analysis.                                      half-mers, sensitivity translation elongation inhibitors, <br />    NMD3 identified screen proteins                           obvious defects mRNA turnover (see below; also <br /> that interact Upf1p (6, 22). The domain Nmd3p re-                             data shown). <br /> sponsible interaction C-terminal 120 amino                              nmd3 mutants display significant defects mRNA <br /> acids (6). This region highly hydrophilic conserved                         turnover. NMD3 identified genetic screens for <br /> among eukaryotic homologs Nmd3p present                          mRNA turnover factors: synthetic lethality xrn1D a <br /> the archaeal proteins (Fig. 1A). Interestingly nmd3-1 mu-                         two-hybrid screen UPF1. Thus, likely nmd3 <br /> tation single nucleotide insertion adenosine run                      mutants display defects mRNA turnover, especially <br /> of adenosines nt 1399 1403. This insertion results                       RNAs nonsense-mediated decay pathway. The tran- <br /> frameshift highly acidic C-terminal 51 amino acids                       script CYH2 gene inefficiently spliced yeast, and <br /> (40% glutamate aspartate) truncated (Fig. 1A). This                           pre-mRNA rapidly degraded cytoplasm non- <br /> deletion domain interacts Upf1p                           sense-mediated decay. The effects temperature-sensitive <br /> thus disrupt interaction. We sequenced                            nmd3-2 nmd3-4 mutants pre-CYH2 degradation were <br /> temperature-sensitive mutants generated PCR (see Mate-                             assessed. Cultures shifted nonpermissive temper- <br /> rials Methods), containing multiple mutations leading                         ature 2 h, time thiolutin added inhibit RNA <br /> to multiple amino acid changes: nmd3-2 (Ile55Thr Ala122Thr                            polymerase II transcription. In experiment, nmd3 mu- <br /> Pro166Ser Leu296Pro), nmd3-3 (Thr105Ser Leu174Phe                                     tant strains showed two- fourfold-higher levels pre-CYH2 <br /> Asp195Gly Val258Asp), nmd3-4 (Tyr136Asn Thr158Ala                                 time thiolutin addition, suggesting steady- <br /> Leu214Trp Ser329Gly Arg441Cys Glu499Gly). Because                              state levels pre-CYH2 slightly elevated (Fig. 2A). A <br /> number changes distribution pro-                          similar elevation ratio pre-CYH2 mature CYH2 was <br />  VOL. 19, 1999                                                                        NMD3 IS REQUIRED FOR STABLE 60S SUBUNITS                             2393 <br />  <br />  <br />  <br />  <br />    FIG. 2. mRNA stability. (A) Strains containing wild-type (WT) NMD3 <br /> (AJY592) temperature-sensitive alleles nmd3-2 (AJY590) nmd3-4 <br /> (AJY596) grown 26&#194;&#176;C shifted 37&#194;&#176;C 2 h, time <br /> thiolutin added. Cells harvested indicated times thiolutin <br /> addition, total RNA prepared, 10 mg RNA analyzed each <br /> sample Northern blotting pre-CYH2 mature CYH2. (B) NMD3 wild- <br /> type temperature-sensitive strains AJY592 AJY590 containing <br /> GAL1::MFA2 plasmid grown 26&#194;&#176;C SC Ura2 medium containing <br /> 2% raffinose shifted 37&#194;&#176;C 2 h, time galactose added to <br /> 2% (final concentration). After 10 min, glucose added 2%. Cells were <br /> harvested indicated times relative addition glucose, total RNA <br /> was prepared, 10 mg RNA analyzed sample Northern <br /> blotting MFA2. <br />  <br />  <br />                                                                                        FIG. 3. Drug diffusion assay. Filters containing indicated amounts of <br />                                                                                     antibiotic placed freshly spread lawns cells YPD plates and <br /> also seen steady-state experiments (data shown). How-                        incubated 33&#194;&#176;C 2 days. Strains used CH1305 (wild type [WT]; <br /> ever, significant difference rates decay                  NMD3), AJY734 (nmd3-4), AJY543 (nmd3-1), RDKY1977 (xrn1D). <br /> pre-CYH2 mature CYH2. The half-lives pre-CYH2 in <br /> wild-type mutant cells 4 5 min, respec- <br /> tively, mature CYH2 cells 9.5 10.5 min,                       elongation yeast cells, mutations affect <br /> respectively. In control experiments, xrn1 null mutant                           translational fidelity confer increased sensitivity these <br /> showed significant stabilization pre-CYH2 (data shown).                      drugs (21, 44, 53, 54). Cycloheximide inhibits translation <br /> The degradation MFA2 mRNA transcription pulse-                              direct interaction large ribosomal subunit, leading <br /> chase experiment examined (Fig. 2B). After growth                          inhibition translational elongation peptidyl <br /> for 2 h 37&#194;&#176;C, transcription galactose-inducible MFA2                        transfer reaction (29). As shown Fig. 3C, nmd3-1 mutant <br /> construct transiently induced galactose followed                        displayed resistance cycloheximide compared of <br /> inhibition transcription glucose. In experiment,                   wild-type cells, xrn1 mutants similar wild-type <br /> was discernible difference mutant wild type.                         cells assay (data shown). The sensitivity nmd3 <br /> Similar results obtained galactose-inducible wild-                       mutants paromomycin hygromycin B resistance to <br /> type nonsense codon-containing MATa1 transcript                           cycloheximide suggest role NMD3 translation and <br /> partially functional mutant nmd3-1 (data shown). Thus,                          ribosome function. <br /> NMD3 does appear general role mRNA                                    nmd3 mutants display decreased levels 60S ribosomal <br /> degradation.                                                                        subunits half-mer polysomes. The role NMD3 trans- <br />    Effects translation inhibitory drugs nmd3 mutants.                         lation analyzed directly examining effects of <br /> Since nmd3 mutants did significant defects mRNA                         nmd3 mutation polysome profiles. Polysomes pre- <br /> degradation, examined effects translation,                     pared wild-type temperature-sensitive nmd3-4 mu- <br /> assaying sensitivity various antibiotics inhibit trans-                 tant cells grown 26 37&#194;&#176;C analyzed ultracentrifu- <br /> lation. Figure 3 shows disk diffusion assay drug sensitivity                  gation sucrose density gradients. At permissive <br /> on plates. In assay, strains carrying nmd3-1                      temperature, profiles nmd3-4 mutant (Fig. 4C) the <br /> temperature-sensitive allele nmd3-4 displayed sensitivity                        wild type (Fig. 4A) similar, 2 h 37&#194;&#176;C, the <br /> paromomycin (Fig. 3A) hypersensitivity hygromycin B                          mutant showed deficit free 60S relative free 40S ribo- <br /> (Fig. 3B) relative wild-type strains. The light halo seen                    somal subunits, overall decrease average number of <br /> the wild type presence paromomycin zone slow                      ribosomes polysomes, appearance half-mers (Fig. <br /> growth distinct zone total growth inhibition seen                    4D). Similar results obtained hypomorphic allele <br /> the nmd3-4 strain. Interestingly, xrn1D mutant showed hy-                        nmd3-1 grown 30&#194;&#176;C (data shown). The presence of <br /> persensitivity hygromycin B sensitive paro-                       half-mers, contain 43S initiation complexes stalled the <br /> momycin concentrations used (Fig. 3A B). Paromo-                         initiator AUG, indicative defect late step the <br /> mycin hygromycin B increase translational error rate                        translation initiation pathway (18). <br />  2394       HO AND JOHNSON                                                                                                              MOL. CELL. BIOL. <br />  <br />  <br />                                                                                     ing Mg21 30 mM EDTA analyzed sucrose <br />                                                                                     gradients. Under conditions, nmd3-4 mutant showed <br />                                                                                     significantly decreased total 60S levels relative 40S 2 h <br />                                                                                     nonpermissive temperature (Fig. 5D) compared the <br />                                                                                     wild type (Fig. 5A B) nmd3-4 mutant 26&#194;&#176;C (Fig. <br />                                                                                     5C). Quantitation peaks Fig. 5 yielded 60S-to-40S ratio <br />                                                                                     1.2 mutant 37&#194;&#176;C, compared 1.7 mutant <br />                                                                                     26&#194;&#176;C wild type 37&#194;&#176;C. The ratio 40S mutant <br />                                                                                     37&#194;&#176;C versus wild type 37&#194;&#176;C 0.94. Thus, 2 h at <br />                                                                                     nonpermissive temperature, 25% reduction in <br />                                                                                     total 60S subunits little change 40S levels. In time <br />                                                                                     course experiment, half-mers formed a <br />                                                                                     measurable decrease 60S subunits (data shown). These <br />                                                                                     data indicate nmd3 mutant, subunit joining se is <br />                                                                                     defective appearance half-mers resulted from <br />                                                                                     depletion 60S levels. <br />                                                                                        Because nmd3 mutants reduced levels 60S subunits <br />                                                                                     nmd3-1 synthetically lethal xrn1D, possible <br />                                                                                     mutation resulted reduction 60S subunits <br />                                                                                     synthetically lethal xrn1D. This tested ge- <br />                                                                                     netically crossing spb2 mutant (YAS398) xrn1D <br />                                                                                     mutant (RDKY1978) determine double mutant was <br />                                                                                     viable. SPB2 encodes ribosomal protein L46, mutations in <br />                                                                                     SPB2 lead reduced levels 60S subunits (51). The xrn1D <br />                                                                                     spb2D double mutants viable did grow apprecia- <br />                                                                                     bly slowly xrn1D single mutants (data shown). <br />                                                                                     Thus, nmd3-1 synthetic lethality xrn1D due <br />                                                                                     simply reduced 60S levels. <br />                                                                                        Nmd3p required stable 25S rRNA. Defects ribo- <br />                                                                                     somal subunit levels mutations nonribosomal pro- <br />                                                                                     teins typically suggest problems rRNA processing and/or <br />    FIG. 4. Polyribosome profiles nmd3-4 mutants restrictive temperature. <br /> Extracts strains AJY735 (NMD3) (A B) AJY734 (nmd3-4) (C and <br />                                                                                     ribosomal subunit assembly. In yeast cells, major portion of <br /> D) grown room temperature (A C) shifted 37&#194;&#176;C 2 h (B D)         rDNA repeat transcribed polymerase I consists of <br /> were fractionated 7 47% sucrose density gradients described Materials   single transcription <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> leading synthesis 35S <br /> and Methods. Peaks representing free 40S, 60S, 80S monoribosomes            rRNA precursor processed form 18S rRNA found <br /> labeled. Peaks representing half-mer polyribosomes labeled arrows. <br />                                                                                     40S ribosomal subunit 25S 5.8S rRNAs <br />                                                                                     60S subunit. 5S rRNA, transcribed polymerase <br />    The reduction free 60S subunits                         III, 60S subunit (62, 66). <br /> presence half-mers indicated imbalance ratio                           Since 25S 18S rRNAs highly methylated during <br /> 40S 60S subunits. This confirmed examining                            processing, pre-rRNA processing easily followed meta- <br /> levels total 60S 40S subunits conditions cause                    bolic labeling [methyl-3H]methionine (61). RNA was <br /> dissociation subunits. Extracts prepared buffer lack-                    pulse-chase labeled 2 min 37&#194;&#176;C [methyl-3H]methi- <br />  <br />  <br />  <br />  <br />    FIG. 5. nmd3-4 mutants display reduced 60S subunit levels restrictive temperature. Extracts prepared presence EDTA Mg21 to <br /> dissociate ribosomal subunits sedimented 7 47% sucrose density gradients. Strains AJY735 (NMD3; A B) AJY734 (nmd3-4; C D) were <br /> grown room temperature (A C) shifted 37&#194;&#176;C 2 h (B D). <br />  VOL. 19, 1999                                                                      NMD3 IS REQUIRED FOR STABLE 60S SUBUNITS                     2395 <br />  <br />  <br />                                                                                   experiments, subsequent processing steps appeared to <br />                                                                                   follow wild-type kinetics. It clear initial cleavage <br />                                                                                   35S rRNA slightly altered mutant; however, the <br />                                                                                   localization Nmd3p cytoplasm (see below) suggests <br />                                                                                   indirect consequence nmd3 mutation. <br />                                                                                      To rule possibility altered methylation re- <br />                                                                                   sponsible apparent instability 25S rRNA, pulse-chase <br />                                                                                   labeling using [3H]uracil carried out. Because [3H]uracil is <br />                                                                                   utilized slower kinetics [methyl-3H]methionine is <br />                                                                                   slowly chased unlabeled uracil, possible to <br />                                                                                   compare directly kinetics processing using [3H]uracil <br />                                                                                   [methyl-3H]methionine. In addition, time course the <br />                                                                                   [3H]uracil labeling experiment longer the <br />                                                                                   [methyl-3H]methionine experiment. As seen Fig. 6B, <span id='am-4' about='rdfs:label' typeof='owl:Thing'>label</span> <br />                                                                                   accumulated normally 18S rRNA mutant 37&#194;&#176;C but <br />                                                                                   failed accumulate 25S rRNA. The lack transient accu- <br />                                                                                   mulation processed 25S rRNA experiment prob- <br />                                                                                   ably rapid rate degradation 25S relative the <br />                                                                                   slow incorporation label [3H]uracil chased with <br />                                                                                   slower kinetics [methyl-3H]methionine. The structure of <br />                                                                                   transient minor product migrating slowly 27S <br />                                                                                   (Fig. 6B) known observed [methyl-3H]me- <br />                                                                                   thionine labeling. <br />                                                                                      The processing stability 5S 5.8S rRNAs also <br />                                                                                   examined vivo pulse-chase labeling [3H]uracil (Fig. <br />                                                                                   6C). Total RNA analyzed electrophoresis dena- <br />                                                                                   turing urea-polyacrylamide gel. Quantitation band intensi- <br />                                                                                   ties Fig. 6C showed 10 min chase, ratio of <br />                                                                                   5.8S 5S 1.2 strains. At 30 60 min, ratio <br />                                                                                   increased 1.3 wild type decreased 1.0 the <br />                                                                                   mutant. Thus, like 25S RNA, 5.8S appears synthesized <br />                                                                                   kinetics similar wild-type kinetics, indicated the <br />                                                                                   similar ratios 5.8S 5S 10 min, fails accumulate to <br />                                                                                   wild-type levels. 5S RNA appears unaffected nmd3 mu- <br />                                                                                   tant. <br />                                                                                      pre-rRNA processing examined Northern <br />    FIG. 6. Pulse-chase labeling analysis rRNAs [methyl-3H]methionine      blot analysis total RNA using oligonucleotide probes spe- <br /> and [3H]uracil. (A) Log-phase cultures AJY735 (wild type [WT]; NMD3)       cific 59ETS, ITSI, ITSII, 39ETS 22 nt immediately <br /> AJY734 (nmd3-4) SC Met2 medium shifted 37&#194;&#176;C 2 h. Cells        downstream 39 end mature 25S RNA re- <br /> labeled 2 min [methyl-3H]methionine, saturating of <br /> unlabeled methionine chase added. RNA analyzed electro- <br />                                                                                   moved 39-exonucleolytic processing (see Materials and <br /> phoresis formaldehyde-agarose gel (1.2% agarose, 6% formaldehyde). (B       Methods). We observed significant accumulation pre- <br /> and C) Log-phase cultures AJY735/pRS416 (NMD3) AJY734/pRS416               rRNA processing intermediates aberrant processing prod- <br /> (nmd3-4) SC Ura2 medium shifted 37&#194;&#176;C 2 h. Cells labeled   ucts nmd3-4 mutants nonpermissive temperature <br /> 3 min [3H]uracil chased saturating unlabeled uracil. <br /> RNA prepared analyzed electrophoresis formaldehyde-agarose <br />                                                                                   (data shown). Thus, nmd3 mutants showed relatively nor- <br /> gel described (B) urea-polyacrylamide gel (6% polyacrylamide, 8.2 M   mal processing rRNAs particular instability 25S and <br /> urea) (C) described Materials Methods. Positions major pre-      5.8S rRNAs. Since subunit assembly happens concerted <br /> rRNAs rRNAs indicated.                                                    fashion rRNA processing, results suggest that <br />                                                                                   Nmd3p necessary late step ribosome assembly, after <br />                                                                                   rRNA processing completed. <br /> onine chased unlabeled methionine. At                              Nmd3p cosediments free 60S ribosomal subunits. De- <br /> indicated times addition unlabeled chase, total                      stabilization ribosomal subunits result mutations in <br /> RNA prepared analyzed formaldehyde-agarose gel                         ribosomal protein genes, factors required rRNA processing <br /> electrophoresis. As shown Fig. 6A, nmd3-4 mutant                           factors required assembly. Since nmd3-4 mutants did <br /> showed dramatic destabilization 25S rRNA 37&#194;&#176;C. The                        defects rRNA processing, investigated the <br /> half-life 25S rRNA mutants approximately 4 min,                     possible association Nmd3p ribosomes. The Nmd3 <br /> whereas wild-type cells label 25S rRNA continued                     protein epitope tagged copy c-Myc epitope <br /> accumulate chase (Fig. 6A). Although 25S                           amino terminus. This construct, expressed single- <br /> rRNA failed accumulate, kinetics processing 18S                      copy centromeric plasmid, complemented temperature <br /> and 25S rRNAs normal nmd3-4 mutant                            sensitivity nmd3 mutant. Extracts prepared from <br /> room temperature 37&#194;&#176;C (Fig. 6A). This seen                         cells expressing c-Myc-Nmd3p fractionated sucrose <br /> transient appearance mature 18S 25S rRNAs 1:1                         density gradients. The A254 profile gradient deter- <br /> molar ratio 2 min similar kinetics disappearance                    mined, fractions collected (Fig. 7A). The protein <br /> of 27S 20S precursor rRNAs, respectively. Similar results                     compositions gradient fractions analyzed immuno- <br /> were obtained separate experiments using different nmd3                        blotting. Western blot analysis showed majority of <br /> mutant strains (data shown). The slight increase 35S                       Nmd3p migrated position free 60S subunits su- <br /> precursor time zero mutant observed separate                     crose gradients observed fractions contain- <br />  2396       HO AND JOHNSON                                                                                                              MOL. CELL. BIOL. <br />  <br />  <br />                                                                                        intensities Fig. 8 revealed 12-fold increased signal Topo <br />                                                                                        II nuclear fraction lane compared total extract lane, <br />                                                                                        Nmd3p G6PDH depleted 0.7- 0.6- <br />                                                                                        fold levels total extract lane. Thus, Nmd3p and <br />                                                                                        G6PDH present nuclear fraction approximately <br />                                                                                        5% level expected nuclear proteins. Al- <br />                                                                                        possible small Nmd3p present <br />                                                                                        nucleus, majority protein cytoplasmic, sup- <br />                                                                                        porting notion apparent association free 60S <br />                                                                                        subunits reflects association cytoplasm. The c-Myc <br />                                                                                        cross-reactive protein Nmd3p signal observed <br />                                                                                        cells lacking c-Myc-tagged Nmd3p (data shown). <br />  <br />  <br />  <br />                                                                                                                DISCUSSION <br />  <br />                                                                                           We identified allele NMD3 screen mutations <br />                                                                                        synthetically lethal deletion XRN1, encodes the <br />                                                                                        major cytoplasmic exoribonuclease responsible mRNA <br />    FIG. 7. Nmd3p cosediments free 60S ribosomal subunits. (A) An extract          degradation yeast (27, 35, 55). NMD3 previously been <br /> from log-phase cells strain CH1305 containing c-Myc-tagged NMD3 the <br /> centromeric plasmid pAJ153 fractionated 7 47% sucrose density <br />                                                                                        identified two-hybrid screen UPF1 (22), gene <br /> gradient. A254 peaks representing free 40S, 60S, 80S monoribosomes             required nonsense-mediated mRNA decay. In eukaryotes, <br /> indicated. (B) Fractions gradient shown panel A collected         premature stop codons mRNA stimulate rapid dead- <br /> precipitated immunoblotting. The Western blot developed using               enylation-independent degradation mRNA, non- <br /> antibodies 60S subunit protein L3, used marker positions of <br /> 60S subunits, c-Myc epitope. <br />                                                                                        sense-mediated decay (3, 25, 42, 48). In yeast, pathway is <br />                                                                                        essential depends UPF1, UPF2/NMD2, UPF3. <br />                                                                                        Deletion genes prevents non- <br />                                                                                        sense-mediated decay confers apparent defects <br /> ing polysomes gradient (Fig. 7B). The large                       cell. NMD3, hand, essential gene. <br /> ribosomal protein L3 used marker sedimen-                                Dominant alleles NMD3 (also referred SRC5) (30) <br /> tation 60S ribosomal proteins. L3                              identified screen suppressors tem- <br /> regions gradient containing 60S subunits, 80S mono-                             perature-sensitive grc5 mutant (31). GRC5, referred as <br /> somes, polysomal ribosomes absent fractions                           QSR1 (30), essential gene protein product, the <br /> containing free 40S subunits (Fig. 7B). These results suggest                          large ribosomal subunit protein L10, required ribosomal <br /> that Nmd3p associated 60S subunits, absence                            subunit joining (13, 16). The loading L10 60S ribo- <br /> from 80S ribosomes polysomes indicates Nmd3p                               somal subunit late cytoplasmic event facilitated the <br /> not integral ribosomal protein.                                                     SQT1 gene product (15). <br />    Cell fractionation. Since ribosome assembly events oc-                            The essential nature NMD3 putative phys- <br /> cur nucleus, apparent association Nmd3p                             ical interaction Nmd3p Upf1p genetic interac- <br /> free 60S subunits explained cosedimen-                          tion xrn1 suggested NMD3 critical <br /> tation Nmd3p nuclear pre-60S complexes. To address                             role mRNA degradation. On hand, genetic <br /> this possibility, cells expressing functional c-Myc-tagged                           interaction QSR1 implied ribosomal function. Although <br /> Nmd3p single-copy vector expressed                             functions mutually exclusive, data sup- <br /> promoter fractionated nuclear cytoplasmic frac-                          port primary role ribosome biogenesis. <br /> tions. The proteins arbitrary amounts fraction                            Examination mRNA stability nmd3 mutants indicated <br /> separated sodium dodecyl sulfate-polyacrylamide gel elec-                           little defect mRNA degradation. nmd3 mutants displayed a <br /> trophoresis (SDS-PAGE) analyzed Western blotting.                               modest (two- fourfold) increase abundance pre- <br /> Topo II G6PDH used nuclear cytoplasmic                                 CYH2 mRNA, subject nonsense-mediated decay. <br /> markers, respectively. As seen Fig. 8, Nmd3p fractionated                           However, increased levels pre-CYH2 appar- <br /> similarly G6PDH, cytoplasmic marker protein. Topo II                            ently impaired mRNA degradation half-life of <br /> was highly enriched nuclear fraction lane,                              pre-CYH2 significantly increased mutants. These <br /> Nmd3p G6PDH depleted. Quantitation band                                modest mRNA stability phenotypes contrast those <br />                                                                                        observed upf1 xrn1 mutants, pre-CYH2 highly <br />                                                                                        stabilized. It clear nmd3 mutants display elevated <br />                                                                                        levels pre-CYH2, defect export <br />                                                                                        pre-CYH2. Dominant alleles nmd3 suppress grc5 muta- <br />                                                                                        tions restore cytoskeletal defects grc5 (31). Thus, it <br />                                                                                        conceivable defect cytoskeleton, caused by <br />                                                                                        nmd3 mutation, affects mRNA export. Although ob- <br />    FIG. 8. Cell fractionation. Cells strain CH1305 containing c-Myc-          served defects mRNA turnover deadenylation-depen- <br /> tagged NMD3 centromeric plasmid pAJ153 fractionated nuclear           dent nonsense-mediated deadenylation-independent <br /> and cytoplasmic fractions. Protein 10 ml total cell lysate (T), nuclear   turnover nmd3-1 hypomorphic allele temperature- <br /> fraction (N), cytoplasmic fraction (C) separated SDS-PAGE 8% <br /> polyacrylamide gels. The separated proteins transferred nitrocellulose <br />                                                                                        sensitive alleles, overexpression truncated form Nmd3p, <br /> membrane, Western blotting performed anti-c-Myc monoclonal                lacking C-terminal 100 amino acids, leads dominant <br /> antibody anti-Topo II G6PDH polyclonal antibodies.                             effect general mRNA stabilization (6). Such stabilization <br />  VOL. 19, 1999                                                          NMD3 IS REQUIRED FOR STABLE 60S SUBUNITS                          2397 <br />  <br />  <br /> appears gain-of-function <span id='am-7' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-8' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>           contain Cx2Cx10&#226;&#8364;&#8220;12Cx2Cx18&#226;&#8364;&#8220;21Cx2Cx75&#226;&#8364;&#8220;89Cx2C. This se- <br /> observed recessive alleles.                                    quence reminiscent various cysteine-rich zinc-binding mo- <br />    60S ribosomal subunits unstable temperature-sen-          tifs including RING fingers, type IV zinc fingers, LIM <br /> sitive nmd3 mutant. Several classes mutations affect          domains (39, 57), spacing Cx2C repeats and <br /> ribosomal subunit stability yeast identified. These      additional residues defining motifs conserved in <br /> include defects integral ribosomal proteins (46, 51), defects      Nmd3p. Because Nmd3p required ribosome assembly <br /> in rRNA processing (4, 8, 17, 20, 24, 52, 56, 59, 67), defects    step, possible motif Nmd3p involved either <br /> in ribosome assembly (15, 33). Our data suggest nmd3             RNA binding protein-protein interaction facilitate the <br /> mutants fall class. The partially functional allele   loading rearrangement 60S ribosomal protein. <br /> nmd3-1 nmd3 temperature-sensitive mutations result                NMD3 nonsense-mediated mRNA decay. The identifi- <br /> reduced levels 60S subunits half-mer polysomes. The            cation NMD3 two-hybrid screen UPF1 suggests <br /> half-mer polysomes appear consequence reduced              role NMD3 nonsense-mediated decay. However, we <br /> 60S subunit levels specifically subunit joining          unable observe defect nonsense mRNA decay in <br /> defect appearance follows time course similar        nmd3 mutants. Thus, NMD3 does appear required <br /> that depletion 60S subunits (unpublished observation).          nonsense-mediated decay pathway. It intriguing that <br /> Pulse-chase analysis rRNA processing indicated 25S            nmd3-1 mutation results truncation protein within <br /> and 18S RNAs normal kinetics temper-            region identified interacting Upf1p. However, <br /> ature-sensitive mutant nonpermissive temperature. How-             UPF1 essential gene NMD3 is, nmd3-1 <br /> ever, 25S rRNA half-life 4 min. Such desta-         upf1 double mutants viable (23). Thus, UPF1 re- <br /> bilization 25S rRNA observed previously cells          quired NMD3 function. The finding overexpression of <br /> depleted 60S subunit protein L16 (46). Since rRNA              truncated Nmd3p gives rise dominant stabilization of <br /> processing ribosome assembly intimately linked pro-           mRNA suggests Nmd3p interact mRNA turn- <br /> cesses, apparently normal processing 25S rRNA nmd3          factors (6), work suggests not <br /> mutants suggest nmd3 defect late assembly          required normal mRNA turnover. <br /> pathway.                                                                 The basis synthetic lethality nmd3-1 xrn1D. <br />    Western blot analysis functional epitope-tagged Nmd3p         At present know nmd3-1 mutation syn- <br /> suggested Nmd3p associated free 60S subunits            thetic lethal xrn1 null mutation. nmd3 temperature- <br /> but polysomes. However, finding Nmd3p               sensitive mutants unable assemble stable 60S subunits, <br /> fractionated cytoplasmic protein supports idea      manifest reduced pool free total 60S subunits. This <br /> association free cytoplasmic 60S subunits. Although           turn leads translation initiation defect. The partial <br /> most 60S subunit proteins loaded particle         function allele, nmd3-1, identified synthetically lethal with <br /> nucleolus, proteins believed exchange-        xrn1, displayed reduced 60S levels, suggesting an <br /> able, raising possibility added cyto-        xrn1 mutant, normal levels 60S subunits essential. This <br /> plasm. One L10, encoded QSR1 (13).                 idea suggests mutations leading reduced 60S <br />    As noted above, NMD3 shows genetic interaction              levels synthetically lethal xrn1 deletion. <br /> QSR1. A second gene, SQT1, identified high-copy-             However, spb2 mutation, leads reduced 60S levels, <br /> number suppressor dominant negative phenotype con-               synthetically lethal xrn1. Thus, synthetic lethality is <br /> ferred overexpressing truncated L10 (15). Sqt1p thought       simply reduced 60S levels, suggesting a <br /> to function cytoplasm loading L10 60S              specific defect conferred nmd3-1 60S subunit. The <br /> subunit. Interestingly, depletion L10 does result de-       identification nmd3 suppressors shed light the <br /> stabilization 60S subunits, depletion Sqt1p does.       specific function NMD3 reason genetic inter- <br /> Thus, addition loading L10 protein, Sqt1p          action XRN1. <br /> additional function required subunit integrity. It has <br /> been suggested Sqt1p contribute novel mecha- <br /> nism translational control modulating ribosome function                               ACKNOWLEDGMENTS <br /> through exchanging protein L10 (13). That SQT1 and <br /> NMD3 appear required late cytoplasmic assembly               We thank Alan Sachs providing strain YAS398, Janet Lindsley <br /> step genetic interaction QSR1 suggests        Jim Wang providing anti-Topo II antibodies, Jonathan Warner <br /> Sqt1p Nmd3p act together. However, preliminary             anti-L3 antibodies, Roy Parker plasmid pRP485. We are <br /> experiments observed physical genetic interac-         grateful Xianmei Yang spb2D cross xrn1D Justin <br /> tion SQT1 NMD3 (23). Additionally,                Brown c-Myc-NMD3 construct pAJ153. We especially appre- <br /> Sqt1p suggested exchange L10 protein         ciative John Woolford critical reading manuscript of <br />                                                                       Allan Jacobson sharing results publication. DNA sequence <br /> ribosomes, depletion 60S subunits nmd3-4 mutants         analysis Core Facility Institute Cellular and <br /> observed hours nonpermissive tem-           Molecular Biology, University Texas Austin. <br /> perature (data shown). This finding suggests nmd3               This work supported NIH grant GM056355 A. W. Johnson. <br /> mutations affect ribosomes formed. Thus, <br /> NMD3 likely required maturation step for <br /> exchanging proteins existing ribosomes. Such maturation                              ADDENDUM IN PROOF <br /> step protein loading, rRNA modification, rear- <br /> rangement existing components. It possible              Because c-Myc signal epitope-tagged <br /> transport 60S subunit affected.                             Nmd3p weak, expressed galactose- <br />    Comparison Nmd3p sequence GenBank sequences                inducible promoter, tagged genomic NMD3 <br /> indicated protein conserved eukaryotes.        13 tandem copies c-Myc verify cytoplasmic lo- <br /> In addition, archaebacteria bacteria contain proteins      calization. The multiply tagged protein functional by <br /> related sequence. Sequence analysis Nmd3p revealed              indirect immunofluorescence techniques the <br /> readily recognizable protein motifs. Intriguingly, proteins     cytoplasm excluded nucleus. <br />  2398       HO AND JOHNSON                                                                                                                         MOL. CELL. BIOL. <br />  <br />  <br />                                  REFERENCES                                              Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. <br />  1. Anderson, J. S. J., R. Parker. 1998. The 39 59 degradation yeast       29. Kaufer, N. F., H. M. Fried, W. F. Schwindinger, M. Jasin, J. R. Warner. <br />     mRNAs general mechanism mRNA turnover requires SKI2                1983. Cycloheximide resistance yeast: gene protein. Nucleic <br />     DEVH box protein 39 59 exonucleases exosome complex.                   Acids Res. 11:3123&#226;&#8364;&#8220;3135. <br />     EMBO J. 17:1497&#226;&#8364;&#8220;1506.                                                            30. Koller, L. Personal communication. <br />  2. Baim, S. B., D. F. Pietras, D. C. Eustice, F. Sherman. 1985. A mutation      31. Koller, L., T. Karl, T. Klade, R. Kodzius, A. Thur, R. Andrejic, M. <br />     allowing mRNA secondary structure diminishes translation Saccharo-             Breitenbach. 1995. Personal communication. <br />     myces cerevisiae iso-1-cytochrome c. Mol. Cell. Biol. 5:1839&#226;&#8364;&#8220;1846.               32. Kranz, J. E., C. Holm. 1990. Cloning function: alternative ap- <br />  3. Barker, G. F., K. Beemon. 1991. Nonsense codons Rous                  proach identifying yeast homologs genes organisms. Proc. <br />     sarcoma virus gag gene decrease stability unspliced viral RNA. Mol.           Natl. Acad. Sci. USA 87:6629&#226;&#8364;&#8220;6633. <br />     Cell. Biol. 11:2760&#226;&#8364;&#8220;2768.                                                        33. Kressler, D., J. la Cruz, M. Rojo, P. Linder. 1998. Dbp6p an <br />  4. Baudin-Baillien, A., D. Tollervey, C. Cullin, F. Lacroute. 1997. Func-           essential putative ATP-dependent RNA helicase required 60S-ribosom- <br />     tional analysis Rrp7p, essential yeast protein involved pre-rRNA            al-subunit assembly Saccharomyces cerevisiae. Mol. Cell. Biol. 18:1855&#226;&#8364;&#8220; <br />     processing ribosome assembly. Mol. Cell. Biol. 17:5023&#226;&#8364;&#8220;5032.                     1865. <br />  5. Belgrader, P., J. Cheng, X. Zhou, L. S. Stephenson, L. E. Maquat. 1994.      34. Kruiswijk, T., R. J. Planta, J. M. Krop. 1978. The course assembly of <br />     Mammalian nonsense codons cis effectors nuclear mRNA half-life.            ribosomal subunits yeast. Biochim. Biophys. Acta 517:378&#226;&#8364;&#8220;389. <br />     Mol. Cell. Biol. 14:8219&#226;&#8364;&#8220;8228.                                                   35. Larimer, F. W., C. L. Hsu, M. K. Maupin, A. Stevens. 1992. Character- <br />  6. Belk, J., F. He, A. Jacobson. 1998. Personal communication.                      ization XRN1 gene encoding 59339 exoribonuclease: sequence data <br />  7. Benard, L., K. Carroll, R. C. P. Valle, R. B. Wickner. 1998. Ski6p          analysis disparate protein mRNA levels gene-disrupted yeast <br />     homolog RNA-processing enzymes affects translation non-poly(A)            cells. Gene 120:51&#226;&#8364;&#8220;57. <br />     mRNAs 60S ribosomal subunit biogenesis. Mol. Cell. Biol. 18:2688&#226;&#8364;&#8220;            36. Lee, B. S., M. R. Culbertson. 1995. Identification additional gene <br />     2696.                                                                                required eukaryotic nonsense mRNA turnover. Proc. Natl. Acad. Sci. <br />  8. Briggs, M. W., K. T. Burkard, J. S. Butler. 1998. Rrp6p, yeast               USA 92:10354&#226;&#8364;&#8220;10358. <br />     homologue human PM-Scl 100-kDa autoantigen, essential effi-        37. Leeds, P., S. W. Peltz, A. Jacobson, M. R. Culbertson. 1991. The product <br />     cient 5.8S rRNA 39 end formation. J. Biol. Chem. 273:13255&#226;&#8364;&#8220;13263.                    yeast UPF1 gene required rapid turnover mRNAs containing <br />  9. Brown, J. T., A. W. Johnson. 1998. Unpublished data.                             premature translational termination codon. Genes Dev. 5:2303&#226;&#8364;&#8220;2314. <br /> 10. Cui, Y., K. W. Hagan, S. Zhang, S. W. Peltz. 1995. Identification        38. Leeds, P., J. M. Wood, B.-S. Lee, M. R. Culbertson. 1992. Gene products <br />     characterization genes required accelerated degradation          promote mRNA turnover Saccharomyces cerevisiae. Mol. Cell. Biol. <br />     mRNAs containing premature translational termination codon. Genes               12:2165&#226;&#8364;&#8220;2177. <br />     Dev. 9:423&#226;&#8364;&#8220;436.                                                                  39. Mackay, J. P., M. Crossley. 1998. Zinc fingers sticking together. <br /> 11. Culbertson, M. R., K. M. Underbrink, G. R. Fink. 1980. Frameshift                Trends Biochem. Sci. 23:1&#226;&#8364;&#8220;4. <br />     suppression Saccharomyces cerevisiae. II. Genetic properties group II      40. Mager, W. H., R. J. Planta, J. Ballesta, J. C. Lee, K. Mizuta, K. Suzuki, J. R. <br />     suppressors. Genetics 95:833&#226;&#8364;&#8220;853.                                                    Warner, J. Woolford. 1997. A new nomenclature cytoplasmic <br /> 12. Czaplinski, K., M. J. Ruiz-Echevarria, S. V. Paushkin, X. Han, Y. Weng,              ribosomal proteins Saccharomyces cerevisiae. Nucleic Acids Res. 25:4872&#226;&#8364;&#8220; <br />     H. A. Perlick, H. C. Dietz, M. D. Ter-Avanesyan, S. W. Peltz. 1998. The          4875. <br />     surveillance complex interacts translation release factors enhance   41. Maquat, L. E. 1995. When cells stop making sense: effects nonsense <br />     termination degrade aberrant mRNAs. Genes Dev. 12:1665&#226;&#8364;&#8220;1677.                     codons RNA metabolism vertebrate cells. RNA 1:453&#226;&#8364;&#8220;465. <br /> 13. Dick, F. A., D. P. Eisinger, B. L. Trumpower. 1997. Exchangeability       42. Maquat, L. E., A. J. Kinneburgh, E. A. Rachmilewitz, J. Ross. 1981. <br />     Qsr1p, large ribosomal subunit protein required subunit joining, sug-          Unstable b-globin mRNA mRNA deficient beta o thalassemia. Cell 27: <br />     gests novel translational regulatory mechanism. FEBS Lett. 419:1&#226;&#8364;&#8220;3.                543&#226;&#8364;&#8220;553. <br /> 14. Dinman, J. D., R. B. Wickner. 1995. 5S rRNA involved fidelity       43. Masison, D. C., A. Blanc, J. C. Ribas, K. Carroll, N. Sonenberg, R. B. <br />     translational reading frame. Genetics 141:95&#226;&#8364;&#8220;105.                                    Wickner. 1995. Decoying cap2 mRNA degradation double- <br /> 15. Eisinger, D. P., F. A. Dick, E. Denke, B. L. Trumpower. 1997. SQT1,              stranded RNA virus poly(A)2 mRNA surveillance yeast antiviral <br />     encodes essential WD domain protein Saccharomyces cerevisiae,            system. Mol. Cell. Biol. 15:2763&#226;&#8364;&#8220;2771. <br />     suppresses dominant-negative mutations ribosomal protein gene             44. Masurekar, M., E. Palmer, B. I. Ono, J. M. Wilhelm, F. Sherman. 1981. <br />     QSR1. Mol. Cell. Biol. 17:5146&#226;&#8364;&#8220;5155.                                                 Misreading ribosomal suppressor SUF46 altered 40S subunit <br /> 16. Eisinger, D. P., F. A. Dick, B. L. Trumpower. 1997. Qsr1p, 60S                 yeast. J. Mol. Biol. 147:381&#226;&#8364;&#8220;390. <br />     ribosomal subunit protein, required joining 40S 60S subunits.      45. Mitchell, P., E. Petfalski, A. Shevchenko, M. Mann, D. Tollervey. 1997. <br />     Mol. Cell. Biol. 17:5136&#226;&#8364;&#8220;5145.                                                       The exosome: conserved eukaryotic RNA processing complex containing <br /> 17. Fabian, G. R., A. K. Hopper. 1987. RRP1, Saccharomyces cerevisiae              multiple 39359 exoribonucleases. Cell 91:457&#226;&#8364;&#8220;466. <br />     gene affecting rRNA processing production mature ribosomal sub-           46. Moritz, M., B. A. Pulaski, J. L. Woolford, Jr. 1991. Assembly 60S <br />     units. J. Bacteriol. 169:1571&#226;&#8364;&#8220;1578.                                                  ribosomal subunits perturbed temperature-sensitive yeast mutants de- <br /> 18. Foiani, M., A. M. Cigan, C. J. Paddon, S. Harashima, A. G. Hinnebusch.           fective ribosomal protein L16. Mol. Cell. Biol. 11:5681&#226;&#8364;&#8220;5692. <br />     1991. GCD2, translational repressor GCN4 gene, general            47. Page, A. M., K. Davis, C. Molineux, R. D. Kolodner, A. W. Johnson. <br />     function initiation protein synthesis Saccharomyces cerevisiae.         1998. Mutational analysis exoribonuclease I Saccharomyces cerevisiae. <br />     Mol. Cell. Biol. 11:3203&#226;&#8364;&#8220;3216.                                                       Nucleic Acids Res. 26:3707&#226;&#8364;&#8220;3716. <br /> 19. Gietz, D., A. St. John, R. A. Woods, R. H. Schiestl. 1992. Improved          48. Peltz, S. W., F. He, E. Welch, A. Jacobson. 1994. Nonsense-mediated <br />     method high efficiency transformation intact yeast cells. Nucleic Acids       mRNA decay yeast. Prog. Nucleic Acid Res. Mol. Biol. 47:271&#226;&#8364;&#8220;297. <br />     Res. 20:1425.                                                                    49. Pulak, R., P. Anderson. 1993. mRNA surveillance Caenorhabditis <br /> 20. Ginisty, H., F. Amalric, P. Bouvet. 1998. Nucleolin functions       elegans smg genes. Genes Dev. 7:1885&#226;&#8364;&#8220;1897. <br />     step ribosomal RNA processing. EMBO J. 17:1476&#226;&#8364;&#8220;1486.                          50. Ruiz-Echevarria, M. J., C. I. Gonzalez, S. W. Peltz. 1998. Identifying the <br /> 21. Gonzalez, A., A. Jimenez, D. Vazquez, J. E. Davies, D. Schindler. 1978.          right stop: determining surveillance complex recognizes de- <br />     Studies mode action hygromycin B, inhibitor translocation         grades aberrant mRNA. EMBO J. 17:575&#226;&#8364;&#8220;589. <br />     eukaryotes. Biochim. Biophys. Acta 521:459&#226;&#8364;&#8220;469.                               51. Sachs, A. B., R. W. Davis. 1989. The poly(A) binding protein required <br /> 22. He, F., A. Jacobson. 1995. Identification novel component            poly(A) shortening 60S ribosomal subunit-dependent translation <br />     nonsense-mediated mRNA decay pathway use interacting protein                initiation. Cell 58:857&#226;&#8364;&#8220;867. <br />     screen. Genes Dev. 9:437&#226;&#8364;&#8220;454.                                                    52. Sachs, A. B., R. W. Davis. 1990. Translatio <br /> </body></html>