<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Jan. 2000, p. 1&#226;&#8364;&#8220;11                                                                             Vol. 20, No. 1 <br /> 0270-7306/00/$04.00&#226;&#171;&#185;0 <br /> Copyright &#194;&#169; 2000, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br /> Scanning Mutagenesis Mcm1: Residues Required DNA Binding, <br />           DNA Bending, Transcriptional Activation by <br />                        MADS-Box Protein <br />               THOMAS B. ACTON, JANET MEAD, ANDREW M. STEINER,                              AND   ANDREW K. VERSHON* <br />                      Waksman Institute Microbiology Department Molecular Biology Biochemistry, <br />                                       Rutgers University, Piscataway, New Jersey 08854-8020 <br />                         Received 11 June 1999/Returned modification 22 July 1999/Accepted 23 September 1999 <br />  <br />              MCM1 essential gene yeast Saccharomyces cerevisiae member MADS-box family <br />           transcriptional regulatory factors. <span id='am-2' about='protege:TO' typeof='owl:Thing'>To</span> understand nature protein-DNA interactions class <br />           proteins, series alanine substitutions DNA-binding domain Mcm1 examined <br />           effects mutations vivo vitro. Our results indicate residues Mcm1 important <br />           viability, transcriptional activation, DNA binding bending. Substitution residues Mcm1 which <br />           highly conserved MADS-box proteins lethal cell abolish DNA binding vitro. <br />           These positions identical interactions DNA serum response factor-DNA and <br />           &#226;?&#163;2-Mcm1-DNA crystal structures, suggesting residues make conserved core protein-DNA <br />           interactions responsible docking MADS-box proteins DNA. Substitution residues well <br />           conserved members MADS-box family play important roles contributing specificity of <br />           DNA binding. These results suggest general model MADS-box proteins recognize bind DNA. We <br />           provide evidence N-terminal extension Mcm1 considerable conformational freedom, <br />           possibly allow binding different DNA sites. Finally, identified mutants positions are <br />           critical Mcm1-mediated DNA bending slow-growth <span id='am-3' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-4' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-5' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. This finding consistent with <br />           earlier results, indicating DNA bending role Mcm1 function cell. <br />  <br />  <br />    The yeast Mcm1 protein member MADS-box                      The middle layer consists hydrophobic four-stranded an- <br /> family transcriptional regulatory factors. This growing class           tiparallel &#226;?&#164;-sheet, formed &#226;?&#164;-strands mono- <br /> of proteins named Mcm1, Arg80, Agamous, Deficiens,                mer aligned roughly parallel coiled-coil alpha- <br /> and serum response factor (SRF),                helices. The &#226;?&#164;-loop linking strands monomer is <br /> proteins isolated containing DNA-binding dimeriza-                involved phosphate contact position outside the <br /> tion motif (5, 16, 18, 23, 32). This class proteins        conserved CArG-box [CC(A/T)6GG] binding site. The top <br /> 60 members organisms range                  layer protein consists extended coil region, fol- <br /> the single-cell yeast Saccharomyces cerevisiae (MCM1, ARG80,               lowed alpha-helix turns. This second alpha-helix <br /> and RLM1) Xenopus (SRF), Drosophila (DRSF MEF2),                    monomer packs corresponding face the <br /> plants (Ag, Def, Glo) humans (SRF) (2, 5, 13, 15, 16,              monomer, adding dimerization interface. In addi- <br /> 18, 20, 23, 28, 32). Although DNA-binding domains                  tion layers protein, N- <br /> highly conserved family proteins, function             terminal extension alpha-helix monomer <br /> varies greatly, ranging regulatory factors control               makes <span id='am-159' about='xsp:base' typeof='owl:Thing'>base</span>-specific phosphate backbone con- <br /> basic cellular processes required development             tacts positions center recognition site. Inter- <br /> and determination cell type (2, 11, 27, 28). Understanding              estingly, position extension different the <br /> how class proteins functions important            proteins. In SRF structure region follows path <br /> understanding mechanism regulation cellular                 minor groove center site. In the <br /> and developmental processes.                                               Mcm1 structure, region crosses minor groove and <br />    The cocrystal SRF protein complex DNA revealed               makes base-specific contacts major groove center <br /> the structure MADS domain time (19).                  site. <br /> More recently, structure member MADS-                   Biochemical structural studies shown many <br /> box family, yeast Mcm1 protein, indicated overall             MADS-box proteins produce large bend DNA upon <br /> fold protein DNA contacts remark-               binding recognition sites (1, 8, 19&#226;&#8364;&#8220;22, 24, 30). Circular <br /> ably conserved yeast human proteins (24).                  permutation assays suggest SRF produces significant <br /> These structures MADS-box domain forms protein                  bend DNA binding site. These results agree <br /> dimer composed layers (Fig. 1). The layer consists <br />                                                                            SRF crystal structure, DNA bent on <br /> of antiparallel coiled-coil alpha-helices, mono- <br />                                                                            CArG recognition site protein to <br /> mer, positioned minor groove center the <br />                                                                            produce overall bend angle 72&#194;&#176; (19). In Mcm1 struc- <br /> recognition site. Residues helices make numerous <br />                                                                            ture, DNA bent protein, edge <br /> phosphate base-specific contacts major groove. <br />                                                                            site second DNA bend away protein <br />                                                                            plane perpendicular bend (24). This unlike <br />   * Corresponding author. Mailing address: Waksman Institute, 190          DNA SRF structure, lies predominantly <br /> Frelinghuysen Rd., Piscataway, NJ 08854-8020. Phone: (732) 445-2905.       plane. Mutations positions outside con- <br /> Fax: (732) 445-5735. E-mail: vershon@mbcl.rutgers.edu.                     served CArG site affect degree bending Mcm1 (1). <br />  <br />                                                                        1 <br />  2      ACTON ET AL.                                                                                                                                MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 1. Mcm1 contacts consensus binding site DNA. (A) The contacts shown predicted observed contacts &#226;?&#163;2-Mcm1-DNA ternary complex <br /> and consensus symmetric Mcm1 binding site used studies described here. One-half site shown axis symmetry &#226;&#171;&#185;1 &#226;&#171;&#186;1 <br /> bp. The contacts shown Mcm1 monomer, Lys46, adjacent monomer complex. The arrows indicate base-specific contacts; <br /> the circles indicate phosphate sugar backbone contacts. (B C) Models Mcm1 dimer binding site derived coordinates crystal structure <br /> of Mcm1 ternary complex (24). Two different views complex shown, positions different layers protein. Side chains residues <br /> required DNA bending, K40, S37, V34 (top bottom), shown right monomer panel C. <br />  <br />  <br />  <br /> These base-pair substitutions large effect tran-                       DNA bending Mcm1. We does appear <br /> scriptional activation, suggesting possible link                       strict correlation simple DNA bending and <br /> between DNA-bending activity Mcm1 transcrip-                               transcriptional activation Mcm1 own. However, mu- <br /> tional regulation MADS-box protein.                                           tants altered DNA bending slow growth phe- <br />    To obtain deeper understanding protein-DNA in-                            notype, indicating important role Mcm1 <br /> teractions requirements DNA-bending                                function cell. <br /> Mcm1 protein, constructed series alanine substi- <br /> tutions DNA-binding domain assayed effects                                                   MATERIALS AND METHODS <br /> these mutations vivo vitro. Our results identify <br />                                                                                          Plasmids. Individual alanine substitutions position DNA-bind- <br /> residues critical supporting viability cell                      ing region Mcm1 constructed cloning double-stranded oligonucleo- <br /> and measure contributions chain DNA                           tides containing desired codon changes pJM231, derivative the <br /> binding transcriptional activation. The results presented                         pRS413 (CEN HIS3) vector contains engineered MCM1 gene expressed <br /> here, SRF-DNA &#226;?&#163;2-Mcm1-DNA crystal                               native promoter. Unique restriction sites, KpnI (bp 24), SacI (bp 51), <br />                                                                                       MluI (bp 91), SpeI (bp 100), MfeI (bp 140), StuI (bp 170), engineered <br /> structures, help define conserved set protein-DNA inter-                         region coding DNA-binding domain Mcm1 introducing <br /> actions critical DNA binding MADS-box proteins. Our                            silent mutations indicated nucleotide positions using recursive PCR as <br /> results identify residues required sequence-specific                     described previously (14). The sequences inserts adjacent regions <br />  VOL. 20, 2000                                                                                           SCANNING MUTAGENESIS OF Mcm1                  3 <br />  <br />  <br /> were determined verify presence specific base-pair substitutions   assayed ability support growth cell (18). <br /> to ensure mutations introduced cloning process.       In order accomplish this, plasmid shuffle technique was <br /> The engineered MCM1 construct complements mcm1 null mutant viability <br /> and activates transcription level unmodified gene, indicating     used introduce mcm1 mutants yeast remove <br /> the codon changes did significantly alter expression activity      wild-type copy gene. The yeast strain YY2052 con- <br /> protein.                                                                             tains chromosomal mcm1::LEU2 null <span id='am-81' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-82' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-84' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>mutation</span> main- <br />    Strains vivo assays. The ability MCM1 mutants maintain cell      tained plasmid containing wild-type MCM1 (4). This plas- <br /> viability activate transcription measured strain YY2052 [MATa <br /> P(PAL)-lacZ::FUS1 leu2-3,112 trp1-1 ura3 his3-11,15 ade2-1 can1-100 mcm1::           mid contains URA3, cells lost this <br /> LEU2/pSL1574-CEN/ARS MCM1 URA3] provided G. Sprague (4).                plasmid selected plating media containing the <br /> Cells transformed mutant derivatives pJM231 (CEN HIS3               drug 5-FOA. Yeast cells transformed derivatives of <br /> MCM1) colonies lost wild-type copy MCM1 plasmid          plasmid pJM231 containing mcm1 alanine substitution mu- <br /> pSL1574 selected plating synthetic medium containing 5-fluoroorotic <br /> acid (5-FOA) absence histidine leucine. Mcm1-dependent tran-           tants tested ability support growth plating <br /> scriptional activation measured strain YY2052 assaying lacZ ex-        5-FOA (3). Transformants fail grow wild- <br /> pression integrated CYC1-lacZ reporter promoter P(PAL)               type MCM1 lost indicates alanine substitu- <br /> Mcm1 binding site liquid &#226;?&#164;-galactosidase assays (4). To assay effects      tion significant effect Mcm1 activity the <br /> mutations Mcm1 protein combination mutant binding sites, mutant <br /> derivatives pJM231 transformed strain JM02 (MATa leu2-3,112             mutant unable complement mcm1 null mutation. <br /> trp1-1 ura3 his3-11,15 ade2-1 can1-100 mcm1::LEU2/pSL1574-CEN/ARS MCM1                  The results alanine scanning mutagenesis indicate that <br /> URA3), loss plasmid pSL1574, containing wild-type gene,       14 40 chains DNA-binding region the <br /> selected medium containing 5-FOA. These strains transformed             Mcm1 MADS-box domain essential ability sup- <br /> with derivatives pTBA43, URA3, 2&#226;?&#174;m, CYC1-lacZ transcription reporter <br /> vector CYC1 upstream activation sequence UAS elements              port growth cell (Table 1). Twelve lethal muta- <br /> been removed replaced consensus Mcm1 binding site containing              tions alpha-helix forms coiled-coil interface <br /> symmetric base-pair substitutions (1). The activation results shown tables    monomers contacts DNA. Many these <br /> are based averages independent transformants mutant         residues contact DNA SRF Mcm1 crystal <br /> and presented percent activity wild-type protein (see Table 1) <br /> or units &#226;?&#164;-galactosidase activity (see Table 2 Fig. 6).                       structures highly conserved MADS-box pro- <br />    Protein purification. Mutant Mcm11&#226;&#8364;&#8220;96 proteins used vitro DNA-          teins (19, 24) (Fig. 1). Alanine substitutions residues K38, <br /> binding assays purified Escherichia coli BL21 cells trans-      R39, K40, G42, K45, K46, make direct contacts with <br /> formed pHA227, protein expression vector sequences coding        DNA Mcm1 crystal structure, support cell via- <br /> for Mcm1 residues 1 96 fused frame gene coding the <br /> maltose-binding protein. The mutant fusion proteins purified 100-ml        bility. Substitutions conserved positions alpha- <br /> cultures described earlier (1). The proteins isolated procedure      helices contact DNA, R32, F36, <br /> &#226;&#172;&#381;95% homogenous consisted nonnative N-terminal amino acids,               I43, M44, L53, T54, fail complement mcm1 null <br /> Gly-Ser, followed native Mcm1 residues 1 96. The protein concentra-        mutant lethal cell. These residues positioned <br /> tion mutant determined Bradford assays (Bio-Rad), normalized, <br /> and verified sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-      dimer interface, possible explanation for <br /> PAGE) gels stained Coomassie blue.                                              inability support growth cell side <br />    EMSAs. The DNA fragments utilized electrophoretic mobility shift           chains critical protein stability proper folding. <br /> assays (EMSAs) (see Fig. 2 Fig. 6) synthesized PCR amplifi-         Only mutations N-terminal extension, F25A and <br /> cation wild-type mutant &#226;?&#163;2/Mcm1 operator sites transcription re- <br /> porter plasmids. As described previously, oligonucleotides W340 (5&#226;&#172;&#732;-CACGCC           I26A, failed complement null mutant. Both these <br /> TGGCGGATCTGC-3&#226;&#172;&#732;), [&#226;?&#165;-32P]ATP end labeled purified                 positions located beginning N-terminal exten- <br /> Sephadex G-25 spin column, W341 (5&#226;&#172;&#732;-GCCCACGCTAGGCAATC-3&#226;&#172;&#732;)                        sion, folds coiled-coil alpha-helices and <br /> were annealed CYC1 promoter region Mcm1 binding         forms dimer interface. These substitutions may <br /> site used amplify 120-bp fragment containing Mcm1 recogni- <br /> tion site (1, 33). The PCR-generated fragments purified native     affect protein stability. Three mutations, V34A, S37A, and <br /> PAGE. The circular permutation assays (see Fig. 4 Fig. 5) performed         E49A, viable slow-growth phenotype, suggesting <br /> with BamHI-, NheI-, HindIII-, EcoRI-generated end-labeled fragments          weakly complement mcm1 null mutant. <br /> derivatives pGD579, contains tandem repeat 375-bp fragment              Effects alanine substitutions Mcm1-mediated tran- <br /> separated 24-bp polylinker containing wild-type mutant Mcm1 binding <br /> sites (1).                                                                           scriptional activation. Although mutants fail to <br />    The purified mutant Mcm11&#226;&#8364;&#8220;96 proteins incubated 1 h room              complement mcm1 null mutation lethal <br /> temperature labeled DNA fragment 20 mM Tris (pH 7.6), 200 mM             cell, large number alanine substitutions are <br /> NaCl, 5 mM MgCl2, 0.1 mM EDTA, 0.1% NP-40, 10% glycerol, 10 &#226;?&#174;g                viable. To determine substitutions affect Mcm1 <br /> herring sperm DNA ml. The circular permutation EMSA performed <br /> with 6% polyacrylamide gel 1&#226;&#171;&#187; Tris-borate-EDTA (TBE) 200 V,        activity, viable mutants assayed ability to <br /> other EMSAs performed 6% polyacrylamide gel 0.5&#226;&#171;&#187; TBE. EMSA              activate transcription vivo. This assay used integrated <br /> gels dried visualized using Molecular Dynamics 425E Phosphor-          lacZ reporter gene control P(PAL) Mcm1 bind- <br /> Imager. The relative binding affinity Mcm1 mutants calculated          ing site (Table 1) (4, 10). Many viable mutants have <br /> linear regression compared wild-type protein. <br />    Cross-linking assays. Oligonucleotides used cross-linking experiments      wild-type near-wild-type levels transcriptional activation <br /> were synthesized using photoreactive reagent 5-bromo-2&#226;&#172;&#732;-deoxyuridine-         reporter promoter, indicating dramat- <br /> &#226;?&#164;-cyanoethylphosphoramidite (Glen Research, Inc.) place thymine phos-          ically alter structure function protein. Most of <br /> phoramadite position &#226;&#171;&#185;1 DNA strand Mcm1 symmetric               substitutions residues surface protein <br /> consensus site (5&#226;&#172;&#732;-GCCCTACCTAABrUTAGGTACCCG-3&#226;&#172;&#732;). The complemen- <br /> tary strand (5&#226;&#172;&#732;-CGGGTACCTAATTAGGTAGGGC-3&#226;&#172;&#732;), did                     contact DNA. As expected, substitutions of <br /> contain 5-bromouracil (BrU), annealed BrU containing oligonucleotides         residues contact DNA, R19, I21, S37, <br /> to produce double-stranded recognition site. Cross-linking experiments        T66, reduced levels activation. Interestingly, the <br /> performed purified proteins described earlier (1).                           slow-growth mutant V34A, contacts DNA, the <br />                                                                                      lowest level activation viable mutant. Alanine substi- <br />                                   RESULTS                                            tutions residues I23, F48, E49 reduced levels <br />                                                                                      activation. These residues near dimer interface, and <br />    Conserved residues Mcm1 essential viability. To                        possible alanine substitutions positions alter <br /> determine residues Mcm1 important activ-                        stability protein, causing decrease transcrip- <br /> ity, residue DNA-binding region MADS-box                          tional activation. Although results correlate with <br /> domain individually substituted alanine. MCM1                         predictions based Mcm1 crystal structure, of <br /> essential gene yeast, alanine mutants                      results unexpected (24). For example, residue R18 <br />  4       ACTON ET AL.                                                                                                                MOL. CELL. BIOL. <br />  <br />  <br />          TABLE 1. Activity Mcm1 alanine substitutions                       expressed purified manner determine the <br />                                                                                    reductions Mcm1-mediated activation vivo function <br />                                                     Relative decrease DNA- <br />     Mutanta      Growthb        % Activationc <br />                                                       binding affinityd (fold)     decreased DNA-binding affinity. An example assays is <br />                                                                                    shown Fig. 2, relative binding affinity calculated for <br /> Wild type           &#226;&#171;&#185;                100                         1                 mutant listed Table 1. <br /> Q14A                &#226;&#171;&#185;                104                         1.5                  In general, results DNA-binding affinity analysis <br /> Q15A                &#226;&#171;&#185;                103                         1.4               good agreement vivo viability activation <br /> K16A                &#226;&#171;&#185;                 80                         2.5 <br /> E17A                &#226;&#171;&#185;                107                         1.5 <br />                                                                                    data correlate predictions based Mcm1 crystal <br /> R18A                &#226;&#171;&#185;                100                         1.6               structure (24). A number alanine substitutions do <br /> R19A                &#226;&#171;&#185;                 21                         4                 support viability, including F25A, I26A, R32A, F36A, <br /> K20A                &#226;&#171;&#185;                 80                         3                 I43A, M44A, L53A, T54A, residues form the <br /> I21A                &#226;&#171;&#185;                 53                         4.5               hydrophobic core protein dimer interface. During <br /> E22A                &#226;&#171;&#185;                 87                         1.4               purification mutants bacteria, the <br /> I23A                &#226;&#171;&#185;                 34                         4.3               proteins present lower levels lysates, exhibited <br /> K24A                &#226;&#171;&#185;                110                         0.8               decreased affinity heparin column, sus- <br /> F25A                &#226;&#171;&#186;                                          NDB                 ceptible degradation. This finding consistent model <br /> I26A                &#226;&#171;&#186;                                          NDB <br /> E27A                &#226;&#171;&#185;                 98                         2 <br />                                                                                    substitutions affect folding protein, therefore <br /> N28A                &#226;&#171;&#185;                 87                        20                 making stable susceptible proteolysis. <br /> K29A                &#226;&#171;&#185;                100                         0.8               All mutants &#226;&#172;&#381;100-fold decrease DNA- <br /> T30A                &#226;&#171;&#185;                100                                           binding affinity. These results suggest loss viability <br /> R31A                &#226;&#171;&#185;                 70                                           mutants function inability produce <br /> R32A                &#226;&#171;&#186;                                          110                 proteins competent DNA binding. <br /> H33A                &#226;&#171;&#185;                120                         0.25                 The remaining nonviable substitutions, K38A, R39A, K40A, <br /> V34A                S                  9                         2.7               G42A, K45A, K46A, binding face coiled- <br /> T35A                &#226;&#171;&#185;                 82                         3                 coil alpha-helices, residues contact DNA the <br /> F36A                &#226;&#171;&#186;                                        NDB <br /> S37A                S                 66                         2.1 <br />                                                                                    crystal structure (24). Many mutants &#226;&#172;&#381;1,000- <br /> K38A                &#226;&#171;&#186;                                        NDB                   fold decrease DNA-binding affinity vitro, indicating that <br /> R39A                &#226;&#171;&#186;                                        NDB                   residues essential role DNA binding Mcm1. <br /> K40A                &#226;&#171;&#186;                                          125                 Our results fact residues highly con- <br /> H41A                &#226;&#171;&#185;                100                         0.12              served MADS-box proteins underscores im- <br /> G42A                &#226;&#171;&#186;                                        NDB                   portance DNA binding supports model con- <br /> I43A                &#226;&#171;&#186;                                          250                 served mechanism DNA recognition family of <br /> M44A                &#226;&#171;&#186;                                       &#226;&#172;&#381;1,000                 proteins. <br /> K45A                &#226;&#171;&#186;                                       &#226;&#172;&#381;1,000                    Many viable mutants decreases DNA- <br /> K46A                &#226;&#171;&#186;                                        NDB <br /> A47                 &#226;&#171;&#185;                100                         1 <br />                                                                                    binding affinity, general effect the <br /> F48A                &#226;&#171;&#185;                 34                         0.8               DNA-binding affinity nonviable mutants. Alanine sub- <br /> E49A                S                 24                        48                 stitutions residues contact DNA the <br /> S51A                &#226;&#171;&#185;                 99                         1.0               crystal structure, R18, R19, I21, T35, S37, T66, <br /> V52A                &#226;&#171;&#185;                 91                         1.2               modest decreases DNA-binding affinity vitro, which <br /> L53A                &#226;&#171;&#186;                                        NDB                   correlate observed decreases activation by <br /> T54A                &#226;&#171;&#186;                                        NDB                   mutants vivo. These residues somewhat con- <br /> T66A                &#226;&#171;&#185;                 77                         3.1               served MADS-box proteins K38, R39, G42, <br />     a <br />     DNA contacts boldface (based &#226;?&#163;2/Mcm1/DNA crystal struc-          K45, K46, essential viability these <br /> ture).                                                                             residues clearly contribute DNA-binding affinity and <br />   b <br />     &#226;&#171;&#185;, Normal growth swapping wild-type copy MCM1; &#226;&#171;&#186;,          transcriptional activation. Additionally, residues <br /> growth 5 days; S, slow-growth phenotype. <br />   c <br />     The level activation based lacZ expression comparison the <br />                                                                                    role determining specificity DNA binding. <br /> wild type.                                                                            Two mutants slow-growth phenotype, V34A <br />   d <br />     The relative decrease DNA-binding affinity comparison wild-type   S37A, produce protein-DNA complexes higher mo- <br /> protein. NDB, detectable binding observed EMSA.                      bility wild-type protein (see Fig. 6). These mutations <br />                                                                                    cause significant decreases transcriptional activation, <br />                                                                                    V34A resulting lowest &#226;?&#164;-galactosidase activity all <br /> makes base-specific phosphate backbone contacts                        viable mutants. In contrast, mutations, N28A <br /> with base pairs center site                  I21A, greater decreases binding affinity V34A <br /> be expected important role function                       similar decreases levels transcrip- <br /> protein. However, alanine substitution position                     tional activation. These results suggest V34A S37A <br /> viable activates transcription wild-type levels, indicating                 affect aspects Mcm1 function. <br /> that R18 chain critical Mcm1 activity                     Loss base-specific contacts mutations N-termi- <br /> site.                                                                              nal extension. One striking differences the <br />    Effects alanine substitutions DNA-binding affinity                    SRF Mcm1 crystal structures location the <br /> in vitro. A significant number mutants alanine substi-                     N-terminal extension protein. In Mcm1 structure, <br /> tutions residues involved protein-DNA contacts fail                       R18 chain making set contacts major <br /> complement mcm1 null mutant cell viability. To deter-                      groove, SRF structure analogous residue is <br /> mine substitutions affect DNA-binding activity                     making series contacts minor groove DNA <br /> the protein, mutants cloned bacterial expres-                      (19, 24). This difference predicted analysis the <br /> sion vector purified, DNA-binding affinity                       SRF structure supported differences protein-DNA <br /> assayed EMSAs. In addition, viable mutants                      cross-linking SRF Mcm1 base pairs center of <br />  VOL. 20, 2000                                                                                                  SCANNING MUTAGENESIS OF Mcm1                           5 <br />  <br />  <br />  <br />  <br />    FIG. 2. EMSA analysis Mcm1 substitutions. DNA binding symmetric consensus site wild-type Mcm1 (lanes 2 6 47 50), K40A (lanes 7 11), <br /> K45A (lanes 12 16), I43A (lanes 17 21), F48A (lanes 22 26), M44A (lanes 27 30), H41A (lanes 31 34), I21A (lanes 35 38), R19A (lanes 39 42), and <br /> R18A (lanes 43 46). All proteins span residues 1 98 Mcm1, contains entire MADS-box domain. Fivefold titrations Mcm1 performed, <br /> increasing concentration 2.2 &#226;&#171;&#187; 10&#226;&#171;&#186;10 M (lanes 2, 7, 12, 17, 22) 2.0 &#226;&#171;&#187; 10&#226;&#171;&#186;9 (lanes 27, 31, 35, 39, 43, 47). The relative binding affinity calculated <br /> for mutant comparison wild-type shown Table 1. <br />  <br />  <br />  <br /> the recognition site (1, 19). Additionally, mutational analysis                     shows decrease activity mutant site com- <br /> the DNA site shows substitutions central base pairs                        parison wild-type site, suggest the <br /> produce large decrease Mcm1-mediated activation, indi-                            base-specific contacts positions mediated other <br /> cating sequence-specific contacts posi-                        residues. The R18A mutation shows decrease activation <br /> tions (1). Residue R18 makes extensive set base-specific                         A1C binding site mutation comparison wild-type <br /> and phosphate backbone contacts center site                           site (Table 2). This reduction activation R18A mutant <br /> the crystal structure (24). However, alanine substitution                        reporter A1C mutation similar the <br /> position R18 does significantly alter transcriptional activa-                      reduction wild-type protein, suggesting resi- <br /> tion vivo DNA-binding affinity vitro. This result                         dues contacting base pairs. The E17A mutant also <br /> unexpected, positions involved base-                          displays similar activity. On hand, R19A muta- <br /> specific contacts significantly larger effects viability                   tion does reduction activation A1C com- <br /> activation. In comparison, substitution adjacent resi-                        pared wild-type binding site. In fact activation of <br /> due, R19, significantly larger effect transcriptional                         mutant A1C binding site slightly increased, <br /> activation DNA-binding affinity.                                                   suggesting steric interference Arg <br />    To investigate contributions residues                       chain mutant DNA site. These results raise the <br /> the N-terminal extension specificity activation, per-                        possibility R19 contact central base pairs the <br /> formed loss-of-contact experiments (6) E17A,                              consensus symmetric binding site used experiments. <br /> R18A, R19A mutations assayed ability                                Amino acid residues Mcm1 required BrU-mediated <br /> activate transcription reporter promoter containing                            cross-linking center site. BrU photoactivat- <br /> A1C base-pair substitution Mcm1 site. In comparison                          able, isosteric analog thymine. Irradiation BrU UV <br /> the wild-type site, A1C substitution causes fourfold re-                         light leads reactive species form covalent <br /> duction level transcriptional activation wild-                        bond groups close proximity (17). We <br /> type Mcm1 protein (1). If R18A mutant shows similar                                previously used property BrU the <br /> transcriptional activation wild-type mutant                       Mcm1 protein contacts thymine methyl groups major <br /> sites, indicate alanine substitution                        groove base pair position center DNA <br /> removed base-specific contacts position                             recognition site (1). In Mcm1 crystal structure NH1 <br /> DNA, protein unable discriminate                           group R18 close proximity C-5 methyl group of <br /> wild-type mutant sites. However, mutant protein                             T&#226;&#171;&#186;1 likely target cross-linking (24). To test which <br />  6           ACTON ET AL.                                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br />            TABLE 2. Activation N-terminal extension mutants <br />                    wild-type mutant DNA sitesa <br />                                            &#226;?&#164;-Gal activity (U) <br />     Site <br />                      WT            E17A                  R18A            R19A <br />  <br /> None                  0.4            0.3                  0.4              0.4 <br /> WT                    5.5            4.6                  4.9              0.3 <br /> A1C                   1.4            0.9                  0.9              0.9 <br />   a <br />     Values shown units &#226;?&#164;-galactosidase activity (&#226;?&#164;-Gal) standard <br /> deviations &#226;&#172;?10%. WT, wild type. <br />  <br />  <br />  <br />  <br /> amino acid N-terminal extension involved a <br /> base-specific contact thymine center the <br /> recognition site, BrU cross-linking assays performed with <br /> the Mcm1 alanine substitution mutants. The E17A R18A <br /> proteins form cross-link DNA (Fig. 3). This result <br /> suggests chains involved direct contact <br /> with center bases site used experi- <br />                                                                                       FIG. 3. Formation cross-linked complexes Mcm1 mutants. Urea- <br /> ments.                                                                             denaturing PAGE BrU-mediated photo-cross-linking reactions the <br />    The R19A mutant, however, did form significant level                      E17A, R18A, R19A mutants labeled site containing BrU substitution <br /> of BrU-mediated cross-linked band, suggesting resi-                      thymine base &#226;&#171;&#186;1. Lanes 1, 6, 11 control reactions omitting UV <br /> due required interaction thymine methyl group                      radiation. Lanes 2, 7, 12 control reactions omitting Mcm1 protein. <br />                                                                                    Lanes 3, 8, 13 control reactions labeled Mcm1 binding site without <br /> at position &#226;&#171;&#186;1. The small cross-linking po-                      BrU (denoted T figure). Lanes 4, 9, 14 control reactions in <br /> sition may, part, result background level                      10-fold excess unlabeled competitor added. Lanes 5, 10, 15 <br /> Mcm1 photo-cross-linking observed absence                           cross-linking reactions. The arrow indicates position cross- <br /> BrU-substituted DNA (Fig. 3, lanes 3, 8, 13). It                       linked Mcm1-DNA products. <br /> possible R19A substitution alters structure the <br /> N-terminal extension, allowing residue to <br /> interact BrU functional group. However,                           bending wild-type protein T7G site. This result <br /> previously shown Mcm1 fragment missing 17                        suggests large drop bending observed T7G <br /> residues protein, Mcm118&#226;&#8364;&#8220;96, forms cross-linked prod-                     base-pair substitution similar loss contacts res- <br /> uct DNA position (1). This result indicates                   idues V34 S37. <br /> 17 residues N-terminal extension involved                           In addition base-specific contacts V34 S37, <br /> the cross-link. The cross-linking results loss-of-contact                  residues K40, T66, lesser degree K29 H33, are <br /> data agreement suggest R19 involved                      close region DNA possibly contribute to <br /> the major groove contact thymine methyl group                          DNA bending (24, 29). The K40A mutation able to <br /> position &#226;&#171;&#185;1 &#226;&#171;&#186;1 site used experiments.                     support viability cell, DNA-binding affinity of <br />    Mcm1 mutants altered bending activity. Base-pair sub-                      mutant decreased 125-fold compared wild- <br /> stitutions positions &#226;&#171;&#185;7 &#226;&#171;&#186;7 Mcm1 DNA-binding site                      type protein. The circular permutation analysis shows that <br /> cause large decrease DNA bending correspond-                        DNA bending decreased 36&#194;&#176; (Fig. 4B). This decrease is <br /> ing effect DNA-binding affinity (1). Interestingly, mutations                   equivalent decrease bending observed V34A- <br /> at position result significant decrease transcrip-               S37A double mutant indicates residue a <br /> tional activation. In Mcm1 crystal structure, residues V34                     critical role DNA bending. In contrast, ala- <br /> and K38 make base-specific contacts position                        substitutions K29, H33, T66 little effect <br /> DNA site (24). Along residue S37, V34 makes base-                        DNA binding, bending, transcriptional activation (Table <br /> specific contacts position 8 site. To investigate                     1, Fig. 4B; DNA bending data shown K29A and <br /> mutations residues affect DNA-bending activity                     H33A). These results residues make a <br /> Mcm1, circular permutation EMSAs performed                               large contribution DNA binding bending protein. <br /> these mutants apparent bend angles calculated                            To determine residues DNA-binding region of <br /> (Fig. 4). The V34A S37A mutants significant de-                           Mcm1 required DNA bending, assayed alanine <br /> creases DNA bending compared wild-type protein.                          substitution EMSA NheI fragment used the <br /> These results support predictions based crystal structure                   circular permutation experiments. This fragment shows the <br /> and residues important producing                       largest differences shift mobility mutations affect <br /> Mcm1-dependent DNA bend. Interestingly, decrease                            DNA bending serves sensitive diagnostic assay for <br /> bending mutants one-half decrease                      identifying mutants altered bending. With exception <br /> in bending resulting T7G mutant DNA (1). One                       mutants described above, mutant <br /> possibility intermediate effect protein mutants                  proteins bind DNA appear alter DNA bending (data <br /> that chain contributing bending. To test                  shown). It possible chains, K38, may <br /> model, V34A-S37A double mutant constructed                             contribute DNA bending. However, failed to <br /> assayed viability bending. Unlike single mutants,                      bind DNA assay conditions, unable to <br /> the V34A-S37A double mutant unable complement                            measure effect bending. <br /> mcm1 null mutant. The V34A-S37A mutant apparent                                Sequence specificity DNA bending Mcm1. In crystal <br /> bend angle 61&#194;&#176;, double observed decrease                          structure, residues V34A, S37A, K40A make base-specific <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> single mutants, comparable                       phosphate backbone contacts positions 7 8 the <br />  VOL. 20, 2000                                                                                          SCANNING MUTAGENESIS OF Mcm1                7 <br />  <br />  <br />                                                                                     DNA site, shown alanine substitutions of <br />                                                                                     residues cause decreases degree DNA bending <br />                                                                                     protein (24). To examine specificity required for <br />                                                                                     bending, tested ability Mcm1 mutants bend <br />                                                                                     DNA mutant binding sites. If V34A S37A mu- <br />                                                                                     tants degree bending mutant site to <br />                                                                                     wild-type site, suggest residues are <br />                                                                                     contacting particular base pair additional <br />                                                                                     base-specific contacts position lost. However, if <br />                                                                                     reduction bending mutant protein to <br />                                                                                     mutant site, indicate base-specific <br />                                                                                     contacts base pair. The V34A substi- <br />                                                                                     tution apparent degree DNA bending the <br />                                                                                     T7G wild-type sites, providing evidence decrease <br />                                                                                     bending mutant site caused loss contact with <br />                                                                                     residue V34 (Fig. 4C). This result suggests K38 contact <br />                                                                                     thymine base pair critical bending. On <br />                                                                                     hand, T7G mutation causes significant decrease <br />                                                                                     DNA bending wild type (35&#194;&#176;) R18A (29&#194;&#176;), S37A <br />                                                                                     (25&#194;&#176;), T66A (32&#194;&#176;) mutants (Fig. 4C). This result indicates <br />                                                                                     residues contact base pair. As <br />                                                                                     expected, V34A-S37A double mutant does a <br />                                                                                     decrease bending T7G mutation. <br />                                                                                        The SRF protein binds Mcm1 site moderate <br />                                                                                     affinity produces significant bend DNA. However, <br />                                                                                     unlike Mcm1, DNA bending SRF affected T7G <br />                                                                                     base-pair substitution (1). This finding suggests even <br />                                                                                     residues contact DNA identical <br />                                                                                     Mcm1 SRF, mechanism bending signif- <br />                                                                                     icantly different. One difference proteins is <br />                                                                                     SRF contains threonine position analogous <br />                                                                                     V34 Mcm1, chain makes contacts position <br />                                                                                     8 DNA position 7 (19). To test differ- <br />                                                                                     ences specificity bending proteins due <br />                                                                                     presence threonine position 34, constructed <br />                                                                                     V34T substitution Mcm1 assayed effects on <br />                                                                                     transcription vivo DNA binding bending vitro. <br />                                                                                     The V34T mutant complements mcm1 null mutant has <br />                                                                                     near-wild-type levels transcriptional activation the <br />                                                                                     P(PAL) site vivo, showing threonine functionally <br />                                                                                     substitute valine position Mcm1 (data shown). <br />                                                                                     The V34T mutant binds wild-type affinity and <br />                                                                                     produces bend wild-type site similar the <br />                                                                                     wild-type protein (Fig. 4B). However, unlike SRF, V34T <br />                                                                                     mutation exhibits decrease apparent bend angle the <br />                                                                                     T7G substitution (Fig. 4C). These results indicate the <br />                                                                                     difference specificity bending pro- <br />                                                                                     teins solely different amino acids position <br />                                                                                     34 MADS-box domain mechanism bend- <br />                                                                                     ing mutant different wild-type Mcm1 or <br />                                                                                     SRF. Furthermore, results suggest mechanisms <br />                                                                                     used proteins bend DNA differs significantly <br />                                                                                     expected proteins near identity in <br />                                                                                     region protein. Interestingly, magnitude of <br />                                                                                     bending V34T T7G site similar V34A binding <br />                                                                                     mutant DNA site. Neither mutants severe <br />                                                                                     decrease bending wild-type protein site. The <br />                                                                                     T7G substitution sterically interfere wild-type <br />                                                                                     valine chain residue 34, substitution residue <br />                                                                                     alanine relieve interference, permitting increased <br />    FIG. 4. Substitutions Mcm1 reduce DNA bending wild-type           bending comparison wild-type protein. It unlikely <br /> and T7G sites. EMSAs performed utilizing position-permuted frag- <br /> ments shown panel A containing wild-type (B) T7G (C) binding sites   V34T substitution relieves steric interference, and <br /> wild-type, V34A, S37A, V34A-S37A, K40A, V34T, T66A, R18A purified pro-          residue make alternative set contacts <br /> teins indicated. Lanes labeled B, N, H, R denote labeled fragments      partially restore bending. <br /> were generated BamHI, NheI, HindIII, EcoRI enzymes, respectively, as <br /> shown panel A. The bend angle protein-DNA complex shown                 Residues V34 S37 contact C-5 methyl group the <br /> below calculated based Thompson Landy relationship (26).         thymine position 8 DNA recognition site, these <br />  8      ACTON ET AL.                                                                                                                                   MOL. CELL. BIOL. <br />  <br />  <br />                                                                                          Mcm1 binding site causes modest reduction level <br />                                                                                          activation wild-type protein (1). Interestingly, even <br />                                                                                          thymine contacted V34 S37 residues has <br />                                                                                          removed T8C substitution, site shows normal <br />                                                                                          levels DNA bending wild-type protein (Fig. 5B). This <br />                                                                                          result suggests alternate contacts protein <br />                                                                                          mutant site restore bending. These alternate con- <br />                                                                                          tacts dependent V34 S37 chains be- <br />                                                                                          cause alanine substitutions residues cause decrease <br />                                                                                          DNA bending T8C site (Fig. 5B). <br />                                                                                             In vivo vitro DNA sequence specificity altered <br />                                                                                          bending mutants. The V34A mutant decreased levels of <br />                                                                                          DNA bending transcriptional activation, suggesting that <br />                                                                                          DNA bending important transcriptional acti- <br />                                                                                          vation protein. To examine possible role and <br />                                                                                          sequence specificity DNA bending, wild type V34A <br />                                                                                          S37A mutants assayed combination T7G <br />                                                                                          T8C DNA sites transcriptional activation vivo and <br />                                                                                          DNA binding vitro (Fig. 6). The wild-type protein shows <br />                                                                                          similar decreases transcriptional activation DNA-bind- <br />                                                                                          ing affinity T7G T8C DNA sites. However, <br />                                                                                          T7G mutant causes large decrease DNA bending, <br />                                                                                          T8C mutant does not. These results consis- <br />                                                                                          tent model directly linking degree DNA bending <br />                                                                                          activation. <br />   FIG. 5. The effects Mcm1 mutants DNA bending T8G T8C                         The S37A protein decreased levels DNA bending and <br /> mutant sites. EMSAs performed utilizing position-permuted fragments              transcriptional activation binding sites. However, <br /> containing T8G (A) T8C (B) binding sites wild-type, V34A, V34T,             decreases activation appear correspond changes <br /> S37A, V34A-S37A purified proteins described Fig. 4. <br />                                                                                          binding affinity protein. Interestingly, find <br />                                                                                          binding affinity wild-type S37A proteins the <br /> interactions contribute Mcm1-dependent DNA                                   mutant DNA sites identical, activation levels the <br /> bending (24). A T8G substitution Mcm1 binding site                                mutant reduced compared wild-type protein. The <br /> results complete loss transcriptional activation vivo                         V34A protein, like S37A protein, decreased <br /> (1). This base-pair substitution causes large decrease DNA                          bending activation sites, levels of <br /> bending wild-type mutant proteins, indicating                          activation roughly correspond changes binding affinity. <br /> is sequence specificity position DNA bending (Fig.                           Although DNA-binding affinity V34A greater the <br /> 5A). In contrast T8G substitution, T8C mutation                          wild-type protein mutant DNA sites, levels ac- <br />  <br />  <br />  <br />  <br />    FIG. 6. DNA-binding affinity, DNA bending, transcriptional activation wild-type, V34A, S37A proteins wild-type mutant binding sites. DNA <br /> binding symmetric consensus site wild-type Mcm1 (lanes 2 13), V34A (lanes 14 25), S37A (lanes 26 37) wild-type site (lanes 2 5, 14 to <br /> 17, 26 29), T7G site (lanes 6 9, 18 21, 30 33), T8C site (lanes 10 13, 22 25, 34 37). The DNA-binding affinity protein <br /> for site calculated binding curves shown fold decrease affinity relative wild-type protein. The apparent bend angle of <br /> the proteins different sites derived data shown Fig. 4 5. The levels transcriptional activation vivo proteins sites <br /> driving expression lacZ measured described Materials Methods shown. Values shown units &#226;?&#164;-galactosidase activity average <br /> of independent isolates. <br />  VOL. 20, 2000                                                                               SCANNING MUTAGENESIS OF Mcm1                 9 <br />  <br />  <br /> tivation V34A sites equal           observed homeodomain proteins (7). Among homeodo- <br /> the wild-type protein. Taken together, results indicate            main proteins, majority contacts DNA are <br /> that DNA-binding affinity important component tran-             highly conserved alpha-helix lies major groove. <br /> scriptional activation Mcm1 simple            The N51 residue, Lys Arg residues in <br /> correlation absolute degree DNA bending               helix, conserved virtually homeodomain pro- <br /> activation. However, fact that, comparison wild-           teins, chains make set base-specific and <br /> type protein, V34A S37A decreased activation               phosphate backbone contacts nearly identical every <br /> the mutant sites binding sites better          homeodomain-DNA structure determined. Dif- <br /> than wild-type protein suggests mutations              ferences sequence-specific binding homeo- <br /> residues affect characteristic Mcm1 important           domains appear result base-specific contacts by <br /> transcriptional activation does strictly correlate         nonconserved residues helix contacts an <br /> DNA-binding affinity.                                                    N-terminal arm minor groove DNA. Based <br />                                                                          similarity contacts SRF Mcm1 <br />                           DISCUSSION                                     crystal structures, high degree conservation <br />                                                                          residues shown essential DNA binding, <br />    The determination crystal structures SRF                appears MADS-box proteins use conserved <br /> Mcm1 bound DNA identified residues MADS-box pro-                   mechanism bind DNA. Residue K38 Mcm1, which <br /> teins make base-specific phosphate backbone contacts            absolutely conserved MADS-box proteins, may <br /> with DNA (19, 24). We performed alanine scanning                function manner similar N51 homeodomain proteins <br /> mutagenesis DNA-binding region Mcm1 examine                 making conserved base-specific contacts DNA. Other <br /> the relative importance of contacts transcrip-          chains highly conserved MADS- <br /> tional activation Mcm1 vivo DNA binding vitro.              box family proteins, residues R39, G42, K45, and <br /> Alanine substitution 14 40 residues mutated          K46, make similar contacts DNA phosphate backbone in <br /> results loss cell viability, indicating chains   SRF Mcm1 crystal structures essential for <br /> are crucial proper function Mcm1 cell. As              DNA binding. These residues similar role dock- <br /> expected, number chains, residues F25, I26,              ing MADS-box proteins DNA conserved lysines in <br /> R32, F36, I43, M44, buried hydrophobic core            recognition helix homeodomain proteins, appear <br /> the protein dimer interface. Although res-           position anchor helix base-specific recognition. <br /> idues directly involved protein-DNA contacts              Although MADS-box proteins contain conserved <br /> crystal structure, detectable levels DNA             mechanism bind DNA, site selection mutagenesis ex- <br /> binding vitro, suggests important          periments shown significant differences in <br /> stability protein folding.                                            sequence specificity (9, 31). The differences specificity be- <br />    Many lethal mutations positions               tween different MADS-box proteins appear largely <br /> highly conserved MADS-box proteins contact                 different contacts N-terminal extension the <br /> the DNA SRF Mcm1 crystal structures (19,                 center recognition site residues coiled-coil <br /> 24). These residues, K38, R39, G42, K45, K46, unable             alpha-helices interact base pairs outside con- <br /> to complement mcm1 null mutant detect-               served CArG site. For example, position DNA con- <br /> able DNA-binding activity vitro. There residues             tacts N-terminal extension center site is <br /> that highly conserved MADS-box proteins               different SRF Mcm1 crystal structures and <br /> make similar contacts DNA Mcm1 SRF                     contribute differences specificity (19, 24). We also <br /> crystal structures. However, results mutagenesis            shown chains residues 34 37 Mcm1 contrib- <br /> of residues suggest chains make             ute sequence specificity positions outside con- <br /> critical contributions function protein. For exam-         served CArG site. In addition, residues <br /> ple, residue T35 Mcm1 position conserved           contacting DNA outside CArG box involved <br /> nearly MADS-box proteins, residue makes identi-             contacts phosphate backbone, contacts may <br /> cal contacts phosphate backbone G4 DNA               contribute sequence specificity binding. For example, <br /> both Mcm1 SRF crystal structures (19, 24). However,              phosphate backbone contacts Mcm1 <br /> an alanine substitution residue Mcm1 complements              occur DNA bent, results have <br /> the mcm1 null mutant near-wild-type levels tran-              shown ability bend sequence specific. Differences <br /> scriptional activation; also, DNA-binding affinity            sequence specificity responsible broad range <br /> mutant threefold wild-type protein.        DNA bending observed various MADS-box pro- <br /> Likewise, residue T66 highly conserved MADS-               teins (29). <br /> box proteins, alanine substitution residue              The effects of mutations correlate <br /> little effect viability, activation, DNA-binding af-         Mcm1 crystal structure (24). However, results on <br /> finity. These results suggest contacts           effects mutations N-terminal extension un- <br /> highly conserved relative importance great        expected. In Mcm1 crystal structure residue R18 making <br /> other contacts common MADS-box proteins.                       base-specific contact center site. Our vivo <br />    The results alanine scanning mutagenesis Mcm1               transcription data vitro cross-linking results suggest that <br /> and structural determination SRF Mcm1 indicates               R18 limited role contacting base pair. In contrast, <br /> which protein-DNA interactions essential roles recog-            data R19 required for, making, <br /> nition binding MADS-box protein. The conserved                  contact major groove center site. One <br /> base-specific contacts K38, phosphate             possible explanation results structural <br /> backbone contacts K39, G42, K45, K46,                    determination Mcm1 MADS-box solved complex <br /> provide proper general framework DNA binding                  &#226;?&#163;2 protein. The DNA contacts N-terminal <br /> MADS-box proteins. The conserved mechanism DNA bind-                  extensions parts proteins altered by <br /> ing MADS-box proteins similar             presence &#226;?&#163;2. Indeed, &#226;?&#163;2-Mcm1-DNA structure, <br />  10     ACTON ET AL.                                                                                                              MOL. CELL. BIOL. <br />  <br />  <br /> the N-terminal extension trans Mcm1 monomer (in-             DNA bend angle circular permutation assay are <br /> teracting &#226;?&#163;2 monomer DNA strand)                 based assumption protein produces single <br /> makes minor groove contacts consequence inter-            bend plane. Therefore, assay, sensitive to <br /> actions trans &#226;?&#163;2 monomer (24). Another possible expla-         relative conformational changes DNA, able to <br /> nation result site used           distinguish contributions type bend towards <br /> experiments varies bases center site         calculated angle. The V34A S37A mutants clearly alter <br /> from used crystal structure. It possible   conformation Mcm1-DNA complex, this <br /> N-terminal extension adopts different conformations make           assay determine affect bend <br /> optimal contacts different sites. The fact posi-      protein, away protein, combination of <br /> tion contacts N-terminal extensions SRF Mcm1            these. However, mutants clearly DNA- <br /> differ greatly position extensions     binding affinity sole determinant Mcm1- <br /> Mcm1 monomers crystal structure significantly         mediated transcriptional activation. <br /> different suggests region considerable con-           An alternative explanation effects V34A and <br /> formational freedom. The conformational freedom ex-            S37A mutants DNA contacts, the <br /> tension important allowing MADS-box proteins            DNA bend, affect position conformation these <br /> recognize different sites complex cofac-       chains. It shown Mcm1 alters conforma- <br /> tors. Interestingly, amino acid chains 17 resi-     tion binding DNA complex &#226;?&#163;1 protein (25). <br /> dues Mcm1 phosphorylated specific conditions          Mutations residues H33 H41 little effect on <br /> vivo, hypothesized different isoforms        DNA-binding affinity complex &#226;?&#163;1 result in <br /> the protein recognize different target sites (12).                decreased levels &#226;?&#163;1-Mcm1-dependent transcriptional acti- <br />    Alanine scanning mutagenesis identified mutants         vation (4). These observations led suggestion that <br /> have altered DNA-bending activity: V34A, S37A, K40A.              region protein role &#226;?&#163;1-induced <br /> These mutants used study mechanism            conformational change Mcm1 (4). It possible V34 <br /> sequence-specific DNA bending. Our results examining DNA          S37 contacts DNA provide similar type of <br /> bending V34A mutant combination T7G site           change structure protein altered <br /> suggests V34 contacts position 7 interaction       conformation important transcriptional activation the <br /> contributes Mcm1-dependent DNA bending. The DNA-                   protein. <br /> bending activity S37A mutant decreased          It important mind address the <br /> the T8C mutation, result consistent model         activation DNA-binding activity Mcm1 alone. In the <br /> interaction S37 T8 contributes DNA-bend-           cell, Mcm1 interacts different proteins regulate <br /> ing activity Mcm1. These results correlate perfectly      diverse pathways. Although results suggest bending of <br /> crystal structure Mcm1 reveal relative importance          DNA overriding factor transcriptional activa- <br /> of interactions producing Mcm1-dependent             tion Mcm1 own, DNA bending play more <br /> DNA bending. Interestingly, bending associated           important role complex factors reg- <br /> K40A mutant does appear sequence specificity,             ulate different pathways. As observed SRF, bend- <br /> since bending wild-type mutant sites identical.        ing Mcm1 needed recruitment other <br /> This finding supported Mcm1 crystal structure,        protein factors promoter regions target genes (21). <br /> shows chain involved phosphate contact           Future studies utilizing mutants help provide deep- <br /> position T8. It possible V34 S37 dictate speci-       er understanding importance DNA bending tran- <br /> ficity bending K40 helps provide necessary            scriptional activation functions Mcm1 cell. <br /> change free energy stabilize DNA altered con- <br /> formation. Without contributions chains                                  ACKNOWLEDGMENTS <br /> wild-type levels DNA bending occur. Predictions               We thank George Sprague providing strains plasmids and <br /> based Mcm1 crystal structure work               Tim Richmond Song Tan providing coordinates of <br /> MADS-box proteins suggest K29 T66 con-              &#226;?&#163;2-Mcm1-DNA crystal structure helpful discussions. <br /> tribute DNA bending (24, 29). However, results indicate          A.M.S. recipient Howard Hughes Medical Institute Under- <br /> that residues contribute little Mcm1 function        graduate Summer Research Fellowship. This research supported <br /> vivo Mcm1-dependent DNA bending vitro.                          grant National Institutes Health (GM49265) to <br />    Our results wild-type protein ma-        A.K.V. <br /> jority mutants, good correlation the <br />                                                                                                        REFERENCES <br /> levels transcriptional activation DNA-binding affinity. <br />                                                                       1. Acton, T. B., H. Zhong, A. K. Vershon. 1997. DNA-binding specificity of <br /> However, observation V34A S37A, mutants               Mcm1: operator mutations alter DNA bending transcriptional ac- <br /> alter DNA bending little effect DNA-binding                tivities MADS box protein. Mol. Cell. Biol. 17:1881&#226;&#8364;&#8220;1889. <br /> affinity, slow-growth phenotypes suggests DNA               2. Affolter, M., J. Montagne, U. Walldorf, J. Groppe, U. Kolter, M. LaRosa, <br /> bending Mcm1 important essential function               W. J. Gehring. 1994. The Drosophila SRF homolog expressed a <br />                                                                          subset tracheal cells maps genomic region required for <br /> of protein. The fact mutations Mcm1              tracheal development. Development 120:743&#226;&#8364;&#8220;753. <br /> similar greater decreases DNA-binding affinity           3. Boeke, J. D., J. Trueheart, G. Natsoulis, G. R. Fink. 1987. 5-Fluoroorotic <br /> result slow growth similar reductions transcrip-           acid selective agent yeast molecular genetics. Methods Enzymol. 154: <br /> tional activation supports idea. However, results          164&#226;&#8364;&#8220;175. <br />       <br /> </body></html>