<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Apr. 1999, p. 2515&#226;&#8364;&#8220;2526                                                                               Vol. 19, No. 4 <br /> 0270-7306/99/$04.0010 <br /> Copyright &#194;&#169; 1999, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />              Esa1p Is Essential Histone Acetyltransferase Required <br />                             Cell Cycle Progression <br />                         ASTRID S. CLARKE,1 JOANNA E. LOWELL,1,2&#226;&#8364;&#160; SANDRA J. JACOBSON,1,3&#226;&#8364;&#161; <br />                                                                  1,3 <br />                                             AND LORRAINE PILLUS * <br />  <br />           Department Molecular, Cellular Developmental Biology1 Department Chemistry Biochemistry,2 <br />                  University Colorado, Boulder, Colorado 80309-0347, Department Biology, University of <br />                                      California San Diego, La Jolla, California 92093-03473 <br />                       Received 20 August 1998/Returned modification 10 October 1998/Accepted 8 December 1998 <br />  <br />              Histones dynamically modified chromatin assembly, specific transcriptional patterns es- <br />           tablished, mitosis development. Modifications include acetylation, phosphorylation, ubiquitina- <br />           tion, methylation, ADP-ribosylation, biological significance of understood. <br />           For example, distinct acetylation patterns correlate nucleosome formation transcriptionally ac- <br />           tivated silenced chromatin, mutations genes encoding yeast histone acetyltransferase (HAT) ac- <br />           tivities result cellular <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span>phenotype</span> modest growth defects. Here report characterization <br />           ESA1, essential gene member MYST family includes yeast silencing genes, human <br />           genes associated leukemia human immunodeficiency virus type 1 Tat protein, Drosophila <br />           mof, gene essential male dosage compensation. Esa1p acetylates histones <span id='am-316' about='xsp:pattern' typeof='owl:Thing'>pattern</span> distinct those <br />           yeast enzymes, temperature-sensitive mutant alleles abolish enzymatic activity vitro result <br />           partial loss acetylated isoform histone H4 vivo. Strains carrying mutations blocked <br />           cell cycle restrictive temperatures, esa1 mutants succeed replicating DNA fail <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br />           proceed normally mitosis cytokinesis. Recent studies Esa1p enhances transcription vitro <br />           modulate expression genes important cell cycle control. These observations link <br />           essential HAT activity cell cycle progression, potentially discrete transcriptional regulatory events. <br />  <br />  <br />    Cytogenetic biochemical differences transcrip-                  assembly chromatin general setting tran- <br /> tionally active inactive chromatin long recog-                   scriptional states specific loci entire chromosomes (21, <br /> nized (for reviews, references 21, 60, 66, 71). Particular             40, 60). Progress defining roles acetylation came <br /> focus defining distinctions core his-                 mapping mutating sites core histones that <br /> tones, context nucleosome serve                     modified recently identification the <br /> scaffold packaging DNA chromatin. Ultimately,                   enzymes, histone acetyltransferases (HATs) deacetylases, <br /> chromatin serves vivo template RNA                   catalyze modifications (for reviews references 40, <br /> transcription DNA replication occur,                    49, 60). The yeast gene HAT1 encode en- <br /> central importance define histones function reg-                 zyme acetylates primarily newly synthesized histone H4, <br /> ulation processes.                                                    potentially assembly nucleosomes (29, 48). <br />    Previous biochemical genetic studies recent                     Hat1p associates Hat2p, Rba48-like protein con- <br /> high-resolution structure nucleosome (37) point                  tributes substrate specificity enzyme (48). Although <br /> amino-terminal tails conserved core histones impor-                  phenotypes observed yeast cells harboring mu- <br /> tant effectors regulation. Extending nucleosomal                  tations genes, similar HAT complex exists human <br /> particle, tails participate internucleosomal interactions             cells (62), suggesting conserved activities function- <br /> and posttranslationally modified result local dif-                ally important. <br /> ferences chromatin structure function influencing                       The transcription-linked HAT identified Tetra- <br /> accessibility activity polymerases regulatory                 hymena homologue yeast Gcn5p (9). GCN5 es- <br /> proteins. Understanding modifications crucial                 sential yeast, necessary transcriptional activa- <br /> understanding cell- gene-specific functions. For example,                  tion genes (17), mutational analyses demonstrate <br /> phosphorylation histone H3 serine 10 occurs late G2                   HAT activity correlates transcriptional function (33, <br /> and proposed promote binding factors                   65). Although Gcn5p provides catalytic activity, only <br /> drive chromatin condensation cells enter mitosis (24).                      component large multisubunit HAT complexes that <br />    Among observed histone modifications, acetylation                   exist cell (19). Interestingly, noncatalytic subunits are <br /> been thoroughly studied, contribute                    important vivo function complexes, suggesting <br />                                                                                finely regulated catalysis speci- <br />                                                                                ficity. <br />    * Corresponding author. Present address: Department Biology,                Many proteins HAT activity iden- <br /> University California San Diego 0347, La Jolla, CA 92093-0347.           tified; include p300 CREB binding protein CBP <br /> Phone: (619) 822-2442. Fax: (619) 534-0555. E-mail: lpillus@biomail <br /> .ucsd.edu. <br />                                                                                (4, 43), SRC-1 family proteins (10, 59), basal transcrip- <br />    &#226;&#8364;&#160; Present address: Laboratory Molecular Parasitology, Rockefel-          tion factor TAFII250 (41). The biological significance the <br /> ler University, New York, NY 10021.                                            activities proteins defined fully, is <br />    &#226;&#8364;&#161; Present address: Department Biology, University California          likely HAT activity common feature gene- <br /> at San Diego, La Jolla, CA 92093-0347.                                         specific basal regulation. <br />  <br />                                                                         2515 <br />  2516       CLARKE ET AL.                                                                                                                          MOL. CELL. BIOL. <br />  <br />  <br />    Sequence analysis identifies superfamily proteins                        Protein expression. Exponentially growing Escherichia coli BL21(DE3)pLysS <br /> that known predicted acetyltransferases (re-                          cells transformed pLP820 (pRSETc vector [Invitrogen]), pLP831 <br />                                                                                     (pRSETc-ESA1), pLP1138 (pRSETc-esa1-L254P), pLP942 (pRSETc-esa1-414), <br /> viewed reference 42). For known                       pLP832 (pRSETc-esa1-L327S) induced addition isopropyl-b-D- <br /> functions, provocative connections                           thiogalactopyranoside final concentration 0.5 mM. Cells harvested <br /> chromatin. We especially interested SAS                            3 h 25&#194;&#176;C, lysate prepared freeze-thaw cycle followed by <br /> (something silencing) genes MYST family (6),                           sonication buffer (20% sucrose, 30 mM Tris [pH 8.0], 1 mM EDTA, 1 mM phe- <br />                                                                                     nylmethylsulfonyl fluoride, 100 ml optical density 600 nm 0.25). Super- <br /> which identified roles transcriptional si-                  natants total cell equivalent lysates clarified centrifugation <br /> lencing yeast Saccharomyces cerevisiae (15, 53). Al-                         used HAT assays. <br /> though SAS2 SAS3 exhibit significant sequence similarity,                          Acetyltransferase activity assays. Fifty micrograms calf thymus histones <br /> they distinct mutant phenotypes suggesting                       (Sigma), 20 mg yeast histones (a gift J. Pilon P. Laybourn), 500 mg <br />                                                                                     unacetylated bovine serum albumin (BSA; Amresco) combined reac- <br /> either promote silence transcription, depending lo-                       tion buffer (32) 15 mg recombinant Esa1p, Esa1-L327Sp, pRSETc total <br /> cus. Both proteins contain A motif, relatively short se-                      soluble cell lysate 0.1 mCi 3H-acetyl coenzyme A (2.3 Ci/mmol; ICN). <br /> quence contribute acetyl coenzyme A binding (42),                       Following incubation 30&#194;&#176;C 10 min, samples processed described <br /> although demonstrated catalytic ac-                        (7) prior scintillation counting. Approximately equivalent as <br /> tivity. We identified yeast SAS family member,                         sevenfold Esa1-L327Sp Esa1p used HAT assays based quan- <br />                                                                                     titation Coomassie blue-stained gels, using Alpha Digital Imaging System <br /> ESA1, encodes essential HAT. Further, Esa1p func-                           2000 program. For fluorography, aliquots HAT assay reactions sep- <br /> tions catalytic subunit 1.3-MDa NuA4 complex                          arated sodium dodecyl sulfate (SDS)&#226;&#8364;&#8220;18% polyacrylamide gel stained <br /> that acetylates free histones chromatin and, notably,                      Coomassie brilliant blue. The gel treated fluorography In- <br /> can promote transcription vitro (14). Our analysis reces-                     tensify enhancer (NEN) processed according manufacturer&#226;&#8364;&#8482;s instructions. <br />                                                                                        Microsequencing. HAT assays performed essentially described above <br /> sive conditional alleles reveals ESA1 important pro-                    3 mg synthetic peptide corresponding N terminus histone <br /> gression cell cycle, cells grown restric-                   H4, H3, H2A combined 3 mg recombinant Esa1p total soluble cell <br /> tive conditions arrest G2/M DNA content partial                          lysate 1.0 mCi 3H-acetyl coenzyme A (2.58 Ci/mmol; NEN). Reaction <br /> depletion acetylated form histone H4. The cell cycle                       mixes incubated 30 min 22&#194;&#176;C (H4 peptide) 3 h 15&#194;&#176;C (H3 and <br />                                                                                     H2A peptides). Individual reactions microsequenced described else- <br /> arrest observed loss ESA1 function dependent                          (32, 58) Protein Chemistry Core Facility Baylor School of <br /> the checkpoint gene RAD9. These observations link                         Medicine. H4 H2A peptides gifts M. Parthun D. Gottschling. <br /> specific histone modifications cell cycle control, potentially                   The H3 peptide synthesized Applied Biosystems instrument, using <br /> mediated transcriptional regulation.                                        standard chemistry manufacturer&#226;&#8364;&#8482;s recommendations. <br />                                                                                        Flow cytometry microscopic analyses. Flow cytometry 10,000 cells/ <br />                                                                                     sample performed described (69). For 49,6-diamidino-2-phe- <br />                                                                                     nylindole (DAPI) staining, cells fixed 70% acetone&#226;&#8364;&#8220;30% methanol dry <br />                         MATERIALS AND METHODS                                       ice 1 h, washed twice phosphate-buffered saline (PBS), then <br />                                                                                     stained DAPI (Boehringer Mannheim) final concentration 0.25 <br />    Yeast methods. An esa1D::HIS3 null <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-243' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-244' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-264' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-266' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-271' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-280' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-286' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-287' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-292' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-294' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-297' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-299' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-312' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span>mutation</span> generated S288c <br />                                                                                     mg/ml. For indirect immunofluorescence, 37% formaldehyde added cul- <br /> diploid strain LPY2553 (MATa/MATa his3D200/his3D200 leu2-3,112/leu2-3,112 <br />                                                                                     tures final concentration 5%. Samples agitated 5 min their <br /> trp1D1/trp1D1 ura3-52/ura3-52) create LPY2639 (MATa/MATa his3D200/ <br />                                                                                     relevant experimental temperatures fixed additional 45 min at <br /> his3D200 leu2-3,112/leu2-3,112 trp1D1/trp1D1 ura3-52/ura3-52 esa1D::HIS3/ESA1) <br />                                                                                     22&#194;&#176;C. Cells washed PBS, washed twice 1.2 M sorbitol PBS <br /> according standard procedures (5). Temperature-sensitive alleles gene- <br />                                                                                     (SPBS), resuspended SPBS 24 mM b-mercaptoethanol and <br /> rated amplifying ESA1 gene using Taq polymerase conditions rec- <br />                                                                                     Zymolyase 70000 (ICN) final concentration 29 mg/ml. After 30 min at <br /> ommended manufacturer (Promega), deoxynucleoside triphos- <br />                                                                                     30&#194;&#176;C, samples washed SPBS twice PBS. Staining per- <br /> phates used final concentration 250 mM. Products subcloned <br /> into TRP1-CEN vector tested complementation esa1 null strain       formed poly-L-lysine-coated multiwell slides. Samples preblocked (PBS <br /> LPY2641 (MATa his3D200 leu2-3,112 trp1D1 ura3-52 esa1D::HIS3 bearing             10 mg BSA ml 0.5% Nonidet P-40). Rabbit anti-Kar2p serum was <br /> ESA1-URA3-CEN plasmid) plasmid shuffle (23). Temperature-sensitive al-           used dilution 1:1,000, rat antitubulin serum (YOL1/34; Serotec, Inc.) <br /> leles identified having complementation defects 34&#194;&#176;C above.          used 1:2,000. Primary incubations 1 h 37&#194;&#176;C followed PBS <br /> Alleles sequenced using Applied Biosystems automated facility.           washes. Secondary antibodies, fluorescein isothiocyanate-conjugated goat anti- <br />    Three alleles, esa1-L327S, esa1-L254P, esa1-414, integrated accord-     rabbit (Jackson Laboratories) 1:300 Texas red-conjugated donkey anti-rat <br /> ing standard procedures (23) LPY2639 esa1D::HIS3 locus URA3-      (Jackson Laboratories) 1:50, incubated 22&#194;&#176;C 30 min. After PBS <br /> marked integrating vectors pLP951, pLP949, pLP952, creating LPY3425,            washes, DAPI added described above. After washing, slides mounted <br /> LPY3454, LPY3443, respectively. Integration events confirmed            Citifluor (Ted Pella, Inc.), staining visualized Zeiss Axiophot flu- <br /> standard methods, diploids LPY3425 LPY3454 sporulated               orescence microscope, images collected COHU charge-coupled <br /> create haploids carrying following alleles experiments: esa1-   device camera using Scion Image 1.57 program. For electron microscopy in <br /> L327S (LPY3430 MATa his3D200 leu2-3,112 trp1D1 ura3-52 esa1D::HIS3 esa1-            collaboration Tom Giddings University Colorado, samples were <br /> L327S::URA3), ESA1 (LPY3431 MATa his3D200 leu2-3,112 trp1D1 ura3-52                 prepared described (44). Sections 80 nm cut examined <br /> ESA1), esa1-L254P (LPY3500 MATa his3D200 leu2-3,112 trp1D1 ura3-52 esa1D::          Philips CM10 microscope 80 kV magnification 310,500 <br /> HIS3 esa1-L254P::URA3), ESA1 (LPY3498 MATa his3D200 leu2-3,112                  325,000. Images collected using Gatan BioScan camera Gatan <br /> trp1D1 ura3-52 ESA1). Strains rad9D esa1-L254P mutant analysis gen-        Digital Micrograph 2.5 (PPC) software. <br /> erated crossing LPY3500 LPY3719 (MATa ade2-101 his3D200 lys2-801                Immunoblotting analysis. Exponentially growing cells split, pelleted, and <br /> trp1D1 ura3-52 rad9D::HIS3) create diploid LPY3779 (MATa/MATa                resuspended 28 37&#194;&#176;C medium incubated appropriate temper- <br /> ADE2/ade2-101 his3D200/his3D200 leu2,3-112/LEU2 LYS2/lys2-801 trp1D1/trp1D1         ature 4 h (except LPY3430, incubated 8 h). Protein 5 3 <br /> ura3-52/ura3-52 esa1D::HIS3 esa1-L254P::URA3/ESA1 RAD9/rad9D::HIS3); af-            108 cells recovered glass bead lysis. Briefly, cells washed lysis <br /> ter sporulation dissection, haploids following mutations      buffer (0.3 M sorbitol, 0.1 M NaCl, 5 mM MgCl2, 10 mM Tris [pH 7.5]), the <br /> recovered tetrads 2 HIS:2 4 HIS:0 segregation         pellets frozen 220&#194;&#176;C resuspended 0.4 0.8 ml lysis buffer <br /> pattern: rad9D esa1-L254P (LPY3780 MATa ade2-101 his3D200 LEU2 lys2-801             containing 1 mg tosylsulfonyl phenylalanyl chloromethyl ketone (TPCK), <br /> trp1D1 ura3-52 rad9D::HIS3 esa1D::HIS3 esa1-L254P::URA3), rad9D (LPY3784            leupeptin, pepstatin A, aprotinin ml 0.2 mM phenylmethylsulfonyl flu- <br /> MATa ade2-101 his3D200 leu2,3-112 LYS2 trp1D1 ura3-52 rad9D::HIS3),             oride. Acid-washed glass beads added, cells vortexed times <br /> esa1-L254P (LPY3785 MATa ADE2 his3D200 LEU2 lys2-801 trp1D1 ura3-52                 1-min bursts. Extracted protein quantitated Bio-Rad protein assay, <br /> esa1D::HIS3 esa1-L254P::URA3). Additional strains used included LPY2991,          diluted sample buffer, boiled 5 min, loaded SDS&#226;&#8364;&#8220;18% <br /> MATa esa1D::HIS3 isolate LPY2639, CEN-ESA1 plasmid (pLP795);            polyacrylamide gel. The separated proteins transferred nylon mem- <br /> LPY3291, LPY2991 plasmid (pLP863)-borne esa1-414            brane, blocked TBSTM (10 mM Tris [pH 8.0], 150 mM NaCl, 0.05% Tween <br /> replaces ESA1 plasmid; LPY3068, LPY2991 ESA1         20, 2% dry milk) 1 h room temperature, incubated antiserum <br /> a TRP1-2mm plasmid, pLP798, replaces pLP795; LPY4184, LPY2991           directed histone H4-acetylated Lys5 residue (1:500 TBSTM; Se- <br /> but MATa isolate esa1-414 TRP1-2mm plasmid, pLP1128, re-            rotec) antiserum directed total histones (0.5 mg/ml TBSTM; Boehr- <br /> places pLP795; mad3D strain (LPY4222 MATa leu2,3-112 his3D200 trp1D1              inger-Mannheim) 8 h 4&#194;&#176;C. Blots washed, incubated horseradish <br /> ura3-52 mad3D::URA3); mad3D esa1-L254P strain (LPY4223 MATa                   peroxidase-conjugated goat anti-rabbit goat anti-mouse antiserum (1:10,000 <br /> leu2,3-112 his3D200 trp1D1 ura3-52 mad3D::URA3 esa1D::HIS3 esa1-L254P::             1:2,500, respectively; Promega) 1 h room temperature, washed, and <br /> URA3).                                                                              processed chemiluminescence described (56). <br />  VOL. 19, 1999                                                                           ESSENTIAL HAT AND CELL CYCLE CONTROL                     2517 <br />  <br />  <br />                                                                                     mixtures resolved using SDS&#226;&#8364;&#8220;18% polyacrylamide gels <br />                                                                                     stained (Fig. 2A) subjected fluorography (Fig. <br />                                                                                     2B). Recombinant Esa1p acetylated histone substrates, the <br />                                                                                     pattern incorporation distinct Gcn5p. <br />                                                                                     Esa1p acetylated primarily histone H4 lesser extent <br />                                                                                     H3 H2A. In contrast, Gcn5p active H3, <br />                                                                                     longer exposures, published reports (32), <br />                                                                                     weak H4 acetylation observed. In parallel liquid HAT <br />                                                                                     assays performed extracts prepared bacteria trans- <br />                                                                                     formed vector alone, histones BSA <br />                                                                                     substituted histones, acetylation observed (Fig. 2C). <br />                                                                                     Therefore, Esa1p&#226;&#8364;&#8482;s activity specific histones these <br />                                                                                     reaction conditions. To identify lysines acetylated re- <br />                                                                                     combinant Esa1p, used synthetic peptides corresponding to <br />                                                                                     lysine-rich amino-terminal tails yeast H4, H3, H2A <br />                                                                                     substrates standard reactions. The labeled peptides were <br />    FIG. 1. Esa1p sequence similarity (gray boxes) human Tip60          subjected microsequencing coupled direct de- <br /> D. melanogaster MOF. A region strong sequence similarity acetyltrans-         termination radioactivity incorporated position of <br /> ferases including yeast Hat1p Gcn5p indicated (black boxes). Similar     peptide (Fig. 2D F). The results clearly Lys5 <br /> identical residues region highlighted. Residues conserved among <br /> known predicted acetyltransferases A motif defined Neuwald         major site Esa1p acetylation yeast H4 pep- <br /> and Landsman (42) noted asterisks. Positions temperature-sensitive      tide. Lysines positions 8, 12, 16 acetylated but <br /> esa1 alleles (see text) noted arrows Esa1p cartoon; position   efficiently, demonstrating acetylatable <br /> of esa1-L327S allele shown arrow sequence expansion.       lysines H4 tail potential target Esa1p. For H3, <br /> The conserved G3E mutation mof (25) marked arrowhead below <br /> the sequence. An N-terminal chromo domain (1, 25, 31, 47)         Lys14 major target, modest acetylation Lys4. <br /> three SAS family members (diagonal stripes). The number amino acids      The yeast H2A peptide acetylated predominantly Lys4 <br /> protein indicated right.                                                  moderately Lys7. This pattern substrate specificity is <br />                                                                                     distinct previously defined yeast HATs and <br />                                                                                     agrees independent results Esa1p obtained assays <br />                                  RESULTS                                            using purified histones, including nonyeast proteins, sub- <br />                                                                                     strates (57). For example, deposition-related Hat1p acety- <br />    ESA1 essential gene MYST family similar-                       lates H4, predominantly Lys12, conditions <br /> ity acetyltransferases. During analysis yeast SAS2                    reduced specificity Lys5 acetylated H2A (29, 48). <br /> SAS3 silencing genes (53), discovered another,                         The transcription-linked HAT, Gcn5p, acetylate H4 on <br /> similar previously uncharacterized open reading frame                           Lys16 significant preference Lys14 histone H3 in <br /> (ORF) (YOR244w) existed yeast chromosome XV. This                                vitro (32). H2A substrate Gcn5p. However, Lys4 of <br /> ORF predicted encode 445-amino-acid protein (Fig. 1)                        H2A identified previously sole vivo site acet- <br /> with sequence features include acetyltransferase motif                      ylation murine leukemia L1210 cells (46), date <br /> and amino-terminal chromo domain, motif                          relevant L1210 HAT reported. Thus, is <br /> proteins thought regulate transcription chromatin                      clear ESA1 encodes HAT distinct substrate speci- <br /> association (1, 31, 47). YOR244w similar [BLASTP (3)                        ficities, indicate novel multiple cellular <br /> scores P 5 102100 better] human MYST family                            functions. To begin understand functions, iden- <br /> proteins Tip60 MOZ Drosophila melanogaster                              tified conditional alleles loss-of-function phenotypes <br /> MOF protein (6, 25, 26). To investigate function                             examined. <br /> YOR244w, constructed <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span>null allele</span> deleting copy                          Temperature-sensitive esa1 mutants lack HAT activity in <br /> the ORF (5) diploid yeast strain. This diploid sporu-                      vitro deplete cellular pools histone H4 acetylated on <br /> lated, spores tetrad, formed                     Lys5 vivo. We screened library plasmids encoding ESA1 <br /> colonies. Microscopic analysis demonstrated spores                    amplified using standard procedures. We <br /> germinated approximately kinetics, cells                     recovered plasmids fully complemented lethality of <br /> two spores tetrad stopped growing                    strains esa1D allele permissive temperatures (28&#194;&#176;C <br /> four rounds division. Genotyping revealed surviving                     below) failed complement restrictive temper- <br /> spores wild type YOR244w, plasmid-borne                          atures (34&#194;&#176;C above). Twenty-two esa1 alleles met cri- <br /> copy wild-type gene rescued lethality associated                    teria, growth phenotypes strains harboring reces- <br /> the deletion. Thus, YOR244w gene product essential                       sive esa1 mutants shown Fig. 3. These mutant <br /> cellular viability, named ESA1, essential SAS                    alleles integrated normal chromosomal <br /> family acetyltransferase.                                                           locus ESA1. The esa1-414 phenotype severe that <br />    Esa1p acetylates histones distinct specificity pattern.                   haploid strains allele viable mu- <br /> A clue cellular role ESA1 came motif                       tated gene plasmid borne presumably present slightly <br /> it shares yeast proteins demonstrated                         increased gene dosage. In esa1-414, deletion single nucle- <br /> vitro catalytic acetyltransferase activity, Hat1p, thought                  otide position 1887 leads <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-166' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-179' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-184' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-191' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-204' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-215' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-230' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-234' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span>frameshift mutation</span> codon <br /> to function chromatin assembly, Gcn5p, functions                       414, altering 10 amino acids terminating ORF 22 <br /> in transcription (9, 29, 48). To test Esa1p HAT activity                     amino acids prematurely. The esa1-L254P esa1-L327S mu- <br /> in vitro, cloned gene vector allowing inducible                   tations substitute proline serine conserved leucines, and <br /> expression E. coli. Extracts prepared bacteria express-                     notably esa1-L327S substitution lies acetyl- <br /> ing Esa1p Gcn5p tested HAT activity standard                       transferase domain itself (Fig. 1). All mutant strains <br /> assay (7, 32) mixing extracts 3H-acetyl coenzyme A                      grew permissive temperatures poorly elevated <br /> and histones yeast calf thymus. These reaction                       temperatures (Fig. 3), suffering 60 80% lethality within <br />  2518                                CLARKE ET AL.                                                                                                                           MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />                                 A                        Esa1p                 Gcn5p                           B                               Esa1p               Gcn5p <br />                                                          yeast         calf yeast calf                                                        yeast      calf yeast           calf <br />  <br />                                                                                                          H3 <br />                                                                                                          H2B <br />                                                                                                          H2A <br />                                                                                                          H4 <br />                                                                   Coomassie stain                                                                       fluorogram <br />                                 C                                                                              D <br />                                                  12 <br />                               3 <br />                               H-acetate (cpm) x 10 <br />  <br />  <br />  <br />  <br />                                                  10                                                                                      30 <br />                                                                                                                                                            Yeast H4 peptide <br />                                                                                                                3 <br />                                                                                                                   H-acetate (cpm) x 10 <br />                                                      8                                                                                   25 <br />  <br />                                                                                                                                          20 <br />                                                      6 <br />                                                                                                                                          15 <br />                                                      4 <br />                                                                                                                                          10 <br />  <br />                                                      2                                                                                    5 <br />                               3 <br />  <br />  <br />  <br />  <br />                                                                                                                3 <br />  <br />  <br />  <br />  <br />                                    0                                                                                                      0 <br />                               histones                        +    +   +   +    -   -    -      -                                             SGRGKGGKG L GKGGA K RHR K I L <br />                                                                                                                                                     5     8    12     16       20 <br />                               BSA                             -    -   -   -    +   +    +      +                                                cycle number/amino acid residue <br />                               vector                          -    +   -   -    -   +    -      - <br />                               Esa1p                           -    -   +   -    -   -    +      - <br />                               L327S                           -    -   -   +    -   -    -      + <br />                                 E                                                                              F <br />                               20                                                                                                         20 <br />                                                                                Yeast H3 peptide                                                            Yeast H2A peptide <br />      3 <br />  <br />  <br />  <br />  <br />                                                                                                               3 <br />        H-acetate (cpm) x 10 <br />  <br />  <br />  <br />  <br />                                                                                                                H-acetate (cpm) x 10 <br />  <br />  <br />  <br />  <br />                               15                                                                                                         15 <br />  <br />  <br />                               10                                                                                                         10 <br />  <br />  <br />                                5                                                                                                          5 <br />      3 <br />  <br />  <br />  <br />  <br />                                                                                                               3 <br />  <br />  <br />  <br />  <br />                                0                                                                                                          0 <br />                                                      A R T K QT A R K S T GG K A PRK Q L A S K AA RK S                                        S GGKGG KAGSA A KA SQ S R S A K AG <br />                                                           4        9       14    18        23       27                                            4     7        13             21 <br />                                                               cycle number/amino acid residue                                                    cycle number/amino acid residue <br />    FIG. 2. Recombinant Esa1p acetylates histones. Products HAT assays using yeast calf histones substrates resolved SDS&#226;&#8364;&#8220;18% <br /> polyacrylamide gel stained Coomassie brilliant blue (A) treated fluorography (B). The activities recombinant Esa1p compared that <br /> of recombinant Gcn5p; Esa1p preferentially acetylates H4, Gcn5p preferentially acetylates H3 (32). (C) Bacterial extracts containing equivalent amounts of <br /> recombinant Esa1p, vector control, recombinant Esa1-L327Sp incubated 3H-acetyl coenzyme A calf histones BSA. Esa1p acetylates calf <br /> histones BSA. No activity observed vector control Esa1-L327Sp lysates mutant proteins tested (data shown). The preferred <br /> targets Esa1p Lys5 histone H4 (D), Lys14 histone H3 (E), Lys4 Lys7 histone H2A (F). Note lysines 8, 12, 16 H4 acetylated. <br /> When repetitive yields cycles Edman degradation calculated, extent acetylation sites equivalent significantly than <br /> that Lys5. <br />  VOL. 19, 1999                                                                                ESSENTIAL HAT AND CELL CYCLE CONTROL                                  2519 <br />  <br />  <br />  <br />                                28&#203;&#353;C                                                                                        37&#203;&#353;C <br />                                                                               ESA1 <br />                                                                        esa1-L254P <br />                                                                        esa1-L327S <br />                                                                           esa1-414 <br />   FIG. 3. esa1 mutants temperature sensitive. Temperature sensitivity observed plating wild-type ESA1 (LPY3498) esa1 mutant (esa1-L254P <br /> [LPY3500], esa1-L327S [LPY3430], esa1-414 [LPY3291]) strains fivefold serial dilutions 28&#194;&#176;C (left) 37&#194;&#176;C (right). All alleles recessive mutations. <br />  <br />  <br /> the 4 h incubation. When recombinant protein en-                              reactive material comparable. Although loss Lys5 <br /> coded esa1 mutant alleles tested standard liquid                             acetylation correlated loss viability esa1 mutant <br /> HAT assays using equivalent excess amounts                                strains, likely sites H4, histone targets, <br /> mutant compared wild-type protein, activity de-                              cumulative effects loss acetylation Esa1p&#226;&#8364;&#8482;s <br /> tected (Fig. 2C data shown) temperature                                 targets contribute essential function Esa1p. <br /> tested 1 37&#194;&#176;C.                                                                 These possibilities distinguished future studies, it <br />    To determine loss HAT activity vitro cor-                         clear Fig. 2 4 esa1 temperature-sensitive <br /> related changes histone acetylation vivo, assayed                        alleles encode proteins defective HAT activity both <br /> protein extracts prepared mutant control strains                           vitro vivo. <br /> immunoblotting. In particular, used antiserum specific                              esa1 mutant strains defective cell cycle control. To <br /> for acetylated Lys5 residue histone H4, preferred                           explore nature striking loss viability that <br /> site recombinant Esa1p activity, compare relative levels                        correlates loss HAT activity, used variety cell <br /> of acetylation wild-type mutant strains permissive                       biological techniques compare wild-type ESA1 esa1 <br /> restrictive temperatures (Fig. 4). Each esa1 mu-                          temperature-sensitive strains. Mutant wild-type cells were <br /> tants detectable levels acetylated isoform                           grown permissive temperature split that <br /> permissive temperature, modest reduc-                              portion shifted restrictive temperature. At <br /> tions relative wild-type control. Under restrictive con-                        multiple time points shift 37&#194;&#176;C, compared the <br /> ditions, mutant strains decreased levels                            cells morphology DNA content. The mutant cells be- <br /> acetylation, severely affected mutant, esa1-                         came distinctly blocked cell cycle restrictive tem- <br /> 414 strain, showing greatest reduction. This reduction                         perature. As shown Fig. 5, 4 h temperature shift, <br /> not simply loss viability, extracts prepared                         ESA1 cells dividing normally peaks, indicative <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> dam1 temperature-sensitive mutant suffers                             G1 G2/M positions cycle, easily distin- <br /> significant loss viability conditions (25a)                         guished flow cytometric analysis. In contrast, mutant <br /> wild-type levels Lys5 acetylation (data shown). The                             cells stopped dividing shifted 37&#194;&#176;C accumu- <br /> change Lys5-acetylated H4 does simply reflect de-                             lated G2/M. This terminal arrest phenotype observed for <br /> crease cellular pools H4. When samples wild-                         esa1 temperature-sensitive strain examined, the <br /> type mutant proteins prepared 37&#194;&#176;C probed                             time required achieve arrest varies alleles (data <br /> an antibody evaluate total histones, levels H4-immuno-                           shown). Because mutant cells arrest G2/M DNA <br />  <br />  <br />  <br />  <br />    FIG. 4. esa1 mutants display conditional decreases level histone H4-Lys5 acetylation vivo. Whole-cell lysates wild-type (WT; lanes 2, 6, 10) <br /> or esa1 temperature-sensitive (lanes 3 5, 7 9, 11 13) strains grown 28&#194;&#176;C (lanes 2 5) 37&#194;&#176;C (lanes 6 13) separated SDS&#226;&#8364;&#8220;18% <br /> polyacrylamide gel transferred nylon probed antiserum directed Ly5-acetylated H4 (A) probed control antiserum directed <br /> against histones (B; upper panel shows H4 immunoreactive band) stained Coomassie brilliant Blue (B, lower panel). Purified yeast histones (YH; approximately <br /> 6 mg; lane 1) run parallel. esa1-L254P (LPY3500; lanes 3, 7, 11), esa1-414 (LPY3291; lanes 4, 8, 12), esa1-L327S (LPY3430; lanes 5, 9, 13) <br /> strain displayed decreased Lys5-acetylated immunoreactivity nonpermissive temperature 37&#194;&#176;C. Arrowheads denote migration histone H4. <br />  2520     CLARKE ET AL.                                                                                                         MOL. CELL. BIOL. <br />  <br />  <br /> content, loss ESA1 function apparently does interfere <br /> with DNA synthesis, esa1 mutants unable pro-         28&#203;&#353;C         ESA1                                     esa1 <br /> ceed mitosis cytokinesis. <br />    To evaluate morphology mutant arrest, cells from <br /> the wild-type strain mutants shown Fig. 3 were <br /> fixed stained DAPI visualize DNA (Fig. 6). The <br /> wild-type ESA1 cells observed positions cell <br /> cycle, normal populations 37% unbudded, 27% small- <br /> budded, 36% large-budded cells. Apparently normal mi- <br /> totic cells, DAPI-staining material extended between <br /> mother daughter buds, seen 8% mutant <br /> cells examined. More 70% mutant cells ar-          37&#203;&#353;C <br /> rested large bud. Of these, 75% contained single <br /> region DAPI-stained chromatin; remaining 25% con- <br /> tained appeared fragmented chromatin. Interest- <br /> ingly, chromatin mother the <br /> daughter bud (data shown) determined Calcofluor <br /> staining (51), indicating normal nuclear migration occur- <br /> ring early mitosis (45, 74) inhibited. Variants the <br /> large-bud arrested mutant cells, additional bud has <br /> formed, observed ,10% population (Fig. 6, <br /> right panels mutants). In cases, DAPI staining was <br /> sometimes seen buds, bud <br />                                                                            |       |                               |       | <br />                                                                           1C      2C                              1C      2C <br /> usually remained unstained. Together, large-budded             FIG. 5. esa1 temperature-sensitive mutants arrest G2/M phase the <br /> variant large-budded forms comprised approximately 80%           cell cycle DNA replication. Wild-type ESA1 (LPY3498) (left) mutant <br /> the esa1 cells restrictive temperature. For       esa1-L254P (LPY3500) (right) cells incubated 28 37&#194;&#176;C 4 h and <br />                                                                     analyzed DNA content flow cytometry. The x y axes represent relative <br /> mutants, increased incubation time restrictive condi-         fluorescence intensity number cells, respectively. esa1 mutants arrested <br /> tions led increased cell volume. We interpret continued      G2/M DNA content DNA replication restrictive temperature of <br /> growth arrested mutant cells indicate Esa1p       37&#194;&#176;C. This mutant phenotype observed esa1-L327S, esa1-414, and <br /> not required cell growth instead necessary cell      temperature-sensitive alleles tested (data shown). The time to <br />                                                                     reach arrest varied somewhat depending allele ESA1. <br /> cycle progression. The observation mutant cells un- <br /> dergo cell cycle arrest replicated DNA shift to <br /> the restrictive temperature reveals Esa1p function      periment prepared electron microscopy using rapid- <br /> required G2 mitosis.                                      freeze protocol (44). Clear differences observed thin <br />    To establish failure mitosis, examined     sections wild-type mutant cells (Fig. 8). Images of <br /> cells indirect immunofluorescence microscopy using anti-         large-budded cells shown comparison, although <br /> tubulin (27, 28) anti-Kar2p (54) staining markers        represent 36% ESA1 population. The <br /> mitotic cytoplasmic microtubules nuclear enve-          cytoplasm esa1 mutant cells consistently observed to <br /> lope-endoplasmic reticulum (ER) membranes (Fig. 7). In wild-        highly vesiculated wild-type cells (Fig. <br /> type cells temperatures esa1 cells permis-    8C), striking differences observed substruc- <br /> sive temperature, staining appeared normal consistent       ture nuclei. In wild-type cells, nucleus relatively <br /> with cell&#226;&#8364;&#8482;s position cell cycle inferred DAPI      uniform density, increased density restricted the <br /> staining. At restrictive temperature,       fairly compact heterochromatic nucleolus (Fig. 8A B). In <br /> 80% abnormal cells esa1 population shown here,           dividing cell, electron-dense material seen at <br /> anti-Kar2p staining encircled DAPI-stained chromatin            poles lobed nucleus extending mother and <br /> (Fig. 7) occasionally extended fingers        daughter cells. However, esa1 cells, nucleus was <br /> cytoplasm bud cell pair lacking DAPI       generally restricted mother-daughter pair <br /> staining. This staining pattern consistent expected    (Fig. 8C D), confirming results light microscopy. <br /> for nuclear envelope ER (54). The microtubular struc-       The mitotic spindles appeared structurally normal, including <br /> tures mutant cells generally oriented long     short bipolar spindles connecting microtubules nor- <br /> axis cell usually region intense         mally nucleated cytoplasmic microtubules. However, some <br /> staining DAPI-stained nucleus, some-          instances, spindle oriented aberrantly even <br /> what lighter, occasionally forked structures extended      observed lie horizontally bud neck than <br /> other bud cell. We interpret spindle cyto-   extending it. The nuclei distinct <br /> plasmic microtubules, respectively, comparable mi-      nucleolus exaggerated, electron- <br /> crotubule structures mutants G2/M arrests (18).       dense material consistently scattered nu- <br /> For wild-type large-budded cells, nucleus migrated          cleus (Fig. 8D) remaining restricted nucle- <br /> between mother daughter buds, long mitotic spindles        olar compartment. This dispersal detected various <br /> stretched them, occasionally short bundles           degrees severity 96% 65 mutant nuclei exam- <br /> cytoplasmic microtubules visible extended cyto-       ined, observed ,1% wild-type nuclei <br /> plasmic faces spindle pole bodies. The mutant cells     seen ultrastructural analyses of <br /> never observed long spindles, consistent fail-     comparably prepared cdc mutants arrest large <br /> ure nucleus migrate normally mother           buds (43, 70a). <br /> daughter buds.                                                         Considered together, enzymatic cell biological anal- <br />    To evaluate morphology arrested esa1 mutant cells         yses esa1 mutants reveal important consequences for <br /> with greater resolution, samples temperature shift ex-       loss ESA1 function. We especially interested the <br />  VOL. 19, 1999        ESSENTIAL HAT AND CELL CYCLE CONTROL                                 2521 <br />  <br />  <br />                 defect cell cycle progression previous research dem- <br />   ESA1          onstrated acetylatable amino-terminal H3 H4 tails <br />                 essential cell cycle control. In particular, cells are <br />                 depleted normal H3 H4 histones left with <br />                 H3 H4 lacking amino-terminal tails (36), rapidly <br />                 assume arrest comparable observed esa1 <br />                 cells. In considering mechanisms leading arrest, it <br />                 relevant consider role checkpoint functions (re- <br />                 viewed references 16 68). <br />                    The esa1 cell cycle arrest dependent RAD9 check- <br />                 point gene. Cell cycle checkpoint functions ordinarily de- <br />                 fined functions perceive damaged struc- <br />                 tures failure complete processes necessary normal <br />                 fidelity cell duplication (16, 68). Frequently, cell cycle arrests <br />                 triggered checkpoint functions allowing time repair <br />                 completion lagging step. We wanted determine <br />                 cell cycle arrest observed esa1 mutants was <br />   esa1-L254P    dependent checkpoint functions. It possible that, <br />                 depending precise role(s) Esa1p HAT activity, <br />                 cells trigger checkpoints registering failure repli- <br />                 cate chromatin failure cell cycle transcriptional control. It <br />                 noted DNA checkpoints defined date, as <br />                 controlled RAD9 gene (39, 67), appear register <br />                 DNA damage, necessarily chromatin damage. Further, the <br />                 esa1 cell cycle arrest observed distinct ordinarily <br />                 seen checkpoints substantial loss cell <br />                 viability esa1 mutants, classically defined <br />                 checkpoint arrests, viability remains high return per- <br />                 missive conditions. <br />                    To assess role checkpoint control esa1 cell cycle <br />                 arrest, constructed isogenic rad9D esa1 double- single- <br />                 mutant strains temperature shift experiments described <br />                 above. At 4 h shift restrictive temperature, the <br />                 rad9D esa1 cells severe losses viability comparable the <br />   esa1-L327S    esa1 single mutants, rad9D cells remained fully <br />                 viable (data shown). Flow cytometry revealed the <br />                 rad9D single-mutant strain unaffected growth 37&#194;&#176;C <br />                 (Fig. 9A), esa1 strain arrested 2C DNA con- <br />                 tent (Fig. 9B). The rad9D esa1 double-mutant strain, however, <br />                 longer arrested simple 2C DNA content (Fig. 9C). <br />                 Instead, 1C 2C peaks apparent, indicative of <br />                 cells G1 G2/M phases cell cycle. Budding <br />                 indices confirmed extended flow cytometry data. <br />                 Rather arresting .80% large-budded abnormal <br />                 cells, rad9D esa1 strain 46% unbudded, 25% small <br />                 budded, 26% large budded. In population, 64% the <br />                 cells clearly abnormal, appearing exaggerated in <br />                 size. <br />                    To determine cell cycle arrest dependent the <br />                 mitotic checkpoint, constructed esa1 strain deleted <br />   esa1-414      MAD3 gene (34, 70). No change observed cell <br />                 cycle arrest restrictive temperature assayed flow cy- <br />                 tometry compared esa1 mutant (Fig. 9D E). <br />                 Therefore, cell cycle arrest observed esa1 mutant <br />                 strain dependent checkpoint function RAD9 but <br />                 MAD3 mitotic checkpoint. <br />  <br />  <br />  <br />                    FIG. 6. esa1 mutants display improper DNA segregation. ESA1 (LPY3498), <br />                 esa1-L254P (LPY3500), esa1-414 (LPY3291) cells incubated 37&#194;&#176;C <br />                 4 h, esa1-L327S (LPY3430) cells incubated 37&#194;&#176;C 8 h, fixed, <br />                 stained DAPI. esa1 mutant cells predominantly large budded as <br />                 visualized Nomarski optics (left center columns, bottom), some <br />                 triple-budded cells (right column, bottom). DAPI staining indicates esa1 <br />                 mutant cells failed segregate abnormally segregated chromatin <br />                 comparison wild-type cells. Bar, 5 mm. <br />  2522     CLARKE ET AL.                                                                                                            MOL. CELL. BIOL. <br />  <br />  <br />                           DISCUSSION <br />  <br />    Histone acetylation posttranslational modification that <br /> historically correlated assembly distinct <br /> transcriptional states chromatin (8, 21). Here report <br /> identification yeast ESA1 gene essential cell <br /> viability encodes protein HAT activity. The enzy- <br /> matic activity Esa1p unique pattern substrate spec- <br /> ificity yeast HATs, intriguingly, essential role <br /> appears tied progression cell cycle DNA <br /> replication complete. ESA1 family highly <br /> conserved genes identified transcrip- <br /> tional regulatory functions yeast, Drosophila, humans (6, <br /> 15, 25, 26, 53). It likely Esa1p&#226;&#8364;&#8482;s essential <br /> functions transcriptional catalytic component <br /> of NuA4, large, multisubunit complex activate tran- <br /> scription (14, 61). These results connect specific <br /> HAT cell cycle progression raise possibility that <br /> targets HAT activity feed checkpoint controls con- <br /> tribute regulation modulation downstream <br /> transcriptional events. <br />    Esa1p acetylates histones distinct pattern sub- <br /> strate specificity. Both recombinant Esa1p native protein <br /> purified NuA4 complex (14) significant HAT ac- <br /> tivity directed histone H4. With synthetic peptides or <br /> purified histones substrates, Esa1p acetylate the <br /> four lysines N-terminal tail significant for <br /> H4 function (this report references 14 57). The pro- <br /> nounced preference Lys5 H4 consistent role <br /> in deposition newly synthesized histones (8), although <br /> the comparable site acetylated transcriptionally active <br /> macronuclei rapidly growing Tetrahymena (12). Esa1p&#226;&#8364;&#8482;s pat- <br /> tern acetylation observed H3 seen HAT ac- <br /> tivity Gcn5p TAFII250 (32, 41), transcriptional <br /> activators. Although H2A&#226;&#8364;&#8482;s acetylation thorough- <br /> ly studied, acetylated vivo animal cells (46) the <br /> same position observe Esa1p&#226;&#8364;&#8482;s activity vitro. These <br /> same specificities observed independently and <br /> recently reported, including using purified Drosophila H4 and <br /> chicken H2A histones substrates (57). Further, catalyti- <br /> cally active recombinant fragment human protein Tip60 <br /> acetylates H4, H3, H2A free histones (72), revealing that <br /> not SAS family proteins highly related,               FIG. 7. esa1 mutants disrupt mitotic cell cycle progression. ESA1 (LPY3498) <br />                                                                         esa1-L254P (LPY3500) cells incubated 37&#194;&#176;C 4 h prepared for <br /> specificities activities well. Although vitro     indirect immunofluorescence microscopy. esa1 mutant cells exhibited aberrant <br /> specificities HATs interpreted cautiously (see,            nuclear short spindle morphologies restrictive temperature. As visu- <br /> e.g., references 48 75), data        alized double-<span id='am-2' about='rdfs:label' typeof='owl:Thing'>label</span> immunofluorescence microscopy, antisera directed against <br /> (57, 72) identify Esa1p candidate H4 HAT, distin-               Kar2p detected nuclear envelope ER a-tubulin detected microtu- <br />                                                                         bules. Both nuclear envelope microtubules extended partially <br /> guished comparatively targets, Lys5 H4 ap-           bud neck comparison wild type coincided the <br /> pears relevant vivo (Fig. 4). Because Lys5-acetylated iso-       DAPI-stained region. Bars represent 5 mm. <br /> form H4 depleted gcn5D mutants (75), likely <br /> must sites substrates that, concert, con- <br /> tribute essential nature ESA1&#226;&#8364;&#8482;s function. Like gcn5           4-h shift restrictive temperature. It possible that <br /> strains display impaired HAT activity vivo res-           H2A H4 relevant substrates vivo Esa1p that <br /> idues A box outside region mu-            H3 acetylation results relaxed stringency vitro with <br /> tated (33, 65), recovered esa1 mutants lesions           recombinant protein. Reduced site specificity observed <br /> the A box (esa1-L327S) outside (esa1-L254P            recombinant Hat1p absence binding partner, <br /> esa1-414) exhibit decreased H4 Lys5 acetylation vivo.           Rba48-like protein Hat2, appears function a <br /> Interestingly, esa1-414 mutant shows strongest decrease,        specificity factor (48). In contrast, broader activity vivo than <br /> suggesting role carboxy terminus maintaining             originally predicted recently suggested Gcn5p <br /> functional complex (14).                                                (75). Thus, Esa1p acetylate H2A H4 context of <br />    The Esa1p-containing NuA4 complex, identified acet-          NuA4 complex act H3 catalytic subunit <br /> ylation nucleosomal histones H2A H4, physically          different complex, influenced different specificity fac- <br /> and genetically distinct large HAT complexes             tors. Determining Esa1p catalytically active more <br /> such Ada SAGA (14, 19) appear active to-               complex distinct substrate speci- <br /> ward H3. The NuA4 complex esa1-414 strains retains HAT               ficities (including nonhistone proteins) depending cellu- <br /> activity permissive temperature, activity lost      lar localization targeted genomic loci critical to <br />  VOL. 19, 1999                                                                                 ESSENTIAL HAT AND CELL CYCLE CONTROL                                   2523 <br />  <br />  <br />  <br />  <br />    FIG. 8. esa1 mutants aberrant ultrastructural morphology. Wild-type ESA1 (LPY3498) (A B) esa1-L254P (LPY3500) (C D) cells incubated <br /> at 37&#194;&#176;C 4 h prepared rapid freezing electron microscopy. In wild-type cells, nucleolar electron-dense material poles dividing nucleus <br /> (n) (A) crescent shape nondividing cell (B, closed arrowhead). However, esa1 mutant (C D, open arrowheads), electron-dense material was <br /> dispersed nucleus. The majority mutant large-budded cells showed nucleus bud neck, contrast wild-type cells, where <br /> the nucleus extended equally bud neck. The cytoplasm esa1 mutant (C, asterisk) highly vesiculated wild-type cells. Each panel <br /> shows different cell, scale bars shown represent 1 mm (A C) 0.5 mm (B D). v, vacuoles. <br />  <br />  <br />  <br /> understanding function. Previously proposed models pre-                             acetylation patterns important chromatin structure. It is <br /> dict HAT complexes regulated subunit                                possible Esa1p modify key nonhistone <br /> exchange (55). Indeed, early studies, single Tetrahymena                           target(s). Alternatively, Esa1p&#226;&#8364;&#8482;s HAT activity function to <br /> HAT activity shown deposition- tran-                               regulate transcription gene group genes, are <br /> scription-related activities (11). In light current molec-                       required proper cell cycle control, although <br /> ular identification HATs, revealing know                          ESA1 itself transcriptionally regulated cell <br /> detailed molecular composition potentially bifunctional                         cycle (13). The closely related genes SAS2 SAS3 in- <br /> activity.                                                                               fluence transcriptional regulation silent mating-type loci <br />    Connecting HATs checkpoint control ESA1&#226;&#8364;&#8482;s                                telomeric reporter genes (15, 53). Preliminary studies in- <br /> role cell cycle progression. The cell cycle arrest temper-                        dicate ESA1 does contribute regulation these <br /> ature-sensitive esa1 mutant cells rapid accompanied                        loci, furthering likelihood functions distinct. <br /> loss viability. These effects suggest appropriate histone                          Insight nature esa1 cell cycle arrest comes <br /> acetylation critical cell cycle progression. Interestingly,                      flow cytometric microscopic analyses. These studies <br /> recent results indicate gcn5 mutants, ordinarily                        reveal arrest occurs cells completed <br /> only modest growth phenotypes, die combination sub-                             DNA synthesis successful completion mitosis. <br /> sets H3 H4 mutants suffer G2/M effects (75).                            Compared overall bud cell size, migration the <br /> Previous studies revealed mutations H4 N-terminal                               nucleus extension mitotic spindle appear abnormal. <br /> lysines lead G2/M delays RAD9 dependent (39).                           Ultrastructural analysis suggests irregulari- <br /> Similarly, deletion N-terminal tails H3 H4                         ties electron-dense chromatin itself. The significance of <br /> delays cell cycle, deletion kills cells G2/M                         observation understood. It remains possible that <br /> (36). Our results provide explanation observations                         Esa1p functions multiple capacities, directly related to <br /> by demonstrating modification N-terminal histone tails                          chromatin structure assembly mitosis tran- <br /> may essential couple acetylation events cell cycle                        scriptionally focused. <br /> control, independent structural changes resulting mu-                              The observation esa1 temperature-sensitive cell cy- <br /> tation histones. Because N-terminal tail H3                        cle arrest dependent RAD9 checkpoint function but <br /> or H4 support viability (36), possible Esa1p                       MAD3 checkpoint function deepens potential <br /> may acetylate histones vivo particular                           significance understanding ESA1&#226;&#8364;&#8482;s role cell cycle. <br />  2524     CLARKE ET AL.                                                                                                             MOL. CELL. BIOL. <br />  <br />  <br /> RAD9 participates sensing DNA damage execution <br /> of functions necessary repair. Our results raise possi-                          A rad9&#226;&#710;&#8224; <br /> bility incompletely inappropriately acetylated chroma- <br /> tin recognized damage, precise mech- <br /> anisms monitoring damage remain defined. It is <br /> formally possible end stages DNA replication <br /> are completed esa1 mutants, degrees detect- <br /> able limits flow cytometric resolution, prefer <br /> the interpretation arrest occurs S phase because <br /> of nuclear positioning mitotic spindle phenotypes. The <br /> high loss viability observed arrest distinguishes from <br /> normal checkpoint-regulated processes. Perhaps damage                                B esa1 <br /> in esa1 mutant cells severe irreparable. Al- <br /> ternatively, Esa1p itself contribute checkpoint functions <br /> akin essential RAD53 gene (2, 68), potentially activat- <br /> ing transcriptional targets required cell cycle transition <br /> points, regulating synthesis factors required for <br /> return normal cell cycle progression. <br />    Essential roles essential HATs. GCN5 HAT1 were <br /> the yeast genes determined encode transcription- dep- <br /> osition-related HAT activities (9, 29, 48). Surprisingly, strains <br /> in genes deleted modest growth phe-                            C rad9&#226;&#710;&#8224; esa1 <br /> notypes special conditions detectable cellular phe- <br /> notypes. The TFIID component TAFII250, essential factor <br /> of basal transcriptional machinery (50, 52), intrinsic <br /> HAT activity (41), essential func- <br /> tion known. It interesting mutants temper- <br /> ature-sensitive alleles yeast TAFII250 gene homologue, <br /> TAF145 arrest cell cycle restrictive tempera- <br /> ture (63). This arrest appears to loss transcrip- <br /> tional activation G1 cyclins loss global RNA poly- <br /> merase II-directed transcription (64),                            D mad3&#226;&#710;&#8224; esa1 <br /> mutant alleles specifically defective HAT activity re- <br /> mains determined. These results, considered together <br /> with esa1 cell cycle arrest, raise possibility that <br /> gene-specific global chromatin modification provides ad- <br /> ditional control transcriptional pathways ac- <br /> tivated DNA damage cell cycle perturbations. <br />    Essential roles HATs revealed male- <br /> specific lethality resulting mutation closely related <br /> Drosophila mof gene. In case, increased transcription <br /> of X chromosome correlated H4 Lys16 acety-                              E mad3&#226;&#710;&#8224; <br /> lation fails, death results dosage compensation is <br /> not achieved (25). Female flies mof mutations appear <br /> completely normal (25), suggesting mof encode a <br /> sex-specific HAT sex-specific requirements <br /> for histone acetylation. <br />    Whether vertebrate MYST family genes essential is <br /> not known. Among human homologues (6), may <br /> be genes encoding essential nonessential functions, as <br /> do yeast genes. By comparison, recently re-                                           |       | <br /> ported murine p300 CBP genes essential,                                        1C 2C <br /> and loss gene results multiple developmental de-              FIG. 9. The esa1 mutant G2/M arrest dependent RAD9 checkpoint <br /> fects early embryonic lethality (73). Although           gene. (A) rad9D (LPY3784), (B) esa1-L254P (LPY3785), (C) rad9D esa1-L254P <br />                                                                         (LPY3780), (D) mad3D esa1-L254P (LPY4223), (E) mad3 (LPY4222) mu- <br /> been possible separate p300&#226;&#8364;&#8482;s transcriptional activation             tant cells incubated 37&#194;&#176;C analyzed DNA content flow <br /> properties HAT activity (38), CBP p300&#226;&#8364;&#8482;s essential         cytometry. The x y axes represent relative fluorescence intensity number <br /> roles likely transcriptional ones, mechanistically de-        cells, respectively. The esa1-L254P mad3D esa1-L254P mutants arrested <br /> pendent HAT activity. CBP known interact p53           DNA replication approximately 80% cells containing 2C DNA <br />                                                                         content, rad9D mad3 cells normal DNA content divid- <br /> potentiate transcriptional activation functions p53 in-       ing cells. The rad9D esa1-L254P double mutant 1C 2C peaks, indi- <br /> hibiting entry G1 (22, 35). Furthermore, p53 acti-         cating esa1-L254P cell cycle arrest dependent RAD9 MAD3. <br /> vated response DNA damage (for review reference <br /> 30). Thus, regulated transcriptional activities HAT com- <br /> plexes participate levels coordinating cell           cilitate fine-tuning transcription profound <br /> cycle regulation responses DNA chromatin damage.              changes chromatin structure. By analogy Gcn5p (20), Esa1p <br />    In considering contribution chromatin modification         participate catalytically complex. Wheth- <br /> biological regulation, clear histone acetylation fa-     er catalytic activity strictly transcription related (14, 61), <br />  VOL. 19, 1999                                                                              ESSENTIAL HAT AND CELL CYCLE CONTROL                                       2525 <br />  <br />  <br /> or roles, chromatin assembly,                           Wodicka, T. G. Wolfsberg, A. E. Gabrielian, D. Landsman, D. J. Lockhart, <br /> be important issue address. The fact closely                         R. W. Davis. 1998. A genome-wide transcriptional analysis mitotic <br />                                                                                           cell cycle. Mol. Cell 2:65&#226;&#8364;&#8220;73. <br /> related genes SAS2, SAS3, ESA1 appear distinct,                           14. Co&#203;&#8224;te&#194;&#180;, J., R. T. Utley, S. Allard, A. Clarke, P. Grant, J. Savard, L. Pillus, and <br /> nonredundant functions unicellular yeast implies                              J. L. Workman. The NuA4 transcription activation/histone H4 acetyltrans- <br /> homologous genes vertebrates distinct func-                              ferase complex contains essential Esa1 protein catalytic subunit <br /> tions. For example, known loss misregulation                          essential ATM-related cofactor Tra1p. Submitted publication. <br />                                                                                       15. Ehrenhofer-Murray, A., D. Rivier, J. Rine. 1997. The role Sas2, an <br /> Tip60 contribute human immunodeficiency virus type 1                               acetyltransferase homolog, silencing ORC function Saccharomyces <br /> Tat transcription (26) MOZ, fused CBP,                                 cerevisiae. Genetics 145:923&#226;&#8364;&#8220;934. <br /> lead acute myeloid leukemia (6). Understanding specific                        16. Elledge, S. J. 1996. Cell cycle checkpoints: preventing identity crisis. <br /> functions proteins, promote changes                              Science 274:1664&#226;&#8364;&#8220;1672. <br />                                                                                       17. Georgakopoulos, T., G. Thieros. 1992. Two distinct yeast transcriptional <br /> chromatin structure transcriptional control, ultimately                          activators require function GCN5 protein promote normal <br /> depend identifying vivo targets complexes                             levels transcription. EMBO J. 11:4145&#226;&#8364;&#8220;4152. <br /> in act.                                                                    18. Ghislain, M., A. Udvardy, C. Mann. 1993. S. cerevisiae 26S protease <br />                                                                                           mutants arrest cell division G2/metaphase. Nature 366:358&#226;&#8364;&#8220;362. <br />                                                                                       19. Grant, P. A., L. Duggan, J. Co    &#203;&#8224;te&#194;&#180;, S. M. Roberts, J. E. Brownell, R. Candau, <br />                           ACKNOWLEDGMENTS                                                 R. Ohba, T. Owen-Hughes, C. D. Allis, F. Winston, S. L. Berger, J. L. <br />                                                                                           Workman. 1997. Yeast Gcn5 functions multisubunit complexes to <br />    A.S.C. J.E.L. contributed equally work.                                    acetylate nucleosomal histones: characterization Ada complex the <br />    We thank colleagues M. Parthun, D. Gottschling, M. Jones,                          SAGA (Spt/Ada) complex. Genes Dev. 11:1640&#226;&#8364;&#8220;1650. <br /> J. Pilon, P. Laybourn, J. Rine, M. Rose, S. Roth, F. Solomon, R. Stern-               20. Grant, P. A., D. E. Sterner, L. J. Duggan, J. L. Workman, S. L. Berger. <br /> glanz, T. Weinert providing reagents advice D. Allis                      1998. The SAGA unfolds: convergence transcription regulators chro- <br /> and J. Lucchesi communicating results prior publication. D.                        matin-modifying complexes. Trends Cell Biol. 8:193&#226;&#8364;&#8220;197. <br /> Wuttke W. Bishop provided invaluable advice assistance                   21. Grunstein, M. 1997. Histone acetylation chromatin structure tran- <br /> peptide synthesis. T. Giddings electron microscopic analysis                     scription. Nature 389:349&#226;&#8364;&#8220;352. <br /> possible. We thank R. Cook colleagues Baylor Protein                   22. Gu, W., X.-L. Shi, R. G. Roeder. 1997. Synergistic activation tran- <br />                                                                                           scription CBP p53. Nature 387:819&#226;&#8364;&#8220;823. <br /> Chemistry Core Facility excellent service. We thank K. Gris-                23. Guthrie, C., G. R. Fink (ed.). 1991. Guide yeast genetics molec- <br /> chuk, S. Mohr, M. Winey assistance microscopy flow                       ular biology, vol. 194. Academic Press, Inc., San Diego, Calif. <br /> cytometry Y. Han assistance sequencing esa1 alleles.                 24. Hendzel, M. J., Y. Wei, M. A. Mancini, A. VanHooser, T. Ranalli, B. R. <br /> We appreciate comments manuscript L. Freeman-Cook,                              Brinkley, D. O. Bazett-Jones, C. D. Allis. 1997. Mitosis-specific phos- <br /> J. Heilig, M. Klymkowsky, L. Leinwand, J. Sherman, E. Stone, M.                       phorylation histone H3 initiates primarily pericentromeric hetero- <br /> Winey.                                                                                    chromatin G2 spreads ordered fashion coincident with <br />    This work initiated support National Science                         mitotic chromosome condensation. Chromosoma 106:348&#226;&#8364;&#8220;360. <br /> Foundation University Colorado Graduate School Council                     25. Hilfiker, A., D. Hilfiker-Kleiner, A. Pannuti, J. C. Lucchesi. 1997. mof, <br />                                                                                           putative acetyltransferase gene related Tip60 MOZ human <br /> for Research Creative Work continued research <br />                                                                                           genes SAS genes yeast, required dosage compensation in <br /> and training grant funding National Institutes Health.                        Drosophila. EMBO J. 16:2054&#226;&#8364;&#8220;2060. <br />                                                                                       25a.Jones, M. Personal communication. <br />                                  REFERENCES                                           26. Kamine, J., B. Elangovan, T. Subramanian, D. Coleman, G. Chinnadu- <br />  1. Aasland, R., A. F. Stewart. 1995. The chromo shadow domain, second              rai. 1996. Identification cellular protein specifically interacts with <br />     chromo domain heterochromatin-binding protein, HP1. Nucleic Acids                  essential cysteine region HIV-1 Tat transactivator. Virology 216: <br />     Res. 23:3168&#226;&#8364;&#8220;3173.                                                                    357&#226;&#8364;&#8220;366. <br />  2. Allen, J. B., Z. Zhou, W. Siede, E. C. Freiberg, S. J. Elledge. 1994. The     27. Kilmartin, J. V., A. E. M. Adams. 1984. Structural rearrangements of <br />     SAD1/RAD53 protein kinase controls multiple checkpoints DNA dam-                  tubulin actin cell cycle yeast Saccharomyces. J. Cell <br />     age-induced transcription yeast. Genes Dev. 8:652&#226;&#8364;&#8220;665.                             Biol. 98:922&#226;&#8364;&#8220;933. <br />  3. Altschul, S. F., W. Gish, W. Miller, E. W. Myers, D. J. Lipman. 1990.         28. Kilmartin, J. V., B. Wright, C. Milstein. 1982. Rat monoclonal antitu- <br />     Basic local alignment search tool. J. Mol. Biol. 215:403&#226;&#8364;&#8220;410.                         bulin antibodies derived using new non-secreting cell line. J. Cell Biol. <br />  4. Bannister, A. J., T. Kouzarides. 1996. The CBP co-activator histone          93:576&#226;&#8364;&#8220;582. <br />     acetyltransferase. Nature 384:641&#226;&#8364;&#8220;643.                                            29. Kleff, S., E. D. Andrulis, C. W. Anderson, R. Sternglanz. 1995. Identi- <br />  5. Baudin, A., O. Ozier-Kalogeropoulos, A. Denouel, F. LaCroute, C.                  fication gene encoding yeast histone H4 acetyltransferase. J. Biol. <br />     Cullen. 1993. A simple efficient method direct gene deletion               Chem. 270:24674&#226;&#8364;&#8220;25677. <br />     Saccharomyces cerevisiae. Nucleic Acids Res. 14:3329&#226;&#8364;&#8220;3330.                        30. Ko, L. J., C. Prives. 1996. p53: puzzle paradigm. Genes Dev. 10: <br />  6. Borrow, J., V. P. Stanton, J. M. Andresen, R. Becher, F. G. Behm, R. S. K.            1054&#226;&#8364;&#8220;1072. <br />     Chaganti, C. I. Civin, C. Disteche, I. Dube, A. M. Frischauf, D. Horsman, F.      31. Koonin, E. V., S. Zhou, J. C. Lucchesi. 1995. The chromo superfamily: <br />     Mitelman, A. E. Watmore, D. E. Housman. 1996. The t(8;16)(p11;p13)                new members, duplication chromo domain possible role deliv- <br />     translocation acute myeloid leukemia fuses putative acetyltransferase         ering transcription regulators chromatin. Nucleic Acids Res. 23:4229&#226;&#8364;&#8220; <br />     CREB-binding protein. Nat. Genet. 14:33&#226;&#8364;&#8220;41.                                       4233. <br />  7. Brownell, J. E., C. D. Allis. 1995. An activity gel assay detects single,   32. Kuo, M.-H., J. E. Brownell, R. E. Sobel, T. A. Ranalli, R. G. Cook, D. G. <br />     catalytically active histone acetyltransferase subunit Tetrahymena macro-          Edmondson, S. Y. Roth, C. D. Allis. 1996. Transcription-linked acety- <br />     nuclei. Proc. Natl. Acad. Sci. USA 92:6364&#226;&#8364;&#8220;6368.                                      lation Gcn5p histones H3 H4 specific lysines. Nature 383:269&#226;&#8364;&#8220; <br />  8. Brownell, J. E., C. D. Allis. 1996. Special HATs special occasions:           272. <br />     linking histone acetylation chromatin assembly gene activation. Curr.      33. Kuo, M.-H., J. Zhou, P. Jambeck, M. E. A. Churchill, C. D. Allis. 1998. <br />     Opin. Genet. Dev. 6:176&#226;&#8364;&#8220;184.                                                          Histone acetyltransferase activity Gcn5p required activation of <br />  9. Brownell, J. E., J. Zhou, T. Ranalli, R. Kobayashi, D. G. Edmondson, S. Y.            targeted genes vivo. Genes Dev. 12:627&#226;&#8364;&#8220;639. <br />     Roth, C. D. Allis. 1996. Tetrahymena histone acetyltransferase A:           34. Li, R., A. W. Murray. 1991. Feedback control mitosis budding yeast. <br />     transcriptional co-activator linking gene expression histone acetylation.          Cell 66:519&#226;&#8364;&#8220;31. (Erratum, 79: following 388, 1994.) <br />     Cell 84:843&#226;&#8364;&#8220;851.                                                                  35. Lill, N. L., S. R. Grossman, D. Ginsberg, J. DeCaprio, D. M. Livingston. <br /> 10. Chen, H., R. J. Lin, R. L. Schiltz, D. Chakravarti, A. Nash, L. Nagy, M. L.           1997. Binding modulation p53 p300/CBP coactivators. Nature 387: <br />     Privalsky, Y. Nakatani, R. M. Evans. 1997. Nuclear receptor coactivator           823&#226;&#8364;&#8220;827. <br />     ACTR novel histone acetyltransferase forms multimeric activation       36. Ling, X., T. A. A. Harkness, M. G. Schultz, G. Fisher-Adams, M. <br />     complex P/CAF CBP/p300. Cell 90:569&#226;&#8364;&#8220;580.                                     Grunstein. 1996. Yeast histone H3 H4 amino termini important for <br /> 11. Chicoine, L. G., R. Richman, R. G. Cook, M. A. Gorovsky, C. D. Allis.             nucleosome assembly vivo vitro: redundant position-indepen- <br />     1987. A single histone acetyltransferase Tetrahymena macronuclei cat-            dent functions assembly gene regulation. Genes Dev. 10:686&#226;&#8364;&#8220; <br />     alyzes deposition-related acetylation free histones transcription-re-          699. <br />     lated acetylation nucleosomal histones. J. Cell Biol. 105:127&#226;&#8364;&#8220;135.             37. Luger, K., A. W. Ma   &#194;&#168;der, R. K. Richmond, D. F. Sargent, T. J. Richmond. <br /> 12. Chicoine, L. G., I. G. Schulman, R. Richman, R. G. Cook, C. D. Allis.             1997. Crystal structure nucleosome core particle 2.8 &#195;&#8230; resolution. <br />     1986. Nonrandom utilization acetylation sites histones isolated            Nature 389:251&#226;&#8364;&#8220;260. <br />     Tetrahymena. J. Biol. Chem. 261:1071&#226;&#8364;&#8220;1076.                                        38. Martinez-Balba    &#194;&#180;s, M. A. A. J. Bannister, K. Martin, P. Haus-Seuffert, M. <br /> 13. Cho, R. J., M. J. Campbell, E. A. Winzler, L. Steinmetz, A. Conway, L.                Meisterernst, T. Kouzarides. 1998. The acetyltransferase activity CBP <br />  2526       CLARKE ET AL.                                                                                                                           MOL. CELL. BIOL. <br />  <br />  <br />     stimulates transcription. EMBO J. 17:2886&#226;&#8364;&#8220;2893.                                       essential growth yeast. Proc. Natl. Acad. Sci. USA 95:3561&#226;&#8364;&#8220;3565. <br /> 39. Megee, P. C., B. A. Morgan, M. M. Smith. 1995. Histone H4             58. Sobel, R. E., R. G. Cook, C. D. Allis. 1994. Non-random acetylation of <br />     maintenance genome integrity. Genes Dev. 9:1716&#226;&#8364;&#8220;1727.                              histone H4 cytoplasmic histone acetyltransferase determined novel <br /> 40. Mizzen, C. A., C. D. Allis. 1998. Linking histone acetylation tran-            methodology. J. Biol. Chem. 269:18576&#226;&#8364;&#8220;18582. <br />     scriptional regulation. Cell Mol. Life Sci. 54:6&#226;&#8364;&#8220;20.                              59. Spencer, T. E., G. Jenster, M. M. Burcin, C. D. Allis, J. Zhou, C. A. Mizzen, <br /> 41. Mizzen, C. A., X.-J. Yang, T. Kokubo, J. E. Brownell, A. J. Bannister, T.             N. J. McKenna, S. A. Onate, S. Y. Tsai, M. Tsai, B. W. O&#226;&#8364;&#8482;Malley. 1997. <br />     Owen-Hughes, J. Workman, L. Wang, S. L. Berger, T. Kouzarides, Y. Na-                 Steroid receptor coactivator-1 histone acetyltransferase. Nature 389:194&#226;&#8364;&#8220; <br />     katani, C. D. Allis. 1996. The TAFII250 subunit TFIID histone              198. <br />     acetyltransferase activity. Cell 87:1261&#226;&#8364;&#8220;1270.                                    60. Turner, B. M. 1998. Histone acetylation epigenetic determinant of <br /> 42. Neuwald, A. F., D. Landsman. 1997. GCN5-related histone N-acetyl-                 long-term transcriptional regulation. Cell Mol. Life Sci. 54:21&#226;&#8364;&#8220;31. <br />     transferases belong diverse superfamily includes yeast SPT10        61. Utley, R. T., K. Ikeda, P. A. Grant, J. Co&#203;&#8224;te&#194;&#180;, D. L. Steger, A. Eberharter, S. <br />     protein. Trends Biochem. Sci. 22:154&#226;&#8364;&#8220;155.                                             John, J. L. Workman. 1998. Transcriptional activators direct histone <br /> 43. Ogryzko, V. O., R. L. Schiltz, V. Russanova, B. H. Howard, Y. Nakatani.           acetyltransferase complexes nucleosomes. Nature 394:498&#226;&#8364;&#8220;502. <br />     1996. The transcriptional coactivators p300 CBP histone acetyltrans-      62. Verrault, A., P. D. Kaufman, R. Kobayashi, B. Stillman. 1997. Nucleo- <br />     ferases. Cell 87:953&#226;&#8364;&#8220;959.                                                             somal DNA regulates core-histone-binding subunit human Hat1 <br /> 44. O&#226;&#8364;&#8482;Toole, E. T., D. N. Mastronarde, T. H. Giddings, M. Winey, D. J. Burke,             acetyltransferase. Curr. Biol. 8:96&#226;&#8364;&#8220;108. <br />     J. R. McIntosh. 1997. Three-dimensional analysis ultrastructural          63. Walker, S. S., J. C. Reese, L. M. Apone, M. R. Green. 1996. Transcrip- <br />     design mitotic spindles cdc20 mutant Saccharomyces cerevisiae.         tion activation cells lacking TAFIIs. Nature 383:185&#226;&#8364;&#8220;188. <br />     Mol. Biol. Cell 8:1&#226;&#8364;&#8220;11.                                                           64. Walker, S. S., W. C. Shen, J. C. Reese, L. M. Apone, M. R. Green. 1997. <br /> 45. Palmer, R. E., M. Koval, D. Koshland. 1989. The dynamics chromo-               Yeast TAF(II)145 required transcription G1/S cyclin genes reg- <br />     movement budding yeast Saccharomyces cerevisiae. J. Cell Biol.            ulated cellular growth state. Cell 90:607&#226;&#8364;&#8220;614. <br />     109:3355&#226;&#8364;&#8220;3366.                                                                    65. Wang, L., L. Liu, S. L. Berger. 1998. Critical residues histone <br /> 46. Pantazis, P., W. M. Bonner. 1981. Quantitative determination histone           acetylation Gcn5, functioning Ada SAGA complexes, also <br />     modification. H2A acetylation phosphorylation. J. Biol. Chem. 256:                required transcriptional function vivo. Genes Dev. 12:640&#226;&#8364;&#8220;653. <br />     4669&#226;&#8364;&#8220;4675.                                                                        66. Weiler, K. S., B. T. Wakimoto. 1995. Heterochromatin gene expres- <br /> 47. Paro, R. 1990. Imprinting determined state chromatin Dro-               sion Drosophila. Annu. Rev. Genet. 29:577&#226;&#8364;&#8220;605. <br />     sophila. Trends Genet. 6:416&#226;&#8364;&#8220;421.                                                 67. Weinert, T. A., L. H. Hartwell. 1988. The RAD9 gene controls cell <br /> 48. Parthun, M. R., J. Widom, D. E. Gottschling. 1996. The major cytoplas-            cycle response DNA damage Saccharomyces cerevisiae. Science 241: <br />     mic histone acetyltransferase yeast: links chromatin replication            317&#226;&#8364;&#8220;322. <br />     histone metabolism. Cell 87:85&#226;&#8364;&#8220;94.                                                68. Weinert, T. A., G. L. Kiser, L. H. Hartwell. 1994. Mitotic checkpoint <br /> 49. Pazin, M. J., J. T. Kadonaga. 1997. What&#226;&#8364;&#8482;s histone               genes budding yeast dependence mitosis DNA replication <br />     deacetylation transcription? Cell 89:325&#226;&#8364;&#8220;328.                                     repair. Genes Dev. 8:652&#226;&#8364;&#8220;665. <br /> 50. Poon, D., Y. Bai, A. M. Campbell, S. Bjorklund, Y. J. Kim, S. Zhou,               69. Weiss, E., M. Winey. 1996. The Saccharomyces cerevisiae spindle pole <br />     R. D. Kornberg, P. A. Weil. 1995. Identification characterization          body duplication gene MPS1 mitotic checkpoint. J. Cell Biol. 132: <br />     TFIID-like multiprotein complex Saccharomyces cerevisiae. Proc.                111&#226;&#8364;&#8220;123. <br />     Natl. Acad. Sci. USA 92:8224&#226;&#8364;&#8220;8228.                                                70. Wells, W. A., A. W. Murray. 1996. Aberrantly segregating centromeres <br /> 51. Pringle, J. R., R. A. Preston, A. E. Adams, T. Stearns, D. G. Drubin, B. K.           activate spindle assembly checkpoint budding yeast. J. Cell Biol. 133: <br />     Haarer, E. W. Jones. 1989. Fluorescence microscopy methods yeast.             75&#226;&#8364;&#8220;84. <br />     Methods Cell Biol. 31:357&#226;&#8364;&#8220;435.                                                    70a.Winey, M., T. Giddings. Personal communication. <br /> 52. Reese, J. C., L. Apone, S. S. Walker, L. A. Griffin, M. R. Green. 1994.       71. Wolffe, A. 1992. Chromatin structure function. Academic Press, New <br />     Yeast TAFIIS multisubunit complex required activated transcrip-              York, N.Y. <br />     tion. Nature 371:523&#226;&#8364;&#8220;527.                                                         72. Yamamoto, T., M. Horikoshi. 1997. Novel substrate specificity the <br /> 53. Reifsnyder, C., J. Lowell, A. Clarke, L. Pillus. 1996. Yeast SAS silencing        histone acetyltransferase activity HIV-1-Tat interactive protein Tip60. <br />     genes human genes associated AML HIV-1 Tat interactions                  J. Biol. Chem. 272:30595&#226;&#8364;&#8220;30598. <br />     homologous acetyltransferases. Nat. Genet. 14:42&#226;&#8364;&#8220;49.                     73. Yao, T.-P., S. P. Oh, M. Fuchs, N.-D. Zhou, L. E. Ch&#226;&#8364;&#8482;ing, D. Newsome, R. T. <br /> 54. Rose, M. D., L. M. Mistra, J. P. Vogel. 1989. KAR2, karyogamy gene,             Bronson, E. Li, D. M. Livingston, R. Eckner. 1998. Gene dosage- <br />     yeast homolog mammalian BiP/GRP78 gene. Cell 57:1211&#226;&#8364;&#8220;1221.              dependent embryonic development proliferation defects mice lacking <br /> 55. Roth, S. Y., C. D. Allis. 1996. The subunit exchange model histone             transcriptional integrator p300. Cel <br /> </body></html>