<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Dec. 2001, p. 8346&#226;&#8364;&#8220;8356                                                                         Vol. 21, No. 24 <br /> 0270-7306/01/$04.00&#226;&#171;&#185;0 DOI: 10.1128/MCB.21.24.8346&#226;&#8364;&#8220;8356.2001 <br /> Copyright &#194;&#169; 2001, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />          Mpe1, Zinc Knuckle Protein, Is Essential Component of <br />           Yeast Cleavage Polyadenylation Factor Required for <br />                 Cleavage Polyadenylation mRNA <br />                                 ` LE MINET,1 JEAN-MARIE SCHMITTER,2 FRANC <br />            LE THUY ANH VO,1 MICHE                                       &#194;&#184; OIS LACROUTE,1 <br />                                                             1 <br />                                       AND FRANC &#194;&#184; OISE WYERS * <br />                Centre Ge&#194;&#180;ne&#194;&#180;tique Mole&#194;&#180;culaire, UPR A2167, CNRS, 91198 Gif sur Yvette,1 Institut Europe&#194;&#180;en de <br />                                  Chimie et Biologie, FRE 2247, CNRS, 33607 Pessac Cedex,2 France <br />                        Received 17 July 2001/Returned modification 22 August 2001/Accepted 24 September 2001 <br />  <br />              In Saccharomyces cerevisiae, vitro mRNA cleavage polyadenylation require poly(A) binding protein, <br />           Pab1p, multiprotein complexes: CFI (cleavage factor I) CPF (cleavage polyadenylation <br />           factor). We characterized novel essential gene, MPE1 (YKL059c), interacts genetically PCF11 <br />           gene encoding subunit CFI. Mpe1p evolutionarily conserved protein, homolog encoded <br />           human genome. The protein sequence contains putative RNA-binding zinc knuckle motif. MPE1 is <br />           implicated choice ACT1 mRNA polyadenylation site vivo. Extracts conditional mutant, <br />           mpe1-1, wild-type extract immunoneutralized Mpe1p defective 3&#226;&#180;&#8226;-end processing. We used <br />           tandem affinity purification (TAP) method strains TAP-tagged Mpe1p Pfs2p Mpe1p, <br />           like Pfs2p, integral subunit CPF. Nevertheless stable CPF, devoid Mpe1p, purified the <br />           mpe1-1 mutant strain, showing Mpe1p directly involved stability complex. Consistently, <br />           Mpe1p necessary processive polyadenylation, nonspecific genuine pre-mRNA 3&#226;&#180;&#8226; end, <br />           displayed CPF alone. However, reconstituted assay purified CFI, CPF, recombinant <br />           Pab1p showed Mpe1p strictly required specific cleavage polyadenylation pre-mRNA. <br />           These results Mpe1p plays crucial role 3&#226;&#180;&#8226; end formation probably promoting specific link <br />           CFI/CPF complex pre-mRNA. <br />  <br />  <br />    In eukaryotic cells, 3&#226;&#172;&#732;-end cleavage polyadenylation              cated correct choice cleavage site (25, 33). Pab1p <br /> essential steps synthesis functional mRNAs. These                  partially copurifies CFI involved regulation of <br /> processes involved transcription termination (21, 34)                 <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> poly(A) tail cleavage (3, 36). The <br /> and export mature mRNAs nucleus (23). The                    3&#226;&#172;&#732;-end processing factors initially identified extensive <br /> poly(A) tail associated poly(A)-binding protein,                    chromatographic purification steps giving different subcom- <br /> Pab1p, directly implicated regulation gene expres-              plexes, contained common subunits (40, 53). <br /> sion, stimulating translation mRNA controlling                 More recently, affinity purification used study pro- <br /> its stability deadenylation-dependent degradation (15,                teins associated component polyadenylation com- <br /> 32, 49). Prior polyadenylation, nascent transcripts               plex, protein A-tagged Pfs2p (38). This technique allowed <br /> cleaved specific site. The cleavage polyadenylation                 direct isolation CPF complex containing the <br /> processes coupled vivo, specificity reac-              previously identified proteins CFI complex pro- <br /> tion ensured recognition cis-acting sequences                teins, showing CFI separate entity. The CPF complex <br /> the transcript broad multiprotein complex. These                    contains poly(A) polymerase enzyme Cft1p/ <br /> steps studied independently cellular extracts. This led            Yhh1p, Cft2p/Ydh1p, Brr5p/Ysh1p, Pta1p, Fip1p, Yth1p, <br /> <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> purification characterization essential fac-              Pfs2p, Pfs1p subunits (6, 38, 40). Only Pfs1 protein <br /> tors (reviewed references 48 52). In yeast Saccha-                remains uncharacterized. The exact functions different <br /> romyces cerevisiae, complexes, CFI (cleavage factor I)               proteins complex unknown, probably they <br /> CPF (cleavage polyadenylation factor), plus Pab1p                    act concerted manner. All subunits are <br /> responsible specific cleavage polyadenylation                 essential, phenotypes available conditional mutants <br /> pre-mRNA. CFI purified shown composed                       that, Fip1p, essential cleav- <br /> two subcomplexes, CFIA CFIB (26). CFIA consists                       age polyadenylation (6, 38, 39, 52, 53). The protein directly <br /> Rna14p, Rna15p, Pcf11p, Clp1p (26, 36). The phenotypes                   involved endonucleolytic cleavage iden- <br /> of Rna14p, Rna15p, Pcf11p mutants confirmed they <br />                                                                              tified. The Pap1 protein known catalyze polyade- <br /> are directly involved 3&#226;&#172;&#732;-end processing (4, 35). CFIB consists <br />                                                                              nylation. However, CFI, CPF, Pab1p absent this <br /> of single protein, Hrp1p/Nab4p, specifically impli- <br />                                                                              polyadenylation occurs unregulated way (33, 38, 40). <br />                                                                                 The 3&#226;&#172;&#732;-end processing apparatus highly conserved be- <br />                                                                              tween yeast mammals, differences in <br />    * Corresponding author. Mailing address: Centre Genetique Mo- <br /> leculaire, UPR A2167, CNRS, 91198 Gif sur Yvette, France. Phone: <br />                                                                              sequences proteins organization the <br /> 33-1-69-82-31-70. Fax: 33-1-69-82-31-40. E-mail: wyers@cgm.cnrs-gif          mRNA 3&#226;&#172;&#732;-end processing signals (24, 44, 48, 52). Rna14p and <br /> .fr.                                                                         Rna15p yeast CFI high degree identity p77 <br />  <br />                                                                       8346 <br />  VOL. 21, 2001                                                                                  Mpe1 IS A NEW SUBUNIT OF YEAST CPF                            8347 <br />  <br />  <br /> and p64 mammalian cleavage stimulation factor,                               Most media genetic methods described Guthrie Fink (19). <br /> CstF. Similarly, Cft1p/Yhh1p, Cft2p/Ydh1p, Brr5p/Ysh1p,                        Yeast strains transformed lithium acetate method (16). <br />                                                                                       Sequence. The Mpe1p sequence analyzed PSI-BLAST CDD <br /> Yth1p CPF yeast homologues subunits                       programs (1). The S. cerevisiae sequence (GenBank accession no. 6322791) was <br /> of mammalian cleavage polyadenylation specificity factor                       lined CLUSTALX corresponding motifs genes Shiz- <br /> (CPSF). Recently, mammalian cleavage factor II, CFIIm,                         osaccharomyces pombe (GenBank accession no. 7490387), Arabidopsis thaliana <br /> was purified contain homologs Pcf11p                           (GenBank accession no. 10178219) Drosophila melanogaster (GenBank ac- <br />                                                                                    cession no. 4972740) cDNAs Homo sapiens (H.s.1, GenBank <br /> Clp1p yeast CFIA (12). A mammalian protein interacts <br />                                                                                    accession no. 13295901; H.s.2, GenBank accession no. 10991320; H.s.3, <br /> with CstF involved assembly                          GenBank accession no. 12002024). The putative human open reading frame <br /> stability 3&#226;&#172;&#732;-end processing complex shares similarities                     (ORF) obtained GENSCANW GeneMark programs. It consists <br /> with Pta1p yeast CPF (46). It clear                    16 exons contig NT_010604.3 (c836544 799544) chromosome 16 <br /> sequence homologies reflect functional identities. For exam-                       following positions: 1, c834386 833812 start codon 833977; <br />                                                                                    2, c828441 828342; 3, c825659 825510 (GENSCANW) c822228 822117 <br /> ple, mammalian CFIIm required cleavage                          (GeneMark); 4, c819257 819169; 5, c819051 818955; 6, c818564 818425; <br /> vitro, yeast CFI necessary cleavage                    6&#226;&#172;&#732;, c816451 816338 (GENSCANW); 7, c815367 815195; 8, c813260 to <br /> polyadenylation. Nevertheless, probably mainly pro-                      813157; 9, c813048 812711; 10, c811673 811577; 11, c811451 811371; 12, <br /> tein-protein interactions acquired different affinities                  c811262 811208; 12&#226;&#172;&#732;, c810075 810007 (GeneMark); 13, c807079 806978; <br /> during evolution, resulting different repartition                     14, c806126 804372; 15, c803995 802056, stop codon 802424 <br />                                                                                    AATAAA polyadenylation signal c802035. <br /> subcomplexes, catalytic functions                             mRNA analysis. RNase H treatment Northern blots carried as <br /> probably modified considerably.                                           previously described (2). The ACT1 mRNA contained 20 &#226;?&#174;g total RNA was <br />    An exhaustive study sequence genes                           cleaved RNase H presence oligo(dT) oligonucleotide <br /> identified domains defining yeast polyadenylation                        5&#226;&#172;&#732;-GAAGATGGAGCCAAAGC-3&#226;&#172;&#732;, complementary nucleotides 158 <br />                                                                                    174 upstream ACT1 stop codon. <br /> signal: efficiency element, positioning element, the <br />                                                                                       Recombinant proteins, antibodies, immunoblots. Purified recombinant <br /> cleavage site itself (reviewed references 48 52). A recent                  Pab1p obtained described previously (3). The C-terminal region Mpe1p <br /> statistical study 3&#226;&#172;&#732; end large number genes                         (amino acids 264 441) expressed His6 fusion peptide being <br /> suggested signals exist,                     cloned pET22b(&#226;&#171;&#185;) (Novagen). The recombinant polypeptide produced <br /> cleavage site (17, 47). The sequences efficiency                        Escherichia coli BL21(DE3) purified nickel-nitriloacetic acid agarose <br />                                                                                    (Qiagen) according manufacturer&#226;&#8364;&#8482;s instructions. Polyclonal anti-Mpe1p <br /> positioning elements degenerate redundant (47). Most                       antiserum obtained immunizing rabbits. Immunoblots processed, <br /> of genes contain potential cleavage sites used                   signals detected chemiluminescence (SuperSignal West Pico Chemi- <br /> by 3&#226;&#172;&#732;-end processing complex different efficiencies                       luminescent Substrate; Pierce) according standard procedures (20). <br /> (47). Modifications <span id='am-240' about='xsp:pattern' typeof='owl:Thing'>pattern</span> signal sequences                       In vitro 3&#226;&#180;&#8226;-end processing assays. 32P-labeled CYC1 RNA precleaved <br />                                                                                    CYC1 RNA synthesized runoff transcripts linearized plasmids as <br /> mutations proteins constituting 3&#226;&#172;&#732;-end processing <br />                                                                                    described previously (35, 40). Proteins extracted fractionated cen- <br /> complex lead identical alterations choice cleavage                    trifugation ammonium sulfate precipitation previously described (11). <br /> site mRNAs (14, 30, 31). This complexity offers possibility                      The CFI CPF complexes purified TAP method (42) starting <br /> for gene regulation providing choice 3&#226;&#172;&#732; cleavage sites.                    strains expressing different TAP-tagged proteins. The complexes were <br /> Indeed, distribution different 3&#226;&#172;&#732;-end transcripts                   eluted immunoglobulin G (IgG) agarose tobacco etch virus proteolytic <br />                                                                                    treatment, dialyzed 20 mM Tris-HCl (pH 7.9)&#226;&#8364;&#8220;0.2 mM EDTA&#226;&#8364;&#8220;50 mM <br /> genes dependent stress responses                        KCl&#226;&#8364;&#8220;0.5 mM dithiothreitol&#226;&#8364;&#8220;20% glycerol, kept &#226;&#171;&#186;80&#194;&#176;C. <br /> growth conditions (22, 45).                                                           In vitro polyadenylation assays carried presence ATP and <br />    We report characterization MPE1, essential                       magnesium acetate. Specific cleavage assayed presence CTP and <br /> yeast gene, encodes protein necessary                       EDTA inhibit formation poly(A) tails, previously described (35). <br />                                                                                       TAP. The TAP-tag contains calmodulin-binding peptide (CBP), spe- <br /> vitro 3&#226;&#172;&#732;-end processing subunit CPF complex. <br />                                                                                    cific cleavage site TEV protease, protein A 25-kDa peptide. <br />                                                                                    It allows complex associated TAP-tagged protein bind IgG <br />                        MATERIALS AND METHODS                                       agarose eluted TEV proteolytic treatment. The complex then <br />    Yeast strains generation conditional mpe1 allele alleles expressing      repurified calmodulin agarose finally eluted EGTA. After this <br /> tandem affinity purification (TAP)-tagged protein. The pcf11-2, mpe1-S, mpe1-1,    procedure, tagged protein contains CBP domain, molecular mass <br /> and mpe1-1-TAP strains derived W303-1B strain (ura3-1 trp1-1         increased 5 kDa. <br /> ade2-1 leu2-3,112 his3-11,15). The MPE1-TAP, MPE1&#226;&#338;&#172;Cterm-TAP, PFS2-TAP,                Complexes purified 1 liter yeast cells expressing TAP-tagged <br /> and PCF11-TAP strains derived BMA-64 strain (ura3-1 &#226;&#338;&#172;trp1            proteins grown optical density 600 nm 2 4 rich <br /> ade2-1 leu2-3,112 his3-11,15).                                                     medium. Complexes purified according standard TAP procedure, <br />    The MPE1 gene disrupted replacing coding region (amino acids 4       adsorption IgG agarose matrix washed twice a <br /> to 438) TRPI cassette diploid BMA64 strain according        higher stringency 400 mM NaCl washing buffer 5 min (42). Purified <br /> previously described PCR-based method (8). To generate mutants, MPE1           proteins precipitated trichloroacetic acid, separated sodium <br /> gene amplified oligonucleotides 5&#226;&#172;&#732;-CCT TAA TCC ATT GCT GGT            dodecyl sulfate (SDS)-10% polyacrylamide gel, stained Coomassie blue. <br /> GC-3&#226;&#172;&#732;, begins 226 bp upstream start codon, 5&#226;&#172;&#732;-CGA CGG CTG            Mass-spectrometric identification. Stained bands excised subjected <br /> CTA GAA TGG-3&#226;&#172;&#732;, begins 204 bp downstream stop codon. PCR              proteolytic digestion endoproteinase Lys-C (Roche Diagnostics, Meylan, <br /> was performed Taq polymerase (Appligene) described manufac-         France) described (18) A microchromatographic separation of <br /> turer deoxynucleoside triphosphate concentration (200 mM con-       proteolytic digests performed C18 ZipTips (Millipore, Bedford, Mass.). <br /> centrations 50 mM dCTP 50 mM dATP). The PCR           Two fractions eluted 3 &#226;?&#174;l 30 70% acetonitrile 0.1% triflu- <br /> products cloned directly vivo gap repair pFL36 MPE1(LEU2)          oroacetic acid, respectively. <br /> vector strain MPE1 coding region deleted          Matrix-assisted laser desorption ionization (MALDI)-mass spectrometry was <br /> contained pFL38-MPE1 (URA3). The LEU2&#226;&#171;&#185; clones transferred           performed Bruker (Bremen, Germany) Reflex III instrument 20-kV <br /> 5-fluoro-orotic acid lose URA3&#226;&#171;&#185; marked plasmid assayed temper-      acceleration voltage 24-kV reflector voltage. External calibration was <br /> ature sensitivity (10).                                                            achieved mixture peptides 942 3,495 Da. Alpha-cyano- <br />    The strains expressing TAP-tagged proteins obtained inserting   4-hydroxy-cinnamic acid used matrix, prepared saturated solution <br /> TAP-tag sequence immediately upstream stop codon corresponding       50% acetonitrile&#226;&#8364;&#8220;0.1% trifluoroacetic acid. Samples prepared the <br /> gene described Puig et al. (41).                                             dried droplet method stainless steel target 26 samples. <br />  8348      VO ET AL.                                                                                                              MOL. CELL. BIOL. <br />  <br />  <br />                                                                                 441 amino acids long, predicted molecular mass 49.5 <br />                                                                                 kDa. To determine function protein, used PCR <br />                                                                                 replace ORF TRP1 gene diploid strain, <br />                                                                                 BMA64. (8). Southern blot analysis confirmed MPE1 had <br />                                                                                 deleted TRP1 inserted locus <br />                                                                                 (data shown). After sporulation, spores with <br />                                                                                 tryptophan auxotrophic <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span>phenotype</span> viable tet- <br />                                                                                 rad. The lethal deletion perfectly complemented the <br />                                                                                 MPE1 gene centromeric vector. Thus, MPE1 encodes for <br />                                                                                 suppressor essential cell viability like the <br />                                                                                 genes involved mRNA 3&#226;&#172;&#732;-end processing. <br />                                                                                    We obtained conditional allele MPE1 PCR mutagen- <br />                                                                                 esis. After integration locus, stable thermosensitive <br />                                                                                 mpe1-1 mutant grew slowly 30&#194;&#176;C unable grow at <br />                                                                                 37&#194;&#176;C (Fig. 1). The double mutant (pcf11-2 mpe1-1) strain <br />                                                                                 showed stronger phenotype slow growth 25&#194;&#176;C each <br />                                                                                 single mutant strain unable grow 30&#194;&#176;C (Fig. 1). The <br />                                                                                 sequence Mpe1-1p revealed point mutations compared <br />                                                                                 wild-type protein (Fig. 2). A <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>nonsense mutation</span> po- <br />                                                                                 sition 354 eliminates 87 amino acids C-terminal end <br />                                                                                 Mpe1p results synthesis shortened protein <br />                                                                                 (see Fig. 6A, lane 4). Nevertheless, truncation prob- <br />                                                                                 ably does determine mutant phenotype the <br />                                                                                 introduction TAP-tag wild-type strain position <br />                                                                                 nonsense codon resulted shortened protein <br />   FIG. 1. Growth characteristics mpe1-1, pcf11-2, pcf11-2                (Fig. 6A, lane 3) detectable phenotypic change (data <br /> mpe1-1 mutant strains. Positions wild-type, mutant, sup-             shown). <br /> pressor strains indicated diagram. Cells grown              Mpe1p evolutionarily conserved protein. A search for <br /> indicated temperatures 2 days minimal medium supplemented                known protein motifs Mpe1p revealed zinc knuckle (CX2 <br /> for auxotrophies W303-1B strain. <br />                                                                                 CX4 HX4 C) amino acids 182 195. This motif has <br />                                                                                 implicated interactions proteins sin- <br />   Measured monoisotopic mass values used 0.15-Da tolerance       gle-stranded nucleic acids (9, 28). A homology search based on <br /> search NCBI database Protein Prospector MSFit search engine <br /> (http://falcon.ludwig.ucl.ac.uk) restricting species S. cerevisiae. <br />                                                                                 zinc knuckle identified genes S. pombe, A. thaliana, <br />                                                                                 D. melanogaster Mpe1p domain between <br />                                                                                 amino acids 176 216 conserved. When protein <br />                                RESULTS <br />                                                                                 sequences aligned Mpe1p sequence, more <br />    The product essential MPE1 gene interacts function-                   regions, A B, contain similarities. The se- <br /> ally Pcf11p. The yeast PCF11 gene encodes CFIA sub-                      quence comparison conserved domains shown in <br /> <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> required cleavage polyadenylation 3&#226;&#172;&#732;                   Fig. 2. The A region encompasses amino acids 5 78 of <br /> end mRNA. A collection conditional thermosensitive                        Mpe1p does contain known motifs. The B domain <br /> PCF11 mutants, including pcf11-2 strain, previously                     extends amino acids 284 343 Mpe1p; cysteine- <br /> selected PCR mutagenesis (4). The pcf11-2 strain grows                       rich presents characteristics RING finger. <br /> slowly 34&#194;&#176;C unable grow 37&#194;&#176;C (Fig. 1). Extracts                 RING fingers implicated protein-protein interac- <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> mutant deficient vitro cleavage                    tions, notably ubiquitination pathway (5). These three <br /> polyadenylation (4). To new genes interact function-                  conserved domains nonoverlapping human <br /> ally CFIA complex, selected extragenic suppres-                     cDNA&#226;&#8364;&#8482;s, belong region chromosome <br /> sors thermosensitivity pcf11-2 mutant UV                    16. Based data sequence human <br /> mutagenesis. A semidominant suppressor found,                       genome, selected set exons reconstituted most <br /> corresponding gene named MPE1 mutant PCF11 ex-                          probable ORF containing conserved motifs (see Ma- <br /> tragenic suppressor 1. The suppressor allele, mpe1-s,                    terials Methods). Like A. thaliana D. melanogaster <br /> phenotype itself (Fig. 1). It suppressed pcf11-3 mutant                  genes, human ORF contains longer C-terminal <br /> not pcf11-1 pcf11-9 mutants. It suppress                  polypeptide downstream B motif MPE1 gene. <br /> rna15-2 weakly rna14-1 rna14-5 mutants (data                    These results strongly indicate human Mpe1p homolog <br /> shown).                                                                         exists. <br />    To clone MPE1 gene, used DNA isolated                             The mpe1-1 <span id='am-162' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-163' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-165' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-170' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-172' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-181' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-182' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-187' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-190' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-195' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-198' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-219' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-223' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-228' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-231' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-232' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-238' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-239' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>mutation</span> modifies choice mRNA poly- <br /> pcf11-2 mpe1-s strain synthesize genomic library                    adenylation site vivo. To determine Mpe1p di- <br /> pFL38 centromeric vector. After transformation ther-                     rectly involved mRNA 3&#226;&#172;&#732;-end processing, studied the <br /> mosensitive pcf11-2 strain selection thermoresistant                     cleavage ACT1 mRNA mpe1-1 mutant strain vivo. <br /> clones, essential ORF, YKL059c (50), con-                     This transcript, encoding yeast actin, contains potential <br /> tained suppressor mpe1-s. The corresponding protein                     cleavage sites 3&#226;&#172;&#732; untranslated region (Fig. 3A). They are <br />  VOL. 21, 2001                                                                       Mpe1 IS A NEW SUBUNIT OF YEAST CPF                    8349 <br />  <br />  <br />  <br />  <br />   FIG. 2. The human genome encodes Mpe1p homolog. The sequence conserved motifs compared S. cerevisiae (Sc), S. <br /> pombe (Sp), A. thaliana (At), D. melanogaster (Dm) polypeptides encoded cDNAs H. sapiens (Hs1 -3). The three <br /> conserved motives designated A, zinc knuckle, B. Identical residues frames marked asterisks, similar residues are <br /> indicated gray shading. The identity scores S. cerevisiae H. sapiens 35% A, 65% zinc knuckle domain, 29% for <br /> B. The arrows positions mutations mpe1-1 mutant (amino acid 9, phenylalanine changed serine; amino acid 268, <br /> glutamine changed lysine; amino acid 337, lysine changed phenylalanine; amino acid 354, lysine changed stop codon). <br />  <br />  <br />  <br /> used variable frequencies vivo depending efficiency          mutants 3&#226;&#172;&#732;-end processing complex reflects their <br /> of cleavage polyadenylation complex. Mutations             inability synthesize mature mRNAs temperature shift <br /> subunits CFI, Rna14p Rna15p, poly(A)              (4, 37, 39). Two ACT1 mRNAs separated con- <br /> polymerase modify sites used (2, 31). We studied               ditions, result cleavage polyadenylation sites 1 5 <br /> vivo distribution ACT1 mRNA polyadenylation sites                   (2). The amounts mRNAs different strains <br /> pcf11-2 mpe1-1 mutants according temperature. The              confirm results shown Fig. 3B. Indeed, frequency of <br /> mRNAs treated RNase H hybridization               cleavage distal site 5 0.08 wild-type strain and <br /> a specific oligonucleotide designed cut transcript near        0.70 0.60, respectively, pcf11-2 mpe1-1 strains <br /> 3&#226;&#172;&#732; end oligo(dT) degrade heterogeneous poly(A)                 180 min 37&#194;&#176;C. Moreover, seen the <br /> tails. After separation polyacrylamide gel, ACT1                 suppression pcf11-2 mpe1-s partially restores accu- <br /> mRNA ends revealed Northern blotting (Fig. 3B). As                mulation actin mRNA correct choice cleavage <br /> already described, wild-type strain tran-              site (Fig. 3C, lanes 13 15). <br /> scripts cleaved proximal site sites                We conclude results MPE1 required <br /> were minor importance regardless temperature (lanes 1               specificity mRNA 3&#226;&#172;&#732;-end cleavage vivo. <br /> to 3). Interestingly, situation reversed mpe1-1               Mpe1p essential vitro cleavage polyadenyla- <br /> and pcf11-2 mutants, distal site                tion. Extracts mpe1-1 mutant strain used test <br /> major site cleaved permissive temperatures site           role Mpe1p specific cleavage polyadenylation in <br /> used 37&#194;&#176;C (lanes 4 6 lanes 7 9, respectively). This          vitro. The cleavage reaction synthetic pre- <br /> demonstrates mpe1-1 pcf11-2 mutations alter              mRNA containing 3&#226;&#172;&#732; end CYC1 showed the <br /> choice cleavage site ACT1 transcripts way           activity mpe1-1 extract reduced 25 32&#194;&#176;C and <br /> as observed mutants polyadenyla-              weak 35&#194;&#176;C (Fig. 4A). At temperatures <br /> tion complex (2, 31).                                                     percentages 5&#226;&#172;&#732; cleaved product respectively 67, 96, <br />    Northern blotting analysis revealed pcf11-2 mpe1-1            98% wild-type strain (lanes 2, 4, 6) 22, 15, <br /> mutants defective accumulation actin mRNA                   7% mpe1-1 mutant strain (lanes 3, 5, 7) The <br /> when shifted restrictive temperatures (Fig. 3C, lanes 5 8           polyadenylation reaction carried precleaved <br /> and lanes 9 12, respectively); decrease              CYC1 synthetic mRNA showed specific polyadenyl- <br /> seen CYC1, URA1, URA2 mRNAs (data shown).                     ation completely abolished 25&#194;&#176;C (Fig. 4B, lane 3). <br /> This defect observed thermosensitive             These results confirmed similar assays carried on <br />  8350     VO ET AL.                                                                                                              MOL. CELL. BIOL. <br />  <br />  <br />                                                                          complex mpe1-1 extract contains active CFI <br />                                                                          complex inactive CPF complex. <br />                                                                            Mpe1p integral subunit CPF complex. TAP was <br />                                                                          used mild method purifying proteins associated <br />                                                                          Mpe1p Pfs2p subunit CPF com- <br />                                                                          plex (42). The TAP-tag sequence inserted immediately <br />  <br />  <br />  <br />  <br />    FIG. 3. MPE1 required choice ACT1 mRNA polyade- <br /> nylation site vivo. (A) Structure 3&#226;&#172;&#732; end ACT1 gene. <br /> Arrows indicate potential polyadenylation sites numbered 1 5 <br /> from 5&#226;&#172;&#732; 3&#226;&#172;&#732;. The position specific oligonucleotide used for <br /> RNase H treatment indicated black bar. (B) Comparison the <br /> polydenylation sites ACT1 mRNA. Total RNA extracted at <br /> time zero various times shift 37&#194;&#176;C. Equal quantities <br /> (20 &#226;?&#174;g) sample treated RNase H hybridization <br /> to complementary oligonucleotide oligo(dT) shorten the <br /> ACT1 mRNA eliminate poly(A) tails. After PAGE and <br /> Northern blotting, different ACT1 mRNA was <br /> estimated. The marker sizes indicated nucleotides left. <br /> (C) Analysis ACT1 mRNA decay shift 37&#194;&#176;C. Equal <br /> quantities total RNA (10 &#226;?&#174;g) extracted indicated times at <br /> 37&#194;&#176;C separated 1.2% agarose gel presence 5% <br /> formaldehyde, ACT1 mRNA visualized Northern blotting. <br /> The polyadenylation sites indicated right. <br />  <br />  <br /> wild-type extracts presence increasing quantities              FIG. 4. Mpe1p necessary vitro 3&#226;&#172;&#732;-end processing. (A) Spe- <br /> specific anti-Mpe1p antibodies. Polyadenylation com-                 cific vitro cleavage assay. Equal quantities wild-type extract (lanes <br /> pletely blocked immunoneutralizing Mpe1p (Fig. 4C, lanes              2, 4, 6) mpe1-1 mutant extract (lanes 3, 5, 7) added to <br />                                                                          cleavage reaction mix containing CYC1 pre-mRNA (lane 1, <br /> 5 6), preimmune serum effect (Fig. 4C,            precursor alone). The temperature reaction indicated the <br /> lanes 3 4). Cleavage slightly modified             top. The positions substrate (CYC1 mRNA) 5&#226;&#172;&#732; <br /> conditions (data shown)                                              cleavage product (5&#226;&#172;&#732;CP) indicated right. The positions and <br />    A rna15-2 mutant extract inactive loss           sizes markers indicated left nucleotides. (B) <br />                                                                          Specific vitro polyadenylation assay. The experiment carried out <br /> of CFIA function (Fig. 4D, lane 4) (35). Nevertheless,            panel A reaction mix specific polyadenylation <br /> could complement mpe1-1 extract (Fig. 4D, lane 6). Iden-             used presence precleaved CYC1 mRNA (CYC1 pre) to <br /> tical results obtained extract rna14-1 mutant            obtain polyadenylated products (pA). Lane 1, precursor alone; lane 2, <br /> (data shown). Similar complementations             wild-type extract; lane 3, mutant mpe1-1 extract. Markers in <br /> shown mutant extracts CFI subunits mutant                 panel A. (C) Mpe1p immunoneutralization inhibits vitro polyade- <br />                                                                          nylation. In vitro polyadenylation carried panel B 25&#194;&#176;C. <br /> extracts CPF subunits (fip1-1, yth1-1, pta1-1, pfs2-1) (7,        Lane 1, precursors alone; lane 2, wild-type extract alone; lanes 3 4, <br /> 38, 39, 53). Conversely, fip1-1 mutant extract inactivated           wild-type extract increasing amounts preimmune serum; lanes <br /> for CPF function (Fig. 4D, lane 5) did complement                5 6, wild-type extract increasing amounts anti-Mpe1p <br /> polyadenylation deficiency mpe1-1 extract (Fig. 4D, lane          antibodies raised C-terminal domain protein. Mark- <br />                                                                          ers described panel A. (D) Complementation polyadenyl- <br /> 7).                                                                      ation activity mpe1-1 mutant extract. In vitro polyadenylation <br />    This leads conclude Mpe1p important                 carried panel B 25&#194;&#176;C. Extracts assayed or <br /> function 3&#226;&#172;&#732;-end formation likely level CPF          combined indicated lane. <br />  VOL. 21, 2001                                                                     Mpe1 IS A NEW SUBUNIT OF YEAST CPF                      8351 <br />  <br />  <br /> upstream MPE1 PFS2 stop codon wild-type strain <br /> (see Materials Methods). This insertion did affect the <br /> growth rate corresponding strains temperature <br /> (data shown). Pfs2p chosen control the <br /> protein A-tagged Pfs2p successfully used to <br /> purify CPF complex (38). <br />    We used SDS-polyacrylamide gel electrophoresis (PAGE) <br /> to separate purified proteins PFS2-TAP strain and <br /> then stained Coomassie blue (Fig. 5A, lane 1). The <br /> pattern identical previously published CPF <br /> purified affinity protein A-tagged Pfs2p or <br /> Yth1p biochemical techniques (6, 38, 40). The purified <br /> Mpe1p-associated proteins shown Fig. 5A, lane 2. It can <br /> be seen profile identical obtained with <br /> Pfs2p (lane 1). In addition, used MALDI-mass spectrom- <br /> etry identify components CPF complex after <br /> in-gel proteolytic digestion endoproteinase LysC. Seven <br /> peptide masses matched values calculated theoretical <br /> (in silico) Mpe1p digest (data shown). Thus, Mpe1p is <br /> clearly proteins associated Pfs2p (lane 1). <br /> Mpe1p migrates gels apparent molecular mass 58 <br /> kDa, calculated mass 49.5 kDa (data shown). <br /> Hence, Mpe1p comigrates Pfs2p, remains associ- <br /> ated calmodulin-binding peptide (Pfs2p-CBP) after <br /> the TAP purification molecular mass was <br /> increased 5 kDa. <br />    The proteins present complexes subjected to <br /> immunoblotting. The repartition Mpe1p, Fip1p, Pap1p <br /> was monitored purification CPF ob- <br /> tained strains PFS2-TAP (Fig. 5B) MPE1-TAP <br /> (Fig. 5C) whole-cell extract (lane 1), effluent the <br /> IgG agarose column (lane 2), purified complex (lane <br /> 3). The proteins mainly purified <br /> complexes present low level effluent. It <br /> is noteworthy Fip1p Pap1p mainly copurified <br /> with MPE1-TAP-purified CPF, suggesting the <br />                                                                           FIG. 5. Mpe1p belongs CPF complex. (A) Analysis the <br /> CPF complex contains Mpe1p protein. The CFIA proteins <br />                                                                        multiprotein complex associated TAP-tagged Pfs2p the <br /> (Rna14p, Rna15p, Pcf11p, Clp1p) (36) Pab1p                TAP-tagged Mpe1p. After purification TAP procedure starting <br /> not detected blots purified CPFs (data shown).           strains PFS2-TAP (lane 1) MPE1-TAP (lane 2), proteins <br /> Moreover, coimmunoprecipitation experiments interac-             separated 10% polyacrylamide-SDS gel stained <br /> tion obtained recombinant Pcf11p vitro-             Coomassie blue. The indicated proteins identified mass <br />                                                                        spectrometry bands excised gel. CBP indicates the <br /> translated Mpe1p vitro-translated proteins          TAP-tagged proteins remained fused CBP purifi- <br /> Rna14p Mpe1p (data shown), indicating               cation. (B C) Western blot analysis TAP. Mpe1p, Fip1p, and <br /> would direct stable interaction Mpe1p             Pap1 detected specific antibodies extract (lane 1), <br /> and Pcf11p.                                                            material bound IgG agarose (lane 2), purified <br />                                                                        complex (lane 3), starting PFS2-TAP strain (B) from <br />    All data clearly association Mpe1p <br />                                                                        MPE1-TAP strain (C). The fractions contain identical quantities of <br /> and CPF complex. This strong association          cells. TAP CBP indicate proteins fused TAP <br /> IgG agarose washed buffer higher ionic               peptide CBP; Mpe1p* truncated form Mpe1p revealed <br /> strength usually used (400 mM NaCl instead 150           anti-Mpe1p antibodies detected variable quantities all <br /> mM NaCl). Mpe1p does appear belong                 extracts. It purified complexes not <br />                                                                        fused C-terminal TAP-tag. <br /> complexes CPF complex proteins puri- <br /> fied Mpe1p-TAP. The lack complementation be- <br /> tween mpe1-1 fip1-1 (see Fig. 4D) shows Mpe1p            CPF modified mpe1-1 mutation, complex was <br /> is assembled CPF, diffuse complex             purified TAP-tagged Pfs2p mpe1-1 mutant <br /> another vitro shown components            background. The pattern obtained gel (Fig. 6A, lane 2) was <br /> the cleavage polyadenylation complex (35, 40). Therefore,          identical wild-type CPF patterns obtained the <br /> Mpe1p considered integral subunit CPF,                MPE1-TAP strain (lane 1) PFS2-TAP strain (Fig. <br /> required cleavage polyadenylation vitro.               5A, lane 1). The apparent stoichiometry Cft1p/Yhh1p, <br />    The mutant protein, Mpe1-1p, tightly associated         Cft2p/Ydh1p, Brr5p/Ysh1p, Pta1 similar the <br /> the CPF complex. To investigate composition             wild-type mutant backgrounds. Since Mpe1-1p trun- <br />  8352     VO ET AL.                                                                                                          MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 6. The mutant protein, Mpe1-1p, stably associated CPF complex. (A) Analysis protein complexes associated the <br /> mutant proteins Mpe1-1p Mpe1&#226;&#338;&#172;Ctermp. The different complexes purified analyzed described Fig. 5A wild-type <br /> strain MPE1-TAP (lane 1) mpe1-1 PFS2-TAP (lane 2), MPE1&#226;&#338;&#172;Cterm-TAP (lane 3), mpe1-1&#226;&#8364;&#8220;TAP (lane 4) mutant strains. (B and <br /> C) Western blot analysis TAP mpe1-1 PFS2-TAP (B) MPE1&#226;&#338;&#172;Cterm-TAP (C) strains described Fig. 5B C. <br />  <br />  <br />  <br /> cated C terminus, apparent molecular mass gel            stringency washing step IgG agarose <br /> was close Fip1p (data shown). Thus, used          reduced (150 mM NaCl instead 400 mM NaCl [data not <br /> immunodetection reveal presence absence                     shown]). The mutant phenotype essentially the <br /> Mpe1-1p PFS2-TAP mpe1-1 strain (Fig. 6B). Mpe1-1p                 Mpe1-1p truncation. Indeed, constructed strain which <br /> was present cellular extract; thus, stable        wild-type ORF TAP tagged position the <br /> accumulates cells. However,               nonsense codon. In strain CPF complex co- <br /> effluent IgG agarose column isolated             purified shortened Mpe1&#226;&#338;&#172;Cterm protein (Fig. 6A, <br /> CPF complex. This result strengthened purification                lane 3), Fip1p mainly detected purified complex <br /> from strain mutant Mpe1-1p protein TAP                immunoblotting (Fig. 6C). <br /> tagged stop codon generated nonsense mutation.                The Mpe1-1p mutant protein largely lost affinity CPF. <br /> Only mutant protein fused CBP, Mpe1-1p&#226;&#8364;&#8220;                Nevertheless, presence relative levels other <br /> CBP, purified proteins               components modified considerably complex <br /> CPF complex (Fig. 6A, lane 4). The result obtained              lacking Mpe1p, prepared (PFS2-TAP <br />  VOL. 21, 2001                                                                  Mpe1 IS A NEW SUBUNIT OF YEAST CPF                 8353 <br />  <br />  <br /> mpe1-1) mutant extract. Thus, conclude that,         previously shown CPFs purified wild-type strains <br /> Mpe1p essential 3&#226;&#172;&#732;-end processing, implicated    (38, 40), polyadenylation precleaved mRNA Mpe1p- <br /> the overall stability CPF.                                        plus CPF processive. Only small substrate <br />    Mpe1p required vitro specificity mRNA            RNA elongated (Fig. 7E, lane 4). (ii) The length the <br /> 3&#226;&#180;&#8226;-end processing. The purification Mpe1p-devoided CPF          poly(A) tails regulated. (iii) This activity com- <br /> allowed analyze role Mpe1p 3&#226;&#172;&#732;-end processing.        pletely blocked excess recombinant Pab1p (Fig. 7E, lane 8). <br /> We tested cleavage polyadenylation activities recon-      Remarkably identical results obtained purified <br /> stituted assays purified factors CFI CPF,        Mpe1p-minus CPF (Fig. 7E, lanes 5 9). Thus, this <br /> with Mpe1p, recombinant Pab1p. We pu-             complex devoided Mpe1p unspecific polyadenylation <br /> rified complexes step TAP procedure     activity maintained, poly(A) addition remained pro- <br /> after elution TEV protease. CPF isolated          cessive (lane 5), suggesting interactions CPF <br /> from MPE1-TAP mpe1-1 PFS2-TAP strains. In              RNA substrate preserved absence of <br /> conditions PAGE did reveal major contaminants          Mpe1p (40). The polyadenylation remained totally inhibited by <br /> Coomassie blue staining, CFIA proteins detected          Pab1p (lane 9). These data indicate Mpe1p be <br /> by immunodetection CPF (data shown). As be-            directly implicated modulation Pap1p activity by <br /> fore, CPF isolated wild-type MPE1-TAP strain            CPF. This consistent observations presented here <br /> contained Mpe1p named Mpe1p-plus CPF,                showing composition stability CPF not <br /> the complex purified mpe1-1 PFS2-TAP mutant strain          affected significantly absence Mpe1p. <br /> did contain mutant Mpe1-1p. It named Mpe1p-minus                As previously shown (33, 38), specific polyadenylation <br /> CPF.                                                                 took place CFI, wild-type Mpe1p-plus CPF, and <br />    Mutant mpe1-1 fip1-1 extracts inactive polyade-       Pab1p combined (Fig. 7E, line 10). CFI necessary to <br /> nylation loss CPF function (Fig. 7A, lanes 3       prevent inhibition polyadenylation Pab1p. The length <br /> 4, respectively) (39). Mpe1p-plus CPF functional com-        synthesized poly(A) tails regulated similar <br /> plemented polyadenylation deficiency mutants             obtained wild-type extract (Fig. 7E, lane 12). It has <br /> (Fig. 7A, lanes 5 6), Mpe1p-minus complex            shown Pab1p directly involved regulation of <br /> was unable provide complementation (Fig. 7A, lanes 7         process (3, 36). Indeed, long poly(A) tails synthesized <br /> and 8). This confirms essential role Mpe1p specific    mutant extract Pab1p extract immunodepleted <br /> processing activity CPF. Neither CPF complemented             protein. In absence Pab1p, long poly(A) tails <br /> cleavage defect mutant rna14-1 extract (data shown).      observed reconstituted assay CFI and <br />    We constructed strain expressing TAP-tagged Pcf11p          Mpe1p-plus CPF (Fig. 7E, lane 6). In assay poly(A) tail <br /> to isolate wild-type CFI TAP method. An rna14-1 ex-           profile identical obtained Mpe1p-plus CPF <br /> tract, mutated CFI subunit, deficient cleavage       (Fig. 7E, compare lanes 4 6). The evidence for <br /> polyadenylation (Fig. 7B, lane 3, Fig. 7C, lane 3, respec-       role CFI activity Mpe1p-plus CPF totally <br /> tively). The purified CFI complemented 3&#226;&#172;&#732;-end process-          inhibited Pab1p (Fig. 7E, lane 8), CPF com- <br /> ing steps rna14-1 mutant extract (Fig. 7B, lane 5, Fig.   bined CFI Pab1p results regulated poly(A) ad- <br /> 7C, lane 7, respectively). Nevertheless, rescue     dition (Fig. 7E, lane 10). The polyadenylation be <br /> polyadenylation defect mpe1-1 fip1-1 mutant ex-           reconstituted assay containing Mpe1p-minus CPF, CFI, <br /> tracts (Fig. 7C, lanes 8 9). Thus, fraction essentially     Pab1p (Fig. 7E, lane 11), demonstrating Mpe1p is <br /> contained active CFI factor used reconsti-        required vitro specific poly(A) addition. The poly- <br /> tuted 3&#226;&#172;&#732;-end processing assays.                                      adenylation activity mixture CFI Mpe1p- <br />    The purified factors showed residual processing activi-     minus CPF (Fig. 7E, lane 7) remained inhibited Pab1p (Fig. <br /> ties. There low level polyadenylation activity      7E, lane 11). <br /> CFI (Fig. 7C, lane 6). This certainly presence of <br /> a small Pap1p purified fraction                                     DISCUSSION <br /> immunoblotting, Mpe1p detected the <br /> same conditions (data shown). Indeed, Pap1p             Our results demonstrate zinc knuckle protein, Mpe1p, <br /> been shown associated purified CFI (27). As previ-          required steps mRNA 3&#226;&#172;&#732;-end processing S. <br /> ously shown (38), low level cleavage activity associated    cerevisiae. Mpe1p essential cell viability. In vitro assays <br /> with wild-type Mpe1p-plus CPF (Fig. 7D, lane 4). How-            demonstrated extract conditionally lethal <br /> ever, cleavage activity restored CFI          mpe1-1 mutant strain impaired cleavage polyadenyl- <br /> Mpe1p-plus CPF associated (Fig. 7D, lane 6). In        ation antibodies raised recombinant Mpe1p <br /> conditions activity reconstituted pres-     inhibit polyadenylation wild-type strain extract. By using <br /> ence Mpe1p-minus CPF (Fig. 7D, lane 7). This definitely           TAP method, Mpe1p copurifies the <br /> confirms Mpe1p necessary cleavage.                       CPF isolated TAP-tagged Pfs2p leaving signif- <br />    Pap1p known catalyze 3&#226;&#172;&#732;-end polyadenylation       icant amounts soluble Mpe1p extract depleted from <br /> of RNA substrates specificity regardless          complex. Conversely, CPF purified TAP- <br /> length poly(A) tails (29, 40). We confirmed RNA      tagged Mpe1p. The protein profile complex iden- <br /> molecules simultaneously polyadenylated re-         tical complexes purified tagged forms <br /> action, showing polymerization distributive (Fig.       different genuine CPF components (6, 38, 40). Thus, Mpe1p <br /> 7E, lane 3). A number conclusions made. (i) As             novel integral subunit CPF. Consistent re- <br />  8354     VO ET AL.                                                                                                            MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 7. Role Mpe1p in vitro 3&#226;&#172;&#732;-end processing reactions reconstituted purified CFI CPF complexes. (A) Complementation of <br /> polyadenylation activity purified CPF. The reactions performed 25&#194;&#176;C precleaved CYC1 mRNA described Fig. 4B. The extracts <br /> were incubated combination purified Mpe1p-plus CPF Mpe1p-minus CPF indicated figure. The positions <br /> of precleaved CYC1 mRNA (CYC1 pre) polyadenylated products (pA) shown right. The positions sizes the <br /> molecular mass markers shown left nucleotides. (B) Complementation cleavage activity purified CFI. The reactions were <br /> performed 25&#194;&#176;C CYC1 pre-mRNA described Fig. 4A. The extracts incubated combination purified CFI as <br /> indicated figure. The positions substrate (CYC1 mRNA) 5&#226;&#172;&#732; cleavage product (5&#226;&#172;&#732;CP) shown right. <br /> Molecular mass markers described panel A. (C) Complementation polyadenylation activity purified CFI. The reactions were <br /> performed described panel A. The extracts incubated combination purified CFI, indicated figure. <br /> Molecular mass markers described panel A. (D) Cleavage activity reconstituted purified complexes. The reactions performed <br /> as described panel B CYC1 pre-mRNA. Purified CFI, Mpe1p-plus CPF, Mpe1p-minus CPF associated indicated top <br /> of figure. Molecular mass markers described panel A. (E) Polyadenylation activity reconstituted purified complexes. The <br /> reactions performed described panel A pre-cleaved CYC1 mRNA. Purified CFI, Mpe1p-plus CPF, Mpe1p-minus CPF, recom- <br /> binant Pap1p (USB, Cleveland, Ohio), recombinant Pab1p (125 ng/assay) associated indicated figure. Molecular mass <br /> markers described panel A. <br />  <br />  <br /> sults, zinc knuckle protein, Pfs1p, molecular mass                 In mutant mpe1-1 strain, Mpe1p associated with <br /> similar Mpe1p repeatedly copurified              CPF. Nevertheless, possible isolate CPF <br /> CPF, corresponding gene char-               subunits formed stable structure devoid Mpe1p. We <br /> acterized (6, 38, 40, 53).                                                conclude (i) mpe1-1 mutation drastically decreases <br />  VOL. 21, 2001                                                                    Mpe1 IS A NEW SUBUNIT OF YEAST CPF                               8355 <br />  <br />  <br /> the affinity protein CPF complex (ii) Mpe1p       strongly suggest Mpe1p highly conserved protein and <br /> is essential overall stability CPF. Thus,         true human homolog exists. It remains determined <br /> processing defect displayed mpe1-1 mutant          human protein functional homolog of <br /> result CPF destabilization, contrast        Mpe1p identified mammalian <br /> observed strains mutated subunits CPF (6, 38,        polyadenylation complex protein assumes different <br /> 53). We showed CPF, Mpe1p, dis-            function higher eukaryotic cells. Interestingly, the <br /> plays polyadenylation activity. It able catalyze   human cDNAs, contains domain conserved Mpe1p, <br /> processive poly(A) addition RNA substrate,            codes Rbbp6 protein, shown this <br /> the poly(A) polymerase, Pap1p, distributive poly-        protein interacts tumor suppressor pRB1 (43). The <br /> adenylation (40). This implies Mpe1p involved         pRB1 protein involved cell differentiation localized <br /> the modulation Pap1p activity CPF complex. CPF,         centers mRNA processing nucleus (13, 51). This <br /> particular, form stable complex RNA             support idea human Mpe1 homolog partic- <br /> free Pap1p does, absence Mpe1p. However,          ipate 3&#226;&#172;&#732;-end processing. <br /> binding CPF substrate specific, CFI <br /> factor required proper substrate recognition                                    ACKNOWLEDGMENTS <br /> accurate 3&#226;&#172;&#732;-end processing (6, 33, 38). Reconstituted assays           We grateful Bertrand Seraphin, Lionel Minvielle-Sebastia, <br /> with purified factors demonstrated Mpe1p strictly re-        Yves D&#226;&#8364;&#8482;Aubenton, Claude Thermes helpful discussions. We <br /> quired cleavage pre-mRNA poly(A)            indebted Marc Bonneu Katell Bathany contributing to <br />                                                                      work. We thank Marie Elisabeth Dufour technical assis- <br /> addition genuine polyadenylation site. This indicates         tance. We thank Bertrand Seraphin Lionel Minvielle-Sebastia <br /> that Mpe1p involved mechanism CFI             providing plasmids antibodies. <br /> CPF function synergistically form correct 3&#226;&#172;&#732; ends. Sev-         This work supported Centre National la Recherche <br /> eral observations suggest role Mpe1p mechanism.         Scientifique, Association pour la Recherche sur le Cancer (grant <br />                                                                      9922), Fondation pour la Recherche Me&#194;&#180;dicale. <br /> (i) Mpe1p required stabilize interactions be- <br /> tween CFI CPF, leading formation stable                                               REFERENCES <br /> specific processing complex. Consistent hypothesis,         1. Altschul, S. F., T. L. Madden, A. A. Schaffer, J. Zhang, Z. Zhang, W. Miller, <br /> some pcf11 mutations suppressed Mpe1p mutant,                  D. J. Lipman. 1997. Gapped BLAST PSI-BLAST: new generation <br />                                                                          protein database search programs. Nucleic Acids Res. 25:3389&#226;&#8364;&#8220;3402. <br /> suggesting proteins interact closely          2. Amrani, N., M. E. Dufour, N. Bonneaud, F. Lacroute. 1996. Mutations <br /> CFI CPF associated. (ii) Mpe1p contains zinc                   STS1 suppress defect 3&#226;&#172;&#732; mRNA processing caused rna15&#226;&#8364;&#8220;2 <br /> knuckle protein putative RNA-binding pro-             mutation Saccharomyces cerevisiae. Mol. Gen. Genet. 252:552&#226;&#8364;&#8220;562. <br />                                                                       3. Amrani, N., M. Minet, M. Le Gouar, F. Lacroute, F. Wyers. 1997. yeast <br /> tein (9, 28). This supported fact zinc knuckle        Pab1 interacts Rna15 participates control poly(A) tail <br /> motif present CPSF 30K subunit mammalian                   length vitro. Mol. Cell. Biol. 17:3694&#226;&#8364;&#8220;3701. <br />                                                                       4. Amrani, N., M. Minet, F. Wyers, M. E. Dufour, L. P. Aggerbeck, F. <br /> cleavage polyadenylation specificity factor involved           Lacroute. 1997. PCF11 encodes protein component yeast cleavage <br /> the interactions protein pre-mRNA (7). Mpe1p            polyadenylation factor I. Mol. Cell. Biol. 17:1102&#226;&#8364;&#8220;1109. <br /> may components required anchor process-          5. Aravind, L., E. V. Koonin. 2000. The U box modified RING <br />                                                                          finger&#226;&#8364;&#8221;a common domain ubiquitination. Curr. Biol. 10:R132&#226;&#8364;&#8220;134. <br /> ing machinery correct location RNA. (iii) Mpe1p          6. Barabino, S. M., M. Ohnacker, W. Keller. 2000. Distinct roles two <br /> involved recognition correct ACT1 mRNA cleav-              Yth1p domains 3&#226;&#172;&#732;-end cleavage polyadenylation yeast pre-mRNAs. <br /> age site vivo. Like ACT1, large number yeast genes               EMBO J. 19:3778&#226;&#8364;&#8220;3787. <br />                                                                       7. Barabino, S. M. L., N. Hub, W. A. Jenny, L. Minvielle-Sebastia, W. <br /> contain multiple potential polyadenylation sites (47). Some           Keller. 1997. The 30-kD subunit mammalian cleavage polyadenyla- <br /> them produce transcripts different lengths choice          tion specificity factor yeast homolog RNA-binding zinc finger <br />                                                                          proteins. Genes Dev. 11:1703&#226;&#8364;&#8220;1716. <br /> between alternative poly(A) sites response modifications        8. Baudin, A., O. Ozier-Kalogeropoulos, A. Denouel, F. Lacroute, C. Cul- <br /> of physiological state cells (22, 45). These observations         lin. 1993. A simple efficient method direct gene deletion Saccha- <br /> suggest regulated 3&#226;&#172;&#732;-end processing mecha-            romyces cerevisiae. Nucleic Acids Res. 21:3329&#226;&#8364;&#8220;3330. <br />                                                                       9. Berg, J. M., Y. Shi. 1996. The galvanization biology: growing <br /> nisms regulation gene expression compo-               appreciation roles zinc. Science 271:1081&#226;&#8364;&#8220;1085. <br /> nents polyadenylation complex, Mpe1p,            10. Boeke, J. D., F. Lacroute, G. R. Fink. 1984. A positive selection for <br /> implicated.                                                              mutants lacking orotidine-5&#226;&#172;&#732;-phosphate decarboxylase activity yeast: <br />                                                                          5-fluoro-orotic acid resistance. Mol. Gen. Genet. 197:345&#226;&#8364;&#8220;346. <br />    The sequence data Mpe1p homologues exist             11. Butler, J. S., P. P. Sadhale, T. Platt. 1990. RNA processing vitro <br /> S. pombe, A. thaliana, D. melanogaster. The zinc knuckle-            produces mature 3&#226;&#172;&#732; ends variety Saccharomyces cerevisiae mRNAs. <br />                                                                          Mol. Cell. Biol. 10:2599&#226;&#8364;&#8220;2605. <br /> containing domain regions conserved        12. Vries, H., U. Ruegsegger, W. Hubner, A. Friedlein, H. Langen, W. <br /> these organisms. Since polyadenylation complex               Keller. 2000. Human pre-mRNA cleavage factor II(m) contains homologs of <br /> yet thoroughly studied species,              yeast proteins bridges cleavage factors. EMBO J. 19:5895&#226;&#8364;&#8220; <br />                                                                          5904. <br /> know putative homologues involved 3&#226;&#172;&#732;-           13. Durfee, T., M. A. Mancini, D. Jones, S. J. Elledge, W. H. Lee. 1994. The <br /> end processing. In mammals, CPSF 30K subunit contains                amino-terminal region retinoblastoma gene product binds novel <br /> five zinc finger repeats zinc knuckle motif,       nuclear matrix protein co-localizes centers RNA processing. <br />                                                                          J. Cell Biol. 127:609&#226;&#8364;&#8220;622. <br /> participate interactions protein pre-mRNA.       14. Duvel, K., G. H. Braus. 1999. Different positioning elements select <br /> The yeast homolog CPSF 30K, Yth1p, contains zinc                 poly(A) sites 3&#226;&#172;&#732;-end GCN4 mRNA yeast Saccharomyces <br />                                                                          cerevisiae. Nucleic Acids Res 27:4751&#226;&#8364;&#8220;4758. <br /> fingers zinc knuckle (7). Since motif        15. Gallie, D. R. 1998. A tale termini: functional interaction the <br /> Mpe1p, speculate role mammalian CPSF              termini mRNA prerequisite efficient translation initiation. <br /> 30K protein assumed yeast proteins, Yth1p               Gene 216:1&#226;&#8364;&#8220;11. <br />                                                                      16. Gietz, R. D., A. St. Jean, R. A. Woods, R. H. Schiestl. 1992. Improved <br /> Mpe1p, function Mpe1p incorporated               method high efficiency transformation intact yeast cells. Nucleic Acids <br /> into CPSF 30K higher eukaryotes. However, results                 Res. 20:1425. <br />  8356       VO ET AL.                                                                                                                             MOL. CELL. BIOL. <br />  <br />  <br /> 17. Graber, J. H., C. R. Cantor, S. C. Mohr, T. F. Smith. 2001. Genomic            Proc. Natl. Acad. Sci. USA 94:7897&#226;&#8364;&#8220;7902. <br />     detection new yeast pre-mRNA 3&#226;&#172;&#732; end processing signals. Nucleic Acids       37. Minvielle-Sebastia, L., B. Winsor, N. Bonneaud, F. Lacroute. 1991. <br />     Res 27:888&#226;&#8364;&#8220;894.                                                                    Mutations yeast RNA14 RNA15 genes result abnormal <br /> 18. Grandier-Vazeille, X., K. Bathany, S. Chaignepain, N. Camougrand, S.               mRNA decay rate: sequence analysis reveals RNA-binding domain the <br />     Manon, J. Schmitter. 2001. Yeast mitochondrial dehydrogenases              RNA15 protein. Mol. Cell. Biol. 11:3075&#226;&#8364;&#8220;3087. <br />     associated supramolecular complex. Biochemistry 40:9758&#226;&#8364;&#8220;9769.             38. Ohnacker, M., S. M. Barabino, P. J. Preker, W. Keller. 2000. The <br /> 19. Guthrie, C., G. R. Fink. 1991. Guide yeast genetics molecular           WD-repeat protein pfs2p bridges essential factors yeast <br />     biology, vol. 194. Academic Press, Inc., New York, N.Y.                            pre-mRNA 3&#226;&#172;&#732;-end-processing complex. EMBO J 19:37&#226;&#8364;&#8220;47. <br /> 20. Harlow, E., D. Lane. 1988. Antibodies: laboratory manual. Cold Spring    39. Preker, P. J., J. Li <br /> </body></html>