<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />            Characterization of the sat Operon in Streptococcus mutans: <br />                  Evidence for a Role of Ffh in Acid Tolerance <br />  <br />              An essential protein translocation pathway in Escherichia coli and Bacillus subtilis involves the signal <br />           recognition particle (SRP), of which the 54-kDa homolog (Ffh) is an essential component. In a previous study, <br />           we found that a transposon insertion in the ylxM-ffh intergenic region of the designated secretion and acid <br />           tolerance (sat) operon of Streptococcus mutans resulted in an acid-sensitive <span id='am-164' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-165' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-167' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-168' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-169' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-174' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-176' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-178' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-186' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-188' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-189' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-193' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-194' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-197' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-199' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-200' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-201' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-202' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-203' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-210' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-211' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-213' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-214' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-216' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-222' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-225' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-226' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-227' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-233' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-236' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-238' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-239' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-240' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span>phenotype</span>. In the present study, we <br />           further characterized this genomic region in S. mutans after construction of bonafide sat operon mutants and <br />           confirmed the role of the SRP pathway in acid resistance. Northern blot and primer extension analyses <br />           identified an acid-inducible promoter upstream of ylxM that was responsible for upregulating the coordinate <br />           expression of all five genes of the sat operon when cells were grown at acid pH. Two constitutive promoters, one <br />           immediately upstream of satD and one just 3&#226;&#180;&#8226; <span id='am-7' about='protege:TO' typeof='owl:Thing'>to</span> the acid-inducible promoter, were also identified. Except for <br />           Ffh, the functions of the sat operon gene products are unknown. SatC, SatD, and SatE have no homology to <br />           proteins with known functions, although YlxM may function as a transcriptional regulator linked to genes <br />           encoding SRP pathway proteins. Nonpolar mutations created in each of the five genes of the sat locus resulted <br />           in viable mutants. Most striking, however, was the finding that a <span id='am-242' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-243' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-245' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-248' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-249' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-254' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-255' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-256' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-258' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-259' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-260' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-261' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-268' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-270' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-273' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-275' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-277' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-279' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-284' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-288' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-289' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-291' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-296' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-298' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-301' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-302' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-303' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-304' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-305' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-306' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-308' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-310' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-311' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-313' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-314' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-316' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-317' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-318' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-319' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span>mutation</span> in ffh did not result in loss of cell <br />           viability, as is the case in all other microbial species in which this pathway has been described. This mutant <br />           also lacked immunologically detectable Ffh and was severely affected in resistance to acid. Complementation <br />           of the mutation resulted in restoration of acid tolerance and reappearance of cytoplasmic Ffh. These data <br />           provide evidence that the SRP pathway plays an important role in acid tolerance in S. mutans. <br />  <br />  <br />    Acid tolerance is regarded as an important determinant for                   phenicol in the medium (17). Sustained mildly acidic condi- <br /> <span id='am-8' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-9' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-41' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-83' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-85' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span>survival</span> of cariogenic bacteria in the biofilm on teeth (7, 37,                 tions increase H&#226;&#171;&#185;/ATPase and glycolytic activities (2, 4, 15, <br /> 49.) The microorganisms in dental plaque are exposed to con-                    16). These metabolic changes permit S. mutans to maintain a <br /> tinual cycles of acid shock resulting <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> the formation of acid                constant transmembrane pH gradient of approximately 1.0 pH <br /> end products during metabolism of dietary carbohydrate by the                   <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> (8, 15). <br /> acidogenic microflora. In vivo pH measurements have revealed                       Previous genetic studies in our laboratory focused on iden- <br /> that the intake of carbohydrates can drop the plaque pH from                    tification of genes that when inactivated by transposon Tn917 <br /> 7 to 4 in less than 20 min (21, 24, 52). In addition, frequent                  rendered the mutants acid sensitive (12). One of the mutants <br /> carbohydrate intake results in a lower pH of &#226;&#8364;&#339;resting plaque,&#226;&#8364;&#65533;                  (AS17) affected in acid tolerance had the transposon inte- <br /> which has been associated with increased dental caries. Early                   grated into the intergenic region between the ylxM and ffh <br /> studies (41) demonstrated that high caries activity is associated               homologs. This locus was subsequently designated sat for se- <br /> with rapid plaque acidification as well as tolerance of low pH                  cretion and acid tolerance (12). <br /> by the plaque microorganisms.                                                      The ffh gene encodes a 54-kDa subunit homolog of the <br />    Streptococcus mutans has been recognized as one of the                       eucaryotic signal recognition particle (SRP) complex, which is <br /> major etiological agents in human dental caries (14, 27). In                    part of a protein translocation pathway that is highly conserved <br /> addition to its ability to colonize tooth surfaces and acidoge-                 throughout the domains of life (6, 9, 29, 54). Detailed studies <br /> nicity, S. mutans is acid tolerant. Glycolysis can occur at pH                  of Escherichia coli and Bacillus subtilis have shown that this <br /> values as low as pH 4.4, whereas the minimum pH for growth                      pathway is involved in protein export and membrane biogen- <br /> is 4.8 (5). Svensa&#194;&#168;ter et al. (42) demonstrated that a 2-h expo-                esis. The protein encoded by ylxM is homologous to a protein <br /> sure of exponential-phase cells to pH 5.5 induced an acid                       of unknown function in Bacillus subtilis that is transcribed as an <br /> tolerance response (ATR) that increased subsequent survival                     operon with ffh (19). YlxM contains helix-turn-helix motifs <br /> at pH 3.0. The mechanisms involved in acid tolerance of S.                      indicative of DNA-binding proteins, therefore suggesting a <br /> mutans are poorly understood. Protein synthesis is required for                 function in transcriptional regulation (35). Remarkably, the <br /> induction of an ATR, since it could be blocked by chloram-                      transposon integration in AS17 resulted in a viable mutant. In <br />                                                                                 E. coli and B. subtilis, ffh is a gene essential for viability (19, 33, <br />                                                                                 51). Our previous data (11) suggested that AS17 had a reduc- <br />   * Corresponding author. Mailing address: Department of Oral Bi-               tion in rather than a complete loss of Ffh expression. <br /> ology, University of Florida, Gainesville, FL 32610. Phone: (352) 846- <br /> 0784. Fax: (352) 392-7357. E-mail: bleiweis@dental.ufl.edu. <br />                                                                                    Further characterization of mutant AS17 revealed that it <br />   &#226;&#8364;&#160; Present address: Division of Immunological and Infectious Dis-              was incapable of an ATR (11). Furthermore, the H&#226;&#171;&#185;/ATPase <br /> eases, TNO-PG, 2301 CE Leiden, The Netherlands.                                 activity in membrane fractions of pH 5.0-shocked AS17 cells <br />  <br />                                                                          2543 <br />  2544      KREMER ET AL.                                                                                                                  J. BACTERIOL. <br />  <br />  <br />                                       TABLE 1. Bacterial strains and plasmids used in this study <br />                                                                                                                                           Source or <br />  Strain or plasmid                                              Description <br />                                                                                                                                           reference <br />  <br /> Strains <br />   E. coli <br />      TOP10           F&#226;&#171;&#186; mcrA &#226;&#338;&#172;(mrr-hsdRMA-mcrBC) &#226;&#65533;&#190;80lacZ&#226;&#338;&#172;M15 &#226;&#338;&#172;lacZ74 deoR recA1 araD139 &#226;&#338;&#172;(ara-leu)7697                               Invitrogen <br />                        galU galK rpsL (Strr) endA1 nupG <br />     INV110           F&#226;&#171;&#186; traD36 proAB&#226;&#171;&#185; lacIqZ&#226;&#338;&#172;M15/rpsL (Strr) thr leu endA thi-1 lacYgalK galTara tonA tsx dam dcm                    Invitrogen <br />                        supE44 &#226;&#338;&#172;(lac-proAB) &#226;&#338;&#172;(mcrC-mrr) 102::Tn10 (Tetr) <br />        INV&#226;&#65533;&#163;F&#226;&#172;&#732;        F&#226;&#171;&#186; endA1 recA1hsdR17 (rK&#226;&#171;&#186; mK&#226;&#171;&#185;) supE44 thi-1 gyrA96 relA1 &#226;&#65533;&#190;80lacZ&#226;&#338;&#172;M15 &#226;&#338;&#172;(lacZYA-argF)U169                         Invitrogen <br />  <br />   S. mutans <br />      NG8             Wild type                                                                                                       K. Knox <br />      MK4             ffh::Kanr; pMK14 in NG8                                                                                         This study <br />      BK110           satC::Kanr; pBK40 in NG8                                                                                        This study <br />      BK120           satD::Kanr; pBK41 in NG8                                                                                        This study <br />      BK130           satE::Kanr; pBK42 in NG8                                                                                        This study <br />      BK140           &#226;&#338;&#172;ylxM::Kanr; pBK45 in NG8                                                                                       This study <br />      BK150           gtfA::ylxM-ffh; pBK55 in NG8                                                                                    This study <br />      BK151           ffh::Kanr gtfA::ylxM-ffh; pMK14 in BK150                                                                        This study <br />      BK152           gtfA::pVA891; pVAgtfA in NG8                                                                                    This study <br />  <br />  <br /> Plasmids <br />   pALH124            aphA3 Kanr cassette in pALH123                                                                                  A. L. Honeyman <br />   pBAD-TOPO          Arabinose-inducible expression of His6 tag protein                                                              Invitrogen <br />   pCR2.1             PCR product A/T cloning vector                                                                                  Invitrogen <br />   pPC185             pUC18 derivative; Emr Apr                                                                                       This study <br />   pPC300             Psat-ylxM-ffh in pDL278                                                                                         11 <br />   PUC18              E. coli cloning vector                                                                                          Invitrogen <br />   pVA891             Streptococcal suicide vector                                                                                    30 <br />   pVAGTFA            gtfA in pVA891                                                                                                  26 <br />   pBK8               ffh in pBAD-TOPO                                                                                                This study <br />   pBK19              Psat-ylxM in pCR2.1                                                                                             This study <br />   pBK22              satD in pCR2.1                                                                                                  This study <br />   pBK23              satE in pCR2.1                                                                                                  This study <br />   pBK40              satC::Kanr in pPC185                                                                                            This study <br />   pBK41              satD::Kanr in pPC185                                                                                            This study <br />   pBK42              satE::Kanr in pPC185                                                                                            This study <br />   pBK43              5&#226;&#172;&#732; ylxM in pCR2.1                                                                                               This study <br />   pBK44              3&#226;&#172;&#732; ylxM in pCR2.1                                                                                               This study <br />   pBK45              &#226;&#338;&#172;ylxM::Kanr in pUC18                                                                                            This study <br />   pBK55              Psat-ylxM-ffh (pPC300-derived) in pVAGTFA                                                                       This study <br />   pMK2               satD in pPC185                                                                                                  This study <br />   pMK4               satC in pCR2.1                                                                                                  This study <br />   pMK6               satE in pPC185                                                                                                  This study <br />   pMK8               satC in pPC185                                                                                                  This study <br />   pMK10              ffh in pCR2.1                                                                                                   This study <br />   pMK11              ffh in pPC185                                                                                                   This study <br />   pMK14              ffh::Kanr in pPC185                                                                                             This study <br />  <br />  <br />  <br />  <br /> was comparable to that in pH 7.5-grown mutant cells, whereas           mutans sat operon consists of five open reading frames (ORFs) <br /> it was twofold lower than the pH 5.0-induced activity in wild-         and confirm the functional role of ffh in acid tolerance. <br /> type cell membranes (11). A lack of Ffh could result in an <br /> inability to rearrange the membrane in the way needed to                                       MATERIALS AND METHODS <br /> generate an ATR, which would include the incorporation of                Bacterial strains and culture conditions. The bacterial strains and plasmids <br /> additional H&#226;&#171;&#185;/ATPase polypeptides. However, the phenotypic             used in this study are listed in Table 1. S. mutans strains were grown routinely in <br /> effects observed could be the result of polar effects of the           Todd-Hewitt broth (BBL, Becton Dickinson Microbiology Systems, Cock- <br />                                                                        eysville, Md.) supplemented with 0.3% (wt/vol) yeast extract (THYE), aerobi- <br /> insertion. In the present study, we addressed this issue by            cally at 37&#194;&#176;C without agitation, or on THYE supplemented with 1.5% (wt/vol) <br /> characterizing the transcriptional organization of the sat             agar, at 37&#194;&#176;C in either candle jars or GasPak anaerobic chambers (BBL). When <br /> operon by Northern blotting and primer extension analysis.             required, antibiotics were used at the following concentrations: erythromycin, 10 <br /> Subsequently, we created nonpolar kanamycin insertions in the          &#226;&#65533;&#174;g/ml; and kanamycin, 500 &#226;&#65533;&#174;g/ml. <br />                                                                          Escherichia coli strains were cultured routinely aerobically at 37&#194;&#176;C with vigor- <br /> individual genes of the sat operon. Growth experiments were            ous shaking in Luria-Bertani (LB) broth or on LB agar (1.5% [wt/vol]) plates <br /> performed to assess the involvement of each of the sat operon          supplemented with the appropriate antibiotics: ampicillin, 100 &#226;&#65533;&#174;g/ml; kanamycin, <br /> genes in acid tolerance of S. mutans. Our data show that the S.        50 &#226;&#65533;&#174;g/ml; erythromycin, 300 &#226;&#65533;&#174;g/ml; or tetracycline, 15 &#226;&#65533;&#174;g/ml. <br />  VOL. 183, 2001                                                                             sat OPERON IN S. MUTANS ACID TOLERANCE                              2545 <br />  <br />  <br />                   TABLE 2. Primers used in this study                                   DNA sequencing was performed by the DNA Sequencing Core Laboratory of <br />                                                                                      the University of Florida&#226;&#8364;&#8482;s Interdisciplinary Center for Biotechnology Research. <br /> Primer                                                                               Sequence analysis was carried out with MacVector V6.5.3 (Oxford Molecular, <br />                                 Sequencea                             Positionb <br />   no. <br />                                                                                      Ltd., Madison, Wis.) and BLAST, version 2.0, software (1) available from the <br /> BK5         5&#226;&#172;&#732;-GCATTGAAATCCTACAGC-3&#226;&#172;&#732;                                   19&#226;&#8364;&#8220;36         National Center for Biotechnology Information (http://www.ncbi.nlm.nih.gov/). <br /> BK6         5&#226;&#172;&#732;-CGTCCAATTGAACTTGTC-3&#226;&#172;&#732;                                  294&#226;&#8364;&#8220;277           RNA isolation. Total RNA was isolated from S. mutans NG8 with the RNeasy <br /> BK13        5&#226;&#172;&#732;-ATGATTTACATCGCTATTATTGG-3&#226;&#172;&#732;                            3163&#226;&#8364;&#8220;3185       Mini or Midi kit (Qiagen), with several modifications. Overnight THYE cultures <br /> BK14        5&#226;&#172;&#732;-GATTAAACAGATAGCATAAAT-3&#226;&#172;&#732;                               735&#226;&#8364;&#8220;715c       were subcultured 1:10 into prewarmed THYE at pH 7.0. Cells were grown to <br /> BK15        5&#226;&#172;&#732;-AAGGTAGCTACGACCTTA-3&#226;&#172;&#732;                                 3781&#226;&#8364;&#8220;3798       mid-exponential phase (optical density at 650 nm of &#226;&#172;&#8225;0.5), pelleted by centrif- <br /> BK18        5&#226;&#172;&#732;-TTTTTTACGTTTTTTGGCTTT-3&#226;&#172;&#732;                              2516&#226;&#8364;&#8220;2496       ugation for 10 min at 4,000 &#226;&#171;&#187; g, and resuspended in fresh THYE at pH 7.0 or 5.0. <br /> BK23        5&#226;&#172;&#732;-GACGACTATTGATAATAGAGAC-3&#226;&#172;&#732;                              832&#226;&#8364;&#8220;853        After a 1-h incubation at 37&#194;&#176;C, bacteria were harvested by centrifugation for 10 <br /> BK25        5&#226;&#172;&#732;-GAGATCGCCAATAATAGCG-3&#226;&#172;&#732;                                3192&#226;&#8364;&#8220;3174       min at 4,000 &#226;&#171;&#187; g, washed once in ice-cold 10 mM Tris&#226;&#8364;&#8220;1 mM EDTA (TE; pH <br /> BK26        5&#226;&#172;&#732;-GACAGGATTACCTGACAAAAAC-3&#226;&#172;&#732;                             3875&#226;&#8364;&#8220;3854       7.0), and resuspended in TE containing lysozyme (5 mg/ml; Sigma Chemical Co., <br /> BK27        5&#226;&#172;&#732;-TTATCTGTCAAAAGCGCCGCAT-3&#226;&#172;&#732;                              591&#226;&#8364;&#8220;570        St. Louis, Mo.) and mutanolysin (200 U/ml; Sigma Chemical Co.). The bacteria <br /> BK28        5&#226;&#172;&#732;-CATTTGCTTATCTGTCAA-3&#226;&#172;&#732;                                  598&#226;&#8364;&#8220;581        were incubated for 10 min at room temperature before proceeding with the <br /> BK36        5&#226;&#172;&#732;-ggtaccATGGCTTTTGAAAGTTTA-3&#226;&#172;&#732;                            969&#226;&#8364;&#8220;986        Qiagen RNeasy kit protocol as provided by the manufacturer. <br /> BK39        5&#226;&#172;&#732;-GTTTTTGACTTTTAGGGCGA-3&#226;&#172;&#732;                               3235&#226;&#8364;&#8220;3216          Northern blot analysis. For Northern blot analysis, 10 &#226;&#65533;&#174;g of total RNA from <br /> PC12        5&#226;&#172;&#732;-ggatccGTTGTGGCTGCTTGGTC-3&#226;&#172;&#732;                             124&#226;&#8364;&#8220;140        both the pH 7.0- and pH 5.0-shocked S. mutans NG8 cultures was separated on <br /> PC18        5&#226;&#172;&#732;-ggatccATGGCTTTTGAAAGTTTA-3&#226;&#172;&#732;                            969&#226;&#8364;&#8220;986        a denaturing gel according to the Qiagen RNeasy kit protocol. Northern blotting <br /> PC19        5&#226;&#172;&#732;-ggtaccGGGAAGACCATTATAACG-3&#226;&#172;&#732;                           2595&#226;&#8364;&#8220;2578       and hybridization were performed according to the protocols provided with the <br /> PC20        5&#226;&#172;&#732;-GCAACATTTGCGGCTTCT-3&#226;&#172;&#732;                                 1669&#226;&#8364;&#8220;1652       Roche DIG system. DIG-labeled PCR probes were generated with the PCR DIG <br />    a <br />                                                                                      labeling kit (Roche) by using primers PC18 and PC20 for the ffh probe, BK15 and <br />      Sequence of primers in the sat locus based on S. mutans JH1005 sequence         BK14 for the satE probe, and BK5 and BK6 for the opuABC probe. The position <br /> (GenBank accession no. U88582), except for primer BK14, which was based on <br /> the UAB159 genomic sequence (http://www.genomic.ou.edu/smutans.html); low-           of the probes is indicated on the genomic map of the sat operon (Fig. 1). <br /> ercase nucleotides indicate added restriction sites.                                    Primer extension. To map the 5&#226;&#172;&#732; termini of sat operon mRNA transcripts, <br />    b <br />      Position of primers based on S. mutans JH1005 sequence (GenBank acces-          primer extension analysis was carried out with 2 pmol of [&#226;&#65533;&#165;-32P]ATP (3,000 <br /> sion no. U88582), unless noted otherwise.                                            Ci/mmol; Amersham Pharmacia Biotech, Inc., Piscataway, N.J.)-labeled BK28 or <br />    c <br />      The position indicated for BK14 is the position within the satE ORF, as based   BK39 on 15 &#226;&#65533;&#174;g of total RNA from pH 7.0- and pH 5.0-grown NG8 cells. After <br /> on the S. mutans UAB159 genomic sequence (http://www.genome.ou.edu                   denaturing for 10 min at 70&#194;&#176;C and subsequent cooling on ice, primer extension <br /> /smutans.html). <br />                                                                                      reactions were performed for 1 h at 42&#194;&#176;C with 200 U of Superscript II reverse <br />                                                                                      transcriptase (Gibco BRL, Grand Island, N.Y.). After heat inactivation, samples <br />                                                                                      were treated with 2 U of Rnase H (Gibco BRL) for 20 min at 37&#194;&#176;C. Following <br />    DNA manipulations. PCRs were performed routinely with the High Fidelity           phenol extraction and ethanol precipitation, the DNA was resuspended in dis- <br /> PCR kit (Roche Molecular Biochemicals, Indianapolis, Ind.) using custom prim-        tilled water and diluted 1:1 with Sequenase stop buffer (U.S. Biochemicals Corp. <br /> ers from Integrated DNA Laboratories, Inc. (Coralville, Iowa). All primers used      Cleveland, Ohio). A sequencing ladder was generated with unlabeled BK28 and <br /> in this study are listed in Table 2. Plasmid DNA preparations were obtained with     pBK19 as a template by using the Sequenase 7-deaza-dGTP DNA sequencing kit <br /> the Qiaprep Spin Miniprep kit (Qiagen, Inc., Valencia, Calif.). S mutans chro-       (U.S. Biochemicals Corp.) with [&#226;&#65533;&#163;-35S]dATP as a <span id='am-5' about='rdfs:label' typeof='owl:Thing'>label</span>, according to the manu- <br /> mosomal DNA was isolated as described previously (12). Natural transformation        facturer&#226;&#8364;&#8482;s protocol. The sequencing ladder and the primer extension reactions <br /> of S. mutans was performed as described previously (32). Transformed cells were      were run on an 8% denaturing polyacrylamide gel at 50 W for 3 h. The gel was <br /> spread on THYE agar supplemented with 10% (wt/vol) sucrose and the appro-            dried and exposed to X-ray film. <br /> priate antibiotics, and the plates were incubated anaerobically at 37&#194;&#176;C for 48 h.       Construction of nonpolar mutations in the sat operon genes. To create a <br />    Southern blot analyses were performed with digoxigenin (DIG)-labeled probes       nonpolar mutation in ffh, a PCR product of BK36 and PC19 first was cloned into <br /> according to the DIG system manufacturer&#226;&#8364;&#8482;s protocols (Roche). Probes were            pCR2.1 T/A TOPO cloning vector (Invitrogen, Carlsbad, Calif.), to give pMK10. <br /> generated by PCR with the PCR DIG labeling kit (Roche) according to the              The KpnI insert fragment was subsequently cloned into streptococcal suicide <br /> manufacturer&#226;&#8364;&#8482;s recommendations.                                                      vector pPC185, a pUC18 derivative in which the bla gene was replaced with an <br />  <br />  <br />  <br />  <br />   FIG. 1. Schematic representation of the sat operon locus in S. mutans. The sequences represented here that make up the sat operon were <br /> compiled from both our sequencing data (GenBank accession no. U88582) and the genome sequence of strain UAB159 (http://www.genome <br /> .ou.edu/smutans.html). Shaded arrows indicate ORFs; their designations are indicated above the sequence. Putative transcription start sites P1, P2, <br /> and P3, are indicated. The probes used in Northern blot experiments in this study are shown by bars below the sequence. sat operon mRNAs and <br /> their sizes as determined by Northern blot analysis are depicted below the sequence. <br />  2546        KREMER ET AL.                                                                                                                               J. BACTERIOL. <br />  <br />  <br /> erythromycin cassette derived from pResEM749 (39). The resulting construct              for 1 h at 100 V, the blots were incubated four times for 30 min at room <br /> (pMK11) was HindIII digested, dephosphorylated, and ligated with HindIII-               temperature in phosphate-buffered saline (PBS) containing 0.3% (vol/vol) <br /> digested promoterless aphA3 Kanr cassette, which had been isolated from                 Tween 20 (TPBS). The blots were incubated overnight at room temperature with <br /> pALH124 (kindly provided by A. L. Honeyman, University of South Florida,                antisera appropriately diluted in TPBS. Subsequently, they were washed four <br /> Tampa), to result in pMK14 (ffh::Kanr). The approach for the satC, satD, and            times with TPBS and incubated for 2 h with horseradish peroxidase-labeled goat <br /> satE insertions was similar to that used for the ffh insertion. PCR products were       anti-mouse immunoglobulin G (1:1,000 in TPBS; Southern Biotechnology As- <br /> generated with primers PC18 and BK25 for satC, BK13 and BK26 for satD, and              sociates, Inc., Birmingham, Ala.). After two washes with TPBS and two washes <br /> BK15 and BK14 for satE and were cloned into pCR2.1 to give pMK4, pBK22,                 with PBS, the antibody-binding antigens were visualized with 4-chloro-1-naph- <br /> and pBK23, respectively. EcoRI insert fragments were subsequently cloned in             thol and H2O2. <br /> pPC185, resulting in pMK8, pMK2, and pMK6, respectively. Thereafter, the                   Ffh antiserum production. To overproduce and purify Ffh, the ffh gene was <br /> PstI-digested Kanr cassette was cloned into the PstI site in satC in pMK8, the          amplified from the S. mutans NG8 genome with primers PC18 and BK18 and <br /> SphI-digested Kanr cassette was cloned into the SphI site in satD in pMK2, and          cloned into the pBAD-TOPO six-histidine tag (His6) expression vector (Invitro- <br /> the BamHI-digested Kanr cassette was cloned into the BclI site of satE in pMK6.         gen). The resulting construct was designated pBK8. Expression of Ffh (Ffh-V5- <br /> These constructs were designated pBK40 (satC::Kanr), pBK41 (satD::Kanr), and            His6) from this construct was analyzed by Western blotting of whole-cell extracts <br /> pBK42 (satE::Kanr).                                                                     with &#226;&#65533;&#163;-V5 antibody (1:5,000 dilution in TTBS; Invitrogen). Maximal induction of <br />    A different approach was taken to construct a kanamycin cassette insertion           expression was achieved by adding 0.2% (wt/vol) arabinose to mid-exponential- <br /> in ylxM. The promoter region and the 5&#226;&#172;&#732; 68 nucleotides (nt) of the ORF were am-         phase E. coli TOP10 cells harboring pBK8 and subsequent incubation for 4 h at <br /> plified with primers PC12 and BK27 and then cloned into pCR2.1 to give pBK43.           37&#194;&#176;C with vigorous shaking. Ffh-V5-His6 was semipurified with HisBind resin <br /> pBK44 contains a PCR fragment of the 3&#226;&#172;&#732; 22 nt plus 800 nt of downstream                 (Novagen, Madison, Wis.) according to the manufacturer&#226;&#8364;&#8482;s denaturing batch <br /> sequence that was amplified with primers BK23 and PC20. The inserts of pBK43            purification protocol. The Ffh-V5-His6 was separated on an SDS&#226;&#8364;&#8220;10% polyacryl- <br /> and pBK44 were isolated by digestion with BamHI-EcoRI and EcoRI-HindIII,                amide gel, the gel was stained with Coomassie brilliant blue R-250, and the 60 <br /> respectively. These fragments were ligated into BamHI-HindIII-digested-pUC18            kDa-protein band of interest was excised. BALB/c mice were immunized with the <br /> in the presence of the EcoRI-digested Kanr cassette, to generate a Kanr-inserted        isolated Ffh-V5-His6 gel slices followed by ascites production at the Hybridoma <br /> ylxM deletion construct, designated pBK45.                                              Core Laboratory of the University of Florida&#226;&#8364;&#8482;s Interdisciplinary Center for Bio- <br />    S. mutans strain NG8 was transformed with the Kanr insertion constructs after        technology Research. <br /> linearization by NarI. To verify the correct integration, Southern blotting was            Nucleotide sequence accession number. The updated nucleotide sequence of <br /> performed with chromosomal DNA isolated from Kanr Ems transformants, or                 the sat operon is available from the GenBank database under accession no. <br /> Kanr transformants in the case of &#226;&#338;&#172;ylxM::Kanr. The resulting mutant strains were        U88582. <br /> designated BK140 (&#226;&#338;&#172;ylxM::Kanr), MK4 (ffh::Kanr), BK110 (satC::Kanr), BK120 <br /> (satD::Kanr), and BK130 (satE::Kanr). <br />    Complementation of the ffh mutation. It was shown in previous work (11) that <br />                                                                                                                          RESULTS <br /> a mutation affecting ffh expression rendered the cell poorly competent for DNA <br /> uptake. In order to perform ffh complementation, we integrated another copy of             Genomic organization of the sat operon locus. In this study, <br /> ylxM-ffh, under the control of its own promoter, into the gtfA locus of the NG8         the nucleotide sequence downstream of the ylxM-ffh operon in <br /> chromosome and then created a mutation in the native ffh locus by insertion of          S. mutans strain JH1005 (GenBank accession no. U88582), <br /> the Kanr cassette as follows. Integration vector pVAGTFA was created by <br />                                                                                         obtained from the DNA fragment in pJG301 containing ylxM- <br /> insertion of a 2.5-kb gtfA fragment into PvuII-digested suicide vector pVA891 <br /> (26, 30). A 2.5-kb BamHI fragment, which contained the ylxM-ffh region, includ-         ffh followed by two complete ORFs and one partial putative <br /> ing its own putative promoters, was then isolated from pPC300 (11) and cloned           ORF, was assessed. These three new ORFs were designated <br /> into the BamHI site of pVAGTFA, to give pBK55. Single-crossover Emr mutants             satC, satD, and satE (of which only the 5&#226;&#172;&#732; end was represent- <br /> were isolated, and integration into the gtfA locus of NG8 was verified by South-        ed), with lengths of 525, 666, and 321 nt, respectively. Southern <br /> ern blotting. The resulting mutant, BK150, was subsequently transformed with <br /> NarI-linearized pMK14; DNA of the Kanr transformants was isolated and used <br />                                                                                         blotting confirmed that the genomic organization of the ylxM- <br /> in Southern blotting to verify the integration of the Kanr cassette into the original   satE region in strain NG8 was identical to that in JH1005 (data <br /> sat operon copy of ffh. The resulting complemented mutant was designated                not shown). The nucleotide sequence of the sat locus in the <br /> BK151. As a control in growth experiments, we used strain BK152, which has              S. mutans UAB159 genomic sequence (http://www.genome <br /> pVAGTFA without an insert integrated in the gtfA locus. <br />                                                                                         .ou.edu/smutans.html) was nearly identical (&#226;&#172;&#381;99%) to the <br />    Acid sensitivity. The effects of pH on the growth of the Kanr insertion mutants <br /> and complementation strains were evaluated in THYE media adjusted by addi-              available JH1005 sequence. A schematic of the genomic orga- <br /> tion of HCl to achieve pH 7.0, 6.0, 5.5, and 5.0. Sixteen-hour cultures of the          nization of the sat operon based on a compilation of the two <br /> strains were subcultured into THYE at 37&#194;&#176;C at the experimental pH levels and            available S. mutans sequences is shown in Fig. 1. <br /> grown to early exponential phase (Klett value of 50 U). The strains then were              Analysis of the genomic database sequence upstream of <br /> subcultured 1:4 into fresh medium at the experimental pH, and their growth at <br />                                                                                         ylxM revealed the presence of two ORFs encoding homologs of <br /> 37&#194;&#176;C was recorded with a Klett colorimeter (Klett-Summerson, New York, <br /> N.Y.). The Klett colorimeter provides a simple means of performing growth               Lactococcus lactis OpuAA (68% identity; GenBank accession <br /> curves on multiple small-volume (5 ml) cultures without removal of any of the           no. AF37878) and OpuABC (66% identity; GenBank acces- <br /> culture to take measurements. Since the Klett instrument scale is logarithmic,          sion no. AF37879.) These opuA genes <span id='am-4' about='Thesaurus:code' typeof='owl:Thing'>code</span> for the two sub- <br /> growth curves are plotted with linear axes (1 scale increment &#226;&#171;&#189; 1 U). Growth            units of a glycine betaine ABC transport system involved in <br /> experiments were repeated three times with duplicate samples. <br />    Western blot analysis. To produce whole-cell extracts of recombinant E. coli, <br />                                                                                         protection against osmotic stress conditions (48). The gene <br /> bacteria were harvested by centrifugation for 10 min at 4,000 &#226;&#171;&#187; g, resuspended          products of satC, satD, and satE have no homologs in the <br /> in TE, and diluted 1:1 in sample buffer (4% [wt/vol] sodium dodecyl sulfate             GenBank database (National Center for Biotechnology Infor- <br /> [SDS], 2% [vol/vol] 2-mercaptoethanol, 20% [vol/vol] glycerol, 125 mM Tris-HCl          mation). In addition, their amino acid sequences contain no <br /> [pH 6.8], 0.1 mg of bromophenol blue per ml). The samples were heated for 10 <br />                                                                                         known protein motifs. Downstream of satE, a homolog of <br /> min at 100&#194;&#176;C, and insoluble debris was removed by centrifugation for 10 min at <br /> 14,000 &#226;&#171;&#187; g. S. mutans whole-cell extracts were obtained by mechanical homog-            Bacillus subtilis RadC is encoded (43% identity; GenBank ac- <br /> enization of bacterial suspensions in TE with a Mini Beadbeater 8 (Biospec              cession no. Q02170). Proteins of the RadC family are involved <br /> Products, Inc., Bartlesville, Okla.) at maximum power with 0.1-mm-diameter              in DNA repair mechanisms (10). <br /> glass beads (Biospec). The resulting cell lysate was diluted 1:1 in sample buffer          Transcriptional organization of the sat operon. Northern <br /> and heated for 10 min at 100&#194;&#176;C. <br />    SDS-solubilized whole-cell proteins were separated on an SDS&#226;&#8364;&#8220;10% polyacryl- <br />                                                                                         blots on total RNA from mid-exponential-phase S. mutans <br /> amide gel and transferred to a nitrocellulose membrane (pore size of 0.45 &#226;&#65533;&#174;m;           NG8 cells that were exposed to pH 7.0 or 5.0 for 1 h following <br /> Schleicher &amp; Schuell, Inc., Keene, N.H.) by Western blotting (45). After blotting       incubation in pH 7.0 medium were performed to identify the <br />  VOL. 183, 2001                                                            sat OPERON IN S. MUTANS ACID TOLERANCE                   2547 <br />  <br />  <br />                                                                      constitutively expressed product mapped to the adenosine of <br />                                                                      the ylxM translation start codon (P2; Fig. 3B). This site is <br />                                                                      preceded by a putative &#226;&#171;&#186;10 promoter sequence (TATGGTA <br />                                                                      AAAT) that is similar to the consensus extended &#226;&#171;&#186;10 se- <br />                                                                      quence (34, 50). No consensus &#226;&#171;&#186;35 sequence was present up- <br />                                                                      stream of this transcription start site. Primer extension with <br />                                                                      primer BK39, which hybridizes within satD, resulted in one <br />                                                                      major product that was constitutively expressed (Fig. 3A). The <br />                                                                      5&#226;&#172;&#732; terminus of this product mapped to an adenosine residue 26 <br />                                                                      nt upstream of the satD gene (P3; Fig. 3C). Two hexameric <br />                                                                      sequences with similarities to consensus &#226;&#171;&#186;10 and &#226;&#171;&#186;35 pro- <br />                                                                      moter sequences were identified upstream of this putative <br />                                                                      transcription start site. <br />                                                                         Construction of nonpolar insertions in sat operon genes. To <br />   FIG. 2. Northern blot analysis of the sat operon. Ten micrograms   determine their function in acid tolerance, insertional muta- <br /> of total RNA from pH 7- or 5-shocked S. mutans NG8 cells was         tions were made in each of the individual genes of the sat <br /> hybridized with an ffh probe amplified with primers PC18 and PC20    operon with a nonpolar kanamycin cassette. This cassette con- <br /> (A), a satE probe amplified with primers BK14 and BK15 (B), and an <br />                                                                      tains a promoterless aphA3 kanamycin resistance gene with <br /> opuABC probe amplified with primers BK5 and BK6 (C). Arrows <br /> indicate major mRNA transcripts. RNA size markers are indicated to   upstream stop codons in all three reading frames and without <br /> the left of panel A.                                                 transcription termination sequences to allow transcription <br />                                                                      readthrough into downstream sequences (A. L. Honeyman, <br />                                                                      personal communication). In the cases of the ffh, satC, satD, <br /> mRNAs transcribed from the sat locus under these conditions.         and satE genes, the insertion was made in a unique restriction <br /> A DIG-labeled ffh probe hybridized with three mRNA species,          site within the gene. The Kanr cassette was inserted at the <br /> i.e., 4.2-, 2.7-, and 0.9-kb transcripts (Fig. 2A). The amount of    HindIII site at position 629 of ffh, the PstI site at position 396 <br /> the 4.2-kb transcript was significantly increased in pH 5.0-         in satC, the SphI site at position 322 in satD, and the BclI site <br /> shocked cells compared to that in pH 7.0-shocked cells,              at position 321 in satE. In the case of ylxM, no useful restriction <br /> whereas the 2.7-kb transcript appeared to be constitutively          sites were present within the ORF. Therefore, a 5&#226;&#172;&#732; fragment <br /> expressed. The amount of the 0.9-kb transcript was decreased         and a 3&#226;&#172;&#732; fragment of this gene were amplified and cloned with <br /> under pH 5.0 conditions. An identical hybridization <span id='am-399' about='xsp:pattern' typeof='owl:Thing'>pattern</span>          the Kanr cassette inserted between the PCR fragments. In this <br /> was observed with a ylxM probe (data not shown).                     construct (pBK45), the Kanr cassette replaces residues 69 to <br />    Based on our sequencing data and the available S. mutans          308 of the ylxM ORF. <br /> UAB159 genome sequence, probes homologous to the up-                    In all cases, transformation with the NarI-linearized Kanr <br /> stream opuABC sequence and the downstream satE sequence              insertion constructs resulted in viable mutations. Transfor- <br /> (Fig. 1) were generated. In addition to a constitutively ex-         mants were visible on THYES plates after 24 h under anaer- <br /> pressed 1.5-kb transcript, the satE probe recognized a 4.2-kb        obic conditions in all cases except for the ffh::Kanr mutation, in <br /> transcript found in greater amounts in pH 5-shocked cells than       which case colonies only became visible after 48 h. Southern <br /> in pH 7-shocked cells (Fig. 2B), as was the case with the ffh        blotting confirmed that integration in the chromosome had <br /> probe described above. Northern blots with total RNA isolated        occurred as schematically depicted in Fig. 4. <br /> from the ffh::Kanr and satC::Kanr mutants revealed the exis-            Effect of insertional inactivations on growth and acid toler- <br /> tence of a 3.5-kb transcript (0.8 kb being contributed by the        ance. The Kanr insertion mutants were grown in THYE at pH <br /> Kanr cassette) which hybridized only with the ylxM/ffh probes        5.0 to 7.0 to assess the importance of the sat operon genes in <br /> and not the satE probe (data not shown). These data confirm          S. mutans for normal growth and acid tolerance. None of the <br /> our conclusion that the 2.7-kb transcript in Fig. 2A is com-         mutations in the genes of the sat operon caused a complete <br /> posed of ylxM, ffh, and satC only and does not include satE. A       inability to grow in THYE at pH 7.0 (Fig. 5A). The wild-type <br /> fainter 2.7-kb band in Fig. 2B is therefore likely an artifact due   strain NG8 had a mean maximum growth rate of 55.4 Klett <br /> to its proximity to the 23S rRNA region. The opuABC probe            units h&#226;&#171;&#186;1 and a final Klett value of 167 &#226;&#171;&#190; 3 U. The ffh mutant <br /> hybridized with a 2.9-kb mRNA. This transcript also was              MK4 had a greatly reduced growth rate of 24.4 Klett units h&#226;&#171;&#186;1 <br /> strongly induced under pH 5.0 conditions (Fig. 2C).                  and a final Klett value of 135 &#226;&#171;&#190; 4 U. A mutation in ylxM <br />    Mapping of 5&#226;&#180;&#8226; termini of sat operon mRNA. Putative tran-          (BK140) had a small effect on the growth rate (47.5 Klett units <br /> scription start sites in the sat operon were identified by primer    h&#226;&#171;&#186;1) and yield (155 &#226;&#171;&#190; 3 U). The satC (BK110), satD (BK120), <br /> extension mapping of the 5&#226;&#172;&#732; mRNA termini. Two major prod-            and satE (BK130) mutants had growth rates and yields similar <br /> ucts were observed with primer BK28, which hybridizes within         to those of parent strain NG8. The same growth patterns were <br /> ylxM (Fig. 3A). The larger of the two was induced under acid         observed at pH 6.0 and 5.5, with gradually reduced growth <br /> conditions, whereas the smaller product was constitutively ex-       rates and yields for all strains compared to those at pH 7.0 <br /> pressed. The 5&#226;&#172;&#732; terminus of the larger acid-inducible product        (data not shown). At pH 5.0 (Fig. 5B), the growth rate of NG8 <br /> mapped to a guanosine residue 104 nt upstream of ylxM (P1;           was reduced to 15.8 Klett units h&#226;&#171;&#186;1, with a final Klett value of <br /> Fig. 3B). This putative transcription start site is preceded by      123 &#226;&#171;&#190; 7 U. The ffh mutant was incapable of growth at this pH. <br /> two hexameric sequences with similarities to consensus &#226;&#171;&#186;10           The growth rate and yield of BK140 were not significantly <br /> and &#226;&#171;&#186;35 promoter sequences. The 5&#226;&#172;&#732; terminus of the smaller           lower than those of the wild type: 15.0 Klett units h&#226;&#171;&#186;1 and <br />  2548     KREMER ET AL.                                                                                                             J. BACTERIOL. <br />  <br />  <br />  <br />  <br />   FIG. 3. Primer extension mapping of the 5&#226;&#172;&#732; end points of transcripts expressing sat operon segments. (A) Primer extension reactions were <br /> performed with 15 &#226;&#65533;&#174;g of total RNA from pH 5- or 7-shocked S. mutans NG8. Reaction mixtures were primed with ylxM primer BK28 or with satD <br /> primer BK39. Lanes C, T, A, and G contain a sequencing ladder primed by primer BK28 on pBK19 plasmid DNA. Arrows indicate 5&#226;&#172;&#732; termini P1, <br /> P2, and P3. Nucleotide sequence for products of primer BK28 is indicated to the left of the products. (B) Sequence upstream of ylxM showing the <br /> two putative transcription start sites P1 and P2 (asterisks) and their putative promoter sites (underlined). The N-terminal amino acid sequence of <br /> YlxM is indicated. A putative inverted repeat terminator sequence for the upstream opu operon is indicated by opposite underlining arrows. (C) <br /> Intergenic sequence between satC and satD showing the putative transcription start site P3 as determined with primer BK39. Promoter sequences, <br /> inverted repeats, and SatC and SatD amino acid sequences are indicated as for panel B. SD, Shine-Dalgarno consensus ribosome binding site. <br />  <br />  <br />  <br /> 118 &#226;&#171;&#190; 6 U, respectively. As at the other pHs, growth of BK110,             tein of the expected size, i.e., 57,000 Da, in whole-cell protein <br /> BK120, and BK130 was similar to that of NG8.                               extracts of S. mutans NG8 (Fig. 5C, lane 1). Therefore, this <br />    Complementation of ffh. Complementation of the ffh muta-                serum was used to verify the effects of the Kanr insertions in <br /> tion was performed to confirm that the acid-sensitive pheno-               the individual genes of the sat operon on Ffh protein expres- <br /> type of mutant MK4 was caused by the disruption of this gene.              sion. As shown in Fig. 5C, the ffh mutant MK4 (lane 3) did not <br /> We tried to complement this mutant directly by using pBK55;                express any full-<span id='am-6' about='xsp:length' typeof='owl:Thing'>length</span> Ffh. Moreover, the anti-Ffh antibodies <br /> however, as was shown previously with a Tn917 mutant (11), ffh             did not bind to smaller antigens representing truncated prod- <br /> mutants in S. mutans are not naturally competent. Therefore,               ucts of the insertionally inactivated ffh gene. In the other in- <br /> a second copy of ffh, including its putative promoter region and           sertion mutants (lanes 2 and 4 to 6), full-length Ffh was ex- <br /> ylxM, was integrated in the gtfA locus of NG8, before interrup-            pressed at levels similar to those in the wild type (lane 1). <br /> tion of the native ffh locus with construct pMK14, to generate                The anti-Ffh serum was also used to verify complementation <br /> complemented ffh-<span id='am-320' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-321' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-322' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-323' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-324' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-325' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-326' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-327' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-328' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-329' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-330' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-331' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-332' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-333' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-334' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-335' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-336' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-337' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-338' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-339' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-340' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-341' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-342' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-343' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-344' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-345' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-346' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-347' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-348' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-349' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-350' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-351' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-352' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-353' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-354' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-355' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-356' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-357' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-358' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-359' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-360' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-361' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-362' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-363' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-364' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-365' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-366' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-367' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-368' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-369' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-370' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-371' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-372' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-373' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-374' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-375' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-376' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-377' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-378' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-379' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-380' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-381' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-382' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-383' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-384' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-385' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-386' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-387' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-388' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-389' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-390' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-391' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-392' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-393' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-394' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-395' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-396' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-397' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span>knockout</span> mutant strain BK151.                             in BK151. As shown in Fig. 6C, integration of the ylxM-ffh <br />    In growth experiments, the intact second copy of ffh in the             region under the control of its own promoter into the gtfA <br /> gtfA locus in BK150 had no marked effect on growth rate and                locus (BK150 [lane 4]) did not affect Ffh expression. Subse- <br /> yield compared to that in wild-type cells (Fig. 6A and B). In              quent interruption of ffh in the sat locus with a Kanr cassette <br /> contrast to mutant MK4, mutant BK151, in which the ffh gene                (BK151 [lane 3]) did not result in the loss of Ffh expression, in <br /> of the sat locus in BK150 had been interrupted, grew at growth             contrast to MK4 (lane 2). These data indicate that comple- <br /> rates and final yields similar to those of wild-type NG8 at both           mentation of Ffh expression has been achieved in strain <br /> pH 7.0 (Fig. 6A) and 5.0 (Fig. 6B). These data indicate that a             BK151. As a control, the insertion of vector pVAGTFA by <br /> copy of ffh in the gtfA locus complemented the ffh mutation,               itself had no effect on Ffh expression (lane 5). <br /> causing growth defects in MK4. Alteration of the gtfA locus did <br /> not affect growth rate and yield, as shown by integration of                                          DISCUSSION <br /> vector pVAGTFA alone (BK152). <br />    Ffh protein in mutants. The calculated molecular mass of                  The ylxM-ffh operon is highly conserved in gram-positive <br /> Ffh is 56,963 Da. The mouse polyclonal antiserum raised                    eubacteria. This gene cluster was first reported in B. subtilis <br /> against the Ffh-V5-His6 protein specifically recognized a pro-             (19), where primer extension analysis implied that these genes <br />  VOL. 183, 2001                                                                sat OPERON IN S. MUTANS ACID TOLERANCE                     2549 <br />  <br />  <br />  <br />  <br />   FIG. 4. Nonpolar insertional inactivation of sat operon genes by a <br /> promoterless Kanr cassette. Schematic representation of the genomic <br /> organization in the wild-type NG8 and mutants BK140 (&#226;&#338;&#172;ylxM::Kanr), <br /> MK4 (ffh::Kanr), BK110 (satC::Kanr), BK120 (satD::Kanr), and BK130 <br /> (satE::Kanr). Restriction sites used for insertion in MK4, BK110, <br /> BK120, and BK130 are indicated. H, HindIII; P, PstI; S, SphI; B, BclI. <br />  <br />  <br /> form an operon. Members of our group previously reported <br /> the same organization in S. mutans (12). The ylxM-ffh gene <br /> cluster can also be found in several other gram-positive spe- <br /> cies, such as Streptococcus pneumoniae, Streptococcus pyogenes, <br /> Staphylococcus aureus, and Enterococcus faecalis (Microbial Ge- <br /> nome Blast Database; http://www.ncbi.nlm.nih.gov/), whereas <br /> the gram-negative species contain no ylxM homologs and thus <br /> lack the ylxM-ffh operon. In the Mycoplasma mycoides and <br /> Mycoplasma pneumoniae genomes, a ylxM homolog is located <br /> immediately downstream of ftsY, the gene encoding the Ffh <br /> docking protein (35). The tight linkage with the components of <br /> the SRP translocation pathway implies a role for YlxM in this <br /> pathway in gram-positive species. However, although YlxM <br /> has characteristics of a transcriptional regulator (35), the func- <br /> tion of this protein remains to be elucidated. <br />    In S. mutans, preliminary primer extension data suggested <br /> that ylxM and ffh are cotranscribed (13). In the present study, <br /> we determined that S. mutans ffh is transcribed in polycistronic <br /> mRNAs containing ylxM-ffh as well as putative ORFs down- <br /> stream of ffh (Fig. 1). A 4.2-kb transcript was composed of <br /> ylxM-ffh-satC-satD-satE, whereas a 2.7-kb mRNA contained                   FIG. 5. Effect of Kanr insertions on growth and Ffh expression. The <br /> the first three genes and a 1.5-kb mRNA contained the last two           growth capacities of mutants MK4, BK110, BK120, BK130, and BK140 <br /> genes. Putative terminator structures are found upstream of              were compared to that of wild-type strain NG8 in THYE adjusted to <br /> ylxM, between satC and satD, and downstream of satE and are              pH 7 (A) and 5 (B). The growth experiments were performed in <br /> consistent with the transcript sizes. The small 0.9-kb transcript        duplicate in three separate experiments. The results of a typical ex- <br />                                                                          periment are shown. (C) Ffh expression was determined by Western <br /> (Fig. 2A) could be the result of preliminary termination or              blot analysis with polyclonal mouse anti-Ffh serum. Fifty micrograms <br /> truncation of a larger transcript. The 5&#226;&#172;&#732; mRNA termini as                of SDS-soluble whole-cell protein was separated on an SDS&#226;&#8364;&#8220;10% <br /> determined by primer extension were consistent with the tran-            polyacrylamide gel and blotted onto nitrocellulose. The full-size Ffh <br /> scriptional organization. The three termini were located up-             protein band is indicated (arrow). Lanes: 1, wild-type strain NG8; 2, <br />                                                                          BK140; 3, MK4; 4, BK110; 5, BK120; 6, BK130. <br /> stream of ylxM (P1), at the adenosine of the translation start <br /> codon of ylxM (P2), and in the satC-satD intergenic region <br />  2550      KREMER ET AL.                                                                                                         J. BACTERIOL. <br />  <br />  <br />                                                                             (P3). Unlike the P1 and P3 termini, the P2 terminus results in <br />                                                                             a leaderless mRNA. Although rare, this type of transcript has <br />                                                                             been reported several times in other species, such as S. pneu- <br />                                                                             moniae (28, 34, 53), E. coli and streptomycetes (22). Although <br />                                                                             leaderless transcripts lack upstream Shine-Dalgarno ribosome <br />                                                                             binding sites, they are functional in translation. It has been <br />                                                                             suggested that a sequence downstream of the translation start <br />                                                                             site plays a role in enhancement of ribosome binding (38, 40). <br />                                                                             Recently, O&#226;&#8364;&#8482;Connor et al. (31) proved by <span id='am-86' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-87' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-100' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-101' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-161' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-163' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span>mutation analysis</span> of <br />                                                                             the proposed downstream ribosome binding site sequence that <br />                                                                             this region was not involved in the initiation of translation. The <br />                                                                             mechanism of translation of these transcripts, therefore, is still <br />                                                                             unclear. <br />                                                                                Transcription of ffh in S. mutans is amplified in response to <br />                                                                             acid stress (11). The data presented here are consistent with <br />                                                                             these findings. Although the amounts of the 2.7- and 1.5-kb <br />                                                                             mRNA appear to be constitutive, upregulation of the 4.2-kb <br />                                                                             full-length sat operon transcript could account for an increase <br />                                                                             in total ffh transcript (Fig. 2A and B). The upregulation of <br />                                                                             transcription appears to be driven by an acid-inducible pro- <br />                                                                             moter upstream of the P1 5&#226;&#172;&#732; terminus, whereas transcription <br />                                                                             starting at P2 and P3 is constitutive, as determined by primer <br />                                                                             extension (Fig. 3). Preliminary data from experiments employ- <br />                                                                             ing modified rapid amplification of cDNA ends confirmed that <br />                                                                             P2 and P3 are genuine transcription start sites with 5&#226;&#172;&#732; cap <br />                                                                             structures (data not shown). However, no further experimental <br />                                                                             evidence has been provided yet for the authenticity of P1 as a <br />                                                                             true transcription start site, suggesting this terminus could <br />                                                                             have been generated by truncation of a larger mRNA. This <br />                                                                             implies that the acid-inducible transcription is driven from a <br />                                                                             promoter further upstream, possibly by the opuA operon pro- <br />                                                                             moter. Induction of opuA promoter activity under pH 5 con- <br />                                                                             ditions is supported by our Northern blot data indicating up- <br />                                                                             regulation of a 2.9-kb transcript containing opuABC (Fig. 2C). <br />                                                                             Although transcription of the opuA genes in L. lactis and B. <br />                                                                             subtilis is regulated by environmental signals, especially os- <br />                                                                             motic stress, a link with acid stress has not been reported. A <br />                                                                             coordinated upregulation of transcription of both SRP trans- <br />                                                                             location and osmoprotection genes might be indicative of a <br />                                                                             general stress response induced by low pH in batch culture. <br />                                                                             Recent cross-protection studies in S. mutans revealed a slight <br />                                                                             cross-protective effect of osmotic stress adaptation on resis- <br />                                                                             tance to acid stress (43). However, acid adaptation did not <br />                                                                             render S. mutans resistant to osmotic stress. Further research <br />                                                                             is needed to reveal the nature of the linkage, if any, between <br />                                                                             the sat operon and the opuA operon, as well as their respective <br />                                                                             roles in acid tolerance of S. mutans. <br />                                                                                Induction of full-length mRNA upon acid stress could result <br />                                                                             in an increased protein translation of all of the sat operon <br />                                                                             ORFs, implying a possible role for all of these gene products in <br />    FIG. 6. Effect of complementation of ffh::Kanr into the gtfA locus       acid tolerance. In acid-sensitive mutant AS17, the transposon <br /> on growth and Ffh expression. Growth capacities of complementation          insertion in the ylxM-ffh intergenic region decreased ffh expres- <br /> strains BK150 (gtfA::ffh), BK151 (ffh::Kanr gtfA::ffh), and BK152 <br />                                                                             sion (11). However, polar effects on the transcription of down- <br /> (gtfA::pVAGTFA), were compared with those of mutant MK4 and <br /> wild-type strain NG8 in THYE adjusted to pH 7 (A) and 5 (B). The            stream genes cannot be ruled out, since satC, satD, and satE <br /> growth experiments were performed in duplicate in three separate            are cotranscribed with ylxM-ffh. Therefore, it was unclear <br /> experiments. The results of a typical experiment are shown. (C) Ffh         which of the sat operon genes was actually associated with the <br /> expression was determined by Western blot analysis with polyclonal          acid-sensitive phenotype, especially since the function of SatC, <br /> mouse anti-Ffh serum. Fifty micrograms of SDS-soluble whole-cell <br /> protein was separated on an SDS&#226;&#8364;&#8220;10% polyacrylamide gel and blotted          SatD, and SatE is unknown. Homologs of the overlapping <br /> onto nitrocellulose. The full-size Ffh protein band is indicated (arrow).   ORFs satD and satE are also found in S. pyogenes, but they <br /> Lanes: 1, wild-type strain NG8; 2, MK4; 3, BK151; 4, BK150; 5, BK152.       have no assigned function (Microbial Genomes Blast Data- <br />  VOL. 183, 2001                                                               sat OPERON IN S. MUTANS ACID TOLERANCE                                  2551 <br />  <br />  <br /> <span id='am-398' about='xsp:base' typeof='owl:Thing'>base</span>; http://www.ncbi.nlm.nih.gov/). In that species, these genes      barely been studied in the model organisms E. coli and B. <br /> are linked, but they are not located near ffh. Interestingly, satC     subtilis. The main SRP substrates in E. coli appear to be a <br /> has no homology to any known sequence or motifs. In the                subset of polytopic inner membrane proteins (46, 47). In B. <br /> present study, we provide evidence that only ffh is involved in        subtilis, the requirement of SRP for the localization of mem- <br /> acid tolerance. The ffh mutant was unable to initiate growth at        brane proteins has not been analyzed. However, Ffh depletion <br /> pH 5.0 (Fig. 5B). Complementation of the ffh mutant with an            inhibits the secretion of over 80% of extracellular proteins, as <br /> intact copy of the gene in the gtfA locus restored Ffh expression      determined by two-dimensional gel electrophoresis (18). To <br /> (Fig. 6C) and abolished any growth defects (Fig. 6A and B).            gain insight into the role of the SRP pathway in S. mutans, <br /> Mutations in satC, satD, or satE did not affect growth at acidic       analysis of the SRP-dependent proteins is of great importance. <br /> conditions, nor did they result in defects in Ffh expression (Fig.     Future research will therefore focus on the differential protein <br /> 5C). Finally, although its mutant was slightly affected in             patterns in membranes and culture supernatants of our ffh <br /> growth, ylxM expression is not essential for Ffh expression.           mutant and the wild-type strain, grown under identical condi- <br />    Although significantly affected in its growth, our ffh mutant       tions. <br /> was viable, whereas in other organisms, ffh knockout attempts <br /> failed, thereby suggesting that Ffh expression is essential for    <br /> viability of the cells (19, 33, 51). It can be argued that insertion   <br /> of a Kanr cassette at position 629 in ffh results in a truncated      <br /> but functional Ffh fragment containing the NH2-terminal 209      <br /> amino acid residues. However, based on homology to E. coli             <br />                                                              <br /> (23, 36) and B. subtilis Ffh (19, 25, 44), this putative truncated <br /> protein would only contain the first two GTP-binding domains                                             <br /> of the G domain. The domains essential for binding to both              <br /> hydrophobic signal sequences of transported proteins (44) and   <br /> small cytoplasmic RNA (25) are encompassed in the C-termi-             <br /> nal (residues 297 to 516) M domain (11). A BLAST search of                 <br /> the S. mutans genomic database, using the E. coli ffh sequence  <br /> as the query, identified some partial ORFs that showed statis-           <br /> tically significant sequence similarity to Ffh; however, they         <br /> appear to be related to the SRP receptor FtsY, the membrane         <br />                                                                       <br /> docking protein for Ffh. Although we cannot rule out the                    <br /> possibility that this protein can take over part of the function of        <br />                                                                 <br /> Ffh in ffh knockout mutants, it is unlikely, because this protein        <br /> lacks significant homology to the essential signal sequence-           <br /> binding domain of Ffh. Taking these results together, we con-     <br />                                                                            <br /> clude that MK4 is a genuine ffh mutant that expresses no                   <br /> functional Ffh.                                                       <br />    The underlying defects in the acid-sensitive phenotype of         <br />                                                                         <br /> MK4 remain to be elucidated. Tn917 mutant AS17 was unable                  <br /> to rapidly increase H&#226;&#171;&#185;/ATPase activity in the membrane upon            <br />                                                                          <br /> acid shock (11), as part of an overall acid tolerance response        <br /> (2, 3, 15). In other S. mutans strains, exposure to acid condi-        <br /> tions induces an upregulation of synthesis of at least 36 pro-         <br />                                                                          <br /> teins, albeit transient in nature for most of them (17). At               <br /> present, the identity and localization of these proteins are           <br /> unknown. The SRP pathway could be involved in the ATR in                 <br /> these strains by translocating the upregulated membrane and                 <br /> extracellular proteins. The strain (NG8) used in the present           <br /> study, however, did not display a classical ATR (data not                   <br /> shown). This strain is atypical in that preincubation at pH 5.0         <br /> to 5.5 does not reduce the number of cells killed by subsequent       <br />                                                                             <br /> incubation at pH 3.5. The genetic regulatory factors responsi-     <br /> ble for the ATR in S. mutans would appear, therefore, to be                <br /> distinct from the regulatory genes controlling expression of the    <br />                                                                         <br /> sat locus. Instead, as mentioned above, upregulation of the sat       <br /> operon may be part of a more generalized stress response              <br />                                                                     <br /> involving the osmoprotection (opuA) locus and possibly other         <br /> loci. Further studies will explore the genetic basis of this gen-     <br /> eralized stress response.                                          <br />                                                                           <br />    In contrast to the mechanism of translocation, the repertoire       <br /> of proteins that are translocated by the SRP pathway has                <br /> </body></html>