<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>         Hybrid Protein between Ribosomal Protein S16 and RimM of <br />               Escherichia coli Retains the Ribosome Maturation <br />                           Function of Both Proteins <br />  <br />             The RimM protein in Escherichia coli is associated with free 30S ribosomal subunits but not with 70S <br />          ribosomes and is important for efficient maturation of the 30S subunits. A mutant lacking RimM shows a <br />          sevenfold-reduced growth rate and a reduced translational efficiency. Here we show that a double alanine-for- <br />          tyrosine substitution in RimM prevents it <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> associating with the 30S subunits and reduces the growth rate <br />          of E. coli approximately threefold. Several faster-growing derivatives of the rimM amino acid substitution <br />          mutant were found that contain suppressor mutations which increased the amount of the RimM protein by two <br />          different mechanisms. Most of the suppressor mutations destabilized a secondary structure in the rimM <br />          mRNA, which previously was shown <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> decrease the synthesis of RimM by preventing the access of the <br />          ribosomes to the translation initiation region on the rimM mRNA. Three other independently isolated sup- <br />          pressor mutations created a fusion between rpsP, encoding the ribosomal protein S16, and rimM on the <br />          chromosome as a result of mutations in the rpsP stop codon preceding rimM. A severalfold-higher amount of <br />          the produced hybrid S16-RimM protein in the suppressor strains than of the native-sized RimM in the original <br />          substitution mutant seems to explain the suppression. The S16-RimM protein but not any native-size ribo- <br />          somal protein S16 was found both in free 30S ribosomal subunits and in translationally active 70S ribosomes <br />          of the suppressor strains. This suggests that the hybrid protein can substitute for S16, which is an essential <br />          protein probably because of its role in ribosome assembly. Thus, the S16-RimM hybrid protein seems capable <br />          of carrying out the important functions that native S16 and RimM have in ribosome biogenesis. <br />  <br />  <br />    The RimM protein is encoded by the second gene of the                    of S16 than of RimM is explained by translational level regu- <br /> trmD operon, which also encodes the ribosomal proteins                      lation (6, 16) through a large mRNA hairpin structure, which <br /> (r-proteins) S16 and L19 and the tRNA(m1G37)methyltrans-                    involves <span id='am-82' about='xsp:base' typeof='owl:Thing'>base</span> paring between the translation initiation region <br /> ferase (TrmD) (5). The RimM protein is important for matu-                  of rimM and sequences approximately 100 nucleotides down- <br /> ration of the 30S ribosomal subunits (4), and it is associated              stream from the rimM start codon (17, 19). This mRNA hair- <br /> with free 30S subunits but not with those in the 70S ribosomes              pin structure prevents access of the ribosomes to the transla- <br /> (3). The amount of RimM in the cell is approximately 12-fold                tion initiation region and thereby reduces the translational <br /> smaller than that of r-proteins (16). These findings suggest that           initiation of rimM. <br /> RimM interacts transiently with the 30S ribosomal subunits                     Here we have found that a RimM protein that contains <br /> during the maturation process. Mutants lacking the RimM                     alanine substitutions for two adjacent conserved tyrosines has <br /> protein show a sevenfold-decreased growth rate and a reduced                a dramatically reduced ability to associate with the 30S ribo- <br /> translational efficiency (3), probably due to incorrectly ma-               somal subunits. Suppressor mutations that increased the growth <br /> tured 30S subunits. The slow growth and translational defi-                 rate of this rimM mutant either increased the synthesis of the <br /> ciency of a &#226;&#338;&#172;rimM mutant can be partially suppressed by al-                 mutant RimM protein up to 10-fold or fused the rpsP gene to <br /> terations in the C-terminal part of r-protein S13, which binds              rimM. The amount of the mutant RimM protein associated <br /> 16S rRNA (3). Increased expression of the ribosome binding                  with the 30S ribosomal subunits in the RimM-overproducing <br /> factor RbfA, which also is important for the maturation of the              strains was similar to that of RimM in wild-type strains. The <br /> 30S ribosomal subunits, also suppresses the translational defi-             hybrid S16-RimM protein in the other suppressor strains was <br /> ciency of a &#226;&#338;&#172;rimM mutant (4). The r-protein S16 is essential for            found both in free 30S ribosomal subunits and in translation- <br /> the viability of Escherichia coli (12) and plays an important role          ally active 70S ribosomes. Since no native-sized S16 was de- <br /> in the assembly of the 30S ribosomal subunits (7). However, in              tected in the ribosomes, this suggests that the hybrid protein <br /> vitro-assembled 30S subunits lacking S16 show functional                    can substitute for both RimM and S16. <br /> properties similar to those of 30S subunits that contain S16, <br /> suggesting that S16 is not directly involved in translation (7). <br /> Further, S16 also binds to cruciform DNA and shows DNA-                                             MATERIALS AND METHODS <br /> nicking activity (2, 11). The observation of 12-fold-higher levels <br />                                                                               Bacterial strains. The bacterial strains used are listed in Table 1. <br />                                                                               Isolation of suppressor mutations. To isolate faster-growing derivatives of the <br />                                                                             rimM120 mutant MW136 (Y106A plus Y107A), several independent colonies <br />   * Corresponding author. Mailing address: Department of Molecular          were grown overnight in Luria broth (1). An aliquot of each overnight culture <br /> Biology, Ume&#195;&#165; University, SE-901 87 Ume&#195;&#165;, Sweden. Phone: 46-90-             was streaked out on rich medium plates, in order to detect revertants, simulta- <br /> 7856754. Fax: 46-90-772630. E-mail: mikael.wikstrom@micro.umu.se.           neously with its reinoculation into fresh medium. This process was repeated until <br />  <br />                                                                      5352 <br />  VOL. 183, 2001                                                S16-RimM PROTEIN RETAINS RIBOSOME MATURATION FUNCTIONS                                   5353 <br />  <br />  <br />                          TABLE 1. Bacterial strains                                  deficiency in the maturation of the 30S subunits evidenced by <br />                                                                         Origin or <br />                                                                                      a reduced processing rate of pre-16S to 16S rRNA (J. M. <br />  Strain                      Genotype (<span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-61' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-73' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span>mutation</span>[s])                                  Lo&#194;&#168;vgren, G. O. Bylund, L. A. C. Lundberg, O. P. Persson, and <br />                                                                        referencea <br />                                                                                      P. M. Wikstro  &#194;&#168;m, unpublished results). Since the wild-type <br /> JML002          MW136 rimM125 (AGC to AAC in codon 2) <br /> JML020          MW136 rpsP877 (TAA to CAA)                                           RimM protein is associated with free 30S ribosomal subunits <br /> JML021          MW136 rpsP878 (TAA to GAA)                                           (3), we examined whether the two amino acid substitutions in <br /> JML022          MW136 rimM128 (AGC to AAC in codon 2)                                the mutant RimM protein affected its ability to associate with <br /> JML023          MW136 rimM129 (ATG to ATT in codon 1) <br /> JML024          MW136 rimM130 (CTC to CTG in codon 29)                               the 30S subunits. The ribosomal subunits in cell extracts from <br /> JML027          MW136 rpsP879 (TAA to TTA)                                           mutant and wild-type strains were separated by sucrose gradi- <br /> JML028          MW136 rimM132 (CTC to CTT in codon 29)                               ent centrifugation under conditions that dissociated the 70S <br /> JML029          MW136 rimM133 (ATG to ATT in codon 1) <br /> MW100           Hfr P4X                                                 18 <br />                                                                                      ribosomes into 50S and 30S subunits, and different fractions <br /> MW136           Hfr P4X rimM120 (TAC to GCT in codon                    Lo <br />                                                                          &#194;&#168;vgren      were screened for the presence of the RimM protein by using <br />                  106 and TAC to GCG in codon 107)                         et al.b    an anti-RimM antiserum. The wild-type RimM protein mole- <br /> MW144           MW136 rimM124 (GGG to GGA in codon                                   cules were found associated with free 30S subunits and also in <br />                  27) <br /> PW109           Hfr P4X &#226;&#338;&#172;rimM102 yfiB::nptI sdr-43 &#226;&#171;&#189;                    4            the cytoplasmic fractions, whereas the mutant RimM protein <br />                  DUP(&#226;&#172;&#732;infB to yhbM&#226;&#172;&#732;)                                                 was found mainly in the cytoplasmic fractions (cf. Fig. 2A and <br />   a <br />       Unless otherwise noted, the origin was this study. <br />                                                                                      B). Thus, the reduced growth rate of the rimM120 mutant and <br />   b <br />       Unpublished results.                                                           the deficiency in the maturation of the 30S subunits correlate <br />                                                                                      with an inability of the mutant RimM protein to interact with <br /> revertants appeared on the plates. Only one revertant was saved from each <br /> original culture for further analyses. <br />                                                                                      the 30S ribosomal subunits. <br />    PCR amplification of chromosomal DNA and DNA sequencing. Regions of the              In the process of isolating second-site suppressor mutations <br /> E. coli chromosome were amplified by PCR (9, 15) from colonies resuspended in        that potentially would restore the binding of the mutant RimM <br /> H2O using Taq DNA polymerase from Roche Diagnostics Scandinavia AB                   protein to the 30S subunits, faster-growing derivatives of strain <br /> (Bromma, Sweden). Obtained fragments were purified using Quantum Prep <br />                                                                                      MW136 (rimM120) were isolated (Fig. 1) that contained mu- <br /> PCR Kleen Spin Columns from Bio-Rad Laboratories (Hercules, Calif.). DNA <br /> sequencing of PCR fragments was carried out with a Thermo Sequenase II dye           tations which were tightly linked to the rimM120 mutation, <br /> terminator cycle sequencing premix kit from Amersham Pharmacia Biotech               as demonstrated by the ability to transfer these mutations <br /> (Buckinghamshire, England), using an ABI 377 XL DNA Sequencer from PE                together with the rimM120 mutation to the wild-type strain <br /> Applied Biosystems (Stockholm, Sweden).                                              MW100 by the phage P1 (data not shown). Several of these <br />    Western blot analysis with an anti-RimM antiserum after sucrose gradient <br /> centrifugation of cellular extracts. Extracts containing dissociated 70S ribosomes <br />                                                                                      suppressor mutations were in the first 29 codons of rimM, <br /> were prepared by disrupting log phase cells in a solution containing 10 mM           some of which did not alter the amino acid sequence of RimM. <br /> Tris-HCl (pH 7.5), 50 mM KCl, 6 mM &#226;&#65533;&#164;-mercaptoethanol, and 1 mM MgCl2                 All of these mutations destabilized a hairpin structure in the <br /> using a French press set at 15 MPa. The S30 extracts obtained after centrifuga-      mRNA (Fig. 3) that previously has been shown to decrease <br /> tion were fractionated by sucrose gradient centrifugation as described previously <br /> (8). Polysome extracts were prepared by freeze-thawing log phase cells in the <br /> presence of lysozyme according to the method of Ron et al. (14) and fractionated <br /> by sucrose gradient centrifugation mainly as described by Powers and Noller <br /> (13). Aliquots from the obtained fractions were analyzed spectrophotometrically <br /> at 260 nm. Selected fractions were subjected to Western blot analysis using the <br /> ECL kit from Amersham Pharmacia Biotech with antisera against the RimM <br /> protein raised in rabbits by Agri Sera AB (Va&#194;&#168;nna&#194;&#168;s, Sweden) using the RimM part <br /> of a thrombin-cleaved glutathione S-transferase&#226;&#8364;&#8220;RimM hybrid protein (data not <br /> shown). <br />    Immunoprecipitation of the r-protein S16 and the S16-RimM hybrid protein <br /> using an anti-S16 antiserum. Strains JML020 (rimM120 rpsP877) and JML024 <br /> (rimM120 rimM130) were grown in morpholinepropanesulfonic acid minimal <br /> medium (10) containing 0.4% glucose, and at an optical density at 600 nm of <br /> 0.03; 15 ml of each culture was labeled with 325 &#226;&#65533;&#174;Ci of [35S]methionine for four <br /> generations. Cells were harvested by centrifugation, washed once in a solution <br /> containing 10 mM Tris-HCl (pH 7.5), 50 mM KCl, 10 mM MgCl2, and 6 mM <br /> &#226;&#65533;&#164;-mercaptoethanol, and disrupted by sonication. 70S ribosomes and free 30S <br /> subunits were purified by sucrose gradient centrifugation, and selected fractions <br /> were pooled. The sucrose was removed using Centricon YM-3 centrifugal filter <br /> devices from Millipore Corporation (Bedford, Mass.). r-proteins were extracted <br /> using acetic acid, precipitated with acetone, solubilized in 8 M urea, and immu- <br /> noprecipitated with the immunoglobulin G fraction of an anti-S16 antiserum <br /> coupled to Sepharose as described previously (18). <br />  <br />  <br />                                    RESULTS <br />    The weak association of a mutant RimM protein with the <br /> 30S ribosomal subunits can be compensated for by increased                              FIG. 1. Relative growth rates of wild-type and different mutant <br /> levels of the mutant protein. The rimM120 mutant MW136,                              strains. The strains were grown in Luria broth (1), and their growth <br />                                                                                      rates were normalized to that for the wild-type strain, MW100, which <br /> which contains alanine substitutions in two of the most con-                         had a specific growth rate k (ln2/g, where g is the mass doubling time <br /> served positions (Y106A and Y107A) of RimM, shows a three-                           in hours) of 1.3 to 1.6 in three independent experiments. The variation <br /> fold-lower growth rate than a rimM&#226;&#171;&#185; strain (Fig. 1) and a                            between the experiments is shown as error bars. <br />  5354       &#194;&#168; VGREN AND WIKSTRO <br />           LO                 &#194;&#168;M                                                                                                   J. BACTERIOL. <br />  <br />  <br />  <br />  <br />                                                                              FIG. 3. Suppressor mutations in the lower part of the mRNA sec- <br />                                                                           ondary structure that prevents access of the ribosomes to the transla- <br />                                                                           tional initiation region of rimM (19). The numbering is relative to the <br />                                                                           transcriptional start site of the trmD operon mRNA (6). The rpsP stop <br />                                                                           codon, SD sequence, and start codon for rimM are shaded. <br />  <br />  <br />                                                                           of the protein. This interpretation is supported by the finding <br />                                                                           that the amount of the mutant protein associated with the <br />                                                                           30S ribosomal subunits is higher in the suppressor mutant <br />                                                                           JML024 (rimM120 rimM130) overexpressing RimM than in the <br />                                                                           rimM120 mutant MW136 (cf. Fig. 2C and B). <br />  <br />  <br />  <br />  <br />   FIG. 2. Cellular localization of RimM proteins in the wild type (A), <br /> rimM120 mutant (B), and rimM120 mutant overexpressing RimM (C). <br /> Cell extracts were fractionated by sucrose gradient centrifugation dur- <br /> ing conditions that dissociated the 70S ribosomes into 50S and 30S <br /> subunits. Selected fractions, indicated by arrows above the A260 curve, <br /> were screened for the presence of the RimM proteins in Western <br /> blotting experiments using a polyclonal anti-RimM antiserum. The <br /> locations of the 50S and 30S ribosomal subunits are indicated below <br /> the A260 curve. <br />  <br />  <br /> translation initiation of rimM, probably by preventing the ac- <br /> cess of the ribosomes to the Shine-Dalgarno (SD) sequence                    FIG. 4. Amounts of RimM proteins in different strains. The amounts <br /> and start codon (17, 19). Accordingly, four out of four of these          of the wild-type RimM and rimM120 mutant proteins in total cell <br /> mutations that were tested for their effect on the expression of          extracts of strains MW100 (wt), MW136 (rimM120), JML002 (rimM120 <br />                                                                           rimM125), JML023 (rimM120 rimM129), JML024 (rimM120 rimM130), <br /> rimM120 were found to increase the levels of the mutant                   JML028 (rimM120 rimM132), and PW109 (&#226;&#338;&#172;rimM102 sdr-43) were <br /> RimM protein (Fig. 4). These results suggest that overexpres-             determined by using an anti-RimM antiserum in a Western blotting <br /> sion of the mutant protein compensates for a reduced function             experiment. <br />  VOL. 183, 2001                                        S16-RimM PROTEIN RETAINS RIBOSOME MATURATION FUNCTIONS                                  5355 <br />  <br />  <br />                                                                           native-sized RimM and no detectable amounts of any degra- <br />                                                                           dation products of the hybrid protein (Fig. 6). Thus, these <br />                                                                           findings suggest that the hybrid S16-RimM protein is respon- <br />                                                                           sible for the observed suppression. <br />                                                                              The S16-RimM hybrid protein replaces S16 in mature ribo- <br />                                                                           somes. In the screening of the cell extract of strain JML020 <br />                                                                           fractionated by sucrose gradient centrifugation, the S16-RimM <br />                                                                           protein was found not only in the 30S subunits but also in 70S <br />                                                                           ribosomes and polysomes (Fig. 6). These results suggested that <br />                                                                           the S16-RimM hybrid protein had replaced the r-protein S16 <br />                                                                           in translationally active 70S ribosomes; however, it was for- <br />                                                                           mally possible that native-size S16 was present in the majority <br />                                                                           of the ribosomes due to proteolytic cleavage of the hybrid <br />                                                                           protein. Therefore, we labeled strains JML020 and JML024 in <br />                                                                           vivo with [35S]methionine and probed for the r-protein S16 and <br />                                                                           the S16-RimM hybrid protein by immunoprecipitation with an <br />                                                                           anti-S16 antiserum in fractions corresponding to 30S ribosomal <br />   FIG. 5. Production of an S16-RimM hybrid protein in the suppres-        subunits and 70S ribosomes after separation of cellular ex- <br /> sor strain JML020 containing a mutation in the rpsP stop codon. The       tracts by sucrose gradient centrifugation. No native-size S16 <br /> presence of the S16-RimM hybrid protein and native-sized RimM             was found in the 30S subunits or in the 70S ribosomes from <br /> proteins was screened for in total cell extracts of strains MW100 (wt),   strain JML020 when they were immunoprecipitated with the <br /> MW136 (rimM120), PW109 (&#226;&#338;&#172;rimM102 sdr-43), and JML020 (rimM120 <br />                                                                           anti-S16 antiserum; however, in agreement with the results <br /> rpsP877) by using an anti-RimM antiserum in a Western blotting <br /> experiment.                                                               from the Western blotting experiments with the anti-RimM <br />                                                                           antiserum, the S16-RimM hybrid protein was found in both the <br />                                                                           30S subunits and the 70S ribosomes (Fig. 7). In the control <br />    An S16-RimM hybrid protein suppresses the slow growth of               strain JML024, native S16 was found in both the 30S subunits <br /> the rimM120 mutant. Surprisingly, three of the chromosomal <br /> suppressor mutations (carried by strains JML020, JML021, <br /> and JML027) changed the stop codon of rpsP preceding rimM <br /> to different sense codons (Fig. 3). These three strains were <br /> among the fastest-growing suppressor strains isolated, having <br /> growth rates that corresponded to approximately 85% of that <br /> of a rimM&#226;&#171;&#185; strain (exemplified by strain JML020 in Fig. 1). <br /> Since rpsP and rimM are in the same reading frame and there <br /> are no additional stop codons between the two genes, the two <br /> genes had been fused in the three suppressor strains. As shown <br /> in Fig. 5, a hybrid S16-RimM protein of the expected size was <br /> produced in large amounts with strain JML020, as demon- <br /> strated by using an anti-RimM antiserum in Western blotting <br /> experiments. Similar results were obtained for the two other <br /> strains, JML021 and JML027, that contain mutations in the <br /> rpsP stop codon (data not shown). Also, a protein of the ex- <br /> pected size for native RimM protein reacted with the antibod- <br /> ies. At this point we cannot distinguish whether this protein <br /> was the result of initiation at the rimM start codon or was a <br /> degradation product of the hybrid protein. We would like to <br /> emphasize that the amount of native-size RimM protein in <br /> the suppressor strain JML020 was not larger than that in the <br /> rimM120 mutant MW136 (Fig. 5), demonstrating that the <br /> mechanism behind the suppression was not an increased level <br /> of the native-sized RimM protein. In the total extract of strain <br /> JML020, there seemed to be degradation products of the S16- <br /> RimM hybrid protein (Fig. 5). To investigate whether the hy- <br />                                                                              FIG. 6. Cellular localization of the S16-RimM hybrid protein. A <br /> brid protein or any of the proposed degradation products were             polysome extract of strain JML020 (rimM120 rpsP877) was fraction- <br /> responsible for the observed suppression, a cellular extract of           ated by sucrose gradient centrifugation, and the indicated fractions <br /> strain JML020 was fractionated by sucrose gradient centrif-               were screened for the presence of the S16-RimM hybrid protein by us- <br /> ugation and different fractions were screened with an anti-               ing an anti-RimM antiserum in a Western blotting experiment. The <br />                                                                           locations of the 50S and 30S ribosomal subunits, 70S ribosomes, and poly- <br /> RimM antiserum in a Western blotting experiment. The                      somes are indicated below the A260 curve. The identity of native-sized <br /> fractions corresponding to the 30S subunits contained large               RimM protein was determined by running a total extract of the wild- <br /> amounts of the S16-RimM protein but very small amounts of                 type strain MW100 and a molecular weight marker on the protein gel. <br />  5356      &#194;&#168; VGREN AND WIKSTRO <br />          LO                 &#194;&#168;M                                                                                                   J. BACTERIOL. <br />  <br />  <br />                                                                          lated hairpin structure increased translation initiation of rimM- <br />                                                                          lacZ fusions (17, 19). Similarly, the suppressor mutations to <br />                                                                          rimM120 that increased the amount of the mutant RimM pro- <br />                                                                          tein decreased the calculated stability of this hairpin structure <br />                                                                          (Fig. 3). Thus, the effect of these suppressor mutations on the <br />                                                                          expression of RimM corroborates the previously proposed <br />                                                                          model in which the mRNA secondary structure prevents the <br />                                                                          access of the ribosomes to the SD sequence and start codon of <br />                                                                          rimM. Further, the suppressor mutations of strains JML023 <br />                                                                          and JML029 changed the AUG start codon of rimM to AUU, <br />                                                                          encoding isoleucine. Probably, the AUG codon just upstream <br />                                                                          from this AUU codon is used as a start codon in these two <br />                                                                          strains. <br />                                                                             Interestingly, three of the isolated suppressor mutations <br />                                                                          changed the stop codon of rpsP (for S16) to sense codons, <br />                                                                          which resulted in the production of a hybrid S16-RimM pro- <br />                                                                          tein. This protein, which was found in large amounts in the cell, <br />                                                                          seems to be responsible for the suppression of the slow growth <br />                                                                          of the cells, implied to be caused by the weak association of the <br />                                                                          mutant RimM protein with the 30S ribosomal subunits. The <br />                                                                          hybrid S16-RimM protein was present in large amounts both in <br />                                                                          free 30S subunits and in translationally active 70S ribosomes, <br />   FIG. 7. Analysis of ribosomes for the presence of the r-protein S16.   probably because of the affinity of the S16 moiety of the hybrid <br /> Acid-soluble r-proteins were subjected to immunoprecipitation with an    protein for the 30S subunits, since RimM is not normally <br /> anti-S16 antiserum after purification of 70S ribosomes and free 30S <br /> subunits from total cell extracts of strains JML020 (rimM120 rpsP877)    present in 70S ribosomes (3). Thus, the ability of the S16- <br /> and JML024 (rimM120 rimM130) labeled in vivo with [35S]methionine        RimM protein to substitute for RimM in ribosome maturation <br /> (see Materials and Methods). The immunoprecipitated proteins were        might depend on the ability of the S16 moiety to guide the <br /> separated on a 10-to-17.5% gradient polyacrylamide gel containing        hybrid protein to the 30S subunits. Alternatively, the suppres- <br /> sodium dodecyl sulfate. <br />                                                                          sion is dependent on RimM-mediated binding of the S16- <br />                                                                          RimM protein to the 30S subunits and is explained by the high <br /> and the 70S ribosomes. The lower intensity of the band corre-            levels in the cell of the hybrid protein, since other suppressor <br /> sponding to S16 in the control strain JML024 than that of the            mutations that increased the amount of the mutant RimM <br /> S16-RimM hybrid protein in strain JML020 results from the                protein restored the level of RimM associated with the 30S <br /> fact that S16 contains only one methionine, whereas the hybrid           subunits to that seen in a wild-type strain (Fig. 2). We favor the <br /> protein is supposed to contain eight. Taken together, these              latter model, since alterations in r-protein S13 suppress the <br /> results suggest that in addition to being able to substitute for <br /> RimM, the hybrid S16-RimM protein also can substitute for <br /> S16 in translationally active 70S ribosomes. <br />  <br />                           DISCUSSION <br />  <br />    Here we show that the RimM protein in the rimM120 mu- <br /> tant, in which alanines have been substituted for the tyrosines <br /> in positions 106 and 107, is only weakly associated with the 30S <br /> ribosomal subunits. The reduced amount of RimM associated <br /> with the 30S subunits most likely explains the slow growth rate <br /> of the rimM120 mutant, since RimM is important for the mat- <br /> uration of the 30S subunits (4). Several suppressor mutations <br /> to the rimM120 mutation increased the amount of the mutant <br /> protein, and for at least one of the suppressor strains, the <br /> amount of the mutant RimM protein associated with the 30S <br /> subunits was comparable to that of the native RimM protein in <br /> wild-type cells. Conceivably, an increased concentration of the <br /> mutant RimM protein in the cell can compensate for its weak                 FIG. 8. Location of the COOH-terminal end of the r-protein S16 in <br /> association with the 30S ribosomal subunits.                             the 30S subunit of T. thermophilus. The structure of the 30S subunit is <br />    An mRNA secondary structure that covers the translation               from reference 20 and was retrieved from the Protein Data Bank (PDB <br /> initiation region for rimM seems to involve base pairing of the          no. 1FJF). Only a part of the structure is shown, in which the 16S <br />                                                                          rRNA is presented as a stick model while the indicated r-proteins are <br /> SD sequence and start codon with sequences around 100 nu-                presented as space-fill models. The T. thermophilus S16 protein is 88 <br /> cleotides downstream from the start codon (19). Previously, it           residues in <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span>, but the most COOH-terminal amino acid in the <br /> was demonstrated that mutations that destabilized the postu-             determined structure is E83. <br />  <br />  <br /> slow growth of a &#226;&#338;&#172;rimM mutant (3), indicating that RimM   <br />                                                                                     <br /> associates with the 30S subunits in the area where S13 is found, <br />                                                                                        <br /> which is on the opposite side from where S16 is located in the   <br /> 30S subunits (20).                                              <br />    E. coli cells lacking the r-protein S16 are not viable (12),  <br /> probably because of the important role the S16 protein plays in   <br /> the assembly of the 30S subunits (7). Thus, the presence of the    <br /> S16-RimM protein in translationally active ribosomes of cells  <br /> lacking native S16 protein and the ability of the hybrid protein         <br /> to support fast growth of the cells suggest that it must be able  <br /> to substitute for the native S16 protein in ribosome assembly.  <br /> Considering that r-proteins to a large extent are embedded in             <br /> the rRNA structure of the ribosome and that the fusion of   <br /> RimM to S16 adds 189 extra amino acids to the 82 normally   <br /> present in S16 of E. coli, it was surprising to find the S16-RimM      <br /> hybrid protein in the 70S ribosomes. However, we note that in       <br /> the 3-&#195;&#8230; resolution structure of the 30S ribosomal subunit from             <br /> Thermus thermophilus (20), the most COOH-terminal amino                  <br /> acid (E83) of those localized in the structure of S16 protrudes   <br /> from the 30S subunits (Fig. 8). Assuming that the structure of    <br /> the 30S subunits in E. coli is similar to that in T. thermophilus,  <br /> the RimM part of the hybrid protein seems therefore to be    <br /> present on the surface of the 30S subunits and might not                  <br /> interfere with the assembly of the 30S subunits. The in vitro                <br /> assembly of the 30S subunits is highly inefficient in the absence     <br /> of S16; however, once assembled, the 30S subunits lacking S16                  <br /> are translationally competent (7), suggesting that S16 has no                    <br /> role or little role in translation as such. Thus, the addition of                     <br /> 189 extra amino acids to the COOH-terminal end of S16 might              <br /> not have any effect on the translational capacity of the ribo-      <br /> somes as long as the assembly function of S16 is not affected.              <br />                                                                                  <br />   <br /> </body></html>