<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>ANTIMICROBIAL AGENTS AND CHEMOTHERAPY, Apr. 2001, p. 1099&#226;&#8364;&#8220;1103                                                                                               Vol. 45, No. 4 <br /> 0066-4804/01/$04.00&#226;&#171;&#185;0 DOI: 10.1128/AAC.45.4.1099&#226;&#8364;&#8220;1103.2001 <br /> Copyright &#194;&#169; 2001, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br /> rho Is Not Essential Viability Virulence Staphylococcus aureus <br />            ROBERT S. WASHBURN,&#226;&#8364;&#160; ANDREA MARRA,&#226;&#8364;&#161; ALEXANDER P. BRYANT, MARTIN ROSENBERG, <br />                                       AND DANIEL R. GENTRY* <br />  <br />                                                     GlaxoSmithKline, Collegeville, Pennsylvania 19426 <br />                         Received 15 November 2000/Returned modification 29 December 2000/Accepted 11 January 2001 <br />  <br />               We identified gene transcription termination factor Rho Staphylococcus aureus. Deletion rho <br />             S. aureus reveals essential viability virulence. We searched available bacterial <br />             genomic sequences homologs Rho broadly distributed highly conserved. Ex- <br />             ceptions include Streptococcus pneumoniae, Streptococcus pyogenes, Mycoplasma genitalium, Mycoplasma pneu- <br />             moniae, Ureaplasma urealyticum, Synechocystis sp. strain PCC6803, appear possess Rho <br />             homolog. Complementation studies indicate S. aureus Rho possesses activity Escherichia coli <br />             Rho Rho inhibitor bicyclomycin active S. aureus Rho. Our results explain lack of <br />             activity bicyclomycin gram-positive bacteria raise possibility essentiality rho <br />             exception rule. <br />  <br />  <br />    The rho gene codes transcription termination factor Rho                              E. coli rho. The plasmid pAU2442 used study genomic <br /> (19) essential viability Escherichia coli (5). The                        library used sequencing project. <br />                                                                                                Deletion mutagenesis. A rho deletion introduced S. aureus RN4220. <br /> function Rho catalyze release RNA                                    This achieved electroporating S. aureus RN4220 tetracycline resistance <br /> transcription complex Rho-dependent terminator se-                                  (Tcr) pRW101, rho gene 500 bp flanking sequence <br /> quence transcribed. Rho RNA-dependent                                       interrupted tetracyline resistance marker replaces rho gene <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> <br /> NTPase activity, required transcript release (10,                              codon 1 codon 313 (Fig. 1). The resulting Tcr colonies screened for <br />                                                                                             erythromycin resistance (Emr), carried pRW101. Tcr Ems <br /> 12), RNA-DNA helicase activity (1). Several Rho- <br />                                                                                             colonies indicated double-crossover event occurred. The deletion rho was <br /> dependent transcription terminators Escherichia coli                                 confirmed PCR analysis primers flanking Tcr marker. <span id='am-4' about='protege:TO' typeof='owl:Thing'>To</span> construct a <br /> coliphage &#226;?&#173; known (4, 20).                                                              &#226;&#338;&#172;rho strain pathogenic background, &#226;&#338;&#172;rho::tet <span id='am-5' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-6' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-18' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-23' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span>mutation</span> moved into <br />    Rho hexamer identical 47-kDa monomers (2, 9, 11,                                 S. aureus clinical isolate WCUH29 bacteriophage &#226;&#338;&#189;11 transduction. The <br /> 17). Its ability bind RNA thought conferred                                  resulting strain, RSW101, used virulence testing described below. <br />                                                                                                Complemention E. coli rho. E. coli rho replaced S. aureus rho by <br /> residues 22 116 419-amino-acid protein (14). The                                  introducing S. aureus rho-containing plasmid pAU2442 E. coli rho <br /> ATP binding domain located residues 167 319                                  deletion strain BLS107(pPMrho). This strain deletion-insertion (rho::kan) <br /> (7). No activity assigned essential C terminus                           mutation chromosomal copy rho. A wild-type copy rho carried on <br /> Rho, speculated involved                                   plasmid pPMrho, temperature-sensitive replicon. This strain fails <br />                                                                                             grow 42&#194;&#176;C loss rho temperature (22). pAU2442 was <br /> subunit interactions (1, 6, 8). <br />                                                                                             introduced BLS107(pPMrho) electroporation plated 30, 37, <br />    The antibiotic bicyclomycin inhibits Rho (23). With ex-                              42&#194;&#176;C. As control, parent strain plated different tem- <br /> ception Micrococcus luteus (15, 16), gram-positive bacteria                              peratures. While parent strain failed form isolated colonies 37 42&#194;&#176;C, <br /> are resistant bicyclomycin, rho essential                              strain containing pAU2442 grew formed isolated colonies all <br /> gram-positive bacterium Bacillus subtilis (13, 21). This study                              temperatures. The loss pPMrho confirmed scoring loss chlor- <br />                                                                                             amphenicol resistance, encoded plasmid. To determine S. <br /> reports sequence characterization rho gene                                aureus Rho protein sensitive bicyclomycin, strain BLS107(pAU2442) was <br /> the clinically important gram-positive pathogen Staphylococcus                              tested sensitivity MIC determination. Liquid MICs determined a <br /> aureus.                                                                                     serial dilution method. Strain BLS107 MIC bicyclomycin <br />  <br />  <br />                         MATERIALS AND METHODS <br />    Bacteria growth conditions. Staphylococcus aureus propagated                             TABLE 1. Strains plasmids used study <br /> either tryptic soy broth (DIFCO) Luria broth (LB). E. coli grown LB. <br /> For auxotrophy measurements S. aureus, M9 minimal medium supple-                     Strain                                                      Source or <br />                                                                                                                           Description <br /> mented arginine, cysteine, glutamate, glycine, isoleucine, leucine, methio-              plasmid                                                        reference <br /> nine, proline, valine, 20 &#226;?&#174;g/ml. Nicotinate thiamine added at <br />                                                                                             Strains <br /> 0.2 &#226;?&#174;g/ml. All cultures routinely incubated 37&#194;&#176;C. All strains used this <br />                                                                                               BLS107         E. coli &#226;&#338;&#172;rho::Kanr                       22 <br /> study described Table 1. <br />                                                                                               MG1655         Wild-type E. coli strain                 Laboratory collection <br />    Identification S. aureus rho gene. The rho gene Staphylococcus                 RN4220         S. aureus laboratory strain              Laboratory collection <br /> aureus identified S. aureus genomic sequence database homology                 RSW100         RN4220 &#226;&#338;&#172;rho::Tetr                        This study <br />                                                                                               RSW101         WCUH29 &#226;&#338;&#172;rho::Tetr                        This study <br />                                                                                               WCUH29         S. aureus clinical isolate               Laboratory collection <br />    * Corresponding author. Mailing address: GlaxoSmithKline, UP1345, <br /> 1250 S. Collegeville Rd., Collegeville, PA 19426. Phone: (610) 917-                         Plasmids <br /> 7504. Fax: (610) 917-7901. E-mail: Dan_R_Gentry@sbphrd.com.                                   pAU2442        4.3-kb genomic clone                This study <br />    &#226;&#8364;&#160; Present address: Institute Cancer Research, Columbia Univer-                                          WCUH29 containing rho <br /> sity, New York, NY 10032.                                                                     pPMrho         Temperature-sensitive replicon,          22 <br />                                                                                                                3.0-kb genomic clone E. coli <br />    &#226;&#8364;&#161; Present address: Protein Design Laboratories, Inc., Fremont, CA <br />                                                                                                                containing rho <br /> 94555. <br />  <br />                                                                                      1099 <br />  1100        WASHBURN ET AL.                                                                                                     ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br />  <br />  <br />   FIG. 1. Sequence S. aureus rho flanking sequences used report. Shown DNA sequence region S. aureus <br /> chromosome contains rho. The rho gene product shown coding sequence. DNA sequence strikethrough line depicts the <br /> sequence deleted rho deletion-insertion strain described paper. <br />  <br />  <br />  <br />  <br /> of 125 &#226;?&#174;g/ml. This twofold lower wild-type E. coli strain           &#226;&#172;&#402;2-cm-long incision animals&#226;&#8364;&#8482; backs. The infected <br /> MG1655 (250 &#226;?&#174;g/ml). The MIC S. aureus strain RN4220             sutures secured end wound tacked midway the <br /> &#226;&#172;&#381;2.5 mg/ml.                                                                             wound underside skin. The suture secured end <br />   Virulence testing. The rho deletion strain RSW101 wild-type parent,           wound, wound closed single staple. After 5 days, the <br /> WCUH29, tested virulence mouse S. aureus infection models.            mice sacrificed CO2 overdose. The skin surrounding wound was <br /> The surgical wound model measures ability grow sutures sewn            excised homogenized 1 ml phosphate-buffered saline (PBS) a <br /> skin healthy mice, hematogenous pyelonephritis model measures          stomacher, bacterial viable counts enumerated. Both infections yielded <br /> ability strains cause kidney infections following parenteral infection   106 bacteria 5 days, significant difference observed the <br /> the bacteria.                                                                           strains. For hematogenous pyelonephritis model, bacterial recovery rates <br />   Surgical wound infections. The mice used infections 4-              (mean &#226;&#171;&#190; standard deviation; n &#226;&#171;&#189; 5) 5.35 &#226;&#171;&#190; 0.41 log10 CFU/ml strain <br /> 6-week-old male CD-1 mice. Approximately 15-cm sutures soaked over-             WCUH29 6.44 &#226;&#171;&#190; 0.81 log10 CFU/ml strain WCUH29 &#226;&#338;&#172;rho. For the <br /> night cultures WCUH29 RSW101 30 min room temperature. The                 surgical wound model, recovery rates 6.58 &#226;&#171;&#190; 0.78 log10 CFU/ml for <br /> backs mice shaved, animals anesthetized isoflurane,           strain WCUH29 7.41 &#226;&#171;&#190; 0.57 log10 CFU/ml strain WCUH29 &#226;&#338;&#172;rho. One <br />  VOL. 45, 2001                                                                                                  S. AUREUS rho DELETION          1101 <br />  <br />  <br />  <br />  <br />  FIG. 2. Alignment S. aureus Rho B. subtilis E. coli Rho. Sequences aligned using MegAlign program the <br /> DNAstar DNA analysis package. Identical residues shaded. <br />  <br />  <br />  <br /> hundred colonies isolated rho mutant infection checked tet-      different bacterial species (18). The predicted ATPase domain <br /> racycline resistance, indicative presence mutation,     S. aureus Rho (residues 180 356) 80% similar 70% <br /> found tetracycline resistant. <br />    Hematogenous pyelonephritis infections. Six- eight-week-old male CD-1 <br />                                                                                    identical E. coli Rho 86% similar 81% identical to <br /> mice used hematogenous pyelonephritis infections. Cultures WCUH29      B. subtilis Rho. Finally, predicted RNA binding domain of <br /> and RSW101 cells adjusted A600 0.6 0.3 ml, 200 &#226;?&#174;l     S. aureus Rho (residues 40 132) 63% similarity 41% <br /> injected mouse tail vein. Mice sacrificed day 5 postin-    identity E. coli Rho 77% similarity 60% identity to <br /> fection CO2 overdose, kidneys aseptically removed then <br /> homogenized 1 ml PBS stomacher, viable counts enumerated <br />                                                                                    B. subtilis Rho. <br /> (described above). The inoculum strains 107 cells. The number         Complementation. The S. aureus rho gene complement <br /> bacteria recovered kidneys infected RSW101 5 days      lack growth E. coli strain rho deleted.. This <br /> decreased. One colonies RSW101 infection checked         determined curing rho-deleted strain plasmid that <br /> the tetracycline resistance marker, 98% tetracycline resis- <br />                                                                                    carries E. coli rho presence S. aureus rho carried on <br /> tant. <br />    Nucleotide sequence accession number. The nucleotide sequence S.         plasmid. The sole Rho activity resulting strain, <br /> aureus rho gene deposited GenBank accession no. AF333962.        BLS107(pAU2442), S. aureus Rho. Strain BLS107 <br />                                                                                    (pAU2442) sensitive bicyclomycin, indicated the <br />                                  RESULTS                                           bicyclomycin MIC 125 &#226;?&#174;g/ml, indicating S. aureus Rho <br />                                                                                    inhibited bicyclomycin. As evidence this, bicy- <br />   The S. aureus Rho protein. S. aureus Rho predicted                         clomycin inhibit ATPase activity partially <br /> <span id='am-2' about='Thesaurus:code' typeof='owl:Thing'>code</span> 443-amino-acid protein predicted molecular                       purified S. aureus Rho (data shown). <br /> mass 50 kDa. S. aureus Rho 53% identical 67%                                The S. aureus rho gene essential vitro growth. A <br /> similar E. coli Rho 66% identical 75% similar                        &#226;&#338;&#172;rho strain, RSW100, constructed replacing nearly all <br /> B. subtilis Rho (Fig. 2). The predicted ATPase domain (resi-                       rho gene tetracycline resistance cassette (Fig. 1). <br /> dues 167 342 E. coli Rho) highly conserved                          The resulting strain viable grows normally 30, 37, 42, <br />  1102      WASHBURN ET AL.                                                                                      ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br />             TABLE 2. Bacterial phyla members                         46&#194;&#176;C. RSW100 grows normally rich medium (LB <br />                     possess Rho homologa                                     tryptic soy broth) minimal medium (supplemented M9 <br /> Aquificales                                                                    medium) anaerobic conditions (data shown). <br />   Aquifex aeolicus                                                                rho required S. aureus virulence. The rho deletion <br />                                                                                transduced pathogenic clinical isolate WCUH29, <br /> Chlamydiales <br />   Chlamydia muridarum <br />                                                                                yielding strain RSW101. Cultures WCUH29 RSW101 <br />   Chlamydia pneumoniae                                                         used vivo models described Materials and <br />   Chlamydia trachomatis                                                        Methods. S. aureus strains mutated variety different <br />                                                                                genes affecting cell metabolism virulence exam- <br /> Cytophaga-Flexibacter-Bacteroides <br />   Porphyromonas gingivalis <br />                                                                                ined laboratories. These strains displayed reduc- <br />                                                                                tions 3 logs pyelonephritis model 1 log in <br /> Green sulfur                                                                   surgical wound infection. In experiments described <br />  Chlorobium tepidum                                                            here, vivo growth RSW101 impaired com- <br />                                                                                pared wild-type WCUH29 model <br /> Gram positive <br />  Low G&#226;&#171;&#185;C (not Streptococcus pyogenes, Mycoplasma pneumoniae,                   systems. From data, conclude that, vitro <br />      Mycoplasma genitalium, Streptococcus pneumoniae)                       growth, rho essential virulence strain <br />    Bacillus subtilis                                                           tested. <br />    Clostridium acetobutylicum <br />    Listeria monocytogenes <br />                                                                                   Review Rho homologs present public sequence data- <br />                                                                                <span id='am-83' about='xsp:base' typeof='owl:Thing'>base</span>. By using S. aureus Rho sequence query, we <br />   High G&#226;&#171;&#185;C (Actinobacteria)                                                    searched public databases proprietary sequence <br />     Micrococcus luteus <br />     Mycobacterium leprae                                                       databases homologs. Rho highly con- <br />     Mycobacterium tuberculosis                                                 served; BLAST scores 5 &#226;&#171;&#187; 10&#226;&#171;&#186;38 lower full- <br />     Streptomyces coelicolor                                                    <span id='am-3' about='xsp:length' typeof='owl:Thing'>length</span> Rho sequences annotated such. As indicated previ- <br />     Streptomyces lividans <br />                                                                                ously Opperman Richardson (18), Rho broadly <br /> Proteobacteria                                                                 distributed major phyla eubacteria (Table <br />   &#226;?&#163; Subdivision                                                                2). A major exception universal presence Rho among <br />      Rhodobacter sphaeroides                                                   eubacteria members low-GC, gram-positive <br />      Rickettsia prowazekii <br />                                                                                bacteria Streptococus pyogenes, Streptococcus pneumoniae, My- <br />   &#226;?&#164; Subdivision                                                                coplasma genitalium, M. pneumoniae, Ureaplasma urealyti- <br />     Neisseria animalis <br />                                                                                cum, possess significant homolog. In the <br />     Neisseria cinerea <br />     Neisseria elongata                                                         case S. pneumoniae, searched proprietary data- <br />     Neisseria gonorrhoeae                                                      bases addition publicly available databases failed to <br />     Neisseria lactamica                                                        identify homolog. In addition mentioned bacteria, <br />     Neisseria meningitidis <br />     Neisseria mucosa                                                           Synechocystis sp. strain PCC6803, cyanobacterium, ap- <br />     Neisseria pharyngis                                                        parently lacks Rho homolog. As indicated previously (18), <br />     Neisseria polysaccharea                                                    ATP binding motif Rho shares homology &#226;?&#163; and <br />     Neisseria subflava <br />                                                                                &#226;?&#164; subunits F1 ATPases. This homology conserved among <br />   &#206;&#181; Subdivision                                                                bacteria well. <br />     Campylobacter jejuni <br />     Helicobacter pylori <br />                                                                                                         DISCUSSION <br />   &#226;?&#165; Subdivision <br />     Actinobacillus actinomycetemcomitans <br />     Actinobacillus pleuropneumoniae <br />                                                                                   In report, S. aureus rho gene can <br />     Allochromatium vinosum                                                     complement E. coli rho deletion strain. The ability S. au- <br />     Buchnera aphidicola                                                        reus rho replace essential E. coli rho gene raises the <br />     Escherichia coli                                                           possibility performs functions S. aureus and <br />     Haemophilus influenzae <br />     Pseudomonas aeruginosa                                                     E. coli. rho essential low-GC, gram-positive bac- <br />     Pseudomonas fluorescens                                                    terium Bacillus subtilis (13, 21), gram-positive spe- <br />     Salmonella enterica serovar Typhimurium                                    cies, including Staphylococcus aureus, resistant Rho <br />     Vibrio cholerae <br />     Xylella fastidiosa <br />                                                                                inhibitor bicyclomycin (15). Here rho re- <br />     Yersinia pestis                                                            quired viability virulence S. aureus. The E. coli strain <br />                                                                                constructed here, rho gene replaced with <br /> Spirochaetales <br />   Borrelia burgdorferi                                                         S. aureus rho, remains sensitive bicyclomycin. This suggests <br />   Treponema pallidum                                                           resistance gram-positive bacteria bicyclomycin is <br />                                                                                result nonessentiality Rho. Supporting conclu- <br /> Thermotogales                                                                  sion extending lack essentiality rho other <br />   Thermotoga maritima <br />                                                                                genera, completed genomes mem- <br /> Thermus/Deinococcus                                                            bers low-GC, gram-positive bacteria lack Rho ho- <br />   Deinococcus radiodurans                                                      molog. The cyanobacterium Synechocystis sp. strain PCC6803 <br />   a <br />     Groupings taken http://www.ncbi.nih.gov/htbin-post/Taxonomy/wgetorg   lack Rho homolog. <br /> and determined BLAST analysis S. aureus Rho.                              An open question rho essential E. coli <br />  VOL. 45, 2001                                                                                                           S. AUREUS rho DELETION                    1103 <br />  <br />  <br /> and gram-negative bacteria, essential                       7. Dombroski, A. J., J. R. LaDine, R. L. Cross, T. Platt. 1988. The ATP <br />                                                                                         binding site rho protein. Affinity labelling Lys181 pyridoxal 5&#226;&#172;&#732;- <br /> gram-positive bacteria. It Rho activity redun-                         diphospho-5&#226;&#172;&#732;-adenosine. J. Biol. Chem. 263:18810&#226;&#8364;&#8220;18815. <br /> dant bacteria essential. This possibility                     8. Dombroski, A. J., T. Platt. 1988. Structure &#226;?&#179; factor: RNA-binding <br /> cannot readily discounted, clear enzyme                           domain separate region strong similarity ATP-binding do- <br />                                                                                         mains. Proc. Natl. Acad. Sci. USA 85:2538&#226;&#8364;&#8220;2542. <br /> responsible redundant activity                        9. Finger, L. R., J. P. Richardson. 1982. Stabilization hexameric <br /> sequence homology Rho case. A second                            form Escherichia coli protein rho ATP hydrolysis conditions. J. Mol. <br /> possibility, Rho essential organisms higher                          Biol. 156:203&#226;&#8364;&#8220;219. <br />                                                                                     10. Galluppi, G. R., C. Lowery, J. P. Richardson. 1976. Nucleoside triphos- <br /> GC content, unlikely, given Synechocystis sp. strain                         phate requirement termination RNA synthesis rho factor, p. 657&#226;&#8364;&#8220; <br /> PCC6803 lacks Rho, genome 47% GC, close                             665. In R. Losick M. Chamberlin (ed.), RNA polymerase. Cold Spring <br />                                                                                         Harbor Laboratory Press, Cold Spring Harbor, N.Y. <br /> to E. coli. The lack rho members                    11. Gogol, E. P., S. E. Seifred, P. H. von Hippel. 1991. Structure and <br /> low-GC, gram-positive bacteria member                                 assembly Escherichia coli transcription termination factor Rho its <br /> distantly related cyanobacteria indicate trend                         interactions RNA. I. Cryoelectron microscopic studies. J. Mol. Biol. <br />                                                                                         221:1127&#226;&#8364;&#8220;1138. <br /> essentiality rho exception rule.                        12. Howa <br /> </body></html>