<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Sept. 2000, p. 5147&#226;&#8364;&#8220;5152                                                                                  Vol. 182, No. 18 <br /> 0021-9193/00/$04.00&#226;&#171;&#185;0 <br /> Copyright &#194;&#169; 2000, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />           Essentiality, Expression, Characterization Class II <br />              3-Hydroxy-3-Methylglutaryl Coenzyme A Reductase of <br />                               Staphylococcus aureus <br />           E. IMOGEN WILDING,1 DONG-YUL KIM,2 ALEXANDER P. BRYANT,1 MICHAEL N. GWYNN,1* <br />         R. DWAYNE LUNSFORD,1 DAMIEN MCDEVITT,1 JOSEPH E. MYERS, JR.,1 MARTIN ROSENBERG,1 <br />                DANIEL SYLVESTER,1 CYNTHIA V. STAUFFACHER,3 AND VICTOR W. RODWELL2 <br />              Department Microbiology, SmithKline Beecham Pharmaceuticals, Collegeville, Pennsylvania 19426,1 and <br />              Departments Biochemistry2 Biological Sciences,3 Purdue University, West Lafayette, Indiana 47907 <br />                                               Received 21 April 2000/Accepted 12 June 2000 <br />  <br />              Sequence comparisons implied presence genes encoding enzymes mevalonate pathway for <br />           isopentenyl diphosphate biosynthesis gram-positive pathogen Staphylococcus aureus. In study we <br />           showed genetic disruption experiments mvaA, encodes putative class II 3-hydroxy-3- <br />           methylglutaryl coenzyme A (HMG-CoA) reductase, essential vitro growth S. aureus. Supplementa- <br />           tion media mevalonate permitted isolation auxotrophic mvaA null mutant attenuated for <br />           virulence murine hematogenous pyelonephritis infection model. The mvaA gene cloned S. aureus <br />           DNA expressed N-terminal His tag Escherichia coli. The encoded protein affinity purified <br />           apparent homogeneity shown class II HMG-CoA reductase, class II eubacterial <br />           biosynthetic enzyme isolated. Unlike HMG-CoA reductases, S. aureus enzyme exhibits dual <br />           coenzyme specificity NADP(H) NAD(H), NADP(H) preferred coenzyme. Kinetic param- <br />           eters determined substrates catalyzed reactions using NADP(H) NAD(H). In <br />           instances optimal activity using NAD(H) occurred pH units acidic using <br />           NADP(H). pH profiles suggested His378 Lys263, apparent cognates active-site histidine and <br />           lysine Pseudomonas mevalonii HMG-CoA reductase, function catalysis general catalytic <br />           mechanism valid S. aureus enzyme. Fluvastatin inhibited competitively HMG-CoA, Ki of <br />           320 &#226;?&#174;M, 104 higher class I HMG-CoA reductase. Bacterial class II HMG-CoA reductases <br />           potential targets antibacterial agents directed multidrug-resistant gram-positive cocci. <br />  <br />  <br />    Isoprenoids, ubiquitous nature, comprise fam-              distinct classes HMG-CoA reductase. Genes appear <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> <br /> ily 23,000 products, composed repeating five-                encode class I HMG-CoA reductases present eu- <br /> carbon, isopentenyl diphosphate (IPP) subunits. The principal                karyotes, archaea, streptomycetes. By <br /> products IPP bacteria include lipid carrier undeca-                contrast, genes encode class II forms enzyme are <br /> prenol, involved cell wall biosynthesis (33), mena-              present eubacteria archaeon Archaeoglobus <br /> quinones ubiquinones involved electron transport (29),                fulgidus (41). Previously characterized class I HMG-CoA re- <br /> and carotenoids (19). Two pathways biosynthesis IPP               ductases include archaea Haloferax volcanii (6) <br /> have described, mevalonate pathway (16)                     Sulfolobus solfataricus (7, 22), gram-positive eubacte- <br /> glyceraldehyde-3-phosphate (GAP)-pyruvate pathway (36, 37).                  rium Streptomyces sp. strain CL190 (40), numerous eu- <br /> Analysis distribution genes encoding enzymes                   karyotes. Characterized class II HMG-CoA reductases include <br /> involved pathways revealed Bacillus subtilis             A. fulgidus (23) biodegradative Pseudomonas <br /> many gram-negative bacteria, including Escherichia coli, Hae-                mevalonii enzyme (4, 20), structure deter- <br /> mophilus influenzae, Helicobacter pylori, possess genes             mined (24, 39) converts mevalonate HMG-CoA, <br /> that encode GAP-pyruvate pathway, low-G&#226;&#171;&#185;C                      permitting growth mevalonate (15). No biosynthetic eubac- <br /> gram-positive cocci Borrelia burgdorferi possess genes              terial class II HMG-CoA reductase has, however, char- <br /> that encode mevalonate pathway (41). However, func-                  acterized. <br /> tionality implied polypeptides mevalonate path-                   Biosynthetic HMG-CoA reductases catalyze reaction 1, the <br /> way bacteria demonstrated.                                   reductive deacylation thioester group HMG-CoA to <br />    Enzymes involved mevalonate pathway number                 primary alcohol mevalonate using 2 mol NADPH <br /> organisms, including humans, isolated studied.                 (35). The putative intermediates, mevaldyl-CoA mevalde- <br /> 3-Hydroxy-3-methylglutaryl coenzyme A (HMG-CoA) reduc-                       hyde, remain bound course reaction. HMG- <br /> tase (EC 1.1.1.34) best-characterized rate-limiting               CoA reductases catalyze additional reactions. Re- <br /> enzyme pathway (9) target statin class              actions 2 3 free mevaldehyde appear model the <br /> of cholesterol-lowering drugs (1). Based amino acid se-                   second reductive stage reverse reductive <br /> quence analysis, Bochar et al. (8) suggested              stage reaction 1, respectively. Reaction 4, oxidative ac- <br />                                                                              ylation (R)-mevalonate (S)-HMG-CoA, reverse of <br />                                                                              reaction 1. <br />   * Corresponding author. Mailing address: Department Microbi- <br /> ology, SmithKline Beecham Pharmaceuticals, 1250 South College Rd.,                &#229;&#8230;&#177;S&#229;&#8230;&#178;-HMG-CoA &#226;&#171;&#185; 2 NADPH &#226;&#171;&#185; 2 H&#226;&#171;&#185;3&#229;&#8230;&#177;R&#229;&#8230;&#178;-mevalonate &#226;&#171;&#185; CoASH &#226;&#171;&#185; 2 NADP&#226;&#171;&#185;   (1) <br /> Collegeville, PA 19426. Phone: (610) 917-7749. Fax: (610) 917-4989. <br /> E-mail: Mick_Gwynn-1@sbphrd.com.                                                      &#229;&#8230;&#177;R&#229;&#8230;&#178;-mevaldehyde &#226;&#171;&#185; NADPH &#226;&#171;&#185; H&#226;&#171;&#185;3&#229;&#8230;&#177;R&#229;&#8230;&#178;-mevalonate &#226;&#171;&#185; NADP&#226;&#171;&#185;         (2) <br />  <br />                                                                       5147 <br />  5148       WILDING ET AL.                                                                                                                               J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 1. Chromosomal analysis S. aureus mvaA null mutant. (A) The chromosome mvaA null mutant contains allelic replacement mvaA (dotted <br /> block arrow) ermC gene (gray block arrow). PCR primers (small arrows) designed anneal regions flanking mvaA. (B) Products expected size <br /> were amplified using chromosomal DNA mutant (2,288 bp [lane 3]) wild type (2,371 bp [lane 8]). Replacement mvaA ermC resulted the <br /> elimination HindIII sites introduction SspI sites chromosome. Amplified fragments digested HindIII SspI, products <br /> of expected size obtained. Lane 1, mutant PCR product incubated HindIII (undigested); lane 2, mutant PCR product cut SspI (1,518 and <br /> 659 bp smaller bands); lane 6, wild-type PCR product cut HindIII (894, 776, 618 bp); lane 7, wild-type PCR product incubated SspI (undigested); <br /> lanes 4 5, 100-bp 1-kb DNA markers, respectively. <br />  <br />  <br />         &#229;&#8230;&#177;R&#229;&#8230;&#178;-mevaldehyde &#226;&#171;&#185; NADP&#226;&#171;&#185; &#226;&#171;&#185; CoASH3&#229;&#8230;&#177;S&#229;&#8230;&#178;-HMG-CoA &#226;&#171;&#185; NADPH &#226;&#171;&#185; H&#226;&#171;&#185;             (3)          Determination essentiality mvaA gene. A DNA construct allelic <br />                                                                                        replacement S. aureus mvaA gene generated overlap three-piece <br />        &#229;&#8230;&#177;R&#229;&#8230;&#178;-mevalonate &#226;&#171;&#185; 2 NADP&#226;&#171;&#185; &#226;&#171;&#185; CoASH3&#229;&#8230;&#177;S&#229;&#8230;&#178;-HMG-CoA &#226;&#171;&#185; 2 NADPH &#226;&#171;&#185; 2 H&#226;&#171;&#185;         (4)       PCR technique. This technique gene fusion procedure (42) extended to <br />                                                                                        include separate long flanking sequences (547 bp directly upstream 591 <br /> where CoASH reduced coenzyme A.                                                     bp directly downstream mvaA gene) surrounding central selectable <br />    We report mvaA gene gram-positive                              cassette containing 1,234-bp ermC gene pE194 (17). The final full- <br /> pathogen Staphylococcus aureus essential growth,                         <span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> products digested cloned BamHI site pBluescript <br /> report cloning, purification, characterization                          II KS(&#226;&#171;&#185;) derivative containing tetracycline resistance gene (tetK) pT181 <br /> mvaA gene product S. aureus, truly biosynthetic                           (21). The resulting plasmids introduced S. aureus RN4220 electro- <br />                                                                                        poration selection resistance erythromycin. All colonies iso- <br /> class II HMG-CoA reductase characterized eubacterium.                           lated, resistant tetracycline (cointegrants), examined for <br />                                                                                        target-specific plasmid integration diagnostic PCR primer pairs based on <br />                         MATERIALS AND METHODS                                          plasmid- locus-specific sequences. Bona fide plasmid cointegrants were <br />                                                                                        transferred S. aureus RN4220 NCTC 8325-4 phage &#226;?&#190;11 trans- <br />    Reagents. Purchased reagents included restriction enzymes, calf intestinal          duction allow second recombination event potentially resolve <br /> alkaline phosphatase, T4 DNA ligase, protein DNA molecular weight mark-            generate allelic-replacement <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-142' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-145' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-159' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-160' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>mutation</span>. Transductants resistant erythromy- <br /> ers (Gibco BRL Life Technologies), Pfu Turbo DNA polymerase (Stratagene),              cin sensitive tetracycline examined allelic replacement PCR. <br /> and lysostaphin (Applied Microbiology, Inc.). Fluvastatin gift Novar-          Murine hematogenous pyelonephritis infection model. Cells overnight <br /> tis. Unless specified, chemicals Sigma.                  cultures S. aureus NCTC 8325-4 mvaA null mutant grown TSB with <br />    Plasmid, bacterial strains, culture media. Expression plasmid pET28         10 mM mevalonate required centrifuged, washed twice sterile phos- <br /> <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> Novagen. Bacterial strains used included E. coli BL21(DE3) DH5&#226;?&#163;           phate-buffered saline, adjusted A600 0.2. Female CD-1 mice weigh- <br /> S. aureus strains RN4220 (31), WCUH29 (NCIMB 40771), NCTC 8325-4.                  ing 18 20 g (Charles River, Quebec, Canada) inoculated tail vein <br /> Luria-Bertani broth agar (38) served growth E. coli, tryptone soy       injection 0.2 ml suspension containing approximately 107 bacteria. Mice <br /> broth (TSB) tryptone soy agar (TSA) (Oxoid) served growth S. aureus.        monitored twice daily signs illness. All animals euthanatized by <br /> Where required, E. coli medium supplemented 50 &#226;?&#174;g kanamycin            carbon dioxide overdose 5 days inoculation. Both kidneys removed <br /> per ml 1% (wt/vol) glucose S. aureus medium supplemented          using aseptic technique homogenized 1 ml PBS, the <br /> 0.5 &#226;?&#174;g erythromycin 5 &#226;?&#174;g tetracycline ml.                                number viable bacteria determined plating TSA, sup- <br />    DNA techniques. Plasmid DNA isolated using RPM kit (Bio 101 Inc.)           plemented 10 mM mevalonate mvaA null mutant. <br /> or Wizard Midiprep DNA purification (Promega). PCR products                    Construction HMG-CoA reductase expression plasmid. The mvaA gene <br /> were isolated horizontal agarose gel electrophoresis purified using         amplified S. aureus WCUH29 chromosomal DNA using primers which <br /> GENECLEAN II kit (Bio 101 Inc.). Chromosomal DNA isolated S.                  introduced NheI BamHI sites ends amplified fragment. The <br /> aureus using standard procedures (27). Incubation 0.1 &#226;?&#174;g lysostaphin       1,278-bp PCR fragment cloned using pCR-Zero Blunt cloning kit (In- <br /> ml included preparation plasmid chromosomal DNA             vitrogen) and, using introduced restriction sites, subcloned pET28 in <br /> S. aureus facilitate cell lysis. Procedures DNA restriction, dephosphoryla-     frame N-terminal histidine tag thrombin cleavage site form pJO1. <br /> tion ligation, agarose gel electrophoresis, PCR, transformation CaCl2-          The insert sequenced verify errors introduced during <br /> competent E. coli, phage transduction performed described               PCR amplification. <br /> Sambrook et al. (38) recommended manufacturers&#226;&#8364;&#8482; instructions. PCR            Expression purification HMG-CoA reductase. E. coli BL21(DE3) cells <br /> employed RoboCycler gradient temperature cycler (Stratagene). Synthetic              containing pJO1 grown mid-log phase, induced addition 1 mM <br /> oligonucleotides prepared, automated DNA sequencing per-                  isopropyl-&#226;?&#164;-D-thiogalactopyranoside (IPTG), harvested 2.5 h postinduction. <br /> formed in-house SmithKline Beecham.                                                 To confirm expression, total cell lysates prepared outlined Sambrook <br />  VOL. 182, 2000                                                                        STAPHYLOCOCCUS AUREUS HMG-CoA REDUCTASE                                       5149 <br />  <br />  <br />                                                                                          Reaction 1, reductive deacylation HMG-CoA mevalonate. Assays using <br />                                                                                       NADPH coenzyme contained 0.25 mM NADPH, 0.25 mM (R,S)-HMG- <br />                                                                                       CoA, 50 mM NaCl, 1 mM EDTA, 5 mM DTT, 25 mM KH2PO4 (pH 7.5). <br />                                                                                       Assays using NADH coenzyme contained 0.25 mM NADH, 0.25 mM <br />                                                                                       (R,S)-HMG-CoA, 50 mM NaCl, 1 mM EDTA, 5 mM DTT, 25 mM KH2PO4 <br />                                                                                       (pH 6.0). <br />                                                                                          Reaction 2, reduction mevaldehyde mevalonate. Assays contained 0.5 <br />                                                                                       mM NADPH, 8.0 mM (R,S)-mevaldehyde, 50 mM NaCl, 1 mM EDTA, 5 mM <br />                                                                                       DTT, 25 mM KH2PO4 (pH 7.0). <br />                                                                                          Reaction 3, oxidative acylation mevaldehyde HMG-CoA. Assays con- <br />                                                                                       tained 0.5 mM NADP&#226;&#171;&#185;, 5.0 mM coenzyme A, 8.0 mM (R,S)-mevaldehyde, 50 <br />                                                                                       mM NaCl, 1 mM EDTA, 5 mM DTT, 100 mM Tris-HCl (pH 9.0). <br />                                                                                          Reaction 4, oxidative acylation mevalonate HMG-CoA. Assays contained <br />                                                                                       5 mM NADP&#226;&#171;&#185;, 5.0 mM coenzyme A, 6.0 mM (R,S)-mevalonate, 50 mM NaCl, <br />                                                                                       1 mM EDTA, 5 mM DTT, 100 mM Tris-HCl (pH 9.0). <br />                                                                                          Unless stated, reactions initiated adding HMG-CoA, <br />    FIG. 2. Expression S. aureus mvaA purification gene product. A       mevaldehyde, mevalonate. For assays, enzyme <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> represents the <br /> protein anticipated size absent induced E. coli BL21(DE3) cells       turnover, 1 min, 1 &#226;?&#174;mol nicotinamide nucleotide coenzyme. This corre- <br /> containing parent vector (lane 1) expressed cells containing pJO1      sponds turnover 1 &#226;?&#174;mol mevaldehyde 0.5 &#226;?&#174;mol HMG-CoA <br /> (lane 2). Lane 3, crude cytosol; lane 4, material failed bind nickel   mevalonate. Reported data mean values triplicate determinations. <br /> column. Bound protein eluted imidazole concentrations 10 mM <br /> (lane 5), 50 mM (lane 6), 100 mM (lane 7), 500 mM (lane 8). Lanes 9 and <br /> 10 contain 20 4 &#226;?&#174;g purified protein, respectively.                                                               RESULTS <br />  <br />                                                                                          Essentiality mvaA gene S. aureus. The genes in- <br /> et al. (38) subjected sodium dodecyl sulfate-polyacrylamide gel electro-       volved synthesis IPP mevalonate pathway are <br /> phoresis. Purification HMG-CoA reductase employed 500-ml culture              essential vitro growth Streptococcus pneumoniae (41). <br /> E. coli BL21(DE3) transformed pJO1. Following induction, cells were <br /> harvested centrifugation HMG-CoA reductase purified using the <br />                                                                                       Therefore, determine S. pneu- <br /> HisTrap kit (Amersham Pharmacia Biotech) essentially described              moniae representative gram-positive cocci if <br /> manufacturer&#226;&#8364;&#8482;s instructions, following modifications. Cell lysis     HMG-CoA reductase essential pathogens. Attempts <br /> performed addition 0.2 mg lysozyme ml incubation 20 min          delete mvaA chromosome S. aureus <br /> at 37&#194;&#176;C. Lysates frozen dry ice-ethanol bath, thawed, sonicated <br /> three times 10 s, 10 s bursts. The freezing-and-sonication <br />                                                                                       using allelic-replacement mutagenesis strategy. For non- <br /> procedure repeated times. Protein eluted column 10,        essential genes cointegrant readily resolved using <br /> 50, 100, 500 mM imidazole. Purified HMG-CoA reductase solution            phage transduction isolate mutant fre- <br /> 50 mM KH2PO4 (pH 7.5), 200 mM NaCl, 1 mM dithiothreitol (DTT), 1 mM                   quency 0.5 5%. In case, however, proved impossible <br /> EDTA, 50% (wt/vol) glycerol stored &#226;&#171;&#186;20&#194;&#176;C. Protein concentrations <br /> were determined using Bradford assay (10) using bovine serum albumin as <br />                                                                                       resolve cointegrant structure isolate mutant on <br /> a standard.                                                                           standard growth media, strongly suggesting mvaA es- <br />   Assays HMG-CoA reductase activities. Spectrophotometric assays HMG-           sential vitro growth. <br /> CoA reductase activity employed Hewlett-Packard model 8452 diode array                 When growth media supplemented meval- <br /> spectrophotometer cell compartment maintained 37&#194;&#176;C mea- <br /> surements 340 nm oxidation reduction NAD(P)H. Assays were <br />                                                                                       onate, mvaA auxotrophic mutants readily obtained and <br /> conducted final volume 200 &#226;?&#174;l. Standard assay conditions             allelic replacement confirmed using diagnostic PCR <br /> reaction studied follows.                                                     restriction analysis. Oligonucleotide primers designed to <br />  <br />  <br />  <br />  <br />    FIG. 3. Effect hydrogen ion concentration activity. The effects indicated concentrations hydrogen ion rates indicated reactions were <br /> assayed standard conditions pH using NADP(H) (F) NAD(H) (E) coenzyme. Assays conducted solution 50 mM KCl, <br /> 25 mM K-PO4, 100 mM Tris-HCl adjusted indicated pH values (x axis). (Top left) Reaction 1, reductive deacylation HMG-CoA mevalonate; (top right) <br /> reaction 2, reduction mevaldehyde mevalonate; (lower left) reaction 3, oxidative acylation mevaldehyde HMG-CoA; (lower right) reaction 4, oxidative <br /> acylation mevalonate HMG-CoA. eu, enzyme units. <br />  5150         WILDING ET AL.                                                                                                                           J. BACTERIOL. <br />  <br />  <br />       TABLE 1. Comparison Km values class II S. aureus <br />       HMG-CoA reductase class II biodegradative <br />          enzyme P. mevalonii class I biosynthetic <br />                          hamster enzymea <br />                                                        Km (&#226;?&#174;M) enzyme from: <br />              Reaction substrates                   S.         P. <br />                                                                              Hamster <br />                                                      aureus    mevalonii <br />  <br /> Reaction 1 (HMG-CoA3mevalonate) <br />  HMG-CoA                                                40         20            20 <br />  NADPH                                                  70                       80 <br />  NADH                                                  100         80 <br />  <br /> Reaction 2 (mevaldehyde3mevalonate) <br />  Mevaldehyde                                         6,900      8,000         1,600 <br />  NADPH                                                 240                      160 <br />                                                                                           FIG. 4. Inhibition statin drug. Double-reciprocal plot inhibition of <br /> Reaction 3 (mevaldehyde3HMG-CoA)                                                       reaction 1, reductive deacylation HMG-CoA mevalonate, using <br />  Mevaldehyde                                           570         80            90    NADPH coenzyme. Analyses conducted pH 7.5 37&#194;&#176;C the <br />                                                                                        indicated concentrations HMG-CoA presence 0 &#226;?&#174;M (E) 500 &#226;?&#174;M <br />  CoASH                                                 490        110            50    (F) fluvastatin. All reactions initiated adding NADPH. eu, enzyme units. <br />  NADP&#226;&#171;&#185;                                                  40                      600 <br />  <br /> Reaction 4 (mevalonate3HMG-CoA) <br />  Mevalonate                                            670        260            20 <br />  CoASH                                                 390         60            10    mevalonate concentration reduced 1 nM, however, <br />  NADP&#226;&#171;&#185;                                                 580                      510    mvaA null mutant unable grow, viability <br />                                                                                        unaffected (data shown). <br />   a <br />       Data P. mevalonii hamster HMG-CoA reductase reference              Virulence attenuation mvaA null mutant. The mvaA <br /> 34. <br />                                                                                        mutation transferred pathogenic strain S. aureus <br />                                                                                        NCTC 8325-4 phage &#226;?&#190;11 transduction. The resulting mu- <br />                                                                                        tants auxotrophic mevalonate. Five mice inocu- <br /> anneal regions flanking mvaA (Fig. 1) internal                         lated intravenously S. aureus NCTC 8325-4 the <br /> regions mvaA ermC (data shown) used am-                              mvaA null mutant sacrificed 5 days postinfection (none of <br /> plify chromosomal DNA templates prepared wild-                                mice died infection). Viable S. aureus NCTC <br /> type mutant strains. The expected products obtained,                          8325-4 harboring deletion recovered the <br /> confirming mvaA replaced ermC auxo-                            kidneys (i.e., level limit detection 1.6 <br /> trophic mutant.                                                                        log10 CFU/mouse) plated presence mevalonate, <br />    Auxotrophy mvaA <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-58' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-60' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span>null mutants</span>. The minimal concentra-                             contrast wild type, S. aureus NCTC 8325-4, was <br /> tion mevalonate required growth S. aureus                            present 5.17 &#226;&#171;&#190; 0.17 log10 CFU/mouse (mean &#226;&#171;&#190; standard <br /> mvaA null mutant investigated TSA containing meval-                             deviation mice). The results indicate auxotro- <br /> onate 0.01, 0.1, 1 mM TSA supplemen-                             phic mvaA null mutant greatly reduced virulence the <br /> tation. Addition mevalonate medium enabled                               hematogenous pyelonephritis infection model. <br /> mutant grow, rate growth lower lower                            Expression mvaA purification S. aureus HMG-CoA <br /> concentrations mevalonate. In absence added me-                              reductase. The mvaA gene S. aureus cloned pET28 <br /> valonate, mutant showed growth 60 h rich                               frame N-terminal histidine tag thrombin cleav- <br /> medium (TSA).                                                                          age site. A soluble protein expressed E. coli BL21(DE3) <br />    The minimum concentration mevalonate required                             high levels following induction 1 mM IPTG was <br /> mvaA null mutant overnight growth TSB 1 mM (data                            readily purified nickel-chelating column, majority <br /> not shown). The effect removing mevalonate me-                             eluting 500 mM imidazole fraction (Fig. 2). Typical <br /> dium viability mvaA null mutants investigated.                           yields 30 mg 90% homogenous protein liter of <br /> An overnight culture mvaA null mutant grown                              induced culture. The mvaA gene encodes putative protein of <br /> presence 1 mM mevalonate diluted 106-fold fresh                            425 residues calculated molecular weight 46,180. <br /> media mevalonate. The number viable                                Matrix-assisted laser desorption ionization mass spectrometry <br /> cells determined plating presence 1 mM                                analysis indicated purified protein molecular <br /> mevalonate. In presence mevalonate mutant grew                              weight 48,503, corresponds full-length HMG- <br /> well, viable count increased 4 logs 24 h. When                         CoA reductase fusion protein minus terminal methionine <br />                                                                                        residue. N-terminal sequencing purified protein con- <br />                                                                                        firmed residue removed N- <br /> TABLE 2. Relative abilities class II HMG-CoA reductases                       terminal sequence intact. <br />                use NADP(H) NAD(H)                                                  Biochemical characterization. The enzymatic activity the <br />                                                                                        purified protein determined measuring oxidation or <br />  Source                     kcat(NADPH)/kcat(NADH) fora: <br />  HMG-CoA                                                                   Reference <br />                                                                                        reduction NAD(P)H 340 nm. Firstly, temperature <br />   reductase              Reaction 1               Reaction 4                           dependency S. aureus HMG-CoA reductase deacy- <br />                                                                                        lation HMG-CoA investigated. Optimal activity the <br /> S. aureus                      25                   25                  This study     catalysis reaction 1 observed 35 45&#194;&#176;C and <br /> A. fulgidus                     0.5                  0.05               23             decreased precipitously 45&#194;&#176;C (data shown). All sub- <br /> P. mevalonii                                     4.4 &#226;&#171;&#187; 10&#226;&#171;&#186;4             14 <br />                                                                                        sequent assays conducted 37&#194;&#176;C. <br />   a <br />       kcat, turnover number.                                                              Coenzyme specificity S. aureus HMG-CoA reductase was <br />  VOL. 182, 2000                                                                    STAPHYLOCOCCUS AUREUS HMG-CoA REDUCTASE                                 5151 <br />  <br />  <br />  <br />  <br />   FIG. 5. Alignment selected amino acid sequences class II HMG-CoA reductases S. aureus (Sa) P. mevalonii (Pm). Conserved paired residues <br /> E81S-E83P, K263S-K267P, D279S-D283P, H378S-H381P, bold, represent active-site residues known function catalysis P. mevalonii enzyme. The <br /> conserved ENVIG, DAMGXN, GTVGG sequences shaded. Asp146P Leu148P, marked asterisks, contribute ability P. mevalonii HMG-CoA <br /> reductase discriminate NADP(H). Additional residues conserved sequences boxed. The sequences S. aureus P. mevalonii <br /> HMG-CoA reductases 39% identical overall. Alignment performed using MegAlign sequence analysis software DNASTAR Inc., Madison, Wis. <br />  <br />  <br />  <br /> investigated range pH values. Figure 3 illustrates                  reductase established functions catalysis (9). We <br /> pH profiles catalysis reactions 1 4 using                        infer mechanism proposed catalysis by <br /> NADP(H) NAD(H) coenzyme. Unlike                              P. mevalonii HMG-CoA reductase (39) valid S. aureus <br /> characterized HMG-CoA reductases, S. aureus enzyme                        HMG-CoA reductase. <br /> use NADP(H) NAD(H) coenzymes catalysis                            All characterized class II HMG-CoA reductases use <br /> all reactions. While significant activity observed using                 NAD(H) (Table 2), S. aureus A. fulgidus <br /> NAD(H), NADP(H) instances preferred coen-                          enzymes use NAD(P)H NAD(H). In respect, the <br /> zyme. For reactions, optimal activity using NAD(H)                       S. aureus enzyme closely resembles archaeal enzyme <br /> occurred pH units acidic using                     mesophilic, biodegradative counterpart. The ability of <br /> NADP(H).                                                                          S. aureus HMG-CoA reductase use coenzyme was <br />    Km values determined substrates                      predicted, Asp146P Leu148P major determinants <br /> reactions catalyzed HMG-CoA reductases. Table 1 summa-                         nucleotide coenzyme specificity P. mevalonii HMG-CoA <br /> rizes data includes comparison Km values                   reductase (14). Crystal structures P. mevalonii enzyme <br /> the class II HMG-CoA reductase P. mevalonii                         revealed Asp146P interacts 2&#226;&#172;&#732;-hydroxyl the <br /> class I Syrian hamster enzyme. In Table 2 coenzyme spec-                      adenosyl ribose NAD(H), discriminating NADP(H) <br /> ificities characterized class II HMG-CoA reduc-                      (24, 39). Consistent ability use NADP(H), sequence <br /> tases compared.                                                               alignments revealed clear cognates Asp146P Leu148P <br />    The effect statin drug fluvastatin S. aureus HMG-                    S. aureus HMG-CoA reductase. Despite significant similar- <br /> CoA reductase investigated. Statin drugs competitive                      ity (39% identity 57% similarity), ability S. aureus <br /> inhibitors HMG-CoA reductase activity (9), ex-                          HMG-CoA reductase accommodate coenzymes sug- <br /> pected, catalysis reaction 1 inhibited fluvastatin.                     gests significant differences coenzyme binding site <br /> Inhibition competitive respect HMG-CoA,                        apparent crystal structure S. aureus <br /> Ki 320 &#226;?&#174;M (Fig. 4).                                                            HMG-CoA reductase determined compared of <br />                                                                                   P. mevalonii enzyme. Finally, suggest ability to <br />                          DISCUSSION                                               use coenzymes turn general property of <br />    We demonstrated mvaA gene essential                        biosynthetic class II forms enzyme. <br /> growth S. aureus mvaA null mutants                              Despite ability use NAD(H), S. aureus HMG-CoA <br /> readily isolated media supplemented mevalonate.                           reductase true biosynthetic enzyme. This inferred from <br /> These results support previous observations mvaA es-                      preference NADPH presence S. aureus of <br /> sential S. pneumoniae (41) strongly suggest gene                  genes encode putative HMG-CoA synthase, mevalonate <br /> is essential gram-positive cocci use meval-                    kinase, phosphomevalonate kinase, mevalonate decarbox- <br /> onate pathway synthesis IPP, GAP-pyruvate                       ylase (41). While oxidoreductases exhibit high order <br /> pathway organisms. The S. aureus                      coenzyme specificity, oxidoreductases use either <br /> mvaA null mutant severely attenuated mouse                             NADP(H) NAD(H) include L-glutamate dehydrogenase <br /> hematogenous pyelonephritis infection model. Although                         (11, 12, 28), isocitrate dehydrogenase (13, 25), glucose-6-phos- <br /> plasma mevalonate concentration mice mea-                         phate dehydrogenase (3, 5), 6-phosphogluconate dehydroge- <br /> sured, concentrations humans rats, 0.02 0.08 &#226;?&#174;M                     nase (5), aldose reductase (30), biliverdin IX&#226;?&#163; reductase <br /> and 0.08 0.50 &#226;?&#174;M, respectively (32),                       (26). With exception liver alcohol dehydrogenase (2), <br /> required support growth mvaA null mutant.                           coenzyme Km values 1 orders magnitude lower <br />    Derived sequence similarities conservation known                        NADP(H) NAD(H), suggesting preference for <br /> active-site residues motifs (Fig. 5) suggested mvaA                  NADPH. The HMG-CoA reductases A. fulgidus S. <br /> gene S. aureus encodes class II HMG-CoA reductase (41).                      aureus mere handful oxidoreductases, and <br /> Expression E. coli mvaA-polyhistidine tag construct                      HMG-CoA reductases, Km essentially <br /> and subsequent purification enzymatic characteriza-                       coenzyme. <br /> tion gene product revealed case.                     The Ki inhibition S. aureus HMG-CoA reductase a <br /> Inspection pH profiles catalysis reactions                 statin drug 4 orders magnitude higher Ki for <br /> revealed functional groups pKa values 7                      class I eukaryotic enzyme, suggesting classes can <br /> and 9 protonated states activity. We pro-                    discriminated chemically. The recent publication the <br /> pose His378 S. aureus (His378S) Lys263S                     crystal structure catalytic portion human HMG- <br /> (Fig. 5), apparent cognates active-site residues His381                 CoA reductase (18) confirms large differences architec- <br /> P. mevalonii (His381P) Lys267P P. mevalonii HMG-CoA                        ture active sites human P. mevalonii enzymes. <br />  5152        WILDING ET AL.                                                                                                                                 J. BACTERIOL. <br />  <br />  <br /> Thus, agents selective inhibition class II bacterial                           map pE194, plasmid specifies inducible resistance macrolide, <br /> enzymes achievable. The essential nature en-                                 lincosamide, streptogramin type B antibiotics. J. Bacteriol. 150:804&#226;&#8364;&#8220;814. <br />                                                                                         18.   Istvan, E. S., M. Palnitkar, S. K. Buchannan, J. Deisenhofer. 2000. <br /> zyme suggests class II HMG-CoA reductases repre-                                     Crystal structure catalytic portion human HMG-CoA reductase: <br /> sent promising target antibacterial agents directed                             insights regulation activity catalysis. EMBO J. 19:819&#226;&#8364;&#8220;830. <br /> multidrug-resistant gram-positive cocci.                                                19.   Johnson, E. A., W. A. Schroeder. 1996. Microbial carotenoids. Adv. <br />                                                                                               Biochem. Eng. Biotechnol. 53:119&#226;&#8364;&#8220;178. <br />                                                                                         20.   Jordan-Starck, T. C., V. W. Rodwell. 1989. Pseudomonas mevalonii <br />                            ACKNOWLEDGMENTS                                                    3-hydroxy-3-methylglutaryl-CoA reductase characterization chemical <br />                                                                                               modification. J. Biol. Chem. 264:17913&#226;&#8364;&#8220;17918. <br />    E. Imogen Wilding Dong-Yul Kim contributed equally this <br />                                                                                         21.   Khan, S., R. P. Novick. 1983. Complete nucleotide sequence pT181, <br /> work.                                                                                         tetracycline-resistance plasmid Staphylococcus aureus. Plasmid 10: <br />    The Purdue contribution funded National Institutes                               251&#226;&#8364;&#8220;259. <br /> Health grants HL 47113 (V.W.R.) 52115 (C.V.S.). The SmithKline                      22.   Kim, D.-Y., D. A. Bochar, C. V. Stauffacher, V. W. Rodwell. 1999. <br /> Beecham contribution funded DARPA grant N65236-                                Expression characterization HMG-CoA reductase thermo- <br /> 97-1-5810.                                                                                    philic archaeon Sulfolobus solfataricus. Protein Exp. Purif. 17:435&#226;&#8364;&#8220;442. <br />    From SmithKline Beecham thank Chantal Petit Howard                            23.   Kim, D.-Y., C. V. Stauffacher, V. W. Rodwell. 2000. Expression and <br /> Kallender helpful suggestions manuscript, Christopher                        characterization class II HMG-CoA reductase hyperthermo- <br /> Traini, Thomas Mathie, Stephanie van Horn synthesis                            philic archaeon Archaeoglobus fulgidus. Protein Sci. 9:1226&#226;&#8364;&#8220;1234. <br />                                                                                         24.   Lawrence, C. M., V. W. Rodwell, C. V. Stauffacher. 1995. Crystal struc- <br /> oligonucleotides sequencing plasmid constructs, Nicole                             ture Pseudomonas mevalonii HMG-CoA reductase 3.0 angstrom reso- <br /> Robertson Dean McNulty matrix-assisted laser desorption                               lution. Science 268:1758&#226;&#8364;&#8220;1762. <br /> ionization mass spectral N-terminal sequence analysis.                              25.   Leyland, M. L., D. J. Kelly. 1991. Purification characterization a <br />                                                                                               monomeric isocitrate dehydrogenase dual coenzyme specificity the <br />                                   REFERENCES                                                  photosynthetic bacterium Rhodomicrobium vannielii. Eur. J. Biochem. 202: <br />  1. Alberts, A. W., J. Chen, G. Kuron, V. Hunt, J. Huff, C. Hoffman, J. Rothrock,             85&#226;&#8364;&#8220;93. <br />     M. Lopez, H. Joshua, E. Harris, A. Patchett, R. Monaghan, S. Currie, E.             26.   Maines, M. D., B. V. Polevoda, T. J. Huang, W. K. McCoubrey, Jr. 1996. <br />     Stapley, G. Albers-Schonberg, O. Hensens, J. Hirshfield, K. Hoogsteen, J.                 Human biliverdin IX&#226;?&#163; reductase zinc-metalloprotein. Characterization of <br />     Liesch, J. Springer. 1980. Mevinolin: highly potent competitive inhib-              purified Escherichia coli expressed enzymes. Eur. J. Biochem. 235:372&#226;&#8364;&#8220;381. <br />     itor hydroxymethylglutaryl-coenzyme A reductase cholesterol-low-           27.   Marmur, J. 1961. A procedure isolation deoxyribonucleic acid <br />     ering agent. Proc. Natl. Acad. Sci. USA 77:3957&#226;&#8364;&#8220;3961.                                     micro-organisms. J. Mol. Biol. 3:208&#226;&#8364;&#8220;218. <br />  2. al-Kassim, L. S., C. S. Tsai. 1993. Purification kinetic characteriza-      28.   Maulik, P., S. Ghosh. 1986. NADPH/NADH-dependent cold-labile <br />     tion pickerel liver alcohol dehydrogenase dual coenzyme specificity.              glutamate dehydrogenase Azospirillum brasilense. Purification prop- <br />     Biochem. Cell Biol. 71:421&#226;&#8364;&#8220;426.                                                           erties. Eur. J. Biochem. 155:595&#226;&#8364;&#8220;602. <br />  3. Anderson, B. M., C. D. Anderson. 1995. Purification characteriza-           29.   Meganathan, R. 1996. Biosynthesis isoprenoid quinones menaquinone <br />     tion Azotobacter vinelandii glucose-6-phosphate dehydrogenase: dual co-                (vitamin K2) ubiquinone (coenzyme Q), p. 642&#226;&#8364;&#8220;656. In F. C. Neidhardt, <br />     enzyme specificity. Arch. Biochem. Biophys. 321:94&#226;&#8364;&#8220;100.                                   R. Curtiss III, J. L. Ingraham, E. C. C. Lin, K. B. Low, B. Magasanik, W. S. <br />  4. Beach, M. J., V. W. Rodwell. 1989. Cloning, sequencing, overex-                   Reznikoff, M. Riley, M. Schaechter, H. E. Umbarger (ed.), Escherichia <br />     pression mvaA, encodes Pseudomonas mevalonii 3-hydroxy-3-meth-                   coli Salmonella: cellular molecular biology, 2nd ed., vol. 1. American <br />     ylglutaryl coenzyme A reductase. J. Bacteriol. 171:2994&#226;&#8364;&#8220;3001.                             Society Microbiology, Washington, D.C. <br />  5. Ben-Bassat, A., I. Goldberg. 1980. Purification properties glu-          30.   Neuhauser, W. D., D. Haltrich, K. D. Kulbe, B. Nidetzky. 1997. <br />     cose-6-phosphate dehydrogenase (NADP&#226;&#171;&#185;/NAD&#226;&#171;&#185;) 6-phosphogluco-                           NAD(P)H-dependent aldose reductase xylose-assimilating yeast <br />     nate dehydrogenase (NADP&#226;&#171;&#185;/NAD&#226;&#171;&#185;) methanol-grown Pseudomonas                           Candida tenuis. Isolation, characterization biochemical properties the <br />     C. Biochim. Biophys. Acta 611:1&#226;&#8364;&#8220;10.                                                       enzyme. <br /> </body></html>