<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Jan. 1999, p. 444&#226;&#8364;&#8220;453                                                                                  Vol. 181, No. 2 <br /> 0021-9193/99/$04.0010 <br /> Copyright &#194;&#169; 1999, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />         A Glucan Synthase FKS1 Homolog Cryptococcus neoformans <br />              Is Single Copy Encodes Essential Function <br />                     JOHN R. THOMPSON,1* CAMERON M. DOUGLAS,1 WEILI LI,1 CHONG K. JUE,1&#226;&#8364;&#160; <br />                             BARNALI PRAMANIK,1 XILING YUAN,1 THOMAS H. RUDE,2 <br />                                   DENA L. TOFFALETTI,2 JOHN R. PERFECT,2 <br />                                                             1 <br />                                             AND MYRA KURTZ <br />  <br />                          Infectious Diseases, Merck Research Laboratories, Rahway, New Jersey 07065,1 and <br />                               Department Medicine, Division Infectious Diseases, Duke University <br />                                           Medical Center, Durham, North Carolina 277102 <br />                                            Received 21 August 1998/Accepted 13 November 1998 <br />  <br />              Cryptococcal meningitis fungal infection, caused Cryptococcus neoformans, prevalent im- <br />           munocompromised patient populations. Treatment failures disease emerging clinic, usually <br />           associated long-term treatment existing antifungal agents. The fungal cell wall attractive target <br />           drug therapy syntheses cell wall glucan chitin processes absent mam- <br />           malian cells. Echinocandins comprise class lipopeptide compounds known inhibit 1,3-b-glucan syn- <br />           thesis, compounds belonging class currently clinical trials therapy for <br />           life-threatening fungal infections. Studies Saccharomyces cerevisiae Candida albicans mutants identify the <br />           membrane-spanning subunit glucan synthase, encoded FKS genes, molecular target echino- <br />           candins. In vitro, echinocandins potent antifungal activity Candida Aspergillus species but <br />           potent C. neoformans. In order examine C. neoformans cells susceptible <br />           echinocandin treatment, cloned homolog S. cerevisiae FKS1 C. neoformans. We have <br />           developed generalized method evaluate essentiality genes Cryptococcus applied FKS1 <br />           gene. The method relies homologous integrative transformation plasmid integrate two <br />           orientations, disrupt target gene function. The results analysis suggest that <br />           C. neoformans FKS1 gene essential viability. The C. neoformans FKS1 sequence closely related the <br />           FKS1 sequences fungal species appears single copy C. neoformans. Furthermore, amino <br />           acid residues known critical echinocandin susceptibility Saccharomyces conserved C. <br />           neoformans FKS1 sequence. <br />  <br />  <br />    Cryptococcus neoformans important fungal pathogen                   karyotic pathogens identification enzymes processes <br /> that produces deep-seated infections immunocompromised                    essential pathogens absent sufficiently <br /> patients. After inhalation spores lung, infection                different human host mechanism-based toxicity is <br /> spreads bloodstream brain causes                     unlikely. In context, cell wall synthesis attractive <br /> life-threatening infection central nervous system.                    fungal specific target continuous synthesis cell <br /> With advent AIDS, infection                      wall glucan chitin required viability pro- <br /> prevalent. Even successful treatment patients,                cesses lack homologous counterparts mammalian cells (8, <br /> life-long suppressive therapy fluconazole required (24).             18, 19, 49). Thus, cell wall inhibitors pose low risk mecha- <br /> Of concern significant number treatment failures                   nism-based toxicity. Furthermore, synthesis assembly <br /> observed long-term therapy attributed flucon-                   cell wall components occur vectorially plasma <br /> azole-resistant strains (28, 53). In addition, cases                  membrane periplasmic space (6). Thus, inhibitors <br /> azole-resistant cryptococcal infection contracted dur-             cell wall synthesis need penetrate cell exert <br /> ing long-term fluconazole prophylaxis therapy recently                  effect. <br /> been reported (4). Multiple modes azole resistance                      The glucan component fungal cell wall well <br /> been reported, including direct effects target enzyme                 characterized Saccharomyces Candida. In organ- <br /> 14a-demethylase mechanisms resulting decreased                 isms, glucan accounts 30 60% cell wall is <br /> cellular concentration drug (28, 58). In cases, azole                comprised predominantly 1,3-b-D-glucan percent <br /> resistance concomitant amphotericin B resistance (23,                1,6-b linkages (17). Echinocandin lipopeptides class of <br /> 58). Thus, search new, safe, effective therapies                 compounds known inhibit 1,3-b-D-glucan synthesis (8). Re- <br /> continues.                                                                   cent clinical success treating Candida infections the <br />    The therapeutic challenge developing agents eu-                lipopeptide MK-0991 (50), potent inhibitor 1,3-b-D-glucan <br />                                                                              synthase (GS), makes enzyme especially attractive tar- <br />                                                                              safe, broad-spectrum antifungal drugs. The spectra of <br />   * Corresponding author. Mailing address: Department Infectious          MK-0991 semisynthetic GS inhibitors broad, <br /> Diseases, Merck Research Laboratories, P.O. Box 2000, R80Y-230, <br />                                                                              including wide range Candida spp., Aspergillus spp. (2, 43), <br /> Rahway, NJ 07076-0900. Phone: (732) 594-4766. Fax: (732) 594-1399. <br /> E-mail: john_thompson@merck.com.                                             common pathogenic molds (9). Unfortunately, <br />   &#226;&#8364;&#160; Present address: Department Biological Sciences Institute         C. neoformans notably susceptible known GS <br /> for Biomolecular Structure Function, Hunter College City          inhibitors (1, 22, 27). Understanding reasons poor <br /> University New York, New York, NY 10021.                                  susceptibility C. neoformans strains GS inhibitors im- <br />  <br />                                                                        444 <br />  VOL. 181, 1999                                                   GLUCAN SYNTHASE FKS1 HOMOLOG IN C. NEOFORMANS                                      445 <br />  <br />  <br /> portant order assess potential glucan synthesis        integrative transformation address issue FKS1 essen- <br /> an anticryptococcal target.                                            tiality C. neoformans. Any strategy attempts disrupt <br />    We gained insight function structure GS        potentially essential gene rely alternative mech- <br /> in fungi studies Saccharomyces cerevisiae,           anism viability, plasmidborne copy failure <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br /> now know structural features enzyme            recover marked disruption sporulated diploid, meth- <br /> <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> model organism C. albicans          ods used extensively S. cerevisiae. For developed molec- <br /> Aspergillus enzymes (11, 25, 34). From studies Saccharomy-          ular genetic systems, C. albicans, statistical arguments <br /> ces, know GS heteromeric enzyme complex com-              based exclusive recovery nondisrupting integrations are <br /> prising large, 215-kDa integral membrane pro-          used strongly suggest essentiality (11, 40, 42). In C. neofor- <br /> tein (12, 32) small subunit, Rho1p,              mans, statistical approach complicated fact <br /> loosely associated membrane (15, 31, 45). Fks1p        majority stable integrative transformants ec- <br /> proposed catalytic subunit, Rho1p small GTP-binding           topic (16, 55, 56). Thus, practical requirement strategy <br /> protein regulates activity GS. Rho1p           facile method screen large number transfor- <br /> regulatory roles S. cerevisiae cell cycle (26, 37, 44, 61).     mants integration FKS1 locus. Our results suggest <br /> Recent evidence derived echinocandin cross-linking stud-          FKS1 gene essential growth C. neoformans and <br /> ies Candida suggest additional components           likely GS essential enzyme. The strategy have <br /> be involved (46).                                                      developed assess FKS1 essentiality generally ap- <br />    In Saccharomyces, highly homologous genes encode            plicable study genes C. neoformans. <br /> large integral membrane subunit (FKS1/GSC1/ETG1/CND1/ <br /> CWH53/PBR1/YLR342W FKS2/GSC2/G4074/YGRO32W).                                               MATERIALS AND METHODS <br /> The enzyme complex containing Fks1p constitutes majority              Strains. C. neoformans MY2062 acapsular mutant clinical isolate and <br /> of activity vegetatively growing cells S. cerevisiae   obtained T. Kozel. C. neoformans H99 clinical serotype A isolate <br /> (12, 32). Lipopeptides affect function Fksp compo-          (41), M001 adenine auxotroph derived H99 UV mutagenesis <br />                                                                        (55). Strains grown synthetic complete medium minus adenine (SC-Ade) <br /> nent FKS gene (13, 32). The FKS2 gene product           yeast peptone dextrose adenine medium (YPAD) (51). <br /> needed sporulation (32). Single disruptions FKS1            Genomic DNA isolations. C. neoformans genomic DNA hybridization anal- <br /> or FKS2 viable; however, double disruption lethal         yses prepared DNA isolation protocol Varma Kwong Chung <br /> event, suggesting genes partially function-       (57). This protocol involves digestion cell wall mureinase followed by <br />                                                                        detergent lysis, phenol-chloroform extraction, ethanol precipitation. <br /> ally redundant (32). A FKS homolog Saccharomyces,                A rapid DNA miniprep procedure established order facilitate <br /> discovered result genome sequencing effort, dis-        screening large numbers transformants. Transformants patched or <br /> pensable (47). In Candida albicans, genes (FKS1/GSC1,            replica plated SC-Ade agar grown 24 h. The patch cells was <br /> GSL1, GSL2) related sequence homology Sac-               scraped transferred microcentrifuge tube containing 500 ml cold <br />                                                                        TES buffer (50 mM Tris HCl [pH 7.5], 20 mM EDTA, 1% sodium dodecyl <br /> charomyces Aspergillus FKS genes (34). However, expres-            sulfate [SDS]). Acid-washed glass beads (0.6 g, 0.45-mm diameter) added, <br /> sion mRNA demonstrated C. albicans FKS1                tubes vortexed vigorously 2 min returned ice. Samples were <br /> and GSL1. Evidence Candida genes, FKS1,             incubated 70&#194;&#176;C 10 min returned ice. Potassium acetate solution <br /> indispensable suggests gene predominate Can-          (200 ml, 3 M) added mixed. The resulting precipitate removed by <br />                                                                        centrifugation 10,000 3 g 20 min room temperature. The supernatant <br /> dida implies GSL1 dispensable possesses          (;500 ml) transferred fresh tube, precipitated addition 2 <br /> a different function (11). In Aspergillus fumigatus, single     volumes ethanol, harvested 10,000 3 g 10 min. The nucleic acid <br /> genomic band observed cross-hybridize S. cerevisiae        pellet washed 70% ethanol, dried, dissolved 50 ml TE <br /> FKS1 probe (12), far, single glucan synthase           buffer (10 mM Tris HCl [pH 7.5], 1 mM EDTA) containing 200 mg RNase A <br />                                                                        ml. <br /> gene cloned A. fumigatus (GenBank                    DNA gel electrophoresis hybridization. DNA restriction digested in <br /> accession no. U79728) Aspergillus nidulans (25).                    0.5- 13 KGB buffer (100 mM potassium glutamate, 10 mM Tris acetate [pH <br />    Since expression genes encoding subunits glucan           7.6], 5 mM MgSO4) (33) separated 1% agarose gels using Tris acetate- <br /> synthase shown essential S. cerevisiae          EDTA (TAE) buffer (40 mM Tris acetate [pH 8.3], 1 mM EDTA). Gels were <br />                                                                        capillary blotted charged nylon membranes (Amersham Hybond N1) after <br /> (32) C. albicans (11), lack susceptibility C. neo-       denaturation 15 min 0.5 N NaOH neutralization 15 min 203 <br /> formans inhibitors glucan synthase differ-         SSPE (13 SSPE 0.18 M NaCl, 10 mM NaPO4, 1 mM EDTA [pH 7.7]) <br /> ences cell wall structure species render           (30). Prehybridization performed solution containing 50% deionized <br /> glucan nonessential. In particular, staining 1,3-b-D-         formamide, 53 SSPE, 103 Denhardt&#226;&#8364;&#8482;s solution (30), 100 mg denatured sal- <br />                                                                        mon testes DNA ml, 0.4% SDS. Hybridization conditions identical <br /> glucan-specific fluorochrome aniline blue suggests 1,3-       Denhardt&#226;&#8364;&#8482;s solution reduced 1/10 used prehybrid- <br /> b-D-glucan component Cryptococcus cell walls           ization. Hybridizations performed 42&#194;&#176;C overnight, blots washed <br /> prevalent Candida (36). On hand, fact            twice 23 SSPE&#226;&#8364;&#8220;0.1% SDS room temperature low-stringency condi- <br /> that production C. neoformans protoplasts requires 1,3-b-D-         tions additional washing 0.23 SSPE&#226;&#8364;&#8220;0.1% SDS 42&#194;&#176;C moderate <br />                                                                        stringency. In cases, hybridization performed formamide at <br /> glucanase argues importance glucan stability         65&#194;&#176;C. Radiolabeled probes prepared random oligonucleotide-labeling <br /> of Cryptococcus cell wall (3). Alternatively, Crypto-          procedure(RandomPrimerDNALabelingSystem[GIBCOBRL]orPrimeIt[Strat- <br /> coccus glucan synthase sufficiently divergent          agene]). <br /> other fungal enzymes lacks sensitivity agents             Cloning C. neoformans homolog S. cerevisiae FKS1. Based Southern <br />                                                                        blot analysis MY2062 genomic DNA S. cerevisiae FKS1 coding region <br /> affect enzyme fungal species, polysac-           probe (data shown), 4-kb BamHI-PstI fragment C. neoformans DNA was <br /> charide capsule Cryptococcus impede penetration           targeted cloning. Genomic DNA digested enzymes separated <br /> of GS inhibitors.                                                      agarose gel, fragments size ca. 4 kb purified (Gene Clean <br />    To begin explore potential C. neoformans Fks             [Bio 101]), size-selected library constructed plasmid pGEM3zf <br />                                                                        (Promega). Colony replicas (20) hybridization screened (;3,500 colonies) <br /> protein susceptibility GS inhibitors cell viability,      S. cerevisiae FKS1 DNA consisting 4.1-kb HindIII-PvuII fragment from <br /> have cloned sequenced FKS1 homolog strain H99             pFF133 (12). Positive clones colony purified, plasmids con- <br /> and attempted disrupt gene. Because C. neoformans               tained subjected dideoxy DNA sequencing (Sequenase [Amersham]) <br /> does propagate diploid, demonstration essentiality         PstI vector-insert junction SP6 primer (Promega). <br />                                                                           C. neoformans FKS1 gene sequencing sequence analysis. The entire puta- <br /> by sporulation heterozygous disruption strain, com-            tive C. neoformans FKS1 gene sequenced strands plasmid <br /> monly performed S. cerevisiae, possible C. neo-           pCG2, pCG3, subclones. DNA sequencing performed ABI <br /> formans. We devised strategy using homologous              model 370 sequencer, sequence data assembled using Sequencer (Gene <br />  446       THOMPSON ET AL.                                                                                                                                J. BACTERIOL. <br />  <br />  <br />                 TABLE 1. Integration screening primers                                screening, primers used single PCR products were <br />                                                                                       analyzed agarose gel electrophoresis. Putative targeted integrants, a <br /> Primer                                          Primer sequence                       single PCR product, verified separate reactions primer pairs A59- <br />                                                                                       A39 B59-B39. For PCR analysis, 50-ml reaction mixtures prepared <br />  A59 .....................................59-GCGGGACACCGACCTCGTGAC-39                 including 2 4 ml miniprep genomic DNA template, 50 pmol primer, <br />  A39 .....................................59-TTTGTTCGCCCATACCAGTAC-39                 2.5 U Taq polymerase (GIBCO-BRL), 300 mM deoxynucleo- <br />  B59 .....................................59-CCGGCACGTGGATCAATCGTT-39                 triphosphates, 1.5 mM MgCl2, manufacturer&#226;&#8364;&#8482;s buffer. PCR cycle <br />  B39 .....................................59-CAACAGTCCAAGCGGGGTAGG-39                 conditions 25 cycles, cycle consisting 50 s 94&#194;&#176;C, 50 s 55&#194;&#176;C, <br />                                                                                       210 s 72&#194;&#176;C. After cycle, reaction mixtures incubated an <br />                                                                                       additional 5 min 72&#194;&#176;C held 4&#194;&#176;C loaded 0.8% agarose&#226;&#8364;&#8220;13 TAE <br />                                                                                       gel analysis. <br /> Codes Corporation). For analysis C. neoformans Fks1p open reading                 Glucan synthase enzyme assays. Glucan synthase enzyme assays per- <br /> frame (ORF), used program GAP (Wisconsin Package version 9.1;                  formed crude microsomal preparation. Enzyme activity quantitated by <br /> Genetics Computer Group, Madison, Wis.) perform pairwise alignments            measuring incorporation UDP-glucose (UDPG) trichloroacetic acid <br /> find reading frame strong identity large predicted cytoplasmic        (TCA)-insoluble fraction. Cells grown 0.01 2 optical density units 600 <br /> domain S. cerevisiae Fks1p. A search frameshifts segments poor           nm 30&#194;&#176;C YPAD medium harvested centrifugation washed with <br /> homology extended ORF suggested possible introns, located           50 mM phosphate buffer (pH 7.0) containing 25 mM KF, 1 mM b-mercapto- <br /> potential splice sites using consensus sequences proposed C. neo-      ethanol, 1 mM EDTA. The pellet suspended buffer con- <br /> formans genes (59). To identify exons end C. neoformans FKS1         taining 10 mM [g-S]GTP protease inhibitors (1 mM Pefabloc com- <br /> ORF, predicted protein identity Fksps weakest, used         plete protease inhibitor mixture Boehringer); cells broken, crude <br /> two approaches. First, PCR primers designed genomic sequence                 membranes prepared, protein concentration determined de- <br /> were used amplify fragments (using Taq polymerase Gibco-BRL               scribed previously (13). GS activity measured reaction mixtures containing <br /> the manufacturer&#226;&#8364;&#8482;s reaction conditions) cDNA library constructed          100 mg crude membrane protein, 50 mM Tris (pH 8.5), 10% glycerol, 25 mM <br /> C. neoformans B-3501 (Clonetech). Second, rapid amplification cDNA ends            KF, 0.25% (wt/vol) bovine serum albumin, 20 mM [g-S]GTP, 18 U a-amylase <br /> (RACE) performed using mRNA prepared C. neoformans H99                       (Sigma), UDP-[3H]glucose (2.2 Ci/mmol; Amersham), 1.5 mM unlabeled <br /> (mRNA Isolation Kit; Boehringer Mannheim) Marathon cDNA ampli-                UDPG (60). After 2 h incubation 30&#194;&#176;C gentle agitation, reactions were <br /> fication kit Clontech. The PCR products approaches cloned         terminated ice-cold TCA, radiolabeled product collected and <br /> into vector pCR2.1 (Invitrogen) sequenced. For RNase protection assays         quantitated described previously (13). To measure glucanase susceptibility <br /> the 59 end C. neoformans FKS1, PCR-amplified DNA probe T7                 reaction products, aliquots replicate GS reaction mixtures removed <br /> promoter element added 59 end transcribed vitro using            following synthesis incubated 6 mg laminarinase (Sigma) <br /> Maxiscript Ambion [32P]UTP Amersham. The resulting               ml 50 mM acetate buffer (pH 5.3) 2 h 37&#194;&#176;C. The TCA-insoluble <br /> radiolabeled RNA probe hybridized total RNA isolated log-phase            material collected, washed, counted (13). <br /> cells C. neoformans H99, RNA-RNA hybrid subjected single-              RNA analysis. For total cellular RNA isolation, RNeasy kit (Qiagen) was <br /> strand RNase digestion, separated 6% acrylamide gel containing 8 M urea,         employed. Cells grown 50 ml YPAD 30&#194;&#176;C mid-log phase har- <br /> and visualized autoradiography.                                                    vested centrifugation (3,000 3 g, 5&#194;&#176;C). All subsequent operations per- <br />    Translation predicted coding sequence produces protein 1,724           formed ice. Cells washed 1 ml cold TE plus 0.1% diethylpyro- <br /> amino acids molecular mass 196,636 daltons. Multiple protein sequence       carbonate (DEPC) transferred microcentrifuge tube. Cell pellets were <br /> alignments Fks proteins fungal species produced             suspended 350 ml Qiagen RLT buffer b-mercaptoethanol (Qiagen) and <br /> ClustalW v1.74 (54), parsimony tree produced PAUP 3.1.1.               0.1% DEPC. An equal volume acid-washed 0.45-mm-diameter glass beads was <br /> Treeview 1.4 used display resulting parsimony tree (39). GAP           added. Cells disrupted 2 cycles 30 seconds minibead beater <br /> used produce pairwise alignments similarity scores.                            (Biospec Products) 30 s ice disruption cycles. The lysate was <br />    Integrative disruption plasmid construction. For PCR screening strategy        transferred fresh tube, glass beads washed additional 350 <br /> (described below), required PCR priming sites near genomic FKS1 gene           ml RLT buffer, wash combined lysate. The pooled lysate <br /> that contained plasmid sequence. To accomplish this, ends       subsequently treated according manufacturer&#226;&#8364;&#8482;s protocol fresh <br /> the 6-kb PstI fragment pCG2 sequenced using pUC18 forward          0.1% DEPC added buffers. Final RNA pellets dissolved 50 ml <br /> reverse primers, fragment slightly smaller 6-kb PstI fragment      TE plus 0.1% DEPC. <br /> subcloned PCR, creating pCG4. The following primers used clone            RNA separated 1% agarose gels containing 13 MOPS buffer [20 mM <br /> fragment pCG4: 59-AAGGAAAAAAGCGGCCGCCAGCCAAATAGTTTC                                3-(N-morpholino)propanesulfonic acid, sodium salt, 5 mM sodium acetate, <br /> TTCTCTGCC-39 59-AAAGGAAAAAAGCGGCCGCCACACTGGTGAT                                   1 mM EDTA (pH 7.5)] 7% formaldehyde. The gels soaked 203 <br /> ATTCGGAATGC-39. Each primer consists 10- 11-bp spacer, NotI site,           SSPE capillary transferred charged nylon membranes (Amersham Hy- <br /> and 24 23 nucleotides (nt) C. neoformans FKS1 sequence. The primers        bond N1) (30). Hybridization conditions identical used South- <br /> used conjunction Expand polymerase (Boehringer Mannheim) buffer           ern DNA analysis. <br /> conditions recommended manufacturer amplify fragment                  Nucleotide sequence accession number. The genomic sequence C. neo- <br /> pCG2 subcloning. The amplification program 120 s 92&#194;&#176;C followed          formans FKS1 gene submitted GenBank (accession no. AF102882). <br /> 25 cycles, cycle consisting 10 s 92&#194;&#176;C, 30 s 50&#194;&#176;C, 18 min at <br /> 68&#194;&#176;C. The PCR product digested NotI enzyme (GIBCO-BRL), purified <br /> from 1% agarose&#226;&#8364;&#8220;13 TAE gels using Gene Clean (Bio 101), ligated                                                RESULTS <br /> NotI-digested pGEM5zf (Promega). <br />    pWL40 constructed digestion pCG4 XhoI XbaI followed by <br /> insertion annealed oligonucleotides (59-TCGATCCCGGGTCGCTCGG <br />                                                                                          Cryptococcus glucan synthase cloning. A Saccharomyces FKS1 <br /> TACCTCGCTAT-39 59-CTAGATAGCGAGGTACCGAGCGACCCGGG                                   probe encompassing portion coding region previ- <br /> A-39). The annealed oligonucleotides introduced unique KpnI, XmaI, XbaI           ously shown hybridize genomic DNA C. neo- <br /> sites site XhoI-XbaI deletion. Deletion 1.5-kb segment DNA   formans MY2062 (12) provided strategy clone the <br /> removes 493 bp FKS1 promoter, 312 codons introns. <br />    The final integration plasmid, pWL42, constructed insertion 3-kb <br />                                                                                       putative C. neoformans FKS1 homolog. Based additional hy- <br /> KpnI-XmaI fragment, plasmid pCnade2DApa, containing cloned Cryp-             bridization analysis Saccharomyces FKS1 gene probe <br /> tococcus ADE2 gene strain B-3501 (55). pWL42 contains ADE2              (data shown), 4-kb BamHI-PstI C. neoformans genomic <br /> marker flanked fragments C. neoformans FKS1 locus. The ADE2             fragment strain MY2062 targeted cloning (see <br /> marker separates 1,206-bp region 59 C. neoformans FKS1 ORF (positions <br /> 21699 2493 respect start codon) 2,478 bp FKS1 coding <br />                                                                                       Materials Methods). Six putative clones isolated, the <br /> sequence downstream (codons 313 1136) (see Fig. 1).                                ends clone sequenced, translated protein <br />    Biolistic transformation. Strain M001 transformed plasmid pWL42           sequences clones (pCG1-2 pCG1-6) <br /> by biolistic transformation procedure (55), adenine prototrophs          aligned Saccharomyces FKS1 protein sequence. <br /> selected plating transformations SC-Ade medium incubating 30&#194;&#176;C <br /> for days. <br />                                                                                       The short stretch amino acid homology provided first <br />    PCR screening targeted integrants. The primers employed identify            verification cloned C. neoformans FKS1 ho- <br /> targeted integrants reveal orientation listed Table 1. All PCR       molog. The location homologous region Saccha- <br /> primers purchased GIBCO-BRL. Primer pair A59-A39 yields                   romyces FKS1 protein sequence size cloned in- <br /> 3,371-bp product (band A) primer pair B59-B39 yields 2,046-bp product <br /> (band B) wild-type C. neoformans H99 genomic DNA used the <br />                                                                                       sert suggested BamHI-PstI fragment contained DNA <br /> template. Targeted integration plasmid pWL42 C. neoformans FKS1           representing amino-terminal half C. neoformans <br /> locus predicted eliminate PCR products (Fig. 1). For initial     FKS1 gene. <br />  VOL. 181, 1999                                                                GLUCAN SYNTHASE FKS1 HOMOLOG IN C. NEOFORMANS                                          447 <br />  <br />  <br />  <br />  <br />    FIG. 1. Integration pWL42 homologous recombination C. neoformans FKS1 (CnFKS1) locus. Two potential homologous integration events are <br /> depicted. In panels, plasmid pWL42 shown circle ADE2 gene (stippled box) inserted cloned fragment CnFKS1 locus (cross-hatched <br /> boxes). The thicker cross-hatched regions linear maps correspond CnFKS1 DNA segments present plasmid. Thick lines arrowheads vertical <br /> bars ends direction transcription approximate size CnFKS1 reading frame. The vertical bars indicate position 59 39 truncation, <br /> and broken lines indicate promoterless reading frames. The ADE2 insertion transcribed direction opposite CnFKS1. In panel A, pWL42 shown <br /> integrating CnFKS1 locus single crossover event (3) cloned 59 region upstream CnFKS1 ORF. The integrated plasmid (bracket) <br /> creates interrupted copy (CnFKS1 59, 39 D) CnFKS1, intact copy ORF remains downstream site integration. The striped box the <br /> intact ORF illustrates approximate size position PCR product obtained A-type integration, wild-type, ectopic integration B-type integration. <br /> In panel B, disrupting integration pWL42 depicted, copies CnFKS1 truncated 39 end (CnFKS1 39 D) 59 end (CnFKS1 59 D) are <br /> created. The striped box illustrates PCR product specific disrupting wild-type ectopic pWL42 integrants A-type integration. <br />  <br />  <br />  <br />  <br />    In C. neoformans, highest reported frequency homol-                           gested Lodge et al. (29), namely, cloning targeted gene <br /> ogous integrative transformation observed                                 genetic background employing selectable <br /> strains derived H99 strain background biolistic                             marker derived different serotype. We need- <br /> transformation. Toffaletti et al. observed frequency 19                         ed isolate additional FKS1 genomic fragments derived from <br /> 22% homologous integration ADE2-containing plas-                              train H99. Based additional Southern analysis H99 DNA <br /> mid genomic ade2 site (55). In experiment                                4-kb HindIII-PstI MY2062 fragment probe <br /> closely analogous situation, Lodge et al. used ADE2                           (data shown), 6-kb PstI fragment 9.4-kb HindIII <br /> a selectable marker homologous integration NMT                               fragment targeted cloning size-selected librar- <br /> locus observed targeted frequency 3% (29). Thus,                           ies constructed pUC18. Restriction mapping (not shown) <br /> majority transformants unstable ectopic. Our <br />                                                                                         suggested 9.4-kb HindIII fragment partially over- <br /> strategy demonstrating essentiality requires isolation of <br />                                                                                         lapped PstI fragment encompass 39 end of <br /> enough homologous integrants yield statistically significant <br /> result. In order maximize potential homologous in-                           gene. Colony screening using 4-kb BamHI-PstI frag- <br /> tegration FKS1 locus, adopted precautions sug-                            ment pCG1-2 probe resulted isolation two <br />  448      THOMPSON ET AL.                                                                                                                         J. BACTERIOL. <br />  <br />  <br />                                                                                   bicans GSL1) formed second distinct grouping. Other clus- <br />                                                                                   ters formed duplicated S. pombe proteins and <br />                                                                                   derived Aspergillus species. <br />                                                                                      Pairwise alignments C. neoformans Fks1p the <br />                                                                                   individual glucan synthase proteins performed <br />                                                                                   provide similarity scores C. neoformans Fks1p and <br />                                                                                   related genes species. The pairwise identity <br />                                                                                   scores pair pathogenic species, C. neo- <br />                                                                                   formans, C. albicans, A. fumigatus, 57 64%. <br />                                                                                      Is putative Cryptococcus FKS1 gene single copy? A ge- <br />                                                                                   nomic Southern blot analysis performed determine if <br />                                                                                   C. neoformans FKS1 gene single copy. The 4-kb BamHI- <br />                                                                                   PstI fragment pCG2 used probe Southern blot of <br />                                                                                   H99 genomic DNA individually digested restriction <br />                                                                                   enzymes (Fig. 3). Employing low-stringency wash conditions, <br />                                                                                   restriction digestions produced single hybridiz- <br />    FIG. 2. Glucan synthase parsimony tree. Abbreviations: SpFKSx1, Schizo-        ing band. Thus, single genomic C. neoformans FKS locus <br /> saccharomyces pombe FKS chromosome 1 (accession no. Z98601); SpFKSx2,             detectable low-stringency hybridization. <br /> S. pombe FKS chromosome 2 (accession no. AL021839); AfFKS, Aspergillus               Is Cryptococcus FKS1 gene essential? An integration <br /> fumigatus FKS (accession no. U79728); AnFKS, Aspergillus nidulans FKS (ac-        plasmid, pWL42, constructed integration ho- <br /> cession no. U51272); CaFKS1, Candida albicans FKS1 (accession no. D88815); <br /> CaGSL1, C. albicans GSL1 (accession no. D88816); CaGSL2, C. albicans GSL2         mologous recombination FKS1 locus occur two <br /> (accession no. AB001077); ScFKS1, Saccharomyces cerevisiae FKS1 (accession        possible orientations. This plasmid contains regions ge- <br /> no. S50235); ScFKS2, S. cerevisiae FKS2 (accession no. S50240); ScFKS3, S. cer-   nomic FKS1 sequence separated ADE2 selectable mark- <br /> evisiae FKS3 (accession no. S53976). (All accession numbers GenBank    er. Integration homologous recombination of <br /> for PIR.) The numbers parentheses percent <br /> identity C. neoformans Fks1p based GCG GAP alignments.                    ADE2 marker leaves intact FKS1 gene, integration <br />                                                                                   results loss function FKS1 gene <br />                                                                                   (Fig. 1). If gene nonessential, recovery integrative <br />                                                                                   transformants orientations possible. Con- <br /> H99 clones; pCG2 contains 6-kb PstI fragment, pCG3                        versely, FKS1 gene encodes essential function, then <br /> contains 9.4-kb HindIII fragment.                                             nondisrupting integration events recovered. Using <br />    C. neoformans FKS1 gene sequence analysis. We determined                       strategy, isolation sufficient number nondisrupting <br /> the sequence 8,023 bp genomic DNA C. neoformans                        transformants isolation disrupting transformants <br /> H99 encompasses FKS1 promoter, ORF inter-                             provides statistical argument gene question is <br /> rupted seven introns, likely transcription termi-                      essential mitotic growth. <br /> nator. At amino carboxy termini, C. neoformans                           Because large number transformants examined <br /> Fks1p shorter Fksp family members roughly 70                     approach, PCR strategy designed facilitate <br /> and 40 residues, respectively (data shown). In analyzing                  identification homologous integrants define <br /> 59 end C. neoformans FKS1 RNase protection, 45 nt                      orientation. Two pairs primers used implement <br /> digested 258-nt probe spanning bases 1805 2063,                         strategy. The key design consideration primer <br /> revealing presence intron (bases 2013 2058).                  pairs primer pair targets FKS1 genomic <br /> Therefore, predicted protein start site position 1940                   sequence contained integration plasmid. The other <br /> FKS1 furthest upstream AUG codon frame                          primer pair targets FKS1 genomic DNA present in <br /> within exon preceding intron. As 39 end,                 integration plasmid. Thus, primer pairs amplify <br /> UAA stop codon position 7468 exon es-                       respective targets normal genomic FKS1 locus, <br /> tablished RACE analysis cDNA sequence. A hydrop-                        yielding different sized products single reaction. How- <br /> athy analysis (52) C. neoformans Fks1 protein reveals <br /> that, like members Fksp family, protein at <br /> least 12 transmembrane helices, amino terminus and <br /> large central hydrophilic region predicted the <br /> cytoplasmic face plasma membrane (data shown). <br /> The highest degree identity Fks proteins within <br /> the central region, genomic sequence encoding this <br /> portion C. neoformans Fks1p interrupted in- <br /> trons. Like Fks proteins, Fks1p C. neofor- <br /> mans does contain proposed UDP-glucose binding <br /> motif QXXRW, does region limited homol- <br /> ogy BcsAp, catalytic subunit cellulose synthase from <br /> Acetobacter xylinium (25). <br />    The deduced C. neoformans FKS1 protein sequence was <br /> aligned 10 fungal glucan synthase sequences derived <br /> from Saccharomyces, Aspergillus, Candida, Schizosaccha- <br /> romyces species. The multiple alignment used construct                        FIG. 3. Cryptococcus FKS Southern blot. Each lane contains 10 mg restric- <br /> the parsimony tree shown Fig. 2. Interestingly, essential                  tion enzyme-digested H99 genomic DNA. The enzymes BamHI (lane 1), <br />                                                                                   PstI (lane 2), NcoI (lane 3), SmaI (lane 4). Hybridization probe 32P- <br /> Saccharomyces genes, FKS1 FKS2, C. albi-                        labeled random-primed 4-kb BamHI-PstI H99 genomic fragment derived from <br /> cans FKS1, formed cluster, FKS homologs                       pCG2 encompasses 1,221 codons (of 1,643) C. neoformans <br /> function elucidated (S. cerevisiae FKS3 C. al-                   FKS1 sequence. Arrows indicate positions HindIII-digested lambda standards. <br />  VOL. 181, 1999                                                               GLUCAN SYNTHASE FKS1 HOMOLOG IN C. NEOFORMANS                       449 <br />  <br />  <br />                                                                                     similar depicted Fig. 5 (data shown). The results <br />                                                                                     established FKS1 locus strains wild <br />                                                                                     type, did match patterns expected for <br />                                                                                     A-type B-type integrations depicted Fig. 1. Thus, a <br />                                                                                     complicated rearrangement FKS1 locus appears to <br />                                                                                     occurred transformants. Further analysis these <br />                                                                                     integrants required ascertain FKS1 expression <br />                                                                                     status transformants. <br />                                                                                        Expression biochemical analysis indeterminate tar- <br />                                                                                     geted integrants. To evaluate expression status three <br />                                                                                     targeted integrants indeterminate genomic structure, GS <br />                                                                                     enzyme activity membranes prepared strains was <br />                                                                                     measured strains analyzed expression of <br />                                                                                     FKS1 mRNA. Membranes transformants con- <br />    FIG. 4. PCR screening profiles. The left panel depicts PCR <span id='am-160' about='xsp:pattern' typeof='owl:Thing'>pattern</span>           tained active GS enzyme measured incorporation of <br /> expected wild-type (WT), A-type (A), B-type (B) integration             glucose UDP-glucose TCA-insoluble product <br /> pWL42. The right gels actual screening examples wild-type control       (Table 2). We showed percentage glucanase- <br /> (WT) A-type integration (A). A B-type integration obtained. The <br /> sizes A- B-specific PCR products 3,371 2,046 bp, respectively.   sensitive product unchanged, confirming TCA- <br />                                                                                     insoluble material unchanged consists predomi- <br />                                                                                     nantly 1,3-b-glucan (Table 2). Similarly, Northern blot <br />                                                                                     analysis detected mRNA band identical size wild- <br /> ever, PCR products eliminated                          type FKS1 message (Fig. 6). <br /> homologous integration plasmid pWL42. The missing prod- <br /> uct reveals orientation integration event. Ectopic <br /> integrants disturb FKS1 locus                                                DISCUSSION <br /> produce PCR bands characteristic wild-type struc- <br /> ture FKS1 locus.                                                                An understanding C. neoformans sus- <br />    Strain M001 transformed pWL42 biolistic                          ceptible known GS inhibitors help evaluate the <br /> method. Targeted integration events orientation in-                      importance GS broad-spectrum antifungal target and <br /> tegration evaluated PCR analysis miniprep genomic                        help focus future drug discovery efforts. We decided to <br /> DNA samples prepared transformants. A total 357                         clone C. neoformans FKS1 homolog means evaluate <br /> transformants screened PCR procedure. This anal-                        glucan synthase required viability C. neofor- <br /> ysis identified 331 ectopic integrants. Twenty-six isolates (7.9%)                  mans compare sequence C. neoformans Fks <br /> produced PCR profile characteristic homologous integra-                        protein fungal enzymes. At issue whether <br /> tion FKS1 locus. Twenty-three isolates displayed                    reduced susceptibility shown Cryptococcus GS inhib- <br /> a PCR pattern expected nondisrupting integration,                         itors attributed divergence Cryptococcus glucan <br /> none transformants consistently displayed PCR pro-                       synthase respect pathogenic fungi the <br /> file expected disrupting integration event. Three                      <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-6' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-7' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-11' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-19' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-30' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-37' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-38' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-46' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-53' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-76' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span>phenotype</span> factors, unique features in <br /> transformants (0.85%) gave inconsistent results                          cell wall structure Cryptococcus make cells <br /> PCR screening remained indeterminate stage. Fig-                        dependent cell wall viability. <br /> ure 4 shows typical patterns obtained PCR wild-                          We isolated C. neoformans gene cross-hybridiza- <br /> type strain A-type, nondisrupting integrants.                          tion S. cerevisiae FKS1. Several lines evidence argue that <br /> Conditions equally amplify primer pairs mixed                          gene functional FKS homolog C. neoformans. The <br /> reaction obtained. However, separate tubes,                         deduced amino acid sequence 58% identical S. cere- <br /> the individual primer pairs yielded bands similar intensity.                     visiae FKS1 C. albicans FKS1, overall <span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span>, <br /> For reason, presumptive integrative transformants                          1,643 amino acids, similar fungal glucan <br /> identified missing band mixed PCR subse-                           synthases. Low-stringency genomic Southern analysis with <br /> quently tested separate PCR primer pair.                              C. neoformans FKS1 hybridization probe failed evi- <br />    To confirm PCR screening results, genomic Southern                           dence additional copies. The hybridization conditions em- <br /> hybridization analysis performed. The genomic structures                        ployed analysis identical conditions used to <br /> of 23 A-type, nondisrupting homologous integration candi-                       detect cross-hybridization S. cerevisiae FKS1 and <br /> dates corroborated Southern blot analysis. Within                       C. neoformans FKS1. Thus, conservatively conclude <br /> this group, 11 confirmed single integration events,                     additional related sequences present C. neofor- <br /> 12 showed pattern consistent homologous A-type inte-                         mans, bear 58% identity shared by <br /> gration C. neoformans FKS1 plus ectopic integration                      S. cerevisiae FKS1 C. neoformans FKS1. Finally, results <br /> second genomic site. Examples single (strain 42-1-28)                       Northern hybridization analysis confirm cloned <br /> and multiple (strain 42-1-3) nondisrupting integrations                         C. neoformans FKS1 gene expressed. <br /> shown Southern blot Fig. 5. The approximately 4-                         In terms genomic organization FKS genes, C. neofor- <br /> 3.5-kb bands lanes 1 Fig. 5A B provide evidence                       mans appears similar Aspergillus species, also <br /> a second ectopically integrated copy plasmid strain                       appear encode FKS single gene (25). The C. neofor- <br /> 42-1-3. In lanes 1 3 Fig. 5D, band 9 kb originates                  mans Fks1 protein sequence shares highest sequence <br /> the endogenous ade2 locus. All bands lanes 1 2                         identity Aspergillus Fks proteins (Fig. 2). <br /> (Fig. 5A D) consistent A-type nondisrupting                            Unequivocal demonstration gene essentially routinely <br /> integration.                                                                        performed S. cerevisiae introducing genetically marked <br />    The indeterminate integrants analyzed                         disruption diploid showing haploid segregants <br /> genomic Southern hybridization probes digestions                           carrying disruption inviable. While C. neoformans pos- <br />  450      THOMPSON ET AL.                                                                                                                            J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 5. Southern blot analysis nondisrupting integrants. Genomic DNA isolated pWL42 transformants (lane 1, strain 42-1-3; lane 2, strain 42-1-28) and <br /> from strain H99 (lane 3) digested PstI (panels A, B, C) XmaI ClaI (panel D), analyzed separate Southern blots. The probes the <br /> blots follows: panel A, 1.5-kb NotI-XmaI CnFKS1 fragment pCG4; panel B, 3.0-kb pGEM5zf vector linear NotI digestion; for <br /> panel C, 0.9-kb XhoI-XhoI CnFKS1 fragment pCG2 represents region FKS1 replaced ADE2 pWL42; panel D, 3.0-kb <br /> KpnI-XmaI CnADE2 fragment pCnADE2DApa. Arrows panels A D denote migration positions HindIII fragments lambda (from top <br /> to bottom, 23.1, 9.4, 6.6, 4.3, 2.3, 2.0 kb). Maps predicted CnFKS1 locus following ectopic, nondisrupting, disrupting integration presented below <br /> panels A D. Black bars the map positions probe fragments disrupted pWL42 integrant map. Features maps (ORFs, <br /> plasmid DNA, regions CnFKS1 contained pWL42) described legend Fig. 1. The brackets map indicate sizes positions of <br /> fragments generated PstI digestion; fragments produced digestion XmaI ClaI shown below. A bar indicating scale shown right. <br />  <br />  <br />  <br />  <br /> sesses sexual cycle, diploid phase cultured. As                      cation coding region insertion selectable <br /> a result, strategies evaluating gene essentiality                     marker remaining coding region. Additionally, con- <br /> be developed organism.                                                      struct contain 59- 39-flanking sequence that <br />    We devised strategy assess essentiality re-                        resulting plasmid integrate genome single <br /> quires sufficient sequence information support con-                          crossover event 59- 39-flanking region. Integra- <br /> struction appropriate integration plasmid define                           tion orientation results gene duplication. In one <br /> suitable PCR primers rapidly identify homologous inte-                            orientation, defects remain copy, pre- <br /> grants. Our strategy involves construction plasmid                     serving intact copy gene. However, integration in <br /> integrate homologous recombination orientations,                           orientation distributes defect re- <br /> only disrupt gene function. This approach                          sulting copies disrupts function gene <br /> involves introduction defects gene, trun-                      copies. Thus demonstration essentiality derives the <br />  VOL. 181, 1999                                                             GLUCAN SYNTHASE FKS1 HOMOLOG IN C. NEOFORMANS                                       451 <br />  <br />  <br />                 TABLE 2. GS specific activities vitro                          parent full-length mRNA FKS1 gene ex- <br />                                                                                   pressed. Glucan synthase activity detected three <br />                                        GS sp acta              % Glucanase- <br />  Strain          Genotype <br />                                    (nmol z mg21 z h21)       sensitive productb   integrants FKS1 expression status question, <br />                                                                                   GS specific activity comparable the <br /> H99          Wild type                 11.4 6 0.7                   69            A-type, nondisrupting integrants analyzed. There- <br /> 42-2-13      A type                     5.6 6 0.6                   83            fore, transformants resulting genomic rearrange- <br /> 42-1-28      A type                     9.6 6 5                     75            ments FKS1 locus functionally disrupted do <br /> 42-1-43      Indeterminate              5.1 6 0.4                   78 <br /> 42-1-48      Indeterminate              6.8 6 0.6                   80 <br />                                                                                   impact conclusion FKS1 gene essential. The <br /> 42-6-60      Indeterminate              7.0 6 0.3                   77            minor (twofold less) quantitative variations observed <br />                                                                                   GS specific activity considered significant. <br />    a <br />      Reactions determination GS specific activity contained 15 mM UDPG        We produced direct evidence GS activity <br /> at 1,000 dpm/nmol. Each value average 6 standard deviation values <br /> from assays. <br />                                                                                   measured vitro derived expression FKS1 gene. <br />    b <br />      Glucanase susceptibility products measured using final concentra-   However, alternative possibility gene, unre- <br /> tion 1.5 mM UDPG 20,000 dpm/nmol.                                           lated known glucan synthases, encodes GS C. neoformans <br />                                                                                   appears unlikely. This require FKS1 gene re- <br />                                                                                   ported provide essential function distinct GS <br /> exclusive recovery integrations nondisrupting orien-                    activity. No bifunctional character attributed to <br /> tation.                                                                           fungal Fks proteins, deduced protein se- <br />    To date, attempts evaluate gene essentiality                     quence C. neoformans Fks1p aligns reasonably with <br /> in Cryptococcus reported. Lodge et al. exploited                      Fks proteins nearly entire length. Given the <br /> temperature-sensitive mutant assess essentiality gene                   high level homology C. neoformans FKS1 gene with <br /> for N-myristoyl transferase (29). They used plasmid construc-                   fungi single-copy status C. neoformans, <br /> tion that, homologous integration, resulted replacement                    logical assume gene providing subunit <br /> of wild-type allele well-characterized temperature-                    required GS activity. Formally, however, exclude <br /> sensitive (ts) <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-112' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-114' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-119' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-157' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span>mutation</span>. Subsequent demonstration loss                      possibility C. neoformans FKS1 gene es- <br /> viability restrictive temperature provided convincing ev-                  sential reasons providing subunit glucan <br /> idence essentiality N-myristoyl transferase. A limi-                   synthase complex. <br /> tation approach, however, need obtain suit-                      If GS essential, C. neoformans cells less <br /> able ts allele demonstration essentiality                    susceptible fungi known GS inhibitors? The <br /> performed. Construction characterization ts alleles                   availability C. neoformans FKS1 sequence gives the <br /> generally follows initial sequencing new gene. There-                    opportunity compare sequence FKS sequences <br /> fore, strategy, relies solely sequence information,                  fungi insight possible differences might <br /> should generally useful applicable earlier stage                  account altered susceptibility known GS inhibitors. <br /> the analysis new gene ts strategy employed                       The primary amino acid sequence C. neoformans Fks1p <br /> Lodge et al.                                                                      shares 58 62% identity, respectively, homologous <br />    Recently, evaluated essentiality topoisomerase                   proteins fungal pathogens, C. albicans FKS1 <br /> I gene showing homologous integrants disruption                      A. fumigatus FKS. The amino acid identity C. al- <br /> plasmid recovered second ectopically inte-                     bicans FKS1 A. fumigatus FKS 64%; thus, degree of <br /> grated functional gene copy present (10). However,                       Fks protein sequence conservation similar pair- <br /> strategy cumbersome required complicated                           wise combination pathogens. <br /> series molecular manipulations proof essential                         In S. cerevisiae, echinocandin-resistant mutations have <br /> feature.                                                                          mapped single amino acid changes FKS genes a <br />    Using strategy, argument essentiality statistical               region spanning amino acids (14). Using multiple se- <br /> in nature depends isolation sufficient number                      quence alignment, closely examined C. neoformans <br /> homologous integrations nondisrupting orientation                       FKS1 sequence important region. Table 3 shows the <br /> order able state confidence disrupting                     sequence region known essential Fks proteins <br /> orientation recovered gene es- <br /> sential. We identified 23 homologous recombination events in <br /> the nondisrupting orientation, integrations dis- <br /> rupting orientation recovered. Assuming equal chance <br /> of recombination orientation, probability that <br /> only orientation isolated 23 attempts (0.5)23 <br /> or 1.19 3 1027. Thus, statistical argument essentiality <br /> of FKS1 gene quite strong. This argument relies as- <br /> sumption roughly equal probability insertion of <br /> the possible orientations. In integration plasmid con- <br /> struct, introduced larger segment flanking sequence <br /> (2,871 bp versus 1,204 bp) recombination <br /> would result gene disruption. Thus, length considerations <br /> introduce bias, favor recovery disrupting integra- <br /> tions. <br />    Three additional homologous integrants C. neofor-                          FIG. 6. CnFKS1 Northern analysis. Lane 1, H99 parent strain; lane 2, 42-1- <br /> mans FKS1 locus contained complicated genomic rear-                        28; lane 3, 42-2-13; lane 4, 42-1-43; lane 5, 42-1-48; lane 6, 42-6-60. Lane 2 and <br />                                                                                   3 A-type, nondisrupting integrations. Lanes 4 6 contain strains with <br /> rangement near FKS1 locus identified. Subse-                       putative genomic rearrangements. The hybridization probe 32P-labeled ran- <br /> quent analysis mRNA expression transformants                          dom-primed 4-kb BamHI-PstI H99 genomic fragment derived pCG2. The <br /> confirmed that, despite genomic rearrangements, ap-                        arrow indicates position ;9-kb FKS mRNA band. <br />  452      THOMPSON ET AL.                                                                                                                                  J. BACTERIOL. <br />  <br />  <br />  TABLE 3. Multiple alignment region known Echr mutations                         quence identities, evolutionary relationships established by <br /> Sequence species     Protein                Amino acid sequence     b               parsimony analysis, detailed examination sequence <br />                                                                                        region known mutations confer echinocandin re- <br />   Consensus  a <br />                                    Phe Leu Thr Leu Ser Leu Arg Asp                     sistance. None analyses support notion the <br />                                                                                        C. neoformans GS enzyme susceptible GS in- <br />   C. neoformans        Fks1p       Phe   Leu   Thr   Leu   Ser   Phe       Arg   Asp <br />   S. cerevisiae        Fks1p       Phe   Leu   Val   Leu   Ser   Leu       Arg   Asp <br />                                                                                        hibitors different fungi. However, dif- <br />   S. cerevisiae        Fks2p       Phe   Leu   Ile   Leu   Ser   Leu       Arg   Asp   ferences intrinsic sensitivity C. neoformans GS could <br />   C. albicans          CaFksp      Phe   Leu   Thr   Leu   Ser   Leu       Arg   Asp   changes discern sequence analysis. <br />   A. fumigatus         AfFksp      Phe   Leu   Thr   Leu   Ser   Phe       Arg   Asp   Supporting notion, Williamson et al. (60) recently <br />   A. nidulans          AnFksp      Phe   Leu   Thr   Leu   Ser   Ile       Arg   Asp <br />                                                                                        shown echinocandins orders magnitude less <br />   S. cerevisiae        Fks1-2p     Ile Leu Val Leu Ser Leu Arg Asp                     potent Cryptococcus glucan synthase vitro than <br />   S. cerevisiae        Fks1-3p     Phe Leu Val Leu Ser Leu Arg Tyr                     enzyme derived Candida (5). Conversely, notion <br />   S. cerevisiae        Fks2-1p     Phe Leu Lys Leu Ser Leu Arg Asp                     capsule contribute relative lack sus- <br />   a <br />     The consensus represents prevalent residue known wild- <br />                                                                                        ceptibility Cryptococcus appears unlikely. Our acapsular <br /> type sequences.                                                                        C. neoformans strain, MY2062, shows susceptibility the <br />   b <br />     Wild-type deviations consensus sequence shown bold type.           echinocandin L-733,560 (MIC 5 16 mg/ml) consistent <br /> The sequences known echinocandin-resistant mutations listed        values determined numerous capsular clinical isolates <br /> bottom substituted residues underlined. <br />                                                                                        (range, 16 32 mg/ml) (2). The genetic data described this <br />                                                                                        work suggest that, despite lower content glucan the <br />                                                                                        Cryptococcus cell wall, glucan synthesis required viability <br /> from S. cerevisiae C. albicans, sole sequences                      Cryptococcus. Thus, data reinforce potential utility <br /> derived Aspergillus species C. neoformans. Six                         glucan synthesis inhibition broad-spectrum antifungal <br /> eight residues region absolutely conserved                      target. Overall, study biochemical differences <br /> FKS sequences. The residues specific substitutions                         fungal glucan synthases appears likely provide <br /> can result echinocandin resistance conserved                         information useful developing broad-spectrum <br /> C. neoformans sequence. The C. neoformans sequence differs                             antifungal agents targeting glucan synthesis. <br /> from consensus eight-residue region position <br /> 6 Phe replaces Leu. A. fumigatus Phe                                                      ACKNOWLEDGMENTS <br /> position. To test possible significance deviation from <br /> the consensus, constructed Leu-to-Phe substitution                           We thank Joseph A. Borkowski helpful advice guidance with <br />                                                                                        multiple alignments parsimony analysis. We thank Joanne <br /> homologous position S. cerevisiae FKS1 sequence. The <br />                                                                                        Williamson Arthur Ram sharing unpublished results. Jean <br /> substitution effect echinocandin sensitivity                          Marrinan provided technical assistance minipreps M001 <br /> either cells enzyme activity vitro (data shown).                       (pWL42) transformants. We thank Forrest Foor gift pFF133. <br />    We used multiple protein sequence alignment                         We thank Jan Onishi Jennifer Nielsen-Kahn helpful dis- <br /> known Fks proteins construct unrooted parsimony tree                            cussions. <br /> in Fig. 2. The C. neoformans Aspergillus proteins occupy a <br /> central position tree, bracketed Schizosaccharo-                                                          REFERENCES <br /> myces pombe proteins Saccharomyces                              1. Bartizal, K., G. Abruzzo, C. Trainor, D. Krupa, K. Nollstadt, D. Schmatz, R. <br /> Candida proteins other. The susceptibility Candida                               Schwartz, M. Hammond, J. Balkovec, F. Vanmiddlesworth. 1992. In <br />                                                                                            vitro antifungal activities vivo efficacies 1,3-b-D-glucan synthesis <br /> and Aspergillus species variety echinocandins                              inhibitors L-671,329, L-646,991, tetrahydroechinocandin B, L-687,781, a <br /> established (2, 43). S. pombe studied                                papulacandin. Antimicrob. Agents Chemother. 36:1648&#226;&#8364;&#8220;1657. <br /> regard known susceptible lipopeptide ac-                            2. Bartizal, K., C. J. Gill, G. K. Abruzzo, A. M. Flattery, L. Kong, P. M. Scott, <br /> uleacin (35, 38), papulacandin B (7, 21, 48). Thus,                         J. G. Smith, C. E. Leighton, A. Bouffard, J. F. Dropinski, J. Balkovec. <br />                                                                                            1997. In vitro preclinical evaluation studies echinocandin antifungal <br /> C. neoformans sequence located centrally parsimony                               MK-0991 (L-743,872). Antimicrob. Agents Chemother. 41:2326&#226;&#8364;&#8220;2332. <br /> tree, bracketed organisms susceptibility known                          3. Bastide, J. M., E. H. Hadibi, M. Bastide. 1979. Taxonomic significance <br /> GS inhibitors. It interesting note known func-                              yeast sphaeroplast release enzymatic treatment intact cells. J. Gen. <br /> tional FKS genes S. cerevisiae (FKS1 FKS2)                                    Microbiol. 113:147&#226;&#8364;&#8220;153. <br />                                                                                         4. Berg, J., C. J. Clancy, M. H. Nguyen. 1998. The hidden danger of <br /> C. albicans (FKS1) clustered together. FKS1 indispensable                            primary fluconazole prophylaxis patients AIDS. Clin. Infect. Dis. 26: <br /> Candida despite presence homologs (GSL1                                   186&#226;&#8364;&#8220;187. <br /> and GSL2). In S. cerevisiae, FKS1 FKS2 func-                               5. Bouffard, F. A., R. A. Zambias, J. F. Dropinski, J. M. Balkovec, M. L. <br /> tional double disruption lethal. Another cluster                                Hammond, G. K. Abruzzo, K. F. Bartizal, J. A. Marrinan, M. B. Kurtz, <br />                                                                                            D. C. McFadden, K. H. Nollstadt, M. A. Powles, D. M. Schmatz. 1994. <br /> consisting C. albicans GSL1 S. cerevisiae FKS3 contains                             Synthesis antifungal activity novel cationic pneumocandin B0 deriv- <br /> proteins nonfunctional respect                               atives. J. Med. Chem. 37:222&#226;&#8364;&#8220;225. <br /> contribution GS enzyme activity capable                          6. Cabib, E., B. Bowers, A. Sburlati, S. J. Silverman. 1988. Fungal cell wall <br /> of providing functional subunits GS complex                                synthesis: construction biological structure. Microbiol. Sci. 5:370&#226;&#8364;&#8220; <br />                                                                                            375. <br /> support growth. FKS3 S. cerevisiae dispensable single                        7. Castro, C., J. C. Ribas, M. H. Valdivieso, R. Varona, F. del Rey, A. <br /> disruption synthetically lethal combination                             Duran. 1995. Papulacandin B resistance budding fission yeasts: iso- <br /> either FKS1 FKS2 deletions (47). In C. albicans, FKS1                             lation characterization gene involved (1,3)b-D-glucan synthesis in <br /> alone indispensable, GSL1 capable                               Saccharomyces cerevisiae. J. Bacteriol. 177:5732&#226;&#8364;&#8220;5739. <br />                                                                                         8. Debono, M., R. S. Gordee. 1994. Antibiotics inhibit fungal cell wall <br /> providing GS activity sufficient support growth                              development. Annu. Rev. Microbiol. 48:471&#226;&#8364;&#8220;497. <br /> nonfunctional different function altogether. This dis-                        9. Del Poeta, M., W. A. Schell, J. R. Perfect. 1997. In vitro antifungal <br /> tinct separate clustering functional unknown func-                              activity pneumocandin L-743,872 variety clinically important <br /> tion Fks homologs lends support validity parsi-                              molds. Antimicrob. Agents Chemother. 41:1835&#226;&#8364;&#8220;1836. <br />                                                                                        10. Del Poeta, M., D. L. Toffaletti, T. H. Rude, C. C. Dykstra, J. Heitman, and <br /> mony tree.                                                                                 J. R. Perfect. The role Cryptococcus neoformans topoisomerase I gene <br />    We compared C. neoformans FKS1 sequence                                   pathobiology antifungal target. Submitted publication. <br /> homologs fungal species terms overall se-                             11. Douglas, C. M., J. A. D&#226;&#8364;&#8482;Ippolito, G. J. Shei, M. Meinz, J. Onishi, J. A. <br />  VOL. 181, 1999                                                                   GLUCAN SYNTHASE FKS1 HOMOLOG IN C. NEOFORMANS                                           453 <br />  <br />  <br />       Marrinan, W. Li, G. K. Abruzzo, A. Flattery, K. Bartizal, A. Mitchell,              cells induced aculeacin A, new antifungal antibiotic. Arch. Microbiol. <br />       M. B. Kurtz. 1997. Identification FKS1 gene Candida albicans           127:11&#226;&#8364;&#8220;16. <br />       essential target 1,3-b-D-glucan synthase inhibitors. Antimicrob. Agents        36.   Nicholas, R., D. Williams, P. Hunter. 1994. Investigation value of <br />       Chemother. 41:2471&#226;&#8364;&#8220;2479.                                                                b-glucan-specific flourochromes predicting b-glucan content cell <br /> 12.   Douglas, C. M., F. Foor, J. A. Marrinan, N. Morin, J. B. Nielsen, A. M. Dahl,           walls zoopathogenic fungi. Mycol. Res. 98:694&#226;&#8364;&#8220;698. <br />       P. Mazur, W. Baginsky, W. Li, M. El-Sherbeini, J. A. Clemas, S. M. Man-           37.   Nonaka, H., K. Tanaka, H. Hirano, T. Fujiwara, H. Kohno, M. Umikawa, A. <br />       dala, F. R. Frommer, M. B. Kurtz. 1994. The Saccharomyces cerevisiae                Mino, Y. Takai. 1995. A downstream target RHO1 small GTP-binding <br />       FKS1 (ETG1) gene encodes integral membrane protein subunit                protein PKC1, homolog protein kinase C, leads activation of <br />       1,3-b-D-glucan synthase. Proc. Natl. Acad. Sci. USA 91:12907&#226;&#8364;&#8220;12911.                  MAP kinase cascade Saccharomyces cerevisiae. EMBO J. 14:5931&#226;&#8364;&#8220; <br /> 13.   Douglas, C. M. Personal communication.                                                  5938. <br /> 14.   Douglas, C. M., S. A. Parent, W. Li, G. Chrebet, J. A. Marrinan, M. B.        38.   Osumi, M., N. Yamada, H. Kobori, A. Taki, N. Naito, M. Baba, T. <br />       Kurtz. Mutations Saccharomyces cerevisiae FKS genes identify residues                Nagatani. 1989. Cell wall formation regenerating protoplasts Schizo- <br />       critical function susceptibility glucan synthase inhibitors. Submit-         saccharomyces pombe: study high resolution, low voltage scanning elec- <br />       ted publication.                                                                    tron microscopy. J. Electron Microsc. 38:457&#226;&#8364;&#8220;468. <br /> 15.   Drgonova, J., T. Drgon, K. Tanaka, R. Kollar, G. C. Chen, R. A. Ford,             39.   Page, R. D. 1996. TreeView: application display phylogenetic trees on <br />       C. Chan, Y. Takai, E. Cabib. 1996. Rho1p, yeast protein                    personal computers. Comput. Appl. Biosci. 12:357&#226;&#8364;&#8220;358. <br />       interface cell polarization morphogenesis. Science 272:277&#226;&#8364;&#8220;279.       40.   Payne, T., R. Calderone. 1997. Characterization phosphoribo- <br /> 16.   Edman, J. C., K. J. Kwon Chung. 1990. Isolation URA5 gene               sylpyrophosphate synthetase gene Candida albicans. J. Med. Vet. My- <br />       Cryptococcus neoformans var. neoformans use selective marker               col. 35:305&#226;&#8364;&#8220;312. <br />       transformation. Mol. Cell. Biol. 10:4538&#226;&#8364;&#8220;4544.                                41.   Perfect, J. R., S. D. Lang, D. T. Durack. 1980. Chronic cryptococcal <br /> 17.   Fleet, G. H. 1985. Composition structure yeast cell walls. Curr. Top.            meningitis: new experimental model rabbits. Am. J. Pathol. 101:177&#226;&#8364;&#8220;194. <br />       Med. Mycol. 1:24&#226;&#8364;&#8220;56.                                                              42.   Petter, R., Y. C. Chang, K. J. Kwon-Chung. 1997. A gene homologous <br /> 18.   Georgopapadakou, N. H., J. S. Tkacz. 1995. The fungal cell wall                Saccharomyces cerevisiae SNF1 appears essential viability of <br />       drug target. Trends Microbiol. 3:98&#226;&#8364;&#8220;104.                                                Candida albicans. Infect. Immun. 65:4909&#226;&#8364;&#8220;4917. <br /> 19.   Gozalbo, D., M. V. Elorza, R. Sanjuan, A. Marcilla, E. Valentin, R.           43.   Pfaller, M. A., S. A. Messer, S. Coffman. 1997. In vitro susceptibilities of <br />       Sentandreu. 1993. Critical steps fungal cell wall synthesis: strategies          435 clinical yeast isolates new echinocandin derivative, LY303366, and <br />       inhibition. Pharmacol. Ther. 60:337&#226;&#8364;&#8220;345.                                          antifungal agents. Antimicrob. Agents Chemother. 41:763&#226;&#8364;&#8220;766. <br /> 20.   Grunstein, M., D. S. Hogness. 1975. Colony hybridization: method        44.   Qadota, H., I. Ishii, A. Fujiyama, Y. Ohya, Y. Anraku. 1992. RHO gene <br />       isolation cloned DNAs contain specific gene. Proc. Natl. Acad.            products, putative small GTP-binding proteins, important activation <br />       Sci. USA 72:3961&#226;&#8364;&#8220;3965.                                                                  CAL1/CDC43 gene product, protein geranylgeranyltransferase in <br /> 21.   Ishiguro, J., A. Saitou, A. Duran, J. C. Ribas. 1997. cps11, Schizosac-           Saccharomyces cerevisiae. Yeast 8:735&#226;&#8364;&#8220;741. <br />       charomyces pombe gene homolog Saccharomyces cerevisiae FKS genes               45.   Qadota, H., C. P. Python, S. B. Inoue, M. Arisawa, Y. Anraku, Y. Zheng, T. <br />       mutation confers hypersensitivity cyclosporin A papulacandin               Watanabe, D. E. Levin, Y. Ohya. 1996. Identification yeast Rho1p <br />       B. J. Bacteriol. 179:7653&#226;&#8364;&#8220;7662.                                                         GTPase regulatory subunit 1,3-b-D-glucan synthase. Science 272:279&#226;&#8364;&#8220; <br /> 22.   Iwata, K., Y. Yamamoto, H. Yamaguchi, T. Hiratani. 1982. In vitro                   281. <br />       studies aculeacin A, new antifungal antibiotic. J. Antibiot. 35:203&#226;&#8364;&#8220;209.     46.   Radding, J. A., S. A. Heidler, W. W. Turner. 1998. Photoaffinity analog <br /> 23.   Joseph-Horne, T., D. Hollomon, R. S. Loeffler, S. L. Kelly. 1995. Cross-            semisynthetic echinocandin LY303366: identification echinocandin <br />       resistance polyene azole drugs Cryptococcus neoformans. Antimi-               targets Candida albicans. Antimicrob. Agents Chemother. 42:1187&#226;&#8364;&#8220;1194. <br />       crob. Agents Chemother. 39:1526&#226;&#8364;&#8220;1529.                                             47.   Ram, A. Personal communication. <br /> 24.   Kauffman, C. A., P. L. Carver. 1997. Antifungal agents 1990s.          48.   Ribas, J. C., C. Roncero, H. Rico, A. Duran. 1991. Characterization a <br />       Current status future developments. Drugs 53:539&#226;&#8364;&#8220;549.                               Schizosaccharomyces pombe morphological mutant altered galacto- <br /> 25.   Kelly, R., E. Register, M. J. Hsu, M. Kurtz, J. Nielsen. 1996. Isolation         mannan content. FEMS Microbiol. Lett. 63:263&#226;&#8364;&#8220;267. <br />       gene involved 1,3-b-D-glucan synthesis Aspergillus nidulans puri-     49.   Ruiz Herrera, J. 1991. Biosynthesis beta-glucans fungi. Antonie Leeu- <br />       fication corresponding protein. J. Bacteriol. 178:4381&#226;&#8364;&#8220;4391.                     wenhoek 60:72&#226;&#8364;&#8220;81. <br /> 26.   Kohno, H., K. Tanaka, A. Mino, M. Umikawa, H. Imamura, T. Fujiwara, Y.            50.   Sable, C. 1997. Presented 37th Interscience Conference Anti- <br />       Fujita, K. Hotta, H. Qadota, T. Watanabe, Y. Ohya, Y. Takai. 1996.                  microbial Agents Chemotherapy, Toronto, Canada, 28 September to <br />       Bni1p implicated cytoskeletal control putative target Rho1p small            1 October 1997. <br />       GTP binding protein Saccharomyces cerevisiae. EMBO J. 15:6060&#226;&#8364;&#8220;6068.            51.   Sherman, F., G. Fink, J. Hicks. 1983. Methods yeast genetics. Cold <br /> 27.   Krishnarao, T. V., J. N. Galgiani. 1997. Comparison vitro                 Spring Harbor Laboratory, Cold Spring Harbor, N.Y. <br />       activities echinocandin LY303366, pneumocandin MK-0991,            52.   Sipos, L., G. von Heijne. 1993. Predicting topology eukaryotic <br />       fluconazole Candida species Cryptococcus neoformans. Antimi-                membrane proteins. Eur. J. Biochem. 213:1333&#226;&#8364;&#8220;1340. <br />       crob. Agents Chemother. 41:1957&#226;&#8364;&#8220;1960.                                             53.   Spitzer, E. D., S. G. Spitzer, L. F. Freundlich, A. Casadevall. 1993. <br /> 28.   Lamb, D. C., A. Corran, B. C. Baldwin, J. Kwon-Chung, S. L. Kelly.                  Persistence initial infection recurrent Cryptococcus neoformans menin- <br />       1995. Resistant P45051A1 activity azole antifungal tolerant Cryptoc <br /> </body></html>