<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                      A Single-Transformation Gene Function Test in <br />                                  Diploid Candida albicans <br />  <br />              The fungal pathogen Candida albicans is naturally diploid, and current gene disruption strategies require two <br />           successive transformations. We describe here a genetic construct (UAU1) for which two copies may be selected. <br />           Insertion of UAU1 into one genomic site, after a single transformation, allows selection for segregants with two <br />           copies of the insertion. Major classes of segregants are those carrying homozygous insertion mutations and <br />           allelic triplications, which have two insertion alleles and a wild-type allele. Thus nonessential and essential <br />           genes may be distinguished rapidly through PCR tests for homozygosis and triplication. We find that homozy- <br />           gous mutations may be isolated at three nonessential loci (ADE2, RIM20, and YGR189), while only allelic <br />           triplications were found at two essential loci (SNF1 and CDC28). We have unexpectedly isolated homozygous <br />           mutants with mutations at CDC25; they are viable but defective in filamentation on serum-containing medium. <br />           The UAU1 cassette is thus useful <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> assess rapidly the essentiality of C. albicans genes. <br />  <br />  <br />    Candida albicans is an opportunistic fungal pathogen. It has                                      MATERIALS AND METHODS <br /> been of experimental interest for two main reasons. First, it is                Plasmids. The UAU1 (ura3-ARG4-ura3) (Fig. 1A) cassette is carried in plas- <br /> a significant pathogen that causes oral mucosal infections, vag-             mid pBME101, which was constructed as follows. Plasmid pRS424ARG4- <br /> initis, nosocomial bloodstream infections, and a variety of deep             URA3-BH1 (4) was digested with BsmI and SpeI, overhangs were blunted with <br />                                                                              Vent polymerase, and blunt ends were ligated to form plasmid pBME98. This <br /> tissue infections (26). Therefore, many experimental studies                 plasmid was then digested with XhoI and SmaI, filled in, and religated to remove <br /> have focused on pathogenesis, drug resistance, and analysis of               a polylinker ClaI site to yield plasmid pBME99. Five hundred <span id='am-163' about='xsp:base' typeof='owl:Thing'>base</span> pairs of the <br /> prospective drug targets. Second, it is a distant cousin of the              3&#226;&#172;&#732; end of URA3 was amplified <span id='am-1' about='protege:FROM' typeof='owl:Thing'>from</span> pRS424ARG4-URA3-BH1 with primers <br /> most well-characterized unicellular eukaryote, Saccharomyces                 Kpn5&#226;&#172;&#732; and Xho3&#226;&#172;&#732;-500 and ligated into the XhoI and KpnI sites of pBME99, <br />                                                                              generating plasmid pBME101. (Oligonucleotide sequences are listed in Table 1.) <br /> cerevisiae, so that the function of a C. albicans gene may be                   Several of the gene disruption constructs described below were generated by <br /> suggested by its role in S. cerevisiae. Yet, surprisingly, C. albi-          in vivo recombination in S. cerevisiae, based on the method of Ma et al. (18). The <br /> cans may use gene products and regulatory pathways in novel                  key feature of this method is that linearization of a plasmid with CEN-ARS or <br /> ways (4, 16, 17, 28, 34, 37). The contrast between C. albicans               2 &#226;&#65533;&#174;m replicons causes a decrease in transformation efficiency in S. cerevisiae, but <br />                                                                              inclusion of a DNA fragment that bridges the linearization site restores efficient <br /> and S. cerevisiae can provide unique insight into regulatory                 transformation. The retrieved plasmids largely contain the resealed plasmid <br /> mechanisms, interpathway relationships, and general aspects                  carrying the bridging DNA fragment inserted through homologous recombina- <br /> of eukaryotic biology.                                                       tion. We used this method to extend the homology that flanked gene disruption <br />    Molecular genetics has played an increasingly prominent                   PCR products. We could thus use gene disruption primers with only 40 bases of <br />                                                                              homology to the targeted locus. The construction of plasmid pBME102, which <br /> role in studies of C. albicans, particularly with the partial ge-            carries the ade2::UAU1 gene disruption cassette, was typical. The UAU1 cassette <br /> nomic sequence as a tool for gene discovery. However, genetic                was amplified by PCR from plasmid pBME101 with primers ade2-3DR and <br /> methods are cumbersome for two reasons (30). First, C. albi-                 ade2-5DR (Table 1) to yield an ade2::UAU1 PCR product with 50 bp of ADE2 <br /> cans strains are diploid (or of higher ploidy) and there is no               homology on each end. Approximately 1 &#226;&#65533;&#174;g of the 4.15-kbp PCR product, <br />                                                                              purified from an agarose gel, was cotransformed with 0.2 &#226;&#65533;&#174;g of EcoRI-linerized <br /> meiotic division, so that gene disruption mutants must be con-               plasmid pRS314-ADE (44) into S. cerevisiae trp1 mutant strain AMP107 (36). <br /> structed through two successive transformations. Second, with                EcoRI digestion linearized the plasmid by cleaving at a single site in the C. al- <br /> current methods, disruption of one allele may be straight-                   bicans ADE2 insert that lies between the regions of homology to primers ade2- <br /> forward, but disruption of the second allele is infrequent if                3DR and ade2-5DR. Transformants were selected on SC-Trp plates, which <br />                                                                              selects for the plasmid TRP1 marker. A control transformation with no DNA <br /> the second disruption construct is homologous to the first.                  yielded 0 colonies/plate, a control with linearized pRS314-ADE yielded 112 <br /> These problems can make it difficult to construct a homozy-                  colonies/plate, and the cotransformation of linearized pRS314-ADE and the <br /> gous mutant and to determine whether a gene is essential                     ade2::UAU1 PCR product yielded 773 colonies/plate. DNA prepared from Trp&#226;&#171;&#185; <br /> for growth.                                                                  transformants (10) was transformed into Escherichia coli strain HB101 (Pro- <br />                                                                              mega), with selection for ampicillin resistance. The structure of retrieved plasmid <br />    We describe here a genetic strategy that circumvents these                pBME102 was confirmed by restriction digestion. <br /> difficulties. It yields homozygous insertion mutations after a                  The snf1::UAU1 gene disruption cassette was carried in plasmid pBME105. <br /> single transformation. The strategy provides a rapid test for                We first constructed plasmid pBME103 as follows: C. albicans genomic DNA was <br /> essential genes and should thus accelerate drug target valida-               diluted 100-fold and PCR amplified with primers SNF1c1aI-5&#226;&#172;&#732; and SNF1ecoRI- <br />                                                                              3&#226;&#172;&#732;, and the PCR product was ligated into plasmid pGEMT-Easy (Promega). We <br /> tion. In addition, the strategy can provide a preliminary assess-            then moved the SNF1 insert as an EcoRI fragment into EcoRI-digested vector <br /> ment of mutant phenotypes, and it lends itself to large-scale                pRS424 (2) to create plasmid pBME104. Finally, plasmid pBME105 was gener- <br /> analysis of gene function.                                                   ated by in vivo recombination of BsmI-linearized pBME104 and a snf1::UAU1 <br />                                                                              PCR product with 40 bp of SNF1 homology on each end, produced with template <br />                                                                              plasmid pBME101 and primers SNF1 dr 5&#226;&#172;&#732; and SNF1 dr 3&#226;&#172;&#732;. <br />                                                                                 The cdc28::UAU1 cassette was carried in plasmid pBME108. We first con- <br />                                                                              structed plasmid pBME100, which carries a 1.5-kbp clone of C. albicans CDC28, <br />   * Corresponding author. Mailing address: Department of Microbi-            by PCR amplification of strain CAI4 DNA with primers cdc28-N and cdc28-C <br /> ology, Columbia University, 701 West 168th St., New York, NY 10032.          and ligation into vector pGEM-T (Promega). We also removed the XhoI site <br /> Phone: (212) 305-8251. Fax: (212) 305-1741. E-mail: apm4@columbia            of vector pRS424 (2) by ligation of XhoI-digested and filled-in pRS424, yield- <br /> .edu.                                                                        ing plasmid pBME106. We then constructed plasmid pBME107 by ligating the <br />  <br />                                                                       5730 <br />  VOL. 182, 2000                                                     C. ALBICANS SINGLE-TRANSFORMATION GENE FUNCTION TEST                                              5731 <br />  <br />  <br />                                                                                         Strains. The C. albicans strains (Table 2) are derivatives of strain BWP17 (ura3&#226;&#338;&#172;:: <br />                                                                                      &#226;&#65533;&#173;imm434/ura3&#226;&#338;&#172;::&#226;&#65533;&#173;imm434 his1::hisG/his1::hisG arg4::hisG/arg4::hisG), which <br />                                                                                      was derived from strain CAI4 (7) as described previously (44). The following <br />                                                                                      UAU1 disruption heterozygotes were constructed through transformation of <br />                                                                                      strain BWP17 with the restriction fragments indicated and selection for Arg&#226;&#171;&#185; <br />                                                                                      transformants: strain BMY7, the NotI/EcoRV insert fragment from plasmid <br />                                                                                      pBME 102; strain BMY18, the PvuII insert fragment from plasmid pBME105; <br />                                                                                      and strain BMY22, the PvuII insert fragment from plasmid pBME108. The <br />                                                                                      following UAU1 disruption heterozygotes were constructed through transforma- <br />                                                                                      tion of strain BWP17 with the PCR product indicated and selection for Arg&#226;&#171;&#185; <br />                                                                                      transformants: strain BMY16, PCR with primers cdc25 5&#226;&#172;&#732;dr and cdc25 3&#226;&#172;&#732;dr on <br />                                                                                      template pBME101; and strain DAY151, PCR with primers YGR189 5&#226;&#172;&#732;DR and <br />                                                                                      YGR 189 3&#226;&#172;&#732; DR on template pBME101. The UAU1 heterozygote BMY17 was <br />                                                                                      constructed by homologous integration of the UAU1 cassette into URA3 se- <br />                                                                                      quences of a rim20::URA3 allele. Specifically, we transformed strain DAY18 <br />                                                                                      (originally called Enx-het1 [44]), of genotype rim20::URA3/RIM20 ura3&#226;&#338;&#172;:: <br />                                                                                      &#226;&#65533;&#173;imm434/ura3&#226;&#338;&#172;::&#226;&#65533;&#173;imm434 his1::hisG/his1::hisG arg4::hisG/arg4::hisG, with the <br />                                                                                      PCR product of template pBME101 amplified with primers 5-Detect and 3-De- <br />                                                                                      tect, selected for Arg&#226;&#171;&#185; transformants, and screened for those that became Ura&#226;&#171;&#186;. <br />                                                                                      (This integration reaction is the reverse of the excision reaction diagrammed in <br />                                                                                      Fig. 1A.) Integration of UAU1 cassettes in heterozygotes was confirmed through <br />                                                                                      genomic DNA PCRs with the primer pairs listed in Table 2. <br />                                                                                         The extents of the deletion mutations are as follows. For ADE2, the dele- <br />                                                                                      tion removes codons 195 to 310; the open reading frame (ORF) is 568 co- <br />                                                                                      dons. For RIM20, the deletion removes codons 520 to 719; the ORF is 785 <br />                                                                                      codons. For YGR189, the deletion removes codons 180 to 316; the ORF is 453 <br />                                                                                      codons. For SNF1, the deletion removes codons 20 to 358; the ORF is 620 co- <br />                                                                                      dons. For CDC28, the deletion removes codons 131 to 181; the ORF is 317 co- <br />                                                                                      dons. For CDC25, the deletion removes codons 1000 to 1333; the ORF is 1333 <br />                                                                                      codons. All of these deletions remove highly conserved regions of the respective <br />                                                                                      proteins, including key residues required for SNF1 and CDC28 protein kinase <br />                                                                                      activity. The deletion of CDC25 removes the GDP-GTP exchange factor domain <br />                                                                                      (15), which is the most highly conserved region of the protein. <br />                                                                                         Media and growth conditions. C. albicans and S. cerevisiae were routinely cul- <br />                                                                                      tured in YPD plus uridine (2% Bacto Peptone, 1% yeast extract, 20% dextrose, <br />                                                                                      and 80 &#226;&#65533;&#174;g uridine per ml). Selection was done on SD synthetic medium (6.7% <br />                                                                                      yeast nitrogen base plus ammonium sulfate, without amino acids, and with 2% <br />                                                                                      dextrose); auxotrophic supplements were added at standard concentrations (14) <br />                                                                                      except that uridine was added at 80 &#226;&#65533;&#174;g per ml. <br />                                                                                         PCR detection. Genotypes at each locus were determined through PCR assays <br />                                                                                      with the primers indicated in Table 2. Flanking primers (illustrated by Ade2amp5 <br />                                                                                      and Ade2amp3 in Fig. 1C) were used to detect wild-type alleles and URA3 <br />                                                                                      insertion alleles. However, we found that detection of UAU1 insertion alleles was <br />                                                                                      unreliable with flanking primers, presumably because of the more efficient syn- <br />                                                                                      thesis of the smaller products from wild-type and URA3 insertion alleles. There- <br />                                                                                      fore, we used a flanking primer and an internal ARG4 primer (illustrated by <br />                                                                                      Ade2amp3 and Arg4det in Fig. 1C) in a second PCR to detect the presence of <br />                                                                                      UAU1 insertion alleles. PCR was performed with total yeast genomic DNA as <br />                                                                                      described previously (44). Reaction mixtures were typically heated to 94&#194;&#176;C for 5 <br />                                                                                      min, followed by 33 cycles of 94&#194;&#176;C for 1 min, 50&#194;&#176;C for 1 min, and 72&#194;&#176;C for 3 min. <br />                                                                                      After a final incubation at 72&#194;&#176;C for 10 min, reaction mixtures were stored at 4&#194;&#176;C. <br />                                                                                         Determination of recombination rates. Recombination rates were determined <br />                                                                                      by the method of Lea and Coulson, as described by Rattray and Symington (33). <br />                                                                                      A UAU1 insertion heterozygote was streaked onto YPD and grown at 30&#194;&#176;C for <br />                                                                                      2 to 3 days. Nine entire single colonies were used to inoculate separate 3-ml YPD <br />    FIG. 1. Genetic properties of UAU1. (A) Conversion of UAU1 to URA3. The           broth cultures that were grown to saturation at 30&#194;&#176;C. One milliliter of saturated <br /> UAU1 marker (top) comprises an intact ARG4 gene flanked by URA3 deletion             culture was washed, suspended in 1 ml of sterile water, and diluted for plating. <br /> derivatives ura3&#226;&#338;&#172;3&#226;&#172;&#732; and ura3&#226;&#338;&#172;5&#226;&#172;&#732;. The URA3 segments are nonfunctional, so the         One hundred microliters of the undiluted suspension was plated on SC-Arg-Ura, <br /> UAU1 cassette confers an Arg&#226;&#171;&#185; Ura&#226;&#171;&#186; <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-15' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-20' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-25' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-32' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-44' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-54' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-69' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-72' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-79' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span>phenotype</span>. The URA3 segments share 530            100 &#226;&#65533;&#174;l of a 1/100 dilution was plated on SC-Ura, and 20 &#226;&#65533;&#174;l (in a total volume of <br /> bp of homology and can thus recombine to yield an intact URA3 gene. Recom-           200 &#226;&#65533;&#174;l) of a 1/10,000 dilution was plated on YPD. Colony counts of these plates <br /> bination excises the ARG4 gene and results in an Arg&#226;&#171;&#186; Ura&#226;&#171;&#185; phenotype. (B)            were used to determine the median mitotic recombinant frequencies (Ura&#226;&#171;&#185;/total <br /> Anticipated use of UAU1 to select for homozygous mutants (double-disruption          or Ura&#226;&#171;&#185; Arg&#226;&#171;&#185;/total) for each strain. Recombination rates (events per cell per <br /> selection). One allele of a gene (here ADE2) is disrupted with a UAU1 insertion      generation) were calculated according to the formula rate &#226;&#171;&#189; (0.4343 &#226;&#171;&#187; median <br /> through transformation with selection for an Arg&#226;&#171;&#185; phenotype. Growth of the           frequency)/(log N &#226;&#171;&#186; log N0), where N is the total number of cells in the culture <br /> transformant yields rare recombinant segregants in which the UAU1 insertion          and N0 is the initial number of cells (one cell) that gave rise to the culture (33). <br /> allele is homozygous. Such segregants may be selected after they undergo re- <br /> combinational excision within one UAU1 cassette to yield a unique Arg&#226;&#171;&#185; Ura&#226;&#171;&#185; <br /> phenotype. (C) Outcome of double-disruption selection with an ade2::UAU1/                                               RESULTS <br /> ADE2 strain. Genotypes were determined with PCR primers depicted at the top. <br /> The ade2::UAU1/ADE2 strain yielded two classes of Arg&#226;&#171;&#185; Ura&#226;&#171;&#185; segregants. One            Selection for homozygous mutants. We constructed a gene <br /> class (homozygote) was Ade&#226;&#171;&#186;, grew into red colonies, and yielded diagnostic          disruption cassette, UAU1 (ura3-ARG4-ura3), that can exist in <br /> ade2::UAU1 and ade2::URA3 PCR products (Fig. 2A, lanes 1, 2, 5, 7, and 9). The       two states (Fig. 1A). The cassette includes three segments: <br /> other class (allelic triplication) was Ade&#226;&#171;&#185;, grew into white colonies, and yielded   ura3&#226;&#338;&#172;3&#226;&#172;&#732;, a nonfunctional 3&#226;&#172;&#732; deletion copy of URA3; ARG4; and <br /> ade2::UAU1, ade2::URA3, and ADE2 PCR products (Fig. 2A, lanes 3, 4, 6, 8, and <br /> 10). The latter class is depicted as a trisome as an example, but it may arise       ura3&#226;&#338;&#172;5&#226;&#172;&#732;, a nonfunctional 5&#226;&#172;&#732; deletion copy of URA3 that shares <br /> through translocation, tandem duplication, or other genetic rearrangements.          530 bp of sequence identity with the ura3&#226;&#338;&#172;3&#226;&#172;&#732; segment. The <br />                                                                                      cassette expresses ARG4, but not URA3, in this state (UAU1). <br />                                                                                      The 530 bp of homology between the ura3 segments permits <br /> pBME100 insert, released with SacII and SpeI, into pBME106 digested with SacII <br /> and SpeI. Plasmid pBME108 was generated by in vivo recombination of XhoI- <br />                                                                                      recombination to yield an intact URA3 gene and excision of <br /> digested pRS424&#226;&#338;&#172;XhoI-CDC28 and a cdc28::UAU1 PCR product produced                    ARG4. The resulting cassette expresses URA3 but not ARG4. <br /> from template plasmid pBME101 and primers cdc28 5&#226;&#172;&#732;DR and cdc28 3&#226;&#172;&#732;DR.                 We reasoned that the cassette would permit identification of <br />  5732      ENLOE ET AL.                                                                                                                J. BACTERIOL. <br />  <br />  <br />                                                       TABLE 1. Oligonucleotide sequences <br />   Oligo- <br />                                                                            Sequence (5&#226;&#172;&#732; to 3&#226;&#172;&#732;) <br />  nucleotide <br />  <br /> 3-Detect ....................TGTGGAATTGTGAGCGGATAACAATTTCAC <br /> 5-Detect ....................GTTTTCCCAGTCACGACGTTGTAAAACGAC <br /> ade2-3DR .................GGGTTGCCTTATCACCCAAGACATTCAACATAATAGCATTGGTGATGGAATGTGGAATTGTGAGCGGATA <br /> ade2-5DR .................GTCCATTATATGCTGAAAAATGGTGTCCTTTCACCAAAGAATTGGCTGTGGTTTTCCCAGTCACGACGTT <br /> Ade2amp3 ................CCATCTTTCGCTCTGGTCTAGTAG <br /> Ade2amp5 ................GTCGATGACTTGTTACACATTGGG <br /> Arg4det .....................GGAATTGATCAATTATCTTTTGAAC <br /> cdc25 3&#226;&#172;&#732;DR ..............TAAATTTTACGTTTTTATTGATTCGCTCTACTGTACAAACACTATGCTTAGCAAATACTATGTGGAATTGTGAGCGGATA <br /> cdc25 5&#226;&#172;&#732;DR ..............AATATTATGAAACTTTTACTTGAAAAAAATTGGTCCATGTCGTATTACAATGAACCTGTAGTTTTCCCAGTCACGACGTT <br /> Cdc25amp3...............CATTGTGTATTAGAAGTCTGTAGTTC <br /> Cdc25amp5...............GGAATTGATATCTTTACTAATTGC <br /> cdc28 3&#226;&#172;&#732;DR ..............CAGACCACATATCTACCCCAGTGGAATATTGTTTCCCTCCTGTGGAATTGTGAGCGGATA <br /> cdc28 5&#226;&#172;&#732;DR ..............ATTCGAGGTATTAAACATTGTCATTCTCATCGAGTTTTACGTTTTCCCAGTCACGACGTT <br /> cdc25del1 ..................TGAAAACGGTTTGAAACTCTTGA <br /> cdc25del2 ..................CATGTGTGCCATTTTTTGGTGTG <br /> cdc25del3 ..................TGTCAAATCAGATAAGTACACACCA <br /> cdc25del4 ..................CCAATCAATTGACTAACTTTGTTGG <br /> cdc25del5 ..................TATCAATGCTCTCACTGAGCCCTG <br /> cdc25del6 ..................TTTGCTCACTCCGACCAAGTTC <br /> Cdc28amp3...............CCCTGAATATCCTGAAAAGCAATCG <br /> Cdc28amp5...............GAAGATGAAGGTGTACCTAGTACC <br /> cdc28-C .....................CCCCTGAATATCCTGAAAAGCAATCG <br /> cdc28-N .....................CCAACTATAGAACACACACATCCCAAGCC <br /> Kpn5&#226;&#172;&#732; ........................GGGGGTACCTGATTTCTAGAAGGACCACC <br /> Rim20amp3 ..............GATTCCATAAACCAGGTTTACTAG <br /> Rim20amp5 ..............GAGTGTAATCATTTGTTGCAAGAG <br /> SNF1 dr 3&#226;&#172;&#732;................TTGATTTCGATTGCTGAGACGATGAGTTTCCTGCAGCAACTGTGGAATTGTGAGCGGATA <br /> SNF1 dr 5&#226;&#172;&#732;................GCAAGTTCCGATCGACCCCGCTGCAAATCCAGCAAATAGAGTTTTCCCAGTCACGACGTT <br /> Snf1amp3 ..................GTCTTTCACCAATGCATGATTC <br /> Snf1amp5 ..................CATAATGAAAATCAATCGCAAC <br /> SNF1claI-5&#226;&#172;&#732; ..............CCATCGATGAATCAATATATAGAAGAAGG <br /> SNF1ecoRI-3&#226;&#172;&#732; ..........GGGGAATTCGCTCATCTTTAATTAGTTTCG <br /> Xho3&#226;&#172;&#732;-500 .................GGGCTCGAGCATCAATTTATGATTTTTGAAG <br /> YGR189 3&#226;&#172;&#732;DR.........AGCTTGAAGAGGAGGAGGAAGATGATGATGATGATGATGAAGTGGTAGATGAAGGTGAGCGTGGAATTGTGAGCGGATA <br /> YGR189 5&#226;&#172;&#732;DR.........CCATACTTATGTTATTGATTGGACCAAAGATGCAGTTACTTGGTCCGTTGACGGTAGTGTTTTCCCAGTCACGACGTT <br /> Ygr189amp3 .............GAACTGCATTGGATTTTCGC <br /> Ygr189amp5 .............CTACTACTTATGATCGTGGT <br />  <br />  <br />  <br />  <br /> homozygous mutants (Fig. 1B). A heterozygous UAU1 inser-                     lane H, and data not shown). Strain BMY7 yielded primarily <br /> tion, introduced by transformation, might occasionally become                white Ade&#226;&#171;&#185; colonies, and loss of ADE2 function creates red <br /> homozygous through gene conversion or a mitotic recombina-                   Ade&#226;&#171;&#186; colonies. We grew 30 independent cultures of the het- <br /> tion event. The homozygous mutant would be uniquely capable                  erozygote and plated aliquots to isolate Ura&#226;&#171;&#185; and Arg&#226;&#171;&#185; Ura&#226;&#171;&#185; <br /> of yielding segregants that express both ARG4 and URA3.                      segregants (Table 2). Ura&#226;&#171;&#185; segregants arose a rate of 3.5 &#226;&#171;&#187; <br /> Thus, homozygous mutants should be found among Arg&#226;&#171;&#185;                          10&#226;&#171;&#186;6 per division as uniformly white Ade&#226;&#171;&#185; colonies. Arg&#226;&#171;&#185; <br /> Ura&#226;&#171;&#185; segregants in a population carrying a UAU1 insertion.                   Ura&#226;&#171;&#185; segregants arose at a rate of 2.0 &#226;&#171;&#187; 10&#226;&#171;&#186;8 per division. <br />    We used this rationale to construct homozygous ade2/ade2                  Among the Arg&#226;&#171;&#185; Ura&#226;&#171;&#185; colonies, 23% were red and Ade&#226;&#171;&#186;. We <br /> mutants after one transformation of reference strain BWP17                   used PCR analysis to determine the genotype of one randomly <br /> (relevant genotype: arg4/arg4 ura3/ura3 ADE2/ADE2). The strain               chosen Arg&#226;&#171;&#185; Ura&#226;&#171;&#185; segregant from each culture (Fig. 2B, lanes <br /> was transformed with an ade2::UAU1 DNA fragment. An Arg&#226;&#171;&#185;                     1 to 10, and data not shown). We found 12 segregants of ge- <br /> transformant (strain BMY7) had the genotype ade2::UAU1/                      notype ade2::UAU1/ade2::URA3, and these were phenotypi- <br /> ADE2, as verified by PCR and Southern blot analyses (Fig. 2A,                cally red and Ade&#226;&#171;&#186;. The 18 remaining segregants were of <br />  <br />  <br />                                                TABLE 2. Double-disruption selection experiments <br />                                                                                                  Ura&#226;&#171;&#185; rate per     Arg&#226;&#171;&#185; Ura&#226;&#171;&#185; rate       No. of homo- <br />  Locus                Strain (genotypea)                       Detection primers <br />                                                                                                    divisionb        per divisionb         zygotesc <br />  <br /> ADE2          BMY7 (ade2::UAU1/ADE2)                  Ade2amp5, Ade2amp3, Arg4det                 3.5   &#226;&#171;&#187;   10&#226;&#171;&#186;6     2.0   &#226;&#171;&#187;   10&#226;&#171;&#186;8          12 <br /> YGR189        DAY151 (ygr189::UAU1/YGR189)            Ygr189amp5, Ygr189amp3, Arg4det             2.3   &#226;&#171;&#187;   10&#226;&#171;&#186;6     1.0   &#226;&#171;&#187;   10&#226;&#171;&#186;8          10 <br /> RIM20         BMY17 (rim20::UAU1/RIM20)               Rim20amp5, Rim20amp3, Arg4det               1.5   &#226;&#171;&#187;   10&#226;&#171;&#186;5     6.8   &#226;&#171;&#187;   10&#226;&#171;&#186;8          11 <br /> SNF1          BMY18 (snf1::UAU1/SNF1)                 Snflamp5, Snflamp3, Arg4det                 2.9   &#226;&#171;&#187;   10&#226;&#171;&#186;6     1.3   &#226;&#171;&#187;   10&#226;&#171;&#186;8           0 <br /> CDC28         BMY22 (cdc28::UAU1/CDC28)               Cdc28amp5, Cdc28amp3, Arg4det               7.9   &#226;&#171;&#187;   10&#226;&#171;&#186;6     3.5   &#226;&#171;&#187;   10&#226;&#171;&#186;7           0 <br /> CDC25         BMY16 (cdc25::UAU1/CDC25)               Cdc25amp5, Cdc25amp3, Arg4det               5.4   &#226;&#171;&#187;   10&#226;&#171;&#186;6     3.6   &#226;&#171;&#187;   10&#226;&#171;&#186;9           2 <br />   a <br />   Strains carried the additional mutations ura3&#226;&#338;&#172;::&#226;&#65533;&#173;imm434/ura3&#226;&#338;&#172;::&#226;&#65533;&#173;imm434 his1::hisG/his1::hisG arg4::hisG/arg4::hisG. <br />   b <br />   Recombination rates were calculated through the method of the median, as described in Materials and Methods. <br />   The number of homozygous mutants found among 30 independent Arg&#226;&#171;&#185; Ura&#226;&#171;&#185; segregants from the heterozygote indicated. Genotypes were determined through <br />   c <br />  <br /> PCR analysis, examples of which are shown in Fig. 2. <br />  VOL. 182, 2000                                                 C. ALBICANS SINGLE-TRANSFORMATION GENE FUNCTION TEST                          5733 <br />  <br />  <br />                                                                                 results indicate that homozygous mutants can be isolated after <br />                                                                                 a single transformation. <br />                                                                                    To determine whether this homozygosis strategy may be <br />                                                                                 useful at other loci, we examined the genes RIM20 and <br />                                                                                 YGR189. Previously constructed rim20/rim20 mutants are via- <br />                                                                                 ble, so this gene is not essential (4). We constructed rim20:: <br />                                                                                 UAU1/RIM20 and ygr189::UAU1/YGR189 heterozygotes through <br />                                                                                 one transformation (strains BMY17 and DAY151, respective- <br />                                                                                 ly) and examined 30 independent cultures of each for produc- <br />                                                                                 tion of Ura&#226;&#171;&#185; and Arg&#226;&#171;&#185; Ura&#226;&#171;&#185; segregants (Table 2). For both <br />                                                                                 strains, these segregants arose at rates similar to those ob- <br />                                                                                 served with the ade2::UAU1/ADE2 heterozygote. PCR analysis <br />                                                                                 of genotype was again carried out on one Arg&#226;&#171;&#185; Ura&#226;&#171;&#185; segregant <br />                                                                                 per culture (Figs. 2B and C and data not shown). For both <br />                                                                                 genes, we found that one-third of the segregants were rim20:: <br />                                                                                 UAU1/rim20::URA3 or ygr189::UAU1/ygr189::URA3 homo- <br />                                                                                 zygotes, respectively, and two-thirds were rim20::UAU1/rim20:: <br />                                                                                 URA3/RIM20 or ygr189::UAU1/ygr189::URA3/YGR189 trip- <br />                                                                                 lication derivatives, respectively (Table 2). The rim20/rim20 <br />                                                                                 homozygotes had a filamentation defect, as found for conven- <br />                                                                                 tionally constructed homozygotes (4); the ygr189/ygr189 homo- <br />                                                                                 zygotes had no obvious phenotype. Therefore, homozygous <br />                                                                                 mutants have been isolated by selection from UAU1 insertion <br />                                                                                 heterozygotes at three different C. albicans loci. <br />                                                                                    Homozygote-trisome (HT) test for essential genes. Iden- <br />                                                                                 tification of essential genes in C. albicans is vital to assess <br />                                                                                 prospective drug targets (30) and to consider more-refined <br />                                                                                 functional tests (1). C. albicans genes have been considered <br />                                                                                 essential if one allele can be disrupted but the second cannot. <br />                                                                                 We reasoned that the UAU1 cassette might simplify and accel- <br />                                                                                 erate this assessment: a UAU1 insertion in nonessential genes <br />                                                                                 yields both homozygous and triplication-bearing segregants, <br />                                                                                 but a UAU1 insertion in an essential gene should yield only <br />                                                                                 triplication-bearing segregants. Thus, we examined the conse- <br />                                                                                 quences of UAU1 insertions in three likely essential genes: <br />                                                                                 SNF1, CDC28, and CDC25. <br />                                                                                    Petter et al. have presented strong experimental evidence <br />                                                                                 that SNF1 is essential for viability (29). We created a snf1:: <br />                                                                                 UAU1/SNF1 heterozygote and found that it produced Ura&#226;&#171;&#185; <br />                                                                                 and Arg&#226;&#171;&#185; Ura&#226;&#171;&#185; segregants at rates similar to those for ADE2 <br />                                                                                 and YGR189 insertions (Table 2). PCR genotyping of 30 inde- <br />                                                                                 pendent Arg&#226;&#171;&#185; Ura&#226;&#171;&#185; segregants revealed that all were snf1:: <br />                                                                                 UAU1/snf1::URA3/SNF1 triplication derivatives (Table 2; Fig. <br />                                                                                 2D). Our failure to obtain snf1/snf1 homozygotes is consistent <br />                                                                                 with the conclusion that SNF1 is an essential gene. <br />                                                                                    CDC28 specifies a cyclin-dependent protein kinase, and ac- <br />                                                                                 tivity of such kinases is vital for cell cycle progression in all <br />                                                                                 eukaryotes (21). The C. albicans CDC28 gene complements <br />                                                                                 corresponding defects in other yeasts (3, 35), so we inferred <br />                                                                                 that C. albicans CDC28 may be essential. A cdc28::UAU1/ <br />                                                                                 CDC28 heterozygote produced Ura&#226;&#171;&#185; and Arg&#226;&#171;&#185; Ura&#226;&#171;&#185; segre- <br />    FIG. 2. PCR analysis of double-disruption selection segregants. Each panel   gants at rates comparable to those for other insertion heterozy- <br /> shows a composite of two agarose gels of PCRs conducted with genomic DNA        gotes; PCR genotyping revealed that 30 independent Arg&#226;&#171;&#185; <br /> templates. The upper-gel PCRs used amp3 and Arg4det primers; the lower-gel <br /> PCRs used amp3 and amp5 primers. (The amp3&#226;&#8364;&#8220;amp5 PCRs are unreliable for         Ura&#226;&#171;&#185; segregants were cdc28::UAU1/cdc28::URA3/CDC28 trip- <br /> detection of full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> UAU1 insertions, presumably because the smaller PCR    lication derivatives (Table 2; Fig. 2E). These results support <br /> products are amplified more efficiently.) Lanes: templates from parent strain   the idea that C. albicans CDC28 is an essential gene. <br /> BWP17 (lanes P), the respective UAU1 disruption heterozygote (lanes H), and        The S. cerevisiae CDC25 gene is essential for viability; Cdc25p <br /> 10 independent Arg&#226;&#171;&#185; Ura&#226;&#171;&#185; segregants from the disruption heterozygote (lanes <br /> 1 to 10). Panels show analyses for ADE2 (A), RIM20 (B), YGR189 (C), SNF1        is an activator of Ras proteins (39). C. albicans CDC25 com- <br /> (D), CDC28 (E), and CDC25 (F).                                                  plements an S. cerevisiae cdc25 mutant (8), so we inferred that <br />                                                                                 CDC25 may be essential in C. albicans as well. A cdc25:: <br />                                                                                 UAU1/CDC25 heterozygote produced Ura&#226;&#171;&#185; segregants at <br />                                                                                 rates comparable to those for other insertion heterozygotes <br /> genotype ade2::UAU1/ade2::URA3/ADE2 and were phenotyp-                          and produced Arg&#226;&#171;&#185; Ura&#226;&#171;&#185; segregants at a slightly lower rate <br /> ically white and Ade&#226;&#171;&#185;. These latter Arg&#226;&#171;&#185; Ura&#226;&#171;&#185; Ade&#226;&#171;&#185; seg-                         (Table 2). PCR genotyping revealed that 2 of 30 independent <br /> regants thus have three copies of the ADE2 locus; the third                     Arg&#226;&#171;&#185; Ura&#226;&#171;&#185; segregants were of genotype cdc25::UAU1/cdc25:: <br /> copy may derive from trisomy or translocation (Fig. 1C). Our                    URA3 (Fig. 2F, lanes 3 and 8); the remaining 28 segregants <br />  5734      ENLOE ET AL.                                                                                                               J. BACTERIOL. <br />  <br />  <br />                                                                                 characterized via a battery of refined phenotypic tests. Here, <br />                                                                                 we have described a double disruption selection strategy with a <br />                                                                                 different purpose: to construct homozygous mutants rapidly for <br />                                                                                 a preliminary assessment of gene function. Our strategy is not <br />                                                                                 a replacement or a substitute for the careful crafting of ho- <br />                                                                                 mozygous mutants. Instead, it is a strategy that lends itself to <br />                                                                                 large-scale analysis of gene function. <br />                                                                                    We have presented here an application of double-disruption <br />                                                                                 selection that facilitates one important goal of C. albicans <br />                                                                                 research: the identification of essential genes. We refer to <br />                                                                                 this test as the HT test. The methodology fills an important <br />                                                                                 need for basic research, because it is otherwise laborious to <br />                                                                                 determine whether a gene is essential (29, 30), and current <br />                                                                                 technology can yield conflicting results (5, 16, 22, 38). The <br />                                                                                 methodology also fills an important need for genome-based <br />                                                                                 pharmaceutical research, in which one must evaluate a large <br />                                                                                 number of gene products as prospective drug targets (23). <br />                                                                                 Finally, we expect that the technical simplicity of our method <br />                                                                                 may encourage those who study a conserved gene in some oth- <br />                                                                                 er organism to test the <span id='am-160' about='Thesaurus:Homologous_Gene' typeof='owl:Thing'>homologous gene</span>&#226;&#8364;&#8482;s function in C. albi- <br />                                                                                 cans. <br />                                                                                    The results of HT tests were largely consistent with expec- <br />                                                                                 tations based on prior studies: SNF1 and CDC28 are essential; <br />                                                                                 ADE2 and RIM20 are not. However, the test also argued that <br />                                                                                 CDC25 is not essential, because homozygous cdc25/cdc25 mu- <br />                                                                                 tants were isolated. (The deletion that we constructed removes <br />                                                                                 much of the domain that is necessary and sufficient for GDP- <br />    FIG. 3. PCR analysis of cdc25&#226;&#338;&#172;/cdc25&#226;&#338;&#172; segregants. (A) Schematic diagram      GTP exchange factor activity [15], so we can formally conclude <br /> of wild-type and mutant CDC25 loci. The solid line represents the genomic       only that Cdc25p exchange factor activity is not essential.) We <br /> CDC25 sequence. The dashed line represents the sequence deleted and replaced    note, though, that the cdc25/cdc25 homozygotes arose a lower <br /> with UAU1 and URA3 in the cdc25::UAU1 and cdc25::URA3 alleles. (B) Loca-        rate than homozygous insertion mutations in other nonessen- <br /> tions of PCR primers. Each line represents a PCR amplification product that <br /> derives from the CDC25 sequences directly above (in panel A). The primer pair   tial genes. This observation might be explained if viability of <br /> amp5 &#226;&#171;&#185; amp3 (Cdc25amp5 and Cdc25amp3) can amplify wild-type and mu-             the cdc25/cdc25 mutant depended upon a genetic suppressor <br /> tant CDC25 alleles. The primer pairs 1&#226;&#171;&#185;2 (cdc25del1 and cdc25del2), 3&#226;&#171;&#185;4         <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-88' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-90' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-93' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-95' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-102' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-103' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-104' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-110' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-111' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-113' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-123' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-125' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-128' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-130' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-139' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-140' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-148' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-149' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-155' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span>mutation</span>, as is the case in S. cerevisiae (39). A second possi- <br /> (cdc25del3 and cdc25del4), and 5&#226;&#171;&#185;6 (cdc25del5 and cdc25del6) amplify regions    bility is that CDC25 is completely dispensable, as is likely the <br /> that are deleted by the cdc25::UAU1 and cdc25::URA3 alleles. (C) PCR products <br /> from genomic templates. Genomic templates derived from strains BWP17            case in Schizosaccharomyces pombe (19), but that the structure <br /> (CDC25/CDC25), BMY16 (cdc25::UAU1/CDC25), and the two cdc25::UAU1/              or position of the locus limits recombination. We note that the <br /> cdc25::URA3 segregants were amplified by PCR with the primer pairs indicated,   same ambiguities apply to traditional (two-transformation) <br /> and PCR products were visualized after agarose gel electrophoresis.             gene disruption strategies: infrequent isolation of a targeted <br />                                                                                 mutant may reflect either the inadvertent selection of the ge- <br />                                                                                 netic suppressor or recombinational, rather than functional, <br /> were cdc25::UAU1/cdc25::URA3/CDC25 triplication deriva-                         properties of the locus. The ambiguities may be resolved by <br /> tives (Table 2; Fig. 2F). In principle, the DNA segment deleted                 protein depletion experiments or use of dominant-negative <br /> from the CDC25 locus might have been translocated to a                          mutants (1, 6). Thus, the advantage of the HT test is that it <br /> different region of the genome in the two segregants. Such a                    provides functional information rapidly that can justify more <br /> translocation would be undetected in flanking-primer PCRs                       definitive but laborious experiments. <br /> but should be detected with PCRs of internal segments. How-                        There is a circumstance&#226;&#8364;&#8221;the case of preexisting triplicated <br /> ever, three internal primer pairs yielded no PCR product from                   alleles&#226;&#8364;&#8221;in which the HT test might yield misleading results. <br /> genomic templates prepared from the cdc25::UAU1/cdc25::                         There are three loci for which triplicated alleles have been <br /> URA3 segregants (Fig. 3). Therefore, CDC25 is not essential                     reported, CHS2, FTR1, and FKS1 (GSC1) (9, 22, 32), although <br /> for viability in C. albicans. A recent study indicates that C. al-              FKS1 (GSC1) is apparently not triplicated in all isolates of <br /> bicans Ras2p is a positive regulator of serum-induced filamen-                  strain CAI4 (5). One can envision that insertion of the UAU1 <br /> tation (6). The two cdc25/cdc25 homozygotes grew at a slightly                  cassette into CHS2, for example, would yield a genetic struc- <br /> reduced rate and were partially defective in filamentation on                   ture consistent with a genotype of chs2::UAU1/CHS2; the ac- <br /> agar containing serum. These findings support the idea that                     tual genotype would be chs2::UAU1/CHS2/CHS2. Subsequent <br /> Cdc25p promotes Ras2p activity and filamentation but indicate                   Arg&#226;&#171;&#185; Ura&#226;&#171;&#185; selection would yield exclusively segregants of ge- <br /> that CDC25 is not an essential gene.                                            notype chs2::UAU1/chs2::URA3/CHS2, not because CHS2 is <br />                                                                                 essential but because the strain initially had triplicated alleles. <br />                              DISCUSSION                                         Thus far, preexisting triplications have seldom been observed <br />                                                                                 in C. albicans, so we expect that this limitation for the HT test <br />    Analysis of gene function in C. albicans is a critical step                  will not outweigh its usefulness. <br /> toward understanding its biology, pathogenesis, and vital pro-                     Our results with double-disruption selection raise an impor- <br /> cesses. In the past, this analysis has required two successive                  tant question that we have not yet resolved: how large a DNA <br /> transformations to create homozygous null mutations or con-                     segment becomes homozygous in the Arg&#226;&#171;&#185; Ura&#226;&#171;&#185; homozygous <br /> ditionally expressed alleles (1, 7, 30, 43). These methods allow                disruption mutants? Given that natural C. albicans isolates are <br /> construction of precisely defined mutant strains that can be                    heterozygous for preexisting mutations (31, 40&#226;&#8364;&#8220;42), it is possi- <br /> ble that selection for homozygosity of a UAU1 insertion mu-        <br /> tation may yield homozygosity of a linked mutation as well. In  <br /> our small survey of known nonessential genes, we found only       <br /> homozygotes that had phenotypes consistent with traditionally                  <br /> constructed mutants. However, the possibility that a linked             <br /> mutation may become homozygous remains an important ca-            <br /> veat for any conclusion derived solely from double-disruption           <br /> selection.                                                                <br />    Our estimates of mitotic recombination rates point to a   <br /> surprising conclusion: that heterozygous mutations become      <br /> homozygous at a rate of 2 &#226;&#171;&#187; 10&#226;&#171;&#186;3 per division. We calculate      <br /> this rate from the rate of production of detectable homozy-     <br /> gotes (&#226;&#172;&#402;1 &#226;&#171;&#187; 10&#226;&#171;&#186;8 per division) and the rate of recombination      <br /> to generate URA3 from UAU1 (&#226;&#172;&#402;3 &#226;&#171;&#187; 10&#226;&#171;&#186;6 per division). This      <br /> homozygosis rate is 102- to 103-fold higher than expected from   <br /> studies of S. cerevisiae (24) and violates anecdotal common         <br /> knowledge derived from use of Ura-blaster cassettes to create            <br /> gene disruptions (7, 43). One simple explanation is that our            <br /> estimate of the rate of recombination to generate URA3 from  <br /> UAU1 may be artificially low because of a selective advantage         <br /> of Arg&#226;&#171;&#185; cells during growth in broth culture. However, in         <br /> coculture experiments, we have not detected such an advan-            <br /> tage (B. Enloe and A. P. Mitchell, unpublished results). A       <br /> second explanation is that our frequency estimates are in error  <br /> because of differences in growth dynamics on the selective               <br /> plates. Thus, for example, Ura&#226;&#171;&#185; colonies might arise only from               <br /> preexisting recombinants, while Arg&#226;&#171;&#185; Ura&#226;&#171;&#185; colonies might                <br /> arise during growth after plating. A third explanation is that   <br /> the event that generates URA3 in a UAU1 insertion heterozy-     <br /> gote is not equivalent to the event that generates URA3 in a        <br /> homozygote or triplication derivative. For example, a single             <br /> recombination event might generate two copies of the in-      <br /> sertion mutation and simultaneously promote conversion of          <br /> UAU1 to URA3 in one of the copies. Therefore, our rate                              <br /> estimates serve as an empirical guide, but we remain skeptical                <br /> that interchromosomal recombination is so frequent in C. al-                     <br /> bicans.                                                                <br />    Allelic triplications were detectable for all of the genes we             <br /> examined, and we are uncertain of their genetic structure. One                        <br /> simple possibility is that they result from trisomy for the re-                 <br /> spective chromosome, perhaps in conjunction with an overall     <br /> increase in ploidy. In support of this idea, we note that several                <br /> different selections yield monosomic and trisomic C. albicans                <br /> derivatives (12, 13, 27). A second possibility is that the tripli-             <br /> cations result from tandem duplication or translocation of a                           <br /> smaller genomic segment, a mechanism that also has exper-                          <br />                                                                                  <br /> imental support (11, 20, 25). The UAU1 cassette may be a                              <br /> useful tool to define the genetic and environmental parameters                      <br /> that influence changes in gene dosage through either of these                          <br /> mechanisms.                                                                        <br />   <br /> </body></html>