<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Role of an Essential Acyl Coenzyme A Carboxylase in the Primary and <br />         Secondary Metabolism of Streptomyces coelicolor A3(2) <br />  <br />              Two genes, accB and accE, that form part of the same operon, were cloned <span id='am-3' about='protege:FROM' typeof='owl:Thing'>from</span> Streptomyces coelicolor A3(2). <br />           AccB is homologous <span id='am-5' about='protege:TO' typeof='owl:Thing'>to</span> the carboxyl transferase domain of several propionyl coezyme A (CoA) carboxylases and <br />           acyl-CoA carboxylases (ACCases) of actinomycete origin, while AccE shows no significant homology to any <br />           known protein. Expression of accB and accE in Escherichia coli and subsequent in vitro reconstitution of <br />           enzyme activity in the presence of the biotinylated protein AccA1 or AccA2 confirmed that AccB was the <br />           carboxyl transferase subunit of an ACCase. The additional presence of AccE considerably enhanced the activity <br />           of the enzyme complex, suggesting that this small polypeptide is a functional component of the ACCase. The <br />           impossibility of obtaining an accB null mutant and the thiostrepton growth dependency of a tipAp accB <br />           conditional mutant confirmed that AccB is essential for S. coelicolor viability. Normal growth <span id='am-6' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-7' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-9' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-12' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-13' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-14' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-16' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-17' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-21' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-24' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-26' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-29' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-31' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-35' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-39' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-40' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-42' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-43' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-48' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-49' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-51' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-52' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-63' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-64' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-65' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-66' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-67' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-68' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-70' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-71' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-77' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-78' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-81' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-82' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-83' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span>phenotype</span> in the <br />           absence of the inducer was restored in the conditional mutant by the addition of exogenous long-chain fatty <br />           acids in the medium, indicating that the inducer-dependent phenotype was specifically related to a conditional <br />           block in fatty acid biosynthesis. Thus, AccB, together with AccA2, which is also an essential protein (E. <br />           Rodriguez and H. Gramajo, Microbiology 143:3109&#226;&#8364;&#8220;3119, 1999), are the most likely components of an ACCase <br />           whose main physiological role is the synthesis of malonyl-CoA, the first committed step of fatty acid synthesis. <br />           Although normal growth of the conditional mutant was restored by fatty acids, the cultures did not produce <br />           actinorhodin or undecylprodigiosin, suggesting a direct participation of this enzyme complex in the supply of <br />           malonyl-CoA for the synthesis of these secondary metabolites. <br />  <br />  <br />    Malonyl coenzyme A (CoA) is an essential metabolite in                       Several complexes with acyl-CoA carboxylase (ACCase) ac- <br /> most living organisms. It is a substrate for fatty acid synthases            tivity have been purified from a number of actinomycetes. <br /> (4, 16), for polyketide synthases in plants, fungi, and bacteria             These complexes also possess the ability to carboxylate other <br /> (19), and for fatty acid chain elongation systems (37). It also              substrates, including propionyl- and butyryl-CoA (12, 18, 20). <br /> plays a role as a modulator of the activity of some proteins (8).            Consequently, these enzymes are referred to as ACCases, and <br /> Since malonyl-CoA is used in the production of many of the                   all of them consist of two subunits, a larger one (the &#226;&#65533;&#163; chain) <br /> pharmaceutically important polyketides made by streptomyce-                  with the ability to carboxylate its covalently bound biotin group <br /> tes (19), there is considerable interest in understanding the                and a smaller one (the &#226;&#65533;&#164; chain) bearing the carboxyl trans- <br /> pathway(s) that leads to its synthesis. Thus, knowledge of the               ferase activity. Little is known about the physiological role of <br /> enzyme(s) involved in the supply of this key metabolite will not             these enzymes. <br /> only provide a better understanding of primary metabolism in                    The pathway for the biosynthesis of malonyl-CoA in Strep- <br /> streptomycetes but will potentially allow for the development                tomyces coelicolor has not been established yet. However, acetyl- <br /> of more rational approaches for improving the level of pro-                  CoA carboxylase activity has been readily measured in crude <br /> duction of many useful secondary metabolites.                                extracts of S. coelicolor (7, 36), confirming the presence of this <br />    Biosynthesis of malonyl-CoA occurs in most species through                enzyme activity in this microorganism. Attempts to purify a <br /> ATP-dependent carboxylation of acetyl-CoA by an acetyl-CoA                   complex with acetyl-CoA carboxylase activity from streptomy- <br /> carboxylase (45). The reaction catalyzed by this enzyme is a                 cetes have been unsuccessful, probably reflecting its high in- <br /> two-step process that involves ATP-dependent formation of                    stability in vitro (7). An alternative pathway for the biosynthe- <br /> carboxybiotin, followed by transfer of the carboxyl moiety to                sis of malonyl-CoA was described in Streptomyces aureofaciens <br /> acetyl-CoA. Acetyl-CoA carboxylase expression is essential for               (2, 25) and involved the anaplerotic enzymes phosphoenol- <br /> the normal growth of bacteria (27, 28, 32), yeasts (17), and                 pyruvate carboxylase and oxaloacetate dehydrogenase. How- <br /> isolated animal cells in culture (33), reflecting the importance             ever, oxaloacetate dehydrogenase could not be detected in S. <br /> of this biosynthetic pathway.                                                coelicolor A3(2) (6), where malonyl-CoA synthesis appears to <br />                                                                              occur exclusively through the acetyl-CoA carboxylase complex. <br />                                                                                 Attempts to identify enzymes with carboxylase activity in <br />    * Corresponding author. Mailing address: IBR, Departamento de             S. coelicolor led to the characterization of two complexes ex- <br /> Microbiolog&#196;&#177;&#194;&#180;a, Facultad de Ciencias Bioqu&#196;&#177;&#194;&#180;micas y Farmace&#194;&#180;uticas, <br /> Universidad Nacional de Rosario, Suipacha 531, 2000-Rosario, Argen- <br />                                                                              hibiting exclusively propionyl-CoA carboxylase (PCCase) ac- <br /> tina. Phone: 54-341-4350661. Fax: 54-341-4390465. E-mail: gramajo            tivity. The PCCase purified by Bramwell et al. (7) consisted of <br /> @infovia.com.ar.                                                             a biotinylated protein, PccA, of 88 kDa and a nonbiotinylated <br />  <br />                                                                       4166 <br />  VOL. 67, 2001                                                      ANALYSIS OF AN ACYL-CoA CARBOXYLASE OF S. COELICOLOR                                           4167 <br />  <br />  <br />                                                       TABLE 1. Strains and plasmids used in this study <br />    Strain or plasmid                                                            Description                                                       Reference or source <br />  <br /> Strains <br />   S. coelicolor <br />      M145                     Parental strain SCP1&#226;&#171;&#186; SCP2&#226;&#171;&#186;                                                                                             22 <br />      T124                     M145 (accB/pTR124); Thr Hygr                                                                                            This work <br />      T149                     T124 containing pTR149 integrated in the att site of &#226;&#65533;&#190;C31; Thr Hygr Amr                                                 This work <br />      T149A                    T149 with the wild-type accB copy of the chromosome replaced by the accB::hyg mutant                                    This work <br />                                 allele; Hygr Amr <br />   E. coli <br />     DH5&#226;&#65533;&#163;                      &#226;&#338;&#172;lacU169 (&#226;&#65533;&#190;80lacZ&#226;&#338;&#172;M15) endA1 recA1 hsdR17 deoR supE44 thi-1 &#226;&#65533;&#173;&#226;&#171;&#186; gyrA96 relA1                                             15 <br />     BL21&#226;&#65533;&#173;(DE3)                ompT(DE3)                                                                                                               43 <br />     ET 12567                  supE44 hsdS20 ara-14 proA2 lacY galK2 rpsL20 xyl-5 mtl-1 &#226;&#338;&#172;dam &#226;&#338;&#172;dcm &#226;&#338;&#172;hsdM Cmr                                            29 <br />     RG7                       DH5&#226;&#65533;&#163; carrying pCL1 and pBA11 plasmids                                                                                   43 <br />  <br /> Plasmids <br />   pBluescript SK(&#226;&#171;&#185;)           Phagemid vector (Apr lacZ&#226;&#172;&#732;)                                                                                             Stratagene <br />   PCR-Blunt                   Used for cloning PCR products                                                                                           Invitrogen <br />   pGEM-T Easy                 Used for cloning PCR products                                                                                           Promega <br />   pIJ2925                     pUC18 derivative (Apr lacZ&#226;&#172;&#732;)                                                                                            22 <br />   pSET151                     Used for the conjugal transfer of DNA from E. coli to Streptomyces spp. (Apr Thr lacZ&#226;&#172;&#732;)                                 3 <br />   pET22b(&#226;&#171;&#185;)                   Phagemid vector (Apr lacZ&#226;&#172;&#732;) for expression of recombinant proteins under control of strong                              Novagen <br />                                 T7 transcription and translation signals <br />   pUZ8002                     RK2 derivative with defective oriT (Kmr)                                                                                31 <br />   pIJ8600                     Used for the conjugal transfer of DNA from E. coli to Streptomyces spp. and for expression of                           44 <br />                                 recombinant proteins under tipA promoter <br />   pBA11                       Vector containing E. coli birA gene                                                                                     1 <br />   pTR45                       pSK(&#226;&#171;&#185;) with a chromosomal Pstl insert carrying accA2                                                                    36 <br />   pRM08                       pSK(&#226;&#171;&#185;) with an Sstl insert carrying accBE                                                                               This   work <br />   pTR82                       pSK(&#226;&#171;&#185;) carrying accBE with an NdeI site in the translation start site of accB                                           This   work <br />   pTR88                       pET22b(&#226;&#171;&#185;) with accBE under control of strong T7 transcription and translation signals                                   This   work <br />   pTR90                       pET22b(&#226;&#171;&#185;) with accB under control of strong T7 transcription and translation signals                                    This   work <br />   pTR94                       pIJ8600 derivative with a deletion of the int and att sites and carrying the accBE genes under                          This   work <br />                                 tipAp control <br />   pTR107                      pET22b(&#226;&#171;&#185;) with accE under control of strong T7 transcription and translation signals                                    This   work <br />   pTR124                      pSET151 with a hyg (Hygr) gene inserted in the accB coding region                                                       This   work <br />   pTR141                      pIJ8600 derivative carrying oriT RK2, ori pUC18, attP site, int &#226;&#65533;&#190;C31, and aac(3)IV (Amr)                                This   work <br />   pTR149                      pTR141 with a KpnI insert carrying accBE                                                                                This   work <br />   pTR204                      pET21a(&#226;&#171;&#185;) with accA2 under control of strong T7 transcription and translation signals                                   This   work <br />   pTR237                      pET28a(&#226;&#171;&#185;) with an accE His tag fusion gene                                                                              This   work <br />  <br />  <br /> component, the carboxyl transferase, of 66 kDa. More recently,                        S. coelicolor conjugation experiments (3). For selection of Streptomyces transfor- <br /> we characterized, genetically and biochemically, the compo-                           mants and exconjugants, media were overlaid with thiostrepton (TH) (300 &#226;&#65533;&#174;g per <br />                                                                                       plate), hygromycin (HYG) (1 mg per plate), or APR (1 mg per plate), respec- <br /> nents of a second PCCase in this bacterium. In vitro reconsti-                        tively. Strains and recombinant plasmids are listed in Table 1. Fatty acid sup- <br /> tution experiments showed that an active complex could be                             plementation studies were performed in SMM containing APR (10 &#226;&#65533;&#174;g ml&#226;&#171;&#186;1) and <br /> obtained by mixing a carboxyl transferase component of 65                             0.075% (vol/vol) Brij 58. The different fatty acids were added at a final concen- <br /> kDa, PccB, with either of the two almost identical biotinylated                       tration of 100 &#226;&#65533;&#174;g ml&#226;&#171;&#186;1. <br /> components, AccA1 and AccA2 (36).                                                        Growth conditions, protein production, and preparation of cell extracts. <br />                                                                                       S. coelicolor M145 was grown at 30&#194;&#176;C in shake flasks in YEME medium for 24 to <br />   Here we present a detailed genetic and biochemical charac-                          48 h. When necessary, 10 &#226;&#65533;&#174;g of APR ml&#226;&#171;&#186;1 or 5 &#226;&#65533;&#174;g of TH ml&#226;&#171;&#186;1 was added to the <br /> terization of an essential ACCase from S. coelicolor. The en-                         medium. Mycelia were harvested by centrifugation at 5,000 &#226;&#171;&#187; g for 10 min at 4&#194;&#176;C, <br /> zyme complex possesses unique characteristics and appears to                          washed in 100 mM potassium phosphate buffer, pH 8, containing 0.1 mM di- <br /> be the main pathway for malonyl-CoA synthesis in this micro-                          thiothreitol (DTT), 1 mM EDTA, 1 mM phenylmethylsulfonyl fluoride, and 10% <br />                                                                                       glycerol (buffer A) and resuspended in 1 ml of the same buffer. The cells were <br /> organism. <br />                                                                                       disrupted by sonic treatment (4- or 5-s bursts) using a VibraCell Ultrasonic <br />                                                                                       Processor (Sonics &amp; Materials, Inc.). Cell debris was removed by centrifugation, <br />                         MATERIALS AND METHODS                                         and the supernatant was used as cell extract. For the expression of heterologous <br />    Bacterial strains, culture, and transformation conditions. S. coelicolor strain    proteins, E. coli strains harboring the appropriate plasmids were grown at 37&#194;&#176;C <br /> M145 (SCP1&#226;&#171;&#186; SCP2&#226;&#171;&#186;) was manipulated as described by Hopwood et al. (19).               in shake flasks in Luria-Bertani medium in the presence of 25 &#226;&#65533;&#174;g of chloram- <br /> The strain was grown on SFM, R2, and R5 agar media and in 50 ml of SMM or             phenicol ml&#226;&#171;&#186;1 or 100 &#226;&#65533;&#174;g of ampicillin ml&#226;&#171;&#186;1 for plasmid maintenance. In order to <br /> YEME liquid medium. Escherichia coli strain DH5&#226;&#65533;&#163; was used for routine sub-            improve the biotinylation of AccA1 and AccA2 in E. coli, the strains containing <br /> cloning and was transformed according to the method of Hanahan (15). Trans-           pCL1 or pTR204 were also transformed with pBA11 (1), which overexpresses the <br /> formants were selected on media supplemented with the appropriate antibiotics         E. coli biotin ligase; 10 &#226;&#65533;&#174;M D-biotin was also added to the medium. Overnight <br /> at the following concentrations: ampicillin, 100 &#226;&#65533;&#174;g ml&#226;&#171;&#186;1; apramycin (APR), 100        cultures were diluted 1:10 in fresh medium and grown to an A600 of 0.4 to 0.5 <br /> &#226;&#65533;&#174;g ml&#226;&#171;&#186;1; chloramphenicol, 25 &#226;&#65533;&#174;g ml&#226;&#171;&#186;1; and kanamycin, 30 &#226;&#65533;&#174;g ml&#226;&#171;&#186;1. Strain               before the addition of IPTG to a final concentration of 0.1 mM. Induction was <br /> BL21(DE3) is an E. coli B strain lysogenized with &#226;&#65533;&#173;DE3, a prophage that               allowed to proceed for 4 h. The cells were harvested, washed, and resuspended <br /> expresses the T7 RNA polymerase from the isopropyl-&#226;&#65533;&#164;-D-thiogalactopyranoside          in 1 ml of buffer A. Cell extracts were prepared as described above. <br /> (IPTG)-inducible lacUV5 promoter (43). ET12567/pUZ8002 (a gift from M.                   Protein methods. Cell extracts were analyzed by denaturing sodium dodecyl <br /> Paget, John Innes Centre, Norwich, United Kingdom) was used for E. coli-              sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) (26) using a Bio-Rad <br />  4168       RODRI&#194;&#180;GUEZ ET AL.                                                                                                         APPL. ENVIRON. MICROBIOL. <br />  <br />  <br /> minigel apparatus. The final acrylamide monomer concentration was 12% (wt/            pTR202 with a full-<span id='am-4' about='xsp:length' typeof='owl:Thing'>length</span> accA2 gene. Finally, the NdeI-HindIII fragment from <br /> vol) for the separating gel and 5% for the stacking gel. Coomassie brilliant blue     pTR202 was cloned into the NdeI-HindIII-digested pET21a(&#226;&#171;&#185;), to yield <br /> was used to stain protein bands. Protein contents were determined by the              pTR204. <br /> method of Bradford (5) with bovine serum albumin as the standard. The relative           To provide an additional copy of accB, pIJ8600 was digested with BglII and <br /> concentration of soluble AccB and AccA2 overexpressed in E. coli was deter-           EcoRI and the fragment containing oriT from RK2, ori from pUC18, the attP site <br /> mined by densitometric scanning of the polyacrylamide-SDS gels.                       and int of &#226;&#65533;&#190;C31, and the aac(3)IV (Amr) gene was ligated with a linker contain- <br />    Acetyl-CoA carboxylase and PCCase assay. Acetyl-CoA carboxylase and                ing sites for the following restriction enzymes to yield pTR141 (Mike Butler, <br /> PCCase activities in cell extracts were measured following the incorporation of       personal communication): BglII, AseI, EcoRI, BglII, NdeI, KpnI, XbaI, PstI, <br /> HCO3&#226;&#171;&#186; into acid nonvolatile material (7, 20). The reaction mixture contained          HindIII, BamHI, SstI, and NotI. A 4.0-kb KpnI fragment containing the complete <br /> 100 mM potassium phosphate (pH 8.0), 300 &#226;&#65533;&#174;g of bovine serum albumin, 3 mM             accBE operon from pRM08 was cloned into KpnI-cleaved pTR141, yielding <br /> ATP, 5 mM MgCl2, 50 mM NaH14CO3 (specific activity, 200 &#226;&#65533;&#174;Ci mmol&#226;&#171;&#186;1 [740               pTR149. To place the chromosomal copy of accBE under the control of the <br /> kBq mmol&#226;&#171;&#186;1]), 1 mM substrate (acetyl-CoA or propionyl-CoA), and 100 &#226;&#65533;&#174;g of             TH-inducible tipA promoter, the synthetic oligonucleotides TC16 (5&#226;&#172;&#732;-ATTCTA <br /> cell-free protein extract in a total reaction volume of 100 &#226;&#65533;&#174;l. The reaction was      GACATATGACCGTTTTGGATGAGG), used to introduce an NdeI site at the <br /> initiated by the addition of NaH14CO3, allowed to proceed at 30&#194;&#176;C for 15 min,         translational start codon of the S. coelicolor accB gene, and TC17 (5&#226;&#172;&#732;-ACCTC <br /> and stopped with 200 &#226;&#65533;&#174;l of 6 M HCl. The contents of the tubes were then               TAGACAACGCTCGTGGACC) were used to amplify an internal fragment of <br /> evaporated to dryness at 95&#194;&#176;C. The residue was resuspended in 100 &#226;&#65533;&#174;l of water,        S. coelicolor accB. The reaction mixture contained 10 mM Tris-HCl (pH 8.3), 50 <br /> 1 ml of Optiphase scintillation liquid (Wallac Oy) was added, and the 14C             mM KCl, 1 mM MgCl2, 6% glycerol, 25 &#226;&#65533;&#174;M (each) deoxynucleoside triphos- <br /> radioactivity was determined in a Beckman liquid scintillation counter. Nonspe-       phate, 2 to 5 U of Taq DNA polymerase, 20 pmol of each primer, and 50 ng of <br /> cific CO2 fixation by crude extracts was assayed in the absence of substrate. One     S. coelicolor chromosomal DNA in a final volume of 100 &#226;&#65533;&#174;l. Samples were <br /> <span id='am-1' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> of enzyme activity catalyzed the incorporation of 1 &#226;&#65533;&#174;mol of 14C into acid-       subjected to 30 cycles of denaturation (95&#194;&#176;C, 30 s), annealing (65&#194;&#176;C, 30 s), and <br /> stable products per min. To confirm that the products of the reactions were           extension (72&#194;&#176;C, 1 min). The 1-kb PCR product was digested with XbaI (these <br /> malonyl- or methylmalonyl-CoA, samples were analyzed by high-performance              sites were introduced by the 5&#226;&#172;&#732; end of the oligonucleotides TC16 and TC17) and <br /> liquid chromatography (24).                                                           cloned in XbaI-cleaved pBluescript SK(&#226;&#171;&#185;), yielding pTR82. An NdeI-XbaI frag- <br />    DNA manipulations. Isolation of chromosomal and plasmid DNA, restriction           ment from the plasmid pTR82 was cloned in NdeI-XbaI-cleaved pIJ8600, yield- <br /> enzyme digestion, and agarose gel electrophoresis were carried out by conven-         ing pTR93. In order to place the chromosomal copy of the accBE operon under <br /> tional methods (22, 38). Southern analyses were performed by using 32P-labeled        the tipA promoter we removed from pTR93 a HindIII fragment containing the <br /> probes made by random oligonucleotide priming (Prime-a-gene kit; Promega).            int gene and att of &#226;&#65533;&#190;C31, yielding pTR94. <br />    Gene cloning and plasmid construction. The synthetic oligonucleotides TC1             Protein purification protocols. The His6-tagged fusion protein H6AccE was <br /> (5&#226;&#172;&#732;-CAGAATTCAAGCAGCACGCCAAGGGCAAG) and TC2 (5&#226;&#172;&#732;-CAGAAT                                 purified from cultures of RG12 [strain BL21(DE3) harboring pTR237] after the <br /> TCGATGCCGTCGTGCTCCTGGTC) were used to amplify an internal frag- <br />                                                                                       addition of 0.1 mM IPTG to induce the DE3-encoded T7 RNA polymerase. Cells <br /> ment of the S. coelicolor pccB gene. The reaction mixture contained 10 mM <br />                                                                                       were pelleted, resuspended in 50 mM phosphate buffer (pH 7.2)&#226;&#8364;&#8220;300 mM NaCl&#226;&#8364;&#8220; <br /> Tris-HCl (pH 8.3), 50 mM KCl, 1 mM MgCl2, 6% glycerol, 25 &#226;&#65533;&#174;M each de- <br />                                                                                       0.75 mM dithiothreitol&#226;&#8364;&#8220;10% glycerol, and disrupted by sonication. Cell debris <br /> oxynucleoside triphosphate, 2 &#226;&#171;&#190; 5 U of Taq or Pfu DNA polymerase, 20 pmol of <br />                                                                                       was removed by centrifugation, and the supernatant was passed through a Ni2&#226;&#171;&#185;- <br /> each primer, and 50 ng of S. coelicolor chromosomal DNA in a final volume of <br />                                                                                       nitrilotriacetic acid-agarose affinity column equilibrated with the same buffer. <br /> 100 &#226;&#65533;&#174;l. Samples were subjected to 30 cycles of denaturation (95&#194;&#176;C, 30 s), an- <br />                                                                                       The H6AccE protein was recovered by elution with 100 mM imidazole and <br /> nealing (65&#194;&#176;C, 30 s), and extension (72&#194;&#176;C, 1 min). A 1-kb PCR fragment was used <br />                                                                                       dialyzed against a solution containing 100 mM sodium phosphate (pH 7.2), 1 mM <br /> as a 32P-labeled probe to screen a size-enriched library. A 2.7-kb BamHI frag- <br />                                                                                       dithiothreitol, 1 mM EDTA, and 20% glycerol. <br /> ment containing an incomplete accB gene was cloned in BamHI-cleaved pBlue- <br />                                                                                          Nucleotide sequencing. The sequence of the SstI fragment containing accB was <br /> script SK(&#226;&#171;&#185;), yielding pTR62. The synthetic oligonucleotides TC16 <br />                                                                                       determined by subcloning ApaI fragments from pRM08 in pSKBluescript SK(&#226;&#171;&#185;). <br /> (5&#226;&#172;&#732;-TATTCTAGACATATGACCGTTTTGGATGAGG), used to introduce an <br />                                                                                       Synthetic oligonucleotides were used where needed to complete the sequence. <br /> NdeI site at the translational start codon of the S. coelicolor accB gene, and TC17 <br />                                                                                       Dideoxy sequencing (39) was carried out using the Promega TaqTrack sequenc- <br /> (5&#226;&#172;&#732;-ACCTCTAGACAACGCTCGTGGACC) were used to amplify an internal <br />                                                                                       ing kit and double-stranded DNA templates. <br /> fragment of the S. coelicolor accB gene. The reaction mixture was the same as the <br />                                                                                          S1 nuclease mapping. For each S1 nuclease reaction, 30 &#226;&#65533;&#174;g of RNA was <br /> one indicated above. Samples were subjected to 35 cycles of denaturation (95&#194;&#176;C, <br />                                                                                       hybridized in trichloroacetic acid-sodium salt (NaTCA) buffer (solid NaTCA <br /> 30 s), annealing (65&#194;&#176;C, 30 s), and extension (72&#194;&#176;C, 1 min). The PCR product was <br /> digested with XbaI and cloned in XbaI-cleaved pBluescript SK(&#226;&#171;&#186;) in E. coli            [Aldrich] was dissolved to 3 M in 50 mM PIPES, 5 mM EDTA, pH 7.0) to about <br /> DH5&#226;&#65533;&#163;, yielding pTR82. This plasmid was digested with BstEII and SacI, ligated         0.002 pmol (approximately 104 cpm) of the following probes. For accA2, the <br /> with a BstEII-SacI fragment cleaved from pMR08, and introduced by transfor-           oligonucleotide 5&#226;&#172;&#732;-GCTTTGAGGACCTTGGCGATG (accA2down) corre- <br /> mation into E. coli DH5&#226;&#65533;&#163;, yielding pTR87. An NdeI-SacI fragment from pTR87            sponding to a sequence within the coding region of accA2 was uniquely labeled <br /> was cloned in NdeI-SacI-cleaved pET22b(&#226;&#171;&#185;) (Novagen) (pTR88), thus placing             at the 5&#226;&#172;&#732; end with [32P]ATP using T4 polynucleotide kinase and then used in <br /> the accBE operon under the control of the powerful T7 promoter and ribosome-          PCR with the unlabeled oligonucleotide 5&#226;&#172;&#732;-GAAGTACAGGCCGAAGACCAC <br /> binding sequences. Expression of accB was achieved by eliminating part of the         (accA2up), which corresponds to a sequence upstream of the accA2 promoter <br /> coding sequence of accE in pTR88. For this, pTR88 was digested with NotI and          region, to generate a 766-bp probe. For accA1, the oligonucleotide 5&#226;&#172;&#732;-GCGAT <br /> the large fragment was religated to obtain pTR90. The synthetic oligonucleotides      TTCGCCACGATTGGCG (accA1down) corresponding to a sequence within <br /> NaccE (5&#226;&#172;&#732;-TTATCTAGACATATGTCCCCTGCCGAC), used to introduce an                          the coding region of accA1 was uniquely labeled at the 5&#226;&#172;&#732; end and used in a PCR <br /> NdeI site at the translational start codon of the S. coelicolor accE gene, and        with the unlabeled oligonucleotide 5&#226;&#172;&#732;-CCGATATCAGCCCCTGATGAC <br /> CaccE (5&#226;&#172;&#732;-ATGAATTCTATGCATCGGGTCAGCGCCAGCTG) were used to                              (accA1down), which corresponds to a sequence upstream of the accA1 promoter, <br /> amplify accE. The reaction mixture was the same as the one indicated above.           to generate a 563-bp probe. For accB, the oligonucleotide 5&#226;&#172;&#732;-CGTCAGCTTG <br /> Samples were subjected to 35 cycles of denaturation (95&#194;&#176;C, 30 s), annealing           CCCTTGGCGTG (accBdown) corresponding to a sequence within the coding <br /> (65&#194;&#176;C, 30 s), and extension (72&#194;&#176;C, 30 s). The PCR product was cloned using the        region of accB was labeled at the 5&#226;&#172;&#732; end and then used in a PCR with the <br /> pGEM-T easy vector (Promega) in E. coli DH5&#226;&#65533;&#163;, yielding pTR106. An NdeI-               unlabeled oligonucleotide 5&#226;&#172;&#732;-CTACGCTCCGGGTGAGCGAAC (accBup), <br /> EcoRI fragment from pTR106 was cloned in NdeI-EcoRI-cleaved pET22b(&#226;&#171;&#185;),                which corresponds to a sequence upstream of the accB promoter, to generate a <br /> yielding pTR107, thus placing the accE gene under the control of the T7 pro-          483-bp probe. For accBE, the oligonucleotide 5&#226;&#172;&#732;-GGAGGGCCGTGATGGCG <br /> moter and ribosome-binding sequences. To generate an accE His tag fusion gene         GCGACTTCCTCGGG (accBEdown) corresponding to a sequence within the <br /> (full-length accE fused to six His codons at its N terminus), the NdeI-EcoRI          coding region of accE was labeled at the 5&#226;&#172;&#732; end and used in a PCR with the <br /> fragment from pTR107 was cloned in NdeI-EcoRI-cleaved pET28a(&#226;&#171;&#185;), yielding             unlabeled oligonucleotide 5&#226;&#172;&#732;-GAGGAACTGGTACGCGCGGGCG[GTACAA <br /> pTR237. For the production of high levels of AccA2, we constructed pTR204.            GCAAGCT] (accBEup) corresponding to a sequence in the coding region of <br /> For that the synthetic oligonucleotides accANd (5&#226;&#172;&#732;- CATATGCGAAAGGTG                   accB (bracketed oligonucleotides constitute a tail added to the probe to differ- <br /> CTCATCGCCAATC) and accABa (5&#226;&#172;&#732;-AAAGCGTTCTCCGAGAGGAATCC                                 entiate probe reannealing from full-length protection) to generate a 563-bp <br /> GTAGC) were used to amplify the N terminus of accA2 and to introduce an               probe. Subsequent steps were performed as described by Strauch et al. (41). <br /> NdeI site at the translational start codon of the gene. The PCR fragment was             Nucleotide sequence accession number. The accB and accE genes were iden- <br /> cloned into PCR-Blunt (Invitrogen) to yield pTR200. A BamHI-KpnI fragment             tified in cosmid SC1C2 (S. coelicolor genome project [http://www.sanger.ac.uk <br /> from pTR45 (36) was cloned into the BamHI-KpnI-digested pTR200, yielding              /Projects/S_coelicolor/]; nucleotide accession number AL031124). <br />  VOL. 67, 2001                                           ANALYSIS OF AN ACYL-CoA CARBOXYLASE OF S. COELICOLOR                              4169 <br />  <br />  <br />  <br />  <br />    FIG. 1. Organization of the region of the S. coelicolor M145 chromosome containing the accB and accE genes. (A) Genetic and physical map <br /> of the 6.2-kb insert in pRM08. The secondary structure downstream of accE may represent a factor-independent transcriptional terminator. Probes <br /> 1 and 2 were generated by PCR using the oligonucleotides accBup-accBdown and accBEup-accBEdown, respectively, uniquely labeled at the 5&#226;&#172;&#732; <br /> end (&#226;&#180;&#177;) and were used in transcriptional analysis of the accBE operon. (B) Map of the DNA fragments cloned in pET22b that were used for <br /> expression of accB and/or accE in E. coli. Only relevant restriction sites are shown: B, BamHI; Bc, BclI; E, EcoRI; K, KpnI; Nd, NdeI; N, NotI; <br /> S, SpHI. <br />  <br />                             RESULTS                                          Heterologous expression of accB, accE, and in vitro recon- <br />                                                                           stitution of an ACCase complex. Recently, we achieved recon- <br />    Cloning the accBE genes. Since pccB mutants of S. coelicolor           stitution of a PCCase complex activity by mixing E. coli cell <br /> produce wild-type levels of acetyl-CoA carboxylase (36), we               extracts containing PccB (the carboxyl transferase) with cell <br /> foresaw that a second gene encoding a different carboxyl trans-           extracts containing the biotinylated subunits AccA1 and <br /> ferase &#226;&#65533;&#164; subunit capable of recognizing acetyl-CoA as a sub-              AccA2 (36). To assess whether AccB and AccE were compo- <br /> strate should exist in this organism. Based on the high level of          nents of a previously uncharacterized carboxylase complex, we <br /> sequence homology shown by genes encoding putative carboxyl               attempted similar in vitro reconstitution experiments with <br /> transferases in the same species (e.g., in Mycobacterium tuber-           crude extracts containing these proteins. Since E. coli does <br /> culosis [10]), we attempted to clone this alternative &#226;&#65533;&#164; subunit <br />                                                                           not contain PCCase and acetyl-CoA carboxylase activity can- <br /> gene using pccB as a hybridization probe. When a BamHI <br />                                                                           not be assayed directly by carboxylation of acetyl-CoA (34), the <br /> digest of S. coelicolor DNA was probed with pccB under con- <br />                                                                           acetyl-CoA carboxylase activity measured in these crude ex- <br /> ditions of low stringency, a second poorly hybridizing band was <br />                                                                           tracts represents the activity of heterologous complexes recon- <br /> readily detected (data not shown). This hybridizing sequence <br />                                                                           stituted in vitro. <br /> was cloned from a size-enriched library as a 2.5-kb BamHI <br />                                                                              Overexpression of accB and accE in E. coli was attempted <br /> fragment. Sequencing revealed the presence of an incomplete <br />                                                                           with strain RG8, a BL21(DE3) strain containing pTR88 (Fig. <br /> open reading frame (ORF) with high homology to pccB; the <br />                                                                           1). SDS-PAGE of crude extracts of RG8, prepared from <br /> complete gene was subsequently cloned on a 6-kb SstI frag- <br /> ment, yielding pRM08 (Fig. 1). Sequencing of this fragment                IPTG-induced cultures, revealed overexpression of a 57-kDa <br /> revealed a putative protein with end-to-end similarity to a               protein, corresponding to the predicted size of AccB. In the <br /> likely decarboxylase of Streptomyces cyanogenus (76% identity             same electrophoretic analysis no clearly identifiable AccE <br /> [46]), to PccB from S. coelicolor (57% identity [36]), and to the         band was observed. In vitro reconstitution of ACCase activity <br /> &#226;&#65533;&#164; subunit (PccB) of the Saccharopolyspora erythraea PCCase                was then obtained by mixing a crude extract prepared from an <br /> (56% identity [11]). The gene encoding this new putative car-             IPTG-induced culture of RG8 with a cell extract of E. coli <br /> boxyl transferase was called accB.                                        strain RG11, which overproduces the biotinylated protein <br />    The sequence also revealed the presence of a small ORF,                AccA2 and the E. coli biotin ligase BirA, harbored in plasmids <br /> accE, whose start codon was only 17 bp downstream of the                  pTR204 and pBA11, respectively. After incubation for 1 h at <br /> termination codon of accB. A 17-nucleotide (nt) inverted re-              4&#194;&#176;C, the mixture was assayed for acetyl-CoA carboxylase and <br /> peat which could function as a factor-independent bidirec-                PCCase activities. As shown in Table 2, an enzyme complex <br /> tional transcriptional terminator separates accE from three               with both acetyl-CoA carboxylase and PCCase activities was <br /> convergent ORFs with homology to putative proteins of                     readily detected, confirming that AccB was the carboxyl trans- <br /> M. tuberculosis of unknown function. The putative AccE pro-               ferase component of an ACCase complex. Similar results were <br /> tein has a deduced molecular mass of 7.5 kDa and does not                 obtained when the reconstitution experiments were performed <br /> resemble any other known protein. The region upstream of                  using cell extracts of strain RG7, a BL21(DE3) strain contain- <br /> accB encodes a putative protein which is highly homologous to             ing pCL1 that provides AccA1 instead of AccA2 as the biotin- <br /> several known hyaluronidases.                                             ylated component of the ACCase. The lower levels of both <br />  4170         RODRI&#194;&#180;GUEZ ET AL.                                                                                             APPL. ENVIRON. MICROBIOL. <br />  <br />  <br /> TABLE 2. Heterologous expression of ACCase components in cell                        but not AccE. Although ACCase activity was readily detected <br />  extracts of E. coli and in vitro reconstitution of enzyme activity                  in this mixture, indicating that AccE is not catalytically neces- <br />                                               Protein expression (mU mg of           sary for the successful reconstitution of an active complex in <br />    E. coli        Protein(s) induced            protein&#226;&#171;&#186;1) in cell extractsb         vitro, the levels of acetyl-CoA carboxylase and PCCase activ- <br />    straina             by IPTG <br />                                          Acetyl-CoA carboxylase        PCCase        ities were considerably lower (approximately 30%) than those <br />        c                                                 e                           obtained with cell extracts that contained AccB and AccE (Fig. <br /> RG7             AccA1                            &#226;&#172;&#65533;0.02                 &#226;&#172;&#65533;0.02e <br /> RG11c           AccA2                            &#226;&#172;&#65533;0.02e                &#226;&#172;&#65533;0.02e        2, compare mixes 1 and 2). Since the levels of AccB in the cell <br /> RG8             AccB, AccE                       &#226;&#172;&#65533;0.02e                &#226;&#172;&#65533;0.02e        extracts of RG8 and RG9 were essentially the same, we in- <br /> RG11-RG8d       AccA2-AccB, AccE               7.53 &#226;&#171;&#190; 0.18           9.10 &#226;&#171;&#190; 0.20     ferred from these experiments that AccE was necessary to <br /> RG7-RG8d        AccA1-AccB, AccE               1.85 &#226;&#171;&#190; 0.12           2.25 &#226;&#171;&#190; 0.13 <br />                                                                                      obtain a fully active ACCase complex. To confirm that the <br />   a <br />     All the RG strains are derived from E. coli BL21(DE3), except RG7, which         absence of AccE was responsible for the lower ACCase activity <br /> derives from DH5&#226;&#65533;&#163;. <br />   b <br />     Results are the means of three determinations &#226;&#171;&#190; standard errors.                 observed, we studied the effect that the addition of cell extracts <br />   c <br />     Contains plasmid pBA11 that expresses BirA constitutively.                       containing high levels of soluble AccE [strain RG10, a BL21 <br />   d <br />     Mix of equal amounts of proteins from cell extracts of each of the strains <br /> indicated. <br />                                                                                      (DE3) strain containing pTR107] had on the ACCase activity <br />   e <br />     The amount of 14C fixed into acid-stable products was not significantly higher   present in a mix of crude extracts containing AccB and AccA2. <br /> than background levels (10 cpm, equivalent to 0.02 mU).                              As shown in Fig. 2 (mixes 2 and 3) the specific activities of both <br />                                                                                      acetyl-CoA carboxylase and PCCase were almost 3.5 times <br /> acetyl-CoA carboxylase and PCCase activity are due to the                            higher in the presence of AccE than in the control experiment <br /> lower level of expression of accA1 by pCL1 (36). These results                       that lacked this protein and resembled those values obtained <br /> confirmed that either AccA1 or AccA2 could be used effi-                             by mixing RG8 (AccBE) and RG11 (AccA2) cell extracts. <br /> ciently, at least in vitro, as the &#226;&#65533;&#163; subunit of the enzyme complex                   Similar results were obtained when purified H6AccE was <br /> (Table 2).                                                                           added to the AccB-AccA2 mix (Fig. 2, mixes 4 to 9). The <br />    Is AccE a functional component of the ACCase complex?                             addition of different amounts of H6AccE (from values ranging <br /> The genetic organization of accB and accE as members of the                          from 0.1 to 10 &#226;&#65533;&#174;g of pure protein) increased the levels of <br /> same transcription <span id='am-2' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> suggested that AccE could also be a                          acetyl-CoA carboxylase activity, reaching saturation when <br /> functional component of the ACCase complex. To investigate                           more than 2 &#226;&#65533;&#174;g of AccE was present in the reaction mix. The <br /> this hypothesis we assayed acetyl-CoA carboxylase and PCCase                         fact that the maximum level of enzyme activity was obtained at <br /> activities in a mixture of cell extracts that contained AccB                         high concentrations of AccE proposed a direct participation of <br /> [strain RG9, a BL21(DE3) strain containing pTR90] and                                this protein in the activation of the complex formed by AccB <br /> AccA2 [strain RG11, a BL21(DE3) strain containing pTR204]                            and AccA2. Although the results presented in this section <br />  <br />  <br />  <br />  <br />    FIG. 2. Effect of AccE on the catalytic activity of the ACCase complex. In mixes 1, 2, and 3, acetyl-CoA carboxylase and PCCase activities were <br /> measured after mixing equal amounts of proteins from cell extracts from each of the strains indicated. In mixes 4 to 9, ACCase activity was <br /> determined using a mix of RG9 and RG11 cell extracts containing different amounts of purified H6AccE. Results are the means of three <br /> determinations. When ACCase activity was measured in individual cell extracts, the amount of 14C fixed into acid-stable products was not <br /> significantly higher than background levels (10 cpm, equivalent to 0.02 mU). <br />  VOL. 67, 2001                                            ANALYSIS OF AN ACYL-CoA CARBOXYLASE OF S. COELICOLOR                               4171 <br />  <br />  <br />  <br />  <br />   FIG. 3. Attempted disruption of accB. (A) Diagram showing integration of pTR124 through one of the accBE flanking regions and resolution <br /> of the cointegrate by a second crossover event. The &#226;&#171;&#187; on top of the arrow indicates the inability to obtain the replacement of the wild-type accB <br /> by the Hygr mutant allele. (B) Integration of a second copy of accBE at the &#226;&#65533;&#190;C31 att site of T124 (to yield strain T149) allowed replacement of <br /> the wild-type accB by the mutant allele. <br />  <br />  <br /> suggest that AccE increases the rate of the ACCase reaction,               permit a second crossover event, leading to the replacement of <br /> kinetic analysis using purified components will be necessary to            the wild-type accB gene by the Hygr mutant allele. To confirm <br /> understand the precise role played in enzyme activity by this              this hypothesis, we first integrated pTR149 (see Materials and <br /> small polypeptide.                                                         Methods; Fig. 3B) containing accBE and the native promoter <br />    accB is an essential gene in S. coelicolor. To study the role of        into the &#226;&#65533;&#190;C31 attB site of T124 (the presence of accE in this <br /> AccB in vivo, we attempted to construct an accB mutant by                  construct would also cater for any polar effect on the expres- <br /> gene replacement (Fig. 3A). A HYG resistance cassette was                  sion of accE caused by disruption of the native copy of accB). <br /> cloned in the unique BamHI site present in the coding se-                  The resulting strain, T149 (Hygr Thr Amr), was subjected to <br /> quence of accB contained in pTR80. After an intermediate                   three rounds of sporulation on SFM agar containing HYG and <br /> cloning step in pIJ2925, a BglII fragment containing the mu-               APR, and after screening approximately 500 colonies, 20 were <br /> tated allele was inserted in the conjugative E. coli vector                found to be Amr Hygr Ths; one of these was designated T149A. <br /> pSET151. The resulting plasmid, pTR124, was introduced into                Disruption of accB, but only in the presence of an additional <br /> the E. coli donor strain ET12567/pUZ8002 and transferred by                copy of the gene (i.e., in strain T149A), was confirmed by <br /> conjugation into M145. Thr Hygr exconjugants were selected in              Southern analysis using an internal fragment of accB as a <br /> which the plasmid had integrated into the chromosome at the                hybridization probe. These results confirmed the essentialness <br /> accB locus by a single crossover. One of the exconjugants,                 of AccB for S. coelicolor viability. <br /> T124, was taken through four rounds of sporulation on SFM                     Construction and characterization of an accBE conditional <br /> medium with HYG to allow for a second crossover and                        mutant. In order to regulate the expression of the putative <br /> replacement of the wild-type accB with the mutant allele.                  accBE operon and study its effect on the physiology of S. coeli- <br /> Although several thousand colonies were screened for TH                    color, we constructed a conditional mutant strain in which the <br /> sensitivity (which would have reflected successful gene replace-           expression of these genes was under the control of the TH- <br /> ment), none were obtained, suggesting that accB could be an                inducible tipA promoter (30). For this, pTR94 was transformed <br /> essential gene in S. coelicolor. If this were true, the presence of        into the E. coli strain ET12567/pUZ8002 and conjugated into <br /> a second copy of accB in the chromosome of T124 ought to                   the S. coelicolor strain M145. Integration of pTR94 by Camp- <br />  4172     RODRI&#194;&#180;GUEZ ET AL.                                                                                             APPL. ENVIRON. MICROBIOL. <br />  <br />  <br />                                                                          were also described for an accBC conditional mutant of Bacil- <br />                                                                          lus subtilis (32). Interestingly, although growth was restored in <br />                                                                          oleate-supplemented medium, the cultures were still impaired <br />                                                                          in antibiotic production and the levels of acetyl-CoA carboxy- <br />                                                                          lase activity were the same as those found in the absence of the <br />                                                                          inducer (Table 3). All these results strongly suggest that AccB <br />                                                                          is the carboxyl transferase component of an essential ACCase <br />                                                                          complex whose main physiological role appears to be the sup- <br />                                                                          ply of malonyl-CoA for both fatty acid and polyketide biosyn- <br />                                                                          thesis. <br />                                                                             Transcriptional analysis of accBE, accA1, and accA2. Bio- <br />                                                                          synthesis of malonyl-CoA in S. coelicolor should occur not only <br />                                                                          during exponential phase, when the synthesis of fatty acids is <br />                                                                          essential, but also during transition and stationary phase to <br />   FIG. 4. Effects of TH and various fatty acids on growth of S. coeli-   provide the elongation units for the synthesis of actinorhodin <br /> color M94 bearing the tipAp-accBE fusion. Cultures of strain M94 were <br /> grown in SMM medium containing 10 mg of APR ml&#226;&#171;&#186;1 (&#228;&#161;&#186;) or the              and undecylprodigiosin. Genetics and biochemical data pro- <br /> same medium supplemented with TH (5 &#226;&#65533;&#174;g/ml) (E) or with the fol-          pose that AccB forms part of the main ACCase of S. coelicolor <br /> lowing fatty acids at 0.01%: octanoic acid (f), palmitic acid (&#198;&#8217;), and   involved in the biosynthesis of malonyl-CoA and that either <br /> oleic acid (F).                                                          AccA2 or AccA1 could function as the biotinylated compo- <br />                                                                          nents of this enzyme complex. In order to study the levels of <br /> bell recombination through the accBE homologous sequences                transcription of the enzyme components and hopefully gain <br /> left the accBE operon under tipAp. The strain obtained was               more information into the subunit composition of the complex <br /> named M94 and the genetic modification introduced was con-               throughout growth we performed transcriptional studies of the <br /> firmed by Southern blot experiments (data not shown).                    accBE, accA1, and accA2 genes. <br />    Normal growth of strain M94 on SMM depended on the                       S. coelicolor A3(2) strain M145 was grown in SMM medium <br /> presence of 5 &#226;&#65533;&#174;g of TH/ml, which derepresses the expression of           and RNA was extracted during the exponential, transition, and <br /> the accBE operon. In the absence of TH growth was strongly               stationary phases of growth. S1 nuclease protection analysis <br /> affected, and the low growth levels observed were probably due           of accB mRNA was performed using a 483-bp PCR product, <br /> to a leakiness of the control system (Fig. 4) (E. Takano and M.          uniquely labeled at the 5&#226;&#172;&#732; end of the downstream oligonucle- <br /> Bibb, unpublished data); no antibiotic production was ob-                otide. Transcription of accB occurred primarily during active <br /> served in these cultures. To determine the effect of TH on the           growth (exponential and transition phases) and then declined <br /> acetyl-CoA carboxylase and PCCase enzyme levels, both activ-             significantly upon entry into stationary phase (Fig. 5A). The tran- <br /> ities were measured in 38-h cultures grown in SMM with or                scripts of the major and essential sigma factor gene of S. coeli- <br /> without the addition of 5 &#226;&#65533;&#174;g of TH/ml. We used this time point           color, hrdB, and of the pathway-specific activator gene for ac- <br /> because both cultures were still in their exponential phase and          tinorhodin biosynthesis, actII-ORF4, were monitored as controls. <br /> we expected, at least for the acetyl-CoA carboxylase activity, its       As expected from previous work, hrdB was expressed through- <br /> maximal levels. As observed in Table 3, the acetyl-CoA car-              out growth (9), while the actII-ORF4 transcript peaked during <br /> boxylase activity present in crude extracts prepared from the            transition phase and disappeared in stationary phase (13). <br /> uninduced cultures was almost 10 times lower than that found                The RNA-protected fragment identified for accB corre- <br /> in the TH-induced cultures. This difference was not observed             sponds to a transcript that would start 1 bp upstream of or at <br /> in the levels of PCCase, a result that was expected considering          the adenine of the most likely translation start codon of accB. <br /> that the ACCase containing AccB as a &#226;&#65533;&#164; subunit is only one of            Putative &#226;&#171;&#186;10 and &#226;&#171;&#186;35 promoter regions similar to those likely <br /> the three known complexes with PCCase activity in S. coeli-              to be recognized by &#226;&#65533;&#180;hrdB (42) are located upstream of the <br /> color (7, 36). These results correlate the growth deficiency of          transcription initiation site (Fig. 5B). <br /> the M94 conditional mutant with the low levels of acetyl-CoA                To determine if accB and accE were cotranscribed, a 563-bp <br /> carboxylase activity in the absence of the inducer and strongly          probe was generated by PCR that spanned the intergenic re- <br /> support the hypothesis that AccB is an essential protein for             gion. For this we used a 5&#226;&#172;&#732; oligonucleotide corresponding to a <br /> S. coelicolor viability.                                                 sequence within the coding region of accB and a 3&#226;&#172;&#732; oligonu- <br />    Since acetyl-CoA carboxylase catalyzes the synthesis of ma- <br /> lonyl-CoA, the primer for the elongation step of fatty acids, we <br /> investigated whether the growth defect showed by M94 could                      TABLE 3. Acetyl-CoA carboxylase and PCCase activities <br /> be corrected by growing it in the presence of different fatty                            in cell extracts of S. coelicolor M94 <br /> acids. M94 grew very poorly in SMM (Fig. 4); however, <br />                                                                                                               Activity (mU mg of protein&#226;&#171;&#186;1)a of: <br /> when the SMM medium was supplemented with oleic acid,                     Growth medium <br /> a straight-chain unsaturated fatty acid, growth was restored                                            Acetyl-CoA carboxylase                 PCCase <br /> to normal levels (Fig. 4). The growth of the mutant was not              SMM                                 0.12 &#226;&#171;&#190; 0.03                     2.20 &#226;&#171;&#190; 0.06 <br /> stimulated by the straight-chain octanoic acid or palmitic acid,         SMM &#226;&#171;&#185; THb                           1.24 &#226;&#171;&#190; 0.06                     3.90 &#226;&#171;&#190; 0.07 <br /> indicating that these saturated fatty acids are not incorporated         SMM &#226;&#171;&#185; Oleateb                       0.15 &#226;&#171;&#190; 0.03                     1.40 &#226;&#171;&#190; 0.05 <br /> efficiently into S. coelicolor membrane phospholipids or that             a <br />                                                                               Results are means of three determinations &#226;&#171;&#190; standard errors. <br /> the resulting membranes are not functional. Similar results               b <br />                                                                               TH, 5 &#226;&#65533;&#174;g/ml; oleate, 0.01% (wt/vol). <br />  VOL. 67, 2001                                              ANALYSIS OF AN ACYL-CoA CARBOXYLASE OF S. COELICOLOR                       4173 <br />  <br />  <br />                                                                            showed that accB and accE were part of the same transcript <br />                                                                            (Fig. 5C). The <span id='am-162' about='xsp:pattern' typeof='owl:Thing'>pattern</span> of transcription of accBE during the <br />                                                                            different growth phases corresponded well to the profile ob- <br />                                                                            served with the accB probe. <br />                                                                               The transcripts of accA2 present during exponential phase <br />                                                                            were studied by high-resolution S1 mapping. The probe used <br />                                                                            was a 766-bp PCR-generated DNA fragment uniquely labeled <br />                                                                            at the 5&#226;&#172;&#732; end of the oligonucleotide corresponding to a se- <br />                                                                            quence within accA2. The experimental data revealed the pres- <br />                                                                            ence of two RNA protected fragments, consistent with tran- <br />                                                                            scripts initiated 25 bp (from accA2p1) and 153 bp (from <br />                                                                            accA2p2) upstream of the putative translation start site of <br />                                                                            accA2 (Fig. 6A and C). The putative &#226;&#171;&#186;10 and &#226;&#171;&#186;35 regions of <br />                                                                            these promoters also show some similarity to the consensus <br />                                                                            sequence of promoters that are likely to be recognized by <br />                                                                            &#226;&#65533;&#180;hrdB. The growth phase-dependent expression of accA2 from <br />                                                                            these two putative promoters closely resembled that observed <br />                                                                            for the accBE operon; i.e., there was a constant and high level <br />                                                                            of expression during the exponential and transition phases of <br />                                                                            growth that declined markedly upon entry into stationary <br />                                                                            phase (Fig. 7A). However, a new RNA protected fragment of <br />                                                                            185 bp was also detected during transition phase. Since the <br />                                                                            nucleotide sequences of accA1 and accA2 are identical from nt <br />                                                                            &#226;&#171;&#186;2 to nt &#226;&#171;&#185;200 with respect to the coding sequence (36), 185 bp <br />                                                                            of this probe should also be protected by the accA1 mRNA. <br />                                                                            Thus, although the existence of a third promoter for accA2 that <br />                                                                            is regulated in a different manner cannot be ruled out, this <br />                                                                            transcript could also correspond to accA1. <br />                                                                               S1 nuclease protection analysis of accA1 was performed <br />                                                                            using a 563-bp PCR product uniquely labeled at the 5&#226;&#172;&#732; end of <br />                                                                            the downstream oligonucleotide corresponding to a sequence <br />                                                                            within accA1. Two major RNA protected fragments were iden- <br />                                                                            tified (as well as a faint band designated accA1p3 in Fig. 6B), <br />                                                                            with the most abundant representing a putative transcriptional <br />                                                                            start site located 88 bp upstream of the GTG initiation co- <br />                                                                            don of AccA1. Putative &#226;&#171;&#186;10 and &#226;&#171;&#186;35 regions resembling those <br />                                                                            likely to be recognized by &#226;&#65533;&#180;hrdB are again located upstream of <br />                                                                            each of the two more prominent start sites (Fig. 6B and C). <br />                                                                            Two direct repeat sequences of 16 bp containing only two <br />                                                                            mismatches flank the putative &#226;&#171;&#186;35 region of accA1p1 and the <br />                                                                            transcription start point of accA1p2 and might represent the <br />                                                                            binding sites of a putative transcriptional regulator (Fig. 6C). <br />                                                                            S1 nuclease protection experiments with RNA from different <br />                                                                            growth phases revealed accA1 transcripts exclusively during <br />                                                                            transition phase (Fig. 7B), showing a completely different reg- <br />    FIG. 5. Growth phase-dependent expression and transcription start <br />                                                                            ulation than accA2 and suggesting that the smallest RNA pro- <br /> site of the accBE operon. (A) S1 nuclease mapping of accB, actII- <br /> ORF4, and hrdB, using RNA isolated from a liquid-grown culture of          tected fragment detected for accA2 during transition phase <br /> S. coelicolor M145. Exp, Trans, and Stat indicate the exponential,         most probably reflects transcription of accA1. <br /> transition, and stationary phases of growth, respectively. (B) The nu- <br /> cleotide sequence of both strands of the accB promoter region is <br /> shown. The arrow indicates the most likely transcription start point for                           DISCUSSION <br /> the accBE promoter, as determined by S1 nuclease mapping. Potential <br /> &#226;&#171;&#186;10 and &#226;&#171;&#186;35 regions for accBEp are underlined. (C) S1 nuclease                In Streptomyces malonyl-CoA is not only an essential metab- <br /> mapping of the accB-accE intergenic region using a 563-nt probe. FLP,      olite used as the main elongation <span id='am-163' about='Thesaurus:Unit' typeof='owl:Thing'>unit</span> for fatty acid biosynthe- <br /> full-length protection of the probe reflecting transcription across the    sis (4, 8) but also one of the most common building blocks <br /> intergenic region. <br />                                                                            utilized in the synthesis of several pharmaceutically important <br />                                                                            polyketide compounds (19). Therefore, the interest in estab- <br /> cleotide corresponding to a sequence within accE. The addi-                lishing the pathway(s) leading to the biosynthesis of this met- <br /> tion of a 13-nt tail to the 5&#226;&#172;&#732; oligonucleotide allowed for facile          abolic intermediate in this microorganism has relevance not <br /> discrimination of full-length protection (reflecting cotranscrip-          only from a fundamental view but also from a more applied <br /> tion) of the probe from probe reannealing. The results clearly             point of view. <br />  4174      RODRI&#194;&#180;GUEZ ET AL.                                                                                           APPL. ENVIRON. MICROBIOL. <br />  <br />  <br />                                                                             ylated proteins AccA1 and AccA2 confirmed that AccB was <br />                                                                             the carboxyl transferase subunit of an ACCase. The additional <br />                                                                             presence of AccE considerably enhanced the activity of the <br />                                                                             enzyme complex (Table 2), suggesting that this small polypep- <br />                                                                             tide is a functional component of the ACCase. Whether this <br />                                                                             protein plays a role as an allosteric regulator of the enzyme or <br />                                                                             as a structural component of the complex remains to be elu- <br />                                                                             cidated. All the actinomycete ACCases studied so far contain <br />                                                                             three functional domains located in two polypeptides (18, 20). <br />                                                                             Thus, AccE, for which there are no known homologues, might <br />                                                                             be a distinctive feature of ACCases from Streptomyces spp. <br />                                                                                Based on these biochemical studies we decided to prove in <br />                                                                             vivo whether AccB was the carboxyl transferase component of <br />                                                                             an essential ACCase. The impossibility of obtaining an accB <br />                                                                             null mutant and the TH growth dependency of a tipAp-accB <br />                                                                             conditional mutant (Fig. 3A and 4) confirmed that AccB is <br />                                                                             essential for S. coelicolor viability. A normal growth phenotype <br />                                                                             in the absence of the inducer was restored in the conditional <br />                                                                             mutant by the addition of exogenous long-chain fatty acids <br />                                                                             in the medium (Fig. 4), indicating that the inducer-depen- <br />                                                                             dent phenotype was specifically related to a conditional block <br />                                                                             in fatty acid biosynthesis and that the acetyl-CoA carboxylase <br />                                                                             activity of the ACCase complex, containing AccB as the car- <br />                                                                             boxyl transferase subunit, is the main pathway of malonyl-CoA <br />                                                                             biosynthesis in S. coelicolor. Although normal growth was re- <br />                                                                             stored by unsaturated fatty acids in liquid SMM medium, we <br />                                                                             were unable to obtain an accB mutant of T124 in the presence <br />                                                                             of oleate after several rounds of sporulation in SFM medium <br />                                                                             (41) supplemented with oleate and APR. We suggest that de <br />                                                                             novo fatty acid synthesis may be essential for an efficient sporu- <br />                                                                             lation of this microorganism, as was shown in B. subtilis in <br />  <br />  <br />  <br />  <br />    FIG. 6. Mapping of the accA2 and accA1 transcription start points. <br /> (A and B) High-resolution S1 nuclease mapping of the 5&#226;&#172;&#732; end of accA2 <br /> transcripts. Lanes 1, RNA protected products of the S1 nuclease pro- <br /> tection assay; lanes 2 to 5, A, C, G, and T lanes of a dideoxy sequencing <br /> ladder using the same oligonucleotide that was used to make the S1 <br /> probe (accA2down for accA2 and accA1down for accA1). &#226;&#180;&#177;, uniquely <br /> labeled with 32P at the 5&#226;&#172;&#732; end. (C) Sequence of the accA2 and accA1 <br /> upstream regions, indicating the most likely transcription start point(s) <br /> for the accA1 and accA2 promoters (bent arrows). Potential &#226;&#171;&#186;10 and <br /> &#226;&#171;&#186;35 regions are underlined. Potential ribosomal binding sites (rbs) are <br /> in bold. The 17-nt direct repeats found upstream of the transcription <br /> start point of accA1p1 are indicated with straight arrows. <br />  <br />  <br />    In most species, malonyl-CoA is synthesized through car- <br /> boxylation of acetyl-CoA by an acetyl-CoA carboxylase (45), <br /> and this enzyme complex has been shown to be essential for <br /> many microorganisms, such as E. coli, B. subtilis, and Saccha- <br /> romyces cerevisiae (17, 28, 32). Based on this knowledge and in <br /> an attempt to characterize the malonyl-CoA biosynthetic path- <br /> way in S. coelicolor we searched for a carboxyl tranferase com- <br /> ponent that could function as the &#226;&#65533;&#164; subunit of an acetyl-CoA <br /> carboxylase complex. Thus, by using pccB (36) as a hybridiza-                  FIG. 7. Growth phase-dependent expression of accA2 and accA1. <br />                                                                             S1 nuclease mapping of accA2 (A) and accA1 (B), using RNA isolated <br /> tion probe we isolated the accBE operon of S. coelicolor. Ex-               from a liquid-grown culture of S. coelicolor M145 harvested at different <br /> pression of accB and accE in E. coli and subsequent in vitro                stages of growth, is shown. Exp, Trans, and Stat indicate the exponen- <br /> reconstitution of enzyme activity in the presence of the biotin-            tial, transition, and stationary phases of growth, respectively. <br />  <br /> which fatty acid synthesis is essential to couple the activation of   ed component of the ACCases. Interestingly, AccA of M. xan- <br /> the mother cell transcription factors with the formation of           thus was not essential for the viability of this microorganism <br /> differentiating cells (40). If this hypothesis was correct accB       and mutants in this subunit only affected the intracellular levels <br /> mutants would not be able to sporulate, even in the presence of       of acetyl-CoA carboxylase but not of the PCCase activity, <br /> oleate, and would be lost in the isolation procedure utilized.        showing a sharp difference with our findings. <br />    Considering the essential role played by AccB and taking <br /> into account the apparent inviability of accA2 mutants in                                        <br /> S. coelicolor (36), we postulate that AccA2 and AccB are the &#226;&#65533;&#163; <br />                                                                        . <br /> and &#226;&#65533;&#164; components of an ACCase, whose main physiological        <br /> role is the synthesis of malonyl-CoA. Transcriptional studies of      <br /> accBE and accA2 showed that the expression of these genes             <br /> occurred principally during the exponential and transition  <br /> phases of growth (Fig. 5A and 6A), in agreement with their   <br /> essential role in this organism. Consistent with these results the <br /> levels of acetyl-CoA carboxylase and PCCase activity through- <br /> out growth were also found to be maximal during exponential                                             <br /> phase (data not shown).                 <br />    In S. coelicolor, in addition to the need for malonyl-CoA  <br /> synthesis during vegetative growth, there is also a requirement   <br /> for this metabolite during transition and stationary phase. At  <br /> least two of the secondary metabolites produced by S. coeli-    <br /> color, undecylprodigiosin and actinorhodin, are synthesized       <br /> during these growth phases and require malonyl-CoA for their  <br /> synthesis. If the essential ACCase characterized in this work is      <br /> the only enzyme capable of synthesizing malonyl-CoA, then it        <br /> will also be required during the production of these two anti-   <br /> biotics. In agreement with this hypothesis fatty acid-supple-       <br /> mented cultures of the M94 conditional mutant, for which             <br /> ACCase activity was barely detectable, were unable to produce   <br /> actinorhodin or undecylprodigiosin. Based on the proposed        <br /> composition of the enzyme complex and on the transcriptional  <br /> studies reported here, we suggest that the low level of expres-          <br /> sion of accA2 and accBE that occurs during stationary phase       <br /> provides enough of the &#226;&#65533;&#163; and &#226;&#65533;&#164; components to produce suffi-  <br /> cient ACCase for secondary metabolism. If this assumption is   <br /> correct, the biosynthesis of polyketide antibiotics in S. coeli-    <br /> color should be improved by overproduction of the ACCase   <br /> components during stationary phase.                    <br />    While the burst of accA1 transcription during transition       <br /> phase could provide a new biotinylated component for the                <br /> ACCase complex during stationary phase, <span id='am-84' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-85' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-86' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-89' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-91' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-92' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-94' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-97' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-98' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-99' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-105' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-106' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-107' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-109' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-117' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-118' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-120' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-121' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-127' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-129' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-131' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-133' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-134' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-135' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-136' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-137' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-143' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-146' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-150' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-152' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-153' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-154' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-158' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-160' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-161' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span>mutation</span> of accA1                <br /> did not change the level of acetyl-CoA carboxylase or PCCase   <br /> throughout growth. Moreover, this mutation has no deleteri-        <br /> ous effect on antibiotic production in S. coelicolor (36); conse-       <br /> quently, the physiological role of AccA1 remains uncertain     <br /> (although its location in cosmid AH10 [35] adjacent to a new      <br /> putative PKS cluster might suggest a role in the synthesis of    <br /> a hitherto unknown polyketide). For instance, a gene, jadJ,            <br /> whose deduced amino acid sequence showed a high degree of                 <br /> similarity with that of AccA1 (70% identity) has been recently    <br /> located in the gene cluster associated with jadomycin B bio-          <br /> synthesis in Streptomyces venezuelae (14). Disruption of jadJ      <br /> had no effect on growth or morphology of the organism, im-           <br /> plying that the product of this gene was not essential for fatty       <br /> acid biosynthesis, but the mutant did show a reduced produc-          <br /> tion of jadomycin.                                                  <br />    Recently, a two-component acetyl-CoA carboxylase was par-       <br /> tially characterized in Myxococcus xanthus (23). The biotinyl-        <br />                                                                      <br /> ated component of this enzyme complex, AccA, also contains           <br /> the BC domain, resembling the organization of the biotinylat-          <br /> </body></html>