<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:protege="http://protege.stanford.edu/plugins/owl/protege#" xmlns:xsp="http://www.owl-ontologies.com/2005/08/07/xsp.owl#" xmlns:Thesaurus="http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Dec. 2001, p. 6979&#226;&#8364;&#8220;6990                                                                             Vol. 183, No. 24 <br /> 0021-9193/01/$04.00&#226;&#171;&#185;0 DOI: 10.1128/JB.183.24.6979&#226;&#8364;&#8220;6990.2001 <br /> Copyright &#194;&#169; 2001, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />       Two Nonredundant SecA Homologues Function Mycobacteria <br />       MIRIAM BRAUNSTEIN,1,2* AMANDA M. BROWN,1&#226;&#8364;&#160; SHERRY KURTZ,2                                    AND   WILLIAM R. JACOBS, JR.1 <br />        Howard Hughes Medical Institute, Department Microbiology Immunology, Albert Einstein College Medicine, <br />                   Bronx, New York 10461,1 Department Microbiology Immunology, University of <br />                                   North Carolina, Chapel Hill, North Carolina 27599-72902 <br />                                            Received 22 June 2001/Accepted 18 September 2001 <br />  <br />              The proper extracytoplasmic localization proteins important aspect mycobacterial physiology and <br />           pathogenesis Mycobacterium tuberculosis. The protein export systems mycobacteria remained <br />           unexplored. The Sec-dependent protein export pathway characterized Escherichia coli is <br />           responsible transport cytoplasmic membrane proteins containing signal sequences their <br />           amino termini. SecA central component pathway, highly conserved bacteria. <br />           Here report unusual property mycobacterial protein export&#226;&#8364;&#8221;the presence homologues of <br />           SecA (SecA1 SecA2). Using allelic-exchange strategy Mycobacterium smegmatis, demonstrate that <br />           secA1 essential gene. In contrast, secA2 deleted example nonessential secA <br />           homologue. The essential nature secA1, consistent conserved Sec pathway, leads <span id='am-3' about='protege:TO' typeof='owl:Thing'>to</span> <br />           believe secA1 represents equivalent E. coli secA. The results phenotypic analysis &#226;&#338;&#172;secA2 <br />           mutant M. smegmatis presented indicate role SecA2 protein export. Based our <br />           study, appears SecA2 assist SecA1 export proteins Sec pathway. However, <br />           SecA2 functional equivalent SecA1. This finding, combination fact SecA2 highly <br />           conserved mycobacteria, suggests second role SecA2. The possibility exists role <br />           SecA2 export specific subset proteins. <br />  <br />  <br />    Mycobacterium tuberculosis causative agent tuber-               Escherichia coli (11, 15). The Sec-dependent protein export <br /> culosis represents severe health threat                 pathway responsible transport majority of <br /> world. Nearly million people die year tuber-                 extracytoplasmic proteins E. coli; consequently, export <br /> culosis (51). M. tuberculosis intracellular pathogen           represents essential function cell (16). The <br /> able survive grow macrophages. Secreted cell                   proteins translocated cytoplasmic membrane the <br /> surface-associated proteins ideally positioned act             Sec pathway initially synthesized precursor proteins con- <br /> host macrophage enable <span id='am-4' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-5' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-6' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-7' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-8' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-9' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-10' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-11' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-12' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-13' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-14' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-15' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-16' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-17' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-18' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-19' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-20' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-21' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-22' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-23' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-24' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-25' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-26' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-27' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-28' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-29' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-30' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-31' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-32' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-33' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-34' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-35' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-36' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-37' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-38' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-39' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-40' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-41' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-42' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-43' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-44' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-45' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-46' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-47' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-48' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-49' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-50' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-51' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-52' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-53' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-54' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-55' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-56' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-57' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-58' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-59' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-60' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-61' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-62' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-63' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-64' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-65' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-66' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-67' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-68' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-69' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-70' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-71' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-72' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-73' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-74' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-75' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-76' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-77' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-78' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-79' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-80' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-81' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span>survival</span> bacillus                  taining conserved amino-terminal signal sequences. These pre- <br /> normally hostile antimicrobial environment. In support               cursor proteins routed translocase, channel <br /> hypothesis, exported repetitive protein (Erp)                   translocating proteins travel mem- <br /> shown participate pathogenesis M. tuberculosis (2).             brane. During translocation, signal sequence cleaved off <br /> An erp mutant M. tuberculosis attenuated mice                   signal peptidase generate mature exported protein. <br /> exhibits significantly reduced ability grow macrophages.             The translocase core composed integral membrane pro- <br /> However, specific role Erp intracellular survival              teins SecY SecE peripheral membrane protein <br /> strategy M. tuberculosis unknown.                                      SecA. SecA plays central essential role export <br />    Protein export important aspect bacterial pathogen-              pathway. It ATPase provides energy protein <br /> esis. Research diverse bacterial pathogens demon-                     translocation. In addition, SecA binds nearly com- <br /> strated majority virulence factors extracytoplas- <br />                                                                              ponents Sec pathway, including precursor proteins, <br /> mic proteins (20, 29). Recently, importance protein <br />                                                                              chaperones SecB, acidic phospholipids mem- <br /> export pathways pathogenicity underscored the <br />                                                                              brane, integral membrane components translo- <br /> identification specialized secretion pathways (types I, II, III, <br />                                                                              case (15, 33, 43). Through cycles ATP binding hydro- <br /> and IV) numerous bacterial pathogens. There many <br />                                                                              lysis, SecA delivers bound precursor proteins translocase <br /> examples types export pathways encoded in <br />                                                                              undergoes cycles membrane insertion deinsertion <br /> pathogenicity islands required proper local- <br />                                                                              lead stepwise export protein (18, 42, 50). Addi- <br /> ization specific virulence factors (28). <br />    Surprisingly little known protein export path-               tional proteins, SecG complex SecD, SecF, and <br /> ways mycobacteria (4). In contrast, protein export path-              YajC, essential Sec-dependent export serve to <br /> ways bacteria better characterized, notably             increase overall efficiency process (14). The Sec- <br />                                                                              dependent export pathway highly conserved different <br />                                                                              bacteria (39, 43). <br />   * Corresponding author. Mailing address: Department Microbi-               In order begin understanding protein export pathways <br /> ology Immunology, University North Carolina, Chapel Hill, NC          operate mycobacteria, set identify char- <br /> 27599-7290. Phone: (919) 966-5051. Fax: (919) 962-8103. E-mail: <br />                                                                              acterize SecA. To surprise, homologues of <br /> braunste@med.unc.edu. <br />   &#226;&#8364;&#160; Present address: Aaron Diamond AIDS Research Center, New                 secA mycobacteria, referred secA1 secA2. This was <br /> York, NY 10016.                                                              unexpected finding, fully sequenced genome, <br />  <br />                                                                       6979 <br />  6980       BRAUNSTEIN ET AL.                                                                                                                                 J. BACTERIOL. <br />  <br />  <br />                                                                TABLE 1. Strains used study <br />         Strain                                                              Description                                                                 Reference <br />                                      &#226;&#171;&#186; <br /> E. coli K-12 DH5&#226;?&#163;                   F [&#226;?&#190;80d&#226;&#338;&#172;lacZM15] &#226;&#338;&#172;(lacZYA-argF) U169 deoR recA1 endA1 hsdR17 glnV44                                        Gibco Life Technologies <br />                                      thi-1 gyrA96 relA1 <br /> M. smegmatis <br />  mc2155                             ept-1                                                                                                      45 <br />  MB526                              mc2155::pMB128, secA1 single-crossover strain                                                              This   work <br />  MB573                              mc2155::pMB160, secA2 single-crossover strain                                                              This   work <br />  mc22522                            mc2155, &#226;&#338;&#172;secA2                                                                                             This   work <br />  mc22718                            mc2155, attB::fbpB-Tn552 &#226;&#172;&#732;phoA&#226;&#172;&#732; oriE1 kan                                                                  5 <br />  mc22724                            mc2155, attB::pepA-Tn552 &#226;&#172;&#732; phoAl oriE1 hyg                                                                 5 <br />  mc22725                            mc2155, attB::rv1566c-Tn552 &#226;&#172;&#732;phoA-oriE1-kan                                                                5 <br />  mc22757                            mc2155 (pMB174), expresses &#226;&#172;&#732;phoA                                                                           5 <br />  MB509                              mc2155(pMV261)                                                                                             This   work <br />  MB624                              mc2155(pMB196), expresses pepA-&#226;&#172;&#732;phoA fusion                                                              This   work <br />  MB625                              mc22522(pMB196), expresses pepA-&#226;&#172;&#732;phoA fusion                                                             This   work <br />  MB634                              mc2155(pMB202), expresses rv1566c-&#226;&#172;&#732;phoA fusion                                                           This   work <br />  MB635                              mc22522(pMB202), expresses rv1566c-&#226;&#172;&#732;phoA fusion                                                          This   work <br />  MB648                              mc2155(pMB206), expresses fbpB-HA-&#226;&#172;&#732;phoA fusion                                                           This   work <br />  MB649                              mc22522(pMB206), expresses fbpB-HA-&#226;&#172;&#732;phoA fusion                                                          This   work <br />  <br />  <br />  <br /> among available time, revealed existence                             sequence), N &#226;&#171;&#189; GAT C, X &#226;&#171;&#189; G A, Y &#226;&#171;&#189; C T. These primers <br /> of secA genes single organism. In report,                                employed touchdown PCR (40) amplify 250-bp products M. <br />                                                                                           bovis BCG M. smegmatis genomic DNAs. <br /> present genetic analysis Mycobacterium smegmatis, fast-                               To clone full-<span id='am-2' about='xsp:length' typeof='owl:Thing'>length</span> genes, resulting PCR products used probes <br /> growing nonpathogenic mycobacteria, demonstrates                                 screen appropriate libraries. The M. bovis BCG product used screen <br /> that mycobacterial secA genes redundant                               pKSII&#226;&#171;&#185;::M. bovis BCG library (7). One hybridizing clone, pH.3, subjected <br /> that function cell.                                                           exonuclease digestion generate collection nested deletions, using the <br />   Very recently, Staphylococcus aureus Streptococcus                              Erase-A-<span id='am-316' about='xsp:base' typeof='owl:Thing'>Base</span> (Promega), sequenced. The M. smegmatis <br />                                                                                           product used screen pYUB282::mc2155 cosmid library (kind gift V. <br /> pneumoniae genomes provided additional examples                                    Balasubramanian). A 4.0-kb fragment hybridizing clone, pYUB495, <br /> bacteria possess secA genes (24, 27, 48). The presence                           subcloned pKSII&#226;&#171;&#185; generate pYUB536. Nested deletions of <br /> of SecA proteins represent new type specialized                              pYUB536 constructed Erase-A-Base sequenced. <br /> protein export pathway.                                                                      Cloning sequencing secA2 homologues. Full-length M. tuberculosis <br />                                                                                           secA2 amplified PCR M. tuberculosis H37Rv genomic DNA using <br />                                                                                           primers SecAS14 (5&#226;&#172;&#732; TCGGATC-CAGCGGAACACCCCGGGCAGACT) and <br />                         MATERIALS AND METHODS                                             SecAS15 (5&#226;&#172;&#732;GCGGATCCAGTGCACGGTTGTCCACGAATTGC). The PCR <br />                                                                                           product cloned pCR2.1 vector using TA cloning kit (Invitro- <br />    Bacterial strains culture methods. All strains used work are <br /> described Table 1. E. coli strain DH5&#226;?&#163; used DNA cloning procedures.            gen) generate pMB144. This PCR product used probe pYUB412::M. <br /> For culturing E. coli, Luria-Bertani (LB) broth agar (Difco) used.              smegmatis genomic cosmid library (kind gift F. C. Bange) low-stringency <br /> When required, following antibiotics used E. coli strains: kanamycin         conditions. A 6.0-kb fragment hybridizing clone, p6-1, subcloned <br /> at 40 &#226;?&#174;g/ml carbenicillin 50 &#226;?&#174;g/ml. A variety media employed           pSKII&#226;&#171;&#185; generate pMB148. Nested deletions pMB148 con- <br /> the growth M. smegmatis included following: Middlebrook 7H9             structed Erase-A-Base used generate sequence of <br /> broth (Difco) 0.2% glycerol, 0.5% dextrose, 0.1% Tween 80;              secA2 (M. smegmatis). <br /> medium supplemented 0.2% glycerol, 1&#226;&#171;&#187; ADS (0.5% bovine serum albu-                      To clone M. smegmatis secA2 control constitutive hsp60 <br /> min, fraction V [Boehringer Mannheim], 0.2% dextrose, 0.85% NaCl), 0.1%               promoter, gene amplified PCR M. smegmatis mc2155 genomic <br /> Tween 80; LB broth 0.1% Tween 80; 7H10 agar 0.2% dextrose                   DNA. The primers used SmSecA2-14 (5&#226;&#172;&#732;ACGGATCCA-GTGGCGAAT <br /> 0.05% Tween 80; LB agar 0.2% dextrose; Mueller-Hinton agar (Difco).              GAGTCCTGGCGAAC) SmSecA2-13 (5&#226;&#172;&#732;CGTCTTCCATGCCTAGAAC- <br /> When appropriate, sucrose incorporated 7H10 plates concentration            CTAATG). The resulting PCR product cloned pCR2.1 create pMB207. <br /> of 4.5%. When required, following antibiotics used grow M. smeg-              The BamHI fragment containing secA2 subcloned pMV261 to <br /> matis strains: kanamycin 20 &#226;?&#174;g/ml hygromycin B 50 &#226;?&#174;g/ml. When                   create pMB208. <br /> appropriate, cultures grown overnight Middlebrook 7H9 0.2%                      Construction &#226;&#338;&#172;secA suicide plasmids. (i) &#226;&#338;&#172;secA1 suicide plasmid pMB128. <br /> glycerol, 1&#226;&#171;&#187; ADS, 0.1% Tween 80, 100 &#226;?&#174;M sodium azide.                                 An approximately 12.0-kb SacI fragment pYUB495 containing secA1 M. <br />    DNA methodologies. Standard molecular biology techniques cloning              smegmatis identified Southern analysis. Restriction digest analysis showed <br /> employed previously described (41). Restriction enzymes Vent polymer-              secA1 located near middle fragment. This fragment cloned <br /> ase obtained New England Biolabs, Inc. The Expand high fidelity PCR             pKSII&#226;&#171;&#185; generate plasmid pMB117. pMB117 linearized BglII (a <br /> system obtained Boehringer Mannheim. Dimethyl sulfoxide (1.0                  unique site near middle secA1) subjected exonuclease III <br /> 10.0%) included selected PCRs. DNA sequencing performed                     digestion directions outward BglII-cut site using Erase- <br /> conventional automated methods. Automated sequencing employed                     A-Base self-ligated. One resulting deletion constructs, <br /> Applied Biosystems Big Dye Terminator Cycle Sequencing kit (Perkin-Elmer)                 pMB120, analyzed PCR sequencing analysis shown contain a <br /> and Applied Biosystems 377 sequencer. Sequence assembly carried                1,278-bp in-frame deletion secA1 gene. The large SacI fragment of <br /> with MacVector Assembly Align software (Genetics Computer Group).                 pMB120 subcloned pSKII&#226;&#171;&#185; create pMB125. The in-frame unmarked <br />    Plasmid construction. The plasmids used study described Table 2.        secA1 deletion fragment cloned counterselectable suicide vector <br />    Cloning sequencing secA1 homologues. Degenerate oligonucleotide                 pYUB657 (37). This achieved cutting pMB125 SpeI ScaI and <br /> primers designed regions identity secA genes E. coli, Bacillus       cloning resulting secA1 fragment SpeI- ScaI-cut pYUB657. The final <br /> subtilis, Pavlova lutherii. The 5&#226;&#172;&#732; primer (5&#226;&#172;&#732;-ACN-GGN-GAX-GGN-AAX-A                   vector produced pMB128. <br /> CN-YT-3&#226;&#172;&#732;) designed peptide TGEGKT (amino acids 104 109                    (ii) &#226;&#338;&#172;secA2 suicide plasmid pMB160. A 3.2-kb BamHI-ScaI fragment of <br /> the E. coli sequence), 3&#226;&#172;&#732; primer (5&#226;&#172;&#732;-AXX-TAX-TCX-AAN-CC-3&#226;&#172;&#732;)                   pMB148 cloned BamHI-EcoRV-cut pKSII&#226;&#171;&#185; generate pMB155. <br /> designed peptide GFDYL (amino acids 183 187 E. coli                    pMB155 used template inverse PCR using primers SmSecA2-5 <br />  VOL. 183, 2001                                                                   SecA1 AND SecA2 IN MYCOBACTERIUM SMEGMATIS                                 6981 <br />  <br />  <br />                                                           TABLE 2. Plasmids used study <br />    Plasmid                <span id='am-82' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-83' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-84' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-85' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-86' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-87' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-88' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-89' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-90' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-91' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-92' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-93' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-94' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-95' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-96' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-97' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-98' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-99' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-100' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-101' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-102' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-103' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-104' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-105' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-106' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-107' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-108' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-109' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-110' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-111' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-112' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-113' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-114' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-115' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-116' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-117' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-118' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-119' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-120' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-121' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-122' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-123' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-124' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-125' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-126' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-127' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-128' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-129' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-130' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-131' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-132' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-133' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-134' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-135' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-136' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-137' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-138' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-139' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-140' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-141' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-142' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-143' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-144' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-145' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-146' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-147' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-148' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-149' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-150' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-151' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-152' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-153' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-154' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-155' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-156' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-157' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-158' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-159' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span>Phenotype</span>/genotype                                       Description                                                Source <br />  <br /> pKSII&#226;&#171;&#185;           bla ColE1                                                                                                      Stratagene <br /> pSKII&#226;&#171;&#185;           bla ColE1                                                                                                      Stratagene <br /> pCR2.1           bla aph ColE1                                                                                                  Invitrogen <br /> pMV261.kan       aph Phsp60 oriM ColE1                   Multicopy mycobacterial shuttle plasmid                                46 <br /> pMV306.kan       aph int attP ColE1                      Single-copy mycobacterial shuttle plasmid                              46 <br /> pMV361.hyg       hyg Phsp60 int attP ColE1               Single-copy mycobacterial shuttle plasmid                              46 <br /> pYUB412          bla hyg int attP ColE1 cos&#226;?&#173;             Single-copy mycobacterial shuttle plasmid                              S. Bardarov W. R. Jacobs <br /> pYUB282          bla ColE1 cos&#226;?&#173;                                                                                                 V. Balasubramanian and <br />                                                                                                                                    W. R. Jacobs <br /> pMD31            aph oriM ColE1                          Multicopy mycobacterial shuttle plasmid                                12 <br /> pMB198           hyg int attP ColE1                      Phsp60 deleted pMV361.hyg                                         This work <br /> pYUB657          bla hyg Phsp60-sacB ColE1               Counterselectable suicide vector mycobacteria                      37 <br /> pYUB495          bla secA1 (M. smegmatis) ColE1          pYUB282 containing M. smegmatis secA1                                  This work <br />                    cos&#226;?&#173;y <br /> pYUB536          bla secA1 (M. smegmatis) ColE1          4.0-kb BamHI fragment containing M. smegmatis secA1               This work <br />                                                            pYUB495 cloned pKSII&#226;&#171;&#185; <br /> pYUB544          aph Phsp60-secA1 oriM ColE1             4.0-kb BamHI fragment pYUB536 cloned pMV261;                 This work <br />                                                            M. smegmatis secA1 control hsp60 promoter <br /> pH.3             bla secA1 (M. bovis BCG) ColE1          pKSII&#226;&#171;&#185; containing 8.0-kb fragment M. bovis BCG secA1           This work <br /> pYUB499          aph secA1 (M. bovis BCG) oriM           3.0-kb HindIII-KpnI fragment M. bovis BCG secA1 cloned              This work <br />                    ColE1                                   pMD31 <br /> pYUB538          aph secA1AzR (M. bovis BCG)             pYUB499 secA1 azide resistance allele                             This work <br />                    oriM ColE1 <br /> pMB117           bla secA1 (M. smegmatis) ColE1          12.0-kb SacI fragment containing M. smegmatis secA1               This work <br />                                                            pYUB495 cloned pKSII&#226;&#171;&#185; <br /> pMB120           bla &#226;&#338;&#172;secA1 (M. smegmatis) ColE1         In-frame deletion mutant form secA1 derived pMB117             This work <br /> pMB125           bla &#226;&#338;&#172;secA1 (M. smegmatis) ColE1         SacI fragment pMB120 subcloned pSKII&#226;&#171;&#185;                          This work <br /> pMB128           bla hyg Phsp60-sacB &#226;&#338;&#172;secA1              &#226;&#338;&#172;secA1 pMB125 cloned pYUB657                                 This work <br />                    (M. smegmatis) ColE1 <br /> p6-1             bla secA2 (M. smegmatis) ColE1          pYUB412 cosmid containing M. smegmatis secA2                           This work <br /> pMB144           bla aph secA2 (M. tuberculosis)         M. tuberculosis secA2 cloned pCR2.1                               This work <br />                    ColE1 <br /> pMB147           aph Phsp60-secA2 (M. tuberculosis)      M. tuberculosis secA2 pMB144 cloned pMV261             This work <br />                    oriM ColE1                              control Phsp60 <br /> pMB148           bla secA2 (M. smegmatis) ColE1          6.0-kb BamHI fragment p6-1 containing M. smegmatis                This work <br />                                                            secA2 cloned pSKII&#226;&#171;&#185; <br /> pMB155           bla secA2 (M. smegmatis) ColE1          3.2-kb BamHI-ScaI fragment pMB148 carrying M. smegmatis             This work <br />                                                            secA2 cloned pKSII&#226;&#171;&#185; <br /> pMB156           bla &#226;&#338;&#172;secA2 (M. smegmatis) ColE1         In-frame deletion secA2 derived pMB155                         This work <br /> pMB160           bla hyg Phsp60-sacB &#226;&#338;&#172;secA2              &#226;&#338;&#172;secA2 cloned pMB156 pYUB657                                 This work <br />                    (M. smegmatis) ColE1 <br /> pMB162           aph Phsp60-secA2 (M. tuberculosis)      HindIII-XbaI fragment carrying Phsp60-secA2 pMB147 sub-           This work <br />                    int attP ColE1                          cloned pMV306.kan <br /> pMB110           aph Phsp60-&#226;&#172;&#732;phoA oriM ColE1             &#226;&#172;&#732;phoA PCR product cloned control Phsp60                       This work <br /> pMB174           aph Phsp60-&#226;&#172;&#732;phoA int attP ColE1         NotI fragment pMB110 carrying Phsp60-&#226;&#172;&#732;phoA cloned              This work <br />                                                            pMV306.kan <br /> pMB175           aph secA1 (M. bovis BCG) int            HindIII/Asp718 M. bovis BCG secA1 containing fragment cloned           This work <br />                    attP ColE1                              pMV306.kan <br /> pMB196           aph pepA::Tn552 &#226;&#172;&#732;phoA oriE1 hyg         pepA::&#226;&#172;&#732;phoA fusion recovered NheI fragment mc22724             This work <br />                    int attP ColE1                          cloned pMB198 <br /> pMB202           hyg rv1566c::Tn552 &#226;&#172;&#732;phoA oriE1          rv1566c::&#226;&#172;&#732;phoA fusion recovered NarI fragment mc22725          This work <br />                    kan int attP ColE1                      cloned pMB198 <br /> pMB203           hyg fbpB::Tn552 &#226;&#172;&#732;phoA oriE kan          fbpB::&#226;&#172;&#732;phoA fusion recovered NarI fragment mc22718             This work <br />                    int attP ColE1                          cloned pMB198 <br /> pMB206           hyg fbpB-HA::Tn552 &#226;&#172;&#732;phoA oriE1          HA tag cloned frame fbpB::&#226;&#172;&#732;phoA fusion pMB203               This work <br />                    kan int attP ColE1 <br /> pMB207           bla aph secA2 (M. smegmatis)            M. smegmatis secA2 cloned pCR2.1                                  This work <br />                    ColE1 <br /> pMB208           aph Phsp60-secA2 (M. smegmatis)         M. smegmatis secA2 pMB207 cloned pMV261                This work <br />                    oriM ColE1                             control Phsp60 <br />  <br />  <br />  <br /> (5&#226;&#172;&#732;-ACTGATATCGCGCAGCACGTCGAAGCCGAT-3&#226;&#172;&#732;) SmSecA2-6 (5&#226;&#172;&#732;-                           Two step allelic exchange create &#226;&#338;&#172;secA mutants M. smegmatis. To con- <br /> TGAGATATCGTCGAGCGCCGCGAGACCCT-3&#226;&#172;&#732;) (underlined residues de-                       struct &#226;&#338;&#172;secA1 mutants, pMB128 electroporated mc2155 hygromy- <br /> note EcoRV site). The resulting PCR product gel purified, cut       cin-resistant transformants selected. Individual transformants sub- <br /> EcoRV, self-ligated. The resulting vector pMB156. Due primer          jected Southern analysis, 14 16 strains evaluated contained the <br /> locations secA2, pMB156 contains in-frame deletion 1,296 bp secA2.   suicide vector integrated secA1 region chromosome means of <br /> A SpeI-ScaI secA2-containing fragment pMB156 subcloned SpeI-         single-crossover event. One strains, MB526, employed the <br /> ScaI-cut pYUB657. The final vector pMB160.                                   subsequent steps. For second homologous recombination event, MB526 was <br />    Electroporation M. smegmatis. M. smegmatis strain mc2155 (45) elec-    grown saturation 7H9 medium hygromycin 50 &#226;?&#174;g/ml. For <br /> troporated previously described (36).                                         MB526 strains additionally carry kanamycin resistance element-containing <br />  6982       BRAUNSTEIN ET AL.                                                                                                                     J. BACTERIOL. <br />  <br />  <br /> plasmids, kanamycin included medium. This culture subcultured               TABLE 3. Amino acid similarities M. tuberculosis <br /> at 1:100 dilution medium lacking hygromycin incubated                       SecA proteins selected SecA homologues <br /> overnight. Dilutions plated 7H10 plates containing 4.5% sucrose to <br /> select sucrose-resistant colonies. To obtain &#226;&#338;&#172;secA2 mutants, pMB160                                                                        % Similarity <br /> electroporated mc2155 hygromycin-resistant transformants ob-                                                           Accession     M. tuber- <br />                                                                                                          Protein                                     culosisa <br /> tained. Three transformants confirmed result                                                           no. <br /> single-crossover event Southern analysis. One clones, MB573,                                                                     SecA1     SecA2 <br /> used subsequent work presented report. The second homologous <br /> recombination event selected using protocol described           Mycobacterium tuberculosis SecA1               Z95121        100.0     50.2 <br /> MB526.                                                                               Mycobacterium bovis BCG SecA1                  U66080         98.6     50.8 <br />    Southern analysis. To analyze secA1 recombinants, genomic DNA iso-            Mycobacterium smegmatis SecA1                  U66081         88.0     50.2 <br /> lated M. smegmatis clones previously described (10) digested        Mycobacterium leprae SecA1                     P57996         90.4     48.5 <br /> BglII. The probe used 669-bp MscI fragment pYUB536 containing secA1         Mycobacterium tuberculosis SecA2               Z78020         50.2    100.0 <br /> of M. smegmatis. To analyze secA2 recombinants, genomic DNA isolated and <br />                                                                                      Mycobacterium smegmatis SecA2                  AF287049       50.2     86.4 <br />                                                                                      Mycobacterium leprae SecA2                     O32922         50.1     90.8 <br /> digested BamHI. The probe used 1.9-kb BamHI-HindIII fragment <br />                                                                                      Corynebacterium glutamicum SecA                D17428         74.9     46.1 <br /> obtained pMB156 contains secA2 M. smegmatis. Southern analysis <br />                                                                                      Streptomyces coelicolor SecA                   U21192         73.4     49.1 <br /> was performed previously described (41), probes labeled with <br />                                                                                      Synechococcus sp. strain PCC7942 SecA          Q55357         63.6     44.8 <br /> [32P]dCTP using Ready-to-Go Labeling kit (Pharmacia). <br />                                                                                      Staphylococcus aureus SecA-1                   BAB41941       63.3     51.2 <br />    Construction &#226;&#180;&#8226;phoA fusion vectors. fbpB-&#226;&#172;&#732;phoA, pepA-&#226;&#172;&#732;phoA, rv1566c-        Staphylococcus aureus SecA-2                   BAB43747       49.9     46.5 <br /> &#226;&#172;&#732;phoA fusions rescued strains mc22718, mc22724, mc22725, re-           Listeria monocytogenes SecA                    P47847         61.1     46.5 <br /> spectively. These strains contain M. tuberculosis DNA-&#226;&#172;&#732;phoA fusions encode      Streptococcus pneumoniae SecA-1                AE005672       60.2     50.8 <br /> active PhoA fusion proteins M. smegmatis recently obtained          Streptococcus pneumoniae SecA-2                AE005672       52.3     47.8 <br /> transposon libraries (5). These phoA fusions present cosmids integrated       Escherichia coli SecA                          P10408         60.9     49.2 <br /> into chromosome. To rescue fusions, genomic DNA isolated            Bacillus subtilis SecA                         P28366         60.6     49.0 <br /> each strain digested NarI (for mc22718 mc22725) NheI (for            Pisum sativum SecA                             Q41062         59.5     46.8 <br /> mc22724), self-ligated, transformed DH5&#226;?&#163;. Due presence E.      Caulobacter crescentus SecA                    P38380         59.4     50.1 <br /> coli origin replication transposon, self-ligated molecules       Actinobacillus actinomycetemcomitans SecA      AF116183       59.2     48.4 <br /> replicative plasmids E. coli. These restriction enzymes cut outside phoA   Haemophilus influenzae SecA                    U32772         58.9     48.4 <br /> fusion promoter element. Once isolated, fusions subcloned        Neisseria meningitidis MC58 SecA               AAF41891       58.2     47.7 <br /> mycobacterial shuttle vector, pMB198, contains attP/int L5 mycobacte-       Helicobacter pylori SecA                       O25475         57.3     47.7 <br /> riophage integration attachment system, promotes integration          Pavlova lutherii SecA                          QO1570         55.5     46.1 <br /> chromosome single copy (25).                                                      Borrelia burgdorferi SecA                      OO7497         54.6     47.6 <br />    Isolation azide-resistant alleles M. bovis BCG SecA. Plasmid pYUB499        Mycoplasma genitalium SecA                     P47318         50.9     44.1 <br /> was subjected N-methyl-N&#226;&#172;&#732;-nitro-N-nitrosoguanidine (Sigma) mutagenesis              a <br />                                                                                            Percent similarity determined Gap Comparison GCG. <br /> (30). Mutagenized DNA electroporated mc2155, azide-resistant <br /> cells selected 7H10 agar plates containing sodium azide 25 &#226;?&#174;g/ml. This <br /> concentration sodium azide chosen basis azide sensitivity disk <br /> assays determined minimum sodium azide capable inhib-          lutherii, degenerate oligonucleotide primers designed a <br /> iting growth M. smegmatis 7H10 agar plates.                                    region identity amino termini proteins (see <br />    Azide sensitivity disk assays. Cells grown LB saturation. A 0.1-ml     Materials Methods). These primers used amplify <br /> volume saturated culture mixed molten LB agar plated an <br />                                                                                      approximately 250-bp PCR product M. bovis <br /> LB plate. Seven-millimeter-diameter paper filter disks (Schleicher &amp; Schuell) <br /> were placed hardened agar. To disk, 10 &#226;?&#174;l 0.3 M sodium       BCG M. smegmatis genomic DNAs. The resulting PCR <br /> azide added. The zone sensitivity calculated diameter           products used probe appropriate M. bovis BCG M. <br /> clearing disk minus diameter disk.                             smegmatis genomic libraries led cloning se- <br />    Preparation whole-cell protein extracts M. smegmatis Western          quencing secA homologues mycobacterial <br /> analysis. Strains grown A600 0.5 1.0 7H9&#226;&#8364;&#8220;1&#226;&#171;&#187; ADS&#226;&#8364;&#8220;glycerol&#226;&#8364;&#8220; <br /> 0.1% Tween 80&#226;&#8364;&#8220;kanamycin 20 &#226;?&#174;g/ml. The number cells, 1.2 &#226;&#171;&#187; 109, was <br />                                                                                      species. These secA genes identified named secA1. <br /> harvested centrifugation. The cells washed twice phosphate-buffered       M. bovis BCG secA1 M. smegmatis secA1 encode proteins <br /> saline&#226;&#8364;&#8220;0.02% Tween 80, pelleted centrifugation, quick-frozen dry-ice         predicted sizes 106 107 kDa, respectively. Subse- <br /> &#226;&#8364;&#8220;ethanol bath, stored &#226;&#171;&#186;20&#194;&#176;C. At later time, pellets resuspended    quently, secA1 homologue (rv3240c) identified the <br /> in 200 &#226;?&#174;l extraction buffer (50 mM Tris-HCl [pH 7.5], 5 mM EDTA, 0.6% <br />                                                                                      complete genome sequence M. tuberculosis (8). Each of <br /> sodium dodecyl sulfate [SDS], 10 &#226;?&#174;g/ml aprotinin, E-64 10 &#226;?&#174;g/ml, leupeptin at <br /> 10 &#226;?&#174;g/ml, Pefabloc SC 500 &#226;?&#174;g/ml, pepstatin A 10 &#226;?&#174;g/ml) 200 &#226;?&#174;l      secA1 genes encodes protein significant sequence <br /> 106-&#226;?&#174;m glass beads added. The cells beads vortexed twice       similarity SecA1 mycobacterial species the <br /> 5 min 4&#194;&#176;C 5-min rest ice. A 200-&#226;?&#174;l volume 2&#226;&#171;&#187; SDS-polyacryl-          SecA proteins bacteria (Table 3). Southern analysis <br /> amide gel electrophoresis (PAGE) sample buffer added sample. The         secA1 probes revealed hybridizing restriction fragments in <br /> samples denatured boiling, 20 &#226;?&#174;l loaded SDS-PAGE <br /> gel. The proteins transferred nitrocellulose, Western analysis to <br />                                                                                      genomic DNAs of mycobacterial species tested. <br /> detect PhoA hemagglutinin (HA) carried anti-PhoA polyclonal          This Southern analysis consistent secA1 sin- <br /> antibodies obtained 5 Prime-3 Prime, Inc., anti-HA monoclonal anti-          gle-copy gene; indication secA gene <br /> bodies obtained Covance, Inc.                                                   existed mycobacterial genome (data shown). <br />    Nucleotide sequence accession numbers. The nucleotide sequences reported <br />                                                                                         Based above-described work fact the <br /> here submitted GenBank database assigned following <br /> accession numbers: M. smegmatis secA1, U66081; M. smegmatis secA2, AF287049;         fully sequenced bacterial genomes available at <br /> M. bovis BCG, U66080.                                                                time contained secA homologue, sur- <br />                                                                                      prised second distinct secA homologue present in <br />                                   RESULTS                                            genome sequence M. tuberculosis (8). This second secA <br />                                                                                      homologue named secA2. M. tuberculosis secA2 cor- <br />   Cloning mycobacterial secA homologues. At start                          responds open reading frame rv1821 encodes protein <br /> this study, minimal number secA genes cloned.                          predicted size 89 kDa. By Southern analysis, we <br /> By using available sequences E. coli, B. subtilis, P.                       identified secA2-hybridizing fragments H37Rv Erd- <br />  VOL. 183, 2001                                                             SecA1 AND SecA2 IN MYCOBACTERIUM SMEGMATIS                         6983 <br />  <br />  <br />  <br />  <br />    FIG. 1. Alignment SecA proteins B. subtilis, E. coli, M. smegmatis (SecA1 SecA2). ABCI (high-affinity ATP-binding site) and <br /> ABCII (low-affinity ATP-binding site) indicated. The lightning bolt indicates threonine residue mutated produce azide <br /> resistance. Triangles represent deletion junction points &#226;&#338;&#172;secA1 <span id='am-160' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-161' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-162' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-163' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-164' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-165' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-166' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-167' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-168' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-169' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-170' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-171' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-172' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-173' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-174' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-175' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span><span id='am-176' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-177' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-178' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-179' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-180' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-181' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-182' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-183' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-184' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-185' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-186' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-187' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-188' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-189' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-190' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-191' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-192' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-193' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-194' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-195' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-196' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-197' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-198' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-199' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-200' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-201' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-202' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-203' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-204' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-205' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-206' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-207' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-208' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-209' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-210' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-211' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-212' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-213' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-214' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-215' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-216' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-217' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-218' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-219' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-220' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-221' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-222' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-223' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-224' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-225' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-226' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-227' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-228' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-229' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-230' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-231' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-232' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-233' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-234' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-235' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-236' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-237' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span>mutation</span>. The arrows pointing identify deletion junction points <br /> in &#226;&#338;&#172;secA2 mutation. <br />  <br />  <br /> man strains M. tuberculosis, M. bovis BCG, M. smegmatis              other. To determine essential nature SecA <br /> (data shown). We used M. tuberculosis secA2 gene               protein mycobacteria, tested possible to <br /> probe screen M. smegmatis genomic library               delete secA gene. <br /> clone sequence M. smegmatis secA2. The predicted size                   We used two-step allelic-exchange strategy determine <br /> M. smegmatis SecA2 87 kDa.                                               essentiality secA gene. This approach previ- <br />    Reproducibly, secA1 secA2 probe used                   ously used M. smegmatis (22, 36). In step, a <br /> Southern analyses (high low stringency nature)                suicide vector containing in-frame deletion given secA <br /> single secA gene, corresponding probe used, iden-                gene, selectable hyg (encoding hygromycin resistance) <br /> tified (data shown). This consistent difference          marker, counterselectable sacB marker integrated into <br /> nucleotide sequence (50%) secA genes. At                chromosome means single homologous recombi- <br /> amino acid level, SecA1 SecA2 50% similar. An                  nation event secA allele vector the <br /> alignment predicted proteins reveals SecA2             wild-type secA allele chromosome. The resulting recom- <br /> smaller protein shorter carboxy terminus (Fig. 1). E. coli           binant strain referred single-crossover strain, it <br /> SecA contains ATP-binding sites (ABCI ABCII)                   contains tandem duplication secA region (both mu- <br /> are important SecA function (31). Both SecA1 SecA2                   tated wild-type allele present chromosome) <br /> exhibit strong homology proposed Walker A B                      separated vector sequences including sacB hyg (dia- <br /> motifs present sites (Fig. 1), suggesting          grams strains shown Fig. 2B 3B). The second <br /> have physiological functions similar E. coli SecA.              step select recombinants undergone second <br />    Determination essentiality secA1 secA2. In                 homologous recombination event secA alleles. <br /> other bacteria SecA studied, SecA essen-               Depending site second recombination event, it <br /> tial encoded single-copy gene (17, 34, 43). One ex-                leaves mutant wild-type allele chromosome. <br /> planation presence secA genes mycobacteria                This second recombination event eliminates interven- <br /> is encode redundant functions. In scenario, de-              ing vector sequences, including sacB hyg. Expression of <br /> letion secA gene lethal                sacB results sensitivity sucrose; therefore, secondary <br /> the remaining homologue compensate absence                    recombinants selected sucrose-resistant colonies (38). <br />  6984     BRAUNSTEIN ET AL.                                                                                                        J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 2. Southern analysis secA1 recombinants M. smegmatis. (A D) The relevant BglII BamHI restriction endonuclease sites are <br /> denoted. The MscI probe used Southern analysis represented vertical lines diagram. Panels: A, diagram chromosomal <br /> wild-type secA1 allele; B, diagram secA1 single-crossover strain MB526; C, diagram chromosomal &#226;&#338;&#172;secA1 allele; D, diagram pYUB544, <br /> a multicopy plasmid present merodiploid strains; E, Southern blot BglII-digested genomic DNAs probed secA1 probe. Sizes of <br /> fragments indicated left. Lanes: 1, mc2155; 2, MB526; 3, &#226;&#338;&#172;secA1 recombinant pYUB544; 4, wild-type recombinant pYUB544. <br />  <br />  <br /> The failure obtain recombinants secA gene              allele remaining resulting recombinants assayed by <br /> deleted strongly suggests essential gene. If            PCR Southern analysis. All recombinants assayed <br /> inability delete chromosomal secA gene overcome              contained wild-type allele (Table 4). This failure obtain <br /> merodiploid strains contain extra copy secA, dem-           secA1 deletion mutants strongly suggests deletion secA1 <br /> onstrates absolute requirement secA gene via-             M. smegmatis lethal event. <br /> bility.                                                                      To confirm interpretation, created secA1 merodip- <br />    secA1 essential M. smegmatis. We applied              loid strains transformation MB526 plasmids express- <br /> approach secA1 gene M. smegmatis. Suicide plasmid               ing secA1 M. smegmatis M. bovis BCG (pYUB544 and <br /> pMB128 contains 1,278-bp in-frame deletion M. smeg-              pYUB499, respectively). <span id='am-1' about='protege:FROM' typeof='owl:Thing'>From</span> merodiploid single-cross- <br /> matis secA1. This deletion generate mutated protein                strains, sucrose-resistant hygromycin-sensitive recom- <br /> comprising 531 amino acids (in comparison wild-type                binants obtained. In contrast results obtained with <br /> protein 957 amino acids). Importantly, deletion involves           starting strain MB526 MB526 transformed control vec- <br /> both ATP-binding sites (Fig. 1). This plasmid electropo-              tor pMV261, merodiploid strains produced classes of <br /> rated M. smegmatis, hygromycin-resistant transfor-               recombinants: &#226;&#338;&#172;secA1 recombinants wild-type secA1 re- <br /> mants obtained. One transformants, MB526,               combinants (Table 4; Fig. 2). From results, concluded <br /> used subsequent experiments. MB526 product               secA1 essential gene M. smegmatis. <br /> a single-crossover event secA1 deletion allele                secA2 functionally replace secA1. The essentiality of <br /> pMB128 wild-type secA1 allele chromosome,               secA1 M. smegmatis suggested secA1 secA2 not <br /> demonstrated Southern analysis (Fig. 2B E).                        redundant genes. However, secA2 expressed under <br />    Following growth MB526 absence hygromy-               experimental conditions, secA1 secA2 still <br /> cin, sucrose-resistant colonies selected screened           function. To address possibility, tested <br /> for hygromycin sensitivity. Sucrose-resistant hygromycin-             &#226;&#338;&#172;secA1 strain rescued secA2 expression. <br /> sensitive colonies undergone second recombination                  Single-crossover strain MB526 independently electropo- <br /> event secA1 alleles. The individual secA1                 rated plasmids pMB208, pMB147, pMB162, <br />  VOL. 183, 2001                                                        SecA1 AND SecA2 IN MYCOBACTERIUM SMEGMATIS                           6985 <br />  <br />  <br />                                                                             TABLE 4. Secondary recombination products produced <br />                                                                                        single-crossover strains <br />                                                                                                         % secA allele present among <br />                                                                           Strain (plasmid)               secondary recombinants               na <br />                                                                                                         Deletion          Wild type <br />  <br />                                                                       secA1 single crossover <br />                                                                         MB526                               0                100              13 <br />                                                                         MB526(pMV261)                       0                100               9 <br />                                                                         MB526(pYUB544)                     62                 38              16 <br />                                                                         MB526(pYUB499)                     59                 41              39 <br />                                                                         MB526(pMB208)                       0                100              20 <br />                                                                         MB526(pMB147)                       0                100              24 <br />                                                                         MB526(pMB162)                       0                100              19 <br />  <br />                                                                       secA2 single crossover <br />                                                                         MB573                              31                 69              13 <br />                                                                         MB575                              50                 50              18 <br />                                                                          a <br />                                                                            Number sucrose-resistant hygromycin-sensitive recombinants ana- <br />                                                                       lyzed. <br />  <br />  <br />  <br />                                                                       strain MB573 constructed (Fig. 3). From MB573, sucrose- <br />                                                                       resistant hygromycin-sensitive secondary recombinants <br />                                                                       identified. Some recombinants contained the <br />                                                                       &#226;&#338;&#172;secA2 deletion. One &#226;&#338;&#172;secA2 mutants, mc22522, was <br />                                                                       used subsequent analyses presented report. <br />                                                                       A separate secA2 single-crossover strain, MB575, similarly led <br />                                                                       production &#226;&#338;&#172;secA2 mutants. The ability delete <br />                                                                       secA2 demonstrates essential gene. To our <br />                                                                       knowledge, secA shown be <br />                                                                       nonessential. <br />                                                                          The &#226;&#338;&#172;secA2 mutant exhibits growth defect rich agar <br />                                                                       plates. Shortly generating mc22522, observed that <br />                                                                       grown rich medium (LB Mueller-Hinton) agar <br />                                                                       plates, produces smaller single colonies wild- <br />                                                                       type M. smegmatis strain mc2155 (Fig. 4). This small-colony <br />                                                                       phenotype medium dependent; seen minimal <br />    FIG. 3. Southern analysis secA2 recombinants M. smegmatis.   agar plates (7H10). Furthermore, observable <br /> (A C) BamHI restriction endonuclease sites denoted. The <br /> BamHI-HindIII probe used Southern analysis indicated    growth defect associated mc22522 grown rich <br /> vertical bars diagram. Panels: A, diagram chromo-    liquid medium. <br /> somal wild-type secA2 allele; B, diagram secA2 single-crossover       We understand basis phenotype. However, <br /> strain MB573; C, diagram chromosomal &#226;&#338;&#172;secA2 allele; D, Southern    clear <span id='am-238' about='Thesaurus:TESSE01' typeof='owl:Thing'><span id='am-239' property="Thesaurus:Preferred_Name" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-240' property="rdfs:label" content="Missense mutation" datatype="rdf:PlainLiteral"></span><span id='am-241' property="Thesaurus:code" content="TEESS06" datatype="rdf:PlainLiteral"></span><span id='am-242' property="Thesaurus:code" content="TEESS14" datatype="rdf:PlainLiteral"></span><span id='am-243' property="Thesaurus:code" content="TEESS02" datatype="rdf:PlainLiteral"></span><span id='am-244' property="Thesaurus:Preferred_Name" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-245' property="rdfs:label" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-246' property="Thesaurus:Preferred_Name" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-247' property="Thesaurus:Preferred_Name" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-248' property="rdfs:label" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-249' property="Thesaurus:code" content="TEESS23" datatype="rdf:PlainLiteral"></span><span id='am-250' property="Thesaurus:Preferred_Name" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-251' property="Thesaurus:code" content="TEESS01" datatype="rdf:PlainLiteral"></span><span id='am-252' property="Thesaurus:Preferred_Name" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-253' property="Thesaurus:code" content="TEESS19" datatype="rdf:PlainLiteral"></span><span id='am-254' property="rdfs:label" content="Mutation analysis" datatype="rdf:PlainLiteral"></span><span id='am-255' property="Thesaurus:Preferred_Name" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-256' property="rdfs:label" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-257' property="Thesaurus:code" content="TEESS04" datatype="rdf:PlainLiteral"></span><span id='am-258' property="Thesaurus:Preferred_Name" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-259' property="rdfs:label" content="Deleterious mutation" datatype="rdf:PlainLiteral"></span><span id='am-260' property="rdfs:label" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-261' property="Thesaurus:Preferred_Name" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-262' property="Thesaurus:Preferred_Name" content="Gene knockout" datatype="rdf:PlainLiteral"></span><span id='am-263' property="Thesaurus:Preferred_Name" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-264' property="Thesaurus:Preferred_Name" content="null mutants" datatype="rdf:PlainLiteral"></span><span id='am-265' property="Thesaurus:code" content="TEESS21" datatype="rdf:PlainLiteral"></span><span id='am-266' property="Thesaurus:Preferred_Name" content="phenotype" datatype="rdf:PlainLiteral"></span><span id='am-267' property="Thesaurus:code" content="TEESS10" datatype="rdf:PlainLiteral"></span><span id='am-268' property="rdfs:label" content="Deletion mutation" datatype="rdf:PlainLiteral"></span><span id='am-269' property="Thesaurus:Preferred_Name" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-270' property="Thesaurus:code" content="TEESS17" datatype="rdf:PlainLiteral"></span><span id='am-271' property="Thesaurus:Preferred_Name" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-272' property="Thesaurus:code" content="TEESS18" datatype="rdf:PlainLiteral"></span><span id='am-273' property="rdfs:label" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-274' property="rdfs:label" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-275' property="Thesaurus:Preferred_Name" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-276' property="rdfs:label" content="null allele" datatype="rdf:PlainLiteral"></span><span id='am-277' property="rdfs:label" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-278' property="Thesaurus:code" content="TEESS12" datatype="rdf:PlainLiteral"></span><span id='am-279' property="rdfs:label" content="survival" datatype="rdf:PlainLiteral"></span><span id='am-280' property="Thesaurus:Preferred_Name" content="Targeted mutation" datatype="rdf:PlainLiteral"></span><span id='am-281' property="Thesaurus:code" content="TEESS11" datatype="rdf:PlainLiteral"></span><span id='am-282' property="Thesaurus:code" content="TEESS20" datatype="rdf:PlainLiteral"></span><span id='am-283' property="Thesaurus:code" content="TEESS22" datatype="rdf:PlainLiteral"></span><span id='am-284' property="Thesaurus:Preferred_Name" content="Mutation" datatype="rdf:PlainLiteral"></span><span id='am-285' property="Thesaurus:code" content="TEESS08" datatype="rdf:PlainLiteral"></span><span id='am-286' property="rdfs:label" content="Gene mutation" datatype="rdf:PlainLiteral"></span><span id='am-287' property="Thesaurus:code" content="TEESS05" datatype="rdf:PlainLiteral"></span><span id='am-288' property="Thesaurus:Preferred_Name" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-289' property="Thesaurus:code" content="TEESS25" datatype="rdf:PlainLiteral"></span><span id='am-290' property="Thesaurus:Preferred_Name" content="Nonsense mutation" datatype="rdf:PlainLiteral"></span><span id='am-291' property="rdfs:label" content="Novel mutation" datatype="rdf:PlainLiteral"></span><span id='am-292' property="rdfs:label" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-293' property="rdfs:label" content="Insertion mutation" datatype="rdf:PlainLiteral"></span><span id='am-294' property="Thesaurus:Preferred_Name" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-295' property="rdfs:label" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-296' property="rdfs:label" content="Germline mutation" datatype="rdf:PlainLiteral"></span><span id='am-297' property="rdfs:label" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-298' property="rdfs:label" content="Knock-out" datatype="rdf:PlainLiteral"></span><span id='am-299' property="rdfs:label" content="RNA interference pathway" datatype="rdf:PlainLiteral"></span><span id='am-300' property="rdfs:label" content="lethal phenotype" datatype="rdf:PlainLiteral"></span><span id='am-301' property="Thesaurus:code" content="TEESS03" datatype="rdf:PlainLiteral"></span><span id='am-302' property="rdfs:label" content="Silent mutation" datatype="rdf:PlainLiteral"></span><span id='am-303' property="Thesaurus:code" content="TEESS16" datatype="rdf:PlainLiteral"></span><span id='am-304' property="Thesaurus:Preferred_Name" content="knockout" datatype="rdf:PlainLiteral"></span><span id='am-305' property="Thesaurus:Preferred_Name" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-306' property="rdfs:label" content="Induced mutation" datatype="rdf:PlainLiteral"></span><span id='am-307' property="Thesaurus:code" content="TEESS09" datatype="rdf:PlainLiteral"></span><span id='am-308' property="Thesaurus:Preferred_Name" content="cell survival" datatype="rdf:PlainLiteral"></span><span id='am-309' property="Thesaurus:code" content="TEESS13" datatype="rdf:PlainLiteral"></span><span id='am-310' property="Thesaurus:code" content="TEESS07" datatype="rdf:PlainLiteral"></span><span id='am-311' property="rdfs:label" content="Point mutation" datatype="rdf:PlainLiteral"></span><span id='am-312' property="Thesaurus:code" content="TEESS15" datatype="rdf:PlainLiteral"></span><span id='am-313' property="Thesaurus:Preferred_Name" content="RNA Interference" datatype="rdf:PlainLiteral"></span><span id='am-314' property="Thesaurus:Preferred_Name" content="Frameshift mutation" datatype="rdf:PlainLiteral"></span><span id='am-315' property="Thesaurus:code" content="TEESS24" datatype="rdf:PlainLiteral"></span>deletion mutation</span> secA2. Even <br /> blot BamHI-digested genomic DNAs probed secA2 probe. <br /> Sizes fragments indicated left. Lanes: 1, mc2155; 2, <br />                                                                       strain mc22522 contains in-frame deletion secA2, <br /> MB573; 3, &#226;&#338;&#172;secA2 (mc22522 produced MB573); 4, wild-type          avoid polar effects genes, per- <br /> recombinant produced MB573.                                      formed complementation analysis. We transformed mc22522 <br />  <br /> which carry secA2 gene control constitutive <br /> hsp60 promoter. Vector pMB208 multicopy vector express- <br /> ing M. smegmatis secA2, pMB147 pMB162 mul- <br /> ticopy single-copy vectors, respectively, express M. <br /> tuberculosis secA2. Secondary recombination products se- <br /> lected recombinant strains. None secA2 <br /> expression constructs enabled production &#226;&#338;&#172;secA1 mu- <br /> tants (Table 4). <br />    secA2 essential M. smegmatis. The essential nature <br /> of secA2 M. smegmatis evaluated. Suicide plasmid <br /> pMB160 contains in-frame deletion 1,296 bp M. <br /> smegmatis secA2. This deletion produces mutated protein of <br />                                                                         FIG. 4. Growth defect &#226;&#338;&#172;secA2 mutant rich medium agar <br /> 366 amino acids (in comparison wild-type protein 798        plates. Single colonies &#226;&#338;&#172;secA2 mutant mc22522, wild-type mc2155, <br /> amino acids). Once again, ATP-binding sites included          complemented &#226;&#338;&#172;secA2 attB::secA2 strain mc22522/pMB162 are <br /> in deletion (Fig. 1). By using pMB160, single-crossover           shown Mueller-Hinton plates grown 37&#194;&#176;C. <br />  6986     BRAUNSTEIN ET AL.                                                                                                  J. BACTERIOL. <br />  <br />  <br />                                                                          azide similarly acts inhibit SecA mycobacteria. We tested <br />                                                                          specific mutations secA1 confer azide resistance <br />                                                                          M. smegmatis. Multicopy plasmid pYUB499, contains <br />                                                                          secA1 M. bovis BCG, subjected chemical mutagen- <br />                                                                          esis introduced mc2155. Azide-resistant transformants <br />                                                                          selected plates containing azide arose fre- <br />                                                                          quency 10&#226;&#171;&#186;4. The azide resistance transformant was <br />                                                                          shown segregate mutagenized plasmid, designated <br />                                                                          pYUB538, means retransformation experiments. Se- <br />                                                                          quence analysis secA1 gene pYUB538 revealed a <br />                                                                          G-to-A mutation base pair 386. This mutation results in <br />                                                                          replacement threonine 129 isoleucine ABCI (Fig. 1). <br />                                                                          The corresponding mutation secA B. subtilis, replacement <br />                                                                          threonine 128 isoleucine, previously reported <br />                                                                          produce azide resistance (32). Thus, test provided com- <br />                                                                          pelling evidence sodium azide acts similar fashion on <br />                                                                          SecA1 mycobacteria. <br />                                                                             To measure impact SecA2 azide sensitivity of <br />                                                                          M. smegmatis, disk assay employed (see Materials and <br />                                                                          Methods). The zone clearing azide-soaked disk <br />                                                                          used measure azide sensitivity. This assay clearly <br />                                                                          demonstrated &#226;&#338;&#172;secA2 mutant supersensitive to <br />                                                                          azide. While wild-type mc2155 zone sensitivity 17.7 <br />    FIG. 5. The &#226;&#338;&#172;secA2 mutant sensitive overexpression secA1.    mm, mc22522 exhibited 31.3-mm zone sensitivity. The <br /> (A) &#226;&#338;&#172;secA2 mutant mc22522 transformed various plasmids. <br /> Lanes: 1, pMV261; 2, pMB175 (single-copy secA1); 3, pYUB499 (mul- <br />                                                                          complemented strain, mc22522 pMB162, exhibited in- <br /> ticopy secA1). (B) mc2155 transformed various plasmids. Lanes: 1,   termediate zone sensitivity 20.3 mm. This increased azide <br /> pMV261; 2, pYUB499. Colonies grown Mueller-Hinton agar           sensitivity mc22522 similarly observed liquid MIC <br /> plates 37&#194;&#176;C.                                                          analysis (data shown). This supersensitivity phenotype is <br />                                                                          specific azide pleiotropic effect altered cell <br /> with pMB162, integrates mycobacterial chro-               wall permeability. The sensitivity mc22522 collection of <br /> mosome expresses secA2 gene M. tuberculosis              drugs including streptomycin, isoniazid, rifampin, ampicillin, <br /> the constitutive hsp60 promoter. The introduction secA2          vancomycin unchanged. Given relationship be- <br /> expression construct successfully rescued small-colony phe-          tween azide SecA, believe supersensitivity <br /> notype mc22522, demonstrating phenotype              phenotype reflects role SecA2 assisting SecA1 the <br /> result secA2 mutation.                                            essential Sec pathway. <br />    Since SecA1 SecA2 share significant homology                 SecA2 functions protein export. The azide supersensitivity <br /> other SecA proteins, tested increased expression           phenotype suggested SecA2 plays role assisting SecA1 <br /> secA1 suppress small-colony phenotype mc22522.              protein export. The strong homology SecA2 other <br /> To surprise, did overexpression SecA1                SecA proteins suggested role protein export. There- <br /> suppress phenotype, exacerbated growth defect (Fig.           fore, set determine cells lacking secA2 ex- <br /> 5). An extra copy secA1 integrated chromosome             hibit defect protein export. <br /> mc22522 causes reduction colony size, expression              To monitor protein export, used immunoblot analysis of <br /> secA1 multicopy vector dramatic                  exported PhoA (E. coli alkaline phosphatase) fusion proteins <br /> effect. This result obtained secA1 genes            ascertain protein export occurs normally or <br /> both M. bovis BCG M. smegmatis. This effect                   unprocessed precursor proteins accumulate intracel- <br /> &#226;&#338;&#172;secA2 phenotype seen rich agar plates               lularly (indicative defect protein export). Each the <br /> specific &#226;&#338;&#172;secA2 background, overexpression                fusions analyzed exported M. tuberculosis protein <br /> secA1 wild-type parent strain mc2155 obvious effect            (FbpB [antigen 85B, Rv1886c], Rv1566c, PepA [Rv0125]) <br /> on colony size. This synthetic phenotype &#226;&#338;&#172;secA2              fused amino terminus truncated PhoA protein <br /> high-level expression secA1 suggestive relationship           lacking endogenous signal sequence. These fusions were <br /> between genes.                                                 recently isolated genetic screen identify M. tuberculosis <br />    The &#226;&#338;&#172;secA2 mutant supersensitive azide. In E. coli,             proteins exported host strain M. smegmatis <br /> sodium azide inhibits protein export vivo vitro           (5). We, others, observed exported <br /> (35). Furthermore, SecA shown major                   PhoA fusion proteins expressed M. smegmatis de- <br /> essential target sodium azide, azide-resistant mutants             tected Western analysis whole-cell extracts, products <br /> map secA E. coli (21, 35). Therefore, investigated              identified migrate expected molecular weight of <br /> whether presence SecA2 influences azide sensitivity           fusion protein (6, 49). Rather, different fusion proteins <br /> of M. smegmatis.                                                         lead production similar, smaller PhoA-reactive prod- <br />    Since azide sensitivity previously studied            ucts near size expected native PhoA. Although <br /> mycobacteria, set determine sodium               size products unexpected, believe they <br />  VOL. 183, 2001                                                            SecA1 AND SecA2 IN MYCOBACTERIUM SMEGMATIS                       6987 <br />  <br />  <br />  <br />  <br />    FIG. 6. Immunoblot analysis PhoA fusions wild-type &#226;&#338;&#172;secA2 (mc22522) strains. Whole-cell extracts individual strains were <br /> prepared, 20-&#226;?&#174;l volumes loaded lane SDS-PAGE Western analysis. (A) Anti-PhoA antibodies used Western analysis <br /> of whole-cell extracts. Lanes: 1, MB509 (mc2155/pMV261); 2, mc22757 (mc2155/pMB174); 3, MB648 (mc2155/pMB206); 4, MB648 azide <br /> treatment; 5, MB649 (mc22522/pMB206); 6, MB634 (mc2155/pMB202); 7, MB634 azide treatment; 8, MB635 (mc22522/pMB202). (B) <br /> Anti-HA Western blot. Lanes: 1, MB648; 2, MB648 azide treatment; 3, MB649. (C) Anti-PhoA Western blot. Lanes: 1, MB509; 2, mc22757; <br /> 3, MB624 (mc2155/pMB196); 4, MB625 (mc22522/pMB196). Marker sizes indicated left kilodaltons. Full-length FbpB-HA-&#226;&#172;&#732;PhoA and <br /> Rv1566c&#226;&#172;&#732;-&#226;&#172;&#732;PhoA fusions indicated arrows. &#226;&#172;&#732;PhoA breakdown products identified bracket near the blots. The asterisks <br /> indicate cross-reacting products. Strains grown overnight presence sodium azide indicated blot. Plasmid pMB174 expresses <br /> &#226;&#172;&#732;phoA, pMB206 expresses fbpB-HA-&#226;&#172;&#732;phoA fusion, pMB202 expresses rv1566c-&#226;&#172;&#732;phoA fusion, pMB196 expresses pepA-&#226;&#172;&#732;phoA fusion. <br />  <br />  <br /> reflect exported PhoA fusions. This based findings              correct-sized full-length fusion product (76.4 kDa for <br /> that PhoA fusion proteins exhibit phosphatase activity              FbpB-HA-&#226;&#172;&#732;PhoA 64.7 kDa Rv1566c-&#226;&#172;&#732;PhoA) observed, <br /> (indicating export cytoplasm) contain            reduced proteolytic PhoA products <br /> recognizable signal sequences. We propose smaller              (Fig. 6A, lanes 5 8). We interpret change size as <br /> PhoA-reactive products arise proteolysis occurs follow-           reflecting accumulation full-length fusion product in <br /> ing export cell wall. In E. coli, similar proteolytic            cytoplasm, protected cell wall proteases, <br /> degradation near PhoA fusion junction exported                showing mc22522 exhibits defect export of <br /> PhoA fusion proteins observed (23). Given prop-             fusion proteins. <br /> erty PhoA fusions M. smegmatis, predicted                    In support interpretation, strains <br /> defect export proteins result decrease         MB648 (mc2155 expressing FbpB-HA-&#226;&#172;&#732;PhoA) MB634 <br /> in degradation fusions, yielding increase            (mc2155 expressing Rv1566c-&#226;&#172;&#732;PhoA) treated azide, <br /> amount full-length fusion inside cells.                            full-length fusion protein appeared (Fig. 6A, lanes 4 <br />    When FbpB-HA-&#226;&#172;&#732;PhoA Rv1566c-&#226;&#172;&#732;PhoA                     7). Since azide treatment cripple export <br /> fusion expressed wild-type mc2155, bands 45             pathway inactivating essential SecA1 protein, pos- <br /> 50 kDa observed full-length fusion product ob-              sibly SecA2 well, believe large product that <br /> served (Fig. 6A, lanes 3 6). In contrast,             appears result accumulation full-length fusion <br /> fusion proteins expressed &#226;&#338;&#172;secA2 mutant (mc22522),             protein cytoplasm. Finally, confirmed prod- <br />  6988     BRAUNSTEIN ET AL.                                                                                                 J. BACTERIOL. <br />  <br />  <br /> uct accumulating mc22522 FbpB-HA-&#226;&#172;&#732;PhoA fusion                housekeeping SecA protein. Our isolation azide-resistant <br /> protein taking advantage in-frame HA tag present           mutations SecA1 supports idea. We believe that <br /> the FbpB portion fusion. Western analysis anti-HA          SecA1 acts mycobacterial Sec family <br /> antibodies carried MB648 (mc2155 expressing                 homologues transport signal sequence-containing proteins <br /> FbpB-HA-&#226;&#172;&#732;PhoA), MB648 following treatment azide,              cytoplasmic membrane. However, definitive evi- <br /> MB649 (mc22522 expressing FbpB-HA-&#226;&#172;&#732;PhoA) (Fig. 6B). This               dence SecA1 functions manner requires further <br /> analysis confirmed large product appears              investigation construction conditional secA1 allele. <br /> mc22522 FbpB-HA-&#226;&#172;&#732;PhoA full-length fusion product.                  Our ability generate &#226;&#338;&#172;secA2 mutant demonstrates that <br />   In contrast, PepA-&#226;&#172;&#732;PhoA fusion similarly affected.         SecA2 essential. Although mutation full <br /> When fusion protein expressed mc2155             deletion, believe truncated protein retains <br /> mc22522, PhoA proteolytic products observed              function. Of ATP-binding sites, A motif <br /> (Fig 6C). Neither strain led production full-length          ABCI deleted, eliminate ATP-binding <br /> 67.5-kDa fusion protein. From studies far, conclude          hydrolysis activity (Fig. 1). Furthermore, subse- <br /> that export some, all, proteins involves SecA2.         quently constructed mutant M. tuberculosis com- <br />                                                                        plete deletion secA2 viable (M. Braunstein and <br />                           DISCUSSION                                   W. R. Jacobs, unpublished data). To knowledge, the <br />                                                                        SecA protein shown nonessential. <br />    Mycobacteria different SecA proteins. The proper              The role SecA2 mycobacteria. To begin understand <br /> localization cell surface secreted proteins impor-        role SecA2 cell, examined SecA2 is <br /> tant aspect mycobacterial physiology pathogenesis. The          capable carrying functions SecA1. The fact <br /> cell envelope mycobacteria unusual structure             mutant deletion secA1 inviable suggested that <br /> presents unique barriers protein secretion. Yet, protein        SecA2 simply equivalent copy SecA1. However, it <br /> export pathways mycobacteria remained uncharacter-             remained possibility result reflected lack ex- <br /> ized (5). The genome M. tuberculosis contains homologues            pression SecA2 experimental conditions. To <br /> of Sec pathway components (5, 8), exception        address directly, expressed secA2 constitutive <br /> of SecB, protein Sec systems (19). In       mycobacterial hsp60 promoter. By Western analysis, saw <br /> addition, mycobacteria known secrete proteins          SecA2 protein produced (data shown); how- <br /> initially synthesized precursors containing recognizable sig-       ever, requirement SecA1 relieved ex- <br /> nal sequences amino termini. Simply based             pression SecA2. This finding SecA2 functionally <br /> information, appear mycobacteria contain typ-          replace SecA1 consistent fact homology <br /> ical Sec pathway protein export cytoplasm.              shared SecA proteins 50% (Table 3). <br /> However, presented report, mycobacterial Sec               If SecA2 simply extra copy SecA1, the <br /> pathway unusual SecA proteins. Although          role SecA2? Based significant sequence homology <br /> SecA ubiquitously present highly conserved bacteria,         SecA2 SecA homologues, believed it <br /> the identification organisms containing secA genes           involved protein export&#226;&#8364;&#8221;perhaps export a <br /> new.                                                                   specific subset proteins. Our subsequent analysis supports <br />    Both mycobacterial SecA1 SecA2 proteins exhibit             idea. <br /> significant sequence similarity bacterial plant              A phenotypic analysis &#226;&#338;&#172;secA2 mutant mc22522 revealed <br /> (plastid) SecA homologues, contain hallmark               phenotypes include growth defect rich agar plates. <br /> ATP-binding motifs (Table 3; Fig. 1). All mycobacteria          The source phenotype unclear. This growth phenotype <br /> evaluated far secA1 secA2, including M. leprae        mc22522 exacerbated secA1 overexpressed, but <br /> (9, 26). This significant, given reductive evolution       overexpression secA1 wild-type background does not <br /> has documented M. leprae&#226;&#8364;&#8221;a smaller genome size             effect. This synthetic relationship high <br /> 44% genes recognizable pseudogenes (9). The               levels SecA1 absence SecA2 suggests these <br /> evolutionary basis SecA proteins mycobacteria            SecA proteins interact proteins or <br /> unknown. If genes arose duplication event, likely      other. This consistent role SecA2 protein <br /> took place long time ago SecA proteins         export. A possible explanation synthetic phenotype is <br /> extremely similar other, highly conserved       SecA2 absent SecA1 overexpressed, non- <br /> different mycobacterial species. SecA1 exhibits higher se-             productive complexes SecA1 essential Sec <br /> quence similarity SecA homologues, including            factors arise, leading overall decrease efficiency of <br /> that distantly related bacteria E. coli B. subtilis,   protein export. This possibility explored future ex- <br /> it does mycobacterial SecA2 (Table 3).                            periments. <br />    SecA1 housekeeping SecA protein, SecA2 extra            The azide supersensitivity phenotype mc22522 sug- <br /> SecA protein. As starting point defining roles played        gests involvement SecA2 protein export. Azide poisons <br /> by SecA1 SecA2, used allelic exchange determine              ATPases, SecA1 appears sensitive essential <br /> whether secA gene essential. These experiments                 ATPase mycobacteria. For reason, interpret this <br /> showed secA1 essential M. smegmatis, secA2         phenotype indicative azide greater effect the <br /> not. In bacteria, SecA essential.                             Sec pathway SecA2 present assist process. <br />    The essentiality SecA1 leads believe repre-           The direct experiment undertook identify role <br /> sents mycobacterial equivalent E. coli SecA              SecA2 export proteins monitor protein <br />  VOL. 183, 2001                                                      SecA1 AND SecA2 IN MYCOBACTERIUM SMEGMATIS                                  6989 <br />  <br />  <br /> processing measure protein translocation         maintained abilities function SecA1 and <br /> cytoplasmic membrane. In looking steady-state levels      Sec family homologues primary Sec pathway <br /> three PhoA fusion proteins, FbpB-HA-               evolved second function utilized under <br /> &#226;&#172;&#732;PhoA fusion Rv1566c-&#226;&#172;&#732;PhoA fusion effi-           certain set conditions export specific subset pro- <br /> ciently exported &#226;&#338;&#172;secA2 mutant. It important em-       teins. <br /> phasize export defect mc22522 complete. The <br /> immunoblots reveal reduced export (PhoA pro-                                      ACKNOWLEDGMENTS <br /> teolytic products), mc22522 strains expressing          We gratefully acknowledge F. C. Bange V. Balasubramanian for <br /> fusions exhibit phosphatase activity (which associated     genomic libraries, T. Weisbrod assistance DNA sequencing, <br /> with transport fusion protein cytoplasm). In      A. Flower M. Pavelka critical reading manuscript. <br /> contrast, did evidence defect export       This work supported NIH grant AI21670 W.R.J. M.B. was <br />                                                                     Howard Hughes Medical Institute Fellow Life Sciences Re- <br /> of PepA-&#226;&#172;&#732;PhoA fusion protein mc22522. This finding re-         search Foundation course work. <br /> veals SecA2 participates protein export some, <br /> perhaps all, signal sequence-containing proteins.                                                REFERENCES <br />    We interpret results indicating role SecA2        1. Akita, M., A. Shinkai, S. Matsuyama, S. Mizushima. 1991. SecA, an <br />                                                                         essential component secretory machinery Escherichia coli, exists as <br /> protein export. An alternative explanation SecA2          homodimer. Biochem. Biophys. Res. Commun. 174:211&#226;&#8364;&#8220;216. <br /> is regulator level SecA1. This idea supported      2. Berthet, F. X., M. Lagranderie, P. Gounon, C. Laurent-Winter, D. Enser- <br /> the demonstration E. coli SecA regulate             gueix, P. Chavarot, F. Thouron, E. Maranghi, V. Pelicic, D. Portnoi, G. <br />                                                                         Marchal, B. Gicquel. 1998. Attenuation virulence disruption the <br /> expression (44). In examining Western analyses cur-            Mycobacterium tuberculosis erp gene. Science 282:759&#226;&#8364;&#8220;762. <br /> rently available reagents, indication levels    3. Blanco, J., A. J. Driessen, J. J. Coque, J. F. Martin. 1998. Biochemical <br /> of SecA1 altered presence absence SecA2             characterization SecA protein Streptomyces lividans&#226;&#8364;&#8221;interaction <br />                                                                         nucleotides, binding membrane vesicles vitro translocation of <br /> the cell; therefore, think explanation role          proAmy protein. Eur. J. Biochem. 257:472&#226;&#8364;&#8220;478. <br /> SecA2 unlikely (data shown).                                     4. Braunstein, M., J. T. Belisle. 2000. Genetics protein secretion, p. <br />                                                                         203&#226;&#8364;&#8220;220. In G. F. Hatfull W. R. J. Jacobs (ed.), Molecular genetics of <br />    Two roles SecA2 mycobacteria. We propose             mycobacteria. ASM Press, Washington, D.C. <br /> role SecA2 facilitator protein export         5. Braunstein, M., T. I. Griffin, J. I. Kriakov, S. T. Friedman, N. D. Grindley, <br /> primary Sec pathway involving SecA1. We believe                W. R. Jacobs, Jr. 2000. Identification genes encoding exported My- <br />                                                                         cobacterium tuberculosis proteins using Tn552&#226;&#172;&#732;phoA vitro transposition <br /> azide supersensitivity phenotype export defect              system. J. Bacteriol. 182:2732&#226;&#8364;&#8220;2740. <br /> signal sequence-containing PhoA fusion proteins reflect         6. Carroll, J. D., R. C. Wallace, J. Keane, H. G. Remold, R. D. Arbeit. 2000. <br /> role SecA2. There collection signal sequence-            Identification Mycobacterium avium DNA sequences encode exported <br />                                                                         proteins using phoA gene fusions. Tuber. Lung. Dis. 80:117&#226;&#8364;&#8220;130. <br /> containing presecretory proteins substrates        7. Cirillo, J. D., T. R. Weisbrod, A. Banerjee, B. R. Bloom, W. R. Jacobs, <br /> SecA1 SecA2. Thus, SecA2 absent,                Jr. 1994. Genetic determination meso-diaminopimelate biosynthetic <br />                                                                         pathway mycobacteria. J. Bacteriol. 176:4424&#226;&#8364;&#8220;4429. <br /> impact overall efficiency pathway. We propose          8. Cole, S. T., R. Brosch, J. Parkhill, T. Garnier, C. Churcher, D. Harris, S. V. <br /> that FbpB-&#226;&#172;&#732;PhoA Rv1566c-&#226;&#172;&#732;PhoA proteins,                Gordon, K. Eiglmeier, S. Gas, C. E. Barry, III, F. Tekaia, K. Badcock, D. <br /> since SecA2 participates, dos exclusively act,         Basham, D. Brown, T. Chillingworth, R. Connor, R. Davies, K. Devlin, T. <br />                                                                         Feltwell, S. Gentles, N. Hamlin, S. Holroyd, T. Hornsby, K. Jagels, B. G. <br /> export. In E. coli, B. subtilis, Streptomyces lividans, SecA        Barrell, et al. 1998. Deciphering biology Mycobacterium tuberculosis <br /> has shown exist dimer (1, 3, 13, 47). Therefore,        complete genome sequence. Nature 393:537&#226;&#8364;&#8220;544. <br /> is possible SecA1 SecA2 form mixed dimers          9. Cole, S. T., K. Eiglmeier, J. Parkhill, K. D. James, N. R. Thomson, P. R. <br />                                                                         Wheeler, N. Honore, T. Garnier, C. Churcher, D. Harris, K. Mungall, D. <br /> that species important recognition certain            Basham, D. Brown, T. Chillingworth, R. Connor, R. M. Davies, K. Devlin, S. <br /> substrates.                                                             Duthoy, T. Feltwell, A. Fraser, N. Hamlin, S. Holroyd, T. Hornsby, K. Jagels, <br />                                                                         C. Lacroix, J. Maclean, S. Moule, L. Murphy, K. Oliver, M. A. Quail, M. A. <br />    The findings SecA2 functionally replace SecA1            Rajandream, K. M. Rutherford, S. Rutter, K. Seeger, S. Simon, M. Sim- <br /> and SecA2 highly conserved mycobacteria              monds, J. Skelton, R. Squares, S. Squares, K. Stevens, K. Taylor, S. White- <br /> lead believe SecA2 second role cell.            head, J. R. Woodward, B. G. Barrell. 2001. Massive gene decay the <br />                                                                         leprosy bacillus. Nature 409:1007&#226;&#8364;&#8220;1011. <br />    This second role export distinct subset    10. Connell, N. D. 1994. Mycobacterium: isolation, maintenance, transforma- <br /> proteins exclusive preferential substrates SecA2.        tion, mutant selection. Methods Cell Biol. 45:108&#226;&#8364;&#8220;125. <br /> The identities proteins established,     11. Danese, P. N., T. J. Silhavy. 1998. Targeting assembly periplasmic <br />                                                                         outer-membrane proteins Escherichia coli. Annu. Rev. Genet. 32:59&#226;&#8364;&#8220; <br /> their identification awaits complete analysis             94. <br /> multitude exported proteins. In order identify pro-     12. Donnelly-Wu, M. K., W. R. Jacobs, Jr., G. F. Hatfull. 1993. Superinfec- <br />                                                                         tion immunity mycobacteriophage L5: applications genetic transfor- <br /> teins, important know conditions          mation mycobacteria. Mol. Microbiol. 7:407&#226;&#8364;&#8220;417. <br /> SecA2-specific protein export functions.                            13. Driessen, A. J. 1993. SecA, peripheral subunit Escherichia coli <br />    Research numerous bacterial pathogens repeatedly              precursor protein translocase, functional dimer. Biochemistry 32: <br />                                                                         13190&#226;&#8364;&#8220;13197. <br /> demonstrated importance specialized protein secretion        14. Duong, F., W. Wickner. 1997. Distinct catalytic roles SecYE, SecG <br /> systems virulence. Many specialized export pathways           SecDFyajC subunits preprotein translocase holoenzyme. EMBO J. <br /> are present pathogenicity islands. The genome M. tuber-           16:2756&#226;&#8364;&#8220;2768. <br />                                                                     15. Economou, A. 1998. Bacterial preprotein translocase: mechanism con- <br /> culosis does appear specialized export systems              formational dynamics processive enzyme. Mol. Microbiol. 27:511&#226;&#8364;&#8220;518. <br /> homologous previously identified pathogens        16. Economou, A. 2000. Bacterial protein translocase: unique molecular ma- <br />                                                                         chine army substrates. FEBS Lett. 476:18&#226;&#8364;&#8220;21. <br /> (4). However, recent identification secA genes    17. Economou, A. 1999. Following leader: bacterial protein export through <br /> gram-positive pathogens S. aureus S. pneumoniae raises              Sec pathway. Trends Microbiol. 7:315&#226;&#8364;&#8220;320. <br /> the interesting possibility SecA2 component new      18. Economou, A., W. Wickner. 1994. SecA promotes preprotein transloca- <br />                                                                         tion undergoing ATP-driven cycles membrane insertion deinser- <br /> type specialized protein export pathway important         tion. Cell 78:835&#226;&#8364;&#8220;843. <br /> pathogenesis (24, 27, 48). SecA2 represent protein       19. Fekkes, P., A. J. Driessen. 1999. Protein targeting bacterial <br />  6990       BRAUNSTEIN ET AL.                                                                                                                            J. BACTERIOL. <br />  <br />  <br />     cytoplasmic membrane. Microbiol. Mol. Biol. Rev. 63:161&#226;&#8364;&#8220;173.                         tial function Mycobacterium smegmatis. J. Bacteriol. 178:6496&#226;&#8364;&#8220;6507. <br /> 20. Finlay, B. B., S. Falkow. 1997. Common themes microbial pathoge-          37. Pavelka, M. S., Jr., W. R. Jacobs, Jr. 1999. Comparison construc- <br />     nicity revisited. Microbiol. Mol. Biol. Rev. 61:136&#226;&#8364;&#8220;169.                             tion unmarked deletion mutations Mycobacterium smegmatis, Mycobac- <br /> 21. Fortin, Y., P. Phoenix, G. R. Drapeau. 1990. Mutations conferring                terium bovis, bacillus Calmette-Guerin, Mycobacterium tuberculosis <br />     resistance azide Escherichia coli occur primarily secA gene. J.         H37Rv allelic exchange. J. Bacteriol. 181:4780&#226;&#8364;&#8220;4789. <br />     Bacteriol. 172:6607&#226;&#8364;&#8220;6610.                                                        38. Pelicic, V., J. M. Reyrat, B. Gicquel. 1996. Expression Bacillus <br /> 22. Gomez, M., L. Doukhan, G. Nair, I. Smith. 1998. sigA essential             subtilis sacB gene confers sucrose sensitivity mycobacteria. J. Bacteriol. <br />     gene Mycobacterium smegmatis. Mol. Microbiol. 29:617&#226;&#8364;&#8220;628.                         178:1197&#226;&#8364;&#8220;1199. <br /> 23. Kaufman, M. R., R. K. Taylor. 1994. Identification bacterial cell-        39. Pohlschroder, M., W. A. Prinz, E. Hartmann, J. Beckwith. 1997. Protein <br />     surface virulence determinants TnphoA. Methods Enzymol. 235:426&#226;&#8364;&#8220;                translocation domains life: variations theme. Cell 91: <br />     448.                                                                                 563&#226;&#8364;&#8220;566. <br /> 24. Kuroda, M., T. Ohta, I. Uchiyama, T. Baba, H. Yuzawa, I. Kobayashi, L. Cui,      40. Roux, K. H. 1994. Using mismatched primer-template pairs touchdown <br />     A. Oguchi, K. Aoki, Y. Nagai, J. Lian, T. Ito, M. Kanamori, H. Matsumaru,            PCR. BioTechniques 16:812&#226;&#8364;&#8220;814. <br />     A. Maruyama, H. Murakami, A. Hosoyama, Y. Mizutani-Ui, N. K. Taka-               41. Sambrook, J. E., E. F. Fritsch, T. Maniatis. 1989. Molecular cloning: a <br />     hashi, T. Sawano, R. Inoue, C. Kaito, K. Sekimizu, H. Hirakawa, S. Kuhara,           laboratory manual, 2nd ed. Cold Spring Harbor Laboratory Press, Cold <br />     S. Goto, J. Yabuzaki, M. Kanehisa, A. Yamashita, K. Oshima, K. Furuya, C.            Spring Harbor, N.Y. <br />     Yoshino, T. Shiba, M. Hattori, N. Ogasawara, H. Hayashi, K. Hira-            42. Schiebel, E., A. J. Driessen, F. U. Hartl, W. Wickner. 1991. Delta mu H&#226;&#171;&#185; <br />     matsu. 2001. Whole genome sequencing methicillin-resistant Staphylococ-           ATP function different steps catalytic cycle preprotein <br />     cus aureus. Lancet 357:1225&#226;&#8364;&#8220;1240.                                                    translocase. Cell 64:927&#226;&#8364;&#8220;939. <br /> 25. Lee, M. H., L. Pascopella, W. R. Jacobs, Jr., G. F. Hatfull. 1991.           43. Schmidt, M. G., K. B. Kiser. 1999. SecA: ubiquitous component of <br />     Site-specific integration mycobacteriophage L5: integration-proficient            preprotein translocase prokaryotes. Microbes Infect. 1:993&#226;&#8364;&#8220;1004. <br />     vectors Mycobacterium smegmatis, Mycobacterium tuberculosis, ba-         44. Schmidt, M. G., D. B. Oliver. 1989. SecA protein autogenously represses <br />     cille Calmette-Guerin. Proc. Natl. Acad. Sci. USA 88:3111&#226;&#8364;&#8220;3115.                      translation normal protein secretion Escherichia coli. J. <br /> 26. Limia, A., F. J. Sangari, D. Wagner, L. E. Bermudez. 2001. Character-            Bacteriol. 171:643&#226;&#8364;&#8220;649. <br />     ization expression secA Mycobacterium avium. FEMS Microbiol.           45. Snapper, S. B., R. E. Melton, S. Mustafa, T. Kieser, W. R. Jacobs, Jr. <br />     Lett. 197:151&#226;&#8364;&#8220;157.                                                                   1990. Isolation characterization efficient plasmid transformation mu- <br /> 27. Mazmanian, S. K., H. Ton-That, O. Schneewind. 2001. Sortase-catalysed            tants Mycobacterium smegmatis. Mol. Microbiol. 4:1911&#226;&#8364;&#8220;1919. <br />     anchoring surface proteins cell wall Staphylococcus aureus. Mol.    46. Stover, C. K., V. F. la Cruz, T. R. Fuerst, J. E. Burlein, L. A. Benson, L. T. <br />     Microbiol. 40:1049&#226;&#8364;&#8220;1057.                                                             Bennett, G. P. Bansal, J. F. Young, M. H. Lee, G. F. Hatfull, et al. 1991. New <br /> 28. Mecsas, J. J., E. J. Strauss. 1996. Molecular mechanisms bacterial            use BCG recombinant vaccines. Nature 351:456&#226;&#8364;&#8220;460. <br />     virulence: type III secretion pathogenicity islands. Emerg. Infect. Dis.     47. Takamatsu, H., A. Nakane, A. Oguro, Y. Sadaie, K. Nakamura, K. <br />     2:270&#226;&#8364;&#8220;288.                                                                           Yamane. 1994. A truncated Bacillus subtilis SecA protein consisting the <br /> 29. Miller, J. F., P. Cossart. 1999. Bacterial pathogenesis: post-        N-terminal 234 amino acid residues forms complex Escherichia coli <br />     genomic era. Curr. Opin. Microbiol. 2:15&#226;&#8364;&#8220;17.                                         SecA51(ts) protein complements protein translocation defect the <br /> 30. Miller, J. H. 1992. A short course bacterial genetics: laboratory manual        secA51 mutant. J. Biochem. (Tokyo) 116:1287&#226;&#8364;&#8220;1294. <br />     handbook Escherichia coli related bacteria. Cold Spring Harbor       48. Tettelin, H., K. E. Nelson, I. T. Paulsen, J. A. Eisen, T. D. Read, S. Peterson, <br />     Laboratory Press, Cold Spring Harbor, N.Y.                                           J. Heidelberg, R. T. DeBoy, D. H. Haft, R. J. Dodson, A. S. Durkin, M. <br /> 31. Mitchell, C., D. Oliver. 1993. Two distinct ATP-binding domains              Gwinn, J. F. Kolonay, W. C. Nelson, J. D. Peterson, L. A. Umayam, O. White, <br />     needed promote protein export Escherichia coli SecA ATPase. Mol.               S. L. Salzberg, M. R. Lewis, D. Radune, E. Holtzapple, H. Khouri, A. M. <br />     Microbiol. 10:483&#226;&#8364;&#8220;497.                                                               Wolf, T. R. Utterback, C. L. Hansen, L. A. McDonald, T. V. Feldblyum, S. <br /> 32. Nakane, A., H. Takamatsu, A. Oguro, Y. Sadaie, K. Nakamura, K.                   Angiuoli, T. Dickinson, E. K. Hickey, I. E. Holt, B. J. Loftus, F. Yang, H. O. <br />     Yamane. 1995. Acquisition azide-resistance elevated SecA ATPase                Smith, J. C. Venter, B. A. Dougherty, D. A. Morrison, S. K. Hollingshead, <br />     activity confers azide-resistance cell growth protein translocation         C. M. Fraser. 2001. Complete genome sequence virulent isolate of <br />     Bacillus subtilis. Microbiology 141:113&#226;&#8364;&#8220;121.                                      Streptococcus pneumoniae. Science 293:498&#226;&#8364;&#8220;506. <br /> 33. Oliver, D. B. 1993. SecA protein: autoregulated ATPase catalysing prepro-        49. Timm, J., M. G. Perilli, C. Duez, J. Trias, G. Orefici, L. Fattorini, G. <br />     tein insertion translocation Escherichia coli inner membrane.         Amicosante, A. Oratore, B. Joris, J. M. Frere, et al. 1994. Transcription and <br />     Mol. Microbiol. 7:159&#226;&#8364;&#8220;165.                                                           expression analysis, using lacZ phoA gene fusions, Mycobacterium <br /> 34. Oliver, D. B., J. Beckwith. 1981. E. coli mutant pleiotropically defective       fortuitum beta-lactamase genes cloned natural isolate high-level <br />     export secreted proteins. Cell 25:765&#226;&#8364;&#8220;772.                                 beta-lactamase producer. Mol. Microbiol. 12:491&#226;&#8364;&#8220;504. <br /> 35. Oliver, D. B., R. J. Cabelli, K. M. Dolan, G. P. Jarosik. 1990. Azide-       50. van der Wolk, J. P., J. G. Wit, A. J. Driessen. 1997. The catalytic cycle <br />     resistant mutants Escherichia coli alter SecA protein, azide-sensi-        Escherichia coli SecA ATPase comprises distinct preprotein trans- <br />     tive component protein export machinery. Proc. Natl. Acad. Sci. USA           location events. EMBO J. 16:7297&#226;&#8364;&#8220;7304. <br />     87:8227&#226;&#8364;&#8220;8231.          <br /> </body></html>