<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             J Nutr. Author manuscript; available PMC 2007 November 1. <br />                            Published final edited form as: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             J Nutr. 2006 November ; 136(11): 2735&#8211;2742. <br />  <br />  <br />                            DROSOPHILA HOLOCARBOXYLASE SYNTHETASE IS A <br />                            CHROMOSOMAL PROTEIN REQUIRED FOR NORMAL HISTONE <br />                            BIOTINYLATION, GENE TRANSCRIPTION PATTERNS, LIFESPAN <br />                            AND HEAT TOLERANCE1,2 <br />  <br />                            Gabriela Camporeale&#8225;, Ennio Giordano&#182;, Rosaria Rendina&#182;, Janos Zempleni&#8225;,||, Joel C. <br />                            Eissenberg&#167;,|| <br />                            &#8225; From Department Nutrition Health Sciences, University Nebraska, Lincoln, Nebraska <br />                            68583-0806, the <br />                            &#182; Dipartimento delle Scienze Biologiche, Universit&#225; di Napoli Federico II, Naples 80134, Italy, and <br />                            the <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            &#167; Edward A. Doisy Department Biochemistry Molecular Biology, Saint Louis University <br />                            Medical School, Saint Louis, MO 63101. <br />  <br />                            Abstract <br />                                  Posttranslational modifications histones play important roles chromatin structure genomic <br />                                  stability. Distinct lysine residues histones targets covalent binding biotin, catalyzed by <br />                                  holocarboxylase synthetase (HCS) biotinidase (BTD). Histone biotinylation implicated <br />                                  heterochromatin structures, DNA repair, mitotic chromosome condensation. To test whether <br />                                  HCS BTD deficiency alters histone biotinylation, characterize phenotypes associated with <br />                                  HCS BTD deficiency, HCS- BTD-deficient flies generated RNA interference <br />                                  (RNAi). Expression HCS BTD decreased 65&#8211;90% RNAi-treated flies, judged by <br />                                  mRNA abundance, BTD activity, abundance HCS protein. Decreased expression HCS and <br />                                  BTD caused decreased biotinylation K9 K18 histone H3. <span id='am-49' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-50'  ></span><span id='am-51' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> associated altered <br />                                  expression 201 genes HCS-deficient flies. Lifespan HCS- BTD-deficient flies decreased <br />                                  32% compared wild-type controls. Heat tolerance decreased 55% HCS- <br />                                  deficient flies compared controls, judged survival times; effects BTD deficiency were <br />                                  minor. Consistent observation, HCS deficiency associated altered expression of <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  285 heat-responsive genes. HCS BTD deficiency did affect cold tolerance, suggesting stress- <br />                                  specific effects chromatin remodeling histone biotinylation. This study provide <br />                                  evidence HCS-dependent histone biotinylation affects gene function phenotype, suggesting <br />                                  complex phenotypes HCS- BTD-deficiency disorders reflect chromatin structure <br />                                  changes. <br />  <br />  <br />  <br />  <br />                            1This work supported NIH grants DK 60447, DK 063945, 1 P20 RR16469, NSF grants EPSCoR EPS-0346476, MCB <br />                            0131414, MCB 6552870. This paper contribution University Nebraska Agricultural Research Division, Lincoln, NE <br />                            68583 (Journal Series No. 15192). <br />                            2Supplemental Tables S1 S2 available Online Supporting Material online posting paper at <br />                            http://jn.nutrition.org. <br />                            || Co-corresponding authors: J. Zempleni, Department Nutrition Health Sciences, University Nebraska-Lincoln, 316 Ruth <br />                            Leverton Hall, Lincoln, NE 68583-0806. Tel. 402-472-3270; Fax: 402-472-1587; E-mail: jzempleni2@unl.edu. J.C. Eissenberg, Edward <br />                            A. Doisy Department Biochemistry Molecular Biology, Saint Louis University Medical School, 1402 S. Grand Blvd., St. Louis, <br />                            MO 63101; Tel. 314-977-9236; Fax: 314-977-9205; E-mail: eissenjc@slu.edu.. <br />                             Camporeale et al.                                                                                                                 Page 2 <br />  <br />  <br />  <br />                               Keywords <br />                                     biotinidase; <span id='am-42' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-43'  ></span><span id='am-44' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span>Drosophila melanogaster</span>; holocarboxylase synthetase; lifespan; stress resistance <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 In eukaryotes, biotin serves covalently bound coenzyme acetyl-CoA carboxylase &#945;, <br />                                                 acetyl-CoA carboxylase &#946;, pyruvate carboxylase (PC)3, propionyl-CoA carboxylase (PCC), <br />                                                 3-methylcrotonyl-CoA carboxylase. Carboxylases play essential roles metabolism <br />                                                 fatty acids, glucose, amino acids. The attachment biotin specific lysine (K) <br />                                                 residues carboxylases catalyzed holocarboxylase synthetase (HCS) ATP- <br />                                                 dependent reaction (1). Recently, view potential biological functions biotin been <br />                                                 expanded observation histones contain covalently bound biotin (2,3). <br />  <br />                                                 The following lysine (K) residues histones targets biotinylation: K9, K13, K125, <br />                                                 K127, K129 histone H2A; K4, K9 K18 histone H3; K8 K12 histone <br />                                                 H4 (4&#8211;6). Evidence suggests K8-biotinylated histone H4 K12-biotinylated histone H4 <br />                                                 enriched heterochromatin participate gene silencing (Camporeale, G., Oommen, <br />                                                 A. M., Griffin, J. B., Sarath, G. &amp; Zempleni, J., unpublished <span id='am-103' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-104'  ></span><span id='am-105' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span>). Moreover, abundance <br />                                                 K8- K12 biotinylated histone H4 cell cycle dependent reaches peak levels during <br />                                                 mitotic chromatin condensation (7). Finally, K12-biotinylated histone H4 participate in <br />                                                 cellular response DNA double-strand breaks (8). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Histone biotinylation mediated HCS (9) biotinidase (BTD) (3). Covalent <br />                                                 modifications histones typically reversible (10); enzymes mediating debiotinylation of <br />                                                 histones largely unknown. Evidence suggests BTD mediate debiotinylation of <br />                                                 histones (11) addition acting histone biotinyl transferase (3). It uncertain cells <br />                                                 control opposing enzymatic activities BTD. Variables the <br />                                                 microenvironment confined regions chromatin, posttranslational modifications BTD, <br />                                                 alternate splicing BTD determine BTD acts biotinyl histone transferase <br />                                                 histone debiotinylase (12). In addition, BTD-binding proteins contribute regulation. <br />  <br />                                                 The biotin protein ligases evolutionarily conserved. A subset enzymes (exemplified <br />                                                 E. coli BirA) known function transcriptional repressors (13,14). The effector BirA <br />                                                 transcriptional repression biotinyl-5&#8242;-adenylate (bio-5&#8242;-AMP). The BirA-bio-5&#8242;-AMP <br />                                                 complex active enzymatic transfer biotin site-specific DNA binding. When <br />                                                 biotin abundant, BirA-bio-5&#8242;-AMP accumulates binds operator sequence the <br />                                                 biotin biosynthesis operon repress transcription. Thus, BirA couples intracellular demand <br />                                                 synthesis biotin E. coli. HCS structural functional homolog BirA <br />                                                 (15,16). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Inborn mutations human HCS BTD decrease histone biotinyl transferase activity of <br />                                                 enzymes (3,5,9). For example, biotinylation histones decreased HCS-deficient <br />                                                 human fibroblasts (5,9) lymphoma (Jurkat) cells (Camporeale, G. &amp; Zempleni, J., <br />                                                 unpublished data). Notwithstanding potential importance observations, effects of <br />                                                 hcs btd gene mutations chromatin structure phenotypes largely unknown. <br />  <br />                                                 The Drosophila <span id='am-98' about='obo:NCBITaxon_80614' typeof='owl:Thing obo:NCBITaxon_80611 obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_452333 obo:NCBITaxon_5204 obo:NCBITaxon_451864 obo:NCBITaxon_68889 obo:NCBITaxon_155619 obo:NCBITaxon_2759'><span id='am-99' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-100'  ></span><span id='am-101' property="rdfs:label" content="Melanogaster" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>melanogaster</span> genome encodes orthologs human HCS (GenBank accession <br />                                                 number AAF52022) BTD (GenBank accession number AAF46130). Consistent this <br />                                                 observation, major classes histones biotinylated Drosophila (17). Biotin <br />  <br />  <br />  <br />                               3The abbreviations used are: BTD, biotinidase; HCS, holocarboxylase synthetase; K9BioH3, lysine-9 biotinylated histone H3; <br />                               K18BioH3, lysine-18 biotinylated histone H3; IR, inverted repeat; K, lysine; PC, pyruvate carboxylase; PCC, propionyl-CoA carboxylase; <br />                               RNAi, RNA interference; y w, wild type. <br />  <br />  <br />                                                         J Nutr. Author manuscript; available PMC 2007 November 1. <br />                             Camporeale et al.                                                                                                  Page 3 <br />  <br />  <br />                                                deficiency associated decreased lifespan fertility flies (17), remains <br />                                                uncertain caused altered biotinylation histones. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                In present study, Drosophila used model investigate knockdown of <br />                                                HCS BTD expression associated altered chromatin structure, evidenced by <br />                                                decreased biotinylation histone H3. Histone H3 chosen model is <br />                                                biotinylated abundantly histones flies (17). We used HCS- BTD- <br />                                                deficient flies investigate effects altered histone biotinylation phenotypes flies. Here, <br />                                                focused lifespan temperature tolerance markers phenotypic changes, given <br />                                                previous studies linked biotin status variables (17). We identified genes whose <br />                                                expression altered loss HCS expression. <br />  <br />                               MATERIALS AND METHODS <br />                               Immunofluorescent staining polytene chromosomes <br />                                                Larval polytene chromosome squashes prepared essentially described (18). Anti-HCS <br />                                                serum used 1:50 dilution; FITC-labeled anti-rabbit secondary antibody used 1:200 <br />                                                dilution. <br />  <br />                               Generation transgenic flies <br />                                                HCS- BTD-deficient flies generated using RNA interference (RNAi) inserting <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                gene-specific cassettes pUAST-sp plasmid (19). For generation vector targeting <br />                                                HCS, 1.4kb DNA fragment Drosophila CG14670 gene (coordinates 35696 37133, <br />                                                GenBank accession no. AE003602) PCR amplified using following PCR primers: <br />                                                EcoRI-5&#8242;-HCS (5&#8242;- ATATGAATTCGGAAGATTACGGTAAGCTAATTGC-3&#8242;) and <br />                                                EcoRI-3&#8217;-HCS (5&#8242;-ATATGAATT CGCGACTTGGTGGACTCCTCGC-3&#8242;) generate the <br />                                                EcoRI-HCS-EcoRI cassette; XbaI-5&#8242;-HCS (5&#8242;- <br />                                                CGCTCTAGATTACGGTAAGCTAATTGC-3&#8242;) Xho-3&#8242;-HCS (5&#8242;- <br />                                                CGCCTCGAGTGCGACTT GGTGGACTCC-3&#8242;) generate XbaI-HCS-XhoI cassette. <br />                                                Both EcoRI-HCS-EcoRI XbaI-HCS-XhoI cassettes inserted inverted repeats <br />                                                (IR) pUAST-sp vector generate plasmid denoted pUAST-IRsp-HCS; the <br />                                                pUAST-IRsp-HCS plasmid HCS cassettes separated 200 bp spacer enhance <br />                                                stability IR (19). For generation vector targeting BTD, 1.5 kb DNA fragment of <br />                                                Drosophila CG3599 gene (coordinates 300621 302187, GenBank accession no. <br />                                                AE003436) PCR amplified using following PCR primers: EcoRI-5&#8242;-BTD (5&#8242;- <br />                                                CGCGAATTCGCCGCCTCTGGTGATTCAATCC-3&#8242;) BglII-3&#8242;-BTD (5&#8242;-CGCAGATC <br />                                                TGTCAGTGCTCTTCAGCAGTTCG-3&#8242;) generate EcoRI-BTD-BglII cassette; and <br />                                                XbaI-5&#8242;-BTD (5&#8242;-TTGTCTTGTCTAGATTATTCGCCGCC-3&#8242;) XhoI-3&#8242;-BTD (5&#8242;- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                CGCCTCGAGTGTCAGTG CTCTTCAGCAGTTCG-3&#8242;) generate XbaI-BTD-XhoI <br />                                                cassette. Both EcoRI-BTD-BglII XbaI-BTD-XhoI cassettes inserted the <br />                                                pUAST-sp vector described generate plasmid denoted pUAST-IRsp-BTD (19). <br />  <br />                                                Germline transformations pUAST-IRsp-HCS pUAST-IRsp-BTD conducted <br />                                                using P elements described (20). In transformed flies, transcription siRNA driven by <br />                                                Gal4. Hence, experiments described conducted mating pUAST-IRsp-HCS <br />                                                pUAST-IRsp-BTD transgenic virgin female flies male Actin5C-Gal4 driver lines. Flies <br />                                                straight wings produce siRNA selected experimentation. As control, <br />                                                used F1 crosses wild-type (y w) virgin females Actin5C-Gal4 male drivers (denoted <br />                                                Actin5C-Gal4 control flies). The following additional controls used some <br />                                                experiments: Act5C-Gal4 driver flies; y w flies; pUAST-IRsp-HCS pUAST-IRsp- <br />                                                BTD flies crossed Act5C-Gal4 driver. <br />  <br />  <br />  <br />                                                       J Nutr. Author manuscript; available PMC 2007 November 1. <br />                             Camporeale et al.                                                                                            Page 4 <br />  <br />  <br />                               Drosophila husbandry <br />                                                Flies housed 25&#186;C standard cornmeal diet transferred fresh tubes every <br />                                                72 h unless noted otherwise. Flies kept 12 h light/dark cycle. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Abundance mRNA coding HCS BTD <br />                                                Total RNA isolated 100 flies TriZol reagent according manufacturer&#8217;s <br />                                                instructions (Invitrogen, Carlsbad, CA). Genomic DNA removed enzymatically using <br />                                                TURBO DNA-free&#8482; (Ambion, Austin, TX). Briefly, cDNA synthesized using the <br />                                                ImProm-II reverse transcriptase Promega (Madison, USA). Primers HCS <br />                                                (forward primer = 5&#8242;-AGTATTGGAATTGGAGAATGC-3&#8242;; reverse primer = 5&#8242;- <br />                                                AACTTACATATTGATGGGAACC-3&#8242;) BTD (forward primer = 5&#8242;- <br />                                                CTACTACACCGATATACCCAGCAC-3&#8242;; reverse primer = 5&#8242;- <br />                                                CGCTGATGAAACAGTTGAGGAC-3&#8242;) designed using Beacon designer 4.0 <br />                                                software (Premier Biosoft International, Palo Alto, CA). Real-time PCR performed using <br />                                                iCycler IQ&#8482; multicolor real-time detection (Bio-Rad, Palo Alto, CA). Histone H4 <br />                                                (GenBank accession number NM_165383) used reference gene (forward primer = 5&#8242;- <br />                                                GTGCTGCGTGATAACATC-3&#8242;; reverse primer = 5&#8242;-TGGCTGTAACTGTCTTCC-3&#8242;). <br />  <br />                               Biotinylated carboxylases <br />                                                The abundance biotinylated carboxylases activity PCC quantified markers <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                HCS deficiency transgenic flies. Briefly, 50 flies homogenized described (17) <br />                                                proteins (~ 100 &#956;g) resolved using 4&#8211;8% Tris-acetate gels (Invitrogen). Transblots <br />                                                probed streptavidin peroxidase (21). Band intensities quantified gel <br />                                                densitometry (22). Activity PCC fly homogenates quantified described (17). <br />  <br />                               Biotinidase activity <br />                                                Whole fly homogenates prepared described above; enzyme activity homogenates <br />                                                quantified monitoring BTD-mediated hydrolysis biotinyl-p-aminobenzoic acid <br />                                                colorimetric assay (2). <br />  <br />                               HCS abundance <br />                                                Abundance HCS fly homogenates quantified Western blot analysis using <br />                                                rabbit anti-human HCS (23); antibody cross-reacts HCS Drosophila. Proteins <br />                                                resolved using 4&#8211;8% Bis-Tris gels (Invitrogen), transblots probed anti- <br />                                                HCS serum (20,000-fold dilution) goat anti-rabbit IgG peroxidase conjugate (Sigma, St. <br />                                                Louis, MO; 10 ng/mL). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Biotinylated histones <br />                                                Histone extracts fly homogenates prepared described (24) minor <br />                                                modifications: 5 mol/L HCl added fly homogenates produce final concentration of <br />                                                0.25 mol/L. Histones extracted overnight (4&#186;C); insoluble proteins removed by <br />                                                centrifugation histones supernatant precipitated trichloroacetic acid (1.2 <br />                                                mol/L final concentration). Histones washed acetone dissolved 8 M urea. <br />                                                Histones resolved gel electrophoresis (6) probed using streptavidin- <br />                                                peroxidase (2) antisera (250-fold dilution) K9-biotinylated histone H3 (K9BioH3) and <br />                                                K18-biotinylated histone H3 (K18BioH3) (5). Histone loading equalized based protein <br />                                                assays, densitometric quantitation gels stained Commassie blue Western blot <br />                                                analysis using antibody histone H3 C-terminal tail. <br />  <br />  <br />  <br />  <br />                                                      J Nutr. Author manuscript; available PMC 2007 November 1. <br />                             Camporeale et al.                                                                                               Page 5 <br />  <br />  <br />                               DNA microarray <br />                                                RNA 50 flies extracted 1 mL TriZol reagent described above. RNA was <br />                                                purified using RNeasy mini kit (Qiagen, <span id='am-83' about='obo:NCBITaxon_195854' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_8087 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_28738 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_7898 obo:NCBITaxon_195853 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-84'  ></span><span id='am-85' property="rdfs:label" content="Valencia" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Valencia</span>, CA) hybridized Drosophila <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                melanogaster Genome 2.0 Array (Affymetrix, Santa Clara, CA) Genomics Core <br />                                                Research Facility, University Nebraska-Lincoln (25). GeneChip operating software 1.4 <br />                                                (Affymetrix) used normalization analysis microarray data. <br />  <br />                               Lifespan <br />                                                The lifespan flies determined described previously minor modifications (17). <br />                                                Briefly, newly eclosed female male flies housed separately groups 20 flies/tube <br />                                                (n = 10 tubes treatment group). Dead flies counted removed daily. <br />  <br />                               Temperature stress <br />                                                Resistance flies cold heat stress determined described previously minor <br />                                                modifications (17). In cold stress experiments, male female flies housed separately <br />                                                4&#176;C 16 h; timed intervals flies transferred room temperature surviving <br />                                                flies counted 24 h. At time point tubes (25 flies each) collected (n=4). <br />                                                For heat stress experiments, tubes (25 flies each) kept 34&#176;C 9 h; dead flies <br />                                                counted 0.5 h. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Statistics <br />                                                Homogeneity variances groups tested using Bartlett&#8217;s test (26). If variances were <br />                                                heterogeneous, data log-transformed statistical analysis. Significance of <br />                                                differences groups tested one-way ANOVA. Fisher&#8217;s Protected Least <br />                                                Significant Difference procedure used posthoc testing (26). Effects treatment over <br />                                                time (e.g., resistance heat stress) analyzed repeated measures ANOVA. If effects <br />                                                treatment time statistically significant, treatment effects individual time points <br />                                                assessed one-way ANOVA Fisher&#8217;s Protected Least Significant Difference <br />                                                procedure individual time points. StatView 5.0.1 (SAS Institute; Cary, NC) used to <br />                                                perform calculations. Differences considered significant P &lt; 0.05. Data expressed <br />                                                means &#177; SD. <br />  <br />                               RESULTS <br />                               HCS chromosomal protein Drosophila <br />                                                Previous studies showed BTD HCS nuclear cytoplasmic <br />                                                fractions (23,27). To test HCS bound chromosomes, immunostained fixed <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                giant polytene chromosomes Drosophila instar larvae antibody HCS. HCS <br />                                                broadly distributed euchromatic arms chromosomes, particular <br />                                                concentrations certain bands, puffs (sites active transcription) telomeres <br />                                                (Fig. 1). Pre-immune serum used concentration produced significant staining <br />                                                polytene chromosomes. This result supports model HCS plays direct role in <br />                                                chromatin remodeling histone biotinylation specific chromosomal loci. <br />  <br />                               Expression HCS BTD RNAi knockdown flies <br />                                                Expression HCS BTD RNAi transformed flies caused substantial decrease the <br />                                                abundance HCS BTD, respectively. The abundance mRNA encoding HCS BTD <br />                                                smaller knockdown flies compared Act5C-Gal4 control flies (Fig. 2A). The following <br />                                                additional controls tested produced amounts HCS BTD mRNA similar those <br />                                                observed Act5C-Gal4 control flies: Act5C-Gal4 driver flies, y w flies, pUAST-IRsp- <br />                                                HCS pUAST-IRsp-BTD flies crossed Act5C-Gal4 driver (data shown). <br />  <br />                                                      J Nutr. Author manuscript; available PMC 2007 November 1. <br />                             Camporeale et al.                                                                                             Page 6 <br />  <br />  <br />                                                HCS protein barely detectable HCS-RNAi flies compared BTD-RNAi Act5C- <br />                                                Gal4 control flies; &#946;-actin used control affected HCS knockdown (Fig. <br />                                                2B). BTD activity homogenates BTD-RNAi flies decreased compared Act5C- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Gal4 control flies; BTD activity unaffected HCS-RNAi flies (Fig. 2C). <br />  <br />                               Biotin-dependent carboxylases <br />                                                Biotinylation carboxylases decreased significantly HCS-RNAi BTD-RNAi flies <br />                                                compared Act5C-Gal4 controls, judged streptavidin blotting (Fig. 3A) gel <br />                                                densitometric analysis (Fig. 3B). 3-Methylcrotonyl-CoA carboxylase PCC co-migrated as <br />                                                single band considered analysis gel densitometry; acetyl-CoA <br />                                                carboxylase barely detectable treatment groups considered <br />                                                analysis gel densitometry. Decreased biotinylation carboxylases HCS- BTD- <br />                                                deficient flies associated decreased PCC activity compared Act5C-Gal4 controls <br />                                                (Figure 3C). The abundance biotinylated carboxylases PCC activity similar in <br />                                                homogenates Act5C-Gal4 control flies various controls Act5C-Gal4 <br />                                                driver stock y w flies (data shown). <br />  <br />                                                Decreased biotinylation carboxylases HCS-RNAi BTD-RNAi flies likely mediated <br />                                                distinct mechanisms. In HCS-RNAi flies, decreased biotinylation carboxylases <br />                                                mediated decreased biotinyl ligase activity. In BTD-RNAi flies, biotinidase deficiency <br />                                                leads biotin deficiency known individuals affected inborn error (28). The <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                mechanism speculated increased urinary excretion biocytin (the biotinyl lysine <br />                                                remnant proteolytic breakdown biotinylated proteins) failure efficiently digest and <br />                                                absorb biotin covalently bound dietary proteins. <br />  <br />                               Biotinylation histones <br />                                                Previous studies provided evidence gender specificity histone biotinylation flies <br />                                                (17). Hence, investigated effects HCS BTD knockdown histone biotinylation <br />                                                separately males females. In males, HCS BTD deficiency caused decrease <br />                                                histone biotinylation. However, effects pronounced HCS deficiency for <br />                                                BTD deficiency. When biotinylated histones probed streptavidin-peroxidase a <br />                                                general probe biotin, signal intensity substantially decreased HCS-RNAi flies <br />                                                moderately decreased BTD-RNAi flies compared Act5C-Gal4 controls (Fig. <br />                                                4A). Biotinylated histone H4 barely detectable transgenic flies controls, <br />                                                consistent previous studies laboratory (17). An <span id='am-108' about='obo:NCBITaxon_32644' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-109' property="oboInOwl:hasRelatedSynonym" content="sonstige nucleic acid" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasRelatedSynonym" content="unspecified" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasRelatedSynonym" content="miscellaneous nucleic acid" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasRelatedSynonym" content="not shown" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasRelatedSynonym" content="other" datatype="xsd:string"></span><span id='am-115' property="rdfs:label" content="unidentified" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasRelatedSynonym" content="unknwon" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasRelatedSynonym" content="none" datatype="xsd:string"></span><span id='am-118'  ></span><span id='am-119' property="oboInOwl:hasRelatedSynonym" content="unknown organism" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasRelatedSynonym" content="unclassified sequence" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasRelatedSynonym" content="not specified" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasRelatedSynonym" content="unknown" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasRelatedSynonym" content="unidentified organism" datatype="xsd:string"></span>unidentified</span> biotinylated protein <br />                                                migrated histones H1 H3; protein denoted &#8220;unknown&#65533;? Fig. 4A. Based <br />                                                size acid solubility protein, speculate band corresponds an <br />                                                unidentified <span id='am-54' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> biotinylated histone histone variant. In contrast males, HCS and <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                BTD deficiency minor effects biotinylation histones female flies (Fig. 4A). <br />  <br />                                                Next, determined HCS BTD deficiency effects biotinylation of <br />                                                specific lysine residues. K9BioH3 K18BioH3 substantially decreased male HCS- <br />                                                RNAi flies, moderately decreased male BTD-RNAi flies compared Act5C- <br />                                                Gal4 controls (Fig. 4B 4C). In female flies, abundance K9BioH3 K18BioH3 <br />                                                decreased moderately HCS-RNAi flies, decrease detectable BTD- <br />                                                RNAi flies compared Act5C-Gal4 controls (Fig. 4B 4C). <br />  <br />                                                Theoretically, small magnitude effects HCS BTD deficiency females compared <br />                                                males caused inefficient knockdown enzymes females. We <br />                                                sought exclude possibility quantifying abundance biotinylated carboxylases <br />                                                separately males females. The percent decrease biotinylated carboxylases was <br />                                                similar males females. For example, activity biotinylated PC decreased about <br />  <br />  <br />                                                      J Nutr. Author manuscript; available PMC 2007 November 1. <br />                             Camporeale et al.                                                                                                 Page 7 <br />  <br />  <br />                                                90% male female HCS-deficient flies compared Act5C-Gal4 controls. Likewise, <br />                                                abundance biotinylated PC decreased 50% male female BTD-deficient <br />                                                flies compared controls (data shown). These data suggest quantitatively minor <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                decrease biotinylated histones female HCS-RNAi BTD-RNAi flies compared with <br />                                                males caused insufficient knockdown HCS BTD. <br />  <br />                               Gene expression affected HCS deficiency <br />                                                We used expression profiling knockdown flies look evidence patterns gene <br />                                                expression altered response changes chromatin modification. As model, we <br />                                                selected treatment group exhibits greatest alteration histone biotinylation the <br />                                                experiments: HCS-deficient male flies. Gene expression profiles flies and <br />                                                Act5C-Gal4 controls obtained using DNA microarrays. The expression 77 genes <br />                                                decreased 50% HCS-RNAi flies compared Act5C-Gal4 controls, whereas <br />                                                expression 124 genes increased 100% compared controls (online <br />                                                supplemental Table S1). We observed clusters HCS-dependent genes, based on <br />                                                biological process molecular function. For example, HCS deficiency associated with <br />                                                substantially increased expression genes play roles signal transduction, transport, and <br />                                                cell death, associated substantially decreased expression genes play roles in <br />                                                defense response (Table 1). <br />  <br />                               Knockdown HCS BTD cause reduced lifespan <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Low abundance HCS BTD associated decreased lifespan compared Act5C- <br />                                                Gal4 controls (Table 2). The relative decreases lifespan similar 75%, 50%, 25%, <br />                                                5% survival times genders. Note HCS deficiency pronounced effect <br />                                                lifespan BTD deficiency. <br />  <br />                               Knockdowns HCS BTD caused increased sensitivity heat stress <br />                                                Knockdown HCS BTD associated decreased survival male flies exposed to <br />                                                heat stress. Consistent sex-specific differences histone biotinylation, effects of <br />                                                knockdowns stress resistance greater male female flies, greater in <br />                                                HCS-deficient flies BTD-deficient flies. For example, 50% survival time 4.6 <br />                                                &#177; 0.3 h male Act5C-Gal4 flies, 3.1 &#177; 0.3 h 4.3 &#177; 0.4 h male HCS-RNAi and <br />                                                BTD-RNAi flies, respectively (Fig. 5A). In female flies, HCS deficiency BTD <br />                                                deficiency increased susceptibility heat stress (Fig. 5B); 50% survival time 5.1 <br />                                                &#177; 0.3 h female Act5C-Gal4 flies, 4.3 &#177; 0.1 h female HCS-RNAi flies, 5.5 &#177; 0.3 h <br />                                                female BTD-RNAi flies (Fig. 5B). HCS BTD deficiency specifically decreased heat <br />                                                tolerance opposed globally decreasing temperature tolerance. Consistent notion, <br />                                                survival flies exposed cold temperatures (4&#186;C 16 hr) did depend HCS <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                BTD (data shown). <br />  <br />                                                Finally, microarray analysis used identify heat-responsive genes affected by <br />                                                HCS deficiency. Heat stress altered expression 398 genes Act5C-Gal4 flies. In addition <br />                                                identified 285 heat-responsive genes uniquely affected HCS-deficient flies. <br />                                                Specifically, 167 heat-responsive genes down-regulated 50% HCS-deficient <br />                                                flies, 118 heat-responsive genes up-regulated 100% HCS-deficient flies <br />                                                (online supplemental Table S2). Heat-responsive, HCS-dependent genes clustered based <br />                                                biological process molecular function. HCS deficiency associated substantial <br />                                                increase expression genes play role proteolysis, substantial decrease <br />                                                genes play role chitin metabolism cuticle structure (Table 3). <br />  <br />  <br />  <br />  <br />                                                      J Nutr. Author manuscript; available PMC 2007 November 1. <br />                             Camporeale et al.                                                                                               Page 8 <br />  <br />  <br />                               DISCUSSION <br />                                                Biotinylation histones HCS BTD hypothesized participate heterochromatin <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                structure, gene silencing, cellular response DNA damage. The phenotypes associated <br />                                                HCS BTD deficiency humans extensively described (29). While <br />                                                deficiency enzyme viable, bewildering array seemingly disparate <br />                                                symptoms associated conditions. In present study, sought new insights into <br />                                                roles histone biotinylation gene function chromatin structure, characterize <br />                                                phenotypes associated HCS BTD deficiency Drosophila melanogaster. The <br />                                                demonstration HCS chromosomal protein Drosophila supports inference that <br />                                                HCS directly affects histone biotinylation chromatin. <br />  <br />                                                Expression HCS BTD dramatically knocked RNAi, judged the <br />                                                abundance mRNA coding HCS BTD, levels HCS protein biotinylated <br />                                                carboxylases, activity BTD. HCS deficiency associated global decrease in <br />                                                histone biotinylation decrease specific biotinylation sites histone H3. The <br />                                                reduction histone biotinylation seen HCS knockdown flies consistent finding <br />                                                HCS chromosomal protein, suggests effect direct. Effects HCS <br />                                                deficiency histone biotinylation modest females males. Generally, <br />                                                effects BTD deficiency histone biotinylation smaller effects observed for <br />                                                HCS deficiency. However, fact knockdowns significant effects demonstrates <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                proteins non-redundant activity vivo. <br />  <br />                                                Both HCS BTD deficiency decrease histone biotinylation greater extent males than <br />                                                females. The reason gender specificity unknown. We excluded possibility of <br />                                                insufficient knockdown HCS BTD females demonstrating biotinylated <br />                                                carboxylases decreased similar level transgenic flies genders. Why, then, <br />                                                histone biotinylation decreased greater extent males females? One possibility <br />                                                nuclear translocation biotin efficient females males, or <br />                                                turnover biotinylated histones greater males females. Another <br />                                                possibility Y chromosome, male-specific entirely heterochromatic <br />                                                somatic tissue, represents major target histone biotinylation Drosophila; reduction <br />                                                HCS BTD expected disproportionate effect males. <br />                                                Finally, exclude possibility deficiency of histone biotinyl <br />                                                transferases compensated biotinyl transferase females. We sought to <br />                                                investigate possibility creating HCS BTD double knockdowns. However, these <br />                                                flies non-viable. <br />  <br />                                                Previous studies provided evidence roles histone biotinylation cell proliferation (2, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                7), DNA repair (8,30), gene silencing (Camporeale, G., Oommen, A. M., Griffin, J. B., <br />                                                Sarath, G. &amp; Zempleni, J., unpublished data). Consistent biological significance of <br />                                                processes, previous study suggested biotin deficiency decreases lifespan and <br />                                                fertility flies (17). However, previous study clear link biotin deficiency and <br />                                                lifespan histone biotinylation evident. Here, time provide evidence <br />                                                reduced biotinylation histones HCS-deficient flies linked phenotypes as <br />                                                decreased lifespan heat tolerance. Also provide evidence alterations chromatin <br />                                                structure decreased biotinylation histones associated unique patterns gene <br />                                                expression. Gene expression analysis indicates HCS deficiency alters expression of <br />                                                201 genes. We identified 285 heat-responsive genes depend HCS normal <br />                                                expression. Currently, unknown genes mediate increased heat <br />                                                susceptibility HCS-deficient flies. <br />  <br />  <br />  <br />  <br />                                                      J Nutr. Author manuscript; available PMC 2007 November 1. <br />                             Camporeale et al.                                                                                                           Page 9 <br />  <br />  <br />                                                A caveat findings HCS BTD deficiency decreases biotinylation both <br />                                                histones carboxylases. Theoretically, effects described to <br />                                                decreased biotinylation carboxylases histones. To address uncertainty we <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                currently investigating genotypes phenotypes flies individual carboxylases <br />                                                knocked down. These ongoing studies provided evidence knockdown pyruvate <br />                                                carboxylase 3-methylcrotonyl-CoA carboxylase associated increased <br />                                                susceptibility heat stress (Camporeale, G., Eissenberg, J. C. &amp; Zempleni, J., unpublished <br />                                                data). Based ongoing studies hypothesize increased heat susceptibility <br />                                                observed HCS-deficient flies direct result changes histone biotinylation and <br />                                                chromatin structure, result secondary changes reduced activities <br />                                                biotin-dependent enzymes. <br />  <br />                                                The hypothesis effects HCS BTD deficiency act level chromatin <br />                                                modification offers novel explanation diverse apparently unrelated deficits seen <br />                                                HCS BTD deficiency patients. Rather connected metabolic <br />                                                pathways, classical vitamin deficiencies, symptoms HCS BTD deficiency may <br />                                                reflect, significant degree, misregulation genes connected use histone <br />                                                biotinylation regulatory mechanism. The development Drosophila models HCS and <br />                                                BTD deficiency, combined favorable cytology genetics Drosophila, help <br />                                                define role histone biotinylation gene regulation disease. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               References <br />                                                1. Camporeale G, Zempleni J. Biotin. In: Bowman BA, Russell RM, editors. Present Knowledge in <br />                                                   Nutrition. 9th ed. Washington, D.C.: International Life Sciences Institute; 2006 (in press). <br />                                                2. Stanley JS, Griffin JB, Zempleni J. Biotinylation histones human cells: effects cell proliferation. <br />                                                   Eur J Biochem 2001;268:5424&#8211;9. [PubMed: 11606205] <br />                                                3. Hymes J, Fleischhauer K, Wolf B. Biotinylation histones human serum biotinidase: assessment <br />                                                   biotinyl-transferase activity sera normal individuals children biotinidase <br />                                                   deficiency. Biochem Mol Med 1995;56:76&#8211;83. [PubMed: 8593541] <br />                                                4. Camporeale G, Chew YC, Kueh A, Sarath G, Zempleni J. Use synthetic peptides identifying <br />                                                   biotinylation sites human histones. In: McMahon RJ, editor. Avidin-Biotin Technology Life <br />                                                   Sciences. Totowa, N.J.: Humana Press; 2005. <br />                                                5. Kobza K, Camporeale G, Rueckert B, Kueh A, Griffin JB, Sarath G, Zempleni J. K4, K9, K18 in <br />                                                   human histone H3 targets biotinylation biotinidase. FEBS J 2005;272:4249&#8211;59. [PubMed: <br />                                                   16098205] <br />                                                6. Camporeale G, Shubert EE, Sarath G, Cerny R, Zempleni J. K8 K12 biotinylated human <br />                                                   histone H4. Eur J Biochem 2004;271:2257&#8211;63. [PubMed: 15153116] <br />                                                7. Kothapalli N, Zempleni J. Biotinylation histones depends cell cycle NCI-H69 small cell <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   lung <span id='am-37' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-41'  ></span>cancer</span> cells. FASEB J 2005;19:A55.[abstract] <br />                                                8. Kothapalli N, Sarath G, Zempleni J. Biotinylation K12 histone H4 decreases response DNA <br />                                                   double strand breaks human JAr choriocarcinoma cells. J Nutr 2005;135:2337&#8211;42. [PubMed: <br />                                                   16177192] <br />                                                9. Narang MA, Dumas R, Ayer LM, Gravel RA. Reduced histone biotinylation multiple carboxylase <br />                                                   deficiency patients: nuclear role holocarboxylase synthetase. Hum Mol Genet 2004;13:15&#8211;23. <br />                                                   [PubMed: 14613969] <br />                                                10. Jenuwein T, Allis CD. Translating histone code. Science 2001;293:1074&#8211;80. [PubMed: 11498575] <br />                                                11. Ballard TD, Wolff J, Griffin JB, Stanley JS, Calcar Sv, Zempleni J. Biotinidase catalyzes <br />                                                     debiotinylation histones. Eur J Nutr 2002;41:78&#8211;84. [PubMed: 12083317] <br />                                                12. Zempleni J. Uptake, localization, noncarboxylase roles biotin. Annu Rev Nutr 2005;25:175&#8211; <br />                                                     96. [PubMed: 16011464] <br />                                                13. Eisenberg MA, Prakash O, Hsiung SC. Purification properties biotin repressor. A <br />                                                     bifunctional protein. J Biol Chem 1982;257:15167&#8211;73. [PubMed: 6129246] <br />  <br />  <br />  <br />                                                       J Nutr. Author manuscript; available PMC 2007 November 1. <br />                             Camporeale et al.                                                                                                   Page 10 <br />  <br />  <br />                                                14. Beckett D. Energetic methods study bifunctional biotin operon repressor. Methods Enzymol <br />                                                    1998;295:424&#8211;50. [PubMed: 9750231] <br />                                                15. Leon-Del-Rio A, Leclerc D, Akerman B, Wakamatsu N, Gravel RA. Isolation cDNA encoding <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                    human holocarboxylase synthetase functional complementation biotin auxotroph of <br />                                                    <span id='am-60' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-61' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-71'  ></span><span id='am-72' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span>Escherichia coli</span>. Proc Natl Acad Sci U S A 1995;92:4626&#8211;30. [PubMed: 7753853] <br />                                                16. Dupuis L, Campeau E, Leclerc D, Gravel RA. Mechanism biotin responsiveness biotin- <br />                                                    responsive multiple carboxylase deficiency. Molec Genet Metabol 1999;66:80&#8211;90. <br />                                                17. Landenberger A, Kabil H, Harshman LG, Zempleni J. Biotin deficiency decreases life span and <br />                                                    fertility increases stress resistance Drosophila melanogaster. J Nutr Biochem 2004;15:591&#8211; <br />                                                    600. [PubMed: 15542350] <br />                                                18. Lis JT, Mason P, Peng J, Price DH, Werner J. P-TEFb kinase recruitment function heat shock <br />                                                    loci. Genes Dev 2000;14:792&#8211;803. [PubMed: 10766736] <br />                                                19. Giordano E, Rendina R, Peluso I, <span id='am-93' about='obo:NCBITaxon_176679' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_34481 obo:NCBITaxon_1264859 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4857 obo:NCBITaxon_2759 obo:NCBITaxon_1399770'><span id='am-94' property="rdfs:label" content="Furia" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-96'  ></span><span id='am-97' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Furia</span> M. RNAi triggered symmetrically transcribed transgenes <br />                                                    Drosophila melanogaster. Genetics 2002;160:637&#8211;48. [PubMed: 11861567] <br />                                                20. Spradling AC. P element-mediated transformation. In: Roberts DB, editor. Drosophila, practical <br />                                                    approach. Oxford: IRL Press; 1986. p. 175&#8211;97. <br />                                                21. Manthey KC, Griffin JB, Zempleni J. Biotin supply affects expression biotin transporters, <br />                                                    biotinylation carboxylases, metabolism interleukin-2 Jurkat cells. J Nutr 2002;132:887&#8211; <br />                                                    92. [PubMed: 11983808] <br />                                                22. Griffin JB, Rodriguez-Melendez R, Zempleni J. The nuclear abundance transcription factors Sp1 <br />                                                    Sp3 depends biotin Jurkat cells. J Nutr 2003;133:3409&#8211;15. [PubMed: 14608051] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                23. Chew YC, Camporeale G, Kothapalli N, Sarath G, Zempleni J. Lysine residues N- C-terminal <br />                                                    regions human histone H2A targets biotinylation biotinidase. J Nutr Biochem <br />                                                    2006;17:225&#8211;33. [PubMed: 16109483] <br />                                                24. Kang HL, Benzer S, Min KT. Life extension Drosophila feeding drug. Proc Natl Acad Sci U <br />                                                    S A 2002;99:838&#8211;43. [PubMed: 11792861] <br />                                                25. Wiedmann S, Rodriguez-Melendez R, Ortega-Cuellar D, Zempleni J. Clusters biotin-responsive <br />                                                    genes human peripheral blood mononuclear cells. J Nutr Biochem 2004;15:433&#8211;9. [PubMed: <br />                                                    15219929] <br />                                                26. SAS Institute. StatView Reference. 3rd ed. Cary, NC: SAS Publishing; 1999. <br />                                                27. Pispa J. Animal biotinidase. Ann Med Exp Biol Fenniae 1965;43:4&#8211;39. <br />                                                28. Wolf B, Grier RE, Allen RJ, Goodman SI, Kien CL. Biotinidase deficiency: An enzymatic defect in <br />                                                    late-onset multiple carboxylase deficiency. Clin Chim Acta 1983;131:273&#8211;81. [PubMed: 6883721] <br />                                                29. Wolf B. Disorders biotin metabolism: Treatable neurological syndromes. In: Rosenberg RN, <br />                                                    Prusiner BS, Mauro SD, Barchi RL, Kunkel LM, editors. The Molecular Genetic Basis of <br />                                                    Neurological Disease. Stoneham, MA: Butterworth; 1992. p. 569&#8211;81. <br />                                                30. Peters DM, Griffin JB, Stanley JS, Beck MM, Zempleni J. Exposure UV light causes increased <br />                                                    biotinylation histones Jurkat cells. Am J Physiol Cell Physiol 2002;283:C878&#8211;C84. [PubMed: <br />                                                    12176744] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                       J Nutr. Author manuscript; available PMC 2007 November 1. <br />                             Camporeale et al.                                                                                               Page 11 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                FIGURE 1. <br />                                                Holocarboxylase synthetase (HCS) binds distinct chromosomal sites wild-type <br />                                                Drosophila melanogaster. Panel A: Immunostaining fixed polytene chromosomes anti- <br />                                                HCS serum. Panel B: Phase contrast image chromosomes A. Panel C: Immunostaining <br />                                                fixed polytene chromosomes pre-immune serum. Panel D: Phase contrast image of <br />                                                chromosomes C. Legend: filled arrowheads = puffs stain HCS; open arrowheads = <br />                                                puffs stain HCS; line = non-puffed site stains HCS; t = staining telomere; <br />                                                N = nucleolus; cc = heterochromatic chromocenter; X = X chromosome; 2L 2R = left and <br />                                                right arms chromosome 2; 3L 3R = left right arms chromosome 3. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                      J Nutr. Author manuscript; available PMC 2007 November 1. <br />                             Camporeale et al.                                                                                           Page 12 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                FIGURE 2. <br />                                                RNAi HCS biotinidase (BTD) decreases expression HCS BTD, respectively, <br />                                                Drosophila melanogaster compared control flies. Panel A: mRNA levels HCS and <br />                                                BTD HCS-RNAi flies, BTD-RNAi flies, Act5C-Gal4 control flies. Panel B: <br />                                                Quantification HCS protein Western blot analysis; &#946;-actin used loading control. <br />                                                Panel C: Quantification BTD activity fly homogenates. PABA = para-amino benzoic <br />                                                acid. a,bColumns sharing letter significantly different (P &lt; 0.05). Values are <br />                                                means &#177; SD (n = 3). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                      J Nutr. Author manuscript; available PMC 2007 November 1. <br />                             Camporeale et al.                                                                                           Page 13 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                FIGURE 3. <br />                                                RNAi knockdown HCS BTD associated decreased biotinylation carboxylases <br />                                                Drosophila melanogaster. Panel A: Abundance biotinylated carboxylases HCS- <br />                                                deficient (HCS-RNAi) BTD-deficient (BTD-RNAi) flies compared Act5C-Gal4 <br />                                                control flies. PCC = Propionyl-CoA carboxylase. Panel B: Gel densitometric quantification of <br />                                                pyruvate carboxylase (PC). Panel C: PCC activity fly homogenates. a,b,cColumns sharing <br />                                                letter significantly different groups (P &lt; 0.05). Values means &#177; SD <br />                                                (n = 3). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                      J Nutr. Author manuscript; available PMC 2007 November 1. <br />                             Camporeale et al.                                                                                      Page 14 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                FIGURE 4. <br />                                                RNAi knockdown HCS BTD associated decreased biotinylation histones in <br />                                                Drosophila melanogaster. Panel A: Abundance biotinylated histones HCS-RNAi and <br />                                                BTD-RNAi flies compared Act5C-Gal4 control flies, judged streptavidin blotting. <br />                                                Panel B: Transblots probed anti-K9BioH3. Panel C: Transblots probed anti- <br />                                                K18BioH3. Note lanes shown blot, <span id='am-57' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> presented <br />                                                changed order facilitate comparisons. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                      J Nutr. Author manuscript; available PMC 2007 November 1. <br />                             Camporeale et al.                                                                                        Page 15 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                FIGURE 5. <br />                                                Effects HCS BTD RNAi knockdown heat tolerance HCS-deficient (HCS-RNAi) <br />                                                BTD-deficient (BTD-RNAi) flies compared Act5C-Gal4 control flies. Panel A: HCS- <br />                                                RNAi BTD-RNAi male flies exhibited decreased heat tolerance compared control <br />                                                flies. Panel B: HCS deficiency BTD deficiency decreased heat tolerance female <br />                                                knockdown flies. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                      J Nutr. Author manuscript; available PMC 2007 November 1. <br />                                Camporeale et al.                                                                                              Page 16 <br />  <br />                                                                                         TABLE 1 <br />                               Select clusters genes increased decreased expression male HCS-deficient (HCS-RNAi) Drosophila <br />                               compared Act5C-Gal4 control flies <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                                                     Expression (HCS-RNAi vs. Act5C-Gal4) <br />  <br />                            Gene Ontology category                                                         Increased                        Decreased <br />  <br />                            Biological process <br />                            Signal transduction                                                                    7                               1 <br />                            Cell cycle                                                                             2                               0 <br />                            Cell organization                                                                      3                               0 <br />                            Cell death                                                                             3                               1 <br />                            Catabolism                                                                             6                               2 <br />                            Defense response                                                                       2                              10 <br />                            Transport                                                                             15                               2 <br />                            Nucleic acid metabolism                                                                2                               0 <br />                            Regulation transcription                                                           14                               2 <br />                            Proteolysis peptidolysis                                                          18                               6 <br />                            Morphogenesis                                                                          5                               1 <br />                            Molecular function <br />                            Protein binding                                                                        7                               0 <br />                            DNA/nucleotide binding                                                                18                               1 <br />                            Transcription factor binding                                                           8                               1 <br />                            Kinase                                                                                12                               4 <br />                            Transferase                                                                            3                              10 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                           J Nutr. Author manuscript; available PMC 2007 November 1. <br />                                  Camporeale et al.                                                                                                                          Page 17 <br />  <br />                                                                                                    TABLE 2 <br />                                 Lifespan HCS-deficient (HCS-RNAi) BTD-deficient (BTD-RNAi) male female Drosophila <br />                                 melanogaster compared Act5c-Gal4 control flies <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                                                           Days survival <br />  <br />                                                                               Male                                                               Female <br />  <br />                            % Survival               HCS-RNAi              BTD-RNAi                 Control              HCS-RNAi               BTD-RNAi                 Control <br />  <br />                            75                        17 &#177; 2.7a              23 &#177; 3.1b              25 &#177; 2.6c             37 &#177; 4.2a               43 &#177; 2.8b             50 &#177; 6.5c <br />                            50                        28 &#177; 2.0a              30 &#177; 2.6a              33 &#177; 2.0b             47 &#177; 3.2a               49 &#177; 3.1a             55 &#177; 2.4b <br />                            25                        32 &#177; 2.4a              35 &#177; 2.3b              41 &#177; 1.6c             57 &#177; 3.0a               56 &#177; 2.1a             63 &#177; 2.0b <br />                            5                         40 &#177; 1.9a              39 &#177; 2.4a              47 &#177; 2.0b             63 &#177; 1.8a               64 &#177; 1.8a             68 &#177; 1.6b <br />  <br />  <br />                            Values means &#177; SD (n = 10 vials containing 20 flies each). <br />                            a,b,c <br />                                 Values sharing superscript significantly different sex % survival category; P &lt; 0.05 determined <br />                            one-way ANOVA Fisher&#8217;s Protected Least Significant Difference procedure posthoc testing. Males compared females. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                  J Nutr. Author manuscript; available PMC 2007 November 1. <br />                                Camporeale et al.                                                                                      Page 18 <br />  <br />                                                                                            TABLE 3 <br />                               Select clusters heat-responsive genes upregulated downregulated HCS-deficient male flies <br />  <br />                            Gene Ontology category                                                         Upregulated          Downregulated <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Biological process <br />                            Signal transduction                                                                      6                        6 <br />                            Cell cycle                                                                               1                        2 <br />                            Cell proliferation                                                                       5                        3 <br />                            Proteolysis peptidolysis                                                            13                       12 <br />                            Chitin metabolism                                                                        2                        9 <br />                            Cell motility                                                                            0                        3 <br />                            Transport                                                                               12                       17 <br />                            Biosynthesis                                                                             7                        8 <br />                            Regulation transcription                                                             10                        3 <br />                            Molecular function <br />                            Protein binding activity                                                                 2                        8 <br />                            DNA/nucleotide binding activity                                                          8                        4 <br />                            Transcription factor activity                                                            9                        2 <br />                            Receptor activity                                                                        6                        9 <br />                            Cuticle structure                                                                        1                       10 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                              J Nutr. Author manuscript; available PMC 2007 November 1. <br />                                 Camporeale et al.                                                                                                                            Page 19 <br />  <br />                                                                                     SUPPLEMENTAL TABLE S1 <br />                                Genes increased decreased expression HCS-deficient (HCS-RNAi) male Drosophila compared with <br />                                Act5C-Gal4 control flies <br />  <br />                            Expression increased &gt;200% HCS-RNAi flies compared controls:1 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            defective chorion 1 (CG2175); lethal (3) malignant blood neoplasm (CG12755); held wings (CG10293); vitelline membrane 34Ca (CG9271); <br />                            pfam01151 (CG8534); vitelline membrane 26Ab (CG9046); RAB (CG32776); ventral veins lacking (CG10037); Rieske iron-sulfur protein (CG7361); <br />                            cytochrome P450-4e3 (CG4105); vitelline membrane 26Aa (CG9048); chorion protein 36 (CG1478); DUF229 (CG9168); chitin binding Peritrophin-A <br />                            (CG9307); <span id='am-1' about='obo:NCBITaxon_490857' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_41142 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_7055 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_75546 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_569629 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_41086 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="Sphaerocanthon" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="Sphaerocanthon Olsoufieff, 1947" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasAlternativeId" content="NCBITaxon:369563" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="Nanos" datatype="xsd:string"></span><span id='am-8'  ></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Nanos Westwood, 1837" datatype="xsd:string"></span>nanos</span> (CG5637); unknown genes (CG17042; CG5480; CG17298; CG14118; CG14934); unidentified transcripts (Affymetrix probe identifiers <br />                            1639811_at; 1635146_at; 1635036_at; 1624084_s_at). <br />                            Expression increased 100 200% HCS-RNAi flies compared controls: <br />                            cutlet (CG33122); <span id='am-10' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-11'  ></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> UAS construct Yu (CG18405); male fertility factor kl3 (AAG29546); oskar (CG10901); mitochondrial <br />                            carrier protein (CG18327); follicle cell protein 3C (CG18508); Kelch (CG17754); emp24/gp25L/p24 <span id='am-34' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-35' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> (CG33105); CHK (CG5126); widerborst <br />                            (CG5643); LeuB (CG7755); odorant-binding protein 83cd (CG15582); DUF233 (CG2650); transferrin 1 (CG6186); predicted Na+-dependent transporter <br />                            (CG9903); white (CAA53795); Nhe2 (CG9256); LRR_RI (CG5096); longest ORF (CG13606); Ccp84Aa (CG2360); Tryp_SPc (CG31267); larval serum <br />                            protein 1 (CG6821); misexpression suppressor KSR 4 (CG31274); AMP deaminase (CG32626); zf-C2H2 (CG10309); lysozyme P (CG9116); fat body <br />                            protein 1 (CG17285); shaking B (CG32508); LRR_RI (CG2471); tropomyosin 1 (CG4898); Tryp_SPc (CG11911); longitudinals lacking (CG12052); <br />                            yolkless (CG1372); peptidase S10 (CG32483); ACBP (CG8628); RH40749p (CG6131); nuclear factor I (CG2380); bicoid (CG1034); peptidase C48 <br />                            (CG11023); Rgk1 (CG9811); Ecdysone-induced protein 74EF (CG32180); His4 (CG31611); misexpression suppressor KSR 4 (CG31274); Myb- <br />                            interacting protein 130 (CG3480); fat body protein 2 (CG3763); LP19160p (CG13044); PycA (CG1516); Sec14p-like lipid-binding domain (CG1211); <br />                            glass multimer reporter construct Nolan (CG8556); pr-set7 (CG3307); RE12806p (CG17086); chorion protein 38 (CG11213); Usf (CG17592); zf- <br />                            C3HC4 (CG14435); Gid (CG4610); RE05031p (CG17475); IP03589p (CG14957); cAMP-dependent protein kinase 1 (CG4379); twin eyeless <br />                            (CG11186); IP05101p (CG32024); pasilla (CG8144); FReD (CG30281); Pleckstrin homology (PH) domain (CG32982); alkPPc (CG3290); female sterile <br />                            (1) homoerotic (CG2252); peptidase S9 (CG11034); checkpoint suppressor homologue (CG12690); brain tumor (CG10719); retrotransposon gag protein <br />                            (CG10102); SEC14 (CG33514); trypsin (CG18211); MICAL (CG33208); unknown genes (CG18348; CG13165; CG13043; CG30395; CG3805; <br />                            CG14406; CG12998; CG33204; CG32694; CG15263; CG13042; CG14752; CG31038); unidentified transcripts (Affymetrix probe identifiers 1634519_at; <br />                            1639606_at; 1639054_s_at; 1641367_at; 1624379_s_at; 1626205_s_at; 1635258_s_at; 1640955_s_at). <br />                            Expression decreased &gt;75% HCS-RNAi flies compared controls: <br />                            PGRP-SB2 (CG9697); dissatisfaction (CG9019); dopamine receptor 2 (CG18741); larval cuticle protein 1 (CG11650); chorion protein 18 (CG6517); <br />                            chorion protein 16 (CG6533); chromosome segregation ATPase (CG13083); chorion protein 19 (CG6524); IP04471p (CG30096); FReD (CG32496); <br />                            Sugar_tr (CG6640); unknown genes (CG32644; CG13084; CG18607; CG14632); unidentified transcripts (Affymetrix probe identifiers 1631178_at; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            1632150_at; 1628898_at). <br />                            Expression decreased 50 75% HCS-RNAi flies compared controls: <br />                            yolk protein 2 (CG2979); abhydro lipase (CG2772); strawberry notch (CG1903); yolk protein 3 (CG11129); trypsin-like serine protease (CG17404); <br />                            GH03623p (CG4784); trypsin-like serine protease (CG1304); yolk protein 1 (CG2985); SAK (CG7186); PSF1 (CG9187); cecropin C (CG1373); <br />                            LP22678p (CG17738); GH01676p (CG17352); serine protease 6 (CG2071); UDPGT (CG17322); Ugt86Dd (CG6633); trypsin-like serine protease <br />                            (CG18179); IGcam (CG15214); SSM4 (CG15059); galactose-specific C-type lectin (CG9976); Smc (CG17177); IP07781p (CG14315); attacin-A <br />                            (CG10146); cecropin B (CAA34845); DUF171 (CG12128); trypsin-like serine protease (CG17571); CBM_14 (CG13675); peritrophin-15a (CG17814); <br />                            turandot C (CG31508); (CG40485); pastrel (CG8588); cytochrome P450-6a8 (CG10248); unknown genes (CG31606; CG7296; CG12439; CG32602; <br />                            CG7567; CG11671; CG32510; CG13305); unidentified transcripts (Affymetrix probe identifiers 1632647_s_at; 1627513_at; 1628989_at; 1623579_at; <br />                            1626392_s_at; 1628534_at; 1632882_at; 1639231_at; 1624069_at; 1637692_at). <br />  <br />                            1 <br />                             Gene symbols provided parentheses; unidentified transcripts Affymetrix probe identifier substituted gene symbols. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                J Nutr. Author manuscript; available PMC 2007 November 1. <br />                                 Camporeale et al.                                                                                                                            Page 20 <br />  <br />                                                                                     SUPPLEMENTAL TABLE S2 <br />                                Heat-responsive genes upregulated downregulated HCS-deficient male flies <br />  <br />                            Expression increased &gt;200% HCS-RNAi flies compared controls:1 <br />                            Stellate 12D orphon (CG33236); yolk protein 2 (CG2979); yolk protein 3 (CG11129); female-specific independent transformer (CG17820); LipA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            (CG10383); LP21536p (CG14963); ANK (CG8679). <br />                            Expression increased 100 200% HCS-RNAi flies compared controls: <br />                            Jonah 66Ci (CG7118); black cells (CG5779); trypsin 29F (CG9564); Drosocin (CG10816); PEX11 (CG33474); emocyanin (CG8193); PAP2 wunen <br />                            (CG11425); takeout (CG11853); CBM14 (CG10725); SMP2 (CG8709); LP11437p (CG10462); Tryp SPc (CG8329); death associated molecule related <br />                            Mch2 (CG18188); FRQ1 (CG32103); astray (CG3705); Rho (CG12102); <span id='am-88' about='obo:NCBITaxon_652916' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6656 obo:NCBITaxon_6681 obo:NCBITaxon_6692 obo:NCBITaxon_6694 obo:NCBITaxon_1206794 obo:NCBITaxon_115581 obo:NCBITaxon_6657 obo:NCBITaxon_6682 obo:NCBITaxon_88770 obo:NCBITaxon_6683 obo:NCBITaxon_98089 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-90'  ></span><span id='am-91' property="rdfs:label" content="Thor" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Thor</span> (CG8846); adenosine 3 (CG31628); DltE (CG13833); Tryp SPc <br />                            (CG4927); LD43023p (CG8321); DUF727 (CG14505); Cyp4ac2 (CG17970); Jonah 99Fi (CG18030); imaginal disc growth factor 1 (CG4472); GH12638p <br />                            (CG3672); RE61294p (CG8112); suppressor cytokine signaling 36E (CG15154); serine pyruvate aminotransferase (CG3926); RH05653p(CG13116); <br />                            Sugar_tr (CG31106); metchnikowin (CG8175); BAH (CG32529); adenosine 2 (CG9127); sugarless (CG10072); naked cuticle (CG11614); att-ORFB <br />                            (CG33488); ia2 (CG31795); Delilah (CG5441); ANK (CG9121); PepO (CG14528); MA3 (CG10990); CLECT (CG8343); sterile (CG1441); SCP <br />                            (CG6628); lethal (2) essential life (CG4533); serpin (CG9460); Cyp12a5 (CG11821); PcbC (CG5346); GH08205p (CG3906); gamma- <br />                            glutamyltranspeptidase (CG17636); Cyp6a22 (CG10240); midline fasciclin (CG3359); Pal (CG5472); GNS1/SUR4 (CG33110); ornithine decarboxylase <br />                            1 (CG8721); Ecdysone-induced protein 75B (CG8127); DM8 (CG14456); Tryp_SPc (CG10587); pancreatic lipase like protein (CG6283); Atf-2 <br />                            (CG30420); hemolysin (CG33173); BTB/POZ (CG31160); Ecdysone-induced protein 75B (CG8127); bunched (CG5461); COG0842 (CG14559); PepO <br />                            (CG5527); scratch (CG1130); odorant-binding protein 8a (CG12665); raptor (CG4320); protein tyrosine phosphatase 99A (CG2005); haem peroxidase <br />                            (CG8913); adh_short (CG3842); paired (CG6716); adh_short (CG2064); martik (CG3361); Jun-related antigen (CG2275); deformed epidermal <br />                            autoregulatory factor-1 (CG8567); X box binding protein-1 (CG9415); Tryp_SPc (CG9672); DSPc (CG10089); Jonah 65Ai (CG10475); ab-hydrolase <br />                            associated lipase (CG5932); CLECT (CG12111); carbonic anhydrase alpha (CG18673); mitochondrial ribosomal protein L51 (CG15434); predicted <br />                            GTPase (CG8801); MADF (CG8281); chitin-binding domain type 2 (CG7874); heat shock construct Kidd (CG17342); unknown genes (CG1648; <br />                            CG13654; CG32185; CG12065; CG15784; CG4285; CG11686; CG13868; CG5597; CG12726; CG10433; CG13947; CG9776; CG31829; CG14499; <br />                            CG13946; CG4577; CG11112); unidentified transcripts (Affymetrix probe identifiers 1638496_at; 1625050_s_at). <br />                            Expression decreased &gt;75% HCS-RNAi flies compared controls: <br />                            four-jointed (CG10917); MdlB (CG10505); fat body protein 1 (CG17285); odorant binding protein (CG2650); GH05104p (CG13607); larval serum <br />                            protein 1 (CG6821); LIM domain (CG30179); LIM domain (CG30174); yellow-d2 (CG9891); unknown genes (CG13049; CG15080; CG8515; CG13043; <br />                            CG12998; CG31775). <br />                            Expression decreased 50 75% HCS-RNAi flies compared controls: <br />                            ACBP (CG8628); krotzkopf verkehrt (CG2666); RKIP (CG18594); peptidase M28 (CG30049); CBM14 (CG31973); Stretchin-Mlck (CG18255); <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            RH01578p (CG15006); larval serum protein 1 (CG6821); CBM14 (CG32499); vanin-like (CG32754); Zim2 (CG3777); MH2 (CG13183); sterile <br />                            (CG17562); sickle (CG13701); His1 (CG31617); MH2 (CG13188); CBM14 (CG13676); predicted trypsin-like serine protease (CG31265); LRRRI <br />                            (CG2471); glyco18 (CG1869); wrinkled (CG5123); PDZ signaling (CG14168); Tryp_SPc (CG17475); CBM14 (CG2989); cd00190 (CG4053); DUF233 <br />                            (CG11852); ecdysis triggering hormone (CG18105); BCM14 (CG17052); myosin light chain 2 (CG2184); smart00631 (CG14820); Gasp (CG10287); <br />                            Cyp4d14 (CG3540); ninaD (CG31783); cp309 (CG33957); Duf227 (CG11893); GILT (CG13822); IP07649p (CG13155); CLECT (CG4115); UDPGT <br />                            (CG5724); CD36 family (CG3829); peroxidase (CG3477); crystallin (CG16963); exportin 6 (CG3923); Orc1 (CG10667); TRP (CG11396); Kelch <br />                            (CG17754); DUF1193 (CG3631); Dimp (CG9850); LD12613p (CG1919); HL04910p (CG7759); mRpL52 (CG1577); SD18375p (CG3773); dachs <br />                            (CG10595); peritrophin-A (CG17058); DltE (CG31810); HL05804p (CG30437); formyl transferase (CG1750); LDLa (CG32209); sugar_tr (CG33281); <br />                            predicted GTPase (CG18528); arginase (CG18104); Obp83g (CG31558); SMC4 (CG18304); shark (CG18247); CO esterase (CG7529); aret (CG31762); <br />                            sugar_tr (CG31530); KU (CG3513); Cks85A (CG9790); longest ORF LD45181p (CG6912); msl-3 (CG8631); aminopeptidase N (CG31198); Pex2/Pex12 <br />                            (CG3639); alpha amylase (CG33138); Ice (CG7788); Tub84D (CG2512); peptidase S28 (CG18493); Jonah 44E (CG8579); Peptidase S28 RE36938p <br />                            (CG3734); SMC4 LD41224p (CG196); GH12942p (CG8179); trans-isoprenyl diphosphate synthase (CG31005); mRpL55 (CG14283); RAB GTPase <br />                            (CG4789); Niemann-Pick type C2 (CG31410); alanine dehydrogenase/PNT (CG7144); ribosome recycling factor (CG4447); DUF227 (CG31087); MOSC <br />                            (CG1665); AraJ (CG18281); pellucida (CG2467); RH50269p (CG6639); RH32892p (CG1674); branchless (CG4608); SD05384p CNH domain <br />                            (CG8060); Paramyosin (CG5939); GalNAc-T2 (CG6394); leucine-rich repeats (CG8561); RE03018p (CG13003); Glutathione S-transferase (CG4688); <br />                            eel-Fucolectin Tachylectin-4 (CG9095); HDc (CG11900); actinin (CG4376); Arabinose efflux permease (CG8051);  <br /> </body></html>