<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Letter <br />  <br /> Origins impact constraints evolution <br /> of gene families <br /> Boris E. Shakhnovich1,3 Eugene V. Koonin2 <br /> 1 <br />  Bioinformatics Program, Boston University, Boston, Massachusetts 02215, USA; 2National Center Biotechnology Information, <br /> National Institutes Health, Bethesda, Maryland 20894, USA <br />  <br />  <br />         Recent investigations high-throughput genomic phenomic <span id='am-161' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-162' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-163' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-164'  ></span><span id='am-165' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> uncovered variety significant but <br />         relatively weak correlations gene&#8217;s functional evolutionary characteristics. In particular, essential genes <br />         genes paralogs slight propensity evolve slowly nonessential genes singletons, <br />         respectively. However, given weakness multiplicity associations, biological relevance remains <br />         uncertain. Here, existence essential paralog used specific strong gauge of <br />         selection. We partition gene families genomes classes: include essential gene <br />         (E-families) essential genes (N-families). We weaker purifying selection causes N-families <br />         evolve dynamic regime higher rates duplicate fixation pseudogenization. Because genes <br />         E-families subject significantly stronger purifying selection N-families, survive longer and <br />         exhibit greater sequence divergence. Longer average survival time allows divergence upstream regulatory <br />         regions, resulting change transcriptional context paralogs E-families. These findings compatible <br />         differential division ancestral functions (subfunctionalization) emergence novel functions <br />         (neofunctionalization) prevalent modes evolution paralogs E-families opposed to <br />         pseudogenization (nonfunctionalization), typical fate paralogs N-families. Unlike other <br />         characteristics genes, essentiality, existence paralogs, expression level, membership E-<span id='am-61' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> or <br />         N-family strongly correlates level selection appears major determinant gene&#8217;s <br />         evolutionary fate. <br />         [Supplemental material available online www.genome.org.] <br />  <br /> The nature connections organismal molecular                           theory molecular evolution, remains open question. There <br /> evolution remains fundamental and, generally, unanswered                           substantial disagreement correlation an <br /> question. The relationship evolution gene                       artifact caused transitive relationships expression level <br /> organism characterized change fitness precipi-                      (Pal et al. 2001) abundance (Drummond et al. 2005). Although <br /> tated deletion mutation gene (Drake et al. 1998;                       recent, sophisticated statistical analyses indepen- <br /> Keightley Eyre-Walker 1999). Although simple defi-                     dent groups suggest correlation gene&#8217;s knockout <br /> nition, quantifying fitness effect gene deletions muta-                    effect purifying selection does exist; studies also <br /> tions remains hard problem. One principal difficulties                   shown correlation probably quite weak (Drummond <br /> that changes deleterious conditions neutral                      et al. 2005; He Zhang 2005; Wall et al. 2005). Most impor- <br /> or beneficial conditions (Macarthur Levins                           tantly, observed correlations taught little evo- <br /> 1964; Levins 1968). As result, notions fitness essen-                   lutionary dynamics governing gene duplication divergence. <br /> tiality inherently ambiguous.                                                          Gene duplication followed divergence pri- <br />       Driven recent availability complete genome                      mary driving forces functional innovation evolu- <br /> sequences, results genome-wide functional as-                        tion (Ohno 1970; Lynch Katju 2004). Currently, diver- <br /> says, researchers observed significant correlations                     gence paralogs duplication thought follow one <br /> functional characteristics genes, essentiality (Hurst                 routes. The common outcome duplication <br /> Smith 1999; Hirsh Fraser 2001; Yang et al. 2003), number                      event nonfunctionalization copy a <br /> protein-interaction partners (Fraser et al. 2002), expression                     pseudogene and, eventually, <span id='am-138' about='obo:NCBITaxon_376232' typeof='obo:NCBITaxon_34667 obo:NCBITaxon_79514 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71528 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_192380 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41088 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-139' property="rdfs:label" content="Goes" datatype="xsd:string"></span><span id='am-140'  ></span><span id='am-141' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>goes</span> extinct (Nei Roychoud- <br /> level (Pal et al. 2001), rate evolution (equivalently                 houry 1973; Petrov Hartl 2000), second copy <br /> strength purifying selection). However, reported correla-                     retains original function. The other, common none- <br /> tions relatively weak, contribution character-                   theless crucial evolutionary scenarios neofunctionalization <br /> istic remains subject vigorous debate (Jordan et al. 2003; Wall                 subfunctionalization. In case neofunctionalization, <br /> et al. 2005; Drummond et al. 2006). In particular, validity                   paralog retains original function, one <br /> the observed negative correlation gene&#8217;s knockout ef-                      evolves new function phase rapid, nearly neutral <br /> fect (essentiality) evolutionary rate, originally predicted                   evolution (Ohno 1970). Under subfunctionalization model, <br /> Kimura Ohta (Kimura 1981) corollary neutral                          multiple functions ancestral gene divided between <br />                                                                                      paralogs, evolve purifying selection (Force <br /> 3 <br />  Corresponding author.                                                               et al. 1999; Lynch Force 2000; He Zhang 2005). Given <br /> E-mail <span id='am-1' about='obo:NCBITaxon_59057' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_59003 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-2' property="rdfs:label" content="Borya" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5'  ></span>Borya</span>@acs.bu.edu; fax (617) 353-4814. <br /> Article published online print. Article publication date http://   sufficiently detailed information, probability functional <br /> www.genome.org/cgi/doi/10.1101/gr.5346206.                                           divergence occurred speciation event estimated <br />  <br />  <br /> 16:1529&#8211;1536 &#169;2006 Cold Spring Harbor Laboratory Press; ISSN 1088-9051/06; www.genome.org                           Genome Research          1529 <br />                                                                                                                           www.genome.org <br />  Shakhnovich Koonin <br />  <br />  <br />  <br />  <br /> Figure 1. The largest E- N-gene families yeast. The family left contains essential genes (N-family), family right includes <br /> two essential members (E-family). The red circles represent essential genes, blue squares represent nonessential genes. The E-family represented <br /> here consists various mitochondrial membrane transporters, N-family includes seripauperins (poorly characterized multigene family repre- <br /> sented, primarily, subtelomeric regions yeast chromosomes). <br />  <br />  <br /> using statistical models sequence evolution (Gu 1999,                       true essential genes paralogs (Yang et <br /> 2001a,b). However, general, relationship selec-                 al. 2003). Since independent line analysis indicated <br /> tion neofunctionalization understood. For ex-                  genes paralogs, average, evolve slowly than <br /> ample, paralogs experience period relaxed purify-                singletons (Davis Petrov 2004; Jordan et al. 2004), nature <br /> ing selection immediately duplication, genes paralogs               relationships variables&#8212;the gene&#8217;s <br /> show greater evolutionary conservation singletons (Davis                  essentiality, existence paralogs, rate evolution&#8212; <br /> and Petrov 2004; Jordan et al. 2004). <span id='am-97' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-98'  ></span><span id='am-99' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-100' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> unexpected find-             remains unclear. We sought explore essentiality or <br /> ing taken indicate that, average, duplications               paralogy necessary sufficient primary determinant of <br /> genes critical biological functions retained               purifying selection. To end, need compare strength <br /> than duplications biologically important genes.                        selection essential nonessential members the <br />       The goal present study elucidate general               gene family. <br /> relationships functional constraints, differential                           Paralogy relationships genes genomes can <br /> strengths purifying selection, gene duplication. First,              conveniently represented general form Diver- <br /> observe membership gene family good predictor                gence Diffusion Graph (DDG). The vertices represent genes, <br /> selection. For example, genes essential paralog               edges represent homology relationships weighed according <br /> under stronger selection genes essential paralogs.                sequence similarity scores (Fig. 1; Supplemental ma- <br /> We families paralogs include es-                terial). All genes partitioned paralogous families (Har- <br /> sential gene (E-families) consist entirely non-              rison Gerstein 2002; Enright et al. 2003) finding strongly <br /> essential genes (N-families) evolve dramatically different regi-            connected components DDG, is, sets vertices in <br /> mens. Although genes E-families, average, evolve substan-                path exists pair (see Supplemental mate- <br /> tially slowly genes N-families, E-families               rial). Partitioning paralogous families (strongly connected com- <br /> a greater divergence paralogs. This attrib-                ponents DDG) classes&#8212;those include least <br /> uted significantly longer average survival time paralogs             essential gene (E-families) essential mem- <br /> in E-families compared N-families. The E-families appear              bers (N-families)&#8212;allows compare essential nonessen- <br /> comprise reservoir genes evolution new functions               tial members family (see Methods). For <span id='am-112' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-113' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> <br /> the subfunctionalization neofunctionalization routes. Fi-                  genome-wide data gene essentiality available, the <br /> nally, relationship evolution               E-family N-family sets included substantial numbers genes <br /> the open reading frames upstream regions. Specifically,                    (Table 1), providing statistical comparison evolutionary <br /> genes E-families stronger selection evolve novel             characteristics essential nonessential genes within <br /> transcriptional regulatory contexts.                                           E-families comparisons sets genes in <br />                                                                                E-families N-families. <br />  <br /> Results                                                                        Table 1. Numbers essential genes paralogs <br />                                                                                sizes E-families N-families <br /> Homogeneity strength selection paralogous <br /> families                                                                                                       Essential     Nonessential <br />                                                                                                All essential   genes       genes         Genes in <br /> While essential genes tend evolve slowly and, implication,               Species             genes       E-families     E-families       N-families <br /> appear stronger purifying selection nonessential <br /> ones (Hirsh Fraser 2001; Jordan et al. 2002), trend                S. cerevisiae      1032             95             183             656 <br />                                                                                C. elegans          861            158             119            1300 <br /> relatively weak. Furthermore, published reports suggest                   E. coli             688             24             328             521 <br /> that negative correlation selection essentiality <br />  <br />  <br /> 1530      Genome Research <br />              www.genome.org <br />                                                                                                               Evolution gene families <br />  <br />  <br /> Table 2. Average strength purifying selection essential     strength selection correlates probability pseudo- <br /> nonessential genes, E-families N-families                  genization M. leprae. Since observed E- <br />                     Essential genes   Nonessential genes       P      families N-families evolve different strengths puri- <br />                                                                       fying selection, hypothesized exhibit <br /> SFP density            0.01567              0.02158          1e-20    different pseudogenization duplication rates. Identification <br /> Ka/Ks &#8211; Yeast          0.10                 0.13             1e-8     pseudogenes (Harrison et al. 2001, 2002; Harrison Ger- <br /> Ka/Ks &#8211; E. coli        0.054                0.099            1.6e-6   stein 2002) derived E-families N-families S. cerevisiae <br />                                                                       showed estimated pseudogenization rate E-families <br /> Only nonessential <br /> genes families      E-families          N-families                 approximately seven times lower N-families. <br /> SFP density            0.012               0.027             1e-40    Although high-precision estimate of <br /> Ka/Ks &#8211; Yeast          0.08                0.12              4e-11    small number pseudogenes E-families, difference in <br /> Ka/Ks &#8211; E. coli        0.056               0.1               1e-8     pseudogenization rates highly statistically significant <br />                                                                       (Table 3). <br /> The table shows essential nonessential members E- <br /> families stronger purifying selection members N-          The larger total number paralogs (Table 1) N-families, <br /> families. The P-values calculated using two-tailed t-test.     coupled observation higher pseudogenization rate, <br />                                                                       suggests recent duplicates N-families enjoy <br />                                                                       higher duplication rate. To test prediction, identified <br />       For example, partitioning paralogous gene sets E-          pairs orthologs lineage-specific paralogs using InPara- <br /> families N-families used test essentiality   noid algorithm (Remm et al. 2001) closely related yeast <br /> a major determinant selection. If strength selection   species (S. cerevisiae, <span id='am-64' about='obo:NCBITaxon_5475' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_118267 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-65' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-66'  ></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="Candida" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Torulopsis" datatype="xsd:string"></span>Candida</span> glabrata, <span id='am-32' about='obo:NCBITaxon_4910' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-33' property="rdfs:label" content="Kluyveromyces" datatype="xsd:string"></span><span id='am-34'  ></span><span id='am-35' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Kluyveromyces</span> lactis, Aphis <br /> characteristic gene family membership opposed essenti-       gossypii, <span id='am-11' about='obo:NCBITaxon_4958' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_766764 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Wingea" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="Debaryomyces" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15'  ></span><span id='am-16' property="oboInOwl:hasAlternativeId" content="NCBITaxon:4945" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasAlternativeId" content="NCBITaxon:28554" datatype="xsd:string"></span>Debaryomyces</span> hansenii). These data num- <br /> ality, predict genes E-families, including nonessen-   ber paralogs fixed species diverg- <br /> tial ones, subject significantly stronger purifying       ing common ancestor. For example, given gene had <br /> selection members N-families. We assessed strength        paralog S. cerevisiae, ortholog paralogs C. <br /> of purifying selection standard measures: single feature   glabrata, infer lineage-specific duplication occurred <br /> polymorphism (SFP) densities <span id='am-37' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-38' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-43'  ></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> genes        divergence C. glabrata. Using approach, we <br /> (Winzeler et al. 2003), K /K s ratios S. cerevisiae&#8211;        calculated number duplicate fixation events mem- <br /> <span id='am-54' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-55' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-60'  ></span>Saccharomyces</span> paradoxus orthologs <span id='am-115' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-116' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-119' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-125'  ></span><span id='am-126' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span>Escherichia coli</span> K12/           bers E-families N-families. We N-families fix <br /> CFT073 orthologs. The genes stronger purifying         duplication events rate approximately twice E- <br /> selection expected lower SFP densities lower Ka/Ks    families (Table 4) For example, 90% E-family members <br /> ratios (Nei 1987).                                                    S. cerevisiae exactly ortholog C. glabrata, while <br />       We essential nonessential members        80% N-families group. In agreement with <br /> E-families substantially stronger purifying selection       this, 17% E-families &gt;46% N-families fixed <br /> than members N-families, independent species analyzed       duplicate (to produce extra paralog) S. cerevisiae <br /> and method used estimate selection (Table 2). Further-         divergence common ancestor D. hansenii. <br /> more, employed criteria, difference strength     The validity analysis hinges assumption the <br /> selection nonessential genes E-families N-             duplication rate E-families N-families. Alter- <br /> families considerably significant difference        natively, differences duplication rates explain the <br /> between essential nonessential genes species      observed pattern invoking differential fixation dupli- <br /> (Table 2). Thus, strength selection        cates; however, hard imagine genes E-families <br /> salient characteristic gene family membership essential-      N-families duplicate different rates. <br /> ity. Moreover, data Table 2 suggest essentiality se         Taken together, duplication pseudogenization data in- <br /> is necessary sufficient impose purifying selection.    dicate N-families evolve significantly &#8220;dynamic&#65533;? <br /> Instead, given E-families exhibit significant biases   regime E-families. Perhaps, weaker purifying se- <br /> characteristics responsible transitive correla-     lection (Table 2), members N-families higher rate of <br /> tions strength selection, <span id='am-102' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-103' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-106'  ></span>Codon</span> Adaptation        pseudogenization (Table 3) duplicate fixation (Table 4). <br /> Index protein abundance (Hahn Kern 2005; Supplemental          Of course, observations entirely indepen- <br /> Table S4), conclude existence essential paralog     dent greater duplicate fixation rate result a <br /> be used specific marker genes subject strong purifying     higher rate pseudogenization. The relevant issue effect of <br /> selection. We systematic differences protein func- <br /> tions E-families N-families; classes fami- <br />                                                                       Table 3.     Genes pseudogenes E-families N-families in <br /> lies showed comparable functional diversity, E- <br />                                                                       yeast <br /> families included greater fraction molecular chaperones and <br /> proteins related functions, N-families en-                        No.           No.             Pseudogene/gene <br /> riched metabolic enzymes transporters (for annotated lists                       genes         pseudogenes                 ratio <br /> of E-family N-family membership, Supplemental ma-         E-families        278           4                            0.014 <br /> terial).                                                              N-families        656           62                           0.095 <br />                                                                       E/N ratio         0.42          0.06                    P-value &lt; 1e-20 <br /> Selection dynamics duplications divergence <br />                                                                       The ratio pseudogenes/genes seven times E-families prob- <br /> The implication observation sets    ability P &lt; 1e-20 difference chance. The calcula- <br /> genes related evolution share characteristics        tion comparing ratio genes types of <br /> impose selection. We recently showed (Shakhnovich 2006)          families pseudogenes. <br />  <br />  <br />                                                                                                              Genome Research             1531 <br />                                                                                                                 www.genome.org <br />  Shakhnovich Koonin <br />  <br />  <br /> Table 4. Rate duplicate fixation E-families N-families              thermore, observed difference distributions synony- <br /> between S. cerevisiae yeasts                                        mous site divergence E-families N-families compatible <br />                  C. glabrata      K. lactis    A. gossypii     D. hansenii    notion characteristic half-life paralogs in <br />                                                                               E-families longer N-families. Taking Ks <br /> Exactly 1 ortholog pair                                                       value measure evolutionary time, estimated pairs <br />   E-families         0.9             0.86          0.87            0.81       paralogs E-families survive, average, times <br />   N-families         0.8             0.6           0.53            0.41       longer (mean Ks = 3.25) paralogs N-families (mean <br />                                                                               Ks = 0.15; P &lt; 1e-40). Thus, increased evolution- <br /> +1 paralog S. cerevisiae <br />   E-families          0.08           0.12          0.11            0.17       ary dynamism results shift shorter life spans genes <br />   N-families          0.17           0.33          0.41            0.46       N-families. <br />  <br /> +2 paralogs S. cerevisiae                                                  Longer life span duplicates E-families allows greater <br />   E-families        &lt; 0.01         &lt; 0.01        &lt; 0.01          &lt; 0.01       divergence sequence transcriptional regulation <br />   N-families          0.02           0.05          0.03            0.07 <br />                                                                                   The observation longer average time survival duplicates <br /> We used InParanoid software estimate number species-                E-families carries significant implications divergence of <br /> specific paralogs S. cerevisiae yeast species.       protein sequences families. In fact, visual examination of <br /> Each difference E-families N-families significant            graph representation largest paralogous families from <br /> P-value &lt; 1e-10 using &#65533;?&#65533;2 tests. <br />                                                                                   S. cerevisiae immediately reveals striking difference the <br />                                                                                   E-families N-families (Fig. 1). Although sizes both <br /> this prodigious rate evolution typical fate para-               families shown Figure 1 similar, N-family much <br /> logs. Does higher pseudogenization rate N-families (Table                  greater number connections node (node degree) the <br /> 3) offset higher duplicate fixation rate (Table 4) resulting             (&#8764;10 compared &#8764;1). We observed substantial <br /> shorter overall survival pairs paralogs? To test this,              difference mean clustering coefficients charac- <br /> assume synonymous sites approximately neutral                        terize density connections genes types <br /> evolve clock-like fashion. Under assumption, distri-                families (&#8764;0.55 N-families compared &#8764;0.21 E-families; <br /> bution synonymous substitutions paralogous gene families                    P &lt; 1e-3). Both number connections node the <br /> should mirror age distribution paralogs.                               clustering coefficient measure transitivity sequence space, sug- <br />       For N-families, distribution Ks shows best fit            gesting that, average, E-family paralogs diverged farther away <br /> an exponential decay curve. This consistent approxi-                   N-family paralogs. <br /> mately constant probability pseudogenization unit time.                           We calculated distribution sequence divergence be- <br /> In sharp contrast, number paralogs E-families correlates                tween pairs paralogs E-families N-families using two <br /> linearly increase synonymous site divergence (Fig. 2).                standard measures nonsynonymous substitution rate, the <br /> The shape distribution Figure 2 E-families                   number nonsynonymous substitutions nonsynonymous <br /> explained model pseudogenization rate                        site (Ka), amino acid sequence identity (see Supplemental <br /> these families drops paralogs diverge (data shown). Fur-                   material). Indeed, criteria, paralogs E-families were <br />                                                                                                             characterized greater diver- <br />                                                                                                             gence (Fig. 3A,B). For example, S. ce- <br />                                                                                                             revisiae, average amino acid se- <br />                                                                                                             quence identity paralogs E- <br />                                                                                                             families &#8764;40% compared &#8764;73% <br />                                                                                                             N-families (Fig. 3B). Qualitatively <br />                                                                                                             similar results observed E- <br />                                                                                                             families N-families E. coli and <br />                                                                                                             <span id='am-77' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-78' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-82'  ></span><span id='am-83' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span>Caenorhabditis elegans</span> (Supplemental <br />                                                                                                             Table S6a,b). Furthermore, differ- <br />                                                                                                             ence function distribution <br />                                                                                                             family size shown Supplemental <br />                                                                                                             Table S7a,b. Finally, nearly identical re- <br />                                                                                                             sults obtained broad range of <br />                                                                                                             cutoffs used classify genes paralogs <br />                                                                                                             (Table 5; Supplemental Table S6a,b). <br />                                                                                                             These results unequivocally that, on <br />                                                                                                             average, paralogs E-families are <br />                                                                                                             separated protein se- <br />                                                                                                             quence space paralogs N- <br />                                                                                                             families. <br />                                                                                                                   In fact, difference sequence <br />                                                                                                             divergence distributions large <br /> Figure 2. Distributions synonymous substitution rates (Ks) pairs paralogs E-familes (gray      wanted assess pre- <br /> circles) N-families (black squares). The x-<span id='am-107' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-108' property="rdfs:label" content="Axis" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-111'  ></span>axis</span> Ks values y-axis number pairs. <br /> The Ks bins 0.3 E-families 0.1 N-families. The black dashed line fit          dictive value differentiating between <br /> exponential decay paralogs N-families (R2 = 0.96, P &lt; 0.001). The solid gray line       E-families N-families absence <br /> linear fit Ks divergence paralogs E-families (R2 = 0.86, P &lt; 0.001).                          essentiality data. We performed re- <br />  <br />  <br /> 1532      Genome Research <br />              www.genome.org <br />                                                                                                                        Evolution gene families <br />  <br />  <br />  <br />  <br /> Figure 3. Sequence divergence E-families N-families. (A) Distributions nonsynonymous substitution rates (Ka) pairs paralogs in <br /> N-families (black squares) E-families (red circles). The mean Ka values &#8764;0.5 E-families &#8764;0.14 N-families (P &lt; 1e-50). (B) Distributions <br /> of sequence identity pairs paralogs E-families (red circles) N-families (black squares). The mean sequence identity &#8764;40% pairs of <br /> paralogs E-families &#8764;73% pairs paralogs N-families (P &lt; 1e-40). <br />  <br />  <br /> ciprocal analysis, is, examined characteristics                In contrast, paralogs N-families appeared coregu- <br /> sequence divergence differentiate E-families                  lated TFs (Fig. 5). Thus, average, E-family <br /> N-families. To end, used Receiver Operating Characteris-                members diverged substantially farther N- <br /> tic (ROC) statistics based average separation paralogs             family members sequence, regulatory <br /> families classify genes E-families N-families               context. This consistent notion longer time of <br /> explicitly invoking essentiality marker. The ROC curve                  survival paralogs E-families allows modification upstream <br /> Figure 4 shows &#8764;80% E-families paralogs                regions increases chances acquisition new regula- <br /> far diverged &#8764;20% N-families. An better separation                tory mechanisms. <br /> E-families N-families obtained using clustering <br /> coefficient classification criterion: 73% E-families            Discussion <br /> were captured single false positive (N-family),              We present evidence gene family membership general <br /> this analysis covered larger families (those                 reliable <span id='am-6' about='obo:NCBITaxon_189528' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_189527 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_9219 obo:NCBITaxon_436491'><span id='am-7' property="rdfs:label" content="Indicator" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10'  ></span>indicator</span> strength purifying selection acting <br /> more members), resulting classification 11 E-families 18 <br /> N-families yeast. Thus, separation paralogs sequence <br /> space E-families N-families dissimilar classifica- <br /> tion based solely sequence divergence, mostly, reproduces the <br /> partitioning based using essential paralogs markers. <br />       Our results E-families explore protein se- <br /> quence space, duplication divergence paralogs, to <br /> a greater depth N-families. This important bear- <br /> ings impact family membership functional diver- <br /> gence. So far, presented evidence paralogs E- <br /> families survive longer, enjoy lower rate pseudogenization, <br /> and diverge sequence farther paralogs N-families. Sub- <br /> functionalization (division ancestral pleiotropy) character- <br /> ized strong purifying selection paralogs dupli- <br /> cation (Force et al. 1999; Lynch Force 2000) be <br /> aided long survival duplicates E-families. Since <br /> sequence divergence taken characteristic sub- <br /> functionalization, reasonably hypothesize paralogs <br /> from E-families preferentially evolve subfunctionaliza- <br /> tion scenario. <br />       If diverging paralogs E-families follow subfunc- <br /> tionalization (or neofunctionalization) routes, expect                Figure 4. ROC curves predicting family belongs E- or <br />                                                                                 N-<span id='am-94' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-95' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-96' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> based average sequence divergence paralogs clustering <br /> to observe divergence sequence expression               coefficient. Paralogous gene families (strongly connected components of <br /> regulation paralogs adapt new biological niches. We                   DDG) sorted average sequence divergence pairs of <br /> compared extent transcription factor (TF) sharing                paralogs (black squares) clustering coefficients (green triangles). <br /> paralogs classes families. In accord notion              Only families members analyzed, families <br />                                                                                 clustering coefficient 0 excluded uninformative. This <br /> of greater functional diversification genes E-families,              results 23 E-families 63 N-families sequence divergence <br /> of paralogs families common transcription                   calculation 12 E-families 18 N-families clustering coeffi- <br /> factors binding upstream regions (Harbison et al. 2004).               cient calculation. The red diagonal line represents random expectation. <br />  <br />  <br />                                                                                                                        Genome Research              1533 <br />                                                                                                                           www.genome.org <br />  Shakhnovich Koonin <br />  <br />  <br /> Table 5. Robustness sequence divergence E-families                       The magnitude observed differences two <br /> and N-families Ks cutoff used building DDG graph S.              types families sharply contrasts previous observations weak <br /> cerevisiae <br />                                                                               moderate correlations various functional evolu- <br />              No. genes      No. genes       &lt; Ka&gt;      &lt;Ka&gt;       tionary characteristics genes determined genome scale <br /> Ks cutoff    E-families     N-families     E-families    N-families     (Hurst Smith 1999; Hirsh Fraser 2001; Pal et al. 2001; <br />                                                                               Jordan et al. 2003; Drummond et al. 2005). We believe the <br /> 5                 275               658          0.509344       0.147632 <br />                                                                               case E-families N-families different this <br /> 4.4               270               659          0.509814       0.147657 <br /> 3.8               264               642          0.507339       0.144439      partitioning attains sharp separation genes classes, <br /> 3.2               246               625          0.500473       0.140768      measured characteristics evolution reflect collective <br /> 2.6               204               585          0.490572       0.128384      behaviors distinct gene sets. The tight link between <br /> 2.3               159               568          0.445489       0.124553 <br />                                                                               gene family membership strength purifying selection <br /> 2                 114               491          0.404539       0.104033 <br /> 1.7                69               434          0.307063       0.091604      likely reflect relationship function con- <br /> 1.4                34               369          0.20307        0.070387      straints. Gene families share set related functions <br />                                                                               (Todd et al. 2001; Shakhnovich Max Harvey 2004) a <br /> Average nonsynonymous sequence divergence (Ka) calculated            similar range functional constraints, translates a <br /> the pairs paralogs. The differences &lt;Ka&gt; E-families         characteristic strength purifying selection. The typical range of <br /> N-families statistically significant (P &lt; 0.001). (An expanded version <br /> is included Supplemental Table S1.)                                        functions associated certain level purifying selection, in <br />                                                                               turn, determines subsequent fate duplicates, is, <br /> on gene. Purifying selection, linked functional constraints,             likelihood gene fixed duplication, diverges a <br /> may affect course molecular evolution                 particular degree paralog, undergoes subfunctional- <br /> influencing speed divergence, affecting fate           ization neofunctionalization. Further investigations the <br /> of paralogs duplication. Specifically, paralogs            origins homogeneity selection acting members paralo- <br /> in E-families include essential genes subject                gous families uncover novel determinants selection <br /> stronger purifying selection genes N-families (Table 2)               rooted commonalities shared members family <br /> without essential paralogs. Thus, salient observation            specific individual genes. <br /> that genes families paralogs experience similar levels                 Another possible conclusion study subfunc- <br /> selection. Although clear-cut difference dis-             tionalization transient phase gene evolution, genes <br /> tribution protein functions E-families N-families,             divide ancestral functions soon undergo neofunctionaliza- <br /> appears presence essential genes corre-             tion. This hypothesis consistent observations the <br /> <span id='am-89' about='obo:NCBITaxon_8186' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_1489905 obo:NCBITaxon_123366 obo:NCBITaxon_1489904 obo:NCBITaxon_1489872 obo:NCBITaxon_8184 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665'><span id='am-90' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-91' property="rdfs:label" content="Lates" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-93'  ></span>lates</span> greater biological importance E-family members                  apparently decreasing rate pseudogenization passage of <br /> (Table 2). In spite stronger selective constraints, paralogs            time (Fig. 2) change transcriptional regulation paralogs <br /> E-families exhibit greater sequence divergence N-families                E-families (Fig. 5). A similar sub-, neofunctionalization model <br /> (Fig. 3). Furthermore, difference exploration se-               recently proposed He Zhang study the <br /> quence space E-families N-families large              evolution protein&#8211;protein interaction networks (He and <br /> we use average sequence divergence paralogs cluster-              Zhang 2005). <br /> ing coefficient predictive manner classify large propor-                  The results presented uncover surprisingly strong link <br /> tion families using existence essential para-            gene&#8217;s membership paralogous family, con- <br /> log criterion (Fig. 4).                                                  straints imposed evolution purifying selection, the <br />       The difference average sequence divergence between <br /> members E-families N-families attributed the <br /> differential dynamics molecular evolution families. <br /> Specifically, N-families evolutionarily dynamic, is, <br /> duplicates families tend pseudogenes shortly <br /> after duplication (Table 3) enjoy higher fixation rate <br /> (Table 4). These results compatible recent evidence in- <br /> dicating that, yeast, functionally important genes tend to <br /> duplicate (He Zhang 2006). The net outcome of <br /> the difference evolutionary dynamics E-families and <br /> N-families paralogs E-families longer life spans <br /> (Fig. 2). Thus, average, paralogs E-families older, is, <br /> have evolving longer time, paralogs N-families <br /> (Fig. 2). It likely that, thanks longer average life <br /> span, paralogs E-families follow subfunctionalization or <br /> neofunctionalization paths paralogs N- <br /> families. This hypothesis supported demonstration of <br /> more extensive transcriptional rewiring (divergence TF- <br /> binding sites) paralogs E-families N-families <br /> (Fig. 5). These findings robust respect method of <br /> family identification and, importantly, species              Figure 5. Shared transcription factors regulating pairs paralogs from <br />                                                                               E-families N-families. In E-families, 92% pairs TF- <br /> families analyzed&#8212;very similar results obtained                 binding sites common, 29% pairs N-families shared <br /> yeast, nematode C. elegans, bacterium E. coli (see            sites TFs (P &lt; 1e-50). The distribution N-families is <br /> Supplemental material).                                                       characterized heavy tail pairs sharing TFs. <br />  <br />  <br /> 1534        Genome Research <br />              www.genome.org <br />                                                                                                                     Evolution gene families <br />  <br />  <br /> characteristics gene family evolution duplication. The clas-                generation protein database search programs. Nucleic Acids Res. <br />                                                                                   25: 3389&#8211;3402. <br /> sification genes E-families N-families useful <br />                                                                               Conery, J.S. Lynch, M. 2001. Nucleotide substitutions the <br /> starting point variety future studies relationships             evolution duplicate genes. Pac. Symp. Biocomput. 2001: 167&#8211;178. <br /> among evolution genes, genomes, phenotypes.                        Cormen, T.H. 2001. Introduction algorithms, 2nd ed. MIT Press, <br />                                                                                   Cambridge, MA. <br />                                                                               Davis, J.C. Petrov, D.A. 2004. Preferential duplication conserved <br />                                                                                   proteins eukaryotic genomes. PLoS Biol. 2: e55. <br /> Methods                                                                       Drake, J.W., Charlesworth, B., Charlesworth, D., Crow, J.F. 1998. <br />                                                                                   Rates spontaneous mutation. Genetics 148: 1667&#8211;1686. <br /> To construct DDG species (S. cerevisiae, C.             Drummond, D.A., Bloom, J.D., Adami, C., Wilke, C.O., Arnold, F.H. <br /> elegans, E. coli), complete sets protein sequences                2005. Why highly expressed proteins evolve slowly. Proc. Natl. Acad. <br />                                                                                   Sci. 102: 14338&#8211;14343. <br /> the respective genomes extracted GenBank data-                  Drummond, D.A., Raval, A., Wilke, C.O. 2006. A single determinant <br /> base, all-against-all sequence comparison performed                    dominates rate yeast protein evolution. Mol. Biol. Evol. <br /> using BLAST program (Altschul et al. 1997; ftp://                             23: 327&#8211;337. <br /> ftp.ncbi.nih.gov). Each protein node DDG,                 Enright, A.J., Kunin, V., Ouzounis, C.A. 2003. Protein families and <br />                                                                                   TRIBES genome sequence space. Nucleic Acids Res. 31: 4632&#8211;4638. <br /> results comparisons, represented BLAST                     Force, A., Lynch, M., Pickett, F.B., Amores, A., Yan, Y.L., and <br /> scores, expectation (e) values, amino acid sequence identities,                Postlethwait, J. 1999. Preservation duplicate genes by <br /> are edges connecting nodes. The edges assigned                       complementary, degenerative mutations. Genetics 151: 1531&#8211;1545. <br /> weights using BLAST scores, amino acid sequence identities,                Fraser, A.G., Kamath, R.S., Zipperlen, P., Martinez-Campos, M., <br />                                                                                   Sohrmann, M., Ahringer, J. 2000. Functional genomic analysis <br /> the ratio nonsynonymous (Ka) synonymous (Ks)                       C. elegans chromosome I systematic RNA interference. Nature <br /> substitution rates. To calculate Ka Ks values, amino acid                 408: 325&#8211;330. <br /> sequence alignments traced corresponding                     Fraser, H.B., Hirsh, A.E., Steinmetz, L.M., Scharfe, C., Feldman, <br /> nucleotide sequence alignments, Ka Ks calculated                     M.W. 2002. Evolutionary rate protein interaction network. <br />                                                                                   Science 296: 750&#8211;752. <br /> using PAML (Yang 1997; Lynch Conery 2000; Yang                        Gerdes, S.Y., Scholle, M.D., Campbell, J.W., Balazsi, G., Ravasz, E., <br /> Nielsen 2000; Conery Lynch 2001). A similar procedure                     Daugherty, M.D., Somera, A.L., Kyrpides, N.C., Anderson, I., <br /> used Enright et al. (2003).                                                    Gelfand, M.S., et al. 2003. Experimental determination system <br />       Paralogous families identified finding strongly                 level analysis essential genes <span id='am-131' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-132' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-135' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-136' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-137'  ></span>Escherichia</span> coli MG1655. J. <br />                                                                                   Bacteriol. 185: 5673&#8211;5684. <br /> connected components DDG described Cormen                        Giaever, G., Chu, A.M., Ni, L., Connelly, C., Riles, L., Veronneau, S., <br /> (2001) (see Supplemental Table S1). We define strongly con-                     Dow, S., Lucau-Danila, A., Anderson, K., Andre, B., et al. 2002. <br /> nected component set nodes path exists                    Functional profiling Saccharomyces cerevisiae genome. Nature <br /> any pair. The principal results presented obtained us-                  418: 387&#8211;391. <br />                                                                               Gu, X. 1999. Statistical methods testing functional divergence after <br /> ing BLAST cutoff 1e-15 dS = 5. However, nearly identical                 gene duplication. Mol. Biol. Evol. 16: 1664&#8211;1674. <br /> results obtained broad range cutoffs (Table 3)              Gu, X. 2001a. Mathematical modeling functional divergence after <br /> After identifying strongly connected components DDG,                   gene duplication. J. Comput. Biol. 8: 221&#8211;234. <br /> high-throughput essentiality data used divide fami-               Gu, X. 2001b. A site-specific measure rate difference gene <br />                                                                                   duplication speciation. Mol. Biol. Evol. 18: 2327&#8211;2330. <br /> lies E- N-classes. The yeast gene essentiality data              Hahn, M.W. Kern, A.D. 2005. Comparative genomics centrality <br /> were obtained high-throughput knockout experiments                           essentiality eukaryotic protein-interaction networks. <br /> (Giaever et al. 2002); C. elegans gene essentiality data                 Mol. Biol. Evol. 22: 803&#8211;806. <br /> from genome-wide RNAi knockdown experiments (Fraser et                    Harbison, C.T., Gordon, D.B., Lee, T.I., Rinaldi, N.J., Macisaac, K.D., <br />                                                                                   Danford, T.W., Hannett, N.M., Tagne, J.B., Reynolds, D.B., Yoo, J., et <br /> al. 2000; Kamath et al. 2003; Simmer et al. 2003), E. coli                al. 2004. Transcriptional regulatory code eukaryotic genome. <br /> essentiality data GenBank (ftp://ftp.ncbi.nih.gov)                      Nature 431: 99&#8211;104. <br /> (Gerdes et al. 2003; details, Supplemental material).             Harrison, P.M. Gerstein, M. 2002. Studying genomes the <br /> The data yeast SNPs Winzeler et al. (2003). The                      aeons: Protein families, pseudogenes proteome evolution. J. Mol. <br />                                                                                   Biol. 318: 1155&#8211;1174. <br /> transcription factor data ChIP-chip experiments (Lee                Harrison, P.M., Echols, N., Gerstein, M.B. 2001. Digging dead <br /> et al. 2002; Harbison et al. 2004).                                               genes: An analysis characteristics pseudogene <br />       We used InParanoid program (http://inparanoid.                          population <span id='am-84' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-85' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-87'  ></span><span id='am-88' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Caenorhabditis</span> elegans genome. Nucleic Acids Res. <br /> cgb.ki.se/) identify orthologs species-specific paralogs                   29: 818&#8211;830. <br />                                                                               Harrison, P., Kumar, A., Lan, N., Echols, N., Snyder, M., Gerstein, <br /> from yeast species (S. cerevisiae, C. glabrata, K. lactis, A.            M. 2002. A small reservoir disabled ORFs yeast genome and <br /> gossypii, D. hansenii). The genomes obtained                    implications dynamics proteome evolution. J. Mol. Biol. <br /> NCBI (http://www.ncbi.nlm/nih.gov).                                               316: 409&#8211;419. <br />                                                                               He, X. Zhang, J. 2005. Rapid subfunctionalization accompanied by <br />                                                                                   prolonged substantial neofunctionalization duplicate gene <br />                                                                                   evolution. Genetics 169: 1157&#8211;1164. <br /> Acknowledgments                                                               He, X. Zhang, J. 2006. Higher duplicability important genes <br />                                                                                   yeast genomes. Mol. Biol. Evol. 23: 144&#8211;151. <br /> The authors thank Eugene Shakhnovich invaluable sup-                  Hirsh, A.E. Fraser, H.B. 2001. Protein dispensability rate of <br /> port review manuscript. We thank Matthew Hahn                     evolution. Nature 411: 1046&#8211;1049. <br /> for proposing duplication rate study helpful                 Hurst, L.D. Smith, N.G. 1999. Do essential genes evolve slowly? <br /> comments regarding manuscript. Additionally, extend                        Curr. Biol. 9: 747&#8211;750. <br />                                                                               Jordan, I.K., Rogozin, I.B., Wolf, Y.I., Koonin, E.V. 2002. Essential <br /> our appreciation Charles DeLisi, Tim Reddy, Joe Mellor, Julian                 genes evolutionarily conserved nonessential genes <br /> Mintseris, Bioinformatics program Boston                     bacteria. Genome Res. 12: 962&#8211;968. <br /> University discussion insights.                                       Jordan, I.K., Wolf, Y.I., Koonin, E.V. 2003. No simple dependence <br />                                                                                   protein evolution rate number protein&#8211;protein <br />                                                                                   interactions: Only prolific interactors tend evolve <br />                                                                                   slowly. BMC Evol. Biol. 3: 1. <br /> References                                                                    Jordan, I.K., Wolf, Y.I., Koonin, E.V. 2004. Duplicated genes evolve <br />                                                                                   slower singletons despite initial rate increase. BMC Evol. <br /> Altschul, S.F., Madden, T.L., Schaffer, A.A., Zhang, J., Zhang, Z., Miller,       Biol. 4: 22. <br />     W., Lipman, D.J. 1997. Gapped BLAST PSI-BLAST: A new              Kamath, R.S., Fraser, A.G., Dong, Y., Poulin, G., Durbin, R., Gotta, M., <br />  <br />  <br />  <br />                                                                                                                     Genome Research              1535 <br />                                                                                                                        www.genome.org <br />  Shakhnovich Koonin <br />  <br />  <br />     Kanapin, A., Le Bot, N., Moreno, S., Sohrmann, M., et al. 2003.          Remm, M., Storm, C.E., Sonnhammer, E.L. 2001. Automatic <br />     Systematic functional analysis Caenorhabditis elegans genome         clustering orthologs in-paralogs pairwise species <br />     using RNAi. Nature 421: 231&#8211;237.                                            comparisons. J. Mol. Biol. 314: 1041&#8211;1052. <br /> Keightley, P.D. Eyre-Walker, A. 1999. Terumi Mukai riddle        Shakhnovich, B.E. 2006. Relative contributions structural <br />     deleterious mutation rates. Genetics 153: 515&#8211;523.                       des <br /> </body></html>