<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>2304                                                                       Biophysical Journal   Volume 91   September 2006      2304&#8211;2311 <br />  <br />  <br /> Using Topology Metabolic Networks Predict Viability <br /> of Mutant Strains <br />  <br /> Zeba Wunderlich* Leonid A. Mirnyy <br /> *Biophysics Program, Harvard University, Cambridge, Massachusetts; yHarvard-MIT Division Health Sciences Technology, <br /> Cambridge, Massachusetts <br />  <br />  <br /> ABSTRACT Understanding relationships structure (topology) function biological networks central <br /> question systems biology. The <span id='am-25' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-29'  ></span>idea</span> topology major determinant systems function attractive and <br /> highly disputed hypothesis. Although structural analysis interaction networks demonstrates correlation the <br /> topological properties node (protein, gene) network functional essentiality, analysis metabolic networks <br /> fails &#65533;?nd correlations. In contrast, approaches utilizing topology biochemical parameters metabolic <br /> networks, e.g., &#64258;ux balance analysis, successful predicting phenotypes knockout strains. We reconcile these <br /> seemingly con&#64258;icting results showing topology metabolic networks <span id='am-57' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-58' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-65'  ></span><span id='am-66' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span>Escherichia coli</span> and <br /> <span id='am-1' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-2' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> are, fact, suf&#65533;?cient predict viability knockout strains accuracy comparable &#64258;ux <br /> balance analysis large, unbiased mutant <span id='am-104' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-105'  ></span><span id='am-106' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> sets. <span id='am-30' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-31'  ></span><span id='am-32' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> surprising result obtained introducing novel topology-based <br /> measure network transport: synthetic accessibility. We popular topology-based characteristics as <br /> node degree, graph diameter, node usage (betweenness) fail predict viability E. coli mutant strains. The success of <br /> synthetic accessibility demonstrates ability capture essential properties metabolic network, the <br /> branching chemical reactions directed transport material inputs outputs. Our results strongly support link <br /> between topology function biological networks and, agreement recent genetic studies, emphasize minimal <br /> role &#64258;ux rerouting providing robustness mutant strains. <br />  <br />  <br /> INTRODUCTION <br /> Many suggested debated idea topology                    single-gene knockout strains. EMA predict viability <br /> determines network function. Although structures               mutant strains comparable accuracy. Because these <br /> biological networks available, remains hard separate            methods use network topology, stoichiometry of <br /> the contributions topology contributions                   metabolic chemical transport reactions, cases, <br /> kinetic equilibrium parameters. Because well-                maximal rates reactions, cannot <br /> established structures wealth related experimental             separate role topology role played other <br /> data, <span id='am-73' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-74' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-75'  ></span><span id='am-76' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span>Escherichia</span> coli <span id='am-18' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-20'  ></span><span id='am-21' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae                   parameters network function. In addition, the <br /> metabolic networks perfect model systems explore               complexity method results, EMA techniques <br /> role network topology. Is topology metabolic network              computationally expensive (12) provide little insight <br /> sufficient predict viability knockout mutants?                  certain mutations lethal, are <br />    Metabolic networks modeled extensively using                 tolerated. <br /> steady-state flux balance approaches (1&#8211;8). To test                      Here untangle topology stoichiometry the <br /> capabilities metabolic network models, groups                metabolic network topology sufficient <br /> compared predicted experimentally measured effects                 predict viability E. coli S. cerevisiae <br /> gene deletions cell growth. Among effective                   mutant strains accurately FBA large, unbiased sets <br /> methods flux balance analysis (FBA) (3,4,6&#8211;9), re-                mutants (9,13,14). This result supports claim that <br /> lated minimization metabolic adjustment (<span id='am-98' about='obo:NCBITaxon_997520' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_95175 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-99' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-101'  ></span><span id='am-102' property="rdfs:label" content="Moma" datatype="xsd:string"></span>MOMA</span>) method                  topology plays central role determining network func- <br /> (10), elementary mode analysis (EMA) (11). Although                   tion malfunction (15,16). We employ novel network <br /> these methods shown useful understand-                 property, synthetic accessibility, intuitive trans- <br /> ing structure dynamics metabolic fluxes,                  parent way understanding effects metabolic muta- <br /> deliver different experimentally testable predictions. FBA                tion (Fig. 1). We define synthetic accessibility, S, total <br /> can accurately predict fluxes individual reactions             number reactions needed transform given set input <br /> the wild-type mutant strains viability              metabolites set output metabolites predict that <br />                                                                           increases S result alterations topology of <br />                                                                           metabolic network adversely affect growth. The term <br /> Submitted December 30, 2005, accepted publication May 24, 2006.   &#8216;&#8216;synthetic accessibility&#8217;&#8217; borrowed field drug <br /> Address reprint requests Leonid Mirny, Harvard-MIT Div. Health      design, defined smallest number chem- <br /> Sciences &amp; Technology, Massachusetts Institute Technology, 77          ical steps needed synthesize drug common labo- <br /> Massachusetts Ave., Cambridge, MA 02139. Tel.: 617-452-4862; Fax:         ratory reactants similar spirit &#8216;&#8216;scope&#8217;&#8217; of <br /> 617-253-2514; E-mail: leonid@mit.edu. <br /> &#211; 2006 Biophysical Society <br /> 0006-3495/06/09/2304/08 $2.00                                                                                doi: 10.1529/biophysj.105.080572 <br />  Topology-Based Metabolic Predictions                                                                                                                         2305 <br />  <br />  <br />                                                                                    Construction graphic metabolism model <br />                                                                                    The reactions included E. coli metabolic network taken from <br />                                                                                    Edwards Palsson (4), reactions included yeast metabolic <br />                                                                                    network taken Duarte et al. (8). Although updated <br />                                                                                    version E. coli metabolic network available (6), chose use the <br />                                                                                    previous version enable comparison synthetic accessibility perfor- <br />                                                                                    mance previous studies (4,9&#8211;11). Each reaction metabolite repre- <br />                                                                                    sented node, directed edges connect reactants reactions and <br />                                                                                    reactions products, accounting reversibility reactions. <br />  <br />  <br />                                                                                    Selection input output metabolite sets <br />                                                                                    The input metabolites E. coli minimal medium, E. coli rich medium, and <br />                                                                                    various yeast medium conditions listed Supplementary Material, <br />                                                                                    Tables S1&#8211;S4. E. coli minimal medium consists energy source <br />                                                                                    (glucose, acetate, glycerol, succinate), components minimal medium, <br />                                                                                    sulfur source, carbon dioxide oxygen, nicotinamide mononucleotide, <br />                                                                                    regulatory protein thioredoxin (Supplementary Material, Table S1). <br />                                                                                    The input metabolites chosen match real composition minimal <br /> FIGURE 1 An illustration synthetic accessibility approach. In          medium closely possible. Nicotinamide mononucleotide thiore- <br /> representation metabolic network, circular nodes represent metabo-          doxin included ensure that, wild-type network, components <br /> lites, rectangular nodes represent reactions, directed edges indicate    output biomass accessible. They chosen specifically because <br /> relationships. Nodes outline (green blue) synthetically        upstream metabolites biomass synthesis pathways. <br /> accessible, nodes outline (red orange) accessible.      E. coli rich medium consists metabolites minimal medium <br /> The algorithm begins identifying reactions neighbor input      biotin, riboflavin, pantoate, pyridoxine, thiamin, dihydrofolate, <br /> metabolites (nodes A&#8211;D) marking reactions                p-aminobenzoic acid, 20 amino acids, nucleotide bases <br /> reactants available accessible (reactions 1 2). All products     included external metabolites metabolic network (external thymine <br /> these reactions marked accessible (nodes F&#8211;H). The algorithm              metabolic network). Rich medium difficult model <br /> examines neighboring reactions newly marked metabolites           accurately, using slightly different input metabolite sets no <br /> the step continues new metabolites marked accessible.       significant effect results (results shown). <br /> The inset demonstrates happens gene produces enzyme               The input metabolites yeast based descriptions Duarte <br /> that catalyzes reaction 2 deleted: metabolites H K reaction 5         et al. (8) include histidine, leucine, uracil compensate the <br /> would accessible more. We define synthetic accessibility, S,         deletions His-3/Leu-2/Ura-3 mutant strains. Additionally, <br /> the number reactions required transform set inputs set        thioredoxin (oxidized), H1 (in endoplasmic reticulum), NADPH (in the <br /> outputs. Synthetic accessibility analogous diameter directed        endoplasmic reticulum), dolichol included inputs, without <br /> graph, contrast graph diameters, synthetic accessibility takes      them, components biomass producible, the <br /> account branching nature chemical reactions purpose              wild-type network. <br /> metabolic networks, produce outputs inputs.                                    The E. coli output metabolites taken components E. coli <br />                                                                                    biomass (Supplementary Material, Table S5) (19). The yeast output metabo- <br /> metabolites (17,18). We demonstrate network                        lites components biomass reaction reported (8). <br /> characteristics node degree change graph <br /> diameter unable predict viability E. coli mutant <br />                                                                                    Synthetic accessibility algorithm <br /> strains better random predictions, suggesting synthetic <br /> accessibility appropriate characteristic networks                    To determine synthetic accessibility outputs given inputs, we <br /> with directed transport, metabolic networks.                               use type iterative breadth-first search, similar previously described <br />                                                                                    &#8216;&#8216;forward-firing&#8217;&#8217; (Fig. 1) (20). The algorithm starts examining the <br />                                                                                    reactions require given input metabolites reactant. It then <br /> MATERIALS AND METHODS                                                              marks reactions reactants available &#8216;&#8216;accessible&#8217;&#8217; <br />                                                                                    marks metabolites produced reactions &#8216;&#8216;accessible&#8217;&#8217; as <br /> De&#65533;?nition synthetic accessibility                                               well. The algorithm examines reactions require newly <br />                                                                                    marked metabolites starting material, determines reaction <br /> Consider metabolic network access certain inputs: substrates         accessible based availability reactants, until <br /> consumed environment (e.g., sugars, oxygen, nitrogen),           new metabolites marked accessible. Concurrently, number of <br /> the aim producing certain outputs amino acids, nucleotides,         steps needed reach accessible metabolite j, synthetic accessibility <br /> other components collectively called biomass. We define synthetic          Sj, recorded; synthetic accessibility network S calculated by <br /> accessibility Sj output j minimal number metabolic reactions       summing synthetic accessibilities outputs. <br /> needed produce j network inputs (Fig. 1). Sj set infinity j <br /> cannot synthesized network inputs. Summing synthetic <br /> accessibility components biomass, obtain total              Comparison experimental results <br /> synthetic accessibility S &#188; +i Si biomass. We propose an <br /> enzyme knockout does change S, i.e., biomass produced               To compare results approach experimental data sets, first <br /> without extra metabolic cost, mutant viable. If S &#188; N,         create adjacency matrix, represents wild-type metabolic net- <br /> essential component biomass produced network                 work topology. Then, mutant strain, create &#8216;&#8216;mutated&#8217;&#8217; adja- <br /> inputs, predict lethal phenotype.                               cency matrix removing reactions catalyzed gene. For all <br />  <br />                                                                                                                       Biophysical Journal 91(6) 2304&#8211;2311 <br />  2306                                                                                                                                       Wunderlich Mirny <br />  <br /> E. coli predictions, previous papers, delete corresponding           unrelated reactions. We measure performance synthetic acces- <br /> genes reactions catalyzed multiple isozymes. We calculate              sibility erroneous network plotting accuracy percentage <br /> viability mutant compare results experimental data (see         shuffled assignments. <br /> Supplementary Material). If Smutant &#188; Swild-type, predict mutant is <br /> viable; predict inviable. In E. coli insertional mutant data set, <br /> phenotype data given competitive growth rates. A mutant                      RESULTS <br /> considered negatively selected (or inviable) twofold decrease <br /> in growth rates 30 generations (9). For Gerdes et al. data set (21),       Performance synthetic accessibility the <br /> create mutated adjacency matrices genes included metabolic             E. coli metabolic network <br /> network model, resulting 598 mutated adjacency matrices. For yeast <br /> experimental data, use preprocessed data set created Duarte et al.           To study performance synthetic accessibility in <br /> (8) simultaneously delete isozymes.                                         predicting viability knockout strains compare to <br />                                                                                        previous studies, applied method E. coli <br />                                                                                        metabolic network. We initially tested data sets <br /> Calculation topology-based predictions <br />                                                                                        used previous studies: large, unbiased data set of <br /> We explore number topology-based measures predictors                  insertional (transposon-induced) mutants (9) smaller <br /> E. coli mutant viability, including node degree, diameter, node usage. <br />                                                                                        data set collected FBA analysis (4), mainly <br /> The degree enzyme calculated summing degrees the <br /> reactions catalyzed enzyme isozymes. We define network                  contained knockouts enzymes involved central metab- <br /> diameter sum metabolites versus metabolites&#8217; shortest paths,         olism. All mutants grown minimal medium. We <br /> and mutant, calculate change network diameter wild-            used data sets specifically used in <br /> type. We define node usage enzyme number times                  previous studies (4,9&#8211;11), compared results. <br /> reactions catalyzed enzyme used produce biomass wild- <br />                                                                                        We used union data sets refer below <br /> type strain, according synthetic accessibility approach, is <br /> essentially analogous betweenness (22,23). For measure, degree,                combined data set. When applied combined data <br /> diameter, usage, predict enzyme essential (and, therefore,             set, approach performed (62% accuracy, p &#188; 6 3 <br /> the corresponding mutant stain inviable), measure greater            10 8) FBA approach (62%, p &#188; 3 3 10 8) (see Table <br /> than given cutoff. We vary cutoff entire range possible         1, Fig. 2 details). On large data set 487 insertional <br /> values value gives optimal performance, measured either <br />                                                                                        mutants (9), synthetic accessibility approach performed <br /> by accuracy significance x 2 statistic. <br />                                                                                        (60% accuracy, p &#188; 3 3 10 5) FBA and <br />                                                                                        MOMA approaches (58% 59% accuracy, p &#188; 1 3 10 3 <br /> Quantitative analysis performance                                                   1 3 10 4, respectively), somewhat higher <br />                                                                                        statistical significance. On smaller data set 79 mutants <br /> To assess performance synthetic accessibility methods in <br /> predicting phenotype mutant stains, use measures: accuracy,             (4), FBA correctly predicted 86% cases, our <br /> sensitivity, specificity, p-value x 2 statistic. We define accuracy     topology-based synthetic accessibility approach 71% <br /> as (TP 1 TN)/(TP 1 TN 1 FP 1 FN), TP number true                       accuracy, providing correct predictions 53/68 &#188; 78% of <br /> positives, TN number true negatives, FP number false               cases predicted correctly FBA. <br /> positives, FN number false negatives. We define positives and <br /> negatives terms experimentally measured phenotypes, where <br />                                                                                           The difference performance synthetic <br /> positives viable strains negatives inviable strains,            accessibility approach compared FBA the <br /> assignment arbitrary reversed. In similar fashion, define           data sets probably related way data sets <br /> sensitivity TP/(TP 1 FP) specificity TN/(TN 1 FN). To calculate              interpreted cases included data sets. In the <br /> the x 2 statistic, use two-by-two contingency tables sort mutant          smaller data set, mutant strains classified viable or <br /> strain based silico vivo phenotypes calculate the <br /> appropriate p-value. <br />                                                                                        inviable, insertional data set, mutants are <br />                                                                                        labeled negatively selected (the population mutant <br />                                                                                        strain one-half wild-type population 30 <br /> Assessment synthetic accessibility robustness                                       generations competitive growth) negatively selected. <br /> To test robustness approach, introduce random mistakes              Because synthetic accessibility approach deems mutant <br /> the E. coli network randomly reassigning certain fraction enzymes           strain inviable negatively selected based path <br />  <br /> TABLE 1 Comparison accuracy statistical signi&#65533;?cance FBA, MOMA, EMA, synthetic accessibility methods <br /> applied E. coli metabolic network <br />                                                                                                           Method <br />  <br />   Mutant data source              No. cases         Synthetic accessibility               FBA                        MOMA                         EMA <br />                                                                                                     11                       <br /> Collected literature               79                 71%, 8 3 10 *  -5 <br />                                                                                       86%, 7 3 10    (4)                                     90%, 3 3 10 14 (11) <br /> Insertional mutants                    481                 60%, 3 3 10 5              58%, 1 3 10 3 (9)          59%, 1 3 10    4 <br />                                                                                                                                     (10)             &#8211; <br /> Combined data sets                     560                 62%, 6 3 10 8              62%, 3 3 10 8                      &#8211;                           &#8211; <br /> Gerdes data set                        598                 74%, 1 3 10 5                      &#8211;                          &#8211;                           &#8211; <br /> *Accuracy, p-value x 2 statistic. <br />  <br /> Biophysical Journal 91(6) 2304&#8211;2311 <br />  Topology-Based Metabolic Predictions                                                                                                                            2307 <br />  <br />  <br />  <br />  <br /> FIGURE 2 Performance synthetic accessibility compared FBA, MOMA, EMA, topology-based measures using E. coli metabolic <br /> network. The graphs illustrate relative performance techniques using measures, accuracy, (TP 1 TN)/(TP 1 TN 1 FP 1 FN), negative <br /> log x 2 statistic&#8217;s p-value, indicates correlation silico predictions vivo observations E. coli mutant strain viability. <br /> The x2 statistic calculated using contingency table like ones Fig. 3 smaller data set (79 data points, 90 data points EMA), insertional <br /> mutant data set (487 data points), combined data set (560 data points) (4,9,11). When larger, representative insertional mutant data set the <br /> combined data set used, synthetic accessibility accurate statistically significant FBA. However, synthetic accessibility performs poorly <br /> on smaller data set, probably data set data points covers central metabolism, small fraction metabolic network. <br /> The topology-based measures, degree diameter, perform worse FBA, MOMA, EMA, synthetic accessibility, indicating poorly <br /> characterize functioning metabolic network. The random predictions using expected values produced FBA x 2 test represent <br /> the expected performance correlation silico vivo predictions. They vary little expected values other <br /> x 2 tests used. <br />  <br />  <br /> lengths inputs outputs accessibility outputs,                     2. Our approach correctly predicted strains with <br /> the classification scheme correspond closely                            longer rerouted pathways inviable, suggesting that <br /> to synthetic accessibility approach: longer path lengths                            rerouting metabolic fluxes plays small role rescuing <br /> may correspond reduced growth rates invia-                               mutant strains. This result consistent results of <br /> bility.                                                                                 FBA analysis yeast mutants (24). <br />    The number type data points included data                           3. Most viable mutants untouched primary <br /> sets different. The insertional data set larger                        synthetic pathways short rerouting (e.g., because <br /> (487 vs. 79 data points) includes fairly random col-                              isozymes). <br /> lection insertions metabolic genes, smaller <br /> data set contains data enzymes used                               Although used previous FBA studies, we <br /> central metabolism (glycolysis, pentose phosphate pathway,                           applied synthetic accessibility approach large- <br /> citric acid cycle, respiration processes) (4). Because                           scale knockout study Gerdes et al. (21), identified <br /> central metabolism contains number alternate pathways,                          genes essential robust growth rich medium using a <br /> some require fewer steps commonly                              genetic footprinting technique based transposon-based <br /> used pathways, surprising synthetic acces-                        mutagenesis. The synthetic accessibility approach performed <br /> sibility approach performs poorly FBA                                 data set (74% accuracy, p &#188; 1 3 10 5). <br /> applied smaller data sets. <br />    In regard combined data set, synthetic accessibility <br /> had greater sensitivity, indicating better FBA                      Performance synthetic accessibility the <br /> or MOMA predicting strains viable,                            yeast metabolic network <br /> lower specificity, indicating good at <br />                                                                                      To ensure success synthetic accessibility <br /> predicting inviable strains (Figs. 3 4). The success of <br />                                                                                      method limited E. coli metabolic network, we <br /> synthetic accessibility combined data set demonstrates <br />                                                                                      tested method metabolic network S. cerevisiae, <br /> three important results, making transparent difference <br />                                                                                      metabolic network reconstructed by <br /> between viable nonviable strains. <br />                                                                                      hand (8). This reconstruction extensively validated <br /> 1. Most nonviable mutants simply lack pathway                                   use FBA predict phenotypes large <br />    synthesize biomass components (S &#188; N),                              number single-gene knockout yeast strains grown a <br />    i.e., essential metabolites produced                        variety conditions (13,14). The conditions include glucose <br />    network inputs (Table 2).                                                     minimal medium (MMD) rich medium defined <br />  <br />                                                                                                                          Biophysical Journal 91(6) 2304&#8211;2311 <br />  2308                                                                                                                                   Wunderlich Mirny <br />  <br />  <br />  <br />  <br /> FIGURE 3 Results synthetic accessibility approach, divided type prediction. These contingency graphs allow exploration types of <br /> errors common. Results reported (A) combined E. coli data set, (B) Gerdes et al. (21) E. coli data set, (C) combined yeast <br /> data set (metabolic genes only). The x <span id='am-35' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-36' property="rdfs:label" content="Axis" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-38'  ></span><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>axis</span> represents phenotypes predicted synthetic accessibility method, y axis represents experimental <br /> phenotypes. The green blocks correspond cases prediction matches experiment, red, hashed blocks correspond errors. The area box <br /> is proportional number cases category. From diagram, common type error synthetic accessibility <br /> approach predicts mutant viable actually inviable. <br />  <br />  <br /> carbon source (YPGal, galactose; YPD, glucose; YPDGE,                              essential slow sets, synthetic accessibility performs <br /> glucose-ethanol-glycerol, YPG, glycerol; YPE, ethanol;                         comparably FBA. When conditions considered <br /> YPL, lactate). Sets essential slow-growth genes                        simultaneously, synthetic accessibility predicts phenotype <br /> also identified experimentally genes                           83.7% accuracy, compared FBA 82.6% <br /> mutant strains constructed genes pro-                         accuracy. <br /> duced slow-growing mutant strains rich (YPD) medium.                               We believe higher overall accuracy synthetic <br /> The results (Table 3) gene sets show,                       accessibility FBA applied yeast metabolic <br />                                                                                    network probably largely result way data sets <br />                                                                                    used. For E. coli data sets, predictions made <br />                                                                                    knockout strains involved genes were <br />                                                                                    included metabolic network model. For yeast data <br />                                                                                    sets, following protocol previous FBA study (8), <br />                                                                                    predictions strains, gene was <br />                                                                                    included metabolic network model not. Because <br />                                                                                    genes nonessential, predict knockouts of <br />                                                                                    genes absent metabolic network model viable, <br />                                                                                    inflates accuracy. We report accuracies for <br />                                                                                    predictions metabolic gene knockouts Table 3, and <br />                                                                                    accuracies higher cases. <br />  <br />                                                                                    Performance topology-based measures <br />                                                                                    E. coli metabolic network <br />  <br /> FIGURE 4 Plot sensitivity specificity synthetic accessibility           We tested ability topology-based graph charac- <br /> (SA) prediction methods. For combined E. coli data set (560          teristics, node degree, graph diameter, node <br /> data points), sensitivity, TP/(TP 1 FP), specificity, TN/(TN 1 FN),        usage (see Materials Methods), predict viability of <br /> calculated predictions using synthetic accessibility, FBA, <br /> degree, diameter. The cutoff values degree diameter selected <br />                                                                                    TABLE 2 E. coli mutants predicted inviable by <br /> to minimize x 2 test p-value. The random values calculated using the <br />                                                                                    synthetic accessibility approach combined data set, <br /> expected values calculated x2 test FBA essentially the <br />                                                                                    divided reason predicting inviability <br /> same values synthetic accessibility used instead. Though both <br /> degree diameter good sensitivity, specificity quite low. Both    Reason predicting inviability    Correct (percent)    Incorrect (percent) <br /> synthetic accessibility FBA moderate values sensitivity <br />                                                                                    No. accessible outputs ,              89 (59%)              63 (41%) <br /> and specificity. In cases, sensitivity greater, implying the <br />                                                                                      wild-type (S &#188; N) <br /> viable predictions reliable inviable predictions, also <br />                                                                                    S . wild-type                            10 (67%)               5 (33%) <br /> be seen Fig. 3. <br />  <br /> Biophysical Journal 91(6) 2304&#8211;2311 <br />  Topology-Based Metabolic Predictions                                                                                                           2309 <br />  <br /> TABLE 3 Accuracy synthetic accessibility FBA methods predicting viability yeast deletion strains <br />                                                                                       Data set <br />  <br />                                         Essential    Slow    MMD      YPGal      YPD        YPDGE        YPG         YPE         YPL          All <br /> No. cases                            118         83       564      564       565         565         565         565        565         4154 <br /> FBA (8)                                  31.4%      19.3%     84.0%    85.6%     84.4%       85.3%       86.5%       85.7%      86.4%        82.6% <br /> Synthetic accessibility                  11.9%       1.20%    94.0%    97.2%     85.3%       84.4%       83.7%       84.1%      84.1%        83.7% <br /> No. cases (only metabolic enzymes)   100         45       459      459       462         462         462         462        462         3373 <br /> FBA                                      33.0%       4.44%    95.0%    97.6%     87.9%       87.9%       89.2%       87.4%      88.3%        87.6% <br /> Synthetic accessibility                  14.0%       2.22%    94.3%    96.9%     88.5%       88.3%       89.0%       88.7%      88.7%        87.1% <br />  <br />  <br />  <br /> E. coli mutant strains. Several studies suggested            topological characteristics node degree diameter <br /> nodes higher degrees important              predictive information-carrying networks (e.g., the <br /> network, removal nodes biological networks              Internet, protein&#8211;protein interaction networks), results <br /> is likely lead lethal phenotype (15,16). To test          suggest characteristics synthetic accessi- <br /> this hypothesis, computed degree enzyme              bility appropriate transport directed networks, <br /> the number metabolites participating reactions cata-             metabolic networks. <br /> lyzed enzyme. A strain predicted inviable if <br /> the degree knocked-out enzyme certain <br />                                                                        Robustness synthetic accessibility <br /> cutoff. Fig. 2 shows optimized cutoff value, this <br /> procedure predicts viability worse random prediction.           Metabolic networks incomplete may <br />    Several theoretical studies focused graph diameter          contain errors. To study predictions using <br /> as measure network performance, defining graph                  synthetic accessibility depend errors network, <br /> diameter mean shortest paths pair             performed robustness analysis using E. coli <span id='am-93' about='obo:NCBITaxon_94025' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_27394 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_74974 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_74975 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-94' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="Meta" datatype="xsd:string"></span><span id='am-96'  ></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>meta</span>- <br /> nodes (15,25,26). To test graph diameter predictor             bolic network. Errors modeled random reassignment <br /> viability, predicted mutant inviable increase         certain percentages enzymes different reactions. Fig. <br /> graph diameter exceeded cutoff. Fig. 2 shows that, similar           5 shows accuracy prediction decreased with <br /> to node degree, graph diameter did perform better              increased fraction introduced mistakes. The method <br /> than random predictions.                                               tolerated assignment error rates 5&#8211;10%, accuracy <br />    Similarly, tested topology-based measure, <br /> enzyme usage, defined number times the <br /> reactions catalyzed enzyme used produce <br /> biomass wild-type strain according synthetic <br /> accessibility approach. Enzyme usage analogous node <br /> betweenness, number shortest paths between <br /> all pairs nodes node (22,23). Enzyme <br /> usage performed somewhat better random predictions <br /> but worse synthetic accessibility, surprising <br /> because basically used subset data produced the <br /> synthetic accessibility approach. <br />    In summary, popular topology-based measures performed <br /> more poorly synthetic accessibility. Moreover, node <br /> degree diameter accurate simply pre- <br /> dicting mutants viable, gives accu- <br /> racy 53.8%, node usage performed better                FIGURE 5 Accuracy synthetic accessibility approach a <br /> than node degree diameter, worse predictor           percentage enzyme-reaction assignments shuffled. To assess robust- <br />                                                                        ness synthetic accessibility method errors topology the <br /> the synthetic accessibility (see Supplementary Material). <br />                                                                        E. coli metabolic network, randomly shuffle given percentage the <br />    These characteristics ignore essential properties              assignments enzymes reactions calculate accuracy of <br /> metabolic network, directionality branching reactions,          synthetic accessibility method 10 trials. We plot average accuracy <br /> and directed transport material cellular substrates            percentage assignments shuffled, error bars noting the <br /> (sugars, oxygen, etc.) products (biomass). Synthetic                minimum maximum observed accuracy. The horizontal line denotes the <br />                                                                        accuracy predicting mutants viable, best expected result a <br /> accessibility, contrast, takes account properties <br />                                                                        random network. The approach relatively robust random errors the <br /> of metabolic network. As such, synthetic accessibility             enzyme-reaction assignments, clear expected trend <br /> can thought generalization concept graph          lower accuracy great variability accuracy number of <br /> diameter directed transport networks. Although certain             shuffled assignments increases. <br />  <br />                                                                                                          Biophysical Journal 91(6) 2304&#8211;2311 <br />  2310                                                                                                              Wunderlich Mirny <br />  <br />  <br /> dropped level random predictions ;50%               tary Material) demonstrates well-known complexities metab- <br /> enzyme-reaction assignments shuffled.                            olism: metabolic pathway used produce specific <br />                                                                       product shortest one; be <br />                                                                       completely characterized sets input output meta- <br /> DISCUSSION <br />                                                                       bolites. Similar rates failure flux balance techniques <br /> In study, topology function          suggest importance regulation adaptation muta- <br /> metabolic network intimately related. By introducing            tions possible role undiscovered metabolic and <br /> novel topology-based measure, synthetic accessibility,             transport reactions. <br /> were able correctly predict viability 443 598 mutant            We explore popular network characteristics <br /> E. coli strains comprehensive, reliable data set (21)          graph diameter, node degree, betweenness (usage) <br /> and 3477 4154 mutant yeast strains grown             predictors mutant viability. Our results demonstrate that <br /> conditions (13,14). Synthetic accessibility, S, essentially      characteristics fail predict mutants&#8217; viability. We <br /> network diameter specifically tailored transport networks,        conclude, agreement recent similar study (28), that <br /> and increase S correlated inviable        node degree used predict viability metabolic <br /> phenotype. A significant increase S mutation suggests           knockout strains. <br /> increased metabolic costs, leading reduction growth            The lack predictive utility node degree graph <br /> rate death. The apparent success synthetic accessibility        diameter metabolic networks easy understand. Both <br /> can attributed contribution network                 concepts widely applied information exchange <br /> topology information used                networks, Internet social networks, every <br /> these predictions.                                                    pair nodes potentially interact. On contrary, the <br />    Synthetic accessibility rapidly computed given        metabolic network transport network products are <br /> network, adjustable parameters, and, contrast            synthesized set initial substrates. Performance of <br /> FBA, MOMA, EMA, does require knowledge                 network determined ability synthesize pro- <br /> stoichiometry maximal uptake rates metabolic               ducts, hence, paths inputs final products of <br /> transport reactions. On E. coli insertional data set,         central importance, contrast diameter, pair of <br /> accuracy synthetic accessibility approach comparable        nodes considered. Because chemical reactions require <br /> to FBA MOMA. The performance synthetic                substrate yield product, linear path used in <br /> accessibility compared FBA EMA smaller E.            information networks needs replaced tree all <br /> coli data set worse, smaller data set data   required substrates. Considering aspects naturally leads to <br /> mutants affecting central metabolism            concept synthetic accessibility study metabolic and <br /> be biased, large data set insertional mutants       similar transport networks, e.g., signaling networks, are <br /> fairly unbiased representative. Synthetic accessibility           webs reactions, input chemical or <br /> also performs comparably FBA yeast data sets.               physical stimulus output group chemical re- <br />    Unlike FBA, synthetic accessibility does assume           sponses stimulus. Synthetic accessibility defined way <br /> optimality regard biomass production. But model           generalization graph diameter directed, branching <br /> assumes long rerouted fluxes efficient native      chemical reactions input&#8211;output transport network. <br /> ones, predicting mutants longer fluxes (larger synthetic            In summary, topology metabolic <br /> accessibility) inviable. Although assumption fails         network central determining viability mutant <br /> certain cases, similar success rates FBA ap-           strains, success widely used flux balance techniques <br /> proach suggest assumption holds true vast ma-           predicting viability primarily attributed to <br /> jority mutant strains. We conclude, agreement recent       topology. The addition stoichiometric parameters <br /> studies (24,27), rerouting does contribute signifi-          does significantly improve accuracy predictions, <br /> cantly robustness knockout mutants.                             used FBA predict fluxes in- <br />    Similar accuracy achieved techniques based flux              dividual reactions. We introduce concept synthetic <br /> balance synthetic accessibility points network topol-      accessibility, allows fast, accurate, easily inter- <br /> ogy primary determinant viability predictions          pretable analysis metabolic networks. Our results suggest <br /> FBA MOMA. Although results suggest network               rerouting metabolic fluxes plays minimal role in <br /> topology sufficient predict strain viability use   providing viability mutant strains. Importantly, results <br /> of stoichiometric coefficients flux balances does im-         strongly support central role network topology in <br /> prove prediction accuracy, detailed prediction fluxes     determining phenotypes biological systems. <br /> in individual reactions FBA/MOMA does require knowl- <br /> edge stoichiometric coefficients maximal uptake rates. <br />                                                                       SUPPLEMENTARY MATERIAL <br />    Importantly, flux balance synthetic accessibility <br /> fail predict viability significant number mutants.         An online supplement article visiting <br /> Analysis incorrect predictions E. coli (see Supplemen-          BJ Online http://www.biophysj.org. <br />  <br /> Biophysical Journal 91(6) 2304&#8211;2311 <br />  Topology-Based Metabolic Predictions                                                                                                                    2311 <br />  <br /> We thank V. Spirin help processing raw data comments        13. Giaever, G., A. M. Chu, L. Ni, C. Connelly, L. Riles, S. Veronneau, <br /> the manuscript, M. Kardar suggestions testing robustness,           S. Dow, A. Lucau-Danila, K. Anderson, B. Andre, A. P. Arkin, <br /> and V. Berube M. Slutsky comments manuscript. We                  A. Astromoff, et al. 2002. Functional profiling Saccharomyces <br /> thank A. Trusina K. Sneppen useful discussions.                               cerevisiae genome. Nature. 418:387&#8211;391. <br />                                                                                   14. Steinmetz, L. M., C. Scharfe, A. M. Deutschbauer, D. Mokranjac, <br /> Z.W. recipient Howard Hughes Medical Institute Predoctoral <br />                                                                                       Z. S. Herman, T. Jones, A. M. Chu, G. Giaever, H. Prokisch, <br /> Fellowship. L.A.M. Alfred P. Sloan Research Fellow.                             P. J. Oefner, R. W. Davis. 2002. Systematic screen human <br />                                                                                       disease genes yeast. Nat. Genet. 31:400&#8211;404. <br />                                                                                   15. Albert, R., H. Jeong, A. L. Barabasi. 2000. Error attack <br />                                                                                       tolerance complex networks. Nature. 406:378&#8211;382. <br /> REFERENCES                                                                        16. Jeong, H., S. P. Mason, A. L. Barabasi, Z. N. Oltvai. 2001. <br />                                                                                       Lethality centrality protein networks. Nature. 411:41&#8211;42. <br />  1. Varma, A., B. O. Palsson. 1994. Metabolic flux balancing&#8212;basic <br />     concepts, scientific practical use. Biotechnology. 12:994&#8211;998.            17. Myatt, G. J. 1994. Computer aided estimation synthetic accessibility. <br />                                                                                       PhD Thesis. University Leeds, Leeds, UK. <br />  2. Edwards, J. S., B. O. Palsson. 1999. Systems properties the <br />     <span id='am-40' about='obo:NCBITaxon_727' typeof='owl:Thing obo:NCBITaxon_135625 obo:NCBITaxon_712 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_724'><span id='am-41'  ></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Haemophilus meningitidis" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium influenzae" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Coccobacillus pfeifferi" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Influenza-bacillus" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="Haemophilus influenzae" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="Bacterium influenzae" datatype="xsd:string"></span>Haemophilus influenzae</span> Rd metabolic genotype. J. Biol. Chem.                  18. Handorf, T., O. Ebenhoh, R. Heinrich. 2005. Expanding metabolic <br />     274:17410&#8211;17416.                                                                  networks: scopes compounds, robustness, evolution. J. Mol. <br />                                                                                       Evol. 61:498&#8211;512. <br />  3. Edwards, J. S., B. O. Palsson. 2000. Metabolic flux balance <br />     analysis silico analysis <span id='am-109' about='obo:NCBITaxon_83333' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-110' property="oboInOwl:hasExactSynonym" content="Escherichia coli K12" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-113' property="rdfs:label" content="Escherichia coli K-12" datatype="xsd:string"></span>Escherichia coli K-12</span> gene             19. Neidhardt, F. C., H. E. Umbarger. 1996. Chemical composition of <br />     deletions. BMC Bioinformatics. 1:1.                                               Escherichia coli. In Escherichia coli <span id='am-80' about='obo:NCBITaxon_590' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-81' property="oboInOwl:hasDbXref" content="PMID:10939679" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasDbXref" content="PMID:10319519" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasDbXref" content="PMID:3231714" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="PMID:15653929" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="PMID:9731304" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-88'  ></span><span id='am-89' property="oboInOwl:hasDbXref" content="PMID:15653930" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasRelatedSynonym" content="Samonella" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasDbXref" content="PMID:12072558" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="Salmonella" datatype="xsd:string"></span>Salmonella</span>: Cellular and <br />                                                                                       Molecular Biology. F. C. Neidhardt R. Curtis, editors. ASM Press, <br />  4. Edwards, J. S., B. O. Palsson. 2000. The Escherichia coli MG1655              Washington, DC. 13&#8211;16. <br />     silico metabolic genotype: <span id='am-103' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span>, characteristics, capabil- <br />     ities. Proc. Natl. Acad. Sci. USA. 97:5528&#8211;5533.                              20. Romero, P. R., P. Karp. 2001. Nutrient-related analysis of <br />                                                                                       pathway/genome databases. Pac. Symp. Biocomput. 471&#8211;482. <br />  5. Forster, J., I. Famili, P. Fu, B. O. Palsson, J. Nielsen. 2003. <br />     Genome-scale reconstruction Saccharomyces cerevisiae meta-             21. Gerdes, S. Y., M. D. Scholle, J. W. Campbell, G. Balazsi, E. Ravasz, <br />     bolic network. Genome Res. 13:244&#8211;253.                                            M. D. Daugherty, A. L. Somera, N. C. Kyrpides, I. Anderson, M. S. <br />                                                                                       Gelfand, A. Bhattacharya, V. Kapatral, et al. 2003. Experimental deter- <br />  6. Reed, J. L., T. D. Vo, C. H. Schilling, B. O. Palsson. 2003. An               mination level analysis essential genes Escherichia <br />     expanded genome-scale model Escherichia coli K-12 (iJR904 GSM/ <br />                                                                                       coli MG1655. J. Bacteriol. 185:5673&#8211;5684. <br />     GPR). Genome Biol. 4:R54. <br />                                                                                   22. Newman, M. E. 2001. Scientific collaboration networks. I. Network <br />  7. Burgard, A. P., C. D. Maranas. 2001. Probing performance <br />                                                                                       construction fundamental results. Phys. Rev. E. 64:016131. <br />     limits Escherichia coli metabolic network subject gene addi- <br />     tions deletions. Biotechnol. Bioeng. 74:364&#8211;375.                           23. Newman, M. E. 2001. Scientific collaboration networks. II. Shortest <br />                                                                                       paths, weighted networks, centrality. Phys. Rev. E. 64:016132. <br />  8. Duarte, N. C., M. J. Herrgard, B. O. Palsson. 2004. Reconstruction <br />     validation Saccharomyces cerevisiae iND750, fully compart-           24. Papp, B., C. Pal, L. D. Hurst. 2004. Metabolic network analysis of <br />     mentalized genome-scale metabolic model. Genome Res. 14:1298&#8211;1309.                causes evolution enzyme dispensability yeast. Nature. <br />                                                                                       429:661&#8211;664. <br />  9. Badarinarayana, V., P. W. Estep 3rd, J. Shendure, J. Edwards, S. Tavazoie, <br />     F. Lam, G. M. Church. 2001. Selection analyses insertional             25. Jeong, H., B. Tombor, R. Albert, Z. N. Oltvai, A. L. Barabasi. <br />     mutants using subgenic-resolution arrays. Nat. Biotechnol. 19:1060&#8211;               2000. The large-scale organization metabolic networks. Nature. <br />     1065.                                                                             407:651&#8211;654. <br /> 10. Segre, D., D. Vitkup, G. M. Church. 2002. Analysis optimality          26. Ma, H., A. P <br /> </body></html>