<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body><span id='am-12' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-22'  ></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> nucleolar protein <br /> Nop7p is necessary for biogenesis <br /> of 60S ribosomal subunits <br />  <br /> ABSTRACT <br /> To identify new gene products that participate in ribosome biogenesis, we carried out a screen for mutations that <br /> result in lethality in combination with mutations in DRS1, a <span id='am-29' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-30' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-35'  ></span>Saccharomyces</span> cerevisiae nucleolar DEAD-box protein <br /> required for synthesis of 60S ribosomal subunits. We identified the gene N0P7 that encodes an essential protein. The <br /> temperature-sensitive nop7-1 mutation or metabolic depletion of Nop7p results in a deficiency of 60S ribosomal <br /> subunits and accumulation of halfmer polyribosomes. Analysis of pre-rRNA processing indicates that nop7 mutants <br /> exhibit a delay in processing of 27S pre-rRNA to mature 25S rRNA and decreased accumulation of 25S rRNA. Thus <br /> Nop7p, like Drs1p, is required for essential steps leading to synthesis of 60S ribosomal subunits. In addition, inac- <br /> tivation or depletion of Nop7p also affects processing at the A0 , A1 , and A2 sites, which may result from the associ- <br /> ation of Nop7p with 35S pre-rRNA in 90S pre-rRNPs. Nop7p is localized primarily in the nucleolus, where most steps <br /> in ribosome assembly occur. Nop7p is homologous to the zebrafish pescadillo protein necessary for embryonic <br /> development. The Nop7 protein contains the BRCT motif, a protein&#8211;protein interaction domain through which, for <br /> example, the human BRCA1 protein interacts with RNA helicase A. <br /> Keywords: BRCT motif; DEAD-box proteins; nucleolus; ribosome biogenesis; synthetic lethality; yeast <br />  <br />  <br />  <br /> INTRODUCTION                                                                proteins to form mature 40S and 60S ribosomal sub- <br />                                                                             units+ These concerted pathways of rRNA processing <br /> Ribosomal RNA processing and ribosome assembly <br />                                                                             and ribosome assembly require a number of nonribo- <br /> have been studied in many different eukaryotes (re- <br />                                                                             somal trans-acting factors, including more than 100 <br /> viewed in Eichler &amp; Craig, 1994), but have been best <br />                                                                             small RNAs and at least 70 different proteins (Kressler <br /> characterized in the yeast Saccharomyces cerevisiae <br />                                                                             et al+, 1999; Venema &amp; Tollervey, 1999)+ The majority of <br /> (reviewed in Kressler et al+, 1999; Venema &amp; Tollervey, <br />                                                                             these molecules reside in the nucleolus, the nuclear <br /> 1999)+ The yeast ribosome consists of the 40S subunit, <br />                                                                             subcompartment where most of the steps in ribosome <br /> which contains the 18S rRNA and 33 different proteins, <br />                                                                             biogenesis occur+ <br /> and the 60S subunit, which contains the 25S, 5+8S, and <br />                                                                                An important <span id='am-70' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-71' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> of nucleolar proteins necessary <br /> 5S rRNAs and 46 different proteins+ The 18S, 5+8S, and <br />                                                                             for ribosome biogenesis is the DEAD-box <span id='am-50' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-51' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> of pu- <br /> 25S rRNAs are transcribed by RNA polymerase I as a <br />                                                                             tative RNA helicases (Fuller-Pace, 1994; de la Cruz <br /> single 35S precursor, whereas 5S rRNA is transcribed <br />                                                                             et al+, 1999)+ Seventeen different DEAD-box proteins <br /> independently from separate genes by RNA polymer- <br />                                                                             (DBPs) have been implicated in biogenesis of yeast <br /> ase III+ The 35S rRNA precursor undergoes sequential <br />                                                                             ribosomes+ The precise functions of these nucleolar <br /> endo- and exonucleolytic cleavages to generate the <br />                                                                             DBPs are not known, but might include rearranging <br /> mature rRNAs+ Concomitant with its synthesis and pro- <br />                                                                             RNA structure to provide access to nucleases, catalyz- <br /> cessing, rRNA is covalently modified by methylation <br />                                                                             ing association or dissociation of snoRNAs and rRNAs, <br /> and pseudouridylation and assembles with ribosomal <br />                                                                             or altering conformation of nucleolar RNA, protein, or <br />                                                                             RNP complexes to permit association or dissociation of <br />   Reprint requests to: John L+ Woolford, Jr+, Department of Bio-            other assembly factors or ribosomal proteins (reviewed <br /> logical Sciences, Carnegie Mellon University, 616 Mellon Institute,         in Schwer, 2001; Tanner &amp; Linder, 2001)+ Nucleolar <br /> 4400 Fifth Avenue, Pittsburgh, Pennsylvania 15213, USA; e-mail: <br /> jw17@andrew+cmu+edu+                                                        DBPs might function in multiprotein complexes, as has <br />   *These authors contributed equally to <span id='am-80' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-81' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-82'  ></span><span id='am-83' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>this</span> work+                          been shown for several other helicases involved in <br />                                                                       150 <br />  Yeast nucleolar protein containing a BRCT motif                                                                              151 <br />  <br /> mRNA processing or turnover (Margossian et al+, 1996; <br /> Mitchell et al+, 1997; Vanzo et al+, 1998)+ The partners <br /> of these helicases include proteins that stimulate heli- <br /> case activity of the DBPs and nucleases that may re- <br /> quire the helicases to alter RNA secondary structure <br /> prior to degradation+ Therefore, to understand the func- <br /> tion of nucleolar DBPs, it is important to identify their <br /> protein partners+ <br />    We previously identified the nucleolar DBP Drs1p, <br /> which is necessary for biogenesis of the 60S ribosomal <br /> subunit (Ripmaster et al+, 1992)+ To identify other mol- <br /> ecules that function together with Drs1p, we carried out <br /> a genetic screen for mutations that are synthetically <br /> lethal with drs1 mutations+ Such synthetic lethal screens <br /> have been especially useful in studies of pathways of <br /> RNA processing (Frank et al+, 1992; Liao et al+, 1993; <br /> Berg&#232;s et al+, 1994; Simos et al+, 1996; Venema &amp; Tol- <br /> lervey, 1996; Venema et al+, 1997; Xu et al+, 1998) or in <br /> studies of multimolecular complexes (Fabre et al+, 1994; <br /> Doye &amp; Hurt, 1995)+ Our screen led to the isolation of <br /> three genes whose protein products play a role in ribo- <br /> some biogenesis, and thus may functionally interact <br /> with Drs1p+ In this report, we describe one of these <br /> genes, NOP7+ We show that Nop7p is an essential, <br /> conserved protein located in the nucleolus+ The nop7-1 <br /> mutation or depletion of Nop7p results in a deficiency <br /> of 60S ribosomal subunits, a kinetic delay in process- <br /> ing of 27S pre-rRNAs to 25S rRNA, and a decrease in <br /> 25S rRNA+ Interestingly, Nop7p contains a BRCT motif <br /> thought to function in other proteins as a structural do-    FIGURE 1. Mutant alleles of DRS1+ Amino acids changed in each of <br />                                                              the 10 new mutant alleles of DRS1, plus the original drs1-1 mutation, <br /> main for protein&#8211;protein interactions, including inter-      are indicated in single-letter code above each altered amino acid+ <br /> actions with an RNA helicase (Anderson et al+, 1998;         The drs1 allele number is indicated above the mutated amino acid+ <br /> Zhang et al+, 1998)+                                         Each of the conserved motifs present in the core helicase domains <br />                                                              of the DEAD-box family is indicated in bold face+ <br />  <br />  <br /> RESULTS <br />                                                              none of the mutations change any of the highly con- <br /> Screen for mutations synthetically lethal <br />                                                              served amino acids, several alter amino acids imme- <br /> with mutations in the nucleolar DEAD-box <br />                                                              diately adjacent to the GKT, PTRELA, SAT, or HRVGR <br /> protein Drs1p <br />                                                              motifs+ Each drs1 mutant is deficient in 60S ribosomal <br /> We expected that the spectrum of mutations that would        subunits and accumulates halfmer polyribosomes (<span id='am-113' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-114' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-115'  ></span><span id='am-116' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-117' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> <br /> cause lethality in combination with a drs1 mutation would    not shown)+ Pulse-chase analysis was performed to <br /> depend upon the particular mutant allele of drs1+ Pre-       analyze rRNA processing in the drs1-5 and drs1-11 <br /> viously, we had identified one mutant allele of DRS1,        mutants+ As observed for the drs1-1 mutant, both the <br /> drs1-1. Therefore, to increase the likelihood of identi-     drs1-5 and drs1-11 mutants accumulate the 27SB pre- <br /> fying different gene products interacting with Drs1p, we     rRNA after 60 min at 13 8C, fail to produce additional <br /> first isolated additional drs1 mutants+ PCR mutagen-         25S rRNA, but generate wild-type levels of 18S rRNA, <br /> esis and gap repair in vivo were used to generate mu-        consistent with a specific defect in biogenesis of 60S <br /> tations in DRS1, screening for cold-sensitive (Cs 2 ) or     ribosomal subunits (Fig+ 2 and data not shown)+ <br /> temperature-sensitive (Ts 2 ) conditional lethal mutant         To screen for mutations lethal in combination with <br /> strains+ We identified the drs1-2&#8211;drs1-11 mutants, all of    drs1 mutations, we constructed ade2 ade3 leu2 ura3 <br /> which are cold-sensitive for growth at 13 8C+ Six of these   drs1D haploid yeast containing a CEN URA3 ADE3 <br /> mutants contain single mutations in DRS1, and the re-        DRS1 plasmid and a CEN LEU2 plasmid bearing either <br /> maining four contain two or more mutations in DRS1           drs1-2 or drs1-5+ We specifically chose the drs1-2 and <br /> (Fig+ 1)+ All of the mutations lie within the central core   drs1-5 alleles because they contain single mutations <br /> domain of DRS1 containing the conserved motifs of            near two different conserved DBP motifs+ drs1-2 con- <br /> the DEAD-box family (de la Cruz et al+, 1999)+ Although      tains a Leu414Ser mutation adjacent to the SAT motif, <br />  152                                                                                                                   C.C. Adams et al. <br />  <br />  <br />  <br />  <br />         FIGURE 2. A: 27SB pre-rRNA accumulates and 25S rRNA is not produced in drs1-1 and drs1-5 cold-sensitive mutants+ <br />         Synthesis, processing, and accumulation of large molecular weight rRNA processing intermediates and products were <br />         assayed in the drs1-1 and drs1-5 mutants (lanes 6&#8211;10 and 16&#8211;20, respectively) or in the wild-type DRS1 strain (lanes 1&#8211;5 <br />         and 11&#8211;15) grown at 30 8C in YEPGlu and shifted to 13 8C for 5 h+ Cells were pulse-labeled with [methyl- 3 H]-methionine for <br />         5 min and chased with excess cold methionine for 2, 5, 10, and 60 min+ RNA was extracted and analyzed as described in <br />         Materials and Methods+ The rRNA primary transcript (35S), processing intermediates (32S, 27SA, 27SB, 20S), and products <br />         (18S, 25S) are labeled+ Identical results were obtained with drs1-11 (data not shown)+ We reproducibly observed slightly <br />         slower rates of processing of all rRNAs in the wild-type strains shifted to 13 8C compared to those grown at 30 8C+ <br />         B: Simplified scheme for steps in processing of rRNA+ <br />  <br />  <br />  <br />  <br /> and drs1-5 contains a Tyr563Cys mutation near the                       other drs1 mutant alleles (Fig+ 3)+ Further character- <br /> HRVGR motif+ After mutagenesis, nonsectoring 5-FOAs                     ization of sl3 and the gene defined by this mutation is <br /> colonies were identified that contained a chromosomal                   described below+ <br /> mutation causing the DRS1 plasmid to be essential+ <br /> One dominant synthetic lethal mutation (SL1) was <span id='am-103' about='obo:NCBITaxon_238707' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_8075 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_1489917'><span id='am-104'  ></span><span id='am-105' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-107' property="rdfs:label" content="Iso" datatype="xsd:string"></span>iso</span>- <br /> lated using the drs1-2 strain and two recessive syn- <br /> thetic lethal mutations (sl2 and sl3) were isolated using <br /> the drs1-5 strain+ Each synthetic lethal mutant strain <br /> was Ts 2 for growth at 37 8C+ In crosses of either reces- <br /> sive sl2 or sl3 mutant to a wild-type strain, the Sl 2 and <br /> the Ts 2 phenotypes co-segregated 2 1 :2 2 , indicating <br /> that each mutant contains a single mutation in a nu- <br /> clear gene responsible for both the Sl 2 and the Ts 2 <br /> phenotypes+ Complementation was observed when <br /> strains bearing the two recessive mutations sl2 and sl3 <br /> were mated to each other; therefore sl2 and sl3 repre-                  FIGURE 3. The nop7-1 mutation causes lethality only in combina- <br /> sent mutations in distinct genes+                                       tion with drs1-3, drs1-5, or drs1-11 mutant alleles+ Yeast strain <br />    To determine whether these synthetic lethal pheno-                   JWY5464 (drs1D ade-2-1 ade3::hisG ura3-1 leu-2-3,112 nop7-1 plus <br />                                                                         plasmid pJW3956 bearing URA3, ADE3, CEN and DRS1 ) was trans- <br /> types were allele specific, we introduced <span id='am-128' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-129' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-131' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>plasmids</span> <span id='am-98' about='obo:NCBITaxon_201850' typeof='owl:Thing obo:NCBITaxon_1125798 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71529 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-99' property="rdfs:label" content="Car" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-101'  ></span><span id='am-102' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>car</span>-                 formed with CEN LEU2 plasmid vector pRS315 or pRS315 bearing <br /> rying each of the 11 different drs1 mutant alleles into                 each of the drs1 alleles drs1-1 through drs1-11, wild-type DRS1, or <br /> each synthetic lethal mutant strain+ Whereas SL1 was                    &#8220;empty&#8221; vector+ Four independent transformants bearing each plas- <br />                                                                         mid were replica-plated to C-leu medium containing 1 mg/mL 5-<span id='am-7' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10'  ></span><span id='am-11' property="rdfs:label" content="Foa" datatype="xsd:string"></span>FOA</span>, <br /> lethal in combination with all alleles of DRS1 tested                   as shown+ Cells that could not grow on these 5 FOA-containing <br /> (data not shown), sl2 and sl3 displayed allele specific-                plates (the drs1-3, drs-5, and drs1-11 strains) are unable to lose the <br /> ity+ sl2 was lethal only in combination with drs1-3, drs1-5,            URA3 ADE3 DRS1 plasmid, indicating that the nop7-1 mutation is <br />                                                                         lethal in combination with the specific plasmid-borne drs1 mutant <br /> drs1-10, and drs1-11 (data not shown), and sl3 was                      allele+ Likewise, the strain carrying the empty vector pRS315 cannot <br /> lethal with drs1-3, drs1-5, and drs1-11 but not with the                lose the DRS1 plasmid+ <br />  Yeast nucleolar protein containing a BRCT motif                                                                                        153 <br />  <br /> NOP7 encodes a novel, essential protein                                 Upon sporulation and tetrad dissection, His 1 spores <br />                                                                         were recovered and were always Leu 1 (data not shown)+ <br /> Transformation of a yeast genomic library into the sl3 <br />                                                                         Thus NOP7 is an essential gene+ <br /> mutant and complementation of its synthetic lethal pheno- <br /> type yielded ORF YGR103w+ Subsequent subcloning, <br /> mutagenesis, and linkage analysis demonstrated that <br />                                                                         Inactivation or depletion of Nop7p results in <br /> YGR103w corresponds to the gene defined by the sl3 <br />                                                                         a deficiency of 60S ribosomal subunits <br /> mutation (see Materials and Methods)+ We named this <br /> gene NOP7, for nucleolar protein 7 (see below), and                     To investigate whether Nop7p is essential for pro- <br /> designated the sl3 mutation nop7-1+                                     duction of ribosomes, we assayed the amounts of <br />   To determine whether NOP7 is an essential gene, we                    ribosomal subunits, 80S monoribosomes, and poly- <br /> used a one-step PCR strategy to precisely replace one                   ribosomes in extracts prepared from the nop7-1 Ts 2 <br /> copy of the wild-type NOP7 open reading frame with                      mutant grown at 23 8 and shifted from 23 8C to 37 8C for <br /> the deletion-insertion allele nop7D::HIS3, in diploid strain            5 h+ We also assayed ribosome production in a GAL- <br /> JWY1401 homozygous for the his3-D200 deletion                           NOP7 strain metabolically depleted of Nop7p+ In this <br /> (Baudin et al+, 1993)+ Genomic southern blotting dem-                   strain, the chromosomal copy of NOP7 was disrupted <br /> onstrated that the expected gene replacement had oc-                    and wild-type NOP7, under the control of a GAL pro- <br /> curred+ Upon sporulation of the nop7D::HIS3/NOP7                        moter, was supplied on a plasmid+ This strain grew in <br /> diploid and dissection of tetrads, two viable His 2 spores              solid or liquid YEPGal at the same rate as a wild-type <br /> and no viable His 1 spores were recovered in each of                    NOP7 strain (Fig+ 4A,B), indicating that the GAL-NOP7 <br /> 20 tetrads (data not shown)+ The diploid was trans-                     construct is fully functional+ However, the GAL-NOP7 <br /> formed with a CEN plasmid bearing NOP7 and LEU2+                        strain did not form colonies on YEPGlu solid medium <br />  <br />  <br />  <br />  <br />         FIGURE 4. Growth of yeast cells is impaired upon depletion of Nop7p+ A: Yeast strains JWY5853 (nop7D::HIS3 plus <br />         pRS315 containing NOP7 ) and JWY6901 (nop7D::HIS3 plus pBM258T containing GAL-NOP7 ) were streaked on YEPGal <br />         or YEPGlu medium and incubated for 4 days at 30 8C+ B: Liquid growth curve of GAL-NOP7 strain JWY6901 grown in <br />         YEPGal or after shifting to YEPGlu medium+ OD600 values are plotted as log[ODt /ODt 0 ], where ODt 0 is the initial OD600 <br />         reading at time 0 before shifting from YEPGal to YEPGlu and ODt is the OD at time t after the shift+ Cell cultures were kept <br />         at OD600 from 0+2&#8211;0+8 to maintain logarithmic phase growth+ C: NOP7 mRNA is depleted upon shifting the GAL-NOP7 <br />         JWY6901 strain from YEPGal to YEPGlu medium+ NOP7 mRNA was assayed by northern blotting using 32 P-labeled NOP7 <br />         ORF as a probe+ U3 snoRNA was used as a loading control+ D: Nop7p is depleted upon shifting the GAL-NOP7 strain <br />         JWY6901 from YEPGal to YEPGlu medium+ Rabbit polyclonal antibodies against a peptide from Nop7p were used to assay <br />         Nop7p+ Yeast PGK was the loading control, assayed using rabbit antibodies+ <br />  154                                                                                                                   C.C. Adams et al. <br />  <br /> (Fig+ 4A) and exhibited a decreased growth rate in liq-                 some profile (Fig+ 5A)+ Likewise, both the wild-type NOP7 <br /> uid media 24 h after shifting from YEPGal to YEPGlu                     strain and the GAL-NOP7 strain exhibited wild-type poly- <br /> (Fig+ 4B)+ NOP7 mRNA was no longer detectable 1 h                       ribosome profiles after growth in galactose-containing <br /> after the media shift (Fig+ 4C) and Nop7 protein was                    medium (Fig+ 5B)+ However, 5 h after shifting the nop7-1 <br /> not detectable by 12 h after the shift (Fig+ 4D)+ The                   strain from 23 8C to 37 8C, and 4 h or 14 h after shifting <br /> nop7-1 strain grown at 23 8C exhibited a wild-type ribo-                the GAL-NOP7 strain from YEPGal to YEPGlu, the num- <br />  <br />  <br />  <br />  <br />        FIGURE 5. The nop7-1 mutation and depletion of Nop7p result in a deficiency of free 60S ribosomal subunits and accu- <br />        mulation of halfmer polyribosomes+ A: Extracts were prepared from the nop7-1 strain JWY5464 grown in YEPGlu at 23 8C <br />        to 4{10 7 cells/mL (left), or grown at 23 8C to 3{10 7 cells/mL and shifted to 37 8C for 5 h (right)+ Cell lysates containing <br />        ribosomes and polyribosomes were separated by centrifugation on 7% to 47% sucrose velocity gradients+ Peaks repre- <br />        senting free 40S and 60S ribosomal subunits and 80S monosomes are labeled+ Fractions containing halfmer polyribosomes <br />        are indicated by vertical arrows+ B: The GAL-NOP7 strain JWY6901 was grown in YEPGal at 30 8C to 5{10 7 cells/mL (left) <br />        and shifted to YEPGlu at 30 8C for 14 h (right)+ Ribosomes and polyribosomes were resolved as described above+ <br />  Yeast nucleolar protein containing a BRCT motif                                                                   155 <br />  <br /> ber of free 60S ribosomal subunits relative to free 40S      HA-NOP7 strain, but not in the untagged NOP7 strain <br /> ribosomal subunits decreased and halfmer polyribo-           (data not shown)+ Indirect immunofluorescence micros- <br /> somes accumulated (Fig+ 5 and data not shown)+ The           copy demonstrated that HA-tagged Nop7p is localized <br /> effect on ribosome production in the GAL-NOP7 strain         to a crescent- or cap-shaped region largely distinct from <br /> was greater 14 h after the shift than 4 h, consistent with   the DAPI-stained DNA in the nucleoplasm (Fig+ 6)+ A <br /> the observation that Nop7p is only completely depleted       small amount of Nop7p is reproducibly observed in the <br /> by 12 or more hours after the shift+                         nucleoplasm+ Thus Nop7p is located within the nucle- <br />    The peaks of halfmer polyribosomes observed in the        olus of yeast where ribosomes are assembled, rather <br /> nop7 mutants contain mRNAs associated with integral          than in the cytoplasm, where ribosomes function in <br /> numbers of ribosomes plus a 43S preinitiation com-           protein synthesis+ This result indicates that Nop7p most <br /> plex, stalled at the AUG initiator <span id='am-90' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-91' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-93'  ></span><span id='am-94' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span> (Helser et al+,     likely is a nonribosomal factor necessary for biogen- <br /> 1981; Rotenberg et al+, 1988)+ Such halfmers accumu-         esis of 60S subunits, rather than a component of ma- <br /> late when translation initiation is blocked at the 60S       ture ribosomes+ <br /> subunit joining step (Kang &amp; Hershey, 1994; Eisinger <br /> et al+, 1997), or when 60S ribosomal subunit assembly <br />                                                              Processing of 27S pre-rRNA is delayed <br /> is blocked (Rotenberg et al+, 1988; Moritz et al+, 1990, <br />                                                              and production of 25S rRNA is decreased <br /> 1991; Ripmaster et al+, 1992, 1993; Deshmukh et al+, <br />                                                              in nop7 mutants <br /> 1993; Sun &amp; Woolford, 1994; Hong et al+, 1997; Vilardell <br /> &amp; Warner, 1997; Weaver et al+, 1997; Zanchin et al+,         To characterize in more detail the role of Nop7p in the <br /> 1997; Daugeron &amp; Linder, 1998; de la Cruz et al+, 1998a,     production of 60S ribosomal subunits, we assayed rRNA <br /> 1998b; Kressler et al+, 1998; Ho &amp; Johnson, 1999)+ The       processing in the nop7-1 mutant and upon depletion of <br /> accumulation of halfmers in the nop7 mutants most            Nop7p in the GAL-NOP7 strain+ We did so using pulse- <br /> likely resulted from a block in 60S ribosomal subunit        chase labeling experiments with [methyl- 3H]-methionine <br /> assembly, as the peak of free 60S ribosomal subunits         and [5,6- 3 H]-uracil, and by steady-state measurements <br /> was reduced in the nop7-1 mutant or upon depletion of        via northern hybridization and primer extension+ The <br /> Nop7p+ We confirmed that there is a deficit of 60S           nop7-1 mutant strain and the NOP7 wild-type strain <br /> ribosomal subunits upon depletion of Nop7p, by quan-         grown overnight at 23 8C were shifted to 37 8C for 2 h <br /> tifying total ribosomal subunits from extracts subjected     and pulse-labeled for 5 min with [methyl- 3H]-methionine, <br /> to centrifugation on high salt sucrose gradients+ The        then chased with an excess of cold methionine for 2, 5, <br /> ratio of 60S subunits to 40S subunits monitored by           10, and 60 min+ In the wild-type NOP7 strain, most of <br /> absorbance at 254 nM was 2+4 in wild-type strains com-       the 27SA precursors were converted to 27SB pre- <br /> pared to 1+3 in the Nop7p-depleted strain (data not          rRNAs after the 5-min pulse, and little 27SA or 27SB <br /> shown)+ Taken together, these results indicate that in-      precursors remained by the 5-min chase (Fig+ 7A)+ In <br /> activation or depletion of Nop7p affects the assembly        contrast, the nop7-1 mutant exhibited a greater amount <br /> of ribosomes rather than their function+ These effects       of 27SA precursor than 27SB precursor after the 5-min <br /> on 60S ribosomal subunits most likely are a direct re-       pulse+ Significant amounts of both of these precursors <br /> sult of inactivation or depletion of Nop7p, rather than      were detected after the 2-min chase, and more 27SB <br /> being an indirect result of impaired growth rate+ The        precursor RNA was present after the 5-min chase than <br /> 60S ribosomal subunits were decreased before any             in the NOP7 strain (Fig+ 7A)+ By the 60-min chase point, <br /> discernible decrease in the rate of division of the nop7-1   substantially less 25S rRNA accumulated relative to <br /> or GAL-NOP7 strains+                                         18S rRNA in the nop7-1 strain compared to the NOP7 <br />                                                              strain+ The decreased production of 25S rRNA relative <br />                                                              to 18S rRNA was quantified by direct measurement of <br /> Nop7p is a nucleolar protein <br />                                                              the radioactivity in these RNAs+ The ratio of 25S to 18S <br /> Most steps in ribosome biogenesis occur in the nucle-        rRNA in the nop7-1 mutant 60 min after the chase was <br /> olus, whereas ribosome function occurs in the cyto-          0+89, compared to a ratio of 2+02 in the wild-type NOP7 <br /> plasm+ Therefore, to distinguish further whether Nop7p       strain+ We also observed a small amount of 35S and <br /> is required for the assembly versus the function of 60S      32S pre-rRNA at the 2-min chase point in the nop7-1 <br /> ribosomal subunits, we determined the subcellular lo-        mutant but not in wild-type NOP7 strain (Fig+ 7A, com- <br /> calization of Nop7p+ To do so, we constructed strain         pare lanes 1 and 2 with lanes 6 and 7)+ There was no <br /> JWY6700 in which the only functional Nop7p protein           difference in the rate of formation of 18S rRNA from the <br /> contains a triple HA epitope tag near its carboxyl ter-      20S rRNA intermediate, or in the amount of 18S rRNA <br /> minus+ This strain grew at the same rate as a control        produced in the nop7-1 strain versus the NOP7 strain <br /> strain containing untagged wild-type NOP7 (data not          (Fig+ 7A)+ Similar results were obtained upon depletion <br /> shown), indicating that the HA-tagged Nop7p is func-         of Nop7p in the GAL-NOP7 strain (data not shown)+ <br /> tional+ Western immunoblotting demonstrated that a              To exclude a defect in rRNA methylation and to as- <br /> polypeptide of the expected size was present in the          say the synthesis of low molecular weight 5S and 5+8S <br />  156                                                                                                           C.C. Adams et al. <br />  <br />  <br />  <br />  <br />        FIGURE 6. HA-tagged Nop7p is localized in the nucleolus+ Indirect immunofluorescence microscopy was carried out with <br />        strain JWY6700 (nop7D::HIS3 plus plasmid pRS315 containing CEN LEU2 and triple HA-tagged NOP7 )+ A: DNA was <br />        stained with 49, 6-diamidino-2-phenylindole dihydrochloride (DAPI)+ B: Nomarski image+ C: HA-Nop7p was detected with <br />        mouse monoclonal antibody 16B12 directed against the influenza hemagglutinin epitope, followed by decoration with a <br />        rhodamine-conjugated rabbit anti-mouse IgG+ D: Merged image+ Pseudocolors were assigned to the digitized micrographs <br />        (red 5 rhodamine; green 5 DAPI)+ <br />  <br />  <br />  <br />  <br /> rRNAs and tRNAs, we performed pulse-chase analysis                 the nop7-1 mutant, there was a slight delay in produc- <br /> using [5,6- 3 H]-uracil with NOP7 and nop7-1 strains 2 h           tion of the 7S rRNA processing intermediate derived <br /> after shifting from 23 8C to 37 8C, and with NOP7 and              from the 27SB rRNA, and a slight delay in production of <br /> GAL-NOP7 strains 14 h after shifting from YEPGal to                5+8S rRNA from 7S rRNA (Fig+ 7B and data not shown)+ <br /> YEPGlu medium+ As observed with the [methyl 23 H]-                 Synthesis of 5S rRNA and tRNAs was not affected in <br /> methionine pulse-chase experiments, processing of 27S              the nop7-1 mutant or upon depletion of Nop7p (Fig+ 7B <br /> rRNA to 25S rRNA was delayed and 25S rRNA accu-                    and data not shown)+ The absence of effects on pro- <br /> mulated at lower levels in the nop7 mutants compared               duction of 5S rRNA and tRNAs, as well as on methyl- <br /> to the wild-type NOP7 control strains (data not shown)+            ation of rRNA, indicate that the effects of nop7 mutants <br /> The ratio of 25S to 18S rRNA was 1+14 after the chase              on 27S rRNA processing are specific+ The accumula- <br /> in the Nop7p-depleted strain compared to 2+14 for the              tion of newly synthesized 5S rRNA at wild-type levels in <br /> wild-type NOP7 strain+ Analysis of low molecular weight            nop7 mutants is consistent with our previous observa- <br /> RNAs indicated that in the Nop7p-depleted strain or                tions that 5S rRNA exists as a stable RNP complex <br />  Yeast nucleolar protein containing a BRCT motif                                                                                           157 <br />  <br />  <br />  <br />  <br />        FIGURE 7. The nop7-1 mutation or depletion of Nop7p delays processing of 27S pre-rRNA to mature 25S rRNA and <br />        impairs accumulation of 25S rRNA+ A: Synthesis, processing, and accumulation of large molecular weight rRNA processing <br />        intermediates and products were assayed in the nop7-1 mutant strain JWY5464 grown at 23 8C in YEPGlu and shifted to <br />        37 8C for 4 h+ Cells were pulse-labeled with [methyl- 3 H]-methionine for 5 min and chased with excess cold methionine for <br />        2, 5, 10, and 60 min+ RNA was extracted and analyzed as described in Materials and Methods+ The rRNA primary transcript, <br />        processing intermediates, and products are labeled+ B: Production of small molecular weight rRNAs and tRNAs was <br />        assayed in the NOP7 strain JWY5853 and the GAL-NOP7 strain JWY6901 grown in YEPGal at 30 8C then shifted to YEPGlu <br />        for 14 h+ Cells were pulse-labeled with [5,6- 3 H]-uracil for 5 min and chased with excess cold uracil for 2, 5, 10, and 60 min+ <br />        RNA was extracted and analyzed as described in Materials and Methods+ The 7S rRNA intermediate, 5+8SL and 5+8SS RNA <br />        products, 5S rRNA, and tRNAs are labeled+ <br />  <br />  <br />  <br />  <br /> with rpL5, independent of ribosomes, which is relatively                 ate upon depletion of Nop7p, consistent with a block in <br /> stable when assembly of 60S ribosomal subunits is                        processing at the A0 , A1 , and A2 sites+ <br /> disrupted (Moritz et al+, 1990; Deshmukh et al+, 1993)+                     Northern hybridization does not distinguish between <br />   To further analyze the steps in pre-rRNA processing                    the 27SA2 and the 27SA3 precursors, or between the <br /> that are affected upon depletion of Nop7p, we exam-                      27SBL and 27SBS precursors+ To assay levels of these <br /> ined the steady-state levels of rRNA precursors, pro-                    precursors and to determine whether processing at the <br /> cessing intermediates, and products by northern blotting+                A2 , A3 , and B sites was correct at the nucleotide level, <br /> To distinguish each rRNA <span id='am-95' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-96' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span>, oligonucleotide                        primer extension assays were performed using oligo- <br /> probes were used that are complementary to different                     nucleotides complementary to sequences 39 of these <br /> portions of the 59 ETS, ITS1, ITS2, 25S rRNA, and 18S                    sites+ Results confirmed that only modest levels of 27S <br /> rRNA+ RNA was extracted from the GAL-NOP7 and                            precursors accumulated and that processing occurred <br /> wild-type NOP7 strains 0, 4, 19, and 26 h after shifting                 precisely at the A2 , A3 , BL , and BS sites in ITS1 (data <br /> the cells from YEPGal to YEPGlu+ Examination of the                      not shown)+ <br /> blots (Fig+ 8) and quantification by phosphorimaging <br /> indicated that the amount of 25S rRNA relative to 18S <br />                                                                          DISCUSSION <br /> rRNA decreased by 50%, and that levels of 27SA3 , <br /> 27SA2 , and 27SB pre-rRNAs increased 3- to 10-fold+                      To identify proteins that function together with Drs1p in <br /> These results suggest that in the absence of Nop7p,                      the synthesis of 60S ribosomal subunits, we screened <br /> processing of the 27S pre-rRNAs is decreased, result-                    for mutations that are lethal in combination with drs1 <br /> ing in diminished amounts of mature 25S rRNA+ We                         mutations+ Three mutant strains were identified in which <br /> also observed increased amounts of the 35S and 32S                       production of 60S ribosomal subunits was compro- <br /> pre-rRNAs and the aberrant 23S pre-rRNA intermedi-                       mised+ We have characterized in detail the NOP7 gene <br />  158                                                                                                  C.C. Adams et al. <br />  <br />  <br />  <br />  <br />                                                                 FIGURE 8. A: Effect of depletion of Nop7p on steady-state <br />                                                                 levels of pre-rRNA and mature rRNA+ RNA was extracted from <br />                                                                 strains JWY5853 (NOP7 ) and JWY6901 (GAL-NOP7 ) grown <br />                                                                 in YEPGal at 30 8C and 4, 19, and 26 h after shifting to <br />                                                                 YEPGlu+ RNAs were resolved by agarose gel electrophoresis <br />                                                                 and analyzed by northern hybridization using the indicated <br />                                                                 oligonucleotide probes complementary to different regions of <br />                                                                 the 35S pre-rRNA+ B: Sequences within the pre-rRNAs and <br />                                                                 mature rRNAs complementary to probes are indicated+ <br />  <br />  <br />  <br />  <br /> corresponding to sl3+ NOP7 is an essential gene en-        tion as scaffolding molecules to hold together compo- <br /> coding a highly conserved protein localized to the nu-     nents of multimolecular complexes+ For example, the <br /> cleolus+ Inactivation of Nop7p in the nop7-1 mutant or     human BRCA1 protein interacts with RNA helicase A <br /> depletion of Nop7p using the GAL-NOP7 conditional          through its BRCT motif (Anderson et al+, 1998)+ Thus a <br /> expression construct results in a deficit of 60S ribo-     likely function for Nop7p is to serve as a scaffolding <br /> somal subunits and accumulation of halfmer poly-           protein to hold together molecules necessary for ribo- <br /> ribosomes+ Processing of the 27S rRNA precursors           some assembly or rRNA processing+ Consistent with <br /> is slightly delayed and production of mature 25S rRNA      this <span id='am-53' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-55'  ></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="Idea" datatype="xsd:string"></span>idea</span>, we have recently found that Nop7p copurifies <br /> is reduced+ However, processing of each rRNA occurs        with 66S preribosomal particles (Harnpicharnchai et al+, <br /> accurately; no aberrant processing intermediates ac-       2001)+ <br /> cumulate+ These results lead us to conclude that Nop7p,       Processing of 27SA and 27SB pre-rRNAs appears to <br /> like Drs1p, is an essential nucleolar protein necessary    be slowed but not completely blocked in nop7 mutants+ <br /> for biogenesis of 60S ribosomal subunits+                  The absence of a strong block in rRNA processing sug- <br />                                                            gests that Nop7p is not directly involved in rRNA pro- <br />                                                            cessing+ It seems more likely that Nop7p helps to <br /> Where does Nop7p function in the pathway <br />                                                            establish the proper context of the assembling ribo- <br /> of 60S ribosomal subunit biogenesis? <br />                                                            some, so that rRNA is properly folded for its processing <br /> Although a few nucleolar proteins have clearly defined     or assembly with other molecules to form a stable, <br /> functions, such as nucleases, the roles of most nucle-     mature 60S subunit+ In the absence of Nop7p func- <br /> olar proteins, including Nop7p, remain unclear+ How-       tion, improperly or incompletely assembled 66S pre- <br /> ever, a few clues are available from the sequence of       ribosomal particles may fall apart and the protein and <br /> the Nop7 protein and from the phenotype of nop7 mu-        RNA components of the assembly intermediate might <br /> tants+ Although Nop7 does not resemble any known           be degraded+ However, our pulse-chase and northern <br /> nuclease and does not contain any motifs known to          data do not allow us to distinguish whether in nop7 <br /> recognize RNA, it does contain the BRCT motif first        mutants 27S pre-rRNA is degraded so that less 25S, <br /> found in the BRCA1 protein (Koonin et al+, 1996; Calle-    7S, and 5+8S rRNAs are produced, or 25S, 7S, and <br /> baut &amp; Mornon, 1997)+ This motif can fold into a well-     5+8S rRNAs are made in normal amounts, but sub- <br /> defined structure that may function as a protein&#8211;protein   sequently degraded+ Alternatively, processing of 27S <br /> interaction domain (Zhang et al+, 1998)+ Consequently,     pre-rRNAs to 25S RNA may simply be occurring in <br /> several BRCT-containing proteins are thought to func-      proper fashion but at a slower rate in nop7 mutants+ It <br />  Yeast nucleolar protein containing a BRCT motif                                                                     159 <br />  <br /> is also possible that Nop7p functions in transport of          that Drs1p cofractionates with Nop7p in 66S preribo- <br /> pre-ribosomes from the nucleolus to the nucleoplasm or         somes (Harnpicharnchai et al+, 2001)+ However, we have <br /> cytoplasm, rather than in assembly+ Mutants defective          found no evidence for direct physical interactions be- <br /> in ribosome transport often also exhibit defects in as-        tween Nop7p and Drs1p; results of two-<span id='am-123' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-124' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-125' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-127'  ></span>hybrid</span> tests <br /> sembly, perhaps due to failure to recycle assembly fac-        were negative+ <br /> tors (Stage-Zimmerman et al+, 2000; Gadal et al+, 2001)+          The allele-specific synthetic lethal phenotypes that <br />    The accumulation of 35S and 32S rRNA precursors             are observed between nop7-1 and the drs1 mutations <br /> in the nop7 mutants suggests that processing at the A0 ,       may offer some clues to the function of Nop7p or to its <br /> A1 , and A2 sites may be delayed when production of            interactions with Drs1p+ nop7-1 is lethal in combination <br /> 25S and 5+8S rRNA is altered+ Similar effects have             with drs1-3, drs1-5, and drs1-11, but not with eight other <br /> been observed in other mutants defective in 60S sub-           mutant alleles of drs1+ Each of the 11 different drs1 <br /> unit biogenesis, suggesting that the pathways for bio-         mutations results in a deficiency of 60S ribosomal sub- <br /> genesis of the two subunits may interact or may be             units and accumulation of halfmer polyribosomes+ There <br /> subject to cross regulatory controls (reviewed in Ve-          are only slight variations in the extent of the ribosome <br /> nema &amp; Tollervey, 1999)+ Recently we have found that           deficiency phenotype in each of these mutants+ How- <br /> Nop7p is associated with 35S pre-rRNA as well as 27S           ever, there is no obvious correlation between these <br /> pre-rRNA, suggesting that Nop7p may first associate            quantitative effects and synthetic lethality with nop7-1+ <br /> with nascent ribosomes in the 90S pre-rRNP before it           Thus the synthetic lethal phenotypes do not merely <br /> functions in 66S pre-rRNPs (Harnpicharnchai et al+,            reflect a general decrease in drs1 function+ DEAD-box <br /> 2001)+ Therefore an alternative explanation for the ef-        proteins contain eight conserved motifs implicated in <br /> fects on A0 , A1 , and A2 processing in the nop7 mutants       ATP binding, ATP hydrolysis, RNA binding, and RNA <br /> is that partial loss of Nop7p function might also impact       helicase activity (Fuller-Pace, 1994; de la Cruz et al+, <br /> the early steps of pre-rRNA processing by affecting            1999)+ drs1-3 contains Val305Asp and Ile306Val mu- <br /> structure or function of 90S pre-rRNPs+                        tations adjacent to the PTRELA motif, drs1-5 contains <br />    It is not surprising that the small decreases in the rate   a Tyr563Cys mutation adjacent to the HRVGR motif, <br /> of production of 25S rRNA and 60S ribosomal subunits           and drs1-11 contains Ala260Val and Leu564Pro muta- <br /> that we observe over a period of minutes to several            tions, the latter adjacent to the HRVGR motif+ The other <br /> hours in nop7 mutants can eventually (after 10&#8211;20 h)           eight drs1 mutations are present in different portions of <br /> result in cessation of growth+ The demands of rapidly          the Drs1 protein, including conserved DBP motifs, but <br /> growing yeast cells for ribosomes are great; approxi-          none of these are in the HRVGR motif+ The HRVGR <br /> mately 2,000 ribosomes are synthesized each minute             motif has been implicated in linking ATP binding and <br /> (Warner, 1999)+ When ribosome assembly is slowed or            hydrolysis with conformational changes important for <br /> stopped, the numbers of ribosomes will be diluted by           helicase or translocase activity of DBPs (Pause &amp; <br /> cell division and eventually drop below a threshold nec-       Sonenberg, 1992; Pause et al+, 1993, reviewed in Tan- <br /> essary for continued cell growth and division+ Similar         ner &amp; Linder, 2001)+ Perhaps Nop7p facilitates the as- <br /> results have been observed for other mutants defec-            sociation of RNA with Drs1p or the hydrolysis of ATP by <br /> tive in 60S ribosomal subunit biogenesis (Rotenberg            Drs1p+ <br /> et al+, 1988; Daugeron &amp; Linder, 1998; de la Cruz et al+, <br /> 1998a, 1998b; Kressler et al+, 1998; Burger et al+, 2000)+ <br />                                                                A function in development for metazoan <br />                                                                homologs of Nop7p <br /> What is the nature of interactions between <br />                                                                Nop7p is a highly conserved protein; homologs are <br /> Nop7p and Drs1p? <br />                                                                found in zebrafish, <span id='am-36' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-37'  ></span><span id='am-38' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Schizosaccharomyces</span> pombe, Cae- <br /> Nop7p, like Drs1p, is a nucleolar protein necessary for        norhabditis elegans, <span id='am-58' about='obo:NCBITaxon_8355' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8342 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_8360 obo:NCBITaxon_2759 obo:NCBITaxon_262014 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8353 obo:NCBITaxon_7776 obo:NCBITaxon_8352 obo:NCBITaxon_30319 obo:NCBITaxon_41666 obo:NCBITaxon_8292'><span id='am-59' property="oboInOwl:hasExactSynonym" content="platanna" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62'  ></span><span id='am-63' property="rdfs:label" content="Xenopus laevis" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="Xenopus laevis (Daudin, 1802)" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="Xenopus leavis" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasExactSynonym" content="common platanna" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasRelatedSynonym" content="Bufo laevis" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasExactSynonym" content="African clawed frog" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasExactSynonym" content="clawed frog" datatype="xsd:string"></span>Xenopus laevis</span>, mouse, and hu- <br /> production of 60S ribosomal subunits+ However, nop7            man+ The zebrafish homolog pescadillo was discovered <br /> mutants are delayed in processing 27S to 25S rRNAs,            in a screen for embryonic lethal mutations (<span id='am-118' about='obo:NCBITaxon_473123' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_27394 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_74974 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_74975 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-119' property="rdfs:label" content="Allende" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-122'  ></span>Allende</span> <br /> whereas drs1 mutants accumulate 27SB pre-rRNA and              et al+, 1996; Gaiano et al+, 1996)+ By the third day of <br /> produce no detectable 25S rRNA (Fig+ 2; Ripmaster              embryogenesis in zebrafish pes mutants, many differ- <br /> et al+, 1992)+ Most combinations of double mutants with        ent organs and tissues, including the brain, liver, and <br /> 60S subunit deficiencies that we and others have con-          gut, are underdeveloped, and by day 6, the embryos <br /> structed do not result in lethality (J+L+ Wolford, Jr+, un-    die+ The ribosome deficiency phenotype of yeast nop7 <br /> publ+; P+ Linder, pers+ comm+)+ Therefore we believe           mutants suggests a molecular basis for the zebrafish <br /> that the synthetic lethality observed between a subset         pes mutant phenotypes+ The zebrafish pescadillo pro- <br /> of the drs1 mutations and nop7-1 reflects specific phys-       tein, like its yeast homolog Nop7p, may be required for <br /> ical or functional interactions between Drs1p and Nop7p,       synthesis of ribosomes+ Zebrafish embryos might in- <br /> rather than the cumulative effects of diminished 60S           herit maternal ribosomes, but could require zygotic <br /> subunits+ In support of this view, we have discovered          expression of ribosomal genes and genes encoding <br />  160                                                                                                        C.C. Adams et al. <br />  <br /> ribosome assembly factors to synthesize additional ribo-          mid into yeast strain JWY2181 containing the drs1::HIS3 null <br /> somes necessary for late stages of development+ A                 allele and plasmid YEP352-DRSI-HA containing URA3 and <br /> shortage of ribosomes might be especially deleterious             HA-tagged DRS1. Plasmid loss assays were performed using <br /> to the development of rapidly proliferating tissues such          5-fluoroorotic acid (5-FOA) at 13 8C, 30 8C, and 37 8C as de- <br /> as those affected in pes embryos+ Analogous situa-                scribed in Deshmukh et al+ (1995) to identify strains contain- <br />                                                                   ing Cs 2 or Ts 2 alleles of DRS1 on the LEU2 plasmid+ <br /> tions have been observed during development of X. <br /> laevis and <span id='am-73' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-74'  ></span><span id='am-75' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span>Drosophila melanogaster</span>+ In Xenopus, ex- <br /> pression of genes encoding rRNA, ribosomal proteins,              Isolation of synthetic lethal mutants <br /> small nucleolar RNAs, and nucleolar proteins is acti- <br /> vated during embryogenesis to synthesize sufficient               Yeast strains JWY5451, JWY5452, JWY5455, and JWY5456 <br /> ribosomes for late stages of development (Amaldi, 1982;           [ ade2-1 ade3::hisG ura3-1 leu2-3,112 1 plasmids pJW3956 <br />                                                                   ( CEN URA3 ADE3 DRS1 ) and either pJW3816 or pJW3828 <br /> Pierandrei-Amaldi et al+, 1985; Caizergues-Ferrer et al+, <br />                                                                   (CEN LEU2 and either drs1-2 or drs1-5 )] form colonies on <br /> 1989, 1991)+ Xenopus anucleolate mutants that con-                YEPGlu medium that exhibit red/white sectoring due to loss <br /> tain a deletion of most of the cluster of reiterated genes        of plasmid pJW3956+ Each strain was grown in synthetic <br /> for 18S and 28S rRNAs die at the swimming tadpole                 complete medium lacking leucine and uracil to 5{10 7 210 8 <br /> stage, due to the inability to make new ribosomes dur-            cells/mL, plated on YEPGlu solid medium at ;300 colony <br /> ing embryogenesis (Elsdale et al+, 1958; Brown &amp; Gur-             forming units per plate, and mutagenized to approximately <br /> don, 1964; Wallace &amp; Birnstiel, 1966; Brown &amp; Weber,              15% survival with UV irradiation+ Alternatively, strains were <br /> 1968)+ Drosophila minifly, Minute, mini, and bobbed mu-           grown in liquid medium as above, mutagenized to approxi- <br /> tants, which contain mutations in genes encoding a                mately 15% survival with 0+3% EMS, and plated on YEPGlu <br /> nucleolar protein, a ribosomal protein, 5S rRNA, and              solid medium as above+ Mutagenized cells were incubated at <br /> 35S rRNA, respectively, have defects in ribosome syn-             30 8C for 5&#8211;8 days+ Approximately 175,000 colonies were <br />                                                                   screened for a nonsectoring phenotype+ Candidates were <br /> thesis that lead to developmental delays and reduction <br />                                                                   streaked for single colonies on YEPGlu medium to confirm <br /> of adult body size (Boncinelli et al+, 1972; Procunier &amp;          nonsectoring and plated on medium containing 1 mg/mL <br /> Tartof, 1975; Kay &amp; Jacobs-Lorena, 1985; Giordano                 5-FOA to determine whether or not growth depended on the <br /> et al+, 1999)+ Further studies of Nop7p and other nu-             presence of plasmid JW3956 containing URA3, ADE3, and <br /> cleolar molecules will be important not only to under-            DRS1. Strains retaining pJW3956 for reasons unrelated to <br /> stand ribosome biogenesis, but to appreciate complex              synthetic lethality with drs1-2 or drs1-5 were identified as <br /> phenotypes resulting from deficiencies in ribosomes+              follows: the nonsectoring, 5-FOA-sensitive strains were grown <br />                                                                   in liquid YEPGlu to allow loss of the CEN LEU2 drs1-2 or <br />                                                                   CEN LEU2 drs1-5 plasmid+ These Leu 2 strains were trans- <br /> MATERIALS AND METHODS                                             formed with pJW3833 (DRS1 LEU2 ), pJW3816 (drs1-2 LEU2 ), <br />                                                                   or pJW3828 (drs1-5 LEU2 )+ Three yeast strains, JWY5459 <br /> Strains, media, and genetic techniques                            (SL1), JWY5461 (sl2), and JWY5463 (sl3), in which sectoring <br />                                                                   was restored when transformed with pJW3833 but was not <br /> Yeast strains used in this study are listed in Table 1+ Yeast     restored when transformed with pJW3816 or pJW3828 were <br /> were grown in YEPGlu, YEPGal, or defined synthetic media          retained as bona fide synthetic lethal mutants+ <br /> and genetically manipulated as described in Sherman et al+ <br /> (1986)+ Yeast cells were transformed with DNA using the <br /> lithium acetate method (Ito et al+, 1983)+ DNA cloning in Esch-   Cloning and disruption of NOP7 <br /> erichia coli and analysis of DNA were performed as de- <br />                                                                   The NOP7 gene defined by the sl3 mutation in JWY5463 was <br /> scribed in Sambrook et al+ (1989)+ <br />                                                                   cloned by replacing plasmid pJW3828 (CEN LEU2 drs1-5 ) <br />                                                                   with plasmid pJW3988 (CEN LYS2 drs1-5 ), generating yeast <br /> Isolation of new mutant alleles of DRS1                           strain JWY5465+ This allowed us to clone NOP7 from a yeast <br />                                                                   genomic library cloned in a CEN LEU2 vector, by transfor- <br /> Mutant alleles of DRS1 were isolated by PCR mutagenesis           mation of JWY5465 and complementation of the nonsector- <br /> and gap repair in vivo, as described in Deshmukh et al+ (1995)+   ing, 5-FOAs phenotypes+ Transformants were selected on <br /> Plasmid pJW3016 (pRS315 containing LEU2 plus a Pst I-             complete synthetic medium lacking lysine and leucine, and <br /> Xho I fragment bearing DRS1 ) was digested with Xba I and         red/white sectoring 5-FOAR colonies were identified+ Plas- <br /> Bam HI and the gapped plasmid was purified by gel electro-        mids were extracted from the candidate strains and trans- <br /> phoresis+ DNA from nt 230 to 12092 of the DRS1 ORF,               formed into E. coli+ Those plasmids containing LEU2 (but not <br /> which includes the coding region for most of the core heli-       LYS2 or URA3 ) were identified by DNA dot blot analysis and <br /> case motifs, was generated by PCR from intact pJW3016,            then transformed into yeast strain JWY5465 to reconfirm sec- <br /> using primers 60A-9 and 60A-6 (Table 2)+ PCR was carried          toring and 5-FOAR phenotypes+ The ends of the genomic <br /> out under standard conditions, except 10-fold molar less dATP,    DNA inserts in each plasmid were sequenced and found to <br /> dGTP, or dTTP was used compared with the other three              encompass an 8 kb region of chromosome VII containing <br /> dNTPs (20 vs+ 200 mM)+ Under these conditions, approxi-           MIC1, YGR101w, YGR102c, YGR103w, and the 39 end of <br /> mately one to two mutations were generated per 1,000 nt+          SRB5+ To identify which if any of these ORFs complemented <br /> PCR fragments were co-transformed with the gapped plas-           the sl3 mutation, several subclones were constructed by mak- <br />  Yeast nucleolar protein containing a BRCT motif                                                                              161 <br />  <br /> TABLE 1+ S. cerevisiae strains used in this study+ <br />  <br /> Strain                                                   Genotype                                                   Source <br />  <br /> YCH125      MATa ade 2-1 ade3::hisG ura3-1 his3-11,15 leu2-3,112 trp1-1 LYS2                                  C+ Hardy &amp; S+ Wente <br /> YCH128      MATa ade2-1 ade3:hisG ura3-1 his3-11,15 leu2-3,112 TRP1 lys2                                      C+ Hardy &amp; S+ Wente <br /> JWY1401     MATa/MATa ade2-101/1 can1/1 his3-D200/his3-D200 leu2-D1/leu2-D1 lys2-801/lys2-801 trp1-D101/      M+ Moritz <br />              trp1-D101 ura3-52/ura3-52 <br /> JWY5436     MATa/MATa ade2-1/ade2-1 ade3::hisG/ade3::hisG ura3-1/ura3-1 his3-11,15/his3-11,15 leu2-3,112/     YCH125 X YCH128 <br />              leu2-3,112 trp1-1/TRP1 LYS2/lys2 <br /> JWY5439     MATa/MATa ade2-1/ade2-1 ade3::hisG/ade3::hisG ura3-1/ura3-1 his3-11,15/his3-11,15 leu2-3,112/ <br />              leu2-3,112 trp1/TRP1 LYS2/lys2 DRS1/drs1D::hisGURA3hisG <br /> JWY5440     MATa/MATa ade2-1/ade2-1 ade3::hisG/ade3::hisG ura3-1/ura3-1 his3-11,15/his3-11,15 leu2-3,112/ <br />              leu2-3,112 trp-1/TRP1 LYS2/lys2 DRS1/drs1D::hisGURA3hisG <br /> JWY5444     MATa ade2-1 ade3::hisG ura3-1 his3-11,15 leu2-3,112 TRP1 lys2 drs1D::hisG plus pJW3956 <br />              (CEN URA3 ADE3 DRS1) <br /> JWY5447     MATa ade2-1 ade3::hisG ura3-1 his3-11,15 leu2-3, 112 trp1-1 LYS2 drs1D::hisG plus pJW3956 <br /> JWY5451     MATa ade2-1 ade3::hisG ura3-1 his3-11,15 leu2-3,112 TRP1 lys2 drs1D::hisG plus pJW3956 <br />              and pJW3816 (CEN LEU2 drs1-2 ) <br /> JWY5452     MATa ade2-1 ade3::hisG ura3-1 his3-11,15 leu2-3,112 TRP1 lys2 drs1D::hisG plus pJW3956 <br />              and pJW3828 (CEN LEU2 drs1-5) <br /> JWY5455     MATa ade2-1 ade3::hisG ura3-1 his3-11,15 leu2-3,112 trp1-1 LYS2 drs1D::hisG plus pJW3956 <br />              and pJW3816 (CEN LEU2 drs1-2) <br /> JWY5456     MATa ade2-1 ade3::hisG ura3-1 his3-11,15 leu2-3,112 trp1-1 LYS2 drs1D::hisG plus pJW3956 <br />              and pJW3828 (CEN LEU2 drs1-5) <br /> JWY5459     MATa ade2-1 ade3::hisG ura3-1 his3-11,15 leu 2-3, 112 TRP1 lys2 drs1D::hisG SL1 plus pJW3956 <br />              and pJW3816 <br /> JWY5460     MATa ade2-1 ade3::hisG ura3-1 his3-11,15 leu2-3,112 TRP1 lys2 drs1D::hisG SL1 plus pJW3956 <br /> JWY5461     MATa ade2-1 ade3::hisG ura3-1 his3-11,15 leu2-3,112 TRP1 lys2 drs1D::hisG sl2 plus pJW3956 <br />              and pJW3828 <br /> JWY5462     MATa ade2-1 ade3::hisG ura3-1 his3-11,15 leu2-3,112 TRP1 lys2 drs1D::hisG sl2 plus pJW3956 <br /> JWY5463     MATa ade2-1 ade3::hisG ura3-1 his3-11,15 leu2-3,112 TRP1 lys2 drs1D::hisG sl3 plus pJW3956 <br />              and pJW3828 <br /> JWY5464     MATa ade2-1 ade3::hisG ura3-1 his3-11,15 leu2-3,112 TRP1 lys2 drs1D::hisG sl3 plus pJW3956 <br /> JWY5465     MATa ade2-1 ade3::hisG ura3-1 his3-11,15 leu2-3,112 TRP1 lys2 drs1D::hisG sl3 plus pJW3956 <br />              and pJW3988 (CEN LYS2 drs1-5) <br /> JWY5802     MATa ade2-1 ade3::hisG ura3-1 his3-11,15 leu2-3,112 trp1-1 LYS2 drs1D::hisG NOP7::TRP1 NOP7 <br />              plus pJW3956 and pJW3828 <br /> JWY5803     MATa/MATa ade2-1/ade2-1 ade3::hisG/ade3::hisG ura3-1/ura3-1 his3-11,15/his3-11,15 leu2-3,112/ <br />              leu2-3,112 trp1-1/TRP1 LYS2/lys2 drs1D::hisG/drs1D::hisG sl3/SL3 NOP7/NOP7::TRP1 NOP7 <br /> JWY5827     MATa/MATa ade2-101/1 can1/1 his3-D200/his3-D200 leu2-D1/leu2-D1 lys2-801/lys2-801 trp1-D101/ <br />              trp1-D101 ura3-52/ura3-52 NOP7/nop7D::HIS3 <br /> JWY5850     Mata his3D-200 leu2 -D1 lys2-801 trp1-D101 ura3-52 nop7::HIS3 plus pJW4915 (CEN URA3 NOP7) <br /> JWY5852     MATa his3D-200 leu2 -D1 lys2-801 trp1-D101 ura3-52 nop7::HIS3 plus pJW4915 (CEN LEU2 <br />              HA-NOP7) <br /> JWY5853     MATa his3D-200 leu2 -D1 lys2-801 trp1-D101 ura3-52 nop7::HIS3 plus pJW4913 (CEN LEU2 NOP7) <br /> JWY6901     MATa his3D-200 leu2 -D1 lys2-801 trp1-D101 ura3-52 nop7::HIS3 plus pJW6004 (CEN URA3 <br />              GAL-NOP7) <br /> JWY6700     MATa his3D-200 leu2 -D1 lys2-801 trp1-D101 ura3-52 nop7::HIS3 plus pJW5814 (CEN LEU2 NOP7-HA3 ) <br />  <br />  <br />  <br />  <br /> ing internal deletions+ Plasmid JW3993 containing a deletion        ORF YGR103w conferred the complementing phenotype, and <br /> from the Sph I site in MIC1 to the Sac II site near the 39 end of   not an RNA molecule transcribed from sequences 59 or 39 to <br /> YGR102c still complemented sl3, whereas plasmid JW3992              the ORF, a frameshift mutation in YGR103w was constructed <br /> containing a deletion from the Sac II site in YGR102c to the        by digesting plasmid pJW3993 with Nsi I, removing the over- <br /> Hin dIII site within SRB5 failed to complement sl3+ These re-       hanging 39 ends with Klenow fragment, and relegating the <br /> sults suggested that YGR103w was the ORF complementing              DNA+ The plasmid containing this frameshift mutation be- <br /> sl3+ To test this hypothesis, YGR103w (and 500 nt of 59 and         tween codons 72 and 73 of YGR103w failed to complement <br /> 39 flanking nontranslated sequences) was amplified by PCR           sl3+ <br /> using oligonucleotides 103 Bam up and 103 Bam down                     To confirm that YGR103w corresponds to sl3 and is not an <br /> (Table 2), digested with Bam HI and cloned into the Bam HI          extragenic multicopy suppressor of sl3, we tested whether <br /> site of plasmid pJW3982 (CEN LEU2 ) to create pJW3998+              YGR103w is linked to the sl3 locus+ The Bam HI fragment <br /> This plasmid containing the YGR103w ORF complemented                containing YGR103w was isolated from pJW3998 and in- <br /> sl3+ Thus, ORF YGR103w was determined to be the comple-             serted into the Bam HI site of yeast integrating plasmid <br /> menting molecule+ To determine that the protein encoded by          pRS304, containing TRP1 and no yeast origins of replication+ <br />  162                                                                                                            C.C. Adams et al. <br />  <br />          TABLE 2+ Oligonucleotides used in this study+ <br />  <br />          60A-9              59-GTGCCTACAACAGCG-39 <br />          60A-6              59-TTTTGCTGTTAGTGAC-39 <br />          103 Bam Up         59-AGACGGATCCTTTTACAGCTTCAGCAGA-39 <br />          103 Bam Down       59-AATTGGATCCCTGCCTTGATGAACTTAA-39 <br />          103 HIS3 pro       59-CCTGTAGAAAATAGTATAGTAACAGCGGTATCCTACTTATACACTCTTGGCCTCCTCTAG-39 <br />          103 HIS3 term      59-GAGAGGCTATTGGAAAAGAAGAGAAAACTATTTCTTGGAATCTAGTCGTTCAGAATGACACG-39 <br />          JJHA1              59-CTTCCGCGGAACAATTTTTTTACAGCTTCAGC-39 <br />          NOP7Up             59-TACTGGATCCGTGTAGTAAATCCTGCTGGTG-39 <br />          JJHA2              59-GGAGGCCTTTTCTTGGAATCTAGTTTATTCAG-39 <br />          NOP7Down           59-TACTGGATCCTATAATCATTGGCACCTGGGC-39 <br />          103 Bam NHA        59-CTTTTTATTTCTTGGATCCCTAGGATAGATACC-39 <br />          1                  59-CATGGCTTAATCTTTGAGAC-39 <br />          2                  59-GCTCTTTGCTCTTGCC-39 <br />          3                  59-TGTTACCTCTGGGCCC-39 <br />          4                  59-AATTTCCAGTTACGAAAATTCTTGT-39 <br />          5                  59-GGCCAGCAATTTCAAGTTA-39 <br />          6                  59-CTCCGCTTATTGATATGC-39 <br />  <br />  <br />  <br />  <br /> This plasmid pJW4909 was linearized and transformed into             pJW4913 ( CEN LEU2 NOP7 ) produced yeast strain JWY6901 <br /> yeast strain JWY5456 [ MATa trp1-1 LYS2 drs1D::hisG 1 plas-          (nop7::HIS3 plus plasmid pJW6004 containing CEN URA3 <br /> mids pJW3956 (CEN URA3 ADE3 DRS1 ) and pJW3828 (CEN                  GAL-NOP7 )+ Levels of NOP7 mRNA in JWY6901 grown in <br /> LEU2 drs1-5 )] to create strain JWY5802+ Integration at the          YEPGal or shifted to YEPGlu were assayed by northern blot- <br /> homologous locus in the genome was confirmed by southern             ting and hybridization using the NOP7 ORF as a probe+ Lev- <br /> blot analysis+ JWY5802 was crossed to JWY5463 (MATa lys2             els of Nop7p in JWY6901 grown in YEPGal or shifted from <br /> TRP1 sl3 plus plasmids pJW3956 and pJW3828) to create                YEPGal to YEPGlu were assayed by western blotting using <br /> diploid JWY5803+ This diploid was sporulated, tetrads dis-           rabbit antibodies, generated by immunization with synthetic <br /> sected, and phenotypes assessed to score segregation of              peptide GIKYSETSEADKDVNKSK containing amino acids <br /> relevant markers (Sl 2 , i+e+, 5-FOAs , Trp 2 , and Ts 2 )+ Tetrad   522 to 539 of Nop7p, linked to KLH (Alpha Diagnostic Inter- <br /> types predicted to occur if the sl3 locus is in fact identical to    national)+ Rabbit polyclonal antibodies against yeast PGK <br /> ORF YGR103w were obtained, rather than the very different            were a kind gift of Dr+ Jeremy Thorner+ <br /> pattern of tetrad types predicted if sl3 were not identical to <br /> YGR103W+ For example, NOP7 TRP1 NOP7 segregated in <br /> opposition to nop7-1, as expected if sl3 is YGR103w+                 Fractionation and characterization <br />    A yeast strain containing a nop7D::HIS3 null allele was           of polyribosomes <br /> constructed using PCR by the method of Baudin et al+ (1993)+ <br /> Oligonucleotides 103HIS3 pro and 103HIS3 term (Table 2)              Yeast strains JWY5464 (nop7-1 ) and JWY5444 (NOP7 ) were <br /> were used as primers with a HIS3 template substrate to syn-          grown in YEPGlu at 23 8C overnight to 3{10 7 cells/mL and <br /> thesize an 1,100 bp HIS3 fragment flanked on the 59 end              shifted to YEPGlu at 23 8C or 37 8C for 5 h+ JWY6901 (GAL- <br /> with nt 243 to nt 21 of NOP7 and on the 39 end with nt               NOP7 ) and JWY5853 (NOP7 ) were grown in YEPGal at <br /> 11799 to nt 11845 of NOP7+ This fragment was transformed             30 8C to 5{10 7 cells/mL and shifted to YEPGlu at 30 8C for <br /> into diploid yeast JWY1401, homozygous for the deletion              4 h or 14 h+ Extracts were prepared from these strains and <br /> allele his3 -D200, with selection for His 1 phenotype+ Replace-      ribosomal subunits, ribosomes, and polyribosomes were as- <br /> ment of one wild-type allele of NOP7 by the nop7D::HIS3 null         sayed by sucrose gradient centrifugation, as described pre- <br /> allele in the transformant JWY5827 was confirmed by geno-            viously (Sun &amp; Woolford, 1994)+ To quantify ratios of total <br /> mic southern blotting+ The lethal phenotype of nop7D::HIS3           subunits, extracts were prepared as above and subjected to <br /> was demonstrated by sporulation of JWY5827 and dissection            sucrose gradient centrifugation, except that 0+5 M KCl was <br /> of tetrads+                                                          included instead of 0+15 M NaCl in the lysis buffer and su- <br />                                                                      crose gradient buffer+ <br />  <br /> Construction and analysis of a conditional <br /> null GAL-NOP7 strain                                                 Pulse-chase and northern analysis <br />                                                                      of rRNA processing <br /> The NOP7 ORF containing Bam HI sites at each end was <br /> amplified by PCR using primers NOP7Up and NOP7Down                   The synthesis, processing, accumulation, and turnover of <br /> (Table 2)+ The DNA was digested with Bam HI and cloned into          rRNAs were assayed by pulse-chase analysis as described <br /> the Bam HI site downstream of the GAL promoter in plasmid            by Venema et al+ (1998), with the following exceptions+ Strains <br /> pBM258T containing URA3, to produce plasmid pJW6004+                 JWY5464 (nop7-1 ) and JWY5444 (NOP7 ) grown in YEPGlu <br /> The resulting GAL-NOP7 plasmid pJW6004 was transformed               at 23 8C to 2{10 7 cells/mL were shifted to YEPGlu at 37 8C for <br /> into yeast strain JWY5853+ Subsequent loss of plasmid                2 h+ Strains JWY6901 (GAL-NOP7 ) and JWY5853 (NOP7 ) <br />  <br />  <br /> grown in YEPGal at 30 8C to 5{10 7 cells/mL were shifted to     <br /> YEPGlu at 30 8C for 14 h+ Both pairs of strains were pulse-      <br /> labeled with 33 mCi/mL of [methyl- 3 H]-methionine or 18         <br /> mCi/mL of [5,6- 3 H]-uracil for 5 min and chased for up to       <br /> 60 min with excess cold methionine or uracil+ RNA was ex-        <br /> tracted and equal cpm of each sample were subjected to           <br /> electrophoresis on 1+2% agarose 6% formaldehyde gels or          <br /> 8% acrylamide 8 M urea gels+ RNA was capillary blotted <br /> (agarose) or electroblotted (acrylamide) onto Nytran Plus and   <br /> blots were exposed to BIOMAX MS film (Kodak) using a            <br /> BIOMAX Transcreen-LE (Kodak) at 280 8C+ Nytran filters were     <br /> aligned with the exposed films, positions of the filters con- <br /> taining the 18S and 25S rRNAs were cut out, and were counted <br /> by liquid scintillation+                                       <br />    Northern analysis and primer extension were carried out <br />                                                          <br /> as described in Venema et al+ (1998)+ Yeast strains JWY5853      <br /> and JWY6901 were grown in YEPGal at 30 8C to 5{10 7               <br /> cells/mL then shifted to YEPGlu for 4 h, 19 h or 26 h+ RNA         <br /> was extracted, subjected to electrophoresis, and blotted as   <br /> above+ Blots were hybridized with 32 P-labeled oligonucleo-     <br /> tides #1 to #6, complementary to rRNA, as described by         <br /> Venema et al+ (1998)+                             <br /> Construction of HA-tagged Nop7p                             <br /> To construct an allele of NOP7 tagged with 3 HA epitopes at        <br /> its 39 end, a Sac II-Stu I DNA fragment containing the NOP7    <br /> ORF was synthesized by genomic PCR using primers JJHA1         <br /> and JJHA2, digested with Sac II and Stu I, and cloned be-         <br /> tween the Sac II and Stu I sites of plasmid pJW5636 (CEN         <br /> LEU2 HA3-tag)+ The resulting plasmid pJW5814 bearing        <br /> NOP7-HA3 was transformed into yeast strain JWY5850             <br /> [ nop7D::HIS3 plus pJW4914 (CEN URA3 NOP7 )]+ Transfor-         <br /> mants were plated on medium containing 5-FOA to select            <br /> for loss of pJW4914, to generate yeast strain JWY6700          <br /> [ nop7D::HIS3 plus pJW5814 (CEN LEU2 NOP7-HA3 )]+          <br /> Indirect immunofluorescence microscopy                        <br /> Strains JWY6700 (NOP7-HA3 ) and JWY5853 (NOP7 ) were               <br /> grown in YEPGlu at 30 8C to 5&#8211;6{10 7 cells/mL, harvested,        <br /> and prepared for microscopy as described by Pringle et al+     <br /> (1989)+ Mouse monoclonal antibody 16B12 directed against             <br /> the hemagglutinin epitope YPYDVPDYA was used at 1:100           <br /> dilution+ The secondary antibody used was rabbit anti-mouse       <br /> immunoglobulin G conjugated with rhodamine at a dilution of        <br /> 1:100+ DAPI (49, 6 diamidino-2-phenyl indole dihydrochloride)   <br /> was used to stain DNA+ Slides were viewed using a fluo-          <br /> rescence microscope (Carl Zeiss) equipped with a 1003           <br /> objective+ Photographs were taken on a Hamamatsu black-           <br /> and-white CCD-coupled <span id='am-85' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-86' property="rdfs:label" content="Camera" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-88'  ></span><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>camera</span>+ Digital images were ac-               <br /> quired in the program Photoshop (Adobe Systems, Mountain       <br /> View, <span id='am-1' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-6'  ></span>California</span>)+                                              <br />   <br /> </body></html>