<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Microbiology                                                                                                                         BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access <br /> The authentic methionine aminopeptidase genes are <br /> differentially expressed <span id='am-173' about='obo:NCBITaxon_1423' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_653685'><span id='am-174' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis/Bacillus globigii" datatype="xsd:string"></span><span id='am-175' property="oboInOwl:hasRelatedSynonym" content="Bacillus uniflagellatus" datatype="xsd:string"></span><span id='am-176' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232825" datatype="xsd:string"></span><span id='am-177' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042380" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232841" datatype="xsd:string"></span><span id='am-179' property="oboInOwl:hasRelatedSynonym" content="Vibrio subtilis" datatype="xsd:string"></span><span id='am-180' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232828" datatype="xsd:string"></span><span id='am-181' property="oboInOwl:hasAlternativeId" content="NCBITaxon:387338" datatype="xsd:string"></span><span id='am-182' property="oboInOwl:hasAlternativeId" content="NCBITaxon:941949" datatype="xsd:string"></span><span id='am-183'  ></span><span id='am-184' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232830" datatype="xsd:string"></span><span id='am-185' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-186' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429427" datatype="xsd:string"></span><span id='am-187' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-188' property="rdfs:label" content="Bacillus subtilis" datatype="xsd:string"></span><span id='am-189' property="oboInOwl:hasRelatedSynonym" content="Bacillus natto" datatype="xsd:string"></span><span id='am-190' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232822" datatype="xsd:string"></span><span id='am-191' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232840" datatype="xsd:string"></span><span id='am-192' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042381" datatype="xsd:string"></span><span id='am-193' property="oboInOwl:hasDbXref" content="PMID:11211269" datatype="xsd:string"></span><span id='am-194' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis8" datatype="xsd:string"></span><span id='am-195' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451272" datatype="xsd:string"></span><span id='am-196' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232844" datatype="xsd:string"></span><span id='am-197' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429428" datatype="xsd:string"></span><span id='am-198' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042382" datatype="xsd:string"></span><span id='am-199' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232819" datatype="xsd:string"></span><span id='am-200' property="oboInOwl:hasAlternativeId" content="NCBITaxon:981081" datatype="xsd:string"></span><span id='am-201' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232470" datatype="xsd:string"></span><span id='am-202' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224247" datatype="xsd:string"></span><span id='am-203' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232824" datatype="xsd:string"></span><span id='am-204' property="oboInOwl:hasAlternativeId" content="NCBITaxon:523730" datatype="xsd:string"></span><span id='am-205' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451270" datatype="xsd:string"></span>Bacillus subtilis</span> <br /> CongHui You&#8224;1,2, HongYan Lu&#8224;3, Agnieszka Sekowska*4, Gang Fang2, <br /> YiPing Wang1, Anne-Marie Gilles*2 Antoine Danchin2 <br />  <br /> Address: 1National Laboratory Protein Engineering Plant Genetic Engineering, The College Life Sciences, Peking University, 100871, <br /> Beijing, P.R. China, 2Institut Pasteur, Unit&#233; G&#233;n&#233;tique des G&#233;nomes Bact&#233;riens, 28 rue du Docteur Roux, 75724 <span id='am-409' about='obo:NCBITaxon_49669' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_50362 obo:NCBITaxon_4447 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_4667 obo:NCBITaxon_3193'><span id='am-410' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-411' property="oboInOwl:hasAlternativeId" content="NCBITaxon:49665" datatype="xsd:string"></span><span id='am-412' property="oboInOwl:hasRelatedSynonym" content="Paris L." datatype="xsd:string"></span><span id='am-413'  ></span><span id='am-414' property="rdfs:label" content="Paris" datatype="xsd:string"></span><span id='am-415' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Paris</span> Cedex 15, France, 3HKU- <br /> Pasteur Research Centre, Dexter H. C. Man Building, 8 Sassoon Road Pokfulam, Hong Kong, S.A.R., P.R. China 4CEA Saclay, Laboratoire Stress <br /> Oxydants et <span id='am-274' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-275' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-276' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-277'  ></span><span id='am-278' property="rdfs:label" content="Cancer" datatype="xsd:string"></span>Cancer</span>, DSV/DBJC/SBMS, Bat 142, 91191 Gif sur Yvette Cedex, France <br /> Email: CongHui You - cyou@pasteur.fr; HongYan Lu - lv_hongyan@hotmail.com; Agnieszka Sekowska* - AS208663@dsvidf.<span id='am-1' about='obo:NCBITaxon_1107390' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_272274 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7422 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_272201 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7423 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="rdfs:label" content="Cea" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>cea</span>.fr; <br /> Gang Fang - fangg@pasteur.fr; YiPing Wang - wangyp@pku.edu.cn; Anne-Marie Gilles* - amgilles@pasteur.fr; <br /> Antoine Danchin - adanchin@pasteur.fr <br /> * Corresponding authors &#8224;Equal contributors <br />  <br />  <br />  <br />  <br /> Published: 05 October 2005                                                    Received: 01 August 2005 <br />                                                                               Accepted: 05 October 2005 <br /> BMC Microbiology 2005, 5:57   doi:10.1186/1471-2180-5-57 <br /> <span id='am-279' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-280' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-281'  ></span><span id='am-282' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-283' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> article available from: http://www.biomedcentral.com/1471-2180/5/57 <br /> &#169; 2005 You et al; licensee BioMed Central Ltd. <br /> This Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), <br /> which permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  Abstract <br />                  Background: Two putative methionine aminopeptidase genes, map (essential) yflG (non- <br />                  essential), identified genome sequence Bacillus subtilis. We investigated they <br />                  function methionine aminopeptidases explored possible reasons their <br />                  essentiality dispensability B. subtilis. <br />                  Results: In silico analysis MAP evolution uncovered coordinated pattern MAP and <br />                  deformylase did correlate pattern 16S RNA evolution. Biochemical assays <br />                  showed MAP (MAP_Bs) YflG (YflG_Bs) B. subtilis overproduced <span id='am-350' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-351' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-352' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-353'  ></span><span id='am-354' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-355' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-356' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span>Escherichia</span> <br />                  coli obtained pure proteins exhibited methionine aminopeptidase activity vitro. <br />                  Compared MAP_Bs, YflG_Bs approximately orders magnitude efficient <br />                  assayed synthetic peptide substrates. Both map yflG genes expressed multi-copy <br />                  <span id='am-506' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-507' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-508' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-509' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-510' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>plasmids</span> complement function defective map gene chromosomes E. <br />                  coli B. subtilis. In contrast, lacZ gene transcriptional fusions showed promoter activity <br />                  map 50 100-fold higher yflG. Primer extension analysis detected the <br />                  transcription start site yflG promoter. Further work identified YvoA acted possible <br />                  weak repressor yflG expression B. subtilis vivo. <br />                  Conclusion: Both MAP_Bs YflG_Bs functional methionine aminopeptidases vitro in <br />                  vivo. The high expression level map low expression level yflG account their <br />                  essentiality dispensality B. subtilis, respectively, cells grown laboratory <br />                  conditions. Their difference activity synthetic substrates suggests different <br />                  protein targets vivo. <br />  <br />  <br />  <br />  <br />                                                                                                                                        Page 1 15 <br />                                                                                                                  (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                  http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br />  <br /> Background                                                      codes methionine aminopeptidase. In attempt to <br /> Ribosome-mediated protein synthesis initiated         better understand functional role MAP, studied <br /> with methionine (in eukaryotes) N-formylme-           evolutionary trend MAPs silico demonstrated <br /> thionine (in prokaryotes eukaryotic organelles) [1].        map yflG genes B. subtilis code for <br /> However, removal N-formyl group           methionine aminopeptidases vitro vivo; further- <br /> polypeptide peptide deformylase (DEF, EC 3.5.1.88),          more, finding high expression level map par- <br /> the N-terminal methionine large number proteins         allel low expression yflG B. subtilis, may <br /> is cleaved methionine aminopeptidase (MAP, EC                account essentiality dispensability, <br /> 3.4.11.18) [2,3]. The efficiency removal initiator    respectively. <br /> methionine defined highly conserved local sub- <br /> strate specificity, determined methionine      Results <br /> and adjacent residue. MAP hydrolytically removes        Sequence alignment evolutionary pattern analyses <br /> N-terminal methionine penultimate resi-           In genome sequence B. subtilis, genes could <br /> due bears small uncharged chain (Gly, <span id='am-416' about='obo:NCBITaxon_1416571' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_116706 obo:NCBITaxon_6804 obo:NCBITaxon_131567 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6682 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_6683 obo:NCBITaxon_88770 obo:NCBITaxon_6752 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_553065 obo:NCBITaxon_6072'><span id='am-417' property="rdfs:label" content="Ala" datatype="xsd:string"></span><span id='am-418' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-419'  ></span><span id='am-420' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Ala</span>, Ser,      code methionine aminopeptidases, map yflG, <br /> Thr, Pro, Val, Cys) [4-9], residues downstream        respectively [21] (Figure 1). The sequence similarity and <br /> the penultimate residue little impact reaction.     identity E. coli MAP (MAP_Ec) B. subtilis <br /> All MAPs studied date reported cobalt-            MAP (MAP_Bs) 65% 46%, respectively. Similarity <br /> dependent metalloproteases [3]. However, reports           MAP_Ec YflG B. subtilis (YflG_Bs) is <br /> showed MAPs exhibited activity presence          55% identity 34% similarity between <br /> other divalent ions, Zn(II), Fe(II) Mn(II) [10-      MAP_Bs YflG_Bs 58% identity 36%. <br /> 12]. According sequence homology, MAPs             However, sequence YflG_Bs differs slightly the <br /> grouped subtypes, type I type II [13,14]. An       PROSITE consensus type I MAPs (PS00680), is <br /> insertion approximately 60 residues C-terminal        [MFY]-x-G-H-G- [LIVMC]-[GSH]-x(3)-H-x(4)-[LIVM]-x- <br /> domain type II methionine aminopeptidases             [HN]-[YWVH]. In particular displays extra residue <br /> only difference types [14]. Eukaryotes          conserved GHG metal-binding motif. Interest- <br /> possess map genes, type I type II      ingly, MAP_Bs similar MAP_Ec YflG_Bs is <br /> [3], map gene prokaryotic       similar putative methionine ami- <br /> genomes, type I (<span id='am-225' about='obo:NCBITaxon_2157' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567'><span id='am-226' property="oboInOwl:hasDbXref" content="PMID:10939673" datatype="xsd:string"></span><span id='am-227' property="oboInOwl:hasDbXref" content="PMID:2112744" datatype="xsd:string"></span><span id='am-228' property="oboInOwl:hasDbXref" content="PMID:10425797" datatype="xsd:string"></span><span id='am-229' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-230' property="oboInOwl:hasDbXref" content="PMID:9103655" datatype="xsd:string"></span><span id='am-231' property="oboInOwl:hasRelatedSynonym" content="Prokaryota" datatype="xsd:string"></span><span id='am-232' property="oboInOwl:hasRelatedSynonym" content="archaea" datatype="xsd:string"></span><span id='am-233' property="oboInOwl:hasDbXref" content="PMID:11760965" datatype="xsd:string"></span><span id='am-234' property="oboInOwl:hasRelatedSynonym" content="prokaryotes" datatype="xsd:string"></span><span id='am-235' property="oboInOwl:hasRelatedSynonym" content="Monera" datatype="xsd:string"></span><span id='am-236' property="oboInOwl:hasRelatedSynonym" content="Procaryotae" datatype="xsd:string"></span><span id='am-237' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-238' property="oboInOwl:hasDbXref" content="PMID:11540071" datatype="xsd:string"></span><span id='am-239' property="oboInOwl:hasRelatedSynonym" content="Prokaryotae" datatype="xsd:string"></span><span id='am-240' property="oboInOwl:hasDbXref" content="PMID:10425795" datatype="xsd:string"></span><span id='am-241'  ></span><span id='am-242' property="oboInOwl:hasDbXref" content="PMID:11321083" datatype="xsd:string"></span><span id='am-243' property="oboInOwl:hasDbXref" content="PMID:10490293" datatype="xsd:string"></span><span id='am-244' property="oboInOwl:hasDbXref" content="PMID:270744" datatype="xsd:string"></span><span id='am-245' property="oboInOwl:hasDbXref" content="PMID:11321113" datatype="xsd:string"></span><span id='am-246' property="oboInOwl:hasDbXref" content="PMID:11411719" datatype="xsd:string"></span><span id='am-247' property="rdfs:label" content="Archaea" datatype="xsd:string"></span><span id='am-248' property="oboInOwl:hasDbXref" content="PMID:11541975" datatype="xsd:string"></span><span id='am-249' property="oboInOwl:hasDbXref" content="PMID:10939651" datatype="xsd:string"></span><span id='am-250' property="oboInOwl:hasDbXref" content="PMID:11211268" datatype="xsd:string"></span><span id='am-251' property="oboInOwl:hasDbXref" content="PMID:11542064" datatype="xsd:string"></span><span id='am-252' property="oboInOwl:hasDbXref" content="PMID:9336922" datatype="xsd:string"></span><span id='am-253' property="oboInOwl:hasRelatedSynonym" content="Metabacteria" datatype="xsd:string"></span><span id='am-254' property="oboInOwl:hasDbXref" content="PMID:8590690" datatype="xsd:string"></span><span id='am-255' property="oboInOwl:hasRelatedSynonym" content="prokaryote" datatype="xsd:string"></span><span id='am-256' property="oboInOwl:hasDbXref" content="PMID:12054223" datatype="xsd:string"></span><span id='am-257' property="oboInOwl:hasDbXref" content="PMID:10843050" datatype="xsd:string"></span><span id='am-258' property="oboInOwl:hasDbXref" content="PMID:11542149" datatype="xsd:string"></span><span id='am-259' property="oboInOwl:hasRelatedSynonym" content="Mendosicutes" datatype="xsd:string"></span><span id='am-260' property="oboInOwl:hasDbXref" content="PMID:8123559" datatype="xsd:string"></span><span id='am-261' property="oboInOwl:hasRelatedSynonym" content="Archaebacteria" datatype="xsd:string"></span><span id='am-262' property="oboInOwl:hasDbXref" content="PMID:10425796" datatype="xsd:string"></span><span id='am-263' property="oboInOwl:hasDbXref" content="PMID:10939677" datatype="xsd:string"></span>Archaea</span>) type II (Bacteria) [15].    nopeptidase <span id='am-104' about='obo:NCBITaxon_1279' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1385 obo:NCBITaxon_90964 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-105' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasDbXref" content="PMID:10319495" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasDbXref" content="PMID:9734063" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasDbXref" content="PMID:10425778" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasDbXref" content="PMID:10319469" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasDbXref" content="PMID:17220435" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasDbXref" content="PMID:12656157" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-113'  ></span><span id='am-114' property="oboInOwl:hasDbXref" content="PMID:10758876" datatype="xsd:string"></span><span id='am-115' property="rdfs:label" content="Staphylococcus" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasRelatedSynonym" content="Aurococcus" datatype="xsd:string"></span>Staphylococcus</span> aureus (labelled &quot;YMAP&quot; <br />                                                                 Figure 1). Overall proteins similar their <br /> MAP distributed living organisms,        physico-chemical properties (they closely similar <br /> plays important physiological role. The deletion      length slightly acidic). Moreover, shown Fig- <br /> single map gene prokaryotes <span id='am-334' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-335' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-336' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-337'  ></span><span id='am-338' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-339' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-340' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-341' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-342' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-343' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-344' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-345' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-346' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-347' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-348' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-349' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Escherichia coli</span> [16]    ure 1, metal (presumably cobalt) binding site (high- <br /> or <span id='am-357' about='obo:NCBITaxon_590' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-358' property="oboInOwl:hasDbXref" content="PMID:10939679" datatype="xsd:string"></span><span id='am-359' property="oboInOwl:hasDbXref" content="PMID:12072558" datatype="xsd:string"></span><span id='am-360' property="oboInOwl:hasDbXref" content="PMID:15653929" datatype="xsd:string"></span><span id='am-361' property="oboInOwl:hasDbXref" content="PMID:10319519" datatype="xsd:string"></span><span id='am-362' property="oboInOwl:hasDbXref" content="PMID:3231714" datatype="xsd:string"></span><span id='am-363' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-364' property="oboInOwl:hasDbXref" content="PMID:15653930" datatype="xsd:string"></span><span id='am-365' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-366' property="oboInOwl:hasRelatedSynonym" content="Samonella" datatype="xsd:string"></span><span id='am-367' property="rdfs:label" content="Salmonella" datatype="xsd:string"></span><span id='am-368' property="oboInOwl:hasDbXref" content="PMID:9731304" datatype="xsd:string"></span><span id='am-369'  ></span>Salmonella</span> typhimurium [17] lethal. In yeast, dele-   lighted residues) conserved. It important <br /> tion map genes (type I type II)        investigate proteins methionine ami- <br /> causes slow growth phenotype, deletion          nopeptidase activity. <br /> genes lethal [18]. Why MAP activity an <br /> essential role living cells? Previous studies showed    This difference evolution proteins considered as <br /> many proteins needed N-terminal methio-           essential related organisms (<span id='am-84' about='obo:NCBITaxon_1239' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-85' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasRelatedSynonym" content="Firmacutes" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasDbXref" content="PMID:11034484" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasRelatedSynonym" content="Gram positive bacteria" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasExactSynonym" content="low GC Gram+" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasRelatedSynonym" content="clostridial firmicutes" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasRelatedSynonym" content="Bacillus/Clostridium group" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasExactSynonym" content="Gram-positive bacteria" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasRelatedSynonym" content="Low G+C firmicutes" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasRelatedSynonym" content="Clostridium group firmicutes" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="PMID:15143038" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasExactSynonym" content="low G+C Gram-positive bacteria" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="PMID:10555317" datatype="xsd:string"></span><span id='am-99' property="rdfs:label" content="Firmicutes" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasRelatedSynonym" content="firmicutes" datatype="xsd:string"></span><span id='am-101'  ></span><span id='am-102' property="oboInOwl:hasAlternativeId" content="NCBITaxon:31968" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasDbXref" content="PMID:11542017" datatype="xsd:string"></span>Firmicutes</span>) intriguing, <br /> nine removed normal biological activity, proper         explored evolution pattern map study <br /> subcellular localization eventual degradation               meant identify persistent genes bacteria [23]. When <br /> (reviewed [19]). As result, possible MAP     matched evolution 16S RNA, map gene <br /> essential essentiality downstream tar-    does follow linear course evolution, most <br /> gets. Besides essentiality, MAP changes dynamics         ribosomal proteins [23], shows erratic pattern (Fig- <br /> the sulfur containing metabolites,            ure 2). This suggests MAP involved differ- <br /> important role homeostasis cell.                  ent biological processes role methionine <br />                                                                 aminopeptidase according organisms. This <br /> In general, map gene genome         prompted explore evolution MAP pro- <br /> prokaryotes, exception <span id='am-61' about='obo:NCBITaxon_1117' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-62' property="oboInOwl:hasExactSynonym" content="blue-green bacteria" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="PMID:9382701" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="Cyanobacteria" datatype="xsd:string"></span><span id='am-65'  ></span><span id='am-66' property="oboInOwl:hasRelatedSynonym" content="cyanobacteria" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasRelatedSynonym" content="Oxyphotobacteria" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="PMID:15388760" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="Cyanophycota" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="Cyanophyta" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasRelatedSynonym" content="Oxygenic photosynthetic bacteria" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasExactSynonym" content="cyanophytes" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasExactSynonym" content="blue-green algae" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasDbXref" content="PMID:11542017" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasRelatedSynonym" content="Cyanophyceae" datatype="xsd:string"></span>cyanobacteria</span> strain,      teins functionally related, using criteria <br /> which functional methionine aminopeptidases           used identify persistent genes. We defined persistent <br /> [20]. Surprisingly, genomes <span id='am-318' about='obo:NCBITaxon_91061' typeof='owl:Thing obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-319' property="oboInOwl:hasRelatedSynonym" content="Firmibacteria" datatype="xsd:string"></span><span id='am-320' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-321' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-322' property="oboInOwl:hasRelatedSynonym" content="Bacillus/Lactobacillus/Streptococcus group" datatype="xsd:string"></span><span id='am-323' property="oboInOwl:hasDbXref" content="PMID:20212322" datatype="xsd:string"></span><span id='am-324' property="rdfs:label" content="Bacilli" datatype="xsd:string"></span><span id='am-325'  ></span>Bacilli</span>,      genes genes present orthologs 85% <br /> more putative MAP genes detected sequence             genomes clade (here 26 Firmicutes), after <br /> alignment. In B. subtilis, putative genes responsible       removing obligatory endosymbionts list (see <br /> for MAP activity, map yflG, identified [21].         Methods). The protein divergence (sequence similarity <br /> Kobayashi et al. [22] reported map essential gene    divided sequence length difference) MAP B. subti- <br /> in B. subtilis yflG not, apparent paradox yflG   lis orthologs Firmicutes compared <br />  <br />  <br />                                                                                                                    Page 2 15 <br />                                                                                              (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                    http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br />  <br /> Figure 1 alignment MAP E. coli, B. subtilis, S. aureus <br /> Sequences <br /> Sequences alignment MAP E. coli, B. subtilis, S. aureus. The line shows protein sequence <br /> of MAP B. subtilis (MAP_Bs), second shows MAP E. coli (MAP_Ec), shows YflG B. subtilis (YflG_Bs) <br /> and shows YflG/MAP <span id='am-117' about='obo:NCBITaxon_1280' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1385 obo:NCBITaxon_90964 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_1279'><span id='am-118' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasRelatedSynonym" content="Staphilococcus aureus" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasRelatedSynonym" content="Micrococcus pyogenes" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasAlternativeId" content="NCBITaxon:325213" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasRelatedSynonym" content="Streptococcus aureus" datatype="xsd:string"></span><span id='am-123' property="rdfs:label" content="Staphylococcus aureus" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasRelatedSynonym" content="Staphylococcus pyogenes aureus" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasRelatedSynonym" content="Micrococcus aureus" datatype="xsd:string"></span><span id='am-127'  ></span><span id='am-128' property="oboInOwl:hasRelatedSynonym" content="Staphylococus aureus" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasDbXref" content="PMID:8573498" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasRelatedSynonym" content="Staphlococcus pyogenes citreus" datatype="xsd:string"></span>Staphylococcus aureus</span> (YMAP_Sa). Identical amino acids shadowed dark conserva- <br /> tive amino acids shadowed grey. The highlighted residues stand conserved amino acids important MAP <br /> activity (bold italic residues conservative basic ones, bold acidic ones). [MFY]-x-G-H-G-[LIVMC]-[GSH]- <br /> x(3)-H-x(4)-[LIVM]-x-[HN]-[YWVH] (where H cobalt ligand) conserved motif protein sequences (placed <br /> under alignment). <br />  <br />  <br />  <br />  <br /> with bacterial evolutionary distance (measured 16S            molecular mass MAP_Bs YflG_Bs was <br /> rRNA distance). Similar analyses performed           27409.17 &#177; 0.92 Da 27209.35 &#177; 2.14 Da, respectively. <br /> general protease, ClpP, DEF, ribosomal protein S4, RpsD          Interestingly, proteins purified E. coli retained <br /> and YkrB, second deformylase present B. subtilis (Fig-      initial methionine, consistent nature the <br /> ure 2). Interestingly, contrast expected evolution-    second residue, isoleucine cases. <br /> ary pattern RpsD, MAP exhibited erratic <br /> evolutionary way Def YkrB, functional             As seen Table 1, enzymes exhibited methionine <br /> deformylases B. subtilis, suggesting co-evolution          aminopeptidase activity vitro synthetic peptide <br /> methionine aminopeptidase deformylase.                       substrates tested. They differed widely, however, the <br />                                                                  extent activity: specific activity YflG_Bs was <br /> Enzyme activity vitro                                         90- 20-fold higher MAP_Bs with <br /> It recognized gene function identification         tetrapeptide MGMM tripeptide MAS sub- <br /> solely based sequence comparisons mislead-           strates, respectively. Both enzymes preferred tetrapep- <br /> ing [24,25]. Therefore, determined map          tide MGMM substrate. With MAS MG the <br /> and yflG authentic MAP genes overproducing           substrates, MAP_Bs activity 76.9% 46.2% that <br /> proteins E. coli characterizing enzymatic activ-    MGMM substrate, respectively, YflG_Bs <br /> ity. Both products map yflG genes purified        retained 17.7% 0.6% relative activity, respec- <br /> more 90% homogeneity. Mass spectrometry showed              tively, substrates. <br />  <br />  <br />                                                                                                                      Page 3 15 <br />                                                                                                (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                     http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br />  <br /> YkrB 2 <br /> Evolutionary <br /> Figure         trend analyses MAP, ClpP, Def, RpsD and <br /> Evolutionary trend analyses MAP, ClpP, Def, RpsD <br /> and YkrB. Map, ClpP, Def, RpsD YkrB picked up <br /> from B. subtilis. Their BBHs Firmicutes were <br /> detected. Their protein divergence (sequence similarity <br /> divided sequence length difference) B. subtilis their <br /> orthologs Firmicutes compared bacte- <br /> rial evolutionary distance individually (measured 16S <br /> rRNA evolution). <br />  <br />  <br />  <br />  <br /> Table 1: Reaction rate MAP_Bs YflG_Bs different <br /> peptides substrates <br />  <br />                                    Specific Activity (U/mg) <br />  <br />  Peptide (4 mM)                  MAP_Bs               YflG_Bs <br />                                                                             activity <br />                                                                    Figure 3of <br />                                                                    Relative <br />                                                                    present                  MAP_Bs (B) YflG_Bs the <br />                                                                                          ions <br />                                                                                      (A) <br />                                                                               different <br />                                                                    Relative activity (A) MAP_Bs (B) YflG_Bs in <br />                                                                    present different ions. MGMM (4 mM) used as <br />  MGMM                            2.6 &#177; 0.3           232.0 &#177; 8.2 <br />  MAS                             2.0 &#177; 0.1            41.0 &#177; 5.5 <br />                                                                    substrate. All ions tested 0.2 mM concentration. <br />  MG                              1.2 &#177; 0.1            1.3 &#177; 0.1 <br />                                                                    Each relative activity defined 100% the <br />                                                                    presence Co2+ (MAP_Bs: 2.6 U/mg; YflG_Bs: 232.0 U/mg). <br />  Three measurements proteins different <br />  substrates made. <br />  <br />  <br />                                                                    methionine aminopeptidase located C-termi- <br /> The effect divalent ions (0.2 mM) enzyme activity            nal polypeptide chain [20,26-29], N- <br /> was investigated (Figure 3). MAP_Bs significant activ-         terminal GFP fusions proteins expected <br /> ity Mn2+ Co2+. MAP_Bs retained                       interfere functions. In addition, con- <br /> appreciable activity (26&#8211;50%) replacement Co2+            structed C-terminal GFP fusions MAP_Bs and <br /> with Cd2+, Zn2+, Ca2+, Mg2+, Cu2+, Sr2+ Ba2+,          YflG_Bs, possible degradation the <br /> Ni2+. In contrast, YflG_Bs showed strong preference          fusion proteins vivo unknown reasons, we <br /> Co2+ decrease activity 20%             failed detect fluorescence case (<span id='am-468' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-469' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-470' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-471' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-472'  ></span>data</span> not <br /> other divalent ions tested.                                        shown). Cells strains BSIP8001 (MAP::GFP fusion) <br />                                                                    BSIP8002 (YflG::GFP fusion) mid-exponential <br /> Subcellular localization                                           growth phase collected visualized fluores- <br /> To investigate MAP_Bs YflG_Bs dif-             cence microscopy, respectively. Both GFP-MAP_Bs (Figure <br /> ference subcellular localization, constructed N-       4A) GFP-YflG_Bs (Figure 4B) proteins distributed <br /> terminal green fluorescent protein (GFP) fusions           evenly cells. As expected, intensity of <br /> proteins. As previously reported, catalytic domain          fluorescence proteins increased the <br />  <br />  <br />                                                                                                                       Page 4 15 <br />                                                                                                 (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                   http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br />  <br /> Figure 4 GFP-MAP_Bs GFP-YflG_Bs fusions <br /> Localization <br /> Localization GFP-MAP_Bs GFP-YflG_Bs fusions. A. Localization GFP-MAP_Bs strain BSIP8001 grown in <br /> minimal medium 0.5% xylose (w/v); B. Localization GFP-YflG_Bs strain BSIP8002 grown minimal medium 1% <br /> xylose (w/v). <br />  <br />  <br />  <br />  <br /> increasing concentration xylose used induce expres-        map expression considerably higher yflG <br /> sion GFP fusions (0.1%- 1% (w/v)) (data               expression <br /> shown).                                                          The fact map essential yflG [22], <br />                                                                  ability yflG gene rescue MAP_Bs <br /> MAP-defective mutants rescued MAP_Bs YflG_Bs           function supplied high level, points substan- <br /> supplied trans                                                tial differences expression level genes. To <br /> The E. coli mutant EM9 contains engineered map gene,          test possibility, expression map yflG genes <br /> whose expression control isopropyl &#946;-D-          B. subtilis vivo studied using lacZ reporter <br /> thiogalactoside (IPTG), strain grow           gene. <br /> unless IPTG added growth medium [16]. Using <br /> the replicative vector pBAD, holds arabinose-           Strains BSHP7042 BFS4611 grown minimal <br /> inducible promoter, map yflG genes B.               medium used monitor expression map <br /> subtilis individually introduced strain EM9. As        yflG, respectively. The map gene showed higher <br /> shown Figure 5, transformants complement            promoter activity (Figure 6). Its activity increased gradu- <br /> the map defect IPTG omitted (leading MAP           ally log phase growth nearly four-fold <br /> deficiency) high concentration arabinose sup-        increase detected (109&#8211;379 U/mg) early <br /> plied, case plasmids carrying B. subtilis map   exponential stationary growth phases. In contrast, the <br /> or yflG genes. Lower concentrations arabinose par-       yflG gene showed low activity the <br /> tially supported growth. This demonstrated map         conditions tested (4&#8211;8 U/mg) (Figure 6). In summary, the <br /> and yflG B. subtilis supply MAP function E.      map gene promoter showed 50 100-fold higher activity <br /> coli vivo.                                                    yflG using lacZ reporter gene. <br />  <br /> In B. subtilis, map essential gene, yflG      Promoter localization yflG gene <br /> [22]. However, back-up map yflG copy         In B. subtilis genome, yflG gene located upstream <br /> control IPTG provided multi-copy plasmid pDG148            yflH gene, followed putative Rho-independent <br /> (30&#8211;50 copies using pUB110 replicon B. subtilis         transcription terminator. RT-PCR experiments demon- <br /> [30]), map::lacZ disruptants chromosome B. sub-        strated yflG yflH belonged common tran- <br /> tilis obtained (BSHP7046 BSHP7037). Never-              scription unit, making operon (data shown). <br /> theless, witnessed cases systematic            Furthermore, lacZ transcription fusion yflH gene <br /> formation variegated blue white streaked colonies          (strain BSHP7043) showed activity changes as <br /> X-gal plates, clones extremely unstable, leading        yflG gene conditions tested (data not <br /> to loss transformants generations (data       shown), substantiating RT-PCR results. In <span id='am-315' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-316' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-317' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> to <br /> not shown).                                                      identify promoter yflG-yflH operon, carried <br />                                                                  primer extension analysis. Two primers complemen- <br />                                                                  tary different regions inside yflG gene used <br />                                                                  yielding identification transcription start <br />  <br />  <br />                                                                                                                      Page 5 15 <br />                                                                                                (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                  http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br />  <br />                                                                Expression <br />                                                                Figure 6 map yflG lacZ transcriptional fusions <br />                                                                Expression map yflG lacZ transcriptional <br />                                                                fusions. &#946;-galactosidase expression map yflG lacZ <br />                                                                transcriptional fusion growth curve strains <br />                                                                BSHP7042 BFS4611 minimal medium. OD600 (&#65533;??) and <br />                                                                &#946;-galactosidase activity (&#65533;?) measured strain <br />                                                                BSHP7042 map lacZ transcription fusion. OD600 (&#65533;?) and <br />                                                                &#946;-galactosidase activity (&#9632;) measured strain <br />                                                                BFS4611 yflG lacZ transcription fusion. <br />  <br />  <br />  <br /> Figure 5rescue experiments <br /> Genetic <br /> Genetic rescue experiments. The bacteria were <br /> streaked M63 minimal solid medium plates were <br /> incubated 37&#176;C overnight. The concentration IPTG     point correlate structure promoter with <br /> 1 mM. The order plates picture described        poor expression yflG conditions tested. <br /> below left right bottom. A. EM9 <br /> host strain M63 IPTG; EM9 M63 IPTG. B. <br /> EM9 strain pBAD vector (ECHP1007) M63            YvoA showed slight repression effect yflG expression <br /> with IPTG; ECHP1007 M63 arabinose (0.5% (W/V));        As expression yflG B. subtilis vivo low, <br /> ECHP1007 M63 IPTG arabinose. C. EM9             endeavoured uncover conditions enhance its <br /> strain caring pBAD map vector (ECHP1005) M63           expression. No significant difference the <br /> arabinose (0.5% (W/V)); ECHP1005 M63 IPTG;             carbon, nitrogen sulfur sources changed, no <br /> ECHP1005 M63 IPTG; ECHP1005 M63             difference conditions sporulation ger- <br /> arabinose (0.02% (W/V)); ECHP1005 M63 arabinose        mination (data shown). As genes proximal the <br /> (0.1% (W/V)). D. EM9 strain caring pBAD yflG vector            chromosome code proteins related <br /> (ECHP1006) M63 arabinose (0.5% (W/V));                 functions, neighbourhood yflG analysed. Gene <br /> ECHP1006 M63 IPTG; ECHP1006 M63             nagP, encodes putative phosphoenolpyruvate- <br /> IPTG; ECHP1006 M63 arabinose (0.1% (W/V)); <br />                                                                dependent transport N-acetylglucosamine-specific <br /> ECHP1006 M63 arabinose (0.02% (W/V)). <br />                                                                enzyme IICB component, located upstream yflG and <br />                                                                transcribed divergently (Figure 8A). It share com- <br />                                                                mon control elements. Exploring corresponding inter- <br />                                                                genic region sliding window 19 nt-long we <br />                                                                uncovered sequence AATTGGTATAGATCACTAG which <br />                                                                significant counterpart (AGCTGGTCTAGAT- <br /> point. As shown Figure 7, start located 31 nt        CACTAG) upstream nagAB operon, encoding two <br /> upstream ATG translation start <span id='am-310' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-311' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-312'  ></span><span id='am-313' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-314' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span> regions        N-acetylglucosamine metabolic genes putative tran- <br /> weakly similar consensus -35 (TTCCTA) -10               scriptional regulator (GntR <span id='am-271' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-272' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-273' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span>) gene, yvoA (Figure <br /> (TAAGCT) regions upstream start point,       8A). This prompted investigate YvoA a <br /> separated 18 nt, AT-rich region. It difficult   possible regulator yflG. Using lacZ fusion, we <br />  <br />  <br />                                                                                                                    Page 6 15 <br />                                                                                              (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                   http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br />  <br /> Figure <br /> Identification <br />          7     transcription start point yflG promoter <br /> Identification transcription start point yflG promoter. Primer extension sequencing reaction were <br /> performed primer. The size extended product compared DNA-sequencing ladder corre- <br /> sponding region. An arrow indicates +1 site. Predicted -10 -35 regions bold boxed; experimental tran- <br /> scription start site bold; translational start site underlined. <br />  <br />  <br />  <br />  <br /> monitored expression yflG yvoA-disrupted            vitro (Table 1 Figure 3). Furthermore, gene rescue <br /> strain, BSIP8004.                                               experiments showed map yflG genes <br />                                                                 expressed replicative plasmids supply MAP <br /> As shown Figure 8B, promoter yflG showed low        function E. coli B. subtilis, demonstrating both <br /> but consistently higher activity yvoA gene disrupted     MAP_Bs YflG_Bs functional methionine ami- <br /> strain BSIP8004 compared wild type par-           nopeptidases vivo (Figure 5). Interestingly, Kobayashi et <br /> ent BFS4611 (two-fold higher activity). This suggested          al. [22] reported disruption map gene B. sub- <br /> that YvoA contribute regulation yflG            tilis lethal, deletion yflG not. In the <br /> expression.                                                     present study, obtained B. subtilis map mutants when <br />                                                                 map yflG genes expressed multi-copy replicative <br /> Discussion                                                      plasmids control IPTG induced promoter <br /> In prokaryotes, usually accepted   (strains BSHP7046 BSHP7037). This observation sug- <br /> gene responsible methionine aminopeptidase. How-            gests single copy yflG gene expressed the <br /> ever, B. subtilis genome program predicted              chromosome supply MAP function B. <br /> existence methionine aminopeptidase genes, map           subtilis, substantiating conditions its <br /> and yflG [21]. We shown MAP_Bs               expression extremely low, making MAP signif- <br /> YflG_Bs functional methionine aminopeptidases            icant methionine aminopeptidase available. This is <br />  <br />  <br />                                                                                                                     Page 7 15 <br />                                                                                               (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                    http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br />                                                                  phosphate deaminase activity modulated amino- <br />                                                                  terminal methionine enzyme [31]. We tried several <br />                                                                  growth conditions involving N-acetylglucosamine a <br />                                                                  carbon nitrogen source, both, did any <br />                                                                  modulation YflG activity (data shown). We also <br />                                                                  explored expression yflG::lacZ fusions sporula- <br />                                                                  tion germination. No enhancement observed <br />                                                                  (data shown). As consequence, surmise the <br />                                                                  high expression level map accounts essentiality in <br />                                                                  B. subtilis. <br />  <br />                                                                  The published MAP_Ec activity [6,11] 5&#8211;10 times <br />                                                                  higher MAP_Bs assay conditions. This <br />                                                                  related observation map gene E. <br />                                                                  coli does belong highly expressed ribosome pro- <br />                                                                  tein operon, contrast situation B. subtilis [32]. <br />                                                                  As result, relative expression level map E. coli <br />                                                                  lower map gene B. subtilis. For <br />                                                                  unknown reasons impossible yflG <br />                                                                  gene function main methionine aminopeptidase <br />                                                                  B. subtilis vivo spite high activity YflG_Bs <br />                                                                  vitro. As shown Table 2, enzyme activity MAPs <br />                                                                  crude cell extracts E. coli (BL5), B. subtilis (168) and <br />                                                                  yflG disrupted B. subtilis strain (BFS4611) showed not <br />                                                                  difference vitro condition tested. <br />                                                                  Different expression levels map genes genome <span id='am-428' about='obo:NCBITaxon_201850' typeof='owl:Thing obo:NCBITaxon_1125798 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71529 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-429' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-430' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-431'  ></span><span id='am-432' property="rdfs:label" content="Car" datatype="xsd:string"></span>car</span>- <br />                                                                  rying methionine aminopeptidase genes <br />                                                                  previously suggested main reason for <br />                                                                  different physiological roles vivo, direct <br />                                                                  evidence            provided       [20,33].     Methionine <br />                                                                  aminopeptidases vivo active proteins, i.e. on <br />                                                                  complex polypeptides [34,35] short pep- <br />                                                                  tides vitro assays. It specific tar- <br /> Expression <br /> Figure <br /> rupted strain <br />          8 yflG lacZ transcriptional fusions yvoA dis-     gets require specific aminopeptidase. A similar <br /> Expression yflG lacZ transcriptional fusions yvoA          conclusion holds apparently paralogous genes. <br /> disrupted strain. A. gene organization yflG           For example, functional peptide <br /> yvoA. The arrows indicate direction gene transcription;   deformylases B. subtilis, probably the <br /> B. &#946;-galactosidase expression yflG lacZ transcriptional fusion   predominant deformylase concomitant high gene <br /> in yvoA disrupted strain BSIP8004 yvoA wild type strain      expression [36]. <br /> BFS4611 grown minimal medium. OD600 (&#65533;??) &#946;-galac- <br /> tosidase activity (&#65533;?) measured strain BSIP8004. <br /> OD600 (&#65533;?) &#946;-galactosidase activity (&#9632;) measured         The finding co-evolutionary parallel trend of <br /> for strain BSIP8004 strain BFS4611.                          methionine aminopeptidase deformylase Firmi- <br />                                                                  cutes (Figure 2) consistent functional relationship <br />                                                                  vivo, correlates fact deformy- <br />                                                                  lase removes N-formyl groups prerequisite the <br />                                                                  subsequent function MAP [33]. This co-evolution may <br /> corroborated lacZ fusion studies showing         signature protein-protein interactions [37,38], <br /> map gene expressed 50&#8211;100-fold higher yflG          intriguing conjecture, supported by <br /> under conditions tested. Despite weak expression         consistent duplication deformylases genomes <br /> of yflG, however, detected sigma A promoter using           methionine aminopeptidase (Table <br /> primer extension (Figure 7). We showed         3). <br /> expression modulated YvoA (Figure 8), suggesting a <br /> possible connection N-acetylglucosamine                  Conclusion <br /> metabolism methionine aminopeptidase. In                We proved MAP_Bs YflG_Bs functional <br /> respect worth noting E. coli glucosamine-6-        methionine aminopeptidases B. subtilis we <br />  <br />  <br />                                                                                                                       Page 8 15 <br />                                                                                                 (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                                   http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br /> Table 2: MAP activity crude cell extracts E. coli B. subtilis <br />  <br />                                                                                 Specific Activity (U/mg) <br />  <br />          Peptide (4 mM)                           E. coli (BL5)                  B. subtilis wild type (168)        B. subtilis yflG mutant (BFS4611) <br />  <br />               MGMM                                 1.5 &#177; 0.2                             1.5 &#177; 0.1                             1.5 &#177; 0.1 <br />                MAS                                 1.1 &#177; 0.2                             1.1 &#177; 0.1                             1.1 &#177; 0.2 <br />                MG                                  0.6 &#177; 0.1                             0.8 &#177; 0.2                             0.4 &#177; 0.1 <br />  <br />  Cells stationary growth phase of strains collected sonically disrupted. Supernatants of cell debris used to <br />  measure MAP activity. Totally, times measurements carried out. <br />  <br />  <br />  <br /> suggested map gene essential high                     All experiments performed accordance the <br /> expression level, yflG nonessential possibly                        European regulation requirements concerning con- <br /> because low expression level making                   tained use Genetically Modified Organisms Group-I <br /> take compensate function map map                      (French agreement N&#176; 2735). <br /> not expressed, specific targets dedicated to <br /> only MAPs. Conservation map-                       DNA techniques <br /> like genes Bacilli suggests involvement process                      DNA purification, restriction enzyme digestion, ligation <br /> specific group organisms, involve                       transformation E. coli performed according to <br /> also peptide deformylase.                                                    standard protocols [40]. For cloning purpose, YieldAce or <br />                                                                              Pfu DNA polymerase (Stratagene) used. All DNA <br /> Methods                                                                      sequences plasmids constructed work were <br /> Bacterial strains, plasmids, culture conditions                          determined using dideoxy-chain termination method <br /> E. coli B. subtilis strains plasmids used             Thermo Sequenase Kit (Amersham Pharmacia <br /> work listed Table 4. E. coli TG1 XL1-Blue                    Biotech). B. subtilis cells transformed plasmid <br /> used cloning experiments (TG1 single cross-over                      DNA following two-steps protocol described previ- <br /> recombination replicative plasmids propagation                        ously [41]. Transformants selected LB plates con- <br /> XL1-Blue double cross-over recombination) E.coli                     taining corresponding antibiotics IPTG needed. <br /> BL5 IPTG-induced protein overproducing. B. subtilis <br /> 168 used wild type strain study. E. coli                  Plasmid strain constructions <br /> cells grown <span id='am-220' about='obo:NCBITaxon_218032' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_69557 obo:NCBITaxon_69555 obo:NCBITaxon_69556 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-221' property="rdfs:label" content="Luria" datatype="xsd:string"></span><span id='am-222'  ></span><span id='am-223' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-224' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Luria</span>-Bertani (LB) medium [39]                     In order clone genes B. subtilis protein pro- <br /> M63 minimal medium (KH2PO4, 4.4 mM; K2HPO4, 8                                ducing E. coli, map yflG genes amplified <br /> mM; (NH4)2SO4, 15 mM; MgSO4, 2 mM; ferric citrate, 34                        PCR genomic DNA B. subtilis 168 cloned <br /> &#181;M; VitB1, 0.0001% (w/v); fructose, 0.4% (w/v); sodium                       expression vector pET24b(+) (Novagen) using NdeI <br /> citrate, 0.3 mM; CaCl2, 50 &#181;M; MnCl2, 5 &#181;M; ZnCl2, 12 &#181;M;                    XhoI restriction sites. The amplified fragments <br /> CuCl2, 2.5 &#181;M; CoCl2, 2.5 &#181;M; Na2MoO4, 2.5 &#181;M)                               included nucleotides +1 +744 nucleotides +1 to <br /> supplemented final concentration arabinose                         +747 relative translational start point map and <br /> from 0.02% 0.5% (w/v) needed. B. subtilis cells                      yflG, respectively. The resulting constructs, plasmids <br /> were grown SP medium minimal medium                                 pIPP8003 pIPP8004, transformed E. coli <br /> (K2HPO4, 8 mM; KH2PO4, 4,4 mM; glucose, 27 mM; Na3-                          BL5 giving strains ECIP8003 ECIP8004 overpro- <br /> citrate, 0.3 mM; L-glutamine, 15 mM; L-tryptophan, 0.244                     ducing MAP YflG, respectively. <br /> mM; ferric citrate, 33.5 &#181;M; MgSO4 L-methionine, 1 <br /> mM; MgCl2, 0.61 mM; CaCl2, 49.5 &#181;M; FeCl3, 49.9 &#181;M;                          To construct map yflG GFP fusion plasmids and <br /> MnCl2, 5.05 &#181;M; ZnCl2, 12.4 &#181;M; CuCl2, 2.52 &#181;M; CoCl2,                       strains, map yflG genes amplified PCR <br /> 2.5 &#181;M; Na2MoO4, 2.48 &#181;M). Antibiotics added                         genomic DNA B. subtilis 168 cloned N- <br /> the following concentrations required: ampicillin,                      terminal GFP fusion vector pSG1729 [42] using XhoI and <br /> 100 &#181;g/ml; spectinomycin, 80 &#181;g/ml; chloramphenicol, 5                       EcoRI restriction sites. The amplified fragments included <br /> &#181;g/ml; phleomycin, 5 &#181;g/ml; kanamycin, 100 &#181;g/ml E.                      nucleotides +1 +744 nucleotides +1 +747 <br /> coli 5 &#181;g/ml B. subtilis. IPTG added 1 mM                     relative translational start point map yflG, <br /> concentration stated needed. Solid media                     respectively. The resulting plasmids, pIPP8001 and <br /> prepared adding 1.5% agar (w/v) respective liq-                    pIPP8002, linearized ScaI restriction enzyme <br /> uid media. Bacteria grown 37&#176;C. The optical den-                     transformed B. subtilis 168. Clones were <br /> sity (OD) bacterial cultures measured 600 nm. <br />  <br />  <br />                                                                                                                                         Page 9 15 <br />                                                                                                                  (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                    http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br /> Table 3: Distribution def map genes selected genomes <br />  <br />                                                                        Numbers Each Genome <br />  <br />  Genome Name                                                       def                    map                        EMBL ID <br />  <br />  <span id='am-16' about='obo:NCBITaxon_1883' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_2062 obo:NCBITaxon_201174 obo:NCBITaxon_85003 obo:NCBITaxon_85011'><span id='am-17' property="oboInOwl:hasAlternativeId" content="NCBITaxon:65506" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="PMID:11411701" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1973" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="PMID:9336904" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="PMID:4886644" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="PMID:8934906" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="PMID:8590678" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="PMID:15950131" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="PMID:16560709" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="PMID:1371059" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="PMID:9731302" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="PMID:10843062" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasAlternativeId" content="NCBITaxon:31945" datatype="xsd:string"></span><span id='am-32'  ></span><span id='am-33' property="rdfs:label" content="Streptomyces" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Chainia" datatype="xsd:string"></span>Streptomyces</span> coelicolor                                           4                        3                        AL645882 <br />  <span id='am-511' about='obo:NCBITaxon_33903' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_2062 obo:NCBITaxon_201174 obo:NCBITaxon_1883 obo:NCBITaxon_85003 obo:NCBITaxon_85011'><span id='am-512' property="oboInOwl:hasRelatedSynonym" content="Streptomyces avermectinius" datatype="xsd:string"></span><span id='am-513' property="oboInOwl:hasDbXref" content="PMID:12508861" datatype="xsd:string"></span><span id='am-514' property="oboInOwl:hasDbXref" content="PMID:464561" datatype="xsd:string"></span><span id='am-515' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-516' property="oboInOwl:hasDbXref" content="PMID:12508884" datatype="xsd:string"></span><span id='am-517' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-518' property="rdfs:label" content="Streptomyces avermitilis" datatype="xsd:string"></span><span id='am-519'  ></span>Streptomyces avermitilis</span>                                          3                        4                        BA000030 <br />  <span id='am-35' about='obo:NCBITaxon_1817' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_85025 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003'><span id='am-36'  ></span><span id='am-37' property="rdfs:label" content="Nocardia" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Nocardia Trevisan 1889" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="PMID:7547284" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="PMID:7537058" datatype="xsd:string"></span>Nocardia</span> farcinica IFM 10152                                      3                        3                        AP006618 <br />  <span id='am-296' about='obo:NCBITaxon_211586' typeof='owl:Thing obo:NCBITaxon_267890 obo:NCBITaxon_1 obo:NCBITaxon_70863 obo:NCBITaxon_22 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_135622'><span id='am-297' property="oboInOwl:hasRelatedSynonym" content="Shewanella oneidensis ATCC 700550" datatype="xsd:string"></span><span id='am-298' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-299' property="oboInOwl:hasExactSynonym" content="Shewanella oneidensis str. MR-1" datatype="xsd:string"></span><span id='am-300' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-301' property="oboInOwl:hasRelatedSynonym" content="Shewanella sp. MR-1" datatype="xsd:string"></span><span id='am-302' property="oboInOwl:hasAlternativeId" content="NCBITaxon:60479" datatype="xsd:string"></span><span id='am-303' property="oboInOwl:hasExactSynonym" content="Shewanella oneidensis strain MR-1" datatype="xsd:string"></span><span id='am-304' property="rdfs:label" content="Shewanella oneidensis MR-1" datatype="xsd:string"></span>Shewanella oneidensis MR-1</span>                                        3                        1                        AE014299 <br />  Silicibacter pomeroyi DSS-3                                       3                        1                        CP000031 <br />  <span id='am-131' about='obo:NCBITaxon_1396' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_86661 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-132' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451271" datatype="xsd:string"></span><span id='am-133'  ></span><span id='am-134' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1271755" datatype="xsd:string"></span><span id='am-135' property="oboInOwl:hasAlternativeId" content="NCBITaxon:941952" datatype="xsd:string"></span><span id='am-136' property="rdfs:label" content="Bacillus cereus" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasAlternativeId" content="NCBITaxon:941951" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1158119" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasAlternativeId" content="NCBITaxon:540964" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1219400" datatype="xsd:string"></span><span id='am-141' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1219392" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1219409" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-144' property="oboInOwl:hasRelatedSynonym" content="Bacillus medusa" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1219408" datatype="xsd:string"></span><span id='am-147' property="oboInOwl:hasAlternativeId" content="NCBITaxon:941950" datatype="xsd:string"></span><span id='am-148' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429436" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:hasRelatedSynonym" content="Bacillus endorhythmos" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:hasDbXref" content="PMID:9542081" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1169733" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:hasRelatedSynonym" content="Bacillus sp. JKR66" datatype="xsd:string"></span>Bacillus cereus</span> ZK                                                2                        3                        CP000001 <br />  <span id='am-6' about='obo:NCBITaxon_180856' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1428 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_86661 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Bacillus thuringiensis serovar konkukian" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Bacillus thuringiensis subsp. konkukian" datatype="xsd:string"></span>Bacillus thuringiensis serovar konkukian</span>                          2                        3                        AE017355 <br />  <span id='am-153' about='obo:NCBITaxon_1392' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_86661 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-154' property="oboInOwl:hasExactSynonym" content="anthrax" datatype="xsd:string"></span><span id='am-155' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-156' property="rdfs:label" content="Bacillus anthracis" datatype="xsd:string"></span><span id='am-157' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-158' property="oboInOwl:hasRelatedSynonym" content="Bacteridium anthracis" datatype="xsd:string"></span><span id='am-159' property="oboInOwl:hasExactSynonym" content="anthrax bacterium" datatype="xsd:string"></span><span id='am-160' property="oboInOwl:hasDbXref" content="PMID:8123566" datatype="xsd:string"></span><span id='am-161' property="oboInOwl:hasRelatedSynonym" content="Bacillus cereus var. anthracis" datatype="xsd:string"></span><span id='am-162'  ></span>Bacillus anthracis</span> &apos;Ames Ancestor&apos;                                2                        3                        AE017334 <br />  Bacillus anthracis Sterne                                         2                        3                        AE017225 <br />  <span id='am-388' about='obo:NCBITaxon_222523' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1396 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_86661 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-389' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-390' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-391' property="oboInOwl:hasExactSynonym" content="Bacillus cereus str. ATCC 10987" datatype="xsd:string"></span><span id='am-392' property="rdfs:label" content="Bacillus cereus ATCC 10987" datatype="xsd:string"></span><span id='am-393' property="oboInOwl:hasDbXref" content="PMID:14960714" datatype="xsd:string"></span><span id='am-394' property="oboInOwl:hasExactSynonym" content="Bacillus cereus strain ATCC 10987" datatype="xsd:string"></span>Bacillus cereus ATCC 10987</span>                                        2                        3                        AE017194 <br />  Bacillus anthracis Ames                                           2                        3                        AE016879 <br />  <span id='am-447' about='obo:NCBITaxon_226900' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1396 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_86661 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-448' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-449' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-450' property="oboInOwl:hasRelatedSynonym" content="Bacillus cereus (strain ATCC 14579 / DSM 31)" datatype="xsd:string"></span><span id='am-451' property="oboInOwl:hasDbXref" content="PMID:12721630" datatype="xsd:string"></span><span id='am-452' property="rdfs:label" content="Bacillus cereus ATCC 14579" datatype="xsd:string"></span>Bacillus cereus ATCC 14579</span>                                        2                        3                        AE016877 <br />  <span id='am-206' about='obo:NCBITaxon_1575' typeof='owl:Thing obo:NCBITaxon_85006 obo:NCBITaxon_1 obo:NCBITaxon_110932 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_85023 obo:NCBITaxon_201174 obo:NCBITaxon_85003'><span id='am-207' property="oboInOwl:hasRelatedSynonym" content="Clavibacter xyli" datatype="xsd:string"></span><span id='am-208' property="oboInOwl:hasDbXref" content="PMID:12501368" datatype="xsd:string"></span><span id='am-209' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-210' property="rdfs:label" content="Leifsonia xyli" datatype="xsd:string"></span><span id='am-211' property="oboInOwl:hasDbXref" content="PMID:10826825" datatype="xsd:string"></span><span id='am-212' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-213'  ></span>Leifsonia xyli</span> subxyli CTCB07                                     2                        3                        AE016822 <br />  <span id='am-78' about='obo:NCBITaxon_1142' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_1301283 obo:NCBITaxon_1118 obo:NCBITaxon_1117'><span id='am-79'  ></span><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="Synechococystis" datatype="xsd:string"></span><span id='am-82' property="rdfs:label" content="Synechocystis" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Synechocystis</span> PCC 6803                                            1                        3                        BA000022 <br />  <span id='am-305' about='obo:NCBITaxon_79880' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-306'  ></span><span id='am-307' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-308' property="rdfs:label" content="Bacillus clausii" datatype="xsd:string"></span><span id='am-309' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Bacillus clausii</span> KSM-K16                                          1                        3                        AP006627 <br />  <br />  <br />  <br />  <br /> selected amylase deficient phenotype double        introducing SalI cloning site 5&apos; end SphI <br /> cross-over recombination amyE site [42], giving strains       cloning site 3&apos; end, respectively. Both fragments <br /> BSIP8001 BSIP8002, respectively.                             inserted SalI SphI sites IPTG- <br />                                                                  inducible replicative vector pDG148 [44], producing <br /> To obtain map yflG expression plasmids           plasmid pHPP7021 pHPP7022, respectively. The <br /> map gene rescue analysis conditional map gene dele-         wild type strain B. subtilis 168 transformed these <br /> tion E. coli strain, EM9 [16], map (nucleotides -30      plasmids, giving strains BSHP7045 BSHP7038, <br /> +744 relative map translation start point) yflG       respectively. <br /> (nucleotides -21 +754 relative yflG translation <br /> start point) genes amplified PCR genomic            To disrupt map gene, 246 bp-long fragment within <br /> DNA B. subtilis 168 creation BamHI            map gene (nucleotides +142 +388 relative the <br /> BglII sites. Both purified PCR fragments cleaved         map translation start point) amplified PCR from <br /> BamHI BglII restriction enzymes inserted                 genomic DNA B. subtilis 168 using primers introducing <br /> between sites plasmid pBAD containing            EcoRI cloning site 5&apos; end BamHI cloning <br /> arabinose inducible promoter (a modified pBAD-&#945;&#946;&#947;                site 3&apos; end fragment. This fragment then <br /> plasmid containing additional BamHI cloning site          inserted EcoRI BamHI sites plasmid <br /> used removing BamHI BglII insert [43]), pro-         pJM783 [45], contains promoterless lacZ reporter <br /> ducing plasmids pHPP1017 pHPP1018, respectively.             gene, producing plasmid pHPP7025. The plasmid was <br /> The proper orientation checked ability             introduced chromosome BSHP7045 and <br /> cloned fragment recovered double digestion              BSHP7038 strains single cross-over event, giving <br /> using BamHI BglII restriction enzymes. Plasmids              strains BSHP7046 BSHP7037, respectively. To ascer- <br /> pHPP1017, pHPP1018 pBAD vector trans-               tain correctly inserted, clones checked <br /> formed E. coli strain EM9 individually selected         PCR. <br /> on LB plates containing ampicillin, chloramphenicol and <br /> IPTG, giving strains ECHP1005, ECHP1006                      To construct map transcriptional fusion lacZ <br /> ECHP1007, respectively.                                          gene, DNA fragment downstream map gene <br />                                                                  (nucleotides + 423 relative translation start <br /> To obtain back-up map yflG copies con-         point gene&apos;s stop codon) amplified PCR <br /> ditional map disruptant construction B. subtilis, map     genomic DNA B. subtilis 168 using primers intro- <br /> gene (nucleotides -43 +744 relative map transla-       ducing EcoRI cloning site 5&apos; end BamHI <br /> tion start point) yflG gene (nucleotides -54 733 rel-     cloning site 3&apos; end fragment, inserted <br /> ative yflG translation start point) amplified      EcoRI BamHI sites plasmid pJM783 [45], <br /> PCR genomic DNA B. subtilis 168 using primers            producing plasmid pHPP7017. The plasmid intro- <br />  <br />  <br />                                                                                                                      Page 10 15 <br />                                                                                                 (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                                        http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br /> Table 4: Bacterial strains plasmids used created study <br />  <br />  Strains             Genotype                                                                                     Source Reference <br />  <br />  Escherichia coli <br />  <br />  XL1-Blue            K12 supE44 hsdR17 recA1 endA1 gyrA46 thi relA1 lac- F&apos;[proAB+ lacIq lacZ&#8710;M15 Tn10(tetR)]     Laboratory collection <br />  TG1                 K12 supE hsd&#8710;5 thi &#8710; (lac-proAB) F&apos;[traD36 proA+ proB+ lacIq lacZ&#8710;M15]                       Laboratory collection <br />  BL5(DE3)            F-                                                                                           Invitrogen <br />  EM9                 PT7-map (on chromosome)                                                                      <span id='am-458' about='obo:NCBITaxon_207382' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_166306 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_7055 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_75546 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_41086 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-459' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-460'  ></span><span id='am-461' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-462' property="rdfs:label" content="Chiron" datatype="xsd:string"></span>Chiron</span> Culture Collection Centre <br />  ECIP8003            F- pIPP8003                                                                                  This work <br />  ECIP8004            F- pIPP8004                                                                                  This work <br />  ECHP1005            EM9 pHPP1017                                                                                 This work <br />  ECHP1006            EM9 pHPP1018                                                                                 This work <br />  ECHP1007            EM9 pBAD                                                                                     This work <br />  <br />  Bacillus subtilis <br />  <br />  168                 trpC2                                                                                        Laboratory collection <br />  BFS4611             trpC2 yflG::lacZ                                                                             Functional analysis projecta <br />  BFS817              trpC2 yvoA::lacZ                                                                             Functional analysis projecta <br />  BSHP7042            trpC2 map-lacZ                                                                               This work <br />  BSHP7045            trpC2 pDG148-map                                                                             This work <br />  BSHP7046            trpC2 map::lacZ pDG148-map                                                                   This work <br />  BSHP7037            trpC2 map::lacZ pDG148-yflG                                                                  This work <br />  BSHP7038            trpC2 pDG148-yflG                                                                            This work <br />  BSHP7043            trpC2 yflH::lacZ                                                                             This work <br />  BSIP8001            trpC2 amyE::gfp-map                                                                          This work <br />  BSIP8002            trpC2 amyE::gfp-yflG                                                                         This work <br />  BSIP8003            trpC2 yvoA::[(lacZ-ery)::pEC23(Kan)]                                                         This work <br />  BSIP8004            trpC2 yflG::lacZ yvoA::[(lacZ-ery)::pEC23(Kan)]                                              This work <br />  <br />  Plasmids            Description                                                                                  Source Reference <br />  <br />  pET24b(+)           expression vector, KanR                                                                      Novagen <br />  pBAD-abg            genetic rescue vector, AmpR                                                                  [43] <br />  pDG148              expression vector, AmpR KanR                                                                 [44] <br />  pJM783              cloning vector, CmR                                                                          [45] <br />  pSG1729             N-terminal GFP fusion vector, AmpR SpecR                                                     [42] <br />  pEC23               integrative plasmid, KanR                                                                    M. Simon P. Stragier, unpublished <br />  pIPP8003            pET24b map                                                                                   This work <br />  pIPP8004            pET24b yflG                                                                                  This work <br />  pHPP1017            pBAD map                                                                                     This work <br />  pHPP1018            pBAD yflG                                                                                    This work <br />  pHPP7017            pJM783 map-lacZ                                                                              This work <br />  pHPP7018            pJM783 yflH::lacZ                                                                            This work <br />  pHPP7021            pDG148 map                                                                                   This work <br />  pHPP7022            pDG148 yflG                                                                                  This work <br />  pHPP7025            pJM783 map::lacZ                                                                             This work <br />  pIPP8001            pSG1729 map                                                                                  This work <br />  pIPP8002            pSG1729 yflG                                                                                 This work <br />  <br />  This   strain constructed frame project functional characterization genome B. subtilis Japan [46]. <br />  <br />  <br />  <br /> duced chromosome B. subtilis 168 single                          nese project functional analysis genome of <br /> crossover event, giving strain BSHP7042.                                          B. subtilis, 2000 genes dis- <br />                                                                                   rupted fusion lacZ reporter gene [22,46,47]. <br /> The yflG transcriptional fusion lacZ gene <br /> (yflG::lacZ disruptant, strain BFS4611) constructed                           To construct yflH transcriptional fusion lacZ <br /> within framework European Union Japa-                              gene, DNA fragment downstream yflH gene <br />  <br />  <br />                                                                                                                                            Page 11 15 <br />                                                                                                                       (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                 http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br /> (nucleotides + 32 +310 relative translation     o-dianisidine dihydrochloride 0.1 M Tris-HCl (pH <br /> start point) amplified PCR genomic DNA          7.4)), tube incubated 37&#176;C 10 min, the <br /> B. subtilis 168 using primers introducing EcoRI cloning     absorbance 440 nm recorded. Standard curve with <br /> site 5&apos; end BamHI cloning site 3&apos; end   known concentration L-methionine colour <br /> the fragment, inserted EcoRI BamHI sites     development buffer plotted quantitative analysis. <br /> of plasmid pJM783 [45], producing plasmid pHPP7018.            The absorbance 1 &#181;mol methionine ml 440 <br /> The plasmid introduced chromosome B.           nm equivalent 8.6 [6]. One Unit activity was <br /> subtilis 168 single cross-over event, giving strain       defined 1 &#181;mol methionine produced min under <br /> BSHP7043.                                                      assay conditions used. Preference metal ions was <br />                                                                tested changing cobalt substrate solution <br /> To obtain disruptant yvoA strain compatible           divalent metal ions final concentration of <br /> BFS4611, BFS817 strain (yvoA::lacZ disruptant,        0.2 mM. <br /> the European Union Japanese consortium, above) <br /> was transformed ScaI linearized pEC23 plasmid         Amylase activity detected growth B. subtilis <br /> (which carries kanamycin resistance gene, M. Simon       strains Tryptose Blood Agar Base (TBAB, Difco) supple- <br /> P. Stragier, unpublished) replacement kan-        mented 10 g/l hydrolyzed starch (Sigma). Starch <br /> amycin resistance gene lacZ Erm genes belong-       degradation detected sublimating iodine the <br /> ing pMutin plasmid (erythromycin resistant)             plates [49]. <br /> integrated genome, homologous recombina- <br /> tion. The resulting clones checked inability    &#946;-galactosidase specific activity measured described <br /> to grow erythromycin chloramphenicol. The               Miller [50]. One Unit &#946;-galactosidase activity was <br /> resulting strain named BSIP8003. The chromosomal           defined enzyme produced 1 nmol <br /> DNA BSIP8003 prepared used transform         o-nitrophenol min 28&#176;C. Specific activity was <br /> BFS4611 strain, producing double yvoA yflG dis-        expressed Units mg protein. The experiments were <br /> rupted strain BSIP8004.                                        performed triplicates. <br />  <br /> Expression purification MAP_BS YflG_Bs              Subcellular localization analysis <br /> proteins                                                       To visualize GFP-MAP_Bs GFP-YflG_Bs fusions loca- <br /> The strains ECIP8003 ECIP8004 grown 37&#176;C           tions living cells, BSIP8001 BSIP8002 strains were <br /> in LB medium containing kanamycin (25 &#181;g/ml)               grown minimal medium 0.5% fructose sole <br /> induced 0.5 mM IPTG hours. Cell pellets          carbon source supplemented xylose different final <br /> were harvested centrifugation stored -20&#176;C.          concentrations (0.1% 1% (w/v)). Cells mid- <br /> Cells suspended 20 mM Tris-HCl (pH 8.0)            exponential growth phase collected visualized <br /> sonically disrupted. The cellular debris removed        fluorescence microscopy. Microscope Axiovert 135 TV <br /> centrifugation 12,000 g 40 min. The supernatants        (ZEISS, Germany) used experiment fluo- <br /> were applied DEAE-sephacel column chromatogra-            rescence filter sets used visualize GFP obtained <br /> phy (Amersham). The protein (MAP_Bs YflG_Bs)            Chroma Technology Corp (USA). Cells visual- <br /> eluted 0.1 M NaCl buffer. Protein con-        ized 1% agarose (w/v) slides [51] using Axiovision 4 <br /> centration determined Bradford&apos;s method [48]        (Allied High Tech Products, Inc.) exposure <br /> using protein assay kit (Bio-Rad Laboratories). The purity     time 3&#8211;10 s. <br /> of isolated proteins determined SDS-PAGE [40]. <br /> Ion mass spectrometry used measure molecular        Genetic rescue experiment <br /> mass purified protein checking exist-   The growth EM9, ECHP1005 (containing B. subtilis map <br /> ence methionine proteins.          gene), ECHP1006 (containing B. subtilis yflG gene) and <br />                                                                ECHP1007 (containing pBAD vector alone), was <br /> Enzyme assays                                                  compared M63 minimal medium plates (37&#176;C, over- <br /> The methionine aminopeptidase assay performed           night) presence absence IPTG (1 mM) or <br /> described Arie Ben-Bassat et al. [6] small modifi-     arabinose (0.02 %, 0.1 % 0.5 %). <br /> cations. 10 &#181;l protein solution added 90 &#181;l the <br /> substrate solution (4 mM peptide 0.1 M K2HPO4               RNA isolation manipulation <br /> (pH7.5), 0.2 mM CoCl2), incubated 37&#176;C         Total cellular RNA extracted B. subtilis 168 cells <br /> 10 min. The reaction stopped incubation 100&#176;C        growing minimal medium OD600 0.5 using <br /> for 2 min. After addition 900 &#181;l colour              High Pure RNA Isolation Kit Roche. The RNA con- <br /> development mixture (containing 0.2 mg L-amino acid         centration determined light absorption 260 nm <br /> oxidase, 0.03 mg horseradish peroxidase, 0.2 mg      280 nm. 2 &#181;g RNA loaded 1.2% agarose <br />  <br />  <br />                                                                                                                  Page 12 15 <br />                                                                                             (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                     http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br /> gel check RNA purity integrity. RT-PCR experi-      enzyme activity analysis, helped draft manu- <br /> ments performed using RT-PCR System (Promega)             script. AD designed supervised study. <br /> as specified manufacturer. <br />                                                                Acknowledgements <br /> Primer extension analysis carried using reverse        The initial stage work supported HKU-Pasteur Research <br /> transcriptase AMV (Roche) described [52]. Two oligo-        Centre (SAR Hong Kong, China) like thank HF Kung and <br /> nucleotides (+33 +62 +97 +126 relative trans-     KY Yuen early moment. CH You supported <br />                                                                EGIDE (France). We like thank JD Huang kind gift of <br /> lation start point yflG) used identification <br />                                                                pBAD-&#945;&#946;&#947; vector, Chiron Culture Collection Center conditional map <br /> of yflG promoter. The primers used gen-      gene deletion E. coli mutant, EM9, U Mechold kind suggestions both <br /> eration sequence ladders. Reaction products            experiment manuscript drafting, YL Yap contribution sil- <br /> separated 7% denaturing polyacrylamide gel contain-         ico analyses discussion. <br /> ing 8 M urea. DNA sequences determined using <br /> Sanger&apos;s dideoxy chain-termination method                 References <br /> &quot;Thermo Sequenase radiolabeled terminator cycle                1.    Kozak M: Comparison initiation protein synthesis pro- <br />                                                                      caryotes, eucaryotes, organelles. Microbiol Rev 1983, <br /> sequencing kit&quot; Amersham Pharmacia Biotech.                     47:1-45. <br />                                                                2.    Link AJ, Robison K, Church GM: Comparing predicted and <br /> DNA sequence silico analysis                                  observed properties proteins encoded genome of <br />                                                                      <span id='am-473' about='obo:NCBITaxon_83333' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-474' property="rdfs:label" content="Escherichia coli K-12" datatype="xsd:string"></span><span id='am-475' property="oboInOwl:hasExactSynonym" content="Escherichia coli K12" datatype="xsd:string"></span><span id='am-476' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-477' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Escherichia coli K-12</span>. Electrophoresis 1997, 18:1259-1313. <br /> DNA sequences analyzed using DNA Strider soft-        3.    Bradshaw RA, Brickey WW, Walker KW: N-terminal processing: <br /> ware [53]. The program BLAST [54] used search             methionine aminopeptidase N alpha-acetyl trans- <br /> homologous sequences database. The program                    ferase families. Trends Biochem Sci 1998, 23:263-267. <br />                                                                4.    Huang S, Elliott RC, Liu PS, Koduri RK, Weickmann JL, Lee JH, Blair <br /> CLUSTALW used multiple alignment [55]. B. sub-               LC, Ghosh-Dastidar P, Bradshaw RA, Bryan KM: Specificity of <br /> tilis sequences analyzed using SubtiList database           cotranslational amino-terminal processing proteins in <br />                                                                      yeast. Biochemistry 1987, 26:8242-8246. <br /> [56,57]. Accepted consensus sequences extracted           5.    Moerschell RP, Hosokawa Y, Tsunasawa S, Sherman F: The specifi- <br /> from PROSITE [58,59].                                                cities yeast methionine aminopeptidase acetylation of <br />                                                                      amino-terminal methionine vivo. Processing altered <br />                                                                      <span id='am-463' about='obo:NCBITaxon_238707' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_8075 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_1489917'><span id='am-464' property="rdfs:label" content="Iso" datatype="xsd:string"></span><span id='am-465'  ></span><span id='am-466' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-467' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>iso</span>-1-cytochromes         c    created           oligonucleotide <br /> Evolutionary pattern analyses                                        transformation. J Biol Chem 1990, 265:19638-19643. <br /> First, MAP orthologs (orthologs defined using         6.    Ben-Bassat A, Bauer K, Chang SY, Myambo K, Boosman A, Chang S: <br />                                                                      Processing initiation methionine proteins: prop- <br /> BBH (Bi-directional Best Hit) strategy, i.e. gene            erties Escherichia coli methionine aminopeptidase <br /> genome A similar hit gene b genome B               gene structure. J Bacteriol 1987, 169:751-757. <br /> and vice versa, gene b regarded pair         7.    Tsunasawa S, Stewart JW, Sherman F: Amino-terminal processing <br />                                                                      mutant forms yeast iso-1-cytochrome c. The specifici- <br /> orthologs [60]) retrieved pair bacte-           ties methionine aminopeptidase acetyltransferase. J <br /> rial genomes. Secondly, BBHs, protein sequence              Biol Chem 1985, 260:5382-5391. <br /> similarity &gt;50% protein length difference &lt;1.3,     8.    Miller CG, Strauch KL, Kukral AM, Miller JL, Wingfield PT, Mazzei GJ, <br />                                                                      Werlen RC, Graber P, Movva NR: N-terminal methionine-spe- <br /> gene map B. subtilis collected. Thirdly,                 cific peptidase Salmonella typhimurium. Proc Natl Acad Sci U <br /> orthologs of BBHs collected              S A 1987, 84:2718-2722. <br />                                                                9.    Hirel PH, Schmitter MJ, Dessen P, Fayat G, Blanquet S: Extent N- <br /> genomes (a total 206 bacterial genomes EMBL              terminal methionine excision Escherichia coli proteins <br /> July 1st, 2005), recursive identification            governed side-chain length penultimate <br /> members cluster longer extended. Last,             amino acid. Proc Natl Acad Sci U S A 1989, 86:8247-8251. <br />                                                                10.   Walker KW, Bradshaw RA: Yeast methionine aminopeptidase I <br /> protein divergence (sequence similarity divided                   utilize Zn2+ Co2+ cofactor: case mis- <br /> sequence length difference) MAP B. subtilis            taken identity? Protein Sci 1998, 7:2684-2687. <br /> orthologs Firmicutes compared bac-        11.   D&apos;Souza V M, Holz RC: The methionyl aminopeptidase from <br />                                                                      Escherichia coli function iron(II) enzyme. Biochem- <br /> terial evolutionary distance (measured 16S rRNA                   istry 1999, 38:11079-11085. <br /> evolution). Same analyses carried ClpP, Def,       12.   Wang J, Sheppard GS, Lou P, Kawai M, Park C, Egan DA, Schneider <br />                                                                      A, Bouska J, Lesniewski R, Henkin J: Physiologically relevant <br /> RpsD YkrB.                                                       metal cofactor methionine aminopeptidase-2 is <br />                                                                      manganese. Biochemistry 2003, 42:5035-5042. <br /> Authors&apos; contributions                                         13.   Bazan JF, Weaver LH, Roderick SL, Huber R, Matthews BW: <br />                                                                      Sequence structure comparison suggest methio- <br /> CHY carried MAPs enzyme activity analysis, subcellu-             aminopeptidase, prolidase, aminopeptidase P, crea- <br /> lar observation, expression level comparisons, drafted           tinase share common fold. Proc Natl Acad Sci U S A 1994, <br />                                                                      91:2473-2477. <br /> the manuscript. HYL did genetic rescue experiment E.        14.   Arfin SM, Kendall RL, Hall L, Weaver LH, Stewart AE, Matthews BW, <br /> coli primer extension analysis. AS designed initial          Bradshaw RA: Eukaryotic methionyl aminopeptidases: two <br /> experiments, completed genetic rescue experiment              classes cobalt-dependent enzymes. Proc Natl Acad Sci U S A <br />                                                                      1995, 92:7714-7718. <br /> B. subtilis, constructed lacZ fusion mutants B. subtilis,   15.   Li JY, Cui YM, Chen LL, Gu M, Li J, Nan FJ, Ye QZ: Mutations the <br /> and wrote manuscript. GF did evolutionary            S1 sites methionine aminopeptidases Escherichia <br /> pattern analyses. YPW helped analyse experimental             coli <span id='am-284' about='obo:NCBITaxon_9606' typeof='owl:Thing obo:NCBITaxon_9605 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-285'  ></span><span id='am-286' property="oboInOwl:hasExactSynonym" content="human" datatype="xsd:string"></span><span id='am-287' property="oboInOwl:hasExactSynonym" content="man" datatype="xsd:string"></span><span id='am-288' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-289' property="rdfs:label" content="Homo sapiens" datatype="xsd:string"></span><span id='am-290' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Homo sapiens</span> reveal residues critical sub- <br />                                                                      strate specificity. J Biol Chem 2004, 279:21128-21134. <br /> data draft manuscript. AMG supervised MAPs <br />  <br />  <br />                                                                                                                          Page 13 15 <br />                                                                                                     (page number citation purposes) <br />  BMC Microbiology 2005, 5:57                                                                         http://www.biomedcentral.com/1471-2180/5/57 <br />  <br />  <br />  <br /> 16.   Chang SY, McGary EC, Chang S: Methionine aminopeptidase                      27.   Liu S, Widom J, Kemp CW, Crews CM, Clardy J: Structure of <br />       gene Escherichia coli essential cell growth. J Bacteriol                 human methionine aminopeptidase-2 complexed with <br />       1989, 171:4071-4072.                                                               fumagillin. Science 1998, 282:1324-1327. <br /> 17.   Miller CG, Kukral AM, Miller JL, Movva NR: pepM essential              28.   Griffith EC, Su Z, Turk BE, Chen S, Chang YH, Wu Z, Biemann K, Liu <br />       gene Salmonella typhimurium.                         J Bacteriol 1989,          JO: Methionine aminopeptidase (type 2) common <br />       171:5215-5217.                                                                     target angiogenesis inhibitors AGM-1470 ovalicin. <br /> 18.   Li X, Chang YH: Amino-terminal protein processing Saccha-                       Chem Biol 1997, 4:461-471. <br />       romyces cerevisiae essential function requires                29.   Zuo S, Guo Q, Ling C, Chang YH: Evidence zinc fingers <br />       distinct methionine aminopeptidases. Proc Natl Acad Sci U S A                      methionine aminopeptidase <span id='am-264' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-265' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-266' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-267' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-268' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-269'  ></span><span id='am-270' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span>Saccharomyces</span> cer- <br />       1995, 92:12357-12361.                                                              evisiae important normal growth. Mol Gen Genet 1995, <br /> 19.   Lowther WT, Matthews BW: Structure function                             246:247-253. <br />       methionine aminopeptidases.                 Biochim Biophys Acta 2000,       30.   McKenzie T, Hoshino T, Tanaka T, Sueoka N: The nucleotide <br />       1477:157-167.                                                                      sequence pUB110: salient features relation to <br /> 20.   Atanassova A, Sugita M, <span id='am-453' about='obo:NCBITaxon_940433' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_6073 obo:NCBITaxon_6074 obo:NCBITaxon_33208 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6097 obo:NCBITaxon_37516 obo:NCBITaxon_940447'><span id='am-454' property="rdfs:label" content="Sugiura" datatype="xsd:string"></span><span id='am-455' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-456' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-457'  ></span>Sugiura</span> M, Pajpanova T, Ivanov I: Molecular                replication regulation. Plasmid 1986, 15:93-103. <br />       cloning, expression characterization distinctive                31.   Lara-Gonzalez S, Dixon HB, <span id='am-442' about='obo:NCBITaxon_1112527' typeof='owl:Thing obo:NCBITaxon_243881 obo:NCBITaxon_243882 obo:NCBITaxon_94015 obo:NCBITaxon_33208 obo:NCBITaxon_94017 obo:NCBITaxon_33213 obo:NCBITaxon_94020 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_156536 obo:NCBITaxon_6072'><span id='am-443' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-444' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-445' property="rdfs:label" content="Mendoza" datatype="xsd:string"></span><span id='am-446'  ></span>Mendoza</span>-Hernandez G, Altamirano MM, <br />       genes encoding methionine aminopeptidases cyanobacte-                           Calcagno ML: On role N-terminal group allos- <br />       rium Synechocystis sp. strain PCC6803. Arch Microbiol 2003,                        teric function glucosamine-6-phosphate deaminase from <br />       180:185-193.                                                                       Escherichia coli. J Mol Biol 2000, 301:219-227. <br /> 21.   Kunst F, Ogasawara N, Moszer I, Albertini AM, Alloni G, Azevedo V,           32.   Suh JW, Boylan SA, Oh SH, Price CW: Genetic transcrip- <br />       Bertero MG, Bessieres P, Bolotin A, Borchert S, Borriss R, Boursier                tional organization Bacillus subtilis spc-alpha region. <br />       L, Brans A, Braun M, Brignell SC, Bron S, Brouillet S, Bruschi CV,                 Gene 1996, 169:17-23. <br />       Caldwell B, Capuano V, Carter NM, Choi SK, Codani JJ, Connerton              33.   Giglione C, Serero A, Pierre M, Boisson B, Meinnel T: Identification <br />       IF, Cummings NJ, Daniel RA, Denizot F, Devine KM, Dusterhoft A,                    eukaryotic peptide deformylases reveals universality N- <br />       Ehrlich SD, Emmerson PT, Entian KD, Errington J, Fabret C, Ferrari E,              terminal protein processing mechanisms. Embo J 2000, <br />       Foulger D, Fritz C, Fujita M, Fujita Y, Fuma S, Galizzi A, Galleron N,             19:5916-5929. <br />       Ghim SY, Glaser P, Goffeau A, Golightly EJ, Grandi G, Guiseppi G,            34.   Palmiter RD, Gagnon J, Walsh KA: Ovalbumin: secreted pro- <br />       Guy BJ, Haga K, Haiech J, Harwood CR, Henaut A, Hilbert H, Holsap-                 tein transient hydrophobic leader sequence. Proc <br />       pel S, Hosono S, Hullo MF, <span id='am-400' about='obo:NCBITaxon_131275' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4710 obo:NCBITaxon_4734 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_417010 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_169700 obo:NCBITaxon_40551 obo:NCBITaxon_3193'><span id='am-401' property="rdfs:label" content="Itaya" datatype="xsd:string"></span><span id='am-402'  ></span><span id='am-403' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-404' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Itaya</span> M, Jones L, Joris B, Karamata D, Kasa-            Natl Acad Sci U S A 1978, 75:94-98. <br />       <span id='am-11' about='obo:NCBITaxon_385297' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_186628 obo:NCBITaxon_1 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7995 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489793 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_385271 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-12'  ></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Hara" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>hara</span> Y, KlaerrBlanchard M, Klein C, Kobayashi Y, Koetter P,                  35.   Jackson R, Hunter T: Role methionine initiation hae- <br />       Koningstein G, Krogh S, Kumano M, Kurita K, Lapidus A, Lardinois S,                moglobin synthesis. Nature 1970, 227:672-676. <br />       Lauber J, Lazarevic V, Lee SM, Levine A, Liu H, Masuda S, Mauel C,           36.   Haas M, Beyer D, Gahlmann R, Freiberg C: YkrB main pep- <br />       Medigue C, <span id='am-395' about='obo:NCBITaxon_569144' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43755 obo:NCBITaxon_27474 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_43917 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_43742 obo:NCBITaxon_50557 obo:NCBITaxon_179436'><span id='am-396' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-397' property="rdfs:label" content="Medina" datatype="xsd:string"></span><span id='am-398' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-399'  ></span>Medina</span> N, Mellado RP, Mizuno M, Moestl D, Nakai S,                      tide deformylase Bacillus subtilis, <span id='am-214' about='obo:NCBITaxon_1730' typeof='owl:Thing obo:NCBITaxon_186806 obo:NCBITaxon_186802 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_186801 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-215' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-216'  ></span><span id='am-217' property="rdfs:label" content="Eubacterium" datatype="xsd:string"></span><span id='am-218' property="oboInOwl:hasDbXref" content="PMID:8863439" datatype="xsd:string"></span><span id='am-219' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>eubacterium</span> contain- <br />       Noback M, Noone D, OReilly M, Ogawa K, Ogiwara A, Oudega B,                        ing functional peptide deformylases. Microbiology 2001, <br />       Park SH, Parro V, Pohl TM, Portetelle D, Porwollik S, Prescott AM,                 147:1783-1791. <br />       Presecan E, Pujic P, Purnelle B, Rapoport G, Rey M, Reynolds S, Rieger       37.   Goh CS, Cohen FE: Co-evolutionary analysis reveals insights <br />       M, Rivolta C, Rocha E, Roche B, Rose M, Sadaie Y, Sato T, Scanlan E,               protein-protein interactions. J Mol Biol 2002, 324:177-192. <br />       Schleich S, Schroeter R, Scoffone F, Sekiguchi J, Sekowska A, Seror SJ,      38.   Ramani AK, Marcotte EM: Exploiting co-evolution inter- <br />       Serror P, Shin BS, Soldo B, Sorokin A, Tacconi E, Takagi T, Takahashi              acting proteins discover interaction specificity. J Mol Biol <br />       H, Takemaru K, Takeuchi M, Tamakoshi A, Tanaka T, Terpstra P,                      2003, 327:273-284. <br />       Tognoni A, Tosato V, Uchiyama S, Vandenbol M, Vannier F, Vassarotti          39.   Bertani G: Studies lysogenesis. I. The mode phage liber- <br />       A, Viari A, Wambutt R, Wedler E, Wedler H, Weitzenegger T, Win-                    ation lysogenic Escherichia coli. J Bacteriol 1951, 62:293-300. <br />       ters P, Wipat A, Yamamoto H, Yamane K, Yasumoto K, Yata K, Yosh-             40.   Sambrook J FEGMT: Molecular Cloning: Laboratory Manual. <br />       <span id='am-377' about='obo:NCBITaxon_426865' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158332 obo:NCBITaxon_158416 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_158394 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-378'  ></span><span id='am-379' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-380' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-381' property="rdfs:label" content="Ida" datatype="xsd:string"></span>ida</span> K, Yoshikawa HF, Zumstein E, Yoshikawa H, Danchin A: The                       NY, Cold Spring Harbor; 1989. <br />       complete genome sequence Gram-positive bacterium                      41.   Kunst F, Rapoport G: Salt stress environmental signal <br />       Bacillus subtilis. Nature 1997, 390:249-256.                                       affecting degradative enzyme synthesis Bacillus subtilis. J <br /> 22.   Kobayashi K, Ehrlich SD, Albertini A, Amati G, Andersen KK, Arnaud                 Bacteriol 1995, 177:2403-2407. <br />       M, Asai K, Ashikaga S, Aymerich S, Bessieres P, Boland F, Brignell SC,       42.   Lewis PJ, Marston AL: GFP <span id='am-405' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-406' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-407' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-408' property="rdfs:label" content="vectors" datatype="xsd:string"></span>vectors</span> controlled expression <br />       Bron S, Bunai K, Chapuis J, Christiansen LC, Danchin A, Debarbouille               dual labelling protein fusions Bacillus subtilis. Gene <br />       M, Dervyn E, Deuerling E, Devine K, Devine SK, Dreesen O, Err-                     1999, 227:101-110. <br />       ington J, Fillinger S, Foster SJ, Fujita Y, Galizzi A, Gardan R, Eschevins   43.   Muyrers JP, Zhang Y, Testa G, Stewart AF: Rapid modification of <br />       C, Fukushima T, Haga K, Harwood CR, Hecker M, Hosoya D, Hullo                      bacterial artificial chromosomes ET-recombination. <br />       MF, Kakeshita H, Karamata D, Kasahara Y, Kawamura F, Koga K,                       Nucleic Acids Res 1999, 27:1555-1557. <br />       Koski P, Kuwana R, Imamura D, Ishimaru M, Ishikawa S, Ishio I, Le            44.   Stragier P, Bonamy C, Karmazyn-Campelli C: Processing a <br />       Coq D, Masson A, Mauel C, Meima R, Mellado RP, Moir A, Moriya S,                   sporulation sigma factor Bacillus subtilis: morpholog- <br />       Nagakawa E, Nanamiya H, Nakai S, Nygaard P, Ogura M, Ohanan T,                     ical structure control gene expression. Cell 1988, <br />       O&apos;Reilly M, O&apos;Rourke M, Pragai Z, Pooley HM, Rapoport G, Rawlins                   52:697-704. <br />       JP, Rivas LA, Rivolta C, Sadaie A, Sadaie Y, Sarvas M, Sato T, Saxild        45.   Perego M: Integrational vectors genetic manipulation in <br />       HH, Scanlan E, Schumann W, Seegers JF, Sekiguchi J, Sekowska A,                    Bacillus subtilis. In Bacillus subtilis Gram-positive bacteria <br />       Seror SJ, Simon M, Stragier P, Studer R, Takamatsu H, Tanaka T,                    Edited by: Sonenshein AL HJALR. Washington, D.C, Amercian Society <br />       Takeuchi M, Thomaides HB, Vagner V, van Dijl JM, Watabe K, Wipat                   Microbiology; 1993:615-624. <br />       A, Yamamoto H, Yamamoto M, Yamamoto Y, Yamane K, Yata K,                     46.   Bacillus Subtilis Genome Database                          [http://bacil <br />       Yoshida K, Yoshikawa H, Zuber U, Ogasawara N: Essential Bacillus                   lus.genome.ad.jp] <br />       subtilis genes. Proc Natl Acad Sci U S A 2003, 100:4678-4683.                47.   Micado Release 6.1 [http://locus.jouy.inra.fr/cgi-bin/genmic/mad <br /> 23.   Fang G, Rocha E, Danchin A: How essential non-essential                        base/progs/madbase.operl] <br />       genes? Mol Biol Evol, press .                                             48.   Bradford MM: A rapid sensitive method quantita- <br /> 24.   Brenner SE: Errors genome annotation. Trends Genet 1999,                        tion microgram quantities protein utilizing princi- <br />       15:132-133.                                              <br /> </body></html>