<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />                         RAD53 Regulates DBF4 Independently of Checkpoint <br />                                Function in <span id='am-1' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-12'  ></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> <br />  <br />  <br />                                                                ABSTRACT <br />                 The Cdc7p and Dbf4p proteins form an active kinase complex in <span id='am-18' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-23'  ></span><span id='am-24' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae that is essential <br />              for the initiation of DNA replication. A genetic screen for mutations that are lethal in combination with <br />              cdc7-1 led to the isolation of seven lsd (lethal with seven defect) complementation groups. The lsd7 <br />              complementation group contained two temperature-sensitive dbf4 alleles. The lsd1 complementation group <br />              contained a new allele of RAD53, which was designated rad53-31. RAD53 encodes an essential protein <br />              kinase that is required for the activation of DNA damage and DNA replication checkpoint pathways, and <br />              that is implicated as a positive regulator of S phase. Unlike other RAD53 alleles, we demonstrate that the <br />              rad53-31 allele retains an intact checkpoint function. Thus, the checkpoint function and the DNA replica- <br />              tion function of RAD53 can be functionally separated. The activation of DNA replication through RAD53 <br />              most likely occurs through DBF4. Two-<span id='am-43' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-44'  ></span><span id='am-45' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>hybrid</span> analysis indicates that the Rad53p protein binds to Dbf4p. <br />              Furthermore, the steady-state level of DBF4 message and Dbf4p protein is reduced in several rad53 mutant <br />              strains, indicating that RAD53 positively regulates DBF4. These results suggest that two different functions <br />              of the cell cycle, initiation of DNA replication and the checkpoint function, can be coordinately regulated <br />              through the common intermediate RAD53. <br />  <br />  <br />  <br /> T     HE initiation of DNA replication is a strictly regu- <br />       lated process that is coupled tightly to cell cycle <br /> progression and results in the accurate duplication of <br />                                                                            Isolation of mutants defective in cell cycle progression <br />                                                                          led to the identification of CDC7, a gene encoding a <br />                                                                          nuclear serine/threonine kinase that is essential for the <br /> the genetic material. Orderly cell cycle events ensure                   initiation of S phase in Saccharomyces cerevisiae (Hart- <br /> that the initiation of DNA replication occurs once, and                  well 1973; Bahman et al. 1988; Hollingsworth and <br /> only once, per cell cycle. At the same time, eukaryotic                  Sclafani 1990; Yoon and Campbell 1991). The kinase <br /> cells also have evolved mechanisms for reducing or elim-                 activity of Cdc7p is required for the initiation of repli- <br /> inating the result of DNA damage or incomplete replica-                  cation (Buck et al. 1991; Hollingsworth et al. 1992). <br /> tion before completing other cell cycle events, such as                  Also, Cdc7p recently has been shown to be required <br /> mitosis. These surveillance mechanisms, termed check-                    throughout S phase for origin firing, but not for elon- <br /> points, ensure that the integrity of the genome is intact                gation (Bousset and Diffley 1998; Donaldson et al. <br /> before proceeding through crucial cellular events (for                   1998). Although the level of CDC7 transcripts appears to <br /> reviews see Hartwell and Weinert 1989; Stewart                           be constitutive throughout the cell cycle, Cdc7p kinase <br /> and Enoch 1996; Weinert 1998). Three classes of DNA                      activity fluctuates in a cell cycle-dependent manner, <br /> checkpoints have been described in yeast. One pathway                    peaking at the time S phase begins (Sclafani et al. 1988; <br /> blocks exit from S phase or entry into mitosis if DNA                    Jackson et al. 1993). <span id='am-33' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-34'  ></span><span id='am-35' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> is brought about by association <br /> replication is incomplete (Weinert 1992; Allen et al.                    of Cdc7p (Jackson et al. 1993) with the product en- <br /> 1994; Weinert et al. 1994). The other two classes pre-                   coded by the DBF4 gene, whose expression varies in a <br /> vent exit from either G1 or G2 in cells containing dam-                  cell cycle-dependent manner (Chapman and Johnston <br /> aged DNA (Weinert and Hartwell 1988, 1990; Siede                         1989). Through genetic studies, CDC7 and DBF4 have <br /> et al. 1993). Failure of the cellular restraints imposed                 been shown to act at the same point (Kitada et al. <br /> normally by a checkpoint can result in genomic instabil-                 1992). Likewise, Cdc7p and Dbf4p have also been shown <br /> ity, increased mutation rates, and ultimately death, if                  to interact physically, and both are required to obtain <br /> cells continue to divide unchecked.                                      active kinase activity (Jackson et al. 1993). The Cdc7p <br />                                                                          kinase complex is brought to origins of replication via <br />                                                                          the interaction of Dbf4p with the origin replication com- <br />                                                                          plex (ORC), where it is thought to phosphorylate mem- <br />                                                                          bers of the prereplication complex (Dowell et al. 1994; <br />    Corresponding author: Robert A. Sclafani, Department of Biochemis-    Hardy et al. 1997). One phosphorylation target is the <br /> try and Molecular Genetics, 4200 E. 9th Ave., Box B121, University <br /> of Colorado Health Sciences Center, Denver, CO 80262.                    product of the MCM2 gene (Lei et al. 1997). Phosphory- <br /> E-mail: robert.sclafani@uchsc.edu                                        lation of the prereplication complex converts it into an <br /> Genetics 151: 965&#8211;977 (March 1999) <br />  966                                                P. R. Dohrmann et al. <br />  <br /> active replication complex, although exactly how this is        cleus, where it presumably functions with other DNA <br /> achieved is still unknown.                                      replication and repair enzymes (Zheng et al. 1993). <br />    Initiation of the checkpoint program requires the ac-        Third, and probably most compelling, a RAD53 disrup- <br /> tivity of many different proteins. The components of            tion gives rise to spores that form inviable microcolonies <br /> the DNA damage checkpoint machinery generally fall              enriched for cells that demonstrate a large budded cell <br /> into three classes: sensors, transducers, and targets (Wein-    morphology with short mitotic spindles (Zheng et al. <br /> ert 1998). RAD9, RAD17, RAD24, and MEC3 are in-                 1993). This phenotype is consistent with other cdc mu- <br /> cluded in the sensor <span id='am-30' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> and are required for response        tants that contain mutations required for DNA synthesis. <br /> to DNA damage (Lydall and Weinert 1995; Navas et al.            Because other checkpoint genes can be deleted, it is <br /> 1995; Paulovich et al. 1997; de la Torre-Ruiz et al.            not entirely clear why RAD53 is essential. However, it <br /> 1998). The POL2, RFD5, and DPB11 classes are required           has been postulated that the essential function of RAD53 <br /> for the response to the arrest of DNA replication (Araki        could be related to some aspect of DNA replication or <br /> et al. 1995; Navas et al. 1995). They are thought to            the timing of mitosis (Zheng et al. 1993; Weinert 1998). <br /> recognize DNA damage or stalled replication forks, and            The two different aspects of the cell cycle, initiation <br /> they initiate a protein kinase signal transduction cascade      of DNA replication and checkpoint function, could be <br /> that activates Mec1p and Rad53p (Sanchez et al. 1996;           coordinately regulated through DNA metabolism events. <br /> Sun et al. 1996). The protein kinases Rad53p and Mec1p          As a starting point for our studies, we were interested <br /> act as transducers and transmit signals to critical down-       in identifying genes that act in the same pathway or <br /> stream targets (Sanchez et al. 1996; Sun et al. 1996),          affect a similar function as CDC7. Here we report the <br /> leading to the transcription of genes involved in DNA           isolation of a new allele of RAD53 in a genetic screen <br /> replication and repair (Aboussekhra et al. 1996; Kiser          for mutants that are lethal in combination with cdc7-1. <br /> and Weinert 1996; Navas et al. 1996). Another conse-            This allele of RAD53 retains a fully functional check- <br /> quence is the activation of targets that slow or halt the       point. In addition, we show that Rad53p interacts with <br /> cell cycle and allow the completion of DNA repair or            Dbf4p and controls the level of expression of DBF4. <br /> replication (Sidorova and Breeden 1997). So far, only           Thus, we demonstrate that the checkpoint functions <br /> one potential target of this cascade has been identified.       and the DNA replication functions can be functionally <br /> Swi6p is modified in a RAD53-dependent manner in                separated in RAD53. These results suggest strongly that <br /> response to the G1 (DNA damage) checkpoint, which               coordination of the two separate functions could occur <br /> results in the delay of entry into S phase by inhibiting        through a common intermediate, namely, RAD53. <br /> CLN transcription (Sidorova and Breeden 1997). <br />    RAD53 (5MEC2, SPK1, SAD1) encodes a dual-speci- <br />                                                                                MATERIALS AND METHODS <br /> ficity protein kinase that is required for all three DNA <br /> damage checkpoints at G1, S phase, and G2 (Stern et                Yeast strains, media, and <span id='am-48' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-49' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span>: Yeast strains were grown <br /> al. 1991; Zheng et al. 1993; Allen et al. 1994; Weinert         in yeast extract/peptone/dextrose (YEPD) with 2% glucose <br /> et al. 1994; Fay et al. 1997). It is thought to be part         or in synthetic defined (SD) minimal media supplemented <br />                                                                 with appropriate amino acids and 2% glucose. All yeast strains <br /> of the transducer class (Weinert 1998). Mutations in            used in this study are listed in Table 1. All strains are congenic <br /> RAD53 have been isolated in screens for checkpoint              with A364a, except as marked (Hartwell 1967). Strains PJ69- <br /> mutations (Allen et al. 1994; Weinert et al. 1994), and         4a ( James et al. 1996), Y187, and Y190 (Clontech, Palo Alto, <br /> null mutants of RAD53 are inviable (Zheng et al. 1993;          CA) are from a different genetic background and are used <br /> Allen et al. 1994; Kim and Weinert 1997). RAD53 mu-             for two-hybrid analysis. Standard genetic methods were used <br />                                                                 for strain construction and tetrad analysis (Sherman et al. <br /> tants continue to divide unchecked in response to DNA-          1986), and transformation of yeast strains was performed by <br /> damaging agents and die as a result (Zheng et al. 1993;         the lithium acetate method (Ito et al. 1983). Disruption of <br /> Allen et al. 1994; Weinert et al. 1994). Rad53p itself          the PEP4 and RAD53 genes was done by the one-step gene <br /> is modified in response to DNA damage in a MEC1- and            replacement method (Rothstein 1983). For the pep4D::URA3 <br />                                                                 disruption, plasmid pBR322-pep4D::URA3 was digested with EcoRI <br /> TEL1-dependent manner, placing RAD53 downstream <br />                                                                 and XhoI before transformation. The presence of pep4D::URA3 <br /> of MEC1 and TEL1 (Sanchez et al. 1996; Sun et al. 1996).        was analyzed using the APE overlay test for CpY activity ( Jones <br />    Several lines of circumstantial evidence suggest that        1991). For the rad53D::hisG-URA3-hisG disruption, pPD84 was <br /> in addition to its checkpoint function, RAD53 could also        digested with BamHI before transformation. The presence of <br /> be involved in the initiation of DNA replication. First,        the rad53D::hisG-URA3-hisG disruption was analyzed by South- <br />                                                                 ern blot analysis (Ausubel et al. 1987). Strains bearing the <br /> RAD53 has been shown to be transcriptionally coregu-            rad53D::hisG-URA3-hisG disruption were grown on 5-fluoro- <br /> lated with DNA synthetic genes (Zheng et al. 1993).             orotic medium to select for loss of the URA3 marker, resulting <br /> This regulation is most likely conferred by a pair of           in a strain carrying rad53D::hisG allele. <br /> consensus MluI cell cycle box (MCB) regulatory ele-                All plasmids used in this study are listed in Table 2. The <br /> ments within its promoter. These elements are found             CDC7-ADE3 sectoring plasmids were constructed in several <br />                                                                 steps. To construct pPD4 (pRS316-CDC7-ADE3), a 2.7-kb Sal I <br /> in the promoters of a number of genes that are tran-            to EcoRI CDC7 fragment from pRH102 (Hollingsworth et al. <br /> scribed in late G1 and include DNA synthetic enzymes.           1992) was cloned into the Sal I to EcoRI sites of pRS316, gener- <br /> Second, immunolocalization places Rad53p in the nu-             ating pPD1. A 3.5-kb Bcl I ADE3 fragment was then removed <br />                                                     RAD53 Regulation of DBF4                                                    967 <br />  <br />                                                             TABLE 1 <br />                                                 List of strains used in this study <br />  <br /> Strains                                                             Genotype (source) <br /> 199                      MATa cdc8-1 can1 gal1 leu2 trp1 ura3 <br /> 209                      MATa cdc7-3 can1 ade1 ade2 gal1 his7 ura3 leu2 <br /> 299                      MATa can1 cyh2 gal1 his3 leu2 trp1 ura3 <br /> 576                      MATa cdc7-4 ade2 gal1 ura3 <br /> 708                      MATa cdc7-7 gal1 his3 leu2 trp1 ura3 <br /> P119                     MATa bob1-1(5mcm5) can1 gal1 his7 leu2 lys2 trp1 ura3 <br /> P257                     MATa bob1-1(5mcm5) ade1 can1 cyh2 gal1 his3 trp1 ura3 <br /> PDY024                   MATa cdc7-1 ade2 ade3 bar1 cyh2 gal1 leu2 trp1 ura3 1 pRS316-CDC7-ADE3 <br /> PDY029                   MATa dbf4-1 ade2 ade3 cyh2 gal1 his7 leu2 lys2 trp1 ura3 <br /> PDY093                   MATa cdc7-1 ade2 ade3 cyh2 gal1 his3 leu2 ura3 <br /> PDY095                   MATa dbf4-1 ade2 ade3 cyh2 gal1 lys2 trp1 <br /> PDY108                   PDY024 lsd1-1 (5rad53-31) <br /> PDY109                   PDY024 lsd6-1 (5cdc7 null) <br /> PDY120                   PDY024 lsd7-1 (5dbf4) <br /> PDY124                   PDY024 lsd2-1 <br /> PDY128                   PDY024 lsd3-1 <br /> PDY129                   PDY024 lsd4-1 <br /> PDY138                   PDY024 lsd5-1 <br /> PDY177                   MATa cdc7-1 ade2 ade3 gal1 his7 leu2 ura3 <br /> PDY201                   MATa lsd1-1 cdc7-1 ade2 ade3 gal1 his3 his7 leu2 trp1 ura3 1 pRS314-CDC7-ADE3 <br /> PDY206                   MATa lsd1-1 cdc7-1 ade2 ade3 gal1 his7 leu2 ura3 1 pRS316-CDC7-ADE3 <br /> PDY207                   MATa lsd1-1 cdc7-1 ade2 ade3 gal1 his7 leu2 trp1 ura3 1 pRS316-CDC7-ADE3 <br /> PDY208                   MATa lsd2-1 cdc7-1 ade2 ade3 gal1 his7 leu2 ura3 1 pRS316-CDC7-ADE3 <br /> PDY209                   MATa lsd2-1 cdc7-1 ade2 ade3 gal1 his7 leu2 trp1 ura3 1 pRS316-CDC7-ADE3 <br /> PDY210                   MATa lsd3-1 cdc7-1 ade2 ade3 gal1 his7 leu2 ura3 1 pRS316-CDC7-ADE3 <br /> PDY211                   MATa lsd3-1 cdc7-1 ade2 ade3 gal1 his7 leu2 trp1 ura3 1 pRS316-CDC7-ADE3 <br /> PDY212                   MATa lsd4-1 cdc7-1 ade2 ade3 gal1 his7 leu2 ura3 1 pRS316-CDC7-ADE3 <br /> PDY213                   MATa lsd4-1 cdc7-1 ade2 ade3 gal1 his7 leu2 trp1 ura3 1 pRS316-CDC7-ADE3 <br /> PDY214                   MATa lsd5-1 cdc7-1 ade2 ade3 gal1 his7 leu2 ura3 1 pRS316-CDC7-ADE3 <br /> PDY215                   MATa lsd5-1 cdc7-1 ade2 ade3 gal1 his7 leu2 trp1 ura3 1 pRS316-CDC7-ADE3 <br /> PDY242                   MATa rad53-31 pep4D::URA3 cdc7-1 ade2 ade3 gal1 his3 his7 leu2 trp1 ura3 1 pRS314-CDC7-ADE3 <br /> PDY258                   MATa rad53-11 pep4D::URA3 gal1 his3 his7 ura3 <br /> PDY289                   MATa rad53-31 pep4D::URA3 ade2 ade3 bar1 gal1 his6 his7 leu2 trp1 ura3 <br /> PDY294                   MATa rad53D::hisG-URA3-hisG bob1-1 cyh2 gal1 his3 lys2 leu2 trp1 ura3 1 pGAP-RNR1 <br /> PDY305                   MATa can1 cyh2 gal1 his3 leu2 trp1 ura3 1 pRS316-7HA-DBF4 <br /> PDY306                   MATa rad53-31 cdc7-1 ade2 ade3 gal1 his3 his7 leu2 trp1 ura3 1 pRS314-CDC7-ADE3, pRS316-7HA-DBF4 <br /> PDY307                   MATa rad53-11 bar1 gal1 his6 his7 ura3 1 pRS316-7HA-DBF4 <br /> PDY308                   MATa rad53D::hisG bob1-1 can1 cyh2 gal1 leu2 lys2 trp1 1 pGAPRNR1, pRS316-7HA-DBF4 <br /> PJ69-4a                  MATa ade2 gal4D gal80D his3 lys2 LYS2::GAL1p-HIS3 GAL2p-ADE2 met2::GAL7p-lacZ ( James et al. 1996) <br /> TWY310                   MATa rad53-11 ura3 his3 his7 (T. Weinert) <br /> Y187                     MATa ade2 gal4D gal80D his3 leu2 met- trp1 ura3 URA3::GAL1UAS-GAL1tata-lacZ (Clontech) <br /> Y190                     MATa ade2 cyh2 gal4D gal80D his3 leu2 lys2 trp1 ura3 LYS2::GAL1UAS-HIS3tata-HIS3 URA3::GAL1UAS-GAL1tata- <br />                            lacZ (Clontech) <br />  <br />  <br /> from pMW29 (gift from David Stillman) and cloned into the           nomic 5322-bp ClaI RAD53 fragment into the ClaI sites of <br /> BamHI site of the polylinker of pRS316, generating pPD3.            pRS316 and pRS314, respectively. <br /> The ADE3 cassette was then removed from pPD3 and cloned                Plasmid pPD84 (pBS-rad53D::hisG-URA3-hisG) was created <br /> into pPD1 as a EcoRI and NotI fragment, generating pPD4             in two steps. First, the 5322-bp genomic RAD53 ClaI fragment <br /> (pRS316-CDC7-ADE3). The pPD7 (pRS314-CDC7-ADE3) plas-               was cloned into the ClaI site of pBS KS1 (Stratagene, La Jolla, <br /> mid was constructed by removing the 6.2-kb NotI to Sal I frag-      CA), generating pPD82. This plasmid was then digested with <br /> ment containing both the CDC7 and ADE3 genes and ligating           SphI to completion and filled in with Klenow. This removes <br /> it into the NotI and Sal I sites of the polylinker of pRS314.       the RAD53 promoter and most of the coding region of RAD53 <br />    The DBF4 genomic plasmids pPD32 (pRS315-DBF4) and                (nt 2477 to 11887). The 3.8-kb BamHI to Bgl II hisG-URA3- <br /> pGO117 (pRS425-DBF4) were constructed by ligating the ge-           hisG fragment from pNKY51 (Alani et al. 1987) was filled in <br /> nomic 5-kb Sal I to EcoRI DBF4 fragment from pDBF4.4 (Chap-         with Klenow and ligated into this site, generating pPD84 (pBS- <br /> man and Johnston 1989) into the Sal I to EcoRI sites of pRS315      rad53D::hisG-URA3-hisG). <br /> and pRS425, respectively.                                              The GAL4DB-RAD53 plasmids were constructed as follows. <br />    The RAD53 genomic plasmids pPD60 (pRS316-RAD53) and              A 1221-bp PvuII fragment of RAD53 was first cloned into the <br /> pPD83 (pRS314-RAD53) were constructed by ligating the ge-           SmaI site in frame of pY2, generating pPD93 [pGAL4DB-RAD53i <br />  968                                                      P. R. Dohrmann et al. <br />  <br />                                                                 TABLE 2 <br />                                                   List of plasmids used in this study <br />  <br />          Plasmids                                        Description                                     Source <br />          pBluescriptKS1                Cloning vector                                        Stratagene <br />          pRS314                        TRP1 yeast shuttle vector                             Sikorski and Hieter (1989) <br />          pRS315                        LEU2 yeast shuttle vector                             Sikorski and Hieter (1989) <br />          pRS316                        URA3 yeast shuttle vector                             Sikorski and Hieter (1989) <br />          pPD4                          pRS316-CDC7-ADE3 (URA3)                               This study <br />          pPD7                          pRS314-CDC7-ADE3 (TRP)                                This study <br />          pPD32                         pRS315-DBF4 (LEU2)                                    This study <br />          pGO117                        pRS425-DBF4 (LEU2)                                    This study <br />          pRS277                        pCDC7 CEN (LEU2)                                      Hollingsworth et al. (1992) <br />          pRS288                        pcdc7-1 CEN (LEU2)                                    Hollingsworth et al. (1992) <br />          pPD58                         YCp50-RAD53 genomic clone                             This study <br />          pPD60                         pRS316-RAD53 (URA3)                                   This study <br />          pPD83                         pRS314-RAD53 (TRP1)                                   This study <br />          pBR322-pep4D::URA3            pep4D::URA3 disruption plasmid                        Gift from B. Tye <br />          pNKY51                        phisG::URA3::hisG cassette                            Alani et al. (1987) <br />          pPD84                         pBS-rad53D::hisG-URA3-hisG                            This study <br />          pY2                           pGAL4DB (TRP1)                                        Sadowski et al. (1992) <br />          pPD93                         pGAL4DB-RAD53i (aa50&#8211;aa457) fusion (TRP1)             This study <br />          pPD94                         pGAL4DB-RAD53 (aa50&#8211;aa822) fusion (TRP1)              This study <br />          pGO109                        pGAL4DB-DBF4 fusion (TRP1)                            Shellman et al. (1998) <br />          pGAD-2F                       pGAL4AD (LEU2)                                        Chien et al. (1991) <br />          pGO119                        pGAL4AD-CDC7 fusion (LEU2)                            Shellman et al. (1998) <br />          pCH441                        pGAL4AD-DBF4 fusion (LEU2)                            Hardy and Pautz (1996) <br />          pGO174                        pRS316-7HA-DBF4 (URA3)                                This study <br />          pGAP-RNR1                     pRNR1 driven off GAP promoter (TRP1)                  T. Weinert <br />  <br />  <br />  <br /> (aa50&#8211;aa457)]. pPD94 [pGAL4DB-RAD53 (aa50&#8211;aa822)] was                  (EMS; Sigma, St. Louis, MO) mutagenesis of strain PDY024 <br /> constructed by ligating in the 2321-bp BamHI-to-HindIII frag-          was performed under conditions that would generate z30% <br /> ment of RAD53 from pPD82 into the BamHI and HindIII                    survival ( Jackson et al. 1993). Survival was measured at 28%. <br /> sites of pPD93, reconstructing the 39 end of RAD53. Plasmid            A total of 4 3 108 cells was mutagenized and split into 10 <br /> pGO174 (pRS316-7HA-DBF4) was constructed in several steps.             different pools. After inactivation of EMS with sodium thiosul- <br /> First, the genomic 5-kb Sal I to HindIII DBF4 fragment from            fate, cells were washed in 50 mm KH2PO4 buffer, pH 7.0, and <br /> pDBF4.4 (Chapman and Johnston 1989) was cloned into the                then were plated at a dilution of 500 colonies per plate on <br /> Sal I to HindIII sites of pRS316, generating pPD2. pPD2 was            YEPD. The plates were incubated at 238 for 7 days until color <br /> then digested with NotI and filled in with Klenow to eliminate         had fully developed. Mutagenized cells that demonstrated a <br /> the NotI site in the polylinker, generating pGO140. Two PCR            rare nonsectored colony morphology were further character- <br /> fragments were generated from this plasmid template using              ized. <br /> the following sets of primers and Pfu DNA polymerase: T3                  Determination of DNA damage-induced cell cycle delay: <br /> primer 59 ATT AAC CCT CAC TAA AGG GA 39, HA-DBF4                       For UV survival studies, cells were grown to late-logarithmic <br /> primer 59 TAC AGC GGC CGC ATG CAT AGT CAG GAA                          phase in liquid culture (108 cells/ml), counted, and plated <br /> CAT CGT ATG GGT ACA TTT TCT TCT TTC TTT TC 39,                         at a dilution of z500 cfu/plate. Cells were mutagenized by <br /> NotI DBF4 primer 59 ATC TGC GGC CGC GTT TCT CCA                        exposure to UV light with a 254-nm source at fluence rate of <br /> ACG AAA ATG 39, and 39 primer DBF4 59 ATC GTT GCA                      0.7 mW/cm2, as measured with a UVP radiometer for doses <br /> TGT GTG AGG 39. The two PCR fragments were cloned into                 of 0, 40, 60, 80, and 100 J/m2 (Ostroff and Sclafani 1995). <br /> pGO140 as a three-part ligation: pGO140 plasmid digested               The plates were wrapped in aluminum foil to prevent light <br /> with Sal I and Bcl I, PCR fragment 1 digested with Sal I and NotI,     repair and were incubated at 238 for 7 days and then counted. <br /> and PCR fragment 2 digested with NotI and Bcl I, generating            All experiments were performed in triplicate. Percent survival <br /> pGO154. Finally, a triple HA tag DNA fragment (Tyers et al.            was determined relative to unirradiated controls. <br /> 1992) with NotI ends was ligated into the NotI site of pGO154.            For hydroxyurea arrest experiments, cells were grown to <br /> The resulting plasmid was shown to contain two contiguous              midlogarithmic phase in liquid YEPD culture (107 cells/ml). <br /> direct repeats of the triple HA tag, resulting in a total of seven     A small aliquot of cells was removed from each culture as a <br /> HA tags. The 59 end of the gene was sequenced past the Bcl I           negative control before adding hydroxyurea. Hydroxyurea was <br /> site to determine whether all HA tags were in frame and                then added to a final concentration of 0.2 m, and aliquots <br /> no mutations were introduced by PCR amplification. This                were removed at timed intervals to determine cell number <br /> plasmid can complement the dbf4-1 temperature-sensitive mu-            and to score for viability as colony-forming units on YEPD <br /> tation, and Dbf4p protein can be easily detected by immu-              plates (Allen et al. 1994). Percentage survival was determined <br /> noblot analysis using the 12CA5 monoclonal antibody directed           relative to cells that were not exposed to hydroxyurea at the <br /> against the HA epitope.                                                beginning of the experiment. <br />    Synthetic lethality screen: The synthetic lethality screen was         Two-hybrid assay: Activation domain and DNA-binding do- <br /> done as described previously (Kranz and Holm 1990; Bender              main fusion plasmids were first transformed sequentially into <br /> and Pringle 1991; Hardy 1996). Ethyl methanesulfonate                  strain PJ69-4a, and selection was carried out on SD-Leu-Trp <br />                                                       RAD53 Regulation of DBF4                                             969 <br />  <br /> plates. To assay for interaction, colonies were plated onto SD-     The screen is based on a red/white adenine colony <br /> Leu-Trp-Ade plates, and prototrophic growth was analyzed            sectoring assay (Kranz and Holm 1990; Bender and <br /> ( James et al. 1996). For the quantitative b-galactosidase analy- <br /> sis, the DNA-binding constructs were transformed into strain        Pringle 1991; Hardy 1996). Nonsectored colonies, in- <br /> Y190 (Clontech), and the activation domain constructs were          dicative of mutants that may require the plasmid-borne <br /> transformed into strain Y187 (Clontech). The transformed            CDC7 gene to survive, were designated lsd for lethal <br /> haploids were mated, and the diploids were selected on SD-          with seven defect. Of the z15,000 colonies screened, <br /> Trp-Leu media. b-Galactosidase activity was measured in Miller      39 demonstrated a nonsectored red colony phenotype. <br /> units, as described previously ( Jackson et al. 1993; Shellman <br /> et al. 1998). Three individual colonies from each diploid strain    These 39 strains were backcrossed to PDY093. All the <br /> were used to perform the b-galactosidase assays.                    diploids analyzed demonstrated a sectored colony mor- <br />    Northern blot analysis: A plasmid containing a 7-hemaglut-       phology, indicating that the mutations are recessive. <br /> tin tag DBF4 gene under the control of the wild-type DBF4           Eighteen of these strains were characterized further. <br /> promoter was transformed into wild-type, rad53-31, rad53-11,        The remaining mutants did not demonstrate 2:2 segre- <br /> and rad53D::hisG strains for Northern, FACS, and immunoblot <br /> experiments (see below). The rad53D::hisG strain was kept           gation of the sectoring phenotype after meiosis, were <br /> alive by overexpression of RNR1 (Sanchez et al. 1996). North-       unable to mate, or were sporulation defective. The 18 <br /> ern (RNA) analysis was performed as described previously,           haploid segregants from the original backcross were <br /> with the following exceptions (Dohrmann et al. 1992). Total         backcrossed two additional times and sorted into com- <br /> RNA (10 mg) was subjected to electrophoresis through a 1%           plementation groups by constructing diploids from <br /> agarose-formaldehyde gel and was transferred to Hybond-N <br /> nylon membranes (Amersham, Arlington Heights, IL). Hy-              mutants of opposite mating types. The diploids were <br /> bridization probes were generated with a Random Prime DNA           tested for their sectoring ability; seven complementation <br /> labeling system (GIBCO BRL, Gaithersburg, MD) and a-32P             groups were identified this way and designated lsd1&#8211;7. <br /> (3000 Ci/mmol; New England Nuclear, Boston, MA) from                   We expected to find mutations in DBF4 on the basis of <br /> the following templates: for DBF4, a 794-bp XhoI internal frag-     the fact that cdc7-1 mutation is known to be synthetically <br /> ment of DBF4; for POL1, an EcoRI 869-bp internal fragment <br /> of POL1. To detect 18s rRNA, an 18s rRNA DNA primer (59             lethal with dbf4-1 from previous experiments (Kitada <br /> GCTTATACTTAGACATGCAT 39; gift from Judith Jaehning)                 et al. 1992). Two of the 18 mutants were temperature <br /> was labeled with [g-32P]ATP (6000 Ci/mmol) and T4 poly-             sensitive and found to contain mutations in DBF4. To <br /> nucleotide kinase. Hybridization and subsequent washes of the       confirm this conclusion, we showed that the sectoring <br /> Northern blots with the 18s rRNA DNA primer were carried at         phenotype was complemented by a DBF4 plasmid. <br /> a temperature of 408. Autoradiographic bands were quanti- <br /> tated on a Molecular Dynamics PhosphorImager using Im-                 The largest complementation group (lsd6) contained <br /> ageQuant software.                                                  11 isolates. Two different types of experiments show <br />    Fluorescence-activated cell sorter (FACS) analysis: Cells        that lsd6 mutants contain null or intragenic mutations <br /> were grown in synthetic defined media at 238, diluted to 106        in cdc7. First, it was noted that 11 diploids, which re- <br /> cells/ml, allowed to grow to a density of 2&#8211;4 3 106 cells/ml        sulted from a backcross to cdc7-1 (after loss of pRS316- <br /> (midlogarithmic phase), and then processed for FACS analysis <br /> as described previously (Nash et al. 1988). Cell numbers and        CDC7-ADE3-URA3 plasmid), generated two viable and <br /> sizes were determined using a Coulter Mulitsizer II using an        two lethal spores upon tetrad dissection (<span id='am-38' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-39' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-42'  ></span>data</span> not shown). <br /> aperture tube with a 100-mm orifice and latex beads as size         All viable spores from the crosses were TRP1, indicating <br /> standards.                                                          linkage to CDC7 (,5 cM). Second, a plasmid bearing <br />    Immunoblot analysis: For immunoblot analysis, cells were         the cdc7-1 mutation could complement the cdc7 null <br /> grown in synthetic defined media to midlogarithmic phase <br /> (2 3 106 cells/ml) and analyzed with a Coulter Multisizer II.       mutation in the chromosome, resulting in a sectored <br /> A total of 2 3 107 cells were processed for yeast extracts. Cells   phenotype and supporting our hypothesis. The re- <br /> were washed twice in H2O and then resuspended in 20 ml SDS          maining five complementation groups contained one <br /> sample buffer plus protease inhibitors [phenylmethyl sufonyl        member each. <br /> fluoride (Sigma), 174 ng/ml; leupeptin (Peptide Institute,             Cloning of LSD1: Colony sectoring was used to iden- <br /> Inc.), 1.3 ng/ml; and pepstatin (Boehringer Mannheim, India- <br /> napolis, IN), 0.3 ng/ml], and then boiled immediately for 5         tify plasmids from a YCp50 yeast genomic library (Ameri- <br /> min. SDS-PAGE and immunoblot analysis was performed as              can Type Culture Collection, Rockville, MD) that could <br /> described previously ( Jackson et al. 1993), except that the        complement the lsd1-1 mutation. Out of z6900 colo- <br /> immunoblots were visualized with the ECL enhanced chemilu-          nies, one plasmid could complement the defect, and <br /> minescence kit (Amersham). The primary antibodies mouse             it was designated pPD58. A primer was then used to <br /> monoclonal 12CA5 anti-HA (Babco) and rabbit anti-G6PD <br /> (Sigma) were used at dilutions of 1:500 and 1:5000, respec-         sequence the regions flanking the insert within the <br /> tively. Secondary horseradish peroxidase-conjugated goat anti-      pPD58 library clone. The DNA sequence within the <br /> rabbit and goat anti-mouse antibodies (Bio-Rad, Richmond,           insert was compared to the Saccharomyces Genome Da- <br /> CA) were both used at a dilution of 1:3000.                         tabase and found to contain five contiguous open read- <br />                                                                     ing frames from chromosome XVI. One of these is <br />                                                                     RAD53, which was found to complement the nonsec- <br />                            RESULTS <br />                                                                     toring defect of strain PDY207. This suggests that the <br />   cdc7-1 synthetic lethal screen: A screen for mutations            lsd1-1 is an allele of RAD53. To prove this, the PEP4 <br /> that are lethal in combination with cdc7-1 was conducted            gene, which is adjacent to RAD53, was disrupted with <br /> to identify genes that are in the same genetic pathway              URA3 in lsd1-1. The lsd1-1 mutation always cosegregated <br /> or that affect the same biochemical process as CDC7.                with the pep4D::URA3 marker in subsequent crosses, <br />  970                                                P. R. Dohrmann et al. <br />  <br /> indicating that the two genes are tightly linked (,2 cM, <br /> data not shown). We conclude that lsd1-1 encodes an <br /> allele of RAD53, which we have named rad53-31. <br />    A null allele of RAD53 can be suppressed by RNR1 <br /> in high copy on plasmid pGAP-RNR1 (Sanchez et al. <br /> 1996; Desany et al. 1998; Zhao et al. 1998). However, <br /> high-copy RNR1 cannot suppress the nonsectoring de- <br /> fect of an lsd1-1 (5rad53-31) cdc7-1 strain, PDY207. This <br /> indicates that both the rad53-31 and cdc7-1 mutations <br /> are required to manifest the synthetic lethal phenotype. <br />    The DNA checkpoint is functional in a rad53-31 strain: <br /> Many alleles of RAD53 have been isolated in different <br /> screens for mutations with a checkpoint defect (Zheng <br /> et al. 1993; Allen et al. 1994; Weinert et al. 1994). Cells <br /> harboring a rad53-11 (5mec2-1) mutation, for example, <br /> exhibit rapid loss of viability in the presence of DNA- <br /> damaging agents, such as UV light, because they lack <br /> the ability to inhibit cell cycle progression (Zheng et al. <br /> 1993; Allen et al. 1994; Weinert et al. 1994). Likewise, <br /> rad53-11 mutants die rapidly in the presence of hydroxy- <br /> urea, a drug that normally stalls replication forks by <br /> limiting nucleotide availability through inhibition of <br /> ribonucleotide reductase (Allen et al. 1994; Weinert <br /> et al. 1994). We therefore tested rad53-31 for a check- <br /> point defect. As expected, the rad53-11 strain showed a <br /> marked decline in viability in response to increasing <br /> UV doses (Figure 1A). Surprisingly, the rad53-31 strain <br /> exhibited a sensitivity to UV damage comparable to that <br /> of the wild-type strain control (Figure 1A). <br />    For analysis of the S phase checkpoint, survival of <br /> wild-type, rad53-11, and rad53-31 strains in the presence <br /> of 200 mm hydroxyurea was examined. Again, as ex-                  Figure 1.&#8212;Absence of checkpoint deficiency in the rad53- <br /> pected, the rad53-11 strain demonstrated a marked de-           31 strain. (A) UV sensitivity of wild-type and rad53 mutant <br /> cline in viability (Figure 1B). In contrast, the rad53-31       strains. Wild-type strain 299 (solid square) and the rad53 mu- <br /> strain, like the wild-type, demonstrated no significant         tant strains TWY310, rad53-11 (open square) and PDY206, <br />                                                                 rad53-31 (open circle) were grown to late logarithmic phase <br /> loss in viability when grown in the presence of hydroxy- <br />                                                                 in liquid culture, plated onto YEPD plates at an appropriate <br /> urea. We conclude that rad53-31 is still proficient in          dilution (200&#8211;500 cfu/plate), irradiated with the indicated <br /> checkpoint function.                                            doses of UV light, and percent survival was determined relative <br />    Genetic interactions among RAD53 alleles and several         to unirradiated controls. Shown are the averages of three- <br /> cell division cycle mutations: A cdc7 rad53-11 (5mec2-1)        independent plates per dose of UV light. (B) Hydroxyurea <br />                                                                 sensitivity of wild-type and rad53 mutant strains. Wild-type <br /> double mutant is viable (Weinert et al. 1994). This <br />                                                                 strain 299 (solid square) and rad53 mutant strains TWY310, <br /> presents an apparent contradiction to our identification        rad53-11 (open square) and PDY206, rad53-31 (open circle) <br /> of rad53-31 as being lethal in combination with cdc7-1.         were grown to midlogarithmic phase in liquid culture, and <br /> However, we used a different cdc7 allele for our studies.       hydroxyurea was added (0.2 m). Aliquots were removed at <br /> To resolve this apparent discrepancy, we crossed rad53-         timed intervals to determine cell number and to score for <br />                                                                 viability as colony forming units on YEPD plates. Percentage <br /> 11 (5mec2-1) to all of the currently available cdc7 mu- <br />                                                                 of survival was determined relative to cells just before adding <br /> tants in our laboratory, and we assayed tetrads for their       hydroxyurea. Three independent plates were scored for each <br /> ability to form rad53-11 cdc7-x spores (Table 3A). Three        individual time point. <br /> alleles of CDC7 demonstrated a synthetic effect in combi- <br /> nation with rad53-11 at permissive temperature. Double <br /> mutants could not be recovered for rad53-11 and cdc7-3          31 found that cdc8-1 rad53-31 double mutants could be <br /> or cdc7-7, and the combination of rad53-11 with cdc7-1          easily recovered (Table 3B). We conclude from this <br /> exhibited a slow-growth phenotype. We confirmed the             experiment that the cdc8-1 rad53-11 lethal interaction <br /> previous report that cdc7-4 rad53-11 double mutants are         must be the result of the loss-of-checkpoint function in <br /> viable (Weinert et al. 1994). We conclude RAD53 dem-            rad53-11 (Weinert et al. 1994). We believe that cdc8-1 <br /> onstrates a genetic interaction with CDC7.                      rad53-31 double mutants are viable because rad53-31 <br />    The rad53-11 (5mec2-1) is synthetically lethal with          retains an intact checkpoint function. <br /> cdc8-1 (Weinert et al. 1994). Crosses of cdc8-1 with rad53-       We reasoned that, because rad53-31 was lethal with <br />                                                      RAD53 Regulation of DBF4                                              971 <br />  <br />                             TABLE 3                                we asked if bob1-1 could bypass the requirement for <br />          Genetic interactions between rad53 and several            the RAD53 gene. A rad53D::URA3 disruption was con- <br />                   cell division cycle mutations                    structed in a heterozygous diploid strain carrying a <br />                                                                    bob1-1, analyzed by Southern blot to confirm the disrup- <br /> Strain                         Xrad53-11 (PDY258)                  tion (data not shown), allowed to sporulate, and individ- <br />                                                                    ual spores were analyzed for viability (Table 4A). The <br />    A. Interaction between rad53-11 and various cdc7 alleles <br /> Wild type (299)      Viable (10 tetrads) <br />                                                                    results are complicated by the fact that strains bearing <br /> cdc7-1 (PDY177)      Slow growth phenotype (18 tetrads)            rad53D::URA3 frequently generate second site suppres- <br /> cdc7-3 (209)         Synthetic lethality/inviable (12 tetrads)     sors (Zheng et al. 1993; Kim and Weinert 1997; Desany <br /> cdc7-4 (576)         Viable (12 tetrads)                           et al. 1998). The number of 3:1 and 4:0 viable:lethal <br /> cdc7-7 (708)         Synthetic lethality/inviable (33 tetrads)     tetrad patterns suggests that bob1-1 is not responsible <br /> Strain                         Xrad53-31 (PDY289)                  for the suppression (Table 4A). This was confirmed by <br />                                                                    performing a second cross where bob1-1 was homozygous <br />          B. Interaction between rad53-31 and cdc8-1                in the diploid. Many 3:1 and 2:2 viable:lethal segregation <br /> cdc8-1 (199)         Viable (18 tetrads) <br />                                                                    patterns were generated, indicating that a second site <br />          C. Interaction between rad53-31 and dbf4-1                suppressor was responsible for the suppression, not <br /> dbf4-1 (PDY029)      Slow growth phenotype (31, 24 tetradsa)       bob1-1 (Table 4B). We conclude that bob1-1 is unable to <br />    Genetic interactions between rad53 and various cell cycle       bypass rad53D::URA3. <br /> mutations were tested by crossing individual strains together.        Interaction of Rad53p and Dbf4p: Given that RAD53 <br /> The diploids were sporulated and dissected and the resulting       interacts genetically with CDC7 and DBF4, we asked if <br /> segregants were grown at the permissive temperature of 238.        Rad53p could interact physically with Cdc7p and/or <br /> The rad53-11 and the rad53-31 alleles were marked by a             Dbf4p, as assayed by the two-hybrid method. The GAL4- <br /> pep4D::URA3 marker, which is located directly adjacent to the <br /> RAD53 gene. The number in parentheses indicates how many           DB-RAD53 (aa50&#8211;aa822) could complement rad53D:: <br /> informative tetrads were dissected for each cross (except in       URA3, indicating that it encodes a functional protein <br /> the case of the dbf4-1 cross, see below for details). In crosses   (data not shown). The GAL4-DB RAD53i (aa50&#8211;aa457), <br /> that demonstrated a synthetic lethality/inviable phenotype,        which lacks C-terminal sequences, failed to complement <br /> no viable double mutants were detected. In addition, no sig-       the rad53D::URA3 (data not shown). The results show <br /> nificant deviation from the 1PD:4T:1NPD ratio was observed <br /> for all the crosses, as predicted for the segregation of two       that Rad53p interacts weakly with Dbf4p, but not with <br /> unlinked genes (except for dbf4-1, see below). An extremely        Cdc7p (Figure 2). In addition, the Rad53p fusion con- <br /> slow growth phenotype was detected in tetrad segregants from       struct that lacks the C-terminal sequences failed to inter- <br /> the cdc7-1 cross. Upon restreaking, double mutants from the        act with either Cdc7p or Dbf4p, suggesting that the <br /> cdc7-1 cross grew very poorly.                                     C-terminal sequences are necessary for interaction with <br />    a <br />      For the dbf4-1 cross, double mutants could be obtained; <br /> however, results were complicated by the fact that out of the      Dbf4p. Previously described interactions were seen be- <br /> original 31 tetrads dissected, the dbf4-1 mutation reverted to     tween Dbf4p and Cdc7p, and between Dbf4p and Dbf4p <br /> wild type in 22 cases, generating many 3:1 and 4:0 segregation     (Jackson et al. 1993; Shellman et al. 1998). <br /> patterns for temperature resistance. To prove that double             Several attempts were made to show that Dbf4p and <br /> mutants were viable, this diploid was then transformed with        Rad53p interact directly using other physical methods. <br /> the pRS425-DBF4 to cover the dbf4-1 mutation, 24 tetrads were <br /> dissected, and plasmids were allowed to be lost from the tetrad    Rad53p/Dbf4p coimmunoprecipitation and GST-Rad <br /> segregants. In this case, dbf4-1 rad53-31 double mutants were      53p/Dbf4p interaction experiments failed to detect an <br /> easily obtained, and dbf4-1 always segregated 2:2.                 interaction (data not shown). This may reflect a tran- <br />                                                                    sient or weak interaction between Rad53p and Dbf4p <br />                                                                    that can only be detected by the sensitive two-hybrid <br /> cdc7-1, it might also be lethal with dbf4-1 because CDC7           interaction assay. Nevertheless, two independent two- <br /> and DBF4 act at the same point in the pathway (Kitada              hybrid assays could detect a significant interaction be- <br /> et al. 1992). The rad53-31 dbf4-1 double mutant could              tween Rad53p and Dbf4p. <br /> be recovered if the dbf4-1 mutation in the diploid was                RAD53 regulates DBF4 expression at the mRNA and <br /> first complemented by wild-type DBF4 on a plasmid,                 protein levels: To understand how rad53-31 manifests <br /> and then the plasmid was allowed to be lost in tetrad              its synthetic lethal effect with cdc7-1, we asked whether <br /> segregants (Table 3C). The rad53-31 dbf4-1 double mu-              DBF4 mRNA and/or Dbf4p protein levels were altered <br /> tant demonstrated a synthetic growth defect. We con-               in various RAD53 mutants. The level of DBF4 message <br /> clude that RAD53 demonstrates a genetic interaction                was significantly reduced in the rad53D::hisG strain, ex- <br /> with DBF4.                                                         hibiting about fivefold less mRNA than the wild type <br />    The bob1-1 mutation cannot suppress a rad53D::URA3              (Figure 3A). This was not a result of overexpression of <br /> null allele: A recessive mutation in BOB1(MCM5/CDC46)              the RNR1 gene because wild-type cells that overexpress <br /> bypasses the requirement for the essential S phase activa-         RNR1 demonstrate no difference in DBF4 expression <br /> tors Cdc7p and Dbf4p at the G1/S boundary ( Jackson                (data not shown). In addition, the Northern blot results <br /> et al. 1993; Hardy et al. 1997). Given the fact that RAD53         demonstrated that the level of DBF4 message in the <br /> interacts genetically with CDC7 and DBF4 (see above),              rad53-11 strain was reduced about twofold. Surprisingly, <br />  972                                                  P. R. Dohrmann et al. <br />  <br />                                                             TABLE 4 <br />                          The bob1-1(mcm5) mutation does not suppress a null mutation in rad53D::URA3 <br />  <br />         Relevant genotype of strain dissected                          Type of tetrad                       Number <br />                          A. bob1-1 or second site suppressor suppresses rad53D::URA3 mutation <br />         bob1/1 1/rad53D::URA3                                   4:0 viable:inviable                             0 <br />         (299 3 P119)                                            3:1 viable:inviable                             9a <br />                                                                 2:2 viable:inviable                            16 <br />                                                                 Total tetrads dissected                        25 <br />             B. Backcross to determine if viable segregants are suppressed by bob1-1 or by a second site suppressor <br />         bob1/bob1 1/rad53D::URA3 sup?/1                           4:0 viable:inviable                           13 <br />                                                                   3:1 viable:inviable                            7 <br />                                                                   2:2 viable:inviable                            4 <br />                                                                   Total tetrads dissected                       24 <br />           a <br />               Viable bob1-1 rad53D::URA3 from part A was backcrossed to bob1-1 strain. <br />  <br />  <br /> the level of DBF4 message was increased in the rad53-              protein expression in the rad53-31 and rad53-11 strains <br /> 31 strain.                                                         was similar to that seen for mRNA expression. Dbf4p <br />    DBF4 is an MCB box-regulated gene that is expressed             protein levels were reduced in the rad53-11 strain, <br /> just before S phase (Lowndes et al. 1992; Zheng et al.             whereas they were slightly increased in the rad53-31 <br /> 1993). Thus, it was possible that RAD53 could be acting            strain. We conclude from these experiments that DBF4 <br /> on the expression of this class of genes. If so, one would         is regulated in a RAD53-dependent manner at both the <br /> expect another MCB box-regulated gene, such as POL1,               mRNA and protein levels. <br /> to be downregulated in the same manner in the rad- <br /> 53D::hisG strain. However, the POL1 mRNA level was <br />                                                                                          DISCUSSION <br /> not significantly different between wild-type and rad- <br /> 53D::hisG strains (Figure 3A). We conclude that RAD53                 The activation of DNA repair mechanisms and simul- <br /> does not universally affect MCB box-regulated genes.               taneous activation of cell cycle arrest by DNA damage <br />    We hypothesized that the elevated levels of DBF4 (and           checkpoints results in minimizing the effects of DNA <br /> POL1) message in the rad53-31 strain could be a result             damage to eukaryotic cells. Failure of the cellular re- <br /> of cells being shifted toward one phase of the cell cycle.         straints imposed normally by a checkpoint can result <br /> To test this possibility, FACS analysis was performed on           in increased mutation rates, genomic instability, and ulti- <br /> the yeast strains that were used to make the extracts              mately death if cells continue to divide unchecked. It <br /> (Figure 3C). Most of the rad53-31 cells were in the G2             is perhaps not so surprising that the signals involved in <br /> phase of the cell cycle. It should be noted that this is           eukaryotic DNA checkpoint control could be integrated <br /> the first phenotypic defect we have observed for strains           with other DNA replication functions. It has been sug- <br /> that bear only the rad53-31 genotype. The level of DBF4            gested that the RAD53 gene, which is involved in the <br /> message was then compared between wild-type and rad-               DNA checkpoint, encodes a signal-transducing kinase <br /> 53-31 mutant extracts that were prepared from cells                that could integrate several of these functions (Zheng <br /> arrested at the same stage of the cell cycle (Figure 3B).          et al. 1993; Weinert 1998). <br /> Cells were grown first to midlogarithmic phase and then               It has been postulated that RAD53 encodes dual func- <br /> arrested in G1/S in the presence of 200 mm hydroxyurea             tions, including positive regulation of replication and <br /> for 4 hr. DBF4 message is only moderately reduced in               negative regulation of cell cycle progression (Sun et al. <br /> the rad53-31 strain (Figure 3B).                                   1996). Teleologically, a cell needs to inhibit cell cycle <br />    To test the level of Dbf4p protein expression in the            progression for the DNA repair, but replication enzymes <br /> four strains, cells carrying a plasmid with a HA-tagged            need to be activated to repair damage. This predicts <br /> DBF4 plasmid were grown to midlogarithmic phase and                that the DNA replication and the checkpoint functions <br /> then harvested to prepare the cells for FACS and the               could be separable. <br /> extracts for immunoblot analysis (Figure 4). FACS pro-                To this point, the evidence is circumstantial regarding <br /> files generated for the four strains were similar to those         the role of RAD53 in the control of DNA replication. <br /> shown in Figure 3. Immunoblots were then probed with               Several lines of evidence suggest that RAD53 is associ- <br /> antibodies against 7HA-DBF4p and G6PDp. Dbf4p was                  ated with DNA replication: transcriptional coregulation <br /> undetectable in the rad53D::hisG strain, indicating that           with other DNA synthetic enzymes, the terminal arrest <br /> RAD53 positively regulates Dbf4p. Longer exposures re-             phenotype or rad53D null mutants, the nuclear localiza- <br /> vealed a very low level of Dbf4p expression in the rad-            tion of Rad53p, and the essential function of RAD53 <br /> 53D::hisG strain (data not shown). The level of Dbf4p              (Zheng et al. 1993). Weinert (1998) suggested that the <br />                                                 RAD53 Regulation of DBF4                                               973 <br />  <br />  <br />  <br />  <br />                                                                                              Figure 2.&#8212;Interaction of <br />                                                                                           Rad53p with Dbf4p. (A) Proto- <br />                                                                                           trophic growth in strain PJ69- <br />                                                                                           4a is dependent on the inter- <br />                                                                                           action between RAD53 and <br />                                                                                           DBF4 two-hybrid fusions. Strain <br />                                                                                           PJ69-4a, which contains ADE2 <br />                                                                                           under GAL2 promoter control, <br />                                                                                           was transformed with the indi- <br />                                                                                           cated two-hybrid fusion plas- <br />                                                                                           mids, and selection was exe- <br />                                                                                           cuted on SD-Leu-Trp plates. <br />                                                                                           To assay for interaction, colo- <br />                                                                                           nies were streaked onto SD- <br />                                                                                           Leu-Trp-Ade plates, shown <br />                                                                                           above. pRad53DB fuses aa50&#8211; <br />                                                                                           aa822 to the Gal4pDNA-bind- <br />                                                                                           ing domain, whereas pRad53i <br />                                                                                           fuses aa50&#8211;aa457. (B) Quanti- <br />                                                                                           tative liquid b-galactosidase <br />                                                                                           assays. The indicated DNA- <br />                                                                                           binding domain and activation <br />                                                                                           domain plasmids were trans- <br />                                                                                           formed into the strains Y187 <br />                                                                                           and Y190, strains were mated, <br />                                                                                           and quantitative b-galactosi- <br />                                                                                           dase measurements were made <br />                                                                                           from diploid yeast extracts. As- <br />                                                                                           says were completed for three <br />                                                                                           independent colonies from each <br />                                                                                           diploid strain. <br />  <br />  <br />  <br />  <br /> essential function of RAD53 could be related to the          bility is that RAD53 acts in the same genetic pathway as <br /> timing of mitosis or some aspect of DNA replication.         CDC7 and DBF4. The allele specificity seen with rad53- <br />    The experiments performed here provided the strong-       11 and different cdc7 alleles illustrates the fact that a <br /> est evidence yet that there is a direct requirement for      certain level of Cdc7p kinase activity is required in com- <br /> RAD53 in the initiation of DNA replication. Our results      bination with rad53-11, or the cells become inviable. As <br /> indicate that the checkpoint function of the rad53-31        measured originally through segregation lag experiments, <br /> allele remains intact. At the same time, the rad53-31        a hierarchy of function of cdc7 mutants at permissive <br /> allele is lethal in combination with cdc7-1, a gene that     temperature was determined (Hollingsworth et al. <br /> is intimately related to the initiation of DNA synthesis.    1992). It is clear that cdc7-3 and cdc7-7 have the least <br /> Indeed, it appears that the checkpoint and replication       activity at permissive temperature, whereas the cdc7-4 <br /> functions of RAD53 can be separated. This effect can         has the highest level of activity. This explains why cdc7-4 <br /> only be seen in combination with cdc7-1 because muta-        rad53-11 (5mec2-1) double mutants were originally <br /> tions that completely knock out the replication function     found to be viable (Weinert et al. 1994). The level of <br /> of RAD53 presumably also knock out the checkpoint            activity of cdc7 mutant alleles most likely determines <br /> function and perhaps the essential function of RAD53.        the allele-specific interaction with rad53-11. In addition, <br /> Similarly, a synthetic growth defect was seen by rad53-      when analyzed by FACS, the rad53-31 allele demon- <br /> 31 in combination with dbf4-1. The RAD53 synthetic           strates a G2 profile that is reminiscent of defects in <br /> defects seen in combination with both CDC7 and DBF4          other DNA replication genes, such as in orc5-1 and mcm2 <br /> imply that RAD53 affects a similar biochemical process       mutants (Yan et al. 1991; Loo et al. 1995). Finally, Rad <br /> or a similar function as CDC7 and DBF4. Another possi-       53p was shown to interact with Dbf4p in two-hybrid ex- <br />  974                                                  P. R. Dohrmann et al. <br />  <br />  <br />  <br />  <br />    Figure 3.&#8212;Steady-state level of DBF4 mRNA is reduced in the rad53D::hisG strain. (A) A Northern blot was analyzed containing <br /> RNA prepared from four isogenic strains: wild-type strain PDY305, rad53-31 strain PDY306, rad53-11 strain PDY307, and rad53D::hisG <br /> strain PDY308. The Northern blot was probed sequentially with a DBF4 probe, a POL1 probe, and then a probe against 18s rRNA <br /> for an internal control. Cells were grown to midlogarithmic phase before harvesting for RNA analysis and FACS (see C). <br /> Autoradiographic bands were quantitated on a Molecular Dynamics PhosphorImager, normalized to the 18s rRNA internal <br /> control, and compared to wild type, which was set at 100%. (B) rad53-31 cells blocked in G1/S demonstrate moderately reduced <br /> levels of DBF4 message. Cells were grown to midlogarithmic phase and cultured for 4 hr in the presence of 0.2 m hydroxyurea, <br /> and were then harvested for RNA analysis and FACS. (C) FACS analysis of isogenic yeast strains used to prepare RNA extracts. <br />  <br />  <br />  <br /> periments. Taken together, the genetic and two-hybrid               URA3 because it is required for integrating the G1, S, <br /> studies strongly suggest that RAD53 has a positive role             and G2 checkpoints. Perhaps the cell cannot survive <br /> in regulating DNA replication.                                      without all three functions. Alternatively, RAD53 may <br />    The predominant G2 population seen in the rad53-                 be required for other essential functions, such as expres- <br /> 31 mutant via FACS analysis could also be a consequence             sion of RNR1 (Desany et al. 1998). <br /> of the rad53-31 allele generating a weak but constitutive              The two lines of evidence presented here also illus- <br /> checkpoint signal. Perhaps the synthetic lethality seen             trate that RAD53 exerts its positive control through <br /> with the cdc7-1 rad53-31 double mutant is a combination             DBF4. First, two-hybrid studies suggest that Rad53p may <br /> of reduced activity of the cdc7-1 gene product and a                interact directly through Dbf4p. Second, RAD53 regu- <br /> dominant but weak checkpoint signal from rad53-31.                  <span id='am-25' about='obo:NCBITaxon_8186' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_1489905 obo:NCBITaxon_123366 obo:NCBITaxon_1489904 obo:NCBITaxon_1489872 obo:NCBITaxon_8184 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665'><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Lates" datatype="xsd:string"></span><span id='am-28'  ></span><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>lates</span> the expression of DBF4 at the message and protein <br />    Given the fact that the bob1-1 mutation can bypass               levels. The DBF4 message is reduced fivefold, and the <br /> the requirement for CDC7 and DBF4 (Jackson et al.                   levels of Dbf4p protein are virtually undetectable in a <br /> 1993; Hardy et al. 1997), why is it that bob1-1 cannot              rad53D::hisG strain. This indicates that RAD53 regulates <br /> suppress rad53D::URA3? It is probable that bob1-1 can-              DBF4 positively. At this point, we do not have an explana- <br /> not bypass the checkpoint function absent in rad53D::               tion for why either DBF4 message or Dbf4p protein is <br />                                                   RAD53 Regulation of DBF4                                                        975 <br />  <br />  <br />  <br />  <br />    Figure 4.&#8212;Steady-state Dbf4p protein levels are virtually undetectable in <br /> the rad53D::hisG strain. (A) Immunoblot analysis of Dbf4p and G6PDp was <br /> performed with extracts prepared from the indicated isogenic strains: wild- <br /> type strain PDY305, rad53-31 strain PDY306, rad53-11 strain PDY307, and rad- <br /> 53D::hisG strain PDY308. Cells were grown to midlogarithmic phase and then <br /> harvested for protein analysis and FACS. (B) FACS analysis of isogenic yeast <br /> strains used to prepare protein extracts. <br />  <br />  <br />  <br />  <br /> slightly increased in the rad53-31 strain. Could a quanti-       altered in various RAD53 mutant strains. This may help <br /> tative difference in the Dbf4p protein be responsible            elucidate the role of RAD53 in controlling Dbf4p ac- <br /> for generating the synthetic lethal defect of a rad53-31         tivity. <br /> cdc7-1 strain? We know that lowering the level of DBF4              In conclusion, we have demonstrated that the multi- <br /> message twofold in a cdc7-1 background is still viable           ple functions of RAD53 can be separated. These results <br /> (P. Dohrmann, unpublished results). This suggests that           suggest strongly that RAD53 is a common intermediate <br /> the reason rad53-31 is lethal with cdc7-1 is not quantita-       between a checkpoint and DNA replication function. <br /> tive because the level of DBF4 message seen in a rad53-      <br /> 31 strain is 80% of wild type (Figure 3B). Therefore,           <br /> it follows that RAD53 must regulate some qualitative         <br /> difference in Dbf4p protein.                                 <br />    We would hypothesize that perhaps Rad53p binds to           <br /> and modifies Dbf4p protein. The qualitative difference         <br /> in Dbf4p activity would be partly responsible for the        <br /> activation of the DNA replication initiation program.       <br /> In addition, the activation of DBF4 transcription and/          <br /> or degradation of DBF4 message through RAD53 could <br /> be through a positive feedback loop based on the level <br /> of activity of the Dbf4p protein. This would explain                               <br /> reasonably why the DBF4 message and protein levels in <br />                                                                <br /> rad53-31 mutants are not affected severely. At the same              <br /> time, in rad53D::hisG mutants, Rad53p is unavailable to       <br /> modify Dbf4p, and as a consequence, DBF4 message           <br /> and Dbf4p protein rapidly disappear.                       <br />    With the dramatic reduction of Dbf4p protein, the                 <br /> cells are still viable in the rad53D::hisG strain. Perhaps       <br /> the essential role of RAD53 is to regulate the expression      <br /> of RNR1 because high-copy expression of RNR1 can                  <br /> bypass the rad53D::hisG defect. At the same time, RAD53       <br /> may also regulate DBF4 levels to ensure rapid S phase           <br /> entry through this positive feedback loop. Future experi-         <br /> ments will address whether Cdc7p kinase activities are      <br /> </body></html>