<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Aug. 1997, p. 4664&#8211;4670                                                                                     Vol. 179, No. 15 <br /> 0021-9193/97/$04.0010 <br /> Copyright &#169; 1997, American Society Microbiology <br />  <br />  <br />  <br />  The <span id='am-23' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-24' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-29'  ></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> Mevalonate Diphosphate Decarboxylase <br />     Is Essential Viability, Single Leu-to-Pro Mutation in <br />           Conserved Sequence Leads Thermosensitivity <br />                                         ` S,* DANIEL GUYONNET, <br />                             THIERRY BERGE                                               AND   FRANCIS KARST <br />            Faculte&#180; des Sciences, Laboratoire Ge&#180;ne&#180;tique Physiologique et Mole&#180;culaire, Institut Biologie Mole&#180;culaire <br />                           et d&#8217;Inge&#180;nierie Ge&#180;ne&#180;tique, Universite&#180; Poitiers, F-86022 Poitiers Cedex, France <br />                                               Received 6 March 1997/Accepted 2 June 1997 <br />  <br />             The mevalonate diphosphate decarboxylase enzyme converts mevalonate diphosphate <span id='am-77' about='obo:NCBITaxon_238707' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_8075 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_1489917'><span id='am-78'  ></span><span id='am-79' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="Iso" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>iso</span>- <br />          pentenyl diphosphate, building block isoprenoids. We used <span id='am-40' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-46'  ></span>Saccharomyces</span> cerevisiae temperature- <br />          sensitive mutant defective mevalonate diphosphate decarboxylase previously described (C. Chambon, V. <br />          Ladeve`ve, M. Servouse, L. Blanchard, C. Javelot, B. Vladescu, F. Karst, Lipids 26:633&#8211;636, 1991) charac- <br />          terize mutated allele. We showed single change conserved amino acid accounts tempera- <br />          ture-sensitive phenotype mutant. Complementation experiments erg19-mutated <br />          background strain ERG19 gene, shown essential gene yeast, was <br />          disrupted. Epitope tagging wild-type mevalonate diphosphate decarboxylase allowed isolate the <br />          enzyme active form versatile one-step immunoprecipitation procedure. Furthermore, the <br />          course study, observed high level expression wild-type ERG19 gene led lower sterol <br />          steady-state accumulation compared wild-type strain, suggesting enzyme key <br />          enzyme mevalonate pathway regulation. <br />  <br />  <br />    The mevalonate (MVA) pathway provides cell nu-                  cumulation fetal brain development phenylketonuria <br /> merous fundamental molecules called isoprenoids, including                  results severe brain damage leading microcephaly and <br /> sterols, chain ubiquinone, dolichol, prenyl groups           mental retardation (24). As MVA-PP decarboxylase an <br /> prenylated proteins ras, plant hormones, aromatic               early enzyme MVA pathway, phenylacetate effects may <br /> compounds, (11, 14). For reason, groups                  lack fundamental intermediary end prod- <br /> focused pathway, potential target inhibi-             ucts MVA pathway. These observations led trials <br /> tors acting antifungal compounds agriculture               using phenylacetate antiproliferative agent malig- <br /> medicine cellular antiproliferative agents means             nant gliomas, form primary brain tumors (23). Recently <br /> for lowering serum cholesterol levels (for review, refer-             Prasanna et al. (22) shown specific inhibition the <br /> ence 20). For instance, simvastatin, inhibitor 3-hy-              MVA-PP decarboxylase phenylacetic acid sodium salt <br /> droxy-3-methylglutaryl&#8211;coenzyme A (HMG-CoA) reductase                    (NaPA), used lovastatin, specific inhibi- <br /> used hypocholesterolemic agent terbinafin, inhibitor               <span id='am-67' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-68' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-70'  ></span><span id='am-71' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>tor</span> HMG-CoA reductase, reverse malignant phe- <br /> of fungal squalene epoxidase, ketoconazole, inhibitor               notype human glioblastoma cells. <br /> of fungal lanosterol-14-demethylase, known anti-                  Although temperature-sensitive yeast strain deficient in <br /> fungal agents commonly used human disease therapy.                   MVA-PP decarboxylase activity isolated characterized <br />    Most structural genes coding enzymes involved             years ago (10), far cloning functional complemen- <br /> in sterol biosynthesis cloned characterized              tation enzyme&#8217;s structural gene failed, mainly be- <br /> many organisms, regulation pathway higher                    cause relatively high reversion rate MN19-34 <br /> eucaryotes documented (14). A substantial con-                mutant deficient MVA-PP decarboxylase. <br /> tribution cloning genes come                      Recently Toth Huwyler (26) reported isolation of <br /> isolation collection ergosterol auxotrophic mutants gen-            cDNAs encoding rat, human, yeast MVA-PP decarbox- <br /> erated yeast Saccharomyces cerevisiae (17). Several sterol           ylase (which called MPD) help peptide se- <br /> biosynthesis structural genes cDNAs cloned                  quence <span id='am-82' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-83'  ></span><span id='am-84' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> obtained enzyme purified rat liver <br /> functional complementation mutants (13, 18, 21).                   (27). <br />    Of enzymes MVA pathway, MVA diphosphate                          In paper report cloning gene encoding the <br /> (MVA-PP) decarboxylase known. <span id='am-47' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-50'  ></span><span id='am-51' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span>                  yeast MVA-PP decarboxylase functional complementation <br /> enzyme converts MVA-PP isopentenyl diphosphate (IPP),                  yeast mutant isolated laboratory that <br /> the building block isoprenoids, requires ATP Mg21                gene essential yeast cell viability. We report the <br /> for activity (8).                                                       cloning sequencing mutated recessive allele. We <br />    MVA-PP decarboxylase shown competitively                  localized mutation conserved amino acid. We <br /> inhibited phenylalanine derivatives, phenylpyruvate,             single change leucine proline suffi- <br /> phenylacetate, phenylalanine itself (9). Phenylacetate ac-              cient confer temperature-sensitive phenotype strain <br />                                                                             harboring allele. <br />  <br />   * Corresponding author. Mailing address: IBMIG, Laboratoire de <br /> Ge&#180;ne&#180;tique Physiologique et Mole&#180;culaire, 40 avenue du Recteur                                   MATERIALS AND METHODS <br /> Pineau, F-86022 Poitiers Cedex, France. Phone: (33) 0549-45-40-29.            Cloning characterization ERG19 gene. The yeast genomic library <br /> Fax: (33) 0549-45-35-03. E-mail: berges@hermes.univ-poitiers.fr.            generous gift F. Lacroute. In library, yeast genomic DNA <br />  <br />                                                                      4664 <br />  VOL. 179, 1997                                                          EFFECTS OF MVA-PP DECARBOXYLASE GENE ON VIABILITY                                          4665 <br />  <br />  <br /> fragments partially digested Sau3A ligated high-copy-number plas-       (3,000 3 g, 2 min), supernatant kept (SUP fraction) pellet was <br /> mid pFL44L (7). The gene marker used select transformants URA3              washed times potassium phosphate buffer. The pellet eventually <br /> gene. This library used transform temperature-sensitive strain MN19-34         resuspended sodium dodecyl sulfate-polyacrylamide gel electrophoresis load- <br /> (MATa ura3 trp1), deficient MVA-PP decarboxylase activity             ing buffer. <br /> which derived original mutant strain MNerg19 (10). Approximately 105            MVA-PP decarboxylase activity determination. To assay enzyme activity, a <br /> independent Ura1 transformants pooled plated selective medium          crude protein extract centrifuged 12,000 3 g prepared described above. <br /> a nonpermissive temperature (36&#176;C). One strains growing 36&#176;C showed         To 200 mg protein mixture 10 mM MgCl2, 20 mM ATP, 0.5 mM NADPH, <br /> plasmid-dependent growth kept analysis. This strain har-          1 mM glucose-6-phosphate, 2 U glucose-6-phosphate dehydrogenase, 23.5 <br /> bored plasmid containing genomic insert size estimated            mM [2-14C]MVA (53 mCi/mmol) potassium phosphate buffer (0.1 M, pH 7.4) <br /> about 6.5 kb. A unique SalI site unique BglII site located approximately     added, total volume 200 ml. <br /> the middle 6.5-kb genomic insert used, addition restriction sites      Following 40-min incubation 37&#176;C, 500 mg cold carrier ergosterol, <br /> of high-copy-number plasmid pFL44L, subclone original insert. Two              lanosterol, squalene, squalene oxide added. The mixture gently <br /> subclones corresponding SmaI-BglII restriction fragment ;3 kb           saponified 30 min 42&#176;C presence methanolic KOH (10% final <br /> SalI-SalI restriction fragment ;3 kb subcloned plasmid pFL44S (7)       concentration) extracted heptane. The heptane phase concentrated <br /> generate, respectively, pTB1 pTB2. These <span id='am-87' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-88' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>plasmids</span> subsequently           vacuum, radioactivity aliquot measured liquid scin- <br /> transformed temperature-sensitive strain MN19-34. Only pTB2 reversed         tillation. The main fraction separated Silica Gel G developed with <br /> phenotype mutated strain, i.e., allowing growth 36&#176;C. The insert con-       dichloromethane. The distribution radioactivity determined a <br /> tained pTB2 partly deleted using hand XbaI site           Berthold 2382 thin-layer chromatography (TLC) scanner. To assay activity of <br /> located insert multiple cloning site generate pTB3      immunoprecipitated protein, experiment performed the <br /> on hand PstI site generate pTB4. The complementing activity        presence immobilized fusion protein immunoprecipitated con- <br /> restricted 1.94-kb PstI-SalI restriction fragment pTB4,      ditions described (200 mg protein). <br /> fully sequenced.                                                                         Sterol extraction analysis. Sterol extraction essentially de- <br />    Disruption ERG19 gene. In <span id='am-60' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> inactivate ERG19 gene,           scribed previously (16). Sterols quantified UV absorption (% 5,7 5 <br /> BglII restriction sites generated PCR. The sequences oligo-        OD281.5 3 volume heptanolic phase milliliters 3 396/11,500 3 dry <br /> nucleotides 59TTTAGATCTGGTGCTCACTGTCTGG39 (195BG),                         weight milligrams, %5,7 percentage dry weight D5,7 <br /> was designed create BglII site position 538 (from PstI site), 7     sterols [sum dry weight percentages ergosterol ergosta-5,7-dienol]), <br /> nucleotides upstream ATG start <span id='am-52' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-53' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-55'  ></span><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span>, 59ATTAGATCTTGAGT                   various <span id='am-57' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> analyzed gas-liquid chromatography. The gas <br /> TGCAAAAGG39 (193BG), designed create second BglII site              chromatograph (GC 6000; Carlo-Erba) supplied flame-ionization <br /> position 1620, 112 nucleotides upstream TAA stop codon. The pUC18-           detector on-column injector. The column RSL 150 (Altech) <br /> 295 construct used template PCRs. PCR product 1 (PCR1)            capillary. Cholesterol (Sigma) used internal standard determine <br /> about 540 bp corresponding promoter ERG19 gene                 relative retention times (in minutes), follows: squalene, 0.72; <br /> obtained universal M13 195BG primers. PCR product 2 (PCR2)                cholesta-8,24-dienol (zymosterol), 1.06; ergosta-5,7,22-trienol (ergosterol), 1.09; <br /> about 320 bp corresponding 39 ERG19 gene obtained         ergosta-8,24(28)-dienol (fecosterol) ergosta-8,14-dienol (ignosterol), 1.13; <br /> the reverse 193BG primers. PCR1 digested PstI BglII,         ergosta-8-enol, 1.14; ergosta-7,24(28)-dienol (episterol) ergosta-5,7-dienol, <br /> PCR2 digested BglII SalI. Both fragments ligated           1.16; 4,4,14-trimethylcholesta-8,24-dienol (lanosterol), 1.24. <br /> pUC18 vector digested PstI SalI, generating pUC18-D19 interme- <br /> diary construct, coding sequence ERG19 gene is <br /> deleted.                                                                                                                RESULTS <br />    The TRP1 gene prepared YDp-W plasmid 0.8-kb restriction <br /> fragment ligated pUC18-D19 cut BglII dephosphorylated,                 Cloning characterization ERG19 gene. We have <br /> generating pUC18-D19W plasmid. The pUC18-D19W plasmid cut                isolated genomic DNA fragment able reverse temper- <br /> with PstI SalI order purify 1.66-kb restriction fragment harboring      ature-sensitive phenotype MN19-34 strain deficient in <br /> the erg19::TRP1 allele, subsequently transformed 5302 (trp1/       MVA-PP decarboxylase activity. <br /> trp1 ura3/ura3) diploid strain (19). Trp1 transformants sporulated, and <br /> tetrads dissected.                                                                  The insert size plasmid library (pDG295) <br />    Epitope tagging MVA-PP decarboxylase. Two Z domains corresponding           carrying complementing activity estimated 6.5 kb. <br /> to immunoglobulin G (IgG) binding domains protein A Staphylo-             A limited restriction map developed permit choice <br /> <span id='am-72' about='obo:NCBITaxon_538669' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_33381 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_33380 obo:NCBITaxon_6072 obo:NCBITaxon_7524 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_33373 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_30189 obo:NCBITaxon_33342 obo:NCBITaxon_50557'><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-74'  ></span><span id='am-75' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="Coccus" datatype="xsd:string"></span>coccus</span> aureus fused frame N terminus MVA-PP decarbox-           convenient restriction sites subcloning high-copy- <br /> ylase. The 1.1-kb BamHI-SacI restriction fragment corresponding NOP1 <br /> promoter Z domains protein A prepared                  number plasmid pFL44S (7). <br /> pProtA-NOP1 plasmid (5). In order fuse frame coding sequence            After subcloning steps (Fig. 1; Materials and <br /> ERG19 gene, designed PA19 oligonucleotide contains SacI                Methods) complementing activity restricted a <br /> restriction site just downstream ATG codon (59TTTGAGCTCGACC                  1.94-kb restriction fragment sequence determined. <br /> GTTTACACAGCATC39). The ERG19 gene amplified PCR the <br /> PA19 reverse primers, pUC18-295 construct template.           Two head-to-head open reading frames (ORFs) on <br /> The 1.4-kb PCR product digested SacI SalI ligated          insert: incomplete ORF encoding 102 amino acids was <br /> the 1.1-kb BamHI-SacI fragment (NOP1 promoter plus Z domains)             identified sequence comparison portion yeast <br /> pFL44L plasmid cut BamHI SalI generate pPA295-1 plasmid.              COQ2 gene coding N-terminal para-hydroxy- <br /> The primers procedure used clone erg19 mutated <br /> allele temperature-sensitive MN19-34 strain (giving pPA295ts-1           benzoate polyprenyltransferase (3). The second ORF com- <br /> construct).                                                                           plete corresponds MVA-PP decarboxylase coding <br />    Each tagged gene prepared 2.5-kb BamHI-SalI restriction fragment          sequence reported course study <br /> and cloned pRS316 single-copy plasmid giving pPA295-2 (wild-type         (MVD1 gene) (26). <br /> ERG19) pPA295ts-2 (erg19 allele temperature-sensitive strain) <br /> constructs.                                                                              Therefore, confirmed gene able complement <br />    Immunoprecipitation tagged protein. Immunoprecipitation pro-         MN19-34 mutant corresponds structural gene coding <br /> tein A fusion proteins described Berge`s et al. (5)          MVA-PP decarboxylase. <br /> modifications. Cells grown saturation selective medium (100 ml),              The ERG19 gene essential viability. In order inac- <br /> washed potassium phosphate buffer (0.1 M, pH 7.4), eventually resus- <br /> pended potassium phosphate buffer (0.1 M, pH 7.4) containing 0.5 mM EDTA <br />                                                                                       tivate ERG19 gene, replaced coding se- <br /> and 10 mM dithiothreitol. The cells broken glass beads, debris      quence TRP1 gene having generated convenient <br /> were eliminated short centrifugation 3,000 3 g. The homogenate       restriction sites PCR. In construct, deletion includes <br /> centrifuged 30 min 12,000 3 g 4&#176;C, protein         ATG start codon ERG19 37 codons of <br /> crude extract (in micrograms milliliter) estimated performing the <br /> following calculation: OD228.5 2 OD234.5 3 0.317 3 dilution factor (TOT frac- <br />                                                                                       C-terminal remain coding sequence. The <br /> tion), OD228.5 optical density 228.5 nm.                              deleted erg19::TRP1 gene inserted homologous <br />    To 700 ml potassium phosphate buffer (0.1 M, pH 7.4) 4 mg protein            recombination 5302 (trp1/trp1 ura3/ura3) diploid strain <br /> A-Sepharose (Sigma) 10 mg rabbit IgG (whole molecule; Sigma)              (19). Tetrad dissection resulted 2:2 segregation the <br /> added, mixture incubated 30 min. Unbound IgG eliminated <br /> by washing potassium phosphate buffer. To pellet 700 ml potassium <br />                                                                                       viability (Fig. 1), viable spores auxotrophs for <br /> phosphate buffer 300 500 mg protein crude extract added,        tryptophan (data shown). Moreover, pTB4 plasmid was <br /> and mixture incubated 4&#176;C 1.5 h. After short centrifugation          transformed ERG19/erg19::TRP1 trp1/trp1 ura3/ura3 <br />  4666           ` S ET AL. <br />            BERGE                                                                                                                                J. BACTERIOL. <br />  <br />  <br />                                                                                    fusion protein efficiently detected Western <br />                                                                                    blot PAP antibodies (peroxidase-antiperoxidase rabbit <br />                                                                                    IgG). The observed molecular mass fusion protein <br />                                                                                    (;60,000 Da) consistent predicted molecular mass <br />                                                                                    chimeric ProtA-ERG19 protein deduced se- <br />                                                                                    quence gene (44,000 1 15,000 Da) (Fig. 2). <br />                                                                                       The pPA295-1 construct transformed diploid <br />                                                                                    strain copy ERG19 gene disrupted <br />                                                                                    (TB1D strain; ura3/ura3 trp1/trp1 ERG19/erg19::TRP1). The <br />                                                                                    haploid Ura1 Trp1 progeny selected sporulation; it <br />                                                                                    carried erg19::TRP1 allele plasmid pPA295-1. <br />                                                                                       The chimeric gene expressing ProtA-ERG19 fusion pro- <br />                                                                                    tein cloned single-copy vector pRS316, giving <br />                                                                                    plasmid pPA295-2. This construct transformed the <br />                                                                                    MN19-34 temperature-sensitive strain. The transformed strain <br />                                                                                    grew normally 36&#176;C, showing fusion protein pro- <br />                                                                                    duced single-copy plasmid fully complements mu- <br />                                                                                    tation (data shown). The construct transformed <br />                                                                                    TB1D (ERG19/erg19::TRP1) diploid strain obtain a <br />                                                                                    strain MVA-PP decarboxylase tagged <br />                                                                                    protein. After sporulation, viable Ura1 Trp1 progeny be <br />                                                                                    obtained, confirming fusion protein functional <br />                                                                                    (Fig. 4B, lane 2). A Western blot analysis showed level <br />                                                                                    production fusion protein single-copy vector <br />                                                                                    severely reduced (data shown). <br />    FIG. 1. Subcloning steps locate ERG19 gene (A) perform gene <br /> disruption (B). The pDG295 plasmid represents original genomic clone <br /> isolated library. The various restriction fragments subcloned into <br /> high-copy-number plasmid pFL44S (see Materials Methods), con- <br /> struct assayed complementation MN19-34 strain. Tetrad dissection <br /> was carried YPD. Each viable spore shown prototrophic for <br /> tryptophan. In panel B, arrows indicate transcription orientation. WT, <br /> wild type. <br />  <br />  <br />  <br />  <br /> diploid strain (TB1D), sporulation induced, hap- <br /> loid progeny analyzed. Haploid Trp1 Ura1 strains could <br /> be obtained did survive 5-<span id='am-1' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-2' property="rdfs:label" content="Foa" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>FOA</span>-containing medium <br /> (data shown), confirming ERG19 gene not <br /> dispensable viability yeast. <br />    Epitope tagging protein encoded ERG19. In order <br /> to immunoprecipitate protein encoded ERG19 gene <br /> and assay MVA-PP decarboxylase activity vitro, we <br /> chose tag protein IgG binding domain pro- <br /> tein A S. aureus. This strategy used <br /> several times allow immunoprecipitation affinity <br /> purification immunolocalization protein or <br /> to compensate lack specific antibodies directed <br /> against protein (15, 25, 29). The epitope domain have <br /> used consists repetitive IgG binding domains (Z-Z) <br /> of 15,000 Da fused N terminus the <br /> amino acid sequence deduced ERG19 gene. In our <br /> construct, transcription chimeric gene driven the <br /> yeast NOP1 promoter. NOP1 housekeeping gene encoding <br /> the yeast nucleolar protein fibrillarin, abundant protein <br /> involved ribosome biogenesis highly conserved through- <br /> out evolution. Although promoter strong                       FIG. 2. Expression ERG19 tagged IgG binding domains protein A <br /> well-known promoters like PGK ADH1 promoters,                         yeast. (A) Schematic drawing ProtA-ERG19 construct. Two IgG bind- <br /> recognized constitutive relatively strong promoter.                       ing domains (Z) fused N terminus ERG19 coding sequence <br />                                                                                    cloned pFL44S plasmid (see Materials Methods). prom, pro- <br /> The chimeric gene inserted high-copy-number plas-                       moter. (B) The pPA295-1 construct fusion protein (3) transformed <br /> mid (pFL44L). The resulting plasmid, pPA295-1, trans-                          MN19-34 strain deficient MVA-PP decarboxylase, growth was <br /> formed MN19-34 strain, replica plating                         assayed YPD 28&#176;C nonpermissive temperature (36&#176;C). Controls <br /> Ura1 transformants nonpermissive temperature showed                           untransformed MN19-34 strain (1) MN19-34 strain transformed <br />                                                                                    untagged Erg19p protein pTB4 (2). (C) Immunoblot analysis. A <br /> that fusion protein fully complements temperature-                         crude protein extract prepared MN19-34 strain transformed with <br /> sensitive phenotype (Fig. 2).                                                      pPA295-1 construct analyzed Western blotting. The fusion pro- <br />    A crude protein extract prepared strain,                      tein (arrow) detected PAP IgG. <br />  VOL. 179, 1997                                                      EFFECTS OF MVA-PP DECARBOXYLASE GENE ON VIABILITY                                    4667 <br />  <br />  <br />                                                                                    TABLE 1. Conversion radiolabeled MVA nonsaponifiable <br />                                                                                                  lipids crude protein extracts <br />                                                                                                     Amt                  % Reaction productb <br />                                                                                         Strain      MVAa <br />                                                                                                     (nmol)   Squalene   Squalene oxide    Lanosterol   Baseline <br />  <br />                                                                                   FL100              0.83       93             2             2            3 <br />                                                                                   erg19::TRP1c       0.42       74             2             5           18 <br />                                                                                   MN19-34            0.07       67             9            NDd          17 <br />                                                                                   MN19-34 1 IPe      0.52       85             5             4            6 <br />                                                                                     a <br />                                                                                       Amount [2-14C]MVA converted nonsaponifiable lipids. <br />                                                                                     b <br />                                                                                       Percent total nonsaponifiable lipids loaded. Separation TLC, and <br />                                                                                   radioactivity measurements performed Berthold 2382 TLC analyzer. <br />                                                                                     c <br />                                                                                       Strain carrying gene erg19::TRP1 (pPA295-1; multicopy, wild-type strain). <br />                                                                                     d <br />                                                                                       ND, detected. <br />                                                                                     e <br />                                                                                       IP, immunoprecipitated protein. <br />  <br />  <br />  <br />    FIG. 3. Immunoprecipitation tagged Erg19p. A crude protein extract      mid (pTB4), strain transformed the <br /> prepared haploid strain carrying erg19::TRP1 disrupted allele trans- <br /> formed pPA295-1 plasmid expressing ProtA-Erg19p fusion protein. <br />                                                                                   pPA295-2 plasmid (tagged ERG19, monocopy). The sterols of <br /> The proteins (500 mg) incubated IgG coupled ProtA-Sepharose      above-mentioned strains analyzed UV absorption <br /> 1 h 4&#176;C. After precipitation, samples corresponding supernatant (SUP    gas chromatography (GC) (see Materials Methods). <br /> fraction) (2) pellet (3) loaded sodium dodecyl sulfate-      The results obtained independent experiments are <br /> polyacrylamide gel electrophoresis gel analyzed Coomassie staining (A) <br /> and Western blotting (B) using PAP antibodies detect fusion protein <br />                                                                                   shown Table 2. <br /> (arrow). The control lane (TOT fraction) (1) corresponds sample          While mean value percentage dry weight the <br /> crude extract immunoprecipitation. Approximately        D5,7 sterols (sum ergosterol ergosta-5,7-dienol) the <br /> protein loaded lane. Standards kilodaltons.                    wild-type strain 0.80%, MN19-34 strain produced <br />                                                                                   significantly smaller sterols (0.25%). The GC <br />                                                                                   analysis revealed mutant strain, ergosterol is <br />    A whole-cell extract (centrifuged 12,000 3 g) pre-                      significantly detected. This contrast wild-type strain, <br /> pared nondenaturing conditions strain over-                        sterol represents 56%. Therefore, it <br /> producing ProtA-ERG19 fusion protein (pPA295-1 plas-                          appears MN19-34 mutant strain produces 6-fold- <br /> mid). Nonspecific rabbit IgGs preincubated protein                           smaller sterol corresponding wild-type iso- <br /> A-Sepharose added protein extract incubated                       genic strain. It noted conditions the <br /> for 90 min 4&#176;C. After sedimentation resin, pro-                     MN19-34 strain sterol auxotrophic 28&#176;C. <br /> teins extract, supernatant,                           The tagged version ERG19 control NOP1 <br /> immunoprecipitation pellet analyzed Coomassie                        promoter, carried single-copy vector (pPA295-2), led a <br /> staining Western blot analysis using PAP antibodies.                       wild-type-like ergosterol production level 0.86% the <br /> The result experiment shows 90%                      strain carrying erg19::TRP1. In contrast, wild-type <br /> fusion protein crude protein extract retained                     ERG19 gene carried multicopy vector (pTB4) the <br /> the pellet (Fig. 3). Moreover, fusion protein signifi-                    strain, ergosterol level remains low (0.38%, i.e., about <br /> cantly abundant pellet visible Coomassie                   half wild type). The GC analysis showed that <br /> staining, heavy chains IgGs,                    sterol biosynthesis intermediates lower levels <br /> only protein detected immunoprecipitated pellet.                           associated wild-type strain. <br />    To check enzyme prepared condi-                           This result suggests high level MVA-PP decar- <br /> tions active, showed crude protein extract                    boxylase gene expression basis small <br /> (centrifuged 12,000 3 g) prepared MN19-34 mu-                         ergosterol produced strain carrying erg19::TRP1 <br /> tant incubated radiolabeled MVA produced                            (pTB4). <br /> small amounts sterols (1.6% MVA converted) compared                            Cloning erg19 allele temperature-sensitive <br /> those produced extract wild-type strain (17%                         strain. A PCR product corresponding erg19 allele coding <br /> MVA converted) produced extract strain                      sequence obtained MN19-34 temperature-sensi- <br /> carrying erg19::TRP1 complemented pPA295-2 (wild-type                        tive strain using oligonucleotides designed to <br /> ERG19; monocopy) (9% MVA converted), confirming                            allow epitope tagging allele. The chimeric ProtA- <br /> leaky phenotype mutant (Table 1). When immuno-                        erg19ts allele construction high-copy-number <br /> precipitated fusion protein added MVA-PP                           plasmid pFL44L single-copy plasmid pRS316, generat- <br /> decarboxylase-deficient mixture, significant sterols                  ing, respectively, pPA295ts-1 (multicopy) pPA295ts-2 <br /> was produced (11% MVA converted), sep-                          (monocopy) plasmids. Two independent PCR products were <br /> arated TLC identified (Table 1). These results showed                      cloned parallel pUC18 sequencing. In addition, <br /> that MVA pathway restored addition                     erg19 coding sequence cloned expression vector <br /> immunoprecipitated MVA-PP decarboxylase.                                          pPA295ts-2 sequence determined. <br />    Overexpression ERG19 gene impairs MVA path-                            In cases, sequencing erg19 allele coding sequence <br /> way. Total sterols extracted control wild-type                        allowed single (T3C) point mutation position <br /> strain (FL100) grown complete medium yeast-peptone-dex-                        779 (from PstI site), resulting replacement leucine <br /> trose (YPD), MN19-34 strain grown permissive                        79 proline (L79P). <br /> temperature YPD addition exogenous ergosterol,                         The pPA295ts-1 pPA295ts-2 plasmids trans- <br /> from strain carrying disrupted erg19::TRP1 gene com-                      formed MN19-34 strain, Ura1 transformants <br /> plemented ERG19 gene high-copy-number plas-                         restreaked complete medium 36&#176;C. The transfor- <br />  4668           ` S ET AL. <br />            BERGE                                                                                                                                        J. BACTERIOL. <br />  <br />  <br />                                                TABLE 2. Comparative sterol compositions various strains <br />                                                                                                            % Sterol 6 SD <br />            Strain                         % 5,7a <br />                                                                          b <br />                                                               Zymosterol            Ergosterol           Ergost-8-enol           Ergosta-5,7-dienol          Lanosterol <br />  <br /> FL100                                  0.80 6 0.06             16 6 2                 56 6 6                10 6 5                    12 6 1                    664 <br /> MN19-34                                0.25 6 0.02               NDc                  99 6 1                 ND                        ND                       161 <br /> erg19::TRP1 (pTB4)d                    0.38 6 0.02            12.0 6 0.6              75 6 2                 ND                        862                      561 <br /> erg19::TRP1 (pPA295-2)e                0.86 6 0.09              22 6 3                58 6 3                 261                      12 6 2                    461 <br />    a <br />      % 5,7 percentage dry weight D5,7 sterols (sum dry weight percentages ergosterol ergosta-5,7-dienol). The equation % 5,7 given <br /> in Materials Methods. <br />    b <br />      Values determined GC. <br />    c <br />      ND, detected. <br />    d <br />      Multicopy. <br />    e <br />      Monocopy. <br />  <br />  <br />  <br /> mants obtained pPA295ts-1 grow 36&#176;C                               long 39-untranslated region (UTR) mRNA (about 600 <br /> same extent wild-type strain, transformants ob-                           nucleotides) reported Toth Huwyler (26). This result, <br /> tained single-copy plasmid pPA295-2 comple-                             obtained cDNA isolated authors, consis- <br /> ment temperature-sensitive phenotype (data shown).                             tent yeast genome sequencing data. Again, aver- <br /> We transformed plasmids TB1D strain. The <br /> Ura1 transformants sporulated, viable haploid Ura1 <br /> Trp1 strains recovered. The strain carrying erg19:: <br /> TRP1 gene produced wild-type ERG19 (pPA295-2; mono- <br /> copy) carrying erg19::TRP1 gene produced from <br /> mutated erg19 (pPA295ts-2; monocopy) restreaked at <br /> 36&#176;C (Fig. 4B). Only strain wild-type allele could <br /> grow 36&#176;C, showing erg19 allele mutant <br /> strain responsible temperature-sensitive phe- <br /> notype MN19-34 strain. <br />    A Western blot analysis confirmed fusion protein is <br /> expressed, apparent molecular mass identical of <br /> the protein produced wild-type ERG19 (Fig. 4C), <br /> although doubling time permissive temperature is <br /> slightly enhanced (by factor 1.2), suggesting en- <br /> zyme MN19-34 mutant, permissive temper- <br /> ature, slightly active wild-type counterpart. <br /> When proteins prepared 6-h growth 36&#176;C, no <br /> difference detected (data shown). We therefore <br /> concluded defect mutant strain a <br /> higher turnover messenger RNA ERG19 the <br /> protein point mutation affect directly the <br /> catalytic site interaction domain necessary ho- <br /> modimerization. It important point Leu-79 <br /> residue residues conserved yeast, <br /> human (26), <span id='am-6' about='obo:NCBITaxon_3701' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_980083 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-7' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45250" datatype="xsd:string"></span><span id='am-8'  ></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Arabidopsis" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Cardaminopsis" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Arabidopsis</span> thaliana MVA-PP decarboxylases <br /> (11a). <br />  <br />                                DISCUSSION <br />  <br />    This paper reports cloning functional complementa- <br /> tion yeast gene encoding MVA-PP decarboxylase. Evi- <br /> dence given mutant strain MN19-34 (10) bears <br /> a defect MVA-PP decarboxylase structural gene.                                     FIG. 4. The mutated erg19 allele corresponds point mutation leading to <br />    The genomic sequence ERG19 identical                           thermosensitivity. (A) Sequence comparison erg19 allele cloned PCR <br />                                                                                        temperature-sensitive MN19-34 strain wild-type allele showing <br /> cDNA, reported Toth Huwyler (26). As reported                             T3C point mutation changes leucine 79 proline. (B) Dilution <br /> these authors, ERG19 COQ2 (encoding para-hydroxyben-                               series showing growth strain carrying erg19::TRP1, contains <br /> zoate polyprenyl transferase) transcribed divergent                           monocopy plasmid pRS316 harboring wild-type ERG19 allele tagged with <br /> promoters, ORFs separated 250 bp. This                           protein A (pPA295-2 plasmid) (lane 2) harboring mutated erg19 allele <br />                                                                                        tagged protein A (pPA295ts-2 plasmid) (lane 3). Growth assayed 28 <br /> is significantly average length data                      36&#176;C YPD. The MN19-34 thermosensitive strain shown control <br /> yeast genome sequence indicate divergent ORFs                              (lane 1). (C) Immunoblot analysis. Crude protein extracts prepared from <br /> average 600 nucleotides apart (12). Yet possible                             strain carrying erg19::TRP1 gene produced wild-type ERG19 <br /> physiological significance peculiarity stud-                      (pPA295-2; monocopy) (lane 2) carrying erg19::TRP1 gene <br />                                                                                        produced erg19 mutated allele (pPA295ts-2; monocopy) (lane 3). The <br /> ied, tempting speculate sharing short                         fusion protein (arrow) detected PAP IgG. A Coomassie-stained gel run <br /> promoter area way coregulate related pathways.                           parallel shown control total protein loaded. Protein <br /> Another noticeable feature ERG19 gene unusually                          standards (lane 1) kilodaltons. <br />  VOL. 179, 1997                                          EFFECTS OF MVA-PP DECARBOXYLASE GENE ON VIABILITY                                           4669 <br />  <br />  <br /> age 39 UTRs yeast potential genes 160      decarboxylase indirectly, i.e., checking ability restore <br /> nucleotides (12). In expression <span id='am-63' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-64' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>vectors</span>,     functional MVA pathway leading sterols (squalene, squa- <br /> original genomic clone (pDG295, Fig. 1), half     lene oxide, lanosterol) radiolabeled MVA incu- <br /> region truncated (;150 nucleotides original 39 UTR        bated nonfunctional, MVA-PP decarboxyl- <br /> are left pPA295 constructs), effect            ase-deficient protein extract. Although obvious this <br /> complementation. The significance long 39 UTR        assay used enzyme biochemical parameter de- <br /> MVA-PP decarboxylase mRNA determined.                  termination, experiment showed tagged <br />    The results presented MVA-PP decarboxyl-          MVA-PP decarboxylase isolated active form <br /> ase gene disruption lethal yeast. This result expected,   immunoprecipitation procedure. Therefore, en- <br /> since ergosterol unique end product MVA           zyme preparation procedure allow researchers as- <br /> pathway yeast cells impermeable organic py-        say MVA-PP decarboxylase activity vitro various con- <br /> rophosphates supplemented IPP,         ditions enzyme prepared native conditions from <br /> the product MVA-PP decarboxylase. A similar result           yeast test activity mutant counterpart. <br /> been described MVA kinase (21), phosphomevalonate ki-           This study help design new inhibitors key <br /> nase (28), IPP isomerase (1), farnesyl diphosphate (FPP)        enzyme study structural functional rela- <br /> synthase (2, 6) gene disruptions, showing essential   tionships vitro-mutagenized ERG19. <br /> genes. However, study, exclude the <br /> presence second gene MVA-PP decarboxylase,                                      ACKNOWLEDGMENTS <br /> the case HMG-CoA reductase (4). Indeed, mu-               We thank Johann Joets helpful advice GC analysis and <br /> tant (MNerg19) defective MVA-PP decarboxylase           various members laboratory helpful discussions during <br /> described yeast. This mutant exhibits temperature-sensi-      course work. We grateful Randy Worobo Mat- <br /> tive phenotype, speculated phenotype      thieu Re&#180;gnacq critically reading manuscript. Ed Hurt (Univer- <br /> was consequence gene inactivation leaky     sity Heidelberg) acknowledged gift pRS316 plasmid. <br /> mutation.                                                              This work supported grants GREG/INRA (no. 9/95) <br />    Surprisingly, overexpression wild-type ERG19          French Ministry National Education Research. <br /> gene leads low sterol content yeast disrupted <br />                                                                                                        REFERENCES <br /> ERG19 gene. One way account result to <br />                                                                      1. Anderson, M. S., M. Muehlbacher, I. P. Street, J. Proffit, C. D. Poulter. <br /> argue MVA-PP decarboxylase limiting enzyme             1989. Isopentenyl diphosphate:dimethylallyl diphosphate isomerase. An im- <br /> sterol pathway yeast overexpression results           proved purification enzyme isolation gene Saccharo- <br /> accumulation IPP, geranyl diphosphate, FPP             myces cerevisiae. J. Biol. Chem. 264:19169&#8211;19175. <br /> feed MVA kinase and/or HMG-CoA reduc-                2. Anderson, M. S., J. G. Yarger, C. L. Burck, C. D. Poulter. 1989. Farnesyl <br />                                                                         diphosphate synthetase. Molecular cloning, sequence expression an <br /> tase. This strain able grow temper-          essential gene Saccharomyces cerevisiae. J. Biol. Chem. 264:19176&#8211; <br /> ature, colonies heterogeneous size, suggesting          19184. <br /> that growth impaired.                                     3. Ashby, M. N., S. Y. Kutsunai, S. Ackerman, A. Tzagoloff, P. A. Edwards. <br />    We mutated gene presents leucine-to-pro-             1992. COQ2 candidate structural gene encoding para-hydroxy- <br />                                                                         benzoate:polyprenyltransferase. J. Biol. Chem. 267:4128&#8211;413 <br /> </body></html>