<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal Vol. 20 No. 24 pp. 7209&#177;7219, 2001 <br />  <br />  <br /> RTG-dependent mitochondria nucleus signaling <br /> is negatively regulated seven WD-repeat <br /> protein Lst8p <br />  <br />  <br /> Zhengchang Liu, Takayuki Sekito1,                                      maintain supply key TCA cycle intermediates, e.g. <br /> Charles B.Epstein2 Ronald A.Butow3                                 oxaloacetate, severely <br />                                                                        limiting cells mitochondrial de&#174;ciencies. The <br /> Department Molecular Biology, University Texas Southwestern <br /> Medical Center, 5323 Harry Hines Blvd, Dallas, TX 75390-9148, USA      expression genes retrograde pathway, <br /> 1 <br />                                                                        CIT2 (encoding glyoxylate cycle isoform of <br />  Present address: Department Cell Biology, National Institute   citrate synthase), DLD3 (encoding D-lactate dehydro- <br /> Basic Biology, Nishigonaka 38, Myodaiji 444-585, Aichi, Japan <br /> 2 <br />  Present address: Aventis Pharmaceuticals, Inc., 26 Landsdowne         genase) PDH1 (a gene involved propionate <br /> Street, Cambridge, MA 02139, USA                                       metabolism) directly dependent regulatory <br /> 3                                                                      genes, RTG1, RTG2 RTG3 (Liao Butow, 1993; Jia <br />  Corresponding author <br /> e-mail: butow@swmed.edu                                                et al., 1997; Chelstowska et al., 1999). <br />                                                                           In cells respiratory de&#174;cient reduced <br /> In cells reduced mitochondrial function, RTG1, 2                  respiratory activity, RTG genes regulate the <br /> and 3 required expression genes involved                 expression CIT1, ACO1, IDH1 IDH2 (Liu and <br /> glutamate synthesis. Glutamate negatively regulates                    Butow, 1999), encode enzymes catalyzing &#174;rst <br /> RTG-dependent gene expression upstream Rtg2p,                       steps TCA cycle lead synthesis a- <br /> which, turn, acts upstream bHLH/Zip tran-                    ketoglutarate, precursor glutamate. In cells with <br /> scription factors, Rtg1p Rtg3p. Here report                     robust mitochondrial respiratory activity, expression of <br /> that mutations [lst8-(2&#177;5)] LST8, essential                 TCA cycle genes control HAP <br /> gene encoding seven WD40-repeat protein required                     transcription complex (Forsburg Guarente, 1989; <br /> for targeting amino acid permeases (AAPs)                    Gangloff et al., 1990; Rosenkrantz et al., 1994). Glu- <br /> plasma membrane, bypass requirement Rtg2p                      tamate homeostasis, central functions RTG- <br /> and abolish glutamate repression RTG-dependent                      dependent gene expression, exempli&#174;es retro- <br /> gene expression. The lst8-1 mutation, however,                   grade pathway serves adapt cells mitochondrial <br /> reduces plasma membrane expression AAP,                      dysfunction. The levels glutamate regulate retro- <br /> bypass Rtg2p requirement, suppresses                     grade response negative feedback loop, low <br /> glutamate repression RTG target gene expression.                    glutamate levels activate high glutamate levels repress <br /> We Lst8p negatively regulates RTG gene                       RTG-dependent gene expression (Liu Butow, 1999). <br /> function, acting sites, upstream Rtg2p,                  Indeed, cells compromised mitochondrial function, <br /> affecting glutamate repression RTG-dependent gene                   mutations RTG genes result glutamate <br /> expression Ssy1p, AAP-like sensor                        auxotrophy (Liao Butow, 1993; Jia et al., 1997). <br /> external amino acids, Rtg2p                         Although exact mechanism glutamate signaling is <br /> and Rtg1p&#177;Rtg3p. These <span id='am-66' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-70'  ></span>data</span>, genome-                     unclear, respiratory-competent cells cells grown in <br /> wide transcription pro&#174;ling, reveal pathways regu-                     presence high levels glutamate medium, the <br /> lated glutamate, provide insight                       basic <span id='am-33' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-34' property="rdfs:label" content="Helix" datatype="xsd:string"></span><span id='am-35'  ></span><span id='am-36' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>helix</span>&#177;loop&#177;helix/leucine zipper (bHLH/Zip) tran- <br /> regulation cellular responses mitochondrial                      scription factors, Rtg1p Rtg3p, present an <br /> dysfunction.                                                           inactive complex cytoplasm (Sekito et al., 2000). In <br /> Keywords: LST8/mitochondria/RTG genes/WD40 repeat/                     respiratory-de&#174;cient cells cells grown medium <br /> yeast                                                                  lacking glutamate, factors accumulate nucleus <br />                                                                        processes requiring Rtg2p, novel cytoplasmic protein <br />                                                                        N-terminal ATP-binding domain similar the <br />                                                                        hsp70/actin/sugar kinase <span id='am-43' about='obo:NCBITaxon_superfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-44' property="rdfs:label" content="superfamily" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>superfamily</span> ATP-binding <br /> Introduction                                                           proteins (Bork et al., 1992). Rtg2p acts proximal <br /> In budding yeast, <span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-10'  ></span><span id='am-11' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>, mito-                  sensor mitochondrial dysfunction, <span id='am-56' about='obo:NCBITaxon_173684' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_41937 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_4011 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-57'  ></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="Gluta" datatype="xsd:string"></span>gluta</span>- <br /> chondrial dysfunction, respiratory de&#174;ciency                mate levels, regulate subcellular localization of <br /> blocks tricarboxylic acid (TCA) cycle, results               Rtg1p Rtg3p (Sekito et al., 2000). <br /> increased expression subset nuclear genes involved                Recent studies shown RTG pathway also <br /> in metabolism, small molecule transport pathways,                      responds quality nitrogen source the <br /> peroxisomal biogenesis stress responses (Liao et al.,              medium target rapamycin (<span id='am-51' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-52' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-55'  ></span>TOR</span>) kinase <br /> 1991; Chelstowska Butow, 1995; Traven et al., 2001;                pathway (Komeili et al., 2000; Shamji et al., 2000). In <br /> Epstein et al., 2001b). <span id='am-38' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-41'  ></span><span id='am-42' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> pathway inter-organelle                particular, cells grown poor nitrogen source, <br /> communication, called retrograde regulation, accommo-                  TOR kinase pathway inhibited by <br /> dates cells mitochondrial defect. For example,                rapamycin, Rtg1p Rtg3p translocate the <br /> many reactions catalyzed target genes                     cytoplasm nucleus Rtg2p-dependent manner <br /> the retrograde pathway serve, directly indirectly,               activate target gene expression, similarly the <br />  <br /> &#227; European Molecular Biology Organization                                                                                     7209 <br />  Z.Liu et al. <br />  <br />  <br /> Rtg2p-dependent nuclear accumulation Rtg1p and <br /> Rtg3p cells dysfunctional mitochondria (Sekito <br /> et al., 2000). Together, studies pathways of <br /> carbohydrate nitrogen metabolism linked together <br /> via RTG-dependent gene expression. <br />    To identify additional regulatory genes RTG <br /> pathway, searched mutations bypass the <br /> requirement RTG2 CIT2 expression. This search led <br /> to identi&#174;cation LST8, essential gene encoding a <br /> seven WD40-repeat protein implicated in <br /> amino acid permease transport Golgi cell <br /> surface (Roberg et al., 1997a). We some, not <br /> all, mutations LST8 bypass requirement of <br /> Rtg2p Rtg1p/Rtg3p-dependent gene expression. We <br /> conclude Lst8p functions negative regulator       Fig. 1. RTB2/LST8 negative regulator CIT2 expression. (A) RTB2 <br /> RTG-dependent gene expression, acting          regulates CIT2 expression upstream RTG3, downstream of <br />                                                                RTG2. Wild-type (PSY142), rtg2D (PSY142-rtg2), rtg2D lst8-5 <br /> sites, upstream Rtg2p level external          (RBY426) rtg2D rtb2-5 rtg3D (RBY427) cells grown YPD <br /> glutamate sensing Rtg2p              medium mid-log phase collected b-galactosidase activity <br /> Rtg1p&#177;Rtg3p.                                                   analysis determine transcriptional activation integrated <br />                                                                CIT2-lacZ reporter gene. b-galactosidase assays whole-cell extracts <br />                                                                carried triplicate described Materials methods. <br /> Results                                                        (B) RTB2 LST8. Wild-type PSY142 cells rtg2D lst8-5 <br />                                                                transformed pRS416-LST8, lst8-5, rtg2D lst8-1 lst8-1 <br /> Isolation RTG2 bypass mutants                               derivatives grown YNBcasD medium plus uracil (omitted in <br /> To identify new genes retrograde pathway,            cells transformed pRS416-LST8) mid-log phase, CIT2-lacZ <br /> carried genetic screen identify mutants    reporter gene expression determined b-galactosidase activities. <br /> bypass requirement Rtg2p CIT2 expression. <br /> Using rtg2D strain CIT2-lacZ reporter gene           suf&#174;cient abolish reporter gene expression the <br /> integrated URA3 locus, cells mutagenized       rtg2D rtb2 isolates, shown rtg2D rtb2-5 cells <br /> screened colonies appeared blue X-gal plates.      (Figure 1B). To verify RTB2 LST8 same, <br /> This screen yielded 14 candidates selected        carried complementation tests examining CIT2- <br /> potential rtg2D bypass mutants, designated rtb (Rtg        lacZ expression diploids obtained cross between <br /> bypass). Northern blot analysis revealed rtb   rtg2D rtb2-5 strain rtg2D lst8D::LEU2 strain <br /> isolates high levels expression             carrying wild-type copy LST8 centromeric <br /> endogenous CIT2 mRNA (data shown), indicating              plasmid. The resulting diploids cured plasmid <br /> that rtg2D bypass phenotype peculiar        tested CIT2-lacZ expression. We CIT2- <br /> reporter gene. Standard genetic tests showed rtg2D    lacZ expression high diploids (data not <br /> bypass phenotype isolate recessive      shown), indicating RTB2 LST8. Finally, as <br /> a single gene mutation. Further, isolates         described section, identi&#174;ed point mutations <br /> placed complementation groups, rtb1 rtb2.         LST8 coding region rtb2 mutant <br /> Five independent rtb2 mutants obtained,           alleles identi&#174;ed screen. Hereafter, refer to <br /> properties described here; details      RTB2 LST8. When lst8 mutant alleles were <br /> the rtb1 complementation group described else-         examined singly, shown lst8-5 (Figure 1B), <br /> where.                                                         CIT2-lacZ expression nearly 10-fold higher its <br />    Preliminary characterization rtb2 isolates    expression wild-type cells. Altogether, data show <br /> (rtb2-5) showed block CIT2-lacZ expression         LST8 negative regulator RTG gene function. <br /> due rtg2D mutation completely reversed the <br /> rtb2-5 mutation (Figure 1A). Similar results obtained <br /> for rtb2 mutants. To exclude possibility    The lst8-1 mutant allele does bypass rtg2D <br /> the rtb2 mutations activated alternate pathway CIT2     mutation <br /> expression, introduced rtg3 null mutation rtg2D     LST8 shown previously encode essential seven <br /> rtb2-5 mutant cells. CIT2-lacZ expression abolished        WD-repeat protein functions delivery Gap1p, <br /> in triple mutant (Figure 1A), suggesting rtg2D    possibly amino acid permeases, cell <br /> bypass does involve recruitment alternative         surface (Roberg et al., 1997a). In study, mutant <br /> transcriptional activators.                                    allele LST8 (lst8-1) identi&#174;ed synthetic lethal <br />                                                                sec13-1 mutation. To determine the <br /> RTB2 LST8                                                   lst8-1 mutation similar effects RTG-dependent <br /> To identify RTB2 gene, rtg2D rtb2-5 mutant cells      gene activity, lst8-1 allele transplaced the <br /> transformed centromere-based wild-type yeast            LST8 locus wild-type rtg2D cells resultant <br /> genomic library screened transformants            transformants assayed CIT2-lacZ expression <br /> restored CIT2-lacZ expression low level observed        (Figure 1B). Surprisingly, unlike lst8 mutants we <br /> in wild-type rtg2D cells. Complementing              obtained, lst8-1 allele unable bypass rtg2D <br /> <span id='am-71' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-72' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span> obtained, subsequent subcloning              mutation, itself resulted ~4-fold <br /> yielded single gene, LST8, necessary           activation CIT2-lacZ expression. These &#174;ndings raise <br />  <br /> 7210 <br />                                                                                                 Lst8p negatively regulates RTG gene function <br />  <br />  <br />  <br />  <br />                                                                              Fig. 3. The glutamate auxotrophy rtg2D mutation rescued by <br />                                                                              lst8-5 lst8-1 mutation. Wild-type PSY142 cells, and <br />                                                                              rtg2D, lst8-5, rtg2D lst8-5, rtg2D rtb2-5 rtg3D, lst8-1 rtg2D lst8-1 <br />                                                                              derivatives streaked YNBD medium supplemented with <br />                                                                              0.02% glutamate indicated, uracil, leucine lysine, and <br />                                                                              incubated 30&#176;C 2&#177;3 days. <br />  <br />                                                                              conserved seven WD repeats Lst8p, as <br />                                                                              WD repeats G protein b-subunit (Sondek et al., <br />                                                                              1996). We designated different lst8 mutant alleles <br />                                                                              obtained study lst8-2 lst8-5, collect- <br />                                                                              ively lst8-(2&#177;5). Structural modeling Lst8p suggests <br />                                                                              lst8-(2&#177;5) mutations clustered localized on <br />                                                                              surface protein, lst8-1 an <br />                                                                              internal position opposite protein <br />                                                                              (Figure 2B). <br /> Fig. 2. Analysis lst8 mutations. (A) Alignment seven WD            lst8-5, lst8-1, rescues glutamate <br /> repeats Lst8p. Five conserved residues seven WD <br /> repeats highlighted gray, consensus sequence formed        auxotrophy rtg2D mutant <br /> shown alignment; j designates hydrophobic residue          A hallmark rtg mutant cells glutamate <br /> y designates hydrophilic residue; T designates serine threonine.      auxotrophs mitochondrial function reduced <br /> Based alignment Lst8p G protein b1 subunit            compromised (Liao Butow, 1993; Liu Butow, <br /> (ter Haar et al., 1998), WD repeat likely consist four <br /> b-strands, indicated arrows A, B, C D. The row gives a <br />                                                                              1999). Given striking difference ability the <br /> canonical numbering residues WD repeat. The red              lst8-(2&#177;5) mutant alleles versus lst8-1 allele to <br /> type indicates mutations identi&#174;ed screen: lst8-2, Gly146          suppress block CIT2 expression rtg2D cells, we <br /> glutamate; lst8-3, Gly138 aspartate; lst8-4 lst8-6, Gly181         asked difference lst8 <br /> aspartate; lst8-5, Gly171 aspartate. The yellow type indicates        mutants ability reverse glutamate auxo- <br /> lst8-1 mutation identi&#174;ed synthetic lethal sec13 (Roberg et al., <br /> 1997a). (B) Three-dimensional model Lst8p. The model created          trophy rtg2D strain grown glucose medium. As <br /> by Swiss-Model (Peitsch, 1996), automated comparative protein             shown lst8-5 (Figure 3), lst8-(2&#177;5) mutant <br /> modeling server, based homology Lst8p G protein           alleles restored glutamate prototrophy rtg2D cells, <br /> b1 subunit. The image prepared using programs Gl_render (http://         depended presence RTG3. In contrast, <br /> www.hhmi.swmed.edu/external/Doc/G1_render.html), BOBSCRIPT <br /> (Esnouf, 1997), MOLSCRIPT (Kraulis, 1991) Raster3D (Meritt, <br />                                                                              lst8-1 mutation unable restore glutamate <br /> 1997). The positions b-strands A&#177;D WD repeat 6              prototrophy rtg2D cells, consistent inability <br /> indicated. N C designate N- C-terminal ends Lst8p.            lst8 allele bypass loss CIT2 expression in <br /> Locations lst8-(2&#177;5) mutations indicated red dots,     rtg2D cells. In cases CIT2 reporter gene <br /> the lst8-1 mutation yellow dot.                                       expression high, Rtg3p localized nucleus <br />                                                                              (data shown). These &#174;ndings underscore the <br />                                                                              difference lst8-(2&#177;5) lst8-1 mutants, but <br /> the possibility Lst8p separable functions, notion                 suggest site negative regulation RTG-depend- <br /> that developed later report.                                 ent gene expression Lst8p Rtg2p the <br />    Sequencing &#174;ve lst8 mutants isolated showed                     Rtg1p&#177;Rtg3p transcription complex. <br /> that contained single missense mutation <span id='am-1' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Glycine" datatype="xsd:string"></span>glycine</span> <br /> residue, WD repeat 4 remainder WD                          Both lst8-5 lst8-1 mutant cells largely <br /> repeat 5 (Figure 2A). Two mutants contained                       insensitive glutamate repression <br /> identical mutation position 181 WD repeat 5. Based                     To investigate regulation RTG-dependent <br /> on design rtg2D bypass screen (see Materials                      gene expression Lst8p, examined effects the <br /> and methods), mutants likely arisen                       addition glutamate growth medium CIT2-lacZ <br /> independently. All mutated glycine residues                      expression LST8, lst8-5 lst8-1 cells. In LST8 cells <br /> localized tight turns protein. WD repeat 5                  grown minimal medium, CIT2-lacZ expression was <br /> most divergent seven, lacking conserved                        repressed &gt;100-fold addition 0.2% glutamate to <br /> His2, Ser/Thr20 Asp24, containing four-residue                     medium (Figure 4A). In lst8-5 mutant cells, however, <br /> insertion linker region leading WD repeat 6. We                    CIT2-lacZ expression largely insensitive glutamate <br /> also determined mutation lst8-1 allele identi&#174;ed                  repression, decreasing ~2-fold addition of <br /> by Roberg et al. (1997a),                         0.2% glutamate. This loss sensitivity glutamate <br /> leucine serine change position 300, residue                          repression result bulk impairment of <br />  <br />                                                                                                                                               7211 <br />  Z.Liu et al. <br />  <br />  <br />                                                                           glutamate uptake, raise possibility glutamate <br />                                                                           repression involve external glutamate sensing. <br />  <br />                                                                           An Ssy1p&#177;Ptr3p signal transduction cascade <br />                                                                           mediates glutamate repression CIT2 expression <br />                                                                           One candidate external glutamate sensor Ssy1p, a <br />                                                                           plasma membrane protein predicted structure <br />                                                                           similar amino acid permeases (Didion et al., <br />                                                                           1998; Iraqui et al., 1999; Klasson et al., 1999). Previous <br />                                                                           studies shown amino acid induction AGP1, <br />                                                                           encodes broad-speci&#174;city amino acid permease, as <br />                                                                           genes encoding amino acid <br />                                                                           permeases, abolished ssy1 mutant cells (Didion <br />                                                                           et al., 1998; Iraqui et al., 1999). These amino acid <br />                                                                           inductive effects shown require Ptr3p, a <br />                                                                           peripheral plasma membrane protein, proposed <br />                                                                           Ptr3p interacts Ssy1p signal <br />                                                                           transduction cascade external amino acid sensing. In <br />                                                                           addition, induction probably requires Shr3p, protein <br />                                                                           function facilitate exit Ssy1p the <br />                                                                           endoplasmic reticulum (Klasson et al., 1999) and, hence, <br />                                                                           targeting plasma membrane. <br /> Fig. 4. The insensitivity glutamate repression CIT2-lacZ reporter      To investigate Ssy1p mediate glutamate <br /> gene expression lst8-5 mutant cells result defect   repression RTG target gene expression, &#174;rst <br /> glutamate uptake. (A) Lst8-5 lst8-1 mutants largely insensitive <br /> to glutamate repression CIT2-lacZ reporter gene expression. Wild- <br />                                                                           determined glutamate repress the <br /> type (PSY142), lst8-1 lst8-5 mutant strains integrated copy   activity CIT2-lacZ reporter gene ssy1D, ptr3D and <br /> of CIT2-lacZ reporter gene grown YNB5%D medium            shr3D mutant cells. Figure 5A shows strong <br /> without 0.2% glutamate. Whole-cell extracts prepared             repression CIT2-lacZ activity glutamate wild-type <br /> b-galactosidase activities determined described Materials      cells essentially blocked mutant strains. <br /> and methods. (B) Glutamate uptake wild-type lst8-5 mutant <br /> cells. Wild-type (PSY142) lst8-5 mutant strains grown         The lack signi&#174;cant glutamate repression CIT2-lacZ <br /> YNBD medium glutamate uptake assayed described              expression ssy1D, ptr3D shr3D mutant cells is <br /> Materials methods. Glutamate present &#174;nal concentration      similar lst8 mutants. However, unlike the <br /> of 0.01%. (C) Glutamate uptake wild-type gap1D dip5D             lst8-(2&#177;5) mutations, loss CIT2 reporter gene <br /> mutant strains. Wild-type (S288C) gap1D dip5D derivative strain <br /> were grown YNBD medium, glutamate uptake assay was <br />                                                                           expression rtg2D cells reversed ssy1D, <br /> carried described (B). (D) CIT2-lacZ reporter gene expression   ptr3D shr3D mutations (Figure 5A), double <br /> is repressed glutamate gap1D dip5D mutant cells. Wild-type    mutants rtg2D ssy1D, rtg2D ptr3D rtg2D shr3D <br /> (S288C) gap1D dip5D mutant derivative strain grown          remained glutamate auxotrophs (Figure 5B). These data <br /> YNB5%D medium glutamate indicated. Whole-cell          suggest that, like lst8-1 mutation, ssy1D, ptr3D and <br /> extracts prepared b-galactosidase activities determined <br /> as described Materials methods.                                    shr3D mutations affect RTG gene function upstream of <br />                                                                           RTG2. <br />                                                                              We measured rate glutamate uptake ssy1D <br />                                                                           cells, glutamate uptake inhibited by <br />                                                                           ~50% (Figure 6A). These data agreement with <br /> glutamate uptake, lst8-5 mutant cells showed                  previous report showing modest decrease glutamate <br /> same kinetics glutamate uptake wild-type cells                      uptake ssy1 mutant cells (Klasson et al., 1999). To <br /> (Figure 4B). Interestingly, lst8-1 mutation did              provide additional evidence glutamate repression of <br /> not bypass requirement Rtg2p, greatly reduced                  CIT2-lacZ expression function glutamate <br /> the glutamate repression CIT2-lacZ expression.                         uptake, overexpressed Dip5p Gap1p ssy1D cells <br />   We asked glutamate repress                     determined rate glutamate uptake cells. <br /> CIT2-lacZ reporter gene expression mutant cells               Overexpression amino acid permeases ssy1D <br /> defective glutamate transport. In yeast,                 cells restored rates glutamate uptake ~80% the <br /> major pathways glutamate uptake, involving                    wild-type rate (Figure 6A), CIT2-lacZ expression <br /> general amino acid permease encoded GAP1                       remained insensitive glutamate repression (Figure 6B). <br /> other speci&#174;c glutamate/aspartate permease encoded                   Although level glutamate repression CIT2-lacZ <br /> the DIP5 gene (Regenberg et al., 1998). Accordingly,                   expression wild-type cells overexpressing Dip5p and <br /> examined effects inactivation genes                    Gap1p somewhat observed wild-type <br /> glutamate repression CIT2-lacZ expression. In gap1D                  control cells, possibly high level expression <br /> dip5D double mutant, rate glutamate uptake                     plasma membrane proteins competes Ssy1p <br /> inhibited ~85% (Figure 4C). Nevertheless, CIT2-lacZ                    targeting, clear ssy1D cells overexpressing <br /> expression double mutant cells sensitive               amino acid permeases, glutamate completely fails to <br /> glutamate repression wild-type cells (Figure 4D).                   repress CIT2-lacZ reporter gene expression. Altogether, <br /> These data suggest glutamate repression RTG-                      data support view sensing the <br /> dependent gene expression strictly dependent                    Ssy1p&#177;Ptr3p signal transduction pathway, than <br />  <br /> 7212 <br />                                                                                            Lst8p negatively regulates RTG gene function <br />  <br />  <br />  <br />  <br /> Fig. 5. Effects shr3D, ssy1D ptr3D mutations glutamate <br /> repression, rtg2D suppression glutamate auxotrophy. (A) CIT2-lacZ <br /> expression. Transformants wild-type (PLY126), shr3D (PLY151- <br /> ura3), ssy1D (HKY20), ptr3D (HKY31), rtg2D (PLY126-rtg2), shr3D <br /> rtg2D (PLY151-rtg2), ssy1D rtg2D (HKY20-rtg2) ptr3D rtg2D <br /> (HKY31-rtg2) strains containing CIT2-lacZ reporter gene the <br /> centromere-based plasmid pCIT2-lacZ grown mid-log phase in <br /> the media indicated &#174;gure, b-galactosidase activities    Fig. 6. The loss glutamate repression CIT2-lacZ reporter gene <br /> determined extracts described Materials methods. Strains    expression ssy1D cells uncoupled glutamate uptake. <br /> rtg2D (PLY126-rtg2), shr3D rtg2D (PLY151-rtg2), ssy1D rtg2D              (A) Glutamate uptake ssy1D cells overexpressing Dip5 Gap1. <br /> (HKY20-rtg2) ptr3D rtg2D (HKY31-rtg2) tested            ssy1D (HKY20) cells transformed 2m control plasmid <br /> CIT2-lacZ expression grown YNB5%D medium            (empty vector), 2m-TEF-DIP5 2m-GAP1 plasmids, resultant <br /> are glutamate auxotrophs. (B) The shr3D, ssy1D ptr3D mutations       transformants, including wild-type cells transformed 2m <br /> cannot rescue glutamate auxotrophy rtg2D cells. Cells streaked   control plasmid, analyzed glutamate uptake described in <br /> on YNBD medium 0.02% glutamate incubated          Figure 4B. (B) Glutamate repression CIT2-lacZ reporter gene <br /> 30&#176;C 2&#177;3 days.                                                       expression ssy1D cells overexpressing Dip5p Gap1p. The <br />                                                                          various transformants described (A) transformed with <br />                                                                          pCIT2-lacZ reporter gene construct transformants were <br />                                                                          grown YNBD medium 0.2% glutamate. Whole-cell <br /> glutamate uptake, major factor responsible                    extracts prepared b-galactosidase activities determined <br /> glutamate repression RTG-dependent gene expression.                   described Materials methods. <br />    We determined amino acid speci&#174;city for <br /> repression CIT2 reporter gene expression wild-type <br />                                                                          addition glutamate medium. A number studies <br /> cells. CIT2-lacZ expression repressed strongly in <br /> cells grown presence glutamate glutamine,                   shown different quality nitrogen sources give <br /> whereas expression reduced &lt;3-fold                        rise different patterns gene expression (Cardenas <br /> presence amino acids (Figure 7). In ssy1D                    et al., 1999; Hardwick et al., 1999; Komeili et al., 2000). <br /> mutant, repression CIT2-lacZ expression glutamate                  Assessment quality different nitrogen sources has <br /> and glutamine eliminated and, instances,                 complicated fact variation among <br /> modest repression amino acids                      strains response utilization different <br /> eliminated. These data suggest speci&#174;c non-                     nitrogen sources, including <span id='am-46' about='obo:NCBITaxon_29189' typeof='obo:NCBITaxon_29178 owl:Thing obo:NCBITaxon_203397 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_69034 obo:NCBITaxon_29185 obo:NCBITaxon_543769 obo:NCBITaxon_2759'><span id='am-47' property="rdfs:label" content="Ammonia" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-50'  ></span>ammonia</span> glutamate. <br /> speci&#174;c repression RTG target gene expression                      Accordingly, experiments <br /> effected Ssy1p.                                                  described effects glutamate RTG- <br />                                                                          dependent gene expression, glutamate added to <br />                                                                          medium containing ammonia, avoid <br /> Genome-wide transcription analysis genes                              comparing different, sole nitrogen sources. <br /> regulated glutamate                                                      Strain S288C grown mid-log phase YNB5%D <br /> Because glutamate act repressor                     medium supplementation 0.2% <br /> activator gene expression (Iraqui et al., 1999),            glutamate. mRNA isolated, labeled cDNAs <br /> effects operating Ssy1p&#177;Ptr3p signal transduc-               prepared reverse transcription presence of <br /> tion pathway, wished obtain global view                  Cy3- Cy5-labeled nucleotides. These labeled cDNAs <br /> glutamate-regulated genes. For purpose, carried                  mixed hybridized duplicate dye <br /> out microarray analysis ~6200 yeast genes determine                con&#174;gurations yeast whole-genome arrays. Applying a <br /> how relative mRNA abundance affected                     stringent cut-off average 2.5-fold difference for <br />  <br />                                                                                                                                         7213 <br />  Z.Liu et al. <br />  <br />  <br /> individual genes, 45 genes expression <br /> was affected glutamate (Table I); these, 31 genes <br /> were repressed 14 induced. Among repressed <br /> genes, 12 known subject nitrogen catabolite <br /> repression (NCR), including MEP2, encoding ammonia <br /> permease, GAP1, encoding general amino acid <br /> permease. The repression NCR genes by <br /> glutamate suggests ammonia preferred <br /> nitrogen source strain S288C, repression <br /> of NCR genes glutamate likely to the <br /> combined effect glutamate ammonia generate <br /> glutamine, preferred nitrogen source. As <br /> expected, RTG target genes, ACO1, CIT2 and <br /> IDH2, glutamate-repressible genes. <br />    Among 14 genes induced glutamate, encode               Fig. 7. Effects different amino acids CIT2-lacZ reporter gene <br /> amino acid permeases, &#174;ve which, BAP2, BAP3, AGP1,            expression wild-type ssy1D cells. Wild-type (PLY126) and <br /> GNP1 VAP1/TAT1, belong Cluster I amino acid           ssy1D (HKY20) strains transformed centromere-based plasmid <br /> permease <span id='am-30' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> shows broad substrate speci&#174;city          pCIT2-lacZ grown mid-log phase YNB5%D medium <br /> (Regenberg et al., 1999). Induction genes            supplemented 0.1% indicated amino acids. Whole-cell <br />                                                                  extracts prepared b-galactosidase activities determined <br /> external amino acids Ssy1p dependent (Iraqui et al.,          described Materials methods. <br /> 1999), underscoring conclusion glutamate can <br /> effect induction repression gene expression. <br />  <br />                                                                  glutamate signaling, binding glutamate Ssy1p <br /> Discussion <br />                                                                  send signal (inhibitory) RTG pathway, as <br /> To identify additional components RTG pathway,            proposed effects amino acids in <br /> we sought mutants bypass dependence            Ssy1p/Ptr3p-dependent activation gene expression <br /> Rtg2p, require Rtg1p&#177;Rtg3p transcriptional         amino acid inductive pathway (Didion et al., 1998; <br /> activators target gene expression. The current work          Klasson et al., 1999; Iraqui et al., 1999). Despite loss <br /> shows certain mutations seven WD40-repeat            glutamate repression RTG pathway ssy1D, <br /> protein, Lst8p, bypass requirement Rtg2p,        ptr3D, shr3D lst8-1 cells, these <br /> establishes wild-type cells, Lst8p negative         mutations bypass requirement Rtg2p CIT2 <br /> regulator RTG-dependent gene expression. As dia-              expression lst8(2&#177;5) mutations. <br /> grammed model Figure 8, suggest Lst8p             In considering effect glutamate RTG-depend- <br /> can regulate RTG-dependent gene expression sites,         ent gene expression, important distinguish between <br /> one downstream Rtg2p upstream            sensing extra- intracellular glutamate. In cells <br /> level external glutamate repression RTG                compromised mitochondrial function, intracellular <br /> pathway.                                                         glutamate supplies maintained RTG regulation of <br />    Because lst8-1 lst8-(2&#177;5) mutant cells,           expression TCA cycle genes responsible the <br /> expression CIT2-lacZ reporter gene constitutive       synthesis a-ketoglutarate (Liu Butow, 1999), by <br /> and insensitive repression glutamate, simple           increased expression genes encoding enzymes that <br /> way Lst8p regulate RTG pathway              function anaplerotic pathways, small molecule transport <br /> controlling cells&apos; ability glutamate. Indeed,   peroxisomal activities converge supply acetyl- <br /> previous study demonstrated general amino acid          CoA, oxaloacetate citrate TCA cycle, by <br /> permease, Gap1p, failed targeted plasma             increased expression genes enable cells utilize <br /> membrane lst8-1 mutant cells (Roberg et al., 1997a).          poor nitrogen sources (Epstein et al., 2001b). In contrast, <br /> Our results suggest, however, glutamate repression       glutamate available medium, RTG- <br /> RTG-dependent gene expression likely occur            dependent gene expression down-regulated the <br /> through glutamate sensing involving Ssy1p&#177;Ptr3p              SSY1&#177;PTR3 signal transduction pathway. <br /> signal transduction pathway processes                Although RTG pathway insensitive to <br /> dependent glutamate uptake. First, inactivation        glutamate repression lst8-(2&#177;5) mutant cells, quite <br /> Ssy1p&#177;Ptr3p pathway eliminated glutamate repression           possible Lst8p regulatory site(s) downstream of <br /> RTG-dependent gene expression, glutamate uptake           Rtg2p direct connection glutamate <br /> ssy1D cells inhibited ~50%; result     regulation pathway. The WD40-repeat motif was <br /> in agreement previous study (Klasson et al., 1999).       identi&#174;ed originally b-subunit heterotrimeric G <br /> Secondly, glutamate repression RTG pathway            proteins (Fong et al., 1986) subsequently been <br /> largely unaffected inactivation general           wide spectrum regulatory proteins, it <br /> (Gap1) glutamate/aspartate-speci&#174;c (Dip5p) amino             functions mediating protein&#177;protein interactions. Thus <br /> acid permeases. Finally, overexpression Gap1p             Lst8p component complex tethers <br /> Dip5p ssy1D cells restored rate glutamate uptake       Rtg1p&#177;Rtg3p cytoplasm. In model, loss of <br /> to ~80% wild-type cells, CIT2 expression,      glutamate repression RTG-dependent gene expression <br /> in ssy1D cells alone, insensitive glutamate repres-       lst8-(2&#177;5) mutant cells consequence the <br /> sion. These data suggest that, Ptr3p-coupled             constitutive localization Rtg1p&#177;Rtg3p complex in <br />  <br /> 7214 <br />                                                                                                  Lst8p negatively regulates RTG gene function <br />  <br />  <br /> Table I. Genome-wide effects glutamate gene expression <br /> Gene expression            ORF                 Gene             Gene product/function                                                     Fold change <br />  <br /> Repressed <br /> NCR target genes <br />                            YNL142W             MEP2             ammonia permease low capacity high af&#174;nity                         14.3 <br />                            YKR039W             GAP1             general amino acid permease                                               14.0 <br />                            YKR034W             DAL80            negative regulator multiple nitrogen catabolic genes                   12.4 <br />                            YLR158C             ASP3C            L-asparaginase II, extracellular                                           9.0 <br />                            YIR032C             DAL3             ureidoglycolate hydrolase                                                  7.7 <br />                            YLR155C             ASP3A            L-asparaginase II, extracellular                                           7.5 <br />                            YLR157C             ASP3B            L-asparaginase II, extracellular                                           7.4 <br />                            YLR160C             ASP3D            L-asparaginase II, extracellular                                           5.8 <br />                            YJR152W             DAL5             allantoate permease                                                        5.1 <br />                            YIR031C             DAL7             malate synthase 2                                                          4.5 <br />                            YLR142W             PUT1             proline oxidase                                                            3.3 <br />                            YOR348C             PUT4             putative proline-speci&#174;c permease                                          2.7 <br /> RTG target genes <br />                            YLR304C             ACO1             aconitase, mitochondrial                                                   6.5 <br />                            YCR005C             CIT2             non-mitochondrial citrate synthase                                         3.3 <br />                            YOR136W             IDH2             NAD+-dependent isocitrate dehydrogenase                                    3.1 <br /> Others <br />                            YKR033C                              protein unknown function                                               11.4 <br />                            YLR348C             DIC1             mitochondrial dicarboxylate transport protein                              5.4 <br />                            YLR349W                              protein unknown function                                                4.4 <br />                            YBR182C             SMP1             probable transcription factor                                              4.3 <br />                            YBR294W             SUL1             probable sulfate transport protein                                         4.2 <br />                            YPL058C             PDR12            multidrug resistance transporter                                           3.6 <br />                            YNL208W                              protein unknown function                                                3.2 <br />                            YOR135C                              protein unknown function                                                2.9 <br />                            YJL172W             CPS1             carboxypeptidase yscS                                                      2.9 <br />                            YGL045W                              protein unknown function                                                2.7 <br />                            YML131W                              putative NAD-dependent oxidoreductase                                      2.7 <br />                            YLR161W                              protein unknown function                                                2.7 <br />                            YER124C                              protein unknown function                                                2.6 <br />                            YLL053C                              aquaporin water channel protein                                            2.6 <br />                            YIL121W                              member MFS-MDR family multidrug permeases                            2.5 <br />                            YGL028C             SCW11            soluble cell wall protein                                                  2.5 <br /> Induced <br /> Amino acid <br /> permeases <br />                            YDR046C             BAP3             valine transporter                                                        22.9 <br />                            YDR068              BAP2             probable amino acid permease leucine, isoleucine valine <br />                            YDR                                                                                                            20.0 <br />                            YCL025C             AGP1             amino acid permease                                                       12.1 <br />                            YBR069C             VAP1             probable amino acid transport protein                                     12.0 <br />                            YDR508C             GNP1             high-af&#174;nity glutamine permease                                            6.1 <br />                            YGR055W             MUP1             high-af&#174;nity methionine permease                                           4.3 <br /> Proteases <br />                            YIR039C             YPS6             GPI-anchored aspartic protease                                             6.6 <br />                            YOL154W                              protein similarity zinc metalloproteinases                         2.8 <br /> Other <br />                            YDR509W                              protein unknown function                                                4.8 <br />                            YNL160W             YGP1             glycoprotein synthesized response nutrient limitation                3.8 <br />                            YMR011W             HXT2             high-af&#174;nity hexose transporter-2                                          3.6 <br />                            YBR092C             PHO3             acid phosphatase, constitutive                                             3.2 <br />                            YOL158C             ENB1             enterobactin transporter                                                   3.0 <br />                            YNL065W                              member MFS-MDR family multidrug permeases                            2.7 <br />  <br /> Four independent cultures strain S288C grown mid-log phase YNB5%D 0.2% glutamate. After isolation poly(A)+ <br /> RNA each, labeled cDNAs Cy3 Cy5 mixed oppositely labeled cDNA treated untreated samples. <br /> The indicated fold changes gene expression determined described Epstein et al. (2001a) using cut-off 2.5-fold change average <br /> of expression ratios independent RNA preparations, repeated reversal Cy3 Cy5 labeling. <br />  <br />  <br />  <br /> the nucleus. An alternative model Lst8p bind                       What molecular basis proposed dual <br /> to regulate activity kinase phosphatase,                 sites action Lst8p diagrammed model of <br /> effecting change phosphorylation state Rtg3p;                     Figure 8? We suggest domains of <br /> this, turn, dictate Rtg3p&apos;s nuclear entry exit.                   Lst8p, functioning assembly activity the <br /> It remains determined internal glutamate                         Ssy1p&#177;Ptr3p signal transduction complex in <br /> sensed RTG pathway Rtg2p functions                             negative regulation RTG-dependent gene expression <br /> to transduce glutamate signals Rtg1p&#177;Rtg3p                             downstream Rtg2p. In addition genetic evidence <br /> transcription complex.                                                        summarized, clearly de&#174;ning sites Lst8p action <br />  <br />                                                                                                                                                  7215 <br />  Z.Liu et al. <br />  <br />  <br />                                                                           Lst8p localized cellular <br />                                                                           compartment. Although speci&#174;c intracellular location <br />                                                                           Lst8p established, preliminary cell fraction <br />                                                                           studies suggest Lst8p localized to <br />                                                                           different cellular compartments (unpublished observa- <br />                                                                           tions). Further studies required clarify these <br />                                                                           issues. <br />                                                                               Ssy1p shown involved amino acid <br />                                                                           inductive expression amino acid permeases <br />                                                                           (Didion et al., 1998; Iraqui et al., 1999), peptide <br />                                                                           transporter, Ptr2p (Didion et al., 1998), and, shown <br />                                                                           here, glutamate repression. These &#174;ndings prompted us <br />                                                                           carry genome-wide transcriptional analysis the <br />                                                                           effects glutamate gene expression. Microarray <br />                                                                           experiments revealed major pathways were <br />                                                                           affected glutamate: NCR; RTG pathway; the <br />                                                                           amino acid inductive pathway. Interestingly, Ssy1p <br />                                                                           appears involved pathways <br />                                                                           (Didion et al., 1998; Iraqui et al., 1999; Klasson et al., <br /> Fig. 8. Model dual regulation RTG gene functions Lst8p.         1999). One set genes expression affected by <br /> Once cell surface, Ssy1p binds glutamate glutamine and <br /> sends inhibitory signal Rtg2p Ptr3p. As revealed     glutamate microarray analysis amino acid- <br /> lst8-1 mutation, Lst8p affect targeting assembly          inducible genes, BAP2, BAP3, GNP1, VAP1/TAT1 and <br /> Ssy1p&#177;Ptr3p system, signal transduction function suggested      AGP1. Their transcription shown dependent <br /> by Roberg et al. (1997a), act upstream Rtg2p. Lst8p        Ssy1p (Iraqui et al., 1999). Although, compared with <br /> also proposed function downstream Rtg2p, revealed the <br /> lst8-(2&#177;5) mutations. Intracellular glutamate hypothesized be <br />                                                                           hydrophobic amino acids, glutamate glutamine are <br /> sensed Ptr3p (Klasson et al., 1999), additional component      strong repressors RTG gene function, weak <br /> of Ssy1p signaling pathway. Repressive signals generated           inducers AGP1 expression. We consistently observed <br /> glutamate lock Lst8p `RTG inhibition&apos; state     addition 0.01% glutamate growth medium <br /> Lst8p available function Ssy1p&#177;Ptr3 signaling;   resulted 4- 5-fold greater induction AGP1 <br /> would effect attenuating glutamate repression RTG gene <br /> functions.                                                                compared addition 0.2% glutamate (data not <br />                                                                           shown), supporting Lst8p switching model <br />                                                                           discussed above. The amino acid induction PTR2 <br /> both upstream downstream Rtg2p, striking                transcription involves transcriptional repressor Cup9p <br /> the lst8-(2&#177;5) mutations clustered localize                  N-end rule pathway component, Ubr1p (Goodman <br /> region Lst8p predicted                  et al., 1997; Byrd et al., 1998). Is cross- <br /> protein opposite lst8-1 mutation (Figure 2B).              regulation pathways? We prelim- <br /> WD40-repeat proteins usually adopt b-propeller struc-                   inary experiments (data shown) ubr1, ure2 gln3 <br /> ture, use blades interact              deletion mutations signi&#174;cant effect the <br /> proteins affecting blades (Goodman                      expression RTG target gene, CIT2, and, conversely, <br /> et al., 1997; ter Haar et al., 1998). We propose                 rtg mutations affect expression NCR <br /> distinctive phenotypes types lst8 mutations                 target gene, GLN1. These &#174;ndings suggest is <br /> due defects interaction protein                     little direct cross-regulation pathways <br /> different factors. The dual functionality Lst8p                    downstream Ssy1p&#177;Ptr3p. <br /> facilitate glutamate homeostasis participating                        Recent work shown TOR (target of <br /> negative feedback loop regulating glutamate levels. In                rapamycin) kinase signaling pathway, involved <br /> the presence high concentrations external glutamate,                nutrient sensing, affects RTG-dependent gene <br /> signals generated Ssy1p `lock&apos; Lst8p form                   expression manner retrograde pathway, <br /> that functions sequester Rtg1p&#177;Rtg3p complex                    affecting subcellular localization Rtg1p <br /> the cytoplasm. Thus, Lst8p available                Rtg3p Rtg2p-dependent manner (Komeili et al., <br /> Ssy1p signal transduction system, plausibly                2000). Inhibition TOR kinase pathway treating <br /> function targeting post-translational modi&#174;cation                cells rapamycin growing poor <br /> one components system. This model                   nitrogen source promotes dissociation Ure2p a <br /> explain observation glutamate blocks targeting               cytoplasmic, phosphorylated form positive regulator <br /> of Gap1p plasma membrane (Roberg et al.,                           NCR pathway, Gln3p, resulting Gln3&apos;s depho- <br /> 1997a,b), suggests means attenuating                    sphorylation nuclear accumulation (Beck Hall, <br /> glutamate repression concentration glutamate                 1999; Cardenas et al., 1999; Hardwick et al., 1999). <br /> the medium decreases. Consistent `switching&apos;                    Dephosphorylation Gln3p believed controlled <br /> model, observed rtg mutant cells grown                 type 2A-related phosphatase, Sit4p, its <br /> rich medium, ~10-fold induction AGP1                       inhibitor, Tap42p, effector protein TOR pathway <br /> expression (data shown). This &#174;nding indicate                   (Beck Hall, 1999). An sit4 deletion mutation, how- <br /> that Lst8p released site(s) downstream               ever, does affect CIT2 expression (unpublished data), <br /> Rtg2p protein available                    suggesting different TOR effectors serve to <br /> SSY1&#177;PTR3 pathway. Altogether, &#174;ndings imply                   regulate NCR RTG pathways. <br />  <br /> 7216 <br />                                                                                               Lst8p negatively regulates RTG gene function <br />  <br />  <br /> Table II. <span id='am-23' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-24' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-27'  ></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae strains used <br /> Strain                   Genotype                                                                                              Source <br />  <br /> PSY142                   MATa ura3-52 leu2 lys2 ura3::CIT2-lacZ                                                                R.Butow <br /> PSY142-rtg2              MATa ura3-52 leu2 lys2 ura3::CIT2-lacZ rtg2D::ura3                                                    R.Butow <br /> RBY346                   MATa ura3-52 leu2::LEU2-RTG2 lys2 ura3::CIT2-lacZ rtgD::ura3 lst8-5                                   study <br /> RBY426                   MATa ura3-52 leu2 lys2 ura3::CIT2-lacZ rtg2D::ura3 lst8-5                                             study <br /> RBY427                   MATa ura3-52 leu2 lys2 ura3::CIT2-lacZ rtg2D::ura3 lst8-5 rtg3D::LEU2                                 study <br /> RBY254                   MATa ura3-52 leu2 lys2 ura3::CIT2-lacZ lst8-1                                                         study <br /> RBY264                   MATa ura3-52 leu2 lys2 ura3::CIT2-lacZ lst8-1 rtg2D::kanMX4                                           study <br /> RBY418                   MATa ura3 leu2 lys2                                                                                   study <br /> RBY428                   MATa/a ura3/ura3 leu2/leu2 lys2/lys2 LST8/lst8D::LEU2 [pLST8]                                         study <br /> RBY419                   MATa ura3 leu2 lys2 rtg2D::kanMX4                                                                     study <br /> RBY420                   MATa ura3 leu2 lys2 lst8D::LEU2 pLST8                                                                 study <br /> RBY421                   MATa ura3 leu2 lys2 lst8D::LEU2 rtg2D::kanMX4 pLST8                                                   study <br /> RBY422                   MATa/a ura3/ura3::CIT2-lacZ leu2/leu2 lys2/lys2 RTG2/rtg2D::ura3 LST8/lst8-5                          study <br /> RBY423                   MATa/a ura3/ura3::CIT2-lacZ leu2/leu2 lys2/lys2 rtg2D::ura3/rtg2D::kanMX4 LST8/lst8-5                 study <br /> RBY424                   MATa/a ura3/ura3::CIT2-lacZ leu2/leu2 lys2/lys2 rtg2D::ura3/rtg2D::kanMX4 lst8D::LEU2/lst8-5          study <br /> ECY408                   MATa ura3 lst8-1                                                                                      C.Kaiser <br /> S288C                    MATa ura3-52 <br /> M4276                    MATa ura3 gap1D dip5D::kanMX                                                                          M.Kielland-Brandt <br /> PLY126                   MATa ura3-52 lys2D201                                                                                 P.Ljungdahl <br /> PLY151-ura3              MATa ura3-52 lys2D201 ade2 shr3D1::ura3                                                               P.Ljungdahl <br /> HKY20                    MATa ura3-52 lys2D201ssy1D13                                                                          P.Ljungdahl <br /> HKY31                    MATa ura3-52 lys2D201 ptr3D15                                                                         P.Ljungdahl <br /> PLY126-rtg2              MATa ura3-52 lys2D201 rtg2::kanMX4                                                                    study <br /> PLY151-rtg2              MATa ura3-52 lys2D201 alde2 shr3D1::ura3 rtg2::kanMX4                                                 study <br /> HKY20-rtg2               MATa ura3-52 lys2D201 ssy1D13 rtg2::kanMX4                                                            study <br /> HKY31-rtg2               MATa ura3-52 lys2D201 ptr3D15 rtg2::kanMX4                                                            study <br />  <br />  <br />  <br /> Materials methods                                                       AAAGCGGGAAGAAGTCA-3&#162; used amplify 2.1 kbp LST8 <br />                                                                             sequence. ECY408 genomic DNA used amplify lst8-1 mutant <br /> Strains                                                                     allele. The ampli&#174;ed DNA digested SpeI SalI ligated <br /> Strains used study listed Table II.                          Yip352 pRs416 form, respectively, Yip352-lst8-1 pRS416- <br />                                                                             lst8-1. Similarly, LST8 ampli&#174;ed PSY142 strain and <br /> Growth media, growth conditions                                             isogenic rtb2 mutant strains construct pRS416-LST8, pRS416-lst8- <br /> Yeast strains grown 30&#176;C YPD (1% yeast extract, 2%               2&#177;6, respectively. pCIT2-lacZ described previously (Liao et al., <br /> bactopeptone 2% dextrose), YNBcasD (0.67% yeast nitrogen base,          1991; Liu Butow, 1999). To obtain plasmid 2m-LYS2, pRS426 <br /> 1% casamino acids 2% dextrose) minimal YNB medium (0.67%             plasmid digested EcoRV StuI remove URA3 <br /> yeast nitrogen base) supplemented 2 5% glucose (YNBD             sequence replaced 5.2 kbp LYS2 sequence SmaI- and <br /> YNB5%D, respectively), Na glutamate (as indicated        PstI-digested YDp-K (Berben et al., 1991). To construct plasmid 2m- <br /> the text &#174;gures), supplemented required nutrients.         TEF-DIP5, primer pair 5&#162;-gtcaACTAGTCTCTAAGTAATGAA- <br />                                                                             GATGCC-3&#162; 5&#162;-gtcaGTCGACGTGATACCTGTACACTATGG-3&#162; <br /> Strain constructions                                                        used amplify DIP5 open reading frame 413 bp 3&#162;- <br /> Standard genetic manipulations carried described Rose        untranslated region. The resultant PCR product digested SpeI <br /> et al. (1990). RBY426 isolated rtg2 bypass screen described    SalI ligated pRS426-TEF plasmid (Mumberg et al., <br /> below. RBY426 transformed rtg3D::LEU2 disruption cassette        1995). The URA3 sequence pRS426-TEF-DIP5 replaced with <br /> to form RBY427. RBY254 lst8-1 mutant obtained two-         LYS2, similarly construction 2m-LYS2. To construct 2m-GAP1 <br /> step gene replacement. RBY418 derived PSY142 lacking           plasmid, 3.5 kbp GAP1 sequence obtained SpeI- SalI- <br /> integrated CIT2-lacZ reporter gene mating-type switching PSY142       digested pPL247 (Ljungdahl et al., 1992) ligated pRS426. The <br /> cells transformed plasmid pGAL-HO (Herskowitz Jensen,          resultant pRS426-GAP1 digested NcoI StuI remove part <br /> 1991). RBY418 transformed rtg2::kanMX4 disruption               URA3 sequence replaced LYS2 sequence <br /> cassette form RBY419. RBY420 meiotic segregant obtained             described above. <br /> following sporulation diploid RBY428 transformed pRS416- <br /> LST8. RBY421 constructed RBY420 transforming cells              Rtg2 bypass mutant screen <br /> with pUC18-rtg2::kanMX4 digested PstI. Matings carried        Ethyl methanesulfonate mutagenesis carried essentially as <br /> between RBY426 RBY418, RBY419 RBY421 form RBY422,                 described previously (Liao Butow, 1993). Cells plated on <br /> RBY423 RBY424, respectively. RTG2 disruptions strains             YNBD plus X-gal amino acid supplements. Fourteen rtb mutants <br /> PLY126-rtg2, PLY151-rtg2, HKY20-rtg2 HKY31-rtg2 carried            obtained grouped complementation groups, <br /> out transformation rtg2::kanMX4 disruption cassette    rtb1 rtb2, described text. The rtb mutants arose a <br /> respective parental strains.                                                frequency ~1 106 cells. <br />  <br /> Plasmid constructs                                                          Isolation RTB2 (LST8) gene <br /> Construction rtg3D::LEU2 disruption cassette described previ-        A URA3 centromeric CIT2-lacZ reporter gene transformed a <br /> ously (Liu Butow, 1999). To construct rtg2D::kanMX4 cassette          RBY285 (rtg2D rtb2-5) derivative lacking integrated version the <br /> derivative, primer set 5&#162;-AGGCACACTCTTCTTCAC-3&#162; 5&#162;-                 reporter gene. To isolate RTB2 gene, wild-type genomic DNA <br /> gactaagcttTGAACAACAAGAAGGTGCCC-3&#162; used amplify                       library constructed centromeric vector p366 carrying LEU2 <br /> RTG2. After digestion BglII HindIII, RTG2 DNA ligated          marker screened transformants RBY285 abolish <br /> into BamHI HindIII sites pRS416 form pRS416-RTG2. A           CIT2-lacZ expression. Two 8000 LEU2+ transformants were <br /> 643 bp NdeI fragment pRS416-RTG2 cloned NdeI site        obtained overlapping restriction maps. Subcloning the <br /> pUC18 form pUC18-RTG2. A kanMX4 disruption cassette module           centromeric plasmid pRS415 retransformation revealed a <br /> inserted SpeI site form pUC18-rtg2::kanMX4. The primer set      fragment containing LST8 gene necessary suf&#174;cient to <br /> 5&#162;-gactactagtACCACCAGGACCAAAGCTTG-3&#162; 5&#162;-gactgtcgacCA-                   complement rtb2-5 mutant phenotype. <br />  <br />  <br />                                                                                                                                             7217 <br />  Z.Liu et al. <br />  <br /> b-galactosidase assays                                                           Doolittle,R.F. Simon,M.I. (1986) Repetitive segmental structure <br /> b-galactosidase assays carried previously described (Liu         transducin b subunit: homology CDC4 gene and <br /> Butow, 1999). For plasmid&#177;strain combination, assays                   identi&#174;cation related mRNAs. Proc. Natl Acad. Sci. USA, 83, <br /> conducted triplicate independent experiments carried             2162&#177;2166. <br /> two times.                                                           Forsburg,S.L. Guarente,L. (1989) Communication between <br />                                                                                  mitochondria nucleus regulation cytochrome genes in <br /> Assays amino acid uptake                                                     yeast Saccharomyces cerevisiae. Annu. Rev. Cell Biol., 5, <br /> Amino acid uptake assayed described Roberg et al. (1997a)              153&#177;180. <br /> except glutamate uptake, equal volumes 0.2 mM 14C-labeled         Gangloff,S.P., Marguet,D. Lauquin,G.J.M. (1990) Molecular cloning <br /> glutamate 59.1 mM unlabeled glutamate mixed diluted 50-             yeast mitochondrial aconitase gene (Aco1) evidence a <br /> fold assay mixture. At 2 min intervals, samples 100 ml      synergistic regulation expression glucose plus glutamate. Mol. <br /> were diluted 3 ml ice-cold 5.9 mM glutamate quench                Cell. Biol., 10, 3551&#177;3561. <br /> reaction, cells collected &#174;ltration washed 33 5 ml       Goodman,O.B., Krupnick,J.G., Gurevich,V.V., Benovic,J.L. and <br /> of ice-cold water. L-[U-14C]glutamic acid (250 mCi/mmol) obtained            Keen,J.H. (1997) Arrestin/clathrin interaction. Localization the <br /> from Amersham Pharmacia (Buckinghamshire, UK).                                   arrestin binding locus clathrin terminal domain. J. Biol. Chem., <br />                                                                                  272, 15017&#177;15022. <br /> Microarray analysis                                                           Hardwick,J.S.,      Kuruvilla,F.G.,     Tong,J.K.,     Shamji,A.F.     and <br /> A fresh culture S288C diluted 250 ml YNB5%D               Schreiber,S.L. (1999) Rapamycin-modulated transcription de&#174;<span id='am-61' about='obo:NCBITaxon_203327' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_8220 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_8219 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489878 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_497223 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665'><span id='am-62' property="rdfs:label" content="Nes" datatype="xsd:string"></span><span id='am-63'  ></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>nes</span> <br /> without 0.2% glutamate. After overnight growth OD600 0.8, cells          subset nutrient-sensitive signaling pathways directly <br /> collected mRNA samples prepared described (Epstein et al.,           controlled Tor proteins. Proc. Natl Acad. Sci. USA, 96, <br /> 2001b). Cy3- Cy5-labeled cDNAs prepared hybridized             14866&#177;14870. <br /> microarray 6219 yeast genes. Replica experiments carried          Herskowitz,I. Jensen,R.E. (1991) Putting HO gene work: <br /> using independent liquid cultures opposite con&#174;guration          practical uses mating-type switching. Methods Enzymol., 194, <br /> Cy3 Cy5. The data analyzed described (Epstein et al.,                132&#177;146. <br /> 2001a,b) using cut-off value average 2.5-fold change         Iraqui,I., Vissers,S., Bernard,F., DeCraene,J.O., Boles,E., Urrestarazu,A. <br /> replica hybridizations.                                                          Andre,B. (1999) Amino acid signaling Saccharomyces <br />                                                                                  cerevisiae: permease-like sensor external amino acids F- <br />                                                                                  box protein Grr1p required transcriptional induction the <br /> Acknowledgements                                                                 AGP1 gene, encodes broad-speci&#174;city amino acid permease. <br />                                                                                  Mol. Cell. Biol., 19, 989&#177;1001. <br /> We thank, G.Fink, C.Kaiser, P.O.Ljungdahl M.C.Kielland-Brandt         Jia,Y., Rothermel,B., Thornton,J. Butow,R.A. (1997) A basic <br /> yeast strains plasmids, Tsan Xiao technical support, members         helix&#177;loop&#177;helix zipper transcription complex functions a <br /> of Butow laboratory helpful discussions. This work supported         signaling pathway mitochondria nucleus. Mol. Cell. <br /> by grants GM22525 CA77811 National Institutes Health,            Biol., 17, 1110&#177;1117. <br /> and I-0642 The Robert A.Welch Foundation.                                Klasson,H., Fink,G.R. Ljungdahl,P.O. (1999) Ssy1p Ptr3p are <br />                                                                                  plasma membrane components yeast senses <br />                                                                                  extracellular amino acids. Mol. Cell. Biol., 19, 5405&#177;5416. <br /> References                                                                    Komeili,A., Wedaman,K.P., O&apos;Shea,E.K. Powers,T. (2000) <br />                                                                                  Mechanism metabolic control: target rapamycin signaling links <br /> Beck,T. Hall,M.N. (1999) The TOR signaling pathway controls                  nitrogen quality activity Rtg1 Rtg3 transcription <br />   nuclear localization nutrient-regulated transcription factors. Nature,      factors. J. Cell Biol., 151, 863&#177;878. <br />   402, 689&#177;692.                                                               Kraulis,P.J. (1991) MOLSCRIPT: program produce detailed <br /> Berben,G., Dumont,J., Gilliquet,V., Bolle,P.A. Hilger,F. (1991) The          schematic plots proein structures. J. Appl. Crystallogr., 24, <br />   YD <br /> </body></html>