<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Biochem. J. (2001) 358, 447&#8211;455 (Printed Great Britain)                                                                                               447 <br />  <br />  <br /> Subnuclear localization mitotic phosphorylation HIRA, human <br /> homologue <span id='am-21' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-28'  ></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> transcriptional regulators <br /> Hir1p/Hir2p <br /> Filomena DE <span id='am-138' about='obo:NCBITaxon_191969' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_27544 obo:NCBITaxon_42300 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_33208 obo:NCBITaxon_191874 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-139' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-140'  ></span><span id='am-141' property="rdfs:label" content="Lucia" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>LUCIA</span>, Ste! phanie LORAIN1, Christine SCAMPS, Fre! de! ric GALISSON, Jan MACHOLD2 Marc LIPINSKI3 <br /> *Interactions Mole! culaires et <span id='am-48' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-49'  ></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Cancer</span>, CNRS UMR 1598, Institut Gustave Roussy, 94805 Villejuif Cedex, France <br />  <br />  <br />  <br />  <br /> The HIRA gene encodes nuclear protein histone-binding                          necessitated 300 mM NaCl 150 mM sufficient in <br /> properties conserved yeast humans.                             mitotic cells. Immunofluorescence staining microscopic <br /> Hir1p Hir2p, HIRA homologues <span id='am-38' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-43'  ></span><span id='am-44' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span>Saccharomyces</span>                             examination mitotic cells revealed HIRA excluded from <br /> cere&#138;isiae, transcriptional co-repressors action resides                    condensed chromosomes, confirming lack association with <br /> at chromatin level occurs cell-cycle-regulated fashion.                  chromatin mitosis. Western-blot analysis indicated that <br /> In mammals, HIRA essential gene early development,                       HIRA molecules hyper-phosphorylated point the <br /> possibly control specific gene-transcription pro-                      cell cycle. Metabolic labelling pulse-chase experiments <br /> grammes, exact function remains deciphered. Here                        characterized HIRA stable protein half-life approx. <br /> we report subnuclear distribution cell-cycle behaviour                     12 h. The mitotic phosphorylation HIRA provide the <br /> of HIRA protein. Using biochemical immuno-                               dividing cell way retarget HIRA-containing multi- <br /> fluorescence techniques, minor fraction HIRA                           protein complexes different chromatin regions daughter <br /> tightly associated nuclear matrix, material                         compared parental cells. <br /> remains nuclease treatment high-salt extraction. How- <br /> ever, HIRA molecules proved extractable. In non-                                 Key words : histone-binding protein, nuclear matrix, protein <br /> synchronized cell populations, extraction chromatin                              half-life. <br />  <br />  <br />  <br />  <br /> INTRODUCTION                                                                          [3,14] ; addition, oligonucleotide-induced inhibition HIRA <br />                                                                                       expression chick neural crest cells results the <br /> The HIRA protein identified eukaryotic                        malformations cono-truncal region heart are <br /> <span id='am-97' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-98' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> [1&#8211;5]. Its structure strikingly conserved                     observed syndromic patients [15]. <br /> evolution. In N-terminal third, seven reiterations module,                      Most functional studies HIRA protein <span id='am-45' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-46' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> been <br /> called WD repeat, define constant region,                       performed Saccharomyces cere&#138;isiae, provides ex- <br /> predicted assemble characteristic &#946;-propeller structure                     ception rule single HIRA, contains structural <br /> [6]. As reviewed recently, &#946;-propellers offer convenient surfaces                     homologues, Hir1p Hir2p, produced separate genes <br /> for inter-molecular contacts constituents macro-                           probably evolved divergently ancient gene-dupli- <br /> molecular complexes [7]. In mammals, HIRA essential                            cation event [1]. The HIR genes originally identified <br /> proper development HIRA-null mice grossly abnormal                          important periodic control histone gene transcrip- <br /> and die early embryogenesis [8]. The human HIRA gene lies                          tion [16,17]. In absence HIR gene products, transcription <br /> within gene-rich region human chromosome 22q11.2                         constitutive HTA1-HTB1 histone H2A\H2B <br /> reduced hemizygosity patients developmental                       gene locus, normally repressed outside late G and <br />                                                                                                                                                     &quot; <br /> conditions known DiGeorge syndrome velocardiofacial                             S phases cell cycle, indicating Hir1p Hir2p <br /> syndrome [9,10]. In patients, organs tissues derived                         participate cell-cycle-dependent transcriptional repression [18]. <br /> from affected neural crest cells pharyngeal region                    The mechanism repression elucidated totally, <br /> the developing embryo affected [11]. Two recent reports                        current view implies role proteins local <br /> have demonstrated haploinsufficiency Tbx1, gene                       chromatin structure. Indeed, maximal transcription locus <br /> within large 22q11 deletion, sufficient generate                         reached Hir1p\Hir2p-dependent repression relieved <br /> mice cardiovascular defects observed DiGeorge                          following recruitment Swi\Snf multi-protein complex (or <br /> syndrome\velocardiofacial syndrome patients [12,13]. HIRA                             Swi\Snf subcomplex) [19]. Swi\Snf major ATPase- <br /> may play additional role, albeit partial, pathogenic                            dependent chromatin-remodelling machinery [20&#8211;22] has <br /> processes, mouse chick expressed                     long known antagonize chromatin-mediated trans- <br /> affected embryological region proper time development                       criptional repression yeast [23]. How Hir1p Hir2p <br />  <br />  <br />  <br />  <br />   Abbreviations used : DAPI, 4h,6-diamidino-2-phenylindole ; CIP, calf intestinal phosphatase ; PARP, poly(ADP-ribose)polymerase ; CSK, cytoskeletal <br /> buffer ; GST, glutathione S-transferase ; FCS, fetal calf serum. <br />   1 <br />     Present address : Imperial Cancer Research Fund, Clare Hall Laboratories, South Mimms, Hertfordshire EN6 3LD, U.K. <br />   2 <br />     Present address : Amersham Pharmacia Biotech Europe, Munzinger Strasse 9, D-79 111 Freiburg, Germany. <br />   3 <br />     To correspondence addressed (e-mail lipinski!igr.fr). <br />  <br />                                                                                                                                     # 2001 Biochemical Society <br />  448             F. De Lucia others <br />  <br /> alternate co-repressor co-activator states precise     0.25 M sucrose. Further extraction digested pellet <br /> times cell cycle remains determined. Because      performed twice high-salt buffer consisting 2 M <br /> is structurally similar Hir1p Hir2p physically interacts   NaCl\0.2 mM MgCl \10 mM Tris\HCl, pH 7.5, leaving nuclear <br />                                                                                            # <br /> with core histones [24] DNA-binding transcription            matrix insoluble pellet. All buffers contained cocktail of <br /> factors [25], HIRA likely function transcriptional      protease inhibitors (Complete4, Roche). <br /> regulation. Having recently identified novel cyclin\ <br /> cyclin-dependent kinase 2 substrate, transcriptional role      Western-blot analysis <br /> the human HIRA protein relate cell-cycle progression <br /> [26], similar yeast counterparts.                              The protein content different fractions determined <br />     We previously reported that, exponentially growing        using protein-assay reagent (Pierce) BSA used as <br /> cells, human HIRA protein largely nuclear ; interphase,    standard. Appropriate protein amounts precipitated with <br /> it seen numerous foci dispersed nucleus,    trichloroacetic acid prior PAGE performed 7.5 % 5.5 % <br /> the exception nucleoli [24]. Using biochemical approach,     polyacrylamide slab gels containing 0.1 % SDS. The gels were <br /> we human HIRA high-molecular-            transferred polyscreen PVDF membranes (Millipore), <br /> mass multi-protein complex present nuclear extracts           incubated affinity-purified 1455 reagent recognizing C- <br /> human tumour cells (J. Machold M. Lipinski, unpublished           terminal peptide HIRA, described previously [24], the <br /> work). To progress understanding HIRA protein           affinity-purified 484 antiserum (Agrobio) prepared described <br /> higher eukaryotes clues possible         below. Commercial monoclonal antibodies used controls <br /> function(s), examined turnover details         dilutions indicated manufacturers. Detection immune <br /> intra-nuclear distribution biochemical properties      complexes accomplished using appropriate dilutions of <br /> cell cycle. HIRA characterized stable protein         peroxidase-coupled goat anti-mouse anti-rabbit secondary <br /> half-life approx. 12 h, fraction associated         reagents (Pierce) followed detection using ECL system <br /> tightly nuclear matrix, insoluble material          (Amersham Pharmacia Biotech). <br /> remains high-salt extraction nuclease-treated nuclei [27]. <br /> Those HIRA molecules soluble phosphorylated            Preparation protein extracts asynchronous mitotic <br /> at time mitosis excluded mitotic chromosomes.         cells <br /> <span id='am-80' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-81' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-82'  ></span><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> provide way retarget HIRA-containing multi-             For exponentially growing HeLa cells, nuclear extracts were <br /> protein complexes following cell cycle.                        prepared resuspending pellet isolated nuclei buffer <br />                                                                       consisting 10 mM Hepes potassium salt, pH 7.9, 1.5 mM <br /> EXPERIMENTAL                                                          MgCl , 0.1 mM EGTA 0.5 mM dithiothreitol the <br />                                                                              # <br />                                                                       addition 150, 300 600 mM NaCl. Extraction performed <br /> Cell culture synchronization <br />                                                                       (30 min, 4 mC) constant agitation. Residual nuclear material <br /> K562 cells grown 37 mC 5 % CO RPMI (Life                pelleted centrifugation (30 min, 10 600 g). Exactly the <br />                                                 # <br /> Technologies) supplemented 5 % fetal calf serum (FCS ;           procedure followed protein preparations cell <br /> Seromed). HeLa cells grown Dulbecco&#8217;s modified Eagle&#8217;s        cultures enriched mitotic cells incubation presence <br /> medium supplemented 10 % FCS.                                    drugs preventing normal microtubule polymerization <br />    For cell-cycle synchronization mitosis, exponentially grow-     process. In situation, large proportion cells are <br /> ing HeLa cells treated 16 h, usually 0.4 &#181;g\ml          blocked stage mitosis nuclear membrane has <br /> nocodazole (Sigma). Some experiments carried          broken down. This results protein preparations technically <br /> shorter incubation period, presence lower drug concen-      referred nuclear. However, anything, the <br /> trations presence 0.25 &#181;M demecolcine (Sigma)       procedure results loss proportion nuclear proteins <br /> equivalent results. Mitotically enriched floating cells          purification process, soluble &#8216; cytosolic &#8217; <br /> harvested away adherent cells, analysed DNA content          proteins. In article, literature, we <br /> by propidium iodide staining flow cytometry using               refer &#8216; nuclei &#8217; &#8216; nuclear preparations &#8217; want a <br /> FACScalibur (Becton-Dickinson) proportion cells        better designation. <br /> in mitosis 4h,6-diamidino-2-phenylindole (DAPI) staining <br /> (0.4 &#181;g\ml, 5 min) observation fluorescence micro-        Antibodies <br /> scope (Olympus). This protocol reproducibly yields greater than <br /> 80 % cells DNA content corresponding G \M            The rabbit antiserum 484 (Agrobio) generated immuni- <br />                                                              #        zation recombinant form glutathione S-transferase <br /> phases, 70 % mitosis (see FACS analysis <br /> in Figure 5, below).                                                  (GST) fused amino acids 10&#8211;1017 HIRA, the <br />                                                                       previously described pGEX-HIRA construct [24], produced <br />                                                                       <span id='am-110' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-111' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-115' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-118'  ></span><span id='am-119' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span>Escherichia coli</span> BL21. Purification fusion protein was <br /> Protein fractionation nuclear matrix preparation                  performed overnight incubation (4 mC) rotating wheel in <br /> Cultured HeLa K562 cells harvested, washed twice           presence glutathione-agarose beads, followed elution <br /> PBS snap-frozen k80 mC. The cell membranes lysed          presence 500 mM Tris, pH 8.0\100 mM glutathione <br /> by thawing low-salt extraction buffer (10 mM Hepes,              (30 min, 4 mC rotating wheel). After trichloroacetic acid <br /> pH 7.9\1.5 mM MgCl \0.1 mM EGTA\0.5 mM dithiothreitol\                precipitation, GST-HIRA protein purified electro- <br />                         # <br /> 5 % glycerol). Nuclei pelleted centrifugation (600 g,         phoresis preparative slab gel stained 0.3 M <br /> 10 min). After wash, nuclei resuspended      CuCl . The negatively stained fusion protein sliced and <br />                                                                            # <br /> same buffer containing 0.35 M NaCl nuclear proteins               recovered gel electrotransfer. A series control <br /> extracted ice 30 min. After centrifugation (14 000 g, 4 mC,    Western-blot stainings demonstrated serum detects the <br /> 25 min), pellet digested (30 min, 30 mC) 300 units\ml    band 112 kDa detected 1455 antiserum [24]. <br /> RNase-free DNase I (Boehringer Mannheim) digestion               Affinity purification performed described [24] the <br /> buffer consisting 20 mM Tris\HCl, pH 7.5\5 mM MgCl \               GST-HIRA fusion protein used immunogen. <br />                                                                 # <br /> # 2001 Biochemical Society <br />                                                                     Subcellular localization mitotic phosphorylation HIRA protein                          449 <br />  <br /> The anti-poly(ADP-ribose)polymerase (PARP) anti-                   overnight presence 500 &#181;g total non-radioactive <br /> lamin B monoclonal antibodies purchased Clontech                 K562 lysate. After 10 washes 50 mM Tris\HCl, pH 8.0, <br /> and Oncogene Research Products, respectively. For immuno-                  0.25 % Igepal Ca-630, 1 mM EDTA 150 mM NaCl, the <br /> fluorescence detection histones, monoclonal reagent recog-            immunoprecipitates submitted SDS\PAGE analysed <br /> nizing core histones used (Roche).                                 autoradiography Western blotting. <br />  <br /> Immunofluorescence analysis                                                RESULTS <br /> For immunofluorescence staining situ-prepared nuclear                Significant amounts HIRA associate nuclear matrix <br /> matrix, HeLa cells extracted directly coverslips following         A number nuclear proteins contain so-called AT hook, a <br /> the procedure described Spector et al. [28]. In brief,     small sequence motif centred conserved <span id='am-16' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-17' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-18'  ></span><span id='am-19' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>glycine</span>- <br /> fractionation step, soluble proteins removed extraction            arginine-proline (GRP) tripeptide preceded and <br /> in cytoskeletal buffer (CSK, 10 mM Pipes, pH 6.8\300 mM                    followed basic amino acids ; motif appears sufficient to <br /> sucrose\100 mM NaCl\3 mM MgCl \1 mM EGTA) containing <br />                                      #                                     confer corresponding proteins DNA-binding property <br /> 0.5 % Triton X-100. Chromatin proteins released DNase              [29]. A nonapeptide (KKKKGRPRK) fitted description <br /> I digestion extraction 0.25 M ammonium sulphate.                  perfectly positions 640&#8211;648 human HIRA <br /> Nuclear matrices fixed 2 % paraformaldehyde                 amino acid sequence equivalent position each <br /> (20 min, room temperature) PBS processed                HIRA homologue identified mouse, Xenopus, chick Fugu <br /> immunofluorescence staining, performed essentially               (Table 1). Since AT-hook-containing proteins found <br /> as described [24]. Developing reagents consisted species-            associated nuclear matrix, insoluble material that <br /> specific <span id='am-11' about='obo:NCBITaxon_149625' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_163739 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Alexa" datatype="xsd:string"></span><span id='am-15'  ></span>Alexa</span> Fluor 488-conjugated (Molecular Probes)                 remains extraction soluble proteins nucleus, <br /> Texas Red-conjugated (Jackson Laboratories) goat anti-mouse                decided investigate subnuclear distribution HIRA <br /> Ig anti-rabbit Ig antisera, used dilution            molecules. Exponentially growing K562 cells treated as <br /> between 1\400 1\800. Stained cells observed, signals              described Experimental section obtain fractions of <br /> acquired optical sections generated described [24]. To           cytosolic, nuclear matrix-associated proteins. Lamin B a <br /> allow visual comparative analysis resistance               component lamina entirely retained <br /> different nuclear molecules nuclear-matrix-preparation              nuclear matrix material [30]. In contrast, PARP been <br /> procedure, image acquisitions performed fixed ac-                reported partition essentially soluble components of <br /> quisition settings.                                                        cell nucleus, usually 4 % molecules <br />                                                                            remaining nuclear matrix [31]. Our Western-blot analysis <br /> Phosphatase treatment                                                      performed commercial antibodies lamin B PARP <br />                                                                            reproduced results, lamin B signal being <br /> Nuclear 0.35 M NaCl extracts (35 &#181;g) incubated (1 h, 37 mC)           detected matrix preparation (Figure 1B, lane 7), whereas <br /> with various amounts calf intestinal phosphatase (CIP ; New             PARP mainly recovered soluble 0.35 M NaCl extract <br /> England Biolabs) buffer consisting 100 mM NaCl, 50 mM              (Figure 1C, lane 2). A longer exposure (Figure 1D) revealed that <br /> Tris\HCl, pH 7.5, 10 mM MgCl 1 mM dithiothreitol. <br />                                  #                                         small fraction nuclear PARP present among <br />                                                                            proteins tightly associated chromatin (Figure 1D, lane <br /> Half-life measurement HIRA protein                                  6) nuclear matrix (Figure 1D, lane 7). Thus nuclear <br />                                                                            proteins correctly partitioned procedure. <br /> For metabolic labelling, K562 cells (30i10&apos;) harvested               Staining blot affinity-purified 484 anti- <br /> washed twice long-term labelling medium consisting               serum raised human HIRA revealed HIRA molecules <br /> 90 % cysteine\methionine-free RPMI (ICN) 10 % complete                 distributed soluble 0.35 M salt extract (Figure <br /> RPMI (Life Technologies) supplemented 10 % FCS. Cells                 1A, lane 2) nuclear matrix (Figure 1A, lane 7). Identical <br /> were incubated 16&#8211;18 h 30 ml medium              results obtained different fractionation procedures, <br /> presence 100 &#181;Ci\ml mixture [$&amp;S]methionine                 including use 0.25 M ammonium sulphate instead 2 M <br /> [$&amp;S]cysteine (Trans $&amp;S-<span id='am-143' about='rdfs:label' typeof='owl:Thing'>Label</span>, ICN) humidified 5 % CO <br />                                                                 #          NaCl protein extraction described Penman and <br /> incubator (at 37 mC). Cells collected, washed          collaborators [32], protein extraction preceded by, instead of <br /> warm RPMI\10 % FCS 5i10&apos; cells seeded separate                 following, DNase treatment [33] (results shown). Although <br /> flasks. The entire cell contents flask harvested           difficult precisely quantify, proportion HIRA molecules <br /> different time points followed washing PBS snap- <br /> freezing k80 mC. <br />    For immunoprecipitation experiments, cells lysed while <br /> vortexing (30 min, 4 mC) 50 mM Tris\HCl, pH 8.0, 300 mM                 Table 1     Conservation AT-hook motif HIRA proteins various <br /> NaCl, 10 mM MgCl , 0.5 % Igepal Ca-630 (Sigma), 1 mM EDTA                  species <br />                      # <br /> and Complete antiprotease cocktail (Roche). After centri-              The AT hooks identified amino acid sequences HIRA homologues indicated <br /> fugation (10 600 g, 10 min, 4 mC), 500 &#181;g supernatant              species aligned. Numbers refer postitions amino acid sequences. <br /> precleared incubation (4 h 4 mC) Protein A\G&#8211; <br /> agarose (Santa Cruz Biotechnology) Protein                   Species               Amino acid position sequence         Reference <br /> A\G&#8211;agarose preincubated preimmune serum recom-                                          640             648 <br />                                                                            <span id='am-85' about='obo:NCBITaxon_9606' typeof='owl:Thing obo:NCBITaxon_9605 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="Homo sapiens" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasExactSynonym" content="man" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasExactSynonym" content="human" datatype="xsd:string"></span><span id='am-91'  ></span>Homo sapiens</span>              KKKKGRPRK                            [1] <br /> mended manufacturer. The precleared lysate adjusted                                   638 <br />                                                                            <span id='am-1' about='obo:NCBITaxon_10090' typeof='owl:Thing obo:NCBITaxon_314147 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_337687 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_862507 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_39107 obo:NCBITaxon_10088 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_10066 obo:NCBITaxon_6072 obo:NCBITaxon_33553 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9989 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="Mus muscaris" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Mus musculus" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="house mouse" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-7'  ></span><span id='am-8' property="oboInOwl:hasExactSynonym" content="mouse" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="mice C57BL/6xCBA/CaJ hybrid" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasAlternativeId" content="NCBITaxon:85055" datatype="xsd:string"></span>Mus musculus</span>              KKKKGRPRK646                         [2] <br /> to 150 mM NaCl 0.25 % Igepal Ca-630 subjected                   <span id='am-53' about='obo:NCBITaxon_8355' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8342 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_8360 obo:NCBITaxon_2759 obo:NCBITaxon_262014 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8353 obo:NCBITaxon_7776 obo:NCBITaxon_8352 obo:NCBITaxon_30319 obo:NCBITaxon_41666 obo:NCBITaxon_8292'><span id='am-54'  ></span><span id='am-55' property="oboInOwl:hasExactSynonym" content="clawed frog" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasExactSynonym" content="common platanna" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="Xenopus laevis" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasRelatedSynonym" content="Xenopus laevis (Daudin, 1802)" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasRelatedSynonym" content="Bufo laevis" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasRelatedSynonym" content="Xenopus leavis" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasExactSynonym" content="African clawed frog" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasExactSynonym" content="platanna" datatype="xsd:string"></span>Xenopus laevis</span>        635 <br />                                                                                                      KRKKGRPRK643                         C. Scamps M. Lipinski, <br /> overnight immunoprecipitation rotating wheel 4 mC using                                                                               unpublished work <br />                                                                                                  644 <br /> an irrelevant rabbit antiserum affinity-purified 484 anti-          <span id='am-65' about='obo:NCBITaxon_9031' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_8976 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_9005 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9030 obo:NCBITaxon_8825 obo:NCBITaxon_9072 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-66' property="oboInOwl:hasExactSynonym" content="bantam" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasExactSynonym" content="chickens" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="Gallus gallus domesticus" datatype="xsd:string"></span><span id='am-71'  ></span><span id='am-72' property="oboInOwl:hasRelatedSynonym" content="Gallus domesticus" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="Gallus gallus" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasExactSynonym" content="chicken" datatype="xsd:string"></span>Gallus gallus</span>               KKKKGRPRK652                       [3] <br /> HIRA reagent. The mixtures incubated (4 h 4 mC)            Fugu rubripes         640 <br />                                                                                                        KRKKGRPRK648                       [5] <br /> with 50 &#181;l Protein A\G&#8211;agarose preincubated <br />  <br />                                                                                                                                         # 2001 Biochemical Society <br />  450                F. De Lucia others <br />  <br />  <br />  <br />  <br /> Figure 1      A fraction HIRA protein associated nuclear matrix <br /> Nuclei prepared K562 cells (lane 1) extracted 0.35 M NaCl (lane 2). The pellet <br /> (lane 3) digested DNase I (lane 4). After centrifugation, supernatant harvested <br /> (lane 5) pellet extracted 2 M NaCl (lane 6), leaving nuclear matrix fraction (lane <br /> 7). Equal amounts proteins (40 &#181;g) resolved electrophoresis 7.5 % <br /> polyacrylamide gel immunoblotted using affinity-purified 484 anti-HIRA reagent (A) and <br /> commercial antibodies lamin B (B) PARP (C, D). <br />  <br />  <br />  <br />                                                                                                     Figure 3      HIRA protein mitotic cells <br />                                                                                                     (A) K562 cells stained immunofluorescently HIRA (lower panel) DNA counterstained <br />                                                                                                     using DAPI (upper panel). (B) Mitotic floating cells collected HeLa cells treated with <br />                                                                                                     demecolcine (Dc) asynchronous population exponentially growing HeLa cells (As) <br />                                                                                                     used prepare cellular extracts presence indicated concentrations NaCl. <br />                                                                                                     The HIRA molecules present 50 &#181;g proteins loaded lane detected using the <br />                                                                                                     affinity-purified 484 anti-HIRA antibody. <br />  <br />  <br />  <br />  <br />                                                                                                     cells. Nuclear matrices prepared situ HeLa cells <br />                                                                                                     grown directly coverslips. These adherent cells submitted <br />                                                                                                     various extraction steps described Experimental <br />                                                                                                     section, prior fixation processing immunofluorescence <br />                                                                                                     staining. In fractionation step, soluble cytosolic and <br />                                                                                                     nucleoplasmic proteins removed extraction CSK <br />                                                                                                     buffer presence detergent Triton X-100. Treatment <br /> Figure 2      Immunofluorescence staining situ-prepared nuclear matrix                        DNase I extraction presence 0.25 M ammonium <br />                                                                                                     sulphate released majority chromatin-associated <br /> Staining DNA, lamin B, HIRA histones performed HeLa cells extracted CSK          proteins. The remaining material consisted structural and <br /> buffer (upper panels) nuclear matrices prepared situ HeLa cells (lower         additional molecules associated matrix. The efficiency of <br /> panels).                                                                                            chromatin digestion assessed disappearance the <br />                                                                                                     nuclear staining produced DNA stain DAPI (Figure 2, <br />                                                                                                     compare upper- lower-left images). Double immuno- <br /> that partitioned nuclear matrix clearly intermediate                                       fluorescence staining experiments performed using sim- <br /> (approx. 10&#8211;20 %) observed soluble                                     ultaneously anti-lamin B monoclonal antibody anti- <br /> PARP enzyme (about 2 % matrix-associated) integral                                          HIRA affinity-purified 1455 antiserum. As expected, lamin <br /> component matrix, lamin B (close 100 % matrix-                                            B signal resisted extraction (Figure 2, compare upper and <br /> associated).                                                                                        lower images). When fields observed HIRA <br />   We sought confirmation presence HIRA                                        staining, apparent staining been <br /> nuclear matrix using independent procedure different                                         retained matrix preparation procedure. Since <br />  <br /> # 2001 Biochemical Society <br />                                                                                         Subcellular localization mitotic phosphorylation HIRA protein                             451 <br />  <br />  <br />  <br />  <br /> Figure 4     Mitotic phosphorylation HIRA <br /> Nuclear extracts (35 &#181;g lane) prepared nocodazole-arrested HeLa cells incubated (1 h, 37 mC) absence (lanes 2) presence (lanes 3&#8211;6) various concentrations CIP <br /> as indicated. Lanes 6 loaded nuclear extracts treated CIP presence 50 mM phosphatase inhibitor sodium EDTA. (A) Immunoblot analysis using affinity-purified <br /> 1455 antibody ; (B) membrane stained Coomassie Brilliant Blue. <br />  <br />  <br />  <br /> DAPI staining left preparation totally unlabelled                                       electrophoretic migration pattern (Figure 3B, As). In <br /> (Figure 2, lower-left image), appeared unlikely                                      cells, 300 mM salt necessary efficient HIRA <br /> HIRA immunofluorescence signal proportion                                    extraction, expected chromatin-associated protein. A <br /> of HIRA molecules associated chromatin. Indeed,                                       higher salt concentration (600 mM) did extract HIRA <br /> under conditions, extremely intense histone signal                                  protein. In corresponding extracts demecolcine-treated <br /> was virtually abrogated (Figure 2, lower-right panel), consistent                                cells, novel band detected addition normally <br /> with notion small fraction, 5 %,                                  migrating species (Figure 3B, Dc). This slower form HIRA <br /> of total nuclear histones resists extraction nuclear                                      readily extractable, seen lane loaded the <br /> matrix preparation [28]. Together, biochemical immuno-                                   150 mM salt extract. In 300 600 mM salt, HIRA molecules <br /> fluorescence <span id='am-151' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-152' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-153' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-154' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-155'  ></span>data</span> demonstrated significant proportion                                     distributed equally normally migrating the <br /> of mammalian HIRA molecules tightly associated                                          slower forms. An increase salt concentration did not <br /> the nuclear matrix.                                                                              yield additional HIRA material. <br />  <br /> The HIRA protein excluded mitotic chromosomes and <br /> more readily extracted synchronized mitotic cells                                           The mitosis-specific form HIRA hyperphosphorylated <br /> We reported previously [24] (see Figure 2)                                     We investigated retarded migration mitotic <br /> interphase cells, HIRA molecules distributed numerous                                     HIRA molecules phosphorylation. Proteins extracted <br /> foci scattered nucleus, exception                                     mitotically enriched HeLa cells submitted Western- <br /> nucleoli. Figure 3(A) displays interphase mitotic cells stained                              blot analysis (Figure 4A). The retarded HIRA <br /> for nuclear DNA HIRA. It seen HIRA                                           molecules somewhat varied experiment other, <br /> distribution pattern differed cell subsets,                                  probably varying proportions (from 70 % 90 %) cells <br /> HIRA signal clearly excluded condensed chromo-                                        blocked mitosis different experiments ; present case, <br /> somes mitotic cells. Since phosphorylation events                                   virtually HIRA molecules extract migrated the <br /> implicated exclusion condensed mitotic chromosomes                                   retarded pattern (Figure 4A, lane 1). Treatment CIP caused <br /> for nuclear proteins, including transcription factors [34]                                 HIRA molecules migrate quickly (Figure 4A, lanes 3&#8211;5) <br /> and members chromatin-remodelling complexes [35],                                     compared untreated (Figure 4A, lane 1) mock-treated <br /> asked case HIRA. The classical                                (Figure 4A, lane 2) extracts, effect abrogated the <br /> procedure prepare nuclei applied HeLa cell population                                presence phosphatase inhibitor sodium EDTA (Figure 4A, <br /> enriched mitotic cells following treatment demecolcine,                                  lane 6). Similar results obtained protein extracts were <br /> followed protein extraction presence different salt                                 submitted action &#955;-phosphatase (results shown). <br /> concentrations. Western-blot analysis various protein                                     Since migration pattern proteins various <br /> fractions extracted demecolcine- mock-treated cells                                     extracts affected, revealed Coomassie Blue staining <br /> was performed reveal HIRA. HIRA molecules isolated                                       (Figure 4B), HIRA migration differences an <br /> asynchronous control cells behaved homogeneously regard                                     electrophoretic artefact. Together, results indicated that <br />  <br />                                                                                                                                                                # 2001 Biochemical Society <br />  452                F. De Lucia others <br />  <br />  <br />  <br />  <br /> Figure 5      The mitotic mobility shift affects HIRA molecules associated nuclear matrix <br /> Fractions prepared loaded gel Figure 1. The left- right-hand panels correspond control floating HeLa cells harvested nocodazole-treated cultures, respectively. <br /> A FACS analysis populations stained DNA content using propidium iodide shown corresponding blots. Equal amounts proteins (35 &#181;g) loaded lane <br /> for immunoblot analysis using affinity-purified 1455 anti-HIRA antibody. The short long arrows point quickly slowly migrating forms HIRA, respectively. <br />  <br />  <br /> mitotically enriched cells contain hyperphosphorylated forms                                      fications affect HIRA time mitosis normally <br /> HIRA biochemical features include increased extract-                                           cycling population. <br /> ability lower-salt buffer (150 mM NaCl) decreased electro- <br /> phoretic mobility. <br />    These experiments performed cells incubated for <br />                                                                                                      The hyperphosphorylated form HIRA associated the <br /> 16 h presence nocodazole. Although long-established <br />                                                                                                      nuclear matrix <br /> procedure obtain large numbers cells blocked mitosis,                                       Having observed differences subnuclear distribution of <br /> this incubation period presence microtubule-                                             HIRA molecules interphase mitosis, light <br /> depolymerizing drug result damaged cells physio-                                      existence hyperphosphorylated forms HIRA in <br /> logically irrelevant hyperphosphorylation HIRA. To address                                        mitotic cells, asked features indicative <br /> this possibility, performed experiments. First,                                        different properties HIRA regard nuclear matrix. <br /> verified nocodazole-treated cells irreversibly                                         Nuclear matrices prepared asynchronous floating <br /> damaged. DNA staining FACS analysis performed                                              (mitotically enriched) HeLa cells obtained nocodazole <br /> replacement nocodazole-containing medium drug-free                                         treatment. As seen FACS analysis displayed Figure 5, <br /> medium indicated 35 % 56 % (compared 3 %                                        greater 80 % cells harvested nocodazole-treated <br /> drug-arrested population) cells G 140 180 min                                     culture DNA content corresponding G \M phases <br />                                                    &quot;                                                                                                        # <br /> after nocodazole removal, respectively. Secondly, repeated                                    cell cycle, compared 11 % control asyn- <br /> nocodazole treatment using shorter incubation periods lower                                       chronous population. Upon microscopic examination, pro- <br /> drug concentrations. The gel retardation obtained                                       portion mitotic cells 3.6 % population, <br /> HIRA molecules 8 h nocodazole treatment                                          compared 72 % former. In control cell extracts <br /> little 10 % usual drug concentration (results shown).                                  (Figure 5, Western blot, left-hand panel), retarded HIRA <br /> A final indication HIRA hyperphosphorylation                                          band observed detectable migration <br /> related mitotis effect nocodazole treatment                                 differences different fractions (Figure 5, left-hand <br /> was provided using classical &#8216; shake-off &#8217; procedure allows                                   panel, lanes 2&#8211;7). HIRA molecules present mitotically enriched <br /> the recuperation mitotically enriched floating cells                                      HeLa cells (Figure 5, Western blot, right-hand panel) also <br /> asynchronous population exponentially growing HeLa cells.                                         distributed soluble matrix-associated <br /> In hands, procedure reproducibly yielded 25&#8211;28 %                                         fractions. However, contrasting asynchronous popu- <br /> mitotic cells, compared approx. 3 % control asyn-                                        lation, migration HIRA molecules varied between <br /> chronous population. Again, similarly retarded HIRA products                                         fractions. Retarded HIRA molecules (Figure 5, right-hand panel, <br /> were detectable corresponding lane gel electro-                                        long arrow) abundant &#8216; nuclear &#8217; fraction (Figure 5, <br /> phoresis (results shown). We conclude                                         right-hand panel, lane 1), accounted than <br /> hyperphosphorylation HIRA molecules occurs mito-                                          half total HIRA signal. It readily apparent vast <br /> tically arrested cells following drug treatment reproduces modi-                                     majority retarded HIRA molecules partitioned the <br />  <br /> # 2001 Biochemical Society <br />                                                                                                Subcellular localization mitotic phosphorylation HIRA protein            453 <br />  <br />                                                                                                       periments performed. In case, radioactive HIRA <br />                                                                                                       band decreased slightly 6 h (Figure 6A, lane 3) the <br />                                                                                                       beginning chase (Figure 6A, lane 2), confirming HIRA <br />                                                                                                       stable protein. Only later time points did intensity of <br />                                                                                                       band clearly diminish (see 24 h time point, Figure 6A, <br />                                                                                                       lane 4). Quantification analyses indicated 25 % initial <br />                                                                                                       radioactive HIRA consistently present material <br />                                                                                                       immunoprecipitated 24 h leading half-life determination <br />                                                                                                       approx. 12 h protein. <br />  <br /> Figure 6 Metabolic labelling pulse&#8211;chase analysis reveals HIRA as <br /> a stable protein                                                                                      DISCUSSION <br /> (A) Autoradiogram (6 h exposure) extracts prepared K562 cells been <br />                                                                                                       A major reorganization chromatin takes place mitosis <br /> metabolically labelled 18 h submitted immunoprecipitation using immunoaffinity-        duplicated chromosomes condensed prior their <br /> purified 484 antiserum 0 (lane 2), 6 (lane 3) 24 (lane 4) h chase. Lane 1         separation incorporation reconstituting nucleus of <br /> loaded material precipitated using irrelevant rabbit antiserum. (B) Materials precipitated    daughter cell. Chromatin-associated proteins protein <br /> with irrelevant antiserum (lane 1) immunoaffinity-purified HIRA-specific 484 antiserum      complexes obviously affected process. Contrasting <br /> (lanes 2&#8211;4) transferred PVDF membranes revealed staining 484. Lanes               situation interphase HIRA molecules are <br /> 2&#8211;4 loaded precipitates obtained 0, 24 36 h beginning chase, <br /> respectively. In panels arrows indicate positions HIRA protein. <br />                                                                                                       scattered nuclear space [24], shown here <br />                                                                                                       histone-binding protein excluded condensed <br />                                                                                                       mitotic chromosomes. In cells blocked mitosis following drug <br />                                                                                                       treatment, electrophoretic migration HIRA retarded <br /> soluble protein subset (Figure 5, right-hand panel, lane 2),                                    phosphorylation events. This post- <br /> totally undetectable matrix preparation (Figure 5, right-                                      translational modification HIRA effect drug <br /> hand panel, lane 7).                                                                                  treatment observed low doses nocodazole, <br />   The observation hyperphosphorylated forms HIRA                                          shorter incubation times non-treated mitotically <br /> detected mitosis excluded nuclear matrix                                          enriched cells obtained shake-off procedure. Hyper- <br /> suggest modification enhances biochemical extract-                                      phosphorylated forms HIRA displayed increased extract- <br /> ability HIRA, probably reflects decreased association                                    ability chromatin, 150 mM NaCl sufficient extract <br /> with chromatin material &#138;i&#138;o.                                                                      significant amounts HIRA protein drug-arrested mitotic <br />                                                                                                       cells, 300 mM necessary non-synchronized cells. <br />                                                                                                          Other nuclear proteins similarly phosphorylated in <br /> The endogenous HIRA long-lived protein                                                           mitosis subsequently degraded totally absent early-G - <br />                                                                                                                                                                         &quot; <br /> Nuclear proteins phosphorylated mitotic phase                                       phase cells [36]. This clearly case HIRA, which <br /> cell cycle targeted degradation immediately fol-                                        remains readily detectable gets progressively dephosphoryl- <br /> lowing mitosis, presumably prevent presence cell                                      ated cells progress G cell cycle <br />                                                                                                                                                &quot; <br /> at untimely stage cell cycle. The finding                                     (results shown). Indeed, pulse&#8211;chase experiments indi- <br /> HIRA protein subject mitotic phosphorylation prompted                                          cated HIRA undergoes slow turnover. The protein not <br /> us explore turnover. Various synchronization protocols                                         actively synthesized takes 16&#8211;18 h labelling &#138;i&#138;o for <br /> revealed lack major quantitative variations amounts                                       sufficient amounts radioactive HIRA detectable <br /> of HIRA protein cell cycle (Y. Le! cluse M.                                        immunoprecipitation autoradiography. The stability of <br /> Lipinski, unpublished work), HIRA rapidly                                           protein reflected approx. 12 h necessary for <br /> synthesized degraded likely stable                                        precipitated HIRA lose half radioactive content the <br /> protein. To answer question, undertook determination                                        chase experiments. In conclusion, HIRA synthesized <br /> of half-life HIRA using classical pulse&#8211;chase approach.                                      degraded actively cell cycle, suggesting its <br /> K562 cells metabolically labelled using $&amp;S followed                                          mitotic phosphorylation affects molecular interactions rather <br /> immunoprecipitation endogenous HIRA. A representative                                              stability. When composition HIRA-containing <br /> autoradiogram shown Figure 6(A). Despite presence                                        high-molecular-mass complex solved (this work progress <br /> numerous cysteine methionine residues amino acid                                           laboratory), interesting examine whether <br /> sequence, long incubation period (16&#8211;18 h) presence                                       HIRA molecular partners vary mitotic interphase <br /> radioactive cysteine methionine required specific                                       cells. <br /> band approx. 112 kDa appear HIRA immuno-                                                    One possible reason HIRA gets dissociated chroma- <br /> precipitates, consistent known migration HIRA [24].                                       tin detrimental to <br /> Thus HIRA protein appears actively synthesized                                       chromosome condensation mitosis-associated <br /> these exponentially growing cells. Probably length                                     chromatin-affecting event. Alternatively, mitotic redistri- <br /> of labelling period, despite extensive preclearing, nu-                                       bution HIRA precede allow retargeting different <br /> merous bands reproducibly visible addition HIRA                                            regions chromatin daughter compared parental cells. <br /> (Figure 6A, arrow), non-specific seen                                           The single HIRA protein higher eukaryotes two <br /> immunoprecipitation carried irrelevant antiserum                                          structural homologues S. cere&#138;isiae, Hir1p Hir2p, define <br /> (Figure 6A, lane 1). A Western-blot analysis performed                                             family nuclear proteins likely function in <br /> similar samples verified position HIRA                                           regulation gene transcription, presumably impact the <br /> migration experimental conditions (Figure 6B).                                         chromatin level, suggested evolutionarily conserved <br /> The experiment demonstrated equivalent amounts                                       histone-binding properties. Studies avian [15] murine [8] <br /> of HIRA protein reproducibly precipitated 36 h                                       models demonstrated HIRA essential normal <br /> the beginning chase (Figure 6B). Three pulse&#8211;chase ex-                                         processes early embryogenesis. We propose way <br />  <br />                                                                                                                                                          # 2001 Biochemical Society <br />  454               F. De Lucia others <br />  <br /> in HIRA important development protein                               8    Scambler, P., Roberts, C., Sutherland, H., Kimber, W., Lui, V., Halford, S., McKie, J., <br /> complexes specify chromatin status genomic regions                                     Snoeren, C. A. S., Lohman, F., Meijers, C. Wynshaw-Boris, A. (1998) Hira, a <br /> targeted sequence-specific DNA-binding transcription factors                                    gene DGS/VCFS region, required normal embryogenesis. Am. J. Hum. <br />                                                                                                    Genet. 63, A7 <br /> such homeodomain protein Pax-3 [25]. A nonapeptide <br />                                                                                               9    Gottlieb, S., Emanuel, B. S., Driscoll, D. A., Sellinger, B., Wang, Z., Roe, B. and <br /> perfectly fitting consensus AT-hook sequences [29]                                      Budarf, M. L. (1997) The DiGeorge syndrome minimal critical region contains a <br /> present evolutionarily conserved HIRA proteins                                         Goosecoid-like (GSCL) homeobox gene expressed early human <br /> higher eukaryotes. AT-hook-containing proteins bind minor                                      development. Am. J. Hum. Genet. 60, 1194&#8211;1201 <br /> groove DNA preference AT-rich sequences                                     10   Carlson, C., Sirotkin, H., <span id='am-105' about='obo:NCBITaxon_442249' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_215788 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_100750 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-106' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-107'  ></span><span id='am-108' property="rdfs:label" content="Pandita" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Pandita</span>, R., Goldberg, R., McKie, J. M., Wadey, R., Patanjali, <br /> without nucleotide specificity. Matrix- scaffold-                                       S. R., Weissman, S. M., Anyane-Yeboa, K., Warburton, D. et al. (1997) Molecular <br /> associated DNA regions (MARs SARs) function                                             <span id='am-144' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> 22q11 deletions 151 velo-cardio-facial syndrome patients. Am. J. <br />                                                                                                    Hum. Genet. 61, 620&#8211;629 <br /> boundary elements transcriptional domains shown <br />                                                                                               11   Lammer, E. J. Opitz, J. M. (1986) The DiGeorge anomaly developmental <br /> to bound AT-hook-containing proteins global roles                                    field defect. Am. J. Med. Genet. suppl. 2, 113&#8211;127 <br /> regulation cellular functions defined lineages [37]. Com-                               12   Lindsay, E., Vitelli, F., Su, H., Morishima, M., Huynh, T., Pramparo, T., Jurecic, V., <br /> ponents yeast [38] mammalian [39] chromatin-remodelling                                     Ogunrinu, G., Sutherland, H. F., Scambler, P. J. et al. (2001) Tbx1 haploinsufficiency <br /> complexes contain AT hooks essential functions.                                  DiGeorge syndrome region causes aortic arch defects mice. Nature (London) <br /> We suggest HIRA utilizes AT hook tether                                        410, 97&#8211;101 <br /> HIRA-containing complex(es) chromatin.                                                     13   Merscher, S., Funke, B., Epstein, J. A., Heyer, J., Puech, A., Lu, M. M., Xavier, R. J., <br />                                                                                                    Demay, M. B., Russell, R. G., Factor, S. et al. (2001) TBX1 responsible for <br />    Using biochemical immunofluorescence techniques <br />                                                                                                    cardiovascular defects velo-cardio-facial/DiGeorge syndrome. Cell 104, 619&#8211;629 <br /> we significant albeit minor number                                   14   Wilming, L. G., Snoeren, C. A. S., Van Rijswijk, A., Grosveld, F. Meijers, C. <br /> nuclear HIRA molecules tightly associated nuclear                                    (1997) The murine homologue HIRA, DiGeorge syndrome candidate gene, is <br /> matrix non-synchronized mitotic cells. From                                      expressed embryonic structures affected human CATCH22 patients. Hum. Mol. <br /> electrophoretic behaviour, particular HIRA molecules                                    Genet. 6, 247&#8211;259 <br /> not subject phosphorylation event modified                               15   Farrell, M. J., Stadt, H., Wallis, K. T., Scambler, P. J., Hixon, R. L., Wolfe, R., <br /> migration biochemical properties extractable HIRA.                                          Leatherbury, L. Kirby, M. L. (1999) HIRA, DiGeorge syndrome candidate gene, <br /> Physically, nuclear matrix provides interfaces at-                                     required cardiac outflow tract septation. Circ. Res. 84, 127&#8211;135 <br />                                                                                               16   Osley, M. A. Lycan, D. (1987) Trans-acting regulatory mutations alter <br /> tachment multi-protein multi-enzyme complexes involved                                      transcription Saccharomyces cerevisiae histone genes. Mol. Cell. Biol. 7, <br /> in DNA RNA metabolism [40] ; functionally, in-                                           4204&#8211;4210 <br /> volved regulation DNA replication [30,41], RNA                                      17   Sherwood, P. W., Tsang, S. V. Osley, M. A. (1993) Characterization HIR1 and <br /> transcription splicing [42,43] cellular differentiation [44].                               HIR2, genes required regulation histone gene transcription in <br /> In regard, HIRA belongs growing group molecules                                       Saccharomyces cerevisiae. Mol. Cell. Biol. 13, 28&#8211;38 <br /> that includes DNA-binding transcription factors [45], ATPases                                 18   Spector, M. S., Raff, A., DeSilva, H., Lee, K. Osley, M. A. (1997) Hir1p and <br /> such mammalian Swi\Snf complexes [46],                                           Hir2p function transcriptional corepressors regulate histone gene transcription in <br />                                                                                                    Saccharomyces cerevisiae cell cycle. Mol. Cell. Biol. 17, 545&#8211;552 <br /> histone acetyltransferases deacetylases [47,48], protein                                  19   Dimova, D., Nackerdien, Z., Furgeson, S., Eguchi, S. Osley, M. A. (1999) A role <br /> kinases [49,50] DNA replication factor PCNA                                                transcriptional repressors targeting yeast Swi/Snf complex. Mol. Cell 4, <br /> (proliferating-cell nuclear antigen) [51]. Those HIRA molecules                                    75&#8211;83 <br /> that remain nuclear matrix non-modified form                               20   Co# te! , J., Quinn, J., Workman, J. L. Peterson, C. L. (1994) Stimulation GAL4 <br /> mitosis likely engage molecular associations different                                   derivative binding nucleosomal DNA yeast SWI/SNF complex. Science 265, <br /> from extractable HIRA. They important                                       53&#8211;60 <br /> in reassembly HIRA-containing multi-protein complexes                                  21   Imbalzano, A. N., Kwon, H., Green, M. R. Kingston, R. E. (1994) Facilitated <br />                                                                                                    binding TATA-binding protein nucleosomal DNA. Nature (London) 370, <br /> following mitosis. These questions clearly additional <br />                                                                                                    481&#8211;485 <br /> studies.                                                                                      22   Wu, C. (1997) Chromatin remodeling control gene expression. J. Biol. <br />                                                                                                    Chem. 272, 28171&#8211;28174 <br /> This work supported grants Human Frontiers Science                    23   Winston, F. Carlson, M. (1992) Yeast SNF/SWI transcriptional activators the <br /> Program Organization Association pour la Recherche sur le Cancer (ARC).                    SPT/SIN chromatin connection. Trends Genet. 8, 387&#8211;391 <br /> S. L. benefitted doctoral fellowships Ministe&apos; la Recherche       24   Lorain, S., Quivy, J.-P., Monier-Gavelle, F., Scamps, C., Le! cluse, Y., Almouzni, G. and <br /> ARC. The expert technical assistance Yann Le! cluse gratefully recognized.                   Lipinski, M. (1998) Core histones HIRIP3, novel histone-binding protein, <br />                                                                                                    directly interact WD repeat protein HIRA. Mol. Cell. Biol. 18, 5546&#8211;5556 <br /> REFERENCES                                                                                    25   Magnaghi, P., Roberts, C., Lorain, S., Lipinski, M. Scambler, P. J. (1998) HIRA, <br />                                                                                                    mammalian homologue Saccharomyces cerevisiae transcriptional co-repressors, <br />  1    Lamour, V., Le! cluse, Y., Desmaze, C., Spector, M., Bodescot, M., Aurias, A., Osley, <br />                                                                                                    interacts Pax3. Nat. Genet. 20, 74&#8211;77 <br />       M. A. Lipinski, M. (1995) A human homolog S. cerevisiae HIR1 HIR2 <br />       transcriptional repressors cloned DiGeorge syndrome critical region.           26   Hall, C., Nelson, D. M., Ye, X., Baker, K., DeCaprio, J. A., Seeholzer, S., Lipinski, M. <br />       Hum. Mol. Genet. 4, 791&#8211;799                                                                  Adams, P. D. (2001) HIRA, human homologue yeast Hir1p Hir2p, a <br />  2    Scamps, C., Lorain, S., Lamour, V. Lipinski, M. (1996) The HIR protein family :          novel cyclin/cdk2 substrate expression blocks S-phase progression. Mol. Cell. <br />       isolation characterization complete murine cDNA. Biochim. Biophys. Acta             Biol. 21, 1854&#8211;1865 <br />       1306, 5&#8211;8                                                                               27   Razin, S. V. (1997) The nuclear matrix spatial organization chromosomal DNA <br />  3    Roberts, C., Daw, S., Halford, S. Scambler, P. J. (1997) Cloning                     domains, R. G. Landes Company, Austin <br />       developmental expression analysis chick Hira (<span id='am-133' about='obo:NCBITaxon_243464' typeof='owl:Thing obo:NCBITaxon_243881 obo:NCBITaxon_94015 obo:NCBITaxon_33208 obo:NCBITaxon_94017 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_94020 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_243329 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-134' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-135' property="rdfs:label" content="Chira" datatype="xsd:string"></span><span id='am-136'  ></span><span id='am-137' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Chira</span>), candidate gene           28   Spector, D. L., Goldman, R. D. Leinwand, L. A. (1998) Cells, Laboratory <br />       DiGeorge syndrome. Hum. Mol. Genet. 6, 237&#8211;246                                               Manual, p. 44.1&#8211;44.8, Cold Spring Harbor Press, Cold Spring Harbor <br />  4    Kirov, N., Shtilbans, A. Rushlow, C. (1998) Isolation characterization     29   Aravind, L. Landsman, D. (1998) AT-hook motifs identified wide variety of <br />       new gene encoding member HIRA family proteins Drosophila                    DNA-binding proteins. Nucleic Acids Res. 26, 4413&#8211;4421 <br />       m <br /> </body></html>