<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>      Rct1, a Nuclear RNA Recognition Motif-Containing Cyclophilin, <br />            Regulates Phosphorylation of the RNA Polymerase II <br />                          C-Terminal Domain <br />   <br />              Phosphorylation of the C-terminal domain (CTD) of RNA polymerase II (RNAP II) is a dynamic process <br />           that regulates transcription and coordinates it with pre-mRNA processing. We show here that Rct1, a <br />           nuclear multidomain cyclophilin from <span id='am-39' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-40' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-42'  ></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Schizosaccharomyces</span> pombe, is encoded by an essential gene that <br />           interacts with the CTD and regulates its phosphorylation in vivo. Downregulation of Rct1 levels results in <br />           increased phosphorylation of the CTD at both Ser2 and Ser5 and in a commensurate decrease in RNAP <br />           II transcription. In contrast, overexpression of Rct1 decreases phosphorylation on both sites. The close <br />           association of Rct1 with transcriptionally active chromatin suggests a role in regulation of RNAP II <br />           transcriptional activity. These <span id='am-125' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-126' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-128'  ></span><span id='am-129' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span>, together with the pleiotropic phenotype upon Rct1 deregulation, <br />           suggest that <span id='am-102' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-103'  ></span><span id='am-104' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-105' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>this</span> multidomain cyclophilin is an important player in maintaining the correct phosphory- <br />           lation code of the CTD and thereby regulating CTD function. <br />  <br />  <br />    Eukaryotic RNA polymerase II (RNAP II), the enzyme                           associates in a phosphorylation-dependent manner directly <br /> responsible for transcription of protein-coding genes, is a                     or indirectly with a variety of pre-mRNA-processing factors, <br /> multisubunit complex. For accurate transcription, RNAP II                       including 5&#11032; capping enzymes, splicing factors, and some <br /> is controlled by many factors through protein-protein inter-                    components of the 3&#11032; end-processing machinery cycle (1, 17, <br /> actions. RNAP II transcripts undergo extensive processing,                      35, 37&#8211;39, 41, 46, 53). <br /> including capping, splicing, and polyadenylation, which are                        Recently, <span id='am-120' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-121' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-123' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-124'  ></span>cis</span>-trans isomerization of prolyl peptide bonds <br /> tightly coordinated with transcription. There is now strong                     in the heptapeptide repeats has been revealed as an addi- <br /> evidence that RNAP II coordinates these processing reac-                        tional level of regulation of CTD structure and function, as <br /> tions and couples them to transcription. Specifically, the                      it has been shown that three CTD-interacting proteins bind <br /> C-terminal domain (CTD) of the largest RNAP II subunit                          to the CTD only in an all-trans proline conformation (10, 31, <br /> has a critical role in targeting pre-mRNA processing factors                    32, 34, 48). One of these proteins is the parvulin-type pep- <br /> to emerging pre-mRNAs. Thus, the CTD coordinates tran-                          tidyl-prolyl cis-trans isomerase (PPIase) Pin1, which has pre- <br /> scription with pre-mRNA processing and is therefore sub-                        viously been shown to be important for keeping the correct <br /> ject to tight regulation during the transcription cycle (1, 17,                 CTD phosphorylation status in mouse and human cells (51, <br /> 35, 37, 39, 41, 46, 53).                                                        52). In addition, mutation of ESS1, a <span id='am-23' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-24' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-28'  ></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span>Saccharomyces</span> cerevi- <br />    In all eukaryotes, the CTD consists of variable numbers of                   siae homolog of Pin1, resulted in defects in pre-mRNA 3&#11032; <br /> YSPTSPS heptapeptide repeats. During the transcription                          end formation (14). These data suggest that enzymes cata- <br /> cycle, the CTD undergoes dynamic phosphorylation and <br />                                                                                 lyzing cis-trans isomerization of proline peptide bonds might <br /> dephosphorylation on Ser2 and Ser5. Transcription initia- <br />                                                                                 be important for correct CTD conformation and phosphor- <br /> tion requires an unphosphorylated CTD. Following initia- <br />                                                                                 ylation and consequently for its recognition by diverse pre- <br /> tion, the CTD becomes phosphorylated at Ser5, a modifica- <br />                                                                                 mRNA-processing factors. Thus, generation of the CTD <br /> tion that is necessary to initiate transcript elongation. <br />                                                                                 code requires not only phosphorylation but also cis-trans <br /> Finally, extensive phosphorylation at Ser2 is accompanied <br />                                                                                 isomerization of the two prolines that follow Ser2 and Ser5 <br /> by transcript elongation and termination. In addition to <br />                                                                                 in the heptapeptide repeats (3, 24, 32, 34). <br /> regulating RNAP II transcription, the CTD phosphorylation <br />                                                                                    The majority of cyclophilins (PPIases) are small proteins <br /> status is also important for coordination of transcription <br /> with pre-mRNA processing, in particular for recruitment of                      containing only a PPIase domain of about 120 amino acids. <br /> RNA-processing factors to nascent transcripts. The CTD                          However, several multidomain cyclophilins from different or- <br />                                                                                 ganisms have been described as well (36, 42). The most com- <br />                                                                                 plex multidomain cyclophilins characterized thus far are <br />    * Corresponding author. Mailing address: Max F. Perutz Laborato-             <span id='am-16' about='obo:NCBITaxon_3701' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_980083 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Arabidopsis" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Cardaminopsis" datatype="xsd:string"></span><span id='am-21'  ></span><span id='am-22' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45250" datatype="xsd:string"></span>Arabidopsis</span> AtCyp59 (12) and its orthologue from <span id='am-90' about='obo:NCBITaxon_5884' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_431838 obo:NCBITaxon_340080 obo:NCBITaxon_33825 obo:NCBITaxon_2759 obo:NCBITaxon_5878 obo:NCBITaxon_33630 obo:NCBITaxon_6020'><span id='am-91'  ></span><span id='am-92' property="rdfs:label" content="Paramecium" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasDbXref" content="GC_ID:6" datatype="xsd:string"></span>Paramecium</span> <br /> ries, Medical University of Vienna, Department of Medical Biochem-              tetraurelia, Kin241p (22). They are characterized by a unique <br /> istry, Bohrgasse 9/3, A-1030 Vienna, Austria. Phone: 43 1 4277 61642.           domain organization, consisting of a PPIase domain at the N <br /> Fax: 43 1 4277 9616. E-mail: zdravko.lorkovic@univie.ac.at. <br />                                                                                 terminus, followed by an RNA recognition motif (RRM) and <br />    &#8224; Supplemental material for this article may be found at http://mcb <br /> .asm.org/.                                                                      a C-terminal domain enriched in charged amino acids and <br />    &#17188; <br />      Published ahead of print on 5 March 2007.                                  serines or RS/RD dipeptide repeats. Homologous proteins <br />  <br />                                                                          3601 <br />  3602       GULLEROVA ET AL.                                                                                                                         MOL. CELL. BIOL. <br />  <br />  <br /> have been found in the majority of eukaryotes, although Sac-                          manipulator. Spores were grown on yeast extract-supplement (YES) medium for <br /> charomyces cerevisiae and some other <span id='am-1' about='obo:NCBITaxon_147537' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_716545 obo:NCBITaxon_2759'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Saccharomycotina" datatype="xsd:string"></span><span id='am-5'  ></span>Saccharomycotina</span> spe-                            4 days at 32&#176;C. <br />                                                                                          pMG was derived from pREP3X (11) by inserting a kanamycin resistance <br /> cies do not encode this kind of protein (12, 22). Kin241p was                         cassette, cut out from pSMRG2&#11001; (25), into the SacI site. The plasmid expressing <br /> identified in <span id='am-95' about='obo:NCBITaxon_5888' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_431838 obo:NCBITaxon_340080 obo:NCBITaxon_33825 obo:NCBITaxon_5884 obo:NCBITaxon_2759 obo:NCBITaxon_5878 obo:NCBITaxon_33630 obo:NCBITaxon_6020'><span id='am-96' property="rdfs:label" content="Paramecium tetraurelia" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasRelatedSynonym" content="Paramecium aurelia syngen 4" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasAlternativeId" content="NCBITaxon:42403" datatype="xsd:string"></span><span id='am-100'  ></span><span id='am-101' property="oboInOwl:hasDbXref" content="GC_ID:6" datatype="xsd:string"></span>Paramecium tetraurelia</span> as a protein involved in                         Rct1 fused to a hemagglutinin (HA) tag was constructed by cloning the rct1 gene, <br /> cell morphogenesis (reference 22 and references therein).                             amplified by PCR with the oligonucleotides 5&#11032;-GTCAGTCTCGAGATGTCTG <br /> AtCyp59 is a nuclear protein and was identified in a yeast                            TACTAATTGAA-3&#11032; (forward) and 5&#11032;-GTCAGTGTCGACTCATGCGTAGTC <br />                                                                                       AGGCACATCATACGGATATCGATATCTATCATCTCTATAACG-3&#11032; (reverse). <br /> two-<span id='am-130' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-131' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-133'  ></span><span id='am-134' property="rdfs:label" content="hybrid" datatype="xsd:string"></span>hybrid</span> screen as an interacting partner of SR proteins, an <br />                                                                                       The PCR fragment was ligated into pMG opened with XhoI/SalI, resulting in <br /> important <span id='am-53' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> of splicing regulators (43). As it also inter-                       pMG1. <br /> acted with the CTD of RNAP II, a function for this protein at                            To generate different cell strains with the rct1-HA gene expressed from the <br /> the interface between transcription and pre-mRNA splicing                             plasmid, WT and rct1&#11001;/&#11002; cells were transformed with pMG1. The haploid <br /> was proposed (12). Interestingly, Arabidopsis cells ectopically                       rct1&#9004;pMG1 strain (h&#11002; leu&#11001; ura&#11001; ade6-704) was obtained from rct1&#11001;/&#11002;pMG1 cells <br />                                                                                       after tetrad dissection. <br /> expressing AtCyp59 are characterized by very slow growth,                                Conditions for PCR genotyping. Loopfuls of freshly growing cells were trans- <br /> presumably due to a decreased phosphorylation of the RNAP                             ferred into Eppendorf tubes and boiled in a microwave for 3 min. Cells were then <br /> II CTD (12). As no Arabidopsis mutant in the gene encoding                            resuspended in 50 &#9262;l of sterile water and used as a template for PCR under the <br /> AtCyp59 exists, and as no homologous protein is present in S.                         following conditions: 95&#176;C for 3 min; 30 cycles of 94&#176;C for 45 s, 55&#176;C for 45 s, and <br />                                                                                       72&#176;C for 1 min; and once at 72&#176;C for 5 min. The presence of WT rct1 and <br /> cerevisiae, we switched to an alternative genetically treatable <br />                                                                                       disrupted rct1&#9004;::ClonNat alleles was checked by PCR by using the oligonucleo- <br /> organism, <span id='am-44' about='obo:NCBITaxon_4896' typeof='owl:Thing obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_1 obo:NCBITaxon_147554 obo:NCBITaxon_34346 obo:NCBITaxon_4890 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4895 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-45'  ></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Schizosaccharomyces pombeP" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="fission yeast" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="Schizosaccharomyces pombe" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Schizosaccharomyces malidevorans" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45042" datatype="xsd:string"></span>Schizosaccharomyces pombe</span>. The S. pombe homolog                             tide pairs 5&#11032;-GAGGCAGAAGCAGAGGCTGTTACA-3&#11032; (forward)&#8211;5&#11032;-AACGTG <br /> is 49% identical and 66% similar to AtCyp59, with the same                            CCGCATTTATGGAG-3&#11032; (reverse) and 5&#11032;-GCGTGGGGACAATTCAACGC-3&#11032; <br /> domain organization as AtCyp59 (see Fig. 1A) except for a Zn                          (forward)&#8211;5&#11032;-AACGTGCCGCATTTATGGAG-3&#11032; (reverse), respectively. The <br /> knuckle which is located downstream of the RRM in all plant                           presence of the pMG1Rct1-HA construct in cells was analyzed by PCR by using <br />                                                                                       the oligonucleotides 5&#11032;-GGAATCCGATTGTCATTCGGC-3&#11032; (forward) and 5&#11032;- <br /> homologs (12). This sequence conservation also suggests a                             CCCGGGGATCCTCTAGAGTC-3&#11032; (reverse). <br /> highly conserved function.                                                               Nuclear transcription run-on analysis. Gene Screen Plus membrane (Perkin- <br />    Here, we show that Rct1 (for &#8220;RRM-containing cyclophilin                           Elmer) was soaked in 1&#11003; SSC (1&#11003; SSC is 0.15 M NaCl plus 0.015 M sodium <br /> regulating transcription&#8221;), the S. pombe ortholog of Arabidop-                        citrate) and fixed in a MINIfold slot blot apparatus (Schleicher and Schuell) to <br />                                                                                       apply the DNA. Ten micrograms of double-stranded DNA prepared by PCR was <br /> sis AtCyp59, is encoded by an essential gene. Reduced levels of <br />                                                                                       denatured and applied per slot. DNA was fixed to membranes by UV cross- <br /> Rct1 in rct1 heterozygous cells result in strong growth and                           linking in a Stratalinker 2400 (Stratagene). Membranes were prehybridized in <br /> morphological defects and have an impact on meiotic differ-                           hybridization buffer (10 mM Tris-HCl [pH 7.5], 250 &#9262;g tRNA, 10&#11003; Denhardt&#8217;s <br /> entiation. Furthermore, phosphorylation of the CTD is in-                             solution [0.2% bovine serum albumin, 0.2% polyvinylpyrrolidone {molecular <br /> creased and ongoing transcription of RNAP II is reduced.                              weight, 40,000}, 0.2% Ficoll 400], 0.5% nonfat dry milk, 0.3 M NaCl, 1% sodium <br />                                                                                       dodecyl sulfate [SDS], 10 mM EDTA) for 2 h at 60&#176;C. S. pombe WT and Rct1&#11001;/&#11002; <br /> Most of the observed phenotypes are rescued by complemen-                             cells were grown in YES medium until reaching exponential phase. The optical <br /> tation assays. In addition, higher levels of Rct1 resulted in                         density at 600 nm (OD600) was measured, and the same amount of cells was <br /> decreased phosphorylation of the CTD and again in reduced                             harvested by centrifugation at 4,000 rpm for 5 min at room temperature. Pellets <br /> growth. Its close association with the active transcription com-                      were washed with 1 ml of ice-cold TMN buffer (10 mM Tris-acetate [pH 7.0], 12 <br />                                                                                       mM MgCl2, 50 mM NH4Cl) and resuspended in 0.5 ml of TMN buffer and <br /> plex places this essential protein at a central position in regu- <br />                                                                                       equilibrated for 10 min on ice. TMN buffer was then removed, and pellets were <br /> lating CTD phosphorylation throughout the transcription cy-                           resuspended in 950 &#9262;l of cold water and 50 &#9262;l of 10% sodium sarcosyl and <br /> cle. Together with recent structural data on CTD-interacting                          incubated on ice for an additional 20 min. Detergent was removed, and perme- <br /> proteins, our data emphasize the importance of PPIases in the                         abilized cells were resuspended in 0.5 ml of run-on buffer (50 mM Tris-HCl [pH <br /> formation of the CTD code.                                                            7.9], 80 mM MgCl2, 500 mM KCl, 1 mM dithiothreitol [DTT], 1 mM ATP, 0.5 <br />                                                                                       mM GTP, 0.5 mM CTP, 100 units of RNasin, and 100 &#9262;Ci of [32P]UTP). The <br />                                                                                       protocol used can be found at http://www.hsph.harvard.edu/wolflab/Protocols <br />                                                                                       /Protocols/Fission%20Yeast/Nurse%20Lab%20Manual.htm. Cells were incu- <br />                         MATERIALS AND METHODS <br />                                                                                       bated at 30&#176;C for 10 min and washed with TMN buffer, and total RNA was <br />    S. pombe strains and handling of cells. The diploid S. pombe strain P136           isolated by using TRIzol reagent (Invitrogen). Isolated RNA was dissolved in 100 <br /> (h&#11001;/h&#11002; leu1-32/leu&#11001; ura4-27/ura&#11001; ade6-704/ade&#11001;) was used as the wild-type             &#9262;l of Tris-EDTA (TE), denatured for 3 min at 95&#176;C, and added to the prehy- <br /> (WT) strain for all experiments. If not stated otherwise, all WT strains (WT,         bridized membrane in 3 ml of hybridization buffer. Hybridization was performed <br /> WTpMG and WTpMG1) used are diploid. Media, growth conditions, and standard            at 60&#176;C for 2 days. Membranes were then washed twice with 2&#11003; SSC&#8211;1% SDS for <br /> genetic methods used throughout were as described previously (33; http://www-rcf      10 min at 60&#176;C and exposed to MR Kodak films. In addition, blots were analyzed <br /> .usc.edu/&#11011;forsburg/). Geneticin (G418; Sigma) and ClonNat (BioAgents) were            by PhosphorImaging and quantified by ImageQuant, version 5.1. A list of genes <br /> used at final concentrations of 155 and 100 &#9262;g/ml, respectively. The nmt1 pro-        analyzed by nuclear transcription run-on and oligonucleotides used for amplifi- <br /> moter was repressed by the addition of thiamine (final concentration, 100 &#9262;g/ml).     cation of corresponding DNA fragments, as well as for reverse transcriptase PCR <br />    Construction of rct1 knockout allele and <span id='am-135' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-136' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-138' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>plasmids</span> and generation of rct1&#11545;/&#11545;pMG1,   (RT-PCR) (see below), is available in Table S1 in the supplemental material. <br /> rct1&#11545;/&#11546;pMG1, and rct1&#9004;pMG1 strains. Fragments corresponding to the 5&#11032; and 3&#11032;          Fragment sizes were 400 to 500 nucleotides (nt) and originated from the middle <br /> untranslated regions (UTR) of the rct1 gene were amplified by PCR. For the 5&#11032;         of the coding region. In case of U1 snRNA, U6 snRNA, and tRNASer, amplified <br /> UTR, oligonucleotides 5&#11032;-GACTAGGGATCCTCTAGAAACGAATGTATCA                              DNA fragments cover the entire sequences of the mature RNAs. <br /> CATTGC-3&#11032; (forward) and 5&#11032;-GACTAGAAGCTTTTTCAATTAGTACAGAC                                 RNA isolation, RT-PCR, and real-time PCR. Total cellular RNA from S. <br /> ATG-3&#11032; (reverse), which introduce a BamHI and a HindIII site, respectively,           pombe cells in exponential phase was isolated with TRIzol (Gibco). RNA was <br /> were used. For the 3&#11032; UTR, oligonucleotides 5&#11032;-GACTAGCTGCAGCATCGT                     dissolved in the proper amount of sterile water and treated with RNase-free <br /> GACGATCAAAGTTCA-3&#11032; (forward) and 5&#11032;-GACTAGTCTAGACAAGTAT                               DNase (Promega) for 30 min at 37&#176;C. One microgram of total RNA was used for <br /> AAGCAATGTAGATTTC-3&#11032; (reverse), which introduce a PstI and an XbaI site,               reverse transcription, which was performed according to the manufacturer&#8217;s <br /> respectively, were used. PCR fragments were cloned into pClonNat1. The                instructions (Reverse Transcription System; Promega) by using oligo(dT)15 <br /> pClonNat15&#11032;UTR3&#11032;UTR construct was then linearized with XbaI and trans-                primer. cDNA was diluted to 100 &#9262;l with TE buffer, and 10 &#9262;l was used for PCR. <br /> formed into WT cells by the lithium acetate method. Correct integration of            Conditions for PCRs were 1 cycle at 95&#176;C for 3 min; 30 cycles at 94&#176;C for 1 min, <br /> pClonNat15&#11032;UTR3&#11032;UTR into the rct1 gene was analyzed by Southern blotting.             50&#176;C for 1 min, and 72&#176;C for 1 min; and 1 cycle at 72&#176;C for 5 min. Oligonucleo- <br /> Heterozygous rct1&#11001;/&#11002; cells were grown on malt extract sporulation medium              tides used for amplification of corresponding DNA fragments (genes 1 to 10 in <br /> plates for 3 days at 26&#176;C, and tetrads were dissected with a Singer micro-            Fig. 5B) are available in Table S1 in the supplemental material. Conditions for <br />  VOL. 27, 2007                                                         CYCLOPHILIN REGULATES RNAP II CTD PHOSPHORYLATION                                       3603 <br />  <br /> RT-PCR amplification of meiotically induced genes were exactly as described          PLRSPSPTPEQC-COOH) and P2 (H2N-CERRYRYDRRYRDDRYR-COOH) <br /> previously (2). Oligonucleotides used for amplification of meiosis-unrelated         (Eurogentec). Antibodies were affinity purified on a SulfoLink (Pierce) column <br /> genes nak1, (AF091345), hob1 (AF275637), and wsp1 (AF038575) were as fol-            coupled with both peptides. After purification, antibodies were concentrated to <br /> lows: nak1for, 5&#11032;-CTGGAAATGTCAAGCTATGTG-3&#11032;; nak1rev, 5&#11032;-CCTGCA                       1 &#9262;g/&#9262;l and were used at 1:1,000 dilutions. Secondary antibodies, goat anti-rat <br /> AACAAGAGGCAATG-3&#11032;; hob1for, 5&#11032;-CAAAGCGTGAAGAAGCAGCA-3&#11032;;                              (Sigma-Aldrich) and goat anti-mouse (Bio-Rad) immunoglobulin G (IgG) and <br /> hob1rev, 5&#11032;-ACCTTGTGCACCGTTAAGTC-3&#11032;; wsp1for, 5&#11032;-CGATAGCTGA                          goat anti-mouse (Sigma-Aldrich) IgM, all conjugated with horseradish peroxi- <br /> ATTGCCTCAAC-3&#11032;; wsp1rev, 5&#11032;-ATCGTCGTCTTCATCCTCTTC-3&#11032;. Condi-                         dase, were used at 1:10,000 dilutions. A chemiluminescence kit (Amersham <br /> tions for PCRs were 1 cycle at 95&#176;C for 3 min; 30 cycles at 94&#176;C for 1 min, 52&#176;C     Pharmacia Biotech) was used for developing the blots. <br /> for 1 min, and 72&#176;C for 1 min; and 1 cycle at 72&#176;C for 5 min. PCR products were <br /> analyzed on 1.5 to 2% agarose gels. <br />    Real-time PCRs were done with the Corbett Research Rotor-Gene GG-3000 <br />                                                                                                                      RESULTS <br /> machine. All samples were run in triplicate to ensure accuracy of the data. PCR of <br /> 45 cycles was done with QuantiTect SYBR green PCR Master Mix (QIAGEN), 2 &#9262;l             Rct1, encoding the Schizosaccharomyces pombe ortholog of <br /> of precipitated DNA, and 1 &#9262;M oligonucleotides. Cycling parameters were 95&#176;C for <br /> 15 min followed by 45 cycles of 94&#176;C for 2 s, 55&#176;C for 11 s, and 68&#176;C for 20 s. <br />                                                                                      AtCyp59, is an essential gene. To analyze knockout mutants of <br /> Oligonucleotides used were as follows: spo4PROfw, 5&#11032;-TGGTTATGACAGGTGT                rct1, we performed rct1 gene disruption in a diploid S. pombe <br /> GCTG-3&#11032;; spo4PROrev, 5&#11032;-CCATGATCATCATTAAACGG-3&#11032;; spo4ORFfw, 5&#11032;-                      strain, as confirmed by Southern blotting and by PCR geno- <br /> ATGCTACATGAACTACGAGG-3&#11032;; spo4ORFrev, 5&#11032;-GGGTGTAAGGATTCC                              typing (data not shown). Integration of pClonNat, containing a <br /> AAGCA-3&#11032;; spo4TERfw, 5&#11032;-TTGTATCGATTAACTCTTAA-3&#11032;; spo4TERrev, 5&#11032;-                     selectable antibiotic gene, resulted in heterozygous cells con- <br /> ATGACTTGACGCTACAGGCAT-3&#11032;; meu13PROfw, 5&#11032;-CCCTTCGCAAGGTTG <br /> GAAA-3&#11032;; meu13PROrev, 5&#11032;-ACTAAGGAAGAAAATCCATG-3&#11032;; meu13ORFfw, <br />                                                                                      taining one rct1&#9004;::ClonNat and one rct1 allele. These cells are <br /> 5&#11032;-GCTTTAAATAACTCACTCAG-3&#11032;; meu13ORFrev, 5&#11032;-TTCCAAGGAATCC                            hereafter referred to as heterozygous rct1&#11001;/&#11002; or short rct1&#11001;/&#11002; <br /> GTAA-3&#11032;; meu13TERfw, 5&#11032;-GCAGACAACTGCTCTAATAT-3&#11032;; meu13TERrev,                        cells. Tetrad dissection from rct1&#11001;/&#11002; heterozygous diploid cells <br /> 5&#11032;-TCATATCCGTTCCGAAATT-3&#11032;; mfr1PROfw, 5&#11032;-GTGGTCAACTGATCAT                            resulted in two growing spores (Fig. 1B). PCR genotyping <br /> TGTT-3&#11032;; mfr1PROrev, 5&#11032;-TGAAACCAGGTGGCTTGAAA-3&#11032;; mfr1ORFfw, 5&#11032;- <br />                                                                                      revealed the presence of only a WT allele in all growing spores. <br /> TAATGTCTTGGCAGTCGGAC-3&#11032;; mfr1ORFrev, 5&#11032;-TTTCGTAGACTCGATA <br /> TCCC-3&#11032;; mfr1TERfw, 5&#11032;-CAATCATCTCAACCCCATCA-3&#11032;; mfr1TERrev, 5&#11032;-A                     In addition, none of the viable spores was resistant to ClonNat, <br /> ATACAGCCTCAAGCCTGAA-3&#11032;; mei2PROfw, 5&#11032;-CTCCTCCTCTAACGTTTG                             further indicating that they were WT and that rct1 is an essen- <br /> TT-3&#11032;; mei2PROrev, 5&#11032;-ACAATAAAGCTGGCCGATAA-3&#11032;; mei2ORFfw, 5&#11032;-CC                      tial gene. Microscopic analysis of nongrowing spores showed <br /> GCTAAATCACTGCGATCT-3&#11032;; mei2ORFrev, 5&#11032;-GTGGACCGAACGTCTGA                              that spores deleted for rct1 germinate but cell division stops <br /> AGT-3&#11032;; mei2TERfw, 5&#11032;-CGAAAGAGATCCTGTATTG-3&#11032;; mei2TERrev, 5&#11032;-GA <br /> TACTCAGAGTGAAGTTGA-3&#11032;; act1PROfw, 5&#11032;-GGTGGTATGAAGCCGTTG <br />                                                                                      after few generations. We failed to grow these cells even at <br /> ATTAC-3&#11032;; act1PROrev, 5&#11032;-TTCTGCCGTGAAGTGCTA-3&#11032;; act1ORFfw, 5&#11032;-TG                     different temperatures or in liquid medium (data not shown). <br /> GGAACAGTGTGGGTAACA-3&#11032;; act1ORFrev, 5&#11032;-AGCACCCTTGCTTGTTGA                             Together, these results indicate that rct1 disruption is lethal <br /> CT-3&#11032;; act1TERfw, 5&#11032;-CCGGACTCGAGAAGAAACAT-3&#11032;; act1TERrev, 5&#11032;-AAC                     and not temperature sensitive. <br /> CACCTTTTTCCGCTCTT-3&#11032;. Fluorescence intensities were plotted against the <br />                                                                                         The cellular localization of Rct1 was analyzed by indirect <br /> number of cycles by using an algorithm provided by the manufacturer. <br />    Cell wall and actin staining, immunofluorescence, and microscopy. Cell wall       immunofluorescence in rct1&#11001;/&#11002; or rct1&#9004; strains harboring a <br /> growth was analyzed by staining living cells with aniline blue. Actin staining was   plasmid expressing Rct1 fused to a HA tag (pMG1; see below). <br /> performed as described (5, 6) by using <span id='am-6' about='obo:NCBITaxon_149625' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_163739 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-7' property="rdfs:label" content="Alexa" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10'  ></span>Alexa</span> Fluor 488-phalloidin (Molecular         Cells were grown for 18 h in minimal medium (EMM) to <br /> Probes). Immunofluorescence was performed exactly as described previously            derepress the nmt1 promoter driving transcription of Rct1-HA <br /> (40). Rat anti-HA monoclonal antibody (3F10; Roche) was used at a 1:200 <br /> dilution. Secondary antibody was goat anti-rat antibody&#8211;Alexa Fluor 563 (Mo- <br />                                                                                      from the plasmid, followed by fixation and indirect immuno- <br /> lecular Probes) at a 1:200 dilution. DNA was visualized by Hoechst 33342             fluorescence with anti-HA antibody. As shown in Fig. 1C <br /> (Molecular Probes) staining. If not stated otherwise, cells were grown until         (shown are only rct1&#9004;pMG1 cells), anti-HA antibody detected <br /> mid-exponential phase in YES and EMM media and analyzed by microscopy                Rct1-HA only in the nucleus. Staining of fixed cells with <br /> (Axioplan epifluorescence microscope; Zeiss) with differential interference con- <br />                                                                                      Hoechst showed that the Hoechst and Alexa 563 signals are <br /> trast (DIC) optics and 63&#11003; oil objective. Images were acquired by a charge- <br /> coupled-device (CCD) <span id='am-107' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-108' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-110' property="rdfs:label" content="Camera" datatype="xsd:string"></span><span id='am-111'  ></span>camera</span> and further processed with Adobe Photoshop.              overlapping (Fig. 1C). Analysis of mitotic cells revealed that <br />    Immunoprecipitation. rct1&#11001;/&#11002;pMG1 cells were grown in 25 ml of EMM for             Rct1 enters new nuclei very rapidly after division, prior to <br /> 15 h, collected by centrifugation, and washed once with ice-cold water. Cells were   septum formation (Fig. 1C, two lower panels). <br /> resuspended in 500 &#9262;l of PEB400 buffer (50 mM HEPES-KOH [pH 7.9], 400                   Disruption of one rct1 allele results in growth, morpholog- <br /> mM KCl, 2.5 mM MgCl2, 1 mM EDTA, 1 mM DTT, 0.1% Triton X-100) (26), <br />                                                                                      ical, and meiotic defects that are rescued by episomal expres- <br /> supplemented with EDTA-free protease inhibitor cocktail (Roche). Two cell <br /> volumes of glass beads were added, and cells were vortexed three times for 15 s.     sion of Rct1. Because we observed that rct1&#11001;/&#11002; cells grow more <br /> Finally, cells were sonicated once and subsequently centrifuged at 4&#176;C for 15 min    slowly than do WT cells, we decided to analyze rct1&#11001;/&#11002; cells <br /> at 13,000 rpm in an Eppendorf centrifuge. Supernatant was mixed with 500 &#9262;l of       more thoroughly. Microscopic analyses revealed that rct1&#11001;/&#11002; <br /> PEB without KCl, and immunoprecipitation was done with anti-HA antibody, as          cells are much shorter and slightly thicker than WT cells. The <br /> described previously (26). <br />    Chromatin immunoprecipitation. Chromatin immunoprecipitation experi- <br />                                                                                      same phenotype was observed with cells grown on rich medium <br /> ments were done as described previously (47). Precipitated DNA was resus-            (YES) plates or in liquid YES medium (Fig. 1D). Growth <br /> pended in 100 &#9262;l of TE buffer, and 10 &#9262;l was used for PCR. Conditions for PCRs       analysis in YES medium revealed that rct1&#11001;/&#11002; cells grow con- <br /> were 1 cycle at 95&#176;C for 3 min; 31 cycles at 94&#176;C for 1 min, 52&#176;C for 1 min, and     siderably more slowly than do WT cells (Fig. 1E). Growth rates <br /> 72&#176;C for 1 min; and 1 cycle at 72&#176;C for 5 min. PCR products were analyzed on <br />                                                                                      in EMM were also affected, though not to the same extent as <br /> 1.5 to 2% agarose gels. Oligonucleotide sequences used for the PCRs shown in <br /> Fig. 4A are available upon request.                                                  in YES (data not shown). The growth rate of an unrelated <br />    SDS-PAGE and Western blotting. Proteins were separated by SDS-polyacryl-          knockout strain (termed 93) was comparable to that of WT <br /> amide gel electrophoresis (PAGE) (10% gels) and transferred to polyvinylidene        cells, indicating that the slower growth of rct1&#11001;/&#11002; cells is not <br /> difluoride membranes (Millipore), followed by Western blotting according to          due to the presence of ClonNat in the medium. Surprisingly, <br /> standard procedures. Rat anti-HA (3F10) (Roche), mouse antitubulin (Sigma- <br /> Aldrich), and mouse anti-CTD monoclonal antibodies (H14, H5, 8WG16; Co- <br />                                                                                      cells grown overnight in minimal medium without a nitrogen <br /> vance) were used at 1:1,000 dilutions. Anti-Rct1 polyclonal antibodies were          source (EMM-N) strongly sporulated even at 32&#176;C (Fig. 1D). <br /> raised in rabbits immunized simultaneously with two peptides, P1 (H2N-DLVE           Sporulation was highly enhanced compared to WT cells, and <br />  3604      GULLEROVA ET AL.                                                                                                       MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 1. Analysis of rct1&#11001;/&#11002; cells. (A) Schematic representation of the domain structure of Rct1. (B) Tetrad analysis of rct1&#11001;/&#11002; diploid S. pombe <br /> cells. Six asci were dissected, and spores were grown on YES plates at 32&#176;C for 5 days. (C) Rct1 is a nuclear protein. rct1&#9004;pMG1 cells were grown <br /> overnight in EMM and processed for immunostaining with anti-HA antibody. Rct1 localizes into the nuclei of interphase cells (upper panels) as <br /> well as of dividing cells (two lower panels). Hoechst staining was performed to determine the position of nuclei, and DIC images were acquired <br /> to determine cell shapes. (D) Microscopic analysis of WT and rct1&#11001;/&#11002; cells grown overnight in liquid YES and EMM media as well as of cells grown <br /> for 2 days on YES plates. Images were acquired with a CCD camera by using DIC optics and a 63&#11003; oil objective. (E) Growth analysis of WT, <br /> rct1&#11001;/&#11002;, and the unrelated S. pombe 93 strains. Cell were grown overnight in YES medium and diluted to an OD600 of 0.1 in YES medium, and <br /> the OD600 was measured every 2.5 h. (F) Analysis of Rct1 levels in WT and rct1&#11001;/&#11002; cells. Analysis of the tubulin level (right) was used as a loading <br /> control. <br />  <br />  <br />  <br /> virtually all cells were sporulated after 34 h, whereas sporula-            ated a potential overexpressing strain, WTpMG1, and, as a <br /> tion of WT cells was around 50% (Fig. 2D and data not                       control, a WTpMG strain by transforming WT diploid cells with <br /> shown).                                                                     plasmids pMG1 (tagged Rct1) and pMG (empty vector), re- <br />    WT S. pombe cells are rod-shaped (cylindrical), and their                spectively. Tetrad dissection of rct1&#11001;/&#11002;pMG1 cells resulted in <br /> polarity is expressed in the pattern of cell growth, which is               only two growing spores. PCR genotyping, as well as growth in <br /> restricted to the ends of the long <span id='am-112' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-113'  ></span><span id='am-114' property="rdfs:label" content="Axis" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>axis</span> of the cell (16, 29). In WT         the presence of ClonNat and G418, revealed that they are <br /> cells grown in YES and EMM, both aniline blue staining and                  rct1&#9004; haploid cells harboring pMG1 (hereafter rct1&#9004;pMG1). <br /> actin localization revealed patterns characteristic of dividing             Although the nmt1 promoter is repressed by thiamine present <br /> asynchronous cells; the strongest staining was at the cell tips             in YES medium and should be off, it is still active to some <br /> and at sites of septum synthesis. In contrast, many rct1&#11001;/&#11002; cells           extent, and detectable amounts of protein are produced (Fig. <br /> were stained asymmetrically either on one cell end or between               2A). The rct1&#9004;pMG1 cells grown overnight in YES or EMM <br /> cell tips (aniline blue). In addition, in rct1&#11001;/&#11002; cells, actin was          were of the size and shape expected for haploid WT cells (Fig. <br /> found throughout the cell or in curved cables running either                2C, top, and Fig. S1 in the supplemental material), although <br /> along the long axis of the cell or curling around the nucleus or            some irregularly shaped cells were observed as well (data not <br /> the cell surface (see Fig. S1 in the supplemental material). All            shown; also see below). This indicates that low expression of <br /> growth and morphological changes with rct1&#11001;/&#11002; cells were ob-                Rct1 from the plasmid in YES medium (Fig. 2A) is sufficient to <br /> served either in the presence or absence of ClonNat, indicating             complement rct1&#9004; cells. Why we never recovered WT spores <br /> that they are not due to the presence of the antibiotic. There-             expressing additional Rct1 from the plasmid is unclear. <br /> fore, we compared Rct1 protein levels in WT and rct1&#11001;/&#11002; cells                  Growth analysis showed that rct1&#11001;/&#11002;pMG1 and rct1&#9004;pMG1 <br /> and found that Rct1 levels are considerably lower in rct1&#11001;/&#11002;                grow at approximately the same rate, somewhat slower than <br /> cells (Fig. 1F, left), obviously leading to haploinsufficiency.             WT but faster than rct1&#11001;/&#11002; (Fig. 2B). The addition of thiamine <br />    To confirm that the observed phenotypes in rct1&#11001;/&#11002; cells are             (100 &#9262;g/ml) significantly reduced the growth of rct1&#9004;pMG1 <br /> indeed a consequence of the reduced levels of Rct1, in vivo                 cells, but, unexpectedly, it did not stop it completely (Fig. 2B). <br /> complementation was performed. A plasmid expressing HA-                     Analysis of the Rct1-HA protein in these cells revealed the <br /> tagged Rct1 (pMG1) under control of the thiamine-repressible                presence of low levels of HA-tagged Rct1 even 24 h after the <br /> nmt1 promoter (Fig. 2A, top) was transformed into rct1&#11001;/&#11002;                   addition of thiamine (Fig. 2C, insert in the left bottom panel). <br /> diploid cells, forming the rct1&#11001;/&#11002;pMG1 strain. We also gener-               This explains why haploid rct1&#9004;pMG1 cells were still growing <br />  VOL. 27, 2007                                               CYCLOPHILIN REGULATES RNAP II CTD PHOSPHORYLATION                              3605 <br />  <br />  <br />  <br />  <br />    FIG. 2. Episomal expression of Rct1 complements rct1&#11001;/&#11002; cells. (A) (Top) Schematic presentation of plasmid pMG1, expressing HA-tagged <br /> Rct1. The nmt1 promoter is repressed by thiamine and is therefore active only in minimal medium (EMM), which does not contain thiamine. <br /> (Bottom) Expression analysis of Rct1-HA in rct1&#11001;/&#11002;pMG1, rct1&#9004;pMG1, and WTpMG1 cells grown overnight in EMM (left) or YES (right) medium. <br /> Western blotting was performed with anti-HA antibody. The lower right panel represents an analysis of Rct1 levels; the band marked with an <br /> asterisk represents an antibody cross-reacting peptide. (B) Growth analysis of WT, WTpMG, WTpMG1, rct1&#11001;/&#11002;, rct1&#11001;/&#11002;pMG1, and rct1&#9004;pMG1 cells <br /> in liquid YES medium. Cells were grown overnight in YES medium, diluted to an OD600 of 0.1, and further grown for 25 h in the presence of <br /> ClonNat and G418. Rct1&#9004;pMG1 cells were also grown in the presence of thiamine (rct1&#9004;pMG1 &#11001; T) to inhibit expression of Rct1-HA from plasmid. <br /> (C) Actin, DNA, and cell wall (Hoechst and aniline blue [H&#11001;AB]) staining of rct1&#9004;pMG1 cells grown overnight in YES (top) or for 24 h in the <br /> presence of thiamine (bottom). Note mycelial, branched, and otherwise unseparated rct1&#9004;pMG1 cells and asymmetric aniline blue staining (AB). <br /> Cells in the top and bottom panels (on the left) are at the same scale. All cells in the right panel are at the same scale. Images were acquired <br /> with a CCD camera by using DIC optics and a 63&#11003; oil objective. The insert in the bottom-left panel represents an analysis of the Rct1-HA <br /> level in rct1&#9004;pMG1 cells grown in the presence of thiamine. (D) Analysis of sporulation defects of rct1&#11001;/&#11002; cells. WT, rct1&#11001;/&#11002;, rct1&#11001;/&#11002;pMG1, <br /> rct1&#11001;/&#11002;pMG, and WTpMG1 cells were inoculated in EMM-N medium at an OD600 of 0.1 and grown at 32&#176;C for 24 h. Sporulation was scored <br /> after 24 h by counting 100 cells for each strain. (E) RT-PCR analysis of meiotically induced genes from cells grown in EMM-N. Samples <br /> were taken at the indicated time points, and RT-PCR was performed as described previously (2). Positions of unspliced and spliced <br /> transcripts are indicated on the right. <br />  3606     GULLEROVA ET AL.                                                                                           MOL. CELL. BIOL. <br />  <br />  <br /> and also shows that very low levels of Rct1 are able to support     phase (Fig. 3C). Analysis of the cytoplasmic tubulin levels (Fig. <br /> some cell growth. However, the rct1&#9004;pMG1 cells grown for 24 h       3B and C, lowest panels), as well as staining of membranes with <br /> or longer in the presence of thiamine in both media became          Ponceau S (data not shown), revealed that equal amounts of <br /> wider (swollen) and much shorter than WT cells, with a variety      protein were loaded in each lane. In addition, analysis of the <br /> of growth shapes ranging from normal and rod-like to                levels of two other RNAP II subunits, Rpb3 and Rpb7, showed <br /> rounded, pear-like, irregularly branched, or highly elongated       that the total levels of RNAP II in WT and rct1&#11001;/&#11002; cells are <br /> (Fig. 2C and data not shown). Also, many cells grow in short        similar (Fig. 3B). Thus, the results described above clearly <br /> chains, indicating defects in cell separation. It is also worth     demonstrate that the CTD phosphorylation status is impaired <br /> noting that WTpMG1 cells that overexpress Rct1 grow very            in rct1&#11001;/&#11002; cells and that reduced levels of Rct1 result in CTD <br /> slowly, in accordance with our previous data on AtCyp59 (12),       hyperphosphorylation. Also, these results are consistent with <br /> although no morphological defects were observed. Finally, the       our studies of AtCyp59, whose overexpression led to CTD <br /> results shown in Fig. S1 in the supplemental material clearly       hypophosphorylation (12). <br /> indicate that all other phenotypes observed in rct1&#11001;/&#11002; cells are       Episomal expression of Rct1-HA rescues the CTD phosphor- <br /> rescued by episomal expression of Rct1-HA.                          ylation defect. Next, we analyzed the CTD phosphorylation <br />    During their life cycle, diploid S. pombe cells heterozygous     status in rct1&#11001;/&#11002;pMG1 and rct1&#9004;pMG1 strains and compared it to <br /> for the mating-type genes can either undergo mitotic prolifer-      that of WT and rct1&#11001;/&#11002; cells. Cells were grown overnight in <br /> ation or enter meiosis. The decision between these two devel-       YES medium (OD600, 0.5 to 0.7 [depending on the strain]), <br /> opmental states depends on nutrient availability, particularly      where expression of Rct1-HA from the plasmid is very low, and <br /> nitrogen, and on growth temperature. We show that the reg-          CTD phosphorylation was analyzed by using H14 and H5 an- <br /> ulation of meiotic differentiation is perturbed in rct1&#11001;/&#11002; cells    tibodies. Episomal expression of Rct1-HA in rct1&#11001;/&#11002;pMG1 and <br /> grown at 32&#176;C in EMM-N (EMM without a nitrogen source)              rct1&#9004;pMG1 cells resulted in a decrease in CTD phosphorylation <br /> (Fig. 1D). In addition, we show that genes important for the        compared to Rct1&#11001;/&#11002; cells (Fig. 3D; compare lanes 2 and 3 <br /> entrance into meiosis were up-regulated in rct1&#11001;/&#11002; cells and        with lane 1). However, CTD phosphorylation levels turned out <br /> demonstrated activated splicing (Fig. 2E). Ectopic expression       to be even lower than in WT cells. As the same effect was <br /> of Rct1-HA in these cells (rct1&#11001;/&#11002;pMG1 strain) resulted in          observed in WT cells ectopically expressing Rct1-HA (Fig. 3D; <br /> complementation of this defect. Interestingly, however, after       compare lane 5 with lanes 2 and 3), we analyzed Rct1 levels in <br /> 24 h in EMM-N, only &#11011;15% of rct1&#11001;/&#11002;pMG1 cells sporulated,           these trains. As shown in Fig. 3D, Rct1 levels in these strains <br /> compared to &#11011;35% of WT cells. Similarly, overexpression of          are somewhat higher than in WT cells (see also Fig. 2A, lower <br /> Rct1 in the WT background (WTpMG1) also reduced sporula-            panel on the right). Therefore, these data are consistent with <br /> tion efficiency (Fig. 2D). Together, these data indicate that       our results with ectopic expression of AtCyp59 in Arabidopsis <br /> precise regulation of Rct1 levels is necessary for correct mei-     cells, which showed decreased CTD phosphorylation upon a <br /> otic differentiation. We also analyzed three spliced transcripts    slight increase of protein level (12) Also, this might explain <br /> that are not induced by meiosis (nak1, hob1, and wsp1). As          why these cells grow at a lower rate than do WT cells. <br /> shown in Fig. 2D, their splicing was not affected by underex-          To further investigate the effects of Rct1 on CTD phosphor- <br /> pression of Rct1, indicating that Rct1 is most probably not a       ylation, we analyzed CTD phosphorylation in rct1&#9004;pMG1 cells <br /> general splicing regulator.                                         in more detail. Cells were grown overnight in YES or EMM <br />    CTD phosphorylation status is affected in rct1&#11545;/&#11546; cells. Our     medium and then diluted to an OD600 of 0.05 in YES contain- <br /> previous results obtained with the Arabidopsis Rct1 homolog,        ing 100 &#9262;g/ml thiamine, and the same amounts of cells were <br /> AtCyp59, showed that it interacted with the CTD of RNAP II          taken for analysis at the indicated time points. Western blot <br /> and that ectopic expression of AtCyp59 in cell suspension           analysis with anti-HA antibody revealed efficient repression of <br /> culture resulted in a highly reduced growth rate. This effect was   Rct1-HA expression, resulting in very low levels of Rct1-HA <br /> attributed to the reduced phosphorylation of the CTD of             12 h later (Fig. 3E, lane 4 [HA panel]). Analysis of the same <br /> RNAP II (12).                                                       blots with H5 and H14 antibodies revealed a concomitant <br />    By immunoprecipitation experiments with HA-antibodies            increase in CTD phosphorylation at Ser2 and Ser5, respec- <br /> using rct1&#11001;/&#11002;pMG1 cell extracts, we show that Rct1 also copre-      tively (Fig. 3E). Again, equal protein loading was controlled by <br /> cipitates RNAP II (Fig. 3A). It was therefore of interest to        anti-tubulin antibody (Fig. 3E). Thus, the described data <br /> analyze the CTD phosphorylation status in rct1&#11001;/&#11002; cells. Over-      clearly show that underexpression of Rct1 increases whereas <br /> night cultures were grown in YES medium to mid-log phase            Rct1 overexpression decreases CTD phosphorylation. <br /> (OD600, 0.7), and equal amounts of cells were directly resus-          Rct1 is associated with transcriptionally active chromatin <br /> pended in SDS-PAGE loading buffer. RNAP II was detected             throughout the RNAP II transcription cycle. As we were able to <br /> by Western blotting using antibodies specific to the unphos-        demonstrate an interaction of Rct1 with the CTD of RNAP II, we <br /> phorylated CTD (8WG16), phosphorylated Ser2 (H5), and               asked next whether Rct1 also associates with active chromatin <br /> phosphorylated Ser5 (H14) (7, 20, 35). Notably, both the H5         and at which stage of the transcription cycle. rct1&#11001;/&#11002;pMG1 cells <br /> and H14 antibodies revealed higher phosphorylation in rct1&#11001;/&#11002;       were grown for 16 h under nmt1-inducible conditions, and <br /> cells, whereas the 8WG16 antibody showed only a modest              chromatin immunoprecipitation (ChIP) was performed with <br /> increase in RNAP II signal (Fig. 3B, lanes 2). This is consistent   anti-HA antibodies. Four RNAP II genes were analyzed with <br /> with the specificities of H5 and H14 antibodies compared to         oligonucleotides covering promoters, open reading frames, <br /> that of 8WG16 (7, 20, 35). Time course analysis of cells grown      and 3&#11032; UTRs. Figure 4A demonstrates that Rct1 is associated <br /> in YES medium showed that CTD hyperphosphorylation per-             with the chromatin at the promoter region and remains asso- <br /> sists from very low OD until the cells reach the stationary         ciated throughout the transcriptional cycle, as all oligonucleo- <br />  VOL. 27, 2007                                              CYCLOPHILIN REGULATES RNAP II CTD PHOSPHORYLATION                              3607 <br />  <br />  <br />  <br />  <br />    FIG. 3. Phosphorylation status of the RNAP II CTD in rct1&#11001;/&#11002; heterozygous cells. (A) Rct1 coprecipitates RNAP II. Immunoprecipitation of <br /> Rct1-HA from rct1&#11001;/&#11002;pMG1 cell extracts results in coprecipitation of RNAP II, as determined by anti-CTD monoclonal antibody H14. Lane 1, input <br /> protein extract; lane 2, protein extract incubated with protein A-Sepharose beads; lane 3, protein extract incubated with anti-HA monoclonal <br /> antibody and protein A-Sepharose. Antibodies used for Western blotting are indicated on the right. Immunoprecipitation with anti-HA antibody <br /> and protein extract from WT cells did not result in precipitation of detectable amounts of RNAP II (lowest panel, analyzed with anti-CTD H14 <br /> antibody). (B) CTD phosphorylation is increased in rct1&#11001;/&#11002; cells. Overnight cultures in the mid-exponential phase (OD600 of 0.7) grown in YES <br /> medium were collected and directly resuspended in 2&#11003; SDS-PAGE loading buffer, and proteins were separated by 8% SDS-PAGE. Blots were <br /> probed with monoclonal antibodies against the CTD of RNAP II, Rpb3, and Rpb7 (indicated on the right side of each panel). Analysis of the <br /> cytoplasmic &#9251;-tubulin level was performed as a loading control. (C) Time course of CTD phosphorylation in WT and rct1&#11001;/&#11002; cells. Overnight <br /> cultures grown in YES medium were diluted to an OD600 of 0.1 and grown further. Cells were taken at indicated ODs, and CTD phosphorylation <br /> was analyzed with monoclonal antibody H14 (specific for Ser5). Analysis of the cytoplasmic tubulin level was used as a loading control (lower <br /> panel). Details are as for panel B. (D) CTD phosphorylation defect in rct1&#11001;/&#11002; cells is rescued by episomal expression of Rct1-HA. Protein blots <br /> were probed with the antibodies indicated on the right. (E) Repression of Rct1-HA expression increases CTD phosphorylation. Protein blots from <br /> rct1&#9004;pMG1 cells grown in the presence of thiamine were taken at the indicated time points and analyzed by Western blotting with the antibodies <br /> indicated on the right. <br />  <br />  <br />  <br /> tide pairs resulted in a comparable amplification of corre-               (act1), which is not affected by meiotic differentiation, did not <br /> sponding DNA fragments in input and coprecipitated DNA                    show any difference between the 10- and 24-h time points (Fig. <br /> samples (compare lanes 3 and 1). Control PCRs performed                   4B). Together, these data are consistent with increased tran- <br /> with oligonucleotides corresponding to promoter and coding                scriptional activity of the spo4, meu13, mfr1, and mei2 genes at <br /> regions of the mitochondrial cox1 gene or the nuclear U6                  the 24-h time point in rct1&#11001;/&#11002;pMG1 cells (Fig. 2E) and clearly <br /> snRNA gene, which is transcribed by RNAP III, did not result              demonstrate that association of Rct1 with the chromatin de- <br /> in significant amplification, indicating that Rct1 specifically           pends on its transcriptional activity. <br /> associates with RNAP II genes. These results are also in agree-              RNAP II transcription is reduced in rct1&#11545;/&#11546; cells. The CTD <br /> ment with the observation that mainly RNAP II transcription               undergoes a complex pattern of phosphorylation/dephosphor- <br /> is reduced in rct1&#11001;/&#11002; cells (see below).                                  ylation during the transcription cycle (see the introduction). In <br />    To investigate whether association of Rct1 with the chroma-            <span id='am-117' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-118' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-119' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> to enable entry of RNAP II into a new initiation com- <br /> tin correlates with transcriptional activity, we tested four mei-         plex, the CTD has to be dephosphorylated after release from <br /> osis-specific genes (spo4, meu13, mfr1, and mei2 [2, 30]) that            the chromatin. Therefore, we investigated the functional con- <br /> showed enhanced transcription when rct1&#11001;/&#11002;pMG1 cells were                 sequence(s) of Rct1 depletion on RNAP II transcriptional <br /> grown for 24 in EMM-N medium (Fig. 2E). We performed                      activity. We measured the levels of nascent transcription for <br /> ChIP analysis with rct1&#11001;/&#11002;pMG1 cells grown for 10 and 24 h in             some known RNAP II transcription units by using nuclear <br /> EMM-N. Figure 4B clearly demonstrates that 10 h after inoc-               run-on analysis. Equal amounts of exponentially growing WT <br /> ulation, Rct1 is associated to some extent with the promoter,             and rct1&#11001;/&#11002; cells were permeabilized with Na-sarcosyl, and <br /> with the coding region, and with the 3&#11032; UTRs of the analyzed              ongoing transcription was measured by incorporation of <br /> genes. However, 24 h after inoculation, a significant increase in         [32P]UTP. DNA fragments corresponding to the coding re- <br /> coprecipitation of promoter, coding regions, and 3&#11032; UTRs has              gions of 13 randomly selected protein-coding genes were PCR <br /> been observed (Fig. 4B). In contrast, analysis of the actin gene          amplified and slot blotted onto membranes. In addition, DNA <br />  3608     GULLEROVA ET AL.                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 4. Rct1 is associated with transcriptionally active chromatin throughout the RNAP II transcription cycle. (A) Association of Rct1 with the <br /> chromatin was analyzed by ChIP. ChIP was performed with anti-HA antibody and rct1&#11001;/&#11002;pMG1 cells grown at 32&#176;C under nmt1-inducible conditions <br /> (EMM medium). Four RNAP II genes previously analyzed by nuclear run-on were analyzed (upper panels). The two lower panels are controls <br /> performed with genes transcribed with mtRNAP (cox1) and RNAP III (U6 snRNA). Lanes 1, PCR with immunoprecipitated chromatin; lanes 2, <br /> PCR with chromatin bound to protein A-Sepharose alone; lanes 3, PCR with input chromatin. (B) ChIP analysis of four meiosis-specific genes. <br /> The rct1&#11001;/&#11002;pMG1 cells were grown in EMM-N for 10 or 24 h, and ChIP with the meiotically induced genes spo4, meu13, mfr1, and mei2 was <br /> performed as described for panel A. Coimmunoprecipitated DNA was analyzed by real-time PCR. All samples were run in triplicate to ensure <br /> accuracy of the data. <br />  <br />  <br />  <br />  <br /> fragments from genes transcribed by RNAP III (tRNASer and <br /> U6 snRNA), by RNAP I (17S rRNA), and by mitochondrial <br /> RNA polymerase (mtRNAP) (cox1 gene) were employed as <br /> controls. Filters were hybridized with total [32P]UTP-labeled <br /> RNA from WT and rct1&#11001;/&#11002; cells and quantified after exposure <br /> to a PhosphorImager. Filters were subsequently exposed to <br /> X-ray film (Fig. 5A). Quantification of results from three in- <br /> dependent experiments clearly showed that transcription of all <br /> 13 RNAP II genes is reduced about 20 to 50% in rct1&#11001;/&#11002; cells <br /> (Fig. 5A, genes 1 to 13). In contrast, both RNAP III (samples <br /> 14 and 16) and mtRNAP (sample 18) transcript synthesis was <br /> not affected, whereas RNAP I transcription (sample 15) was <br /> also slightly inhibited (Fig. 5A). Thus, transcription run-on <br /> data show that partial depletion of Rct1, which results in hy- <br /> perphosphorylation of the CTD, affects RNAP II transcription. <br /> However, an additional effect on RNAP I transcription cannot <br /> be excluded (see Discussion). <br />    Because ongoing RNAP II transcription was reduced in <br /> rct1&#11001;/&#11002; cells, we wanted to analyze the impact on the steady- <br /> state levels of RNAP II transcripts. To do so, total RNA <br /> isolated from WT and rct1&#11001;/&#11002; cells was reverse transcribed and <br /> used for PCR amplification of 10 RNAP II genes that were <br /> previously analyzed by nuclear run-on assay (Fig. 5A, genes 1 <br /> to 10). The semiquantitative RT-PCR experiment was re-                        FIG. 5. RNAP II transcription is reduced in rct1&#11001;/&#11002; cells. (A) Au- <br />                                                                            toradiograph of a representative nuclear run-on experiment. Samples <br /> peated several times with newly isolated RNA. Quantification               1 to 13, RNAP II transcripts. In addition, DNA fragments from the <br /> of results shown in Fig. 5B revealed that the RNA steady-state             RNAP I gene (17S rRNA; sample 15), RNAP III genes (tRNASer and <br /> levels for all 10 genes are reduced from 15 to 40% in rct1&#11001;/&#11002;              U6 snRNA genes, samples 14 and 16), the RNAP II gene for U1 <br />                                                                            snRNA (sample 17), and mtRNAP (cox1 gene; sample 18), were used <br /> cells. Measurements of ongoing transcription and steady-state <br />                                                                            as controls. Slot blots were exposed to X-ray film. Quantification of <br /> levels of mRNAs provide strong evidence that reduction of the              three independent nuclear run-on experiments is shown below. Inten- <br /> Rct1 level results in inhibition of RNAP II transcription. These           sities on the y axis are arbitrary units. (B) Steady-state levels of 10 <br /> results, together with the increased phosphorylation status of             RNAP II genes (genes 1 to 10 in panel A) were analyzed by semi- <br />                                                                            quantitative RT-PCR. Total RNA isolated from WT and rct1&#11001;/&#11002; cells <br /> the CTD, which is predicted to influence its interaction with <br />                                                                            was reverse transcribed with an oligo(dT) primer, and the resulting <br /> other proteins, could be the cause of the observed growth and              cDNA was used for PCR with gene-specific primers. Intensities on the <br /> morphological defects of rct1&#11001;/&#11002; cells.                                    y axis are arbitrary units. <br />  VOL. 27, 2007                                           CYCLOPHILIN REGULATES RNAP II CTD PHOSPHORYLATION                              3609 <br />  <br />  <br />                         DISCUSSION <br />  <br />    The data presented here, together with our previous data on <br /> Arabidopsis AtCyp59, provide genetic and biochemical evi- <br /> dence that the S. pombe nuclear multidomain cyclophilin Rct1 <br /> is required for fine tuning of RNAP II CTD phosphorylation. <br /> Specifically, we show that rct1 is an essential gene. Further- <br /> more, deletion of one rct1 allele in a diploid strain results in <br /> elevated phosphorylation of the CTD, reduced RNAP II tran- <br /> scriptional activity, and severe growth and morphological de- <br /> fects. These data, which describe phenotypes of a heterozy- <br /> gote, provide evidence that Rct1 levels are under tight dosage <br /> control and that correct levels of Rct1 are absolutely required        FIG. 6. Model of how Rct1 might influence CTD structure and <br /> for a balanced CTD phosphorylation/dephosphorylation cycle.         function. We propose that the &#8220;CTD code&#8221; is a result of coordinated <br /> Finally, the presence of Rct1 along the entire length of actively   activities of CTD kinases, phosphatases, and PPIases, which together <br />                                                                     generate high-affinity binding sites for factors involved in the regula- <br /> transcribed genes is consistent with a direct interaction be-       tion of RNAP II transcription and its coupling with pre-mRNA pro- <br /> tween Rct1 and the elongating RNAP II complex. Thus, our            cessing. <br /> results suggest that this multidomain cyclophilin is an essential <br /> regulator of RNAP II transcription. <br />    Cyclophilins are ubiquitous proteins that possess PPIase ac-     particular, it has been shown that Pcf11 binds the CTD in a <br /> tivity; e.g., they catalyze cis-trans isomerization of peptide      phosphorylation-dependent manner, whereby phosphorylation <br /> bonds preceding proline. In eukaryotic cells, cyclophilins have     of the two adjacent repeats contributes to an increase in the <br /> been found in all cellular compartments with a variety of func-     affinity of Pcf11 for CTD (34). These new crystallographic and <br /> tions being ascribed to them, including protein trafficking and     biochemical data provide evidence of how changes in phosphor- <br /> maturation, receptor signaling, receptor complex stabilization,     ylation and proline isomerization could contribute to the struc- <br /> apoptosis, RNA processing, spliceosome assembly, and tran-          tural plasticity of the CTD, which is important for recruitment of <br /> scription (see references 15, 19, 26, 27, 28, 44, 45, 49, and 52    many different proteins to the CTD and the transcription complex <br /> and references therein). Despite the obviously widespread im-       (3, 24, 32). These prerequisites suggest that kinases, phosphatases, <br /> portance of PPIases, genetic analysis of 13 immunophilin genes      and peptidy-prolyl isomerases are important regulators of <br /> in S. cerevisiae revealed only ESS1 (homolog of the human           CTD functions (3, 24, 32, 34). <br /> Pin1) as essential (13). However, recent reexamination of              Several kinases and phosphatases have been shown to influ- <br /> ESS1 disruption revealed that it is not absolutely essential for    ence phosphorylation/dephosphorylation of the CTD (32, 34, <br /> cell viability and that its deletion rather causes a severe tem-    38), but only one immunophilin, Pin1/ESS1, has been impli- <br /> perature-sensitive growth phenotype (18). ESS1 is also not          cated in the regulation of these processes (50, 51). Pin1 is a <br /> essential in S. pombe (18) and <span id='am-56' about='obo:NCBITaxon_5207' typeof='owl:Thing obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_5204 obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_5215 obo:NCBITaxon_5206 obo:NCBITaxon_5234 obo:NCBITaxon_2759 obo:NCBITaxon_552466 obo:NCBITaxon_155616'><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasRelatedSynonym" content="Filobasidiella neoformans" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasRelatedSynonym" content="Torulopsis neoformans" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasRelatedSynonym" content="Torula neoformans" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="Cryptococcus neoformans" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasRelatedSynonym" content="Filobaxidiella neoformans" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="Blastomyces neoformans" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="Filobasidiella neoformornans" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="Debaryomyces neoformans" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces neoformans" datatype="xsd:string"></span><span id='am-67'  ></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Lipomyces neoformans" datatype="xsd:string"></span>Cryptococcus neoformans</span>, but it      small parvulin-type immunophilin that has been shown to in- <br /> is in some other <span id='am-30' about='obo:NCBITaxon_4751' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2759'><span id='am-31' property="oboInOwl:hasDbXref" content="PMID:11062127" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="Fungi" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="fungi" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="fungi" datatype="xsd:string"></span><span id='am-35'  ></span><span id='am-36' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="PMID:12684019" datatype="xsd:string"></span>fungi</span>, e.g., <span id='am-76' about='obo:NCBITaxon_5475' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_118267 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-77' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-78'  ></span><span id='am-79' property="oboInOwl:hasRelatedSynonym" content="Torulopsis" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="Candida" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Candida</span> albicans and <span id='am-70' about='obo:NCBITaxon_5052' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_5042 obo:NCBITaxon_33154 obo:NCBITaxon_147545 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_1131492 obo:NCBITaxon_451871 obo:NCBITaxon_2759'><span id='am-71'  ></span><span id='am-72' property="oboInOwl:hasAlternativeId" content="NCBITaxon:37503" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="Aspergillus" datatype="xsd:string"></span>Aspergillus</span>      teract with many different proteins involved in a wide variety of <br /> nidulans. Similarly, Rct1 is essential in S. pombe, but its ho-     cellular processes (4, 27, 28, 52). As pin1&#11002;/&#11002; cells accumulate <br /> molog in P. tetraurelia, Kin241, is not (22). It is not clear why   hypophosphorylated RNAP II, whereas moderate overexpres- <br /> deletion of a particular PPIase is lethal in some organisms and     sion of Pin1 resulted in the appearance of a novel hyper- <br /> not in others. One possibility is that in certain genetic back-     hyperphosphorylated form of RNAP II (51), its influence on <br /> grounds, high expression levels of other PPIases compensate         CTD phosphorylation was investigated. In vitro experiments <br /> for the loss of the other(s), thus acting as natural suppressors.   suggested that human Pin1 inhibits the CTD phosphatase <br /> This would also indicate that the functions of certain PPIases      FCP1 and stimulates CTD phosphorylation by cdc2/cyclin B <br /> are partially redundant or overlapping. Alternatively, it is pos-   (51). Here, we show that reduced levels of the cyclophilin Rct1 <br /> sible that some PPIases are essential only under certain con-       result in increased CTD phosphorylation at both Ser2 and <br /> ditions.                                                            Ser5, whereas increased levels of Rct1 (and also of AtCyp59) <br />    The highly repetitive primary sequence of the CTD suggests       (12) reduced CTD phosphorylation. This indicates that, under <br /> that it can adopt different conformations. The significance of      physiological conditions, Rct1 stimulates CTD dephosphoryla- <br /> Rct1 regulating CTD function is very convincing in light of         tion. Thus, these two PPIases have entirely opposite effects on <br /> recent structural data that suggest that the free CTD, which        the CTD phosphorylation status; Rct1p has a positive effect on <br /> contains a multitude of heptapeptide repeats (YSPTSPS), is          CTD dephosphorylation (this work), as opposed to Pin1, which <br /> highly flexible. Significantly, the peptidyl-proline isomerase      has a negative effect (51). However, there is genetic evidence <br /> Pin1, the capping enzyme guanylyltransferase Cgt1, and Pcf11,       from S. cerevisiae indicating that ESS1 stimulates dephosphor- <br /> an essential component of the mRNA cleavage factor <span id='am-11' about='obo:NCBITaxon_360966' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_9431 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="Ia" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15'  ></span>IA</span>, all          ylation of the CTD (50), as Rct1 does in S. pombe. Nonethe- <br /> bind to CTD peptides only in an all-trans conformation in vitro,    less, in both cases it is not clear how these effects are brought <br /> providing the first direct evidence for the importance of cis-      about. Rct1 could either bind to the CTD, thereby facilitating <br /> trans isomerization of prolyl bonds in CTD repeats in protein-      access of other CTD-modifying enzymes, or, alternatively, the <br /> protein interactions (10, 31, 32, 34, 48). Furthermore, interac-    CTD kinases and phosphatases are direct targets of Rct1 (Fig. <br /> tions of the CTD with regulatory factors have been found to         6). Intriguingly, PTPA, an unusual PPIase, has recently been <br /> depend on Ser2 and Ser5 phosphorylation (23, 37, 38). In            shown to function as an activator of PP2A phosphatase (21). <br />  3610     GULLEROVA ET AL.                                                                                                       MOL. CELL. BIOL. <br />  <br />  <br /> The increase/decrease in CTD phosphorylation by reducing/           changes in binding of proteins regulating splicing of meiosis- <br /> elevating the level of Rct1 indicates that Rct1 could be in-        specific genes. Some meiosis-specific genes are transcribed un- <br /> volved in either inhibiting a CTD phosphatase or stimulating a      der vegetative growth; however, their pre-mRNAs are spliced <br /> CTD kinase activity. However, further experiments are neces-        only upon meiotic induction (2, 30). The factors regulating <br /> sary to answer these questions more precisely.                      these events may bind the CTD only under specific conditions. <br />    As cis-trans isomerization of peptide bonds takes place spon-    Thus, the deregulation of CTD phosphorylation observed in <br /> taneously, but at a much lower rate (42, 44, 49), it is also        rct1&#11001;/&#11002; cells might induce their binding under vegetative <br /> possible that PPIases are not absolutely required in vivo.          growth, consequently leading to splicing and induction of mei- <br /> Rather, they might be generally required for selective prolyl       osis. Similarly, it has recently been demonstrated that the CTD <br /> peptide bond isomerization in processes where fast changes in       promotes alternative splicing of fibronectin pre-mRNA by re- <br /> protein-protein interactions or protein phosphorylation/de-         cruiting SR protein SRp20 (9). Interestingly, the human ho- <br /> phosphorylation take place, e.g., coupled transcription and         molog of Rct1, PRIL4, has recently been identified in affinity- <br /> pre-mRNA processing. Many genetic and biochemical data              purified spliceosomal B complex (8), further supporting our <br /> have accumulated over the last decade that indicate the phys-       hypothesis that Rct1 plays an important role at the interface of <br /> ical and functional coupling of RNAP II transcription with          transcription and pre-mRNA processing. Whether Rct1 is in- <br /> pre-mRNA processing and chromatin structure through the             deed involved in pre-mRNA processing and, if so, how it con- <br /> CTD of RNAP II. A highly dynamic CTD phosphorylation                tributes to splicing regulation requires further investigation. <br /> pattern is required for coordination of transcription and nu- <br /> clear pre-mRNA processing (1, 4, 17, 35, 37, 39, 46, 53). In                                 <br /> addition, it has recently been demonstrated that cis-trans <br />                                                                      <br /> isomerization of prolyl peptide bonds may also contribute to       <br /> establishing or disrupting binding interactions with the CTD        <br /> (3, 24, 32, 34). This, together with the data presented here and     <br /> previously with Pin1, highlights the general importance of            <br /> PPIases in the regulation of CTD structure and function (Fig. 6). <br />    We were able to show here that Rct1 is required for correct                                    <br /> CTD phosphorylation and consequently for efficient RNAP II         <br /> transcription. However, the highly pleiotropic phenotype ob-            <br /> served with rct1&#11001;/&#11002; cells, together with the observed tight dos-      <br /> age control of Rct1, is unusual for PPIases and hints at an-           <br /> other, as-yet-unknown function(s) besides RNAP II                  <br /> transcription regulation. Indeed, we show that RNAP I tran-           <br /> scription is reduced as well in rct1&#11001;/&#11002; cells. The reason for the       <br /> decrease in transcription of the 17S rRNA is not clear at the         <br /> moment. However, this is consistent with our previous obser-            <br /> vation that green fluorescent protein-tagged AtCyp59 localizes          <br /> to the nucleolus to some extent (12), where it could possibly      <br /> affect transcription of RNAP I as well. Also of note is that            <br /> deletion of one rct1 allele, which results in reduced Rct1 pro-     <br /> tein levels, causes strong growth and morphological pheno-      <br /> types, presumably due to deregulated CTD phosphorylation             <br /> and RNAP II transcription. However, the possibility that Rct1      <br /> is also involved in regulation of factors important for cell           <br /> morphology and differentiation is not excluded. Episomal ex-           <br /> pression of Rct1-HA in rct1&#11001;/&#11002;pMG1 cells partially restored all    <br /> growth and morphological defects observed in rct1&#11001;/&#11002; cells,           <br /> whereas moderate overexpression in WT cells caused a strong      <br /> negative effect on their growth and meiotic differentiation,            <br /> although cell size, polarity, and mitotic division were as in WT    <br /> cells. Thus, Rct1 levels are clearly under tight dosage control.       <br />    Rct1 is an unusual cyclophilin, as it contains an RRM and a         <br /> C-terminal domain rich in RS/RD dipeptide repeats and                  <br /> charged amino acids. The high conservation of these multido-       <br /> main proteins, in particular in the RRM region, suggests ad-            <br /> ditional possible functions apart from PPIase activity. Indeed,   <br /> our observation that AtCyp59 also interacts with SR proteins            <br /> in Arabidopsis suggested that it might regulate events at the          <br /> interface of transcription and splicing (12). In fact, the ob-     <br /> served meiotic phenotype in rct1&#11001;/&#11002; cells might be explained         <br /> by the deregulation of CTD phosphorylation and concomitant            <br /> </body></html>