<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>RNA (2001), 7 :1397&#8211;1402+ Cambridge University Press+ Printed USA+ <br /> Copyright &#169; 2001 RNA Society+ <br /> DOI: 10+1017+S1355838201011074 <br />  <br />  <br />  <br />  <br /> Distinct roles RDE-1 RDE-4 RNA <br /> interference <span id='am-28' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-29' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-31'  ></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span>Caenorhabditis elegans</span> <br />  <br />  <br /> SUSAN PARRISH 1,2 ANDREW FIRE 1 <br /> 1 <br />     Department Embryology, Carnegie Institution Washington, Baltimore, Maryland 21210, USA <br /> 2 <br />     Biology Graduate Program, Johns Hopkins University, Baltimore, Maryland 21218, USA <br />  <br />  <br />  <br />  <br /> ABSTRACT <br /> RNA interference (RNAi) cellular defense mechanism uses double-stranded RNA (dsRNA) sequence- <br /> specific trigger guide degradation homologous single-stranded RNAs. RNAi multistep process involving <br /> several proteins type RNA intermediate, population small 21&#8211;25 nt RNAs (called siRNAs) that <br /> are initially derived cleavage dsRNA trigger. Genetic screens <span id='am-35' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-36' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-39'  ></span>Caenorhabditis</span> elegans identified <br /> numerous mutations cause partial complete loss RNAi. In work, analyzed cleavage injected <br /> dsRNA produce initial siRNA population animals mutant rde-1 rde-4, genes essential for <br /> RNAi required organismal viability fertility. Our results suggest distinct roles RDE-1 and <br /> RDE-4 interference process. Although null mutants lacking rde-1 phenotypic response dsRNA, the <br /> amount siRNAs generated injected dsRNA trigger comparable wild-type. By contrast, <br /> mutations rde-4 substantially reduced population siRNAs derived injected dsRNA trigger. Injection <br /> of chemically synthesized 24- 25-nt siRNAs circumvent RNAi resistance rde-4 mutants, bypass <br /> was observed rde-1 mutants. These results support model RDE-4 involved production <br /> of siRNAs, RDE-1 acts siRNAs formed. <br /> Keywords: dsRNA; Piwi; PTGS; RNAi; silencing; siRNA <br />  <br />  <br />  <br /> INTRODUCTION                                                            Bosher &amp; Labouesse, 2000)+ The ancient origin RNAi <br />                                                                         PTGS processes suggests mecha- <br /> Defense mechanisms protect deleterious <br />                                                                         nisms represent primitive nucleic-acid-based im- <br /> gene expression essential long-term survival of <br />                                                                         mune response+ <br /> a <span id='am-53' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>+ These genome surveillance mechanisms <br />                                                                            Although precise molecular mechanisms RNAi <br /> allow selective silencing destruction aberrant nu- <br />                                                                         remain elucidated, recent observations <br /> cleic acids, including molecules endogenous <br />                                                                         provided beginnings working model+ A <br /> and parasitic origin+ RNA interference (RNAi), ex- <br />                                                                         particularly significant observation Ham- <br /> ample mechanism, uses double-stranded RNA <br />                                                                         ilton &amp; Baulcombe (1999), identified population <br /> (dsRNA) trigger sequence-specific degradation <br />                                                                         small (;25-nt) RNAs (now referred siRNAs), <br /> of homologous transcripts (Fire et al+, 1998; reviewed <br />                                                                         presence correlates ongoing silencing+ <br /> in Sharp, 2001)+ dsRNAs frequently byproduct of <br />                                                                         siRNAs observed Drosophila extracts <span id='am-56' about='obo:NCBITaxon_201850' typeof='owl:Thing obo:NCBITaxon_1125798 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71529 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="Car" datatype="xsd:string"></span><span id='am-59'  ></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>car</span>- <br /> viral transposon gene expression+ One proposed <br />                                                                         rying RNAi-like reaction (Hammond et al+, 2000; <br /> function RNAi recognize dsRNAs in- <br />                                                                         Zamore et al+, 2000)+ siRNA formation vitro in <br /> dicators unwanted gene expression, providing <br />                                                                         vivo shown result cleavage <br /> a means silence parasitic genetic elements (re- <br />                                                                         dsRNA trigger (Parrish et al+, 2000; Yang et al+, <br /> viewed Baulcombe, 1996, 1999; Ketting et al+, 1999; <br />                                                                         2000; Zamore et al+, 2000)+ Intriguingly, Hamilton &amp; Baul- <br /> Tabara et al+, 1999)+ Aspects RNAi related post- <br />                                                                         combe (1999) reported example ap- <br /> transcriptional silencing (PTGS) mechanisms been <br />                                                                         pearance strong siRNA signal dependent on <br /> conserved diverse phyla eukary- <br />                                                                         presence RNA target+ <span id='am-43' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-44'  ></span><span id='am-45' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> suggests a <br /> otic kingdoms (reviewed Cogoni &amp; Macino, 2000; <br />                                                                         population siRNAs formed initial cleavage of <br />                                                                         trigger RNA subjected subsequent (a) <br />                                                                         amplification target-dependent copying process, <br />    Reprint requests to: Andrew Fire, Carnegie Institution Washing- <br /> ton, Department Embryology, 115 West University Parkway, Bal-        and/or (b) stabilization presence mRNA <br /> timore, Maryland 21210, USA; e-mail: fire@ciwemb+edu+                   target+ <br />                                                                     1397 <br />  1398                                                                                              S. Parrish A. Fire <br />  <br />    The siRNAs possess characteristic structure,            The description molecular intermediates re- <br /> a 59 phosphate, 39 hydroxyl, two- three-base          action, combined availability numerous mu- <br /> 39 overhang strand duplex (Elbashir             tants affecting mechanism provides excellent <br /> et al+, 2001b)+ This siRNA structure bears striking          starting point construction functional path- <br /> resemblance products formed RNase III cleavage           way (Edgar &amp; Wood, 1966)+ The straightforward <br /> of long dsRNAs+ Consistent hypothesis         starting point analysis genes re- <br /> RNase III-like activity involved RNAi, Bernstein         quired RNAi essential viability+ Such cases <br /> and colleagues (2001) demonstrated pro-            allow clear analysis reaction living system <br /> tein homology RNase III (DICER) present          completely lacking specific gene product+ Mutations <br /> Drosophila extracts capable producing short          partial loss RNAi accom- <br /> RNA products longer dsRNA trigger+ Impor-               panying fertility defects somewhat chal- <br /> tantly, synthetic 21- 22-nt RNA duplexes mimic         lenge analysis organism+ For case, <br /> the structure siRNAs capable effectively trig-       rescue maternal dowry, overlapping gene function, <br /> gering RNAi vitro vivo, implicating siRNAs        considerations prevent complete loss RNAi <br /> direct participants RNAi process (Caplen et al+,        activity achieved living system+ <br /> 2001; Elbashir et al+ 2001a, 2001b)+ The siRNAs, prob-            In work, utilized assays production of <br /> ably assembled ribonucleoprotein complex,            siRNAs interference siRNAs rde-1 rde-4 <br /> been shown direct cleavage cognate mRNA              mutant animals begin constructing pathway the <br /> within region homology, single cleavage        RNAi reaction+ <br /> site located center protected area (Zamore <br /> et al+, 2000; Elbashir et al+, 2001b)+ <br />                                                                RESULTS AND DISCUSSION <br />    One approach understand silencing mecha- <br /> nism relied genetic screens genes can <br />                                                                Genetic requirements production of <br /> mutate silencing-deficient phenotype (Cogoni &amp; <br />                                                                siRNAs RNA interference <br /> Macino, 1997; Elmayan et al+, 1998; Tabara et al+, 1999)+ <br /> One <span id='am-40' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-41' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> mutations Caenorhabditis elegans, con-         We previously described vivo assay fol- <br /> sisting alleles genes rde-1 rde-4 (rde:          low fate dsRNA trigger C. elegans (Parrish <br /> RNAi defective), shows global RNAi resistance          et al+, 2000)+ Briefly, C. elegans adults injected with <br />                                                                32 <br /> any evidence additional phenotypic consequences                P-labeled dsRNA harvested RNA extraction <br /> (Tabara et al+, 1999)+ The rde-1 gene cloned          12 h growth 16 8C+ RNA molecules the <br /> and encodes member large PIWI/STING/                  injected animals resolved denaturing polyacryl- <br /> Argonaute/Zwille/eIF2c protein <span id='am-25' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-26' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> (e+g+, 24 rep-      amide gels visualized phosphorimager analy- <br /> resentatives C. elegans; Tabara et al+, 1999)+ Analyses     sis+ In wild-type recipient animals, assay detects a <br /> of PIWI/STING/Argonaute/Zwille/eIF2c homologs               small proportion (approximately 1%) dsRNA trig- <br /> other systems suggested similar             ger processed small RNAs (Parrish et al+, <br /> to rde-1 exhibiting specific role RNAi/PTGS,           2000)+ These small RNAs approximately 25 nt in <br /> whereas involved developmental regula-           length, similar siRNAs observed RNAi di- <br /> tion gene expression (Cox et al+, 1998; Catalanotto         verse species (Hamilton &amp; Baulcombe, 1999; Ham- <br /> et al+, 2000; Fagard et al+, 2000; Grishok et al+, 2001)+      mond et al+, 2000; Parrish et al+, 2000; Yang et al+, <br /> Certain members family dual function          2000; Zamore et al+, 2000)+ To investigate genetic <br /> in development genome surveillance (Fagard et al+,         requirements production siRNAs, utilized this <br /> 2000)+                                                         vivo assay assess siRNA formation rde-1 and <br />    Consistent hypothesis mechanistic com-          rde-4 mutants+ <br /> monality RNAi developmental control pro-              In mutant strains carrying different rde-1 <br /> cesses, large class mutations shown           alleles (rde-1 (ne219 ) rde-1 (ne300 ); Tabara et al+, <br /> affect RNAi developmental progression+ In C.          1999), levels ;25 nt siRNAs produced the <br /> elegans, mutations genes rde-2, rde-3, mut-2,           dsRNA trigger comparable wild-type N2 <br /> mut-7, mut-8, mut-9, ego-1, exhibit variable de-           worms (ne300 : Fig+ 1; ne219 : <span id='am-61' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-62' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-64'  ></span><span id='am-65' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> shown)+ This <br /> grees RNAi resistance accompanied host              indicates RDE-1 required initial pro- <br /> other phenotypes, defective chromosome seg-            cessing trigger RNA siRNAs+ The rde-1 <br /> regation (all), increased transposon mobilization (rde-2,      alleles used distinct molecular lesions: rde- <br /> rde-3, mut-7, -8, -9), chromatin condensation          1 (ne300 ) predicted null contains premature <br /> defects (mut-7 ) (Ketting et al+, 1999; Tabara et al+, 1999;   stop <span id='am-48' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-49'  ></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span>, rde-1 ( ne219 ) harbors mis- <br /> Dernburg et al+, 2000; Ketting &amp; Plasterk, 2000; Smar-         sense mutation conserved PIWI domain (Tabara <br /> don et al+, 2000)+ All mutations produce steril-      et al+, 1999)+ <br /> ity, does loss function C. elegans DICER            When rde-4 ( ne299 ) animals (Tabara et al+, 1999) were <br /> homolog dcr-1 (Grishok et al+, 2001)+                          examined assay, reduction of <br />  Functions RDE-1 RDE-4 RNAi                                                                                               1399 <br />  <br />  <br />  <br />  <br />                                                                          FIGURE 2. Reduced levels ;25-nt siRNA products dsRNA <br />                                                                          trigger rde-4 mutants+ Labeled dsRNA injected N2 and <br />                                                                          rde-4(ne299) adults (grown 16 8C 20 8C) analyzed de- <br />                                                                          scribed Figure 1+ End-labeled ssRNA oligos (zr1 zr2, each <br />                                                                          26 nt length zr10, 27 nt length; Parrish et al+, 2000) were <br />                                                                          utilized markers+ <br /> FIGURE 1. Processing injected dsRNA ;25-nt siRNAs not <br /> impaired rde-1 mutants+ 32 P-labeled dsRNA injected N2 <br /> (wild-type) rde-1(ne300) C. elegans adults+ RNAs uni- <br /> formly labeled vitro synthesis presence 32 P-UTP (to a <br /> specific activity 15 Ci/mMol UTP) injected concentration     temperatures paralleled overall RNAi sensitivity+ At 16 8C, <br /> of ;2+5 mg/mL+ Animals allowed recover 12 h 16 8C         siRNA products injected RNA detected <br /> and subsequently harvested RNA extraction+ Samples re-          rde-4 ( ne299 ) animals, present level <br /> solved electrophoresis 12% denaturing polyacrylamide gels <br /> and visualized phosphorimager analysis+ Two different images       substantially levels wild-type (Fig+ 2)+ <br /> the gel shown; high gain image located left low    When animals grown 20 8C (Fig+ 2) 23 8C <br /> gain image right+ The t 5 0 sample contained mixture      (data shown), levels small RNA generated in <br /> C. elegans labeled dsRNA subjected stan- <br /> dard RNA extraction+ End-labeled ssRNA oligos (zr1 zr2,         rde-4 ( ne299 ) consistently detection <br /> 26 nt length; Parrish et al+, 2000) utilized markers+         threshold assay+ These results suggest in- <br />                                                                          volvement (direct indirect) RDE-4 initial <br />                                                                          processing dsRNA trigger+ <br /> siRNA generated trigger observed, a <br /> more dramatic decrease seen elevated tempera- <br />                                                                          Assays interference siRNAs <br /> tures (Fig+ 2)+ The RNAi process rde-4 (ne299 ) ap- <br />                                                                          distinguish roles RDE-1 <br /> pears somewhat temperature sensitive, allowing <br />                                                                          RDE-4 RNAi <br /> no interference 20 8C 23 8C, permitting weak <br /> interference grown 16 8C (data shown)+                       In experiments performed Drosophila embryo ex- <br /> At present data distinguish                        tracts, siRNAs derived dsRNA trigger have <br /> temperature effects ne299 allele reflect                      shown possess characteristic structure with <br /> temperature-sensitive protein (i+e+, residual RDE-4 pro-                 59 phosphate, 39 hydroxyl, 39 two- three- <br /> tein activity 16 8C) temperature-sensitive pro-                  base overhang strand duplex (Elbashir <br /> cess (ability RNAi reaction proceed low                   et al+, 2001b)+ Synthetic dsRNA oligos siRNA <br /> level absence RDE-4 16 8C)+ Assays                      structure shown capable triggering <br /> initial siRNA formation rde-4 mutant different                 RNAi injection C. elegans (Caplen et al+, 2001)+ <br />  1400                                                                                                           S. Parrish A. Fire <br />  <br /> To determine roles RDE-1 and/or RDE-4                  tion 24-nt 25-nt siRNA+ Although injection <br /> be bypassed precleavage trigger RNA,                      siRNAs produced interference rde-4 mutant <br /> examined chemically synthesized siRNAs                   background, bypass block interference ap- <br /> produce phenotypic effect rde-1 rde-4 mutants+                 peared incomplete+ Evidence incomplete resto- <br />   The siRNAs able bypass RNAi de-                      ration interference comes comparison the <br /> fects rde-1 (ne300 ) animals (Fig+ 3)+ siRNAs contain-              fraction progeny affected wild-type versus rde-4 <br /> ing 24 25 nt unc-22 sequence, produced                     mutant animals injected 25-nt siRNA (approx- <br /> the characteristic specific twitching phenotype in-                imately threefold increase wild-type animals)+ In- <br /> dicative unc-22 interference wild-type animals (Fire             terestingly, blunt-ended 26-nt dsRNA did trigger <br /> et al+, 1998; Caplen et al+, 2001), produced interfer-              significant interference rde-4 mutant animals, de- <br /> ence rde-1 (ne300 ) animals+ Similarly, blunt-ended               spite triggering modest interference wild-type worms+ <br /> dsRNA oligo containing 26 nt mediate inter-                  This result implies structural modifications im- <br /> ference rde-1 ( ne300 ), dsRNA oligo 81 nt              parted RNase III cleavage event important <br /> in length+ The inability siRNAs bypass RNAi                  generating efficient RNAi+ For rde-4 ( ne299 ), there <br /> resistance rde-1 mutants suggests defect                slight background level interference the <br /> mutant lies downstream trigger processing+ This hy-                 26-nt 81-nt dsRNAs; resulted from <br /> pothesis consistent observation (Fig+ 1)              degradation RNA preparations, result- <br /> that ;25-nt siRNAs present injection la-                  ing small population RNAs resembling struc- <br /> beled dsRNA rde-1 mutants+                                        ture siRNAs+ The ability siRNAs bypass the <br />   We synthetic siRNAs able partially                RNAi defect rde-4 mutant, addition re- <br /> bypass requirement RDE-4 interference                   duced ;25-nt siRNAs generated RNAi <br /> response (Fig+ 3)+ For alleles rde-4 (ne299                 mutant (Fig+ 2), suggests RDE-4 involved <br /> ne301 ), interference induced 20 8C injec-                generating siRNAs dsRNA trigger molecules+ <br />  <br />  <br />  <br />  <br />        FIGURE 3. siRNAs bypass RNAi resistance rde-4 rde-1 mutants+ Shown structures of <br />        synthetic RNAs injected C. elegans (Parrish et al+, 2000, Caplen et al+, 2001)+ Shown interference <br />        activities synthetic RNAs wild-type mutant C. elegans (wild-type data described, Caplen et al+, 2001)+ Double- <br />        stranded RNAs (5 mg/mL) injected gonad C. elegans adults grown 20 8C+ Percentages represent the <br />        fraction F1 progeny exhibiting unc-22 twitching phenotype 0+33 mM levamisole+ Numbers parentheses indicate <br />        total number animals scored+ <br />  Functions RDE-1 RDE-4 RNAi                                                                                 1401 <br />  <br /> Sequential activity RDE-4                                   subsequent steps utilize short segment of <br /> and RDE-1 RNAi                                              guide RNA means recognize and/or act a <br />                                                                target transcript+ <br /> Our data consistent sequential model which <br /> RDE-4 involved step prior initial <br /> cleavage dsRNA trigger produce siRNA-type            MATERIALS AND METHODS <br /> molecules, RDE-1 involved subsequent <br /> step interference (Fig+ 4)+ As complex <br />                                                                Strains <br /> multiprotein/multi-RNA assemblies, events RNAi <br /> might occur short time range large           The C. elegans stock N2 (Brenner, 1974) used the <br /> complex (e+g+, Bernstein et al+, 2001)+ There          representative wild-type strain+ Strains carrying RNAi- <br /> likely degree freedom formation           defective mutations rde-1 ( ne219 ), rde-1 ( ne300 ), rde- <br /> and action protein:RNA complexes execute RNAi+         4 ( ne299 ), rde-4 ( ne301 ) (Tabara et al+, 1999) each <br /> It pointed experiments           extensively backcrossed wild-type (and gifts H+ <br />                                                                Tabara C+ Mello)+ Each strain contained genetic <br /> rule association RDE-1 precleavage <br />                                                                markers+ <br /> RNAi complex, rule continued as- <br /> sociation RDE-4 complex cleavage+ <br />    Figure 4 shows working model roles           Assays fate dsRNA trigger <br /> RDE-1, RDE-4, siRNAs interference pro- <br /> cess: RDE-4 invoked recognition long              The vivo fate radioactively labeled trigger molecules was <br /> duplex RNA molecules aberrant RNAs (allowing                analyzed described previously (Parrish et al+, 2000)+ Ra- <br />                                                                dioactive RNA synthesized incorporation 32 P- <br /> cleavage form characteristic siRNA guide), <br />                                                                UTP (specific activity, 15 Ci/mMol UTP) standard T3/ T7 <br /> whereas RDE-1 (and homologous factors) invoked <br />                                                                polymerase reactions followed RNA extraction, sense/ <br />                                                                antisense hybridization, injection C. elegans a <br />                                                                concentration ;2+5 mg/mL+ The injected dsRNA mol- <br />                                                                ecules contained sequence gfp (326 nt), unc-22 (743 <br />                                                                nt), restriction enzyme linkers (103 nt)+ Following <br />                                                                growth 12 h 16 8C, injected animals harvested for <br />                                                                RNA extraction+ RNAs resolved 12% denaturing poly- <br />                                                                acrylamide gels visualized using Molecular Dynamics <br />                                                                Storm Phosphorimager+ <br />  <br />  <br />                                                                Assays interference small RNAs <br />                                                                RNA oligonucleotides containing sequences unc-22 <br />                                                                gene (Fig+ 3) chemically synthesized using 29-O -(tri- <br />                                                                isopropyl)silyloxy-methyl chemistry Xeragon AG (Zurich, <br />                                                                Switzerland; Parrish et al+, 2000; Caplen et al+, 2001)+ The <br />                                                                26-nt 81-nt RNA oligomers purified anion <br />                                                                exchange HPLC+ RNA oligos annealed injected <br />                                                                C. elegans strains (grown 20 8C) concentration of <br />                                                                ;5 mg/mL+ The degree interference measured the <br />                                                                fraction progeny exhibiting Unc-22 twitching phenotype <br />                                                                0+33 mM levamisole+ <br />  <br />  <br />                                                                ACKNOWLEDGMENTS <br />                                                                We thank Hiroaki Tabara, Craig Mello, Lisa Timmons, Rich- <br />                                                                ard Morgan, Natasha Caplen numerous suggestions <br />                                                                continued collaboration, R+ Alcazar, J+ Fleenor, S+ <br />                                                                Kostas, K+ Liu, J+ Yanowitz, T+ Tuschl, J+ Bender, C+ Fan, P+ <br />                                                                Sharp, R+ Plasterk valuable discussions+ Research <br />                                                                support National Institutes Health (Grants <br />                                                                GM37706 GM07231) Carnegie Institution of <br />                                                                Washington gratefully acknowledged+ <br />  <br />                                                                Received June 18, 2001; returned revision <br /> FIGURE 4. A working model showing roles RDE-1, RDE-4,   June 27, 2001; revised manuscript received <br /> siRNAs interference reaction+                           July 9, 2001 <br />  1402                                                                                                              S. Parrish A. Fire <br />  <br /> REFERENCES                                                                   Vaucheret H+ 1998+ <span id='am-9' about='obo:NCBITaxon_3701' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_980083 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Cardaminopsis" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45250" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Arabidopsis" datatype="xsd:string"></span><span id='am-15'  ></span>Arabidopsis</span> mutants impaired cosuppres- <br />                                                                              sion+ Plant Cell 10 :1747&#8211;1757+ <br /> Baulcombe D+ 1996+ RNA target initiator post-             Fagard M, Boutet S, Morel J, Bellini C, Vaucheret H+ 2000+ AGO1, <br />    transcriptional gene silencing transgenic plants+ Plant Mol Biol       QDE-2, RDE-1 related proteins required post- <br />    32 :79&#8211;88+                                                                transcriptional gene silencing plants, quelling <span id='am-16' about='obo:NCBITaxon_4751' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2759'><span id='am-17' property="oboInOwl:hasDbXref" content="PMID:11062127" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="fungi" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Fungi" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="PMID:12684019" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="fungi" datatype="xsd:string"></span><span id='am-24'  ></span>fungi</span>, and <br /> Baulcombe D+ 1999+ <span id='am-1' about='obo:NCBITaxon_10239' typeof='owl:Thing obo:NCBITaxon_1'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Viruses" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="Viridae" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="Vira" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="viruses" datatype="xsd:string"></span>Viruses</span> gene silencing plants+ Arch Virol          RNA interference animals+ Proc Natl Acad Sci USA 97 :11650&#8211; <br />    Suppl 15 :189&#8211;201+                                                        11654+ <br /> Bernstein E, Caudy A, Hammond S, Hannon G+ 2001+ Role              Fire A, Xu S, Montgomery M, Kostas S, Driver S, Mello C+ 1998+ <br />    bidentate ribonuclease initiation step RNA interf <br /> </body></html>