<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />  <br />  <br />  <br />                            A high-efficiency method to replace essential genes with mutant <br />                            alleles in yeast <br />  <br />  <br />                            Abstract <br />                                 Temperature-sensitive (TS), internally deleted and truncated alleles are important tools to facilitate <br />                                 the characterization of essential genes. We have developed a straightforward method to replace a <br />                                 wild-type gene with a mutant allele at the endogenous locus. <span id='am-11' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-12'  ></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> method is an efficient alternative <br />                                 to the two-step method for integration of alleles that are compromised in function or contain multiple <br />                                 mutations. A strain is constructed that has the essential gene of interest disrupted by a selectable <br />                                 marker. Strain viability is maintained by a plasmid carrying a copy of the essential wild-type gene <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 and theADE3 gene. The mutant allele is cloned into an integratable vector carrying a selectable/ <br />                                 counter-selectable marker, such asURA3. The plasmid is linearized and transformed, directing <br />                                 integration to the 5&#8242; or 3&#8242; region flanking the essential open reading frame (ORF). Transformants that <br />                                 have integrated the mutant gene at the endogenous locus can lose the autonomous plasmid carrying <br />                                 the wild-type copy of the essential gene and theADE3gene. These transformants are identifiable as <br />                                 white sectoring colonies, display the mutant phenotype and may be characterized. An optional second <br />                                 selection step on 5-fluoroorotic acid (5-<span id='am-1' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-2' property="rdfs:label" content="Foa" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5'  ></span>FOA</span>) selects for popouts of the integrating vector sequences, <br />                                 leaves the mutant allele at the endogenous locus, and recycles selectable markers. We have used this <br />                                 method to integrate a TS allele of SPC110 that could not be integrated by standard methods. <br />  <br />  <br />                            Keywords <br />                                 temperature-sensitive; gene replacement; integration; plasmid shuffle; conditional allele; <br />                                 counterselection <br />  <br />  <br />                            Introduction <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Mutant alleles are important tools to provide insight into the functions of essential genes. A <br />                                             number of simple and high-efficiency methods have been developed for the replacement of <br />                                             non-essential genes with mutant alleles in yeast (Struhl, 1983; Szent-Gyorgyi, 1996). However, <br />                                             when working with essential genes, a classic two-step gene replacement method has been the <br />                                             main method used, since a functional copy of the gene in question must be present at all times <br />                                             to maintain strain viability (Scherer and Davis, 1979; Winston et al., 1983; Boeke et al., <br />                                             1987; Rothstein, 1991). The basic strategy is to introduce the mutant allele by linearizing an <br />                                             integratable plasmid containing the mutant allele and URA3, a selectable/counter-selectable <br />                                             marker. Linearization directs integration to the endogenous locus of the gene, resulting in a <br />                                             tandem array of the wild-type and mutant allele flanking the URA3 marker. After counter- <br />                                             selection on 5-FOA, the strains carrying the mutant allele are then identified by screening. <br />  <br />                                             There are a few notable drawbacks to the two-step approach. The screen for the mutant allele <br />                                             can be time consuming if the mutant does not display an obvious phenotype. Alternatively, <br />  <br />  <br />                            *Correspondence to: Trisha N. Davis, Department of Biochemistry, Box 357350, University of Washington, Seattle, WA 98195-7350, <br />                            USA. E-mail: tdavis@u.washington.edu <br />                             Widlund and Davis                                                                                             Page 2 <br />  <br />  <br />                                                one may have to screen through a large number of colonies before isolating the strain with the <br />                                                gene-replaced allele because the mutant allele is selected against if the mutant is less fit <br />                                                compared to wild-type. In this case, it may be extremely difficult to replace the wild-type with <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                the mutant allele. Also, if mutations are spread through a significant portion of the essential <br />                                                ORF, any number of them may be lost during integration or in the recombination event required <br />                                                for growth on 5-FOA. Other gene replacement strategies will work in many cases, but they are <br />                                                susceptible to gene conversion because wild-type sequence is present at the site of integration <br />                                                (Shortle et al., 1984; Erdeniz et al., 1997). <br />  <br />                                                We have developed a high-efficiency method that overcomes these difficulties and allows <br />                                                replacement of an essential gene with any non-lethal mutation. <br />  <br />                               Materials and methods <br />                               Media <br />                                                YPD medium and SD medium were made as described (Sherman et al., 1986). SD complete <br />                                                is SD medium supplemented with 50 &#956;g/ml adenine, 25 &#956;g/ml uracil, 100 &#956;g/ml tryptophan, <br />                                                and 0.1% casamino acids. SD-tryptophan is SD complete lacking tryptophan. SD-uracil low <br />                                                adenine is SD complete lacking uracil and contains 5 &#956;g/ml adenine. 5-FOA medium is SD <br />                                                complete with 100 &#956;g/ml uracil and supplemented with 1 mg/ml 5-FOA (Boeke et al., 1987). <br />                                                YP glycerol medium is YPD medium with 2% glycerol substituted for glucose. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               <span id='am-39' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-40' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>Plasmids</span> <br />                                                Plasmids used in this study are listed in Table 1. Plasmid pHS55 is the integratable version of <br />                                                plasmid pHS54. Plasmid pHS54 carries the spc110-227 allele, which was recovered in the TS <br />                                                screen described previously(Sundberg and Davis, 1997). <br />  <br />                               Strains <br />                                                The yeast strain used in this study is HSY2-12C. Its genotype is MATa ade2-1oc ade3&#916; <br />                                                can1-100 his3-11,15 leu2-3,112 lys2&#916;::HIS3 spc110&#916;/I&gt;::TRP1 trp1-1 ura3-1 carrying <br />                                                plasmid pHS26. It is a haploid isolate of HSY2 and is a derivative of W303(Geiser et al., <br />                                                1993). The <span id='am-16' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-26'  ></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span>Escherichia coli</span> strain used for plasmid preparations was XL-1 Blue (Stratagene). <br />  <br />                               Transformation and selection <br />                                                pHS55 (1 &#956;g) was linearized with SnaBI. The linearized DNA was transformed into the <br />                                                equivalent of 5 ml of a culture of HSY2-12C at 100 Klett units, using the LiAc TRAFO method <br />                                                (Schiestl and Gietz, 1989). The transformation reaction was plated on five SD-uracil low <br />                                                adenine plates and incubated at room temperature for 7 days. Sectored colonies were isolated <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                and re-streaked on SDuracil low adenine to obtain single, uniformly white colonies. Single <br />                                                white colonies from each sectored isolate were tested for growth on YP glycerol to eliminate <br />                                                isolates that were petite. The remaining white isolates were re-streaked on YPD in duplicate. <br />                                                One set was incubated at room temperature for 2 days to allow for recombination and loss of <br />                                                the URA and TRP markers at the SPC110 locus, while the other set was incubated at 37 &#176;C for <br />                                                2 days to test temperature sensitivity. A sample from each room-temperature streak was struck <br />                                                heavily on 5-FOA plates and incubated at room temperature for 5 days. Single colonies were <br />                                                isolated from the 5-FOA plates and tested to see whether they had lost the ability to grow on <br />                                                medium lacking tryptophan. <br />  <br />                               PCR <br />                                                Colony PCR using two pairs of primers was performed at three steps during the gene <br />                                                replacement. Primer positions are illustrated in Figure 2d. Primer 1 anneals in the middle of <br />  <br />  <br />                                                      Yeast. Author manuscript; available in PMC 2006 December 12. <br />                             Widlund and Davis                                                                                                  Page 3 <br />  <br />  <br />                                                the TRP1 ORF and has the sequence 5&#8242;-TAATTTCACAGGTAGTTC-3&#8242;. Primer 2 is primer <br />                                                T7 with sequence 5&#8242;-TAATACGACTCACTATAGGG-3&#8242;. Primers 3 and 4 span the SPC110 <br />                                                ORF and have the sequences 5&#8242;-TGCGCGGATATCTTAAGCAA-3&#8242; and 5&#8242;- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                TTGAAGATCAGTCTGCGTAGG-3&#8242;, respectively. Colony PCR was performed in 50 &#956;l <br />                                                reactions containing 1 &#215; GC buffer, 200 &#956;M dNTPs, 0.5 &#956;M each primer, and 0.5 &#956;l Phusion&#8482; <br />                                                DNA polymerase (Finnzymes). Reactions were cycled 30 times at 98&#176;C for 10 s, 50&#176;C for 20 <br />                                                s and 72&#176;C for 2 min, with a final extension step of 72&#176;C for 7 min. PCR products were separated <br />                                                on a 1% agarose gel withm a 1 kb ladder (NEB). <br />  <br />                               Sequencing <br />                                                Colony PCR was performed using primers 3 and 4 as described above. The 3 kb products were <br />                                                purified with the QIAquick&#174; PCR purification kit (Qiagen). Sequencing was done with primer <br />                                                5&#8242;-TGAAATCTGAACAGAGTA-3&#8242;. <br />  <br />                               Results and discussion <br />                                                We have developed a new method for gene replacement that is based on a plasmid shuffle <br />                                                strategy (Bender and Pringle, 1991). The essential ORF of interest is replaced with any <br />                                                selectable marker except URA3 in a diploid strain that is homozygous for the ade2-1oc mutation <br />                                                and ade3&#916;. PCR cassette based knock-out strategies using kanamycinor hygromycin resistance <br />                                                genes have made this process simple and efficient (Wach et al., 1994; Goldstein and McCusker, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                1999). The strain is then transformed with a plasmid containing the essential ORF with flanking <br />                                                upstream and downstream sequence (typically &#8764;500 bp 5&#8242; and 3&#8242;), and the ADE3 gene. The <br />                                                ADE3 gene can be used for selection on medium lacking histidine if no HIS genes are <br />                                                compromised. If not, a selectable marker other than URA3 must be included on the ADE3 <br />                                                plasmid. We routinely use a plasmid carrying a 2 &#956;m origin of replication, ADE3 and LYS2 for <br />                                                selection after transformation, which is made using plasmids pLI831, pTD29 and the essential <br />                                                gene of interest (Muller 1996). Selection for the ADE3 plasmid is followed by sporulation and <br />                                                isolation of a haploid carrying the disrupted ORF and the ADE3 plasmid. The strain must <br />                                                maintain the ADE3 plasmid for viability, since it carries the only copy of the essential ORF. <br />                                                Due to the maintenance of ADE3 in an ade2-1oc background, colonies of this strain are red, <br />                                                especially when supplied minimal amounts of adenine. This strain is called the &#8216;plasmid shuffle <br />                                                strain&#8217; (Figure 1a). <br />  <br />                                                Following strain construction, the mutant ORF with flanking sequence is cloned into an <br />                                                integratable vector containing the URA3 gene, such as pRS306 (Sikorski and Hieter, 1989). <br />                                                The plasmid is then linearized with a unique restriction site in the 5&#8242; or 3&#8242; region of the essential <br />                                                gene. If a unique site is not available, one must be engineered. We generally use a unique site <br />                                                in the 3&#8242; region. Transformation of the linearized plasmid into the plasmid shuffle <span id='am-6' about='obo:NCBITaxon_189528' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_189527 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_9219 obo:NCBITaxon_436491'><span id='am-7' property="rdfs:label" content="Indicator" datatype="xsd:string"></span><span id='am-8'  ></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>indicator</span> <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                strain directs integration into the 3&#8242; region of the essential gene, which is present at the <br />                                                endogenous locus or into the 3&#8242; region of the copy of the gene in the ADE3 plasmid (Figure <br />                                                1b). Integration into the endogenous locus allows for loss of the ADE3 plasmid, while <br />                                                integration into the ADE3 plasmid makes its loss lethal, since it carries both functional copies <br />                                                of the essential gene. Consequently, when the transformation reaction is plated on SD-uracil <br />                                                low adenine plates, the former event will result in white sectored colonies, while the latter will <br />                                                give solid red colonies (Figure 1c). Solid white colonies are isolated from the sectored colony <br />                                                by re-streaking for single colonies. Loss of mitochondrial DNA can cause a white sectoring <br />                                                colony, but these colonies are easily identified by their inability to grow on medium with <br />                                                glycerol as the sole carbon source. The white colony able to grow on YP glycerol has the <br />                                                URA3 marker flanked by the disrupted ORF and the mutant gene at the endogenous locus <br />                                                (Figure 1c). The mutant phenotype will be apparent at this stage. Plating on medium containing <br />                                                5-FOA selects for recombination events that remove the URA3 marker (Figure 1d). One of <br />                                                these events will remove the sole functional copy of the essential gene, resulting in a non-viable <br />  <br />                                                      Yeast. Author manuscript; available in PMC 2006 December 12. <br />                             Widlund and Davis                                                                                                 Page 4 <br />  <br />  <br />                                                strain. The other event will result in a viable strain that carries the desired mutant allele at the <br />                                                endogenous locus. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                As proof of principle, we integrated spc110-227, which is a temperature-sensitive allele of a <br />                                                gene encoding a spindle pole body component (Sundberg and Davis, 1997). This allele falls <br />                                                within the same complementation group as the spc110-226 allele, which has defects in spindle <br />                                                assembly and cannot achieve a bioriented spindle (Yoder et al., 2005). Previous attempts to <br />                                                integrate the spc110-227 allele by the classical two-step method had failed. <br />  <br />                                                The plasmid shuffle indicator strain was made as described previously (Geiser et al., 1993). In <br />                                                this case, TRP1 replaces the SPC110 ORF and plasmid pHS26 is the ADE3 plasmid. <br />                                                Linearization and transformation of 1 &#956;g plasmid pHS55, which contains the spc110-227 allele, <br />                                                yielded 715 transformants. Of those, 21 sectored white (Figure 2a). Sectored colonies were re- <br />                                                streaked to isolate single, uniformly white colonies. Seven of the isolates generated only white <br />                                                colonies unable to grow on YP glycerol, indicating that these colonies were petite and not of <br />                                                interest. All remaining 14 isolates were temperature-sensitive at 37&#176;C (Figure 2b). The 14 <br />                                                isolates were all able to generate colonies on 5-FOA that were also subsequently unable to <br />                                                grow on medium lacking tryptophan, confirming loss of the spc110::TRP1 allele by <br />                                                recombination. We monitored this process by colony PCR (Figure 2c,d). Finally, all 14 isolates <br />                                                contained all of the mutations present in the SPC110 ORF in plasmid pHS55, as confirmed by <br />                                                sequencing. The spc110-227 allele has seven mutations spanning approximately 350 base pairs. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Five of the seven mutations (A2305G, T2405C, A2434G, G2452A and T2491A) result in the <br />                                                published amino acid changes (R784G, M802T, K812E, E818K and Y831N) (Sundberg and <br />                                                Davis, 1997). The remaining two mutations (A2322G and A2673G) are silent. Base pairs and <br />                                                residues are numbered from the start of the SPC110 ORF. <br />  <br />                                                The increased efficiency of this method compared to the classic two-step method can be <br />                                                attributed mainly to the extent of selection against the mutant allele. In the two-step approach, <br />                                                mutations can be lost during integration because wild-type sequence is present at the site of <br />                                                recombination. Degradation of the ends of the linearized integratable vector increases this <br />                                                possibility. After selection for the integration event, one must allow sufficient cell divisions <br />                                                for a popout event to occur. Selection against the mutant allele occurs directly after <br />                                                recombination, progressively decreasing the fraction of the total cells with the mutant allele <br />                                                with each cell division. Depending on the severity of the selection, this may make the possibility <br />                                                of isolating the mutant very low. Our method minimizes these problems. During integration, <br />                                                mutations in the ORF of the mutant allele are not lost because wild-type sequence is not present <br />                                                at the endogenous locus. Gene conversion with the wild-type sequence on the autonomous <br />                                                plasmid is possible but this event is very infrequent, since we rarely see the plasmid shuffle <br />                                                strain sector on its own and this rare event must also result in a URA+ colony for it to be picked <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                up in our screen. After transformation, selection against the mutant allele only affects the extent <br />                                                of sectoring in the colony that arose from a cell that integrated the mutant allele at the <br />                                                endogenous locus. Even a colony containing an exceptionally compromised allele at the <br />                                                endogenous locus will eventually sector to an extent that can be seen, making this assay very <br />                                                sensitive. As soon as sectoring occurs, selection against the mutant allele cannot occur and no <br />                                                reversions are possible, since no wild-type sequence is present in the mutant white colony. <br />  <br />                                                This method can be used as both a one-step and a two-step method. The white colonies display <br />                                                the mutant phenotype; consequently, any allele may be characterized after a single <br />                                                transformation and selection. The optional second popout step on 5-FOA can be done at any <br />                                                time and is useful when one needs to make the two selectable markers available. The method <br />                                                is also readily available to those already using a red/white screen for TS mutants. One only <br />                                                needs to convert recovered mutant alleles that are on ARS/CEN plasmids to integratable <br />                                                versions. Conveniently, it can also be modified for use in the URA-based TS screen, where a <br />  <br />  <br />                                                      Yeast. Author manuscript; available in PMC 2006 December 12. <br />                             Widlund and Davis                                                                                                            Page 5 <br />  <br />  <br />                                                2 &#956;m or ARS/CEN plasmid contains URA3 and the wild-type allele, and the ARS/CEN vector <br />                                                with the mutant allele has another selectable marker (Boeke et al., 1987). Essentially, <br />                                                integration is performed as described, but selection for the integrants into the genomic locus <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                occurs on 5-FOA medium. If a popout of the integratable plasmid is desired, then the <br />                                                integratable vector must contain a selectable/counter-selectable marker other than URA3, such <br />                                                as LYS2. <br />  <br />   <br />  <br />  <br />                                                Figure 1. <br />                                                (a) The plasmid shuffle strain: a strain has the ORF of interest replaced with any selectable <br />                                                marker except URA3. In the example shown here, TRP1 is used. The 5&#8242;- and 3&#8242; regions flanking <br />                                                the ORF are indicated. A 2&#956;m plasmid carrying the wild-type gene is required to maintain <br />                                                strain viability. The 2&#956;m plasmid also carries the strain&apos;s sole copy of ADE3. The strain carries <br />                                                the ade2-1oc mutation and therefore remains red, since cells losing the 2&#956;m plasmid are not <br />                                                viable. (b) Integration: an integratable plasmid carrying URA3 for selection and the mutant <br />                                                gene is linearized at a unique restriction site marked with an asterisk. Linearization directs <br />                                                integration into the 3&#8242; region of either the endogenous locus or into the ADE3 plasmid. (c) <br />                                                Organization of the essential locus: cells losing the ADE3 plasmid are viable only if integration <br />                                                occurs at the endogenous locus, resulting in a sectored colony. The white colony has the deleted <br />                                                ORF, followed by URA3 and the mutant gene at the endogenous locus. (d) Popout: the white <br />                                                colony is put on 5-FOA to select against the URA3 marker. Recombination between the 3&#8242; <br />                                                regions results in an inviable strain, while recombination between the 5&#8242; regions results in a <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                viable strain with the desired mutations <br />  <br />  <br />  <br />  <br />  <br />  <br />  <br />                                                Figure 2. <br />                                                (a) Sectoring: a single transformation of the linearized integration vector gave 21 sectored <br />                                                colonies. A sample of transformants is shown. The arrow indicates a sectored colony. (b) <br />                                                Phenotype: temperature sensitivity of three representative white colonies compared to a wild- <br />                                                type (wt) strain. Strains were incubated at 37&#176;C for 2 days. (c) Colony PCR using two sets of <br />                                                primers was performed at three stages of the replacement. Lane 1, the plasmid shuffle strain <br />                                                is the template, using primers 1 and 2; lane 2, a URA+ white isolate is the template, using <br />                                                primers 1 and 2; lane 3, a TRP&#8211;URA&#8211;5-FOA isolate is the template, using primers 1 and 2; <br />                                                lane 4, the plasmid shuffle strain is the template, using primers 3 and 4; lane 5, a URA+ white <br />                                                isolate is the template, using primers 3 and 4; lane 6, a TRP&#8211;URA isolate is the template, using <br />                                                primers 3 and 4. Lane M contains a 1 kb ladder. (d) Primer positions at the SPC110 locus. The <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                5&#8242; and 3&#8242; regions on SPC110 are in green and blue, respectively. The mutant spc110-227 <br />                                                ORF is in yellow <br />  <br />  <br />  <br />  <br />                                                      Yeast. Author manuscript; available in PMC 2006 December 12. <br />                                Widlund and Davis                                                                                                            Page 8 <br />  <br />                                                                                           Table 1 <br />                               Plasmids used in this study <br />  <br />                            Plasmid                Parent plasmid                       Relevant markers                              Source or reference <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            pHS26                  pTD29                        SPC110 LYS2, ADE3, 2 &#956;m origin             Geiser et al., 1993 <br />                            pHS54                  pHS32                        spc110-227                                 Sundberg and Davis, 1997 <br />                            pHS55                  pRS306                       spc110-227                                 Sundberg and Davis, 1997 <br />                            pLI831                 DK202                        ADE3                                       Muller, 1996 <br />                            pRS306                                              URA3, f1 origin                            Sikorski and Hieter, 1989 <br />                            pRS316                                              CEN6 ARSH4, URA3, f1 origin                Sikorski and Hieter, 1989 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br /> </body></html>