<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Mar. 2005, p. 2288&#8211;2296                                                                       Vol. 25, No. 6 <br /> 0270-7306/05/$08.00&#11001;0 doi:10.1128/MCB.25.6.2288&#8211;2296.2005 <br /> Copyright &#169; 2005, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />    Inactivation Pre-mRNA Cleavage Polyadenylation Factor <br />           Pfs2 Fission Yeast Causes Lethal Cell Cycle Defects <br />    Shao-Win Wang,1&#8224; Kazuhide Asakawa,2&#8224;&#8225; Thein Z. Win,1&#8224; Takashi Toda,2 Chris J. Norbury3* <br />       Department Zoology1 Sir William Dunn School Pathology,3 University Oxford, Oxford, Cell Regulation <br />                     Laboratory, London Research Institute, <span id='am-44' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-45' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-48'  ></span>Cancer</span> Research UK, London,2 United <span id='am-75' about='obo:NCBITaxon_kingdom' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-76' property="rdfs:label" content="kingdom" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Kingdom</span> <br />                    Received 12 October 2004/Returned modification 17 November 2004/Accepted 14 December 2004 <br />  <br />             Faithful chromosome segregation fundamentally important maintenance genome integrity and <br />          ploidy. By isolating conditional mutants defective chromosome segregation fission yeast Schizosaccha- <br />          romyces pombe, identified role essential gene pfs2 chromosome dynamics. In absence <br />          offunctionalPfs2,chromosomalattachmenttothemitoticspindlewasdefective,withconsequentchromosomemis- <br />          segregation. Under circumstances, multiple intracellular foci spindle checkpoint proteins Bub1 and <br />          Mad2 seen, deletion bub1 exacerbated mitotic defects loss cell viability resulted <br />          loss pfs2 function. Progression G1 S phase following release nitrogen starvation also <br />          required pfs2&#11545; function. The product orthologous <span id='am-1' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-14'  ></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> gene PFS2 component of <br />          multiprotein complex required 3&#11541;-end cleavage polyadenylation pre-mRNAs and, keeping with <br />          conservation essential function, S. pombe pfs2 mutant defective mRNA 3&#11541;-end processing. <br />          Mutations pfs2 suppressed overexpression putative mRNA 3&#11541;-end cleavage factor Cft1. These <br />          <span id='am-83' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-84' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-85' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-87'  ></span>data</span> suggest unexpected links mRNA 3&#11541;-end processing chromosome replication segregation. <br />  <br />  <br />    Following DNA replication eukaryotic cells, accurate mi-              origins replication establishment and <br /> totic chromosome segregation requires bivalent attach-                  maintenance cohesion sister chromatids follow- <br /> ment replicated chromosomes spindle sister                ing DNA replication. The integrity chromosome-as- <br /> kinetochores, followed anaphase movement chromo-                  sociated complexes potentially threatened traversal the <br /> somes opposite spindle poles. Defects process result             chromosomes DNA replication forks RNA polymerases. <br /> either catastrophic failure mitosis cell death              <span id='am-54' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-55'  ></span><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> view emerged identification budding <br /> aneuploidy. Spindle checkpoint mechanisms provide protec-                   yeast <span id='am-18' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-19' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-21'  ></span><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae DNA elements serve to <br /> tion eventualities imposing delay               prevent transcriptional read-through centromeres ori- <br /> onset anaphase chromosomes established sym-               gins replication (27). Maintenance heterochromatin at <br /> metrical, bivalent attachments tense spindle microtubules                centromeres fission yeast <span id='am-25' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-26' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span><span id='am-29'  ></span>Schizosaccharomyces</span> pombe is <br /> (14). The molecular basis checkpoint incompletely                required repression transcription for <br /> understood, proteins involved iden-               accurate chromosome segregation (1). Furthermore, recent <br /> tified characterized yeast models shown perform               study demonstrated cohesin, multiprotein complex <br /> analogous checkpoint functions diverse <span id='am-69' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-70' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span>, including                maintains sister chromatid cohesion, accumulates the <br /> metazoans plants. Such proteins include Bub1, Bub3,                     3&#11032; ends convergent active transcription units (13). <br /> Mps1, Mad1, Mad2, Mad3, thought form                        The mutual incompatibility transcription centromere <br /> heterooligomeric complex unattached kinetochores (2).                    function capacity read-through transcription in- <br /> During progression normal, unperturbed mitosis,                terfere expression downstream genes emphasize <br /> complexes expected short-lived                    importance appropriate termination transcription. <br /> highly dynamic interactions spindle microtubules                For RNA polymerase II (Pol II) transcription units, termina- <br /> kinetochores. Once kinetochores achieved bivalent                  tion generally coupled endonucleolytic cleavage the <br /> attachment, reduction Mad- Bub-dependent signal-                 primary transcript site defining 3&#11032; terminus the <br /> ing allows activation multisubunit ubiquitin ligase known <br />                                                                             mRNA. Termination typically occurs zone lacks <br /> as anaphase-promoting complex/cyclosome subse- <br />                                                                             distinctive features level primary sequence is <br /> quent progression anaphase. <br />                                                                             situated bases downstream cleavage <br />    Normal chromosome behavior requires assembly spe- <br />                                                                             site. In cases, cleavage followed polyadenylation of <br /> cialized protein-DNA complexes telomeres, centromeres, <br />                                                                             nascent mRNA, cleavage polyadenylation <br />                                                                             steps coordinated multisubunit protein complexes. <br />   * Corresponding author. Mailing address: Sir William Dunn School <br />                                                                             These complexes conserved overall organization <br /> of Pathology, University Oxford, South Parks Rd., Oxford OX1             diverse eukaryotes (23) comprise cleavage and <br /> 3RE, United Kingdom. Phone: 44 1865 275540. Fax: 44 1865 275501.            polyadenylation specificity factor (CPSF) cleavage stim- <br /> E-mail: chris.norbury@path.ox.ac.uk.                                        ulation factor (CstF) mammals. In S. cerevisiae, orthologues <br />   &#8224; S.-W.W., K.A., T.Z.W. contributed equally work. <br />   &#8225; Present address: Division Molecular Developmental Biol- <br />                                                                             of subunits CPSF CstF identified <br /> ogy, Department Developmental Genetics, National Institute            components cleavage polyadenylation factor <br /> Genetics, Mishima, Shizuoka, Japan.                                         (CPF). In absence efficient cleavage polyadenyla- <br />  <br />                                                                      2288 <br />  VOL. 25, 2005                                                                                                        Pfs2 AND THE CELL CYCLE                     2289 <br />  <br />  <br />                                                          TABLE 1. S. pombe strains used study <br />        Strain                                                                  Genotype                                                                 Source <br />                                                 &#11002; <br /> 972                                            h                                                                                                Laboratory stock <br /> 428h                                           h&#11002; ade6-M210 leu1-32 ura4-D18 his7                                                               Laboratory stock <br /> 429h                                           h&#11001; ade6-M216 leu1-32 ura4-D18 his7                                                               Laboratory stock <br /> KZ2                                            h&#11001; leu1-32 ade6-M210 bub1&#11001;-GFP-kanr Ch16                                                         This study <br /> KZ50                                           h&#11002; ade6-M210 leu1-32 pfs2-3169 bub1&#11001;-GFP-kanr                                                    This study <br /> KZ52                                           h&#11002; his2 pfs2-3169 mad2&#11001;-GFP-LEU2 bub1&#11001;-HA3-kanr                                                  This study <br /> KZ67                                           h&#11002; ade6-M216 leu1-32 ura4-D18 pfs2-3169 bub1::ura4&#11001;                                              This study <br /> pfs2&#9004;                                          h&#11001;/h&#11002; pfs2::ura4&#11001;/pfs2&#11001; ade6-M210/ade-M216 leu1-32 ura4-D18                                      This study <br /> pfs2&#11001;-GFP                                      h&#11002; pfs2&#11001;-GFP::kanr leu1-32                                                                       This study <br /> P41-pfs2                                       h&#11002; pfs2::ura4-nmt41-pfs2 ade6-M210 leu1-32 ura4-D18 his7                                         This study <br /> P81-pfs2                                       h&#11002; pfs2::ura4-nmt81-pfs2 leu1-32 ura4-D18                                                        This study <br /> pfs2-11                                        h&#11002; pfs2-11                                                                                       This study <br /> bub1&#9004;                                          h&#11002; bub1::ura4&#11001; ade6-M210 leu1-32 ura4-D18 his1                                                   Jean-Paul Javerzat <br /> pfs2-11 bub1&#9004;                                  h&#11002; pfs2-11 bub1::ura4&#11001; ade6-M210 leu1-32 ura4-D18 his1                                           This study <br /> bub1-GFP                                       h&#11002; bub1&#11001;-GFP::kanr leu1-32 ura4-D18                                                              Laboratory stock <br /> pfs2-11 bub1-GFP                               h&#11002; pfs2-11 bub1&#11001;-GFP::kanr leu1-32 ura4-D18                                                      This study <br /> Ch16                                           h&#11002; ade6-M210 ura4-D18 Ch16                                                                       Laboratory stock <br /> pfs2-11 Ch16                                   h&#11002; pfs2-11 ade6-M210 leu1-32 ura4-D18 Ch16                                                       This study <br />  <br />  <br />  <br /> tion, Pol II continues transcribe normal termi-                           Disruption modification pfs2. One-step gene disruption modification <br /> nation zone, potentially disrupting functions genes                       homologous recombination performed following PCR-mediated gener- <br />                                                                                      ation ura4&#11001; kanMX selectable cassettes flanked 80-bp segments from <br /> other chromosomal features downstream. <br />                                                                                      appropriate regions pfs2&#11001; using oligonucleotides described Table 2. <br />    Here characterization fission yeast mu-                      Following transformation diploid strain (428h/429h), Ura&#11001; progeny were <br /> tant identified inability execute mitosis faithfully                  screened desired integration pattern diagnostic PCRs primer <br /> and primary biochemical defect mutant                       pairs spanning presumptive recombination sites (details additional <br /> in component cleavage polyadenylation machin-                           primers used purpose available authors request). Meiosis <br /> ery. These data indicate proper regulation mRNA                          sporulation induced plating malt extract agar, tetrad dissec- <br />                                                                                      tion performed MSM micromanipulator (Singer Instruments, <br /> 3&#11032;-end processing essential normal chromosome                          Watchet, United Kingdom) described (15). The pfs2&#11001; gene was <br /> segregation.                                                                         disrupted using primers DELA DELB, construction chromosomal- <br />                                                                                      ly GFP-tagged pfs2&#11001; strain (pfs2&#11001;-GFP) accomplished using primers <br />                         MATERIALS AND METHODS                                        TAGA TAGB, nmt1 promoter shutoff strains P41-pfs2 P81-pfs2 <br />    Fission yeast methods. Conditions growth, maintenance, genetic        constructed using primers PSOA PSOB (Table 2). In <span id='am-72' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> to <br /> manipulation fission yeast described previously (15). A complete list     generate ts alleles pfs2, low-fidelity PCRs performed primers <br /> of strains used study given Table 1. Except        MUTA MUTB, flank pfs2&#11001; locus, genomic S. pombe <br /> stated, strains grown 30&#176;C yeast extract (YE) Edinburgh minimal        DNA template. The resulting pool mutagenized pfs2 sequences used <br /> medium (EMM2) appropriate supplements. When necessary, gene expres-             transform strain P81-pfs2, transformants selected growth 25&#176;C <br /> sion nmt1 promoter repressed addition 5 &#65533;?&#65533;M thiamine        YE agar containing thiamine, allowing growth cells nmt1 <br /> the growth medium.                                                                   promoter-driven pfs2 allele replaced homologous recombination <br />    The pfs2-3169 mutant identified course large-scale mu-        mutagenized PCR product. The resulting colonies screened ts <br /> tagenesis screen identify essential genes required accurate chromosome        growth replica plating checked loss P81-pfs2 allele PCR, <br /> segregation. Parental strain KZ2 (Table 1), contains minichromosome            sequence change(s) pfs2 identified following PCR amplification of <br /> Ch16 (19) Bub1-green fluorescent protein (GFP), mutagenized             mutant alleles. <br /> nitrosoguanidine previously described (24), spread rich yeast extract        Microscopy. Cells fixed 3.8% formaldehyde washed phosphate- <br /> five supplements (YE5S) plates, incubated 27&#176;C. Colonies replica         buffered saline stained 4&#11032;,6-diamidino-2-phenylindole (DAPI) before <br /> plated YE5S plates 36&#176;C YE plates (rich medium lacking exogenously         examination fluorescence microscopy. Images acquired using Zeiss <br /> added supplements) 32&#176;C, cells lost Ch16 turned red             Axioskop microscope equipped Planapochromat &#11003;100 objective, Axio- <br /> because chromosomal ade6-210 mutation. A total 221 isolates of <br />                                                                                      cam cooled charge-coupled device <span id='am-59' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-62'  ></span><span id='am-63' property="rdfs:label" content="Camera" datatype="xsd:string"></span>camera</span>, Axiovision software (Carl Zeiss <br /> 500,000 mutagenized colonies showed minichromosome loss tempera- <br />                                                                                      Ltd., Welwyn Garden City, United Kingdom). Images assembled using <br /> ture-sensitive (ts) phenotypes; furthermore, 39 mutants showed cosegregation of <br />                                                                                      Adobe Photoshop. In experiments, living cells growing EMM2 medium <br /> minichromosome loss ts phenotypes backcrossing. Among <span id='am-78' about='obo:NCBITaxon_238707' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_8075 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_1489917'><span id='am-79' property="rdfs:label" content="Iso" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-81'  ></span><span id='am-82' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>iso</span>- <br />                                                                                      stained addition 5 &#65533;?&#65533;g/ml bis-benzimide (Hoechst 33342; Sigma) <br /> <span id='am-49' about='obo:NCBITaxon_8186' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_1489905 obo:NCBITaxon_123366 obo:NCBITaxon_1489904 obo:NCBITaxon_1489872 obo:NCBITaxon_8184 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665'><span id='am-50'  ></span><span id='am-51' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="Lates" datatype="xsd:string"></span>lates</span>, mutations 10 complementation groups (cin1 cin10) showed <br />                                                                                      examination fluorescence microscopy. Visualization Pfs2-GFP in <br /> hypersensitivity thiabenzadole compared wild-type strain. cin2-3169 is <br />                                                                                      living cells embedded 0.6% low-melting-point agarose performed room <br /> allelic pfs2. For analysis pfs2-3169, strain KZ52, contains <br /> Mad2-GFP Bub1-hemagglutinin (HA), used.                                      temperature (&#11011;22&#176;C). <br />    Minichromosome loss assays performed strains containing non-           Flow cytometry. Cells fixed 70% ethanol rehydrated 10 mM <br /> essential ade6-M216-marked Ch16 minichromosome derivative chromosome 3            EDTA (pH 8.0)&#8211;0.1 mg RNase A/ml&#8211;1 &#65533;?&#65533;M Sytox green incubated 37&#176;C <br /> (19). The ade6-M216 allele complements unlinked ade6-M210 marker         2 h. Cells analyzed using Epics XL-MCL flow cytometer (Coulter, <br /> strains remain Ade&#11001; long minichromosome main-            Fullerton, Calif.). <br /> tained. Chromosome loss measured progeny single Ade&#11001; cell             RT-PCR. Total RNA isolated hot phenol extraction purified by <br /> after known number generations selection adenine prot-         using RNeasy kit (Qiagen). A total 0.5 &#65533;?&#65533;g total S. pombe RNA was <br /> otrophy relaxed growth YE agar. Rates chromosome loss          reverse transcribed SuperScript reverse transcriptase (RT) (Invitrogen) and <br /> generation calculated described (16, 28) formula 1 &#11002;      random hexamers according manufacturer&#8217;s instructions. Reverse-tran- <br /> e(1/n)lnRn, Rn proportion Ade&#11001; cells n generations         scribed S. pombe cDNA products (2 &#65533;?&#65533;l) PCR amplified described <br /> removal selection. For strain tested, mean rates calculated   (9) specific primers (Table 2) 25 cycles 95&#176;C 1 min, <br /> independent measurements.                                                            64&#176;C 1 min, 72&#176;C 3 min. <br />  2290     WANG ET AL.                                                                                                 MOL. CELL. BIOL. <br />  <br />                                            TABLE 2. Oligonucleotides used study <br />   Name                                                              Sequence (5&#11032;-3&#11032;) <br />  <br /> DELA .........................TTTAAGTTGGTTTTATTTAAATTAAAGATTGATTTTATTGCTGCTAAATAAGGAATTTCACATCTTTGGGGAA <br />                                  TTGGTTACAAAGCGACTATAAGTCAGAAAGTG <br /> DELB..........................GGATGATATTGCATACTCCAAATACCCAAATCATGAGCAAATGGAATTTCAGCAGTAGGGTATAGTAAATA <br />                                  AGAAGACGCAATTCTAAATGCCTTCTGAC <br /> TAGA .........................ACGCTGAGCCGTCAACTCAAAACTCTTTTATCCCAGGTCTCACCTCAAAGTCACAAGATGGGTATCCTCAAA <br />                                  ACTATCGACGGATCCCCGGGTTAATTAA <br /> TAGB .........................GCTGCGGTGTGAATTTTTTTTTTTTTAAATCTAATTAATGCCAAGTAAATATGTTTTTCAAATAACAAAGAAA <br />                                  TGTATTAGAATTCGAGCTCGTTTAAAC <br /> PSOA ..........................CTAGGTGGATTAATTTATTTGGCGTGCCTAATTTGGAACGAAATTAATATCATCTTCAACATCCCACCGATTT <br />                                  ACTTATAATACTAATTCTAAATGCCTTCTGAC <br /> PSOB ..........................TTTGATAGCCCTGATCCATAGTCCACAGTCCTACGAGTCATAGGCTTTTGAATAATTCTAGCATTTTCAGCTC <br />                                  TTTCCATCAAAGCGACTATAAGTCAGAAAGTG <br /> MUTA ........................ATCAAGCATGGATAGATGATCAAAGATATC <br /> MUTB.........................TCCTCTCGCCGAATCGTAATTTCTTAA <br /> URA4F .......................GTCCCCTGGTATCGGCTTGGATGTTAAAGG <br /> URA4R1 ....................TGCCTTCTGACATAAAACGCCTAG <br /> URA4R2 ....................TGTAGGAGCATGTTTAATAAATTACTATAGCAAATTACTTTTTATT <br /> URA4R3 ....................CATACATCTTTCATTGGCTTTGTAC <br /> URA4R4 ....................GTCTACATGGTATTTTACATTCATC <br /> URA4R5 ....................CCAACACCAATGTTTATAACCAAG <br /> URA4R6 ....................GCTTGTGATATTGACGAAACTTTTTGACATC <br /> CDT1F........................CTCATCTCGCTCAGATTTTAACCGTATGGC <br /> CDT1R .......................CTCGCGACTTGTTGATTTCTCCAAATTGAG <br /> CDC18F......................CATATCGTTCAGCAGGAGTCGTGGGTCGTG <br /> CDC18R .....................CGGACTGGGTGAAATGCGACTCTAGTTCAC <br /> CDC18F1....................GGTCTTCAACAGAAAGCCATCCTTTGTACG <br /> CDC18R1 ...................GGTACCAATGTCTCCAACAGCTGTAATGAC <br /> CDC18R2 ...................CGGCAAAATTTGAAAACCGTAACGCGTGAA <br /> CDC18R3 ...................CAACCTGAAAAGTTTAAACACCAAGTTGGA <br /> CDC18R4 ...................ACCACTTGAGACTGAACCACGGGTATGCGT <br /> CDC18R5 ...................CCCCAAAATTGAGAAGTGCTTTTGCTACGT <br /> CDC18R6 ...................TTCCGCAGCTTATATGATTACTATGTCGTC <br />  <br />  <br />  <br />                          RESULTS                                   category frequently contained aberrantly segregated chromo- <br />                                                                    somes (data shown). <br />    Fission yeast Pfs2 essential nuclear protein required <br />                                                                       Complementation cin2-3169 mutant genomic <br /> for normal chromosome segregation. In order identify novel <br />                                                                    DNA library (kindly provided Taro Nakamura Chikashi <br /> genes essential roles chromosome segregation, used <br />                                                                    Shimoda) yielded 10 transformants grow the <br /> the fluorescence microscopy-based screen search collec- <br />                                                                    restrictive temperature. Of these, contained plasmid in- <br /> tion ts S. pombe mutants strains displaying chromosome <br />                                                                    sert spanning S. pombe orthologue S. cerevisiae PFS2 <br /> missegregation unusually high frequency Mad2 or <br /> Bub1 nuclear dots restrictive temperature. This screen      gene, term pfs2&#11001;, contained insert <br /> yielded cin2-3169 strain, number         spanning cft1&#11001;, S. pombe orthologue S. cerevisiae <br /> mutant strains (a complete account screen         CFT1 gene. Both PFS2 CFT1 (also known YHH1) en- <br /> characterization mutants form basis fu-     code components S. cerevisiae CPF clear coun- <br /> ture report). After asynchronous cin2-3169 cells expressing        terparts metazoan CstF CPSF (Cstf50 CPSF160, <br /> Mad2-GFP shifted 26&#176;C restrictive temper-         respectively) (23). Backcrossing cin2-3169 strain a <br /> ature 36&#176;C, cells containing Mad2-GFP dots accumulated          strain pfs2&#11001; gene tagged GFP (by use of <br /> and accounted 8% population 4 h            kanamycin resistance gene marker) demonstrated that <br /> temperature shift (Fig. 1A). The presence one, two,          pfs2 gene mutated cin2-3169 cells, cft1&#11001; repre- <br /> occasionally Mad2-GFP foci cells correlated     sents extragenic suppressor. We assigned des- <br /> in cases gross defects chromosome segregation,     ignation pfs2-3169 pfs2 allele identified cin2-3169. <br /> judged DAPI staining (Fig. 1A). Visualization HA                The identification ts pfs2 allele suggested pfs2&#11001; is <br /> epitope-tagged Bub1 cin2-3169 cells expressing Mad2-       likely essential cell viability. To address point, we <br /> GFP revealed variety combinatorial phenotypes       constructed ura4-D18/ura4-D18 diploid fission yeast strain in <br /> shift 36&#176;C. Most cells Bub1-HA foci contained         pfs2 alleles disrupted ura4&#11001; <br /> Mad2-GFP spots, case,      marker. Following meiosis tetrad dissection, found <br /> most cases numbers Bub1-HA Mad2-GFP foci                set haploid spores viable, <br /> within individual cells equal (Fig. 1B C). After      case Ura&#11002; (Fig. 2A). Thus, pfs2&#11001;, like <br /> incubation 4 h 36&#176;C, 85% cin2-3169 cells         PFS2 S. cerevisiae, essential gene, pfs2-3169 a <br /> microtubular arrays typical interphase, 10% post-     loss-of-function allele. <br /> anaphase arrays 4% contained mitotic spindles. The         To monitor localization Pfs2 living fission yeast, the <br />  VOL. 25, 2005                                                                                          Pfs2 AND THE CELL CYCLE               2291 <br />  <br />  <br />                                                                             tion living pfs2&#11001;-GFP cells fluorescence microscopy <br />                                                                             showed Pfs2-GFP (and, inference, Pfs2) localized <br />                                                                             nucleus excluded nucleolus (Fig. 2B). <br />                                                                             There obvious differences pattern cells <br />                                                                             different cell cycle stages. <br />                                                                                To examine influence pfs2 function progression <br />                                                                             mitosis detail, pfs2-3169 control pfs2&#11001; <br />                                                                             cells expressing GFP localized centromere chromo- <br />                                                                             2 (cen2-GFP) synchronized arrest hydroxyu- <br />                                                                             rea (HU) 26&#176;C released HU-free medium at <br />                                                                             36&#176;C. Progression subsequent mitosis mon- <br />                                                                             itored fluorescence microscopy examine cen2-GFP sep- <br />                                                                             aration nuclear morphology (Fig. 3). Under condi- <br />                                                                             tions, control pfs2&#11001; cells exhibited uniformly symmetrical <br />                                                                             segregation cen2-GFP signal daughter nuclei, a <br />                                                                             peak binucleate cells seen 120 min release the <br />                                                                             HU block; 180 min, cells completed <br />                                                                             mitosis cytokinesis (Fig. 3A B). In contrast, approxi- <br />                                                                             mately 40% pfs2-3169 cells initiated cen2-GFP separa- <br />                                                                             tion, cells displayed severe chromosome <br />                                                                             missegregation 240 min release HU block (Fig. <br />                                                                             3A C). In remaining 60% cells, cen2-GFP <br />                                                                             remained unseparated. The population cells failed to <br />                                                                             complete mitosis 240 min release HU block <br />                                                                             (Fig. 3C). These data suggested HU block re- <br />                                                                             lease, pfs2-3169 cells showed defects. One mitotic <br />                                                                             sister chromatid missegregation, accompanied in- <br />                                                                             creased frequencies Mad2 Bub1 foci, was <br />                                                                             block interphase, resulting failure enter mitosis. <br />                                                                             Among cells exhibited cen2-GFP separation, timing of <br />                                                                             event significantly altered pfs2 mutant, <br />                                                                             suggesting gross defect sister chromatid co- <br />                                                                             hesion. <br />                                                                                As initially independent investigation pfs2&#11001; <br />  <br />  <br />  <br />  <br />    FIG. 1. Chromosome missegregation accumulation Mad2 <br /> and Bub1 foci cin2-3169 (pfs2-3169) cells. (A) An asynchronous <br /> culture cin2-3169 mad2&#11001;-GFP (KZ52) cells growing 26&#176;C was <br /> shifted restrictive temperature 36&#176;C 4 h, stained DAPI <br /> to reveal DNA, examined fluorescence microscopy. (B) Termi- <br /> nal phenotypes cin2-3169 mad2&#11001;-GFP bub1&#11001;-HA strain (KZ52) <br /> after 4 h incubation 36&#176;C. Fixed cells processed anti-HA <br /> immunofluorescence DAPI staining. Individual cells containing <br /> one fluorescent spots assigned categories: <br /> cells containing multiple spots, larger number Mad2-GFP <br /> spots Bub1-HA spots (I); equal numbers multiple <br /> (II) single (V) spots type; multiple spots, a <br /> larger number Bub1-HA spots Mad2-GFP spots (III); and <br /> those single Mad2-GFP spot (IV) single Bub1-HA spot <br /> (VI). (C) Quantification phenotypes indicated panel B; total <br /> of 55 cells fluorescent spots scored. <br />                                                                               FIG. 2. Fission yeast Pfs2 essential nuclear protein. (A) Tet- <br />                                                                             rads derived h&#11001;/h&#11002; pfs2::ura4&#11001;/pfs2&#11001; diploid strains micro- <br /> one-step gene replacement method used generate                     dissected YE5S agar. Colonies resulting tetrads <br />                                                                             photographed 7 days growth 30&#176;C. The genotypes of <br /> strain (pfs2&#11001;-GFP) C-terminally GFP-tagged version                segregants determined replica plating. (B) Merged images <br /> Pfs2. The GFP-tagged protein appeared functional,                  fluorescence micrographs showing Pfs2-GFP DNA (Hoechst <br /> judged viability pfs2&#11001;-GFP strain. Examina-              33342) localization living cells. Bar, 10 &#65533;?&#65533;m. <br />  2292      WANG ET AL.                                                                                                           MOL. CELL. BIOL. <br />  <br />  <br />                                                                           spindle checkpoint-defective strains genetic interactions <br />                                                                           pfs2-11 cells. Strains deletion bub1 (bub1&#9004;) were <br />                                                                           inherently ts, combination bub1&#9004; pfs2-11 <br />                                                                           resulted reduction restrictive temperature relative to <br />                                                                           seen pfs2-11 (Fig. 4A). This genetic interaction <br />                                                                           reflected rapid loss viability double <br />                                                                           mutant compared of single mutants after <br />  <br />  <br />  <br />  <br />    FIG. 3. Chromosome segregation defects pfs2-3169 cells. (A) <br /> Segregation patterns sister centromeres. Wild-type (KZ72) pfs2- <br /> 3196 (KZ71) cells carrying cen2-GFP grown 26&#176;C treated <br /> with HU 3 h. After HU washed away, cultures shifted to <br /> 36&#176;C incubation continued. cen2-GFP localization DAPI <br /> staining visualized fluorescence microscopy. Representative <br /> examples unseparated (a), equally segregated (b), missegre- <br /> gated (c d) centromeres shown. Bar, 5 &#65533;?&#65533;m. (B C) Quan- <br /> tification sister centromere segregation patterns following cen2-GFP <br /> and DAPI staining wild-type (B) pfs2-3169 (C) cells. The per- <br /> centage type segregation pattern (open squares, unsepa- <br /> rated; blue circles, equally segregated; red triangles, missegregated) <br /> was plotted time point. <br />  <br />  <br />  <br /> function, used targeted homologous recombination re- <br /> place endogenous pfs2&#11001; promoter at- <br /> tenuated versions thiamine-repressible nmt1 promoter, <br /> P41 P81 (3, 4). The resulting strains, P41-pfs2 P81- <br /> pfs2, grew healthily absence thiamine unable <br /> to grow attenuated nmt1 promoters repressed <br /> by addition thiamine medium (data shown). <br /> The availability promoter shutoff strains possi- <br /> ble generation ts alleles pfs2, following tar- <br /> geted recombination randomly mutated chromosomal pfs2 <br /> sequences simultaneous restoration pfs2&#11001; promoter <br /> (see Materials Methods). This approach gave rise a <br /> second tightly ts allele, pfs2-11. These cells ceased divide <br /> within 1 h shifted restrictive temperature of <br /> 36&#176;C and, like pfs2-3169 cells, displayed chromosome segrega- <br /> tion defects later time points (Fig. 4 data shown). <br /> These data indicate phenotypes associated both <br />                                                                              FIG. 4. The spindle checkpoint required cell survival the <br /> alleles resulted similar losses pfs2 function. The con-           inactivation pfs2. (A) Wild-type strain 972 strains <br /> nection pre-mRNA 3&#11032;-end processing chromo-                    indicated streaked parallel YE5S agar plates photo- <br /> some segregation suggested data pfs2-3169 cells             graphed incubation 3 5 days indicated temperatures. <br /> therefore based unrepresentative, allele-specific effects.         (B) At 2-h intervals, samples cultures pfs2-11, bub1&#9004;, pfs2- <br />    The spindle checkpoint required cell survival             11 bub1&#9004; cells incubated restrictive temperature 36&#176;C were <br />                                                                           taken score percentage aberrant mitosis culture. One <br /> inactivation pfs2. Despite chromosome segregation de-              thousand cells indicated times plated YES agar plates <br /> fects described above, pfs2 mutants retained viability             assess cell survival. Colonies counted 5 days growth at <br /> protracted incubation restrictive temperature (Fig. 4B).           26&#176;C. (C) Fluorescence micrographs DAPI-stained pfs2-11, bub1&#9004;, <br /> The increased frequencies Mad2 Bub1 foci cells            pfs2-11 bub1&#9004; cells grown permissive temperature 26&#176;C <br />                                                                           shift 36&#176;C 6 h. Bar, 10 &#65533;?&#65533;m. (D) Merged images of <br /> suggested involvement aspects spin-            fluorescence micrographs showing wild-type pfs2-11 cells carrying <br /> dle checkpoint machinery events following loss              Bub1-GFP. Cells grown 36&#176;C 2 h, fixed methanol, and <br /> pfs2 function. To investigate notion further, tested              stained DAPI. <br />  VOL. 25, 2005                                                                                        Pfs2 AND THE CELL CYCLE                2293 <br />  <br />  <br />                                                                           results earlier study cleavage polyadenyl- <br />                                                                           ation gene (9), transcripts extending 3&#11032; end <br />                                                                           mature ura4&#11001; mRNA undetectable control <br />                                                                           pfs2&#11001; cells grown 3 h 36&#176;C (Fig. 6B, upper panel). In <br />                                                                           contrast, steady-state ura4&#11001; transcripts extended 300 <br />                                                                           bases normal 3&#11032; end pfs2-11 cells grown under <br />                                                                           conditions (Fig. 6B, lower panel). These data sug- <br />                                                                           gested S. pombe, S. cerevisiae, pfs2&#11001; function is <br />                                                                           required normal 3&#11032;-end processing pre-mRNA. <br />                                                                              Pfs2 required entry S phase. Although defective <br />                                                                           chromosome segregation striking aspect the <br />                                                                           pfs2 mutant phenotype cellular level, 85% cells <br />                                                                           arrested interphase pfs2 inactivated asyn- <br />                                                                           chronous populations (data shown). Similarly, 60% pfs2- <br />                                                                           3169 cells released HU (early-S-phase) block 36&#176;C <br />                                                                           unable progress mitosis (Fig. 3). This interphase <br />                                                                           function pfs2&#11001; investigated using nitrogen starvation <br />                                                                           arrest pfs2-11 pfs2&#11001; cells G1 26&#176;C releas- <br />                                                                           ing starved cells nitrogen refeeding 36&#176;C (Fig. 7). <br />                                                                           Flow cytometry indicated pfs2&#11001; population en- <br />                                                                           tered S phase 2 h nitrogen refeeding com- <br />                                                                           pleted DNA replication 4 h (Fig. 7A). In contrast, few <br />                                                                           pfs2-11 cells left G1 4-h time point, and <br />                                                                           substantial numbers G1-arrested cells present after <br />                                                                           6 h. Thus, pfs2&#11001; function required transit normal <br />                                                                           rate G1 phase S phase. The likely requirement of <br />                                                                           Pfs2 general pre-mRNA processing suggest ni- <br />                                                                           trogen-starved cells lacking pfs2&#11001; function simply fail to <br />    FIG. 5. Increased chromosome segregation failure rate pfs2-11 <br /> cells. (Upper panel) Rates minichromosome loss calculated for <br /> pfs2&#11001; pfs2-11 cells grown semipermissive temperature of <br /> 32&#176;C. Mean loss rates 9.6 &#11003; 10&#11002;4 7.5 &#11003; 10&#11002;5 generation <br /> for pfs2-11 cells control pfs2&#11001; cells, respectively. (Lower <br /> panels) Examples pfs2&#11001; colonies (left) Ade&#11002; pfs2-11 colonies <br /> indicative chromosome loss (right). <br />  <br />  <br /> a shift cells 36&#176;C (Fig. 4B). Loss viability was <br /> correlated appearance &#8220;cut&#65533;? cells, septa- <br /> tion occurred absence nuclear division (Fig. 4B <br /> and C). Consistent results obtained pfs2-3169 <br /> cells (Fig. 1), cells multiple Bub1-GFP foci observed <br /> by 2 h pfs2-11 cells shifted 36&#176;C (Fig. 4D). <br />    The failure chromosome segregation seen pfs2 mu- <br /> tants shifted restrictive temperature suggested that <br /> even partial loss pfs2 function severely compromise <br /> the fidelity chromosome transmission. pfs2&#11001; (control) and <br /> pfs2-11 cells containing nonessential minichromosome (Ch16) <br /> carrying adenine biosynthetic marker used <br /> to measure rates chromosome loss growth semi- <br />                                                                              FIG. 6. Pfs2 required transcriptional termination. (A) Sche- <br /> permissive temperature 32&#176;C (Fig. 5). This assay showed           matic representation 547-bp region encompassed the <br /> the spontaneous loss minichromosome 13-fold               EcoRV-HindIII fragment downstream ura4&#11001; gene. The po- <br /> frequent pfs2-11 cells control pfs2&#11001; cells. We conclude        sitions translation stop <span id='am-64' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-65' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-66'  ></span><span id='am-67' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span> (TAA) major site of <br /> that proper functioning pfs2&#11001; makes important contri-           pre-mRNA 3&#11032;-end cleavage polyadenylation (vertical arrowhead) <br />                                                                           indicated, oligonucleotides used RT-PCR anal- <br /> bution maintenance genome stability S. pombe. <br />                                                                           ysis (horizontal arrows). (B) Electrophoretic separation RT-PCR <br />    Pfs2 required prevent transcriptional read-through.              products total RNA isolated cultures wild-type (upper <br /> Since Pfs2 previously implicated pre-mRNA cleavage                 panel) pfs2-11 (lower panel) cells grown restrictive temper- <br /> and polyadenylation S. cerevisiae, likely           ature 36&#176;C 3 h. RT-PCR products separated 1.5% <br /> essential function pfs2&#11001; S. pombe involve                agarose gel. Lane numbers correspond 3&#11032; oligonucleotides <br />                                                                           (URA4R1 URA4R6) used RT PCR amplification; re- <br /> pre-mRNA processing. To investigate possibility, steady-             actions included common 5&#11032; primer (URA4F), indicated panel <br /> state RNA well-characterized downstream region                A. In negative control reactions (lane C), reverse transcriptase <br /> the ura4&#11001; gene examined RT-PCR (Fig. 6). In keeping                added RT-PCR. Lane M, size markers. <br />  2294      WANG ET AL.                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />                                                                             achieve level macromolecular biosynthesis needed for <br />                                                                             commitment mitotic cell cycle start. This inter- <br />                                                                             pretation was, however, supported measurements of <br />                                                                             RNA transcripts cdt1&#11001; cdc18&#11001; (Fig. 7B), genes <br />                                                                             transcribed start-dependent manner (8). RT-PCR <br />                                                                             analysis indicated levels transcripts of <br />                                                                             genes peaked 2 h nitrogen refeeding control <br />                                                                             pfs2&#11001; cells; pfs2-11 cells, transcript levels increased with <br />                                                                             similar kinetics 2 h continued rise, transcripts <br />                                                                             maintained high levels 6 h release. <br />                                                                             These data suggested that, conditions, pfs2-11 cells <br />                                                                             substantially delayed cell cycle progression point <br />                                                                             start close initiation S phase. Al- <br />                                                                             start-dependent transcription activated these <br />                                                                             cells, findings ura4&#11001; gene (Fig. 6) suggested that <br />                                                                             3&#11032;-end processing transcripts defective. <br />                                                                             We able address notion transcripts from <br />                                                                             cdc18&#11001;, 3&#11032; end cDNA mapped this <br />                                                                             gene (11). RT-PCR indicated steady-state cdc18&#11001; tran- <br />                                                                             scripts extended 200 bases normal mRNA <br />                                                                             3&#11032; end pfs2-11 cells released nitrogen starvation 2 h <br />                                                                             36&#176;C (Fig. 7C D). <br />  <br />                                                                                                        DISCUSSION <br />                                                                                                    &#11001; <br />                                                                                The S. pombe pfs2 gene described encodes 509- <br />                                                                             amino-acid protein containing seven WD (&#65533;?&#65533;-transducin) re- <br />                                                                             peats. Previous studies identified CstF50, human counter- <br />                                                                             Pfs2, involvement pre-mRNA 3&#11032;-end <br />                                                                             processing S. cerevisiae Pfs2p component CPF <br />                                                                             complex (22, 29). Mutation pfs2 S. pombe caused the <br />                                                                             accumulation transcripts extending 3&#11032; ends of <br />                                                                             mature mRNAs ura4&#11001; cdc18&#11001; genes (Fig. 6 7). <br />                                                                             While formally possible finding represents the <br />                                                                             stabilization read-through RNAs normally present <br />                                                                             low levels, straightforward explanation be <br />                                                                             pfs2 function required pre-mRNA cleavage and <br />                                                                             polyadenylation. The identification cft1&#11001; multicopy <br />                                                                             suppressor adds weight <span id='am-39' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-40' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-42'  ></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>idea</span>. <br />                                                                                This study revealed unexpected connections between <br />                                                                             pfs2 cell cycle progression. The point cycle at <br />                                                                             pfs2 function appears critical onset of <br />                                                                             S phase activation start-dependent transcrip- <br />                                                                             tion (Fig. 7). The start-dependent transcripts accumulated <br />    FIG. 7. Pfs2 required entry S phase. (A) pfs2&#11001; (strain       pfs2-11 cells released nitrogen starvation restric- <br /> 972) pfs2-11 cells arrested G1 nitrogen starvation       tive temperature did appear appropriately processed <br /> released nitrogen-rich medium nonpermissive temperature         (Fig. 7C D). The impaired function transcripts <br /> of 36&#176;C. Cells harvested hourly intervals processed DNA         contributed observed cell cycle arrest, <br /> staining flow cytometry. (B) (Upper panels) Total RNA samples <br /> from cultures shown panel A isolated indicated           Cdt1 Cdc18 required assembly of <br /> times. RT-PCR analysis performed primers specific cod-         functional prereplicative complexes chromosomal origins of <br /> ing sequences cdt1&#11001; (CDT1F CDT1R; left panels) cdc18&#11001;            replication (10, 11, 18). The observation cells released <br /> (CDC18F CDC18R; right panels), products separated              G1 arrest nitrogen refeeding able progress <br /> on 1.5% agarose gel. (Lower panels) Normalized levels RT-PCR <br /> products. (C D) Transcriptional read-through cdc18&#11001; pfs2-11 <br />                                                                             start (as judged activation cdt1&#11001;/cdc18&#11001; <br /> cells. (C) Schematic representation 743-bp region containing the <br /> StyI-HinDIII fragment encompassing 3&#11032; end cdc18&#11001; gene. <br /> The positions translation stop codon (TAG) site of <br /> pre-mRNA 3&#11032;-end cleavage polyadenylation (vertical arrowhead)           Lane numbers correspond 3&#11032; oligonucleotides (CDC18R1 to <br /> (11) indicated, oligonucleotides used RT-PCR         CDC18R6) used RT PCR amplification; reactions included <br /> analysis (horizontal arrows). (D) RT-PCR products obtained total       common 5&#11032; primer (CDC18F1), indicated panel C. Negative <br /> RNA isolated cultures wild-type (upper panel) pfs2-11           control reactions (lane C) panels B D contained reverse <br /> (lower panel) cells harvested 2 h nitrogen refeeding panel A.   transcriptase. <br />  VOL. 25, 2005                                                                                      Pfs2 AND THE CELL CYCLE                    2295 <br />  <br />  <br /> transcription) absence pfs2 function surprising,        link specifically pre-mRNA cleavage poly- <br /> since efficient pre-mRNA 3&#11032;-end processing expected         adenylation chromosome segregation recently estab- <br /> to prerequisite gene expression needed cell         lished affinity purification cleavage polyadenylation <br /> cycle commitment. It possible nitrogen-starved cells         complexes S. cerevisiae S. pombe (6, 7, 17, 25). In both <br /> retain stable stockpile mature mRNAs facilitate G1         yeasts, serine-threonine protein phosphatase Glc7/Dis2 <br /> progression nutritional status improves. Alterna-           identified component complexes puri- <br /> tively, loss pfs2 function affect processing         fication variety tagged subunits, including Cft1 and <br /> different transcripts different degrees, efficient ex-    Pfs2. In S. pombe, dis2&#11001; gene identified a <br /> pression genes required G1 transit comparatively       screen mutants defective mitotic chromosome separation <br /> independent pfs2. Further experiments required         (21). The screen identified dis3&#11001;, encodes pro- <br /> distinguish possibilities. Defects cell cycle     tein homologous S. cerevisiae protein Dis3p/Rrp44p, a <br /> machinery, opposed defects cell growth, generally lead      component 3&#11032;-5&#11032; exoribonuclease termed exosome <br /> to cell elongation S. pombe. Cells lacking pfs2 function          (12, 20, 21). It interesting determine the <br /> (either result shift ts strain restrictive   chromosome segregation defects dis2 dis3 mutants, like <br /> temperature promoter shutoff) did appre-         pfs2 mutants, associated aberrant RNA pro- <br /> ciably elongated, however. This observation suggested           cessing. <br /> general macromolecular synthesis partially com-            A partial loss pfs2 function fission yeast gave rise a <br /> promised circumstances, arguing highly         dramatic elevation rate chromosome loss (Fig. 5). It <br /> specific involvement pfs2 regulated expression cell     interesting determine scenario also <br /> cycle-related genes.                                                 occurs mammals; case, defects cleavage and <br />    Chromosome segregation second cell cycle stage dur-         polyadenylation machinery contribute chromo- <br /> ing pfs2 mutant phenotype manifested. Two              somal instability characterizes cancer cells. <br /> general explanations account defective mitosis <br />                                                                                               ACKNOWLEDGMENTS <br /> seen loss pfs2 function. One realistic possibility is <br /> that loss transcriptional termination direct effect      We grateful Nick Proudfoot members group for <br /> some aspect chromosome structure kinetochore function.         valuable discussions, Paul Russell bringing attention the <br />                                                                      reannotation pfs2 cid14 genes, Stephen Kearsey for <br /> The recent observation cohesin clusters sites conver-     comments manuscript, Jean-Paul Javerzat providing the <br /> gent transcription (13) suggests intimate relationship be-        bub1 deletion strain, Taro Nakamura Chikashi Shimoda for <br /> tween sister chromatid cohesion distribution active       providing genomic DNA library. <br /> Pol II complexes. In addition, centromere function S. pombe         This work supported Cancer Research UK, Human <br /> may perturbed deregulated transcriptional read-           Frontiers Science Program (research grant T.T.), Association <br />                                                                      International Cancer Research, Wellcome Trust (research <br /> through, S. cerevisiae (27). Alternatively, impact   career development award S.-W.W.). <br /> of loss pfs2 function mitosis indirect, through <br />                                                                                                     REFERENCES <br /> impaired expression genes required faithful <br />                                                                       1. Allshire, R. C., E. R. Nimmo, K. Ekwall, J. P. Javerzat, G. Cranston. <br /> chromosome segregation. This situation analogous             1995. Mutations derepressing silent centromeric domains fission yeast <br /> a phenotype previously described S. cerevisiae, muta-           disrupt chromosome segregation. Genes Dev. 9:218&#8211;233. <br /> tion pre-mRNA splicing factors seen cause spindle           2. Amon, A. 1999. The spindle checkpoint. Curr. Opin. Genet. Dev. 9:69&#8211;75. <br />                                                                       3. Bahler, J., J. Q. Wu, M. S. Longtine, N. G. Shah, A. McKenzie III, A. B. <br /> checkpoint activation secondary consequence defective            Steever, A. Wach, P. Philippsen, J. R. Pringle. 1998. Heterologous <br /> tubulin gene expression (5). These general interpretations           modules efficient versatile PCR-based gene targeting Schizosac- <br />                                                                          charomyces pombe. Yeast 14:943&#8211;951. <br /> are mutually exclusive.                                           4. Basi, G., E. Schmid, K. Maundrell. 1993. TATA box mutations the <br />    The chromosome segregation defects pfs2 mutant cells               <span id='am-30' about='obo:NCBITaxon_4896' typeof='owl:Thing obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_1 obo:NCBITaxon_147554 obo:NCBITaxon_34346 obo:NCBITaxon_4890 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4895 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Schizosaccharomyces malidevorans" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="Schizosaccharomyces pombe" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Schizosaccharomyces pombeP" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45042" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="fission yeast" datatype="xsd:string"></span><span id='am-38'  ></span>Schizosaccharomyces pombe</span> nmt1 promoter affect transcription efficiency <br /> resulted accumulation multiple foci Mad2                transcription start point thiamine repressibility. Gene 123: <br />                                                                          131&#8211;136. <br /> Bub1, presumably marking unattached kinetochores (Fig. 1              5. Dahan, O., M. Kupiec. 2002. Mutations genes Saccharomyces <br /> and 4). Many cells multiple Mad2 Bub1 foci            cerevisiae encoding pre-mRNA splicing factors cause cell cycle arrest <br /> proceeded anaphase-like state, suggesting               activation spindle checkpoint. Nucleic Acids Res. 30:4361&#8211; <br />                                                                          4370. <br /> spindle checkpoint properly enforced pfs2 inac-         6. Dichtl, B., D. Blank, M. Sadowski, W. Hubner, S. Weiser, W. Keller. <br /> tivation. Nonetheless, severity mitotic abnormalities         2002. Yhh1p/Cft1p directly links poly(A) site recognition RNA poly- <br />                                                                          merase II transcription termination. EMBO J. 21:4125&#8211;4135. <br /> in loss viability pfs2 mutant cells           7. Gavin, A. C., M. Bosche, R. Krause, P. Grandi, M. Marzioch, A. Bauer, J. <br /> accentuated deletion bub1 (Fig. 4). These observa-             Schultz, J. M. Rick, A. M. Michon, C. M. Cruciat, M. Remor, C. Hofert, M. <br /> tions suggested spindle checkpoint partially           Schelder, M. Brajenovic, H. Ruffner, A. Merino, K. Klein, M. Hudak, D. <br />                                                                          Dickson, T. Rudi, V. Gnau, A. Bauch, S. Bastuck, B. Huhse, C. Leutwein, <br /> activated pfs2 mutant cells mitotic abnormalities,             M. A. Heurtier, R. R. Copley, A. Edelmann, E. Querfurth, V. Rybin, G. <br /> conclusion supported observation Mad2 Bub1               Drewes, M. Raida, T. Bouwmeester, P. Bork, B. Seraphin, B. Kuster, G. <br /> were colocalized presumptive unattached kinet-             Neubauer, G. Superti-Furga. 2002. Functional organization yeast <br />                                                                          proteome systematic analysis protein complexes. Nature 415:141&#8211;147. <br /> ochores (Fig. 1).                                                     8. Hofmann, J. F., D. Beach. 1994. cdt1 essential target the <br />    Additional connections RNA processing chro-               Cdc10/Sct1 transcription factor: requirement DNA replication inhi- <br />                                                                          bition mitosis. EMBO J. 13:425&#8211;434. <br /> mosome segregation suggested characterization          9. Humphrey, T., C. E. Birse, N. J. Proudfoot. 1994. RNA 3&#11032; end signals of <br /> S. pombe mutants defective 5&#11032;-3&#11032; exoribonuclease en-              S. pombe ura4 gene comprise site determining efficiency element. <br /> coded dhp1 (26). These mutants accumulated polyadenyl-                EMBO J. 13:2441&#8211;2451. <br />                                                                      10. Kearsey, S. E., S. Montgomery, K. Labib, K. Lindner. 2000. Chromatin <br /> ated RNA nuclei exhibited chromosome segrega-               binding fission yeast replication factor mcm4 occurs anaphase <br /> tion defects similar reported pfs2 mutants. A          requires ORC cdc18. EMBO J. 19:1681&#8211;1690. <br />  2296       WANG ET AL.                                                                                                                               MOL. CELL. BIOL. <br />  <br /> 11. Kelly, T. J., G. S. Martin, S. L. Forsburg, R. J. Stephen, A. Russo, P.            implicated mitotic control, binds directly Ran enhances GEF <br />     Nurse. 1993. The fission yeast cdc18&#11001; gene product couples S phase                  activity RCC1. EMBO J. 15:5595&#8211;5605. <br />     START mitosis. Cell 74:371&#8211;382.                                              21.   Ohkura, H., Y. Adachi, N. Kinoshita, O. Niwa, T. Toda, M. Yanagida. <br /> 12. Kinoshita, N., M. Goebl, M. Yanagida. 1991. The fission yeast dis3&#11001;                1988. Cold-sensitive caffeine-supersensitive mutants Schizosac- <br />     gene encodes 110-kDa essential protein implicated mitotic control. Mol.           charomyces pombe dis genes implicated sister chromatid separation dur- <br />     Cell. Biol. 11:5839&#8211;5847.                                                              ing mitosis. EMBO J. 7:1465&#8211;1473. <br /> 13. Lengronne, A., Y. Katou, S. Mori, S. Yokobayashi, G. P. Kelly, T. Itoh, Y.       22.   Ohnacker, M., S. M. Barabino, P. J. Preker, W. Keller. 2000. The <br />     Watanabe, K. Shirahige, F. Uhlmann. 2004. Cohesin relocation                  WD-repeat protein Pfs2p bridges essential factors yeast <br />     sites chromosomal loading places convergent transcription. Nature             pre-mRNA 3&#11032;-end-processing complex. EMBO J. 19:37&#8211;47. <br />     430:573&#8211;578.                                                                     23.   Proudfoot, N., J. O&#8217;Sullivan. 2002. Polyadenylation: tail com- <br /> 14. Millband, D. N., L. Campbell, K. G. Hardwick. 2002. The awesome                    plexes. Curr. Biol. 12:R855&#8211;R857. <br />     power multiple model systems: interpreting complex nature spindle      24.   Radcliffe, P., D. Hirata, D. Childs, L. Vardy, T. Toda. 1998. Identifica- <br />     checkpoint signaling. Trends Cell Biol. 12:205&#8211;209.                                    tion novel temperature-se <br /> </body></html>