<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, May 2003, p. 2961&#8211;2966                                                                                Vol. 185, No. 9 <br /> 0021-9193/03/$08.00&#11001;0 DOI: 10.1128/JB.185.9.2961&#8211;2966.2003 <br /> Copyright &#169; 2003, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />                                                             NOTES <br />  <br />               Essentiality clpX, Not clpP, clpL, clpC, clpE, in <br />                             <span id='am-8' about='obo:NCBITaxon_1301' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_1300 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186826'><span id='am-9' property="rdfs:label" content="Streptococcus" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="PMID:1720654" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="PMID:7537076" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="PMID:8995803" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="PMID:19620365" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="PMID:14657115" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-17'  ></span><span id='am-18' property="oboInOwl:hasDbXref" content="PMID:10555340" datatype="xsd:string"></span>Streptococcus</span> pneumoniae R6 <br />                           Gregory T. Robertson,1&#8224; Wai-Leung Ng,1,2 Raymond Gilmour,1 <br />                                           Malcolm E. Winkler1,2* <br />                          Lilly Research Laboratories, Eli Lilly Company, Indianapolis, Indiana 46285,1 <br />                             Department Biology, Indiana University, Bloomington, Indiana 474052 <br />                                          Received 16 September 2002/Accepted 19 February 2003 <br />  <br />             We using regulated promoter clpX <span id='am-19' about='obo:NCBITaxon_1313' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1301 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_1300 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186826'><span id='am-20'  ></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Micrococcus pneumoniae" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Diplococcus pneumoniae" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Streptococcus pneumoniae" datatype="xsd:string"></span>Streptococcus pneumoniae</span> R6 essential, clpP, <br />          clpL, clpC, clpE disrupted. The essentiality clpX initially missed duplication and <br />          rearrangement region chromosome containing clpX. Depletion ClpX resulted rapid loss of <br />          viability overt changes cell morphology. Essentiality clpX, clpP, reported <br />          previously. <br />  <br />  <br />    We reported previously clpP-mediated proteolysis plays              rangements low frequency. clpX required temper- <br /> an important role pneumococcal stress responses,                 atures tested, depletion ClpX resulted rapid loss of <br /> the development competence, virulence (22). ClpP pro-                cellular viability. <br /> teases consist ATPase regulatory subunit                        Duplications rearrangements clpX region the <br /> Hsp100 <span id='am-93' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-94' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> chaperones ClpP protease subunit,                  S. pneumoniae R6 chromosome. The putative ClpX polypep- <br /> which contains serine protease active site (18; reviewed               tide S. pneumoniae R6 shows high identity similarity <br /> references 8, 9, 11, 21, 24). Streptococcus pneumoniae                  entire length ClpX subunits defined bac- <br /> contains putative orthologs ATPase specificity factors              teria (e.g., 58 76, 60 76, 65 81, 75 87% <br /> (ClpC, ClpE, ClpL, ClpX) (14, 28) single clpP gene                identity similarity, respectively, ClpX <span id='am-109' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-110' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-115'  ></span><span id='am-116' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-119' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span>Escherichia coli</span> <br /> that monocistronic operon (22). clpP mutants showed                 K-12, <span id='am-135' about='obo:NCBITaxon_75' typeof='owl:Thing obo:NCBITaxon_204458 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_28211 obo:NCBITaxon_76892'><span id='am-136' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-137' property="rdfs:label" content="Caulobacter" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasDbXref" content="PMID:10319468" datatype="xsd:string"></span><span id='am-140'  ></span><span id='am-141' property="oboInOwl:hasDbXref" content="PMID:10425763" datatype="xsd:string"></span>Caulobacter</span> crescentus, <span id='am-32' about='obo:NCBITaxon_1423' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_653685'><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis/Bacillus globigii" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042382" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasAlternativeId" content="NCBITaxon:387338" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Bacillus uniflagellatus" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasAlternativeId" content="NCBITaxon:981081" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224247" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232470" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="PMID:11211269" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Vibrio subtilis" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Bacillus natto" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042381" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429427" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451272" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232830" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232819" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasAlternativeId" content="NCBITaxon:941949" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042380" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232840" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232824" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232844" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasAlternativeId" content="NCBITaxon:523730" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232822" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232828" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451270" datatype="xsd:string"></span><span id='am-59'  ></span><span id='am-60' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429428" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis8" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232841" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="Bacillus subtilis" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232825" datatype="xsd:string"></span>Bacillus subtilis</span>, <span id='am-26' about='obo:NCBITaxon_1357' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_1300 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186826'><span id='am-27'  ></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="lactic streptococci" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="Lactococcus" datatype="xsd:string"></span>Lactococcus</span> <br /> temperature sensitivity growth 40&#176;C Trypticase soy                lactis, respectively, GenBank comparisons). The intercis- <br /> agar blood (TSA-BA) plates appreciably im-                     tronic regions separate clpX surrounding genes are <br /> paired growth 37&#176;C (22). <span id='am-96' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-97'  ></span><span id='am-98' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> observation in-               minimal, obvious factor-independent tran- <br /> dependently Chastanet coworkers, showed                  scription terminators region (Fig. 1A). Thus, clpX may <br /> that clpE mutants temperature sensitive, suggesting role              multifunctional operon extends dpr through <br /> for ClpE ClpEP temperature tolerance (5). Roles                   hypothetical gene spr1422 (Fig. 1A). The open reading <br /> ClpL ClpX S. pneumoniae discovered,                frame immediately downstream clpX designated spr1426 <br /> while function ClpC specificity factor present          recently shown essential encode GTP <br /> question (4, 5, 22). <br />                                                                             binding protein (29). The dfr gene upstream clpX (Fig. <br />    We described construction &#9004;clpX::ermAM <br />                                                                             1A) encodes dihydrofolate reductase, required for <br /> mutation (erythromycin resistance; Ermr) S. pneumoniae R6 <br />                                                                             growth number bacterial <span id='am-106' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-107' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> (12). The known <br /> (strain EL873; Table 1) mutant did not <br />                                                                             gene upstream clpX dpr, implicated in <br /> show temperature-sensitive growth like &#9004;clpP mutant (22). <br />                                                                             H2O2 resistance (33) critical growth S. <br /> However, reported that, reasons did not <br />                                                                             pneumoniae, produces H2O2 aerobic growth un- <br /> understand, &#9004;clpX::ermAM amplicon <br />                                                                             der laboratory conditions (26). <br /> into S. pneumoniae D39 (Table 1) transformation (22) and <br /> that transfer &#9004;clpX::ermAM mutation R6                    Previously, confirmed presence &#9004;clpX::ermAM <br /> D39 using chromosomal DNA isolated strain                      allele EL873 PCR primers located immediately <br /> EL873 (22). Further analyses problem shown                outside clpX reading frame (22). Although thought that <br /> clpX actually essential S. pneumoniae R6 clpX            sufficient resolution gels resolve &#9004;clpX:: <br /> region chromosome forms duplications rear-                 ermAM construct clpX&#11001; gene, later analyses fur- <br />                                                                             ther digestion PCR products restriction enzymes <br />                                                                             revealed copies &#9004;clpX::ermAM clpX&#11001; <br />   * Corresponding author. Mailing address: Department Biology,           genes present EL873 (Fig. 2, lane 5). Thus, EL873 a <br /> Indiana University, Jordan Hall 142, Bloomington, IN 47405. Phone: <br />                                                                             merodiploid likely arose recombination the <br /> (317) 508-7812. Fax: (812) 855-6705. E-mail: mwinkler@bio.indiana <br /> .edu.                                                                       &#9004;clpX::ermAM allele copy duplication clpX <br />   &#8224; Present address: <span id='am-142' about='obo:NCBITaxon_509369' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_40096 obo:NCBITaxon_40092 obo:NCBITaxon_33208 obo:NCBITaxon_40093 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-143' property="rdfs:label" content="Cumbre" datatype="xsd:string"></span><span id='am-144'  ></span><span id='am-145' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Cumbre</span>, Inc., Dallas, TX 75235.                        region occurred spontaneously chromosome S. <br />  <br />                                                                      2961 <br />  2962       NOTES                                                                                                                                    J. BACTERIOL. <br />  <br />                                                                  TABLE 1. Bacterial strainsa <br />  Strain                 Genotype phenotype                                                   Description                                      Source reference <br />  <br /> EL59         S. pneumoniae R6, avirulent unen-                  Derived D39 isolate                                                          A. Tomasz (14) <br />                 capsulated parent strain <br /> EL161        S. pneumoniae D39, virulent encap-                 Subclone original clinical isolate                                             J. Yother (25) <br />                 sulated type 2 parent strain <br /> EL539        EL59 &#9004;clpP::aad9 (Spcr)                            EL59 transformed linear &#9004;clpP::aad9 amplicon                                 22 <br /> EL854        EL59 &#9004;clpC::ermAM (Ermr)                           EL59 transformed linear &#9004;clpC::ermAM amplicon                                22 <br /> EL873        EL59 &#9004;clpX::ermAM clpX&#11001; (Ermr)                     EL59 containing previously undetected merodiploid &#9004;clpX::ermAM                 22 <br />                                                                  clpX&#11001; <br /> EL1039       EL161 &#9004;clpX::ermAM clpX&#11001; (Ermr)                    EL161 transformed genomic DNA EL873;                                    22 <br />                                                                  contains previously undetected merodiploid &#9004;clpX::ermAM <br />                                                                  clpX&#11001; <br /> EL1082       EL59 &#9004;clpL::ermAM (Ermr)                           EL59 transformed linear &#9004;clpL::ermAM amplicon                                This study <br /> EL1259       EL59 &#9004;clpE::aad9 (Spcr)                            EL59 transformed linear &#9004;clpE::aad9 amplicon                                 This study <br /> EL1383       EL59 &#9004;bgaA::(Pc::tetL::T1T2::PfcsK-clpX&#11001;)          EL59 transformed linear &#9004;bgaA::(Pc::tetL::T1T2::PfcsK-clpX&#11001;)                 This study <br />                (Tetr)                                            amplicon <br /> EL1387       EL1383 &#9004;clpX&#65533;?&#11022;ermAM (Ermr Tetr)                    EL1383 transformed linear &#9004;clpX&#65533;?&#11022;ermAM amplicon                              This study <br /> EL1392       &#9004;clpX&#65533;?&#11022;ermAM clpX&#11001; (Ermr)                          EL59 transformed &#9004;clpX&#65533;?&#11022;ermAM, resulting partially                      This study <br />                                                                  characterized duplication &#9004;clpX&#65533;?&#11022;ermAM clpX&#11001; <br />   a <br />     Strains constructed transformation indicated recipients linear double-stranded synthetic PCR amplicon DNA. Following single colony isolation, all <br /> constructions confirmed PCR analysis chromosomal DNA using flanking internal primers additional restriction digestion amplicons (details <br /> available http://www.streppneumoniae.com). <br />  <br />  <br /> pneumoniae R6 (e.g., reference 2). This conclusion                           duced synthetic competence stimulatory peptide 1 (see ref- <br /> fully supported observation EL873 reverted rapidly                       erence 13). The frequency transformation amplicon <br /> to sensitivity erythromycin antibiotic selection                         (&#65533;?2.0 &#11003; 10&#11002;6 &#9004;clpX&#65533;?&#11022;ermA transformants) the <br /> removed cultures (Table 2), reversion ac-                          level detection compared control genomic DNA <br /> companied restoration wild-type clpX&#11001; chromo-                           imparting resistance novobiocin (7.1 &#11003; 10&#11002;4 Norr transfor- <br /> some (Fig. 2, lanes 5 6).                                                        mants) R6 parent strain EL59. The frequency transfor- <br />    Because potential polarity clpX insertions                       mation calculated based number recovered an- <br /> expression essential downstream spr1426 gene (Fig. 1A),                       tibiotic-resistant colonies 1 ml transformation mixture <br /> we attempted cross &#9004;clpX&#65533;?&#11022;ermAM amplicon depicted                             divided total CFU milliliter. However, Ermr <br /> in Fig. 1A, clpX reading frame replaced exactly                        colony did appear (EL1392; Table 1), contained another <br /> from start stop <span id='am-101' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-102' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-103'  ></span><span id='am-104' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span> ermAM reading frame,                        kind rearrangement clpX region. Analysis clpX <br /> the chromosome S. pneumoniae R6 transformation in-                             region EL1392 PCR showed presence anom- <br />  <br />  <br />  <br />  <br />    FIG. 1. Organization clpX operon S. pneumoniae R6 construction nonpolar replacement clpX reading frame with <br /> ermAM (Ermr) (A) strain containing ectopic copy clpX&#11001; control fucose-inducible PfcsK promoter located bgaA <br /> locus (B) (see text Table 1 details). Genes predicted directions transcription depicted arrows (drawn scale). The lytC and <br /> trxB genes likely mark boundaries clpX region based large intergenic region direction transcription, respectively. The <br /> shaded regions depict flanking genes surrounding clpX, hatched region corresponds ermAM open reading frame. Pc-tetL imparts <br /> constitutive resistance 0.25 &#65533;?&#65533;g tetracycline ml used selection marker. T1T2 indicates transcription terminators protect <br /> regulated expression PfcsK. <br />  VOL. 185, 2003                                                                                                                         NOTES         2963 <br />  <br />  <br />  <br />  <br />    FIG. 2. ScaI restriction maps clpX regions genomic DNA S. pneumoniae strains EL59, EL873, EL1392 amplified PCR with <br /> the oligonucleotide primer pair clpX-a clpX-c (22). Lanes 1 7, Gibco 1-kb Plus standard ladder; lane 2, wild-type clpX&#11001; locus parent <br /> strain EL59; lane 3, duplicated spr1426 locus EL1392, contains copy clpX&#11001; clpX&#65533;?&#11022;ermAM cassette; lane 4, Erms revertant <br /> of EL1392 containing wild-type clpX&#11001; region; lane 5, clpX::ermAM duplicated clpX&#11001; locus EL873; lane 6, Erms revertant EL873 <br /> containing wild-type clpX&#11001; region. Shaded arrows, clpX reading frame; hatched arrows, ermAM reading frame; open arrows, genes surrounding <br /> clpX (Fig. 1); black dots, &#11011;0.9 kb unknown DNA present EL1392 clpX&#11001; region. ScaI restriction sites indicated letter S a <br /> horizontal line. The break denotes unknown chromosomal DNA EL873 merodiploid. Predicted restriction fragment sizes and <br /> indicated DNA standards kilobase pairs. <br />  <br />  <br /> alous &#11011;5-kb amplicon contained ermAM cassette,                    peared significantly lower frequency &#11349;10&#11002;6. Thus, for- <br /> clpX&#11001;, duplication flanking downstream gene spr1426,              mation &#9004;clpX&#65533;?&#11022;ermAM clpX&#11001; duplications occurred a <br /> and uncharacterized region &#11011;0.9 kb containing DNA                 low detectable frequency comparable essen- <br /> unknown origin (Fig. 2, lane 3). This arrangement probably                 tial genes, &#9004;gyrB::ermAM gyrB&#11001;, chro- <br /> arose aberrant recombination event clpX region.               mosome S. pneumoniae R6. <br /> Unlike EL873, reversion EL1392 erythromycin sensitivity <br /> occurred low frequency antibiotic selection was <br /> removed cultures (Table 2), again, reversion was <br /> accompanied restoration wild-type clpX&#11001; chromo-                    TABLE 2. Reversion frequency S. pneumoniae mutants <br /> some (Fig. 2, lanes 3 4).                                                  containing duplications clpX region chromosome <br />    We determined frequency &#9004;clpX&#65533;?&#11022;ermAM                                                                                No. (%) Ermr and <br /> clpX&#11001; merodiploids arose transformations compared                            Selection                      No.      Erms isolates following <br />                                                                                                                                    spontaneous resolution <br /> for unlinked essential gyrB gene (29). Ten independent                  Strain              CFU/mlb        colonies <br />                                                                                                                                        duplications <br />                                                                                          growtha                     screened <br /> exponential cultures EL59 (R6 parent strain) brain <br /> heart infusion (BHI) medium diluted 1:20 1 ml                                                                             Ermr           Erms <br />  <br /> BHI medium containing 10% heat-inactivated horse serum                     EL873          None        4.7 &#11003; 106         31          4 (13)        27 (87) <br /> (Sigma), 10 mM glucose, 100 ng competence stimu-                                   ERY         4.9 &#11003; 105         31         31 (100)        0 (0) <br /> latory peptide 1 cell density &#11011;7.0 &#11003; 105 CFU/ml               EL1392         None        2.0 &#11003; 107        130        129 (99.2)       1 (0.8) <br />                                                                                           ERY         7.9 &#11003; 106         61         61 (100)        0 (0) <br /> (22). Competent cell suspensions mixed the <br /> nonpolar &#9004;clpX&#65533;?&#11022;ermAM amplicon, &#9004;gyrB::ermAM dele-                         a <br />                                                                                Bacteria grown statically BHI containing lacking 0.3 &#65533;?&#65533;g eryth- <br />                                                                            romycin (ERY) ml &#11015;7 h 37&#176;C 5% CO2, serially diluted 0.9% <br /> tion-insertion amplicon, control genomic DNA bearing                  (wt/vol) NaCl, spread TSA-BA plates containing lacking 0.3 &#65533;?&#65533;g of <br /> point mutation confers resistance novobiocin                      erythromycin ml, respectively. Random colonies screened erythro- <br /> (Novr) (16). Novr transformants recovered                  mycin resistance (Ermr) sensitivity (Erms) patched TSA-BA <br />                                                                            plates containing lacking 0.3 &#65533;?&#65533;g erythromycin ml. <br /> transformations average frequency &#11011;6 &#11003; 10&#11002;4,                        b <br />                                                                                Number CFU milliliter culture cells recovered at <br /> whereas &#9004;clpX&#65533;?&#11022;ermAM &#9004;gyrB::ermAM transformants ap-                     time plating. <br />  2964      NOTES                                                                                                                       J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 3. Impaired growth S. pneumoniae R6 strain EL1387 containing regulated replacement clpX (&#9004;clpX PfcsK::clpX&#11001;) medium <br /> lacking fucose. (A) EL59 (clpX&#11001; parent), EL1383 (regulated merodiploid clpX&#11001; PfcsK::clpX&#11001;), EL1387 (regulated replacement &#9004;clpX PfcsK:: <br /> clpX&#11001;) streaked TSA-BA plates containing 0.25% (wt/vol) fucose (inducing) lacking additional fucose (noninducing). Plates were <br /> photographed 24 h incubation 37&#176;C atmosphere 5% CO2. (B) EL1387 (regulated replacement &#9004;clpX PfcsK::clpX&#11001;) grown <br /> statically overnight BHI broth containing 0.1% (wt/vol) fucose 37&#176;C 5% CO2. Cultures diluted 100-fold fresh BHI containing <br /> or lacking 0.1% (wt/vol) fucose, static incubation continued 37&#176;C 5% CO2. Filled circles squares represent optical densities <br /> and viable cell counts, respectively, culture containing 0.1% (wt/vol) fucose. Open circles squares represent optical densities and <br /> viable cell counts, respectively, culture lacking fucose (0.001% [wt/vol]) carryover fucose starting culture). Results representative <br /> of independent experiments. OD620 (1.4 cm), optical density 620 nm tube 1.4-cm diameter. <br />  <br />  <br />  <br />    clpX essential S. pneumoniae R6. The fact             construction, yield strain EL1383 (Table 1). Tetr transfor- <br /> not obtain simple transformant replaced clpX                     mants selected TSA-BA plates containing 0.25% (wt/ <br /> &#9004;clpX&#65533;?&#11022;ermAM suggested clpX essential. To dem-                      vol) L-fucose 37&#176;C atmosphere 5% CO2. The loca- <br /> onstrate essentiality, needed construct merodiploid                  tion construction confirmed PCR (<span id='am-170' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-171' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-172' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-173'  ></span><span id='am-174' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> not <br /> that clpX&#11001; transcription control regulatable              shown). <br /> promoter. Several catabolic promoters shown else-                     The &#9004;clpX&#65533;?&#11022;ermAM amplicon crossed the <br /> where regulated sugars S. pneumoniae related                 chromosome EL1383, Ermr mutants selected on <br /> species, including galactose, maltose, fucose uti-             TSA-BA plates containing inducer 0.25% (wt/vol) L-fucose. <br /> lization (1, 3, 3a, 20; P. F. Chan, K. A. Ingraham, C. Y. So, M.             Now frequency recovery Ermr transformants in- <br /> Lonetto, M. Rosenberg, D. J. Holmes, M. Zalacain, Abstr.                 creased dramatically (2.1 &#11003; 10&#11002;3 &#9004;clpX&#65533;?&#11022;ermA transfor- <br /> 100th Gen. Meet. Am. Soc. Microbiol., abstr. H-97, p. 370,                   mants compared &#65533;?2.0 &#11003; 10&#11002;6 EL59 [see above]) was <br /> 2000). We chose promoter upstream fucose ki-                    comparable Novr control (4.5 &#11003; 10&#11002;4). The <br /> nase gene (fcsK spr1973) (PfcsK) (14), previously             resulting regulated replacement EL1387 (Table 1) grew like <br /> been reported provide regulation S. pneumoniae (3a; W.                 EL59 (clpX&#11001; parent) EL1383 [&#9004;bgaA::(Pc-tetL::T1T2::Pf- <br />                                                                                       &#11001;       &#11001; <br /> Bae, P. Chan, L. Palmer, V. Clausen, J. Throup, M.                           csK-clpX ) clpX     merodiploid] plates containing L-fucose <br /> Noordewier, K. Koretke, D. Lunsford, A. Bryant, K. Ingraham,                 completely failed grow L-fucose omitted from <br /> D. Holmes, M. Rosenberg, M. Burnham, Abstr. 101st                        plates (Fig. 3A). The lack growth EL1387 absence <br /> Gen. Meet. Am. Soc. Microbiol., abstr. B-381, p. 126, 2001).                 L-fucose consistent polar effect expression of <br /> We constructed cassette shown Fig. 1B                    essential downstream spr1426 gene, expres- <br /> promoter region clpX&#11001; replaced PfcsK, pro-                sion clpX&#11001; gene control L-fucose the <br /> tected transcription terminators T1T2 connected                  &#9004;bgaA locus. Thus, expression clpX&#11001; essential the <br /> tetL (tetracycline resistance [Tetr]) gene driven consti-             growth S. pneumoniae R6. <br /> tutive promoter (Pc) (16). The Pc-tetL::T1T2::PfcsK-clpX&#11001; cas-                  Although clpX essential clpP not, formally <br /> sette crossed bgaA locus, encodes dispens-              possible essentiality clpX depend func- <br /> able &#65533;?&#65533;-galactosidase (34) partly deleted                     tional clpP&#11001;, clpC&#11001;, clpE&#11001;, clpL&#11001; gene. For example, ClpX <br />  VOL. 185, 2003                                                                                                                         NOTES          2965 <br />  <br />  <br /> could interact ClpP protease subunit                  Implications. Our finding clpX essential, clpP is <br /> other ATPase specificity subunits limit ClpP proteolysis               not, unusual, ClpX usually functions regulatory <br /> activity. To test notion, transformed EL1387 grown             subunit ClpXP protease (10, 32). We constructed and <br /> fucose-containing medium clpP::kan::rpsL&#11001;, clpC::kan::             confirmed presence single-copy insertion-deletion mu- <br /> rpsL&#11001;, clpE::kan::rpsL&#11001;, clpL::kan::rpsL&#11001; <span id='am-175' about='obo:NCBITaxon_1385265' typeof='owl:Thing obo:NCBITaxon_27524 obo:NCBITaxon_33340 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_222823 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7399 obo:NCBITaxon_50557 obo:NCBITaxon_6072 obo:NCBITaxon_6960'><span id='am-176' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-177' property="rdfs:label" content="Janus" datatype="xsd:string"></span><span id='am-178'  ></span><span id='am-179' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Janus</span> cassette               tations inactivate clpP, clpC, clpL, clpE (strain EL539, <br /> amplicon (27). For transformation, recovered hun-                 EL854, EL1082, EL1259, respectively [Table 1]). None was <br /> dreds transformants TSA-BA plates containing fucose                 essential growth S. pneumoniae R6 37&#176;C (data not <br /> and kanamycin transformants TSA-BA plates con-                  shown), clpP clpE imparted temperature sensi- <br /> taining kanamycin lacking fucose (data shown). Thus,              tivity BHI liquid medium reported previously (5, 22; data <br /> clpX essentiality did depend functional clpP&#11001;, clpC&#11001;,            shown). Curiously, clpE mutant temperature <br /> clpE&#11001;, clpL&#11001; gene.                                                     sensitive TSA-BA plates 40&#176;C (data shown), whereas <br />    Phenotypes clpX underexpression. We examined pheno-                 clpP mutant (22). The <span id='am-164' about='obo:NCBITaxon_102234' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_1301283 obo:NCBITaxon_1118 obo:NCBITaxon_1117'><span id='am-165' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-167'  ></span><span id='am-168' property="oboInOwl:hasDbXref" content="PMID:6416126" datatype="xsd:string"></span><span id='am-169' property="rdfs:label" content="Cyanobacterium" datatype="xsd:string"></span>cyanobacterium</span> <span id='am-1' about='obo:NCBITaxon_1129' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_1301283 obo:NCBITaxon_1118 obo:NCBITaxon_1117'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="Anacystis" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasDbXref" content="PMID:11411708" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="Synechococcus" datatype="xsd:string"></span>Synechococcus</span> <br /> types bacteria depleted ClpX learn               sp. strain PCC 7942 genes encoding ClpP isoenzymes <br /> basis essentiality clpX. Repetition experiment          single copy ClpX. Viable deletion-insertion mutants of <br /> shown Fig. 3A 30&#176;C showed clpX expression                  Synechococcus obtained clpPI clpPII, sug- <br /> essential lower temperature 37&#176;C bac-            gesting clpPIII clpX indispensable growth (6, <br /> teria spread TSA-BA plates (data shown). We                23). ClpX ClpP essential C. crescentus (7, 15), <br /> that 0.05% (wt/vol) fucose minimum concentration                  chaperone functions ascribed ClpX besides <br /> that supported growth regulated replacement mutant                     role ClpXP proteolysis (17, 24, 30). A recent report shows <br /> EL1387 BHI broth 37&#176;C; however, concentration               E. coli K-12 lacking functional ClpX ClpP lost viability <br /> fucose led lower growth yields EL1387 reg-              rapidly did wild-type parent strains extended <br /> ulated merodiploid EL1383 (final culture optical density 620           stationary phase (31). <br /> nm, 0.17 compared 0.47, respectively). We determined                 The results reported indicate ClpX play es- <br /> the viability cells regulated replacement mutant                sential roles independent ClpP S. pneumoniae R6 (Fig. <br />                                                                           3A; above). Furthermore, depletion ClpX leads to <br /> EL1387 following downshift low fucose concentration <br />                                                                           rapid cell death overtly affecting cell morphology (Fig. <br /> (Fig. 3B). Addition 0.1% (wt/vol) fucose supported full <br />                                                                           3B; data shown). Unlike clpP, clpC, clpE, clpL, clpX <br /> growth viable cells BHI broth 37&#176;C (Fig. 3B, closed <br />                                                                           appears member heat shock regulon in <br /> symbols). In contrast, cultures EL1387 downshifted to <br />                                                                           S. pneumoniae (22), microarray analyses indicated that <br /> 0.001% (wt/vol) fucose prematurely stopped increasing op- <br />                                                                           clpX transcript significantly changed by <br /> tical density concomitantly rapidly lost cellular viability <br />                                                                           phase growth chemically defined medium suble- <br /> (Fig. 3B). The plateau optical density suggested EL1387 <br />                                                                           thal concentrations antibiotic triclosan novobiocin <br /> was undergoing rapid autolysis. Consistent obser- <br />                                                                           common translation inhibitors, macro- <br /> vation, examination EL1387 phase-contrast microscopy <br />                                                                           lides erythromycin roxithromycin, caused marginal <br /> at various times fucose downshift did reveal signifi- <br />                                                                           1.7-fold induction clpX transcript amounts (19; data not <br /> cant changes cellular morphology compared EL59              shown). Genetic physiological experiments progress <br /> and EL1383 controls (data shown). At 360 min fucose             learn essential function(s) ClpX understand <br /> downshift, &#65533;?1% EL1387 cells irregularly shaped                organization possible regulation clpX operon <br /> and large, especially end short chains, compared           (Fig. 1) S. pneumoniae. <br /> those EL59 EL1383 controls (data shown); how- <br /> ever, point significant death occurring                  We thank colleagues listed Table 1 bacterial strains; D. A. <br /> EL1387 cultures.                                                          Morrison Pc-Tetr cassette used studies; K. Kaz- <br />    The rapid cell death EL1387 ClpX depletion (Fig.               mierczak, D. Mytelka T. Nicas, G. Zhao comments the <br /> 3B) analysis global transcription patterns microar-            manuscript. <br />                                                                             This work supported resources provided Lilly Research <br /> rays (19, 22) problematic. Instead tested potential            Laboratories Indiana University Bloomington. Gregory T. Rob- <br /> overexpression ClpX regulated merodiploid EL1383                ertson supported Lilly Postdoctoral Fellowship. <br /> changed transcription pattern. We compared microarray <br /> patterns RNA isolated cultures EL1383 grown                                              REFERENCES <br /> optical density 620 nm &#11011;0.2 BHI 37&#176;C lacking               1. Ajdic, D., J. J. Ferretti. 1998. Transcriptional regulation Strepto- <br /> induced 60 min 0.2% (wt/vol) fucose. On basis                <span id='am-159' about='obo:NCBITaxon_538669' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_33381 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_33380 obo:NCBITaxon_6072 obo:NCBITaxon_7524 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_33373 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_30189 obo:NCBITaxon_33342 obo:NCBITaxon_50557'><span id='am-160'  ></span><span id='am-161' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-162' property="rdfs:label" content="Coccus" datatype="xsd:string"></span><span id='am-163' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>coccus</span> mutans gal operon GalR repressor. J. Bacteriol. 180:5727&#8211; <br />                                                                              5732. <br /> two independent experiments, unable detect sig-                2. Anderson, R. P., J. R. Roth. 1977. Tandem genetic duplications phage <br /> nificant differences transcript patterns EL1383 cultured               bacteria. Annu. Rev. Microbiol. 31:473&#8211;505. <br /> with fucose, exception clpX tran-             3. Bettenbrock, K., C. A. Alpert. 1998. The gal genes Leloir pathway <br />                                                                              <span id='am-65' about='obo:NCBITaxon_1578' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_33958 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186826'><span id='am-66' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="PMID:21724957" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="PMID:19244449" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="PMID:18048724" datatype="xsd:string"></span><span id='am-70'  ></span><span id='am-71' property="oboInOwl:hasDbXref" content="PMID:19060088" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="Lactobacillus" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Lactobacillus</span> casei 64H. Appl. Environ. Microbiol. 64:2013&#8211;2019. <br /> script (increased &#11011;1.7-fold) genes spr1963                3a.Chan, P. F., K. M. O&#8217;Dwyer, L. M. Palmer, J. D. Ambrad, K. A. Ingraham, <br /> spr1973, comprise fucose regulon (fcsK, fucA, fucU,                C. So, M. A. Lonetto, S. Biswas, M. Rosenberg, D. J. Holmes, M. <br />                                                                              Zalacain. 2003. Characterization novel fucose-regulated promoter <br /> PTS-EII [spr1970], PTS-EII [spr1969], PTS-EII [spr1968], PTS-                (PfcsK) suitable gene essentiality antibacterial mode-of-action studies <br /> EII [spr1967], spr1966, spr1965, fucI, adh2 increased &#11011;23-,              Streptococcus pneumoniae. J. Bacteriol. 185:2051&#8211;2058. <br /> 16-, 15-, 11-, 39-, 25-, 10-, 10-, 36-, 28-, 3-fold, respectively).   4. Charpentier, E., R. Novak, E. Tuomanen. 2000. Regulation growth <br />                                                                              inhibition high temperature, autolysis, transformation adherence in <br /> It possible overexpression clpX sufficient             Streptococcus pneumoniae clpC. Mol. Microbiol. 37:717&#8211;726. <br /> these experiments elicit phenotype.                                  5. Chastanet, A., M. Prudhomme, J. P. Claverys, T. Msadek. 2001. Reg- <br />  2966         NOTES                                                                                                                                         J. BACTERIOL. <br />  <br />       ulation Streptococcus pneumoniae clp genes role competence        21. Porankiewicz, J., J. Wang, A. K. Clarke. 1999. New insights the <br />       development stress survival. J. Bacteriol. 183:7295&#8211;7307.                         ATP-dependent Clp protease: <span id='am-125' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-126' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-127'  ></span><span id='am-128' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Escherichia</span> coli beyond. Mol. Microbiol. <br />  6.   Clarke, A. K., J. Schelin, J. Porankiewicz. 1998. Inactivation clpP1       32:449&#8211;458. <br />       gene proteolytic subunit ATP-dependent Clp protease         22. Robertson, G. T., W. L. Ng, J. Foley, R. Gilmour, M. E. Winkler. 2002. <br />       cyanobacterium Synechococcus limits growth light acclimation. Plant               Global transcriptional analysis clpP mutations type 2 Streptococcus <br />       Mol. Biol. 37:791&#8211;801.                                                                pneumoniae effects physiology virulence. J. Bacteriol. 184: <br />  7.   Fuchs, T., P. Wiget, M. Osteras, U. Jenal. 2001. Precise amounts             3508&#8211;3520. <br />       novel member phosphotransferase <span id='am-132' about='obo:NCBITaxon_superfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-133' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-134' property="rdfs:label" content="superfamily" datatype="xsd:string"></span>superfamily</span> essential growth         23. Schelin, J., F. Lindmark, A. K. Clarke. 2002. The clpP multigene family <br />       normal morphology Caulobacter crescentus. Mol. Microbiol. 39:679&#8211;              ATP-dependent Clp protease cyanobacterium Synechococcus. <br />       692.                                                                                  Microbiology 148:2255&#8211;2265. <br />  8.   Gottesman, S. 1996. Proteases targets Escherichia coli. Annu.        24. Schirmer, E. C., J. R. Glover, M. A. Singer, S. Lindquist. 1996. HSP100/ <br />       Rev. Genet. 30:465&#8211;506.                                                               Clp proteins: common mechanism explains diverse functions. Trends Bio- <br />  9.   Gottesman, S. 1999. Regulation proteolysis: developmental switches.                chem. Sci. 21:289&#8211;296. <br />       Curr. Opin. Microbiol. 2:142&#8211;147.                                                 25. Smith, M. D., W. R. Guild. 1979. A plasmid Streptococcus pneumo- <br /> 10.   Gottesman, S., W. P. Clark, V. Crecy-Lagard, M. R. Maurizi. 1993.              niae. J. Bacteriol. 137:735&#8211;739. <br />       ClpX, alternative subunit ATP-dependent Clp protease Esch-          26. Spellerberg, B., D. R. Cundell, J. Sandros, B. J. Pearce, I. Idanpaan-Heik- <br />       erichia coli. Sequence vivo activities. J. Biol. Chem. 268:22618&#8211;22626.        kila, C. Rosenow, H. R. Masure. 1996. Pyruvate oxidase, determi- <br /> 11.   Gottesman, S., M. R. Maurizi. 1992. Regulation proteolysis: energy-            nant virulence Streptococcus pneumoniae. Mol. Microbiol. 19:803&#8211;813. <br />       dependent proteases targets. Microbiol. Rev. 56:592&#8211;621. <br />                                                                                         27. Sung, C. K., H. Li, J. P. Claverys, D. A. Morrison. 2001. An rpsL <br /> 12.   Hartman, P. G. 1993. Molecular aspects mechanism action dihy- <br />                                                                                             cassette, Janus, gene replacement negative selection Strepto- <br />       drofolate reductase inhibitors. J. Chemother. 5:369&#8211;376. <br />                                                                                             coccus pneumoniae. Appl. Environ. Microbiol. 67:5190&#8211;5196. <br /> 13.   Havarstein, L. S., P. Gaustad, I. F. <span id='am-147' about='obo:NCBITaxon_203327' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_8220 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_8219 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489878 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_497223 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665'><span id='am-148' property="rdfs:label" content="Nes" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-151'  ></span>Nes</span>, D. A. Morrison. 1996. Identi- <br />                                                                                         28. Tettelin, H., K. E. Nelson, I. T. Paulsen, J. A. Eisen, T. D. Read, S. Peterson, <br />       fication streptococcal competence-pheromone receptor. Mol. Micro- <br />                                                                                             J. Heidelberg, R. T. DeBoy, D. H. Haft, R. J. Dodson, A. S. Durkin, M. <br />       biol. 21:863&#8211;869. <br />                                                                                             Gwinn, J. F. Kolonay, W. C. Nelson, J. D. Peterson, L. A. Umayam, O. White, <br /> 14.   Hoskins, J., W.  <br /> </body></html>