<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal Vol. 20 No. 15 pp. 4204&#177;4213, 2001 <br />  <br />  <br /> Processing 20S pre-rRNA 18S ribosomal RNA <br /> in yeast requires Rrp10p, essential non-ribosomal <br /> cytoplasmic protein <br />  <br />  <br /> Emmanuel Vanrobays, Pierre-Emmanuel                                     25S 5.8S large ribosomal subunit of <br /> Gleizes, Ce&#194;cile Bousquet-Antonelli1,                                   60S, 18S rRNA small ribosomal <br /> Jacqueline Noaillac-Depeyre,                                            subunit 40S. Concomitantly maturation <br /> Miche&#65533;?le Caizergues-Ferrer                                          process, numerous ribosomal non-ribosomal proteins <br /> Jean-Paul Ge&#194;lugne2                                                     associate pre-rRNA intermediates generate the <br />                                                                         pre-60S 43S pre-ribosomal subunits, precursors the <br /> LBME du CNRS, 118 route Narbonne, 31062 Toulouse cedex 04,           60S 40S ribosomal subunits, respectively. These pre- <br /> France <br />                                                                         ribosomal subunits eventually exported cyto- <br /> 1 <br />  Present address: Institute Cell Molecular Biology, University   plasm, &#174;nal assembly maturation steps occur <br /> of Edinburgh, King&apos;s Buildings, Edinburgh EH9 3JR, UK                   (for recent thorough reviews Kressler et al., 1999; <br /> 2 <br />  Corresponding author                                                   Venema Tollervey, 1999). <br /> e-mail: jpg@ibcg.biotoul.fr                                                Processing intermediates formed 35S pre-rRNA <br />                                                                         maturation characterized early course rRNA <br /> Numerous      non-ribosomal      trans-acting    factors                processing studies. The identi&#174;cation functional <br /> involved pre-ribosomal RNA processing                      characterization trans-acting factors involved in <br /> characterized, speci&#174;cally                          process constituted `key questions&apos; the <br /> required cytoplasmic steps 18S rRNA                     early nineties (Woolford Warner, 1991). During the <br /> maturation. Here demonstrate Rio1p/Rrp10p                       past decade, thanks development new biochem- <br /> is factor. Previous studies showed                      ical, genetic silico technologies, dozens these <br /> RIO1 gene essential cell viability conserved                 components identi&#174;ed yeast. These small <br /> from archaebacteria man. We isolated RIO1                          nucleolar components ribonucleoprotein particles <br /> mutant screen mutations synthetically lethal                   (snoRNPs) non-ribosomal nucleolar proteins: putative <br /> with mutant allele GAR1, essential gene                         helicases, ribonucleases factors, actual <br /> required 18S rRNA production rRNA pseudo-                       functions de&#174;ned (for reviews <br /> uridylation. We RIO1 encodes cytoplas-                      Kressler et al., 1999; Venema Tollervey, 1999). <br /> mic non-ribosomal protein, depletion                        Although processing cleavage sites 35S <br /> Rio1p blocks 18S rRNA production leading 20S                         precursor precisely determined, endoribo- <br /> pre-rRNA accumulation. In situ hybridization reveals                    nucleases acting sites identi&#174;ed very <br /> that, Rio1p depleted cells, 20S pre-rRNA localizes                   them, snoRNP MRP site A3 and <br /> in cytoplasm, demonstrating accumulation                   Rnt1p 3&#162; external transcribed spacer (3&#162;ETS) <br /> is export defect. <span id='am-98' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-99' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-101'  ></span><span id='am-102' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> strongly suggests                  (for review Venema Tollervey 1999). <br /> that Rio1p involved cytoplasmic cleavage                      Early cleavage steps 35S pre-rRNA sites A0, <br /> 20S pre-rRNA site D, producing mature 18S                            A1 A2 yield 20S 27SA pre-rRNAs, precursors <br /> rRNA. Thus, Rio1p renamed Rrp10p (ribo-                        rRNAs small ribosomal subunit large <br /> somal RNA processing #10). Rio1p/Rrp10p &#174;rst                     ribosomal subunit, respectively. In S.cerevisiae, mature <br /> non-ribosomal factor characterized speci&#174;cally                          18S rRNA formed endoribonucleolytic <br /> required 20S pre-rRNA processing.                                   cleavage 20S pre-rRNA site D (Figure 1), which <br /> Keywords: pre-rRNA processing/ribosome/                                 occurs cytoplasm (Udem Warner, 1973; <br /> <span id='am-50' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-54'  ></span><span id='am-55' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>/synthetic lethal screen                        Trapman et al., 1975; Stevens et al., 1991). Impairment <br />                                                                         processing step results 20S pre-rRNA accumu- <br />                                                                         lation concomitant 18S rRNA depletion. This <br />                                                                         phenotype stem defects 43S pre-subunit <br /> Introduction                                                            assembly and/or nucleocytoplasmic transport well, as <br /> In eukaryotic cells, steps ribosomal subunit                    demonstrated Ran regulators nucleoporin mutants <br /> biogenesis place specialized nuclear structure,               (Moy Silver, 1999). While numerous factors involved <br /> the nucleolus, seen primarily,                     early cleavages 35S pre-rRNA been <br /> exclusively, ribosome factory. In yeast                      identi&#174;ed, factors speci&#174;cally required the <br /> <span id='am-67' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-68' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-72'  ></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae, transcription RNA                          processing 20S pre-rRNA 18S rRNA been <br /> polymerase I repeated rDNA genes produces                      described. Deletion DRS2 (Ripmaster et al., 1993) of <br /> primary transcript contains mature                    RPS31/UBI3, encodes ribosomal protein the <br /> ribosomal RNAs (rRNAs), internal external                  small subunit (Finley et al., 1989), results decreased <br /> spacer sequences. The primary transcript chemically                  ef&#174;ciency conversion 20S pre-rRNA 18S <br /> modi&#174;ed undergoes endo- exonucleolytic reac-                    rRNA, produced, slower rates. On <br /> tions (see Figure 1) lead ultimately mature rRNAs             contrary, deletion RPS0A RPS0B, two <br />  <br /> 4204                                                                                           Published Oxford University Press <br />                                                                                        Site D cleavage 20S pre-rRNA requires Rrp10p <br />  <br />  <br />  <br />  <br /> Fig. 1. Pre-rRNA processing S.cerevisiae. (A) Structure 35S pre-rRNA. In primary transcript sequences mature 18S, 5.8S and <br /> 25S rRNAs &#175;anked external transcribed spacers (5&#162; 3&#162;ETS) separated internal transcribed spacers (ITS1 ITS2). Cleavage <br /> sites indicated uppercase letters A0&#177;E, oligonucleotide probes used northern blot hybridizations numbers 1&#177;6. (B) Pre-rRNA <br /> processing pathway. Sequential cleavages 35S pre-rRNA sites A0 A1 generate 33S 32S pre-rRNAs. Cleavage 32S <br /> pre-rRNA site A2 ITS1 yields 27SA2 20S pre-rRNAs, precursors RNA components large small ribosomal <br /> subunits, respectively. The 27SA2 precursor processed site A3 RNase MRP, generating 27SA3 pre-rRNA rapidly digested the <br /> 5&#162;&#177;3&#162; exonucleases Rat1p Xrn1p site B1S yielding 27SBS pre-rRNA. This constitutes major pathway. Approximately 15% 27SA2 <br /> molecules processed site B1L leading 27SBL intermediate. Processing site B2, 3&#162; end 25S rRNA, occurs concomitantly with <br /> 27SB 5&#162;-end formation. The 27SBS 27SBL pre-rRNAs follow processing pathway 25S 5.8SS/L cleavage site C2 in <br /> ITS2, followed 3&#162;&#177;5&#162; exonucleolytic digestion 7SS 7SL site C2 E exosome complex, 5&#162;&#177;3&#162; exonucleolytic digestion the <br /> 5&#162; end 25S rRNA. The &#174;nal maturation 20S pre-rRNA endonucleolytic cleavage site D occurs cytoplasm produces the <br /> mature 18S rRNA fragment D-A2 (5&#162; ITS1). The D-A2 fragment degraded 5&#162;&#177;3&#162; exonuclease Xrn1p. <br />  <br /> redundant genes encoding rpS0 ribosomal proteins,                        inhibition 18S rRNA production. Rio1p/Rrp10p is <br /> leads arrest 18S rRNA production correlated                    &#174;rst non-ribosomal protein speci&#174;cally required for <br /> 20S pre-rRNA accumulation (Ford et al., 1999).                               20S pre-rRNA processing cytoplasm character- <br />    Here report characterization new protein                      ized. We suggest Rio1p/Rrp10p catalyse, be <br /> speci&#174;cally required process 20S pre-rRNA 18S                      complex catalysing, endonucleolytic cleavage <br /> rRNA. This factor, encoded essential RIO1/RRP10                       20S pre-rRNA site D. <br /> gene (Angermayr Bandlow, 1997), homologues in <br /> all eukaryotes archaebacterial organisms sequenced                    Results <br /> far. We demonstrate RIO1/RRP10 encodes cyto-                          A synthetic lethal screen gar1-10 mutant <br /> plasmic non-ribosomal protein. Its inactivation results                   identi&#174;es RIO1 <br /> drastic inhibition cleavage site D, leading strong                Gar1p, core proteins H/ACA snoRNPs <br /> accumulation 20S pre-rRNA cytoplasm                     (Lubben et al., 1995; Ganot et al., 1997; Henras et al., <br />  <br />                                                                                                                                             4205 <br />  E.Vanrobays et al. <br />  <br />  <br /> 1998), essential protein required pseudo-         &#174;gure 1 Supplementary <span id='am-178' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-179'  ></span><span id='am-180' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-181' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-182' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span>, available The EMBO <br /> uridylation rRNAs 18S rRNA synthesis (Girard         Journal Online). <br /> et al., 1992; Bousquet-Antonelli et al., 1997). In <span id='am-128' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-129' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-130' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span>        The predicted polypeptide unusual composition, <br /> characterize factors involved processes,            containing 18% glutamic aspartic acids 11.5% <br /> carried synthetic lethal (sl) screens mutant alleles   serine threonine. The mutation the <br /> of GAR1. Strain YO89 non-sectoring,         rio1-1 allele causes lethality gar1-10 strains a <br /> 5&#162;-&#175;uoroorotic acid (5-<span id='am-1' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Foa" datatype="xsd:string"></span>FOA</span>)-sensitive clones, isolated     missense mutation leading change universally <br /> screen undertaken gar1-10 temperature-sensitive        conserved threonine 62 S.cerevisiae protein to <br /> allele (see Materials methods Venema et al.,            isoleucine. <br /> 1997). This clone regained sectoring phenotype               Since mutant allele RIO1 retrieved as <br /> segregated FOA-resistant (FOAr) cells transform-           synthetically lethal mutant allele GAR1 <br /> ation pJPG67 carrying GAR1+ allele,       gene, product clearly required pre- <br /> transformation vector carrying gar1-10 allele,       rRNA processing post-transcriptional modi&#174;cation <br /> demonstrating carries mutation(s) synthetically         (Girard et al., 1992; Bousquet-Antonelli et al., 1997), we <br /> lethal gar1-10. Strain YO89 crossed             investigated possible involvement Rio1p the <br /> MATa derivative parental strain YO126, diploid       production mature rRNAs. <br /> clones obtained segregate FOAr cells,               As shown below, RIO1 appears involved the <br /> tetrad dissection, 2:2 segregation sl character    processing pre-rRNA consequently been <br /> observed, demonstrating caused single           renamed RRP10 (ribosomal rNA processing #10). <br /> recessive mutation. <br />    To clone wild-type allele sl mutation, strain      Depletion Rrp10p affects 18S rRNA production <br /> YO89-13, derivative YO89                      Since RIO1/RRP10 shown essential gene <br /> LEU2::gar1-D disruption replaced                  (Angermayr Bandlow, 1997), conditional RIO1/ <br /> TRP1::gar1-D allele, transformed yeast               RRP10 allele constructed replacing promoter by <br /> multi-copy genomic library carried 2m LEU2 vector        inducible GAL10 promoter. The 5&#162; end construct <br /> pFL46S (Bonneaud et al., 1991). Eight leu+ sectoring            tagged protein A epitopes (see <br /> colonies grow 5-FOA plates obtained          Supplementary data description construc- <br /> (loss pJPG203: ARS, CEN, ADE3, GAR1). Among                  tion). Transcription driven GAL10 promoter allows <br /> these, seven received library GAR1-              high expression gene culture media contain <br /> containing plasmid segregate lysine-           galactose (YPG, YNB Gal), repressed culture <br /> requiring cells (loss pJPG225: ARS CEN, LYS2,                media containing glucose (YPD, YNB Glu), leading the <br /> gar1-10). The library plasmid recovered            depletion ProtA-Rrp10p (see Supplementary &#174;gure 2). <br /> remaining transformant shown complement sl              We use conditional RRP10 allele to <br /> phenotype retransformation strain YO89-13.            determine Rrp10p necessary pre-rRNA <br /> This plasmid, pEV6, partially sequenced shown        processing. We &#174;rst assessed steady-state levels of <br /> carry fragment chromosome XV (from nucleotide            mature rRNAs ethidium bromide staining northern <br /> 548 151 nucleotide 552 896) containing complete        blotting RNAs prepared cells grown permissive <br /> open reading frames (ORFs): (i) GCY1, encodes          YPG medium shifted non-permissive YPD medium <br /> aldo/keto reductase unknown substrate speci&#174;city             2&#177;24 h. As shown Figure 2A, depletion Rrp10p <br /> (Angermayr Bandlow, 1997); (ii) essential gene           effect 25S rRNA accumulation drastically <br /> RIO1 (Angermayr Bandlow, 1997); (iii)                affects level 18S rRNA. Furthermore, 16 h <br /> uncharacterized ORF, YOR121C (included GCY1                  culture YPD medium, RNA molecules migrate <br /> ORF). When plasmid carrying subcloned RIO1                similarly 20S pre-rRNA accumulate suf&#174;cient <br /> wild-type allele (pEV21) introduced sl strain        level detected ethidium bromide staining <br /> YO89, red/white sectoring growth 5-FOA medium            (Figure 2A). <br /> were restored, case plasmid           To determine RNA <span id='am-125' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-126' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> related the <br /> pEV22 carrying rio1-1 mutant allele derived            20S pre-rRNA, precursor 18S rRNA, northern <br /> YO89 genomic DNA (see Materials methods).                   hybridizations using probe 1 complementary 18S <br /> Furthermore, plasmid pEV22 allows growth           rRNA, probe 2 complementary ITS1 sequence <br /> glucose-containing medium strain YO296,                sites D A2 (see Materials methods and <br /> carries RIO1+ allele control GAL10           Figure 1), performed. Results hybridizations <br /> promoter, demonstrating mutation responsible       shown Figure 2B C. The RNA species that <br /> the colethality gar1-10 lies RIO1        accumulates Rrp10p depleted cells detected with <br /> gene.                                                           probes 1 2, migrates position the <br />    This gene previously identi&#174;ed S.cerevisiae,     normal 20S pre-rRNA intermediate (Figure 2B C, lane <br /> and shown essential cell viability                Gal). Probing blot probe 3 (complementary <br /> (Angermayr Bandlow, 1997). Occurrence genes              ITS1 sequences sites A2 A3) did reveal <br /> homologous RIO1 archaebacteria, <span id='am-83' about='obo:NCBITaxon_5071' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_5042 obo:NCBITaxon_33154 obo:NCBITaxon_147545 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_1131492 obo:NCBITaxon_451871 obo:NCBITaxon_2759'><span id='am-84' property="rdfs:label" content="Emericella" datatype="xsd:string"></span><span id='am-85'  ></span><span id='am-86' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Emericella</span>                pre-rRNA intermediate size close (data <br /> nidulans (formerly <span id='am-77' about='obo:NCBITaxon_5052' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_5042 obo:NCBITaxon_33154 obo:NCBITaxon_147545 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_1131492 obo:NCBITaxon_451871 obo:NCBITaxon_2759'><span id='am-78' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasAlternativeId" content="NCBITaxon:37503" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Aspergillus" datatype="xsd:string"></span><span id='am-82'  ></span>Aspergillus</span> nidulans), <span id='am-93' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-94' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span><span id='am-95'  ></span><span id='am-96' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Caenorhabditis</span>        shown), indicating RNA species accumulating <br /> elegans <span id='am-103' about='obo:NCBITaxon_9606' typeof='owl:Thing obo:NCBITaxon_9605 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-104' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-105' property="rdfs:label" content="Homo sapiens" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasExactSynonym" content="human" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasExactSynonym" content="man" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-109'  ></span>Homo sapiens</span> reported (Anaya et al.,       Rrp10p depleted cells 20S pre-rRNA extends <br /> 1998). Our search databases identi&#174;ed homologues          5&#162; end 18S site A2 ITS1 (Figure 1). These <br /> Rio1p eukaryotic archaebacterial sequenced           data depletion Rrp10p severely impairs 18S <br /> genomes (for sequence alignments Supplementary              rRNA production leads accumulation 20S <br />  <br /> 4206 <br />                                                                                      Site D cleavage 20S pre-rRNA requires Rrp10p <br />  <br />  <br />  <br />  <br />                                                                            Fig. 3. Depletion Rrp10p results reduced synthesis 18S rRNA. <br />                                                                            (A) YO296 (GAL::PROTA-RRP10) strain grown 30&#176;C YNB Gal <br />                                                                            methionine; (B) YO296 grown 30&#176;C YPG medium (Gal) <br />                                                                            shifted 14 h YPD medium (Glu), &#174;nally grown 9 h <br />                                                                            YNB Glu methionine. Cells labelled 4 min with <br />                                                                            (methyl-3H) methionine chased large excess unlabelled <br />                                                                            methionine 1&#177;20 min. <br />  <br />  <br />  <br /> Fig. 2. Depletion Rrp10p speci&#174;cally affects steady-state           35S pre-rRNA molecules. This suggests <br /> levels mature 18S rRNA species results 20S pre-rRNA              20S pre-rRNA intrinsically unstable not <br /> accumulation. Northern blot analysis pre-rRNA processing: YO296         converted 18S rRNA degraded exosome <br /> (GAL::PROTA-RRP10) cells grown YPG medium (Gal),             complex (Allmang et al., 2000), Rrp10p depletion <br /> YPD medium 24 h (Glu). At indicated time points, total <br /> RNA extracted separated 1% agarose-formaldehyde gels         leads stable form precursor, possibly to <br /> analyse 35S, 32S, 27SA2, 25S, 20S 18S species, 6%               defect assembly 43S pre-subunits, or <br /> polyacrylamide gels 7S(L), 7S(S) 5.8S species analysis. Equal      both. <br /> amounts total RNA (5 mg) loaded lane. (A) Ethidium           The high molecular weight blot shown Figure 2B was <br /> bromide staining gel. Northern blots gel hybridized <br /> to: (B) probe 2 complementary ITS1 upstream site A2; (C) probe 1     probed probes 5 6, complementary the <br /> complementary 18S rRNA, 7 h exposure (upper panel C)         ITS2 25S rRNA, respectively (see Figure 1 and <br /> 90 min exposure (lower panel C) blot shown;             Materials methods). A low molecular weight blot was <br /> (D) probe 4 complementary 5.8S, probe 5 complementary ITS2           probed probes 4 (complementary 5.8S rRNA) <br /> and probe 6 complementary 25S rRNA. Pre-rRNA rRNA <br /> species schematically represented right side; rectangles   5 (complementary ITS2). Apart what <br /> represent mature rRNAs lines transcribed spacers.         observed 18S 20S rRNAs, neither <br />                                                                            changes levels normal pre-rRNA processing <br />                                                                            intermediates, accumulation aberrant species was <br /> pre-rRNA precursor. Phosphorimager quanti&#174;cations                       detected depletion Rrp10p experiments <br /> the 20S 18S bands northern blot shown                        (Figure 2D). <br /> Figure 2C reveal 40-fold reduction 18S                     The kinetics pre-rRNA processing rRNA accu- <br /> rRNA 20-fold increase 20S pre-rRNA levels,                    mulation analysed (methyl-3H) methionine pulse&#177; <br /> in cells grown 16 h YPD medium compared                    chase experiments (Figure 3). In GAL::PROTA-RRP10 <br /> ones cells grown galactose medium. Considering                   strain grown presence galactose, the <br /> observed evolution 18S/25S ratio,                 labelled 27S 20S pre-rRNAs disappeared a <br /> amount total cellular RNA loaded lane                 10 min chase. In contrast, 20 min chase, 20S pre- <br /> and corresponding number cells used                     rRNA present 18S rRNA faintly <br /> amount RNA (given dramatic inhibition 18S                        detectable cells GAL::PROTA-RRP10 <br /> rRNA production depletion Rrp10p, cells,                      strain grown 14 h glucose-containing medium. On <br /> collected later time point samples culture              contrary, Rrp10p depletion does affect time <br /> YPD medium, needed total                    course 27S pre-rRNA conversion 25S rRNA. <br /> cellular RNA), actual drop 18S rRNA level                   Moreover, pre-rRNA intermediate observed <br /> more dramatic 40-fold. Moreover, small                     chase experiment, indicating processing <br /> of 18S rRNA cells (Figure 2A) probably                      pathways leading 20S pre-rRNA 25S rRNA are <br /> stems incomplete depletion Rrp10p leaks                  fully functional Rrp10p depleted cells, 18S rRNA <br /> glucose transcriptional repression GAL10 promoter                   formation strongly inhibited conditions a <br /> and/or survival Rrp10p molecules produced                     defect 20S pre-rRNA processing. <br /> early switch glucose medium. Consequently                        In order determine Rrp10p depletion specif- <br /> we think deletion Rrp10p, production 18S                   ically affects processing 20S pre-rRNA, levels of <br /> rRNA completely inhibited. It appears 20S                 small nuclear nucleolar RNAs assessed in <br /> pre-rRNA does accumulate stoichiometrically                       cells strain YO296 shifted YPD medium. All tested <br /> respect 25S rRNA, arising processing                  snoRNAs unaffected (data shown). <br />  <br />                                                                                                                                           4207 <br />  E.Vanrobays et al. <br />  <br />  <br />                                                                            subsequently processed site D unknown <br />                                                                            endonuclease (Udem Warner, 1973; Trapman et al., <br />                                                                            1975), leading 18S rRNA D-A2 5&#162;-ITS1 fragment <br />                                                                            degraded cytoplasmic 5&#162;&#177;3&#162; exoribonuclease <br />                                                                            Xrn1p (Stevens et al., 1991). Thus, accumulation the <br />                                                                            20S pre-rRNA nuclear retention of <br />                                                                            43S particles, precursors small ribosomal <br />                                                                            subunit, defect cytoplasmic processing of <br />                                                                            20S pre-rRNA. To sort possibilities, the <br />                                                                            localization 20S pre-rRNA accumulating Rrp10p <br />                                                                            depleted cells determined situ hybridization with <br />                                                                            probe complementary D-A2 fragment. <br />                                                                            Ultrastructural localization probe analysed by <br />                                                                            <span id='am-173' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-174' property="rdfs:label" content="Electron" datatype="xsd:string"></span><span id='am-175' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-176' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-177'  ></span>electron</span> microscopy (Figure 4). In wild-type cells <br />                                                                            (Figure 4A) YO296 cells grown galactose <br />                                                                            medium (not shown), labelling, corresponds to <br />                                                                            35S, 32S 20S precursors, essentially restricted to <br />                                                                            nucleolus. In contrast, Rrp10p depleted cells exhibit <br />                                                                            nucleolar cytoplasmic labelling (Figure 4C), a <br />                                                                            phenotype similar observed Dxrn1 strain <br />                                                                            (Figure 4B). Thus, 20S pre-rRNA accumulation Rrp10p <br />                                                                            depleted cells does result defect nuclear <br />                                                                            export 43S particle, impairment <br />                                                                            cytoplasmic endonucleolytic processing 20S <br />                                                                            pre-rRNA. <br />  <br />                                                                            Rrp10p non-ribosomal cytoplasmic protein <br />                                                                            Subcellular localization Rrp10p investigated using <br />                                                                            green &#175;uorescent protein (GFP)-Rrp10p fusion protein. <br />                                                                            Strain YO296 (GAL::PROTA-RRP10) regained ability <br />                                                                            grow YPD transformation pEV24 <br />                                                                            plasmid, directs expression GFP-Rrp10p <br />                                                                            construct, showing fusion functional. In strain <br />                                                                            JG540 (wild type) expressing GFP-RRP10 fusion allele <br />                                                                            (pEV24), green &#175;uorescence detected the <br />                                                                            cytoplasm (Figure 5A). In strain, fusion of <br />                                                                            nucleolar protein Gar1p GFP localizes the <br />                                                                            nucleolus, expected (Girard et al., 1992) (Figure 5B). <br />                                                                            The cytoplasmic localization Rrp10p con&#174;rmed by <br /> Fig. 4. Cells depleted Rrp10p accumulate 20S pre-rRNA            immunodetection ProtA-Rrp10p fusion protein. <br /> cytoplasm. Electron microscopic detection ITS1: (A) wild-        Immunodetection ProtA-Rrp10p Gar1p-ProtA <br /> type yeast cells, labelling exclusively detected nucleolus <br /> (35S, 33S, 32S 20S pre-rRNAs); (B) Dxrn1 cells, labelling <br />                                                                            fusions electron microscopy shown Figure 5C <br /> is nucleolus cytoplasm (fragment D-A2         D, respectively. Gar1p nucleolus, but <br /> the ITS1); (C) GAL::PROTA-RRP10 cells grown 16 h YPD             Rrp10p exclusively localizes cytoplasm. Since <br /> medium (Glu), labelling detected nucleolus        Rrp10p required maturation 20S pre-rRNA <br /> cytoplasm. np, nucleoplasm; no, nucleolus; ct, cytoplasm. Bar, 200 nm.     located cytoplasm, assessed possibility <br />                                                                            Rrp10p ribosomal protein. In analysis ProtA- <br />                                                                            Rrp10p detected ribosomal subunit, <br /> Likewise, levels RNase MRP RNA U1                            whereas, expected, ribosomal proteins rpL3 and <br /> snRNA altered 24 h growth YPD                    rpL32 present 60S subunit, rpS4 40S <br /> medium. Northern hybridizations probes speci&#174;c                    subunit (see Supplementary &#174;gure 3). Moreover, Rrp10p is <br /> ACT1 rpS11 ribosomal protein mRNAs did reveal                      80S ribosomes fractions gradient <br /> any defect pre-mRNA processing accumulation (data                    shown Figure 6A (see below). These results that <br /> not shown). Altogether data depletion                   localized cytoplasm required for <br /> Rrp10p strongly speci&#174;cally affects processing                  cleavage site D 20S pre-rRNA, Rrp10p a <br /> 20S pre-rRNA 18S rRNA.                                                  ribosomal protein. In agreement data, Rrp10p <br />                                                                            referenced recent compilation ribosomal <br /> Cells depleted Rrp10p accumulate 20S pre-rRNA                           proteins (Planta Mager, 1998). <br /> in cytoplasm <br /> In yeast, shown previously steps              Rrp10p associated higher order structures <br /> 18S rRNA maturation occur cytoplasm. After                          translating ribosomes <br /> cleavage site A2, 20S pre-rRNA exported                 To assess Rrp10p present cytoplasm as <br /> nucleolus 43S pre-ribosomal particle                   free molecules associated higher order structures, a <br />  <br /> 4208 <br />                                                                                          Site D cleavage 20S pre-rRNA requires Rrp10p <br />  <br />  <br />  <br />  <br /> Fig. 5. Rrp10p cytoplasmic protein. Subcellular localization using GFP-tagged proteins (A B). (A) GFP-Rrp10p: cells transformed pEV24 <br /> (GFP::RRP10) exhibit cytoplasmic pattern &#175;uorescence. (B) Gar1p-GFP: cells transformed pZUT3 (GAR1::GFP) punctate pattern <br /> characteristic nucleolar staining observed. Positions nuclei determined DAPI staining (blue). Overlay images shown by <br /> superposition blue green stainings. Immunolocalization electron microscopy (C D). (C) ProtA-Rrp10p; (D) Gar1p-ProtA. Tagged <br /> proteins detected treatment anti-protein A antibodies followed incubation colloidal gold-conjugated protein A. no, nucleolus; np, <br /> nucleoplasm; ct, cytoplasm. Bars C D, 200 nm. <br />  <br />  <br />  <br />                                                                               cellular extract strain YO296 expressing ProtA- <br />                                                                               Rrp10p fractionated sedimentation through <br />                                                                               10&#177;30% glycerol gradients, fractions this <br />                                                                               gradient analysed protein RNA contents. <br />                                                                               The relative positions 40S, 60S ribosomal subunits and <br />                                                                               80S monosomes determined probing RNA <br />                                                                               extracted fractions 18S- 25S-speci&#174;c <br />                                                                               probes. Probing blot 20S pre-rRNA <br />                                                                               speci&#174;c probe (fragment D-A2 Figure 1) revealed that <br />                                                                               precursor 18S rRNA exclusively in <br />                                                                               particles exhibit sedimentation coef&#174;cient ~40S <br />                                                                               (Figure 6B). Likewise, immunodetection PAP anti- <br />                                                                               body showed Rrp10p sediments same <br />                                                                               fractions 20S pre-rRNA (Figure 6A). Less 5% of <br />                                                                               ProtA-Rrp10p fusion protein &#174;rst <br />                                                                               fractions gradient corresponding soluble factors, <br />                                                                               detected 80S fraction (Figure 6A). Thus, <br />                                                                               Rrp10p associated higher order structures <br />                                                                               sedimentation coef&#174;cient ~40S, found <br />                                                                               monosomes polysomes, indicating that <br />                                                                               Rrp10p integral component translating <br />                                                                               ribosomes. Since Rrp10p depletion blocks 20S pre-rRNA <br /> Fig. 6. Sedimentation pro&#174;les ProtA-Rrp10p 20S pre-rRNA a <br /> glycerol gradient. A total extract loaded 10&#177;30% glycerol <br />                                                                               processing, Rrp10p monosomes or <br /> gradient subjected centrifugation. Fractions collected,       polysomes, observed cosedimentation Rrp10p and <br /> proteins RNA fraction analysed. (A) Sedimentation            20S pre-rRNA glycerol gradient suggests that <br /> pro&#174;les ProtA-Rrp10p, rpS4p, rpL3p rpL32p. Proteins pre-               Rrp10p associated, transiently, the <br /> cipitated TCA separated SDS&#177;PAGE revealed                 43S particle, precursor 40S ribosomal subunit. In <br /> western blotting. (B) Sedimentation pattern 18S 25S rRNAs <br /> and 20S pre-rRNA. Fractions containing peaks 40S, 60S               order assess association, cell lysates from <br /> ribosomal subunits 80S ribosomes indicated. (C) 20S pre-RNA           RRP10-protA RRP10+ strains incubated IgG <br /> selectively co-puri&#174;es Rrp10-ProtAp. 1/400 clari&#174;ed cell          Sepharose beads salt concentration high to <br /> lysate (T) bulk immunoprecipitated RNA (IP) probed          disrupt unspeci&#174;c weak interactions (0.2 M KCl), the <br /> with mix probes 1 6 Figure 1 (18S, 25S) D-A2 <br /> probe prepared multiprime labelling (20S). The fraction            RNA material retained IgG&#177;Sepharose beads in <br /> rRNA pre-rRNA recovered immunoprecipitate                       conditions extracted, separated agarose gel <br /> determined phosphoimager quanti&#174;cation (IP/T).                             probed 18S-, 25S- 20S-speci&#174;c probes. As <br />  <br />                                                                                                                                                4209 <br />  E.Vanrobays et al. <br />  <br />  <br /> shown Figure 6C, RNA immunoprecipitated             sudD/RRP10 structural homology correlated a <br /> lysate prepared RRP10-protA cells selectively          functional one. In C.elegans, systematic investigation of <br /> enriched 20S pre-rRNA. Such effect               loss function phenotypes predicted genes RNA- <br /> observed using RRP10+ cell lysate (not shown),          mediated interference (RNAi), revealed inhibition by <br /> indicating Rrp10p 20S pre-rRNA              RNAi RRP10 putative homologues (Ce2 in <br /> found molecular structure, 43S           Supplementary &#174;gure 1) leads post-embryonic growth <br /> precursor small ribosomal subunit. Another small         defect Sick phenotype (Fraser et al., 2000), no <br /> RNA fragment hybridizes D-A2 probe          molecular function ascribed gene. By <br /> retrieved immunoprecipitated material (not             speci&#174;cally inhibiting functions selected genes, <br /> shown). Such small fragment observed           RNAi provide way analyse function(s) the <br /> fractions glycerol gradient containing 20S pre-      RRP10 homologues molecular level, especially <br /> rRNA. This suggests slight enrichment 18S           early embryogenesis Drosophila, in <br /> rRNA compared 25S rRNA (600/250 = 2.4)              C.elegans. <br /> immunoprecipitated RNA immunoprecipitation               The functional analysis RRP10 described above <br /> of precursor particles 40S ribosomal subunit       shows RRP10 encodes cytoplasmic protein. It also <br /> containing Rrp10p products processing         demonstrates Rrp10p-depleted cells exhibit strong <br /> 20S pre-rRNA: 18S rRNA D-A2 fragment. This              accumulation 20S pre-rRNA. Such accumulation <br /> suggests Rrp10p directly involved site D             20S pre-rRNA stem from: (i) defects 43S pre- <br /> cleavage precursor 18S rRNA.                         subunit assembly impairing export the <br />                                                                 nucleus &#174;nal processing both; (ii) alteration of <br />                                                                 elements export machinery exempli&#174;ed for <br /> Discussion                                                      prp20-1, rna1-1 yrb1-1 mutants (Moy Silver, <br /> In study, using sl screen, identi&#174;ed &#174;rst non-   1999); (iii) direct inhibition endonucleolytic <br /> ribosomal factor speci&#174;cally required 20S pre-rRNA          cleavage site D. <br /> processing yeast. sl screens developed S.cerevisiae          To knowledge, handful characterized <br /> (Guarente, 1993) proved ef&#174;cient analysis      S.cerevisiae mutants impaired pre-rRNA processing <br /> of large multi-molecular structures like nuclear pore       speci&#174;cally accumulate 20S pre-rRNA, are <br /> (Doye Hurt, 1995) complex pathways tRNA          mutant alleles genes encoding small subunit ribosomal <br /> processing export (Simos et al., 1996) cell polarity     proteins. Inactivation non-essential DRS2 gene leads <br /> and morphogenesis (Madden Snyder, 1998). With               delayed processing 20S pre-rRNA 18S rRNA <br /> respect ribosome biogenesis, screening mutations         impairment 27SA pre-rRNA maturation, <br /> synthetically lethal NOP1 mutant alleles led        lesser extent (Ripmaster et al., 1993). <br /> identi&#174;cation NOP4/NOP77 (Berges et al., 1994),              Drs2p localizes late Golgi membranes (Chen <br /> RRP5 (Venema Tollervey, 1996), NOP56 NOP58              et al., 1999), deletion DRS2 lethal when <br /> (Gautier et al., 1997) functional partners NOP1.          coupled thermosensitive mutations CHC1, dele- <br /> Using GAR1 mutant alleles baits sl screens,            tion ARF1. Since CHC1 ARF1 encode factors <br /> already characterized mutations ROK1, essential           involved formation coated transport vesicles, the <br /> gene encoding putative RNA helicase required 18S          20S pre-rRNA processing defect observed DRS2 <br /> rRNA production (Venema et al., 1997), RRP8         mutants probably indirect re&#175;ects coupling of <br /> encodes putative methyltransferase (Bousquet-Antonelli        ribosome biogenesis membrane synthesis (Mizuta and <br /> et al., 2000). As reported here, sl mutations        Warner, 1994). The CLP-8 mutant described decades <br /> obtained screens resides RIO1/RRP10,              ago (Carter Cannon, 1980; Carter et al., 1980) exhibits <br /> constitutively expressed essential gene (Angermayr          decreased rates 20S pre-rRNA export cytoplasmic <br /> Bandlow, 1997), conserved yeasts man           processing. Although mutant studied <br /> and archaebacteria (Anaya et al., 1998 data herein),     further, phenotype suggests 43S pre-subunits <br /> pointing possible involvement evolutionarily       produced altered way are <br /> conserved biological process. So far function       inef&#174;ciently recognized export the <br /> ascribed Rrp10p homologues identi&#174;ed,             processing machineries. Deletion RPS31/UBI3, which <br /> either based occurrence putative functional motifs     encodes small ribosomal subunit proteins, <br /> on vivo analyses. However, E.nidulans, cold-            speci&#174;cally delays maturation 20S pre-rRNA but <br /> sensitive lethal mutation gene sudD, RRP10           does affect nuclear export 43S pre-ribosomal <br /> homologue, characterized extragenic              subunit. Enough mature 40S subunits synthesized <br /> suppressor thermosensitive mutation bimD,             permit growth, greatly decreased rate <br /> gene possibly involved control chromosome             (Finley et al., 1989). Presumably pre-40S subunits devoid <br /> segregation (Holt, 1996). At non-permissive temperatures,       rpS31 poor substrates processing machinery. <br /> this sudD7 mutation itself leads abnormal nuclear         Finally, depletion rpS0 results block 18S rRNA <br /> morphology. Whether phenotypes displayed             synthesis, pre-40S subunits containing 20S pre-rRNA <br /> sudD7 mutant re&#175;ect direct involvement RRP10        accumulate. The inef&#174;cient incorporation rpS0 <br /> homologue chromosome segregation indirect             depleted pre-40S subunits translating ribosomes leads to <br /> consequences impairment general                shift smaller polysomes, eventually translation <br /> pathway remains determined. In view results,       inhibition (Ford et al., 1999). Clearly proper assembly of <br /> an analysis pre-rRNA processing sudD7 strain          rpS0 pre-40S particles required their <br /> would time reveal        processing mature 40S subunits. <br />  <br /> 4210 <br />                                                                                Site D cleavage 20S pre-rRNA requires Rrp10p <br />  <br />  <br /> Table I. Saccharomyces cerevisiae strains used study <br /> Strain                Genotype                                                                 Reference <br />  <br /> YO24                  MATa ade2 ade3 his4-260 leu2 lys2 trp1-1 ura3-52 can1-100                Henras et al. (1998) <br />                       URA3::GAL::GAR1 <br /> YO89                  MATa ade2 ade3 his3 leu2 lys2 tyr7-1 trp1-1 ura3-52 can1-100             study <br />                       LEU2::gar1-D rrp10-1/pJPG203/pJPG225 <br /> YO89-13               MATa ade2 ade3 his3 leu2 lys2 tyr7-1 trp1-1 ura3-52 can1-100             study <br />                       TRP1::gar1-D rrp10-1/pJPG203/pJPG225 <br /> YO126                 MATa ade2 ade3 his3 leu2 lys2 tyr7-1 trp1-1 ura3-52 can1-100             Venema et al. (1997) <br />                       LEU2::gar1-D/pJPG203/pJPG225 <br /> YO296                 MATa leu2-Dtrp1 his3-D200 gal2 gal-D108 HIS3::GAL::PROTA-RRP10           study <br /> JG540                 MATa ade2 ade3 his4-260 leu2 lys2 trp1-1 ura3-52 can1-100                Bousquet-Antonelli et al. (2000) <br /> R934 (Dxrn1)          MATa ade2-1 his3-11,15 trp1-1 ura3-52 URA3::xrn1-D                       provided S.Kearsey (Petfalski et al., 1998) <br /> YDL402                MATa leu2-Dtrp1 his3-D200 gal2 gal-D108                                  Lafontaine Tollervey (1996) <br />  <br />  <br />  <br />  <br />    Concerning molecular role Rrp10p, 20S                16S equivalent bacteria <span id='am-6' about='obo:NCBITaxon_2157' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567'><span id='am-7' property="oboInOwl:hasDbXref" content="PMID:11211268" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="PMID:11321083" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="PMID:8590690" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="PMID:2112744" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="prokaryotes" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="prokaryote" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="PMID:10843050" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Procaryotae" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="PMID:270744" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Prokaryotae" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="PMID:11540071" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="PMID:12054223" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Metabacteria" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="PMID:11321113" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Mendosicutes" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="PMID:10425797" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="PMID:10425795" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="archaea" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="PMID:9336922" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="PMID:11760965" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="PMID:10490293" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="PMID:9103655" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Prokaryota" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="PMID:11542064" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="PMID:11542149" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Monera" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="Archaea" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="PMID:11541975" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="PMID:10939651" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="PMID:8123559" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="PMID:10939673" datatype="xsd:string"></span><span id='am-40'  ></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Archaebacteria" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="PMID:11411719" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="PMID:10939677" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="PMID:10425796" datatype="xsd:string"></span>archaea</span>, been <br /> pre-rRNA cytoplasm Rrp10p-depleted                characterized. Implication snoRNAs U3, U22 and <br /> cells, involvement pre-40S subunit nuclear export         E2 processing pre-rRNA site 2 Xenopus <br /> machinery itself unlikely. Likewise, Rrp10p             oocytes (the vertebrate equivalent site D S.cerevisiae <br /> inactivation led improper assembly pre-40S                 pre-rRNA) reported (Tycowski et al., 1994; <br /> ribosomal subunit, major consequence              Mishra Eliceiri, 1997; Borovjagin Gerbi, 1999). <br /> on export nucleus. RRP10 expressed                   Whether observed impairment site 2 cleavage a <br /> constitutively (Angermayr Bandlow, 1997) low              direct indirect consequence snoRNA depletions <br /> level. According transcriptome analysis (Holstege et al.,         remains established. An endoribonucleolytic activity, <br /> 1998), average molecule RRP10 mRNA                      partially puri&#174;ed Ehrlich ascites tumor cells, which <br /> present cell, compared amounts range           cleaves vitro 3&#162; region vitro transcribed pre-18S <br /> of 13&#177;15 molecules cell mRNAs encoding snoRNP                rRNA, reported (Shumard et al., 1990) not <br /> components Gar1p, Nhp2p Nop1p, 30&#177;50                       documented. It mentioned that <br /> molecules cell mRNAs encoding ribosomal                      maturation 3&#162; end 18S rRNA occurs in <br /> proteins rpS0. In accordance RRP10 low                  cytoplasm yeast cells, cleavage pre-rRNA at <br /> transcriptional expression level, <span id='am-115' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-116' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-117'  ></span><span id='am-118' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-119' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span> <span id='am-45' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-46' property="rdfs:label" content="Bias" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49'  ></span>bias</span> index            site 2 (3&#162; end vertebrates 18S rRNA) nuclear event <br /> 0.098, calculated according Bennetzen Hall                    vertebrate cells. Thus, vertebrate homologues of <br /> (Bennetzen Hall, 1982; Coghlan Wolfe, 2000),                 Rrp10p functional ones, nuclear <br /> indicates RRP10 mRNA translated low rate,                 proteins. Unfortunately, information concerning the <br /> suggesting cellular Rrp10p concentration             cellular localization proteins presently avail- <br /> be low. Thus, Rrp10p low abundance protein                 able. Considering occurrence RRP10 homologues <br /> localizes cytoplasm, structures          eukaryotic archaebacterial genomes <br /> with sedimentation coef&#174;cient ~40S, selectively             sequenced date, functional homology the <br /> interacts 20S pre-rRNA-containing particles. Rrp10p             members protein <span id='am-74' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-75' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> investigated by <br /> inactivation rapidly leads speci&#174;c total inhibition        assaying functional complementation RRP10 <br /> cleavage site D. These data suggest Rrp10p              S.cerevisiae mutants using cloned cDNAs encoding <br /> factor transiently associated pre-40S subunit,            RRP10 homologues, using RNAi approaches. <br /> involved catalytic step cytoplasmic maturation <br /> of 20S pre-rRNA. In support data been <br /> shown genome-wide two-<span id='am-183' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-184' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-185' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-186' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-187'  ></span>hybrid</span> analysis of <br /> S.cerevisiae (Ito et al., 2001) Rrp10p rpS1ap,            Materials methods <br /> protein 40S ribosomal subunit, physically <br /> interact. We propose inhibition maturation 20S            Media, strains <span id='am-188' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-189' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-190' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-191' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-192' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>plasmids</span> <br /> pre-rRNA arising Rrp10p depletion direct              Yeast media prepared described previously (Bousquet-Antonelli <br /> inhibition activity endonuclease endonu-            et al., 2000). <span id='am-141' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-142' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-144' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-147' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-148'  ></span><span id='am-149' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-153' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-154' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-155' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span>Escherichia coli</span> DH5a used molecular cloning and <br /> cleolytic complex, cleaves pre-rRNA site D.            propagation plasmids. Yeast strains used constructed study <br /> The tempting hypothesis Rrp10p itself carries           listed Table I. Constructions <span id='am-169' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-170' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-171' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-172' property="rdfs:label" content="vectors" datatype="xsd:string"></span>vectors</span> pJPG203 (CEN, ADE3, <br />                                                                      URA3, GAR1), pJPG225 (CEN, LYS2, gar1-10), pJPG219 (CEN, TRP1, <br /> this catalytic activity, key component                gar1-10), pJPG67 (CEN, TRP1, GAR1) pZUT3 (CEN, URA3, <br /> catalytic complex. Another possibility Rrp10p,               GAR1::GFP) described (Girard et al., 1994; Venema et al., <br /> transiently associated pre-40S particle, recruits           1997; Trumtel et al., 2000). Construction pEV24, plasmid directing <br /> the endonucleolytic factor(s). The development                 production GFP-Rrp10p fusion protein described the <br /> in vitro assay site D processing, characteriza-          Supplementary data. For Rrp10p depletion, cells growing exponentially <br />                                                                      YPG medium 30&#176;C harvested centrifugation, washed, and <br /> tion factors interacting Rrp10p allow           resuspended YPD medium. During growth, cells periodically <br /> de&#174;ne better role 18S rRNA synthesis.                         diluted pre-warmed medium order kept exponential <br />    Interestingly, factors involved matur-            growth phase. Yeast cells transformed previously described <br /> ation 3&#162; end 18S rRNA eukaryotic organisms,             (Bousquet-Antonelli et al., 2000). <br />  <br />  <br />                                                                                                                                        4211 <br />  E.Vanrobays et al. <br />  <br /> Synthetic lethal screen                                                        Immunoprecipitations <br /> This screen, based ADE2/ADE3 red/white sectoring coupled                 Yeast cells harvested 0.5 l cultures exponential growth <br /> &#175;uoroorotate sensitivity/resistance tests (Kranz Holm, 1990),          phase (OD600 nm = 0.8), washed extraction buffer (20 mM <br /> been previously described (Venema et al., 1997), brie&#175;y       HEPES pH 8.0, 1 mM MgCl2, 0.2 M KCl, 0.2% Triton X-100, 1 mM <br /> summarized here. Strain YO126, carries insertional disruption         dithiothreitol, supplemented protease inhibitors cocktail), <span id='am-120' about='obo:NCBITaxon_9822' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9821 obo:NCBITaxon_35497 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-121' property="rdfs:label" content="Sus" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-124'  ></span>sus</span>- <br /> of GAR1 gene, wild-type GAR1 allele CEN, ADE3, URA3               pended buffer broken `one shot cell disrupter&apos; (Henras <br /> plasmid (pJPG203), thermosensitive gar1-10 allele CEN,            et al., 1998). To 30 000 g supernatants broken cells, 0.1 ml IgG&#177; <br /> LYS2 plasmid (pJPG225), UV mutagenized. Among 17 300 screened              Sepharose beads added, suspensions incubated 1 h <br /> colonies, failed form white sectors YPsec did grow      6&#176;C rocking table. The IgG&#177;Sepharose beads washed <br /> 5-FOA medium.                                                                  repeatedly buffer, bound RNAs extracted as <br />                                                                                described previously (Ganot et al., 1997). <br /> Cloning sequencing RRP10 rrp10-1 <br /> Strain YO89-13, derivative YO89, transformed multi-            Western blotting <br /> copy yeast genomic library cloned pFL46S (2mS, LEU2) (Bonneaud              Proteins total extracts separated SDS&#177;PAGE. Fractions <br /> et al., 1991), leu+ transformants replica plated YPSec        sucrose gradients contained ribosomal subunits, from <br /> identify sectoring colonies, streaked 5-FOA-                glycerol gradients, trichloroacetic acid precipitated protein <br /> containing plates. Cloning RRP10 ORFs pEV6 (2mS, LEU2,                 content analysed SDS&#177;PAGE western blotting. Samples were <br /> insert) YO89 genomic DNA leading plasmids pEV21 (CEN               transferred nitrocellulose membranes (Hybond C-super; Amersham/ <br /> ARS LEU2 PROTA::RRP10) pEV22 (CEN ARS LEU2                                 Pharmacia). ProtA-Rrp10p detected using rabbit HRP-conjugated <br /> PROTA::rrp10-1), respectively, described Supplementary data.         anti-protein A antibody (PAP; Dako, SA) diluted 1/5000, Gar1p using a <br />                                                                                rabbit polyclonal anti-Gar1 antiboby (Girard et al., 1992) diluted to <br /> RNA analysis                                                                   1/200; ribosomal proteins rpL3, rpL32 rpS4 detected a <br /> RNA isolation northern hybridizations carried previously       mouse monoclonal anti-rpL3 antiboby diluted 1/5000, rabbit <br /> described (Henras et al., 1998). The following oligonucleotides used      polyclonal anti-rpL32 antibody cross reacting rpS4 (kindly provided <br /> for northern hybridizations (see Figure 1): 5&#162;-CATGGCTTAATC-                   J.Warner) diluted 1/2000, followed ECL detection (Amersham/ <br /> TTTGAGAC-3&#162; (probe 1); 5&#162;-TTAAGCGCAGGCCCGGCTGG-3&#162;                              Pharmacia). <br /> (probe 2); 5&#162;-GATTGCTCGAATGCCCAAAG-3&#162; (probe 3); 5&#162;-TGC- <br /> GTTCAAAGATTCGATG-3&#162; (probe 4); 5&#162;-GGCCAGCAATTTCAAG-                            Supplementary data <br /> TTA-3&#162; (probe 5); 5&#162;-CTCACGACGGTCTAAACCC-3&#162; (probe 6). For                     Supplementary data paper available The EMBO Journal <br /> analysis fractions glycerol gradients                   Online. <br /> immunoprecipitated RNAs 20S pre-rRNA probe used prepared <br /> by multiprime labelling (Amersham) using D-A2 DNA fragment as <br /> template.                                                                      Acknowledgements <br />                                                                                We grateful J.Warner, Ed Hurt D.Lafontaine providing us <br /> Pulse&#177;chase labelling pre-rRNA                                              antibodies, plasmids yeast strains. We indebted to <br /> Metabolic labelling RNA performed described previously               C.Normand carried sl screen, R.Jahangir who <br /> (Bousquet-Antonelli et al., 2000). Strain YO296 (GAL::PROTA-RRP10)             participated 20S situ localization. We thank members the <br /> was grown 30&#176;C YPG medium, shifted YPD medium, grown               Ferrer laboratory helpful discussions Y.Henry critical reading <br /> this medium 14 h grown additional 9 h YNB Glu           manuscript. We like thank Y.de Pre&#194;val (LBME, Toulouse, <br /> medium methionine (GAL::RRP10 Glu; Figure 3), grown              France) synthesis oligonucleotides, D.<span id='am-164' about='obo:NCBITaxon_117093' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_117077 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_50704 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_50674 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_50671'><span id='am-165' property="rdfs:label" content="Villa" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-167'  ></span><span id='am-168' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Villa</span> photographs <br /> 30&#176;C YPG medium shifted YNB Gal medium                       A.Rivals technical support. Immunodetection electron <br /> methionine grown 9 h medium (GAL::RRP10 Gal;                   microscopy performed laboratory Professor N.Gas <br /> Figure 3). Labelling, chase processing samples carried     (LBME, Toulouse, France). This work supported CNRS, the <br /> as previously described (Bousquet-Antonelli et al., 2000).                     Universite&#194; Paul Sabatier grants Association pour la <br />                                                                                Recherche sur le <span id='am-88' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-90'  ></span><span id='am-91' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Cancer</span> (ARC) P.-E.G., Programme de <br /> Immunoelectron microscopy                                                      Recherche Fondamentale en Microbiologie et Maladies Infectieuses et <br /> The ProtA-Rrp10p Gar1p-ProtA expressing strains used were,                 Parasitaires du Ministe&#65533;?re l&apos;Education Nationale l&apos;Enseignement <br /> respectively, strain YO296 (GAL::PROTA-RRP10) grown galactose-              Supe&#194;rieur et la Recherche (MENRT). E.V. supported grants <br /> containing medium, strain YO24 transformed pMCGZZ1 (CEN,              MENRT, C.B.-A. supported grants MENRT <br /> TRP1, GAR1-PROTA) (Henras et al., 1998) grown glucose-containing            ARC. <br /> medium. Cell &#174;xation, cutting mounting sections nickel grids <br /> were performed described Leger-Silvestre et al. (1999). Grids were <br /> incubated 2 h room temperature anti-protein A antibodies <br /> (Sigma) diluted 1/1000 phosphate saline buffer containing 2% <br />                                                                                References <br /> bovine serum albumin. The sections washed buffer,         Allmang,C., Mitchell,P., Petfalski,E. Tollervey,D. (2000) <br /> incubated 1 h colloidal gold-conjugated protein A (British              Degradation ribosomal RNA precursors exosome. Nucleic <br /> Biolcell International) diluted 1/20. Sections contrasted 5%        Acids Res., 28, 1684&#177;1691. <br /> aqueous uranyl acetate imaged JEOL-1200 EX electron                   Anaya,P., Evans,S.C., Dai,C., Lozano,G. May,G.S. (1998) Isolation <br /> microscope operating 80 kV.                                                   <span id='am-131' about='obo:NCBITaxon_162425' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_5042 obo:NCBITaxon_5052 obo:NCBITaxon_33154 obo:NCBITaxon_147545 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_1131492 obo:NCBITaxon_451871 obo:NCBITaxon_2759'><span id='am-132' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasRelatedSynonym" content="Aspergilus nidulans" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-135' property="oboInOwl:hasRelatedSynonym" content="A. nidulans" datatype="xsd:string"></span><span id='am-136' property="oboInOwl:hasRelatedSynonym" content="Aspergillus nidulellus" datatype="xsd:string"></span><span id='am-137'  ></span><span id='am-138' property="oboInOwl:hasAlternativeId" content="NCBITaxon:5072" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasRelatedSynonym" content="Emericella nidulans" datatype="xsd:string"></span><span id='am-140' property="rdfs:label" content="Aspergillus nidulans" datatype="xsd:string"></span>Aspergillus nidulans</span> sudD gene human homologue. <br />                                                                                  Gene, 211, 323&#177;329. <br /> In situ hybridization                                                          Angermayr,M. Bandlow,W. (1997) The type basal promoter <br /> To obtain probe complementary 5&#162; region ITS1, D-A2           determines regulated constitutive mode transcription the <br /> rDNA fragment PCR ampli&#174;ed subcloned pGEM4                          common control region yeast gene pair GCY1/RIO1. J. Biol. <br /> (Promega). The resulting plasmid linearized XbaI restriction            Chem., 272, 31630&#177;31635. <br /> site site D, used multiple cycles DNA synthesis             Bennetzen,J.L. Hall,B.D. (1982) Codon selection yeast. J. Biol. <br /> in thermocycler using Goldstar DNA polymerase, primer                          Chem., 257, 3026&#177;3031. <br /> 5&#162;-CTCCACAGTGTGTTGTATTG-3&#162;, encompasses site A2,                     Berges,T., Petfalski,E., Tollervey,D. Hurt,E.C. (1994) Synthetic <br /> a mix dNTPs containing dUTP-biotin. The single-stranded DNA probe             lethality &#174;brillarin identi&#174;es NOP77p, nucleolar protein <br /> obtained puri&#174;ed free dUTP-biotin precipitation used         required pre-rRNA processing modi&#174;cation. EMBO J., 13, <br /> in situ hybridization ultra-thin sections described previously (Leger-     3136&#177;3148. <br /> Silvestre et al., 1999).                                                       Bonneaud,N., Ozier-Kalogeropoulos,O., Li,G.Y., Labouesse,M., <br />                                                                                  Minvielle-Sebastia,L. Lacroute,F. (1991) A family low and <br /> Glycerol gradients protein RNA analyses                           high copy replicative, integrative single-stranded S. cerevisiae/ <br /> fractions                                                                        E. coli shuttle vectors. Yeast, 7, 609&#177;615. <br /> Sedimentation pro&#174;les glycerol gradients obtained described         Borovjagin,A.V. Gerbi,S.A. (1999) U3 small nucleolar RNA is <br /> by Bousquet-Antonelli et al. (2000).                                             essential cleavage sites 1, 2 3 pre-rRNA determines <br />  <br />  <br /> 4212 <br />                                                                                        Site D cleavage 20S pre-rRNA requires Rrp10p <br />  <br />    rRNA processing pathway taken Xenopus oocytes. J. Mol.          characterization small nucleolar ribonucleoprotein particle <br />    Biol., 286, 1347&#177;1363.                                                      snR30 Saccharomyces cerevisiae. J. Biol. Chem., 270, <br /> Bousquet-Antonelli,C., Henry,Y., Gelugne J.P., Caizergues-Ferrer,M.            11549&#177;11554. <br />    Kiss,T. (1997) A small nucleolar RNP protein required          Madden,K. Snyder,M. (1998) Cell polarity morphogenesis in <br />    pseudouridylation eukaryotic ribosomal RNAs. EMBO J., 16,                budding yeast. Annu. Rev. Microbiol., 52, 687&#177;744. <br />    4770&#177;4776.                                                                Mishra,R.K. Eliceiri,G.L. (1997) Three small nucleolar RNAs that <br /> Bousquet-Antonelli,C., Vanrobays,E., Gelugne,J.P., Caizergues-                 involved ribosomal RNA precursor processing. Proc. Natl Acad. <br />    Ferrer,M. Henry,Y. (2000) Rrp8p yeast nucleolar protein            Sci. USA, 94, 4972&#177;4977. <br />    functionally linked Gar1p involved pre-rRNA cleavage         Mizuta,K. Warner,J.R. (1994) Continued functioning the <br />    site A2. RNA, 6, 826&#177;843.                                                   secretory pathway essential ribosome synthesis. Mol. Cell. <br /> Carter,C.J. Cannon,M. (1980) Maturation ribosomal precursor             Biol., 14, 2493&#177;2502. <br />    RNA Saccharomyces cerevisiae. A mutant defect       Moy,T.I. Silver,P.A. (1999) Nuclear export small ribosomal <br />    transport terminal processing 20 S species. J. Mol. Biol.,       subunit requires ran-GTPase cycle certain nucleoporins. Genes <br />    143, 179&#177;199.                                                               Dev., 13, 2118&#177;2133. <br /> Carter,C.J., Cannon,M. Jimenez,A. (1980) A trichodermin-resistant        Petfalski,E., Dandekar,T., Henry,Y. Tollervey,D. (1998) Processing <br />    mutant Saccharomyces cerevisiae abnormal distribution         precursors small nucleolar RNAs rRNAs requires <br />    native ribosomal subunits. Eur. J. Biochem., 107, 173&#177;183.                  common components. Mol. Cell. Biol., 18, 1181&#177;1189. <br /> Chen,C.Y., Ingram,M.F., Rosal,P.H. Graham,T.R. (1999) Role           Planta,R.J. Mager,W.H. (1998) The list cytoplasmic ribosomal <br />    Drs2p, P-type ATPase potential aminophospholipid translocase,         proteins Saccharomyces cerevisiae. Yeast, 14, 471&#177;477. <br />    yeast late Golgi function. J. Cell Biol., 147, 1223&#177;1236.              Ripmaster,T.L., Vaughn,G.P. Woolford,J.L.,Jr (1993) DRS1 to <br /> Coghlan,A. Wolfe,K.H. (2000) Relationship codon bias mRNA            DRS7, novel genes required ribosome assembly function in <br />    concentration protein length Saccharomyces cerevisiae. Yeast,        Saccharomyces cerevisiae. Mol. Cell. Biol., 13, 7901&#177;7912. <br />    16, 1131&#177;1145.                                                            Shumard,C.M., Torres,C. Eichler,D.C. (1990) In vitro processing at <br /> Doye,V. Hurt,E.C. (1995) Genetic approaches nuclear pore                3&#162;-terminal region pre-18S rRNA nucleolar <br />    structure function [published erratum appears Tren <br /> </body></html>