<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>KREPA4, RNA binding protein essential editosome <br /> integrity survival <span id='am-20' about='obo:NCBITaxon_5690' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33682 obo:NCBITaxon_5654 obo:NCBITaxon_2759 obo:NCBITaxon_5653'><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-23'  ></span><span id='am-24' property="rdfs:label" content="Trypanosoma" datatype="xsd:string"></span>Trypanosoma</span> brucei <br />  <br />  <br /> REZA SALAVATI,1,2,3 NANCY LEWIS ERNST,1 JEFF O&#8217;REAR,1 TROY GILLIAM,1 SALVADOR TARUN JR.,1 <br /> and KENNETH STUART1,2 <br /> 1 <br />  Seattle Biomedical Research Institute, Seattle, Washington 98109, USA <br /> 2 <br />  Department Pathobiology, University Washington, Seattle, Washington 98195, USA <br />  <br />  <br />  <br />  <br /> ABSTRACT <br /> The 20S editosome, multiprotein complex, catalyzes editing mitochondrial mRNAs trypanosomatids by <br /> uridylate insertion deletion. RNAi mediated inactivation expression KREPA4 (previously TbMP24), component the <br /> 20S editosome, procyclic form <span id='am-25' about='obo:NCBITaxon_5691' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_39700 obo:NCBITaxon_33682 obo:NCBITaxon_5654 obo:NCBITaxon_5690 obo:NCBITaxon_2759 obo:NCBITaxon_5653'><span id='am-26' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-27'  ></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Trypanosoma (Trypanozoon) brucei" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Trypanosoma brucei subgroup" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="Trypanosoma brucei" datatype="xsd:string"></span>Trypanosoma brucei</span> resulted inhibition cell growth, loss RNA editing, disap- <br /> pearance 20S editosomes. Levels MRP1 REAP-1 proteins, roles editing editosome <br /> components, unaffected. Tagged KREPA4 protein incorporated 20S editosomes vivo preference either <br /> insertion deletion subcomplexes. Consistent S1-like motif, recombinant KREPA4 protein binds synthetic gRNA with <br /> a preference 39 oligo (U) tail. These <span id='am-85' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-88'  ></span><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> suggest KREPA4 RNA binding protein specific the <br /> gRNA Utail important 20S editosome stability. <br /> Keywords: 20S editosome; RNA binding; RNA editing; editosome integrity <br />  <br />  <br />  <br /> INTRODUCTION                                                                   et al. 2001; Cruz-Reyes et al. 2002; Drozdz et al. 2002; <br />                                                                                Schnaufer et al. 2003), KRET2 mRNA terminal uridylyl <br /> Mitochondrial (mt) mRNAs trypanosomes undergo <br />                                                                                transferase (TUTase) (Ernst et al. 2003), KREX1 KREX2 <br /> RNA editing, inserts deletes uridylates (Us) to <br />                                                                                U-specific exonucleases (exoUases) (Kang et al. 2005; <br /> produce mature functional mRNAs (Madison-Antenucci <br />                                                                                N.L. Ernst, B. Panicucci, J. Carnes, K. Stuart, unpubl.), <br /> et al. 2002; Simpson et al. 2004; Stuart et al. 2005). The <br />                                                                                KREN1 KREN2 deletion insertion endonu- <br /> number Us inserted deleted sites of <br />                                                                                cleases, respectively (Carnes et al. 2005; Trotter et al. 2005), <br /> their insertion deletion specified guide RNAs <br />                                                                                KREH1 RNA helicase (Missel et al. 1997). The six <br /> (gRNAs). RNA editing occurs multiple cycles key <br />                                                                                20S editosome KREPA proteins C-terminal OB- <br /> catalytic steps, endonucleolytic cleavage pre-mRNA <br />                                                                                fold-like motifs, KREPB proteins have <br /> followed addition removal Us at <br />                                                                                U1-like zinc finger motif (Schnaufer et al. 2003; Worthey <br /> the 39 end 59 cleavage fragment insertion deletion <br />                                                                                et al. 2003). Their potential functions described below. <br /> editing, respectively, ligation processed 59 <br />                                                                                Other complexes proteins function RNA editing and <br /> RNA fragment 39 RNA fragment. Kinetoplastid <br />                                                                                interact 20S editosomes. These include the <br /> RNA editing (KRE) proteins occur pairs sets related <br />                                                                                KRET1 complex adds gRNA 39oligo(U) tails, the <br /> proteins multiple complexes. The enzymes editing <br />                                                                                MRP1 proteins anneal gRNA mRNA, <br /> are contained multiprotein complexes, 20S edito- <br />                                                                                complex RBP16 affects abundance of <br /> somes, accurately edit single sites vitro. 20S <br />                                                                                edited unedited mRNAs (Koller et al. 1997; Pelletier <br /> editosomes contain 20 proteins specific <br />                                                                                et al. 2000; Mu&#168;ller et al. 2001; Aphasizhev et al. 2003b; <br /> functions elucidated. KREL1 KREL2 are <br />                                                                                Pelletier Read 2003; Vondruskova et al. 2005). Addi- <br /> deletion insertion mRNA ligases, respectively (Huang <br />                                                                                tional proteins TbRGG1 REAP1 function <br />     3                                                                          RNA editing regulation (Madison-Antenucci et al. <br />     Present address: McGill University, Institute Parasitology, Mon- <br /> treal, Quebec H9X 3V9, Canada.                                                 1998; Vanhamme et al. 1998; Madison-Antenucci and <br />    Reprint requests to: Kenneth Stuart, Seattle Biomedical Research            Hajduk 2001). <br /> Institute, 307 Westlake Ave. N, Suite 500, Seattle, WA 98109-5219, USA;           The 20S editosomes physically functionally <br /> e-mail: ken.stuart@sbri.org; fax: (206) 256-7229. <br />    Article published online ahead print. Article publication date   organized subcomplexes. Heterotrimeric KRET2, <br /> at http://www.rnajournal.org/cgi/doi/10.1261/rna.2244106.                      KREL2, KREPA1 insertion subcomplexes KREX2, <br />  <br /> RNA (2006), 12:819&#8211;831. Published Cold Spring Harbor Laboratory Press. Copyright &quot; 2006 RNA Society.                                   819 <br />  Salavati et al. <br />  <br />  <br />  <br /> KREL1, KREPA2 deletion subcomplexes evident              KREPA4 N-terminal region low compositional <br /> from variety vitro vivo studies (Cruz-Reyes       complexity midregion potential S1 motif. <br /> et al. 2002; Huang et al. 2002; Aphasizhev et al. 2003a; Ernst   Recombinant KREPA4 protein binds synthetic gRNA that <br /> et al. 2003; Schnaufer et al. 2003). KREPA1 KREPA2,          contains U tail specificity oligo (U). These data <br /> which C2H2 zinc fingers, suggested       suggest KREPA4 function protein&#8211;protein <br /> to coordinate <span id='am-52' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-53' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> catalytic steps occur       interactions interact RNA, potentially 39 U <br /> during insertion deletion editing, respectively (Schnaufer   tail gRNA. <br /> et al. 2003; Law et al. 2005). A recent report suggests that <br /> the 20S editosome protein KREPA3 contributes both <br /> endo- exoribonuclease activities RNA editing (Brecht <br />                                                                  RESULTS <br /> et al. 2005). KREPA4, focus article, was <br /> previously designated TbMP24 based preprocessed <br />                                                                  KREPA4 essential trypanosome growth <br /> molecular weight 23.7 kDa identified mass <br />                                                                  RNA editing <br /> spectrometric analyses editosomes biochemi- <br /> cally purified immunoprecipitated monoclonal             The role KREPA4 editing explored RNAi <br /> antibody (MAb) specific editosome protein KREPA2             mediated repression (knockdown) gene expression by <br /> (Panigrahi et al. 2003). KREPA4 belongs <span id='am-6' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span>       generating double-stranded RNA (dsRNA) the <br /> proteins (KREPA1&#8211;6) conserved C-terminal             entire KREPA4 coding region. The dsRNA expressed <br /> region OB-fold-like domain, largest         using construct opposing tetracycline (tet)-inducible <br /> which C2H2 zinc fingers (Panigrahi et al. 2001b;       T7 RNA polymerase promoters stably integrating it <br /> Schnaufer et al. 2003; Worthey et al. 2003). The KREPA4          nontranscribed spacer rDNA locus (Wang <br /> gene immediately upstream KREPA6 gene           et al. 2000). <span id='am-44' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-45'  ></span><span id='am-46' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> procyclic form Trypanosoma <br /> three trypanosomatid genomes, suggesting       brucei (PF) strain 29&#8211;13 (Wirtz et al. 1999), contains <br /> result gene duplication event (Worthey et al. 2003).        T7 RNA polymerase tet repressor genes dsRNA <br /> The functions proteins unknown. KREPB1&#8211;8            induced addition tet. Knockdown KREPA4 <br /> all U1-like zinc finger motifs KREPB1&#8211;5        mRNA expression growth inhibition observed <br /> have RNase III-like motifs, motif KREPB4        expression dsRNA (Fig. 1A). Constructs with <br /> and -5 diverges substantially canonical RNase III       59 315 bp including 60 bp 59 UTR 39 327 bp <br /> motif (Panigrahi et al. 2003; Worthey et al. 2003). KREPB1&#8211;      KREPA4 did inhibit growth despite reduction <br /> 3 putative double-stranded RNA binding motifs          mRNA abundance (data shown). Northern <br /> and endonuclease activity (Worthey et al. 2003;             analysis RNA full-length KREPA4 RNAi cell <br /> Carnes et al. 2005; Trotter et al. 2005). Otherwise, func-   lines oligonucleotide probe complementary <br /> tions KREPB proteins unknown, KREPB5      KREPA4 59 UTR region discriminate the <br /> is essential editosome integrity (Wang et al. 2003).      dsRNA target mRNA showed KREPA4 <br />    Some information emerging proteins            mRNA reduced cells full-length RNAi <br /> essential editosome structural integrity. RNAi inactiva-     induced, relative uninduced cell lines (days 2 and <br /> tion KREPA1 expression shown preferen-            4 &quot;RNAi lanes). <br /> tially inhibit insertion editing result loss          Expression full-length KREPA4 dsRNA dramati- <br /> adenylatable editing ligase KREL2 shift editosomes        cally inhibited cell proliferation (Fig. 1A). This with <br /> to z15S (Drozdz et al. 2002; O&#8217;Hearn et al. 2003).               mRNA knockdown suggests KREPA4 protein <br /> Similarly, inactivation KREPA2 leads substantial           expression repressed (down-regulated) induction <br /> disruption 20S editosome loss editing ligase       full-length dsRNA, assess this <br /> KREL1 (Huang et al. 2002). The effect down-regulation         directly lack antibody specific for <br /> of KREX1 expression results progressive decrease     KREPA4. Repression KREPA4 protein expression also <br /> S value 20S editosome time preferential          suggested absence editosome proteins as <br /> reduction vitro precleaved U-deletion editing (Kang        described below. Cell growth essentially ceased 6 d of <br /> et al. 2005).                                                    growth presence tet resumed day 19, <br />    We report genetic functional studies       possibly loss RNAi knockdown (Chen et al. 2003). <br /> that KREPA4 essential RNA editing 20S edito-          RNA editing vivo significantly reduced after <br /> some structural integrity. Tagged KREPA4 cosediments          KREPA4 repression RNAi induction 6 d shown <br /> z20S glycerol gradients key enzyme              real-time PCR analysis (Fig. 1B). The levels edited A6, <br /> activities catalyze vitro editing. RNAi-mediated         RPS12, COIII, CYb, COII mRNAs reduced by <br /> inactivation KREPA4 vivo results editosome              z60%&#8211;80% levels pre-edited transcripts <br /> disintegration, consequent loss editing cell         essentially unchanged somewhat increased (CYb <br /> growth. Comparative silico analyses predict              COII). The levels COI ND4 mRNAs not <br />  <br /> 820      RNA, Vol. 12, No. 5 <br />                                                                                                      KREPA4 integrity editosome <br />  <br />  <br />  <br />  <br /> FIGURE 1. Down-regulation KREPA4 expression affects trypanosome growth edited RNA abundance. (A) Growth curve cells which <br /> KREPA4 expressed absence RNAi induction (open diamonds) repressed RNAi induction 1 mg/mL tet (solid diamonds). <br /> The cumulative cell numbers reflect normalization cell densities dilution factor. (Inset) Northern blot total RNA hybridized <br /> with oligonucleotide probe complementary 59 UTR region KREPA4. RNA collected indicated days induced (+RNAi) <br /> or uninduced cells (&quot;RNAi). Ribosomal RNA shown loading control. (B) KREPA4 repression reduces RNA editing vivo. Real-time PCR <br /> analysis RNA KREPA4 RNAi cells absence presence RNAi induction 6 d. The abundance pre-edited (pre, white bars) <br /> and edited (edit, black bars) mRNAs repressed cells shown relative cells KREPA4 expressed. The relative <br /> abundance edited COI ND4 mRNAs (medium gray bars) shown. The RNA levels normalized 18S rRNA. The thick <br /> gray line 1 indicates relative change mRNA level, line representing increase decrease mRNA <br /> levels, respectively. Error bars represent standard deviation replicates. <br />  <br />  <br />                                                                                                                  www.rnajournal.org         821 <br />  Salavati et al. <br />  <br />  <br />  <br /> undergo editing unaffected. The level edited                  role KREPA4 restricted z20S <br /> MURFII mRNA did decrease, possibly reflecting greater             editosome. <br /> stability. Northern analysis showed KREPA4 mRNA                     Repression KREPA4 expression resulted the <br /> levels reduced RNAi induction real-                loss vitro RNA editing activities normally <br /> time PCR using primer set 39 UTR did               sediment z20S. Precleaved insertion editing, which <br /> a corresponding reduction (data shown)            entails linked TUTase ligase activities, precleaved <br /> this region RNA targeted RNAi. In               deletion editing, entails linked exoUase ligase <br /> addition, real-time PCR showed mRNA level                 activities, greatly diminished glycerol gradient <br /> KREPA6, editosome protein closely related                 20S fractions (centered fraction 13) whole-cell <br /> KREPA4, changed following KREPA4 RNAi knock-                  lysates 4 d RNAi induction (Fig. 3A,B, lanes labeled <br /> down (data shown). Thus, knockdown KREPA4                      KREPA4 repressed). Western analyses activity profiles <br /> expression substantially inhibited growth editing            identical relative abundance of <br /> in vivo, confirming role editing.                              non-editosome proteins complex mixtures well <br />                                                                       proteins affect activity (e.g., ligases) differen- <br />                                                                       tially affect Western versus activity signals. Nevertheless, <br /> Inactivation KREPA4 expression results loss <br />                                                                       traces ligated products precleaved insertion <br /> of ;20S editosomes <br />                                                                       deletion substrates detected, substantial <br /> Western analyses glycerol gradient fractions whole-           exonuclease activity. The reflect residual <br /> cell lysates MAbs specific editosome proteins           editosome ligase activity (see above), latter <br /> revealed dramatic reduction editosome proteins             probably reflects contaminating nucleases whole <br /> 4 d growth presence tet. The KREPA1, KREPA2,             cell lysates U specific result primarily in <br /> KREPA3, KREPA6, KREL1 proteins cosediment                 &quot;3U products shown previously (Wang et al. 2003). <br /> z20S cells grown absence tet dramatically           Thus, partial loss catalytic activities associated <br /> reduced abundance following re- <br /> pression KREPA4 expression (Fig. 2A). <br /> Some KREL1, KREPA2, lesser <br /> extent KREPA1 KREPA3 remained <br /> but shifted higher region the <br /> gradient. Thus, loss editing is <br /> paralleled loss editosomal <br /> proteins z20S editosomes. <br /> These data indicate KREPA4 is <br /> important editosome structural in- <br /> tegrity. Western analyses total cell <br /> lysates studies not <br /> sufficiently sensitive directly deter- <br /> mine overall abundance edi- <br /> tosome proteins (data shown). <br /> Western analyses glycerol gradient <br /> fractions whole-cell lysates with <br /> MAbs specific RNA binding <br /> proteins involved editing <br /> but integral components the <br /> editosome including MRP1 (previously <br /> gBP21) REAP-1, showed differ- <br /> ence protein levels sedimentation <br /> between cells KREPA4 was <br /> expressed repressed (Fig. 2B,C). The <br /> slightly greater signal intensity of <br /> samples KREPA4 repressed FIGURE 2. Loss 20S editosome down-regulation KREPA4 expression. Western <br /> appears to loading analysis glycerol gradient fractions partially purified editosome fraction (control, c) with <br /> protein gradients, indicated KREPA1, -2, -3, -6, KREL1 (A), MRP1     (B), REAP1 (C), PGK (D) antibodies. Lysates of <br />                                             equivalent numbers cells (3 3 109) protein KREPA4 expressed (exp.) or <br /> by greater signal PGK loading <br />                                             repressed (rep.) RNAi induction 4 d subjected glycerol gradient fractionation <br /> control, protein involved followed Western blot analysis. The PGK isoforms A, B, C used loading <br /> in editing (Fig. 2D). These data indicate control. <br />  <br /> 822      RNA, Vol. 12, No. 5 <br />                                                                                                 KREPA4 integrity editosome <br />  <br />  <br />  <br />                                                                                                   partner(s). To distinguish KREPA3 <br />                                                                                                   signal tagged KREPA4, glycerol <br />                                                                                                   gradient fractions probed a <br />                                                                                                   mixture KREPA1 KREL1 MAbs. <br />                                                                                                   Tagged KREPA4 detected <br />                                                                                                   cells induced tet <br />                                                                                                   (Fig. 4C,D). Overall data suggest <br />                                                                                                   tagged KREPA4 in <br />                                                                                                   z20S editosomes, implied <br />                                                                                                   below. <br />                                                                                                      Tagged editosomes purified <br />                                                                                                   induced KREPA4 TAP-tagged <br />                                                                                                   cells sequential IgG-sepharose affin- <br />                                                                                                   ity chromatography followed TEV <br />                                                                                                   protease elution, glycerol gradient sedi- <br />                                                                                                   mentation, calmodulin resin <br />                                                                                                   affinity chromatography described <br />                                                                                                   previously (Schnaufer et al. 2003). The <br />                                                                                                   TEV eluate IgG sepharose <br />                                                                                                   contained complexes KREPA1, <br />                                                                                                   KREPA2, KREPA3, KREL1 pro- <br /> FIGURE 3. Reduction RNA editing catalytic activities vitro KREPA4 repression. teins cosedimented gradients <br /> Lysates cells KREPA4 expressed repressed RNAi induction 4 d (Fig. 4E). The reduced signal the <br /> fractionated glycerol gradients 5 h fractions assayed editing activities. KREPA3 region compared Figure 4B <br /> (A) Precleaved U insertion RNA editing. Input RNA added Us (+2U), edited RNA <br /> with insertion Us (E), ligated product 59 39 RNA fragments removal Protein A <br /> any added Us (L) indicated. (B) Precleaved U deletion editing. Input RNA Us portion tag TEV protease <br /> removed (&quot;4U), RNA edited deletion 4 Us (E), ligated RNAs Us digestion, implying integration <br /> removed (L) indicated.                                                                        tagged protein 20S edito- <br />                                                                                                   somes. This suggested the <br /> with RNA editing confirms involvement KREPA4                     presence KREPA1, KREPA2, KREPA3, KREL1 <br /> editing.                                                                   proteins pooled fractions 11&#8211;14 further <br />                                                                            purified calmodulin affinity chromatography (Fig. 4F). <br />                                                                            Some KREL1 KREPA2 detected MAbs <br /> Physical functional association KREPA4 with <br />                                                                            adenylation fraction 5 (Fig. 4E). None the <br /> the editosome proteins <br />                                                                            KREPA1, KREPA2, KREPA3, KREL1 proteins were <br /> The structural association KREPA4 protein                    pooled fractions 3&#8211;6 calmodulin affinity <br /> editosome components assessed using cell line                        chromatography (Fig. 4F). Perhaps abundance these <br /> containing tet-inducible version KREPA4 gene                      proteins low detection (possibly pro- <br /> fused C-terminal TAP tag (Rigaut et al. 1999). Whole-                 teolysis) loosely associated tagged KREPA4 <br /> cell lysates uninduced induced cells                       fractions did remain associated the <br /> fractionated glycerol gradient fractionation 5 h                    tagged KREPA4 final purification step. Taken <br /> (Fig. 4A&#8211;D). Western analysis glycerol gradient                     results indicate KREPA4 integral <br /> fractions mixture MAbs specific                    component 20S editosomes does appear <br /> for 20S editosome proteins revealed increased signal                stable component smaller subcomplexes. <br /> position expected tagged KREPA4 20S (centered                      We assayed pooled glycerol gradient fractions (frac- <br /> at fraction 11) 20S region gradient                tions 3&#8211;6 11&#8211;14) calmodulin affinity <br /> induced compared uninduced cells (Fig. 4A,B). This                      purification precleaved insertion deletion editing. <br /> increased signal likely tagged KREPA4,                 Pooled fractions 11&#8211;14 catalyzed precleaved insertion <br /> comigrates KREPA3, primary                         deletion subjected cal- <br /> secondary antibodies bind Protein A moiety                      modulin purification contain exoUase, TUTase, <br /> the tagged KREPA4 protein. The stronger signal frac-                    ligase editing activities (Fig. 4G). These activities were <br /> tions 3 5 induced cells MAbs implies                detected pooled fractions 3&#8211;6 prior calmodulin <br /> tagged KREPA4 present gradient                     purification, consistent lack detectable <br /> some tagged protein near top,                 editosome proteins fractions (Fig. 4F). On other <br /> have led degradation absence interacting              hand, pooled fractions 11&#8211;14 contain exoUase/TUTase <br />  <br />                                                                                                             www.rnajournal.org        823 <br />  Salavati et al. <br />  <br />  <br />  <br />  <br /> FIGURE 4. Characterization proteins associated TAP-tagged KREPA4 vivo. Whole-cell extracts equivalent numbers cells (3 3 109) <br /> and protein expression TAP-tagged KREPA4 induced (A,C) induced 1 mg/mL tet 2 d (B,D) subjected to <br /> glycerol gradient fractionation followed Western blot analysis antibodies indicated. (E) Proteins associated tagged KREPA4 in <br /> induced cells purified IgG affinity chromatography eluted TEV protease. The TEV eluate separated 10%&#8211;30% <br /> glycerol gradient subjected Western analysis. (Lower panel) The pooled glycerol gradient fractions (3&#8211;4 11&#8211;14) analyzed the <br /> presence editing ligases adenylation [a-32P]ATP. The RNA editing ligases indicated KREL1 KREL2. (+) Partially purified <br /> editosome fraction used Western adenylation assays (Panigrahi et al. 2001a). (F) Pooled glycerol gradient fractions 3&#8211;6 11&#8211;14 were <br /> further purified calmodulin affinity eluates subjected Western analysis MAbs indicated. (G) In vitro editing assays <br /> of second eluate calmodulin purification step (cal) precleaved insertion (left panel) precleaved deletion (right panel) editing <br /> assays. The negative control (&quot;) contains extract positive control (+) partially purified mitochondrial extract. Pooled glycerol <br /> gradient fractions 3&#8211;6 11&#8211;14 assayed directly (GG) purification calmodulin affinity (cal). Editing intermediates end <br /> products indicated Figure 2. <br />  <br />  <br /> 824       RNA, Vol. 12, No. 5 <br />                                                                                                     KREPA4 integrity editosome <br />  <br />  <br />  <br /> and ligase editing activities. Overall, results demon-              RNA binding protein. Low complexity regions proteins <br /> strate association KREPA4 20S editosomes.                     typically lack catalytic activities; however, been <br />                                                                           important biological functions, often <br />                                                                           associated protein&#8211;protein interactions (Wootton and <br /> Compositional analysis structural prediction <br />                                                                           Federhen 1996). This sequence analysis suggests that <br /> of KREPA4 <br />                                                                           KREPA4 capacity interact RNA <br /> Structural prediction compositional analysis                       proteins editosome. <br /> KREPA4 identified amphipathic N terminus                        The S1 motif type OB fold, in <br /> expected mt protein (Fig. 5A, underlined; data                  large number RNA-associated proteins. Motif scanning <br /> shown) OB fold previously reported (Worthey                       T. brucei KREPA4 protein (TbKREPA4) using Profile- <br /> et al. 2003) putative S1 motif OB fold              Scan (Pagni et al. 2004) detected weak similarity match to <br /> (amino acids 75&#8211;148, highlighted yellow, below)                S1 motif (score 6.7 10 highest score) <br /> and low compositional complexity regions (amino                       midregion (amino acids 75&#8211;148). However, the <br /> acids 25&#8211;36 53&#8211;67, highlighted green). In addition,                <span id='am-15' about='obo:NCBITaxon_5693' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33682 obo:NCBITaxon_5654 obo:NCBITaxon_5690 obo:NCBITaxon_2759 obo:NCBITaxon_47570 obo:NCBITaxon_5653'><span id='am-16' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-17'  ></span><span id='am-18' property="rdfs:label" content="Trypanosoma cruzi" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Trypanosoma cruzi</span> <span id='am-32' about='obo:NCBITaxon_5664' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_38581 obo:NCBITaxon_131567 obo:NCBITaxon_5658 obo:NCBITaxon_33682 obo:NCBITaxon_1286322 obo:NCBITaxon_5654 obo:NCBITaxon_2759 obo:NCBITaxon_38568 obo:NCBITaxon_5653'><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Leishmania tropica major" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Leishmania (Leishmania) major" datatype="xsd:string"></span><span id='am-36'  ></span><span id='am-37' property="rdfs:label" content="Leishmania major" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Leishmania major</span> KREPA4 ortho- <br /> KREPA4 contains 24 arginines 24 glycines                  logs (TcKREPA4 LmKREPA4) diverged and <br /> abundant amino acids (11% number amino                        appear contain S1 motif (see Discussion). A <br /> acid). The presence arginine-rich sequence                      multiple sequence alignment potential TbKREPA4 S1 <br /> a putative S1 motif strongly suggests KREPA4                   motif (residues 75&#8211;148) S1 motifs containing <br />  <br />  <br />  <br />  <br /> FIGURE 5. Amino acid sequence analysis structural model T. brucei KREPA4 S1 motif. (A) The putative 18-amino acid <br /> mitochondrial import presequence underlined. The predicated low compositional complexity regions highlighted green putative <br /> S1 motif yellow. (B) A multiple sequence alignment S1 motifs E. coli PNPase (1sro), RNase E (1smx), KREPA4. Residues 1sro <br /> and 1smx critical OB fold indicated triangles numbered E. coli 1sro. (C) Comparison 1sro structure <br /> (yellow), KREPA4 model (green) superimposed the1sro structure (1sro/KREPA4), modeled KREPA4 structure (green). The side <br /> chains <span id='am-1' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="rdfs:label" content="Glycine" datatype="xsd:string"></span>glycine</span> residues (red) important OB fold shown ball stick. <br />  <br />  <br />                                                                                                                 www.rnajournal.org         825 <br />  Salavati et al. <br />  <br />  <br />  <br /> proteins <span id='am-55' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-56' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-60'  ></span><span id='am-61' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span>Escherichia coli</span> PNPase (1sro, residues 6&#8211;76)       homodimer (Fig. 6A, lane 6). The homodimer signal, which <br /> and RNase E (1smx, residues 10&#8211;96) known              represents small portion overall KREPA4 signal, <br /> structures solved NMR X-ray crystallog-        incomplete denaturation protein <br /> raphy, respectively, shown Figure 5B. The            high concentration protein the <br /> conserved residues S1 motif glycine residues        sensitivity Western blots, corresponding <br /> (G11, G20, G31, G53; numbered E. coli 1sro)          signal absent gel stained Coomassie (see lane 5). <br /> which conserved KREPA4 (G80, G91, G120;            In addition, signal detected cells <br /> numbered T. brucei) fourth conservatively      induced (data shown). Finally, higher molec- <br /> replaced (S102). A valine occasionally replaced      ular weight band detected Western blots when <br /> isoleucine, KREPA4 (I82), contributes          larger fragment KREPA4 expressed E. coli, and <br /> hydrophobic core five-stranded antiparallel b barrel,    signal potential homodimer band corre- <br /> is conserved. S1 motifs unique turn 310   spondingly larger (data shown). <br /> <span id='am-39' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-40' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Helix" datatype="xsd:string"></span><span id='am-43'  ></span>helix</span> end strand 3            Purified KREPA4 protein assayed RNA editing- <br /> OB-fold proteins, isoleucine valine (I106       associated catalytic activities RNA binding. Enzymatic <br /> V109) form helix conserved KREPA4.       assays known catalytic activities RNA editing using <br />    The three-dimensional (3-D) structure TbKREPA4            vitro RNA editing substrates did detect the <br /> was predicted basis alignment known      major catalytic activities editing, endoribonuclease, <br /> structures S1 motifs 1sro 1smx templates.      TUTase, exonuclease, RNA ligase (data shown). <br /> This homology modeling superposition TbKREPA4           Gel mobility shift assays using radiolabeled gA6[14] <br /> and 1sro 3-D structures, average carbon distance     guide RNA oligo (U) tail performed in <br /> 1.7 A&#730;, reveals conserved structural core shared   presence increasing amounts unlabeled compet- <br /> features S1 motif. Our modeling data suggest        itors (Fig. 6B). Incubation recombinant KREPA4 with <br /> they similar structures, conserved glycines   gA6[14] RNA resulted shifted product absence <br /> distributed interior OB fold (Fig. 5C). These     competitor. Quantification gel shift signals nor- <br /> glycines probably involved structural organization       malized lanes competitor RNA was <br /> similar <span id='am-78' about='obo:NCBITaxon_superfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="superfamily" datatype="xsd:string"></span>superfamily</span> OB-fold proteins (Bycroft         absent. The oligo (U) tail region gRNA molecule <br /> et al. 1997). The TbKREPA4 model notably lacks         appears major determinant binding the <br /> fifth b-strand conserves features strands 1 4     protein. Formation shifted product specific for <br /> that contribute homodimerization S1 fold           gRNA U tail formation inhibited <br /> RNase E aspects strands 2 3 exposed       z75% addition 100-fold molar excess unlabeled <br /> residues likely RNA binding region. Based       homologous RNA heterologous Bluescript RNA <br /> this structural model TbKREPA4 predicted              (Fig. 6B). Furthermore, KREPA4 resulted gel shifts of <br /> contain S1 RNA binding motif mediate RNA            labeled gRNA U tail nonspecific manner, <br /> substrate binding specificity capacity     binding unaffected presence of <br /> interact editosome proteins bring      100-fold molar excess unlabeled homologous RNA (data <br /> RNA catalysts proximity.                           shown). Consistent data, competition a <br />                                                                 100-fold molar excess gA6[14] gRNA devoid oligo (U) <br />                                                                 tail A6 pre-mRNA mRNA inhibited product formation <br /> KREPA4 purification functional characterization <br />                                                                 z10%. However, addition equimolar amount <br /> An inducible N-terminal His6-KREPA4 fusion protein              poly (U) inhibited product formation 35% 10- <br /> lacking 40 amino acids expressed E. coli       100-fold molar excess poly (U) inhibited product <br /> and soluble (Fig. 6A). Despite attempts,    formation z80%. Radiolabeling poly U indicated <br /> unable obtain sufficient soluble full-length recombinant     size range 10&#8211;70 nt (data shown). This <br /> protein version N-terminal 18 amino         partially explain poly (U) efficient <br /> acids removed. While purified protein lacked       competitor guide RNA average oligo (U) <br /> predicted mitochondrial import signal low         tail length 10&#8211;15 nt. <br /> compositional complexity region, retained second low        In contrast, 100-fold molar excess poly (G), <br /> compositional complexity region predicted S1            poly (C), poly (A) inhibited product formation by <br /> motif. Small amounts protein expressed              maximum 23%. Thus KREPA4 specifically binds the <br /> without induction, suggesting leaky expression T7        poly (U) tail gA614 gRNA. <br /> promoter (Fig. 6A, lane 1). The protein purified its <br /> His tag near homogeneity (Fig. 6A, lanes 5,6; see <br />                                                                 DISCUSSION <br /> Materials Methods), Western analysis using an <br /> anti-His antibody detected primarily protein expected    This study shows KREPA4 essential protein <br /> size smaller apparent size           component z20S RNA editing complex T. brucei. <br />  <br /> 826      RNA, Vol. 12, No. 5 <br />                                                                                                 KREPA4 integrity editosome <br />  <br />  <br />  <br />                                                                                                   recombinant protein binds to <br />                                                                                                   synthetic gRNA, specifically gRNA <br />                                                                                                   39-(U)-tail. <br />                                                                                                       The S1 motif TbKREPA4 OB <br />                                                                                                   fold identified using Profile Scan <br />                                                                                                   protein databases. The mo- <br />                                                                                                   tif, identified E. <br />                                                                                                   coli ribosomal protein S1 (Subramanian <br />                                                                                                   1983), related cold shock <br />                                                                                                   domain (CSD), consists anti- <br />                                                                                                   parallel b-barrel b-strands. <br />                                                                                                   The S1 motif present pro- <br />                                                                                                   teins unrelated functions, all <br />                                                                                                   bind RNA (Bycroft et al. 1997). Here <br />                                                                                                   shown recombinant <br />                                                                                                   KREPA4 interacts gRNA vitro. <br />                                                                                                   We previously shown <br />                                                                                                   (Salavati et al. 2002; Aphasizhev et al. <br />                                                                                                   2003b) nuclease treatment of <br />                                                                                                   editing complexes shifts position of <br />                                                                                                   endonuclease, vitro editing, and <br />                                                                                                   RNA ligase activities. These studies <br />                                                                                                   suggested RNA degradation <br />                                                                                                   results loss editing complex com- <br />                                                                                                   ponents. However, data not <br />                                                                                                   allow conclude U <br />                                                                                                   specificity KREPA4 directed in <br />                                                                                                   vivo oligo (U) tail Us added <br />                                                                                                   mRNA deleted it. KREPA4 <br />                                                                                                   key component 20S edito- <br />                                                                                                   somes loss results their <br />                                                                                                   disintegration; KREPA4 does <br />                                                                                                   appear stable component of <br />                                                                                                   deletion insertion subcom- <br /> FIGURE 6. Expression RNA binding characteristics recombinant KREPA4. (A) <br />                                                                                                   plexes. The S1 motif polynucleotide <br /> Recombinant KREPA4 lacking 40 amino acids induced E. coli 1 mM phosphorylase stimulates exonu- <br /> IPTG time point 0. Proteins cells time 0 (lane 1), 1 h (lane 2), 2 h (lane 3), 3 h cleolytic degradation mRNA and <br /> (lane 4) post-induction separated SDS-PAGE stained Coomassie Brilliant help molecules involved in <br /> Blue. Purified recombinant KREPA4 separated SDS-PAGE stained Coomassie <br /> Brilliant Blue (lane 5). Western blot purified recombinant A4 monoclonal antibody <br />                                                                                                   RNA degradation. For instance, E. <br /> against His-tag (lane 6). The arrow indicates position recombinant KREPA4, coli, S1 stimulates activity T4 <br /> asterisk represents potential recombinant KREPA4 dimer. Size markers kilodaltons phage regB RNA-specific endonuclease <br /> shown left. (B) Recombinant KREPA4 (20 ng, 800 fmol) incubated 1 fmol (Ruckman et al. 1994). By analogy, <br /> radiolabeled gA6[14] U tail presence varying amounts competitor RNAs. I <br /> indicates radiolabeled input RNA, P, radiolabeled RNA presence KREPA4 KREPA4 interact RNA and <br /> without competitor RNA, numbers panels indicate fold excess protein(s) stimulate otherwise <br /> unlabeled competitor RNAs. RNA&#8211;protein complexes resolved electrophoresis 9% affect (their) activity editing. This <br /> nondenaturing acrylamide gels visualized PhosphorImager. The arrows indicate superficially similar roles pro- <br /> gel shifts radiolabeled RNA KREPA4, numbers panels indicate percent <br /> of shift normalized shifts absence competitor (P).                               posed KREPA1 KREPA2 in <br />                                                                                                   substrate binding coordination of <br />                                                                                                   order U addition/ligation U <br /> Regulated loss KREPA4 results dramatic reduction                     deletion/ligation, respectively (Schnaufer et al. 2003). At <br /> in growth PF trypanosomes, loss editing vivo                    present binding partners KREPA4 20S <br /> and vitro editing-associated catalytic activities,                 editosome unknown identification advance <br /> well loss editosomes. We conclude                       understanding role. <br /> KREPA4 essential stability structure                      The divergence T. cruzi L. major KREPA4 <br /> of editosome. KREPA4 contains predicted S1 motif                       ortholog (TcKREPA4 LmKREPA4) sequences from <br />  <br />                                                                                                             www.rnajournal.org        827 <br />  Salavati et al. <br />  <br />  <br />  <br /> T. brucei similar previous report TcKREPA4            (pH 8.0) added. The cleared lysate added an <br /> and LmKREPA4 proteins homology               Invitrogen ProBond Resin Column rotated 1 h 4&#176;C. Resin <br /> family OB-fold-containing proteins (Worthey et al.                column previously prepared adding 1 mL ProBond <br /> 2003). TcKREPA4 identified homology searches                  resin column, washing twice 10 mL 20 mM Tris <br />                                                                      (pH 8.0) 1 3 5 mL 20 mM Tris (pH 8.0), 300 mM NaCl, <br /> but LmKREPA4 identified syntenic location <br />                                                                      10 mM imidazole stored 4&#176;C. The column drained <br /> compared T. cruzi T. brucei. The lack apparent <br />                                                                      gravity flow washed 2 3 4 mL wash buffer (20 mM <br /> S1 motif TcKREPA4 LmKREPA4 reflect                        Tris pH 8.0, 300 mM NaCl, 20 mM imidazole). KREPA4 was <br /> development divergent novel functions T. cruzi              eluted adding 4 3 500 mL elution buffer (20 mM Tris pH <br /> and L. major compared T. brucei paralogs com-               8.0, 300 mM NaCl, 250 mM imidazole). Glycerol added to <br /> monly evolve new functions (Kondrashov et al. 2002). The             final concentration 30%, samples stored &quot;80&#176;C. <br /> divergence, including S1 motif region, may <br /> reflect differences KREPA4&#8217;s interactions RNA            KREPA4 gel shift assay <br /> protein <span id='am-49' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>, accommodate <br /> divergence editing proteins and/or RNA substrates           gA6[14] guide RNA, gA6[14] lacking oligo(U) tail, A6 pre- <br /> or possibly regulatory processes species.                edited mRNA (A6 short/TAG.1) transcripts used assay were <br />                                                                      prepared T7 polymerase (Promega) transcription PCR- <br />    Depletion component T. brucei 20S <br />                                                                      generated templates previously described (Seiwert et al. 1996), <br /> editosomes (e.g., KREL2) results cell death         90-nt pBlueScript SK+ (Stratagene) RNA generated <br /> a dramatic reduction growth rate associated              vitro transcription NotI linearized plasmid. Gel shift <br /> defects RNA editing. In cases, specific editosome            assays performed incubating 20 ng (800 fmol) purified <br /> components depleted major structural                  KREPA4 1 fmol (10K cpm) internally [a-32P]UTP-labeled <br /> consequences 20S editosome retention              gA6[14] RNA substrate (heated 95&#176;C 5 min) presence <br /> catalytic activities. However, loss KREPA4, like KREPB5,          absence unlabeled competitor buffer GS-RBB50 (20 mM <br /> has dramatic structural consequences result loss         Tris-HCl pH 7.6, 50 mM KCl, 5 mM MgCl2, 100 mg/mL BSA, <br /> of editosomes (Fig. 3). Nevertheless, levels        10% glycerol), 100 mM KCl, 20 units RNasin (Promega) in <br /> the KREPA1, -2, -3, -6, KREL1 editosome proteins            20-mL volume 30 min room temperature. In competition <br /> dramatically decreased KREPA4 deficient cells,            experiments, unlabeled gA6[14] guide RNA, gA6[14] RNA lacking <br />                                                                      oligo(U) tail, A6short/TAG.1 pre-edited RNA, pBluescript RNA, <br /> MRP1 REAP-1 proteins complexes with <br />                                                                      poly(N) competitor RNAs added 1-, 10-, 100-fold <br /> possible roles editing unaffected. These results             molar excess [a-32P]UTP-labeled gA6[14] RNA followed the <br /> suggest critical role KREPA4 20S core complex,          addition KREPA4. Samples loaded 9% native <br /> which entail editosome assembly role associated           polyacrylamide minigels (Bio-Rad) 0.53 TBE run at <br /> with dynamic events occur core complex               100 V 1 h 4&#176;C. Gels dried visualized Storm <br /> during editing T. brucei. Its presence editosome           PhosphorImager screens (Molecular Dynamics). The intensity of <br /> and dependence editosome integrity indicates           bands quantified using ImageQuant software. <br /> a likely protein interaction domain. However, KREPA4&#8217;s <br /> binding partner(s) location 20S editosome         Plasmid constructs, transfections, induction <br /> as unknown. Nevertheless, results KREPA4             RNAi <br /> required z20S editosome integrity suggests associa- <br />                                                                      Three KREPA4 dsRNA-containing <span id='am-81' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-82' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>vectors</span> constructed that <br /> tion subcomplexes. Thus KREPA4 does not <br />                                                                      contained full-length KREPA4 ORF, 59 315 bp <br /> appear catalytic role editing, data suggest      including 60 bp 59 UTR, 39 327 bp KREPA4. The <br /> is important editosome structure substrate binding.          inserts PCR amplified T. brucei PF 29.13 genomic <br />                                                                      DNA. The 657-bp full-length ORF amplified using 59-CCG <br />                                                                      CTCGAGCGGATGCGGGTGCGTTCACTCCT-39 forward <br /> MATERIALS AND METHODS                                                primer 59-AACTGCAGAACCAATGCATTGGTTAACACTC <br />                                                                      CAACTCCTGC-39 reverse primer, 315-bp 59 fragment <br /> Purification recombinant KREPA4                                   amplified using 59-CCGCTCGAGCGGTCTCTGAAGGAAA <br />                                                                      GGTGTGT-39 forward primer 59-CCCAAGCTTGGGC <br /> E. coli BL21 (DE3) pLysS cells (200 mL) containing N-terminal        GTGAGGCAAATTGGATTCGCC-39 reverse primer, and <br /> truncated KREPA4 pRSETc plasmid (Invitrogen) grown           327-bp 39 fragment amplified 59-CCGCTCGAGCGG <br /> to logarithmic phase LB induced 1 mM IPTG 3 h.       TTGGTGCATCGCCAACTGTGAC-39 forward primer and <br /> Cells collected centrifugation 6000g 15 min 4&#176;C.   59-CCCAAGCTTGGGGTCGACGTAGCGGGGAGCACCA-39 the <br /> The cell pellet resuspended 5 mL lysis buffer (50 mM     reverse primer. The resulting PCR products digested with <br /> Tris pH 8.0, 300 mM NaCl, 10 mM imidazole, 1 mg/mL            XhoI PstI HindIII ligated pZJM vector (Wang <br /> lysozyme), incubated ice 30 min, sonicated ice (6 3    et al. 2000) digested compatible restriction <br /> 10-sec bursts 200 W 10-sec cooling period). Lysate       enzymes create final constructs. <br /> centrifuged 10,000g 25 min 4&#176;C. The supernatant           The 29.13 strain PF T. brucei integrated genes T7 <br /> then transferred new tube equal volume 50 mM Tris     RNA polymerase tetracycline (tet) repressor used for <br />  <br />  <br /> 828      RNA, Vol. 12, No. 5 <br />                                                                                                KREPA4 integrity editosome <br />  <br />  <br />  <br /> all transfections (Wirtz et al. 1999). Plasmid DNA (15 mg)     59-labeled 59CL18, 39CL13pp gPCA6-2A RNAs, respectively <br /> was linearized NotI, cells transfected cloned 1 d   (Igo et al. 2000, 2002). The reaction products resolved on <br /> after transfection limiting dilution. The cells grown     polyacrylamide&#8211;urea gels visualized Storm PhosphorIm- <br /> in SDM-79 medium 27&#176;C transfectants selected          ager screens (Molecular Dynamics). RNA ligases adenylated <br /> in 25 mg/mL hygromycin, 15 mg/mL neomycin, 2.5 mg/mL              previously described (Sabatini Hajduk 1995) 25 mM <br /> phleomycin. Double-stranded RNA induced 1 mg/mL tet          HEPES, pH 7.9, 10 mM Mg(OAc)2, 50 mM KCl, 0.5 mM DTT, <br /> and uninduced induced cells counted daily obtain      10% dimethylsulfoxide 2.5 mCi [a-32P]ATP. The proteins <br /> growth curves. The cells maintained 1.0 3 106        resolved 10% SDS&#8211;PAGE gels radiolabeled <br /> 2.0 3 107 cells/mL course RNAi induction. The       proteins detected PhosphorImaging. <br /> cumulative cell number determined multiplying the <br /> number cells dilution.                                      Analysis editing complex <br />                                                                       TAP-tagged KREPA4 complexes purified 2 3 109 cells <br /> RNA analysis                                                          previously described (Rigaut et al. 1999; Schnaufer et al. 2003). <br /> Trizol (Invitrogen) reagent used isolate total RNA    For glycerol gradient analysis, cells lysed 1 mL Lysis <br /> parasites. Northern blot analysis performed 10 mg RNA          buffer (20 mM HEPES pH 7.9, 10 mM Mg(OAc)2, 50 mM KCl, <br /> from cells grown 2 4 d absence            1 mM EDTA, 1% Triton-X) containing protease inhibitors (1 mg/mL <br /> presence 1 mg/mL tet. Preparation RNA, transfer       Pepstatin, 2 mg/mL Leupeptin, 1 mM Pefabloc). Centrifugation <br /> the RNA membrane (Hybond-N+; Amersham Pharmacia                used clear lysate, equal protein was <br /> Biotech), hybridization detection performed accord-      loaded 11 mL 10%&#8211;30% glycerol gradient. The gradients <br /> ing manufacturer&#8217;s protocol (ULTRAhyb, Ambion). The            centrifuged 38,000 rpm 5 9 h 4&#176;C (SW40 rotor; <br /> oligonucleotide complementary 59 UTR region KREPA4          Beckman), 500-mL fractions collected top, flash <br /> used Northern analysis TAATACGACTCACTATAGGGG                  frozen liquid nitrogen, stored &#8211;80&#176;C. Every other <br /> GACTGAGCACTGCATCAA.                                                   fraction gradient (33 mL) resolved 10% SDS- <br />                                                                       PAGE gels. The proteins transferred PVDF membranes <br />                                                                       reacted MAbs specific KREPA1&#8211;3, KREL1 (Panigrahi <br /> Real-time PCR <br />                                                                       et al. 2001a), REAP1 (Madison-Antenucci et al. 1998), MRP1 <br /> Real-time PCR carried essentially described (Carnes        (Allen et al. 1998), KREPA6 (Schnaufer et al. 2003) PGK <br /> et al. 2005). Total RNA isolated cells using Trizol LS       (Parker et al. 1995) polyclonal antibodies. The membranes were <br /> Reagent (Invitrogen) 10 mg DNase I treated using         developed ECL kit (Amersham) manufacturer&#8217;s <br /> DNAfree kit (Ambion). The integrity RNA confirmed          instructions. <br /> using RNA nanochip BioAnalyzer (Agilent Technologies). <br /> The cDNA templates real-time PCR reverse transcribed         KREPA4 modeling <br /> from 4.5 mg RNA using random hexamers Taqman Reverse <br />                                                                       A structural model KREPA4 built MODELLER <br /> Transcription Reagents (Applied Biosystems) 30 mL reaction. <br />                                                                       program using software programs Accelrys Inc., DS Mod- <br /> Each experiment reaction reverse transcriptase as <br />                                                                       eling 1.1 (Sali Blundell 1993). The model generated based <br /> a control. The 30 mL reaction diluted sevenfold water. <br />                                                                       multiple alignments shown Figure 5, using S1 motifs <br /> For PCR reaction, 12.5 mL SYBR Green Master Mix <br />                                                                       E. coli PNPase (PDB code 1sro) E. coli RNase E (PDB <br /> (Applied Biosystems), 5 mL 1.5 mM forward oligo, 5 mL 1.5 mM <br />                                                                       code 1smx) templates. The quality modeled structure was <br /> reverse oligo, 2.5 mL cDNA template (or -RT control) were <br />                                                                       checked Profiles_3D program (Bowie et al. 1991) DS <br /> combined 96-well plate (Applied Biosystems). ABI <br />                                                                       Modeling 1.1. <br /> Prism 7000 thermocycler conditions reactions 50&#176;C <br /> for 2 min, 95&#176;C 10 min, followed 40 cycles 95&#176;C for <br /> 15 sec 60&#176;C 1 min. Template diluted 1:50      ACKNOWLEDGMENTS <br /> the 18S rRNA internal control Ct values similar less <br /> abundant edited pre-edited RNAs. Thermal dissociation             We thank E. Wirtz cell line 29.13, P. Englund pZJM <br /> confirmed PCR generated single amplicon. Primers            vector, S. Hajduk REAP1, M. Parsons PGK <br /> real-time PCR designed using ABI Primer Express v2.0             antiserum. We thank Rosemary Proff excellent technical <br /> software. The sequences of primers described         assistance. We grateful T. White A. Schnaufer <br /> previously (Carnes et al. 2005) KREPA6 forward: TTGTT          critical reading manuscript members the <br /> GGCGTCGTTCATGAT KREPA6 reverse: TGCGTGACGG                        Stuart laboratory fruitful discussions. This work sup- <br /> CATCTTCATA. Relative changes target amplicons deter-          ported NIH grants AI14102 K.S., AI053784 R.S., the <br /> mined using Pfaffl method, PCR efficiencies calculated    M.J. Murdock Charitable Trust. <br /> linear regression using LinRegPCR. <br />                                                                       Received September 29, 2005; accepted February 8, 2006. <br /> Assays enzymatic activities <br /> Precleaved deletion editing specifying removal Us     REFERENCES <br /> insertion editing specifying addition Us assayed   Allen, T.E., Heidmann, S., Reed, R., Myler, P.J., Go&#168;ringer, H.U., and <br /> vitro 59-labeled U5 59CL, U5 39CL gA6[14]PC-del             Stuart, K.D. 1998. Association guide RNA binding protein <br />  <br />  <br />                                                                                                             www.rnajournal.org         829 <br />  Salavati et al. <br />  <br />  <br />  <br />    gBP21 active RNA editing complexes Trypanosoma brucei.             Madison-Antenucci, S. Hajduk, S. 2001. RNA editing-associated <br />    Mol. Cell. Biol. 18: 6014&#8211;6022.                                                protein 1 RNA binding protein specificity preedited <br /> Aphasizhev, R., Aphasizheva, I., Nelson, R.E., Gao, G., Simpson, A.M.,            mRNA. Mol. Cell 7: 879&#8211;886. <br />    Kang, X., Falick, A.M., Sbicego, S., Simpson, L. 2003a. Isola-         Madison-Antenucci, S., Sabatini, R.S., Pollard, V.W., Hajduk, S.L. <br />    tion U-insertion/deletion editing complex Leishmania                 1998. Kinetoplastid RNA-editing-associated protein 1 (REAP-1): <br />    tarentolae mitochondria. EMBO J. 22: 913&#8211;924.                                  A novel editing complex protein repetitive domains. EMBO J. <br /> Aphasizhev, R., Aphasizheva, I., Nelson, R.E., Simpson, L. 2003b.             17: 6368&#8211;6376. <br />    A 100-kD complex RNA-binding proteins mitochon-                Madison-Antenucci, S., Grams, J., Hajduk, S.L. 2002. Editing <br />    dria <span id='am-9' about='obo:NCBITaxon_5689' typeof='owl:Thing obo:NCBITaxon_5688 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_5658 obo:NCBITaxon_33682 obo:NCBITaxon_1286322 obo:NCBITaxon_5654 obo:NCBITaxon_2759 obo:NCBITaxon_5653'><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-11'  ></span><span id='am-12' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Sauroleishmania tarentolae" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Leishmania tarentolae" datatype="xsd:string"></span>Leishmania tarentolae</span> catalyzes RNA annealing                      machines: The complexities trypanosome RNA editing. Cell <br />    interacts RNA editing components. RNA 9: 62&#8211;76.                   108: 435&#8211;438. <br /> Bowie, J.U., Luthy, R., Eisenberg, D. 1991. A method identify          Missel, A., Souza, A.E., Norskau, G., Go&#168;ringer, H.U. 1997. <br />    protein sequences fold known three-dimensional                     Disruption gene encoding novel mitochondrial DEAD-box <br />    structure. Science 253: 164&#8211;170.                                               protein Trypanosoma brucei affects edited mRNAs. Mol. Cell. <br /> Brecht, M., Niemann, M., Schlu&#168;ter, E., Mu&#168;ller, U.F., Stuart, K.,            Biol. 17: 4895&#8211;4903. <br />    Go&#168;ringer, H.U. 2005. TbMP42, protein component                   Mu&#168;ller, U.F., Lambert, L., Go&#168;ringer, H.U. 2001. Annealing of <br />    RNA editing complex African trypanosomes endo-                          RNA editing substrates facilitated guide RNA-binding protein <br />    exoribonuclease activity. Mol. Cell 17: 621&#8211;630.                               gBP21. EMBO J. 20: 1394&#8211;1404. <br /> Bycroft, M., Hubbard, T.J., Proctor, M., Freund, S.M.,                    O&#8217;Hearn, S., Huang, C.E., Hemann, M., Zhelonkina, A., Sollner- <br />    Murzin, A.G. 1997. The solution structure S1 RNA binding                Webb, B. 2003. Trypanosoma brucei RNA editing complex: Band II <br />    domain: A member ancient nucleic acid-binding fold. Cell                 structurally critical maintains band V ligase, is <br />    88: 235&#8211;242.                                                                   nonessential. Mol. Cell. Biol. 23: 7909&#8211;7919. <br /> Carnes, J., Trotter, J.R., Ernst, N.L., Steinberg, A.G., Stuart,          Pagni, M., Ioannidis, V., Cerutti, L., Zahn-Zabal, M., Jongeneel, C.V., <br />    K. 2005. An essential RNAse III insertion editing endonuclease              Falquet, L. 2004. MyHits: A new interactive resource for <br />    Trypanosoma brucei. Proc. Natl. Acad. Sci. 102: 16614&#8211;16619.                   protein annotation domain identification. Nucleic Acids Res. <br /> Chen, Y., Hung, C.H., Burderer, T., Lee, G.S. 2003. Development               32: W332&#8211;W335. <br />    RNA interference revertants Trypanosoma brucei cell lines            Panigrahi, A.K., Gygi, S., Ernst, N., Igo Jr., R.P., Palazzo, S.S., Schnaufer, A., <br />    generated double stranded RNA expression construct                      Weston, D., Carmean, N., Salavati, R., Aebersold, R., et al. 2001a. <br />    driven opposing promoters. Mol. Biochem. Parasitol. 126:                Association novel proteins. TbMP52 TbMP48, the <br />    275&#8211;279.                                                                       Trypanosoma brucei RNA editing complex. Mol. Cell. Biol. 21: 380&#8211;389. <br /> Cruz-Reyes, J., Zhelonkina, A.G., Huang, C.E., Sollner-Webb,              Panigrahi, A.K., Schnaufer, A., Carmean, N., Igo Jr., R.P., Gygi, S.P., <br />    B. 2002. Distinct functions RNA ligases active Trypanosoma           Ernst, N.L., Palazzo, S.S., Weston, D.S., Aebersold, R., Salavati, R., <br />    br <br /> </body></html>