<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Lifespan Regulation Evolutionarily <br /> Conserved Genes Essential Viability <br /> Sean P. Curran1,2, Gary Ruvkun1,2* <br /> 1 Department Molecular Biology, Massachusetts General Hospital, Boston, Massachusetts, United States America, 2 Department Genetics, Harvard Medical School, <br /> Boston, Massachusetts, United States America <br />  <br />  <br /> Evolutionarily conserved mechanisms control aging predicted prereproductive functions <span id='am-104' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-105' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-106' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> be <br /> subject natural selection. Genes essential growth development highly conserved evolution, <br /> but role longevity previously assessed. We screened 2,700 genes essential <span id='am-75' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-76'  ></span><span id='am-77' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Caenorhabditis</span> <br /> elegans development identified 64 genes extend lifespan inactivated postdevelopmentally. These <br /> candidate lifespan regulators highly conserved yeast humans. Classification candidate lifespan <br /> regulators functional groups identified expected insulin metabolic pathways revealed enrichment <br /> for translation, RNA, chromatin factors. Many essential gene inactivations extend lifespan the <br /> strongest known regulators aging. Early gene inactivations essential genes caused growth arrest larval <br /> stages, arrested animals live longer wild-type adults. daf-16 required enhanced <br /> survival arrested larvae, suggesting increased longevity physiological response essential gene <br /> inactivation. These results suggest insulin-signaling pathways play role regulation aging stage life. <br /> Citation: Curran SP, Ruvkun G (2007) Lifespan regulation evolutionarily conserved genes essential viability. PLoS Genet 3(4): e56. doi:10.1371/journal.pgen.0030056 <br />  <br />  <br />                                                                                          inhibit mitochondrial function adults shown <br /> Introduction <br />                                                                                          increase lifespan [34]. <br />    The lifespan organism regulated genetic                                 Recent genome-wide RNAi screens increased longevity <br /> and environmental in&#64258;uences <span id='am-101' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-102' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-103' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> [1]. Recent                                  identi&#65533;?ed ;100 potential regulators lifespan C. <br /> work identi&#65533;?ed speci&#65533;?c components variety                                  elegans diverse cellular pathways, are <br /> cellular processes regulate lifespan. In C. elegans loss-of-                        evolutionarily conserved [37,38], genes essential for <br /> function mutations insulin/insulin-like growth factor-1/                          viability underrepresented genome-wide RNAi screens <br /> daf-2 signaling pathway double lifespan                             postdevelopmental phenotypes aging. These RNAi <br /> an animal [2&#8211;6]. The regulation lifespan DAF-2 occurs                              screens adult longevity preclude identi&#65533;?cation gene <br /> during adulthood [7]. The insulin-signaling pathway nega-                                inactivations cause lethality, larval arrest, sterility, and/or <br /> tively regulates forkhead (FOXO) transcription factor                                developmental pleiotropies (i.e., essential genes), unless <br /> DAF-16, ultimately functions positively                                genes inactivated required developmen- <br /> negatively regulate transcription metabolic, chaperone,                               tal roles. <br /> cellular defense, genes [8&#8211;11]. The regulation                                 Here report 64 genes that, inactivated postdeve- <br /> lifespan insulin-like signaling cascade                                 lopmentally RNAi, increase adult lifespan. We report <br /> evolutionarily conserved mechanism demon-                                   enhanced survival phenotype animals arrested <br /> strated &#64258;ies mice [12&#8211;15]. The regulation DAF-16                               development essential gene inactivations. <br /> activity modulated JNK signaling pathway, the <br /> SIR-2.1 deacetylase, HSF-1, LIN-14, SMK-1 the <br />                                                                                          Results/Discussion <br /> nucleus [16&#8211;20]. <br />    In organisms rate aging tied reproduc-                              Postdevelopmental RNAi Screen Clones That Increase <br /> tion. In C. elegans germline proliferation produces DAF-16                             Lifespan <br /> and KRI-1 mediated signal negatively regulates lifespan                               To identify essential genes function adulthood to <br /> while somatic gonad promotes lifespan extension [21&#8211;23].                             regulate lifespan, selected 2,700 RNAi clones that, fed <br />    Caloric restriction (CR) extends lifespan species <br /> including yeast, worms, &#64258;ies, mice [24&#8211;29]. The sir-2.1 and <br />                                                                                          Editor: Stuart K. Kim, Stanford University School Medicine, United States of <br /> let-363 genes C. elegans regulate lifespan CR [30,31].                            America <br /> Unlike insulin signaling, &#64258;ies, imposing CR anytime                            Received February 19, 2007; Accepted February 26, 2007; Published April 6, 2007 <br /> increase lifespan [32]. Finally, perturbations mitochondrial <br />                                                                                          A previous version article appeared Early Online Release February <br /> function shown increase lifespan [33&#8211;36].                                   27, 2007 (doi:10.1371/journal.pgen.0030056.eor). <br /> Diminished function mitochondrial <span id='am-160' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-161' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-162' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-163' property="rdfs:label" content="Electron" datatype="xsd:string"></span><span id='am-164'  ></span>electron</span> transport <br />                                                                                          Copyright: &#211; 2007 Curran Ruvkun. <span id='am-91' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-92' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-93'  ></span><span id='am-94' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> open-access article distributed <br /> system extend mutations insulin-like                                  terms Creative Commons Attribution License, permits <br /> signaling pathway, mutations ubiquinone                                   unrestricted use, distribution, reproduction medium, provided the <br /> biosynthesis gene clk-1 increase longevity                            original author source credited. <br /> phenotype CR. The mechanism longevity induced                                 Abbreviations: CR, caloric restriction; RNAi, RNA interference <br /> defective mitochondria thought occur develop-                               * To correspondence addressed. E-mail: ruvkun@molbio.mgh. <br /> ment, previous attempts use RNA interference (RNAi)                             harvard.edu <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                       0479                                           April 2007 | Volume 3 | Issue 4 | e56 <br />                                                                                                                   Essential Longevity Genes C. elegans <br />  <br />  <br /> Author Summary <br />  <br /> The lifespan animal determined environmental and <br /> genetic factors, mechanisms identified increase <br /> lifespan evolutionarily conserved organisms. Previous <br /> longevity screens C. elegans identified 100 genes, but <br /> ;2,700 essential normal development excluded from <br /> analysis. Paradoxically, essential genes times more <br /> likely highly conserved phylogeny genes no <br /> obvious developmental phenotypes. We screened 2,700 <br /> essential genes increased adult lifespan initiating gene <br /> knockdown animal reached adulthood, bypassing <br /> earlier developmental roles. We identified 64 genes extend <br /> lifespan inactivated postdevelopmentally. More 90% of <br /> the genes identified conserved yeast humans. Many <br /> of newly identified longevity genes extend lifespan robustly <br /> as well-characterized longevity mutants. It possible that <br /> the homologues genes regulate lifespan other <br /> organisms well. Genetic analysis places genes in <br /> known pathways regulated insulin-like signaling, many <br /> of gene inactivations function independently this <br /> mechanism lifespan extension. Surprisingly, subset these <br /> gene inactivations induce potent developmental arrest also <br /> facilitate enhanced survival arrested state, suggesting that <br /> aging stage subject regulatory control. <br />  <br />  <br />                                                                                Figure 1. Representation Postdevelopmental RNAi Screen for <br /> from previous generation L1 larval stage,                      Longevity <br /> cause arrest embryonic larval stages, screened                  Embryos gravid C. elegans hermaphrodites isolated by <br /> for increased lifespan initiating RNAi L4 larval/                 hypochlorite treatment allowed hatch overnight absence <br /> young adult stage (Figure 1). We observations                      food. The synchronized L1 larvae placed OP50 agar <br />                                                                                plates 20 8C allowed develop L4/young adult stage. The L4/ <br /> that validate approach: First, C. elegans lifespan                 young adult animals washed plates, cleaned from <br /> extended fed dsRNA targeting insulin receptor/daf-2                   bacteria sucrose flotation, placed six-well RNAi plates (;30 <br /> at developmental larval stage adulthood [7];                       worms/well). From replicates pass, 470 RNAi clones were <br />                                                                                identified positive retested second pass similar <br /> second, conditional daf-2 alleles cause dauer arrest gene               conditions increasing stringency scoring positive. A total of <br /> is inactivated L1 stage increased longevity                      134 RNAi clones passed second pass criteria scored <br /> inactivated adults [39]; third, null alleles daf-2               longitudinally pass blind positive (daf-2 RNAi) and <br />                                                                                negative (empty vector, eri-1 RNAi, daf-16 RNAi) controls. A total of <br /> lethal [40] demonstrating necessity uncoupling                      64 RNAi clones increased mean lifespan .10% compared the <br /> developmental aging phenotypes essential genes                       negative controls. <br /> well utility producing non-null phenotypes RNAi.                  doi:10.1371/journal.pgen.0030056.g001 <br />    We performed RNAi screen utilizing eri-1(mg366) <br /> [41] strain improve RNAi cells types including                       89 gene inactivations 16,000 screened (;0.6%), a <br /> neurons, normally refractory RNA interference.                    higher proportion gene inactivations cause large <br /> In fact, longevity genes identify expressed                 increases longevity (percent increase compared vector <br /> in nervous system, implicated insulin                    control denoted parentheses RNAi clone), <br /> regulation longevity (Figure S1; Table S1) [42].                            use enhanced RNAi strain also <br />    Gene inactivations caused increases mean lifespan                   contribute increased sensitivity. <br /> of 10% scored positive screen. <br /> Endocrine signaling, stress adaptation, metabolism,                        Functional Classes Postdevelopmental Regulators of <br /> reproduction potent evolutionarily conserved regu-                     Longevity <br /> lators aging [43]. Our screen identi&#65533;?ed genes                       Protein synthesis. Sorting lifespan increase, inacti- <br /> canonical longevity-promoting pathways addition                          vation genes involved protein synthesis caused most <br /> pathways previously implicated aging (Figures S2                    potent lifespan increase. We identi&#65533;?ed RNAi clones <br /> S3; Table 1).                                                                  target components translation initiation factor <br />    Essential genes conserved phylogeny                        (eIF) complex; egl-45 (eIF3, 52%), eif-3.F (eIF3, 32%), eif-3.B <br /> genes obvious developmental phenotype;                       (eIF3, 51%), ifg-1 (eIF4G, 55%), T27F7.3 (eIF1, 25%), two <br /> 90% genes identify conserved yeast                       clones targeting inf-1 (eIF4A, 46% 28%). Inhibition of <br /> humans. The theory antagonistic pleiotropy suggests                    genes increased lifespan 50% longer with <br /> any genes function postreproductive longevity control                  vector control RNAi. In mammals, insulin stimulates protein <br /> should natural selection prereproductive stages.                   synthesis vivo vitro increasing rate of <br /> Our <span id='am-175' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-176' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-177'  ></span><span id='am-178' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-179' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> support notion: yield major lifespan                      translation initiation regulating eIF4F complex <br /> regulators 64 gene inactivations 2,700 tested                        consisting eIF4G, eIF4E, eIF4A, 40S ribosome <br /> (;2.4%), 4-fold increased yield compared previous                     [44]. eIF4F recruits eIF3 generate 43S ribosome. <br />  <br />       PLoS Genetics | www.plosgenetics.org                              0480                                       April 2007 | Volume 3 | Issue 4 | e56 <br />                                                                                                        Essential Longevity Genes C. elegans <br />  <br />  <br /> Protein synthesis closely monitored cell                 glp-1 genes function reproductive (Table <br /> allow energy equivalents redirected processes              S1) disrupt signaling germline increase <br /> such genomic maintenance stress adaptation. Mice                   lifespan. <br /> lacking translational inhibitor 4E-BP1 lean                 Vacuolar H&#254;-ATPases potent lifespan regu- <br /> increased metabolism mitochondrial biogenesis,                  lator emerge screen. These proteins acidify <br /> are opposing phenotypes long-lived mutants [45].                 intracellular compartments act synaptic transmission <br /> We clones target components 40S              cell death signaling cascades [51]. UNC-32, subunit of <br /> subunit ribosome, rps-3 (32%), rps-8 (18%), rps-11             vacuolar ATPase, regulates male longevity [52]. The <br /> (28%) increased mean lifespan. Although components                vacuolar ATPase comprised V0/V1 heteromultimer. We <br /> the 80S ribosome complex represented RNAi                     identi&#65533;?ed type subunit screen; vha-6 <br /> library screened, components 43S                      (V0, 24%) tag-300 (V1, 23%). In addition, identi&#65533;?ed <br /> complex, predominantly translation initiation factors                     candidate longevity genes linked cell-death <br /> emerged regulators lifespan.                                        pathways; apoptotic cysteine protease ced-3 (19%) the <br />    Under conditions ER stress transient block protein             cell death related nuclease crn-5 (41%). We surprised to <br /> synthesis occurs. Inactivation ER oxidoreductase <span id='am-170' about='obo:NCBITaxon_1112515' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_336669 obo:NCBITaxon_6854 obo:NCBITaxon_336668 obo:NCBITaxon_6843 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-171' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-172'  ></span><span id='am-173' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-174' property="rdfs:label" content="Ero" datatype="xsd:string"></span>ero</span>-1             identify RNAi clone targeting ced-3 screen since <br /> (32%) increased lifespan (Table 1). In mammals, C. elegans,          ced-3(0) mutants fail undergo programmed cell death but <br /> and yeast, ATF4/atf-5/GCN4 gene contains extensive                 lethal [53]. However, noted synthetic lethality <br /> 59UTR multiple upstream open reading frames. Under                   ced-3(717) daf-2(e1370) indicating genetic <br /> normal conditions upstream open reading frames                    interaction longevity-promoting pathways <br /> preferentially utilized translation initiation yielding               (unpublished data). In support &#65533;?nding, increased <br /> truncated proteins, low nutrient conditions                 lifespan phenotype ced-3 (RNAi) dependent daf- <br /> the proper ATG used synthesize ATF4/atf-5/GCN4. eIF1                16 (Figure 2). <br /> has shown regulate initiation <span id='am-96' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-97' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-98'  ></span><span id='am-99' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-100' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span> selection [46]. In               Mitochondria. We surprised &#65533;?nd late <br /> support &#65533;?ndings recent studies identi&#65533;?ed ifg-1            inactivation mitochondrial genes increased lifespan. <br /> and rps-11 regulators longevity [47,48]. Therefore,                 RNAi clones targeting F26E4.6 (28%) atp-3 (21%) were <br /> although complex, translation effective regulator <br />                                                                           previously identi&#65533;?ed increase lifespan [37,38]. However, <br /> of cellular stress adaptation. <br />                                                                           previous work suggested perturbations disable <br />    Signaling. Genes insulin-signaling pathway well- <br />                                                                           mitochondria increase lifespan occur larval <br /> established potent regulators longevity worm <br />                                                                           development, L4 stage observe <br /> [43]. Our screen identi&#65533;?ed signaling molecules that <br />                                                                           [34]; differences attributed enhanced <br /> extend adult lifespan (Figure S2). For example, identify <br />                                                                           RNA interference phenotype eri-1(mg366) strain. In <br /> SEM-5 (24%), SH2- SH3-like protein negatively <br />                                                                           addition, identi&#65533;?ed seven mitochondrial RNAi <br /> regulates RAS-MAP -IP3 signal transduction. sem-5 was <br />                                                                           clones signi&#65533;?cantly increased adult lifespan the <br /> also identi&#65533;?ed enhancer dauer formation (S.S. Lee, <br />                                                                           gene inactivation occurred postdevelopmentally: sub- <br /> unpublished data). In addition, inactivation annotated <br />                                                                           units respiratory complex V (atp-2 [43%] phi-37 [27%]); <br /> kinases&#8212;the serine/threonine protein kinases tpa-1 (28%) and <br />                                                                           subunits NADH:ubiquinone oxidoreductase <br /> tag-181 (21%), ARK kinase <span id='am-65' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-66' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> member sel-5 (29%), and <br /> a previously identi&#65533;?ed integrin-linked kinase pat-4 (13%)                 complex (F59C6.5 [19%], nuo-1 [23%], Y56A3A.19 <br /> [38]&#8212;increased lifespan. daf-16(mgDf47) epistasis (discussed              [21%]); predicted mitochondrial carrier protein (F43E2.7, <br /> below) reveals kinases act                  21%); mitochondrial AAA-protease (spg-7, 22%). In <br /> insulin pathway.                                                          support role nuo-1 negative regulation lifespan, <br />    The C. elegans genome contains insulin-like receptor,              overexpression nuo-1 causes decreased fecundity and <br /> daf-2, 39 insulin-like peptides              shortened lifespan [40]. <br /> known functions [49]. Loss-of-function mutations daf-2                Gene expression. RNA binding/processing chromatin- <br /> of strongest enhancers lifespan C. elegans,          associated factors largest <span id='am-80' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-81' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> regulators; <br /> screen RNAi daf-2 increased lifespan ;79%. In humans,               comprising 20% genes identi&#65533;?ed screen <br /> the relaxin family peptides structurally similar                yielding potent extension lifespan <br /> insulin bind seven transmembrane receptors                      (Figures S2 S3; Table 1). The strong enrichment RNA <br /> initiate signaling cascades [50]. We identi&#65533;?ed serpentine             factors intriguing identi&#65533;?ed the <br /> receptors, str-49 (15%) sre-25 (23%), act lifespan            previous genome-wide RNAi screens longevity [37,38]. We <br /> control. Since lifespan extension phenotype gene             identi&#65533;?ed clones targeting predicted RNA helicases, dic- <br /> inactivations requires DAF-16 (Figure 2), conceivable          1 (22%), B0511.6 (50%), ZK686.2 (24%) scored as <br /> these seven transmembrane receptors utilize subset               strongest gene inactivations increase lifespan <br /> insulin-like peptides, analogy relaxin humans.                   (Table 1). We clones predicted conserved <br />    Signals proliferating germ cells negatively regulate C.           RNA binding domains; hrp-1 (49%) contains RRM domain, <br /> elegans aging, insulin-signaling pathway [43]. We                 C56G2.1 (20%) KH Tudor domain may <br /> identi&#65533;?ed glp-1 (33%), codes DSL-family ligand                facilitate RNA chromatin interactions, respectively. In <br /> receptor germline controls germ cell proliferation            addition, protein products large number genes <br /> and negatively regulates lifespan [22]. The design                 identi&#65533;?ed predicted modify and/or process RNA; these <br /> screen fortunate somatic gonad                    include predicted mRNA splicing factor D1054.14 (36%); <br /> establishment initial pool proliferating germ cells          ribonucleases crn-5 (41%) Y54E10BR.4 (15%); mRNA <br /> adulthood perturbed. Down-regulating genes                cleavage polyadenylation factor F25C6.2 (24%); RNA <br />  <br />       PLoS Genetics | www.plosgenetics.org                         0481                                 April 2007 | Volume 3 | Issue 4 | e56 <br />                                                                                                            Essential Longevity Genes C. elegans <br />  <br />  <br />  <br /> Table 1. 64 RNAi Clones That Increase Adult Lifespan When Fed Postdevelopmentally <br />  <br /> Sequence         Locus     Chromosome   Group   Brief Description                      Meana   Standard   Median/    Total             p-Valuee <br /> Name                                                                                           Errorb     25%c       n/Censored nd <br />  <br /> Y55D5A.5         daf-2     III          SIG     Insulin/IGF receptor tyrosine kinase   29.63   0.90       31/35      141/11            ,0.0001 <br /> M110.4           ifg-1     II           TRN     Translation-initiation factor 4G       25.74   1.18       27/31       49/4             ,0.0001 <br /> C27D11.1         egl-45    III          TRN     Translation-initiation factor 3a       25.20   0.79       27/31       94/10            ,0.0001 <br /> Y54E2A.11        eif-3.B   II           TRN     Translation-initiation factor 3b       25.00   0.87       25/31       86/7             ,0.0001 <br /> B0511.6                    I            RNA     ATP-dependent RNA helicase             24.78   0.85       27/31       80/7             ,0.0001 <br /> F42A6.7          hrp-1     IV           RNA     RRM domain                             24.67   0.74       25/29       67/7             ,0.0001 <br /> F57B9.6f         inf-1     III          TRN     Translation-initiation factor 4A       24.19   1.18       23/31       42/0             ,0.0001 <br /> C34E10.6         atp-2     III          MIT     F0F1-type ATP synthase                 23.74   0.78       25/29       82/6             ,0.0001 <br /> C14A4.5          crn-5     II           RNA     Exosomal 39-59 exoribonuclease         23.36   0.75       23/27       81/11            ,0.0001 <br /> T28F12.2         unc-62    V            TRX     HOX transcription factor               22.91   0.74       21/29      104/20            ,0.0001 <br /> C34C6.6          prx-5     II           MET     TPR repeat-containing                  22.88   0.66       23/27       61/12            ,0.0001 <br /> D1054.14                   V            RNA     Predicted PRP38-like splicing factor   22.50   1.00       23/27       48/8             ,0.0001 <br /> F19B6.1                    IV           SIG     <span id='am-136' about='obo:NCBITaxon_96820' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_6820 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_41116 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_88770 obo:NCBITaxon_29979 obo:NCBITaxon_526425 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_526418 obo:NCBITaxon_6072'><span id='am-137'  ></span><span id='am-138' property="rdfs:label" content="Armadillo" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Armadillo</span>/<span id='am-20' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-21' property="rdfs:label" content="Beta" datatype="xsd:string"></span><span id='am-22'  ></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>beta</span>-catenin/plakoglobin     22.30   0.77       23/27       65/2             ,0.0001 <br /> F02A9.6          glp-1     III          SIG     Fibrillins, Ca2&#254;-binding EGF-like      22.00   0.70       19/25       93/15            ,0.0001 <br /> D2013.7          eif-3.F   II           TRN     Translation-initiation factor 3f       21.90   0.94       19/29       70/5             ,0.0001 <br /> Y105E8B.8        ero-1     I            TFC     ER oxidoreductin                       21.89   0.94       21/27       58/5             ,0.0001 <br /> VC5.3            npa-1     V            MET     Fatty acids vitamin A binding      21.84   0.70       23/25       82/38            ,0.0001 <br /> C23G10.3         rps-3     III          TRN     40S ribosomal subunit S3               21.80   0.74       21/27       76/15            ,0.0001 <br /> W07E6.1                    II           RNA     tRNA rRNA cytosine-C5-methylase    21.66   0.84       21/27       79/14            ,0.0001 <br /> F35B12.5         sas-5     V            NUC     Chromatin-binding snw family           21.56   0.60       21/27       95/9             ,0.0001 <br /> F35G12.3         sel-5     III          SIG     ARK protein kinase family              21.44   0.69       21/25       85/15            ,0.0001 <br /> F57B9.6f         inf-1     III          TRN     Translation-initiation factor 4A       21.25   0.89       19/27       49/1             ,0.0001 <br /> F40F11.1         rps-11    IV           TRN     40S ribosomal-protein S11              21.22   0.66       21/25       71/8             ,0.0001 <br /> K01C8.10         cct-4     II           CHP     Chaperonin complex                     21.19   0.80       21/27       72/10            ,0.0001 <br /> B0545.1          tpa-1     IV           SIG     Serine/threonine protein kinase        21.19   0.87       23/25       55/17            ,0.0001 <br /> F26E4.6                    I            MIT     Cytochrome c oxidase                   21.15   0.66       19/27       94/0             ,0.0001 <br /> ZK792.3g         inx-9     IV           SIG     Innexin-type channels                  21.12   0.90       19/27       63/11            ,0.0001 <br /> Y105C5B.12h                IV           UNK                                            21.09   0.81       21/25       54/10            ,0.0001 <br /> H28O16.1         phi-37    I            MIT     F0F1-type ATP synthase                 21.04   0.65       21/27      102/7             ,0.0001 <br /> F26A3.4                    I            SIG     Dual-specificity phosphatase           20.95   0.71       19/27       70/10            ,0.0001 <br /> F53F4.11                   V            UNK                                            20.94   0.76       21/25       63/13            ,0.0001 <br /> ZC132.3                    V            UNK                                            20.82   0.79       19/25       61/9             ,0.0001 <br /> T27F7.3                    II           TRN     Translation-initiation factor 1        20.72   1.00       19/27       66/18             0.0002 <br /> ZK686.2                    III          RNA     DEAD-box ATP-dependent RNA helicase    20.62   0.77       19/25       56/4             ,0.0001 <br /> F25G6.2                    V            RNA     mRNA cleavage polyadenylation      20.59   0.80       19/27       56/7             ,0.0001 <br /> ZK1127.5                   II           RNA     RNA 39-terminal phosphate cyclase      20.54   0.85       17/27       71/22            ,0.0001 <br /> T07A9.8                    IV           RNA     RNA methylase                          20.52   0.85       19/25       78/22            ,0.0001 <br /> VW02B12L.1       vha-6     II           TFC     Vacuolar H&#254;-ATPase                     20.49   0.80       19/27       64/6              0.0001 <br /> D2030.9                    I            UNK     WD40 repeat-containing                 20.49   0.78       19/25       43/0              0.0002 <br /> C14F5.5          sem-5     X            SIG     SH2 SH3 domains                    20.47   0.66       19/25      111/28             0.0001 <br /> C09H10.3         nuo-1     II           MIT     NADH:ubiquinone oxidoreductase         20.37   0.86       21/25       46/8             ,0.0001 <br /> Y110A7A.12i      tag-300   I            TFC     Vacuolar H&#254;-ATPase                     20.34   0.74       19/25       72/8             ,0.0001 <br /> Y57A10C.5        sre-25    II           SIG     Serpentine receptor                    20.31   0.77       19/25       58/5              0.0003 <br /> Y47G6A.10        spg-7     I            PTO     AAA&#254;-type ATPase peptidase             20.28   0.78       21/25       60/2              0.0001 <br /> F08B4.1          dic-1     IV           RNA     DEAD box RNA helicase                  20.16   0.71       19/25       77/11            ,0.0001 <br /> F27C1.7          atp-3     I            MIT     Mitochondrial F1F0-ATP synthase        20.12   0.76       19/25       74/19             0.003 <br /> T20F10.1         tag-181   I            SIG     Warts/lats-like S/T kinases            20.09   0.61       21/23       75/5              0.0001 <br /> Y56A3A.19                  III          MIT     NADH-ubiquinone oxidoreductase         20.08   0.71       19/25       76/11            ,0.0001 <br /> F43E2.7                    II           MIT     Mitochondrial carrier protein          20.07   0.86       19/25       61/9              0.0001 <br /> F52E1.1          pos-1     V            NUC     CCCH-type Zn-finger                    19.90   0.89       19/25       65/18             0.002 <br /> F01F1.8          cct-6     III          CHP     Chaperonin complex                     19.88   0.72       19/25       85/26             0.0008 <br /> C56G2.1                    III          RNA     KH Tudor RNA-binding               19.88   0.58       21/27      107/15            ,0.0001 <br /> F59C6.5                    I            MIT     NADH-ubiquinone oxidoreductase         19.77   0.74       17/25       60/3              0.0006 <br /> F57C9.5          htp-3     I            NUC     HORMA domain                           19.71   0.74       19/23       50/1              0.0001 <br /> C48D1.2          ced-3     IV           SIG     Caspase, apoptotic cysteine protease   19.68   0.85       19/23       61/10             0.0009 <br /> F42C5.8j         rps-8     IV           TRN     40S ribosomal-protein S3               19.55   0.64       19/23       78/9              0.0001 <br /> Y65B4BR.5                  I            TRX     NAC/TS-N transcription factor          19.46   0.70       19/23       68/7              0.0001 <br /> T21B10.1                   II           UNK                                            19.39   0.61       19/23       82/14             0.0001 <br /> C34D4.7k         str-49    IV           SIG     Serpentine receptor                    19.10   0.49       19/21       99/28             0.0001 <br /> Y17G7B.5         mcm-2     II           NUC     DNA-replication licensing factor       19.09   0.63       19/23       79/15             0.0067 <br /> C01G8.5          erm-1     I            SIG     Radixin, moesin                        19.01   0.71       19/21       64/19             0.0022 <br /> Y54E10BR.4                 I            PTO     RNA-binding Nob10                      19.00   0.70       19/21       92/29             0.0021 <br /> C29F9.7          pat-4     III          SIG     Integrin-linked kinase                 18.78   0.69       17/23       76/15             0.0193 <br /> M03A8.4          gei-15    X            UNK                                            18.78   0.68       19/23       56/10             0.0188 <br /> ZC101.2          unc-52    II           SIG     Basement-membrane (HSPG) core          18.47   0.62       17/23       70/14             0.036 <br /> Vector Control                                                                         16.57   0.48       17/21      150/38            NA <br /> T07A9.5          eri-1     IV           RNA     RNA nuclease                           16.22   0.55       15/21      162/30             * <br />  <br />  <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              0482                               April 2007 | Volume 3 | Issue 4 | e56 <br />                                                                                                                                     Essential Longevity Genes C. elegans <br />  <br /> RNAi clones identified outlined Figure 1. Functional groups brief description based KOG codes [40]: CHP, chaperone; MET, metabolism; MIT, mitochondria; NUC, <br /> nuclear chromatin; PTO, protein turnover; RNA, RNA biology; SIG, signaling; TFC, trafficking; TRX, transcription; TRN, translation protein synthesis; UNK, unknown. <br /> a <br />   The mean lifespan days. <br /> b <br />    The standard error mean days. <br /> c <br />   The median lifespan time 25% individuals remain alive days. <br /> d <br />    The total number individuals scored shown (followed number individuals censored bursting vulva, bagging, crawling agar). <br /> e <br />   The p-value log rank test comparing RNAi treatment populations vector control population. <br /> f <br />   F57B9.6 RNAi clones hybridize F57B9.3. <br /> g <br />    ZK792.3 RNAi clone hybridize ZK792.2. <br /> h <br />   Y105C5B.12 RNAi clone hybridize T13A10.11. <br /> i <br />  Y110A7A.12 RNAi clone hybridize F20B6.2. <br /> j <br />   F42C5.8 RNAi clone hybridize F24C5.9. <br /> k <br />   C34D4.7 RNAi clone hybridize C34D4.6 C34D4.16. <br /> IGF, insulin-like growth factor; NA, applicable. *, p-value significant. <br /> doi:10.1371/journal.pgen.0030056.t001 <br />  <br />  <br /> cyclase ZK1127.5 (24%); RNA methylases, T07A9.8                                      lating small RNA mechanisms gene regulation eukar- <br /> (24%) W07E6.1 (31%).                                                                     yotes [54&#8211;56]. Inactivation chromatin-associated <br />    Chromatin-associated factors conserved players                                     factors caused increased lifespan: C56G2.1 (20%), sas-5 <br /> altering gene expression, coordinating cell fate, modu-                                  (30%), htp-3 (19%), DNA replication licensing factor <br />                                                                                              mcm-2 (15%). In support &#65533;?ndings, mcm-2 also <br />                                                                                              identi&#65533;?ed Serial Analysis Gene Expression analysis be <br />                                                                                              transcriptionally repressed daf-2/insulin-receptor mutants <br />                                                                                              contribute longevity phenotype [57]. <br />                                                                                                 Two transcription factors, unc-62/homothorax/MEIS (38%) <br />                                                                                              Y65B4BR.5/NAC/TS-N (17%) negatively <br />                                                                                              regulate lifespan. Recently, Dubcovsky laboratory re- <br />                                                                                              ported mutations NAC transcription factor NAM- <br />                                                                                              B1 delay senescence weeks <span id='am-50' about='obo:NCBITaxon_4564' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_147368 obo:NCBITaxon_359160 obo:NCBITaxon_1 obo:NCBITaxon_4479 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4734 obo:NCBITaxon_3398 obo:NCBITaxon_147389 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_38820 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasRelatedSynonym" content="Triticum L." datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="Triticum" datatype="xsd:string"></span><span id='am-54'  ></span><span id='am-55' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Triticum</span> <br />                                                                                              turgidum, ancestral wild wheat allele functions to <br />                                                                                              accelerate senescence increase nutrient remobilization <br />                                                                                              [58], suggesting evolutionary conserved <br />                                                                                              mechanism regulating lifespan. Although role unc-62 <br />                                                                                              previously described adult animal, unc- <br />                                                                                              62 promoter element active adult neuronal tissues <br />                                                                                              (Figure S1). While mechanism RNA chromatin- <br />                                                                                              associated factors controlling lifespan unknown, there <br />                                                                                              signi&#65533;?cant alterations transcriptional pro&#65533;?le of <br />                                                                                              aging populations [59]; genes regulate such <br />                                                                                              changes. <br />  <br />                                                                                              Phenotypic Analysis Candidate Longevity Genes <br />                                                                                                 One established method validating genes emerge <br />                                                                                              RNAi screens test gene knockouts same <br />                                                                                              phenotype. For essential genes, prove possible to <br />                                                                                              study longevity arrested animals. However, study <br />                                                                                              adult longevity conditional alleles required. Such <br />                                                                                              alleles tend emerge detailed genetic analysis rather <br />                                                                                              genomic knockout projects currently <br />                                                                                              available. <br />                                                                                                 The aging research community characterized several <br />                                                                                              central mechanisms mediate lifespan regulation includ- <br />                                                                                              ing insulin/insulin-like growth factor signaling, CR, and <br />                                                                                              mitochondrial function. To classify pathways represented <br />                                                                                              new genes, performed secondary assays: DAF-16 <br />                                                                                              localization, sod-3 expression, arrested larval survival, sup- <br />                                                                                              pression polyglutamine aggregation, aberrant fat <br /> Figure 2. Identification RNAi Clones Act DAF-16                              metabolism clustered genes phenotypes <br /> (A) Shading indicates daf-16(mgDf47); eri-1(mg366) epistatic                  observed (Figure 2; Figure S4; Table 2; Tables S1 S2). <br /> lifespan extension observed eri-1(mg366) (Table S2).                                         Identi&#65533;?cation clones act converge the <br /> (B) DAF-16 nuclear localization feeding RNAi clone shown. Light <br /> shading indicates nuclear localization, medium shading                              insulin-signaling pathway. The DAF-16/FOXO transcription <br /> indicates nuclear localization (Figure S4).                                             factor key player biological processes that <br /> (C) sod-3p::gfp expression RNAi feeding shown. Light shading                        regulate lifespan including stress adaptation metabolism <br /> indicates weak expression, medium shading indicated modest expres- <br /> sion, dark shading indicates strong expression.                                          [10,11,39,60]. Because loss function daf-16 alleles are <br /> doi:10.1371/journal.pgen.0030056.g002                                                        epistatic longevity promoting mutations, in- <br />  <br />         PLoS Genetics | www.plosgenetics.org                                          0483                                            April 2007 | Volume 3 | Issue 4 | e56 <br />                                                                                                                                     Essential Longevity Genes C. elegans <br />  <br />  <br />                                                                                                       DAF-16 extend lifespan, translocation DAF-16 the <br /> Table 2. Larval Arrest Survival Phenotypes RNAi Clones                                         nucleus general stress represent new <br /> during Development                                                                                    regulator DAF-16 activity. <br />                                                                                                          Larval arrest survival phenotypes RNAi clones <br /> Sequence Name                Locus           eri-1(mg366)             daf-16(mgDf47);                 development. Loss DAF-2 development <br />                                                                       eri-1(mg366)                    causes constitutive arrest dauer stage. daf-16(lf) <br />                                                                                                       mutations defective dauer arrest induced low <br /> F27C1.7                      atp-3           Lva,   Age               Lva, Age <br />                                                                                                       insulin signaling fail arrest L1 stage larvae in <br /> H28O16.1                     phi-37          Lva,   Age               Lva, Age <br /> Y47G6A.10                    spg-7           Lva                      non-Lva, n.c.                   absence food [60,61]. Many 2,700 gene <br /> T27F7.3                                      Lva,   Age               Lva, non-Age                    inactivations initiated feeding RNAi L1 stage <br /> M110.4                       ifg-1           Lva,   Age               Lva, non-Age                    cause highly penetrant developmental arrest stages ranging <br /> VW02B12L.1                   vha-6           Lva,   Age               Lva, non-Age <br />                                                                                                       early larval sterile adult stage (Table S1). A subset of <br /> F01F1.8                      cct-6           Lva,   Age               Lva, non-Age <br /> B0511.6                                      Lva                      Lva, n.c.                       gene inactivations caused arrested larvae survive <br /> Y54E2A.11                    eif-3.B         Lva,   Age               Lva, non-Age                    longer control animals facilitated DAF-16 nuclear <br /> C34E10.6                     atp-2           Lva                      Lva, n.c.                       localization arrested stage (Figure 2; Table 2). We tested <br /> F57B9.6a                     inf-1           Lva,   Age               non-Lva, non-Age <br />                                                                                                       larval arrest increased survival arrested <br /> C27D11.1                     egl-45          Lva                      Lva <br /> F42C5.8                      rps-8           Lva                      Lva, n.c.                       larvae induced gene inactivations depend DAF-16 <br /> C23G10.3                     rps-3           Lva,   Age               Lva, non-Age                    activity feeding RNAi clones cause larval arrest in <br /> D2013.7                      eif-3.F         Lva                      Lva                             eri-1(mg366) daf-16(mgDf47);eri-1(mg366) double mutant. <br /> F40F11.1                     rps-11          Lva                      Lva, n.c. <br />                                                                                                       The absence DAF-16 abrogated long-lived phenotype <br />                                                                                                       arrested larvae (Table 2). For gene <br /> Age, aging alteration; Lva, larval arrest; n.c., change survival compared eri-               inactivations, daf-16 mutant animals arrested fed <br /> 1(mg366). <br /> a <br />  Both RNAi clones targeting inf-1 identified screen increased longevity <br />                                                                                                       RNAi clone; did survive long the <br /> scored similarly.                                                                                     arrested state. The daf-16(mgDf47) allele did, however, weakly <br /> doi:10.1371/journal.pgen.0030056.t002                                                                 suppress larval arrest inf-1 spg-7 inactivation (Table <br />                                                                                                       2). These data suggest stage, stress pathways <br />                                                                                                       requiring daf-16 triggered gene inactivations to <br /> activated 64 candidate longevity genes eri-1(mg366)                                         ensure survival arrested larvae. <br /> strain carrying null daf-16(mgDf47) mutation scored                                           We noted gene inactivations cause <br /> for lifespan extension (Figure 2; Table S2). In accord                                           increased survival arrested larvae encode translation <br /> previous studies, RNAi clones targeting mitochondrial                                            factors. Translation major target antibiotics, which <br /> genes increase adult lifespan did depend DAF-16                                           produced wide range <span id='am-56' about='obo:NCBITaxon_4751' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2759'><span id='am-57' property="rdfs:label" content="Fungi" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="PMID:11062127" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasExactSynonym" content="fungi" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasRelatedSynonym" content="fungi" datatype="xsd:string"></span><span id='am-62'  ></span><span id='am-63' property="oboInOwl:hasDbXref" content="PMID:12684019" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>fungi</span> microbes that <br /> [34,35]. The gene inactivations targeting protein synthesis                                       <span id='am-130' about='obo:NCBITaxon_333870' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_195229 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41087 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_71193 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-131'  ></span><span id='am-132' property="oboInOwl:hasRelatedSynonym" content="Nematodes Berthold, 1827" datatype="xsd:string"></span><span id='am-133' property="rdfs:label" content="Nematodes" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-135' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>nematodes</span> encounter environment. As larvae or <br /> machinery increase lifespan absence DAF-16.                                            adult enters environment antibiotic, be <br /> However, RNAi representative genes                                                 signaling pathways detect ribosomal de&#65533;?ciency trigger <br /> classes increase lifespan daf-16(mgDf47);eri-                                        developmental arrest xenobiotic protective path- <br /> 1(mg366) strain represent DAF-16 independent                                                 ways. The induced stress adaptation survival pathways <br /> pathways lifespan extension (Figure 2). We classi&#65533;?ed                                           ensure animal live long to <br /> remaining RNAi clones DAF-16 dependent                                                escape antibiotic resume reproductive development. <br /> failed extend mean lifespan absence DAF-16                                         Inhibition translation RNAi ribosomal other <br /> least 10% compared vector controls. Among DAF-16-                                              translation factors mimic ribosomal de&#65533;?ciencies <br /> dependent genes signaling molecules                                              induced antibiotics normal C. elegans ecosystem, and <br /> RNA-related genes including mcm-2, hrp-1, pos-1, sas-5, crn-5,                                        trigger physiological response, developmental arrest, and <br /> T07A9.8, ZK686.2, ZK1127.5. These genes act                                             cessation aging. Because prereproductive arrest, this <br /> converge insulin-signaling pathway lifespan                                              response natural selection trigger longevity <br /> control.                                                                                              enhancing pathways [62]. It signi&#65533;?cant gene <br />    Although role DAF-16 insulin-signaling best                                       inactivations potent arrested larval lifespan increases, a <br /> understood, DAF-16 receives inputs multiple cellular                                             vacuolar ATPase mitochondrial ATP synthase, are <br /> pathways functions outside adult lifespan                                                  targets natural antibiotics [63,64]. <br /> extension. DAF-16 transcriptional activity negatively <br /> regulated cytoplasmic sequestration. Upon loss inhib-                                           Summary <br /> ition, DAF-16 translocates nucleus. Therefore,                                                 In study, screened large portion the <br /> tested gene inactivations identi&#65533;?ed                                         genome previously underrepresented genome-wide based <br /> screen induce DAF-16 nuclear localization, regardless                                           RNAi screens aging phenotypes identi&#65533;?ed 64 genes <br /> of necessity DAF-16 adult lifespan extension.                                              normally function shorten lifespan. Characterization <br /> Despite independent daf-16 gene activity                                                 gene knockdowns epistasis insulin- <br /> increasing adult lifespan, inactivation atp-3, ifg-1, vha-6,                                       signaling pathway use biomarkers aging places them <br /> F25G6.2, B0511.6, eif-3.B, inf-1, egl-45, rps-8, rps-3, eif-3.F, rps-                              distinct classes. Because molecules predicted to <br /> 11 induced localization DAF-16 nucleus                                                  function complex cellular pathways (Figure S5), future <br /> consequently drove expression DAF-16 target gene                                               work focus dissect mechanisms employed these <br /> sod-3 (Figure 2). Since genes function                                           essential processes regulate lifespan. <br />  <br />         PLoS Genetics | www.plosgenetics.org                                                   0484                                  April 2007 | Volume 3 | Issue 4 | e56 <br />                                                                                                                      Essential Longevity Genes C. elegans <br />  <br />  <br /> Materials Methods                                                                 c56g2.1: F: 59-CAGACAGGTGAAGCTGAGCGTGGC-39, R: 59- <br />                                                                                     AAACGCAGAAAACGTCGGTGACGGAATG-39; <br />    Strains. Strains maintained cultured using standard                       egl-45: F: 59-CCAGCCAGGAAAAATCGATTATATTAAG, R: 59- <br /> techniques [65]. We used following C. elegans strains mutant                AGTTGTGCTCGGATTACCGCTGAATTG-39; <br /> alleles: wild-type N2 Bristol, eri-1(mg366)IV, daf-16(mgDf47)I;eri-                   unc-62: F: 59-CCCTGAAATTGTTGCGAAAGTTTCTG, R: 59- <br /> 1(mg366)IV; zIs356, (daf-16p::daf-16:: g fp; rol-6[su1006]); muIs84, (sod-          GTTCCTGCAAGAGAGAAATATTAAATTTTG-39; <br /> 3p:: g fp); huIs33, (sod-3p::gfp; rol-6[su1006]); rmIs133: (unc-                  htp-3: F: 59-CTCCCGAAGATTCCGCATTTGCTC, R: 59-TTTGA- <br /> 54p:: Q40yfp).                                                                      CACTTAAAATATTTTAAAACATTTTTTTTAA-39; <br />    Postdevelopmental RNAi lifespan screen. Eggs isolated                    f26a3.4: F: 59-GACCGGAACAGGTGGGCAATGTCGAC, R: 59- <br /> gravid eri-1(mg366) worms synchronized hatching overnight                 TTTGTAGTGTATCTGTAATCATATTAAATTTGATTC-39; <br /> the absence food. The synchronized L1 larvae placed                   inf-1: F: 59-GTATGTGTTTATGGTGTGTGCACAAG, R: 59-GA- <br /> OP50-containing agar plates allowed develop L4-stage                      CAGGTGGGTTGAAAAGTTAAAAATTAAC-39; <br /> larvae 20 8C. The L4-stage larvae washed thoroughly, cleaned                  f08b4.1: F: 59-CTAACCGATTCCTCAAGCCACGTGGG, R: 59- <br /> by sucrose &#64258;otation, placed 12-well plates <span id='am-107' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-108' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-109' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-114'  ></span><span id='am-115' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span>Escherichia coli</span>             TTTTGATTATTGATATTTCATTCGAATTTGCCAG-39; <br /> expressing double-stranded RNA (dsRNA) (described below). We                          y54e10br.4: F: 59-CTCTCCCGATTCCGCCATAATGCCCG, R: 59- <br /> carried large-scale RNAi screen using enhanced RNAi strain                TCTCTGAAATATCGAAAAGAAATGAGATAATTG-39; <br /> eri-1(mg366) (Figure S1). Brie&#64258;y, RNAi colony grown                          sem-5: F: 59-GGGTTAGAGCACTCTTAATGAGTCATG, R: 59- <br /> overnight LB 50 lg/ml ampicillin seeded 12-                   CGTCTCGCTACCTGAAATATACTCTT-39; <br /> well RNAi agar plates containing 5 mM isopropylthiogalactoside                        zk686.2: F: 59-GGTTCCGGAGATAACCAAGCAGTATTGG, R: 59- <br /> (IPTG). The RNAi bacteria induced overnight room temper-                    TATCTGGAGAAATTAAAATATGAACCAAAAAATGCG-39. <br /> ature dsRNA expression. We added ;30 synchronized L4- <br /> stage animals well, allowed worms develop adults, and <br /> then added 5-&#64258;uorodexoyuridine (FUdR) solution &#65533;?nal concen-                    Supporting Information <br /> tration 0.1 mg/ml. Worms kept 20 8C, lifespan was <br /> monitored. Worms feeding bacteria carrying vector                   Figure S1. A Subset Longevity-Promoting RNAi Clones Are <br /> targeting eri-1 used negative controls. At 96 wells            Expressed Neuronal Metabolic Tissues. <br /> empty vector control included. At time                     Found doi:10.1371/journal.pgen.0030056.sg001 (16.2 MB AI). <br /> control worms dead, containing different RNAi <br /> bacteria scored live worms. RNAi wells live worms                  Figure S2. Candidate Longevity Genes Constitute Diverse Novel <br /> were observed scored positives. RNAi clones scored                Functional Groups <br /> as positive &#65533;?rst second passes screening (Figure S1) were <br />                                                                                     The 64 RNAi clones identi&#65533;?ed classi&#65533;?ed broad functional <br /> retested duplicate pass using conventional <br />                                                                                     groups: chaperones protein turnover (6.3%); metabolism and <br /> longitudinal RNAi lifespan assays (see below). <br />                                                                                     mitochondria (15.6%); RNA, transcription factors, nuclear <br />    RNAi lifespan assay. RNAi bacteria prepared described <br />                                                                                     chromatin (26.6%); signaling (21.9%); traf&#65533;?cking (4.7%); protein <br /> above. daf-2, eri-1, daf-16 RNAi clones included positive <br />                                                                                     synthesis (15.6%); unknown (9.4%). <br /> and negative controls blind fashion conventional lifespan <br /> assays (daf-2 RNAi construct kindly provided M. Vidal, Harvard                   Found doi:10.1371/journal.pgen.0030056.sg002 (829 KB AI). <br /> Medical School, Boston, Massachusetts, United States). Synchronous <br /> L4/young adult stage eri-1(mg366) animals treated FUdR                    Figure S3. Survival Curves Novel Classes Lifespan Regulators <br /> prepared placed 6-well plates containing positive               Survival eri-1(mg366) adult worms &#65533;?rst day adulthood <br /> RNAi clones controls duplicate. Approximately 40&#8211;60 adult                    (day 0) fed control vector (blue line), daf-2(RNAi) (red line), RNAi <br /> animals scored (two wells RNAi clone,                 genes identi&#65533;?ed screen increased mean lifespan. <br /> two biological replicates). The animals kept 20 8C scored               Representative genes protein synthesis RNA and <br /> every days gentle prodding platinum wire test                  chromatin class shown. <br /> viability. To ensure continued ef&#65533;?cacy RNAi knockdown,                       Found doi:10.1371/journal.pgen.0030056.sg003 (958 KB AI). <br /> animals fed freshly induced RNAi bacteria &#65533;?ve seven <br /> days. Lifespan de&#65533;?ned &#65533;?rst day adulthood (adult lifespan &#188;             Figure S4. Representative DAF-16-GFP Localization Phenotypes <br /> 0) scored dead. Worms died protruding/                 Under &#8216;&#8216;normal&#8217;&#8217; conditions DAF-16 predominantly cytoplasmic <br /> bursting vulva, bagging, crawling agar censored from <br />                                                                                     (score &#188; 0). Under conditions stress DAF-16 translocated from <br /> the analysis. For epistasis analysis, RNAi lifespan assays were <br />                                                                                     cytoplasm nucleus (scores 1&#8211;3). <br /> performed described above, daf-16(mgDf47);eri- <br /> 1(mg366) worms used.                                                           Found doi:10.1371/journal.pgen.0030056.sg004 (3.7 MB AI). <br />    Statistical analysis. Statistical analyses performed using SPSS <br /> software (http://www.spss.com). The survival experience RNAi-               Figure S5. Two-<span id='am-180' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-181'  ></span><span id='am-182' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-183' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-184' property="rdfs:label" content="hybrid" datatype="xsd:string"></span>Hybrid</span> Interaction Map 64-Candidate <br /> treated population compared population treated                  Longevity Genes <br /> with control RNAi using log rank test. A p-value ,0.05                      Candidate genes identi&#65533;?ed shown red outline. This interaction <br /> considered signi&#65533;?cantly different control.                                  map generated using Interactome query page M. Vidal&#8217;s <br />    Arrested larvae survival. Synchronized L1-stage eri-1(mg366) eri-             research group (http://vidal.dfci.harvard.edu). <br /> 1(mg366);daf-16(mgDf47) animals placed RNAi clones                     Found doi:10.1371/journal.pgen.0030056.sg005 (284 KB AI). <br /> induce larval arrest phenotypes 20 8C. To ensure animals <br /> bypassing lethality did reproduce plates moved 25 8C                Table S1. Phenotypes Associated Candidate RNAi Clones <br /> to exploit temperature-sensitive sterility associated eri-                 Adult expression patterns identi&#65533;?ed study highlighted in <br /> 1(mg366). The wells qualitatively monitored days                 red determined stated Materials Methods. <br /> arrested larvae survival.                                                           Developmental RNAi phenotypes expression patterns other <br />    Fluorescence microscopy assays. Synchronized L1-stage animals                 genes previously determined [40]. All, cell types; C, <br /> freshly egg-prepped embryos carrying integrated transgenes sod-                 coelomocytes; E, excretory cell; H, hypodermis; I, intestine; M, muscle; <br /> 3p:: g fp, daf-16p::daf-16- g fp, myo-3p::Q40yfp placed onto <br />                                                                                     N, nervous system; P, pharynx; R, reproductive system; RG, renal <br /> RNAi bacteria described above. The &#64258;uorescence intensity each <br />                                                                                     gland; U, unidenti&#65533;?ed cell types. Homology indicated shading <br /> population monitored days following RNAi <br />                                                                                     BLASTP e-values ,10 6 related worm (Cb), yeast (Sc), &#64258;y (Dm), <br /> treatment. <br />                                                                                     mouse (Mn), human (Hs). sod-3p:: g fp expression determined <br />    L4 fat accumulation. Nile Red experiments performed as <br />                                                                                     independent replicates. Adult fat accumulation was <br /> previously described [66] L4-stage eri-1(mg366) animals <br />                                                                                     determined Nile Red staining. Increased Nile Red staining is <br /> were fed RNAi clones induced plates containing 5 mM IPTG. <br />                                                                                     indicated dark shading, decreased Nile Red staining is <br />    Promoter fusions. Promoter elements ampli&#65533;?ed PCR from <br /> wild-type genomic DNA fused RFP [67]. The transcriptional                    indicated light shading. Clones decreased polyQ aggregation <br /> reporters injected gonads wild-type adult                     adults compared vector controls indicated shading. <br /> hermaphrodites. Transgenic animals harboring extrachromoso-                     Found doi:10.1371/journal.pgen.0030056.st001 (205 KB DOC). <br /> mal arrays imaged. <br />    Primer pairs used amplify promoter elements. The primer pairs                 Table S2. daf-16(mgDf47);eri-1(mg366) Lifespan Epistasis <br /> used amplify promoter elements follows:                                   The mean lifespan shown days. The standard error mean <br />  <br />        PLoS Genetics | www.plosgenetics.org                                  0485                                      April 2007 | Volume 3 | Issue 4 | e56 <br />                                                                                                                                  Essential Longevity Genes C. elegans <br />  <br /> lifespan shown days. The median lifespan time                            technical assistance, engaging discussions, critical reading of <br /> 25% individuals remain alive shown days. The total number                         manuscript; Xiaoyun Wu, Annie Lee Conery, Alexander Soukas, <br /> of individuals scored shown (followed number individuals                       Sascha Russel, Justine <span id='am-141' about='obo:NCBITaxon_927630' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_57614 obo:NCBITaxon_57612 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-142' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-143'  ></span><span id='am-144' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-145' property="rdfs:label" content="Melo" datatype="xsd:string"></span>Melo</span> critical reading manuscript; <br /> censored bursting, bagging, crawling agar). The p-                        members Ruvkun Kaplan laboratories scienti&#65533;?c <br /> value log rank test comparing RNAi treatment populations                          discussions. We thankful John Kim, Harrison Gabel, Ravi <br /> the control population shown. Percent increase mean lifespan                       Kamath identi&#65533;?cation eri-1(mg366) lethal RNAi clones. We <br /> RNAi treated populations compared population fed vector                              thank Brian Kennedy suggesting stress pathways regulate <br /> control shown.                                                                           upstream ORF translation. <br /> Found doi:10.1371/journal.pgen.0030056.st002 (167 KB DOC).                                  Author contributions. SPC GR conceived designed the <br />                                                                                             experiments. SPC performed experiments, analyzed data, <br />                                                                                             contributed reagents/materials/analysis tools, wrote paper. <br /> Acknowledgments                                                                                Funding. This work supported grants NIH F32- <br /> Some strains provided Caenorhabditis Genetics Center,                           AG026207 SPC R01-AG016636 GR. <br /> which funded National Institutes Health (NIH) National                            Competing interests. The authors declared competing <br /> Center Research Resources. We thank David J. Simon                                  interests exist. <br />  <br /> References                                                                                  25. Partridge L, Gems D, Withers DJ (2005) Sex death: What the <br /> 1. Finch CE, Ruvkun G (2001) The genetics aging. Annu Rev Genomics                           connection? Cell 120: 461&#8211;472. <br />     Hum Genet 2: 435&#8211;462.                                                                   26. Howitz KT, Bitterman KJ, Cohen HY, Lamming DW, Lavu S, et al. (2003) <br /> 2. Kenyon C, Chang J, Gensch E, Rudner A, Tabtiang R (1993) A C. elegans                        Small molecule activators sirtuins extend <span id='am-26' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-41'  ></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> <br />     mutant lives twice long wild type. Nature 366: 461&#8211;464.                          lifespan. Nature 425: 191&#8211;196. <br /> 3. Morris JZ, Tissenbaum HA, Ruvkun G (1996) A phosphatidylinositol-3-OH                    27. Wood JG, Rogina B, Lavu S, Howitz K, Helfand SL, et al. (2004) Sirtuin <br />     kinase family member regulating longevity diapause Caenorhabditis                    activators mimic caloric restriction delay ageing metazoans. Nature <br />     elegans. Nature 382: 536&#8211;539.                                                               430: 686&#8211;689. <br /> 4. Kawano T, Kataoka N, Abe S, Ohtani M, Honda Y, et al. (2005) Lifespan                    28. Lakowski B, Hekimi S (1998) The genetics caloric restriction in <br />     extending activity substances secreted nematode Caenorhabditis                    <span id='am-68' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-69'  ></span><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Caenorhabditis elegans</span>. Proc Natl Acad Sci U S A 95: 13091&#8211;13096. <br />     elegans include dauer-inducing pheromone. Biosci Biotechnol                    29. Houthoofd K, Gems D, Johnson TE, Van&#64258;eteren JR (2007) Dietary <br />     Biochem 69: 2479&#8211;2481.                                                                      restriction nematode Caenorhabditis elegans. Interdiscip Top <br /> 5. Paradis S, Ruvkun G (1998) Caenorhabditis elegans Akt/PKB transduces                         Gerontol 35: 98&#8211;114. <br />     insulin receptor-like signals AGE-1 PI3 kinase DAF-16                       30. Vellai T, Takacs-Vellai K, Zhang Y, Kovacs AL, Orosz L, et al. (2003) <br />     transcription factor. Genes Dev 12: 2488&#8211;2498.                                              Genetics: In&#64258;uence <span id='am-165' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-166' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-167' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-168'  ></span><span id='am-169' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>TOR</span> kinase lifespan C. elegans. Nature 426: 620. <br /> 6. Paradis S, Ailion M, Toker A, Thomas JH, Ruvkun G (1999) A PDK1                          31. Wolff S, Dillin A (2006) The trifecta aging Caenorhabditis elegans. Exp <br />     homolog necessary suf&#65533;?cient transduce AGE-1 PI3 kinase signals                    Gerontol 41: 894&#8211;903. <br />     regulate diapause Caenorhabditis elegans. Genes Dev 13: 1438&#8211;1452.              32. Mair W, Goymer P, Pletcher SD, Partridge L (2003) Demography dietary <br /> 7. Dillin A, Crawford DK, Kenyon C (2002) Timing requirements insulin/                      restriction death Drosophila. Science 301: 1731&#8211;1733. <br />     IGF-1 signaling C. elegans. Science 298: 830&#8211;834.                                    33. Feng J, Bussiere F, Hekimi S (2001) Mitochondrial electron transport a <br /> 8. Lee SS, Kennedy S, Tolonen AC, Ruvkun G (2003) DAF-16 target genes                      key determinant life span Caenorhabditis elegans. Dev Cell 1: 633&#8211; <br />     control C. elegans life-span metabolism. Science 300: 644&#8211;647.                          644. <br /> 9. Murphy CT, McCarroll SA, Bargmann CI, Fraser A, Kamath RS, et al. (2003)                 34. Dillin A, Hsu AL, Arantes-Oliveira N, Lehrer-Graiwer J, Hsin H, et al. (2002) <br />                                                                                                 Rates behavior aging speci&#65533;?ed mitochondrial function during <br />     Genes act downstream DAF-16 in&#64258;uence lifespan of <br />                                                                                                 development. Science 298: 2398&#8211;2401. <br />     Caenorhabditis elegans. Nature 424: 277&#8211;283. <br />                                                                                             35. Lee SS, Lee RY, Fraser AG, Kamath RS, Ahringer J, et al. (2003) A systematic <br /> 10. Lin K, Dorman JB, Rodan A, Kenyon C (1997) daf-16: An HNF-3/forkhead <br />                                                                                                 RNAi screen identi&#65533;?es critical role mitochondria C. elegans <br />     family member function double life-span Caenorhabditis <br />                                                                                                 longevity. Nat Genet 33: 40&#8211;48. <br />     elegans. Science 278: 1319&#8211;1322. <br />                                                                                             36. Larsen PL, Clarke CF (2002) Extension life-span Caenorhabditis <br /> 11. Ogg S, Paradis S, Gottlieb S, Patterson GI, Lee L, et al. (1997) The Fork head <br />                                                                                                 elegans diet lacking coenzyme Q. Science 295: 120&#8211;123. <br />     transcription factor DAF-16 transduces insulin-like metabolic longev- <br />                                                                                             37. Hamilton B, Dong Y, Shindo M, Liu W, Odell I, et al. (2005) A systematic <br />     ity signals C. elegans. Nature 389: 994&#8211;999. <br />                                                                                                 RNAi screen longevity genes C. elegans. Genes Dev 19: 1544&#8211;1555. <br /> 12. Bluher M, Kahn BB, Kahn CR (2003) Extended longevity mice lacking the <br />                                                                                             38. Hansen M, Hsu AL, Dillin A, Kenyon C (2005) New genes tied endocrine, <br />     insulin receptor adipose tissue. Science 299: 572&#8211;574. <br />                                                                                                 metabolic, dietary regulation lifespan Caenorhabditis elegans <br /> 13. Tatar M, Kopelman A, Epstein D, Tu MP, Yin CM, et al. (2001) A mutant                       genomic RNAi screen. PLoS Genet 1: e17. doi:10.1371/journal.pgen. <br />     Drosophila insulin receptor homolog extends life-span impairs                      0010017 <br />     neuroendocrine function. Science 292: 107&#8211;110.                                          39. Larsen PL, Albert PS, Riddle DL (1995) Genes regulate both <br /> 14. Clancy DJ, Gems D, Harshman LG, Oldham S, Stocker H, et al. (2001)                          development longevity Caenorhabditis elegans. Genetics 139: 1567&#8211; <br />     Extension life-span loss CHICO, Drosophila insulin receptor                      1583. <br />     substrate protein. Science 292: 104&#8211;106.                                                40. WormBase Web site, http://www.wormbase.org, release WS170, 01.19.2007. <br /> 15. Holzenberger M, Dupont J, Ducos B, Leneuve P, Geloen A, et al. (2003) IGF-              41. Kennedy S, Wang D, Ruvkun G (2004) A conserved siRNA-degrading RNase <br />     1 receptor regulates lifespan resistance oxidative stress mice.                   negatively regulates RNA interference C. elegans. Nature 427: 645&#8211;649. <br />     Nature 421: 182&#8211;187.                                                                    42. Wolkow CA, Kimura KD, Lee MS, Ruvkun G (2000) Regulation C. elegans <br /> 16. Boehm M, Slack F (2005) A developmental timing microRNA target                      life-span insulinlike signaling nervous system. Science 290: 147&#8211; <br />     regulate life span C. elegans. Science 310: 1954&#8211;1957.                                   150. <br /> 17. Hsu AL, Murphy CT, Kenyon C (2003) Regulation aging age-related                  43. Kenyon C (2005) The plasticity aging: Insights long-lived mutants. <br />     disease DAF-16 heat-shock factor. Science 300: 1142&#8211;1145.                            Cell 120: 449&#8211;460. <br /> 18. Wolff S, Ma H, Burch D, Maciel GA, Hunter T, et al. (2006) SMK-1,                    44. Harris TE, Chi A, Shabanowitz J, Hunt DF, Rhoads RE, et al. (2006) mTOR- <br />     essential regulator DAF-16-mediated longevity. Cell 124: 1039&#8211;1053.                      dependent stimulation association eIF4G eIF3 insulin. <br /> 19. Oh SW, Mukhopadhyay A, Svrzikapa N, Jiang F, Davis RJ, et al. (2005) JNK                    Embo J 25: 1659&#8211;1668. <br />     regulates lifespan Caenorhabditis elegans modulating nuclear                      45. Tsukiyama-Kohara K, Poulin F, Kohara M, DeMaria CT, Cheng A, et al. <br />     translocation forkhead transcription factor/DAF-16. Proc Natl Acad                       (2001) Adipose tissue reduction mice lacking translational inhibitor <br />     Sci U S A 102: 4494&#8211;4499.                                                                   4E-BP1. Nat Med 7: 1128&#8211;1132. <br /> 20. Tissenbaum HA, Guarente L (2001) Increased dosage sir-2 gene                       46. Pestova TV, Kolupaeva VG (2002) The roles individual eukaryotic <br />     extends lifespan Caenorhabditis elegans. Nature 410: 227&#8211;230.                            translation initiation factors ribosomal scanning initiation codon <br /> 21. Hsin H, Kenyon C (1999) Signals reproductive regulate                   selection. Genes Dev 16: 2906&#8211;2922. <br />     lifespan C. elegans. Nature 399: 362&#8211;366.                                            47. <span id='am-83' about='obo:NCBITaxon_9596' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-84' property="oboInOwl:hasAlternativeId" content="NCBITaxon:317028" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasAlternativeId" content="NCBITaxon:445518" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="Pan" datatype="xsd:string"></span><span id='am-88'  ></span><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasExactSynonym" content="chimpanzees" datatype="xsd:string"></span>Pan</span> KZ, Palter JE, Rogers AN, Olsen A, Chen D, et al. (2007) Inhibition of <br /> 22. Arantes-Oliveira N, Apfeld J, Dillin A, Kenyon C (2002) Regulation life-                 mRNA translation extends lifespan Caenorhabditis elegans. Aging Cell 6: <br />     span germ-line stem cells Caenorhabditis elegans. Science 295: 502&#8211;                   111&#8211;119. <br />     505.                                                                                    48. Hansen M, Taubert S, Crawford D, Libina N, Lee SJ, et al. (2007) Lifespan <br /> 23. Berman JR, Kenyon C (2006) Germ-cell loss extends C. elegans life span                      extension conditions inhibit translation Caenorhabditis elegans. <br />     regulation DAF-16 kri-1 lipophilic-hormone signaling.                     Aging Cell 6: 95&#8211;110. <br />     Cell 124: 1055&#8211;1068.                                                                    49. Pierce SB, Costa M, Wisotzkey R, Devadhar S, Homburger SA, et al. (2001) <br /> 24. Lin SJ, Defossez PA, Guarente L (2000) Requirement NAD SIR2                      Regulation DAF-2 receptor signaling human insulin ins-1, a <br />     life-span extension calorie restriction <span id='am-43' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-44' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-45'  ></span><span id='am-46' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae.                     member unusually large diverse C. elegans insulin gene family. <br />     Science 289: 2126&#8211;2128.                                                                     Genes Dev 15: 672&#8211;686. <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                          0486                                          April 2007 | Volume 3 | Issue 4 | e56 <br />                                                                                                                                 Essential Longevity Genes C. elegans <br />  <br /> 50. Hsu SY, Nakabayashi K, Nishi S, Kumagai J, Kudo M, et al. (2002) Activation            59. Lund J, Tedesco P, Duke K, Wang J, Kim SK, et al. (2002) Transcriptional <br />     orphan receptors hormone relaxin. Science 295: 671&#8211;674.                          pro&#65533;?le aging C. elegans. Curr Biol 12: 1566&#8211;1573. <br /> 51. Syntichaki P, Samara C, Tavernarakis N (2005) The vacuolar H&#254; -ATPase                  60. Vowels JJ, Thomas JH (1992) Genetic analysis chemosensory control of <br />     mediates intracellular acidi&#65533;?cation required neurodegeneration C.                   dauer formation Caenorhabditis elegans. Genetics 130: 105&#8211;123. <br />     elegans. Curr Biol 15: 1249&#8211;1254.                                                      61. Baugh LR, Sternberg PW (2006) DAF-16/FOXO regulates transcription of <br /> 52. Gems D, Riddle DL (2000) Genetic, behavioral environmental                             cki-1/Cip/Kip repression lin-4 C. elegans L1 arrest. Curr Biol <br />     determinants male longevity Caenorhabditis elegans. Genetics 154:                    16: 780&#8211;785. <br />     1597&#8211;1610.                                                                             62. Tercero JA, <span id='am-155' about='obo:NCBITaxon_1118665' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_272259 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_7422 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_272201 obo:NCBITaxon_7400 obo:NCBITaxon_7423 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-156' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-157'  ></span><span id='am-158' property="rdfs:label" content="Espinosa" datatype="xsd:string"></span><span id='am-159' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Espinosa</span> JC, Lacalle RA, Jimenez A (1996) The biosynthetic <br /> 53. Ellis HM, Horvitz HR (1986) Genetic control programmed cell death                    pathway aminonucleoside antibiotic puromycin, deduced from <br />     nematode C. elegans. Cell 44: 817&#8211;829.                                                 molecular analysis pur cluster <span id='am-1' about='obo:NCBITaxon_1883' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_2062 obo:NCBITaxon_201174 obo:NCBITaxon_85003 obo:NCBITaxon_85011'><span id='am-2'  ></span><span id='am-3' property="oboInOwl:hasDbXref" content="PMID:4886644" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="PMID:16560709" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="PMID:11411701" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="Streptomyces" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasAlternativeId" content="NCBITaxon:31945" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="PMID:1371059" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="PMID:10843062" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="PMID:8590678" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1973" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="PMID:9731302" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="PMID:9336904" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="PMID:8934906" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="PMID:15950131" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasAlternativeId" content="NCBITaxon:65506" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Chainia" datatype="xsd:string"></span>Streptomyces</span> alboniger. J Biol <br /> 54. Wang D, Kennedy S, Conte D Jr., Kim JK, Gabel HW, et al. (2005) Somatic                    Chem 271: 1579&#8211;1590. <br />     misexpression germline P granules enhanced RNA interference                  63. Huss M, Ingenhorst G, Konig S, Gassel M, Drose S, et al. (2002) <br />     retinoblastoma pathway mutants. Nature 436: 593&#8211;597.                                       Concanamycin A, speci&#65533;?c inhibitor V-ATPases, binds V(o) <br /> 55. Narbonne P, Roy R (2006) Inhibition germline proliferation C.                    subunit c. J Biol Chem 277: 40544&#8211;40548. <br />     elegans dauer development requires PTEN, LKB1 AMPK signalling.                     64. Breen GA, Miller DL, Holmans PL, Welch G (1986) Mitochondrial DNA of <br />     Development 133: 611&#8211;619.                                                                  <br /> </body></html>