<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Inhibition of HIV-1 reverse transcriptase by RNA aptamers <br /> in <span id='am-31' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-32' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-40'  ></span><span id='am-41' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span>Escherichia coli</span> <br />  <br /> ABSTRACT <br /> A better understanding of aptamer function in bacteria would help to establish simple model systems for screening RNA&#8211;protein <br /> interactions within an intracellular context. <span id='am-47' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-48' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-52'  ></span><span id='am-53' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span>Escherichia</span> coli DNA polymerase I mutants (Pol Its) fail to grow at 37&#176;C unless an <br /> exogenous DNA polymerase such as HIV-1 reverse transcriptase (RT) is expressed within the cell. Here, we show that four RNA <br /> aptamers that inhibit HIV-1 RT in vitro block complementation by HIV-1 RT when expressed in vivo. No other essential <br /> functions are impaired by aptamer expression at either temperature. Intracellular aptamer RNA concentrations from induced <br /> cultures were measured to range from 76 to 180 nM, which is comparable with exogenously expressed HIV-1 RT levels in these <br /> cells. RT polymerase activity was reduced to background levels in cell-free extracts prepared from cultures expressing both <br /> HIV-1 RT and the 70.28 aptamer, compared with extracts from cultures expressing HIV-1 RT alone. Intracellularly expressed <br /> RNA aptamers can thus be used to generate conditional null mutants in bacteria by titrating an essential protein. <br /> Keywords: SELEX; aptamer; RNA&#8211;protein interactions; bioassay; HIV-1 reverse transcriptase <br />  <br />  <br />  <br /> INTRODUCTION                                                           exerts observable influence on the biology of those cells <br />                                                                        (Burke and Nickens 2002). Less is known about aptamer <br /> Treatment of infection by <span id='am-9' about='obo:NCBITaxon_12721' typeof='owl:Thing obo:NCBITaxon_11632 obo:NCBITaxon_11652 obo:NCBITaxon_1 obo:NCBITaxon_35268 obo:NCBITaxon_10239 obo:NCBITaxon_693660 obo:NCBITaxon_327045 obo:NCBITaxon_11646'><span id='am-10'  ></span><span id='am-11' property="oboInOwl:hasBroadSynonym" content="HIV" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="AIDS virus" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Human immunodeficiency virus" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>human immunodeficiency virus</span> <br />                                                                        function in bacterial cells. Whereas translational regulation <br /> (HIV) generally includes small molecule inhibitors of the <br />                                                                        by small molecules has been observed both in engineered <br /> viral reverse transcriptase (RT). However, drug-resistant <br />                                                                        (Werstuck and Green 1998) and natural systems (Nahvi et <br /> forms of RT are selected by prolonged treatment, prompt- <br />                                                                        al. 2002; Winkler et al. 2002a,b), enzyme antagonism by <br /> ing widespread efforts to generate new anti-RT drugs. Both <br />                                                                        aptamers within bacteria has not been reported. Bacteria <br /> RNA and ssDNA aptamers that bind HIV-1 RT with high <br />                                                                        offer several advantages for studying RNA&#8211;protein interac- <br /> affinity have been identified through the SELEX protocol <br />                                                                        tions, including ease of genetic manipulation and lower cost <br /> (Tuerk et al. 1992; Schneider et al. 1995; Burke et al. 1996; <br />                                                                        of maintaining cell lines. At the same time, a number of <br /> Andreola et al. 2001). When any of several different RNA <br />                                                                        characteristics of the bacterial microenvironment could af- <br /> aptamers was expressed in human T293 cells, the suscepti- <br />                                                                        fect aptamer function, including the short average lifetime <br /> bility of these cells to HIV infection was reduced, and the <br />                                                                        of bacterial mRNA transcripts, counterion concentrations, <br /> infectivity of <span id='am-1' about='obo:NCBITaxon_10239' typeof='owl:Thing obo:NCBITaxon_1'><span id='am-2'  ></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="Vira" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Viruses" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="viruses" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Viridae" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>viruses</span> produced by infected T293 cells is re- <br />                                                                        and macromolecular constituents. A better understanding <br /> duced 90%&#8211;99% compared with cells not expressing <br />                                                                        of aptamer function in bacteria would help to establish the <br /> aptamers (Joshi and Prasad 2002). Human T-lymphoid cell <br />                                                                        extent to which such model systems can be used in rapid <br /> lines expressing the 33-nt TPK1.1 aptamer also demon- <br />                                                                        screens of intracellular aptamer&#8211;protein interactions. <br /> strated intracellular inhibition of RT and protection against <br />                                                                           <span id='am-26' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-27' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-30'  ></span>This</span> study utilizes Escherichia coli strain BK148, which <br /> HIV-1 infection (Chaloin et al. 2002). <br />                                                                        carries a temperature-sensitive DNA polymerase I (polA12) <br />    The aptamer-RT interaction represents one of more than <br />                                                                        that allows normal growth at 30&#176;C but restricts the <span id='am-63' about='obo:NCBITaxon_forma' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="forma" datatype="xsd:string"></span>forma</span>- <br /> a dozen examples wherein an aptamer retains its target- <br />                                                                        tion of isolated colonies at 37&#176;C (Witkin and Roegner- <br /> binding function when expressed in eukaryotic cells and <br />                                                                        Maniscalco 1992). Expression of an exogenous DNA poly- <br />                                                                        merase such as HIV-1 RT (Kim and Loeb 1995; Kim et al. <br />   Reprint requests to: Donald H. Burke, Department of Chemistry, In-   1996) or rat DNA polymerase &#9252; (Sweasy and Loeb 1992) <br /> diana University, Bloomington, IN 47405-7102, USA; e-mail: dhburke@    restores replication and normal growth at 37&#176;C. Small mol- <br /> indiana.edu; fax: (812) 855-6590. <br />   Article and publication are at http://www.rnajournal.org/cgi/doi/    ecule nucleoside prodrug inhibitors of HIV-1 RT re-estab- <br /> 10.1261/rna.5550103.                                                   lished temperature sensitivity of the Pol Its/HIV-1 RT strain <br />  <br /> RNA (2003), 9:1029&#8211;1033. Published by Cold Spring Harbor Laboratory Press. Copyright &#169; 2003 RNA Society.                      1029 <br />  Nickens et al. <br />  <br />  <br />  <br /> at 37&#176;C (Kim and Loeb 1995; Kim et al. 1996), demonstrat-                dilutions of freshly saturated cultures resulted in resatura- <br /> ing that the DNA polymerase activity of the RT is respon-                tion at all dilutions assayed when HIV-1 RT was expressed <br /> sible for growth at the restrictive temperature. We demon-               in the strain. In contrast, at least three to four orders of <br /> strate here that RT complementation of the Pol I defect is               magnitude higher initial density of BK148 was required for <br /> blocked by inhibitory RNA aptamers, creating a conditional               growth in culture when the strain did not express HIV-1 RT <br /> null phenotype suitable for evaluating libraries of HIV-1 RT             (Table 1). <br /> mutants for aptamer-resistant variants. <br />  <br /> RESULTS                                                                  Temperature sensitivity restored by RNA aptamers <br />                                                                            Four aptamers were cloned individually into the expression <br /> Complementation of Pol Its by HIV-1 RT <br />                                                                            vector pERLAC (Fig. 1). Aptamers 70.28 and 70.24 are 118- <br /> Plasmid pRT5 was constructed to direct low-level expres-                   nt isolates selected to bind HIV-1 RT, whereas 70.28min is <br /> sion of HIV-1 RT (Fig. 1A). When duplicate concentric                      the 34-nt pseudoknot core of isolate 70.28 (Burke et al. <br /> spiral streak plates of E. coli strain BK148 (Pol Its) were                1996). Aptamer TPK1.1 is a related 33-nt pseudoknot <br /> incubated at 30&#176;C and 37&#176;C, cells expressing HIV-1 RT                      aptamer (Tuerk et al. 1992), which has been reported to <br /> from plasmid pRT5 formed isolated colonies out to the edge                 bind HIV-1 RT in solution with enhanced affinity (Kd &#8776; 25 <br /> of the spiral at both temperatures, whereas cells from con-                pM; Kensch et al. 2000). Plasmid-derived transcripts all <br /> trol cultures with plasmid pHSG576 formed isolated colo-                   contain a 28-nt 5&#11032; leader and a 50-nt 3&#11032; tail, 40 nt of which <br /> nies only at the permissive temperature (Fig. 2A,B). Growth                form the stable rrnB transcriptional terminator (Fig. 1). <br /> at the center of spiral streak plates is observed irrespective of          This design was intended to minimize nucleolytic degrada- <br /> temperature or plasmid. These results are consistent with                  tion and to maximize the probability that nascent tran- <br /> previously observed density-dependent growth (Kim and                      scripts would present the aptamers to the intracellular en- <br /> Loeb 1995; Kim et al. 1996; Kim 1997). End-point serial                    vironment in their native conformations. Nitrocellulose <br />                                                                                                 filter-binding assays of transcripts synthe- <br />                                                                                                 sized in vitro with these flanking se- <br />                                                                                                 quences gave Kd values (KdTPK1.1 = 1.1 <br />                                                                                                 nM; Kd70.28 = 5.3 nM; Kd70.28min = 25 <br />                                                                                                 nM; Kd70.24 = 43 nM) that are similar to <br />                                                                                                 those originally reported for the core <br />                                                                                                 aptamers (Tuerk et al. 1992; Burke et al. <br />                                                                                                 1996). <br />                                                                                                     Aptamer expression <span id='am-71' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-72' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>plasmids</span> were <br />                                                                                                 introduced into BK148 harboring plas- <br />                                                                                                 mid pRT5. RT-expressing strains carry- <br />                                                                                                 ing the empty aptamer cassette (pERLAC) <br />                                                                                                 grew to the edges of spiral streak plates <br />                                                                                                 at both 30&#176;C and 37&#176;C, indicating that <br />                                                                                                 HIV-1 RT remains active in the pres- <br />                                                                                                 ence of transcripts from pERLAC. Cells <br />                                                                                                 expressing aptamer 70.28min (Fig. 2C) <br />                                                                                                 or any of the other aptamers (<span id='am-66' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-68'  ></span><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> not <br />                                                                                                 shown) grew normally at 30&#176;C, but were <br />                                                                                                 restricted to the high-density centers of <br />                                                                                                 the spiral streak at 37&#176;C. Similarly, end- <br />                                                                                                 point dilution assays demonstrated that <br />                                                                                                 these strains required at least three to four <br />                                                                                                 orders of magnitude higher initial cell <br />                                                                                                 density for growth in culture than did <br /> FIGURE 1. (A) Sequences in plasmid pRT5 controlling translation of HIV-1 RT expression. cells expressing HIV-1 RT alone (Table 1), <br /> Restriction sites used in cloning are indicated. Translational start sites are indicated with <br /> arrows; Shine-Dalgarno sequence for translation of HIV-1 RT is double-underlined. (B) suggesting little or no HIV-1 RT activity <br /> Aptamer RNA expressed from construct p70.28min. Nucleotides corresponding to EcoRI and in cells expressing inhibitory RNA ap- <br /> SacI sites used in cloning this and other aptamers are underlined. AptQuant oligonucleotide tamers. HIV-1 RT complementation in <br /> used as a probe for aptamer detection is shown adjacent to its hybridization target, which is <br />                                                                                                 BK148 at 37&#176;C is thus blocked by indi- <br /> present in all aptamer expression constructs. (C) Sequences of aptamer inserts used in this <br /> study. For aptamers 70.28 and 70.24, the notations x and y represent the following sequences: vidual expression of aptamer 70.24, 70.28, <br /> (x) 5&#11032;-gggaaaagcgaaucauacacaaga-3&#11032;, (y) 5&#11032;-gggcauaagguauuuaauuccaua-3&#11032;.                         70.28 min, or TPK1.1. <br />  <br /> 1030       RNA, Vol. 9, No. 9 <br />                                                                                     HIV reverse transcriptase RNA aptamers in bacteria <br />  <br />  <br />  <br />                                                                          Consistent with this interpretation, RT activity in cell-free <br />                                                                          extracts was reduced to the HIV RT&#8722; background level in <br />                                                                          cultures expressing both RT and aptamers (Table 3). <br />  <br />  <br />                                                                          DISCUSSION <br />                                                                          HIV-1 RT can be inhibited by RNA aptamers inside bacte- <br />                                                                          rial cells, as reported by loss of growth in a DNA polymerase <br />                                                                          I complementation assay. By extension, it should be pos- <br />                                                                          sible to monitor aptamer&#8211;protein interactions for other es- <br />                                                                          sential proteins by assessing the growth of conditional null <br />                                                                          phenotypes. The four RNA aptamers in this study are ex- <br />                                                                          pressed as components of small transcripts (94&#8211;208 nt). <br />                                                                          They all inhibit HIV-1 RT polymerase activity in vitro, and <br />                                                                          they all prevent HIV-1 RT complementation of BK148 at <br />                                                                          nonpermissive temperatures both on plates and in serial <br />                                                                          dilution assays (Fig. 2; Table 1). Aptamer-expressing BK148 <br />                                                                          cultures always grew at 30&#176;C, as did an isogenic Pol+ strain <br />                                                                          (JS295) carrying any of the aptamer expression constructs at <br />                                                                          30&#176;C or 37&#176;C. Thus, no other essential cellular function is <br />                                                                          significantly inhibited by these aptamers at either tempera- <br />                                                                          ture. <br /> FIGURE 2. Effect of aptamer expression on complementation of the <br /> Pol Its mutation in HIV-1 RT. Photographs show concentric dilution          The binding affinity and the relative expression levels of <br /> plates of strain BK148 with the following plasmids: (A) pHSG576, (B)     an aptamer and its target protein can profoundly influence <br /> pRT5, (C) pRT5 and p70.28.                                               the biological effects of aptamer expression (Burke and <br />                                                                          Nickens 2002). For high-affinity interactions, aptamer RNA <br />                                                                          concentration is more important than the precise value of <br /> Quantification of intracellular aptamer concentrations <br />                                                                          Kd in determining the fraction of RT bound by aptamer. <br /> Quantitative measurements of intracellular aptamer RNA                   Western blot analysis of BK148 cells carrying plasmid <br /> levels can help delineate the parameters that govern in vivo             pHIVRT (essentially identical to pRT5) gave estimates of <br /> inhibition. To this end, total cellular RNA (RNAT) was                   &#8764;100 copies of HIV-1 RT per induced cell (Kim and Loeb <br /> isolated from each strain, applied to a membrane, and                    1995; Kim 1997). Assuming a mean cell volume of 1.5 &#181;m3/ <br /> probed with radiolabeled oligonucleotides. Aptamer RNA                   cell (Kubitschek and Friske 1986), this corresponds to a <br /> was consistently detected using a probe complementary to                 concentration of 110 nM for the RT. Intracellular concen- <br /> the 5&#11032; leader of the aptamer expression cassette common to               trations of aptamer and control RNA were determined here <br /> all constructs (Fig. 1B). Comparison with purified RNA                   to range between 76 and 180 nM (Table 2). Given the Kd <br /> standards revealed intracellular concentrations ranging                  values reported here for these aptamers in the context of <br /> from 76 &#177; 10 nM to 180 &#177; 47 nM (70&#8211;160 copies per cell)                  their 5&#11032; and 3&#11032; flanking sequences (1&#8211;40 nM), approxi- <br /> for all four aptamers (Table 2). These values are similar to             mately 50%&#8211;98% of the expressed HIV-1 RT is expected to <br /> RT concentrations measured previously (100 copies/cell;                  be sequestered by aptamers. It is not known how much <br /> Kim and Loeb 1995). Control RNA <br /> expressed from the aptamer-free vec- <br /> <span id='am-58' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-59' property="rdfs:label" content="Tor" datatype="xsd:string"></span><span id='am-60'  ></span><span id='am-61' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>tor</span> (pERLAC) was measured to be at        TABLE 1. Effects of aptamer expression on growth on HIV-1 RT complemented pol Its mutants <br /> a concentration of 179 &#177; 11 nM;           in liquid culture end-point dilution assays <br /> thus, the growth of cells expressing                                                               Dilution from initial culturesa <br /> this plasmid at 37&#176;C cannot be attrib- <br />                                           Plasmids                 RT gene/aptamer 10&#8722;1 10&#8722;3 10&#8722;4 10&#8722;5 10&#8722;6 10&#8722;7 10&#8722;8 <br /> uted to failure to accumulate RNA, <br /> but rather to the absence of aptamer      pHSG576                     None/none           +       +       +       &#177;        &#8722;       &#8722;      &#8722; <br /> in the expressed RNA. As these RNA        pRT5                        Yes/none            +       +       +       +        +       +      + <br /> concentrations are well in excess of      pRT5 and pERLAC             Yes/vector          +       +       +       +        +       +      + <br />                                           pRT5 and PERTPK1.1          Yes/TPK1.1          +       +       +       &#8722;        &#8722;       &#8722;      &#8722; <br /> the measured Kd values for the            pRT5 and p70.24             Yes/70.24           +       +       +       &#8722;        &#8722;       &#8722;      &#8722; <br /> aptamer-RT interactions, inhibitory       pRT5 and p70.28             Yes/70.28           +       +       +       +        &#8722;       &#8722;      &#8722; <br /> RNA concentrations are therefore          pRT5 and p70.28min          Yes/28min           +       +       +       +        &#8722;       &#8722;      &#8722; <br /> high enough to sequester most or all      a <br />                                             Growth at 37&#176;C for 48 hr following dilution from initial cultures at densities of 8 &#215; 108 cfu/mL. <br /> of the HIV-1 RT produced in the cell. <br />  <br />                                                                                                             www.rnajournal.org          1031 <br />  Nickens et al. <br />  <br />  <br />  <br />                                                                         mented with chloramphenicol (30 &#181;g/mL), carbenicillin (200 &#181;g/ <br />  TABLE 2. Steady-state concentration (nM) of aptamer RNAa <br />                                                                         mL), tetracycline (10 &#181;g/mL), or IPTG (0.1 mM) as appropriate. <br />  Plasmids present            Normalized to           Normalized to <br />  in BK148                     cell number             rrnB signal <br />                                                                         Plasmid construction <br />  pERLAC + pRT5                  174 &#177; 14                179 &#177; 11 <br />  pERTPK1.1 + pRT5               108 &#177; 22                112 &#177; 17 <br />                                                                         All plasmid constructs were confirmed by DNA sequencing. Vec- <br />  p70.24 + pRT5                  200 &#177; 52                180 &#177; 47        tor pERLAC is a derivative of pUHE-21.2 (Lanzer and Bujard <br />  p70.28 + pRT5                   75 &#177; 10                 76 &#177; 10        1988), a transcription-translation vector that is itself a derivative of <br />  p28min + pRT5                   91 &#177; 6                 105 &#177; 7         the ColE1 replicon pBR322. Chloramphenicol resistance was re- <br />  a <br />                                                                         moved by divergent PCR using oligonucleotides pUHE-PCR1 (5&#11032;- <br />   All concentrations assume a cell volume of 1.5 &#181;m3.                   GGGGTACCCGGGATCTAAGTATCATTGTT-3&#11032;) and pUHE- <br />                                                                         PCR-2 (5&#11032;-GGGGTACCTGCAGAGCTCATAAAACGAAAGGCT <br />                                                                         CAGTC-3&#11032;). Digestion with KpnI (underlined) and ligation <br /> DNA polymerase activity the HIV-1 RT needs to provide                   yielded a 2502-bp plasmid designated pER1 that retains a &#9252;-lac- <br />                                                                         tamase gene for selection. The lacO1 operator was restored using <br /> for complementation of the Pol Its phenotype, or inversely, <br />                                                                         synthetic oligonucleotides to create plasmid pERLAC. Transcrip- <br /> what fraction of the RT needs to be blocked to inhibit <br />                                                                         tion of aptamers cloned into pERLAC is driven by a heterologous <br /> growth. However, the lack of RT activity in cell-free extracts          phage T7 promoter A1 (which utilizes E. coli RNA polymerase), <br /> from cells expressing both HIV-1 RT and aptamer relative                combined with two lac operators (PA1-04/03) to generate transcripts <br /> to cells expressing HIV-1 RT only (Table 3), suggests that              of the form shown in Figure 1B. <br /> most of the RT is bound by aptamer.                                        The gene for HIV-1 RT was amplified by PCR from plasmid <br />    Bioassays for screening new drugs have a long history,               p83-2 (AIDS Research and Reference Reagent Program), which <br /> and their use for testing aptamer efficacy is a logical devel-          carries the pol gene from HIV-1 clinical isolate NY5 (Adachi et al. <br /> opment. Whereas aptamers against HIV-1 RT have been                     1986; Gibbs et al. 1994). This fragment was cloned along with <br /> available for 10 yr, their bioactivity has only recently been           appropriate oligonucleotides into HindIII&#8211;BamHI-digested <br /> reported. As noted above, RNA aptamers show potent anti-                pHSG576, a low-copy number plasmid that utilizes a Pol I inde- <br />                                                                         pendent pSC101 origin of replication (Takeshita et al. 1987), to <br /> viral activity in human cell culture. We have speculated that <br />                                                                         create the HIV-1 RT expression plasmid pRT5 (Fig. 1A). This <br /> HIV-1 RT may not be able to mutate to aptamer resistant <br />                                                                         construct is nearly identical to another low-level RT expression <br /> forms without adversely affecting viral replication activity            vector, pHIVRT (Kim and Loeb 1995). Transcription of the HIV-1 <br /> (Burke et al. 1996). This hypothesis appears to have been               RT gene is under the control of the lacP/O from the pHSG576 <br /> born out in at least one instance (Fisher et al. 2002). The             vector. Plasmid copy number for pRT5 was steady under the con- <br /> system described here offers for the first time a methodol-             ditions used here, whereas copy number of the aptamer-encoding <br /> ogy to test this hypothesis by screening libraries of RT mu-            <span id='am-54' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-55' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="vectors" datatype="xsd:string"></span>vectors</span> diminished by 80% in BK148 compared with aptamer <br /> tants for resistance to aptamer inhibitors.                             plasmid levels in the cloning strain NM522. <br />  <br />                                                                         RNA detection and quantification <br /> MATERIALS AND METHODS <br />                                                                         Total RNA (RNAT) was prepared from overnight cultures using a <br />                                                                         MasterPure RNA purification kit (Epicentre), then resuspended in <br /> Strains and medium <br />                                                                         ddH2O with RNasin (Promega) at 1 U/&#181;L and stored at &#8722;80&#176;C. <br /> Genetic complementation experiments used the Pol Its E. coli B/r        RNA standards were transcribed in vitro from PCR-amplified <br /> strain BK148 (recA718 polA12 fad751&#11019;Tn10 trpE65 uvrA155 mal&#8722;            templates derived from plasmids pERLAC (for RNA T7-Lac) or <br /> lon&#8722; sulA11; Witkin and Roegner-Maniscalco 1992). E. coli strain        pERTPK1.1 (for RNA T7-TPK1.1). Slot blots were prepared using <br /> NM522 (Stratagene) was used for cloning and for plasmid con-            a Bio-Dot SF microfiltration apparatus with <span id='am-76' about='obo:NCBITaxon_743421' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_50638 obo:NCBITaxon_7438 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7434 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_34725 obo:NCBITaxon_50557'><span id='am-77'  ></span><span id='am-78' property="rdfs:label" content="Zeta" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Zeta</span>-Probe blotting <br /> struction. <span id='am-16' about='obo:NCBITaxon_218032' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_69557 obo:NCBITaxon_69555 obo:NCBITaxon_69556 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-19'  ></span><span id='am-20' property="rdfs:label" content="Luria" datatype="xsd:string"></span>Luria</span> broth and Difco nutrient broth were supple-            membranes using a slight modification of the Epicentre protocol. <br />                                                                         RNAT, or standard RNA was mixed with yeast tRNA, so that each <br />                                                                         slot on the grid received 6 &#181;g of RNA. Cell number per microgram <br />                                                                         of RNAT was determined from optical density of culture assuming <br />  TABLE 3. RT activity in cell-free extracts                             5 &#215; 108 cells&#183;mL&#8722;1 when A600nm = 1.0. Blotted membranes were <br />  Cell-free extracts            Total RT activitya   Relative activity   probed with radiolabeled AptQuant oligonucleotide (Fig. 1A). Af- <br />                                                                         ter washing off nonspecifically bound probe, the blot was dried <br />  BK148 alone (no plasmids)          1.9 &#177; 1.2              1.0          and analyzed using a Molecular Dynamics PhosphorImager and <br />  BK148 (pRT5)                      12.1 &#177; 2.9              6.4          ImageQuant software. Calibration curves for converting Phosphor- <br />  BK148 (pRT5, p70.28)               1.9 &#177; 0.1              1.0 <br />                                                                         Imager pixels to picomoles of target RNA were plotted using the <br />  BL21DE3 (pRT(p))                 253.0 &#177; 46             133.0 <br />                                                                         signal from the standard RNA. A faint signal in RNAT prepared <br />  a <br />   Unit activity defined as fmole of dTTP incorporated into TCA          from BK148 carrying only plasmid pRT5 defined the nonspecific <br />  precipitable material &#11080; min&#8722;1 &#11080; &#181;g protein&#8722;1.                          background associated with the AptQuant probe, and was sub- <br />  Values are reported as the mean &#177; standard deviation reported          tracted from each sample before calculating aptamer concentra- <br />  based on at least three assays. <br />                                                                         tions for each strain. <br />  <br />    After calculating the picomoles of aptamer RNA present in each          <br /> applied sample, blots were stripped and reprobed with the rrnB1               <br /> probe specific for ribosomal RNA, allowing the rRNA to serve as          <br /> an internal loading standard. Molar concentrations of aptamers <br />                                                                              <br /> inside cells were calculated from (1) the number of cells that             <br /> contributed RNAT to a given blot, (2) the calculated moles of          <br /> aptamer RNA detected, and (3) an assumed average cell volume of             <br /> 1.5 &#181;m3 for exponentially growing cultures in nutrient broth (Ku-         <br /> bitschek and Friske 1986). Excluded volume due to intracellular        <br /> molecular crowding was omitted from the calculation.                <br /> Reverse transcriptase assays and Kd determination                         <br /> Duplicate reverse transcriptase assays were performed using cell-           <br /> free extracts prepared from duplicate cultures as described previ-         <br /> ously (Hizi et al. 1988) using 20 &#181;g of total protein per assay. Cells     <br /> carrying plasmid pRT(p), an RT-overexpressing plasmid (gift of S.           <br /> Hughes, National <span id='am-21' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-25'  ></span>Cancer</span> Institute), served as positive control for         <br /> RT activity. Dissociation constants (Kd) were determined from <br />                                                                      <br /> nitrocellulose filter-binding assays as described (Burke et al. 1996)     <br /> using gel-purified, radiolabeled aptamer RNA and purified HIV-1               <br /> RT carrying a His6 tag on its carboxyl terminus.                         <br /> </body></html>