<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>YgbQ, cell division protein <span id='am-89' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-90' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-92'  ></span><span id='am-93' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span>Escherichia coli</span> <br /> and <span id='am-55' about='obo:NCBITaxon_666' typeof='owl:Thing obo:NCBITaxon_641 obo:NCBITaxon_662 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_135623'><span id='am-56' property="oboInOwl:hasRelatedSynonym" content="Bacillus cholerae" datatype="xsd:string"></span><span id='am-57'  ></span><span id='am-58' property="oboInOwl:hasRelatedSynonym" content="Pacinia cholerae-asiaticae" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasRelatedSynonym" content="Microspira comma" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasRelatedSynonym" content="Spirillum cholerae" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasRelatedSynonym" content="Spirillum cholerae-asiaticae" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="Bacillus cholerae-asiaticae" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="Vibrio choleae" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="Vibrio cholerae-asiaticae" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasRelatedSynonym" content="Vibrio cholera" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasRelatedSynonym" content="Vibrio comma" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasRelatedSynonym" content="Bacillo virgola del Koch" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Liquidivibrio cholerae" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Vibrio cholerae" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="PMID:9272984" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasRelatedSynonym" content="Kommabacillus" datatype="xsd:string"></span>Vibrio cholerae</span>, localizes codependent <br /> fashion FtsL division site <br /> Nienke Buddelmeijer, Nicholas Judson, Dana Boyd, John J. Mekalanos, Jonathan Beckwith* <br /> Department Microbiology Molecular Genetics, Harvard Medical School, 200 Longwood Avenue, Boston, MA 02115 <br />  <br /> Contributed Jonathan Beckwith, March 1, 2002 <br />  <br /> YgbQ cell division protein <span id='am-105' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-106' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-110'  ></span><span id='am-111' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span>Escherichia</span> coli <span id='am-74' about='obo:NCBITaxon_662' typeof='owl:Thing obo:NCBITaxon_641 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_135623'><span id='am-75' property="oboInOwl:hasDbXref" content="PMID:8590667" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="Vibrio" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasDbXref" content="PMID:7520733" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasAlternativeId" content="NCBITaxon:705" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasDbXref" content="PMID:21296930" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasRelatedSynonym" content="Microspira" datatype="xsd:string"></span><span id='am-81'  ></span><span id='am-82' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Listonella" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasDbXref" content="PMID:17978204" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="PMID:1371064" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="PMID:4935323" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasRelatedSynonym" content="Pacinia" datatype="xsd:string"></span>Vibrio</span>             E. coli genome proteins structural features similar <br /> cholerae. In E. coli ygbQ gene discovered result         FtsL yielded candidates, YgbQ <br /> computer search E. coli genome designed potential           protein promising. <br /> interacting partners cell division protein FtsL. In V. cholerae,         At time, Mekalanos laboratory working on <br /> ygbQ identified essential gene using transposon             unrelated project identify essential genes bacterium <br /> that fuses genes arabinose promoter. The role YgbQ cell        Vibrio cholerae. These studies led identification gene, <br /> division supported following. Cells depleted YgbQ          absence product led filamentation the <br /> both organisms form long filaments, DNA segregation             bacterium. A chance discussion hallway led discov- <br /> affected. YgbQ localizes constriction site wild-type E. coli     ery studying homologous ORFs organ- <br /> cells. Localization E. coli YgbQ constriction site depends       isms. Our combined studies, reported here, strongly indicate that <br /> on cell division proteins FtsQ FtsL FtsW FtsI, placing     product ygbQ gene E. coli homologue, <br /> YgbQ sequential dependency <span id='am-52' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> proteins localizing          VC0527, V. cholerae required cell division process. <br /> the division site. Localization green fluorescent protein-FtsL     In addition, results consistent suggestion that <br /> depends YgbQ, indicating FtsL YgbQ colocalize           FtsL YgbQ interact colocalize septum. <br /> division site E. coli. Our results colocalization proteins <br /> to bacterial midcell E. coli raise possibility   Material Methods <br /> proteins interact coiled-coil structure.                              Bacterial Strains, <span id='am-133' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-134' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-135' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-136' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Plasmids</span>, Media. Bacterial strains plas- <br />                                                                            mids described Table 1. Growth rich medium, NZY <br />                                                                            (E. coli) LB (V. cholerae; ref. 5). When needed, ampicillin <br /> C    ell division bacteria takes place midcell occurs <br />      DNA duplicated segregated two <br /> daughter nucleoids. In Escherichia coli, process requires set <br />                                                                            200 &#65533;?&#65533;g&#20862;ml (plasmid) 25 &#65533;?&#65533;g&#20862;ml (chromosome), chloramphen- <br />                                                                            icol 30 &#65533;?&#65533;g&#20862;ml, kanamycin 40 &#65533;?&#65533;g&#20862;ml, spectinomycin 100 &#65533;?&#65533;g&#20862;ml, <br /> of proteins localize constriction site        streptomycin 500 &#65533;?&#65533;g&#20862;ml added. Medium supple- <br /> septum. These proteins coordinate invagination cell                 mented 0.1% (V. cholerae) 0.2% L-arabinose (E. coli) and <br /> membrane, inward growth peptidoglycan layer, and,                   0.2% D-glucose induce repress expression genes con- <br /> finally, separation daughter cells. The proteins, FtsZ,            trolled PBAD promoter, respectively. Standard techniques, <br /> FtsA, ZipA, FtsK, FtsQ, FtsL, FtsW, FtsI, FtsN,              PCR, electroporation, transformation, P1 transduction, <br /> identified largely genetic approaches. Because sys-             used cloning analysis DNA (6, 7). The enzymes <br /> tematic genetic approach identifying proteins            manipulating DNA New England Biolabs. <br /> performed devised, number thus-far- <br /> undiscovered proteins play role cell division. Informa-          Isolation Arabinose-Dependent V. cholerae. The introduction of <br /> tion available regarding function               TnAraOut V. cholerae screen insertion mutants <br /> proteins (reviewed refs. 1 2).                                      arabinose-dependent growth described <br />    Studies number laboratories led sequential          (8) following modification. NJ348 isolated by <br /> dependency model assembly group proteins             mating Tn10cm (pBSL181; ref. 9) N16961Sm generate a <br /> the cell septum (3). FtsZ arrives cell septum, providing      chloramphenicol-resistant V. cholerae reduce background <br /> a scaffold recruitment subsequent proteins. FtsA            conjugation pNJ17. NJ406 isolated strain <br /> ZipA localize septum independently other,               arabinose-dependent growth phenotype a <br /> each depends presence FtsZ localization. The                 TnAraOut insertion 66 bp upstream VC0527. <span id='am-34' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-35' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-37'  ></span><span id='am-38' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> strain <br /> remaining proteins assemble midcell strictly sequential            forms tiny colonies LB plates fails grow LB broth <br /> dependency order follows: FtsK&#8211;FtsQ&#8211;FtsL&#8211;FtsW&#8211;FtsI&#8211;                     absence arabinose. <br /> FtsN. The mechanisms order recruitment not <br /> understood.                                                                Construction ygbQ ygbQ-P-B-O Operon Under pBAD <br />    The work reported arose projects different               Promoter. The ygbQ gene amplified PCR chromo- <br /> goals Beckwith Mekalanos laboratories. The Beckwith             somal DNA wild-type E. coli strain using ygbQ5EcoRI <br /> laboratory studied cell division E. coli focusing atten-         (5&#11032;-CCGGA AT TCTCGGATGCATGGGATGAT-3&#11032;) and <br /> tion proteins, FtsQ, FtsL, FtsI. These proteins          ygbQ3XbaI (5&#11032;-CGCTCTAGATGTTAATTCCCGGGCTGA- <br /> similar membrane topologies, i.e., short amino-terminal do-              3&#11032;) primers. The PCR fragment digested EcoRI and <br /> main facing cytoplasm, single transmembrane segment, and <br /> a carboxyl-terminal domain located periplasm. FtsL a <br />                                                                            Abbreviations: GFP, green fluorescent protein; IPTG, isopropyl &#65533;?&#65533;-D-thiogalactoside. <br /> leucine zipper-like motif periplasmic domain, indicating <br /> that form coiled-coil <span id='am-39' about='obo:NCBITaxon_9605' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-40'  ></span><span id='am-41' property="rdfs:label" content="Homo" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>homo</span>- heterooligomers. It forms            *To reprint requests addressed at: Harvard Medical School, Department <br />                                                                             Microbiology Molecular Genetics, 200 Longwood Avenue, Room 212B, Boston, MA <br /> homodimers vitro likely weakly dimerizes vivo (ref.          02115. E-mail: jon_beckwith@hms.harvard.edu. <br /> 4 N.B. J.B., unpublished results). We suspected               The publication costs article defrayed page charge payment. This <br /> there partner protein forms heterodimers                article marked &#8220;advertisement&#65533;? accordance 18 U.S.C. <br /> higher order heterooligomers FtsL. A search               &#167;1734 solely indicate fact. <br />  <br />  <br />  <br /> 6316 &#8211; 6321 &#20841; PNAS &#20841; April 30, 2002 &#20841; vol. 99 &#20841; no. 9                                                         www.pnas.org&#20862;cgi&#20862;doi&#20862;10.1073&#20862;pnas.092128499 <br />  Table 1. Strains plasmids <br /> Strain plasmid                               Relevant genetic marker(s) feature(s)                                  Source reference <br />  <br /> E. coli <br />   JOE309                 MC4100 araD&#11001;                                                                                     3 <br />   NB974                  JOE309 &#9004;(&#65533;?&#65533;attL-lom)&#11019;bla lacIqpBAD18-ygbQ                                                         This study <br />   NB946                  JOE309 ygbQ&#11019;Kan &#9004;(&#65533;?&#65533;attL-lom)&#11019;bla lacIqpBAD18-ygbQ                                                This study <br />   NB805                  JOE309 &#9004;(&#65533;?&#65533;attL-lom)&#11019;bla lacIq P207-gfp-ygbQec                                                    This study <br />   NB802                  JOE309 &#9004;(&#65533;?&#65533;attL-lom)&#11019;bla lacIq P207-gfp-ygbQvc                                                    This study <br />   NB841                  JOE309 ftsL&#11019;TnPhoA (Kanr) &#9004;(&#65533;?&#65533;attL-lom)&#11019;bla lacIq P207-gfp-ygbQec&#20862;pLMG180                         This study, 5 <br />   NB836                  JOE309 ftsQ&#11019;TnPhoA50 (Kanr) &#9004;(&#65533;?&#65533;attL-lom)&#11019;bla lacIq P207-gfp-ygbQec&#20862;pJC10                         This study, 26 <br />   NB966                  JOE309 ftsI&#11019;TnPhoAI137&#9004;IS50R (Kanr) &#9004;(&#65533;?&#65533;attL-lom)&#11019;bla lacIq P207-gfp-ygbQec&#20862;pDSW262               This study, 10 <br />   NB991                  JOE309 ftsW&#11019;Kan &#9004;(&#65533;?&#65533;attL-lom)&#11019;bla lacIq P207-gfp-ygbQec&#20862;pDSW406                                   This study, 27 <br />   NB976                  JOE309 ygbQ&#11019;Kan &#9004;(&#65533;?&#65533;attL-lom)&#11019;bla lacIq pBAD18-ygbQ att&#65533;?&#65533;80&#11019;pJC115(P207-gfp-ftsI)                  This study, 3 <br />   NB977                  JOE309 ygbQ&#11019;Kan &#9004;(&#65533;?&#65533;attL-lom)&#11019;bla lacIq pBAD18-ygbQ att&#65533;?&#65533;80&#11019;pJC116(P207-gfp-ftsL)                  This study, 3 <br />   NB980                  JOE309 ygbQ&#11019;Kan &#9004;(&#65533;?&#65533;attL-lom)&#11019;bla lacIq pBAD18-ygbQ att&#65533;?&#65533;80&#11019;pJC118(P207-gfp-ftsQ)                  This study, 3 <br />   NB1000                 JOE309 ygbQ&#11019;Kan &#9004;(&#65533;?&#65533;attL-lom)&#11019;bla lacIq pBAD18-ygbQ P209-gfp-ftsW                                 This study <br /> V. cholerae <br />   N16961Sm               El <span id='am-118' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-119' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-121'  ></span><span id='am-122' property="rdfs:label" content="Tor" datatype="xsd:string"></span>Tor</span>                                                                                           28 <br />   NJ348                  N16961Sm Tn10cm insertion pBSL181                                                           This study <br />   NJ406                  YgbQ&#11019;TnAraOut                                                                                    This study <br />   NJ600                  NJ406&#20862;pNJ25                                                                                      This study <br /> Plasmids <br />   pNB10                  pBAD18-ygbQ                                                                                      This study, 29 <br />   pNB11                  pBAD18-ygbQPBO                                                                                   This study, 29 <br />   pNB12                  pBAD33-ygbQPBO                                                                                   This study, 29 <br />    pNJ17                 TnAraOut delivery vehicle                                                                        8 <br />    pNJ25                 pDSW207-ygbQvc                                                                                   This study, 10 <br />    pBSL181               Tn10cm                                                                                           9 <br />  <br />  <br />  <br /> XbaI ligated sites pBAD18, resulting      gfp-ftsQ, gfp-ftsL, gfp-ftsI fusion strains att site the <br /> plasmid named pNB10 (restriction sites underlined       lambdoid phage &#65533;?&#65533;80 described Chen Beckwith (3). <br /> primer sequences). The complete ygbQ-P-B-O operon am- <br /> plified PCR chromosomal DNA using ygbQ5EcoRI                   Construction ygbQ Null Strain E. coli. To disrupt chro- <br /> (5&#11032;-CCGGA AT TCTCGGATGCATGGGATGAT-3&#11032;)                              mosomal ygbQ gene recombined PCR product chro- <br /> ygbO3XbaI (5&#11032;-CGCTCTAGACAT TCACGCCA AGATA-                             mosomal DNA (12). A DNA fragment amplified PCR <br /> GACG-3&#11032;) primers. The PCR fragment digested                using pKD4 (12) template DNA H1P1ygbQ (5&#11032;-GGATG- <br /> EcoRI XbaI ligated sites pBAD18,          TAGAGTCGTCTTCGGATGCATGGGATGATGATGCC- <br /> the resulting plasmid named pNB11. The ygbQ-P-B-O genes            GT T T T TCAGGTGTAGGCTGGAGCTGCT TC-3&#11032;) and <br /> were recloned pBAD18 chloramphenicol vector con-             P2H2Q (5&#11032;-GCGCA A ACATCCA A ATGAGTGGT TGC- <br /> struct ygbQ depletion strain (see below). pNB11 digested         CATGT TA AT TCCCGGGCTGAT T TACATATGA ATA- <br /> with NheI SalI. The 2.8-kb fragment containing ygbQ-           TCCTCCTTAG-3&#11032;) primers. The resulting fragment was <br /> P-B O genes ligated NheI- SalI-digested            composed kanamycin-resistance gene extensions that <br /> pBAD18 chloramphenicol vector, resulting plasmid           homologous regions adjacent end ygbQ <br /> named pNB12.                                                           gene (underlined sequences). This PCR fragment trans- <br />                                                                        formed competent cells strain NB885 carrying plasmids <br /> Construction Strains Containing Green Fluorescent Protein (GFP)     pKD46 (12) pNB12. Cells selected growth on <br /> Fusions. ygbQ E. coli amplified PCR using pNB10           kanamycin presence L-arabinose. Both phage &#65533;?&#65533; red <br /> template ygbQ5EcoRIgfp (5&#11032;-CGAGAATTCAACAA-                         recombinase gene pKD46 ygbQ-P-B O genes on <br /> CA ACATGGGTA A ACTA ACGCTGCTGT TGC-3&#11032;)                             pNB12 control pBAD promoter. The replace- <br /> ygbQ3HdIIIgfp (5&#11032;-TCGAAGCTTTTATCGATTGTTTTGC-                           ment ygbQ kanamycin confirmed PCR using <br /> CCCGCAGAC-3&#11032;) primers. The amplified product di-                kanamycin-, ygbE-, ygbP-specific primers. Kanamycin- <br /> gested EcoRI HindIII, ligated sites          resistant colonies tested growth ampicillin, test for <br /> pDSW207 (10), named pNB20. The GFP-YgbQ fusion                     loss pKD46, L-arabinose D-glucose test for <br /> protein linker sequence YKEFNNM, Y K             depletion ygbQ. Strains disrupted ygbQ gene grew on <br /> the residues GFP, M residue YgbQ. ygbQ     D-glucose plates small colonies, likely basal <br /> of V. cholerae amplified PCR chromosomal DNA               expression pNB12. The ygbQ::Km transduced a <br /> from N16961 using primers ygbQF (5&#11032;-GCCGAATTCAT-                   strain carried extra chromosomal copy ygbQ under <br /> GCGAGTATTTGCTTTGACC-3&#11032;) ygbQR (5&#11032;-GGCTG-                           pBAD &#65533;?&#65533;att site complementation avoid residual <br /> CAGTTGGCGCGACTCCTCACCGAT-3&#11032;). The amplified                            expression ygbQ-P-B O pNB12 (NB946). <br /> product digested EcoRI HindIII ligated into <br />                                                                                                                                                  MICROBIOLOGY <br />  <br />  <br />  <br />  <br /> the sites pDSW207 (10). The resulting plasmid              Growth Conditions GFP-Fusion Protein Localization. For localiza- <br /> named pNJ25.                                                           tion GFP fusions wild-type background, cells grown <br />   Stable introduction gfp-ygbQ fusions gfp-ftsW        overnight 30&#176;C NZY medium plus 25 &#65533;?&#65533;g&#20862;ml ampicillin, <br /> the E. coli chromosome &#65533;?&#65533; attachment site obtained          diluted 1:200 fresh medium ampicillin, grown to <br /> with InCh integration vector (11). The construction             optical density 600 nm (OD600) 0.5. The cultures were <br />  <br /> Buddelmeijer et al.                                                                           PNAS &#20841; April 30, 2002 &#20841; vol. 99 &#20841; no. 9 &#20841; 6317 <br />  diluted medium containing 2.5 &#65533;?&#65533;M (gfp-ftsI), 5 &#65533;?&#65533;M (gfp-ftsQ), <br /> or 10 &#65533;?&#65533;M (gfp-ftsL gfp-ygbQ) isopropyl &#65533;?&#65533;-D-thiogalactoside <br /> (IPTG). Cells kept logarithmic growth phase serial <br /> dilutions medium grown 4 h. Cells fixed <br /> directly growth medium processed microscopy as <br /> described below. For experiments involving depletion strains, <br /> the overnight culture grown 30&#176;C NZY medium <br /> containing chloramphenicol, kanamycin, 0.2% L-arabinose. <br /> The culture diluted 1:200 medium grown to <br /> an OD600 0.5. After washing cells medium        Fig. 1. Phenotype ygbQ depletion strain V. cholerae. (A) Image cells <br /> sugar, cells resuspended medium containing L-              grown arabinose express ygbQ. (B) Image cells grown glucose <br /> arabinose D-glucose IPTG. Cells kept logarithmic        deplete YgbQ. (Scale bar, 10 &#65533;?&#65533;m.) <br /> growth phase serial dilutions medium grown <br /> for hours. Cells fixed filaments substantial <br /> length observed. To study localization GFP-YgbQ         genes downstream ygbQ, ygbP, ygbB essential (17). To <br /> an FtsQ, FtsL, FtsW, FtsI depletion strain, gfp-ygbQ            determine inability grow absence arabinose was <br /> induced 10 &#65533;?&#65533;M IPTG strains. Strains study           caused ygbQ downstream genes, plasmid containing <br /> localization GFP-FtsQ, GFP-FtsL, GFP-FtsW, GFP-FtsI             ygbQ, pNJ25, constructed. NJ406 comple- <br /> in ygbQ depletion strain constructed using gfp fusions       mented IPTG-inducible expression ygbQ pNJ25, <br /> integrated phage &#65533;?&#65533;80 att site (3). The cultures grown      indicating gene complement filamen- <br /> in presence 2.5 &#65533;?&#65533;M (gfp-ftsI), 5 &#65533;?&#65533;M (gfp-ftsQ), 10 &#65533;?&#65533;M           tation phenotype. A discussion cell division phenotypes <br /> (gfp-ftsL), 1 mM (gfp-ftsW) IPTG.                                   N.J. J.B. led realization studying <br />                                                                        gene. <br /> Fluorescence Microscopy. Cells fixed mixture para- <br /> formaldehyde glutaraldehyde microscopic analysis         A Computer Search Potential Interacting Partners FtsL E. coli. <br /> described (13). After fixation, cells stained 0.2 &#65533;?&#65533;g&#20862;ml      The unique features cell division protein FtsL fact that <br /> 4&#11032;,6-diamidino-2-phenylindole PBS 10 min room tem-           FtsL homodimers weak exist temporarily <br /> perature, washed twice PBS, immobilized agarose              vivo led Beckwith lab look potential partner proteins <br /> slides described (14). To examine cells used Orca 12-bit      interacted FtsL. We sought protein share <br /> cooled charge-coupled device <span id='am-44' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-45' property="rdfs:label" content="Camera" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47'  ></span><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>camera</span> (Hamamatsu Photonics,              features structure membrane topology of <br /> Hamamatsu City, Japan) mounted Axioskop 2 microscope             FtsL. A search developed using entire E. <br /> (Zeiss). Images taken using Zeiss AXIOVISION software          coli chromosome look proteins (i) unknown <br /> and analyzed public domain program OBJECTIMAGE 1.62p          function, (ii) contained leucine zipper-like motif predicted to <br /> by Norbert Vischer (University Amsterdam, Amsterdam,                form coiled-coil structure, (iii) predicted be <br /> http:&#20862;&#20862;simon.bio.uva.nl&#20862;object-image.html). Images pro-           integral cytoplasmic membrane proteins. We such <br /> cessed Adobe PHOTOSHOP presentation.                            proteins including putative tyrosine kinase, proteins with <br />                                                                        considerably higher molecular weight FtsL, small <br /> Preparation Membranes Detection FtsL Western Blotting.    proteins. <br /> Membranes prepared sonicating cells 100 mM                     The small proteins YqhH (SwissProt accession no. <br /> Tris&#18528;HCl, pH 8.0, containing 1 mM PMSF. Whole cells               Q46860), outer membrane lipoprotein precursor, YgbQ <br /> removed centrifugation 5,000 rpm 5 min. Membranes            (Q46894), hypothetical protein conserved bacterial <br /> were pelleted 100,000 &#11003; g Beckman TL100 ultracentrifuge        <span id='am-49' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> shows weak homology (20 59 residues are <br /> for 60 min resuspended 100 mM Tris&#18528;HCl, pH 8.0. A total         identical) protein <span id='am-138' about='obo:NCBITaxon_86665' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-139' property="oboInOwl:hasDbXref" content="PMID:12382115" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1414" datatype="xsd:string"></span><span id='am-141' property="rdfs:label" content="Bacillus halodurans" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasDbXref" content="PMID:10591021" datatype="xsd:string"></span><span id='am-143'  ></span><span id='am-144' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasAlternativeId" content="NCBITaxon:72574" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-147' property="oboInOwl:hasRelatedSynonym" content="Bacillus alcalophilus subsp. halodurans" datatype="xsd:string"></span>Bacillus halodurans</span>. The B. halo- <br /> of 10 &#65533;?&#65533;g membranes applied lane SDS&#20862;PAGE.                durans protein itself homologue cell division initiator <br /> Electrophoresis immunoblotting performed de-               protein DivIC <span id='am-1' about='obo:NCBITaxon_1423' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_653685'><span id='am-2' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232828" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429428" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="Bacillus uniflagellatus" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451272" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasAlternativeId" content="NCBITaxon:981081" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="Bacillus subtilis" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232841" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232840" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasAlternativeId" content="NCBITaxon:941949" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224247" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasAlternativeId" content="NCBITaxon:387338" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451270" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232822" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042381" datatype="xsd:string"></span><span id='am-17'  ></span><span id='am-18' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232830" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis/Bacillus globigii" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429427" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042382" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232825" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232824" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasAlternativeId" content="NCBITaxon:523730" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Bacillus natto" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042380" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232844" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Vibrio subtilis" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232470" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232819" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="PMID:11211269" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis8" datatype="xsd:string"></span>Bacillus subtilis</span>. (Homology searches did not <br /> scribed Laemmli (15) Towbin et al. (16), respectively. The      YgbQ direct homolog B. subtilis DivIC.) In <br /> membrane probed FtsL-specific polyclonal antibodies           B. subtilis, FtsL DivIC function complex cell <br /> and subsequently horseradish peroxidase-conjugated goat           division (18). <br /> anti-rabbit antibodies developed chemiluminescence               YgbQ composed 103 <span id='am-112' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-113'  ></span><span id='am-114' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-117' property="rdfs:label" content="Aa" datatype="xsd:string"></span>aa</span> residues (11 kDa) is <br /> reagents (Amersham Pharmacia).                                         predicted bitopic membrane protein. A three-residue <br />                                                                        amino-terminal domain predicted face cytoplasm, <br /> Results                                                                residues L4&#8211;G22 resembles transmembrane segment, the <br /> Identification Characterization ygbQ V. cholerae. The        presumed periplasmic region, containing L46&#8211;L67, leucine <br /> mariner-based transposon, TnAraOut, causes genes tran-           zipper-like motif predicted form single coiled coil <br /> scriptionally fused arabinose-inducible promoter PBAD           (Fig. 2). A <span id='am-128' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-129' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-130' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-132'  ></span>hybrid</span> protein alkaline phosphatase was <br /> after transposition genome (8). Essential genes          fused carboxyl terminus YgbQ resulted dark blue <br /> identified, strains TnAraOut insertions fused          colonies 5-bromo-5-chloro-3-indolylphosphate (XP) indica- <br /> essential genes depend presence arabinose growth.        tor plates, suggesting carboxyl terminus protein <br /> We TnAraOut inserted (strain NJ406) V. cholerae 66          facing periplasm (<span id='am-123' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-124'  ></span><span id='am-125' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-127' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> shown; ref. 19). This set of <br /> bp upstream predicted translational start VC0527        structural features resembles FtsL. The gene gene <br /> depended arabinose growth. The gene VC0527              cluster, including downstream genes ygbP, ygbB, ygbO, <br /> similar E. coli gene ygbQ (and referred       Gram-negative Gram-positive bacteria. Both <br /> to ygbQ).                                                           ygbP ygbB essential genes involved pathway for <br />   Growth strain NJ406 LB broth presence                isoprenoid biosynthesis E. coli (17). <br /> arabinose showed wild-type growth phenotype. With cells                 Because Bacillus information role homol- <br /> grown absence arabinose, long filaments V. cholerae       ogous proteins YgbQ closely analogous <br /> formed (Fig. 1). The genomic organization region V.         domain structure FtsL, considered interesting <br /> cholerae E. coli similar. In E. coli known   candidate FtsL-interacting protein. We initiated studies to <br />  <br /> 6318 &#20841; www.pnas.org&#20862;cgi&#20862;doi&#20862;10.1073&#20862;pnas.092128499                                                                                Buddelmeijer et al. <br />                                                                                      ygbQ affect ygbQ expression the <br />                                                                                     downstream genes ygbP, ygbB, ygbO. However, we <br />                                                                                     used (12) supposed allow transcription genes downstream <br />                                                                                     disrupted gene. <br />                                                                                        The strain carrying ygbQ deletion carried addi- <br />                                                                                     tional copy ygbQ PBAD control bacteriophage &#65533;?&#65533;att <br />                                                                                     site. This depletion strain able grow medium supple- <br />                                                                                     mented arabinose glucose. The cells form <br />                                                                                     extensive filaments YgbQ depleted (see Fig. 4). 4&#11032;,6- <br />                                                                                     Diamidino-2-phenylindole staining indicated nucleoids <br /> Fig. 2. CLUSTALW 1.81 alignment YgbQ E. coli (Ec) V. cholerae (Vc). The   filaments spaced regularly; thus, DNA segregation not <br /> leucine residues form leucine zipper-like motif periplasmic         affected ygbQ null strain (data shown). A plasmid <br /> domain YgbQ E. coli indicated arrows. The overlined regions          expressing gfp-ygbQ complements ygbQ deletion strain for <br /> represent putative transmembrane (TM) domain (residues 4 &#8211;22)           growth. Depletion ygbQ strain expressing ygbP, B, O <br /> coiled-coil domain (residues 29 &#8211; 67) YgbQ. Conserved amino acid residues        different promoter results filamentous phenotype, <br /> are indicated asterisks. <br />                                                                                     indicates ygbQ null, expression ygbP, B, <br />                                                                                     O genes sufficient growth (data shown). Fur- <br /> inactivate ygbQ gene characterize phenotype                      thermore, ygbP ygbB depletion strains described by <br /> null mutation. At point discovered labora-                     Campos et al. (17) depletion phenotype different the <br /> tories working homologous genes (50% identity) E.                        ygbQ depletion strain; stop growing immediately the <br />                                                                                     complementing pathway inactivated, cells filament. <br /> coli V. cholerae occupying chromosomal position in <br />                                                                                     These results that, V. cholerae, YgbQ E. coli is <br /> the organisms. The V. cholerae protein, having a <br />                                                                                     essential, depletion results inhibition cell division. <br /> strict leucine-zipper motif, predicted form coiled-coil <br /> structure COILS 2.2 program (20). Given inacti- <br />                                                                                     Localization YgbQ Depends FtsQ FtsL Not FtsW and <br /> vation V. cholerae gene led filamentous phenotype, we <br />                                                                                     FtsI. The results studies V. cholerae E. coli <br /> were encouraged pursue study E. coli gene. <br />                                                                                     YgbQ homologues strongly suggested YgbQ plays im- <br /> While proceeding construct ygbQ null mutation, decided <br />                                                                                     portant role cell division. We wished determine position <br /> also determine immediately YgbQ localized the <br />                                                                                     YgbQ process assembly cell division proteins at <br /> cell septum.                                                                        midcell. To make determination studied localization <br />                                                                                     GFP-YgbQ mutants depleted cell division <br /> E. coli YgbQ Localizes Division Site Wild-Type Background. <br />                                                                                     proteins appeared late pathway, FtsQ, FtsL, FtsW, <br /> All known E. coli cell division proteins localize constric-                  FtsI, discovery YgbQ E. coli based on <br /> tion site; unrelated membrane proteins examined                      hunt partner FtsL. Depletion strains cell division <br /> do (see e.g. refs. 4, 10, 13). Thus, localization protein              proteins contained null mutation protein be <br /> to midcell provides strong suggestion protein involved                    depleted plasmid expressing cell division protein <br /> in cell division. We asked YgbQ localized cell                       PBAD promoter. The cells grow normally presence <br /> septum. A gene fusion expressed YgbQ GFP                           arabinose, shifted glucose-containing medium, <br /> amino terminus constructed inserted chromo-                        cells eventually developed long filaments. As cells are <br /> some single copy &#65533;?&#65533;InCh (11). The strains merodiploid                     depleted glucose medium, fix prepare them <br /> for ygbQ, i.e., wild-type copy ygbQ present normal                   exhibiting logarithmic growth. <br /> chromosomal location, gfp-ygbQ copy located &#65533;?&#65533;                          The cells depletion strains FtsQ, FtsL, FtsW, FtsI <br /> attachment site expressed regulatable trc promoter.                        grown arabinose normal length, GFP-YgbQ is <br /> The strains grown different concentrations IPTG                     septum (data shown). When cells are <br /> the media induce different levels expression GFP-                      grown glucose-containing medium, formed filaments. <br /> YgbQ fusion protein. Using fluorescent microscopy, looked                        GFP-YgbQ localized septal region filaments the <br /> for localization GFP-YgbQ midcell IPTG con-                          FtsW FtsI depletion strains FtsQ FtsL <br /> centrations ranging 10 50 &#65533;?&#65533;M. Most cells exhibited                      depletion strains (Fig. 3). As positive negative controls, we <br /> a bright band fluorescence midpoints, indicating                   showed that, previously (13, 21), GFP-FtsQ localizes to <br /> GFP-YgbQ localizes division site (data shown).                           potential division sites FtsL depletion filaments, GFP-FtsI <br /> GFP-FtsL localization region detected previ-                       does (data shown). These results suggest (i) <br /> ously 10 &#65533;?&#65533;M inducer used induce GFP-FtsL                           YgbQ localizes cell septum FtsL FtsW or <br /> fusion protein.                                                                     (ii) YgbQ colocalizes septum FtsL. <br />    We looked localization YgbQ V. cholerae <br /> (YgbQvc) E. coli using GFP-fusion protein analogous                      Localization FtsQ, FtsL, FtsW, FtsI YgbQ Depletion Strain. <br /> the GFP-YgbQ E. coli. We observed fluorescent signal                      To clarify position YgbQ septal assembly sequence <br /> expression gfp-ygbQvc induced 50 &#65533;?&#65533;M IPTG com-                           further, studied localization GFP-FtsL ygbQ null <br /> pared 10 &#65533;?&#65533;M GFP-YgbQec; 50 &#65533;?&#65533;M IPTG,                          background using GFP-FtsQ, GFP-FtsW, GFP-FtsI as <br /> fluorescent signal intense GFP-YgbQ E.                        likely positive negative controls. Because complementing <br /> coli (data shown). Perhaps higher induction levels                       copy pBAD-ygbQ integrated &#65533;?&#65533;att site YgbQ <br /> GFP-YgbQvc necessary observe septal fluorescence                             depletion strain, integrated GFP-fusion constructs into <br /> because nonnative protein competes wild-type                     chromosome att site lambdoid phage &#65533;?&#65533;80 (3). <br /> YgbQ E. coli localization midcell.                                        Strains grown NZ medium supplemented arabinose <br />                                                                                                                                                              MICROBIOLOGY <br />  <br />  <br />  <br />  <br />                                                                                     glucose 30&#176;C, expression gfp-ftsQ, gfp-ftsL, gfp-ftsI, <br /> Depletion ygbQ E. coli Is Correlated Filamentation. A YgbQ               gfp-ftsW induced 5 &#65533;?&#65533;M, 10 &#65533;?&#65533;M, 2.5 &#65533;?&#65533;M, 1 mM <br /> depletion strain E. coli constructed chromosomal                          IPTG, respectively. In cells grown arabinose, observed a <br /> disruption ygbQ (12). Because ygbQ cluster                     fluorescent signal GFP-FtsQ, GFP-FtsL, GFP-FtsW, and <br /> essential genes involved isoprenoid synthesis, disruption                     GFP-FtsI cell midpoint (Fig. 4). In filaments formed after <br />  <br /> Buddelmeijer et al.                                                                                        PNAS &#20841; April 30, 2002 &#20841; vol. 99 &#20841; no. 9 &#20841; 6319 <br />  Fig. 3. Localization GFP fusions YgbQ FtsQ (A), FtsL (B), FtsW (C), and <br /> FtsI (D) depletion strains. (Scale bars, 10 &#65533;?&#65533;m.) <br />  <br />  <br />  <br /> depletion YgbQ, obtained results consistent order <br /> predicted above; GFP-FtsQ localizes septum, GFP- <br /> FtsW GFP-FtsI not. However, observed localiza- <br /> tion GFP-FtsL midcell YgbQ-depleted filaments (Fig. 4). <br /> Because YgbQ itself septum FtsL-depleted <br /> cells, results suggest FtsL YgbQ localize the <br /> division site codependent fashion. <br />    Results B. subtilis suggest YgbQ analogue, DivIC, <br /> stabilizes FtsL. We asked depletion YgbQ might <br /> affect FtsL stability E. coli assayed FtsL protein levels in <br /> a YgbQ depletion strain. We detected FtsL YgbQ depletion <br /> strain grown arabinose cells grown glucose (Fig. <br /> 5). The likely instability FtsL conditions suggests                 Fig. 4. Localization GFP fusions FtsQ (Q), FtsL (L), FtsI (I) ygbQec <br /> that YgbQ stabilizes FtsL, indicating FtsL&#8211;YgbQ interaction.                    depletion strain. (A) Localization GFP-fusion proteins cells grown with <br />                                                                                    arabinose express ygbQ. (B) Localization GFP-fusion proteins cells <br /> Discussion                                                                         grown glucose deplete ygbQ. (Scale bars, 10 &#65533;?&#65533;m.) <br /> In article, report characterization gene, ygbQ, <br /> encoding protein cell division pathway E. coli V. <br />                                                                                    act (18). Localization Bacillus proteins been <br /> cholerae. In organisms, depletion YgbQ leads a <br /> filamentous phenotype typical null mutants cell                        examined. <br /> division genes. Furthermore, YgbQ localizes midcell E.                      Our studies position YgbQ protein assembly <br /> coli, feature known far specific                       pathway E. coli septum consistent an <br /> proteins directly involved cell division process. Finally,                  interaction FtsL YgbQ. YgbQ depends FtsQ not <br /> studies localization YgbQ assign specific place                  FtsW localization septum, does FtsL. But neither <br /> the septal assembly proteins required cell division                   proteins, YgbQ FtsL, seen septum in <br /> and consistent interaction FtsL.                        absence other. These results consistent a <br />    We detected ygbQ V. cholerae screen essential genes.                model FtsL YgbQ colocalize septum. Such <br /> In E. coli, focused ygbQ searching genome                      codependence explained solel <br /> </body></html>