<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Deletions in the S1 domain of Rrp5p cause <br /> processing at a novel site in ITS1 of yeast pre-rRNA <br /> that depends on Rex4p <br />  <br />  <br />                                                                           action of the processing complex responsible for <br /> Rrp5p is the only protein so far known to be                                       cleavage at site A3. Furthermore, they clearly <br /> required for the processing of yeast pre-rRNA at                                   implicate Rex4p in both pre-rRNA processing and <br />                                                                                    ribosome assembly, even though <span id='am-72' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-73'  ></span><span id='am-74' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="This" datatype="xsd:string"></span>this</span> protein is not <br /> both the early sites A0, A1 and A2 leading to 18S <br />                                                                                    essential for yeast. <br /> rRNA and at site A3, the &#174;rst step speci&#174;c for the <br /> pathway leading to 5.8S/25S rRNA. Previous in vivo <br /> mutational analysis of Rrp5p demonstrated that the <br /> &#174;rst 8 of its 12 S1 RNA-binding motifs are involved                                INTRODUCTION <br /> in the formation of the `short&apos; form of 5.8S rRNA                                  Ribosome production is one of the major metabolic activities <br /> (5.8SS), which is the predominant <span id='am-82' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-83' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> under                                    in all types of cells and this process has been highly conserved <br /> normal conditions. We have constructed two strains                                 during evolution. The mature rRNA species (with the <br /> in which the genomic RRP5 gene has been replaced                                   exception of eukaryotic 5S rRNA) are encoded by multiple, <br /> by an rrp5 deletion mutant lacking either S1 motifs                                polycistronic units that each give rise to a single precursor <br />                                                                                    transcript. This transcript then undergoes a complex series of <br /> 3&#177;5 (rrp5-D3) or 5&#177;8 (rrp5-D4). The &#174;rst mutant <br />                                                                                    modi&#174;cations including methylation and pseudouridylation as <br /> synthesizes almost exclusively 5.8SL rRNA, whereas                                 well as endo- and exonucleolytic cleavages to remove the <br /> the second one still produces a considerable                                       transcribed spacer regions. Concomitantly, the various pre- <br /> amount of the 5.8SS species. Nevertheless, both                                    cursor intermediates are assembled with the ribosomal <br /> mutations were found to block cleavage at site A3                                  proteins in an ordered fashion. In eukaryotes the majority of <br /> completely. Instead, a novel processing event                                      these reactions take place in the nucleolus and require the <br /> occurs at a site in a conserved stem&#177;loop structure                                participation of numerous non-ribosomal, trans-acting factors. <br /> located between sites A2 and A3, which we have                                        The processing pathway leading to the eukaryotic, mature <br /> named A4. A synthetic lethality screen using the                                   rRNA species is best characterized in the yeast <br />                                                                                    <span id='am-37' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-40'  ></span><span id='am-41' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> (1&#177;3; Fig. 1). The genome of <br /> rrp5-D3 and rrp-D4 mutations identi&#174;ed the REX4 <br />                                                                                    S.cerevisiae contains 150&#177;200 tandemly repeated rDNA units, <br /> gene, which encodes a non-essential protein                                        located on chromosome XII, each consisting of an 18S, 5.8S <br /> belonging to a <span id='am-69' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> of related yeast proteins that                                and 25S rRNA gene separated by two internal transcribed <br /> includes several known 3&#162;&#174;5&#162; exonucleases.                                         spacers (ITS1 and ITS2) and &#175;anked by the 5&#162;- and 3&#162;-external <br /> Inactivation of the REX4 gene in rrp5-D3 or rrp-D4                                 transcribed spacers (ETS; Fig. 1A). Transcription of an rDNA <br /> cells abolished cleavage at A4, restored cleavage at                               repeat by RNA polymerase I results in a pre-rRNA that is co- <br /> A3 and returned the 5.8SS:5.8SL ratio to the wild-                                 transcriptionally processed at its 3&#162;-end by Rnt1p to produce <br /> type value. The sl phenotype of the rrp5D/rex4 &#177;                                   the 35S precursor (4,5). Cleavage of the 35S pre-rRNA at the <br /> double mutants appears to be due to a severe dis-                                  early sites A0 within the 5&#162;-ETS, A1, the mature 5&#162;-end of 18S <br />                                                                                    rRNA, and A2 within ITS1 then gives rise to the 20S and <br /> turbance in ribosomal subunit assembly, rather <br />                                                                                    27SA2 precursor species. Components essential for these <br /> than pre-rRNA processing. The <span id='am-88' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-89'  ></span><span id='am-90' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-91' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> provide direct                                  processing cleavages include several snoRNP species as well <br /> evidence for a crucial role of the multiple S1 motifs                              as a host of protein factors, of which the precise role of many is <br /> of Rrp5p in ensuring the correct assembly and                                      still unclear (1,2,6). Inactivation of any of these trans-acting <br />  <br /> *To whom correspondence should be addressed. Tel: +31 20 444 7545; Fax: +31 20 444 7553; Email: raue@chem.vu.nl <br /> Present address: <br /> Jaap Venema, Solvay Pharmaceuticals, Department of Biotechnology, C.J. van Houtenlaan 36, 1382 CP Weesp, The Netherlands <br />  <br /> The authors wish it to be known that, in their opinion, the &#174;rst two authors should be regarded as joint First Authors <br />                                                                                       Nucleic Acids Research, 2002, Vol. 30 No. 19       4223 <br />  <br />  <br />                                                                              present in the 25.5S precursor is removed exonucleolytically <br />                                                                              by Rat1p and Xrn1p (15). Again, a considerable number of <br />                                                                              additional trans-acting factors have been implicated in these <br />                                                                              steps (1,2,20&#177;27) <br />                                                                                 Although under normal conditions RNaseMRP acts upon <br />                                                                              the 27SA2 precursor, it is also able to use the 35S pre-rRNA as <br />                                                                              its substrate when cleavage at the early sites is blocked (1,2). <br />                                                                              Thus, the set of trans-acting factors required for these early <br />                                                                              cleavages on the one hand and by RNaseMRP on the other <br />                                                                              appear to be able to act independently from each other. This <br />                                                                              conclusion is supported by the fact that correct formation of <br />                                                                              18S and 5.8S/25S rRNA occurs even if the genes are encoded <br />                                                                              in trans (28). Nevertheless, there is evidence for interdepend- <br />                                                                              ence between formation of the small and large subunit rRNAs <br />                                                                              both from mutational analysis of ITS1 (29) and experiments <br />                                                                              involving genetic depletion of the snR30 snoRNP (13). A <br />                                                                              direct link between the processing reactions required for 18S <br />                                                                              and 5.8S rRNA formation was established with the identi&#174;- <br />                                                                              cation of the yeast RRP5 gene (30). Genetic depletion of <br />                                                                              Rrp5p inhibits production of both 18S and 5.8SS rRNA by <br />                                                                              blocking cleavage at sites A0, A1 and A2 as well as A3. The <br />                                                                              193 kDa Rrp5p protein has a bipartite structure: the <br />                                                                              N-terminal two-thirds contain 12 S1 RNA-binding motifs, <br />                                                                              while the remaining C-terminal region consists mainly of <br />                                                                              seven tetratricopeptide (TPR) motifs (31,32). In vivo muta- <br />                                                                              tional analysis has established a direct correlation between this <br />                                                                              domain structure and the function of Rrp5p in formation of the <br />                                                                              18S and 5.8SS rRNAs, respectively. Mutations in some of the <br />                                                                              TPR motifs block the former, whereas deletions in the S1 <br /> Figure 1. Processing of pre-rRNA in S.cerevisiae. (A) Structure of the       domain strongly reduce the latter (31,32), but stimulate the <br /> rDNA transcription unit. Mature rRNA sequences are indicated by the thick    production of 5.8SL rRNA, which then becomes the dominant <br /> bars, transcribed spacer sequences by the thin lines. (B) The pre-rRNA       species in the mutant cells. We have now further analyzed the <br /> processing pathway. The processing cleavages requiring Rrp5p are show in <br /> large type. Straight arrows indicate endonucleolytic cleavages, kinked <br />                                                                              effect on pre-rRNA processing of two of these mutations, <br /> arrows represent exonucleolytic processing events. Note that the manner in   rrp5-D3, lacking motifs 3&#177;5, and rrp5-D4, lacking motifs 5&#177;8, <br /> which the 27SA2 pre-rRNA is converted into 27SBL is not known.               which change the 5.8SS:5.8SL ratio to different extents. <br />  <br /> factors inhibits formation of mature 18S rRNA indicating a                   MATERIALS AND METHODS <br /> strong functional interdependence. <br />    The 20S pre-rRNA is exported to the cytoplasm where it is                 Strains and <span id='am-93' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-94' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-96' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>plasmids</span> <br /> cleaved at site D to remove the remaining portion of ITS1 and                Strain YJV154 carrying a genomic, wild-type copy of the <br /> generate the mature 18S rRNA (7,8). The majority (~90%) of                   RRP5 gene under control of the GAL promoter has been <br /> the 27SA2 precursor undergoes endonucleolytic cleavage at                    described previously (31). Strains YJV306 and YJV515 were <br /> site A3 by RNaseMRP (9&#177;12). The resulting 27SA3 pre-rRNA                     obtained by respective replacement of the wild-type RRP5 <br /> is further processed by the 5&#162;&#174;3&#162; exonucleases Rat1p and                     gene of strain YJV140 by either the rrp5-D3 or rrp-D4 allele <br /> Xrn1p into the 27SBS species having the mature 5&#162;-end of                     constructed previously (31). Strain FVY8C (GAL-RRP5/rex&#177;) <br /> 5.8SS rRNA (13,14). The remaining 10% of the 27SA2 pre-                      was obtained by crossing YJV154 with YAV41 (kindly <br /> rRNA is converted into the 27SBL precursor, which has the                    provided by Dr van Hoof), in which the REX4 gene had been <br /> 5&#162;-end of 5.8SL rRNA located 6 nt upstream from that of its                  inactivated by replacing it with the kanamycin gene (19), and <br /> 5.8SS counterpart. The precise mechanism of this alternative                 selecting for spores that were prototrophic for uracil (the <br /> processing pathway is unknown. It is still unclear why                       marker for the GAL-rrp5 gene) and resistant against geneticin. <br /> eukaryotic cells contain two forms of 5.8S rRNA. However,                    Strain yRP840, the parent strain of YAV41 was also obtained <br /> differences in the pattern of translation have been observed                 from Dr van Hoof. Cloning and transformation procedures <br /> between yeast cells having the normal, high 5.8SS:5.8SL ratio                were essentially as described previously (31). <br /> and those in which this ratio is reversed (10). <br />    Both 27SBS and 27SBL pre-rRNA are further processed in                    Isolation of sl mutants and identi&#174;cation of the gene <br /> an identical manner. Cleavage at site C2 within ITS2 separates               Isolation of sl mutants was performed essentially as described <br /> the 7S and 25.5S pre-rRNAs (15). The former is converted                     previously (33). Strain YJV515 carrying the rrp5-D4 allele on <br /> into 5.8S rRNA by multistep 3&#162;&#174;5&#162; exonucleolytic trimming,                   its genome, as well as the ade2, ade3 and ura3 markers, was <br /> which involves the exosome (16&#177;18) as well as the small                      transformed with plasmid pHT-RRP5 containing the wild- <br /> exonucleases Rex1p and Rex2p (19). The portion of ITS2 still                 type RRP5, as well as the ADE3 and URA3 genes. <br />  4224   Nucleic Acids Research, 2002, Vol. 30 No. 19 <br />  <br />  <br /> Transformants were irradiated with 254 nm UV light at a dose      Table 1. Oligonucleotides used for northern hybridization and primer <br />                                                                   extensions analysis <br /> resulting in ~50% survival. A total of about 900 non-sectoring <br /> (red) colonies were selected by microscopic screening of          No.a                             Sequence (5&#162;&#174;3&#162;) <br /> approximately 1.9 3 105 survivors. After growth on yeast <br /> extract/peptone/dextrose (YPD) plates, red colonies were          1                                GCTCTCATGCTCTTGCCAAAAC <br />                                                                   2                                GATATGAAAACTCCACAGTG <br /> restreaked on plates containing 5-&#175;uoroorotic acid (5-<span id='am-1' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="rdfs:label" content="Foa" datatype="xsd:string"></span>FOA</span>).       3                                TTTGGGCATTCGAGCAATCGG <br /> Thirty-&#174;ve colonies unable to grow on these plates because of     4                                CCAGTTACGAAAATTCTTGTTTTTGAC <br /> their inability to lose the pHT-RRP5(URA3) plasmid were           5                                CTGCGTTCTTGATCGATGCG <br /> obtained. These strains were then transformed with a plasmid      6                                GAATGTTTGAGAAGGAAATGACGCTC <br /> containing either the wild-type RRP5 gene or the rrp5-D4          7                                CGCCTAGACGCTCTCTTCTTA <br /> mutant allele and 16 strains in which the former, but not the     aThe   positions of the oligonucleotides are shown in Figure 2A. <br /> latter, gene restored sectoring were selected. <br />    The selected strains were transformed with the pUN100 <br />                                                                   gradients followed by centrifugation for 17 h at 13 000 r.p.m. <br /> yeast genomic library. About 8 3 104 colonies were screened <br />                                                                   Gradients were analyzed by continuous scanning of the OD254. <br /> for restoration of sectoring resulting in 20&#177;50 candidates for <br /> each of the sl mutants tested. Further selection on 5-FOA <br /> plates reduced the number to 20&#177;30 candidates per strain. To      RESULTS <br /> exclude strains containing a library-encoded RRP5 gene the <br /> BglII restriction pattern of the recovered library plasmids was   The rrp5-D3 and rrp5-D4 deletions shift processing <br /> compared with that of pHT-RRP5. The plasmids were                 within ITS1 from site A3 to a novel, upstream site <br /> analyzed by PCR using RRP5-speci&#174;c primers. About 25%             Using a yeast strain in which the genomic RRP5 gene is <br /> of the recovered plasmids indeed contained the wild-type          conditionally expressed under control of the GAL promoter, <br /> RRP5 gene.                                                        we have shown that deletion of S1 motifs 1&#177;2 (rrp5-D2), 3&#177;5 <br />    Characterization of several library plasmids recovered from    (rrp5-D3) or 5&#177;8 (rrp5-D4) from the N-terminal region of the <br /> sl strain Y11A6 identi&#174;ed a 10 kb fragment originating from       Rrp5p protein lowers production of 5.8SS rRNA to different <br /> chromosome XV, containing &#174;ve complete open reading               extents. At the same time formation of its 5.8SL counterpart is <br /> frames (ORFs). Further subcloning and testing of the resulting    increased to approximately the same extent (31). The &#174;rst two <br /> plasmids for their ability to restore sectoring in Y11A6          mutations also lowered the growth rate ~2-fold, whereas the <br /> resulted in limitation to a fragment containing the overlapping   rrp5-D4 mutation did not signi&#174;cantly affect growth. We, <br /> ORFs YOL079 and YOL080 present on the Watson and the              therefore, decided to use the rrp5-D3 and rrp5-D4 mutations to <br /> Crick strand, respectively. A deletion construct removing a       study the role of Rrp5p in the formation of 5.8S rRNA further. <br /> large portion of YOL080 without affecting YOL079 proved           As neither deletion is lethal we constructed mutant yeast <br /> unable to restore sectoring, demonstrating that the former gene   strains in which the genomic copy of the RRP5 gene had been <br /> is the one responsible for the sl phenotype.                      replaced by either of the mutant alleles (see Materials and <br />                                                                   Methods). The resulting YJV306 (rrp5-D3) and YJV515 <br /> RNA analysis <br />                                                                   (rrp5-D4) strains were found to grow 2 and 1.4 times more <br /> Isolation of RNA, northern hybridization and primer extension     slowly than the parent (YJV140) wild-type strain, respectively <br /> analysis were carried out as described previously (34). The       (data not shown). In both cases this is slightly below the rate <br /> position of the primers is indicated in Figure 2A. The            observed previously for yeast cells that have been switched <br /> sequences of these primers are given in Table 1.                  from wild-type to the respective mutant Rrp5p by transfer <br />                                                                   from galactose- to glucose-based medium. This difference is <br /> Sucrose gradient analysis <br />                                                                   probably due to residual production of wild-type Rrp5p in <br /> Strains YJV154 or FVY88C transformants were grown on              cells containing the GAL-rrp5 gene even when grown on <br /> minimal medium with galactose as the carbon source and            glucose, whereas the integrants of course do not contain any <br /> shifted to glucose-based medium. After 24 h, cycloheximide        wild-type protein. <br /> was added at the &#174;nal concentration of 50 mg/ml and the cells        Total RNA was isolated from exponentially growing <br /> were cooled to 0&#176;C. Cells were harvested by centrifugation,       YJV140, YJV306 and YJV515 cells and subjected to northern <br /> washed with 10 ml lysis buffer [20 mM HEPES pH 7.4,               analysis. Because the two rrp5-D mutations predominantly <br /> 100 mM KCl, 2 mM MgAc2, 50 mg/ml cycloheximide,                   affect synthesis of 5.8SS rRNA (31), we focused our attention <br /> 200 mg/ml heparin, 1 mM dithiothreitol (DTT), 1 mM                on processing within ITS1, using the set of probes depicted in <br /> phenylmethylsulfonyl &#175;uoride (PMSF)] and stored at &#177;20&#176;C.         Figure 2A. RNA was also isolated after treating the cultures <br /> Extracts were prepared by adding 1 ml of lysis buffer to the      with LiCl for 2 and 6 h to facilitate detection of 5&#162;-extended <br /> cell pellet and vortexing with 0.5 mg glass beads 10 times for    forms of 5.8S rRNA (35). <br /> 30 s with intermittent cooling on ice. The resulting lysate was      The results of the northern analysis are depicted in <br /> cleared by centrifuging twice in an Eppendorf centrifuge at       Figure 2B&#177;H. In agreement with our previous results (31), <br /> 13 000 r.p.m. for 20 min and the OD260 was determined. About      YJV306 contains only a very small amount of the 5.8SS <br /> 20 OD260 units of extract were layered on 15&#177;35% sucrose          species, while the level of 5.8SL is increased ~10-fold (Fig. 2B, <br /> gradients in 10 mM Tris&#177;HCl pH 7.4, 70 mM NH4Cl, 4 mM             lanes 4&#177;6). The rrp5-D4 mutant also shows a clear predom- <br /> MgAc2, 1 mM DTT, and the gradients were centrifuged for           inance of 5.8SL rRNA, but still produces a considerable <br /> 17 h at 18 000 r.p.m. in an SW28 rotor (Beckman Instruments)      amount of the 5.8SS species (Fig. 2B, lanes 7&#177;9). Again the <br /> at 4&#176;C. For analysis of polysome pro&#174;les we used 10&#177;50%           effects of the integrated rrp5-D3 and rrp5-D4 mutant alleles <br />            Nucleic Acids Research, 2002, Vol. 30 No. 19                 4225 <br />  <br />  <br /> are more pronounced than seen previously in the carbon- <br /> source shift experiments. <br />    Addition of LiCl to yeast cells results in the accumulation of <br /> 5&#162;-extended forms of 5.8S rRNA because LiCl strongly <br /> inhibits the 5&#162;&#174;3&#162; exonucleases Rat1p and Xrn1p, responsible <br /> for the 27SA3&#174;27SBL processing step, as well as, to a lesser <br /> extent, RNase MRP which converts 27SA2 into 27SA3 pre- <br /> rRNA (35). However, cleavage of the 27SA2 and 27SA3 <br /> precursor species within ITS2 and the subsequent removal of <br /> the ITS2 spacer sequences from 7S pre-rRNA can still <br /> proceed. Figure 2C shows that upon LiCl treatment both the <br /> wild-type strain and the rrp5 deletion mutants indeed accu- <br /> mulate a product that hybridizes to probe 2, located between <br /> site A2 and A3, and that according to its mobility corresponds <br /> to the A2-E fragment. The same product is also recognized by <br /> probes 3 and 4 (Fig. 2D and E). Probe 6 (Fig. 2G) should not <br /> detect the A2-E fragment because it hybridizes only to RNA <br /> molecules that still contain at least a portion of ITS2. The band <br /> seen in Figure 2G, therefore, represents 7S pre-rRNA, which <br /> has almost the same size as the A2-E fragment. Probe 5 <br /> (Fig. 2F) detects both the A2-E fragment and 7S pre-rRNA. <br /> Furthermore, we can also detect the complementary D-A2 <br /> fragment of ITS1 in all three strains upon hybridization with <br /> probe 1 (Fig. 2H). Somewhat unexpectedly, we did not <br /> observe accumulation of the A3-E fragment upon hybridiza- <br /> tion with probes 4 or 5 (Fig. 2E and F). We surmise that this is <br /> due to the genetic makeup of our strains, because, as shown in <br /> Figure 2E (lane 10), the A3-E fragment could be visualized by <br /> probe 4 in another RRP5 wild-type strain, yRP840 (19), after <br /> treatment with LiCl for 2 h. Interestingly, however, probes 4 <br /> and 5 did detect a fragment in both the rrp5-D3 and rrp5-D4 <br /> strains that is absent from the wild-type cells (Fig. 2E and F, <br /> lanes 4&#177;9). The same fragment also hybridizes with probe 3, <br /> complementary to the region of ITS1 directly upstream from <br /> site A3 (Fig. 2D, lanes 4&#177;9), but not with either probe 2 <br /> (Fig. 2C) or probe 6 (Fig. 2G). The data indicate that in the <br /> rrp5-D3 and rrp5-D4 cells a novel processing event occurs <br /> within ITS1, either in addition to or instead of cleavage at A3. <br /> As judged from the electrophoretic mobility of the novel <br /> fragment its 5&#162;-end, which we have called A4, is located <br /> approximately midway between A2 and A3. <br />    In <span id='am-85' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> to establish whether processing at A4 indeed <br /> replaces that at A3 and to map the location of the A4 site more <br />  <br />  <br /> Figure 2. Northern analysis of low-molecular-weight processing <br /> intermediates in rrp5-D3 and rrp-D4 cells. Cultures of strains YJV306 <br /> (rrp5-D3), YJV515(rrp5-D4) and the wild-type parental strain YJV140 were <br /> grown on YPD medium to mid-exponential phase. Samples containing <br /> identical amounts of cells were taken before addition of 0.2 M LiCl and <br /> after further incubation in the presence of LiCl for 2 and 6 h. Total RNA <br /> was isolated, separated on 8% polyacrylamide gels and blotted onto <br /> nitrocellulose &#174;lters which were then hybridized in sequence with the <br /> probes indicated in (A). (A) The region of the pre-rRNA spanning ITS1, <br /> 5.8S rRNA and part of ITS2, including the processing sites. The position of <br /> the different oligonucleotides used for northern hybridization and primer <br /> extension experiments is indicated. The sequences of these oligonucleotides <br /> are shown in Table 1. (B&#177;H) Northern hybridization. Lanes 1&#177;3, YJV140; <br /> lanes 4&#177;6, YJV306; lanes 7&#177;9, YJV515. Lanes 1, 4 and 7, samples taken <br /> before addition of LiCl; lanes 2, 5 and 8, samples taken 2 h after addition of <br /> LiCl; lanes 3, 6 and 9, samples taken 4 h after addition of LiCl. Lane 10 in <br /> (E) shows hybridization of probe 4 to RNA isolated from the wild-type <br /> strain yRP840 (19) 2 h after addition of LiCl. <br />  4226   Nucleic Acids Research, 2002, Vol. 30 No. 19 <br />  <br />  <br /> precisely, we carried out primer extension experiments on <br /> total RNA isolated from wild-type, rrp5-D3 and rrp5-D4 cells <br /> either before or after treatment with LiCl using probe 4. As <br /> shown in Figure 3A a stop corresponding to A3 is readily <br /> detectable in the wild-type strain upon LiCl treatment. In the <br /> two mutant strains, however, this stop has disappeared <br /> completely, indicating that cleavage at A3 is blocked <br /> completely. Instead, a new set of stops appears 34&#177;36 nt <br /> farther upstream. Together with the results of the northern <br /> hybridization shown in Figure 2 the data &#174;rmly establish the <br /> occurrence of an alternative-processing event within ITS1 in <br /> the rrp5 deletion mutants. Furthermore, our results show that <br /> there is no absolute link between cleavage at A3 and the <br /> formation of 5.8SS rRNA. Despite the absence of any <br /> detectable processing at site A3 in the rrp5-D4 mutant, in <br /> these cells ~40% of 5.8S rRNA still is the short form, which, <br /> therefore, should originate from the 27SA2 and/or 27SA4 <br /> intermediate. The existence of such a 27SA4 processing <br /> intermediate was corroborated by primer extension experi- <br /> ments using probes 6 and 7, located upstream and downstream <br /> of site C2 within ITS2, respectively (Fig. 2A). In both cases <br /> we observe a clear signal corresponding to site A4 in the <br /> samples isolated from LiCl-treated rrp5-D3 and rrp5-D4 <br /> mutant cells, while the signal corresponding to site A3 has <br /> disappeared. In particular the detection of site A4 with probe 7 <br /> is convincing evidence for the presence of a 27SA4 precursor. <br />    As shown in Figure 4 the A4 processing site maps to a <br /> region of the spacer that has been well conserved with respect <br /> to both primary and secondary structure in a wide variety of <br /> yeast species. The A4 site is located within a 5&#162;-UCUUUG-3&#162; <br /> consensus sequence that is part of a stem shared by all known <br /> yeast ITS1 regions. There is no obvious structural similarity <br /> between the A4 site and either the A2 or the A3 site. <br /> Synthetic lethality with rrp5-D4 identi&#174;es the REX4 gene <br /> In order to obtain more information on the manner in which <br /> Rrp5p functions in pre-rRNA processing we carried out a <br /> synthetic lethality screen with the rrp5-D4 mutation using the <br /> ade2/ade3 red-white colony sectoring assay, essentially as <br /> described previously (33) (see Materials and Methods for <br /> details). Strain YJV515 carrying the rrp5-D4 mutation as well <br /> as the wild-type RRP5 gene on an ADE3/URA3 plasmid were <br /> mutagenized by UV irradiation and surviving colonies were          Figure 3. Primer extension analysis of precursor intermediates in the rrp5-D3 <br /> screened microscopically for non-sectoring mutants, indicat-       and rrp-D4 cells. Total RNA was isolated from exponentially growing wild- <br /> ing the inability to lose the plasmid. The presence of a           type and mutant cells immediately before and 2 h after addition of 0.2 M <br /> mutation that is sl with the rrp5-D4 deletion was then             LiCl. Primer extension analysis was carried out using probes 4 (A), 6 (B) or <br />                                                                    7 (C; see Fig. 2A for location of the probes). The positions of the primer <br /> con&#174;rmed by streaking the non-sectoring mutants onto plates        extension stops corresponding to processing sites A2, A3 and A4 are <br /> containing 5-&#175;uorouracil which should inhibit growth because       indicated. In (B) and (C) only the samples from LiCl-treated cells are shown. <br /> of the obligatory presence of the URA3 plasmid carrying the <br /> wild-type RRP5 gene. In this manner we isolated 16 mutants <br /> that regained sectoring when transformed with a wild-type          partial (IRA2 and DEC1) and four complete ORFs (YOL077, <br /> RRP5 gene but not with an empty vector, demonstrating that         YOL078, YOL079, ATP19 and YOL080). By subcloning <br /> they carry an unlinked mutation, which is sl with the rrp5-D4      portions of this region the ability to restore sectoring in the <br /> mutation. These mutations were then characterized by trans-        Y11A6 strain was limited to a fragment that contained only <br /> forming the strains in question with a yeast genomic library       ORFs YOL079 and YOL080, which are present on opposite <br /> and recovering the library plasmid from resectoring colonies.      DNA strands and overlap almost completely. To distinguish <br /> Library plasmids containing the wild-type RRP5 gene were           between these two ORFs, we &#174;rst created a deletion that <br /> identi&#174;ed by PCR analysis using RRP5-speci&#174;c primers.              removed a large C-terminal portion of YOL080 without <br /> Restriction mapping and sequence analysis of the remaining         affecting YOL079. The resulting plasmid was unable to <br /> plasmids recovered from one of the sl strains (Y11A6)              restore sectoring in Y11A6 cells. Secondly, mutation of the <br /> identi&#174;ed a 10 kb region of chromosome XV containing two           translation start <span id='am-77' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-78' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-79'  ></span><span id='am-80' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span> of YOL079 did not affect the ability of <br />                                                                                        Nucleic Acids Research, 2002, Vol. 30 No. 19       4227 <br />  <br />  <br />                                                                               strain YJV306 led to the same result, showing that absence of <br />                                                                               Rex4p is also sl with the rrp5-D3 deletion (data not shown). <br />                                                                               Inactivation of REX4 restores normal ITS1 processing in <br />                                                                               the rrp5-D strains <br />                                                                               To study the functional relationship between Rrp5p and Rex4p <br />                                                                               we used strain YJV154 in which the genomic, wild-type RRP5 <br />                                                                               gene is under control of the inducible GAL promoter (31). This <br />                                                                               strain was crossed with the rex4 null mutant YAV41 (19) to <br />                                                                               give FVY8C (GAL-rrp5/rex4&#177;). The latter strain was then <br />                                                                               transformed with a plasmid carrying either the rrp5-D3 or <br />                                                                               rrp5-D4 mutant allele under control of the authentic RRP5 <br />                                                                               promoter. Transformants were grown on galactose and then <br />                                                                               shifted to glucose-based medium to block production of wild- <br />                                                                               type Rrp5p. For the FVY8C transformants this causes growth <br />                                                                               to stop after ~23 h (data not shown). Total RNA was isolated <br />                                                                               immediately before and 24 h after the shift and analyzed by <br />                                                                               northern hybridization as well as primer extension analysis; <br />                                                                               the results are shown in Figures 5 and 6, respectively. <br />                                                                                  As is evident from Figure 5, in cells expressing wild-type <br />                                                                               Rrp5p the absence of Rex4p does not in&#175;uence the formation <br />                                                                               of the mature rRNA species. When grown on galactose, the <br />                                                                               REX4 (YJV154) and rex4&#177; (FVY8C) cells have the same, <br /> Figure 4. Structural conservation around site A4 among various yeast          normal ratio of 5.8S:5.8SL rRNA as well as the same levels of <br /> species. (A) Sequence comparison of the region of ITS1 spanning site A4       18S and 25S rRNA (Fig. 5A, C and D, compare lanes 5 and 7 <br /> that is highlighted in (B). Nucleotides in upper case are part of the stem,   with lanes 1 and 3, respectively). Also, northern hybridization <br /> nucleotides in lower case form the loop of the conserved hairpin structure <br /> containing site A4. S.c., S.cerevisiae; S.t., <span id='am-54' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-55' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-56'  ></span><span id='am-57' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span>Saccharomyces</span> transvaalensis;   did not detect any signi&#174;cant differences in the levels of the <br /> T.d., Torulospora delbru&#200;ckii; K.m., <span id='am-16' about='obo:NCBITaxon_4911' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_4910 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-17'  ></span><span id='am-18' property="rdfs:label" content="Kluyveromyces marxianus" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Kluyveromyces fragilis" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Zygorenospora marxiana" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Guilliermondella marxiana" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Candida pseudotropicalis" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="NCBITaxon:39396" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Kluyveromyces cicerisporus" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Dekkeromyces marxianus" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces marxianus" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Candida kefyr" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Zygofabospora marxiana" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Zygosaccharomyces marxianus" datatype="xsd:string"></span>Kluyveromyces marxianus</span>; H.w.,           various processing intermediates between galactose-grown <br /> Hansenula wingei; A.t., Arxioma telluris; D.h., Debaromyces hansenii; Z.b.,   YJV154 and FVY8C transformants (data not shown), in <br /> <span id='am-6' about='obo:NCBITaxon_4957' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759 obo:NCBITaxon_4953'><span id='am-7' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Zygosaccharomyces bisporus" datatype="xsd:string"></span><span id='am-10'  ></span>Zygosaccharomyces bisporus</span>. (B) Schematic representation of the structure     agreement with previous observations (19). However, when <br /> of the S.cerevisiae ITS1 according to Yeh et al. (40). The stem&#177;loop <br /> structure formed by the sequence shown in (A) is highlighted. <br />                                                                               the transformants are made dependent upon the Rrp5pD3 or <br />                                                                               Rrp5pD4 mutant protein by growing them on glucose, the lack <br />                                                                               of Rex4p does have a surprising effect: in the absence of the <br />                                                                               Rex4p protein the shift in the ratio of 5.8S:5.8SL characteristic <br /> the plasmid to restore sectoring. Therefore, YOL080 clearly <br />                                                                               of the respective rrp5-D mutation does not occur (Fig. 5B, <br /> carries the mutation that is sl with rrp5-D4 in these cells. <br />                                                                               lanes 6 and 8). Moreover, the A4-E fragment, which can be <br /> Sequencing of the sl allele of YOL080 revealed a single <br />                                                                               clearly seen in the rrp5-D3 and rrp-D4 cells that carry an intact <br /> mutation that changes the codon for Lys274 to a stop codon. <br />                                                                               REX4 gene (Fig. 5B, lanes 2 and 4), does not accumulate in <br />    Recently, van Hoof et al. (19) showed that YOL080 <br />                                                                               their rex4&#177; counterparts (Fig. 5B, lanes 6 and 8). Thus, <br /> encodes a member of a set of &#174;ve related, non-essential, yeast <br />                                                                               inactivation of the REX4 gene corrects the abnormal process- <br /> proteins that includes Pan2p, the 3&#162;&#174;5&#162; exonuclease involved                  ing within ITS1 caused by the rrp5-D3 and rrp-D4 mutations. <br /> in initial shortening of the poly(A) tails of mRNA (36). The                  There is no change in the levels of the mature, high-molecular- <br /> other four members were called Rex1p, Rex2p, Rex3p and                        weight rRNA species in the rrp5-D3/rex4&#177; or rrp5-D4/rex4&#177; <br /> Rex4p, the last one being the product of YOL080. The Rex                      mutants (Fig. 5C and D). <br /> proteins also contain the 3&#162;&#174;5&#162; exonuclease signature domain                     Figure 6 shows the results of primer extension experiments, <br /> at their C-terminus (37,38). Rex1p&#177;Rex3p were indeed found                    which are in full agreement with the conclusion drawn from <br /> to be 3&#162;&#174;5&#162; exonucleases, which have distinct, but over-                      the northern hybridization data. Using primer 4 (Fig. 2A) we <br /> lapping, roles in 3&#162;-end maturation of several rRNA, snRNA                    analyzed RNA isolated from YJV154 and FVY8C cells <br /> and snoRNA species (19). A rex4 null mutant, however, does                    carrying the rrp5-D3 or rrp-D4 plasmid either before or 24 h <br /> not display any detectable disturbance in RNA maturation (19;                 after a shift from galactose to glucose. Cells containing the <br /> our unpublished data), leaving the enzymatic activity of the                  intact REX4 gene show a clear reduction in the signal <br /> Rex4p protein to be established.                                              corresponding to the stop at A3 after the shift and the <br />    We crossed strain YJV515 (rrp5-D4) with strain YAV41 in                    concomitant appearance of the stops corresponding to the A4 <br /> which REX4 had been inactivated by insertion of the                           site (compare lanes 2 and 4 with lanes 1 and 3, respectively). <br /> kanamycin gene (kindly provided by Dr van Hoof). Analysis                     However, in the rex4&#177; transformants we do not see any change <br /> of 20 different tetrads demonstrated that viable spores                       in the signals corresponding to the A3 and A4 cleavage sites <br /> producing cells that were auxotrophic for histidine (thus                     upon repression of the wild-type RRP5 gene (compare lanes 6 <br /> containing the rrp5-D4 allele) invariably were also sensitive to              and 8 with lanes 5 and 7, respectively). We conclude that the <br /> geneticine (indicating the presence of an intact REX4 gene).                  Rex4p protein is essential for redirecting ITS1 processing in <br /> This further supports the existence of a genetic interaction                  both the rrp5-D3 and rrp5-D4 mutant from the normal (A3) to <br /> between the RRP5 and REX4 genes. A similar cross using                        the alternative (A4) pathway. <br />  4228    Nucleic Acids Research, 2002, Vol. 30 No. 19 <br />  <br />  <br />  <br />  <br /> Figure 5. Effect of inactivation of REX4 on formation of mature rRNA in <br /> rrp5-D3 and rrp-D4. Strain FVY8C carrying a genomic wild-type RRP5 <br /> gene under control of the GAL promoter, a rex4 null allele and a plasmid- <br /> encoded rrp5-D3 or rrp-D4 gene under control of the authentic RRP5 <br /> promoter was shifted from galactose- to glucose-based medium and total <br /> RNA was isolated immediately before and 24 h after the shift. The RNA <br /> was subjected to northern analysis using probes complementary to the <br /> mature 5.8S, 18S and 25S rRNAs [(A), (C) and (D), respectively] as well as <br /> probe 3 (see Fig. 2A) located directly upstream from cleavage site A3 (B). <br /> Strain YJV154, which is isogenic with FVY8C except for the presence of a <br /> wild-type REX4 gene, was used as a control. Lanes 1, 3, 5 and 7, RNA <br /> from galactose-grown cells. Lanes 2, 4, 6 and 8, RNA from cells shifted to <br /> glucose for 24 h. The nature of the products detected is indicated. <br />  <br />  <br /> Inactivation of REX4 in cells dependent on the Rrp5pD3 <br /> or Rrp5pD4 protein leads to severe underproduction of <br /> 80S ribosomes <br /> The observation that rex4&#177; cells expressing either of the <br /> Rrp5pD mutant proteins possess normal levels of the pre- <br /> rRNA intermediates and mature rRNA species came as a <br /> surprise in view of the sl phenotype of such cells. In order to              Figure 6. Effect of inactivation of REX4 on ITS1 processing in rrp5-D3 and <br /> see whether the sl phenotype might nevertheless be due to a                  rrp-D4 mutant cells. Strain FVY8C carrying a genomic wild-type RRP5 <br />                                                                              gene under control of the GAL promoter, a rex4 null allele and a plasmid- <br /> disturbance in ribosome biogenesis we carried out sucrose                    encoded rrp5-D3 or rrp-D4 gene under control of the authentic RRP5 <br /> gradient analysis of extracts from YJV154 (GAL-rrp5/REX4)                    promoter was shifted from galactose- to glucose-based medium and total <br /> as well as FVY8C (GAL-rrp5/rex4&#177;) cells carrying the wild-                   RNA was isolated immediately before and 24 h after the shift. The RNA <br /> type RRP5 gene or either of the two rrp5 deletion alleles on a               was subjected to primer extension analysis using probe 4, located between <br />                                                                              sites A3 and B1 (see Fig. 2A). The position of the stops corresponding to <br /> plasmid. All strains were initially grown on galactose and then              cleavage sites A2, A3 and A4 is indicated. Lanes 1, 3, 5 and 7, RNA from <br /> shifted to glucose-based medium for 24 h prior to preparation                galactose-grown cells. Lanes 2, 4, 6 and 8, RNA from cells shifted to <br /> of the extracts. As shown in Figure 7 the rrp5-D3 and rrp-D4                 glucose for 24 h. <br /> mutations themselves already have an effect on ribosome <br /> biogenesis. The sucrose gradient pro&#174;les of glucose-grown <br /> YJV154 transformants that carry either of the plasmid- <br /> encoded rrp5 deletion alleles reveal a signi&#174;cant de&#174;cit in                  in agreement with the relative growth rates of cells expressing <br /> 60S subunits apparent from the lower 60S and 80S peaks                       either protein (31; this paper). <br /> relative to the 40S subunit peak (Fig. 7, compare C and E with                  Introduction of the rex4 null mutation into cells producing <br /> A). We have con&#174;rmed this by analyzing the polysome pro&#174;le                   wild-type Rrp5p has no detectable effect on the ribosome <br /> of these cells, which in the case of the rrp5-D3 mutant clearly              pro&#174;le (Fig. 7B). A distinct change can be seen, however, in <br /> shows the presence of `halfmer&apos; polyribosomes, mRNA                          glucose-grown FVY8C (GAL-rrp5/rex4&#177;) cells expressing the <br /> molecules carrying a single 40S subunit in addition to one                   Rrp5pD3 or Rrp5pD4 mutant protein (Fig. 7D and F). Apart <br /> or more 80S ribosomes, which is characteristic of the                        from the decreased level of 60S subunits discussed above, <br /> underproduction of 60S subunits (Fig. 7H). We did not detect                 these cells show a dramatic reduction in the amount of 80S <br /> halfmer polyribosomes in the rrp5-D4 mutant (Fig. 7I)                        ribosomes. Despite the absence of detectable abnormalities in <br /> suggesting that the effect of this mutation on 60S subunit                   pre-rRNA processing in these cells, therefore, ribosome <br /> formation is less severe than that of its D3 counterpart. This is            biogenesis is severely disturbed, which explains the sl <br />                                                                                         Nucleic Acids Research, 2002, Vol. 30 No. 19       4229 <br />  <br />  <br />                                                                                speci&#174;cally involved in 18S rRNA synthesis, whereas the <br />                                                                                N-terminal region, encompassing 12 S1 RNA-binding motifs, <br />                                                                                is crucial for production of the 5.8SS rRNA. Thus, the <br />                                                                                C-terminal domain might interact with the snoRNP complex <br />                                                                                that carries out the early processing cleavages, while the <br />                                                                                N-terminal domain assists RNase MRP. The data presented in <br />                                                                                this paper constitute direct experimental proof for the latter <br />                                                                                suggestion as they demonstrate that removal of either S1 <br />                                                                                motifs 3&#177;5 (rrp5-D3) or 5&#177;8 (rrp5-D4) leads to a complete <br />                                                                                block in the RNase MRP-directed cleavage at site A3 (Fig. 3). <br />                                                                                   Interestingly, our experiments reveal a second effect of the <br />                                                                                rrp5-D3 and rrp-D4 mutations, namely the occurrence of a <br />                                                                                hitherto unobserved processing event in ITS1 at a site about <br />                                                                                midway between A2 and A3 which we have designated A4 <br />                                                                                (Fig. 3). The simultaneous presence of substantial amounts of <br />                                                                                an A4-E fragment (Fig. 2D, E and F), normal amounts of the <br />                                                                                D-A2 fragment (Fig. 2H) and, in particular, the detection of <br />                                                                                the A4 site in primer extension experiments using probes 6 and <br />                                                                                7 complementary to different regions of ITS2 clearly support <br />                                                                                the conclusion that the rrp5-D3 and rrp-D4 mutations cause <br />                                                                                the processing machinery to bypass A3 and instead to convert <br />                                                                                the 27SA2 pre-rRNA into an alternative, novel 27SA4 <br />                                                                                precursor species. <br />                                                                                   The currently available data do not allow us to decide <br />                                                                                whether formation of the 27SA4 species is an endo- or an <br />                                                                                exonucleolytic event. However, we favor the former possibil- <br />                                                                                ity for the following reasons. First, the processing intermediate <br />                                                                                having its 5&#162;-end at A4 accumulates in the presence of LiCl, <br />                                                                                which strongly inhibits the major 5&#162;&#174;3&#162; exonucleases Xrn1p <br />                                                                                and Rat1p known to be involved in pre-rRNA processing <br />                                                                                (Figs 2 and 3). Second, the 27SA2 pre-rRNA, the probable <br />                                                                                immediate precursor of the 27SA4 species, appears to be a <br />                                                                                poor substrate for these exonucleases, in any case, at least <br />                                                                                under normal conditions (13,14). Finally, as shown in Figure 4, <br /> Figure 7. Effect of inactivation of the REX4 gene on ribosome biogenesis <br />                                                                                the A4 site is located in a region of ITS1 that has been <br /> in rrp5-D3 and rrp-D4 cells. Cultures of YJV154 (GAL-rrp5/REX4) and            relatively well conserved with respect to both primary and <br /> FVY8C (GAL-rrp5/rex4&#177;) cells carrying either the wild-type RRP5 gene or        secondary structure over a wide spectrum of yeast species, <br /> one of the rrp5-D deletion alleles on a plasmid were shifted from galactose-   suggesting that it could be the recognition site for an <br /> to glucose-based medium and extracts were prepared 24 h later. 20 OD260        endonuclease, the nature of which remains to be identi&#174;ed. <br /> units of extract were analyzed on 15&#177;35% (A&#177;F) or 10&#177;50% sucrose <br /> gradients (G&#177;I). Sedimentation is from right to left. (A, C and E) Extracts    There is no obvious structural similarity of this region with <br /> from strain YJV154 carrying the wild-type RRP5 (A), rrp5-D3 (B) or             those containing other known processing sites within the pre- <br /> rrp-D4 (C) allele. (B, D and F) Extracts from strain FVY8C carrying the        rRNA. It should also be noted that processing at A4 might be <br /> wild-type RRP5 (B), rrp5-D3 (D) or rrp5-D4 (F) allele. (G&#177;I) Polysome          less precise than the standard processing events. In all cases <br /> pro&#174;les for YJV154 carrying the wild-type RRP5 (G), rrp5-D3 (H) or <br /> rrp5-D4 (I) allele. The arrows in panel H indicate the `halfmers&apos;. <br />                                                                                we observe a set of three bands corresponding to consecutive <br />                                                                                nucleotides, whose relative intensity varies somewhat from <br />                                                                                one experiment to another (Figs 3 and 6). <br />                                                                                   Although the rrp5-D3 and rrp-D4 deletions cause identical <br /> phenotype of combining the rrp5-D3 or rrp5-D4 and rex4&#177; <br />                                                                                changes in ITS1 processing the fate of the resulting precursors <br /> mutations. <br />                                                                                differs in the two mutants. In the rrp5-D3 strain subsequent <br />                                                                                processing almost exclusively follows the `long&apos; pathway <br />                                                                                leading to 5.8SL rRNA. In the rrp5-D4 mutant, on the other <br /> DISCUSSION <br />                                                                                hand, substantial processing still occurs via the `short&apos; <br /> Genetic depletion of Rrp5p blocks synthesis of both 18S                        pathway, presumably by exonucleolytic degradation of the <br /> rRNA, requiring the snoRNP-dependent cleavages at A0&#177;A2,                       27SA4 precursor, resulting in ~40% 5.8SS rRNA. We propose <br /> as well as 5.8SS rRNA, which depends upon the RNaseMRP-                        that the presence of mutant Rrp5p protein causes a structural <br /> directed cleavage at A3. This led to the hypothesis that the                   alteration in the processing complex that reduces either the <br /> major role of Rrp5p is to ensure functional integration of the                 rate of entry of the exonucleases or the rate of exonucleolytic <br /> many trans-acting factors required for accurate and ef&#174;cient                   digestion, allowing processing at site B1L to get the upper <br /> pre-rRNA processing at these sites (30). Subsequent muta-                      hand. Clearly, the extent of this negative effect on B1S <br /> tional analysis (31,32) demonstrated that the C-terminal                       processing is different for the two mutant proteins, being <br /> region of the protein, containing seven TPR motifs, is                         largest for the rrp5-D3 mutation. <br />  4230   Nucleic Acids Research, 2002, Vol. 30 No. 19 <br />  <br />  <br />    A synthetic lethality screen using the rrp5-D4 mutation         its cleavage at site A3. The involvement of Rrp5p in this <br /> resulted in the isolation of the REX4 gene, which encodes a        processing step seems to be rather complex, however, because <br /> protein belonging to a <span id='am-61' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> of non-essential 3&#162;&#174;5&#162;               the requirement for the S1 motifs in question can be abrogated <br /> exonucleases. Although other members of this family were           by removing Rex4p. The data also indicate a further role for <br /> shown to be involved in pre-rRNA processing, no evidence for       Rrp5p, either directly or indirectly, in the subsequent <br /> such a role was found in the case of the Rex4p protein (19).       exonucleolytic processing to site B1S and demonstrate a <br /> The data presented in Figures 5 and 6, however, clearly            negative effect of the two deletion mutations on 60S subunit <br /> demonstrate that inactivation of the REX4 gene does have a         biogenesis. Furthermore, both genetic and biochemical evi- <br /> rather surprising effect on pre-rRNA processing in strains         dence clearly indicates the involvement of Rex4p in the <br /> expressing either of the mutant Rrp5pD proteins. Both the          assembly of yeast ribosomes, albeit in an unusual manner. <br /> rrp5-D3/rex4&#177; and rrp5-D4/rex4&#177; double mutants show normal <br /> ITS1 processing as well as a wild-type 5.8SS:5.8SL ratio. We <br />                                                                    <br /> conclude that the change in ITS1 processing induced by the <br /> rrp5-D3 and rrp-D4 mutations requires intact Rex4p and that        <br /> in the absence of Rex4p the mutant processing complex              <br /> containing either mutant Rrp5p protein regains the ability to      <br /> direct RNaseMRP to the A3 site. The molecular basis for this      <br /> phenomenon remains a matter of speculation. Rex4p could        <br /> participate directly in ribosome biogenesis, a conceivable <br /> hypothesis in view of the fact that its human homolog resides <br />                                                                   <br /> in the nucleolus (39). Another possibility is that Rex4p plays a <br /> role in the formation of an as yet unidenti&#174;ed trans-acting         <br />                                                                  <br /> factor that forms part of the processing complex containing        <br /> Rrp5p. In wild-type cells, however, the role of Rex4p is  <br /> critical.                                                         <br />    The lack of any detectable abnormalities in pre-rRNA         <br /> processing in the rrp5-D3/rex4&#177; and rrp5-D4/rex4&#177; double           <br />                                                                   <br /> mutants left us without an obvious explanation for the sl            <br /> phenotype of these mutants. Therefore, we considered the           <br /> possibility of a defect in ribosome assembly, which for            <br /> obvious reasons we studied in strains in which the wild-type        <br />                                                                      <br /> RRP5 gene is conditionally expressed. Sucrose gradient              <br /> analysis demonstrated that in fact the rrp5-D3 and rrp-D4            <br /> mutations by themselves already cause a signi&#174;cant defect in        <br /> 60S subunit assembly. Extracts from cells expressing the             <br /> Rrp5pD3 protein show a clear de&#174;cit in the large, relative to   <br />                                                                    <br /> the small, subunits as well as the characteristic presence of           <br /> halfmer polyribosomes (Fig. 7). This could be a consequence           <br /> of the almost exclusive presence of 5.8SL rRNA in these cells       <br /> which might be assembled less ef&#174;ciently than its smaller             <br />                                                                       <br /> counterpart, similar to 3&#162;-extended 5.8S rRNA (17). This <span id='am-64' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-65' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-66'  ></span><span id='am-67' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>idea</span>        <br /> is supported by the fact that the de&#174;cit in 60S subunits, in         <br /> particular as judged from the absence of detectable amounts of        <br /> halfmers, is less severe in the rrp5-D4 mutant, which contains       <br /> a higher proportion of 5.8SS rRNA (31; this paper). On the         <br />                                                                       <br /> other hand, the structural abnormality of the Rrp5p protein            <br /> might also directly affect 60S subunit assembly in the            <br /> mutant cells.                                                         <br />    Sucrose gradient analysis of extracts prepared from the       <br />                                                                   <br /> rrp5-D3/rex4&#177; and rrp5-D3/rex4&#177; double mutants revealed a         <br /> very low amount of 80S ribosomes (Fig. 7). The combination        <br /> of mutant Rrp5p and the absence of Rex4p, therefore, appears     <br /> to cause a severe defect in ribosome assembly, which would             <br /> explain the sl phenotype of the double mutants. This &#174;nding             <br />                                                                    <br /> further supports the hypothesis that Rex4p is involved in             <br /> ribosome biogenesis, although its role only becomes manifest         <br /> in the presence of the mutant Rrp5p proteins. We are presently    <br /> studying the role of Rex4p in ribosome biogenesis further             <br />                                                                     <br /> using both genetic and biochemical approaches.                      <br />    In summary, the data presented in this paper &#174;rmly establish       <br /> the role of the S1 domain of Rrp5p in assisting RNaseMRP in         <br /> </body></html>