<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright 0 1988 Genetics Society America <br />  <br />  <br />  <br />  Conditional Mutants RPCl60, Gene Encoding Largest Subunit of <br />               RNA Polymerase C <span id='am-1' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-2' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-14'  ></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> <br />  <br />  Rosmarie Gudenus,&apos; Sylvie Mariotte, Alejandra Moenne, Anny Ruet, Sylvie Memet, Jean-Marie <br />                        Buhler, Andre Sentenac Pierre Thuriaux <br />  Service Biochimie, Dbpartement Biologie, C.E.N. Saclay, Commissariat I&apos;Energie Atomique, F91191 G$ sur Yvette (cedex), <br />                                                                France <br />                                             Manuscript received September 24, 1987 <br />                                                   Revised copy accepted March 12, 1988 <br />  <br />                                                               ABSTRACT <br />                  A 18.4-kb fragment yeast genome containing gene largest subunit RNA <br />               polymerase C (RPCI6O) cloned hybridization previously isolated fragment gene. <br />               RPG160 maps chromosome XV, tightly linked allelictotheessentialgene             TSM8740. <br />               Temperature sensitive (ts) mutant alleles constructed vitro mutagenesis NaHSOs <br />               substituted wild-type allele chromosome.     Four wereunambiguously <br />               identified rpcl60 mutants failure complement fully defective mutation rpcl60:: URA3. <br />               The faithful transcription yeast tRNA gene mutant cell-free extracts strongly reduced as <br />               compared wild-type. I n vivo, rpcl60 mutations specifically affect synthesis tRNA a <br />               temperature sensitive way, comparatively little effect synthesis 5s rRNA no <br />               effect on5.8s rRNA. An unlinked mutation (pcil-3)suppresses thetemperature sensitive phenotype <br />               rpc160-41 mutation. <br />  <br />  <br />  <br /> I   N yeast eukaryotes, genes of <br />      small RNAs tRNA 5 s rRNAare <br /> specifically transcribed nuclear <br />                                                                         yeast RNA polymerase C participates in <br />                                                                         transcription genes corre- <br />                                                                         sponding tRNA 5s rRNA. Given impor- <br /> RNA polymerases, RNA polymerase C (or 111). <span id='am-48' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-51'  ></span><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span>                        tance yeasts model systems tostudymajor <br /> enzyme extensively purified contains                       cellular processes eukaryotes, undertaken <br /> fourteen distinct polypeptides (hereafter called sub-                   isolation characterization mutantspro- <br /> units), sharedwith RNA polymer-                       ducing conditionally defective RNA polymerase C <br /> ase A RNA polymerase A B (SEN-                         S. cerevisiae. Ten genes potentially corresponding <br /> TENAC 1985). T h e promotersrecognized               RNA             distinct subunits RNA polymerase C been <br /> polymeraseC       intragenic.For        transfer RNAs               cloned immunological screening (RIVAet al. 1986). <br /> corresponding tononsensesuppressors,mutations                           One (RPC160) encodes largest subunit <br /> with defective promoters lead antisuppressor                      C160 identical gene previously identified <br /> phenotype relatively easily                     homology structural gene largest <br /> isolated characterized yeasts <span id='am-18' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-19' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-24'  ></span>Saccharomyces</span>                  subunit RNA polymerase B Drosophila melano- <br /> cerevisiae (ALLISON   et al. 1983) SchizosaccharomyceJ              gaster (INGLES    et al. 1984). [RNA polymerase subunits <br /> pombe (PEARSON    et al. 1985).The intragenic promoters                 symbolized capital A, B C followed by <br /> are directly recognized RNA polymerase C                         number correspondingto estimated molecular <br /> itself, additionalproteins(transcription           fac-          weight x 10&apos;. Several symbols havebeen used to <br /> tors) added thepurified RNA                            designate corresponding genes (RUET      et al. 1980; <br /> polymerase C achieve faithful transcription vitro                 INCLES   et al. 1984; RIVAet al. 1986; NONETet al. 1987). <br /> (KLEKAMP     WEIL1982; KOSKIet al. 1982; RUETet                     Pending general agreement this, others <br /> al. 1984; CAMIER    et al. 1985) thereforeessen-                (MANNet al. 1987) adopted nomenclature <br /> tial gene         specificity RNA polymer-                    proposed RIVAet al. (1986).] Both genes a <br /> ase C.                                                                  striking sequence homology gene encoding <br />    I n spite extensive enzymological immuno-                     largestsubunit         bacterial RNA polymerase <br /> logical studies (SENTENAC     1985), little known              (ALLISON    et al. 1985).We report theconstruction <br /> the interaction RNA polymerase C                        temperature-sensitive mutant alleles RPCl60 <br /> cognate transcription factors. Moreover,       clear          affect synthesis tRNAs vivo vitro, <br />                                                                         isolation extragenic suppressor restoring <br />    &apos; <br />    Present address: Immuno AG, Uferstrasse 15, A2304, Orth der       growth restrictive temperature theprecise <br /> Donau, Austria.                                                         localization RPC160 gene chromosome <br /> Genetics 119 517-326 (July, 1988). <br />  518                                                      R. Gudenus et al. <br />  <br />                            TABLE 1                                  described SHERMAN, FINK HICKS          (1979). The <br />                           Yeast strains                             proteinase-less phenotype pep4-3 mutation fol- <br />                                                                     lowed method Of JOHNSTON (1977) using 2% com- <br />                                                                     mercial powder milk caseine source.Strain D27-7C <br />       Name                            Genotype <br />                                                                     carries pep4::HIS3 null allele (kindly sent C. WOOL- <br />                                                                                         + <br />  <br />  <br />  <br />  <br /> YNN281&quot;           ura3-52 his3-200 trpl-901 lys2-801 ade2-101       FORD) instead ofpep4-3.      Free spores canRICAN+ <br />                      MATa                                           heterozygous diploids obtained digesting asci <br /> YNN281canRb       ura3-52 his3-200 trpl-901 lys2-801 ade2-101       100-fold diluted <span id='am-33' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-34' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-35'  ></span><span id='am-36' property="rdfs:label" content="Helix" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Helix</span> pomatia juice (Industrie Biolo- <br />                      canR MAT                                       gique Francaise) forat 1 hrat 30&#176;, checking for <br /> YNN282&quot;           ura3-52 his3-200 trpl-901 lys2-801 ade2-101       complete digestion microscope separating <br />                      MAT                                            aggregated sporesby sonication. The spores plated <br /> YNN281-15         uru3-52 his3-200 trpl-901 lys2-801 ade2-I01       selective mediumcontaining canavanine isolate <br />                      rpc160-15 MATa                                 canR haploid segregants (KLAPHOLZ      EASTON-ESPOSITO <br /> YNN281-20         ura3-52 his-3-200 trpl-901 lys2-801 ade2-101       1982), occasional formation MATclIMATa diploids <br />                      rpcl60-20 MATa                                 mating adjacent spores plating. Yeast <br /> YNN281-31         ura3-52 his3-200 trpl-901 lys2-801 ade2-101       transformation according ITO et al. (1983). <br />                      rpcl60-31 MATa                                 Integrative transformants symbolized :: the <br /> YNN281-41         ura3-52 his3-200 trpl-901 lys2-801 ade2-I01       symbol  integrated <br />                                                                            of <br />                                                                                                     DNA fragment     (e.g., <br />                      rpcl60-41 MATa                                 rpcl60:: URA3 ). Transformants distinguished from <br />                                                                                   + <br />  <br />  <br />  <br />  <br /> D27-7C            ura3-52 his3-200 trpl-901 lys2-801 ade2-I01       potential revertants suppressors (such obtained when <br />                      rpc160-41 MATa pep4 ::HIS3+ MAT                transforming amber lys2-801 mutants) testing the <br /> YNN281-41-3       urd-52 his3-200 trpl-901 lys2-801 ade2-I01        mitotic instability transformedcharacter and, if <br />                      rpcI60-41 MATa pcil-3 MATa                     needed, genomic hybridization analysis appro- <br /> D25-13A           his3-200 trpl ade2-I01 pep4-3 rpc160-31 MATa      priate &quot;P-labeled probe. <br /> 382-23A&apos;          spoll ura3-52 can1 cyh2 ade2hi57 h o d MATa          Plasmidsandphages: Phage <span id='am-113' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-114' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-117' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span> A EMBL3 <br /> 393-35C&apos;          spoll ura3-52 his2 leu1 lysl met4 pet8 MAT        (FRISCHAUF   et al. 1983), pSP65 (MELTON   et al. 1984), YRp7 <br /> 381-9D&quot;           spol1 ura3-52 ade6 arg4 a707 asp5 met14 lys2      (PARENT,    FENIMORE    BOSTIAN1985) YCp631 <br />                     pet1 7 trpl MAT                                 (BARNESandTHORNER           1986) previously de- <br /> 396-22B&apos;          spoll ura3-52 adel his1 leu2 lys7 met3trp5        scribed. pEMBLYi32 similar pEMBLYi21 contains <br />                     MAT                                             pUC19 polylinker (BALDARI CESARENI           1985 and <br /> Be3 14d           tsm 8740 MATa                                     personal communication). Plasmid p02 pPBR322 de- <br /> NC93SPl&apos;          ura3-351,273,328 MATa                             rivate 1881 bp EcoRV PvuII fragment has <br /> T12-2B(pSP4)      ura3-52 his3-200 trip1 -901 lys2-801 ade2-IO1     deleted tandem du lication 635-bp ClaI <br />                     rpcl60 ;: URA3+ MAT (pSP4)                      fragment carrying tRNA$yeast           gene[originating <br /> 20B-12f           pep4-3 trpl MAT                                   plasmid pY20 (FELDMAN,OLAHandFIEDENREICH <br /> D25-13A           pep4-3 trpl his3-200 ade2-I01 MAT                 1982)] inserted ClaI site (0.       GABRIELSEN <br /> CMY214&apos;           YNN281 canR X YNN282                              C. MARCK,personal communication). Several plasmids <br /> D8                382-23A X 393-35C                                 carrying inserts overlappingwith RPC160 constructed <br /> D9                382-23A X 396-22B                                 study (Figure 1). pSPl derived pSP65 and <br /> Dl0               382-23A X 381-9D                                  carries 2.5-kb EcoRI insert RPC160 interrupted by <br /> T2                rpcl60 :: URA3 + transformants CMY2 14         1. l-kb Hind111 URA3 fragment inserted by <br />                                                                                             + <br />  <br />  <br />  <br />  <br />                                                                     blunt-end ligation StuI site. YIp32-1 derives from <br /> TDU               rpcl60::URA3 + transformants DU <br /> TD9                              + <br />                   rpcl60 :: URA3 transformants D9                   pEMBLYi32 insertion 5.3-kb XbaI fragment <br />                                                                     starts 1.2 kb upstream initiation <span id='am-53' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-54' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56'  ></span><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span> ofRPCI60 <br /> TDlO              rpcl60 :: URA3 + transformants D 10 <br />                                                                     ends coding sequence. Finally, pSP4 carries <br />     Yeast Genetic Center, Berkeley.                                 14.4-kb fragment containingthe RPC160 gene <br />   &apos; MANNet al. (1987).                                              insertedat Sal1 site YCp631. This fragment was <br />    KLAPHOLZand EASTON-ESPOSITO  (1982).                             isolated genomic library (M. SNYDER,       unpublished <br />    HILGER,PREVOTand MORTIMER(1982).                                 <span id='am-103' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-104'  ></span><span id='am-105' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-107' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span>) yeast strain X2180C contains Sau3A <br />               unpublished data. <br />    F. LACROUTE, <br />                                                                     partialfragments cloned BamHIsite A <br />   JONES (1976).                                                     EMBL3 (this library obtained C. MANN).Recom- <br />                                                                     binantphage plasmids obtained standard <br />                                                                     techniques (MANIATIS,    FRITSCH  SAMBROOK     1982). Plas- <br /> X V . I n parallel work, mutations RPC40 gene              mids kept amplified <span id='am-61' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-62' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-75'  ></span><span id='am-76' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span>Escherichia coli</span> strains <br /> (which encodes subunit shared RNA polymerase                   RR1 HB 101 (MANIATIS,      FRITSCH SAMBROOK      1982). <br /> A C) obtained, shown t o                     For NaHSOs mutagenesis, used E. coZi strain JM107 <br /> affect vivo assembly RNA polymerases                 (YANISCH-PERRON,VIEIRA             and <br />                                                                                                       MESSING   1985) prepare <br /> (MANN et al. 1987). Temperature-sensitivemutants                    single stranded DNA ung strain BW310 [defective <br />                                                                     uracil N-glucosylase (DUNCAN,    ROCKSTROH     WARNER <br /> of thegeneencodingthelargestsubunit            yeast              1978)] amplification mutagenized DNA. <br /> RNA polymerase B described recently                     Media andgrowth conditions S. cerevisiae: Yeast <br /> (HIMMELFARB, SIMPSON       FRIESEN1987; NONETet                 strains grown 30&quot; using standard media growth <br /> al. 1987).                                                          conditions (SHERMAN, FINK HICKS      1979). Temperature <br />                                                                     sensitive mutants grown YPD 25&quot; tested at <br />                                                                      37&quot;. Nutritional requirements determined onminimal <br />              MATERIALS AND METHODS <br />                                                                      medium SD supplemented appropriate nutrients <br />    Yeast strains: The yeast strains used study           (SHERMAN, FINK HICKS      1979). Except tryptophan, <br /> listed Table 1. The standard genetic techniques yeast        amino acid requirements met 0.5% casamino <br />                                      Polymerase <br />                                           RNA                Mutants Yeast                                                             5 19 <br />  <br /> acid. Respiratory deficient mutants, resistance canavan-                   E         E    E E            E            E            E <br />                                                                  B    B <br /> ine cycloheximidewere tested described                   I <br /> KLAPHOLZ    EASTON-ESPOSITO     (1982). Colonies resistant   1                                                                          1 <br /> to 5-fluoroorotate selected described BOEKE, <br />                                                                               I <br />                                                                               n <br />                                                                                   I I <br />                                                                                   nn <br />                                                                                                                                  I1 <br />                                                                                                                                  n <br /> LACROUTE     FINK (1985) using 50-mm Petri dishes in-                 K                                         K            s <br /> cubated 5 days 25&quot;. Sporulation solid <br /> medium K-acetate (10 g/liter) supplemented the <br /> appropriate nutrients quarter concentration <br />                                                                                                  Fii                    pspl <br />  <br />  <br /> used SD medium.                                                                            URA 3 <br />    NaHSOSmutagenesis: Single-stranded DNAwas pre- <br /> pared 100-ml culture J M l O l (YIp32-1) super-                                   1 <br />                                                                                             -i                          yIp32- <br />  <br /> infected phage IRl described DENTE,CESARENI                                       E E       St   E <br />                                                                                                                     C <br /> and CORTESE    (1983). 10 pg DNA treated 10 min at <br /> 37&quot; 3.0 M NaHSO3 (SHORTLE       BOTSTEIN   1983) and <br /> the mutagen removed spun-column chromatogra- <br /> phy 1-ml Sephadex G-50 medium column. Upon <br /> transformation E. coli strain BW310, 2000           L                                                                I   PSP4 <br />                                                                       1                                         1 <br /> pooled ampicillin-resistant transformants harvested              B                                         B                 S <br /> in LB supplemented ampicillin double-stranded <br />                                                                     FIGUREI.-Simplified restriction map 18.4-kb region <br /> plasmid DNAwas prepared alkaline lysis CsCl <br />                                                                  spanning RPCl60. The black arrow corresponds coding <br /> density gradient (MANIATIS,    FRITSCH  SAMBROOK    1982). <br />                                                                  sequence RPCl60 (ALLISON   et al. 1985). Endonuclease cleavage <br /> The uru3-52 yeast strain YNN281 transformed by <br />                                                                  sites symbolized B (BamHI), K (KfinI), H (HindIII), E <br /> about 10 pg mutagenized plasmidDNA linearized at <br />                                                                  (EcoRI), S (SalI), X (XbaI), St (StuI) C (ClaI). The inserts <br /> theunique CZaI site promote genomic integration by <br />                                                                  corresponding plasmids (pSP1, YIp32-1, YRp7-160 and <br /> targeted crossing-over R P C l 6 0 gene (ORR-WEAVER, <br />                                                                  pSP4) used study drawn scale lower of <br /> SZOSTAK   ROTHSTEIN     1981). About 5000 transformants <br />                                                                  figure. <br /> selected uracil omission medium scored tem- <br /> perature-sensitivity replica plating YPD 25&quot; and <br /> 37&quot;. <br />    I n vivo labeling [14C]lysine [5,6-&apos;H]uracil:Cul- <br />                                                                  Cells washed distilled waterand disrup- <br /> tures (25 ml) yeast(about 2 X lo7 cells/ml) exponentially     tion buffer (200 mM Tris-HC1 pH 8.0, 1 mM EDTA, 10 <br /> growing 25&quot; SD medium (supplemented casa-             mM mgC12, 10 mM 2-mercaptoethanol, 10% v/v glycerol, 4 <br /> <span id='am-89' about='obo:NCBITaxon_245039' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491 obo:NCBITaxon_9170'><span id='am-90'  ></span><span id='am-91' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="Mino" datatype="xsd:string"></span>mino</span> acids 20 mg/ml adenine, trypotophan            mM phenylmethylsulfonylfluoride)and finally resus- <br /> uracil) cultivated 30 min water baths        pended 40 mlof thatbuffer. After disruption a <br /> at 25&quot; 37&quot;. The cellswere filtered 0.45-pm            Manton-Gaulin homogenizer, ionic strength ad- <br /> filters transferred medium       justed 300 mM ammonium sulfate. The cell lysate was <br /> uracil concentration 0.5 mg/ml. [&apos;HIUracil (5.104 Bq/        left 15 min ice centrifuged 90 min 35,000 <br /> ml) [&apos;4C]lysine(2.104Bqiml)         addedandthe         rpm. The transcription assaywas final volume <br /> cultures grown 45 min. In pulse-chase           40 pl, presence reaction mixture which <br /> experiments,the labeling pulse 15 min             contained 40 mM HEPES pH 7.8,70 mM KCl, 5 mM MgC12, <br /> chased 30 min adding unlabeled uracil 200         10 mM dithiotreitol, 0.1 mM EDTA, 10 v/v glycerol, 3 mM <br /> mg/ml.Aliquots (1 ml)were precipitated 4 ml 10%               ATP, 3 mM GTP, 3 mM CTP, 0.15 mM [&quot;PIUTP (6.7 lo3 <br /> trichloracetic acid, filtered 0.45-prn filters, thoroughly    Bq/nmol), 0.1 pg plasmid p02 containing yeast <br /> washed 5% trichloracetic acid counted scintil-       tRNA?&apos;&quot; gene varying concentrations cell-free ex- <br /> lation acid-precipitable 14Cand &apos;H activity. The incor-       tract. Incubation 25&quot; 45 min. The synthesized <br /> poration &apos;H insmallRNA          (5.8S, 5 s tRNA)       RNA separated electrophoresis 6% polyacryla- <br /> measured extracting total RNA follows. Cells resus-         mide gelin presence 7 M ureaand subjected to <br /> pended 5 ml extraction buffer (50 mM Tris-HC1 pH <br />                                                                   autoradiography described above. <br /> 7.4, 10 mM EDTA, 0.10 M NaCl 5% sodium dodecyl <br /> sulfate) crushed Eaton press extracted twice <br /> with 5 ml phenol-chloroform-isoamyl alcohol mixture                                         RESULTS <br /> (24:24: 1). The nucleic acids aqueous phase were <br /> precipated ethanol resuspended 100 to200 pI <br />                                                                    Cloning genetic localization 18.4-kb re- <br /> of water. Aliquots corresponding 5 20 pg RNA          gion spanning gene RPCl60 onchromosome X V : <br /> vacuum dried, dissolved 20pl 90% formamide             The 2.5-kb EcoRI fragment RPClGO described by <br /> run 6% polyacrylamidegel containing 7 M urea             RIVAet al. (1986) corresponds half <br /> 0.09 M Tris-Borate (pH 8.3)with 2.5 M EDTA 90 min            coding sequence (Figure 1 ALLISON    et al. <br /> at 200 V. The gel fixed 7% acetic acid, immersed <br /> for 30 min Amplifier T M (Amersham) dried 80&quot;          1985). To isolate gene, fragment was <br /> under vacuum. Labeled RNA bands detected                 used probe genomic library strain X2 18OC <br /> autoradiography using kodak X-Omat S film,            S . cerevisiae phage A EMBL3 (M. SNYDER, <br /> quantified densitometry.                                      unpublisheddata).Two      partly overlapping clones <br />    Preparation cell-free extracts vitro assay      analyzed restriction mapping using 18 dif- <br /> specific transcriptionof tRNA genes: Cultures (5-liter)      ferent enzymes. They define 18.4-kb region which <br /> of strains 20B-12 (pep4-3,trpl), D25-13A (rpc160-31, <br /> pep4-3, trpl) D27-7C (rpc160-41, pep4::HZS?+ )          bears gene. A simplified restriction map is <br /> grown YPD collected centrifugation optical       given Figure 1 (a detailed map available on <br /> density 2.0 600 nm, yielding 30 g wet cells.      request). <br />  520                                                 R. Gudenus et al. <br />     The 3.6-kp EcoRI fragment containing proxi-                                              TABLE 2 <br />  mal R P C l 6 0 interrupted 1.1-kb URA3  + <br />  <br />  <br />  <br />  <br />                                                                 Localization rpcl60 right arm chromosome XV by <br />  insert (Figure 1) subsituted                                    tetrad analysis <br />  wild-type alleles R P C l 6 0 ura3-52 diploid <br />  strain CMY214. In experiment,the yeastcells                               Segregation genetic markers <br />  transformed linearized EcoRI fragment <br />                                                                                     RPCl60                RPCl60               tsm8740 <br /> and URA3 transformants selected onthe <br />             + <br />  <br />                                                                                          X                   X                         X <br /> appropriate medium (ROTHSTEIN          1983). One trans-                           tsm8740                 ade2                    ade2 <br /> formant (strain T2) analyzed <span id='am-108' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-109' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-111'  ></span><span id='am-112' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>hybrid</span>-                               P T N P                 T N P           T        N <br />  ization ofits genomic DNA EcoRI digestion,              Our data     68           4       0       142 21    0          59    13     0 <br />  using 2.5-kb EcoRI fragment RPCl6O ra-              HILGER, PR~VOT                                                 38    13     0 <br /> dioactive probe. In theuntransformedstrain,                 and <br /> hybridization revealed 2.5-kb band corre-               MORTIMER <br /> sponding R P C l 6 0 + alleles, whereasan           Pooled data                                                    97    26     0 <br /> additional 3.6-kb band corresponding inter-             Average map        (1.2-7.3) <br />                                                                                  2.8     6.4                (3.0-9.8) <br />                                                                                                                    10.6            (7.0-14.2) <br /> rupted rpcl60:: URA3 allele present strain <br />                         + <br />                                                                 distances <br /> T 2 (data shown). Fourteen tetrads      dissected        Map distances given centimorgans (% recombinants) <br /> after sporulationof T2. All showed 2:2 segregation           confidence interval 95% given parentheses. P stands <br />                                                                parental ditypes, T tetratypes N nonparental <br /> of viable spores (auxotrophic uracil) andtwo           ditypes. Confidence intervals calculated assuming Poisson <br /> lethal spores,as expected rpcl60 : : URA3 allele <br />                                                 +              distribution. Average map distances calculated data pooled <br /> were haplolethal agreement equivalent              crosses using ts alleles small duplication <br />                                                                R P C l 6 0 (see RESULTS) <br />                                                                                           including 41 tetrads analyzed by <br /> results INCLES et al. (1984). Since sporelethality      HILGER,PR~VOT       MORTIMER(1982), ignoring possible <br /> observed reflect defective germination          heterogeneities differences genetic back- <br /> than afailuretoundergo            vegetative growth,        grounds. <br /> constructed haploid strain T12-2B (pSP4)            chromosomes. About 100 viable haploid segregants <br /> carries rpcl60 :: URA3 chromosomal markerbut <br />                        + <br />                                                               isolated transformed diploids <br /> is viable contains functionalRPClGO         described KLAPHOLZand EASTON-ESPOSITO <br /> geneonthecentromeric              plasmid pSP4 (CEN3,         (1982). The mutantand wild-type allele each <br /> LYS2+, RPCl6O+).T12-2B cultivated               marker segregated roughly equal frequencies       in <br /> ten generations liquid YPD, plated YPD              offspring diploid transformants TD8,TD9 <br /> about 2 X 10&apos; colonies wereexamined sponta-               TD10, ade2 pet1 7 both <br /> neous plasmid loss leading lysine auxotrophy. No           located chromosome XV. For markers, <br /> auxotrophic colony detected, control            offspring 100% wild type 100% <br /> experiment RPCl6O strain carrying pSP4 <br />                                 + <br />                                                               mutant, depending diploid strain considered. <br /> yielded about5%auxotrophs.This             indicates     This indicates RPCl60 itself located chro- <br /> pSP4 belost rpcl60 : : URA3 background, <br />                                         + <br />                                                               mosome XV. <br /> showing R P C l 6 0 allele essential <br />                             +                                    T o precisely localize RPCl6O chromosome XV, <br /> to viability vegetatively growing cells.               constructed haploid strain T4-1 (ura3- <br />     To <br />      map          RPCl60&apos; chromosome, <br />                               ato                          52, his3-A200,ade2-101) carrying haplolethal <br /> rpcl60 :: URA3 allele constructed tech- <br />                  +                                            allele rpcl60::URA3          andthe <br />                                                                                              +       wild-type allele <br /> nique described diploid strains (D8,           R P C l 6 0 + tandemduplication.     In strain, <br /> D9 D10) homozygous ura3-52              R P C l 6 0 genetically marked URA3 insert                 + <br />  <br />  <br />  <br />  <br /> s p o l l , heterozygous variety genetic         butthe haploid mutant viable the <br /> markers located 16chromosomes                  adjacent RPCl60 allele. Preliminary results indi- <br />                                                                                      + <br />  <br />  <br />  <br />  <br /> identified S . cerevisiae (Table 1). The spoll muta-       catedthat RPCl6O meiotically linked with <br /> tion eliminates meiotic crossing-over com-            ADE2. This prompted cross T4-1 ura3-52 <br /> pletely preventing production viable haploid           tsm 8740 partner.This cross heterozygousfor <br /> spores meiotic offspring, providesa              RPC160, ADE2, TSM 8740 HIS3 all <br /> simple chromosome allocation technique (KLAPHOLZ              chromosome XV, TSM8740 close ADE2. <br /> and EASTON-ESPOSITO        1982). In particular case,    T h e segregation RPCl6O followed the <br /> the viable haploid spores harboring            segregation URA3 character. Seventy-two <br />                                                                                                      + <br />  <br />  <br />  <br />  <br /> the R P C l 6 0 allele consequently auxo- <br />              +                                                tetrads analyzed: 55were parental R P C l 6 0 , <br /> trophic uracil. Since meiotic crossing        ADE2 TSM8740. Four undergone crossing- a <br /> over,the R P C l 6 0 + allele present viable         RPCl6O TSM8740, remained <br /> spores cosegregate genetic markers            parental theRPCl60-ADE2 interval. The remain- <br /> present <span id='am-84' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-85' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-88'  ></span>cis</span> chromosome,             ing 13 undergone crossing ADE2 <br /> show linkage markers located             R P C l 6 0 remained parental theRPCIGO- <br />                                                      RNA Polymerase Mutants Yeast                                               52 1 <br />  <br />                              URA 3                                    FRIESEN 1987; NONET            et al. 1987). As sumarized <br />                                                                       Figure2, 5.3-kbXbaI fragment RPCl60 lacking <br />                                                                       386 nucleotides C-terminal end but <br />  <br />  <br />                        w <br />                                                                       including N-terminal endwas prepared single- <br />                                                                       stranded DNA plasmid YIp32-1 (URA3+), tak- <br />                                     I <br />                                                                       ing advantage       f l origin replication harbored <br />                                                                       plasmid (BALDARI CESARENI              1985). This <br />                 I ,                                                   DNA treated NaHS03and amplified by <br />                                                                       transformation ung strain E . coli generate <br />                        t.rg.1.d Intogntlon <br />                                                                       pool 2000 clones mutagenized double- <br />                                                                       stranded plasmids. The mutagenized plasmids were <br />                                                                       integrated genome          ura3-52 yeast strain <br />                                URA3          5&apos;       3&apos; <br />                                                                       YNN281 transformation targeted crossing- <br />                                                                       (ORR-WEAVER,          SZOSTAK     ROTHSTEIN      1981) <br />                Cla I                                Cldl <br />                                                                       RPC160 unique ClaI site. This generated a <br />                                 I                                     tandem duplication RPCl60, inactive <br />  <br />  <br />  <br />  <br />               - <br />                                                                       copy lacking C-terminal end functional <br />                                1                                      copy 3.7 kb coding sequence <br />                                                                        ( . e , , region proximal ClaI) arosefrom the <br />                                                                       mutagenized plasmid. A plasmid-born mutation will <br />                   5&apos;                3&apos; <br />                                                                       phenotypically expressed gen- <br />                                                                       omic integration provided located upstream <br />                mutrnt 811010    Ch10mOWm8l   bCUS                     ClaI site recombine functional <br />     FIGURE2.-Construction      fo chromosomal mutant alleles       copy RPCl60. Mutant            excised URA3   + <br />  <br />  <br />  <br /> RPC160 integrative crossover plasmid-born RPC160            plasmid fromthechromosome                   aspontaneous <br /> insert. A single-stranded preparation plasmid YIp32-1 contain-     crossover direct repeats RPC160 <br /> ing XbaI insert RPC160 lacking 3&apos; end mutagenized <br /> with NaHSOs amplified ung E . coli strain generate     selected described BOEKE, LACROUTE              and <br /> pool double-stranded mutant plasmids. Plasmid integration        FINK (1984). Provided thatthe               excisive crossing <br /> targeted crossover (ORR-WEAVER,    SZOSTAK  ROTHSTEIN 1981)        located upstream ts mutation in <br /> at ClaI site generates tandem duplication RPC160        RPCl60, mutation remain chromo- <br /> one truncated complete RPC160 sequence. Potential ts <br />                                                                        somal copy RPC160 plasmid excision. The <br /> mutations [symbolized (*)I present plasmid trans- <br /> mitted nontruncated RPC160 copy provided          final result precise substitution wild-type <br /> located upstream ClaI site (ORR-WEAVER,      SZOSTAKand        RPCl60 gene mutagenized RPCl60 fragment <br /> ROTHSTEIN1981; SHORTLE,NOVICKand BOTSTEIN1984). The                    (Figure 2). <br /> resulting transformant express ts phenotype.            From 5000 URA3 transformants isolated <br />                                                                                                        + <br />  <br />  <br />  <br />  <br /> Spontaneous excision plasmid intergenic crossover <br />                                                                        1 week incubation 25&quot;, showed little or <br /> between RPCIGO repeats restores single copy RPC160 <br /> which harbors ts mutation provided excisive crossing      growth replica plated YPD 37&quot; but <br /> over occured upstream mutated site.                             grew 25&quot;. The temperature-sensitive character was <br />                                                                        uracil auxotrophy. Tetrad analysis from <br /> TMS8740 interval. Ninety-one tetrads various                      cross ura3 tester strainNC93SP1 showed <br /> two-factor crosses ADE2 ts alleles                  that,for mutants,the ts charactersegregated <br /> RPCI6O (see below) analyzed. These                      independentlyfromthe              URA3 markerand was <br />                                                                                                               + <br />  <br />  <br />  <br />  <br /> data, sumarized Table 2, allow precise localiza-                  born integrated plasmid. How- <br /> tion RPCl60 chromosome X V , TSM                         ever, genetic evidence establishes re- <br /> 8740 ADE2, 3 cM marker                        maining ts mutations located integrated <br /> (Table 2). The tsm8740 mutation complemented                    YIp32-1 plasmid (Table3).(1)Theyare                  recessive, <br /> upon transformation thecentromeric LYS2 +,                          2:2 segregation, cosegregate URA3      + <br />  <br />  <br />  <br />  <br /> RPCl60 plasmid pSP4 notallelic to <br />           + <br />                                                                        strongly linked ade2. (2) Spontaneous plasmid <br /> RPCl60.                                                                excision strongly correlates reversion ts <br />    Construction temperature-sensitive mutant al-                    character. (3)Conversely, reversion ts mutation <br /> leles RPC160 random mutagenesis vitro:                        frequent (between5 X                   andlop3per <br /> Temperature-sensitive (ts) mutant alleles RPCl60                    cell) accompanied by <br /> were isolated anapproach previously usedto                          plasmid excision. Reversion plasmid excision <br /> obtain ts mutants genes encoding actin,   to-                  ascribed tointergenic conversion unequal <br /> poisomerase I1 largest subunit RNA poly-                     crossing directrepeats                  of <br /> merase B S. cerevisiae (SHORTLE, NOVICK   BOT-                  RPCI6O (KLEINand PETES1981). (4)For trans- <br /> STEIN 1984; HOLMet al. 1985; HIMMELFARB,      SIMPSON                  formant, obtained spontaneous subclones which <br />      522                                                                 R. Gudenus et al. <br />                                                                             TABLE 3 <br />                                                                    Properties t s mutants <br />  <br />                                        Doubling time                              Meiotic linkage&apos;                     Mitotic instability <br />                                                YPD <br />                                               (in hr)&quot;                            URA3    +       add                                 Fre uency (%) of <br />                                                                              ~~ <br />  <br />                                                                                                                  Frequency           Jasmid loss <br />                                                                                                                 reversion after <br />                                                                                                                             among <br />                                                                                                                                                  L   S   + <br />  <br />  <br />  <br />  <br />               Strain                    25&quot;              37&quot;                      P T N                          plasmid loss            revertants&apos; <br />  <br /> YNN281-15 (rpcl60-15)                  4.0               15                       160     0    151 0       17       22125 <br /> YNN281-20 (96160-20) 16            8.8 3.5                                           1*   0    16 1 0               21/25                       26 <br /> YNN281-31 (rp~160-31)                  4.3          &gt;20                            8 0    0     8 0 0               2 1/25                      62 <br /> YNN281-41 (rpc160-41)                  4.5            1230 <br />                                                     &gt;20                           150     0        29               24/25 <br />        YNN288: 2.5 hr (25&quot;)and 2.7 hr (37&quot;). <br />      &apos; P, T N defined Table 2. The lJRA3+ character allows follow segregation chromosomally integrated plasmid <br /> bearing ts mutation original construction. Tetrad analysis based asci viable spores. The tetrad labeled <br /> with (*) classified &quot;tetratype&quot; corresponded ascus temperature resistant spore prototrophic for <br /> uracil (ie., harbored plasmid YiP32-1).This segregant resulted spontaneous reversion ts allele concomitant <br /> plasmid excision, possibly intergenic conversion (KLEINand PETES1981). <br /> ~   Average thiee experiments set 50 revertants each. <br />  <br />  <br />  <br /> had excised plasmid remained ts (due                                      generations. Minor differences growth rates (Table <br /> crossover localized upstream ts mutation). As                                   3) suggest mutants correspond    todistinct <br /> expected, ts mutation stable                                      mutations. Cell morphology normal cell <br /> these subclones (less revertant lo6 cells),                               appeared arrested atvarious cell cycle stages as <br /> since intergenic conversion unequal crossing                                  judged percentageof budded cells the <br /> would longer possible copy                                      size buds. Diploid cells YNN281-41 X D27-7c <br /> RPCl6O left chromosome.                                                         (homozygous rpcl60-41) precultured 25&quot; <br />    These constructions yielded recessive mutant                                  liquid YPD plated sporulation medium at <br /> alleles (rpcl60-15, rpcl60-20, rpcl60-31 rpcl60-                                  temperature. This yielded 5% asci <br /> 4 1 ) result precise substitution                                (most mature four-spore asci) compared <br /> wild-type RPCl60 gene mutated gene. In <br />                            +                                                          45%of asci observed RPCl60+ diploid <br /> contrast initial transformants, genet-                                YNN281 X YNN282 experimental <br /> ically stable harbor foreign    DNA sequence                                   conditions. Practically asci observedwhen <br /> (originatingfromthetransforming         vector) inthe                                 mutant diploid precultured 30&quot; or <br /> vicinityof RPCl6O. Admittedly,the          ts mutations                               25&quot; followed 16-hr shift 37&quot;. <br /> could map gene fragment immediately proximal                                        We measuredtheincorporation        labeled <br /> to RPCl6O mutagenized insert. T o eliminate                                    uracil tRNAs, 5s rRNA 5.8s rRNA wild- <br /> this possibility, established rpcl60                                 type,rpc160-31 rpc160-41 cells exponentially <br /> mutants define single complementation group                                      growing 25&quot; cells shifted 37&quot; 1 hr. <br /> mating rpcl60-41 segregant andshowing                                     We used eithera45imin pulse a15-min pulse <br /> that resulting diploids temperature-sensitive.                                followed 30-min chase 400-fold excess of <br /> We crossed lys2-801, rpcl60-15 mutant                                       unlabeled uracil. Under conditions, was <br /> strain T12-2B (pSP4)which rpcl60:: URA3                   + <br />                                                                                       effect rpcl60-41 mutation protein syn- <br /> viable harborsthecentromeric        plasmid                                thesis total RNA synthesis measured by <br />  pSP4 (LYS2 +, R P C 1 6 0 + ) . The   resulting                                      incorporation [14C]lysine [3H]uracil acid <br /> rpcl60 : : URA3 +lrpc160-15 (pSP4) diploid tem-                                   precipitablematerial(datanotshown).       The incor- <br /> perature-resistant RPCl60 allele             +                      poration [3H]uracil tRNAs, 5 s rRNA 5.8s <br /> pSP4. However, spontaneous curing plasmid                                      rRNA 45-min pulse measured au- <br />  (leading tolysine auxotrophy) yielded ts phenotype,                                toradiography quantified densitometry (Fig- <br /> indicating rpcl60-15 notcomplemented                                       ure 3). As compared wild type, accumulation <br /> rpcl60 :: URA3 mutations <br />                        + <br />                                                                                       tRNAs reduced times rpcl60- <br />  define gene.                                                                41 mutant 25&quot; reduced factor <br />     Phenotype characterizationof ts mutants: The                                  cells shifted to37&quot; for <br />  ts mutantshardly grow 37&quot; anabout                                         1 hr 16 hr. There effect 5.8s rRNA <br />  twofold reduced growth rate 25&quot; compared                                    accumulation 25&quot; o r 37&quot; and, surprisingly, <br />  RPCl60+ strain YNN281. Upon temperature                                        little effect 5s rRNA accumulation 25&quot; at <br />  shift, growth (as measured    turbidometry 600                                 37&quot; 1 hr, rRNA known to <br />  nm) arrestedafterthreetofour           cell                              synthesized RNA polymerase C. The 5srRNA <br />      huI[  W T(16-c) <br />                                           LA <br />                                                   RNA <br />                                                    Mutants <br />                                                       Polymerase <br />  <br />  <br />  <br />  <br />                                           41 (irh 37&apos;C) <br />                                                                           ~   A <br />                                                                                   Yeast <br />  <br />                                                                                              llill               M <br />                                                                                                                  B <br />                                                                                                                                     323 <br />  <br />  <br />  <br />  <br />                                                                                                                  0       *a <br />       A                               n                                                                                   #   - <br />  <br />  <br />  <br />  <br />     k A Lk J ,4v   (15.c)                  41   (th .t 37&apos;C) <br />                                                                               1 2 3 4 <br />                                                                          FIGLRE   4.&quot;In vitro transcription yeast tRNA$&quot; gene in <br />                                                                       cell-free extracts RPC160 strain D20B- 12 the <br />                                                                       rpc160-31 strain D25-13A <br />                                                                                               5 6 7 8 <br />  <br />                                                                                                         + <br />  <br />  <br />  <br />  <br />                                                                                                 collected <br />                                                                                                                   9101112 <br />  <br />  <br />                                                                                                              log phase cultureson <br />                                                                       YPD 25&quot;. Panel A: 5 , 15, 25 35 pg protein wild- <br />   FIGURE3.-Incorporation    [&apos;HH-5,6]uracil small-sized       type cell-free extract (lanes 1-4) mutant cell free extract <br /> RNA 45 min vivo labeling 25&quot; 37&quot; strain      (lanes 5-8) assayed specific transcription tRNA&quot;&apos;&quot; <br /> YNN281 ( R P C 1 6 0 + ) D27-7C (rpc160-41). The R N A        gene plasmid p02 25&quot;. The R N A synthesized separated <br /> prepared, separatedona         polyacrylamide gel autoradi-       6% polyacrylamide gel (see MATERIALS ASD METHODS). The <br /> ographed described YATERIALS ASD METHODS. Autoradi-             arrow points tRNA band. Panel B: assay done <br /> ograms quantified densitometry. Left panel: wild type         presence 20 pg wild-type proteins (lane 9),of 20 k g <br /> (top) mutant (bottom) exponential cultures 25&quot;. Right     mutant proteins (lane lo), mixture (lane 11) and <br /> panel: mutant cells shifted 37&quot; 16 hr (top)or 1 hr (bottom).   40 pg wild-type proteins (lane 12). <br />  <br />  <br /> synthesis strongly reduced cells                    Selection extragenicsuppressorisolated <br /> transferredfor 16 hrat 37&quot;. Similar results                      therpcl60-41 strain: An exponential culture of <br /> obtained 15-min pulse chase,                 strain YNN281-41 (rpcl60-41) UV mutagenized <br /> indicating 5.8s rRNA, 3s rRNA, rRNA                      toabout 30% survival plated YPD 37&quot;. <br /> are stable mutant andwild-type cells (data                Colonies growing temperature appeared after <br /> not shown). Therefore, differences observed                    1 week frequency 1 10&apos; viable cells. <br /> the accumulation tRNA 5 s rRNA                          Ten reisolated doublingtime <br /> mutant cells likely entirely differences              YPD measured 37&quot;. Three grew rapidly <br /> in rate synthesis molecules. Similar                  wild-type strain gave ts segregants when <br /> results obtained rpcl60-31 mutant D25-                  backcrossed RPCI6O&apos; strain YNN282. They <br /> 13A (data shown).                                                 likely result reversion rpcl60-41 to <br />    RNA polymerase C activity cell-free extracts:                   R P C l 6 0 &apos;. Out seven thatgrew slowly <br /> Two mutants (rpc160-31 rpcl60-41) tested                     YPD 37&quot; (doubling times 360-400 min), one <br /> for activity RNA polymerase C cell-free                     particular revertant (Y NN28 1-41-3) backcrossed <br /> extracts cells exponentially growing 25&quot;, using                 rpcl60-41 strain D27-7C YNN282 <br /> a specific assay transcription tRNA genes.                     (RPC160 &apos;). Tetrad analysis showed thecapacity <br /> Figure 4 shows thatthetranscription         activity             grow 37&quot; monogenic (2:2) segregation in <br /> cell-free extract rpcl60-31 strain strongly                cross, YNN281-41-3 was <br /> reduced comparedto RPC160&apos; (a similar                          crossed YNN282, ts phenotype recovered <br /> somewhat markedeffect          observed                 25% spores. These results indicated <br /> rpcl60-41; datanot       shown). Mixing experiments                   capacity grow 37&quot; monogenic <br /> showed reduced activity                  suppressor rpcl60-41 genetically unlinked <br /> presence inhibitor transcription mutant               RPC160. The corresponding mutation been <br /> extract (Figure 4, lanes 9-12). The assay                    called pcil-3 (for polymerase C interaction). The <br /> at 25&quot;, similar results obtained                    growth rate pcil-3 RPC160&apos; segregant YPD <br /> extract incubated 37&quot;, showing marked                  25&quot; 37&quot; indistinguishable wild type <br /> temperature sensitivity mutant enzyme                    showing pcil-3 mutation de- <br /> these experimental conditions (data shown).                       tectable growth phenotype. This segregant be <br />  524                                              R. Gudenus et al. <br /> identified capacity restore original sup-    integrated plasmid, close linkage ADE2 on <br /> pressorphenotype crossed intoa rpc160-41              chromosome XV, frequentreversion spon- <br /> background.                                                taneous plasmid excision failure com- <br />                                                            plement rpcl60:: URA3 null allele. These four <br />                       DISCUSSION <br />                                                                                        + <br />  <br />  <br />  <br />  <br />                                                            alleles distinct mutations judged <br />    Previous work (INGLES     et al. 1984; ALLISONet al.    differences reversion patterns in <br />  1985;RIVA et al. 1986; THURIAUX        et al. 1986)   growth rates 25&quot; 37&quot;. <br /> identified RPC160 auniquegeneencodingthe                   The ts mutants analyzed (rpcl60- <br /> largest subunit RNA polymerase C. This gene         31 rpc160-41) defective tRNA synthesis <br /> amarked homology onesencodingthe                p&apos;  vivo cell-free extracts. The synthesis <br /> subunit E . coli RNA polymerase (ALLISON         et al. 5.8s rRNA catalyzed RNA polymerase A was <br />  1985) large subunitof RNA polymerase A         affected. As anticipated, fully defective rpcl60 <br /> B yeast (ALLISON     et al. 1985; RIVAet al. 1986).     mutants unable undergo spore germination <br /> R P C l 6 0 maps right arm chromosome XV,        (INGLES  et al. 1984) vegetative growth (this work). <br /> close allelic TSM8740, gene encoding        Yet, ts mutant rpcl60-41 fivefold <br /> presumably essential hitherto <span id='am-118' about='obo:NCBITaxon_32644' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-119' property="oboInOwl:hasRelatedSynonym" content="unclassified sequence" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasRelatedSynonym" content="none" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasRelatedSynonym" content="unknown" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasRelatedSynonym" content="unknwon" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasRelatedSynonym" content="unidentified organism" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasRelatedSynonym" content="not specified" datatype="xsd:string"></span><span id='am-125'  ></span><span id='am-126' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-128' property="rdfs:label" content="unidentified" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasRelatedSynonym" content="other" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasRelatedSynonym" content="sonstige nucleic acid" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasRelatedSynonym" content="unspecified" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasRelatedSynonym" content="not shown" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasRelatedSynonym" content="miscellaneous nucleic acid" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasRelatedSynonym" content="unknown organism" datatype="xsd:string"></span>unidentified</span> func-       reduction tRNA synthesis vivo 25&quot; less <br /> tion (HILGER,PREVOTand MORTIMER,1982). SPO-                twofold reduction growthrate rich <br /> T I 4 [which involved sporulation (TSUBOI mediumatthesametemperature,indicatingthat <br />  1983)] maps veryclose TSM8740 (possibly           tRNA synthesis (or step, the <br /> between R P C l 6 0 ) allelic    transcription thecorrespondinggenes         RNA <br /> RPC160 homozygous rpcl60 diploidshavea               polymerase C) major rate limiting factor in <br /> strongly reduced level sporulation 25&quot;. RIVAet       cellular growth. <br /> al. (1986) cloned 22 genes potentially             Temperature-sensitive mutants need ther- <br /> encoding various subunits RNA polymerase A, B           mosensitive forthe activity thecorresponding <br /> and C. Their restriction maps overlap         enzyme, instead affect synthesis (or its <br /> the 18.4-kb insert cloned present work (our         stability), especially case complex multimeric <br /> unpublished results). Thus, data        far  enzymes RNApolymerases. Several ts mutants <br /> suggest remarkable feature organization         defective assembly RNA polymerase have <br /> of surrounding region chromosome,           described E. coli (KIRSCHBRAUM       et al. <br /> certainly argue cluster genes encoding        1975). In yeast, MANNet al. (1987) isolated <br /> RNA polymerase subunits.                                   plasmid-born ts alleles RPC40, gene encoding <br />    Temperature-sensitive rpcl60 mutants ob-           AC 40 subunit common RNA polymerase A <br /> tained vitro mutagenesis NaHS03            C. The corresponding mutations prevent in <br /> non-replicative plasmid subsequently in-         vivo assembly RNA polymerase A C do <br /> serted chromosome integrative transfor-        affect activity preassembled enzyme. <br /> mation (SHORTLE,    NOVICKand BOTSTEIN        1984). This  A similar conclusion reachedforts mutations <br /> procedure somewhat cumbersome              gene encoding        largestsubunit ofRNA <br /> alternativeapproach      based plasmid exchange         polymerase B S. cerevisiae (HIMMELFARB,      SIMPSON <br /> (MANNet al. 1987;BUDD CAMPBELL1987)                FRIESEN 1987).     In contrast, mutation isolated <br /> has advantage yielding mutations         gene NONET et al. (1987) appears to <br /> been precisely substituted wild-type allele     affect activity intracellular accu- <br /> the chromosome, thanyielding mutations born         mulation enzyme, completely lacks RNA <br /> onan artificial extrachromosomalreplicon. As al-           polymerase B activity vitro cells grown <br /> readyobserved(SHORTLE, NOVICK BOTSTEIN permissive temperature. <br />  1984) ts mutants unlinked mutagenized gene            In present case, ts mutants analyzed <br /> were generated (2 5000 transformants).         additional fourfold decrease tRNA syn- <br /> One explanation unlinked ts mutants        thesis vivo hour shift 37&quot;. Thus, it <br /> integrative transformation restricted small      unlikely mutations merely prevent the <br /> cell subpopulation competentfor mitotic           assembly RNA polymerase C affecting the <br /> recombination (HENACT LUZZATI1972; MINET,              properties enzyme present temper- <br /> GROSSENBACHER-GRUNDER                 and <br />                                    THURIAUX    1980)   ature shift. In case, preexisting RNA <br /> which prone high level sponta-         polymerase molecules continuetoproduce <br /> neous mutation. However, observed mutagenic          RNA hoursbeforebecominglimiting, <br /> effect transformation spontaneous mutation        assuming RNA polymerase C unusually <br /> monitored resistance canavanine. In addition         labile enzyme. However, exclude the <br /> to unlinked ts mutants, ts alleles RPClGO      mutation itself temperature-sensitive effect on <br /> have identified cosegregation       stability enzyme (alone associated with <br />                                          RNA Polymerase Mutants Yeast                                                    525 <br /> its cognate transcription complexes)          possibility pczl-3 encodes protein directly <br /> direct effect transcriptional activity. In partic-   interacting C160 subunit, example a <br /> ular, hypothesis explain reduced        transcription factor subunit poly- <br /> RNA polymerase C activity mutant cell-free           merase. <br /> extracts particularly temperature-sensitive.            We thank CARLMANNand PIERRE      FROMAGEOT   stimulating <br />    A striking feature mutants analyzed        discussions encouragement. This work partly supported <br /> that largelyspecific forthe synthesis           long term EMBO Fellowship R.G. grant the <br /> tRNA, synthesis 5s rRNA eventually          Centre National la Recherche Scientifique (ATP 033-542). <br /> became arrested doubling times 37&#65533;?. <br /> This indicates mutantenzyme (and, possibly, <br /> the wild type form well) discriminates                           LITERATURE CITED <br /> structurally related initiation complexes formed be-        ALLISON,D. S., S. H. GOH B.D. HALL,1983 The promoter <br /> tween tRNA 5s rRNAgenes                    sequence yeast tRNA Tyr gene. Cell 34: 655-664. <br /> proteins acting transcription factors (SENTENAC          ALLISON,L. <br />                                                                      A.,      M. MOYLE,M. SHALESand C. J. INGLES, <br />                                                                1985 Extensive homology amongthe largest subunits of <br />  1985). One possibility mutationally modi- <br />                                                                eukaryotic prokaryotic RNA polymerases Cell 42: 599- <br /> fied C160 subunit specifically altered recogni-           610. <br /> tion site initiation complexes. Alternatively,      BALDARI,   C., G. CESARENI,     1985 Plasmids pEMBLY:new <br /> the affinity wild type RNA polymerase C            single stranded shuttle <span id='am-94' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-95' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-96' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>vectors</span> recovery analysis <br /> be lower tRNA initiation complexes,             yeast DNA sequences. Gene 35: 27-32. <br /> account reduced synthesis tRNAs ts        BARNES,   D.A., J. THORNER,     1986 Genetic manipulation of <br />                                                                 Saccharomyces cerevisiae use LYSZ gene. Mol.Cell. <br /> mutations lead decreased intracellular concen-             Biol. 6: 2828-2838. <br /> tration enzyme.                                      BOEKE,J. D., F. LACROUTE       G.R. FINK,1984 A positive <br />    Within ribosomal repeat unit, 5s rRNA                selection mutants lacking orotidine -5&#8217;-phosphate decar- <br /> gene lies major promoter element                boxylase activity yeast: 5-fluoro-orotic acid resistance. Mol. <br />                                                                 Gen. Genet. 197: 345-346. <br /> the initiation site large rRNA gene,          BUDD,M., J. L. CAMPBELL,     1987 Temperature-sensitive mu- <br /> transcribed opposite orientation (ELIONand WAR-              tations yeast DNA polymerase I gene. Proc. Natl. Acad. <br /> NER 1984). This suggests transcription          Sci. USA 84: 2838-2842. <br /> 5s rRNA gene RNA polymerase C berequired             CAMIER,  S., 0. GABRIELSEN,  R. BAKER   A. SENTENAC,     1985 A <br /> for thesynthesis ofthe 35s rRNA RNA polymerase               split binding site transcription factor tRNAsG&#8217;&#65533;? <br /> A, ensuring coordinate production              gene. EMBO J. 4: 491-500. <br />                                                             DENTE,L.,G. CESARENI      R. CORTESE,    1983 pEMBL: new <br /> types rRNA. However, data establish             <span id='am-30' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-31' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> single stranded plasmids. NucleicAcidsRes. 11: <br /> synthesis 5.8s rRNA (and, consequently,               1645-1655. <br /> 35s rRNA) affected severe reduction          DUNCAN,K.,  B. <br />                                                                         P.           A. ROCKSTROH       H. R. WARNER, <br /> the rate 5s rRNA synthesis. Therefore, pe-               1978 <span id='am-77' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-78' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-81'  ></span><span id='am-82' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span>Escherichia</span> coli K12 mutants deficient Uracil-DNA <br /> culiar organization rRNA repeat unit S .              glycosylase.J. Bacteriol. 134: 1039-1046. <br />                                                             ELION,E. A., J. R. WARNER,1984 The major promoter <br /> cerevisiae does result tight coordination           element rRNA transcription yeast lies 2 kb upstream. <br /> the transcription rDNA genes byRNA poly-                 Cell 39: 663-673. <br /> merases A C.                                            FELDMAN,   H., J. OLAH andH. FIEDENREICH,      1982 Sequence a <br />    In conclusion, present work provides             yeast DNA fragment containing achromosomal replicator and <br /> characterization conditional mutants specifically            tRNA Glu gene. Nucleic Acids Res. 9: 2249-2959. <br />                                                             FRISCHAUF,    A.M., H. LEHRBACH,     A. POUTSKA    N. MURRAY, <br /> defective RNA polymerase C. The properties                1983 Lambda replacement vectors carrying polylinker se- <br /> the mutant RNA polymerase C enzymes                   quences. J. Mol. Biol. 170: 827-842. <br /> analyzed biochemically <span id='am-58' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-59' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> identify mo-         H~NAUT     A., <br />                                                                          , M. LUZZATI,     1972 ContrBle I&#8217;aptitude A <br /> lecular defect leads temperature-sen-           recombiner pendant la phase vkgktativechez Saccharomyces <br /> sitive phenotype, parallel study            cerevisiae. Mol. Gen. Genet. 116: 26-34. <br />                                                             HILGER,F., M. P R ~ V Oand  T R. K. MORTIMER,         1982 Genetic <br /> gene RPC40 (MANNet al. 1987). Yeast genes corre-               mapping arg, cpa, <span id='am-98' about='obo:NCBITaxon_201850' typeof='owl:Thing obo:NCBITaxon_1125798 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71529 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-99' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-100' property="rdfs:label" content="Car" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-102'  ></span>car</span> tsm genes Saccharomyces cerevisiae. <br /> sponding subunits RNA polymerase C                 Curr. Genet. 6: 93-98. <br /> have cloned immunological screening (RIVA           HIMMELFARB,      H. J., E. M. SIMPSONand J. D. FRIESEN, <br /> et al. 1986) currently characterized           1987 Isolation characterization temperature-sensitive <br /> this laboratory. Still genes related RNA              RNA polymerase I1 mutants Saccharomyces cerevisiae. Mol. Cell. <br />                                                                Biol. 7: 2155-2164. <br /> polymerase C function identified selecting        HOLM,C., T. GOTO,J. C. WANCand D. BOTSTEIN,1985 DNA <br /> for extragenic suppressors rpcl60 ts mutants. In-           topoisomerase I1 required time mitosis yeast. <br /> deed, ts alleles described (rpcl60-41)         Cell 41: 169-173. <br /> is suppressible extragenic suppressor mutation,       INGLES, C. J., H. J. HIMMELFARB,   M. SHALES,  A. L. GREENLEAF   and <br /> pcil-3, itself detectable growth phe-          J. I3. FRIESEN,1984 Identification, molecular cloning and <br />                                                                mutagenesis Saccharomyces cereviriae RNA polymerase genes. <br /> notype. We indication asto molecular               Proc. Natl. Acad. Sci. USA 81: 2157-216. <br /> mechanismofthis        suppression, obvious         ITO, H., Y. FCKUDA,K. MURATA A. KIMURA. <br />  526                                                      R. Gudenus et al. <br />     1983 Transformation intact yeast cells treated alcali     PEARSON, D.,    I. WILLIS,H. HOTTINGER, J.       BELL,A. KUMAR,      U. <br />     ions J. Bacteriol. 153: 163-168.                                     LEUPOLD     D. SOLL,1985 Mutations preventing expression <br /> JOHNSTON, L. H., 1977 The use novel plate assay search         sup3 tRNASernonsense suppressors <span id='am-25' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-26' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-28'  ></span><span id='am-29' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span>Schizosaccharomyces</span> <br />     yeast mutants defective desoxyribonucleases. Mol. Gen.        pombe. Mol. Cell. Biol. 5: 808-815. <br />     Genet. 152: 219-222.                                              RIVA,M., S. MEMET,     J. Y. MICOUIN,J. HUET,I. TREICH,       J. DASSA, <br /> JOSES, E., 1976 Proteinase mutants Saccharomyces cerevztiae.          R. YOUNG,J. M. BUHLER,          A. SENTENAC      P. FROMAGEOT, <br />     Genetics. 85: 23-33.                                                 1986 Isolation structural genes yeast RNA polymerases <br />  KIRSCHBAUM,   J. B., I. V. CLAEYS,  S. NASI,B. MOLHOLTand J. H.         immunological screening. Proc. Natl. Acad.Sci. USA 83: <br />     MILLER,1975 Temperature-sensitive RNA polymerase mu-                  1554-1558. <br />     tants alteredsubunit synthesis degradation. Proc.        ROTHSTEIN,   R.L. 1983 One-step gene disruption yeast. Meth- <br />     Natl. Acad. Sci. USA 72: 2375-2379.                                  ods Enzymol. 101: 202-210. <br />  KLAPHOLZ,   S., R. EASTON-ESPOSITO,     1982 A new mapping       RUET,A., A. SENTENAC,        P. FROMAGEOT,    B. WINSORand F. LAC. <br />     method employing meiotic rec mutant ofyeast. Genetics              ROUTE,1980 A mutation B 220 subunit gene affects <br />     100: 387-412.                                                        structural functional properties yeast RNA poly- <br />  KLEIN, H. L., T. D. PETES,1981 Intrachromosomalgene                 merase B vitro. J. Biol. Chem. 255: 6450-6455. <br />     conversion yeast. Nature 289: 144-148.                         RUET, A., S. CAMIER,        W. S. MAGOWICZ,A. SENTENACandP. <br />  KLEKAMP,  M. S., P.A. WEILL,1982 Specific transcription          FROMAGEOT,       1984 Isolation <span id='am-45' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-46' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> C transcription factor <br />     homologous class 111 genes yeast-soluble cell-free extracts.      forms stable complex tRNA genes. EMBO J. 3: <br />     J. Biol. Chem. 257: 8432-8441.                                       343-350. <br />  KOSKI,R. A,, D. S. ALLISON,     M. WORTHINGTON     B. D. HALL,   SENTENAC,   A,, 1985 Eukaryotic RNA polymerases. CRC Crit. <br />      1982 An vitro RNA polymerase 111 Saccharomyces        Rev. Biochem. 18: 31-90. <br />     cereuisiae: effects deletions point mutations SUP4    SHERMAN,   F., G. R. FINK J.B. HICKS,1979 Methods Yeast <br />     transcription. Nucleic Acids Res. 10: 8127-8143.                     Genetics. Cold Spring Harbor Laboratory, Cold Spring Harbor, <br />  MANIATIS,T.,E.F. FRITSCHand J. SAMBROOK,            1982 Molecular      N.Y. <br />     Cloning: A Laboratory Manual. Cold Spring Harbor Laboratory,      SHORTLE,   D., D. BOTSTEIN,     1983 Directed mutagenesis with <br />     Cold Spring Harbor, N.Y.                                              sodium bisulfite Methods Enzymol. 100: 457-468. <br />  MANN, C., J. M. BUHLER,I. TREICHand A. SENTENAC,                     SHORTLE,   D., P. NOVICKand D. BOTSTEIN,           1984 Construction <br />      1987 RPC40, auniquegene asubunitshared                  genetic characterization temperature-sensitive mutant <br />     yeast RNA polymerases A C. Cell 48: 627-637.                      alleles yeast actin gene. Proc. Natl. Acad. Sci. USA 81: <br />  MELTON,D.A., P. A. KRIEG,M. R. REBAGLIATI,         T. MANIATIS, K.      4889-4893. <br />     ZINN M.R. GREEN,1984 Efficient invitro synthesis           THURIAUX    P., C. MANN,    J. M. BUHLER,  1. TREICH,   R. GUDENUS,  S. <br />     biologically active RNA RNA hybridization probes             MARIOTTE,      M. RIVAand A. SENTENAC,        1986 Gene cloning <br />      plasmids containing bacteriophage SP6 promoter. Nucleic            mutant isolation subunits RNA polymerases the <br />     Acids Res. 12: 7035-7056.                                             yeast Saccharomyces cerevisiae. pp. 5 19-53 1 . In Extrachromosomal <br />  MINET, M., A. M. GROSSE <br /> </body></html>