<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Systems Biology 3; Article number 106; doi:10.1038/msb4100147 <br /> Citation: Molecular Systems Biology 3:106 <br /> &amp; 2007 EMBO Nature Publishing Group All rights reserved 1744-4292/07 <br /> www.molecularsystemsbiology.com <br />  <br />  <br /> Genome-wide transcriptional plasticity underlies <br /> cellular adaptation novel challenge <br /> Shay Stern1, Tali Dror1, Elad Stolovicki2, Naama Brenner1 Erez Braun2,* <br /> 1 <br />   Department Chemical Engineering, Technion-Israel Institute Technology, Haifa, Israel 2 Department Physics, Technion-Israel Institute of <br /> Technology, Haifa, Israel <br /> * Corresponding author: Department Physics, Technion-Israel Institute Technology, Haifa 32000, Israel. Tel.: &#254; 972 48292879; Fax: &#254; 972 48295755; <br /> E-mail: erez@physics.technion.ac.il <br />  <br /> Received 11.12.06; accepted 16.2.07 <br />  <br />  <br />  <br />  <br />                Cells adjust transcriptional state accommodate environmental genetic perturbations. <br />                An open question extent transcriptional response perturbations specifically <br />                selected evolution. To test possibility transcriptional reprogramming does need to <br />                &#8216;pre-designed&#8217; lead adaptive metabolic state physiological timescales, confronted <br />                yeast cells novel challenge previously encountered. We rewired genome by <br />                recruiting essential gene, HIS3, histidine biosynthesis pathway foreign regulatory <br />                system, GAL network responsible galactose utilization. Switching medium glucose a <br />                chemostat caused repression essential gene presented cells severe challenge to <br />                adapted approximately 10 generations. Using genome-wide expression arrays, we <br />                global transcriptional reprogramming (41200 genes) underlies adaptation. <br />                A large fraction responding genes nonreproducible repeated experiments. These results <br />                nonspecific transcriptional response reflecting natural plasticity regulatory <br />                network supports adaptation cells novel challenges. <br />                Molecular Systems Biology 24 April 2007; doi:10.1038/msb4100147 <br />                Subject Categories: metabolic regulatory networks; chromatin &amp; transcription <br />                Keywords: adaptation; cellular metabolism; expression arrays; plasticity; transcriptional response <br />  <br />  <br />  <br /> Introduction                                                                     genes (Chu et al, 1998; Spellman et al, 1998; Hughes et al, 2000; <br />                                                                                  Jansen et al, 2002; Segal et al, 2003; Ihmels et al, 2004; <br /> Recent genome-wide expression measurements revealed                         Kharchenko et al, 2005). <span id='am-33' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-37'  ></span>This</span> regulatory mode been <br /> that broad range environmental perturbations gene                       selected evolution &#8216;pre-designed&#8217; allow transcrip- <br /> deletions lead global transcriptional response                   tional reprogramming meet specific environmental and <br /> sizeable fraction genome responds perturbation                  genetic challenges. An alternative mutually exclusive <br /> induction repression (Gasch et al, 2000; Jelinsky et al, 2000;                possibility massive transcriptional response to <br /> Causton et al, 2001; Ideker et al, 2001; Lai et al, 2005). The                   perturbations universal feature underlying regulatory <br /> hallmark large-scale responses rapid transient                     network, necessity large-scale interactions across <br /> followed relaxation, cell generation time,                      functional groups (intermodular interactions). A genetic or <br /> steady state (Gasch et al, 2000; Koerkamp et al, 2002; Wu et al,                 environmental perturbation causes response is <br /> 2004). It known, however, fraction genes                   largely nonspecific perturbation. This response of <br /> such massive transcriptional responses essential                       genetic network turn allows efficient plastic adaptation of <br /> specific cellular demands. Using known gene annotations,                     cellular metabolism broad range unforeseen challenges. <br /> fraction responding genes directly relevant perturba-                  There indications literature similar phenotypes, <br /> tion usually small, raising question                including similar metabolic states growth rates micro- <br /> is functional role rest. On general grounds,                 organisms, supported different underlying expression <br /> wonders transcriptional reprogramming following                      states (Fong et al, 2005). The relation metabolism and <br /> a perturbation specifically selected evolution                    transcription simple one-to-one, owing flexibility <br /> toward challenge, reflects fundamental large-scale                   robustness metabolic network cells generate <br /> connectivity dynamic characteristics transcriptional                  multiple transcriptional states achieve desired metabolic <br /> regulatory network (Kafri et al, 2005). If transcriptional                   condition. The question arises extent cells use this <br /> response specifically selected, expects                 transcriptional flexibility address unforeseen challenges on <br /> coexpression genes cofunctionality reflecting                        physiological timescales (not new mutations). Such <br /> specific demand imposed challenge. This requires                     freedom crucial high evolvability potential (Gerhart and <br /> degree overlapping regulatory cascades responding                       Kirschner, 1997; Kirschner Gerhart, 1998). <br />  <br /> &amp; 2007 EMBO Nature Publishing Group                                                                                      Molecular Systems Biology 2007 1 <br />  Transcriptional plasticity underlies adaptation <br /> S Stern et al <br />  <br />  <br />  <br />  <br />    To study generality plasticity transcriptional           A                                                           IV <br /> reprogramming, challenged yeast cells novel <br /> perturbation encountered                                           I                                       1 <br />                                                                                        1                              III <br /> history evolution. This approach eliminates possibility <br />                                                                                                                II <br />  <br />  <br />  <br />  <br />                                                                                                                                                 log2 expression <br /> that regulatory specifically selected to <br />  <br />  <br />  <br />  <br />                                                                        OD (600 nm) <br /> address perturbation. A strain yeast <span id='am-18' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-19' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-20'  ></span><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span>Saccharomyces</span> <br /> cerevisiae engineered recruit gene HIS3, essential                                                                           0 <br /> enzyme histidine biosynthesis pathway (Hinne-                                0.1 <br /> busch, 1992), GAL system, responsible galactose <br /> utilization (Stolovicki et al, 2006). The arbitrary regulatory <br /> linkage conserved highly specific                                                                               &#8211;1 <br />                                                                                      0.01 <br /> modules represents unforeseen perturbation cells. In                                10 gen <br /> our strain, HIS3 exclusive control GAL <br /> regulatory entirely detached natural                             &#8211;50        0    50          100     150     200 <br /> regulation. The GAL behaves strong switch                                       I                              IV       2 <br />                                                                        B <br /> heavily repressed switched glucose-containing <br /> medium (Johnston et al, 1994; Carlson, 1999; Braun                                  1 <br /> Brenner, 2004). Therefore, switching medium                                                                                      1 <br /> containing glucose lacking histidine, GAL system <br />  <br />  <br />  <br />  <br />                                                                                                                                                 log2 expression <br />                                                                        OD (600 nm) <br />                                                                                                                      III <br /> and HIS3 highly repressed immediately following                                               II <br /> the switch cells encounter severe challenge. We                        0.1                                                  0 <br /> recently shown cell population carrying rewired <br /> genome adapt grow competitively chemostat a <br /> medium containing pure glucose (Stolovicki et al, 2006). This                                                                              &#8211;1 <br /> adaptation population occurred timescale B10                          0.01 <br /> generations ability grow competitively                                 20 gen <br /> glucose stably inherited population. Moreover,                                                                                  &#8211;2 <br /> introducing 3-amino-triazole (3AT), competitive inhibitor                          &#8211;100       0    100    200          300     400 <br /> HIS3p, caused significantly larger environmental pressure                                                Time (h) <br /> and led eventually similar adaptive response, albeit on <br />                                                                    Figure 1 Adaptive population dynamics associated global transcriptional <br /> somewhat longer timescales (Stolovicki et al, 2006). It        response. Population dynamics measured cell density chemostat <br /> shown adaptation process involved tuning       (blue line) (A) 3AT (B) 40 mM 3AT. The population each <br /> HIS3 GAL genes transcription levels according       experiment exhibited phases (I&#8211;IV) dynamics depicted. Expression <br /> the inhibitor medium, mechanism        arrays measured time points course population <br />                                                                    adaptation following medium switch galactose glucose (at t&#188;0). <br /> this transcriptional reprogramming process remained un- <br />                                                                    A SOM clustering method (see Materials methods Supplementary <br /> known. Yeast cells encountered recruitment HIS3         Figure S1) led dominant global clusters: induced (red; a&#8212;543, b&#8212;701 <br /> to GAL evolutionary history       genes) repressed (green; a&#8212;692, b&#8212;998 genes). Note symmetry <br /> genome possibly selected specifically       induced repressed clusters (correlation coefficient two <br /> address glucose repression HIS3. Nevertheless,          clusters&#8217; mean expression profiles, a:  0.92, b:  0.98). The error bars present <br />                                                                    standard deviation expression values genes belonging each <br /> here adaptation process following switch           cluster. Note logarithmic scale. The generation time equals chemostat dilution <br /> glucose-based medium chemostat involves global            time   ln2B5 h. <br /> transcriptional response hundreds genes, emerging the <br /> onset medium switch relaxing timescale of <br /> the population adaptation. This global transcriptional re-         responses converged 10&#8211;20 generations stable <br /> sponse sensitive applied pressure HIS3p inhibition    steady state glucose. There phases population <br /> and underlies adaptation cells unforeseen   dynamics (Stolovicki et al, 2006). In phase I, The GAL genes <br /> challenge. The observed response largely nonspecific;           HIS3 linked repressed, OD transiently <br /> repeated experiments result low reproducibility        increased following switch glucose metabolism, to <br /> transcriptional states showing large fraction        existing resources (e.g. histidine) population the <br /> responding genes, enabling adaptation process,        previous galactose steady state. Then, population fitness <br /> are nonspecific challenge.                              decayed phase II improper expression levels HIS3, <br />                                                                    followed recovery generations (III) steady- <br />                                                                    state level (IV) cell density similar peak phase I. <br />                                                                    In phase IV, cells fully adapted grow pure glucose, <br /> Results                                                            tuning expression HIS3 appropriate level <br /> Figure 1 shows dynamics population&#8217;s cell density       according environmental pressure (level 3AT) <br /> (blue lines, measured optical density (OD) reflecting       (Stolovicki et al, 2006). <br /> the integrated metabolic state population) following         To study genome-wide transcriptional response, eight <br /> medium switch galactose glucose chemostat           samples cells harvested chemostat (see Materials <br /> without (A) (B) 3AT. The transient population             methods) time points different phases the <br />  <br /> 2 Molecular Systems Biology 2007                                                                               &amp; 2007 EMBO Nature Publishing Group <br />                                                                                                                          Transcriptional plasticity underlies adaptation <br />                                                                                                                                                            S Stern et al <br />  <br />  <br />  <br />  <br /> population adaptation analyzed expression arrays                           repression (Gasch et al, 2000; Nautiyal et al, 2002) <br /> the experiments, 3AT. Genome-wide                                 suggests global conservation principle, possibly <br /> measurements mRNA expression levels normalized                               competition cellular resources, involved dynamics <br /> their expression galactose steady state revealed                             genetic regulatory system. These results sensitive <br /> sizeable fraction genome responded induction                              clustering method (<span id='am-69' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-72'  ></span><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> shown). <br /> repression switch glucose (Figure 2). Superimposed                            We analyzed clusters appeared the <br /> on OD traces, Figure 1 shows results clustering                           transcriptional response based known annotations. In <br /> analysis (Tamayo et al, 1999; Shamir et al, 2005)                               cluster transiently repressed, genes of <br /> expression genes measured arrays time                            GAL system, exhibiting expected transient repression <br /> the experiments (see details clustering analysis                         switch glucose followed recovery the <br /> Supplementary Figure S1). This analysis revealed                                   expression, correlated population dynamics <br /> dominant clusters, containing hundreds genes                           (see Supplementary Figure S2 agreement previous k- <br /> experiment. The genes clusters responded                               PCR results; Stolovicki et al, 2006). An example transiently <br /> medium switch glucose strong transient induction                            induced modules provided glycolysis ribosomal <br /> repression followed relaxation steady state                               protein genes (Supplementary Figure S3). The glycolysis and <br /> timescale population adaptation process, approximately                          ribosomal genes known overexpressed glucose <br /> 10 generations. During relaxation period, external                            compared galactose (Yin et al, 2003). Surprisingly, following <br /> conditions kept constant chemostat. The                                transient induction, groups exhibited relatively fast <br /> clusters experiment characterized similar                           relaxation steady-state level, similar galactose <br /> opposite dynamics. The striking symmetry induction                             medium. More globally, characterized biological <br />  <br />  <br />  <br />  <br />                                               No 3AT                                                40 mM 3AT <br />                              1      2     3      4      5      6      7     8      1      2 3          4      5       6      7 8 <br />  <br />  <br />  <br />  <br />                            &gt; &#8211;2.51                                                                                              &lt;2.36 <br />                              Gal                            Glu                        Gal                    Glu <br />  <br />                                                                            Time <br /> Figure 2 The genome-wide transcription pattern. The raw transcription levels time points experiments, (left) 3AT, (right) 40 mM 3AT, color <br /> code: red&#8212;induced, green&#8212;repressed. There total 4148 genes passed filters (see Materials methods). The medium switch galactose to <br /> glucose marked numbers columns measurement points shown Figure 1. Note differences patterns expression for <br /> the experiments (rows correspond gene experiments). <br />  <br /> &amp; 2007 EMBO Nature Publishing Group                                                                                             Molecular Systems Biology 2007 3 <br />  Transcriptional plasticity underlies adaptation <br /> S Stern et al <br />  <br />  <br />  <br />  <br /> functions main clusters Figure 1, using                  carbon sources galactose glucose, regulatory <br /> Gene Ontology annotations (SGD) (Materials methods). In                        changes induced increase environmental pressure <br /> both experiments, transiently induced cluster sig-                         applied directly rewired gene, HIS3. Both populations <br /> nificantly enriched general energetic biosynthesis                         grown 3AT initiated similar steady- <br /> modules glycolysis, ribosome biogenesis, macromo-                          state patterns expression galactose (Supplementary <br /> lecule biosynthesis regulatory genes involved                      Figure S4). Thus, 3AT itself does cause significant <br /> the proteins translational machinery. By contrast, transi-                     modification expression state. This consistent with <br /> ently repressed cluster significantly enriched modules                     previous work showing 3AT does introduce significant <br /> that assigned DNA replication, DNA repair                               effects direct inhibition HIS3p, has <br /> regulation transcription. This distribution biological                       similar effect reducing HIS3 effective expression levels <br /> functionalities clusters suggests                         (Marton et al, 1998). The gene content overlap the <br /> induced cluster corresponds mainly cytoplasmic processes,                       transiently induced transiently repressed clusters both <br /> whereas repressed cluster processes nucleus                          experiments (with 3AT) partial, 15% (389 <br /> (Po0.005 cell compartment enrichment analysis). The                             genes 2545 assigned induced repressed <br /> functional enrichment analysis statistical method                     clusters), significantly larger expected by <br /> based tradition assigning well-defined functions                      chance (Supplementary Figure S1c). This partial overlap <br /> specific genes. This approach, however, explains small                      contains genes responding carbon source <br /> fraction global response shown below.                                    switch galactose glucose. This result indicates the <br />    One led ask observed expression dynamics                      switch carbon sources caused small the <br /> reflect known stress response (Gasch et al, 2000; Causton                        response, major effect HIS3 <br /> et al, 2001). In particular, repression HIS3 raises                     repression pressure. <br /> possibility cells amino-acid starvation                            To assess reproducibility transcriptional response, <br /> (Natarajan et al, 2001). We performed detailed                              repeated chemostat experiment 3AT. The <br /> comparison genomic transient response observed                         population density exhibited reproducible adaptation dy- <br /> here 3AT, different time points                         namics glucose steady state, nevertheless <br /> dynamics, published large-scale results known stress                        transcriptional response showed significant variations. We <br /> responses (Supplementary Table S1). The maximal number                          quantified degree reproducibility computing the <br /> induced genes overlapping transient response                        correlation coefficient expression patterns (Supple- <br /> our experiments stress response 39 31 (out                      mentary Figure S5). Within single experiment, correla- <br /> 201) 3AT 40 mM 3AT, respectively. Similarly,                    tion time-separated glucose steady states high <br /> number overlapping repressed genes 85 48 (out                        (R&#188;0.8&#8211;0.9). This high value close reproducibility in <br /> 320) 3AT 40 mM 3AT, respectively. This analysis                     duplicate arrays (R&#188;0.9). Even time points separated as <br /> shows transcriptional response time point                       237 h, correlation coefficient R&#188;0.9, showing <br /> our experiment low overlap universal                           stability pattern generations. By contrast, <br /> stress response observed wide range environmental                         different experiments, correlation coefficient is <br /> stresses (Gasch et al, 2000; Causton et al, 2001). It                  significantly lower (R&#188;0.32). The reproducibility transient <br /> correlated typical response amino-acid starvation                      transcriptional patterns showed similar trends. The correlation <br /> (Natarajan et al, 2001) (Supplementary Table S2). In particular,                   coefficient initial transient responses following <br /> the general amino-acid response (regulated Gcn4) (Hinne-                        medium switch glucose (phase I dynamics in <br /> busch, 1992) does operate experiments point                      Figure 1) repeated experiments low (Ro0.1) <br /> even high level 3AT applied. Additionally,                        compared high correlation transient points <br /> known specific responses stress characterized                            chemostat experiment (R&#188;0.5&#8211;0.9; Supple- <br /> transient induction repression relaxation steady                        mentary Figure S6). An additional way assess the <br /> state generation time (Gasch et al, 2000; Koerkamp                        reproducibility compare total number genes <br /> et al, 2002; Wu et al, 2004). Thus, given different                            significantly induced repressed case compute <br /> transcription patterns extremely long timescales                        degree overlap groups genes. Table I <br /> the transient response observed here, conclude                   shows number genes significantly induced (4twofold) <br /> a typical stress starvation response.                                           repressed (otwofold) repeated experiments the <br />    We used 3AT control parameter, enabled                         transients steady states. It shows numbers <br /> distinguish specific metabolic effects, including switch                    vary significantly experiments, consistent the <br />  <br />  <br /> Table I Comparison total number genes significantly induced repressed degree overlap groups genes different experiments <br />  <br /> Total number                No-3AT,                  No-3AT,             40 mM              Fraction overlap <br /> of genes                  repeat            second repeat           3AT                  repeats <br />  <br /> 3344                           418                    830                  524              7% (85 genes)                Transiently induced (42-fold) <br />                                514                    780                  960             14% (160 genes)               Transiently repressed (o2-fold) <br /> 3728                            90                    136                  217             10% (20 genes)                Induced steady state (42-fold) <br />                                262                    432                  464             30% (159 genes)               Repressed steady state (o2-fold) <br />  <br />  <br /> 4 Molecular Systems Biology 2007                                                                                       &amp; 2007 EMBO Nature Publishing Group <br />                                                                                                                                          Transcriptional plasticity underlies adaptation <br />                                                                                                                                                                            S Stern et al <br />  <br />  <br />  <br />  <br /> Figure 3 Environmental pressure leads highly correlated transcriptional response. (A) Color-coded figure transcriptional response time points the <br /> two experiments, 3AT (top) 40 mM 3AT (bottom). The genes ordered experiment according clusters presented Figure 1. The <br /> significant increase coherency response increase environmental pressure 3AT apparent image. Expression profiles genes belonging to <br /> glycolysis (B), histidine (C) purine (D) pathways. Note emergence highly correlated patterns transcription owing environmental pressure lower <br /> panel. A given functional module simultaneously contains correlated anti-correlated trajectories. <br />  <br />  <br />  <br /> behavior correlation coefficients. The fraction                                                                                                          No 3AT <br />                                                                                                                                                                    40 mM 3AT <br /> overlapping induced repressed genes different <br />                                                                                               Probability distribution <br />  <br />  <br />  <br />  <br /> experiments small (10&#8211;30%; Table I), significant                                                              0.04 <br /> enrichment biological functionality (P40.001) outside of <br /> the expected induction glycolysis genes repression <br /> of GAL genes. Thus, majority responding genes <br /> do overlap stage repeated experiments. We <br /> conclude non-negligible portion genes                                                                 0.02 <br /> changed expression adaptation process does <br /> not well-defined reproducible function the <br /> challenging environment. <br />    Our experiment enables characterize dynamics                                                             0.00 <br /> the transcriptional response addition gene content.                                                                   &#8211;1.0   &#8211;0.5      0.0         0.5         1.0 <br /> Upon switch glucose, average dynamics                                                                                       Corr co. <br /> transcriptional response experiments, and <br />                                                                                        Figure 4 The correlations transcriptional response. A Pearson correlation <br /> without 3AT, somewhat similar (Figure 1 Supple-                               coefficient computed pairs genes experiments, with <br /> mentary Figure S4). However, larger pressure applied                            3AT. The figure shows distributions correlation coefficient <br /> the 3AT results markedly higher correlation                             possible pairs, 3AT (black) 40 mM 3AT (red). <br /> temporal patterns hundreds responding genes. <br /> Figure 3A compares array data color code                            low-pressure (Supplementary Figure S7). Figure 4 <br /> experiments. It seen emergent pattern transcrip-                        compares distributions correlation coefficients among <br /> tion exhibited higher degree <span id='am-38' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> introduction                         genes experiments. Note significant peaks at <br /> high external pressure form 3AT; 25%                            high positive negative correlation edges 3AT <br /> gene pairs exhibited absolute correlation coefficient larger                           experiment compared distribution peaked zero <br /> than 0.9 high-pressure experiment compared 5%                             correlation experiment 3AT. <br />  <br /> &amp; 2007 EMBO Nature Publishing Group                                                                                                             Molecular Systems Biology 2007 5 <br />  Transcriptional plasticity underlies adaptation <br /> S Stern et al <br />  <br />  <br />  <br />  <br />    Observation transcriptional temporal trajectories                    specific one. Interestingly, genes belonging same <br /> specific metabolic pathways experiments illustrates                     metabolic pathway exhibited simultaneous positively and <br /> different contributions correlated dynamics (Figures 3B&#8211;                    negatively correlated dynamics. Even genes encoding for <br /> D). A general energetic module, glycolysis, exhibited                      enzymes residing linear metabolic pathway exhibited <br /> similar patterns induction relaxation experi-                       negatively correlated dynamics (Ihmels et al, 2004; Kharch- <br /> ments (Figure 3B). This, however, general                              enko et al, 2005). These general features 3AT <br /> behavior. We one-third known                           experiment, highly correlated expression dynamics and <br /> metabolic gene modules (30 88 modules described                          simultaneous positively negatively correlated trajectories, <br /> KEGG (Kanehisa et al, 2004), Po0.05, Materials                             appeared genes belonging protein complex <br /> methods) exhibited high correlation expression                      (Jansen et al, 2002). For example, Figure 5 shows the <br /> genes environmental pressure high (40 mM 3AT,                         expression trajectories nucleotide excision repair <br /> average absolute correlation 0.5570.06),                       (NER) protein complex. The generality results are <br /> low (no 3AT, average absolute correlation 0.3970.04). As                        demonstrated comparing gene-pair correlations be- <br /> example, Figure 3C shows histidine biosynthesis pathway                        tween GAL1 genes experiments, with <br /> and Figure 3D purine pathway. Note highly ordered                          3AT (Figure 6). Note low- <br /> trajectories lower panels (with 3AT) compared                        correlated gene pairs 3ATenvironment exhibited high <br /> disordered ones upper panels (no 3AT). These pathways                       correlation 3AT present. <br /> are directly connected gene pressure, HIS3,                       Taken together, highly correlated response observed <br /> mentioned above, modules unique exhibit-                         reveals coexpression does necessarily imply co- <br /> ing high correlation environmental pressure                          functionality (Niehrs Pollet, 1999) exist among <br /> was high (see Figure 3A). The increase expression                               genes residing different functional modules. Indeed, the <br /> correlation sizeable fraction genome higher                        highly correlated pattern expression spans entire <br /> pressure HIS3 indicates global transcriptional                           metabolic network modules correlations not <br /> regulatory mechanism operation, local                          decay genes residing remote modules (Kharchenko et al, <br />                                                                                    2005) (Supplementary Figure S8). These results indicate that <br />                                                                                    crosstalk functional modules plays important role <br />     A                                                                              enabling emergence proper metabolic state. The <br />                        2 <br />                                                                                    opposite true&#8212;the symmetric patterns induced and <br />                                                                         KIN28 <br />                                                                         TFB1 <br />                                                                                    repressed transcriptional responses observed modules <br />     log2 expression <br />  <br />  <br />  <br />  <br />                        1                                                TFB3       means co-functionality does necessarily imply <br />                                                                         RAD23 <br />                                                                         RAD4 <br />                                                                                    coexpression simple connection between <br />                        0                                                RAD2       transcription metabolism. <br />                                                                         SSL2 <br />                                                                         RAD7 <br />                                                                         RAD10 <br />                       &#8211;1 <br />                                                                         RAD14                             1.0 <br />                                                                         TFB2 <br />                                                                         CCL1 <br />                       &#8211;2 <br />                                                                                                           0.5 <br />                                                                                       Corr (40 mM 3AT) <br />  <br />  <br />  <br />  <br />                        &#8211;50       0     50       100       150    200 <br />  <br />     B <br />                        1                                                                                  0.0 <br />                                                                        KIN28 <br />                                                                        TFB1 <br />     log2 expression <br />  <br />  <br />  <br />  <br />                                                                        TFB3 <br />                        0                                               RAD23                             &#8211;0.5 <br />                                                                        RAD4 <br />                                                                        RAD2 <br />                                                                        SSL2 <br />                       &#8211;1                                               RAD7                              &#8211;1.0 <br />                                                                        RAD10                                &#8211;1.0   &#8211;0.5        0.0           0.5            1.0 <br />                                                                        RAD14 <br />                                                                        TFB2                                               Corr (no 3AT ) <br />                       &#8211;2                                               CCL1 <br />                                                                                    Figure 6 Correlation coefficient GAL1 genes. The <br />                                                                                    Pearson correlation coefficient computed transcriptional <br />                       &#8211;100   0       100     200    300    400    500              dynamic profiles GAL1 genes experiments, and <br />                                            Time (h)                                3AT. The correlation coefficient value pair 3AT <br />                                                                                    experiment plotted correlation coefficient value no-3AT <br /> Figure 5 Expression profiles protein complex. Expression profiles      experiment. Only pairs absolute Pearson correlation 0.8 least <br /> genes belonging NER protein complex defined MIPS (http://             experiments plotted clarity. There 339 genes 3AT <br /> mips.gsf.de) superimposed population density curves (blue lines), (A)   experiment 109 genes no-3AT experiment absolute correlations <br /> no-3AT (B) 40 mM 3AT experiments. Note significant increase order       0.9. Note genes highly correlated experiment exhibit low <br /> among transcriptional response genes 3AT experiment              negative correlation experiment. The GAL1 gene served <br /> compared no-3AT experiment. Note genes belonging           example here, similar results obtained genes <br /> same protein complex exhibited anti-correlated expression patterns.                belonging different functional modules. <br />  <br /> 6 Molecular Systems Biology 2007                                                                                           &amp; 2007 EMBO Nature Publishing Group <br />                                                                                                                 Transcriptional plasticity underlies adaptation <br />                                                                                                                                                   S Stern et al <br />  <br />  <br />  <br />  <br />                                                                                    order 10 generations. This response cells the <br />                    3 <br />                                                                                    chemostat genuine transcriptional reprogramming process <br />                                                                                    population selection mechanism. Increasing <br />                    2                                                               pressure applied rewired gene HIS3 resulted a <br />                                                                                    significant increase correlations hundreds genes <br />    log2 (glu_ss) <br />  <br />  <br />  <br />  <br />                    1                                                               residing different modules. This sensitivity level of <br />                                                                                    pressure imposed novel challenge shows transcrip- <br />                    0                                                               tional response directly involved cellular adaptation <br />                                                                                    process. The highly correlated global transcriptional response <br />                    &#8211;1                                                              consistent massive crosstalk known exist <br />                                                                                    functional modules (Ideker et al, 2001; Tong et al, <br />                    &#8211;2                                                              2001) dynamic properties regulatory network <br />                                                                                    (Luscombe et al, 2004). The characteristics transcrip- <br />                    &#8211;3                                                              tional reprogramming observed support conclusion <br />                                                                                    universal feature transcriptional regulatory <br />                         &#8211;3   &#8211;2     &#8211;1      0        1        2        3 <br />                                                                                    reflecting plasticity transgeneration stability. <br />                                   log2 (glu_transient)                                The novelty challenge presented cells excludes <br /> Figure 7 Steady-state transcriptional pattern predicted           possibility transcriptional response been <br /> transient response. The steady-state expression level (average          specifically selected evolution. Indeed, analyzing details <br /> measurement points glucose steady state) versus transient expression        transcriptional response revealed that, large <br /> level (4th time point Figure 1B) experiment 40 mM 3AT. The <br /> Pearson correlation transient response steady state 0.76. The <br />                                                                                    fraction responding genes, simple biological <br /> slope deviation reference dashed black line represents overshoot   interpretation, connecting specific cellular <br /> undershoot transient expression levels compared steady state.            demands imposed novel challenge (e.g., amino-acid <br />                                                                                    production). In particular, surprisingly genes <br />                                                                                    belonging functional module, active <br />    The long timescale transcriptional response raises                   linear chain biosynthesis pathway, respond <br /> question result population level selection                  opposite directions; highly induced the <br /> process (Ferea et al, 1999) cellular process faithfully               highly repressed. Thus, important conclusion of <br /> transmitted generations (Molinier et al, 2006). In                         work global transcriptional response an <br /> selection process, initial state average mixture                  unforeseen challenge explained simple cellular or <br /> of large number subpopulations,                             metabolic logic. This expected response not <br /> eventually determine final state. Therefore,                     specifically selected evolution pre- <br /> one expects significant correlation initial                     designed challenge. Support conclusion found <br /> final states level population averages. Figure 7                     fact repeating experiments reproduce similar <br /> shows final steady state experiment highly                      transcription patterns transient phase the <br /> correlated initial transient transcription level follow-                  adapted state glucose. <br /> ing medium switch galactose- glucose-containing                           Nevertheless, data raise following apparent paradox <br /> medium. The final steady state predicted initial                    conclusion: adaptation novel perturbation <br /> transient response. A gene exhibiting higher (lower) steady-                       pre-designed, &#8216;correct&#8217; direction of <br /> state transcription level glucose relative galactose                 transcriptional response (the &#8216;correct&#8217; set induced and <br /> exhibited overshoot (undershoot) relaxed                        repressed genes) instantaneously generated (Figure 7)? A <br /> proper steady state population adaptation timescale                         plausible explanation direction transcriptional <br /> (Figure 1B). Thus, transcriptional reprogramming                           response itself adaptive, reflects large- <br /> initiated instantaneously medium switch                              scale interactions functional modules regulatory <br /> galactose glucose, emergent pattern transcription                    network. This possibility supported symmetry <br /> was faithfully transmitted generations till conver-                      dynamics induced repressed genes. The <br /> gence steady state. We conclude observed                               mechanism underlying broad nonspecific transcriptional <br /> expression dynamics result selection process                      response determined. One possibility small <br /> on population, plastic transcriptional response                   number master regulators responsible response. <br /> of cellular regulatory network.                                                Alternatively, absence specific transcriptional <br />                                                                                    regulatory response shaped evolution specific <br />                                                                                    demand, parallel regulatory pathways freely <br />                                                                                    respond initially having constraining feedback. <br /> Discussion                                                                         Global mechanisms, chromatin remodeling, be <br /> We presented yeast cells severe perturbation                       involved process. Once global transcriptional <br /> not encountered evolutionary history                      response emerges, adaptive metabolic response be <br /> they adapted physiological timescales. We                        generated provided underlying expression pattern broad <br /> that underlying adaptation global transcriptional                        support it. Metabolic requirements feeding back <br /> response involving sizeable fraction genome,                        transcription regulation cause relaxation tran- <br /> relaxed timescale population adaptation,                      scriptional response magnitude, affect its <br />  <br /> &amp; 2007 EMBO Nature Publishing Group                                                                                    Molecular Systems Biology 2007 7 <br />  Transcriptional plasticity underlies adaptation <br /> S Stern et al <br />  <br />  <br />  <br />  <br /> directionality. The transcriptional reprogramming observed                 Expression arrays <br /> here reflects complex interplay transcrip-                For sample, 15 mg total RNA isolated cells using hot <br /> tional regulation metabolism. This <span id='am-25' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-26' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29'  ></span>idea</span>, correct,                   phenol extraction. mRNA reverse transcribed (superscript II, <br /> explains ability cells overcome broad range                 Invitrogen) labeled indirectly cy5/3 dyes (Amersham) using <br /> novel challenges stress environments. If                   amino-allyl dUTP (Ambion). For time point, cDNA <br />                                                                            microarrays (yeast 6.4 k, UHN microarray center, www.microarrays.- <br /> specific local solution perturbation,                    ca) containing B6400 yeast ORFs duplicate (a total spots <br /> specifically shaped evolution perturbation,             ORF) hybridized overnight (421C) sample <br /> cells alternative general mechanism. A mismatch                    labeled cy5 reference sample labeled cy3. Arrays were <br /> between metabolic requirements cellular state                  scanned using commercial scanner software (GenePix 4000B, <br />                                                                            Axon instruments). For microarray, cy5/cy3 intensity ratios were <br /> works driving force causing nonspecific large-scale                 normalized using Acuty software (Axon instruments), ratio <br /> transcriptional response. The drastic change expression             medians 1. log2(cy5/cy3) values spots gene were <br /> state cell opens multiple new metabolic pathways.                   averaged time point genes high- <br /> Physiological selection works multiple <span id='am-64' about='obo:NCBITaxon_94025' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_27394 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_74974 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_74975 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-65' property="rdfs:label" content="Meta" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-68'  ></span>meta</span>-                 quality spots time point dynamic path the <br />                                                                            experiments subject analysis. Duplicate arrays were <br /> bolic pathways stabilize adaptive state causes <br />                                                                            checked yield high correlated signal gene (Supplementary <br /> relaxation perturbed expression pattern. This scenario,             Figure S5e) data array compared real-time <br /> involving creation library possibilities                   PCR measurements various genes sample. There a <br /> physiological selection library, compatible              total 4148 genes passed filters experiments. For all <br /> our understanding broad <span id='am-30' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-31' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> biological systems,                  analyses log2(cy5/cy3) values time point normalized <br />                                                                            galactose steady-state time point. <br /> placing cellular metabolic/regulatory networks the <br /> same footing neural immune systems (Gerhart <br /> and Kirschner, 1997). <br />                                                                            Clustering analysis <br />    The way organisms deal unforeseen changes of <br /> the intriguing questions biology. Recruitment gene            All genes two-fold change time point clustered <br />                                                                            using EXPANDER software (Shamir et al, 2005). The self-organized <br /> foreign regulatory considered powerful driving              maps (SOM) clustering method (Tamayo et al, 1999; Shamir et al, <br /> force evolution, responsible example, evolution          2005) applied gene profiles, 16 clusters pre-defined <br /> developmental systems (Carroll et al, 2001; Davidson, 2001;                parameter (Supplementary Figure S1). Clusters same <br /> True Carroll, 2002; Wilkins, 2002; Carroll, 2005). Such                fundamental mean expression profile joined large <br />                                                                            clusters presented Figure 1. These results sensitive the <br /> genomic perturbation presents severe challenges cell;               SOM clustering method similar results obtained with <br /> the evolvability potential organisms (Gerhart Kirschner,            different methods (data shown). Enrichment biological process <br /> 1997; Kirschner Gerhart, 1998) depends ability             computed clusters genes using GO TermFinder (SGD). <br /> cope them. The global transcriptional plasticity described            All P-values computed using hypergeometric distribution. <br /> here shows efficient adaptive response emerge <br /> on physiological timescales, prerequisite efficient <br /> evolutionary process.                                                      Comparing coherency KEGG groups <br />                                                                            experiments <br />                                                                            We computed P-value mean difference two <br /> Materials methods                                                      distributions absolute pairwise correlation specific KEGG <br />                                                                            (Kanehisa et al, 2004) group, 3AT 40 mM 3AT <br /> Strain chemostat growth conditions                                     experiment, using Wilcoxon&#8217;s test. All P-values corrected for <br /> Experiments carried haploid yeast strain YPH499          multiple hypothesis testing using false discovery rate. <br /> [Mata, ura3-52, lys2-801, ade2-101, trp1-D63, his3D200, leu2D1] <br /> carrying plasmid vector pESC-LEU (Stratagene) containing the <br /> pGAL1-pGAL10 divergent promoter HIS3 pGAL1 (Stolo-              Supplementary information <br /> vicki et al, 2006). his3D200 deletion removed entire HIS3- <br /> coding region plus upstream promoter region, including Gcn4        Supplementary information available Molecular Systems <br /> regulatory sequence. Cells grown homemade chemostat              Biology website (www.nature.com/msb). <br /> (Stolovicki et al, 2006) synthetic dropout medium lacking histidine <br /> and leucine appropriate amino-acid supplement 2% of <br /> either pure galactose pure glucose sole carbon source. <br /> Throughout experiments, sugar (either galactose glucose)        Acknowledgements <br /> was excess (maximal consumption cells 25%       We thank L David, E Ruppin, R Sharan, M Kupiec, K Keren Y Soen <br /> sugar fed). The medium following (concentrations g/l): 1.7      useful remarks manuscript Y Arava assistance with <br /> yeast nitrogen base amino acids ammonium sulfate, 5            array experiments. This research supported Israeli <br /> ammonium sulfate, 1.4 amino-acid dropout powder (without trypto-           Science Foundation (ISF EB), German-Israeli Fund (GIF) the <br /> phan, histidine, leucine uracil; Sigma), 0.01 L-tryptophan, 0.005      Yeshaya Horowitz Association Center Complexity <br /> uracil. Growth chemostat limited concentration    Science. S Stern acknowledges support Sherman scholar- <br /> amino-acid supplement. In 3AT experiment, 40 mM                 ship. <br /> competitive inhibitor 3-amino-1,2,4-triazole (Sigma), sterilized by <br /> filtration, introduced feeding medium. An online <br /> measurement (Stolovicki et al, 2006) used measure <br /> the OD cells chemostat producing blue curves Figure 1. <br /> A homemade cell collector (Stolovicki et al, 2006) used             References <br /> automatically collect samples cells chemostat precise <br /> time points experiment instantaneously freeze them.          Braun E, Brenner N (2004) Transient responses adaptation to <br /> These samples used microarray experiments.                       steady state eukaryotic gene regulation system. Phys Biol 1: 67 <br />  <br /> 8 Molecular Systems Biology 2007                                                                             &amp; 2007 EMBO Nature Publishing Group <br />                                                                                                          Transcriptional plasticity underlies adaptation <br />                                                                                                                                            S Stern et al <br />  <br />  <br />  <br />  <br /> Carlson M (1999) Glucose repression yeast. Curr Opin Microbiol 2:       Koerkamp MG, Rep M, Bussemaker HJ, Hardy GPMA, Mul A, <br />     202                                                                       Piekarska L, Szigyarto CAK, Mattos JMTd, Tabak HF (2002) <br /> Carroll SB (2005) Evolution levels: genes form. PLoS Biol       Dissection transient oxidative stress Saccharomyces <br />     3: 1159&#8211;1166                                                              cerevisiae using DNA microarrays. Mol Biol Cell 13: 2783&#8211;2794 <br /> Carroll SB, Grenier JK, Weatherbee SD (2001) From DNA Diversity:        Lai LC, Kosorukoff AL, Burke PV, Kwast KE (2005) Dynamical <br />     Molecular Genetics Evolution Animal Design. Malden,            remodeling transcriptome short-term anaerobiosis <br />     MA: Blackwell Science Inc                                                 <span id='am-1' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-16'  ></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>: different response role Msn2 <br /> Causton HC, Ren B, Koh SS, Harbison CT, Kanin E, Jennings EG, Lee TI,         and/or Msn4 factors galactose glucose media. Mol <br />     True HL, Lander ES, Young RA (2001) Remodeling yeast genome            Cell Biol 25: 4075&#8211;4091 <br />     expression response environmental changes. Mol Biol Cell 12:     Luscombe NM, Babu MM, Yu H, Snyder M, Telchmann SA, Gerstein M <br />     323&#8211;337                                                                   (2004) Genomic analysis regulatory network dynamics reveals <br /> Chu S, DeRisi J, Eisen M, Mulholland J, Botstein D, Brown PO,                 large topological changes. Nature 431: 308&#8211;312 <br />     Herskowitz I (1998) The transcriptional program sporulation      Marton MJ, DeRisi JL, Bennet HA, Iyer VR, Meyer MR, Roberts CJ, <br />     budding yeast. Science 282: 699&#8211;705                                       stoughton R, Burchard J, Slade D, Dai Jr H, DEB, Hartwell LH, <br /> Davidson EH (2001) Genomic Regulatory Systems, Development                Brown PO, Friend SH (1998) Drug target validation and <br />     Evolution. London: Academic Press                                         identification secondary drug target effects using DNA <br /> Ferea TL, Botstein D, Brown PO, Rosenzweig RF (1999) Systematic               microarrays. Nature Med 4: 1293&#8211;1301 <br />     changes gene expression patterns following adaptive evolution       Molinier J, Ries G, Zipfel C, Hohn B (2006) Transgeneration memory of <br />     yeast. Proc Nat Acad Sci USA 96: 9721                                  stress plants. Nature 442: 1046&#8211;1049 <br /> Fong SS, Joyce AR, Palsson BO (2005) Parallel adaptive evolution           Natarajan K, Meyer MR, Jackson BM, Slade D, Roberts C, Hinnebusch <br />     cultures <span id='am-41' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-42' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-46'  ></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span>Escherichia coli</span> lead convergent growth phenotypes         AG, Marton MJ (2001) Transcriptional profiling shows Gcn4p is <br />     different gene expression states. Genome Res 15: 1365&#8211;1372           master regulator gene expression amino acid starvation <br /> Gasch AP, Spellman PT, Kao CM, Carmel-Harel O, Eisen MB, Botstein             yeast. Mol Cell Biol 21: 4347&#8211;4368 <br />     D, Brown PO (2000) Genomic expression programs response         Nautiyal S, DeRisi JL, Blackburn EH (2002) The genome-wide <br />     yeast cells environmental changes. Mol Biol Cell 11: 4241&#8211;4257      expression response telomerase deletion Saccharomyces <br /> Gerhart J, Kirschner M (1997) Cells, Embryos, Evolution. Malden,          cerevisiae. Proc Nat Acad Sci USA 99: 9316&#8211;9321 <br />     MA: Blackwell Science Inc                                              Niehrs C, Pollet B (1999) Synexpression groups eukaryotes. Nature <br /> Hinnebusch AG (1992) General pathway-specific regulatory                  402: 483&#8211;487 <br />     mechanisms controlling synthesis amino acid biosynthesis        Segal E, Shapira M, Regev A, Pe&#8217;er D, Botstein D, Koller D, Friedman N <br />     enzymes Saccharomyces cerevisiae. In The Molecular                 (2003) Module networks: identifying regulatory modules their <br />     Cellular Biology Yeast Saccharomyces Gene Expression,              condition-specific regulators gene expression data. Nat Genet <br />     Jones EW, Pringle JR, Broach JR (eds). Cold Spring Harbor,                34: 166&#8211;176 <br />     Woodbury, NY: Cold Spring Harbor Laboratory Press                      SGD www.yeastgenome.org <br /> Hughes TR, Marton MJ, Jones AR, Roberts CJ, Stoughton R, Armour            Shamir R, Maron-Katz A, ATanay Linhart C, Steinfeld I, Sharan R, <br />     CD, Bennet HA, Coffey E, Dai H, He YD, Kidd MJ, King AM, Meyer            Shiloh Y, Elkon R (2005) EXPANDER&#8212;an integrative program suite <br />     MR, Slade D, Lum PY, Stepaniants SB, Shoemaker DD, Gachotte D,            microarray data analysis. BMC Bioinform 6: 232 <br />     Chakraburtty K, Simon J, Bard M, Friend SH (2000) Functional           Spellman PT, Sherlock G, Zhang MQ, Iyer VR, Anders K, Eisen MB, <br />     discovery compendium expression profiles. Cell 102:              Brown PO, Botstein D, Futcher B (1998) Comprehensive <br />     109&#8211;126                                                                   identification cell cycle-regulated genes yeast <br /> Ideker R, Thorsson V, Ranish JA, Christmas R, Buhler J, Eng JK,               Saccharomyces cerevisiae microarray hybridization. Mol Biol <br />     Bumgarner R, Goodlett DR, Aebersold R, Hood L (2001) Integrated           Cell 9: 3273&#8211;3297 <br />     genomic proteomic analyses systematically perturbed           Stolovicki E, Dror T, Brenner N, Braun E (2006) Synthetic gene <br />     metabo <br /> </body></html>