<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>EUKARYOTIC CELL, July 2007, p. 1239&#8211;1247                                                                                            Vol. 6, No. 7 <br /> 1535-9778/07/$08.00&#11001;0 doi:10.1128/EC.00414-06 <br /> Copyright &#169; 2007, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />  Development Highly Efficient Gene Targeting System Induced by <br />    Transient Repression YKU80 Expression <span id='am-118' about='obo:NCBITaxon_5475' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_118267 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-119'  ></span><span id='am-120' property="oboInOwl:hasRelatedSynonym" content="Torulopsis" datatype="xsd:string"></span><span id='am-121' property="rdfs:label" content="Candida" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Candida</span> glabrata&#17188;&#8224; <br />                         Keigo Ueno,1 Jun Uno,1 Hironobu Nakayama,2 Kaname Sasamoto,1 <br />                                      Yuzuru Mikami,1 Hiroji Chibana1* <br />     Research Center Pathogenic <span id='am-47' about='obo:NCBITaxon_4751' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2759'><span id='am-48' property="oboInOwl:hasDbXref" content="PMID:11062127" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasExactSynonym" content="fungi" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="fungi" datatype="xsd:string"></span><span id='am-52'  ></span><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Fungi" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="PMID:12684019" datatype="xsd:string"></span>Fungi</span> Microbial Toxicoses, Chiba University, 1-8-1 Inohana, Chuo-ku, Chiba 260-8673, <br />                   Japan,1 Suzuka National College Technology, Shiroko, Suzuka, Mie 510-0294, Japan2 <br />                                               Received 30 December 2006/Accepted 4 May 2007 <br />  <br />              In pathogenic yeast <span id='am-132' about='obo:NCBITaxon_5478' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_374468 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_600669 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-133' property="oboInOwl:hasRelatedSynonym" content="Torulopsis glabrata" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-135'  ></span><span id='am-136' property="oboInOwl:hasDbXref" content="PMID:14654427" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-138' property="rdfs:label" content="Candida glabrata" datatype="xsd:string"></span>Candida glabrata</span>, gene targeting generate knockouts &#8220;knockins&#65533;? poten- <br />           tially powerful method analysis gene function. Its importance increased C. glabrata genome <br />           sequence project, progress field hampered inefficient mechanisms gene targeting. With the <br />           use 40-bp homologous flanking DNA, gene targeting identified. To address issue, YKU80 was <br />           disrupted, leading increase targeting efficiency 5.1% using 40-bp flanking homologous DNA. To <br />           harness beneficial effects YKU80 inactivation gene targeting frequency incurring negative <br />           effects, synthetic sickness lethality, developed new expression YKU80 was <br />           restored following transient knockdown expression transformation. Strains used new <br />           carried SAT1 flipper YKU80 promoter region, used repress expression during <br />           transformation spontaneously excised locus transformation. By using strain, <br />           DNA damage induced methyl methane sulfonate, H2O2, UV irradiation, hydroxyurea during <br />           gene targeting evaluated mutation rate URA3 determined. No significant effects SAT1 <br />           flipper processes identified. After SAT1 flipper excised, 34-bp FLP recombination <br />           target sequence left promoter region. However, levels mRNA transcription restored and <br />           difference survival ratio vivo compared YKU80 wild-type strain identified. <br />  <br />  <br />    The incidence opportunistic fungal infection                    haploid fungus, C. albicans diploid, making gene <br /> increasing, particularly patients immunocompro-                     targeting C. albicans twice difficult C. <br /> mised <span id='am-1' about='obo:NCBITaxon_12721' typeof='owl:Thing obo:NCBITaxon_11632 obo:NCBITaxon_11652 obo:NCBITaxon_1 obo:NCBITaxon_35268 obo:NCBITaxon_10239 obo:NCBITaxon_693660 obo:NCBITaxon_327045 obo:NCBITaxon_11646'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="AIDS virus" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasBroadSynonym" content="HIV" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="Human immunodeficiency virus" datatype="xsd:string"></span>human immunodeficiency virus</span> infection leukemia                    glabrata (4). The genome sequence C. glabrata has <br /> or receiving immunosuppressive therapy organ                       completed, allowing comprehensive identifica- <br /> transplantation chemotherapy <span id='am-139' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-140' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-141' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-143'  ></span>cancer</span>. Systemic fungal                    tion genes (10). Taking attributes together, C. <br /> infections patients life threatening. Candida                glabrata come age feasible model organism for <br /> albicans Candida glabrata opportunistic                      genome-wide study pathogenicity fungi the <br /> pathogenic fungi cause candidiasis (7). C. albicans remains               identification antifungal drug targets. <br /> the commonly encountered <span id='am-159' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-160' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-161' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> clinical practice                    Gene targeting important approaches for <br /> other species (non-C. albicans species), C. glabrata,                  analysis gene function. Genome-wide functional analyses <br /> now cause significant levels disease (35). In study, C.                 facilitated efficient gene targeting using constructs <br /> glabrata second commonly documented species,                      flanked end short homologous sequences, the <br /> isolated 20% candidiasis patients (31). It                lengths typically 40 60 bp. These short homol- <br /> known C. glabrata low susceptibility fluconazole,                ogous sequences easily produced single-step PCR <br /> which drug administered clinical situations (12);              using pair oligonucleotide DNA primers containing ho- <br /> thus, candidiasis caused C. glabrata associated high <br />                                                                                mologous flanking sequences. However, targeted integration <br /> mortality (17). Despite increase incidence, little is <br />                                                                                short homologous sequences relatively inefficient C. <br /> known molecular basis C. glabrata virulence in <br />                                                                                glabrata, characteristic shares higher eukaryotes and <br /> contrast C. albicans. <br />                                                                                fungi S. cerevisiae (8, 22, 28, 40). S. cerevisiae has <br />    C. glabrata offers advantages study fungal <br />                                                                                highly efficient gene targeting, exogenous DNA fragments <br /> pathogenesis. It closely related <span id='am-40' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-41' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-42'  ></span><span id='am-43' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span>Saccharomyces</span> cer- <br />                                                                                correctly integrated targeted locus efficiency <br /> evisiae C. albicans (2, 12), suggesting the <br />                                                                                close 100% requiring short target sequences 40 <br /> wealth information accumulated decades study S. <br /> cerevisiae transferable C. glabrata. C. glabrata              bp length (3, 40). As consequence, complete set gene <br />                                                                                knockout strains S. cerevisiae successfully con- <br />                                                                                structed (13). <br />    * Corresponding author. Mailing address: Research Center                   The targeted integration DNA fragment requires ho- <br /> Pathogenic Fungi Microbial Toxicoses, Chiba University, 1-8-1              mologous recombination, repair mecha- <br /> Inohana, Chuo-ku, Chiba 260-8673, Japan. Phone: 81-43-226-2792.                nisms DNA double-strand breaks (DSB). Nonhomologous <br /> Fax: 81-43-226-2486. E-mail: chibana@faculty.chiba-u.jp. <br />                                                                                end joining (NHEJ) mechanism DSB repair, but <br />    &#8224; Supplemental material article http://ec <br /> .asm.org/.                                                                     does require homologous DNA sequences (9, 30). <br />    &#17188; <br />      Published ahead print 18 May 2007.                                  We observed C. glabrata, NHEJ predominate <br />  <br />                                                                         1239 <br />  1240         UENO ET AL                                                                                                                           EUKARYOT. CELL <br />  <br />  <br />                                                              TABLE 1. Strains used study <br />     Strain                       Parent                                                 Genotype                                                 Source reference <br />  <br /> 2001H                        2001U                       &#9004;his3::ScURA3 &#9004;ura3                                                                          21 <br /> 2001T                        2001U                       &#9004;trp1::ScURA3 &#9004;ura3                                                                          21 <br /> 2001HT                       2001TU                      &#9004;his3::ScURA3 &#9004;trp1 &#9004;ura3                                                                    21 <br /> ACG4                         2001HT                      his3 trp1 PScADH1::tetR::GAL4AD::TRP1                                                        28 <br /> tADE2-38                     ACG4                        his3 trp1 PScADH1::tetR::GAL4AD::TRP1 97t::ADE2-CgHIS3                                       <span id='am-144' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-145'  ></span><span id='am-146' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-147' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-148' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span>   study <br /> HET80                        ACG4                        yku80 ILV2(P182L)                                                                            This   study <br /> ACG4-SF                      ACG4                        his3 trp1 PScADH1::tetR::GAL4AD::TRP1 SAT1 flipper                                           This   study <br /> HETS146                      ACG4                        his3 trp1 PScADH1::tetR::GAL4AD::TRP1 YKU80::SAT1 flipper                                    This   study <br /> HETS202                      HET80                       his3 trp1 PScADH1::tetR::GAL4AD::TRP1 yku80::SAT1 flipper                                    This   study <br /> ACG4-SFP                     ACG4-SF                     his3 trp1 PScADH1::tetR::GAL4AD::TRP1 FRT                                                    This   study <br /> POP-SF1467                   HET-SF146                   his3 trp1 PScADH1::tetR::GAL4AD::TRP1 YKU80::FRT                                             This   study <br /> POP-SF221                    HET-SF22                    His3 trp1 PScADH1::tetR::GAL4AD::TRP1 YKU80::FRT                                             This   study <br /> KUE100                       2001H                       his3 yku80::SAT1 flipper                                                                     This   study <br /> KUE200                       2001HT                      his3 trp1 yku80::SAT1 flipper                                                                This   study <br /> KPOP11538                    KUE100                      his3 yku80::FRT cyb2::CgHIS3                                                                 This   study <br />  <br />  <br />  <br /> at integration events involving DNA fragments flanking                         yeast nitrogen base amino acids, 2% [wt/vol] dextrose) media. Various <br /> short homologous sequences. Therefore, reasoned                          growth supplements added SD medium described standard <br />                                                                                     experimental manual (6). For screening sulfometuron methyl (SM) resis- <br /> the activity NHEJ suppressed, frequency                        tance performed HET80, cells plated SD plates plus 20 <br /> of homologous recombination increase. Several genes,                         &#65533;?&#65533;g/ml histidine 20 &#65533;?&#65533;g/ml SM (Wako, Inc.; dissolved dimethyl sulfoxide to <br /> such KU70, KU80, LIG4 homologues,                             prepare 2 mg/ml stock solution). For selection nourseothricin (NSTC)- <br /> been identified key genes NHEJ eukaryotes, including                      resistant strains, cells plated YPD plates containing 100 &#65533;?&#65533;g/ml NSTC <br />                                                                                     (HKI Jena, Germany; 10 mg/ml water stock solution). In <span id='am-162' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-163' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-164' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> test for <br /> mammals, plants, fungi. Recently, mutants con- <br />                                                                                     recovery live C. glabrata infected mice, homogenized kidney was <br /> structed genes homologous human KU70                             spread agar plates consisting 1.5% (wt/vol) agar, 0.5% (wt/vol) yeast <br /> KU80 disrupted <span id='am-80' about='obo:NCBITaxon_5140' typeof='obo:NCBITaxon_222544 owl:Thing obo:NCBITaxon_5139 obo:NCBITaxon_1 obo:NCBITaxon_5148 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_147550 obo:NCBITaxon_715989 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_2759'><span id='am-81' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="Neurospora" datatype="xsd:string"></span><span id='am-84'  ></span>Neurospora</span> <span id='am-8' about='obo:NCBITaxon_688987' typeof='owl:Thing obo:NCBITaxon_37581 obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_116123 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_57992 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-9' property="rdfs:label" content="Crassa" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-11'  ></span><span id='am-12' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>crassa</span> (28); <span id='am-18' about='obo:NCBITaxon_4910' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-19'  ></span><span id='am-20' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Kluyveromyces" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Kluyveromyces</span>                        extract, 1% (wt/vol) polypeptone, 1% (wt/vol) dextrose, 1,000 units/ml penicillin <br /> lactis (22); <span id='am-105' about='obo:NCBITaxon_5052' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_5042 obo:NCBITaxon_33154 obo:NCBITaxon_147545 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_1131492 obo:NCBITaxon_451871 obo:NCBITaxon_2759'><span id='am-106'  ></span><span id='am-107' property="rdfs:label" content="Aspergillus" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasAlternativeId" content="NCBITaxon:37503" datatype="xsd:string"></span>Aspergillus</span> fumigatus (8); koji molds, including As-                   G, 100 &#65533;?&#65533;g/ml streptomycin sulfate (Wako Chemicals, Japan). For re- <br /> pergillus niger (25), <span id='am-188' about='obo:NCBITaxon_41058' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_5042 obo:NCBITaxon_5052 obo:NCBITaxon_33154 obo:NCBITaxon_147545 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_1131492 obo:NCBITaxon_451871 obo:NCBITaxon_2759'><span id='am-189' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-190' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-191'  ></span><span id='am-192' property="rdfs:label" content="Aspergillus sojae" datatype="xsd:string"></span>Aspergillus sojae</span>, <span id='am-111' about='obo:NCBITaxon_5062' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_5042 obo:NCBITaxon_5052 obo:NCBITaxon_33154 obo:NCBITaxon_147545 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_1131492 obo:NCBITaxon_451871 obo:NCBITaxon_2759'><span id='am-112' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-113' property="rdfs:label" content="Aspergillus oryzae" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasRelatedSynonym" content="Eurotium oryzae" datatype="xsd:string"></span><span id='am-116'  ></span><span id='am-117' property="oboInOwl:hasRelatedSynonym" content="Aspergillus oryza" datatype="xsd:string"></span>Aspergillus oryzae</span> (38);               pression genes interest, inactivation tetracycline regulatable pro- <br />                                                                                     moter (Tet-P) carried appropriate medium 20 &#65533;?&#65533;g/ml doxycycline <br /> <span id='am-91' about='obo:NCBITaxon_5207' typeof='owl:Thing obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_5204 obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_5215 obo:NCBITaxon_5206 obo:NCBITaxon_5234 obo:NCBITaxon_2759 obo:NCBITaxon_552466 obo:NCBITaxon_155616'><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces neoformans" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasRelatedSynonym" content="Torula neoformans" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasRelatedSynonym" content="Filobasidiella neoformans" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasRelatedSynonym" content="Blastomyces neoformans" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasRelatedSynonym" content="Debaryomyces neoformans" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasRelatedSynonym" content="Lipomyces neoformans" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasRelatedSynonym" content="Filobaxidiella neoformans" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasRelatedSynonym" content="Filobasidiella neoformornans" datatype="xsd:string"></span><span id='am-102'  ></span><span id='am-103' property="rdfs:label" content="Cryptococcus neoformans" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasRelatedSynonym" content="Torulopsis neoformans" datatype="xsd:string"></span>Cryptococcus neoformans</span> (16); <span id='am-70' about='obo:NCBITaxon_5147' typeof='obo:NCBITaxon_222544 owl:Thing obo:NCBITaxon_5139 obo:NCBITaxon_1 obo:NCBITaxon_5146 obo:NCBITaxon_5148 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_147550 obo:NCBITaxon_715989 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_2759'><span id='am-71' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-72'  ></span><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="Sordaria macrospora" datatype="xsd:string"></span>Sordaria macrospora</span> (32). <br />                                                                                     ([ICN Biomedicals Inc.] dissolved 50% ethyl alcohol prepare 2 mg/ml stock <br /> Drastically improved efficiency gene targeting                      solution). <br /> these Ku mutants. It shown NHEJ                            <span id='am-165' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-166' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-167' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-168' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-169' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-170' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-171' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-172' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-173' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-174' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-175'  ></span><span id='am-176' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-177' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-179' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-180' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span>Escherichia coli</span> DH5&#65533;?&#65533; used DNA manipulation. General recombinant <br /> suppressed, led sensitization DNA-damaging                            DNA procedures performed described previously (15, 36). <br /> agents, including methyl methane sulfonate (MMS) (5, 16, 37).                          Primers. For list primers used study, Table S1 the <br />                                                                                     supplemental material. <br /> These findings enabled important advances meeting                              Plasmid construction. The <span id='am-201' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-202' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-203' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-204' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-205' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>plasmids</span> used study listed Table 2. All <br /> the objective high-throughput construction targeted re-                       plasmids based pBluescriptII SK(&#11001;) (Stratagene). <br /> combinant fungal strains. After construction desired                            pKUILV2-1. A 6-kb DNA fragment, including YKU80-ILV2 regions, am- <br /> recombinant strain, NHEJ genes usually returned                          plified PCR using primer pair preKu80HindIII preILV2HindIII and <br />                                                                                     genomic DNA tADE2-38 template. After PCR, product was <br /> the cell mating. Although sexual cycle been <br />                                                                                     digested HindIII inserted pBluescriptII SK(&#11001;) cleaved HindIII <br /> identified C. glabrata, developed new                      yield plasmid pKUILV2-1. <br /> that delivers efficient gene targeting restores expres-                        pILV2-7. PCR carried pKUILV2-1 primer pair of <br /> sion YKU80 single transformation.                                         preKuR2 preILV2F2. The primers anneal sites flanking open reading <br />                                                                                     frame (ORF) YKU80 outside ORF pKUILV2-1. The PCR <br />                                                                                     product circularized self-ligation, ORF YKU80 removed <br />                        MATERIALS AND METHODS <br />                                                                                     plasmid produce pILV2-7. <br />   Strains media. All strains used study described Table 1.          pBS-SF.4. PCR carried primer pair pSFS2AF6(Bgl-Kpn) <br />   C. glabrata strains grown routinely 37&#176;C YPD (1% [wt/vol] yeast        pSFS2AR5(Nhe-Sac) plasmid pSFS2A (34) template DNA, includ- <br /> extract, 2% [wt/vol] Bacto peptone, 2% [wt/vol] dextrose) SD (0.67% [wt/vol]    ing SAT1 flipper. The PCR product digested KpnI SacI and <br />  <br />  <br />  <br />                                                             TABLE 2. Plasmids used study <br />         Plasmid                                                               Description                                                        Source reference <br />  <br /> pBluescriptII SK(&#11001;)                                 Cloning vector                                                                                    Stratagene <br /> pCGH97                                              Carries CgHIS3 marker Tet-P                                                                   28 <br /> pKUILV2-1                                           YKU80-ILV2 region cloned pBluescriptII SK(&#11001;)                                                 This study <br /> pILV2-7                                             YKU80 ORF eliminated pKUILV2                                                                 This study <br /> pBS-SF.4                                            SAT1 flipper cloned pBluescriptII SK(&#11001;)                                                      This study <br /> pKU-self.5                                          pKUILV2-1 containing BglII site NheI site                                                 This study <br /> pKU-self(ORF2)                                      pKUILV2-1 containing BglII site NheI site                                                 This study <br /> pSFS2A                                              Contains FRT site, FLP1, CaSAT1 SAT1 flipper                                               34 <br /> pKUSF-18                                            SAT1 flipper integrated pKUILV2-1                                                            This study <br /> pKUSForf2-10                                        SAT1 flipper integrated pKUILV2-1                                                            This study <br />  VOL. 6, 2007                                               DEVELOPMENT OF A GENE TARGETING SYSTEM FOR C. GLABRATA                                              1241 <br />  <br /> inserted pBluescriptII SK(&#11001;) cleaved KpnI SacI         YPD plates containing 100 &#65533;?&#65533;g/ml NSTC, plates incubated <br /> generate pBS-SF.4.                                                                  37&#176;C 1 day. <br />    pKU-self.5. The plasmid pKU-self.5 constructed add BglII NheI            Strain construction. Strain tADE2-38, ADE2 expression able be <br /> sites YKU80. PCR carried primer pair pYKU80F2 (Bgl)          controlled Tet-P, constructed ACG4 (26) integration of <br /> and pYKU80R2 (Nhe) plasmid pKUILV2-1 template. The PCR prod-               Tet-P cassette ADE2 promoter locus (Table 1). The Tet-P cassette <br /> ucts treated T4 polynucleotide kinase circularized self-ligation   amplified PCR following primers template: pCGHade2F2 <br /> to construct pKU-self.5.                                                            pCgTETade2R1 primers plasmid DNA p97CGH (26) template. <br />    pKU-self(ORF2). PCR carried primer pair pYKU80ORF2.F            tADE2-38 screened SD plate histidine streaked on <br /> (Bgl) pYKU80ORF2.R (Nhe-Bgl) plasmid pKUILV2-1 template.               SD plate containing adenine doxycycline verify red colony <br /> PCR products digested circularized self-ligation.                       color resulted knockdown ADE2 expression. To confirm correct <br />    pKUSF-18 pKUSForf2-10. pBS-SF4 digested BglII NheI          integration Tet-P cassette, PCR carried using primers <br /> the inserted DNA, including SAT1 flipper, isolated plasmid.        pCgADE2checkF1 pCgADE2checkR1. <br /> The isolated fragment ligated plasmids pKU-self.5 pKU-               A disruption cassette amplified PCR using pILV2-7 template and <br /> self(ORF2) cleaved BglII NheI yield pKUSF-18 pKUSForf2-10,            using preKU80HindIII pILV2mR2 primers (see Table S1 supple- <br /> respectively.                                                                       mental material). This cassette included point mutation ILV2 changed <br />    Genomic DNA isolation C. glabrata DNA manipulation. Cells          amino acid 182 (proline) leucine. The cassettes used transfor- <br /> suspended 40 70 &#65533;?&#65533;l TE (10 mM Tris, pH 7.5, 1 mM EDTA), equal        mation parent strain ACG4 integrated YKU80-ILV2 locus by <br /> volume 0.5-mm glass beads added 2-ml collection tube. The cells       homologous recombination cassette entered cell. The trans- <br /> were disrupted Multi-Beads Shocker (Yasui Kikai, Inc.) using cycles    formant designated HET80 screened plates containing 20 &#65533;?&#65533;g/ml SM, and <br /> of 2,700 rpm 15 s 3-s brake time boiled 10 min. From    genomic DNA extracted order check YKU80 deletion. The <br /> resulting suspension, 1 &#65533;?&#65533;l added 25-&#65533;?&#65533;l PCR mixture.                       verification YKU80 deletion performed PCR sets primers; <br />    DNA amplification performed using KOD-plus-DNA polymerase                    check PCR 1 used preKU80checkR2 pKU80checkF2, check PCR 2 used <br /> (Toyobo), GoTaq green master mix (Promega), ready-to-go PCR beads (Am-           pKU80checkR1 pmILV2checkR1, check PCR 3 used preKU80HindIII <br /> ersham Biosciences) according manufacturers&#8217; protocols. BigDye Termi-        Ku80checkmidR3. <br /> nator cycle sequencing kits (version 3.1; Applied Biosystems, Inc.) Ap-         SAT1 flipper C. glabrata. The SAT1 flipper module 4.3 kb long and <br /> plied Biosystems model 3100 automated capillary sequencer used             contained following DNA elements: NSTC resistance gene SAT1, FLP1 <br /> nucleotide sequencing. Other standard recombinant-DNA procedures per-          encoding site-specific recombinase, FLP recombination target (FRT) <br /> formed described previously (15, 36).                                            (34-bp) sites recognized Flp1 recombinase (34). To examine the <br />    RNA manipulations quantitative real-time RT-PCR. For RNA extraction,         utility module C. glabrata, ACG4 transformed SAT1 <br />                                                                                     flipper, leading illegitimate integration. The transformant ACG4-SF was <br /> the hot-phenol method performed described previously (18). For quan- <br />                                                                                     confirmed PCR grown permissive conditions (YPD plate without <br /> titative real-time reverse transcription-PCR (RT-PCR), mRNA purified <br />                                                                                     NSTC). NSTC-sensitive clones screened subsequent testing growth <br /> from total RNA extracted Oligotex-dT(Super) mRNA purification kit <br />                                                                                     restrictive conditions YPD plates NSTC. Those clones lost <br /> (TaKaRa, Japan) according manufacturer&#8217;s protocol. To quantify mRNA, <br />                                                                                     SAT1 flipper, confirmed PCR analysis. To examine efficiency the <br /> we used Syber RT-PCR kit (TaKaRa) synthesis cDNA and <br />                                                                                     SAT1 flipper module C. glabrata, cells HETS202, SAT1 flipper <br /> real-time PCR. The real-time PCR ACT1 performed primers ACT1 <br />                                                                                     integrated, cultured permissive liquid medium overnight 37&#176;C, <br /> 59F ACT1 249R; YKU80 performed primers YKU80 622F <br />                                                                                     spread YPD plates, replicated restrictive plates (YPD, <br /> and YKU80 798R (see Table S1 supplemental material) using ABI <br />                                                                                     including 100 &#65533;?&#65533;g/ml NSTC). Only 0.5% colonies grow the <br /> PRISM 7000 (Applied Biosystems). <br />                                                                                     restrictive medium. Twelve colonies grow selective plates <br />    5&#11541; RACE. 5&#11032; RACE (rapid amplification cDNA ends) performed using <br />                                                                                     checked PCR, strains produced PCR product proved <br /> the GeneRacer kit (Invitrogen). Briefly, total RNA parent strain ACG4 was <br />                                                                                     loss SAT1 flipper. <br /> treated calf intestinal phosphatase treated tobacco acid <br />                                                                                        Knockdown using SAT1 flipper. To knock YKU80 gene, the <br /> pyrophosphatase remove 5&#11032; cap structure intact, full-length mRNA. <br />                                                                                     YKU80 knockdown cassette (Y-kd cassette) integrated just upstream of <br /> The GeneRacer RNA oligonucleotide, known priming site, ligated to <br />                                                                                     the second ATG <span id='am-149' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-150' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-151' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-152'  ></span><span id='am-153' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span>, yielding HETS146 HETS202, <br /> the 5&#11032; end mRNA using T4 RNA ligase. The ligated mRNA reverse <br />                                                                                     respectively. The Y-kd cassettes comprising SAT1 flipper flanked long <br /> transcribed cDNA random primers (N6). To obtain 5&#11032; ends, first-         homologous sequences amplified PCR using following templates and <br /> strand cDNA strands amplified using GeneRacer 5&#11032; nested primer         primers. For amplification Y-kd cassette, pKUSF-18 used as <br /> primer kdyku80checkR1. Purified RACE PCR products cloned              template, kdYKU80-1 kdYKU80-4 used primers. For the <br /> pCR4-TOPO vector (Invitrogen) sequencing.                                       amplification second Y-kd cassette, pKUSForf2-10 used template, <br />    C. glabrata transformation. The C. glabrata transformation protocol         preKU80HindIII pILV2mR2 used primers. After selection <br /> slightly modified version transformation protocol described previous      transformants YPD plates containing 100 &#65533;?&#65533;g/ml NSTC, PCR analysis was <br /> studies (20). Cells streaked YPD agar plate grown 2 3 days     performed confirm site-specific integration Y-kd cassettes. Check <br /> at 37&#176;C. A colonies incubated 10 ml YPD liquid medium       PCR 1 used primers pSFS-2AF2(PacI) kdYKU80checkF1; check PCR 2 <br /> cultured overnight shaking 37&#176;C. The cells resuspended 10 ml     used primers preku80checkR2 ku80checkmidR3. YKU80 knockdown strains <br /> fresh YPD (starting optical density 600 nm [OD600] 0.4) shaken      gene disruption, HETS202, KUE100, KUE200, constructed by <br /> OD600 1.0, approximately 2 3 h 37&#176;C. The cells harvested           using C. glabrata strains 2001H 2001HT (21). <br /> centrifugation 6,000 &#11003; g 5 min. The pellets rinsed 10 ml TE        POP-SF1467 POP-SF221 isolated following procedure. Trans- <br /> buffer harvested centrifugation. The cells resuspended 10 ml      formants derived HETS146 HETS202 incubated YPD liquid <br /> 0.15 M lithium acetate dissolved TE buffer (LiOAc/TE) shaken lightly     medium NSTC 37&#176;C 24 h shaking. Under condition, the <br /> 1 h 30&#176;C. The cells harvested resuspended 400 &#65533;?&#65533;l 0.15 M     FLP1-mediated excision SAT1 flipper occurs automatically. The cells were <br /> LiOAc. Sixty microliters suspension dispensed new 1.5-ml tube,      spread YPD agar plates NSTC incubated 37&#176;C 24 h. The <br /> supplemented 5 10 &#65533;?&#65533;g (3 &#65533;?&#65533;l) disruption cassettes 20 &#65533;?&#65533;g (2 &#65533;?&#65533;l)    colonies replicated YPD plates 100 &#65533;?&#65533;g/ml NSTC. <br /> of carrier DNA (salmon sperm DNA [Wako], boiled use 10 min           Clones, did grow plate containing NSTC, confirmed to <br /> chilled 10 min denature double-strand DNA) mixed gently. This            excised SAT1 flipper PCR: check PCR 3 used primers pFRT1 F1 and <br /> solution incubated 30 min 37&#176;C added 120 &#65533;?&#65533;l 52.5%         kdYKU80checkF1; check PCR 4 used primers kdYKU80checkF1 and <br /> polyethylene glycol 4000-0.15 M LiOAc mixed entirely pipetting. The cells    kdYKU80checkR1. POP-SF1467 POP-SF221 derived HETS146 <br /> were incubated 45 min 37&#176;C. After mixing carefully, cells heat      HETS202, respectively. <br /> shocked incubating 45 min 42&#176;C spread appropriate              Determination gene targeting efficiency. The inactivation ADE2, is <br /> selection plates incubated 37&#176;C days. For NSTC-resistant strains   genes required adenine biosynthesis, induces color colonies <br /> like HETS202, cells harvested resuspended 1 ml fresh YPD         turn red accumulation intermediate metabolite phosphori- <br /> liquid medium 42&#176;C heat shock suspension shaken 37&#176;C      bosyl aminoimidazole (19). On agar plate containing doxycycline, trans- <br /> for 12 24 h. The cells harvested, washed YPD liquid medium,     formant (tADE2), Tet-P integrated promoter region of <br /> resuspended 400 &#65533;?&#65533;l YPD liquid medium. The suspension spread          ADE2 homologous recombination, forms colonies turn red. On other <br />  1242        UENO ET AL                                                                                                                               EUKARYOT. CELL <br />  <br />  <br />  <br />  <br />    FIG. 1. Gene targeting efficiency depends length the <br /> homologous flanking sequence. The numbers shown right side <br /> describe number correct integrants total number colonies <br /> tested. The lengths homologous tags shown left side. The <br /> designation 80-500 means 80-bp flanking sequence a <br /> 500-bp flanking sequence attached 5&#11032; 3&#11032; sides, respec- <br /> tively, DNA cassette. The efficiency gene targeting accompa- <br /> nied homologous recombination measured determining                            FIG. 2. Construction YKU80 deletion mutant using SM resis- <br /> frequency integration DNA cassette bearing Tet-P                      tance. (A) Procedure construction YKU80 deletion <br /> endogenous promoter region ADE2. When Tet-P integrated                         strain, HET80, PCR-mediated cassette. On chromosome K C. <br /> into correct locus, colonies turn red plates supplemented                     glabrata, YKU80 (CAGL0K03443g) adjacent ILV2 (CAGL0K <br /> with doxycycline. As S. cerevisiae, red colony color caused                     03465g), followed 1 kb intergenic space. A point mutation in <br /> inactivation ADE2 gene C. glabrata (19). Closed open                       Ilv2p, proline 182 replaced leucine, confers resistance to <br /> boxes represent gene targeting efficiency HET80 ACG4,                         SM S. cerevisiae (41). The cassette DNA fragment contained 5&#11032; <br /> respectively.                                                                            homologous flanking region YKU80-ILV2 locus the <br />                                                                                          ILV2 gene carrying point mutation changed proline182 to <br />                                                                                          leucine. Integration cassette removes YKU80 induces the <br />                                                                                          point mutation Ilv2p, confers SM resistance. The parent strain <br /> hand, transformants Tet-P integrated region form         ACG4 transformed deletion cassette, transformants <br /> colonies remain white. The gene targeting efficiency (%) calculated          selected plates containing SM. The asterisks indicate point <br /> the following formula: number tADE2 strains obtained transforma-             mutation. (B) PCR analysis yku80 deletion strain three <br /> tion/total transformants &#11003; 100%. For list primer pairs used attach           sets primers. Lanes h results ACG4 HET80, <br /> homologous tags Tet-P cassette, Table S1 supplemental mate-            respectively. The lengths expected PCR products shown on <br /> rial. The calculation targeting efficiency using loci carried   right-hand panel A. <br /> with PCR using appropriate primer sets. <br />    Evaluation DNA damage &#9004;yku80 mutants vitro. To measure the <br /> sensitivity strains DNA-damaging agents, employed microdilution <br /> method based CLSI (formerly NCCLS) standard M27-A (27). Drop titer                killed cervical dislocation. Both kidneys extirpated, homogenized, di- <br /> tests spread titer tests sensitivity carried out. Cells        luted saline, spread agar plates determine number CFU <br /> overnight culture harvested, rinsed YPD, adjusted               C. glabrata kidneys. This analysis carried days 1, 7, 14 <br /> OD600 0.3. This suspension diluted 10&#11002;3, 3 &#65533;?&#65533;l dilution            injection C. glabrata. The animal experiment complied all <br /> dropped plate containing reagent. The plates incubated 37&#176;C         relevant guidelines policies Animal Welfare Committee Faculty <br /> 2 days. The spread titer test measure sensitivity UV rays performed        Medicine Chiba University, Japan. <br /> as follows. Cells overnight culture harvested, rinsed YPD, and <br /> adjusted OD600 4. This suspension diluted 10&#11002;4. Fifty microliters of <br />                                                                                                                           RESULTS <br /> the diluted suspension spread plates. These plates exposed to <br /> 0, 20, 40, 60 J/m2 254-nm UV rays (CL-1000 UV cross-linker [UV Products])                Length homologous sequence required efficient gene <br /> and incubated 37&#176;C 1 day. The CFU determined each <br /> exposed plate, survival rate (%) calculated following formula: <br />                                                                                          targeting correct locus ACG4. The length require- <br /> CFU exposed plate/CFU unexposed plate &#11003; 100%.                             ment homologous sequence induce correct gene <br />    Infection experiment. Seven male CD-1 (ICR) mice, 5 weeks old weighing            targeting measured ACG4 (Fig. 1). The DNA cassette <br /> 28.5 30.9 g, used virulence studies group. Cyclophosphamide         consists Tet-P, HIS3 gene selection marker, and <br /> (Shionogi &amp; Co., Ltd., Japan) hydrocortisone sodium phosphate (Banyu <br />                                                                                          homologous sequences ranging size 40 1,000 bp on <br /> Pharmaceutical Co., Ltd., Japan) concomitantly administered lateral <br /> tail vein doses 200 mg/kg body weight 125 mg/kg, respectively. Three        side. When flanking sequences 40 bp, cor- <br /> days administration, 100 mg/kg 125 mg/kg cyclophospha-            rectly targeted integrants identified. The highest effi- <br /> mide hydrocortisone, respectively, 1 &#11003; 107 cells C. glabrata             ciency gene targeting 63.5% length the <br /> administered lateral tail veins mice. The amounts            flanking sequences 500 bp. The efficiency decreased to <br /> immunosuppressive agents administered 4 days. The cells were <br /> suspended saline final volume 250 &#65533;?&#65533;l prior administration to <br />                                                                                          16.3% homologous sequence 5&#11032; 80 bp, <br /> the mice. Out seven mice allocated date, heaviest            3&#11032; 500-bp homologous sequence (Fig. 1). <br /> lightest mice removed remaining mice etherized                Unexpectedly, length homologous sequences <br />  VOL. 6, 2007                                        DEVELOPMENT OF A GENE TARGETING SYSTEM FOR C. GLABRATA                                     1243 <br />  <br />  <br />  <br />  <br />   FIG. 3. Phenotypic characterization &#9004;yku80 mutant using DNA-damaging agents. The experiment conducted measuring the <br /> OD600 time point. Closed circles open circles indicate results ACG4 HET80 (&#9004;yku80), respectively, panels. (A) The results <br /> of growth rate analyses 37&#176;C 42&#176;C indicated solid dotted lines, respectively. The vertical <span id='am-154' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-155' property="rdfs:label" content="Axis" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-157'  ></span><span id='am-158' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>axis</span> represented logarithm <br /> scale. Sensitivity reagent UV radiation evaluated determining cell survival ratio described percentage compared <br /> to cells grown DNA-damaging reagents UV radiation. The panels assigned sensitivities (B) MMS, (C) hydrogen <br /> peroxide, (D) UV radiation, (E) hydroxyurea. The sensitivity UV radiation calculated independent experiments shown <br /> as means &#11006; standard deviations. <br />  <br />  <br />  <br />  <br /> was 1,000 bp, targeting efficiency lower effi-                Gene targeting efficiency short homologous sequences <br /> ciency length 500 bp. From results,               HET80. The yku80 deletion mutant HET80 evaluated <br /> suggested efficient gene targeting required              gene targeting efficiency (Fig. 1). The DNA cassette, in- <br /> base pairs homologous flanking sequence ACG4.                         cluding Tet-P homologous flanking sequences <br />    Effect carrier DNA concentration transformation effi-              ranging 40 200 bp, integrated promoter <br /> ciency. Denatured salmon sperm DNA used carrier DNA                   region ADE2. The efficiency gene targeting 5.1% with <br /> for experiments S. cerevisiae K. lactis (14, 22). The              40 bp homologous flanking sequence HET80, this <br /> requirement carrier DNA efficient gene                length homologous sequences provided correct integration <br /> targeting C. glabrata investigated work. The                 ACG4. Likewise, 80-bp flanking homologous sequences, <br /> efficiency measured integration rate Tet-P                efficiency gene targeting 8.6% HET80, approxi- <br /> cassette ADE2 promoter 200-bp homologous                    mately sevenfold efficient ACG4. The efficiency rose <br /> sequence. The highest efficiency identified 20 &#65533;?&#65533;g               80% 500 bp homologous flanking sequence HET80. <br /> carrier DNA used. About two-times-higher efficiency                 The relative increase rate gene targeting efficiency <br /> achieved using carrier DNA                 caused YKU80 deletion greater shorter length of <br /> carrier DNA used.                                                       homologous flanking sequence HET80 (Fig. 1). <br />    Construction YKU80 deletion mutant efficient gene                To evaluate targeting efficiency various targets C. <br /> targeting. A search C. glabrata genomic sequences                 glabrata, DNA cassettes targeting 27 different loci were <br /> Genolevures (http://cbi.labri.fr/Genolevures/) revealed            produced introduced HET80. Between 5 30 col- <br /> gene homology S. cerevisiae YKU80. It designated YKU80            onies transformants checked correct inte- <br /> (CAGL0K03443g) C. glabrata. The amino acid sequences de-                 gration site PCR. The average efficiency 13% 60 bp <br /> duced genes shared 39% identity 58% similarity.              homologous flanking sequence. These results suggest that <br /> A DNA cassette designed delete YKU80 simulta-                    gene targeting HET80 effective various loci. <br /> neously confer resistance sulfometuron methyl. Recipient                    Susceptibilities yku80 mutant DNA-damaging agents. <br /> strain ACG4 transformed cassette construct yku80            To characterize yku80 deletion mutant C. glabrata, the <br /> mutant HET80. To confirm construction yku80 deletion,                effects temperature DNA-damaging agents exam- <br /> PCR analysis performed using sets primers (Fig. 2).            ined HET80. Since C. glabrata grows 37&#176;C, 37&#176;C and <br />  1244     UENO ET AL                                                                                                           EUKARYOT. CELL <br />  <br />  <br />  <br />  <br />   FIG. 4. Insertion pop-out SAT1 flipper. The ATG YKU80 shown Genolevures (http://cbi.labri.fr/Genolevures/). A second <br /> ATG identified 81 bp downstream ATG codon frame. To construct yku80 knockdown strains (HETS strains), <br /> the SAT1 flipper integrated sites 5&#11032; the second ATG YKU80 ORF yield HETS146 HETS202, respectively. <br /> The SAT1 flipper removed HETS146 HETS202 strains, producing POP-SF1467 POP-SF221, respectively, were <br /> generated following permissive conditions: YPD medium NSTC overnight shaking 37&#176;C. The 34-bp palindrome DNA, <br /> the FRT, presented arrowheads face face, remains POP-SF1467 POP-SF221. <br />  <br />  <br />  <br /> 42&#176;C chosen permissive restrictive temperatures,              cassette containing Tet-P flanking 200-bp homolo- <br /> respectively. Whereas growth slightly delayed               gous sequence. With use HETS146 POP-SF1467, <br /> 42&#176;C HET80, distinct growth defect relative wild-            efficiency rates 1.97 27.4%, respectively, unex- <br /> type strain identified. No valid differences HET80            pectedly lower recipient included SAT1 flipper <br /> and ACG4 recognized mortality cells exposed             lost it. On hand, using HETS202 and <br /> UV irradiation, MMS, H2O2, hydroxyurea (Fig. 3). These                 POP-SF221, rates 40.3 25.0%, respectively. These <br /> results suggest YKU80 deletion mutant C. glabrata             results indicate expression YKU80 repressed <br /> is disabled general cellular functions DNA repair. <br />    Construction YKU80 knockdown strain insertion of <br /> SAT1 flipper. To employ beneficial effects YKU80 inac- <br /> tivation gene targeting complicating effects, <br /> including possible synthetic sickness lethality, necessary <br /> to construct strain expression YKU80 is <br /> restored following transient knockdown transforma- <br /> tion. To realize convenient strain, module SAT1 <br /> flipper integrated just ATG anno- <br /> tated open reading frame YKU80, creating strain <br /> HETS146. Unfortunately, gene targeting efficiency was <br /> greatly decreased strain (Fig. 4 5). This an <br /> unexpected result, constructed strain, <br /> HETS202, SAT1 flipper integrated just before <br /> the second ATG, 81 nucleotides downstream the <br /> first ATG frame. During incubation non- <br /> selective conditions, HETS146 HETS202 lost SAT1 <br /> flipper produced POP-SF1467 POP-SF221, respec- <br /> tively.                                                                      FIG. 5. Gene targeting efficiency yku80 knockdown strains <br />                                                                           revertant strains. The determination gene targeting effi- <br />    Evaluation gene targeting efficiency YKU80 knock-            ciency construct measured shown Fig. 1. Targeting <br /> down strains. The efficiency gene targeting measured            efficiency 200 bp homologous flanking sequence corresponding <br /> the integration promoter region ADE2 DNA                 ADE2 promoter region measured. <br />  VOL. 6, 2007                                      DEVELOPMENT OF A GENE TARGETING SYSTEM FOR C. GLABRATA                                     1245 <br />  <br />  <br />  <br />  <br />   FIG. 6. Relative quantification YKU80 transcription real-time     FIG. 7. Effect FRT sequence survival ratio vivo. A total of <br /> RT-PCR. mRNA harvested cells overnight culture       1 &#11003; 107 cells strain administered lateral tail veins <br /> 37&#176;C. <span id='am-196' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-197' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-198'  ></span><span id='am-199' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-200' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Data</span> shown means &#11006; standard deviations          mice. Fungal burden kidney measured days 1, 7, and <br /> independent experiments. Transcription detected HET80,       14 infection. Outliers statistically dismissed using the <br /> the yku80-deleted strain. *, P 0.042 versus HETS202        Thompson test colony counting, P values 0.05 were <br /> particular condition. #, P 0.581 versus ACG4 particular    considered significant. ACG4, POP-SF221, KPOP11538 indi- <br /> condition.                                                              cated closed circles, open circles, open squares (cyb2 a <br />                                                                         positive control), respectively. Error bars indicate standard deviations. <br />  <br />  <br /> when SAT1 flipper located second ATG and <br /> restored lost (Fig. 5).                                     host strain strain genetically stable, ele- <br />    Transcription initiation site YKU80. To confirm site       vated mutation frequency occurring maintenance. In <br /> the promoter region YKU80, initiation site transcrip-         order confirm genetic stability YKU80 knockdown <br /> tion YKU80 investigated 5&#11032; RACE experiment. The             strain, mutation rate HETS202 investigated. The <br /> sequences clones, including 5&#11032; RACE products,               mutation rates HETS202 ACG4 (as control) were <br /> were determined. This showed transcription initi-          measured frequency 5-fluoroorotic acid (5-<span id='am-13' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-14' property="rdfs:label" content="Foa" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-16'  ></span><span id='am-17' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>FOA</span>) <br /> ated downstream ATG (data shown). These                resistance. The strains subcultured YPD plates con- <br /> results indicated ATG initiation codon      taining 100 &#65533;?&#65533;g/ml NSTC 10 times, number CFU was <br /> YKU80.                                                                  determined YPD plates containing 0.2% 5-FOA. The mu- <br />    Evaluation yku80 knockdown effect gene targeting           tation rates ACG4 HETS202 (2.97 &#11006; 1.38) &#11003; 10&#11002;7 <br /> to various loci. The gene targeting efficiency 3.5% (4/114)         (4.15 &#11006; 1.08) &#11003; 10&#11002;7 (P &#11005; 0.31), respectively, indicating <br /> in HETS202 40 bp homologous sequence correspond-                significant difference mutation rate caused repres- <br /> ing ADE2 promoter region. To extend results             sion YKU80. <br /> other loci, gene targeting efficiency measured 15 loci          In vivo survival stains carrying FRT YKU80 promoter. <br /> using KUE200 SAT1 flipper integrated upstream              To evaluate virulence strains FRT remains <br /> of second ATG. The DNA cassettes included, cases,            ORF YKU80, vivo experiment carried <br /> 56 bp homologous flanking sequence             out. Cells ACG4 POP-SF221 KPOP11538 were <br /> HIS3 selective marker. After transformation, 3 30               administered lateral tail veins mice. KPOP11538 <br /> colonies checked PCR. The efficiency gene targeting          cyb2 mutation used control growth <br /> was approximately 5 10% genes higher               defects previously identified vitro (Ueno, unpub- <br /> 30% genes. The results suggest gene targeting             lished). The fungal burdens ACG4 POP-SF221 were <br /> with short homologous sequences yku80 knockdown strains              transiently decreased day 7 increased day 14, while <br /> is effective multiple loci.                                          fungal burden KPOP11538 gradually decreased and <br />    Effect remaining FRT repression YKU80 tran-             eventually fell limit detection. The pattern the <br /> scription. After SAT1 flipper removed genome,           survival curve different ACG4 POP-SF221 <br /> the FRT sequence, 34-bp short palindrome, remains            (Fig. 7). These results indicate FRT located YKU80 <br /> in sequence upstream second ATG YKU80                  promoter region effect survival C. glabrata in <br /> POP-SF221. The effect gene expression caused FRT              vivo. <br /> remaining 5&#11032; end YKU80 ORF quantified by <br /> real-time PCR. Since mRNA YKU80 tran-                                               DISCUSSION <br /> scribed stationary growth phase exponen- <br /> tial growth phase (K. Ueno et al., unpublished data), mRNA                 To facilitate systematic C. glabrata mutant construction a <br /> was harvested cells stationary phase. The            high-throughput manner, new recipient strain permits <br /> of mRNA relative ACG4 shown Fig. 6. The YKU80                  efficient gene targeting required. As step, demon- <br /> mRNA level 0.38-fold HETS202, POP-SF221                  strated targeting efficiency short homologous <br /> 1.09-fold, mRNA identified HET80. Thus,               sequences increased yku80 deletion mutant. This made <br /> no effect FRT transcription identified POP-SF202           strain useful efficient gene targeting, leaving the <br /> in comparison wild-type progenitor.                              gene crippled misleading results disruptants. In S. <br />    Mutation rate measured spontaneous 5-FOA resistance               cerevisiae, genetic interactions YKU80 other <br /> in HETS202. It important criterion transformation            genes, including synthetic lethal combinations, re- <br />  1246     UENO ET AL                                                                                                               EUKARYOT. CELL <br />  <br />  <br /> ported (29, 33). There effects, including unknown         function analyses, elucidating pathogenicity facilitating the <br /> synthetic sickness lethality, induced YKU80 inactivation.       identification antifungal drug targets C. glabrata. <br /> Therefore, YKU80 restored activity the <br /> analysis specific function gene mutant                                ACKNOWLEDGMENTS <br /> construction. Cotransformation DNA cassette Tet-P             We thank P. T. Magee (University Minnesota) critical reading <br /> integration DNA cassette YKU80 restoration did          manuscript, O. Reuss (Universitat Wurzburg, Germany) the <br /> succeed (data shown), decided construct strain         gift plasmid pSFS2A, K. Kitada sharing C. glabrata strains <br /> in expression YKU80 repressed            2001H, 2001T, 2001HT. <br />                                                                          This work supported grant-in-aid scientific research for <br /> transformation optimal gene targeting expression          following priority areas, infection host response, genome bi- <br /> easily restored gene targeting.                             ology microbial genome, Ministry Education, Culture, <br />     The SAT1 flipper FLP1-mediated pop-out          Sports, Science Technology, Japan. <br /> is efficiently removed chromosomal sites C. albicans <br />                                                                                                       REFERENCES <br /> (34). In work, used SAT1 flipper the <br />                                                                        1. Barnes, G., D. Rio. 1997. DNA double-strand-break sensitivity, DNA <br /> repression YKU80 expression C. glabrata. The SAT1 flip-             replication, cell cycle arrest phenotypes Ku-deficient Saccharomyces <br /> per previously reported useful gene                cerevisiae. Proc. Natl. Acad. Sci. USA 94:867&#8211;872. <br />                                                                        2. Barns, S. M., D. J. Lane, M. L. Sogin, C. Bibeau, W. G. Weisburg. 1991. <br /> manipulation C. glabrata. The maltose promoter likely           Evolutionary relationships pathogenic Candida species relatives. <br /> to regulate transcription C. glabrata C. albicans,       J. Bacteriol. 173:2250&#8211;2255. <br /> because DNA cassette SAT1 flipper frequently lost           3. Baudin, A., O. Ozier-Kalogeropoulos, A. Denouel, F. Lacroute, C. Cul- <br />                                                                           lin. 1993. A simple efficient method direct gene deletion Saccha- <br /> medium, including glucose. Even cells cultured              romyces cerevisiae. Nucleic Acids Res. 21:3329&#8211;3330. <br /> in YPD, including NSTC, 26.3% lost SAT1 flipper.           4. Berman, J., P. E. Sudbery. 2002. <span id='am-124' about='obo:NCBITaxon_5476' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_5475 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_118267 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-125' property="oboInOwl:hasAlternativeId" content="NCBITaxon:39399" datatype="xsd:string"></span><span id='am-126' property="rdfs:label" content="Candida albicans" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasDbXref" content="GC_ID:12" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasRelatedSynonym" content="Candida stellatoidea" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasRelatedSynonym" content="Candida stellatoidea type I" datatype="xsd:string"></span><span id='am-130'  ></span><span id='am-131' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Candida albicans</span>: molecular revolu- <br /> The high frequency pop-out convenient work,            tion built lessons budding yeast. Nat. Rev. Genet. 3:918&#8211;930. <br />                                                                        5. Boulton, S. J., S. P. Jackson. 1996. Identification Saccharomyces <br /> since step strain plated maltose medium           cerevisiae Ku80 homologue: roles DNA double strand break rejoining and <br /> induce pop-out skipped. For uses, however,               telomeric maintenance. Nucleic Acids Res. 24:4639&#8211;4648. <br />                                                                        6. Burke, D., D. Dawson, T. Stearn. 2000. Methods yeast genetics: Cold <br /> the promoter needs replaced controllable pro-                Spring Harbor Laboratory course manual, 2000 ed. Cold Spring Harbor <br /> moter order obtain better regulation FLP1. These                 Laboratory Press, Cold Spring Harbor, NY. <br /> results suggest MAL2 promoter SAT1 flipper             7. Calderone, R. A. 2002. Candida candidiasis. ASM Press, Washington, DC. <br />                                                                        8. da Silva Ferreira, M. E., M. R. Kress, M. Savoldi, M. H. Goldman, A. Hartl, <br /> cassette can&#8217;t repressed medium, including glucose,           T. Heinekamp, A. A. Brakhage, G. H. Goldman. 2006. The akuBKU80 <br /> it adequately leaky generate derivatives SAT1          mutant deficient nonhomologous end joining powerful tool for <br /> flipper removed.                                                       analyzing pathogenicity <span id='am-211' about='obo:NCBITaxon_746128' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_5042 obo:NCBITaxon_5052 obo:NCBITaxon_33154 obo:NCBITaxon_147545 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_1131492 obo:NCBITaxon_451871 obo:NCBITaxon_2759'><span id='am-212' property="oboInOwl:hasRelatedSynonym" content="Aspergillus fumigates" datatype="xsd:string"></span><span id='am-213' property="rdfs:label" content="Aspergillus fumigatus" datatype="xsd:string"></span><span id='am-214' property="oboInOwl:hasAlternativeId" content="NCBITaxon:945814" datatype="xsd:string"></span><span id='am-215' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-216' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-217' property="oboInOwl:hasRelatedSynonym" content="Sartorya fumigata" datatype="xsd:string"></span><span id='am-218'  ></span><span id='am-219' property="oboInOwl:hasAlternativeId" content="NCBITaxon:5085" datatype="xsd:string"></span><span id='am-220' property="oboInOwl:hasRelatedSynonym" content="Neosartorya fumigata" datatype="xsd:string"></span>Aspergillus fumigatus</span>. Eukaryot. Cell 5:207&#8211;211. <br />                                                                        9. Duda &#180;sova&#180;, Z., A. Dudas, M. Chovanec. 2004. Non-homologous end- <br />     In yku80 mutants S. cerevisiae, severe growth defect          joining factors <span id='am-23' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-24' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-31'  ></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>. FEMS Microbiol. Rev. 28:581&#8211; <br /> 37&#176;C identified telomere shortening,                601. <br />                                                                       10. Dujon, B., D. Sherman, G. Fischer, P. Durrens, S. Casaregola, I. Lafontaine, <br /> other causes (1, 5, 9, 11). These cells exhibit hypersensi-          J. De Montigny, C. Marck, C. Neuveglise, E. Talla, N. Goffard, L. Frangeul, <br /> tivity DNA-damaging agents, MMS, UV                 M. Aigle, V. Anthouard, A. Babour, V. Barbe, S. Barnay, S. Blanchin, J. M. <br /> radiation hydroxyurea (5, 16, 37). In &#9004;yku80 mutant             Beckerich, E. Beyne, C. Bleykasten, A. Boisrame, J. Boyer, L. Cattolico, F. <br />                                                                           Confanioleri, A. De Daruvar, L. Despons, E. Fabre, C. Fairhead, H. Ferry- <br /> C. glabrata, obvious growth delay identified 37&#176;C            Dumazet, A. Groppi, F. Hantraye, C. Hennequin, N. Jauniaux, P. Joyet, R. <br /> 42&#176;C. No sensitization MMS, H2O2, UV radiation, hy-                 Kachouri, A. Kerrest, R. Koszul, M. Lemaire, I. Lesur, L. Ma, H. Muller, <br /> droxyurea identified study (Fig. 3). These results            J. M. Nicaud, M. Nikolski, S. Oztas, O. Ozier-Kalogeropoulos, S. Pellenz, S. <br />                                                                           Potier, G. F. Richard, M. L. Straub, A. Suleau, D. Swennen, F. Tekaia, M. <br /> suggest DNA repair essential cell functions            Wesolowski-Louvel, E. Westhof, B. Wirth, M. Zeniou-Meyer, I. Zivanovic, <br /> not critically affected deletion yku80 C. glabrata. It       M. Bolotin-Fukuhara, A. Thierry, C. Bouchier, B. Caudron, C. Scarpelli, C. <br />                                                                           Gaillardin, J. Weissenbach, P. Wincker, J. L. Souciet. 2004. Genome <br /> seems likely DSB repair supported path-                evolution yeasts. Nature 430:35&#8211;44. <br /> way(s) independent YKU80, microhomology end                11. Feldmann, H., L. Driller, B. Meier, G. Mages, J. Kellermann, E. L. <br /> joining (23). In <span id='am-56' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span><span id='am-60'  ></span>Schizosaccharomyces</span> pombe, sensitiv-            Winnacker. 1996. HDF2, second subunit Ku homologue from <br />                                                                           Saccharomyces cerevisiae. J. Biol. Chem. 271:27765&#8211;27769. <br /> ity MMS did occur Ku single mutants, NHEJ               12. Fidel, P. L., Jr., J. A. Vazquez, J. D. Sobel. 1999. Candida glabrata: <br /> almost suppressed, verified plasmid repair assay           review epidemiology, pathogenesis, clinical disease comparison <br /> (24, 39).                                                                 C. albicans. Clin. Microbiol. Rev. 12:80&#8211;96. <br />                                                                       13. Giaever, G., A. M. Chu, L. Ni, C. Connelly, L. Riles, S. Veronneau, S. Dow, <br />     The single deletion yku80 C. glabrata obvious            A. Lucau-Danila, K. Anderson, B. Andre, A. P. Arkin, A. Astromoff, M. <br /> effect growth rate, DNA repair ability, mutation               El-Bakkoury, R. Bangham, R. Benito, S. Brachat, S. Campanaro, M. Cur- <br />                                                                           tiss, K. Davis, A. Deutschbauer, K. D. Entian, P. Flaherty, F. Foury, D. J. <br /> rate (Fig. 3). The SAT1 flipper maintained YKU80              Garfinkel, M. Gerstein, D. Gotte, U. Guldener, J. H. Hegemann, S. Hempel, <br /> knockdown strains selection 100 &#65533;?&#65533;g/ml NSTC                  Z. Herman, D. F. Jaramillo, D. E. Kelly, S. L. Kelly, P. Kotter, D. LaBonte, <br /> and automatically lost cells nonselective             D. C. Lamb, N. Lan, H. Liang, H. Liao, L. Liu, C. Luo, M. Lussier, R. Mao, <br />                                                                           P. Menard, S. L. Ooi, J. L. Revuelta, C. J. Roberts, M. Rose, P. Ross- <br /> conditions. This construct allows efficient gene targeting            Macdonald, B. Scherens, G. Schimmack, B. Shafer, D. D. Shoemaker, S. <br /> restoration expression YKU80 single transforma-            Sookhai-Mahadeo, R. K. Storms, J. N. Strathern, G. Valle, M. Voet, G. <br /> tion. Although FRT left promoter region            Volckaert, C. Y. Wang, T. R. Ward, J. Wilhelmy, E. A. Winzeler, Y. Yang, G. <br />                                                                           Yen, E. Youngman, K. Yu, H. Bussey, J. D. Boeke, M. Snyder, P. Philippsen, <br /> YKU80 SAT1 flipper excised chromosome,                 R. W. Davis, M. Johnston. 2002. Functional profiling Saccharo- <br /> the FRT detectable effect survival C. glabrata           myces cerevisiae genome. Nature 418:387&#8211;391. <br />                                                                       14. Gietz, R. D., R. H. Schiestl, A. R. Willems, R. A. Woods. 1995. Studies <br /> in vivo (Fig. 7). The YKU80 knockdown strain, HETS202,               transformation intact yeast cells LiAc/SS-DNA/PEG pro- <br /> new recipient strain enables rapid construction desired           cedure. Yeast 11:355&#8211;360. <br /> recombinants high-efficiency gene targeting; furthermore,         15. Glover, D. M., B. D. Hames. 1995. DNA cloning 1, practical approach: <br />                                                                           core techniques, 2nd ed. Oxford Universiy Press, Oxford, United <span id='am-193' about='obo:NCBITaxon_kingdom' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-194' property="rdfs:label" content="kingdom" datatype="xsd:string"></span><span id='am-195' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Kingdom</span>. <br /> strains deploying NHEJ regulation, e.g., HETS202, facili-        16. Goins, C. L., K. J. Gerik, J. K. Lodge. 2006. Improvements gene <br /> tate gene manipulation contribute comprehensive gene               deletion fungal pathogen Cryptococcus neoformans: absence Ku <br />  VOL. 6, 2007                                                  DEVELOPMENT OF A GENE TARGETING SYSTEM FOR C. GLABRATA                                               1247 <br />  <br />       proteins increases homologous recombination, cotransformation in-         30. Pastink, A., J. C. Eeken, P. H. Lohman. 2001. Genomic integrity the <br />       dependent DNA molecules allows rapid complementation deletion phe-                repair double-strand DNA breaks. Mutat. Res. 480&#8211;481:37&#8211;50. <br />       notypes. Fungal Genet. Biol. 43:531&#8211;544.                                         31. Pfaller, M. A., R. N. Jones, G. V. Doern, H. S. Sader, S. A. Messer, A. <br /> 17.   Gumbo, T., C. M. Isada, G. Hall, M. T. Karafa, S. M. Gordon. 1999.               Houston, S. Coffman, R. J. Hollis. 2000. Bloodstream infections to <br />       Candida glabrata fungemia. Clinical features 139 patients. Medicine (Bal-         Candida species: SENTRY Antimicrobial Surveillance Program North <br />       timore) 78:220&#8211;227.                                                                  America Latin America, 1997&#8211;1998. Antimicrob. Agents Chemother. <br /> 18.   Hanaoka, N., T. Umeyama, K. Ueno, K. Ueda, T. Beppu, H. Fugo, Y. Uehara,             44:747&#8211;751. <br />       M. Niimi. 2005. A putative dual-specific protein phosphatase encoded      32. Po&#168;ggeler, S., U. Kuck. 2006. Highly efficient generation signal trans- <br />       YVH1 controls growth, filamentation virulence Candida albicans.               duction knockout mutants using fungal strain deficient mammalian <br />       Microbiology 151:2223&#8211;2232. <br />                                                                                            ku70 ortholog. Gene 378:1&#8211;10. <br /> 19.   Hanic-Joyce, P. J., P. B. Joyce. 1998. A high-copy-number ADE2- <br />                                                                                        33. Polotnianka, R. M., J. Li, A. J. Lustig. 1998. The yeast Ku heterodimer <br />       bearing plasmid transformation Candida glabrata. Gene 211:395&#8211;400. <br /> 20.   Ito, H., Y. Fukuda, K. Murata, A. Kimura. 1983. Transformation                essential protection telomere nucleolytic recombi- <br />       intact yeast cells treated alkali cations. J. Bacteriol. 153:163&#8211;168.           national activities. Curr. Biol. 8:831&#8211;834. <br /> 21.   Kitada, K., E. Yamaguchi, M. Arisawa. 1995. Cloning Candida           34. Reuss, O., A. Vik, R. Kolter, J. Morschhauser. 2004. The SAT1 flipper, <br /> </body></html>