<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>tRNAHis guanylyltransferase catalyzes <br /> a 3&#11541;-5&#11541; polymerization reaction that is <br /> distinct from G&#11546;1 addition <br />  <br />  <br /> Yeast tRNAHis guanylyltransferase, Thg1, is an essential protein <br /> that adds a single guanine to the 5&#11541; end (G&#11546;1) of tRNAHis. <span id='am-104' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-105' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-108'  ></span>This</span> G&#11546;1 <br /> residue is required for aminoacylation of tRNAHis by histidyl-tRNA <br /> synthetase, both in vitro and in vivo. The guanine nucleotide <br /> addition reaction catalyzed by Thg1 extends the polynucleotide <br /> chain in the reverse (3&#11541;-5&#11541;) direction of other known polymerases, <br /> albeit by one nucleotide. Here, we show that alteration of the 3&#11541; <br /> end of the Thg1 substrate tRNAHis unleashes an unexpected re- <br /> verse polymerase activity of wild-type Thg1, resulting in the 3&#11541;-5&#11541; <br /> addition of multiple nucleotides to the tRNA, with efficiency <br /> comparable to the G&#11546;1 addition reaction. The addition of G&#11546;1 forms <br /> a mismatched G&#18528;A base pair at the 5&#11541; end of tRNAHis, and, with <br /> monophosphorylated tRNA substrates, it is absolutely specific for <br /> tRNAHis. By contrast, reverse polymerization forms multiple G&#18528;C or             Fig. 1. Nucleotide addition reactions catalyzed by 5&#11032;-3&#11032; polymerases and <br /> C&#18528;G base pairs, and, with preactivated tRNA <span id='am-109' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-110' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span>, it can initiate           tRNAHis guanylyltransferase (Thg1). <br /> at positions other than &#11546;1 and is not specific for tRNAHis. Thus, <br /> wild-type Thg1 catalyzes a templated polymerization reaction <br /> acting in the reverse direction of that of canonical DNA and RNA               consistent with the demonstration that G&#11002;1 is necessary and <br /> polymerases. Surprisingly, Thg1 can also readily use dNTPs for                 sufficient for histidyl-tRNA synthetase activity in vitro (11&#8211;13). <br /> nucleotide addition. These results suggest that 3&#11541;-5&#11541; polymeriza-                 Because G&#11002;1 modification is a strong determinant of histidyl- <br /> tion represents either an uncharacterized role for Thg1 in RNA or              tRNA synthetase activity, Thg1 must be extraordinarily specific <br /> DNA repair or metabolism, or it may be a remnant of an earlier                 for tRNAHis to prevent misacylation of other tRNAs. Indeed, <br /> catalytic strategy used in nature.                                             Thg1 prefers its physiological substrate, monophosphorylated <br />                                                                                tRNAHis (p-tRNAHis), which arises after cleavage by RNase P, <br /> polymerase &#20841; <span id='am-46' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-47' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-51'  ></span><span id='am-52' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> &#20841; THG1 &#20841; tRNA modification               by &#11022;10,000-fold over other p-tRNA species (14). As for amino- <br />                                                                                acyl-tRNA synthetases, where the anticodon is often a major <br />                                                                                determinant of tRNA specificity, the tRNAHis GUG anticodon <br /> A     ll known polymerases catalyze the addition of nucleotides in <br />       the 5&#11032;-3&#11032; direction, including classical DNA and RNA <br /> polymerases, as well as telomerase, reverse transcriptase, and a <br />                                                                                is the source of Thg1 specificity because it is both necessary and <br />                                                                                sufficient for Thg1 activity in vitro (14). The three-step enzymatic <br /> number of template-independent RNA polymerases (1&#8211;3).                          reaction catalyzed by Thg1 involves the activation of the 5&#11032; end <br /> These polymerases universally catalyze the same reaction, in-                  of the tRNA by adenylylation, attack of the 3&#11032;-OH of GTP on <br /> volving attack by the polynucleotide 3&#11032;-OH on the 5&#11032;-                          the activated intermediate, and removal of the 5&#11032;-pyrophos- <br /> triphosphate of an incoming NTP, to yield an extended polynu-                  phate, yielding mature G&#11002;1-containing tRNAHis and AMP (Fig. <br /> cleotide and a pyrophosphate side product from the NTP (Fig.                   1B) (7, 8). These chemical steps are also strikingly similar to <br /> 1A). However, polymerization could just as easily occur chem-                  those catalyzed by aminoacyl-tRNA synthetases (14). However, <br /> ically in the reverse (3&#11032;-5&#11032;) direction with the same functional               despite these similarities between Thg1 and synthetases, as well <br /> groups, by attack of the 3&#11032;-OH of the incoming nucleotide on a                 as the proximity of their sites of action at the top of the tRNA <br /> 5&#11032;-triphosphorylated polynucleotide (Fig. 1B). We show here                    aminoacyl-acceptor stem, there is no identifiable homology <br /> that tRNAHis guanylyltransferase can act in this capacity as a                 between Thg1 and the synthetases or even between Thg1 and any <br /> reverse polymerase, catalyzing the 3&#11032;-5&#11032; addition of multiple                  other enzyme <span id='am-75' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-76' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span>. <br /> nucleotides to the polynucleotide chain in a reaction that is                     Formally, the Thg1 G&#11002;1 addition reaction is the 3&#11032;-5&#11032; extension <br /> distinct from its known physiological role.                                    of a polynucleotide chain, albeit by one nucleotide. Moreover, <br />    tRNAHis species are unique among tRNAs because they                         there is an alternative mode of Thg1 activity that more closely <br /> contain an additional universally conserved G residue at the &#11002;1                resembles a polymerase, in which triphosphorylated tRNAHis <br /> position (G&#11002;1), whereas only one other characterized tRNA has                  (ppp-tRNAHis) is used directly for the addition of G&#11002;1 (Fig. 1B), <br /> any nucleotide at that position (4, 5). In prokaryotes, G&#11002;1 is                 bypassing the ATP requirement for activation (7). <br /> genome-encoded, forms a standard base pair with C73, and is                       We report here that wild-type Thg1 can catalyze multiple <br /> retained during tRNAHis maturation because RNase P cleaves                     rounds of reverse polymerization with certain substrates. While <br /> pre-tRNA at the &#11002;1 position (6). By contrast, in eukaryotes, G&#11002;1 <br /> is added posttranscriptionally by tRNAHis guanylyltransferase, <br /> which is encoded in yeast by the essential THG1 gene (7), and                  Conflict of interest statement: No conflicts declared. <br /> it catalyzes the addition of a guanine nucleotide to the 5&#11032; end of             Abbreviations: CIP, calf intestinal alkaline phosphatase; PEI, polyethyleneimine; p-tRNA, <br /> tRNAHis across from residue A73 of the tRNA, the nucleotide                    monophosphorylated tRNA; ppp-tRNA, triphosphorylated tRNA. <br />  <br /> immediately preceding the CCA 3&#11032; end (7&#8211;9). G&#11002;1 addition                       &#8224;To whom correspondence should be addressed. E-mail: eric&#14061;phizicky@urmc.rochester.edu. <br />  <br /> activity is crucial for aminoacylation of tRNAHis in vivo (10),                &#169; 2006 by The National Academy of Sciences of the USA <br />  <br />  <br />  <br /> 8640 &#8211; 8645 &#20841; PNAS &#20841; June 6, 2006 &#20841; vol. 103 &#20841; no. 23                                                             www.pnas.org&#20862;cgi&#20862;doi&#20862;10.1073&#20862;pnas.0603068103 <br />  Fig. 2. Thg1 exhibits 3&#11032;-5&#11032; reverse polymerase activity with tRNAHis variants <br /> containing altered 3&#11032; ends. (A) Schematic of Thg1 assay with 5&#11032;-32P-labeled <br /> tRNAHis. After RNase A and phosphatase treatment, the Thg1 reaction prod- <br /> ucts [G[32P]pGpC (Gp*GpC) and longer species] are resolved from 32Pi derived <br />  <br />  <br />  <br />  <br />                                                                                                                                                                         BIOCHEMISTRY <br /> from substrate by TLC. (B) Thg1 adds guanine nucleotides to A73C tRNAHis <br /> variants. Activity with 5&#11032;-32P-labeled wild-type tRNAHis (a), tRNAHis C73CCA (b), <br /> tRNAHis C73CAA (c), or tRNAHis C73ACA (d) variants is shown using 5-fold serial <br /> dilutions of purified Thg1 (50 &#9262;g&#20862;ml to 0.4 &#9262;g&#20862;ml); &#11002;, no Thg1 added. (C) <br /> Confirmation of identity of additional 5&#11032; reaction products formed by Thg1. <br /> Comigration of tRNAHis C73CCA G&#11002;1, G&#11002;2, and G&#11002;3 nucleotide addition prod- <br /> ucts with [3&#11032;-32P]Cp-labeled, calf intestinal alkaline phosphatase (CIP)-treated <br /> oligonucleotide standards (GpGp*C, GpGpGp*C, and GpGpGpGp*C; lanes c, <br /> a, and b, respectively) is shown after resolution by silica TLC.                    Fig. 3. Thg1 adds guanine nucleotides to the 5&#11032; end of tRNAHis variants in a <br />                                                                                     template-dependent reaction. (A) Thg1 reverse polymerization does not re- <br />                                                                                     quire ATP with ppp-tRNA. Primer extension analysis of Thg1 reaction products <br /> investigating the tRNAHis determinants for Thg1 activity, we                        is shown with wild-type (a), C73CCA (b), C73CAA (c), or C73ACA (d) ppp-tRNAHis <br /> found that alteration of the conserved nucleotide A73 of tRNAHis                    species in the presence of ATP (a&#8211;d) or in the absence of ATP (e). The horizontal <br /> unleashes a previously unexpected reverse polymerase function                       line denotes the expected primer extension stop with unreacted transcript <br /> of wild-type Thg1, resulting in the 3&#11032;-5&#11032; extension of the polynu-                  (position &#11001;1), and additional nucleotide incorporations are indicated by (&#11002;1), <br /> cleotide chain by multiple nucleotides. Characterization of Thg1                    (&#11002;2), and (&#11002;3). The primer position is indicated by an arrow on the tRNA <br />                                                                                     diagram; G&#11002;1, G&#11002;2, and G&#11002;3 nucleotides incorporated by Thg1 with the tRNAHis <br /> reverse polymerization shows that this activity, unlike the addi- <br />                                                                                     C73CCA variant are indicated in parentheses. (B) Thg1 reverse polymerization <br /> tion of G&#11002;1, is template-dependent, recognizing G&#18528;C Watson&#8211;                         does not depend on the source of purified Thg1 protein. Shown is the primer <br /> Crick base pairs. Moreover, reverse polymerization is not spe-                      extension of reaction products from the addition of G&#11002;1 (with wild-type <br /> cific for tRNAHis or for starting at the &#11002;1 position of tRNA,                       tRNAHis) and reverse polymerization (with tRNAHis C73CCA) using yeast Thg1 <br /> provided that the activated (triphosphorylated) form of the                         purified from either <span id='am-63' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-64' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-68'  ></span><span id='am-69' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae (Sc) or <span id='am-112' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-113' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-124'  ></span><span id='am-125' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span>Escherichia coli</span> (Ec). (C) <br /> tRNA substrate is used in the assays.                                               Thg1 adds guanosine residues according to the number of C residues at the 3&#11032; <br />                                                                                     end of the tRNA species. Thg1 reaction products with tRNAHis C73CCCCA were <br /> Results                                                                             analyzed by primer extension, demonstrating the addition of up to five <br />                                                                                     nucleotides to the 5&#11032; end of the tRNA. <br /> Thg1 Adds Multiple Guanine Residues to the 5&#11541; End of A73C Variant <br /> tRNAHis Species. Although Thg1 normally adds a single guanine <br /> nucleotide to the 5&#11032; end of tRNAHis, Thg1 can add multiple                          a tRNAHis C73CAA variant results in a G&#11002;2 product at higher <br /> guanine nucleotides to a variant tRNAHis that bears C73 instead                     concentrations of Thg1 but no G&#11002;3 product (Fig. 2Bc), whereas <br /> of the A73 that is universally conserved in eukaryotic tRNAHis                      reaction with a tRNAHis C73ACA variant yields only a G&#11002;1 <br /> species (4, 15). We demonstrated this phenomenon with a <br />                                                                                     product (Fig. 2Bd). Thus, it appears that formation of additional <br /> sensitive assay that exploits the phosphatase resistance that <br />                                                                                     products depends on the identity of the A73CCA 3&#11032; end of tRNA. <br /> occurs when G&#11002;1 is added to 5&#11032; end-labeled tRNAHis (14). After <br />                                                                                        Examination of Thg1 activity with the corresponding ppp- <br /> incubation of tRNAHis with Thg1, treatment with RNase A and <br /> phosphatase yields the internally labeled trimer G[32P]pGpC if                      tRNA substrates shows that Thg1 has the same spectrum of <br /> G&#11002;1 has been added, or 32Pi from unreacted substrate (Fig. 2A),                     products with this alternatively activated tRNAHis substrate as <br /> which are well separated by TLC (Fig. 2Ba). However, reaction                       with the p-tRNAHis species, which require activation by ATP <br /> of Thg1 with a tRNAHis variant bearing A73C (tRNAHis C73CCA)                        (Fig. 3A). As measured by primer extension product length and <br /> also yields a prominent second product and a small amount of                        intensity, Thg1 efficiently adds a single guanine nucleotide <br /> a third product at higher concentrations of Thg1 (Fig. 2Bb).                        residue to ppp-tRNAHis (Fig. 3Aa), whereas up to three nucle- <br /> These additional products comigrate on TLC with chemically                          otides are added to a ppp-tRNAHis C73CCA variant (Fig. 3Ab), <br /> synthesized oligonucleotide standards corresponding to G&#11002;2 and                      two nucleotides are added to a ppp-tRNAHis C73CAA variant <br /> G&#11002;3 reaction products (Fig. 2C), and they are presumed to be                        (Fig. 3Ac), and a single nucleotide is added to a ppp-tRNAHis <br /> these species because only GTP and ATP are present in the                           C73ACA variant (Fig. 3Ad). We conclude that Thg1 catalyzes <br /> reactions, and all reaction products are sensitive to RNase T1,                     authentic 3&#11032;-5&#11032; reverse polymerization, defined as extension of <br /> producing Gp (<span id='am-141' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-142' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-143' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-144' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-145'  ></span>data</span> not shown). Similarly, reaction of Thg1 with                     the 5&#11032; end of a polynucleotide chain by more than a single <br />  <br /> Jackman and Phizicky                                                                                          PNAS &#20841; June 6, 2006 &#20841; vol. 103 &#20841; no. 23 &#20841; 8641 <br />  nucleotide. Because addition of multiple guanine nucleotides to <br /> the ppp-tRNAHis variants is at least as efficient in the absence of <br /> ATP (Fig. 3Ae), we conclude that Thg1 does not require ATP to <br /> activate the 5&#11032; end of the RNA for multiple-nucleotide additions; <br /> rather, we infer that the second and third guanine nucleotide <br /> additions, like the first guanine nucleotide addition, are powered <br /> by hydrolysis of pyrophosphate from the growing polynucleotide <br /> chain. Moreover, the slightly enhanced addition activity ob- <br /> served with the ppp-tRNAHis C73CCA variant in the absence of <br /> ATP suggests that ATP in fact inhibits the multiple-addition <br /> reaction. Because yeast Thg1 purified from E. coli exhibits the <br /> same multiple-nucleotide addition activity, we conclude that this <br /> activity is intrinsic to Thg1 protein (Fig. 3B). <br />    The addition of multiple guanine nucleotides by Thg1 is an <br /> efficient RNA reverse polymerization activity. Thg1 reverse <br /> polymerization with tRNAHis C73CCA and other variants occurs <br /> with similar efficiency (Figs. 2B and 3A) and on the same time <br /> scale (data not shown) as for the addition of G&#11002;1 to wild-type <br /> tRNAHis, whether the compared substrates are ppp-tRNAHis <br /> species or p-tRNAHis species. Furthermore, there is an element <br /> of template dependence in the reverse polymerization because <br /> the number of guanine nucleotides added to the 5&#11032; end of both <br /> p-tRNAHis and ppp-tRNAHis variants appears to be directed by <br /> the number of consecutive cytidine residues at the 3&#11032; end, <br /> starting at nucleotide 73 (Figs. 2B and 3). Indeed, up to five <br /> guanine residues are added across from the 3&#11032; end of tRNAHis <br /> C73CCCCA (Fig. 3C).                                                   Fig. 4. Nucleotide dependence of Thg1 reverse polymerization compared <br />                                                                       with the addition of G&#11002;1. (A) Nucleotide dependence of Thg1 activity with <br />                                                                       ppp-tRNAHis species. Thg1 reaction products formed in the presence of indi- <br /> Thg1 Reverse Polymerase Activity Is Template-Dependent Beyond <br />                                                                       vidual NTPs and ppp-tRNAHis substrates (with 3&#11032; end sequence indicated below <br /> Position &#11546;1. Closer examination of the nucleotide dependence of       each panel) were analyzed by primer extension. Lanes G, A, U, and C show each <br /> the reverse polymerase activity of Thg1 demonstrates that it is       NTP added as indicated; &#11002;, no nucleotide; NP, no Thg1. (B) Thg1 is specific for <br /> strictly template-dependent and recognizes G&#18528;C and C&#18528;G base           the addition of G at position &#11002;1 in the presence of all four NTPs. Thg1 reaction <br /> pairs. Consistent with a templated nucleotide addition reaction,      products with 5&#11032;-32P-labeled wild-type tRNAHis are shown in the presence of <br /> Thg1 can catalyze multiple rounds of reverse polymerization at        GTP and ATP (a) or a mixture of all four NTPs (GTP, ATP, UTP, and CTP) (b&#8211;d), <br /> the 5&#11032; end of tRNAHis C73CCA only in the presence of GTP (Fig.        each at a final concentration of 1 mM (b), 0.4 mM (c), or 0.1 mM (d); &#11002;, no <br />                                                                       added nucleotide; NP, no Thg1. (Left) RNase A and CIP treatment of reaction <br /> 4Ab), whereas with each of the other NTPs, Thg1 can catalyze <br />                                                                       products, resolved by standard silica TLC. (Right) RNase T2 digestion of reac- <br /> only a single addition reaction. Furthermore, reverse polymer-        tion products, to release 3&#11032;-labeled Np resulting from the addition of nucle- <br /> ization is not restricted to the addition of guanine nucleotide       otide at position &#11002;1, resolved by polyethyleneimine (PEI)-cellulose TLC in 0.5 <br /> because up to three cytidine nucleotides are added to the 5&#11032; end      M sodium formate (pH 3.5). Gp, Ap, Up, and Cp, migration of 3&#11032;-phosphory- <br /> of tRNAHis G73GGA with CTP (Fig. 4Ac), but only one nucle-            lated standards as visualized by fluorescence quenching. <br /> otide is added with each of the other NTPs. However, Thg1 does <br /> not appear to recognize U&#18528;A base pairs under these conditions <br /> because tRNAHis U73UUA or A73AAA species are not sub-                 3&#11541;-5&#11541; Reverse Polymerization of ppp-tRNA Substrates by Thg1 Is Not <br /> strates for reverse polymerization in the presence of ATP or          Specific for tRNAHis. Although our previous results demonstrate <br /> UTP, respectively (Fig. 4 Ad and Ae). Thus, polymerization            that the addition of Thg1 G&#11002;1 activity is at least 10,000-fold more <br /> beyond the &#11002;1 position appears to require formation of a              specific for tRNAHis than for other tRNAs when assayed in vitro <br /> canonical G&#11002;1&#18528;C73 or C&#11002;1&#18528;G73 base pair and appears to extend          with the physiological p-tRNAHis substrate (14), Thg1 is not <br /> further only if the next templated nucleotides at position 74 and     nearly as specific when assayed with ppp-tRNA substrates for <br /> beyond are G or C residues. Moreover, Thg1 is not restricted to       either G&#11002;1 addition activity (14) or reverse polymerization <br /> multiple-nucleotide additions of a single NTP because addition        activity. Thus, Thg1 adds a single G residue to wild-type <br /> to a tRNAHis G73CCA variant is somewhat more extensive in the         ppp-tRNAPhe, which normally contains an A73CCA 3&#11032; end (Fig. <br /> presence of both GTP and CTP than it is with GTP alone (Fig.          5A), and Thg1 efficiently adds up to three guanine residues to a <br /> 4Af ). The relatively weaker extension observed with this sub-        ppp-tRNAPhe C73CCA variant (Fig. 5A). These results demon- <br /> strate suggests that some nontemplated G&#11002;1 addition occurs            strate that reverse polymerase activity does not inherently <br /> opposite G73, which would create a nonpaired and therefore            depend on recognition of tRNAHis and therefore could poten- <br /> nonextendable substrate. Indeed, as shown below, the addition         tially occur with other substrates in vivo. <br /> of G is preferred at the &#11002;1 position.                                    Reverse polymerization by Thg1 also does not need to initiate <br />    Templated reverse polymerization by Thg1 beyond position           at the &#11002;1 position. Indeed, a G&#11002;1-containing tRNAHis C73CCA <br /> &#11002;1 is in sharp contrast to the addition reaction that occurs at the   transcript, which is the same molecule as the G&#11002;1 addition <br /> &#11002;1 position because Thg1 can add any nucleotide at the &#11002;1             product that results from Thg1 activity on tRNAHis C73CCA, is <br /> position with each of the ppp-tRNA species examined for reverse       a substrate for continued addition of up to two additional <br /> polymerization (Fig. 4 Ab&#8211;Af ), as well as with wild-type tRNAHis     nucleotides as demonstrated by primer extension (Fig. 5B). <br /> (Fig. 4Aa). However, consistent with the known role of Thg1,          However, the corresponding G&#11002;1-containing wild-type tRNAHis <br /> when all four NTPs are present, Thg1 adds exclusively G at            A73CCA 76-mer transcript cannot be extended further, presum- <br /> position &#11002;1 of wild-type p-tRNAHis opposite A73, as indicated by      ably because it is the same molecule as mature tRNAHis, with an <br /> RNase T2 analysis of reaction products, which yields only Gp          unpaired G&#11002;1 opposite A73 (Fig. 5B). Likewise, Thg1 adds two <br /> (Fig. 4B).                                                            nucleotides to a tRNAHis transcript that initiates at residue &#11001;2; <br />  <br /> 8642 &#20841; www.pnas.org&#20862;cgi&#20862;doi&#20862;10.1073&#20862;pnas.0603068103                                                                             Jackman and Phizicky <br />                                                                                                                                                                      BIOCHEMISTRY <br /> Fig. 5. Thg1 reverse polymerase activity is not specific for tRNAHis and does <br /> not need to begin at the &#11002;1 position. (A) tRNAPhe C73CCA is a substrate for <br /> Thg1 reverse polymerization. Thg1 catalyzes the addition of G&#11002;1 to wild-type <br /> ppp-tRNAPhe and addition of multiple G residues to ppp-tRNAPhe C73CCA, as <br /> analyzed by primer extension. (B) Reverse polymerization can initiate at <br /> position &#11002;1. Thg1 catalyzes addition of G&#11002;2 and G&#11002;3 to a tRNAHis 76-mer <br /> transcript that already contains G&#11002;1 when this G&#11002;1 residue is base-paired with <br /> C73, but not when it is opposite A73, as analyzed by primer extension. (C) <br /> Reverse polymerization can initiate at position &#11001;2. Thg1 catalyzes the addi- <br /> tion of G&#11001;1 and G&#11002;1 to a tRNAHis variant that begins at the &#11001;2 position, yielding <br /> the same G&#11002;1-containing reaction product as a control RNA species that begins <br /> with G&#11001;1, as analyzed by primer extension. (D) Thg1 does not add additional         Fig. 6. Thg1 adds deoxynucleotides and ribonucleotides to the 5&#11032; end of <br /> residues to p-tRNAHis species that do not begin with G&#11001;1. 5&#11032;-32P-labeled            tRNAHis. (A) Comparison of Thg1 activity in the presence of GTP or dGTP. Thg1 <br /> wild-type or variant tRNAHis and tRNAPhe species, as indicated below each           was assayed with 5&#11032;-32P-labeled wild-type or C73CCA tRNAHis and the standard <br /> panel, were assayed in the presence of GTP and ATP. GpGpC is the reaction           RNase A&#20862;CIP assay in the presence of either GTP or dGTP as indicated. (B) <br /> product observed with wild-type tRNAHis; GpGpGpC is the tetramer RNase A            Verification of deoxyguanosine nucleotide incorporation into tRNA. (a) Sam- <br /> reaction product expected for G&#11001;2 control RNA, which has an altered G2&#18528;C71          ples from A were resolved by TLC on PEI-cellulose in 0.5 M sodium formate (pH <br /> base pair.                                                                          3.5), where dG-containing reaction products migrate differently from the <br />                                                                                     corresponding ribonucleotide-containing products. (b) Samples from A were <br />                                                                                     treated with RNase T1 and CIP and resolved in the same system. <br /> the first nucleotide addition restores the G&#11001;1 5&#11032; end, and the <br /> second adds G&#11002;1 opposite A73, yielding the same G&#11002;1-containing <br /> product as is observed with a control tRNAHis transcript that                       incorporation are resistant to RNase T1 treatment (Fig. 6Bb). <br /> starts at G&#11001;1 (Fig. 5C). However, the addition of a nucleotide                      Thus, dGTP is incorporated and not contaminating GTP. <br /> initiating at either G&#11002;1 or at G&#11001;2 or to the tRNAPhe C73CCA                         Discussion <br /> variant was not detected when the corresponding p-tRNA <br /> species were assayed for activity with 5&#11032;-end-labeled p-tRNA; in                    We have demonstrated that wild-type Thg1 has an unexpected <br /> this assay, the only p-tRNA species that was a substrate for Thg1                   second catalytic activity in addition to its known role of adding <br /> was the wild-type control p-tRNAHis bearing the altered 2&#18528;71                        G&#11002;1 to the 5&#11032; end of tRNAHis (7, 10): Thg1 can catalyze a reverse <br /> base pair (Fig. 5D). The lack of reaction with all other p-tRNA                     (3&#11032;-5&#11032;) polymerase activity, adding multiple nucleotides to the 5&#11032; <br /> variants is consistent with the high selectivity of Thg1 for mature                 end of variant tRNA species in a template-dependent reaction. <br /> p-tRNAHis.                                                                             Like the normal G&#11002;1 addition reaction, reverse polymeriza- <br />                                                                                     tion requires ATP with p-tRNAHis variants that begin at &#11001;1. <br /> Thg1 Efficiently Incorporates dGTP in both the G&#11546;1 Addition and                     Reverse polymerization with all other substrate tRNAs requires <br /> Reverse Polymerization Reactions. To explore further the general-                   ppp-tRNA and can occur in the absence of ATP. Presumably, as <br /> ity of the Thg1 reaction, we investigated Thg1 activity with                        with the normal G&#11002;1 addition, these triphosphorylated sub- <br /> deoxynucleotides. Surprisingly, Thg1 appears to incorporate                         strates bypass the ATP requirement because they are already <br /> dGTP with efficiency comparable with that of GTP, as indicated                      activated for the first nucleotide addition, and this and subse- <br /> by the similar amounts of addition products observed with both                      quent nucleotide addition steps are powered by pyrophosphate <br /> nucleotides, using 5&#11032;-end-labeled wild-type p-tRNAHis and p-                        release of the &#9252;- and &#9253;-phosphates from the triphosphorylated <br /> tRNAHis C73CCA substrates (Fig. 6A). Although the deoxy- and                        nucleotide at the 5&#11032; end of the growing polynucleotide chain <br /> ribonucleotide reaction products are not resolved from one                          (Fig. 1B). In support of this mechanism for reverse polymeriza- <br /> another in this TLC system (Fig. 6A), they are easily resolved                      tion, a substrate tRNA that begins at &#11002;1 and therefore emulates <br /> with another TLC assay system to analyze the same reaction                          the product of the first addition reaction is active only if it is <br /> products (Fig. 6Ba), and as expected, the products of dGTP                          triphosphorylated (Fig. 5B). This mechanism of reverse poly- <br />  <br /> Jackman and Phizicky                                                                                        PNAS &#20841; June 6, 2006 &#20841; vol. 103 &#20841; no. 23 &#20841; 8643 <br />  merization is very similar to the pyrophosphate release that           tRNAHis and tRNAPhe C73CCA substrates at least as efficiently <br /> powers conventional 5&#11032;-3&#11032; polymerization, although in 5&#11032;-3&#11032;            as it catalyzes its known physiological G&#11002;1 addition reaction on <br /> polymerization the pyrophosphate originates from the incoming          wild-type tRNAHis (Figs. 2B, 3A, and 5A) and that both of these <br /> NTP instead of the polynucleotide (Fig. 1B).                           reactions occur on the same time scale (data not shown). Thus, <br />    Under normal circumstances in the cell, tRNAs other than            it is tempting to speculate that the reverse polymerase activity <br /> tRNAHis are presumably not substrates for reverse polymeriza-          has a function. It is possible that the reverse polymerization <br /> tion or G&#11002;1 addition by Thg1 because there is no ready source          activity of Thg1 might simply be a remnant of an earlier 3&#11032;-5&#11032; <br /> of activated 5&#11032; ends of the tRNAs. There is no naturally occurring     polymerase activity of this family of proteins, which could have <br /> mature ppp-tRNA that would be a substrate for Thg1, and                competed with or acted together with the activities of conven- <br /> activation by adenylylation occurs only with 75-mer p-tRNAHis          tional 5&#11032;-3&#11032; polymerases for replication or repair functions. <br /> through Thg1 recognition of the anticodon (14), consistent with        However, it is particularly difficult to reconcile the evolutionary <br /> the crucial role of G&#11002;1-containing tRNAHis in charging tRNAHis         retention of a distinct Thg1 reverse polymerization activity that <br /> in vitro (11&#8211;13) and in vivo (10). Furthermore, tRNAHis nucle-         forms G&#18528;C and C&#18528;G base pairs because such Watson&#8211;Crick base <br /> otide addition is restricted to a single guanosine residue because     pair recognition and use would not be expected of an enzyme <br /> Thg1 adds only the single required G&#11002;1 opposite the discrimi-          whose only role is to add G&#11002;1 opposite A73 of tRNAHis in vivo. <br /> nator A73 (Figs. 2B and 3A). This feature emphasizes the               Rather, the reverse polymerization described here may reflect an <br /> importance of the conserved A73 in eukaryotic tRNAHis because          additional Thg1 activity that is used in the cell. Indeed, there is <br /> Thg1 could add additional residues opposite the conserved              an apparent 3&#11032;-5&#11032; polymerization activity that functions to edit <br /> prokaryotic discriminator, C73.                                        the 5&#11032; end of mitochondrial tRNA in <span id='am-92' about='obo:NCBITaxon_5754' typeof='owl:Thing obo:NCBITaxon_555280 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_555407 obo:NCBITaxon_2759 obo:NCBITaxon_33677 obo:NCBITaxon_1485168 obo:NCBITaxon_554915'><span id='am-93' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="Acanthamoeba" datatype="xsd:string"></span><span id='am-95'  ></span><span id='am-96' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Acanthamoeba</span> castellanii <br />    The incorporation of G&#11002;1 into tRNAHis in archaeal species is        and <span id='am-70' about='obo:NCBITaxon_4815' typeof='obo:NCBITaxon_33154 obo:NCBITaxon_4761 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_34479 obo:NCBITaxon_34478 obo:NCBITaxon_2759 obo:NCBITaxon_451435'><span id='am-71' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="Spizellomyces" datatype="xsd:string"></span><span id='am-74'  ></span>Spizellomyces</span> punctatus although, unlike Thg1, this activity <br /> more complicated. Despite the nearly universal conservation of         does not proceed beyond the normal 5&#11032; end of the tRNA <br /> C73 in <span id='am-7' about='obo:NCBITaxon_2157' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567'><span id='am-8' property="oboInOwl:hasDbXref" content="PMID:11760965" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="PMID:11542064" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="PMID:10490293" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="PMID:11542149" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="PMID:10843050" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Monera" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="PMID:10939673" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="PMID:270744" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="prokaryotes" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="PMID:8123559" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="PMID:11411719" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Archaea" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Metabacteria" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="prokaryote" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="PMID:10939677" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="PMID:12054223" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="PMID:11211268" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="PMID:9103655" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Mendosicutes" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="PMID:11321083" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="PMID:8590690" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="PMID:10425795" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="PMID:10939651" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Prokaryota" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="PMID:10425797" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="PMID:11541975" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="PMID:11321113" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="PMID:10425796" datatype="xsd:string"></span><span id='am-38'  ></span><span id='am-39' property="oboInOwl:hasDbXref" content="PMID:11540071" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasRelatedSynonym" content="Archaebacteria" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="archaea" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Procaryotae" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="PMID:9336922" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Prokaryotae" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="PMID:2112744" datatype="xsd:string"></span>Archaea</span>, there are at least some species that appear to         (19&#8211;21). Because the gene product(s) that catalyze this reaction <br /> require posttranscriptional G&#11002;1 addition opposite C73 because          are unknown, the relationship between these activities is unclear. <br /> their tRNAHis genes lack a genomically encoded G&#11002;1, and the            Thg1 might act in a similar or related capacity in 3&#11032;-5&#11032; polymer- <br /> corresponding organisms have a predicted Thg1 ortholog. Thus,          ization of tRNA substrates, although this role would depend on <br /> archaebacterial Thg1 homologs may possess additional mecha-            some additional mechanism to activate the 5&#11032; ends of the tRNA <br /> nisms to ensure addition of only a single guanine residue across       species. <br /> from C73. Consistent with this possibility, the archaeal THG1             The ability of Thg1 to use deoxynucleotides may also have <br /> orthologs are much less conserved than eukaryotic orthologs,           implications for Thg1 function. Thg1 incorporates deox- <br /> and, although both human and <span id='am-78' about='obo:NCBITaxon_5475' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_118267 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-79'  ></span><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Candida" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasRelatedSynonym" content="Torulopsis" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Candida</span> albicans THG1 or-                 yguanosine with an efficiency similar to its ribonucleotide coun- <br /> thologs can complement a thg1 conditional yeast strain, the            terpart for both reverse polymerization with tRNAHis C73CCA <br /> archaeal ortholog from <span id='am-1' about='obo:NCBITaxon_2160' typeof='owl:Thing obo:NCBITaxon_2158 obo:NCBITaxon_2159 obo:NCBITaxon_1 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_183925 obo:NCBITaxon_28890'><span id='am-2'  ></span><span id='am-3' property="rdfs:label" content="Methanobacterium" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="PMID:15653928" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Methanobacterium</span> thermoautotrophicum            and the addition of G&#11002;1 with wild-type tRNAHis (Fig. 6). This <br /> does not complement a thg1 mutant under the same expression            property of Thg1 is in contrast to the highly selective preferences <br /> conditions (data not shown).                                           exhibited by most 5&#11032;-3&#11032; RNA and DNA polymerases for their <br />    The results presented here, coupled with previous results, illus-   nucleotide substrates (22). However, several members of the <br /> trate a bewildering array of nucleotide recognition and use by Thg1    polymerase X family of DNA damage repair polymerases also <br /> protein. During the addition of G&#11002;1, Thg1 recognizes the 5&#11032; end of     exhibit a lack of discrimination toward nucleotide sugar species <br /> ATP for adenylylation, the 3&#11032; end of incoming NTPs for nucleotidyl     in vitro, which may be important for their proposed functions in <br /> transfer, and the 5&#11032;-pyrophosphate of the product tRNA for             vivo, including polymerization at points in the cell cycle when <br /> pyrophosphatase activity (Fig. 1). In addition, reverse polymeriza-    cellular dNTP levels are low and signaling the site of repair by <br /> tion of ppp-tRNAHis substrates by Thg1 at positions other than &#11002;1      marking the chromosome with incorporated ribonucleotides (23, <br /> requires formation of a G&#18528;C or C&#18528;G base pair and the potential to      24). Thus, the ability of Thg1 to use deoxynucleotides suggests <br /> form similar base pairs at subsequent addition sites, whereas any      that any additional functions of Thg1 might employ either <br /> nucleotide can be added at position &#11002;1 whether or not a base pair      dNTPs or rNTPs. In this regard, Thg1 has recently been shown <br /> can be formed (Fig. 4). Thus, Thg1 has the potential both to           to associate with the origin recognition complex and to be <br /> recognize and select for nucleotides that participate in Watson&#8211;       implicated in the G2&#20862;M transition in yeast (25), suggesting a <br /> Crick base pairing for polymerization and to recognize and select      potential role at this site. <br /> guanosine in a non-Watson&#8211;Crick interaction opposite A73. Al- <br /> though Thg1 base pair recognition is limited to G&#18528;C but apparently     Materials and Methods <br /> not U&#18528;A base pairs (Fig. 4), it is not limited by the normal tRNA 3&#11032;   Sources of Thg1. Thg1 protein was purified from both S. cerevisiae <br /> end in the number of nucleotides that can be added because a tRNA      and E. coli as described in ref. 14. Unless otherwise indicated, the <br /> with five C residues at the 3&#11032; end (tRNAHis C73CCCCA) can be           Thg1 protein used for all assays was the yeast purified protein. <br /> extended by up to five nucleotides (Fig. 3C). The multiple modes of <br /> nucleotide selection by the relatively small (28 kDa) Thg1 suggest     Variant tRNA Species. All tRNA variants with the exception of <br /> conformational changes during the course of the reaction, and they     C73CCCCA tRNAHis were cloned into a previously described <br /> are reminiscent of the multiple use of different nucleotides by the    pUC13-based plasmid under control of the T7 RNA polymerase <br /> CCA-adding enzyme, which restructures its single active site for       promoter for purposes of in vitro transcription (7). tRNAHis and <br /> accommodation of each new nucleotide substrate during nontem-          tRNAPhe variants were all created by mutagenic PCR, with the <br /> plated addition to the 3&#11032; end of the tRNA (16&#8211;18).                     T7 RNA polymerase promoter encoded by the 5&#11032; PCR primer <br />    We emphasize that the reverse polymerase activity described         and subsequent ligation of the amplified PCR product into the <br /> here is distinct from the addition of G&#11002;1 to tRNAHis that occurs       PstI&#20862;BamHI sites of the previously described EMP835 (7). All <br /> in vivo during tRNAHis maturation because the addition of G&#11002;1          tRNA-containing <span id='am-146' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-147' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-148' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-149' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>plasmids</span> were verified by sequencing. The <br /> is not dependent on base pair formation, does not require prior        tRNAHis C73CCCCA RNA substrate was made by ligation of a <br /> activation of the tRNAHis 5&#11032; end, and is absolutely specific for       phosphorylated synthetic RNA oligonucleotide (5&#11032;-ggagauggc- <br /> p-tRNAHis that arises after cleavage of pre-tRNA by RNase P.           CCCCCA) to the 3&#11032; end of a truncated tRNAHis transcript, <br /> By contrast, reverse polymerization depends on base pair for-          ending at A63, using 10 units&#20862;&#9262;l T4 DNA ligase (USB Corp.) in <br /> mation and is not specific for the correct 5&#11032; end of tRNAHis. We       the presence of a complementary DNA oligonucleotide span- <br /> also emphasize that Thg1 catalyzes reverse polymerization on           ning the ligation site. <br />  <br /> 8644 &#20841; www.pnas.org&#20862;cgi&#20862;doi&#20862;10.1073&#20862;pnas.0603068103                                                                      Jackman and Phizicky <br />  Thg1 Activity Assay with [5&#11541;-32P]tRNA. [5&#11032;-32P]tRNA species were                      Determination of 5&#11541; Nucleotide Identity by RNase T2 Digestion. 5&#11032; <br />                                                                                       32P-labeled <br /> prepared by phosphatase treatment of in vitro transcripts fol-                                    wild-type tRNAHis was used in Thg1 reactions in the <br /> lowed by labeling with [&#9253;-32P]ATP using T4 polynucleotide                             buffer described above with varied concentrations of all four <br /> kinase (Roche Applied Biosciences) to a final specific activity of                    NTPs (1 mM, 0.4 mM, or 0.1 mM each) and 1 &#9262;M purified Thg1 <br /> &#11015;7,000 Ci&#20862;mmol (1 Ci &#11005; 37 GBq). The [5&#11032;-32P]tRNA was used                             in a 10-&#9262;l reaction mixture for 6 h at room temperature. After <br /> as the substrate in a Thg1 assay containing 10 mM MgCl2, 3 mM                         6 h, a portion (4 &#9262;l) of the reaction was stopped, digested, and <br /> DTT, 125 mM NaCl, 0.2 mg&#20862;ml BSA, and, unless otherwise                                analyzed as described above with EDTA and RNase A. Another <br /> indicated, 1 mM each GTP and ATP. Enzyme titration assays                             4-&#9262;l aliquot was removed to a new tube containing 0.5 unit of <br /> consisted of 5-fold serial dilutions (&#11015;50 &#9262;g&#20862;ml to 0.4 &#9262;g&#20862;ml) of                      CIP in 1&#11003; alkaline phosphatase buffer in a final volume of 10 &#9262;l <br /> purified Thg1. Reactions were started by the addition of enzyme                       to remove the background from the remaining unreacted sub- <br /> and carried out at room temperature. After 5&#8211;6 h, the reaction                        strate tRNA. After 30 min at 37&#176;C, the reaction products were <br /> was stopped by removal of an aliquot (4 &#9262;l) to a new tube                             purified by extraction with phenol&#20862;chloroform&#20862;isoamyl alcohol <br /> containing 0.5 &#9262;l of 0.5 M EDTA and 0.5 &#9262;l of 10 mg&#20862;ml RNase                          and precipitation with ethanol. The resulting tRNA was resus- <br /> A followed by digestion for 10&#8211;30 min at 50&#176;C and treatment                           pended in a 10-&#9262;l reaction mixture containing 1 unit of RNase <br /> with 0.5 unit of CIP in 1&#11003; phosphatase reaction buffer in a final                     T2 and 0.5 &#9262;g of RNase A in 20 mM sodium acetate (pH 5.2)&#20862;1 <br /> volume of 6 &#9262;l at 37&#176;C for 30 min. Reactions were spotted to                          mM EDTA and digested for 30 min at 50&#176;C. Two microliters of <br /> silica TLC plates (EM Science) and resolved in an n-propyl                            each digestion reaction mixture was spotted to PEI-cellulose <br /> alcohol&#20862;NH4OH&#20862;H2O (55:35:10) solvent system. The plates                               plates and developed in a 0.5 M sodium formate (pH 3.5) solvent <br /> were visualized and quantified with a PhosphorImager and                              system. The positions of unlabeled nucleotide standards were <br /> IMAGEQUANT software.                                                                  determined by spotting 2 &#9262;l of 10 mM solutions of GMP, UMP, <br />                                                                                       AMP, or CMP and visualizing the resultant spots by fluores- <br /> 5&#11541; End Analysis by Primer Extension. Thg1 reactions were carried                      cence quenching. <br /> out for 6 h at room temperature in the same buffer described <br /> above with 1 &#9262;M unlabeled triphosphorylated tRNA in a 10-&#9262;l <br />  <br />  <br />  <br />  <br />                                                                                                                                                                              BIOCHEMISTRY <br />                                                                                       Determination of Deoxyguanosine Incorporation by Resistance to <br /> reaction volume containing 1 &#9262;M purified Thg1. Unless other-                          RNase T1 Digestion. Thg1 reactions were carried out as described <br /> wise noted, each reaction contained a 1 mM concentration of the                       above with 5&#11032;-32P-labeled tRNAHis (wild-type and C73CCA <br /> indicated NTP. Thg1-treated tRNAs were purified by extraction                         species) in the presence of 0.1 mM ATP and either 1 mM GTP <br /> with phenol&#20862;chloroform&#20862;isoamyl alcohol, concentrated by pre-                          or 1 mM dGTP for 5 h at room temperature. After digestion with <br /> cipitation with ethanol, and resuspended in 10 &#9262;l of buffer                           RNase A and CIP, as described above, a 2-&#9262;l portion of the <br /> containing 10 mM Tris&#18528;HCl (pH 7.5) and 1 mM EDTA. Two                                 reaction mixture was removed to a new tube containing 2 &#9262;g of <br /> microliters of each purified tRNA was used as the template in                         <span id='am-135' about='obo:NCBITaxon_430997' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_1 obo:NCBITaxon_171187 obo:NCBITaxon_4890 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_716545 obo:NCBITaxon_2759'><span id='am-136' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-137' property="rdfs:label" content="Torula" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-139'  ></span><span id='am-140' property="oboInOwl:hasRelatedSynonym" content="Torula Pers." datatype="xsd:string"></span>torula</span> yeast tRNA and 0.5 unit of RNase T1 in 30 mM sodium <br /> primer extension reactions employing a 5&#11032;-end-labeled primer                          acetate (pH 5.2) buffer in a final volume of 5 &#9262;l. The RNase T1 <br /> specific for tRNAHis (5&#11032;-ACTAACCACTATACTAAGA-3&#11032;) as                                   digestion was allowed to proceed for 30 min at 50&#176;C, and then the <br /> shown in Fig. 2 A or a similar tRNAPhe-specific primer (5&#11032;-                           reaction mixtures were treated with 0.5 unit of CIP in 1&#11003; alkaline <br /> TGGCGCTCTCCCAACTG-3&#11032;). After the Thg1-treated tRNA                                    phosphatase reaction buffer in a final volume of 6 &#9262;l. After 30 <br /> template was annealed to 1 pmol of primer in a final volume of                        min at 37&#176;C, 2 &#9262;l of each reaction mixture (both the original <br /> 5 &#9262;l, extension reactions were carried out with 8 units of avian                      RNase A-treated and RNase A- &#11001; RNase T1-treated reactions) <br /> myeloblastosis virus reverse transcriptase (AMV-RT) (20 units&#20862;                        were spotted to PEI-cellulose TLC plates and resolved in the 0.5 <br /> &#9262;l; Promega) in the presence of a 0.4 mM concentration of each                        M sodium formate (pH 3.5) solvent system. <br /> dNTP in 1&#11003; AMV-RT reaction buffer (Promega) in a final <br /> volume of 10 &#9262;l at 37&#176;C for 1&#8211;2 h. The resulting products were                    <br /> resolved on 10% polyacrylamide&#20862;4 M urea gels, which were                            <br /> dried and visualized with the PhosphorImager.                                    <br />  <br /> </body></html>