<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />  <br /> Pyruvate Carboxylase Is an Essential Protein in the <br /> Assembly of Yeast Peroxisomal Oligomeric Alcohol <br /> Oxidase <br />  <br />  <br />  <br />  <br />         Hansenula polymorpha ass3 mutants are characterized by the accumulation of inactive alcohol <br />         oxidase (AO) monomers in the cytosol, whereas other peroxisomal matrix proteins are <br />         normally activated and sorted to peroxisomes. These mutants also have a glutamate or <br />         aspartate requirement on minimal media. Cloning of the corresponding gene resulted in the <br />         isolation of the H. polymorpha PYC gene that encodes pyruvate carboxylase (HpPyc1p). <br />         HpPyc1p is a cytosolic, anapleurotic enzyme that replenishes the tricarboxylic acid cycle with <br />         oxaloacetate. The absence of <span id='am-54' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-58'  ></span>this</span> enzyme can be compensated by addition of aspartate or <br />         glutamate to the growth media. We show that HpPyc1p protein but not the enzyme activity <br />         is essential for import and assembly of AO. Similar results were obtained in the related yeast <br />         <span id='am-40' about='obo:NCBITaxon_4919' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_1156497 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-42'  ></span><span id='am-43' property="rdfs:label" content="Pichia" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Hansenula" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasAlternativeId" content="NCBITaxon:34359" datatype="xsd:string"></span>Pichia</span> pastoris. In vitro studies revealed that HpPyc1p has affinity for FAD and is capable to <br />         physically interact with AO protein. These <span id='am-110' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-111'  ></span><span id='am-112' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-113' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span> suggest that in methylotrophic yeast pyruvate <br />         carboxylase plays a dual role in that, besides its well-characterized metabolic function as <br />         anapleurotic enzyme, the protein fulfils a specific role in the AO sorting and assembly process, <br />         possibly by mediating FAD-binding to AO monomers. <br />  <br />  <br /> INTRODUCTION                                                             Both in vivo and in vitro experiments suggested that <br />                                                                       assembly of AO into active octamers is most likely not a <br /> The yeast Hansenula polymorpha is able to use methanol as             spontaneous process (Distel et al., 1987; van der Klei et al., <br /> sole carbon and energy source. Growth on this compound is             1989b). Several independent experiments suggested that <br /> accompanied by the induction of peroxisomes that contain              specific helper proteins (tentatively called assembly fac- <br /> the key enzymes of methanol metabolism. Alcohol oxidase               tors) are required to mediate AO assembly. Studies on an <br /> (AO) is a major constituent of these organelles and catalyzes         H. polymorpha riboflavin (Rf) auxotrophic mutant revealed <br /> the oxidation of methanol into formaldehyde and hydrogen              that Rf limitation interfered with the assembly and the <br /> peroxide. Inactive AO monomers are synthesized in the                 import of AO (Evers et al., 1994) and suggested that <br /> cytosol and posttranslationally imported into the target or-          cofactor binding, oligomerization, and translocation of <br /> ganelle, where the protein is activated. The active enzyme is         AO are tightly coupled processes. However, in all H. <br /> an octamer, containing eight identical subunits, which each           polymorpha peroxisome-deficient (pex) mutants analyzed <br /> contains a FAD molecule as cofactor (reviewed by van der              so far AO is normally assembled and active in the cytosol. <br /> Klei et al., 1991a).                                                  This suggests that AO assembly does not require the <br />                                                                       specific (acidic) microenvironment of the peroxisomal ma- <br />                                                                       trix (van der Klei et al., 1991c). <br />   Article published online ahead of print. Mol. Biol. Cell 10.1091/      Previous biochemical approaches to identify AO assembly <br /> mbc.E02&#8211; 07&#8211; 0417. Article and publication date are at www.molbi-     factors failed so far. We therefore sought to isolate these <br /> olcell.org/cgi/doi/10.1091/mbc.E02&#8211; 07&#8211; 0417.                         components by a genetic approach. To this end we have <br />   &#167; <br />     These authors contributed equally to this paper. <br />   &#182; <br />     Corresponding author. E-mail address: M.Veenhuis@biol.rug.nl.     isolated a collection of mutants that displayed reduced AO <br />     Abbreviations used: AO, alcohol oxidase; BSA, bovine serum        activities (van Dijk et al., 2002). <br />     albumin, CAT, catalase; DHAS, dihydroxyacetone synthase;             Here, we report the functional complementation of one of <br />     FCS, fluorescence correlation spectroscopy.                       these mutants. We show that the protein product of the <br />  <br />  <br /> 786                                                                                   &#169; 2003 by The American Society for Cell Biology <br />                                                                                                                    Alcohol Oxidase Assembly <br />  <br />  <br /> complementing gene, pyruvate carboxylase, has a dual func-              (blunt ended)&#8211;PstI fragment and ligated into pBKS-PYC (StyI [blunt <br /> tion in that the protein, but not the enzyme activity, is crucial       ended]&#8211;PstI fragment). From this plasmid (pBKS-&#9004;PYC::URA3) a <br /> for sorting and subsequent assembly of AO protein in per-               3.1-kb BglII fragment was isolated and used to transform H. poly- <br /> oxisomes of H. polymorpha.                                              morpha NCYC 495 (leu1.1 ura3) or pex3::LEU2 (ura3 &#9004;pex3). Trans- <br />                                                                         formants were selected for uracil prototrophy and inability to grow <br />                                                                         on minimal methanol media. Correct integration was confirmed by <br /> MATERIALS AND METHODS                                                   Southern blotting. <br />                                                                            A mutated HpPYC1 gene (R316Q), in which the <span id='am-59' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-61'  ></span><span id='am-62' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span> encoding <br /> Organisms and Growth Conditions                                         residue 316 (arginine) was substituted by glutamine, was con- <br />                                                                         structed by overlap extension PCR (Horton et al., 1990) using primer <br /> <span id='am-67' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-68' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-71'  ></span><span id='am-72' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span>Escherichia coli</span> strains DH5&#9251; and C600 were cultivated as described <br />                                                                         PYC-1 (5&#11032;GACATTATTTCATCGAAATTAATCCTCAGATCCAGGTC <br /> (Sambrook et al., 1989). The H. polymorpha strains used in this study <br />                                                                         GAGCACACC3&#11032;) and both pBKS-40 universal (MF) and -50 reverse <br /> are NCYC 495 (leu1.1), NCYC 495 (leu1.1 ura3; Gleeson and Sudbery, <br />                                                                         primers (MR). From the PCR product a 0.3-kb PstI/NarI fragment <br /> 1988) and mutants derived from these strains: ass3-110.leu 1.1 (van <br />                                                                         was exchanged with the same fragment in pBKS-PYC, resulting in <br /> Dijk et al., 2002), &#9004;pex3 ura3 (Baerends et al., 1996), &#9004;pyc1 leu1.1, <br />                                                                         pBKS-PYCR316Q. For introduction into H. polymorpha NCYC 495 <br /> &#9004;pex3 &#9004;pyc1, and &#9004;pyc1.PAMOPYC. <br />                                                                         (leu1.1 &#9004;pyc1) plasmid pYT3-PYCR316Q was constructed by ligation <br />    H. polymorpha cells were grown on minimal media containing <br />                                                                         of the full-length mutated HpPYC1 gene (XbaI/EcoRI blunt ended) <br /> 0.67% (wt/vol) Yeast Nitrogen Base without amino acids (Difco, <br />                                                                         from the pBKS plasmid into pYT3 (XbaI/BamHI blunt ended; Tan et <br /> Sparks, MD) containing 1% glucose (YND) or 0.5% methanol <br />                                                                         al., 1995). <br /> (YNM); on YPD containing 1% yeast extract, 1% peptone, and 1% <br />                                                                            Plasmid pHIPX5 carrying HpPYC1 under control of AMO pro- <br /> glucose or mineral medium (van Dijken et al., 1976) supplemented <br />                                                                         moter (PAMO) was constructed as follows: PYC1, amplified by PCR <br /> with 0.5% carbon source and 0.25% nitrogen source. For the induc- <br />                                                                         using primers &#8220;PYC-ATG&#8221; (CTTCCATGGCCCAGGTCG) and <br /> tion of peroxisomes mutant strains were precultivated in YPD me- <br />                                                                         &#8220;PYC-STOP&#8221; (CCGCATGCGCAGAGCGAGACGC), was digested <br /> dium and shifted to methanol-containing mineral medium for 16 h. <br />                                                                         by NcoI and SphI and cloned into pHIPX5 digested with the same <br /> To accumulate monomeric AO in the cytosol of &#9004;pyc1::PAMOPYC, <br />                                                                         restriction enzymes. The resulting plasmid was linearized with <br /> cells were grown for 6 h on media containing 0.1% glycerol/0.5% <br />                                                                         BsiWI and transformed into H. polymorpha NCYC 495 &#9004;pyc1(leu1.1). <br /> methanol/0.25% ammonium sulfate. Subsequently, the cells were <br />                                                                         A strain in which a single copy of the expression construct was <br /> incubated for 30 min in media without carbon or nitrogen sources <br />                                                                         integrated was selected, based on Southern blot analysis. <br /> (Waterham et al., 1993) followed by transfer to mineral media con- <br /> taining 0.5% glucose and 0.25% ethylamine. <br />    Pichia pastoris wild-type MP 36 (his3)and MP 36-&#9004;pyc(pyc::his3)      Isolation of His6-tagged HpPyc1p <br /> (Menendez et al., 1998) were cultivated as described by Faber et al. <br /> (1998).                                                                 For isolation of C-terminal His6-tagged HpPyc1p, PYC1 was ampli- <br />    When needed uracil (20 mg/l), leucine (20 mg/l), histidine (40       fied by PCR, using primers &#8220;PYC-ATG&#8221; and &#8220;PYC-STOP,&#8221; digested <br /> mg/l), aspartate (60 mg/l), or glutamate (60 mg/l) were added to        with NcoI/PvuII and inserted into NcoI/BglII (blunt ended)&#8211; di- <br /> the media.                                                              gested pQE-60 (Qiagen GmbH, Hilden, Germany), resulting in plas- <br />                                                                         mid pQE60-PYC. E. coli Sq13009[pREP4] containing pQE60-PYC <br />                                                                         was grown as detailed in The QIAexpressionist. Cell pellets were <br /> Isolation and Characterization of the Pyruvate                          suspended in a 50 mM potassium phosphate buffer, pH 7.4, con- <br /> Carboxylase (PYC1) Gene                                                 taining 300 mM NaCl; 0.2 mM b-ME, 10% glycerol, 0.2 mM EDTA, <br /> Genetic manipulations of H. polymorpha were performed as de-            and Complete (Roche). Cells were disrupted using a French press. <br /> scribed previously (Gleeson and Sudbery, 1988; Faber et al., 1992;      Cell debris and other insoluble material were removed by centrif- <br /> Titorenko et al., 1993; Faber et al., 1994) Standard recombinant DNA    ugation. The cell extract was loaded onto a 1 ml Ni2&#11001; containing <br /> techniques were carried out essentially as described (Sambrook et.      HiTrap Chelating column (Amersham Pharmacia Biotech AB, Upp- <br /> al, 1989). Endonuclease restriction enzymes and biochemicals were       sala, Sweden), washed with 10 ml buffer containing 75 mM imida- <br /> obtained from Roche (Almere, the Netherlands) and used as de-           zole and 150 mM NaCl in 50 mM potassium phosphate, pH 7.4, and <br /> tailed by the manufacturer. To clone the complementing genomic          eluted with the same buffer containing 500 mM imidazole. The peak <br /> fragment, mutant ass3-110 was transformed with an H. polymorpha         fractions contained highly purified HpPyc1p as determined by SDS- <br /> genomic library (Tan et al., 1995). Leucine prototrophs were tested     PAGE and Coomassie brilliant blue staining (unpublished data). <br /> on YNM plates for the ability to grow on methanol (Mut&#11001;). Mut&#11001; <br /> transformants were selected. Their plasmid content was isolated <br /> and reintroduced in mutant ass3-110. Four <span id='am-115' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-116' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-117' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>plasmids</span> that comple- <br />                                                                         Biochemical Methods <br /> mented ass3-110 again were selected for further analysis. These         Crude extracts were prepared according to van der Klei et al.(1991b). <br /> plasmids contained overlapping genomic fragments ranging in size        AO activity was measured as described by Verduyn et al. (1984). AO <br /> from 6.5 to 9.0 kb. A 4.2-kb complementing DNA fragment was             monomers and octamers were separated by sucrose density centrif- <br /> subcloned as an EcoRI-XbaI fragment into pBluescript II KS&#11001; (pBSII      ugation (Goodman et al., 1984). Protein concentrations were deter- <br /> KS&#11001;; Stratagene Inc., San Diego, CA). Sequencing of both strands        mined using the Bio-Rad protein assay kit (Bio-Rad Gmbh, Munich, <br /> was carried out on a LiCor automated DNA-sequencer using dye-           Germany) using bovine serum albumin (BSA) as standard. The FAD <br /> primer chemistry (LI-COR Inc., Lincoln, NE). For DNA and amino          content of AO was determined in immunoprecipitates by fluores- <br /> acid sequence analysis, the PC-GENETM program (Release 6.70;            cence spectroscopy as detailed previously (van der Klei et al., 1989a). <br /> IntelliGenetics, Mountain View, CA) was used. The TBLASTN al-           SDS-PAGE (Laemmli, 1970) and Western blotting (Kyhse-Andersen, <br /> gorithm (Altschul et al., 1997) was used to search the databases at     1984) was performed as described. Blots were decorated using <br /> the National Center for Biotechnology Information (Bethesda, MD).       antisera against various H. polymorpha proteins or <span id='am-47' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-48' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-53'  ></span>Saccharomyces</span> <br /> The nucleotide sequence of H. polymorpha PYC1 (HpPYC1) was              cerevisiae pyruvate carboxylase (Rohde et al., 1991) and the chromo- <br /> deposited at GenBank and was assigned Accession No. AF 221670.          genic or chemiluminescent Western blotting kit (Boehringer Mann- <br />                                                                         heim BV, Almere, the Netherlands). Cell fractionation was per- <br />                                                                         formed as described by van der Klei et al. (1998). A postnuclear <br /> Construction of Mutants                                                 supernatant (10 ml in total) was loaded onto a discontinuous su- <br /> A PYC1 disruption strain was constructed as follows: the H. poly-       crose gradient (25 ml). After centrifugation 1.5-ml fractions were <br /> morpha URA3 gene (Merckelbach et al., 1993) was isolated as a BglII     taken from the bottom. <br />  <br />  <br /> Vol. 14, February 2003                                                                                                                     787 <br />  P. Ozimek et al. <br />  <br />  <br />  <br />  <br /> Figure 1. Morphology and immunocytochemistry of methanol-induced cells of WT H. polymorpha (Figure 1A) and ass3-110 cells (B and C). <br /> WT cells (A) are characterized by the presence of several large peroxisomes that harbor AO protein (&#9251;-AO antibodies). Only small <br /> peroxisomes are found in cells of the mutant strain (B; KMnO4-fixation). <span id='am-100' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-101' property="rdfs:label" content="Electron" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-104'  ></span>Electron</span> micrographs are taken of glutaraldehyde-fixed cells, <br /> poststained with uranylacetate unless otherwise indicated. M, mitochondrion; N, nucleus; P, peroxisome, V, vacuole, *AO protein aggregate. <br /> Bar, 0.5 &#9262;m. <br />  <br />  <br />  <br /> AO/HpPyc1p Binding Studies                                              proteins and gold-conjugated goat anti-rabbit (GAR) antibodies <br />                                                                         (Waterham et al., 1994). Cytochemical staining experiments for the <br /> AO and BSA columns were prepared as described by Evers et al.           detection and localization of AO activity were performed by the <br /> (1993). On binding the proteins were denatured by incubation for        CeCl3-based method (Veenhuis et al., 1976). <br /> 16 h at 4&#176;C in a buffer containing 8 M urea in 25 mM Tris-HCl, pH <br /> 7.0, followed by extensive washing with buffer A (25 mM Tris-HCl, <br /> 50 mM KCl, 1 mM DTT, pH 7.0) to remove the urea. For binding <br /> studies 100 &#9262;l purified HpPyc1p (50 &#9262;g/ml) was loaded onto 50-&#9262;l        RESULTS <br /> columns, followed by washing with 20 column volumes buffer A <br /> and elution with 20 column volumes of a solution containing 8 M <br />                                                                         The H. polymorpha Pyruvate Carboxylase Gene <br /> urea. Proteins were precipitated with TCA and analyzed by West-         Functionally Complements a Mutant Defective in <br /> ern blotting.                                                           AO Assembly <br />                                                                         In a genetic approach to identify proteins involved AO <br /> Fluorescence Correlation Spectroscopy                                   assembly/activation, we have isolated a collection of H. <br /> The fluorescence correlation spectroscopy (FCS) setup was basically <br />                                                                         polymorpha mutants that are impaired to utilize methanol as <br /> as described by Hink and Visser (1998). For excitation of FAD, <span id='am-95' about='obo:NCBITaxon_354775' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_40096 obo:NCBITaxon_40092 obo:NCBITaxon_33208 obo:NCBITaxon_40093 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-96' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="Argon" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-99'  ></span>argon</span>    sole carbon source (Mut&#11002; phenotype) because of strongly <br /> ion <span id='am-105' about='obo:NCBITaxon_62990' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_364270 obo:NCBITaxon_131567 obo:NCBITaxon_241799 obo:NCBITaxon_91882 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_241789 obo:NCBITaxon_4037 obo:NCBITaxon_241778 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4036 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-106'  ></span><span id='am-107' property="rdfs:label" content="Laser" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>laser</span> lines of 488 nm were used. The light intensity was adjusted   reduced or absent AO activities (Van Dijk et al., 2002). Four <br /> by using various neutral density filters. Measurements were made        mutants were characterized by normal AO protein levels but <br /> in a 96-well chamber in 50 mM potassium phosphate buffer, pH 7.0,       strongly reduced AO enzyme activities. Localization studies <br /> at room temperature. For the analysis of FAD binding to HpPyc1p,        revealed that the import of AO into peroxisomes was spe- <br /> purified HpPyc1p (200 or 400 nM) was incubated with FAD (100            cifically blocked in these mutants (Van Dijk et al., 2002). The <br /> nM) for 1 h at 37&#176;C before the measurements. The concentrations of      overall morphology of methanol-induced cells of a represen- <br /> FAD and pyruvate carboxylase were calculated based on their ex-         tative strain of these mutants, ass3-110, compared with WT <br /> tinction coefficients (FAD &#9280;450 &#11005; 11.3 mM&#11002;1 cm&#11002;1, HpPyc1p mono- <br /> mer &#9280;280 &#11005; 77 mM&#11002;1 cm&#11002;1). Diffusion constants of individual fluo- <br />                                                                         cells, is shown in Figure 1. <br /> rescent molecules were calculated from the time-dependent                  Growth experiments revealed that these four mutants also <br /> fluctuation of the fluorescent signal. Experimental autocorrelation     displayed, in addition to the Mut&#11002; phenotype, a severe <br /> curves were then fitted by theoretical autocorrelation functions        growth defect on minimal glucose-ammonium sulfate me- <br /> using the FCS Data Processor 1.3 software. In all series of experi-     dia, which could be restored by addition of aspartate or <br /> ments the alignment and focusing of the setup was frequently            glutamate. Addition of these amino acids, however, did not <br /> checked by measuring the autocorrelation function of 7.6 nM rho-        result in the complementation of the Mut&#11002; phenotype. The <br /> damin 110. The dimensions of the excitation volume were deter-          amino acid requirement (Asp&#11002;/Glu&#11002;) could not be sepa- <br /> mined by the known diffusion coefficient of rhodamin 110.               rated from the Mut&#11002; phenotype through backcrosses with <br />                                                                         parental strains, indicating that both phenotypes were <br /> Electron Microscopy                                                     closely linked. Complementation analysis revealed that all <br /> Whole cells were fixed and prepared for electron microscopy as          four mutants with this phenotype (Mut&#11002;, Asp&#11002;/Glu&#11002;) fell <br /> described (Waterham et al., 1994). Immunolabeling was performed         in one complementation group, designated ass3. <br /> on ultrathin sections of Unicryl-embedded cells, using specific poly-      To isolate the defective gene, strain ass3-110 was trans- <br /> clonal antibodies against various H. polymorpha and S. cerevisiae       formed with a genomic H. polymorpha library. Transformants <br />  <br />  <br /> 788                                                                                                         Molecular Biology of the Cell <br />                                                                                                           Alcohol Oxidase Assembly <br />  <br />  <br /> capable to grow on mineral media containing methanol <br /> (Mut&#11001;, Asp&#11001;/Glu&#11001;) were selected. Subcloning and reintro- <br /> duction of the complementing fragments into ass3-110 re- <br /> sulted in a 4.2-kb genomic fragment that contained the com- <br /> plementing activity. This fragment was sequenced and the <br /> sequence was deposited at GenBank (Accession No. AF <br /> 221670). <br />    Sequence analysis revealed that the complementing frag- <br /> ment contained a potential open reading frame (ORF) en- <br /> coding a protein of 1175 amino acids with a predicted MW <br /> of 130 kDa. A database search revealed that this protein was <br /> highly similar throughout the entire protein to pyruvate <br /> carboxylases (Pyc) from various organisms ranging from <br /> bacteria (e.g., <span id='am-1' about='obo:NCBITaxon_1423' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_653685'><span id='am-2' property="oboInOwl:hasAlternativeId" content="NCBITaxon:523730" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="Bacillus uniflagellatus" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042380" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis8" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232825" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232841" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232819" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232470" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232844" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Bacillus natto" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Vibrio subtilis" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232840" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429427" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042381" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232828" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis/Bacillus globigii" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Bacillus subtilis" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasAlternativeId" content="NCBITaxon:981081" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451270" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232824" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasAlternativeId" content="NCBITaxon:941949" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasAlternativeId" content="NCBITaxon:387338" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224247" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042382" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429428" datatype="xsd:string"></span><span id='am-29'  ></span><span id='am-30' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451272" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="PMID:11211269" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232822" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232830" datatype="xsd:string"></span>Bacillus subtilis</span> 50% identity) and yeast (P. <br /> pastoris, 81% identity; S. cerevisiae Pyc1p and Pyc2p, both <br /> 77% identity, to man (53% identity). On the basis of the <br /> results of this search, we designated the gene PYC1 and its <br /> translation product HpPyc1p. <br />    An H. polymorpha PYC1 disruption mutant (&#9004;pyc1) was <br /> constructed in which approximately half of the H. polymor- <br /> pha PYC1 gene was deleted (the region encoding amino <br /> acids 298 &#8211;905). Growth experiments indicated that cells of <br /> &#9004;pyc1 showed the same phenotype as the original mutant <br /> ass3-110: no growth on minimal media containing glucose <br /> and ammonium sulfate unless aspartate or glutamate were <br /> added and a defect in growth on methanol independent of <br /> the presence of aspartate or glutamate. <br />    Mating of the &#9004;pyc1 strain with the original ass3-110 mu- <br /> tant resulted in diploids that were all Mut&#11002;. After sporula- <br /> tion, no Mut&#11001; cells were observed, demonstrating that ass3- <br /> 110 and &#9004;pyc1 are closely linked and most likely are alleles <br /> of the same gene. <br />    Pyc is an anapleurotic enzyme that replenishes the tricar- <br /> boxylic acid (TCA) cycle with oxaloacetate from pyruvate. <br /> For S. cerevisiae it has been shown that the absence of Pyc <br />                                                                  Figure 2. Western blot analysis of crude extracts prepared from <br /> results in the inability of cells to grow on minimal media       methanol-induced H. polymorpha WT, &#9004;pyc1, and &#9004;pex3&#9004;pyc1 cells. <br /> containing glucose and ammonium sulfate, although they           The blots were decorated with antibodies against various H. poly- <br /> do grow on glucose-aspartate containing media (Stucka et         morpha proteins. HpPyc1p was detected using antibodies against S. <br /> al., 1991). Also P. pastoris mutants lacking Pyc were reported   cerevisiae Pycp that cross-react with the H. polymorpha protein. The <br /> to be unable to grow on glucose-ammonium sulfate media,          proteins are present in virtually equal amounts except that Pex5p <br /> whereas growth is possible with aspartate or glutamate as        levels are slightly enhanced in &#9004;pex3&#9004;pyc1cells. Equal amounts of <br /> nitrogen source.                                                 protein were loaded per lane. <br />    These data indicate that the Asp&#11002;/Glu&#11002; phenotype of H. <br /> polymorpha &#9004;pyc1 is due to the absence of Pyc enzyme activ- <br /> ity. However, it does not explain why H. polymorpha &#9004;pyc1        subjected to sucrose density gradient centrifugation in <br /> cells are unable to grow on methanol media that contain Asp      <span id='am-64' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-65' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> to separate AO monomers from octamers (Good- <br /> or Glu.                                                          man et al., 1984). Western blot analysis revealed that in <br />                                                                  gradients prepared from WT control cells almost all AO <br />                                                                  protein was found in the bottom fractions (Figure 3, frac- <br /> Properties of AO in H. polymorpha &#9004;pyc1 Cells                    tion 6) where octameric AO sediments. However, in gra- <br /> AO enzyme activity measurements in crude extracts pre-           dients prepared from methanol-induced &#9004;pyc1 cells AO <br /> pared from methanol-induced &#9004;pyc1 and WT control cells,          protein was found in the top fractions, indicative for a <br /> revealed that the AO activity of &#9004;pyc1 cells was &#11021;2% of the      monomeric state (Figure 3, fractions 2 and 3). In the <br /> activity found in WT cells (0.07 and 4.1 U/mg protein,           fractions where octameric AO sediments no AO protein <br /> respectively). Western blot analysis of these extracts re-       was detected. However, because the cells still display <br /> vealed, however, that the AO protein levels were normal in       some enzyme activity (&#11021;2% of the enzyme activity in WT <br /> &#9004;pyc1 cells (Figure 2). Also, the amounts of other peroxiso-     cells) and only octameric AO is active, the octameric AO <br /> mal matrix enzymes (dihydroxyacetone synthase [DHAS]             apparently is below the level of detection in the experi- <br /> and catalase [CAT]) and the peroxins Pex3p, Pex5p, and           ment shown in Figure 3. <br /> Pex14p were virtually identical in WT and &#9004;pyc1 cells (Fig-         Fluorescence analysis of the FAD content of AO protein, <br /> ure 2).                                                          immunoprecipitated from crude extracts of WT and &#9004;pyc1 <br />   To analyze whether the absence of AO activity was due          cells, revealed that in precipitates of equal amounts of AO <br /> to a defect in AO oligomerization, crude extracts were           protein from crude extracts prepared from &#9004;pyc1 or WT cells <br />  <br />  <br /> Vol. 14, February 2003                                                                                                           789 <br />  P. Ozimek et al. <br />  <br />  <br />  <br />  <br /> Figure 3. Analysis of the oligomerization state of AO protein in H. <br /> polymorpha WT, &#9004;pyc1, &#9004;pex3&#9004;pyc1, and &#9004;pex3 cells by sucrose den- <br /> sity gradient centrifugation of crude extracts prepared from meth- <br /> anol-induced cells. All fractions of the gradient were analyzed for <br /> the presence of AO protein by Western blotting. Fraction 1 repre- <br /> sents the top fraction, fraction 8 the bottom fraction. Monomeric AO <br /> sediments to fractions 2&#8211;3, octameric AO to fractions 6 &#8211;7. Equal <br /> portions of the fractions were loaded per lane. <br />  <br />  <br />  <br /> the concentration of FAD was &#11011;25-fold lower in precipitates <br /> from &#9004;pyc1 cells compared with WT controls. <br />    The overall morphology of methanol-induced H. poly- <br /> morpha &#9004;pyc1 cells was highly comparable to that of the <br /> original ass3-110 cells. Immunocytochemically, anti-AO&#8211; <br /> specific labeling was predominantly localized in the cy- <br /> tosol, with very little labeling on peroxisomes (Figure 4A).           Figure 4. Immunocytochemical demonstration of peroxisomal ma- <br /> However, the other major PTS1 proteins DHAS (Figure                    trix proteins in methanol-induced H. polymorpha &#9004;pyc1 cells. Using <br /> 4B) and CAT (Figure 4C) showed a normal peroxisomal                    anti-AO antisera labeling was predominantly observed in the cy- <br /> location. Cytochemical staining experiments revealed that              tosol, whereas only a minor portion of the protein was localized at <br /> in &#9004;pyc1 AO enzyme activity was, like in WT cells, con-                peroxisomes (A). Characteristically, the labeling intensities of the <br /> fined to peroxisomes (Figure 4D), although the amount of               peroxisomal profiles strongly varied. Specific anti-DHAS (B) or <br />                                                                        anticatalase (C) dependent labeling was confined to peroxisomes. <br /> activity fluctuated between individual organelles, re- <br />                                                                        Cytochemical staining for the detection of AO enzyme activity (D) <br /> flected by variations in staining intensity. Although the              revealed that the enzyme activity was invariably confined to per- <br /> level of AO activity is very low in &#9004;pyc1 cells, this can              oxisomes. Like AO protein, also the staining intensity varied (indi- <br /> easily be detected by cytochemical staining, because this              cated by arrows) among the peroxisomal population, present in one <br /> technique is extremely sensitive.                                      cells, indicative for variations in the levels of active AO protein. <br />    Taken together, these data indicate that in H. polymorpha <br /> &#9004;pyc1 cells bulk of the AO protein is in the cytosol in an <br /> inactive, FAD-lacking, monomeric form while a minor frac- <br /> tion is present as enzymatically active, FAD-containing oc-            &#9004;pex3&#9004;pyc1 cells showed that the level of various perox- <br /> tamers inside peroxisomes.                                             isomal enzymes and peroxins was normal (Figure 2). <br />                                                                        However, AO assembly was not restored, because very <br />                                                                        low specific AO activities were detected (unpublished <br /> The AO Assembly Failure in &#9004;pyc1 Cells Is Not                          data). Also, sucrose density gradient analysis of crude <br /> Indirect and Due to an Import Defect                                   extracts prepared from methanol-induced &#9004;pex3&#9004;pyc1 <br /> The failure of AO assembly in &#9004;pyc1 cells may be related               cells revealed that, like in &#9004;pyc1, AO was predominantly <br /> to a spatial separation of the AO monomers (in the cy-                 present in a monomeric state (Figure 3). Controls, pre- <br /> tosol) and putative peroxisomal assembly factor(s). In H.              pared from crude extracts of &#9004;pex3 cells, confirmed that <br /> polymorpha pex mutants these peroxisomal factor(s) most                the absence of peroxisomes in these cells did not influence <br /> likely are also mislocalized to the cytosol, thus explaining           AO oligomerization, because bulk of the AO protein was <br /> why in these cells AO assembly/activation normally oc-                 found in fractions where octameric AO sediments (Figure <br /> curs in this compartment (van der Klei et al., 1991c). In              3, fraction 6 and 7). These results suggest that the AO <br /> &#9004;pyc1 cells, however, normal peroxisomes are still present             assembly defect in &#9004;pyc1 cells is not an indirect effect, due <br /> that may contain the putative AO assembly factor(s), as a              to a specific AO protein import block. <br /> peroxisomal protein import defect other than for AO pro- <br /> tein was not observed. To test this possibility, we con- <br /> structed a H. polymorpha &#9004;pex3&#9004;pyc1 double mutant, in                  AO Assembly Does not Require HpPyc1p Enzyme <br /> which all peroxisomal matrix proteins are predicted to be              Activity <br /> mislocalized to the cytosol (Baerends et al., 1996). Bio-              To test whether AO assembly is dependent on HpPyc1p <br /> chemical analysis of crude extracts of methanol-induced                enzyme activity, we introduced a point mutation in H. poly- <br />  <br />  <br /> 790                                                                                                         Molecular Biology of the Cell <br />                                                                                                                Alcohol Oxidase Assembly <br />  <br />  <br /> morpha PYC1 that replaces the active site residue arginine <br /> 316 by glutamine (R316Q). Indeed, this mutant was unable <br /> to grow on minimal medium containing glucose unless sup- <br /> plemented with aspartate or glutamate (unpublished data). <br /> However, in the presence of these amino acids, cells of the <br /> mutant strain could grow on methanol, indicative for the <br /> restoration of the AO assembly defect (Figure 5A). Western <br /> blotting experiments using antibodies against S. cerevisiae <br /> Pyc protein, which cross-react with HpPyc1p (see Figure 2), <br /> revealed that HpPyc1pR316Q was synthesized in methanol- <br /> induced cells to levels comparable to WT cells (Figure 5B). <br /> Immunocytochemical analysis confirmed that AO protein <br /> was exclusively present in peroxisomes of these cells, indis- <br /> tinguishable from WT cells (Figure 5C). Hence, not the en- <br /> zyme activity but another function of the protein is required <br /> for AO assembly. <br />  <br /> HpPyc1p Is a Cytosolic Enzyme <br /> To determine the subcellular location of HpPyc1p, homog- <br /> enized protoplasts prepared from methanol-grown WT cells <br /> were subjected to sucrose density centrifugation followed by <br /> Western blot analysis of the various fractions obtained. As <br /> shown in Figure 6, HpPyc1p was only detected in the low- <br /> density fractions at the top of the gradient (Figure 6, frac- <br /> tions 18 &#8211;24), indicative for a cytosolic location. Analysis of <br /> the peroxisomal peak fractions did not reveal any HpPyc1p, <br /> indicating that the protein does not partially cosediment <br /> with peroxisomes. A cytosolic location is in line with the <br /> reported location of Pyc protein in S. cerevisiae (Walker et al., <br /> 1991). <br />  <br /> Newly Induced HpPyc1p Can Mediate Assembly of <br /> Cytosolically Accumulated AO Monomers <br /> The intriguing question concerns the function of HpPyc1p in <br /> AO assembly: does it act like a chaperone or does it serve <br /> other functions, e.g., in cofactor binding? This question was <br /> addressed in a &#9004;pyc1 strain that contained a copy of the <br /> PYC1 gene under control of the inducible amine oxidase <br /> promoter (PAMO). In this strain, &#9004;pyc1::PAMOPYC1, the syn- <br /> thesis of AO monomers and the HpPyc1p protein can be <br />                                                                     Figure 5. Enzymatically active HpPyc1p is not required for AO <br /> separated in time. First, cells of this strain were induced on      assembly. (A) Growth of H. polymorpha WT (F), H. polymorpha &#9004;pyc1 <br /> methanol/ammonium sulfate, conditions that induce the               producing the mutant protein HpPyc1pR316Q ( ) and H. polymorpha <br /> synthesis of AO monomers, but largely represses HpPyc1p             &#9004;pyc1 containing an empty expression plasmid (E) on mineral <br /> synthesis due to the presence of ammonium sulfate. Subse-           media containing methanol supplemented Asp and Glu. H. polymor- <br /> quently, the cells were shifted to glucose/ethylamine, con-         pha &#9004;pyc1 cells are severely hampered in growth (OD660 &#11005; 0.6 after <br /> ditions that strongly repress AO synthesis (due to the pres-        36 h). The initial increase in cell density is due to the presence of <br /> ence of glucose) but induce HpPyc1p synthesis due to the            small amounts of yeast extract in the media. In contrast, cells of H. <br /> induction of the PAMO by ethylamine. This allowed address-          polymorpha WT and the &#9004;pyc1 strain producing HpPyc1pR316Q are <br />                                                                     able to grow on methanol and reached a comparable final yield <br /> ing the question whether existing, monomeric AO molecules <br />                                                                     (OD660 3.2 and 2.5 after 36 h of growth, respectively; the cell density <br /> that had accumulated in the cytosol, were still accessible for      is expressed as optical density at 660 nm; OD660). (B) Comparison of <br /> the HpPyc1p function in AO assembly. The results depicted           the levels of HpPyc1p in crude extracts prepared from methanol- <br /> in Figure 7 show that HpPyc1p was below the limit of                induced H. polymorpha strains using Western blotting and antibod- <br /> detection in &#9004;pyc1::PAMOPYC1 cells before the shift (Figure         ies against S. cerevisiae Pycp. Lane 1, WT; lanes 2&#8211;5, &#9004;pyc1 cells <br /> 7A, lane 4, T &#11005; 0 min). After the shift to ethylamine-contain-      containing no plasmid (2), an empty expression plasmid (3), a <br /> ing media, PYC1 was rapidly induced and HpPyc1p levels              plasmid containing WT HpPYC1 (4) or a plasmid containing mutant <br /> comparable to WT were detected within 60 min. (compare              PYC1-R316Q (5). Equal amounts of protein were loaded per lane. <br /> lanes 1&#8211;3 with lane 6). Western blot analysis of a native gel       (C) Immunocytochemical demonstration of AO protein in metha- <br />                                                                     nol-induced &#9004;pyc1 cells producing HpPyc1p containing the muta- <br /> confirmed that significant amounts of monomeric AO had <br />                                                                     tion R316Q. Anti-AO specific labeling was confined to peroxisomes. <br /> accumulated before the shift to ethylamine as nitrogen <br /> source. However, within 30 min after the induction of <br /> HpPyc1p synthesis, the monomeric AO band had disap- <br />  <br />  <br /> Vol. 14, February 2003                                                                                                                 791 <br />  P. Ozimek et al. <br />  <br />  <br />  <br />  <br /> Figure 6. Sucrose gradient, prepared from a postnuclear superna- <br /> tant obtained from methanol-grown H. polymorpha WT cells. The <br /> graph shows the distribution of the peroxisomal marker enzyme <br /> AO (&#11001;), the mitochondrial marker enzyme cytochrome c oxidase <br /> ([itrig]), the protein (0) and sucrose concentrations (dotted line). The <br /> Western blot shows the distribution of HpPyc1p in the even frac- <br /> tions of the gradient. The protein was only detected in the upper <br /> part of the gradient (fractions 18 &#8211;24), which corresponds to the <br /> cytosol. Sucrose concentrations are expressed as % (wt/wt), the <br /> protein concentrations as mg/ml and the specific activities of AO <br /> and cytochrome c oxidase as percentages of the value in the peak <br /> fractions that were arbitrarily set at 100.                                Figure 7. Biochemical demonstration of the assembly of cytosoli- <br />                                                                            cally accumulated monomeric AO upon subsequent artificial induc- <br />                                                                            tion of PYC1. Cells of the &#9004;pyc1::PAMOPYC1 strain were pregrown <br />                                                                            on methanol/ammonium sulfate to induce AO synthesis under <br /> peared. At the same time the level of AO enzyme activity                   conditions that PYC1 expression is strongly repressed by ammo- <br /> (Figure 7C) had significantly increased. In a control experi-              nium sulfate. Subsequently the cells were shifted (at T &#11005; 0 min) to <br /> ment using WT cells, the level of AO activity and octameric                media containing glucose and ethylamine to induce PAMO, and thus <br />                                                                            HpPyc1p production and to repress AO synthesis. (A) A Western <br /> AO decreased as a result of glucose-induced degradation of                 blot of an SDS-PAA gel prepared from crude extracts of WT and <br /> peroxisomes. This process most likely also occurs in                       &#9004;pyc1::PAMOPYC1 cells taken before (T &#11005; 0 min) or 30 and 60 min <br /> &#9004;pyc1::PAMOPYC1 cells, as indicated by the subsequent re-                  after the shift. The blot, decorated with anti-Pyc antibodies, shows <br /> duction in AO protein and activity levels that follow the                  the induction profile of HpPyc1p in &#9004;pyc1::PAMOPYC1 cells, com- <br /> initial strong increase (Figure 7C).                                       pared with WT levels. (B) A Western blot of a native gel of the same <br />                                                                            samples. This blot, decorated with anti-AO antibodies, which allows <br />                                                                            to detect both octameric AO (O) or monomeric AO (M), shows that <br /> HpPyc1p Physically Interacts with AO and FAD                               monomeric AO present at T &#11005; 0 min is undetectable at T &#11005; 30 min. <br /> To study whether HpPyc1p has affinity for AO protein, in                   (C) The enzyme activities of AO in the same samples (black bar: WT <br /> vitro binding studies were performed. To this purpose a                    cells; gray bar: &#9004;pyc1::PAMOPYC1 cells). The reduction in AO activ- <br /> Sepharose column containing AO protein was prepared                        ities in WT cells is due to selective peroxisome degradation, induced <br /> (Evers et al., 1993). As a control, immobilized BSA was used.              by glucose. The values in both strains at T &#11005; 0 min were set to 100% <br />                                                                            (absolute values at T &#11005; 0 h for WT ,2.0 U/mg protein and <br /> Purified HpPyc1p was loaded onto these columns. On ex- <br />                                                                            &#9004;pyc1::PAMOPYC1 cells 0.5 U/mg protein). <br /> tensive washing, the bound protein was eluted by a buffer <br /> containing 8 M urea. As shown in Figure 8, a significant <br /> portion of the loaded HpPyc1p protein had bound to the AO <br /> column, whereas in the control experiment using BSA all                    normalized fluorescence autocorrelation curves of FAD in <br /> HpPyc1p was found in the flow-through fraction. These                      the absence of HpPyc1p revealed that the average diffu- <br /> findings indicate that HpPyc1p is capable of binding to AO                 sion time of FAD, obtained by a one-component fit anal- <br /> protein.                                                                   ysis, was 41.53 &#9262;s. This value is in agreement with the <br />    To study whether HpPyc1p could play a role in the                       molecular weight of FAD molecules. After addition of <br /> association of FAD to AO, we tested whether HpPyc1p,                       purified HpPyc1p, the fluorescence autocorrelation curve <br /> which contains an ATP-binding motif, is capable of bind-                   changed and could be best fitted with a two-component <br /> ing FAD. FAD binding was measured using fluorescence                       fit. Fixing the average diffusion time of FAD, the diffusion <br /> correlation spectroscopy (FCS), a technique that allows to                 time of the second component was 191&#8211;213 &#9262;s. Using the <br /> measure diffusion constants of fluorophores. Analysis of                   equation MWPYC &#11005; (&#9270;PYC/&#9270;FAD)3 &#11003; MWFAD the molecular <br />  <br />  <br /> 792                                                                                                              Molecular Biology of the Cell <br />                                                                                                              Alcohol Oxidase Assembly <br />  <br />  <br />                                                                       strain of P. pastoris (Menendez et al., 1998). The P. pastoris <br />                                                                       &#9004;pyc strain, like its H. polymorpha counterpart, was unable to <br />                                                                       grow on glucose unless aspartate or glutamate was added. <br />                                                                       Growth on methanol was fully prevented, irrespective of the <br />                                                                       presence of these amino acids in the media. Immunocyto- <br />                                                                       chemical experiments revealed that, like in H. polymorpha, P. <br />                                                                       pastoris &#9004;pyc cells did not import AO in peroxisomes (Figure <br />                                                                       9B) as in WT cells (Figure 9A). <br /> Figure 8. In vitro interaction of AO and HpPyc1p. Purified               Moreover, a P. pastoris pyc1 suppressor mutant (Menen- <br /> HpPyc1p was loaded onto columns containing AO or BSA protein.         dez et al., 1998), in which the aspartate requirement was <br /> The columns were subsequently extensively washed. The flow            restored, still failed to assemble AO and thus, to grow on <br /> through and wash fractions were pooled (F). Bound HpPyc1p was         methanol (unpublished data), indicating that also in P. pas- <br /> eluted using a buffer containing 8 M urea (E). Equal portions of F <br /> and E were subjected to SDS-PAGE and blotted. The blots were <br />                                                                       toris Pyc protein, but not the enzyme activity, is required for <br /> decorated using anti-Pyc antibodies. Most HpPyc1p was found in        AO assembly. <br /> the elution fraction (E) when AO columns were used, in contrast all      Remarkably, P. pastoris &#9004;pyc cells grew normally on oleic <br /> HpPyc1p was found in the flow through and wash fraction (F) when      acid, at rates similar to WT controls. Hence, the absence of <br /> a control column containing BSA was used.                             Pyc1p has no general deteriorating effect on peroxisome <br />                                                                       biogenesis or function. Also, one of the key enzymes of <br />                                                                       oleate metabolism is a peroxisomal flavoprotein, namely <br /> weight of the second component was calculated to be                   acyl-CoA oxidase. This protein displays normally activities <br /> 105&#8211;130 kDa. Assuming the HpPyc1p molecules to be                     (unpublished data) and is located in peroxisomes (Figure <br /> spherical in shape, this is in agreement with the apparent            9C). Therefore, the PYC1 deletion seems to interfere specif- <br /> molecular weight of monomeric HpPyc1p calculated from                 ically with AO assembly and not with that of other peroxi- <br /> its amino acid sequence (130 kDa). Hence, these data                  somal flavin proteins. <br /> indicate that part of the FAD had bound to the added <br /> HpPyc1p. In control experiments using lysozyme instead                DISCUSSION <br /> of HpPyc1p no change of fluorescence autocorrelation <br /> curve and diffusion time were observed.                               We have identified pyruvate carboxylase (Pyc) as the first <br />                                                                       protein that has an essential function in assembly of perox- <br />                                                                       isomal AO in methylotrophic yeast. Pyc is an anapleurotic <br /> Also in P. pastoris Pyc1p Is Essential for AO                         enzyme that replenishes the tricarboxylic acid cycle with <br /> Import and Assembly                                                   oxaloacetate from pyruvate. As a consequence, yeasts lack- <br /> To determine whether the Pyc1p-dependent AO assembly                  ing Pyc enzyme activity cannot grow on minimal glucose <br /> defect is limited to H. polymorpha or represents a common             media unless aspartate or glutamate is added, amino acids <br /> feature of methylotrophic yeast, we analyzed a PYC deletion           that can be converted into oxaloacetate. Unexpectedly, H. <br />  <br />  <br />  <br />  <br /> Figure 9. Immunocytochemical localization of AO (A and B) and acyl-CoA oxidase (C) in P. pastoris WT (A) and &#9004;pyc cells (B and C). In <br /> methanol-grown WT cells anti-AO-dependent labeling is confined to peroxisomes (A), whereas in &#9004;pyc cells (B) the protein was predomi- <br /> nantly detected in a cytosolic aggregate (*) but not in peroxisomes. In oleic acid-grown &#9004;pyc cells the antiacyl-CoA oxidase&#8211; dependent <br /> labeling is confined to peroxisomal profiles (C). <br />  <br />  <br /> Vol. 14, February 2003                                                                                                             793 <br />  P. Ozimek et al. <br />  <br />  <br /> polymorpha and P. pastoris strains lacking Pyc are also unable       An alternative explanation for the phenotype of H. poly- <br /> to grow on methanol, independent of the presence of aspar-        morpha &#9004;pyc1 is that HpPyc1p is essential to mediate binding <br /> tate or glutamate in the medium. We demonstrated that this        of AO to the receptor Pex5p and that FAD-binding occurs <br /> growth defect is due to a severe block in the assembly of AO,     after import mediated by a peroxisomal factor. However, <br /> a key enzyme in methanol metabolism. Our data convinc-            this explanation is less likely in view of the fact that in cells <br /> ingly show that Pyc protein but not its enzyme activity is        of the &#9004;pex3&#9004;pyc1 double mutant AO remains monomeric, <br /> necessary for AO assembly. The import and activation of           whereas it is normally active in single pex mutants, e.g., <br /> another peroxisomal flavin enzyme, acyl-CoA oxidase, was          &#9004;pex3 (compare Figure 3) and &#9004;pex5 (van der Klei et al., <br /> not affected in the absence of Pyc. Hence, Pyc seems to           1995). <br /> function specifically in the AO assembly pathway in methy-           We showed before, that FAD-containing AO monomers <br /> lotrophic yeast.                                                  can assemble spontaneously into octamers in vitro (Evers et <br />    The current model of AO assembly hypothesizes that AO          al., 1995). Hence, it can be envisaged that this also can take <br /> monomers but not octamers are imported into peroxisomes.          place in intact cells in vivo. However, when FAD-binding <br /> This is based on the finding that octameric AO protein is not     indeed occurs in the cytosol the cell has to deal with the <br /> incorporated into peroxisomes in vivo (Douma et al., 1990;        problem how to prevent premature spontaneous assembly <br /> Waterham et al., 1993) and on the results of elegant pulse        of the FAD-containing monomers. There is a strong <span id='am-90' about='obo:NCBITaxon_94025' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_27394 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_74974 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_74975 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-91' property="rdfs:label" content="Meta" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-94'  ></span>meta</span>- <br /> chase experiments that provided evidence that assembly            bolic need for the cell to postpone this event until import has <br /> into the active octamer takes place inside the organelle          occurred, because only minor amounts of active AO in the <br /> (Goodman et al., 1984; Stewart et al., 2001).                     cytosol give rise to severe energetical disadvantages due to <br />    FAD most likely binds to monomeric AO, as is indicated         a cytosolic H2O2 metabolism that would retard or even <br /> by the finding that FAD cannot reassociate in vivo to AO          prevent growth on methanol (van der Klei et al., 1991b). One <br /> octamers, from which FAD has been chemically removed              possibility is that HpPyc1p remains bound to AO after co- <br /> (van der Klei et al., 1989a). Whether in H. polymorpha FAD        factor binding. A second option is that, upon FAD binding, <br /> binds to AO monomers before or upon translocation across          the protein is immediately donated to Pex5p, which pre- <br /> the peroxisomal membrane was so far still speculative.            vents octamer formation. Because other PTS1 proteins <br />    We show here that in H. polymorpha &#9004;pyc1 cells bulk of the     (DHAS, CAT) are normally imported in &#9004;pyc1 cells and also <br /> AO protein accumulated as inactive, FAD-lacking mono-             their Pex5p levels are similar to WT cells, it is indeed un- <br /> mers in the cytosol. Only a minor portion had assembled           likely that Pex5p molecules are bound to the large pool of <br /> into FAD-containing, enzymatically active octamers,               FAD-lacking AO monomers in the cytosol of &#9004;pyc1 cells. <br /> which&#8212; based on cytochemical experiments&#8212;were local-                 Based on the above reasoning, our adapted hypothetical <br /> ized in the peroxisomal matrix. This suggests that the pres-      model of AO assembly predicts that the first step in AO <br /> ence of HpPyc1p is important for FAD-binding to AO mono-          import is HpPyc1p-mediated FAD binding to newly synthe- <br /> mers, the subsequent translocation into the peroxisomal           sized AO protein in the cytosol (Figure 10). Subsequently, <br /> matrix and finally the assembly into octamers.                    the FAD-containing monomers bind to the PTS1 receptor <br />    Interestingly, the phenotype of H. polymorpha &#9004;pyc1 cells is   Pex5p, followed by the translocation of the Pex5p-AO cargo <br /> highly comparable to that of the H. polymorpha riboflavin         complex into peroxisomes. After dissociation of the cargo <br /> auxotrophic mutant, rif1 (Evers et al., 1996), in which also      from Pex5p in the organellar matrix, the FAD-containing AO <br /> bulk of the AO protein accumulates as FAD-lacking mono-           monomers may spontaneously oligomerize into enzymati- <br /> mers in the cytosol. A likely way to explain both observa-        cally active octamers followed by shuttling of Pex5p back to <br /> tions is that FAD binding to AO monomers in the cytosol is        the cytosol to mediate another round of PTS1 protein im- <br /> a prerequisite to allow efficient translocation into peroxi-      port. This model is in line with the previously proposed <br /> somes. Our current findings suggest that cytosolically lo-        &#8220;extended shuttle model&#8221; for Pex5p (van der Klei et al., 1995; <br /> cated HpPyc1p is required to bind FAD to AO monomers.             van der Klei and Veenhuis, 1996), which was recently ex- <br />    We found that in HpPyc1p-deficient cells soluble, mono-        perimentally proven by Dammai and Subramani (2001) for <br /> meric AO accumulated that could be activated upon subse-          human cells. <br /> quent artificial induction of PYC1 expression. Together with         A further implication of the present study is that in <br /> the observation that HpPyc1p physically interacts with AO         methylotrophic yeasts Pyc has multiple functions. The rele- <br /> protein and is capable of binding FAD, our current data           vance of this is obvious in view of the data on the human <br /> strongly suggest that HpPyc1p functions as a FAD-binding          genome that have become available recently (Lander et al., <br /> protein in the cytosol.                                           2001; Venter et al., 2001). The 30,000 genes that have been <br />    Relatively little is known on proteins that play a role in     identified cannot cope for the multitude of functions that are <br /> cofactor binding. It is generally assumed that cofactor bind-     predicted to be essential in men unless specific genes encode <br /> ing occurs spontaneously upon formation of the correct            multiple proteins or proteins with multiple functions. Pro- <br /> binding site in a protein molecule. So far no proteins have       teins that fulfill two different functions are the housekeeping <br /> been described that play a role in noncovalent binding of         enzyme lactate dehydrogenase B in duck, which is also the <br /> FAD; also, only one is yet identified that is essential for       <span id='am-34' about='obo:NCBITaxon_3863' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_163743 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-35' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-36'  ></span><span id='am-37' property="rdfs:label" content="Lens" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Lens Mill." datatype="xsd:string"></span>lens</span> structural protein epsilon-crystallin (Hendriks et al., <br /> covalent FAD binding (Kim et al., 1995). Several examples         1988); glyceraldehyde 3-phosphate dehydrogenase, which <br /> are known of proteins involved in binding of heme, for            has been shown to play a role in endocytosis in CHO cells <br /> instance, mitochondrial heme lyases that are necessary for        (Robbins et al., 1995); and the &#9251;-subunit of phosphofructoki- <br /> the attachment of heme to cytochrome c or cytochrome c1           nase (but not enzyme activity), which is required for the <br /> (Page et al., 1998). However, despite extensive research mo-      onset of glucose-mediated selective pexophagy (microauto- <br /> lecular details on their mode of action are still lacking.        phagy; Yuan et al., 1997). The switch in function may even be <br />  <br />  <br /> 794                                                                                                   Molecular Biology of the Cell <br />                                                                                                                  Alcohol Oxidase Assembly <br />  <br />  <br />                                                                         nal is not the only targeting information essential for import <br />                                                                         (Waterham et al., 1997). Therefore, specific additional pro- <br />                                                                         teins may exist that exclusively function in AO import and <br />                                                                         assembly. We are currently trying to identify these proteins <br />                                                                         by cloning the encoding genes by functional complementa- <br />                                                                         tion of the other ass mutants. <br />  <br />  <br />  <br /> Figure 10. Hypothetical model of AO import and assembly in H. <br />                                                                        <br /> polymorpha. AO monomers are synthesized on free ribosomes in the <br /> cytosol. In the cytosol HpPyc1p assists in binding of the cofactor    <br /> FAD to newly synthesized AO monomers. Subsequently, the FAD-         <br /> containing monomers are bound to the PTS1 receptor, Pex5p. Both     <br /> the receptor and its cargo are translocated across the peroxisomal      <br /> membrane (I), followed by dissociation of the AO-Pex5p complex in     <br /> the organellar matrix. The FAD-containing AO monomers then              <br /> assemble into octamers. Pex5p recycles to the cytosol (II) to mediate   <br /> another round of PTS1 protein import. <br />  <br /> temperature dependent, as demonstrated for heat shock             <br /> protein DegP, which normally functions as a chaperone but              <br /> displays proteolytic activity at elevated temperatures (Spiess         <br /> et al., 1999).                                                        <br />    Our collection of Ass mutants that comprises 10 different           <br /> complementation groups (Van Dijk et al., 2002), suggests               <br /> that additional proteins may be involved in AO biosynthesis             <br /> (e.g., targeting or translocation). This view is also based on          <br /> the finding that in P. pastoris the carboxyterminal PTS1 sig-          <br /> </body></html>