<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Gene Duplication Zebrafish <br /> kit ligand Partitioning Melanocyte <br /> Development Functions kit ligand a <br /> Keith A. Hultman1, Nathan Bahary2, Leonard I. Zon3,4, Stephen L. Johnson1* <br /> 1 Department Genetics, Washington University School Medicine, Saint Louis, Missouri, United States America, 2 Department Molecular Genetics Biochemistry, <br /> University Pittsburgh School Medicine, Pittsburgh, Pennsylvania, United States America, 3 Department Pediatrics, Children&#8217;s Hospital, Boston, Massachusetts, <br /> United States America, 4 Howard Hughes Medical Institute, Boston, Massachusetts, United States America <br />  <br />  <br /> The retention particular genes genome duplication zebrafish given insights genes <br /> may evolve partitioning ancestral functions. We examine partitioning expression patterns and <br /> functions zebrafish kit ligands, kit ligand (kitla) kit ligand b (kitlb), discuss possible coevolution <br /> with duplicated zebrafish kit receptors (kita kitb). In situ hybridizations kitla mRNA expressed in <br /> the trunk adjacent notochord middle somite stages melanocyte migration later <br /> expressed skin, receptor required melanocyte survival. kitla expressed regions <br /> complementary kita receptor expression, including pineal gland, tail bud, ear. In contrast, kitlb mRNA is <br /> expressed brain ventricles, ear, cardinal vein plexus, regions generally complementary zebrafish <br /> kit receptor ortholog. However, like kitla, kitlb expressed skin stages consistent melanocyte <br /> survival. Thus, appears kita kitla maintained congruent expression patterns, kitb kitlb have <br /> evolved divergent expression patterns. We demonstrate interaction kita kitla morpholino knockdown <br /> analysis. kitla morphants, kitlb morphants, phenocopy null allele kita, defects melanocyte <br /> migration survival. Furthermore, kitla morpholino, kitlb morpholino, interacts genetically sensitized <br /> allele kita, confirming kitla functional ligand kita. Last, examine kitla overexpression embryos, <br /> which results hyperpigmentation caused increase number size melanocytes. <span id='am-112' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-113'  ></span><span id='am-114' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-116' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> <br /> hyperpigmentation dependent kita function. We conclude following genome duplication, kita kitla <br /> have maintained receptor&#8211;ligand relationship, coevolved complementary expression patterns, that <br /> functional analysis reveals kita receptor&#8217;s function embryo promoted its <br /> interaction kitla. <br /> Citation: Hultman KA, Bahary N, Zon LI, Johnson SL (2007) Gene duplication zebrafish kit ligand partitioning melanocyte development functions kit ligand a. <br /> PLoS Genet 3(1): e17. doi:10.1371/journal.pgen.0030017 <br />  <br />  <br />                                                                                           mutations neural crest derived pigment cells make the <br /> Introduction <br />                                                                                           zebra&#65533;?sh attractive model analysis genetic <br />    Vertebrate melanocyte pigment patterns long                                  mechanisms pigment cell development. Additionally, the <br /> studied model pattern formation morphogenesis.                               zebra&#65533;?sh genome thought duplicated early the <br /> Melanocytes arise neural crest immediately begin <br />                                                                                           teleost lineage [9&#8211;12]. Since then, redundant genes <br /> to migrate ventrally distinct migratory pathways to <br />                                                                                           lost, small percentage duplicates <br /> establish embryonic pigment pattern. Mutations genes <br /> required melanocyte morphogenesis increased                                  retained, presumably evolving independ- <br /> understandings cell&#8211;cell communication, signal trans-                                  ently. One co-orthologous pair zebra&#65533;?sh kit genes <br /> duction, differentiation, cell migration. Mice null                              [13,14]. The zebra&#65533;?sh mutant these, kita (sparse), <br /> mutations receptor tyrosine kinase Kit (Dominant                                   defects embryonic melanocyte development; however, <br /> spotting, W) embryonic lethal homozygotes the <br /> failure hematopoiesis [1]. Animals heterozygous null                            Editor: Gregory S. Barsh, Stanford University School Medicine, United States of <br /> homozygous hypomorphic alleles viable, however,                               America <br /> display white spotting coat color phenotype. Further                                    Received September 14, 2006; Accepted December 12, 2006; Published January <br /> analysis mutants revealed early failure                                 26, 2007 <br /> melanocyte lineage: fewer melanoblasts arise neural                                Copyright: &#211; 2007 Hultman et al. This open-access article distributed under <br /> crest, defects ventral dispersion, die                            terms Creative Commons Attribution License, permits unrestricted <br /> melanization [2&#8211;4]. The locus encoding ligand                                use, distribution, reproduction medium, provided original author <br />                                                                                           source credited. <br /> receptor, Kitl (Steel, Sl), displays nearly identical de&#65533;?ciencies <br /> [5&#8211;8].                                                                                    Abbreviations: dpf, days postfertilization; hpf, hours postfertilization; kita, <br />                                                                                           zebrafish kit receptor; kitb, zebrafish kit b receptor; Kitl, mouse kit ligand; kitla, <br />    More recently, use zebra&#65533;?sh, <span id='am-69' about='obo:NCBITaxon_7955' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7954 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="Brachydanio rerio" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="Danio frankei" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasExactSynonym" content="zebrafish" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasRelatedSynonym" content="Danio rerio (Hamilton, 1822)" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasRelatedSynonym" content="Cyprinus rerio" datatype="xsd:string"></span><span id='am-75'  ></span><span id='am-76' property="oboInOwl:hasAlternativeId" content="NCBITaxon:27702" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasRelatedSynonym" content="Brachidanio rerio" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="Danio rerio" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasAlternativeId" content="NCBITaxon:37966" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="Danio rerio frankei" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasExactSynonym" content="leopard danio" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Cyprinus rerio Hamilton, 1822" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasExactSynonym" content="zebra danio" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasRelatedSynonym" content="Brachydanio rerio frankei" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasExactSynonym" content="zebra fish" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Danio rerio</span>,                               zebrafish kit ligand a; kitlb, zebrafish kit ligand b; MO, morpholino oligonucleotide; <br /> been successful extension understanding                                       RACE, rapid amplification cDNA ends <br /> vertebrate pigment pattern biology. The optical clarity                                * To correspondence addressed. E-mail: sjohnson@genetics.wustl. <br /> zebra&#65533;?sh embryos, oviparous development, genetic                                      edu <br />  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                       0089                                          January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                            Zebrafish kit ligand Duplication/Evolution <br />  <br />  <br /> Author Summary                                                                 rapidly cleaved membrane. In vitro experi- <br />                                                                                ments shown cells expressing variant produce <br /> Gene duplication events provide useful substrate identify             soluble ligand product [17]. The Kitl-2 isoform splices out <br /> effects evolution reshaping genes roles physiology          exon 6, does contain secondary cleavage <br /> or development. Thus, dozens receptor tyrosine kinases,                site exon 7, expression shown more <br /> differing roles development, generated animal                  restricted cell membrane Kitl-1 [17,18]. <br /> lineages. Less clear associated ligands duplicated <br />                                                                                Furthermore, genetic analysis alleles Kitl <br /> and evolved evolution constrained match <br /> that cognate receptors. This report demonstrates                  reveals splice forms involved different <br /> duplication kit ligand gene zebrafish shows                 functions Kit signaling. Mice Steel-dickie (Sld) <br /> expression function specific development                     mutation, lack transmembrane domain, produce <br /> melanocyte partitioned ligands, kitla.               soluble form Kitl similar Kitl-1 [15,16] <br /> By means, kitla coevolved duplicates kit           melanocytes migrate properly subse- <br /> receptor tyrosine kinase regulate zebrafish melanocyte develop-             quently die [17]. This indicates survival requirement for <br /> ment. In contrast, expression pattern ligand, kitlb,          properly expressed Kitl basolateral surface the <br /> which kitla approximates expression           epidermis. Transgenic mice expressing Kitl lacking <br /> mouse kit ligand gene, evolved independently kit <br />                                                                                exon 6, however, appear wild-type pigmentation as <br /> receptor gene. <br />                                                                                adults qualitative migration de&#65533;?ciency <br />                                                                                [18]. The current hypothesis soluble ligand product <br />                                                                                required establish chemotactic chemokinetic <br />                                                                                gradient promoting cell migration, cell survival <br /> there differences role Kit mice. In                 sustained cell&#8211;cell contact membrane- <br /> contrast mouse, zebra&#65533;?sh homozygous kitab5 (null)                   tethered ligand. The alternative splicing primary <br /> viable de&#65533;?ciencies primordial germ cells                     cleavage site exon 6 Kitl contribute the <br /> hematopoiesis. Zebra&#65533;?sh kita mutants retain 60% wild-                       regulation switch development. <br /> type embryonic melanocyte numbers [13]. These melano-                             In investigation, ortholog tetrapod <br /> cytes fail migrate periphery embryo                       Kitl duplicated zebra&#65533;?sh genome co- <br /> remain dorsal regions neural tube                      orthologs undergone splice form degeneration. One co- <br /> the otocysts. After 4 days postfertilization (dpf),                      ortholog, kitla, retained exon 6, kitlb appears have <br /> melanocytes undergo programmed cell death                              lost exon. Given suggested roles mouse <br /> completely absent 12 dpf. The second kit ortholog, kitb,                    isoform, hypothesized kitla, having retained exon 6, <br /> is expressed Rohon-Beard neurons, trigeminal gan-                        better suited promote melanocyte migration, <br /> glion, ear does involved                      similar mouse Kitl-1, kitlb, having lost exon 6, would <br /> melanocyte development [14].                                                   required melanocyte survival, like mouse Kitl-2. We test <br />    The primary roles kita zebra&#65533;?sh melanocyte migration                  hypothesis &#65533;?rst examining mRNA expression of <br /> and survival separable. Genetic analysis receptor                   gene using situ hybridization targeting <br /> using temperature-sensitive allele kita (kita jle99) revealed             antisense morpholino oligonucleotides (MOs) zebra- <br /> that kita signaling required migration 2 dpf                 &#65533;?sh co-ortholog Kitl. <br /> then required cell survival stage [15].                   We kitla expressed trunk the <br /> Furthermore, identi&#65533;?cation migration-speci&#65533;?c                        medial melanocyte migration pathway early migra- <br /> survival-speci&#65533;?c alleles kita suggests                       tory stages, 18 30 hours postfertilization (hpf), then <br /> functions involve separate molecular domains kita                       later expressed skin stages (4 5 dpf) that <br /> [15]. Although migration-speci&#65533;?c alleles contained                     require kita receptor function survival. MO knockdown <br /> lesions near putative ligand-binding domain, allele                    kitla con&#65533;?rms required melanocyte <br /> that defective melanocyte survival lesion                 migration survival. We overexpression of <br /> intracellular kinase domain kita. One possible explanation                  kitla results hyperpigmented embryo increase in <br /> for &#65533;?nding survival ligand independent                      number size melanocytes. A demonstra- <br /> zebra&#65533;?sh.                                                                      tion kitla acting kita pathway achieved <br />    Further evidence Kitl Kit separable functions                 genetic interaction low dose kitla <br /> in migration survival revealed mice mutant                       MOs sensitized allele kita receptor, by <br /> Neuro&#65533;?bromin sustained RAS pathway activity [16].                    revealing hyperpigmentation phenotype kitla <br /> These mice melanocytes require Kit signaling                     requires kita function. <br /> migration independent Kit signaling melano-                        In contrast, kitlb expressed cells lining brain <br /> cyte survival, indicating activating RAS pathway                   ventricles, sensory epithelium ear, the <br /> suf&#65533;?cient melanocyte survival migration                     cardinal vein plexus regions associated trunk <br /> mouse.                                                                         melanocyte migration. However, like kitla, kitlb expressed in <br />    Separable functions migration survival                     skin 4 5 dpf larvae. MO knockdowns kitlb indicate <br /> established mouse Kit ligand, Kitl (also                        required embryonic melanocyte <br /> referred mast cell growth factor [Mgf], steel factor [SlF],              migration survival. Finally, discuss possible signal- <br /> and stem cell factor [Scf]). Kitl mouse expressed                 ing kit ligands kit receptors zebra&#65533;?sh <br /> cell surface splice variants. The larger variant,                partitioning functions gene duplication <br /> Kitl-1, contains primary cleavage site encoded exon 6                     events receptor&#8211;ligand pairs. <br />  <br />       PLoS Genetics | www.plosgenetics.org                              0090                                January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                                Zebrafish kit ligand Duplication/Evolution <br />  <br />  <br /> Results <br /> Identifying Zebrafish kit ligand Orthologs <br />    We interested kita-dependent pigment <br /> pattern larval zebra&#65533;?sh required kit ligand (Kitl mouse, <br /> KITLG human, referred Mgf, Scf). After searching <br /> the zebra&#65533;?sh genetic trace database Kitl ortholog using <br /> protein sequence multiple <span id='am-129' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-130' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span>, identi&#65533;?ed two <br /> partial candidate genes Chromosomes 25 4, we <br /> refer kitla kitlb, respectively. Each candidate locus <br /> was ampli&#65533;?ed 24-hpf embryonic cDNA con&#65533;?rm the <br /> annotated sequence test expressed. <br /> Full-length cDNA obtained candidate 59 and <br /> 39 rapid ampli&#65533;?cation cDNA ends (RACE) by <br /> amplifying exons. To test these <br /> candidates orthologous mammalian Kitl, con- <br /> structed phylogenetic tree known tetrapod <br /> sequences, including mouse, human, chick, copies in <br /> Xenopus, axolotl (Figure 1). We able identify <br /> two candidate copies predicted kitl fugu, medaka, <br /> and stickleback genomes. Since copies present the <br /> &#65533;?sh genomes closely related the <br /> tetrapod clade, appears zebra&#65533;?sh copies arose <br /> from duplication event teleost lineage the <br /> divergence tetrapod lineage teleost <br /> radiation. This observation consistent notion a <br /> whole genome duplication teleost lineage time <br /> [9&#8211;11]. We conclude zebra&#65533;?sh kitla kitlb co-                     Figure 1. Phylogenetic Tree kitl Protein Sequences <br /> orthologous mammalian Kitl.                                                The topology tree suggests kitl locus duplicated before <br />                                                                               teleost radiation. Dr, <span id='am-88' about='obo:NCBITaxon_7954' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-89' property="oboInOwl:hasRelatedSynonym" content="Celestichthys" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="Danio" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasAlternativeId" content="NCBITaxon:487617" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasRelatedSynonym" content="Brachydanio" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-94'  ></span><span id='am-95' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasAlternativeId" content="NCBITaxon:46777" datatype="xsd:string"></span>Danio</span> rerio (zebrafish), Fr, Fugu rubripes (fugu), <br /> kitl Co-orthologs Correspond Different Mammalian Kitl                      Ga, <span id='am-155' about='obo:NCBITaxon_69292' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_8100 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489922 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_8111 obo:NCBITaxon_33511 obo:NCBITaxon_1490020 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_69291'><span id='am-156' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-157'  ></span><span id='am-158' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-159' property="rdfs:label" content="Gasterosteus" datatype="xsd:string"></span>Gasterosteus</span> aculeatus (stickleback), Ol, <span id='am-51' about='obo:NCBITaxon_8089' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_28781 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_8088 obo:NCBITaxon_47757 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_76071 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665'><span id='am-52' property="oboInOwl:hasAlternativeId" content="NCBITaxon:123687" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasRelatedSynonym" content="Xenopoecilus" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Oryzias" datatype="xsd:string"></span><span id='am-55'  ></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasRelatedSynonym" content="Orizias" datatype="xsd:string"></span>Oryzias</span> latipes (medaka), Am, <br />                                                                               <span id='am-25' about='obo:NCBITaxon_8295' typeof='owl:Thing obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_7711 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7776 obo:NCBITaxon_30367 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_8293 obo:NCBITaxon_8294 obo:NCBITaxon_41666 obo:NCBITaxon_8292 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-26' property="rdfs:label" content="Ambystoma" datatype="xsd:string"></span><span id='am-27'  ></span><span id='am-28' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="big salamanders" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="mole salamanders" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Ambystoma</span> mexicanum (axolotl), Gg, <span id='am-97' about='obo:NCBITaxon_9031' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_8976 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_9005 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9030 obo:NCBITaxon_8825 obo:NCBITaxon_9072 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-98' property="oboInOwl:hasExactSynonym" content="chicken" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasExactSynonym" content="chickens" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasRelatedSynonym" content="Gallus domesticus" datatype="xsd:string"></span><span id='am-102' property="rdfs:label" content="Gallus gallus" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasRelatedSynonym" content="Gallus gallus domesticus" datatype="xsd:string"></span><span id='am-105'  ></span><span id='am-106' property="oboInOwl:hasExactSynonym" content="bantam" datatype="xsd:string"></span>Gallus gallus</span> (chick), Xl, Xenopus <br /> Isoforms                                                                      laevis (Xenopus), Mm, <span id='am-1' about='obo:NCBITaxon_10090' typeof='owl:Thing obo:NCBITaxon_314147 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_337687 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_862507 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_39107 obo:NCBITaxon_10088 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_10066 obo:NCBITaxon_6072 obo:NCBITaxon_33553 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9989 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-2' property="oboInOwl:hasExactSynonym" content="mouse" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="rdfs:label" content="Mus musculus" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasAlternativeId" content="NCBITaxon:85055" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="Mus muscaris" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="mice C57BL/6xCBA/CaJ hybrid" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="house mouse" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Mus Musculus</span> (mouse), Hs, <span id='am-117' about='obo:NCBITaxon_9606' typeof='owl:Thing obo:NCBITaxon_9605 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-118' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasExactSynonym" content="man" datatype="xsd:string"></span><span id='am-120'  ></span><span id='am-121' property="oboInOwl:hasExactSynonym" content="human" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-123' property="rdfs:label" content="Homo sapiens" datatype="xsd:string"></span>Homo sapiens</span> (human). <br />    Because different mammalian Kitl functions                       Bootstrap values 1,000 replicates labeled. <br /> shown correspond alternative splicing exon 6,                   doi:10.1371/journal.pgen.0030017.g001 <br /> examined genetic structure splice forms zebra&#65533;?sh <br /> kitla kitlb. To test zebra&#65533;?sh kitla kitlb                kitla retains exon 6 clear, predicted transcript <br /> alternatively spliced exon 6 larval pigment develop-                terminates exon 5. In contrast zebra&#65533;?sh transcripts <br /> ment, RT-PCR conducted gene primers                          con&#65533;?rmed RT-PCR experiments, fugu, <br /> &#64258;anking exon 6 larva 1, 2, 3, 4, 5 dpf. Each                    stickleback, medaka sequences predicted <br /> gene displayed single splice form (unpublished <span id='am-168' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-169'  ></span><span id='am-170' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-171' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-172' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span>). We                    GENSCAN sequences (http://genscanw.biosino.org http:// <br /> then aligned complete cDNA coding sequence kitla                   genes.mit.edu/GENSCAN). <br /> kitlb genomic sequence reveal exon&#8211;intron                        The &#65533;?nding zebra&#65533;?sh kitla kitlb appear resemble <br /> structure. A comparison genomic structure mouse                   mammalian alternative splice forms compelled notion <br /> Kitl reveals kitla contains exons, aligning               genes functioning similarly roles <br /> the murine exons (Figure 2). In contrast, kitlb                      suggested mouse isoforms. In mouse, alternative <br /> retained exons, exons 1 5 aligning                    splicing appears regulate ef&#65533;?ciency Kitl is <br /> with murine exons 1 5 exons 6 8                           cleaved cell membrane, isoform exon 6 <br /> aligning murine exons 7 9, suggesting kitlb                 (Kitl-1) contains primary cleavage site rapidly cleaved <br /> is missing sequence homologous mouse exon 6 (Figure 2).                    cell membrane [17]. This site identi&#65533;?ed <br /> Thus, duplication kit ligand gene zebra&#65533;?sh                      (amino acid sequence VAAS) mouse conserved in <br /> followed splice form degeneration similar                 human chick unrecognizable Xenopus axolotl <br /> shown zebra&#65533;?sh duplicated genes [19].                                [20,21]. We examined protein sequences the <br />    The presence absence exon 6 associated                 zebra&#65533;?sh genes possible cleavage sites. We not <br /> with kitla, kitlb, respectively, teleost. Both stickleback          identify cleavage site zebra&#65533;?sh kitla exon 6 (see Dataset <br /> and fugu sequence de&#65533;?ned kitla phylogenetic tree                    S1 multiple sequence alignment). We did, however, notice <br /> are predicted contain exon 6, kitlb species                  similar site fugu (amino acid sequence VISS) exon 6. In <br /> does not. However, predicted sequence medaka kitlb                    addition, secondary cleavage site, present both <br /> appears contain candidate exon 6. This indicate                    mouse isoforms exon 7 cleaved lower ef&#65533;?ciency, <br /> that splice form degeneration exon 6 occurred                          conserved species. Therefore, dif&#65533;?cult to <br /> independently taxa, radiation teleost               predict zebra&#65533;?sh kitla kitlb cleaved the <br /> clade. However, possible exon falsely                      membrane comparable ef&#65533;?ciency. <br /> identi&#65533;?ed gene-&#65533;?nding algorithm. Whether medaka                           The existence copies kit ligand, correspond- <br />  <br />       PLoS Genetics | www.plosgenetics.org                             0091                                     January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                                    Zebrafish kit ligand Duplication/Evolution <br />  <br />  <br />                                                                                        pathway. We distinguish expression levels higher <br />                                                                                        background regions lateral migration <br />                                                                                        pathway appropriate stages (18 30 hpf). Thus, <br />                                                                                        &#65533;?nd evidence change kitla expression is <br />                                                                                        responsible change migration medial to <br />                                                                                        lateral pathway. Instead, appears likely this <br />                                                                                        speci&#65533;?c migratory behavior controlled factors other <br /> Figure 2. Genomic Structure Alignment Mouse Kitl                                  kitla. We examined kitla expression is <br /> Exons (boxes) drawn scale labeled according                    present 4 dpf, kita required melanocyte <br /> homology mouse sequence. Pairwise similarity (percent identical <br /> residues plus conserved amino acid substitutions based                          survival. By 4 5 dpf, kitla expression lost the <br /> Blossum40 matrix) zebrafish protein mouse protein                     medial trunk regions evident skin (Figure 3I). <br /> presented exon parentheses. Full-length values mouse                This expression restricted melanocyte stripe regions <br /> zebrafish kitla: 29% identical 43% similar. Full-length values for <br /> mouse zebrafish kitlb: 20% identical 50% similar. Although quite                occurs skin, including interstripe <br /> diverged sequence, zebrafish paralogs display well-conserved                    regions (Figure 3I, black arrowhead). We faint <br /> intron site locations with mouse ortholog. The                 staining dorsal regions somite (Figure 3I, red <br /> best alignment kitlb reveals lost exon 6, is <br /> alternatively spliced mouse human. Exon 5 kitla expanded                 arrowhead). We suggest early expression medial <br /> 39 corresponding contraction exon 6. Kit binding domain                      cells supports medial lateral migration the <br /> contains residues interact Kit determined crystal                later switch expression skin provides subsequent <br /> structures mouse proteins [26,27]. The locations major <br /> cleavage site mouse exon 6 minor cleavage site mouse                     cues melanocyte survival. <br /> exon 7 indicated. We identify cleavage site                          In addition expression patterns kitla suggestive of <br /> sequence conservation zebrafish gene (see Dataset S1).                  roles pigment pattern, expression places that <br /> MOs (red bars) targeted overlap ATG start site kitlb and <br /> the exon 3&#8211;intron 3 boundary kitla kitlb. The splice site MOs                   role melanocyte development. Thus, &#65533;?rst <br /> resulted splicing exon 2 exon 4 (red lines), resulting shorter,          observe kitla mRNA expression tail bud 18 hpf <br /> in-frame, transcript (see Figure 4E 4F).                                           embryos presomitic mesoderm (Figure 3A 3B) as <br /> doi:10.1371/journal.pgen.0030017.g002 <br />                                                                                        pineal gland 24-h embryo (Figure 3C). <br />                                                                                        These regions complementary kita expression the <br /> ing alternative splice form mouse human, lead                          developing caudal notochord pineal gland [13], suggest- <br /> to hypothesize zebra&#65533;?sh kitla retains function                         ing possible role communication presomitic <br /> mammalian isoform 1 kitlb, having lost exon 6,                          mesoderm (expressing ligand, kitla) notochord <br /> retain function mammalian isoform 2. Thus,                                   (expressing receptor, kita) pineal gland <br /> predicted kitla required melanocyte                                  (expressing ligand receptor). However, the <br /> migration, kitlb required melanocyte                                kita mutant kitla morphants, below, display <br /> survival. These ideas explored examining                                   tail-bud notochord-related phenotypes. kitla mRNA also <br /> expression patterns MO knockdown phenotypes                                 expressed sensory epithelium ear, most <br /> zebra&#65533;?sh kitl co-orthologs.                                                            prominently ventral otic vesicle (Figure 3D, arrow- <br />                                                                                        head). Interestingly, otic expression near where <br /> kitla kitlb Expression Patterns                                                    second zebra&#65533;?sh kit receptor, kitb, expressed [14]. This <br />    To explore kitla kitlb involved                             suggests functional communicative role <br /> establishing embryonic pigment pattern,                                 conserved kitla kitb, no <br /> examined mRNA expression patterns embryonic                               morphological phenotype ear kitla morphants. <br /> development using mount situ hybridization (Figure                               In contrast kitla expression, kitlb expressed in <br /> 3). In mouse, Kitl mRNA expression &#65533;?rst trunk                          regions suggest role promoting melanocyte migra- <br /> at embryonic day 10.5 dorsal somitic dermatome                                  tion trunk. We &#65533;?nd kitlb mRNA expressed the <br /> [22,23]. This expression lateral neural crest                         ventricular proliferation zones, pallium, brain 22 <br /> present approximately time kita mRNA-                                 hpf (Figure 3G 3H, black arrowheads). This region has <br /> positive melanocyte precursors begin migrate. Later, Kitl                        described previously proliferative zone neurons <br /> expressed ectoderm, support melanocytes                          glial cells [25]. As kitla, kitlb expression also <br /> migration skin [23].                                                            detected ear (Figure 3G, red arrow). However, unlike <br />    Consistent role kitla, observe kitla mRNA                       kitla, sections expresssion limited ventral <br /> two groups cells somite trunk 20 hpf                        otic vesicle, sections kitlb expression found <br /> 30 hpf (Figure 3E). Whole mount situ analysis shows                            otic vesicle epithelium (unpublished data). <br /> the kitla-expressing cells appear center somite,                        We observe robust expression, roughly half of <br /> and sections reveal cells adjacent                                 embryos, kitlb cardinal vein plexus 26 hpf (Figure <br /> notochord (Figure 3F). It interesting kitla mRNA                               3G, red arrowhead). These expression results suggest kitlb <br /> expression restricted middle somite;                             expressed locations appropriate for <br /> earlier observations melanocyte migration showed                               signaling melanocytes migration the <br /> melanocytes appear funnel middle somite                           medial lateral pathway trunk. However, 4 dpf, kitlb <br /> they migrate medial pathway [24]. That kitla mRNA                            mRNA expression evident skin (Figure 3J). kitlb <br /> expression overlaps locations melanocyte migration                             expression stage appears intense kitla the <br /> through medial pathway supports notion kitla                           skin stages. These results consistent roles for <br /> involved promoting migration melanocytes                               kitla, kitlb, melanocyte survival. <br />  <br />        PLoS Genetics | www.plosgenetics.org                                     0092                                January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                                    Zebrafish kit ligand Duplication/Evolution <br />  <br />  <br />  <br />  <br /> Figure 3. kitla kitlb Whole Mount In Situ Hybridizations <br /> (A&#8211;F) kitla mRNA expression stages migration. <br /> (A) kitla expression seen 19 hpf presomitic mesoderm tail bud (black arrowhead). <br /> (B) Section 19-hpf tail bud. <br /> (C) High magnification shows expression kitla mRNA pineal gland 26 hpf. <br /> (D) High magnification shows kitla mRNA ear 26 hpf sensory epithelium, pronounced staining ventral otic vesicle (black <br /> arrowhead). <br /> (E) kitla mRNA expressed groups cells horizontal myoseptum middle somite (black arrowheads) trunk beginning 22 <br /> hpf 30 hpf (image 26 hpf). We observe kitla-positive cells dorsal locations posterior somites (red arrowheads). <br /> (F) Cross section trunk shows expression near notochord 26 hpf. <br /> (G H) kitlb mRNA expression stages migration. kitlb expressed ventricles brain (black arrowheads), ear (red arrow), <br /> and cardinal vein plexus (red arrowhead) 24 hpf (G). Cross section brain ventricle shows kitlb expressed cells lining brain ventricles (H). <br /> (I J) kitla kitlb mRNA expression 4 dpf, stage kita-dependent survival. kitla mRNA expressed skin (black <br /> arrowhead) dorsal myotome (red arrowhead) (I). kitlb mRNA expressed faintly skin (black arrowhead) (J). <br /> Scale bars: (A) 100 lm, (B) 25 lm, (C) 100 lm, (D) 50 lm, (E) 100 lm, (F) 25 lm, (G) 100 lm, (H) 10 lm, (I) 25 lm, (J) 25 lm. <br /> nt, neural tube; nc, notochord; pm, presomitic mesoderm <br /> doi:10.1371/journal.pgen.0030017.g003 <br />  <br />  <br /> kitla Is Required Melanocyte Migration Survival                            zebra&#65533;?sh kitla kitlb resembled alternative splice forms <br />   To test kitla kitlb provides ligand                       mammalian Kitl foregoing analysis kitlb <br /> function kita melanocyte development, explored                           expression consistent role melanocyte survival, <br /> their roles embryonic development MO gene                           kitla expression consistent migration survival, <br /> knockdown (Figure 4). Our &#65533;?nding duplicated                               compelled possibility gene corresponded <br />  <br />        PLoS Genetics | www.plosgenetics.org                                 0093                                    January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                                    Zebrafish kit ligand Duplication/Evolution <br />  <br />  <br />  <br />  <br /> Figure 4. kitla Morphant Phenocopies kitab5 Migration <br /> (A) Wild-type embryonic pigment pattern 2 dpf shows melanocytes migrating yolk (red arrowhead). <br /> (B) kitab5 mutants migration phenotype, melanocytes remaining near ear (red arrow) dorsum absent yolk head (black <br /> arrowheads). <br /> (C) Wild-type embryos injected kitla MOs (6.1 ng) exhibit migratory phenotype similar kitab5 melanocytes present near ear (red arrow) <br /> and absent head yolk (black arrowheads). <br /> (D) Wild-type embryos injected kitlb MOs (6.0 ng) indistinguishable wild-type showing melanocytes present yolk (red arrowhead) <br /> by 2 dpf. <br /> (E&#8211;G) RT-PCR morphant embryos shows MO specificity: (E) kitla RT-PCR wild-type, kitla MO, kitlb MO 3 dpf; (F) kitlb RT-PCR wild-type, kitla <br /> MO, kitlb MO 3 dpf; (G) kitla RT-PCR kitla MO 2, 3, 4, 5, 6, 7 dpf, revealing aberrant splice product caused MO dominant <br /> until 5 dpf, wild-type message visible. <br /> (H) Regions embryo used define migrated nonmigrated melanocytes quantitative analysis melanocyte migration. Red areas <br /> indicate nonmigrated melanocytes dorsal lateral stripe hind yolk ear. Green areas define migrated melanocytes on <br /> the head, yolk, ventral yolk sac stripe hind yolk. Note melanocytes migrated positions the <br /> dorsum horizontal myoseptum, region typically melanocytes, scored nonmigrated embryo, melanocyte <br /> that migrates past horizontal myoseptum scored migrated, migration appropriate not. <br /> (I) Quantitative analysis melanocyte migration negative control MOs (6.8 ng), kitab5, kitla MO (6.1 ng). kitla MO embryos display similar loss of <br /> migration kitab5. Mean values 95% confidence interval reported, n &#188; 10. <br /> Scale bars: 150 lm. <br /> doi:10.1371/journal.pgen.0030017.g004 <br />  <br />  <br />  <br /> functionally different mammalian splice forms. To                           multiple protein sequence alignment different <br /> further test hypothesis kitla kitlb required                  DNA sequence confer speci&#65533;?city. RT-PCR <br /> melanocyte migration survival, respectively, designed                        analysis MO-injected embryos showed MO to <br /> splice-blocking MOs targeting E3I3 exon-intron boun-                           gene caused aberrant splicing, joining exon 2 exon 4; <br /> dary speci&#65533;?c gene injected wild-type                         morphant splices exon 3 targeted gene <br /> embryos. We chose boundary kit-                          (Figure 4E kitla Figure 4F kitlb). Sequencing the <br /> binding domain [26,27] conserved region                      PCR product con&#65533;?rmed splice product results an <br />  <br />        PLoS Genetics | www.plosgenetics.org                                 0094                                   January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                            Zebrafish kit ligand Duplication/Evolution <br />  <br />  <br />  <br />  <br /> Figure 5. kitla Morphant Phenocopies kitab5 Survival Phenotype <br /> (A) Wild-type larva 8 dpf. <br /> (B) Higher magnification dorsal melanocyte stripe 8 dpf wild-type larva healthy melanocytes (black arrowhead). <br /> (C) kitla MO shows fewer melanocytes 8 dpf, similar kitab5. <br /> (D) Higher magnification kitla MO showing melanocytes blebbing skin (red arrow), characteristic melanocyte programmed cell death. <br /> (E F) kitlb MO 8 dpf (E) higher magnification (F) kitlb MO 8 dpf, indistinguishable wild-type. <br /> (G) Total melanocyte counts survival phenotypes kitab5 kitla MO similar. <br /> Scale bars: 100 lm. <br /> doi:10.1371/journal.pgen.0030017.g005 <br />  <br />  <br />  <br /> in-frame deletion exon 3 (amino acids 41 59                    result total number melanocytes kitab5. <br /> both kitla kitlb). Furthermore, result speci&#65533;?c,            This indicate migration phenotype kitla is <br /> kitla MOs did interfere kitlb splicing exon 3 and,                sensitive abrogation melanocyte number <br /> conversely, kitlb MOs did interfere kitla splicing               phenotype. <br /> (Figure 4E kitla Figure 4F kitlb).                                  In addition defects migration, melanocytes kitla MO <br />    We examined consequences MO knockdown                       embryos undergo programmed cell death manner similar <br /> zebra&#65533;?sh embryonic melanocyte development. Melanocytes                        kita-null mutants (Figure 5). Wild-type larva four <br /> in wild-type embryos begin migrate neural crest                   distinct stripes 8 dpf (Figure 5A) healthy <br /> at 18 hpf 2 dpf establish dorsal, lateral, ventral,            melanocytes (Figure 5B). Melanocytes kita mutant larvae <br /> yolk sac stripes (Figure 4A). In contrast, melanocytes kitab5              begin die 4 dpf, characteristically blebbing the <br /> mutants fail migrate remain near ear                 skin larvae completely free melanocytes after <br /> dorsal stripe, appearing ventral stripe               12 dpf. Melanocytes kitla morphants begin die after <br /> yolk sac stripe (Figure 4B). Embryos injected kitla MOs (6               4 dpf, fewer melanocytes 8 dpf (Figure 5C), bleb <br /> ng) similar phenotype kitab5 (Figure 4C). Melanocyte                skin kita mutants (Figure 5D). Melanocyte counts <br /> counts regions outlined Figure 4H reveal signi&#65533;?-                    reveal kitla morphants extent of <br /> cantly fewer migrated melanocytes wild-type, similar                  melanocyte loss kita mutants (Figure 5G). Although <br /> kitab5 (Figure 4I). Interestingly, melanocytes remain                 usually melanocytes kita mutants disappear, very <br /> the dorsal regions near ear kitab5 animals,                melanocytes kitla morphant embryos tend survive. <br /> resulting total melanocyte number similar wild-               This possibly partial restoration kitla function, as <br /> type. However, &#65533;?nd slightly higher doses kitla MOs                 MOs tend lose ef&#65533;?cacy time [28,29]. We &#65533;?rst see <br />  <br />       PLoS Genetics | www.plosgenetics.org                             0095                                January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                              Zebrafish kit ligand Duplication/Evolution <br />  <br />  <br />  <br />  <br /> Figure 6. kitla MO Enhances Temperature-Sensitive kit j1e99 Allele <br /> (A) kit j1e99 embryos reared 28 8C appear similar wild-type melanocyte pattern (see Figure 4A wild-type) 3 dpf. <br /> (B) A submaximal dose (0.5 0.8 ng) kitla MOs shows little effect wild-type embryos. <br /> (C) Submaximal dose kitla MOs shows significant migration phenotype kit j1e99. <br /> (D E) Quantitative analysis kit j1e99&#8211;kitla enhancement. Melanocytes counted 3 dpf scored migrated green regions or <br /> nonmigrated regions shown red (Figure 4H). Mean values 95% confidence interval reported (n &#188; 10). <br /> (D) Using metric, wild-type embryos average 94.4 (69% total) migrating melanocytes. A submaximal dose kitla MOs results 16.4 fewer <br /> migrated melanocytes wild-type. kit j1e99 embryos reared 28 8C 3.7 fewer migrated melanocytes. Neither number total melanocytes <br /> or migrating melanocytes kit j1e99 submaximal dose kitla MOs significantly different compared wild-type. <br /> (E) Migration effect defined difference migrating melaocytes compared wild-type. If combined effects kit j1e99 the <br /> submaximal dose kitla MOs additive, expect migration effect  20.1 kit j1e99&#8211;kitla MO larvae ( 16.4 6 3.7). Instead, observe a <br /> migration effect  40.7. The v2 test number migrated melanocytes kit j1e99&#8211;kitla MO larvae expected number reveals <br /> this difference significantly (p , 0.0002) greater additive. <br /> Scale bars: 150 lm. <br /> doi:10.1371/journal.pgen.0030017.g006 <br />  <br />  <br />  <br /> restoration wild-type kitla transcript 3 dpf                     expected additive de&#65533;?cit ( 20.1 melanocytes) (Figure 6E). This <br /> increasing wild-type transcript 7 dpf (Figure 4G),                     result con&#65533;?rms genetic interaction kita kitla. <br /> during stages melanocyte survival kita dependent. <br /> These results indicate kitla promotes migration                  kitla Overexpression Induces Hyperpigmentation <br /> survival melanocytes similar zebra&#65533;?sh kita receptor.                    To explore role kitla establishing the <br />                                                                                melanocyte pattern, examined kitla overexpression in <br /> kitla MOs Enhance kita jle99                                                   wild-type embryos (Figure 7). Accordingly, constructed an <br />    The phenotypic analysis kitla MOs wild-type embryos                   expression vector kitla fusion red &#64258;uorescent <br /> suggests kitla functionally orthologous                         protein, JRed [30]. Injection vector, capped mRNA <br /> mammalian Kitl, ligand kita receptor tyrosine                   vector, wild-type embryos results embryos <br /> kinase. To test kitla kita pathway,               hyperpigmentation body (Figure 7B). To <br /> investigated genetic interaction effects                test increase cell number, <br /> kitla MOs sensitized kita allele (Figure 6). kita jle99             melanocytes counted dorsal stripe. Wild-type <br /> temperature-sensitive allele kita, lesion C-                  embryos average 200 dorsal melanocytes <br /> terminal tyrosine kinase domain (Leu754Pro) [15]. kita jle99 &#65533;?sh               (Figure 7A 7E). Hyperpigmented animals a <br /> are phenotypically wild-type permissive temperature                     dramatic increase number dorsal melanocytes, <br /> (25 8C) defects melanocyte migration                          averaging 300 cells increase greater 50% <br /> survival restrictive temperature (33 8C) similar null              (Figure 7E). The number melanocytes regions also <br /> kita mutant. For purposes, advantage                        appears increased embryos, not <br /> intermediate phenotype 28 8C (Figure 6A), slightly                 quanti&#65533;?ed increase. <br /> fewer melanocytes migrate compared wild-type. This                             In transient injection experiments, typically see <br /> hypomorphic phenotype likely partial kita signaling                  little red &#64258;uorescence cells yolk. <br /> activity used assay testing                     Because JRed kitla expected act dimers <br /> whether genes act genetic pathway kita.                   [30,31], possible that, steric hindrance, kitla <br /> Our analysis shows interaction kita j1e99                 dimerization interferes ability JRed dimerize <br /> kitla MOs synergistic, greater de&#65533;?cit                        correctly visible locations of <br /> migrating melanocytes ( 41.7) mutant ( 3.7                   expression. Whole mount situ analysis shows abundant and <br /> melanocytes) morphant ( 16.4 melanocytes),                     highly mosaic mRNA expression (unpublished data), suggest- <br />  <br />        PLoS Genetics | www.plosgenetics.org                             0096                                  January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                              Zebrafish kit ligand Duplication/Evolution <br />  <br />  <br />  <br />  <br /> Figure 7. kitla Overexpression Causes Hyperpigmentation Wild-type Not kitab5 Embryos <br /> (A) Wild-type larva. <br /> (B) Larva injected kitla::JRed fusion construct shows hyperpigmentation melanocytes covering larger area wild-type. <br /> (C) kita larva. <br /> (D) kitla::JRed injected kitb5 embryos results kitb5 phenotype. <br /> (E) kitla::JRed hyperpigmented larvae melanocytes dorsal stripe wild-type. When kitla expression vector injected kita <br /> embryos, change number melanocytes. All larvae 6 dpf (n &#188; 10). <br /> Scale bars: 150 lm. <br /> doi:10.1371/journal.pgen.0030017.g007 <br />  <br />  <br />  <br /> ing kitla protein expressed embryo                   process. Indeed, expression data 4 dpf that <br /> these experiments.                                                             kitlb involved melanocyte survival. When injected <br />                                                                                kitlb E3I3 MOs block normal splicing kitlb resulting in <br /> kitla-Induced Hyperpigmentation Is kita Dependent                              transcripts (approximately 75%) lacking exon 3 (Figure <br />    To &#65533;?rmly kitla acts kita receptor,                 4F), effect melanocyte phenotypes. kitlb <br /> explored effect overexpressing kitla kitab5 embryos <br />                                                                                morphant embryos signi&#65533;?cantly different wild- <br /> (Figure 7). When kitla expression construct injected <br />                                                                                type melanocyte migration (Figure 4D) survival (Figure <br /> into kitab5 embryos (Figure 7D), melanocytes develop identi- <br />                                                                                5E 5F). Additionally, kitlb morphants appear morpholog- <br /> cally kita-null phenotype (Figure 7C), including an <br />                                                                                ically normal obvious blood defects <span id='am-16' about='obo:NCBITaxon_12939' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3290 obo:NCBITaxon_33090 obo:NCBITaxon_693762 obo:NCBITaxon_131221 obo:NCBITaxon_693766 obo:NCBITaxon_58023 obo:NCBITaxon_241806 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-17' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Anemia Sw." datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-20'  ></span><span id='am-21' property="rdfs:label" content="Anemia" datatype="xsd:string"></span>anemia</span> at <br /> identical number melanocytes dorsal stripes (Figure <br />                                                                                2 dpf (unpublished data). Because rule the <br /> 7E). This consistent notion effects of <br />                                                                                possibility kitlb functional despite ability E3I3 <br /> ligand, kitla, mediated receptor, kita. <br />                                                                                MOs splice exon 3, designed additional kitlb MOs <br /> Together phylogenetic analysis, MO phenocopy <br /> of kitab5, MO enhancement kita j1e99, depend-                   designed block translation. Neither showed an <br /> ence kita kitla-induced hyperpigmentation,                       effect melanocyte migration survival conditions <br /> conclude kitla functional kita ligand zebra&#65533;?sh                  tested. For sensitive test, took advantage of <br /> embryonic melanocyte development kitla promotes                       conditional kita allele injecting kitlb MOs into <br /> all kita-dependent functions embryonic melanocyte                    embryos homozygous kita jle99, examine they <br /> development. The &#65533;?nding kitla MO largely phenocopies                      enhance sensitized kita phenotype. We failed to <br /> the kita mutant tends argue kitla ligand                   detect enhancement kita jle99 kitlb MOs, either <br /> required embryonic melanocyte development,                     independently combined mix (unpublished data). <br /> is dif&#65533;?cult rule small effect kitlb                   Taken together, expression kitlb mRNA, <br /> revealed experiments.                                                   data suggest kitlb does necessary role in <br />                                                                                embryonic melanocyte development. <br /> kitlb Is Not Required Embryonic Melanocyte Patterning <br /> or Survival                                                                    Discussion <br />   The demonstration kitla necessary for <br /> melanocyte migration survival roles kita receptor                     We identi&#65533;?ed explored roles kitl <br /> does preclude possibility kitlb participates              orthologs zebra&#65533;?sh. These co-orthologs notable by <br />  <br />        PLoS Genetics | www.plosgenetics.org                             0097                                  January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                                   Zebrafish kit ligand Duplication/Evolution <br />  <br />  <br />                                                                                       primary cleavage site alters soluble expression the <br />                                                                                       ligand [17,18]. There evidence alternative splice <br />                                                                                       forms, Kitl-1, containing primary secondary <br />                                                                                       cleavage sites, Kitl-2, secondary cleavage <br />                                                                                       site, functioning promoting melanocyte migration <br />                                                                                       survival, respectively [22]. It shown, <br />                                                                                       however, exclusive expression short splice form, <br />                                                                                       lacking exon 6, Kitl-2, results normal adult pigment <br />                                                                                       pattern [18]. Indeed, isoforms appear cleaved at <br />                                                                                       membrane [17], albeit different rates, differences <br /> Figure 8. Stylized Drawing Zebrafish Embryo Displaying mRNA                        functions isoforms quantitative, <br /> Expression Patterns kita kitb Receptor Tyrosine Kinases Their              qualitative. <br /> Candidate Ligands, kitla kitlb                                                       Since zebra&#65533;?sh kita melanocytic functions appear be <br /> Coincident expression noted kita (blue) kitla (green)               molecularly separable, explored possibility the <br /> trunk, tail bud, ear, pineal gland. kitb (red) kitlb (yellow) not <br /> appear coincident. kitlb does appear coincident receptor                migration survival functions partitioned between <br /> for expression brain ventricles cardinal vein plexus. kitb           zebra&#65533;?sh kitl co-orthologs. Although primary cleavage <br /> be coincident kitla expression trunk.                                     site conserved species, unable to <br /> doi:10.1371/journal.pgen.0030017.g008 <br />                                                                                       identify site protein sequence &#65533;?sh <br />                                                                                       sequences examined (see Dataset S1). This entirely <br />                                                                                       surprising given rate evolution ligands. It is <br /> their differences genomic structure, expression pattern,                           possible site exists conserved. <br /> and phenotypes MO knockdown experiments.                                            Our analysis expression patterns, MO knockdown <br />    It clear gene duplications played role                       phenotypes, interaction MO knockdown sensitized <br /> creation large number receptor tyrosine kinases                             kita allele, &#65533;?nding overexpression phenotypes kitla <br /> [32,33]. Less clear role gene duplication generating                     require kita function argue kitla provides the <br /> their protein ligands, evolve rapidly                         necessary ligand roles kita receptor promoting <br /> phylogenetic relationships characterized.                             embryonic melanocyte development, kitlb to <br /> However, instances, receptor&#8211;ligand pairs                            provide little none. These roles include establishing the <br /> could arisen duplication                                 initial cell number, promoting migration, promoting <br /> receptor ligand. Such case Kit/                            survival. Temperature shift experiments kit j1e99 <br /> Kitl Fms/M-Csf. While Kit clearly closely related                      temperature-sensitive allele kit revealed melanocytes <br /> Fms [34], protein sequences Kitl M-Csf dif&#65533;?cult                        require kit signaling 2 dpf migration 2 <br /> to align. However, secondary structures ligands                          dpf survival [15]. The &#65533;?nding kitla required both <br /> indicate that, short-chain helical cytokines, M-Csf                     indicates involved &#65533;?rst promote <br /> the similar Kitl [26]. The recent duplication                            migration 2 dpf later promote <br /> leading kita/kitb kitla/kitlb zebra&#65533;?sh, therefore,                          survival 2 dpf. That migration survival <br /> provides opportunity understanding evolutionary                            defect kitl MOs phenocopy kita null suggests <br /> mechanisms coevolution receptor&#8211;ligand pairs.                              difference time functions to <br />    Both kit receptor ligand appear                                differential expression display ligand 2 dpf. It <br /> duplicated teleost lineage. These duplication events                           likely switch mediated downstream kita <br /> are consistent notion genome duplication                          signaling. We note explored individual <br /> event prior teleost radiation [9&#8211;11]. It                              roles kitla kitlb promoting adult melanocyte develop- <br /> hypothesized gene duplication events followed                          ment, different kita-dependent kita-independent <br /> by relaxed evolutionary selection result changes                         melanocyte populations [41,42]. <br /> in expression function co-orthologs [35]. Preserva-                            Previously, exhaustive noncomplementation screen for <br /> tion copies result subfunctionalization,                          additional alleles kita resulted migration-speci&#65533;?c <br /> whereby multiple functions controlled single ancestral                           alleles lesions near ligand-binding domain, the <br /> gene partitioned gene duplicates [36]. Subfunc-                           single survival-speci&#65533;?c allele lesion intracellular <br /> tionalization gene expression described                               domain [15]. One possible explanation results was <br /> several zebra&#65533;?sh co-ortholog pairs [37&#8211;39]. Similar case                       migration-promoting function kita required <br /> for kitl here, zebra&#65533;?sh mitf gene                                ligand interaction, melanocyte survival function <br /> duplicated, co-ortholog generates transcript                          kita ligand independent. The &#65533;?nding kitla <br /> corresponds splice forms produced                                MO abrogates melanocyte migration survival rules <br /> mammalian Mitf gene [19,40].                                                          possibility. In light results here, more <br />                                                                                       likely intracellular lesion disrupt binding of <br /> kitla Is Functional Ligand kita Melanocyte                                 downstream targets kita survival speci&#65533;?c. It also <br /> Migration Survival                                                                appears separable functions kitla/kita <br />    Examination genetic structure kitla kitlb                        pathway diverge kita, activity kitla. <br /> loci revealed likely splice form degeneration followed <br /> the duplication event, kitla retained exon 6                            Coincident Gene Expression kit&#8211;kitl Pairs <br /> kitlb gene lost exon. Interestingly, presence absence                        We interested expressions zebra&#65533;?sh <br /> of exon 6 mouse transcript regulates inclusion                        kitl co-orthologs evolved duplication event. <br />  <br />        PLoS Genetics | www.plosgenetics.org                                    0098                                January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                       Zebrafish kit ligand Duplication/Evolution <br />  <br />  <br /> In analysis, assume Kitl expression                  include cardinal vein plexus cells lining the <br /> patterns mouse [22,23,43] chick [43] similar           ventricles brain (Figure 8, yellow shading). Although no <br /> of tetrapod/&#65533;?sh ancestral pattern examine               obvious morphological phenotypes seen kitlb mor- <br /> gene expression domains partitioned zebra&#65533;?sh                  phants, sensitive assays, instance, behavior or <br /> Kitl co-orthologs. Thus, Kitl expressed trunk            blood histology, reveal subtle defects. <br /> lateral intermediate mesoderm mouse chick                    We conclude coincident mRNA expression <br /> [23,43,44] dorsal dermomyotome mouse [22].                  patterns kitla kita, similar phenotypes <br /> Kitl weakly expressed brain                 caused mutation MO knockdown, kitla kita <br /> expressed near otic vesicle mesenchyme                 coevolved expression patterns functions, presumably by <br /> the hindbrain surface ectoderm, pro-                    partitioning select expression domains functions from <br /> nounced expression regions lateral ventral otic             ancestral receptor ligand genes. In contrast, except <br /> vesicle [45].                                                             expression ear, little coincident <br />    In composite, early expression domains kitl             expression kitlb kitb embryo. This raises the <br /> genes zebra&#65533;?sh match mouse chick,                possibility kitlb does signal kitb receptor. <br /> that assumed ancestral form. These domains,              By logic, kitlb kitb interact receptors <br /> with domains kit receptors, shown stylized            ligands, related ancestral paralogs. <br /> embryo (Figure 8) representing expression patterns seen                   It interesting know structures kitla <br /> from 18 30 hpf. Additionally, diagram reveals                 kita coevolved maintain effective ligand&#8211;receptor <br /> coincident expression kita kitla, consistant kitla-           interactions and, contrast, kitlb kitb structures <br /> positive cells signaling cells expressing kita. Most impor-            evolved as-yet-unidenti&#65533;?ed receptor ligand <br /> tant, kita expressed migrating melanocytes (Figure 8, blue          partners. <br /> shading trunk) kitla expressed cells               In addition early expression domains, kitla <br /> medial pathway melanocyte migration (green shading                    kitlb mRNAs expressed 4 5 dpf kita- <br /> along trunk). Another location kita kitla                   dependent melanocyte survival. At stage, kita-expressing <br /> coincident tail bud, kita expressed                melanocytes established embryonic melanocyte <br /> notochord [13] (Figure 8, blue) kitla surrounding           pattern. However, kitla (abundantly) kitlb (weakly) <br /> presomitic mesoderm (Figure 8, green shading tail).                    expressed skin restricted to <br /> Additionally, kitla kita expressed pineal             regions adjacent near melanocyte stripes. The presence of <br /> gland (Figure 8, green/blue regions), suggesting            kitla expression interstripe region skin indicates <br /> be interacting promote aspect pineal gland                     gene required (perhaps permissively) for <br /> function development. Such involvement kita                     migration, kitla expression does establish prepattern for <br /> pathway does appear required normal develop-                melanocyte stripe formation. <br /> ment, observe morphological changes                       The partitioning expression domains the <br /> these tissues kitla morphants kitab5 mutants.                zebra&#65533;?sh kit/kitl co-orthologs result 300 million <br /> Moreover, like kitla, expression kita subset           years evolution zebra&#65533;?sh genome duplicated [12]. <br /> expression domains observed mouse [13]. Thus,                       We compare recent duplication kitl <br /> conclude coevolution kita kitla results retained           Xenopus (called Xsl-1 Xsl-2) [21]. In contrast the <br /> complementary expression patterns, following partition                 teleost, Xenopus genes appear contain exon 6 (see <br /> the ancestral kit kitl expression. These expression                   Dataset S1). Overall, difference expression domains <br /> patterns suggest share functional roles embryonic                 Xsl-1 Xsl-2 between <br /> development.                                                              zebra&#65533;?sh kitla kitlb. Like zebra&#65533;?sh, copies appear <br />    Next, examined possibility expression patterns             expressed otic placode skin. Unlike <br /> of kitb kitlb coevolved another. We                zebra&#65533;?sh, Xenopus genes expressed dermo- <br /> previously reported receptor kitb expressed                myotome trunk, Xsl-1 expressed the <br /> Rohon-Beard neurons, trigeminal ganglion, otic                    length trunk Xsl-2 restricted anterior <br /> vesicle [14] (Figure 8, red shading). With possible                   somites. It clear expression domains are <br /> exception otic vesicle, observe kitla                functioning Xenopus relation melanocyte develop- <br /> and kitlb expression, locations appears             ment, reported co-ortholog Kit appears be <br /> coincident expression kitlb. Like kitla, &#65533;?nd kitlb                expressed melanocytes [46,47]. <br /> expressed epithelium lining otic vesicle (Figure 8, <br /> yellow shading). It interesting note mouse, Kitl            kitla-Induced Hyperpigmentation <br /> seems expressed mesenchyme near ear,                     As &#65533;?rst step understanding morphological <br /> than otic vesicle epithelium, kitla            roles kitla melanocyte development, begun to <br /> kitlb. These expression domains clearly homologous,               generate animals overexpress ligand. Following <br /> yet similar positioning raises possibility            injection kitla expression vector, embryos over- <br /> re&#64258;ect homology function. If so,            express kitla hyperpigmented larger <br /> know expression domains evolved different                    melanocytes. The &#65533;?nding hyperpigmentation is <br /> tissues. Since kita expressed ear, remains               dependent kita, kitla MO knockdown <br /> possible ligands signaling kitb          phenotype interaction sensitized temper- <br /> this context.                                                             ature-sensitive kita allele, provides compelling evidence that <br />    We locations kitlb mRNA expression             kitla ligand kita embryonic melanocyte develop- <br /> is coincident kita kitb receptor expression. These areas            ment. It interesting know similar <br />  <br />       PLoS Genetics | www.plosgenetics.org                         0099                                January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                                    Zebrafish kit ligand Duplication/Evolution <br />  <br />  <br /> overexpression kitlb results hyperpigmentation,                          amplify kitla kitlb cDNA adapted T7 RNA polymerase <br /> which shed light kitlb capacity                      recognition sequences 39 end coding sequence: kitla <br />                                                                                     forward primer 59-TCTCGTTCCATATGAAGAAGTCAA-39, reverse <br /> to bind kita transduce signal.                                                primer 59-taatacgactcactataggTCAGATATCCCCACATCTAATGG-39, <br />    As yet, unclear kitla-overexpressing embryos                       kitlb forward primer 59-CACATGAGGGAGGTTAAGATTG-39, and <br /> have melanocytes proliferation precursor                         reverse primer 59-taatacgactcactataggACACCCTAAAGAATCCAGC- <br />                                                                                     CA-39, lowercase letters indicate T7 sequence. <br /> cells, recruitment differentiation additional stem cells,                       Antisense probe synthesized PCR products using <br /> transdifferentiation nonmelanized cells melanocytes,                        DIG-labeled NTPs transcription reaction T7 polymerase. <br /> or protection cell death. Regardless mechanism,                         The sense strand synthesized using T7 adaptors 59 end <br />                                                                                     coding sequenced used negative control. Whole mount <br /> this result suggests normal development, kitla rate                      situ hybridization modi&#65533;?ed previously published <br /> limiting, correct expression necessary                      methods [55] use Abimed situ robot (Abimed In Situ <br /> produce wild-type embryonic pigment pattern. Notably                            Pro; Intavis AG, http://www.intavis.com). Color development was <br /> absent variety large-scale [48] focused [49,50]                    conducted manually. <br />                                                                                        For sectioning, embryos embedded agar [56], 10-lm <br /> mutant screens zebra&#65533;?sh mutants melanocyte                              sections cut using Leica CM 1850 Cryostat (http://www.leica. <br /> hyperpigmentation, increased number cells                           com). <br /> larger cells. It interesting phenotype produced                         MO knockdown analysis. Antisense MOs (Gene Tools, http://www. <br />                                                                                     gene-tools.com) designed reduce functional <br /> kitla DNA injection, expected produce mosaic                          protein expressed embryos [57]. We chose target mRNA splicing <br /> expression kitla, effect entire                            <span id='am-132' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-133' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> test ef&#65533;?cacy MOs RT-PCR analysis [58]. <br /> animal. Our analysis understand role kitla                             The MOs kitla E3I3 (59-CTGGATAACAACACTCACCACTTCT-39) <br />                                                                                     kitlb E3I3 (59-CACATGTATACTTACCACATCCTTT-39) were <br /> melanocyte development ongoing, making                            designed block processing splice donor site exon 3. <br /> stable transgenic lines expressing construct                         Additional MOs designed target kitlb ATG start site (59- <br /> allow explore role kitla/kita signaling cell                        ATGTCTTAACCTCCCTCATGGGGAACAT-39) 59 UTR (59- <br /> number, cell size control, directing melanocyte                           CCTGTAAATGACTGGAGATAATCAC-39). Negative control MO <br />                                                                                     (59-CCTCTTACCTCAGTTACAATTTATA-39) used control <br /> migration.                                                                          nonspeci&#65533;?c, MO effects. MO injection solution 20 <br />                                                                                     lM MO 13 Danieau buffer 1% Phenol Red visual tracer. <br />                                                                                     Embryos injected MO injection solution yolk the <br /> Materials Methods                                                               one- two-cell stage. <br />    Fish rearing stocks. Embryos obtained vitro                     To test ef&#65533;?cacy MO altering transcript kitla and <br /> fertilization reared 14:10-h dark/light cycle egg water              kitlb, RT-PCR conducted embryos injected kitla E3I3 MOs, <br /> constant temperature 28 8C, unless noted. Wild-type                    kitlb E3I3 MOs, negative control MO (Cont. MO). Primers RT- <br />                                                                                     PCR kitla forward primer 59-CACAGTTGCTGCCTATTCCA-39, <br /> &#65533;?sh obtained sjC inbred line sjA/sjC hybrids <br />                                                                                     kitla reverse primer 59-TGAATCCTCCAAACCAGGTC-39, kitlb for- <br /> [51,52]. Mutants kit receptor tyrosine kinase, kit b5 kit j1e99, <br />                                                                                     ward primer 59-ACCTGCTCAGGTGTTTTTGG-39, kitlb reverse <br /> have previously described [15]. <br />                                                                                     primer 59-CATTCTGTCCTCCAGGTCGT-39. A semiquantitative <br />    Cloning kitla kitlb. Candidate zebra&#65533;?sh kitl sequences were <br />                                                                                     measure kitla kitlb MO ef&#65533;?cacy measured pixel <br /> found searching Whole Genome Shotgun Trace Database (http:// <br />                                                                                     intensity calculation ImageJ (Research Services Branch, National <br /> www.ncbi.nlm.nih.gov/Traces/trace.cgi?&amp;cmd&#188;retrieve&amp;val&#188;species_ <br />                                                                                     Institute Mental Health, http://www.rsb.info.nih.gov/ij). <br /> code%3D%22DANIO%20RERIO%22&amp;retrieve&#188;Submit; June 2002)                                 kitla overexpression. kitla cDNA sequence cloned frame into <br /> using tblastn program [43] Kitl protein sequences                     pJRed-N vector (Evrogen, http://www.evrogen.com) produce a <br /> human, mouse, chick, Xenopus, axolotl query. Each query                  C-terminal fusion protein. This resulted expression vector with <br /> sequence appeared produce conserved alignments                        early promoter <span id='am-11' about='obo:NCBITaxon_10358' typeof='owl:Thing obo:NCBITaxon_10357 obo:NCBITaxon_1 obo:NCBITaxon_548681 obo:NCBITaxon_35237 obo:NCBITaxon_10239 obo:NCBITaxon_10292'><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Cytomegalovirus" datatype="xsd:string"></span><span id='am-15'  ></span>cytomegalovirus</span> driving expression the <br /> zebra&#65533;?sh genome, signi&#65533;?cant pair human                  kitla::JRed fusion protein. DNA (100 pg) injected directly wild- <br /> KITLG exon 4, E-values 4e 9 1e 5. This pair hits                 type kit b5 embryos one- two-cell stage analysis, <br /> mapped LN54 radiation <span id='am-173' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-174' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-175' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-176' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-177'  ></span>hybrid</span> panel Chromosomes 25 4                   using 1% Phenol Red tracer. Alternatively, injected 59-capped <br /> and named kitla kitlb, respectively.                                            mRNA (30 pg) vector (mMessage mMachine; Ambion, <br />    To obtain full-length cDNA sequence loci, PCR primers                   http://www.ambion.com). For simplicity, refer embryos injected <br /> were designed amplify candidate kitl                    mRNA vector &#8216;&#8216;kitla injected,&#8217;&#8217; unless <br /> exons embryonic caudal &#65533;?n poly(T)-primed cDNA. RACE                        noted. <br /> was used attain 59 39 ends genes. In case kitlb,              Melanocyte counts. Melanocyte migration determined by <br /> where RACE failed, primers designed candidate open                        counting melanocytes 48 hpf seven distinct regions the <br /> reading frames downstream upstream known exons                    embryo: yolk, near ear, head anterior eye, and <br /> genomic region (assembly z06s024825). Single RT-PCR fragments                       stripes hind yolk. Melanocytes the <br /> were isolated reaction 1% agarose gel electro-                     yolk, head, ventral, yolk sac stripes classi&#65533;?ed migrated <br /> phoresis sequenced.                                                             melanocytes colored green Figure 4H. Those that <br />    Phylogenetic sequence analyses. A multiple sequence align-                   remained near ear dorsal lateral trunk stripes, <br /> ment kit ligand protein sequences generated ClustalW                    marked red Figure 4H, considered nonmigratory. <br /> and adjusted manually (see Dataset S1). Because alignment                Melanocyte survival determined counting total number <br /> protein sequence C-terminal exon 5 ambiguous, Kyte-Doolittle                 melanocytes larvae 4 dpf 11 dpf. Hyperpigmented <br /> hydropathy plots hidden Markov modeling program                             animals kitla overexpression placed 1% epinephrine to <br /> HMMTOP [53] used con&#65533;?rm alignment trans-                         retract melanosomes, dorsal stripe melanocytes counted <br /> membrane domain exon 7. Exons 6 9 omitted                      compared number dorsal melanocytes wild-type <br /> the phylogenetic analysis. Phylogenetic relationships                     embryo. <br /> determined using maximum likelihood method Phylip <br /> package [54]. Bootstrap support topology conducted with <br /> maximum likelihood 1,000 replicates. We built trees using                 Supporting Information <br /> the neighbor joining parsimony methods. Neighbor joining <br /> produced tree zebra&#65533;?sh kitla outgroup &#65533;?sh                 Dataset S1. Multiple Sequence Alignment kit Ligands <br /> genes, parsimony produced tree zebra&#65533;?sh kitlb                   Found doi:10.1371/journal.pgen.0030017.sd001 (83 KB RTF). <br /> outgroup. The maximum likelihood tree presented is <br /> most consistent pattern gene duplication previously <br /> reported teleost lineage require               Accession Numbers <br /> evolutionary changes explain copies &#65533;?sh species.                 GenBank (http://www.ncbi.nlm.nih.gov/Genbank) accession numbers <br />    In situ hybridization. kitla kitlb probe template generated              genes discussed paper kitla (AY929068) kitlb <br /> by PCR ampli&#65533;?cation 24-hpf embryonic cDNA. Primers used                     (AY929069). Protein sequences Entrez protein accession <br />  <br />        PLoS Genetics | www.plosgenetics.org                                  0100                                   January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                                                Zebrafish kit ligand Duplication/Evolution <br />  <br /> numbers human KITLG (NP_000890), mouse Kitl                                            Acknowledgments <br /> (NP_038626), chick Kitl (NP_990461), axolotl Kitl (AAD17253), <br /> Xenopus Xsl-1 (AAH44108), Xenopus Xsl-2 (AAT66411)                               We thank Jonathan Gitlan, Chao Tsung Yang, Eve Mellgren for <br /> found http://www.ncbi.nlm.nih.gov/entrez/query.                                          useful comments manuscript. We thank Scott Higdon for <br /> fcgi?CMD&#188;search&amp;DB&#188;protein.                                                                 help maintaining &#65533;?sh stocks. <br />                                                                                                Author contributions. KAH SLJ conceived designed the <br />    Protein sequences Fugu rubripes (fugu), <span id='am-160' about='obo:NCBITaxon_69293' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_8100 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489922 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_8111 obo:NCBITaxon_33511 obo:NCBITaxon_1490020 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_69292 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_69291'><span id='am-161' property="oboInOwl:hasExactSynonym" content="three-spined stickleback" datatype="xsd:string"></span><span id='am-162'  ></span><span id='am-163' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-164' property="oboInOwl:hasRelatedSynonym" content="Gasterosteus aculeatus Linnaeus, 1758" datatype="xsd:string"></span><span id='am-165' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-166' property="rdfs:label" content="Gasterosteus aculeatus" datatype="xsd:string"></span><span id='am-167' property="oboInOwl:hasExactSynonym" content="three spined stickleback" datatype="xsd:string"></span>Gasterosteus aculeatus</span> <br />                                                                                             experiments, analyzed data, wrote paper. KAH <br /> (stickleback), <span id='am-59' about='obo:NCBITaxon_8090' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_28781 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_8089 obo:NCBITaxon_2759 obo:NCBITaxon_8088 obo:NCBITaxon_47757 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_76071 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665'><span id='am-60' property="oboInOwl:hasExactSynonym" content="Japanese rice fish" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasRelatedSynonym" content="Orizias latipes" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasExactSynonym" content="Japanese medaka" datatype="xsd:string"></span><span id='am-64'  ></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="Poecilia latipes" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="Oryzias latipes" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasExactSynonym" content="medaka" datatype="xsd:string"></span>Oryzias latipes</span> (medaka) searching                     performed experiments. NB, LIZ, SLJ contributed reagents/ <br /> predicted protein database species                              materials/analysis tools. <br /> Ensembl (http://www.ensembl.org). Fugu kitla (NEWSINFR-                                        Funding. This work funded National Institutes Health <br /> UP00000171441), stickleback kitla (ENSGACP00000012357), medaka                              grant GM56988 SLJ. <br /> kitla (ENSORLP00000016705), medaka kitlb (EN-                                              Competing interests. The authors declared competing <br /> SORLP00000016705).                                                                          interests exist. <br />  <br />  <br /> References                                                                                  23. Matsui Y, Zsebo KM, Hogan BL (1990) Embryonic expression a <br /> 1. Little CC (1915) The inheritance black-eyed white spotting mice. Am                    haematopoietic growth factor encoded Sl locus ligand for <br />     Nat 49: 727&#8211;740.                                                                            c-kit. Nature 347: 667&#8211;669. <br /> 2. Geissler EN, Ryan MA, Housman DE (1988) The dominant-white spotting                      24. Raible DW, Wood A, Hodsdon W, Henion PD, Weston JA, et al. (1992) <br />     (W) locus mouse encodes c-kit proto-oncogene. Cell 55: 185&#8211;192.                  Segregation early dispersal neural crest cells embryonic <br /> 3. Qiu FH, Ray P, Brown K, Barker PE, Jhanwar S, et al. (1988) Primary                          zebra&#65533;?sh. Dev Dyn 195: 29&#8211;42. <br />     structure c-kit: Relationship CSF-1/PDGF receptor kinase                    25. Wullimann MF, Puelles L (1999) Postembryonic neural proliferation the <br />     <span id='am-22' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-23' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span>: Oncogenic activation v-kit involves deletion extracellular                    zebra&#65533;?sh forebrain relationship prosomeric domains. Anat <br />     domain C terminus. EMBO J 7: 1003&#8211;1011.                                                 Embryol (Berl) 199: 329&#8211;348. <br /> 4. Orr-Urtreger A, Avivi A, Zimmer Y, Givol D, Yarden Y, et al. (1990)                      26. Zhang Z, Zhang R, Joachimiak A, Schlessinger J, Kong XP (2000) Crystal <br />     Developmental expression c-kit, proto-oncogene encoded W                        structure human stem cell factor: Implication stem cell factor <br />     locus. Development 109: 911&#8211;923.                                                            receptor dimerization activation. Proc Natl Acad Sci U S A 97: 7732&#8211; <br /> 5. Sarvella PA, Russell LB (1956) Steel, new dominant gene house                       7737. <br />     mouse. J Hered 47: 123&#8211;128.                                                             27. Jiang X, Gurel O, Mendiaz EA, Stearns GW, Clogston CL, et al. (2000) <br /> 6. Anderson DM, Lyman SD, Baird A, Wignall JM, Eisenman J, et al. (1990)                        Structure active core human stem cell factor analysis of <br />     Molecular cloning mast cell growth factor, hematopoietin active                binding receptor kit. EMBO J 19: 3192&#8211;3203. <br />     membrane bound soluble forms. Cell 63: 235&#8211;243.                             28. Yan YL, Miller CT, Nissen RM, Singer A, Liu D, et al. (2002) A zebra&#65533;?sh sox9 <br /> 7. Copeland NG, Gilbert DJ, Cho BC, Donovan PJ, Jenkins NA, et al. (1990)                       gene required cartilage morphogenesis. Development 129: 5065&#8211;5079. <br />     Mast cell growth factor maps near steel locus mouse chromosome 10                29. Mellgren EM, Johnson SL (2004) A requirement kit embryonic <br />     deleted number steel alleles. Cell 63: 175&#8211;183.                              zebra&#65533;?sh melanocyte differentiation revealed melanoblast delay. Dev <br /> 8. Huang E, Nocka K, Beier DR, Chu TY, Buck J, et al. (1990) The hematopoietic                  Genes Evol 214: 493&#8211;502. <br />     growth factor KL encoded Sl locus ligand c-kit              30. Shagin DA, Barsova EV, Yanushevich YG, Fradkov AF, Lukyanov KA, et al. <br />     receptor, gene product W locus. Cell 63: 225&#8211;233.                                (2004) GFP-like proteins ubiquitous metazoan <span id='am-147' about='obo:NCBITaxon_superfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-148' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-149' property="rdfs:label" content="superfamily" datatype="xsd:string"></span>superfamily</span>: Evolution of <br /> 9. Amores A, Force A, Yan YL, Joly L, Amemiya C, et al. (1998) Zebra&#65533;?sh hox                     functional features structural complexity. Mol Biol Evol 21: 841&#8211;850. <br />     clusters vertebrate genome evolution. Science 282: 1711&#8211;1714.                       31. Lemmon MA, Pinchasi D, Zhou M, Lax I, Schlessinger J (1997) Kit receptor <br /> 10. Taylor JS, Braasch I, Frickey T, Meyer A, Van Peer Y (2003) Genome                       dimerization driven bivalent binding stem cell factor. J Biol Chem <br />     duplication, trait shared 22000 species ray-&#65533;?nned &#65533;?sh. Genome Res                   272: 6311&#8211;6317. <br />     13: 382&#8211;390.                                                                            32. Rousset D, Agnes F, Lachaume P, Andre C, Galibert F (1995) Molecular <br /> 11. Jaillon O, Aury JM, Brunet F, Petit JL, Stange-Thomann N, et al. (2004)                     evolution genes encoding receptor tyrosine kinase immuno- <br />     Genome duplication teleost &#65533;?sh <span id='am-140' about='obo:NCBITaxon_99883' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_31031 obo:NCBITaxon_1489922 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_32517 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_31028 obo:NCBITaxon_31022 obo:NCBITaxon_7898 obo:NCBITaxon_47144 obo:NCBITaxon_32443 obo:NCBITaxon_41665'><span id='am-141' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasExactSynonym" content="spotted green pufferfish" datatype="xsd:string"></span><span id='am-143'  ></span><span id='am-144' property="rdfs:label" content="Tetraodon nigroviridis" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasRelatedSynonym" content="Tetraodon nigroviridis Marion de Proce, 1822" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Tetraodon nigroviridis</span> reveals early              globulinlike domains. J Mol Evol 41: 421&#8211;429. <br />     vertebrate proto-karyotype. Nature 431: 946&#8211;957.                                        33. Grassot J, Gouy M, Perriere G, Mouchiroud G (2006) Origin molecular <br /> 12. Taylor JS, Van Peer Y, Braasch I, Meyer A (2001) Comparative genomics                    evolution receptor tyrosine kinases immunoglobulin-like domains. <br />     provides evidence ancient genome duplication event &#65533;?sh. Philos                    Mol Biol Evol 23: 1232&#8211;1241. <br />     Trans R Soc Lond B Biol Sci 356: 1661&#8211;1679.                                             34. Mellgren EM, Johnson SL (2002) The evolution morphological complex- <br /> 13. Parichy DM, Rawls JF, Pratt SJ, Whit&#65533;?eld TT, Johnson SL (1999) Zebra&#65533;?sh                     ity zebra&#65533;?sh stripes. Trends Genet 18: 128&#8211;134. <br />     sparse corresponds orthologue c-kit required                    35. Ohno S (1970) Evolution gene duplication. New York: Springer-Verlag. <br />     morphogenesis subpopulation melanocytes, essential               36. Force A, Lynch M, Pickett FB, Amores A, Yan YL, et al. (1999) Preservation <br />     hematopoiesis primordial germ cell development. Development 126:                         duplicate genes complementary, degenerative mutations. Genetics <br />     3425&#8211;3436.                                                                                  151: 1531&#8211;1545. <br /> 14. Mellgren EM, Johnson SL (2005) kitb, second zebra&#65533;?sh ortholog mouse                37. Serluca FC, Sidow A, Mably JD, Fishman MC (2001) Partitioning tissue <br />     Kit. Dev Genes Evol 215: 470&#8211;477.                                                           expression accompanies multiple duplications Na&#254;/K&#254; ATPase alpha <br /> 15. Rawls JF, Johnson SL (2003) Temporal molecular separation kit                    subunit gene. Genome Res 11: 1625&#8211;1631. <br />     receptor tyrosine kinase&#8217;s roles zebra&#65533;?sh melanocyte migration                   38. Chiang EF, Yan YL, Guiguen Y, Postlethwait J, Chung B (2001) Two Cyp19 <br />     survival. Dev Biol 262: 152&#8211;161.                                                            (P450 aromatase) genes duplicated zebra&#65533;?sh chromosomes ex- <br /> 16. Wehrle-Haller B, Meller M, Weston JA (2001) Analysis melanocyte                          pressed ovary brain. Mol Biol Evol 18: 542&#8211;550. <br />     precursors Nf1 mutants reveals MGF/KIT signaling promotes                       39. Chiang EF, Pai CI, Wyatt M, Yan YL, Postlethwait J, et al. (2001) Two sox9 <br />     directed cell migration independent function cell survival. Dev Biol              genes duplicated zebra&#65533;?sh chromosomes: Expression similar tran- <br />     232: 471&#8211;483.                                                                               scription activators distinct sites. Dev Biol 231: 149&#8211;163. <br /> 17. Huang EJ, Nocka KH, Buck J, Besmer P (1992) Differential expression                 40. Lister JA, Close J, Raible DW (2001) Duplicate mitf genes zebra&#65533;?sh: <br />     processing cell-associated forms kit-ligand: KL-1 KL-2.                   Complementary expression conservation melanogenic potential. <br />     Mol Biol Cell 3: 349&#8211;362.                                                                   Dev Biol 237: 333&#8211;344. <br /> 18. Tajima Y, Moore MA, Soares V, Ono M, Kissel H, et al. (1998) Consequences               41. Johnson SL, Africa D, Walker C, Weston JA (1995) Genetic control adult <br />     exclusive expression vivo Kit-ligand lacking major proteolytic                 pigment stripe development zebra&#65533;?sh. Dev Biol 167: 27&#8211;33. <br />     cleavage site. Proc Natl Acad Sci U S A 95: 11903&#8211;11908.                                42. Rawls JF, Johnson SL (2001) Requirements kit receptor tyrosine <br /> 19. Altschmied J, Delfgaauw J, Wilde B, Duschl J, Bouneau L, et al. (2002)                      kinase regeneration zebra&#65533;?sh &#65533;?n melanocytes. Development 128: <br />     Subfunctionalization duplicate mitf genes associated differential                   1943&#8211;1949. <br />     degeneration alternative exons &#65533;?sh. Genetics 161: 259&#8211;267.                        43. Reedy MV, Johnson RL, Erickson CA (2003) The expression patterns c- <br /> 20. Parichy DM, Stigson M, Voss SR (1999) Genetic analysis steel PG-                 kit Sl chicken embryos suggest unexpected roles genes in <br />     M/versican-encoding gene AxPG candidates white (d) pigmenta-                     somite limb development. Gene Expr Patterns 3: 53&#8211;58. <br />     tion mutant salamander <span id='am-32' about='obo:NCBITaxon_8296' typeof='owl:Thing obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_7711 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7776 obo:NCBITaxon_30367 obo:NCBITaxon_8295 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_8293 obo:NCBITaxon_8294 obo:NCBITaxon_41666 obo:NCBITaxon_8292 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-33' property="oboInOwl:hasExactSynonym" content="axolotl" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="Ambystoma mexicanum" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Ambystoma mexicanum (Shaw &amp; Nodder, 1798)" datatype="xsd:string"></span><span id='am-36'  ></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Ambystoma mexicanum</span>. Dev Genes Evol 209:                  44. Motro B, van der Kooy D, Rossant J, Reith A, Bernstein A (1991) Contiguous <br />     349&#8211;356.                                                                                    patterns c-kit steel expression: Analysis mutations W Sl <br /> 21. Martin BL, Harland RM (2004) The developmental expression                            loci. Development 113: 1207&#8211;1221. <br />     <span id='am-39' about='obo:NCBITaxon_8355' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8342 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_8360 obo:NCBITaxon_2759 obo:NCBITaxon_262014 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8353 obo:NCBITaxon_7776 obo:NCBITaxon_8352 obo:NCBITaxon_30319 obo:NCBITaxon_41666 obo:NCBITaxon_8292'><span id='am-40' property="oboInOwl:hasExactSynonym" content="platanna" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasExactSynonym" content="clawed frog" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="Xenopus laevis" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="African clawed frog" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Xenopus laevis (Daudin, 1802)" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Bufo laevis" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-48'  ></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="Xenopus leavis" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasExactSynonym" content="common platanna" datatype="xsd:string"></span>Xenopus laevis</span> steel homologues, Xsl-1 Xsl-2. Gene Expr Patterns 5: 239&#8211;            45. Steel KP, Davidson DR, Jackson IJ (1992) TRP-2/DT, new early <br />     243.                                                                                        melanoblast marker, shows steel growth factor (c-kit ligand) a <br /> 22. Wehrle-Haller B, Weston JA (1995) Soluble cell-bound forms steel                     survival factor. Development 115: 1111&#8211;1119. <br />     factor activity play distinct roles melanocyte precursor dispersal               46. Kao KR, Bernstein A (1995) Expression Xkl-1, Xenopus gene related to <br />     survival lateral neural crest migration pathway. Development 121:                    mammalian c-kit, dorsal embryonic tissue. Mech Dev 50: 57&#8211;69. <br />     731&#8211;742.                                                                                47. Baker CV, Sharpe CR, Torpey NP, Heasman J, Wylie CC (1995) A Xenopus c- <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                          0101                                       January 2007 | Volume 3 | Issue 1 | e17 <br />                                                                                                                                Zebrafish kit ligand Duplication/Evolution <br />  <br />       kit-related receptor tyrosine kinase expressed migrating stem cells          53. Tusnady GE, Simon I (2001) The HMMTOP transmembrane topology <br />       lateral line system. Mech Dev 50: 217&#8211;228.                                                 prediction server. Bioinformatics 17: 849&#8211;850. <br /> 48.   Kelsh RN, Brand M, Jiang YJ, Heisenberg CP, Lin S, et al. (1996) Zebra&#65533;?sh              54. Felsenstein J (2005) Phylip (Phylogeny Inference Package), version 3.6 <br />       pigmentation mutations proce <br /> </body></html>