<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> GCD14p, a Repressor of GCN4 Translation, Cooperates with Gcd10p <br />     and Lhp1p in the Maturation of Initiator Methionyl-tRNA <br />                    in <span id='am-11' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-12' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-20'  ></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>&#8224; <br />  <br />             Gcd10p and Gcd14p were first identified genetically as repressors of GCN4 mRNA translation in Saccha- <br />           romyces cerevisiae. Recent findings indicate that Gcd10p and Gcd14p reside in a nuclear complex required for <br />           the presence of 1-methyladenosine in tRNAs. Here we show that Gcd14p is an essential protein with predicted <br />           binding motifs for S-adenosylmethionine, consistent with a direct function in tRNA methylation. Two different <br />           gcd14 mutants exhibit defects in cell growth and accumulate high levels of initiator methionyl-tRNA (tRNAMet <br />                                                                                                                      i  ) <br />           precursors containing 5* and 3* extensions, suggesting a defect in processing of the primary transcript. Dosage <br />           suppressors of gcd10 mutations, encoding tRNAMet  i  (hcIMT1 to hcIMT4; hc indicates that the gene is carried <br />           on a high-copy-number plasmid) or a homologue of human La protein implicated in tRNA 3*-end formation <br />           (hcLHP1), also suppressed gcd14 mutations. In fact, the lethality of a GCD14 deletion was suppressed by <br />           hcIMT4, indicating that the essential function of Gcd14p is required for biogenesis of tRNAMet <br />                                                                                                        i  . A mutation in <br />           GCD10 or deletion of LHP1 exacerbated the defects in cell growth and expression of mature tRNAMet   i  in gcd14 <br />           mutants, consistent with functional interactions between Gcd14p, Gcd10p, and Lhp1p in vivo. Surprisingly, the <br />           amounts of NME1 and RPR1, the RNA components of RNases P and MRP, were substantially lower in gcd14 <br />           lhp1::LEU2 double mutants than in the corresponding single mutants, whereas 5S rRNA was present at <br />           wild-type levels. Our findings suggest that Gcd14p and Lhp1p cooperate in the maturation of a subset of RNA <br />           polymerase III transcripts. <br />  <br />  <br />    Eukaryotic cells respond to different conditions of starvation            short upstream open reading frames (uORFs) present in the <br /> and stress by down-regulating the rates of protein biosynthesis.             GCN4 mRNA leader sequence prevent ribosomes from initi- <br /> One aspect of <span id='am-54' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56'  ></span><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="This" datatype="xsd:string"></span>this</span> response in mammalian cells involves <span id='am-117' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-118' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-119'  ></span><span id='am-120' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-121' property="rdfs:label" content="Phos" datatype="xsd:string"></span>phos</span>-                ating translation at the GCN4 start <span id='am-59' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-61'  ></span><span id='am-62' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span> under conditions of <br /> phorylation on serine 51 of the alpha subunit of translation                 amino acid sufficiency. Amino acid starvation triggers activa- <br /> initiation factor 2 (eIF2a) (53). This phosphorylation event                 tion of the protein kinase Gcn2p, which phosphorylates eIF2a <br /> reduces the activity of eIF2B, the guanine nucleotide exchange               and thereby inhibits eIF2-GTP-tRNAMet   i   ternary complex for- <br /> factor for eIF2 that recycles eIF2-GDP to the active form                    mation. This situation specifically favors GCN4 translation be- <br /> eIF2-GTP (20, 40, 46, 59). Only eIF2-GTP can bind initiator                  cause, after translating uORF1, many ribosomes cannot ac- <br /> methionyl-tRNA (tRNAMet     i  ) to produce the eIF2-GTP-                    quire the ternary complex in time to recognize the start codons <br /> tRNAMeti   ternary complex that delivers initiator tRNA to the               at uORF2, -3, and -4 and thus continue scanning downstream. <br /> 43S preinitiation complex; thus, phosphorylation of eIF2 in-                 The majority of these ribosomes rebind the ternary complex by <br /> hibits general protein synthesis initiation. In the budding yeast            the time they reach the GCN4 AUG codon and reinitiate there <br /> <span id='am-28' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-33'  ></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae, eIF2a becomes phosphorylated                       instead (19, 20). <br /> when cells are deprived of an amino acid or purine, leading to                  Recessive mutations in any of the genes encoding the four <br /> depletion of ternary complex levels, just as occurs in mamma-                essential subunits of eIF2B, GCD1, GCD2, GCD6, and GCD7, <br /> lian cells (19, 31). Interestingly, it also specifically stimulates          mimic the effects of eIF2 phosphorylation and derepress <br /> translation of the distinctive mRNA encoding Gcn4p, a tran-                  GCN4 mRNA translation in the absence of Gcn2p and amino <br /> scriptional activator of more than 40 genes involved in amino                acid starvation (29, 30). These gcd mutations also lead to un- <br /> acid biosynthesis (29), thereby alleviating the nutrient limita-             conditional slow growth or temperature sensitivity on nutrient- <br /> tion conditions that trigger phosphorylation of eIF2a in yeast.              rich medium, phenotypes that result from defects in general <br />    The molecular mechanism that couples GCN4 translation to                  translation initiation (9, 16, 22, 27). It is thought that gcd1, <br /> amino acid availability has been explored in great detail over               gcd2, gcd6, and gcd7 mutations impair the ability of eIF2B to <br /> past several years (reviewed in references 31 and 32). Four <br />                                                                              recycle eIF2-GDP to eIF2-GTP and thereby decrease the rate <br />                                                                              of ternary complex formation in vivo (31, 46). This causes <br />                                                                              lower rates of translation initiation on most mRNAs but leads <br />   * Corresponding author. Mailing address: Instituto de Microbio- <br />                                                                              to increased translation of GCN4 mRNA, which is inversely <br /> log&#305;&#180;a Bioqu&#305;&#180;mica del CSIC/Universidad de Salamanca, Edificio De- <br /> partamental de Biolog&#305;&#180;a, Campus Miguel de Unamuno, 37007                    coupled to the availability of ternary complexes in the cell (20). <br /> Salamanca, Spain. Phone: 34-923-121673. Fax: 34-923-224876. E-mail:             Recessive mutations in GCD10 also lead to temperature- <br /> tamame@gugu.usal.es.                                                         sensitive growth and constitutive derepression of GCN4 trans- <br />   &#8224; M.T. dedicates this paper to the memory of Toma&#180;s Santos.                lation in the absence of Gcn2p (26, 42). GCD10 is an essential <br />  <br />                                                                       4167 <br />  4168       CALVO ET AL.                                                                                                                         MOL. CELL. BIOL. <br />  <br />  <br /> gene and was shown to encode a 62-kDa protein that copuri- <br /> fied and coimmunoprecipitated with subunits of eIF3 (23). <br /> eIF3 stimulates several steps of the initiation pathway in mam- <br /> malian cells (among others, binding of ternary complexes to <br /> 40S ribosomes [47]). Accordingly, we proposed that gcd10 mu- <br /> tations might derepress GCN4 translation by decreasing the <br /> ability of eIF3 to promote rebinding of ternary complexes to <br /> 40S subunits that have translated uORF1 and continued scan- <br /> ning downstream on the GCN4 mRNA leader. This would <br /> allow these subunits to skip uORF4 and reinitiate translation <br /> at the GCN4 AUG without any reduction in the level of ter- <br /> nary complex formation (23). More recently, we found that <br /> gcd10 mutations reduce the expression of mature tRNAMet       i   at <br /> a posttranscriptional step and that the phenotypes of gcd10 <br /> mutants are suppressed by multiple copies of the genes IMT1 <br /> to IMT4, encoding tRNAMet    i    (3). Interestingly, gcd10 mutants <br /> lack 1-methyladenosine (m1A) at position 58 in tRNAMet      i   and <br /> in all other tRNAs (17 or more) containing this modified base. <br /> The absence of m1A seems to impair specifically the expression <br /> of mature tRNAMet  i    through degradation of the unprocessed <br /> precursors containing 59 and 39 extensions (3). The reduction <br /> in mature tRNAMet  i    levels in gcd10 mutants should diminish <br /> ternary complex formation, decreasing the rate of general <br /> translation initiation and producing constitutive derepression <br /> of GCN4 translation (20). Thus, the impaired maturation of <br /> tRNAMeti    can also explain the known phenotypes of gcd10 <br /> mutants (3). <br />    We previously identified GCD14 and GCD15 as novel genes                             FIG. 1. Analysis of the GCD14 coding region. A partial restriction map of the <br /> in S. cerevisiae that are required for translational repression of                  GCD14 region is shown in the center, indicating the positions of the two ORFs, <br />                                                                                     GCD14 and SPT10 (44), identified in the plasmid isolated from the genomic <br /> GCN4 mRNA under amino acid-replete conditions (17). Here                            library bearing GCD14 (pRC5). The direction of transcription is indicated by the <br /> we report the molecular cloning of GCD14 and show that                              arrows. The top line depicts the genomic DNA insert in pRC5, and below that <br /> it encodes an essential protein containing binding motifs for                       are depicted the fragments present in subclones constructed from pRC5 in <br /> S-adenosylmethionine (S-AdoMet). We found that gcd14 mu-                            low-copy-number vector pRS316 (pRC55 and pRC56). The ability of each sub- <br />                                                                                     clone to complement the phenotypes of gcd14-1 and gcd14-2 mutants is shown on <br /> tants have reduced levels of mature tRNAMet     i   and accumulate                  the right. Below the map of the GCD14 region are shown enlargements of the <br /> tRNAMeti   precursors containing 59 and 39 extensions. Similar to                   2.6-kb SpeI-ClaI region in plasmid pRC56 and of the 3.2-kb SpeI-ClaI region in <br /> the case for gcd10 mutants, the presence of IMT1 to IMT4 in                         pRC560 bearing the gcd14::URA3 allele, in which a 1.1-kb URA3 fragment <br /> high copy number suppresses the phenotypes of gcd14-1 and                           replaces ;730 bp of the GCD14 coding sequence. Restriction enzyme sites: B, <br />                                                                                     BamHI; Bg, BglII; C, ClaI; H, HindIII; Hp, HpaI; S, Sau3A; X, XbaI; Sp, SpeI. <br /> gcd14-2 mutants, and overexpression of IMT4 overcomes the <br /> lethality of a gcd14::URA3 deletion. Gcd14p is physically asso- <br /> ciated with Gcd10p in cell extracts (3), and we observed addi- <br /> tive effects of gcd14 and gcd10 mutations on cell growth and                        containing a NotI site (59CGATCCACGGAAAAACGCGGCCGCTAATTAAA <br /> expression of mature tRNAMet    i   . Interestingly, we found that                  TGATTAAC39; the NotI site is underlined, and the GCD14 stop codon is in <br /> gcd14 mutations also lead to reduced expression of other                            boldface). The resulting plasmid was named pRC61. (To avoid methylation of <br />                                                                                     the ClaI site located downstream of the GCD14 gene in plasmid pRC61, it was <br /> tRNAs and the RNA components of RNases P (38) and MRP                               always amplified in the dam mutant <span id='am-75' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-76' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-82' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-90'  ></span>Escherichia coli</span> strain GM119.) (iii) A 2.4-kb <br /> (54) when combined with a deletion of LHP1, encoding a yeast                        ClaI fragment containing all of the GCD14 coding sequences was subcloned in <br /> homologue of the human autoantigen La. These findings and                           YCp50 or in the high-copy-number YEp24 vector (50), generating <span id='am-112' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-113' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-114' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>plasmids</span> <br /> others (3) suggest that Gcd10p and Gcd14p functionally coop-                        pRC62 and pRC63, respectively. (iv) A 115-bp NotI fragment, encoding three <br />                                                                                     in-tandem copies of the hemagglutinin (HA) epitope, was inserted at the NotI <br /> erate with Lhp1p to promote the maturation of a subset of                           site in pRC62 and pRC63, to create plasmids pRC64 and pRC65, respectively. <br /> RNA polymerase III transcripts.                                                     Sequence analysis confirmed the in-frame insertion of the HA-coding sequence <br />                                                                                     just upstream of the GCD14 stop codon. <br />                         MATERIALS AND METHODS                                          A 670-bp HpaI-EcoRI fragment from the 39 coding region of the GCD14 gene <br />                                                                                     was fused in frame to the trpE coding sequence in pATH2 (21) to generate <br />    Plasmids. Plasmids constructed in this work are represented in Fig. 1. Plasmid   pTRPC14. Construction of the correct in-frame fusion was confirmed by DNA <br /> pRC5 was isolated from a yeast genomic library in the low-copy-number vector        sequencing. <br /> YCp50 (51) by selecting for complementation of the 3-amino-1,2,4-triazol resis-        Plasmid pJA128 was constructed by inserting the ca. 3.5-kb XbaI-XhoI frag- <br /> tance (3ATr) and Slg2 phenotypes of strain Hm316 (gcd14-2). Plasmids pRC55          ment bearing GCD10 from pMG107 (23) into YCplac111 (24). Plasmid pJA103, <br /> and pRC56 contain, respectively, the 2.3-kb XbaI and the 2.8-kb SpeI-ClaI           containing the lhp1::hisG::URA3::hisG allele, was constructed by PCR amplifi- <br /> fragments from pRC5 subcloned into the corresponding sites of pRS316 (56).          cation of 200- and 350-bp fragments containing, respectively, the 59 and 39 <br />    Plasmid pRC56 was digested with HpaI and BglII to delete an internal 730-bp      flanking sequences at LHP1 by using the appropriate primers. The 59 PCR <br /> fragment of the GCD14 coding sequence, and the BglII end was filled in with         product was digested with EcoRI and BglII and cloned into EcoRI- and BglII- <br /> Klenow fragment. The resulting linearized plasmid was ligated to a blunt-ended      digested pNKY51 (1) to create pJA1039. The 39 PCR product was digested with <br /> 1.1-kb HindIII-HindIII fragment containing the URA3 gene, generating pRC560.        BamHI and SphI and cloned into BamHI- and SphI-digested pJA1039 to create <br />    Plasmid pRC57 was generated by subcloning into pRS306 a ;2.8-kb HpaI-            pJA103. <br /> BamHI fragment from pRC5, extending from 1 kb upstream of the GCD14 ATG                All other plasmids used in this work have been previously described: pE107 <br /> to ;30 nucleotides 59 of the stop codon. Plasmids pRC64 and pRC65 were              (GCD10) (23); p1775 (IMT4) (20); p2632 (IMT1), p2633 (IMT2), p2634 (IMT3), <br /> constructed as follows. (i) The 2.8-kb SpeI-ClaI fragment from pRC56 was            p2635 (IMT4), and p2626(LHP1) (3); pNK985 (1); pRS315 (56); pRS426 (13); <br /> cloned into the corresponding sites of the SK1 polylinker of Bluescript M131        and pKOIII (60). <br /> (Stratagene), generating plasmid pRC6. (ii) A NotI site was introduced upstream        Yeast strain constructions. The genotypes and origins of all yeast strains used <br /> of the GCD14 stop codon by in vitro mutagenesis with a MUTA-GENE in vitro           in this study are summarized in Table 1. Strains Hm295 (gcd14-1), Hm296 <br /> mutagenesis kit (Bio-Rad), using plasmid pRC6 and an oligonucleotide (RC 22)        (gcd14-2), and Hm316 (gcd14-2) were selected as 3ATr and Slg2 ascospores from <br />  VOL. 19, 1999                                                                  Gcd14p, Gcd10p, Lhp1p, AND tRNA PROCESSING IN YEAST                                4169 <br />  <br />  <br />                                                                        TABLE 1. Yeast strains <br />   Strain                                                                   Genotype                                                                Source or reference <br />  <br /> H117              MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 (HIS4::lacZ URA3)                                                                    26 <br /> H160              MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 gcd14-1 (HIS4::lacZ URA3)                                                            17 <br /> H168              MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 gcd14-2 (HIS4::lacZ URA3)                                                            17 <br /> H752              MATa gcn2::LEU2 leu2-3 leu2-112 ura3-52                                                                                          26 <br /> H753              MATa gcn2::LEU2 leu2-3 leu2-112 ura3-52                                                                                          26 <br /> H1515             MATa leu2,3-112 trp1D63 ura3-52                                                                                                  A. G. Hinnebusch <br /> Hm295             MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 gcd14-1 (HIS4::lacZ ura3-52)                                                         17 <br /> Hm296             MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 gcd14-2 (HIS4::lacZ ura3-52)                                                         17 <br /> Hm296g            MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 gcd14-2 (gcd14 URA3)                                                                 This study <br /> Hm296G            MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 gcd14-2 (GCD14 URA3)                                                                 This study <br /> Hm298             MATa gcn2-101 gcn3-101 his1-29 ura3-52 gcd10-505 (HIS4::lacZ ura3-52)                                                            23 <br /> Hm316             MATa gcd2-101 his1-29 ino1 leu2-3 leu2-112 ura3-52 gcd14-2                                                                       17 <br /> Hm397             MATa gcn2-101 gcn3-101 his1-29 ino1 leu2::hisG ura3-52 gcd10-505 gcd14-2                                                         This study <br /> Hm406             MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 (HIS4::lacZ ura3-52) leu2::hisG::URA3::hisG lhp1::LEU2                               This study <br /> Hm407             MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 gcd14-1 (HIS4::lacZ ura3-52) leu2::hisG::URA3::hisG lhp1::LEU2                       This study <br /> Hm408             MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 gcd14-2 (HIS4::lacZ ura3-52) leu2::hisG::URA3::hisG lhp1::LEU2                       This study <br /> F104              MATa ura3-52                                                                                                                     G. Fink <br /> F105              MATa ura3-52                                                                                                                     G. Fink <br /> F294              MATa prt1-1 ura3-52 ade2-1 leu2-3 leu2-112                                                                                       G. Johnston <br /> JAy113            MATa leu2,3-112 trp1D63 ura3-52 lhp1::hisG::URA3::hisG (p2705::GCD10-HA)                                                         This study <br /> JAy206            MATa gcd10D1 ura3-52 trp1 leu2D 1 his3D200 pep4::HIS4 prb1D6 can1 (pJA128::GCD10) <br /> JAy142            MATa gcd10D1 ura3-52 trp1 leu2D1 his3D200 pep4::HIS4 prb1D6 can1(GCD10-HA URA3)                                                  3 <br /> JAy143            MATa gcd10D1 ura3-52 trp1 leu2D1 his3D200 pep4::HIS4 prb1D6 can1 (GCD10 URA3)                                                    3 <br /> W303 1A           MATa ura3-52 leu2-3 leu2-112 trp1-289 his3-11 ade2-1 can1-100                                                                    F. del Rey <br /> W303 1B           MATa ura3-52 leu2-3 leu2-112 trp1-289 his3-11 ade2-1 can1-100                                                                    F. del Rey <br /> YNG1              MATa/a ura3-52 /ura3-52 leu2-3 leu2-112/leu2-3 leu2-112 trp1-289 /trp1-289 his3-11/his3-11 ade2-1/ade2-1                         This study <br />                    can1-100/can1-100 <br /> YNG174            MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 (HIS4::lacZ ura3-52)                                                                 This   study <br /> YNG175            MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 (HIS4::lacZ ura3-52) gcd14-1                                                         This   study <br /> YNG176            MATa gcn2-101 gcn3-101 his1-29 ino1 ura3-52 (HIS4::lacZ ura3-52) gcd14-2                                                         This   study <br /> YRC1              MATa/a gcn2-101/gcn2::LEU2 ura3-52/ura3-52 leu2-3 leu2-112/leu2-3 leu2-112 his1-29/HIS1 ino1/INO1                                This   study <br />                    gcd14-2/GCD14 <br /> YRC2              MATa/a ura3-52/ura3-52                                                                                                           This study <br />  <br />  <br />  <br /> genetic crosses previously described (17). Isogenic Hm296G and Hm296g strains         coding sequence was completely replaced with LEU2 (60) as follows. A Ura2 <br /> were obtained by transforming Hm296 to Ura1 with the integrating plasmid              derivative of strain H117 was selected in 5-<span id='am-1' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Foa" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>FOA</span> medium, and this strain, named <br /> pRC57, containing an incomplete copy of GCD14. Depending on the location of           YNG174 (GCD14), along with Hm295 (gcd14-1) and Hm296 (gcd14-2), was <br /> the crossover, integration of pRC57 yielded a nontandem duplication of wild-          transformed with the 6.5-kb BglII fragment from pNK985 (1) described above to <br /> type GCD14 and the truncated gcd14 allele, or the gcd14-2 and truncated gcd14         disrupt the LEU2 gene in each strain with hisG::URA3::hisG sequences. Leu2 <br /> alleles, separated by plasmid sequences. The former (in Hm296G) led to a              Ura1 transformants were selected and transformed with a 4.6-kb SalI-XbaI <br /> 3AT-sensitive phenotype, whereas the latter (in Hm296g) conferred 3AT resis-          fragment from pKOIII (60) containing the null allele lhp1::LEU2. Total RNA <br /> tance.                                                                                was isolated from Leu1 transformants and analyzed by Northern blotting to <br />    A 3.2-kb SpeI-ClaI fragment from pRC560 containing gcd14::URA3 was used            verify the absence of LHP1 mRNA in isogenic strains Hm406 (GCD14 <br /> for one-step gene disruptions (52) of one of the two GCD14 alleles in diploid         lhp1::LEU2), Hm407 (gcd14-1 lhp1::LEU2), and Hm408 (gcd14-2 lhp1::LEU2). <br /> strain YRC1 or YRC2, constructed by crossing Hm316 with H753 and F104 with               Strain JAy113 was constructed by transforming H1515 (MATa ura3-52 leu2- <br /> F105, respectively (Table 1). Total DNA was isolated from Ura1 transformants          3,112 trp1D63) to Ura1 with a 4.3-kb SphI/EcoRI fragment isolated from pJA103 <br /> and analyzed by Southern blotting to verify that replacement by the null allele       containing lhp1::hisG::URA3::hisG. Disruption of LHP1 was confirmed by PCR <br /> was successful. Heterozygous GCD14/gcd14::URA3 or gcd14/gcd14::URA3 dip-              amplification of fragments diagnostic of lhp1::hisG::URA3::hisG from JAy113 <br /> loids were sporulated, and tetrad analysis was conducted as described in Results.     genomic DNA. Strain JAy206 was constructed by transforming JAy143 (3) to <br />    The diploid strain YNG1 was constructed by crossing W3031A (ura3-52 leu2-          Leu1 with pJA128, bearing single-copy GCD10 and LEU2, and a stable Ura2 <br /> 3,112) with W3031B (ura3-52 leu2-3,112) (Table 1). One allele of GCD14 in             derivative exhibiting wild-type growth was isolated on medium containing 1 mg <br /> YNG1 was replaced by gcd14::URA3 (as described above), and a Ura1 Leu2                of 5-FOA per ml (6). <br /> heterozygous GCD14/ gcd14::URA3 diploid was transformed with the high-copy-              Media and genetic techniques. Sensitivity to 3AT (Sigma; catalog no. A8056) <br /> number LEU2 plasmid p1775 containing IMT4 (20). A Leu1 diploid transfor-              was tested as previously described (28). Transformation of yeast strains was <br /> mant was sporulated, and tetrad analysis was conducted with the following             carried out as described by Ito et al. (34). Standard genetic techniques and media <br /> results: a 4:0 segregation for viability was observed in ascospores of 7 tetrads of   were as previously described (55). <br /> 12 analyzed, showing that p1775 (IMT4) rescued the lethality of a gcd14::URA3            Chromosomal mapping of GCD14. GCD14 was physically mapped to chromo- <br /> deletion.                                                                             some X by using a Chromo-Blot (Clontech), in which yeast chromosomes were <br />    To construct strain Hm397, a gcd10-505 gcd14-2 ascospore clone was isolated        fractionated by clamped homogeneous electrical field electrophoresis and blot- <br /> from a tetratype tetrad obtained from a cross between Hm296 and Hm298 (23).           ted to a nylon membrane. Two individual blot lanes were probed, hybridizing a <br /> The genotypes of all four ascopore clones from this tetrad were transformed with      radiolabeled 2.8-kb HpaI-BamHI fragment from pRC5 that contains most of the <br /> empty <span id='am-98' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-99' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-100' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>vectors</span> or low-copy-number plasmids containing GCD10 or GCD14, fol-             GCD14 coding sequence. The same probe and a radiolabeled 2.4-kb XbaI frag- <br /> lowed by phenotype testing of the transformants. The LEU2 gene was disrupted          ment from pRC55 were used to hybridize a set of filters containing an ordered <br /> in the gcd10-504 gcd14-2 strain by transforming it with a 6.5-kb BglII fragment       lambda library of yeast genomic DNA (49). The first probe hybridized to clone <br /> from pNK985 (1), containing the 3.8-kb hisG::URA3::hisG cassette inserted at          70692, and the second hybridized with the same clone and also with clone 70091, <br /> the EcoRI site of the LEU2 gene, and Leu2 Ura1 transformants were selected.           both bearing inserts corresponding to containing a region encompassing TIF2 <br /> The transformants were plated on SD medium containing 1 mg of 5-fluoro-orotic         and URA2 sequences on in the left arm of chromosome X. To map GCD14 <br /> acid (5-FOA) per ml (6) to select the Ura2 derivative Hm397 (see Table 1 for          genetically, we crossed H168 (gcn2-101 gcn3-101 gcd14-2 ino1) with H753 <br /> genotypes). Hm397 was transformed with low-copy-number plasmids containing            (gcn2::LEU2 GCD14 INO1) and analyzed cosegregation of the 3ATr, Slg2, and <br /> GCD10 (pJA128), GCD14 (pRC56), or empty vectors (pRS315 or pRS316) to                 Ino2 phenotypes in 45 tetrads. We observed 32 parental ditype, 0 nonparental <br /> generate the isogenic transformants with the genotypes GCD10 gcd14-2                  ditype, and 13 tetratype asci, thus locating GCD14 at 14.4 centimorgans from <br /> (Hm420), gcd10-505 GCD14 (Hm421), GCD10 GCD14 (Hm422), and gcd10-505                  INO1. <br /> gcd14-2 (Hm423).                                                                         DNA sequence analysis and cloning of gcd14 alleles. The nucleotide sequence <br />    LHP1 was replaced in GCD14 and gcd14 strains with a null allele in which the       of a 2.8-kb SpeI-ClaI fragment in pRC56 was obtained for both strands by using <br />  4170        CALVO ET AL.                                                                                                                               MOL. CELL. BIOL. <br />  <br />  <br /> a Sequenase version 2.0 kit (U.S. Biochemical Corp). Reverse primer, M13-40 <br /> primer, and several oligonucleotides derived from the previously determined <br /> sequences were used as primers, and pRC56 was used as double-stranded DNA <br /> template. The DNA sequence was analyzed by using the DNAsis sequence <br /> analysis program (Hitachi software), and comparisons of the GCD14 sequence <br /> were made with BLAST programs (2). <br />    The gcd14-1 and gcd14-2 alleles were cloned by PCR with genomic DNAs from <br /> strains H160 and H168, respectively, as templates and the Expand High Fidelity <br /> PCR System (Boehringer Mannheim). The primers consisted of oligonucleotides <br /> having 20 bases corresponding to sequences at the beginning (200 bp upstream <br /> of the ATG) or the end of the GCD14 coding sequence. Amplification products <br /> corresponding to gcd14-1 or gcd14-2 were isolated from two independent PCRs <br /> in each case, subcloned into the pGEM vector (Promega), and sequenced on <br /> both strands by using the appropriate oligodeoxynucleotides derived from the <br /> previously determined GCD14 sequence as primers. <br />    Production of Gcd14p-specific antiserum. The TrpE-Gcd14p fusion protein <br /> encoded in plasmid pTRPC14 was overexpressed in E. coli HB101 and partially <br /> purified from the insoluble fraction of whole-cell extracts by sodium dodecyl <br /> sulfate (SDS)-polyacrylamide gel electrophoresis as already described (21). Gel <br /> slices containing ;200 mg of the fusion protein were used to inoculate rabbits. <br /> Immunization of the rabbits and collection of the immune antisera were con- <br /> ducted by Hazelton Laboratories (Vienna, Va.). <br />    RNA isolation and Northern blot analysis. Total RNA was isolated as previ- <br /> ously described (36). Samples were dried, resuspended in loading buffer (90% <br /> formamide, 1 mM EDTA [pH 8.0], 0.1% bromophenol blue, 0.1% xylene cya- <br /> nol), and heated to .90&#176;C for 10 min before separation on a 6 or 8% polyacryl- <br /> amide-bisacrylamide (19:1)&#8211;8.3 M urea gels by electrophoresis at 250 V for <br /> approximately 3.5 h. Gels were run at 250 V for 20 min prior to separation of                 FIG. 2. Deduced amino acid sequence of Gcd14p. The predicted amino acid <br /> RNAs. Following electrophoresis, gels were soaked in 0.53 Tris-borate-EDTA                 sequence of S. cerevisiae Gcd14p is given in single-letter code. Two regions of <br /> for 20 min, and RNA was transferred to positively charged nylon membranes                  sequence similarity observed in several S-AdoMet-dependent methyltransferases <br /> (Boehringer Mannheim) in the same buffer by electrotransfer (Trans-Blot Cell;              are boxed and indicated as motif I and motif II (35). The conserved G residue at <br /> Bio-Rad) for 3.5 h at 19 V. RNA was immobilized on membranes by UV                         position 5 of motif I is G118 in Gcd14p. A glutamate residue commonly located <br /> cross-linking with a UV-Stratalinker 2400 (Stratagene) according to the manu-              17 to 19 residues C terminal to motif I is found 17 residues C terminal to motif <br /> facturer&#8217;s instructions. tRNAs on the membranes were detected by hybridization             I in Gcd14p, at position 139 (*), and a cluster of hydrophobic residues, hhXh <br /> with radiolabeled oligonucleotides in hybridization buffer (0.25 M Na2HPO4 [pH             (D/E), at positions 135 to 138 (underlined, LFSF) precedes the E element. The <br /> 7.5], 7% SDS, 1% bovine serum albumin, 1 mM EDTA) at 50 to 52&#176;C for 12 to                  central invariant aspartate in motif II is conserved at position 209 in Gcd14p. <br /> 20 h. Oligonucleotides were radiolabeled at the 59 ends with [g-32P]ATP (6,000             Motifs I and II are separated by 83 residues in Gcd14p. A putative nuclear <br /> Ci/mmol) and T4 polynucleotide kinase (Pharmacia). Following hybridization,                localization signal (7) (shaded box) is located between residues K282 R294 in the <br /> membranes were washed once with 23 SSC (13 SSC is 0.15 M NaCl plus 0.015                   Gcd14p sequence. <br /> M sodium citrate)&#8211;0.1% SDS for 30 min at room temperature and once in 13 <br /> SSC&#8211;0.1% SDS at 60&#176;C for 20 min prior to detection by autoradiography. Direct <br /> quantitation of all hybridization signals was conducted by phosphorimager anal- <br /> ysis with a BAS-1500 PhosphorImager and MacBAS Ver.2.x software (Fuji <br /> Film). For reprobing, membranes were stripped of radioactive probes by incu-               we had cloned authentic GCD14, we mapped the chromosomal <br /> bation at 100&#176;C in 1.0% SDS and cooled to room temperature.                                location of the genomic insert in pRC5 to chromosome X of S. <br />    The oligonucleotides used as probes in Northern blots and the corresponding             cerevisiae. A radiolabeled 2.8-kb HpaI-BamHI fragment ob- <br /> transcript <span id='am-69' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> to which they specifically hybridize (RNA) were as follows: (i)         tained from pRC5 hybridized exclusively to chromosome X in <br /> 59-TCGTTTCGATCCCGAGGACATCAGGGTTATGA-39 (tRNAMet                              ), (ii) 59- <br /> TCGGTTTCGATCCCGAGGACATCAGGGTTATGA-39 (tRNAMet <br />                                                                         i <br />                                                                             ), (iii) 59-   a yeast Chromo-Blot (see Materials and Methods). The cloned <br />                                                                        e <br /> TGCTCGAGGTGGGGWTTGAACCCACGACGG-39 (tRNAUAU                         Ile <br />                                                                         ), (iv) 59-CA      sequence was mapped to the left arm of chromosome X, in a <br /> GTTGATCGGACGGGAAAC-39 (5S rRNA), (v) 59-TGCGTTCTTCATCGA                                    region close to SPT10/PBS2/TRK1, by hybridizing the same <br /> TGCGAGAACC-39 (5.8S rRNA), (vi) 59-GACGTCCTACGATTGCACTC-39                                 HpaI-BamHI fragment to filters containing an ordered lambda <br /> (RPR1 RNA), (vii) 59-TCCCCCGGGTGAATCCATGGACCAAGA-39 (NME1 <br /> RNA), (viii) 59-GGTTCATCCTTATGCAGGG-39 (U6 RNA), (ix) 59-AGCCG                             library of yeast genomic clones (see Materials and Methods). <br /> AACTTTTTATTCCATTCG-39 (pre-tRNACGA             Ser <br />                                                    ) (61), and (x) 59-AGCCCAAG             From tetrad analysis we estimated that gcd14-2 is about 14.4 <br /> AGATTTCGAGTCTCTCG-39 (tRNACGA            Ser <br />                                               ) (61).                                      centimorgans from INO1 (see Materials and Methods). The <br />    Nucleotide sequence accession number. The EMBL accession number for the                 fact that INO1 is located 50 kb from the cloned sequence on <br /> GCD14 nucleotide sequence is Z54149. <br />                                                                                            the left arm of chromosome X verifies that the insert in pRC5 <br />                                                                                            contains the wild-type allele of GCD14. <br />                                     RESULTS                                                   To define the boundaries of GCD14, subclones of the <br />                                                                                            genomic insert in pRC5 were constructed in low-copy-number <br />    Isolation and characterization of the wild-type GCD14 gene                              plasmids and tested for complementation of the gcd14-1 and <br /> and of the gcd14-1 and gcd14-2 mutant alleles. Mutations in                                gcd14-2 mutations (Fig. 1). The results of this analysis localized <br /> GCD14 were isolated as suppressors of the 3AT phenotype of                                 GCD14 to the 2.8-kb SpeI-ClaI fragment in pRC56 (Fig. 1 and <br /> a gcn2-101 gcn3-101 double mutant (17). All gcn mutations                                  <span id='am-102' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-103' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-106'  ></span>data</span> not shown), since deletions that remove sequences from <br /> confer sensitivity to 3AT because they impair derepression of                              the 59 or the 39 end (pRC55) of that fragment completely <br /> GCN4, and of histidine biosynthetic genes regulated by Gcn4p,                              abolished complementation activity (Fig. 1). The SpeI-ClaI <br /> in response to histidine starvation imposed by 3AT (28). gcd14                             fragment was sequenced on both strands and found to contain <br /> mutations restore derepression of GCN4 translation and thus                                a single ORF of 1,149 bp that is predicted to encode a protein <br /> confer 3AT resistance in a gcn2 gcn3 background. In addition,                              of 383 amino acids with a molecular mass of 43,917 Da. (Fig. <br /> recessive gcd14 mutations lead to a slow-growth phenotype                                  2). Comparison of the deduced amino acid sequence of <br /> (Slg2) on rich media at 28&#176;C which is exacerbated at both                                  Gcd14p with protein sequences in the GenBank, EMBL, and <br /> lower and higher temperatures (18 and 37&#176;C) (Ts2 phenotype)                                SWIS-PROT databases revealed that the gene had 60% simi- <br /> (data not shown). We cloned the wild-type allele of GCD14                                  larity and 47.5% identity with <span id='am-35' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span><span id='am-39'  ></span>Schizosaccharomyces</span> pombe <br /> from a yeast genomic library on plasmid pRC5 (Fig. 1) by                                   CPD1, a gene proposed to encode the 42-kDa subunit of DNA <br /> complementing the 3ATr and Slg2 phenotypes conferred by                                    polymerase d (62). <br /> gcd14-2 in the gcn2-101 gcd14-2 strain Hm316. To prove that                                   The predicted polypeptide sequence of Gcd14p contains two <br />  VOL. 19, 1999                                                Gcd14p, Gcd10p, Lhp1p, AND tRNA PROCESSING IN YEAST                                   4171 <br />  <br />  <br /> regions of sequence similarity described for several S-AdoMet- <br /> dependent methyltransferases (35). These motifs always occur <br /> in the same <span id='am-72' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> separated by intervals of similar lengths, and <br /> it was suggested that they contribute to binding of the substrate <br /> S-AdoMet or the product S-adenosylhomocysteine of S-Ado <br /> Met-dependent methyltransferases. Motif I (VIEAGTGSG), <br /> located between residues 114 and 122 in Gcd14p (Fig. 2), is <br /> similar to conserved regions described for DNA adenine and <br /> cytosine methyltransferases (33) and was also found in 69 of 84 <br /> non-DNA methyltransferases (35). Gcd14p also contains a <br /> conserved glutamate (E) residue located 17 residues C termi- <br /> nal to motif I and a cluster of hydrophobic residues, LFSF, at <br /> positions 135 to 138 preceding the E element, thus matching <br /> the established consensus hhXh (D/E). Motif II (APWDAI <br />                                                                        FIG. 3. Polysome profiles of isogenic gcd14-2 and GCD14 strains. Isogenic <br /> PH), located between amino acid residues 206 and 213 in             strains Hm296g and Hm296G were cultured overnight in yeast extract-peptone- <br /> Gcd14p (Fig. 2), is present in 46 of the 84 enzyme sequences        dextrose at 23&#176;C and used to inoculate two flasks containing 300 ml of yeast <br /> analyzed (35), including one bacterial tRNA methyltransferase       extract-peptone-dextrose to an optical density at 600 nm of ;0.1. Cultures were <br /> (33). Generally, motifs I and II are separated by 57 6 13 amino     incubated at room temperature in a rotary shaker at 250 rpm. At an optical <br />                                                                     density at 600 nm of ;4.0, cells were collected and resuspended in the appro- <br /> acids, whereas in Gcd14p the separation is 82 residues; how-        priate volume of yeast extract-peptone-dextrose prewarmed to 39&#176;C and grown <br /> ever, the RNA methyltransferases and porphyrin precursor            to an optical density at 600 nm of ;2.0. From both flasks, 150 ml was collected <br /> methyltransferases are known exceptions to this rule.               immediately for the 39&#176;C, t 5 0 samples, and 150 ml of fresh prewarmed yeast <br />    We identified the mutations in gcd14-1 and gcd14-2 by PCR        extract-peptone-dextrose was added back to each flask. Samples of 150 ml were <br />                                                                     collected from both flasks after 1.5 h and processed for polysome analysis by <br /> amplification of both alleles from genomic DNAs of H160 and         velocity sedimentation of whole-cell extracts on sucrose gradients (22). Gradients <br /> H168, respectively, followed by automatic DNA sequencing            were fractionated while being scanned at 254 nm, and the resulting absorbance <br /> (see Materials and Methods). The gcd14-1 allele contains a          profiles are shown, with the tops of the gradients on the left. The positions of 40S, <br /> single point mutation, consisting of a transversion from G to C     60S, and 80S ribosomal species are indicated by arrows. For each strain, the top <br />                                                                     two gradients correspond to samples incubated at 39&#176;C for 0 h (t 5 0) and the <br /> at nucleotide 265, replacing a histidine codon (GAC) with an        two bottom gradients contain samples obtained after 1.5 h of incubation at 39&#176;C. <br /> aspartate codon (CAC) at amino acid residue 89. This muta-          Areas delimited inside the profiles correspond to the fractions of the total <br /> tion falls in a region of ca. 23 amino acid residues that is very   absorbance profiles represented by 2-mer and 3-mer polysomes. In the gcd14-2 <br /> well conserved between CPD1 of S. pombe and GCD14; how-             strain, the 2-mers and 3-mers represented 26% of the polysome mass at the <br />                                                                     permissive temperature but comprised 32% of the polysomes after 1.5 h at 39&#176;C. <br /> ever, it does not alter the predicted S-AdoMet binding motifs       In contrast, 2-mers and 3-mers represented 24 and 25% of the polysome mass at <br /> in Gcd14p. The gcd14-2 allele contains a single G-to-T trans-       23&#176;C and after 1.5 h at 39&#176;C, respectively, in the wild-type strain Hm296G. The <br /> version in nucleotide 3 of the GCD14 coding sequence, replac-       P/M ratios were calculated at t 5 0 and t 5 1.5 h for three independent <br /> ing the GCD14 methionine start codon ATG (Met) with ATT             experiments, and the following mean values and standard errors (in parentheses) <br />                                                                     were obtained: at t 5 0, P/M 5 4.2 (0.83) for GCD14 and 2.8 (0.44) for gcd14-2; <br /> (Ile).                                                              at t 5 1.5 h, P/M 5 5.2 (0.35) for GCD14 and 3.5 (0.72) for gcd14-2. <br />    Immunoblot analysis of whole-cell extracts with antibodies <br /> against Gcd14p (see Materials and Methods) showed that the <br /> gcd14-1 product was expressed at a level greater than or equal <br /> to that of the wild type, whereas the gcd14-2 product was           formed with a URA3 low-copy-number plasmid containing only <br /> substantially diminished in abundance (data not shown). Pre-        the GCD14 coding region (pRC56 [Fig. 1]), and induced to <br /> sumably, a low level of gcd14-2 protein can be produced by          sporulate. Tetrads that contained four viable ascospores were <br /> inefficient translation initiation at the ATT start codon.          dissected, showing that GCD14 complements the lethal phe- <br />    GCD14 is an essential gene. The slow-growth phenotype of         notype of haploid spores bearing the gcd14::URA3 mutation. <br /> the gcd14-1 and gcd14-2 mutations suggests that Gcd14p has an          gcd14-2 cells exhibit a modest defect in general translation <br /> essential function beyond its role in GCN4-specific transla-        initiation. The essential nature of the GCD14 gene together <br /> tional control. To test this possibility, we constructed a dele-    with its role as a repressor of GCN4 mRNA translation (17) <br /> tion-insertion null allele, gcd14::URA3, in plasmid pRC560          suggested to us that Gcd14p could be required for general <br /> (Fig. 1) and used it to replace one allele of GCD14 in ura3-        initiation of translation. To investigate that possibility, we an- <br /> 52/ura3-52 diploid strains YRC1 (gcn2/gcn2 GCD14/gcd14-2            alyzed total polysome profiles from wild-type and mutant <br /> ura3-52/ura3-52) and YRC2 (GCN2/GCN2 GCD14/GCD14                    gcd14-2 strains by fractionating whole-cell extracts on sucrose <br /> ura3-52/ura3-52) (see Materials and Methods). We verified by        gradients by velocity sedimentation. The gcd14-2 mutants do <br /> Southern blot analysis that the gcd14::URA3 allele replaced         not form colonies at 37&#176;C; however, they exhibited little or no <br /> one of the two copies of GCD14 in the Ura1 transformants of         growth defect in liquid medium after several hours at 37&#176;C <br /> YRC1 and YRC2 that were obtained (data not shown). Tetrad           (data not shown). Quantitation of total polysome profiles for <br /> analysis of these transformants revealed that in 24 asci from       gcd14-2 and GCD14 strains grown at 23&#176;C to mid-logarithmic <br /> each diploid, only two of the four spores formed colonies on        phase and then incubated for 4 h at 37&#176;C revealed no signifi- <br /> rich medium after 4 days at 28&#176;C (data not shown). All viable       cant differences in the polysome/monosome (P/M) ratio. Ac- <br /> spores were Ura2 (ura3-52) (and thus contain GCD14), and            cordingly, we next examined polysome profiles for the isogenic <br /> those from the YRC1 transformant were also 3ATs (ura3-52            strains Hm296g (gcd14-2) and Hm296G (GCD14) grown at <br /> gcn2 GCD14), indicating that the gcd14-2 allele had been            39&#176;C, where the gcd14-2 mutant exhibits a more detectable <br /> replaced by gcd14::URA3. These results demonstrate that             growth defect in liquid medium. As shown in Fig. 3, the P/M <br /> GCD14 is essential for growth.                                      ratio was considerably lower in the gcd14-2 mutant (P/M ratio, <br />    To confirm that GCD14 was the only essential gene whose          2.4) than in its isogenic GCD14 parent strain (P/M ratio, 5.7) <br /> function had been disrupted, a Ura2 derivative of the het-          after 1.5 h at 39&#176;C. There was also a small reduction in the <br /> erozygous gcd14::URA3/GCD14 transformant of diploid strain          average size of the polysomes in the gcd14-2 mutant at 39&#176;C <br /> YRC2 was isolated by growth on 5-FOA medium (6), trans-             (see Fig. 3 legend). Although these differences in polysome <br />  4172       CALVO ET AL.                                                                                                                     MOL. CELL. BIOL. <br />  <br />  <br />                                                                                     gcd10 alleles (data not shown). Furthermore, suppression by <br />                                                                                     hcIMT and hcLHP1 seems to be specific for gcd10 and gcd14, <br />                                                                                     as temperature-sensitive mutations in genes encoding subunits <br />                                                                                     of translation initiation factors eIF2 and eIF3 (sui2-1 and <br />                                                                                     prt1-1, respectively) were not suppressed by these dosage sup- <br />                                                                                     pressors (3). <br />                                                                                        gcd14 mutants are defective in processing the primary pre- <br />                                                                                     cursors of the initiator tRNAMet. High-copy-number IMT1, -2, <br />                                                                                     -3, and -4 and LHP1 suppress the phenotypes of gcd10 mutants <br />                                                                                     because these mutants contain reduced amounts of mature <br />                                                                                     tRNAMeti    , and the dosage suppressors compensate for this <br />                                                                                     defect by increasing mature tRNAMet              i   to nearly wild-type <br />                                                                                     (hcLHP1) or higher-than-wild-type (hcIMT) levels (3). Ac- <br />                                                                                     cordingly, we used Northern analysis to investigate whether <br />                                                                                     steady-state levels of mature tRNAMet          i   were also reduced in <br />                                                                                     gcd14 mutants H160 (gcd14-1) and H168 (gcd14-2) compared <br />                                                                                     to that in their parental wild-type strain (H117). The probe for <br />                                                                                     tRNAMeti    , containing sequences complementary to nucleotides <br />                                                                                     33 to 64 in tRNAMet    i     , hybridizes to both mature tRNAMet  i    and <br />                                                                                     different precursor tRNAMet       i    species bearing unique 59-39 ex- <br />                                                                                     tensions encoded by the different IMT genes (3) (see Materials <br />                                                                                     and Methods). The Northern data in Fig. 5 were quantified <br />                                                                                     with a phosphorimager, and the results are given in Table 2. <br />                                                                                     Relative to the level of elongator tRNAMet (tRNAMet              e   ), we <br />                                                                                     calculated that the amounts of mature tRNAMet           i   in the mutant <br />                                                                                     strains were ca. half of that in the wild-type strain both at 28&#176;C <br />                                                                                     and after 1.5 h at 37&#176;C (Fig. 5A; Table 2). In addition, the <br />    FIG. 4. Phenotypes of gcd14 dosage suppressors. Eight independent trans- <br /> formants of strains Hm295 (gcd14-1) and Hm296 (gcd14-2) carrying empty vector       levels of tRNAMet    i     precursors were 2.3- and 3.2-fold higher at <br /> pRS426 (gcd14), a low-copy-number plasmid bearing GCD14 (pRC56), or high-           28 and 37&#176;C, respectively, leading to a fivefold increase in the <br /> copy-number plasmids bearing IMT1 (p2632), IMT2 (p2633), IMT3 (p2634),              ratio of precursor to mature tRNAMet           i   in the gcd14 mutants <br /> IMT4 (p2635), GCD10 (pE107), or LHP1 (p2636) were streaked for single               versus the wild type (Fig. 5A; Table 2). These data strongly <br /> colonies on minimally supplemented SD plates and incubated at 28&#176;C (top <br /> plates) or 37&#176;C (bottom plates) for 2 days. The locations of the transformants on   suggest that Gcd14p is required for efficient processing of the <br /> plates are indicated inside the central schematic.                                  nascent tRNAMet    i      transcripts. It is noteworthy that tRNAMet    i <br />                                                                                     species migrating faster than the wild-type mature form were <br />                                                                                     detected in both gcd14 mutants under all conditions (species f <br />                                                                                     in Fig. 5A), suggesting that fully 59-39-processed tRNAMet          i     is <br /> size and content are not dramatic, they were consistently ob-                       either unstable or incorrectly processed in these mutants. Sim- <br /> served in three independent experiments (see Fig. 3 legend).                        ilar results were obtained for a gcd14-2 mutant by Anderson et <br /> These data suggest that the gcd14-2 mutants exhibit a modest                        al. (3), although the accumulation of pre-tRNAMet         i   at 37&#176;C was <br /> defect in general translation initiation. The magnitude of this                     less extensive than that observed here. <br /> defect is less than that observed in gcd1 or gcd2 mutants,                             As described for gcd10-504 (3), the gcd14-1 and gcd14-2 <br /> consistent with the lesser derepression of GCN4 translation                         mutations had little or no effect on the size or steady-state <br /> associated with gcd14 versus gcd1 or gcd2 mutations (17, 26).                       amount of elongator tRNAMet (Fig. 5B; Table 2). In the case <br />    Genes encoding initiator tRNAMet or Lhp1p are dosage sup-                        of tRNAIle UAU, the gcd14-1 and gcd14-2 mutations led to small <br /> pressors of the Gcd2 and Slg2 phenotypes of gcd14 mutants.                          increases in the amounts of the primary transcript containing <br /> gcd10 and gcd14 mutations lead to constitutive derepression of                      both 59 and 39 extensions plus the intron (slower-migrating <br /> GCN4 translation in the absence of Gcn2p (Gcd2 phenotype),                          species of pretRNAIle       UAU) and possibly a small reduction in the <br /> conferring resistance to 3AT in a gcn2 strain background (17,                       levels of mature tRNAIle         UAU in the mutant strains (Fig. 5C; <br /> 26). We found recently (3) that the 3ATr and Ts2 phenotypes                         Table 2). Consequently, the precursor tRNA/mature tRNA <br /> conferred by gcd10 mutations in a gcn2 background can be fully                      ratios for this tRNA were slightly elevated in the mutant versus <br /> suppressed by the presence in high copy number of any of the                        the wild-type strain (Table 2), albeit not to the extent observed <br /> four genes, IMT1 to IMT4, encoding tRNAMet  i  (11, 14) and can                     for tRNAMet   i  . A similar result was obtained for tRNATyr          GTA <br /> be partially suppressed by high-copy-number LHP1 (39, 60). In                       (data not shown). These results raise the possibility that <br /> view of the physical interaction found between Gcd10p and                           Gcd14p is required primarily for efficient removal of 59 and 39 <br /> Gcd14p (3), we asked whether the dosage suppressors of gcd10                        extensions from initiator tRNAMet but may make a small con- <br /> mutations could also suppress the 3ATr and Slg2 phenotypes                          tribution to the processing of certain other tRNAs besides the <br /> conferred by gcd14 mutations. Each of the four IMT genes on                         initiator. <br /> high-copy-number plasmids (hcIMT) suppressed the 3ATr                                  We confirmed by Northern analysis that the presence of <br /> (data not shown) and Slg2 (Fig. 4) phenotypes in gcd14-1                            hcIMT or hcLHP1 plasmids in the gcd14 mutants led to in- <br /> gcn2-101 and gcd14-2 gcn2-101 mutants to about the same                             creased levels of mature tRNAMet         i   , accounting for their sup- <br /> extent as did a low-copy-number plasmid containing GCD14,                           pressor phenotypes. Transformants of the gcd14-1 strain con- <br /> whereas high-copy-number LHP1 (hcLHP1) only partially sup-                          taining hcIMT1, hcIMT2, hcIMT3, or hcIMT4 had levels of <br /> pressed the mutant Slg2 phenotype (Fig. 4 and data not                              mature tRNAMet    i      that were 3- to 7-fold higher than that ob- <br /> shown). hcGCD10 had no suppressor activity in the gcd14 mu-                         served in a vector transformant and 1.5- to 3.5-fold higher than <br /> tants (Fig. 4), and neither single-copy GCD14 nor hcGCD14                           that seen in the wild-type GCD14 transformant (data not <br /> suppressed the phenotypes of mutants containing different                           shown). High-copy LHP1 led to only a 1.5-fold increase in the <br />  VOL. 19, 1999                                                                    Gcd14p, Gcd10p, Lhp1p, AND tRNA PROCESSING IN YEAST                                    4173 <br />  <br />  <br />                                                                                          wild-type strain (Fig. 6B; Table 3), similar to the results de- <br />                                                                                          scribed above. Compared to the gcd14-2 mutant, the gcd10-505 <br />                                                                                          single mutant at 37&#176;C showed a similar accumulation of the <br />                                                                                          precursors but a somewhat greater reduction in mature tRNAMet    i <br />                                                                                          expression at 37&#176;C (Fig. 6B, lanes 6 to 8). Both mutants showed <br />                                                                                          7- to 10-fold increases in the precursor tRNA/mature tRNA <br />                                                                                          ratios for tRNAMeti    (Table 3). These last findings are interest- <br />                                                                                          ing because they suggest that Gcd10p is also required for <br />                                                                                          efficient end processing of tRNAMet   i   . The gcd10-505 gcd14-2 <br />                                                                                          double mutant at 37&#176;C showed a greater reduction in mature <br />                                                                                          tRNAMeti   levels than did either single mutant, exhibiting levels <br />                                                                                          only 30% of that seen in the wild-type transformant (Fig. 6B, <br />                                                                                          lanes 5 to 8; Table 3). The additivity of these defects in mature <br />                                                                                          tRNAMeti   expression suggests that Gcd10p and Gcd14p partici- <br />                                                                                          pate in the same process involved in pre-tRNAMet    i   processing. <br />                                                                                             Following the temperature shift, the double-mutant culture <br />                                                                                          increased in mass by only about 30% (data not shown), <br />                                                                                          whereas the level of mature tRNAMet    i   was reduced by ca. 69% <br />                                                                                          (Table 3). Therefore, it appears that the reduction in mature <br />                                                                                          tRNAMeti    levels cannot be accounted for by dilution of the <br />                                                                                          preexisting mature molecules during cell growth following the <br />                                                                                          temperature shift. This implies that a substantial fraction of the <br />                                                                                          mature tRNAMet  i    was degraded at 37&#176;C in the double mutant. <br />                                                                                          It is noteworthy that the level of tRNAMet   i    precursors in the <br />                                                                                          double mutant at 37&#176;C, while higher than that in the wild-type <br />                                                                                          strain, was lower than that seen in the single mutants. This may <br />                                                                                          indicate that the unprocessed tRNAMet    i     precursors which ac- <br />                                                                                          cumulate in the double mutant also are degraded more rapidly <br />                                                                                          at 37&#176;C than in the single mutants. Finally, it is interesting that <br />                                                                                          the single and double mutants showed ca. 10-fold-higher levels <br />    FIG. 5. gcd14 mutants are defective in processing the primary precursors of           of the tRNAMeti     precursors at 28&#176;C compared to the wild-type <br /> initiator tRNAMet and tRNAUAU    Ile <br />                                       . Northern blot analysis of total RNA (10 mg) <br /> isolated from strains H117 (GCD14), H160 (gcd14-1), and H168 (gcd14-2) grown <br />                                                                                          strain, even though there was little or no reduction in mature <br /> in yeast extract-peptone-dextrose medium at 28&#176;C to mid-exponential phase (0 h           tRNAMeti   expression under these conditions (Table 3). Perhaps <br /> at 37&#176;C) and shifted to 37&#176;C for 1.5 h is shown. The blot was probed with a              processing of tRNAMet  i   is slower in the mutants at all temper- <br /> radiolabeled oligonucleotide that specifically hybridized to tRNAMet     i    (A) and    atures but this leads to a deficit in mature tRNAMet          levels <br />                                                                                                                                                    i <br /> then was stripped and reprobed with radiolabeled oligonucleotides specific for <br />        Met                Ile <br /> tRNAe (B) or tRNAUAU (C) (see Materials and Methods). The positions of <br /> pre-tRNAMeti   species, mature tRNAMet   i  , tRNAMet <br />                                                     e  , and primary (upper band) <br />                                                                              Ile <br /> and 59- and 39-end-processed intron-containing (lower band) pre-tRNAUAU           and <br />                Ile <br /> mature tRNAUAU are indicated on the left. Indicated on the right are the <br />                          Met <br /> positions of pre-tRNAi containing 59 and 39 extensions encoded by IMT2 and                     TABLE 2. Comparison of tRNAMet <br />                                                                                                                            i  and tRNAUAU  Ile <br />                                                                                                                                                levels <br /> IMT3 (b), pre-tRNAMet  i     containing 59 and 39 extensions encoded by IMT1 and                      in GCD14 and gcd14 isogenic strainsa <br /> IMT4 (c), mature tRNAMet    i   (e), and aberrantly processed tRNAMet  i   species (f) <br /> (see text for details).                                                                                                                     Value for: <br />  <br />                                                                                                                              GCD14              gcd14-1            gcd14-2 <br />                                                                                                Parameter and <br />                                                                                                                               strain             strain             strain <br />                                                                                                    tRNA <br /> level of mature tRNAMet  i   in the gcd14-1 mutant Hm295 at                                                             28&#176;C <br />                                                                                                                                   37&#176;C, <br />                                                                                                                                           28&#176;C <br />                                                                                                                                                      37&#176;C, <br />                                                                                                                                                              28&#176;C <br />                                                                                                                                                                         37&#176;C, <br /> 37&#176;C, reaching a level slightly below that observed in the                                                                        1.5 h              1.5 h              1.5 h <br /> GCD14 strain (data not shown). This last finding is in accor-                            Relative level (% of <br /> dance with the fact that hcLHP1 is a relatively weak suppressor                              wild type at 28&#176;C) <br /> of the Gcd2 (data not shown) and Slg2 (Fig. 4) phenotypes of                              pre-tRNAMet i                100       126      237       320      234       324 <br /> gcd14-1.                                                                                  tRNAMeti                     100        95       48        46       51        46 <br />                                                                                                  Met <br />                                                                                           tRNAe                        100       102      100        82      101       165 <br />    Additive effects of gcd10-504 and gcd14-2 mutations on pro-                            Primary pre-tRNAUAU Ile <br />                                                                                                                        100       172      172       192      167       172 <br /> cessing and accumulation of tRNAMet   i   . In view of the similar                        Intron-containing            100        80      113        76      139        94 <br /> phenotypes of gcd14 and gcd10 mutations, we investigated pos-                                pre-tRNAUAU <br />                                                                                                  Ile <br />                                                                                                         Ile <br />  <br />  <br /> sible genetic interactions between them. The gcd10-505                                    tRNAUAU                      100       115      75         79      101        87 <br /> gcd14-2 double mutant Hm397 was transformed with low-copy- <br />                                                                                          Ratio of primary precur- <br /> number plasmids containing GCD10, GCD14, or empty vectors                                    sors to mature tRNA <br /> to generate four isogenic transformants with the following ge-                            tRNAMeti                        0.03     0.04     0.16      0.23    0.15       0.23 <br /> notypes: (i) gcd10-505 gcd14-2, (ii) GCD10 gcd14-2, (iii) gcd10-                          tRNAUAUIle <br />                                                                                                                           0.05     0.08     0.12      0.13    0.10       0.11 <br /> 505 GCD14, and (iv) GCD10 GCD14. As shown in Fig. 6A, the                                  a <br />                                                                                              The relative intensities of the hybridization signals in the autoradiograms <br /> gcd10-505 gcd14-2 double mutant grew more slowly at 28 and                               shown in Fig. 5 were quantified by phosphorimaging analysis, and the data were <br /> 34&#176;C than did either single mutant, indicating additivity of the                         normalized to the tRNAMet e   signals. The amounts of each tRNA species (pre- <br /> growth defects conferred by these mutations. Northern analy-                             cursor or mature forms) measured in each strain at 28&#176;C or after 1.5 h at 37&#176;C <br />                                                                                          were expressed relative to the corresponding values in the wild-type strain at <br /> sis of the transformants grown at 37&#176;C showed that the gcd14-2                           28&#176;C, which were set to 100%. The ratios of pre-tRNAMet      to mature tRNAMet    , <br /> single mutant contained high levels of tRNAMet  i   precursors and                                               Ile <br />                                                                                          or of primary pre-tRNAUAU     to mature tRNAUAUIle <br />                                                                                                                                                   i                    i <br />                                                                                                                                              are listed in the lower half of <br /> a modest reduction in mature tRNAMet    i    levels relative to the                      the table. <br />  4174       CALVO ET AL.                                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 6. Exacerbation of gcd14-2 phenotypes by a gcd10-505 mutation. (A) The double mutant Hm423 (gcd10-505 gcd14-2) and isogenic GCD10 gcd14-2 (Hm420), <br /> gcd10-505 GCD14 (Hm421), and GCD10 GCD14 (Hm422) strains were streaked for single colonies on minimally supplemented SD plates and incubated at 28, 34, or <br /> 37&#176;C for 2 days. The genotypes of the transformants are indicated adjacent to the appropriate sectors of the 28&#176;C plate. (B) Northern blot analysis of total RNA (10 <br /> mg) isolated from the same strains analyzed in panel A, conducted as described for Fig. 5 except that cells were grown in minimally supplemented SD at 28&#176;C (t 5 0) <br /> (lanes 1 to 4) and then shifted to 37&#176;C for 4 h (lanes 5 to 8). The blot was probed for 5S rRNA, tRNAMet i <br />                                                                                                                         Ile <br />                                                                                                              , and tRNAUAU   as described for Fig. 5. <br />  <br />  <br />  <br /> relative to other stable RNAs only at the high growth rates                         analyzed in Fig. 6B; however, we did not observe additive <br /> occurring at elevated temperatures.                                                 reductions in the levels of mature tRNAIle <br />                                                                                                                             UAU upon combining <br />   Similar to the results shown above for the gcd14 single mu-                       gcd14-2 and gcd10-505 mutations (Table 3). There were also <br /> tants (Fig. 5), we observed modest (15 to 35%) reductions in                        small increases in the amounts of the tRNAIle <br />                                                                                                                                UAU primary tran- <br /> mature tRNAIle UAU expression in the single and double mutants                      script in the single and double mutants at 28&#176;C and in the <br />  <br />  <br />                    TABLE 3. Effects of combining gcd10-505 and gcd14-2 mutations on expression of tRNAMet <br />                                                                                                       i <br />                                                                                                                   Ile <br />                                                                                                           and tRNAUAU a <br />  <br />  <br />                                                                                                         Value for: <br />  <br />            Parameter and tRNA                         gcd10 gcd14 strain                gcd14 strain                    gcd10 strain         GCD10 GCD14 strain <br />  <br />                                                      28&#176;C         37&#176;C, 4 h        28&#176;C         37&#176;C, 4 h            28&#176;C       37&#176;C, 4 h     28&#176;C        37&#176;C, 4 h <br />  <br /> Relative level (% of wild type at 28&#176;C) <br />  Pre-tRNAMet i                                     1,031           535            661           1,236            639             930        100            198 <br />  tRNAMeti                                            145            31            101              60             95              45        100             92 <br />                      Ile <br />  Primary pre-tRNAUAU                                 182            82            164             260            195             216        100            160 <br />                                Ile <br />  Intron-containing pre-tRNAUAU                       104            62            103              98            100              71        100             69 <br />         Ile <br />  tRNAUAU                                              76            85             80              77             82              65        100             95 <br />  <br /> Ratio of primary precursor to mature <br />  tRNA <br />  tRNAMeti                                              0.37           0.88           0.34              1.06           0.35         1.06        0.05           0.11 <br />         Ile <br />  tRNAUAU                                               0.35           0.14           0.30              0.50           0.35         0.50        0.15           0.25 <br />   a <br />     The relative intensities of the hybridization signals in the autoradiograms shown in Fig. 6B were quantified by phosphorimaging analysis, and the data were <br /> normalized to the 5S rRNA signals. The amounts of each tRNA species (precursor or mature forms) measured in each strain at 28&#176;C or after 4 h at 37&#176;C were expressed <br /> relative to the corresponding values in the wild-type strain at 28&#176;C, which were set to 100%. The ratios of pre-tRNAMet <br />                                                                                                                     i   to mature tRNAMet <br />                                                                                                                                       i <br />                                                                                                                                                                 Ile <br />                                                                                                                                           or of primary pre-tRNAUAU <br />                   Ile <br /> to mature tRNAUAU      are listed in the lower half of the table. <br />  VOL. 19, 1999                                                             Gcd14p, Gcd10p, Lhp1p, AND tRNA PROCESSING IN YEAST                               4175 <br />  <br />  <br />  <br />  <br />    FIG. 7. GCD14 is not essential in the presence of hcIMT4. (A) Ascospore clones from two four-spored tetrads (designated 5 and 7) were obtained from a <br /> heterozygous GCD14/gcd14::URA3 diploid (YNG1) containing high-copy-number plasmid p1775 (hcIMT4) (20). The four strains from each tetrad (5A to 5D and 7A <br /> to 7D), all containing hcIMT4, were streaked for single colonies on minimally supplemented SD medium and incubated at 28 or 37&#176;C for 3 days. The position of each <br /> spore clone and its GCD14 genotype is indicated adjacent to the appropriate plate sector. 1, wild-type GCD14; D, gcd14::URA3. (B) Northern blot analysis of total <br /> RNA (10 mg) isolated from the spore clones of tetrad 5 (strains 5A to 5D) bearing hcIMT4 (described for panel A). The strains were grown to mid-logarithmic phase <br /> in minimally supplemented SD medium at 28&#176;C (t 5 0 at 37&#176;C) and shifted to 37&#176;C for 4 h. The blot was probed for tRNAMet i  , tRNAMet <br />                                                                                                                                     e <br />                                                                                                                                                 Ile <br />                                                                                                                                        , or tRNAUAU   as described <br /> for Fig. 5. <br />  <br />  <br />  <br />  <br /> single mutants at 37&#176;C (Fig. 6B), suggesting that processing of                    from one tetrad grew similarly at 28 and 37&#176;C (Fig. 7A, spores <br /> this precursor occurred more slowly in the mutants. Interest-                      7A to 7D). In a second tetrad, however, the two gcd14::URA3 <br /> ingly, the primary tRNAIle UAU precursor did not accumulate                        hcIMT4 strains grew more slowly than did the GCD14 hcIMT4 <br /> in the double mutant at 37&#176;C, perhaps indicating that it is                        strains, particularly at 37&#176;C (Fig. 7A, spores 5A to 5D). Thus, <br /> more susceptible to degradation, as suggested above for pre-                       at least in certain genetic backgrounds (e.g., spores 7C and <br /> tRNAMet i . Thus, as concluded above, the gcd10 and gcd14                          7D), overexpression of tRNAMet   i    from IMT4 makes Gcd14p <br /> mutations have considerably smaller effects on the maturation                      dispensable for growth at 28 and 37&#176;C. <br /> of pre-tRNAIle UAU versus pre-tRNAi <br />                                         Met <br />                                             , particularly in the                     We conducted Northern blot analysis of the eight strains <br /> gcd10-505 gcd14-2 double mutant (Table 3). In fact, as shown                       from the two tetrads just described, all containing hcIMT4. As <br /> next, maturation of tRNAMet i   appears to be the only essential                   shown in Fig. 7B, the gcd14::URA3 hcIMT4 strains accumu- <br /> function of Gcd14p.                                                                lated 3- to 10-fold-higher levels of the pre-tRNAMet      species, <br />                                                                                                                                          i <br />    GCD14 is dispensable in yeast strains overexpressing initi-                     most of which derive from hcIMT4 (3), than did the congenic <br /> ator tRNAMet. Having shown that GCD14 is essential and that                        GCD14 hcIMT4 strains. Thus, it appears that processing of this <br /> gcd14 mutations impair the maturation of pre-tRNAMet      i   , we                 overproduced precursor is impaired in strains lacking GCD14. <br /> asked whether overexpression of an IMT gene would allow <br />                                                                                    The mature tRNAMet   i   was reduced by ca. half at 37&#176;C in only <br /> cells to survive in the absence of Gcd14p. To test this possi- <br />                                                                                    two of the four gcd14::URA3 hcIMT4 strains (spore clones 5D <br /> bility, we replaced one of the two GCD14 alleles in the wild- <br />                                                                                    and 7D) versus the four GCD14 hcIMT4 strains (Fig. 7B and <br /> type diploid strain YNG1 with the gcd14::URA3 deletion-in- <br /> sertion allele and verified the gene replacement by Southern                       data not shown). Thus, in certain genetic backgrounds (spore <br /> blot analysis (data not shown). After sporulation of the resulting                 clones 5B and 7C), overproduction of pre-tRNAMet        i    from <br /> GCD14/gcd14::URA3 diploid, tetrad analysis revealed the ex-                        hcIMT4 completely overcame the requirement for Gcd14p to <br /> pected 21:22 segregation for cell viability, and all viable spores                 express high levels of mature tRNAMet   i  . However, species mi- <br /> were Ura2 (bearing GCD14). When the GCD14/gcd14::URA3                              grating faster than mature tRNAMet   i    were visible in the two <br /> diploid, which is homozygous for leu2, was first transformed                       deletion strains in tetrad 5 (Fig. 7B) and also in tetrad 7 (data <br /> with a high-copy-number LEU2 plasmid p1775 bearing IMT4                            not shown), suggesting that a fraction of the mature fully pro- <br /> (20) and then subjected to tetrad analysis, 20 of 32 tetrads                       cessed tRNAMet i   present in the gcd14::URA3 hcIMT4 strains <br /> showed a 41:02 segregation for cell viability and 21:22 segre-                     may be improperly processed or partially degraded and thus be <br /> gation for the Ura phenotype. Importantly, all of the Ura1                         nonfunctional in translation. The gcd14::URA3 hcIMT4 strains <br /> spores (bearing gcd14::URA3) were Leu1 (bearing hcIMT4 on                          showed little or no defect in expression of elongator tRNAMet <br /> p1775) (data not shown). These results showed that p1775                           and the mature form of tRNAIle     UAU (Fig. 7B); however, there <br /> (hcIMT4) overcame the lethality of the gcd14::URA3 deletion.                       was a modest accumulation of the primary (slower-migrating) <br /> The gcd14::URA3 hcIMT4 and GCD14 hcIMT4 strains derived                            precursor for tRNAIle  UAU, particularly at 37&#176;C, reaching at lev- <br />  4176     CALVO ET AL.                                                                                                            MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />                                                                        FIG. 8. Synthetic interactions of gcd14 and lhp1::LEU2 mutations. (A) <br />                                                                     Strains YNG174 (GCD14), YNG175 (gcd14-1), and YNG176 (gcd14-2) and the <br />                                                                     isogenic lhp1::LEU2-containing derivatives Hm406 (GCD14 lhp1::LEU2), <br />                                                                     Hm407 (gcd14-1 lhp1::LEU2), and Hm408 (gcd14-2 lhp1::LEU2) were streaked <br />                                                                     for single colonies on minimally supplemented SD plates and incubated at 28 or <br />                                                                     37&#176;C for 2 days. The locations of the strains are indicated by their relevant <br />                                                                     genotypes in the schematic at the top. (B) Northern blot analysis of total RNA <br />                                                                     (20 mg) isolated from the strains described for panel A and separated by elec- <br />                                                                     trophoresis in a 6% polyacrylamide&#8211;8.3 M urea gel. Strains were grown in <br />                                                                     minimally supplemented SD at 28&#176;C. The blot was probed for tRNAMet            i   , <br />                                                                            Ile <br />                                                                     tRNAUAU     , tRNAMet <br />                                                                                         e <br />                                                                                                        Ser <br />                                                                                            , and tRNACGA    by using the appropriate oligonucleotides <br />                                                                     (see Materials and Methods). Indicated on the right of the top panel are the <br />                                                                     positions of various precursor and processed forms of tRNAMet i  , as described in <br />                                                                                                                                           Ser <br />                                                                     the legend to Fig. 5. The positions of precursor and mature tRNACGA        species <br />                                                                     are indicated on the right of the bottom panel: a primary precursor containing 59 <br />                                                                     and 39 extensions (g), a processing intermediate containing only the 39 extension <br />                                                                     (h), a 59- and 39-end-processed intron-containing precursor (i), and mature <br />                                                                            Ser <br />                                                                     tRNACGA      (j). (C) The Northern blot in panel B was probed for RPR1 RNA, <br />                                                                     NME1 RNA, 5S rRNA, and U6 RNA by using the appropriate oligonucleotides <br />                                                                     (see Materials and Methods). The positions of precursor and mature RPR1 <br />                                                                     species are indicated on the left. <br /> els ca. 1.5- to 4-fold higher than those seen in GCD14 hcIMT4 <br /> strains (Fig. 7B). <br />    Deletion of LHP1 exacerbates the phenotypes of gcd14 mu- <br /> tations. To investigate the role of Lhp1p in processing of          tions of 77, 86, and 68% in the GCD14, gcd14-1, and gcd14-2 <br /> tRNAMeti    precursors in gcd14 mutants, we disrupted LHP1          strains, respectively (Fig. 8B; Table 4). Lhp1p has been impli- <br /> with the LEU2 gene in wild-type GCD14 and gcd14 mutant              cated in stabilizing precursors and promoting correct 39 end <br /> strains (see Materials and Methods). The lhp1::LEU2 mutation        processing of certain tRNAs (61). Thus, to account for the <br /> did not affect the growth rate of the wild-type strain H117, in     reduced amounts of mature tRNAMet     i   seen in the double mu- <br /> agreement with previous results (60); however, it greatly exac-     tants, we suggest that deletion of LHP1 leads to degradation of <br /> erbated the Slg2 phenotypes of the gcd14-1 and gcd14-2 mu-          a substantial fraction of the tRNAMeti    precursors and that this <br /> tants (Fig. 8A). In addition, we observed additive effects of the   degradation is more extensive in the gcd14 mutants than in the <br /> lhp1::LEU2 and gcd14 mutations on the levels of mature              GCD14 strain. <br /> tRNAMeti   . Whereas the lhp1::LEU2 allele reduced mature              In contrast to the findings for tRNAMet <br />                                                                                                             i   , there were relatively <br /> tRNAMeti    by only ca. 27% in GCD14 cells, it caused reductions    small effects of deletion of LHP1 on the levels of the precursor <br /> of ca. 60% in gcd14-1 and gcd14-2 cells (Fig. 8B; Table 4). The     and mature forms of tRNAIle   UAU in all three strains, and there <br /> levels of tRNAMeti    precursors also were greatly diminished by    was little indication that the lhp1::LEU2 and gcd14 mutations <br /> the lhp1::LEU2 mutation in all three backgrounds, with reduc-       had additive effects on expression of the fully processed <br />  VOL. 19, 1999                                                               Gcd14p, Gcd10p, Lhp1p, AND tRNA PROCESSING IN YEAST                                  4177 <br />  <br />  <br />          TABLE 4. Effects of combining gcd14 and lhp1::LEU2 mutations on expression of tRNAMet <br />                                                                                            i   , tRNAMet <br />                                                                                                      e <br />                                                                                                                Ile <br />                                                                                                          , tRNAUAU , tRNASer <br />                                                                                                                          CGA, <br />                                                    and RPR1, NME1, and U6 RNAsa <br />                                                                                                          Value for: <br />               Parameter and tRNA                        GCD14        GCD14 lhp1::LEU2         gcd14-1      gcd14-1 lhp1::LEU2        gcd14-2      gcd14-2 lhp1::LEU2 <br />                                                          strain           strain               strain             strain              strain             strain <br />  <br /> Relative level (% of wild type at 28&#176;C) <br />  Pre-tRNAMet i                                          100                23                 126                 18                 236                 74 <br />  tRNAMeti                                               100                73                  37                 15                  52                 20 <br />         Met <br />  tRNAe                                                  100                96                 112                114                 120                 92 <br />                      Ile <br />  Primary pre-tRNAUAU                                    100                62                 118                 79                 116                 79 <br />                                Ile <br />  Intron-containing pre-tRNAUAU                          100                96                  58                 61                  91                 71 <br />         Ile <br />  tRNAUAU                                                100               104                  65                 66                  97                 59 <br />  Primary pre-tRNASer CGA                                100                38                  57                 39                  60                 26 <br />  Intermediate precursor                                 100                 &#8212;b                 59                  &#8212;                  56                  &#8212; <br />  Intron-containing pre-tRNASer CGA                      100                61                  49                 24                  30                 33 <br />  tRNASerCGA                                             100                94                  94                 32                  75                 50 <br />  Pre-RPR1                                               100                51                  49                 15                  82                 46 <br />  RPR1                                                   100                73                  68                 24                  60                 52 <br />  NME1                                                   100                61                 100                 28                  98                 28 <br />  U6                                                     100                96                  85                 73                  80                 55 <br />  <br /> Ratio of primary precursor to mature RNA <br />  tRNAMeti                                                 0.08               0.025               0.30                 0.08             0.35               0.31 <br />         Ile <br />  tRNAUAU                                                  0.04               0.03                0.08                 0.05             0.05               0.06 <br />  tRNASerCGA                                               0.28               0.11                0.17                 0.34             0.22               0.15 <br />  RPR1                                                     0.51               0.36                0.36                 0.33             0.70               0.45 <br />    a <br />      The relative intensities of the hybridization signals in the autoradiograms shown in Fig. 8B and C were quantified by phosphorimaging analysis, and the data were <br /> normalized to the 5S rRNA signals. The amounts of each tRNA or RNA species (precursors or mature forms) measured in each strain at 28&#176;C were expressed relative <br /> to the corresponding values in the wild-type strain at 28&#176;C, which were set to 100%. The ratios of pre-tRNAMet <br />                                                                                                            i   to mature tRNAMet <br />                                                                                                                                i <br />                                                                                                                                                         Ile <br />                                                                                                                                    , of primary pre-tRNAUAU  to mature <br />        Ile                          Ser                     Ser <br /> tRNAUAU    , of primary pre-tRNACGA      to mature tRNACGA      , and of pre-RPR1 to mature RPR1 are listed in the lower half of the table. <br />    b <br />      &#8212;, undetectable. <br />  <br />  <br />  <br />  <br /> form of this tRNA (Fig. 8B; Table 4). Thus, similar to our                           effect was especially marked in the gcd14-1 background. As <br /> findings for gcd10 gcd14 double mutants (Fig. 6), expression                         suggested above for tRNAMeti  , Lhp1p may increase the stabil- <br /> of tRNAIle UAU was relatively insensitive to mutations which                         ity of pre-RPR1 RNA, and this function may be critically <br /> showed strong additive effects on the production of mature                           required for production of mature RPR1 in gcd14-1 mutants. <br /> tRNAMeti  . No defects whatsoever in expression of elongator                         Finally, U6 RNA showed modest reductions in both gcd14 <br /> tRNAMet in the single or double mutants relative to that in the                      lhp1::LEU2 double mutants compared to that in the corre- <br /> wild type were observed (Fig. 8B; Table 4). However, we ob-                          sponding single mutants, similar in magnitude to those de- <br /> served a substantial reduction (70%) in the level of mature                          scribed above for tRNAIle UAU (Fig. 8B; Table 4). Taken to- <br /> tRNASerCGA in the gcd14-1 lhp1::LEU2 double mutant compared                          gether, the results in Fig. 8B and C strongly suggest that <br /> to the corresponding gcd14-1 single mutant and an additive                           Gcd14p and Lhp1p cooperate in maturation and accumulation <br /> effect of a lesser degree for this tRNA in the gcd14-2                               of a subset of RNA polymerase III transcripts. In the case of <br /> lhp1::LEU2 double mutant (Fig. 8B; Table 4). Unlike the sit-                         tRNAMeti  , a requirement for Gcd14p can be readily observed <br /> uation for tRNAMet  i   , the amounts of tRNASer   CGA precursors                    in mutants defective for only this protein, whereas for <br /> were decreased, rather than increased, by the gcd14 single                           tRNASerCGA, NME1, and RPR1, a strong dependence on Gcd14p <br /> mutations; however, for both tRNAs, deletion of LHP1 greatly                         is revealed only in the absence of Lhp1p. <br /> reduced the amounts of precursors in both GCD14 and gcd14                               We showed previously that Gcd14p copurified with a poly- <br /> backgrounds. Thus, Gcd14p and Lhp1p may both function in                             histidine-tagged form of Gcd10p on nickel affinity resin and <br /> stabilizing the tRNASer  CGA precursors. It is unknown whether                       that neither protein was stably associated with the PRT1-en- <br /> tRNASer                   1 <br />        CGA contains m A at position 58 (57).                                         coded subunit of eIF3 (3). Moreover, Gcd10p and Gcd14p <br />    We next considered the possibility that expression of other                       were not detected in a highly purified eIF3 complex (48), and <br /> RNA polymerase III transcripts was also reduced in the gcd14                         both proteins showed prominent nuclear localization (3). Thus, <br /> lhp1::LEU2 double mutants. As shown in Fig. 8C, the levels of                        it appears that Gcd10p and Gcd14p reside in a nuclear protein <br /> 5S rRNA were nearly indistinguishable among the gcd14 and                            complex distinct from eIF3. In view of the genetic interactions <br /> lhp1::LEU2 single and double mutants. In contrast, we ob-                            between gcd14 mutations and the lhp1::LEU2 allele, it was of <br /> served strong additive effects of gcd14-1 and lhp1::LEU2 on                          interest to determine whether Lhp1p is tightly associated with <br /> expression of both the RPR1 and NME1 transcripts. These are                          the Gcd10p-Gcd14p complex. To answer this question, we <br /> the RNA components of RNase P and RNase MRP, which are                               investigated whether Lhp1p could be coimmunoprecipitated <br /> involved in the processing of tRNAs and rRNA, respectively                           from cell extracts with an HA-tagged form of Gcd10p. It was <br /> (38, 54). We observed a similar interaction between gcd14-2                          shown previously that the GCD10-HA allele employed for this <br /> and lhp1::LEU2 for NME1 RNA, but a lesser effect for RPR1                            study and wild-type GCD10 were indistinguishable in the abil- <br /> RNA, in this double mutant versus the gcd14-1 lhp1::LEU2                             ity to complement the lethal phenotype of a gcd10::URA3 mu- <br /> strain (Table 4). It is interesting that deletion of LHP1 reduced                    tation (3). Whole-cell extracts were prepared from isogenic <br /> the amount of RPR1 precursor in all three strains, and this                          GCD10-HA or GCD10 strains containing either a high-copy- <br />  4178       CALVO ET AL.                                                                                                                           MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 9. Gcd14p, but not Lhp1p, is tightly associated with Gcd10p in cell extracts. (A) Whole-cell extracts were prepared as described previously (47) from strains <br /> H1515 (LHP1) and YJA113 (lhp1) and from transformants of YJA142 (GCD10HA) or YJA143 (GCD10) bearing high-copy-number plasmid p2626 containing LHP1 <br /> or empty vector YEp24. Aliquots containing 200 mg of total cell protein were mixed with 2 to 4 mg of anti-HA monoclonal antibody HA.11 (Babco), which was prebound <br /> to protein A-Sepharose for 2 h on ice, and the mixture was incubated for 1 h at 4&#176;C. After three washes with lysis buffer (20 mM Tris-HCl [pH 7.4], 100 mM KCl, 1.0 <br /> mM Mg acetate, 0.1% [vol/vol] Triton X-100) containing one complete protease inhibitor tablet (Boehringer Mannheim) per 25 ml, immunoprecipitates were collected <br /> by centrifugation and proteins were eluted by boiling in Laemmli buffer (37). The total proteins immunoprecipitated from 200 mg (pellet [P]; lanes 4, 6, and 8) or 50 <br /> mg of the starting whole-cell extracts (input [I]; lanes 3, 5, and 7) were separated by SDS-polyacrylamide gel electrophoresis (36) and transferred to a nitrocellulose <br /> membrane (Millipore) in 25 mM Tris&#8211;192 mM <span id='am-6' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>glycine</span>&#8211;0.1% SDS containing 20% (vol/vol) methanol. Lanes 1 and 2 contain 50 mg of the starting whole-cell extracts <br /> from strains H1515 and YJA113, respectively, which were included to establish the identity of Lhp1p. The membrane was blocked overnight at 4&#176;C in BLOTTO (5% <br /> [wt/vol] nonfat dry milk, 10 mM Tris-HCl [pH 7.4], 150 mM NaCl, 0.05% [vol/vol] Tween 20). A single immunoblot was probed with 3 mg of rabbit anti-HA antibody <br /> HA.11 (Babco) per ml, stripped, reprobed with a 1:1,000 dilution of anti-Gcd14p antibodies (see Materials and Methods), and reprobed with a 1:1,000 dilution of <br /> anti-Lhp1p antibodies (60). Immune complexes were detected with horseradish peroxidase-conjugated sheep antimouse (Amersham) or donkey antirabbit (Amersham) <br /> secondary antibodies and an enhanced chemiluminescence kit (ECL; Amersham). (B) A longer exposure of lanes 5 and 6 of the blot in panel A probed with anti-Lhp1p <br /> antibodies, included to detect small amounts of Lhp1p in the immunoprecipitates. <br />  <br />  <br />  <br /> number plasmid bearing LHP1 or an empty vector and were                               phenotypes of nonlethal gcd14 mutations also were partially <br /> immunoprecipitated with monoclonal anti-HA antibodies. As                             suppressed by hcLHP1, which increased the level of mature <br /> expected, comparable proportions of Gcd14p and HA-Gcd10p                              tRNAMeti    to just below wild-type levels. Together, these find- <br /> were immunoprecipitated with anti-HA antibodies from the                              ings indicate that the essential function of Gcd14p is required <br /> extracts prepared from strains containing GCD10-HA, but nei-                          for efficient and accurate production of mature tRNAMet  i   . <br /> ther protein was immunoprecipitated from the GCD10 extract                               The fact that gcd14 mutants exhibit tRNAMet         precursor <br />                                                                                                                                          i <br /> (Fig. 9A, lanes 4, 6, and 8). No detectable Lhp1p was coim-                           tRNA/mature tRNA ratios 5- to 10-fold higher than wild type <br /> munoprecipitated with HA-Gcd10p and Gcd14p, even when                                 (Fig. 5; Table 2) is most easily explained by a defect in pre- <br /> Lhp1p was greatly overexpressed (Fig. 9A, lanes 4 and 6, and                          tRNAMet     processing. The precursor molecules that accumu- <br />                                                                                              i <br /> B, lane 6). In accordance with this finding, no Lhp1p was <br />                                                                                       late in gcd14 mutants contain both 59 and 39 extensions, indi- <br /> detectable in a preparation of the Gcd10p-Gcd14p complex <br />                                                                                       cating that both cleavage of the 59 extension by RNase P (18) <br /> purified to homogeneity (48a). We conclude that Lhp1p is not <br />                                                                                       and removal of the 39 extension by endo- or exonucleases occur <br /> tightly associated with the Gcd10p-Gcd14p complex in cell <br /> extracts.                                                                             less efficiently in gcd14 mutants. The accumulation of mole- <br />                                                                                       cules shorter than wild-type mature tRNAMet  i  in gcd14 mutants <br />                                                                                       could indicate that a portion of the 59- and 39-processed <br />                               DISCUSSION                                              tRNAMet     lacks the CCA triplet added posttranscriptionally to <br />                                                                                              i <br />    Evidence that Gcd14p is required for efficient processing of                       the 39 ends of all tRNAs (18). Alternatively, these shorter <br /> tRNAMet <br />       i  . Recessive gcd14 mutations confer constitutive dere-                        molecules could be degradation intermediates. <br /> pression of GCN4 mRNA translation in the absence of eIF2a                                Evidence that Gcd14p and Gcd10p function together to pro- <br /> phosphorylation (17), a phenotype generally indicating re-                            mote maturation of tRNAMet  i    in vivo. Nonlethal gcd10 muta- <br /> duced formation of the eIF2-GTP-tRNAMet      i   ternary complex                      tions lead to reduced expression of mature tRNAMet   i   and can <br /> (20). gcd14 mutants also have a Slg2 phenotype, and we found                          be suppressed by hcIMT and hcLHP1 (3), just as shown here <br /> that general translation initiation is slightly impaired in these                     for gcd14 mutations (Fig. 4). Moreover, the lethal effect of <br /> mutants, at least at elevated temperatures (Fig. 3). In addition,                     deleting GCD10 also could be suppressed by overexpression of <br /> deletion of GCD14 is lethal. These findings suggest that                              tRNAMet     from high-copy-number IMT4 (3). Thus, the essen- <br />                                                                                              i <br /> Gcd14p has an essential function required for ternary complex <br />                                                                                       tial functions of both proteins, at least at low growth temper- <br /> formation in vivo. In agreement with this conclusion, we found <br />                                                                                       atures, are required only for expression of mature tRNAMet     . In <br /> that gcd14 mutants contain approximately 50% lower steady-                                                                                       i <br />  <br /> state levels of mature tRNAMet     than are present in isogenic                       addition to these genetic similarities, we found that Gcd10p <br />                                i <br /> wild-type strains (Tables 2 to 4). They also contain processed                        and Gcd14p can be coimmunoprecipitated (Fig. 9) and copu- <br /> tRNAMet    molecules that appear to be smaller than wild-type                         rified (3) from whole-cell extracts. The physical association <br />        i <br /> mature tRNAMet      (Fig. 5 to 7, species f) and thus may be                          between these two proteins has been confirmed by using a <br />                 i <br /> defective in aminoacylation or ternary complex formation. The                         polyhistidine-tagged form of Gcd10p to isolate a Gcd10p- <br /> Gcd2 and Slg2 phenotypes of the gcd14 mutants were fully                              Gcd14p complex from whole-cell extracts by affinity chroma- <br /> suppressed by introducing multiple copies of the IMT genes                            tography (3) and by yeast two-<span id='am-107' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-108' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-110' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-111'  ></span>hybrid</span> analysis (4). In addition, <br /> encoding tRNAMet  i   , which restored expression of mature                           epitope-tagged forms of both proteins Gcd10p and Gcd14p <br /> tRNAMeti   to greater-than-wild-type levels. Moreover, the pres-                      were shown by immunofluorescence to exhibit prominent nu- <br /> ence of hcIMT4 overcame the lethality of a gcd14 deletion. The                        clear localization in yeast cells (3). These findings strongly <br />  VOL. 19, 1999                                                         Gcd14p, Gcd10p, Lhp1p, AND tRNA PROCESSING IN YEAST                    4179 <br />  <br />  <br /> suggest that Gcd10p and Gcd14p reside in a nuclear complex                   Gcd10p copurified with eIF3 activity (23), as stabilizing the <br /> that functions directly in the maturation of tRNAMet      i    .             ternary complex is a function ascribed to eIF3 (47). <br />    Here we provided additional in vivo evidence for this con-                   Evidence that Lhp1p is required to protect tRNAMet       i   from <br /> clusion by showing that gcd14-2 and gcd10-505 mutations had                  degradation in gcd14 mutants. In otherwise wild-type cells, <br /> additive effects on cell growth and expression of mature                     deletion of LHP1 led to substantially decreased levels of pre- <br /> tRNAMeti  . The double mutant at 28&#176;C accumulated higher                     cursor tRNAMet  i    and to only a small reduction in mature <br /> levels of pre-tRNAMet i   than did either single mutant, suggest-            tRNAMeti    levels (Fig. 8B; Table 4). This could be explained by <br /> ing compounded defects in processing. Moreover, the gcd10-                   postulating a role for Lhp1p in transcription of the IMT genes. <br /> 505 single mutant accumulated tRNAMet       i     precursors concom-         However, in view of the allele-specific interactions between <br /> itant with diminished amounts of mature tRNAMet         i     (Fig. 6B),     gcd14 mutations and the lhp1::LEU2 deletion (Fig. 8B), plus <br /> implicating Gcd10p in pre-tRNAMet    i     processing. In a previous         the evidence that Gcd14p functions in methylation and pro- <br /> study we showed that expression of mature tRNAMet           i      was re-   cessing of pre-tRNAMet  i   , we favor the <span id='am-49' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-50'  ></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>idea</span> that Lhp1p affects <br /> duced by ca. fivefold after several hours at 37&#176;C in a gcd10-504             tRNAMeti     expression posttranscriptionally. As suggested for <br /> mutant; however, we observed no accumulation of pre-                         other yeast tRNAs (61), binding of Lhp1p to the poly(U) <br /> tRNAMeti   at the restrictive temperature in that strain (3). This           stretch at the 39 end of pre-tRNAMet     i   could stabilize the con- <br /> could be explained by proposing that the gcd10-504 and gcd10-                formation needed for precise endonucleolytic 39 cleavage and <br /> 505 (studied here) mutations both lead to defects in processing              block access by 39-59 exonucleases. In the absence of Lhp1p, a <br /> of pre-tRNAMet      but that the unprocessed molecules are more              significant fraction of the pre-tRNAMet    i   would be incorrectly <br />               i <br /> susceptible to degradation in the gcd10-504 mutant than in the               trimmed at the 39 end and subsequently degraded, reducing <br /> gcd10-505 or gcd14 strains analyzed here. In accordance with                 the yield of mature tRNAMet    i    . <br /> this explanation, we obtained evidence that both precursor and                  Deletion of LHP1 in the gcd14 mutants led to larger reduc- <br /> mature tRNAMet       were highly susceptible to degradation in the           tions in both precursor and mature tRNAMet       i   than in GCD14 <br />                 i <br /> gcd10-505 gcd14-1 double mutant at 37&#176;C (Fig. 6B), similar to                cells (Fig. 8B), suggesting a greater need for the putative chap- <br /> our previous findings for the gcd10-504 single mutant (3).                   erone function of Lhp1p in gcd14 versus GCD14 cells. It was <br />    Thus far, we have observed no defect in pre-tRNAMet                syn-   shown previously that a mutation in yeast tRNASer      CGA makes its <br />                                                                  i <br /> thesis or processing in cell extracts prepared from gcd10 or                 processing completely dependent on Lhp1p, and the process- <br /> gcd14 mutants (3a). Therefore, Gcd10p and Gcd14p do not                      ing intermediates appear to be rapidly degraded when Lhp1p <br /> appear to be essential components of the processing machin-                  is depleted from cells (61). It was proposed that Lhp1p was <br /> ery. Our discovery that Gcd14p contains two sequence motifs                  essential to stabilize the conformation of the mutant tRNASer    CGA <br />                                                                              to permit correct endonucleolytic cleavage removal of the 39 <br /> conserved among S-AdoMet-dependent methyltransferases, <br />                                                                              trailer. Similarly, the absence of m1A at position 58 could <br /> including at least one tRNA methyltransferase (35), raised the <br />                                                                              perturb the structure of the initiator tRNAMet in gcd14 mu- <br /> possibility that one or more nucleotides in pre-tRNAMet                 is <br />                                                                     i <br />                                                                              tants and create a stronger requirement for binding of Lhp1p <br /> undermethylated in gcd14 mutants. This could alter the con- <br />                                                                              to the 39 trailer to achieve precise endonucleolytic 39 end <br /> formation of pre-tRNAMet       in a way that impairs removal of the <br />                           i                                                  processing. When Lhp1p was overexpressed in gcd14-1 cells, it <br /> 59 or 39 extensions or addition of the CCA triplet to the 39 end. <br />                                                                              led to increased steady-state levels of both precursor and ma- <br /> Consistent with this possibility, we recently found that m1A is              ture full-length tRNAMet         and of an intermediate precursor <br />                                                                                                        i <br /> lacking in total tRNA isolated from gcd10D cells (3). This base              transcribed from IMT2 and IMT3 at 37&#176;C containing the 39 <br /> modification occurs at position 58 in the initiator and elongator            extension (data not shown). Through increased binding to the <br /> forms of tRNAMet, in tRNATyr     GTA (also examined here), and in            39 trailers of these precursors, the overproduced Lhp1p could <br /> 15 other tRNAs but is not found at any other positions in yeast              promote correct endonucleolytic cleavage, as suggested above, <br /> tRNAs (57). In the initiator, the m1A58 residue is involved in               accounting for the ability of hcLHP1 to increase expression of <br /> a unique tertiary substructure not observed in any elongator                 mature tRNAMet       in gcd14 mutants at elevated temperatures. <br />                                                                                               i <br /> tRNAs, involving three residues unique to eukaryotic initia-                 This type of functional interaction between Gcd14p and Lhp1p <br /> tors, A60, A54, and A20 (5). Thus, the lack of methylation at                in the maturation of tRNAMet          is consistent with the lack of <br />                                                                                                                i <br /> position 58 could perturb the structure of initiator tRNAMet,                strong physical interaction between these two proteins (Fig. 9). <br /> and impair its processing and stability, without similarly affect-              Combining the deletion of LHP1 with the gcd14-1 mutation <br /> ing elongator tRNAMet or other tRNAs bearing m1A58. It is                    led unexpectedly to large reductions in the expression of <br /> unknown whether tRNAIle                       Ser <br />                             UAU and tRNACGA, whose processing                NME1 RNA, RPR1 RNA, and tRNASer                 CGA, in addition to <br /> and accumulation were slightly impaired in gcd14 mutants,                    tRNAMeti   . The gcd14-2 lhp1::LEU2 mutant also contained sub- <br /> contain m1A58.                                                               stantially reduced amounts of NME1 RNA. The double mu- <br />    It remains to be determined whether the Gcd14p-Gcd10p                     tants exhibited small reductions in levels of tRNAIle   UAU and U6 <br /> complex is directly responsible for m1A58 formation in yeast                 RNA compared to the corresponding single mutants and had <br /> tRNA and whether Gcd14p has methylase activity, as pre-                      wild-type levels of elongator tRNAMet and 5S rRNA (Fig. 8B). <br /> dicted from the presence of S-AdoMet binding motifs. Con-                    Thus, it appears that Gcd14p and Lhp1p cooperate in the <br /> sidering that Gcd10p has RNA binding activity (23), it might                 maturation of a subset of RNA polymerase III transcripts. Of <br /> have a chaperone function that would facilitate methylation of               the transcripts analyzed thus far, only tRNAMet    i  shows a strong <br /> pre-tRNAMeti     by Gcd14p in wild-type cells. The results in Fig.           requirement for Gcd14p in cells containing Lhp1p. The re- <br /> 6B seem to indicate that the gcd10-505 mutation leads to deg-                moval of Lhp1p reveals additional strong requirements for <br /> radation of the unprocessed precursors that accumulate in                    Gcd14p in the production of mature RPR1 RNA, NME1 <br /> GCD10 gcd14-2 cells. Thus, Gcd10p may also be required to                    RNA, and tRNASer    CGA and a lesser requirement in U6 expres- <br /> protect hypomethylated pre-tRNAMet       i      from exonucleolytic          sion. Similarly to the tRNAs, RPR1 RNA is transcribed as a <br /> degradation. If Gcd10p performs a chaperone function, it                     369-nucleotide precursor RNA with 84 nucleotides of 59 leader <br /> could conceivably interact with mature tRNAMet        i   in the cyto-       and up to 30 nucleotides of 39 trailing sequences (38). An <br /> plasm and promote formation of the ternary complex with                      intriguing possibility is that Gcd14p is required for methylation <br /> eIF2 and GTP. This hypothetical function might explain why                   of pre-RPR1 RNA and that the hypomethylated RPR1 RNA <br />  <br />  <br />  <br /> present in gcd14-2 mutants would be highly susceptible to                        <br /> degradation in cells lacking Lhp1p, as postulated above for                           <br />                                                                                  <br /> tRNAMeti   . It remains to be determined whether RPR1 RNA is                          <br /> methylated. Obviously, we cannot rule out the possibility that                          <br /> combining the gcd14 and lhp1::LEU2 mutations in the same                             <br /> strains leads to an unforeseen indirect effect on transcription                         <br />                                                                                        <br /> or maturation of NME1 RNA, RPR1 RNA, or tRNASer            CGA.                      <br />    The fact that gcd14 mutations alone lead to 30 to 40%                                 <br /> reductions in the levels of mature RPR1 RNA (Fig. 8B; Table                          <br /> 4) raises the possibility that a reduction in RNase P activity                        <br />                                                                                      <br /> contributes to the defect in processing of tRNAMet    i   in gcd14                       <br /> single mutants. Perhaps the absence of m1A58, with the atten-                           <br /> dant defects in tertiary structure, renders tRNAMet i   more <span id='am-64' about='obo:NCBITaxon_9822' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9821 obo:NCBITaxon_35497 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-65'  ></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="Sus" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>sus</span>-                        <br /> ceptible than other tRNAs to reductions in RNase P activity.                             <br />                                                                                     <br /> By the same token, it could be proposed that the dramatic                                <br /> reductions in RPR1 RNA levels observed in the gcd14-1                                    <br /> lhp1::LEU2 double mutant could largely account for the addi-                         <br /> tive effects of these mutations in reducing mature tRNAMet     i <br />                                                                                         <br />                                                                                     <br /> levels. However, it was shown previously that depletion of                          <br /> Rpp1p, a protein subunit of RNase P (12, 58), or a mutation in                            <br /> the POP1-encoded subunit of RNase P (41) leads to accumu-                      <br /> lation of unprocessed tRNA precursors concomitant with re-                               <br />                                                                                           <br /> duced amounts of the mature tRNAs. The fact that both pre-                           <br /> cursors and mature tRNAMet    i    were greatly diminished in the                      <br /> gcd14-1 lhp1::LEU2 double mutant (Fig. 8B) suggests that the                        <br /> unprocessed tRNA precursors, which should accumulate in                            <br />                                                                                         <br /> response to reduced RNase P levels, are rapidly degraded in                       <br /> the absence of both Lhp1p and wild-type Gcd14p.                                     <br />    It will be interesting to identify the complete role of the                          <br /> Gcd10p-Gcd14p complex in modification and processing of                                 <br />                                                                                   <br /> different RNA polymerase III transcripts and to investigate                             <br /> further functional molecular interactions between this complex                        <br /> and Lhp1p. We recently found that deletion of CPD1, the                              <br /> GCD14 homologue in fission yeast, is not lethal but leads to                           <br /> accumulation of tRNAMet         precursors (11a). Thus, it appears                       <br />                           i                                                          <br /> that the function of Gcd14p in tRNAMet    i   maturation is evolu-                       <br /> tionarily conserved.                                                                     <br />   <br /> </body></html>