<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Haploinsufficiency Flap endonuclease (Fen1) leads <br /> to rapid tumor progression <br /> Melanie Kucherlapati*&#8224;, Kan Yang&#8225;, Mari Kuraguchi&#8225;, Jie Zhao&#167;, Maria <span id='am-76' about='obo:NCBITaxon_177434' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_60751 obo:NCBITaxon_71543 obo:NCBITaxon_71541 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_41073 obo:NCBITaxon_33208 obo:NCBITaxon_41071 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_535382 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-77' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-78'  ></span><span id='am-79' property="rdfs:label" content="Lia" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Lia</span>&#167;, Joerg Heyer&#182;, Michael F. Kane&#20648;, <br /> Kunhua Fan&#8225;, Robert Russell&#167;, Anthony M. C. Brown&#8225;**, Burkhard Kneitz&#8224;&#8224;, Winfried Edelmann&#167;, <br /> Richard D. Kolodner&#20648;, Martin Lipkin&#8225;, Raju Kucherlapati* <br /> *Departments Medicine Genetics Harvard Partners Center Genetics Genomics, Harvard Medical School, Boston, MA 02115; &#8225;Strang <br /> <span id='am-51' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-55'  ></span>Cancer</span> Research Laboratory The Rockefeller University, New York, NY 10021; &#167;Department Cell Biology Institute Animal Studies, Albert <br /> Einstein College Medicine, 1300 Morris Park Avenue, Bronx, NY 10461; &#182;Memory Pharmaceuticals Corporation, Montvale, NJ 07645; &#20648;Ludwig <br /> Institute Cancer Research, University <span id='am-6' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-7' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>California</span> San Diego School Medicine, La Jolla, CA 92093; **Department Cell Biology and <br /> Anatomy, Weill Medical College Cornell University, New York, NY 10021; &#8224;&#8224;Physologische Chemie I, Biozentrum Am Hubland, <br /> 97074 Wurzburg, Germany <br />  <br /> Contributed Richard D. Kolodner, May 29, 2002 <br />  <br /> Flap endonuclease (Fen1) required DNA replication                     FEN1 functions processing 5&#11032; ends Okazaki <br /> repair, defects gene encoding Fen1 cause increased                 fragments lagging strand synthesis long patch-base excision <br /> accumulation mutations genome rearrangements. Because                  type repair (32&#8211;37). FEN1 protein binds proliferating nuclear cell <br /> mutations genes involved processes cause cancer              antigen (38&#8211;39) potentially competes p21, xeroderma <br /> predisposition, investigated possibility Fen1                 pigmentosum gene product, 5&#11032; methyl cytosine methyl transferase, <br /> function tumorigenesis gastrointestinal tract. Using gene           proteins specific binding motif, implying roles for <br /> knockout approaches, introduced null mutation murine                FEN1 DNA replication, repair, epigenetic inheritance, cell <br /> Fen1. Mice homozygous Fen1 mutation obtained,                cycle control. Null mutations gene (also referred rad27) <br /> suggesting absence Fen1 expression leads embryonic lethal-              <span id='am-27' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-29'  ></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> cause temperature-sensitive viability, <br /> ity. Most Fen1 heterozygous animals appear normal. However,                   increased sensitivity UV light, mutagen sensitivity, genomic <br /> when combined mutation adenomatous polyposis coli               instability, plasmid loss, destabilization telomeric repeats <br /> (Apc) gene, double heterozygous animals increased numbers                nonviable combination mutations genes involved <br /> of adenocarcinomas decreased survival. The tumors              homologous recombination (40&#8211;46). rad27 mutants exhibit a <br /> mice microsatellite instability. Because copy Fen1            complex mutator phenotype. They increased frequencies of <br /> gene remained intact tumors, Fen1 haploinsufficiency appears            accumulating frame-shift mutations <span id='am-56' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-58'  ></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="This" datatype="xsd:string"></span>this</span>, been <br /> lead rapid progression cancer.                                          suggested rad27 mutations cause partial MMR defect <br />                                                                               (41&#8211;42, 47). rad27 mutants accumulate insertion mutations that <br />  <br /> T    forms human colorectal cancer, most <br />      frequent sporadic form. Germ line mutations the <br /> Apc gene responsible rare cancer predisposition syn- <br />                                                                               result duplication sequences flanked repeated sequences <br />                                                                               deletion mutations result deletion sequences <br />                                                                               flanked repeated sequences (42), thought arise <br /> drome, familial adenomatous polyposis (FAP) (1&#8211;7),                   improper processing Okazaki fragments (42, 48). Finally, <br /> additional genetic changes required tumor formation.                  rad27 mutants accumulate extensive genome rearrangements that <br /> Mutations Apc genes involved tumor progression            suggested result errors DNA replication <br /> a prerequisite majority sporadic colorectal tumors. Hered-           (43). The phenotypes caused rad27 mutations make Fen1 a <br /> itary nonpolyposis colorectal cancer (HNPCC) colorec-              potential candidate cancer susceptibility gene. <br /> tal cancer predisposition syndrome. Germ line mutations                   To examine role FEN1 tumor initiation progression, <br /> DNA mismatch repair (MMR) genes Msh2 Mlh1 major                   generated mice carry null mutation Fen1 gene. We <br /> cause HNPCC (refs. 8&#8211;12; http:&#20862;&#20862;www.nfdht.nl&#20862;database&#20862;                     homozygosity Fen1 mutation leads early <br /> mdbchoice.htm). Most HNPCC cases accounted                      embryonic lethality. Mice heterozygous mutation <br /> missense, nonsense, frameshift, splice mutations             viable mild tumor predisposition phenotype. A <br /> genes deletion mutations, primarily Msh2 (9, 13&#8211;16). A small <br />                                                                               proportion mice (17%) develop non-Hodgkin&#8217;s lymphoma of <br /> proportion HNPCC cases appear caused germ line <br />                                                                               B cell type, mice premature <span id='am-66' about='obo:NCBITaxon_49990' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_216706 obo:NCBITaxon_216718 obo:NCBITaxon_3398 obo:NCBITaxon_4136 obo:NCBITaxon_4143 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-69'  ></span><span id='am-70' property="rdfs:label" content="Thymus" datatype="xsd:string"></span>thymus</span> involution. <br /> mutations MMR genes, Msh6 Pms2, muta- <br />                                                                               To assess role Fen1 GI tumor progression, generated <br /> tions Msh6 familial non-HNPCC cases (17&#8211;21). <br />                                                                               mice double heterozygotes mutations Apc and <br /> Tumors HNPCC patients exhibit microsatellite insta- <br />                                                                               Fen1 genes. Mice heterozygous Apc mutation <br /> bility (MSI) (22&#8211;26). MSI results expansion contraction of <br /> mono- multinucleotide repeats failure repair insertion&#20862;          (Apc1638N) tumor predisposition phenotype. They develop <br /> deletion mismatches DNA. Although large number HNPCC                  colonic polyps adenomas small intestine show <br /> kindreds accounted mutations Msh2, Mlh1,                 MSI, median survival 13 mo. When Apc1638N and <br /> other MMR genes, reports suspected HNPCC cases                   Fen1null combined, median survival resulting double <br /> that mutations genes tumors              heterozygous mice reduced 9 mo, tumors more <br /> with MSI, raising possibility mutations genes               advanced, tumors MSI. The wild-type (WT) copy of <br /> involved MMR implicated cases (27&#8211;30).                   Fen1 allele remains intact tumors. These results suggest <br />    A proportion sporadic tumors intestinal tract         haploinsufficiency Fen1 important contributor to <br /> of tumors sites exhibit MSI (16, 23, 24, 31), suggesting           gastrointestinal (GI) tumor progression mouse models. <br /> that sporadic defects MMR genes play role the <br /> initiation progression number tumor types. Indeed, <br />                                                                               Abbreviations: FAP, familial adenomatous polyposis; HNPCC, hereditary nonpolyposis <br /> mutations Msh2 Mlh1 silencing Mlh1 detected              colorectal cancer; MMR, mismatch repair; MSI, microsatellite instability; GI, gastrointesti- <br /> some tumor types (8&#8211;12). This observation           nal; ES, embryonic stem; IVTT, vitro transcription translation; WT, wild type. <br /> suggests DNA repair genes involved               &#8224;To  reprint requests addressed. E-mail: mkucherlapati@rics.bwh. <br /> cases. One gene Flap Endonuclease 1 (Fen1).                            harvard.edu. <br />  <br />  <br />  <br /> 9924 &#8211;9929 &#20841; PNAS &#20841; July 23, 2002 &#20841; vol. 99 &#20841; no. 15                                                               www.pnas.org&#20862;cgi&#20862;doi&#20862;10.1073&#20862;pnas.152321699 <br />                                                                                   Fig. 2. Genotyping embryonic day 3.5 blastocysts PCR. Blastocysts were <br />                                                                                  placed distilled H2O, boiled 10 min, used directly substrate for <br />                                                                                  PCR reactions. A 260-bp fragment amplified Neomycin gene in <br />                                                                                  Fen1null locus, 120-bp fragment amplified Fen1 WT locus. <br />  <br />  <br />  <br />                                                                                  sections flat mucosa adjacent tumors. Statistical analyses were <br />                                                                                  performed using Fisher exact probability &#65533;?&#65533;2 tests for <br />                                                                                  analysis tumor incidence Mann&#8211;Whitney test and <br /> Fig. 1. Strategy production Fen1null mutant mice. (A) Gene-           binomial exact calculation tumor multiplicity. <br /> targeting strategy. (B) Southern blot liver DNA Fen1 WT (12-kb <br /> fragment) Fen1null mutant mouse (6-kb fragment). DNA digested <br /> with HindIII hybridized 1,800-bp probe 3&#11032; flanking          MSI Analysis. DNA extracted tumor tissue used for <br /> region Fen1 gene. (C) PCR analysis tail DNA Fen1 WT (WT, 2.2-kb   PCR amplification (49) limiting dilution conditions. Approx- <br /> fragment) Fen1null mutant mouse (1.7-kb fragment).                           imately 25&#8211;30 reactions performed tumor DNA <br />                                                                                  sample dilution 10&#11002;5. The D7Mit91 dinucleotide repeat <br />                                                                                  locus (left: TCTTGCTTGCATACACTCACG right: <span id='am-1' about='obo:NCBITaxon_1255062' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_1204416 obo:NCBITaxon_131567 obo:NCBITaxon_13819 obo:NCBITaxon_3290 obo:NCBITaxon_33090 obo:NCBITaxon_3268 obo:NCBITaxon_131221 obo:NCBITaxon_241806 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="rdfs:label" content="Gaga" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>GAGA</span>- <br /> Materials Methods                                                            CAAACCGCAGTCTCCT) amplified using end-labeled <br /> Fen1 Gene Targeting. A 9-kb fragment mouse genomic DNA                    left primer. Amplified products separated denaturing <br /> isolated BAC 325J22 (RP-22 BAC library) cloned                     polyacrylamide gel autoradiographed analysis. Instability <br /> pUC19. The fragment contained Fen1 coding region, shown                   judged determining proportion PCR reactions that <br /> by DNA sequencing. A 1.2-kb KpnI fragment located 5&#11032; end                  contained microsatellite allele differed WT allele. <br /> of flanking region deleted facilitate identification of <br /> target vector insertion embryonic stem (ES) cells. ClaI                   Sequencing Remaining Fen1 Allele Tumors. Primer pairs were <br /> HindIII sites encompass coding region 826                  constructed spanned Fen1 gene mouse&#20862; <br /> bp 3&#11032; flanking sequence. The ClaI site filled converted            human homology blocks implicated transcriptional control <br /> to XbaI site. The Fen1 coding region deleted                      (50). PCR products sequenced. <br /> digestion XbaI HindIII replaced neomycin <br /> gene control phosphoglycerol kinase promoter. The                   Analysis Apc Truncation Mutations. Codons 677-1674 the <br /> 826 bp 3&#11032; flanking region previously deleted reinserted            mouse Apc gene analyzed protein truncating mutations <br /> HindIII site using PCR product containing HindIII sites             PCR vitro transcription translation (IVTT) <br /> end. Proper orientation sequence confirmed. The targeting                (51&#8211;52). PCR amplification WT Apc allele performed <br /> vector (Fig. 1A) linearized KpnI transfecting ES cells.             stages eliminate coamplification inactivated <br /> G418-resistant clones screened PCR using primers                      Apc1638N allele. Ten nanograms genomic DNA amplified <br />  <br />  <br />  <br />  <br />                                                                                                                                                                     GENETICS <br /> within targeting vector 3&#11032; flanking region. Of 165 ES cell               10-&#65533;?&#65533;l reactions containing Pfu (<span id='am-12' about='obo:NCBITaxon_2260' typeof='owl:Thing obo:NCBITaxon_183968 obo:NCBITaxon_1 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_2259 obo:NCBITaxon_2258 obo:NCBITaxon_28890'><span id='am-13' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-14'  ></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Pyrococcus" datatype="xsd:string"></span>Pyrococcus</span> furiosus) DNA <br /> clones screened, 8 contained correctly targeted mutant allele                polymerase reaction buffer [20 mM Tris&#18528;HCl (pH 8.8)&#20862;2 mM <br /> (Fen1null). Chimeric mice generated standard techniques                  MgSO4&#20862;10 mM KCl&#20862;10 mM (NH4)2SO4&#20862;0.1% Triton X-100&#20862;0.1 <br /> with C57&#20862;BL6 blastocysts germ line transmission monitored                 mg&#20862;ml nuclease-free BSA] 0.05 units&#20862;&#65533;?&#65533;l Pfu turbo <br /> using coat color markers. Chimeric males bred C57&#20862;BL6                    (Stratagene). Cycling conditions cycle 94&#176;C 5 <br /> females offspring genotyped PCR (Fig. 1C) Southern                     min, followed 20 cycles 94&#176;C 1 min, 60&#176;C 1 min, and <br /> hybridization (Fig. 1B). Offspring intercrossed required.                72&#176;C 5 min, final extension cycle 72&#176;C 5 min. <br />                                                                                  This procedure resulted amplification products &#11015;3.2 kb. <br /> Generation Fen1null&#20862;Apc1638N Animals. Mice heterozygous               Two overlapping segments Apc gene covering codons <br /> Fen1null allele mated Apc1638N animals spontaneously              677-1223 1100&#8211;1674 subsequently amplified from <br /> develop intestinal colonic polyps colon cancer. The                      aliquots reactions using pairs PCR primers <br /> Apc1638N allele C57&#20862;BL6 background. The Fen1null mice                 specific IVTT. For amplification 677-1223 fragment, <br /> were mixed genetic background estimated 60% C57&#20862;BL6,                  forward primer 5&#11032;-CGGGATCCTAATACGACTCACTAT- <br /> 37.5% 129&#20862;SV, 2.5% SJL&#20862;J. All animals maintained                  AGGGAGACCACCATGGATGCATGTGGAACTTTGTGG <br /> Association Assessment Accreditation Laboratory An-                   reverse primer: 5&#11032;-CGAACAGCTAGCATTAGATG- <br /> imal Care-approved facility barrier conditions.                            GAGGTACAGC used. For amplification 1100&#8211;1674 <br />                                                                                  fragment, forward primer: 5&#11032;-GCGGATCCTAATACGACT- <br /> Analysis Tumors. After mice killed, GI tract                     CACTATAGGGAGACCACCATGGGTATGATGATGT- <br /> removed, opened longitudinally, fixed 10% neutral-buffered                ATAGGTCAAGGGGAACCAGT reverse primer: 5&#11032;- <br /> formalin. The gross specimens examined dissecting                   CAACTTGCTAGCTCTGACCCCATCTCCAG used. <br /> microscope tumors, number location tumors                     PCR performed 20-&#65533;?&#65533;l reactions containing 1-&#65533;?&#65533;l aliquots <br /> were recorded. Representative tumors chosen histological                first-stage reactions. Cycling conditions segments <br /> and molecular analysis. A histological diagnosis                 above, 25 cycles PCR performed and <br /> tissue sections hematoxylin eosin staining. Early invasive             annealing temperature 57&#176;C. One microliter the <br /> adenocarcinomas invaded submucosa,                  resulting PCR products directly used template 6-&#65533;?&#65533;l <br /> and invasive adenocarcinomas invaded                IVTT reactions (TNT T7 Quick PCR DNA, Promega) <br /> muscularis beyond. Microadenomas counted serial                 containing 2.5 &#65533;?&#65533;Ci [35S]-methionine (Amersham Pharmacia). <br />  <br /> Kucherlapati et al.                                                                                        PNAS &#20841; July 23, 2002 &#20841; vol. 99 &#20841; no. 15 &#20841; 9925 <br />                                                                                   Table 2. Multiplicity GI tumors Fen1null mice <br />                                                                                                                                        Small           Large <br />                                                                                                         Overall       Stomach        intestine       intestine <br />  <br />                                                                                  WT                  0.00 &#11006; 0.00     0.00 &#11006; 0.00    0.00 &#11006; 0.00    0.00 &#11006; 0.00 <br />                                                                                  Fen1null            0.00 &#11006; 0.00     0.00 &#11006; 0.00    0.00 &#11006; 0.00    0.00 &#11006; 0.00 <br />                                                                                  Fen1null&#20862;Apc1638N   8.44 &#11006; 7.13*    0.56 &#11006; 1.13&#8224;   7.32 &#11006; 6.20*   0.56 &#11006; 1.01&#8224; <br />                                                                                  Apc1638N            7.13 &#11006; 2.85     1.88 &#11006; 2.10    5.00 &#11006; 1.85    0.25 &#11006; 0.71 <br />  <br />                                                                                     Statistical results comparison binomial exact calcula- <br />                                                                                  tion: compared Fen1null: *, P &#65533;? 0.001; &#8224;, P &#65533;? 0.02. No significant difference <br />                                                                                  Fen1null&#20862;Apc1638N double heterozygous Apc1638N mice <br />                                                                                  Mann-Whitney test. Mean &#11006; SD. <br />  <br />  <br />  <br />                                                                                  severely underrepresented (X2 &#11005; 26.9, P &#65533;? 0.05) (Fig. 2.). We <br />                                                                                  attempted establish cell lines 60 blastocysts obtained from <br />                                                                                  intercrosses; able establish 14 (23%). Six these <br />                                                                                  wild type heterozygotes, were <br /> Fig. 3. Kaplan&#8211;Meier survival plot Fen1 &#11001;&#20862;&#11001; Apc &#11001;&#20862;&#11001;; Fen1 &#11001;&#20862;&#11002; Apc &#11001;&#20862;&#11001;; <br />                                                                                  homozygous Fen1null allele. These results suggest Fen1 <br /> Fen1 &#11001;&#20862;&#11001; Apc &#11001;&#20862;&#11002;; Fen1 &#11001;&#20862;&#11002; Apc &#11001;&#20862;&#11002; mice. Time death mice              expression necessary continued cell viability. <br /> became moribund recorded. The colors different genotypes the <br /> number mice follows: green, (106) Fen1 &#11001;&#20862;&#11001; Apc &#11001;&#20862;&#11001;; red, (161)         Properties Fen1null Heterozygotes. Mice heterozygous <br /> Fen1 &#11001;&#20862;&#11002; Apc &#11001;&#20862;&#11001;; dark blue, (33) Fen1 &#11001;&#20862;&#11001; Apc &#11001;&#20862;&#11002;; light blue, (25)             Fen1null mutation viable, fertile, life span <br /> Fen1 &#11001;&#20862;&#11002; Apc &#11001;&#20862;&#11002;.                                                                indistinguishable WT littermates. Among two <br />                                                                                  small groups mice Fen1null heterozygotes, mice <br />                                                                                  moribund. In group seven mice, total <br /> The reactions incubated 30&#176;C 1 h. Aliquots                    heterozygotes WT animals; three <br /> IVTT reactions analyzed 12% SDS&#20862;PAGE                            heterozygotes moribund 11 mo age, was <br /> fluorography. For characterization tumor-specific                     moribund 5 mo age. All mice had <br /> mutations, PCR products cloned vector, indi-                     thymuses times smaller their <br /> vidual clones screened IVTT identify mutations,                  WT littermates healthy animals comparable age. <br /> DNA sequence determined.                                                     Histological examination revealed severe atrophy thymus, <br />                                                                                  medulla cortex, overall reduction the <br /> Results <br />                                                                                  density uniformity lymphocyte population. The <br /> Homozygous Fen1null Animals Are Not Viable. We generated mouse                   overall observations favor interpretation premature thy- <br /> ES cells copy Fen1 gene replaced null               mus involution mechanism accelerated loss individual <br /> allele entire gene completely deleted (Fig. 1 A).               lymphocytes. A second cohort contained 36 mice, 12 <br /> Correct gene targeting events identified Southern <span id='am-92' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-93' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-95'  ></span><span id='am-96' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>hybrid</span>-                Fen1null heterozygotes. When examined an <br /> ization (Fig. 1B) PCR (Fig. 1C). We obtained gene-                     average age 12.3 mo, heterozygotes non- <br /> targeted cell lines. Four, designated 126, 111, 102, 146,               Hodgkins lymphoma B cell type, a <br /> used experiments. ES cells injected mouse                  Paneth cell hyperplasia small intestine. <br /> blastocysts generate chimeric mice. Chimeric mice cell lines <br /> 126 111 transmitted mutant allele (Fen1null)               Effect Fen1 Haploinsufficiency Apc1638N Mice. We gener- <br /> germ line. The resulting Fen1null heterozygotes viable                  ated mice heterozygous mutation Apc gene <br /> fertile. They intercrossed offspring genotyped                 (Apc1638N) (53) heterozygous Fen1null. Kaplan&#8211;Meier <br /> PCR. We examined 219 mice 34 matings. Of these, 71                     survival curves different groups mice shown Fig. <br /> wild type 148 Fen1null heterozygotes. Fen1null homozy-                  3. Fen1null&#20862;&#11001; mice survival curves indistinguishable <br /> gotes detected. These results suggest expression                WT littermates. Apc1638N mice median survival <br /> Fen1 necessary normal embryogenesis, lack                     13 mo. The addition Fen1null allele mice <br /> functional copies Fen1 gene leads embryonic lethality.                 reduces median survival 9 mo (P &#65533;? 0.07). <br />    To determine lack gene expression leads early                  Mice different genetic compositions killed exam- <br /> lethality, examined 37 blastocysts Fen1null intercrosses.            ined approximately 1 yr age. The tumor incidence types <br /> Although detected blastocysts homozygous (Fen1                 tumors observed different types mice summarized <br /> &#11001;&#20862;&#11001;: Fen1 &#11002;&#20862;&#11001;: Fen1 &#11002;&#20862;&#11002; &#11005; 23:10:4), blastocysts                        Table 1. The WT Fen1null heterozygous mice tumors <br />  <br />  <br />                        Table 1. Tumor incidence Fen1 mice <br />                                                                                                  No., % mice tumors <br />                                                          Sex <br />                        Genotype (n)                     (M&#20862;F)          Age, mo            Overall               GI             Extra-GI <br />  <br />                        WT (7)                           1:6           12.6 &#11006; 1.1         0 (0%)            0 (0%)              0 (0%) <br />                        Fen1null (12)                    1:2           12.3 &#11006; 2.5         2 (17%)           0 (0%)              2 (17%) <br />                        Fen1null&#20862;Apc1638N (9)            1:0.5         11.1 &#11006; 1.7         9 (100%)          9 (100%)*           0 (0%) <br />                        Apc1638N (8)                     1:0.6         13.0 &#11006; 1.9         8 (100%)          8 (100%)            0 (0%) <br />  <br />                          n, number mice studied. Statistical results comparison Fisher exact probability test: compared to <br />                        Fen1null: *, P &#65533;? 0.001. No significant difference Fen1null&#20862;Apc1638N double heterozygous and <br />                        Apc1638N mice. Mean &#11006; SD. <br />  <br />  <br /> 9926 &#20841; www.pnas.org&#20862;cgi&#20862;doi&#20862;10.1073&#20862;pnas.152321699                                                                                             Kucherlapati et al. <br />  Table 3. Histologic types tumors Fen1null mice <br />                                       Apc1638N     Fen1null&#20862;Apc1638N <br />  <br /> Total no. tumors           57              76 <br /> No. (%) tumors histological 36 (63%)        48 (63%) <br />     examination <br /> Histologic types tumors: <br />  <br /> Total                              36 (100%)        48 (100%) <br /> Malignant                           1 (2.8%)        12 (25%) <br />   Early invasive carcinoma          0 (0%)           2 (4.2%) <br />   Adenocarcinoma                    1 (2.8%)        10 (20.8%) <br /> Benign <br />   Villous-tubular adenoma           4 (11.1%)        7 (14.6%) <br />   Tubular adenoma                  25 (69.4%)       23 (47.9%) <br />   Microadenoma                      6 (16.7%)        6 (12.5%) <br />  <br />  <br />  <br /> in GI tract. Both Apc1638N double heterozygotes      Fig. 5. Invasive adenocarcinoma small intestine Fen1nullApc1638N of <br /> substantial tumor burden. Tumor multiplicity slightly higher    double mutant mouse. Tumor tissue composed neoplastic glands with <br /> Fen1null&#20862;Apc1638N animals Apc1638N mice (Table 2). The         irregular shape size infiltrating muscularis. Mucinous pools formed. <br /> range tumor multiplicity considerably greater Fen1null&#20862;     Ulceration shown surface tumor. Desmoplastic response was <br /> Apc1638N mice Apc1638N mice alone. The range tumors        observed near neoplastic glands invasive area. Hematoxylin&#20862;eosin &#11003;100. <br /> per animal Apc1638N mice 4&#8211;13 Fen1null&#20862;Apc1638N <br /> double heterozygotes, range 3&#8211;20. A definitive difference      The Remaining Fen1 Allele Tumors Is Intact. We examined the <br /> between strains mice obtained histological     status WT Fen1 allele DNA tumors derived from <br /> typing tumors (Table 3; Fig. 5). The Fen1null&#20862;Apc1638N double       Apc1638N allele Fen1null heterozygous mice. DNA 15 <br /> heterozygous mice 9-fold greater incidence malignant          Fen1null Apc1638N tumors 5 Apc1638N tumors exam- <br /> tumors (early invasive invasive adenocarcinomas) compared          ined. The entire coding region Fen1 gene additional <br /> with Apc1638N heterozygotes (P &#65533;? 0.0053). Apc1638N animals         820 bp 5&#11032; untranslated region contains homology <br /> more benign tumors (tubular adenomas) (3.13 &#11006; 1.89)               blocks conserved mouse human Fen1 genes were <br /> Fen1null&#20862;Apc1638N double heterozygotes (2.22 &#11006; 1.48, P &#65533;? 0.05). An     analyzed DNA sequencing. In cases, Fen1 gene was <br /> increase number adenomas large intestine     intact detectable mutations (<span id='am-87' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-88' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-90'  ></span><span id='am-91' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> shown). <br /> observed double heterozygous mice (0.56 &#11006; 1.01) compared with <br /> Apc1638N (0.25 &#11006; 0.71, P &#65533;? 0.05, Table 2).                             Apc Mutations Tumor DNA. We examined fate WT <br />                                                                        Apc allele tumors groups mice. A total 41 <br /> Tumors Fen1null&#20862;Apc1638N Double Heterozygous Animals Have         tumors analyzed Fen1null&#20862;Apc1638N mice, results <br /> MSI. We determined tumors double heterozy-        compared obtained analysis 15 tumors <br /> gotes exhibited MSI using limiting dilution PCR. DNA       Apc1638N mice (51). IVTT analysis indicated 15 (37%) of <br /> normal intestinal mucosa healthy C57&#20862;BL6 mice (0 60          tumors double heterozygotes Apc truncation <br />  <br />  <br />  <br />  <br />                                                                                                                                                          GENETICS <br /> PCR reactions positive), tumors Apc1638N mice (7     mutations region encoding codons 876-1274, se- <br />                                                                        quence alterations shown Table 4. In general, Apc <br /> 91 PCR reactions positive) showed instability (Fig. 4 A C). <br />                                                                        truncation mutations detected groups tumors point <br /> DNA Mlh1 tumors (42 62 PCR reactions positive)         mutations. Insertion&#20862;deletion mutations identified 2 15 <br /> four tumors Fen1null&#20862;Apc1638N heterozygotes (93 123 PCR        tumors double heterozygous mice (Fig. 6). We detected <br /> reactions positive) extensive MSI (Fig. 4 B D). Normal liver   insertion&#20862;deletions tumors Apc1638N animals. <br /> tissue Apc1638N mouse adenomas showed MSI <br /> (0 31 PCR reactions positive; data shown). Normal liver         Discussion <br /> tissue Fen1null Apc1638N mouse adenocarcinomas did     We generated mice loss function mutation Fen1 <br /> not MSI (0 30 PCR reactions).                                  gene examined phenotype caused mutation. Mice <br />  <br />  <br />  <br />  <br />                                                                                                Fig. 4. MSI Fen1nullApc1638N tumor DNA. Primers <br />                                                                                                D7Mit91 locus used amplify 262-bp <br />                                                                                                fragment limiting dilution conditions (10&#11002;5). <br />                                                                                                (A) Five PCR amplifications (normal intestine) no <br />                                                                                                MSI. (B) Twenty-one PCR amplifications two <br />                                                                                                Fen1null Apc1638N tumor DNAs MSI indicated. <br />                                                                                                (C) Twenty PCR amplifications Apc1638N <br />                                                                                                tumor DNA MSI. (D) Thirty-one PCR amplifi- <br />                                                                                                cations Mlh1 tumor MSI. <br />  <br />  <br /> Kucherlapati et al.                                                                              PNAS &#20841; July 23, 2002 &#20841; vol. 99 &#20841; no. 15 &#20841; 9927 <br />                       Table 4. Sequence Apc mutations Apc1638N Fen1null&#20862;Apc1638N tumors <br />                      <span id='am-61' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-62' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-64'  ></span><span id='am-65' property="rdfs:label" content="Codon" datatype="xsd:string"></span>Codon</span>      Mutation      Consequence            WT sequence*            Apc1638N      Fen1null&#20862;Apc1638N <br />  <br />                      874      C-T               Arg-Stop       TCA AAA CGA GGT CTG                &#8211;                 1 <br />                      921      C-T               Arg-Stop       GCG GCA CGA AGA AGC              1                   1 <br />                      934      ins&#8224; TACA         Frameshift     AAC ACA TAC AAC TTC              1                   &#8211; <br />                      939      G-T               Glu-Stop       AAG TCG GAA AAT TCA              1&#167;                  &#8211; <br />                      941      C-A               Ser-Stop       GAA AAT TCA AAT AGG               &#8211;                  1 <br />                      956      C-T               Arg-Stop       TAT AAA CGA TCT TCA               &#8211;                  4 <br />                      982      G-T               Glu-Stop       TCA GTT GAA TCC TAT               &#8211;                  1 <br />                      992      del&#8225; 8 bp &#11001; A     Frameshift     AAA TTTTGCAGTTAT GGT             1&#167;                  &#8211; <br />                      995      del&#8225; 10 bp        Frameshift     AGT TATGGTCAGTAT CCA              &#8211;                   1 <br />                      1047     G-A               Trp-Stop       GAA AGG TGG GCA AGA              1&#167;                  &#8211; <br />                      1112     C-T               Arg-Stop       ACA AAT CGA ATG GGT               &#8211;                  1 <br />                      1154     G-T               Glu-Stop       GAA GAA GAA GAA GAG               &#8211;                  1 <br />                      1202     ins&#8224; ATCA         Frameshift     AAT TCA TCA GCA CAA               &#8211;                  1 <br />                      1227     C-T               Gln-Stop       AAA AGG CAG AAT CAG               &#8211;                  1 <br />                      1242     C-T               Gln-Stop       CAG ACT CAA AAA GGC               &#8211;                  1 <br />                      1274     C-A               Ser-Stop       TGC AGT TCA TTA TCA               &#8211;                  1 <br />                      Total                                                                      5                  15 <br />  <br />                      *The WT sequence surrounding mutation shown site mutation shown bold. <br />                      &#8224;ins, insertion. <br />                      &#8225;del, deletion. <br />                      &#167;Data Kuraguchi et al. (51). <br />  <br />  <br />  <br />  <br /> lacking copies Fen1 gene die early embryogenesis,            mice similar, appears significant increase rate <br /> indicating Fen1 gene product essential development.        progression tumors double mutant mice. The had <br /> The relative paucity homozygous blastocysts,          times adenocarcinomas carrying Apc1638N <br /> failure establish Fen1null homozygous ES cell lines, suggests      mutation alone. These results suggest reduction Fen1 does <br /> Fen1 expression necessary cell survival proliferation.     alter tumor initiation process profound effect on <br /> The role FEN1 DNA replication repair explain           tumor progression. None tumors examined showed <br /> importance cell survival. The Fen1 heterozygotes normal          mutation loss WT Fen1 allele. This observation is <br /> life span exhibit phenotypes. Several mice            consistent view complete absence Fen1 function <br /> showed features premature thymus involution                 lead impaired proliferation. A clue mechanism by <br /> the result early T cell depletion. The proposed function             Fen1 heterozygosity results rapid tumor progression came <br /> Fen1 nonhomologous end joining (54) V(D)J re-                    observation tumors double heterozy- <br /> combination explain observed result. A proportion                gotes exhibit MSI level seen tumors Mlh1 <br /> of Fen1 heterozygotes B cell lymphomas. The precise               mutant mice; suggests reduction Fen1 expression <br /> mechanism Fen1 hemizygosity leads B cell lym-                  results impairment DNA repair. <br /> phomas needs investigation.                                           That Fen1 heterozygotes tumor susceptibility that <br />    We obtained strong evidence role Fen1 tumor            reduction copy number sufficient result advanced tumors <br /> progression. When Fen1null mutation combined              Apc1638N background suggest conditions, <br /> Apc1638N mutation, double heterozygous mice exhibited signif-          haploinsufficiency Fen1 gene causes mutator phenotype in <br /> icant differences heterozygous Apc1638N        mammalian cells. A total 41 tumors analyzed Fen1null&#20862; <br /> mutation alone. The double heterozygotes lower median                Apc1638N mice. Fifteen (37%) Apc truncation muta- <br /> survival. Although average tumor burden groups              tions. Several mutations novel. They include 4-bp <br />  <br />  <br />  <br />  <br />                                                                                                                 Fig. 6. Distribution Apc mutations <br />                                                                                                                 Fen1null Apc1638N tumor DNA <br />                                                                                                                 tumor DNA Apc1638N mice. <br />                                                                                                                 Diagram Apc codons 677 <br />                                                                                                                 1674 showing positions char- <br />                                                                                                                 acteristics mutations detected in <br />                                                                                                                 Fen1nullApc1638N (Upper) Apc1638N <br />                                                                                                                 (Lower) intestinal tumors. &#8218;, deletion; <br />                                                                                                                  , insertion; , substitution. Each sym- <br />                                                                                                                 bol represents independent muta- <br />                                                                                                                 tion. The 15-<span id='am-81' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-82'  ></span><span id='am-83' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-85' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>aa</span> (A&#8211;C) four <br />                                                                                                                 20-aa (1&#8211; 4) &#65533;?&#65533;-catenin-binding repeats <br />                                                                                                                 SAMP (Ser, <span id='am-71' about='obo:NCBITaxon_1416571' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_116706 obo:NCBITaxon_6804 obo:NCBITaxon_131567 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6682 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_6683 obo:NCBITaxon_88770 obo:NCBITaxon_6752 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_553065 obo:NCBITaxon_6072'><span id='am-72' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-74'  ></span><span id='am-75' property="rdfs:label" content="Ala" datatype="xsd:string"></span>Ala</span>, Met, Pro se- <br />                                                                                                                 quence) repeat (S1) segment of <br />                                                                                                                 Apc indicated. <br />  <br />  <br /> 9928 &#20841; www.pnas.org&#20862;cgi&#20862;doi&#20862;10.1073&#20862;pnas.152321699                                                                                   Kucherlapati et al. <br />  insertion 10-bp deletion. Because truncation                                      individual components make complexes. Haploinsuf- <br /> mutations detected tumors similar alterations                                        ficiency component make difference cell <br /> previously identified remaining Apc allele tumors                                        undergoing normal replication; however, cell cycle per- <br /> Apc1638N mice, Fen1 heterozygous mice                                         turbed mutations oncogenes tumor suppressor genes (e.g., <br /> appreciable tumor burden, suggest Fen1 does play                                        Apc), additional levels gene products be <br /> significant role GI tumor initiation. The advanced nature                                     necessary accommodate change rates cell division. If <br /> of tumors double heterozygotes suggests greater role                                     critical product necessary replication repair is <br /> Fen1 tumor progression.                                                                            sufficient quantity, result detrimental to <br />    The maintenance genomic stability prime importance                                     genomic stability. Replication repair factors involved the <br /> the cell, surprising genes involved replication                             maintenance genomic stability, results described imply <br /> repair contribute process. We observed MSI Fen1null&#20862;                                        quantitative measure expression gene <br /> Apc1638N haploinsufficient tumors consistent                                        products useful prognostic <span id='am-22' about='obo:NCBITaxon_189528' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_189527 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_9219 obo:NCBITaxon_436491'><span id='am-23' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Indicator" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-26'  ></span>indicator</span>, particularly genes <br /> increased accumulation frameshift mutations observed S.                                         controlling cell cycle concomitantly genes involved replica- <br /> cerevisiae rad27 mutants. That Fen1 WT allele intact                                  tion repair. <br /> tumors suggests quantity product plays important <br /> role tumor progression. Eukaryotic replication initiates 2 &#11003; 104                                This work supported grants R.K. (CA84301-04 ES11040) <br /> to 1 &#11003; 105 sites human haploid genome. The formation                                        R.D.K. (GM50006 ES11040) National Institutes of <br /> competent replication complexes leading lagging strands                                    Health, fellowship M.K. American Association for <br /> of DNA depend concentration availability                                       Cancer Research&#8211;Cancer Research Foundation America. <br />  <br />  <br />  1. Yang, K., Edelmann, W., Fan, K., Lau, K., Kolli, V., Fodde, R., Khan, P. M., Kucherlapati,        26. Terdiman, J. P., Gum, J. R., Jr., Conrad, P. G., Miller, G. A., Weinberg, V., Crawley, S. C., <br />     R. &amp; Lipkin, M. (1997) J. Exp. Zool. 277, 245&#8211;254.                                                    Levin, T. R., Reeves, C., Schmitt, A., Hepburn, M., et al. (2001) Gastroenterology 120, <br />  2. Bussey, H. J. R. (1975) Familial Polyposis Coli (Johns Hopkins Univ. Press, Baltimore).               21&#8211;30. <br />  3. Bodmer, W. F., Bailey, C. J., Bodmer, J., Bussey, H. J. R., Ellis, A., Gorman, P., Lucibello,     27. Park, J. G., Vasen, H. F., Park, K. J., Peltomaki, P., Ponz Leon, M., Rodriguez-Bigas, <br />     F. C., Murday, V. A., Rider, S. H., Scambler, P., et al. (1987) Nature (London) 328,                  M. A., Lubinski, J., Beck, N. E., Bisgaard, M. L., Miyaki, M., et al. (1999) Dis. Colon <br />     614&#8211;616.                                                                                              Rectum 42, 710&#8211;715. <br />  4. Leppert, M., Dobbs, M., Scambler, P., O&#8217;Connell, P., Nakamura, Y., Stauffer, D.,                  28. Wijnen, J., Khan, P. M., Vasen, H., van der Klift, H., Mulder, A., van Leeuwen- <br />     Woodward, S., Burt, R., Hughes, J., Gardner, E., et al. (1987) Science 238, 1411&#8211;1413.                Cornelisse, I., Bakker, B., Losekoot, M., Moller, P. &amp; Fodde, R. (1997) Am. J. Hum. <br />  5. Groden, J., Thliveris, A., Samowitz, W., Carlson, M., Gelbert, L., Albertsen, H., Joslyn,             Genet. 61, 329&#8211;335. <br />     G., Stevens, J., Spiro, L., Robertson, M., et al. (1991) Cell 66, 589&#8211;600.                        29. Wu, Y., Berends, M. J., Mensink, R. G., Kempinga, C., Sijmons, R. H., van Der Zee, A. G., <br />  6. Nishisho, I., Nakamura, Y., Miyoshi, Y., Miki, Y., Ando, H., Horii, A., Koyama, K.,                   Hollema, H., Kleibeuker, J. H., Buys, C. H. &amp; Hofstra, R. M. (1999) Am. J. Hum. Genet. <br />     Utsunomiya, J., Baba, S., Hedge, P., et al. (1991) Science 253, 665&#8211;669.                              65, 1291&#8211;1298. <br />  7. Kinzler, K. W., Nilbert, M. C., Su, L., Vogelstein, B., Bryan, T. M., Levy, D. B., Smith,         30. Farrington, S. M., Lin-Goerke, J., Ling, J., Wang, Y., Burczak, J. D., Robbins, D. J. &amp; <br />     K. J., Preisinger, A. C., Hedge, P., McKechnie, D., et al. (1991) Science 253, 661&#8211;665.               Dunlop, M. G. (1998) Am. J. Hum. Genet. 63, 749&#8211;759. <br />  8. Kuismanen, S. A., Holmberg, M. T., Salovaara, R., la Chapelle, A. &amp; Peltomaki, P.              31. Eshleman, J. R. &amp; Markowitz, S. D. (1995) Curr. Opin. Oncol. 7, 83&#8211;89. <br />     (2000) Am. J. <br /> </body></html>