<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Methods <br />  <br /> Target-selected mutant screen TILLING <br /> in Drosophila <br /> Sylke Winkler,1,2 Anja Schwabedissen,1 Dana Backasch,1 Christian B&#246;kel,1 <br /> Claudia Seidel,1,2 Stefanie B&#246;nisch,1,2 Maximilian F&#252;rthauer,1 Antje Kuhrs,1 <br /> Laura Cobreros,1 Michael Brand,1,2 Marcos Gonz&#225;lez-Gait&#225;n1,3 <br /> 1 <br />  Max Planck Institute Molecular Cell Biology Genetics, 01307 Dresden, Germany; 2University Technology Dresden, <br /> 01062 Dresden, Germany <br />  <br />  <br />       The availability Drosophila genomic DNA sequence prompts development method efficiently <br />       obtain mutations genes identified sequence homologies biochemically. To date, molecularly <br />       characterized mutations generated 6000 &#8764;15,000 annotated fly genes, around <br />       one-third essential viability. To obtain mutations essential nonessential genes interest, took a <br />       reverse genetics approach, based large-scale detection point mutations Cel-I-mediated heteroduplex <br />       cleavage. A library genomic DNA 2086 EMS-mutagenized lines established. The library screened for <br />       mutations genes. A total 6.1 Mb screened, 44 hits different mutagenesis <br />       conditions. Optimal conditions yielded average mutation 156 kb. For essential gene tested, of <br />       25 mutations turned cause lethality, confirming EMS mutagenesis leads high frequency gene <br />       inactivation. We established Cel-I-mediated TILLING used efficiently obtain mutations in <br />       genes Drosophila. <br />  <br />  <br />       [Supplemental material available online www.genome.org.] <br />  <br /> In Drosophila, generally applicable              induced local lesions genomes) used successfully high- <br /> method generate mutant strains directly DNA sequence              throughput screening libraries mutagenized EMS N- <br /> <span id='am-48' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-49' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-52'  ></span>data</span>. Two approaches inherent advantages limitations             ethyl-N-nitroso-urea (ENU) (for review, Henikoff et al. 2004; <br /> (for review, Bellen et al. 2004) used target        Stemple 2004). The method detects single-nucleotide polymor- <br /> mutations known gene Drosophila, i.e., imprecise excision         phisms (SNP) genes amplified PCR. <br /> of transposable elements (Robertson et al. 1988) homologous           TILLING optimized high throughput using Cel-I <br /> recombination (Rong et al. 2002). Various screens provided              endonuclease (Colbert et al. 2001), cleaves mismatches <br /> collection transposable element insertions close 6000            heteroduplexes formed mutant wild-type <br /> genes, number soon approaching saturation,               strands (Oleykowski et al. 1998). We took Cel-I-based Tilling <br /> these mobile elements different degrees target specificity        approach identify mutations large scale Drosophila. <br /> (Bellen et al. 2004). These transposon insertions used as <br /> direct tools affect gene function, utilized to <br /> generate deletions imprecise excision. On hand, <br />                                                                           Results Discussion <br /> homologous recombination routine approach             We established library fly lines containing isogenized second <br /> Drosophila community. Ethyl methanesulfonate (EMS)               chromosomes carrying mutations caused 50-mM EMS treat- <br /> extensively used chemical mutagen results high-point         ment, resulted 2.24 lethal hits chromosome (Ta- <br /> mutational densities. EMS shows low-target specificity (it gen-         ble 1). <span id='am-20' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-23'  ></span><span id='am-24' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> frequency lethal hits consistent previous <br /> erates G:C-to-A:T transitions [Loechler et al. 1984; Snow et al.          mutagenesis (Grigliatti 1986). To build mutant balanced <br /> 1984]), genome accessible it. Flybase        stocks (see Methods), males females proper genotypes <br /> reports 3323 Drosophila genes mutated EMS.              selected eliminating genotypes expres- <br /> For 2601 them, related transposon insertion            sion apoptotic gene hid driven heat-shock promotor <br /> described. Furthermore, EMS point mutations generate null             (Moore et al. 1998). This allowed generate high number of <br /> alleles, lead conditional alleles (e.g., thermo-     balanced mutant lines necessity collecting virgin <br /> sensitive alleles) affect specific protein domains. Allelic series     females right genotype (see Methods). <br /> generated way useful elucidate function             To screen library, followed &#8220;nested PCR/Cel-I strat- <br /> a gene interest.                                                       egy&#65533;? similar recently reported Zebrafish TILLING approach <br />       In <span id='am-6' about='obo:NCBITaxon_3701' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_980083 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_3700 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_3699 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-7' property="rdfs:label" content="Arabidopsis" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45250" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Cardaminopsis" datatype="xsd:string"></span><span id='am-10'  ></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Arabidopsis</span>, zebrafish, rat (McCallum et al. 2000;           (Wienholds et al. 2003; Fig. 1). In approach, Cel-I het- <br /> Wienholds et al. 2003; Smits et al. 2004), TILLING (targeting             eroduplex cleavage, fragments labeled different fluoro- <br />                                                                           phores generated sizes sum length of <br /> 3 <br />  Corresponding author.                                                    amplicon, giving robustness analysis (Figs. 1, 2; also <br /> E-mail Gonzalez@mpi-cbg.de; fax 49-351-2101389. <br /> Article publication http://www.genome.org/cgi/doi/10.1101/     Methods). In <span id='am-35' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> establish genomic DNA preparation as <br /> gr.3721805.                                                               PCR Cel-I digest conditions, applied the <br />  <br />  <br /> 718     Genome Research                                15:718&#8211;723 &#169;2005 Cold Spring Harbor Laboratory Press; ISSN 1088-9051/05; www.genome.org <br />           www.genome.org <br />                                                                                                                                  Tilling Drosophila <br />  <br />  <br /> method reidentify known point Table 1. Tilling results: Overview <br /> mutations, i.e., Mad12, cniAR55, tkv7 <br />                                                                                     125 mM EMS                       50 mM EMS                   Total <br /> (Nellen et al. 1994; Roth et al. 1995; Sekel- <br /> sky et al. 1995) (see Supplemental mate- G0 mutagenized #                       10000                           10000 <br /> rial). Then, screened total 5.0 Mb G0 survivers #                       &#8764;2500 (25%)                     &#8764;9000 (90%) <br />                                                    a <br /> in library looking SNPs F1 #                                   700                             3920 <br />                                               Number linesb                  171                             1915                        2086 <br /> genes (Sara, Alp23B, Arf6;                                c <br />                                               F2 viable chromosomes (f)         9/171 (5.2%)                    41/384 (10.6%) <br /> Fig. 3, Table 1) detected 32 novel        Lethal hits chrom. (&#65533;?&#65533;) d <br />                                                                                 2.94                            2.24 <br /> SNPs, corresponding frequency 1 Amplicons screenede                  Sara-1, 2, 3, 4, 5              Sara-1, 2, 3 (187) <br /> 155.6 kb.                                                                       Alp23B-1, 2, 3                  Sara-4, 5 (1915) <br />       In order study effect                                       Arf6-1, 2                       Alp23B-1, 3 (187) <br />                                                                                                                 Alp23B-2 (1915) <br /> EMS dosage frequency SNPs,                                                                         Arf6-1, 2 (187) <br /> considered 1.5-fold higher EMS concen- Candidates tilling     f <br />                                                                                 1 signal: 1                     1 signal: 73 <br /> tration treatment (125 mM) gener- (LI-COR hits)                             2 signals: 13                   2 signals: 33 <br /> ated pilot library 171 lines. A concen-                                    Total 14                        Total 106                   120 <br />                                                                          f <br />                                               Confirmed sequencing           1 signal: 0                     1 signal: 15 (20.5%) <br /> tration 125 mM lead average                                            2 signals: 12 (92.3%)           2 signals: 17 (51.5%) <br /> number lethal mutations chromo-                                          Total 12                        Total 32                    44 <br /> some 2.94 frequency SNPs 1 Total screenedf                       1086.534 kb                     4977.702 kb                 6064.236 kb <br />                                                           f <br /> in 90.5 kb (12 SNPs 1.1 Mb; Table 1). Frequency                          1 90.5 kb                    1 155.6 kb               &#8212; <br />                                               Missense SNPsf                    10 (83.3%)                      23 (71.9%)                  33 (75.0%) <br /> Therefore, 1.5-fold higher concentration Splice acceptorf                     0                               1 (3.1%)                    1 (2.2%) <br /> of EMS causes 1.5-fold increase Silentf                                2 (16.7%)                       8 (25%)                     10 (22.7%) <br />                                                                     f <br /> in frequency SNPs lethal hits, C/G T/A transition                 9 (75%)                         28 (87.5%)                  37 (84.1%) <br />                                                                  f <br /> implying effects EMS increase Other mutations                       1 T/A C/G                    1 T/A G/C                5 T/A A/T <br />                                                                                 2 T/A A/T                    3 T/A A/T                1 T/A C/G <br /> linearly concentration. However, <br />                                                                                                                                             1 T/A G/C <br /> we obtain lines (171) after <br /> feeding 125 mM EMS 10000 G0 males aTotal number b p cn sca*/CyO, hb-lacZ F males. <br />                                                                                                 1 <br /> (Table 1). In addition, effects bTotal number F2 crosses produced progeny. <br />                                               c <br /> 125 mM EMS health G0         Total number lines homozygous b p cn sca* adults survive. Denominator, total number of <br /> F1 progeny variable typi- dlines inspected. <br />                                                 Lethal hits chromosome: &#65533;?&#65533; = &#11502;lnf. <br /> cally deleterious subsequent re- e171 lines screened amplicon case 125 mM mutagenesis. For 50 mM <br /> peats (data shown). In contrast, mutagenesis, number lines screened particular amplicon indicated parenthesis. <br /> fertility viability progeny fRepeated signals overlap amplicons represented once. <br /> the 50 mM treated males robust and <br /> reproducible (Table 1). Therefore, decided 50 mM EMS <br /> as standard condition. <br />                                                                                           II <br />                                                                                Total SNPORF       = 5065 ORFsII &#215; 1.69 kb &#65533;? ORF &#215; 1SNP 156 kb <br />                                                                                                   = 54.9 SNPORF chromosome II <br />                                                                                                                                               &#65533;? <br />       We studied effect type genomic DNA <br />                                                                            (b) 54.9 SNPs, approximate number located in <br /> library (isogenic/balanced), type LI-COR signal (single/ <br />                                                                            ORFs essential genes (one-third) (Miklos Rubin 1996) <br /> double channel), pooling factor SNP-detec- <br /> tion efficiency (see Supplemental material). Based these <br /> studies, decided (1) use genomic DNA libraries the <br />                                                                                  Total SNPessential <br />                                                                                                II                        &#65533;? <br />                                                                                                         = 54.9 SNPORF 3 = 18.3SNPORF  essential <br />                                                                                                                                                 and <br />  <br /> balanced fly stocks, outcrossing lines origi-           (c)  percentage         18.3  SNPs   ORFs    cause   gene <br /> nal isogenized chromosomes (see Methods), (2) consider             inactivation   (r ), estimated     frequency   hits   chro- <br /> single double LI-COR signals (i.e., signals             mosome lead lethality essential gene (2.24 lethals/ <br /> only complementary amplicon fragments la-                   chromosome II; Methods), assuming inactivat- <br /> beled different fluorophores detected) subse-            ing base-pair changes affect ORFs <br /> quently reconfirm mutants sequencing (see Supple-                                             2.24 lethals &#65533;? chromosome II <br /> mental material details), (3) pool future                           ri =                essential <br />                                                                                                                                     = 12.2% <br /> the genomic DNA preparations pairs PCR reac-                                                18.3 SNPORF <br /> tions.                                                                         corresponds inactivating hit 8.2 SNPs (100/12.2). <br />      The Cel-I-based screen mutant library detected SNPs                     This percentage 12.2% inactivating mutations EMS in <br /> with relatively high frequency (Table 1). We wondered                ORF consistent expected average frequency of <br /> the SNP frequency consistent known frequencies mu-                  inactivating mutations predicted PARSESNP algorithm <br /> tations previously reported EMS.                                           (http://www.proweb.org/parsesnp/) based <span id='am-25' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-26'  ></span><span id='am-27' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>codon</span> compo- <br />      Given (1) average size Drosophila ORFs (1.69 kb)                   sition Drosophila genes, taking account EMS causes <br /> (Misra et al. 2002), (2) approximate number genes                G:C T:A transitions. This implies average recovery least <br /> second chromosome according flybase (5065 genes; ORFsII),                   2.8 inactivating mutations typical Drosophila gene with <br /> (3) ratio essential genes Drosophila (Miklos Rubin               1.69 kb ORF library 2086 fly lines (1915 + 171 the <br /> 1996) (around three), (4) frequency SNPs generated               EMS concentration experiments): <br /> by EMS (1 156 kb; Table 1), (5) number lethal hits <br /> per chromosome II mutagenesis (2.24 hits/chromosome;                                                      kb   1 SNP 1 SNPinactivating <br />                                                                                            2086 lines &#215; 1.69        &#215;      &#215; <br /> Table 1), estimate, (a) average number total SNPs                                              ORF 156 kb     8.2 SNP <br /> ORFs mutagenized chromosome II mutagenesis                                                      = 2.8 SNPinactivating typical ORF <br />  <br />  <br />                                                                                                                          Genome Research           719 <br />                                                                                                                               www.genome.org <br />  Winkler et al. <br />  <br />  <br />                                                                                   Alp23B, Arf6 represent loss functions respective <br />                                                                                   genes (Fig. 3). In case Sara, previously knew an <br />                                                                                   essential gene (C. B&#246;kel M. Gonz&#225;lez-Gait&#225;n, unpubl.) whose <br />                                                                                   mutations lead embryonic lethality removing ma- <br />                                                                                   ternal zygotic components. Five 21 missense Sara muta- <br />                                                                                   tions cause semilethality maternal zygotic compo- <br />                                                                                   nents removed (see Methods). This represents efficiency of <br />                                                                                   20% total number 25 Sara SNPs screening 3.2 Mb <br />                                                                                   (Fig. 3). For Alp23B, Drosophila TGF-&#65533;?&#65533; homolog, not <br />                                                                                   known essential gene. We identified 14 Alp23B <br />                                                                                   SNPs (2.5 Mb screened), missense alleles and <br />                                                                                   nonsense mutation. The nonsense Alp23B mutation leads <br />                                                                                   stop codon prodomain region, resulting truncated <br />                                                                                   protein lacks mature, secreted TGF-&#65533;?&#65533; peptide. Therefore, <br />                                                                                   mutation represents &#8220;signaling null&#65533;? allele Alp23B. Us- <br />                                                                                   ing deletions Alp23B genomic region (see Methods), we <br />                                                                                   performed complementation tests null allele order to <br />                                                                                   address Alp23B essential gene. This mutation leads <br />                                                                                   85%&#8211;100% lethality, depending genetic background, <br />                                                                                   indicating Alp23B essential viability (see Methods). In <br />                                                                                   case Arf6, determined gene essential for <br />                                                                                   viability, fertility males (M. Gonz&#225;lez-Gait&#225;n, un- <br />                                                                                   publ.). After pilot screening 0.4 Mb, identified Arf6 <br />                                                                                   SNPs, missense mutations mu- <br />                                                                                   tation conserved residue splice acceptor. None the <br />                                                                                   missense mutations overtly affect Arf6 function male <br /> Figure 1. Cel-I-based Tilling strategy. We prepared genomic DNA <br /> library mutagenized b pr cn sca/CyO, hb-lacZ fly lines per- <br />                                                                                   spermatogenesis. These results indicate Tilling approach <br /> formed three-step (1,2,3) nested PCR reaction (A) particular gene         used obtain inactivating mutations genes interest. <br /> of interest. The PCR reactions included IRDye700 (red)              Of course, missense mutations candidates hypomor- <br /> IRDye800 (blue) fluorophores bound forward reverse primers,            phic conditional alleles. <br /> thereby differentially labeling ends amplicons (B). Then we <br />                                                                                         EMS reported alkylate guanine residues, producing O6- <br /> denatured reannealed fragments generate balancer/mutant <br /> heteroduplexes (C) digested Cel-I (D). When SNP pres-           ethylguanine, pairs T, C, leading the <br /> ent, cleavage heteroduplexes generates fragments labeled            replacement G/C base pair A/T (Snow et al. 1984). This <br /> IRDye700 IRDye800, respectively, detected de-               predicts strong G/C-to-A/T <span id='am-1' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-2' property="rdfs:label" content="Bias" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>bias</span> EMS-induced mutations as <br /> naturing polyacrylamide gel electrophoresis LI-COR setup. A typical          observed numerous mutagenesis studies. Indeed, 37 (84%) of <br /> LI-COR signal shows fragments complementary sizes must <br /> sum original amplicon size. Subsequently, sequenced PCR             44 mutations observed correspond G/C-to-A/T transitions, <br /> products confirm molecular lesions (see Methods).                          confirming prediction (Table 1). Another mutations are <br />                                                                                   A/T-to-T/A transversions kinds changes. <br />                                                                                   This result contrast previous report 100% were <br />              1.66 SNPinactivating ORF kb                                   G/C-to-A/T transitions, based 16 novel EMS mutations Dro- <br />  <br />                              &#20873;   1.66 SNPinactivating = <br />                                                           2.8 SNPinactivating <br />                                                             1.69 kb &#65533;? ORF <br />                                                                             &#20874; . <br />                                                                                   sophila detected DHPLC (Bentley et al. 2000). This be <br />                                                                                   different sensitivity detect mutations two <br />                                                                                   methods (Cel-I vs. DHPLC). Among 44 mutations obtained, <br />     This corresponds frequency 1.3 &#11503; 10&#11502;3 inactivating                    33 represent missense mutations, nonsense, 10 are <br /> mutations typical Drosophila gene haploid genome                         silent map noncoding regions. Only represented mu- <br />  <br />  <br />       &#20873; <br />                                                                                   tation affecting splice acceptor. This spectrum corresponds <br />           1.3 &#215; 10&#8722;3 inactivating mutation haploid genome                     roughly frequency mutations expected EMS treat- <br />                                                                                   ment predicted analysis PARSESNP algorithm <br />  <br />                = <br />                    2.8 SNPinactivating <br />                       2086 lines <br />                                        &#20874; <br />                                        , <br />                                                                                   amplicons considered (data shown). <br />                                                                                         Being rapid cost-effective, generation mutants by <br />                                                                                   Tilling will, future, routine approach reverse genet- <br /> which consistent average number inactivating                   ics fly community. Indeed, currently generating <br /> hit gene 1000 lines EMS mutagenesis                     mutant libraries X chromosome. Much like the <br /> experience previous reports (Grigliatti 1986). Therefore,               Arabidopsis field, results enable fly researchers organize the <br /> typical gene interest, probability recovering              generation mutants Tilling screens efficient resource <br /> inactivating mutation library estimated, assuming                   entire community. <br /> a Poisson distribution, as <br />  <br />                 P(n &#11350; 1) = 1 &#11502; e&#11502;&#65533;?&#65533; = 1 &#11502; e&#11502;2.8 = 0.94,                            Methods <br /> i.e., library 94% saturated. In order reach 98% satura-                 Isogenization <br /> tion, currently extending library 3000 lines.                       The b pr cn sca chromosome mutagenized isogenized <br />       We studied mutations obtained Sara,                    according standard procedures. In short, single male was <br />  <br />  <br /> 720        Genome Research <br />              www.genome.org <br />                                                                                                                                 Tilling Drosophila <br />  <br />  <br />  <br />  <br /> Figure 2. Tilling screen. Example gel used mutation detection Sara-4 amplicon 96 balanced mutant lines; 700 (left) 800 nm <br /> (right) emission images showing fragments generated Cel-I-based heteroduplex cleavage. (Arrowheads) Fragments generated cleavage 14 <br /> different silent background polymorphisms balancer reidentified sequencing; (circles) novel mutations. Notice the <br /> polymorphisms novel mutations appear channels complementary sizes. In case second mutation, longer fragment <br /> (800-nm channel) distinguishable (asterisk). After sequencing, could, however, reconfirm signal corresponds bona fide mutation. <br /> Note silent polymorphisms present 96 lines, reflecting fact balancer itself isogenic. New mutations appear in <br /> individual lines. Full-length product, 747 bp (arrow). Cleavage new mutants yields 440 bp (700 nm) + 315 bp (800 nm) (lane 4) 90 bp <br /> (700 nm) 660 bp (800 nm) (lane 80). <br />  <br />  <br /> crossed CyO/In(2L)nocSco females 10 single F1 b pr cn sca/               water transferred vials containing paper tissue <br /> CyO males crossed independently CyO/Df(2L) females                   soaked 10 mL 40% sucrose solution containing 50 or <br /> amplify different individual b pr cn sca chromosomes. F3 b pr cn                125 mM EMS (Sigma), kept 24 h. Afterward, <br /> sca/CyO males virgin females crossed generate 10                    transferred vials let recover and <br /> stocks, scored viability homozygous b pr cn               defecate 30 min, finally transferred food vials groups <br /> sca chromosome.                                                                 30 30 CyO, hb-lacZ/Sp, hs-hid virgin females (G0 <br />                                                                                 cross) maintained 25&#176;C. After 4 d, G0 males were <br /> Mutagenesis                                                                     discarded avoid mutation clustering, G0 females trans- <br /> Mutagenesis performed according standard protocols                       ferred new food vial. The F1 b p cn sca/CyO, hb-lacZ males <br /> (Grigliatti 1986). Isogenized b pr cn sca males (100 vial)             containing mutagenized chromosome collected and <br /> kept 25&#176;C overnight vials containing paper tissue soaked                crossed individually CyO, hb-lacZ/Sp, hs-hid virgin fe- <br />  <br />  <br />  <br />  <br /> Figure 3. Mutation spectrum genes. Genomic organization Sara, Alp23B, Arf6. White boxes represent ORFs. Black boxes below <br /> (Sara-1 5, Alp23B-1 3, Arf6-1,2) correspond position amplicons considered study. Total DNA screened, number SNPs, <br /> missense, nonsense, lethal mutations listed. Lines ORF introns represent position balancer polymorphisms. <br />  <br />  <br />                                                                                                                          Genome Research             721 <br />                                                                                                                             www.genome.org <br />  Winkler et al. <br />  <br />  <br /> males (F1 cross) room temperature. After 4 d, crosses       DNA preparation. Genomic DNA diluted 1:50 ultrapure <br /> heat shocked incubating vials water bath 38.3&#176;C      H2O, 5-&#181;L aliquots arrayed 384-well PCR plates using <br /> 75 min. If dead larvae observed vial, heat     Tecan Genesis workstation 200. The plates sealed with <br /> treatment prolonged 10 minutes water bath.          aluminum foil tape (3MM) stored &#11502;20&#176;C. A copy the <br /> Individual mutant fly stocks maintained 18&#176;C, trans-         original DNA preparation (masterplate) stored indepen- <br /> ferred new food vial month. A total 25 flies        dently &#11502;20&#176;C. <br /> each fly stock frozen processed genomic DNA <br /> extraction.                                                             Primer design <br />      To determine number lethal hits chromosome II, <br />                                                                         Nested primer sets span respective genomic region were <br /> the frequency stocks containing homozygous viable b pr cn <br />                                                                         designed automatically using modified PRIMER3-based ap- <br /> sca* chromosome (f) determined. From null term the <br />                                                                         plication (http://primers.niob.knaw.nl/; Wienholds et al. 2003). <br /> Poisson distribution, f = P(0) = e&#11502;&#65533;?&#65533;, &#65533;?&#65533; hit frequency <br />                                                                         This application searches gene-specific outer primers, well <br /> per chromosome. Therefore, &#65533;?&#65533; = &#8211;lnf. <br />                                                                         corresponding inner primers universal 5&#11032; overhangs (M13 <br />      To compare detection efficiency flies containing and <br />                                                                         forward: TGTAAAACGACGGCCAGT, M13 TILLING: AGGA <br /> not containing polymorphisms balancer, b p cn sca*/ <br />                                                                         AACAGCTATGACCAT). A set primers comprises the <br /> CyO, hb-lacZ males 125 mM EMS fly lines (171 <br />                                                                         M13-forward M13-TILLING sequence, primer is <br /> lines) crossed virgins original b pr cn sca isogenic <br />                                                                         linked IRDye700 IRDye800 dyes, respectively. <br /> stock b pr cn sca*/b pr cn sca F1 flies selected extract <br />                                                                              In total, 10 sets primers automatically designed <br /> genomic DNA prepare &#8220;isogenic library&#65533;? described below. <br />                                                                         different genes (Sara, Alp23B, Arf6) screen the <br />                                                                         Drosophila TILLING library new mutations (Fig. 3; Table 1; <br /> Drosophila strains lethality tests                                  Supplemental Table 2). <br /> The alleles used work described reference (Lindsley <br /> and Zimm 1992) flybase (http://flybase.bio.indiana.edu/).           Cel-I-mediated screen mutations <br /> The Sara&#11502; allele deletion caused imprecise excision a <br />                                                                         DNA amplified nested PCR approach de- <br /> transposable element. <br />                                                                         scribed in Wienholds et al. (2003). The PCR was <br />       To test lethality Sara alleles, Sara&#11502;/Sara&#11502; virgins <br />                                                                         performed gene-specific primers. The PCR product <br /> were crossed <span id='am-30' about='obo:NCBITaxon_1046922' typeof='owl:Thing obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_1206794 obo:NCBITaxon_131567 obo:NCBITaxon_88770 obo:NCBITaxon_6854 obo:NCBITaxon_768140 obo:NCBITaxon_61977 obo:NCBITaxon_6843 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-31' property="rdfs:label" content="Sarax" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-34'  ></span>Sarax</span>/CyO, hb-lacZ males containing allele of <br />                                                                         served template reaction using mix gene-specific <br /> interest. The number Sara&#11502;/Sarax versus Sara&#11502;/CyO, hb-lacZ <br />                                                                         inner primers linked universal 5&#11032; overhangs (second <br /> flies recorded assess lethality/semilethality associated <br />                                                                         PCR), IRDye-labeled primers described (third <br /> with allele. <br />                                                                         PCR). The resulting labeled PCR product denatured re- <br />       To test lethality Alp23B alleles, complementation <br />                                                                         annealed allow heteroduplex formation, digested Cel-I- <br /> tests performed using deletions Alp23B genomic <br />                                                                         mediated cleavage, fragments visualized LI-COR <br /> region, i.e., Df(2)C144 (Lindsley Zimm 1992) deletion <br />                                                                         4200 DNA sequencers. <br /> generated imprecise excision P-element (Df(2)) which <br />                                                                               Pipeting steps performed Tecan Genesis worksta- <br /> uncovers Alp23B-flanking genes. The Alp23B stop codon muta- <br />                                                                         tion 200 manually multichannel pipettes (Eppendorf). <br /> tion leads lethality (0 Df(2)/b pr cn sca* vs. 59 CyO/b pr cn sca* <br />                                                                         PCRs amplified Multicycler PTC 225 Tetrad four <br /> progeny) semilethality (five Df(2)C144/b pr cn sca* vs. 36 CyO/b <br />                                                                         MotoBlocks384 (MJ Research) integrated Tecan <br /> pr cn sca*) depending background. Semilethality the <br />                                                                         Genesis workstation, alternatively GeneAmp9700 (Ap- <br /> case Df(2)C144 known dpp second hit on <br />                                                                         plied Biosystems). <br /> the Defeciency chromosome, behave dominant <br />                                                                               Cycling conditions PCR follows: 94&#176;C for <br /> suppressor mutation (M. Gonz&#225;lez-Gait&#225;n M. F&#252;rthauer, unpubl.). <br />                                                                         1 min, 35 cycles 94&#176;C 20 sec, 57&#176;C 30 sec, 72&#176;C 1 <br />       The Arf6 alleles tested monitoring male sterility a <br />                                                                         min, followed final extension 3 min 72&#176;C (volume: 10 <br /> complementation test Arf6 null alleles generated lab. <br />                                                                         &#181;L). The second/third, nested PCR final volume of <br />                                                                         10 &#181;L (cycling conditions: 94&#176;C 1 min, 30 cycles 94&#176;C for <br /> Genomic DNA preparation construction Drosophila              20 sec, 57&#176;C 30 sec, 72&#176;C 1 min, final extension for <br /> DNA library                                                             3 min 72&#176;C). Heteroduplex formation performed de- <br /> A total 10 (Mad12, cniAR55, tkv7) 25 flies (50 mM, 125        naturing slow reannealing PCR fragments (conditions: <br /> mM EMS library) anesthetized, collected Eppendorf tubes         95&#176;C 3 min, 30 cycles 95&#176;C 20 sec decrement of <br /> and frozen &#11502;80&#176;C 20 min. Flies ground 100 (10            0.5&#176;C cycle, 50 cycles 80&#176;C 1 min decrement of <br /> flies), 200 &#181;L (25 flies) lysis buffer (100 mM Tris-Hcl pH     0.5&#176;C cycle, 55&#176;C 20 min, 20 cycles 55&#176;C 1 min with <br /> 7.5, 100 mM EDTA, 100 mM NaCl, 0.5% SDS) disposable              decrement 0.5&#176;C cycle, finally, 40 cycles 45&#176;C for <br /> tissue grinder (Eppendorf) incubated 65&#176;C 30 min. A          30 sec decrement 0.5&#176;C cycle). A total 5 &#181;L the <br /> total 200 (10) 400 (25) &#181;L 1.4 M KAc, 4.3 M LiCl solution      processed PCR fragments digested 1:1000 dilution of <br /> was added, followed incubation ice 10 min. Cell debris        Cel-I enzyme. Cel-I isolated described (Wienholds et al. <br /> was precipitated spinning 13,000 rpm 15 min room           2003), tested activity, titrated dilution series on <br /> temperature. Supernatant (0.5 mL) transferred Ep-         proof principle samples. The Cel-I digest stopped add- <br /> pendorf tubes (Mad12, cniAR55, tkv7 primary hits geno-      ing 5 &#181;L 75 mM EDTA sample. Sample purification was <br /> typing) deep 96-well plates (1.1 mL cavities, 50 mM, 125 mM          using Sephadex G50 (medium coarse) columns, and <br /> EMS library). A total 200 (10) 400 (25) &#181;L Isopropanol     samples concentrated final volume 2 &#181;L 35 min <br /> added, mixed, DNA precipitated centrifugation             85&#176;C. In total, 0.5 &#181;L loaded 5.5% polyacrylamide gels <br /> 3671g 1 h room temperature. The DNA pellets washed          (25 cm) LI-COR 4200 DNA sequencers. The LI-COR slab gel <br /> with 70% ethanol, dried, resuspended 50 (10 flies) 150        sequencers used perform high-resolution DNA electropho- <br /> &#181;L (25 flies) TE (pH 7.5). Aliquots genomic DNA sepa-           resis combination two-channel infrared detection the <br /> rated 0.7% agarose gels check yield variability    fluorescently labeled (700 800 nm emission) DNA fragments. <br />  <br />  <br /> 722     Genome Research <br />           www.genome.org <br />                                                                                                                            Tilling Drosophila <br />  <br />  <br /> Potential mutations detected scored gel pictures                 recovery mutations Drosophila. Genetics 156: 1169&#8211;1173. <br /> that produced raw TIFF images using Adobe Photoshop.            Colbert, T., Till, B.J., Tompa, R., Reynolds, S., Steine, M.N., Yeung, A.T., <br />                                                                              McCallum, C.M., Comai, L., Henikoff, S. 2001. High-throughput <br />                                                                              screening induced point mutations. Plant Physiol. 126: 480&#8211;484. <br /> DNA sequencing                                                            Grigliatti, T. 1986. Mutagenesis. In Drosophila. A practical approach (ed. <br /> Presumptive SNPs detected Cel-I-mediated digestion              D.B. Roberts), pp. 39&#8211;59. IRL Press Ltd., Oxford, UK. <br />                                                                           Henikoff, S., Till, B.J., Comai, L. 2004. TILLING. Traditional <br /> were reamplified respective PCR product veri-             mutagenesis meets functional genomics. Plant Physiol. 135: 630&#8211;636. <br /> fication. The second PCR way described            Lindsley, D.L. Zimm, G.G. 1992. The genome Drosophila. <br /> before, fluorescently labeled primers. A total 8 &#181;L           Academic Press, London, UK. <br /> of second PCR product digested 0.3 U Exonuclease             Loechler, E.L., Green, C.L., Essigmann, J.M. 1984. In vivo <br />                                                                              mutagenesis O6-methylguanine built unique site viral <br /> 1 1.6 U Shrimp Alkaline Phosphatase 15 min 37&#176;C,                  genome. Proc. Natl. Acad. Sci. 81: 6271&#8211;6275. <br /> followed 15 min 72&#176;C inactivate enzymes. A total          McCallum, C.M., Comai, L., Greene, E.A., Henikoff, S. 2000. <br /> 0.5&#8211;1 &#181;L (about 50 ng DNA) enzyme-purified PCR prod-               Targeted screening induced mutations. Nat. Biotechnol. <br /> ucts used template sequencing. Sequencing                  18: 455&#8211;457. <br />                                                                           Miklos, G.L. Rubin, G.M. 1996. The role genome project in <br /> performed universal sequencing primers (M13 forward,                    determining gene function: Insights model organisms. Cell <br /> M13 TILLING) gene-specific internal primers cover com-                 86: 521&#8211;529. <br /> plete amplicons (sequencing mix: 0.5 pmol primer, 1 &#181;L BigDye             Misra, S., Crosby, M.A., Mungall, C.J., Matthews, B.B., Campbell, K.S., <br /> terminator version 3.1 Sequencing kit, 2 &#181;L 5&#11503; Sequencing                    Hradecky, P., Huang, Y., Kaminker, J.S., Millburn, G.H., Prochnik, <br />                                                                              S.E., et al. 2002. Annotation <span id='am-13' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span><span id='am-16'  ></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Drosophila melanogaster</span> <br /> buffer [Applied Biosystems] total volume 10 &#181;L cycling               euchromatic genome: A systematic review. Genome Biol. <br /> conditions, i.e., 96&#176;C 1 min, 30 cycles 96&#176;C 10 sec, 52&#176;C         3: 0083.0081&#8211;0083.0022. <br /> for 5 sec, 60&#176;C 3 min). After purification ethanol             Moore, L.A., Broihier, H.T., Van Doren, M., Lunsford, L.B., and <br /> precipitation, sequencing products analyzed 3730 DNA               Lehmann, R. 1998. Identification genes controlling germ cell <br />                                                                              migration embryonic gonad formation Drosophila. <br /> Analyzer (Applied Biosystems) DNA sequencing facility              Development 125: 667&#8211;678. <br /> the MPI-CBG. Eventually, sequence traces screened het-           Nellen, D., Affolter, M., Basler, K. 1994. Receptor serine/threonine <br /> erozygous SNPs using Polyphred (http://droog.mbt.washington.                 kinases implicated control Drosophila body pattern by <br /> edu/PolyPhred.html).                                                         decapentaplegic. Cell 78: 225&#8211;237. <br />                                                                           Oleykowski, C.A., Bronson Mullins, C.R., Godwin, A.K., Yeung, A.T. <br />       To reconfirm presumptive mutants, genomic DNA <span id='am-38' about='obo:NCBITaxon_238707' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_8075 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_1489917'><span id='am-39' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-40'  ></span><span id='am-41' property="rdfs:label" content="Iso" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>iso</span>-                 1998. Mutation detection using novel plant endonuclease. Nucleic <br /> lated corresponding fly stock described (10               Acids Res. 26: 4597&#8211;4602. <br /> flies preparation), diluted 1:20 ultra pure water, used         Robertson, H.M., Preston, C.R., Phillis, R.W., Johnson-Schlitz, D.M., <br /> template nested PCR approach. Verification hits               Benz, W.K., Engels, W.R. 1988. A stable genomic source P <br />                                                                              element transposase Drosophila <span id='am-43' about='obo:NCBITaxon_80614' typeof='owl:Thing obo:NCBITaxon_80611 obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_452333 obo:NCBITaxon_5204 obo:NCBITaxon_451864 obo:NCBITaxon_68889 obo:NCBITaxon_155619 obo:NCBITaxon_2759'><span id='am-44'  ></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Melanogaster" datatype="xsd:string"></span>melanogaster</span>. Genetics <br /> always resequencing sort false-positive hits             118: 461&#8211;470. <br /> resulted presumptive Taq-polymerase errors.                          Rong, Y.S., Titen, S.W., Xie, H.B., Golic, M.M., Bastiani, M., <br />                                                                              Bandyopadhyay, P., Olivera, B.M., Brodsky, M., Rubin, G.M., and <br />                                                                              Golic, K.G. 2002. Targeted mutagenesis homologous <br /> Acknowledgments               <br /> </body></html>