<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Sept. 1998, p. 4406&#8211;4412                                                                                     Vol. 180, No. 17 <br /> 0021-9193/98/$04.0010 <br /> Copyright &#169; 1998, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />          Contribution Pmra Promoter Expression Genes in <br />                <span id='am-19' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-20' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-22'  ></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span>Escherichia coli</span> mra Cluster Cell Envelope <br />                      Biosynthesis Cell Division Genes <br />                         DOMINIQUE MENGIN-LECREULX,1* JUAN AYALA,2 AHMED BOUHSS,1 <br />                                 JEAN VAN HEIJENOORT,1 CLAUDINE PARQUET,1 <br />                                                             3 <br />                                             AND HIROSHI <span id='am-6' about='obo:NCBITaxon_385297' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_186628 obo:NCBITaxon_1 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7995 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489793 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_385271 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-7' property="rdfs:label" content="Hara" datatype="xsd:string"></span><span id='am-8'  ></span><span id='am-9' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>HARA</span> &#8224; <br />  <br />                  Laboratoire des Enveloppes Bacte&#180;riennes, Centre National la Recherche Scientifique, Universite&#180; <br />                       <span id='am-47' about='obo:NCBITaxon_49669' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_50362 obo:NCBITaxon_4447 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_4667 obo:NCBITaxon_3193'><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasAlternativeId" content="NCBITaxon:49665" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="Paris L." datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="Paris" datatype="xsd:string"></span><span id='am-53'  ></span>Paris</span>-Sud, 91405 Orsay Cedex, France1; Centro Biolog&#305;&#180;a Molecular Severo Ochoa, <br />                               Consejo Superior Investigationes Cient&#305;&#180;ficas, Universidad Auto <br />                                                                                                &#180;noma <br />                                   Madrid, Canto Blanco, 28049 Madrid, Spain2; National <br />                                      Institute Genetics, Mishima, Shizuoka-ken 411, Japan3 <br />                                                Received 7 May 1998/Accepted 1 July 1998 <br />  <br />             Recently, promoter essential gene ftsI, encodes penicillin-binding protein 3 <span id='am-35' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-36' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-37'  ></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span>Escherichia</span> coli, <br />          precisely localized 1.9 kb upstream gene, beginning mra cluster cell division and <br />          cell envelope biosynthesis genes (H. Hara, S. Yasuda, K. Horiuchi, J. T. Park, J. Bacteriol. 179:5802&#8211;5811, <br />          1997). Disruption promoter (Pmra) chromosome replacement lac promoter <br />          (Pmra::Plac) led isopropyl-b-D-thiogalactopyranoside (IPTG)-dependent cells lysed absence of <br />          inducer, defect complemented region Pmra ftsW, fifth gene <br />          downstream ftsI, provided trans plasmid. In present work, levels various proteins <br />          involved peptidoglycan synthesis cell division precisely determined cells Pmra::Plac <br />          promoter expression repressed fully induced. It confirmed Pmra promoter required for <br />          expression genes mra cluster: mraZ (orfC), mraW (orfB), ftsL (mraR), ftsI, murE, murF, <br />          mraY, murD, ftsW. Interestingly, three- sixfold-decreased levels MurG MurC enzymes were <br />          observed uninduced Pmra::Plac cells. <span id='am-14' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-17'  ></span><span id='am-18' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> correlated accumulation nucleotide precursors <br />          UDP&#8211;N-acetylglucosamine UDP&#8211;N-acetylmuramic acid, substrates enzymes, depletion <br />          pool UDP&#8211;N-acetylmuramyl pentapeptide, resulting decreased cell wall peptidoglycan synthesis. <br />          Moreover, expression ftsZ, penultimate gene cluster, significantly reduced Pmra <br />          expression repressed. It concluded transcription genes located downstream ftsW <br />          mra cluster, murG ftsZ, mainly (but exclusively) dependent Pmra promoter. <br />  <br />  <br />    The rigid, shape-determining material bacterial cell walls            crucial question genes large mra cluster <br /> is giant polymer periodic structure named peptidoglycan                transcribed open. A promoter ftsI gene <br /> or murein. Its biosynthesis complex process involving             (also named pbpB), encodes penicillin-binding protein 3 <br /> different cytoplasmic membrane steps (33). Conditional-                 (PBP3) (7, 19, 28), genes upstream was <br /> lethal mutants Escherichia coli altered different levels           recently identified (13). Interestingly, inactivation this <br /> this metabolic pathway described previously,                  promoter (Pmra) chromosome replacement an <br /> most mutations mapped regions                 inducible promoter (Plac) led isopropyl-b-D-thiogalactopy- <br /> chromosome (10, 27, 36). One them, 2 min E. coli               ranoside (IPTG)-dependent cells grow ab- <br /> map, studied great contained large               sence inducer plasmid carrying the <br /> cluster genes, mraZ envA, code proteins                 mraZ-ftsW region present (13). It concluded that <br /> involved cell envelope biosynthesis cell division. It            Pmra promoter essential expression nine <br /> designated mra murein region A (7, 19, 27) dcw                genes mra cluster. In fact, report uninduced <br /> for division cell wall (1, 9, 34, 35). Through earlier work          Pmra::Plac cells significantly depleted products of <br /> our laboratories, complete physical map                   different genes located downstream ftsW, indicating <br /> DNA sequence 17-kb region determined                      main proportion transcription genes <br /> and function of genes present cluster               (from murG ftsZ) derives Pmra promoter. <br /> were identified. However, exception cell divi- <br /> sion genes ftsQ, ftsA, ftsZ, transcription                                      MATERIALS AND METHODS <br /> investigated great (4, 9, 11, 29, 31, 34, 35, 37), the <br />                                                                                Materials. Acetyl coenzyme A (acetyl-CoA), glucosamine-1-phosphate (GlcN- <br />                                                                             1-P), UTP, UDP&#8211;N-acetylglucosamine (UDP-GlcNAc) bought from <br />                                                                             Sigma. Peptidoglycan nucleotide precursors dipeptide D-<span id='am-54' about='obo:NCBITaxon_1416571' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_116706 obo:NCBITaxon_6804 obo:NCBITaxon_131567 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_6682 obo:NCBITaxon_6657 obo:NCBITaxon_1206794 obo:NCBITaxon_6683 obo:NCBITaxon_88770 obo:NCBITaxon_6752 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_553065 obo:NCBITaxon_6072'><span id='am-55' property="rdfs:label" content="Ala" datatype="xsd:string"></span><span id='am-56'  ></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Ala</span>&#8211;D-Ala were <br />   * Corresponding author. Mailing address: Laboratoire des Envel-           prepared described previously (20&#8211;24). UDP-[14C]GlcNAc (7.4 GBq z <br /> oppes Bacte&#180;riennes, Centre National la Recherche Scientifique,          mmol21), [3H]UMP (0.5 TBq z mmol21), L-[14C]alanine (5.6 GBq z mmol21) <br />                                                                             purchased Amersham; [14C]acetyl-CoA (1.9 GBq z mmol21) from <br /> Universite&#180; Paris-Sud, Ba&#710;timent 432, 91405 Orsay Cedex, France. <br />                                                                             ICN; D-[14C]glutamic acid (1.9 GBq z mmol21) obtained American <br /> Phone: 33-1-69-15-61-34. Fax: 33-1-69-85-37-15. E-mail: dominique           Radiolabeled Chemicals (St. Louis, Mo.); [meso-14C]diaminopimelic acid <br /> .mengin-lecreulx@ebp.u-psud.fr.                                             ([meso-14C]A2pm; 11.5 GBq z mmol21) <span id='am-1' about='obo:NCBITaxon_1107390' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_272274 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7422 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_272201 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7423 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="rdfs:label" content="Cea" datatype="xsd:string"></span>CEA</span> (Saclay, France). <br />   &#8224; Present address: Department Biochemistry Molecular Bi-              Bacterial strains <span id='am-70' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>plasmids</span>. JE7968 (Pmra::Plac) JE7970 (Pmra::Plac <br /> ology, Saitama University, Urawa, Saitama 338-8570, Japan.                  recA1) derivatives W3110 carry Plac cassette inserted the <br />  <br />                                                                      4406 <br />  VOL. 180, 1998                                                                  EXPRESSION OF GENES IN THE E. COLI mra CLUSTER                                     4407 <br />  <br />  <br />  <br />  <br />    FIG. 1. lacZ operon fusion experiments Pmra-murG region mra cluster. The orientation approximate sizes mra cluster genes (black arrows) <br /> and rho-independent terminator (a bracket T beneath it) shown top. Below, filled bars denote chromosomal fragments cloned promoter <br /> assay mini-F vector pFZY1DH. Their left ends, cut HpaI, joined filled EcoRI site vector. Short open bars denote SmaI-BamHI V <br /> interposon polylinker (8). Open arrowheads P&#8217;s beneath represent Pmra, disrupted Pmra (Pmra::4 bp), Pmra replaced Plac (Pmra::Plac). The plasmid <br /> numbers left Pmra, Pmra::4 bp, Pmra::Plac, respectively. b-Gal activities averages measurements transformants MC1061-5 <br /> grown buffered L broth-glucose-thymine medium (13) containing ampicillin, (1) (2) IPTG concentration 1 mM. The figure drawn scale <br /> except galK9-9lacZ fusion gene (large white arrows) short upstream region, including polylinker (dotted line) vector (13, 17). Abbreviations <br /> for restriction sites (only relevant sites shown): B, BamHI; E, EcoRI; Ev, EcoRV; H, HindIII; S, SmaI; Sn, SnaBI. <br />  <br />  <br />  <br /> HindIII site Pmra chromosome (13). Thus, dependent           presence absence 1 mM IPTG. Strains requiring IPTG growth first <br /> a lac inducer growth. The Plac cassette composed cat gene followed       grown presence, cultures diluted 100-fold into <br /> by transcriptional terminators rrnB operon, lacIq gene           prewarmed medium lacking inducer. The effects depletion of <br /> orientation opposite Plac orientation Pmra. MC1061-5             IPTG cell morphology (loss rod shape) cell growth (arrest growth <br /> (DlacX74) (17) used host lacZ operon fusion experiment. pHR416         followed onset cell lysis) observed time period that <br /> is plasmid derived pSY396 carries entire mra cluster (21-kb           depended strain used: 2 h JE7968 JE7970 3 h <br /> AatII fragment) (13). pHR477, pHR478, pHR479, pHR431, pHR439, pHR427,                  JE7970(pHR477/pHR485). The 1-h delay observed strain was <br /> and pHR426 mini-F plasmids carrying chromosomal fragment Pmra             fact biosynthetic activity (MurD) depleted that <br /> to mraY, murD, ftsW, murG, ftsQ, ftsA, ftsZ, respectively. pHR485 carries          strain IPTG deprivation. At time (the final optical density culture <br /> ftsW control aadA promoter vector pGB2 (13). The               approximately 0.7), cells harvested washed 40 ml cold <br /> mini-F plasmids used operon fusion experiments depicted Fig. 1.          20 mM potassium phosphate buffer (pH 7) containing 0.3 mM MgCl2 1 mM <br /> The vector pFZY1DH (13), derivative pFZY1 (17).                               b-mercaptoethanol. The wet cell pellet suspended 5 ml buffer <br />    Recombinant DNA procedures. These methods essentially based                 disrupted sonication (Sonicator 150; T. S. Ultrasons, Annemasse, France) <br /> those Sambrook et al. (30). The b-galactosidase (b-Gal) assay unit          5 min cooling. The resulting suspension centrifuged 4&#176;C 30 <br /> <span id='am-64' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> used described Koop et al. (17).                          min 200,000 3 g Beckman TL100 centrifuge. The supernatant was <br />    Growth conditions. Unless noted, 2YT medium (26) used             dialyzed overnight 4&#176;C 100 volumes phosphate buffer, and <br /> growing cells. Cell growth monitored 600 nm spectrophotometer            resulting solution (40 50 mg protein 5 ml), designated crude <br /> (model 240; Gilford Instrument Laboratories, Inc., Oberlin, Ohio). For strains         soluble enzyme, stored 220&#176;C. The pellet, consisting membrane pro- <br /> carrying drug resistance genes, antibiotics used following concentra-      teins, resuspended 1 ml buffer. Protein concentrations were <br /> tions (in micrograms milliliter): ampicillin, 100; kanamycin, 30; spectinomy-      determined method Lowry et al. (18), using bovine serum albumin as <br /> cin, 30; tetracycline, 12.5; chloramphenicol, 10.                                  standard. <br />    Preparation crude enzyme. Exponential-phase cells (0.5-liter cultures)           Enzymatic assays. (i) GlcN-1-P acetyltransferase (GlmU). The two-step for- <br /> the different strains listed Table 1 grown 37&#176;C 2YT medium,       mation UDP-GlcNAc GlcN-1-P, catalyzed bifunctional enzyme <br />  <br />  <br />  <br />                                   TABLE 1. Specific activities peptidoglycan-synthesizing enzymes E. coli strainsa <br />                                                                                                          Relative amt <br />             Strain                      IPTG <br />                                                        MurE            MurF           MraY            MurD           MurG            MurC            Ddl          GlmU <br />  <br /> W3110                                     2             1.0             1.0            1.0            1.0               1.0           1.0            1.0           1.0 <br /> JE7968                                    2             0.25            0.2            0.35           0.25              0.4           0.3            0.7           1.0 <br />                                           1             3.0             2.0            1.8            2.1               2.0           3.4            1.3           1.0 <br /> JE7970                                    2             0.20            0.3            0.4            0.3               0.4           0.4            0.7           1.0 <br />                                           1             2.5             2.1            2.2            2.1               2.2           3.5            1.2           1.0 <br /> JE7970(pHR479)                            2             1.0             1.1            1.0            1.2               0.3           0.15           0.8           1.0 <br />                                           1             2.5             2.2            1.5            2.7               1.5           3.2            1.2           1.0 <br /> JE7970(pHR477/pHR485)                     2             1.0             1.2            0.9            0.05              0.4           0.4            0.8           1.0 <br />                                           1             2.5             2.2            1.5            2.0               2.0           3.2            1.3           1.0 <br /> JE7970(pHR439)                            2             1.0             1.1            1.1            1.0               1.1           1.1            1.1           1.0 <br />    a <br />      Cells grown exponentially 37&#176;C 2YT medium supplemented (1) supplemented (2) 1 mM IPTG. IPTG-requiring strains grown in <br /> the presence inducer diluted prewarmed medium IPTG. Growth continued effects cell growth observed, <br /> from 2 3 h later, depending strain used (see text). Crude extracts (soluble membrane fractions) prepared analyzed different enzyme <br /> activities. A value 1 wild-type strain W3110 corresponds (in nanomoles z minute21 z milligram protein21) 1.5 MurE, 3 MurF, 0.6 MurD, 0.04 <br /> for MraY, 0.02 MurG, 0.06 MurC, 0.8 Ddl, 103 GlmU, respectively. The specific activity GlmU, enzyme involved early steps peptidoglycan <br /> synthesis encoded gene separate chromosomal region (25), used internal standard control absence variability extracts. <br />  4408       MENGIN-LECREULX ET AL.                                                                                                          J. BACTERIOL. <br />  <br />  <br /> GlcN-1-P acetyltransferase (GlmU), standard assay mixture containing 50                                      RESULTS <br /> mM Tris-HCl buffer (pH 8.0), 2 mM UTP, 3 mM MgCl2, 0.5 mM [14C]acetyl- <br /> CoA (700 Bq), 2 mM GlcN-1-P, enzyme (10 mg protein) final volume <br /> of 100 ml monitored. <br />                                                                                           Levels peptidoglycan-synthesizing enzymes Pmra::Plac <br />    (ii) L-Alanine-adding enzyme (MurC). The formation UDP&#8211;N-acetyl-                 cells. To examine far mra cluster function of <br /> muramyl (MurNAc)&#8211;L-Ala standard assay mixture containing 100 mM Tris-             Pmra promoter required, recently disrupted this <br /> HCl buffer (pH 8.6), 5 mM ATP, 20 mM MgCl2, 2 mM L-[14C]alanine (2 KBq),               promoter chromosome replaced induc- <br /> 1 mM UDP-MurNAc, enzyme (250 mg protein) final volume 50 ml             ible lac promoter (13). An IPTG-dependent strain, JE7968, <br /> was monitored. <br />    (iii) D-Glutamic acid-adding enzyme (MurD). The formation UDP&#8211; <br />                                                                                        resulted construction; strain shown lyse <br /> MurNAc&#8211;L-Ala&#8211;D-Glu standard assay mixture containing 100 mM Tris-HCl              deprived lac inducer exponential growth, most <br /> buffer (pH 8.6), 5 mM ATP, 5 mM MgCl2, 25 mM D-[14C]glutamic acid (500 Bq),            probably expression genes <br /> 25 mM UDP&#8211;MurNAc&#8211;L-alanine, enzyme (5 mg protein) final volume             involved peptidoglycan synthesis repressed. To control <br /> of 50 ml monitored.                                                                point, levels enzymes encoded different <br />    (iv) meso-A2pm-adding enzyme (MurE). The formation UDP-MurNAc <br /> tripeptide standard assay mixture containing 100 mM Tris-HCl buffer (pH <br />                                                                                        genes mra cluster determined cells strain <br /> 8.6), 5 mM ATP, 100 mM MgCl2, 0.1 mM [meso-14C]A2pm (500 Bq), 0.2 mM                   JE7968 grown continuously presence of <br /> UDP&#8211;MurNAc&#8211;L-Ala&#8211;D-Glu, enzyme (50 mg protein) final volume             1 mM IPTG depleted lac inducer approximately 2 h <br /> 75 ml monitored.                                                                   (until effects cell growth observed). In most <br />    (v) D-Alanyl&#8211;D-alanine-adding enzyme (MurF). The formation UDP-                  cases (for MurE, MurF, MraY, MurD, MurG, MurC), the <br /> MurNAc pentapeptide standard assay mixture containing 100 mM Tris-HCl <br /> buffer (pH 8.6), 5 mM ATP, 100 mM MgCl2, 70 mM D-[14C]Ala&#8211;D-Ala (500 Bq), <br />                                                                                        levels enzymes observed induced JE7968 cells two- <br /> 70 mM UDP&#8211;MurNAc&#8211;L-Ala&#8211;g-D-Glu&#8211;meso-A2pm, enzyme (20 mg pro-                    threefold higher parental W3110 cells <br /> tein) final volume 100 ml monitored.                                       (Table 1). One exception concerned Ddl, activity was <br />    (vi) D-Alanine:D-alanine ligase (Ddl). The standard assay mixture contained         increased 20 30%. As expected, inverse variations <br /> 50 mM Tris-HCl buffer (pH 8.6), 5 mM ATP, 20 mM MgCl2, 50 mM D-[14C]Ala                observed IPTG-depleted cells, contained three- <br /> (1 KBq), 0.12 mM UDP&#8211;MurNAc&#8211;L-Ala&#8211;g-D-Glu&#8211;meso-A2pm, enzyme (20 <br />                                                                                        fivefold different enzymes mentioned than <br /> mg protein) final volume 50 ml. Because reaction product, D-Ala&#8211; <br /> D-Ala, inhibits activity D-alanine:D-alanine ligase (Ddl) (20), quan- <br />                                                                                        wild-type strain W3110, exception Ddl, <br /> titatively converted UDP-MurNAc pentapeptide coupling Ddl activity           decreased 30% (Table 1). The concomitant <br /> to MurF present extract.                                            similar variations enzymes clearly indicated <br />    (vii) Phospho-MurNAc pentapeptide translocase (MraY). The reaction          transcription corresponding genes mainly de- <br /> substitution [3H]UMP UMP moiety UDP-MurNAc pentapeptide <br />                                                                                        pendent Pmra promoter. <br /> was used assay translocase activity 20-ml reaction mixture consisting <br /> of 50 mM Tris-HCl buffer (pH 7.5), 12.5 mM MgCl2, 13 mM [3H]UMP (500 Bq),                 As reported previously, IPTG requirement strain <br /> 0.16 mM UDP-MurNAc pentapeptide, membranes (60 mg protein).                     JE7968 complemented chromosomal fragment <br />    In cases (assays vii), reaction mixtures incubated 37&#176;C        extending Pmra ftsW (inclusive) provided in <br /> 30 min, reactions terminated addition 10 ml acetic acid,         trans plasmid (13). Some constructs in <br /> reaction products separated high-voltage electrophoresis Whatman <br />                                                                                        expression genes impaired when <br /> 3MM filter paper 2% formic acid (pH 1.9) 1 1.5 h 40 V/cm, using an <br /> LT36 apparatus (Savant Instruments, Hicksville, N.Y.). The radioactive spots           IPTG absent&#8212;for instance, ftsW strain JE7970(pHR478) <br /> were located overnight autoradiography type R2 films (3M, St. Paul,            murD strain JE7970(pHR477/pHR485). Each these <br /> Minn.) radioactivity scanner (Multi-Tracermaster model LB285;                strains required IPTG growth filamented lysed when <br /> EG&amp;G Wallac/Berthold, Evry, France). The radioactive spots cut out,           deprived lac inducer, depending function gene <br /> the radioactivity counted Betamatic IV liquid scintillation <br />                                                                                        product cell division peptidoglycan synthesis. <br /> spectrophotometer (Kontron Instruments) solvent consisting 2 <br /> ml water 13 ml Aqualyte mixture (J. T. Baker Chemicals, Deventer,            For instance, observed cells JE7970(pHR477/ <br /> The Netherlands).                                                                      pHR485) stopped growing lysed 3 h expo- <br />    (viii) N-Acetylglucosaminyltransferase (MurG). The standard reaction mix-           nential growth absence IPTG. An analysis cell <br /> ture contained, final volume 25 ml, 100 mM Tris-HCl buffer (pH 7.5),           content time consistent defective expression <br /> 40 mM MgCl2, 30 mM ATP, 0.7 mM UDP-MurNAc pentapeptide, 2 mM UDP- <br />                                                                                        D-glutamic acid-adding enzyme (MurD): 20-fold-reduced <br /> [14C]GlcNAc (1 KBq), membranes (150 mg protein). Membranes were <br /> incubated UDP-MurNAc pentapeptide 10 min 35&#176;C generate            level enzyme detected (Table 1), large amounts of <br /> undecaprenyl-pyrophosphoryl MurNAc pentapeptide (via MraY) addi-                UDP&#8211;MurNAc&#8211;L-Ala (the nucleotide substrate MurD) <br /> tion time zero radioactive substrate 10-min incubation 35&#176;C.        accumulated, precursors located downstream in <br /> The reaction stopped placing tubes boiling-water bath 2 min,       pathway depleted. This resulted 40% lower pep- <br /> the reaction mixtures analyzed descending chromatography 16 h           tidoglycan content, probably minimum value <br /> Whatman 1 filter paper isobutyric acid&#8211;1 M NH4OH (5:3, vol/vol). Spots <br /> corresponding products (peptidoglycan lipid intermediate) remaining         compatible cell viability (<span id='am-65' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-66' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-67'  ></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span> shown). <br /> UDP-GlcNAc substrate detected radioactivity counted                 Cells JE7968 JE7970 (a recA derivative former) <br /> described above.                                                                       IPTG independent growth transformed with <br />    Pool levels peptidoglycan precursors. Cells W3110 JE7970 deriva-          plasmid (pHR479) carrying minimal complementing <br /> tives (1-liter cultures) grown exponentially 37&#176;C 2YT medium         fragment Pmra-ftsW region. However, growth rate of <br /> absence presence IPTG. When optical density cultures reached <br /> 0.7, cells rapidly chilled 0&#176;C harvested cold. The extraction   strain absence IPTG relatively low compared <br /> of peptidoglycan nucleotide precursors boiling water cold trichloroace-       wild-type strain rate presence <br /> tic acid analytical procedure used quantitation previ-       IPTG (generation times 37&#176;C 70, 35, 39 min, <br /> ously described (20, 21).                                                              respectively). As expected, JE7970(pHR479) cells contained <br />    Isolation sacculi quantitation peptidoglycan. Exponential-phase cells     wild-type levels MurE, MurF, MraY, MurD enzymes <br /> (0.5-liter cultures) W3110 JE7970 derivatives grown described <br /> above, absence presence IPTG. Harvested cells washed          three- sixfold-reduced levels MurG MurC ac- <br /> cold 0.85% NaCl solution centrifuged again. Bacteria rapidly             tivities, respectively (Table 1). Since cells grown con- <br /> resuspended vigorous stirring 20 ml hot (95 100&#176;C) aqueous 4%          tinuously absence IPTG, values represented the <br /> sodium dodecyl sulfate solution 30 min. After standing overnight room           production enzymes conditions expres- <br /> temperature, suspensions centrifuged 30 min 200,000 3 g        sion Pmra promoter maximally repressed. The <br /> pellets washed times water. The final suspensions, 2-ml <br /> volumes water, homogenized brief sonication. Aliquots hydro-           cells viable conditions, implying that <br /> lyzed analyzed previously described (24), peptidoglycan content         MurG MurC activities somewhat excess wild- <br /> of sacculi expressed terms muramic acid content.                     type strain residual activity sustain <br />  VOL. 180, 1998                                                                     EXPRESSION OF GENES IN THE E. COLI mra CLUSTER                                  4409 <br />  <br />  <br />       TABLE 2. Abnormal peptidoglycan content pools                             Pmra activity (13), displaced insertion Plac cassette <br />              peptidoglycan precursors uninduced                                     HindIII site (Pmra::Plac; pHR582, pHR584, pHR585 <br />                     JE7970(pHR479) cellsa                                              respectively). <br />                                         Amt (nmol/liter culture) of:                   b-Gal assays, absence functional Pmra (Pmra::4bp) <br />        Strain          IPTG <br />                                                                                        Pmra::Plac activity repressed, indicated the <br />                                 UDP-       UDP-       UDP-MurNAc          Peptido-     level transcription detectable EcoRV site low <br />                                GlcNAc     MurNAc      pentapeptide         glycan <br />                                                                                        compared background b-Gal activity measured the <br /> W3110                   2         70          25           230             2,600       pFZY1DH vector (Fig. 1). Interestingly, Boyle et al. recently <br /> JE7970(pHR479)          2        240         150            90             1,900       localized promoter ftsW gene EcoRI and <br />                         1         75          30           190             3,000       BglII sites mraY-murD intergenic region (3). Since this <br /> JE7970(pHR439)          2         75          30           230             2,500       region just upstream EcoRV site (Fig. 1), contri- <br />   a <br />     Cells grown exponentially 37&#176;C 2YT medium absence (2)            bution promoter low-level transcription detected <br /> or presence (1) 1 mM IPTG. In cases, cells harvested optical         likely. At SnaBI SmaI sites, hand, <br /> density 0.7, corresponding approximately 230 mg bacterial cell (dry           small significant transcription detected. <br /> weight) liter culture. Cell peptidoglycan extracted quantitated      There likely promoter(s), the <br /> terms muramic acid content (20, 24). Nucleotide precursors extracted, <br /> purified, quantitated previously described (20, 21).                            EcoRV site murD SnaBI site ftsW, which <br />                                                                                        responsible residual expression murG down- <br />                                                                                        stream genes observed uninduced Pmra::Plac cells. <br />                                                                                           In presence functional Pmra operon fusion <br /> peptidoglycan synthesis, rate sufficient cell                        plasmids, higher b-Gal activity detected SmaI site <br /> integrity. Further analyses showed cells accumulated                        murG (Fig. 1). The transcription originating Pmra <br /> UDP-GlcNAc UDP-MurNAc, nucleotide substrates                                proceed murG probably beyond, is <br /> the MurG MurC enzymes, respectively (Table 2). This                                necessarily required growth division cell. <br /> finding confirmed partial depletion enzymes                          This consistent increase MurG, MurC, Ddl, and <br /> the cell content indicated significant reduction                          enzymatic activities observed induced JE7970 cells. <br /> flow metabolites pathway peptidoglycan synthesis                         The activity Pmra high Plac fully induced <br /> at enzymatic steps. Consequently, pool UDP&#8211;N-                              1 mM IPTG. <br /> acetylmuramyl pentapeptide, end product cytoplas-                              The finding b-Gal activity detected SmaI site, <br /> mic steps, decreased twofold cell peptidoglycan                            presence functional Pmra Pmra::Plac ac- <br /> content decreased 30% (Table 2). It previously                              tivity induced, half EcoRV and <br /> established peptidoglycan content E. coli cells                      SnaBI sites intriguing. Transcriptional attenuation dif- <br /> be reduced 40 50% loss cell integrity                           ferential degradation mRNA occurring. <br /> (reference 25 references therein). Table 2 shows                             Relative amounts MraW, PBP3, FtsZ E. coli mu- <br /> pools precursors peptidoglycan content strain                            tant strains. We characterized expression genes <br /> JE7970(pHR439), carries murG murC addition                                encoding MraW, PBP3, FtsZ strain JE7970 deple- <br /> on plasmid, normal. The results observed                            tion IPTG inducer 2 h, just lysis cells, by <br /> when strain JE7970(pHR479) grown presence                                measuring relative amounts proteins detected with <br /> IPTG. This finding confirmed variations described                             specific antibodies immunoblotting assay. Increases in <br /> above clearly correlated partial depletion                        levels proteins, compared levels in <br /> MurG MurC enzymes cell content. At ap-                              parental strain, W3110, induction con- <br /> proximation, reduced growth rate attributed                            ditions, decreases observed depletion the <br /> the depletion enzymes, strain carrying                          inducer (Table 3). Note case protein, <br /> in addition murG murC plasmid, JE7970(pHR439), <br /> grew faster, generation time 50 min. <br />    All results taken suggested, first, tran- <br />                                                                                                  TABLE 3. Relative amounts MraW, PBP3, and <br /> scription gene located Pmra-ftsW region                                           FtsZ E. coli mutant strainsa <br /> under control sole Pmra promoter and, second, that <br /> transcription genes distal ftsW originated                                                                        Relative amt 6 SD of: <br />                                                                                               Strain             IPTG <br /> Pmra promoter(s). The finding 70                                                                MraW             PBP3              FtsZ <br /> 80% MurG MurC activities lost repression <br /> of Plac Pmra::Plac cells indicated Pmra ab-                    W3110                       2        100             100               100 <br /> solutely required for, governed main of, tran-                                                    1        106 6 13         88 6 9           109 6 16 <br /> scription corresponding genes.                                                  JE7970                      2         71 6 11         49 6 3            70 6 12 <br />    A promoter Pmra contributes expression                                                     1        207 6 23        660 6 192         148 6 11 <br />                                                                                        JE7970(pHR431)              2        114 6 20        334 6 36           72 6 8 <br /> the murG murC genes. To examine contribution                                JE7970(pHR439)              2         88 6 2         361 6 28           59 6 10 <br /> Pmra expression genes mra cluster,                            JE7970(pHR427)              2        141 6 22         75 6 3           209 6 13 <br /> made fusions proximal region mra cluster                            JE7970(pHR426)              2        132 6 15         70 6 5           217 6 16 <br /> and promoterless galK9-9lacZ gene promoter assay                              JE7970(pHR416)              2        117 6 19         71 6 13          208 6 31 <br /> mini-F vector, pFZY1DH (Fig. 1). The cloned DNA                                 a <br />                                                                                            Cells grown exponentially 37&#176;C 2YT medium supplemented (1) or <br /> the HpaI site just upstream Pmra, EcoRV site                          supplemented (2) 1 mM IPTG. Crude extracts (total MraW and <br /> the 59 end murD (pHR571), SnaBI site                              FtsZ membrane fractions PBP3) prepared differential centrifu- <br /> middle ftsW (pHR577), SmaI site 59                            gation disrupting cells French press. Proteins fractionated a <br /> end murG (pHR578). Pmra disrupted                                   Tricine&#8211;sodium dodecyl sulfate&#8211;8% acrylamide gel detected immuno- <br />                                                                                        blotting specific antibodies chemiluminiscence method. The <br /> HindIII site filling-in reaction T4 DNA                            absorbances corresponding bands film different <br /> polymerase (Pmra::4bp; pHR586, pHR588, pHR589 re-                                  experiments measured expressed relative of <br /> spectively), shown completely abolish                            protein detected parental strain W3110. <br />  4410     MENGIN-LECREULX ET AL.                                                                                             J. BACTERIOL. <br />  <br />  <br /> and enzymatic activity, measured. About twofold           mainly (or completely) dependent Pmra promoter. In <br /> increase MraW sevenfold increase                particular, provided evidence repression the <br /> evident PBP3, suggesting specific regulation          Pmra promoter resulted dramatic depletion murD <br /> two genes level transcription and/or translation. FtsZ        gene product, arrest peptidoglycan synthesis <br /> also showed small (1.5-fold) increase presence              attendant cell lysis. This suggested expression the <br /> inducer. Levels gene products depletion              chromosomal murD gene exclusively dependent this <br /> conditions induction IPTG agreement              promoter and, consequently, putative promoters the <br /> with results obtained mur gene products                region 59 murD functional vivo present in <br /> suggested transcription Pmra proceed ftsZ.          copy. Strains defective product ftsI <br />    Two revertants strains JE7967 JE7970          ftsW gene previously constructed (13). In each <br /> chloramphenicol resistant IPTG independent              case, depletion lac inducer resulted cell division <br /> isolated simply growing cells absence IPTG           defect, corrected addition plasmid <span id='am-59' about='obo:NCBITaxon_201850' typeof='owl:Thing obo:NCBITaxon_1125798 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71529 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-60' property="rdfs:label" content="Car" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-62'  ></span><span id='am-63' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>car</span>- <br /> stationary phase plating 2YT-chloramphenicol                rying defective gene. Strains conditionally defective <br /> medium. Expression genes strains             expression murE, murF, mraY gene been <br /> became IPTG independent, levels proteins            constructed, likely phenomena observed with <br /> quite similar absence presence inducer (data           strains confirmed genes essential and <br /> not shown). This result indicated defect rever-         exclusively expressed Pmra promoter. <br /> tant strains modification lac promoter                We confirmed Pmra promoter absolutely <br /> and/or operator did allow correct binding ex-           required expression genes distal ftsW mra <br /> pression LacI. Whatever changes strains,           cluster. However, clear major proportion the <br /> confirmed expression Pmra::Plac promoter              transcription genes originated promot- <br /> proceed ftsZ.                                                     er, murG murC genes, expression <br />    Expression gene products studied        reduced three- sixfold, respectively, repression <br /> Pmra::Plac strain carrying plasmids different fragments         Pmra. The residual expression gene products sug- <br /> the mra cluster. The results presented Table 3. MraW             gested presence efficient promoter. As shown by <br /> was expressed level strains. Only       lacZ fusion experiments described work, pro- <br /> a small increase expression (15 30%) extracts        moter EcoRV site lying with- <br /> of JE7970(pHR431), JE7970(pHR427), JE7970(pHR426),                  murD SnaBI site ftsW. Since ftsW gene is <br /> JE7970(pHR416) cells, small decrease expression (less          essentially dependent Pmra promoter expression, this <br /> than 10%) evident JE7970(pHR439) difficult           promoter probably resides sequence of <br /> correlate presence plasmids. However, expres-           ftsW, future work devoted identification. It <br /> sion PBP3 FtsZ showed opposite variations               noteworthy genes mra cluster are <br /> strains. In strains JE7970(pHR431) JE7970(pHR439), PBP3             tightly packed 100% DNA in <br /> was increased three- fourfold FtsZ decreased              chromosomal region (EMBL entry EC2MIN) coding. In <br /> by 30 40%. Inversely, strains JE7970(pHR427), JE7970              particular, murG gene, coding N-acetylglucos- <br /> (pHR426), JE7970(pHR416), FtsZ increased twofold                aminyltransferase, shown overlap preceding ftsW gene <br /> and level PBP3 reduced 25 30%. The main                4 bases (15, 23). Since gene cluster, murC, <br /> difference plasmids presence absence          separated murG 100 bases (14), was <br /> ftsA-encoding fragment main ftsZ promoters            tempting consider murG gene cotranscribed <br /> previously identified (4, 9, 29, 37).                                   exclusively Pmra, preceding genes. <br />                                                                         In case, assumed residual transcrip- <br />                           DISCUSSION                                    tion occurred IPTG deprivation Pmra::Plac <br />                                                                         strain sufficient provide MurG molecules to <br />    Previous data transcription genes mra             support normal cell growth. However, discussed previously <br /> cluster mainly concerned distal part, including cell division       (13), basal level expression genes depending the <br /> genes ftsQ, ftsA, ftsZ envA (2, 4, 9, 11, 29, 32, 37).   Pmra promoter unlikely sufficient absence of <br /> How mur genes proximal region expressed                IPTG, considering presence lacIq strain the <br /> was known recent demonstration pro-              addition glucose growth medium used, which <br /> moter Pmra, originally identified required ex-         lead maximal repression promoter. The fact that <br /> pression ftsI, directs expression downstream        JE7970(pHR479) cells contained three- sixfold-lower levels <br /> genes (13). Earlier suggested existence pro-          MurG MurC enzymes consistent another <br /> moters ftsI-murC region (22), based observation           promoter contributing expression. <br /> that multicopy plasmids (pUC18 derivatives) individual                Under conditions IPTG induction Pmra::Plac <br /> genes cloned orientation opposite vector             promoter strain JE7970, levels MraW PBP3 proteins <br /> promoter fully complemented specific defects thermo-             increased largely different factors, two- sevenfold, re- <br /> sensitive mutant strains allowed significant overproduction         spectively. This effect observed pro- <br /> of corresponding enzymes. This observed particular           teins. The simple explanation greater stability PBP3 <br /> with murE, murD, murG, murC genes. One exception                accounts differential expression unlikely the <br /> concerned murF, expression apparently required                turnover PBP3 shown high (unpublished <br /> whole sequence upstream gene murE, suggesting               observation). Then, data suggest specific regulation <br /> cotranscription occur case (22). In            expression genes level transcription <br /> other cases, conceivable cumulative effect        and/or translation. In sense, weak promoters identi- <br /> weak promoters high copy numbers transcriptional read-            fied genes mra cluster (12), but <br /> through promoter plasmid vector resulted            involvement differential expression two <br /> in sufficient expression functional complementation. In         proteins remains established. <br /> present work, expression of genes shown             We observed specific activity Ddl de- <br />  VOL. 180, 1998                                                      EXPRESSION OF GENES IN THE E. COLI mra CLUSTER                                     4411 <br />  <br />  <br /> creased 30% Pmra::Plac cells deprived IPTG and, in-                                              REFERENCES <br /> versely, increased 30% induced cells. These levels          1. Ayala, J. A., T. Garrido, M. A. Pedro, M. Vicente. 1994. Molecular <br /> variation low compared enzymes encoded                biology bacterial septation. New Compr. Biochem. 27:73&#8211;101. <br />                                                                          2. Beall, B., J. Lutkenhaus. 1987. Sequence analysis, transcriptional orga- <br /> by genes mra cluster. This showed expression              nization, insertional mutagenesis envA gene Escherichia coli. J. <br /> of ddlB was, part, dependent Pmra promoter,              Bacteriol. 169:5408&#8211;5415. <br /> as demonstrated genes murG murC. However,                     3. Boyle, D. V., M. M. Khattar, S. G. Addinall, J. Lutkenhaus, W. D. <br /> should noted genes, ddlA ddlB, ligases              Donachie. 1997. ftsW essential cell-division gene Escherichia coli. <br />                                                                             Mol. Microbiol. 24:1263&#8211;1273. <br /> with identical kinetic properties identified              4. Dai, K., J. Lutkenhaus. 1991. ftsZ essential cell division gene in <br /> E. coli (38), theoretically contributing Ddl activity           Escherichia coli. J. Bacteriol. 173:3500&#8211;3506. <br /> determined cell extracts. Since ddlA gene belongs            5. Dai, K., J. Lutkenhaus. 1992. The proper ratio FtsZ FtsA is <br /> completely separate chromosomal region, variations ob-                  required cell division occur Escherichia coli. J. Bacteriol. 174:6145&#8211; <br />                                                                             6151. <br /> served likely represented specific contribution            6. Dewar, S. J., K. J. Begg, W. D. Donachie. 1992. Inhibition cell division <br /> of DdlB enzyme total activity. Since null mutation             initiation imbalance ratio FtsA FtsZ. J. Bacteriol. 174: <br /> in ddlA ddlB apparent effect cell growth             6314&#8211;6316. <br /> morphology, laboratory growth conditions,             7. Donachie, W. D. 1993. The cell cycle Escherichia coli. Annu. Rev. Micro- <br />                                                                             biol. 47:199&#8211;230. <br /> reason duplication remains unclear.                             8. Fellay, R., J. Fray, H. Krisch. 1987. Interposon mutagenesis soil and <br />    The levels FtsZ, essential cell division protein en-              water bacteria: <span id='am-11' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-12' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> DNA fragments designed vitro insertional <br /> coded penultimate gene cluster, varied               mutagenesis Gram-negative bacteria. Gene 52:147&#8211;154. <br /> repression induction Pmra promoter. These variations           9. Flardh, K., T. Garrido, M. Vicente. 1997. Contribution individual <br />                                                                             promoters ddlB-ftsZ region transcription essential cell- <br /> paralleled mur gene products, levels            division gene ftsZ Escherichia coli. Mol. Microbiol. 24:927&#8211;936. <br /> comparatively lower (Table 3). This suggested portion         10. Fletcher, G., C. A. Irwin, J. M. Henson, C. Fillingim, M. M. Malone, and <br /> the transcription genes cluster derived             J. R. Walker. 1978. Identification Escherichia coli cell division gene sep <br /> from Pmra promoter. Dai Lutkenhaus (4) previously                   organization cell division-cell envelope genes sep-mur-ftsA- <br />                                                                             envA cluster determined specialized transducing lambda bacterio- <br /> reported strain null allele ftsZ chromo-            phages. J. Bacteriol. 133:91&#8211;100. <br /> some complemented lambda phage (l16-2)                11. Garrido, T., M. Sanchez, P. Palacios, M. Aldea, M. Vicente. 1993. <br /> carrying 6 kb DNA upstream ftsZ, including ftsA               Transcription ftsZ oscillates cell cycle Escherichia coli. <br /> ddlB genes, promoters ftsZ                EMBO J. 12:3957&#8211;3965. <br />                                                                         12. Go &#180;mez, M. J. 1991. Doctoral thesis. Universidad Auto      &#180;noma Madrid, <br /> identified (9, 29, 37). This suggested involvement                 Madrid, Spain. <br /> far-upstream promoter, contribution ftsZ expression            13. Hara, H., S. Yasuda, K. Horiuchi, J. T. Park. 1997. A promoter the <br /> was estimated 30 40% (4). Pmra participates                  genes Escherichia coli mra cluster cell division cell <br /> but apparently essential ftsZ expression,              envelope biosynthesis genes, including ftsI ftsW. J. Bacteriol. 179:5802&#8211; <br />                                                                             5811. <br /> Pmra::Plac strain carrying region ftsW plas-        14. <span id='am-42' about='obo:NCBITaxon_1269045' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_6427 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_41799 obo:NCBITaxon_41798 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_6341 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6340 obo:NCBITaxon_6072'><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Ikeda" datatype="xsd:string"></span><span id='am-46'  ></span>Ikeda</span>, M., M. Wachi, H. K. Jung, F. Ishino, M. Matsuhashi. 1990. <br /> mid grew absence IPTG. The 59 end insert              Nucleotide sequence involving murG murC mra gene cluster E. <br /> present l16-2 contained ftsZ failed complement               coli. Nucleic Acids Res. 18:4014. <br /> a null allele ftsZ middle ftsW gene (3, 15).       15. Ishino, F., H. K. Jung, M. Ikeda, M. Doi, M. Wachi, M. Matsuhashi. <br />                                                                             1989. New mutations fts-36, lts-33, ftsW clustered mra region the <br /> When considering data obtained lacZ fusions,              Escherichia coli chromosome induce thermosensitive cell growth divi- <br /> be assumed promoter, responsible                 sion. J. Bacteriol. 171:5523&#8211;5530. <br /> portion expression genes murG ftsZ,               16. Jung, H. K., F. Ishino, M. Matsuhashi. 1989. Inhibition growth of <br /> present upstream SnaBI site sequence ftsW.               ftsQ, ftsA, ftsZ mutant cells Escherichia coli amplification a <br />                                                                             chromosomal region encompassing closely aligned cell division cell <br /> As shown above, levels PBP3 FtsZ proteins showed                 growth genes. J. Bacteriol. 171:6379&#8211;6382. <br /> opposite variations Pmra::Plac strain carrying plasmids          17. Koop, A. H., M. E. Hartley, S. Bourgeois. 1987. A low-copy-number <br /> with different fragments mra cluster. The main differ-               vector utilizing b-galactosidase analysis gene control elements. <br /> ence plasmids presence absence                    Gene 52:245&#8211;256. <br />                                                                         18. Lowry, O. H., N. J. Rosebrough, A. L. Farr, R. J. Randall. 1951. Protein <br /> fragment carrying ftsA main promoters ftsZ. Then,                measurement Folin phenol reagent. J. Biol. Chem. 193:265&#8211;275. <br /> we propose expression FtsA absence Pmra           19. Matsuhashi, M., M. Wachi, F. Ishino. 1990. Machinery cell growth <br /> promoter elicits regulatory mechanism involves repres-               division: penicillin-binding proteins proteins. Res. Microbiol. <br /> sion specific promoter PBP3 FtsA. In regard,             141:89&#8211;103. <br />                                                                         20. Mengin-Lecreulx, D., B. Flouret, J. van Heijenoort. 1982. Cytoplasmic <br /> it previously shown amplification fragment cor-               steps peptidoglycan synthesis Escherichia coli. J. Bacteriol. 151:1109&#8211; <br /> responding genes cluster modified plating           1117. <br /> capacity thermosensitive ftsA mutant restrictive            21. Mengin-Lecreulx, D., B. Flouret, J. van Heijenoort. 1983. Pool levels of <br /> temperature (16) elevated levels FtsA protein              UDP-N-acetylglucosamine UDP-N-acetylglucosamine-enolpyruvate in <br />                                                                             Escherichia coli correlation peptidoglycan synthesis. J. Bacteriol. <br /> blocked cell division early stage (5, 6). An imbalance              154:1284&#8211;1290. <br /> between FtsA FtsZ similarly affect expression            22. Mengin-Lecreulx, D., C. Parquet, L. R. Desviat, J. Pla      &#180;, B. Flouret, J. A. <br /> the genes cluster, including ftsI.                             Ayala, J. van Heijenoort. 1989. Organization murE-murG region <br />                                                                             Escherichia coli: identification murD gene encoding D-glutamic- <br />                                                                             acid-adding enzyme. J. Bacteriol. 171:6126&#8211;6134. <br />                                                                         23. Mengin-Lecreulx, D., L. Texier, M. Rousseau, J. van Heijenoort. 1991. <br />                      ACKNOWLEDGMENTS                                        The murG gene Escherichia coli codes UDP-N-acetylglucosamine: <br />                                                                             N-acetylmuramyl-(pentapeptide) pyrophosphoryl-undecaprenol N-acetyl- <br />    This work supported grant Centre National la        glucosamine transferase involved membrane steps peptidoglycan <br /> Recherche Scientifique (URA 1131) grant &#8220;Biotechnologies&#65533;?             synthesis. J. Bacteriol. 173:4625&#8211;4636. <br /> from Ministe`re l&#8217;Education Nationale la Recherche et la   24. Mengin-Lecreulx, D., J. van Heijenoort. 1985. Effect growth condi- <br /> Technologie (no. 97.C.0177), France. Work J.A.&#8217;s laboratory         tions peptidoglycan content cytoplasmic steps biosynthesis in <br />                                                                             Escherichia coli. J. Bacteriol. 163:208&#8211;212. <br /> was supported grant BI094-0789 Comision Interministerial <br />                                                                         25. Mengin-Lecreulx, D., J. van Heijenoort. 1993. Identification glmU <br /> de Ciencia y Tecnologia, Spain. J.A. acknowledges institutional         gene encoding N-acetylglucosamine-1-phosphate uridyltransferase Esch- <br /> help Fundacio&#180;n Ramo &#180;n Areces Centro Biolog&#305;&#180;a Mo-        erichia coli. J. Bacteriol. 175:6150&#8211;6157. <br /> lecular Severo Ochoa.                                                   26. Miller, J. H. 1992. A short course bacterial genetics: laboratory manual <br />    We than <br /> </body></html>