<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright 8 1988 Genetics Society America <br />  <br />  <br />  <br />  The unc-ZZ(IV)Region <span id='am-1' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-7'  ></span>Caenorhabditis elegans</span>: Genetic Analysis Lethal <br />                                Mutations <br />  <br />                     Denise V. Clark, Teresa M. Rogalski,&apos; Leo M. Donati David L. Baillie <br />                  Department Biological Sciences, Simon Fraser University, Bumby, British Columbia, Canada V5A IS6 <br />                                                       Manuscript received November 10, 1987 <br />                                                           Accepted February 11, 1988 <br />  <br />                                                                ABSTRACT <br />                   The organization essential genes unc-22 region, definedby deficiency sDj2 linkage <br />                group IV, studied. Using balancer nT1 (ZV;V), suppresses recombination over <br />                49 map units, 294 lethal mutations LGIV(right)and LGV(1eft) wererecovered using EMS <br />                mutagenesis. Twenty-six mutations fell unc-22 region. Together previously <br />                isolated lethal mutations, total 63 lethal mutations fall 31 complementation <br />                groups. Mutations positioned map using overlapping deficiencies addition to <br />                sDj2. The lethal alleles deficienciesin unc-22 region characterized respectto their <br />                terminal phenotypes. Mappingof lethal mutations shows sDj2 deletes minimum 1.8 <br />                map units maximum 2.5 map units. A minimum estimate essential gene number for <br />                region using truncated Poisson calculation 48. The <span id='am-81' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-82' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-85'  ></span>data</span> indicate minimum estimate of <br />                approximately 3500 essential genes <span id='am-8' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-11'  ></span><span id='am-12' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span>Caenorhabditis</span> elegam genome. <br />  <br />                                                                                 BAILLIE1985), 20 essential genes identified in <br />  0    UR laboratory interested revealing how <br />        genes essential development organized <br /> at genetic molecular levels inthe genome <br />                                                                                 2 map unit interval defined deficiency <br />                                                                                 sDj2. From studies, estimated there <br /> of nematode Caenorhabditis elegans. Several re-                             32 essential genes unc-22 region, based on <br /> gions C . elegans genome genetically                           Poisson distribution lethal mutations. <br /> characterized respect essential genes                            Therefore, region saturated with <br /> past, particularly linkage group (LG) I (ROSEand                             mutations essential genes. The reasons a <br /> BAILLIE1980; HOWELL      et al. 1987), LGII (SIGURDSON,                         continued genetic characterization unc-22 re- <br /> SPANIER HERMAN      1984),LGV (ROSENBLUTH         et al.                    gion threefold. First, approaching saturation of <br /> 1988), LGX (MENEELY      HERMAN     1979, 1981)                         essential genes lethal mutations allows one <br /> LGIV m - 2 (ROGALSKI       RIDDLE1988)                          make accurate estimate gene number <br /> well unc-22 gene cluster (ROGALSKI,       MOER-                       unc-22 region. Second, identification ofall <br /> MAN BAILLIE     1982; ROGALSKI    BAILLIE    1985)                      essential genes, positioning defi- <br /> which subject study.                                             ciency breakpoints genetic map, valuable <br />    The unc-22 gene codes muscle component.                                molecular characterization region. Third, <br /> It characterized genetically (MOERMAN                              collection alleles ofeach essential gene provides <br /> BAILLIE1979; ROGALSKIand BAILLIE1985)                                   information pattern expression devel- <br /> been cloned (MOERMAN,         BENIANand WATERSTON                               opment perhapsabout relationships between <br /> 1986). In addition cloning unc-22, restriction                        adjacent genes region. <br /> fragment length differences (RFLDs) ge-                                  In paper isolation set of <br /> netically mapped cloned 2.5-map-unit                                 EMS induced lethal mutations using balancer <br /> interval surrounding unc-22, unc-43 unc-                            nTZ(ZV;V)(FERGUSON HORVITZ       1985).Along with <br /> 32 (BAILLIE,  BECKENBACH      ROSE 1985). With                          updated map essential genes sDj2 region <br /> continued addition overlapping cosmids                                andmapping       data newly identified essential <br /> original clones (COULSON     et al. 1986),                       genes, present compiled list lethal muta- <br /> total 1,100 kilobases cloned DNA                             tions sDj2 region isolated date, with <br /> unc-43 unc-32 interval. However, cosmids                             terminal phenotypes. By using distribution <br /> not contiguous region.                                       allele number essential genes sDj2 <br />    In previous work unc-22 region (Ro-                               region, arrive Poisson estimate total essen- <br /> GALSKI, MOERMAN      BAILLIE1982; ROCALSKIand                               tial gene number region. The extent the <br />                                                                                 nT2 balanced region refined provide estimate <br />    &apos;Present address: Division Biological Sciences, University Missouri,   ofessential gene number balanced region. <br /> Columbia, Missouri 652 1 1.                                                     <span id='am-18' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-21'  ></span><span id='am-22' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> estimate derived determining pro- <br /> Genetics 119 345-353 (June, 1988). <br />  346                                                   D. V. Clark et al. <br /> portion lethal mutations (isolated present               (1974). Double mutants, heterozygous nTI, con- <br /> screen) fall sDj2 region                structed following manner. Homozygous double <br />                                                                    mutant ( b ) hermaphrodites mated unc-22 unc-3ll <br /> Poisson estimateof essential gene number sDj2               nTI (ZV);+ /nTI ( V )males. Double mutant heterozygotes [a <br /> region.                                                            bhTI (ZV);+ lnTI ( V ) + /nTI (ZV);   blnT1 ( V ) ] se- <br />                                                                    lected 1% nicotine solution aswild type individuals <br />                                                                    lacking twitching phenotype. The pseudolinkage be- <br />              MATERIALS AND METHODS                                 tween LGIV LGV caused nTI allowed construction <br />                                                                    strains. <br />     The genetic nomenclature inthis paper follows                  Mappingandcomplementationtests:              Recombination <br /> recommendations HORVITZ          et al. (1979).                 mapping carried constant 20 degrees the <br />     Stock maintenance strains: <span id='am-59' about='obo:NCBITaxon_333870' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_195229 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41087 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_71193 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-60' property="oboInOwl:hasRelatedSynonym" content="Nematodes Berthold, 1827" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="Nematodes" datatype="xsd:string"></span><span id='am-62'  ></span><span id='am-63' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Nematodes</span> main-            recommendations ROSEand BAILLIE          (1979). Initial map- <br /> tained petri plates containing nematode growth medium           ping lethal mutations isolated screen either <br /> (NGM) streaked <span id='am-36' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-51'  ></span>Escherichia coli</span> OP50 (see BRENNER             LGIV LGV involved crossing strain N2 males <br />  1974).                                                            scoring progeny hermaphrodites of <br />     The wild-type strain N2 (var. Bristol) strains carrying    genotype (let-x) unc-22 unc-31/      +++   ( I V ) ;+/+ ( V ) or <br /> the following mutations used study obtained                              + <br />                                                                    unc-22 unc-31/+ (ZV);let-x/+( V ) .A greater 3 : 1 ratio <br /> from stock collection Medical Research Council,         wild type Unc-22 Unc-31 progeny indicated linkage <br /> Cambridge, England, Caenorhabditis Genetics            LGIV. AllLGIV lethal mutations retained and <br /> Center University Missouri,Columbia. LGIV: dpy-          4map units unc-22 positioned <br /> 4(e1166), dpy-9(e424),dpy-I3(e184),dpy-20(eI282), mDj7             complementation testswith dj7, sDj2 <br /> (ROGALSKI     RIDDLE    1988),mec3(e1338), unc-5(e152), unc-   sDj21. Positioning lethal mutations sDj2 was <br /> 17(e933), unc-3I(e169),unc-33(e204);LGV: dpy-II(e224),             accomplished complementation tests set of <br /> dpy-2I(e428),  unc-76(e911);  nTI(ZV;V)          (FERGUSON     deficiencies breakpoints divide sDj2 region into <br /> HORVITZ      1985): fromthe strain CB3608whichhas              zones (see Figure 2). Once mutation positioned <br /> genotype dpy-ZO(e1282) dpy-26(nl99)/nTI(ZV); +/nTI (V).            zone, assigned complementation group <br /> The strains carrying nDj27(ZV)and lzn-I ( e l 777)ZV       complementation testswithallelesofall                genes <br /> strain lzn-3(n378) unc-22(e66)were obtained R.            zone. For complementation tests alleles of <br /> Horvitz laboratory, M.I.T. The remaining LGIV mutations            previously identified genes multiple alleles (see <br /> were isolated Simon Fraser University: unc-22(s7)(MOER-         Table 2), representative allele used (see <br /> MAN BAILLIE       1979),sDj2, sDj7, sDf8, sDj9, sDfl0 (MOER-   RESULTS discussion). The positions recently <br /> MAN BAILLIE        1981), sDj21 sDj22 (G. WILDand D.       identified essential genes confirmed refined by <br /> BAILLIE,    unpublished results). The sDj2 region lethal mu-       factor mapping. The progeny (let-x) unc- <br /> tations LGIV isolated previously Simon Fraser Uni-                         + <br />                                                                    22 unc-31/+ + individuals scored (see Table 3). <br /> versity isolated screen described           Effective lethal stages: Effective lethal stages <span id='am-23' about='obo:NCBITaxon_9605' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-24' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-25'  ></span><span id='am-26' property="rdfs:label" content="Homo" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>homo</span>- <br /> listed Table 2 corresponding references.                   zygotes hemizygotes EMS induced lethal mutations <br />     It noted work unc-          ofhomozygotes deficiencies sDj2 region <br /> 22(s7) mutation usedas marker (MOERMANand                    determined. For homozygous lethal phenotypes ar- <br /> BAILLIE     1979). It conditionally semidominant mutation     resting later thanthe      earlylarval     stage, heterozygous <br /> where unc-22(~7)/+individual &#8220;twitcher&#65533;? phe-             parents stock strain selfed 20&#65533;?, Unc- <br /> notype 1% nicotine solution (Sigma)          22 Unc-31 larvae sterile adults selected their <br /> phenotypically wild type. Individuals homozygous          average length measured 3 days later (toensure <br /> wild type contracted paralyzed nicotine,        completion growth) dissecting microscope (ac- <br /> unc22(s7) homozygotes unconditional twitchers.                 curacy 2 0.05 mm). For lethal mutations arresting the <br />    Screen recessive lethal mutations:      Young gravid her-   early larval stage, putative Unc-22 Unc-31 homozygotes <br /> maphrodites genotype unc-22 unc-3llnTI (ZV);+ /             easily identified using nTI balanced strain <br /> n T I ( V ) exposed 0.012 M EMS 4 hr. Po her-          occurrence egg early larval stage arrested <br /> maphrodites allowed lay eggs 24-hr            worms probably nTI segregation aneuploids. <br /> broods. This procedure prevented overcrowding             Therefore,for      mutations, (let-x) unc-22 unc-3Il <br /> ensured Fls recovered parent          nT1 (ZV);+/nTl(V) DflnTI hermaphrodites mated <br /> the 48-hr period. All wild type Fls individually         unc-22 unc-3llnTI (ZV); +/nTI    ( V ) males, gravid Unc-22 <br /> allowed self-fertilize F2s screened        Unc-31 F, progeny selected allowed lay eggs <br /> absence gravid Unc-22 Unc-31 individuals. At        5-12-hr broods. As viable progeny matured, were <br /> wild-type individuals strain carrying putative         removed theplate developmentally arrested <br /> recessive lethal mutation selected. The progeny            individuals left. These measured had <br /> these screened forthe absenceofUnc-22Unc-31                   hatched fromthe        egg. To determinethe           hemizygous <br /> recombinants confirm mutation lies          phenotype EMS induced lethal mutations sDj2, (let- <br /> nTI balanced region.                                               X ) unc-22 unc-3llsDj2 individuals selected <span id='am-28' about='obo:NCBITaxon_1178312' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_33154 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_572719 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41011 obo:NCBITaxon_41197 obo:NCBITaxon_41012 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-29'  ></span><span id='am-30' property="rdfs:label" content="Mea" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>mea</span>- <br />    To determine induction frequency spontaneous             sured. If stage arrest egg early larva, <br /> recessive lethal mutations nT1, protocol         determined couldbesome <br /> used described above, omitting 0.012 M EMS.                 developmentally arrested aneuploids parental <br />     Extent nT2 balanced region LGIV LGV:             hermaphrodite [sDj2/nTI(IV);+ /nTI ( V ) ]self progeny. <br /> In orderto determine extentof recombination suppres-               The larval stage arrest determined comparing <br /> sion nT1 LGIV LGV precisely, various                length mutant individuals unc-22(s7) growth <br /> pairs morphological mutations placed trans           curve (ROGALSKI,    MOERMANand BAILLIE1982). It should <br /> nTI wild-type N2 chromosome comparison.               noted developmental stage arrest larval <br /> Table1 shows various double mutant chromosomes                 lethal mutations classified L1, L2, L3 or <br /> used. These constructed described BRENNER               L4 attempt observe record <br />                                                            Lethal Mutations C. elegans                                              347 <br />                                                                     TABLE I <br />                                                           nT1 recombination suppression <br />                         ~~~ <br />  <br />  <br />  <br />  <br />                                      Interval                        bi+   fa                              blnTlb <br />               Linkage <br />                group                          b   A       B      Total          Distance     A     B    Total           Distance <br />                 IV              dpy-9-unc-17        247     197    1715    31 (29-33)          71 <br />                                                                                                 462 43                   35 (30-40) <br />                                unc-17-dpy-4         152     159    2244    15 (14-16)           2    0    3568          -0 <br />                                unc-22-dpy-4                                 5&#8216;                  1    0    1172          -0 <br />                                unc-33-dpy-13        20       27    1892(1.8-3.2) <br />                                                                             2.5                 0    0    1073            0 <br />                               -dpy-I3 <br />                                  lin-1              47       67             7.8 <br />                                                                    1766 (6.0-9.6)d              1    0     724          -0 <br />                  V            dpy-11-unc-76 <br />                                    75                       48     1784(6.8-10.4)d <br />                                                                              8.6                0    0     1243           0 <br />                               unc-76-dpy-21 64      60             1069     12 25 <br />                                                                                (10-14)              44      701          1 1 (9-13) <br />      Distance map units calculated loop p = 1 - ( 1 - 2R)&#65533;?&#8217; R = (A + B)/total progeny scored (BRENNER      1974). The <br /> 95% confidence intervals shown parentheses. <br />    * Distance calculated assuming nTI behaves like reciprocal translocation eTl(II1; V ) (see ROSENBLUTH <br />                                                                                                           BAILLIE <br />                                                                                                                    1981) segregates <br /> inviable aneuploids p = 0.6-0.6[1 - (10/3)R]&#8217;/* R = (A + B)/total progeny scored. <br />      From EDGLEY   RIDDLE1987. <br />      Dpy recombinants used calculate p = 1 - ( 1 - 3R&#8217;)&#8217;/*and R&#8217; = A/(A + number wild types). <br />  <br />  <br /> the larval moults. Instead, classified length                    screenwas conducted theabsence EMS. Using <br /> individual early, mid late larval based position                 strain used EMS screen, unc- <br /> on thegrowthcurve.        The growth unc-22(s7) urn-                       22 unc-31 lnT1   (ZV);+ InTl ( V ) ,two spontaneous lethal <br /> 31(e169) individuals follows curve unc- <br /> 22(s7) individuals 20&#65533;? (data shown).                                   mutations isolated 1736chromosomes <br />                                                                               screened. One mutation mapped 17 map units from <br />                                                                               unc-22 LGIV mappedin region <br />                                RESULTS                                        deleted sDj21 sDj2 right unc- <br />                                                                               22. The spontaneouslethalmutationfrequency             in <br />     Screen lethal mutations: Using screening                          nT1 balanced region 0.1%. Therefore, <br /> protocol described     MATERIALS AND METHODS,                           294 mutations isolated EMS screen, <br /> established recessive lethal mutation frequency                         lethalmutations expectedto of <br /> the nT1 balanced region. Out 3398 FI &#8220;chromo-                              spontaneous origin. <br /> somes&#65533;?[LGIV(right)and LGV(left)] screened, 294                                   Extent nT1 balanced region:Originally, FER- <br /> lethal mutations recovered nT1 balanced                           GUSON HORVITZ         (1985) determinedthat nT1 <br /> region (see Figure l), giving recessive lethal mu-                          causedpseudolinkage         LGIV(right)    and <br /> tation frequency 8.7% 0.012 M EMS.                                    LGV(1eft) suppressed recombination unc- <br />     The types ofmutationsrecoveredincludeegg,                                 17 dpy-4 unc-60 dpy-11 intervals (see Figure <br /> larval andadult sterile recessive lethalmutations.                             1). T h e pseudolinkage LGIV(right) and <br /> Owing transparency Unc-22 Unc-31 indi-                              LGV(1eft) wasobserved nT1 heterozygote self prog- <br /> viduals visibility fertilized eggs                         eny. All aneuploid zygotes resultingfrom various <br /> the body wall, adult sterile mutations recovered                         combinations half translocations normal chro- <br /> since fertilized eggs visible greatly                        mosomes appeared arrest development egg. <br /> reducedinnumber          individualscarryingthese                          The euploidprogeny consist homozygotes for <br /> mutations. Any maternal effect mutations leading                           normal LGIV LGV, parental heterozygotes and <br /> fertilized eggsthat subsequently fail developwould                         nT1 homozygotes (nT1 associated vulvaless <br /> thus recovered screen.                                         phenotype) 1 :4: 1 ratio, respectively. In orderto <br />     T h e relative lethal <br />                         mutation <br />                                frequencies                                 furtherrefinetheextent            nT1 recombination <br /> LGIV(right)and LGV(1eft) 4.9% (169muta-                                  suppression, intervals LGIV LGV were <br /> tions) 3.8% (125 mutations) respectively. There                           examined results listed Table 1. On <br /> is probability strains carrying lethal                     LGIV, appears nT1 effectively balances <br /> mutations LGIV carry mutation LGV,                               2 1map unitinterval lin-1 dpy-4 (mapping <br /> since chromosome marked mor-                              data includes EDGLEY         RIDDLE1987). <br /> phological mutation. The probability occurring                        It noted        occasional Unc <br /> is low (approximately 6 169 LGIV muta-                             recombinantfromthe             unc-22dpy-4lnTl (ZV);+ I <br /> tions carry additional lethal mutation                           nT1 ( V ) unc-17 dpy-41nTl(ZV);+ InTl ( V ) individ- <br /> LGV).                                                                         uals. In cases, recombinant unc chromosome <br />     In orderto establish numberof lethal mutations                        didnotcarry       nT1. This demonstrated the <br /> that EMS induced vs. spontaneous,                          inability recombinant chromosomes sup- <br />  348                                                     D. V. Clark et al. <br />   LGIV                                                                                                 unc-22 <br />                                                                                                      dpy - 20 <br />                                                                                                     1 in-3                     dPY-4 <br />                                                                        unc-17                      mec-3                    t ra-3 <br />  dPY-9                                             1 in-1             unc-33    dpy-13            unc-43 unc-31           lev-1 <br />                                                                                                daf-/4(  11 <br />                                                                                          nT1 <br />                                                        t&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot; <br />                                                                                                         6 <br />                                                                                                                   sDf21 <br />                                                                                                              l--&quot;i <br />  <br />  LGV <br />  <br />  <br />  unc-60                                                     dpy- 11             unc-76         dpy-21                          unc-til <br />  <br />            nll <br />       ~&quot;&quot;&quot;&quot;&quot;&quot;&quot;-&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;&quot;i <br />       5 map units <br />  <br />    FIGURE1 .-Genetic map LGIV LGV showing region balanced nTl(1V;V). Nonessential genes importance study <br />                      -. <br /> are shown map positions derived EDCLEYand RIDDLE(1987) study. The overlapping deficiencies dj7,sDfz and <br /> sDj21 shown. <br />  <br /> press recombination (data shown) lack                  that, thesample 169mutations, mutations <br /> ofvulvaless progeny Unc recombinants.                        sDj2 region identified. <br /> Therefore, itseems recombination occur                          A comparison density lethal mutations <br /> between nT1 normal LGIV homologue                        recovered &quot;nT1 screen&quot;between h f - 1 4 and <br /> a small interval left 4 9 - 4 . This probably               lev-1 shows lethal mutations clustered the <br /> means nT1 complex rearrangement                        left unc-22 relative region right (data <br /> than simple reciprocal translocation. It                shown). The clustered region likely due <br /> be noted nT1 completely stablein balancing                relatively low frequency recombination (see <br /> tru-3, lies approximately map unit                   DISCUSSION). Until analysis lethals the <br /> left dpy-4   u. <br />                  HODGKIN,   personal communication).                  rest LGIV(right) complete, say <br /> On LGV, nT1 suppresses recombination atleast                     clustered regions. Alleles of <br /> unc-60 unc-76, interval 28 map units                   essential genes near ama-l(ZV) char- <br /> (R. ROSENBLUTH     R. JOHNSEN, personal commu-                    acterized form dense cluster those <br /> nication). Therefore, extent nT1 balanced                  near unc-22 (ROGALSKI     RIDDLE   1988). <br /> region 49 map units.                                            In <span id='am-33' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-34' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> establish complementation groups, the <br />    Mapping complementationtests: The 169                          26 lethal mutations sDj2 mapped to <br /> strains carrying mutations LGIV subjected                     smaller intervals zones using deficiencies sDf7, <br /> to analysis. An approximate two-factor map                    sDf8, sDj9, sDfl0, sDj21,sDj22, dj7 nDj27. The <br /> distance unc-22was determinedfor                         unc-22 region subdivided 10 zones <br /> these lethal mutations (data shown). Complemen-                   defined deficiency breakpoints (Figure 2). To <br /> tation tests carried mu-                    define complementation groups zone, rep- <br /> tations map units unc-22 (125 mu-                     resentative alleles previously characterized essen- <br /> tations) overlapping deficiencies mDj7,                 tial genes used. The representative allele, inthe <br /> sDj2 sDj21. All26 lethal mutations failingto                      cases multiple alleles withdifferent <br /> complement sDj2 2 map units unc-                    phenotypes, exhibited earliest <br /> 22. None mutations 2-4-map-unit                     developmental arrest ormost severelethal phenotype <br /> intervals unc-22 lie                  likely null allele. The inter se <br /> within sDj2. This indicates initial approxi-                 complementation tests (data shown) 11 <br /> mate mapping mutations served decrease                      essential genes (in addition Zin-3; below) <br /> number necessary complementation tests                     identified sDj2 region ROGAL- <br /> lethal mutations sDf2. We couldsafelyassume                       SKI BAILLIE 1985. Table 2 lists essential <br />                                                                  Lethal Mutations C. elegans                                                     349 <br />      lpec-3        let-308 let-100 lin-3 <br />    let-71 let-72 let-307 let-98 let-74let-65                        let-92 let42 <br />                                                                                                  let-66      let-68        let-97 <br />                                                                                                                                                  unc-31 <br />  <br />  <br />  <br />    I                                                                    sDfZ                                                                 I <br />                                                                             sDf7                          I&#65533;?------------                 sDf21 <br />                                           -1       -1            sDf81 -                                                                 sDf22 <br />                                                                             sDf9    1 <br />                                                    -1               sDflO 1 - <br />                                                -            1                                             nDf27 <br />                            1 <br />                          mDf7    - <br /> Zone: <br />                                      1                  2       3      4    5   6       7          8                9               10 <br />  <br /> Number o f essential genes           19                 1       1      1    1   1       1          2                2               2 <br /> i n zone: <br />                0 . 5 map units <br />    FIGURE2.-Genetic map essential genes urn-22 region defined deficiency sDj2. The 3 1 essential genes shown along <br /> with positions deficiency breakpoints divide region 10 zones. The number essential genes zone is <br /> indicated map. Brackets map indicate genes positioned relative other. The <br /> order genes brackets based two-factor mapping distances unc-22. <br />  <br />  <br /> gene, lethal allele, zone, effective lethal stage                                   sDj2 deletes region minimum <br /> homozygotes hemizygotes(let-xlsDj2).Some                                           1.8 map units maximum 2.5 map units <br /> changes thedatafrom         previous reportsare                                       size (Figure 2). The left breakpoint sDj2 well <br /> follows. In ROGALSKI      BAILLIE(1985), s175                                       defined. unc-43 positioned just outside of <br /> assigned allele let-62, possibly                                       sDf2, 1.2 map units unc-22, furthest <br /> formed complex locus let-59. Reex-                                           essential gene left unc-22 stillwithin <br /> amination results showed s175                                         sDf2 let-69, 1.2 map units unc-22 (ROGALSKI <br /> other let-59 allelesfail complement other.                                          BAILLIE1985). The right breakpoint sDj2 is <br /> Thus,the gene designation let-62 hassincebeen                                               clearlydefined. This relative <br /> discontinued. Also, s172 shown allele                                       sparsity genes right unc-22. let-97, 0.6 <br /> let-69 let-59, indicated ROGALSKI,   MOER-                                    map unit right unc-22, thefurthest <br /> MAN BAILLIE     (1982).                                                                 essential gene unc-22 lies withinsDj2, while <br />    lin-3 essential gene viable alleles                                  unc-31 lies 1.3 map units unc-22 most <br /> with vulvaless phenotype (FERGUSON       HORVITZ                                      proximal gene outside right sDj2 breakpoint. <br />  1985). In course work, s751 s1263                                          Therefore, right sDj2 breakpoint 0.6 <br /> were originally assigned tolet-94. Subsequently, com-                                       1.3 map units unc-22. <br /> plementation testswith &amp;-3(n378) (FERGUSON                                                 There locus shows complex comple- <br /> HORVITZ1985) showed let-94 lin-3.                                             mentation pattern. There basic explanations <br /> Therefore, discontinued let-94 designa-                                         phenomenon: alleles are <br /> tion adopted thename lin-3 complemen-                                          small deletions spanning complementation <br /> tation group.                                                                               groups somepairs alleles exhibit intragenic <br />    The mappingdataforthe           elevennewessential                                       complementation. This locus, let-91) consid- <br /> genes lin-3 listed Table 3. Mapping data                                         ereda single,complex gene forthepurpose               of <br /> for essential genes canbe                                            Poisson analysis data. There alleles <br /> corresponding references listed Table 2. The                                             let-91, twoofwhich      arrest inmid-larvalstages <br /> mutations let-91(s753), let-93(~734)and lin-3(~751)                                         homozygousand complement other <br /> were isolated preliminary study using screen                                         (sf578and ~ 1 1 8 5 )The <br />                                                                                                                   . allele s753, hand, <br /> identical described paper (DONATI                                        arrests late larval development fails com- <br /> 1985). The remaining let genes identified                                          plement s678 ~1185.Because s753 arrests <br /> in study.                                                                              later development twoalleles,it <br />    The two-factor mapping data lethal mutations                                          seemsunlikely deletion oftwo distinct, <br />  350                                                                          D. V. Clark et al. <br />                                                                                      TABLE 2 <br />                                                                    Lethal alleles unc-22 region <br />  <br />                                                      Phenotype lethal arrest&#8216;                                                               Phenotype lethal arrestc <br />                                                                    ~ <br />  <br />  <br />  <br />  <br /> Zone&#8220;     Gene Reference&#8217; <br />                       Allele                 ICt-xlsDf2 <br />                                                      let-xllet-x                           Zone&#8216;      Gene          Allele   Referenceb        let-xllet-x        Lt-xlsDf2 <br />  <br />   6     let-52    s42                   1       Early larva            Early larva             1   let-70    s689               2         Mid larva          ND <br />   5     let-56    s46                   1       Late larva             Mid larva                             dl32                         Early larva        Mid larva <br />                   s50, s168             1       Late larva             ND                      1   let-71    s692               2         Sterile            Sterile <br />                   sl7?                  1       Mid larva              Early larva                                                          (leaky)            (leaky) <br />                   ~1192, ~1210,                 Mid larva              Mid larva               1   let-72    s52                1         Late larva         ND <br />                   ~1223,s1262                                                                                s695               2         Mid larva          Mid larva <br />   1     let-59    s4 9                  I       Early larva            Early larva             1   let-73    s685               2         Sterile            Sterile <br />                   sl75                  1       Multiple               Egg early            1   let-74    s697               2         Late larva         Late larva <br />                                                   stages                 larvad                                                             (leaky) <br />                   s681                  2       Early larva            ND                      1   let-91    s678               2         Mid larva          Mid larva <br />                   slI74, dl97                   Early larva            Egg early                          s753               3         Late larva         Late larva <br />                                                                          larva                               dl65                         ND                 Mid larva <br />   3     let-60    s5 9                  1       Mid larva              Mid larva                             dl85                         Early larva        Egg early <br />                                                   (leaky)                                                                                                      larva <br />                   s1124                         Early larva            Egg early            4   let-92    s504               2         Early larva        Early larva <br />                                                                          larva                               s677               2         Early larva        ND <br />                   sl155                         Late larva             Mid larva               7   let-93    s734               3         Mid larva          Mid larva <br />   1     Let-61    s65                   1       Late larva             Late larva              1   let-96    51112                        Mid larva          Mid larva <br />   1     let-63    sl70                  1       Mid larva              Mid larva           IO      let-97    s1121                        Early larva        Early larva <br />                   s679                 2        Late larva             ND                   1      let-98    ~1117                        Late larva         Late larva <br />   1     let-64    sl71                  1       Sterile                Sterile             10      let-99    S12OI                        Maternal           Maternal <br />                                                   (leaky)                                                                                   effect let         effect let <br />                   s216                  1       Sterile                Sterile                 1   let-100   dl60                         Early larva        Egg early <br />   2     kt65      s174, s254            1       Mid larva              Mid larva                                                                               larva <br />                   s694                 2        Sterile                ND                      1   let-307   dl71                         Mid larva          Mid larva <br />                   sl154, s1222                  Mid larva              Mid larva               1   let-308   sl705                        Mid larva&#8220;         Mid larva <br />   8     let-66    sl76                 1        Early larva            Mid larva               9   let-309   dl15                         Late larva         Late larva <br />   8     let-67    s214                 1        Sterile                Mid larva               1   let-311   s1195                        Late larva         Late larva <br />   9     let-68    s680                 2        Sterile                Mid larva               1   let-312   ~1234                        Late larva         Late larva <br />                   s693                 2        Late larva             ND                      1   lin-3     s751               3         Late larva         Early larva <br />                   s696                 2        Sterile                ND                                    s1263                        Late larva         ND <br />                   698                  2        Mid larva              ND                                    mDp,sDf2,                    Egg <br />                   s1258                         Early larva            Early larva                           nDf27, sDf21 <br />   1     let-69    sl72                  1       Early larva            ND                                    sDP, sofa,                   Early larva <br />                   s684                 2        Late larva             Mid larva                             SOP,sDfl0 <br />                   sllll                         Early larva            Egg early <br />                                                                         larva <br />  <br />       Refer Figure 2 location zone map. <br />    * For information isolation, mapping characterization lethal mutations reference  1, ROGALSKI, MOERMAN    BAILLIE(1982) and <br /> MOERMAN(1980). For isolation mapping lethal mutations reference 2, ROCALSKIand BAILLIE        (1985). The reference 3 (DONATI    1985) <br /> lethal mutations isolated identical manner screen described paper. The mutations listed reference 26 isolated in <br /> the screen described paper. The sources deficiencies listed indicated MATERIALS AND METHODS. <br />     Lethal mutations described references 1 2 <span id='am-65' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-66'  ></span><span id='am-67' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>cis</span> unc-22(~7).   Those described reference 3 paper cis unc-22(s7) <br /> and unc-jl(e169).N D = determined. <br />      The egg early phenotype let-xlsDf2 individuals indicates precise developmental arrest stage determined (see MATERIALS <br /> AND METHODS). <br />    &#8216;The phenotype let-308(~1705)   homozygotes hemizygotes background let-99(~1201)     LL( (see RESULTS). <br />  <br />  <br />  <br /> noninteracting loci. In addition, allele sl165                                   ODS.  Details previously isolated mutations <br /> fails complement sl185                                        corresponding references listed <br /> two alleles.                                                                              Table 2. Note areno egg lethal mutations <br />     Lethal phenotypes essential gene alleles     de-                               inthis region exception someof the <br /> ficiencies: Table 2 list EMS-induced lethal                                   deficiencies. The phenotypes described &#8220;leaky&#65533;?are <br /> mutations isolated date laboratory                                       occasional fertile progeny. It <br /> sDj2 region, deficiencies. The stages                                     noted lethal phenotypes of <br /> at individuals homozygous recessive lethal                                      mutations maybe synthetic effect, <br /> alleles sDf2 region block development range                                        mutations incis unc-22(~7) <br /> from early larva sterile adult. The homo-                                      unc-22(s7)and unc-31(e169). <br /> zygous (let-dlet-x) and, most, hemizygous (let-                                      let-308(~1705) let-99(~1201)  shown be <br /> x l s D j 2 ) phenotype lethal allele indicated.                              carried chromosome deficiency map- <br /> The phenotypes mutations isolated study                                       ping. Homozygous individuals chromosome <br /> were established described MATERIALS <br />                                               AND METH-                                midlarval lethal phenotype. Individuals of <br />                                                      Lethal Mutations C. elegans                                                       35 1 <br />                                                                   TABLE 3 <br />                                                        Two-factor mapping data <br />  <br />                                           Recombinants                                        Position <br />                                                                                              relative to <br />              Gene       Allele       Unc-22Unc-31        Unc-22      Unc-31      Total        UnC-2.Y      Distance ~ n c - 2 2 ~ <br />  <br />               lin-3      s751            11                            10        2769         left         0.60 (0.30-1.07) <br />              let-93      s734                             15           17        366 1        right        0.62 (0.31-0.93) <br />              let-96     SI 112           19                             8        3289         left         0.87 (0.48-1.26) <br />              let-97     SI 121                            15            3        3925         right        0.57 (0.29-0.87) <br />              let-98     SI 11 7          17                             8        3858         left         0.66 (0.35-0.97) <br /> SI201        let-99                                         8          10        4714         right        0.25 (0.10-0.49) <br />             let-1 00    SI I60           17                            21        4143         left         0.62 (0.33-0.91) <br />             let-307     ~lI71            24                            13        4615         left         0.78 (0.47-1.09) <br />             let-308     SI 705           18                            12        3677         left         0.74 (0.40- 1.08) <br />             let-309     sl I15                             12           8        3693         right        0.49 (0.25-0.85) <br />             let-311     ~1195             11                           12        3353         left         0.49 (0.25-0.88) <br />             let-312     s1234             16               2442        16                     left         0.99 (0.51-1.47) <br />      The hermaphrodites used two-factor mapping lethal mutations genotype [let-a(sx)]unc-22(s7)unc-?l(el69)/+    + +. <br /> The classes recombinant progeny enable placement mutations right left unc-22. The presence Unc-22 recombinants <br /> indicates mutation lies unc-22 unc-31 Unc-22Unc-31 recombinants indicate mutation lies left of <br /> unc-22. <br />      Distance map units 1 0 9 p = 1 - [ l - (3U/t0tal)]&#8217;/~and U = number Unc-22 Unc-22Unc-31 recombinants and <br /> total = number individuals scored. The 95% confidence intervals shownin parentheses calculated using formula <br /> 1.96(Npq)&#65533;?*where N = wild-type individuals (W), p = U/W q = 1 - p using table STEVENS        (1942). <br />  <br />  <br />  <br /> the genotype let-308(sZ705) unc-22 let-99(~1201)  unc-                  interval right unc-22 gene sparse <br /> 3Z/sDj2Z adult Unc-22 Unc-31sthat lay dead eggs                     relative left interval. <br /> whereas let-308(sZ705) unc-22 let-99(~1201)   unc-3ZI                      The apparentclustering genes left unc- <br /> sDj2 mDj7 individuals arrest development                      22, noted previously (ROGALSKI,      MOERMANand <br /> mid-larval stage. let-99(~1201),when separated re-                      BAILLIE 1982; ROGALSKIand BAILLIE1985), could <br /> combinationally let-308(sZ705),also ma-                      result following: (1) lower frequency of <br /> ternal egg lethal phenotype homozygote. The                        recombination left unc-22 relative the <br /> lethal phenotype let-308(sZ705)                   right,(2)a    higher number ofmutationallysilent <br /> determined.                                                             genes right unc-22, and(3)a          physical <br />                                                                         clustering genes, genes length of <br />                                                                         DNA left interval right. The <br />                          DISCUSSION <br />                                                                         clustering genes recombination map C. <br />    We recovered set 0.012 M EMS-induced                       elegans noted BRENNER      (1974) forthe <br /> recessivelethal      over <br />              mutations                  the <br />                                           balancer                      nonessential genes linkage group. Essential <br /> nT1 (ZV;V).The lethal mutation frequency 8.7%                     gene clustering subsequently coincide <br /> this 49-map-unit region. This frequency compa-                       clustering nonessential genes LGI <br /> rable 44-map-unit eTZ(ZZZ;V)region:                     (ROSEand BAILLIE1980; HOWELL         et al. 1987), LGII <br /> 6.3% EMS dose (R. ROSENBLUTH                          (SIGURDSON,   SPANIER HERMAN       1984) LGV <br /> R. JOHNSEN, personal communication). The unc-22                         (ROSENBLUTH    et al. 1988). Recent molecular work on <br /> region lethal mutations isolated screen                    LGIII gene cluster containing lin-12 shows that <br /> further characterized.                                                  clustering region appears tobe to <br />    Forthis study, definethe unc-22 region                         recombinational suppression uneven <br /> linkage group IV interval spanned                         distribution genes physical map (GREEN- <br /> deficiency sDj2. This region minimum 1.8                    WALD et al. 1987). <br /> a maximum 2.5 map units. To date, 63 lethal                         The unc-22 region divided ten <br /> sterile mutations isolated fall                    zones defined breakpoints nine <br /> 31 complementation groups. There 23 essential                       overlapping deficiencies including sDj2. Deficiency <br /> genes nonessential genes (dpy-20,                    breakpoints serve unambiguously order comple- <br /> him-8, mec-3) left unc-22 8 essential                 mentation groups genetic map. It apparent <br /> genes onenon-essential gene (dpy-26)to right                    Figure 2 deficiency breakpoints are <br /> (see EDGLEY    RIDDLE1987 information                        required, particularly zone 1, order com- <br /> nonessential genes). The right breakpoint sDj2                    plementation groups. <br /> not clearly defined (0.6-1.3 map units unc-22)                        Using truncated Poisson formula MENEELY <br />  352                                             D. V. Clark et al. <br /> and HERMAN(1979) number alleles         isolated 37, egg lethal alleles. How- <br /> each essential gene identified, predict  ever, sDf2 homozygotes arrest development the <br /> is minimum 48 essential genes sDj2 region. egg, sDj7, sDf8, sDj9 sDflU homozygotes <br /> The Poisson distribution inherently carries as-    arrest early larvae (Figure 2 Table 2). This <br /> sumption essential genes equally   indicates maybe asyet <span id='am-86' about='obo:NCBITaxon_32644' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-87' property="oboInOwl:hasRelatedSynonym" content="not specified" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasRelatedSynonym" content="unidentified organism" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasRelatedSynonym" content="none" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasRelatedSynonym" content="unknwon" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasRelatedSynonym" content="other" datatype="xsd:string"></span><span id='am-92'  ></span><span id='am-93' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasRelatedSynonym" content="unknown" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="unidentified" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasRelatedSynonym" content="unknown organism" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasRelatedSynonym" content="unclassified sequence" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasRelatedSynonym" content="sonstige nucleic acid" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasRelatedSynonym" content="not shown" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasRelatedSynonym" content="unspecified" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasRelatedSynonym" content="miscellaneous nucleic acid" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>unidentified</span> <br /> mutable. Work withDrosophila melunogaster shown    essential gene egg lethal mutant phenotype <br /> that assumption attempting    zone 1, 8, 9 10. Since calculated <br /> to accurately estimate gene number (HILLIKER, CHOV-    essential genes identified, this <br /> NICK CLARK      1981; LEFEVRE  WATKINS  1986). essential gene identified thefuture. <br /> Some genes clearly &#8220;hot spots&#65533;? EMS      Alternatively, possible 31 identi- <br /> mutagenesis (see Table 2). This particularly true   fied essential genes egg lethal null phenotype <br /> for let-56 haseight alleles whenthe mean allele  null allele isolated. A <br /> frequency one. It empha-    explanation egg lethal phenotype of <br /> sized estimate minimum one.         sDf2 homozygotes synthetic effect <br /> Another factor distort gene number esti-    deleting number essential genes. The possi- <br /> mate, making high, interallelic complemen-   bility second mutation sDj2 <br /> tation. With addition deficiency break-    chromosome causing egg lethal phenotype is <br /> points alleles gene, presence     small,since sDj2 originally maintained an <br /> of complementing alleles revealed   unbalanced strain, <br /> exist. let-91 locus sDj2 region    Of 63 sDj2 region lethal mutations, 49 were <br /> shows complex complementation pattern. For       analyzed respect phenotypes let- <br /> purpose calculating essential gene number,   xllet-x let-xlsDj2 individuals. In 1 1 cases, the <br /> been considered single locus alleles.      observable stage developmental arrest varies from <br />    By comparing thenumber lethal mutations          homozygote hemizygote.Nine lethal mutations <br /> obtained nTl(ZV;V) region sDj2   exhibit severe phenotype hemizygote, <br /> region, arrive estimate number    indicating hypomorphic mutations. <br /> of essential genes nT1 balanced region    Two mutations, Zet-66(sI7 6 )and let-7U(sl132),exhibit <br /> can extended estimate genome.    mid larval stage ofarrest hemizygote an <br /> This calculation dependenton uniform          early larval homozygote. Either sl76 and <br /> distribution essential genes   nT1 balanced  s1132 hypermorphic (or antimorphic) there <br /> region, depends assumption    second mutations nT1 balanced region in <br /> the genes equally mutable. The estimated number    strains carrying mutations. <br /> of essential genes sDj2 region 48, 25       It lethal alleles Table <br /> chromosomes carrying lethal mutations sDj2      2 havevarying phenotypes homozygotesversus <br /> region isolated nT1 screen. In parallel,   hemizygotes,since method usetoclassify <br /> since 294 lethal mutations isolated nT1    lethal phenotypes, length nematode, <br /> balanced region, minimum 565 essential   approximate. Any fine distinctions betweenpheno- <br /> genes 49 map unit nT1 region, whichis           typeswouldprobablybemissed.What             present <br /> approximately sixth genome. This leads      guideline lethal phenotypes each <br /> to minimum estimate approximately 3500 essen-     essential gene. This guideline gives indication <br /> tial genes C. elegans genome, far stages development essential <br /> from original estimate 2000 essentialgenes      gene product required. We make assumption <br /> (BRENNER     1974). There problem extending    mutant phenotype essential gene an <br /> the essential gene estimate nT1 region  indication time expression development <br /> for entire genome. The nT1 region includes     gene product required at <br /> of autosomal gene clusterswhileitonly         time developmental arrest. <br /> includes autosomal arms. Therefore,        The continued analysis unc-22 region our <br /> the average essential gene density nT1 region  laboratory allow correlation genetic and <br /> is probably higher theoverall genome.    physical maps. This accomplished several <br /> For reason, essential gene number estimate    ways.   First,genetically   mapped deficiency break- <br /> for genome overestimate.                 points serve identify extent essential <br />    The lethal phenotypes EMS induced mutations      genes physical map, aswas accomplished in <br /> in sDj2 region range early larvae sterile  rosy-ace region D. <span id='am-76' about='obo:NCBITaxon_80614' typeof='owl:Thing obo:NCBITaxon_80611 obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_452333 obo:NCBITaxon_5204 obo:NCBITaxon_451864 obo:NCBITaxon_68889 obo:NCBITaxon_155619 obo:NCBITaxon_2759'><span id='am-77' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="Melanogaster" datatype="xsd:string"></span><span id='am-80'  ></span>melanogaster</span> (GAUSZ       et al. <br /> adults. Previously (ROGALSKI,    MOERMANand BAILLIE 1986). Second, isolation lethal alleles mutator <br />  1982), noted lethal allele sDj2 strain high mobilityof transposable <br /> region egg lethal phenotype. With addi-     element T c 1 (MOERMANand WATERSTON         1984) allows <br /> tion 26 mutations study previously tagging cloning essential genes (K. MCKIM, <br />                                                      Lethal Mutations C. elegans                                                  353 <br /> D. CLARK,R.JOHNSEN D. BAILLIE,unpublished                                                                                 15: 2295- <br />                                                                           12 region ofCaenorhabditis elegans. Nucleic Acids Res. <br /> results). Third, cosmidsassociatedwith geneti-                       2307. <br /> cally mapped RFLDs used rescue mutant                       HILLIKER,A. J., A. CHOVNICK        S. H. CLARK,1981 The <br />                                                                          relative mutabilities vitalgenes D. melanogarter. Drosophila <br /> strains germ line transformation (FIRE1986).                     Inform. Serv. 56: 64-65. <br />    Correlation genetic physicalmapsof                      HORVITZ,H. R., S. BRENNER,       J. HODGKIN    R. K. HERMAN, <br /> the unc-22 region willallowus       determinethe                      1979 Auniform genetic nomenclature forthe nematode <br /> density essential genes kilobase. We                     Caenorhabditis elegans. Mol. Gen. Genet. 175: 129-133. <br /> determine variabilityinessential                     HOWELL,  A. M., S. G. GILMOUR,     R.A. MANCEBOand A. M. ROSE, <br />                                                                          1987 Genetic analysis large autosomal region Caenor- <br /> gene density unc-22 physical                    hubditiselegans use free duplication. Genet. Res. <br /> map genetic map. Additionally,                       49: 207-213. <br /> the identification coding elements, finding                     LEFEVRE, G., W. WATKINS,      1986 The question total gene <br /> regions sequence identity C. elegans                      number Drosophila melanogasler. Genetics 113: 869-895. <br /> Caenorhabditisbriggsae (S. PRASADand D. BAILLIE,                      MENEELY,P.M., R.K. HERMAN,          1979 Lethals, steriles, and <br />                                                                          deficiencies aregion X-chromosome Caenorhabditis <br /> unpublished results), enable determine                    elegans. Genetics 92: 99- 115. <br /> number mutationally silent genes unc-22                     MENEELY,   P. <br />                                                                                   M.,    R. K. HERMAN,        1981 Suppression and <br /> region. Eventually, association ofcoding elements                    function X-linked lethal sterile mutations C. elegans. <br /> with genetic map  study expression                  Genetics 97: 65-84. <br /> at themolecular level revealmuch genome                    MOERMAN,   D. G., 1980 A genetic analysis unc-22 region in <br />                                                                          Caenorhabditis elegans. Ph.D. thesis, Simon Fraser University, <br /> organization C. elegans.                                              Burnaby, B.C., Canada. <br />    The authors wish thank <span id='am-13' about='obo:NCBITaxon_7780' typeof='owl:Thing obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_7711 obo:NCBITaxon_7858 obo:NCBITaxon_131567 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_33154 obo:NCBITaxon_7776 obo:NCBITaxon_7777 obo:NCBITaxon_7778 obo:NCBITaxon_117893 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_30475'><span id='am-14'  ></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Raja" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>RAJA</span> ROSENBLUTH,     ANNROSE,BOB         MOERMAN,    D.G., D.L. BAILLIE,1979 Genetic organization <br /> JOHNSEN,  KIM MCKIMand LINDAADDISON        helpful discussion        Caenorhabditis elegans: Fine structure analysis urn-22 <br /> throughout work M. R. ROCALSKI   technical assistance.      gene. Genetics 91: 95-103. <br /> We thanks reviewers useful helpful     MOERMAN,    D.G., D.L. BAILLIE,1981 Formaldehyde muta- <br /> comments manuscript. This research supported               genesis Caenorhabditis elegans. Mut. Res. 8 0 273-279. <br /> Natural Sciences Engineering Research Council Canada           MOERMAN,    D. G., R. H. WATERSTON,          1984 Spontaneous <br /> and Muscular Dystrophy Association Canada grants D.L.B.            unstable urn-22 IV mutations C. elegans var. Bergerac. <br /> and Muscular Dystrophy Association Canadapredoctoral                Genetics 108: 859-877. <br /> fellowship D.V.C.                                                  MOERMAN,D. G., G. M. BENIAN             R. H. WATERSTON, <br />                                                                          1986 Molecular cloning muscle gene urn-22 C. <br />                                                                          elegans Tcl transposon tagging. Proc. Natl. Acad. Sci. USA <br />                                                                          83: 2579-2583. <br />                    LITERATURECITED                                    ROGALSKI,  T. M., D.L. BAILLIE,1985 Genetic organization <br />                                                                          urn-22 IV gene theadjacent region Caenorhabditis <br /> BAILLIE, D. L., K. A. BECKENBACH   A. M. ROSE,1985 Cloning           elegans. Mol. Gen. Genet. 201: 409-414. <br />     urn43 unc-31 interval (linkage group IV)     ROGALSKI,  T. M., D. L. RIDDLE,1988 A Caenorhabditis elegans <br />     Caenorhabditis elegansgenome using Tcl linkage selection. Can.       RNA polymerase gene, m - 1 I V ,and nearby essential genes. <br />    J. Genet. Cytol. 27: 457-466.                                         Genetics 118: 61-74. <br /> BRENNER,  S., 1974 The genetics Caenorhabditis elegans. Genetics   ROCALSKI,T. M.,   G. <br />                                                                                         D.             MOERMAN D. L. BAILLIE, <br />     77: 71-94.                                                            1982 Essential genes deficiencies unc-22 IV region <br /> COULSON,  A., J. SULSTON,S. BRENNER    J. U R N ,1986 Toward         Caenorhabditis elegans. Genetics 102: 725-736. <br />     physical map genome nematode Caenorhabditis       ROSE, A. M., D. L. BAILLIE,1979 The effect temperature <br />     ehgans. Proc. Natl. Acad. Sci. USA 83: 7821-7825.                    parental age recombination nondisjunction in <br /> DONATI,L.A.M.,       1985 A genetic analysis right arm         Caenorhabditis elegans. Genetics 92: 409-418. <br />     linkage group IV Caenorhabddtis elegans, emphasis      ROSE,A. M., D.L. BAILLIE,1980 Genetic organization of <br />     sDj2 region. M.Sc. Thesis, Simon Fraser University, Bur-         region unc-15(1),a gene affecting paramyosin in <br />     naby, B.C., Canada.                                                  Caenorhabditis elegans. Genetics 96: 639-648. <br /> EDCLEY,  M. L., D.L. RIDDLE,1987 Caenorhabditis elegans. In:      ROSENBLUTH,    R.E., D.L. BAILLIE,1981 The genetic analysis <br />     Genetic Maps 1987, Vol. 4, Edited S. J. OBRIEN.Cold Spring        reciprocal translocation, eTl(III;V), Caenorhabditis ele- <br />     Harbor Laboratory, Cold Spring Harbo <br /> </body></html>