<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Activity Reaction Core Plasticity <br /> of Metabolic Networks <br /> Eivind Almaas1,2, Zolta&#180;n N. Oltvai3*, Albert-La&#180;szlo&#180; Baraba&#180;si2,4 <br /> 1 Microbial Systems Division, Biosciences Directorate, Lawrence Livermore National Laboratory, Livermore, <span id='am-1' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>California</span>, United States America, 2 Center Complex <br /> Network Research Department Physics, University Notre Dame, Notre Dame, Indiana, United States America, 3 Department Pathology, University Pittsburgh <br /> School Medicine, Pittsburgh, Pennsylvania, United States America, 4 Center <span id='am-36' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-37' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-40'  ></span>Cancer</span> Systems Biology, Dana-Farber Cancer Institute, Harvard University, Boston, <br /> Massachusetts, United States America <br />  <br />  <br /> Understanding system-level adaptive changes taking place organism response variations the <br /> environment key issue contemporary biology. Current modeling approaches, constraint-based flux- <br /> balance analysis, proved highly successful analyzing capabilities cellular metabolism, including its <br /> capacity predict deletion phenotypes, ability calculate relative flux values metabolic reactions, the <br /> capability identify properties optimal growth states. Here, use flux-balance analysis thoroughly assess the <br /> activity <span id='am-73' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-74' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-77'  ></span><span id='am-78' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span>Escherichia coli</span>, <span id='am-49' about='obo:NCBITaxon_210' typeof='owl:Thing obo:NCBITaxon_72293 obo:NCBITaxon_213849 obo:NCBITaxon_209 obo:NCBITaxon_1 obo:NCBITaxon_68525 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_29547'><span id='am-50' property="oboInOwl:hasAlternativeId" content="NCBITaxon:219" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="PMID:1995031" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasRelatedSynonym" content="Campylobacter pylori" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="PMID:8186097" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="PMID:8494747" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasRelatedSynonym" content="Campylobacter pyloridis" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasRelatedSynonym" content="Campylobacter pylori subsp. pylori" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="PMID:11931154" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasRelatedSynonym" content="Helicobacter nemestrinae" datatype="xsd:string"></span><span id='am-60'  ></span><span id='am-61' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="Helicobacter pylori" datatype="xsd:string"></span>Helicobacter pylori</span>, <span id='am-12' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-25'  ></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> metabolism 30,000 diverse simulated <br /> environments. We identify set metabolic reactions forming connected metabolic core carry non-zero fluxes <br /> under growth conditions, flux variations highly correlated. Furthermore, enzymes <br /> catalyzing core reactions display considerably higher fraction phenotypic essentiality evolutionary <br /> conservation catalyzing noncore reactions. Cellular metabolism characterized large number of <br /> <span id='am-46' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-47' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span>-specific conditionally active reactions organized evolutionary conserved, active, <br /> metabolic core. Finally, current antibiotics interfering bacterial metabolism target core <br /> enzymes, indicating findings important implications antimicrobial drug-target discovery. <br /> Citation: Almaas E, Oltvai ZN, Baraba&#180;si A (2005) The activity reaction core plasticity metabolic networks. PLoS Comput Biol 1(7): e68. <br />  <br />  <br />                                                                                           changes induce structural plasticity, resulting in <br /> Introduction <br />                                                                                           changes metabolism&#8217;s active wiring diagram, turning <br />   Constraint-based modeling approaches, &#64258;ux-bal-                                  previously zero-&#64258;ux reactions inhibiting previously <br /> ance analysis (FBA) [1,2], proved highly successful                               active pathways. For example, shifting E. coli cells from <br /> analyzing capabilities cellular metabolism, including                          glucose- succinate-rich media, 11 previously active <br /> capacity predict deletion phenotypes, ability                                   reactions turned completely, previously <br /> calculate relative &#64258;ux values metabolic reactions,                             inactive reactions turned on. <br /> the capability identify properties alternate optimal                                   The types response mechanisms described &#64258;ux <br /> growth states wide range simulated single-carbon-                                 structural plasticity imply possible existence a <br /> source environmental conditions [3&#8211;5]. Recent analyses                               group reactions subject structural plasticity, <br /> indicate deletion phenotype substantial subset                           active environmental conditions. Indeed, <br /> metabolic enzymes growth-condition-dependent [6], argu-                                metabolic reactions carry non-zero &#64258;uxes <br /> ing selective use metabolic reactions distinct                                S. cerevisiae different growth conditions [6]. In <br /> environments.                                                                             itself, however, &#65533;?nding compatible scenario in <br />                                                                                           active reactions randomly distributed: if <br /> Results                                                                                   typically q fraction metabolic reactions active <br />                                                                                           speci&#65533;?c growth condition, n distinct conditions <br />    To thoroughly examine utilization relative &#64258;ux                                 predict non-zero overlap encompassing qn <br /> rates metabolic reaction wide range simulated                             fraction reactions. Indeed, &#65533;?nd increase <br /> environmental conditions, sampled 30,000 randomly                                  number inspected conditions, n, number of <br /> uniformly chosen optimal growth conditions, all <br /> single-carbon-source minimal medium conditions suf&#65533;?cient <br />                                                                                           Received June 27, 2005; Accepted November 2, 2005; Published December 16, <br /> for growth, using FBA [1,2] reconstructed metabolic                                2005 <br /> networks <span id='am-63' about='obo:NCBITaxon_209' typeof='owl:Thing obo:NCBITaxon_72293 obo:NCBITaxon_213849 obo:NCBITaxon_1 obo:NCBITaxon_68525 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_29547'><span id='am-64' property="oboInOwl:hasDbXref" content="PMID:17329766" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="PMID:11156001" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="Helicobacter" datatype="xsd:string"></span><span id='am-68'  ></span><span id='am-69' property="oboInOwl:hasDbXref" content="PMID:15143020" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="PMID:1704793" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasAlternativeId" content="NCBITaxon:28203" datatype="xsd:string"></span>Helicobacter</span> pylori, <span id='am-89' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-90' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-91'  ></span><span id='am-92' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span>Escherichia</span> coli, <span id='am-29' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-30' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-31'  ></span><span id='am-32' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Saccharomyces</span>                      DOI: 10.1371/journal.pcbi.0010068 <br /> cerevisiae [7&#8211;9] (see Materials Methods). We                               Copyright: &#211; 2005 Almaas et al. <span id='am-41' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-42'  ></span><span id='am-43' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span> open-access article distributed the <br /> when assuming optimal growth microorganisms,                                 terms Creative Commons Attribution License, permits unrestricted <br />                                                                                           use, distribution, reproduction medium, provided original author <br /> their metabolism adapts environmental changes                                  source credited. <br /> two distinct mechanisms. The common mechanism is <br />                                                                                           Abbreviation: FBA, flux-balance analysis <br /> &#64258;ux plasticity, involving changes &#64258;uxes already <br /> active reactions organism shifted                                    Editor: Daniel Segre, Boston University, United States America <br /> growth condition another. For example, changing                                   * To correspondence addressed. E-mail: oltvai@pitt.edu <br /> glucose- succinate-rich media alters &#64258;ux 264 E. coli                            A previous version article appeared Early Online Release November <br /> reactions 20%. Less commonly, environmental                                  2, 2005 (DOI: 10.1371/journal.pcbi.0010068.eor). <br />  <br />  <br />         PLoS Computational Biology | www.ploscompbiol.org                          0557                                     December 2005 | Volume 1 | Issue 7 | e68 <br />                                                                                                                                      Metabolic Network Core <br />  <br />  <br /> Synopsis                                                                               &#64258;ux-based plasticity, rest reactions are <br />                                                                                        conditionally active, turned speci&#65533;?c <br /> Although cellular metabolism investigated                            growth conditions subject structural- <br /> cellular functions, understood changes                       &#64258;ux-based plasticity. <br /> adapts systems level response environmental variations. In                     Figure 2 displays metabolic reactions E. coli that <br /> this study, authors advantage reconstructed                            remain active 30,000 simulated growth conditions. The <br /> metabolic networks Helicobacter pylori, Escherichia coli,                       striking feature diagram fact reactions <br /> Saccharomyces cerevisiae computational method flux- <br />                                                                                        form single connected cluster, encompassing 90 <br /> balance analysis investigate functional plasticity these <br /> three metabolic networks large number simulated growth <br />                                                                                        reactions. This unique feature E. coli; H. pylori, all <br /> conditions.                                                                            138 reactions form single cluster, S. cerevisiae 33 <br />                                                                                        reactions connected. Given relatively low number of <br /> Within approach, authors identify metabolic cores                      reactions active, likelihood form <br /> these organisms. These metabolic cores represent connected sets                     single large cluster chance negligible, p , <br /> reactions used tested environments. When cross-                        1e 6 H. pylori E. coli, p ; 2e 6 S. cerevisiae. Given <br /> correlating predicted cores experimental <span id='am-101' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-102' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-103' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-105'  ></span>data</span>, authors                    compact clustered nature group reactions <br /> find cores display significantly higher fraction, of <br />                                                                                        active, refer collectively the <br /> essential evolutionary conserved enzymes, noncore <br /> counterparts. Additionally, extended mRNA half-lives core <br />                                                                                        metabolic core. <br /> enzymes support notion core reactions                            The metabolic core contains types reactions: The <br /> represent main integrators metabolic activity.                                      &#65533;?rst type consists essential biomass <br />                                                                                        formation environmental conditions (81 90 <br /> Since metabolic core reactions indispensable                   reactions E. coli), second type reaction is <br /> growth microorganism, existing antibiotics target                   required assure optimal metabolic performance <br /> select bacterial enzymes core, authors suggest                     (Table S1). In case inactivation second type, <br /> metabolic core important applications antimicrobial <br />                                                                                        alternative sub-optimal pathways used ensure <br /> drug-target discovery. <br />                                                                                        cellular survival. Interestingly, compact core S. <br />                                                                                        cerevisiae, 33 reactions predicted indispensable <br />                                                                                        biomass formation growth conditions. More- <br /> reactions carrying non-zero &#64258;ux condition decreases                            over, assuming 10% reduction growth rate <br /> rapidly. However, eventually saturates constant value                          (compared optimal growth), size identity E. <br /> (Figure 1A&#8211;C), identifying group reactions non-                         coli metabolic core remains largely unchanged, retaining 83 of <br /> zero &#64258;ux 30,000 simulated growth conditions.                                 original 90 reactions, nonessential. This <br /> Speci&#65533;?cally, &#65533;?nd 138 381 H. pylori (36.2%), 90                           indicates concept metabolic core valid under <br /> of 758 E. coli (11.9%), 33 1,172 S. cerevisiae (2.8%),                          optimal suboptimal growth, some <br /> metabolic reactions active (Figure 1D 1E).                              difference identity individual reactions. Of note, the <br />                                                                                        metabolic core represents subset minimal reaction <br /> While reactions respond environmental changes only <br />                                                                                        sets [10,11] overlap alternative (i.e., degenerate) <br />                                                                                        minimal reaction sets [4,5]. The minimal reaction set [11] <br />                                                                                        contains metabolic core addition, however, to <br />                                                                                        reactions necessary sustained growth chosen <br />                                                                                        substrate, minimal reaction set [5] consists of <br />                                                                                        201 reactions active E. coli 136 <br />                                                                                        aerobic anaerobic single-carbon-source minimal envi- <br />                                                                                        ronments capable sustaining optimal growth. The latter <br />                                                                                        &#65533;?nding excellent agreement Figure 1B (upper <br />                                                                                        curve), demonstrates analysis low number of <br />                                                                                        growth conditions signi&#65533;?cantly overestimate set of <br />                                                                                        reactions active. <br />                                                                                           To identify factors determine size of <br />                                                                                        metabolic core, note number core <br />                                                                                        metabolic reactions systematically decreases from <br />                                                                                        H. pylori S. cerevisiae (Figures 3A 1E), relative size of <br />                                                                                        core decreasing 36.2% reactions H. pylori to <br /> Figure 1. The Emergence Metabolic Core <br />                                                                                        11.9% E. coli 2.8% S. cerevisiae. This trend be <br /> (A&#8211;C) The average relative size number reactions are <br /> always active function number sampled conditions (black                 explained collective network effect: relatively small <br /> line) (A) H. pylori, (B) E. coli, (C) S. cerevisiae. As number          size H. pylori metabolic network leaves little &#64258;exibility <br /> conditions increases, curve converges constant denoted                 biomass production, requiring continuous activity of <br /> dashed line, identifying metabolic core organism. The red line <br /> denotes number reactions active activity is <br />                                                                                        high fraction available metabolic reactions. Indeed, <br /> randomly distributed metabolic network. The fact                        &#65533;?nd average approximately 61% H. pylori <br /> converges zero indicates real core represents collective                 reactions active given environmental condition. The <br /> network effect, forcing group reactions active conditions.           larger number metabolic reactions present E. coli offers <br /> (D E) The number metabolic reactions (D) number of <br /> metabolic core reactions (E) studied organisms.                           higher degree metabolic &#64258;exibility, allowing a <br /> DOI: 10.1371/journal.pcbi.0010068.g001                                                 signi&#65533;?cant fraction biomass produced by <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                        0558                               December 2005 | Volume 1 | Issue 7 | e68 <br />                                                                                                                                    Metabolic Network Core <br />  <br />  <br />  <br />  <br /> Figure 2. The Metabolic Core E. coli <br /> All reactions active 30,000 investigated external conditions shown. Metabolites contribute directly to <br /> biomass formation [8] colored blue, core reactions (links) catalyzed essential (or nonessential) enzymes [13] colored red (or green). <br /> (Black-colored links denote enzymes unknown deletion phenotype.) The blue dashed lines indicate multiple appearances metabolite, while <br /> links arrows denote unidirectional reactions. Note 20 51 metabolites necessary biomass synthesis present core, <br /> indicating produced (or consumed) growth-condition-specific manner. See Protocol S1 Table S1 abbreviations of <br /> metabolites list core reactions E. coli, H. pylori, S. cerevisiae. The folate peptidoglycan biosynthesis pathways indicated blue <br /> and brown shading, respectively, white numbered arrows denote current antibiotic targets inhibited by: (1) sulfonamides, (2) trimethoprim, (3) <br /> cycloserine, (4) fosfomycin. Note reactions appear disconnected omitted drawing cofactors. <br /> DOI: 10.1371/journal.pcbi.0010068.g002 <br />  <br />  <br />  <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                   0559                                December 2005 | Volume 1 | Issue 7 | e68 <br />                                                                                                                                Metabolic Network Core <br />  <br />  <br />                                                                                    alternative pathways. Indeed, average utilization E. <br />                                                                                    coli metabolic network given growth condition only <br />                                                                                    35.3%. For S. cerevisiae, metabolic network size <br />                                                                                    signi&#65533;?cantly exceeds H. pylori E. coli, there <br />                                                                                    higher metabolic &#64258;exibility, activity only <br />                                                                                    19.7% reactions required typical environment. <br />                                                                                       The fact core reactions active all <br />                                                                                    investigated environmental conditions suggests they <br />                                                                                    play key role maintaining metabolism&#8217;s overall <br />                                                                                    functional integrity. Therefore, absence individual <br />                                                                                    reactions metabolic core lead to <br />                                                                                    signi&#65533;?cant metabolic disruptions. Indeed, using genome-scale <br />                                                                                    deletion-phenotype data obtained rich growth media <br />                                                                                    [12,13], &#65533;?nd 74.7% E. coli enzymes that <br />                                                                                    catalyze core metabolic reactions (i.e., core enzymes) are <br />                                                                                    essential, compared 19.6% lethality fraction character- <br />                                                                                    izing noncore enzymes. The gap core and <br />                                                                                    noncore enzymes signi&#65533;?cant S. cerevisiae, which <br />                                                                                    essential enzymes catalyze 84% core reactions, whereas <br />                                                                                    average essentiality conditionally active enzymes is <br />                                                                                    15.6% (Figure 3B). Note likelihood random <br />                                                                                    concentration essential enzymes core is <br />                                                                                    extremely small, p-values 3.3e 23 9.0e 13 E. coli <br />                                                                                    yeast, respectively, presume presence of <br />                                                                                    errors deletion-phenotype data (Protocol S1). Taken <br />                                                                                    together, results indicate organism&#8217;s ability to <br />                                                                                    adapt changing environmental conditions rests, large <br />                                                                                    extent, continuous activity metabolic core, <br />                                                                                    regardless environmental conditions. <br />                                                                                       Intuitively, assume core represents a <br />                                                                                    subset high-&#64258;ux reactions characterizing activity of <br />                                                                                    metabolic networks [3]. However, measurements indicate <br />                                                                                    that, average, &#64258;uxes core noncore reactions <br />                                                                                    highly comparable (Figure 3C). Alternatively, could <br />                                                                                    assume main biological role metabolic <br />                                                                                    core ensure continuous production biomass under <br />                                                                                    growth conditions. In contrast, &#65533;?nd 20 the <br />                                                                                    51 metabolites (39%) considered necessary biomass <br />                                                                                    production E. coli [8] produced core <br />                                                                                    reactions; instead&#8212;in growth-condition-dependent fash- <br />                                                                                    ion&#8212;they produced various alternative metabolic <br />                                                                                    pathways. The core, however, contains large number of <br />                                                                                    reactions selected anabolic pathways, including of <br />                                                                                    membrane lipid-, cell envelope-, peptidoglycan-biosyn- <br />                                                                                    thesis pathways (Table S1). These core reactions represent <br />                                                                                    network bottlenecks, paths synthesis of <br />                                                                                    certain biomass components. Therefore, appears the <br />                                                                                    composition metabolic core determined two <br />                                                                                    factors. First, metabolic reactions directly contrib- <br />                                                                                    <span id='am-111' about='obo:NCBITaxon_365374' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_86011 obo:NCBITaxon_33208 obo:NCBITaxon_1 obo:NCBITaxon_6040 obo:NCBITaxon_45124 obo:NCBITaxon_131567 obo:NCBITaxon_73655 obo:NCBITaxon_27929 obo:NCBITaxon_2759'><span id='am-112'  ></span><span id='am-113' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-115' property="rdfs:label" content="Ute" datatype="xsd:string"></span>ute</span> biomass production tend metabolic <br />                                                                                    core organism. Second, however, tendency offset <br />                                                                                    network-induced redundancy: reactions pathways <br />                                                                                    end-products synthesized two <br />                                                                                    alternative pathways environmental redundancy and <br /> Figure 3. Characterizing Metabolic Cores                                       structural plasticity, eliminated core. <br /> (A) The number overlapping metabolic reactions metabolic core            Therefore, reactions metabolic network possesses <br /> of H. pylori, E. coli, S. cerevisiae.                                          (see Figure 1D), stronger network-induced <br /> (B) The fraction metabolic reactions catalyzed essential enzymes          redundancy, smaller core (Figure 1E). <br /> the cores (black) outside core E. coli S. cerevisiae. <br /> (C) The distribution average metabolic fluxes core the <br />                                                                                       Given important functional role played the <br /> noncore reactions E. coli.                                                      metabolic core given organism, expect <br /> DOI: 10.1371/journal.pcbi.0010068.g003                                             signi&#65533;?cant parts core conserved different <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                    0560                             December 2005 | Volume 1 | Issue 7 | e68 <br />                                                                                                                      Metabolic Network Core <br />  <br />  <br /> organisms. Indeed, E. coli H. pylori cores 63            redundant core, regulatory mechanisms shifted the <br /> reactions common, 18 33 core reactions S.            mRNA decay rate nature regulatory <br /> cerevisiae present E. coli H. pylori          interactions, offering higher regulatory stability the <br /> metabolic cores (see Figure 3A). Also, considering               core enzymes. <br /> enzyme orthologs 32 divergent bacteria [13], &#65533;?nd                The &#65533;?nding core reactions form single cluster <br /> that metabolic core enzymes E. coli display high             suggest activity participating reactions <br /> degree evolutionary conservation, average core enzyme          highly synchronized, changes &#64258;ux one <br /> having orthologs 71.7% reference bacteria (p ,              reaction affecting &#64258;ux reactions well. Thus, we <br /> 1e 6). In contrast, conditionally active noncore enzymes          able discover core inspecting the <br /> have evolutionary retention 47.7% [13]. Given          correlations activities metabolic reactions. <br /> observed correlation evolutionary retention           To test hypothesis, calculated &#64258;ux correlation <br /> essentiality gene products [13], difference          coef&#65533;?cient, Cij, reaction pair E. coli metabolism <br /> simple consequence high-lethality fraction core         [17] inspecting &#64258;ux values reactions each <br /> enzymes. However, analysis indicates         30,000 simulated growth conditions. Using Cij <br /> the case: random selection 90 enzymes 74.7%                 matrix metric hierarchical clustering algorithm <br /> lethality ratio average evolutionary retention         [18], observed emergence group reactions <br /> 63.4%. Taken together, results indicate major              &#64258;uxes change simultaneously environmental <br /> portions metabolic core conserved,                   shifts. Interestingly, highly correlated reactions signi&#65533;?- <br /> displaying higher evolutionary retention individual        cantly overlap metabolic core (Figure 4A), indicating <br /> essentiality participating enzymes indicate,             environmental conditions altered, the <br /> suggesting maintaining core&#8217;s integrity collective      &#64258;uxes core reactions increased decreased a <br /> need organism.                                                 synchronized fashion. <br />    The requirement continuous activity core               The synchrony observed inspect exper- <br /> reactions impact regulation catalytic        imental mRNA data core reactions changes <br /> enzymes. We expect activity core                    environmental conditions. Calculating correlations <br /> reactions&#8212;for &#64258;ux magnitude needs                core noncore enzymes using correspond- <br /> modulated&#8212;should display higher degree stability           ing mRNA copy number data (see Materials Methods) for <br /> different regulatory control reactions outside        41 experiments [19], &#65533;?nd correlations among <br /> core. Evidence regulatory effects provided         mRNA copy numbers core enzymes systematically <br /> measurements. First, inspected experimentally deter-           higher observed correlations noncore <br /> mined mRNA half-lives E. coli metabolic enzymes           counterparts (Figure 4B), average correlation ,C. <br /> cells grown <span id='am-7' about='obo:NCBITaxon_218032' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_69557 obo:NCBITaxon_69555 obo:NCBITaxon_69556 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-10'  ></span><span id='am-11' property="rdfs:label" content="Luria" datatype="xsd:string"></span>Luria</span>-Bertani medium [14]. We               &#188; 0.23 core enzymes ,C. &#188; 0.07 noncore enzymes <br /> that distribution mRNA half-life times core            (p , 1e 4). Notably, &#65533;?nding different average <br /> enzymes displays shift higher values, average           correlation mRNA expression 66 &#8216;&#8216;correlated <br /> half-life time mRNAs 14.0 min compared            reaction sets&#8217;&#8217; (groups reactions that <br /> 10.5 min noncore mRNAs (p &#188; 0.016). Furthermore,              turned together), frequency with <br /> only 2% core enzymes corresponding mRNA half-             correlated reaction set used does affect its <br /> life times shorter 5 min, compared 23% ones           mRNA correlation [5]. <br /> catalyzing conditionally active noncore reactions (see Materi- <br /> als Methods). Therefore, continuous metabolic need                Discussion <br /> for reactions apparently affected mechanisms <br /> responsible mRNA decay catalytic enzymes,                       Previous studies &#65533;?rmly established environmental <br /> providing higher dynamic inertia environmental                    changes induce &#64258;ux plasticity, altering &#64258;ux levels of <br /> changes.                                                                  individual reactions (reviewed [20]). Similarly, fact that <br />    Additionally, enzyme-encoding operon,                      metabolism displays structural plasticity, turning off <br /> counted number activating repressive regulatory                reactions growth conditions altered, been <br /> links E. coli transcriptional regulatory network [15,16].          observed [3,4,6]. However, demonstration a <br /> As regulatory interactions currently known 13             group reactions predicted active environmental <br /> the core enzyme-encoding operons, considered                       conditions forming connected metabolic core could <br /> extended core, representing set 234 reactions                 substantially improve understanding organization <br /> are active 90% 30,000 simulated growth                utilization metabolic networks. The emergence the <br /> conditions. The results indicate fraction repressive          core represents collective network effect, channeling the <br /> regulatory links extended core 52.3%,                 production indispensable biomass components a <br /> fraction activating interactions 35.7%,                key reactions replaced alternative <br /> remaining 12% represents regulatory links                 pathways environmental conditions. The collective <br /> activate suppress enzyme&#8217;s mRNA synthesis rate. In                 origin core supported observed changes the <br /> contrast, noncore enzyme-encoding operons,                core size. While number biomass components the <br /> difference fraction activating repressing              metabolism needs produce comparable three <br /> links, representing 45% regulatory interactions.              organisms (49 H. pylori, 51 E. coli, 44 S. <br /> These results offer evidence co-evolution core             cerevisiae), number metabolic reactions contributing to <br /> metabolic network corresponding regulatory network:               differs. The larger size metabolic network <br /> given requirement continuously active non-                    signi&#65533;?cantly increases capacity redundancy, decreas- <br />  <br />       PLoS Computational Biology | www.ploscompbiol.org            0561                             December 2005 | Volume 1 | Issue 7 | e68 <br />                                                                                                                                          Metabolic Network Core <br />  <br />  <br />                                                                                       bacterial yeast gene products essential, a <br />                                                                                       high fraction essential speci&#65533;?c environ- <br />                                                                                       ments. For example, recent measurements indicate 76% <br />                                                                                       S. cerevisiae genes inactive nutrient-rich <br />                                                                                       conditions fact active, essential in <br />                                                                                       growth conditions [6]. However, effective <br />                                                                                       antimicrobial drug needs able kill target organism <br />                                                                                       physiological conditions exist. Thus <br />                                                                                       pharmacological interventions targeting speci&#65533;?c pathway <br />                                                                                       effective environments pathway not <br />                                                                                       needed corresponding enzyme turned off. Instead, <br />                                                                                       effective antimicrobials target activity of <br />                                                                                       core reactions, disruption impact on <br />                                                                                       microorganism&#8217;s ability function environ- <br />                                                                                       mental conditions. Indeed, currently used anti- <br />                                                                                       biotics, fosfomycin cycloserine act inhibiting cell-wall <br />                                                                                       peptidoglycan, sulfonamides trimethoprim inhibit <br />                                                                                       tetrahydrofolate biosynthesis, pathways present in <br />                                                                                       H. pylori E. coli core (see Figure 2). In addition, <br />                                                                                       core pathways involved synthesis &#64258;avin adenine <br />                                                                                       dinucleotide, 2-dehydro-3-deoxy-D-octonate, lipopolysac- <br />                                                                                       charide potential antibiotic targets [21]. <br />                                                                                          This indicates core enzymes essential biomass <br />                                                                                       formation, optimal suboptimal growth (Table <br />                                                                                       S1), prove effective antibiotic targets given cell&#8217;s <br />                                                                                       need maintain activity enzymes all <br />                                                                                       conditions. The fact core reactions are <br />                                                                                       shared bacteria offers possibility identify <br />                                                                                       bacterium-speci&#65533;?c drug targets. Finally, results pertain- <br />                                                                                       ing existence core characteristics by <br />                                                                                       means limited studied organisms, the <br />                                                                                       analysis carried organisms high- <br />                                                                                       quality metabolic reconstructions available. Given the <br />                                                                                       large number sequenced bacterial genomes, studies <br />                                                                                       open new avenues rapid silico antimicrobial <br />                                                                                       drug-target identi&#65533;?cation. <br />  <br />                                                                                       Materials Methods <br /> Figure 4. Correlations E. coli Metabolic Reactions                                FBA. Starting published stoichiometric matrices the <br /> (A) We calculated Pearson correlation using flux values 30,000               reconstructed E. coli MG1655, H. pylori, S. cerevisiae metabolic <br /> conditions reaction pair grouping reactions                       networks [7&#8211;9], steady-state concentrations internal <br /> according hierarchical average-linkage clustering algorithm. The                 metabolites organism satisfy <br /> values flux-correlation matrix range  1 (red) 0                                           d         X <br /> (white) unity (blue). The horizontal color bar denotes reaction                                    &#189;Ai   &#188;    Sij mj &#188; 0                  &#240;1&#222; <br />                                                                                                              dt          j <br /> member core (green), vertical color bar denotes whether <br /> the enzymes catalyzing reaction essential (red).                              Sij stoichiometric coef&#65533;?cient metabolite Ai reaction <br /> (B) Distribution Pearson correlation mRNA copy numbers 41                  j, mj &#64258;ux reaction j. We use convention if <br /> experiments [14]. The correlations core reactions clearly shifted          metabolite Ai substrate (or product) reaction j, Sij , 0 (Sij . 0). <br /> towards higher values, average correlation coefficient ,C. &#188;               Any vector &#64258;uxes fmjg satis&#65533;?es equation 1 corresponds a <br /> 0.23 compared average noncore coefficient ,C. &#188; 0.07                      state metabolic network, hence, potential state of <br /> DOI: 10.1371/journal.pcbi.0010068.g004                                                operation cell. Using linear programming, choose the <br />                                                                                       solution maximizes biomass production respective <br />                                                                                       organisms. Reactions active, likely re&#64258;ecting annota- <br />                                                                                       tion errors incomplete data, ignored &#64258;ux analysis. <br /> ing size metabolic core. Therefore, metabolic                             Metabolic core identi&#65533;?cation. We model possible cellular <br /> core contains reactions necessary optimal cellular                       environments investigated FBA models, described before <br /> performance regardless environmental conditions,                               [3]. Brie&#64258;y, metabolic-uptake reaction, &#65533;?x uptake rate <br />                                                                                       randomly chosen value 0 20 mmol/g/h (dry <br /> while conditionally active metabolic reactions represent                          weight) calculating optimal &#64258;uxes con&#65533;?guration <br /> the different ways cell capable optimally                          using linear programming. Each reaction subsequently tagged as <br /> utilizing substrates environment.                                            active (non-zero &#64258;ux) inactive (zero &#64258;ux). Since a <br />                                                                                       large number possible combinations selected uptake <br />   The identi&#65533;?cation metabolic core important                          rates, repeat process 30,000 times. In addition, calculate <br /> practical implications: given continuous activity high                        optimal &#64258;uxes single-carbon-source con&#65533;?gurations a <br />                                                                                       minimal-uptake medium consisting unlimited <span id='am-96' about='obo:NCBITaxon_29189' typeof='obo:NCBITaxon_29178 owl:Thing obo:NCBITaxon_203397 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_69034 obo:NCBITaxon_29185 obo:NCBITaxon_543769 obo:NCBITaxon_2759'><span id='am-97'  ></span><span id='am-98' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-99' property="rdfs:label" content="Ammonia" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>ammonia</span>, sulfate, <br /> degree essentiality core enzymes, represent                            phosphate, carbon dioxide, potassium, restricted oxygen E. <br /> potential targets antimicrobial intervention. Speci&#65533;?cally,                        coli, H. pylori, S. cerevisiae, plus unlimited ergosterol zymosterol <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                       0562                                  December 2005 | Volume 1 | Issue 7 | e68 <br />                                                                                                                                               Metabolic Network Core <br />  <br /> for S. cerevisiae. For reaction, assign value, qi, zero             database (http://www.shigen.nig.ac.jp/ecoli/pec) given Table S2 of <br /> and unity, representing relative number conditions                      Gerdes et al. [13], resulting total 62 essential enzymes and <br /> reaction active. The preliminary metabolic core de&#65533;?ned                    seven enzymes determine deletion <br /> set reactions active sampled conditions (qi &#188; 1). We                  phenotype. The deletion phenotype yeast enzymes taken <br /> determine &#65533;?nal metabolic core removing preliminary                       CYGD database [12]. <br /> core reactions that, &#64258;ux rate constrained zero <br /> over total set initial conditions, leave growth rate <br /> unchanged.                                                                               Supporting Information <br />    Correlations clustering. We evaluate FBA &#64258;ux correlations <br /> by calculating Pearson coef&#65533;?cient possible reaction pair                     Protocol S1. Supplementary Analyses List Metabolite <br /> combinations 30,000 different simulated growth conditions.                        Abbreviations <br /> We subsequently group reactions employing correlation <br /> values metric standard hierarchical average-linkage                          Found DOI: 10.1371/journal.pcbi.0010068.sd001 (628 KB DOC). <br /> clustering algorithm [22].                                                               Table S1. Metabolic Core Reactions <br />    Data analysis. The annotated metabolic FBA models specify the <br /> enzymes genes catalyzing various metabolic reactions. We                         Found DOI: 10.1371/journal.pcbi.0010068.st001 (57 KB XLS). <br /> represent mRNA activity level half-life time metabolic <br /> enzyme averaging available experimental values, Ek, the <br /> catalyzing gene products,                                                             Acknowledgments <br />                                     1 X <br />                                ri &#188;           Ek                         &#240;2&#222;             We thank S. Mobashery S. Vakulenko discussions. Work the <br />                                     Ni k&#188;1;Ni                                            University Notre Dame University Pittsburgh was <br />                                                                                          supported US Department Energy, National Institutes of <br /> where Ni number genes. We assess essentiality                         Health, National Science Foundation. <br /> reaction considering essential catalyzing <br /> enzymes (or gene products) essential survival                             Competing interests. The authors declared competing <br /> organism (see Protocol S1 error analysis).                                           interests exist. <br />   Core noncore essentiality. The deletion phenotype E. coli                          Author contributions. EA, ZNO, ALB conceived study. EA <br /> enzymes taken Gerdes et al. [13]. For nonlethal core                            ALB designed study. EA performed computational <br /> reactions, cross-checked deletion phenotype PEC                          work. EA, ZNO ALB wrote paper.                             &amp; <br />  <br />  <br />  <br />  <br /> References                                                                               12. Gu&#168;ldener U, Mu&#168;nsterko&#168;tter M, Kastenmu&#168;ller G, Strack N, van Helden J, et <br /> 1. Edwards JS, Palsson BO (2000) The Escherichia coli MG1655 silico                       al. (2005) CYGD: The Comprehensive Yeast Genome Database. Nucleic <br />     metabolic genotype: Its de&#65533;?nition, characteristics, capabilities. Proc               Acids Res 33: D364&#8211;D368. <br />     Natl Acad Sci U S A 97: 5528&#8211;5533.                                                   13. Gerdes SY, Scholle MD, Campbell JW, Balazsi G, Ravasz E, et al. (2003) <br /> 2. Schilling CH, Letscher D, Palsson BO (2000) Theory systemic                       Experimental determination level analysis essential genes in <br />     de&#65533;?nition metabolic pathways use interpreting metabolic                  Escherichia coli MG1655. J Bacteriol 185: 5673&#8211;5684. <br />     function pathway-oriented perspective. J Theor Biol 203: 229&#8211;248.             14. Selinger DW, Saxena RM, Cheung KJ, Church GM, Rosenow C (2003) Global <br /> 3. Almaas E, Kova&#180;cs B, Vicsek T, Oltvai ZN, Baraba&#180;si AL (2004) Global                      RNA half-life analysis Escherichia coli reveals positional patterns of <br />     organization metabolic &#64258;uxes bacterium Escherichia coli. Nature                transcript degradation. Genome Res 13: 216&#8211;223. <br />     427: 839&#8211;843.                                                                        15. Shen-Orr S, Milo R, Mangan S, Alon U (2002) Network motifs the <br /> 4. Burgard AP, Nikolaev EV, Schilling CH, Maranas CD (2004) Flux coupling                    transcriptional regulation network Escherichia coli. Nat Genet 31: 64&#8211;68. <br />     analysis genome-scale metabolic reconstructions. Genome Res 14: 301&#8211;              16. Salgado H, Gama-Castro S, Martinez-Antonio A, Diaz-Peredo E, Sanchez- <br />     312.                                                                                     Solano F, et al. (2004) RegulonDB (version 4.0): Transcriptional regulation, <br /> 5. Reed JL, Palsson BO (2004) Genome-scale silico models E. coli                  operon organization growth conditions <span id='am-106' about='obo:NCBITaxon_83333' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-107' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-108' property="rdfs:label" content="Escherichia coli K-12" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasExactSynonym" content="Escherichia coli K12" datatype="xsd:string"></span>Escherichia coli K-12</span>. Nucleic <br />     multiple equivalent phenotypic states: Assessment correlated reaction                 Acids Res 32: D303&#8211;D306. <br />     subsets comprise network states. Genome Res 14: 1797&#8211;1805.                      17. Wiback SJ, Famili I, Greenberg HJ, Palsson BO (2004) Monte Carlo <br /> 6. Papp B, Pal C, Hurst LD (2004) Metabolic network analysis causes <br />                                                                                              sampling used determine size shape steady-state <br />     evolution enzyme dispensability yeast. Nature 429: 661&#8211;664. <br />                                                                                              &#64258;ux space. J Theor Biol 228: 437&#8211;447. <br /> 7. Schilling CH, Covert MW, Famili I, Church GM, Edwards JS, et al. (2002) <br />                                                                                          18. Eisen MB, Spellman PT, Brown PO, Botstein D (1998) Cluster analysis and <br />     Genome-scale metabolic model <span id='am-116' about='obo:NCBITaxon_85962' typeof='owl:Thing obo:NCBITaxon_72293 obo:NCBITaxon_213849 obo:NCBITaxon_209 obo:NCBITaxon_1 obo:NCBITaxon_68525 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_29547 obo:NCBITaxon_210'><span id='am-117' property="oboInOwl:hasRelatedSynonym" content="Helicobacter pylori KE26695" datatype="xsd:string"></span><span id='am-118' property="rdfs:label" content="Helicobacter pylori 26695" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasExactSynonym" content="Helicobacter pylori (strain 26695)" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasDbXref" content="PMID:9252185" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasRelatedSynonym" content="Helicobacter pylori ATCC 700392" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasExactSynonym" content="Helicobacter pylori str. 26695" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasExactSynonym" content="Helicobacter pylori strain 26695" datatype="xsd:string"></span>Helicobacter pylori 26695</span>. J Bacteriol 184: <br />                                                                                              display genome-wide expression patterns. Proc Natl Acad Sci U S A 95: <br />     4582&#8211;4593. <br /> 8. Reed JL, Vo TD, Schilling CH, Palsson BO (2003) An expanded genome-                       14863&#8211;14868. <br />     scale model Escherichia coli K-12 (iJR904 GSM/GPR). Genome Biol 4:                19. Allen TE, Herrgard MJ, Liu MZ, Qiu Y, Glasner JD, et al. (2003) Genome- <br />     R54.1&#8211;R54.12.                                                                            scale analysis uses Escherichia coli genome: Model-driven <br /> 9. Duarte NC, Herrgard MJ, Palsson BO (2004) Reconstruction validation                   analysis heterogeneous data sets. J Bacteriol 185: 6392&#8211;6399. <br />     Saccharomyces cerevisiae iND750, fully compartmentalized genome-scale           20. Price ND, Reed JL, Palsson BO (2004) Genome-scale models microbial <br />     metabolic model. Genome Res 14: 1298&#8211;1309.                                               cells: Evaluating consequences constraints. Nat Rev Microbiol 2: 886&#8211; <br /> 10. Burgard AP, Maranas CD (2001) Probing performance limits                      897. <br />     Escherichia coli metabolic network subject gene additions deletions.           21. Gerdes SY, Scholle MD, D&#8217;Souza M, Bernal A, Baev MV, et al. (2002) From <br />     Biotechnol Bioeng 74: 364&#8211;375.                                                           genetic footprinting antimicrobial drug targets: Examples cofactor <br /> 11. Burgard AP, Vaidyraman S, Maranas CD (2001) Minimal reaction sets                    biosynthetic pathways. J Bacteriol 184: 4555&#8211;4572. <br />     Escherichia coli metabolism different growth requirements uptake           22. Dunn G, Everitt BS (1982) An introduction mathematical taxonomy. <br />     environments. Biotechnol Prog 17: 791&#8211;797.                                               Cambridge: Cambridge University Press. 152 p. <br />  <br />  <br />  <br />  <br />        PLoS Computational Bio <br /> </body></html>