<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Dec. 2006, p. 8683&#8211;8696                                                                           Vol. 26, No. 23 <br /> 0270-7306/06/$08.00&#11001;0 doi:10.1128/MCB.00940-06 <br /> Copyright &#169; 2006, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />     Breast <span id='am-43' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-44'  ></span><span id='am-45' property="rdfs:label" content="Cancer" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Cancer</span> Metastasis Suppressor 1 Functions Corepressor <br />       Enhancing Histone Deacetylase 1-Mediated Deacetylation <br />                 RelA/p65 Promoting Apoptosis&#17188;&#8224; <br />                                       Yuan Liu, Philip W. Smith, David R. Jones* <br />                        Department Surgery, University <span id='am-6' about='obo:NCBITaxon_183605' typeof='owl:Thing obo:NCBITaxon_1329950 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8504 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_8509 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_1329912 obo:NCBITaxon_33208 obo:NCBITaxon_1329961 obo:NCBITaxon_1329911 obo:NCBITaxon_7711 obo:NCBITaxon_8570 obo:NCBITaxon_169862 obo:NCBITaxon_7776 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_8578 obo:NCBITaxon_34989'><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8'  ></span><span id='am-9' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Virginia" datatype="xsd:string"></span>Virginia</span> School Medicine, Charlottesville, Virginia <br />                         Received 26 May 2006/Returned modification 1 August 2006/Accepted 11 September 2006 <br />  <br />              The antiapoptotic transcription factor NF-&#65533;?&#65533;B constitutively activated cancers important <br />           cytokine-mediated progression metastatic movement tumors. Breast cancer metastasis suppressor <br />           1 (BRMS1) metastasis suppressor gene mechanisms action poorly understood. In report, <br />           demonstrate BRMS1 decreases transactivation potential RelA/p65 ameliorates expres- <br />           sion NF-&#65533;?&#65533;B-regulated antiapoptotic gene products. BRMS1 immunoprecipitates RelA/p65 subunit <br />           NF-&#65533;?&#65533;B protein-protein interactions occurring C terminus region rel homology domain but <br />           known transactivation domains. Moreover, BRMS1 functions corepressor promoting binding <br />           HDAC1 RelA/p65, deacetylates lysine K310 RelA/p65, suppresses RelA/p65 transcrip- <br />           tional activity. Selective small interfering RNA knockdown BRMS1 confirms chromatin-bound BRMS1 <br />           required deacetylation RelA/p65, enhancing chromatin occupancy HDAC1 NF-&#65533;?&#65533;B- <br />           regulated promoters cIAP2 Bfl-1/A1. We observed cells lacking BRMS1 dramatic increase cell <br />           viability loss attachment extracellular matrix. Collectively, results suggest that <br />           BRMS1 suppresses metastasis ability function transcriptional corepressor antiapoptotic <br />           genes regulated NF-&#65533;?&#65533;B. <br />  <br />  <br />    Aberrant constitutive activation nuclear transcrip-                cytosolic I&#65533;?&#65533;B, results proteasomal degradation and <br /> tion factor NF-&#65533;?&#65533;B detected human malignant                     subsequent liberation NF-&#65533;?&#65533;B which, turn, translocates <br /> tumors (7, 14, 16, 46, 58, 61). We previously shown                   nucleus, enhances transcription (4, 21). In- <br /> NF-&#65533;?&#65533;B plays important role resistance non-small-                   tranuclear NF-&#65533;?&#65533;B activity regulated transcriptional co- <br /> cell lung cancer (NSCLC) chemotherapy histone                            regulators function connecting sequence-specific acti- <br /> deacetylase (HDAC) inhibition (30, 31, 41). Recently, im-                   vators basal transcriptional machinery, addition to <br /> portance cytokine-mediated activation IKK&#65533;?&#65533; subse-                     modifying chromatin intrinsic histone acetyl- <br /> quently NF-&#65533;?&#65533;B promotion progression inflam-                       transferase (HAT) HDAC activity (50). RelA/p65 binds to <br /> matory-associated epithelial cancers identified (23,                   CBP/p300 transcriptional coactivator, p/CAF, <br /> 32, 48). Furthermore, low levels cytokine tumor necrosis                 overexpression coactivators enhances trans- <br /> factor (TNF) produced tumor-associated macrophages                      activation potential NF-&#65533;?&#65533;B (47, 56). The transcriptional <br /> been suggested actually increase tumor growth <span id='am-95' about='obo:NCBITaxon_94025' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_27394 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_74974 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_74975 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-96' property="rdfs:label" content="Meta" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-99'  ></span>meta</span>-                      activity RelA/p65 modulated HDACs, <br /> static progression NF-&#65533;?&#65533;B-dependent signaling cas-                       including HDAC-1, -2, -3, SIRT1 (2, 10, 64). Post- <br /> cades (5, 32). Purported mechanisms NF-&#65533;?&#65533;B                         translational modifications pivotal chromatin-associ- <br /> promotes tumor progression metastatic movement include                      ated activities RelA/p65 acetylation RelA/p65 is <br /> the upregulation antiapoptotic gene products,                     important transcriptional activity DNA binding, as <br /> proangiogenic factors (1). Thus, increasing evidence                   impairing assembly I&#65533;?&#65533;B&#65533;?&#65533; nuclear exporta- <br /> that upregulation cell survival signals NF-&#65533;?&#65533;B                  tion (11, 33). <br /> the initial critical events occur metastatic                  In contrast prometastatic role NF-&#65533;?&#65533;B, breast cancer <br /> process.                                                                        metastasis suppressor 1 (BRMS1) gene mapped to <br />    Transcriptionally active NF-&#65533;?&#65533;B typically composed                    chromosome 11q13 originally identified metasta- <br /> heterodimeric protein complex, best studied                    sis suppressor gene (55). The BRMS1 protein product lo- <br /> most common form RelA/p65/p50 heterodimer,                         calized predominantly nucleus contains imperfect <br /> RelA/p65 contains transactivation domains. Classic activa-                  leucine zipper motif various coiled-coil domains (43, 55). <br /> tion NF-&#65533;?&#65533;B occurs IKK isoform phosphorylation                       BRMS1 mRNA expression shown markedly <br />                                                                                 reduced melanoma breast cancer cell lines, stable <br />                                                                                 overexpression BRMS1 cell lines significantly in- <br />    * Corresponding author. Mailing address: Department Surgery,              hibited metastatic potential (13, 54). Interestingly, there <br /> Box 800679, University Virginia, Charlottesville, VA 22908-0679.             evidence date BRMS1 regulates metastatic move- <br /> Phone: (434) 243-6443. Fax: (434) 982-1026. E-mail: djones@virginia             ment alterations matrix metalloproteinase levels, <br /> .edu. <br />    &#8224; Supplemental material article http://mcb <br />                                                                                 cellular adhesion profiles, alterations tumor cell invasion <br /> .asm.org/.                                                                      (52). Recent studies indicate BRMS1 selective com- <br />    &#17188; <br />      Published ahead print 25 September 2006.                             ponent mSin3a/HDAC corepressor complex resulting in <br />  <br />                                                                          8683 <br />  8684       LIU ET AL.                                                                                                                         MOL. CELL. BIOL. <br />  <br />  <br /> basal transcription repression measured GAL4 <span id='am-100' about='obo:NCBITaxon_117980' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6656 obo:NCBITaxon_6681 obo:NCBITaxon_6684 obo:NCBITaxon_111521 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6682 obo:NCBITaxon_88770 obo:NCBITaxon_111523 obo:NCBITaxon_6683 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-101' property="rdfs:label" content="Lucifer" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-103'  ></span><span id='am-104' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>lucifer</span>-                     Ac-H3(Lys9/Lys14), Ac-H4 (Lys8) (Cell Signaling Technology, Beverly, <br /> ase reporter myelomonocytic growth factor minimal                          MA); M2 Flag-epitope tag, &#65533;?&#65533;-tubulin, &#65533;?&#65533;-tubulin (Sigma Aldrich, St. Louis, <br />                                                                                   MO); HA-epitope tag (BD Biosciences, Palo Alto, CA). Acetyl-p65 (K310) <br /> promoter (42). BRMS1 shown negatively                            antibody kindly provided Marty W. Mayo (Charlottesville, VA). Recom- <br /> regulate nuclear translocation NF-&#65533;?&#65533;B urokinase-                        binant TNF purchased Sigma (St. Louis, MO). The TNT T7 Quick- <br /> type plasminogen activator transcripts processes in-                      Coupled transcription/translation obtained Promega Bio- <br /> volving inhibition I&#65533;?&#65533;B&#65533;?&#65533; phosphorylation (13).                                  sciences (San Luis Obispo, CA). <br />                                                                                      Total RNA isolation, quantitative reverse transcriptase PCR (RT-PCR), and <br />    Given BRMS1 functions metastasis suppressor and <br />                                                                                   NF-&#65533;?&#65533;B-regulated gene expression assays. Total RNA extracted using <br /> has associated known corepressor complex                          TRIzol (Invitrogen) according manufacturer&#8217;s protocol. In brief, 106 cells <br /> that NF-&#65533;?&#65533;B activation associated development                       100 mg tissue lysed 1 ml TRIzol, proteins were <br /> metastases, sought determine BRMS1                           separated chloroform. RNAs precipitated isopropanol, cDNAs <br /> regulating NF-&#65533;?&#65533;B transcriptional activity effects                      synthesized using Advantage RT PCR enzyme kit (Clontech, Palo <br />                                                                                   Alto, CA). BRMS1 expression determined real-time PCR iCycler <br /> RelA/p65 acetylation. Using NSCLC model system,                             IQ (Bio-Rad, Hercules, CA). The human BRMS1 primers TGCAGCGG <br /> demonstrate BRMS1 represses basal induc-                        AGCCTCAAG (forward) TCACATCCAGACAGAAGCCCT (reverse). <br /> ible transactivation potential RelA/p65 functioning                    Human HPRT gene amplified reference gene (38). Real-time PCR <br /> corepressor HDAC1 addition decreasing HDAC1                             quantification processed. The expression BRMS1 (&#9004;TC) normalized <br /> binding RelA/p65. Using site-directed mutagenesis se-                      endogenous reference gene (HPRT) (TCR). The &#9004;TC value calculated <br />                                                                                   subtracting TC value reference (TCR) TC value the <br /> lective small interfering RNA (siRNA) knockdown specific                       sample (TCS): &#9004;TC &#11005; TCS &#11002; TCR. The relative expression (2&#9004;&#9004;TC) calibrator <br /> HDACs, identified BRMS1/HDAC1 core-                              (placenta RNA; Clontech, Palo Alto, CA) determined subtracting the <br /> pressor complex reduces NF-&#65533;?&#65533;B-dependent transcription                     &#9004;TCC (calibrator, &#9004;TCC &#11005; TCCS &#11002; TCCR) &#9004;TC value (&#9004;&#9004;TC &#11005; <br /> deacetylation RelA/p65 lysine K310. Futhermore, siRNA                       &#9004;TCC (calibrator) &#11002; &#9004;TC). <br />                                                                                      For NF-&#65533;?&#65533;B-regulated gene expression assays, HEK 293T cells H1299 cells <br /> knockdown BRMS1 recruits endogenous acetyl-K310-RelA/ <br />                                                                                   plated 24 h transfection. For BRMS1 overexpression assays, BRMS1 <br /> p65 chromatin NF-&#65533;?&#65533;B-dependent antiapoptotic genes                       expression vector vector control transiently transfected by <br /> cIAP2 Bfl-1/A1 promoters simultaneously inhibiting                      using Polyfect reagent (QIAGEN, <span id='am-109' about='obo:NCBITaxon_195854' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_8087 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_28738 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_7898 obo:NCBITaxon_195853 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-110' property="rdfs:label" content="Valencia" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-112'  ></span><span id='am-113' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Valencia</span>, CA) according manufactur- <br /> endogenous HDAC-1 chromatin occupancy. Finally, using                          er&#8217;s instructions. For BRMS1 knockdown assay, siRNA BRMS1 siRNA con- <br /> anoikis cell model metastasis, demonstrate BRMS1                       trol (100 nM) transfected using Oligofectamine (Invitrogen, Carlsbad, <br />                                                                                   CA) according manufacturer&#8217;s instructions. The day transfection, an <br /> significantly increases apoptosis suspended NSCLC cells                        additional 100 nM siRNA transfected H1299 cells described above. <br /> after cytokine stimulation.                                                       At 36 h transfection, TNF (10 ng/ml) added cells an <br />    Collectively, results indicate BRMS1 functions                   additional 12 h. Total RNA extracted using TRIzol, followed real-time <br /> a corepressor modulates NF-&#65533;?&#65533;B-dependent antiapoptotic                        RT-PCR described above. The PCR primers used follows: cIAP-2, the <br />                                                                                   5&#11032;-GCTGTGATGGTGGACTCAGG-3&#11032; (forward) 5&#11032;-TGGCTTGAACTT <br /> transcription chromatin level. These observations sug- <br />                                                                                   GACGGATG-3&#11032; (reverse); Bfl-1/A1, 5&#11032;-TCATATTTTGTTGCGGAGTTCA-3&#11032; <br /> gest BRMS1 expression prevent metastases                          (forward) 5&#11032;-TCCAGCCAGATTTAGGTTCAAA-3&#11032; (reverse); Bcl-xL, <br /> ability corepressor regulate NF-&#65533;?&#65533;B transcription                   5&#11032;-GCATTGTGGCCTTTTTCTCC-3&#11032; (forward) 5&#11032;-GCTGCTGCATTGTT <br /> cell survival loss cellular adhesion.                                CCCATA-3&#11032; (reverse); VCAM-1, 5&#11032;-GCTGCTCAGATTGGAGACTCA-3&#11032; <br />                                                                                   (forward) 5&#11032;-CGCTCAGAGGGCTGTCTATC-3&#11032; (reverse); ICAM-1, 5&#11032;-C <br />                                                                                   TGTGTCCCCCTCAAAAGTC-3&#11032; (forward) 5&#11032;-GGGGTCTCTATGCCCA <br />                        MATERIALS AND METHODS                                      ACAA-3&#11032; (reverse); E-Selectin, 5&#11032;-TGAAGCTCCCACTGAGTCCAA-3&#11032; (for- <br />    Cell culture, surgical specimens, reagents, plasmid constructs. Human      ward) 5&#11032;-GGTGCTAATGTCAGGAGGGAGA-3&#11032; (reverse); GAPDH, <br /> NSCLC lines (NCI-H157, NCI-H358, NCI-H460, NCI-A549, NCI-H1299),            5&#11032;-CTACACCTTGCCTGTGAGCA-3&#11032; (forward) 5&#11032;-GACACGTGTGGCC <br /> normal human bronchial epithelial cell line (NL-20), tumorigenic non-     ATTGTAG-3&#11032; (reverse). <br /> metastatic human embryonic kidney cells (HEK 293T) (63) obtained           Transfection luciferase reporter assays. <span id='am-125' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-126' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>Plasmids</span> reporters were <br /> the American Type Culture Collection (Manassas, VA). NSCLC cell lines         transiently cotransfected cells 40 60% confluence using Polyfect <br /> HEK 293T cells grown described previously (64). NL20 cells           reagent described above. Luciferase reporter activity assays performed as <br /> cultured Ham F-12 medium (Invitrogen, Carlsbad, CA) supplements           previously described (41). &#65533;?&#65533;-Galactosidase activities analyzed control for <br /> according manufacturer&#8217;s protocol. Human NSCLC specimens adja-         efficiency transfection. Luminescence normalized protein concen- <br /> cent noncancerous lung preserved according standard surgical resection    trations, transfection <span id='am-120' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-121' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-122'  ></span><span id='am-123' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-124' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> means &#11006; standard deviations (SD) of <br /> from patients Division Thoracic Surgery, University Virginia,   independent experiments performed triplicate analyses. <br /> with informed consent Human Investigations Committee approval. The               Immunoprecipitation Western blot. HEK 293T cells transfected with <br /> 3x-&#65533;?&#65533;B luciferase reporter (3x-&#65533;?&#65533;B-Luc), Gal-4 luciferase construct (Gal4-Luc),     expression <span id='am-105' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-106' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-107' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>vectors</span> encoding Flag-tagged p65, HA-tagged BRMS1, both. Cells <br /> expression vectors encoding Gal4-p65 fusion protein (1-286, 286-520, 286-551,     harvested 48 h transfection. In select experiments, HEK 293T cells <br /> 520-551, 1-551, 286-551/K310R), expression vectors (pGEX) encoding GST-       transfected siRNA BRMS1 control. For immunoprecipitation as- <br /> p65 fusion proteins (1-305, 245-455, 354-551), plasmids (pCMV) encod-     says, primary antibody (2 &#65533;?&#65533;g anti-HA-epitope tag, 3.4 &#65533;?&#65533;g anti-Flag-epitope <br /> ing Flag-tagged p65 previously described (41, 51, 64). Human BRMS1           tag, 3.1 &#65533;?&#65533;g anti-&#65533;?&#65533;-tubulin, 4 &#65533;?&#65533;g anti-myc, 4 &#65533;?&#65533;g anti-p65, 4 &#65533;?&#65533;g of <br /> cDNA cloned PCR (the primers 5&#11032;-GTATGAATTCGACCTGTC                    anti-BRMS1, 4 &#65533;?&#65533;g anti-HDAC1, 4 &#65533;?&#65533;g anti-HDAC3) mixed with <br /> CAGCCTCCAAGC-3&#11032; [forward] 5&#11032;-GTATCTCGAGTCAAGGTCCATCC                          precleared cell lysates hour 4&#176;C addition 30 &#65533;?&#65533;l protein <br /> GATTTTC-3&#11032; [reverse]; restriction sites underlined) inserted     agarose A/G (Santa Cruz, Santa Cruz, CA), reactions tumbled over- <br /> hemagglutinin (HA)-tagged pCMV vector (Clontech, Palo Alto, CA)               night 4&#176;C. The agarose beads extensively washed, eluted protein <br /> pcDNA3.1(&#11001;) vectors (Invitrogen, Carlsbad, CA) using restriction enzymes      resolved sodium dodecyl sulfate-polyacrylamide gel electrophoresis <br /> EcoRI XhoI (New England Biolabs, Beverly, MA). Human HDAC1                (SDS-PAGE), followed immunoblot analysis. All Western blots per- <br /> cloned PCR (the HDAC1 primers 5&#11032;-CGGAATTCACGATGGCGCA                      formed according standard procedures. Proteins separated SDS&#8211; <br /> GACGCAGGGCAC-3&#11032; [forward] 5&#11032;-CGGAATTCGGCCAACTTGACCT                           12% PAGE. After transfer, nitrocellulose membranes (Bio-Rad, Hercules, <br /> CCTCCTTG-3&#11032; [reverse]) inserted pcDNA3.1(&#11001;) vectors using EcoRI          CA) probed primary antibodies 1:1,000 dilution secondary <br /> sites. siRNA SMART pool human BRMS1, HDAC1, HDAC3, siCONTROL                  antibodies (Promega, Madison, WI) 1:5,000 dilution blocking solution. <br /> nontargeting siRNA purchased Dharmacon (Chicago, IL). The anti-         Proteins visualized enhanced chemiluminescent substrate (Pierce <br /> bodies used present study follows: BRMS1 (Abnova Corp., Tai-       Biotechnology, Rockford, IL). <br /> wan); RelA/p65, mSin3A, p300, myc, normal rabbit immunoglobulin G (IgG;          GST binding assays. Glutathione S-transferase (GST)&#8211;p65 fusion proteins <br /> Santa Cruz Biotechnology, Santa Cruz, CA); &#65533;?&#65533;-acetyl-lysine, HDAC1, HDAC3,         expressed <span id='am-72' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-73' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-86'  ></span><span id='am-87' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span>Escherichia coli</span> BL21 (Amersham, Piscataway, NJ) puri- <br />  VOL. 26, 2006                                                                                                                BRMS1 AND NF-&#65533;?&#65533;B            8685 <br />  <br /> fied described previously (18). The relative abundance fusion protein to <br /> GST evaluated using SDS&#8211;10% PAGE gel stained Coomassie <br /> blue. pcDNA3.1(&#11001;)-BRMS1 expression vector translated TNT <br /> reticulocyte binding 35S-labeled methionine (Amersham, Piscataway, <br /> NJ) according manufacturer&#8217;s protocol. Labeled BRMS1 rocked for <br /> 2 h room temperature GST-p65 fusion proteins total volume 500 <br /> &#65533;?&#65533;l NETN buffer (20 mM Tris-HCl [pH 7.5], 100 mM KCl, 0.7 mM EDTA, <br /> 0.05% Nonidet P-40, 1 mM phenylmethylsulfonyl fluoride). The reactions <br /> were extensively washed NETN buffer, bound proteins resolved <br /> on 15% denaturing SDS-PAGE gels analysis autoradiography. <br />    RelA/p65 deacetylation assays. RelA/p65 deacetylation assays performed <br /> as previously described (11). HEK 293T cells transfected expression <br /> vector encoding HA-tagged BRMS1 vector. In select experiments, cells <br /> were cotransfected siRNA HDAC1, HDAC3, siRNA control de- <br /> scribed above. After treatment TNF (0, 20, 40 ng/ml) 1 h, cell lysates <br /> were resolved SDS-PAGE gels immunoblotted specific acetyl- <br /> p65(K310) antibody. For vitro deacetylation assays, HEK 293T cells were <br /> cotransfected expression vectors encoding Flag-tagged p65 without <br /> p300. Cell lysates immunoprecipitated 4 &#65533;?&#65533;g Flag antibody. RelA/ <br /> p65 deacetylation assays performed incubating 20 &#65533;?&#65533;l immunoprecipi- <br /> tated Flag-RelA/p65 protein 10 &#65533;?&#65533;l vitro-translated BRMS1 protein <br /> and/or increasing amounts HDAC1 protein using TNT reticulocyte <br /> system HDAC buffer (10 mM Tris-HCl [pH 8.0], 10 mM NaCl, 10% glycerol) <br /> for 2 h room temperature. The reactions washed times HDAC <br /> buffer analyzed immunoblotting <span id='am-51' about='obo:NCBITaxon_9596' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-52' property="oboInOwl:hasAlternativeId" content="NCBITaxon:445518" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="Pan" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasAlternativeId" content="NCBITaxon:317028" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-57'  ></span><span id='am-58' property="oboInOwl:hasExactSynonym" content="chimpanzees" datatype="xsd:string"></span>pan</span>-acetyl-lysine antibody. <br />    Caspase-3, DNA fragmentation, cell death assays. HEK 293T cells or <br /> H1299 cells transiently transfected pCMV-BRMS1 expression vector <br /> or vector control. Cells left untreated treated TNF (50 <br /> ng/ml) 24 h transfection. At 48 h posttransfection, apoptosis quantified <br /> by quantitation nucleosomes released cytoplasm using cell death <br /> detection ELISA Plus kit (Roche Applied Science) according manufac- <br /> turer&#8217;s directions. Caspase-3 activity determined addition APC- <br /> DEVD protein conjugate (Calbiochem, San Diego, CA) cellular extracts <br /> containing 30 &#65533;?&#65533;g protein. The fluorescence caspase-3-cleaved protein con- <br /> jugates measured. For cell death assays, BRMS1 overexpressed by <br /> transfecting pCMV-BRMS1 expression vector vector control or <br /> suppressed transfecting siRNA BRMS1 siRNA control HEK 293T and <br /> H1299 cells described above. At 48 h transfection, cells suspended, <br /> respectively, 0, 0.5, 1, 2, 3 h. Cells replated, percentage <br /> of cell death obtained enumerating trypan blue-positive cells versus <br /> the total number cells additional 24 h incubation. <br />    ChIP. For overexpression experiments, HEK 293T cells transfected with <br /> pCMV-BRMS1 vector. For RNA interference experiments, HEK 293T <br /> cells transfected siRNA BRMS1 siRNA control described above. <br /> After 36 h, cells left untreated treated TNF (10 ng/ml) for <br /> an additional 12 h. Chromatin immunoprecipitation (ChIP) assays per- <br /> formed described previously (34). DNA immunoprecipitated 4 &#65533;?&#65533;l of <br /> antibody (anti-BRMS1, anti-p65, anti-p50, anti-mSin3A, anti-acetyl-p65 K310,              FIG. 1. BRMS1 expression reduced NSCLC cells tumor <br /> anti-HDAC1, anti-HDAC3, Ac-H3, Ac-H4, normal rabbit IgG) purified.              tissues. (A) BRMS1 mRNA protein expression tumorigenic but <br /> The regions human cIAP-2 Bfl-1/A1 promoters containing &#65533;?&#65533;B binding           nonmetastatic cell lines (NL20 HEK 293T) NSCLC cell lines <br /> sites targeted amplification described previously (38). PCR data      (H358, H1299, H157, A549, H460). mRNA levels evaluated <br /> analyzed previously described (8).                                                  quantitative RT-PCR calculated ratio human pla- <br />    Statistical analysis. The results experiments represent means &#11006;      centa control (top). Protein levels BRMS1 detected West- <br /> SD separate experiments performed triplicate unless noted.       ern blotting &#65533;?&#65533;-tubulin loading control (bottom). Real-time <br /> Statistical differences control BRMS1 groups determined          RT-PCR data represent mean &#11006; SD experiments <br /> two-tailed, unpaired Student t test appropriate. P values &#65533;?0.05           performed triplicate. (B) BRMS1 mRNA protein expression in <br /> considered significant.                                                                tumor tissues patient-matched adjacent lung tissues. mRNA and <br />                                                                                        protein levels patient-matched samples determined in <br />                                                                                        panel A. The pathological stage presented using international <br />                                   RESULTS                                              staging lung cancer (44). Stage <span id='am-11' about='obo:NCBITaxon_360966' typeof='owl:Thing obo:NCBITaxon_9397 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_9431 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_30560 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-12' property="rdfs:label" content="Ia" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15'  ></span>IA</span> NSCLC limited local <br />                                                                                        disease metastasis lymph nodes. Stage IIB NSCLC includes <br />    BRMS1 expression decreased NSCLC. NSCLC                                  spread local lymph nodes. Stage IIIB disease includes metastasis to <br /> highly malignant cancer proclivity metastasizing                         regional lymph nodes outside lung itself. <br /> both lymph nodes distant organs. Since BRMS1 sup- <br /> pressor metastasis, suggests aggressiveness of <br /> NSCLC related intratumoral BRMS1 levels. To                              NSCLC cell lines examined. Parallel results shown in <br /> test hypothesis, BRMS1 mRNA protein expression                                BRMS1 protein expression (Fig. 1A). <br /> levels determined different human NSCLC cell                                To determine expression pattern also <br /> lines real-time RT-PCR immunoblotting. Compared                              present surgically resected human NSCLC adjacent <br /> the immortalized bronchial epithelial NL20 HEK293T                                 noncancerous lung tissue, BRMS1 mRNA protein levels <br /> cells, BRMS1 mRNA levels markedly decreased                                 detected using techniques. As shown Fig. <br />  8686      LIU ET AL.                                                                                                              MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 2. BRMS1 inhibits constitutive TNF-induced NF-&#65533;?&#65533;B transcriptional activity NF-&#65533;?&#65533;B-dependent gene expression. (A) 293T and <br /> H1299 cells transiently cotransfected NF-&#65533;?&#65533;B-responsive reporter (3x-&#65533;?&#65533;B-Luc) mutant 3x-&#65533;?&#65533;B-Luc reporter expression vector <br /> encoding BRMS1. At 18 h transfection cells treated treated TNF (10 ng/ml) 12 h. The luciferase activity was <br /> determined. All transfections normalized <span id='am-1' about='obo:NCBITaxon_10358' typeof='owl:Thing obo:NCBITaxon_10357 obo:NCBITaxon_1 obo:NCBITaxon_548681 obo:NCBITaxon_35237 obo:NCBITaxon_10239 obo:NCBITaxon_10292'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Cytomegalovirus" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5'  ></span>cytomegalovirus</span> (CMV)&#8211;&#65533;?&#65533;-galactosidase activity. (B) 293T cells H1299 cells were <br /> transiently cotransfected Gal4-Luc reporter expression vector encoding fusion protein Gal-4-p65 BRMS1. At 18 h <br /> after transfection cells treated treated TNF (10 ng/ml) 12 h. The luciferase activity determined. All transfections were <br /> normalized CMV&#8211;&#65533;?&#65533;-galactosidase activity. (C) 293T cells transiently transfected BRMS1 CMV-driven vector (Control). <br /> Cells treated treated TNF (10 ng/ml) 12 h. Total RNA extracted, NF-&#65533;?&#65533;B regulated antiapoptotic gene mRNA <br /> levels analyzed quantitative RT-PCR. The data plotted fold control, results cells transfected control <br /> vector normalized 1 gene. ##, P &#65533;? 0.01 compared control; &#11569;&#11569;, P &#65533;? 0.01 compared control &#11001; TNF. (D) 293T cells were <br />  VOL. 26, 2006                                                                                                  BRMS1 AND NF-&#65533;?&#65533;B            8687 <br />  <br />  <br /> 1B, expression BRMS1 mRNA protein decreased                    BRMS1 coimmunoprecipitates RelA/p65 p50 <br /> in nearly tumor tissues compared adjacent non-              subunits NF-&#65533;?&#65533;B. We determined BRMS1 <br /> cancerous tissues. Interestingly, reduction                coimmunoprecipitate components NF-&#65533;?&#65533;B that <br /> BRMS1 protein mRNA levels patient (patient                 known antiapoptotic function, RelA/p65 p50. <br /> 4) early-stage disease. Therefore, BRMS1 expression               Exogenous RelA/p65 BRMS1 reciprocally immuno- <br /> both NSCLC cell lines tumor tissues markedly decreased             precipitated overexpression HA-tagged BRMS1 <br /> compared noncancerous cells tissues, suggesting               Flag-tagged RelA/p65 (Fig. 3A). The negative control, <br /> the development progression NSCLC involve                  &#65533;?&#65533;-tubulin monoclonal antibody, myc polyclonal antibody <br /> loss BRMS1 expression.                                                 failed detect immunoprecipitation protein complexes, <br />    BRMS1 represses transactivation potential RelA/p65.             confirming specificity anti-Flag anti-HA anti- <br /> Previous studies indicated nuclear translocation            bodies, respectively. Although coimmunoprecipitation of <br /> and DNA-binding activity NF-&#65533;?&#65533;B suppressed breast                 BRMS1 RelA/p65 occurred overexpression as- <br /> cancer MDA-MB-231 melanoma C8161.9 cell lines                    says, wanted determine proteins <br /> stably overexpress BRMS1 (13). To determine                       interact similar fashion endogenously. As shown in <br /> BRMS1 abolish NF-&#65533;?&#65533;B transcriptional activity                 reciprocal coimmunoprecipitation assays Fig. 3B, endog- <br /> system, transient-transfection 3x-&#65533;?&#65533;B luciferase assays per-          enous BRMS1 nuclear extracts does coimmunoprecipitate <br /> formed treatment TNF. We chose TNF stimulus               endogenous RelA/p65 p50. <span id='am-59' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-62'  ></span><span id='am-63' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> association <br /> based ability enhance metastatic progression epi-           BRMS1 nuclear RelA/p65 p50 subunits <br /> thelial cancers NF-&#65533;?&#65533;B-dependent manner (32). As shown               NF-&#65533;?&#65533;B implies BRMS1 affect NF-&#65533;?&#65533;B transcription <br /> in Fig. 2A, BRMS1 dramatically diminished basal TNF-                  directly interacting RelA/p65-p50 heterodimer. <br /> induced NF-&#65533;?&#65533;B transcriptional activity cell lines. Fail-             BRMS1 directly binds RelA/p65 subunit NF-&#65533;?&#65533;B. <br /> ure BRMS1 affect transcription cells expressing                RelA/p65 contains N-terminal Rel homology domain <br /> mutant 3x-&#65533;?&#65533;B-Luc reporter gene confirms NF-&#65533;?&#65533;B specificity.                (RHD) consisting approximately 286 amino acids which <br /> Given BRMS1 shown affect NF-&#65533;?&#65533;B                   mediates DNA binding dimerization (22). Three transac- <br /> decreased cytosolic I&#65533;?&#65533;B processing (13), sought specifi-            tivation domains present C terminus region RelA/ <br /> cally examine ability BRMS1 regulate transacti-             p65, regulate interaction NF-&#65533;?&#65533;B various <br /> vation potential RelA/p65 only. To accomplish this, Gal4/              basal transcriptional components associates with <br /> p65-Luc reporter gene assays performed (Fig. 2B). These              p300/CBP transcriptional coactivators (3, 20, 22, 47, 56). To <br /> demonstrated BRMS1 significantly represses transac-              characterize protein binding domains between <br /> tivation potential RelA/p65 TNF stimulation.                     BRMS1 RelA/p65, performed GST pull-down assays. <br />    In <span id='am-69' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> determine BRMS1 decrease                GST truncated GST-RelA/p65 fusion proteins (Fig. 3C) <br /> the expression classic NF-&#65533;?&#65533;B-dependent antiapoptotic genes             used examine interaction BRMS1 in <br /> (cIAP-2, Bfl-1/A1, BclxL) (9, 60, 67), 293T cells trans-         vitro rabbit reticulocyte transcription/translation system. As <br /> fected expression vector encoding BRMS1 quantita-                shown Fig. 3D, BRMS1 strongly bind GST- <br /> tive RT-PCR performed (Fig. 2C). BRMS1 significantly                  RelA/p65(245-455) fusion protein modestly interact <br /> decreased TNF-induced cIAP-2, Bfl-1/A1, Bcl-xL tran-                  GST-RelA/p65(1-305) fusion protein GST <br /> scripts. To determine BRMS1 decreased                  GST-RelA/p65(354-551) construct. This sug- <br /> NF-&#65533;?&#65533;B-regulated genes known involved metastasis,                 gests region RelA/p65 directly interacts with <br /> RT-PCR experiments repeated intracellular ad-                BRMS1 primarily located amino acids 245 354 <br /> hesion proteins ICAM, VCAM, E-selectin (36, 37, 59). As               RelA/p65. This region corresponds C-terminal end of <br /> shown Fig. 2D, significant effect BRMS1                RHD does include transactivation domains of <br /> overexpression transcripts, effect          RelA/p65. <br /> non-NF-&#65533;?&#65533;B regulated gene GAPDH. To confirm regula-                      To determine correlation the <br /> tion NF-&#65533;?&#65533;B-regulated antiapoptotic genes specific                BRMS1-binding regions RelA/p65 protein the <br /> BRMS1, selective silencing BRMS1 expression using siRNA                transactivation potential RelA/p65, Gal4/p65 constructs <br /> resulted robust rescue of antiapoptotic genes           ectopically expressed 293T system, Gal4/ <br /> (Fig. 2E). These results indicate BRMS1 inhibits                 p65 luciferase activity determined presence ab- <br /> transactivation potential RelA/p65 establishes           sence BRMS1. Maximal BRMS1-mediated repression of <br /> evidence BRMS1 modulate cancer cell survival                     RelA/p65 transactivation occurred cells overexpressing the <br /> through transcriptional repression NF-&#65533;?&#65533;B-regulated anti-               Gal4-RelA/p65(286-551) Gal4-RelA/p65(286-520) pro- <br /> apoptotic genes.                                                          teins (Fig. 3E). BRMS1 little ability repress <br />  <br />  <br />  <br />  <br /> transiently transfected BRMS1- CMV-driven vector (control). Cells treated treated TNF (10 ng/ml) 12 h. Total <br /> RNA extracted, NF-&#65533;?&#65533;B regulated gene housekeeping gene (GAPDH) mRNA levels analyzed quantitative RT-PCR. The data <br /> were analyzed described above. No statistical difference control BRMS1 groups. (E) 293T cells H1299 cells were <br /> transiently transfected siRNA BRMS1 siRNA control. Cells treated treated TNF (10 ng/ml) 12 h. The total RNA was <br /> extracted, NF-&#65533;?&#65533;B regulated antiapoptotic gene mRNA levels analyzed quantitative RT-PCR. The data plotted described for <br /> panel C. Western blots whole-cell lysates effective knockdown siRNA &#65533;?&#65533;-tubulin loading control. <br />  8688     LIU ET AL.                                                                                                         MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 3. BRMS1 colocalizes RelA/p65 p50 subunits directly interacts RelA/p65 subunit NF-&#65533;?&#65533;B. (A) 293T cells were <br /> cotransfected expression vector encoding HA-tagged BRMS1, Flag-tagged RelA/p65, both. Cell lysates immunoprecipitated Flag <br /> polyclonal antibody HA monoclonal antibody, followed immunoblotting presence BRMS1 RelA/p65, respectively. Input protein <br /> showed relatively equal amounts HA-tagged BRMS1 Flag-tagged RelA/p65 expressed immunoprecipitated. The cell <br /> lysates immunoprecipitated &#65533;?&#65533;-tubulin monoclonal antibody myc polyclonal antibody, respectively, controls nonspecific <br /> interference antibody. (B) 293T cell isolated nuclear lysates coimmunoprecipitated negative control (IgG) antibody BRMS1, <br /> RelA/p65, p50, followed immunoblotting presence endogenous RelA/p65, p50, BRMS1. Input protein showed relatively <br /> equal amounts endogenous, RelA/p65, p50, BRMS1 expressed immunoprecipitated. (C) Schematic illustration RelA/p65 <br /> proteins. The 1-551 construct encodes full-length Rel/p65 protein. RHD, Rel homology domain (u); TAD, transactivation domains (o). (D) In <br /> vitro-translated [35S]methionine-labeled BRMS1 incubated GST-RelA/p65 fusion proteins (1-305, 245-455, 354-551) GST alone <br /> (negative control). Proteins retained GST GST-RelA/p65 fusion proteins eluted analyzed SDS-PAGE autoradiography <br /> (n &#11005; 3) (Top). GST-RelA/p65 fusion proteins separated SDS&#8211;15% PAGE gel stained Coomassie blue indicating relative <br /> amounts GST fusion protein. (E) 293T cells transiently cotransfected Gal4-Luc reporter expression vector encoding <br /> Gal4-RelA/p65 fusion proteins (1-286, 286-520, 286-551, 520-551, 1-551). In addition, cells transfected expression vector <br /> encoding BRMS1 vector (control). All transfections normalized CMV&#8211;&#65533;?&#65533;-galactosidase activity. The fold repression was <br /> calculated basal transcription values Gal4-RelA/p65 vector normalized 1. Western blot analysis confirms the <br /> expression various Gal4-p65 fusion proteins. <br />  <br />  <br />  <br />  <br /> RelA/p65 transactivation N-terminus C-terminus                   BRMS1 deacetylates RelA/p65 lysine residue 310. The <br /> regions protein. Collectively, data suggest            transactivation potential RelA/p65 regulated part <br /> BRMS1-mediated repression transactivation potential               reversible acetylation (11, 64). The RelA/p65 pro- <br /> of RelA/p65 result direct binding            tein known acetylated lysine (K) residues, 122, <br /> two proteins region located amino acids               123, 218, 221, 310. K310 known important <br /> 245 354 RelA/p65.                                                 transactivation function RelA/p65 (33, 64). <br />  VOL. 26, 2006                                                                                                         BRMS1 AND NF-&#65533;?&#65533;B             8689 <br />  <br />  <br />  <br />  <br />   FIG. 4. BRMS1 abolishes TNF-induced NF-&#65533;?&#65533;B acetylation lysine 310. (A) 293T cells cotransfected expression plasmid encoding <br /> HA-tagged BRMS1 vector control. At 48 h posttransfection, cells treated TNF 1 h indicated concentrations. Western <br /> blots whole-cell lysates demonstrate acetylation lysine 310 p65 using specific acetyl-p65(K310) antibody relatively constant total p65. <br /> Other blots confirmed expression HA-BRMS1 &#65533;?&#65533;-tubulin loading control. (B) 293T H1299 cells transiently transfected <br /> with Gal4-Luc reporter plasmid encoding Gal4-p65 fusion protein (286-551) mutant Gal4-p65 fusion protein (286-551, K310R). In <br /> addition, plasmid encoding BRMS1 vector (control) cotransfected indicated. At 18 h transfection, cells treated or <br /> not treated TNF (10 ng/ml) 12 h. The luciferase activity determined. All transfections normalized CMV&#8211;&#65533;?&#65533;-galactosidase <br /> activity. The fold repression calculated basal transcription values Gal4-RelA/p65 vector K310R mutant vector were <br /> normalized 1. Western blot analysis confirmed expression various Gal4-p65 fusion proteins. <br />  <br />  <br />  <br />  <br /> Based data point, hypothesized                       BRMS1 functions corepressor RelA/p65 deacetyla- <br /> BRMS1 repress transactivation potential RelA/p65                   tion. To determine <span id='am-48' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-49' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> HDACs primarily respon- <br /> by modulation acetylation state. Using anti-acetyl                 sible BRMS1-mediated deacetylation RelA/p65, 3x-&#65533;?&#65533;B <br /> RelA/p65 K310 antibody, deacetylation assays performed                   luciferase assays repeated presence HDAC <br /> which endogenous RelA/p65 acetylated K310                     inhibitors. Trichostatin A (TSA) typical inhibitor class I <br /> in dose-dependent manner stimulation TNF. Impor-                   II HDACs class III (40). Nicotinamide (NAM) is <br /> tantly, addition BRMS1 effectively deacetylated RelA/                  specific negative regulator class III HDACs (6). TSA signif- <br /> p65 K310 despite escalating doses TNF (Fig. 4A).                        icantly rescued BRMS1 repression basal TNF-induced <br />    We investigated deacetylation RelA/p65                     NF-&#65533;?&#65533;B-dependent transcription, NAM did (Fig. 5A). <br /> K310 important target BRMS1-mediated repression                     Thus, inhibition class I/II HDACs, class III, is <br /> of RelA/p65 activity. As shown Fig. 4B, ectopic expression              involved BRMS1-mediated deacetylation RelA/p65. <br /> BRMS1 dramatically reduced basal TNF-induced                           Previous studies reported BRMS1 forms com- <br /> RelA/p65 transactivation potential, supporting prior ex-                  plexes retinoblastoma-binding protein 1 mSin3A <br /> periments (Fig. 2A B). In addition, overexpression                   histone deacetylase complex (42), corepressor <br /> site-directed mutant Gal4-p65 (K310R) fusion protein                   function mSin3A derived associations HDAC1 and <br /> resulted ca. 50% repression RelA/p65 transactivation                -2 (24, 35, 45). In addition, HDAC3 known deacetylate <br /> potential compared wild-type constructs, finding             RelA/p65 (10, 33) interactions corepres- <br /> in agreement prior reports (11). Importantly, transient                  sors NCoR SMRT (25, 26). Coimmunoprecipitation assays <br /> transfection BRMS1 failed repress basal                    performed identify HDAC interacted with <br /> the TNF-induced transactivation potential RelA/p65 af-                     BRMS1. We HDAC1, HDAC3, associated <br /> ter loss lysine K310 (Fig. 4B). These data strongly                    endogenous BRMS1 (Fig. 5B). Understanding that <br /> suggest deacetylation K310 RelA/p65 required                    BRMS1 corepressor function, sought deter- <br /> for BRMS1-mediated repression transactivation po-                      BRMS1 affect binding HDAC1 to <br /> tential RelA/p65.                                                          RelA/p65. As shown Fig. 5C, siRNA BRMS1 dramatically <br />  8690     LIU ET AL.                                                                                                           MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 5. HDAC1 required BRMS1-mediated RelA/p65 deacetylation. (A) 293T H1299 cells transiently cotransfected with <br /> 3x-&#65533;?&#65533;B-Luc reporter plasmid encoding BRMS1 vector control. At 18 h transfection, cells treated treated TSA <br /> (500 nM) NAM (500 &#65533;?&#65533;M) additional 12 h. Cells simultaneously treated treated TNF (10 ng/ml). The luciferase activity <br /> was determined. All transfections normalized CMV&#8211;&#65533;?&#65533;-galactosidase activity. The data represent typical experiment performed in <br /> duplicate, means &#11006; SD. (B) 293T whole-cell lysates coimmunoprecipitated control (IgG), anti-HDAC1, anti-HDAC3 <br /> antibody, followed immunoblotting presence endogenous BRMS1. Input protein showed relatively equal amounts endogenous <br /> BRMS1, HDAC1, HDAC3 expressed immunoprecipitated. (C) 293T cells transfected siRNA BRMS1 siRNA control. Cell <br /> lysates coimmunoprecipitated anti-p65, presence endogenous HDAC1 probed Western blotting. (D) 293T cells were <br /> transfected expression vector encoding Flag-tagged RelA/p65 HA-tagged p300. Cell lysates immunoprecipitated with <br /> anti-Flag antibody. On left, pellets incubated rabbit reticulocyte (lane 3), vitro-translated HDAC1 (10 &#65533;?&#65533;l) alone, or <br />  VOL. 26, 2006                                                                                                       BRMS1 AND NF-&#65533;?&#65533;B             8691 <br />  <br />  <br /> decreases binding HDAC1 RelA/p65, suggesting                      NSCLC cells. The H1299 cells chosen have <br /> BRMS1 required process.                                          reduced BRMS1 expression relative 293T cells. There <br />    Although demonstrated BRMS1 abolishes                        putative NF-&#65533;?&#65533;B binding sites identified cIAP-2 <br /> acetylation RelA/p65 K310, unclear                  promoter positions &#11002;147, &#11002;197, &#11002;210 putative <br /> deacetylation histone deacetylases known associate              NF-&#65533;?&#65533;B binding element Bfl-1/A1 promoter position &#11002;823 <br /> with BRMS1 or, alternatively, BRMS1 itself in-                   (15, 27). As noted previously, unrelated downstream sequences <br /> trinsic deacetylase activity. To experimentally address                 genes amplified control (see Fig. S1 in <br /> question, RelA/p65 acetylated using p300 vivo               supplemental material). <br /> then incubated vitro translated BRMS1, increasing                       As shown Fig. 6A B, RNA interference inhibition of <br /> doses HDAC1, both. The addition BRMS1 protein                       BRMS1 dramatically increases TNF-mediated endogenous <br /> alone unable deacetylate RelA/p65. HDAC1 pro-                   acetylation RelA/p65 (K310) chromatin demonstrated <br /> duced modest deacetylation RelA/p65,                      using specific anti-K310 antibody. In addition, selective <br /> recombinant proteins added, significant deacetylation                silencing BRMS1 concomitantly decreases chromatin occu- <br /> RelA/p65 observed (Fig. 5D). To confirm requirement                  pancy HDAC1 affecting HDAC3 loading. This oc- <br /> for BRMS1 deacetylating RelA/p65, vitro deacety-                 curs cIAP2 Bfl-1/A1 promoters. In con- <br /> lation assays repeated increasing doses HDAC1                   trast, ectopic expression BRMS1 resulted enhanced <br /> alone BRMS1, modest deacetylation                        recruitment HDAC1 promoters significant <br /> observed (Fig. 5D). These results indicate BRMS1                  reduction acetylated RelA/p65 (Fig. 6C). Interestingly, <br /> does possess intrinsic HDAC activity; however, ex-                  RelA/p65 chromatin loading increased ectopic ex- <br /> pressed HDAC1, effectively deacetylate                    pression silencing BRMS1, suggesting BRMS1 does <br /> RelA/p65.                                                                    govern basal inducible loading RelA/p65 chroma- <br />    In order confirm ability BRMS1 deacetylate              tin. We corepressor, BRMS1 function <br /> RelA/p65 binding HDAC1, used RNA                          global regulator chromatin structure, evidenced <br /> interference separately knockdown HDAC1 HDAC3 ex-                      ability decrease promoter occupancy Ac-H3 and <br /> pression presence BRMS1. As shown Fig. 5E,                      Ac-H4 cIAP2 Bfl-1/A1 promoters. These <br /> selective HDAC1 knockdown presence BRMS1 re-                       ChIP findings cell type specific similar results <br /> sulted partial rescue endogenous RelA/p65 acetylation                observed H1299 NSCLC cells (see Fig. S2 the <br /> (compare lanes 10 11), HDAC3 knockdown did                       supplemental material). <br /> (compare lanes 10 12). In similar experimental design,                    These data provide compelling evidence BRMS1 a <br /> the transcriptional activity NF-&#65533;?&#65533;B partially rescued            required corepressor regulates NF-&#65533;?&#65533;B transcription by <br /> the loss HDAC1 presence BRMS1 overex-                      modulating acetylation status RelA/p65 HDAC1- <br /> pression demonstrable rescue following HDAC3                         dependent manner. <br /> knockdown (Fig. 5F). Interestingly, deacetylation (Fig.                 Modulation BRMS1 expression sensitizes cells TNF- <br /> 5E) NF-&#65533;?&#65533;B transcriptional activity assays (Fig. 5F), knock-              induced apoptosis. Understanding importance TNF- <br /> down HDAC1 unable completely abolish BRMS1-                        mediated tumor growth (32, 48) observation that <br /> mediated RelA/p65 deacetylation repression NF-&#65533;?&#65533;B tran-                 ectopic expression BRMS1 inhibits NF-&#65533;?&#65533;B-dependent anti- <br /> scriptional activity, suggesting HDACs                  apoptotic gene expression, hypothesized BRMS1 <br /> that tether BRMS1. Despite results, data presented                 sensitize cancer cells apoptotic cell death. To exper- <br /> indicate BRMS1/HDAC1 complex functions regu-                     imentally address hypothesis, apoptosis assays per- <br /> late deacetylation RelA/p65.                                          formed 293T cells H1299 NSCLC cells transiently <br />    Silencing chromatin-bound BRMS1 promotes acetylation                   overexpressed BRMS1. As shown Fig. 7A B, BRMS1 <br /> of RelA/p65 decreases HDAC1 loading NF-&#65533;?&#65533;B-depen-                      overexpression sensitized cells TNF-induced apoptosis, as <br /> dent promoters. To elucidate direct cor-                 indicated enhanced DNA fragmentation caspase-3 ac- <br /> relation BRMS1 deacetylation RelA/p65 re-                     tivity, respectively. <br /> pression NF-&#65533;?&#65533;B mediated transcription, ChIP assays                      After detachment cellular matrix, suspended nor- <br /> performed promoter NF-&#65533;?&#65533;B-regulated anti-                   mal endothelial epithelial cells enter programmed cell <br /> apoptotic cIAP-2 Bfl-1/A1 genes 293T H1299                    death, process known anoikis (19). Oncogenic transfor- <br />  <br />  <br />  <br />  <br /> BRMS1 (10 &#65533;?&#65533;l) plus increasing doses HDAC1 (0, 1, 5, 10, 15 &#65533;?&#65533;l). On right, pellets incubated rabbit reticulocyte alone <br /> (lane 3) increasing doses vitro-translated HDAC1 (0, 1, 5, 10, 15 &#65533;?&#65533;l). A pan-acetyl-lysine antibody probed immunoblotting. <br /> RelA/p65 control detected equal amounts RelA/p65 IP. (E) 293T cells transfected expression vector encoding <br /> HA-tagged BRMS1or vector control. In addition, siRNA HDAC1, siRNA HDAC3, siRNA control cotransfected indicated. At <br /> 48 h transfection, cells treated treated TNF (20 ng/ml) 1 h. Western blots whole-cell lysates showed relative <br /> differences acetylation K310 p65 using specific acetyl-p65(K310) antibody relatively constant levels total p65 protein. Western blots <br /> for HDAC1 HDAC3 effective knockdown siRNA. A Western blot HA shows expression HA-BRMS1 tubulin a <br /> loading control. (F) 293T cells transiently cotransfected 3x-&#65533;?&#65533;B-Luc reporter plasmid encoding BRMS1 vector control. Cells <br /> were cotransfected siRNA HDAC1, siRNA HDAC3, siRNA control. Cells treated treated TNF (10 ng/ml) 12 h <br /> before harvesting luciferase activity assays. All transfections normalized CMV&#8211;&#65533;?&#65533;-galactosidase activity. Western blots indicate <br /> that siRNA HDAC1 HDAC3 specifically knocks protein expression. <br />  8692     LIU ET AL.                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 6. BRMS1 potentiates recruitment HDAC1 diminishes chromatin-associated acetyl-p65 specific K310 cIAP-2 and <br /> Bfl-1/A1 promoters. (A) Western blots indicate siRNA BRMS1 specifically knocks BRMS1 expression. (B) 293T cells transfected <br /> with siRNA BRMS1 siRNA control (siControl). Cells treated TNF (10 ng/ml) 12 h harvesting. ChIP analysis was <br /> performed previously described antibodies. Chromatin-associated protein complexes assessed quantitative PCR using PCR <br /> primers specific cIAP-2 Bfl-1/A1 promoters. The fold expression calculated amplified promoter fragment no <br /> treatment normalized 1. The results represent means &#11006; SD separate experiments performed triplicate. (C) Similar <br /> experiments repeated described above, exception expression vector encoding BRMS1 vector (Control) was <br /> ectopically expressed transient transfection instead siRNA. <br />  <br />  <br />  <br /> mation known result ability cells survive          mediated loss BRMS1 expression de- <br /> suspension. This ability cancer cells evade anoikis             creased tumor cell apoptosis anoikis, potentially resulting in <br /> required development metastatic disease. To deter-            larger number viable cells implantation meta- <br /> mine relative contribution BRMS1 anoikis, cell               static site. <br /> model tumor cells suspension created. As shown in <br /> Fig. 7C, 293T H1299 cells transiently expressing BRMS1                                         DISCUSSION <br /> had significantly cell death longer <span id='am-64' about='obo:NCBITaxon_9822' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9821 obo:NCBITaxon_35497 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-65'  ></span><span id='am-66' property="rdfs:label" content="Sus" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>sus</span>- <br /> pension compared controls. Conversely, siRNA knockdown                 In order explore signal transduction pathways related to <br /> of BRMS1 resulted little cell death despite detached            BRMS1 NF-&#65533;?&#65533;B, chose examine lung cancer cell <br /> for 3 h. This suggests mechanism metastasis             model lung cancer highly metastatic disease <br />     FIG. 7. BRMS1 sensitizes NSCLC cells apoptosis. (A) 293T cells H1299 cells transiently transfected expression vector <br /> encoding BRMS1 vector control. At 24 h transfection, cells treated treated TNF (50 ng/ml) additional 24 h. <br /> Cell extracts analyzed DNA fragmentation marker apoptosis. &#11569; &#11569;&#11569;, P &#65533;? 0.05 P &#65533;? 0.01, respectively, compared CMV <br /> plus No Add groups; # ##, P &#65533;? 0.05 P &#65533;? 0.01, respectively, compared CMV plus TNF groups. (B) 293T cells H1299 cells were <br /> transfected treated described above. Cell extracts analyzed caspase-3 activation marker apoptosis. (C) 293T H1299 <br /> cells transfected pCMV-BRMS1 versus pCMV vector (Control) siRNA-BRMS1 versus siRNA-control indicated. At <br /> 48 h transfection, cells suspended various lengths time. Cell death measured trypan blue exclusion assays. Western blots <br /> show siRNA specifically knocks BRMS1 protein expression. &#11569; &#11569;&#11569;, P &#65533;? 0.05 P &#65533;? 0.01, respectively, compared CMV siRNA <br /> control time point; #, P &#65533;? 0.05 compared time zero groups. <br />  <br />                                                                       8693 <br />  8694     LIU ET AL.                                                                                                    MOL. CELL. BIOL. <br />  <br />  <br /> (28). One explanation tumorigenesis           acetylation K310 playing prominent role regulation of <br /> there loss metastasis suppressor function, priming       transcriptional activity RelA/p65 (11, 33). In addi- <br /> the tumor develop distant metastases. Evidence         tion, TNF promote acetylation K310 demon- <br /> shown Fig. 1, BRMS1 mRNA protein                strated (12). Realizing binding do- <br /> decreased NSCLC cell lines human lung cancer tissues           main BRMS1 RelA/p65 included K310, wanted to <br /> compared normal epithelial lung cell line adjacent non-        determine BRMS1 repressing RelA/p65 tran- <br /> cancerous lung tissues.                                               scriptional activity promoting deacetylation lysine <br />    The separate steps required cancer success-         residue. Our findings indicate BRMS1 completely abol- <br /> fully metastasize migration, invasion, avoidance cell      ishes TNF stimulated acetylation RelA/p65 K310 and <br /> death apoptosis (17). Prior studies examining putative             deacetylation results dramatic decrease the <br /> mechanisms BRMS1-mediated suppression metastasis                transactivation potential RelA/p65 TNF treatment <br /> have demonstrated effect cancer cell invasion properties,       (Fig. 4). This observation interesting given TNF, along <br /> although breast cancer melanoma cells overexpressing              interleukin-6 cytokines, released tumor- <br /> BRMS1 modest decrease motility compared               associated macrophages shown stimulate tumor <br /> parental cells (52, 57). Other studies shown breast      progression NF-&#65533;?&#65533;B-dependent pathways (32). Our <br /> cancer cells BRMS1 restores gap junctional intercellular         data using siRNA knockdown BRMS1 support premise <br /> communication increases mRNA expression phosphor-              cancers lacking BRMS1 likely robust <br /> ylated connexin 43, responsible closing gap          response TNF cytokine-mediated transcriptional <br /> junction communication (39, 53, 62). We demonstrated             activation NF-&#65533;?&#65533;B-dependent antiapoptotic gene products, <br /> here additional mechanism BRMS1                 known promote tumor cell survival. <br /> functions involves ability corepressor modulate           While BRMS1 shown associate mSin3a/ <br /> NF-&#65533;?&#65533;B-dependent transcription genes known regulate              HDAC complex (42), demonstrate BRMS1 itself func- <br /> cell survival. Casey coworkers reported BRMS1          tions corepressor regulating binding HDAC1 to <br /> could modulate NF-&#65533;?&#65533;B nuclear translocation                 RelA/p65 promoting HDAC1-mediated deacetylation of <br /> IKK&#65533;?&#65533;-independent inhibition phosphorylation I&#65533;?&#65533;B,            RelA/p65. In contrast NCoR SMRT nuclear corepres- <br /> cytosolic protein sequesters NF-&#65533;?&#65533;B, preventing          sors, conserved nuclear receptor interaction motif <br /> nuclear translocation (13). We hypothesized BRMS1,             BRMS1. In addition, classic hydrophobic re- <br /> predominantly nuclear chromatin-associated protein,               gions BRMS1 associated core- <br /> could regulating NF-&#65533;?&#65533;B transcriptional activity.              pressors (29). This observation, fact BRMS1 teth- <br />    The present study demonstrates BRMS1 significantly            ers HDAC1 instead HDAC3, provides sharp contrast to <br /> inhibits NF-&#65533;?&#65533;B-dependent transcription, specifically trans-       described corepressors, NCoR SMRT. <br /> activation potential RelA/p65. Moreover, se-             Our data demonstrate chromatin-associated BRMS1 <br /> lective silencing BRMS1 restores expression NF-&#65533;?&#65533;B-          regulates acetyl-K310 RelA/p65 status NF-&#65533;?&#65533;B-de- <br /> dependent antiapoptotic genes cIAP-2, Bfl-1/A1, BclxL          pendent cIAP-2 Bfl-1/A1 promoters (Fig. 6). We cannot <br /> HEK 293T NSCLC cells sensitizes cell lines            dismiss role mSin3a potential contribution the <br /> TNF-induced apoptosis (Fig. 2 7). Previous studies           inhibition RelA/p65-dependent transcription a <br /> shown antiapoptotic proteins protect cancer cells           concomitant decrease mSin3a chromatin loss <br /> anoikis prolong survival, increasing likeli-       BRMS1. In addition, given corepressor function of <br /> hood survive following invasion through,           BRMS1, surprising modify <br /> detachment from, extracellular matrix (49, 66). The regu-         chromatin structure enhanced deacetylation his- <br /> lation tumor cell viability cancer cells migrate, invade,   tone tails H3 H4. However, noted Fig. 5, vitro <br /> then detach enter bloodstream key event tumor         coexpression BRMS1 HDAC1 sufficient ef- <br /> metastatic progression. Thus, demonstrated present          ficiently deacetylate RelA/65. Thus, BRMS1 enhances re- <br /> study, BRMS1-mediated suppression NF-&#65533;?&#65533;B-dependent an-              pression endogenous NF-&#65533;?&#65533;B-dependent antiapoptotic genes <br /> tiapoptotic gene expression results proapoptotic state,          promoting HDAC-1-mediated deacetylation RelA/p65 <br /> priming tumor TNF-mediated cell death.                        directly chromatin. <br />    We BRMS1 coimmunoprecipitates endogenous                    In summary, demonstrate metastasis suppressor <br /> RelA/p65 p50 (Fig. 3). Moreover, BRMS1 directly inter-            BRMS1 directly interacts RelA/p65 inhibits trans- <br /> acts C-terminal region RHD RelA/p65                activation potential, part, functioning corepressor for <br /> (amino acids 254 354). The functional significance              HDAC1-mediated deacetylation RelA/p65. The loss of <br /> BRMS1 binding region RelA/p65 shown Fig. 3E,         BRMS1 likely affect transcription factors (i.e., AP1, <br /> where maximal repression RelA/p65 transactivation po-          STAT, etc.) regulated HDAC1 involved in <br /> tential occurs constructs overexpressing binding domain.      cancer cell survival signal transduction pathways. Future stud- <br /> Interestingly, HDAC1 described bind          ies focus upstream events BRMS1 is <br /> region RelA/p65 (65).                                      regulated, particularly relate ability BRMS1 to <br />    While NF-&#65533;?&#65533;B transcription regulated interactions        decrease tumor cell survival. <br /> with inhibitor I&#65533;?&#65533;B, recent studies identified the <br /> importance acetylation regulating activity (10, 11, 64).                         ACKNOWLEDGMENTS <br /> The RelA/p65 subunit NF-&#65533;?&#65533;B acetylated coactiva-            We thank Marty W. Mayo (University Virginia) review of <br /> tors p300, CBP, p/CAF known lysine residues,         manuscript constructive comments. <br />  VOL. 26, 2006                                                                                                                          BRMS1 AND NF-&#65533;?&#65533;B                 8695 <br />  <br />   This study supported grant Commonwealth                           I&#65533;?&#65533;B kinase alpha: prerequisite NF-&#65533;?&#65533;B transcription survival. Mol. <br /> Foundation Cancer Research (D.R.J.) NIH Surgery Re-                                Cell 16:245&#8211;255. <br /> search Training grant T32 HL007849 (P.W.S.).                                            27.   Hong, S. Y., W. H. Yoon, J. H. Park, S. G. Kang, J. H. Ahn, T. H. Lee. <br />                                                                                               2000. Involvement NF-&#65533;?&#65533;B binding elements tumor necrosis factor <br />                                                                                               alpha-, CD40-, Epstein-Barr virus latent membrane protein 1-mediated <br />                                   REFERENCES                                                  induction cellular inhibitor apoptosis protein 2 gene. J. Biol. Chem. <br />  1. Aggarwal, B. B. 2004. Nuclear factor-&#65533;?&#65533;B: enemy within. Cancer Cell                    275:18022&#8211;18028. <br />     6:203&#8211;208.                                                                          28.   Jemal, A., R. Siegel, E. Ward, T. Murray, J. Xu, C. Smigal, M. J. Thun. <br />  2. Ashburner, B. P., S. D. Westerheide, A. S. Baldwin, Jr. 2001. The p65                 2006. Cancerstatistics, 2006. CA Cancer J. Clinicians 56:106&#8211;130. <br />     (RelA) subunit NF-&#65533;?&#65533;B interacts histone deacetylase (HDAC)               29.   Jepsen, K., M. G. Rosenfeld. 2002. Biological roles mechanistic <br />     corepressors HDAC1 HDAC2 negatively regulate gene expression.                      actions co-repressor complexes. J. Cell Sci. 115:689&#8211;698. <br />     Mol. Cell. Biol. 21:7065&#8211;7077.                                                      30.   Jones, D. R., R. M. Broad, L. D. Comeau, S. J. Parsons, M. W. Mayo. <br />  3. Baeuerle, P. A. 1998. I&#65533;?&#65533;B-NF-&#65533;?&#65533;B structures: interface inflammation              2002. Inhibition nuclear factor &#65533;?&#65533;B chemosensitizes non-small cell lung <br />     control. Cell 95:729&#8211;731.                                                                 cancer cytochrome c release caspase activation. J. Thorac. <br />  4. Baldwin, A. S., Jr. 1996. The NF-&#65533;?&#65533;B I &#65533;?&#65533;B proteins: new discoveries                Cardiovasc. Surg. 123:310&#8211;317. <br />     insights. Annu. Rev. Immunol. 14:649&#8211;683.                                           31.   Jones, D. R., R. M. Broad, L. V. Madrid, A. S. Baldwin, Jr., M. W. Mayo. <br />  5. Balkwill, F. 2002. Tumor necrosis factor tumor promoting factor? Cyto-                 2000. Inhibition NF-&#65533;?&#65533;B sensitizes non-small cell lung cancer cells to <br />     kine Growth Factor Rev. 13:135&#8211;141.                                                       chemotherapy-induced apoptosis. Ann. Thorac. Surg. 70:930&#8211;937. <br />  6. Bitterman, K. J., R. M. Anderson, H. Y. Cohen, M. Latorre-Esteves,              32.   Karin, M., F. R. Greten. 2005. NF-&#65533;?&#65533;B: linking inflammation immu- <br />     D. A. Sinclair. 2002. Inhibition silencing accelerated aging nicotin-           nity cancer development progression. Nat. Rev. Immunol. 5:749&#8211;759. <br />     amide, putative negative regulator yeast sir2 human SIRT1. J. Biol.        33.   Kiernan, R., V. Bres, R. W. Ng, M. P. Coudart, S. El Messaoudi, C. Sardet, <br />     Chem. 277:45099&#8211;45107.                                                                    D. Y. Jin, S. Emiliani, M. Benkirane. 2003. Post-activation turn-off of <br />  7. Bours, V., E. Dejardin, F. Goujon-Letawe, M. P. Merville, V. Cas-                     NF-&#65533;?&#65533;B-dependent transcription regulated acetylation p65. J. Biol. <br />     tronovo. 1994. The NF-&#65533;?&#65533;B transcription factor cancer: high expression              Chem. 278:2758&#8211;2766. <br />     NF-&#65533;?&#65533;B- I &#65533;?&#65533;B-related proteins tumor cell lines. Biochem. Pharmacol.           34.   Kuo, M. H., C. D. Allis. 1999. In vivo cross-linking immunoprecipi- <br />     47:145&#8211;149.                                                                               tation studying dynamic protein-DNA associations chromatin envi- <br />  8. Chakrabarti, S. K., J. C. James, R. G. Mirmira. 2002. Quantitative                    ronment. Methods 19:425&#8211;433. <br />     assessment gene targeting vitro vivo pancreatic transcrip-      35.   Laherty, C. D., W. M. Yang, J. M. Sun, J. R. Davie, E. Seto, R. N. <br />     tion factor, Pdx1. Importance chromatin structure directing promoter                Eisenman. 1997. Histone deacetylases associated mSin3 corepressor <br />     binding. J. Biol. Chem. 277:13286&#8211;13293.                                                  mediate mad transcriptional repression. Cell 89:349&#8211;356. <br />  9. Chen, C., L. C. Edelstein, C. Gelinas. 2000. The Rel/NF-&#65533;?&#65533;B <span id='am-40' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-41' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span>               36.   Lee, C. W., W. N. Lin, C. C. Lin, S. F. Luo, J. S. Wang, J. Pouyssegur, and <br />     directly activates expression apoptosis inhibitor Bcl-xL. Mol. Cell. Biol.         C. M. Yang. 2006. Transcriptional regulation VCAM-1 expression by <br />     20:2687&#8211;2695.                                                                             tumor necrosis factor-alpha human tracheal smooth muscle cells: involve- <br /> 10. Chen, L., W. Fischle, E. Verdin, W. C. Greene. 2001. Duration nuclear              ment MAPKs, NF-&#65533;?&#65533;B, p300, histone acetylation. J. Cell Physiol. <br />     NF-&#65533;?&#65533;B action regulated reversible acetylation. Science 293:1653&#8211;1657.                  207:174&#8211;186. <br /> 11. Chen, L. F., Y. Mu, W. C. Greene. 2002. Acetylation RelA discrete         37.   Lee, D. K., J. E. Kang, H. J. Park, M. H. Kim, T. H. Yim, J. M. Kim, M. K. <br />     sites regulates distinct nuclear functions NF-&#65533;?&#65533;B. EMBO J. 21:6539&#8211;6548.                Heo, K. Y. Kim, H. J. Kwon, M. W. Hur. 2005. FBI-1 enhances tran- <br /> 12. Chen, L. F., S. A. Williams, Y. Mu, H. Nakano, J. M. Duerr, L. Buckbinder,                scription nuclear factor-&#65533;?&#65533;B (NF-&#65533;?&#65533;B)-responsive E-selectin gene by <br />     W. C. Greene. 2005. NF-&#65533;?&#65533;B RelA phosphorylation regulates RelA acet-                   nuclear localization p65 subunit NF-&#65533;?&#65533;B. J. Biol. Chem. 280:27783&#8211; <br />     ylation. Mol. Cell. Biol. 25:7966&#8211;7975.                                                   27791. <br /> 13. Cicek, M., R. Fukuyama, D. R. Welch, N. Sizemore, G. Casey. 2005. <br />                                                                                         38.   Liu, Y., C. E. Denlinger, B. K. Rundall, P. W. Smith, D. R. Jones. 21 <br />     Breast cancer metastasis suppressor 1 inhibits gene expression targeting <br />                                                                                               August 2006, posting date. SAHA induces Akt-mediated phosphorylation of <br />     nuclear factor-&#65533;?&#65533;B activity. Cancer Res. 65:3586&#8211;3595. <br />                                                                                               p300 promotes acetylation transcriptional activation RelA/p65. <br /> 14. Dejardin, E., G. Bonizzi, A. Bellahcene, V. Castronovo, M. P. Merville, and <br />                                                                                               J. Biol. Chem. [Online.] doi: 10.1074/jbc.M604478200. <br />     V. Bours. 1995. Highly-expressed p100/p52 (NFKB2) sequesters NF- <br />                                                                                         39.   Locke, D. 1998. Gap junctions normal neoplastic mammary gland. <br />     &#65533;?&#65533;B-related proteins cytoplasm human breast cancer cells. Oncogene <br />                                                                                               J. Pathol. 186:343&#8211;349. <br />     11:1835&#8211;1841. <br />                                                                                         40.   Marks, P., R. A. Rifkind, V. M. Richon, R. Breslow, T. Miller, W. K. <br /> 15. D&#8217;Souza, B. N., L. C. Edelstein, P. M. Pegman, S. M. Smith, S. T. Loughran, <br />                                                                                               Kelly. 2001. Histone deacetylases cancer: causes therapies. Nat. <br />     A. Clarke, A. Mehl, M. Rowe, C. Gelinas, D. Walls. 2004. Nuclear factor <br />                                                                                               Rev. Cancer 1:194&#8211;202. <br />     &#65533;?&#65533;B-dependent activation antiapoptotic bfl-1 gene Epstein-Barr <br />     virus latent membrane protein 1 activated CD40 receptor. J. Virol.              41.   Mayo, M. W., C. E. Denlinger, R. M. Broad, F. Yeung, E. T. Reilly, Y. Shi, <br />     78:1800&#8211;1816.                                                                             D. R. Jones. 2003. Ineffectiveness histone deacetylase inhibitors to <br /> 16. Duffey, D. C., Z. Chen, G. Dong, F. G. Ondrey, J. S. Wolf, K. Brown, U.                   induce apoptosis involves transcriptional activation NF-&#65533;?&#65533;B through <br />     Siebenlist, C. Van Waes. 1999. Expression dominant-negative mu-                  Akt pathway. J. Biol. Chem. 278:18980&#8211;18989. <br />     tant inhibitor-&#65533;?&#65533;Balpha nuclear factor-&#65533;?&#65533;B human head neck squa-            42.   Meehan, W. J., R. S. Samant, J. E. Hopper, M. J. Carrozza, L. A. Shevde, <br />     mous cell carcinoma inhibits survival, proinflammatory cytokine expression,               J. L. Workman, K. A. Eckert, M. F. Verderame, D. R. Welch. 2004. <br />     tumor growth vivo. Cancer Res. 59:3468&#8211;3474.                                       Breast cancer metastasis suppressor 1 (BRMS1) forms complexes ret- <br /> 17. Fidler, I. J. 1995. Critical factors biology human cancer metastasis.           inoblastoma-binding protein 1 (RBP1) mSin3 histone deacetylase <br />     Am. Surgeon 61:1065&#8211;1066.                                                                 complex represses transcription. J. Biol. Chem. 279:1562&#8211;1569. <br /> 18. Fischle, W., S. Emiliani, M. J. Hendzel, T. Nagase, N. Nomura, W. Voelter,          43.   Meehan, W. J., D. R. Welch. 2003. Breast cancer metastasis suppressor <br />     E. Verdin. 1999. A new family human histone deacetylases related                1: update. Clin. Exp. Metastasis 20:45&#8211;50. <br />     <span id='am-16' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-27'  ></span><span id='am-28' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> HDA1p. J. Biol. Chem. 274:11713&#8211;11720.                     44.   Mountain, C. F. 1997. Revisions international staging lung <br /> 19. Frisch, S. M., H. Francis. 1994. Disruption epithelial cell-matrix                 cancer. Chest 111:1710&#8211;1717. <br />     interactions induces apoptosis. J. Cell Biol. 124:619&#8211;626.                          45.   Nagy, L., H. Y. Kao, D. Chakravarti, R. J. Lin, C. A. Hassig, D. E. Ayer, S. L. <br /> 20. Gerritsen, M. E., A. J. Williams, A. S. Neish, S. Moore, Y. Shi, T.                   Schreiber, R. M. Evans. 1997. Nuclear receptor repression mediated by <br />     Collins. 1997. CREB-binding protein/p300 transcriptional coactivators              complex containing S <br /> </body></html>