<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>A genome-scale analysis identification of <br /> genes required growth survival of <br /> <span id='am-286' about='obo:NCBITaxon_727' typeof='owl:Thing obo:NCBITaxon_135625 obo:NCBITaxon_712 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_724'><span id='am-287'  ></span><span id='am-288' property="rdfs:label" content="Haemophilus influenzae" datatype="xsd:string"></span><span id='am-289' property="oboInOwl:hasRelatedSynonym" content="Haemophilus meningitidis" datatype="xsd:string"></span><span id='am-290' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-291' property="oboInOwl:hasRelatedSynonym" content="Bacterium influenzae" datatype="xsd:string"></span><span id='am-292' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-293' property="oboInOwl:hasRelatedSynonym" content="Coccobacillus pfeifferi" datatype="xsd:string"></span><span id='am-294' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium influenzae" datatype="xsd:string"></span><span id='am-295' property="oboInOwl:hasRelatedSynonym" content="Influenza-bacillus" datatype="xsd:string"></span>Haemophilus influenzae</span> <br /> Brian J. Akerley*, Eric J. Rubin&#8224;, <span id='am-138' about='obo:NCBITaxon_4173' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_216795 obo:NCBITaxon_4143 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_156152 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-139' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-140' property="rdfs:label" content="Veronica" datatype="xsd:string"></span><span id='am-141' property="oboInOwl:hasAlternativeId" content="NCBITaxon:165315" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasExactSynonym" content="brooklimes" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasAlternativeId" content="NCBITaxon:274338" datatype="xsd:string"></span><span id='am-144' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-145'  ></span><span id='am-146' property="oboInOwl:hasRelatedSynonym" content="Veroncia" datatype="xsd:string"></span><span id='am-147' property="oboInOwl:hasExactSynonym" content="speedwells" datatype="xsd:string"></span>Veronica</span> L. Novick&#8225;, Kensey Amaya*, Nicholas Judson&#8225;, John J. Mekalanos&#8225;&#167; <br /> *Department Microbiology Immunology, University Michigan Medical School, Ann Arbor, MI 48109; &#8224;Department Immunology Infectious <br /> Diseases, Harvard School Public Health, Boston, MA 02115; &#8225;Department Microbiology Molecular Genetics, Harvard Medical School, <br /> Boston, MA 02115 <br />  <br /> Contributed John J. Mekalanos, November 9, 2001 <br />  <br /> A high-density transposon mutagenesis strategy applied                 We described approach (9) detecting essential <br /> the <span id='am-296' about='obo:NCBITaxon_724' typeof='owl:Thing obo:NCBITaxon_135625 obo:NCBITaxon_712 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236'><span id='am-297'  ></span><span id='am-298' property="rdfs:label" content="Haemophilus" datatype="xsd:string"></span><span id='am-299' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-300' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-301' property="oboInOwl:hasRelatedSynonym" content="Haemophilus Winslow et al. 1917" datatype="xsd:string"></span><span id='am-302' property="oboInOwl:hasDbXref" content="PMID:1736960" datatype="xsd:string"></span>Haemophilus</span> influenzae genome identify genes required           genes applies high-density vitro transposon mutagenesis <br /> growth viability. <span id='am-148' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-149' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-150'  ></span><span id='am-151' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> analysis detected putative essential roles       genetic footprinting discrete chromosomal segments of <br /> for products 259 ORFs unknown function. Comparisons              genomes naturally transformable organisms. This approach <br /> between complete genomes defined subset proteins             rapidly detected known previously unknown genes essen- <br /> H. influenzae having homologs <span id='am-61' about='obo:NCBITaxon_1763' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1762 obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003'><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="PMID:8863452" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="PMID:1883713" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="PMID:1581193" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="PMID:2275850" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="PMID:7547304" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium Lehmann and Neumann 1896" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="Mycobacterium" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="PMID:1380284" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="PMID:7547284" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="PMID:16014496" datatype="xsd:string"></span><span id='am-74'  ></span><span id='am-75' property="oboInOwl:hasDbXref" content="PMID:7907223" datatype="xsd:string"></span>Mycobacterium</span> tuberculosis           tial growth viability vitro. We report application <br /> are absent <span id='am-114' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-115'  ></span><span id='am-116' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-126' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>, distribution pattern        method complete genome H. influenzae, a <br /> favors use development antimicrobial therapeutics.             common human pathogen causing range diseases includ- <br /> Three genes set essential viability           ing otitis media pneumonia (10, 11). The <span id='am-377' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-378' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-379' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-380' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-381'  ></span>data</span> obtained <br /> bacteria. Interfacing set essential gene products H.             study indicate &#11015;54% essential genes lack <br /> influenzae distribution homologs microorgan-          defined functional roles. Sequence comparison detects poten- <br /> isms detect components unrecognized cellular pathways               tial orthologs large number putatively essential <br /> essential diverse bacteria. This genome-scale phenotypic anal-          genes bacterial <span id='am-158' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-159' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-160' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span>. This information relevant to <br /> ysis identifies potential roles large set genes unknown        defining common essential pathways life and <br /> function.                                                                  potential targets development antimicrobial therapeu- <br />                                                                            tics. This genome-scale database provides experimen- <br />                                                                            tal evidence putative biological roles large group of <br /> T    availability complete microbial genome sequences <br />      allows large-scale functional analysis genes known only <br /> by DNA sequences. A growing number high- <br />                                                                            genes unknown function H. influenzae. <br />  <br /> throughput approaches developed use in-                  Methods <br /> formation. Expression profiling DNA microarrays (1) <span id='am-153' about='obo:NCBITaxon_1178312' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_33154 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_572719 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41011 obo:NCBITaxon_41197 obo:NCBITaxon_41012 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-154' property="rdfs:label" content="Mea" datatype="xsd:string"></span><span id='am-155' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-157'  ></span>mea</span>-                Plasmid Construction. To increase efficiency in <br /> sures gene expression highly parallel fashion                 vitro transposon mutagenesis procedure constructed a <br /> identify possible functional relationships genes                mariner-based minitransposon containing H. influenzae <br /> evaluation coexpression (2). Two-<span id='am-392' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-393'  ></span><span id='am-394' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-395' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-396' property="rdfs:label" content="hybrid" datatype="xsd:string"></span>hybrid</span> protein interaction             DNA uptake consensus sequence (US) increases trans- <br /> technologies determine proteins physically interact              formation frequency (12). A KpnI site pENT3 (9) was <br /> providing possible means linking genes known pheno-                removed digestion SpeI SapI followed blunt-end <br /> types. Construction specific gene-knockouts genomic                creation T4 polymerase, religation, cloning a <br /> scale initiated <span id='am-354' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-355' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-356' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-357' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-358' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-359' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-360' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-361' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-362' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-363'  ></span><span id='am-364' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-365' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-366' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-367' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-368' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-369' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span>Escherichia coli</span> (3) <span id='am-28' about='obo:NCBITaxon_1423' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_653685'><span id='am-29' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451270" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429428" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232819" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429427" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasAlternativeId" content="NCBITaxon:523730" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Bacillus natto" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232470" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Bacillus uniflagellatus" datatype="xsd:string"></span><span id='am-38'  ></span><span id='am-39' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451272" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232840" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042380" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis8" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224247" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Vibrio subtilis" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232841" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Bacillus subtilis" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232822" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042382" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasAlternativeId" content="NCBITaxon:941949" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232828" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis/Bacillus globigii" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="PMID:11211269" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasAlternativeId" content="NCBITaxon:387338" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232825" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasAlternativeId" content="NCBITaxon:981081" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232824" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232844" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232830" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042381" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Bacillus subtilis</span>          derivative containing unique KpnI site transposon, <br /> (4). This strategy provides direct means assess genes           pENT3kpnI. Primers (US1-BA, 5&#11032;-GGTACCA A A AC- <br /> are required given phenotype laborious                 CAGTTTCAGGTCAGCAAG-3&#11032;; US2-BA, 5&#11032;-GGTAC- <br /> generally impractical individual laboratories. Several al-             CTCGCCTTTGTTCCTTGTGTGCC-3&#11032;) used PCR- <br /> ternative approaches recently used. In Saccharo-                 amplify 177-bp segment H. influenzae chromosome <br /> myces cerevisiae, genetic footprinting applied chromosome               containing strong match US consensus distrib- <br /> V detected &#11015;53 essential genes (5). Sequencing randomly                 uted genome (13). This US segment cloned <br /> generated mutants <span id='am-85' about='obo:NCBITaxon_2093' typeof='owl:Thing obo:NCBITaxon_2092 obo:NCBITaxon_1 obo:NCBITaxon_2085 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_544448 obo:NCBITaxon_31969'><span id='am-86' property="oboInOwl:hasDbXref" content="PMID:11931184" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasRelatedSynonym" content="Haemobartonella" datatype="xsd:string"></span><span id='am-88'  ></span><span id='am-89' property="rdfs:label" content="Mycoplasma" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasRelatedSynonym" content="Asterococcus" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasAlternativeId" content="NCBITaxon:29500" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasRelatedSynonym" content="Eperythrozoon" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasAlternativeId" content="NCBITaxon:57371" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasDbXref" content="PMID:8995799" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasDbXref" content="PMID:16350067" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasRelatedSynonym" content="Bovimyces" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasDbXref" content="PMID:15176735" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="GC_ID:4" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasDbXref" content="PMID:11321109" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasRelatedSynonym" content="Asteromyces" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasDbXref" content="PMID:16403858" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasDbXref" content="PMID:13403276" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasDbXref" content="PMID:11411711" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasRelatedSynonym" content="Borrelomyces" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasRelatedSynonym" content="Pleuropneumonia" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasDbXref" content="PMID:10826816" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasDbXref" content="PMID:8863441" datatype="xsd:string"></span>Mycoplasma</span> genitalium Mycoplasma                  KpnI site pENTkpnI create pENTUS carrying <br /> pneumoniae shown allow rapid identification                 Tnmagellan4 minitransposon. <br /> putatively nonessential genes (6), technique feasible <br /> for laboratories access high-throughput DNA sequenc- <br /> ing capabilities. This approach potentially identify essen-            Mutagenesis H. influenzae Genomic Segments Genetic Foot- <br /> tial genes confidence level depends character-              printing. H. influenzae Rd genome (ATCC #9008) mu- <br /> izing transposon junctions cover complete                    tagenized vitro transposition, using series overlapping <br /> genome high level redundancy (6). Several random                   &#11015;10-kb PCR products targets (see Fig. 1 overview). <br /> genetic screens applicable essential gene identification.           Primers designed create overlapping &#11015;10-kb PCR <br /> A transposon-based conditional expression used               products representing complete H. influenzae genome <br /> identify 16 essential genes <span id='am-320' about='obo:NCBITaxon_666' typeof='owl:Thing obo:NCBITaxon_641 obo:NCBITaxon_662 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_135623'><span id='am-321'  ></span><span id='am-322' property="oboInOwl:hasRelatedSynonym" content="Vibrio comma" datatype="xsd:string"></span><span id='am-323' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-324' property="oboInOwl:hasDbXref" content="PMID:9272984" datatype="xsd:string"></span><span id='am-325' property="rdfs:label" content="Vibrio cholerae" datatype="xsd:string"></span><span id='am-326' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-327' property="oboInOwl:hasRelatedSynonym" content="Spirillum cholerae-asiaticae" datatype="xsd:string"></span><span id='am-328' property="oboInOwl:hasRelatedSynonym" content="Spirillum cholerae" datatype="xsd:string"></span><span id='am-329' property="oboInOwl:hasRelatedSynonym" content="Microspira comma" datatype="xsd:string"></span><span id='am-330' property="oboInOwl:hasRelatedSynonym" content="Pacinia cholerae-asiaticae" datatype="xsd:string"></span><span id='am-331' property="oboInOwl:hasRelatedSynonym" content="Kommabacillus" datatype="xsd:string"></span><span id='am-332' property="oboInOwl:hasRelatedSynonym" content="Vibrio choleae" datatype="xsd:string"></span><span id='am-333' property="oboInOwl:hasRelatedSynonym" content="Vibrio cholerae-asiaticae" datatype="xsd:string"></span><span id='am-334' property="oboInOwl:hasRelatedSynonym" content="Bacillo virgola del Koch" datatype="xsd:string"></span><span id='am-335' property="oboInOwl:hasRelatedSynonym" content="Vibrio cholera" datatype="xsd:string"></span><span id='am-336' property="oboInOwl:hasRelatedSynonym" content="Bacillus cholerae-asiaticae" datatype="xsd:string"></span><span id='am-337' property="oboInOwl:hasRelatedSynonym" content="Liquidivibrio cholerae" datatype="xsd:string"></span><span id='am-338' property="oboInOwl:hasRelatedSynonym" content="Bacillus cholerae" datatype="xsd:string"></span>Vibrio cholerae</span> (7). A screen <br /> based inhibition gene function antisense RNA applied <br /> to <span id='am-1' about='obo:NCBITaxon_1279' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1385 obo:NCBITaxon_90964 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-2' property="oboInOwl:hasDbXref" content="PMID:10319495" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="PMID:10758876" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="PMID:9734063" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="PMID:17220435" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="PMID:10319469" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Staphylococcus" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Aurococcus" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="PMID:12656157" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="PMID:10425778" datatype="xsd:string"></span>Staphylococcus</span> aureus ascribed roles viability &#11015;150               Abbreviations: US, uptake consensus sequence; BHI, brain heart infusion. <br />  <br /> genes (8). These approaches offer ability easily identify           &#167;To <br />                                                                              reprint requests addressed at: Department Microbiology and <br />                                                                            Molecular Genetics, Harvard Medical School, Building D1, Room 411, 200 Longwood <br /> the subset essential genes amenable conditional expres-              Avenue, Boston, MA 02115. E-mail: jmekalanos@hms.harvard.edu. <br /> sion inhibition. However, methods stochastic and, <br />                                                                            The publication costs article defrayed page charge payment. This <br /> therefore, provide information genes encoun-                article marked &#8220;advertisement&#65533;? accordance 18 U.S.C. <br /> tered screen.                                                   &#167;1734 solely indicate fact. <br />  <br />  <br />  <br /> 966 &#8211;971 &#20841; PNAS &#20841; January 22, 2002 &#20841; vol. 99 &#20841; no. 2                                                          www.pnas.org&#20862;cgi&#20862;doi&#20862;10.1073&#20862;pnas.012602299 <br />  Fig. 1. Strategy whole-genome mutagenesis vitro mariner mutagenesis functional analysis genetic footprinting. (A) In vitro transposon <br /> mutagenesis Tnmagellan4 conducted PCR products corresponding regions H. influenzae chromosome. Each mutagenized DNA segment <br /> was introduced separately H. influenzae natural transformation homologous recombination. The Tnmagellan4 element encodes kanamycin <br /> resistance (KmR) contains DNA US, flanked transposon-inverted repeats (black triangles). (B) PCR products (10 kb) were <br /> generated targets mutagenesis &#11015;5 kb overlap adjacent segments. These products mutagenized (as A) generate 366 <br /> individual pools transposon mutants. The primers used obtain genetic footprinting data &#11015;2.5 kb genome, primer <br /> was used independent pools mutants. <br />  <br />  <br />  <br /> sequence (14). Automated primer design conducted                    30 cycles. A GIBCO&#20862;BRL 1-kb DNA ladder used size <br /> the MACVECTOR program desired melting point                         marker gel electrophoresis. <br /> parameter set minimum 62&#176;C. The resulting primer set <br />  <br />  <br />  <br />  <br />                                                                                                                                                           MICROBIOLOGY <br /> was filtered using Microsoft EXCEL select primers that <br />                                                                              Results <br /> produced &#11015;10-kb products &#11015;5-kb overlap. This pro- <br /> cedure produced set primers generate targets vitro            Generation Ordered Collection Transposon Insertion Mutants <br />                                                                              Encompassing H. influenzae Genome. Our previous analysis of <br /> transposition querying 2.5-kb segment the <br /> chromosome genetic footprinting. In vitro transposition                   H. influenzae genome analysis mapping vitro <br />                                                                              transposition (GAMBIT) procedure indicated ap- <br /> reactions conducted essentially described (9) and <br />                                                                              proach applied complete genome define <br /> contained 500 ng 10-kb PCR product, 100 ng of <br />                                                                              functional roles genes based DNA sequence information <br /> pENTUS, purified hyperactive C9 mutant (15) Himar1 <br />                                                                              sole input data (9). Genes essential viability under <br /> transposase, gift D. Lampe (Duquesne Univ., Pittsburgh). <br />                                                                              defined growth condition (in case, ability form <br /> The C9 mutant transposase increases efficiency this <br />                                                                              colonies BHIs agar medium) sustain markedly reduced <br /> mutagenesis procedure 10 &#8211;50-fold (15). Each reaction                 frequency transposon insertions detected genetic <br /> transformed competent H. influenzae (16) outgrown                   footprinting. Results approach correlate with <br /> 1 h brain heart infusion (BHI) supplemented hemin (10                independent methods assigning essentiality (9, 17) and, <br /> &#65533;?&#65533;g&#20862;ml) nicotinamide adenine dinucleotide [10 &#65533;?&#65533;g&#20862;ml                       therefore, genes sustain transposon insertions are <br /> (BHIs)], selected colony formation 36 h growth              designated putatively essential. Adjacent nonessential genes <br /> at 37&#176;C BHIs agar containing 20 &#65533;?&#65533;g&#20862;ml kanamycin                        intergenic regions DNA segment serve internal <br /> sulfate. Transformants pooled BHIs 15% glycerol                 controls sustain insertions. Extended-length PCR <br /> and stored ordered set &#11002;80&#176;C. Dilutions frozen              reactions used amplify 366 &#11015;10-kb DNA segments <br /> stock plated obtain 2,000 colonies. Colonies                representing H. influenzae genome. The segments were <br /> were pooled adjusted approximate OD600 0.1,                 designed overlap &#11015;5-kb (Fig. 1). This design achieves two <br /> 1 &#65533;?&#65533;l added PCR reactions containing                       objectives. (i) Mutations segments genome <br /> chromosomal primer corresponding mutagenized region                   represented independent pools transpo- <br /> and primer specific Himar1 inverted repeat                  son mutants. (ii) Primers set complementary to <br /> ends Tnmagellan4, Marout (5&#11032;-CCGGGGACTTATCAGC-                            sites H. influenzae chromosome spaced, average, <br /> CAACC-3&#11032;). Genetic footprinting conducted essentially                 2.5 kb. Analysis gene independent pools <br /> described (9) following reaction cycle used:             provides verification. The distance 2.5 kb primer <br /> 94&#176;C 30 s followed 68&#176;C 5 min &#11001; 15 s cycle                binding sites allows obtain overlapping data some <br />  <br /> Akerley et al.                                                                                      PNAS &#20841; January 22, 2002 &#20841; vol. 99 &#20841; no. 2 &#20841; 967 <br />                                                                               Table 1. Summary putative essential nonessential genes <br />                                                                              Functional designation                                           No. <br />  <br />                                                                              Essential gene candidates (no insertions)                        478 <br />                                                                                Hypothetical genes (unknown functions)                         259 <br />                                                                                Genes known functions                                     219 <br />                                                                              Nonessential genes (insertions distributed ORF)            538 <br />                                                                              One insertion detected                                           191 <br />                                                                              Two insertions detected                                           65 <br />                                                                              Not scored (too far primers)                                453 <br />  <br />  <br />  <br />  <br />                                                                              ping segments, contain relatively high <br />                                                                              proportion putatively essential genes (22 27 genes <br />                                                                              examined). Therefore, relative scarcity nonessential genes <br />                                                                              regions account recovery fewer <br />                                                                              colonies. The regions yielded colonies were <br />                                                                              adjacent segments corresponding putative phage gene prod- <br />                                                                              ucts homologous phage Mu (14). The significance <br />                                                                              reduced insertional mutagenesis frequency Mu-like <br />                                                                              locus clear. <br />  <br />  <br />                                                                              Analysis Transposon Insertion Mutants Each Subgenomic Chro- <br />                                                                              mosomal Region. Mutants representing chromosomal re- <br />                                                                              gion pooled analyzed genetic footprinting as <br />                                                                              described (9). Distances chromosome-specific <br />                                                                              primer calculated, map positions ORF <br />                                                                              respect primer directly superimposed the <br /> Fig. 2.   Scoring genetic footprinting H. influenzae <br />                                                                              gel images shown Fig. 2. Each region scored and <br /> genome. Maps ORF locations respect binding sites primer   color-coded (Fig. 2) according insertions be <br /> were superimposed gel images. Dashed (putative nonessential regions)      detected. The intensity bands footprinting gels, <br /> or solid (putative essential regions) lines gel image represent       precisely quantitative, provides indication of <br /> predicted segments H. influenzae coding sequences. Primer reference       relative number mutants containing insertion mutations <br /> numbers (red) listed near primer positions gene map (Left)    chromosomal position. As previous observations (5, 9) <br /> on gel image lane. Tick marks gene map <br /> represent approximate positions insertions detected. Pools of <br />                                                                              indicated essential genes able sustain insertion <br /> mutants derived independent transposition reactions named        mutations 3&#11032; end, exclusively scored 5&#11032; 75% of <br /> the primer pairs used generate targets vitro transposon mutagen-   length ORF criteria. Genes sustaining no <br /> esis (top, brackets). The gel image shows analysis overlapping        insertions region designated putatively essential <br /> &#11015;10-kb regions corresponding mutant pools 327&#8211;328 329 &#8211;330.           growth viability. Those capable sustaining insertions <br /> Colors denote summarized results. Red , insertions; blue,         single apparent position likely essential <br /> insertion site; white, insertion mutations partially reduce growth; and <br /> yellow, evenly distributed insertions&#20862;putative nonessential gene. Absence <br />                                                                              essential genes encode nonessential domains or <br /> of insertions 300 bp end target PCR product       divided complementing ORFs (although some <br /> considered significant minimum distance required          bands represent insertions multiple adjacent sites <br /> homologous recombination (e.g., blue line marked asterisk lane       depending resolution gel system). We scored <br /> 327).                                                                        genes containing single permissive insertion sites when <br />                                                                              resolve bands 10% length gene. <br />                                                                              Sixty-five ORFs contained insertions apparently limited two <br /> primers average resolution limit electro- <br />                                                                              locations coding region. This group includes both <br /> phoresis gels &#11015;4 kb. <br />                                                                              genes known essential (e.g., secA; ref. 9) those <br />   Each PCR product mutagenized vitro transposition <br /> in reactions containing hyperactive C9 mutant                     demonstrated nonessential (such hktE gene <br /> Himar1 transposase mariner-based minitransposon                        encoding catalase; ref. 18) and, therefore, information <br /> (Tnmagellan4) carrying H. influenzae DNA uptake signal                    considered equivocal. Genes encoding untranslated RNA <br /> and gene encoding kanamycin resistance. Mutagenized DNA                    scored. Small genes (&#11015;300 bp less) scored <br /> was transformed H. influenzae selected colony                   unless located sufficiently close analysis primers <br /> formation BHIs agar containing kanamycin (Methods). The                   allow accurate resolution insertion site positions. <br /> majority (351&#20862;366) reactions yielded 80                      The overall numbers results distribution known <br /> 5,000 colonies average 564. Six reactions yielded                 vs. unknown functional categories listed Table 1. Zero <br /> fewer 50 colonies 9 target PCR products differed               insertion detected 38% 15%, respectively, of <br /> in size predicted segments based genome                      total 1,272 ORFs scored. Thus, identified 670 <br /> sequence omitted. Variation numbers colonies                      putative essential genes 478 genes contained no <br /> obtained, part, ref lects 10-kb stretches DNA                insertions detected method. The complete list of <br /> contain different proportions essential nonessential                  putative essential genes available Table 3, is <br /> genes. For example, 4 6 regions yielded fewer               published supporting information PNAS web site, <br /> than 50 colonies analyzed means adjacent overlap-                 www.pnas.org. <br />  <br /> 968 &#20841; www.pnas.org&#20862;cgi&#20862;doi&#20862;10.1073&#20862;pnas.012602299                                                                                    Akerley et al. <br />  Table 2. H. influenzae putative essential genes conserved M. tuberculosis absent S. cerevisae <br /> Hin          Mtu     Bsu     Mge       Cpn     Ctr     Nme       Cje     Hpy     Eco      Pmu      Pae      Vch     Bbu      Tpa               PROSITE <br />  <br /> HI0004           &#11001;   &#11001;        &#11001;E        &#11001;       &#11001;        &#11001;        &#11001;       &#11001;       &#11001;N       &#11001;       &#11001;         &#11001;       &#11001;        &#11001;      UPF0054 <br /> HI0175           &#11001;   &#11001;N                 &#11001;       &#11001;        &#11001;        &#11001;               &#11001;        &#11001;       &#11001;         &#11001;       &#11001; <br /> HI0303           &#11001;   &#11001;                                   &#11001;        &#11001;       &#11001;       &#11001;        &#11001;       &#11001;         &#11001;       &#11001;        &#11001; <br /> HI0338           &#11001;   &#11001;                                   &#11001;                        &#11001;        &#11001;       &#11001;         &#11001;       &#11001;               Predicted transmembrane <br /> HI0466           &#11001;                                       &#11001;                        &#11001;        &#11001;       &#11001;         &#11001; <br /> HI0570           &#11001;                                                                &#11001;        &#11001;       &#11001;         &#11001; <br /> HI0767           &#11001;   &#11001;N                 &#11001;       &#11001;        &#11001;        &#11001;       &#11001;       &#11001;N       &#11001;       &#11001;         &#11001;       &#11001;        &#11001;      N6&#14061;MTASE <br /> HI1086           &#11001;                                       &#11001;                        &#11001;        &#11001;       &#11001;         &#11001;                       PROKAR&#14061;LIPOPROTEIN <br /> HI1282           &#11001;   &#11001;N                                  &#11001;        &#11001;       &#11001;       &#11001;N       &#11001;       &#11001;         &#11001;       &#11001;        &#11001; <br /> HI1608           &#11001;   &#11001;E                         &#11001;        &#11001;        &#11001;       &#11001;       &#11001;E       &#11001;       &#11001;         &#11001;                &#11001; <br /> HI1654           &#11001;   &#11001;N       &#11001;E                         &#11001;        &#11001;       &#11001;       &#11001;N       &#11001;       &#11001;N        &#11001;       &#11001;        &#11001;      UPF0011 <br /> HI1731           &#11001;                                       &#11001;        &#11001;               &#11001;                &#11001;         &#11001; <br />  <br />    Presence (&#11001;) absence (blank) homologs putatively essential H. influenzae proteins range bacteria. Homologs H. influenzae proteins <br /> included excluded table yielded PAM250 DP values &#11022; 80 P values &#65533;?10&#11002;3 based precomputed BLASTP comparisons 37 microbial genomes <br /> (http:&#20862;&#20862;mbgd.genome.ad.jp&#20862;). Growth phenotypes previously determined mutants deficient homologs species (see text) summarized as <br /> essential (E) nonessential (N) data available. H. influenzae (hin), M. tuberculosis (mtu), B. subtilis (bsu), <span id='am-271' about='obo:NCBITaxon_810' typeof='obo:NCBITaxon_204428 owl:Thing obo:NCBITaxon_204429 obo:NCBITaxon_809 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_1113537 obo:NCBITaxon_51290 obo:NCBITaxon_51291'><span id='am-272' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-273' property="oboInOwl:hasRelatedSynonym" content="Rickettsiaformis" datatype="xsd:string"></span><span id='am-274' property="oboInOwl:hasDbXref" content="PMID:21048221" datatype="xsd:string"></span><span id='am-275' property="oboInOwl:hasDbXref" content="PMID:9103637" datatype="xsd:string"></span><span id='am-276' property="rdfs:label" content="Chlamydia" datatype="xsd:string"></span><span id='am-277' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-278' property="oboInOwl:hasRelatedSynonym" content="Miyagawanella" datatype="xsd:string"></span><span id='am-279' property="oboInOwl:hasDbXref" content="PMID:9103632" datatype="xsd:string"></span><span id='am-280' property="oboInOwl:hasDbXref" content="PMID:11211265" datatype="xsd:string"></span><span id='am-281'  ></span><span id='am-282' property="oboInOwl:hasRelatedSynonym" content="Rakeia" datatype="xsd:string"></span><span id='am-283' property="oboInOwl:hasDbXref" content="PMID:21048222" datatype="xsd:string"></span><span id='am-284' property="oboInOwl:hasDbXref" content="PMID:10319462" datatype="xsd:string"></span><span id='am-285' property="oboInOwl:hasRelatedSynonym" content="Bedsonia" datatype="xsd:string"></span>Chlamydia</span> pneumoniae (cpn), Chlamydia <br /> trachomatis (ctr), N. meningitidis (nme), <span id='am-251' about='obo:NCBITaxon_197' typeof='owl:Thing obo:NCBITaxon_213849 obo:NCBITaxon_72294 obo:NCBITaxon_1 obo:NCBITaxon_68525 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_194 obo:NCBITaxon_29547'><span id='am-252' property="rdfs:label" content="Campylobacter jejuni" datatype="xsd:string"></span><span id='am-253' property="oboInOwl:hasRelatedSynonym" content="Vibrio hepaticus" datatype="xsd:string"></span><span id='am-254'  ></span><span id='am-255' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-256' property="oboInOwl:hasDbXref" content="PMID:11321120" datatype="xsd:string"></span><span id='am-257' property="oboInOwl:hasRelatedSynonym" content="Vibrio jejuni" datatype="xsd:string"></span><span id='am-258' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-259' property="oboInOwl:hasRelatedSynonym" content="Ampylobacter jejuni" datatype="xsd:string"></span><span id='am-260' property="oboInOwl:hasRelatedSynonym" content="Campylobacter fetus subsp. jejuni" datatype="xsd:string"></span>Campylobacter jejuni</span> (cje), H. pylori (hpy), E. coli (eco), <span id='am-303' about='obo:NCBITaxon_745' typeof='owl:Thing obo:NCBITaxon_135625 obo:NCBITaxon_712 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236'><span id='am-304' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-305'  ></span><span id='am-306' property="rdfs:label" content="Pasteurella" datatype="xsd:string"></span><span id='am-307' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-308' property="oboInOwl:hasDbXref" content="PMID:1736960" datatype="xsd:string"></span>Pasteurella</span> multocida (pmu), P. aeruginosa (pae), V. cholerae <br /> (vch), <span id='am-229' about='obo:NCBITaxon_139' typeof='owl:Thing obo:NCBITaxon_203691 obo:NCBITaxon_1 obo:NCBITaxon_64895 obo:NCBITaxon_203692 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_137 obo:NCBITaxon_136 obo:NCBITaxon_138'><span id='am-230' property="oboInOwl:hasExactSynonym" content="Lyme disease spirochete" datatype="xsd:string"></span><span id='am-231' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-232' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-233' property="oboInOwl:hasRelatedSynonym" content="Borrelia burgdorffragment" datatype="xsd:string"></span><span id='am-234' property="oboInOwl:hasDbXref" content="PMID:8995796" datatype="xsd:string"></span><span id='am-235' property="oboInOwl:hasDbXref" content="PMID:7981102" datatype="xsd:string"></span><span id='am-236' property="oboInOwl:hasExactSynonym" content="Borrelia burgdorferi sensu stricto" datatype="xsd:string"></span><span id='am-237' property="oboInOwl:hasRelatedSynonym" content="Borrelia burdorferi" datatype="xsd:string"></span><span id='am-238'  ></span><span id='am-239' property="oboInOwl:hasDbXref" content="PMID:1380285" datatype="xsd:string"></span><span id='am-240' property="oboInOwl:hasDbXref" content="PMID:7699027" datatype="xsd:string"></span><span id='am-241' property="oboInOwl:hasDbXref" content="PMID:1503971" datatype="xsd:string"></span><span id='am-242' property="oboInOwl:hasDbXref" content="PMID:9336916" datatype="xsd:string"></span><span id='am-243' property="rdfs:label" content="Borrelia burgdorferi" datatype="xsd:string"></span>Borrelia burgdorferi</span> (bbu), <span id='am-211' about='obo:NCBITaxon_160' typeof='owl:Thing obo:NCBITaxon_203691 obo:NCBITaxon_1 obo:NCBITaxon_203692 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_137 obo:NCBITaxon_136 obo:NCBITaxon_157'><span id='am-212'  ></span><span id='am-213' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-214' property="rdfs:label" content="Treponema pallidum" datatype="xsd:string"></span><span id='am-215' property="oboInOwl:hasRelatedSynonym" content="Spirillum pallidum" datatype="xsd:string"></span><span id='am-216' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-217' property="oboInOwl:hasRelatedSynonym" content="Spirochaeta pallida" datatype="xsd:string"></span><span id='am-218' property="oboInOwl:hasRelatedSynonym" content="Spironema pallidum" datatype="xsd:string"></span><span id='am-219' property="oboInOwl:hasRelatedSynonym" content="Microspironema pallidum" datatype="xsd:string"></span>Treponema pallidum</span> (tpa). <br />  <br />  <br />  <br /> Genome Analysis Mapping Vitro Transposition (GAMBIT)                     Essential genes unknown function this <br /> Analysis Gene Products Homologous Genes Assigned                        category important characterize better understand <br /> Functions Other Organisms. Many putative essential                       microbial cell biology defining optimal bacterial <br /> genes detected H. influenzae essential homologs                      targets development antimicrobial therapeutic <br /> other organisms. These include cell division proteins, murein                      agents. We determined putatively essential gene prod- <br /> synthesis enzymes, enzymes one-carbon metabolism,                           ucts unknown functions conserved a <br /> many (19 &#8211;21) (Table 3). Essentiality gene                       phylogenetically distant pathogen, <span id='am-76' about='obo:NCBITaxon_1773' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1762 obo:NCBITaxon_1763 obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003 obo:NCBITaxon_77643'><span id='am-77' property="rdfs:label" content="Mycobacterium tuberculosis" datatype="xsd:string"></span><span id='am-78'  ></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium tuberculosis typus humanus" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="Bacterium tuberculosis" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium tuberculosis var. hominis" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasRelatedSynonym" content="Bacillus tuberculosis" datatype="xsd:string"></span>Mycobacterium tuberculosis</span>. <br /> products initially unexpected based functional                          To exclude proteins potentially present in <br /> annotation provided genome sequence. HI0119                           humans, homologs S. cerevisiae removed. <br /> detected essential gene screen, annotated                     Absence proteins humans evaluated TBLASTN <br /> as fimA, encoding major subunit fimbrial surface                          comparisons list (Table 2) human genome <br /> structure nonessential growth organisms.                       sequence database (September 2001). Table 2 notes roles in <br /> However, HI0119 shown Lu et al. (22) encode                            viability conserved genes (homologs HI0004, <br /> protein critical zinc transport. Deletion ORF H.                    HI0767, HI1282, HI1608, HI1654) exam- <br /> influenzae confers severe growth defect aerobic con-                       ined B. subtilis (ref. 4; http:&#20862;&#20862;locus.jouy.inra.fr&#20862;cgi-bin&#20862; <br /> ditions absence supplementation high con-                         genmic&#20862;madbase&#14061;home.pl), E. coli (27), M. genitalium (6), or <br />  <br />  <br />  <br />  <br />                                                                                                                                                                       MICROBIOLOGY <br /> centration zinc (22). Large-scale functional screens                    <span id='am-172' about='obo:NCBITaxon_286' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_135621'><span id='am-173' property="oboInOwl:hasDbXref" content="PMID:10939664" datatype="xsd:string"></span><span id='am-174' property="oboInOwl:hasRelatedSynonym" content="Loefflerella" datatype="xsd:string"></span><span id='am-175' property="oboInOwl:hasRelatedSynonym" content="RNA similarity group I" datatype="xsd:string"></span><span id='am-176' property="oboInOwl:hasDbXref" content="PMID:9103607" datatype="xsd:string"></span><span id='am-177' property="oboInOwl:hasRelatedSynonym" content="Liquidomonas" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:hasDbXref" content="PMID:23918787" datatype="xsd:string"></span><span id='am-179' property="oboInOwl:hasDbXref" content="PMID:7727274" datatype="xsd:string"></span><span id='am-180' property="oboInOwl:hasDbXref" content="PMID:15950132" datatype="xsd:string"></span><span id='am-181' property="oboInOwl:hasDbXref" content="PMID:10758879" datatype="xsd:string"></span><span id='am-182' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-183' property="oboInOwl:hasDbXref" content="PMID:18048745" datatype="xsd:string"></span><span id='am-184' property="oboInOwl:hasRelatedSynonym" content="Peudomonas" datatype="xsd:string"></span><span id='am-185' property="rdfs:label" content="Pseudomonas" datatype="xsd:string"></span><span id='am-186'  ></span><span id='am-187' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Pseudomonas</span> aeruginosa (17). Both putatively essential <br /> GAMBIT aid reconciling genome annotation                               H. influenzae genes list conserved M. <br /> subsequent experimental data literature detecting                        genitalium potentially essential M. genitalium. Homologs <br /> such apparent inconsistencies.                                                     HI1608 gene product essential other <br />    We detected putatively essential genes H. influenzae                    organisms examined, B. subtilis E. coli. <br /> that nonessential bacteria. The smpB gene H.                       This gene characterized independently other <br /> influenzae identified essential screen                        groups course study shown encode a <br /> nonessential E. coli B. subtilis (23, 24). Transposon                       component essential isoprenoid biosynthesis pathway in <br /> insertions smpB M. pneumoniae                             E. coli (27&#8211;29). It possible genes identified our <br /> detected (6). SmpB essential activity tmRNA (23,                     analysis encode previously unrecognized components sim- <br /> 25). The tmRNA functions mRNA tRNA                             ilarly characterized cellular pathways. Conversely, highly <br /> adds short peptide tag prematurely terminated translation                     conserved products gene HI1654 putatively <br /> products, targeting degradation. Although tmRNA                        essential Mycoplasma spp. B. subtilis, E. coli, or <br /> nonessential E. coli essential viability <span id='am-161' about='obo:NCBITaxon_482' typeof='owl:Thing obo:NCBITaxon_481 obo:NCBITaxon_28216 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_206351'><span id='am-162' property="oboInOwl:hasRelatedSynonym" content="Nesseira" datatype="xsd:string"></span><span id='am-163'  ></span><span id='am-164' property="oboInOwl:hasDbXref" content="PMID:7520730" datatype="xsd:string"></span><span id='am-165' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasRelatedSynonym" content="Gonococcus" datatype="xsd:string"></span><span id='am-167' property="oboInOwl:hasRelatedSynonym" content="&quot;Merismopedia&quot; Zopf 1885" datatype="xsd:string"></span><span id='am-168' property="oboInOwl:hasRelatedSynonym" content="Neisseria Trevisan 1885" datatype="xsd:string"></span><span id='am-169' property="oboInOwl:hasRelatedSynonym" content="&quot;Gonococcus&quot; Lindau 1898" datatype="xsd:string"></span><span id='am-170' property="rdfs:label" content="Neisseria" datatype="xsd:string"></span><span id='am-171' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Neisseria</span>                 P. aeruginosa. Products HI1654, therefore, participate <br /> gonorrhoeae (26). H. influenzae N. gonorrhoeae                        potential essential pathway bacteria <br /> obligate human parasites relatively small genomes (&#11015;1.8                       produce functionally redundant components. Although com- <br /> and &#11015;2.2-Mbp, respectively) potentially lack factors                     parisons kind subject specific experimental <br /> present E. coli suppress growth defect smpB                      conditions methods used, results indicate the <br /> mutant.                                                                            requirement given gene viability varies between <br />                                                                                    organisms. <br /> Identification Putative Essential Genes Unknown Functions. <br /> Comparison list essential genes (Table 3) current                   Discussion <br /> genome annotation revealed 259 putatively essential genes                          Based results, estimate 38% H. influenzae genes <br /> lacking defined functional roles. At 159                       critical growth viability colonies rich medium. <br /> initially conserved organisms (14),                         Putatively essential genes likely been <br /> number increases additional genomes sequenced.                         previously characterized genes nonessential functions. <br /> Genes conserved evolutionarily divergent genera                        In cases, putatively essential genes identified <br /> potentially encode components basic cellular processes.                         nonessential organisms. Studies H. influenzae can <br />  <br /> Akerley et al.                                                                                              PNAS &#20841; January 22, 2002 &#20841; vol. 99 &#20841; no. 2 &#20841; 969 <br />  potentially detect essential cellular pathways                        targets narrow-spectrum antibiotics (31). However, as <br /> alternate mechanisms organisms larger genomes. Alter-                           genomes sequenced, likely fewer genes will <br /> natively, H. influenzae contain additional factors be-                         considered organism-specific. Although directed ap- <br /> come lethal essential gene depleted. Moreover,                              proaches examining single genes produce high <br /> organisms larger genome sizes compared with                             confidence level given gene needed growth, low <br /> smaller coding capabilities increase gene func-                          output essential gene candidates makes finding ideal can- <br /> tions redundant given growth condition.                               didates likely. To obtain reliable assessment of <br />    An estimate essential gene content 55&#8211;73%                                  critical biological pathways organism requires studies <br /> generated Mycoplasma spp. based large-scale sequenc-                           natural environment growth pathogen <br /> ing project identified genes containing transposon inser-                          infected host. Our studies identified com- <br /> tions (6). Two small-scale surveys examining total 53                              prehensive list factors essential vitro a <br /> selected genes E. coli indicated lower proportions                         standard growth condition. These results provide information <br /> (&#11015;22%) genes examined essential (6&#20862;27 6&#20862;26,                              used identify genes essential during <br /> respectively) (27, 30). In fact, true proportion E. coli                         infection. We completed functional genomic analysis of <br /> could potentially lower authors chose targets                          H. influenzae provide publicly available database of <br /> conserved divergent bacterial species attempt favor                         putatively essential nonessential genes organism <br /> detection essential genes. Analysis chromosome V S.                            (see Table 3). Detailed analysis gene required to <br /> cerevisiae indicated &#11015;20% genes examined                                confirm specific functions, list provides starting <br /> essential (5). The causes presence greater propor-                         point targeted analysis subsets genes interesting to <br /> tion essential genes H. influenzae Mycoplasma spp.                            investigators examining critical biological processes. For ex- <br /> are likely complex. It likely obligate                      ample, kinetic approach monitoring loss mutants from <br /> parasites reliant nutrients host,                        pool directly vitro transposon mutagenesis can <br /> fewer biosynthetic capabilities overlapping functions                              provide rapid means initial verification essentiality (5, <br /> compared bacteria environmental niches                                32). In addition, insertions specific DNA regions be <br /> water soil. It possible compare H.                               precisely mapped PAGE examine genes were <br /> influenzae data list &#11015;150 critical genes detected                        evaluated (17, 33). This study integrates genomic sequence <br /> the Gram-positive pathogen, S. aureus, exact gene                              data genome-scale mutagenesis. Other global analyses <br /> identifiers genes mediating antisense RNA-inhibited phe-                            genomic expression profiling combined with <br /> notypes bacterium available (8). However,                          results define genetic functions. Genome <br /> important note experiments diverse bacteria                          analysis mapping vitro transposition (GAMBIT) and <br /> were conducted different culture conditions growth                             genetic footprinting results study essenti- <br /> media, factors predicted major impact require-                             ality bacterial genes cultures grown complex medium <br /> ments viability.                                                                    does follow simple patterns. Essentiality predicted <br />                                                                                         exclusively conservation species, genes needed in <br />    A number &#8216;&#8216;postgenomic&#8217;&#8217; approaches ad- <br />                                                                                         organism necessarily needed another. However, <br /> vanced refine list potential targets antimicrobial <br />                                                                                         comparing essential gene databases divergent organisms can <br /> drug development small numbers genes be <br />                                                                                         used prioritize genetic bioch <br /> </body></html>