<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>           Identification of the FKS1 Gene of <span id='am-34' about='obo:NCBITaxon_5475' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_118267 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-35'  ></span><span id='am-36' property="rdfs:label" content="Candida" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Torulopsis" datatype="xsd:string"></span>Candida</span> albicans as the <br />              Essential Target of 1,3-b-D-Glucan Synthase Inhibitors <br />  <br />              Pneumocandins and echinocandins are fungicidal antibiotics, currently in clinical development, that inhibit <br />           1,3-b-D-glucan synthase (GS) in several human fungal pathogens. We have identified a gene from the diploid <br />           organism <span id='am-40' about='obo:NCBITaxon_5476' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_5475 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_118267 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Candida albicans" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasAlternativeId" content="NCBITaxon:39399" datatype="xsd:string"></span><span id='am-44'  ></span><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Candida stellatoidea" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="GC_ID:12" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasRelatedSynonym" content="Candida stellatoidea type I" datatype="xsd:string"></span>Candida albicans</span> that encodes a target of these inhibitors. A 2.1-kb portion of <span id='am-56' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-57' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-60'  ></span>this</span> gene, designated <br />           CaFKS1, has significant homology to the <span id='am-1' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-2'  ></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> FKS1 and FKS2 genes, which encode partially <br />           functionally redundant subunits of GS. To evaluate the role of CaFks1p in susceptibility to echinocandins, we <br />           disrupted CaFKS1 on one homolog each of the spontaneous pneumocandin-resistant C. albicans mutants <br />           CAI4R1, NR2, NR3, and NR4. These mutants had been selected previously on agar plates containing the <br />           pneumocandin L-733,560. The clones derived from this transformation were either resistant (Echr) or fully <br />           sensitive (Echs) to inhibition by L-733,560 in both liquid broth microdilution and in vitro GS assays. The site <br />           of plasmid insertion in the transformants was mapped by Southern blot analysis, using restriction site <br />           polymorphisms in the CaFKS1 gene to distinguish between the two alleles (designated CaFKS1h and CaFKS1b). <br />           For strains CAI4R1 and NR2, the CaFKS1b allele was disrupted in each Echr transformant; for strain NR4, <br />           CaFKS1h was disrupted in each Echr transformant. We conclude that (i) strains CAI4R1, NR2, and NR4 are <br />           heterozygous for a dominant or semidominant pneumocandin resistance mutation at CaFKS1, (ii) drug <br />           resistance mutations can occur in either CaFKS1 allele, and (iii) CaFks1p is a target of the echinocandins. For <br />           transformants of strain NR3, all the clones we analyzed were uniformly Echr, and only the CaFKS1h allele, <br />           either in disrupted or wild-type form, was detected on genomic Southern blots. We believe gene conversion at <br />           the CaFKS1 locus may have produced two Cafks1h alleles that each contain an Echr mutation. Transformants <br />           derived from the mutants were analyzed for susceptibility to pneumocandin treatment in a mouse model of <br />           disseminated candidiasis. Strains heterozygous for the resistant allele (i.e., C. albicans CAI4R1, NR2, and NR4) <br />           were moderately resistant to treatment, while strains without a functional Echs allele (i.e., strain NR3 and <br />           derivatives of strain CAI4R1 with the disruption plasmid integrated in the Echs allele) displayed strong in vivo <br />           echinocandin resistance. Finally, we were unable to inactivate both alleles at CaFKS1 by two-step integrative <br />           disruption, suggesting that CaFks1p is likely to be an essential protein in C. albicans. <br />  <br />  <br />    Semisynthetic derivatives of pneumocandin B0 (MK0991;                     following model has emerged. S. cerevisiae has two genes, FKS1 <br /> formerly L-743,872) and echinocandin B (LY303366) are li-                    and FKS2, which encode highly homologous, large integral <br /> popeptide antifungals that are currently in clinical trials for the          membrane proteins that function as the catalytic subunit of GS <br /> treatment of systemic fungal infections (5, 33). These com-                  (11, 22, 37). A small GTP-binding subunit, Rho1p, is also <br /> pounds inhibit the synthesis of 1,3-b-D-glucan, which is an                  necessary for activity (13, 20, 24, 36, 41). FKS1 expression is <br /> essential polysaccharide in the walls of many, if not all, fungal            cell cycle regulated and more abundant during vegetative <br /> pathogens (3, 10, 18). The broader-spectrum semisynthetic                    growth, while FKS2 expression is calcineurin dependent and <br /> derivatives have increased potency in vitro and in animal mod-               important for efficient sporulation (37). Mutations conferring <br /> els of fungal infection, and the improved potency is a function              high-level resistance to lipopeptides (the Echr phenotype) map <br /> of increased activity against 1,3-b-D-glucan synthase (GS) (3,               to FKS1 or FKS2, and the GS complexes containing either <br /> 30). Despite widespread interest in this enzyme target for an-               Fks1p or Fks2p are sensitive to inhibition by lipopeptides (11, <br /> tifungal therapy, many questions remain about the components                 37). The mutant alleles confer resistance to GS inhibitors, both <br /> of the synthetic complex, the mechanism of echinocandin in-                  in whole cells and in assays of GS activity, and the resistance <br /> hibition, and the potential for resistance in the fungal patho-              phenotype is semidominant in diploids. The role for Fks1p in <br /> gens.                                                                        glucan synthesis was further supported by Inoue et al. (22), <br />    The susceptibility of <span id='am-18' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-23'  ></span><span id='am-24' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae to GS inhibi-              who identified the gene products of FKS1 and FKS2 in enzyme <br /> tors has been used to identify and characterize genes that play              preparations enriched for GS activity. As would be expected <br /> a role in 1,3-b-D-glucan synthesis (9, 11, 12, 15). Based on                 for an essential cellular function, mutants can survive with <br /> studies of resistant mutants (9, 11, 12, 15), gene disruption                either FKS1 or FKS2 alone but disruption of both genes is <br /> experiments (11, 22, 37), and biochemical characterization, the              lethal (22, 37). <br />                                                                                 By using a portion of S. cerevisiae FKS1 as a probe, genes <br />                                                                              with strong homology were identified from the fungal patho- <br />    * Corresponding author. Mailing address: Department of Biochem- <br /> istry, Merck Research Laboratories RY80Y-220, P.O. Box 2000, Rah-            gens Candida albicans, <span id='am-28' about='obo:NCBITaxon_5052' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_5042 obo:NCBITaxon_33154 obo:NCBITaxon_147545 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_1131492 obo:NCBITaxon_451871 obo:NCBITaxon_2759'><span id='am-29'  ></span><span id='am-30' property="rdfs:label" content="Aspergillus" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasAlternativeId" content="NCBITaxon:37503" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Aspergillus</span> fumigatus, and Cryptococcus <br /> way, NJ 07065-0900. Phone: (732) 594-5124. Fax: (732) 594-1399. E-           neoformans (11). The homolog fksA has recently been cloned <br /> mail: Myra_Kurtz@Merck.com.                                                  from <span id='am-66' about='obo:NCBITaxon_162425' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_5042 obo:NCBITaxon_5052 obo:NCBITaxon_33154 obo:NCBITaxon_147545 obo:NCBITaxon_451864 obo:NCBITaxon_147538 obo:NCBITaxon_4890 obo:NCBITaxon_716546 obo:NCBITaxon_716545 obo:NCBITaxon_1131492 obo:NCBITaxon_451871 obo:NCBITaxon_2759'><span id='am-67' property="oboInOwl:hasRelatedSynonym" content="Aspergillus nidulellus" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasAlternativeId" content="NCBITaxon:5072" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="A. nidulans" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="Emericella nidulans" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="Aspergilus nidulans" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="Aspergillus nidulans" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-75'  ></span>Aspergillus nidulans</span> (25). The deduced amino acid se- <br />    &#8224; Present address: Chrysalis Corporation, Princeton, NJ 08544.            quence for FksAp is significantly conserved with the S. cerevi- <br />  <br />                                                                       2471 <br />  2472       DOUGLAS ET AL.                                                                                                    ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br /> siae Fks proteins, particularly in a central domain which is                            DNA was transformed into C. albicans by either the spheroplast method (29) <br /> predicted to be hydrophilic and on the cytoplasmic face of the                       or the lithium acetate method (14). Transformants selected on SC Ura2 medium <br />                                                                                      were purified and tested for growth on YPDA agar plates containing L-733,560 <br /> protein. To explore the role of FksAp in glucan synthesis, Kelly                     at 0.5 mg/ml. <br /> et al. (25) raised antisera against synthetic peptides designed                         DNA sequence analysis was performed as per the manufacturer&#8217;s directions, <br /> from the FksAp sequence and demonstrated that a cross-re-                            using Sequenase v. 2.0 (U.S. Biochemicals), [32P]dCTP (Amersham) and oligo- <br /> acting protein with the expected molecular weight was de-                            nucleotide primers synthesized at Merck or purchased from New England Bio- <br />                                                                                      labs or Gibco BRL. Software from Mac Vector (Oxford Molecular Group) and <br /> tected and strongly enriched during partial purification of GS                       the Genetics Computer Group was used to analyze DNA sequences. <br /> from A. nidulans. This enzyme activity was susceptible to li-                           Northern blotting was performed according to the manufacturer&#8217;s directions, <br /> popeptide inhibition, and the hyphae of Aspergillus spp. be-                         using mRNA prepared with the polyATract System (Promega). Samples were <br /> come distended and highly branched, both in vitro (32) and in                        separated on a 1% agarose gel containing formaldehyde, transferred to a nylon <br />                                                                                      membrane (Amersham), and probed with a DNA fragment radiolabeled with <br /> vivo (28), when exposed to GS inhibitors. The results support                        [32P]dCTP by random priming. <br /> the <span id='am-48' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-49' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-52'  ></span>idea</span> that FksAp is both a component of the GS complex in                            Plasmid constructions. Total genomic DNA from C. albicans CAI4 was di- <br /> A. nidulans and a target of the echinocandins.                                       gested with KpnI and BamHI, and fragments were separated on a 0.6% agarose <br />    A recent in vitro selection for spontaneous L-733,560-resis-                      gel. DNA from one lane was probed for fragments homologous to a 1.3-kb <br />                                                                                      ClaI-SalI probe from the S. cerevisiae FKS1 gene (11). We used the signal from <br /> tant mutants of the C. albicans Ura2 strain CAI4 yielded rare                        the Southern blot as a guide to excise KpnI-BamHI fragments of the appropriate <br /> isolates whose phenotypes were similar to those of S. cerevisiae                     size from the gel and clone them into YEplac195 (17). E. coli clones transformed <br /> lipopeptide-resistant mutants (27). The whole-cell resistance                        with this minilibrary were pooled and probed with the mWave colony hybridiza- <br /> of one such mutant (strain CAI4R1) reflects resistance in en-                        tion kit (Invitrogen). Plasmid DNA was isolated from putative clones, and the <br />                                                                                      insert was identified by Southern blotting. The 2.1-kb KpnI-BamHI fragment <br /> zyme activity, and results from spheroplast fusion experiments                       from one of these clones (pGJ1) was isolated and subcloned into pUC19 (New <br /> are consistent with a semidominant mutation in a single gene                         England Biolabs) to yield pJAD1. <br /> (27). As with the S. cerevisiae mutants, the resistance pheno-                          Plasmid pJAD1DXH was derived from plasmid pJAD1 by ligating filled-in <br /> type is specific to GS inhibitors. To understand the nature of                       XbaI and HindIII sites to yield a vector with a unique SalI site within the insert <br />                                                                                      DNA. pJAD1DXH was digested with BamHI and phosphatase and ligated to the <br /> echinocandin resistance in these spontaneous mutants, and to                         URA3 gene carried on a 2.9-kb BamHI fragment from plasmid pMB-7 (16) to <br /> explore the role of the C. albicans homolog of FKS1 in <span id='am-61' about='obo:NCBITaxon_9822' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9821 obo:NCBITaxon_35497 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-62'  ></span><span id='am-63' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="Sus" datatype="xsd:string"></span>sus</span>-                          create <span id='am-118' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-119' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-120' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>plasmids</span> pAM600 and pAM601, which differ only in the orientation of the <br /> ceptibility to GS inhibitors, we cloned and sequenced a portion                      URA3 fragment. The disruption plasmid pAM602 was created by removing the <br /> of the homologous gene (CaFKS1) and used it for integrative                          0.6-kb SalI-BclI fragment from plasmid pJAD1DXH and replacing it with the <br />                                                                                      3.9-kb SalI-BglII fragment from plasmid pMB-7, which carries the hisG-URA3- <br /> disruption in several C. albicans Echr mutants. Our results                          hisG cassette. <br /> indicate that mutations in either CaFKS1 allele can confer in                           Plasmid pJAM15 (35), with the C. albicans URA3 gene inserted into vector <br /> vitro resistance to L-733,560, which identifies CaFks1p as a                         pBR322, was used to transform strains CAI4, CAI4R1, NR2, NR3, and NR4 to <br /> target of the echinocandins in C. albicans. All of the transfor-                     uridine prototrophy. <br />                                                                                         GS assays. We measured the in vitro synthesis of 1,3-b-D-glucan from UDP- <br /> mants retained the ability to produce a lethal infection in mice,                    [3H]glucose, using crude membranes prepared as described previously (12). The <br /> and disruption of the Echs allele correlated with significant                        assay measures the formation of radiolabeled trichloroacetic acid (TCA)-insol- <br /> resistance to treatment of infected animals with L-733,560. The                      uble material. Briefly, cells in logarithmic phase were harvested by centrifuga- <br /> absence of DNA highly homologous to CaFKS1 on Southern                               tion, washed, and broken with glass beads. Debris and unbroken cells were <br />                                                                                      removed by centrifugation at 4,000 3 g, and crude membranes were harvested <br /> blots of C. albicans total genomic DNA, along with our inabil-                       from the supernatant fraction by centrifugation at 100,000 3 g. The resulting <br /> ity to recover null mutations in both CaFKS1 alleles, suggests                       pellet was resuspended at 1/50 the original culture volume and stored in aliquots <br /> that there may be no functional redundancy for this gene in C.                       at 280&#176;C. For the GS assay, reaction mixtures incubated in microtiter dishes with <br /> albicans. The clinical implications of these results are dis-                        gentle agitation for 2 h were quenched with an equal volume of ice-cold 20% <br />                                                                                      TCA. Precipitated product was harvested onto glass-fiber filter mats (Pharmacia) <br /> cussed.                                                                              and washed with water in a 96-channel cell harvester (Cambridge Technology). <br />                                                                                      The mats were dried, sealed in bags with scintillation cocktail, and counted with <br />                         MATERIALS AND METHODS <br />                                                                                      a Betaplate scintillation counter (LKB/Wallac). The counting efficiency was <br />    Strains, media, antibiotics, and growth conditions. C. albicans CAI4 (16) was     estimated at 40%. <br /> a gift from W. Fonzi (Georgetown University). The Echr mutants CAI4R1, NR2,             In vivo analysis. Uridine prototrophic derivatives of wild-type and mutant <br /> NR3, and NR4 have been described previously (27). The semisynthetic pneu-            strains were tested as described previously (27) for virulence in mice. The 50% <br /> mocandin L-733,560 (6) was synthesized at Merck (Rahway, N.J.) and was shown         lethal doses (LD50s) were calculated by the Knudsen-Curtis method (26). Strains <br /> to be .95% pure by high-performance liquid chromatography. Amphotericin B,           were also tested for susceptibility to L-733,560 or amphotericin B treatment in a <br /> itraconazole, ketoconazole, nystatin, and 5-fluorocytosine were obtained from        disseminated candidiasis target-organ assay described previously (4). The 99% <br /> Sigma. Fluconazole was from the Merck chemical collection. Strains were grown        effective dose (ED99) was determined based on the reduction of the number of <br /> at 30&#176;C in yeast extract-peptone medium with adenine, with or without uridine        CFU/gram of kidney compared to sham-treated control mice as described pre- <br /> supplementation (YPDA and YPDAUd, respectively), or in a defined medium              viously (2). Five mice were used per group per experiment. All procedures were <br /> (SC Ura2) as described previously (27). Values for the MIC or for 50% inhibi-        performed in accordance with the highest standards for the humane handling, <br /> tion of growth by L-733,560 were determined by a liquid broth microdilution          care, and treatment of research animals and were approved by the Merck Insti- <br /> assay in either YPDA, YPDAUd, or Yeast Nitrogen Base (Difco) with 2%                 tutional Animal Care and Use Committee. The care and use of research animals <br /> glucose. Briefly, cells grown to mid-logarithmic phase were subcultured to an        at Merck meet or exceed all applicable local, national, and international laws and <br /> initial optical density at 600 nm (A600) of 0.01 and seeded in wells of microtiter   regulations. <br /> dishes that contained serial twofold dilutions of L-733,560. Growth inhibition          Nucleotide sequence accession number. The nucleotide sequence of the <br /> was determined after 24 h at 30&#176;C by eye (for MICs) or by measuring the A600         2,069-bp KpnI-BamHI fragment of CaFKS1 has been assigned GenBank acces- <br /> in a spectrophotometer (Tecan) and comparing the results to those for a control      sion no. AF027295. <br /> well containing no drug. <br />    DNA and RNA methods. All DNA manipulations were performed as described <br /> by Maniatis et al. (34). Wizard Preps (Promega) and kits from Qiagen, Inc., were                                       RESULTS <br /> used to isolate plasmids, and DNA fragments were purified with the Gene Clean <br /> Kit (Bio101). Enzymes for digestion, ligation, and dephosphorylation of DNA             A C. albicans homolog of S. cerevisiae FKS1. The Fks proteins <br /> were from Gibco BRL. Competent cells of <span id='am-85' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-86' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-91'  ></span><span id='am-92' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-99' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span>Escherichia coli</span> DH5a were used for <br /> transformations. <br />                                                                                      from S. cerevisiae (Fks1p and Fks2p) and A. nidulans (FksAp) <br />    Total genomic DNA was isolated from C. albicans strains by the method of          are large integral membrane proteins that share a highly ho- <br /> Holm et al. (21). For Southern blot analysis, probes prepared by random priming      mologous internal domain (domain I) which is predicted to be <br /> (Stratagene) with [32P]dCTP (Amersham) were hybridized to DNA that had               on the cytoplasmic side of the plasma membrane. Regions <br /> been digested with restriction enzymes and transferred to <span id='am-123' about='obo:NCBITaxon_743421' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_50638 obo:NCBITaxon_7438 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7434 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_34725 obo:NCBITaxon_50557'><span id='am-124' property="rdfs:label" content="Zeta" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-126'  ></span><span id='am-127' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Zeta</span> Probe GT mem- <br /> branes (Bio-Rad) with a PosiBlot pressure blotter (Stratagene) according to the <br />                                                                                      within domain I have homology to BcsAp, the catalytic subunit <br /> manufacturer&#8217;s directions. Blots were washed under conditions of high stringency     of <span id='am-76' about='obo:NCBITaxon_434' typeof='owl:Thing obo:NCBITaxon_433 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_28211 obo:NCBITaxon_204441'><span id='am-77' property="rdfs:label" content="Acetobacter" datatype="xsd:string"></span><span id='am-78'  ></span><span id='am-79' property="oboInOwl:hasRelatedSynonym" content="Mycoderma" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="Termobacterium" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasRelatedSynonym" content="Ulvina" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasRelatedSynonym" content="Acetimonas" datatype="xsd:string"></span>Acetobacter</span> xylinium cellulose synthase (25). We used the <br /> and analyzed by autoradiography.                                                     1.3-kb ClaI-SalI DNA fragment from FKS1, which encodes a <br />  VOL. 41, 1997                                                                                                         FKS1 GENE OF C. ALBICANS                    2473 <br />  <br />  <br />  <br />  <br />    FIG. 1. Alignment of CaFks1p and predicted Fks protein sequences from S. cerevisiae FKS1 (U12893), S. cerevisiae FKS2 (U16783), and A. nidulans fksA (U51272). <br /> GenBank accession numbers are given in parentheses. The 2,069-bp KpnI-BamHI CaFKS1 fragment from C. albicans CAI4 was analyzed by using software from the <br /> Genetics Computer Group. A 689-amino-acid uninterrupted reading frame was identified and aligned with other Fksp sequences by using the clustal program. Domain <br /> I of S. cerevisiae Fks1p begins at position 261 in this figure and extends beyond position 688. Conserved residues are boxed, and gaps are indicated with dashes. The <br /> regions shown in the alignment are as follows: S. cerevisiae Fks1p, residues 460 to 1147; S. cerevisiae Fks2p, residues 479 to 1166; and A. nidulans Fks1p, residues 492 <br /> to 1178. <br />  <br />  <br />  <br />  <br /> portion of the large cytoplasmic domain, to clone a KpnI-                              between the two CaFKS1 alleles at a HindIII site in the 59 <br /> BamHI fragment of the homologous gene from C. albicans and                             region (Fig. 2B). We used this restriction fragment length <br /> create plasmid pJAD1. As expected, the protein sequence of                             polymorphism to distinguish the alleles in subsequent experi- <br /> the uninterrupted reading frame predicted from this fragment                           ments. For our purposes, the 6.7-kb HindIII-HindIII fragment <br /> shows strong homology to Fks1p, Fks2p, and FksAp, particu-                             produced by digestion of genomic DNA from strain CAI4 with <br /> larly within domain I (Fig. 1). A Northern blot of poly(A) RNA                         HindIII and PvuII defines the CaFKS1h allele, while the 8.5-kb <br /> from C. albicans CAI4 was probed with the KpnI-BamHI frag-                             PvuII-HindIII fragment defines the CaFKS1b allele. <br /> ment from pJAD1, and a single transcript of ca. 7 kb was                                  Integrative transformation at the CaFKS1 locus of Echr <br /> detected (<span id='am-108' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-109' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-110'  ></span><span id='am-111' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span> not shown). We have chosen FKS1 as the name                             mutants. Strain CAI4R1 is a spontaneous Echr mutant derived <br /> for the open reading frame that contains this KpnI-BamHI                               from strain CAI4 as described previously (27). We used the <br /> fragment, given the significant degree of homology to members                          cloned fragment of CaFKS1 to characterize the mutation in <br /> of the FKS gene <span id='am-25' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-26' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> and the convention of using the same                            CAI4R1. Plasmid pAM600 (Fig. 2C), which contains the C. <br /> nomenclature for C. albicans and S. cerevisiae genes. We refer                         albicans URA3 gene, an internal fragment of CaFKS1, and the <br /> to it here as CaFKS1 to avoid confusion with the S. cerevisiae                         pUC19 backbone, was linearized within the CaFKS1 insert by <br /> gene.                                                                                  digestion with SalI and transformed into CAI4R1. Integration <br />    We used the CaFKS1 fragment in pJAD1 to map the gene in                             at the Cafks1-1 locus is expected to disrupt one of the two <br /> strain CAI4 by Southern blotting. The region between the                               alleles according to the scheme outlined in Fig. 2D and E. Five <br /> KpnI and BamHI sites of the plasmid clone and the genome                               of the eight Ura1 transformants we characterized were Echr, <br /> were colinear. Digestion of genomic DNA from strain CAI4                               while the other three were Echs. We mapped the site of inte- <br /> with several enzymes for which there were no sites within the                          gration for two Echr isolates (T25 and T26) and two Echs <br /> KpnI-BamHI fragment of pJAD1 produced only a single cross-                             isolates (T28 and T32) by Southern blotting. In Fig. 2A, the <br /> hybridizing band on autoradiograms; no additional fragments                            two alleles at the CaFKS1 locus of strains CAI4 and CAI4R1 <br /> with weak homology to the probe were visible (data not                                 and the transformants are distinguished by digestion with <br /> shown). However, digestion of genomic DNA with HindIII and                             PvuII and HindIII. For transformants T25 and T26, the <br /> PvuII generated two distinct fragments on Southern blots (Fig.                         6.7-kb HindIII-HindIII fragment is intact but the 8.5-kb <br /> 2A, lane 1), despite the fact that there were no sites for these                       PvuII-HindIII fragment is replaced by two novel fragments of <br /> enzymes within the KpnI-BamHI CaFKS1 fragment in pJAD1.                                7.9 and 6.1 kb. Transformants T28 and T32 retained the 8.5-kb <br /> The fragment pattern was consistent with a polymorphism                                PvuII-HindIII fragment, but the 6.7-kb HindIII-HindIII frag- <br />  2474       DOUGLAS ET AL.                                                                                                  ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br />                                                                                      than the values for T28 and T32. It was shown previously that <br />                                                                                      the inhibition curve for CAI4R1 was unlike the titration curves <br />                                                                                      described for S. cerevisiae Echr mutants or for the C. albicans <br />                                                                                      Echr mutant strain CA-2 (27). The IC50 of L-733,560 for the <br />                                                                                      enzyme from strain CAI4R1 was increased only fourfold rela- <br />                                                                                      tive to that from strain CAI4, but inhibition did not exceed <br />                                                                                      55%, even at the highest concentrations of drug we tested (27). <br />                                                                                         We used disruption with pAM600 to characterize the Cafks1 <br />                                                                                      alleles in three additional spontaneous L-733,560-resistant mu- <br />                                                                                      tants derived from C. albicans CAI4: strains NR2, NR3, and <br />                                                                                      NR4 (27) (Table 1). C. albicans NR2 and NR4, like strain <br />                                                                                      CAI4R1, yielded both Echr and Echs isolates when trans- <br />                                                                                      formed with pAM600. Strain NR2 was equivalent to CAI4R1 <br />                                                                                      in that the CaFKS1b allele was disrupted in all Echr Ura1 <br />                                                                                      transformants, and the Cafks1h allele was disrupted in all Echs <br />                                                                                      Ura1 transformants (data not shown). In contrast, when strain <br />                                                                                      NR4 was transformed with pAM600, loss of the resistance <br />  <br />  <br />  <br />  <br />     FIG. 2. Integration of plasmid pAM600 at the CaFKS1 locus of C. albicans <br /> CAI4R1. (A) Southern blot of pAM600 transformants. Genomic DNAs from a <br /> wild-type strain (CAI4 [lane 1]), the original Echr mutant strain (CAI4R1 [lane <br /> 2]), two Echr transformants (T25 [lane 3] and T26 [lane 4]), and two Echs <br /> transformants (T28 [lane 5] and T32 [lane 6]) were digested with PvuII and <br /> HindIII and probed with the 2.1-kb KpnI-BamHI CaFKS1 fragment from <br /> pJAD1. A portion of the autoradiogram is shown; arrows indicate the migration <br /> positions of plasmid pAM602 digested with KpnI (8.2 kb) and plasmid <br /> pJAD1DXH digested with BamHI (4.7 kb). (B) Schematic maps of the CaFKS1h <br /> and CaFKS1b alleles. Abbreviations for restriction enzyme cleavage sites are as <br /> follows: P, PvuII; H, HindIII; K, KpnI; S, SalI; and B, BamHI. The K, S, and B <br /> sites are included for reference only and should not be considered unique within <br /> the regions shown. Restriction fragments detected following digestion with PvuII <br /> and HindIII (circled) and probing with the KpnI-BamHI CaFKS1 fragment are <br /> illustrated with dashed lines. The stippled boxes depict the regions homologous <br /> to the probe. (C) Map of plasmid pAM600. The pUC19 vector sequences (      &#9632;    ), <br /> the 2.9-kb BamHI fragment (&#9632;) containing the C. albicans URA3 gene, and <br /> the 2.1-kb CaFKS1 KpnI-BamHI fragment (o) are shown. Arrows within the <br /> circle illustrate the directions of transcription of the URA3, CaFKS1, and E. coli <br /> bla genes. The plasmid was linearized by digestion with SalI (thick line inter- <br /> secting the circle) prior to transformation. (D and E) Chromosomal maps fol- <br /> lowing pAM600 integration at CaFKS1. The maps depict pAM600 integration at <br /> the CaFKS1b allele (D) and at the CaFKS1h allele (E). <br />  <br />  <br />  <br />  <br /> ment has been replaced by novel fragments of 6.1 and 5.8 kb. <br /> These fragment sizes are consistent with integration of <br /> pAM600, which has two internal PvuII sites, into the CaFKS1b <br /> allele of T25 and T26 and the Cafks1h-1 allele of T28 and T32 <br /> (Fig. 2D and E). <br />    To test the phenotypes of Cafks1::URA3/CaFKS1 transfor- <br /> mants T25, T26, T28, and T32, we performed a liquid broth                               FIG. 3. Susceptibilities of CAI4R1 transformants to pneumocandin L-733,560. <br /> microdilution assay and an in vitro GS assay (Fig. 3). Esti-                         (A) Strains T25 (E), T26 (F), T28 (&#8218;), T32 (&#338;), CAI4 (h), and CAI4R1 (&#9632;) <br /> mated values for 50% inhibition of growth by L-733,560 (Fig.                         were grown in YPDA and assayed as described in Materials and Methods. The <br /> 3A) were roughly 1,000-fold higher for transformants T25 and                         drug was prepared from a 10-mg/ml stock, and the final concentration ranged <br />                                                                                      from 6.1 3 1022 to 8.0 3 103 ng/ml. (B) Membranes were prepared from <br /> T26 (ca. 3,000 ng/ml) than for strains T28 and T32 (ca. 0.2 to                       mid-logarithmic-phase cells and glucan synthase activity was measured as de- <br /> 0.8 ng/ml). The original heterozygous mutant (CAI4R1) was                            scribed in Materials and Methods. The reaction mixtures contained 0.3 mg of <br /> also resistant, albeit to a slightly lesser extent. Figure 3B shows                  crude membrane protein per ml and 0.6 mM UDP-glucose with UDP-[3H]glu- <br /> the curves for L-733,560 inhibition of GS in crude membrane                          cose added to 25,000 dpm/nmol; final concentrations of L-733,560 were from <br />                                                                                      5.1 3 1022 to 1.0 3 105 nM. After 2 h at 22&#176;C, the TCA-insoluble product was <br /> preparations from each strain. Much like the results with whole                      harvested and radioactive counts were performed. The specific activity for each <br /> cells, the concentrations of L-733,560 that inhibit GS activity by                   enzyme is given in Table 3. The symbols in panel B are the same as those used <br /> 50% (IC50s) for strains T25 and T26 were ca. 1,000-fold higher                       in panel A; error bars show standard deviations. <br />  VOL. 41, 1997                                                                                                   FKS1 GENE OF C. ALBICANS                    2475 <br />  <br />  <br />  TABLE 1. Susceptibilities of CAI4R1 and pAM600 transformants <br />       of spontaneous Echr mutants NR2, NR3, and NR4 <br />                  Susceptibility to            No. of <br />                     L-733,560             transformants       Disrupted allele <br />  Strain                                                           in Echr <br />                      a <br />                 MIC           IC50                             transformants <br />                                          Echr       EchS <br />                (mg/ml)       (nM) <br />  <br /> CAI4R1            2          1,000         5          3          CaFKS1b <br /> NR2               4         40,000         5          3          CaFKS1b <br /> NR3             .16         30,000        75          0 <br /> NR4               8             40         6          2          CaFKS1h <br />   a <br />    The MIC was determined in YPDA medium and measured as outlined in <br /> Materials and Methods. <br />  <br />  <br />  <br />  <br /> phenotype among four Ura1 transformants correlated with <br /> disruption of the Cafks1b allele (data not shown). These data <br /> suggest that strains NR2 and NR4, like mutant CAI4R1, are <br /> heterozygous for a semidominant mutation conferring the Echr                         FIG. 5. Inhibition by L-733,560 of GS activity from mutant strains NR2 (F), <br /> phenotype and that one allele (Cafks1h) contains the mutation                     NR3 (&#9632;), and NR4 (&#338;). Reactions were performed as described in the legend to <br /> in strains NR2 and CAI4R1 while the other allele (Cafks1b)                        Fig. 3B, except the membrane protein concentration ranged from 0.4 to 0.8 <br />                                                                                   mg/ml, incubation was at 30&#176;C, UDP-[3H]glucose was at 7,500 dpm/nmol, and <br /> contains the mutation in strain NR4. With strain NR3, no Echs                     L-733,560 was titrated from 5.5 3 1021 to 4.3 3 104 nM. Results with the <br /> clones were recovered among 75 Ura1 pAM600 transfor-                              wild-type strain (CAI4 [h]) are also shown. The specific activity of each enzyme <br /> mants; i.e., all the isolates grew on SC Ura2 plates containing                   is given in Table 3. <br /> 0.5 mg of L-733,560 per ml. The CaFKS1 locus from three <br /> NR3(pAM600) clones and the original NR3 mutant was eval- <br /> uated by Southern blotting, and the results are shown in Fig. <br />                                                                                   was pneumocandin resistant, with increases in the IC50s rang- <br /> 4A. Restriction fragments of 5.8, 6.1, and 6.7 kb were detected <br />                                                                                   ing from 20-fold (NR4) to roughly 10,000-fold (NR2 and <br /> in PvuII-HindIII-digested genomic DNA from the transfor- <br />                                                                                   NR3). However, inhibition of GS activity from strains NR2 and <br /> mants, and a single 6.7-kb fragment was present in the digest of <br />                                                                                   NR4 reaches a maximum of ca. 55%, with a shallow slope over <br /> NR3 DNA. The faint thin line above this fragment is an arti- <br />                                                                                   a wide range of L-733,560 concentrations. This profile resem- <br /> fact from film development that was present in several lanes of <br />                                                                                   bles the curve for pneumocandin inhibition of CAI4R1 GS <br /> the autoradiogram (data not shown). These results are consis- <br />                                                                                   (Fig. 3B). In contrast, GS activity from strain NR3 was entirely <br /> tent with the genomic maps shown in Fig. 4B and C, which <br />                                                                                   resistant to inhibition with L-733,560 at concentrations below <br /> depict a CaFKS1 locus with a 59 HindIII site in both alleles <br />                                                                                   10 mM. The shape of this curve is very similar to those of the <br /> (strain NR3) (Fig. 4B), and pAM600-mediated integrative dis- <br />                                                                                   titration curves shown for inhibition of enzyme activity from <br /> ruption at this locus (Fig. 4C). <br />                                                                                   strains T25 and T26 (Fig. 3B). <br />    The inhibition of GS activity from strains NR2, NR3, and <br />                                                                                      We also evaluated the susceptibility of several transformants <br /> NR4 by L-733,560 is shown in Fig. 5. Relative to GS activity <br />                                                                                   and their parents to antifungal antibiotics currently in clinical <br /> from the wild-type strain CAI4, the enzyme from each mutant <br />                                                                                   use (Table 2). Amphotericin B, itraconazole, fluconazole, nys- <br />                                                                                   tatin, ketoconazole, and 5-fluorocytosine produced equivalent <br />                                                                                   zones of inhibition against all nine strains. Strains T25 and T26 <br />                                                                                   and the Ura1 pJAM15 transformants of strains NR2, NR3, <br />                                                                                   and NR4 were also resistant to several known GS inhibitors, <br />                                                                                   including echinocandin B, several pneumocandins, and dihy- <br />                                                                                   dropapulacandin (data not shown). This specificity of resis- <br />                                                                                   tance is characteristic of S. cerevisiae mutants selected for re- <br />                                                                                   sistance to GS inhibitors, such as strains R560-1C (12) and <br />                                                                                   pbr1-1 (9). <br />                                                                                      Two-step integrative transformation at CaFKS1. We sought <br />                                                                                   to determine whether CaFKS1 is an essential gene. Our strat- <br />                                                                                   egy was to transform Ura2 strains bearing one disrupted allele <br />                                                                                   at the CaFKS1 locus with a URA3-containing disruption plas- <br />                                                                                   mid that also targets CaFKS1 and determine the genotypes of <br />                                                                                   the Ura1 transformants by Southern blot analysis. If CaFKS1 <br />                                                                                   is not required for viability, we expected to recover clones with <br />                                                                                   disruptions of both copies of the gene. To disrupt the first <br />                                                                                   allele, strain CAI4 was transformed to Ura1 with plasmid <br />    FIG. 4. CaFKS1 locus in Echr transformants of mutant NR3. (A) Southern <br />                                                                                   pAM602, which had been digested with PvuII and KpnI to <br /> blot of DNAs from mutant strain NR3 (lane 1) and three pAM600 transformants       release the Cafks1::hisG-URA3-hisG insert. Clones were puri- <br /> (strains T82 [lane 2], T83 [lane 3], and T85 [lane 4]) digested with PvuII and    fied on SC Ura2, and genomic DNA was analyzed on Southern <br /> HindIII and probed with the 2.1-kb KpnI-BamHI CaFKS1 fragment from                blots (Fig. 6). Transformant T6 had a disruption of the <br /> pJAD1. The markers used are described in the legend to Fig. 2A. (B) Restriction <br /> map of homozygous Cafks1h locus from strain NR3. The symbols and features of <br />                                                                                   Cafks1b allele (CaFKS1h/Cafks1b::hisG-URA3-hisG); diges- <br /> the map are described in the legend to Fig. 2C. (C) Map of pAM600 integration     tion of genomic DNA from strain T6 with HindIII and PvuII <br /> at Cafks1h-3 of strain NR3.                                                       yielded 6.7- and 12-kb fragments. Several independent colonies <br />  2476       DOUGLAS ET AL.                                                                                                     ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br />                                TABLE 2. Susceptibilities of CAI4-derived strains to antifungal antibiotics in clinical use <br />  <br />                              Amt                                                      Diam of zone of inhibition (mm) with: <br />     Compound <br />                            (mg/spot)        CAI4         CAI4R1          T25         T26       T28      T32       NR2(Ura1)         NR3(Ura1)     NR4(Ura1) <br />  <br /> Amphotericin B                 5             19            16            17          17         17       17            16              18            18 <br /> Itraconazole                  10             20            18            19          20         19       18            17              18            20 <br /> Fluconazole                    5             25            20            20          20         24       21            22              20            19 <br /> Nystatin                      10             24            20            20          21         20       20            18              24            23 <br /> Ketoconazole                   5             33            31            33          32         32       31            31              29            30 <br /> 5-Fluorocytosine              10             16            14            16          16         15       14            14              16            16 <br />  <br />  <br />  <br />  <br /> were transferred to plates containing 5-fluoroorotic acid                                 ment patterns, but all had an intact 8.5-kb CaFKS1b allele. We <br /> (5FOA) and uridine to select for loss of the URA3 gene (16).                              obtained similar results with strain T6FOA1; among 13 trans- <br /> The Southern blot of HindIII-PvuII-digested DNA from one                                  formants, none had the pattern expected for a disruption of the <br /> Ura2 derivative (T6FOA1) revealed two fragments (6.7 and                                  functional 6.7-kb CaFKS1h allele (Fig. 6 and data not shown). <br /> 9.3 kb), consistent with loss of the URA3 gene but retention of                           Clones were obtained by both spheroplast and lithium acetate <br /> one hisG insertion (CaFKS1/Cafks1b::hisG). The CAI4 trans-                                transformation methods. Because we were able to isolate nu- <br /> formant T1 had a disruption of the CaFKS1h allele; genomic                                merous Cafks1::URA3/CaFKS1 transformants but no Cafks1:: <br /> DNA from T1 yielded 8.5- and 15-kb fragments upon digestion                               hisG/Cafks1::URA3 transformants, we propose that a func- <br /> with HindIII and PvuII (Fig. 6). The size of the insertion-                               tional CaFKS1 allele is essential for viability under these <br /> bearing allele was consistent with tandem integration of two                              growth and transformation conditions. <br /> cassettes (Cafks1h::hisG-URA3-hisG-URA3-hisG/CaFKS1b). This                                  Growth rates, virulence, and susceptibility to L-733,560 in <br /> structure was confirmed with an additional Southern blot; di-                             vivo. The reduced growth rate of S. cerevisiae fks1 null mutants <br /> gestion of DNA from strain T1 with each of three restriction                              and the poor growth and weak virulence of the C. albicans Echr <br /> enzymes that cut once in the 3.9-kb hisG-URA3-hisG cassette                               mutant M2 (27) confirm that changes in GS activity can be <br /> produced a 3.9-kb fragment that hybridized with a hisG probe                              detrimental to the cell. For strain CAI4R1, it was shown that <br /> (data not shown). A Ura2 derivative (T1FOA3) obtained fol-                                the spontaneous Echr mutation had no significant effect on <br /> lowing selection with 5FOA was also analyzed (Fig. 6), and the                            growth in vitro or on the ability to establish a disseminated <br /> fragment pattern (7.5- and 8.5-kb HindIII-PvuII fragments)                                infection in immunocompromised mice (27). Animals infected <br /> was consistent with retention of a single hisG cassette                                   with a Ura1 derivative of this heterozygous mutant required an <br /> (Cafks1h::hisG/CaFKS1b).                                                                  ED99 of L-733,560 approximately eightfold higher than that for <br />    A second round of targeted disruption at the Cafks1 locus                              animals infected with strain CAI4(Ura1) (27). We evaluated <br /> was performed by transforming plasmid pAM601, linearized by                               the consequences of single-allele disruption at the CaFKS1 <br /> digestion with SalI, into strains T6FOA1 and T1FOA3. Uri-                                 locus on both growth rate and GS activity in vitro and on the <br /> dine prototrophs were selected on SC Ura2, and genomic                                    LD50 and ED99 of L-733,560 in vivo (Table 3). The doubling <br /> DNA was analyzed on Southern blots. Among 22 transfor-                                    times of all strains we tested were between 80 and 110 min in <br /> mants of strain T1FOA3, 16 yielded the pattern expected for                               YPDAUd medium; there was no dramatic effect of the Echr <br /> the genotype Cafks1h::URA3/CaFKS1b (Fig. 6 and data not                                   mutations or loss of either the CaFKS1h or CaFKS1b allele on <br /> shown). The remaining transformants had more complex frag-                                the growth rate. The specific activity of GS from these strains <br />                                                                                           (Table 3) ranged from 13 to 55 nmol z mg21 z h21; for each <br />                                                                                           strain, the GS activity varied for replicate membrane prepara- <br />                                                                                           tions, such that differences of less than twofold should not be <br />                                                                                           considered significant. Therefore, cells with only a functional <br />                                                                                           CaFKS1b allele (T32) or CaFKS1h allele (T25) were not sig- <br />                                                                                           nificantly impaired in their ability to grow in liquid culture or <br />                                                                                           express GS activity. In an animal model of disseminated can- <br />                                                                                           didiasis, the LD50s ranged from a low of 4,500 CFU/mouse <br />                                                                                           [strain NR2(Ura1)] to a high of 330,000 CFU/mouse (strain <br />                                                                                           T6). Lethality relative to the functional diploid CAI4(Ura1) <br />                                                                                           was slightly reduced (less than twofold) when the CaFKS1h <br />                                                                                           allele was disrupted in strain T1, but disruption of the <br />                                                                                           CaFKS1b allele in strain T6 produced a nearly 65-fold reduc- <br />                                                                                           tion in the LD50. Among the heterozygous Echr mutants, <br />                                                                                           strain NR4(Ura1), with a mutant Cafks1b allele (Cafks1b-4/ <br />    FIG. 6. Southern blot of clones from two-step integration at the CaFKS1                CaFKS1h) was nearly 32-fold less virulent than strain CAI4 <br /> locus. DNA from each strain was analyzed as described in the legend to Fig. 2.            (Ura1), while the LD50s for two strains with wild-type <br /> pAM602 transformation of wild-type strain CAI4 (lane 1) produced transfor-                CaFKS1b alleles [CAI4R1(Ura1), Cafks1h-1/CaFKS1b; NR2 <br /> mants with plasmid integration in either the CaFKS1b allele (strain T6 [lane 2]) <br /> or the CaFKS1h allele (strain T1 [lane 10]). Clones selected for 5FOA resistance <br />                                                                                           (Ura1), Cafks1h-2/CaFKS1b] were essentially equivalent to <br /> (strain T6FOA1 [lane 3] and strain T1FOA3 [lane 11]) were transformed with                that of CAI4(Ura1). Strain NR3(Ura1), which was homozy- <br /> pAM602, and the fragment profiles for seven Ura1 transformants of strain                  gous for the Cafks1h-3 allele, was only twofold less virulent <br /> T6FOA1 (T54 through T59 [lanes 4 to 9, respectively]) and six transformants of            than the wild-type diploid. Finally, the CAI4R1-derived trans- <br /> strain T1FOA3 (T3 through T8 [lanes 12 to 17, respectively]) are shown. The <br /> positions of markers (as described in the legend to Fig. 2, with the addition of a <br />                                                                                           formants T25 and T26 (Cafks1h-1/Cafks1b::URA3), as well as <br /> 3.2-kb standard from pJAD1DXH digested with BamHI and SalI) are indicated                 strains T28 and T32 (Cafks1h-1::URA3/CaFKS1b), were all <br /> (in kilobases).                                                                           roughly 3- to 10-fold less virulent than strain CAI4(Ura1). <br />  VOL. 41, 1997                                                                                               FKS1 GENE OF C. ALBICANS                2477 <br />  <br />  <br />        TABLE 3. Analysis of mutant strains for growth and GS activity in vitro and virulence and pneumocandin susceptibility in vivo <br />                                                                                     In vitro                                    In vivo <br />  <br />      Strain                                Genotype                       Doubling time                          14-day LD50              ED99 (mg/kg) <br />                                                                                                GS sp acta <br />                                                                              (min)                            (104 CFU/mouse)       L-733,560      AMBc <br />           1 <br /> CAI4(Ura )           CaFKS1h/CaFKS1b (pJAM15)                                  90                 31               0.51                0.06         0.05 <br /> T1                   Cafks1h::hisG-URA3-hisG-URA3-hisG/CaFKS1b                NDb                 ND               0.63                ND           ND <br /> T6                   CaFKS1h/Cafks1b::hisG-URA3-hisG                          ND                  ND              33                   ND           ND <br /> CAI4R1(Ura1)         Cafks1h-1/CaFKS1b (pJAM15)                                93                 55               0.47                0.25         0.03 <br /> T25                  Cafks1h-1/Cafks1b::URA3                                   85                 24               1.7                .6.0          0.14 <br /> T26                  Cafks1h-1/Cafks1b::URA3                                   90                 22               5.1                .6.0          0.05 <br /> T28                  Cafks1h-1::URA3/CaFKS1b                                  110                 13               2.8                 0.11         0.04 <br /> T32                  Cafks1h-1::URA3/CaFKS1b                                  100                 35               3.7                 0.03         0.02 <br /> NR2(Ura1)            Cafks1h-2/CaFKS1b (pJAM15)                               ND                  52               0.45                0.74         0.12 <br /> NR3(Ura1)            Cafks1h-3/Cafks1h-3 (pJAM15)                             ND                  53               1.1                .6.0          0.05 <br /> NR4(Ura1)            CaFKS1h/Cafks1b-4 (pJAM15)                                80                 19              16                   0.75         0.07 <br />  a <br />    Expressed as nanomoles of product z hour21 z milligram of protein21. <br />  b <br />    ND, not determined. <br />  c <br />    AMB, amphotericin B. <br />  <br />  <br />  <br />    We evaluated the susceptibilities of these strains to treat-            CAI4 has considerable homology to FKS1. The amino acid <br /> ment with L-733,560 in a mouse model. Animals were infected                sequence predicted from this fragment is 689 residues long, <br /> with a suspension of each strain at 0.5 to 1 times the 14-day              with ca. 75% identity to the Fksp family of proteins (Fks1p, <br /> LD50 and treated with L-733,560 or amphotericin B according                Fks2p, and FksAp). The identity between these proteins is <br /> to the regimen outlined in Materials and Methods. The ED99s                higher (88%) if the comparison is limited to the region of each <br /> for L-733,560 (Table 3) can be used to categorize the strains              protein within the large, predicted cytoplasmic domain. While <br /> into three distinct classes: strains T28, T32, and CAI4(Ura1)              this fragment of CaFKS1 was sufficient for our analysis, it is not <br /> were very susceptible to the antibiotic; strains CAI4R1(Ura1),             the entire open reading frame. It remains to be seen if other <br /> NR2(Ura1), and NR4(Ura1) were less susceptible, requiring 4                domains of CaFks1p have significant identity to the equivalent <br /> to 13 times more L-733,560 for 99% efficacy; and strains T25,              regions of the proteins from A. nidulans and S. cerevisiae. <br /> T26, and NR3(Ura1) were not significantly eliminated from                  Preliminary data from cloning and sequencing of an FKS ho- <br /> infected mice, even at the highest dose of L-733,560 we tested.            molog from C. neoformans suggests that this gene product is <br /> The Echr mutations did not confer cross-resistance to ampho-               also remarkably conserved (23). Unlike genes of the FKS fam- <br /> tericin B; ED99s for all strains, including the T25, T26, and              ily, genes encoding proteins involved in chitin biosynthesis <br /> NR3(Ura1) mutants, were within the range of 0.02 to 0.14 mg                (CHS genes) are significantly divergent across different fungal <br /> per kg. Therefore, the four independent mutations conferred                genera (7), and three isozymes from C. albicans (CaChs1p, <br /> in vitro pneumocandin resistance in a liquid broth microdilu-              CaChs2p, and CaChs3p) are significantly different from one <br /> tion assay and a GS assay and caused an increase in the ED99s              another (7, 19, 38, 44, 45). <br /> for L-733,560 treatment of disseminated candidiasis in an an-                 Integration of the URA3-based disruption plasmid pAM600 <br /> imal model.                                                                into the Cafks1 locus of three spontaneous C. albicans Echr <br />                                                                            mutants yielded a mixture of Echs and Echr transformants. <br />                             DISCUSSION                                     Loss of the resistance phenotype in Echs transformants was <br />    Numerous studies have identified the echinocandins as GS                correlated with disruption of the Cafks1h allele for strains <br /> inhibitors with the potential for use as novel antifungal drugs            CAI4R1 and NR2 and with disruption of the Cafks1b allele for <br /> to treat human mycoses (31). With the development of potent                strain NR4. The results with strain CAI4R1 confirm a previous <br /> inhibitors of GS and the growing repertoire of genetic and                 study based on spheroplast fusions with genetically marked <br /> molecular methods for C. albicans, we have the tools to study              Echs strains (27), in which the Echr phenotype was found to be <br /> the role of this essential enzyme for cell wall synthesis. Work            semidominant. Even though the diploid strain CAI4R1 <br /> with the nonpathogen S. cerevisiae has shown that the GS                   (Cafks1h-1/CaFKS1b) is consistently more susceptible in vitro <br /> subunit Fks1p is the likely target of echinocandin inhibition.             to L-733,560 than strains lacking a functional Echs allele [T25 <br /> The S. cerevisiae etg1-1 strain contains a semidominant muta-              and T26 (Cafks1h-1/Cafks1b::URA3)], the difference is small <br /> tion that leads to echinocandin-resistant whole cells with re-             (Fig. 3). Because we recovered both resistant and sensitive <br /> sistant GS activity. Recent studies with the C. albicans strain            pAM600 transformants of CAI4R1, NR2, and NR4, the <br /> CAI4 have identified rare spontaneous Echr mutants that are                Cafks1h-1, Cafks1h-3, and Cafks1b-4 mutations must be con- <br /> amenable to genetic analysis and can be evaluated for viru-                sidered dominant. In addition, both alleles must be expressed <br /> lence. Because these mutants had phenotypes similar to those               under the growth conditions used in our study, since a muta- <br /> of the S. cerevisiae etg1-1 mutant, we suspected that the mutant           tion in either the CaFKS1h allele or the CaFKS1b allele per- <br /> Echr allele in these isolates might be a homolog of FKS1. Here,            mits Echr/Echs heterozygotes to grow on medium supple- <br /> we have provided evidence that the pathogen C. albicans has a              mented with L-733,560 sufficient to kill Echs/Echs cells. <br /> genetic segment specifying an Fks1p homolog and that the rare                 Strain NR3 was unique among the echinocandin-resistant <br /> mutations are indeed dominant alleles of this gene (CaFKS1).               mutants we isolated. First, only one allele was detected on <br /> Our inability to isolate homozygous null mutants in CaFKS1                 Southern blots of NR3 genomic DNA digested with HindIII <br /> suggests that it is essential for survival of C. albicans.                 and PvuII, and the size was consistent with that of the <br />    The CaFKS1 gene fragment we cloned from C. albicans                     CaFKS1h allele. Second, pAM600 transformants of strain NR3 <br />  2478     DOUGLAS ET AL.                                                                                       ANTIMICROB. AGENTS CHEMOTHER. <br />  <br />  <br /> were uniformly Echr; we recovered no Echs derivatives among           that other mutations, either spontaneous or induced during <br /> the 75 clones we evaluated. The fragment pattern observed on          transformation, are responsible for the differences in virulence. <br /> Southern blots of three transformants suggested that these            We also were unable to predict from liquid broth microdilution <br /> strains contained both an intact and a disrupted copy of the          assays or GS IC50s which strains would be most resistant in vivo <br /> CaFKS1h allele (Fig. 4). While we cannot rule out the possi-          to treatment with pneumocandin. Strain CAI4R1(Ura1) re- <br /> bility of a dominant Echr mutation at another locus, it seems         sembles strains T25 and T26 in its susceptibility in vitro to <br /> likely that strain NR3 has two resistant alleles at CaFKS1.           L-733,560 (Fig. 3A), but the in vivo susceptibilities were not <br /> Under selective pressure imposed by L-733,560, an original            equivalent; infections with strains T25 and T26 required at <br /> mutant allele could have been duplicated by gene conversion.          least 24-fold more L-733,560 for 99% effective treatment than <br /> Such a mutation at the CaFKS1 locus could be either recessive         infections with strain CAI4R1(Ura1) (Table 3). Likewise, the <br /> or dominant. If the mutation were recessive, it would support         IC50s of L-733,560 for GS activities from strains NR2, NR3, <br /> the idea that two types of mutations in FKS genes can confer          and NR4 (40,000, 30,000, and 40 nM, respectively [Table 2]) <br /> resistance to glucan synthase inhibitors in the laboratory, i.e.,     predict that strains NR2(Ura1) and NR3(Ura1) should be <br /> recessive or very weakly dominant mutations, such as the S.           equally resistant in vivo and that strain NR4(Ura1) should be <br /> cerevisiae etg1-4 (15) and pbr1-1 (9) mutations, as well as dom-      significantly more susceptible. However, in infected animals, <br /> inant mutations typified by the Cafks1h-1 mutation.                   strain NR3(Ura1) was the most resistant to L-733,560 (ED99 <br />    Our Southern blots revealed a fortuitous restriction frag-         . 6.0), while ED99s for strains NR2(Ura1) and NR4(Ura1) <br /> ment length polymorphism at the CaFKS1 locus that was in-             were equivalent (0.74 and 0.75, respectively), with values at <br /> strumental for distinguishing the two alleles and demonstrating       least eightfold lower than that for strain NR3(Ura1). A qual- <br /> that disruption of either allele was not overtly deleterious un-      itative measure of enzyme resistance, the shape of the GS <br /> der laboratory growth conditions, but a second forced integra-        titration curves for L-733,560 inhibition, shown in Fig. 3B and <br /> tion yielded only transformants with the plasmid integrated           5, represents the best prediction of pneumocandin susceptibil- <br /> into the preexisting disrupted allele. These results suggest that     ity in vivo. The Echr/Echs heterozygous mutants are of inter- <br /> loss of both copies of the CaFKS1 gene was either a lethal            mediate resistance in vivo, and their GS inhibition curves sug- <br /> event or sufficiently disabling that viable transformants could       gest intermediate resistance, with a shallow slope and a <br /> not be detected. This observation is consistent with our inabil-      maximum of 55% inhibition. We do not know what extent of <br /> ity to detect a homolog of CaFKS1 under stringent conditions,         GS inhibition is sufficient to clear a cell from an infected <br /> like the FKS2 gene of S. cerevisiae which can rescue cells            animal, so the use of GS IC50s as we have defined them may <br /> lacking a functional FKS1 gene. For A. nidulans (25), only one        not be warranted to predict phenotypes in vivo. In addition, <br /> FKS homolog was detected by restriction mapping and <span id='am-113' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-114'  ></span><span id='am-115' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>hybrid</span>-           variables of the in vitro GS assay such as the concentration of <br /> ization analysis. A 0.4-kb C. albicans gene fragment with 47%         crude membrane protein and the incubation temperature can <br /> homology to the S. cerevisiae FKS2 gene was identified by             have a significant effect on the absolute IC50, as evidenced by <br /> random sequence analysis of the C. albicans genome (42), but          the results for GS from strain CAI4 shown in Fig. 3B and 5. <br /> the role of this gene (CaFKS2) is not understood.                        The Echr mutants we have characterized in this paper were <br />    The disruption strategy we used has been employed by oth-          isolated at high cell densities on agar plates containing the <br /> ers to identify essential genes in C. albicans, including HSP90,      pneumocandin L-733,560. Although these strains represent a <br /> a member of the heat shock protein 90 family of molecular             useful tool for genetic characterization, their relevance to the <br /> chaperones (46), and KRE6, a gene required for b-1-6 glucan           potential for clinical pneumocandin resistance is unclear. Echr <br /> synthesis (39). Besides KRE6, several other important but non-        organisms were not recovered in a C. albicans target-organ <br /> essential genes for C. albicans cell wall synthesis and structure     model of infection when pneumocandin MK0991 (formerly <br /> have been identified. The CHS2 (19) and CHS3 (44) genes play          L-743,872) was given to neutropenic mice via intraperitoneal <br /> a role in chitin synthesis; there is evidence that CHS3 plays an      administration (1, 43). The outcome of current and future use <br /> important role in virulence in animal models of infection (8). A      of echinocandins and pneumocandins in humans will provide <br /> third gene (CHS1) has been identified, but its role remains           relevant information about the potential for clinical resistance <br /> unclear (38, 45). Mutants in the C. albicans PHR1 gene are            to this new <span id='am-53' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-54' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> of antifungal antibiotics. <br /> unable to conduct apical growth of either the yeast or hyphal <br /> growth form at alkaline pH. The Phr1 protein is produced at                          <br /> more alkaline pHs and shows strong identity to the Gas1p <br />                                                                        <br /> glycosylphosphatidylinositol-anchored cell wall protein from S. <br /> cerevisiae. It is interesting that GAS1 is not an essential gene in  <br /> S. cerevisiae, but loss of GAS1 combined with disruption of      <br /> either KRE6 or PKC1 is synthetically lethal (40). <br />    Virulence and in vivo susceptibility to treatment with L-                                     <br /> 733,560 were assessed for several strains in a mouse model of          <br /> disseminated candidiasis. Although each strain we tested was            <br />                                                                           <br /> able to establish a lethal infection, the LD50s were not equiv-       <br /> alent, ranging from 0.45 3 104 to 33 3 104 CFU/g of kidney,              <br /> and neither growth rate nor GS specific activity was predictive   <br /> of the relative virulence. Reduced virulence also does not seem        <br /> to be associated with disruption or mutation of a specific allele.        <br /> While the wild-type CaFKS1b allele is preserved in several                <br /> strains that have low LD50s [CAI4(Ura1), T1, CAI4R1(Ura1),           <br /> and NR2(Ura1)], a strain that lacks the CaFKS1b allele en-                <br /> tirely [NR3(Ura1)] and two transformants with a disrupted              <br /> Cafks1b::URA3 allele (strains T25 and T26) still showed sig-           <br /> nificant virulence in the mouse model (Table 3). It is possible           <br /> </body></html>