<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL OF BACTERIOLOGY, Apr. 2007, p. 2844&#8211;2853                                                                                               Vol. 189, No. 7 <br /> 0021-9193/07/$08.00&#11001;0 doi:10.1128/JB.01713-06 <br /> Copyright &#169; 2007, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />  Essentiality Ribosomal Transcription Antitermination Proteins <br />     Analyzed Systematic Gene Replacement <span id='am-102' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-103' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-114'  ></span><span id='am-115' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span>Escherichia coli</span>&#17188; <br />                              Mikhail Bubunenko,1,2 Teresa Baker,2 Donald L. Court2* <br />        Basic Research Program, SAIC-Frederick, Inc., National <span id='am-74' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-75' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-77'  ></span><span id='am-78' property="rdfs:label" content="Cancer" datatype="xsd:string"></span>Cancer</span> Institute&#8212;Frederick, Frederick, Maryland 21702,1 and <br />             Molecular Control Genetics Section, Gene Regulation Chromosome Biology Laboratory, Center for <br />                        Cancer Research, National Cancer Institute&#8212;Frederick, Frederick, Maryland 217022 <br />                                              Received 6 November 2006/Accepted 18 January 2007 <br />  <br />              We details method used identify distinguish essential nonessential genes <br />           bacterial <span id='am-118' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-119' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-121'  ></span><span id='am-122' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-124' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span>Escherichia</span> coli chromosome. Three key features characterize method: high-efficiency <br />           recombination, precise replacement just open reading frame chromosomal gene, presence <br />           naturally occurring duplications bacterial genome. We targeted genes encoding functions critical <br />           processes transcription translation. Proteins complexes evaluated determine if <br />           essential cell deleting individual genes. The transcription elongation Nus proteins and <br />           termination factor Rho, involved rRNA antitermination, ribosomal proteins small 30S <br />           ribosome subunit, minor ribosome-associated proteins analyzed. It concluded five <br />           bacterial transcription antitermination proteins essential, minor ribosome-associated <br />           proteins examined (RMF, SRA, YfiA, YhbH), unlike ribosomal proteins, dispensable. Interest- <br />           ingly, 30S ribosomal proteins essential, knockouts ribosomal protein genes, rpsF <br />           (S6), rpsI (S9), rpsM (S13), rpsO (S15), rpsQ (S17), rpsT (S20), viable. <br />  <br />  <br />    A gene essential nonessential viability              transcription translation; (iii) naturally occurring du- <br /> a cell. An essential gene encodes function required                  plications regions bacterial genome cells ex- <br /> under growth conditions, elimination lethal                isting culture (34). <br /> the cell. Hence, generally interesting, dif-                   We examined functions cellular systems that <br /> ficult, type genes identify characterize. In era              important bacterial macromolecular syntheses. <br /> many genomes sequenced coding regions                       One Nus-dependent transcription antitermination sys- <br /> identified, ability distinguish essential nonessential              tem, components interact RNA polymer- <br /> genes requires careful experimental assessment (23, 24, 29,               ase transcription elongation termination modify <br /> 48). Here, create gene replacements distinguish                     polymerase, allowing read-through transcription termi- <br /> two classes genes direct selection Escherichia coli using             nators (21, 50, 54, 55). The second examined the <br /> phage &#65533;?&#65533; Red-mediated homologous recombination, termed                           ribosome, particular small subunit (30S) ribosomal <br /> recombineering (12, 20, 69).                                                    proteins (r-proteins) known minor ribo- <br />    Recombineering uses Red recombination functions                       some-associated proteins. The minor r-proteins, name <br /> phage &#65533;?&#65533; manipulate DNA chromosomes <span id='am-147' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-148' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-151' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span>                        implies, equimolar amounts conventional <br /> of enteric bacteria (12, 18). In work, recombineering                  30S 50S r-proteins, ratio does increase station- <br /> adapted directly identify essential genes E. coli. The                    ary phase specific conditions growth <br /> procedure intended high-throughput identification                  (1, 28, 35, 64). We shown E. coli transcription <br /> of essential genes, certainly upgraded                antitermination factors, NusB, conditionally <br /> larger-scale analysis. It useful targeted,                 lethal, essential bacterial cell. In contrast, minor <br /> knowledge-based analysis cellular systems individ-                 ribosomal proteins dispensable cell viability agree- <br /> ual components, demonstrated paper. Our                      ment published results. Unexpectedly, six <br /> method developed using features: (i) extremely                     r-proteins 30S subunit absolutely required for <br /> high efficiency recombineering (69); (ii) ability re-                 growth and, like NusB, exhibit conditional cell lethality. <br /> combineering provide precise replacement just open                        (Research performed Theresa Baker partial ful- <br /> reading frame (ORF) chromosomal gene orf                    fillment requirements Master Science Bio- <br /> antibiotic resistance cassette, taking care design express               medical Sciences Hood College, Frederick, MD.) <br /> such replacement orf regulatory regions the <br /> replaced gene (69) avoid polar effects insert on <br />                                                                                                         MATERIALS AND METHODS <br />                                                                                    Media bacteriological techniques. Standard bacteriological media and <br />   * Corresponding author. Mailing address: Molecular Control                techniques used cultivate cells (37). Cells grown LB broth on <br /> Genetics Section, Gene Regulation Chromosome Biology Labo-                  LB M63 agar plates. The M63 plates supplemented 0.2% glucose, <br /> ratory, Center Cancer Research, National Cancer Institute Fred-          vitamin B1, biotin. The following final concentrations antibiotics were <br /> erick, Frederick, MD 21702. Phone: (301) 846-5940. Fax: (301) 846-              used, unless stated: ampicillin (Ap), 25 &#65533;?&#65533;g/ml; tetracycline (Tc), 12.5 <br /> 6988. E-mail: court@ncifcrf.gov.                                                &#65533;?&#65533;g/ml; kanamycin (Km), 25 50 &#65533;?&#65533;g/ml, chloramphenicol (Cm), 10 30 <br />   &#17188; <br />     Published ahead print 2 February 2007.                                &#65533;?&#65533;g/ml, recombinant strains maintenance plasmids, respectively. <br />  <br />                                                                          2844 <br />  VOL. 189, 2007                                                                                        IDENTIFICATION OF ESSENTIAL GENES                         2845 <br />  <br />  <br />                   TABLE 1. Strains plasmids used <br />                                                                                                               TABLE 2. List primers <br />    Strain                           Description                          Source <br />                                                                                             Primera                          Homology sequenceb <br /> W3110              Wild type <br /> DY330              W3110 &#9004;lacU169 gal490 pgl&#9004;8 &#65533;?&#65533; &#20851;&#65533;?&#65533;&#20852;cI857               69            <span id='am-61' about='obo:NCBITaxon_468772' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_33154 obo:NCBITaxon_43733 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_50680 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50673 obo:NCBITaxon_50557 obo:NCBITaxon_50671'><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-63'  ></span><span id='am-64' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="Nusa" datatype="xsd:string"></span>nusA</span>-Cm(f) ............CCCCACTTTTAATAGTCTGGATGAGGTGAAAAGCCCGCG- <br />                     &#9004;(cro-bioA)                                                                                  cat(f) <br /> DY331              DY330 recA                                           69            nusA-Cm(r)............CGTTACATCTGTCATGCTGTTCCTTCCTGCTACAGTTTA- <br /> DY432              DY330 &#9004;N-kil                                         C. McGill                                cat(r) <br />                                                                            (NCI)      nusA-pCR(f) ..........GCTGCCCGTAATATTTGCTGGTTCGGTGACGAAGCGTAA- <br /> IQ527              MC4100 ssyB63 (nusB::IS10) zba525::Tn10              59                                       pCR(f) <br /> NB132              DY330 rpsF&#65533;?&#11022;kan                                      <span id='am-84' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-85' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-88'  ></span>This</span> work     nusA-pCR(r) ..........ATTGGATACGGCTTCAACTACAGCCAAAATTTCTTTGTT- <br /> NB149              DY330 rpsJ(nusE)&#65533;?&#11022;kan/rpsJ(nusE)&#11001;                    This work                                pCR(r) <br />                      partial diploid                                                  nusA-check(f) ........TAAAGATGAAGTGTTCGCGCTGAG <br /> NB207              DY432 nusB&#65533;?&#11022;cat/nusB&#11001; partial diploid                This   work   nusA-check(r) ........GCTGTACCAGGCGTTCCACGGAGG <br /> NB208              DY330 nusA&#65533;?&#11022;cat/nusA&#11001; partial diploid                This   work   nusB-Cm(f) ............TGAAAGCCATCAAGGCCTGAAATTAGTAAGGGGAAATCC- <br /> NB209              DY330 nusA&#65533;?&#11022;cat pAB116                               This   work                              cat(f) <br /> NB210              NB207 nusB&#65533;?&#11022;cat/nusB&#11001; thrA::Tn10                     This   work   nusB-Cm(r) ............CATGGAACGGTCTTCCGTGAATCTACCGGCCTGGATATCA- <br />                      thrA&#11001;                                                                                       cat(r) <br /> NB213              DY330 rpsM&#65533;?&#11022;kan                                      This   work   nusB-check(f).........AAGGTGCAGAAGCTGCACTGACCGCGC <br /> NB216              DY330 rho&#65533;?&#11022;kan/rho&#11001;                                  This   work   nusB-check(r).........GGCAATCAGGGAGAACTCGCCACATGC <br /> NB217              DY330 rhoL-rho&#65533;?&#11022;kan/rho&#11001;                             This   work   nusE-Km(f) ............AATCATTTTCGTTTATAAAATAATTGGAGCTCTGGTCT- <br /> NB218              DY330 rhoL-rho&#65533;?&#11022;kanSD/rho&#11001;                           This   work                              kan(f) <br /> NB345              DY330 rpsT&#65533;?&#11022;kan                                      This   work   nusE-Km(r) ............CAATCATTGTTTCAACCTCTCAATCGCTCAATGACCTGA- <br /> NB350              DY330 rpsQ&#65533;?&#11022;kan                                      This   work                              kan(r) <br /> NB421              W3110 nusB&#65533;?&#11022;cat                                      This   work   nusE-exp(f).............CAGAAGGAGATATTCATATGCAGAACCAAAGAATCCGTATC <br /> NB429              DY330 rpsO&#65533;?&#11022;kan                                      This   work   nusE-exp(r) ............TTGCATGCGCATCTAGATTAACCCAGGCTGATCTGCACGTC <br /> NB438              W3110 rpsV&#65533;?&#11022;kan                                      This   work   nusE-check(f).........CTCCCATCAATCGTAATGGGTCTG <br /> NB439              W3110 yfiA&#65533;?&#11022;kan                                      This   work   nusE-check(r).........ACGGGTCATACCCACTTTTTTACC <br /> NB440              DY330 rpsV&#65533;?&#11022;kan                                      This   work   nusG-Cm(f) ............GGTGAAAATGTTTGTAGAAAACTTCTGACAGGTTGGTTT- <br /> NB441              DY330 yfiA&#65533;?&#11022;kan                                      This   work                              cat(f) <br /> NB469              DY330 rpsJ(nusE)&#65533;?&#11022;kan pAB37                          This   work   nusG-Cm(r)............TTGTGCAACGATTAAATCGCCGCTTTTTTGATCGCTGGG- <br /> NB470              DY330 rpsI&#65533;?&#11022;kan                                      This   work                              cat(r) <br /> NB483              DY330 yhbH&#65533;?&#11022;cat                                      This   work   nusG-check(f) ........CGCAGTAATGTCACTGATCCTGTG <br /> NB484              DY330 rpsV&#65533;?&#11022;spc                                      This   work   nusG-check(r) ........GCGAAATTGTATTCCAATCTCACG <br /> NB485              DY330 rmf&#65533;?&#11022;bla                                       This   work   rho-Km(f) ...............ACATTAAGTTCGAGATTTACCCCAAGTTTAAGAACTCACACCA <br /> NB493              W3110 yhbH&#65533;?&#11022;cat                                      This   work                              CT-kan(f) <br /> NB494              W3110 rmf&#65533;?&#11022;bla                                       This   work   rhoL-Km(f) ............TAAGTTTGAATCTTGTAATTTCCAACGCTTCCCGTTTTATCTT <br /> NB495              W3110 rpsV&#65533;?&#11022;spc                                      This   work                              <span id='am-125' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-126' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-128'  ></span><span id='am-129' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span>AA</span>-kan(f) <br /> NB539              W3110 rmf&#65533;?&#11022;bla rpsV&#65533;?&#11022;spc yfiA&#65533;?&#11022;kan                   This   work   rhoL-KmSD.............rhoL-Km(f)-CAGGATGAGGATCGTTTCGCATG <br />                      yhbH&#65533;?&#11022;cat                                                        rho-Km(r)...............AGCAAAACGCCACGTAAACACGTGGCGTTTTTGGCATAAGACA <br /> NB606              DY432 nusB&#65533;?&#11022;cat                                      This work                                AA-kan(r) <br /> NB610              DY330 nusG&#65533;?&#11022;cat nusG &#11001; partial diploid               This work     rho-check(f)............TGTTGACTTCGTATTAAACATACC <br /> NB611              DY330 nusG&#65533;?&#11022;cat pAB90                                This work     rho-check(r) ...........GATGAGATCAGTACTCACTGTCAG <br /> NB747              IQ527 ssyB63&#65533;?&#11022;nusB&#65533;?&#11022;cat                              This work     rmf-Ap(f)................TTTCTTTTCCACCAGAAACCAGTATGAGGGAAACGAGGC- <br /> pCR-Script         oriColE1 Cmr                                         Stratagene                               amp(f) <br /> pPCR-Blunt         oriColE1 Kmr                                         Invitrogen    rmf-Ap(r)................TCCTCCGCAATGCGGAGGTTTCTTTTTAAAGAGACAGAA- <br /> pAB37              pCR Cmr nusE(rpsJ)                                   This work                                amp(r) <br /> pAB90              pPCR Kmr nusG                                        30            rmf-check(f) ...........TGACGGCAGTTATGATTCGCGGTA <br /> pAB116             pPCR Kmr nusA                                        This work     rmf-check(r) ...........AGGATACGTCTGCCTTCTGATTAT <br />                                                                                       rpsV-Km(f).............TGTTGTCCTTAAAACTAGCTACAGGATTGAGGAGTTAAA- <br />                                                                                                                  kan(f) <br />                                                                                       rpsV-Km(r) ............AATGGTGTTTAATCGTCATTGAGGACTGATGGTTATGAA- <br />                                                                                                                  kan(r) <br />                                                                                       rpsV-check(f) .........TTATGTGGTCAGTGGCCAGCACC <br />    Strains plasmids. Strains plasmids used study shown         rpsV-check(r) .........TTGGTTCCATGTCACTCACTCTT <br /> Table 1.                                                                              yfiA-Km(f)..............GCTGAATTCACCAAGACGGGAAGACAAGAGGTAAAATTT- <br />    Gene disruption recombineering. Genes disrupted chromosome                                     kan(f) <br /> as described (60) modifications. Briefly, DY330 cells        yfiA-Km(r)..............CGCCCGAAGGCGCGTTGGCGATACACTCAATATAAAGGA- <br /> induced &#65533;?&#65533; Red functions A600 0.5 prepared electroporation                                   kan(r) <br />                                                                                       yfiA-check(f) ..........CACATTTTGACATCAGGAACGG <br /> with 300 ng linear DNA antibiotic resistance cassette. The cells were <br />                                                                                       yfiA-check(r) ..........GTACTGTTTTCACGCTGTCAAC <br /> recovered 0.9 ml LB, grown 2 3 h 32&#176;C, spread aliquots LB agar     yhbH-Cm(f)............AACAACTCGTTTGACCCAACCGATAAGGAAGACACT-cat(f) <br /> plates appropriate antibiotic, incubated 7 days 34&#176;C. The 34&#176;C      yhbH-Cm(r) ...........AACAGACCGCCATGCACATGCTAATTGCCCGGACAA-cat(r) <br /> temperature chosen defective &#65533;?&#65533; prophage contained DY330            yhbH-check(f) ........CACTGTTGCGAAGTACCGAGAGTCT <br /> has killing function expressed 37&#176;C. The antibiotic resistance        yhbH-check(r)........ATTAACCGTACAAATGGCCCGTTGT <br /> cassettes gene disruption PCR amplification (Expand PCR kit; <br /> Roche Applied Science, Indianapolis, IN) antibiotic-resistant orfs (see        amp(f) .....................ATGAGTATTCAACATTTCCGT <br /> reference 60 description antibiotic resistance orfs) <span id='am-142' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-143'  ></span><span id='am-144' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-146' property="rdfs:label" content="hybrid" datatype="xsd:string"></span>hybrid</span> primers   amp(r).....................TTACCAATGCTTAATCAGTGA <br /> (Table 2) contained 40- 50-nucleotide (nt) sequence 5&#11032; ends        cat(f) .......................ATGGAGAAAAAAATCACTGGATATAC <br />                                                                                       cat(r) .......................TTACGCCCCGCCCTGCCACTCATC <br /> homologous chromosomal regions surrounding ORF gene targeted <br />                                                                                       kan(f) ......................ATGATTGAACAAGATGGATTG <br /> for disruption, followed 22- 25-nt priming sequence complementary      kan(r) ......................TCAGAAGAACTCGTCAAGAAG <br /> 5&#11032; 3&#11032; ends antibiotic resistance gene ORF. Primers inactivation      pCR(f) ....................CCTGAATTCTGCAGATATCCATCAC <br /> the 30S r-protein genes designed manner; sequences         pCR(r) ....................CATAGCTGTTTCCTGTGTGAAATTG <br /> available request. The resulting PCR-generated cassettes antibiotic <br />                                                                                         a <br /> resistance orf flanked 40- 50-nt regions homology needed        f r indicate forward reverse primers, respectively. <br />                                                                                         b <br /> recombination chromosomal gene, chromosomal orf                 Sequences primers used generate gene cassettes recombineering are <br />                                                                                       shown parts. The shown normal font represents <br /> precisely replaced antibiotic resistance orf. <br />                                                                                       sequences complementary flanking regions gene interest. The <br />    PCR analysis gene replacements. Recombinant colonies purified           second bold used amplification respective PCR <br /> LB plates appropriate antibiotic 34&#176;C avoid contamination     fragments; sequences primers shown the table. <br /> background cells. Individual colonies resuspended 30 &#65533;?&#65533;l sterile H2O, and <br />  2846       BUBUNENKO ET AL.                                                                                                                           J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 1. Gene disruption recombineering E. coli. Replacement entire chromosomal gene ORF antibiotic resistance cassette orf <br /> by recombineering shown upper row. If gene dispensable (A), antibiotic resistance orf simply replaces gene ORF without <br /> affecting cell survival. In case essential gene replacement, cells replaced gene survive additional wild-type copy <br /> of gene, chromosomal duplication (B) complementing plasmid expressing gene trans (C). <br />  <br />  <br />  <br /> 1 &#65533;?&#65533;l used template PCR analysis using primers flanking targeted     counting colonies LB-Cm versus LB plates. The experiments re- <br /> gene&#8217;s ORF (the set checking primers shown Table 2) 50 &#65533;?&#65533;l-reaction     peated twice. <br /> mix. PCR products visualized 1% agarose gel.                                  Phenotypic analysis knockout mutants. The nusB, rmf, rpsV, yfiA, yhbH <br />    Construction multiple knockouts minor ribosome-associated proteins.        knockout mutants P1 transduced W3110, selecting respective <br /> The individual gene knockouts recombineering described         antibiotic resistance marker. They analyzed phenotypic microarrays (7) <br /> using different antibiotic resistance gene cassettes. These knockouts se-       major metabolic functions, recommended manufacturer (Biolog, <br /> quentially combined W3110 P1 transduction (37) obtain quadruple           Inc.). Cell growth analyzed LB M63 agar plates antibiotic at <br /> gene knockout, rmf&#65533;?&#11022;bla, rpsV&#65533;?&#11022;spc, yfiA&#65533;?&#11022;kan, yhbH&#65533;?&#11022;cat. After             30, 37, 42&#176;C. <br /> transduction, transduced gene replacement selected LB plates            To study viability knockouts minor r-proteins stationary phase, 3 <br /> antibiotic verified PCR analysis. The final knockout confirmed     ml LB inoculated wild-type W3110 mutant knockout and <br /> same manner replaced genes.                                             grown 37&#176;C 24 h. Equal volumes of mutant W3110 cell <br />    Cloning nus genes. Cloning nusA nusG open reading frames        cultures mixed continued incubate condi- <br /> done retrieval (gap repair) recombineering (60) linear pPCR-Blunt          tions 10 days. Controls included initial mutant W3110 <br /> vector (Invitrogen, Inc., Carlsbad, CA) exactly described nusG (30). The      monocultures. Samples taken 2 days, diluted, plated LB agar, <br /> ORF nusE, known rpsJ, amplified translation initiation   appropriate antibiotic, determine total number <br /> region PCR chromosome, using Expand PCR kit (Roche Ap-               cells number mutant cells survived mixed cultures over <br /> plied Science, Indianapolis, IN), PCR fragment cloned pCR-          incubation time. <br /> Cam (Stratagene, La Jolla, CA). Plasmids correct orientation            Bioinformatic analysis. The older version COGnitor (58) available the <br /> insert relative Plac screened restriction analysis verified DNA    NCBI website adapted study. The basic procedure included the <br /> sequencing. See Table 1 plasmids cloned genes Table 2 primers       generation phylogenetic pattern protein screen- <br /> used clone genes.                                                             ing protein conservation minimal eubacterial genomes 1.2 <br />    Disruption chromosomal nusA nusG recombineering presence         Mb size. Since genomes severely limited number genes <br /> of plasmid-expressed genes. The DY330 cells transformed electropora-         (38), contain higher percentage essential genes. The orthologous <br /> tion pAB90 pAB116 expressing nusG nusA, respectively. The freshly         proteins encoded genomes additionally analyzed tree analysis of <br /> transformed cells used recombineering replace chromosomal nusA           protein similarity followed sequence analysis protein <span id='am-71' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-72' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span>, using <br /> and nusG genes cat cassette, described above. Recombinants            ClustalW 1.74 manual adjustment aligned sequences. As result these <br /> selected 34&#176;C LB-Cm plates analyzed PCR above. Constitutive          manipulations, truly orthologous proteins selected. The results of <br /> expression Plac allowed sufficient expression NusA NusG proteins          COG analysis verified expansive Blastn Blastp analyses <br /> from plasmid DY330 cells. Note homologies used replace        bacterial genomes available date, using E. coli ortholog as <br /> chromosomal nus genes recombineering present plasmid               bait. It basic procedure screening proteins are <br /> clones.                                                                              conserved 90% time minimal genomes sufficient reliably <br />    Transfer nusE disruption chromosome P1 transduction           predict proteins essential determined gene replacement tech- <br /> presence plasmid-expressed nusE. The nusE&#65533;?&#11022;kan allele crossed        nology. <br /> recipient DY330 bearing nusE-expressing plasmid, pAB37, P1 transduction <br /> with lysate grown NB149, nusE&#65533;?&#11022;kan/nusE&#11001; partial diploid. DY330 re- <br /> cipient cells used negative control transduction nusE&#65533;?&#11022;kan. <br /> Transductants spread LB-Km plates incubated 34&#176;C                                            RESULTS <br /> days. They purified LB-Km plates verified PCR configuration <br /> of nusE gene. <br />                                                                                         Rationale use recombineering analyze essential <br />    Analysis size stability chromosomal duplications. The extent nusG    genes. An essential gene encodes function required <br /> and nusB chromosomal duplications analyzed set mini-Tn10-              growth conditions, elimination lethal to <br /> generated auxotrophic markers originally designed fine gene mapping P1        cell. However, experimental conditions de- <br /> transduction (40, 52). Each markers P1 transduced nusG <br />                                                                                      fine essential genes functions required for <br /> and nusB duplication-containing strains, transductants selected on <br /> LB-Tc plates scored prototrophy M63 minimal medium. Growth           colony formation LB agar 34&#176;C. Genes inac- <br /> of Tcr cells minimal medium indicated duplication.                          tivated manner affect expression adja- <br />    We examined stability duplications strains. Cultures nusG    cent genes. In general, gene coding region start to <br /> or nusB partial diploids grown overnight LB-Cm LB-Cm, Tc broth.           stop <span id='am-89' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-90'  ></span><span id='am-91' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span> replaced precise in-frame substitution of <br /> The overnight cell cultures washed antibiotic, rate duplication <br /> segregation determined serial 300-fold dilution fresh LB medium <br />                                                                                      antibiotic resistance cassette orf (69), allowing direct selec- <br /> every 12 h antibiotic 8 days incubation. Every 24 h,     tion replacement. Nonessential genes routinely <br /> number frequency Cmr cells total cell population determined       replaced substitution technique (Fig. 1A) frequencies <br />  VOL. 189, 2007                                                                         IDENTIFICATION OF ESSENTIAL GENES                  2847 <br />  <br />  <br />                                                                             knockout chromosome recreated the <br />                                                                             presence plasmid clone (Fig. 1C). These procedures <br />                                                                             used analysis gene essentiality, outlined Fig. 2. <br />                                                                                Use precise gene replacement selectable antibiotic <br />                                                                             resistance cassette yield recombinants, if <br />                                                                             gene essential. Whenever ORF target gene <br />                                                                             question replaced precisely ORF cassette, <br />                                                                             promoter translation signals target gene are <br />                                                                             used express cassette. We expect genes, <br />                                                                             expression insufficient allow antibiotic resistance. In <br />                                                                             cases recombinant colonies found, alternative <br />                                                                             cassette structures used contain promoter <br />                                                                             ribosome-binding site cassette, its <br />                                                                             transcription terminator (see examples reference 69). When <br />                                                                             transcription terminator used, terminator block <br />                                                                             expression downstream genes operon. However, the <br />                                                                             absence terminator, cassette&#8217;s promoter too <br />                                                                             strong cause toxic overexpression distal genes. For all <br />                                                                             genes tested below, native expression sufficient <br />                                                                             express antibiotic resistance cassette, insertion of <br />                                                                             additional regulatory elements required. If recombi- <br />                                                                             nants presence cloned copy, is <br />                                                                             possible region chromosome refractory to <br />    FIG. 2. Flow chart procedure analysis gene essential-      recombination. Although systematic search been <br /> ity. A gene disrupted recombineering exactly replacing       completed, gene replacements attempted in <br /> gene ORF antibiotic resistance cassette orf. A group genes       laboratory, example refractory gene region <br /> can analyzed choosing different antibiotic resistance cassettes for <br /> disruption (see &#8220;Construction multiple knockouts minor ribo-         found. <br /> some-associated proteins&#65533;? text). A COG-based prediction pro-            Three patterns observed orf replacements. Three typical <br /> cedure aid selecting genes analysis. Essential versus        patterns ascribed essential, growth-impaired, <br /> dispensable genes determined different recombination           nonessential gene knockouts described below <br /> frequencies detection partial duplication chromosomal <br /> region targeted gene gene essential. If needed, gene <br />                                                                             (Fig. 3). The pattern observed inactivation essential <br /> essentiality tested gene disruption presence    gene (Fig. 3B) resultant low number normally growing <br /> a complementing plasmid carrying wild-type allele gene.         recombinant colonies (&#65533;?102/108 viable cells), bear <br />                                                                             duplication replaced gene carrying wild-type allele <br />                                                                             (gene&#65533;?&#11022;antibiotic resistance cassette/gene&#11001;). Another pat- <br /> that usually generate 104 recombinants 108                   tern characterized impaired growth caused gene <br /> electroporated cells (69). Essential genes, hand,              replacement (Fig. 3C). In case, recombination few <br /> cannot replaced recombination second wild-                  normally growing recombinant colonies observed among <br /> type copy gene present cell (Fig. 1B C).            thousands slow-growing ones. The large normal colonies <br />    Within population cells growing culture, approxi-                contain duplication region replaced gene, <br /> mately 10&#11002;3 cells undergo duplication large genomic                  small slow-growing colonies contain dis- <br /> segment, creating partial diploid region chromosome                rupted gene. The nonessential gene pattern (Fig. 3A) char- <br /> (34). Independent duplications most, all, regions              acterized high recombination frequency (103 104 re- <br /> the genome occur exist significant subpopulations             combinants/108 viable cells) normally growing recombinant <br /> bacterial culture. When essential genes targeted gene               colonies bearing single replaced chromosomal gene an <br /> replacement recombineering, partial diploid subpopu-               antibiotic resistance cassette (gene&#65533;?&#11022;antibiotic resistance cas- <br /> lations present available recombination. Recombi-               sette). <br /> nants survive essential gene deleted resides             Essential nus genes. Replacement nusA, nusD/rho, nusE/ <br /> in region genome diploid. Such recombinants                rpsJ, nusG cat kan cassette resulted low <br /> are low frequency &#11011;5 50 108 cells              frequency antibiotic-resistant recombinants (Table 3). PCR <br /> antibiotic-resistant colonies, duplicated state             analysis showed (Fig. 3B, lane 1) antibiotic-resistant <br /> stabilized selection antibiotic resistance marker            recombinants nusA, nusE, nusG, rho contained a <br /> for expression essential gene (Fig. 1B). If gene            wild-type antibiotic resistance-substituted copy the <br /> be replaced efficiently recombineering rare recombi-                 respective targeted gene, indicating genes essen- <br /> nants appear partial diploids, assume gene              tial <span id='am-131' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span>. Our results rho disagree those <br /> be essential. Alternatively, possible gene            Baba et al. (5), reported rho nonessential. <br /> essential, particular replacement caused unex-                 We different knockouts rho. The removed <br /> pected defect adjacent essential gene. To test possi-            just rho gene, second removed rho gene the <br /> bility, open reading frame candidate essential gene             rho leader region including small &#8220;gene&#65533;? rhoL. For the <br /> is individually cloned plasmid recombineering. The                construct, rhoL does distinctive Shine- <br />  2848       BUBUNENKO ET AL.                                                                                                            J. BACTERIOL. <br />  <br />  <br />  <br />  <br />   FIG. 3. Three patterns gene knockouts observed analysis gene essentiality recombineering. A. Nonessential gene pattern <br /> characterized standard high recombination frequency LB plates (left panel) single configuration replaced gene analyzed by <br /> agarose gel electrophoresis (right panel). The agarose gel shows gene replaced antibiotic resistance cassette (lane 1) versus original <br /> wild-type gene (lane 2). B. Essential gene pattern characterized low recombination frequency colonies duplicated configuration the <br /> disrupted gene (lane 1) versus wild-type allele (lane 2) gel analysis. The duplication includes gene replaced antibiotic resistance <br /> cassette (lane 1, upper band) wild-type gene (lane 1, lower band). C. Growth-impaired pattern characterized mixed pattern two <br /> colony sizes different recombination frequencies. The large colonies appear low recombination frequency duplicated <br /> configuration (lane 1) essential gene. The small colonies high recombination frequency single configuration replaced <br /> gene (lane 2) versus wild-type allele (lane 3). <br />  <br />   TABLE 3. Results gene disruption recombineering applied to <br />      antitermination ribosome-associated protein genes                         Dalgarno region, rhoL-rho knockout the <br />                      Recombination                                   Gene <br />                                                                                    kan cassette containing Shine-Dalgarno re- <br />        Gene                             Gene configurationb                        gion. Recombinants constructs made, all <br />                        efficiency                               essentiality <br />                                                                                    showed pattern essential gene knockouts carried the <br /> nusA                   1.8 &#11003; 101       nusA&#65533;?&#11022;cat/nusA&#11001;          Essential <br /> nusBc                  0.9 &#11003; 101       nusB&#65533;?&#11022;cat/nusB&#11001;          NAe                gene duplication. <br />                        6.5 &#11003; 103       nusB&#65533;?&#11022;cat                Growth                Initially thought duplicated regions might <br />                                                                   impaired         formed unequal crossing small <br /> nusD/rho               3.7 &#11003; 102       nusD&#65533;?&#11022;kan/nusD&#11001;          Essential          homologous regions chromosome mediated host <br /> rhoL-nusD/rho          2.7 &#11003; 102       nusD&#65533;?&#11022;kan/nusD&#11001;          Essential <br /> rhoL-nusD/rhod         4.2 &#11003; 102       nusD&#65533;?&#11022;kanSD/             Essential <br />                                                                                    recombination system. However, repeated rho <br />                                           nusD&#11001;                                    gene replacement exactly reported recA <br /> nusE(rpsJ)             0.7 &#11003; 101       nusE&#65533;?&#11022;kan/nusE&#11001;          Essential          mutant derivative, result. Red-mediated re- <br /> nusG                   1.2 &#11003; 101       nusG&#65533;?&#11022;cat/nusG&#11001;          Essential          placements found, rarely, cells du- <br /> yfiA                   7.3 &#11003; 104       yfiA&#65533;?&#11022;kan                Nonessential       plications. <br /> yhbH                   6.6 &#11003; 104       yhbH&#65533;?&#11022;cat                Nonessential <br /> rmf                    3.2 &#11003; 104       rmf&#65533;?&#11022;amp                 Nonessential          The growth-impaired nusB gene. In case nusB replace- <br /> rpsV                   8.5 &#11003; 104       rpsV&#65533;?&#11022;kan                Nonessential       ments (Table 3), normal-sized colonies appeared first <br />                                                                                    day incubation, mimicking character nus genes. <br />   a <br />     Number antibiotic-resistant DY330 recombinants 5 &#11003; 108 viable cells. <br />   b <br />     Determined PCR analysis chromosomal region replaced         However, unlike nus gene replacements, 2 days a <br /> gene, shown Fig. 3. Partial diploids shown gene&#65533;?&#11022;antibiotic-resis-    large number much-smaller-sized colonies appeared (&#11011;104 <br /> tance cassette/gene&#11001;. <br />   c <br />     Two gene knockout patterns observed nusB knockouts. Cells a <br />                                                                                    colonies/108 viable cells). These types isolates the <br /> diploid gene configuration appeared low frequency day. After   nusB gene replacement showed distinct PCR patterns. The <br /> 2 days, followed numerous nusB&#65533;?&#11022;cat recombinants.                     larger colonies contained wild-type nusB cat- <br />   d <br />     rhoL-rho replaced kan containing Shine-Dalgarno sequence, <br /> kanSD.                                                                             replaced gene, small nusB&#65533;?&#11022;cat colonies only <br />   e <br />     NA, applicable.                                                            replaced gene copy (Fig. 3C, lanes 1 2). <br />  VOL. 189, 2007                                                                          IDENTIFICATION OF ESSENTIAL GENES                    2849 <br />  <br />  <br />  <br />  <br />    FIG. 4. Characterization gene duplications essential (nusG) impaired (nusB) gene knockout patterns. A. Genetic mapping nusG <br /> and nusB duplications set Tn10 (Tcr) auxotrophic markers plating cells LB-Tc M63 minimal agar select for <br /> Tcr prototrophy, respectively. The gene markers position (min) E. coli chromosome indicated. Positions nusG nusB <br /> are shown bold. The dotted line defines inverted chromosomal region W3110 (25). The duplicated regions shown bold. The <br /> bold dotted line indicates duplication precisely mapped extend flanking area. B. Stability nusG nusB diploids. <br /> Every 12 h diploid cultures passed LB medium. Every 24 h number diploids cultures estimated ratio Cmr <br /> cells total number cells cultures plating LB-Cm plain LB plates, respectively. Note very <br /> different stabilities diploids, days incubation nusG nusB diploids plotted percentage actual number Cmr <br /> cells cell cultures shown linear log plot, respectively. <br />  <br />  <br />  <br />    Replacement 30S ribosomal protein genes. Each 21              tion replaced nus genes colonies tested (Table 3). <br /> 30S ribosomal protein genes evaluated essential func-              In contrast, absence complementing plasmid, <br /> tion replacement orf kan cassette. Fifteen             recombineering generated gene replacement dupli- <br /> these gene replacements displayed pattern essential                 cated configuration; however, P1 transduction did generate <br /> gene knockout: low recombinant frequency duplication               recombinants low frequency relative re- <br /> as confirmed PCR analysis (Fig. 3). Six knockouts (rpsF,                combineering. Note nus gene plasmid not <br /> rpsI, rpsM, rpsO, rpsQ, rpsT) formed numerous,                 detected PCR analysis primers used in <br /> small, slow-growing colonies, containing disrupted                chromosomal flanking regions outside nus gene <br /> gene, described growth-impaired nusB knockout.                  tested. <br /> From <span id='am-132' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-133' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-135' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-136'  ></span>data</span> conclude majority 30S                      Characteristics gene duplication. We attempted P1 <br /> protein genes essential, rpsF, rpsI, rpsM, rpsO, rpsQ,          transduce number essential gene replacements, including <br /> and rpsT.                                                                  nus genes, strain. In case nusE <br />    Nonessential ribosome-associated protein genes. Replace-                nusG replacements transduced unless recipient strain <br /> ment minor ribosome-associated protein genes kan                 contained plasmids expressing respective nus gene. Since <br /> yielded high frequency (&#11022;103 108 viable cells) Kmr                P1 known transduce nearly 100-kb segment chromo- <br /> recombinants (Table 3). All isolates contained inac-              somal DNA, suggests duplication 50 kbp. On <br /> tivated form targeted gene formed normal-sized                  hand, nusB&#65533;?&#11022;cat replacement trans- <br /> colonies (Fig. 3A). A fully viable quadruple knockout                   duced strains, transductants always <br /> all minor protein genes (rmf&#65533;?&#11022;amp, rpsV&#65533;?&#11022;spc,                         small colony formers lacking intact nusB gene. <br /> yfiA&#65533;?&#11022;kan, yhbH&#65533;?&#11022;cat) generated serial P1 trans-                   To determine extensive duplications be, the <br /> duction individual knockouts. Thus, minor              duplicated nusG nusB mutants subjected further <br /> ribosome-associated proteins appear dispensable.                     tests. Several auxotrophic strains Tn10 <br />    Inactivation nus genes presence complementing              transposon insertion (40, 52). We transduced the <br /> plasmids. To inactivate essential chromosomal nus genes,               tet markers auxotrophs duplicated <br /> methods used required presence comple-                  nusG&#65533;?&#11022;cat nusB&#65533;?&#11022;cat strains, selecting tetracycline <br /> menting plasmid bearing cloned nus genes. In                 resistance (Tcr), respectively, LB-Tc agar scoring for <br /> method, nusA nusG genes replaced cat                    prototrophy M63 minimal agar. Transductants were <br /> cassette recombineering. In second method, nusE              Tcr prototrophic presumed diploid that <br /> nusG genes replaced transduction P1 grown                  region. The results indicated duplicated regions ex- <br /> the previously partial diploid nusE&#65533;?&#11022;kan/nusE&#11001;                     tend far nus genes tested segments compris- <br /> nusG&#65533;?&#11022;cat/nusG&#11001;. Both methods presence com-                  ing 10 4 min chromosome nusB nusG <br /> plementing plasmid yielded single chromosome configura-                duplications, respectively (Fig. 4A). The duplications, <br />  2850     BUBUNENKO ET AL.                                                                                                     J. BACTERIOL. <br />  <br />  <br />                                                                          These data consistent published data func- <br />                                                                          tion rpsV product, SRA (28), recently <br />                                                                          published data characterization yfiA yhbH knock- <br />                                                                          outs (62), function rmf product. The <br />                                                                          rmf::kan cells shown gradually lose viability in <br />                                                                          stationary phase (68). Our rmf knockout viable could <br />                                                                          competed <span id='am-97' about='obo:NCBITaxon_1306155' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-98' property="rdfs:label" content="mixed culture" datatype="xsd:string"></span><span id='am-99'  ></span><span id='am-100' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>mixed culture</span> wild-type W3110 cells <br />                                                                          4 8 days incubation (Fig. 5). Other r-protein knock- <br />                                                                          outs grew normally mixed cultures period <br />                                                                          time (data shown). In rmf&#65533;?&#11022;bla knockout, rmf <br />                                                                          orf precisely replaced bla, Yamaguchi et al. (68) <br />                                                                          placed promoter-containing kan cassette middle the <br />                                                                          rmf gene, potentially causing polar effects expression of <br />                                                                          genes surrounding rmf. The competitive growth disadvantage <br />                                                                          rmf knockout evident quadruple gene <br />   FIG. 5. Survival rmf&#65533;?&#11022;bla quadruple ribosome-associated         knockout, defect quadruple <br /> protein mutants stationary phase. Stationary-phase cultures rmf    knockout experimental conditions (Fig. 5). Proper- <br /> and quadruple knockouts, W3110, grown 37&#176;C     ties conditionally essential ribosomal protein gene <br /> 24 h. The mixed cultures obtained combining 1 ml of      knockouts, rpsF, rpsI, rpsM, rpsO, rpsQ, rpsT, reported <br /> the 24-h culture knockouts 1 ml W3110 grown at <br />                                                                          (10). <br /> 37&#176;C 10 days (bottom panel) equal volume (2 ml) of <br /> the original monoculture (top panel). Samples taken indicated, <br /> and number Apr survivors total cell population                                  DISCUSSION <br /> counted plating LB-Ap plain LB plates, respectively. <br />                                                                              An understanding complex biological process be <br />                                                                          greatly facilitated identification essential components. <br /> however, exhibited different stability time (Fig. 4B). The          We developed procedure identify key components of <br /> nusG duplication quite stably inherited per-            bacterial systems, based essentiality growth and <br /> sisted 70% cells population 9           viability cells. Central procedure (Fig. 1) tech- <br /> days growth. The nusB duplication, contrary,               nique gene replacement chromosome using recom- <br /> quite unstable, 50% cells bearing duplication           bineering (12) antibiotic resistance orfs. The high effi- <br /> after 12 h growth (see Materials Methods).              ciency recombineering lack polar effects the <br />    Functional studies replacement mutants. An advantage               chromosomal orf gene replacements allow positive selection <br /> of single-copy, nonpolar, gene replacements          inactivated essential gene. Normally, essen- <br /> transferred different genetic background directly used          tial gene, recombination event replaces gene cell <br /> for functional studies. In work, transferred dis-            cell lost form colony. How- <br /> rupted alleles nusB nonessential ribosomal protein                ever, rare cases, recombination event occurs cell <br /> genes wild-type W3110 background investi-              naturally occurring duplication, replacing copy of <br /> gate properties.                                                   essential gene antibiotic resistance cassette and <br />    We growth single-copy nusB&#65533;?&#11022;cat              leaving second copy intact (Fig. 2). This gene duplication is <br /> impaired normal growth conditions greatly impaired             stably inherited comprise large segment the <br /> at temperatures 23&#176; 30&#176;C, indicating              chromosome. <br /> these circumstances NusB important cell growth,                  This approach utilized evaluate components two <br /> although absolutely essential. Initially,              macromolecular cellular systems, Nus transcription anti- <br /> nusB&#65533;?&#11022;cat cold sensitive ssyB63, loss-             termination functions (21, 50, 55) ribosomal proteins (9, <br /> of-function nusB::IS10 insertion mutant strain MC4100 (59).           16, 47), including set minor ribosome-associated proteins <br /> However, difference growth caused strain               (64). The Nus antitermination functions originally iden- <br /> background, demonstrated directly replacing ssyB63              tified play critical role life cycle phage <br /> in MC4100 nusB&#65533;?&#11022;cat using P1 transduction. In MC4100,               &#65533;?&#65533; temporal activation genes distal terminators (21). In <br /> nusB&#65533;?&#11022;cat grows like ssyB63 mutant (data shown).                 E. coli, factors modulate synthesis rRNA (44, 54, <br />    The minor ribosome-associated protein gene replacements               70). Although rRNA synthesis vitally important bacterial <br /> were fully functional temperatures rich           cells, E. coli tolerate substantial variability expression <br /> minimal medium did gross impairment major                rRNA genes, able survive deletion one <br /> biochemical pathways judged phenotypic microarrays                 copy seven rRNA genes (4). The NusA NusD/Rho <br /> (data shown). Although proteins increased lev-            functions previously suspected essential bacte- <br /> els stationary phase, gene knockouts grew normally           rial growth, conditional lethal mutants genes had <br /> stationary phase long period time. Even quadruple             isolated (13, 17, 39, 56). A nusG knockout shown to <br /> knockout mutant defective minor r-proteins              nonviable (72), nusA knockout obtained <br /> fully viable normal phenotypic microarray              presence attenuating rho mutations (71). At the <br /> analysis (data shown), demonstrating mod-             time, rho (45) nusG (27) knockouts Bacillus sub- <br /> ule minor r-proteins dispensable cell functions.          tilis, <span id='am-1' about='obo:NCBITaxon_1279' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1385 obo:NCBITaxon_90964 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-2' property="oboInOwl:hasDbXref" content="PMID:10319469" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Staphylococcus" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="PMID:10758876" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="PMID:12656157" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="PMID:9734063" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Aurococcus" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="PMID:10425778" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="PMID:10319495" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="PMID:17220435" datatype="xsd:string"></span><span id='am-13'  ></span>Staphylococcus</span> aureus rho knockout (66), had <br />  VOL. 189, 2007                                                                  IDENTIFICATION OF ESSENTIAL GENES                  2851 <br />  <br />  <br /> been reported viable. The NusB function             tively, <span id='am-94' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-95' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> protein assembly ribosomes be <br /> shown essential (44, 46, 59).                                 somewhat different vivo. <br />    In study, demonstrate nusA, nusD/rho, nusE/            Among set minor ribosome-associated proteins, <br /> rpsJ, nusG essential cell growth,        RMF (65, 68) YfiA (1, 2) proposed important <br /> nusB required temperatures 30&#176;C. Since         functions, directly inhibiting protein synthesis reducing <br /> NusB required rRNA antitermination vitro (54)        translation errors vitro. However, shown this <br /> and vivo (61), cell growth, Nus-dependent        study, minor proteins required vivo, in <br /> rRNA type antitermination essential cellular       stationary phase. They weakly conserved bac- <br /> process. Residual transcription read-through sufficient      teria, genes inactivated without <br /> in absence antitermination higher tempera-       affecting growth physiology cells. Within the <br /> tures. NusB binds NusE RNA sequence, boxA,          group, RMF needed cell survival stationary <br /> the rRNA leader (41). The boxA site important antiter-       phase, albeit competitive growth conditions, in <br /> mination (6, 22, 63). NusA NusG known interact           respect distinguishes itself proteins. <br /> with RNA polymerase involved antitermi-         At time, proteins, SRA, YfiaA, and <br /> nation elongation termination steps tran-       YhbH, support RMF function cells, rmf <br /> scription (11, 32, 56, 63). Rho transcription termination      knockout itself survives stationary phase somewhat bet- <br /> factor binds nascent RNA moves RNA            ter quadruple knockout minor r-proteins. <br /> interact RNA polymerase, causing termination &#8220;Rho              While paper preparation, global analysis of <br /> sites&#65533;? (49). It tempting hypothesize NusB     individual gene knockouts E. coli genes reported (5). <br /> only specific antitermination factor, Nus         Those authors employed approach orf gene replacement <br /> factors function generally modulating basic RNA         chromosome kan cassette carrying pro- <br /> polymerase activities transcription elongation. Perhaps          moter used plasmid-based Red recombination system, <br /> these functions NusA, NusG, Rho proteins de-           10-fold lower recombination efficiency than <br /> termine essential nature. NusE essential prob-    prophage-based (18) used work. Although <br />                                                                     method lacks means direct analysis essential genes <br /> ably structural component ribosomes. <br />                                                                     (see introduction features gene replacement <br />    Ribosomal proteins long thought being <br />                                                                     assay), list essential genes compiled default <br /> essential (16, 31, 67). We examined essentiality 30S <br />                                                                     genes knockout obtained. Their <br /> r-proteins E. coli. Six 21 ribosomal protein gene <br />                                                                     analysis nonessential essential genes does entirely <br /> knockouts, rpsF (S6), rpsI (S9), rpsM (S13), rpsO (S15), <br />                                                                     agree analysis genes tested work, i.e., for <br /> rpsQ (S17), rpsT (S20), viable defined our <br />                                                                     nus ribosomal protein genes. They identified rpsI (S9), <br /> conditions colony growth LB medium 34&#176;C. However, <br />                                                                     rpsM (S13), rpsQ (S17) essential, of <br /> these knockouts dramatically reduced cell growth 37&#176;C <br />                                                                     genes dispensable assay work by <br /> in W3110 did grow temperatures (data <br />                                                                     researchers (14, 26). Another gene list essen- <br /> not shown). The 15 r-proteins (S1 S5, S7, S8, S10 to <br />                                                                     tial genes viable E. coli knockout reported <br /> S12, S14, S16, S18, S19, S21) essential. The essential     RNase III gene, rnc, gene replacement was <br /> properties S1 (53), S2 (33), S16 (43),        tested way described (69). Baba <br /> viability S9 (26), S13 (14), S15 (36), S20 (51) gene     et al. (5) rho rpsU dispensable, although <br /> knockouts, demonstrated previously. Spontaneous           genes essential analyses. <br /> mutants lacking S6, S9, S13, S17, S20 protein           We suggest number reasons source disagree- <br /> been obtained Dabbs (16).                                        ment assessing gene essentiality method Baba et al. <br />    Analysis viable ribosomal protein gene knockouts      (5). The rnc gene known translationally coupled the <br /> provides interesting implications understanding mech-       essential era gene downstream (8, 57). The rpsM rpsQ <br /> anisms ribosome biogenesis functioning detailed       genes parts large gene operons translation- <br /> elsewhere (10). Initial analysis essentiality 30S ribo-   ally coupled essential neighboring genes, well. The <br /> somal proteins revealed number intriguing observa-         fusions generated Baba et al. (5) ensure translational <br /> tions. All nonessential proteins located rim   coupling downstream gene intermediate <br /> the 30S subunit platform (47), away          antibiotic resistance cassette replacement. The rpsI, rpsM, and <br /> important functional centers ribosome. Three      rpsQ knockouts problems condi- <br /> viable knockouts affected primary assembly proteins S15,        tions knockout selection. All knockouts exhibit poor <br /> S17, S20, supposed crucial initiating   growth hands quite sensitive concentra- <br /> subsequent cooperative binding secondary late assem-         tion Km selection medium. <br /> bly proteins. The secondary late assembly proteins             In case rho rpsU, unusually high <br /> essential, exception secondary protein S6. So    frequency antibiotic-resistant recombinants two <br /> seems ribosomes assemble vivo primary         genes compared essential gene knockouts tested (Ta- <br /> assembly proteins, directly demonstrated ribo-      ble 3, rho data). This explained regions <br /> somes lacking S15 (10) S20 (51). The principles           chromosome undergo recombination efficiently, as <br /> cooperative ordered assembly proteins ribosomes         shown segments chromosomal DNA (20). <br /> that deduced vitro experimentation (15, 42)        Alternatively, frequency partial diploids region <br /> may quite mimic ribosome biogenesis vivo. Alterna-       higher regions chromo- <br />  2852       BUBUNENKO ET AL.                                                                                                                             J. BACTERIOL. <br />  <br />  <br /> some. Note according assay Baba et al. (5),                                 effects NusA NusG transcription elongation Rho-dependent <br />                                                                                            termination Escherichia coli. J. Mol. Biol. 278:307&#8211;316. <br /> dispensable gene knockouts ones detected.                        12.   Court, D. L., J. A. Sawitzke, L. C. Thomason. 2002. Genetic engineering <br /> Their definition does account possibility                             using homologous recombination. Annu. Rev. Genet. 36:361&#8211;388. <br /> partial diploids. Experimental approaches analyze gene es-                        13.   Craven, M. G., D. I. Friedman. 1991. Analysis Escherichia coli <br />                                                                                            nusA10(Cs) allele: relating nucleotide changes phenotypes. J. Bacteriol. <br /> sentiality examined carefully, assay direct                            173:1485&#8211;1491. <br /> selection screening essential dispensable genes,                         14.   Cukras, A. R., R. Green. 2005. Multiple effects S13 modulating the <br /> like described study, required. We want                            strength intersubunit interactions ribosome translation. J. <br />                                                                                            Mol. Biol. 349:47&#8211;59. <br /> to point analysis gene knockout                         15.   Culver, G. M. 2003. Assembly 30S ribosomal subunit. Biopolymers <br /> account problems polarity transcription trans-                             68:234&#8211;249. <br />                                                                                      16.   Dabbs, E. R. 1991. Mutants lacking individual ribosomal proteins tool to <br /> lation, expression level antibiotic resistance cassette                         investigate ribosomal properties. Biochimie 73:639&#8211;645. <br /> used, presence duplications, specific growth                          17.   Das, A., D. Court, S. Adhya. 1976. Isolation characterization of <br /> conditions used.                                                                           conditional lethal mutants Escherichia coli defective transcription ter- <br />                                                                                            mination factor rho. Proc. Natl. Acad. Sci. USA 73:1959&#8211;1963. <br />    This study clarified understanding importance                      18.   Datta, S., N. Costantino, D. L. Court. 2006. A set recombineering <br /> of cellular systems: transcription antitermination                             plasmids gram-negative bacteria. Gene 379:109&#8211;115. <br /> system conventional ribosome minor ribosome-asso-                            19.   Deutscher, M. P., Z. Li. 2001. Exoribonucleases multiple roles <br />                                                                                            RNA metabolism. Prog. Nucleic Acid Res. Mol. Biol. 66:67&#8211;105. <br /> ciated proteins. Many cellular systems analyzed                      20.   Ellis, H. M., D. Yu, T. DiTizio, D. L. Court. 2001. High efficiency <br /> this manner, e.g., modules ribosome biogenesis net-                             mutagenesis, repair, engineering chromosomal DNA using single- <br /> work, rRNAs (4, 47) rRNA maturation                                  stranded oligonucleotides. Proc. Natl. Acad. Sci. USA 98:6742&#8211;6746. <br />                                                                                      21.   Friedman, D. I., D. L. Court. 1995. Transcription antitermination: the <br /> system (3, 19) partially understood.                                                   lambda paradigm updated. Mol. Microbiol. 18:191&#8211;200. <br />                                                                                      22.   Friedman, D. I., M. Gottesman. 1981. Lytic mode lambda develop- <br />                                                                                            ment, p. 21&#8211;51. In R. W. Hendrix, J. W. Roberts, F. W. Stahl, R. A. <br />                           ACKNOWLEDGMENTS                                                  Weisberg (ed.), Lambda II. Cold Spring Harbor Laboratory, Cold Spring <br />                                                                                            Harbor, N.Y. <br />   We thank Nina Costantino, Simanti Datta, Craig Laufer, James                       23.   Gerdes, S. Y., M. D. Scholle, J. W. Campbell, G. Balazsi, E. Ravasz, M. D. <br /> Sawitzke, Lynn Thomason, Xiaomei Zhou discussions                             Daugherty, A. L. Somera, N. C. Kyrpides, I. Anderson, M. S. Gelfand, A. <br /> and help manuscript. We especially thankful Mathias                        Bhattacharya, V. Kapatral, M. D&#8217;Souza, M. V. Baev, Y. Grechkin, F. Mseeh, <br /> Springer valuable comments ribosomal                            M. Y. Fonstein, R. Overbeek, A. L. Barabasi, Z. N. Oltvai, A. L. Oster- <br /> manuscript. We wish thank Nina Bubunenko help                           man. 2003. Experimental determination level analysis essential <br /> preparation purified oligonucleotides bioinformatics analysis                    genes Escherichia coli MG1655. J. Bacteriol. 185:5673&#8211;5684. <br /> essential genes.                                                                     24.   Glass, J. I., N. Assad-<span id='am-66' about='obo:NCBITaxon_316718' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3646 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_235631 obo:NCBITaxon_3977 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_235892 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-68'  ></span><span id='am-69' property="rdfs:label" content="Garcia" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Garcia</span>, N. Alperovich, S. Yooseph, M. R. Lewis, M. <br />   This research supported Intramural Research                           Maruf, C. A. Hutchinson III, H. O. Smith, J. C. Venter. 2006. Essential <br />                                                                                            genes minimal bacterium. Proc. Natl. Acad. Sci. USA 103:425&#8211;430. <br /> Program NIH National Cancer Institute, Center Cancer <br />                                                                                      25.   Hill, C. W., B. W. Harnish. 1981. Inversions ribosomal RNA <br /> Research (N01-CO-12400). The research supported Trans-                            genes Escherichia coli. Proc. Natl. Acad. Sci. USA 78:7069&#8211;7072. <br /> NIH/FDA Intramural Biodefense Program grant National                        26.   Hoang, L., K. Fredrick, H. F. Noller. 2004. Creating ribosomes with <br /> Institute Allergy Infectious Diseases (D.L.C.).                                     all-RNA 30S subunit P-site. Proc. Natl. Acad. Sci. USA 101:12439&#8211;12443. <br />   The content publication does necessarily reflect views             27.   Ingham, C. J., J. Dennis, P. A. Furneaux. 1999. Autogenous regulation <br /> or policies Department Health Human Services, does                       transcription termination factor Rho requirement Nus factors <br /> mention trade names, commercial products, organizations imply                        <span id='am-28' about='obo:NCBITaxon_1423' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_653685'><span id='am-29' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429428" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429427" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232841" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232825" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232822" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasAlternativeId" content="NCBITaxon:387338" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042382" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis/Bacillus globigii" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Bacillus natto" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Bacillus uniflagellatus" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis8" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasRelatedSynonym" content="Vibrio subtilis" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasAlternativeId" content="NCBITaxon:941949" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="PMID:11211269" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042381" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasAlternativeId" content="NCBITaxon:981081" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232819" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232840" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Bacillus subtilis" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451272" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasAlternativeId" content="NCBITaxon:523730" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224247" datatype="xsd:string"></span><span id='am-51'  ></span><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232824" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232828" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232470" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451270" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232830" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232844" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042380" datatype="xsd:string"></span>Bacillus subtilis</span>. Mol. Microbiol. 31:651&#8211;663. <br /> endorsement U.S. Government.                                                  28.   Izutsu, K., C. Wada, Y. Komine, T. Sako, C. Ueguchi, S. Nakura, A. <br />                                                                                            Wada. 2001. Escherichia coli ribosome-associated protein SRA, copy <br />                                                                                            number increases stationary phase. J. Bacteriol. 183:2765&#8211;2773. <br />                                  REFERENCES                                          29.   Judson, N., J. J. Mekalanos. 2000. Transposon-based approaches to <br />  1. Agafonov, D. E., V. A. Kolb, A. S. Spirin. 2001. A novel stress-response           identify essential bacterial genes. Trends Microbiol. 8:521&#8211;526. <br />     protein binds ribosomal subunit interface arrests translation.   30.   Knowlton, J. R., M. Bubunenko, M. Andrykovitch, W. Guo, K. M. Routzahn, <br />     Cold Spring Harbor Symp. Quant. Biol. 66:509&#8211;514.                                      D. S. Waugh, D. L. Court, X. Ji. 2003. A spring-loaded state NusG in <br />  2. Agafonov, D. E., A. S. Spirin. 2004. The ribosome-associated inhibitor A           functional cycle suggested X-ray crystallography supported by <br />     reduces translational errors. Biochem. Biophys. Res. Commun. 320:354&#8211;358.              site-directed mutants. Biochemistry 42:2275&#8211;2281. <br />  3. Apirion, D., A. Miczak. 1993. RNA processing prokaryotic cells.           31.   Koonin, E. V. 2000. How genes make cell: minimal-gene set <br />     Bioessays 15:113&#8211;120.                                                                  concept. Annu. Rev. Genomics Hum. Genet. 1:99&#8211;116. <br />  4. Asai, T., D. Zaporojets, C. Squires, C. L. Squires. 1999. An Escherichia     32.   Li, J., S. W. Mason, J. Greenblatt. 1993. Elongation factor NusG inter- <br />     coli strain chromosomal rRNA operons inactivated: complete ex-                acts termination factor Rho regulate termination antitermination <br />     change rRNA genes bacteria. Proc. Natl. Acad. Sci. USA 96:                  transcription. Genes Dev. 7:161&#8211;172. <br />     1971&#8211;1976.                                                                       33.   Link, A. J., D. Phillips, G. M. Church. 1997. Methods generating <br />  5. Baba, T., T. <span id='am-79' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-80' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-82'  ></span><span id='am-83' property="rdfs:label" content="Ara" datatype="xsd:string"></span>Ara</span>, M. Hasegawa, Y. Takai, Y. Okumura, M. Baba, K. A.                    precise deletions insertions genome wild-type Escherichia coli: <br />     Datsenko, M. Tomita, B. L. Wanner, H. Mori. 2006. Construction                  application open reading frame characterization. J. Bacteriol. 179:6228&#8211; <br />     <span id='am-137' about='obo:NCBITaxon_83333' typeof='owl:Thing obo:NCBITaxon_562 obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-138' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasExactSynonym" content="Escherichia coli K12" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-141' property="rdfs:label" content="Escherichia coli K-12" datatype="xsd:string"></span>Escherichia coli K-12</span> in-frame, single-gene knockout mutants: Keio                 6237. <br />     collection. Mol. Syst. Biol. 2:2006.0008.                                        34.   Lupski, J. R., J. R. Roth, G. M. Weinstock. 1996. Chromosomal dupli- <br />  6. Berg, K., C. Squires, C. L. Squires. 1989. Ribosomal RNA operon                    cations bacteria, fruit flies, humans. Am. J. Hum. Genet. 58:21&#8211;27. <br />     antitermination. Function leader spacer region boxB-boxA sequences        35.   Maki, Y., H. Yoshida, A. Wada. 2000. Two proteins, YfiA YhbH, <br />     conservation diverse microorganisms. J. Mol. Biol. 209:345&#8211;358.           associated resting ribosomes stationary phase Escherichia coli. Genes <br />  7. Bochner, B. R., P. Gadzinski, E. Panomitros. 2001. Phenotype microar-              Cells 5:965&#8211;974. <br />     rays high throughput phenotypic testing assay gene function.          36.   Mathy, N., O. Pellegrini, A. Serganov, D. J. Patel, C. Ehresmann, C. <br />     Genome Res. 11:1246&#8211;1255.                                                              Portier. 2004. Specific recognition rpsO mRNA 16S rRNA Esch- <br />  8. Britton, R. A., B. S. Powell, S. Dasgupta, O. Sun, W. Margolin, J. R. Lupski,          erichia coli ribosomal protein S15 relies mimicry site differen- <br />     D. L. Court. 1998. Cell cycle arrest Era GTPase mutants: potential            tiation. Mol. Microbiol. 52:661&#8211;675. <br />     growth rate-regulated checkpoint Escherichia coli. Mol. Microbiol. 28:        37.   Miller, J. H. 1972. Experiments molecular genetics, Cold Spring Harbor <br />     1391&#8211;1393.                                                                             Laboratory, Cold Spring Harbor, N.Y. <br />  9. Brodersen, D. E., P. Nissen. 2005. The social life ribosomal proteins.    38.   Moran, N. A.. Microbial minimalism: genome reduction bacterial patho- <br />     FEBS J. 272:2098&#8211;2108.                                                                 gens. Cell 108:583&#8211;586. <br /> 10. Bubunenko, M., A. Korepanov, D. L. Court, I. Jagannathan, D. Dickinson,          39.   Nakamura, Y., A. Mizusawa, A. Tsugawa, M. Imai. 1986. Conditionally <br />     B. R. Chaudhuri, M. B. Garber, G. M. Culver. 2006. 30S ribosomal                   lethal nusAts mutation Escherichia coli reduces transcription termination <br />     subunits assembled vivo primary binding ribosomal                  does affect antitermination bacteriophage lambda. Mol. Gen. <br />     protein S15. RNA 12:1229&#8211;1239.                                                         Genet. 204:24&#8211;28. <br /> 11. Burns, C. M., L. V. Richardson, J. P. Richardson. 1998. Combinatorial        40.   Nichols, B. P., O. Shafiq, V. Meiners. 1998. Sequence analysis Tn10 <br />  VOL. 189, 2007                                                                                        IDENTIFICATION OF ESSENTIAL GENES                              2853 <br />  <br />       insertion sites collection Escherichia coli strains used genetic      59. Taura, T., C. Ueguchi, K. Shiba, K. Ito. 1992. Insertional disruption of <br />       mapping strain construction. J. Bacteriol. 180:6408&#8211;6411.                         nusB (ssyB) gene leads cold-sensitive growth Escherichia coli and <br /> 41.   Nodwell, J. R., J. Greenblatt. 1993. Recognition boxA antiterminator           suppression secY24 mutation. Mol. Gen. Genet. 234:429&#8211;432. <br />       RNA E. coli antitermination factors NusB ribosomal protein S10.        60. Thomason, L. C., M. Bubunenko, N. Costantino, H. R. Wilson, A. Openheim, <br />       Cell 72:261&#8211;268.                                                                      S. Datta, D. L. Court. 2005. Recombineering: genetic engineering in <br /> 42.   Nomura, M. 1973. Assembly bacterial ribosomes. Science 179:864&#8211;873.                bacteria using homologous recombination, p. 1.16.11&#8211;1.16.21. In F. M. <br /> 43.   Persson, B. C., G. O. Bylund, D. E. Berg, P. M. Wikstrom. 1995.                   Ausubel, R. Brent, R. E. Kingston, D. D. Moore, J. G. Seidman, J. A. <br />       Functional analysis ffh-trmD region Escherichia coli chromo-            Smith, K. Struhl (ed.), Current protocols molecular biology. Wiley, <br />       using reverse genetics. J. Bacteriol. 177:5554&#8211;5560.                          New York, N.Y. <br /> 44.   Quan, S., N. Zhang, S. French, C. L. Squires. 2005. Transcription             61. Torres, M., J.-M. Balada, M. Zellars, C. Squires, C. L. Squires. 2004. In <br />       polarity rRNA operons Escherichia coli nusA nusB mutant strains.            vivo effect NusB NusG rRNA transcription termination. J. Bac- <br />       J. Bacteriol. 187:1632&#8211;1638.                                                          teriol. 186:1304&#8211;1310. <br /> 45.   Quirk, P. G., E. A. Dunkley, Jr., P. Lee, T. A. Krulwich. 1993. Identifi- <br />                                                                                         62. Ueta, M., H. Yoshida, C. Wada, T. Baba, H. Mori, A. Wada. 2005. <br />       cation putative Bacillus subtilis rho gene. J. Bacteriol. 175:647&#8211;654. <br />                                                                                             Ribosome binding proteins YhbH YfiA opposite functions during <br /> 46.   Rajapandi, T., D. Oliver. 1994. ssaD1, suppressor secA51(Ts) that <br />       renders growth Escherichia coli cold sensitive, early amber mutation         100S formation stationary phase Escherichia coli. Genes Cells <br />       transcription factor gene nusB. J. Bacteriol. 176:4444&#8211;4447.                   10:1103&#8211;1112. <br /> 47.   Ramakrishnan, V. 2002. Ribosome structure mechanism transla-           63. Vogel, U., K. F. Jensen. 1997. NusA required ribosomal antiter- <br />       tion. Cell 108:557&#8211;572.                                                               mination modulation transcription elongation rate both <br /> 48.   Reich, K. A. 2000. The search essential genes. Res. Microbiol. 151:319&#8211;           antiterminated RNA mRNA. J. Biol. Chem. 272:12265&#8211;12271. <br />       324.                                                                              64. Wada, A. 1998. Growth phase coupled modulation Escherichia coli ribo- <br /> 49.   Richardson, J. P. 2002. Rho-dependent termination ATPase tran-                 somes. Genes Cells 3:203&#8211;208. <br />       script termination. Biochim. Biophys. Acta 1577:251&#8211;260.                          65. Wada, A., K. Igarashi, S. Yoshimura, S. Aimoto, A. Ishihama. 1995. <br /> 50.   Roberts, J. W. 1993. RNA protein elements E. coli lambda                   Ribosome modulation factor: stationary growth phase-specific inhibitor of <br />       transcription antitermination complexes. Cell 72:653&#8211;655.                             ribosome functions Escherichia coli. Biochem. Biophys. Res. Commun. <br /> 51.   Ryden-Aulin, M., Z. Shaoping, P. Kylsten, L. A. Isaksson. 1993. Ribo-             214:410&#8211;417. <br />       activity modification 16S RNA influenced deletion           66. Washburn, R. S., A. Marra, A. P. Bryant, M. Rosenberg, D. R. Gentry. <br />       ribosomal protein S20. Mol. Microbiol. 7:983&#8211;992.                                     2001. rho essential viability virulence <span id='am-14' about='obo:NCBITaxon_1280' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1385 obo:NCBITaxon_90964 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_1279'><span id='am-15' property="oboInOwl:hasDbXref" content="PMID:8573498" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Staphilococcus aureus" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Micrococcus pyogenes" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Staphylococcus pyogenes aureus" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Staphylococus aureus" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasAlternativeId" content="NCBITaxon:325213" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Micrococcus aureus" datatype="xsd:string"></span><span id='am-23'  ></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Staphlococcus pyogenes citreus" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Staphylococcus aureus" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Streptococcus aureus" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Staphylococcus aureus</span>. <br /> 52.   Singer, M., T. A. Baker, G. Schnitzler, S. M. Deischel, M. Goel, W. Dove,             Antimicrob. Agents Chemother. 45:1099&#8211;1103. <br />       K. J. Jaacks, A. D. Grossman, J. W. Erickson, C. A. Gross. 1989. A            67. Wilson, D. N., K. H. Nierhaus. 2005. Ribosomal proteins spotlight. <br />       collection strains containing genetically linked alternating antibiotic re-        Crit. Rev. Biochem. Mol. Biol. 40:243&#8211;267. <br />       sistance elements genetic mapping Escherichia coli. Microbiol. Rev.        68. Yamaguchi, M., H. Matsushima, A. Wada, M. Sakagami, N. Fujita, A. <br />       53:1&#8211;24.                                                                              Ishihama. 1993. Regulation Escherichia coli rmf gene encoding the <br /> 53.   Sorensen, M. A., J. Fricke, S. Pedersen. 1998. Ribosomal protein S1            ribosome modulation factor: growth phase- growth rate-dependent con- <br />       required translation most, all, natural mRNAs Escherichia coli <br />                                                                                             trol. EMBO J. 12:625&#8211;630. <br />       vivo. J. Mol. Biol. 280:561&#8211;569. <br />                                                                                         69. Yu, D., H. M. Ellis, E. C. Lee, N. A. Jenkins, N. G. Copeland, D. L. <br /> 54.   Squires, C. L., J. Greenblatt, J. Li, C. Condon, C. Squires. 1993. Ribo- <br />                                                                                             Court. 2000. An efficient recombination chromosome engineering <br />       somal RNA antitermination vitro: requirement Nus factors or <br />       <span id='am-152' about='obo:NCBITaxon_32644' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-153' property="rdfs:label" content="unidentified" datatype="xsd:string"></span><span id='am-154' property="oboInOwl:hasRelatedSynonym" content="unknown organism" datatype="xsd:string"></span><span id='am-155' property="oboInOwl:hasRelatedSynonym" content="none" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-157' property="oboInOwl:hasRelatedSynonym" content="unclassified sequence" datatype="xsd:string"></span><span id='am-158' property="oboInOwl:hasRelatedSynonym" content="sonstige nucleic acid" datatype="xsd:string"></span><span id='am-159' property="oboInOwl:hasRelatedSynonym" content="unidentified organism" datatype="xsd:string"></span><span id='am-160' property="oboInOwl:hasRelatedSynonym" content="not shown" datatype="xsd:string"></span><span id='am-161' property="oboInOwl:hasRelatedSynonym" content="unspecified" datatype="xsd:string"></span><span id='am-162' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-163' property="oboInOwl:hasRelatedSynonym" content="unknown" datatype="xsd:string"></span><span id='am-164' property="oboInOwl:hasRelatedSynonym" content="miscellaneous nucleic acid" datatype="xsd:string"></span><span id='am-165' property="oboInOwl:hasRelatedSynonym" content="unknwon" datatype="xsd:string"></span><span id='am-166'  ></span><span id='am-167' property="oboInOwl:hasRelatedSynonym" content="not specified" datatype="xsd:string"></span><span id='am-168' property="oboInOwl:hasRelatedSynonym" content="other" datatype="xsd:string"></span>unidentified</span> cellular components. Proc. Natl. Acad. Sci. USA 90:970&#8211;             Escherichia coli. Proc. Natl. Acad. Sci. USA 97:5978&#8211;5983. <br />       974.                                                                              70. Zellars, M., C. L. Squires. 1999. Antiterminaor-dependent modulation <br /> 55.   Squires, C. L., D. Zaporojets. 2000. Proteins shared transcription         transcription elongation rates NusB NusG. Mol. Microbiol. 32: <br />       translation machines. Annu. Rev. Microbiol. 54:775&#8211;798.                           1296&#8211;1304. <br /> 56.   Sullivan, S. L., M. Gottesman. 1992. Requirement E. coli NusG             71. Zheng, C., D. I. Friedman. 1994. Reduced Rho-dependent transcription <br />       protein factor-dependent transcription termination. Cell 68:989&#8211;994.               termination permits NusA-independent growth Escherichia coli. Proc. <br /> 57.   Takiff, H. E., S. M. Chen, D. L. Court. 1989. Genetic analysis rnc         Natl. Acad. Sci. USA 91:7543&#8211;7547. <br />       operon Escherichia coli. J. Bacteriol. 171:2581&#8211;2590.                          72. Zhou <br /> </body></html>