<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />    RNA Polymerase II Elongation Factors of <span id='am-33' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-34' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-36'  ></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>: <br />                  a Targeted Proteomics Approach <br />  <br />              To physically characterize the web of interactions connecting the <span id='am-50' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-51'  ></span><span id='am-52' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae proteins suspected <br />           to be RNA polymerase II (RNAPII) elongation factors, subunits of Spt4/Spt5 and Spt16/Pob3 (corresponding <br />           to human DSIF and FACT), Spt6, TFIIF (Tfg1, -2, and -3), TFIIS, Rtf1, and Elongator (Elp1, -2, -3, -4, -5, and <br />           -6) were affinity purified under conditions designed to minimize loss of associated polypeptides and then <br />           identified by mass spectrometry. Spt16/Pob3 was discovered to associate with three distinct complexes: his- <br />           tones; Chd1/casein kinase II (CKII); and Rtf1, Paf1, Ctr9, Cdc73, and a previously uncharacterized protein, <br />           Leo1. Rtf1 and Chd1 have previously been implicated in the control of elongation, and the sensitivity to <br />           6-azauracil of strains lacking Paf1, Cdc73, or Leo1 suggested that these proteins are involved in elongation by <br />           RNAPII as well. Confirmation came from chromatin immunoprecipitation (ChIP) assays demonstrating that <br />           all components of <span id='am-65' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-66'  ></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="This" datatype="xsd:string"></span>this</span> complex, including Leo1, cross-linked to the promoter, coding region, and 3&#11541; end of the <br />           ADH1 gene. In contrast, the three subunits of TFIIF cross-linked only to the promoter-containing fragment of <br />           ADH1. Spt6 interacted with the uncharacterized, essential protein Iws1 (interacts with Spt6), and Spt5 <br />           interacted either with Spt4 or with a truncated form of Spt6. ChIP on Spt6 and the novel protein Iws1 resulted <br />           in the cross-linking of both proteins to all three regions of the ADH1 gene, suggesting that Iws1 is likely an <br />           Spt6-interacting elongation factor. Spt5, Spt6, and Iws1 are phosphorylated on consensus CKII sites in vivo, <br />           conceivably by the Chd1/CKII associated with Spt16/Pob3. All the elongation factors but Elongator copurified <br />           with RNAPII. <br />  <br />  <br />    The synthesis of functional mRNA precursors by eukaryotic                 sylbenzimidazole [DRB] sensitivity-inducing factor [DSIF], <br /> RNA polymerase II (RNAPII) is a multistage process consist-                  which is required for activation of elongation by human im- <br /> ing of three major steps: initiation, elongation, and termina-               munodeficiency virus Tat) (46), Elongator (30), Spt6 (15), and <br /> tion. Although much has been learned about steps involved in                 Spt16/Pob3 (corresponding to human FACT [facilitates chro- <br /> initiation, including formation of the initiation complex, initi-            matin transcription]) (28). Elongator copurified with the elon- <br /> ation of the RNA chain, and promoter clearance, much less is                 gating form of RNAPII, and it was shown to bind preferentially <br /> known about later stages in transcription, especially chain                  to the hyperphosphorylated form of RNAPII in vitro (30). The <br /> elongation. It is known, however, that the transition from tran-             Elp3 subunit of Elongator has histone acetyltransferase (HAT) <br /> scriptional initiation to elongation is associated with a change             activity in vitro (48), suggesting that the HAT activity of Elp3 <br /> in the factors that are associated with RNAPII.                              may assist RNAPII during transcription elongation on a chro- <br />    Whereas general transcription factors are required for pro-               matin template. Human FACT stimulates transcriptional elon- <br /> moter-directed transcription initiation, and the multisubunit                gation on a chromatin template in a partially defined in vitro <br /> Srb/Mediator complex is associated with the RNAPII holoen-                   system and has been shown to interact with nucleosomes and <br /> zyme that binds to promoters in the yeast Saccharomyces cer-                 histone H2A/H2B dimers (28, 29). Recently, FACT has also <br /> evisiae, RNAPII can elongate the transcript unaided. However,                been identified in a complex containing human casein kinase II <br /> several factors have been discovered that stimulate elongation               (CKII) that phosphorylates the tumor suppressor protein p53 <br /> on nonchromatin, &#8220;naked&#8221; DNA templates (e.g., TFIIS and                      (18). <br /> TFIIF) (32, 47). Recently, other factors have been identified                   Purification of proteins interacting with and affecting the <br /> that are thought to affect RNAPII transcript elongation via                  activity of RNAPII has generally involved either conventional <br /> effects on chromatin. These include the yeast factors Spt4 and <br />                                                                              chromatography or RNAPII affinity chromatography, a proce- <br /> Spt5 (corresponding to human 5,6-dichloro-1-&#9252;-D-ribofurano- <br />                                                                              dure that identifies human TFIIF (RAP30/RAP74) and TFIIS <br />                                                                              (38). However, conventional or affinity purifications usually <br />                                                                              require either ion exchangers and hydrophobic columns (in the <br />   * Corresponding author. Mailing address: Banting and Best Depart- <br /> ment of Medical Research, University of Toronto, 112 College St., <br />                                                                              former case) or high salt concentrations (in the latter case) <br /> Rm. 210, Toronto, Ontario, Canada M5G 1L6. Phone: (416) 978-4141.            to reduce nonspecific protein binding to the column matrix. <br /> Fax: (416) 978-8528. E-mail: jack.greenblatt@utoronto.ca.                    These conditions could potentially disrupt interactions that <br />  <br />                                                                       6979 <br />  6980      KROGAN ET AL.                                                                                                             MOL. CELL. BIOL. <br />  <br />  <br />             TABLE 1. Yeast strains used in this study                                           MATERIALS AND METHODS <br />  <br />                                                           Reference        Yeast strain construction. All of the strains used in this study (Table 1) are <br />  Strain                      Genotype                                   congenic with W303-1A (42). Tags were fused to genes with a PCR-based <br />                                                           or source <br />                                                                         one-step in vivo tagging strategy (3) using the TAP tag vector pBS1479 (33). <br /> W303-1A       MATa ura3-1 leu2-3,112 his3-11,15 trp1-1   39             Transformed cells (11) were plated onto synthetic medium lacking tryptophan. <br />                 ade2-1 can1-100                                         Integration of the tag-encoding DNA was confirmed by PCR and then Western <br /> YCK245        As W303-1A but trp1 &#9004;                      23             blotting with nonspecific rabbit immunoglobulin to recognize the protein A <br /> YNJK1         As W303-1A, but trp1 &#9004; elp1-TAP::TRP1      23             component of the TAP tag. Strains with the following genes replaced by a <br /> YNJK2         As W303-1A, but trp1 &#9004; elp3-TAP::TRP1      23 <br />                                                                         kanamycin resistance cassette were obtained from Research Genetics (http: <br /> YNJK3         As W303-1A, but trp1 &#9004; spt4-TAP::TRP1      This   study <br />                                                                         //sequence-www.stanford.edu/group/yeast_deletion_project/deletions3.html): <br /> YNJK4         As W303-1A, but trp1 &#9004; spt5-TAP::TRP1      This   study <br /> YNJK6         As W303-1A, but trp1 &#9004; spt6-TAP::TRP1      This   study   CHD1, DST1, PAF1, CDC73, RTF1, and LEO1. Sensitivity to 6-azauracil was <br /> YNJK7         As W303-1A, but trp1 &#9004; spt16-TAP::TRP1     This   study   tested by plating strains harboring pRS316 (37) onto plates lacking uracil and <br /> YNJK8         As W303-1A, but trp1 &#9004; pob3-TAP::TRP1      This   study   containing 25 &#9262;g of 6-azauracil/ml. <br /> YNJK9         As W303-1A, but trp1 &#9004; dst1-TAP::TRP1      This   study      Protein purification. Tagged complexes implicated in transcriptional elonga- <br /> YNJK11        As W303-1A, but trp1 &#9004; iws1-TAP::TRP1      This   study   tion were purified on immunoglobulin G (IgG) and calmodulin columns from <br /> YNJK12        As W303-1A, but trp1 &#9004; rpb3-TAP::TRP1      This   study   extracts of yeast cells (3 liters) grown in yeast extract-peptone-dextrose (YEPD) <br /> YNJK15        As W303-1A, but trp1 &#9004; tfg1-TAP::TRP1      This   study   medium to an optical density at 600 nm (OD600) of 1.0 to 1.5. Cell pellets (7 to <br /> YNJK16        As W303-1A, but trp1 &#9004; tfg2-TAP::TRP1      This   study   10 g) were frozen in liquid nitrogen and lysed by grinding with dry ice in a Krups <br /> YNJK17        As W303-1A, but trp1 &#9004; tfg3-TAP::TRP1      This   study   coffee grinder (model 203-70). An equal volume of YEB (250 mM KCl, 100 mM <br /> YNJK23        As W303-1A, but trp1 &#9004; chd1-TAP::TRP1      This   study   HEPES-KOH [pH 7.9], 1 mM EDTA, 2.5 mM dithiothreitol [DTT]) was added, <br /> YNJK44        As W303-1A, but trp1 &#9004; elp2-TAP::TRP1      23             and following centrifugation in a Beckman 70 Ti rotor at 4&#176;C for 2 h at 34,000 <br /> YNJK45        As W303-1A, but trp1 &#9004; elp4-TAP::TRP1      23             rpm, the supernatant was dialyzed against IPP buffer (10 mM Tris-Cl [pH 7.9], <br /> YNJK46        As W303-1A, but trp1 &#9004; elp5-TAP::TRP1      23 <br />                                                                         0.1% Triton X-100, 0.5 mM DTT, 0.2 mM EDTA, 20% glycerol containing 100, <br /> YNJK74        As W303-1A, but trp1 &#9004; rtf1-TAP::TRP1      This   study <br />                                                                         125, 150, or 200 mM NaCl). After <span id='am-60' about='obo:NCBITaxon_124307' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7147 obo:NCBITaxon_7203 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_92613 obo:NCBITaxon_43736 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_125599 obo:NCBITaxon_33392 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_50557 obo:NCBITaxon_6960 obo:NCBITaxon_6072'><span id='am-61' property="rdfs:label" content="Dialysis" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-64'  ></span>dialysis</span>, the extract was again centrifuged in <br /> YNJK78        As W303-1A, but trp1 &#9004; leo1-TAP::TRP1      This   study <br /> YNJK81        As W303-1A, but trp1 &#9004; cdc73-TAP::TRP1     This   study   a 70 Ti rotor at 4&#176;C for 30 min at 34,000 rpm, and the supernatant was mixed for <br /> YNJK91        As W303-1A, but trp1 &#9004; paf1-TAP::TRP1      This   study   3 h with 200 &#9262;l of IgG-Sepharose (Pharmacia) equilibrated with IPP buffer. <br /> YNJK130       As W303-1A, but trp1 &#9004; ctr9-TAP::TRP1      This   study   Following binding, the IgG-Sepharose was washed with 1 ml of IPP buffer <br />                                                                         followed by 400 &#9262;l of TEV protease cleavage buffer (50 mM Tris-Cl [pH 7.9], <br />                                                                         1 mM DTT, and 0.1% Triton X-100, containing 200, 150, 125, or 100 mM NaCl). <br />                                                                         The beads were then incubated overnight at 4&#176;C with 100 U of TEV protease <br />                                                                         (Life Technologies) in 200 &#9262;l of TEV cleavage buffer. The eluate was combined <br />                                                                         with a 200-&#9262;l wash with TEV cleavage buffer. To this was added 200 &#9262;l of <br /> stabilize some protein complexes. To circumvent these prob-             calmodulin binding buffer (10 mM Tris-Cl [pH 7.9], 10 mM &#9252;-mercaptoethanol, <br /> lems, affinity tags that are fused to selected target proteins have     2 mM CaCl2, and 0.1% Triton X-100 containing 200, 150, 125, or 100 mM NaCl) <br /> been widely used to purify protein complexes. In most cases,            and 200 &#9262;l of calmodulin beads (Pharmacia) equilibrated with the same buffer. <br />                                                                         After binding for 1 to 2 h at 4&#176;C, the calmodulin beads were washed with 200 &#9262;l <br /> however, it has again been necessary to use high salt concen-           of calmodulin binding buffer and 200 &#9262;l of calmodulin wash buffer (10 mM <br /> trations to reduce nonspecific binding of proteins to solid sup-        Tris-Cl [pH 7.9], 10 mM &#9252;-mercaptoethanol, 0.1 mM CaCl2, and 0.1% Triton <br /> ports.                                                                  X-100, containing 200, 150, 125, or 100 mM NaCl). The purified protein com- <br />    Recently, a tandem affinity purification (TAP) scheme, in            plexes were eluted from the calmodulin beads with 5 times with 100 &#9262;l of <br />                                                                         calmodulin elution buffer (10 mM Tris-Cl [pH 7.9], 10 mM &#9252;-mercaptoethanol, <br /> which two adjacent tags (calmodulin-binding peptide and <br />                                                                         3 mM EGTA [pH 8.0], and 0.1% Triton X-100, containing 200, 150, 125, or 100 <br /> <span id='am-1' about='obo:NCBITaxon_1279' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1385 obo:NCBITaxon_90964 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-2' property="oboInOwl:hasDbXref" content="PMID:10319495" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="PMID:17220435" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="PMID:10758876" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="Staphylococcus" datatype="xsd:string"></span><span id='am-7'  ></span><span id='am-8' property="oboInOwl:hasDbXref" content="PMID:10425778" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="PMID:9734063" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="PMID:10319469" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Aurococcus" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="PMID:12656157" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Staphylococcus</span> aureus protein A) separated by a tobacco etch            mM NaCl), and the eluted protein was lyophilized or concentrated by precipi- <br /> virus (TEV) protease site were fused to the protein of interest,        tation with acetone or trichloroacetic acid (TCA). Purified proteins were sepa- <br /> was described (33). The use of two tags and, consequently, two          rated by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS- <br /> independent purification steps allowed protein purification to          PAGE) on gels containing 10% polyacrylamide (except where indicated <br />                                                                         otherwise), and the proteins were visualized by silver staining. Western blotting <br /> be carried out under native conditions at near-physiological            was performed using standard techniques with monoclonal antibody 8WG16 <br /> ionic strengths while still maintaining a low background (de-           (43). <br /> fined as the protein yield from an extract derived from an                 Protein identification. Protein bands were reduced, alkylated, and subjected to <br /> untagged strain). Under these circumstances, protein com-               in-gel tryptic digestion, and the peptides were then purified and identified by <br />                                                                         matrix-assisted <span id='am-88' about='obo:NCBITaxon_62990' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_364270 obo:NCBITaxon_131567 obo:NCBITaxon_241799 obo:NCBITaxon_91882 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_241789 obo:NCBITaxon_4037 obo:NCBITaxon_241778 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4036 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-89' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="Laser" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-92'  ></span>laser</span> desorption ionization&#8211;time-of-flight (MALDI-TOF) mass <br /> plexes containing proteins that are weakly associated in vivo <br />                                                                         spectrometry with a PerSeptive DE STR instrument (26). For tandem mass <br /> have a greater chance of remaining associated during the pu-            spectrometry, the TCA-precipitated protein was resuspended in 100 mM <br /> rification procedure.                                                   NH4HCO3/1 mM CaCl2 buffer, pH 8.5, and digested overnight at 37&#176;C with 2 &#9262;l <br />    Most of the RNAPII elongation factors of S. cerevisiae are           of immobilized Poros trypsin beads (PerSeptive). The entire digest was fraction- <br /> poorly characterized. Moreover, conventional purifications of           ated as described previously (10) on a 7.5-cm (inner diameter, 100 &#9262;m) reverse- <br />                                                                         phase C18 capillary column attached in-line to a Finnigan LCQ-Deca ion trap <br /> human proteins previously implicated in transcriptional elon-           mass spectrometer by ramping a linear gradient from 2 to 60% solvent B in 90 <br /> gation may have resulted in the loss of one or more associated          min. Solvent A consisted of 5% acetonitrile, 0.5% acetic acid, and 0.02% hep- <br /> proteins. In <span id='am-80' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-81' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> to systematically characterize the yeast tran-       tafluorobutanoic acid (HFBA), and solvent B consisted of acetonitrile-water <br /> scriptional elongation factors, we devised a modification of the        (80:20) containing 0.5% acetic acid and 0.02% HFBA. The flow rate at the tip of <br />                                                                         the needle was set to 300 nl/min by programming the high-performance liquid <br /> TAP procedure (33) in which dilution of the yeast extract <br />                                                                         chromatography pump and using a split line. The mass spectrometer cycled <br /> during the purification procedure was minimized and high salt           through four scans as the gradient progressed. The first was a full mass scan <br /> concentrations were avoided. In most cases, purification using          followed by successive tandem mass scans of the three most intense ions. A <br /> this procedure unveiled new protein-protein interactions and            dynamic exclusion list was used to limit collection of tandem mass spectra for <br /> sometimes new polypeptides involved in transcriptional elon-            peptides that eluted over a long period of time. All tandem mass spectra were <br />                                                                         searched by using the SEQUEST computer algorithm against the complete yeast <br /> gation in S. cerevisiae. Furthermore, the <span id='am-93' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-94' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-97'  ></span>data</span> revealed that the        protein database (6/2,000). Each high-scoring peptide sequence was manually <br /> salt concentrations used during purification were critical in           compared with the corresponding tandem mass spectrum to ensure that the <br /> certain cases.                                                          match was correct. <br />  VOL. 22, 2002                                                                                     YEAST RNAPII ELONGATION FACTORS                       6981 <br />  <br />  <br />                                                                                         FIG. 1. Isolation of protein complexes containing Spt4, Spt5, and <br />                                                                                      Spt6. (A) TAPs of the Spt4/Spt5 complex were carried out on strains <br />                                                                                      containing either no tagged protein or TAP-tagged versions of Spt4 or <br />                                                                                      Spt5. Protein complexes were purified in the presence of 100 mM NaCl <br />                                                                                      and were then analyzed by SDS-PAGE and silver staining. Spt4, Spt5, <br />                                                                                      subunits of RNAPII, and a truncated form of Spt6 were identified by <br />                                                                                      trypsin digestion and MALDI-TOF mass spectrometry. In the purifi- <br />                                                                                      cation of tagged Spt5, the presence of Spt4 was identified by subjecting <br />                                                                                      an aliquot of the eluate from the final column directly to trypsin <br />                                                                                      digestion and tandem mass spectrometry. (B) TAP of Spt6. Purifica- <br />                                                                                      tion using an extract containing a TAP-tagged version of Spt6 in 100 <br />                                                                                      mM NaCl resulted in isolation of three forms of the tagged protein and <br />                                                                                      a previously uncharacterized protein encoded by ORF YPR133c, <br />                                                                                      whose gene product we have called Iws1 (interacts with Spt6). (C) TAP <br />                                                                                      of Iws1. Tagging and isolation of Iws1 in 100 mM salt resulted in <br />                                                                                      copurification of a stoichiometric amount of one form of Spt6. Because <br />                                                                                      both Spt6-TAP and Iws1-TAP were run on the same gel, it could be <br />                                                                                      concluded that Iws1 copurified with only the slowest-migrating form of <br />                                                                                      Spt6. <br />  <br />  <br />  <br />  <br />    ChIP. Chromatin immunoprecipitation (ChIP) and PCRs were performed as             that contained no tagged protein. The protein bands absent <br /> described previously (22). All yeast strains were grown at 30&#176;C to an OD595 of 0.4   from the control preparation and corresponding to the tagged <br /> to 0.6. Sequences for the ADH1 3&#11032; untranslated region primer pair are as follows: <br /> ADH11231, ACCGGCATGCCGAGCAAATGCCTG; ADH11400, CCCAACT <br />                                                                                      protein and any associated proteins were then identified by <br /> GAAGGCTAGGCTGTGG.                                                                    MALDI-TOF and ion trap mass spectrometry. Proteins that <br />                                                                                      copurified with the tagged proteins were then themselves <br />                                                                                      tagged, purified, and analyzed so as to confirm their presence <br />                                   RESULTS <br />                                                                                      in the complex and create a picture of the interaction relation- <br />    Purification and identification of the yeast elongation fac-                      ships among all the putative elongation factors of S. cerevisiae. <br /> tors. We used single-step transformation with PCR products                              Spt5 is a phosphoprotein that associates with Spt4 or Spt6. <br /> and a TRP1 selective marker to place TAP tags containing a                           Human DSIF and NELF (negative elongation factor) have <br /> calmodulin-binding peptide and S. aureus protein A, separated                        previously been identified and purified as inhibitory elongation <br /> by a TEV protease cleavage site, at the C termini of TFIIS, the                      factors that make transcription sensitive to DRB (46, 49). <br /> three subunits of TFIIF, all six known subunits of Elongator                         DSIF consists of 160- and 14-kDa subunits and can also stim- <br /> (23), Spt4, Spt5, Spt6, Spt16, and Pob3. To confirm the success                      ulate transcriptional elongation under certain conditions. The <br /> of each tagging procedure, Western blotting was performed on                         yeast homologues of the DSIF subunits, Spt4 and Spt5, also <br /> extracts derived from the tagged strains, making use of an                           form a complex, and mutations in the genes encoding these <br /> irrelevant immunoglobulin to recognize the protein A compo-                          proteins result in sensitivity to 6-azauracil (15), often consid- <br /> nent of the TAP tag. Each tagged protein, together with any                          ered to be an important criterion for mutations in yeast genes <br /> associated proteins, was then purified on IgG and calmodulin                         that encode elongation factors (2, 9). More recently, the Dro- <br /> columns and analyzed by SDS-PAGE followed by staining with                           sophila homologue of Spt5 was shown to colocalize with stalled, <br /> silver. Control purifications were always carried out in parallel                    hypophosphorylated RNAPII on Drosophila heat shock genes <br /> by using extracts derived from an otherwise congenic strain                          in the absence of induction and with hyperphosphorylated, <br />  6982    KROGAN ET AL.                                                                                            MOL. CELL. BIOL. <br />  <br />  <br /> actively elongating RNAPII during salivary gland develop-         during transcriptional elongation. Moreover, the yeast SPT16 <br /> ment, also implying that Spt5 is an elongation factor (1, 17).    gene interacts genetically with the genes encoding the yeast <br />    Purification of tagged Spt4 from yeast extracts in the pres-   elongation factors TFIIS and Spt4 (41). <br /> ence of 100 mM NaCl resulted in copurification of three forms        When the yeast counterparts of FACT, Spt16 and Pob3, <br /> of Spt5 as well as a substoichiometric amount of RNAPII (Fig.     were tagged and subsequently purified from yeast extracts in <br /> 1A). The three electrophoretically distinct forms of Spt5 were    buffers containing 150 mM NaCl, large and approximately <br /> consistently observed, but they are probably caused by C-ter-     stoichiometric amounts of Spt16 and Pob3 were isolated and <br /> minal degradation, because the tagging and purification of        no other proteins were detected by silver staining (Fig. 3A). <br /> Spt5 resulted in the isolation of only one form of tagged Spt5    However, when the NaCl concentration used throughout the <br /> (Fig. 1A). Copurifying with the tagged Spt5 were Spt4, a sub-     purification was reduced to 125 mM, many other proteins <br /> stoichiometric amount of RNAPII, and, surprisingly, a trun-       copurified with Spt16 and Pob3 in substoichiometric amounts <br /> cated form of Spt6. The small untagged Spt4 polypeptide ran       (Fig. 3B). When the NaCl concentration was further reduced <br /> off the gel shown in Fig. 1A, but the presence of Spt4 in the     to 100 mM (Fig. 3C and D), the yields of the proteins associ- <br /> sample applied to the gel was confirmed by tandem mass spec-      ated with Spt16 and Pob3 increased substantially. In each case, <br /> trometry. These results implied that Spt5 can associate either    none of the Spt16/Pob3-associated proteins were detected in <br /> with Spt4 or with a modified form of Spt6. Analysis of the        parallel purifications carried out with extracts derived from an <br /> complexes containing tagged Spt4 or Spt5 by use of tandem         untagged strain (Fig. 3A, B, and C). <br /> mass spectrometry served to identify a phosphorylation site in       Tandem and MALDI-TOF mass spectrometry were used to <br /> Spt5 on serine 188 in the sequence SDDE, which is the start of    identify the following proteins that copurified with Spt16 and <br /> a consensus CKII phosphorylation site [(S/T)-(D/E)-X-(D/E)]       Pob3: Chd1, Ctr9, Paf1, Cdc73, Rtf1, CkaI, CkaII, CkbI, CkbII, <br /> (see Fig. 2B) (12).                                               histones, and proteins encoded by two uncharacterized ORFs, <br />    Spt6 is a phosphoprotein that interacts with the uncharac-     YOL054w and YOR123c (also known as LEO1) (25). Interest- <br /> terized phosphoprotein Iws1. SPT6 was originally identified       ingly, Paf1 and Cdc73, as well as Ctr9, have previously been <br /> along with SPT4 and SPT5 in a screen for suppressors of <span id='am-83' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-84' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-85' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-87'  ></span>cis</span>-      identified as components of a complex that binds to RNAPII <br /> and trans-acting mutations that affected promoter function (15,   (21, 36). Chd1 is a member of the chromodomain-helicase- <br /> 41). More recently, Drosophila Spt6 was shown to colocalize       DNA-binding <span id='am-57' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-58' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span> and has been shown to be involved in <br /> with hyperphosphorylated, actively elongating RNAPII on           ATP-dependent nucleosome remodeling (44). Recently, an- <br /> polytene chromosomes (1, 17).                                     other component of the complex, Rtf1, was implicated genet- <br />    Tagging and purification of yeast Spt6 resulted in copurifi-   ically in transcriptional elongation (8). <br /> cation of substoichiometric amounts of a previously uncharac-        The plethora of proteins we identified in association with <br /> terized protein, encoded by the essential open reading frame      Spt16 and Pob3 raised the possibility that several distinct pro- <br /> (ORF) YPR133c, which we have called Iws1 (interacts with          tein complexes were associated with Spt16 and Pob3. To re- <br /> Spt6) (Fig. 1B). Conversely, tagging and purification of Iws1     solve these complexes, we tagged and purified from yeast ex- <br /> resulted in the isolation of Spt6, further confirming that this   tracts the various proteins that were associated with Spt16 and <br /> interaction occurs in vivo (Fig. 1C). There were three electro-   Pob3. Tagging of Chd1 resulted in copurification of approxi- <br /> phoretically distinct forms of tagged Spt6, but only the form     mately stoichiometric amounts of the four subunits of CKII <br /> with the lowest electrophoretic mobility copurified with tagged   (CkaI, CkaII, CkbI, and CkbII), as well as substoichiometric <br /> Iws1. It is not clear whether the higher-mobility forms of        amounts of Spt16 and Pob3 (Fig. 3E), but none of the other <br /> tagged Spt6 are a consequence of posttranslational modifica-      polypeptides that had copurified with Spt16 and Pob3. Tagging <br /> tion or in vitro or in vivo degradation.                          and purification of Ctr9 resulted in the coisolation of Rtf1, <br />    Tandem mass spectrometry was used to identify three tryptic    Leo1, Paf1, and Cdc73, as well as Spt16 and Pob3, but neither <br /> fragments in the N-terminal region of Spt6 that were <span id='am-98' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-99'  ></span><span id='am-100' property="rdfs:label" content="Phos" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>phos</span>-        the histones nor CKII and Chd1 (Fig. 3F). However, tagging <br /> phorylated on serine residues (serines 95 [SEDD], 135 [SESD],     and purification of histone H3 did result in the copurification <br /> and 207 [SDED]) (Fig. 2A). As was the case for Spt5, each         of Spt16 and Pob3 (data not shown). These results indicated <br /> phosphorylated serine residue in Spt6 was at the beginning of     that there are at least three separable complexes that associate <br /> a consensus CKII site. Many more putative CKII sites exist in     with Spt16 and Pob3: histones and nucleosomes, Chd1/CKII, <br /> the N-terminal region of Spt6, but tandem mass spectrometry       and the complex containing Ctr9, Paf1, Cdc73, Rtf1, and Leo1. <br /> did not detect phosphorylation at these sites. We did detect a    Confirmation came from experiments in which we labeled and <br /> phosphorylation site on serine 89 (SDDD) of Iws1, which is        purified Leo1 and Rtf1 (Fig. 3F), Paf1 (Fig. 3G), and Cdc73 <br /> also a putative CKII site. Interestingly, the acidic N-terminal   (Fig. 3H). In each case, we observed all of the untagged com- <br /> regions of Spt5 and Iws1 have sequence similarity (Fig. 2B),      ponents of this subcomplex, as well as amounts of Spt16 and <br /> and surprisingly, their phosphorylated serine residues align      Pob3 that varied from near-stoichiometric to clearly substoi- <br /> perfectly, perhaps explaining why Spt6 can interact with both     chiometric. In certain cases, there were two distinct forms of <br /> of them.                                                          Paf1, Cdc73, or Ctr9, although we have not yet determined <br />    Spt16/Pob3 associates with multiple protein complexes. Hu-     whether these are consequences of degradation, processing, or <br /> man FACT was purified as a factor that overcame a nucleo-         posttranslational modification. Although Yol054w had copuri- <br /> somal block to elongation by RNAPII in vitro and is composed      fied with tagged Spt16 and Pob3, we did not observe copurifi- <br /> of human homologues of yeast Spt16/Cdc68 and Pob3 (28, 29).       cation of Yol054w with tagged Paf1, Ctr9, Cdc73, Rtf1, or <br /> FACT interacts with nucleosomes and histone H2A-H2B               Leo1, suggesting that Yol054w may be directly associated with <br /> dimers and might, therefore, promote nucleosome disruption        Spt16 or Pob3. <br />  VOL. 22, 2002                                                                               YEAST RNAPII ELONGATION FACTORS                        6983 <br />  <br />  <br />  <br />  <br />    FIG. 2. Detection of phosphorylation on putative CKII sites in Spt5, Spt6, and Iws1. (A) Identification of phosphorylation sites in Spt6. Tandem <br /> mass spectrometry was used to identify three tryptic fragments in the N-terminal region of Spt6 that were phosphorylated on serine residues (se- <br /> rines 95 [SEDD], 135 [SESD], and 207 [SDED]). Each phosphorylated serine residue in Spt6 was at the beginning of a consensus CKII site. (B) <br /> Phosphorylation of a conserved serine present in both Spt5 and Iws1. Mass spectrometry successfully detected a phosphorylation site on serine 89 <br /> (SDDD) of Iws1 as well as on serine 188 (SDDE) of Spt5; each of these occurs at the start of a putative CKII site. The acidic N-terminal regions of Spt5 <br /> and Iws1 have sequence similarity, and their phosphorylated serine residues align perfectly. The regions compared have 23% identity and 40% similarity. <br />  <br />  <br />    To determine if the polypeptides copurifying with Spt16/                   reductions in UTP and GTP concentrations in yeast cells, leads <br /> Pob3 might be involved in transcriptional elongation in vivo,                 to induction of the PUR5 gene (35), and has been shown to <br /> strains harboring knockouts of genes encoding some of the                     affect the elongation efficiency of RNAPII (9). Therefore, <br /> nonessential subunits of the Spt16/Pob3 complex were tested                   sensitivity to this compound has been used as an <span id='am-28' about='obo:NCBITaxon_189528' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_189527 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_9219 obo:NCBITaxon_436491'><span id='am-29'  ></span><span id='am-30' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="Indicator" datatype="xsd:string"></span>indicator</span> <br /> for sensitivity to 6-azauracil. The drug 6-azauracil results in               for involvement in transcriptional elongation. Studies using <br />  6984   KROGAN ET AL.   MOL. CELL. BIOL. <br />  VOL. 22, 2002                                                                               YEAST RNAPII ELONGATION FACTORS                        6985 <br />  <br />  <br />  <br />  <br />    FIG. 3. Spt16/Pob3 associates with several protein complexes. (A) TAPs of Spt16 and Pob3 from TAP-tagged strains in buffers containing 150 <br /> mM NaCl. Purified preparations were analyzed by SDS-PAGE and silver staining. Spt16 and Pob3 were identified by mass spectrometry. <br /> (B) Purification of Spt16/Pob3 complexes in buffers containing 125 mM NaCl. Analysis by SDS-PAGE and silver staining revealed that a number <br /> of other polypeptides copurified with tagged Spt16 and Pob3. (C and D) Tagged Spt16 was purified in buffers containing 100 mM NaCl, resulting <br /> in an increased yield of proteins associated with Spt16/Pob3. The purified preparation was electrophoresed on gels containing 10% (C) or 15% <br /> (D) polyacrylamide. The additional proteins identified by mass spectrometry were Chd1, Ctr9, Paf1, Cdc73, Rtf1, Leo1, Yol054w, all four subunits <br /> of CKII, and histones. (E) TAP of tagged Chd1 in the presence of 100 mM salt. The purified protein was analyzed by SDS-PAGE, silver staining, <br /> and mass spectrometry. Tagged Chd1 copurified with seemingly stoichiometric amounts of the four subunits of CKII (CkaI, CkaII, CkbI, and <br /> CkbII) as well as substoichiometric amounts of Spt16 and Pob3. (F through H) TAPs of Leo1, Rtf1, and Ctr9 (F), Paf1 (G), and Cdc73 (H) (all <br /> in the presence of 100 mM NaCl) resulted in isolation of complexes containing Rtf1, Leo1, Ctr9, Paf1, and Cdc73, as well as Spt16 and Pob3, but <br /> no histones. (I) Sensitivities of deletion strains to 6-azauracil. Strains containing the rtf1&#9004;, paf1&#9004;, cdc73&#9004;, leo1&#9004;, chd1&#9004;, or dst1&#9004; deletion, as well <br /> as plasmid pRS316, were plated on SD-uracil medium with or without the drug 6-azauracil (50 &#9262;g/ml) and were grown at 30&#176;C for 2 to 4 days. WT, <br /> wild type. <br />  <br />  <br /> 6-azauracil have helped to show that genes like DST1, which                      In yeast, TFIIF is comprised of the three subunits Tfg1, <br /> encodes TFIIS, encode factors that have a role in transcrip-                  Tfg2, and Tfg3. When each subunit was tagged and purified <br /> tional elongation (2). When dst1&#9004; was used as a control, de-                  from yeast extracts in buffers containing 150 mM NaCl, a <br /> letions in rtf1, paf1, cdc73, and leo1, but not chd1, resulted in             complex containing all three subunits of TFIIF was isolated <br /> significant or marked sensitivity to 50 &#9262;g of 6-azauracil/ml (Fig.            along with a nearly stoichiometric amount of RNAPII (Fig. <br /> 3I).                                                                          4A). <br />    Purification of TFIIF and TFIIS. TFIIF is a general tran-                     At certain arrest sites encoded in the template DNA, RNA- <br /> scription factor that interacts with RNAPII and can recruit                   PII slides backwards, resulting in loss of contact between the 3&#11032; <br /> RNAPII to a preinitiation complex containing TFIID and                        end of the nascent transcript and the catalytic center of the <br /> TFIIB (14). TFIIF also participates in promoter clearance and                 polymerase (34, 45, 47). The transcriptional elongation factor, <br /> can associate with RNAPII in vitro during chain elongation,                   TFIIS, facilitates escape of arrested RNAPII from such a <br /> enhancing its polymerization rate and processivity (7, 32). It is             pause site by promoting cleavage of the nascent transcript by <br /> thought that TFIIF acts during the early stages of elongation                 the catalytic center of RNAPII, followed by reextension of the <br /> and prevents abortive transcription, perhaps by increasing the                RNA from the newly created 3&#11032; end. When TFIIS was tagged <br /> rate of nucleotide addition.                                                  and purified from yeast extracts, a predominantly free form of <br />  6986     KROGAN ET AL.                                                                                              MOL. CELL. BIOL. <br />  <br />  <br /> the protein was isolated (Fig. 4B), even though TFIIS is known      immunoglobulin G-agarose. Finally, reversal of the cross-links <br /> to interact directly with RNAPII (38).                              was carried out, followed by PCR analyses of the coprecipi- <br />    Isolation of a holo-Elongator complex. Because the three-        tated DNA. The gene investigated was ADH1, encoding alco- <br /> subunit Elongator complex was originally found associated           hol dehydrogenase, and three sets of primer pairs were used to <br /> with the elongating form of RNAPII and was shown to pref-           analyze the gene: one directed against the promoter region, <br /> erentially bind to the hyperphosphorylated form of RNAPII in        one directed further downstream within the coding region, and <br /> vitro (30), it was hypothesized that Elongator replaces the         finally, one used to amplify a region corresponding to DNA in <br /> mediator during the switch from transcription initiation to         the 3&#11032; untranslated region. <br /> elongation. By using extracts from individual strains containing       All three subunits of TFIIF, namely, Tfg1, Tfg2, and Tfg3, <br /> tagged versions of the three known Elongator subunits, Elp1,        cross-linked to the promoter region of ADH1 but not to the <br /> Elp2, and Elp3, a six-subunit yeast holo-Elongator complex          coding region or 3&#11032; untranslated region (Fig. 5A). These re- <br /> containing three additional polypeptides, Elp4, Elp5, and Elp6,     sults suggested that, whereas TFIIF is a stable component of <br /> was subsequently isolated (23). When purification was carried       the initiation complex and may play a role in early elongation, <br /> out at higher salt concentrations, Elongator could be separated     it does not remain stably associated with elongating RNAPII in <br /> into two subcomplexes, one consisting of Elp1, -2, and -3 and       vivo. In contrast, all components of the Paf1 complex were <br /> the other consisting of Elp4, -5, and -6 (23). Interestingly, no    found to cross-link specifically to all three regions of the ADH1 <br /> RNAPII detectable by silver staining was present in our Elon-       gene that were tested, including the novel, uncharacterized <br /> gator preparations, even when microgram quantities of Elon-         protein Leo1 (Fig. 5B); that is, all three fragments of the ADH1 <br /> gator were loaded onto the gel (23) (data not shown).               gene were enriched by immunoprecipitation relative to a DNA <br />    Copurification of the yeast elongation factors with RNAPII.      fragment from a nontranscribed region, no matter which sub- <br /> Human TFIIS, TFIIF, and Spt5 are known to bind directly to          unit of the Paf1/Ctr9/Cdc73/Rtf1/Leo1 complex was tagged. <br /> RNAPII (32, 38, 46). As well, Ctr9, Paf1, and Cdc73 were            Similarly, the ChIP patterns of Spt6 and Iws1 were almost <br /> identified as components of a yeast protein complex that binds      identical, and both displayed the same degree of cross-linking <br /> to RNAPII (21, 36), and Elongator copurified with chromatin-        to all three regions of the ADH1 gene (Fig. 5C). Interestingly, <br /> bound elongating RNAPII (30). Enough RNAPII copurified              ChIP assays demonstrated that Elongator subunits did not <br /> with tagged TFIIF, Spt4, and Spt5 for us to detect its two          cross-link to any part of the ADH1 gene or to a number of <br /> largest subunits, Rpb1 and Rpb2, by silver staining and identify    other genes that were tested (our unpublished data). <br /> them by mass spectrometry (Fig. 1A and 4A). We used the <br /> more sensitive Western blot technique with a monoclonal an-                                 DISCUSSION <br /> tibody directed against Rpb1 to determine which yeast elon- <br /> gation factors were associated more weakly with RNAPII (Fig.           At the outset of this study, a number of genes were known or <br /> 4C). Purification of RNAPII itself from a strain containing a       suspected to encode elongation factors for RNAPII in S. cere- <br /> TAP tag on Rpb3, the third-largest subunit of RNAPII,               visiae. TFIIS had been shown to stimulate elongation by yeast <br /> showed that the soluble yeast extracts used for our purifica-       RNAPII in vitro (34, 45, 47). TFIIF from human and Drosoph- <br /> tions indeed contained RNAPII recognized by monoclonal              ila cells had been shown to stimulate the rate of elongation by <br /> antibody 8WG16 (Fig. 4C). All of the tagged elongation factor       their respective RNAPII enzymes in vitro (4, 32), and we have <br /> subunits copurified with at least some RNAPII except Chd1           found that the same is true for purified yeast TFIIF (our <br /> and Elongator. Chd1 may not copurify with RNAPII simply             unpublished data). Nevertheless, we found that TFIIF is stably <br /> because its association with RNAPII is too indirect: it is weakly   associated only with the promoter region and not with the <br /> associated with Spt16/Pob3, which in turn is weakly associated      downstream transcribed region of the ADH1 gene in vivo, and <br /> with Paf1/Ctr9/Cdc73/Rtf1/Leo1, which in turn is weakly asso-       similar observations were reported recently by Pokholok et al. <br /> ciated with RNAPII (see Fig. 6). Remarkably, no detectable          (31). Therefore, it is not clear that TFIIF behaves as an elon- <br /> RNAPII copurified with Elongator when either Elp1 or Elp5           gation factor in vivo in spite of its in vitro activity. Spt4 and <br /> was tagged and purified, even when as much as 1 &#9262;g of purified      Spt5 are yeast homologues of the two subunits of DSIF, a <br /> Elongator was loaded onto the gel (Fig. 4C), and similar results    human factor with both positive and negative effects on chain <br /> were obtained with tagged Elp2 and Elp3 (data not shown). In        elongation by human RNAPII in vitro (46), and purified yeast <br /> other experiments (data not shown), we used antibodies spe-         Spt4/Spt5 also stimulates elongation by yeast RNAPII in vitro <br /> cific for the phosphorylated C-terminal domain (CTD) on             (our unpublished data). The genetic properties of the yeast <br /> Rpb1. Although purification of tagged Rpb3 revealed that our        SPT6 gene are similar to those of SPT4 and SPT5 (15, 41). <br /> extracts contained some phosphorylated Rpb1, phosphory-             Moreover, the Drosophila homologues of Spt5 and Spt6 appear <br /> lated Rpb1, like the mostly unphosphorylated Rpb1 recog-            to be associated with Drosophila elongation complexes on poly- <br /> nized by the antibody (8WG16) used in Fig. 4C, did not copu-        tene chromosomes (1, 17). Spt16 and Pob3 are yeast <span id='am-70' about='obo:NCBITaxon_9605' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="Homo" datatype="xsd:string"></span><span id='am-73'  ></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>homo</span>- <br /> rify with Elongator tagged on Elp1 or Elp5.                         logues of the two subunits of human FACT, a factor that <br />    ChIP assays. Analysis of the in vivo distribution of the novel   stimulates elongation by human RNAPII on a chromatin tem- <br /> elongation factors described here along a transcribed gene was      plate in a partially reconstituted system in vitro (28, 29). Ge- <br /> carried out by ChIP assays in which the proteins were cross-        netic studies on the yeast RTF1 gene have implicated Rtf1 as a <br /> linked to DNA in vivo by use of formaldehyde (22). Yeast cells      possible RNAPII elongation factor in vivo (8). Finally, bio- <br /> were then lysed, and following isolation and shearing of chro-      chemical copurification experiments indicated that Elongator <br /> matin, the presence of the proteins near specific DNA se-           is associated with elongating RNA polymerase II in vivo (30). <br /> quences was monitored by immunoprecipitation with rabbit               Collectively, these previously published studies implicated <br />  VOL. 22, 2002                                                                           YEAST RNAPII ELONGATION FACTORS                     6987 <br />  <br />  <br />  <br />  <br />    FIG. 4. TAPs of TFIIF and TFIIS and detection of RNAPII interactions by Western blot analysis. (A) Purification of TFIIF by using TAP- <br /> tagged versions of Tfg1, Tfg2, and Tfg3. Purified proteins were analyzed by SDS-PAGE, silver staining, and mass spectrometry. Purification of any <br /> of the three subunits of TFIIF in buffers containing 150 mM NaCl results in isolation of all three components of TFIIF as well as a substoichio- <br /> metric amount of RNAPII. (B) Tagging and purification of TFIIS. Purification of TFIIS (encoded by the DST1 gene) in buffers containing 100 <br /> mM NaCl resulted in the isolation of predominantly TFIIS. Further development of the silver stain revealed only background proteins and none <br /> that had copurified with TFIIS. (C) Western blot analysis of transcription elongation factors purified in the presence of 100 mM NaCl by using <br /> monoclonal antibody 8WG16, which binds to the CTD of the largest subunit of RNAPII, Rpb1. As a positive control, RNAPII was purified from <br /> a strain with a TAP tag on its third-largest subunit, Rpb3. Equal volumes of the column eluates from the various purifications were loaded onto <br /> the gel, except that approximately 10 times less of the Rpb3 purification was loaded. RNAPII, in the presence of 100 mM NaCl, copurified with <br /> TFIIF (Tfg1, -2, and -3), Spt4/Spt5, Spt6/Iws1, TFIIS, Spt16, and Paf1/Rtf1, but not with Elongator (Elp1 and Elp5) or Chd1. <br />  <br />  <br /> 13 polypeptides as having possible roles in chain elongation by            unknown function, Leo1 and possibly Yol054w. A relatively <br /> yeast RNAPII. However, only TFIIS, TFIIF, and Elongator                    small change in the NaCl concentration used for the purifica- <br /> had ever been purified; moreover, none of the yeast elongation             tion, from 150 to 125 or 100 mM, enabled us to identify all <br /> factors had been purified under conditions designed to mini-               these Spt16/Pob3-associated proteins rather than just Spt16 <br /> mize subunit loss during the purification procedure, raising the           and Pob3. Tagging each of the other proteins associated with <br /> possibility that there were intersubunit associations and new              Spt16/Pob3 revealed that the Spt16/Pob3 complex associates <br /> elongation factor subunits that had yet to be discovered. Based            with three distinct protein complexes: the core histones, Chd1/ <br /> on the experiments we have described here and other experi-                CKII, and Rtf1/Paf1/Ctr9/Cdc73/Leo1 (see Fig. 6). It is not <br /> ments (N. J. Krogan and J. F. Greenblatt, unpublished data),               clear whether these three protein complexes associate simul- <br /> we believe that complexes containing proteins that are weakly              taneously with Spt16/Pob3. Simultaneous association would <br /> associated in vivo have a greater chance of remaining associ-              imply that Spt16/Pob3 is a scaffold for the coassembly of all <br /> ated during purification when dilution is minimized and puri-              these factors. <br /> fication is carried out using near-physiological ionic strengths.             The interaction of Spt16/Pob3 with Chd1 and the core his- <br /> The web of interactions among the RNAPII elongation factors                tones is in accord with similar findings for human FACT (19, <br /> that we uncovered by these procedures is illustrated in Fig. 6.            28, 29). Kelley et al. (19) showed that the human homologues <br />    Most importantly, these procedures enabled us to discover               of Pob3 and Chd1 interact physically in vitro and in vivo. As <br /> that Spt16/Pob3 associates with 14 other polypeptides, namely,             well, a chd1&#9004; mutation suppressed the growth defect conferred <br /> Rtf1 and the core histones, as well as Chd1, the four subunits             by a particular pob3 point mutation (8) and may also result in <br /> of yeast CKII, a three-protein complex (Paf1/Ctr9/Cdc73)                   resistance to 6-azauracil in some strain backgrounds (although <br /> known to associate with RNAPII, and one or two proteins of                 not particularly in the one we have used here), implicating <br />  6988     KROGAN ET AL.                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br /> Chd1 in transcriptional elongation (44). FACT was also ob-               components of the Paf1 complex were tested for sensitivity to <br /> served to copurify with human CKII in a complex that phos-               6-azauracil. The strains containing the tagged proteins did not <br /> phorylated p53 (18), but it is not clear whether human CHD1              confer any significant sensitivity to the drug, implying the tag is <br /> was also part of this complex. The results we obtained by                not very detrimental to the function of the complex (data not <br /> tagging and purifying Chd1 imply that yeast Chd1 is directly             shown). It can similarly be inferred that C-terminal tags on <br /> associated with CKII (see Fig. 6). One possibility is that Chd1          Spt16, Pob3, Spt5, Spt6, Iws1, Tfg1, and Tfg2 are not very <br /> controls elongation by controlling the phosphorylation of an             detrimental, because these proteins are all encoded by essen- <br /> elongation factor by CKII. Definitive proof that Chd1 is an              tial genes, and tagged haploid strains are viable and grow well <br /> elongation factor awaits experiments showing that it can di-             in all cases. Although the uncharacterized polypeptide Yol054w <br /> rectly affect elongation by RNAPII in vivo or in vitro.                  was detected when tagged Spt16 was purified in buffers con- <br />     It was recently shown that Paf1 (RNAPII-associated factor),          taining 100 mM NaCl, it was not observed to copurify with any <br /> Cdc73, and Ctr9 form a complex that associates with RNAPII               of the other tagged proteins. Efforts to tag Yol054w were un- <br /> (21, 36). However, the roles of these proteins in transcription          successful, and so we cannot conclude that Yol054w is a bona <br /> by RNAPII were unclear. The observation that combining a                 fide subunit of any of these complexes. It is possible, however, <br /> PAF1 deletion with deletion of the mediator subunit SRB5 is              that Yol054w is directly associated with Spt16 or Pob3 (see Fig. <br /> lethal was used to argue that the Paf1 complex is involved in            6). <br /> initiation (36). However, we found that Paf1, Cdc73, and Ctr9               When all three subunits of TFIIF (Tfg1, Tfg2, and Tfg3) <br /> are associated with Spt16/Pob3 in a complex that also contains           were tagged and purified in buffers containing 150 mM NaCl, <br /> Rtf1 and Leo1, and so it is likely that this complex mediates the        the three proteins copurified with a near-stoichiometric amount <br /> interaction between Spt16/Pob3 and RNAPII. Rtf1 (restores                of RNAPII in each case. This showed that TFIIF interacts with <br /> TBP function) was originally shown to affect the function of             RNAPII in vivo, as suggested by experiments in many systems <br /> the TATA-binding protein (TBP) (40). However, recent work                (7, 38) which demonstrated that TFIIF interacts with RNAPII <br /> has suggested that it is likely to play a role in elongation by          in vitro. However, Tfg3 has also been reported to be associated <br /> RNAPII, since a strain with a deletion of RTF1 is sensitive to           with a number of other complexes, including the transcription <br /> 6-azauracil, a phenotype associated with defects in transcrip-           initiation factor TFIID and the chromatin-modifying com- <br /> tional elongation (8). Also, a deletion of RTF1 has been shown           plexes SWI/SNF and NuA3 (5, 13, 16). When an extract from <br /> to genetically interact with point mutations in the essential            a Tfg3-tagged strain was used to purify the smallest subunit of <br /> genes SPT16 and POB3. In view of the genetically defined role            TFIIF in buffers containing 75 to 100 mM NaCl, other poly- <br /> of Rtf1 in elongation and the association of Paf1/Ctr9/Cdc73             peptides were detected by silver staining of the SDS-polyacryl- <br /> with RNAPII, it is very likely that the Rtf1/Paf1/Ctr9/Cdc73/            amide gel, and these may correspond to subunits of these other <br /> Leo1 complex has an important role in the behavior of Spt16/             factors (Krogan and Greenblatt, unpublished). <br /> Pob3 as an elongation factor. Consistent with this, we found                The Elongator complex reported to copurify with RNAPII <br /> that paf1&#9004;, leo1&#9004;, and cdc73&#9004; strains are sensitive to 6-aza-            was originally shown to have three subunits, Elp1, Elp2, and <br /> uracil, implicating the products of these genes in transcrip-            Elp3, but is now known to have three additional subunits, Elp4 <br /> tional elongation in vivo. Further confirmation that Paf1/               (YPL101w), Elp5 (YHR187w), and Elp6 (YMR312w), which <br /> Cdc73/Ctr9/Rtf1/Leo1 is involved in elongation came from                 can be separated from Elp1/Elp2/Elp3 in buffers containing <br /> ChIP experiments showing that these proteins are associated              200 mM NaCl (23, 30). Genetic and DNA microarray analysis <br /> with the transcribed regions of ADH1 and other genes (our                of strains containing deletions of the Elongator subunits re- <br /> data and reference 31) and from recently published genetic               vealed that all are likely to be components of the same func- <br /> and coimmunoprecipitation experiments examining the roles                tional complex. Interestingly, Western blot analysis showed <br /> of these proteins (27, 39). However, in contrast to this work,           that no detectable RNAPII copurified with tagged Elp1, Elp2, <br /> neither study was able to detect a stable physical association           Elp3, or Elp5. Given our observation that purified Elongator, <br /> between the Paf1 complex and Spt16/Pob3. Nevertheless, de-               unlike purified TFIIF, also did not stimulate elongation by <br /> finitive proof that Paf1, Cdc73, Ctr9, Rtf1, and Leo1 are elon-          RNAPII in vitro (our unpublished data), this result implies <br /> gation factors awaits experiments showing directly that they             either that Elongator, unlike TFIIS, TFIIF, Spt4/Spt5, Spt6/ <br /> have elongation activity in vivo or in vitro.                            Iws1, and the Spt16/Pob3 complex, does not truly associate <br />     Tagging and purification of Rtf1, Paf1, Ctr9, or Cdc73 re-           with RNAPII or else that Elongator associates with RNAPII <br /> sulted in copurification of the same set of polypeptides that            only once RNAPII is engaged in transcription and then per- <br /> included these proteins as well as Leo1, Spt16, and Pob3,                haps only on some genes. In our hands, deletion of the genes <br /> confirming that all are truly associated with Spt16/Pob3. Since          encoding subunits of Elongator does not confer resistance or <br /> it is possible that the presence of the TAP tag on a protein may         sensitivity to 6-azauracil (23), suggesting that Elongator may <br /> impair its function in vivo, strains harboring individual tagged         not generally be an elongation factor. Yeast Elongator sub- <br />  <br />  <br />    FIG. 5. ChIP assays with TFIIF, Paf1/Leo1/Ctr9/Cdc73/Rtf1, and Spt6/Iws1. ChIP was performed with strains containing TAP-tagged versions <br /> of the TFIIF subunits Tfg1, Tfg2, and Tfg3 (A), Paf1, Cdc73, Rtf1, Leo1, and Ctr9 (B), or Spt6 and Iws1 (C). To monitor the presence of each <br /> protein along the ADH1 gene, chromatin was immunoprecipitated with rabbit IgG-agarose and PCR amplified with primer pairs recognizing <br /> promoter (P), coding (C), and 3&#11032; untranslated (U) regions (see schematic in panel A). Each PCR contained a second primer pair that amplified <br /> a region of chromosome V devoid of ORFs (marked by asterisks), thus providing an internal control for background. Input, signal from chromatin <br /> before immunoprecipitation. Primer pairs used are as follows: for P, ADH1-235 and ADH1-13; for C, ADH1844 and ADH11013; for U, ADH11231 <br /> and ADH11400; and for the nontranscribed region, Intergenic V-1 and Intergenic V-2. <br />  VOL. 22, 2002   YEAST RNAPII ELONGATION FACTORS   6989 <br />  6990     KROGAN ET AL.                                                                                                    MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />                     FIG. 6. Comprehensive interaction map for the RNAPII elongation factors. For details, see the text. <br />  <br />  <br />  <br /> units are mainly localized in the cytosol (24, 31), and ChIP          it copurified with Spt6, confirming that this is a bona fide in- <br /> experiments did not reveal association of Elongator subunits          teraction. Western blotting demonstrated that Spt6-Iws1, like <br /> with the transcribed region of the ADH1 gene (our unpub-              Spt4/Spt5, copurifies with RNAPII. Conversely, purification of <br /> lished data). However, human Elongator has been shown to              tagged RNAPII resulted in copurification of Spt4, Spt5, Iws1, <br /> weakly stimulate transcription by human RNAPII in vitro (20),         and Spt6, confirming these associations (data not shown). Our <br /> and one of its subunits has been shown to move from the               ChIP experiments showing that Iws1 is associated with the <br /> cytosol to the nucleus in response to interleukin-6 treatment         transcribed region of the ADH1 gene, together with our ob- <br /> (6). Therefore, Elongator may regulate the transcription of           servations that Iws1 is associated with Spt6 and RNAPII, sug- <br /> specific genes in response to signaling pathways. This would be       gest that Iws1, like Spt6, is an RNAPII elongation factor. Proof <br /> consistent with microarray experiments indicating that Elon-          that Spt6 and Iws1 are elongation factors awaits experiments <br /> gator affects the expression of only a small set of yeast genes       showing that these proteins influence elongation by RNAPII in <br /> (23).                                                                 vivo and in vitro. <br />    Tagging and purification of Spt4 resulted in the copurifica-          Interestingly, the sequence of the acidic N-terminal region <br /> tion of Spt5. When purification was carried out in buffers            of Iws1 displays similarity to the N-terminal region of Spt5. <br /> containing 100 mM NaCl, Spt4/Spt5 copurified with RNAPII              Moreover, tandem mass spectrometry revealed that a con- <br /> and was detectable by silver staining. Similarly, tagged Spt5         served serine residue in a consensus CKII phosphorylation site <br /> copurified with Spt4 and RNAPII. These findings are in accord         in this region was phosphorylated in both Spt5 and Iws1. Three <br /> with previous observations that a mutated form of RPB2 ge-            phosphorylated serines detected by mass spectrometry in the <br /> netically interacts with SPT5 and that Spt5 coimmunoprecipi-          N-terminal region of Spt6 are also located in consensus CKII <br /> tates with Rpb1 (15), the largest subunit of RNAPII. Similarly,       sites (12). Therefore, phosphorylation by CKII may be impor- <br /> DSIF, the human counterpart of Spt4/Spt5, physically interacts        tant for the association of Spt5 and Iws1 with Spt6 or may have <br /> with human RNAPII (46).                                               a regulatory role in the activities of Spt5, Spt6, and Iws1. One <br />    Tagged Spt5 also copurified with a truncated form of Spt6. A       interesting possibility is that the CKII associated with Chd1 in <br /> physical interaction between Spt5 and Spt6 was detected in one        the Spt16/Pob3 elongation factor complex may be involved in <br /> previously published study (41) but not in a follow-up study          the phosphorylation of Spt5, Spt6, and Iws1 (see Fig. 6). <br /> (15). Our tagging and purification of Spt6 did not result in the         Our targeted proteomics approach has led to the description <br /> copurification of Spt5, probably because the form of Spt6 that        of a web of interactions, comprising 25 polypeptides (31 if <br /> copurified with tagged Spt5 is missing its C-terminal region and      Elongator is included), among the putative transcriptional <br /> we have used C-terminal tags exclusively. However, we did             elongation factors of S. cerevisiae. Construction of this inter- <br /> discover a previously uncharacterized 40-kDa polypeptide,             action web was made possible by our use of affinity tagging and <br /> Iws1, associated with Spt6. When Iws1 was tagged and purified,        purification for each of the polypeptides, followed by electro- <br />  <br /> phoresis, silver staining, and mass spectrometry. Determining                    <br />                                                                                       <br /> which interactions are direct and which are indirect would not              <br /> have been possible if we had simply relied on the sensitive                 <br /> Western blot technique. In some cases, (for example, TFIIS,                           <br />                                                                     <br /> TFIIF, and Spt4/Spt5), our results confirmed known or <span id='am-75' about='obo:NCBITaxon_9822' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_117570 obo:NCBITaxon_32524 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_314145 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_9821 obo:NCBITaxon_35497 obo:NCBITaxon_7776 obo:NCBITaxon_9347'><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-78'  ></span><span id='am-79' property="rdfs:label" content="Sus" datatype="xsd:string"></span>sus</span>-                        <br /> pected interactions. In other cases, tagging and purification of    <br /> a known elongation factor resulted in the discovery of novel                            <br />                                                                                   <br /> information regarding that factor. In particular, Iws1, Paf1,                              <br /> Cdc73, Ctr9, and Leo1 were not previously suspected to be                            <br />                                                                                      <br /> RNAPII elongation factors, and Paf1, Cdc73, Ctr9, Leo1, and                          <br /> Rtf1 were not known to associate with Spt16/Pob3. It is in-                              <br /> triguing that there are probably so many elongation factors for                     <br />                                                                                           <br /> RNAPII. It is not known whether all the RNAPII elongation                             <br /> factors can associate simultaneously with the elongating tran-                           <br /> scription complex, nor is it known whether a given factor ac-                  <br />                                                                                            <br /> companies RNAPII all the way from its initiation site to its                              <br /> termination site. The nonessentiality in yeast of many of the                   <br /> elongation factor polypeptides may imply that some of them                                   <br />                                                                                         <br /> are functionally redundant. However, it is also possible that                       <br /> some of them specifically regulate genes whose expression is                                 <br />                                                                                              <br /> not essential under laboratory conditions. The yeast genetic                           <br /> system as well as ChIP assays may make it possible to resolve                         <br /> some of these issues.                                                                       <br /> </body></html>