<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>In vitro and in vivo function of the C-terminus of <br /> <span id='am-24' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-35'  ></span><span id='am-36' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span>Escherichia coli</span> single-stranded DNA binding protein <br />  <br />  <br />  <br />                                                           <br /> We constructed several deletion mutants of <span id='am-40' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-41'  ></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span>Escherichia</span>           Due to its role in DNA replication, repair and recombination, <br /> coli single-stranded DNA binding protein (EcoSSB)                single-stranded DNA binding protein of Escherichia coli <br /> lacking different parts of the C-terminal region. <span id='am-11' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13'  ></span><span id='am-14' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>This</span>           (EcoSSB) is essential for survival of the cell (1&#8211;3). EcoSSB <br /> region of EcoSSB is composed of two parts: a <span id='am-1' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-5'  ></span>glycine</span>-            consists of 177 amino acids and forms a very stable homotetramer <br /> and proline-rich sequence of &#8764;60 amino acids followed            (4). Secondary structure prediction indicates that the sequence of <br /> by an acidic region of the last 10 amino acids which is          EcoSSB can be divided into two parts, an N-terminal domain <br /> highly conserved among the bacterial SSB proteins.               (&#8764;120 amino acids) rich in &#945; helices and &#946; sheets and a more or <br /> The single-stranded DNA binding protein of human                 less unstructured C-terminus (5). The N-terminal two thirds <br /> mitochondria (HsmtSSB) lacks a region homologous                 contain the DNA-binding domain (6). The function of the <br /> to the C-terminal third of EcoSSB. Therefore, we also            C-terminal third is not yet characterized in detail. The importance <br /> investigated a chimeric protein consisting of the                of the C-terminus is illustrated by the ssb113 mutation (7), in <br /> complete sequence of the human mitochondrial                     which the penultimate residue of the EcoSSB protein (proline <br /> single-stranded DNA binding protein (HsmtSSB) and                176) is replaced by serine (8). This mutation results in a UV- and <br /> the C-terminal third of EcoSSB. Fluorescence titrations          temperature-sensitive phenotype. The DNA-binding affinity of <br /> and DNA-melting curves showed that the C-terminal                the mutated protein is only slightly affected (8). Williams et al. (6) <br /> third of EcoSSB is not essential for DNA-binding in              showed that the C-terminal third of EcoSSB becomes more <br /> vitro. The affinity for single-stranded DNA and RNA is           accessible to proteolysis after DNA-binding. Removing the <br /> even increased by the removal of the last 10 amino               C-terminal fragment resulted in a protein with higher affinity <br /> acids. Consequently, the nucleic acid binding affinity           towards single-stranded DNA. <br /> of HsmtSSB is reduced by the addition of the C-terminus             The C-terminal third of EcoSSB consists of two regions: a <br /> of EcoSSB. All mutant proteins lacking the last 10               sequence of &#8764;50 amino acids rich in proline and glycine residues <br /> amino acids are unable to substitute wild-type EcoSSB            is followed by an acidic region of 10 amino acids containing four <br /> in vivo. Thus, while the nucleic acid binding properties         aspartate residues. In contrast with the proline- and glycine-rich <br /> do not depend on an intact C-terminus, this region is            sequence, the region of the last 10 amino acids is highly conserved <br /> essential for in vivo function. Although the DNA                 among procaryotic SSB proteins (1). <br /> binding properties of HsmtSSB and EcoSSB are quite                  To characterize further the functions of the C-terminal third of <br /> similar, HsmtSSB does not function in E.coli. This               EcoSSB, we constructed several mutant EcoSSB proteins in which <br /> failure cannot be overcome by fusing the C-terminal              different parts of this region were deleted. <br /> third of EcoSSB to HsmtSSB. Thus differences in the                 Information about the influence of the N-terminal DNA <br /> N-terminal parts of both proteins must be responsible            binding domain on the functions of the C-terminal third can be <br /> for this incompatibility. None of the mutants was                obtained by replacing this domain with the human mitochondrial <br /> defective in tetramerization. However, mixed tetramers           single-stranded DNA binding protein (HsmtSSB). HsmtSSB, <br /> could only be formed by proteins containing the same             although sharing only 36% sequence homology with the N-terminus <br /> N-terminal part. This reflects structural differences            of EcoSSB, is a homotetrameric single-stranded DNA binding <br /> between the N-terminal parts of HsmtSSB and                      protein (SSB) with in vitro DNA binding properties almost <br /> EcoSSB. These results indicate that the region of the            indistinguishable from those of EcoSSB (9). The size of HsmtSSB <br /> last 10 amino acids, which is highly conserved among             (133 amino acids) corresponds to the size of the N-terminal domain <br /> bacterial SSB proteins, is involved in essential protein&#8211;        of EcoSSB and the protein is completely lacking any sequences <br /> protein interactions in the E.coli cell.                         homologous to the C-terminal third of EcoSSB. <br />  <br />  <br />  <br />  <br /> * To   whom correspondence should be addressed <br />                                                                                                             2707 <br />  <br />                                                                                Nucleic Acids <br />                                                                                Nucleic       Research,1994, <br />                                                                                        Acids Research, 1996,Vol.  24,No. <br />                                                                                                              Vol.22,  No.114       2707 <br />  <br /> MATERIALS AND METHODS                                                    Nucleic acid concentrations are given in units of monomers and <br />                                                                        determined using the following absorption coefficients: <br /> Poly(dT), poly(dA&#8211;dT) and poly(rU) were purchased from                 &#949;max = 8600 M&#8211;1cm&#8211;1 for poly(dT) (16), &#949;260 nm = 9200 M&#8211;1cm&#8211;1 <br /> Pharmacia (Freiburg). Experiments were performed in a standard         for poly(rU) (17) and &#949;260 nm = 6700 M&#8211;1cm&#8211;1 for poly(dA&#8211;dT). <br /> buffer containing 0.1 mM EDTA, 20 mM potassium phosphate <br /> pH 7.4 and sodium chloride as indicated. For fluorescence titrations <br />                                                                        Physicochemical experiments <br /> 100 p.p.m. Tween 20 (Serva) was added. <br />                                                                        Fluorescence titrations were carried out in a Schoeffel RRS 1000 <br /> Media                                                                  spectrofluorimeter as described earlier (18). Excitation wavelength <br />                                                                        was 295 nm and emission was observed at 350 nm. Theoretical <br /> LB medium (10) was used for all microbiological experiments.           binding isotherms were calculated using the model of Schwarz <br /> The concentration of antibiotics used for selection purposes was       and Watanabe (19) with the binding site size n, cooperative <br /> 100 &#181;g/ml ampicillin and 30 &#181;g/ml chloramphenicol respectively.        binding affinity (K &#969;), and the fluorescence quench (Qf ) as <br />                                                                        parameters. The fluorescence quench is the difference between the <br /> Site-directed mutagenesis                                              normalized fluorescences of the free protein and the protein&#8211;DNA <br />                                                                        complex. <br /> Site-directed mutagenesis was performed by the gapped-duplex             DNA melting curves were measured in a DMR10 (Zeiss) <br /> method using the pBR322 derived <span id='am-57' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-58' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span> pSF1 (11) and                 spectrophotometer as described previously (20) using a heating rate <br /> pSF1(HsmtSSB) (9) both conferring ampicillin resistance. The           of 20 K/h. No significant differences between heating and cooling <br /> truncated EcoSSB mutants EcoSSB Q152* and EcoSSB G117*                 curve could be observed confirming the reversibility of melting. <br /> were constructed by changing the codons for Q152 or G117 into            Analytical ultracentrifugation was carried out in a Spinco/ <br /> the stop <span id='am-16' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-20'  ></span>codon</span> TAG and TGA respectively. For construction of           Beckman model E centrifuge and evaluated as described earlier <br /> the deletion mutant EcoSSB &#8710;116&#8211;167 and HsEcSSB, we first              (21). <br /> introduced cleavage sites for blunt end restriction endonucleases <br /> at the respective positions in the genes. After cleavage the desired   Subunit exchange experiments <br /> gene fragments were purified using preparative agarose gel <br /> electrophoresis and ligated. EcoSSB &#8710;116&#8211;167 is a fusion protein       For subunit exchange experiments, two different SSB proteins <br /> where the amino acids 1&#8211;115 of EcoSSB are followed by the last         were mixed at a concentration of 50 &#181;M each in a buffer <br /> 10 amino acids (168&#8211;177) of EcoSSB. In HsEcSSB, the complete           containing 0.5 M NaCl, 1 mM EDTA, 20% glycerol and 20 mM <br /> sequence of HsmtSSB (1&#8211;133) is followed by amino acids                 potassium phosphate, pH 7.5 in a volume of 10 &#181;l. The mixture <br /> 113&#8211;177 of EcoSSB.                                                     was allowed to react for 3 days at room temperature for those <br />                                                                        exchange experiments containing only EcoSSB derivatives and <br /> Protein preparation                                                    for 2 weeks at 37 C for exchanges with HsmtSSB and HsEcSSB. <br />                                                                        The products were diluted by addition of 40 &#181;l 1 M triethanolamine <br /> The plasmids used for overproduction of SSB protein carry the          pH 8.5, 20 &#181;l dimethylsuberimidate (30 mg/ml in water) and 130 &#181;l <br /> respective gene under control of the &#955;PL-promotor. After               water. Crosslinking then was performed by a 3 h incubation at <br /> transformation into E.coli TGE900 (12) carrying the thermo-            room temperature and the products were analyzed by SDS&#8211;PAGE <br /> sensitive &#955;cI857 repressor protein, production was induced by a        (12%) (22). <br /> temperature shift from 30 to 42 C and cells were harvested 3 h <br /> after induction. The sequence of the ssb gene was confirmed from       Complementation of ssb defective strains <br /> plasmid prepared from an aliquot of the cells taken before <br /> induction.                                                             For complementation analysis two E.coli strains were used. In <br />    Wild-type EcoSSB and HsEcSSB were prepared using                    RDP268 (23) the chromosomal ssb gene is deleted and SSB <br /> poly(ethyleneimine) precipitation of the protein and subsequent        function is restored by pACYCssb plasmid conferring chloram- <br /> extraction with 0.4 M NaCl as described by Lohman et al. (13).         phenicol resistance. CS149 carries the ssb-3 mutation (G15D) <br /> The deletion mutant EcoSSB &#8710;116&#8211;167 was prepared like                  (24) on the chromosome rendering the cells unable to survive in <br /> wild-type EcoSSB but had to be extracted from the poly(ethylene-       presence of 0.9 &#181;g/ml mitomycin C. The strains were transformed <br /> imine) precipitation with 0.8 M NaCl. HsmtSSB was prepared as          with plasmids containing the respective ssb genes. For high copy <br /> described earlier (9). All EcoSSB mutants lacking the acidic           number plasmids the respective overproduction <span id='am-47' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-48' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="vectors" datatype="xsd:string"></span>vectors</span> (pSF1 <br /> region of the last 10 amino acids were purified like HsmtSSB.          derivatives v.s.) were used. In the defective strains the PL <br />                                                                        promotor is not repressed leading to high intracellular SSB <br /> Determination of concentrations                                        concentrations. Monocopy plasmids were constructed as follows: <br />                                                                        pSBL5 (25) is a derivative of the mini-F plasmid pRE432 (26) <br /> Protein concentrations are given in units of tetramers throughout      conferring ampicillin resistance and contains the wild-type <br /> the text. They were determined spectrophotometrically using the        EcoSSB gene under control of its natural promotor. In this <br /> following absorption coefficients at 280 nm: &#949; = 113 000 M&#8211;1cm&#8211;1       plasmid the nucleic acid sequence coding for 160 C-terminal <br /> for wild-type EcoSSB (14) and EcoSSB Q152*, &#949; =                        amino acids of wild-type EcoSSB was replaced by the respective <br /> 88 800 M&#8211;1cm&#8211;1 for EcoSSB G117* and EcoSSB &#8710;116&#8211;167,                   mutant gene fragments. Unfortunately the absence of suitable <br /> &#949; = 76 240 M&#8211;1cm&#8211;1 for HsmtSSB and &#949; = 99 000 M&#8211;1cm&#8211;1 for              restriction sites for exchanging the complete EcoSSB gene <br /> HsEcSSB. Unless stated otherwise, the absorption coefficients          precluded the construction of HsmtSSB and HsEcSSB derivatives <br /> were calculated from the known amino acid composition and the          of pSBL5. Since pSBL5 is a monocopy vector and the ssb gene <br /> absorption coefficients of the aromatic amino acids (15).              is under the control of the natural ssb promotor, intracellular SSB <br />  2708 Nucleic Acids Research, 1996, Vol. 24, No. 14 <br />  <br />                                                                                 due to the fact that the last 10 amino acids render SSB protein net <br />                                                                                 negatively charged whereas the DNA binding domains are net <br />                                                                                 positively charged. The proteins were 98% pure as judged from <br />                                                                                 Coomassie-stained denaturing PAGE, the main impurity being <br />                                                                                 wild-type EcoSSB expressed from the chromosomal ssb gene. In <br />                                                                                 preparation of HsmtSSB this impurity was removed in the <br />                                                                                 poly(ethyleneimine) precipitation step. For EcoSSB G117* and <br />                                                                                 EcoSSB Q152* the precipitation did not remove wild-type <br />                                                                                 EcoSSB indicating the formation of mixed tetramers (v.i.). <br />                                                                                 Sedimentation analysis in the analytical ultracentrifuge showed <br />                                                                                 all proteins being homotetramers. <br />  <br />                                                                                 Binding to single-stranded DNA <br />  <br />  <br /> Figure 1. Schematic depiction of the SSB proteins investigated in this study.   The binding of all SSB proteins to poly(dT) was observed by <br /> Hatched: N-terminal part (&#8764;120 <span id='am-51' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-53'  ></span><span id='am-54' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>aa</span>) of EcoSSB. Cross-hatched: HsmtSSB.           inverse fluorescence titrations at 0.3 M NaCl (Fig. 2). Upon <br /> White: proline- and glycine-rich region of EcoSSB (&#8764;50 aa). Black: acidic       binding to poly(dT) the intrinsic tryptophan-fluorescence is <br /> C-terminal region of EcoSSB (10 aa).                                            quenched by &#8764;90%; the exact amount depending on the <br />                                                                                 respective protein. The highest quench is shown by those proteins <br />                                                                                 that contain tryptophan only in the DNA binding domain. <br /> concentrations in this case correspond to the SSB level in a                    Tryptophan 135 of EcoSSB resides in the proline and glycine rich <br /> normal E.coli cell.                                                             region and its fluorescence is scarcely quenched upon DNA <br />   For ssb-3 complementation 100&#8211;200 transformed cells were                      binding (18). Thus wild-type EcoSSB, EcoSSB Q152* and <br /> grown on plates containing ampicillin. Colonies were replica plated             HsEcSSB show the least quench. The affinity of the proteins to <br /> on LB containing 0.9 &#181;g/ml mitomycin C and incubated at 37 C                    poly(dT) is too large to allow a discrimination of the binding <br /> overnight.                                                                      properties of the different proteins from the shape of the <br />   For &#8710;ssb complementation transformed cells were grown in the                  fluorescence titrations (18). The binding site sizes are 65 nucleotides <br /> presence of ampicillin but omitting chloramphenicol. After four <br />                                                                                 per tetramer for wild-type EcoSSB, EcoSSB Q152* and HsEcSSB, <br /> subsequent inoculations of 4 ml medium allowing the cells to <br />                                                                                 and 59 nucleotides for the others. The C-terminal third of EcoSSB <br /> grow for &#8764;50 generations, 100&#8211;200 cells were plated. Clones <br />                                                                                 is not a part of the DNA binding domain (v.i.). Thus it is the sheer <br /> which lost the helper plasmid pACYCssb were identified by <br />                                                                                 size of this region that leads to the requirement of larger stretches <br /> replica plating on LB plates containing both chloramphenicol and <br />                                                                                 of poly(dT) to reach all four DNA binding sites of the tetramer. <br /> ampicillin. <br />   If the plasmid tested could complement &#8710;ssb &#8764;25% of the                          A more indirect, but more sensitive, method to compare the <br /> colonies had lost the helper plasmid indicated by chloramphenicol               affinities of different SSB proteins towards single-stranded DNA <br /> sensitivity. In this case the absence of wild-type EcoSSB was                   is the measurement of the ability of the proteins to depress the <br /> confirmed by western blot analysis.                                             melting temperature of double-stranded DNA. Differences in the <br />                                                                                 affinities not detectable at room temperature are amplified in the <br />                                                                                 melting experiment by the strong decrease of affinity with <br /> RESULTS                                                                         increasing temperature (27). Figure 3 shows the destabilization of <br /> A schematic depiction of the different deletion mutants of                      poly(dA&#8211;dT) by the different SSB proteins. <br /> EcoSSB, wild-type HsmtSSB and the chimeric HsEcSSB                                 The C-terminally truncated proteins EcoSSB Q152* and <br /> constructed for this work is shown in Figure 1.                                 EcoSSB G117* cause a drastic reduction of the melting temperature <br />   The C-terminally truncated proteins EcoSSB Q152* and                          of poly(dA&#8211;dT). This is probably due to the removal of electrostatic <br /> EcoSSB G117* lack the highly conserved region of the last 10                    repulsions between the acidic C-terminus and the single-stranded <br /> amino acids and the complete C-terminal third respectively. In the              DNA. The same effect can be observed with HsmtSSB where the <br /> deletion mutant EcoSSB &#8710;116&#8211;167 the amino acids 116&#8211;167 are                     addition of the C-terminal third of EcoSSB strongly diminished <br /> missing, leading to a protein which lacks the proline- and                      the <span id='am-6' about='obo:NCBITaxon_6534' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_87862 obo:NCBITaxon_131567 obo:NCBITaxon_6533 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-7'  ></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Helix" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>helix</span> destabilization activity (HsEcSSB). <br /> glycine-rich region but does contain the last 10 amino acids.                      In the deletion mutant EcoSSB &#8710;116&#8211;167 the negatively <br /> HsmtSSB is homologous to the N-terminal domain of EcoSSB                        charged C-terminus is directly fused to the DNA binding domain. <br /> and thus corresponds to EcoSSB G117*. The chimeric protein                      Although at room temperature this mutant binds strongly to <br /> HsEcSSB is a fusion protein composed of the complete HsmtSSB                    poly(dT) its helix destabilizing activity in the melting experiment <br /> and the C-terminal third of EcoSSB. This protein corresponds to                 is the lowest of all proteins in this study. This again reflects the <br /> EcoSSB in containing an N-terminal DNA binding domain, a                        electrostatic repulsion between the acidic C-terminus and the <br /> proline- and glycine-rich region and an acidic C-terminus of 10                 single-stranded DNA which is most strongly pronounced in this <br /> amino acids.                                                                    mutant. <br />   All proteins were overexpressed in E.coli. Those proteins that                   The fact that HsmtSSB does not destabilize poly(dA&#8211;dT) as <br /> contained the last 10 amino acids of EcoSSB (wild-type EcoSSB,                  strongly as EcoSSB Q152* and EcoSSB G117* do is a reflection <br /> EcoSSB &#8710;116&#8211;167, HsEcSSB) could be precipitated by                              of the different structures of the DNA binding domains of these <br /> poly(ethyleneimine) while the others could not. This is probably                proteins. <br />                                                                                                                                  2709 <br />  <br />                                                                                               Nucleic Acids <br />                                                                                               Nucleic       Research,1994, <br />                                                                                                       Acids Research, 1996,Vol.  24,No. <br />                                                                                                                             Vol.22,  No.114                   2709 <br />  <br />  <br />  <br />  <br />                                                                                        Figure 3. Destabilization of poly(dA&#8211;dT) by SSB proteins. Poly(dA&#8211;dT) <br /> Figure 2. Fluorescence titrations with poly(dT). Protein (0.3 &#181;M) was titrated         (38 &#181;M) was melted in standard buffer containing 0.1 M NaCl in presence of <br /> with poly(dT) in standard buffer containing 0.3 M NaCl. The solid lines                1.27 &#181;M protein. (A) Tm values: EcoSSB Q152* (a): 35_C; EcoSSB (b): 47_C; <br /> represent theoretical binding isotherms according to the model of Schwarz and          HsEcSSB (c): 56_C; without protein (d): 61_C. (B) Tm values: without protein <br /> Watanabe (19) using the following parameters: (A) EcoSSB (F): n = 65, K &#969;              (d): 61_C; EcoSSB G117* (e): 38_C; HsmtSSB (f): 48_C; EcoSSB &#8710;116&#8211;167 <br /> = 2.4 &#215; 108 M&#8211;1, Qf = 0.88; EcoSSB Q152* (J): n = 65, K &#969; = 1.6 &#215; 108 M&#8211;1,             (g): 59_C. <br /> Qf = 0.91; HsEcSSB (&#9830;): n = 65, K &#969; = 1.7 &#215; 108 M&#8211;1, Qf = 0.85 (B) EcoSSB <br /> &#8710;116&#8211;167 (F): n = 59, K &#969; = 1.9 &#215; 108 M&#8211;1, Qf = 0.95; HsmtSSB (J): n = 59, <br /> K &#969; = 1.7 &#215; 108 M&#8211;1, Qf = 0.96; EcoSSB G117* (&#9830;): n = 59, K &#969; = 1.1 &#215; <br /> 108 M&#8211;1, Qf = 0.99. As discussed previously (18), the relatively strong binding       In vivo properties <br /> does not allow the interpretation of the figures for K &#969; as exact numbers but <br /> rather as lower limits. In contrast, the values for n and Qf are accurate within &quot;2   Physiological abilities of the mutant proteins can be investigated <br /> and &quot;0.02 respectively. <br />                                                                                       by complementation of E.coli strains with defective or lacking ssb <br />                                                                                       gene. However, a strain lacking the ssb gene is not viable. Therefore, <br /> Binding to single-stranded RNA                                                        we used a strain where the chromosomal ssb gene is deleted and <br />                                                                                       substituted by an ssb gene coded on a plasmid conferring <br /> The affinity of EcoSSB to single-stranded RNA is much weaker                          chloramphenicol resistance (23). If this plasmid could be replaced <br /> than to single-stranded DNA (3). Thus differences in affinity not                     by a plasmid carrying a mutant ssb gene and a different resistance <br /> detectable in fluorescence titrations with poly(dT) are observed                      we regarded a respective mutant to be able to complement &#8710;ssb. <br /> with poly(rU). Figure 4 shows fluorescence titrations of the                             A severe defect of the ssb gene is ssb-3 (24) where glycine 15 <br /> different proteins with poly(rU).                                                     is replaced by aspartate leading to a high UV and mitomycin C <br />    The proteins missing the acidic C-terminus (EcoSSB Q152*,                          sensitivity. Cells carrying this defect were not viable in the <br /> EcoSSB G117*) show a higher affinity to poly(rU) as compared                          presence of 0.9 &#181;g/ml mitomycin C. We regard complementation <br /> with wild-type EcoSSB. The affinity of the deletion mutant                            as positive if a plasmid coding for a mutant SSB confers <br /> EcoSSB &#8710;116&#8211;167 is drastically reduced. Once more this reflects                       resistance to this concentration of mitomycin C. <br /> the influence of the electrostatic repulsion between the nucleic                         Complementation was tested with high cellular concentrations <br /> acid and the negatively charged C-terminus.                                           of mutant SSB using the expression plasmids without repression <br />    The same holds for the comparison of HsmtSSB and HsEcSSB                           of the PL promotor. For lower amounts of mutant SSB we used <br /> although the affinities of these proteins to poly(rU) are very weak                   a monocopy plasmid where the ssb gene is under control of the <br /> and only marginal binding can be observed in fluorescence titrations.                 chromosomal E.coli ssb promotor. <br />  2710 Nucleic Acids Research, 1996, Vol. 24, No. 14 <br />  <br />  <br />  <br />  <br />                                                                                  Figure 5. Subunit exchange between wild-type and mutant EcoSSB proteins <br />                                                                                  detected by chemical crosslinking. Lane 1: EcoSSB Q152* and wild-type <br />                                                                                  EcoSSB, lane 2: EcoSSB Q152*, lane 3: EcoSSB G117*, lane 4: EcoSSB <br />                                                                                  G117* and wild-type EcoSSB, lane 5: wild-type EcoSSB, lane 6: molecular <br />                                                                                  mass standard, lane 7: EcoSSB &#8710;116&#8211;167, lane 8: EcoSSB &#8710;116&#8211;167 and <br />                                                                                  wild-type EcoSSB. Bands corresponding to heterodimers are indicated by arrows. <br />  <br />  <br />                                                                                  one could envisage EcoSSB G15D as complementing the <br />                                                                                  otherwise lethal mutations EcoSSB Q152* and EcoSSB G117* . In <br />                                                                                  this mechanism, high concentrations of EcoSSB Q152* and <br />                                                                                  EcoSSB G117* dilute intact C-terminal regions to such an extent <br />                                                                                  that correct EcoSSB function is suppressed (cf. Table 1). <br />  <br />                                                                                  Table 1. Complementation of ssb defective strains <br />  <br />                                                                                   Protein              Complementation <br />                                                                                                        ssb-3                         &#8710;ssb <br />                                                                                                        monocopy      high copy       monocopy     high copy <br />                                                                                   EcoSSB-wt            +             +               +            + <br />                                                                                   EcoSSB Q152*         +             &#8211;               &#8211;            &#8211; <br /> Figure 4. Fluorescence titrations with poly(rU). Protein (0.3 &#181;M) was titrated    EcoSSB G117*         +             &#8211;               &#8211;            &#8211; <br /> with poly(rU) in standard buffer containing 0.2 M NaCl. Solid lines represent <br /> theoretical binding isotherms calculated according to the model of Schwarz and    EcoSSB &#8710;116&#8211;167      +             +               +            + <br /> Watanabe (19) using the following parameters: (A) EcoSSB (F): n = 65, <br /> K &#969; = 7.0 &#215; 10 6 M&#8211;1 , Qf = 0.64; HsmtSSB (J): n.d.; HsEcSSB (&#9830;): n.d.            HsmtSSB              n.d.          &#8211;               n.d.         &#8211; <br /> (B) EcoSSB Q152* (J): n = 65, K &#969; = 2.1 &#215; 107 M&#8211;1, Qf = 0.67; EcoSSB              HsEcSSB              n.d.          &#8211;               n.d.         &#8211; <br /> &#8710;116&#8211;167 (F): n.d.; EcoSSB G117* (&#9830;): n = 59, K &#969; = 2.6 &#215; 107 M&#8211;1, Qf = <br /> 0.71. Note that the titration points for HsmtSSB, HsEcSSB and EcoSSB <br /> &#8710;116&#8211;167 are located on nearly straight lines and there is almost no bending <br /> toward saturation in the accessible range of poly(rU) concentration. This <br /> indicates that HsmtSSB and EcoSSB &#8710;116&#8211;167 bind very weakly but with             Subunit exchange <br /> reasonable fluorescence quench to poly(rU). <br />                                                                                  Formation of mixed tetramers can be demonstrated in vitro by <br />                                                                                  chemical crosslinking of the subunits with dimethylsuberimidate <br />                                                                                  and analyzing the products on a denaturing PAGE. Figure 5 (lanes <br />    Table 1 shows the results of these complementation experiments.               2, 3, 5 and 7) and Figure 6 (lanes 1, 3 and 6) show that the <br /> Wild-type EcoSSB was used as control and could complement in                     chemical crosslinking of homotetramers leads to the appearence <br /> every case. The only mutant showing the same complementation                     of covalently connected dimers, trimers and tetramers in addition <br /> ability as wild-type EcoSSB was EcoSSB &#8710;116&#8211;167. &#8710;ssb could                      to the monomeric band. This supports the result from analytical <br /> not be complemented by any of the other proteins. We conclude                    ultracentrifugation that all proteins of this study form stable <br /> that the proline- and glycine-rich region of the C-terminal third is             homotetramers (v.s.). Thus the C-terminal third of EcoSSB is not <br /> not essential in vivo and that the DNA binding domain of                         required for tetramer formation. <br /> HsmtSSB cannot replace the corresponding binding domain of                          Formation of heterotetramers in this experiment can best be <br /> EcoSSB.                                                                          demonstrated by the appearence of an additional covalently <br />    Small amounts of EcoSSB Q152* and EcoSSB G117*,                               linked heterodimer. If two different SSB proteins form tetramers <br /> however, could complement the N-terminal mutation G15D. A                        in all possible combinations then chemical crosslinking leads to <br /> likely mechanism of this complementation is the formation of                     four trimeric and five tetrameric <span id='am-21' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>. These products cannot <br /> mixed tetramers containing intact N-terminal domains from the                    be resolved on the gel. While all C-terminally modified <br /> mutants EcoSSB Q152* or EcoSSB G117* respectively as well                        EcoSSB-proteins are able to exchange subunits with wild-type <br /> as the correct C-terminal region from the EcoSSB G15D. Thus                      EcoSSB (Fig. 5, lanes 1, 4 and 8) the human mitochondrial SSB <br />                                                                                                                              2711 <br />  <br />                                                                                           Nucleic Acids <br />                                                                                           Nucleic       Research,1994, <br />                                                                                                   Acids Research, 1996,Vol.  24,No. <br />                                                                                                                         Vol.22,  No.114                   2711 <br />  <br />                                                                                  of mutations localized in different regions of EcoSSB can be <br />                                                                                  overcome by the formation of mixed tetramers. <br />                                                                                    Thus a number of defined functional properties of these SSB <br />                                                                                  proteins can be assigned to different structural regions. <br />  <br /> Figure 6. Subunit exchange between EcoSSB, HsmtSSB and HsEcSSB <br /> detected by chemical crosslinking. Lane 1: wild-type EcoSSB, lane 2:             <br /> HsmtSSB and wild-type EcoSSB, lane 3: HsmtSSB, lane 4: HsEcSSB and               <br /> wild-type EcoSSB, lane 5: molecular mass standard, lane 6: HsEcSSB, lane 7:       <br /> HsEcSSB and HsmtSSB. The band corresponding to a heterodimer is indicated           <br /> by an arrow. Lack of heterodimer formation in lanes 2 and 4 is indicated by an     <br /> asterisk.                                                                          <br /> protein cannot form mixed tetramers with EcoSSB (Fig. 6, lanes 2                  <br /> and 4). However, fusion of the EcoSSB C-terminal region to                         <br /> HsmtSSB does not impair heterotetramer formation between                          <br /> HsmtSSB and HsEcSSB (Fig. 6, lane 7). All these results show                       <br /> that subunit interaction is restricted to the N-terminal DNA                       <br /> binding domain. However, the N-terminal domains of procaryotic                       <br /> and eucaryotic mitochondrial SSB proteins are too different to                      <br /> allow subunit interaction.                                                          <br />                                                                                     <br /> The C-terminal third of EcoSSB is neither essential for the                        <br /> binding of nucleic acids nor for tetramer formation. The negative                <br /> charges of the last 10 amino acids weaken the binding of the                     <br /> protein to nucleic acids. The electrostatic repulsion depends on                <br /> the distance of these charges from the N-terminal binding domain               <br /> in the amino acid sequence.                                                     <br />                                                                             <br />   However, the last 10 amino acids of the C-terminal third are                    <br /> essential for in vivo function. Mutant SSB proteins missing this               <br /> acidic region are not functional in the E.coli cell. The vital role of    <br /> these last 10 amino acids must be some function different from                    <br /> DNA binding, probably in protein&#8211;protein interactions. The                      <br />                                                                                    <br /> sequence between the DNA binding domain and the last 10 amino                   <br /> acids serves only as a spacer keeping the negative charges away                 <br /> from the bound DNA.                                                                <br />   DNA binding and tetramer formation are both localized in the                  <br /> N-terminal domain. Structural differences in this domain between                 <br /> eucaryotic mitochondrial and procaryotic SSB proteins prohibit          <br /> interactions of the heterologous subunits. The phenotypical effect       <br /> </body></html>