<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR AND CELLULAR BIOLOGY, Mar. 2000, p. 2066&#8211;2074                                                                                    Vol. 20, No. 6 <br /> 0270-7306/00/$04.00&#11001;0 <br /> Copyright &#169; 2000, American Society Microbiology. All Rights Reserved. <br />  <br />  <br />  <br />               RRS1, Conserved Essential Gene, Encodes Novel <br />              Regulatory Protein Required Ribosome Biogenesis in <br />                              <span id='am-1' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-15'  ></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> <br />            AKIKO TSUNO, KEITA MIYOSHI, ROTA TSUJII, TOKICHI MIYAKAWA,                                         AND   KEIKO MIZUTA* <br />          Department Molecular Biotechnology, Graduate School Advanced Sciences Matter, Hiroshima University, <br />                                              Higashi-Hiroshima 739-8527, Japan <br />                    Received 18 October 1999/Returned modification 30 November 1999/Accepted 28 December 1999 <br />  <br />             A secretory defect causes specific significant transcriptional repression ribosomal protein and <br />          rRNA genes (K. Mizuta J. R. Warner, Mol. Cell. Biol. 14:2493&#8211;2502, 1994), suggesting coupling of <br />          plasma membrane ribosome syntheses. In <span id='am-69' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> elucidate molecular mechanism signaling <br />          pathway, isolated cold-sensitive mutant mutation gene termed RRS1 (regulator ribosome <br />          synthesis), appeared defective signaling pathway. The rrs1-1 mutation greatly reduced <br />          transcriptional repression rRNA ribosomal protein genes caused secretory defect. RRS1 <br />          novel, essential gene encoding nuclear protein 203 amino acid residues conserved eukaryotes. <br />          A conditional rrs1-null mutant constructed placing RRS1 control GAL1 promoter. <br />          Rrs1p depletion caused defects processing pre-rRNA assembly ribosomal subunits. <br />  <br />  <br />    Balanced synthesis cellular components required                difference <span id='am-61' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-65'  ></span>codon</span> 114, resulting stop codon. The <br /> normal cell growth. A temperature-sensitive mutation SLY1,               amino acid sequence Rrs1p significantly similar of <br /> whose gene product involved endoplasmic reticulum-to-                 putative protein encoded human cDNA. In order an- <br /> Golgi trafficking (26), causes transcriptional repression            alyze functions Rrs1p, constructed conditional rrs1-null <br /> both ribosomal protein rRNA genes <span id='am-18' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-19' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-24'  ></span>Saccharomyces</span>                      mutant. Depletion Rrs1p affects pre-rRNA processing and <br /> cerevisiae (20). Further examination using various sec mutants              assembly ribosomal subunits, indicating Rrs1p re- <br /> showed defect secretory pathway,                quired proper ribosome biogenesis. <br /> a step prior insertion nascent peptide endo- <br /> plasmic reticulum step involved formation                                       MATERIALS AND METHODS <br /> plasma membrane, prevents continued synthesis                       Yeast strains, <span id='am-92' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-93' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>plasmids</span>, media. The yeast strains used study are <br /> components ribosome. Similar results obtained                   listed Table 1. Yeast cells grown YPD rich medium, YPG, synthetic <br /> following treatment wild-type cells secretory inhib-            complete medium containing 2% glucose (SC) 2% galactose (SCGal), SC <br />                                                                             dropout medium, depending plasmid markers (10). Yeast transformation <br /> itors tunicamycin brefeldin A (20). Furthermore,                   performed lithium acetate procedure (9). <br /> temperature-sensitive mutants transcription ribo-                  A library consisting partial Sau3A fragments S. cerevisiae genomic DNA <br /> somal protein genes temperature sensitive appear                   inserted single-copy yeast vector, YCp50, generously provided by <br /> defective secretory pathway (17). As membrane                 M. D. Rose (29). <br />                                                                                For epitope tagging, NheI site introduced PCR pRRS1 just after <br /> the end product of secretory pathway, results             initiation codon. The DNA cassette encoding copies nine-amino- <br /> suggest important coupling plasma membrane ribo-                  acid influenza virus hemagglutinin (HA) epitope obtained digestion of <br /> some biosynthesis. We proposed existence signal trans-             pYT11 (33) NheI inserted frame NheI site pRRS1. <br /> duction pathway plasma membrane nucleus.                    The digested DNA fragment including epitope-tagged RRS1 subcloned into <br />                                                                             pRS316 (32). <br /> According model, signal generated defect                A plasmid containing GAL1 promoter-controlled RRS1 constructed as <br /> novo synthesis membrane transmitted nu-                 follows. pAT-35 (see below) digested XbaI ScaI 0.8-kb <br /> cleus cause specific significant transcriptional repres-            fragment containing open reading frame (ORF) RRS1 placed under <br /> sion ribosomal genes. It recently suggested stress           GAL1-promoter multi-copy vector pNV7 (25). The DNA fragment <br />                                                                             containing GAL1 promoter, RRS1, terminator cloned a <br /> the plasma membrane monitored Pkc1, initiates                 single-copy vector, pRS313 (32). <br /> signal transduction pathway leads repression (24).                 Cloning mutant allele. A mutant allele chromosomal RRS1 gene <br /> In order elucidate molecular mechanism signal                 isolated PCR. Total chromosomal DNAs isolated wild-type <br /> transduction pathway, screened mutants defective                rrs1-1 mutant cells. DNA fragments including RRS1 rrs1-1 amplified <br />                                                                             PCR cloned pUC19. PCR carried twice, DNA <br /> in response secretory defect.                                      sequence determined using independent clones. <br />    Here isolation molecular characteriza-                  Indirect immunofluorescence. Indirect immunofluorescence microscopy was <br /> tion RRS1, encoding essential nuclear protein 203                  modification procedure described Pringle et al. (27). Cultures <br /> amino acids. In rrs1-1 mutant, secretory defect fails              early-log-phase cells growing SC medium fixed addition form- <br />                                                                             aldehyde medium final concentration 3.2%, followed agitation <br /> cause transcriptional repression rRNA ribosomal                30 min room temperature. Fixed cells harvested centrifugation, <br /> protein genes. The mutant gene, rrs1-1, single nucleotide             washed KPS buffer (100 mM potassium phosphate buffer, pH 7.5, 1.2 <br />                                                                             M sorbitol) resuspended KPS. To remove cell wall, cell suspension <br />                                                                             incubated 20 &#65533;?&#65533;g Zymolyase 100T ml 0.2% &#65533;?&#65533;-mercaptoetha- <br />                                                                             nol 37&#176;C 10 min. Spheroplasts washed KPS harvested <br />   * Corresponding author. Mailing address: Department Molecular <br />                                                                             gentle centrifugation. Resuspended spheroplasts KPS spotted onto <br /> Biotechnology, Graduate School Advanced Sciences Matter, Hi-          polylysine-coated multiwell slides blocked 5 min PBS-BSA (0.285% <br /> roshima University, Kagamiyama 1-4-1, Higashi-Hiroshima 739-8527,           Na2HPO4, 0.02% KH2PO4, 0.8% NaCl, 0.02% KCl [pH 7.5], 1 mg bovine <br /> Japan. Phone: 81 824 24 7765. Fax: 81 824 22 7196. E-mail: kmizuta          serum albumin ml). The cells incubated 10 &#65533;?&#65533;l primary antibody <br /> @ipc.hiroshima-u.ac.jp.                                                     (anti-HA.11; Babco) diluted 1:1,000 PBS-BSA 1 h room temperature. <br />  <br />                                                                      2066 <br />  VOL. 20, 2000                                                               NOVEL REGULATORY PROTEIN OF RIBOSOME SYNTHESIS                                         2067 <br />  <br />  <br />                                                              TABLE 1. Yeast strains used study <br />  Strain                                                                                           Genotype <br />  <br /> W303 ..............................................MATa/&#65533;?&#65533; his3-11,15/his3-11,15 ade2-1/ade2-1 ura3-1/ura3-1 leu2-3,112/leu2-3,112 trp1-1/trp1-1 can1-100/can1-100 <br /> W303a ............................................MATa his3-11,15 ade2-1 ura3-1 leu2-3,112 trp1-1 can1-100 <br /> KM003a ..........................................MATa his3-11,15 ade2-1 ura3-1 leu2-3,112 trp1-1 can1-100 sly1 <br /> KM007a ..........................................MATa/&#65533;?&#65533; his3-11,15/&#11001; ade2-1/ade2-1 ura3-1/ura3-1 leu2-3,112/leu2-3,112 trp1-1/&#11001; can1-100/can1-100 sly1/sly1 <br /> KM101 ...........................................MATa his3-11,15 ade2-1 ura3-1 leu2-3,112 trp1-1 can1-100 sly1 rrs1-1 <br /> KM111 ...........................................MATa his3-11,15 ade2-1 ura3-1 leu2-3,112 trp1-1 can1-100 rrs1::LEU2 pRS316-RRS1-HA <br /> KM113 ...........................................MATa his3-11,15 ade2-1 ura3-1 leu2-3,112 trp1-1 can1-100 rrs1::LEU2 sly1 pRS316-RRS1-HA <br /> KM114 ...........................................MATa his3-11,15 ade2-1 ura3-1 leu2-3,112 trp1-1 can1-100 rrs1::LEU2 sly1 pRS313-rrs1-1-HA <br /> KM123 ...........................................MATa ade2-1 ura3-1 leu2-3,112 can1-100 rrs1-1 <br /> KM129 ...........................................MATa his3-11,15 ade2-1 ura3-1 leu2-3,112 trp1-1 can1-100 rrs1::LEU2 pRS313-GAL1-RRS1 <br />   a <br />       Derived KM001 (20). <br />  <br />  <br />  <br /> After 10 washes PBS-BSA, cells incubated 1 h secondary          caused transfer cells restrictive tempera- <br /> antibody (rhodamine-conjugated goat-anti-mouse immunoglobulin G; Jackson               ture background sly1 (Fig. 1B) addition a <br /> ImmunoResearch Laboratory, Inc.) diluted 1:300. After 10 washes PBS- <br /> BSA PBS, cells stained 1 &#65533;?&#65533;g DAPI (4&#11032;,6&#11032;-                 secretory inhibitor, tunicamycin (Fig. 1C). The rrs1-1 mutation <br /> diamidino-2-phenylindole) ml mounting medium.                                   does suppress temperature sensitivity sly1 (<span id='am-77' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-78' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-81'  ></span>data</span> not <br />    Western blot analysis. Western blotting followed standard techniques,           shown), indicating rrs1-1 suppressor secre- <br /> signals visualized enhanced chemiluminescence (Amersham).                      tory defect sly1 mutation. Genetic analysis showed that <br />    Northern blot [methyl-3H]methionine pulse-chase analysis. Northern blot <br /> analysis carried using 1.5% agarose gels mRNAs 1.2% agarose            mutation recessive; mutant cells crossed <br /> gels steady-state level rRNAs formaldehyde described previously (4,       parental sly1 cells opposite mating type, RPL28 was <br /> 20). The oligonucleotides used analyzing mature rRNAs pre-rRNAs <br /> were follows; probes a, b, c correspond oligonucleotides B, C D, <br /> respectively, described Zanchin Goldfarb (41), probes d e corre- <br /> spond oligonucleotides g, respectively, described Berge`s et al. (1). <br />    [methyl-3H]methionine pulse-chase analysis carried using strains <br /> grown SC lacking methionine (SC&#11002;Met). Each culture pulsed with <br /> [methyl-3H]methionine chased nonradioactive methionine (500 <br /> &#65533;?&#65533;g/ml). Samples taken pouring cultures crushed sterile ice. Total <br /> RNA prepared analyzed agarose gels. <br />    Polyribosome analysis. Yeast cells grown 100 ml medium mid-log <br /> phase harvested immediately following addition cycloheximide (100 <br /> &#65533;?&#65533;g/ml). The pellet washed twice CH buffer (10 mM Tris-HCl [pH 7.4], <br /> 100 mM NaCl, 30 mM MgCl2, 50 &#65533;?&#65533;g cycloheximide ml, 200 &#65533;?&#65533;g heparin <br /> per ml) suspended 0.3 ml CH buffer. After glass-bead lysis yeast cells, <br /> aliquots supernatant corresponding 10 A260 units overlaid 12 <br /> ml 10 40% (wt/wt) sucrose gradient 50 mM Tris-HCl (pH 7.6)&#8211;12 <br /> mM magnesium acetate&#8211;50 mM ammonium acetate&#8211;1 mM dithiothreitol and <br /> centrifuged 2 h 40,000 rpm 4&#176;C Beckman SW40 Ti rotor. <br />  <br />  <br />                                   RESULTS <br />    Isolation yeast mutants unable respond secre- <br /> tory defect. In order obtain genes involved the <br /> signaling pathway, screened mutants secre- <br /> tory defect did cause transcriptional repression ri- <br /> bosomal protein genes. The strategy shown Fig. 1A. A <br /> plasmid generated HIS3 gene driven by <br /> the promoter ribosomal protein gene RPL28 (according <br /> to new nomenclature reference 18; called CYH2). <br /> <span id='am-44' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-45'  ></span><span id='am-46' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> plasmid transformed strain genotype <br /> his3&#11002; sly1ts. These cells phenotypically His&#11001; 25&#176;C, at <br /> 31&#176;C, just nonpermissive temperature sly1, they <br /> grow extremely slowly SC&#11002;His plates containing 3-amino-                                FIG. 1. Isolation rrs1-1 mutant. (A) The strategy used isolate yeast <br /> triazole HIS3 transcription repressed. The cells                        mutants defective signaling response secretory defect. The reporter <br /> were treated ethyl methanesulfonate survival fre-                            gene shown diagram consists coding sequence yeast HIS3 <br />                                                                                        gene fused promoter RPL28. Haploid KM003 (sly1 his3) cells carrying <br /> quency 25% grown SC&#11002;His plates containing 3-ami-                             reporter construct grow 31&#176;C SC&#11002;His plates containing 5 mM <br /> notriazole 31&#176;C. A mutant defective signaling pathway                        3-aminotriazole. Mutant cells respond secretory defect are <br /> was expected grow faster parental strain. Of                          expected grow. (B C) Northern analysis rrs1-1 mutant the <br /> an estimated 8 &#11003; 104 cells plated, 24 colonies grew rapidly                       secretory defect. (B) Yeast KM003 (sly1 RRS1) KM101 (sly1 rrs1-1) cells <br />                                                                                        grown YPD medium log phase (optical density 600 nm &#11005; 0.4 0.6) <br /> were picked. Each mutant tested ability repress                         25&#176;C. Half culture shifted 36&#176;C, 90 min, cells were <br /> transcription ribosomal protein genes RPL28 RPL3                                harvested. (C) Yeast strains W303a (SEC&#11001; RRS1) KM123 (SEC&#11001; rrs1-1) <br /> (formerly TCM1) response secretory defect; mutant                         grown log phase (optical density 600 nm &#11005; 0.4) 25&#176;C. Half the <br /> strains selected. In study, them, termed rrs1-1                       culture treated tunicamycin final concentration 1 &#65533;?&#65533;g/ml 4 h <br />                                                                                        25&#176;C, cells harvested. Total RNA prepared separated by <br /> (regulator ribosome synthesis), chosen anal-                        gel electrophoresis. Northern blot analysis carried using 32P-labeled <br /> ysis. In rrs1-1 cells, transcription ribosomal protein genes                    DNA probes specific RPL28 RPL3. SnoRNA U3 used marker <br /> is significantly repressed response secretory defect                       check equal loading. Ten micrograms total RNA loaded lane. <br />  2068       TSUNO ET AL.                                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />                                                                                        FIG. 3. The rrs1-1 mutation affects secretory defect response the <br />                                                                                     heat shock response. Yeast KM003 (sly1 RRS1) KM101 (sly1 rrs1-1) cells <br />                                                                                     grown log phase (optical density 600 nm &#11005; 0.5 0.8) YPD medium <br />                                                                                     25&#176;C, culture shifted 36&#176;C. Aliquots taken the <br />                                                                                     times indicated, total RNA prepared. Northern blot analysis carried <br />                                                                                     using 32P-labeled DNA probe specific RPL28. U3 used marker <br />                                                                                     check equal loading. Five micrograms total RNA loaded lane. <br />  <br />  <br />  <br />                                                                                     rrs1-1 mutation affects signaling pathway se- <br />                                                                                     cretory defect repression rDNA transcription. When <br />                                                                                     cells pulse-labeled [3H]uracil, incorporation <br />                                                                                     trichloroacetic acid-insoluble fraction also <br />                                                                                     strongly reduced conditions secretory <br />                                                                                     pathway blocked, indicating secretory defect causes <br />                                                                                     repression rDNA transcription inhibition of <br />                                                                                     pre-rRNA methylation (data shown). <br />                                                                                        RRS1 required repression ribosomal protein <br />                                                                                     genes response secretory defect heat <br />                                                                                     shock response. Mild heat shock causes temporary repres- <br />                                                                                     sion transcription ribosomal protein genes (7, 11). To <br />                                                                                     determine repression acts way that <br />                                                                                     failure secretory pathway, examined the <br />                                                                                     effect rrs1-1 mutation repression heat <br />                                                                                     shock 36&#176;C. As shown Fig. 3, ribosomal protein gene, <br />                                                                                     RPL28, significantly repressed 15 min tempera- <br />    FIG. 2. The rrs1-1 mutation affects transcriptional repression       ture shift-up RRS1 rrs1-1 strains, indicating <br /> rDNA response secretory defect. Strains KM003 (sly1 RRS1) KM101 <br /> (sly1 rrs1-1) grown log phase (optical density 600 nm &#11005; 1.0)          rrs1-1 mutation effect heat shock re- <br /> SC&#11002;Met medium 25&#176;C. Aliquots culture shifted 37&#176;C 90       sponse. In background sly1, RRS1 strain did not <br /> min. Each culture pulsed [methyl-3H]methionine (60 &#65533;?&#65533;Ci/ml) 5 min       recover RPL28 mRNA level response secretory <br /> and chased nonradioactive methionine (500 &#65533;?&#65533;g/ml). Samples taken           defect, rrs1-1 recovered defect secre- <br /> and chilled pouring crushed sterile ice time addition cold <br /> methionine (t &#11005; 0) chase time 10 min. Total RNA prepared,        tory response, consistent data Fig. 1B. <br /> and 20 &#65533;?&#65533;g sample analyzed electrophoresis. The gel divided          Cloning RRS1 gene. rrs1-1 mutant cells exhibited a <br /> into pieces. The upper gel soaked En3Hance (NEN), dried,             prolonged generation time (about 7 h) 18&#176;C cultured <br /> exposed film 3 days (top panel) 10 days (middle panel). The lower   YPD liquid medium. At higher temperatures, growth <br /> gel blotted probed U3 (bottom panel). <br />                                                                                     rate severely reduced; generation time rrs1-1 <br />                                                                                     cells 40% longer wild-type cells 30&#176;C and <br />                                                                                     20% longer 37&#176;C. On YPD plate, rrs1-1 cells ceased to <br /> significantly repressed restrictive temperature (data                    proliferate 18&#176;C. The cold sensitivity growth segregated <br /> shown). On sporulation dissection asci, phenotype                        2:2 sets tetrads RRS1/rrs1-1 diploid cells, and <br /> for defect response segregated 2:2, indicating                      cold sensitivity linked phenotype defect <br /> this phenotype single mutation nuclear gene.                       signaling detected Northern blot analysis (data not <br />    The rrs1-1 mutation affects repression rDNA                          shown). Therefore, wild-type RRS1 gene cloned by <br /> transcription response secretory defect. As secretory                     complementation cold sensitivity rrs1-1 cells. rrs1-1 <br /> defect causes transcriptional repression DNA encoding                            cells transformed library yeast genomic DNA <br /> rRNA (rDNA) ribosomal protein genes (20),                             constructed URA3&#11001; centromere-based vector, YCp50 <br /> examined rrs1-1 mutation effect                          (29). Of 2 &#11003; 104 Ura&#11001; transformants, grow 18&#176;C <br /> repression rDNA. We monitor synthesis pro-                           YPD plate. Restriction maps plasmid DNAs re- <br /> cessing rRNA [methyl-3H]methionine pulse-chase analy-                         covered transformants revealed plasmid <br /> sis, newly synthesized precursor rRNA methylated                           identical 18-kb insert. We determined partial DNA <br /> immediately (30, 37). As shown Fig. 2, secretory defect                     sequence plasmid. A database search revealed the <br /> shifting sly1 cells restrictive temperature leads strong                  insert carries complete ORFs partial ORFs. To <br /> repression rDNA transcription (lane 3), newly synthe-                        identify region 18-kb DNA insert required for <br /> sized precursor rRNA processed slowly (lane 4), con-                        complementation, subclones constructed their <br /> sistent previous data (20). On hand, rrs1-1                       complementing activities cold-sensitivity rrs1-1 were <br /> mutant cells, secretory defect does cause repression                       checked. The complementing activity fully recovered the <br /> rDNA transcription (Fig. 2, lanes 7 8), indicating                     plasmid, termed pAT-35, containing 1.6-kb EcoRI-SacI <br />  VOL. 20, 2000                                                          NOVEL REGULATORY PROTEIN OF RIBOSOME SYNTHESIS                                         2069 <br />  <br />  <br />  <br />  <br />    FIG. 4. Alignment Rrs1p related sequences <span id='am-66' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span>. The amino acid sequence Rrs1p predicted nucleotide sequence determined <br /> using plasmid pAT-35 dideoxy chain termination method Pharmacia DNA sequencer. The sequence aligned predicted sequences database; <br /> <span id='am-25' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-26' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-27'  ></span><span id='am-28' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Schizosaccharomyces</span> pombe SPBC29A3.16 (S.p.) (DDBJ/EMBL/GenBank accession number AL022299), C. elegans cosmid C15H11 (C.e.) (39), H. sapiens <br /> KIAA0112 (H.s.) (23). (A) Schematic alignment open reading frames. Amino acid residue numbers marked. The initial codon H. sapiens sequence <br /> has determined amino acids numbered methionine ORF. Identities Rrs1p highlighted regions listed the <br /> right. (B) Alignment single-letter code. Deletions needed alignment indicated dashes. Identical amino acids boxed black. When two <br /> sets identical amino acids, set boxed black shaded gray. <br />  <br />  <br />  <br /> fragment. Northern analysis revealed plasmid comple-                       Rrs1p C-terminal domain, sequence similarity <br /> ments defect signaling rrs1-1 (data shown). To                    (identity, 26.7%) lower. <br /> confirm pAT-35 contains wild-type version                              rrs1-1 produces truncated protein. In order determine <br /> rrs1-1 mutant gene, pAT-35 subcloned YIp5                            mutation point(s), rrs1-1 gene amplified PCR <br /> fragment resultant plasmid integrated                         using chromosomal DNA prepared mutant cells. Se- <br /> rrs1-1 strain. After transformant crossed wild-                      quence analysis showed rrs1-1 nucleotide <br /> type haploid, dissection 17 tetrads yielded 17 sets                      difference (a G-to-A transition), codon 114, resulting in <br /> spores, grow 18&#176;C (data shown). These                     stop codon. In order compare expression molec- <br /> results indicate pAT-35 contains RRS1.                                         ular size wild-type mutant Rrs1p, tandem <br />    Sequences RRS1. We determined entire nucleotide                           copies sequence encoding epitope influenza <br /> sequence insert pAT-35. RRS1 corresponds ORF                           virus HA protein inserted just initial codon of <br /> YOR294w, identified yeast genome project. RRS1 en-                           RRS1 rrs1-1. Each fused genes subcloned a <br /> codes novel protein consisting 203 amino acids                          CEN plasmid. The plasmid containing RRS1-HA com- <br /> predicted molecular mass 23 kDa isoelectric point                      plement lethality rrs1-null mutation the <br /> 9.9. Although conserved motifs detected using                            cold sensitivity rrs1-1 mutation. The plasmid containing <br /> Prosite, coiled-coil structure predicted amino acid                      rrs1-1-HA complement lethality rrs1-null mu- <br /> 34 63 using CoilScan (GCG). A database search revealed                        tation resulted cold sensitivity growth a <br /> that DNA sequences <span id='am-49' about='obo:NCBITaxon_9606' typeof='owl:Thing obo:NCBITaxon_9605 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="Homo sapiens" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-53'  ></span><span id='am-54' property="oboInOwl:hasExactSynonym" content="human" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasExactSynonym" content="man" datatype="xsd:string"></span>Homo sapiens</span>, <span id='am-39' about='obo:NCBITaxon_6237' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-40' property="rdfs:label" content="Caenorhabditis" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-43'  ></span>Caenorhabditis</span> el-                            defect signaling response secretory defect (data <br /> egans, <span id='am-30' about='obo:NCBITaxon_4896' typeof='owl:Thing obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_1 obo:NCBITaxon_147554 obo:NCBITaxon_34346 obo:NCBITaxon_4890 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4895 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-31' property="oboInOwl:hasAlternativeId" content="NCBITaxon:45042" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Schizosaccharomyces malidevorans" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="Schizosaccharomyces pombe" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="fission yeast" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasRelatedSynonym" content="Schizosaccharomyces pombeP" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-38'  ></span>Schizosaccharomyces pombe</span> encode proteins                            shown). These results indicate constructions are <br /> with significant similarity Rrs1p (Fig. 4) 36.8, 30.9,                  biologically functional. As shown Fig. 5, HA-tagged <br /> 43.6% identities, respectively. The amino acid sequences                     proteins, designated Rrs1-HA rrs1-1-HA, detected as <br /> putative human C. elegans homologs longer                         40-kDa 26-kDa bands sodium dodecyl sulfate-polyac- <br />  2070       TSUNO ET AL.                                                                                                                          MOL. CELL. BIOL. <br />  <br />  <br />                                                                                      DAPI, suggesting Rrs1-HA localized nu- <br />                                                                                      cleoplasm nucleolus. <br />                                                                                         rrs1-1 defective rRNA processing ribosome assem- <br />                                                                                      bly. Because Rrs1 protein appears localized the <br />                                                                                      nucleolus, Rrs1p expected role ribosome bio- <br />                                                                                      genesis. Furthermore, shown Fig. 2, maturation 25S <br />                                                                                      rRNA rrs1-1 mutant cells (lanes 5 6) appears be <br />                                                                                      slower wild-type cells (lanes 1 2) permissive <br />                                                                                      temperature. Thus, performed [methyl-3H]methionine <br />                                                                                      pulse-chase polyribosome analyses wild-type and <br />                                                                                      rrs1-1 cells 18&#176;C. rrs1-1 cells proliferate 18&#176;C a <br />                                                                                      YPD plate, continue grow 18&#176;C liquid culture <br />                                                                                      extremely low rate. As shown Fig. 7A, rrs1-1 cells, <br />                                                                                      rate processing pre-rRNA low mature <br />    FIG. 5. Western blot analysis Rrs1-HA rrs1-1-HA. Haploid SEC&#11001; <br /> cells expressing RRS1-HA (KM111; lanes 1 2) sly1 cells expressing            rRNA produced wild-type cells. The polyribosome <br /> RRS1-HA (KM113; lanes 3 4) rrs1-1-HA (KM114; lanes 5 6)              profile showed rrs1-1 cells, 80S ribosomes polyri- <br /> grown SC medium log phase 25&#176;C (lanes 1, 3, 5). Half culture     bosomes decreased, 40S subunits accumulated, and <br /> was shifted 36&#176;C 90 min (lanes 2, 4, 6). Crude extracts denatured <br /> in SDS sample buffer heated 95&#176;C 5 min. Equal amounts protein (44      half-mer polyribosomes appeared (Fig. 7B), suggesting that <br /> &#65533;?&#65533;g) subjected SDS-PAGE Western blotting using anti-HA. An                60S subunit production compromised 40S pro- <br /> arrow indicates nonspecific bands. The positions size markers shown        duction. <br /> the left. <br />                                                                                         RRS1 required ribosome biogenesis. In order to <br />                                                                                      achieve conditional expression RRS1, constructed a <br />                                                                                      strain chromosomal RRS1 gene disrupted and <br /> rylamide gel electrophoresis (SDS-PAGE) gels, considerably                           plasmid containing GAL1-promoter driven RRS1 in- <br /> larger predicted molecular masses proteins.                          troduced. This strain, KM129, grow YPG (galac- <br /> The levels Rrs1-HA rrs1-1-HA change re-                             tose) plate YPD (glucose) plate (data shown). <br /> sponse secretory defect. It noteworthy truncated                    In liquid galactose medium, KM129 cells grew fast as <br /> protein appears stability similar wild-type                   wild-type cells. After shift glucose medium, growth <br /> protein.                                                                             KM129 cells slowed gradually (Fig. 8A). We performed <br />    The RRS1 gene product essential nuclear protein. An                         [methyl-3H]methionine pulse-chase analysis investigate pre- <br /> rrs1-null allele created replacing XbaI-BstEII frag-                      rRNA synthesis, processing, stability rRNAs. <br /> ment LEU2 gene. The DNA fragment containing                             In Rrs1p-depleted cells, rate processing pre-rRNA <br /> rrs1::LEU2 disruption used transform diploid W303 and <br />                                                                                      low 25S rRNA produced wild-type <br /> KM007 (sly1/sly1). Southern blot analysis genomic DNA <br />                                                                                      cells (Fig. 8B). After 30-min chase, amounts 25S rRNA <br /> isolated transformants W303 KM007 <br /> demonstrated resultant diploid cells carried intact                     appeared somewhat decreased compared level at <br /> RRS1 gene disrupted insertion LEU2 (data                           20-min chase, suggesting newly formed 25S rRNA un- <br /> shown). Thirteen tetrads transformant W303                           stable Rrs1p-depleted cells. This experiment in <br /> nine tetrads transformant KM007 dissected.                          duplicate, similar results obtained. Accumulation of <br /> All tetrads yielded viable spores (data shown),                         18S rRNA Rrs1p-depleted cells slower in <br /> both Leu&#11002;. Microscopic inspection                         wild-type cells, defect appeared severe than <br /> spores showed stopped dividing divisions                     25S rRNA. The polyribosome profile Rrs1p-depleted <br /> (data shown), indicating RRS1 essential vegeta-                      cells showed decrease levels 80S ribosomes and <br /> tive growth. Indirect immunofluorescent microscopy revealed                          polyribosome, accumulation 40S subunits, appear- <br /> that Rrs1-HA localized nucleus 25&#176;C (Fig. 6). The                      ance half-mer polyribosomes (Fig. 8C). These results sug- <br /> staining region Rrs1-HA somewhat larger                          gest Rrs1p depletion affects 25S rRNA 60S subunit <br />  <br />  <br />  <br />  <br />   FIG. 6. Intracellular localization Rrs1-HA detected indirect immunofluorescence. Haploid cells expressing RRS1-HA (KM111) grown SC medium at <br /> 25&#176;C early log phase (optical density 600 nm &#11005; 0.1 0.15) stained anti-HA antibodies (A), DAPI (B), differential-interference-contrast (DIC) (C). <br /> Arrowheads positions panels A B. <br />  VOL. 20, 2000                                                           NOVEL REGULATORY PROTEIN OF RIBOSOME SYNTHESIS                                          2071 <br />  <br />  <br />  <br />  <br />    FIG. 7. rrs1-1 mutation causes defect rRNA processing ribosome assembly. (A) Pulse-chase analysis rRNA synthesis. W303a (RRS1, wild type [WT]) and <br /> KM123 (rrs1-1) cultured SC&#11002;Met medium 18&#176;C overnight. Each culture pulsed [methyl-3H]methionine (10 &#65533;?&#65533;Ci/ml) 3 min chased with <br /> nonradioactive methionine (500 &#65533;?&#65533;g/ml). Samples taken chilled pouring crushed sterile ice time addition cold methionine (t &#11005; 0) after <br /> the indicated chase times. Total RNA prepared 20 &#65533;?&#65533;g sample analyzed electrophoresis blotted Nytran membrane. The membrane was <br /> sprayed En3Hance (NEN) exposed film 2 days. The lower gel blotted probed U3. (B) Polyribosome profiles W303a (a) KM123 <br /> (b). The cells cultured YPD 18&#176;C overnight. Free ribosomal subunits polyribosomes cell extracts separated sucrose density gradient <br /> centrifugation. Arrows indicate half-mer polyribosomes. <br />  <br />  <br />  <br /> production 18S rRNA 40S subunit pro-                              results derepression ribosomal protein genes the <br /> duction.                                                                             secretory pathway blocked using tunicamycin by <br />    Steady-state levels pre-rRNAs GAL-RRS1 cells                           shifting sly1 cells restrictive temperature (Fig. 1). West- <br /> analyzed shift glucose medium 48 h.                           ern blotting analysis functional epitope-tagged Rrs1-HA <br /> Following transfer GAL-RRS1 strain glucose medium,                         rrs1-1-HA indicates rrs1-1-HA stability similar to <br /> the 35S pre-rRNA accumulated slightly 24 h (Fig. 9A                         wild-type version secretory defect does <br /> C), consistent results pulse-chase labeling. The                         cause significant change level, suggesting the <br /> reduced levels 27SA2 (Fig. 9B) 20S (Fig. 9A)                             effect rrs1-1 mutation response secretory defect <br /> suggest cleavage A2 retarded accordance                       change rrs1-1 concentration cell. <br /> depletion Rrs1p. In Rrs1p-depleted cells, 23S RNA appears                            Rrs1-HA appears localized nucleus, <br /> via aberrant pathway 35S pre-rRNA                             including nucleolus (Fig. 6). The nucleolus site of <br /> processed sites A0, A1, A2 instead cleaved site                     rRNA synthesis, rRNA processing, ribosomal subunit as- <br /> A3 internal transcribed spacer 1 (Fig. 9A B). The                         sembly (reviewed reference 38). Ribosomal proteins newly <br /> steady-state level 25S rRNA decreased time                         synthesized cytoplasm need transported the <br /> course experiments levels 20S pre-rRNA                        nucleolus assembled rRNAs. The assembled preribo- <br /> and mature 18S rRNA reduced (Fig. 9A D).                           somal subunits transported cytoplasm. Dur- <br /> The level 25S rRNA declined rapidly 18S                         ing ribosome assembly, pre-rRNAs modified cleaved <br /> rRNA; 12 h growth glucose medium,                          mature rRNAs. Pre-rRNA processing assembly ri- <br /> 25S rRNA reduced 39% 18S rRNA                               bosomal subunits tightly linked many <br /> was 57% (Fig. 9F). On hand, level ribosomal                         ribosomal proteins associate pre-rRNA early steps <br /> protein L28 mRNA significantly decreased 24 h                          prior cleavage. For example, mutants 60S subunit <br /> (Fig. 9E), indicating Rrs1p required transcrip-                      protein L11 (according new nomenclature reference 18; <br /> tion ribosomal protein genes.                                                     called L16) exhibit shortage mature 25S rRNA <br />                                                                                      accumulation rRNA precursors (22). Many trans-acting <br />                               DISCUSSION                                             factors involved pre-rRNA processing <br />                                                                                      ribosomal subunit assembly steps (reviewed reference <br />    Because huge ribosome biosynthesis                            34). Some enzymatic activity RNA <br /> growing cells, regulation essential economy                    cleavage, methylation, pseudouridine formation (2, 12, 13, <br /> cells. A functional secretory pathway required normal                     19, 31). Others role maintaining structure of <br /> synthesis ribosomes, suggesting coupling plasma                         complex containing pre-rRNA(s) ribosomal pro- <br /> membrane ribosome syntheses (20). In order elucidate                          teins. Depletion factors cause structural changes <br /> this regulatory mechanism, isolated rrs1-1 mutant                          complex prevent processing pre-rRNA. In <br /> which secretory defect (sly1ts) failed repress transcription                    Rrs1p-depleted cells, pre-rRNA processing retarded the <br /> of ribosomal protein genes. RRS1 essential gene                          A0, A1, A2 sites, consequently, 25S 18S <br /> function previously described. Comparison                        mature rRNA produced. Instead, aberrant cleavage pro- <br /> Rrs1p sequence database sequences showed pro-                          duces 23S rRNA, shown mutants (5, 14, 16, 41). <br /> tein conserved eukaryotes, information                          In [methyl-3H]methionine-pulse chase analysis rrs1-1 and <br /> about function available. The amino acid sequence                          GAL-RRS1 (Fig. 7A 8B), 20S band appears a <br /> Rrs1p readily recognizable protein motifs.                                    doublet. The upper band 21S produced aber- <br />    It unlikely rrs1-1 suppresses defect secretory                 rant cleavage (35). Steady-state levels 18S 25S rRNAs <br /> pathway sly1, rrs1-1 does suppress temperature                        decline depletion Rrs1p. These results suggest <br /> sensitivity growth (data shown) rrs1-1 mutation                      Rrs1p role maintaining structure preribo- <br />  2072       TSUNO ET AL.                                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />     FIG. 8. Depletion Rrs1p causes growth arrest defect ribosome biogenesis rRNA processing. (A) Growth curves W303a (RRS1, wild type [WT]) <br /> and KM129 (GAL-RRS1) cultured 25&#176;C YPGal shifted YPD. The change optical density 600 nm monitored shift. The cell cultures were <br /> diluted optical density lower 1.0, values calculated changes initial values. (B) Pulse-chase analysis rRNA synthesis. W303a <br /> and KM129 (GAL-RRS1) cultured SCGal&#11002;Met shifted SC&#11002;Met 12 h. Each culture pulsed [methyl-3H]methionine (10 &#65533;?&#65533;Ci/ml) 3 min <br /> and chased nonradioactive methionine (500 &#65533;?&#65533;g/ml). Samples taken chilled pouring crushed sterile ice time addition cold methionine <br /> lite(t &#11005; 0) indicated chase times. Total RNA prepared 20 &#65533;?&#65533;g sample analyzed electrophoresis blotted Nytran membrane. <br /> The membrane sprayed En3Hance (NEN) exposed film 2 days. The lower gel blotted probed U3. (C) Polyribosome profiles from <br /> W303a (a) KM129 (GAL-RRS1) (b). The cells cultured YPGal shifted YPD 12 h. Free ribosomal subunits polyribosomes cell extracts <br /> were separated sucrose density gradient centrifugation. Arrows indicate half-mer polyribosomes. <br />  <br />  <br /> somal subunit particles role specific enzyme                     primarily 25S rRNA maturation 60S subunit assembly. <br /> in step pre-rRNA processing.                                                   The decline 18S rRNA level Rrs1p-depleted cells may <br />    Interestingly, rrs1-1 mutation strongly diminished tran-                     secondary effect. The cold-sensitive phenotype the <br /> scriptional repression ribosomal protein genes                          rrs1-1 mutant explained defect assembly of <br /> rDNA response secretory defect (Fig. 1 2). These                        ribosomal subunits; yeast mutants defects in <br /> results suggest Rrs1p important role tran-                      ribosomal subunit assembly isolated screening a <br /> scriptional regulation rRNA ribosomal protein                           collection cold-sensitive strains (28). <br /> genes. However, rrs1-1 mutation depletion                          Transcription ribosomal protein genes temporarily re- <br /> of Rrs1p causes significant defect transcription ribo-                      pressed heat shock (7, 11). The rrs1-1 allele little effect <br /> somal protein genes conditions secre-                        temporary repression transcription brought by <br /> tory pathway normal. Rrs1p role maintenance                        mild heat shock, contrast effect secretory re- <br /> of balanced transcription components ri-                         sponse (Fig. 3). This suggests RRS1 required the <br /> bosome. Depletion Rrs1p using conditional rrs1-null mu-                        temporary repression ribosomal protein genes temper- <br /> tation caused defects pre-rRNA processing ribosomal                          ature shift-up. This indicates mechanism the <br /> subunit assembly; slow processing pre-rRNA, reduced pro-                         secretory response different heat shock <br /> duction 25S rRNA, decrease levels 80S ribosomes                        response, consistent results obtained using a <br /> polyribosomes, accumulation 40S ribosomal subunits,                          rap1-17 mutant (21). <br /> the appearance half-mer polyribosomes contain 43S                             The great majority ribosomal protein genes driven by <br /> initiation complexes stalled AUG start codon. The result                     Rap1-binding site(s) (15, 40). A ribosomal protein genes, <br /> indicates Rrs1p depletion strongly affects maturation                       including RPL3 (encoding ribosomal protein L3), site <br /> 25S rRNA 60S subunit assembly 18S                           Rap1 single Abf1-binding site instead (3, 6, 8). <br /> rRNA 40S subunit assembly. [methyl-3H]methionine                                Yet transcription ribosomal protein genes ap- <br /> pulse-chase analysis polyribosome pattern                          pears coordinated experimental conditions. <br /> rrs1-1 mutant cells cultured 18&#176;C suggest defect                     A secretory defect causes coordinated repression of <br /> 25S rRNA production 60S subunit assembly (Fig. 7B).                             ribosomal protein genes (20). We showed repression <br /> These results suggest rrs1-1 Rrs1p depletion affects                        RPL28 (Rap1 driven) RPL3 (Abf1 driven) was <br />  VOL. 20, 2000                                                          NOVEL REGULATORY PROTEIN OF RIBOSOME SYNTHESIS                                        2073 <br />  <br />  <br />  <br />  <br />    FIG. 9. Northern analysis steady-state level pre-rRNAs. The structure pre-rRNA locations oligonucleotide hybridization probes shown on <br /> the top. W303a (RRS1, wild type [WT]) KM129 (GAL-RRS1), cultured 25&#176;C YPGal, shifted YPD cultured 48 h. Total RNA equivalent <br /> to cells optical density 600 nm 0.5 used sample. Northern blot analysis carried using 32P-labeled DNA probes: probe (A), probe b <br /> (B), probe c (C), probe d 18S probe e 25S (D), probe RPL28 probe U3 (E). (F) RNA levels quantified using BAS-1000 (Fuji Photo <br /> Film Co.), ratio radioactivity value time point time zero shown. <br />  <br />  <br /> diminished C-terminally truncated rap1-17 mutation                           rrs1-1 result insensitivity secretory defect. However, <br /> (21). This suggests Rap1 essential repression                    mutants exhibit slow growth respond a <br /> response secretory defect Rap1-binding sites                       defect secretory pathway (data shown). The second <br /> necessary <span id='am-72' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-73'  ></span><span id='am-74' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>cis</span>-acting element repression. In pa-                     possibility feedback effect defect ribosome <br /> per, demonstrated rrs1-1 mutation affects                      synthesis rrs1-1 mutation overcome effect caused by <br /> repression RPL28 RPL3 (Fig. 1 2), suggesting                        secretory defect. We eliminate possibility com- <br /> that common mechanism regulates transcription                         pletely, rrs1-1 allele little effect temporary <br /> types genes.                                                                     repression transcription ribosomal protein genes brought <br />    The mechanism transcriptional repression rRNA                         mild heat shock (Fig. 3). This suggests rrs1-1 <br /> and ribosomal protein genes response secretory defect                       cells potential lead repression ribosomal <br /> remains elucidated. Our data suggest Rrs1p in-                        protein genes temperature shift-up. Thus, prefer an <br /> volved mechanism, alterna-                       alternative possibility: signal secretory defect might <br /> tive possibilities. One possibility slow growth                      transduced ribosome assembly machinery, in- <br />  2074        TSUNO ET AL.                                                                                                                                MOL. CELL. BIOL. <br />  <br />  <br /> cluding Rrs1p. A two-<span id='am-87' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-88' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-89' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-91'  ></span>hybrid</span> screen using RRS1 bait                                    ments yeast protein Rap1p selected silico ribosomal protein <br /> suggested Rrs1p interact Rpl11 (Y. Matsui et al.,                                gene promoter sequences. Bioinformatics 15:267&#8211;277. <br />                                                                                          16.   Lee, S. J., S. J. Baserga. 1999. Imp3p Imp4p, specific compo- <br /> unpublished data). As Rpl11 ribosomal protein localized                                   nents U3 small nucleolar ribonucleoprotein essential for <br /> on surface large subunit (36), Rrs1p associate                                pre-18S rRNA processing. Mol. Cell. Biol. 19:5441&#8211;5452. <br /> with ribosome interaction Rpl11. The possi-                             17.   Li, B., J. R. Warner. 1996. Mutation Rab6 homologue Sac- <br /> ble association Rrs1p ribosome in-                                    charomyces cerevisiae, YPT6, inhibits early Golgi function ribosome <br />                                                                                                biosynthesis. J. Biol. Chem. 271:16813&#8211;16819. <br /> vestigation. It possible ribosomal assembly machinery                            18.   Mager, W. H., R. J. Planta, J.-P. G. Ballesta, J. C. Lee, K. Mizuta, K. Suzuki, <br /> that involves interaction Rrs1p Rpl11                                      J. R. Warner, J. Woolford. 1997. A new nomenclature cytoplas- <br /> important regulation. According model,                                     mic ribosomal proteins Saccharomyces cerevisiae. Nucleic Acids Res. 25: <br /> mechanism transcriptional repression response se-                                   4872&#8211;4875. <br />                                                                                          19.   Mitchell, P., E. Petfalski, A. Shevchenko, M. Mann, D. Tollervey. 1997. <br /> cretory defect tightly linked normal regulatory mech-                                The exosome: conserved eukaryotic RNA processing complex containing <br /> anism maintains ribosome synthesis. The C-terminal re-                                    multiple 3&#11032;-5&#11032; exoribonucleases. Cell 91:457&#8211;466. <br /> gion Rrs1p deleted rrs1-1 responsible                         20.   Mizuta, K., J. R. Warner. 1994. Continued functioning secretory <br /> the signaling. In case, transcriptional repression ribo-                               pathway essential ribosome synthesis. Mol. Cell. Biol. 14:2493&#8211;2502. <br />                                                                                          21.   Mizuta, K., R. Tsujii, J. R. Warner, M. Nishiyama. 1998. The C-terminal <br /> somal protein genes follow repression rRNA genes                                    silencing domain Rap1p essential repression ribosomal pro- <br /> and C-terminal region Rrs1p required                                     tein genes response defect secretory pathway. Nucleic Acids <br /> signaling.                                                                                     Res. 26:1063&#8211;1069. <br />                                                                                          22.   Moritz, M., B. A. Pulaski, J. L. Woolford, Jr. 1991. Assembly 60S <br />                                                                                                ribosomal subunits perturbed temperature-sensitive yeast mutants de- <br />                            ACKNOWLEDGMENTS                                                     fective ribosomal protein L16. Mol. Cell. Biol. 11:5681&#8211;5692. <br />                                                                                          23.   Nagase, T., N. Miyajima, A. Tanaka, T. Sazuka, N. Seki, S. Sato, S. Tabata, <br />    We thank E. Tsuchiya, D. Hirata, A. Wada valuable <span id='am-82' about='obo:NCBITaxon_84399' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_87877 obo:NCBITaxon_6527 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216305 obo:NCBITaxon_216366 obo:NCBITaxon_216307 obo:NCBITaxon_84398 obo:NCBITaxon_977775 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_120490 obo:NCBITaxon_6072'><span id='am-83' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-84'  ></span><span id='am-85' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="Discus" datatype="xsd:string"></span>discus</span>- <br />                                                                                                K.-I. Ishikawa, Y. Kawarabayasi, H. Kotani, N. Nomura. 1995. Predic- <br /> sion, M. D. Rose yeast genomic DNA library, K. Matsumoto Y.                            tion coding sequences <span id='am-97' about='obo:NCBITaxon_32644' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-98' property="oboInOwl:hasRelatedSynonym" content="unknwon" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasRelatedSynonym" content="other" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasRelatedSynonym" content="unknown organism" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasRelatedSynonym" content="unknown" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasRelatedSynonym" content="sonstige nucleic acid" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasRelatedSynonym" content="unclassified sequence" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasRelatedSynonym" content="unspecified" datatype="xsd:string"></span><span id='am-105'  ></span><span id='am-106' property="oboInOwl:hasRelatedSynonym" content="miscellaneous nucleic acid" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasRelatedSynonym" content="none" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasRelatedSynonym" content="not shown" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasRelatedSynonym" content="unidentified organism" datatype="xsd:string"></span><span id='am-111' property="rdfs:label" content="unidentified" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasRelatedSynonym" content="not specified" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>unidentified</span> human genes. III. The coding <br /> Ohya plasmid, H. Yoshida Y. Maki help polyribosome                            sequences 40 new genes (KIAA0081&#8211;KIAA0120) deduced analysis of <br /> analysis, N. Tanaka help database search. We thank                         c <br /> </body></html>