<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Biophysical Journal: Biophysical Letters <br />  <br />  <br /> Properties Metabolic Networks: Structure versus Function <br />  <br /> R. Mahadevan*y B. O. Palssony <br /> *Genomatica Inc., San Diego, <span id='am-1' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2'  ></span><span id='am-3' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>California</span> 92121; yDepartment Bioengineering, University California, La Jolla, California 92093 <br />  <br />  <br /> ABSTRACT Biological <span id='am-70' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-71'  ></span><span id='am-72' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> high-throughput technologies describing network components (genes, proteins, <br /> metabolites) associated interactions driven reconstruction study structural (topological) properties of <br /> large-scale biological networks. In article, address relation functional structural properties using <br /> extensively experimentally validated genome-scale metabolic network models compute observable functional states of <br /> a microorganism compare &#8216;&#8216;structure versus function&#8217;&#8217; attributes metabolic networks. It observed that, functionally <br /> speaking, essentiality reactions node correlated node connectivity structural analyses biological <br /> networks suggested. These &#65533;?ndings illustrated analysis genome-scale biochemical networks three <br /> <span id='am-39' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-40' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> distinct modes metabolism. These results suggest fundamental differences different biological <br /> networks arising representation functional constraints. <br />  <br />                     Received publication 4 November 2004 &#65533;?nal form 18 November 2004. <br />                Address reprint requests inquiries R. Mahadevan, E-mail: rmahadevan@genomatica.com. <br />  <br />  <br />    The combination various high-throughput technologies            2. Flow networks, specific variable mass or <br /> has enabled coordinated study cellular components              energy flow conserved node, <span id='am-65' about='obo:NCBITaxon_94025' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_27394 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_74971 obo:NCBITaxon_74974 obo:NCBITaxon_6854 obo:NCBITaxon_6843 obo:NCBITaxon_74975 obo:NCBITaxon_33154 obo:NCBITaxon_6905 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6893 obo:NCBITaxon_6072'><span id='am-66' property="rdfs:label" content="Meta" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-68'  ></span><span id='am-69' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>meta</span>- <br /> the level genes, proteins, metabolites, associated            bolic networks. Thus, fundamental properties bio- <br /> interactions them. These studies resulted              logical networks classes significantly <br /> generation large-scale data sets served              different. <br /> foundation reconstruction metabolic, regulatory, <br />                                                                          In report, argue metabolic networks have <br /> signaling, protein-protein interaction networks. The avail- <br />                                                                       unique properties resulting a), conservation con- <br /> ability interaction networks spurred analysis <br />                                                                       straints satisfied node, b), way <br /> of structural, i.e., topological, properties networks <br />                                                                       metabolic networks represented, nodes are <br /> using quantitative approaches (1). Since functional states of <br />                                                                       metabolites links reactions catalyzed by <br /> metabolic networks (that correspond phenotypic functions) <br />                                                                       specific gene products. <span id='am-34' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-35' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-38'  ></span>This</span> representation different from <br /> can assessed, compare important <br />                                                                       protein-protein interaction networks, nodes the <br /> structural properties networks comes to <br />                                                                       gene products links correspond interactions. The <br /> interpreting functional states. <br />                                                                       analysis protein-protein interaction networks suggested <br />    Network analysis suggested biological networks <br />                                                                       deletion highly connected proteins <br /> have important structural properties. First, been <br />                                                                       correlates lethal phenotype (2). In contrast, node <br /> shown networks, including metabolic net- <br />                                                                       (i.e., metabolite) metabolic networks deleted by <br /> works, scale-free possess &#8216;&#8216;small world&#8217;&#8217; property <br />                                                                       genetic techniques, links can. <br /> (1). Second, scale-free networks suggested high <br />                                                                          These topological properties derived network <br /> error tolerance (tolerance random failure) low <br />                                                                       structure, functional, phenotypic, states. <br /> attack tolerance (vulnerability failure highly <br />                                                                       Recent studies indicated metabolic networks have <br /> connected nodes). However, biological networks have <br />                                                                       flux distributions average path length longer <br /> differences functional states. In general, network <br />                                                                       length obtained consideration network <br /> of interactions biological entities (genes, proteins, <br />                                                                       structure (3) functional states have <br /> metabolites, etc.) classified basis nature of <br />                                                                       scale-free characteristics. The proposed error attack <br /> the interaction broad categories: <br />                                                                       tolerance properties metabolic networks assessed in <br /> 1. Influence networks, nature interactions           context functional states using flux balance analysis of <br />    &#8216;&#8216;influence-based&#8217;&#8217; protein-protein interaction        genome-scale metabolic networks (4). Such silico models <br />    signaling networks, links represent          currently available organisms (4). The silico <br />    interaction present not. This <span id='am-31' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-32' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> extended        models <span id='am-42' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-43' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-49'  ></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span>Escherichia coli</span> <span id='am-7' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-20'  ></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> in <br />    include cases type interaction important        particular, extensively validated physiological <br />    addition presence/absence interaction (for ex- <br />    ample, gene regulatory networks transcription fac-                                                2005 Biophysical Society <br />    tors activate repress genes expression).                                                 doi: 10.1529/biophysj.104.055723 <br />  <br /> Biophysical Journal: Biophysical Letters                                                                                              L07 <br />  Biophysical Journal: Biophysical Letters <br />  <br />  <br /> data including data knockout phenotypes (5, 6). These                  From results, observed lethality fraction <br /> genome-scale models metabolic networks                    (fL) connected metabolites higher than <br /> found compute observable functional network states (7,              highly connected metabolites irrespective the <br /> 8) used assess systematically attack            size complexity metabolic network. In fact, <br /> and error tolerance nodes high number con-           surprisingly, networks studied, the <br /> nections relative low number con-              points fall narrow range 0.2&#8211;0.5. There are <br /> nections.                                                              metabolites connectivity have <br />    As biological networks, metabolic networks            lethality fraction 1. These metabolites occur in <br /> satisfy functions simultaneously. To support growth,              linear pathway, end metabolite essential <br /> a genome-scale metabolic network produce              biomass component (e.g., amino acid histidine). The <br /> 35 different compounds (9) (Fig. 1, right panel). The                  relation lethality fraction connectivity is <br /> genome-scale silico models E. coli S. cerevisiae             illustrated Fig. 1, right panel, representative <br /> have used analyze lethality gene deletion           metabolites. Additionally, investigate the <br /> have shown correctly predict experimental                  behavior metabolic network specific metab- <br /> phenotype 78.7% (6) 82.6% (5) 13,750                 olite eliminated network removing the <br /> 4,154 cases, respectively. The computed lethal gene knock-             reactions consume/produce metabolite. To realize <br /> outs used compare connectivity, Ci, node (a         experimentally, genes corresponding re- <br /> metabolite) lethality links (reactions) it. The      actions simultaneously deleted. The <br /> number lethal reactions (or connections, CL,i)            results analysis depicted  <br /> </body></html>