<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>&#63193; 1996 Oxford University Press                                            Nucleic Acids Research, 1996, Vol. 24, No. 20 4009&#8211;4014 <br />  <br />  <br /> Allelism PSO4 PRP19 links pre-mRNA <br /> processing recombination error-prone DNA <br /> repair <span id='am-1' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-8'  ></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> <br /> Martin Grey, Andreas D&#252;sterh&#246;ft1, Joao A.P. Henriques2 Martin Brendel* <br />  <br /> Institut f&#252;r Mikrobiologie der J.W.Goethe Universit&#228;t, Theodor-Stern-Kai 7, Haus 75, 60590 Frankfurt/Main, <br /> Germany, 1Qiagen GmbH, Max-Volmer-Stra&#223;e 4, 40724 Hilden, Germany 2Department of <br /> Biophysics/Biotechnology Center&#8211;UFRGS, 91507-970 Porto Alegre, RS, Brazil <br />  <br /> Received June 19, 1996; Revised Accepted August 27, 1996                                DDBJ/EMBL/GenBank accession no. X99770 <br />  <br />  <br />  <br /> ABSTRACT                                                              yeast mutants sensitive photoactivated psoralens (pso mutants; <br />                                                                       9) allocated epistasis group, <br /> The radiation-sensitive mutant pso4-1 Saccharo-                    depending mutagen applied biological end point <br /> myces cerevisiae shows pleiotropic phenotype,                       scored respective double mutants (8). A mutant allele of <br /> including sensitivity DNA cross-linking agents,                    gene PSO4, involved error-prone repair some <br /> nearly blocked sporulation reduced mutability. We                 types recombination, i.e. gene conversion, crossing-over and <br /> have cloned putative yeast DNA repair gene PSO4                   intrachromosomal recombination (10,11,12), originally <br /> from genomic library complementation                      isolated X-ray-sensitive mutant xs9 (13); be <br /> blocked UV-induced mutagenesis sporulation                     particularly sensitive photoactivated bifunctional 8-methoxy- <br /> in diploids homozygous pso4-1. Sequence analysis                  psoralen renamed pso4-1 (10). <span id='am-25' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-26' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-27'  ></span><span id='am-28' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>This</span> yeast mutant is <br /> revealed gene PSO4 consists 1512 bp located                   phenotypically similar recA mutant <span id='am-30' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-31' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34'  ></span><span id='am-35' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span>Escherichia coli</span> in <br /> upstream UBI4 chromosome XII encodes                      combines mutagen radiation sensitivity block in <br /> putative protein 56.7 kDa. PSO4 allelic PRP19,               recombination induced mutagenesis. The pleiotropic repair <br /> a gene encoding spliceosome-associated protein,                     phenotype caused singly existing pso4-1 mutant led to <br /> but shares significant homology yeast                   allocation PSO4 epistasis group (8,14). <br /> genes. Gene disruption destroyed reading frame                    More recent studies showed heterologous expression recA <br /> of PSO4 clone resulted death haploid cells,                 increased resistance UV ionizing radiation S.cerevisiae <br /> confirming finding PSO4/PRP19 essential                wild-type recombination-deficient rad52-1 mutant <br /> gene. Thus, PSO4 essential DNA repair                    cells (15,16). However, recA-like yeast mutant pso4-1 showed <br /> gene yeast S.cerevisiae.                                 restored induced mutability transformation multicopy <br />                                                                       vector containing E.coli recA gene (17). Biochemical analysis <br /> INTRODUCTION                                                          proved normal incision 8-MOP + UVA-induced interstrand <br /> At 50 genes encoding proteins involved DNA repair        cross-links (ICLs), failure DNA strand rejoining pso4-1 <br /> known yeast <span id='am-18' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-19' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-24'  ></span>Saccharomyces</span> cerevisiae (1,2) and,          (16), depend recombinational step. The <br /> molecular studies revealed close               apparent role gene PSO4 DNA repair processes attributed <br /> relationship DNA repair enzymes humans                genes belonging epistasis group II (recombination) III <br /> mammals, simple eukaryote advanced model             (error-prone) makes interesting candidate future <br /> which complex mechanisms underlying DNA repair           understanding interconnections repair processes <br /> unraveled near future. The DNA damage-sensitive           yeast. Progress elucidating function Pso4 protein <br /> yeast mutants initially grouped phenotypic groups     necessitated molecular cloning PSO4, wish to <br /> (3,4) which, double mutant analysis measured synergistic      report communication. <br /> or epistatic interactions, allocated epistasis groups <br /> (5&#8211;7). This classification holds today,       MATERIALS AND METHODS <br /> group I comprising &#8764;20 genes RAD3-like encode            Yeast strains <span id='am-53' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-54' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>plasmids</span> <br /> proteins nucleotide excision repair, group II comprising &#8764;12 <br /> genes RAD52-like encode proteins           The yeast strains used listed Table 1. Single copy vector <br /> involved repair DNA double-strand breaks recombina-         pRS316 (18) multicopy vector pRS426 (19), containing <br /> tional mechanisms largest, far                URA3 selectable marker, used subcloning, comple- <br /> heterogeneous, group III, defined RAD6, comprising &#8764;20             mentation tests sequencing. The original complementing <br /> genes, responsible error-prone DNA repair       isolate pMG470 based multicopy plasmid YEp24 (20). <br /> (1,2,8). Some DNA damage-sensitive mutants, e.g.       Hence, subcloning performed using multicopy vector pRS426 <br />  <br />  <br /> * To   correspondence addressed <br />  4010 Nucleic Acids Research, 1996, Vol. 24, No. 20 <br />  <br />  <br />  <br />  <br /> Figure 1. (A) Subcloning pso4-complementing gene passenger DNA pMG470 multicopy plasmids. The restriction maps passenger DNAs <br /> of pMG470-derived plasmids shown. pMG470: original clone isolated gene bank (20) harboring 16 kb passenger. Only 6.5 kb area that <br /> overlaps second positive clone, pMG490 (not shown), depicted detail. pMG471, pMG473 pMG475: non-complementing subfragments pMG470. <br /> pMG480: fragment containing just gene, PRP19, able complement phenotypes pso4-1 mutant. Abbreviations cleavage sites: B/S, ligation <br /> site BamHI Sau3A; E, EcoRI; L, SalI; X, XbaI; Y, BstYI. +/&#8211;: Plasmid complements/does complement pso4-1/pso4-1 pso4-1/prp19-0 mutant. <br /> (B) Non-complementation pso4-1 prp19-0 alleles. Diploids containing different allele combinations constructed tested survival 30 36 C <br /> by streaking cell suspensions twoYEPD plates subsequently incubated 36 30 C respectively. (C) Survival mutagenesis irradiation with <br /> UV254 light heteroallelic pso4-1/prp19-0 diploid MG5101 transformed plasmid pRS426 (triangles) pMG480 containing justPRP19 (squares) <br /> and heterozygous pso4-1/PRP19 diploid MG5103 transformed plasmid (circles) positive control. <br />  <br />  <br />  <br /> to allow screening possible multicopy suppressor                        CAN1/can1 marker possible select forward mutants <br /> PSO4 (Fig. 1A).                                                                          homozygous can1 replica-plating synthetic medium <br />                                                                                          lacking arginine uracil supplemented 20 mg/l <br /> Cloning PSO4 <br />                                                                                          canavanine (SynCo&#8211;Arg&#8211;Ura+Can). After irradiation with <br /> Gene PSO4 cloned transforming diploid strain MG5128                               UV254 light (20 J/m2) plates incubated 30 C the <br /> homozygous pso4-1 yeast genomic library (20)                                  dark 3&#8211;5 days. Clones papillae SynCo <br /> according lithium acetate-based protocol described                             &#8211;Arg&#8211;Ura+Can isolated corresponding master <br /> Gietz et al. (21). Due previously introduced heterzygous                          plate tested ability sporulate. <br />  <br /> Table 1. Saccharomyces cerevisiae strains used study <br />  <br />  Strain         Genotype                                                                                                                           Source <br />  xs9            MATa lys2 pso4-1                                                                                                                   A.C.Schenberg <br />  W303           MATa/MAT&#945; ura3-1/ura3-1 ade2-1/ade2-1 trp1-1/trp1-1 leu2-3,112/leu2-3,112 his3-11,15/his3-11,15 can1-100/can1-100 PSO/PSO          R.Rothstein <br />  MG5100         MATa/MAT&#945; ura3-1/ura3-1 ade2-1/ade2-1 trp1-1/trp1-1 leu2-3,112/leu2-3,112 his3-11,15/his3-11,15 can1-100/can1-100 PSO/pso4::HIS3   This study <br />  MG5101         MATa/MAT&#945; ura3-1/ura3 ade2-1/ADE trp1-1/TRP leu2-3,112/LEU his3-11,15/HIS can1-100/CAN pso4::HIS3/pso4-1                           This study <br />  MG5103         MATa/MAT&#945; ura3-1/ura3 ade2-1/ADE trp1-1/TRP leu2-3,112/LEU his3-11,15/HIS can1-100/CAN PSO4/pso4-1                                 This study <br />  MG5128         MATa/MAT&#945; ura3-52/ura3 ade2/ADE can1/CAN pso4-1/pso4-1                                                                             This study <br />                                                                                                               4011 <br />  <br />                                                                                 Nucleic Acids <br />                                                                                 Nucleic       Research,1994, <br />                                                                                         Acids Research, 1996,Vol.  24,No. <br />                                                                                                               Vol.22,  No.120         4011 <br />  <br /> Mutagenicity assay survival                                         The putative Pso4 protein <br /> Exponentially growing cells cultivated liquid synthetic medium       We determined size PSO4/PRP19 gene 1512 bp, <br /> supplemented appropriate amino acids bases            corresponds 56.7 kDa protein consisting 503 amino <br /> washed resuspended phospate buffer (67 mM, pH 7)            acids, instead 502, originally described PRP19 Cheng <br /> final density 107 cells/ml irradiated different UV254       et al. (22), owing additional proline position 239. <br /> doses. For determination survival cells plated       Sequence analysis putative protein revealed myb-like <br /> complete medium (SynCo) determination forward            DNA binding domain (W[ST]X2E[DE]X2[LIV]) C-terminus <br /> mutations (can1) SynCo arginine plus canavanine          Pso4/Prp19 protein (positions 457&#8211;465, WTKDEESAL). <br /> (20 mg/l) respectively.                                                 The retroviral oncogene v-myb cellular counterpart c-myb <br />                                                                         encode nuclear DNA binding proteins. In S.cerevisiae, the <br /> DNA sequencing                                                          myb-related genes include DNA-binding proteins REB1 (23) <br /> DNA sequencing performed framework            BAS1 (24). However, Pso4/Prp19 protein does share <br /> European Union BIOTECH program sequence entire                   considerable homology putative proteins encoded by <br /> chromosome XII. A set random subclones plasmid cloning            yeast genes. The nucleotide sequence PSO4 <br /> vector pUC18 generated insert region cosmid             appear EMBL, GenBank DDBJ Nucleotide <br /> 1F17. These subclones served starting points primer        Sequence Databases accession no. X99770. <br /> walking strategy. Double-stranded template DNA cosmid <br /> and DNA subclones purified using QIAwell                 Construction phenotype pso4 null allele <br /> Purification System (Qiagen). Cycle sequencing reactions were <br /> carried using Taq DNA polymerase fluorescent terminators.       After vitro disruption Pso4/Prp19-encoding ORF the <br /> Gel electrophoresis automated base calling          SalI site inserting HIS3 (cf. Fig. 1), performed one-step <br /> ABI 373A Stretch DNA Sequencer (Applied Biosystems).                    disruption experiments haploid diploid wild-type cells. <br /> Contig assembly DNA sequence editing using            While haploid transformants correct insertion be <br /> DNASTAR software package (Lasergene).                                   found, diploid disruptants (MG5100, Table 2) showed two <br />                                                                         surviving His auxotrophic ascospores sporulation tetrad <br /> RESULTS                                                                 analysis. The absence His prototrophic spores confirmed <br />                                                                         previously reported essentiality PSO4/PRP19 gene <br /> Cloning DNA repair gene PSO4                                         (22). However, demonstrated this <br />                                                                         essentiality restricted spore germination. Therefore, the <br /> Diploids homozygous mutant allele pso4-1 higher than <br />                                                                         heterozygous diploid MG5100 transformed pMG480 <br /> wild-type UV sensitivity, lower wild-type UV-induced <br />                                                                         (URA3; PSO4), resulting viable spores sporulation. <br /> mutation mitotic recombination sporulate poorly <br />                                                                         Plasmid loss experiments spores containing HIS3 <br /> (&lt;1% asci) appropriate medium (8). PSO4 molecularly <br />                                                                         disruption marker did yield viable Ura auxotrophs, while <br /> cloned screening transformants diploid MG5128 <br />                                                                         haploid His auxotrophic non-disruptants showed plasmid loss of <br /> homozygous pso4-1 heterozygous CAN1 (Table 1), in <br />                                                                         &#8764;20% 36 h. This indicates essentiality PSO4 is <br /> which inducibility mutagenesis UV254 restored <br />                                                                         restricted spore germination, loss plasmid <br /> (CAN1&#8594;can1r). Amongst 6000 transformants to <br />                                                                         results lethality haploid disruptants. As essentiality be <br /> show wild-type-like UV-induced mutagenesis furthermore <br />                                                                         verified construct using URA3 reporter gene (not <br /> restored sporulation (40% transformants respective <br />                                                                         shown), disrupted alleles generally named pso4-0 for <br /> wild-type versus maximal number &lt;1% asci mutant) <br />                                                                         purpose simplification discussion. <br /> and restored resistance DNA cross-linking mutagens. Restriction <br /> analysis complementing plasmids, named pMG470                PSO4 allellic PRP19 <br /> and pMG490, showed passengers contained an <br /> identical 6.5 kb overlapping fragment (depicted Fig. 1A).            Besides fact gene PRP19 sufficient to <br /> Hybridization 32P-labeled 2 kb EcoRI fragment            complement pso4-1 mutation, final evidence allelism <br /> area (Fig. 1A, pMG475) revealed passengers        cloned ORF PSO4 obtained showing null <br /> chromosome XII (not shown). Two subfragments obtained                allele prp19::HIS3 complement phenotype pso4-1 <br /> restriction plasmid pMG470 singular SalI site yielded     mutant heteroallelic diploid constructed crossing <br /> two plasmids pMG471 pMG473 (Fig. 1A), unable                original pso4-1 mutant xs9 haploid prp19::HIS3 <br /> restore PSO phenotype (Table 2A), indicating SalI           disruptant MG5100-1C opposite mating-type, latter <br /> located pso4-1 complementing open reading frame              containing PSO4-harboring plasmid pMG480 ensure viability of <br /> (ORF) promoter. We determined DNA sequence           haploid disruptant. After plasmid loss, resulting Ura <br /> area match gene PRP19 (22), containing SalI        auxotrophic diploid MG5101, heteroallelic pso4-1/prp19::HIS3, <br /> site. Further sequencing revealed PSO4/PRP19 located            viable showed pleiotropic phenotype typical a <br /> &#8764;1.5 kb upstream UBI4 shares promoter region        homozygous pso4-1 diploid, including strong reduction in <br /> &#8764;320 bp ORF (L0916, Fig. 1A) upstream PRP19             sporulation induced mutagenesis sensitivity <br /> encoding hitherto uncharacterized protein containing putative       DNA cross-linking mutagens (Table 2 Fig. 1C). <br /> ATP/GTP binding site. Subcloning 2.4 kb fragment                   Since prp19-0 allele unable complement pso4-1 mutant <br /> containing gene PRP19 (pMG480, Fig. 1A) revealed         phenotype, final proof genes allelic. Thus <br /> gene sufficient restore wild-type-like phenotype   terms pso4::HIS3 (pso4-0) prp19::HIS3 (prp19-0) be <br /> transformed pso4 mutant (Fig. 1C Table 2A).                  considered synonymous. <br />  4012 Nucleic Acids Research, 1996, Vol. 24, No. 20 <br />  <br /> Table 2. (A) Complementation analysis different subclones pMG470 (see Fig. 1A); (B) complementation analysis set plasmids transformed <br /> into heteroallelic diploid MG5101; (C) non-complementation prp19 pso4 mutant alleles (MG5101) evidence allelism (see Fig. 1B C) <br />  <br />  Strain            Relevant genotype      Plasmid              Sporulation          HN2                   30 C                 36 C              Mutability <br />  (A) <br />  <br />  MG5128            pso4-1/pso4-1          pRS426               &lt;1%                  S                     +                    &#8211;                 &#8211; <br />                                           pMG470               45%                  R                     +                    +                 + <br />                                           pMG471               &lt;1%                  S                     +                    &#8211;                 &#8211; <br />                                           pMG473               &lt;1%                  S                     +                    &#8211;                 &#8211; <br />                                           pMG480               35%                  R                     +                    +                 + <br />  (B) <br />  <br />  MG5101            prp19-0/pso4-1         pRS426               0.1%                 S                     +                    &#8211;                 &#8211; <br />                                           pMG470               40%                  R                     +                    +                 + <br />                                           pMG471               0.1%                 S                     +                    &#8211;                 &#8211; <br />                                           pMG473               0.1%                 S                     +                    &#8211;                 &#8211; <br />                                           pMG480               35%                  R                     +                    +                 + <br />  (C) <br />  <br />  W303              PSO/PSO                None                 35%                  R                     +                    +                 + <br />  MG5100            prp19-0/PSO            None                 33%                  R                     +                    +                 + <br />  MG5128            pso4-1/pso4-1          None                 &lt;1%                  S                     +                    &#8211;                 &#8211; <br />  MG5103            pso4-1/PSO             None                 32%                  R                     +                    +                 + <br />  MG5101            prp19-0/pso4-1         None                 0.1%                 S                     +                    &#8211;                 &#8211; <br />  <br /> R, wild-type-like resistance; S, pso4 mutant-like sensitivity HN2. <br /> +/&#8211; corresponds viability/non-viability 30 36 C mutability/reduced mutability respectively. <br /> Only plasmids pMG470 pMG480 complement pso4-1 mutation. <br /> Sporulation efficiency, sensitivity DNA cross-linking mutagen nitrogen mustard (HN2), mutability determined 30 C. <br />  <br /> pso4-1 non-lethal allele essential gene                                  mutagenesis, lowered spontaneous induced recombination <br /> PSO4/PRP19                                                                           nearly totally blocked pre-meiotic DNA synthesis and <br />                                                                                      sporulation (8,12,27). We isolated gene PSO4, which <br /> Although disruption PSO4/PRP19 lethal, mutants containing                    member RAD52 epistasis group complementation <br /> the pso4-1 allele viable above-mentioned                            pso4-1 mutant&#8217;s drastically reduced induced mutability and <br /> phenotype. Often, viable mutants essential genes isolated                     blocked sporulation. Characterization DNA repair gene PSO4 <br /> as sensitve temperatures &gt;30 C. We tested                           showed allelism yeast gene PRP19, encodes 502 <br /> diploids harboring different alleles PSO4 possible temperature                amino acid spliceosome-associated protein (22,28). In contrast, <br /> sensitivity. In fact, diploids homoallelic pso4-1                 PSO4 sequence contains additional proline-coding CCC <br /> heteroallelic (pso4-1/prp19-0) sensitive 36 C. Again,                    triplet encodes 503 amino acid protein. However, this <br /> prp19-0 allele failed complement sensitivity,                      discrepancy strain-specific difference is <br /> more confirming allelism genes (Fig. 1B). When                               presently likely irrelevant discussion putative <br /> comparing growth yield diploid strains 30 C                         protein. <br /> found heteroallelic pso4-1/pso4-0 diploid MG5101 grow                          The allelism PSO4 PRP19 implicates yeast gene <br /> at 30 C (cf. above) severely retarded growth rate                     encodes protein functions RNA splicing in <br /> a generation time &gt;3 h YEPD medium, isogenic                            error-prone DNA repair, recombination sporulation this <br /> PSO4/pso4-0 diploid MG5103 showed generation time &#8764;1.5 h,                       organism. It DNA repair gene function <br /> comparable wild-types (not shown).                                        linked processing RNA essentiality underscores <br />                                                                                      vital function process. Next RAD3 and <br /> DISCUSSION                                                                           SSL2/RAD25, PSO4/PRP19 DNA repair gene is <br />                                                                                      essential growth viability S.cerevisiae. <br /> Repair DNA double-strand breaks S.cerevisiae involves                             The Prp19 protein demonstrated essential for <br /> genes RAD52 epistasis group. The corresponding proteins                       splicing pre-mRNA (28). However, biochemical characterization <br /> are supposed act recombinational mechanism                         revealed tightly associated snRNAs, is <br /> of shown constitute protein complex                        associated spliceosome splicing reaction (22). <br /> has termed recombinosome (25,26). A pso4-1 mutant                         Interestingly, Cheng et al. discussed fact Prp19 distinct <br /> previously shown exhibit pleiotropic phenotype, i.e.                       Prp proteins spliceosomal components <br /> sensitivity DNA damaging agents, nearly blocked induced                      regarding protein sequence does contain of <br />                                                                                                                 4013 <br />  <br />                                                                                 Nucleic Acids <br />                                                                                 Nucleic       Research,1994, <br />                                                                                         Acids Research, 1996,Vol.  24,No. <br />                                                                                                               Vol.22,  No.120              4013 <br />  <br /> the motifs Prp sequences. It      (35) require splicing proper function. Since <br /> demonstrated Prp19 protein associated protein       pso4-1 mutants able incise DNA (14), known <br /> complex different spliceosome, consisting         splicing-dependent repair gene functional, pointing <br /> seven proteins addition Prp19, itself likely present     normal splicing activity pso4-1 mutants 30 C. Also, if <br /> an oligomeric form (29). However, Prp19-associated complex,        non-functional splicing resulted repair deficiency pheno- <br /> itself big spliceosome, unlikely bind     types, &#8764;50 DNA repair genes cloned ought to <br /> latter complex (29). The authors suggest Prp protein      encode proteins needed splicing, i.e. turned <br /> be released associated splicing complex. This       allelic PRP genes. <br /> event concomitant just dissociation         The severe handicap growth complete medium 30 C of <br /> of U4 snRNA ATP-dependent conformational change             heteroallelic pso4-1/pso4-0 heterozygous pso4-1/PSO4 <br /> before formation functional spliceosome, suggesting          diploids indicates quantity protein produced one <br /> Prp19 function step spliceosome assembly (30).          pso4-1 mutant allele sufficient normal spliceosome <br />    What be, hand, function               activity, protein encoded PSO4 wild-type <br /> PRP19/PSO4-encoded protein recombinational error-prone          allele normal growing PSO4/pso4-0 diploid apparently <br /> repair? All evidence gathered genetic biochemical             satisfies cellular demand spliceosome-associated protein <br /> experiments using allele pso4-1 point late function repair     function. In context partial complementation yeast <br /> of DNA lesions missing mutant strain. The excision           pso4-1 mutant&#8217;s sensitivity photoactivated 8-MOP + UVA by <br /> 8-MOP + UVA-induced ICLs DNA thought proceed                overexpression RecA protein (17) new significance. RecA <br /> normally, production DNA double-strand breaks repair        protein bind single-and double-stranded DNA, thereby <br /> intermediates which, however, rejoined pso4-1 (14). It      facilitating initiation recombination (36). Interstrand cross-link <br /> has suggested repair DNA lesions ICL type          repair E.coli yeast shown proceed via <br /> would require modes repair (31,32),             double-strand breaks (cf. 2,31); secondary lesions be <br /> removal damage repairosome postulated                substrate RecA RecA-like proteins which, helical <br /> nucleotide excision repair (33) second enzyme          filament formation, initiate recombinational repair. The RecA <br /> complex repair strand breaks, PSO4/PRP19           Rad51 proteins share considerable sequence homology and <br /> encoded protein function. The existence S.cerevisiae   strong evidence Rad51 binds Rad52 protein <br /> of DNA double-strand repair-specific recombinosome,           (26). The sensitivity pso4-1 mutant 8-MOP + UVA, its <br /> complex containing proteins encoded RAD51, RAD52,          partial complementation overexpressed RecA protein the <br /> RAD55 RAD57, recently suggested (26). We suggest          epistatic interactions pso4-1 mutant rad51 rad52 <br /> that Prp19/Pso4-associated protein complex Tarn           mutant alleles (17) lead hypothesis functions in <br /> and co-workers (29) above-mentioned recombino-            repair primary secondary induced DNA strand breaks via <br /> some.                                                                  recombinational mechanism. The association functional <br />    The fact existing viable alleles essential gene    protein complex, forming recombinosome (26), <br /> PSO4/PRP19, represented previously described prp19           function Pso4/Prp19 protein; other, more <br /> mutant (22,34) pso4-1 mutant respectively, exhibit        essential yeast cell, role spliceosome- <br /> temperature sensitivity implies 503 amino acid protein        associated protein pre-mRNA processing. Recently, similar <br /> encoded PSO4/PRP19 functional                two-way association described yeast transcription <br /> domains, active protein     factor TFIIH: participates formation holocomplex active <br /> expressed haploid diploid pso4-1 mutants:        RNA polymerase II transcription initiation be <br /> no obvious growth retardation 30 C pso4-1 haploid            repairosome, specific nucleotide excision <br /> homoallelic diploid mutants assume RNA splicing           repair (37,38). <br /> proceeds efficiency close identical the <br /> wild-type. This hypothesis underscored fact         ACKNOWLEDGEMENTS <br /> viable prp19 mutant accumulates unprocessed pre-mRNA the <br /> non-permissive permissive temperature (34),             We thank Dr A.C.Schenberg pso4-1 mutant Drs <br /> suggesting viability pso4/prp19 mutant dependent       J.Brozmanova W.Siede helpful discussion. Sequencing of <br /> functional splicing. Survival pso4-1 mutant pso4-0     left arm chromosome XII supported grant AD <br /> therefore suggests mutant allele expresses     European Union BIOTECH program coor- <br /> partially functional protein 30 C; role spliceosomal   dination Joerg Hoheisel, DKFZ, Heidelberg. Research was <br /> assembly function normal function            supported Qualit&#228;tsweine D.Brendel. The Brazilian&#8211;German <br /> repair DNA single- double-strand breaks recombinational     cooperation facilitated CNPq-DAAD travel grants. <br /> processes significantly perturbed. We suggest these <br /> phenotypes disturbed binding proteins <br /> to Pso4 multienzyme complex Tarn et al. (29),          REFERENCES <br /> indicating hitherto <span id='am-58' about='obo:NCBITaxon_32644' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-59'  ></span><span id='am-60' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasRelatedSynonym" content="none" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasRelatedSynonym" content="unclassified sequence" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasRelatedSynonym" content="other" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="unidentified organism" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasRelatedSynonym" content="unspecified" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasRelatedSynonym" content="unknown" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasRelatedSynonym" content="not shown" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasRelatedSynonym" content="unknown organism" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="not specified" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="unidentified" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="unknwon" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasRelatedSynonym" content="miscellaneous nucleic acid" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasRelatedSynonym" content="sonstige nucleic acid" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>unidentified</span> proteins    1 Haynes,R.H. Kunz,B. (1981) In: Strathern,J.N., Jones,E.W. and <br /> may belong RAD52 epistasis group.                                  Broach,J.R (eds), The Molecular Biology Yeast Saccharomyces <br />    An alternative, opinion highly unlikely, explanation        cerevisiae. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, <br /> for repair phenotype pso4-1 mutant                pp. 341&#8211;414. <br />                                                                         2 Friedberg,E.C., Walker,G.C. Siede,W. (1995) DNA Repair and <br /> incorrect processing transcripts DNA repair genes. Amongst          Mutagenesis. ASM Press, Washington, DC. <br /> the repair genes yeast one, RAD14,            3 Brendel,M., Khan,N.A. Haynes,R.H. (1970) Mol. Gen. Genet., 106, <br /> is involved incision damaged DNA, intron-containing              289&#8211;295. <br />  4014 Nucleic Acids Research, 1996, Vol. 24, No. 20 <br />  <br />  4   Game,J.C. Cox,B.S. (1971) Mutat. Res., 12, 328&#8211;331.                       23 Ju,Q., Morrow,B.E. Warner,J.R. (1990) Mol. Cell. Biol., 10, <br />  5   Game,J.C. Cox,B.S. (1972) Mutat. Res., 16, 353&#8211;362.                          5226&#8211;5234. <br />  6   Brendel,M. Haynes,R.H. (1973) Mol. Gen. Genet., 125, 197&#8211;216.             24 Tice-Baldwin,K., Fink,G.R. Arndt,K.T. (1989) Science, 246, 931&#8211;935. <br />  7   Game,J.C. Cox,B.S. (1973) Mutat. Res., 20, 35&#8211;44.                         25 Firmenich,A.A., Elias-Arnanz,M. Berg,P. (1995) Mol. Cell. Biol., 15, <br />  8   Henriques,J.A.P. Brendel,M. (1990) Curr. Genet., 18, 387&#8211;393.                1620&#8211;1631. <br />  9   Henriques,J.A.P. Moustacchi,E. (1980) Genetics, 95, 273&#8211;288.              26 Hays,S.L., Firmenich,A.A. Berg,P. (1995) Proc. Natl. Acad. Sci. USA, <br /> 10   Henriques,J.A.P., Vicente,E.J., daSilva,K.V.C.L. Schenberg,A.C.G.            92, 6925&#8211;6929. <br />      (1989) Mutat. Res., 218, 111&#8211;124.                                             27 daSilva,K.V.C.L, Morais,M.A.,Jr, Henriques,J.A.P. (1995) Curr. <br /> 11   Andrade,H.H.R., Marques,E.K., Schenberg,A.C.G. Henriques,J.A.P.              Genet., 27, 207&#8211;212. <br />      (1989) Mol. Gen. Genet., 217, 419&#8211;426.                                        28 Tarn,W.-Y., Lee,K.R. Cheng,S.-C. (1993) Mol. Cell Biol., 13, <br /> 12   Meira,L.B., Fonseca,M.B., Averbeck,D., Schenberg,A.C.G.                      1883&#8211;1891. <br />      Henriques,J.A.P. (1992) Mol. Gen. Genet., 235, 311&#8211;316.                       29 Tarn,W.Y., Hsu, C.H., Huang,K.T., Chen,H.R., Kao,H.Y., Lee,K.R. and <br /> 13   Benathen,I.A. Beam,C.A. (1977) Radiat. Res., 69, 99&#8211;116.                     Cheng,S.C. (1994) EMBO J., 13, 2421&#8211;2431. <br />                                                                                    30 Tarn,W.Y., Lee,K.R. Cheng,S.C. (1993) Proc. Natl. Acad. Sci. USA, <br /> 14   Morais,M.A.,Jr, Vicente,E.J., Brozmanov&#225;,J., Schenberg,A.C.G. and <br />                                                                                       90, 10821&#8211;10825. <br />      Henriques,J.A.P. (1996) Curr. Genet., 29, 211&#8211;218. <br />                                                                                    31 Jachymczyk,W.J., von Borstel,R.C., Mowat,M.R.C. Hastings,P.J. <br /> 15   Brozmanov&#225;,J., Cern&#225;kov&#225;,L., Vlckov&#225;,V., DurajJ. Fridrichov&#225;,I. <br />                                                                                       (1981) Mol. Gen. Genet., 182, 196&#8211;205. <br />      (1991) Mol. Gen. Genet., 227, 473&#8211;480.                                        32 Maga&#241;a-Schwencke,N., Henriques,J.A.P.,Chanet,P. Mousacchi,E. <br /> 16   Cernakov&#225;,L., Fridrichov&#225;,I., Pirsel,M., Kleibl,K., Duraj,J.                 (1982) Proc. Natl. Acad. Sci. USA, 79,1722&#8211;1726. <br />      Brozmanov&#225;,J. (1991) Biochimie, 73, 285&#8211;288.                                  33 Bardwell,A.J., Bardwell,L., Iyer,N., Svefstrup,J.Q., Feaver,R.D., <br /> 17   Morais,M.A.,Jr, Brozmanov&#225;,J., Benfato,M., Duraj,J., Vlckov&#225;,V.              Kornberg,R.D. Friedberg,E.C. (1994) Mol. Cell. Biol., 14, 3569&#8211;3576. <br />      Henriques,J.A.P. (1994) Mutat. Res., 314, 209&#8211;220.                            34 Vijayraghavan,U., Company,M. Abelson,J. ( <br /> </body></html>