<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>   Functional Annotation of <span id='am-283' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-284' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-285' property="rdfs:label" content="class" datatype="xsd:string"></span>Class</span> I Lysyl-tRNA Synthetase Phylogeny <br />               Indicates a Limited Role for Gene Transfer <br />  <br />             Functional and comparative genomic studies have previously shown that the essential protein lysyl-tRNA <br />          synthetase (LysRS) exists in two unrelated forms. Most prokaryotes and all eukaryotes contain a class II <br />          LysRS, whereas most <span id='am-196' about='obo:NCBITaxon_2157' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567'><span id='am-197' property="oboInOwl:hasDbXref" content="PMID:10425796" datatype="xsd:string"></span><span id='am-198' property="oboInOwl:hasDbXref" content="PMID:10425795" datatype="xsd:string"></span><span id='am-199' property="oboInOwl:hasDbXref" content="PMID:270744" datatype="xsd:string"></span><span id='am-200' property="rdfs:label" content="Archaea" datatype="xsd:string"></span><span id='am-201'  ></span><span id='am-202' property="oboInOwl:hasRelatedSynonym" content="Monera" datatype="xsd:string"></span><span id='am-203' property="oboInOwl:hasRelatedSynonym" content="Metabacteria" datatype="xsd:string"></span><span id='am-204' property="oboInOwl:hasDbXref" content="PMID:8590690" datatype="xsd:string"></span><span id='am-205' property="oboInOwl:hasDbXref" content="PMID:11760965" datatype="xsd:string"></span><span id='am-206' property="oboInOwl:hasDbXref" content="PMID:11321083" datatype="xsd:string"></span><span id='am-207' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-208' property="oboInOwl:hasDbXref" content="PMID:2112744" datatype="xsd:string"></span><span id='am-209' property="oboInOwl:hasDbXref" content="PMID:10425797" datatype="xsd:string"></span><span id='am-210' property="oboInOwl:hasDbXref" content="PMID:10843050" datatype="xsd:string"></span><span id='am-211' property="oboInOwl:hasRelatedSynonym" content="Procaryotae" datatype="xsd:string"></span><span id='am-212' property="oboInOwl:hasRelatedSynonym" content="archaea" datatype="xsd:string"></span><span id='am-213' property="oboInOwl:hasDbXref" content="PMID:11540071" datatype="xsd:string"></span><span id='am-214' property="oboInOwl:hasDbXref" content="PMID:11211268" datatype="xsd:string"></span><span id='am-215' property="oboInOwl:hasRelatedSynonym" content="Prokaryotae" datatype="xsd:string"></span><span id='am-216' property="oboInOwl:hasDbXref" content="PMID:10490293" datatype="xsd:string"></span><span id='am-217' property="oboInOwl:hasRelatedSynonym" content="Archaebacteria" datatype="xsd:string"></span><span id='am-218' property="oboInOwl:hasDbXref" content="PMID:8123559" datatype="xsd:string"></span><span id='am-219' property="oboInOwl:hasDbXref" content="PMID:10939673" datatype="xsd:string"></span><span id='am-220' property="oboInOwl:hasDbXref" content="PMID:10939677" datatype="xsd:string"></span><span id='am-221' property="oboInOwl:hasDbXref" content="PMID:12054223" datatype="xsd:string"></span><span id='am-222' property="oboInOwl:hasDbXref" content="PMID:11321113" datatype="xsd:string"></span><span id='am-223' property="oboInOwl:hasRelatedSynonym" content="Mendosicutes" datatype="xsd:string"></span><span id='am-224' property="oboInOwl:hasRelatedSynonym" content="prokaryote" datatype="xsd:string"></span><span id='am-225' property="oboInOwl:hasDbXref" content="PMID:9103655" datatype="xsd:string"></span><span id='am-226' property="oboInOwl:hasDbXref" content="PMID:11541975" datatype="xsd:string"></span><span id='am-227' property="oboInOwl:hasRelatedSynonym" content="prokaryotes" datatype="xsd:string"></span><span id='am-228' property="oboInOwl:hasDbXref" content="PMID:11542149" datatype="xsd:string"></span><span id='am-229' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-230' property="oboInOwl:hasDbXref" content="PMID:11542064" datatype="xsd:string"></span><span id='am-231' property="oboInOwl:hasDbXref" content="PMID:9336922" datatype="xsd:string"></span><span id='am-232' property="oboInOwl:hasDbXref" content="PMID:11411719" datatype="xsd:string"></span><span id='am-233' property="oboInOwl:hasDbXref" content="PMID:10939651" datatype="xsd:string"></span><span id='am-234' property="oboInOwl:hasRelatedSynonym" content="Prokaryota" datatype="xsd:string"></span>archaea</span> and a few bacteria contain a less common class I LysRS. In bacteria the class <br />          I LysRS is only found in the &#9251;-<span id='am-114' about='obo:NCBITaxon_1224' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-115' property="oboInOwl:hasExactSynonym" content="purple photosynthetic bacteria" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasDbXref" content="PMID:16280474" datatype="xsd:string"></span><span id='am-117'  ></span><span id='am-118' property="rdfs:label" content="Proteobacteria" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasDbXref" content="PMID:11542017" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasExactSynonym" content="purple bacteria" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasDbXref" content="PMID:11321122" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasDbXref" content="PMID:11837318" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasExactSynonym" content="purple photosynthetic bacteria and relatives" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasExactSynonym" content="purple bacteria and relatives" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasRelatedSynonym" content="proteobacteria" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasExactSynonym" content="purple non-sulfur bacteria" datatype="xsd:string"></span>proteobacteria</span> and a scattering of other groups, including the spirochetes, while <br />          the class I protein is by far the most common form of LysRS in archaea. To investigate <span id='am-295' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-296' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-297' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-298'  ></span><span id='am-299' property="rdfs:label" content="This" datatype="xsd:string"></span>this</span> unusual <br />          distribution we functionally annotated a representative phylogenetic sampling of LysRS proteins. Class I <br />          LysRS proteins from a variety of bacteria and archaea were characterized in vitro by their ability to recognize <br />          <span id='am-437' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-438' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-439' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-440'  ></span><span id='am-441' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-442' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-443' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-444' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-445' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-446' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-447' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-448' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-449' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-450' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-451' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-452' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span>Escherichia coli</span> tRNALys anticodon mutants. Class I LysRS proteins were found to fall into two distinct groups, <br />          those that preferentially recognize the third anticodon nucleotide of tRNALys (U36) and those that recognize <br />          both the second and third positions (U35 and U36). Strong recognition of U35 and U36 was confined to the <br />          <span id='am-167' about='obo:NCBITaxon_2260' typeof='owl:Thing obo:NCBITaxon_183968 obo:NCBITaxon_1 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_2259 obo:NCBITaxon_2258 obo:NCBITaxon_28890'><span id='am-168'  ></span><span id='am-169' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-170' property="rdfs:label" content="Pyrococcus" datatype="xsd:string"></span><span id='am-171' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>pyrococcus</span>-spirochete grouping within the archaeal branch of the class I LysRS phylogenetic tree, while U36 <br />          recognition was seen in other archaea and an example from the &#9251;-proteobacteria. Together with the corre- <br />          sponding phylogenetic relationships, these results suggest that despite its comparative rarity the distribution <br />          of class I LysRS conforms to the canonical archaeal-bacterial division. The only exception, suggested from both <br />          functional and phylogenetic <span id='am-511' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-512' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-513' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-514' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-515'  ></span>data</span>, appears to be the horizontal transfer of class I LysRS from a pyrococcal <br />          progenitor to a limited number of bacteria. <br />  <br />  <br />    Aminoacyl-tRNA (<span id='am-505' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-506' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-507' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-508' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-509' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-510'  ></span>aa</span>-tRNA) synthesis is the process                        comparable indirect pathways exist for the production of Asn- <br /> whereby an amino acid is esterified to the 3&#11032; end of a tRNA                 tRNAAsn and selenocysteinyl-tRNASec (reviewed in reference <br /> containing the corresponding anticodon sequence (15). After                 11). While the use of these pathways was initially assumed to <br /> their synthesis, aa-tRNAs are screened and selected by elon-                be idiosyncratic to a small minority of organisms, recent anal- <br /> gation factor Tu (16) and then taken to the ribosomal decoding              yses of completed microbial genome sequences has shown <br /> site, where they are paired with the complementary mRNA                     them to be widespread in prokaryotes with the corresponding <br /> codons (19). Consequently, the availability of a cellular pool of           direct pathways absent. GlnRS is absent from all characterized <br /> correctly aminoacylated tRNAs is essential for the faithful                 archaea and the majority of bacteria, while asparaginyl-tRNA <br /> translation of the genetic information encoded in mRNA into                 synthetase (AsnRS) is absent from most archaea and a signif- <br /> the corresponding polypeptide sequence (14). Aminoacyl-                     icant number of bacteria (31). Initial analyses of the complete <br /> tRNAs are primarily made by the aminoacyl-tRNA synthetase                   genome sequences of the archaea <span id='am-188' about='obo:NCBITaxon_2184' typeof='owl:Thing obo:NCBITaxon_183939 obo:NCBITaxon_2183 obo:NCBITaxon_1 obo:NCBITaxon_2182 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_28890'><span id='am-189' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-190'  ></span><span id='am-191' property="oboInOwl:hasDbXref" content="PMID:15653928" datatype="xsd:string"></span><span id='am-192' property="oboInOwl:hasDbXref" content="PMID:8782682" datatype="xsd:string"></span><span id='am-193' property="oboInOwl:hasDbXref" content="PMID:10319480" datatype="xsd:string"></span><span id='am-194' property="rdfs:label" content="Methanococcus" datatype="xsd:string"></span><span id='am-195' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Methanococcus</span> jannaschii <br /> (aaRS) protein <span id='am-280' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-281' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-282' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> (9). The accuracy of aa-tRNA synthesis                (3) and <span id='am-182' about='obo:NCBITaxon_2160' typeof='owl:Thing obo:NCBITaxon_2158 obo:NCBITaxon_2159 obo:NCBITaxon_1 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_183925 obo:NCBITaxon_28890'><span id='am-183' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-184' property="rdfs:label" content="Methanobacterium" datatype="xsd:string"></span><span id='am-185' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-186'  ></span><span id='am-187' property="oboInOwl:hasDbXref" content="PMID:15653928" datatype="xsd:string"></span>Methanobacterium</span> thermoautotrophicum (24) sug- <br /> by the aaRSs is ensured by their extremely high substrate                   gested that in addition to AsnRS and GlnRS, genes encoding <br /> specificity, which is further enhanced in some cases by the                 the cysteinyl-tRNA synthetase (CysRS) and lysyl-tRNA syn- <br /> existence of proofreading mechanisms. The 20 aaRS proteins,                 thetase (LysRS) might also be absent from some organisms. <br /> as for example found in <span id='am-453' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-454' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-455' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-456' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-457' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-458'  ></span><span id='am-459' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span>Escherichia</span> coli (6), are divided into              Experimental studies showed that while both CysRS (17, 26) <br /> two mutually exclusive structural groups of 10 members each,                and LysRS (13) are present in archaea, they are encoded by <br /> termed class I and class II (reviewed in references 1 and 5).               novel or reassigned genes, making the original annotation of <br /> The assignment of an aaRS specific for a particular amino acid              the corresponding open reading frames difficult. In the case of <br /> to one of the two structural classes is almost completely con-              LysRS experimental approaches identified a gene encoding a <br /> served, reflecting the ancient evolutionary origin of this family           class I aaRS (lysK) in the archaeon <span id='am-516' about='obo:NCBITaxon_39152' typeof='owl:Thing obo:NCBITaxon_183939 obo:NCBITaxon_2183 obo:NCBITaxon_2184 obo:NCBITaxon_1 obo:NCBITaxon_2182 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_28890'><span id='am-517' property="oboInOwl:hasDbXref" content="PMID:8782682" datatype="xsd:string"></span><span id='am-518' property="oboInOwl:hasAlternativeId" content="NCBITaxon:43442" datatype="xsd:string"></span><span id='am-519' property="rdfs:label" content="Methanococcus maripaludis" datatype="xsd:string"></span><span id='am-520' property="oboInOwl:hasRelatedSynonym" content="Methanosarcina maripaludis" datatype="xsd:string"></span><span id='am-521' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-522'  ></span><span id='am-523' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-524' property="oboInOwl:hasRelatedSynonym" content="Methanococcus deltae" datatype="xsd:string"></span>Methanococcus maripaludis</span>, <br /> of proteins (21).                                                           in contrast to all previously characterized LysRS proteins that <br />    The 20 canonical aaRS proteins do not represent the sole                 belonged to class II. Comparative genomic analyses showed <br /> means of cellular aa-tRNA synthesis. It was first observed in               class II LysRS-encoding genes (lysS) to be absent from the <br /> <span id='am-129' about='obo:NCBITaxon_1404' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-130' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-131' property="rdfs:label" content="Bacillus megaterium" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1191421" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasRelatedSynonym" content="Bacillus fructosus" datatype="xsd:string"></span><span id='am-135'  ></span><span id='am-136' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1219390" datatype="xsd:string"></span>Bacillus megaterium</span> that Gln-tRNAGln could be synthesized                   majority of archaea and some bacteria, with lysK being found <br /> via Glu-tRNAGln without the use of glutaminyl-tRNA syn-                     instead in all cases (e.g., see reference 10). Functional (12) and <br /> thetase (GlnRS) (32), and subsequent studies showed that                    structural (29) characterizations have shown that the class I <br />                                                                             and II LysRS proteins are functionally equivalent but structur- <br />                                                                             ally unrelated. <br />   * Corresponding author. Mailing address: Department of Microbi- <br /> ology, The Ohio State University, 484 West 12th Ave., Columbus, OH <br />                                                                                Primarily as a result of the sequencing of numerous micro- <br /> 43210-1292. Phone: (614) 292-2120. Fax: (614) 292-8120. E-mail:             bial genomes, over 30 lysK gene sequences are now known. The <br /> ibba.1@osu.edu.                                                             distribution of organisms containing lysK, which in all but the <br />  <br />                                                                      4594 <br />  VOL. 184, 2002                                                                             EVOLUTION OF LYSYL-tRNA SYNTHETASES                                 4595 <br />  <br />                                                                                    nas aromaticivorans cells were a gift from Margaret Romine (Pacific Northwest <br />                                                                                    National Lab, Richland, Wash.) and <span id='am-313' about='obo:NCBITaxon_74968' typeof='owl:Thing obo:NCBITaxon_183967 obo:NCBITaxon_1 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_2301 obo:NCBITaxon_28890 obo:NCBITaxon_90142'><span id='am-314' property="oboInOwl:hasRelatedSynonym" content="Ferriplasma" datatype="xsd:string"></span><span id='am-315' property="oboInOwl:hasDbXref" content="PMID:10843038" datatype="xsd:string"></span><span id='am-316' property="rdfs:label" content="Ferroplasma" datatype="xsd:string"></span><span id='am-317' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-318'  ></span><span id='am-319' property="oboInOwl:hasDbXref" content="PMID:18175672" datatype="xsd:string"></span><span id='am-320' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-321' property="oboInOwl:hasDbXref" content="PMID:16721487" datatype="xsd:string"></span>Ferroplasma</span> acidarmanus genomic DNA <br />                                                                                    was a gift from Paul F. Predki (DOE Joint Genome Institute, Walnut Creek, <br />                                                                                    Calif.). S. aromaticivorans cells were grown in 50% <span id='am-177' about='obo:NCBITaxon_218032' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_69557 obo:NCBITaxon_69555 obo:NCBITaxon_69556 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-178' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-179'  ></span><span id='am-180' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-181' property="rdfs:label" content="Luria" datatype="xsd:string"></span>Luria</span> broth for 36 h at 30&#176;C <br />                                                                                    and genomic DNA was then prepared according to standard procedures (23). <br />                                                                                       Cloning of lysK and lysS genes. LysRS encoding genes were amplified from <br />                                                                                    genomic DNAs as indicated under standard PCR conditions using the fol- <br />                                                                                    lowing primers (engineered restriction enzyme recognition sites are indicated <br />                                                                                    and underlined for each primer): for S. aromaticivorans lysK, Forward: (NdeI) <br />                                                                                    5&#11032;-GCATATGACAGACGAGATCCGCACC, and Reverse: (BamHI) 5&#11032;-GG <br />                                                                                    ATCCTCAGTCGAGCGCCTCGGC; for M. barkeri lysK, Forward: (XhoI) <br />                                                                                    5&#11032;-GCTCGAGATGGCTGACACAATTCACTGG, and Reverse: (BlpI) 5&#11032;-G <br />                                                                                    CTGAGCTTATGCCTGTTTTTCGGGGC; for M. barkeri lysS, Forward: (Nd <br />                                                                                    eI) 5&#11032;-GCATATGAGCATGGAAATTAACAATGAAAAAATGTC, and Re- <br />                                                                                    verse: (BamHI) 5&#11032;-GGATCCTCAGTCTTCCCTTTTCATCTGTGG; for F. <br />                                                                                    acidarmanus lysK, Forward: (XhoI): 5&#11032;-GCTCGAGATGTACTGGGCAGAT <br />                                                                                    GCACTG, and Reverse: (BamHI): 5&#11032;-GGATCCTTACTGTGCAGAAT- <br />                                                                                    TATTCTGGATAAG; for M. magnetotacticum lysK, Forward: (NdeI) 5&#11032;- <br />                                                                                    GCATATGTCCTCCGATTCCCGC, and Reverse: (BamHI) 5&#11032;-GGATCCC <br />                                                                                    TACATATCCTCGCCCGC). <br />                                                                                       The amplified fragments were cloned into Topo TA (Invitrogen) vector and <br />                                                                                    subsequently subcloned into expression vector pET15b (Invitrogen) using the <br />                                                                                    restriction sites introduced by PCR. Clones were sequenced prior to transfor- <br />                                                                                    mation into E. coli BL21 C&#11001; strain. This procedure allowed the overproduction <br />                                                                                    of N-terminally His6-tagged derivatives, which have previously been found not to <br />                                                                                    perturb substrate recognition by LysRS (14). <br />                                                                                       Overexpression and purification of LysRS proteins. Transformants were <br />                                                                                    grown in Luria broth supplemented with ampicillin (100 &#9262;g/ml) and chloram- <br />                                                                                    phenicol (34 &#9262;g/ml) at 37&#176;C to an optical density at 600 nm of 0.6. The expression <br />                                                                                    of the recombinant protein was then induced by IPTG (isopropyl-&#9252;-D-thiogalac- <br />                                                                                    topyranoside) (final concentration of 1 mM) for 3 h at 37&#176;C. The cells were <br />                                                                                    harvested by centrifugation and washed with phosphate-buffered saline; this and <br />                                                                                    subsequent steps were performed at 4&#176;C. The cell paste was resuspended into <br />                                                                                    lysis buffer (50 mM Na2H2PO4, 300 mM NaCl, 5 mM 2-mercaptoethanol, pro- <br />                                                                                    tease inhibitor [Roche]) and sonicated for 10 cycles of 30 s. The cell extract was <br />    FIG. 1. Secondary structure of undermodified E. coli tRNALys. Ab-               obtained by ultracentrifugation at 100,000 &#11003; g for 1 h and subsequently applied <br /> breviations: D, dihydrouridine; &#9274;, pseudouridine; m7G, 7-methyl-                   to an Ni-nitrilotriacetic acid agarose column (Qiagen), according to the standard <br /> guanosine; X, putative 3-(3-amino-3-carboxypropyl) uridine modifica-               procedure. The protein was eluted with an elution buffer (containing 750 mM <br /> tion. The undermodified variants used in this study contain uridine in             imidazole), dialyzed overnight against storage buffer (100 mM Tris-HCl [pH 8.0], <br /> place of 5-methylaminomethyl-2-thiouridine at position 34 and aden-                50 mM KCl, 25 mM MgCl2, 5 mM 2-mercaptoethanol, 50% glycerol), and then <br /> osine in place of N6-threonylcarbamoyladenosine at position 37.                    stored at &#11002;20&#176;C. The purity of all the recombinant enzyme preparations was <br /> Changes in the anticodon nucleotides (nucleotides 34 to 36) are indi-              judged to be 95% or greater by sodium dodecyl sulfate-polyacrylamide gel <br /> cated.                                                                             electrophoresis followed by staining with Coomassie brilliant blue. <br />                                                                                       Cloning and in vitro transcription of tRNALys genes. tRNA genes for in vitro <br />                                                                                    T7 RNA polymerase transcription were cloned by annealing of two cDNA <br /> archaeal <span id='am-235' about='obo:NCBITaxon_2207' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_2206 obo:NCBITaxon_94695 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_224756 obo:NCBITaxon_28890'><span id='am-236' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-237'  ></span><span id='am-238' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-239' property="rdfs:label" content="Methanosarcina" datatype="xsd:string"></span><span id='am-240' property="oboInOwl:hasDbXref" content="PMID:11536640" datatype="xsd:string"></span>Methanosarcina</span> group is to the exclusion of lysS, en-                     oligonucleotides and direct ligation into pUC18 plasmid using BamHI and Hin- <br />                                                                                    dIII restriction sites. The clones were transformed into E. coli DH5&#9251;, sequenced, <br /> compasses most of the archaea and a scattering of bacteria but <br />                                                                                    and amplified. The 3&#11032; end of the transcription template was digested overnight <br /> no eukaryotes. Previous studies have attributed this unusual                       either with BstNI at 60&#176;C or NsiI at 37&#176;C. In vitro T7 RNA polymerase runoff <br /> evolutionary pattern to either horizontal gene transfer (12, 33,                   transcription was conducted according to standard procedure (18). tRNALys <br /> 34) or selective retention from the last common ancestor in                        transcripts were purified on a denaturing 12% polyacrylamide gel and recovered <br /> particular lineages (22). To further investigate the evolution of                  by electroelution (Elutrap; Schleicher and Schuell), followed by desalting on a <br />                                                                                    Nap-5 Column (Pharmacia Biotech). Plasmid T7-911 for preparation of the <br /> the lysK gene family, we have now extended our previous                            His6-tagged recombinant T7 RNA polymerase was a gift from T. Shrader (De- <br /> preliminary studies and attempted to differentiate a range of                      partment of Biochemistry, Albert Einstein College of Medicine, New York, <br /> class I LysRS proteins based upon their mode of recognition of                     N.Y.). Nucleoside triphosphates were from Sigma. <br /> tRNALys. Comparison of these results with a revised phyloge-                          Genomic and phylogenetic analyses. All predicted amino acid sequences were <br />                                                                                    obtained from publicly available databases. Sequences were initially aligned <br /> netic tree of class I LysRS sequences suggests that selective <br />                                                                                    using the program CLUSTALX (30). The alignment was then used to generate <br /> retention was the major factor in the evolution of this protein                    a neighbor-joining phylogeny using the PHYLIP 3.57c package (distributed by <br /> family with horizontal gene transfer playing a relatively minor                    the author [J. Felsenstein]; Department of Genetics, University of Washington, <br /> role.                                                                              Seattle) with parameters previously used for aaRS phylogenies (20). Maximum- <br />                                                                                    likelihood phylogeny was generated using the program Tree-Puzzle 5.0 (www <br />                                                                                    .tree-puzzle.de) (28) (10,000 puzzle steps, JTT substitution matrix). <br />                        MATERIALS AND METHODS <br />    General. In vivo overproduced E. coli tRNALys variants, in which the antico- <br /> don loop nucleotides 34 and 37 have previously been shown to be unmodified <br />                                                                                                                      RESULTS <br /> (Fig. 1) (4), were prepared as described before (4), as was <span id='am-415' about='obo:NCBITaxon_139' typeof='owl:Thing obo:NCBITaxon_203691 obo:NCBITaxon_1 obo:NCBITaxon_64895 obo:NCBITaxon_203692 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_137 obo:NCBITaxon_136 obo:NCBITaxon_138'><span id='am-416' property="oboInOwl:hasDbXref" content="PMID:9336916" datatype="xsd:string"></span><span id='am-417' property="oboInOwl:hasDbXref" content="PMID:8995796" datatype="xsd:string"></span><span id='am-418' property="oboInOwl:hasDbXref" content="PMID:1380285" datatype="xsd:string"></span><span id='am-419' property="oboInOwl:hasDbXref" content="PMID:1503971" datatype="xsd:string"></span><span id='am-420' property="oboInOwl:hasExactSynonym" content="Borrelia burgdorferi sensu stricto" datatype="xsd:string"></span><span id='am-421' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-422' property="oboInOwl:hasExactSynonym" content="Lyme disease spirochete" datatype="xsd:string"></span><span id='am-423' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-424' property="oboInOwl:hasDbXref" content="PMID:7699027" datatype="xsd:string"></span><span id='am-425' property="oboInOwl:hasDbXref" content="PMID:7981102" datatype="xsd:string"></span><span id='am-426' property="oboInOwl:hasRelatedSynonym" content="Borrelia burdorferi" datatype="xsd:string"></span><span id='am-427' property="oboInOwl:hasRelatedSynonym" content="Borrelia burgdorffragment" datatype="xsd:string"></span><span id='am-428'  ></span><span id='am-429' property="rdfs:label" content="Borrelia burgdorferi" datatype="xsd:string"></span>Borrelia burgdorferi</span> <br /> class I LysRS protein (10). Aminoacylation assays were performed as previously       Aminoacylation activity and tRNA specificity of recombi- <br /> described (12). <span id='am-241' about='obo:NCBITaxon_2208' typeof='owl:Thing obo:NCBITaxon_2207 obo:NCBITaxon_1 obo:NCBITaxon_2206 obo:NCBITaxon_94695 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_224756 obo:NCBITaxon_28890'><span id='am-242' property="oboInOwl:hasRelatedSynonym" content="Methanobacterium barkeri" datatype="xsd:string"></span><span id='am-243' property="rdfs:label" content="Methanosarcina barkeri" datatype="xsd:string"></span><span id='am-244' property="oboInOwl:hasRelatedSynonym" content="Methanosarcina barkerii" datatype="xsd:string"></span><span id='am-245' property="oboInOwl:hasRelatedSynonym" content="Sarcina barkeri" datatype="xsd:string"></span><span id='am-246' property="oboInOwl:hasDbXref" content="PMID:1537550" datatype="xsd:string"></span><span id='am-247' property="oboInOwl:hasAlternativeId" content="NCBITaxon:29289" datatype="xsd:string"></span><span id='am-248' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-249'  ></span><span id='am-250' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Methanosarcina barkeri</span> fusaro genomic DNA was a gift from <br /> Kevin Sowers (University of Maryland Biotechnology Institute, Baltimore), Mag- <br />                                                                                    nant LysRS proteins. The recombinant LysRS proteins were <br /> netospirillum magnetotacticum genomic DNA was a gift from Elizabeth Bertani        tested for in vitro activity as determined by their ability to <br /> (Division of Biology, <span id='am-21' about='obo:NCBITaxon_337343' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_41943 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_91836 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4027 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="California Aldasoro et al." datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="California" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-26'  ></span>California</span> Institute of Technology, Pasadena), Sphingomo-    aminoacylate transcripts of their own (cognate) tRNALys genes <br />  4596        AMBROGELLY ET AL.                                                                                                        J. BACTERIOL. <br />  <br />  <br />       TABLE 1. In vitro lysylation of tRNALys substrates by LysRS                 tRNA (Fig. 2C), while mutation of U35 to C did not result in <br />                                 proteins                                          any change in activity (Fig. 2D). The only significant reduction <br />                                        Formation of Lys-tRNALyse                  in activity with F. acidarmanus LysRS was observed with <br /> LysRSa protein source                                                             tRNALys where U36 was replaced with C, a change that virtu- <br />                          Cognate                                   E. coli <br />                                        Noncognate transcriptc                     ally eliminated recognition. <br />                         transcriptb                             tRNALysUUUd <br />                                                                                      Anticodon recognition by M. barkeri class II lysyl-tRNA syn- <br /> F. acidarmanus             &#11002;          &#11002;                               &#11001; <br />                                                                                   thetase. Recognition of the tRNALys anticodon by the class II <br /> M. magnetotacticum         &#11002;          &#11002;                               &#11001; <br /> M. barkeri                 &#11002;          &#11002;                               &#11002;           LysRS of M. barkeri was investigated as described above for <br /> M. barkeri class II        &#11002;          &#11002;                               &#11001;           class I LysRS proteins. Mutation of the nucleotide U34 to G <br /> S. aromaticivorans         &#11001;          &#11001; (M. magnetotacticum)          &#11001;           (Fig. 3A) led to a significant reduction in recognition by M. <br /> B. burgdorferi             &#11001;f         &#11001; (F. acidarmanus)              &#11001;f          barkeri class II LysRS, whereas the replacement of U36 with C <br />   a <br />      All LysRS proteins are class I except as indicated for M. barkeri.           (Fig. 3B) caused only a slight change, indicating the relative <br />   b <br />      In vitro-transcribed tRNALys genes (UUU anticodon sequence) derived          importance of these positions. The strongest effects on amino- <br /> from the same organism as the LysRS. <br />    c <br />      In vitro-transcribed tRNALys genes (UUU anticodon sequence) derived from     acylation were observed for changes in U35 of tRNALys, the <br /> an organism different from that from which the LysRS was derived, as indicated.   G35 and C35 variants both showing significant reductions in <br />    d <br />      Synthesized in vivo; see text for details.                                   activity compared to wild-type (Fig. 3B). <br />    e <br />      Lysylation of tRNA was performed at 37&#176;C for 15 min with 1 &#9262;M enzyme, <br /> 100 &#9262;M lysine, and 4 &#9262;M tRNA. Symbols: &#11002;, no detectable formation of                 Molecular phylogenies of class I LysRS. Phylogenetic anal- <br /> Lys-tRNALys compared to an enzyme-free control; &#11001;, detectable formation of        yses of the class I LysRS protein family were performed using <br /> Lys-tRNALys. <br />    f <br />      Shown for comparison, taken from reference 13. <br />                                                                                   28 of the 31 sequences currently available in public databases. <br />                                                                                   The three excluded sequences are all from <span id='am-430' about='obo:NCBITaxon_138' typeof='owl:Thing obo:NCBITaxon_203691 obo:NCBITaxon_1 obo:NCBITaxon_203692 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_137 obo:NCBITaxon_136'><span id='am-431' property="oboInOwl:hasDbXref" content="PMID:8863416" datatype="xsd:string"></span><span id='am-432' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-433' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-434'  ></span><span id='am-435' property="rdfs:label" content="Borrelia" datatype="xsd:string"></span><span id='am-436' property="oboInOwl:hasDbXref" content="PMID:8863409" datatype="xsd:string"></span>Borrelia</span> genospe- <br />                                                                                   cies and show more than 75% amino acid identity to B. burg- <br />                                                                                   dorferi (N. Mejlhede, A. Montha&#180;n, M. Thesien, and M. Ibba, <br /> or other noncognate tRNALys <span id='am-310' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-311' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-312' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span> (Table 1). Of the pro-                        submitted for publication). As extensive tRNA anticodon rec- <br /> teins tested, only the class I LysRS of S. aromaticivorans was                    ognition data are available for the class I LysRS of B. burgdor- <br /> able to aminoacylate its own in vitro-transcribed tRNALys. To                     feri (25), this sequence was chosen for inclusion in this study. <br /> investigate whether lack of aminoacylation in the other cases                     Structural studies have revealed a high degree of similarity <br /> resulted from either inactive transcripts and/or inactive en-                     between the two <span id='am-13' about='obo:NCBITaxon_subclass' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-14' property="rdfs:label" content="subclass" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>subclass</span> Ib aaRSs LysRS and glutamyl-tRNA <br /> zymes, cross-charging experiments were performed. B. burg-                        synthetase (GluRS) (29), suggesting that GluRS sequences <br /> dorferi class I LysRS, which has previously been shown to                         may be appropriate for the rooting of LysRS phylogenetic <br /> aminoacylate a variety of tRNALys species, was found to ami-                      trees. We therefore generated separate sequence alignments <br /> noacylate in vitro-transcribed F. acidarmanus tRNALys, indi-                      using CLUSTALX of 28 class I LysRS and 11 GluRS proteins <br /> cating that this tRNA was active. In addition, F. acidarmanus                     (chosen as a representative sampling based upon existing phy- <br /> LysRS was able to charge E. coli tRNALys, indicating that the                     logenies [33]) and then manually merged these into a single <br /> enzyme was active. Similar tests with the two M. barkeri LysRS                    alignment using a structure-based sequence alignment of Py- <br /> proteins and in vitro-transcribed M. barkeri tRNALys showed                       rococcus horikoshii LysRS and <span id='am-352' about='obo:NCBITaxon_274' typeof='owl:Thing obo:NCBITaxon_188786 obo:NCBITaxon_188787 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_68933 obo:NCBITaxon_270 obo:NCBITaxon_1297'><span id='am-353' property="rdfs:label" content="Thermus thermophilus" datatype="xsd:string"></span><span id='am-354' property="oboInOwl:hasRelatedSynonym" content="Thermus aquaticus (SUBSP. FLAVUS)" datatype="xsd:string"></span><span id='am-355' property="oboInOwl:hasDbXref" content="PMID:4334342" datatype="xsd:string"></span><span id='am-356' property="oboInOwl:hasRelatedSynonym" content="Thermus aquaticus flavus" datatype="xsd:string"></span><span id='am-357' property="oboInOwl:hasRelatedSynonym" content="Thermua flavus" datatype="xsd:string"></span><span id='am-358' property="oboInOwl:hasRelatedSynonym" content="Thermus aquaticus thermophilus" datatype="xsd:string"></span><span id='am-359' property="oboInOwl:hasRelatedSynonym" content="Thermus themophilus" datatype="xsd:string"></span><span id='am-360' property="oboInOwl:hasRelatedSynonym" content="Thermus aquaticus (SUBSP. THERMOPHILUS)" datatype="xsd:string"></span><span id='am-361' property="oboInOwl:hasRelatedSynonym" content="Thermus flavus" datatype="xsd:string"></span><span id='am-362' property="oboInOwl:hasRelatedSynonym" content="Thermus aquaticus subsp. thermophilus" datatype="xsd:string"></span><span id='am-363' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-364' property="oboInOwl:hasAlternativeId" content="NCBITaxon:273" datatype="xsd:string"></span><span id='am-365' property="oboInOwl:hasDbXref" content="PMID:11594628" datatype="xsd:string"></span><span id='am-366' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-367' property="oboInOwl:hasDbXref" content="PMID:8590676" datatype="xsd:string"></span><span id='am-368'  ></span>Thermus thermophilus</span> GluRS <br /> the class I LysRS and tRNA to be inactive but the class II                        (29). Positions containing gaps in any of the sequences were <br /> enzyme to be active. Investigation of M. magnetotacticum                          then removed, leaving a total of 287 aligned residues present in <br /> showed the class I LysRS to be active, albeit at a significantly                  all sequences. This sequence alignment was then used to gen- <br /> lower level than the other active proteins tested, and revealed                   erate neighbor-joining and maximum-likelihood phylogenies. <br /> that in vitro-transcribed M. magnetotacticum tRNALys could be                     In both cases the GluRS sequences formed a single distinct <br /> charged by S. aromaticivorans LysRS.                                              outgroup with high confidence for the corresponding node <br />    Anticodon recognition by class I LysRSs. Anticodon recog-                      (84.3% by neighbor joining, 93% by maximum likelihood). A <br /> nition was investigated using undermodified E. coli tRNALys                       maximum-likelihood tree rooted by GluRS was then generated <br /> variants generated in vivo, which have previously been shown                      for the class I LysRS sequences (Fig. 4). The overall class I <br /> to be efficient substrates for a variety of both class I and class                LysRS phylogeny resembles a somewhat-reduced version of <br /> II LysRSs (4, 25). Initial tests using M. magnetotacticum LysRS                   the universal phylogenetic tree with distinct bacterial and ar- <br /> showed that while it was able to charge the wild-type tRNA                        chaeal groupings. The only exceptions are the placement of the <br /> and nucleotide 34 mutant (UUU and GUU anticodons, re-                             crenarchaeon <span id='am-483' about='obo:NCBITaxon_46769' typeof='obo:NCBITaxon_651137 owl:Thing obo:NCBITaxon_205957 obo:NCBITaxon_205948 obo:NCBITaxon_1 obo:NCBITaxon_2157 obo:NCBITaxon_131567'><span id='am-484' property="oboInOwl:hasRelatedSynonym" content="Candidatus Cenarchaeum" datatype="xsd:string"></span><span id='am-485' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-486' property="oboInOwl:hasDbXref" content="PMID:8692799" datatype="xsd:string"></span><span id='am-487' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-488'  ></span><span id='am-489' property="rdfs:label" content="Cenarchaeum" datatype="xsd:string"></span>Cenarchaeum</span> symbiosum within the major bac- <br /> spectively), the level of activity for the other mutants was                      terial grouping and a subgroup of several bacteria close to the <br /> insufficient to detect charging above background. Since the M.                    pyrococcal proteins in the archaeal group. <br /> barkeri class I LysRS had been shown to be inactive, the pat- <br /> tern of anticodon recognition was only investigated in more <br />                                                                                                           DISCUSSION <br /> detail for the F. acidarmanus and S. aromaticivorans proteins <br /> (Fig. 2). Mutation of U34 to G in tRNALys was found to have                          Comparative genomics of tRNALys recognition. While all of <br /> no effect on recognition by S. aromaticivorans LysRS (Fig. 2A),                   the LysRSs, with the exception of the class I protein from M. <br /> changes at U35 resulted in reduced charging and the most-                         barkeri, were found to be active when heterologously pro- <br /> pronounced effect was observed for replacement of U36, which                      duced, only one was found to recognize its cognate in vitro- <br /> almost completely eliminated activity (Fig. 2B). With F. aci-                     transcribed tRNA substrate. Previous studies showed that the <br /> darmanus LysRS replacements of either U34 or U35 by G both                        B. burgdorferi (12) and P. horikoshii (29) class I LysRSs both <br /> led to improvements in recognition compared to wild-type                          recognized cognate in vitro-transcribed tRNALys, while the M. <br />  VOL. 184, 2002                                                                  EVOLUTION OF LYSYL-tRNA SYNTHETASES                     4597 <br />  <br />  <br />  <br />  <br />    FIG. 2. Aminoacylation of E. coli tRNALys variants by class I His6-LysRS proteins. Aminoacylation reactions were performed as described in <br /> the text (20-&#9262;l samples) in the presence of 4 &#9262;M tRNA and 100 &#9262;M [14C]lysine. Comparison of charging of wild-type and mutant tRNALys using <br /> class I LysRS enzymes from S. aromaticivorans (200 nm of enzyme) (A and B) and F. acidarmanus (2 &#9262;M enzyme) (C and D). <br />  <br />  <br />  <br /> maripaludis enzyme did not (12). These differences in tRNA               vitro and those that did not with respect to potential nucleotide <br /> gene transcript recognition may result from the absence of               modification sites. This was done by comparing known posi- <br /> essential RNA modifications necessary for recognition or fold-           tions of nucleotide modifications described for tRNALys from <br /> ing, as reported for human mitochondrial tRNALys (8). We                 E. coli (Fig. 1) and <span id='am-153' about='obo:NCBITaxon_2251' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_183963 obo:NCBITaxon_2236 obo:NCBITaxon_28890 obo:NCBITaxon_2235'><span id='am-154' property="rdfs:label" content="Haloferax" datatype="xsd:string"></span><span id='am-155' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-156'  ></span><span id='am-157' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-158' property="oboInOwl:hasDbXref" content="PMID:19244452" datatype="xsd:string"></span>Haloferax</span> volcanii (7) with positions con- <br /> attempted to correlate sequence differences between tRNALys              served in either the &#8220;active&#8221; or &#8220;inactive&#8221; tRNALys gene sets, <br /> genes (UUU anticodon) that gave rise to active transcripts in            but not in both (Fig. 5). These comparisons identified a notable <br />  <br />  <br />  <br />  <br />    FIG. 3. Aminoacylation of E. coli tRNALys variants by M. barkeri class II His6-LysRS. Aminoacylation reactions were performed as described <br /> in the text (20-&#9262;l samples) in the presence of 4 &#9262;M tRNA, 100 &#9262;M [14C]lysine, and 2 &#9262;M enzyme. (A) Position 34 variants; (B) position 35 and <br /> 36 variants. <br />  4598     AMBROGELLY ET AL.                                                                                                  J. BACTERIOL. <br />  <br />  <br />                                                                          during recognition by the class I LysRS from S. aromati- <br />                                                                          civorans, while only U36 was found to be important for the F. <br />                                                                          acidarmanus enzyme. This predominant role for U36 in <br />                                                                          tRNALys recognition was also observed for the class I LysRS of <br />                                                                          M. maripaludis but differs from the B. burgdorferi (25) and P. <br />                                                                          horikoshii (29) enzymes, for which both U35 and U36 were <br />                                                                          found to be critical recognition elements. Taken together with <br />                                                                          previous studies, the data presented here suggest that the class <br />                                                                          I LysRSs can be divided into two broad groups defined by <br />                                                                          whether either U36, or U35 and U36 are the preferred anti- <br />                                                                          <span id='am-305' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-306' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-307' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-308' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-309'  ></span>codon</span> nucleotides for recognition. This functional divergence <br />                                                                          among the class I LysRSs can also be correlated with the <br />                                                                          overall phylogeny of this family (Fig. 4, discussed below), in <br />                                                                          contrast to the class II LysRS family where the pattern of <br />                                                                          anticodon recognition is more highly conserved. <br />                                                                             Phylogenetic conservation of anticodon recognition by class <br />                                                                          II LysRSs. Previous studies on the activity of tRNALys antico- <br />                                                                          don variants with bacterial (E. coli [4] and <span id='am-328' about='obo:NCBITaxon_210' typeof='owl:Thing obo:NCBITaxon_72293 obo:NCBITaxon_213849 obo:NCBITaxon_209 obo:NCBITaxon_1 obo:NCBITaxon_68525 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_29547'><span id='am-329' property="oboInOwl:hasRelatedSynonym" content="Campylobacter pylori" datatype="xsd:string"></span><span id='am-330' property="oboInOwl:hasRelatedSynonym" content="Campylobacter pyloridis" datatype="xsd:string"></span><span id='am-331' property="oboInOwl:hasDbXref" content="PMID:8186097" datatype="xsd:string"></span><span id='am-332' property="oboInOwl:hasDbXref" content="PMID:1995031" datatype="xsd:string"></span><span id='am-333' property="oboInOwl:hasDbXref" content="PMID:11931154" datatype="xsd:string"></span><span id='am-334' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-335' property="oboInOwl:hasAlternativeId" content="NCBITaxon:219" datatype="xsd:string"></span><span id='am-336' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-337' property="rdfs:label" content="Helicobacter pylori" datatype="xsd:string"></span><span id='am-338'  ></span><span id='am-339' property="oboInOwl:hasRelatedSynonym" content="Campylobacter pylori subsp. pylori" datatype="xsd:string"></span><span id='am-340' property="oboInOwl:hasRelatedSynonym" content="Helicobacter nemestrinae" datatype="xsd:string"></span><span id='am-341' property="oboInOwl:hasDbXref" content="PMID:8494747" datatype="xsd:string"></span>Helicobacter pylori</span> <br />                                                                          [25]) class II LysRSs have shown that all three positions of the <br />                                                                          anticodon are recognized, with U35 being the predominant <br />                                                                          recognition site. A similar pattern was also observed during <br />                                                                          studies of the human class II LysRS (27), with U35 and U36 <br />                                                                          both recognized and the former being the main recognition <br />                                                                          element. The data presented here (Fig. 3) indicate that an <br />                                                                          archaeal class II LysRS recognizes the anticodon of tRNALys <br />                                                                          in the same way as its bacterial and eukaryotic counterparts do, <br />                                                                          with U35 being the most important position. This suggests that, <br />                                                                          in contrast to their class I counterparts, class II LysRSs share <br />                                                                          a pattern of tRNA anticodon recognition which is conserved in <br />                                                                          the three primary domains of the universal phylogenetic tree. <br />                                                                          This difference in the evolutionary divergence of a function <br />                                                                          duplicated in the two LysRS classes supports earlier evidence <br />   FIG. 4. Maximum-likelihood phylogenetic tree of class I LysRS          (12) suggesting that these two families are functionally but not <br /> sequences. The tree has been rooted using GluRS sequences. Numbers       mechanistically equivalent. <br /> indicate the percentage occurrence of nodes after 10,000 puzzle steps.      Evolution of the class I LysRS family. The phylogenetic tree <br />                                                                          for class I LysRS sequences contains two major branches <br />                                                                          broadly corresponding to the bacteria and the archaea (Fig. 4). <br /> difference at position 26, which is found as N2,N2-dimethyl-             The inclusion of GluRS sequences places the <span id='am-300' about='obo:NCBITaxon_1' typeof='owl:Thing'><span id='am-301' property="oboInOwl:hasRelatedSynonym" content="all" datatype="xsd:string"></span><span id='am-302' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-303' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-304' property="rdfs:label" content="root" datatype="xsd:string"></span>root</span> of the tree <br /> guanosine in H. volcanii and as adenosine in E. coli. All                between the bacterial and archaeal branches, indicating that <br /> tRNALys genes whose transcripts are recognized by their cog-             the evolutionary pattern of class I LysRS sequences is in agree- <br /> nate class I LysRSs contain A26, with the exception of M.                ment with that of the universal phylogenetic tree derived from <br /> magnetotacticum, while those that are not recognized contain             rRNA sequences (see, for example, reference 2). While the <br /> G26. While the possible role of a modified G26 in tRNALys                class I LysRS tree appears canonical it is somewhat restricted, <br /> folding and/or recognition now requires a more extensive cat-            lacking any known eukaryotic examples and being mainly lim- <br /> aloguing of in vivo RNA modification patterns, the strong                ited to &#9251;-proteobacteria in the bacterial branch and euryarcha- <br /> correlation between experimental and comparative genomic                 eota in the archaeal branch. This observation of a limited <br /> approaches provides a testable hypothesis for the observed               canonical phylogeny among the class I LysRSs suggests that <br /> differences in the activity of in vitro transcribed tRNALys              selective retention of the lysK gene in certain lineages has <br /> genes. This is further supported by the observation that the             contributed substantially to their present distribution, as pre- <br /> distribution of A versus G at position 26 broadly correlates             viously suggested (22). <br /> with the molecular phylogeny of class I LysRSs (Fig. 4 and 5).              Closer examination of the class I LysRS phylogeny, however, <br />    Evolutionary divergence of anticodon recognition among                suggests that limited gene transfer has also contributed to the <br /> class I LysRSs. The data presented above (Fig. 2) support                phylogenetic distribution of class I LysRSs. One possible ex- <br /> earlier proposals (e.g., those in reference 25) that the antico-         ception to the canonical universal phylogeny among the class I <br /> don is a crucial site in tRNALys for recognition by class I              LysRSs is seen for the C. symbiosum protein, which is posi- <br /> LysRSs. In agreement with studies using class I LysRSs from              tioned within the main bacterial branch. Earlier analyses based <br /> the <span id='am-16' about='obo:NCBITaxon_336400' typeof='owl:Thing obo:NCBITaxon_49928 obo:NCBITaxon_1 obo:NCBITaxon_2323 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-17' property="rdfs:label" content="bacterium B" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-20'  ></span>bacterium B</span>. burgdorferi and the archaeon M. maripaludis,            upon smaller data sets placed C. symbiosum as a branch pre- <br /> the first nucleotide of the anticodon (position 34) was not              ceding the &#9251;-proteobacteria (33, 34), possibly indicating hori- <br /> found to play a significant role during recognition. Nucleotides         zontal transfer from an ancestral crenarchaeote. While our <br /> U35 and especially U36 were both shown to be important                   present analysis suggests horizontal transfer from rather than <br />  VOL. 184, 2002                                                                      EVOLUTION OF LYSYL-tRNA SYNTHETASES                                4599 <br />  <br />  <br />  <br />  <br />    FIG. 5. Alignment of tRNALys sequence (UUU anticodon) from organisms encoding class I LysRSs. Nucleotide 26 is indicated and shown in <br /> boldface type, and the anticodon sequence is underlined. Abbreviations: Rpa, <span id='am-64' about='obo:NCBITaxon_1076' typeof='owl:Thing obo:NCBITaxon_356 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_41294 obo:NCBITaxon_28211 obo:NCBITaxon_1073'><span id='am-65' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1077" datatype="xsd:string"></span><span id='am-66'  ></span><span id='am-67' property="oboInOwl:hasRelatedSynonym" content="Rhodovibrio parvus" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasRelatedSynonym" content="Rhodobacillus palustris" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="Rhodopseudomonas palustris" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="Rhodopseudomonas rutila" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="PMID:21986724" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasRelatedSynonym" content="Rhodomonas palustris" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasRelatedSynonym" content="Rhodobacter palustris" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasRelatedSynonym" content="Rhodobacterium capsulatum" datatype="xsd:string"></span>Rhodopseudomonas palustris</span>; CC, <span id='am-476' about='obo:NCBITaxon_75' typeof='owl:Thing obo:NCBITaxon_204458 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_28211 obo:NCBITaxon_76892'><span id='am-477' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-478' property="rdfs:label" content="Caulobacter" datatype="xsd:string"></span><span id='am-479' property="oboInOwl:hasDbXref" content="PMID:10319468" datatype="xsd:string"></span><span id='am-480' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-481' property="oboInOwl:hasDbXref" content="PMID:10425763" datatype="xsd:string"></span><span id='am-482'  ></span>Caulobacter</span> crescentus; MMa, M. <br /> magnetotacticum; RP, R. prowazekii; Wol, <span id='am-460' about='obo:NCBITaxon_953' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_28211 obo:NCBITaxon_766 obo:NCBITaxon_952 obo:NCBITaxon_942'><span id='am-461' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-462' property="oboInOwl:hasDbXref" content="PMID:17684235" datatype="xsd:string"></span><span id='am-463' property="oboInOwl:hasRelatedSynonym" content="Wolbachia Hertig 1936" datatype="xsd:string"></span><span id='am-464'  ></span><span id='am-465' property="oboInOwl:hasDbXref" content="PMID:17329802" datatype="xsd:string"></span><span id='am-466' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-467' property="rdfs:label" content="Wolbachia" datatype="xsd:string"></span>Wolbachia</span> sp.; SA, S. aromaticivorans; ML, <span id='am-322' about='obo:NCBITaxon_381' typeof='owl:Thing obo:NCBITaxon_356 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_68287 obo:NCBITaxon_28211 obo:NCBITaxon_69277'><span id='am-323' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-324' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-325' property="rdfs:label" content="Mesorhizobium loti" datatype="xsd:string"></span><span id='am-326' property="oboInOwl:hasRelatedSynonym" content="Rhizobium loti" datatype="xsd:string"></span><span id='am-327'  ></span>Mesorhizobium loti</span>; RS, <span id='am-89' about='obo:NCBITaxon_1063' typeof='obo:NCBITaxon_204455 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_1060 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_31989 obo:NCBITaxon_28211'><span id='am-90' property="oboInOwl:hasRelatedSynonym" content="ATCC 55304" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasRelatedSynonym" content="Rhodorhagus minor" datatype="xsd:string"></span><span id='am-94'  ></span><span id='am-95' property="oboInOwl:hasRelatedSynonym" content="Rhodococcus capsulatus" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasRelatedSynonym" content="Rhodosphaera minor" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasRelatedSynonym" content="Rhodosphaera capsulata" datatype="xsd:string"></span><span id='am-98' property="rdfs:label" content="Rhodobacter sphaeroides" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasDbXref" content="PMID:16350090" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasRelatedSynonym" content="ATCC BAA-808" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasRelatedSynonym" content="Rhodopseudomonas spheroides" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasDbXref" content="PMID:16558776" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasRelatedSynonym" content="Rhodococcus minor" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasRelatedSynonym" content="Rhodorhagus capsulatus" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasRelatedSynonym" content="Rhodopseudomonas sphaeroides" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasRelatedSynonym" content="Rhodorrhagus spheroides" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasRelatedSynonym" content="Rhodorrhagus capsulatus" datatype="xsd:string"></span>Rhodobacter sphaeroides</span>; PF, <br /> <span id='am-172' about='obo:NCBITaxon_2261' typeof='owl:Thing obo:NCBITaxon_183968 obo:NCBITaxon_2260 obo:NCBITaxon_1 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_2259 obo:NCBITaxon_2258 obo:NCBITaxon_28890'><span id='am-173'  ></span><span id='am-174' property="rdfs:label" content="Pyrococcus furiosus" datatype="xsd:string"></span><span id='am-175' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-176' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Pyrococcus furiosus</span>; PA, Pyrococcus abysii; PH, <span id='am-286' about='obo:NCBITaxon_53953' typeof='owl:Thing obo:NCBITaxon_183968 obo:NCBITaxon_2260 obo:NCBITaxon_1 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_2259 obo:NCBITaxon_2258 obo:NCBITaxon_28890'><span id='am-287' property="oboInOwl:hasRelatedSynonym" content="Pyrococcus shinkaii" datatype="xsd:string"></span><span id='am-288' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-289' property="rdfs:label" content="Pyrococcus horikoshii" datatype="xsd:string"></span><span id='am-290' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-291'  ></span><span id='am-292' property="oboInOwl:hasAlternativeId" content="NCBITaxon:37789" datatype="xsd:string"></span><span id='am-293' property="oboInOwl:hasRelatedSynonym" content="&apos;Pyrococcus shinkaii&apos;" datatype="xsd:string"></span><span id='am-294' property="oboInOwl:hasDbXref" content="PMID:9672687" datatype="xsd:string"></span>Pyrococcus horikoshii</span>; BB, B. burgdorferi; SC, <span id='am-27' about='obo:NCBITaxon_1883' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_2062 obo:NCBITaxon_201174 obo:NCBITaxon_85003 obo:NCBITaxon_85011'><span id='am-28'  ></span><span id='am-29' property="oboInOwl:hasDbXref" content="PMID:8590678" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasAlternativeId" content="NCBITaxon:65506" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="PMID:1371059" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="PMID:11411701" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="PMID:10843062" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="PMID:9731302" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="PMID:16560709" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Chainia" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="PMID:9336904" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1973" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="PMID:8934906" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="PMID:15950131" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="PMID:4886644" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasAlternativeId" content="NCBITaxon:31945" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Streptomyces" datatype="xsd:string"></span>Streptomyces</span> coelicolor; TD, <span id='am-385' about='obo:NCBITaxon_158' typeof='owl:Thing obo:NCBITaxon_203691 obo:NCBITaxon_1 obo:NCBITaxon_203692 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_137 obo:NCBITaxon_136 obo:NCBITaxon_157'><span id='am-386' property="oboInOwl:hasRelatedSynonym" content="Spirochaete denticola" datatype="xsd:string"></span><span id='am-387' property="oboInOwl:hasDbXref" content="PMID:8494734" datatype="xsd:string"></span><span id='am-388' property="oboInOwl:hasRelatedSynonym" content="Treponema orthodontum" datatype="xsd:string"></span><span id='am-389' property="oboInOwl:hasRelatedSynonym" content="Spirochaeta orthodonta" datatype="xsd:string"></span><span id='am-390' property="oboInOwl:hasRelatedSynonym" content="Treponema microdentium" datatype="xsd:string"></span><span id='am-391' property="oboInOwl:hasRelatedSynonym" content="Spirochaeta dentium" datatype="xsd:string"></span><span id='am-392' property="oboInOwl:hasRelatedSynonym" content="Treponema dentium-stenogyratum" datatype="xsd:string"></span><span id='am-393'  ></span><span id='am-394' property="oboInOwl:hasRelatedSynonym" content="Treponema dentium" datatype="xsd:string"></span><span id='am-395' property="oboInOwl:hasRelatedSynonym" content="Treponema comandonii" datatype="xsd:string"></span><span id='am-396' property="oboInOwl:hasRelatedSynonym" content="Spirochaete dentium" datatype="xsd:string"></span><span id='am-397' property="oboInOwl:hasRelatedSynonym" content="Treponema ambiguum" datatype="xsd:string"></span><span id='am-398' property="oboInOwl:hasRelatedSynonym" content="Spirochaeta microdentium" datatype="xsd:string"></span><span id='am-399' property="oboInOwl:hasRelatedSynonym" content="Spirochaeta ambigua" datatype="xsd:string"></span><span id='am-400' property="oboInOwl:hasRelatedSynonym" content="Spirochaeta comandonii" datatype="xsd:string"></span><span id='am-401' property="rdfs:label" content="Treponema denticola" datatype="xsd:string"></span><span id='am-402' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-403' property="oboInOwl:hasRelatedSynonym" content="Spirillum dentium" datatype="xsd:string"></span><span id='am-404' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-405' property="oboInOwl:hasRelatedSynonym" content="Spironema dentium" datatype="xsd:string"></span>Treponema denticola</span>; TP, <br /> <span id='am-376' about='obo:NCBITaxon_160' typeof='owl:Thing obo:NCBITaxon_203691 obo:NCBITaxon_1 obo:NCBITaxon_203692 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_137 obo:NCBITaxon_136 obo:NCBITaxon_157'><span id='am-377' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-378' property="rdfs:label" content="Treponema pallidum" datatype="xsd:string"></span><span id='am-379' property="oboInOwl:hasRelatedSynonym" content="Microspironema pallidum" datatype="xsd:string"></span><span id='am-380' property="oboInOwl:hasRelatedSynonym" content="Spirochaeta pallida" datatype="xsd:string"></span><span id='am-381'  ></span><span id='am-382' property="oboInOwl:hasRelatedSynonym" content="Spironema pallidum" datatype="xsd:string"></span><span id='am-383' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-384' property="oboInOwl:hasRelatedSynonym" content="Spirillum pallidum" datatype="xsd:string"></span>Treponema pallidum</span>; HB, <span id='am-251' about='obo:NCBITaxon_2239' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_183963 obo:NCBITaxon_2236 obo:NCBITaxon_28890 obo:NCBITaxon_2235'><span id='am-252' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-253'  ></span><span id='am-254' property="oboInOwl:hasDbXref" content="PMID:15653928" datatype="xsd:string"></span><span id='am-255' property="oboInOwl:hasRelatedSynonym" content="Haloarchaeum" datatype="xsd:string"></span><span id='am-256' property="oboInOwl:hasRelatedSynonym" content="Flavobacterium (subgen. Halobacterium)" datatype="xsd:string"></span><span id='am-257' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-258' property="oboInOwl:hasDbXref" content="PMID:18485204" datatype="xsd:string"></span><span id='am-259' property="oboInOwl:hasDbXref" content="PMID:19244452" datatype="xsd:string"></span><span id='am-260' property="oboInOwl:hasDbXref" content="PMID:18768637" datatype="xsd:string"></span><span id='am-261' property="rdfs:label" content="Halobacterium" datatype="xsd:string"></span><span id='am-262' property="oboInOwl:hasRelatedSynonym" content="Halobacter" datatype="xsd:string"></span>Halobacterium</span> sp.; AP, <span id='am-1' about='obo:NCBITaxon_56636' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28889 obo:NCBITaxon_114380 obo:NCBITaxon_2157 obo:NCBITaxon_2272 obo:NCBITaxon_131567 obo:NCBITaxon_56635 obo:NCBITaxon_183924'><span id='am-2' property="rdfs:label" content="Aeropyrum pernix" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="PMID:8863437" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-5'  ></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Aeropyrum pernix</span>; MT, M. thermoautotrophicum; MJ, M. jannaschii; MM, M. maripaludis; AF, <br /> <span id='am-263' about='obo:NCBITaxon_2234' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_2233 obo:NCBITaxon_2232 obo:NCBITaxon_183980 obo:NCBITaxon_2231 obo:NCBITaxon_28890'><span id='am-264'  ></span><span id='am-265' property="oboInOwl:hasDbXref" content="PMID:11540072" datatype="xsd:string"></span><span id='am-266' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-267' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-268' property="rdfs:label" content="Archaeoglobus fulgidus" datatype="xsd:string"></span>Archaeoglobus fulgidus</span>; MB, M. barkeri; FA, F. acidarmanus; TV, <span id='am-146' about='obo:NCBITaxon_2302' typeof='owl:Thing obo:NCBITaxon_183967 obo:NCBITaxon_1 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_2301 obo:NCBITaxon_46659 obo:NCBITaxon_28890'><span id='am-147' property="oboInOwl:hasRelatedSynonym" content="Thermoplasma Darland et al. 1970" datatype="xsd:string"></span><span id='am-148'  ></span><span id='am-149' property="oboInOwl:hasDbXref" content="PMID:5481857" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-151' property="rdfs:label" content="Thermoplasma" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Thermoplasma</span> volcanium; TA, <span id='am-137' about='obo:NCBITaxon_2303' typeof='owl:Thing obo:NCBITaxon_183967 obo:NCBITaxon_1 obo:NCBITaxon_2157 obo:NCBITaxon_131567 obo:NCBITaxon_2302 obo:NCBITaxon_2301 obo:NCBITaxon_46659 obo:NCBITaxon_28890'><span id='am-138'  ></span><span id='am-139' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasRelatedSynonym" content="Thermoplasma acidophila" datatype="xsd:string"></span><span id='am-141' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasRelatedSynonym" content="&quot;Thermoplasma acidophila&quot; (sic) Darland et al. 1970" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasRelatedSynonym" content="Thermoplasma acidophilum Darland et al. 1970" datatype="xsd:string"></span><span id='am-144' property="oboInOwl:hasDbXref" content="PMID:5481857" datatype="xsd:string"></span><span id='am-145' property="rdfs:label" content="Thermoplasma acidophilum" datatype="xsd:string"></span>Thermoplasma acidophilum</span>. Sequences of genes <br /> encoding tRNALys (UUU) were obtained from The Institute for Genomic Research (http://www.tigr.org), the Sanger Center (http://www.sanger <br /> .ac.uk/Projects/), and public databases (http://www.ncbi.nlm.nih.gov/cgi-bin/Entrez/genom_table_cgi). <br />  <br />  <br />  <br />  <br /> to the &#9251;-proteobacteria, a more reliable interpretation of the                Parts of this work were supported by funds from The Ohio State <br /> exact positioning of the C. symbiosum protein within the class              University (M.I.) and the National Institutes of Health (GM-22854 to <br />                                                                             Dieter So <br />                                                                                     &#168;ll). <br /> I LysRS tree, and of possible gene transfer events, is now <br /> dependent on the availability of other closely related archaeal                                              <br /> sequences. Within the archaeal branch of the class I LysRS <br />                                                                              <br /> tree, the group containing Pyrococcus species also contains                    <br /> several bacterial examples of LysRS. Functional analyses have              <br />                                                                             <br /> shown that bacterial (B. burgdorferi) and archaeal (P. horiko-                <br /> shii) enzymes from within this group share common tRNALys                     <br /> anticodon recognition properties (see above) not seen for class               <br />                                                                           <br /> I LysRSs from elsewhere in the phylogenetic tree, supporting a            <br /> possible common origin. The positioning of this group within              <br /> the canonical archaeal branch suggests that the corresponding              <br />                                                                            <br /> bacterial LysRSs (Borrelia, <span id='am-406' about='obo:NCBITaxon_157' typeof='owl:Thing obo:NCBITaxon_203691 obo:NCBITaxon_1 obo:NCBITaxon_203692 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_137 obo:NCBITaxon_136'><span id='am-407' property="rdfs:label" content="Treponema" datatype="xsd:string"></span><span id='am-408' property="oboInOwl:hasDbXref" content="PMID:9019153" datatype="xsd:string"></span><span id='am-409' property="oboInOwl:hasRelatedSynonym" content="Microspironema" datatype="xsd:string"></span><span id='am-410' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-411' property="oboInOwl:hasDbXref" content="PMID:23961314" datatype="xsd:string"></span><span id='am-412'  ></span><span id='am-413' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-414' property="oboInOwl:hasDbXref" content="PMID:9734025" datatype="xsd:string"></span>Treponema</span>, and Streptomyces) orig-               <br /> inated from a gene transfer event from the pyrococcal progen-         <br />                                                                          <br /> itor into an ancestral bacterium. Thus, while selective retention   <br /> appears to have been the main determinant of the distribution               <br /> of class I LysRSs, horizontal gene transfer has also played a <br />                                                                          <br /> significant but less widespread role.                                       <br />   <br /> </body></html>