<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#63193; 2004 Genetics Society America <br /> DOI: 10.1534/genetics.104.032888 <br />  <br />  <br />  <br />          Genetic Analysis Yeast Yip1p Function Reveals Requirement for <br />                    Golgi-Localized Rab Proteins Rab-Guanine <br />                            Nucleotide Dissociation Inhibitor <br />  <br />               Catherine Z. Chen,* Monica Calero,* Carol J. DeRegis,* Matthew Heidtman,&#8224; <br />                                 Charles Barlowe&#8224; Ruth N. Collins*,1 <br />           *Department Molecular Medicine, Cornell University, Ithaca, New York 14853-6401 &#8224;Department Biochemistry, <br />                                      Dartmouth Medical School, Hanover, New Hampshire 03755 <br />                                                          Manuscript received June 27, 2004 <br />                                                       Accepted publication August 19, 2004 <br />  <br />  <br />                                                               ABSTRACT <br />                Yip1p identified Rab-interacting membrane protein founder member YIP1 <br />              <span id='am-48' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-49' property="rdfs:label" content="family" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>family</span>, orthologs paralogs eukaryotic genomes. The exact role Yip1p is <br />              unclear; YIP1 essential gene defective alleles severely disrupt membrane transport inhibit <br />              ER vesicle budding. Yip1p ability physically interact Rab proteins nature <span id='am-82' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-83'  ></span><span id='am-84' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-85' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>this</span> <br />              interaction led suggestions Yip1p function process Rab proteins translocate <br />              cytosol membranes. In study investigated physiological requirements for <br />              Yip1p action. Yip1p function requires Rab-GDI Rab proteins, mutations abrogate <br />              Yip1p function lack Rab-interacting capability. We previously shown Yip1p detergent extracts <br />              capability physically interact Rab proteins promiscuous manner; however, genetic <br />              analysis covers yeast Rab reveals Rab requirement vivo exclusively confined a <br />              subset Rab proteins localized Golgi apparatus. <br />  <br />  <br />  <br /> R    AS <span id='am-102' about='obo:NCBITaxon_superfamily' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-103' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-104' property="rdfs:label" content="superfamily" datatype="xsd:string"></span>superfamily</span> members proven critical <br />       players variety fundamental cellular pro- <br /> cesses, influence modulation GTP <br />                                                                             proteins manner dependent COOH-terminal <br />                                                                             prenylation (Calero et al. 2002) compete <br />                                                                             Rab-GDI Rab protein interactions vivo. <br /> binding hydrolysis cycle. These small GTPases                          YIP1 essential gene <span id='am-41' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-42' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-45'  ></span><span id='am-46' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span>Saccharomyces</span> cerevsiae and <br /> commonly cycle cytosolic pool                      extremely conserved evolution; human or- <br /> membrane-associated pool activated GTPases                        tholog fully replace loss yeast gene (Calero <br /> recruit downstream effectors. The highly hydro-                       et al. 2003). In cellular lysates, Yip1p capable promis- <br /> phobic geranylgeranyl groups Rho Rab GTPases                         cuous interaction Rab proteins, specific re- <br /> render energetically unfavorable partition                     quirement double prenylation motif contained <br /> the cytosol individual monomers. Cytosolic Rho                       COOH termini defining features <br /> Rab proteins exist high-affinity dimeric complex                       Rab protein family. Recent studies combining <br /> with guanine nucleotide dissociation inhibitor (GDI)                        vitro reconstitution cell biological genetic <br /> proteins (Olofsson 1999; Alory Balch 2001).                             analyses demonstrated Yip1p functions an <br /> These GDI.GTPase complexes represent inactive cy-                        early stage ER vesicle budding (Barrowman et al. <br /> tosolic reservoir GTPase GDI in-                         2003; Heidtman et al. 2003). This function Yip1p is <br /> duced release GTPase membrane site enable                    difficult reconcile connection Rab proteins. <br /> cells draw reservoir. Mechanisms pro-                     Although Rab proteins required vesicle budding <br /> mote GDI displacement help recruit GTPases                         cargo selection systems, possible to <br /> membranes great importance mech-                        form fusion-competent transport vesicles ER <br /> anisms determine GTPases                        absence Ypt1p function, Rab protein <br /> activated. Possible candidates role Rab protein                    thought regulate particular transport step vivo <br /> membrane recruitment include Rab-interacting                            (Cao Barlowe 2000). Moreover, known <br /> YIP1 family membrane proteins. Certainly YIP1 <br />                                                                             Yip1p action vivo requires Rab proteins, leaving <br /> family possess features suggest capability par- <br />                                                                             open question apparent significance bio- <br /> ticipation Rab membrane recruitment; capa- <br />                                                                             chemical interactions observed proteins. <br /> ble biochemical interaction variety Rab <br />                                                                                In study sought understand two <br />                                                                             aspects Yip1p function determine interac- <br />  1 <br />   Corresponding author: Department Molecular Medicine, Cornell           tion Yip1p Rab proteins biologically significant. <br /> University, Ithaca, NY 14853-6401. E-mail: rnc8@cornell.edu                 We report physiological requirement for <br /> Genetics 168: 1827&#8211;1841 (December 2004) <br />  1828                                                        C. Z. Chen et al. <br />  <br />                                                                 TABLE 1 <br />                                                  S. cerevisiae strains used study <br />  <br /> Strain                                                   Genotype                                                        Source <br /> BY24            MATa/&#65533;?&#65533; ura3-52/ura3-52 leu2-3,112/leu2-3,112                                                     Brennwald laboratory <br /> RCY883          MAT&#65533;?&#65533; ura3-52 leu2-3,112 his3&#9004;200 GDI1&#9004;HISMX6 [pRS316 GDI1]                                       This study <br /> RCY1610         MAT&#65533;?&#65533; ura3-52 leu2-3,112 YIP1&#9004;KAN R [YCP50 (pRC1245) YIP1]                                        Calero et al. (2003) <br /> RCY1612         MATa ura3-52 leu2-3,112 YIP1&#9004;KAN R [pRC1245]                                                     This study <br /> RCY1633         MATa ura3-52 leu2-3,112 his3&#9004;200 YIP1&#9004;KAN R [pRC1908]                                            This study <br /> RCY1634         MAT&#65533;?&#65533; ura3-52 leu2-3,112 his3&#9004;200 YIP1&#9004;KAN R [pRC1908]                                            This study <br /> RCY1718         MATa ura3-52 leu2-3,112 his3&#9004;200 GDI1&#9004;HISMX6 YIP1&#9004;KAN R [pRC1908]                                This study <br /> RCY1769         MAT&#65533;?&#65533; ura3-52 leu2-3,112 lys2-801 his3&#9004;200                                                        This study <br /> RCY1764         MATa ura3-52 leu2-3,112 YIP1&#9004;KAN R [pRS315 yip1-4]                                               Calero et al. (2003) <br /> RCY1768         MATa ura3-52 leu2-3,112 YIP1&#9004;KAN R [pRS315 YIP1]                                                 Calero et al. (2003) <br /> RCY1880         MATa ura3-52 leu2-3,112 his3&#9004;200 lys2 GDI1&#9004;HISMX6 YIP1&#9004;KAN R [pRC1908]                           This study <br /> RCY2057         MATa ura3-52 leu2-3,112 his3&#9004;200 lys2-801 YIP1&#9004;KAN R [pRC1908]                                   This study <br /> RCY2137         MAT&#65533;?&#65533; ura3-52 leu2-3,112 his3&#9004;200 sec4-8 YIP1&#9004;KAN R [pRC1245]                                     This study <br /> RCY2155         MAT&#65533;?&#65533; ura3-52 leu2-3,112 sec2-41 YIP1&#9004;KAN R [pRC1245]                                             This study <br /> RCY2300         MAT&#65533;?&#65533; ura3 leu2 his3 lys2 sec14-3 YIP1&#9004;KAN R [pRC1908]                                            This study <br /> RCY2305         MATa ura3-52 leu2-3,112 his3&#9004;200 lys2-801 sec7-1 YIP1&#9004;KAN R [pRC1908]                            This study <br /> RCY2313         MAT&#65533;?&#65533; ura3 leu2 his3 lys2 YPT11&#9004;KAN R YIP1&#9004;KAN R [pRC1245]                                        This study <br /> RCY2319         MATa ura3-52 leu2-3,112 bet2-1 YIP1&#9004;KAN R [pRC1908]                                              This study <br /> RCY2321         MAT&#65533;?&#65533; ura3-52 leu2-3,112 bet3-1 YIP1&#9004;KAN R [pRC1908]                                              This study <br /> RCY2477         ura3-52 leu2-3,112 his3&#9004;200 lys2-801 YPT31&#9004;HIS3 YIP1&#9004;KAN R [pRC1908]                             This study <br /> RCY2498         MATa leu2&#9004;0 ura3&#9004;0 his3&#9004;0 lys2&#9004;0 met15&#9004;0 yip1&#9004;HISMX6 [pRC1245]                                   This study <br /> RCY2530         MAT&#65533;?&#65533; leu2&#9004;0 ura3&#9004;0 his3&#9004;0 lys2&#9004;0 ypt6&#9004;KAN R yip1&#9004;HISMX6 [pRC1245]                                This study <br /> RCY2540         MATa leu2&#9004;0 ura3&#9004;0 his3&#9004;0 lys2&#9004;0 gyp1&#9004;KAN R yip1&#9004;HISMX6 [pRC1245]                                This study <br /> RCY2542         MATa leu2&#9004;0 ura3&#9004;0 his3&#9004;0 lys2&#9004;0 met15&#9004;0 vps21&#9004;KAN R yip1&#9004;HISMX6 [pRC1245]                       This study <br /> RCY2678         ypt7&#9004;TRP1 yip1&#9004;KAN R leu2 ura3 his3&#9004;200 trp1 ade2-101 [pRC1245]                                  This study <br /> RCY2677         MAT&#65533;?&#65533; leu2&#9004;0 ura3&#9004;0 his3&#9004;0 lys2&#9004;0 met15&#9004;0 ypt10&#9004;KAN R yip1&#9004;HISMX6 [pRC1245]                       This study <br /> RCY2665         opi1&#9004;KAN R yip1&#9004;HISMX6 ura3 leu2 his4 his3? [pRC1245]                                            This study <br /> RCY2675         ura3-52 leu2-3,112 his4 ypt1A136D yip1&#9004;KAN R [pRC1245]                                           This study <br /> RCY2678         leu2 ura3 his3&#9004;200 trp1 ade2-101 ypt7&#9004;TRP1 yip1&#9004;KAN R [pRC1245]                                  This study <br /> RCY2837         MATa ura3-52 leu2-3,112 sec31-2 yip1&#9004;KAN R [pRC1908]                                             This study <br /> RCY3128         leu2&#9004;0 ura3&#9004;0 his3&#9004;0 lys2&#9004;0 ypt52&#9004;KAN R yip1&#9004;HISMX6 [pRC1245]                                    This study <br /> RCY3129         leu2&#9004;0 ura3&#9004;0 his3&#9004;0 lys2&#9004;0 ypt53&#9004;KAN R yip1&#9004;HISMX6 [pRC1245]                                    This study <br />  <br />  <br />  <br /> Rab protein function Yip1p action. We Rab-                 GAC 3&#11032;) S2YIP1 (5&#11032; GTTCAGAAAAACATATATACAAAT <br /> GDI critical subset Rab proteins crucial               ATCGCCCCTAAGCCAATTCCCTTCAATCGATGAATTCG <br />                                                                         AGCTCG 3&#11032;) primers (Wach et al. 1994). Site-directed deletions <br /> the action Yip1p and, moreover, mutations                    carried BY24 diploid strain precisely eliminate <br /> Yip1p impact vesicle budding lost ability                 YIP1 ORF. Genomic PCR using internal deletion primers <br /> to interact Rab proteins. A genetic analysis, com-                 RNC228 (5&#11032; CTATGGAACTGCCTCGGTGA 3&#11032;) RNC263 <br /> prising Rab gene identified S. cerevisiae, reveals             (5&#11032; CAGAAACAACTCTGGCGCATC 3&#11032;) paired flanking <br /> that requirement Rab proteins apparently                     primers YFYIP1 (5&#11032; CGGCCGCTCTAGAACTAGTGGATCCC <br />                                                                         GTATCTCGTTAGTACTTGTT 3&#11032;)and YRYIP1 (5&#11032; TCACAC <br /> confined subset Rab proteins operate                    AGGAAACAGCTATGACCATGAAGCTTGACCTTAGAGTAC <br /> the level Golgi apparatus. In discussion,                 AGACGATG 3&#11032;) performed check correct integra- <br /> suggest hypotheses reconcile light discovery               tion KANMX cassette ends. Haploid yip1&#9004; strains <br /> of roles Yip1p Golgi ER membranes.                      created transformation URA3 YIP1 YCp50 <br />                                                                         plasmid (pRC1245). A gdi1&#9004; strain RCY883 created a <br />                                                                         similar fashion HIS3MX6 module (Wach et al. 1997) <br />                                                                         PCR amplified using primers S1GDI1 (5&#11032; TCCATCA <br />                MATERIALS AND METHODS                                    TAACTGTTAGTGAATAACCACTTATATAGCATAACACAC <br />                                                                         GTACGCTGCAGGTC 3&#11032;)and S2GDI1 (5&#11032; ACCCTCCAATTG <br />    Yeast strains plasmid constructs: The S. cerevisiae strains      CTGCTTTAGTCGTAAAGGTATGAATTTACTGATCGA <br /> and <span id='am-129' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-130' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-131' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>plasmids</span> used studies listed Table 1. For           TGAATTCGAGCT 3&#11032;). Colonies checked correct <br /> all plasmid shuffle experiments, plates used synthetic             HIS3MX6 integration genomic PCR using primers 33942 <br /> complete media containing 1.5 mg/ml 5-fluoroorotic acid                 (5&#11032; GCGATGGCAACGCTGA 3&#11032;) RNC233 (5&#11032; GACCAT <br /> (5-<span id='am-9' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-10' property="rdfs:label" content="Foa" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-12'  ></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>FOA</span>). To create yip1&#9004; tester strains RCY1610 RCY             CATCGTGCTG 3&#11032;) paired flanking primers YFGDI1 (5&#11032; <br /> 1612, KANMX module (Wach et al. 1994) PCR amplified               CGGCCGCTCTAGAACTAGTGGATCCCGTATCTCGTTAG <br /> with S1YIP1 (5&#11032; GCTACAAATTGGACGGGAAGTACTGCAA                            TACTTGTT 3&#11032;) YRGDI1 (5&#11032; TACCGGGCCCCCCCTC <br /> GACAACTATTAGTCCCTCTCGAGCGTACGCTGCAGGTC                                  GAGGTCGACAGACTGACAGTTATACCCAAG 3&#11032;). Haploid <br />                                   Yip1p Function S. cerevisiae Requires Rab Proteins Rab-GDI                                1829 <br />  <br /> gdi1&#9004; strains created transformation GDI1                                     TABLE 2 <br /> URA3 plasmid (pRC1052C) sporulation. To obtain yip1&#9004; <br /> gdi1&#9004; double-deletion strains, RCY1612 mated RCY-                             Plasmids used study <br /> 883 create heterozygous yip1&#9004; gdi1&#9004; diploids. The <br /> resulting diploid plated 5-FOA containing media             Plasmid               Description             Source <br /> eliminate URA3 plasmid, transformed plasmid <br /> pRC1908 containing YIP1 GDI1 cloned pRS316,             pRC2411               GFP-yip1-4 pRS315        This study <br /> and sporulated yield yip1&#9004; gdi1&#9004; double-knockout strain        pRC2409               GFP-yip1-9 pRS315        This study <br /> RCY1718, yip1&#9004; strains RCY1633 RCY1634. Geno-          pRC2416               GFP-yip1-19 pRS315       This study <br /> mic PCR primers CC14 (5&#11032; TGAGCGCTAGGAGTCAC                       pRC2504               GFP-yip1-6 pRS315        This study <br /> TGCC 3&#11032;) CC15 (5&#11032; TCGCTCAGTTCAGCCATAATAT                          pRC2685               GFP-yip1-40 pRS315       This study <br /> GAA 3&#11032;) RCY1718 confirmed his3&#9004;200 auxotrophy               pRC2551               GFP-yip1&#9004;N65             This study <br /> RCY1718. To create yip1&#9004; gdi1&#9004; double-knockout strains           pRC940                YIP1 pRS426              This study <br /> additional auxotrophic markers, RCY1718 mated RCY-             pRC1037               YIF1 pRS426              This study <br /> 1769. Both gdi1&#9004; yip1&#9004; double-knockout strain RCY1880             pRC1040               YIP4 pRS426              This study <br /> and yip1&#9004; single-knockout strain RCY2057 isolated            pRC1739               YIF1 2&#65533;?&#65533; LEU2             This study <br /> from dissection. Strains synthetic lethality analysis   pRC122                YIP1 pRS317              This study <br /> constructed mating yip1&#9004; tester strains strain bearing      pRC1052C              GDI1 pRS316              This study <br /> the mutant allele consideration (Table 1). Diploids             pRC1039               yip1-2 pRS315            This study <br /> were sporulated create yip1&#9004; haploid strain            pRC1916               yip1-111 pRS315          This study <br /> contained secondary gene defect. Copper-dependent pro-            pRC1918               yip1&#9004;N4-7 pRS315         This study <br /> tein expression carried using 281 bp CUP1 <br />                                                                       pRC1920               yip1-6 pRS315            This study <br /> promoter region drive construct interest. Induction <br />                                                                       pRC1987               yip1-9 pRS315            This study <br /> was initiated addition CuSO4 media. Plasmids <br /> are listed Table 2.                                                pRC1989               yip1-3 pRS315            This study <br />    Phenotypic analysis: For analysis novel yip1 allele,       pRC1992               yip1-4 pRS315            This study <br /> at strains derived independent transformants           pRC1993               yip1-5 pRS315            This study <br /> were examined. To analyze growth, nonlethal yip1 strains         pRC1995               yip1-7 pRS315            This study <br /> plated YPD YPD supplemented 3% (vol/vol)                 pRC1997               yip1-8 pRS315            This study <br /> formamide passage 5-FOA-containing media elimi-           pRC2114               PCUP1 yip1-6 pRS315      This study <br /> nate wild-type plasmid. Plates incubated 25&#11034;, 30&#11034;,        pRC2174               gdi1-11 pRS317           This study <br /> 34&#11034;, 37&#11034; 3 days, 15&#11034; 11 days, 37&#11034; YPD &#11001;      pRC2212               yip1-43 pRS315           This study <br /> 3% formamide 5 days.                                              pRC2214               yip1-41 pRS315           This study <br />    Yeast two-<span id='am-124' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-125' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-127'  ></span><span id='am-128' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>hybrid</span> analysis: The mutant yip1 sequences          pRC2263A              yip1-15 pRS315           This study <br /> subcloned pACT2 &#8220;prey&#65533;? constructs assayed           pRC2264A              yip1-151 pRS315          This study <br /> YIF1, YPT1, YPT31, YPT32 &#8220;bait&#65533;? constructs reporter          pRC2265A              yip1-16 pRS315           This study <br /> yeast strain Y190. Due variability Y2H system,          pRC2266A              yip1-17 pRS315           This study <br /> independently generated identical yip1 constructs used           pRC2283               yip1-4 pRS317            This study <br /> to confirm interactions observed experiments. Plasmid          pRC2286               yip1-18 pRS315           This study <br /> pairs cotransformed yeast strain 30        pRC2289               yip1-12 pRS315           This study <br /> independent colonies assayed &#65533;?&#65533;-galactosidase activity.       pRC2293               yip1-19 pRS315           This study <br /> Filters photographed using Alpha Innotech DigiDoc             pRC2398               yip1-13 pRS315           This study <br /> system Olympus C-4040 <span id='am-87' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-88'  ></span><span id='am-89' property="rdfs:label" content="Camera" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>camera</span> settings reflective          pRC2400               yip1-40 pRS315           This study <br /> white light, 15 msec exposure, 105 mm zoom. &#65533;?&#65533;-Galactosi-          pRC2419               yip1-42 pRS315           This study <br /> dase activity quantified using ImageJ, public domain            pRC2452               yip1-44 pRS315           This study <br /> National Institutes Health imaging program (http://rsb.            pRC2454               yip1-31 pRS315           This study <br /> info.nih.gov/ij/), scale 0&#8211;255. Values plotted represent <br />                                                                       pRC2484               yip1&#9004;N29 pRS315          This study <br /> the mean difference measurements taken 10 colo- <br />                                                                       pRC2486               yip1&#9004;N40 pRS315          This study <br /> nies background value. <br />    Microscopy: For fluorescence microscopy, GFP NH2-              pRC2488/pRC2490       yip1&#9004;N55 pRS315          This study <br /> terminal fusions YIP1 yip1 alleles constructed         pRC2528/pRC2530       yip1&#9004;C18 pRS315          This study <br /> episomal <span id='am-105' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-106' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-107' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>vectors</span>. Each construct contains 238 amino acids          pRC2492               yip1-16 pRS315           This study <br /> yEGFP (Cormack et al. 1997), fused start methionine            pRC2493               yip1-20 pRS315           This study <br /> of tagged protein preceded GGPGG linker. Cells           pRC2496               yip1-15 pRS315           This study <br /> containing GFP-fusion plasmids examined Nikon             pRC2526               yip1-24 pRS315           This study <br /> Eclipse E600 equipped 60&#11003; objective 1.5&#11003; optovar.          YCp-HAC1                                       Kawahara et al. <br /> A Spot-RT monochrome CCD camera software version                                                                  (1998) <br /> 3.5 used image capture. All images shown represen-        YCp-HAC1&#9004;intron                                Kawahara et al. <br /> tative images cells logarithmic phase growth                                                            (1998) <br /> supplemented minimal media. <br />    Western blotting: Total cellular lysates analyzed by <br /> SDS-PAGE Western blotting BioTrace 0.45 &#65533;?&#65533;m poly- <br /> vinylidene difluoride (Pall). Membranes probed              ogy), diluted 1/5000 TBS 0.2% Tween-20 5% <br /> MAB3580 anti-GFP antibody (Chemicon, Temecula, CA)                 NF-milk, followed washing chemiluminescent develop- <br /> 1/6000 Tris-buffered saline (TBS) containing 0.2% Tween-           ment phenylphosphate substituted 1,2 dioxetane (CDP- <br /> 20 0.1% nonfat (NF)-milk detect GFP-tagged proteins.           Star; Perkin-Elmer, Norwalk, CT). To provide membrane <br /> Blots developed secondary alkaline phosphatase              protein-loading control, blots stripped 5-min incu- <br /> conjugated goat anti-mouse antibodies (Southern Biotechnol-           bation 0.2 m NaOH subsequently probed mono- <br />  1830                                                     C. Z. Chen et al. <br />  <br />                                                              TABLE 3 <br />                                                         Yip1 mutant alleles <br />  <br />          Allele                         Mutation                       Growth (YPD)a            Restriction enzyme site b <br />          YIP1                 None                                     Wild type <br />          yip1-111             Y4F, T6A, S7A                            Wild type                           &#8212; <br />          yip1&#9004;N4-7            &#9004; Residues 4&#8211;7                           Wild type                           &#8212; <br />          yip1&#9004;N29             &#9004; Residues 1&#8211;29                          Wild type                           &#8212; <br />          yip1&#9004;N40             &#9004; Residues 1&#8211;40                          Wild type                           &#8212; <br />          yip1&#9004;N55             &#9004; Residues 1&#8211;55                          Wild type                           &#8212; <br />          yip1&#9004;N65             &#9004; Residues 1&#8211;65                          Wild type                           &#8212; <br />          yip1&#9004;C18             &#9004; Residues 231&#8211;248                       Lethal                              &#8212; <br />          yip1-3               P51G, P53G, P55G                         Wild type                 NgoMIV <br />          yip1-4               E70K                                     Conditional c,d           XhoI <br />          yip1-41              E70G                                     Lethal                    KpnI, eliminate EcoRV <br />          yip1-40              E70G, K130A                              Conditional               NgoMIV <br />          yip1-44              E70K, K130E                              Lethal                    NgoMIV <br />          yip1-42              E70V                                     Conditional               KpnI, eliminate EcoRV <br />          yip1-43              E70A                                     Wild type e               KpnI, eliminate EcoRV <br />          yip1-5               E75K                                     Wild type                           &#8212; <br />          yip1-6               E76K                                     Lethal f                  AseI <br />          yip1-7               P71V                                     Wild type                 XhoI <br />          yip1-8               P72V                                     Wild type                 XhoI <br />          yip1-9               F81Y                                     Lethal                    Bcl I BbsI <br />          yip1-21              K87A                                     Wild type                 KpnI <br />          yip1-151             K87G                                     Wild type                 KpnI <br />          yip1-16              T88A                                     Wild type                 AseI <br />          yip1-23              D44G, I58T, L73I                         Wild type                           &#8212; <br />          yip1-15              D108A                                    Wild type                 Bgl II <br />          yip1-11              D110A                                    Wild type                 ApaI <br />          yip1-17              P114A                                    Wild type                 SacI <br />          yip1-13              K130A                                    Wild type                 NgoMIV <br />          yip1-31              K130E                                    Wild type                 NgoMIV <br />          yip1-12              L153G, M154G, S155G                      Wild type e               BseRI <br />          yip1-21              D158A                                    Wild type                 Sal I <br />          yip1-2 g             G175E                                    Wild type e                         &#8212; <br />          yip1-18              C177A                                    Wild type                 Nsi I <br />          yip1-19              P180G                                    Lethal                    StuI <br />          yip1-24              W208A, W211A                             Wild type                 Sal I <br />          yip1-20              P233A                                    Wild type                 Nsi I <br />          yip1-14              I167L, I189N, G198S, S239R               Conditional                         &#8212; <br />            a <br />               Growth phenotype haploid cells. The permissive temperature range indicated parentheses. RCY1610 <br />          cells, genotype MATa ura3-52 leu2-3,112 yip1&#9004;KANR [YCp50 YIP1] transformed [pRS315 yip1 mutant <br />          allele]. Transformants passaged 5-FOA eliminate wild-type YIP1 plasmid, surviving <br />          cells tested growth 15&#11034;, 25&#11034;, 30&#11034;, 34&#11034;, 37&#11034;, 40&#11034; described materials methods. <br />             b <br />               Silent restriction enzyme sites introduced screen constructs created site-directed mutagenesis, <br />          constructs verified DNA sequencing. <br />             c <br />               Recessive allele, suppressed multi-copy plasmids encoding YPT1, YPT31, YPT6, YIF1. <br />             d <br />               Cannot suppressed high-copy plasmids encoding YIF1, YIP4, YIP5, YOP1, PRA1/YIP3, GDI1, YPT1, <br />          YPT31, YPT32, co-overexpression SEC23 SEC24 activated (intronless) HAC1. <br />             e <br />               Very sick YPD &#11001; 3% formamide 37&#11034;. Formamide membrane-permeant metabolically inert <br />          molecule known weaken protein-protein interactions destabilizing noncovalent bonds; useful for <br />          screening conditional mutants (Aguilera 1994). <br />             f <br />               Dominant-negative allele. <br />             g <br />               Allele isolated R. Sternglanz transferred Collins lab strain background. <br />  <br />  <br /> clonal antibody 10D7 (Molecular Probes, Eugene, OR)         assays, <span id='am-119' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-120'  ></span><span id='am-121' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> points average duplicate determinations <br /> the membrane protein V-ATPase concentration 0.25            error bars represent range. <br /> &#65533;?&#65533;g/ml. <br />    In vitro vesicle budding transport assays: Yeast semi- <br /> intact cells wild-type yip1 mutant strains pre-                                RESULTS <br /> pared described (Baker et al. 1988). Vesicle transport and <br /> budding assays following [35S]gp&#65533;?&#65533;f carried pre-           Mutagenesis Yip1p reveals residues critical for <br /> viously described (Barlowe 1997; Cao et al. 1998). For vitro     function: To define minimal functional unit of <br />                               Yip1p Function S. cerevisiae Requires Rab Proteins Rab-GDI                        1831 <br />  <br /> Yip1p, performed Yip1p deletion analysis           reason lysine position 70 substitute for <br /> discovered able delete approximately            glutamic acid, opposite charge, unclear. <br /> one-quarter protein detectable effect            During course studies, conditional lethal <br /> (Table 3). Progressive truncations revealed            allele yip1, yip1-3, reported Ferro-Novick and <br /> NH2-terminal 65 residues (Yip1p 248 residues total)             colleagues (Barrowman et al. 2003). This allele was <br /> were fully dispensable growth. In contrast, trunca-       generated random mutagenesis contains four <br /> tion 18 residues COOH terminus resulted            mutations include <span id='am-19' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-20'  ></span><span id='am-21' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Glycine" datatype="xsd:string"></span>glycine</span> residue position 70. <br /> a gene unable complement deletion              Knowing E70G lethal mutation, reasoned <br /> YIP1 (Table 3). These results defined minimal func-           mutations yip1-3 represent <br /> tional unit consisting 183 amino acids         second-site intragenic suppressor. Our attention was <br /> used selective mutagenesis identify critical        drawn residue position 130 residue <br /> residues.                                                       conserved Yip1p orthologs (Figure 1A). Figure <br />    As basis mutagenesis study, targeted           2B shows cells bearing double-mutation E70G, <br /> residues conserved human YIP1A             K130A viable, conditional lethal. These <br /> yeast YIP1. Our previous work shown human          data demonstrate residue 130 fact suppressor <br /> sequence functionally replace deletion yeast           lethal mutation position 70, suggesting the <br /> YIP1, indicating phylogenetic analysis pro-        residues directly indirectly interact. Because the <br /> vide rational approach dissecting molecular           naturally occurring amino acids positions 70 130 <br /> action Yip1p (Calero et al. 2003). An alignment           opposite charge, sought determine they <br /> HsYIP1A YIP1 shown Figure 1A. We mu-             form charge pair Yip1p tertiary structure. Al- <br /> tated charged residue Yip1p rationale            single point mutation K130E innocuous, <br /> that residues highest probability pro-         apparent phenotype, charge reversal both <br /> tein surface exposure intermolecular contact; there-        amino acids simultaneously gave lethal phenotype <br /> fore mutation likely perturb overall       (Figure 2B), leading conclude resi- <br /> protein fold (Chothia 1976). Silent restriction enzyme          dues interact indirectly contribute func- <br /> sites included possible facilitate diagnos-       tionality Yip1p. <br /> tic analysis mutant construct creation veri-            Expression localization defective yip1 alleles: <br /> fied DNA sequencing. Constructs transformed           To determine growth defects associated <br /> into haploid yip1&#9004; tester strain (RCY1610)              yip1 allele attributable aberrant Yip1p <br /> streaked 5-FOA-containing media 25&#11034; assess           levels, analyzed whole-cell lysates expressing GFP- <br /> for functionality comparison yeast transformed            tagged yip1 alleles affinity-purified antibody that <br /> with vector negative control wild-          recognizes GFP. Of temperature conditional-lethal <br /> type YIP1 positive control. Viable mutant yip1 con-        haploid yip1 strains, displayed aberrant Yip1p lev- <br /> structs tested rich media range tempera-        els permissive temperature (Figure 3A) upon <br /> tures YPD containing 3% formamide. The yip1-4            shift restrictive temperature period time <br /> allele (E70K) described previously (Calero et          known sufficient elicit terminal phenotype <br /> al. 2003). The yip1-2 allele described thermosensi-      (Table 3 unpublished results). Of null al- <br /> tive Gallwitz co-workers (Yang et al. 1998); how-        leles, yip1-9 reduced Yip1p levels respect a <br /> ever, wild type rich media           wild-type control strain 25&#11034; (Figure 3A). Both yip1-9 <br /> recreated strain background.                             yip1-19 migrate faster rate wild-type <br />    An overview mutagenesis data shown             Yip1p (Figure 3A), indicative protein degradation, <br /> Figure 1B detailed summary site-         result defective protein null <br /> directed mutagenesis Table 3. Of mutants,        phenotype. The lack correlation Yip1p levels <br /> point mutants (yip1-41, yip1-9, yip1-19)          phenotypes conditional mutant strains <br /> be recessive null mutants, (yip1-6, Figure 1C)          indicates observed phenotypes to <br /> a dominant-negative null allele, (yip1-4, yip1-40,         altered Yip1p levels result defective Yip1p- <br /> yip1-14, yip1-42; Figure 2A) thermosensitive,          protein interactions. <br /> three alleles (yip1-43, yip1-12, yip1-2) sensitive         A combination subcellular fractionation vivo <br /> formamide 37&#11034;. The conditional temperature           fluorescence experiments previously established <br /> yip1-40 allele YPD 37&#11034; 40&#11034; yip1-42,       Yip1p cycles ER Golgi (Heidt- <br /> the yip1-4 allele viable 25&#11034; lethal 34&#11034;         man et al. 2003). Because GFP-tagged Yip1p fully func- <br /> rich media. Glutamic acid position 70 particularly        tional sole cellular copy Yip1p, indicating that <br /> interesting, thermosensitive mutated         use tag does impair function (Heidtman <br /> to lysine valine, lethal mutated glycine,          et al. 2003), GFP tagged yip1 mutant alleles to <br /> and small effect mutated alanine. Glycine         determine steady-state localization live cells. To <br /> can characterized &#8220;flexible&#65533;? amino acid             make comparisons different constructs, both <br /> might disrupt secondary structure Yip1p            null alleles alleles able function single copy <br /> when inserted position 70 (Fersht 1999). However,            expressed single copy background con- <br />  1832                                                    C. Z. Chen et al. <br />  <br />  <br />  <br />  <br />   Figure 1.&#8212;(A) Alignment YIP1 orthologs identifying residues targeted mutagenesis: sequence Yip1p comparison <br /> with full-length cDNAs human YIP1 (HsYIP1), mouse YIP1 (MmYIP1), <span id='am-70' about='obo:NCBITaxon_6239' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_119089 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_55879 obo:NCBITaxon_131567 obo:NCBITaxon_55885 obo:NCBITaxon_6237 obo:NCBITaxon_6236 obo:NCBITaxon_33154 obo:NCBITaxon_6231 obo:NCBITaxon_1206794 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6243 obo:NCBITaxon_6072'><span id='am-71' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="Caenorhabditis elegans" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasRelatedSynonym" content="Rhabditis elegans" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasExactSynonym" content="roundworm" datatype="xsd:string"></span><span id='am-76'  ></span>Caenorhabditis elegans</span> YIP1 (CeYIP1). The <br /> sequences aligned MegAlign (DNASTAR) using Clustal analysis gap-length penalty 10. Amino acid residues <br /> are numbered according yeast Yip1p primary sequence. The residues black background exactly match consensus <br /> sequence. (B) Mutagenized Yip1p residues distributed NH2- COOH-terminal domain Yip1p. YIP1 <br /> family members share common domain topology NH2-terminal hydrophilic COOH-terminal hydrophobic domain <br /> indicated Membrane Protein Explorer (MPEx) profile Yip1p, generated octanol-interface scale (White and <br /> Wimley 1999; Jayasinghe et al. 2001). Yeast Yip1p residues mutagenized study indicated coded according to <br /> physiological impact designated symbol placed amino acid position MPEx profile Yip1p. Table 3 contains <br /> a detailed description mutant alleles. (C) Expression dominant-negative allele, yip1-6. Cells containing plasmids <br /> expressing yip1-6 (E76K) control copper-inducible CUP1 promoter grown early log phase adjusting <br /> and adding CuSO4 media indicated. Growth assessed OD600 turbidity measurements. At 0.5 mm Cu2w&#11001; levels, <br /> growth ceases 5 hr 30&#11034; cells expressing yip1-6, vector-only control cells. <br />                                 Yip1p Function S. cerevisiae Requires Rab Proteins Rab-GDI                              1833 <br />  <br />                                                                                                          Figure 2.&#8212;(A) Thermo- <br />                                                                                                       sensitivity yip1 alleles. <br />                                                                                                       Growth yip1-40, yip1-42, <br />                                                                                                       yip1-4 cells YPD are <br />                                                                                                       compared isogenic wild- <br />                                                                                                       type controls temper- <br />                                                                                                       atures indicated. yip1-4 mu- <br />                                                                                                       tant cells bearing single <br />                                                                                                       point mutation E70K are <br />                                                                                                       thermosensitive re- <br />                                                                                                       strictive temperature 34&#11034; <br />                                                                                                       rich media; yip1-40 cells <br />                                                                                                       (E70G, K130A) yip1-42 <br />                                                                                                       cells (E70A) severely <br />                                                                                                       growth retarded 37&#11034; and <br />                                                                                                       nonviable 40&#11034;. (B) Inter- <br />                                                                                                       action Yip1p residues 70 <br />                                                                                                       130: growth phenotype <br />                                                                                                       haploid cells muta- <br />                                                                                                       tions residue positions 70 <br />                                                                                                       130 Yip1p. RCY1610 <br />                                                                                                       cells, genotype MAT&#65533;?&#65533; ura3- <br />                                                                                                       52 leu2-3,112 YIP1&#9004;KAN R <br />                                                                                                       [YCP50 YIP1], trans- <br />                                                                                                       formed pRS315 yip1 <br />                                                                                                       plasmids indicated. Trans- <br />                                                                                                       formants streaked onto <br />                                                                                                       5-FOA containing media at <br />                                                                                                       25&#11034; assess growth the <br />                                                                                                       mutant versions genes <br /> as sole copy. All genes expressed control endogenous promoter terminator. The single point <br /> mutations E70K (yip1-4, pRC1992), K130A (yip1-13, pRC2398), K130E (yip1-31, pRC2454) viable contrast E70G <br /> (yip1-41, pRC2214), lethal single point mutant. K130A suppress lethality E70G (yip1-40, pRC2400); however, <br /> the combination mutants positions create charge reversal naturally occurring amino acids, E70K/ <br /> K130E (yip1-44, pRC2452), yields lethal phenotype, suggesting intragenic suppression occurs indirect interaction <br /> of residues. <br />  <br />  <br />  <br /> taining wild-type copy YIP1. The results               port <span id='am-1' about='obo:NCBITaxon_11276' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_11157 obo:NCBITaxon_10239 obo:NCBITaxon_439488 obo:NCBITaxon_11270 obo:NCBITaxon_11271 obo:NCBITaxon_35301 obo:NCBITaxon_327822'><span id='am-2' property="oboInOwl:hasRelatedSynonym" content="vesicular stomatitis virus VSV" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasAlternativeId" content="NCBITaxon:12872" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasRelatedSynonym" content="vesicular stomatitis virus, VSV" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="Vesicular stomatitis virus" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>vesicular stomatitis virus</span> envelope glycoprotein <br /> analysis shown Figure 3B. GFP-yip1&#9004;N65, -yip1-4,             physically associates mammalian Sec23 and <br /> -yip1-40, -yip1-6 showed localization similar       Sec24 proteins (Tang et al. 2001). A conserved domain <br /> previously established wild-type protein,            NH2 terminus Yip1A required interac- <br /> numerous fluorescent Golgi puncta nuclear rim                   tion Sec23p/Sec24p, residue this <br /> staining indicative ER, demonstrating yip1            domain mutated thermosensitive allele <br /> alleles normally packaged vesicles deliv-           yip1-4. Sec23p Sec24p components COPII <br /> ered Golgi.                                                  coat complex heteromeric complex <br />    The null alleles GFP-yip1-19 -yip1-9 infor-         (Barlowe 2002a,b). In contrast results obtained <br /> mative analysis expression level appeared          mammalian YIP1A, dominant-negative <br /> somewhat reduced compared wild type                     effect overexpression cytoplasmic portion <br /> showed diffuse cytoplasmic staining suggestive in-             (data shown). SEC23 SEC24 effect on <br /> correctly folded membrane inserted proteins (Figure              growth yip1-4 expressed singly or <br /> 3B) targeted degradation, consistent ap-             simultaneously multi-copy plasmids. <br /> parent migration SDS-PAGE gels (Figure 3A).                         Mutants defective vesicle budding ER <br />    Directed test yip1 suppression high-copy plas-           sec12 sec23 appear activate unfolded <br /> mids: To establish possible sequence events               protein response (Belden Barlowe 2001) can <br /> which Yip1p functions, asked yip1 mutant              suppressed IRE1 activated HAC1 (Higashio <br /> be suppressed multi-copy plasmids encoding               Kohno 2002; Sato et al. 2002). Yip1p been <br /> candidate genes, act conjunction                  implicated ER vesicle budding yip1-4 shows selec- <br /> YIP1.                                                               tive genetic interactions subgroup se- <br />    The mouse homolog Yip1p, Yip1A, localized               cretory genes; however, yip1-4 suppressed by <br /> ER exit sites vesicular-tubular structures,               activated HAC1, indicating role Yip1p ER <br /> carry cargo Golgi (Tang et al. 2001). Overexpres-            vesicle biogenesis distinct COPII coat. <br /> sion cytoplasmic portion Yip1A inhibits trans-               We tested (i) multi-copy plasmids encoding Rab <br />  1834                                                     C. Z. Chen et al. <br />  <br />                                                                     ported physically associate Yip1p, Yop1p <br />                                                                     Yip3p (Calero et al. 2001, 2002), restore the <br />                                                                     functionality lacking yip1-4 single point mutation. <br />                                                                        Genetic analysis yip1-4 reveals interaction Rab <br />                                                                     GDI mutants: Yip1p binds Rab proteins cellular <br />                                                                     lysates little discrimination (Calero et al. 2002, <br />                                                                     2003) required COPII vesicle biogenesis <br />                                                                     (Heidtman et al. 2003), process vitro recon- <br />                                                                     stitution studies shown Rab proteins not <br />                                                                     play critical roles (Cao Barlowe 2000). To address <br />                                                                     puzzle, initiated genetic analysis determine <br />                                                                     Rab genes necessary Yip1p function. In the <br />                                                                     secretory pathway, analyses demonstrated <br />                                                                     synthetic genetic interactions largely stage specific <br />                                                                     (Kaiser Schekman 1990; Finger Novick <br />                                                                     2000). Even ER-to-Golgi stage transport, <br />                                                                     genetic interactions occur mutations af- <br />                                                                     fected vesicle budding ER mutants <br />                                                                     affected fusion ER-derived vesicles Golgi, <br />                                                                     classes mutations (Kaiser <br />                                                                     Schekman 1990). We use this <br />                                                                     approach establish role YIP1 biogenesis <br />                                                                     COPII vesicles ER (Heidtman et al. 2003). <br />                                                                     In experiments, wished evaluate physio- <br />                                                                     logical relevance Rab proteins Yip1p function. <br />                                                                     We examined mutant alleles Rab genes genes <br />                                                                     encoding proteins regulate function Rab <br />                                                                     proteins, GTPase activators (GAPs) guanine <br />                                                                     nucleotide exchange factors (GEFs; Collins 2003). <br />                                                                        yip1-4 especially suited genetic studies this <br />                                                                     mutant does revertants thermosensitive <br />                                                                     growth wide range temperatures the <br />    Figure 3.&#8212;Analysis tagged yip1 mutant alleles. (A, top)       protein product stable restrictive temperature. For <br /> Immunoblots total lysates congenic log-phase GFP-tagged       mutant tested, created strain deleted <br /> yip1 haploid strains, grown minimal medium probed            genomic copy YIP1 wild-type copy of <br /> with anti-GFP antibody visualize GFP-tagged yip1 allele. <br /> Blots stripped 0.2 m NaOH reprobed mono-         YIP1 episomal URA3 plasmid counter- <br /> clonal antibody 10D7 recognizes V-ATPase loading          selected drug 5-FOA. Each strain trans- <br /> control (bottom). Negative control indicates lysates        formed plasmids containing: (1) yip1 ts allele, (2) <br /> isogenic cell line expressing GFP. (B) The localization      YIP1 wild-type plasmid, (3) no-insert control plas- <br /> GFP-YIP1 wild-type mutant alleles visualized vivo         mid. The resulting transformants plated 5-FOA- <br /> fluorescence microscopy wild-type (RCY239) cells. (Left) <br /> GFP fluorescence; (right) DIC optics field.             containing media determine double mutant <br />                                                                     viable. The advantages assay (i) <br />                                                                     double-mutant colonies ana- <br /> genes, (ii) plasmids encoding proteins known               lyzed single plate (see Figure 4A); (ii) comparisons <br /> potentially interact Yip1p, (iii) YIP1           isogenic strains mutant wild- <br /> family paralogs yeast, YIF1, YIP4, YIP5 (Calero              type allele introduced episomal plasmid, <br /> et al. 2002). The selected basis            avoiding necessity multiple backcrosses com- <br /> genes encoding interacting proteins                   pare mutations different strain backgrounds; <br /> suppress defective alleles overexpressed               (iii) assay flexible, impact several <br /> latter selected test possible functional overlap            different alleles tested simultaneously. Figure 4A <br /> among YIP1 family members single organism.                 shows synthetic interaction assay cells grown <br /> No genes deliberate suppression testing showed              plates. Synthetic interactions revealed lack <br /> any level suppression yip1-4 (Table 3). These data            growth presence 5-FOA plate media <br /> indicate yip1-4 allele defective unique            ypt31&#9004; yip1-4, ypt32 A134D yip1-4 double mutants as <br /> function Yip1p. The yip1-4 defect bypassed             examples complete lethality mutants <br /> by YIP1 family members, Yip1p acting up-               vps21&#9004; yip1-4, sec4-8 yip1-4, ypt11&#9004; yip1-4 double <br /> stream Rab proteins. In addition, proteins re-             mutants shown comparison examples specificity <br />                                   Yip1p Function S. cerevisiae Requires Rab Proteins Rab-GDI                           1835 <br />  <br />                                                                      defective yip1 alleles. Only yip1-4 shows synthetic lethal <br />                                                                      interactions ypt1 A136D. Of conditional yip1 <br />                                                                      alleles, yip1-42 appears detrimental effect <br />                                                                      double mutant slightly sicker wild- <br />                                                                      type control. This rapid analysis alleles simulta- <br />                                                                      neously helps understand genetic interactions <br />                                                                      observed, allowing dissect individual contri- <br />                                                                      butions various regions protein overall func- <br />                                                                      tion. <br />                                                                         To provide complete analysis, investigated ge- <br />                                                                      netic interactions yip1-4 Rab ORF <br />                                                                      identified yeast genome. There 11 Rab genes <br />                                                                      complete S. cerevisiae genome (<span id='am-14' about='obo:NCBITaxon_316718' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3646 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_235631 obo:NCBITaxon_3977 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_235892 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-15' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Garcia" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18'  ></span>Garcia</span>-Ranea <br />                                                                      <span id='am-109' about='obo:NCBITaxon_195854' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_8087 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_28738 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_7898 obo:NCBITaxon_195853 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-110' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-111' property="rdfs:label" content="Valencia" datatype="xsd:string"></span><span id='am-112'  ></span><span id='am-113' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Valencia</span> 1998). Table 4 summarizes analysis of <br />                                                                      genetic interactions Rab-encoding genes and <br />                                                                      yip1. No synthetic lethality observed sec4-8, <br />                                                                      ypt10&#9004;, ypt11&#9004;, vps21&#9004;, ypt52&#9004;, ypt53&#9004;, ypt7&#9004;. In con- <br />                                                                      trast, synthetic lethality observed ypt1 A136D, <br />                                                                      ypt31&#9004;, ypt6&#9004;, ypt32 A134D. The ypt1 A136D allele confers <br />                                                                      tight block growth 37&#11034; defect <br />                                                                      permissive temperature (Jedd et al. 1995); also <br />                                                                      observed deleterious genetic interactions another <br />                                                                      ypt1 allele, ypt1-3. Ypt1p, Ypt31/32p, Ypt6p oper- <br />                                                                      ate different membrane trafficking pathways: Ypt1p <br />                                                                      ER Golgi early Golgi transport, Ypt31/32p in <br />                                                                      intra-Golgi Golgi vacuole transport (Benli et al. <br />                                                                      1996; Jedd et al. 1997), Ypt6p retrograde Golgi <br />                                                                      transport (Luo Gallwitz 2003). The feature that <br />                                                                      proteins common distin- <br />                                                                      guishes Rab genes syntheti- <br />                                                                      cally lethal yip1-4 localized the <br />                                                                      Golgi apparatus steady-state levels. These data suggest <br />                                                                      Yip1p action vivo does require Rab pro- <br />                                                                      tein functionality requirement confined <br />                                                                      subset Rab proteins act Golgi appa- <br />    Figure 4.&#8212;Synthetic lethality assay yip1 alleles Rab- <br /> encoding genes genes regulating Rab protein function. <br />                                                                      ratus. <br /> (A) Synthetic lethality plate assay: strains carrying thermosensi-      Because known dependence Yip1p-Rab pro- <br /> tive mutations Rab protein ORFs various genes involved        tein interactions prenylation (Calero et al. 2003), <br /> in Rab protein function deletion YIP1         examined influence defects prenyl <br /> wild-type YIP1 URA3 plasmid transformed vec-          machinery yip1-4 using mutant allele gera- <br /> <span id='am-114' about='obo:NCBITaxon_172918' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_186627 obo:NCBITaxon_186626 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_7952 obo:NCBITaxon_7953 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_32519 obo:NCBITaxon_30727 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_186634 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_41665 obo:NCBITaxon_32443'><span id='am-115'  ></span><span id='am-116' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-118' property="rdfs:label" content="Tor" datatype="xsd:string"></span>tor</span>-only control (pRS315), wild-type YIP1 control (pRC- <br /> 1838B), thermosensitive allele yip1-4 (pRC1992). All <br />                                                                      nylgeranyl transferase, bet2-1 (Rossi et al. 1991). Because <br /> transformants assessed plasmid shuffling 5-FOA            yip1 defects cause ER membrane proliferation, we <br /> at 25&#11034;. Plates bearing vps21&#9004; sec4-8, ypt11&#9004; double mutants          examined interactions opi1&#9004; sec14 ts, proteins <br /> with yip1-4 shown examples genetic interaction       involved phospholipid metabolism interact with <br /> was observed. Similarly, ypt31&#9004; ypt32 A134D yip1-4 dou-     regulate secretory pathway genes (Henry and <br /> ble mutants shown examples complete lethality be- <br /> tween mutants. (B) Testing multiple alleles synthetic <br />                                                                      Patton-Vogt 1998; Carman Henry 1999; Li et al. <br /> lethality: isogenic strains bearing various yip1 conditional le-     2000). These mutants negative genetic interac- <br /> thal alleles indicated tested genetic interactions       tions yip1-4, sec7-1, ADP-ribosylation factor <br /> at 25&#11034; temperature-sensitive gene ypt1A136D. Wild-type      exchange factor profound effects Golgi struc- <br /> YIP1 vector included control positive         ture (Rambourg et al. 1993; Table 4). <br /> and negative growth, respectively. <br />                                                                         Genetic interactions yip1-4 known Rab <br />                                                                      regulators analyzed. As small GTPase <br />                                                                      superfamily members, Rab regulators modulate the <br /> controls genetic interaction observed. Fig-             GTPase nucleotide cycle interact GTPase at <br /> ure 4B demonstrates comparisons different               defined stages assigned defined classes <br /> yip1 alleles simultaneously. In example, genetic in-            effectors, GEFs, GAPs. We focused Ypt1p <br /> teractions examined ypt1 A136D               previous work established synthetic <br />  1836                                                        C. Z. Chen et al. <br />  <br />                                                                   TABLE 4 <br />                                                 Summary genetic interaction data <br />  <br /> Mutant category                    Lethal combined yip1-4                          Viable combined yip1-4 a <br /> Rab genes                         ypt1 A136D, ypt1-3, ypt31&#9004;, ypt32 A134D, ypt6&#9004;       vps21&#9004;, ypt52&#9004;, ypt53&#9004;, ypt7&#9004;, ypt10&#9004;, ypt11&#9004;, sec4-8 <br /> Rab regulators modifiers      gdi1-11                                              sec2-41, bet3-1, gyp1&#9004;, bet2-1 <br /> Other                             None identified                                      opi1&#9004;, sec7-1, sec14 ts, dnm1&#9004; <br />   a <br />    Lethality indicates double mutant inviable 25&#11034;; viability indicates examination double mutant showed <br /> no deleterious synthetic effects. <br />  <br />  <br />  <br /> lethal relationship yip1-4 uso1-1, en-                   series genetic crosses using novel mutant <br /> codes yeast homolog p115, Rab1 effector (Allan                      alleles yip1. Our results summarized Table 5. <br /> et al. 2000), GEF GAP proteins                        We new alleles yip1 failed show <br /> have identified Rab protein. Mutations                    synthetic lethality Rab mutants re- <br /> BET3, Ypt1p GEF (Jones et al. 2000; Wang et al.                        taining synthetic lethality COPII mutants. Con- <br /> 2000) Gyp1p, GAP protein Ypt1p (Du                          versely, however, did yip1 alleles that <br /> Novick 2001), effect combination yip1-4,                    showed synthetic lethality selectively Rab GTPases <br /> and did sec2-41, mutant allele GEF                    COPII mutants. Strikingly, condi- <br /> Sec4p, close homolog Ypt1p (Walch-Solimena et                         tional yip1 allele tested showed synthetic lethality with <br /> al. 1997).                                                                 COPII mutant sec13-1. These data provide additional <br />    We tested mutant alleles Rab-GDI combina-                    support Yip1p acting COPII vesicle biogenesis <br /> tion yip1-4. Rab-GDI involved membrane                      pathway suggest action Yip1p on <br /> retrieval attachment Rab proteins (see Introduc-                    Golgi Rab proteins distinguished its <br /> tion) encoded S. cerevisiae single gene                   role COPII vesicle biogenesis. <br /> GDI1. Both gdi1-11 gdi1-29 generated ran-                         Interaction analysis yip1 alleles Rab proteins: <br /> dom mutagenesis GDI1 proposed                       To investigate basis Rab Yip1p interac- <br /> defective GDI-mediated extraction loading                    tion, asked yip1 mutant alleles retained an <br /> Rab proteins, respectively (Gilbert Burd 2001).                        ability interact Rab proteins. For test, we <br /> We unable recreate gdi1-29 allele YIP1                    determined ability yip1 mutant interact <br /> tester strain conclude strain bearing                    Rab proteins yeast hybrid (Y2H). Previous <br /> allele grows adaptations unfortunately make                      work laboratory demonstrated Yip1p <br /> it unsuitable genetic analyses. However, gdi1-                     Y2H assays reflect protein interactions reca- <br /> 11 mutant allele lethal combination yip1-4                     pitulated pull-down experiments yeast lysates <br /> (Table 4). These data indicate Yip1p action vivo                   represent valid method analysis Yip1p-pro- <br /> intersects activity Rab-GDI supports                   tein interactions (Calero et al. 2002, 2003). We tested <br /> suggestion proteins act common                         interactions yip1 mutant alleles Rab <br /> pathway influence Rab protein function.                                 proteins Ypt1p Ypt31p, genetic analysis <br />    To directly address issue Yip1p works                    shown physiologically relevant binding part- <br /> in conjunction Rab proteins COPII proteins,                     ners, Yif1p, YIP1 family member binds Yip1p <br />  <br />  <br />  <br />                                                                   TABLE 5 <br />                                 Summary genetic interaction data conditional yip1 alleles <br />  <br />          yip1 allele           Mutation              ypt1 A136D    ypt6&#9004;      gdi1-11       sec13-1   sec23-1    sec31-2   sec12-4 <br />          yip1-2        G175K                            &#11001;           &#11001;              &#11001;          &#11002;         &#11001;          &#11001;          &#11001; <br />          yip1-4        E70K                             &#11002;           &#11002;              &#11002;          &#11002;         &#11002;          &#11002;          &#11002; <br />          yip1-12       L153G, M154G, S155G              &#11001;           &#11001;              &#11001;          &#11002;         &#11001;          &#11001;          &#11001; <br />          yip1-40       E70G, K130A                      &#11001;           &#11001;              &#11001;          &#11002;         &#11002;          &#11006;          &#11006; <br />          yip1-42       E70V                             &#11006;           &#11006;              &#11001;          &#11002;         &#11002;          &#11006;          &#11001; <br />          yip1-43       E70A                             &#11001;           &#11001;              &#11001;          &#11002;         &#11001;          &#11006;          &#11001; <br />             (&#11002;), double mutant inviable 25&#11034;; (&#11001;), double mutant viable deleterious synthetic <br />          effects; (&#11006;), double mutant viable 25&#11034;; however, growth impaired comparison individual <br />          mutants. <br />                                 Yip1p Function S. cerevisiae Requires Rab Proteins Rab-GDI                              1837 <br />  <br />  <br />  <br />  <br />   Figure 5.&#8212;Y2H interactions yip1 mutant alleles Rab-interacting partners. Pairs constructs coexpressed <br /> in reporter strain Y190 &#65533;?&#65533;-galactosidase activity (arbitrary units) resulting transformants measured described <br /> in materials methods. The protein &#8220;bait&#65533;? constructs tested &#8220;prey&#65533;? constructs Yip1p mutant alleles, Yip1p, <br /> or vector-only (pAS2-1) controls indicated y-<span id='am-92' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-93'  ></span><span id='am-94' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="Axis" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>axis</span>. Shaded bars, yip1 alleles containing mutations NH2 terminus; <br /> solid bars, COOH-terminal alleles. Plasmid constructs listed Ypt1p interactions; constructs used all <br /> other experiments. <br />  <br />  <br />  <br />  <br /> (Matern et al. 2000; Calero et al. 2002). For mu-              All interactions required COOH-terminal 18 amino <br /> tant allele, tested independently created con-               acids critical function (Table 3), the <br /> structs previously observed slight variations            point mutant showed interaction both <br /> between individual constructs system. The results           Rab proteins Yif1p null allele yip1-41. All <br /> of experiment shown Figure 5, yip1                alleles yip1 point mutations NH2 <br /> alleles containing mutations NH2 terminus                terminus showed level interaction Rab <br /> indicated shaded bars COOH-terminal alleles                  proteins, exception thermosensitive <br /> are indicated solid bars (see Figure 1B represen-          allele yip1-42 null mutant yip1-41 inter- <br /> tation Yip1p domains). Rab proteins showed robust                action Rab proteins detected. <br /> interactions positive control wild-type Yip1p.             yip1-40 yip1-42 cells display defect COPII <br /> In contrast, subset mutant yip1 alleles dimin-            vesicle budding vitro: To investigate corre- <br /> ished abrogated yip1 function did display Rab                lation Rab protein interaction, genetic analysis, <br /> protein interactions. Diminished Rab protein interac-               ER vesicle biogenesis, examined ability of <br /> tion correlated strongly point mutations                membranes yip1 mutant cells produce COPII <br /> hydrophobic COOH terminus protein. In con-                   vesicles vitro. For purpose, investigated the <br /> trast, point mutants hydrophobic COOH-termi-                 mutant alleles yip1-40 yip1-42 these <br /> nal region showed ability interact Yif1p.          mutants differences Rab protein interaction <br />  1838                                                       C. Z. Chen et al. <br />  <br />  <br />  <br />  <br />                                                                          Figure 7.&#8212;Two models Yip1p function. Genetic analysis <br />                                                                       shows Yip1p &#8220;buffers&#65533;? (Hartman et al. 2001) action <br />                                                                       COPII genes ER vesicle biogenesis genes encoding <br />                                                                       Golgi Rab proteins. Although mechanisms underlie <br />                                                                       genetic relationships known, data suggest <br />                                                                       models: (A) Yip1p operates independently ER vesicle <br />                                                                       biogenesis Golgi Rab proteins, (B) Yip1p func- <br />                                                                       tion links ER vesicle biogenesis action Golgi Rab <br />                                                                       proteins. <br />  <br />  <br />  <br />                                                                       efficiency &#11601;11%. These data indicate that <br />                                                                       transport defect yip1-42 strain, previously <br />                                                                       observed yip1-4 mutant strains (Heidtman et al. <br />                                                                       2003), defect ER vesicle budding. Moreover, from <br />                                                                       analysis mutants, yip1-40 mini- <br />                                                                       mal yip1-42 severe defects Rab protein inter- <br />                                                                       action, appear distinction made <br />                                                                       ER vesicle budding Rab protein interac- <br />                                                                       tion. The degree defect ER vesicle budding thus <br />    Figure 6.&#8212;yip1 membranes display defect COPII vesicle <br /> budding vitro. Washed semi-intact cells containing [35S]gp&#65533;?&#65533;f       appears correlate Rab-interacting ability of <br /> were prepared wild-type yip1-40 (A) yip1-42 (B)           mutant allele. <br /> strains. Semi-intact cells incubated buffer (NA) or <br /> a reconstitution mixture (Recon), containing COPII, Uso1p, <br /> LMA1, ATP regeneration system, measure ER-                                         DISCUSSION <br /> to-Golgi transport. After 75 min 23&#11034;, Golgi- <br /> modified [35S]gp&#65533;?&#65533;f measured determine transport effi-             The molecular function Yip1p puzzling; has <br /> ciency. (C) Semi-intact cells wild-type yip1-42 mutant       reported physically interact Rab proteins <br /> strains prepared B incubated COPII pro-           required biogenesis ER <br /> teins measure vesicle budding efficiency. After 30 min 23&#11034;,     vesicles. Although recent work implicated Rab pro- <br /> freely diffusible vesicles containing [35S]gp&#65533;?&#65533;f separated        teins vesicle budding (Pfeffer 2001; Segev 2001a,b; <br /> from semi-intact cell membranes centrifugation 18,000 &#11003; <br /> g [35S]gp&#65533;?&#65533;f quantified ConA precipitation.                     Smythe 2002), previous vitro studies established <br />                                                                       functional Ypt1p required ER-to-Golgi <br />                                                                       transport (Cao Barlowe 2000). These considera- <br />                                                                       tions raise question physiological relevance <br /> and mutant alleles defective Yif1p             biochemical interactions observed Yip1p <br /> interaction (Figure 5). As shown Figure 6, yip1-40                 Rab proteins. To address puzzle, initiated <br /> exhibited mild defect yip1-42 exhibited                  genetic analysis evaluate physiological relevance <br /> significant defect transport [35S]gp&#65533;?&#65533;f Golgi             Rab proteins Yip1p function. Our genetic interac- <br /> complex comparison isogenic wild-type strain                 tion results indicate Yip1p function requires Rab <br /> at 23&#11034;. Specifically, yip1-42 membranes displayed               proteins mutational analysis demonstrating a <br /> reduction [35S]gp&#65533;?&#65533;f transport &#11601;40%                 subset yip1 mutants impaired biological function <br /> wild-type level. To identify stage transport             lost Rab protein-interacting capability. <br /> was compromised, examined ability yip1-                 Through genetic analysis covers single Rab <br /> 42 strain bud COPII vesicles (Figure 6C). We ob-                   protein yeast genome (Lazar et al. 1997), we <br /> served addition COPII proteins,                  establish Rab proteins required Yip1p func- <br /> wild-type membranes budded vesicles efficiency                  tion specifically localized Golgi, <br /> of 43%, yip1-42 membranes budded vesicles                 Ypt1p, Ypt31p, Ypt32p, Ypt6p. These data indicate <br />                                Yip1p Function S. cerevisiae Requires Rab Proteins Rab-GDI                          1839 <br />  <br /> that, Yip1p interact promiscuously           p115 well-established function tethering of <br /> with Rabs vitro, interactions biological         ER-derived vesicles Golgi, demon- <br /> significance. For example, yip1-4 does synthetic        strated recently Uso1p second <br /> lethal effects sec4-8 (a Golgi-to-plasma membrane-          function cargo sorting ER (Allan et al. 2000; <br /> acting Rab protein), Yip1p            Morsomme Riezman 2002; Kondylis Rabou- <br /> be capable physical association Sec4p cellular        ille 2003). Thus cargo sorting, vesicle budding, tar- <br /> lysates (Calero et al. 2003).                                    geting, fusion vivo interconnected <br />    Our data indicating Yip1p factor interacts     previously appreciated, providing precedent for <br /> with Golgi Rab proteins suggest site action          Yip1p functionality Golgi ER mem- <br /> of Yip1p Golgi complex. Previous studies          branes. <br /> demonstrated defects Yip1p disrupt ER vesicle               Another possible hypothesis reconcile role of <br /> biogenesis (Heidtman et al. 2003), conclusion          Yip1p ER vesicle biogenesis Golgi Rab action is <br /> supported study novel yip1 mutant alleles           propose Yip1p mediates interactions between <br /> (Table 5). These results suggest different models            SNARE Rab proteins. However, genetic <br /> of Yip1p function, illustrated Figure 7. One model            interactions Yip1p various v- t-SNARE <br /> is Yip1p operates independently COPII               proteins (sec22 ts, bet1-1, bos1-1, sec5 ts) operating ER/ <br /> vesicle biogenesis Golgi Rab proteins. Another               Golgi transport genes involved <span id='am-97' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-98' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-99'  ></span><span id='am-100' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>cis</span> SNARE disas- <br /> model Yip1p acts common pathway                sembly (sec17 ts sec18-1) Yip1p been <br /> involves COPII vesicle proteins Golgi Rab pro-          reported physically interact v-SNARE Bos1p <br /> teins. We favor model reasons.            vitro (Barrowman et al. 2003). It possible Yip1p <br /> First, extensive mutagenesis spanning entire        &#8220;piggyback&#65533;? SNARE proteins facilitate its <br /> Yip1p protein, unable generate condi-             transport ER Golgi. No known COPII <br /> tional allele selectively defective Golgi     uptake signals apparent inspection primary <br /> Rab COPII gene interaction. Second, vitro assays           amino acid sequence Yip1p recent discover- <br /> comparing membranes yip1-40 yip1-42 cells,              ies demonstrating COPII coat multiple inde- <br /> two mutant alleles share mutated residue           pendent sites cargo recognition (Miller et al. 2003; <br /> (E70G K130A case yip1-40 E70V              Mossessova et al. 2003) opens possibility that <br /> the case yip1-42) indicate degree severity        sorting signals remain identified. It should <br /> of ER vesicle biogenesis defect correlates positively        stressed speculative models, gaps <br /> with ability interact Golgi Rab proteins          remain understanding YIP1 family function, <br /> yeast hybrid.                                                studies necessary resolve poten- <br />    It cycling Yip1p           tial modes Yip1p action. <br /> compartments critical function. Perhaps Golgi-           Because Yip1p requires Rab proteins prenylated <br /> localized Rab GTPases transmit signal Yip1p, signal       productive interactions defective alleles yip1 <br /> that Yip1p carries ER. This signal turn           cause ER membrane accumulation, possi- <br /> is required continued flow membrane               bility Yip1p influences membrane traffic through <br /> material Golgi ER. Thus Yip1p            dysregulated lipid metabolism. However, genetic in- <br /> affect ER vesicle budding Yip1p          teractions observed yip1-4 opi1&#9004; or <br /> of minimal machinery COPII vesicles created           sec14 ts activated HAC1 unable suppress yip1-4 <br /> vitro recombinant components (Matsuoka et al.               currently evidence Yip1p plays role <br /> 1998). In model, Yip1p required overall ho-          interface protein lipid biogenesis <br /> meostasis anterograde retrograde membrane-            secretory pathway involved membrane <br /> trafficking pathways, explaining appears in-           proliferation. <br /> teract Golgi Rab proteins regulate diverse                A recent study identified mammalian Yip3 as <br /> membrane-trafficking pathways. It possible          GDI-displacement factor Rab GTPase Rab9 <br /> Yip1p physically recycle Rab protein           (Sivars et al. 2003). The yeast homolog Yip3 no <br /> ER where, Rab protein does participate          sequence relationship Yip1p; however, does share <br /> in vesicle biogenesis, important component              physical characteristics role Yip1p as <br /> vesicle targeting fusion, incorporated            possible GDI-displacement factor provocative. <br /> into budding vesicle downstream events. This          Because Yip1p (i) Rab-binding membrane protein, <br /> consistent model suggested Barrowman et              (ii) recognizes prenyl groups Rab proteins <br /> al. (2003), propose Yip1p required           (iii) apparent nucleotide preference for <br /> fusion competence ER-derived vesicles, makes           nucleotide-bound state Rab, previously <br /> sense vesicle budding linked &#8220;pro-          proposed Yip1p participate membrane <br /> gramming&#65533;? targeting fusion. In respect,             retrieval pathway Rab proteins cytosol. How- <br /> there appears parallelism Yip1p              ever, apparent shift membrane/cytosolic <br /> Uso1p. Although Uso1p mammalian homolog                  ratios Rab proteins mild mutant allele yip1 <br />  1840                                                         C. Z. Chen et al. <br />  <br /> is examined subcellular fractionation (Barrowman                         p115 cis-SNARE complex: programming budding COPII <br />                                                                             vesicles fusion. Science 289: 444&#8211;448. <br /> et al. 2003), yip1-4 allele                 Alory, C., W. E. Balch, 2001 Organization Rab-GDI/ <br /> to mislocalize GFP-Ypt1p shift restrictive tem-                     CHM superfamily: functional basis choroideremia disease. <br /> perature (Calero et al. 2003). In study,                      Traffic 2: 532&#8211;543. <br />                                                                         Baker, D., L. Hicke, M. Rexach, M. Schleyer R. Schekman, <br /> do demonstrate Rab proteins required                        1988 Reconstitution SEC gene product-dependent intercom- <br /> the physiological function Yip1p, demon-                     partmental protein transport. Cell 54: 335&#8211;344. <br /> strate complete synthetic lethality yip1-4                Barlowe, C., 1997 Coupled ER Golgi transport reconstituted <br />                                                                             purified cytosolic proteins. J. Cell Biol. 139: 1097&#8211;1108. <br /> mutant Rab-GDI, gdi1-11. Taken together, data                  Barlowe, C., 2002a COPII-dependent transport endoplas- <br /> support <span id='am-65' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-66' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-68'  ></span><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>idea</span> Yip1p Rab-GDI participate                      mic reticulum. Curr. Opin. Cell Biol. 14: 417&#8211;422. <br /> similar processes consistent Yip1p                    Barlowe, C., 2002b Three-dimensional structure COPII pre- <br />                                                                             budding complex. Dev. Cell 3: 467&#8211;468. <br /> potential candidate participate process Rab                Barrowman, J., W. Wang, Y. Zhang S. Ferro-Novick, 2003 <br /> membrane attachment detachment,                            The Yip1p.Yif1p complex required fusion competence <br /> remains proven. Yip1p ER recruit                           endoplasmic reticulum-derived vesicles. J. Biol. Chem. 278: <br />                                                                             19878&#8211;19884. <br /> the Ypt1p required cargo sorting (Morsomme                      Belden, W. J., C. Barlowe, 2001 Deletion yeast p24 genes <br /> Riezman 2002) perform similar task Golgi                       activates unfolded protein response. Mol. Biol. Cell 12: 957&#8211; <br /> for Ypt6p Ypt31/32p. Although clear Rab                      969. <br />                                                                         Benli, M., F. Doring, D. G. Robinson, X. Yang D. Gallwitz, <br /> proteins transit membranes cytosol,                       1996 Two GTPase isoforms, Ypt31p Ypt32p, essential <br /> may complex cycle membrane organ-                          Golgi function yeast. EMBO J. 15: 6460&#8211;6475. <br /> elle engagement previously envisaged. For exam-                    Bivona, T. G., M. R. Philips, 2003 Ras pathway signaling on <br />                                                                             endomembranes. Curr. Opin. Cell Biol. 15: 136&#8211;142. <br /> ple, Ypt1p recruited Golgi membranes                      Calero, M., G. R. Whittaker R. N. Collins, 2001 Yop1p, the <br /> from cytosol, travel retrograde vesicles                     yeast homolog polyposis locus protein 1, interacts with <br />                                                                             Yip1p negatively regulates cell growth. J. Biol. Chem. 276: <br /> the ER, released membranes                              12100&#8211;12112. <br /> again ER-derived vesicle carrying Ypt1p docks                 Calero, M., N. J. Winand R. N. Collins, 2002 Identification <br /> at Golgi. An analogy processes suggested                   novel proteins Yip4p Yip5p Rab GTPase interacting <br />                                                                             factors. FEBS Lett. 515: 89&#8211;98. <br /> in case small GTPase Ras,                     Calero, M., C. Z. Chen, W. Zhu, N. Winand, K. A. Havas et al., 2003 <br /> recently appreciated complex intracellular                        Dual prenylation required rab protein localization and <br /> trafficking pathway used different regula-                  function. Mol. Biol. Cell 14: 1852&#8211;1867. <br />                                                                         Cao, X., C. Barlowe, 2000 Asymmetric requirements Rab <br /> tory purposes (Bivona Philips 2003). In                          GTPase SNARE proteins fusion COPII vesicles with <br /> scenario, different Rab-interacting factors                   acceptor membranes. J. Cell Biol. 149: 55&#8211;66. <br /> influence organelle recruitment Rab proteins                     Cao, X., N. Ballew C. Barlowe, 1998 Initial docking ER- <br />                                                                             derived vesicles requires Uso1p Ypt1p independent of <br /> (Calero et al. 2003) different stages round                      SNARE proteins. EMBO J. 17: 2156&#8211;2165. <br /> of membrane transport.                                                  Carman, G. M., S. A. Henry, 1999 Phospholipid biosynthesis <br />    Further biochemical experiments required                      yeast <span id='am-24' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-25' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-35'  ></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> interrelationship with <br />                                                                             metabolic processes. Prog. Lipid Res. 38: 361&#8211;399. <br /> understand precise impact Yip1p Rab                       Chothia, C., 1976 The nature accessible buried surfaces <br /> GTPase cycle, direct indirect nature inter-                   proteins. J. Mol. Biol. 105: 1&#8211;12. <br /> actions Yip1p Rab proteins, relative                Collins, R. N., 2003 Rab ARF GTPase regulation exocytosis. <br />                                                                             Mol. Membr. Biol. 20: 105&#8211;115. <br /> role Rab-GDI. Our studies provide platform direct               Cormack, B. P., G. Bertram, M. Egerton, N. A. Gow, S. Falkow <br /> future questions mutations provide                      et al., 1997 Yeast-enhanced green fluorescent protein (yEGFP): <br /> tools continued genetic biochemical studies                         reporter gene expression <span id='am-51' about='obo:NCBITaxon_5475' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_118267 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-52' property="rdfs:label" content="Candida" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-54'  ></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasRelatedSynonym" content="Torulopsis" datatype="xsd:string"></span>Candida</span> albicans. Microbiology <br />                                                                             143 (Pt 2): 303&#8211;311. <br /> aimed understanding precise function Yip1p                    Du, L. L., P. Novick, 2001 Yeast Rab GTPase-activating protein <br /> in cells.                                                                   Gyp1p localizes Golgi apparatus negative regulator <br />                                                                             Ypt1p. Mol. Biol. Cell 12: 1215&#8211;1226. <br />    Many thanks Pat Brennwald, Chris Burd, Chris Kaiser, Susan     Fersht, A., 1999 Structure Mechanism Protein Science: A Guide to <br /> Henry, Kazutoshi Mori, Nava Segev, Lois Weisman generously          Enzyme Catalysis Protein Folding. W. H. Freeman, San Francisco. <br /> providing strains Kristin Cerione experimental advice.       Finger, F. P., P. Novick, 2000 Synthetic interactions the <br /> C.Z.C. participant Cornell Undergraduate Biology Honors         post-Golgi sec mutations Saccharomyces cerevisiae. Genetics 156: <br /> Program. M.C. recipient Army Predoctoral Fellowship            943&#8211;951. <br /> DAMD17-00-1-0218. C.J.D. supported National Institutes         Garcia-Ranea, J. A., A. Valencia, 1998 Distribution func- <br /> Health (NIH) grant T32 RR07059-08. This work supported           tional diversification Ras superfamily Saccharomyces <br />                                                                             cerevisiae. FEBS Lett. 434: 219&#8211;225. <br /> U.S. Department Agriculture Animal Health Disease Research <br />                                                                         Gilbert, P. M., C. G. Burd, 2001 GDP dissociation inhibitor <br /> Program, American Heart Association grant no. 0030316T, Na-             domain II required Rab GTPase recycling. J. Biol. Chem. 276: <br /> tional Science Foundation grant no. MCB-0079045 R.C. NIH             8014&#8211;8020. <br /> grant no. GM52549 C.B.                                               Hartman, J. L. IV, B. Garvik L. Hartwell, 2001 Principles <br />                                                                             buffering genetic variation. Science 291: 1001&#8211;1004. <br />                                                                         Heidtman, M., C. Z. Chen, R. N. Collins C. Barlowe, 2003 <br />                                                                             A role Yip1p COPII vesicle biogenesis. J. Cell Biol. 163: <br />                     LITERATURE CITED                                        57&#8211;69. <br />                                                                         Henry, S. A., J. L. Patton-Vogt, 1998 Genetic regulation of <br /> Aguilera, A., 1994 Formamide sensitivity: novel conditional phe-          phospholipid metabolism: yeast model eukaryote. Prog. Nu- <br />     notype yeast. Genetics 136: 87&#8211;91.                                   cleic Acid Res. Mol. Biol. 61: 133&#8211;179. <br /> Allan, B. B., B. D. Moyer W. E. Balch, 2000 Rab1 recruitment        Higashio, H., K. Kohno, 2002 A genetic link the <br />                                     Yip1p Function S. cerevisiae Requires Rab Proteins Rab-GDI                                       1841 <br />      unfolded protein response vesicle formation endo-       Olofsson, B., 1999 Rho guanine dissociation inhibitors: pivotal mol- <br />      plasmic reticulum. Biochem. Biophys. Res. Commun. 296: 568&#8211;              ecules cellular signalling. Cell. Signal. 11: 545&#8211;554. <br />      574.                                                                 Pfeffer, S. R., 2001 Rab GTPases: specifying deciphering organ- <br /> Jayasinghe, S., K. Hristova S. H. White, 2001 Energetics,                 elle identity function. Trends Cell Biol. 11: 487&#8211;491. <br />      stability, predict <br /> </body></html>