<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#63193; 2001 Genetics Society America <br />  <br />  <br />  <br /> Yeast Frameshift Suppressor Mutations Genes Coding Transcription <br />     Factor Mbf1p Ribosomal Protein S3: Evidence Autoregulation <br />                              S3 Synthesis <br />  <br />             James L. Hendrick, <span id='am-129' about='obo:NCBITaxon_344662' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_344711 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_42315 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-130'  ></span><span id='am-131' property="oboInOwl:hasRelatedSynonym" content="Patricia Fox 1940" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-134' property="rdfs:label" content="Patricia" datatype="xsd:string"></span>Patricia</span> G. Wilson,1 Irving I. Edelman, Mark G. Sandbaken, <br />                                <span id='am-6' about='obo:NCBITaxon_146270' typeof='owl:Thing obo:NCBITaxon_6514 obo:NCBITaxon_33208 obo:NCBITaxon_216331 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_146272 obo:NCBITaxon_33154 obo:NCBITaxon_680346 obo:NCBITaxon_1206795 obo:NCBITaxon_70849 obo:NCBITaxon_216305 obo:NCBITaxon_216307 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_6072'><span id='am-7' property="rdfs:label" content="Doris" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Doris</span> Ursic Michael R. Culbertson <br />                       Laboratories Genetics Molecular Biology, University Wisconsin, Madison, Wisconsin 53706 <br />                                                       Manuscript received July 26, 2000 <br />                                                   Accepted publication December 1, 2000 <br />  <br />  <br />                                                              ABSTRACT <br />                 The SUF13 SUF14 genes identified extragenic suppressors &#11001;1 frameshift mutations. <br />              SUF13 synonymous MBF1, single-copy nonessential gene coding POLII transcription factor. <br />              The suf13-1 mutation two-nucleotide deletion SUF13/MBF1 coding region. A suf13::TRP1 null <br />              mutant suppresses &#11001;1 frameshift mutations, indicating suppression caused loss SUF13 function. <br />              The suf13-1 suppressor alters sensitivity aminoglycoside antibiotics reduces accumulation of <br />              his4-713 mRNA, suggesting suppression mediated translational level. The SUF14 gene is <br />              synonymous RPS3, single-copy essential gene codes ribosomal protein S3. The suf14-1 <br />              mutation missense substitution coding region. Increased expression S3 limits accumulation <br />              SUF14 mRNA, suggesting expression autoregulated. A frameshift mutation SUF14 prevents <br />              full-length translation eliminated regulation, indicating S3 required regulation. Using CUP1- <br />              SUF14 SUF14-lacZ fusions, run-on transcription assays, estimates mRNA half-life, results <br />              transcription plays minor role regulation 5&#11032;-UTR necessary not <br />              sufficient regulation. A change mRNA decay rate primary mechanism regulation. <br />  <br />  <br />                                                                          ations lead suppression &#11001;1 frameshift muta- <br /> S   UPPRESSORS frameshift nonsense muta- <br />       tions long used bacteria, phage, and <br /> yeast identify RNAs proteins important in <br />                                                                          tions. SUF16 <span id='am-16' about='obo:NCBITaxon_3846' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_163735 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-17' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-20'  ></span>glycine</span> tRNA suppressors represent <span id='am-45' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-46' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> <br />                                                                          tRNA contains extra nucleotide the <br /> the accuracy translation. In yeast <span id='am-38' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-39'  ></span><span id='am-40' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span>Saccharomyces</span>                  anticodon loop loop extended from <br /> cerevisiae, frameshift suppressors distributed 25 differ-             seven unpaired nucleotides. By analyzing sup- <br /> ent loci (SUF1 SUF25) identified previ-                  pression using possible combinations four-base <br /> ous studies laboratory (Culbertson et al. 1977,                   anticodons four-base glycine codons, found <br /> 1980, 1990; Cummins et al. 1980). Suppressors mapping                    base-pairing fourth nucleotide required <br /> at 22 loci exhibited allele-specific suppression             suppression (Gaber Culbertson 1984). SUF8 <br /> &#11001;1G:C insertions glycine proline codons (Gaber et                  proline tRNA suppressors result base substitutions <br /> al. 1983; Mathison Culbertson 1985). Among                           positions 31 39 anticodon stem SUF8 <br /> these, 16 suppressor genes shown encode isoac-                   tRNA (Cummins et al. 1985; Mathison et al. 1989). The <br /> ceptors glycine tRNA, including 2 glycine                        mutations disrupt base-pairing base pair of <br /> tRNACCC, 3 glycine tRNAUCC, 11 glycine                       anticodon stem, resulting mature tRNA with <br /> tRNAGCC (Gaber Culbertson 1982; Mendenhall                           novel secondary structure consisting four-base <br /> et al. 1987; Ball et al. 1988; Mendenhall Culbert-                   anticodon stem nine-base anticodon loop. A <br /> son 1988). Six suppressor genes shown encode                     model based alternate three-dimensional base stack- <br /> isoacceptors proline tRNA, including 4 proline                   ing conformations tRNA originally proposed to <br /> tRNAUGG, 2 proline tRNAIGG (Cummins et al.                       explain tRNAs unpaired nucleo- <br /> 1982, 1985; Winey et al. 1989).                                          tides anticodon loop cause four-base trans- <br />    Detailed analyses yeast suppressors,                    location ribosome (Curran Yarus 1987; <br /> SUF16 (glycine tRNAIGG) SUF8 (proline tRNAUGG),                      Culbertson et al. 1990). A recent interpretation <br /> revealed different kinds tRNA structural alter-                   based new information frameshift suppressor <br />                                                                          tRNAs instead cause slippage peptidyl-tRNA by <br />                                                                          mechanism similar proposed programmed <br />   Corresponding author: Michael R. Culbertson, R.M. Bock Labs, 1525      &#11001;1 frameshifting (Farabaugh 2000). <br /> Linden Dr., University Wisconsin, Madison, WI 53706. <br /> E-mail: mrculber@facstaff.wisc.edu <br />                                                                             In <span id='am-87' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-88' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-98'  ></span><span id='am-99' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span>Escherichia coli</span>, suppressor mutations ribosomal <br />   1 <br />    Present address: Department Biology, Georgia State University,     RNA regions small ribo- <br /> <span id='am-1' about='obo:NCBITaxon_649065' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_649145 obo:NCBITaxon_69555 obo:NCBITaxon_413196 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-2' property="rdfs:label" content="Atlanta" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-5'  ></span>Atlanta</span>, GA 30303.                                                       somal subunit associated ribosomal pro- <br /> Genetics 157: 1141&#8211;1158 (March 2001) <br />  1142                                                 J. L. Hendrick et al. <br />  <br /> teins known affect translational accuracy mu-             act bridge transcription factors and <br /> tated (S4, S5, S12; Oakes et al. 1990; Lodmell            TATA-binding protein. Despite primary role tran- <br /> Dahlberg 1997). Similar rRNA mutations                  scription, present evidence mutations the <br /> created S. cerevisiae, studies indicate trans-      SUF13 gene suppress frameshift mutations transla- <br /> lational accuracy eukaryotes complex interplay            tional mechanism. SUF14 single-copy essential gene <br /> between rRNA, ribosomal proteins, translation factors,            codes ribosomal protein S3 (Fingen-Eigen et <br /> and tRNAs brought three-dimen-               al. 1996). Evidence presented mutations SUF14 <br /> sional space ribosome (Chernoff et al. 1994).              cause frameshift suppression altering translational <br />    In S. cerevisiae, mutations 3 original 25            accuracy ribosomes. <br /> frameshift suppressor genes (suf12, suf13, suf14)                We examined expression SUF13 SUF14 <br /> exhibited patterns suppression typical             genes using changes gene dosage <span id='am-11' about='obo:NCBITaxon_189528' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_189527 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_9219 obo:NCBITaxon_436491'><span id='am-12'  ></span><span id='am-13' property="rdfs:label" content="Indicator" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>indicator</span> of <br /> tRNA suppressors described (Culbertson et al.               regulated expression. Whereas SUF13 failed exhibit <br /> 1982). These suppressors recessive semidomi-              altered expression gene dosage increased, <br /> nant allele-specific like tRNA suppres-          expression SUF14 regulated. <br /> sors. Mutations suf12 broadest spectrum             We present evidence ribosomal protein S3, the <br /> suppression, acting &#11001;1 insertions glycine pro-          product SUF14, inhibits synthesis limiting <br /> line codons UAG UGA nonsense al-                accumulation SUF14 mRNA. Using SUF14lacZ <br /> leles. The suf12 gene code GTP-                promoter fusion, run-on transcription assays, esti- <br /> binding protein significant structural similarity         mates mRNA half-life, results suggest the <br /> the elongation factor EF-1A (Wilson Culbertson                5&#11032; untranslated region (5&#11032;-UTR) necessary not <br /> 1988). suf12 synonymous sup35, codes            sufficient regulation. Transcription appears play <br /> translation termination factor eRF3 (Zhouravieva et               minor role regulation, S3-mediated <br /> al. 1995). Mutations suf12 presumably suppress non-            changes SUF14 mRNA decay rate the <br /> sense mutations promoting readthrough prema-              primary mechanism regulation. <br /> ture stop <span id='am-71' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-72' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-74'  ></span><span id='am-75' property="rdfs:label" content="Codon" datatype="xsd:string"></span>codon</span>. It clear suf12 mutations <br /> cause frameshift suppression, requires change <br /> in decoding elongation correct reading                             MATERIALS AND METHODS <br /> frame.                                                               Strains, media, reagents: The S. cerevisiae strains used <br />    In S. cerevisiae, frameshift/nonsense suppressor muta-         study listed Table 1. <span id='am-150' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-151' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-153' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-154' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Plasmids</span> listed Table <br /> tions identified previously TEF2 gene,           2. Standard yeast genetic techniques described <br /> which encodes elongation factor EF-1A (Sandbaken              (Sherman 1991). Yeast transformation performed elec- <br /> and Culbertson 1988). Similar mutations                 troporation (Grey Brendel 1992). Media growth <br />                                                                   S. cerevisiae described Gaber Culbertson <br /> reported E. coli (Hughes et al. 1987; Vijgenboom <br />                                                                   (1982). Nomenclature media follows: SC, synthetic <br /> and Bosch 1989). EF-1A directly involved insuring           complete; SC-ura, synthetic complete-uracil omitted; SC-ura/ <br /> translational accuracy, involve kinetic            leu, synthetic complete uracil leucine omitted. <br /> proofreading (Thompson et al. 1986) allosteric in-            Growth assayed plating serial dilutions cells from <br /> teractions aminoacyl-tRNA ribosomal A              cell cultures described Shirley et al. (1998). Bacterial <br /> site E-site bound deacyl-tRNA (Nierhaus 1990).                strains used cloning strain MC1066a (leuB600, <br />                                                                   trpC9830, pyrF74::Tn5, kanr, <span id='am-55' about='obo:NCBITaxon_9225' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_9224 obo:NCBITaxon_117571 obo:NCBITaxon_9223 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-58'  ></span><span id='am-59' property="rdfs:label" content="Ara" datatype="xsd:string"></span>ara</span>, hsdS, hsdM&#11001;, recA13; Sandba- <br /> Mutations EF-1A affect &#11001;1 frameshifting               ken Culbertson 1988) strain 6507 (leu&#11002; pro&#11002; <br /> amino acid misincorporation, possibly affecting            pyr23::Tn5 recA rK mK KanR). Paromomycin sulfate, cyclohexi- <br /> rate GTP hydrolysis EF-1A:GTP:aminoacyl-tRNA                mide, hygromycin B, streptomycin obtained from <br /> ternary complexes, critical feature kinetic         Sigma (St. Louis). Geneticin (G418) obtained Boeh- <br /> proofreading, possibly affecting binding affin-         ringer Mannheim (Indianapolis). Enzymes obtained <br />                                                                   New England Biolabs (Beverly, MA) Promega (Madi- <br /> ity EF-1A ribosomal components A site.               son, WI). <br /> Recessive mutations yeast TEF2 identified                DNA/RNA methods: Plasmid DNA isolated described <br /> in screens yielded 25 frameshift suppressor          Maniatis et al. (1982). Yeast genomic plasmid DNA was <br /> genes described genetic redundancy.              prepared method Sherman et al. (1982). Southern <br /> EF-1A encoded nontandem repeated genes,                 analysis performed described Maniatis et al. (1982) <br />                                                                   modified Van tol et al. (1987). DNA probes isolated <br /> TEF1 TEF2 (Schirmaier Philippsen 1984). <br />                                                                   restriction fragments random primer kit <br />    In article suf13 suf14 genes          (Pharmacia, Piscataway, NJ). Deoxyoligonucleotides 5&#11032;- <br /> and genes code proteins. We               end labeled polynucleotide kinase (New England Bio- <br /> have determined sequences frameshift muta-             labs). <br /> tions, leu2-3 met2-1, suppressed               DNA sequenced using ABI sequenase kit (Perkin <br /> by mutations SUF12, SUF13, SUF14 genes                 Elmer, Norwalk, CT) ABI 377 automated fluorescent <br />                                                                   sequencer. Universal sequencing primers obtained from <br /> (Culbertson et al. 1982). SUF13 single-copy nones-           New England Biolabs. Additional oligonucleotides (Operon <br /> sential gene codes transcriptional activator         Technologies, Inc., Alameda, CA) used primers where <br /> Mbf1 (Takemaru et al. 1998), shown                 necessary complete sequencing. Sequence analysis <span id='am-145' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-146' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-147'  ></span><span id='am-148' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span>- <br />                                                  Frameshift Suppressors Yeast                                            1143 <br />  <br />                                                            TABLE 1 <br />                                                             Strains <br />  <br />          Strain                                                   Genotype <br />          IEY73         MAT&#65533;?&#65533; leu2-3 ura3-52 <br />          1160          MATa leu2-3,leu2-112 his4-713 met2-1 ura3-52 lys2 <br />          JHY1a         MATa suf13-1 ura3-52 leu2-3 his4-713 trp1-1 <br />          JHY1a         MATa/MAT&#65533;?&#65533; SUF13/suf13-1 ura3-52/ura3-52 leu2-3/leu2-3 met2-1/met2-1 <br />          JHY2          MATa/MAT&#65533;?&#65533; ura3-52/ura3-52 leu2-&#9004;1/leu2-&#9004;1 trp1-&#9004;1/trp1-&#9004;1 lys2-801/lys2-801 his3&#9004;200/ <br />                         his3&#9004;200 ade2-101/ade2-101 SUF13/suf13::TRP1 <br />          JHY3b         MATa SUF13::TRP1 ura3-52 leu2-&#9004;1 trp1-&#9004;1 lys2-801 his3&#9004;200 ade2-101 <br />          JHY6c         MATa suf14-1 ura3-52 leu2-3 his4-713 trp1-1 <br />          JHY13d        MATa rpb1-1 upf1::URA3 ura3-52 leu2-3 <br />          JHY14e        MATa/MAT&#65533;?&#65533; ura3-52/ura3-52 leu2-&#9004;1/leu2-&#9004;1 trp1-&#9004;1/trp1-&#9004;1 lys2-801/lys2-801 his3&#9004;200/ <br />                         his3&#9004;200 ade2101/ade2-101 SUF14/suf14::TRP1 <br />          JHY16c        MATa suf14::TRP1 ura3-52 leu2-&#9004;1 trp1-&#9004;1 lys2-801 his3&#9004;200 ade2-1 [YCpCUP1pSUF14/2] <br />          JHY16e        MATa suf14::TRP1 ura3-52 leu2-&#9004;1 trp1-&#9004;1 lys2-801 his3&#9004;200 ade2-1 [YCpSUF14/2] <br />          JHY16f        MATa suf14::TRP1 ura3-52 leu2-&#9004;1 trp1-&#9004;1 lys2-801 his3&#9004;200 ade2-1 [YEpSUF14/5] <br />          JHY21f        MATa ura3-52 upf1::URA3 leu-3,-112 trp1-1 <br />          JHY22g        MATa ura3-52 leu2-3,-112 trp1-1 <br />          JHY23h        MATa ura3-52 leu2-3,-112 trp1-1 <br />          JH85.9i       MATa suf14-1 his4-713 ura3-52 trp1-1 <br />          YPH274j       MATa/MAT&#65533;?&#65533; ura3-52/ura3-52 leu2-&#9004;1/leu2-&#9004;1 trp1-&#9004;1/trp1-&#9004;1 lys2-801/lys2-801 his3&#9004;200/ <br />                         his3&#9004;200 ade2-101/ade2-101 <br />          YPH501j       MATa/MAT&#65533;?&#65533; ura3-52/ura3-52 leu2-&#9004;1/leu2-&#9004;1 trp1-&#9004;63/trp1-&#9004;63 lys2-801/lys2-801 his3&#9004;200/ <br />                         his3&#9004;200 ade2-101/ade2-101 <br />            a <br />              HY1b contains plasmid YCp50. JHY1c contains YEpSUF13. <br />            b <br />              JHY3a contains plasmid YCpSUF13. JHY3b contains YCpsuf13-1. JHY3c contains pRS316. <br />            c <br />              JHY6a contains plasmid YCp50. JHY6b contains YCpSUF14. <br />            d <br />              JHY13a contains plasmid YEp351. JHY13b contains YEpSUF14/5. <br />            e <br />              The genotype JHY15 referred text identical JHY14. <br />            f <br />              Transformants strain PLY102 (Leeds et al. 1992) renamed JHY21a-e. JHY21a contains plasmid <br />          PRS315. JHY21b contains YEpSUF14/5. JHY21c contains YEpSUF14/6. JHY21d contains YEpsuf14fs/2. JHY21e <br />          contains YEpSUF13/3. <br />            g <br />              HY22b contains plasmids, pRS315 YEpSUF14placZ. JHY22d contains plasmids, YEpSUF14/5 <br />          YEpSUF14placZ. <br />            h <br />              JHY23a contains plasmid pRS314. JHY23b contains YCpCUP1-SUF14/1. JHY23c contains YEpCUP1- <br />          pSUF14/1. <br />            i <br />              JH85.9a contains plasmid YCp50. JH85.9b contains wild-type SUF14 plasmid pW14.1. <br />            j <br />              YPH274 YPH501 obtained P. Hieter (Sikorski Hieter 1989). YPH274a identical to <br />          YPH274 contains SUF14/suf14:TRP1/SUF14. <br />  <br />  <br /> base searches sequence similarities performed using      Northern blots analyzed Molecular Dynamics (Sun- <br /> the National Center Biotechnology Saccharomyces       nyvale, CA) phosphorImager. <br /> Genome Database.                                                     Run-on transcription assays performed using perme- <br />    To measure accumulation his4-713 mRNA North-         abilized cells presence [32&#65533;?&#65533;]UTP described by <br /> ern analysis, cells grown SC-ura containing histidine     Warner (1991). Labeled RNA transcripts whole-cell <br /> and washed resuspended SC-ura medium lacking          reactions detected hybridization 2 &#65533;?&#65533;g target <br /> histidine. RNA isolated hot phenol extraction (Leeds       DNA bound GeneScreen Plus (DuPont) using hybridization <br /> et al. 1991) time resuspension 3 hr        conditions identical used Northern blotting. <br /> incubation medium lacking histidine. The his4-713 mRNA            Decay rates mRNA determined using methods <br /> was probed using 2234-bp XhoI/XbaI DNA fragment            Herrick et al. (1990) Parker et al. (1991) which <br /> was radiolabeled using random-primed DNA labeling kit           POLII transcription blocked shifting strain carrying <br /> (Boehringer Mannheim). A total 15 &#65533;?&#65533;g RNA fraction-      rpb1-1 restrictive temperature 37&#11034;. The rate mRNA <br /> ated 1% agarose/formaldehyde gel transferred          decay monitored Northern blotting. Decay curves gen- <br /> GeneScreen Plus (DuPont, Wilmington, DE; Ursic et al.             erated using Cricket Graph software plotted per- <br /> 1997). The ACT1 (actin) transcript used loading control      centage RNA remaining compared t &#11005; 0 vs. time. Lines <br /> was detected radiolabeled actin-specific 1144-bp XhoI/     generated using least-squares fit. <br /> KpnI DNA fragment probe. All Northern blotting              Analysis leu2-3 met2-1 mutations: Plasmid YIp26- <br /> experiments carried described Maniatis et al.      LEU2, containing yeast LEU2 URA3 genes, cleaved <br /> (1982) modified Van tol et al. (1987). The actin tran-      unique KpnI site LEU2 coding sequence. The <br /> script detected using oligonucleotide 5&#11032;-TGTTAATT         linear plasmid used transform strain IEY73, <span id='am-124' about='obo:NCBITaxon_201850' typeof='owl:Thing obo:NCBITaxon_1125798 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71529 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-125' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-127'  ></span><span id='am-128' property="rdfs:label" content="Car" datatype="xsd:string"></span>car</span>- <br /> CAGTAAATTTTC-3&#11032;, complementary sequence             ries leu2-3 mutation, Ura&#11001; phenotype site-directed <br /> in ACT1 open reading frame (ORF). All Southern            integration LEU2 locus. All Ura&#11001; transformants were <br />  1144                                                     J. L. Hendrick et al. <br />  <br />                                                                TABLE 2 <br />                                                                Plasmids <br />  <br />          Plasmid                             Vectora                      Yeast genes                     Commentsb <br />          YIp26-LEU2                      YIp26                      URA3 LEU2                        Library clone <br />          YIp26-leu2-3                    YIp26                      URA3 leu2-3                      Cloned gap repair <br />          YCp50-MET2                      YCp50                      CEN4 URA3 MET2                   Library clone <br />          YCp50-met2-1                    YCp50                      CEN4 URA3 met2-1                 Cloned gap repair <br />          pJH13.1                         YCp50                      CEN4 URA3 SUF13                  Library clone <br />          YCpSUF13                        pRS316                     CEN6 URA3 SUF13                  HindIII-HindIII <br />          YIpSUF13                        YIp5                       URA3 SUF13                       HindIII-HindIII <br />          YEpSUF13/1                      YEp352                     2&#65533;?&#65533;o URA3 SUF13                   SphI-PvuII <br />          YEpSUF13/2                      YEpSUF13/1                 2&#65533;?&#65533;o URA3 SUF13                   EcoRI-EcoRI deletion <br />          YEpSUF13/3                      YEp351                     2&#65533;?&#65533;o LEU2 SUF13                   HindIII-HindIII <br />          YEpsuf13::TRP1                  YEpSUF13/2                 2&#65533;?&#65533;o LEU2 SUF13                   TRP1 insertion <br />          PW14.1                          YCp50                      CEN4 URA3 SUF14                  Library clone <br />          YEp352&#9004;R15                      YEp352                     2&#65533;?&#65533;o URA3                         EcoRI site filled in <br />          YEp3352&#9004;R1H                     YEp352                     2&#65533;?&#65533;o URA3                         HindIII site filled in <br />          YEpSUF14B                       YEp352&#9004;R1H                 2&#65533;?&#65533;o URA3 SUF14                   SphI-DraI <br />          YEPSUF14/1                      YEp352                     2&#65533;?&#65533;o URA3 SUF14                   SphI-KpnI <br />          YEpSUF14/2                      pRS315                     CEN6 LEU2 SUF14                  SphI-PvuII <br />          YEpSUF14/4                      YEp352&#65533;?&#65533;R1H                 2&#65533;?&#65533;o URA3 SUF14                   SphI-DraI <br />          YEpSUF14/5                      YEp351                     2&#65533;?&#65533;o LEU2 SUF14                   SphI-KpnI <br />          YEpSUF14/6                      YEp351                     2&#65533;?&#65533;o LEU2 SUF14                   BglII-BglII <br />          YEpsuf14fs/1                    YEpSUF14/4                 2&#65533;?&#65533;o URA3 suf14fs                 SphI-DraI <br />          YEpsuf14fs/2                    YEp351                     2&#65533;?&#65533;o LEU2 suf14fs                 SphI-KpnI <br />          YCpCUP1-SUF14/1                 pRS314                     CEN6 TRP1 CUP1-SUF14             SphI-Kpn1 <br />          YCpCUP1-SUF14/2                 pRS315                     CEN6 LEU2 CUP1-SUF14             SphI-PvuII <br />          YEpCUP1-SUF14/1                 pRS424                     2&#65533;?&#65533;o TRP1 CUP1-SUF14              SphI-Kpn1 <br />          YEpCUP1-SUF14/2                 YEp351                     2&#65533;?&#65533;o LEU2 CUP1-SUF14              SphI-PvuII <br />          YEpSUF14-lacZ                   YEp352                     2&#65533;?&#65533;o URA3 SUF14-lacZ              Promoter fusion <br />             a <br />               YIp5 YIp26 described Botstein et al. (1979), YCp50 Rose et al. (1987), YEp351 YEp352 <br />          Hill et al. (1986), pRS315 pRS316 Sikorski Hieter (1989), pRS424 Christianson et <br />          al. (1991). <br />             b <br />               Locations restriction sites SUF13 SUF14 shown Figures 1 3, respectively. <br />  <br />  <br /> Leu&#11002; phenotype, suggesting integrant <span id='am-155' about='obo:NCBITaxon_237249' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_30083 obo:NCBITaxon_33154 obo:NCBITaxon_35567 obo:NCBITaxon_35568 obo:NCBITaxon_33355 obo:NCBITaxon_236659 obo:NCBITaxon_33354 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_7524 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_33351 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_33349 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33345 obo:NCBITaxon_33347 obo:NCBITaxon_33342 obo:NCBITaxon_173680 obo:NCBITaxon_50557 obo:NCBITaxon_33343'><span id='am-156' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-157' property="rdfs:label" content="Proba" datatype="xsd:string"></span><span id='am-158'  ></span><span id='am-159' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>proba</span>-          suf13-1. A plasmid designated pJH13.1 isolated a <br /> bly homogenotized gene conversion, yielding               YCp50 yeast DNA library complemented suf13-1 allele <br /> homologous leu2-3 alleles separated vector sequences fol-          indicated plasmid-dependent slow-growth phenotype <br /> lowing transformation. Chromosomal DNA Ura&#11001;                  SC-ura/leu medium. YCpSUF13 contains HindIII/Hin- <br /> Leu&#11002; transformant digested BamHI, ligated, used          dIII fragment derived pJH13.1, subcloned <br /> to transform E. coli 6507 ampicillin resistance. A representa-     pRS316. To construct YEpSUF13/1, 3.0-kb fragment <br /> tive plasmid, YIp26-leu2-3, restriction map identi-       containing SUF13 removed pJH13.1 cleavage at <br /> cal parental YIp26-LEU2 plasmid, used determine         SphI PvuII sites vector inserted YEp352. <br /> the DNA sequence leu2-3 gene.                                  To construct YEpSUF13/2, contains unique BglII site <br />    The wild-type MET2 gene cloned complementation              SUF13 gene, EcoRI fragment removed from <br /> of met2-1 strain 1160 using YEp24 library (Carlson           YEpSUF13/1 contained second BglII site. To construct <br /> Botstein 1982). Using vivo gene conversion mapping              YEpSUF13/3, 1.42-kb HindIII fragment containing SUF13 <br /> method (Sandbaken Culbertson 1988), met2-1 mu-                removed YEpSUF13/2 inserted YEp351. <br /> tation showed 23% coconversion repair double-               YIpSUF13 constructed inserting 1.42-kb Hin- <br /> stranded break XbaI restriction site 5&#11032; noncoding         dIII fragment YIp5. <br /> region 37% coconversion repair double-stranded             To generate SUF13 disruption, fragment carrying the <br /> break SacI site coding region. These results          TRP1 gene inserted unique BglII site SUF13. To <br /> indicated met2-1 mutation resided XbaI           accomplish <span id='am-60' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-63'  ></span><span id='am-64' property="rdfs:label" content="This" datatype="xsd:string"></span>this</span>, single-stranded ends 1.45-kb EcoRI <br /> and SacI restriction sites 5&#11032; one-third MET2 gene.      fragment carrying TRP1 filled using T4 DNA polymer- <br /> DNA carrying met2-1 mutation cloned using gapped-             ase. YEpSUF13/2 digested BglII, ends were <br /> duplex repair (Orr-Weaver et al. 1983) transforming            filled using T4 DNA polymerase. The blunt-ended frag- <br /> met2-1 strain 1160 YCp50-MET2 linearized XbaI        ments ligated, resulting plasmid YEpsuf13::TRP1, <br /> and SacI restriction sites. The complete sequence XbaI-        carries suf13::TRP1 disruption allele. One-step gene <br /> SacI restriction fragment YCp50-met2-1 plasmid             replacement used integrate suf13::TRP1 suf13 <br /> determined.                                                           locus (Rothstein 1983). A 2.3-kb HindIII/BamHI fragment <br />    SUF13 gene: A restriction map SUF13 region shown         containing suf13::TRP1 removed YEpsuf13::TRP1 by <br /> in Figure 1A. SUF13 cloned complementation                  codigestion HindIII/BamHI used transform strain <br />                                                     Frameshift Suppressors Yeast                                              1145 <br />  <br /> YPH274 tryptophan prototrophy. The presence suf13::            moter SUF14 coding region, constructed fol- <br /> TRP1 suf13 locus verified Southern blotting.           lows. Starting plasmid YEpSUF14/4, BamHI restriction <br /> The resulting diploid, JHY2, sporulated tetrads         site inserted immediately 5&#11032; SUF14 initiator AUG <br /> analyzed determine haploid strains carrying disrup-        methionine codon using inverse PCR, resulting plasmid <br /> tion viable.                                                    YEpSUF14B. The plasmid YpJ166 derivative YEp352 <br />    The suf13-1 mutation cloned using gap repair (Orr-            contains CUP1 promoter sequences plasmid <br /> Weaver et al. 1983). YCpSUF13 partially digested SacI       pCUP1pgaIKCYC1 (obtained D. Ecker) EcoRI <br /> followed digesting completion HpaI. The digestion         site located transcriptional start site CUP1 <br /> products used transform strain JHY1 uracil prototro-      gene. To construct YEpCUP1pSUF14, YEpSUF14B di- <br /> phy. Twenty-three 72 transformants grew SC-ura/            gested BamHI SacI site vector sequence. The <br /> leu medium rate strain JHY1 transformed          BamHI site blunt filling using Klenow the <br /> a control vector, indicating plasmid conferred uracil       3&#11032; overhangs removed using T4 DNA polymerase. <br /> prototrophy longer contained wild-type SUF13 allele         Plasmid YpJ166 digested EcoRI, site filled <br /> and likely contained suf13-1 allele instead. Plasmids       using Klenow. The blunt-ended fragment YEpSUF14B <br /> from transformants rescued E. coli strain 6507        ligated EcoRI-digested blunt-ended YpJ166. This re- <br /> (Hoffman Winston 1987). Four plasmids                 sulted fusion CUP1 promoter SUF14 ORF. <br /> restriction patterns identical YCpSUF13. These plasmids           There AUG codons 5&#11032;-UTR CUP1-SUF14 <br /> were retransformed strain JHY1. Growth SC-ura/leu            fusion AUG fusion mRNA the <br /> was identical strains carrying suf13-1 allele. One    SUF14 AUG. A 2.2-kb BamHI fragment containing CUP1-SUF14 <br /> of plasmids used determine DNA sequence             removed YEpCUP1-SUF14 inserted pRS314, <br /> of suf13-1 gene.                                                 pRS315, pRS424, YEp351, resulting plasmids named <br />    SUF14 gene: A restriction map SUF14 region shown        YCpCUP1-SUF14/1, YCpCUP1-SUF14/2, YEpCUP1-SUF14/1, <br /> in Figure 1B. To construct SUF14 disruption, plasmid               YEpCUP1-SUF14/2, respectively (see Table 2). Since CUP- <br /> YEp352&#9004;R15 constructed digesting YEp352 EcoRI            1pSUF14 SUF14 mRNA contain different 5&#11032;-UTR se- <br /> followed filling recessed ends religation. A        quences, possible detect mRNAs separately on <br /> 1.1-kb SphI/PvuII fragment carrying SUF14 subcloned         Northern blots. SUF14 mRNA detected using oligomer <br /> YEp352&#9004;R15, resulting plasmid YEpSUF14/2. A 1.45-kb           oSUF145UTR (5&#11032;-ACCATGGATCAATTCGTTAC-3&#11032;), is <br /> EcoRI fragment carrying yeast TRP1 gene inserted          complementary 20-nucleotide (nt) long 5&#11032;-UTR of <br /> the unique EcoRI site located SUF14 coding region.        SUF14 mRNA (Fingen-Eigen et al. 1996). CUP1-SUF14 mRNA <br /> One-step gene replacement used integrate                  detected using oligomer oCUP1-SUF14 (5&#11032;-ACCATG <br /> suf14::TRP1 allele suf14 locus (Rothstein 1983). To           GATCAATTCGTTAC-3&#11032;), complementary 5&#11032;-UTR <br /> accomplish this, fragment carrying suf14:TRP1 removed          CUP1-SUF14 mRNA. <br /> from YEpSUF14/2 digestion SphI used trans-               The SUF14 promoter 5&#11032;-UTR fused E. coli <br /> form strains YPH274 YPH501 (Sikorski Hieter                  lacZ ORF. To construct multicopy plasmid carrying fu- <br /> 1989) tryptophan prototrophy. The insertion disrup-        sion, SUF14 ORF removed YEpSUF14B by <br /> tion suf14 locus verified Southern blotting. The       codigestion BamHI HindIII. The ends made <br /> resulting diploids sporulated tetrads analyzed         blunt filling overhangs. A 3.0-kb SalI fragment con- <br /> to determine haploid strains carrying disruption         taining lacZ ORF blunt using T4 DNA poly- <br /> viable.                                                              merase, blunt-ended fragments ligated to- <br />    The suf14-1 mutation cloned using gap repair (Orr-            gether inserted linearized blunt-ended YEp352, <br /> Weaver et al. 1983). To accomplish this, plasmid YEp             resulting plasmid YEpSUF14placZ. <br /> 352&#9004;R1H constructed digesting YEp352 HindIII                Growth presence antibiotics: A filter disk assay was <br /> followed filling recessed ends religation. A        used determine extent growth presence of <br /> 1.5-kb SphI/DraI fragment containing SUF14 cloned           antibiotics (Singh et al. 1979). Exponentially growing cells <br /> YEp352&#9004;R1H, resulting plasmid YEpSUF14/4. The                 directly spread plates (Figure 2A) mixed <br /> SUF14 coding region removed plasmid codi-            appropriate molten medium containing 0.8% agar <br /> gestion EcoRI HindIII, cut unique          overlaid corresponding solid medium (Figure <br /> sites plasmid. The plasmid lacking SUF14 coding se-           2, B C). Twenty-five microliters (Figure 2A) 12.5 or <br /> quences gel purified used transform strain JHY6           25 &#65533;?&#65533;l (Figure 2, B C) drug applied filter <br /> to uracil prototrophy. Twenty-two 23 transformants            disks (Schleicher &amp; Schuell, Keene, NH; 3/8 inch) water- <br /> grew SC-ura/leu medium rate strain JHY6            soluble stock solutions follows: 10 &#65533;?&#65533;g/ml cycloheximide; <br /> transformed control vector, indicating plasmid       20 mg/ml hygromycin B; 20 mg/ml Geneticin [G418]; 40 <br /> conferred uracil prototrophy longer contained wild-         mg/ml paromomycin; 20 mg/ml streptomycin). The disks <br /> type SUF14 allele, likely contained suf14-1 allele      placed plates preseeded lawns strains <br /> instead. Plasmids transformants rescued E.        tested. The plates incubated 30&#11034; 3 days. <br /> coli strain 6507 (Hoffman Winston 1987). All plasmids        The relative extent growth inhibition antibiotic <br /> had restriction patterns identical YEpSUF14/1. These plas-        isogenic mutant wild-type strains quantitated by <br /> mids retransformed strain JHY6. Growth SC-ura/          comparing zone growth inhibition millimeters after <br /> leu identical strains carrying suf14-1 allele.    subtracting 9.5 mm diameter disk. <br /> One plasmids used determine DNA se- <br /> quence suf14-1 gene. <br />    The suf14fs allele constructed digesting YEpSUF14/4 <br /> with EcoRI, filling base overhangs DNA poly-                                   RESULTS <br /> merase I large fragment religating plasmid T4              Suppression frameshift nonsense mutations <br /> DNA ligase, resulting plasmid YEpsuf14fs/1. A 3.34-kb SphI/ <br /> KpnI fragment containing suf14fs removed subcloned           suf13-1 suf14-1: Alleles SUF13 SUF14 <br /> into YEp351, resulting plasmid YEpsuf14fs/2.                      genes originally identified cosuppressors the <br />    The allele CUP1pSUF14, fusion CUP1 pro-         his4-713, leu2-3, met2-1 mutations. To test pattern <br />  1146                                                         J. L. Hendrick et al. <br />  <br />                            TABLE 3                                        met2-1 &#11001;1 frameshift mutations (Table 4). <br />      Suppression frameshift nonsense mutations                     The ICR-170-induced mutation corresponding leu2-3 <br />                                                                           single base-pair insertion G/C consecutive <br />                                             Suppressionb                  run G/C base pairs. The mutation causes a <br /> Frameshift/nonsense                                                       change wild-type amino acid sequence Lys84- <br /> mutationa                         suf13-1                  suf14-1 <br />                                                                           Trp-Gly-Thr-Gly-Ser-Val Lys84-Trp-Gly-Tyr-Arg. The ar- <br /> his4-506                          NS                     NS               ginine codon followed UAG stop codon that <br /> his4-520                       Suppressed             Suppressed          causes premature termination translation. Likewise, <br /> his4-208                          NS                     NS               spontaneous mutation corresponding met2-1 a <br /> his4-507                       Suppressed             Suppressed          single base-pair insertion G/C consecutive run <br /> his4-38                           NS                     NS <br />                                                                           G/C base pairs. The mutation causes change <br /> his4-519                          NS                     NS <br /> his4-504                       Suppressed             Suppressed          wild-type amino acid sequence Gly85-Pro- <br /> his4-713c                      Suppressed             Suppressed          Leu-Leu-Gly Gly85-Pro-Ser-Ser-Gly. The final glycine <br /> his4-712                          NS                     NS               codon followed UAA stop causes premature <br /> leu2-3c                        Suppressed             Suppressed          termination translation. <br /> arg4-20                           NS                     NS                  The SUF13 gene synonymous MBF1 gene: A <br /> met2-1c                        Suppressed             Suppressed          heterozygous SUF13/suf13-1 leu2-3/leu2-3 diploid strain <br /> ade2-1                            NS                     NS <br />                                                                           JHY1a (Table 1) grew SC-leu reduced rate <br /> trp1-1                         Suppressed             Suppressed <br /> leu2-2                            NS                     NS               compared JHY1 (the suf13-1 leu2-3 parent JHY1a). <br /> his4-260                          NS                     NS               This result indicated suf13-1 exhibits semidominant <br /> his4-305                          NS                     NS               suppression leu2-3 diploid. Since extent of <br /> his4-306                          NS                     NS               suppression distinguishable observed in <br />    a <br />      Alleles listed <span id='am-84' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-85' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> his4-506 met2-1 all <br />                                                                           haploid strain, possible clone SUF13 com- <br /> ICR-induced &#11001;1 frameshift mutations. Mutations cause                 plementation mutant suf13-1 allele. To accom- <br /> premature chain termination include ade2-1 (UAG), trp1-1                  plish this, strain JHY1 transformed Ura&#11001; phe- <br /> (UAA), leu2-2 (UGA), his4-260 (UGA). The his4-305                 notype using YCp50-based yeast genomic library <br /> his4-306 mutations change HIS4 AUG initiation codon                containing yeast DNA inserts SUF13 strain. Trans- <br /> GUG UUG, respectively. Strains carry alleles <br /> (81-10C 167-4C) obtained T. F. Donahue and <br />                                                                           formants screened identify having in- <br /> are described Castillo-Valavicius et al. (1990).                       termediate growth rate SC-ura/leu medium. Four <br />    b <br />      NS indicates suppression indicated growth             &#11601;10,000 transformants displayed phenotype. <br /> selective medium extensive incubation 30&#11034;. Suppressed            Plasmids rescued transformants into <br /> indicates growth selective medium 3&#8211;4 days incuba-               E. coli (Hoffman Winston 1987). One four <br /> tion 30&#11034;. <br />    c <br />      Suppression null allele SUF13::TRP1 observed. <br />                                                                           clones recovered E. coli, designated pJH13.1, con- <br />                                                                           ferred expected partial complementation re- <br />                                                                           transformation yeast strain JHY1. <br />                                                                              To confirm clone likely contain the <br /> of suppression suf13-1 suf14-1, haploid strains <br />                                                                           SUF13 gene, showed plasmid called YIpSUF13 <br /> were constructed contained suppressors <br />                                                                           carrying URA3 1.42-kb HindIII yeast DNA frag- <br /> in combination wide variety &#11001;1 frameshift                       ment pJH13.1 (Figure 1A) integrated following <br /> mutations types nonsense mutations.                      transformation site closely linked suf13 locus <br /> We patterns suppression suf13-1                     chromosome 15 (Gaber et al. 1983; Orr-Weaver et <br /> and suf14-1 identical (Table 3). Both mutations                      al. 1983). URA3 suf13-1 failed recombine a <br /> suppressed &#11001;1 frameshift mutations his4-520, his4-                    cross. The presence DNA insertion resulting from <br /> 507, his4-504, his4-713, leu2-3, met2-1. Among non-                   integration SUF13 region confirmed by <br /> sense mutations, trp1-1 (UAG) suppressed.                        Southern blotting (data shown). DNA sequence <br /> Since suppressible frameshifts &#11001;1 insertions                  analysis 1.42-kb HindIII fragment YIpSUF13 <br /> but different types codons, altered tRNAs                     revealed 453-nt ORF DNA fragment com- <br /> not theory suppress mutations. This led                  plemented suf13-1. <br /> to suspect SUF13 SUF14 code proteins                     A search Saccharomyces Genome Database <br /> than tRNAs. The suf13-1 suf14-1 suppressors failed                    showed ORF identical MBF1 gene, <br /> to suppress his4-305 his4-306. These mutations                        codes known transcription factor (Take- <br /> change initiator methionine codon AUG                         maru et al. 1998). When 850-bp BsaA1 fragment of <br /> GUG UUG, respectively (Castillo-Valavicius et                         SUF13 used probe Northern blot, &#11601;800-bp <br /> al. 1990).                                                                mRNA detected (data shown). The predicted <br />    The leu2-3 met2-1 alleles previously                  protein product 151 amino acids length. It was <br /> analyzed DNA sequence level. When DNA se-                          shown previously MBF1 gene single-copy <br /> quence analysis performed, leu2-3                       gene essential growth (Takemaru et al. <br />                                                 Frameshift Suppressors Yeast                                           1147 <br />  <br />                                                           TABLE 4 <br />                       mRNA sequence changes associated suf13- suf14-suppressible mutations <br />  <br />         Suppressible allelea                                   Sequence (wild type &#8594; mutant)b <br />         leu2-3                            N-85 . . . Trp Gly Thr &#8594; Trp Gly Tyr Arg end <br />                                           5&#11032;-253 . . . UGG GGU ACC &#8594; UGG GGG UAC CGG UAG <br />         met2-1                            N-84 . . . Trp Gly Pro Leu Leu Gly &#8594; Trp Gly Pro Ser Ser Gly end <br />                                           5&#11032;-250 . . . UGG GGC CCU CUU CUG GGU &#8594; UGG GGC CCC UCU UCU GGG UAA <br />         his4A-520                         N-113 . . . Ser Ile Lys &#8594; Ser His Lys <br />                                           5&#11032;-337 . . . UCC AUA AAG &#8594; UCC CAU AAA <br />         his4A-507                         N-149 . . . Asp Gly Leu &#8594; Asp Gly Phe <br />                                           5&#11032;-445 . . . GAC GGU UUA &#8594; GAC GGG UUU <br />         his4A-504                         N-196 . . . Lys Gly Glu &#8594; Lys Gly end <br />                                           5&#11032;-586 . . . AAG GGU GAA &#8594; AAG GGG UGA <br />         his4C-713                         N-757 . . . Thr Pro Glu &#8594; Thr Pro end <br />                                           5&#11032;-2267 . . . ACC CCU GAA &#8594; ACC CCC UGA <br />           a <br />             The sequence changes associated his4-520, his4-507, his4-713 reported previously (Mathison <br />         Culbertson 1985). All mutations induced ICR-170 (Culbertson et al. 1977) for <br />         met2-1, spontaneous mutation originally obtained Berkeley strain collection. The positions <br />         translation termination mutant mRNAs shown leu2-3, met2-1, his4-504, his4-713. The his4- <br />         520 his4-507 mutations cause translation termination downstream UGA codon (nucleotides 423&#8211;425 <br />         480&#8211;482, respectively; Mathison Culbertson 1985). The his4 allele names include A C to <br />         indicate locations mutations HIS4A HIS4C coding regions. <br />           b <br />             Sequence changes mRNAs shown. The sequences nucleotides inserted the <br />         result mutations underlined. Subscripted numbers N-terminal 5&#11032; ends sequences <br />         refer nucleotide shown mRNA sequence amino acid shown protein <br />         sequence using initiator methionine mRNA &#11001;1 position. <br />  <br />  <br /> 1998). We confirmed copy number using Southern                wild type. Furthermore, strain carrying <br /> blotting (data shown). When JHY2, diploid hetero-           suf13-1 differed strains carrying suf13::TRP1 <br /> zygous suf13:TRP1 disruption, sporulated,             SUF13 sensitivity aminoglycoside <br /> all spores viable segregated 2:2 SC-           antibiotics. The strain carrying suf13-1 resis- <br /> Trp, indicating disruption SUF13 function did             tant paramomycin, G418, hygromycin B than <br /> not impair viability.                                             strain carrying SUF13. In contrast, suf13::TRP1 <br />    The suf13-1 mutation corresponds deletion              mutation caused cells sensitive these <br /> two nucleotides: The suf13-1 allele cloned (materi-           drugs wild type. <br /> als methods) DNA sequence analyzed.                 Overall, cells function SUF13 dis- <br /> The suf13-1 mutant differs wild type two-               rupted sensitive antibiotics wild <br /> base deletion positions 208&#8211;209 coding region           type, cells containing frameshift mutation <br /> resulting frameshift reading frame (Figure            resistant. These results suggest SUF13 <br /> 1A). The predicted peptide produced suf13-1               plays role maintaining fidelity translation. <br /> 80 amino acids long 69 residues derived            In addition, suf13-1 identical phenotype a <br /> from wild-type sequence followed 11 residues derived           null mutant. Despite presence 2-nt deletion <br /> from sequence corresponding &#11002;2 reading                 predicted cause frameshifting, trigger <br /> frame. Translation predicted terminate out-           nonsense-mediated mRNA decay, allele pro- <br /> of-frame stop codon.                                              duce truncated protein responsible the <br />    The suf13-1 mutant differs phenotypically com-          differential sensitivities aminoglycoside antibi- <br /> plete loss-of-function mutant: To examine effects             otics. <br /> of suf13 mutants translational fidelity, growth tests            Suppression his4-713 suf13-1 an <br /> were performed presence antibiotics, in-           elevated his4-713 transcript level: To test suf13- <br /> cluding streptomycin, cycloheximide, hygromycin B, ge-            mediated frameshift suppression increased <br /> neticin (G418), paromomycin (Figure 2A; Table                 abundance suppressible frameshift mRNA, <span id='am-76' about='obo:NCBITaxon_1178312' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_33154 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_572719 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41011 obo:NCBITaxon_41197 obo:NCBITaxon_41012 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-77'  ></span><span id='am-78' property="rdfs:label" content="Mea" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>mea</span>- <br /> 5). The induce changes fidelity            sured accumulation his4-713 mRNA Northern <br /> translation increased misreading genetic               blotting. We chose examine his4-713 accu- <br /> code (Singh et al. 1979). A strain carrying suf13::TRP1           mulation affected nonsense-mediated mRNA <br /> exhibited increased sensitivity cycloheximide com-             decay (NMD) analyses suf13-sup- <br /> pared isogenic wild-type strain,          pressible mutations potentially complicated because <br /> the sensitivity strain carrying suf13-1 comparable       subject NMD (Leeds et al. 1991, 1992). In <br />  1148                                                  J. L. Hendrick et al. <br />  <br />                                                                                                  Figure 1.&#8212;(A) The SUF13 gene <br />                                                                                               surrounding DNA. The box <br />                                                                                               represents yeast genomic DNA <br />                                                                                               solid line represents the <br />                                                                                               vector sequence YEp352. The <br />                                                                                               arrow represents position of <br />                                                                                               SUF13 open reading frame <br />                                                                                               direction transcription. <br />                                                                                               DNA leftward HindIII <br />                                                                                               site BamHI site se- <br />                                                                                               quenced. The site insertion of <br />                                                                                               TRP1 gene nucleotide &#11001;76 <br />                                                                                               suf13::TRP1 indicated. The <br />                                                                                               site 2-nt deletion suf13-1 <br />                                                                                               positions &#11001;208&#8211;209 shown. <br />                                                                                               (B) The SUF14 gene sur- <br />                                                                                               rounding DNA. The box repre- <br />                                                                                               sents yeast genomic DNA. The <br />                                                                                               arrow represents position of <br /> the SUF14 open reading frame direction transcription. The locations restriction sites indicated. DNA the <br /> SphI site rightward KpnI site sequenced. The site insertion TRP1 gene nucleotide &#11001;68 suf14::TRP1 is <br /> indicated. The site G &#8594; A transition mutation corresponding suf14-1 nucleotide &#11001;322 indicated. <br />  <br />  <br /> the presence histidine, his4-713 mRNA accumulated                A heterozygous SUF14/suf14-1 leu2-3/leu2-3 his4-713/ <br /> at threefold higher level wild-type SUF13 strain             his4-713 diploid strain grew SC-leu/his medium but <br /> compared suf13-1 strain (Table 6). When SUF13                  reduced rate compared haploid strains carrying <br /> cells shifted medium lacking histidine 3 hr,            suf14-1 leu2-3 his4-713. This result indicated suf14-1 <br /> transcription derepressed indicated               exhibits semidominant suppression leu2-3 his4- <br /> fourfold increase level his4-713 mRNA. The                713 diploid. To clone SUF14, strain JHY6 carrying <br /> same extent derepression observed suf13-1                leu2-3, his4-713, ura3-52 transformed Ura&#11001; <br /> cells 3 hr following starvation histidine. However,             phenotype YCp50 plasmid library (Rose et al. <br /> the overall level his4-713 mRNA substantially re-            1987). One &#11601;10,000 transformants harbored a <br /> duced presence absence histidine.                plasmid complemented suf14-1 allele indi- <br /> These results suggest suppression his4-713               cated plasmid-dependent slow-growth phenotype <br /> suf13-1 increased abundance                    SC-his/leu/ura medium. A plasmid designated <br /> frameshift mRNA observed isogenic                  pPW14.1 rescued E. coli. YepSUF14/1, con- <br /> wild-type SUF13 strain. Suppression               tains 3.34-kb SphI/KpnI yeast DNA fragment from <br /> likely change fidelity translation            pPW14.1, conferred expected partial complementa- <br /> the his4-713 message (see discussion).                              tion suf14-1 retransformation strain JHY6 <br />    The SUF14 gene codes ribosomal protein S3:               (Figure 1B). A 720-nt open reading frame correspond- <br />  <br />  <br />                                                                                                 Figure 2.&#8212;Effects antibi- <br />                                                                                                 otics growth of <br />                                                                                                 strains carrying suf13 and <br />                                                                                                 suf14 alleles. (A) Filter discs <br />                                                                                                 containing 25 &#65533;?&#65533;l of <br />                                                                                                 antibiotics placed <br />                                                                                                 lawns cells (materi- <br />                                                                                                 als methods). S, strep- <br />                                                                                                 tomycin; C, cycloheximide; <br />                                                                                                 H, hygromycin B; G, genet- <br />                                                                                                 icin (G418); P, paromo- <br />                                                                                                 mycin. The diameter the <br />                                                                                                 zone growth inhibition <br />                                                                                                 used quantitative <br />                                                                                                 measure sensitivity of <br />                                                                                                 strain antibi- <br />                                                                                                 otic (Table 5). (A) Lawns of <br /> cells prepared strains JHY3a (SUF13), JHY3b (suf13-1), JHY3c (suf13::TRP1). (B) Lawns cells prepared <br /> from haploid strains JH85.9a [YCp50] (suf14-1) JH85.9a [pPW14.1] (SUF14). (C) Lawns cell prepared diploid <br /> strains 2033 (SUF14/SUF14) 2033.2c3c (SUF14/suf14::TRP1). In B C, discs contain 12.5 &#65533;?&#65533;l antibiotic/disc. <br /> The discs panels contain 25 &#65533;?&#65533;l antibiotic. <br />                                                   Frameshift Suppressors Yeast                                             1149 <br />  <br />                                                             TABLE 5 <br />                          Growth strains carrying suf13 suf14 alleles presence antibiotics <br />  <br />                                                                             Allele <br />          Antibiotica                suf13-1b            suf13::TRP1b              suf14-1c           SUF14/suf14::TRP1d <br />          Cycloheximide               1.00                   1.40                   1.35                        1.00 <br />          Paramomycin                 0.00                   1.40                   0.66                        0.00 <br />          G418                        0.00                   1.40                   0.80                        0.95 <br />          Hygromycin B                0.25                   1.70                   1.00                        1.00 <br />          Streptomycin              No effect              No effect              No effect                   No effect <br />            a <br />              The relative growth rates shown calculated experiments described Figure 2 using 25 &#65533;?&#65533;l of <br />          antibiotic spotted filter discs described materials methods. The relative growth rates <br />          calculated diameter zone growth inhibition mutant divided diameter of <br />          zone growth inhibition corresponding wild-type strain. The strains used listed Figure 2 <br />          described Table 1. Numbers &#11022;1.00 indicate mutant strain exhibited increased sensitivity an <br />          antibiotic, number &#65533;?1.00 indicates increased resistance. &#8220;No effect&#65533;? indicates no <br />          growth inhibition mutant corresponding wild type. A value 0.00 indicates the <br />          mutant completely resistant antibiotic, corresponding wild-type strain exhibited some <br />          sensitivity. <br />            b <br />              Haploid strains carrying mutant suf13 alleles compared isogenic haploid strain carrying wild- <br />          type SUF13 (see Figure 2). <br />            c <br />              A haploid strain carrying suf14-1 compared isogenic haploid strain carrying wild-type SUF14 on <br />          plasmid (see Figure 2). <br />            d <br />              A heterozygous SUF14/suf14::TRP1 diploid strain compared isogenic homozygous SUF14/SUF14 <br />          diploid strain (see Figure 2). <br />  <br />  <br /> ing suf14-1 complementing activity identified               loids segregated 2:2 lethality. The viable spores <br /> DNA sequence analysis. A search Saccharomyces                  tetrad failed grow SC-trp shown <br /> Genome Database showed ORF identical                     Southern blotting carry wild-type SUF14 allele <br /> to RPS3 gene chromosome 14, encodes                      SUF14::TRP1 disruption (data shown). <br /> ribosomal protein S3 (Otaka et al. 1984). The suf14                      The suf14-1 mutation corresponds single nucleo- <br /> locus mapped previously region                       tide substitution: The entire open reading frame for <br /> (Gaber et al. 1983). A 1.0-kb SUF14 transcript de-                RPS3 surrounding DNA clone carrying <br /> tected Northern blotting (data shown).                         suf14-1 examined DNA sequence analysis (see <br />    It shown previously RPS3 single-copy                 materials methods). A single A G transition <br /> essential gene (Fingen-Eigen et al. 1996). We con-                    position 322 open reading frame <br /> firmed copy number using Southern blotting (data                  (Figure 1B). This results nonconservative amino <br /> not shown). A SUF14::TRP1 disruption constructed                  acid substitution positively charged lysine nega- <br /> (materials methods) used confirm                      tively charged glutamic acid. <br /> SUF14 essential viability. When diploid strains                   Mutant suf14 alleles confer increased resistance to <br /> JHY14 JHY15 heterozygous disrup-                aminoglycoside antibiotics: We determined relative <br /> tion sporulated, tetrads derived dip-             sensitivity aminoglycoside antibiotics haploid strains <br />                                                                       carry suf14-1 diploid strains carry SUF14/ <br />                                                                       suf14::TRP1 SUF14/SUF14 (Figure 2, B C; Table <br />                           TABLE 6 <br />                                                                       5). All strains resistant streptomycin. <br />   Effect suf13-1 accumulation his4-713 mRNA              There difference sensitivity hygromycin B <br />                                                                       haploid strains carry suf14 SUF14. <br />                              his4-713 mRNA accumulationa              However, suf14-1 strain twice resistant paro- <br /> Strain                      &#11001; Histidine           &#11002; Histidine         momycin G418 SUF14 strain using differ- <br />                                                                       ent concentrations antibiotics applied filter <br /> suf13-1 [pJH13.1]   b <br />                             1.00 &#11006; 0.07           3.88 &#11006; 0.08 <br />                                                                       disks (twice drug needed produce <br /> suf13-1 [YCp50]c            0.33 &#11006; 0.06           1.42 &#11006; 0.17 <br />                                                                       killing zone). Conversely, suf14-1 strain <br />    a <br />      Cells grown described materials methods.          twice sensitive cycloheximide SUF14 <br /> mRNA accumulation determined Northern blotting                 strain. <br /> and normalized level actin mRNA. Standard devia- <br /> tions determined experiments.                            When SUF14/SUF14 homozygous diploid com- <br />    b <br />      pJH13.1 carries wild-type SUF13 gene (Table 2).              pared SUF14/suf14::TRP1 diploid, cycloheximide <br />    c <br />      YCp50 vector lacking SUF13 gene.                   hygromycin B inhibited growth extent, <br />  1150                                                 J. L. Hendrick et al. <br />  <br /> unlike suf14-1 haploid strain hypersensitive <br /> to cycloheximide. At antibiotic concen- <br /> trations tested, SUF14/suf14::TRP1 strain slightly <br /> more resistant G418 twice resistant <br /> to paromomycin SUF14/SUF14 strain. Overall, <br /> these results resemble effects ribosomal muta- <br /> tions alter translational fidelity, suggesting ri- <br /> bosomal protein S3 affects translational accuracy. <br />    Effect gene dosage expression SUF13 <br /> and SUF14: Regulated genes fail expressed <br /> in proportion gene dosage. To examine effects <br /> of gene dosage expression, compared levels <br /> of expression SUF13 SUF14 strains carrying <br /> a single chromosomal copy gene strains <br /> carrying multiple copies 2&#65533;?&#65533; plasmids. Southern and <br /> Northern blotting used quantitate compare <br /> gene copy number levels mRNA accumulation <br /> (Figure 3). <br />    Strain JHY21e containing multicopy plasmid YEp- <br /> SUF13/3 Southern blotting contain <br /> 11.0 &#11006; 1.0 copies SUF13 gene compared un- <br /> transformed JHY21, contains single chromo- <br /> somal gene copy. When RNA accumulation assessed <br /> by Northern blotting, level accumulation in <br /> JHY21e increased 10 &#11006; 0.5-fold. These results sug- <br /> gest SUF13 gene expressed proportion to <br /> the number gene copies. Strain JHY21b containing <br /> the multicopy plasmid YEpSUF14/5 South- <br /> ern blotting contain 7.2 &#11006; 0.6 copies SUF14 <br /> gene compared single chromosomal copy present <br /> in JHY21. However, level RNA accumulation in- <br /> creased 2.8 &#11006; 0.3-fold. Although represents a <br /> modest increase, proportional number <br /> of gene copies. We repeated experiment using trans-             Figure 3.&#8212;Comparison gene dosage mRNA accu- <br /> formants strains JHY16e JHY16f, carry              mulation SUF13 SUF14. Strains JHY21a, JHY21b, and <br /> single-copy centromeric plasmid YCpSUF14/2                JHY21e transformants JHY21 contain plasmids <br /> multicopy plasmid YEpSUF14/5. In strains,                pRS315 (vector control), multicopy YEpSUF14/5 (wild- <br />                                                                   type SUF14), multicopy YEpSUF13/3 (wild-type SUF13), <br /> accumulation SUF14 mRNA Northern blots                  respectively. (A) A representative Southern blot determine <br /> significantly expected compared num-             gene copy number Northern blot determine the <br /> ber gene copies determined Southern blotting.               levels mRNA accumulation. A 0.85-kb Bsa1 DNA fragment <br />    To test promoter sequences re-             used probe SUF13. A 0.65-kb EcoRI/HindIII <br /> quired transcription SUF14 missing             DNA fragment used probe SUF14 (see Figures <br />                                                                   1A 3A). ACT1 (actin) mRNA served loading control <br /> the YEpSUF14/5 plasmid, contains 0.4 kb up-              (not shown). DNA used Southern blotting digested <br /> stream sequences, performed experiment                EcoRI (SUF14) HindIII (SUF13). (B) A histogram of <br /> described Figure 4 using strain JHY21c. This strain            SUF13 SUF14 gene copy number JHY21a, JHY21b, <br /> contains YEpSUF14/6, includes 1.3 kb up-                 JHY21e compared accumulation SUF14 and <br /> stream sequences likely contain          SUF13 mRNA strains. Whiskers indicate standard <br />                                                                   deviation n &#11005; 6. <br /> all required promoter sequences. The lack pro- <br /> portion mRNA accumulation relative gene dosage <br /> was observed (not shown). We compared single-copy                 dent expression SUF14: Since yeast ribo- <br /> vs. multicopy expression URA3 LEU2 genes.              somal proteins shown autoregulate their <br /> In cases, levels mRNA accumulation corre-             synthesis (Eng Warner 1991; Dabeva and <br /> lated number gene copies (not shown).            Warner 1993; Li et al. 1995; Presutti et al. 1995; Few- <br /> Since dosage independence SUF14 appeared                ell Woolford 1999), tested ribosomal <br /> be unique, examined SUF14 expression                protein S3 autoregulates expression SUF14 gene. <br /> see indicative regulatory mechanism              If occurred, explain SUF14 exhibits <br /> controlling expression.                                           gene dosage-independent expression. <br />    Ribosomal protein S3 required dosage-depen-                To accomplish this, frameshift mutation consisting <br />                                                  Frameshift Suppressors Yeast                                       1151 <br />  <br />                                                                   mulation strains JHY21a, JHY21b, JHY21d, of <br />                                                                   carry chromosomal upf1::URA3 gene disruption <br />                                                                   (Figure 4). These isogenic strains carry chromo- <br />                                                                   somal copy wild-type SUF14 gene, carry either <br />                                                                   vector (JHY21a), multiple plasmid copies of <br />                                                                   wild-type SUF14 (JHY21b), chromosomal copy of <br />                                                                   SUF14 multiple plasmid copies nonfunctional <br />                                                                   suf14fs frameshift allele (JHY21d). mRNA levels in <br />                                                                   JHY21d 9.0 &#11006; 0.8-fold higher compared JHY21a <br />                                                                   3- 4-fold higher JHY21b. These results indi- <br />                                                                   cate mRNA accumulation JHY21d commen- <br />                                                                   surate gene copy number. Since difference <br />                                                                   JHY21b JHY21d strain <br />                                                                   produces functional SUF14 product latter <br />                                                                   does not, result suggests ribosomal protein S3, <br />                                                                   protein product SUF14, necessary gene <br />                                                                   dosage-independent expression. <br />                                                                      SUF14 5&#11541; noncoding sequences required dos- <br />                                                                   age-dependent expression: To test SUF14 <br />                                                                   5&#11032; noncoding sequences required gene dosage- <br />                                                                   independent expression, fused CUP1 promoter <br />                                                                   immediately upstream AUG initiation codon of <br />                                                                   SUF14 create allele CUP1-SUF14, lacks the <br />                                                                   sequences corresponding SUF14 promoter and <br />    Figure 4.&#8212;Dosage-dependent expression frameshift          5&#11032;-UTR (materials methods). CUP1-SUF14 pro- <br /> allele SUF14. Strains JHY21a, JHY21b, JHY21e trans-    duces functional product based ability confer <br /> formants JHY21 contain plasmids pRS315 (vector <br /> only control), multicopy YEpSUF14/5 (wild-type SUF14),        growth strain JHY16c, carries chromosomal <br /> multicopy YEpsuf14fs (a frameshift allele SUF14), respec-      suf14::TRP1 disruption CUP1-SUF14 allele on <br /> tively. (A) A representative Southern blot determine gene      plasmid YCpCUP1-SUF14/2. In presence 0.2&#8211;0.6 <br /> copy number Northern blot determine levels        mm exogenous copper, mRNA transcribed CUP1- <br /> mRNA accumulation. The blots probed simultaneously           SUF14 induced level fivefold higher that <br /> with 0.65-kb EcoRI/HindIII SUF14 fragment 0.85-kb <br /> BsaA1 SUF13 fragment. ACT1 (actin) mRNA served load-         wild-type SUF14 transcript. Some CUP1-SUF14 <br /> ing control Northern blots (not shown). DNA used          transcript detected added copper, pre- <br /> for Southern blotting codigested EcoRI HindIII.      sumably growth medium contains <br /> (B) A histogram SUF14 gene copy number strains          small copper (see Figure 6). <br /> JHY21a, JHY21b, JHY21e compared accumulation              The accumulation SUF14 mRNA examined <br /> of SUF14 mRNA strains. Whiskers indicate stan- <br /> dard deviation n &#11005; 6.                                          wild-type SUF14 CUP1-SUF14 ex- <br />                                                                   pressed single-copy multicopy plasmids (Figure <br />                                                                   5). The results shown wild-type SUF14 mRNA resem- <br /> of four-base insertion EcoRI site SUF14           ble shown Figure 4 mRNA accumulation <br /> ORF constructed (see materials methods                commensurate gene copy number. However, <br /> Figure 1B). This mutation causes premature termina-               expression CUP1-SUF14 single- and <br /> tion translation downstream out-of-frame UGA              multicopy plasmids compared, mRNA accumula- <br /> codon predicted produce truncated protein             tion increased 10.2 &#11006; 0.7-fold strain JHY23c, <br /> 30 amino acids long. The suf14fs allele nonfunctional          carries multicopy plasmid. This increase was <br /> based inability complement suf14::TRP1              mirrored similar increase gene copy number, <br /> null allele. To avoid destabilization suf14fs mRNA             suggesting mRNA accumulates dosage- <br /> due NMD (Leeds et al. 1991), performed                  dependent manner. This result indicates 5&#11032; non- <br /> experiments frameshift allele transformants           coding sequences including SUF14 promoter and/ <br /> of strain PLY102. This strain carries upf1::URA3 al-          wild-type 5&#11032;-UTR SUF14 mRNA required <br /> lele (Leeds et al. 1992), inactivates NMD path-         dosage-independent accumulation SUF14 mRNA. <br /> way. The NMD pathway effect expression                     Induction CUP1-SUF14 reduces accumulation of <br /> of wild-type SUF14 (Lelivelt Culbertson 1999),                SUF14 mRNA: The expression CUP1-SUF14 ana- <br /> indicating suf14fs mRNA wild-type SUF14                  lyzed assess effect increased synthesis ribo- <br /> mRNA stability strain PLY102.                    somal protein S3 accumulation of <br />    To assess effects multiple nonfunctional copies         mRNA derived endogenous SUF14 gene. Strains <br /> of SUF14 mRNA accumulation, compared                    JHY23a containing pRS314 vector JHY23b con- <br /> number gene copies extent mRNA accu-                taining YCpCUP1-SUF14 grown presence <br />  1152                                                   J. L. Hendrick et al. <br />  <br />  <br />  <br />  <br />    Figure 5.&#8212;Dosage-dependent expression CUP1-SUF14. <br /> Strains JHY21a JHY21b transformants JHY21 that <br /> contain plasmids pRS315 (vector control) YEp- <br /> SUF14/5 (wild-type SUF14), respectively. Strains JHY23b and <br /> JHY23c transformants JHY23 contain single-copy <br /> plasmid YCpCUP1-SUF14/1 multicopy plasmid YEp-                     Figure 6.&#8212;Copper-mediated induction CUP1-SUF14 re- <br /> CUP1-SUF14/1 (CUP1 promoter fused SUF14 ORF),                duces accumulation endogenous SUF14 mRNA. Strains <br /> respectively. (A) Representative Southern blots determine        JHY23a JHY23b carry single chromosomal copy of <br /> gene copy number Northern blots determine levels         SUF14 gene. JHY23a contains vector pRS314 and <br /> of mRNA accumulation. DNA used Southern blotting            JHY23b contains YCpCUP1-SUF14, carries copper- <br /> digested EcoRI HindIII. Northern blots stripped       responsive CUP1-SUF14 gene. (A) Northern blots used de- <br /> and reprobed using oligomer hybridizes ACT1 (ac-         termine relative level mRNA accumulation SUF14 <br /> tin) mRNA, served loading control (not shown).           mRNA derived endogenous SUF14 gene for <br /> To detect SUF14 mRNA JHY21a JHY21b, blots           CUP1-SUF14 mRNA. Blots probed oligomer <br /> probed 0.65-kb EcoRI/HindIII DNA fragment           oSUF145UTR, hybridizes 5&#11032;-UTR SUF14 mRNA <br /> SUF14 coding region. To detect CUP1-SUF14 mRNA JHY23b            fails anneal CUP1-SUF14 mRNA. The blots were <br /> and JHY23c, blots probed 20-mer anneals         stripped reprobed oligomer oCUP1-SUF14, <br /> the CUP1 5&#11032;-UTR CUP1-SUF14 mRNA (this probe does             anneals CUP1-SUF14 5&#11032;-UTR. ACT1 (actin) <br /> anneal wild-type SUF14 mRNA). (B) A histogram comparing          mRNA served loading control (not shown). The apparent <br /> gene copy number mRNA accumulation. For strain                 band shift right artifact electrophoresis <br /> JHY21b, gene copy number mRNA accumulation level            does indicate altered size SUF14 mRNA. (B) The <br /> were calculated relative single chromosomal copy          histograms relative levels accumulation SUF14 <br /> SUF14 corresponding transcript JHY21a. For strain        mRNA CUP1-SUF14 mRNA increasing copper concen- <br /> JHY23c, gene copy number mRNA accumulation level            trations. The open solid boxes endoge- <br /> were calculated relative single centromeric plasmid copy     nous wild-type SUF14 mRNA levels strain lacking SUF14- <br /> of CUP1-SUF14 corresponding transcript JHY23b.           CUP1 strain carrying SUF14-CUP1, respectively. The <br /> Whiskers indicate standard deviation n &#11005; 4.                  shows level accumulation CUP1-SUF14 <br />                                                                     mRNA response added copper. <br />  <br /> of 0.0&#8211;1.0 mm exogenous copper added SC-Trp me- <br /> dium. The accumulation CUP1-SUF14 mRNA                       exogenous copper added. The reduc- <br /> mRNA derived chromosomal SUF14 gene                    tion mRNA accumulation absence added <br /> monitored separately Northern blotting (Figure 6).               copper probably presence residual cop- <br />   As copper concentration increased, accu-              medium causes induction CUP1- <br /> mulation mRNA derived CUP1-SUF14 increased                  SUF14. The addition copper medium caused <br /> up 0.6 mm copper, slight decrease               decrease endogenous SUF14 mRNA accu- <br /> accumulation observed. In strain JHY23a containing              mulation ultimately &#11601;30&#8211;50% level accumu- <br /> the pRS314 vector, accumulation endogenous                   lation observed absence YCpCUP1-SUF14 <br /> SUF14 mRNA derived chromosomal SUF14                       plasmid. These results indicate increased synthesis <br /> gene unaffected copper concentrations ranging                ribosomal protein S3 derived CUP1-SUF14 <br /> from 0.0 0.6 mm level began de-               causes reduced accumulation endogenous SUF14 <br /> crease. In strain JHY23b containing CUP1-SUF14,                 mRNA. <br /> accumulation endogenous SUF14 mRNA reduced                      Effect copy number transcription decay <br />                                                  Frameshift Suppressors Yeast                                             1153 <br />  <br /> of SUF14 mRNA: We assessed lower than <br /> expected levels SUF14 mRNA accumulation oc- <br /> cur SUF14 gene copy number increased are <br /> due change transcription mRNA decay. Run- <br /> on transcription assays (Warner 1991) performed <br /> to determine relative rates synthesis newly syn- <br /> thesized transcripts permeabilized cells strains <br /> JHY21a JHY21b, carry single multiple <br /> copies SUF14 gene. We examined strain <br /> JHY21d, carries frameshift allele suf14fs a <br /> multicopy plasmid. All derivatives JHY21 carry upf1:: <br /> URA3, inactivates NMD pathway that <br /> suf14fs mRNA stability SUF14 mRNA. <br />    The rates SUF14 transcription strains <br /> were compared levels SUF14 mRNA accumula- <br /> tion determined Northern blotting SUF14 <br /> gene copy number determined Southern blotting <br /> (Figure 7). JHY21b JHY21d, carry SUF14 or <br /> suf14fs multicopy plasmid, showed 7.4 &#11006; 0.3- <br /> fold 10.5 &#11006; 0.8-fold increase gene copy number, <br /> respectively. The levels mRNA accumulation these <br /> strains increased 3.0 &#11006; 0.2-fold 9.5 &#11006; 0.7-fold, <br /> respectively. These results consistent previous <br /> experiments showing increases mRNA levels <br /> were expected SUF14 gene copy number <br /> was increased, approximately commensurate <br /> with increases gene copy number SUF14                    Figure 7.&#8212;Transcription SUF14. Transcriptional run-on <br /> gene contained frameshift mutation blocked syn-            assays performed assess effects increased gene <br /> thesis S3 protein. By comparison, transcript levels        copy number rate transcription. JHY21a contains <br />                                                                   chromosomal copy SUF14 gene vector <br /> produced run-on transcription increased &#11601;6.3 &#11006; 0.9-            YEp351. JHY21b contains multicopy plasmid YEpSUF14/5. <br /> fold JHY21b 8.2 &#11006; 1.0-fold JHY21d. While            JHY21d contains multicopy plasmid YEpsuf14fs/2. Each <br /> increases levels run-on transcripts some-           strain carries upf1::URA3 frameshift mRNA in <br /> what increases gene copy number,           JHY21d unaffected NMD pathway. (A) A representa- <br /> experiments suggest lower expected level            tive run-on transcription experiment detect labeled RNA <br />                                                                   permeabilized cells hybridized nylon mem- <br /> of mRNA accumulation observed JHY21b                 brane 2 &#65533;?&#65533;g target DNA bound (see materials <br /> explained solely mechanism involving inhibition              methods). The target DNAs included plasmid DNAs <br /> of transcription. Transcriptional inhibition            carrying SUF14 gene ACT1 gene. pUC19 plasmid <br /> increased concentration ribosomal protein S3,            DNA (2 &#65533;?&#65533;g) served negative control. (B) A histogram <br /> occurs, modest.                                                strains JHY21a, JHY21b, JHY21d comparing mRNA <br />                                                                   accumulation levels detected Northern blotting, gene copy <br />    To examine potential role transcription         number detected Southern blotting (data Figure 4), <br /> in S3-mediated autoregulation, constructed fusions             newly synthesized transcripts detected run-on transcrip- <br /> between SUF14 promoter 5&#11032;-UTR E. coli              tion. Whiskers indicate standard deviation n &#11005; 4. <br /> lacZ ORF. The accumulation SUF14-lacZ fusion mRNA <br /> was monitored Northern blotting strains JHY22b <br /> and JHY22d, carry SUF14 gene single <br /> chromosomal copy multiple copies plasmid               rates determined strains JHY13a JHY13b, <br /> (Figure 8). The results indicate accumulation            carry single multiple copies SUF14, respec- <br /> of SUF14-lacZ mRNA does vary SUF14 gene              tively (Figure 9B). Both strains carry mutation rpb1-1, <br /> copy number increased. These results suggest              prevents transcription restrictive tempera- <br /> ribosomal protein S3 does regulate synthesis          ture 37&#11034; (Herrick et al. 1990). The decline mRNA <br /> when SUF14 ORF 3&#11032;-UTR replaced lacZ              accumulation following temperature shift deter- <br /> sequences SUF14 promoter 5&#11032;-                  mined Northern blotting (Figure 9A). Decay rates <br /> UTR present fusion transcript. Although            determined separate experiments that <br /> SUF14 5&#11032;-UTR appears play role regulation              SUF14 mRNA half-life 15.9 &#11006; 2.6 min the <br /> shown previous experiments using CUP1 promoter               single-copy SUF14 strain 9.9 &#11006; 1.3 min the <br /> fusion, sufficient itself mediate regulation.     multicopy SUF14 strain. This represents 40% relative <br />    We measured effect changes SUF14 gene                decline mRNA half-life SUF14 gene copy <br /> copy number half-life SUF14 mRNA. Decay                 number increased. <br />  1154                                                   J. L. Hendrick et al. <br />  <br />  <br />  <br />  <br />                                                                        Figure 9.&#8212;The half-life SUF14 mRNA decreases the <br />                                                                     presence multiple copies wild-type SUF14 gene. Strain <br />                                                                     JHY13a contains YEp351 (vector control) single chromo- <br />                                                                     somal copy SUF14 gene. JHY13b contains multicopy <br />                                                                     plasmid YEpSUF14/5. Both strains carry rpb1-1, inhibits <br />                                                                     transcription restrictive temperature 37&#11034;. (A) mRNA <br />    Figure 8.&#8212;Accumulation SUF14-lacZ promoter fusion             accumulation determined Northern blotting the <br /> mRNA unaffected changes SUF14 gene copy number.            time intervals indicated (in minutes) following shift 37&#11034;. <br /> Strains JHY22b JHY22d contain plasmid YEpSUF14-             Blots probed using oSUF145UTR (materials meth- <br /> lacZ. Strain JHY22d contains SUF14 gene             ods). (B) Data plotted percentage RNA remaining <br /> multicopy plasmid YEpSUF14/5. The accumulation SUF14             vs. time following temperature shift. The lines drawn <br /> mRNA SUF14-lacZ mRNA determined Northern                 using linear regression. Half-lives determined four <br /> blotting using probe oSUF145UTR (materials                  separate experiments shown. <br /> methods), anneals transcripts. ACT1 (actin) <br /> mRNA served loading control (not shown). (A) Northern <br /> blot RNA single-colony isolates JHY22b (lanes <br /> 1, 2, 3) JHY22d (lanes 4, 5, 6). (B) The       coding sequence. In addition, common fea- <br /> histogram shows relative levels accumulation SUF14        ture suggesting suppressible &#11001;1 frame- <br /> mRNA SUF14-lacZ mRNA.                                           shifts located regions especially prone to <br />                                                                     reading frame correction slippage (Qian et al. <br />                                                                     1998; Farabaugh Bjork 1999; Sundararajan et <br />                                                                     al. 1999; Farabaugh 2000). It the <br />                        DISCUSSION <br />                                                                     future determine suf13- suf14-mediated <br />    Suppression &#11001;1 frameshift mutations: In addition              suppression occurs slippage- nonslippage-based <br /> to suppressible his4 frameshifts sup-             mechanism proposed Sundararajan et al. (1999). <br /> pressed suf13 suf14 mutations (Table 4),                  SUF13 codes transcription factor Mbf1p: In <br /> are seven his4 &#11001;1 frameshifts sup-               <span id='am-48' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-49' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-54'  ></span>Drosophila melanogaster</span>, shown Mbf1p <br /> pressed. These include his4-506 (CC &#8594; CCC), his4-38                 functions bridging factor recruits TATA- <br /> (GG &#8594; GGG), his4-518 (CC &#8594; CCC), his4-208 (GG &#8594;                     binding protein (TBP) promoter binding site <br /> GGG), his4-707 (CCC &#8594; CCC), his4-206 (GGG &#8594;                         factor FTZ-F1, results transcriptional <br /> GGGG), his4-519 (GGG &#8594; GGGG) (Mathison                      activation fushi tarazu gene (Li et al. 1994; Take- <br /> Culbertson 1985). There obvious correlation                   maru et al. 1997). In humans, isoforms Mbf1p <br /> evident sequences suggests              exist, bind TBP function a <br /> frameshifts suppressed sup-                manner similar Mbf1p insects (Kabe et al. <br /> pressed. There similarity lengths G C            1999). hMbf1 localizes cytoplasm recruited <br /> runs, range consecutive nucleo-            nucleus needed transcriptional activa- <br /> tides, nature adjacent codons, positions              tion. In S. cerevisiae, MBF1 gene functional <br /> stop codons brought register &#11001;1 frameshift,             homologue insect human versions the <br /> or position frameshift overall mRNA               gene (Takemaru et al. 1998). In vivo vitro studies <br />                                                  Frameshift Suppressors Yeast                                         1155 <br />  <br /> indicate Mbf1p tethers TBP DNA-binding                communication). Further experiments required <br /> domain transcription factor Gcn4p. This pro-               test model. <br /> motes transcriptional activation HIS3 gene,             Role ribosomal protein S3 reading frame mainte- <br /> is known transcriptional targets regula-           nance: To test frameshift suppression caused <br /> tion Gcn4p.                                                    change translational fidelity, examined the <br />    In article suf13-1, contains         effects suf14-1 suppressor mutation growth <br /> 2-nt deletion position &#11001;208-209 yeast SUF13             presence aminoglycoside antibiotics. When <br /> (MBF1) ORF, causes suppression &#11001;1 frameshift muta-             antibiotics type interact ribosomes, they <br /> tions HIS4, LEU2, MET2 genes. The suf13-1              cause phenotypic suppression elevated codon <br /> mutation located middle domain              misreading. We suf14-1 causes increased re- <br /> is required binding Mbf1p Gcn4p                sistance aminoglycoside antibiotics, result con- <br /> brings downstream out-of-frame stop codon                sistent interpretation suf14-1-mediated sup- <br /> MBF1 ORF register upstream TBA binding                pression results increased misreading. <br /> region. Given this, function Mbf1p                  Similar effects growth presence aminogly- <br /> severely impaired (Takemaru et al. 1998). Furthermore,            coside antibiotics paromomycin G418 been <br /> a complete null allele SUF13(MBF1) resulting              shown associated mutations 18S rRNA <br /> insertion TRP1 gene position &#11001;76 upstream               (Chernoff et al. 1994), suggesting presence a <br /> of Mbf1p functional domains causes suppres-              decoding center ribosomes consisting rRNA <br /> sion frameshift mutations. These results indicate         ribosomal proteins. At present, structure of <br /> suppression caused loss function Mbf1p. How-          yeast ribosomes insufficiently characterized know <br /> ever, suf13-1 allele apparently retains function         S3 physically located near decoding <br /> since confers differential resistance aminoglycoside        center. However, bacterial human homologues <br /> antibiotics compared suf13::TRP1.                           S. cerevisiae S3 shown located the <br />    Mechanism suppression transcription factor               decoding center (Choi et al. 1998; Smolenskaya et al. <br /> Mbf1p: The HIS4, LEU2, MET2 genes regulated               1998). We propose S3 play key <br />                                                                   role maintaining fidelity translation. <br /> by Gcn4p, indicating expression levels should <br />                                                                      In contrast phenotype suf14-1, suf13:: <br /> be reduced Mbf1p function impaired (Take- <br />                                                                   TRP1 null mutant caused increased sensitivity amino- <br /> maru et al. 1998). We examined level accumula- <br />                                                                   glycoside antibiotics, suf13-1 suppressor mu- <br /> tion his4-713 mRNA suf13-1 strain found <br />                                                                   tation caused increased resistance. The suppression re- <br /> that substantially reduced compared same <br />                                                                   sulting complete loss Mbf1p function therefore <br /> mRNA his4-713 SUF13 strain. This makes very <br />                                                                   appears mechanistically different sup- <br /> unlikely suppression caused simply overpro- <br />                                                                   pression mutational alteration ribosomal pro- <br /> duction frameshift mRNA frameshift <br />                                                                   tein S3. <br /> mutation read produce sufficient <br />                                                                      Evidence autoregulation S3 expression: The <br /> functional HIS4 product confer growth absence <br />                                                                   expression genes coding ribosomal proteins is <br /> of exogenous histidine. <br />                                                                   coordinately regulated primarily level transcrip- <br />    It appears likely fidelity translation        tion S. cerevisiae (Li et al. 1999; Nomura 1999; Warner <br /> of mRNA affected. Our results testing             1999). In addition, relatively rapid turnover free <br /> sensitivity suf13 alleles antibiotics induce trans-    ribosomal proteins corresponding mRNAs en- <br /> lational misreading support view. Our results sug-           sures majority cellular ribosomal pro- <br /> gest impaired Mbf1p function indirectly affect           tein assembled ribosomes. <br /> the translation mRNAs. Whereas MBf1p TBP                      Some ribosomal proteins subject additional <br /> function POLII transcription (Takemaru et             layers regulation. The expression genes coding <br /> al. 1998), TBP component RNA polymer-           ribosomal proteins S. cerevisiae shown <br /> ase complexes involved transcription                autoregulated. Ribosomal protein L30 (formerly <br /> of POLIII genes including tRNA genes (Cormack                 L32) regulates expression levels (Eng <br /> Struhl 1992). If Mbf1p play role expres-         Warner 1991; Dabeva Warner 1993). A se- <br /> sion POLIII transcripts binding TBP            quence intron pre-mRNA forms stem <br /> that tRNA transcription reduced suf13 mutants,             loop sequence exon I promote binding of <br /> then reduced levels tRNAs induce increased           L30, inhibits splicing. The sequence exon <br /> rates frameshift misreading leading suppression             I forms stem loop sequence exon II the <br /> (Farabaugh 2000). This shown occur                mature mRNA promote binding L30, inhib- <br /> Maf1p, nuclear protein affects tRNA biosynthesis           translation. Free L30 inhibits new produc- <br /> in yeast (K. Pluta, N. C. Martin, J. Weislaw, W. J.               tion L30 levels. The synthesis ribosomal <br /> Smagowicz, O. Lefebvre, D. R. Stanford, S. R. Ellis,              protein S14 coordinated ribosome assembly <br /> A. K. Hopper, A. Sentenac M. Boguta, personal                 high affinity binding S14 18S rRNA and <br />  1156                                                 J. L. Hendrick et al. <br />  <br /> low affinity binding mRNA, encoded            ducing S3 accumulate &#11601;50% wild-type endogenous <br /> by RPS14B(CRY2). Free S14 inhibits expression bind-            SUF14 transcript, (2) accumulation CUP1-SUF14 <br /> ing noncontiguous sequences mRNA               transcript regulated, (3) change accumula- <br /> form regulatory stem loop, exon I                tion explained change transcription rate, <br /> other intron (Li et al. 1995; Fewell Wool-             (4) accumulation SUF14-lacZ transcript not <br /> ford 1999). Finally, ribosomal protein L4 (formerly               regulated. Taken together, results interpre- <br /> L2), autoregulated binding L4                ted mean regulation primarily post-transcrip- <br /> mRNA, promotes endonucleolytic cleavage                 tional 5&#11032;-UTR required S3 <br /> end trimming leading degradation (Presutti et al.              regulation, 5&#11032;-UTR sufficient by <br /> 1995).                                                            itself. The 5&#11032;-UTR ORF arranged in <br />    In article present evidence ribosomal             <span id='am-115' about='obo:NCBITaxon_245896' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71527 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_219428 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_577181 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-116' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-117' property="rdfs:label" content="Cis" datatype="xsd:string"></span><span id='am-118'  ></span><span id='am-119' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>cis</span> regulation occur. By analogy regula- <br /> protein S3 autoregulated. When S3 overproduced             tory stem loops described playing a <br /> from multicopy plasmid, accumulation SUF14               role regulation expression S13 L30, <br /> mRNA failed increase proportion gene copy            possibility sequence 5&#11032;-UTR forms a <br /> number. SUF13(MBF1) autoregulated              stem loop sequence ORF mediate <br /> manner, mRNA accumulation proportional                  regulation. According model, ORF is <br /> to gene copy number. We showed S3 mediates                   present SUF14-lacZ, fusion regu- <br /> regulation analyzing allele SUF14 contains          lated. The RNA-folding program MFOLD (provided by <br /> a frameshift mutation. mRNA accumulation exam-                M. Zuker) predicts stable RNA stem loops that <br /> ined frameshift allele overexpressed            span initiation codon S. cerevisiae human <br /> upf1&#11002; genetic background accumulation              RPS3 mRNA. The stem loops share common feature <br /> the frameshift mRNA independent non-              AUG initiation codon based-paired stem <br /> sense-mediated mRNA decay pathway. The accumula-                  despite complete lack sequence conservation be- <br /> tion frameshift mRNA proportional gene              tween 5&#11032;-UTRs organisms. Our data are <br /> dosage. In addition, CUP1 promoter/5&#11032;-UTR                 consistent model stem loop type <br /> fused SUF14 ORF just upstream AUG initia-           serve binding site S3. However, further <br /> tion codon. It possible distinguish fusion             experiments required verify is <br /> mRNA wild-type SUF14 mRNA using different oli-               case. <br /> gonucleotides anneal uniquely CUP1                   We measured half-life SUF14 mRNA found <br /> SUF14 5&#11032;-UTRs. We induction CUP1-SUF14              statistically significant reduction t1/2 &#11601;40% <br /> in response exogenous copper caused decline               SUF14 overexpressed. The magnitude the <br /> the accumulation wild-type SUF14 mRNA. Taken to-               change half-life differs magnitude change <br /> gether, results indicate level SUF14            mRNA accumulation &#11601;1.5-fold. This likely <br /> mRNA accumulation depends intracellular con-               method measuring half-life, which <br /> centration ribosomal protein S3.                               involves temperature shift strain carrying mu- <br />    The regulation presumed to                tant RNA polymerase subunit encoded rpb1-1. The <br /> transcriptional inhibition S3 accelerated mRNA           temperature shift itself affects relative mRNA half- <br /> decay. Since SUF14 intron, splicing in-             life, addition shown shifting a <br /> volved. To distinguish possibilities,            strain carrying rpb1-1 restrictive temperature <br /> examined gene fusions, performed run-on                 causes stabilization mRNAs encoding ribosomal <br /> transcription assay, measured half-life SUF14          proteins (Li et al. 1999). This expected to <br /> mRNA. We CUP1-SUF14 fusion produced                partially counteract effects accelerated decay re- <br /> a mRNA accumulates proportion gene dosage,             sulting autoregulatory system. If not <br /> indicating regulated. Since fusion lacks      stabilizing effect rpb1-1, magnitude the <br /> the SUF14 promoter 5&#11032;-UTR, regulation                change half-life likely mirror change <br /> due transcriptional inhibition post-          mRNA accumulation. <br /> transcriptional mechanism involving 5&#11032;-UTR. The                  We suggest primary mechanism <br /> run-on transcription assay shows little          autoregulation involves mRNA degradation. On the <br /> no change rate transcription SUF14              basis current results, propose regula- <br /> overexpressed. This result indirectly implicates role           tory requires ribosomal protein S3, 5&#11032;-UTR <br /> for 5&#11032;-UTR regulation. When fusion               mRNA, additional region the <br /> the SUF14 promoter 5&#11032;-UTR E. coli lacZ               mRNA arranged cis 5&#11032;-UTR. Further studies <br /> ORF analyzed, overexpression wild-           required identify sequences mRNA <br /> type SUF14 effect accumulation               required regulation, demonstrate binding S3 <br /> fusion mRNA, indicating 5&#11032;-UTR suffi-             mRNA, identify sequences required for <br /> cient regulation.                                             binding, establish series events that <br />    In summary, results (1) cells overpro-           constitute proposed decay pathway. <br />                                                          Frameshift Suppressors Yeast                                                        1157 <br />  <br />   This work supported College Agricultural Life               <span id='am-21' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-22' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-36'  ></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> regulates expression binding to <br /> Sciences, University Wisconsin Medical School, National Sci-             RPS14B pre-mRNA 18S rRNA. Mol. Cell. Biol. 19: 826&#8211;834. <br /> ence Foundation grant MCB 9870313 (M.R.C.), Public Health              Fingen-Eigen, M., H. Domdey K. Kohrer, 1996 The ribosomal <br /> Service Grant NIH GM59019 (M.R.C.). This Laboratory Genetics              protein gene RPS3 essential single copy gene yeast <br />                                                                                 Saccharomyces cerevisiae. Biochem. Biophys. Res. Commun. 223: <br /> paper no. 3573.                                                                 397&#8211;403. <br />                                                                            Gaber, R. F., M. R. Culbertson, 1982 Frameshift suppression <br />                                                                                 Saccharomyces cerevisiae. IV. New suppressors spontane- <br />                       LITERATURE CITED                                          ous co-revertants Group II his4-206 leu2-3 frameshift <br />                                                                                 mutations. Genetics 101: 345&#8211;367. <br /> Ball, C. B., M. D. Mendenhall, M. G. Sandbaken M. R. Cul-              Gaber, R. F., M. R. Culbertson, 1984 Codon recognition dur- <br />     bertson, 1988 The yeast SUF5 frameshift suppressor encodes                  ing frameshift suppression Saccharomyces cerevisiae. Mol. Cell. <br />     mutant glycine tRNA(CCC). Nucleic Acids Res. 16: 8712.                    Biol. 4: 2052&#8211;2061. <br /> Botstein, D., S. <span id='am-65' about='obo:NCBITaxon_8952' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_8949 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_8948 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-66'  ></span><span id='am-67' property="oboInOwl:hasExactSynonym" content="falcons" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Falco" datatype="xsd:string"></span>Falco</span>, S. E. Stewart, M. Brennan, S. Scherer et           Gaber, R. F., L. Mathison, I. Edelman M. R. Culbertson, 1983 <br />     al., 1979 Sterile host yeasts (SHY): eukaryotic bio-            Frameshift suppression Saccharomyces cerevisiae. VI. Complete <br />     logical containment recombinant DNA experiments. Gene                   genetic map twenty-five suppressor genes. Genetics 103: 389&#8211; <br />     8: 17&#8211;24.                                                                   407. <br /> Carlson, M., D. Botstein, 1982 Two differentially regulated            Grey, M., M. Brendel, 1992 A minute protocol trans- <br />     mRNAs different 5&#11032; ends encode secreted intracellular              forming Saccharomyces cerevisiae electroporation. Curr. Genet. <br />     forms yeast invertase. Cell 28: 145&#8211;154.                                 22: 335&#8211;336. <br /> Castillo-Valavicius, B., H. Yoon T. F. Donahue, 1990 Ge-               Herrick, D., R. Parker A. Jacobson, 1990 Identification and <br />     netic characterization Saccharomyces cerevisiae translational        comparison stable unstable mRNAs Saccharomyces cerevis- <br />     initiation suppressors sui1, sui2, SUI3 effects            iae. Mol. Cell. Biol. 10: 2269&#8211;2284. <br />     HIS4 expression. Genetics 124: 483&#8211;495.                                Hill, J. E., A. M. Myers, T. J. Koerner A. Tzagoloff, 1986 <br /> Chernoff, Y. O., A. Vincent S. W. Liebman, 1994 Mutations                   Yeast/E. coli shuttle <span id='am-120' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-121' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-123' property="rdfs:label" content="vectors" datatype="xsd:string"></span>vectors</span> multiple unique restriction sites. <br />     eukaryotic 18S ribosomal RNA affect translational fidelity           Yeast 2: 163&#8211;167. <br />     resistance aminoglycoside antibiotics. EMBO J. 13: 906&#8211;913.         Hoffman, C. S., F. Winston, 1987 A ten-minute DNA prepara- <br /> Choi, K. M., J. F. Atkins, R. F. Gesteland R. Brimacombe, 1998              tion yeast efficiently releases autonomous plasmids trans- <br />     Flexibility nascent polypeptide chain ribo-               formation <span id='am-103' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-104' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-109'  ></span>Escherichia</span> coli. Gene 57: 267&#8211;272. <br />     some&#8212;contacts peptide N-terminus specific region         Hughes, D., J. F. Atkins S. Thompson, 1987 Mutants elonga- <br />     30S subunit. Eur. J. Biochem. 255: 409&#8211;413.                          tion factor Tu promote ribosomal frameshifting nonsense <br /> Christianson, T. W., R. S. Sikorski, M. Dante, J. H. Shero P.               readthrough. EMBO J. 20: 4235&#8211;4239. <br />     Hieter, 1991 Multifunctional yeast high-copy-number shuttle            Kabe, Y., M. Goto, D. Shima, T. Imai, T. Wada et al., 1999 The <br />     vectors. Gene 110: 119&#8211;122.                                                 role human MBF1 transcriptional coactivator. J. Biol. <br /> Cormack, B. P., K. Struhl, 1992 The TATA-binding protein                    Chem. 274: 34196&#8211;34202. <br />     required transcription nuclear RNA polymerases     Leeds, P., S. W. Peltz, A. Jacobson M. R. Culbertson, 1991 <br />     yeast cells. Cell 69: 685&#8211;696.                                           The product yeast UPF1 gene required rapid turnover <br /> Culbertson, M. R., L. Charnas, M. T. Johnson G. R. Fink, 1977               mRNAs containing premature translational termination co- <br />     Frameshifts frameshift suppressors Saccharomyces cerevisiae.         don. Genes Dev. 5: 2303&#8211;2314. <br />     Genetics 86: 745&#8211;764.                                                  Leeds, P., J. M. Wood, B. S. Lee M. R. Culbertson, 1992 Gene <br /> Culbertson, M. R., K. M. Underbrink G. R. Fink, 1980                        products promote mRNA turnover Saccharomyces cerevisiae. <br />     Frameshift suppression Saccharomyces cerevisiae. II. Genetic proper-        Mol. Cell. Biol. 12: 2165&#8211;2177. <br />     ties group II suppressors. Genetics 95: 833&#8211;853.                    Lelivelt, M. J., M. R. Culbertson, 1999 Yeast Upf proteins <br /> Culbertson, M. R., R. F. Gaber C. M. Cummins, 1982                          required RNA surveillance affect global expression the <br />     Frameshift suppression Saccharomyces cerevisiae. V. Isolation        yeast transcriptome. Mol. Cell. Biol. 19: 6710&#8211;6719. <br />     genetic properties nongroup-specific suppressors. Genetics          Li, B., C. R. Nierras J. R. Warner, 1999 Transcriptional ele- <br />     102: 361&#8211;378.                                                               ments involved repression ribosomal protein synthesis. <br /> Culbertson, M. R., P. Leeds, M. G. Sandbaken P. G. Wilson,                  Mol. Cell. Biol. 19: 5393&#8211;5404. <br />     1990 Frameshift suppression, pp. 559&#8211;570 The Ribosome: Struc-       Li, F., H. Ueda S. Hirose, 1994 Mediators activation of <br />     ture, Function, Evolution, edited A. D. W. Hill, R. Garret,          fushi tarazu gene transcription BmFTZ-F1. Mol. Cell. Biol. 14: <br />     P. Moore, D. Schlessinger J. Warner. American Society                   3013&#8211;3021. <br />     Microbiology, Washington, DC.                                      Li, Z., A. G. Paulovich J. L. Woolford, 1995 Feedback inhibi- <br /> Cummins, C. M., R. F. Gaber, M. R. Culbertson, R. Mann G. R.                tion yeast ribosomal protein gene CRY2 mediated by <br />     Fink, 1980 Frameshift suppression Saccharomyces cerevisiae.              nucleotide sequence secondary structure CRY2 pre- <br />     III. Isolation genetic properties group III suppressors.             mRNA. Mol. Cell. Biol. 15: 6454&#8211;6464. <br />     Genetics 95: 855&#8211;879.                                                  Lodmell, J. S., A. E. Dahlberg, 1997 A conformational switch <br /> Cummins, C. M., T. F. Donahue M. R. Culbertson, 1982 Nucle-                 Escherichia coli 16S ribosomal RNA decoding mes- <br />     otide sequence SUF2 frameshift suppressor gene Saccha-            senger RNA. Science 277: 1262&#8211;1267. <br />     romyces cerevisiae. Proc. Natl. Acad. Sci. USA 79: 3565&#8211;3569.          Maniatis, T., E. F. Fritsch J. Sambrook, 1982 Molecular Clon- <br /> Cummins, C. M., M. R. Culbertson G. Knapp, 1985 Frameshift                  ing: A Laboratory Manual. Cold Spring Harbor Laboratory Press, <br />     suppressor mutations outside anticodon yeast proline                 Cold Spring Harbor, NY. <br />     tRNAs containing intervening sequence. Mol. Cell. Biol. 5:          Mathison, L., M. R. Culbertson, 1985 Suppressible non- <br />     1760&#8211;1771.                                                                  suppressible &#11001;1 G-C base pair insertions induced ICR-170 <br /> Curran, J., M. Yarus, 1987 Reading frame selection trans-               his4 locus Saccharomyces cerevisiae. Mol. Cell. Biol. 5: <br />     fer RNA anticodon loop base stacking. Science 238: 1545&#8211;1550.               2247&#8211;2256. <br /> Dabeva, M. D., J. R. Warner, 1993 Ribosomal protein L32             Mathison, L., M. Winey, C. Soref, M. R. Culbertson G. Knapp, <br />     Saccharomyces cerevisiae regulates splicing translation         1989 Mutations anticodon stem affect removal introns <br />     transcript. J. Biol. Chem. 268: 19669&#8211;19674.                        pre-tRNA Saccharomyces cerevisiae. Mol. Cell. Biol. 9: 4220&#8211; <br /> Eng, F. J., J. R. Warner, 1991 Structural basis regulation          4228. <br />     splicing yeast messenger RNA. Cell 65: 797&#8211;804.                Mendenhall, M. D., M. R. Culbertson, 1988 The yeast SUF3 <br /> Farabaugh, P. J., 2000 Translational frameshifting: implications            frameshift suppressor encodes mutant glycine tRNA(CCC). Nu- <br />     mechanism translational frame maintenance. Prog. Nu-                 cleic Acids Res. 16: 8713. <br />     cleic Acid Res. Mol. Biol. 64: 131&#8211;170.                                Mendenhall, M. D., P. Leeds, H. Fen, L. Mathison, M. Zwick et <br /> Farabaugh, P. J., G. R. Bjork, 1999 How translational accuracy              al., 1987 Frameshift suppressor mutations affecting major <br />     influences reading frame maintenance. EMBO J. 15: 1427&#8211;1434.                glycine transfer RNAs Saccharomyces cerevisiae. J. Mol. Biol. 194: <br /> Fewell, S. W., J. L. Woolford, 1999 Ribosomal protein S14                   41&#8211;58. <br />  1158                                                             J. L. Hendrick et al. <br /> Nierhaus, K. H., 1990 The allosteric three-site model ribo-           Sikorski, R. S., P. Hieter, 1989 A shuttle vectors and <br />     somal elongation cycle: features future. Biochemistry 29:                 yeast host strains designed efficient manipulation DNA in <br />     4997&#8211;5008.                                                                    Saccharomyces cerevisiae. Genetics 122: 19&#8211;27. <br /> Nomura, M., 1999 Regulation ribosome biosynthesis Esche-                Singh, A., D. Ursic J. Davies, 1979 Phenotypic suppression <br />     <span id='am-135' about='obo:NCBITaxon_1430853' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_95179 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_7100 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-136'  ></span><span id='am-137' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-138' property="rdfs:label" content="Richia" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>richia</span> coli Saccharomyces cerevisiae: diversity common prin-          misreading Saccharomyces cerevisiae. Nature 277: 146&#8211;148. <br />     ciples. J. Bacteriol. 181: 6857&#8211;6864.                                     Smolenskaya, I. A., K. N. Bulygin, D. M. Graifer, A. V. Ivanov, <br /> Oakes, M., A. Scheinman, T. Atha, G. Shankweiler J. Lake,                     A. G. Ven&#8217;yaminova et al., 1998 Localization template the <br />     1990 Three dimensional locations rRNA proteins, pp.                    decoding area affinity modification human ribosomes with <br />     168&#8211;179 The Ribosome: Structure, Function, Evolution, edited           photoactivated derivative oligoribonucleotide pGUGUUU. <br />     A. D. W. Hill, R. Garret, P. Moore, D. Schlessinger                    Mol. Biol. 32: 200&#8211;207. <br />     J. Warner. American Society Microbiology, Washington, DC.             Sundararajan, A., W. A. Michaud, Q. Qian, G. Stahl P. J. <br /> Orr-Weaver, T., J. W. Szostak R. J. Rothstein, 1983 Genetic                   Farabaugh, 1999 Near-cognate peptidyl-tRNAs promote &#11001;1 <br />     applications yeast transformation linear gapped plas-             programmed translational frameshifting yeast. Mol. Cell 4: <br />     mids. Methods Enzymol. 101: 228&#8211;245.                                          1005&#8211;1015. <br /> Otaka, E., K. Higo T. Itoh, 1984 Yeast ribosomal proteins.                Takemaru, K., F. Li, H. Ueda S. Hirose, 1997 Multiprotein <br />     VIII. Isolation proteins sequence characterization                 bridging factor 1 (MBF1) evolutionarily conserved transcrip- <br />     twenty-four proteins cytoplasmic ribosomes. Mol. Gen.                 tional coactivator connects regulatory factor TATA <br />     Genet. 195: 544&#8211;546.                                                          element binding protein. Proc. Natl. Acad. Sci. USA 94: 7251&#8211; <br /> Parker, R., D. Herrick, S. Peltz A. Jacobson, 1991 Measure-                   7256. <br />     ment mRNA decay rates Saccharomyces cerevisiae. Methods             Takemaru, K., S. Harashima, H. Ueda S. Hirose, 1998 Yeast <br />     Enzymol. 194: 415&#8211;423.                                                        coactivator MBF1 mediates GCN4-dependent transcriptional acti- <br /> Presutti, C., T. <span id='am-110' about='obo:NCBITaxon_117093' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_117077 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_50704 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_50674 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_50671'><span id='am-111' property="rdfs:label" content="Villa" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-114'  ></span>Villa</span>, D. Hall, C. Pertica I. Bozzoni, 1995                  vation. Mol. Cell. Biol. 18: 4971&#8211;4976. <br />     Identification cis-elements mediating autogenous con-          Thompson, R. C., D. B. Dix A. M. Karim, 1986 The reaction <br />     trol ribosomal protein L2 mRNA stability yeast. EMBO J.                 ribosomes elongation factor Tu*GTP complexes. J. Biol. <br />     14: 4022&#8211;4030.                                                                Chem. 261: 4868&#8211;4874. <br /> Qian, Q., J. N. Li, H. Zhao, T. G. Hagervall, P. J. Farabaugh et              Ursic, D., K. L. Himmel, K. A. Gurley, F. Webb M. R. Culbert- <br />     al., 1998 A new model phenotypic suppression frameshift                son, 1997 The yeast SEN1 gene required processing <br />     mutations mutant tRNAs. Mol. Cell 1: 471&#8211;482.                              diverse RNA classes. RNA 25: 4778&#8211;4785. <br /> Rose, M. D., P. Novick, J. H. Thomas, D. Botstein G. R. Fink,             Van tol, H., N. Stange, H. J. Gross H. Beier, 1987 A human <br />     1987 A Saccharomyces cerevisiae genomic plasmid bank based                 plant intron-containing tRNATyr gene transcribed <br />     centromere-containing shuttle vector. Gene 60: 237&#8211;243.                     HeLa cell extract spliced different pathways. EMBO <br /> Rothstein, R. J., 1983 One-step gene disruption yeast. Methods                 J. 6: 35&#8211;41. <br />     Enzymol. 101: 202&#8211;209.                                                    Vijgenboom, E., L. Bosch, 1989 Translational frameshifts in- <br /> Sandbaken, M. G., M. R. Culbertson, 1988 Mutations elon-                   duced mutant <span id='am-81' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-82' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> polypeptide chain elongation <br />     gation factor EF-1 alpha affect frequency frameshifting            factor Tu Escherichia coli. J. Biol. Chem. 264: 13012&#8211;13017. <br />                                                                               Warner, J. R., 1991 Labelling RNA phosphoproteins Sac- <br />     amino acid misincorporation Saccharomyces cerevisiae. Genetics <br />                                                                                   charomyces cerevisiae. Methods Enzymol. 194: 423&#8211;428. <br />     120: 923&#8211;934. <br />                                                                               Warner, J. R., 1999 The economics ribosome biosynthesis in <br /> Schirmaier, F., P. Philippsen, 1984 Identification genes <br />                                                                                   yeast. Trends Biochem. Sci. 24: 437&#8211;440. <br />     coding translational elongation factor EF-1&#65533;?&#65533; S. cerevisiae. <br />                                                                               Wilson, P. G., M. R. Culbertson, 1988 SUF12 suppresso <br /> </body></html>