<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright 6 1995 Genetics Society America <br />  <br />  <br />  <br />             Sodium Orthovanadate-Resistant Mutants <span id='am-1' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-7'  ></span>Saccharomyces</span> cerevisiue <br />                  Show Defects Golgi-Mediated Protein Glycosylation, <br />                          Sporulation Detergent Resistance <br />  <br />                                Cynthia Kanik-Ennulat, Ermelinda Montalvo Norma Neff <br />              Program Molecular Biology Memorial Sloan-Kettering <span id='am-8' about='obo:NCBITaxon_6754' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_72041 obo:NCBITaxon_33154 obo:NCBITaxon_6681 obo:NCBITaxon_6682 obo:NCBITaxon_6683 obo:NCBITaxon_29964 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_116706 obo:NCBITaxon_116704 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_6692 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6753 obo:NCBITaxon_88770 obo:NCBITaxon_6752'><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10'  ></span><span id='am-11' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="Cancer" datatype="xsd:string"></span>Cancer</span> Center Cornell University Graduate School <br />                                           Medical Sciences, New Ymk, New York 10021 <br />                                                         Manuscript received January 3, 1995 <br />                                                       Accepted publication March 31, 1995 <br />  <br />  <br />                                                              ABSTRACT <br />                   Orthovanadate smalltoxic molecule competes biologically important oxyanion <br />                orthophosphate. Orthovanadate resistance arises spontaneously Saccharomycescerevisiae haploid cells <br />                mutation number genes. Mutations selected 3 mM sodium orthovanadate different <br />                degrees vanadate resistance, hygromycinsensitivity, detergent sensitivity sporulation defects. <br />                Recessive vanadate-resistant mutants belong genetic loci. Most mutants defective in <br />                outer chain glycosylation secreted invertase ( v n l , van2, van4, van5, van6, VAN7-116 others), <br />                phenotype &quot;vor lrRG mutants. The phenotypes vanadate-resistant mutants <br />                consistent alteration permeability specificity Golgi apparatus. The previously <br />                published VAN1 gene product 200 amino acid domain 40% identity MAN9 gene <br />                product 70% identity A w l gene product. Cells containing vanl-18, mnn9 (wg6)or <br />                anpl mutations phenotypic similarities. The VAN2 gene isolated coding region <br />                identified reported. It essential gene chromosome XV translated amino acid <br />                sequence predicts unique 337 amino acid protein multiple transmembrane domains. <br />  <br />  <br />  <br /> V    ANADIUM trace element eukaryotic <br />       cells required growth (WILLSKY  1990). <br /> The biologically significant molecule orthovanadate, <br />                                                                             mers (WILLSKY    et al. 1984, 1985; WILLSKY   1990). Poly- <br />                                                                             oxovanadate (IV)can form number higher <span id='am-21' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-22' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="order" datatype="xsd:string"></span>order</span> <br />                                                                             structures including nanoscale magnetic particles (GAT- <br /> which structural similarities orthophosphate.                        TESCHI et al. 1994). These particles large threedi- <br /> Oxyanions orthovanadate arsenate                          mensional molecular clusters oxometal ions that <br /> toxic cells competing endogenous oxyanions                       form multiple magnetic centers. <br /> such phosphate sulfate. Monomeric orthovana-                             These magnetic particles investigation for <br /> date interfere anynumber orthophosphate-                        physical pharmaceutical properties. One fea- <br /> mediated processes cell ionic mimicry                      ture magnetic centers attraction and <br /> mechanism (CLARKSON 1993).is Ita potent vitro in-                        capture heavy metals. One best studied mole- <br /> hibitor enzymes bind ATP                        cules named VI5 [VI5          (H20)I-,whichhas cap- <br /> recognize utilize high-energy phosphate residues                         tured arsenic atoms. <span id='am-16' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-17'  ></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="This" datatype="xsd:string"></span>This</span> feature orthovanadate <br />  (WILLSKY  1990).                                                           suggests secretory vacuolar network with <br />    Sodium orthovanadate potent inhibitor P-type                       lowered pH environment favor Polyoxovanadate <br /> ATPases thought energy poison                         (IV) formation potential site synthesisof <br /> when added cells. In vivo orthovanadate                        actual toxic compound vivo. <br /> been documented numerous effects, including                            Three laboratories isolated vanadate-resistant <br /> the increase oftyrosine phosphorylation proteins                         mutants S. cereuisiae (WILLSKY et aL 1985;KANIK-ENNU- <br /> and induction transformed insulin-mimic phe-                        LAT NEFF1990; BALLOUet aL 1991). These studies <br /> notype mammalian cells (WALTON     DIXON1993).                       shownthat number genes mutate tovana- <br /> Sodium orthovanadate reported RIEZMAN                           date-resistance. WILLSKY   colleagues studied ortho- <br />  (1985) increase uptake <span id='am-47' about='obo:NCBITaxon_117980' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_6656 obo:NCBITaxon_6681 obo:NCBITaxon_6684 obo:NCBITaxon_111521 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_6682 obo:NCBITaxon_88770 obo:NCBITaxon_111523 obo:NCBITaxon_6683 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49'  ></span><span id='am-50' property="rdfs:label" content="Lucifer" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Lucifer</span> yellow Saccharo-                   vanadate metabolism isolated complementa- <br /> myces cereuisiae.                                                           tion groups vanadate-resistant mutants. We isolated <br />    Another important chemical property orthovana-                        vanadate-resistant mutants reported sequence <br /> date spontaneous polymerization response                       locus ( VAN1), screening yeast mutants <br /> low pH (POPE1983).Vanadium exist yeast cells                         altered protein phosphorylation, especially tyrosine <br /> in number oxidation states, including higher poly-                     protein phosphorylation (KANIK-ENNULAT         NEW <br />                                                                             1990). BALLOU colleagues spontane- <br />   Corresponding author: Norma Neff, Box 86,Memorial Sloan-Ketter-           ous vanadate-resistant mutants ( IRG) noncondi- <br /> ing Cancer Center, 1275 York Ave., New York, NY 10021.                      tional defect secreted invertase glycosylation (BAL <br /> Genetics 140 933-943 (July, 1995) <br />  anik-Ennulat, <br />       C.    934                                                          E. Montalvo N. Neff <br />  <br />                     1991).Some wgmutations are <br />            LOU et al.                             allelic mnn              striction enzyme EcoRI dilute religation. Plasmid pNV93- <br />            mutations, isolatedby laboratory using biochemical               12 contains 3kb Hind111 fragment pNV93-6 ligated <br />                                                                                 Hind111 site yips. PlasmidpNV93-12swas        con- <br />            screens forviable cells defectivein outer chain          synthesis <br />                                                                                 structed digestion pNV93-12 DNA restriction en- <br />             (BALLOU1990).A subset mutants            (mnn7, mnn8,      zyme SnuB1, followed dilute religation. Plasmid pNV93- <br />            mnn9and mnnl0) secrete proteins             cell wall manno-      14 contains 2.2-kb EcoRI-Hind111 fragment pNV93- <br />            proteins significant deficiencies outer chainad-             6 ligated EcoRI Hind111 sites ofYCp50, and <br />            dition. The orthovanadate-resistant         alleles <br />                                                                          pNV93-15 contains fragment EcoRIHindIIIdi- <br />                                                                                 gested Stratagene vector pBluescript SK( -). <br />            MNN8, MNN9and MNNlOgenes showed similar defects                         To create null allele VAN2 locus, mostof open <br />            outer chain addition (BALLOUet al. 1991).                         reading frame deleted plasmid pNV93- 15 using     two <br />               Recently MNN9 MNNl genes                        SnuBl sites (Figures 3 4). The TRPI+ gene isolated <br />            cloned sequenced (YIP et al. 1994). There re-               YRpl7 using restriction enzymes leave blunt <br />            gionofsignificant        amino acididentitybetween               ends  (Ssp1and StuI) ligated deleted vector DNA. <br />                                                                                 The null construction excised vector polylinker <br />            MNN9 VAN1 predicted gene products. Based o n                     sequences using restriction enzymes SalI BamHI before <br />            results laboratories, examined                 adding lithium acetate-treated cells. <br />            glycosylation secreted invertase mutants,                 Polymerasechainreactionamplification:        Diploid trans <br />            compared phenotypes               van mutants        formants van2 TRP+ null construction analyzed <br />            vrg/mnn mutants.                                             reactions containing -10&#8217; cellequivalents genomic <br />                                                                                 DNA, 100 pm primer reagents Perkin <br />                                                                                 Elmer Cetus &#8220;GeneAmp&#65533;? kit. Thirty cycles 1&#8216; 94&#65533;?, 1&#8217; at <br />                            MATERIALSANDMETHODS                                  50&#65533;? 2&#8216; 72&#8220; Perkin-Elmer DNA Thermal Cycler were <br />               Strains: The S. cerevisiue strains used study listed  used. The sequences oligonucleotides were:VAN2sense <br />            Table 1. The haploid strains NF134,NF147, diploid             = 5&#8217;-CAAGTTCGTCGTCAACTT-3&#8216; VAN2 antisense = 5&#8217;- <br />            strain NF3623 S288c background (SCHATZ           et al. 1986; CACTCGAGACAGGTATCA-3&#8217;. The mutant van2-93 gene <br />            SHIHet al. 1988). NFV mutant strains derived fromboth           amplified total nucleic acid NFV293 cells de- <br />            NF134 NF147.NFV220-224were                 selectedfrom vrg      scribed sequenced directly. The sequence the <br />            strains (BALLOU   et al. 1991).                                      oligonucleotides were: VAN2 sense = 5&#65533;?ACGCGTITCAGA- <br />               Media genetic techniques: Media growth, selection         TCCTCT-3&#8217; VAN2 antisense = 5&#8217;-CAATACGTTGCCTGA- <br />            uracil auxotrophs sporulation wereas described             GAT-3&#8216;. <br />            ROSE et al. (1990), yeast plates containing sodium          Invertase activity gels: Active forms invertase re- <br />            orthovanadate; plates weredescribed previously (KANIK-         solved  25 X 37 cm 5% polyacrylamide gels. Sample prepa- <br />            ENNULATand NEFF 1990).Yeast mating, sporulation tet-             ration, gel solutions assay conditions described <br />            rad analysis performed using standard methods (ROSE             BALLOU(1990). Gels run 4&#65533;?at 10 V 18-24 hr. <br />            et al. 1990). Vanadate resistance phenotypes were <br />                                                                                 In experiments, cell extracts 0.1% glucose <br />            scored spotting suspensions cells disposable micro-       grown cells used. Spheroplasts (ESMON      et al. <br />            titer wells plates using 48-prong metal replicator.           1981) distinguishinternal external forms invertase. <br />               Isolation sodium orthovanadate-resistant mutantsSpon- <br />            taneous sodiumorthovanadate-resistant mutants wereiso- <br />            lated individual colonies strains NF134 NF147.                                       RESULTS <br />            Each small colony ( -lo6 cells) suspended 200 ml of <br />            sterile water spread YPD plate containing 3 mM                 Isolation genetic analysis sodium orthovana- <br />            sodium orthovanadate. The plates incubated 26&#65533;?           dateresistant mutants: Spontaneous vanadate-resistant <br />            5 days. One vanadate-resistant colonywas chosen            mutants appeared YPD+3mM sodium orthovana- <br />            plate. After purification YPD plates, individual colo-      date plates frequency         Two primary <br />            nies retested vanadate resistance growth YPD <br />                                                                                 resistant colonies isolated retested. A total of <br />            plates 14,26 35&#65533;?.All vanadate-resistant mutations tested <br />            behaved single mutations crosses wild-typeand        104 mutant strains haploid parents demon- <br />            mutants. All phenotypes cosegregated vana-          stratedconsistentvanadate          resistance.Twenty-three <br />            date resistance alleles tested.                            showed leaky phenotype andwere studied further. <br />               Dominance complementation tests: To test domi-            No temperature-sensitive vanadatedependent mu- <br />            nance recessiveness vanadate mutations, mu- <br />            tant mixed 100 ml saturated liquid YPD culture         tants found. <br />            wild-type parent opposite mating type ina micro-          The remaining 81 strains mated opposite <br />            titer well. The mating mixtures spotted SD +              wild-type haploid parent heterozygous diploids were <br />            uracil plateto select diploid cells auxotrophic complemen-        tested vanadate resistance/sensitivity. Forty-nine re- <br />                                                           + <br />            tation. The Ura- diploids purified SD uracil plates          cessive strains formed vanadate-sensitive diploids with <br />            colonies diploid tested YPD + <br />                                                                                 wild type. The recessive mutants mated in <br />            vanadate plates. <br />               Plasmidconstructions: Bacterialstrains,plasmidvectors             combinatorial manner determine number               of <br />            YCp50-based yeastgenomic plasmid library           genes represented group. Thirty-two dominant <br />            described previously (SHIHet al. 1988; KANIK-ENNULAT             strains formed vanadate-resistant diploids wild type. <br />            NEW 1990). Yeast transformation recombinant DNA                  The results complementation analysis summa- <br />            methods described (KANIK-ENNULAT          NEFF1990). <br />                                                                                 rized Table 2. Our collection vanadate-resistant <br />            PlasmidpNV93-9 constructed digesting pNV93-6 <br />            restriction enzyme SphI dilute religation. Plasmid          mutants similar reportedby labora- <br />            pNV93-10 constructed digesting pNV93-9with re-                tories groups isolated recessive vanadate-resis- <br />                                                      Vanadate-Resistant Yeast Mutants                                        935 <br />  <br />  <br />                                                                TABLE 1 <br />                                                                strain list <br /> ber                     Strain <br />                            NF134                        MATa his4-539 lys2-801 ura3-52 <br />                            NF147                        MATa ade2-101 ura3-52 <br />                            NF3623                       MATa ade2-101 <br />                                                         -        ~     his3-200 leu2-3,112 <br />                                                                                    _       lys2-801- <br />                                                                                            &quot;        trpl-1 ura3-52 <br />                                                                                                     _                   _ <br />                                                         MATa     + his3-200 leu2-3,112          +   t q l - 1 ura3052 <br />                            NFVl1                        MATa his4539 lys2-801 ura3-52 van61 1 <br />                            NFVl4                        MATa his4539 lys2-801 ura3-52 uan2-14 <br />                            NFVl8                        MATa his4-539 lys2-801 ura3-52 uanl-18 <br />                            NFV93                        MATO ade2-101 ura3-52 uan2-93 <br />                            NFV96                        MATa ade2-101 ura3-52 van5-96 <br />                            NFV97                        MATa ade2-101 ura3-52 van497 <br />                            NFVl12                       MATa his4-539 lys2-801 ura3-52 uan5-112 <br />                            NFvl17                       MATa his4-539 lys2-801 ura3-52 van3-117 <br />                            NFvl20                       MATa his4-539 lys2-801 ura3-52 uan5-120 <br />                            NFvl12                       MATa his4-539 lys2-801 ura3-52 van4-122 <br />                            NFVl28                       MATa his4-539 4.52-801 ura3-52 van5-128 <br />                            NFVl33                       MATa his4-539 lys2-801 ura3-52 uan5-133 <br />                            NFV146                       MATa his4-539 lys2-801 ura3-52 uan5-146 <br />                            NFV150                       MATa his4-539 lys2-801 ura3-52 uan5-150 <br />                            NFVl52                       MATa his4-539 lys2-801 ura3-52 van4152 <br />                            NFV153                       MATa his4539 lys2-801 ura3-52 uan4-153 <br />                            NFV2 18                      MATa ade2-101 ura3-52 vanl-18 <br />                            NFV220                       MATa urg6 (mnn9) mnn2 ura3 <br />                            NFV221                       MATa vrg2 (mnnlO) mnn2 ura3 <br />                            NFv222                       MATa urg3 (mnn8) mnn2 ura3 <br />                            NFV223                       MATa urgl mnnl ura3 <br />                            NFV224                       MATa wg4 mnn2 ura3 <br />                            NFV293                       MATa his4539 ura3-52 van2-93 <br />                            NFV294                       MATa his4539 lys2-801 <br />                                                         ~                   ~  ura3-52 van2-93 _     +                  _ _ <br />                                                         MATa     +           + ura3-52      +    ade2-101 <br />                            NFv917                       MATa ade2-101 ura3-52 van3-117 <br />                            NFv920                       MATa ade2-101 ura3-52 van6920 <br />                            NFV923                       MATa ade2-101 ura3-52 uanl-923 <br />                            NFV936                       MATa ade2-101 ura3-52 van4936 <br />                            NFV941                       MATa ade2-101 ura3-53 vanl-941 <br />                            NFV949                       MATa ade2-101 ura3-52 uan5-949 <br />                            NFV956                       MATa ade2-101 ura3-52 van6956 <br />                            NFV963                       MATa ade.2-101 ura3-52 uan4-963 <br />                            NFV969                       MATa ade2-101 ura3-52 van6969 <br />                            NFV980                       MATa ade2-101 ura3-52 vanl-980 <br />                            NFV981                       MATa ade2-101 ura3-52 uan3-981 <br />                            RSY271                       MATa secl8-1 ura3-52 his4-619 <br />  <br />  <br />                                                                         tant mutants genes. Our collection different <br />                                  TABLE 2                                contains large percentage dominant mutants <br />                                                                         reported group, unusual <br />               Classification vauadatexesistant mutants               type recessive mutation (van3 117, below). <br />                                                                            The ?&apos;ANI+ gene product significant amino acid <br />                Mutation <span id='am-13' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-14' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span>                No. isolates <br />                                                                         identity MNN9+ ANp1+ gene products:Work <br />       mutations <br />              Dominant                              32                   MacKay (YIP et al. 1994) BALLOU <br />             VAN1                                    4                   (BALLOUet al. 1991) laboratories prompted comparison <br />             VAN2                                    3                   vanadate-resistant mutants mnn <br />             VAN3                                   10                   phenotypes. The &quot; V 9 + gene isolated and <br />             VAN4                                    8                   sequenced (YIP et al. 1994). The predicted VANl+ and <br />             VAN5                                   14 <br />             VAN6                                    7 <br />                                                                         MNN9+ gene products limited amino acid iden- <br />             Other                                   3                   tity, especiallyin carboxyl-terminal ends two <br />                                                                         predicted amino acid sequences (Figure 1) (YIP et al. <br />  936                                                                   C.   Kanik-Ennulat, E. Montalvo N. Neff <br />  <br />  <br />  <br />   gpcHm <br />  <br />   gpVANl <br />                           r&#8216;                                                                                                 535 <span id='am-63' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-64' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-67'  ></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>aa</span>&#8217;s <br />  <br />  <br />  <br />                                                                                                                              493 an&#8216;s <br />  <br />  <br />                                                                                                                                             FIGURE1.-Amino acid sequence identitv <br />                                                                                                                                           predicted gene products cloned <br />                                                                                                                                           genes MNN9,VANlandANPI.          Position a <br />                                                                                                                                           region predicted amino acid identity (top) <br />                                                                                                                                           open reading frame gene. <br />                                                                                                                                           The Q22 marked carboxyl-terminal re- <br /> gpMNN9 199   ERHALDVQKERRAAMALARNELLFSTIGPKTSWVLWLDAIIETPPSLIQDMTKHNK-AI                                                                  gion gpANPl indicates presence o f 22 <br />                .. .. .. .. .. .. .. .. .. .. ..                                 .. .. .. .. .. .. .. .. .. .. .. .. ..                    consequtiveglutamineresidues       t h e pre- <br /> @AN1 324     DRHAVKV~IRRKLMGRARNWITANAIKPYHSWV~D~VELCPGS~QDLM-SKNYDV                                                                      dicted protein sequence. The amino acid se- <br />              .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. ....................        .. .. .. .. ..        quence alignments abetween thc genes <br /> gpANPl 167   DRHGFGAQGPRRKLMARARNLTANALKPYHSWVYWRDVDVELIPTTIMEDLMQSGQGNV                                                                  displayed (bottom). Amino acid identities <br />                                                                                                                                           indicated colons. The original database <br />              LAANIYQRFYDEEKKQPSIRPYDLNNWQESDTGLEIASQMGDDEIIVEGYAEIATYRPLM                                                                 submission VANZ gene sequence was <br />                                                          .. .. .. .. .. .. ..... .. .. . . .. .. .. . . . . . . .                         missing &#8220;C&#65533;? residue sense strand be- <br />              IVPNVWRPL: - -PTFLGTEQPYDLNSWMESQEALALMTLDEDDVIVEGYAEYPTWRVHL                                                                tween nucleotides 1748 1749 (USINK-EN- <br />              .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. ..   NUIAT NEFF 19%)). The   second open read- <br />              IVPNVWRPL- - -PMJLGNTQPYDFNSWKESEGGLQLADSLDEDAVIVEGYPEYATWRPHL                                                               ing frame (ORF2) identified KIN1 is <br />                                                                                                                                           ATRl (KANMAWA et nl. 1988). predicted <br />                                                                                                                                           transmembrane protein gene mutate <br />              AHFYDANGVPGEEMALDGVGGGCTLVKAEVHRDGAMFPNFPFYHLIETEGFAKMAKRLNY                                                                 aminotriazole resistance. <br />              ..     ..    .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. .. <br />              AYIRDAEGDPNEAVDLDGVGGVSIW(AK1FRNGVQFPAFTFENHAETEAFGKMAKKm;Y <br />               ..   ..    .. .. .. ..        .............................. <br />                                               ............................                                    .. .. <br />              AYMRDPNGNPEDEMELffiIGGVSIW(AKVLRTGSHFPAFSFEKHAETEAFGRLSRRMNY <br />  <br />  <br />              DVFGLPNYLVYHIEEENH-                           395 gpMNN9 <br />               .. .. .. .. .. .. .. .. .. <br />              RVGGLPHYTIWHIYEPSDDDLK 521 gpVANl <br />                ................... <br />                  ................. <br />              NVIGLPHYVIWHIYEPSSDDLK 365 gpANPl <br />  <br />  <br /> 1994). These gene products unexpected                                                                       initial report (KANIK-ENNUIAT    NEFF 1990) <br /> identity predicted gene product  ANPI+                                                                  consistent alterations posttranslational mod- <br /> gene (MELNICKand SHERMAN         1993). All genes                                                                 ificationof membrane-proteincomplexes. The anti- <br /> are nonessentialforhaploid     cell growth. The VAN1                                                                    body cross-reacted sugar-phosphate linkages and <br /> and ANPl gene products 70% identical                                                                    screen picked cells accumulating unknown <br /> region both40% identical toMNN9. The VAN1                                                                       phosphorylated sugar-protein modification. Changes <br /> and MNN9 gene productshave similar carboxyl-termi-                                                                    posttranslational modifications proteins lead <br /> nal position homologous region                                                                    misfolding alterations specific activity in <br /> open reading frames, ANPl gene prod-                                                                     membrane-associated proteins (HAMMOND        e/ nl. 1994). <br /> uct region positioned centrally open                                                                    Cells containing mutations ANPl gene have <br /> reading frame (Figure 1).                                                                                               phenotypic similarities unnl mnn9 mu- <br />    Along sequence identity pheno-                                                                   tant$ drugresistant (aminonitrophenyl pro- <br /> types commonbetween         mnn9 mutants (BAI.I.OU                                                                   panediol), clumpy osmotically sensitive (MCKNIGHT <br />  1982) van1 mutants (KANIK-ENNUIAT        NEFF                                                                  P/ nl. 1981). This conserved regionof -200 amino acids <br />  1990) (Table 3, Figure 2). Both types mutant cells                                                                  common phenotypes recessive mutations sug- <br /> are resistant high levels sodium orthovanadate,                                                                   gests domain nonessential dis- <br /> clumpy, osmoticor detergentsensitive, hygromycin sen-                                                                   tinct genes similar o r overlapping osmotic f h c - <br /> sitive secrete underglycosylated invertase (Table 3,                                                                tion cell. <br /> Figure 2). Some combinations ofalleles sporulate                                                                      Analysis invertase vanadate-resistant mutants: <br /> two viable deformed nonviable spores(BALLOU                                                                     Mutants collection vanadate-resistant mu- <br />  1990; KANIK-ENNULAT    NEFF1990). The phenotypic                                                                   tants (Figure 2, lanes d-f h), wg o r mnn. <br /> alterations ourvanadate-resistant mutants described                                                                  mutants (lanes b c) analyzed active in- <br />                                                    \&apos;anadate-Resistant Yeast Mutants                                              937 <br />  <br />                                                                 TABLE 3 <br />                                                   Phenotypes van zng mutants <br />  <br />                                                                                                 YPD+          SD+ <br />                              YPD+           YPD+        YPD +         YPD+          YPD+         \Q      :&apos;   \&apos;o.l:&apos;   Invertase <br />   Genotype         YPD       10 Hyg        2.5 H y g    50 Hyg       100 Hyg      0.1% Doc       (nnl)        (mu)        cl;1ss <br />  <br /> M&apos;t MA Ta           +           +             +            +              -            +         1-3           1-2         IVt <br /> M&apos;t &apos;\M Ta          +           +             +           +/-             -            +         1-3           1-2         IVt <br /> 7 l N n 1-18        +           -             -            -              -            -         6- I O       6-8        n/ I <br /> rrml-923            +           -             -            -              -            -         4-6          3-4        9/ 1 <br /> z q 6 (m n n 9)     +           -             -            -              -            -         6- 10        6-8        9/ 1 <br /> 71rrn2-93           +           +             -            -              -           +/- <br />                                                                                       2-3        3-4                     9/ I <br /> r1nn2-14            +           +             -            -              -           +/-        3-4          2-3        9/ 1 <br /> 11nn3-1I7           +           +             +            -              -            +         3-4          2-3          \VI <br /> lInn3-98I           +           +             +            -              -            +         3-4          2-3          \$?t <br /> lI/lIl4-97          +           +             +            -              -           +/-        4-6          3-4        9/ 1 <br /> lMlll5-96           +           +             +           +/-             -            +         3-4          2-3        9/ 1 <br /> 7111 n 5- I50       +           +             +           +/-             -            +         3-4          2-3        9/ 1 <br /> vnnh-l I            +           +             +           +/-             -            +         3-4          2-3        8/ 10 <br /> wrnh-920            +           +             +            -              -            +         3-4          2-3        8/ I O <br />  V A W - I I6       +          +/-            -            -              -            -         6-10         6-8        9/ 1 <br /> 71rg.2(mnn I O )    +           +             -            -              -            +         3-4          2-3        8/10 <br /> vrg3 (mn18)         +           +             -            -              -            +         3-4          2-3        8/10 <br /> urgI/vrg7           +           +           +/-            -              -           +/-        3-4          2-3        9/ 1 <br /> z1rc4/71rc5         +           +           +/-            -              -           +/-        4-6          3-4        9/ 1 <br />   Strains wcre grown o n 2.5-mI YPD plates SD plates 26&quot; additions. Hygromvcin (Hyg) addrrl to <br /> platesfrom I O 100 pg/ml. Sodium rleoxycholate (Doc) addedto plates 0.01-0.5%. Sodium orthovanatdate <br /> (V0.T:&apos;) addccl plates 1 10 m x t . The concentrations reported highest level vanadate cells <br /> c m grow 26&apos;. The invertase class indicates approximate degreeof invertase glycosylation. Wt, wild-typeheterogeneous <br /> profile; 8/10, intcrmediate truncation glycosylation profile similar seen m n n 8 mnnl0 mutants; 9/1, <br /> greater truncation o f invertase glycosylation profile similar seen mnn9and mnnl mutants (BAIMX.PI 01. 1991). <br />  <br />                                                                      vertase native gel electrophoresis. Mutant swl8-I <br />                 b c d e f g h                                      cells grown permissive (26&quot;; lane a) restrictive <br />                                                                      (57&quot;; lane g) temperatures grown parallel as <br />                                                                      controls. Wild-type cells, secl8-1 mutant cells grown <br />                                                                      permissive temperature, secretea heterogeneous <br />                                                                      population glycosylated invertase (Figure 2, lane a). <br />                                                                      The heterogeneous size secreted invertase to <br />                                                                      theaddition 2 5 0 mannoseunits ouhvard <br />                                                                      bound protein proceeds        Golgi (KL&apos;Nw,- <br />                                                                      ZINSKA et nl. 1987). At restrictive temperature cells <br />                                                                      sec18-I mutation blocked secretory <br />                                                                      process level endoplasmic reticulum Golgi <br />                                                                      transport accumulate core ER glycosylated in- <br />                                                                      vertase (ESMONet nl. 1981) (Figure 2, lane g). <br />                                                                         As shown BAILOU associates ( B A I , I , o 1982, <br />                                                                                                                            ~~ <br />                                                                      1990; BAILOUpt nl. 1991), cells containing mnn9 ( 7 q 6 ) <br />                                                                      mutations secrete underglycosylated invertase relative <br />                                              V                       wild-type cells (Figure 2, lane b). In mnn9 mu- <br />                                                                      tants, secretedinvertase core orendoplasmic reticu- <br />                                                                      lum modification, blocked early stage of <br />                                                                      outer chain elongation. Cells containing mnn8 ( v r g l ) <br />                                                                      mnnlO (vrg2) mutations secrete invertase, is <br />   FIG~IU    2.--Native <br />               ~.        gel electrophoresis activein-         somewhat larger mnn9 cells sig- <br /> vertase n cell extracts representative strains. (a)     nifican tly underglycosylated (BAI.I.OL~ et crl. 1991) (Fig- <br /> swl8-I cells induced 26&quot;; (g) m 1 8 - I cells induced 37&quot;; <br />                                                                      ure 2, lane c). <br /> (h) vrg6 (mnn9) cells; (c) vrg2 (mnnl0) cells; (d) unn5-150 <br /> cells; (e) rmn2-93 cells; ( f ) v n n l - 1 8 cells; (h) VAN7-116       We distinct invertase glycosylation pro- <br /> cells. The internal form invertase rims dye        files classes vanadate-resistant mutants. Cells <br /> front shown.                                                  containing van1 mutations produce invertase (Figure <br />  938                                     C. Kanik-Ennulat, E.and <br />                                                              Montalvo <br />                                                                   Neff   N. <br />  <br /> 2, lane f ) similar pattern underglycosylation       (1991). The resistant mutations v n l - 1 8 and <br /> as mnn9 vrg6 (lane b). Cells containing van2        mnn9 (vrgb). These strains resistant 210 mM <br /> mutations (lane e), van4 (not shown), VAN7-116 (lane           Na3V04 YPD media. Other classes mutants show <br /> h) mutations secrete underglycosylated invertase       intermediate levels resistance (Table 3). All strains <br /> well. Other vanadate-resistant mutants (vun3-117           resistant SD media (1-2 mM Na3V04lower, <br /> van3-981) showed wild-type distribution glycosyl-         Table 3). Both dominant andrecessive mutants high <br /> ated invertase (not shown). Cells containing van5          levels vanadate resistance severely af- <br /> vun6 (Figure 2, lane d) mutations produced invertase           fected level cell viability, sporulation, hygro- <br /> that nearly identical distribution somewhat              mycin-sensitivity pattern secreted protein glyco- <br /> smaller wild type, similar glycosylation pro-      sylation. <br /> file seen mnn8 mnn10 mutants. These results                The VAN3 class recessive mutations un- <br /> and phenotypes summarized Table 3.                usual alleles: The vanadate-resistant mutants show <br />    Many van mutants hygromycin sensitive           lowest level resistance highest levels of <br /> (Table 3 ) vrg mutants reported BALLOU          spore viablilty, secrete invertase, iswildtype in <br /> et al. (1991). All strains tested grew onYF&apos;Df1Msorbitol       appearance (Tables 3 4). The van3 mutants are <br /> plates showed differential sensitivity WD+O.l%          wild type phenotypes tested un- <br /> sodium deoxycholate plates.All strains tested resis-      usual complementation alleles. <br /> tant toYPD+O.l% Triton X-100 0.5% Tween20 plates.              One recessive mutation (van3- 11 7) failed comple- <br /> The significance sensitivity mutants     ment nearly recessive mutation. This type <br /> known. The hygromycin deoxycholate sensitivities           mutation times collection of <br /> may leaky incorrectly assembledmembrane-          NF134derived mutants twice NF147-derived <br /> protein complexes. These phenotypes           mutants. By tetrad analysis mutation unlinked to <br /> reflection general osmotic-sensitivity asseen loss     mutants tested NFV981 (Table 4). Multi- <br /> of viability long-term liquid culture.                      ple examples NFV981 mutation (complements <br />    Spheroplasts wild-type, secl8-1 (26          classes NFVl17) the <br /> and 37&quot;), vanl -18, van2- 93 mnn9 cells test            NF147-derived mutations (NFV929). This unexpected <br /> whether underglycosylated invertase           complexity complementation analysis this <br /> an intracellular periplasmic compartment. All wild-         particular allele class vanadate-resistant muta- <br /> type vanadate-resistant strains tested secreted glyco-     tions partial dominant phenotype or <br /> sylated invertase. Only unglycosylated intracellular       reflect phenotypic interaction dis- <br /> form invertase cell wall di-        tinct genetic loci. Thetetrad analysisshown is <br /> gested, 37&quot; blocked secl8-1 cells,         totally exhaustive does demonstrate the <br /> accumulate core glycosylated intermediates intracellu-         phenotypic complementation vanadate resistance is <br /> larly (<span id='am-69' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-70' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-73'  ></span>data</span> shown). Thevanadate-resistant mutants          simple. <br /> export underglycosylated invertase    accumu-          Most van mutations cause sporulationdefects Many <br /> late intracellularly.                                       spontaneous mutations isolated showed domi- <br />    The effect vanadate secretion invertase        nant vanadate-resistance phenotype. This class ofmuta- <br /> in wild-type cellswas tested. The kinetics profile      tion reported investigators (WILL <br /> invertase compared wild type, NFV18 NFV93           SKY et al. 1984;BALLOU et al. 1991).Tetrad analysis <br /> induced presence absence 2 mM sodium              proved difficult sporulation efficiency ~ 1 0 % , <br /> orthovanadate. Addition vanadate wild-type re-        sporeviability 10-25%. One dominant mutant <br /> sistant mutant cells effect time course        (NFV116) analyzed tetrad analysis not <br /> invertase induction glycosylation pattern      linked vanl, van2, van3 van4 mutants tested (Ta- <br /> 4 h r induction time. The invertase induced          ble 4).The heterozygous diploid didnotsporulate <br /> was appreciably different strains          mated van5 van6 mutant strains. <br /> tested presence absence vanadate (0.5-0.8         This mutation interesting phenotypic properties <br /> pmol glucose released lo6 cells). These results             (Figure 2, Table 3) tentatively designated VAN7- <br /> agree observations BALLOU      (1990) un-      I 16. This strain showed high level vanadate resis- <br /> derglycosylation secreted invertase mnn mu-          tance clumpy cellsand growth properties simi- <br /> tant cells does notaffect amountor specific activity.      lar van1 -18 mnn9 mutants (Table 4).We are <br />    These results collection mutants de-       unable say precisely genes represented <br /> scribed havesimilar invertase glycosylation pat-          dominantmutants inconsistent aspects <br /> terns seen mnn mutants. In             poor sporulation. <br /> general inverse correlation degree              Heterozygous diploid strains wild-type and <br /> of vanadate-resistance extent outer chain           dominant mutations sporulated YPD YF&apos;D + 3 <br /> elongation. Different degrees vanadate resistance       mM vanadate plates low frequency (&lt;().I%). After <br /> found different mutants seen BALLOU      et al.   weeks plates 26&quot; red sectors appeared in <br />                                                 Vanadate-Resistant Yeast Mutants                                           939 <br />  <br />                                                            TABLE 4 <br />                                         Tetrad analysis vanadateresistant mutations <br />  <br />                                                      No. complete tetrads <br />           strain <br />              Diploid                    4:O         3: 1       2:2           1:3        0:4        Spore <br />                                                                                                viability         (%) <br />         WT (Y x uan3-117                  0          0          16            0          0                  93 <br />         uan3-117 X uanl-18                2          7           1            0          0                 100 <br />         uan3-117 X uan2-93                1          8           1            0          0                 100 <br />         uan3-117 x uan4-97                2          6           2            0          0                 100 <br />         uan3-117 X uan5-96                2          5           2            0          0                 85 <br />         uan3-117 X van6920                1          5           0            0          0                 75 <br />         uan3-117 X uan3-981              10          0           0            0          0                 100 <br />         WT (Y X VAN7-116                  0          0           4            0          0                 50 <br />         VAN7-116 X uanl-18                0          2           0            0          0                 20 <br />         VAN7-116 X uan2-93                0          2           0            0          0                 20 <br />         VAN7-116 X uan3-981               0          0           3            0          0                 25 <br />         VAN7-116 X uan4-97                0          2           1            0          0                 25 <br />              Tetrad columns vanadate-resistant: vanadate-sensitive spores. <br />  <br /> the white diploid cell patches heterozygous forthe                   taining 1 mM sodium orthovanadate. Ten large white <br /> a&amp;2- 101 marker. These vanadate-resistant mutations                  colonies thousands smaller <br /> in diploid cells allow normal sporulation con-               yellow ones. Upon retesting URA&apos; plasmid trans- <br /> trols bypassed low frequency. Heterozygous                formant colonies showed consistent sensitivity SD + <br /> and homozygous diploids containing van1 - 18                 2 mM sodium orthovanadate. These transformants were <br /> uan2-93 mutations, diploids containing null               grown total nucleic acid extracted. Aliquots of <br /> alleles genes, showed phenotype also. The              preparations transformed CaC12-treated <br /> exact cause sporulation phenotype unknown.                <span id='am-24' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-34'  ></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span>Escherichia coli</span> cells <span id='am-79' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-80' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="plasmids" datatype="xsd:string"></span>plasmids</span> recovered by <br /> It possible sporulationdefects seen com-             selection ampicillin-resistant colonies. <br /> binations vanadate-resistant mutants struc-               Three related plasmids wereisolated library. <br /> tural defects spore coats ascus walls,               Restriction enzyme analysis revealed overlapping yeast <br /> allow premature germination. BALLOU(1982) char-                  genomic Sau3A DNA fragments plasmids <br /> acterized defects ascus walls mnn mu-              (pNV93-3,4,6, Figure 3). A complete restriction <br /> tants mutate tovanadate resistance. Other yeast             map plasmid pNV93-6 generated, series <br /> mutants sporulate presence glucose                dropout deletions (pNV93-9,lO) subclones <br /> have reported (ESPOSITOand KLAPHOLTZ 1981).                      (pNV93-12,14,15) generated. The resulting plas- <br />    The diploids vanadate-resistant genes                 mids transformed NFV93 tested vana- <br /> identified identified BALLOU       et al.          date sensitivity/resistance. All plasmids tested comple- <br /> (1991) sporulated poorly all. Independent Ura-                 mented uan2-93 mutation. These data showed the <br /> isolates ofurgstrains isolated tested confirm            complementation activitywas contained 2.2-kb <br /> vanadate resistance. These strains (NFV220-224)                 EcoRI/HindIII fragment. This fragment direction- <br /> transformed   YCp50       (vector control),pNVll                ally cloned phage <span id='am-59' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-60' property="rdfs:label" content="vectors" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>vectors</span> Mp18and Mp19 DNA <br />          + <br /> (YCp50 VANlf) pNV93-3 (Figure 3, YCp50 +                         sequence analysis. <br /> VAN2+) plasmid DNAs. None Ura3&apos;trans-                            The single-stranded phage DNAs sequenced <br /> formants vanadate sensitive. None vanadate-              sequential oligonucleotides spaced approximately <br /> resistant mutations urgstrains tested compli-            every300 nucleotides. The open reading frame <br /> mented wild-type VANI+ VAN2+ genes.                        &gt;25 codons predicts protein 337 amino acids (Fig- <br />    Isolation VAN2+ gene: Haploid cells con-                   ure 4). The predicted gene productof VAN2+ open <br /> taining uan2-93 mutation turn yellow SD plates                reading frame showed significant amino acid iden- <br /> containing 1-3 mM sodium orthovanadate, grow                     tity protein sequence time sequence <br /> more slowly wild type. Heterozygous diploid cells               submission. The open reading frame      predicts protein <br /> (uan2- 93/ +) white SD plates grow plates              multiple potential transmembrane domains. The <br /> containing 1 mM sodium orthovanadate. A YCp50li-                     EcoRI/HindIII fragment used hybridization <br /> brary Sau3A wild-type yeastgenomic DNA fragments                  probe yeast chromosomes transferred nylon <br /> (SHIHet aL 1988) transformed NFV293 cells,                  filter. Chromosome XVDNA bound probe (<span id='am-74' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-75' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-76'  ></span><span id='am-77' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>hybrid</span>- <br /> selecting URA+ colonies SD media. The resulting                   ization data shown). <br /> 25,000 transformant colonies washedoff plates                  The DNA fragment identilied library con- <br /> and pooled. These cells plated SD plates con-                tains VAN2+ gene: PlasmidpNV93-12 con- <br />  940                                         C. Kanik-Ennulat, E. Montalvo N. Neff <br />  <br />  <br />  <br /> 4 ; <br />       P <br />           H <br />           lkb         H B E <br />                       I <br />                       I <br />                           I <br />                           I <br />                               1 <br />                               1 <br />                                   - <br />                                   Sn <br />                                   I <br />                                   I <br />                                        Sn <br />                                         I <br />                                         I <br />                                             H P S p B <br />                                              I 1 <br />                                              II 11 <br />                                                      I   1 <br />                                                              X <br />                                                              I <br />                                                              I <br />                                                                  X <br />                                                                  1 <br />                                                                  1 <br />                                                                      X <br />                                                                      I <br />                                                                      1 <br />                                                                            * <br />  <br />  <br />  <br />  <br />                                                                            pNV93-3           FIGURE3.-Restriction map the <br />                                                                                           KAN2 locusandplasmidconstruc- <br />                                                                            DNV93-4 <br />                                                                            I   ~~ <br />                                                                                           tions. Theupper   represents <br />                                                                                                          line         the <br />                                                                                           region yeast genomic&apos;DNA fomd <br />                                                                            pNV93-6        the <br />                                                                                                three <br />                                                                                                   complementing            library <br />                                                                                           plasmids. Restriction enzyme ahbre- <br />                    P                                                       pNV93-9        viations P, PUuII; H, HindIII; l3, <br />                                                                                           BgflI; E, EmRI; Sn, SnnB1; Sp, SphI; X, <br />                                                                            pNVg3&quot;10       Xl,nl. T h e position <br />                                                                                                               of <br />                                                                                                                the <br />                                                                                                                  predicted <br />                                                                                           open reading frame VAN2 gene <br />                                                                            pNV93-12       shown small arrow. <br />                                                                            pNV93-14.15 <br />  <br />                               7                                            pNV93-16 <br />  <br />  <br />  <br /> structed cloning 3 k b Hind111 fragment                  gonucleotides flanking SnnBlsiteswereused           to <br /> integrating vector Mp5. This plasmid contains unique                   analyze genomic DNA diploid transformants <br /> BglII site yeast genomic DNA outside pre-                  polymerase chain reaction technique. Only two <br /> dicted coding region VAN2&apos; gene (Figure 3,                       diploid transformants examined contained both <br /> pNV93-12). PlasmidpNV93-12 DNA linearized                            wildtype (1.3kb DNA fragment) null allele <br /> with restriction enzyme BglII used transform                      (1.5-kb DNA fragment; data shown). The others <br /> NFV294 selecting Ura+ transformants. Two indepen-                        contained wild-type size allele likely <br /> dent diploid transformants sporulated tetrads                   represent conversion events t @ I - l locus. <br /> dissected. In cases tetrads (20/20) contained                      Both diploid transformant strains sporulated poorly <br /> two Ura&apos; Ura- spores. In case, spores                    (&lt; 10% relative wild-type diploid). Of 20 <br /> were vanadate sensitive. In case, Ura+ spores                  tetrads analyzed diploid, or <br /> were vanadate sensitive ura- spores vana-                   viable spores. Sixteen tetrads viable <br /> date resistant. All auxotrophic markers MAT                      spores noviable spores. The invia- <br /> locus segregated 2:2. In case integration                  ble spores failed germinate YPD dissection <br /> event occurredonthechromosomecontainingthe                               plate, remaining small refractile. All viable spores <br /> van2-9? mutation. In second case, integration oc-                    trp- (34/34). Other auxotrophic        markers were <br /> curred chromosome containing        wild-type                 2:2. This result demonstrates essential nature the <br /> gene.Thesedatademonstratethatthefragment                              open reading frame identified VAN2+ gene for <br /> yeast genomic DNA isolated white-yellow colony                    haploid cell viability. <br /> color screen contains VAN2+ gene.                                       Sequence vun2-93: The van2-93 gene ampli- <br />    The predicted open reading frame      VAN2&apos;                   fied NFV293 cells. A comparison wild-type <br /> gene contains SnaBl restriction enzyme sites                  (NF134) mutant sequences revealed single G to <br /> DNA sequence. These sites unique plas-                    A change position 1010. The wild-type protein se- <br /> midpNV93-12. Dilute religation SnaBl digested                         quence predicts arginine residue (CGT = Arg) at <br /> pNV93-12 DNA creates deleted VAN2 + gene                        position 274 mutant sequencepredict5 histi- <br /> predicted protein product approximately 60 amino                      dine residue (CAT = His). This charged amino acid <br /> acids (pNV93-12s). Plasmid DNAs pNV93-12                             anchor transmembrane domain (Figure 4). The <br /> pNV93-12s digested restriction enzyme BglII                    alteration arginine histidine position <br /> and used transform vanadate-resistant strain NFV293                   effect overall function structure this <br /> to Ura+. All Ura&apos; transformants tested                   essential gene product, butmay create leaky unsta- <br /> integration pNV93-12 DNA strain NV293were                          ble structure. A histidine residue potential <br /> vanadate sensitive (20/20). All Ut-a&apos; trans-                      hydrophobic character does arginine residue, <br /> formants integrationof pNV93- 12s DNAwere                       effective transmembrane anchor <br /> vanadate resistant (20/20). These results demonstrate                    stoptransfer sequence. <br /> that open reading frame identified VAN2&apos; <br /> gene contains complimentingactivity reces-                                          DISCUSSION <br /> sive mutant van2-9? vanadate-resistant gene. <br />    The V M 2 + gene essential haploid cell viabil-                  Sodium orthovanadate toxic yeast cellsand spon- <br /> ity Strain NF3623 transformed selecting Trp&apos; us-                     taneous mutations vanadate resistance arise a <br /> ing van2 TRP+ null allele construction. Twooli-                      number genes. The target structure in <br />                                                      Vanadate-Resistant Yeast Mutants                                           941 <br />  <br />  <br />  <br />   61   AACGAAAAACAUTATATATMCGCTGCCAAGCCTATTTCTTGTTGCCGATTAATTGTT <br />  <br />  121   GACATCCTCTTAATTACCAAAAGAWCTAAGAUACAAACACACTMCCACACAGTACCT <br />  <br />  181   TTCWCCGAATGTCTWTTG~CAGGTCATGCAWCATAACCCm~TTCAGTT <br />    1                M   S   E   L   K   T   G   H   A   G   H   N   P   W    A   S   Y <br />  <br />  241   GCACCAATTCCGGTCCGATCTCTATTTTATCCTATCCTACTGTGGTTCCTCCATTTTAATGACGGTG <br />   18    &amp; N S G P I S I L S Y C G S S I L ~ T Y <br />  <br />  301   ACTAACAAGTTCGTCGTCAACTTGAAGGATTTCAACATGAACTTTGTTATGCTTTTCGTG <br />   38    U K F V V N L K D P N M N F V M L F V <br />  <br />  361   CAATCTTTGGTTTGTACTATAACCTTGATTATC-TACTWCTATWWGTTC <br />   58    Q   S   L   V   C   T   I   T   L   I   I   L   R   I   L   G   Y    A   K   F <br />  <br />  421   CGTTCATTAAACAAAACAGACGCCAAGAACTGGTTCCCTATTTCCTTTTTACTGGTCTTG <br />   78    R S L N K T D A K N W F P I S F L L V L <br />  <br />  481   ATGATCTACACCTCTTCOAAGGCTTTACAATACTTWTGTTCCAATTTACACCATTTTC <br />   98    M   I   Y   T   S   S   K   A   L   Q   Y   L   A   V   P   I <br />                                                                        FIGURE4.-DNA sequence VAN2 <br />                                                                         Y    T   I   F <br />                                                                     gene. The DNA sequence predicted trans- <br />  541 AAOMTTTTGACTATTATCTTGATTGCTTATOOTOAOOTTCTCTTTTTT~TGGCTCTGTC lation open reading frame shown. <br />  118  K N L T I I L I A Y G E V L F F G G S V                       The SnaBl sites usedto make null construc- <br />                                                                     tion (pNV93-16) dropout deletion <br />  601 ACCTCCATGGAATTGTCATCATTTTTGTTGATGGTCCTTTCTTCTGTCGTTGACCAACTTGG (pNV93-12s) underlined. Seven potential <br />  13 8 T S M E L S S F L L Y V L S S V V A T W                       transmembrane domains 20-25 amino acids <br />                                                                     underlined also. Potential Nglycoslylation <br />  661 GGTGACCAGCAAGCTGTGGCTGCCAAGGCTGCTTCATTTGGCTGAAGQAGCAGCCGGTGCT  sites (NXT/S) marked asterisks. The po- <br />  158  G D Q O A V A A K A A S L A E G A A G A <br />                                                                     sition van2-93 mutation nucleotide <br />  721   GTTGCCTCCTTTAACCCAGGTTATTTCTGQATGTTCACCAACTGTATCACTTCTGCATTA                      &apos;010             The       DNA sequence <br />  178    V   A   S   F   N   P   G   Y   P   W   W   F   T   N   C   I   T    S   A   L <br />                                                                                          deposited GenBankAccession <br />                                                                                          Number U15599. This sequence to <br />  781   TTCGTTCTTATMTGAGUAGAGAATTAAGTTAACTAACTTCAAWATTTCGACACTATG                         benearly identical L33915 (GOG5). <br />  198    p V L I l 4 R K R I K L T N F K D F D T y <br />  <br />  841   TTTTACAACAATGTTTTGGCTCTACCTATTCTATTATCTGCTGTTTTCTTTCTGTGTGGMGAT <br />  218    F Y N N V L A L P I L L L F S F C Y E D <br />  <br />  901   TGGTCTTCAGTTAATTTGACACCAATAACTTTTCTMCGATTCGCTAACTGCTATGATCATC <br />  238    W   S   S   V   N   L   T   N   N   F   S   N   D   S   L   T   A    M   I   I <br />                         *               * <br />  961   AGTGGTOTTGCATCCGTCGGTA~TCTTACTGTTCCOOTTTTCG <br />  258    S   G   V   A   S   V   G   I   S   Y   C   S   G   W   C   V   R    V   T   S <br />  <br /> 1021   TCTACTACATATTCGATGGTAGGGGCTTTGMCAAGCTGCCAATTGCCTTGTCTGGTTTG <br />  278    S   T   T   Y   S   Y   V   G   A   L   N   K   L   P   I   A   L    S   G   L <br />  <br /> 1081   ATTTTCTTTOAGTCTCCCTTCTTATCTATCTATT~CTCCATTTTTATTGGTTTCCTA <br />  298    I   F   F   D   S   P   R   N   F   L   S   I   L   S   I   F   I    G   F   L <br />  <br /> 1141   TCAGGTATTATTTATGCTGTTGCCAAAUGAAGCAACAAGCCCAACCTT- <br />  318    S   G   I   I   Y   A   V   A   K   Q   K   K   Q   Q   A   Q   P    L   R   K <br />  <br /> 1201   TGAGAACTTACGGGGGGTOTTATTTTTTTTTTTTTGGTTTATTTTATTTTTATA <br />  <br />  <br />  <br /> uivo toxic molecule unknown, logically mono-                   mechanism accumulates toxic molecule. <br /> meric orthovanadate compete monomeric or-                          Growth yeast cellsin 5 mM orthovanadate causes the <br /> thophosphate active uptake metabolism                         accumulation oftoxic vanadium compounds un- <br /> mimics structure. Many enzymes form <span id='am-84' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-85' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="Phos" datatype="xsd:string"></span><span id='am-87'  ></span><span id='am-88' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>phos</span>-                         known structure wild-type cells vanadate- <br /> phoenzyme intermediates sensitivityto orthovana-                       resistant cells. This work ourwork (KANIK-ENNULAT <br /> date. Given large number enzymes                         NEFF1990) demonstrate representative vana- <br /> cell, unlikely mutationof single                    date-resistant mutants alterations phos- <br /> would create overall cellular resistance.                                phate uptake. These results make unlikely vana- <br />    The work WILLSKYand collaborators (WILLSKY           et               date resistance yeast isdue inhibition <br />                                                                                                                            uptake <br /> al. 1984, 1985) demonstrates vanadate resistance                       orthovandate. <br /> in yeast likely arise alteration cellular                 The phenotype underglycosylated invertase inthe <br />  942                                    C. Kanik-Ennulat, E.and <br />                                                             Montalvo       N. Neff <br />  <br /> vanadate-resistant mutants arise number         receptors, known involved se- <br /> molecular defects. Secretion underglycosylated in-         cretion andmembrane         fusion processes (WARREN <br /> vertase reported yeast genes        1993) potential target molecules orthovandate <br /> the mnn vrgand van mutants (ERDl, HARWICK et al.           considered possible alternative genes for <br /> 1990; KRE2, HILLet al. 1992; PMRl, RUDOLPH           et   mutationto      vanadate resistance inthisspecificity <br /> al. 1989). All genes knownor poten-       model. <br /> tial roles secretory pathway. Mutations           Either model toxicity consistent vanadate <br /> GTP-binding proteins secretory pathway produce         resistance arising alteration loss function <br /> underglycosylated invertase (FERRO-NOVICK          No-    certain integral membrane-associated proteins. <br /> VICK 1993). The otherphenotypes hygromycin sensi-          The phenotypes vanadate-resistant mutants are <br /> tivity, detergent sensitivity, sporulation defects low    consistent models. The major orthophos- <br /> viability prolonged liquid culture consistent      phate storage compartment yeast vacuole, <br /> with weakness plasma membrane cell wall           polymerizes polyphosphate (URECHet <br /> due defects membrane-protein complexes.                 al. 1978). Polyphosphate <span id='am-52' about='obo:NCBITaxon_263156' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_63226 obo:NCBITaxon_131567 obo:NCBITaxon_6820 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_72041 obo:NCBITaxon_525156 obo:NCBITaxon_6681 obo:NCBITaxon_6656 obo:NCBITaxon_1206794 obo:NCBITaxon_6657 obo:NCBITaxon_88770 obo:NCBITaxon_29979 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_263155 obo:NCBITaxon_6072'><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasRelatedSynonym" content="Chelator Hessler, 1970" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasAlternativeId" content="NCBITaxon:525459" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="Chelator" datatype="xsd:string"></span><span id='am-58'  ></span>chelator</span> cations <br />    We support model resistance elimi-      Mg2+or Ca2+,or basic amino acids, helping to <br /> nation toxic molecule formed intracellularly.    concentrate store molecules vacuole. <br /> The toxic molecule formed late endoplas-        The low pH environment vacuole vacuolar <br /> mic reticulum early Golgi apparatus ionic        network convert orthovanadate polyvanadate or <br /> metabolic environment. Alternatively Golgi appara-        vanadyl (WILLSKY   1990) providing competitor nor- <br /> tus contain sensitive target toxic        mal vacuolar phosphate metabolism. Cells be <br /> molecule. The observation vanadate-resistant        sensitive vanadate toxic molecule accu- <br /> mutants secrete underglycosylated invertase          mulates acidic compartments vacuo- <br /> cell wall plasma membrane defects, supports model        lar network.Cells vanadate resistant pre- <br /> that vanadate-resistant cells eliminate toxic mole-       venting concentrationof vanadate laterstages <br /> cule cell alterations secretory pathway    secretory pathway. The exact molecular role of <br /> at stage Golgi apparatus.                          MNN9+, VANl+ VAN2+ genes known <br />    There aretwo simple models fit current data       DNA sequence analysis predicted gene <br /> for toxic vanadate molecule eliminated        products does point known function. The <br /> the cell. The change permeability model. If     VAN2 gene identified N. DEANand col- <br /> the wild-type Golgiapparatus creates toxic molecule       leagues (J. B. POSTER   N. DEAN,unpublished data) <br /> or directtarget, dilution organelle space    gene, called GOG5, mutate vanadate <br /> or loss permeability control compart-      resistance. The cells containinga vanadate-resistant <br /> ment eliminate toxicity. By model          GOG5 mutation hygromycin sensitive, accumulate <br /> normal posttranslational modifications secreted pro-       secreted proteins endoplasmic reticulum and <br /> teins truncated pools soluble molecules     Golgi-related defects. The localization and <br /> are diluted aswell (ABEIJON et al. 1989). This          study function proteinsidentified study <br /> create general plasma membrane problems, struc-            help understand structure function of <br /> tures containing proteins proper modification         membrane-associated proteins Golgi apparatus. <br /> could inaccurately assembled membrane                We thank members yeast laboratoriesat Sloan-Ketter- <br /> integrity itself compromised. By model,         ing Institute comments especially MONA SPECTOR    for <br /> the genes mutate vanadate resistance            help dissection sporulation. We thank VIVIANMACKAY <br /> should encodetransporters membrane-associated              CLINTBALLOU     communicating unpublished results, LUN <br /> proteins regulating intraorganellesoluble molecule        BALLOU   IRENE GRIFTfor yeast strains.We thank BOBBY  BAUMfor <br />                                                               information ATRl open reading frame (ORF2) adja- <br /> content.                                                      cent VANI, noting identity VANl ANPl. <br />    The second model requires change vesicle           We thank PHILROBBINSfor advice information VANl <br /> membrane targeting (FRANZUSOFF         SCHEKMAN   1989;   ANPl identity also. <br /> PRYER   et al. 1992), toxic molecules shunted <br /> out secretory pathway level Golgi                               LITERATURE CITED <br /> apparatus. This model requires alteration Golgi pro- <br />                                                               ABEIJON,C., P. ORLEAN,  P. W. ROBBINS     C. B. HIRSCHBERG,   1989 <br /> teins normal pathway constricted, setting           Topology glycosylationin yeast: characterizationof GDP-man- <br /> up accumulation intermediates. These inter-              nose transport lumenal guanosine diphosp <br /> </body></html>