<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#63193; 2000 Genetics Society America <br />  <br />  <br />  <br />      Proteasome Mutants, pre4-2 ump1-2, Suppress Essential Function <br />               Not Mitochondrial RNase P Function the <br />                       <span id='am-17' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-18' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-33'  ></span>Saccharomyces cerevisiae</span> Gene RPM2 <br />  <br />                                   Mallory S. Lutz, Steven R. Ellis Nancy C. Martin <br />                     Department Biochemistry Molecular Biology, University Louisville, Louisville, Kentucky 40292 <br />                                                      Manuscript received August 25, 1999 <br />                                                   Accepted publication November 2, 1999 <br />  <br />  <br />                                                               ABSTRACT <br />                 The <span id='am-34' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-35' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-40'  ></span>Saccharomyces</span> cerevisiae nuclear gene RPM2 encodes component mitochondrial tRNA- <br />              processing enzyme RNase P. Cells grown fermentable carbon sources require mitochondrial <br />              tRNA processing activity, require RPM2, indicating additional function Rpm2 protein. <br />              RPM2-null cells arrest 25 generations fermentable media. Spontaneous mutations suppress <br />              arrest occur frequency &#11601;9 &#11003; 10&#11002;6. The resultant mutants grow nonfermentable carbon <br />              sources. We identified loci responsible suppression, encode proteins influence <br />              proteasome function assembly. PRE4 essential gene encoding &#65533;?&#65533;-7 subunit 20S proteasome <br />              core. A Val-to-Phe substitution highly conserved region Pre4p disrupts proteasome function <br />              suppresses growth arrest RPM2-null cells fermentable media. The locus, UMP1, encodes <br />              chaperone involved 20S proteasome assembly. A nonsense mutation UMP1 disrupts proteasome <br />              function suppresses &#9004;rpm2 growth arrest. In RPM2 wild-type background, pre4-2 ump1-2 strains <br />              fail grow restrictive temperatures nonfermentable carbon sources. These <span id='am-129' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-130' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-131'  ></span><span id='am-132' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-133' property="rdfs:label" content="Data" datatype="xsd:string"></span>data</span> link proteasome <br />              activity Rpm2p mitochondrial function. <br />  <br />  <br />  <br /> M      ITOCHONDRIA vital organelles eukaryo- <br />         tic cells. In addition role respiration <br /> and oxidative phosphorylation, mitochondria the <br />                                                                           organelle formation and/or maintenance disrupt <br />                                                                           growth carbon sources (De Pinto et al. 1999). <br />                                                                             Yeast mitochondrial RNase P ribonucleoprotein <br /> site diverse cellular functions heme biosyn-                  complex required removing 5&#11032; leader sequences <br /> thesis, metabolite transport, amino acid biosynthesis,                    mitochondrial tRNA precursors (Morales et al. <br /> and lipid catabolism (Attardi Schatz 1988; Pon                        1992; Dang Martin 1993). The RNA component <br /> and Schatz 1991). Mitochondrial biogenesis func-                      mitochondrial RNase P (Rpm1r) encoded the <br /> tion require input nuclear mitochondrial                    mitochondrial genome (Shu et al. 1991). The protein <br /> genomes. In yeast Saccharomyces cerevisiae, mitochon-                 component mitochondrial RNase P (Rpm2p) en- <br /> drial DNA codes proteins required oxidative                       coded nucleus. Biochemical genetic analyses <br /> phosphorylation, <span id='am-98' about='obo:NCBITaxon_1118549' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_8936 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_57387 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-99' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-100' property="rdfs:label" content="Electron" datatype="xsd:string"></span><span id='am-101'  ></span><span id='am-102' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>electron</span> transport, mitochondrial pro-                   established Rpm2p subunit mitochondrial RNase <br /> tein synthesis, mRNA processing. Ribosomal RNA,                       P (Morales et al. 1992; Dang Martin 1993). <br /> tRNA genes, RNase P RNA gene necessary                         Rpm2p major protein highly purified <br /> mitochondrial protein synthesis coded                     preparations mitochondrial RNase P (Morales et <br /> organelle genome. The maintenance wild-type mi-                      al. 1992). Moreover, antibodies raised Rpm2p <br /> tochondrial genome dependent mitochondrial                          immunoprecipitated RNase P activity Rpm1r <br /> protein synthesis (Myers et al. 1985).                                    mitochondrial extracts (Dang Martin 1993). <br />   In facultative aerobes like S. cerevisiae, mutations                 Finally, strains insertional disruption midway <br /> mitochondrial DNA tolerated cells grown                     RPM2 reading frame accumulated mito- <br /> fermentable carbon sources. Consequently, nuclear                         chondrial tRNA precursors 5&#11032; extensions, Rpm1r <br /> genes encoding respiratory components factors re-                     precursors, unable grow respiratory <span id='am-109' about='obo:NCBITaxon_201850' typeof='owl:Thing obo:NCBITaxon_1125798 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_71529 obo:NCBITaxon_41084 obo:NCBITaxon_33154 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6960 obo:NCBITaxon_6072 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_6656 obo:NCBITaxon_41088 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_7041'><span id='am-110' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-111' property="rdfs:label" content="Car" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-113'  ></span>car</span>- <br /> quired mitochondrial gene expression affect respira-                  bon sources (Morales et al. 1992; Stribinskis et al. <br /> tory growth growth fermentable carbon                          1996). These data demonstrate RPM1 RPM2 <br /> sources. In contrast, nuclear mutations interfere                    encode subunits RNase P activity <span id='am-53' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56'  ></span><span id='am-57' property="rdfs:label" content="This" datatype="xsd:string"></span>this</span> <br />                                                                           ribonucleoprotein complex required respiratory <br />                                                                           fermentative growth. <br />                                                                             Unexpectedly, cells null allele RPM2 could <br />   Corresponding author: Nancy Martin, Department Biochemistry          grow respiratory fermentable carbon <br /> and Molecular Biology, University Louisville, 319 Abraham Flexner <br /> Way, Bldg. A, Rm. 708, Louisville, KY 40292.                              sources, indicating RPM2 function addition <br /> E-mail: ncmart01@gwise.louisville.edu                                     role subunit mitochondrial RNase P (Kas- <br /> Genetics 154: 1013&#8211;1023 (March 2000) <br />  1014                                         M. S. Lutz, S. R. Ellis N. C. Martin <br />  <br /> senbrock et al. 1995). This function provided                tions Inc.) added autoclaved plate media concen- <br /> by N-terminal half Rpm2p, explaining strains             trations reported results. <br />                                                                        Construction null allele RPM2: YMW1 YMW2 are <br /> with insertional disruption grew fermentable                 haploid derivatives W303 cells generous gift <br /> carbon sources. The nature second function               Dr. Michael Walberg (Zieler et al. 1995). Deletion of <br /> unclear, related isolation RPM2           RPM2 performed YMW1/YMW2 diploid, YML9. The <br /> as high-copy suppressors temperature-               complete RPM2 open reading frame (ORF) replaced with <br /> sensitive (ts) allele TOM40/ISP42 (Kassenbrock et                heterologous dominant G418 resistance gene kanMX using <br />                                                                     SFH-PCR-based gene disruption technique (Wach et al. <br /> al. 1995). TOM40 encodes essential membrane com-                 1994; Wach 1996). RPM2/kanMX primers 5&#11032; CAA <br /> ponent mitochondrial import machinery (Vest-                 CAAACAGAAAAGAAAAAACAAGCATACACAAACGAAAAT <br /> weber et al. 1989; Moczko et al. 1992; Rapaport et al.              GCGTACGCTGCAGGTCGAC 3&#11032; 5&#11032; GTTATAACAGTG <br /> 1997; Hill et al. 1998). The N-terminal portion RPM2             AAATAAATAAAATATCAAATTGTAAAGGTCAATCGAT <br /> that sufficient growth fermentable carbon                 GAATTCGAGCTCG 3&#11032;. Underlined sequences oligonucle- <br />                                                                     otides mark parts sequence complemen- <br /> sources suppresses TOM40 mitochondrial im-                 tary pFA6-MCS <span id='am-75' about='obo:NCBITaxon_45419' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384 obo:NCBITaxon_36549'><span id='am-76' property="oboInOwl:hasRelatedSynonym" content="pFA6 plasmid" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="Plasmid pFA6" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-80'  ></span>plasmid pFA6</span>-kanMX6 (Wach 1996). <br /> port defect (Kassenbrock et al. 1995). Rpm2p                    Southern blot analysis PCR confirmed proper inser- <br /> Tom40p appear interact assayed                  tion kanMX RPM2 locus, specifically starting one <br /> the two-<span id='am-134' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-135'  ></span><span id='am-136' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-137' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>hybrid</span> system, suggesting RPM2 sup-                nucleotide (nt) 3&#11032; ATG RPM2 ORF through <br /> press mutant allele physically interacting              penultimate nt 5&#11032; STOP <span id='am-58' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-59' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-60'  ></span><span id='am-61' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>codon</span>. <br />                                                                        Suppressors RPM2-null mutation: Three different <br /> with Tom40p (Groom 1995). Thus, requirement                     diploid RPM2/&#9004;rpm2::kanMX strains, YML34, YML37, and <br /> for RPM2 growth remains unclear.                                YML41, sporulated haploid spores separated <br />    RPM2-null mutants display significant rate pheno-           tetrad analysis YPD plates. The sporulation pattern ob- <br /> typic reversion glucose-containing media. This rate              served cases 2:2 ratio big/small colonies on <br /> is indicative spontaneous second-site loss-of-function           YPD 30&#11034; 3 days. The small colonies increase <br />                                                                     size additional week 30&#11034;. A total 14 small <br /> suppressor mutations genes functionally linked                &#9004;rpm2 colonies individually suspended 200, 300, or <br /> the essential growth function RPM2. We labeled             400 &#65533;?&#65533;l YPD. A small aliquot cells counted using a <br /> second-site suppressors, suppressors arrest (<span id='am-114' about='obo:NCBITaxon_342054' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_159970 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_30259 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_6656 obo:NCBITaxon_159969 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33317 obo:NCBITaxon_33342 obo:NCBITaxon_2759 obo:NCBITaxon_50557 obo:NCBITaxon_6072 obo:NCBITaxon_6960'><span id='am-115' property="rdfs:label" content="Soa" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-118'  ></span>soa</span>) mu-            hemocytometer. The remaining cells plated YPD at <br /> tants.                                                              30&#11034;. Suppression &#9004;rpm2-arrested growth results pheno- <br />    Here, independent soa suppressor                typic revertant colonies, likely occurring sponta- <br />                                                                     neous second-site suppressors called suppression arrest <br /> strains fall different complementation              alleles (soa alleles). A maximum different colony sizes <br /> groups, suppress lack growth &#9004;rpm2               seen revertant populations. Individual colonies <br /> strains glucose media. We demonstrate                picked replated score homogeneous growth. <br /> these mutant alleles affect 26S proteasome function.                Only &#9004;rpm2 soa colonies grew homogeneously were <br /> PRE4 identified wild-type allele soa1             studied further. To determine soa mutations reces- <br />                                                                     sive dominant &#9004;rpm2, &#9004;rpm2 soa cells mated <br /> encodes &#65533;?&#65533;-7 subunit 20S proteasome core <br />                                                                     &#65533;?&#65533;&#11001; &#9004;rpm2 SOA strains containing wild-type copy RPM2 <br /> (Groll et al. 1997). UMP1 wild-type allele soa2           YEp352 covering plasmid. A 5-<span id='am-1' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Foa" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>FOA</span> shuffle performed <br /> and encodes chaperone necessary assembly               lose YEp352 RPM2 plasmid (Sikorski Boeke 1991). <br /> of 20S proteasome core complex (Ramos et al.                       Identification wild-type SOA alleles: Seventeen indepen- <br /> 1998). Thus, defects proteasome activity compensate              dent &#9004;rpm2 soa phenotypic revertants backcrossed with <br /> for lack RPM2.                                               isogenic wild-type strains, YMW1 YMW2, isolate soa <br />                                                                     mutations wild-type background deter- <br />                                                                     suppression originated single-site mutations. Dip- <br />               MATERIALS AND METHODS                                 loids sporulated haploid progeny separated by <br />                                                                     tetrad analysis. The growth haploid progeny evaluated <br />    Strains, <span id='am-139' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-140' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-141' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>plasmids</span>, media: Standard yeast manipulations       rich minimal glucose glycerol media 16&#11034; and <br /> were used (Sherman 1991; Sherman Hicks 1991; Kaiser             37&#11034;. Tight conditional growth phenotypes observed in <br /> et al. 1994). Yeasts transformed plasmid DNA using        independent backcrosses, permitting pursue these <br /> a lithium acetate method (Chen et al. 1992). The yeast strains      soa strains further. Tetrad analysis patterns parental, nonpa- <br /> used study listed Table 1. We used centro-       rental, tetratype indicated mutations originated <br /> meric pRS300 series plasmids subcloning (Sikorski        single, nuclear loci. The ability RPM2 SOA/RPM2 soa <br /> Hieter 1989).                                                       diploids grow restrictive conditions demonstrated <br />    Rich glucose media, YPD, included 1% Bacto-yeast extract,        soa1, soa2, soa3, soa4, soa5 recessive. <br /> 2% Bacto-peptone, 2% glucose. Rich glycerol-ethanol me-            RPM2 soa1 yeast (37B1.14OCD3) transformed a <br /> dia, YPGE, contained 1% Bacto-yeast extract, 2% Bacto-pep-          pRS200 centromeric genomic library, YPH1, (Sikorski and <br /> tone, 3% (v/v) glycerol, 2% (v/v) ethanol. Synthetic com-       Hieter 1989; American Type Culture Collection (ATCC) no. <br /> plete (SC), synthetic complete lacking specific amino acids         77165) screened suppression cold-sensitive (cs) <br /> (SC trp&#11002;), synthetic dextrose minimal media (SD) contain        phenotype 16&#11034; YPGE. A plasmid containing 8.8-kb <br /> 0.67% Bacto-yeast nitrogen base amino acids, 2% glu-        insert suppressed conditional growth phenotype. Subclon- <br /> cose, appropriate supplements (Kaiser et al. 1994). Solid       ing identified 2.2-kb fragment containing single ORF, <br /> media plates included 2% Bacto-agar. Culture media re-          PRE4, supported growth RPM2 soa1 cells 16&#11034; on <br /> agents Fisher Scientific (Pittsburgh) Difco (Detroit)       YPGE. Linkage mutant phenotype PRE4 locus <br /> brand. G418, canavanine, CdCl2 aqueous solutions           confirmed integrating PRE4 URA3 linearized plasmid <br /> sterilized passage 0.2-&#65533;?&#65533;m filter (Micron Separa-       PRE4 locus RPM2 soa1 cells (Rothstein 1991). These <br />                                                   Proteasome Mutants Suppress &#9004;rpm2                                               1015 <br />  <br />                                                                 TABLE 1 <br />                                                     Yeast strains used study <br />  <br /> Strain                                               Genotype                                             Source reference <br /> YMW1                    MAT&#65533;?&#65533; ade2-1 ade3&#9004;22 his3-11,15 leu2-3,11 trp1-1 ura3-1 can1-100             M. Walberg; Zieler et al. (1995) <br /> YMW2                    MATa ade2-1 ade3&#9004;22 his3-11,15 leu2-3,11 trp1-1 ura3-1 can1-100             M. Walberg; Zieler et al. (1995) <br /> YMW9                    MATa/MAT&#65533;?&#65533; ade2-1 ade3&#9004;22 his3-11,15 leu2-3,11 trp1-1 ura3-1                 This study <br />                            can1-100 <br /> YML34                   MATa/MAT&#65533;?&#65533; ade2-1 ade3&#9004;22 his3-11,15 leu2-3,11                               This study <br /> YML37                   trp1-1 ura3-1 can1-100 RPM2/&#9004;rpm2::kanMX <br /> YML41 <br /> 37B1.14OCD3             MAT&#65533;?&#65533; ade2-1 ade3&#9004;22 his3-11,15 leu2-3,11 trp1-1 ura3-1 can1-100             This study <br />                          pre4-2 <br /> 34D1.51OCC1             MAT&#65533;?&#65533; ade2-1 ade3&#9004;22 his3-11,15 leu2-3,11 trp1-1 ura3-1 can1-100             This study <br />                          ump1-2 <br /> 41B3.2OCC1              MAT&#65533;?&#65533; ade2-1 ade3&#9004;22 his3-11,15 leu2-3,11 trp1-1 ura3-1 can1-100             This study <br />                          soa3 <br />  <br />  <br />  <br /> cells mated isogenic YMW2, haploid progeny            inhibitors diluted 1:10 (Complete mini cocktail tablets; <br /> scored uracil auxotrophy growth restrictive          Boehringer Mannheim, Indianapolis). An equal volume of <br /> conditions.                                                            sterile glass beads added, cells disrupted 3 <br />    RPM2 soa2 yeast (34D1.51OCC1) similarly trans-                 min mini-bead beater (Biospec Products) followed 15 <br /> formed YCp50 centromeric genomic library, CEN BANK              min &#11002;70&#11034; 3 min bead beater. Samples <br /> A (Rose et al. 1987; ATCC no. 37415), screened 37&#11034;              boiled cleared particulate matter centrifu- <br /> on YPGE. Plasmid-based suppression seen 11-                gation 12,000 &#11003; g microfuge. The concentration of <br /> kb insert. Subcloning identified UMP1 wild-type allele          proteins supernatant determined (Bio-Rad protein <br /> suppressing soa2 ts phenotype. Linkage mutant               assays). A total 1 &#65533;?&#65533;g protein loaded 10% <br /> phenotype UMP1 locus confirmed integrating             SDS/polyacrylamide gel, run, blotted Immobilon-P <br /> a UMP1 URA3 linearized plasmid UMP1 locus RPM2               (Millipore, Bedford, MA) membranes. Probes included rab- <br /> soa2 cells (Rothstein 1991). These cells mated <span id='am-119' about='obo:NCBITaxon_238707' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_1489908 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_1489913 obo:NCBITaxon_8075 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_1489917'><span id='am-120' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-121' property="rdfs:label" content="Iso" datatype="xsd:string"></span><span id='am-122'  ></span><span id='am-123' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>iso</span>-            bit polyclonal antiubiquitin antibody (Haas Bright <br /> genic YMW1, haploid progeny scored uracil                 1985) generously provided Dr. A. Haas, mouse mono- <br /> auxotrophy growth restrictive conditions.                clonal antiactin antibody (Boehringer Mannheim). Enhanced <br />    Nucleic acid techniques DNA sequencing: Standard                chemiluminescence (Amersham Life Sciences, Arlington <br /> DNA manipulations methodologies used (Sam-                    Heights, IL) used signal detection. <br /> brook et al. 1989). Restriction enzymes T4 DNA ligase                 Microscopy: Nuclear mitochondrial DNA stained <br /> were New England Biolabs (Beverly, MA) MBI Fer-                vital dye DAPI (4&#11032;,6-diamidino-2-phenylindole) at <br /> mentas. PCR DNA polymerase enzymes used include TAQ                    independent experiments strain (Kaiser et <br /> (Fisher), Klentaq-LA polymerase mix (Sigma, St. Louis),            al. 1994). Approximately 107 cells pelleted microfuge <br /> Vent (New England Biolabs). The primers PCR recovery               5-sec pulse resuspended 70% ethanol ice. <br /> of sequence PRE4 locus upstream primer 5&#11032; CATT             Cells fixed 10&#8211;30 min ice washed twice with <br /> CCGTGTAGTGCCAATA 3&#11032; downstream primer 5&#11032;                           H2O. Cells resuspended 1 &#65533;?&#65533;g/ml DAPI and <br /> GGTTACCCGTCAATCCTTTT 3&#11032;. The equivalent PCR prim-                      mounted using aqueous Light antifade Prolong anti- <br /> ers UMP1 locus upstream primer 5&#11032; CGGCAC                  fade (Molecular Probes, Eugene, OR). Observations were <br /> TAATCTAATCTTTCAA 3&#11032; downstream primer 5&#11032;                           Nikon Optiphot light microscope &#11003;60 Plan- <br /> TGGCAATTTTATCCTACCTGAG 3&#11032;.                                             Apo 1.4 NA objective. <br />    DNA sequencing performed University Wiscon- <br /> sin Biotechnology Center, DNA Synthesis Sequencing <br /> Facility (Dr. Charles Nicolet). PCR products sequenced <br /> directly cloning PCR-TA cloning vector (In-                                     RESULTS <br /> vitrogen). Both strands PCR product completely <br /> sequenced assure accuracy. Plasmid DNA isolated using              Identification isolation spontaneous second- <br /> either Bio-Rad (Richmond, CA) Qiagen miniprep kits.             site suppressors &#9004;rpm2: Sporulation RPM2/ <br /> BLAST searches compared aligned nucleotide sequences               &#9004;rpm2::kanMX diploids subsequent growth hap- <br /> (Altschul et al. 1990), SIM alignment portion             loid progeny rich glucose yielded 2:2 ratio small/ <br /> the ExPASy home page used amino acid comparisons. <br /> Amino acid numbering mature Pre4p based infor- <br />                                                                        large colonies. More 90% cells the <br /> mation Protein Data Bank (Brookhaven, NY)               small colonies &#9004;rpm2 allele lacked buds, indicat- <br /> accession no. 1RYP (Groll et al. 1997).                                ing ceased growth. <br />    Protein analysis: Protein isolated log-phase cul-             As step understanding phenotype, <br /> tures. Cells grown 30&#11034; OD640 0.5             determined number cells small colo- <br /> transferred 16&#11034; 3.5 hr. Cells collected centrifuga- <br /> tion stored &#11002;70&#11034;. Frozen pellets resuspended            nies. An average &#11601;4.4 &#11003; 107 cells small colony was <br /> 200 &#65533;?&#65533;l NET-NP (150 mm NaCl, 5 mm EDTA, 50 mm Tris-                     obtained direct counting cells 14 independent <br /> HCl, pH 7.5, 0.5% NP-40) plus protease inhibitors. Protease            colonies. This number consistent &#11601;25 rounds of <br />  1016                                        M. S. Lutz, S. R. Ellis N. C. Martin <br />  <br />                                                             TABLE 2 <br />                                   Growth phenotypes RPM2 soa complementation groups <br />  <br />          Strain genotype                    YPD 16&#11034;             YPD 37&#11034;             YPGE 16&#11034;              YPGE 37&#11034; <br />          37B1.14OCD3 (RPM2 soa1)                    &#11001;                  &#11001;&#11001;                   &#11002;                    &#11001; <br />          34D1.51OCC1 (RPM2 soa2)                   &#11001;&#11001;                  &#11001;&#11001;                   &#11001;                    &#11002; <br />          41B3.2OCC1 (RPM2 soa3)                    &#11001;&#11001;                  &#11001;&#11001;                   &#11001;                    &#11002; <br />            Cultures grown late log phase YPGE 30&#11034; diluted OD640 &#11005; 0.1. Aliquots (4-&#65533;?&#65533;l) 10- <br />          fold serial dilutions spotted appropriate plates. Growth colonies monitored 1 wk. Values <br />          comparisons wild-type YMW2 growth (RPM2 SOA), &#11001;&#11001;&#11001;&#11001;. <br />  <br />  <br /> cell division initial spore. The mean spontane-           soa complementation groups (Table 2). Analysis <br /> ous phenotypic reversion rate 14 independent                  &#9004;rpm2 soa/&#9004;rpm2 SOA diploid strains showed that <br /> &#9004;rpm2::kanMX colonies, plated rich glucose 30&#11034;,              mutations complementation groups soa1 soa3 <br /> was 1.5 &#11003; 10&#11002;7 1.4 &#11003; 10&#11002;6 revertants/                  recessive unable suppress &#9004;rpm2 pheno- <br /> original spore. Phenotypic revertants designated              type presence wild-type alleles. The soa2 <br /> suppressor arrest (soa). At 30&#11034; rich glucose media           allele, hand, semidominant par- <br /> revertant colonies grew different rates               tially able suppress &#9004;rpm2 presence wild- <br /> other isogenic wild-type strains. Seventeen               type SOA2. <br /> &#9004;rpm2 soa double mutants demonstrated growth                     The SOA1 locus PRE4, encodes &#65533;?&#65533; subunit <br /> defects YPD, 16&#11034; 37&#11034;, selected            20S proteasome: RPM2 soa1 cells trans- <br /> further study.                                                     formed low-copy centromeric genomic library <br />    Of 17 soa mutants, 5 demonstrated tight temperature-            (YPH1) screened complementation Cs&#11002; <br /> sensitive (Ts&#11002;) cold-sensitive (Cs&#11002;) phenotypes              respiratory growth phenotype (materials meth- <br /> rich YPGE media linked soa phenotype                   ods). A single plasmid containing 8.8-kb insert com- <br /> separate &#9004;rpm2 mutation. Subsequent crosses               plemented restrictive growth 16&#11034; YPGE. Subclon- <br /> demonstrated soa mutations reces-            ing ORFs insert revealed PRE4 was <br /> sive (materials methods). The conditional phe-                 responsible complementation. PRE4 encodes <br /> notypes used screening yeast genomic libraries             &#65533;?&#65533;-7 subunit 20S proteasome core complex <br /> for wild-type alleles soa loci.                             essential cell viability (Hilt et al. 1993; Groll <br />    Pairwise crosses of soa haploids        et al. 1997). Figure 1A shows PRE4 suppression Cs&#11002; <br /> made, diploids plated restrictive growth            Ts&#11002; respiratory growth phenotypes RPM2 soa1 <br /> conditions. These crosses indicated                cells. To determine soa1 allele PRE4, plasmid <br />  <br />  <br />                                                                                                   Figure 1.&#8212;Permissive and <br />                                                                                                   restrictive growth condi- <br />                                                                                                   tions RPM2 pre4-2 and <br />                                                                                                   RPM2 ump1-2 cells YPGE <br />                                                                                                   media. Genomic library <br />                                                                                                   screens led identifi- <br />                                                                                                   cation PRE4 wild- <br />                                                                                                   type allele soa1 sup- <br />                                                                                                   pression RPM2 soa1 cs <br />                                                                                                   phenotype UMP1 was <br />                                                                                                   identified wild-type <br />                                                                                                   allele soa2 suppression <br />                                                                                                   RPM2 soa2 ts pheno- <br />                                                                                                   type. Logarithmically grow- <br />                                                                                                   ing cultures diluted to <br />                                                                                                   OD640 &#11005; 0.1 4 &#65533;?&#65533;l 10- <br />                                                                                                   fold serial dilutions were <br />                                                                                                   spotted YPGE plates. <br />                                                                                                   Plates incubated at <br />                                                                                                   given temperatures fol- <br />                                                                                                   lows: 16&#11034;, 5 days; 30&#11034;, 2 days; <br />                                                                                                   37&#11034;, 3 days. (A) Wild-type <br /> cells original parent strain YMW2; soa1 represents strain 37B1.14OCD3; pRS314 PRE4 CEN plasmid PRE4 <br /> locus. (B) Wild-type cells original parent strain YMW1; soa2 represents strain 34D1.51OCC1; pRS316 CEN plasmid <br /> with UMP1 locus. <br />                                               Proteasome Mutants Suppress &#9004;rpm2                                          1017 <br />  <br />  <br />  <br />  <br />                                                                                        Figure 2.&#8212;Hypersensitivity pre4-2 <br />                                                                                     ump1-2 arginine analogue ca- <br />                                                                                     navanine oxidative damage <br />                                                                                     caused CdCl2. pre4-2, ump1-2, soa3 <br />                                                                                     cultures actively growing YPD were <br />                                                                                     spotted 1:10 serial dilutions mini- <br />                                                                                     mal media plates. (A) Canavanine (30 <br />                                                                                     &#65533;?&#65533;g/ml); (B) CdCl2 (30 &#65533;?&#65533;m). Incubation <br />                                                                                     permissive temperatures 3 <br />                                                                                     days. The higher concentrations cana- <br />                                                                                     vanine used study the <br />                                                                                     can1-100 mutant allele strains <br />                                                                                     derived W303 background. The <br />                                                                                     can1-100 allele confers resistance low <br />                                                                                     levels canavanine. <br />  <br />  <br />  <br />  <br /> containing wild-type PRE4 URA3 integrated            crease mutant proteins normally degraded <br /> the PRE4 locus soa1 cells. These cells crossed          proteasome. Relative wild-type cells, pre4-2 <br /> to wild type, sporulated, tetrads dissected. The       ump1-2 cells showed hypersensitivity canavan- <br /> haploid progeny exhibited 4:0 ratio wild-type/mu-          ine, phenotypic characteristic cells defective <br /> tant segregation pattern YPGE growth 16&#11034;, demon-          proteasome activity (Figure 2; Chen Hochstrasser <br /> strating soa1 mutant allele PRE4. The soa1         1996; Ramos et al. 1998). RPM2 pre4-2 cells de- <br /> allele second allele PRE4 identified          creased colony size relative wild-type strains 5 &#65533;?&#65533;g/ <br /> termed pre4-2 (Hilt et al. 1993). The previously identi-        ml canavanine growth 30 35 <br /> fied pre4-1 allele produces truncated protein de-        &#65533;?&#65533;g/ml canavanine (Figure 2A). Similar results seen <br /> creased peptidylglutamyl peptidase hydrolyzing (PGPH)           RPM2 ump1-2 mutants grow on <br /> activity does affect cell growth (Hilt et al. 1993).   media containing 40 &#65533;?&#65533;g/ml canavanine. In contrast, <br />    The SOA2 locus UMP1, encodes 20S <span id='am-41' about='obo:NCBITaxon_4331' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4328 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_71241 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_232378 obo:NCBITaxon_3193'><span id='am-42' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-43'  ></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Protea" datatype="xsd:string"></span>protea</span>-          RPM2 soa3 cells grow near-wild-type levels when <br /> some maturation factor: Two overlapping plasmids com-           spotted SD-canavanine plates. The combined <br /> plementing soa2 defect isolated (materials             effect null mutation RPM2 soa <br /> and methods). Subcloning possible ORFs re-              mutations led poor growth SD media and <br /> vealed UMP1 complementing gene. Figure             growth presence low concentrations of <br /> 1B shows UMP1 suppression Ts&#11002; respiratory                canavanine (5&#8211;10 &#65533;?&#65533;g/ml; data shown). This implies <br /> growth phenotype RPM2 soa2 cells. UMP1 sup-             cells lacking RPM2 increased sensitivity to <br /> presses Ts&#11002; respiratory phenotype mem-         stressful conditions. <br /> ber soa2 complementation group. UMP1, like                  A second test proteasome function sensitivity <br /> PRE4, necessary function 20S core pro-        cells oxidative damage caused CdCl2. RPM2 <br /> teasomal complex. UMP1 encodes chaperone involved             pre4-2 RPM2 ump1-2 cells grow poorly presence <br /> in 20S proteasome assembly (Ramos et al. 1998). To              30 &#65533;?&#65533;m CdCl2 (Figure 2B). The double &#9004;rpm2 soa <br /> determine soa2 mutations alleles UMP1,             mutants failed grow conditions (data <br /> plasmid containing wild-type UMP1 integrated             shown). Similar results obtained canavanine, <br /> the UMP1 locus soa2 cells. These cells crossed          RPM2 soa3 cells sensitive CdCl2. Both the <br /> to wild type, sporulated, tetrads dissected. The       canavanine CdCl2 sensitivities overcome if <br /> haploid progeny revealed 4:0 ratio wild-type/mu-           RPM2 pre4-2 RPM2 ump1-2 cells transformed <br /> tant segregation pattern growth YPGE growth            PRE4 UMP1, respectively, low-copy <span id='am-81' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-82' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-84' property="rdfs:label" content="vectors" datatype="xsd:string"></span>vectors</span> <br /> 37&#11034;, demonstrating soa2 locus UMP1. We              (Figure 2, A B). The soa3 complementation group <br /> named soa2 allele ump1-2.                                   does evidence defective 26S proteasome <br />    Sensitivity soa mutants canavanine CdCl2:          activity. Our investigations relationship soa3 <br /> To determine proteasome function compromised             suppressor &#9004;rpm2 yeast ongoing. <br /> in pre4-2, ump1-2, soa3 strains, cultures incu-           Degradation ubiquitinated proteins inhibited in <br /> bated variety stressful conditions in-          pre4-2 yeast: As test impaired proteasome <br /> crease cellular dependence proteasomal degrada-              activity pre4-2 cells, examined level ubiquiti- <br /> tion. Exposure cells increasing concentrations         nated proteins permissive restrictive growth con- <br /> the arginine analogue canavanine causes general in-           ditions (Haas 1988; chen Hochstrasser 1995; <br />  1018                                         M. S. Lutz, S. R. Ellis N. C. Martin <br />  <br />  <br />  <br />  <br />                                                                        Figure 4.&#8212;Morphology &#9004;rpm2 soa mutants per- <br />                                                                     missive growth conditions. Cultures grown YPD 30&#11034; <br />                                                                     OD640 &#11005; 0.5. (A) Wild-type YMW2; (B) &#9004;rpm2 pre4-2; <br />                                                                     (C) &#9004;rpm2 ump1-2; (D) &#9004;rpm2 soa3. Fluorescence images reveal <br />                                                                     nuclear mitochondrial DNA stained DAPI. In &#9004;rpm2 <br />                                                                     pre4-2 &#9004;rpm2 ump1-2 cultures interconnected cells <br />                                                                     appear contain nuclear DNA (B C). &#9004;rpm2 pre4-2 cells <br />                                                                     &#65533;?&#65533;&#11002; (B) &#9004;rpm2 ump1-2 &#9004;rpm2 soa3 cells &#65533;?&#65533;o <br />                                                                     (C D). Bar, 5 &#65533;?&#65533;m. <br />    Figure 3.&#8212;Accumulation ubiquitinated proteins the <br /> pre4-2 strain 37B1.14OCD3. A Western blot total yeast ex- <br /> tracts probed polyclonal antibody ubiquitin. <br /> Yeast cell extract (1 &#65533;?&#65533;g) mid-log-phase cultures              Sequence ump1-2 ump1-3 reveals contain <br /> loaded 10% polyacrylamide gel. (Lanes 1 <br /> and 3) YMW2 wild-type cells; (lanes 2 4) 37B1.14OCD3 <br />                                                                     nonsense mutations: Cells lacking UMP1 viable but <br /> (pre4-2) cells. Lanes 1 2 grown 30&#11034; cultures      grow slowly (Ramos et al. 1998). Ump1p postulated to <br /> reached OD640 &#11005; 0.5. Lanes 3 4 shifted           interact proPre2p block protease maturation <br /> restrictive temperature 16&#11034;. The shift 16&#11034; 3 hr       proteasome assembly complete. Both ump1-2 <br /> from identical 30&#11034; OD640 &#11005; 0.5 cultures. High-molecular-mass        ump1-3 contained nonsense mutations resulting in <br /> ubiquitin-protein conjugates indicated left. Molecu- <br /> lar mass standards kilodaltons indicated right.       premature stop codons codons 10 36, respectively. <br />                                                                        Morphology &#9004;rpm2 soa RPM2 soa mutants: <br />                                                                     Cells lacking RPM2 fail produce mature mitochon- <br /> Rinaldi et al. 1998). Wild-type mutant cells               drial tRNA, leading absence mitochondrial pro- <br /> grown 30&#11034; YPGE media. Aliquots log-phase                 tein synthesis petite S. cerevisiae strains. The state <br /> cultures transferred 16&#11034; 3 hr main-           mitochondrial genome analyzed using com- <br /> tained 30&#11034;. Total protein isolated, fractionated             bination DAPI staining genetic crosses. DAPI <br /> by SDS-PAGE, transferred polyvinyldifluoride mem-                staining revealed mitochondrial DNA &#9004;rpm2 pre4-2 <br /> brane, probed antiubiquitin antibody. The blot             cells, &#9004;rpm2 ump1-2 &#9004;rpm2 soa3 showed no <br /> showed increased signal RPM2 pre4-2 extracts vs.              extranuclear fluorescent staining (Figure 4). Mitochon- <br /> wild-type cells 16&#11034; (Figure 3; Haas Bright 1985).            drial DNA distributed elongated, <br /> The diffuse signal molecular weight range 35&#8211;250             interconnected &#9004;rpm2 pre4-2 cells nuclear <br /> kD, reflects general increase ubiquitinated              DNA did appear segregating properly. To <br /> proteins, typical cells impaired proteasome             determine &#9004;rpm2 pre4-2 maintained wild-type mito- <br /> activity.                                                           chondrial genome, cells mated YMW2 &#65533;?&#65533;o, <br />    The pre4-2 allele contains missense mutation: The              isogenic strain containing wild-type nuclear ge- <br /> pre4-2 mutation guanine-to-thymidine base change               nome lacking mitochondrial DNA. The resultant <br /> at nucleotide 157 PRE4 ORF, results               diploids unable grow YPGE, demonstrating <br /> valine-to-phenylalanine substitution amino acid (<span id='am-103' about='obo:NCBITaxon_152839' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4447 obo:NCBITaxon_4747 obo:NCBITaxon_158359 obo:NCBITaxon_158331 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_158350 obo:NCBITaxon_73496 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_3193'><span id='am-104' property="oboInOwl:hasRelatedSynonym" content="Aa Rchb.f." datatype="xsd:string"></span><span id='am-105' property="rdfs:label" content="Aa" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-107'  ></span><span id='am-108' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>aa</span>)             &#9004;rpm2 pre4-2 cells &#65533;?&#65533;&#11002;. <br /> 12 mature protein. Sequence comparison pre4-                 Proteasome degradation cyclins involved cell <br /> 2p closely related 20S proteasome               cycle regulation documented (Chun et <br /> &#65533;?&#65533;-7 subunits archtype &#65533;?&#65533; subunit Ther-               al. 1996; King et al. 1996; Hershko 1997). The &#9004;rpm2 <br /> moplasma <span id='am-68' about='obo:NCBITaxon_522' typeof='owl:Thing obo:NCBITaxon_433 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_28211 obo:NCBITaxon_204441'><span id='am-69' property="oboInOwl:hasExactSynonym" content="Acidiphilum" datatype="xsd:string"></span><span id='am-70'  ></span><span id='am-71' property="oboInOwl:hasRelatedSynonym" content="Acidiphilium Harrison 1981 emend. Kishimoto et al. 1996" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="Acidiphilium" datatype="xsd:string"></span>acidiphilium</span> showed amino acid substi-           pre4-2 &#9004;rpm2 ump1-2 mutants displayed range of <br /> tution resides highly conserved region. This               morphologies indicative cells defective cell cycle <br /> change introduces bulky Phe start &#65533;?&#65533;-strand              progression (Figure 4). Approximately 5&#8211;10% &#9004;rpm2 <br /> 2 &#65533;?&#65533;-sandwich tertiary structure described             pre4-2 cells grown 30&#11034; YPD multiply budded <br /> Groll et al. (1997).                                                showed interconnected, elongated morphologies <br />                                                Proteasome Mutants Suppress &#9004;rpm2                                        1019 <br />  <br />                                                                  soa mutants respiratory competent &#65533;?&#65533;&#11001; mito- <br />                                                                  chondrial genomes. We used DAPI staining observe <br />                                                                  nuclear mitochondrial DNA patterns these <br />                                                                  soa mutants. During log-phase growth 30&#11034;, pre4-2 and <br />                                                                  ump1-2 cultures noticeably different wild-type <br />                                                                  cells contain multiply budded elongated <br />                                                                  cells (Figure 5, A, C, E). However, mutant <br />                                                                  phenotype displayed degree was <br />                                                                  &#9004;rpm2 pre4-2 &#9004;rpm2 ump1-2 cultures. As Figure <br />                                                                  4, cell bodies contain nuclear DNA, mito- <br />                                                                  chondrial DNA interconnected <br />                                                                  cells. Under restrictive conditions (3&#8211;16 hr 16&#11034; or <br />                                                                  37&#11034;, respectively) RPM2 pre4-2 RPM2 ump1-2 <br />                                                                  cultures contained elongated, multiply budded cells <br />                                                                  similar &#9004;rpm2 counterparts. Consistent the <br />                                                                  results double mutant, RPM2 soa3 retains a <br />                                                                  wild-type morphology permissive restrictive <br />                                                                  conditions, supporting conclusion soa3 not <br />                                                                  proteasome-functional mutant (Figure 5, G H). <br />  <br />  <br />                                                                                        DISCUSSION <br />                                                                     The results presented demonstrate growth <br />                                                                  fermentable carbon sources restored null <br />                                                                  mutants RPM2 mutations different genes <br />                                                                  disrupt 20S proteasome function. One encodes the <br />                                                                  proteasome &#65533;?&#65533;-7 subunit; encodes chaperone <br />                                                                  necessary assembly active 20S proteasome com- <br />                                                                  plex (Ramos et al. 1998). <br />                                                                     Proteasome function essential eukaryotic cells <br />                                                                  regulated degradation targeted proteins. The <br />                                                                  ATP- ubiquitin-dependent proteasome pathway <br />                                                                  plays key role degradation misfolded/mutant <br />                                                                  proteins, controlling cell growth divi- <br />                                                                  sion degradation regulatory proteins <br />   Figure 5.&#8212;DAPI staining RPM2 soa mutants per- <br /> missive restrictive growth conditions. Cultures grown   (Chun et al. 1996; Hilt Wolf 1996; King et al. 1996; <br /> in YPD 30&#11034; mid-log phase observation permissive     Hershko 1997). The 26S proteasome proteolytic <br /> conditions (A, C, E, G). Identically grown cultures     component ubiquitin-mediated protein degrada- <br /> shifted 16&#11034; (B H) 37&#11034; (D, F, I) 3&#8211;16    tion pathway. It dumbbell-shaped symmetrical struc- <br /> hr observation fluorescent microscopy. (A B)       ture composed barrel-shaped 20S proteolytic core <br /> RPM2 pre4-2, (C D) RPM2 ump1-2, (E F) RPM2 soa3, <br /> (G&#8211;I) wild-type YMW2 cells. Bar, 5 &#65533;?&#65533;m.                           capped 19S regulatory particles. The yeast 20S protea- <br />                                                                  crystalized recently contains 14 dis- <br />                                                                  tinct structurally related subunits: 7 type &#65533;?&#65533; and <br /> (Figure 4B). After overnight shift restrictive         7 type &#65533;?&#65533; organized stacked heptomeric rings <br /> temperature 16&#11034;, entire culture displayed aber-           configuration &#65533;?&#65533;&#65533;?&#65533;&#65533;?&#65533;&#65533;?&#65533; (Groll et al. 1997). The 20S <br /> rant morphologies dead, serially connected             core complex assembled proprotein precursors <br /> ghosts observed (data shown). &#9004;rpm2 ump1-2 cul-              including inactive precursors proteasome prote- <br /> tures interconnected cells continue divide 37&#11034;          ases (Nandi et al. 1997). All &#65533;?&#65533; subunits encoded by <br /> but quite slowly decreased viability (data        essential genes. <br /> shown). At 30&#11034; interconnected, elongated morphol-                Subunits 20S core complex both <br /> ogy evident &#9004;rpm2 ump1-2 cultures              structural catalytic roles proteasome function. <br /> reached stationary phase (Figure 4C). &#9004;rpm2 soa3 cul-            Three different &#65533;?&#65533; subunits provide distinct N-ter- <br /> tures appeared wild-type cells permissive         minal threonine proteases proteasome catalytic <br /> restrictive temperatures (Figure 4D).                            core. The PUP1, PRE2, PRE3 genes encode &#65533;?&#65533;-sub- <br />    Mutant morphologies cells permissive re-            unit proteases trypsin-like, chymotrypsin-like, and <br /> strictive conditions prominent            PGPH specificities, respectively (Arendt Hoch- <br /> soa1 soa2 strains wild-type RPM2. All single        strasser 1997; Heinemeyer et al. 1997; Ciechanover <br />  1020                                       M. S. Lutz, S. R. Ellis N. C. Martin <br />  <br /> and Schwartz 1998). Pre4p catalytically active                Rpm2p multifunctional protein required <br /> as protease necessary PGPH activity (Hilt            mitochondrial RNase P activity essential <br /> et al. 1993; Heinemeyer et al. 1997). Mutations protea-        function, which, like mitochondria, required during <br /> some &#65533;?&#65533; subunits, Pre4p, thought influence          fermentative growth. The amino-terminal 714-aa por- <br /> the conformation catalytic subunits affect bind-        tion RPM2, supports growth glucose, is <br /> ing orientation substrate peptides. Consistent             sufficient serve high-copy suppressor mito- <br /> with critical role noncatalytic subunits protea-         chondrial protein import defect resulting mutant <br /> some structure observation null mutant           allele TOM40 (Kassenbrock et al. 1995). The pre4-2 <br /> PRE4 lethal. The requirement PRE4 not, how-             ump1-2 suppression &#9004;rpm2 suggests model relat- <br /> ever, solely related role PGPH activity,      ing proteasomal activity essential feature mito- <br /> truncated allele pre4-1 lacks PGPH activity displays          chondrial biogenesis. <br /> no detectable effects cell growth degrada-              Two proteasome mutants suppress mito- <br /> tion reporter protein vivo (Hilt et al. 1993).            chondrially related phenotypes reported <br />    Our work suggests Val-to-Phe substitution          (Campbell et al. 1994; rinaldi et al. 1998). A mutant <br /> &#65533;?&#65533;-7 global effects proteasome activity. Cells         allele RPT3 (ynt1-1) identified suppressor of <br /> a pre4-2 allele exhibit biochemical morphological             enhanced rate escape mitochondrial DNA to <br /> stress-related phenotypes characteristic cells re-        nucleus strains mutant alleles YME1 <br /> duced proteasome activity. Moreover, pre4-2 cells accu-           (Thorsness et al. 1993; Campbell et al. 1994). The <br /> mulated higher levels ubiquitin-protein conjugates             RPT3 mutant suppressed abnormal mitochondrial <br /> relative wild type. We interpret data indicate        morphologies growth defects associated yme1 <br /> that pre4-2 mutation affects just PGPH          mutations. Yme1p ATP- zinc-dependent prote- <br /> activity proteasome. Biochemical, immunologi-              ase component i-AAA proteolytic complex, which <br /> cal, crystallographic studies shown Pre4p           localized inner mitochondrial membrane <br /> is close proximity Pre3p Pup1p (Groll et al.            (Thorsness et al. 1993; Leonhard et al. 1996; Weber <br /> 1997; Heinemeyer et al. 1997; Kopp et al. 1997). Thus,            et al. 1996). Unassembled subunits ATP synthase and <br /> the pre4-2 mutation potentially disrupt multiple            respiratory chain components degraded, part, <br /> proteolytic activities proteasome           pathway involving Yme1p (Pearce Sher- <br /> overall structural integrity. Similar &#8220;poisoning&#65533;?          man 1995; Weber et al. 1996). RPT3, hand, <br /> <span id='am-46' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-47' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span><span id='am-48'  ></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Drosophila melanogaster</span> 20S proteasome mutant &#65533;?&#65533;-2              encodes ATPase activities base <br /> or &#65533;?&#65533;-6 subunits reported (Smyth Belote               subcomplex proteasome 19S regulatory particle <br /> 1999). However, unlike D. <span id='am-124' about='obo:NCBITaxon_80614' typeof='owl:Thing obo:NCBITaxon_80611 obo:NCBITaxon_1 obo:NCBITaxon_5302 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_33154 obo:NCBITaxon_452333 obo:NCBITaxon_5204 obo:NCBITaxon_451864 obo:NCBITaxon_68889 obo:NCBITaxon_155619 obo:NCBITaxon_2759'><span id='am-125' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-126' property="rdfs:label" content="Melanogaster" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-128'  ></span>melanogaster</span> mutants,               (Glickman et al. 1998). Like cells pre4-2 allele, <br /> pre4-2 recessive PRE4 presence absence            mutant allele RPT3 leads cold-sensitive growth <br /> of RPM2, indicating wild-type protein prefer-         nonfermentable carbon sources (Campbell et al. <br /> entially incorporated 20S proteasome              1994). It unlikely RPT3 mutant sup- <br /> assembly.                                                         presses YME1 mutations replacing mitochondrial <br />    Proper proteasome assembly critical proteasome           protein functional component i-AAA com- <br /> function. Unlike Pre4p, Ump1p integral com-             plex. Instead, RPT3 function suppress YME1 <br /> ponent 20S core complex. However, Ump1p              mutants disrupting pathway used degrade ab- <br /> is involved assembly maturation 20S             normal mitochondria (Campbell Thorsness <br /> core complex, mutations affect expression             1998). Recent studies indicated vacuolar <br /> function global effects proteasome function               protease, Pep4p, plays important role escape <br /> (Ramos et al. 1998). Thus, ump1-2, like pre4-2, appears           mitochondrial DNA nucleus YME1 mutants. <br /> to suppress lack growth fermentative carbon             Mutants PEP4 suppress DNA transfer mitochon- <br /> sources &#9004;rpm2 cells disrupting proteasome func-             dria nucleus YME1 mutants suppressing <br /> tion. Yeast carrying null alleles UMP1 accumulated             growth defects abnormal mitochondrial morpholo- <br /> 15S precursors 20S core complex proPre2p               gies. This suggests vacuole function the <br /> (Ramos et al. 1998). They grow slower rates               degradation abnormal mitochondria YME1 strains <br /> and sensitive stressful conditions. In agreement           (Campbell et al. 1994; Campbell Thorsness <br /> with results, ump1-2 mutant reported           1998). In contrast, RPT3 mutants suppress pheno- <br /> sensitivities canavanine CdCl2 characteristic           types associated YME1 mutation, indicating <br /> proteasome mutants, presumably cells              proteasome mutants help promote mitochondrial <br /> fewer fully assembled, proteolytically active 20S protea-         integrity. <br /> some complexes. The semidominant nature                       A proteasome mutant shown suppress <br /> ump1-2 allele suppressing RPM2-null mutations sug-             mutation mitochondrial genome (Rinaldi et <br /> gests single wild-type UMP1 gene &#9004;rpm2/               al. 1994, 1995). A search MnCl2-generated suppres- <br /> &#9004;rpm2 diploid does support assembly wild-              sors mitochondrial tRNAASP point mutant led to <br /> type levels 20S proteasome complexes.                          isolation yeast proteasomal gene RPN11/MPR1 <br />                                             Proteasome Mutants Suppress &#9004;rpm2                                       1021 <br />  <br /> (Zennaro et al. 1989; Rinaldi et al. 1994, 1998; Glick-       tion maintenance mitochondria. This model ac- <br /> man et al. 1998). RPN11/MPR1 non-ATPase subunit          counts results encompassing re- <br /> of lid portion 19S proteasomal regulatory          sults obtained YME1 tRNAASP mutants <br /> particle (Glickman et al. 1998). Similar pre4-2, mpr1-1    (Zennaro et al. 1989; Campbell et al. 1994; Campbell <br /> cultures grown stationary phase accumulate ubiquiti-       Thorsness 1998). <br /> nated proteins growth phenotypes prom-             Nuclear gene products comprise majority mito- <br /> inent glycerol medium. Cells mpr1-1 allele        chondrial proteins needed respiratory nonrespi- <br /> contain increased quantity total mitochondrial          ratory growth. A mitochondrial import integral <br /> DNA cell, suggesting decrease proteasome        transport nuclear-encoded proteins into <br /> activity elevate copy number mitochondrial         different mitochondrial compartments. Increased <br /> DNA (Rinaldi et al. 1998).                                    stabilization mitochondrial outer membrane proteins <br />    The proteasome mutants isolated unique            increased rates mitochondrial import affect <br /> with respect work described         growth characteristics yeast respira- <br /> specifically affect 20S proteolytic core vs. 19S      tory fermentative conditions. An RPM2 essential <br /> regulatory cap subunits 26S proteasome. A sup-         growth function directed level compen- <br /> pression mechanism stabilizes mitochondrial DNA          sated increasing half-life specific protea- <br /> through loss proteasome function, observed          targets. This hypothesis supported previous <br /> in Rinaldi et al. (1998), account suppression      results RPM2 isolated high-copy suppressor <br /> in &#9004;rpm2 yeast &#9004;rpm2 ump1-2 &#9004;rpm2 soa3-1        defective allele essential mitochondrial im- <br /> cells &#65533;?&#65533;o. Alternatively, ynt1-1 proteasome mu-     port channel protein Tom40p (Kassenbrock et al. <br /> tant suppresses yme1 supporting global mitochondrial       1995). <br /> integrity, bypassing vacuolar degradation            Proteasomes regulate levels proteins tar- <br /> pathway, similar mechanism stabilizing mitochondria         geted mitochondrial compartments. At case <br /> in presence decreased proteasomal degradation          control mitochondrial protein levels cystosolic <br /> could account phenotypic suppression &#9004;rpm2 pre4-       proteasome activity established. Regulation to <br /> 2 &#9004;rpm2 ump1-2 strains.                                   achieve proper levels isoforms cytochrome c, <br />    We suggest mechanism proteasome          encoded different nuclear genes localized to <br /> mutants suppress RPM2-null allele indirect, affect-    intermembrane space, occurs ubiquitin- <br /> ing steady-state level key protein(s) compen-     proteasome degradation mechanism (Pearce Sher- <br /> sate loss RPM2. Unlike mammalian cells, lower      man 1997). It likely proteins <br /> eukaryotes, including S. cerevisiae, degrade long-     destined mitochondria regulated protea- <br /> lived proteins ubiquitin-proteasome path-         activity mitochondrial import. <br /> way (reviewed Goldberg et al. 1997). Thus, pre4-2         The direct turnover mitochondria utilizing path- <br /> ump1-2 strains accumulate short-lived proteins,        way balancing proteasomal vacuolar degradation <br /> including regulatory proteins known profound          activities provide cells mechanism mon- <br /> effects cell growth. Consistent interpreta-      itoring removing unneeded improperly function- <br /> tion observation null allele RPM2      ing mitochondria (Campbell Thorsness 1998; <br /> also suppressed multiple copies YBL066,        Sutovsky et al. 1998). A hypothesis mitochondrial <br /> encodes putative transcription factor, PAB1,      turnover involves ubiquitin-tagged proteins supported <br /> encodes poly (A)-binding protein (Groom et al.            studies ubiquitin localization. Ubiquitin conjugates <br /> 1998; H. C. Heyman N. C. Martin, personal com-            ubiquitin-conjugating enzymes localized <br /> munication). Both genes potential        mitochondrial membrane fractions mito- <br /> increase steady-state level certain proteins, which,   chondrial outer membrane (Zhaung McCauley <br /> in turn, suppress RPM2 mutants.                         1989; Magnani et al. 1991). Fisk Yaffe (1999) dem- <br />    Proteasome degradation outer mitochondrial              onstrated loss E3 ubiquitin ligase affects <br /> membrane proteins act general       mitochondrial inheritance. A mechanism protein <br /> of mitochondrial maintenance similar described        ubiquitination cytosolic mitochondria <br /> for ER membrane proteins, insuring improperly        useful regulated degradation mitochon- <br /> folded proteins misassembled complexes de-             drial proteins proteasome targeted turn- <br /> graded rapidly (Kopito 1997; Plemper Wolf                 mitochondria vacuole. <br /> 1999). Essential functions mitochondria depend           Cell fractionation experiments previously shown <br /> on outer mitochondrial membrane proteins include im-          detectable Rpm2p localized mitochondrial <br /> port, fusion, distribution. Decreased proteasome          fraction (Kassenbrock et al. 1995). Rpm2p, compo- <br /> activity impact mitochondrial function          nent mitochondrial RNase P activity, localized to <br /> increasing longevity mitochondrially associated        matrix. However, presence low levels of <br /> proteins, potentially suppressing lethal phenotypes        Rpm2p mitochondrial fractions cyto- <br /> strains containing mutations adverse proper func-         sol ruled out. The limited growth the <br />  1022                                                M. S. Lutz, S. R. Ellis N. C. Martin <br />  <br /> small &#9004;rpm2 colonies suggests proteasome target                         Structure 20S proteasome yeast 2.4 A resolution. Nature <br />                                                                                386: 463&#8211;471. <br /> affecting mitochondrial function, mitochondria                          Groom, K. R., 1995 Genetic approaches yield new insights the <br /> themselves, diluted 25 generations. The spe-                      functional expression biogenesis Rpm1r Rpm2p <br /> cific role Rpm2p cell growth remains un-                             subunits yeast mitochondrial RNase P. Thesis, University of <br />                                                                                Louisville, Louisville. <br /> known clearly demonstrated decreased pro-                      Groom, K. R., H. C. Heyman, M. C. Steffen, L. Hawkins N. C. <br /> teasome function provide resources necessary                           Martin, 1998 <span id='am-12' about='obo:NCBITaxon_4910' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-13'  ></span><span id='am-14' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Kluyveromyces" datatype="xsd:string"></span>Kluyveromyces</span> lactis SEF1 Saccharomyces cere- <br /> in absence RPM2. Our results suggest Rpm2p                         visiae homologue bypass unknown essential function, not <br />                                                                                mitochondrial RNase P function, S. cerevisiae RPM2 <br /> has role regulating synthesis turnover                          gene. Yeast 14: 77&#8211;87. <br /> proteins important growth. This role by-                        Haas, A. L., 1988 Immunochemical probes ubiquitin pool dynam- <br /> passed novel proteasome mutants character-                          ics, pp. 173&#8211;206 Ubiquitin, edited M. Rechsteiner. Plenum <br />                                                                                Press, Salt Lake City. <br /> ized here.                                                                 Haas, A. L., P. M. Bright, 1985 The immunochemical detec- <br />   We thank Ms. <span id='am-63' about='obo:NCBITaxon_1308092' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_33154 obo:NCBITaxon_85772 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_32409 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_7399 obo:NCBITaxon_50557 obo:NCBITaxon_6072 obo:NCBITaxon_6960'><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="Clarissa" datatype="xsd:string"></span><span id='am-66'  ></span><span id='am-67' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Clarissa</span> Moxely Mr. Michael Mindrum techni-             tion quantitation intracellular ubiquitin-protein conju- <br /> cal assistance complementation assays library screens.            gates. J. Biol. Chem. 260: 12464&#8211;12473. <br />                                                                            Heinemeyer, W., M. Fischer, T. Krimmer, U. Stachon D. H. <br /> We thank Dr. A. Haas generous donation affinity purified <br />                                                                                Wolf, 1997 The active sites eukaryotic 20S proteasome <br /> polyclonal antiubiquitin antibody Dr. M. Walberg gener-            involvement subunit precursor processing. J. Biol. <br /> ous donation YMW1 YMW2 yeast strains. This work sup-                Chem. 272: 25200&#8211;25209. <br /> ported National Institutes Health grant 992332 N.C.M.             Hershko, A., 1997 Roles ubiquitin-mediated proteolysis cell <br />                                                                                cycle control. Curr. Opin. Cell Biol. 9: 788&#8211;799. <br />                                                                            Hill, K., K. Model, M. T. Ryan, K. Dietmeier, F. Martin et al., <br />                                                                                1998 Tom40 forms hydrophilic channel mitochon- <br />                       LITERATURE CITED                                         drial import pore preproteins. Nature 395: 516&#8211;521. <br />                                                                            Hilt, W., D. H. Wolf, 1996 Proteasomes: destruction pro- <br /> Altschul, S. F., W. Gish, W. Miller, E. W. Myers D. J. Lipman,             gramme. Trends Biochem. Sci. 21: 96&#8211;102. <br />     1990 Basic local alignment search tool. J. Mol. Biol. 215: 403&#8211;        Hilt, W., C. Enenkel, A. Gruhler, T. Singer D. H. Wolf, 1993 <br />     410.                                                                       The PRE4 gene codes subunit yeast proteasome <br /> Arendt, C. S., M. Hochstrasser, 1997 Identification                 necessary peptidylglutamyl-peptide-hydrolyzing activity. Muta- <br />     yeast 20S proteasome catalytic centers subunit interactions            tions link proteasome stress- ubiquitin-dependent pro- <br />     required active-site formation. Proc. Natl. Acad. Sci. USA 94:         teolysis. J. Biol. Chem. 268: 3479&#8211;3486. <br />     7156&#8211;7161.                                                             Kaiser, C., S. Michaelis A. Mitchell, 1994 Methods Yeast <br /> Attardi, G., G. Schatz, 1988 Biogenesis mitochondria.                   Genetics. Cold Spring Harbor Laboratory Press, Cold Spring Har- <br />     Annu. Rev. Cell Biol. 4: 289&#8211;333.                                          bor, NY. <br /> Campbell, C. L., P. E. Thorsness, 1998 Escape mitochon-             Kassenbrock, C. K., G. J. Gao, K. R. Groom, P. Sulo, M. G. Douglas <br />     drial DNA nucleus yme1 yeast mediated vacuolar-            et al., 1995 RPM2, independently mitochondrial RNase P <br />     dependent turnover abnormal mitochondrial compartments.                 function, suppresses ISP42 mutant defective mitochondrial <br />     J. Cell Sci. 111: 2455&#8211;2464.                                               import essential normal growth. Mol. Cell. Biol. 15: <br /> Campbell, C. L., N. Tanaka, K. H. White P. E. Thorsness, 1994              4763&#8211;4770. <br />     Mitochondrial morphological functional defects yeast            King, R. W., R. J. Deshaies, J. M. Peters M. W. Kirschner, 1996 <br />     caused yme1 suppressed mutation 26S protease                How proteolysis drives cell cycle. Science 274: 1652&#8211;1659. <br />     subunit homologue. Mol. Biol. Cell 5: 899&#8211;905.                         Kopito, R. R., 1997 ER quality control: cytoplasmic connection. <br /> Chen, D. C., B. C. Yang T. T. Kuo, 1992 One-step transforma-               Cell 88: 427&#8211;430. <br />     tion yeast stationary phase. Curr. Genet. 21: 83&#8211;84.             Kopp, F., K. B. Hendil, B. Dahlmann, P. Kristensen, A. Sobek et <br /> Chen, P., M. Hochstrasser, 1995 Biogenesis, structure                  al., 1997 Subunit arrangement human 20S proteasome. <br />     function yeast 20S proteasome. EMBO J. 14: 2620&#8211;2630.               Proc. Natl. Acad. Sci. USA 94: 2939&#8211;2944. <br /> Chen, P., M. Hochstrasser, 1996 Autocatalytic subunit pro-             Leonhard, K., J. M. Herrmann, R. A. Stuart, G. Mannhaupt, W. <br />     cessing couples active site formation 20S proteasome             Neupert et al., 1996 AAA proteases catalytic sites oppo- <br />     completion assembly. Cell 86: 961&#8211;972.                                  site membrane surfaces comprise proteolytic the <br /> Chun, K. T., N. Mathias M. G. Goebl, 1996 Ubiquitin-depen-                 ATP-dependent degradation inner membrane proteins mi- <br />     dent proteolysis cell cycle control yeast. Prog. Cell Cycle         tochondria. EMBO J. 15: 4218&#8211;4229. <br />     Res. 2: 115&#8211;127.                                                       Magnani, M., G. Serafini, A. Antonelli, M. Malatesta G. <br /> Ciechanover, A., A. L. Schwartz, 1998 The ubiquitin-protea-                Gazzanelli, 1991 Evidence particulate location ubiqui- <br />     pathway: complexity myriad functions protein               tin conjugates ubiquitin-conjugating enzymes rabbit brain. <br />     death. Proc. Natl. Acad. Sci. USA 95: 2727&#8211;2730.                           J. Biol. Chem. 266: 21018&#8211;21024. <br /> Dang, Y. L., N. C. Martin, 1993 Yeast mitochondrial RNase P.           Moczko, M., K. Dietmeier, T. Sollner, B. Segui, H. F. Steger et <br />     Sequence RPM2 gene demonstration product            al., 1992 Identification mitochondrial receptor complex <br />     protein subunit enzyme. J. Biol. Chem. 268: 19791&#8211;19796.          Saccharomyces cerevisiae. FEBS Lett. 310: 265&#8211;268. <br /> De Pinto, B., S. B. Malladi N. Altamura, 1999 MitBASE pilot:           Morales, M. J., Y. L. Dang, Y. C. Lou, P. Sulo N. C. Martin, <br />     database nuclear genes involved mitochondrial biogenesis           1992 A 105-kDa protein required yeast mitochondrial <br />     regulation Saccharomyces cerevisiae. Nucleic Acids Res.         RNase P activity. Proc. Natl. Acad. Sci. USA 89: 9875&#8211;9879. <br />     27: 147&#8211;149.                                                           Myers, A. M., L. K. Pape A. Tzagoloff, 1985 Mitochondrial <br /> Fisk, H. A., M. P. Yaffe, 1999 A role ubiquitination                protein synthesis required maintenance intact mitochon- <br />     mitochondrial inheritance Saccharomyces cerevisiae. J. Cell Biol.       drial genomes Saccharomyces cerevisiae. EMBO J. 4: 2087&#8211;2092. <br />     145: 1199&#8211;1208.                                                        Nandi, D., E. Woodward, D. B. Ginsburg J. J. Monaco, 1997 <br /> Glickman, M. H., D. M. Rubin, V. A. Fried D. Finley, 1998 The              Intermediates formation mouse 20S proteasomes: impli- <br />     regulatory particle Saccharomyces cerevisiae proteasome. Mol.       cations assembly precursor <span id='am-6' about='obo:NCBITaxon_3554' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_3563 obo:NCBITaxon_91827 obo:NCBITaxon_71240 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_3524 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_3193'><span id='am-7'  ></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasRelatedSynonym" content="Beta L. 1753" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Beta" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>beta</span> subunits. EMBO J. 16: <br />     Cell. Biol. 18: 3149&#8211;3162.                                                 5363&#8211;5375. <br /> Goldberg, A. L., T. N. Akopian, A. F. Kisselev, D. H. Lee              Pearce, D. A., F. Sherman, 1995 Degradation cytochrome <br />     M. Rohrwold, 1997 New insights mechanisms                     oxidase subunits mutants yeast lacking cytochrome c and <br />     importance proteasome intracellular protein degrada-             suppression degradation mutation yme1. J. Biol. <br />     tion. Biol. Chem. 378: 131&#8211;140.                                            Chem. 270: 20879&#8211;20882. <br /> Groll, M., L. Ditzel, J. Lowe, D. Stock, M. Bochtler et al., 1997          Pearce, D. A., F. Sherman, 1997 Differential ubiquitin-depen- <br />                                                        Proteasome Mutants Suppress &#9004;rpm2                                                      1023 <br />     dent degradation yeast apo- cytochrome c isozymes. J. Biol.      Sikorski, R. S., J. D. Boeke, 1991 In vitro mutagenesis and <br />     Chem. 272: 31829&#8211;31836.                                                     plasmid shuffling: cloned gene mutant yeast. Methods <br /> Plemper, R. K., D. H. Wolf, 1999 Retrograde protein transloca-              Enzymol. 194: 302&#8211;318. <br />     tion: ERADication secretory proteins health disease.          Sikorski, R. S., P. Hieter, 1989 A shuttle vectors and <br />     Trends Biochem. Sci. 24: 266&#8211;270.                                           yeast host strains designed efficient manipulation DNA in <br /> Pon, L., G. Schatz, 1991 Biogenesis yeast mitochondria, pp.              Saccharomyces cerevisiae. Genetics 122: 19&#8211;27. <br />     333&#8211;406 The Molecular Cellular Biology Yeast Saccharo-    Smyth, K. A., J. M. Belote, 1999 The dominant temperature- <br />     myces: Genome Dynamics, Protein Synthesis, Energetics, edited        sensitive lethal DTS7 Drosophila melanogaster encodes altered <br />     J. R. Broach, J. R. Pringle E. W. Jones. Cold Spring Harbor             20S proteasome beta-type subunit. Genetics 151: 211&#8211;220. <br />     Press, Cold Spring Harbor, NY.                                          Stribinskis, V., G. J. Gao, P. Sulo, Y. L. Dang N. C. Martin, <br /> Ramos, P. C., J. Hockendorff, E. S. Johnson, A. Varshavsky                  1996 Yeast mitochondrial RNase P RNA synthesis altered in <br />     R. J. Dohmen, 1998 Ump1p required proper maturation                  RNase P protein subunit mutant: insights biogenesis <br />     20S proteasome substrate comple-                mitochondrial RNA-processing enzyme. Mol. Cell. Biol. 16: <br />     tion assembly. Cell 92: 489&#8211;499.                                     3429&#8211;3436. <br /> Rapaport, D., W. Neupert R. Lill, 1997 Mitochondrial protein            Sutovsky, P., R. Moreno G. Schatten, 1998 Sperm mitochon- <br />     import. Tom40 plays major role targeting translocation             drial ubiquitination model explaining strictly maternal <br />     preproteins forming specific binding site prese-            mtDNA inheritance mammals. Mol. Biol. Cell 9: 309a. <br />     quence. J. Biol. Chem. 272: 18725&#8211;18731.                                Thorsness, P. E., K. H. White T. D. Fox, 1993 Inactivation of <br /> Rinaldi, T., S. Francisci, E. Sennaro, L. Frontali M. <br /> </body></html>