<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Why Do Hubs Tend Be Essential Protein <br /> Networks? <br /> Xionglei He, Jianzhi Zhang* <br /> Department Ecology Evolutionary Biology, University Michigan, Ann Arbor, Michigan, United States America <br />  <br />  <br /> The protein&#8211;protein interaction (PPI) network small number highly connected protein nodes (known hubs) <br /> and poorly connected nodes. Genome-wide studies deletion hub protein likely lethal <br /> than deletion non-hub protein, phenomenon known centrality-lethality rule. <span id='am-44' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-45' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-48'  ></span>This</span> rule widely believed to <br /> reflect special importance hubs organizing network, turn suggests biological significance of <br /> network architectures, key notion systems biology. Despite popularity explanation, underlying <br /> cause centrality-lethality rule critically examined. We propose concept essential PPIs, <br /> which PPIs indispensable survival reproduction organism. Our network analysis suggests <br /> that centrality-lethality rule unrelated network architecture, explained simple fact hubs <br /> have large numbers PPIs, high probabilities engaging essential PPIs. We estimate ; 3% PPIs <br /> are essential yeast, accounting ; 43% essential genes. As expected, essential PPIs evolutionarily more <br /> conserved nonessential PPIs. Considering role essential PPIs determining gene essentiality, the <br /> yeast PPI network functionally robust random networks, far robust potential optimum. <br /> These findings provide new perspectives biological relevance network structure robustness. <br /> Citation: He X, Zhang J (2006) Why hubs tend essential protein networks? PLoS Genet 2(6): e88. DOI: 10.1371/journal.pgen.0020088 <br />  <br />  <br />                                                                                       That is, functional importance node thought arise <br /> Introduction <br />                                                                                       structural importance network [6,7,9,10]. If <br />    A network composed multiple nodes connected                               true, interpretation suggests biological signi&#65533;?cance of <br /> edges. Most complex networks scale-free, power-                            network structures fundamental systems <br /> law distribution number edges node, node                             biology. We challenge view proposing an <br /> connectivity [1,2]. That is, scale-free network contains                          alternative explanation centrality-lethality rule that <br /> small number highly connected nodes (hubs) large                             does invoke network architecture. We evaluate <br /> number poorly connected nodes (non-hubs). The relative                             new explanation empirical <span id='am-57' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-58' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-61'  ></span>data</span> demonstrate <br /> importance node network measured                            prevailing interpretation centrality-lethality <br /> magnitude changes network structure caused                               rule unlikely correct. <br /> removal node. More accurately, measure should <br /> be termed structural importance node. For instance, <br /> computational analysis shows removing hubs increases                             Results/Discussion <br /> the proportion unreachable pairs nodes mean                             An Alternative Explanation Centrality-Lethality Rule <br /> shortest path length pairs reachable nodes                          Based Essential PPIs <br /> the network (i.e., network diameter) removing                                  The current analysis PPI networks treats edges <br /> non-hubs [3]. Hence, hubs important non-hubs                            equally. But reality, PPIs important than <br /> to maintenance global network structure. In                                others. This consideration particularly meaningful <br /> biomolecular networks, genes proteins nodes                              PPIs essential (indispensable) the <br /> and molecular interactions edges, importance                             survival reproduction organism. An essential <br /> node measured magnitude changes                              interaction proteins makes proteins <br /> network function organismal &#65533;?tness caused removal                           essential, removal protein causes <br /> of node. Such measure called functional <br /> importance node. For example, genome-wide gene <br /> deletion studies small faction genes                              Editor: Takashi Gojobori, National Institute Genetics, Japan <br /> genome indispensable survival reproduction                           Received January 24, 2006; Accepted April 26, 2006; Published June 2, 2006 <br /> an organism [4,5]; genes referred essential                           A previous version article appeared Early Online Release April 26, <br /> genes. It scale-free protein&#8211;protein                            2006 (DOI: 10.1371/journal.pgen.0020088.eor). <br /> interaction (PPI) network [6&#8211;8], hubs tend essential [6].                       DOI: 10.1371/journal.pgen.0020088 <br /> This phenomenon observed yeast, nematode, <br />                                                                                       Copyright: &#211; 2006 He Zhang. This open-access article distributed under <br /> and &#64258;y [9&#8211;11] commonly referred centrality-                          terms Creative Commons Attribution License, permits unrestricted <br /> lethality rule [6]. Using terms described above,                              use, distribution, reproduction medium, provided original author <br /> centrality-lethality rule indicates correlation                           source credited. <br /> node&#8217;s structural importance PPI network                               Abbreviations: ER, Erdo&#168;s-Re&#180;nyi; IBEP, interaction essential proteins; PPI, <br /> functional importance. Without critical analysis, corre-                         protein&#8211;protein interaction <br /> lation widely interpreted causal relationship.                          * To correspondence addressed. E-mail: jianzhi@umich.edu <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                    0826                                            June 2006 | Volume 2 | Issue 6 | e88 <br />                                                                                                               Essential Protein&#8211;Protein Interactions <br />  <br />  <br /> Synopsis                                                                      IBEPs randomly rewired networks (Figure 1B). The <br />                                                                               mean m 592.6. None 10,000 m values greater than <br /> Proteins interactions form protein&#8211;protein interaction            number IBEPs real network, strongly suggesting <br /> network, proteins nodes interactions            excess IBEPs real network (p , 0.0001). This <br /> the edges. Genomic studies deleting highly connected              excess evident different datasets yeast PPIs in <br /> protein node (hub) likely lethal organism            nematode PPIs [14,15] (see Figures S1 S2). <br /> deleting lowly connected node (non-hub), phenomenon known                 Under assumption excess IBEPs entirely <br /> as centrality-lethality rule. Because hubs important <br />                                                                               caused essential PPIs, estimate &#188; (807&#8211;592.6)/7356 <br /> than non-hubs organizing global network structure, the <br />                                                                               &#188; 2.92% interactions yeast PPI network essential. <br /> centrality-lethality rule widely believed reflect significance <br /> of network architecture determining network function, key                The standard error 0.23%. Here used random <br /> notion systems biology. In work, authors proposed               rewiring estimate easy ways to <br /> small fraction randomly distributed essential interactions,        calculate analytically unless self-interactions allowed. <br /> which lethal organism disrupted. Under scenario,              In network, 836 proteins, 20.3% nodes, are <br /> a hub likely essential non-hub simply            essential. In addition essential PPIs, factors <br /> the hub interactions higher chance engage              (e.g., protein-DNA interaction) render protein <br /> in essential interaction. Hence, centrality-lethality rule          essential. Let b probability node becomes <br /> explained involvement network architecture. Using              essential factors. To estimate b, &#65533;?rst remove <br /> yeast data, authors provided empirical evidence supporting their <br />                                                                               information gene essentiality yeast PPI network. <br /> hypothesis. Their proposal results challenge prevailing view in <br /> systems biology provide new perspective role of <br />                                                                               We randomly assign 807-m essential edges this <br /> network structures biology.                                                network, m randomly drawn distribution <br />                                                                               Figure 1B. Note 807-m estimated number of <br />                                                                               essential edges. Nodes having essential edges marked <br />                                                                               essential. Next, mimic in&#64258;uence factors <br /> lethality infertility disruption interaction. <br />                                                                               cause gene essentiality randomly marking nodes as <br /> Empirical data indicate existence essential PPIs. For <br />                                                                               essential, total number essential nodes the <br /> example, yeast proteins SPT16 POB3 essential <br />                                                                               network 836. Repeating procedure 10,000 times, <br /> and form heterodimers function DNA replica- <br />                                                                               estimate essential PPIs render 8.7% nodes <br /> tion; genetic studies showed interaction critical <br />                                                                               essential, factors render b &#188; 12.64% nodes <br /> for function [12]. Essential PPIs potentially explain <br />                                                                               essential. The standard error b 0.63%. Note some <br /> the centrality-lethality rule, proteins PPIs <br />                                                                               nodes (1.1%) affected essential PPIs other <br /> have greater probability engage essential <br />                                                                               factors. Approximately 43% (8.7%/20.3%) protein essen- <br /> PPI, having higher chance essential. Note               tiality attributable essential interactions PPI <br /> network architecture invoked explanation.                      network. <br /> Evaluation Number Essential PPIs Their                             Our estimates b biased factors. <br />                                                                               First, features yeast PPI network been <br /> Contribution Gene Essentiality <br />                                                                               distorted random rewiring, affecting estimation <br />   It dif&#65533;?cult identify essential PPIs experimentally the <br />                                                                               a. For example, known links highly <br /> genomic scale, identi&#65533;?cation requires the <br />                                                                               connected nodes suppressed PPI networks [16,17]. <br /> demonstration disrupting interaction two <br />                                                                               Because highly connected nodes tend essential [6,9&#8211;11], <br /> essential proteins affecting aspects the <br />                                                                               suppression reduces number IBEPs. Hence, this <br /> protein functions causes lethality infertility. Here use              suppression accounted rewiring, may <br /> computational approach evaluate prevalence                          overestimated m, consequently underestimated a. <br /> essential PPIs contribution essential PPIs gene                 However, <span id='am-1' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-2' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-3'  ></span><span id='am-4' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Bias" datatype="xsd:string"></span>bias</span> probably small, suppression <br /> essentiality genomic level. Our analysis focuses                appears largely limited nonessential proteins [14]. <br /> yeast <span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-9'  ></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> PPI gene                  Second, quality PPI data affect reliability of <br /> essentiality data complete <span id='am-49' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-50' property="rdfs:label" content="species" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>species</span>.                          estimates. In particular, transforming protein complex <br />   We built yeast PPI network, 4,126 protein                      information binary PPI data using &#8216;&#8216;spoke&#8217;&#8217; <br /> nodes linked 7,356 edges. The PPI data used                    model (the bait predicted interact members a <br /> compiled manually Comprehensive Yeast Genome                           complex) &#8216;&#8216;matrix&#8217;&#8217; model (all members complex <br /> Database [13] literature published large-scale                   predicted interact members the <br /> experiments. As mentioned, proteins forming                            complex) [18] tends generate extra IBEPs large <br /> essential PPI essential (Figure 1A). On contrary,                 complexes, lead overestimation a. <br /> interactions essential proteins (IBEPs)                    However, data include protein <br /> not essential, essentiality protein                complex information recently produced high-throughput <br /> due factors essential PPIs (Figure 1A). This                    methods [19,20], largely immune this <br /> feature allows estimate number essential PPIs               problem. Third, assumed excess observed IBEPs <br /> network, number IBEPs increases number                     real network entirely essential PPIs, it <br /> of essential PPIs. There 807 IBEPs network. We                     caused nonrandom features real <br /> generated control network randomly rewiring edges                    network [14,15]. Finally, estimation b based the <br /> of real network keeping node connectivity (k)                   assumption factors causing protein essentiality <br /> unchanged node. By repeating procedure                         affect nodes PPI network equally random <br /> 10,000 times, obtained distribution number (m)                  manner. <br />  <br />       PLoS Genetics | www.plosgenetics.org                                 0827                              June 2006 | Volume 2 | Issue 6 | e88 <br />                                                                                                                      Essential Protein&#8211;Protein Interactions <br />  <br />  <br />                                                                                     yeast PPI network largely captured two-step <br />                                                                                     procedure, involves essential PPIs randomly <br />                                                                                     distributed edges essentiality-determining <br />                                                                                     factors randomly distributed nodes. The &#65533;?nal <br />                                                                                     critical evaluation estimates b to <br />                                                                                     test protein essentiality predicted using these <br />                                                                                     estimates. For protein nonessential, conditions <br />                                                                                     satis&#65533;?ed. First, protein essential PPI. <br />                                                                                     Second, protein affected factors that <br />                                                                                     cause essentiality. Thus, probability (PE) protein <br />                                                                                     k PPIs essential is: <br />  <br />                                                                                                         PE &#188; 1   &#240;1   b&#222;&#240;1   a&#222;k ;                    &#240;1&#222; <br />                                                                                     b estimated earlier. Thus, PE values <br />                                                                                     predicted k using equation. Our <br />                                                                                     observed PE yeast PPI network matched the <br />                                                                                     predicted PE (Figure 2A). We did compare PE values k <br />                                                                                     . 10, nodes k value k . <br />                                                                                     10. Equation 1 rewritten natural logarithm as: <br />  <br />                                                                                                  ln&#240;1   PE &#222; &#188; k ln&#240;1   a&#222; &#254; ln&#240;1   b&#222;:               &#240;2&#222; <br />  <br />                                                                                       Equation 2 predicts ln(1-PE) changes linearly k. <br />                                                                                     This linear relationship con&#65533;?rmed yeast PPI <br />                                                                                     network (correlation coef&#65533;?cient &#188; 0.927, p &#188; 0.0001; Figure <br />                                                                                     2B). We estimate &#188; 3.29% b &#188; 12.8% slope <br />                                                                                     Y-intersect linear regression, respectively (Figure <br />                                                                                     2B). These estimates signi&#65533;?cantly different our <br />                                                                                     earlier estimates based simulated networks (p . 0.5). <br />                                                                                     Taken together, tests con&#65533;?rm estimates of <br />                                                                                     b reasonably good. <br /> Figure 1. Essential Edges (Interactions) PPI Networks <br /> (A) A hypothetical PPI network 12 proteins. Black white nodes <br />                                                                                     Essential PPIs Are Evolutionarily More Conserved than <br /> refer essential nonessential proteins, respectively. Thick          Nonessential PPIs <br /> edges depict essential nonessential interactions, respectively.                    It interesting predict PPIs essential. <br /> Proteins linked essential interaction essential, whereas <br /> an interaction essential proteins (IBEP) be <br />                                                                                     But prediction naturally dif&#65533;?cult estimating <br /> essential.                                                                          percentage PPIs essential, the <br /> (B) More IBEPs yeast PPI network randomly rewired                    scarcity information individual PPIs. Nonetheless, is <br /> networks. &#8216;&#8216;Observed&#8217;&#8217; indicates observed number (807) IBEPs              clear IBEPs essential. The probability an <br /> the real network. The gray bars distribution number (m) <br /> of IBEPs 10,000 randomly rewired networks.                                       IBEP essential (807&#8211;592.6)/807 &#188; 0.27. Here 807 total <br /> DOI: 10.1371/journal.pgen.0020088.g001                                              number IBEPs (807&#8211;592.6) estimated number of <br />                                                                                     essential interactions. If interacting essential proteins do <br />   To examine potential biases evaluate                             interact essential proteins (observation O), the <br /> reliability estimates, conducted tests. First,                      posterior probability interaction essential (event <br /> according analysis, factors essential PPIs                        E) derived Bayes theorem as: <br /> render b &#188; 12.6% proteins essential. There 1,952 yeast <br /> proteins PPIs included                                           P&#240;OjE&#222;P&#240;E&#222;       13a <br />                                                                                           P&#240;EjO&#222; &#188;               &#188; <br /> PPI network. Interestingly, 11.9% 6 0.8 % (233/1,952)                                                 P&#240;O&#222;      &#254; &#240;1   a&#222;b2 <br /> these proteins essential, number statistically indistin-                                                   0:029 <br /> guishable b (p . 0.4, chi-squares test). This congruence                                     &#188;                                &#188; 0:65:            &#240;3&#222; <br />                                                                                                       0:029 &#254; &#240;1   0:029&#222; 3 0:1262 <br /> suggests estimate b reliable assumption <br /> of stochastically equal in&#64258;uences factors                       The yeast PPI network contains 38 &#8216;&#8216;probably <br /> nodes acceptable. Second, estimation relied                       essential&#8217;&#8217; PPIs (see Table S1 gene names functions). <br /> simulated networks, compared network features                            Compared nonessential PPIs, essential PPIs expected to <br /> simulated real networks. In particular, node essentiality                       conserved evolution importance to <br /> was randomly reassigned estimation b,                        organismal survival reproduction. To test this <br /> network structure unaltered. We                                  hypothesis, assembled PPI network fruit &#64258;y <br /> frequency distribution node connectivity similar                              <span id='am-30' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-31' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-35'  ></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span>Drosophila melanogaster</span>. There 1,066 PPIs yeast <br /> between reassigned networks real network                            proteins orthologs fruit &#64258;y, 4.3% these <br /> both essential nonessential nodes (Figure S3). This result                      PPIs conserved species (Table 1 and <br /> suggests determination node essentiality                         Table S2). In comparison, 7.6% IBEPs 26.3% of <br />  <br />        PLoS Genetics | www.plosgenetics.org                                  0828                                   June 2006 | Volume 2 | Issue 6 | e88 <br />                                                                                                                                              Essential Protein&#8211;Protein Interactions <br />  <br />  <br />                                                                                                    probably essential PPIs conserved species, <br />                                                                                                    con&#65533;?rming prediction essential PPIs evolutio- <br />                                                                                                    narily conserved nonessential PPIs (Table 1 and <br />                                                                                                    Table S2). Other phylogenetic conservation, 38 <br />                                                                                                    probably essential interactions special <br />                                                                                                    features. They apparently enriched functional <br />                                                                                                    categories, biological processes, stable protein complexes. <br />                                                                                                    For example, 45% 38 probably essential interactions <br />                                                                                                    involve proteins appear protein <br />                                                                                                    complexes, compared 47% 748 IBEPs (p . <br />                                                                                                    0.5, v2 test). It possible certain enrichment does exist, <br />                                                                                                    dif&#65533;?cult discern small sample size. <br />  <br />                                                                                                    Essential PPIs Explain Centrality-Lethality Rule <br />                                                                                                       Our analysis yeast PPI network suggests the <br />                                                                                                    centrality-lethality rule simple fact highly <br />                                                                                                    connected nodes involved PPIs poorly <br />                                                                                                    connected nodes, having greater probabilities engag- <br />                                                                                                    ing essential PPIs. One Equation 1 PE is <br />                                                                                                    determined factors. One protein <br />                                                                                                    connectivity, arising solely essential PPIs, the <br />                                                                                                    factor independent protein connectivity. The <br />                                                                                                    success equation describing empirical observa- <br />                                                                                                    tions (Figure 2) congruence estimates and <br />                                                                                                    b obtained different approaches suggest factors <br />                                                                                                    dependent protein interactions, unrelated essential <br />                                                                                                    PPIs, trivial, implying gene essentiality unlikely due <br />                                                                                                    cumulative pleiotropic effects PPI level. <br />                                                                                                    Furthermore, suggest structural features <br />                                                                                                    PPI network, protein connectivity sole <br />                                                                                                    determinant protein essentiality, determi- <br />                                                                                                    nation essential PPIs. These results argue the <br />                                                                                                    hypothesis centrality-lethality rule attributable to <br />                                                                                                    relative importance hub proteins maintenance <br />                                                                                                    network architecture [6,7,9,10]. In support our <br />                                                                                                    hypothesis, node centrality, measured betweenness or <br />                                                                                                    closeness, higher essential nodes for <br />                                                                                                    nonessential nodes yeast PPI network, control <br /> Figure 2. The Relationship Probability That Protein Is <br />                                                                                                    node connectivity (Tables 2 3). Here, betweenness a <br /> Essential (PE) Connectivity (k) Protein <br />                                                                                                    node proportion shortest paths pairs of <br /> (A) Observed predicted PE values. The observed values were <br /> estimated yeast PPI network predicted values                                 reachable nodes node, closeness <br /> computed using Equation 1 parameters &#188; 2.92% b &#188; 12.6%.                                 node mean shortest path length node <br /> Error bars standard (sampling) error observed values.                              reachable nodes network. Both betweenness <br /> (B) Linear regression ln(1-PE) k. Using Equation 2, we <br /> estimated regression &#188; 3.29% b &#188; 12.8%. The 95% <br />                                                                                                    closeness measure centrality node global <br /> confidence interval 2.23%&#8211;4.35%. The 95% confidence                               network structure. Further support hypothesis comes <br /> interval b 6.7%&#8211;18.6%. Proteins k . 10 (; 5%                            recent analysis yeast PPI network, hubs <br /> proteins) considered small sample sizes.                                        classi&#65533;?ed types according coexpression <br /> DOI: 10.1371/journal.pgen.0020088.g002 <br />  <br />  <br /> Table 1. Conservation Essential PPIs Yeast Fruit Fly <br />  <br /> Types Interactions           Number Yeast Protein Interactions                   Number Conserved Interactions                  Proportion Conserved        p-Valuec <br />                                 Conserved Proteinsa                              Two Speciesb                          Interactions <br />  <br /> PPIs                            1,066                                                  46                                                0.043 <br /> IBEPsd                            340                                                  26                                                0.076                          0.0006 <br /> Probably essential PPIse           19                                                   5                                                0.263                          0.0023 <br />  <br />  <br /> a <br />  We identified 1,764 orthologous gene pairs yeast fruit fly genes form 1,066 PPIs yeast. <br /> b <br />   The proteins forming yeast PPI orthologs fruit fly, fruit fly proteins form PPI. <br /> c <br />  Fisher&#8217;s exact test hypothesis proportion conserved interactions identical row. <br /> d <br />   Interactions essential proteins. 27% IBEPs essential. In comparison, 3% PPIs essential. <br /> e <br />   These IBEPs. In addition, proteins forming interaction interacting proteins essential. 65% PPIs essential. <br /> DOI: 10.1371/journal.pgen.0020088.t001 <br />  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                                0829                                            June 2006 | Volume 2 | Issue 6 | e88 <br />                                                                                                                                               Essential Protein&#8211;Protein Interactions <br />  <br />  <br />  <br /> Table 2. Normalized Betweenness Essential Nonessential Proteins Yeast PPI Network <br />  <br /> Connectivity (k)               Essential Proteins                                                      Nonessential Proteins <br />                                Number Proteins                  Mean Betweenness                    Number Proteins                   Mean Betweenness                    p-Valuea <br />  <br />  1                             243                                 0.000000                            1,383                                0.000000                            1.0000 <br />  2                             170                                 0.000210                              784                                0.000264                            0.0080 <br />  3                             119                                 0.000513                              401                                0.000545                            0.3120 <br />  4                              76                                 0.000762                              238                                0.000810                            0.4725 <br />  5                              51                                 0.000914                              118                                0.000996                            0.1016 <br />  6                              29                                 0.001316                               87                                0.001204                            0.6997 <br />  7                              24                                 0.001345                               54                                0.001346                            0.7867 <br />  8                              18                                 0.001380                               46                                0.002016                            0.1393 <br />  9                              16                                 0.001287                               25                                0.002157                            0.0283 <br /> 10                              18                                 0.001329                               29                                0.002540                            0.0073 <br /> . 10                            72                                 0.009365                              125                                0.009740                            0.9772 <br /> Total                          836                                 0.001214                            3,290                                0.000715 <br />  <br />  <br /> a <br />  The p-value computed two-tailed Mann-Whitney test. When proteins considered, partial rank correlation protein essentiality betweenness control <br /> for connectivity  0.089 (p , 10 7). This result indicates centrality measured betweenness higher nonessential proteins essential proteins control of <br /> connectivity. <br /> DOI: 10.1371/journal.pgen.0020088.t002 <br />  <br />  <br /> patterns interacting proteins [21]. It                                    loss edge. Thus, evidence essential <br /> although removing type hub increases network                                          PPIs important nonessential PPIs maintain- <br /> diameter removing type, types                                        ing network architecture. <br /> have similar essentiality [21,22]. <br />    One argue essentiality PPI                                  The Yeast PPI Network Is Functionally More Robust than <br /> its special location network removing                                         Random Networks <br /> essential PPI disturb network architecture                                        It said scale-free networks robust against <br /> removing nonessential PPI. Unfortunately, unknown                                        random removals nodes, majority nodes are <br /> with certainty PPIs essential yeast network.                                    poorly connected, play relatively unimportant roles <br /> Because IBEPs essential, removing IBEPs                                           organizing global network structure [3]. Since PPI <br /> expected increase network diameter                                              networks factor determining protein essentiality is <br /> removing non-IBEPs, essential PPIs important                                         essential PPIs, possible examine PPI network is <br /> than nonessential PPIs maintaining network architec-                                      structured particularly robust fashion. Based the <br /> ture. However, trend (Figure 3A). Moreover,                                     estimates network rewiring linear <br /> removing IBEPs generates fewer unreachable pairs nodes                                        regression, assume 220 edges (3% edges) the <br /> than removing non-IBEPs (Figure 3B). This probably                                            yeast PPI network essential. If randomly assign 220 <br /> because IBEPs tend occur highly connected                                             essential edges yeast PPI network, average 368 nodes <br /> nodes, affected lowly connected nodes                                        essential (Figure 4A). If connectivity distribution <br />  <br />  <br /> Table 3. Normalized Closeness Essential Nonessential Proteins Yeast PPI Network <br />  <br /> Connectivity (k)                 Essential Proteins                                                     Nonessential Proteins <br />                                  Number Proteins                    Mean Closeness                   Number Proteins                     Mean Closeness                   p-Valuea <br />  <br />  1                               243                                   0.145943                         1,383                                  0.157117                          0.0260 <br />  2                               170                                   0.180588                           784                                  0.192916                         ,0.0001 <br />  3                               119                                   0.192545                           401                                  0.202836                          0.0025 <br />  4                                76                                   0.200897                           238                                  0.207470                          0.0093 <br />  5                                51                                   0.208483                           118                                  0.212829                          0.1196 <br />  6                                29                                   0.218147                            87                                  0.215018                          0.2344 <br />  7                                24                                   0.211802                            54                                  0.216865                          0.2358 <br />  8                                18                                   0.211058                            46                                  0.224958                          0.0006 <br />  9                                16                                   0.216784                            25                                  0.228474                          0.1147 <br /> 10                                18                                   0.211258                            29                                  0.227951                          0.0019 <br /> .10                               72                                   0.236238                           125                                  0.236972                          0.8113 <br /> Total                            836                                   0.184768                         3,290                                  0.184522 <br />  <br />  <br /> a <br />  The p-value computed two-tailed Mann-Whitney test. When proteins considered, partial rank correlation protein essentiality closeness control for <br /> connectivity  0.093 (p , 10 8). This result indicates centrality measured closeness higher nonessential proteins essential proteins control of <br /> connectivity. <br /> DOI: 10.1371/journal.pgen.0020088.t003 <br />  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                              0830                                               June 2006 | Volume 2 | Issue 6 | e88 <br />                                                                                                                      Essential Protein&#8211;Protein Interactions <br />  <br />  <br />  <br />  <br />                                                                                  Figure 4. Robustness PPI Networks <br /> Figure 3. Effects Random Removal Edges Global Structure          (A) Numbers essential nodes generated 220 essential edges in <br /> the Yeast PPI Network                                                            various networks. Black gray bars depict distribution the <br />                                                                                  number essential nodes 10,000 replications random assign- <br /> (A) Effects network diameter, mean shortest path length          ments 220 essential edges real yeast PPI network simulated <br /> among reachable pairs nodes network.                               ER networks, respectively. An ER network number nodes <br /> (B) Effects proportion unreachable pairs nodes               edges real network, distribution node <br /> network. Note total number IBEPs 807 network.              connectivity follows Poisson distribution. Also shown minimal <br /> DOI: 10.1371/journal.pgen.0020088.g003                                           maximal numbers essential nodes produced 220 essential <br />                                                                                  edges possible network numbers nodes and <br />                                                                                  edges yeast PPI network. The minimum 22, number <br /> does follow power-law scale-free networks,                     edges 22 nodes high 21 3 22/2 &#188; 231 . 220. The <br /> follows Poisson distribution Erdo&#168;s-Re&#180;nyi (ER)                        maximum 220 3 2 &#188; 440. <br />                                                                                  (B) Proportions essential nodes generated given numbers of <br /> random networks [23], average 417 essential nodes                       essential edges scale-free (power-law) ER networks. Both networks <br /> result 220 essential edges (Figure 4A). In fact,                        contain 4,000 nodes 4,352 edges. The scale-free network its <br /> expected number essential nodes generated given                          node connectivity following power-law distribution P(k) } k  c, where <br />                                                                                  P(k) probability node k edges. We used c &#188; 2.29, the <br /> number essential edges lower scale-free                          real yeast PPI network (see Figure S4). The ER network has <br /> networks ER networks (Figure 4B). This suggest                       connectivity distribution following Poisson distribution mean <br /> that scale-free network robust ER                           connectivity node 2.176. The result essential nodes <br />                                                                                  produced ER networks scale-free networks given <br /> network, consider underlying mechanism                       number essential edges applies c values (see Figure S5). <br /> node essentiality. Note interpretation                         DOI: 10.1371/journal.pgen.0020088.g004 <br /> network robustness different previous analyses. In <br /> previous investigations, robustness measured terms of <br /> network structure [3], measured network                        ness [6]. More likely, robustness emerged byproduct of <br /> function. We caution higher robustness scale-                    evolutionary processes contingencies. Furthermore, <br /> free yeast PPI network ER networks does imply                      interesting note yeast PPI network far from <br /> the robustness originated natural selection robust-                     robust network possible. For instance, design <br />  <br />       PLoS Genetics | www.plosgenetics.org                                0831                                      June 2006 | Volume 2 | Issue 6 | e88 <br />                                                                                                                Essential Protein&#8211;Protein Interactions <br />  <br />  <br /> a network 220 essential edges link 22 essential nodes              average 1.01 essential partners, signi&#65533;?cantly the <br /> (Figure 4A). Obviously, evolution did work way.                 expected number (0.94) estimated randomly rewired <br />                                                                             networks (p &#188; 0.002). This analysis suggests essential PPIs <br /> Caveats                                                                     potentially contribute higher essentiality singletons <br />    Our analysis based PPI data Comprehensive               duplicates, supporting view singleton genes are <br /> Yeast Genome Database [13]. To examine results                  intrinsically important duplicate genes [29]. <br /> are similar different yeast PPI datasets used, we <br /> tried datasets, nodes                      Implications <br /> edges [24] fewer nodes edges                       In biological networks networks, <br /> [21]. We using simulated networks using linear               different edges different levels importance. <br /> regression gave similar estimates b given                    Treating edges quantitatively qualitatively <br /> dataset, different datasets provided different esti-               different way reveal previously unknown patterns and <br /> mates (Figures S1 S2). These results unexpected,                provide new insights. In work, propose concept of <br /> given datasets used vary greatly                   essential protein interactions demonstrate computa- <br /> numbers nodes edges, mean connectivity,                          tional network analysis large faction gene <br /> proportion essential nodes. These variations re&#64258;ect                      essentiality essential PPIs. It important stress <br /> different numbers false-negative false-positive data                 using essential PPIs explain gene essentiality not <br /> about protein essentiality PPI different datasets.                tautological, explanation provides molecular <br /> The noise incompleteness data potentially                  understanding certain genes essential offers a <br /> undermine ability predict PE. However, long                    conceptual framework future experimental proofs. <br /> essential PPIs randomly distributed edges                 Logically, question essential PPIs essential. <br /> other essentiality-causing factors affect nodes equally            We essential PPIs likely occupy <br /> random fashion, Equation 1 work. In fact,                    central locations PPI network nonessential PPIs. <br /> congruence estimates b simulated                  Thus, essentiality PPI does determined <br /> networks regression analysis datasets              network structures particular functions <br /> strongly suggest explanation cause protein               interaction. Alternatively, in&#64258;uence network <br /> essentiality largely correct. Under assumption                  architecture subtle require further <br /> false-negative false-positive PPIs randomly distrib-                scrutiny larger accurate PPI data. Similarly, our <br /> uted network, false-negative PPIs affect a,                   results suggest simpler explanation centrality- <br /> because essential nonessential PPIs affected                 lethality rule does invoke role protein hubs <br /> same extent. On contrary, false-positive PPIs lead                organizing global network structure. Furthermore, our <br /> underestimation a, number essential PPIs               hypothesis quantitatively explains centrality-lethality <br /> not affected, total number PPIs in&#64258;ated. Both              rule, network architecture hypothesis lacks such <br /> these predictions con&#65533;?rmed simulation 50%                   quantitative model. Our &#65533;?nding appears argue against <br /> of yeast PPIs randomly removed added. These                         biological signi&#65533;?cance PPI network architecture. <br /> &#65533;?ndings suggest estimated dataset                      However, pointed gene <br /> minimal false-positive PPIs [21] accurate.                      essentiality important phenomenon deter- <br /> Nonetheless, dataset contains fewer nodes                mines organismal survival reproduction, signi&#65533;?cance <br /> other datasets estimated                         network architecture lie aspects the <br /> applicable subset nodes. A recent study                  cellular life explored. Furthermore, our <br /> pure high-throughput yeast two-<span id='am-62' about='obo:NCBITaxon_37965' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-63' property="rdfs:label" content="hybrid" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-66'  ></span>hybrid</span> data PPIs showed                 analysis focuses PPI networks, unclear whether <br /> weaker centrality-lethality relationship previously              results extend biomolecular networks. There- <br /> from better corroborated data [25]. This result expected                 fore, role network architecture biology and <br /> because pure high-throughput yeast two-hybrid data                      dismissed time. Rather, studies are <br /> contain high proportions false-positive PPIs, resulting             needed nascent &#65533;?eld systems biology address such <br /> lower (e.g, 1.2% Ito et al.&#8217;s data [26]) consequently           important questions biological meaning evolu- <br /> weaker in&#64258;uence k PE (see Equation 1).                                tionary origin architecture robustness biological <br />    It known singleton genes likely              networks [7,30&#8211;32]. <br /> essential duplicate genes [4,27,28]. It interesting ask <br /> whether singletons likely duplicates engage                Materials Methods <br /> in essential interactions. However, singletons and <br />                                                                                The yeast PPI data downloaded ftp://ftpmips.gsf.de/yeast/ <br /> duplicates form separate PPI networks,                     PPI. Although self- interactions contain important biological <br /> impossible estimate separate values them. Further-                 information, considered analysis, mainly because <br /> more, potential functional compensations duplicates                 approach using IBEPs infer essential interactions not <br />                                                                             work self-interactions. Because centrality-lethality rule is <br /> could mask true essentiality duplicate gene. That is,              observed self-interactions excluded, analysis still <br /> many nonessential duplicate genes actually essential               biologically meaningful. We excluded analysis 43 <br /> PPIs. To avoid problems, classify genes single-               interactions involving Ty elements involving mitochondrial <br /> tons duplicates examine interaction partners,                 genes, resulting 7,356 non-redundant PPIs linking 4,126 yeast <br />                                                                             nuclear genes, 836 genes essential. The mean <br /> while ignoring essentiality genes themselves. We               connectivity protein 3.57. Yeast genes subject to <br /> found yeast duplicate genes average 0.89                       single-gene deletion studies listed in: http://www-deletion. <br /> essential partners, signi&#65533;?cantly fewer expected                    stanford.edu/YDPM. Essential genes listed in: http://www. <br />                                                                             sequence.stanford.edu/group/yeast_deletion_project/ <br /> number (0.94) estimated 5,000 randomly rewired                         Essential_ORFs.txt. There 162 genes protein network <br /> networks (p &#188; 0.004). On contrary, yeast singletons                lacked essentiality information treated as <br />  <br />       PLoS Genetics | www.plosgenetics.org                           0832                                     June 2006 | Volume 2 | Issue 6 | e88 <br />                                                                                                                              Essential Protein&#8211;Protein Interactions <br />  <br /> nonessential analyses. This strategy rendered ;                      Figure S2. Relationship Probability That Protein Is <br /> 0.8% genes network misclassi&#65533;?ed terms gene                         Essential (PE) Connectivity (k) Protein Yeast PPI <br /> essentiality. All results virtually identical 162               Network <br /> genes excluded protein network. Essential genes are <br />                                                                                        The yeast PPI information compiled Han colleagues [21] was <br /> those indispensable growth yeasts YPD-rich media. <br />                                                                                        downloaded from: http://www.nature.com/nature/journal/v430/n6995/ <br /> This set genes apparently fundamental cellular processes <br /> of yeast, additional genes indispensable                    suppinfo/nature02555.html. There 2.493 interactions among <br /> adverse conditions [33]. Yeast stable protein complex dataset                      1,379 genes (including 530 essential genes). (A) Observed and <br /> downloaded <span id='am-23' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-24' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span><span id='am-27'  ></span><span id='am-28' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Saccharomyces</span> Genome Database (ftp://genome-                           predicted PE values. The observed values estimated the <br /> ftp.stanford.edu/pub/yeast/data_download/literature_curation/                          yeast PPI network. Error bars standard (sampling) error of <br /> go_protein_complex_slim.tab), contained 188 complexes                            observed values. The predicted values computed using <br /> comprising 1,226 genes.                                                                Equation 1 parameters &#188; 7.4% 6 0.5 % b &#188; 21.8% 6 1.4 <br />    Singleton genes duplicate genes de&#65533;?ned all-against-all                  %, estimated using rewired essentiality-reassigned <br /> BLASTP searches yeast proteins, following [34]. Speci&#65533;?cally, gene                 networks described main text (10,000 replications). (B) <br /> was considered singleton non-self hits E-value &#188;                 Linear regression ln(1-PE) k. We estimated the <br /> 0.1. A gene considered duplicate non-self                 regression Equation 2 parameters &#188; 7.3% b &#188; 24.9%. <br /> hit E-value &#188; 10 20.                                                                Because paucity proteins high connectivity, with <br />    The fruit &#64258;y PPI network [35] included 4,579 proteins connected                  seven edges considered counted 6.5 <br /> 4,663 non-self high-con&#65533;?dence interactions. We conducted genome-                     edges, edges considered together <br /> wide all-against-all BLASTP search (E-value cutoff &#188; 10 10)                    8.5 edges. Proteins   10 edges (; 8% proteins) not <br /> 5,773 yeast 13,434 fruit &#64258;y proteins, downloaded                   considered paucity data k. <br /> Saccharomyces Genome Database (http://www.yeastgenome.org)                         Found DOI: 10.1371/journal.pgen.0020088.sg002 (11 KB PDF). <br /> ENSEMBL (http://www.ensembl.org), respectively. 1,764 reciprocal <br /> best hits found, considered orthologous proteins                 Figure S3. Similarity Node Connectivity Yeast PPI <br /> between species. To control fact essential genes                  Network Simulated Networks (A) Essential (B) Non- <br /> tend evolutionarily conserved, examined yeast PPIs                 essential Nodes <br /> for partners orthologs fruit &#64258;y. The 1,764                To construct simulated networks, removed node <br /> proteins form 1,066 PPIs yeast 156 PPIs fruit &#64258;y.                    essentiality information real network reassigned <br />    Network parameters diameter, closeness, betwe-                      node essentiality two-step random fashion (see main text). The <br /> enness calculated using software Pajek, down-                        mean frequencies shown 10,000 simulated networks. <br /> loaded from: http://vlado.fmf.uni-lj.si/pub/networks/pajek. The node <br /> connectivity yeast PPI network approximated                         Found DOI: 10.1371/journal.pgen.0020088.sg003 (10 KB PDF). <br /> power-law distribution parameter c &#188; 2.29 (Figure S4). To <br /> simulate scale-free (power-law) network parameter c (for                        Figure S4. Frequency Distribution Connectivity (k) Protein in <br /> Figure 4B), &#65533;?rst computed P(k), expected frequency nodes                     Yeast PPI Network Follows Power-Law P(k) } k  2.29 <br /> with k edges (k &#188; 1, 2, 3, . . .), using P(k) &#188; ak  c, constant           The network contains 4,126 nodes connected 7,356 edges. <br /> determined                                                                          Found DOI: 10.1371/journal.pgen.0020088.sg004 (41 KB PDF). <br />                                        X&#8216; <br />                                &#188; 1=      P&#240;k&#222;:                         &#240;4&#222;           Figure S5. Proportions Essential Nodes Generated Given <br />                                      k&#188;1                                               Numbers Essential Edges Scale-Free ER Networks <br /> We decided connectivity of 4,000 nodes                     The scale-free network node connectivity following the <br /> network following P(k) distribution randomly paired                      power-law distribution, ER network node connectiv- <br /> the nodes considering connectivity. When generating                         ity following Poisson distribution. (A) Comparison the <br /> corresponding ER network, randomly paired 4,000 nodes                     power-law network c &#188; 2 ER network. Both networks <br /> the total number edges reached corresponding scale-                     contain 4,000 nodes 5,995 edges randomly generated <br /> free network.                                                                          following respective connectivity distributions. (B) Comparison <br />                                                                                        power-law network c &#188; 2.5 ER network. <br />                                                                                        Both networks 4,000 nodes 3,620 edges. <br /> Supporting Information                                                                 Found DOI: 10.1371/journal.pgen.0020088.sg005 (130 KB PDF). <br /> Figure S1. Relationship Probability That Protein Is <br /> Essential (PE) Connectivity (k) Protein Yeast PPI                Table S1. Probably Essential PPIs Yeast <br /> Network                                                                                Found DOI: 10.1371/journal.pgen.0020088.st001 (53 KB PDF). <br /> The yeast PPI information downloaded GRID (General                            Table S2. Conserved PPIs Yeast Fruit Fly <br /> Repository Interaction Datasets) [24] http://biodata.mshri.on.ca/ <br /> yeast_grid/&#65533;?les/Full_Data_Files/interactions.txt. After excluding                      Found DOI: 10.1371/journal.pgen.0020088.st002 (70 KB PDF). <br /> self-interactions interactions involving Ty elements mito- <br /> chondrial genes, total 13,189 physical PPIs connecting 4,674 <br /> genes (including 972 essential genes) obtained. (A) Observed                  Acknowledgments <br /> predicted PE values. The observed values estimated the <br /> yeast PPI network. Error bars standard (sampling) error                    We thank members Zhang lab discussions Wendy <span id='am-37' about='obo:NCBITaxon_9114' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_9108 obo:NCBITaxon_2759 obo:NCBITaxon_9109 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-38'  ></span><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Tetrapteryx" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasAlternativeId" content="NCBITaxon:9110" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="Grus" datatype="xsd:string"></span>Grus</span>, <br /> the observed values. The predic <br /> </body></html>