<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Kei1: A Novel Subunit of Inositolphosphorylceramide <br /> Synthase, Essential for Its Enzyme Activity and Golgi <br /> Localization <br />  <br />  <br /> Fungal sphingolipids have inositol-phosphate head groups, which are essential for the viability of cells. These head <br /> groups are added by inositol phosphorylceramide (IPC) synthase, and AUR1 has been thought to encode <span id='am-72' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-73' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-76'  ></span>this</span> enzyme. <br /> Here, we show that an essential protein encoded by KEI1 is a novel subunit of IPC synthase of <span id='am-21' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-22' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-23'  ></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>. <br /> We find that Kei1 is localized in the medial-Golgi and that Kei1 is cleaved by Kex2, a late Golgi processing endopeptidase; <br /> therefore, it recycles between the medial- and late Golgi compartments. The growth defect of kei1-1, a temperature- <br /> sensitive mutant, is effectively suppressed by the overexpression of AUR1, and Aur1 and Kei1 proteins form a complex <br /> in vivo. The kei1-1 mutant is hypersensitive to aureobasidin A, a specific inhibitor of IPC synthesis, and the IPC synthase <br /> activity in the mutant membranes is thermolabile. A part of Aur1 is missorted to the vacuole in kei1-1 cells. We show that <br /> the amino acid substitution in kei1-1 causes release of Kei1 during immunoprecipitation of Aur1 and that Aur1 without <br /> Kei1 has hardly detectable IPC synthase activity. From these results, we conclude that Kei1 is essential for both the activity <br /> and the Golgi localization of IPC synthase. <br />  <br />  <br /> INTRODUCTION                                                           suggested that certain proteins are assembled in the lipid <br />                                                                        rafts and fulfill specific and essential functions such as signal <br /> The budding yeast <span id='am-38' about='obo:NCBITaxon_4930' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-39' property="rdfs:label" content="Saccharomyces" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasRelatedSynonym" content="Pachytichospora" datatype="xsd:string"></span><span id='am-42'  ></span><span id='am-43' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasAlternativeId" content="NCBITaxon:36915" datatype="xsd:string"></span>Saccharomyces</span> cerevisiae, a unicellular eu-          transduction (Simons and Toomre, 2000; Allen et al., 2007). <br /> karyote widely used as a model organism in cell biology, has           The existence of lipid raft is still controversial (Munro, 2003), <br /> three major types of membrane lipids. The most abundant                but sphingolipids are important and interesting irrespective <br /> are glycerophospholipids that have glycerol backbones. The             of whether or not lipid rafts really exist. <br /> others are sterols and sphingolipids. Ergosterol is specific for          In S. cerevisiae, sphingolipids are composed of inositol- <br /> <span id='am-45' about='obo:NCBITaxon_4751' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2759'><span id='am-46'  ></span><span id='am-47' property="rdfs:label" content="Fungi" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="fungi" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="PMID:11062127" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="PMID:12684019" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasExactSynonym" content="fungi" datatype="xsd:string"></span>fungi</span> and corresponds to cholesterol in mammals (Holthuis              phosphorylceramide (IPC) and its mannosylated deriva- <br /> et al., 2001). Sphingolipids are phospholipids derived from            tives, namely, mannose inositolphosphorylceramide (MIPC) <br /> ceramides, and the yeast sphingolipids are characterized by            and mannose di(inositolphosphoryl)ceramide [M(IP)2C] <br /> modification with inositol-phosphate (IP). The composition             (Figure 1). There are five types of ceramide backbones with <br /> of membrane lipids changes along with the secretory path-              different positions and number of hydroxide groups in these <br /> way in S. cerevisiae; e.g., sphingolipids and ergosterol are           IP-containing sphingolipids (Funato et al., 2002). De novo <br /> more abundant in the plasma membrane than in other mem-                synthesis of ceramide is carried out by ceramide synthase <br /> branes (Hechtberger et al., 1994). The deficiency of sphingo-          composed of Lag1, Lac1, and Lip1 (Guillas et al., 2001; <br /> lipid biosynthesis results in the instability of a subset of           Schorling et al., 2001; Vallee and Riezman, 2005). The con- <br /> proteins localized to the plasma membrane (Lauwers et al.,             version of ceramide to IPC is catalyzed by IPC synthase that <br /> 2007). It is proposed that sphingolipids and ergosterols form          transfers IP from phosphatidylinositol to ceramide. AUR1 is <br /> specific domains called &#8220;lipid rafts&#8221; that are distinct from           the essential gene responsible for this enzymatic activity <br /> glycerophospholipid-rich regions both structurally and                 (Nagiec et al., 1997; Dickson and Lester, 1999; Levine et al., <br /> functionally in the plasma membrane (Edidin, 2003). It is              2000; Funato et al., 2002). Aur1 is localized to the medial- <br />                                                                        Golgi compartment, indicating that modifications of cer- <br />                                                                        amide occur after it is transported to the Golgi (Levine et al., <br /> This article was published online ahead of print in MBC in Press       2000). Conversion of IPC to MIPC is suggested to be exe- <br /> (http://www.molbiolcell.org/cgi/doi/10.1091/mbc.E09 &#8211; 03&#8211; 0235)        cuted by the Csg1&#8211;Csg2 or Csg1&#8211;Csh1 complex (Uemura et <br /> on September 2, 2009.                                                  al., 2003), and subsequent conversion of MIPC to M(IP)2C is <br /> * Present address: Faculty of Life Sciences, University of Manches-    catalyzed by Ipt1 (Dickson et al., 1997; Uemura et al., 2003). <br /> ter, The Michael Smith Building, Oxford Road, Manchester M13           In this pathway in which ceramides change to complex <br /> 9PT, United <span id='am-131' about='obo:NCBITaxon_kingdom' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-132' property="rdfs:label" content="kingdom" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Kingdom</span>.                                                   sphingolipids, only the first step, i.e., conversion of ceramide <br /> Address correspondence to: Koji <span id='am-108' about='obo:NCBITaxon_1291067' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_1118802 obo:NCBITaxon_131567 obo:NCBITaxon_10219 obo:NCBITaxon_10220 obo:NCBITaxon_2759 obo:NCBITaxon_6072'><span id='am-109' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-111' property="rdfs:label" content="Yoda" datatype="xsd:string"></span>Yoda</span> (asdfg@mail.ecc.u-tokyo.ac.jp).   to IPC, is essential for the viability of the cell. Because it is <br />                                                                        thought that both the accumulation of ceramides and the <br /> Abbreviations used: CHAPS, 3-[(3-cholamidopropyl)dimethylammo- <br /> nio]propanesulfonate; ER, endoplasmic reticulum; GDP, guanosine <br />                                                                        loss of sphingolipids exert fatal effects on yeast cells, it seems <br /> diphosphate; GFP, green fluorescent protein; GST, glutathione trans-   obvious that the conversion of ceramide into IPC is strin- <br /> ferase; HA, hemagglutinin; IPC, inositolphosphorylceramide; MIPC,      gently regulated. To understand the regulation of IPC syn- <br /> mannose inositolphosphorylceramide; M(IP)2C, mannose di(inositol-      thesis, detailed characterization of the enzyme responsible is <br /> phosphoryl)ceramide; TLC, thin layer chromatography.                   crucial. <br />  <br /> 4444                                                                                    &#169; 2009 by The American Society for Cell Biology <br />                                                                                                          Novel Subunit of Yeast IPC Synthase <br />  <br />  <br />                                                                      show that the product of YDR367w is a novel component of <br />                                                                      IPC synthase. It interacts with Aur1 in vivo, is essential for <br />                                                                      the enzymatic activity that transfers IP to ceramide, and it is <br />                                                                      also involved in the localization of IPC synthase to the <br />                                                                      medial-Golgi. We also found that it undergoes cleavage by <br />                                                                      Kex2, a late Golgi endoprotease. Therefore, we have desig- <br />                                                                      nated YDR367w as Kex2-cleavable protein Essential for IPC <br />                                                                      synthesis 1 (KEI1). <br />  <br />  <br />                                                                      MATERIALS AND METHODS <br />                                                                      Strains, <span id='am-158' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-159' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-160' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-161' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-162' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>Plasmids</span>, Media, and Reagents <br />                                                                      S. cerevisiae and plasmids used in this study are listed in Table 1 and 2, <br />                                                                      respectively. The construction of the strains and plasmids are described in <br />                                                                      Supplemental <span id='am-139' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-140' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-141' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-142'  ></span><span id='am-143' property="rdfs:label" content="Data" datatype="xsd:string"></span>Data</span>. <br />                                                                        Yeast cells were grown in YPD medium (1% Bacto yeast extract [BD <br />                                                                      Biosciences, Franklin Lakes, NJ], 2% Bacto peptone [BD Biosciences], and 2% <br />                                                                      glucose] or SD medium [0.17% yeast nitrogen base without amino acids [BD <br />                                                                      Biosciences], 0.5% ammonium sulfate, 2% glucose, and appropriate supple- <br />                                                                      ments) at 30&#176;C unless other temperatures were indicated. YPD was used as a <br />                                                                      growth media unless otherwise specified. Solid media were made with 1.5% <br />                                                                      agar. Twenty micrograms per milliliter each of histidine, tryptophan, and <br />                                                                      uracil, and 0.1% 5-fluoroorotic acid (Sigma-Aldrich, St. Louis, MO) were <br />                                                                      added in SD to make 5-fluoroorotic acid (5-<span id='am-6' about='obo:NCBITaxon_1147086' typeof='owl:Thing obo:NCBITaxon_186623 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186625 obo:NCBITaxon_117570 obo:NCBITaxon_7742 obo:NCBITaxon_89593 obo:NCBITaxon_117571 obo:NCBITaxon_123369 obo:NCBITaxon_33154 obo:NCBITaxon_123367 obo:NCBITaxon_123368 obo:NCBITaxon_123365 obo:NCBITaxon_123366 obo:NCBITaxon_475176 obo:NCBITaxon_1489872 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_1489388 obo:NCBITaxon_1489875 obo:NCBITaxon_1489876 obo:NCBITaxon_1489877 obo:NCBITaxon_1489341 obo:NCBITaxon_7776 obo:NCBITaxon_7898 obo:NCBITaxon_32443 obo:NCBITaxon_41665 obo:NCBITaxon_83881'><span id='am-7' property="rdfs:label" content="Foa" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-10'  ></span>FOA</span>) plates for the Ura3&#11001; coun- <br />                                                                      terselection. Twenty micrograms per milliliter phloxine B (Sigma-Aldrich, St. <br />                                                                      Louis, MO) was added in YPD to make phloxine plates (Tsukada and Oh- <br />                                                                      sumi, 1993) for the selection of temperature-sensitive mutants. Aureobasidin <br />                                                                      A was purchased from Takara Bio (Shiga, Japan). <span id='am-85' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-86' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-98'  ></span><span id='am-99' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span>Escherichia coli</span> DH5&#9251; (F&#11002;, <br />                                                                      supE44 &#9004;lacU169 &#9278;80lacZ&#9004;M15 hsdR17 recA1 endA1 gyrA96 thi-1 relA1) was <br />                                                                      used in plasmid propagation. E. coli was grown in a <span id='am-11' about='obo:NCBITaxon_218032' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_1206795 obo:NCBITaxon_216294 obo:NCBITaxon_69557 obo:NCBITaxon_69555 obo:NCBITaxon_69556 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6448 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-12'  ></span><span id='am-13' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Luria" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Luria</span>-Bertani (LB) <br />                                                                      medium (1% Bacto tryptone [BD Biosciences], 0.5% Bacto yeast extract [BD <br />                                                                      Biosciences], and 0.5% NaCl). <br />  <br />                                                                      Antibodies <br /> Figure 1. Biosynthetic pathway of sphingolipids in S. cerevisiae.    Antisera against Scs2, Ret1, and Sec21 were kindly provided by Dr. Satoshi <br /> Precursors, products, and proteins required for the individual en-   Kagiwada (Nara Women&#8217;s University, Nara, Japan), Dr. Pierre Cosson (Basel <br /> zymatic steps are shown. DHS, dihydrosphingosine; PHS, <span id='am-163' about='obo:NCBITaxon_1262320' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43755 obo:NCBITaxon_43738 obo:NCBITaxon_43758 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_43742 obo:NCBITaxon_50557'><span id='am-164' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-165' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-166'  ></span><span id='am-167' property="rdfs:label" content="Phyto" datatype="xsd:string"></span>phyto</span>-        Institute for Immunology, Basel, Switzerland), and Dr. Rainer Duden (Uni- <br /> sphingosine; IPC, inositolphosphorylceramide.                        versity of Cambridge, Cambridge, United Kingdom), respectively. Anti-3- <br />                                                                      phosphoglycerate kinase mouse (anti-PGK, 22C5; Invitrogen, Carlsbad, CA), <br />                                                                      anti-myc mouse (9E10; Berkeley Antibody, Richmond, CA), anti-hemaggluti- <br />                                                                      nin (HA) mouse (12CA5; Roche Diagnostics, Indianapolis, IN), and anti- <br />                                                                      carboxypeptidase Y (CPY) rabbit (Rockland Immunochemicals, Gilbertsville, <br />    Sphingolipids have attracted the attention of many re-            PA) polyclonal antibody were purchased. The anti-green fluorescent protein <br /> searchers, and striking advances in this field have been             (GFP), anti-Sed5, anti-Van1, anti-Kex2, and anti-Gas1 antisera were prepared <br /> made in the past couple of decades. Initially, genetic ap-           by immunizing rabbits with the glutathione transferase (GST) fusion protein <br />                                                                      as the antigen. For Western blotting, these antisera were diluted at 1:1000. <br /> proaches such as isolating mutants, screening their suppres- <br /> sors, and searching for homologues to known genes related <br /> to sphingolipids have mainly contributed the advances; but           Construction of Temperature-sensitive Mutant Allele <br /> more recently, different approaches have discovered novel            of KEI1 <br /> proteins involved in the sphingolipid biosynthetic pathway.          The DNA fragment containing the whole KEI1 gene with the authentic <br />                                                                      promoter and terminator was amplified under an error-prone polymerase <br /> Riezman&#8217;s group found Lip1 through the purification of the           chain reaction (PCR) condition (Muhlrad et al., 1992). KSY66 was transformed <br /> ceramide synthase complex (Vallee and Riezman, 2005), and            with the mixture of the amplified fragment and the SmaI-cleaved pKS26 to <br /> Weissman&#8217;s group uncovered a previously <span id='am-173' about='obo:NCBITaxon_32644' typeof='owl:Thing obo:NCBITaxon_12908 obo:NCBITaxon_1'><span id='am-174' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-175' property="oboInOwl:hasRelatedSynonym" content="miscellaneous nucleic acid" datatype="xsd:string"></span><span id='am-176'  ></span><span id='am-177' property="oboInOwl:hasRelatedSynonym" content="not shown" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:hasRelatedSynonym" content="none" datatype="xsd:string"></span><span id='am-179' property="oboInOwl:hasRelatedSynonym" content="unidentified organism" datatype="xsd:string"></span><span id='am-180' property="oboInOwl:hasRelatedSynonym" content="unknown organism" datatype="xsd:string"></span><span id='am-181' property="oboInOwl:hasRelatedSynonym" content="other" datatype="xsd:string"></span><span id='am-182' property="oboInOwl:hasRelatedSynonym" content="unknown" datatype="xsd:string"></span><span id='am-183' property="oboInOwl:hasRelatedSynonym" content="unspecified" datatype="xsd:string"></span><span id='am-184' property="oboInOwl:hasRelatedSynonym" content="unknwon" datatype="xsd:string"></span><span id='am-185' property="oboInOwl:hasRelatedSynonym" content="not specified" datatype="xsd:string"></span><span id='am-186' property="rdfs:label" content="unidentified" datatype="xsd:string"></span><span id='am-187' property="oboInOwl:hasRelatedSynonym" content="unclassified sequence" datatype="xsd:string"></span><span id='am-188' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-189' property="oboInOwl:hasRelatedSynonym" content="sonstige nucleic acid" datatype="xsd:string"></span>unidentified</span>                 integrate fragments into the plasmid by gap repair. The His&#11001; transformants <br /> 3-hydroxyacyl-CoA dehydratase Phs1 through a large-scale             were replicated to 5-FOA plates to remove pKS23 at 26.5&#176;C and then repli- <br />                                                                      cated to phloxine B plates and incubated at 37&#176;C for 4 &#8211;12 h. Cells with <br /> analysis of essential genes whose products are located in the        reduced viability become unable to protect the entry of this dye. The cells that <br /> early secretory pathway (Denic and Weissman, 2007). These            formed pink colonies were tested for single colony formation at 26.5 and 37&#176;C. <br /> findings indicate the possibility that any essential genes           After confirming that the temperature sensitivity was due to the plasmid, the <br /> involved in sphingolipid biosynthesis remain unidentified.           nucleotide sequences of the mutant kei1 were determined. The allele kei1-1 <br />                                                                      was integrated into the chromosome of KSY66 at the &#9004;kei1::kanMX4 region by <br /> It is probably difficult to find them by conventional ap-            recombination with kei1-1::LEU2 fragment by double crossing-over at the <br /> proaches.                                                            surrounding homologous sequences. The Leu&#11001; cells were streaked on a <br />    In the S. cerevisiae genome, &#11022;200 genes encode membrane           5-FOA plate to remove pKS23, grown cells were tested for genotype, and a <br /> proteins essential for cell viability. In spite of enormous          correct strain was stocked as KSY271. <br /> efforts in genome-wide analyses, several essential mem- <br /> brane proteins remain uncharacterized. We have tried to              Subcellular Fractionation <br /> assign functions to such essential proteins that are inte-           Cells were converted to spheroplasts and burst in B88 (20 mM HEPES, pH 6.8, <br />                                                                      150 mM potassium acetate, 5 mM magnesium acetate, and 200 mM sorbitol) <br /> grated into membranes by our strategy of using tempera-              containing protease inhibitors (1 &#9262;g/ml each of chymostatin, aprotinin, leu- <br /> ture-sensitive mutants. To date, we have reported the char-          peptin, pepstatin A, antipain, and 1 mM phenylmethylsulfonyl fluoride). <br /> acterization of two previously uncharacterized endoplasmic           Unbroken cells were removed by centrifugation at 1000 &#11003; g for 5 min. For <br /> reticulum (ER) proteins Pga1 and Keg1 (Nakamata et al.,              differential fractionation, the cleared lysate was sequentially centrifuged to <br />                                                                      generate 10,000 &#11003; g &#11003; 10-min pellet (P10), 100,000 &#11003; g &#11003; 60-min pellet (P100), <br /> 2007; Sato et al., 2007). YDR367w is one of such uncharacter-        and 100,000 &#11003; g &#11003; 60-min supernatant (S100). Each fraction was adjusted to <br /> ized essential gene, encoding a 221-amino acid polypeptide           the original volume of the lysate, and the same amount was applied for <br /> with four predicted transmembrane domains. Here, we                  SDS-polyacrylamide gel electrophoresis (PAGE) and Western blotting. <br />  <br />  <br /> Vol. 20, October 15, 2009                                                                                                                      4445 <br />  K. Sato et al. <br />  <br />  <br />  <br /> Table 1. Strains used in this study <br />  <br /> Strain name                                                            Description                                                                Source <br />  <br /> KA31a                      MATa, &#9004;his3&#9004; leu2&#9004; trp1&#9004; ura3                                                                                    EUROSCARFa <br /> BY4741                     MATa, his3&#9004;1 leu2&#9004;0 met15&#9004;0 ura3&#9004;0                                                                               EUROSCARF <br /> BY4742                     MATa, his3&#9004;1 leu2&#9004;0 met15&#9004;0 ura3&#9004;0                                                                               EUROSCARF <br /> BY4743                     MATa/MAT&#9251;, his3&#9004;1/his3&#9004;1 leu2&#9004;0/leu2&#9004;0 met15&#9004;0/MET15 LYS2/lys2&#9004;0 ura3&#9004;0/ura3&#9004;0                                   EUROSCARF <br /> KSY66                      As BY4741, &#9004;kei1::kanMX4, pKS23 (CEN, URA3 KEI1)                                                                 This study <br /> KSY271                     As BY4741, &#9004;kei1::kei1-1 LEU2                                                                                    This study <br /> KSY431                     As BY4741, &#9004;kei1::kanMX4, pKS243 (CEN, HIS3 KEI1-GFP)                                                            This study <br /> KSY506                     As BY4743, &#9004;kei1::kanMX4/KEI1                                                                                    This study <br /> KSY507                     As BY4741, &#9004;kei1::kanMX4, pKS301 (CEN, HIS3 KEI1&#11002;intron-GFP)                                                     This study <br /> KSY512                     As BY4741, &#9004;aur1::AUR1-GFP HIS3                                                                                  This study <br /> KSY513                     As KSY271, &#9004;aur1::AUR1-GFP HIS3                                                                                  This study <br /> KSY525                     As BY4741, &#9004;aur1::AUR1&#8211;3HA HIS3                                                                                  This study <br /> KSY527                     As BY4741, &#9004;kei1::KEI1-GFP URA3                                                                                  This study <br /> KSY530                     As KSY271, &#9004;aur1::AUR1&#8211;3HA HIS3                                                                                  This study <br /> KSY531                     As BY4741, &#9004;aur1::AUR1&#8211;3HA HIS3, &#9004;kei1::KEI1-GFP URA3                                                            This study <br /> KSY548                     As BY4741, &#9004;kex2::kanMX4                                                                                         EUROSCARF <br /> KSY552                     As KSY271, &#9004;ubc7::kanMX4                                                                                         This study <br /> KSY553                     As KSY271, &#9004;ubc7::kanMX4                                                                                         This study <br /> KSY554                     As KSY271, &#9004;doa10::kanMX4                                                                                        This study <br /> KSY559                     As KSY271, &#9004;hrd1::kanMX4                                                                                         This study <br /> KSY560                     As KSY271, &#9004;hrd1::kanMX4                                                                                         This study <br /> KSY561                     As KSY271, &#9004;doa10::kanMX4                                                                                        This study <br /> KSY562                     As KSY271, &#9004;ubx2::kanMX4                                                                                         This study <br /> KSY563                     As KSY271, &#9004;ubx2::kanMX4                                                                                         This study <br /> KSY571                     As KSY512, &#9004;pep4::LEU2                                                                                           This study <br /> KSY572                     As KSY513, &#9004;pep4::LEU2                                                                                           This study <br /> KSY607                     As BY4741, pKS342 (CEN, URA3 kei1K141S-GFP)                                                                      This study <br /> KSY609                     As BY4741, pKS344 (CEN, URA3 kei1R135S-GFP)                                                                      This study <br /> KSY611                     As BY4741, pKS346 (CEN, URA3 kei1K131S-GFP)                                                                      This study <br /> KSY614                     As KSY548, pKS342 (CEN, URA3 kei1K141S-GFP)                                                                      This study <br /> KSY616                     As KSY548, pKS344 (CEN, URA3 kei1R135S-GFP)                                                                      This study <br /> KSY618                     As KSY548, pKS346 (CEN, URA3 kei1K131S-GFP)                                                                      This study <br /> KSY625                     As BY4741, pKS353 (CEN, URA3 KEI1-GFP)                                                                           This study <br /> KSY627                     As KSY548, pKS353 (CEN, URA3 KEI1-GFP)                                                                           This study <br /> KSY686                     As BY4741, &#9004;kei1::kanMX4, ura3-52:: KEI1-GFP, URA3                                                               This study <br /> KSY692                     As BY4741, &#9004;kei1::kanMX4, ura3-52:: kei1-1-GFP, URA3                                                             This study <br /> KSY694                     As BY4741, &#9004;kei1::kanMX4, ura3-52:: kei1&#9004;C-GFP, URA3                                                             This study <br /> KSY696                     As BY4741, &#9004;kei1::kanMX4, ura3-52:: KEI1-GFP, URA3                                                               This study <br /> KSY698                     As BY4741, &#9004;kei1::kanMX4, ura3-52:: kei1F103I-GFP, URA3                                                          This study <br /> KSY714                     As KSY527, &#9004;kre2::KRE2&#8211;3HA LEU2                                                                                  This study <br /> KSY721                     As KSY525, ura3-52::GFP-KEI1, URA3                                                                               This study <br /> KSY723                     As BY4741, ura3-52::GFP-KEI1, URA3                                                                               This study <br /> KSY725                     As KSY723, &#9004;kre2::KRE2&#8211;3HA LEU2                                                                                  This study <br /> KSY728                     As KSY692, &#9004;pep4::LEU2                                                                                           This study <br /> KSY730                     As KSY694, &#9004;pep4::LEU2                                                                                           This study <br /> KSY732                     As KSY696, &#9004;pep4::LEU2                                                                                           This study <br /> KSY753                     As BY4741, &#9004;kei1::kanMX4, ura3-52:: GFP-kei1F103I, URA3                                                          This study <br /> KSY765                     As KSY723, &#9004;pep4::LEU2                                                                                           This study <br /> KSY767                     As KSY753, &#9004;pep4::LEU2                                                                                           This study <br /> KSY771                     As BY4741, &#9004;kei1::kanMX4, pKS380 (CEN, LEU2 KEI1-GFP), &#9004;aur1::AUR1-3HA HIS3                                      This study <br /> KSY773                     As BY4741, &#9004;kei1::kanMX4, pKS381 (CEN, LEU2 kei1F103I-GFP), &#9004;aur1::AUR1-3HA HIS3                                 This study <br /> KSY775                     As BY4741, &#9004;kei1::kanMX4, pKS382 (CEN, LEU2 kei1&#9004;C-GFP), &#9004;aur1::AUR1-3HA HIS3                                    This study <br /> KSY780                     As KSY698, &#9004;pep4::LEU2                                                                                           This study <br /> KSY784                     As KSY271, &#9004;kre2::KRE2-3HA HIS3                                                                                  This study <br /> KSY790                     As BY4741, &#9004;kre2::KRE2-3HA HIS3                                                                                  This study <br /> KSY792                     As KSY696, &#9004;aur1::AUR1-3HA HIS3                                                                                  This study <br /> KSY794                     As KSY698, &#9004;aur1::AUR1-3HA HIS3                                                                                  This study <br /> KSY799                     As BY4741, &#9004;kei1, ura3-52:: GFP-kei1F103I, URA3                                                                  This study <br /> KSY801                     As KSY753, &#9004;aur1::AUR1-3HA HIS3                                                                                  This study <br /> KSY805                     As BY4741, &#9004;kei1::kei1 F103I LEU2, &#9004;aur1::AUR1-3HA HIS3                                                          This study <br /> THY4-2                     As KA31a, &#9004;kre2::KRE2-3HA LEU2                                                                                   This study <br /> YNY309                     sec12-4                                                                                                          Randy Schekman <br /> YNY401                     As &#9010;&#9001;31&#9251;, &#9004;emp24                                                                                                 This study <br /> a <br />     EUROpean Saccharomyces Cerevisiae ARchive for Functional Analysis, Frankfurt, Germany. <br />  <br />  <br />   Fractionations using 22&#8211; 60% sucrose density gradient centrifugation were      al., 1999). In brief, spheroplasts were lysed in 1 ml of ice-cold B88 containing <br /> performed basically as described previously (Becherer et al., 1996; Conchon et   protease inhibitors by incubation for 30 min at 4&#176;C and sequential 20 strokes <br />  <br />  <br /> 4446                                                                                                                         Molecular Biology of the Cell <br />                                                                                                                       Novel Subunit of Yeast IPC Synthase <br />  <br />  <br />  <br /> Table 2. Plasmids used in this study <br />  <br /> Plasmid name                                                          Description                                                               Source <br />  <br /> pKS23                                               CEN-URA3-KEI1, own promoter                                                        This study <br /> pKS26                                               Gap-repair for KEI1, CEN, HIS3                                                     This study <br /> pKS181                                              kei1-1 integration (LEU2)                                                          This study <br /> pKS243                                              CEN-HIS3-KEI1-GFP, own promoter                                                    This study <br /> pKS262                                              2&#9262;URA3-AUR1, own promoter                                                          This study <br /> pKS290                                              AUR1-GFP integration (HIS3)                                                        This study <br /> pKS293                                              AUR1-3HA integration (HIS3)                                                        This study <br /> pKS301                                              CEN, HIS3 KEI1&#11002;intron-GFP, own promoter                                            This study <br /> pKS307                                              KEI1-GFP integration (URA3)                                                        This study <br /> pKS342                                              CEN, URA3 kei1K141S-GFP, own promoter                                              This study <br /> pKS344                                              CEN, URA3 kei1R135S-GFP, own promoter                                              This study <br /> pKS346                                              CEN, URA3 kei1K131S-GFP, own promoter                                              This study <br /> pKS349                                              KEX2 disruption (LEU2)                                                             This study <br /> pKS353                                              CEN, URA3 KEI1-GFP, own promoter                                                   This study <br /> pKS360                                              PEP4 disruption (LEU2)                                                             This study <br /> pKS361                                              URA3-kei1-1-GFP                                                                    This study <br /> pKS362                                              URA3-kei1&#9004;C-GFP                                                                    This study <br /> pKS363                                              URA3-KEI1-GFP                                                                      This study <br /> pKS364                                              URA3-kei1F103I-GFP                                                                 This study <br /> pKS378                                              URA3-GFP-KEI1, YPT1 promoter                                                       This study <br /> pKS380                                              CEN-LEU2-KEI1, own promoter                                                        This study <br /> pKS381                                              CEN-LEU2-kei1F103I, own promoter                                                   This study <br /> pKS382                                              CEN-LEU2-kei1&#9004;C, own promoter                                                      This study <br /> pKS384                                              URA3-GFP-KEI1F103I, YPT1 promoter                                                  This study <br /> pKS393                                              KRE2&#8211;3HAs integration (HIS3)                                                       This study <br /> pKS394                                              kei1F103I integration (LEU2)                                                       This study <br /> pHI63                                               KEX2 disruption (URA3)                                                             Tomishige et al. (2003) <br /> pTH6-3                                              KRE2-3HA integration                                                               This study <br />  <br />  <br />  <br /> in a Dounce homogenizer. The lysate was centrifuged at 1000 &#11003; g for 5 min           for others in SDS-PAGE. Indicated proteins were detected by Western blot- <br /> to remove unlysed cells. Then, 0.65 ml of the supernatant was recovered, and        ting. <br /> the pellet was extracted again with 0.45 ml of ice-cold B88 containing protease <br /> inhibitors and centrifuged at 1000 &#11003; g for 5 min. The combined supernatant          In Vitro Assay for IPC Synthesis <br /> was centrifuged at 10,000 &#11003; g for 10 min, and 1 ml of the resultant supernatant <br />                                                                                     For membrane fractions, cells were grown at 26.5&#176;C to an OD600 nm &#11005; 1.0. Half <br /> was loaded on a sucrose step gradient, which was generated using the <br />                                                                                     of the cells were grown at 26.5&#176;C, and the other half were grown at 37&#176;C for <br /> following steps (all sucrose solutions were made [wt/wt, %], in 10 mM <br />                                                                                     2 h. Cells were collected, washed with water and B88, and suspended in <br /> HEPES-KOH, pH 7.6, 1 mM MgCl2): 1 ml 60%, 1 ml 40%, 1.2 ml 37%, 1.8 ml <br /> 34%, 2 ml 32%, 1.8 ml 29%, 1.2 ml 27%, and 1 ml 22%. After 16-h centrifuga-         ice-cold B88 containing protease inhibitors. The mixture was added with 1 g <br /> tion in an SW40Ti rotor (Beckman Coulter, Fullerton, CA) at 170,000 &#11003; g, 12         of glass beads and broken with Multi-Beads Shocker (Yasui Kikai, Osaka, <br /> fractions of 1 ml were sequentially collected from the top of the gradient.         Japan). Unbroken cells were removed by centrifugation at 1000 &#11003; g for 5 min <br /> Aliquots of each fraction were mixed with the SDS sample buffer, and pro-           at 4&#176;C, and the supernatant was centrifuged on 60% sucrose cushion at <br /> teins were resolved by SDS-PAGE and detected by immunoblotting using                100,000 &#11003; g for 30 min (TLS-55 rotor; Beckman Coulter). The membrane <br /> anti-Kex2, anti-Sed5, anti-GFP, and anti-HA antisera. Enhanced chemilumi-           fraction was suspended in ice-cold B88 containing protease inhibitors, dis- <br /> nescence signals were captured by an image analyzer equipped with a cooled          pensed to 60 &#9262;l each, frozen with liquid N2, and preserved at &#11002;80&#176;C until <br />                                                                                     being used. The membrane fraction was mixed with 60 &#9262;l of the reaction mix <br /> charge-coupled-device <span id='am-77' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-78' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-79'  ></span><span id='am-80' property="rdfs:label" content="Camera" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>camera</span> (LAS-1000plus; Fuji Film, Tokyo, Japan), and <br />                                                                                     (10 mM HEPES-KOH, pH 7.2, 1 mM EDTA, 250 mM sucrose, 10 mg/ml <br /> digital images were quantified using ImageJ software (National Institutes of <br />                                                                                     bovine serum albumin BSA [fatty acid-free; Sigma-Aldrich], 2 mM 3-[(3- <br /> Health, Bethesda, MD) and graphed on Excel (Microsoft, Redmond, WA). <br />                                                                                     cholamidopropyl)dimethylammonio]propanesulfonate [CHAPS, Dojindo, <br />    For fractionation using 18 &#8211; 60% sucrose density gradient centrifugation, <br />                                                                                     Kumamoto, Japan], 2 mM MnCl2, and 2 mM MgCl2) with or without 10 <br /> spheroplasts were lysed as described above, and 1 ml of the supernatant of <br /> 1000 &#11003; g was loaded on a sucrose step gradient that was made as described <br />                                                                                     &#9262;g/ml aureobasidin A (Takara Bio), and incubated at the indicated temper- <br />                                                                                     ature for 30 min (preincubation). Then, 0.6 &#9262;l of 1 M 6-((N-(7-nitrobenz-2- <br /> previously (Inadome et al., 2005). The quantification of the digital images and <br />                                                                                     oxa-1,3-diazol-4-yl)amino)hexanoyl)sphingosine (C6-NBD-ceramide; Invitro- <br /> creation of the graph were carried out as described above. <br />                                                                                     gen) dissolved in dimethyl sulfoxide was added to the mixture and incubated <br />                                                                                     at the indicated temperature for 1 h. The reaction was stopped by adding 400 <br />                                                                                     &#9262;l of 4:10:1 mixture of chloroform:methanol:1 M HCl. Lipids were extracted <br /> Immunoprecipitation                                                                 by sequential additions of 50 &#9262;l of 1 M HCl and 100 &#9262;l of chloroform and <br /> Cells were grown at 30&#176;C in the YPD medium to an OD600 nm &#11005; 1.0, and 50             vigorously vortexing, after which phase separation was achieved by centrif- <br /> OD units of cells were converted to spheroplasts and suspended in 1 ml of B88       ugation for 5 min at 16,100 &#11003; g. The chloroform layer was collected and dried <br /> containing 1% Triton X-100 and protease inhibitors. After rotating the mixture      by evaporation with Speed-Vac, and the resulting pellet was resuspended in <br /> for 30 min at 4&#176;C, unbroken cells were removed by centrifugation at 1000 &#11003;          15 &#9262;l of chloroform. Recovered lipids were separated on a thin layer chro- <br /> g for 5 min at 4&#176;C and then centrifuged at 16,100 &#11003; g for 10 min. The               matography (TLC) plate (Silica gel 60; Merck, White Station, NJ) in an 11:9:2 <br /> supernatant was centrifuged at 100,000 &#11003; g for 60 min. A part of the super-         mixture of chloroform:methanol:30 mM KCl. Fluorescent bands on the TLC <br /> natant was recovered as the fraction &#8220;supernatant.&#8221; The rest of the superna-        were detected using LAS-1000plus. <br /> tant was mixed with anti-HA antibody and kept gently rotating at 4&#176;C for 1 h.          For the assay of immunologically purified IPC synthase, cells were grown <br /> Protein A-Sepharose beads (Amersham Biosciences, Uppsala, Sweden)                   at 26.5&#176;C to an OD600 nm &#11005; 1.0, and 50 OD units of cells were converted to <br /> washed with B88 containing 1% Triton X-100 were added, and incubation was           spheroplasts and suspended in 1 ml of B88 containing 1% Triton X-100 and <br /> continued at 4&#176;C overnight. The mixture was centrifuged at                          protease inhibitors. After rotating the mixture for 30 min at 4&#176;C, unbroken <br /> 500 &#11003; g for 1 min, and the supernatant was recovered as the fraction &#8220;un-           cells were removed by centrifugation at 1000 &#11003; g for 5 min at 4&#176;C and then <br /> bound.&#8221; The beads were washed five times with 500 &#9262;l of B88 containing 1%           centrifuged at 16,100 &#11003; g for 10 min at 4&#176;C. The supernatant was centrifuged <br /> Triton X-100. The bound proteins were eluted twice by incubating at 4&#176;C for         at 100,000 &#11003; g for 60 min at 4&#176;C. The supernatant was mixed with washed <br /> 10 min in 50 &#9262;l of SDS sample buffer and recovered as the fraction &#8220;bound.&#8221;         protein A-Sepharose beads for 30 min at 4&#176;C, and then centrifuged at 500 &#11003; <br /> Material derived from 10-fold more cells was loaded for sample bound than           g for 1 min at 4&#176;C. The supernatant was mixed with anti-HA antibody and <br />  <br />  <br /> Vol. 20, October 15, 2009                                                                                                                                  4447 <br />  K. Sato et al. <br />  <br /> kept gently rotating at 4&#176;C for 1 h. Washed protein A-Sepharose beads were       shown). Interestingly, Kei1-GFP yielded two major bands <br /> added, and incubation was continued at 4&#176;C overnight. The mixture was            when resolved and detected by SDS-PAGE and Western <br /> centrifuged at 500 &#11003; g for 1 min, and the beads were washed twice with 1 ml <br /> of B88 containing 1% Triton X-100. The beads were further washed three           blotting at the estimated molecular masses of 36.5 and 27 <br /> times with 1 ml of B88 containing 1 mM CHAPS. The supernatant was                kDa (Figure 3A). When fractionated by the differential cen- <br /> removed and resuspended in 1 ml of B88 containing 1 mM CHAPS and                 trifugation, the 36.5-kDa polypeptide was mainly recovered <br /> dispensed into four tubes (500, 250, 2 &#11003; 125 &#9262;l). Each tube was centrifuged at   in the P10 fraction, and 27-kDa was almost evenly found in <br /> 500 &#11003; g for 1 min, and the supernatant was completely removed. Bound <br /> proteins on beads of one 125-&#9262;l dispensed tube were eluted with 1&#11003; SDS-          P10 and P100 fractions (Figure 3A). From this result, it is <br /> PAGE sample buffer to monitor the amount of bound proteins. Beads in other       likely that 36.5-kDa polypeptide is localized to the ER and <br /> tubes were resuspended in 120 &#9262;l of the reaction mix (10 mM HEPES-KOH,           27-kDa polypeptide is localized to the Golgi. Thus, there <br /> pH 7.2, 1 mM EDTA, 250 mM sucrose, 5 mg/ml fatty acid-free BSA, 1 mM             are at least two types of Kei1 polypeptides, which are <br /> CHAPS, 2 mM MnCl2, 2 mM MgCl2, 0.15 mg/ml l-&#9251;-phosphatidylinositol <br /> ammonium salt from bovine liver [Sigma-Aldrich], and 0.05% Triton X-100).        different not only in molecular mass but also in subcellu- <br /> After preincubation at 26.5&#176;C for 30 min, the reaction was started by adding     lar localization. <br /> 0.6 &#9262;l of 1 M C6-NBD-ceramide. After incubation at 26.5&#176;C for 1 h, the lipids       KEI1 contains an intron of 31 nucleotides after the predicted <br /> were extracted and analyzed as explained above.                                  start site of translation. We tested whether the intron would <br />                                                                                  result in two types of Kei1 polypeptides, but deletion of the <br /> GST Pull-Down Assay                                                              intron did not affect the pattern of Kei1-GFP on SDS-PAGE <br /> E. coli BL21 (DE3) cells transformed with the plasmid pKSEC7 (GST-Kei1CT)        (Figure 3B). This result is consistent with the analyses of <br /> or pGEX4T-3 (GST) were grown at 37&#176;C in the LB medium containing ampi- <br /> cillin (100 &#9262;g/ml), and protein production was induced by the addition of        transcripts (Miura et al., 2006) and transcription start sites <br /> isopropyl &#9252;-d-thiogalactoside (final concentration, 0.5 mM) when OD600 nm        (Zhang and Dietrich, 2005) in S. cerevisiae, which indicate <br /> reached 0.5. The cultures were grown at 37&#176;C for an additional 2 h. The cells    that there is only one transcript for KEI1. Next, we suspected <br /> were then collected by centrifugation, resuspended in the STE buffer (10 mM      that Kei1 undergoes processing after being transported to <br /> Tris-HCl, pH 8.0, 150 mM NaCl, and 1 mM EDTA) containing 100 &#9262;g/ml <br /> lysozyme, and kept on ice for 20 min. Protease inhibitors and dithiothreitol     the Golgi. Because Kex2 is a major protease in the Golgi <br /> (final concentration, 1 mM) were added to the mixture, which was then            (Fuller et al., 1989; Redding et al., 1991), we tested whether <br /> sonicated for 1 min in an ice bath with a UD-201 Ultrasonic Disruptor (Tomy,     Kex2 would process Kei1. Kei1-GFP was expressed in wild- <br /> Tokyo, Japan) with the appropriate setting (output, 4; duty cycle, 40%). The     type or &#9004;kex2 cells, and the molecular mass of Kei1-GFP was <br /> cell homogenate was centrifuged, and the supernatant was mixed with one- <br /> ninth volume of 10% Triton X-100. Glutathione-Sepharose 4B beads (GE             analyzed by SDS-PAGE. As a result, the 27-kDa band was <br /> Healthcare, Little Chalfont, Buckinghamshire, United Kingdom) were added         diminished when KEI1-GFP was expressed in &#9004;kex2 cells <br /> to the mixture and rotated gently for 2 h at 4&#176;C. Protein-bound beads were       (Figure 3C). Kex2 prefers basic residues, especially dibasic <br /> washed three times in ice-cold phosphate-buffered saline. The yeast cytosol      sequences, as its cleavage site (Rockwell and Fuller, 1998). <br /> prepared from BY4741 was mixed with Triton X-100 (final concentration, 1%), <br /> added to the beads containing purified GST-Kei1CT, and gently rotated for        As it is predicted from the apparent molecular masses of the <br /> 2 h at 4&#176;C. After extensive washing, bound proteins were eluted by boiling the   two <span id='am-82' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-83' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-84' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> of Kei1-GFP that the cleavage occurs between <br /> beads in the SDS sample buffer and subjected to SDS-PAGE. The released           the third and fourth predicted transmembrane segments, we <br /> proteins were detected by Western blotting.                                      replaced each of the three basic residues in this region <br />                                                                                  (131Lys, 135Arg, and 141Lys) by Ser. The modified Kei1-GFP <br /> Indirect Immunofluorescence <br />                                                                                  constructs were produced in wild-type and &#9004;kex2 cells. As <br /> The localization of Aur1-3HA and Kre2-3HA was observed by the indirect <br /> immunofluorescence as described previously (Sato et al., 2007) except that <br />                                                                                  shown in Figure 3C, only the R135S substitution resulted in <br /> anti-HA antibody and <span id='am-1' about='obo:NCBITaxon_149625' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_163739 obo:NCBITaxon_3814 obo:NCBITaxon_1 obo:NCBITaxon_72025 obo:NCBITaxon_131567 obo:NCBITaxon_3803 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_91835 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71275 obo:NCBITaxon_3193'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Alexa" datatype="xsd:string"></span><span id='am-5'  ></span>Alexa</span> 568-conjugated goat antibody to mouse immu-           the disappearance of the 27-kDa Kei1-GFP band in the wild- <br /> noglobulin G were used as the primary antibody and the secondary antibody,       type cells, indicating that Kex2 recognizes this 135Arg resi- <br /> respectively.                                                                    due to cleave Kei1. From these results, we propose that <br />                                                                                  Kei1 is transported to the late Golgi and subsequently <br /> Invertase Detection                                                              processed by Kex2. We will call the full-length Kei1 as <br /> Secretory invertase was recovered and analyzed as described previously (Sato     Kei1F, and the N- and C-terminal fragments as Kei1N and <br /> et al., 2007). <br />                                                                                  Kei1C, respectively. <br />                                                                                     Next, we carried out a solubility test of Kei1-GFP and <br /> RESULTS                                                                          found that both Kei1F-GFP and Kei1C-GFP were found in <br />                                                                                  the precipitate after 100,000 &#11003; g centrifugation even when <br /> Kei1/Ydr367w Is Conserved among Fungi                                            the cell lysate was treated with 1 M sodium chloride, 0.1 M <br /> According to Saccharomyces Genome Database, KEI1/                                sodium carbonate, pH 11.0, or 2 M urea. However, upon <br /> YDR367w encodes a 221-amino acid polypeptide of molec-                           treatment with 1% Triton X-100, both proteins were found in <br /> ular weight 25,484 and pI 8.06, the biochemical function of                      the supernatant (Supplemental Figure S1A). In addition, <br /> which remains unknown. Homologous sequences were                                 GFP-Kei1F and GFP-Kei1N were also found in the superna- <br /> found in several fungal genome databases by FASTAP                               tant only when treated with 1% Triton X-100 (Supplemental <br /> search (Figure 2A), but no homologues were found in data-                        Figure S1B). Therefore, after Kex2 cleavage, both fragments <br /> bases of higher eukaryotes. The hydropathy plot indicated                        of Kei1 remain integrated in the membrane, as predicted by <br /> that there are four hydrophobic segments, which are pre-                         the hydropathy profile. <br /> dicted as transmembrane helices by the TMHMM program <br /> (http://www.cbs.dtu.dk/services/TMHMM/) (Figure 2B).                             Kei1 Localizes to the Medial-Golgi after Kex2 Cleavage <br />                                                                                  To determine the localization of Kei1 more precisely, we <br /> Kei1 Is an Integral Membrane Protein and Undergoes                               constructed a strain that express KEI1-GFP and KRE2-3HA <br /> Cleavage by Kex2                                                                 from their own loci on the chromosome by their own pro- <br /> We constructed a plasmid that produces Kei1 which is C-                          moters, and investigated where Kei1 localizes within the cell <br /> terminally fused with green fluorescent protein (Kei1-GFP)                       in several ways. <br /> and introduced it to a diploid strain in which one of the KEI1                      First, we compared the localization of Kei1C-GFP with the <br /> alleles was disrupted by kanMX4. By tetrad analysis, a hap-                      early Golgi-marker Van1, the medial-Golgi-marker Kre2- <br /> loid disruptant strain producing the fusion protein was ob-                      3HA and the late Golgi-marker Kex2 by subcellular fraction- <br /> tained. Kei1-GFP fully rescued the &#9004;kei1 disruption because                      ation. The cell lysate was centrifuged at 10,000 &#11003; g for 10 <br /> KEI1-GFP cells showed no apparent growth defects (data not                       min to remove the ER fraction, and the resulting supernatant <br />  <br /> 4448                                                                                                                Molecular Biology of the Cell <br />                                                                                            Novel Subunit of Yeast IPC Synthase <br />  <br />  <br />  <br />  <br /> Figure 2. Sequence alignment of Kei1 pro- <br /> tein and its homologues and the hydropathy <br /> profile of Kei1 protein. (A) Amino acid se- <br /> quence alignment of the Kei1 homologues. <br /> Transmembrane domains predicted by the <br /> TMHMM program are indicated by bars over <br /> the sequences. The C-terminal peptide that <br /> fused to GST in Figure 9 (Kei1CT) is indicated <br /> by a dotted line. The sequence alterations in <br /> kei1-1 (F103I and &#9004;C) are indicated by arrows. <br /> The Kex2 recognition residue (135R) is indi- <br /> cated by an arrowhead. The identical amino <br /> acids were highlighted with reverse letters. <br /> S.c., S. cerevisiae; C.g., <span id='am-54' about='obo:NCBITaxon_5475' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_118267 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-55' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasRelatedSynonym" content="Torulopsis" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="Candida" datatype="xsd:string"></span><span id='am-58'  ></span><span id='am-59' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Candida</span> glabrata; V.p., <br /> <span id='am-112' about='obo:NCBITaxon_374469' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-113' property="rdfs:label" content="Vanderwaltozyma" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasDbXref" content="PMID:14654427" datatype="xsd:string"></span><span id='am-116'  ></span><span id='am-117' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Vanderwaltozyma</span> <span id='am-144' about='obo:NCBITaxon_107825' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_27065 obo:NCBITaxon_3398 obo:NCBITaxon_41945 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-145' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-147' property="rdfs:label" content="Polyspora" datatype="xsd:string"></span><span id='am-148'  ></span>polyspora</span>; A.g., Ashbya gossy- <br /> pii; and K.l., <span id='am-16' about='obo:NCBITaxon_4910' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_4751 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_2759 obo:NCBITaxon_147537'><span id='am-17'  ></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="Kluyveromyces" datatype="xsd:string"></span>Kluyveromyces</span> lactis. (B) The hy- <br /> drophobicity index was calculated according <br /> to Kyte and Doolittle (1982),with a window <br /> size of 19 amino acids. Bold lines indicate the <br /> hydrophobic domains. <br />  <br />  <br />  <br /> was loaded on the 22&#8211; 60% sucrose density gradient, and        that Kei1 localizes to the medial-Golgi compartment after <br /> then proteins were fractionated by ultracentrifugation. Dis-   Kex2 cleavage. <br /> tribution of the indicated proteins was analyzed by SDS- <br /> PAGE and Western blotting. As shown in Figure 4A, the          Isolation of kei1-1, a Temperature-sensitive Allele of KEI1 <br /> peak of Kei1C-GFP was found between the peaks of Sed5          To investigate what function Kei1 exerts in the medial-Golgi, <br /> and Kex2 and exhibited a very similar distribution to Kre2-    we sought temperature-sensitive (ts) alleles of KEI1 by error- <br /> 3HA, indicating that Kei1C-GFP localizes to the medial-        prone PCR mutagenesis. We obtained one ts allele kei1-1, in <br /> Golgi. Next, we examined the localization of Kei1-GFP,         which a base substitution caused replacement of 103Phe with <br /> GFP-Kei1, and Kre2-3HA by confocal <span id='am-134' about='obo:NCBITaxon_62990' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_364270 obo:NCBITaxon_131567 obo:NCBITaxon_241799 obo:NCBITaxon_91882 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_241789 obo:NCBITaxon_4037 obo:NCBITaxon_241778 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_4036 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-135' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-136' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-137'  ></span><span id='am-138' property="rdfs:label" content="Laser" datatype="xsd:string"></span>laser</span> microscopy            Ile (F103I) and a nucleotide deletion caused truncation of <br /> (Figure 4B). Kei1-GFP and GFP-Kei1 signals showed              the C-terminal region 193Gln- . . . -221Glu-COOH to 193Lys- <br />                                                                194 <br /> punctate patterns throughout the cytoplasm, most of                Thr-COOH (&#9004;C). Both of the mutations are required for <br /> which overlapped with Kre2-3HA signals. All Kei1-GFP           temperature sensitivity, because the cells carrying only the <br /> dots were apparently coincident with Kre2-3HA dots, but        F103I or &#9004;C mutation can form colonies to a similar extent as <br /> some GFP-Kei1 signals did not coincide with Kre2-3HA.          wild-type cells at 37&#176;C (Supplemental Figure S2A). The chro- <br /> They probably correspond to GFP-Kei1F in the ER, be-           mosomal KEI1 allele was replaced with the mutant kei1-1 by <br /> cause a considerable amount of the uncleaved GFP-Kei1F         homologous recombination for further analysis. <br /> was cofractionated with the ER-marker Scs2 in our su- <br /> crose density gradient centrifugation experiments (Sup-        Kei1 Genetically and Physically Interacts with Aur1 <br /> plemental Figure S1C). GFP-Kei1F is more abundant than         To test whether Kei1 is involved in protein transport or <br /> Kei1F-GFP in KRE2-3HA strains (see Figure 6), which also       modification, the processing of carboxypeptidase Y, glyco- <br /> supports this possibility. From these results, we conclude     sylphosphatidylinositol (GPI)-anchored protein Gas1, and <br />  <br /> Vol. 20, October 15, 2009                                                                                               4449 <br />  K. Sato et al. <br />  <br />  <br />  <br />  <br /> Figure 3. Kei1 is a novel substrate of the late Golgi endoprotease <br /> Kex2. (A) KEI1-GFP was expressed from CEN plasmid. The total cell <br /> lysate was subjected to differential centrifugation to obtain indicated <br /> fractions. Proteins were resolved and detected by SDS-PAGE and <br /> Western blotting. Two bands at the molecular mass of 36.5 kDa and <br /> 27 kDa were detected by anti-GFP antibody. The 36.5-kDa band was <br /> mainly recovered in the P10 fraction, and the 27-kDa band was <br /> recovered evenly in P10 and P100 fractions. The early Golgi mem- <br /> brane protein Van1 was detected by anti-Van1 antibody. (B) Effect of <br /> intron on Kei1 protein size. KEI1 with or without its intron sequence <br /> was expressed from the CEN plasmid and proteins were detected by          Figure 4. Kex2-cleaved Kei1 is localized to the medial-Golgi. (A) <br /> the C-terminal GFP tag. (C) Kei1 protein and its derivatives without      The lysate of cells expressing KEI1-GFP and KRE2-3HA (KSY714) <br /> possible Kex2 cleavage sites. KEI1-GFP, KEIK131S-GFP, KEI1R135S-          was centrifuged at 10,000 &#11003; g for 10 min, and then the supernatant <br /> GFP, or KEI1K141S-GFP was expressed from CEN plasmid in wild-             was subjected to 22&#8211; 60% sucrose density gradient fractionation (see <br /> type (&#11001;) and &#9004;kex2 (&#11002;) cells.                                             Materials and Methods for details). Each fraction was subjected to <br />                                                                           SDS-PAGE, and proteins were detected by Western blotting. The <br />                                                                           signal intensity of indicated proteins was quantified with ImageJ <br />                                                                           software and graphed with Excel. (B) Localization of Kei1 with <br /> secretory invertase were analyzed, but no apparent defects                C-terminal GFP tag (Kei1-GFP, top; KSY714) or Kei1 with N-termi- <br /> were observed in any of the analyses (Supplemental Figure                 nal GFP tag (GFP-Kei1, bottom; KSY725) was shown with indirect <br /> S2, B&#8211;D).                                                                 immunofluorescence of the medial-Golgi protein Kre2-3HA (Al- <br />    Next, we screened for multicopy suppressor genes of the                exa568 anti-mouse). Bar, 5 &#9262;m. <br /> temperature sensitivity of the kei1-1 mutant, because it often <br /> becomes an effective clue to elucidate the functional role of <br /> uncharacterized gene products. We constructed a 2-&#9262;m                      Aur1-3HA were almost identical (Supplemental Figure <br /> genomic library of the kei1-1 chromosome and introduced it                S1C). So, we next investigated the physical interaction of <br /> to kei1-1 cells and screened for colonies whose viability at              Kei1 with Aur1 in vivo. The lysate of KEI1-GFP AUR1-3HA <br /> 37&#176;C depended on the introduced plasmids. Consequently,                   or GFP-KEI1 AUR1-3HA cells were treated with 1% Triton <br /> we identified AUR1, PSD1, GDA1, and SFH1 (YKL091c) as                     X-100 to solubilize the membrane. Aur1-3HA was immuno- <br /> multicopy suppressors of the temperature sensitivity of                   precipitated with anti-HA antibody, and the GFP signal was <br /> kei1-1 (Figure 5A and Supplemental Figure S3). Importantly,               analyzed. Cells expressing KRE2-3HA instead of AUR1-3HA <br /> AUR1 could significantly recover the growth defect of the                 were used as negative controls. As shown in Figure 6, A and <br /> kei1-1 mutant at 37&#176;C (Figure 5A). AUR1 is now thought to                 B, both Kei1C-GFP and GFP-Kei1N were coprecipitated <br /> encode IPC synthase (Funato et al., 2002; Cowart and Obeid,               with Aur1-3HA, indicating that Kei1 and Aur1 form a com- <br /> 2007). High copy introduction of AUR1 gene could not sup-                 plex in vivo. Coprecipitation of Kei1F-GFP or GFP-Kei1F <br /> press the lethality of the &#9004;kei1 null mutant (Figure 5B),                 with Aur1-3HA was much less efficient, but we reproducibly <br /> indicating that Kei1 has an indispensable role related to                 observed the coprecipitated band. This may indicate that <br /> Aur1.                                                                     Aur1 interacts with Kei1F that has not yet arrived in the late <br />    The strong suppressor activity of AUR1 suggests that the               Golgi compartment where Kex2 cleaves Kei1. <br /> kei1-1 mutant has defects in IPC synthesis. We therefore <br /> tested whether the kei1-1 mutant has altered sensitivity to               Activity of the IPC Synthase Significantly Decreases <br /> aureobasidin A, a specific inhibitor of IPC synthase (Nagiec              in the kei1-1 Mutant <br /> et al., 1997). The kei1-1 mutant showed a strong hypersensi-              The in vivo interaction of Kei1 and Aur1, in addition to the <br /> tivity to this inhibitor (Figure 5C), which further supports              hypersensitivity to aureobacidin A of kei1-1 cells, led us to <br /> the direct involvement of Kei1 in IPC synthesis.                          suppose that Kei1 is essential for IPC synthase activity. <br />    When the total cell lysate was subjected to sucrose density            Therefore, we tested whether the membrane preparation <br /> gradient centrifugation, the distribution of Kei1C-GFP and                from the kei1-1 mutant has any difference in the activity of <br />  <br /> 4450                                                                                                           Molecular Biology of the Cell <br />                                                                                                       Novel Subunit of Yeast IPC Synthase <br />  <br />  <br />  <br />  <br />                                                                         Figure 6. Kei1 is a subunit of the IPC synthase complex. (A) The <br />                                                                         lysate of KRE2-3HA KEI1-GFP (KSY714) or AUR1-3HA KEI1-GFP <br />                                                                         (KSY525) cells were treated with 1% Triton X-100 and centrifuged to <br />                                                                         prepare the supernatant (S) for immunoprecipitation with anti-HA <br />                                                                         antibody. U, unbound; B, bound materials. Material derived from <br />                                                                         10-fold more cells was loaded for sample B than for samples S and <br />                                                                         U. (B) Immunoprecipitation was performed from the cleared lysate <br />                                                                         of KRE2-3HA GFP-KEI1 (KSY725) or AUR1-3HA KEI1-GFP (KSY721) <br />                                                                         cells and analyzed as described in A. <br />  <br />  <br />  <br />                                                                         activity of mutant membrane was reduced (lanes 4 and 5). <br />                                                                         Furthermore, when the reaction was carried out at 37&#176;C, it <br />                                                                         was greatly reduced (lanes 8 and 9). Pre-incubation of the <br />                                                                         cells for 2 h at 37&#176;C before collection also reduced the IPC <br />                                                                         synthesis activity of the mutant membrane at 26.5&#176;C (lanes 6 <br />                                                                         and 7), and the membrane had no detectable activity at 37&#176;C <br />                                                                         (lanes 10 and 11). Therefore, the kei1-1 mutant has heat-labile <br />                                                                         IPC synthesis activity. <br /> Figure 5. Isolation of kei1-1, a temperature-sensitive mutant allele       It is yet unclear whether the specific activity of the IPC <br /> of KEI1 and identification of AUR1 as a strong multicopy suppressor     synthase was actually reduced in the kei1-1 mutant, because <br /> of kei1-1 temperature sensitivity. (A) KEI1 was introduced to kei1-1    the amount of Aur1-GFP concomitantly decreased in the <br /> mutant on CEN plasmid and AUR1 was introduced to kei1-1 mutant <br /> on 2&#9262; plasmid. pRS426 is the empty vector. Cells were spotted onto <br />                                                                         kei1-1 membrane, although that of Van1 was not affected <br /> the YPD plates with 10&#11003; serial dilution and incubated at 26.5 and       (Figure 7A). To avoid this problem and to directly assess the <br /> 37&#176;C. (B) KEI1/&#9004;kei1::kanMX4 diploid cells were introduced with         activity of IPC synthase, the enzyme was isolated by immu- <br /> empty vector pRS426, KEI1 on CEN plasmid, or AUR1 on 2&#9262; plas-           noprecipitation of Aur1-3HA with anti-HA antibody. As <br /> mid, sporulated, and then the viability of the generated tetrads were   shown in Figure 7B, the immunoprecipitates from the lysate <br /> analyzed. Four spores from a single ascus were vertically aligned.      of AUR1-3HA cells converted C6-NBD-ceramide to NBD- <br /> (C) BY4741 and kei1-1 mutant (KSY271) were streaked onto the SC         IPC in a dose-dependent manner (lanes 4 &#8211; 6). However im- <br /> plates with or without 0.08 &#9262;g/ml aureobasidin A.                       munoprecipitates from the lysate of KRE2-3HA cells pro- <br />                                                                         duced no detectable NBD-IPC (lanes 1&#8211;3), which confirms <br />                                                                         the validity of the IPC synthase assay system. Therefore we <br /> IPC synthesis. We used C6-NBD-ceramide as a substrate of                then assessed the IPC synthase activity isolated from the <br /> IPC synthase, because it has been widely used to assay IPC              kei1-1 mutant cells. As a result, it showed no detectable IPC <br /> synthesis activity of yeast microsomal membrane in vitro                synthesis activity (lanes 10 &#8211;12), even though the cultivation <br /> (Zhong et al., 1999; Levine et al., 2000; Aeed et al., 2004).           of cells and the IPC synthesis reaction were carried out at the <br /> Synthesis of NBD-IPC is completely blocked by aureobasi-                permissive temperature, and the amount of Aur1-3HA pro- <br /> din A (Figure 7A, lane 1). When cells were grown at 26.5&#176;C,             tein in the kei1-1 sample was not less than that in the wild- <br /> no apparent difference was observed in the IPC synthesis                type sample (lanes 7&#8211;12, immunoblots). The kei1-1 IPC syn- <br /> activity at 26.5&#176;C between the wild-type and kei1-1 mutant              thase may be very labile in the absence of membrane lipids <br /> membranes (Figure 7A, lanes 2 and 3). However, when the                 and loses its activity during the solubilization and immu- <br /> membrane was incubated at 37&#176;C before the reaction, the                 noisolation steps. <br />  <br /> Vol. 20, October 15, 2009                                                                                                             4451 <br />  K. Sato et al. <br />  <br />  <br />  <br />  <br /> Figure 7. Kei1 is essential for the activity of IPC synthase. (A) AUR1-3HA&#8211; expressing cells (KSY512 and KSY513) were grown at 26.5&#176;C and <br /> further incubated at indicated temperatures for 2 h. Lysate was prepared from the cells, and membrane fraction was recovered by <br /> ultracentrifugation. Membrane fractions were preincubated at the indicated temperatures, after which the reaction was performed at the <br /> indicated temperatures for 1 h. (B) Immunoisolation with anti-HA was performed from cells expressing KRE2-3HA (THY4-2) or AUR1-3HA <br /> (KSY525 and KSY530) grown at 26.5&#176;C, and the activity of IPC synthase on the beads at 26.5&#176;C for 1 h was tested. The product NBD-IPC was <br /> detected on the TLC by LAS-1000plus. The proteins on the beads were monitored by Western blotting. (C) Immunoisolation with anti-HA <br /> was performed from KEI1 AUR1-3HA (KSY771), kei1F103I AUR1-3HA (KSY773), or kei1&#9004;C AUR1-3HA (KSY775) cell lysate, and the activity of <br /> IPC synthase was determined as described in B. (D) Immunoprecipitation of Aur1-3HA was performed from the cleared lysate of KEI1-GFP <br /> AUR1-3HA (KSY792) or kei1F103I-GFP AUR1-3HA (KSY794) cells, and the indicated proteins were monitored by Western blotting. S, <br /> supernatant; U, unbound; and B, bound. Material derived from 10-fold more cells was loaded for sample B than for samples S and U. (E) <br /> Immunoprecipitation of Aur1-3HA was performed from the cleared lysate of GFP-KEI1 AUR1-3HA (KSY799) or GFP-kei1F103I AUR1-3HA <br /> (KSY801) cells and analyzed as described in D. <br />  <br />  <br />  <br />    We next tested which of the two mutations of kei1-1 is             apparent growth defect (Supplemental Figure S2A). To ex- <br /> responsible for the loss of IPC synthase activity. Whereas the        amine whether the activity of the IPC synthase of the <br /> IPC synthase from kei1&#9004;C cells showed no significant reduc-           kei1F103I cells is also significantly affected in vivo, we assayed <br /> tion of the IPC synthase activity than that from KEI1 cells           conversion of C6-NBD-ceramide to NBD-IPC by the cells as <br /> (Figure 7C, lanes 1&#8211;3 and 7&#8211;9), the activity of the IPC syn-          described by Levine et al. (2000). As a result, no significant <br /> thase from kei1F103I cells was significantly decreased (lanes         decrease in NDB-IPC production in kei1F103I cells was ob- <br /> 4 &#8211; 6). This result clearly indicates that the F103I replacement      served (Supplemental Figure S4), suggesting that the inacti- <br /> is responsible for inactiveness of IPC synthase isolated from         vation of IPC synthase from kei1F103I cells occurs in the <br /> kei1-1 cells.                                                         process of immunoisolation. <br />    The inactivation of isolated IPC synthase by the F103I                If Kei1 is an essential subunit for the IPC synthase activity, <br /> replacement is surprising because kei1F103I cells show no             it is possible that Kei1F103I may be released from Aur1-3HA <br />  <br /> 4452                                                                                                       Molecular Biology of the Cell <br />                                                                                               Novel Subunit of Yeast IPC Synthase <br />  <br />  <br />  <br />  <br /> Figure 8. Aur1-GFP in kei1-1 cells is deliv- <br /> ered to and degraded in the vacuole. (A) The <br /> Aur1-GFP levels in wild type (lane 1; <br /> KSY512), &#9004;pep4 (lane 2; KSY571), kei1-1 (lane <br /> 3; KSY513), kei1-1 &#9004;ubc7 (lanes 4 and 5; <br /> KSY552, KSY553), kei1-1 &#9004;doa10 (lanes 6 and <br /> 7; KSY554, KSY561), kei1-1 &#9004;hrd1 (lanes 8 and <br /> 9; KSY559, KSY560), kei1-1 &#9004;ubx2 (lanes 10 <br /> and 11; KSY562, KSY563), and kei1-1 &#9004;pep4 <br /> (lane 12; KSY572) cells were compared. Two <br /> independent isolates of ERAD mutants were <br /> used and cells were incubated at 26.5&#176;C. Sim- <br /> ilar results were obtained after further incu- <br /> bation at 37&#176;C for 2 h (data not shown). (B) <br /> The localization of Aur1-GFP was observed <br /> by fluorescence microscopy at the left panels. <br /> Nomarski images are shown at the right, and <br /> the merge of fluorescence and Nomarski im- <br /> ages at the middle. The wild-type (KSY512), <br /> &#9004;pep4 (KSY571), kei1-1 (KSY513), and kei1-1 <br /> &#9004;pep4 (KSY572) cells were incubated at <br /> 26.5&#176;C. (C) The localization of Kre2-3HA in <br /> the KEI1 (KSY790) or kei1-1 (KSY784) cells <br /> was observed by fluorescence microscopy. <br /> Cells were incubated at 26.5&#176;C and then at <br /> 26.5 or 37&#176;C for 2 h. Bars, 5 &#9262;m. <br />  <br />  <br />  <br /> during immunoisolation, which may result in enzyme inac-          for 2 h, the amount of Aur1-GFP in kei1-1 cells decreased <br /> tivation. To test this possibility, we investigated the associ-   further (lanes 6 and 7). These results suggest that degrada- <br /> ation of Kei1F103I to Aur1-3HA on the beads. Aur1-3HA was         tion of Aur1 occurred in the kei1-1 mutant. There are two <br /> immunoprecipitated from AUR1-3HA KEI1-GFP cells or                possible explanations for this. One explanation is that the <br /> AUR1-3HA kei1F103I-GFP cells as in Figure 7C, and the             newly produced Aur1 accumulates in the ER and is subse- <br /> amount of bound proteins were analyzed by Western blot-           quently degraded by ER-associated degradation (ERAD) <br /> ting. As expected, a significant amount of Kei1-GFP was           pathway. The other explanation is that the Golgi localization <br /> found in the bound fraction (Figure 7D, lane 5), but the          of Aur1 is unstabilized, and Aur1 is degraded in the vacuole. <br /> signal of Kei1F103IC-GFP in bound fraction could be hardly        To determine which possibility actually occurs in the kei1-1 <br /> detected (Figure 7D, lane 6). Similar result was obtained in      mutant, we deleted genes involved in the ERAD pathway or <br /> the case of GFP-Kei1F103IN (Figure 7E). These results dem-        protein degradation in the vacuole. Deletion of genes encod- <br /> onstrate that Aur1 alone does not have IPC synthase activity      ing Ubc7 (ubiquitin-conjugating enzyme E2; Bays et al., <br /> and that Kei1 is indeed the essential subunit of IPC synthase     2001), Doa10 (ubiquitin ligase E3; Carvalho et al., 2006), Hrd1 <br /> for its activity.                                                 (ubiquitin ligase E3; Bays et al., 2001), or Ubx2 (membrane <br />                                                                   protein that recruits AAA ATPase Cdc48 to substrate pro- <br /> The Golgi Localization of Aur1 Is Diminished in kei1-1            teins; Neuber et al., 2005; Schuberth and Buchberger, 2005) <br /> Mutant and Aur1 Is Degraded in the Vacuole                        did not result in any stabilization of Aur1-GFP in the kei1-1 <br /> As shown in Figure 7A, the amount of Aur1-GFP protein             mutant (Figure 8A, lanes 4 &#8211;11). Aur1-GFP was restored to <br /> had decreased in kei1-1 cells, even when they grew at 26.5&#176;C      the wild-type level by deletion of PEP4, which encodes a <br /> and almost an equal amount of Van1 was found in the               master protease of the vacuole (Van Den Hazel et al., 1996) <br /> membrane (lanes 2 and 3). When cells were grown at 37&#176;C           (Figure 8A, lane 12). This result suggests that Aur1 is unable <br />  <br /> Vol. 20, October 15, 2009                                                                                                   4453 <br />  K. Sato et al. <br />  <br />  <br />                                                                                        Figure 9. The C-terminal region of Kei1 is im- <br />                                                                                        portant for stable Golgi localization of Kei1 and <br />                                                                                        Aur1. (A) PEP4 (&#11001;) or &#9004;pep4 (&#11002;) with KEI1-GFP <br />                                                                                        (&#11001;: KSY696, &#11002;: KSY732), kei1-1-GFP (&#11001;: KSY698, <br />                                                                                        &#11002;: KSY780), kei1&#9004;C-GFP (&#11001;: KSY694, &#11002;: <br />                                                                                        KSY730), and kei1F103I-GFP (&#11001;: KSY692, &#11002;: <br />                                                                                        KSY728) cells were incubated at 26.5&#176;C. The <br />                                                                                        amounts of Kei1F-GFP and Kei1C-GFP were <br />                                                                                        analyzed by Western blotting. Lanes 1&#8211; 6 were <br />                                                                                        tested on the same gel and membrane and du- <br />                                                                                        plicated lanes were removed. The same applies <br />                                                                                        to lanes 7&#8211;10. (B) PEP4(&#11001;) or &#9004;pep4(&#11002;) with <br />                                                                                        GFP-KEI1 (&#11001;: KSY723, &#11002;: KSY765) and GFP- <br />                                                                                        KEI1F103I (&#11001;: KSY753, &#11002;: KSY767) cells were <br />                                                                                        incubated at 26.5&#176;C. The amounts of GFP-Kei1F <br />                                                                                        and GFP-Kei1N were analyzed by Western <br />                                                                                        blotting. Lanes 1&#8211; 4 were tested on the same gel <br />                                                                                        and membrane and duplicated lanes were re- <br />                                                                                        moved. (C) The amounts of Aur1-3HA in the <br />                                                                                        KEI1-GFP (KSY771), kei1F103I-GFP (KSY773) and <br />                                                                                        kei1&#9004;C-GFP (KSY775) cells were analyzed by <br />                                                                                        Western blotting. Duplicate samples were <br />                                                                                        tested. (D) GST or GST-fusion protein of C ter- <br />                                                                                        minus of Kei1 (GST-Kei1CT) expressed in E. coli <br />                                                                                        was purified and mixed with the yeast cell ly- <br />                                                                                        sate. Bound proteins were precipitated with <br />                                                                                        glutathione-Sepharose and detected by West- <br />                                                                                        ern blotting with indicated antibodies to &#9251;-COP <br />                                                                                        and &#9253;-COP. Phosphoglycerate kinase (PGK) <br />                                                                                        was also detected as a negative control. <br />  <br />  <br />  <br /> to stably localize to the Golgi in the absence of wild-type       kanMX4, and the amounts of Golgi-localized Kei1-1C-GFP in <br /> Kei1, and consequently it tends to be delivered to and de-        PEP4 and &#9004;pep4 cells were compared. If Kei1-1C-GFP is <br /> graded in the vacuole. To verify that Aur1 is transported to      unable to stay in the Golgi and is delivered to the vacuole <br /> the vacuole in kei1-1 cells, we observed the localization of      like Aur1 in kei1-1 cells, the amount of Kei1-1C-GFP should <br /> Aur1-GFP in kei1-1 cells at the permissive temperature. In        be restored by the deletion of PEP4. As expected, the amount of <br /> the KEI1 cells, regardless of the presence or absence of PEP4,    Kei1-1C-GFP was significantly increased by &#9004;pep4 (Figure 9A, <br /> Aur1-GFP displayed the punctate Golgi patterns (Figure 8B),       lanes 1 and 2), whereas the amount of wild-type Kei1C-GFP <br /> which is consistent with the previous observation that            was not affected by the deletion of the PEP4 gene (Figure 9A, <br /> showed Aur1-3HA localizes to the medial-Golgi (Levine et          lanes 5 and 6). From this result, we conclude that Kei1-1C <br /> al., 2000). In the kei1-1 PEP4 cells, the fluorescence of Aur1-   itself is unable to stably localize in the Golgi and is delivered <br /> GFP was obscure and Golgi pattern could be hardly detected        to the vacuole. <br /> (Figure 8B); in the kei1-1 &#9004;pep4 cells, Aur1-GFP was accu-           To test whether either the F103I substitution or &#9004;C trun- <br /> mulated in the vacuole (Figure 8B, bottom). In contrast,          cation causes loss of Golgi localization, we checked the <br /> Kre2-3HA, a control medial-Golgi protein that shows similar       amount of GFP-Kei1F103IN and Kei1&#9004;CC-GFP in PEP4 and <br /> punctate immunofluorescence pattern, was not affected by          &#9004;pep4 cells. The amount of GFP-Kei1F103IN in &#9004;pep4 cells <br /> kei1-1 at either temperature (Figure 8C). From these results,     was slightly increased compared with that in PEP4 cells <br /> we concluded that the stable Golgi localization of Aur1 is        (Figure 9B, lanes 1 and 2); however, this increase was also <br /> compromised in kei1-1 cells and a fraction of Aur1 is deliv-      observed in GFP-Kei1N (Figure 9B, lanes 3 and 4), which <br /> ered to the vacuole and degraded in a Pep4-dependent              suggests that even the wild-type Kei1 is constitutively de- <br /> manner. It may be worth noting that, when GFP-fusion              graded in a Pep4-dependent manner when GFP is fused to <br /> proteins are mislocalized and degraded in the vacuole, the        its N terminus. Therefore it is unlikely that F103I substi- <br /> fluorescence signal of the GFP remnants remains detectable        tution affects the Golgi localization. In contrast, the <br /> in many cases (Reggiori and Pelham, 2002; Sato and Nakano,        amount of Kei1&#9004;CC-GFP significantly increased when <br /> 2002; Valdez-Taubas and Pelham, 2005). However, in this           PEP4 was disrupted (Figure 9A, lanes 3 and 4), although <br /> case the GFP remnants also seemed to be digested.                 that of Kei1F103IC-GFP or Kei1C-GFP did not (Figure 9A, <br />                                                                   lanes 5&#8211;10). These results indicate that the C-terminal region <br /> Deletion of the C Terminus of Kei1 Causes                         of Kei1 is important for localization to the Golgi. The obser- <br /> Mislocalization of Kei1 and Aur1                                  vation that the amount of Aur1-3HA in kei1&#9004;C cells was less <br /> How does Kei1 contribute to the stable localization of Aur1       than that in KEI1 and kei1F103I cells (Figure 9C) also sup- <br /> to the Golgi? To address this question, we first tested           ported the importance of the Kei1 C-terminal region in its <br /> whether Golgi localization of Kei1 is also unstable due to the    Golgi localization. <br /> kei1-1 mutant. Kei1-1-GFP was used in this experiment. To <br /> make the construct of Kei1-1-GFP, the nucleotide sequence         Cytosolic C-Terminal Peptide of Kei1 Can Interact <br /> was manipulated so that F103I, Q193K, and K194T replace-          with Coat Protein (COP)I Coatomer <br /> ments were introduced in the Kei1 sequence, and GFP was           Kex2 exerts its protease activity in the lumen of the Golgi; <br /> directly fused to the 194Thr residue in Kei1. Kei1-1-GFP was      therefore, the Kex2 recognition site of Kei1, the 135Arg resi- <br /> produced in cells whose genomic KEI1 were replaced with           due, should also locate in the lumen. Because there is only <br />  <br /> 4454                                                                                                   Molecular Biology of the Cell <br />                                                                                                 Novel Subunit of Yeast IPC Synthase <br />  <br />  <br /> one hydrophobic segment between 135Arg and the C termi-               Although the cleavage of Kei1 by Kex2 does not seem to <br /> nus, the C terminus of Kei1 is very likely to be exposed to the    be biologically important, it has allowed us to investigate the <br /> cytoplasm. Many cases are known, where the cytoplasmic             localization of IPC synthase in detail. Most Kei1 in the Golgi <br /> C-terminal peptide of the ER and Golgi membrane proteins           fractions has been cleaved by the Kex2 protease that resides <br /> interacts with coatomer, which is responsible for retrograde       in the late Golgi compartment, although Kei1 is mostly <br /> trafficking in the secretory pathway. For example, the dil-        found in the medial-Golgi (Figure 4). This suggests that Kei1 <br /> ysine motif KKXX or KXKXX binds to coatomer directly and           is temporarily transported to the late Golgi whereby it is <br /> supports the incorporation of proteins with this motif into        cleaved by Kex2. Kei1 is then efficiently transported retro- <br /> the COPI vesicle (Jackson et al., 1990; Cosson and Letour-         grade to the medial-Golgi. Because Kei1C-GFP was hardly <br /> neur, 1994). Furthermore, although the dilysine motif does         detected in Sed5-rich fractions when the Golgi compart- <br /> not exist in Rer1, Emp24, or Vrg4, their relatively basic          ments were fractionated by sucrose density gradient centrif- <br /> C-terminal tails interact with coatomer and are important for      ugation (Figure 4A), Kei1 is likely to recycle between the <br /> their Golgi localization (Belden and Barlowe, 2001; Sato et        medial- and late Golgi in a COPI-dependent manner. This <br /> al., 2001; Abe et al., 2004). We hypothesized that Kei1 also       <span id='am-67' about='obo:NCBITaxon_76236' typeof='owl:Thing obo:NCBITaxon_37567 obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_30248 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_33415 obo:NCBITaxon_33317 obo:NCBITaxon_41191 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_37572 obo:NCBITaxon_304572 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_41196 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_127218 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-68' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Idea" datatype="xsd:string"></span><span id='am-71'  ></span>idea</span> is consistent with the previous report that Aur1 does <br /> interacts with coatomer through its C-terminal tail like these     not recycle back to the ER in secretion block experiments <br /> proteins, and tested this by GST pull-down assay. The pep-         (Levine et al., 2000). In mammalian cells, sphingomyelin <br /> tide from 176Gln to 221Glu of Kei1 was fused to the C termi-       (SM) synthase 1, which produces SM from newly synthe- <br /> nus of GST. The fusion protein was produced in E. coli and         sized ceramide, localizes to the trans-Golgi network <br /> purified with glutathione-Sepharose affinity beads. The            (Huitema et al., 2004). Ceramide transfer protein (CERT) <br /> beads were mixed with the yeast cell lysate, and proteins          directly transfers newly synthesized ceramide from the ER <br /> that bound to the beads were analyzed. As shown in Figure          to the site of SM synthesis in the trans-Golgi network <br /> 9D, we found that the coatomers, &#9251;-COP Ret1 and &#9253;-COP              (Hanada et al., 2003). As it has been reported that ceramide <br /> Sec21, bound to the C-terminal peptide of Kei1. Phospho-           is transferred from the ER to the Golgi by both vesicular and <br /> glycerate kinase, a negative control, was not detected. This       nonvesicular transport in yeast (Funato and Riezman, 2001), <br /> result suggests that Kei1 helps the medial-Golgi localization      it is possible that an unidentified functional homologue of <br /> of IPC synthase in a COPI vesicle-dependent manner.                CERT carries ceramide from the ER to the medial-Golgi. <br />                                                                    However, no protein homologous to CERT in sequence has <br />                                                                    been found thus far. <br /> DISCUSSION                                                            We identified GDA1, PSD1, and SFH1 as multicopy sup- <br />                                                                    pressors of the kei1-1 mutant. Their suppressor activities <br /> Genetic and biochemical approaches using temperature-sen-          were obviously lower than AUR1 but were both reproduc- <br /> sitive mutants provide effective clues to elucidate the bio-       ible and significant. Currently, the mechanism of the sup- <br /> logical roles of unknown essential proteins, as we reported        pression is not clear, but sphingolipid metabolism may be <br /> previously on the study of Pga1, a novel subunit of GPI-           somehow involved. GDA1 encodes a GDPase that converts <br /> mannosyltransferase II (Sato et al., 2007). The kei1-1 mutant,     guanosine diphosphate (GDP) to guanosine monophosphate <br /> which was obtained by our screening for temperature-sen-           after mannose is transferred to the substrate of mannosyl- <br /> sitive mutants of KEI1, has a F103I substitution and &#9004;C            transferases in the Golgi, and this conversion is required for <br /> truncation, and both are required for the temperature sen-         the transport of cytosolic GDP-mannose into the Golgi lu- <br /> sitivity. The F103I substitution seems to be silent in the cell,   men (Abeijon et al., 1993; Berninsone et al., 1994). We sup- <br /> and the mutant IPC synthase has enough activity to supply          pose that the multicopy expression of GDA1 may activate <br /> IPC for cell growth. However, Kei1F103I is released from           the sphingolipid synthetic pathway in the Golgi by increas- <br /> Aur1 during immunoisolation and abolishes IPC synthase             ing the supply of GDP-mannose, the substrate of MIPC <br /> activity. This clearly shows that Aur1 alone is unable to          synthase. PSD1 encodes a phosphatidylserine decarboxylase <br /> catalyze IPC synthesis. The &#9004;C truncation disturbs Golgi           of the mitochondrial inner membrane, which converts <span id='am-168' about='obo:NCBITaxon_128483' typeof='owl:Thing obo:NCBITaxon_33208 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_216303 obo:NCBITaxon_37796 obo:NCBITaxon_1206795 obo:NCBITaxon_69555 obo:NCBITaxon_6447 obo:NCBITaxon_33317 obo:NCBITaxon_6448 obo:NCBITaxon_2759 obo:NCBITaxon_6479 obo:NCBITaxon_216293 obo:NCBITaxon_6072'><span id='am-169' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-170' property="rdfs:label" content="Phos" datatype="xsd:string"></span><span id='am-171'  ></span><span id='am-172' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>phos</span>- <br /> localization and causes degradation of the enzyme in the           phatidylserine to phosphatidylethanolamine (Clancey et al., <br /> vacuole. Binding of the C-terminal peptide to coatomer sug-        1993). No relationship between Psd1 and Kei1 has been <br /> gests that Kei1 may have a role to incorporate IPC synthase        reported, but it is possible that the mitochondrial lipid com- <br /> in the COPI vesicles for recycling. Thus F103I and &#9004;C have         position affects the lipid homeostasis of the Golgi or that the <br /> different effects on Kei1 protein, and when combined, syn-         reduction of IPC impairs the function of mitochondria and <br /> ergistically cause temperature sensitivity. We conclude that       Psd1 repairs it. Sfh1 is one of five homologues of the lipid <br /> Aur1 requires Kei1 both for IPC synthase activity and cor-         transfer protein Sec14 in S. cerevisiae and has the highest <br /> rect localization to the Golgi.                                    similarity but the lowest functional relationship with Sec14. <br />    The finding that Kex2 cleaves Kei1 (Figure 3) led us to         Moreover, Sfh1 lacks the lipid transfer activity in vitro; <br /> hypothesize that IPC synthase activity is regulated through        therefore, the molecular function of Sfh1 in vivo is unclear <br /> its proteolytic processing. However, contrary to this, no          (Li et al., 2000). However, the multicopy suppressor activity <br /> apparent difference has been found in the activity, the sub-       of SFH1 may indicate that Sfh1 is actually involved in lipid <br /> unit interaction, or the localization of IPC synthase (our         metabolism. Previous investigations using DNA microarray <br /> unpublished data). Therefore, we have found no evidence            analysis revealed that SFH1 gene expression was signifi- <br /> that the Kex2 cleavage regulates IPC synthesis. Although           cantly enhanced by various stress conditions (Griac, 2007). <br /> Kei1 sequence is highly homologous to its fungal counter-          Considering that a number of studies demonstrated that <br /> parts, the hydrophilic region that has the 135Arg cleavage         sphingolipids are important for stress responses, Sfh1 may <br /> site is not conserved (Figure 2A). Arginine is only found in       regulate genes related to sphingolipid metabolism. Al- <br /> <span id='am-60' about='obo:NCBITaxon_5478' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_374468 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_600669 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-61'  ></span><span id='am-62' property="oboInOwl:hasDbXref" content="PMID:14654427" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="Candida glabrata" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasRelatedSynonym" content="Torulopsis glabrata" datatype="xsd:string"></span>Candida glabrata</span>, and no basic residue is found in Vanderwal-      though this hypothesis has not been examined, the observa- <br /> tozyma polyspora. We suggest that generation of Kex2 cleav-        tion of Sfh1 localization to the nucleus and Golgi (Schnabl et <br /> age site in S. cerevisiae Kei1 was an accidental event in its      al., 2003) suggests a possible role of Sfh1 that links the lipid <br /> evolution without specific advantages or disadvantages in          composition in the Golgi membrane and the regulation of <br /> IPC synthesis.                                                     gene expression. <br />  <br />    Concerning the role of sphingolipids in the stress re-                       <br /> sponse, interesting findings are reported in experiments us-                    <br /> ing a &#9004;lcb1 slc1-1 double mutant that has no sphingolipids <br /> but produces mannosylated phosphatidylinositol deriva-                     <br /> tives containing a C26 fatty acid (Dickson et al., 1990; Patton            <br /> et al., 1992; Lester et al., 1993). This mutant can grow nor- <br />                                                                                 <br /> mally in adequate growth conditions, but the growth is                           <br /> significantly inhibited at high temperature or low and high                   <br /> pH, indicating that cells cannot adapt to such immoderate <br />                                                                                <br /> environments without sphingolipids. These results demon-                        <br /> strate that ceramides are a key mediator of signaling in the            <br /> stress response. IPC synthase activity can define the balance                   <br /> of ceramide and complex sphingolipids; therefore, it should                    <br /> be regulated stringently in response to stimuli caused by                        <br /> intracellular and environmental changes. Kei1 is responsible                  <br /> for both the localization and the activity of IPC synthase,                     <br /> which is an ideal feature to be a regulatory subunit. Inter-                     <br /> estingly, we observed that the amount of Kei1 was signifi-                    <br /> cantly increased in kei1-1 cells compared with wild-type cells                    <br /> (our unpublished data), suggesting that cells up-regulate                        <br /> KEI1 in response to the decreasing amounts of IPC or the                          <br /> increase of ceramide. This hypothesis is currently under                           <br /> investigation.                                                                    <br />    Because tagging 3HA to the C terminus of Aur1 affected                        <br /> the ratio of Kei1C to Kei1F (Figure 6), it is possible that Aur1                  <br /> is responsible for in the transport of Kei1. Indeed, our pre-                  <br /> liminary data show that Kei1 accumulates in the ER by                             <br /> reduction of Aur1 in promoter shut-off experiments, sug- <br /> gesting that the exit of IPC synthase from the ER to the Golgi              <br /> requires Aur1&#8211;Kei1 complex formation. It must be interest-                        <br /> ing to investigate further how cells regulate the production, <br /> complex formation, and localization of Aur1 and Kei1.                             <br />   <br /> </body></html>