<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>6570&#177;6577 Nucleic Acids Research, 2003, Vol. 31, No. 22 <br /> DOI: 10.1093/nar/gkg859 <br />  <br />  <br /> Gene essentiality determines chromosome <br /> organisation bacteria <br /> Eduardo P. C. Rocha1,2,* Antoine Danchin1 <br /> 1 <br />  Unite&#194; Ge&#194;ne&#194;tique des Ge&#194;nomes Bacte&#194;riens, Institut Pasteur, 28, rue du Dr Roux, 75724 <span id='am-589' about='obo:NCBITaxon_49669' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_50362 obo:NCBITaxon_4447 obo:NCBITaxon_3398 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_131221 obo:NCBITaxon_58024 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_1437197 obo:NCBITaxon_4667 obo:NCBITaxon_3193'><span id='am-590' property="oboInOwl:hasAlternativeId" content="NCBITaxon:49665" datatype="xsd:string"></span><span id='am-591' property="rdfs:label" content="Paris" datatype="xsd:string"></span><span id='am-592' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-593' property="oboInOwl:hasRelatedSynonym" content="Paris L." datatype="xsd:string"></span><span id='am-594' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-595'  ></span>Paris</span> Cedex 15, France <br /> and 2Atelier Bioinformatique, Universite&#194; Pierre et Marie Curie, 12, Rue Cuvier, 75005 Paris, France <br />  <br /> Received August 1, 2003; Revised Accepted September 25, 2003 <br />  <br />  <br /> ABSTRACT                                                                        techniques allowed de&#174;nition set essential <br />                                                                                 genes bacteria. <span id='am-305' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-306' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-307' property="rdfs:label" content="This" datatype="xsd:string"></span><span id='am-308' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-309'  ></span>This</span> achieved through <br /> In <span id='am-517' about='obo:NCBITaxon_562' typeof='owl:Thing obo:NCBITaxon_561 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-518' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-519' property="oboInOwl:hasRelatedSynonym" content="bacterium E3" datatype="xsd:string"></span><span id='am-520' property="rdfs:label" content="Escherichia coli" datatype="xsd:string"></span><span id='am-521' property="oboInOwl:hasRelatedSynonym" content="Bacillus coli" datatype="xsd:string"></span><span id='am-522' property="oboInOwl:hasRelatedSynonym" content="Escherchia coli" datatype="xsd:string"></span><span id='am-523' property="oboInOwl:hasExactSynonym" content="Escherichia/Shigella coli" datatype="xsd:string"></span><span id='am-524'  ></span><span id='am-525' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli commune" datatype="xsd:string"></span><span id='am-526' property="oboInOwl:hasRelatedSynonym" content="Eschericia coli" datatype="xsd:string"></span><span id='am-527' property="oboInOwl:hasRelatedSynonym" content="Bacterium coli" datatype="xsd:string"></span><span id='am-528' property="oboInOwl:hasRelatedSynonym" content="Enterococcus coli" datatype="xsd:string"></span><span id='am-529' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-530' property="oboInOwl:hasDbXref" content="PMID:10319482" datatype="xsd:string"></span><span id='am-531' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662104" datatype="xsd:string"></span><span id='am-532' property="oboInOwl:hasAlternativeId" content="NCBITaxon:662101" datatype="xsd:string"></span>Escherichia coli</span> <span id='am-125' about='obo:NCBITaxon_1423' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_653685'><span id='am-126' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232822" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042381" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasRelatedSynonym" content="Vibrio subtilis" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429428" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasRelatedSynonym" content="Bacillus uniflagellatus" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasAlternativeId" content="NCBITaxon:387338" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232844" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis/Bacillus globigii" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232828" datatype="xsd:string"></span><span id='am-135' property="oboInOwl:hasRelatedSynonym" content="Bacillus natto" datatype="xsd:string"></span><span id='am-136' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451272" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1224247" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1451270" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232470" datatype="xsd:string"></span><span id='am-141'  ></span><span id='am-142' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232825" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1429427" datatype="xsd:string"></span><span id='am-144' property="oboInOwl:hasAlternativeId" content="NCBITaxon:523730" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042382" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232841" datatype="xsd:string"></span><span id='am-147' property="rdfs:label" content="Bacillus subtilis" datatype="xsd:string"></span><span id='am-148' property="oboInOwl:hasDbXref" content="PMID:11211269" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232824" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232819" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasAlternativeId" content="NCBITaxon:941949" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232840" datatype="xsd:string"></span><span id='am-153' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-154' property="oboInOwl:hasAlternativeId" content="NCBITaxon:981081" datatype="xsd:string"></span><span id='am-155' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1232830" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:hasRelatedSynonym" content="Bacillus subtilis8" datatype="xsd:string"></span><span id='am-157' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1042380" datatype="xsd:string"></span>Bacillus subtilis</span>, essen-                               different techniques. Saturation transposon mutagenesis, fol- <br /> tiality, expressivity, drives distribution                           lowed sequencing, allowed de&#174;nition essential genes <br /> genes replicating strands.                                      <span id='am-229' about='obo:NCBITaxon_2093' typeof='owl:Thing obo:NCBITaxon_2092 obo:NCBITaxon_1 obo:NCBITaxon_2085 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_544448 obo:NCBITaxon_31969'><span id='am-230' property="oboInOwl:hasDbXref" content="PMID:11931184" datatype="xsd:string"></span><span id='am-231' property="oboInOwl:hasAlternativeId" content="NCBITaxon:57371" datatype="xsd:string"></span><span id='am-232' property="oboInOwl:hasRelatedSynonym" content="Borrelomyces" datatype="xsd:string"></span><span id='am-233' property="oboInOwl:hasDbXref" content="PMID:8995799" datatype="xsd:string"></span><span id='am-234' property="oboInOwl:hasRelatedSynonym" content="Eperythrozoon" datatype="xsd:string"></span><span id='am-235' property="oboInOwl:hasDbXref" content="PMID:11321109" datatype="xsd:string"></span><span id='am-236' property="oboInOwl:hasDbXref" content="PMID:15176735" datatype="xsd:string"></span><span id='am-237'  ></span><span id='am-238' property="oboInOwl:hasAlternativeId" content="NCBITaxon:29500" datatype="xsd:string"></span><span id='am-239' property="oboInOwl:hasRelatedSynonym" content="Pleuropneumonia" datatype="xsd:string"></span><span id='am-240' property="oboInOwl:hasDbXref" content="PMID:8863441" datatype="xsd:string"></span><span id='am-241' property="oboInOwl:hasDbXref" content="PMID:16403858" datatype="xsd:string"></span><span id='am-242' property="oboInOwl:hasDbXref" content="PMID:16350067" datatype="xsd:string"></span><span id='am-243' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-244' property="oboInOwl:hasDbXref" content="GC_ID:4" datatype="xsd:string"></span><span id='am-245' property="oboInOwl:hasDbXref" content="PMID:10826816" datatype="xsd:string"></span><span id='am-246' property="oboInOwl:hasDbXref" content="PMID:11411711" datatype="xsd:string"></span><span id='am-247' property="oboInOwl:hasDbXref" content="PMID:13403276" datatype="xsd:string"></span><span id='am-248' property="oboInOwl:hasRelatedSynonym" content="Asteromyces" datatype="xsd:string"></span><span id='am-249' property="oboInOwl:hasRelatedSynonym" content="Asterococcus" datatype="xsd:string"></span><span id='am-250' property="oboInOwl:hasRelatedSynonym" content="Haemobartonella" datatype="xsd:string"></span><span id='am-251' property="oboInOwl:hasRelatedSynonym" content="Bovimyces" datatype="xsd:string"></span><span id='am-252' property="rdfs:label" content="Mycoplasma" datatype="xsd:string"></span>Mycoplasma</span> genitalium (1) <span id='am-474' about='obo:NCBITaxon_724' typeof='owl:Thing obo:NCBITaxon_135625 obo:NCBITaxon_712 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236'><span id='am-475' property="oboInOwl:hasDbXref" content="PMID:1736960" datatype="xsd:string"></span><span id='am-476' property="oboInOwl:hasRelatedSynonym" content="Haemophilus Winslow et al. 1917" datatype="xsd:string"></span><span id='am-477' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-478'  ></span><span id='am-479' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-480' property="rdfs:label" content="Haemophilus" datatype="xsd:string"></span>Haemophilus</span> in&#175;uenzae <br /> Although essential genes tend coded                                (2). Antisense RNA used speci&#174;cally silence genes <br /> leading replicating strand, underlying selective                            Staphylococccus aureus (3). These techniques fast and <br /> constraints evolutionary extent                                easy implement, especially bacteria hard to <br /> &#174;ndings subject comparative                              cultivate. However, require important resources, tend <br /> studies. Here, extend previous analysis                               over-represent number essential genes of <br /> the genomes low G + C &#174;rmicutes g-proteo-                                polarity transcription operons insertion biases. For <br /> bacteria, second step sequenced                                 example, studies identi&#174;ed 658 essential <br /> bacterial genomes. The inference essentiality                             genes S.aureus 478 H.in&#175;uenzae, precise <br />                                                                                 systematic gene inactivation larger genome free- <br /> homology allows essential genes <br />                                                                                 living Bacillus subtilis resulted identi&#174;cation only <br /> are frequent leading strand                               277 essential genes (4). In <span id='am-533' about='obo:NCBITaxon_561' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-534' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-535' property="rdfs:label" content="Escherichia" datatype="xsd:string"></span><span id='am-536' property="oboInOwl:hasDbXref" content="PMID:19700542" datatype="xsd:string"></span><span id='am-537' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-538' property="oboInOwl:hasRelatedSynonym" content="Escherchia" datatype="xsd:string"></span><span id='am-539'  ></span>Escherichia</span> coli, equivalent <br /> other genes, compared non-                                       published work unavailable, PEC database the <br /> essential highly expressed genes. Smaller biases                                Japan National Institute Genetics provides extensive <br /> were genomes obligatory intra-                                  information essentiality two-thirds genes. <br /> cellular bacteria, assignment essen-                           This database classi&#174;es 206 genes essential E.coli, of <br /> tiality homology fast growing free-living                               82% classi&#174;ed essential B.subtilis. The <br /> bacteria problematic. Cross-comparisons                                 study essentiality provides important information cell <br /> used assess potential errors assignment                            functioning; used develop new antibiotic targets, <br /> essentiality homology revealed that,                                 substantial evolutionary interest. For example, is <br /> cases, variations assignment criteria                               currently subject intense debate essentiality plays <br /> little in&#175;uence overall results. Essential                               important role constraining evolutionary rate of <br />                                                                                 proteins (5&#177;8). Also, patterns loss `essential&apos; genes in <br /> genes tend conserved leading <br />                                                                                 process genome degradation important the <br /> strand average genes, consistent with <br />                                                                                 understanding co-evolution bacteria their <br /> selection positioning impose                                 hosts (9). <br /> strong constraint chromosomal rearrangements.                                   The existence larger number genes leading <br /> These results indicate essentiality plays                                strand, relative lagging strand, known two <br /> fundamental role distribution genes                                decades. It &#174;rst observed rDNA ribosomal <br /> most bacterial genomes.                                                         proteins systematically coded leading strand the <br />                                                                                 E.coli chromosome (10). This speculated result from <br />                                                                                 selection prevent collisions DNA polymerase <br /> INTRODUCTION                                                                    (DNAP) RNA polymerase (RNAP) (11). DNA replication <br /> The concept `essential&apos; gene ambiguous,                      transcription occur simultaneously DNA <br /> genes needed basic supplies indispensable cell                       molecule. DNAP E.coli proceeds 10&#177;20 times faster than <br /> metabolites essential special circumstances.                          RNAP (12), head-on co-oriented collisions will <br /> However, uses medium enriched presumably                      occur replicating bacteria. However, outcome of <br /> necessary components, essentiality pertains core                  collision different depending poly- <br /> cell&apos;s functions, particular processes involved                    merases co-oriented not, resulting transcription <br /> information transfer compartmentalisation. The avail-                       abortion severe replication slow-down genes <br /> ability complete genome sequences high-throughput                        lagging strand (13,14). Thus, preferential <br />  <br /> *To correspondence addressed Atelier Bioinformatique, Universite&#194; Pierre et Marie Curie, 12 rue Cuvier, 75005 Paris, France. <br /> Tel: +33 1 44 27 65 36; Fax: +33 1 44 27 63 12; Email: <span id='am-560' about='obo:NCBITaxon_735486' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_95209 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-561' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-562'  ></span><span id='am-563' property="rdfs:label" content="Erocha" datatype="xsd:string"></span><span id='am-564' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>erocha</span>@abi.snv.jussieu.fr <br />  <br /> Nucleic Acids Research, Vol. 31 No. 22 &#227; Oxford University Press 2003; rights reserved <br />                                                                             Nucleic Acids Research, 2003, Vol. 31, No. 22       6571 <br />  <br />  <br /> positioning translation-related highly expressed genes           frequent leading strand genes. Thirdly, we <br /> (rDNA ribosomal proteins) leading strand             tested non-highly expressed essential genes biased <br /> been widely deemed result selection lower               highly expressed non-essential genes. Fourthly, tested <br /> transcription abortion higher replication rates. The            conservation essential genes leading strand is <br /> observations genes coding rDNA ribosomal               important genes. To check the <br /> proteins leading strand bacterial genomes        dif&#174;culties associated assignment essentiality by <br /> seemed con&#174;rm hypothesis (15,16). As selection        homology biasing results, set tests and <br /> only effective highly expressed genes acts        cross-comparisons available experimental <span id='am-601' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-602' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-603' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-604'  ></span><span id='am-605' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>data</span> on <br /> positioning genes leading strand, create     B.subtilis, E.coli, M.genitalium H.in&#175;uenzae. <br /> gene strand <span id='am-258' about='obo:NCBITaxon_272805' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1329799 obo:NCBITaxon_1 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_436489 obo:NCBITaxon_117570 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_9126 obo:NCBITaxon_33154 obo:NCBITaxon_8782 obo:NCBITaxon_436486 obo:NCBITaxon_1338369 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_461255 obo:NCBITaxon_8492 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_192204 obo:NCBITaxon_7711 obo:NCBITaxon_8825 obo:NCBITaxon_7776 obo:NCBITaxon_436492 obo:NCBITaxon_32561 obo:NCBITaxon_8457 obo:NCBITaxon_436491'><span id='am-259' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-260' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-261'  ></span><span id='am-262' property="rdfs:label" content="Bias" datatype="xsd:string"></span>bias</span> proportional expression level in <br /> replicating bacteria. The bias proportional       MATERIALS AND METHODS <br /> the frequency replication. Fast-growing bacteria should <br /> therefore higher strand bias.                                Data <br />    Several lines evidence recently challenged           Sequence data corresponding annotations taken <br /> emphasis model replication slow-down basis        GenBank Genomes (ftp://ftp.ncbi.nih.gov/genomes/ <br /> of gene strand bias. First, gene strand bias strongly depends    Bacteria/). The following genomes low G + C &#174;rmicutes <br /> the composition DNAP, bacteria containing               analysed: B.subtilis 168, <span id='am-606' about='obo:NCBITaxon_86665' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1386 obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-607' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-608' property="oboInOwl:hasDbXref" content="PMID:10591021" datatype="xsd:string"></span><span id='am-609'  ></span><span id='am-610' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-611' property="oboInOwl:hasDbXref" content="PMID:12382115" datatype="xsd:string"></span><span id='am-612' property="oboInOwl:hasRelatedSynonym" content="Bacillus alcalophilus subsp. halodurans" datatype="xsd:string"></span><span id='am-613' property="oboInOwl:hasAlternativeId" content="NCBITaxon:72574" datatype="xsd:string"></span><span id='am-614' property="rdfs:label" content="Bacillus halodurans" datatype="xsd:string"></span><span id='am-615' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1414" datatype="xsd:string"></span>Bacillus halodurans</span>, <br /> different a-subunits exhibiting higher strand biases (17).     <span id='am-108' about='obo:NCBITaxon_1488' typeof='owl:Thing obo:NCBITaxon_186802 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_186801 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_31979 obo:NCBITaxon_1485'><span id='am-109'  ></span><span id='am-110' property="oboInOwl:hasRelatedSynonym" content="Clostridium acetonobutylicum" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="PMID:7547288" datatype="xsd:string"></span><span id='am-113' property="rdfs:label" content="Clostridium acetobutylicum" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasRelatedSynonym" content="Clostridium acetobutyricum" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasRelatedSynonym" content="Granulobacter pectinovorum" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasDbXref" content="PMID:11411721" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasDbXref" content="PMID:11760952" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasDbXref" content="PMID:9103631" datatype="xsd:string"></span>Clostridium acetobutylicum</span> ATCC824, <span id='am-120' about='obo:NCBITaxon_1485' typeof='owl:Thing obo:NCBITaxon_186802 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_186801 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_31979'><span id='am-121' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-123' property="rdfs:label" content="Clostridium" datatype="xsd:string"></span><span id='am-124'  ></span>Clostridium</span> perfrin- <br /> In bacteria, a-subunit dedicated       gens 13, <span id='am-93' about='obo:NCBITaxon_1358' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1357 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_1300 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186826'><span id='am-94' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasRelatedSynonym" content="Lactococcus lactis (Lohnis) Schleifer et al. 1986" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-97'  ></span><span id='am-98' property="oboInOwl:hasRelatedSynonym" content="Streptococcus lactis" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasRelatedSynonym" content="Bacterium lacti" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasDbXref" content="PMID:11594628" datatype="xsd:string"></span><span id='am-101' property="rdfs:label" content="Lactococcus lactis" datatype="xsd:string"></span>Lactococcus lactis</span> IL1403, <span id='am-173' about='obo:NCBITaxon_1637' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186820'><span id='am-174' property="oboInOwl:hasDbXref" content="PMID:15709360" datatype="xsd:string"></span><span id='am-175' property="oboInOwl:hasRelatedSynonym" content="Listerella" datatype="xsd:string"></span><span id='am-176' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-177'  ></span><span id='am-178' property="oboInOwl:hasDbXref" content="PMID:8782674" datatype="xsd:string"></span><span id='am-179' property="oboInOwl:hasDbXref" content="PMID:10408878" datatype="xsd:string"></span><span id='am-180' property="oboInOwl:hasDbXref" content="PMID:9542083" datatype="xsd:string"></span><span id='am-181' property="rdfs:label" content="Listeria" datatype="xsd:string"></span><span id='am-182' property="oboInOwl:hasDbXref" content="PMID:1713054" datatype="xsd:string"></span><span id='am-183' property="oboInOwl:hasDbXref" content="PMID:9226919" datatype="xsd:string"></span><span id='am-184' property="oboInOwl:hasDbXref" content="PMID:8427807" datatype="xsd:string"></span><span id='am-185' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-186' property="oboInOwl:hasDbXref" content="PMID:1899799" datatype="xsd:string"></span>Listeria</span> inocua <br /> to replication DNA strand (18), average 78%        Clip11262, <span id='am-158' about='obo:NCBITaxon_1639' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_1637 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186820'><span id='am-159' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-160' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-161' property="oboInOwl:hasDbXref" content="PMID:8782698" datatype="xsd:string"></span><span id='am-162' property="oboInOwl:hasRelatedSynonym" content="Corynebacterium parvulum" datatype="xsd:string"></span><span id='am-163'  ></span><span id='am-164' property="oboInOwl:hasDbXref" content="PMID:17773427" datatype="xsd:string"></span><span id='am-165' property="oboInOwl:hasRelatedSynonym" content="Erysipelothrix monocytogenes" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasRelatedSynonym" content="Bacterium monocytogenes" datatype="xsd:string"></span><span id='am-167' property="oboInOwl:hasRelatedSynonym" content="Corynebacterium infantisepticum" datatype="xsd:string"></span><span id='am-168' property="rdfs:label" content="Listeria monocytogenes" datatype="xsd:string"></span><span id='am-169' property="oboInOwl:hasRelatedSynonym" content="Listerella hepatolytica" datatype="xsd:string"></span><span id='am-170' property="oboInOwl:hasRelatedSynonym" content="Bacterium monocytogenes hominis" datatype="xsd:string"></span><span id='am-171' property="oboInOwl:hasDbXref" content="PMID:1906732" datatype="xsd:string"></span><span id='am-172' property="oboInOwl:hasRelatedSynonym" content="Lysteria monocytogenes" datatype="xsd:string"></span>Listeria monocytogenes</span> EGD, <span id='am-1' about='obo:NCBITaxon_182709' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_186817 obo:NCBITaxon_1385 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-2' property="oboInOwl:hasDbXref" content="PMID:16014475" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="PMID:11750818" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="oboInOwl:hasDbXref" content="PMID:16403894" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="PMID:12054225" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Oceanobacillus" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Oceanobacillus</span> <br /> of genes coded leading strand,     iheyensis, <span id='am-25' about='obo:NCBITaxon_1279' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1385 obo:NCBITaxon_90964 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-26' property="oboInOwl:hasDbXref" content="PMID:9734063" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="PMID:12656157" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="PMID:10758876" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="PMID:10319469" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="Staphylococcus" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="PMID:10319495" datatype="xsd:string"></span><span id='am-33'  ></span><span id='am-34' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="PMID:17220435" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="PMID:10425778" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasRelatedSynonym" content="Aurococcus" datatype="xsd:string"></span>Staphylococcus</span> aureus N315, <span id='am-52' about='obo:NCBITaxon_1301' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_1300 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186826'><span id='am-53' property="oboInOwl:hasDbXref" content="PMID:8995803" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="PMID:19620365" datatype="xsd:string"></span><span id='am-56'  ></span><span id='am-57' property="rdfs:label" content="Streptococcus" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="PMID:1720654" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="PMID:7537076" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="PMID:14657115" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="PMID:10555340" datatype="xsd:string"></span>Streptococcus</span> aga- <br /> not highly expressed. Secondly, largest gene            lactiae serotype V, <span id='am-76' about='obo:NCBITaxon_1313' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1301 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_1300 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186826'><span id='am-77' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-79'  ></span><span id='am-80' property="rdfs:label" content="Streptococcus pneumoniae" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasRelatedSynonym" content="Micrococcus pneumoniae" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasRelatedSynonym" content="Diplococcus pneumoniae" datatype="xsd:string"></span>Streptococcus pneumoniae</span> TIGR4, <br /> strand biases slow-growing bacteria. Among             <span id='am-83' about='obo:NCBITaxon_1314' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1301 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_1300 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186826'><span id='am-84'  ></span><span id='am-85' property="oboInOwl:hasRelatedSynonym" content="Micrococcus scarlatinae" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasRelatedSynonym" content="Streptococcus scarlatinae" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="Streptococcus pyogenes" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasRelatedSynonym" content="Streptococcus hemolyticus" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasRelatedSynonym" content="Streptococcus pyrogenes" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasRelatedSynonym" content="Streptococcus erysipelatos" datatype="xsd:string"></span>Streptococcus pyogenes</span> M1 <span id='am-187' about='obo:NCBITaxon_1754' typeof='owl:Thing obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_186801 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_68295 obo:NCBITaxon_186814'><span id='am-188'  ></span><span id='am-189' property="oboInOwl:hasDbXref" content="PMID:17625170" datatype="xsd:string"></span><span id='am-190' property="oboInOwl:hasRelatedSynonym" content="Thermoanaerobium" datatype="xsd:string"></span><span id='am-191' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-192' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2324" datatype="xsd:string"></span><span id='am-193' property="rdfs:label" content="Thermoanaerobacter" datatype="xsd:string"></span><span id='am-194' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-195' property="oboInOwl:hasRelatedSynonym" content="Thermobacteroides" datatype="xsd:string"></span><span id='am-196' property="oboInOwl:hasDbXref" content="PMID:7981107" datatype="xsd:string"></span><span id='am-197' property="oboInOwl:hasRelatedSynonym" content="Clostridium cluster V" datatype="xsd:string"></span><span id='am-198' property="oboInOwl:hasExactSynonym" content="Acetigenium" datatype="xsd:string"></span>Thermoanaerobacter</span> teng- <br /> bacteria containing different DNAP a-subunits, slow-        congensis. The following genomes g-<span id='am-10' about='obo:NCBITaxon_1224' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2'><span id='am-11'  ></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="purple photosynthetic bacteria and relatives" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="purple non-sulfur bacteria" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="purple photosynthetic bacteria" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Proteobacteria" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="purple bacteria" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="PMID:16280474" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="PMID:11542017" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="purple bacteria and relatives" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="PMID:11321122" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasRelatedSynonym" content="proteobacteria" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="PMID:11837318" datatype="xsd:string"></span>proteobacteria</span> were <br /> growing M.genitalium highest biases (80%          analysed: E.coli MG1655, Buchnera Ap, H.in&#175;uenzae Rd, <br /> genes leading strand), fast-growing              <span id='am-481' about='obo:NCBITaxon_745' typeof='owl:Thing obo:NCBITaxon_135625 obo:NCBITaxon_712 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236'><span id='am-482' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-483'  ></span><span id='am-484' property="oboInOwl:hasDbXref" content="PMID:1736960" datatype="xsd:string"></span><span id='am-485' property="rdfs:label" content="Pasteurella" datatype="xsd:string"></span><span id='am-486' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Pasteurella</span> multocida PM70, <span id='am-398' about='obo:NCBITaxon_286' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_135621'><span id='am-399' property="oboInOwl:hasDbXref" content="PMID:23918787" datatype="xsd:string"></span><span id='am-400' property="oboInOwl:hasDbXref" content="PMID:10758879" datatype="xsd:string"></span><span id='am-401' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-402' property="rdfs:label" content="Pseudomonas" datatype="xsd:string"></span><span id='am-403' property="oboInOwl:hasDbXref" content="PMID:18048745" datatype="xsd:string"></span><span id='am-404' property="oboInOwl:hasRelatedSynonym" content="RNA similarity group I" datatype="xsd:string"></span><span id='am-405' property="oboInOwl:hasDbXref" content="PMID:15950132" datatype="xsd:string"></span><span id='am-406' property="oboInOwl:hasDbXref" content="PMID:10939664" datatype="xsd:string"></span><span id='am-407'  ></span><span id='am-408' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-409' property="oboInOwl:hasRelatedSynonym" content="Peudomonas" datatype="xsd:string"></span><span id='am-410' property="oboInOwl:hasDbXref" content="PMID:9103607" datatype="xsd:string"></span><span id='am-411' property="oboInOwl:hasRelatedSynonym" content="Liquidomonas" datatype="xsd:string"></span><span id='am-412' property="oboInOwl:hasRelatedSynonym" content="Loefflerella" datatype="xsd:string"></span><span id='am-413' property="oboInOwl:hasDbXref" content="PMID:7727274" datatype="xsd:string"></span>Pseudomonas</span> aeruginosa <br /> B.subtilis lowest (74%). For genomes           PA01, <span id='am-367' about='obo:NCBITaxon_303' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_286 obo:NCBITaxon_1236 obo:NCBITaxon_72274 obo:NCBITaxon_135621 obo:NCBITaxon_136845'><span id='am-368' property="oboInOwl:hasAlternativeId" content="NCBITaxon:79218" datatype="xsd:string"></span><span id='am-369' property="oboInOwl:hasAlternativeId" content="NCBITaxon:760256" datatype="xsd:string"></span><span id='am-370' property="oboInOwl:hasAlternativeId" content="NCBITaxon:29437" datatype="xsd:string"></span><span id='am-371' property="oboInOwl:hasAlternativeId" content="NCBITaxon:691260" datatype="xsd:string"></span><span id='am-372' property="oboInOwl:hasDbXref" content="PMID:9734035" datatype="xsd:string"></span><span id='am-373' property="oboInOwl:hasAlternativeId" content="NCBITaxon:126995" datatype="xsd:string"></span><span id='am-374' property="oboInOwl:hasAlternativeId" content="NCBITaxon:931964" datatype="xsd:string"></span><span id='am-375' property="oboInOwl:hasDbXref" content="PMID:17978216" datatype="xsd:string"></span><span id='am-376' property="oboInOwl:hasAlternativeId" content="NCBITaxon:931954" datatype="xsd:string"></span><span id='am-377' property="oboInOwl:hasAlternativeId" content="NCBITaxon:105847" datatype="xsd:string"></span><span id='am-378' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-379' property="oboInOwl:hasDbXref" content="PMID:1177780" datatype="xsd:string"></span><span id='am-380' property="oboInOwl:hasRelatedSynonym" content="Bacillus fluorescens putidus" datatype="xsd:string"></span><span id='am-381' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas sp. PX-A-Ps.sp2" datatype="xsd:string"></span><span id='am-382' property="oboInOwl:hasAlternativeId" content="NCBITaxon:1416793" datatype="xsd:string"></span><span id='am-383'  ></span><span id='am-384' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas incognita" datatype="xsd:string"></span><span id='am-385' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas eisenbergii" datatype="xsd:string"></span><span id='am-386' property="oboInOwl:hasAlternativeId" content="NCBITaxon:668614" datatype="xsd:string"></span><span id='am-387' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas striata" datatype="xsd:string"></span><span id='am-388' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas rugosa" datatype="xsd:string"></span><span id='am-389' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas convexa" datatype="xsd:string"></span><span id='am-390' property="oboInOwl:hasRelatedSynonym" content="Bacillus putidus" datatype="xsd:string"></span><span id='am-391' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-392' property="oboInOwl:hasRelatedSynonym" content="Arthrobacter siderocapsulatus" datatype="xsd:string"></span><span id='am-393' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas ovalis" datatype="xsd:string"></span><span id='am-394' property="oboInOwl:hasRelatedSynonym" content="Pseudomonas arvilla" datatype="xsd:string"></span><span id='am-395' property="oboInOwl:hasRelatedSynonym" content="Pseudomanas putida" datatype="xsd:string"></span><span id='am-396' property="oboInOwl:hasDbXref" content="PMID:11211254" datatype="xsd:string"></span><span id='am-397' property="rdfs:label" content="Pseudomonas putida" datatype="xsd:string"></span>Pseudomonas putida</span> KT2440, <span id='am-540' about='obo:NCBITaxon_590' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-541' property="oboInOwl:hasDbXref" content="PMID:15653929" datatype="xsd:string"></span><span id='am-542' property="oboInOwl:hasDbXref" content="PMID:12072558" datatype="xsd:string"></span><span id='am-543' property="oboInOwl:hasDbXref" content="PMID:10319519" datatype="xsd:string"></span><span id='am-544' property="oboInOwl:hasDbXref" content="PMID:10939679" datatype="xsd:string"></span><span id='am-545' property="rdfs:label" content="Salmonella" datatype="xsd:string"></span><span id='am-546' property="oboInOwl:hasRelatedSynonym" content="Samonella" datatype="xsd:string"></span><span id='am-547' property="oboInOwl:hasDbXref" content="PMID:15653930" datatype="xsd:string"></span><span id='am-548' property="oboInOwl:hasDbXref" content="PMID:3231714" datatype="xsd:string"></span><span id='am-549' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-550' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-551' property="oboInOwl:hasDbXref" content="PMID:9731304" datatype="xsd:string"></span><span id='am-552'  ></span>Salmonella</span> enterica <br /> replicated gene coding DNAP a-subunit,        Typhimurium LT2, <span id='am-310' about='obo:NCBITaxon_211586' typeof='owl:Thing obo:NCBITaxon_267890 obo:NCBITaxon_1 obo:NCBITaxon_70863 obo:NCBITaxon_22 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236 obo:NCBITaxon_135622'><span id='am-311' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-312' property="rdfs:label" content="Shewanella oneidensis MR-1" datatype="xsd:string"></span><span id='am-313' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-314' property="oboInOwl:hasAlternativeId" content="NCBITaxon:60479" datatype="xsd:string"></span><span id='am-315' property="oboInOwl:hasExactSynonym" content="Shewanella oneidensis str. MR-1" datatype="xsd:string"></span><span id='am-316' property="oboInOwl:hasExactSynonym" content="Shewanella oneidensis strain MR-1" datatype="xsd:string"></span><span id='am-317' property="oboInOwl:hasRelatedSynonym" content="Shewanella sp. MR-1" datatype="xsd:string"></span><span id='am-318' property="oboInOwl:hasRelatedSynonym" content="Shewanella oneidensis ATCC 700550" datatype="xsd:string"></span>Shewanella oneidensis MR-1</span>, <br /> the slow-growing <span id='am-437' about='obo:NCBITaxon_139' typeof='owl:Thing obo:NCBITaxon_203691 obo:NCBITaxon_1 obo:NCBITaxon_64895 obo:NCBITaxon_203692 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_137 obo:NCBITaxon_136 obo:NCBITaxon_138'><span id='am-438' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-439' property="oboInOwl:hasDbXref" content="PMID:9336916" datatype="xsd:string"></span><span id='am-440' property="oboInOwl:hasExactSynonym" content="Borrelia burgdorferi sensu stricto" datatype="xsd:string"></span><span id='am-441' property="oboInOwl:hasExactSynonym" content="Lyme disease spirochete" datatype="xsd:string"></span><span id='am-442' property="oboInOwl:hasDbXref" content="PMID:7699027" datatype="xsd:string"></span><span id='am-443' property="oboInOwl:hasDbXref" content="PMID:7981102" datatype="xsd:string"></span><span id='am-444' property="oboInOwl:hasRelatedSynonym" content="Borrelia burgdorffragment" datatype="xsd:string"></span><span id='am-445' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-446' property="rdfs:label" content="Borrelia burgdorferi" datatype="xsd:string"></span><span id='am-447' property="oboInOwl:hasDbXref" content="PMID:8995796" datatype="xsd:string"></span><span id='am-448' property="oboInOwl:hasDbXref" content="PMID:1503971" datatype="xsd:string"></span><span id='am-449'  ></span><span id='am-450' property="oboInOwl:hasDbXref" content="PMID:1380285" datatype="xsd:string"></span><span id='am-451' property="oboInOwl:hasRelatedSynonym" content="Borrelia burdorferi" datatype="xsd:string"></span>Borrelia burgdorferi</span> shows highest bias        <span id='am-297' about='obo:NCBITaxon_53413' typeof='owl:Thing obo:NCBITaxon_32033 obo:NCBITaxon_135614 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_338 obo:NCBITaxon_1236'><span id='am-298' property="oboInOwl:hasDbXref" content="PMID:2275852" datatype="xsd:string"></span><span id='am-299' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-300' property="oboInOwl:hasRelatedSynonym" content="Xanthomonas axonoperis" datatype="xsd:string"></span><span id='am-301' property="rdfs:label" content="Xanthomonas axonopodis" datatype="xsd:string"></span><span id='am-302' property="oboInOwl:hasDbXref" content="PMID:10758892" datatype="xsd:string"></span><span id='am-303' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-304'  ></span>Xanthomonas axonopodis</span>, <span id='am-343' about='obo:NCBITaxon_338' typeof='owl:Thing obo:NCBITaxon_32033 obo:NCBITaxon_135614 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236'><span id='am-344' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-345' property="oboInOwl:hasDbXref" content="PMID:11931143" datatype="xsd:string"></span><span id='am-346' property="rdfs:label" content="Xanthomonas" datatype="xsd:string"></span><span id='am-347' property="oboInOwl:hasDbXref" content="PMID:2275852" datatype="xsd:string"></span><span id='am-348' property="oboInOwl:hasDbXref" content="PMID:9103617" datatype="xsd:string"></span><span id='am-349' property="oboInOwl:hasDbXref" content="PMID:18048743" datatype="xsd:string"></span><span id='am-350' property="oboInOwl:hasDbXref" content="PMID:10758874" datatype="xsd:string"></span><span id='am-351'  ></span><span id='am-352' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Xanthomonas</span> campestris, <span id='am-223' about='obo:NCBITaxon_2370' typeof='owl:Thing obo:NCBITaxon_32033 obo:NCBITaxon_135614 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_1236'><span id='am-224' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-225' property="rdfs:label" content="Xylella" datatype="xsd:string"></span><span id='am-226' property="oboInOwl:hasRelatedSynonym" content="Xylella Wells et al. 1987" datatype="xsd:string"></span><span id='am-227'  ></span><span id='am-228' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span>Xylella</span> <br /> (65%). Comparing equally sized chromosomes, &#174;nds           fastidiosa 9a5c <span id='am-287' about='obo:NCBITaxon_214092' typeof='owl:Thing obo:NCBITaxon_629 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236 obo:NCBITaxon_632'><span id='am-288' property="oboInOwl:hasExactSynonym" content="Yersinia pestis strain CO92" datatype="xsd:string"></span><span id='am-289' property="rdfs:label" content="Yersinia pestis CO92" datatype="xsd:string"></span><span id='am-290' property="oboInOwl:hasDbXref" content="PMID:11586360" datatype="xsd:string"></span><span id='am-291' property="oboInOwl:hasExactSynonym" content="Yersinia pestis str. CO92" datatype="xsd:string"></span><span id='am-292' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-293' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>Yersinia pestis CO92</span>. Because different <br /> the slow-growing <span id='am-199' about='obo:NCBITaxon_1763' typeof='obo:NCBITaxon_85007 owl:Thing obo:NCBITaxon_1762 obo:NCBITaxon_1 obo:NCBITaxon_1760 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_2037 obo:NCBITaxon_201174 obo:NCBITaxon_85003'><span id='am-200' property="rdfs:label" content="Mycobacterium" datatype="xsd:string"></span><span id='am-201' property="oboInOwl:hasDbXref" content="PMID:16014496" datatype="xsd:string"></span><span id='am-202' property="oboInOwl:hasDbXref" content="PMID:7547304" datatype="xsd:string"></span><span id='am-203' property="oboInOwl:hasDbXref" content="PMID:1581193" datatype="xsd:string"></span><span id='am-204' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-205'  ></span><span id='am-206' property="oboInOwl:hasRelatedSynonym" content="Mycobacterium Lehmann and Neumann 1896" datatype="xsd:string"></span><span id='am-207' property="oboInOwl:hasDbXref" content="PMID:7907223" datatype="xsd:string"></span><span id='am-208' property="oboInOwl:hasDbXref" content="PMID:1883713" datatype="xsd:string"></span><span id='am-209' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-210' property="oboInOwl:hasDbXref" content="PMID:8863452" datatype="xsd:string"></span><span id='am-211' property="oboInOwl:hasDbXref" content="PMID:1380284" datatype="xsd:string"></span><span id='am-212' property="oboInOwl:hasDbXref" content="PMID:7547284" datatype="xsd:string"></span><span id='am-213' property="oboInOwl:hasDbXref" content="PMID:2275850" datatype="xsd:string"></span>Mycobacterium</span> tuberculosis (59%)           strains <span id='am-329' about='obo:NCBITaxon_species' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-330' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-331' property="rdfs:label" content="species" datatype="xsd:string"></span>species</span> co-linear, one <br /> biased fast-growing E.coli (55%). Thirdly,         strain species analysed. Essentiality data were <br /> recently shown gene strand bias B.subtilis E.coli       taken literature following genomes: B.subtilis <br /> stems `essentiality&apos; genes. Controlling         (4), M.genitalium (1), H.in&#175;uenzae (2) E.coli (http:// <br /> essentiality revealed signi&#174;cant role expression levels   www.shigen.nig.ac.jp/ecoli/pec/). Identi&#174;cation origins of <br /> gene strand bias (19). This suggests anti-oriented             replication leading strands use of <br /> collisions selected problems associated          linear discriminant analysis GC skews, detailed <br /> with existence aborted transcripts, replication          (23). <br /> pausing. It suggests essentiality major deter- <br /> minant chromosome structure. Rearrangements result-          Analysis orthology <br /> ing important inversions leading strands            Orthologues identi&#174;ed reciprocal best hits. <br /> extremely deleterious (20,21). The strand switch essential       First, BlastP search identi&#174;ed 10 best hits protein <br /> genes partly responsible observations.             genome. These proteins aligned using a <br />    To substantiate previous observation concerning essen-       global alignment end gaps facing largest <br /> tial genes, limited fast-growing bacteria     sequence ignored. Two proteins regarded as <br /> similar genome sizes, tested essentiality plays similar     orthologues reciprocal best hits least <br /> role bacterial genomes. Putative essentiality          40% similarity amino acid sequence &lt;20% of <br /> identi&#174;ed homology genomes B.subtilis (for           difference protein length (24). Essentiality derived by <br /> low G + C &#174;rmicutes) E.coli (for g-proteobacteria). As         homology B.subtilis genomes low G + C <br /> working hypothesis, simplifying assump-             &#174;rmicutes E.coli genomes g-proteobacteria. <br /> tions. First, de&#174;ned set putative orthologues        In second step, analysed set complete bacterial <br /> genomes, using reciprocal best hits. Secondly, assumed           genomes origin terminus replication can <br /> that orthologues essential genes likely       reliably identi&#174;ed. In case, essentiality derived by <br /> essential genomes. Thirdly, de&#174;ned putative         homology B.subtilis subset. The classi&#174;cations the <br /> highly expressed genes genome <span id='am-319' about='obo:NCBITaxon_79338' typeof='owl:Thing obo:NCBITaxon_78536 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_3398 obo:NCBITaxon_4130 obo:NCBITaxon_4069 obo:NCBITaxon_1437183 obo:NCBITaxon_33090 obo:NCBITaxon_91888 obo:NCBITaxon_91827 obo:NCBITaxon_131221 obo:NCBITaxon_71240 obo:NCBITaxon_58024 obo:NCBITaxon_1437201 obo:NCBITaxon_58023 obo:NCBITaxon_2759 obo:NCBITaxon_35493 obo:NCBITaxon_71274 obo:NCBITaxon_3193'><span id='am-320' property="rdfs:label" content="Codon" datatype="xsd:string"></span><span id='am-321' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-322' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-323'  ></span>codon</span> adaptation         E.coli B.subtilis genes used reference be <br /> index (CAI) values computed ribosomal proteins (22). At        obtained http://wwwabi.snv.jussieu.fr/~erocha/essential/ <br /> this point, tested different hypotheses concerning      Supplementary Material NAR Online. <br /> higher frequency genes leading strand. First, we <br /> tested essential genes frequent leading     Analysis codon usage <br /> strands non-essential genes. Secondly, tested      CAI values computed using EMBOSS package <br /> the non-essential genes, highly expressed genes            (http://www.uk.embnet.org/Software/EMBOSS). The reference <br />  6572   Nucleic Acids Research, 2003, Vol. 31, No. 22 <br />  <br />  <br /> values codon usage highly expressed genes (understand-      two-tailed c2 tests 2 3 2 contingency tables the <br /> ing high expression cells growing exponen-       comparisons <span id='am-294' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-295' property="rdfs:label" content="class" datatype="xsd:string"></span><span id='am-296' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>class</span> orthologous <br /> tially) computed using ribosomal proteins (22). It       essential genes versus remaining orthologues. Because for <br /> recently shown yeast correlation CAI     type tests multiple tests being <br /> (de&#174;ned using ribosomal proteins) transcriptome data      (one genome), used Bonferroni correction <br /> good, good different transcriptome data sets        taxonomic group. Thirdly, test all <br /> among (under experimental conditions)         sequenced genomes group genes biased than <br /> (25). Another recent work indicated CAI values      another, performed single paired t-test. In case, <br /> robust small changes learning set (26). We         test carried hypothesis no <br /> previously checked transcriptome proteome data,          Bonferroni correction used. <br /> when available, con&#174;rmed analysis gene strand bias <br /> made CAI values (19). Here, gene regarded as <br /> potentially highly expressed CAI 10% <br /> highest values genome. The use 10% threshold       RESULTS <br /> somewhat arbitrary, re&#175;ects results classi- <br /> &#174;cations authors, using CAI factorial          Distribution essential genes replicating <br /> analysis, indicating 5&#177;10% genes particularly         strands <br /> highly expressed (27&#177;29). We shown previously           Low G + C &#174;rmicutes. We explored link between <br /> varying threshold range 5&#177;15% did change      essentiality gene strand bias genomes closely <br /> results signi&#174;cantly (19).                                        related low G + C &#174;rmicutes (excluding distant <br />                                                                   Mycoplasmas). For this, identi&#174;ed orthologues between <br /> Tests CAI <br />                                                                   B.subtilis genomes, assigning essentiality <br /> Codon usage bias good estimator gene              according B.subtilis classi&#174;cation. Then performed <br /> expression bacteria (30). Hence, designed set     tests. First, tested essential genes more <br /> tests verify codon usage ribosomal proteins       frequently leading strand non-essential <br /> signi&#174;cantly biased, codon usage classi&#174;cation      genes. This case genomes (Table 1). <br /> the genome using set discriminant using       Therefore, essentiality plays important role gene strand <br /> random set genes. For this, 100 random                 bias low G + C &#174;rmicutes. Secondly, tested the <br /> experiments. In each, randomly took 50 genes (corres-          non-essential genes, highly expressed genes frequent <br /> ponding number ribosomal proteins used           leading strand non-highly expressed genes. This <br /> previous analysis) build reference table codon usage.   test identi&#174;ed case signi&#174;cantly higher frequency <br /> Using table, computed CAI values genes    highly expressed genes leading strand <br /> then frequency 50 initial genes 10%          (C.acetobutylicum) (Table 1). Therefore, expression levels <br /> highest CAI. A CAI indicating signi&#174;cant codon usage bias         contribute signi&#174;cantly gene strand bias, <br /> should high consistency (i.e. genes initially    essentiality taken account. Thirdly, tested <br /> hypothesised highly expressed classed          essential non-highly expressed genes biased <br /> highly expressed) genome average value smaller         non-essential highly expressed genes. With the <br /> expected chance (to discriminate minority highly        above-mentioned exception C.acetobutylicum, which <br /> expressed genes rest genome). Signi&#174;cant          difference statistically signi&#174;cant, test showed <br /> codon usage bias ribosomal proteins met conditions.      generally true. <br /> First, &gt;40% ribosomal proteins classed highly <br /> expressed 10% CAI values. Secondly, frequency      g-Proteobacteria. The analysis g- <br /> of ribosomal proteins classed 10% CAI values      proteobacteria, starting PEC database E.coli <br /> be signi&#174;cantly higher obtained random        essential genes. This database contains 1700 genes for <br /> experiments genes used build codon index           essentiality character unknown, were <br /> (signi&#174;cant difference taken &gt;3 SDs away       removed simplicity. The resulting sample contains 2410 <br /> mean random experiments). Thirdly, average CAI      E.coli genes, used identify orthologous genes in <br /> all genes, CAI computed using ribosomal proteins,     g-proteobacteria. Because incomplete classi- <br /> must signi&#174;cantly lower obtained random       &#174;cation lack extensive experimental gene <br /> experiments (&gt;3 SDs away).                                        disruption programme E.coli, analysis more <br />                                                                   error prone. Still, results qualitatively similar those <br /> Statistical tests <br />                                                                   observed low G + C &#174;rmicutes (Table 1). With the <br /> We performed types statistical tests adapted   exception Buchnera, genomes high relative <br /> different situations. To test groups different        frequencies leading strand essential genes. When analysing <br /> frequency leading strand genes genome, carried     non-essential genes gene strand bias function of <br /> two-tailed c2 tests 2 3 2 contingency tables. Strand           expression levels, genomes higher biases among <br /> conservation groups genes genomes               highly expressed genes, genomes lower biases <br /> computed frequency orthologues          highly expressed genes genomes non- <br /> same replicating strand genomes. We tested    signi&#174;cant differences. Thus, non-essential genes, <br /> value different expected given average           expression does play signi&#174;cant role gene <br /> strand conservation orthologues. This test consisted    strand bias. Finally, genomes showed a <br />                                                                                               Nucleic Acids Research, 2003, Vol. 31, No. 22                 6573 <br />  <br />  <br /> Table 1. Distribution genes leading strands genomes <br /> Genome                  Genes                                           % Genes lead                                 Statistical tests <br />                         EH           EnH        nEH        nEnH         EH      EnH          nEH        nEnH         Ess        Exp      EvH         SC       CB <br />  <br /> B.subtilis               78          199         119       3710          94       94         72         74           +          0        +                    + <br /> B.halodurans            240           15        1503        402          93       93         81         79           +          0        +           +        + <br /> C.acetobutylicum        218           10         918        251          93       91         91         80           +          +        0           +        + <br /> C.perfringens           208            9         770        196          95       93         88         85           +          0        +           +        + <br /> L.lactis                225           11         698        186          88       89         74         81           +          0        +           +        + <br /> L.inocua                242           16        1069        255          95       94         75         80           +          0        +           +        + <br /> L.monocytogenes         243           18        1035        243          96       93         73         79           +          0        +           +        + <br /> O.iheyensis             248           14        1366        395          96       93         79         76           +          0        +           +        + <br /> S.agalactiae            219           11         665        186          93       92         76         79           +          0        +           +        + <br /> S.aureus                237           13         956        261          95       89         75         75           +          0        +           +        + <br /> S.pneumoniae            214           10         654        153          93       90         72         80           +          0        +           +        + <br /> S.pyogenes              208            9         575        131          92       93         78         80           +          0        +           +        + <br /> T.tengcongensis         212           11         771        166         100       94         90         87           +          0        +           0        0 <br />  <br /> E.coli                  112           94         249       1955          80       71         57         56           +          0        +                    + <br /> Buchnera Ap             127           33         169        116          70       47         73         61           0          0        &#177;           0        0 <br /> H.in&#175;uenzae             149           44         389        289          79       67         52         51           +          0        +           +        + <br /> P.multocida             152           44         450        318          67       72         65         54           +          0        +           +        + <br /> P.aeruginosa            150           40         628        437          75       82         51         59           +          &#177;        +           +        + <br /> P.putida                145           39         615        422          75       78         54         61           +          &#177;        +           +        + <br /> S.enterica              154           49         967        718          80       70         57         57           +          &#177;        +           +        + <br /> S.oneidensis            148           40         569        393          81       72         62         57           +          +        +           +        + <br /> X.axonopodis            145           35         474        323          79       77         56         60           +          0        +           +        + <br /> X.campestris            141           34         482        321          79       79         53         59           +          0        +           +        + <br /> X.fastidiosa            139           35         342        216          99       63         98         51           +          +        &#177;           +        + <br /> Y.pestis                156           46         730        567          80       74         59         53           +          +        +           +        + <br />  <br /> The classi&#174;cation low G + C &#174;rmicutes homology B.subtilis g-proteobacteria E.coli. The genes classed according <br /> to expression level essential phenotype highly expressed (H) versus non-highly expressed (nH) essential (E) versus non-essential (nE). The <br /> acronyms concatenated classes (e.g. EnH indicates essential non-highly expressed genes). The columns `Statistical tests&apos; (two-tail c2 <br /> tests contingency tables) indicate signi&#174;cance test P &lt; 0.05, applying Bonferroni correction. The + &#177; signs indicate rejection the <br /> hypothesis (right- left-side, respectively), 0 indicates rejection. The tests correspond to: E &#185; nE (Ess), nEH &#185; nEnH (Exp), EnH &#185; nEH (EvH), <br /> larger strand conservation essential genes (SC) signi&#174;cant codon usage bias ribosomal proteins (CB). <br />  <br />  <br />  <br />  <br /> signi&#174;cantly higher frequency leading strand essential                      (Buchnera, <span id='am-459' about='obo:NCBITaxon_810' typeof='obo:NCBITaxon_204428 owl:Thing obo:NCBITaxon_204429 obo:NCBITaxon_809 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_1113537 obo:NCBITaxon_51290 obo:NCBITaxon_51291'><span id='am-460' property="oboInOwl:hasDbXref" content="PMID:11211265" datatype="xsd:string"></span><span id='am-461' property="oboInOwl:hasDbXref" content="PMID:9103637" datatype="xsd:string"></span><span id='am-462' property="rdfs:label" content="Chlamydia" datatype="xsd:string"></span><span id='am-463' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-464' property="oboInOwl:hasDbXref" content="PMID:10319462" datatype="xsd:string"></span><span id='am-465' property="oboInOwl:hasDbXref" content="PMID:21048221" datatype="xsd:string"></span><span id='am-466' property="oboInOwl:hasRelatedSynonym" content="Bedsonia" datatype="xsd:string"></span><span id='am-467'  ></span><span id='am-468' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-469' property="oboInOwl:hasRelatedSynonym" content="Rakeia" datatype="xsd:string"></span><span id='am-470' property="oboInOwl:hasRelatedSynonym" content="Miyagawanella" datatype="xsd:string"></span><span id='am-471' property="oboInOwl:hasRelatedSynonym" content="Rickettsiaformis" datatype="xsd:string"></span><span id='am-472' property="oboInOwl:hasDbXref" content="PMID:9103632" datatype="xsd:string"></span><span id='am-473' property="oboInOwl:hasDbXref" content="PMID:21048222" datatype="xsd:string"></span>Chlamydia</span>, Spirochetes <span id='am-498' about='obo:NCBITaxon_780' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_33988 obo:NCBITaxon_28211 obo:NCBITaxon_766 obo:NCBITaxon_775'><span id='am-499' property="oboInOwl:hasDbXref" content="PMID:9103608" datatype="xsd:string"></span><span id='am-500' property="oboInOwl:hasDbXref" content="PMID:10939649" datatype="xsd:string"></span><span id='am-501' property="rdfs:label" content="Rickettsia" datatype="xsd:string"></span><span id='am-502' property="oboInOwl:hasRelatedSynonym" content="Rickettsia da Rocha-Lima 1916" datatype="xsd:string"></span><span id='am-503' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-504' property="oboInOwl:hasDbXref" content="PMID:11491333" datatype="xsd:string"></span><span id='am-505' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-506'  ></span>Rickettsia</span>), also <br /> non-highly expressed genes non-essential highly                           obligatory pathogen (X.fastidiosa). <br /> expressed genes. The exceptions trend non- <br /> free-living Bacteria Buchnera X.fastidiosa, not <br />                                                                                    The positioning essential genes leading strand <br /> showing suf&#174;cient codon usage bias reliably determine <br />                                                                                    highly conserved <br /> expression levels. <br />                                                                                    These observations suggest essentiality important in <br /> Extension procedure. The assignment essentiality                      low G + C &#174;rmicutes g-proteobacteria, possibly <br /> homology distant phylogenetic domains prone                        bacterial domains. However, closely related genomes <br /> more important errors, essentiality relative set                   resemble E.coli B.subtilis phylogenetic <br /> of functions lifestyle de&#174;nition                            inertia, common selective pressure a <br /> orthology increasingly error prone evolutionary                         positioning essential genes leading strand. Some <br /> distances increase. In cases, de&#174;nition                       close genomes, E.coli Y.pestis (31) or <br /> high expression based CAI problematic. Still,                    S.agalactiae S.pneumoniae (32), severely <br /> made preliminary analysis entire set sequenced                         shuf&#175;ed speciation. Interestingly, genome rearrange- <br /> bacteria, including 30 fully sequenced genomes                            ments tend symmetrical origin replication <br /> different species. For genome, determined                              change replicating strand gene is <br /> orthologues B.subtilis. These orthologues classed                     coded, disrupt gene <span id='am-332' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-333' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-334' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> (33,34). Therefore, <br /> as essential non-essential according classi&#174;cation                  performed test determine strand <br /> B.subtilis. The results qualitatively similar                        conservation stronger essential genes among <br /> presented above. First, essential genes biased                       genes (SC column Table 1). The results indicate that <br /> non-essential genes nearly genomes (Fig. 1). Secondly,                      21 23 genomes, conservation leading strand <br /> among highly expressed genes, essential genes                           character essential genes signi&#174;cantly important <br /> more biased non-essential ones. Thirdly, highly expres-                       that genes (P &lt; 0.05, applying the <br /> sed non-essential genes biased essential                     Bonferroni correction multiple tests) (Table 1). The two <br /> highly expressed genes. In analysis, observed                        exceptions concern Buchnera T.tengcongensis. In the <br /> some exceptions, concerning intracellular bacteria                          latter, conservation important but, genome <br />  6574     Nucleic Acids Research, 2003, Vol. 31, No. 22 <br />  <br />  <br />  <br />  <br />                                                                                  Figure 2. Comparison essential genes data set E.coli and <br />                                                                                  B.subtilis. Upper panel: assignment essentiality orthologues (black <br />                                                                                  squares represent essential genes B.subtilis). Lower panel: presence or <br />                                                                                  absence orthologues depending classi&#174;cation essentiality the <br />                                                                                  genomes (black squares represent genes present B.subtilis absent <br />                                                                                  E.coli). The area blocks proportional relative frequency of <br />                                                                                  category. <br />  <br />  <br />                                                                                  Methodological problems lead erroneous infer- <br />                                                                                  ence orthology gene duplication non- <br />                                                                                  orthologous gene displacement (36). Hence, important <br />                                                                                  test robustness assumptions concerning <br />                                                                                  essentiality. The low G + C &#174;rmicutes sharing fewer <br />                                                                                  orthologues B.subtilis essential genes S.pyogenes <br />                                                                                  C.acetobutylicum (Table 1). Yet, cases, could <br />                                                                                  identify orthologues 78% B.subtilis essential genes. <br /> Figure 1. Frequency leading strand genes 53 genomes bacteria. For <br />                                                                                  Among g-proteobacteria, genome containing fewer <br /> the bacteria chromosome, pooled               orthologues E.coli essential genes Buchnera. Buchnera <br /> chromosomes. Essentiality assigned homology B.subtilis,           relatively close E.coli, evolutionary history of <br /> only genes presenting orthologous gene B.subtilis considered        extensive genome reduction led genome only <br /> the analysis. (A) Essential versus non-essential genes. (B) Essential versus     approximately 500 genes (37). Although 13% the <br /> non-essential genes putative highly expressed genes. (C) Highly <br /> expressed non-essential genes versus essential non-highly expressed genes.       E.coli non-essential genes orthologue Buchnera, <br /> The diagonal line indicates similar frequency sets. Intracellular   77% essential genes orthologue. These <br /> obligatory pathogens symbionts indicated open circles          results indicate majority essential genes are <br /> other genomes circles. In panels, essential genes     conserved evolutionary distances, very <br /> biased set (P &lt; 0.001, paired t-tests). <br />                                                                                  genes conserved. We turned comparison of <br />                                                                                  distantly related E.coli B.subtilis genomes. <br />  <br /> shows highest overall strand bias (35), difference                Comparison essential genes B.subtilis E.coli. Among <br /> signi&#174;cant application Bonferroni correction.                       1129 orthologues E.coli B.subtilis, found <br />                                                                                  75% B.subtilis essential genes 26% non- <br /> Tests                                                                            essential genes (Fig. 2). Thus, expected, presence of <br /> The previous analyses strongly suggest majority                   essential genes conserved other <br /> bacteria, essentiality drives gene strand bias. Yet, assigning                   genes large evolutionary distances. Further, 82% the <br /> essentiality homology prone errors. For example,                   E.coli essential genes classed essential B.subtilis <br /> all genomes lack orthologues essential B.subtilis                    (and 7% non-essential genes regarded essential <br /> genes, indicating non-essentiality bacteria.                      B.subtilis) (Fig. 2). Given large evolutionary <br />                                                                                          Nucleic Acids Research, 2003, Vol. 31, No. 22                6575 <br />  <br />  <br />                      Table 2. Cross-comparisons frequency genes leading strand using experimental data <br />                      sets essential genes genomes <br />                                            Experimental data              Homology                        Both <br />                                            Essential   Non-essential      Essential     Non-essential     Essential     Non-essential <br />  <br />                      M.genitalium (Bs)     82%          75%               85%           78%               86%           77% <br />                      H.in&#175;uenzae (Ec)      55%          55%               70%           51%               73%           53% <br />                      B.subtilis (Ec)       94%          72%               97%           83%               97%           80% <br />                      E.coli (Bs)           76%          56%               75%           56%               84%           56% <br />  <br />                      Three types information essentiality used compute frequency leading strand essential <br />                      genes: (i) using experimental data set genome; (ii) homology data set genome <br />                      (Bs, B.subtilis; Ec, E.coli); (iii) making intersection (i) (ii). For example, M.genitalium shows <br />                      82, 85 86% essential genes leading strand essentiality de&#174;ned respectively its <br />                      experimental data, homology essential genes B.subtilis, using intersection two <br />                      criteria (essential B.subtilis M.genitalium). <br />  <br />  <br />  <br /> distance bacteria, &#174;nding                        Table 3. Percentage genes leading strand genomes, <br /> large fraction essential genes conserved                      according essentiality <br /> genomes suggests assigning essentiality homology                    Comparison                             Leading       Leading       Expected <br /> reasonable, free-living bacteria.                                                                    essential     others <br />  <br /> Comparisons experimental data sets. We used                        B.subtilis versus M.genitalium         85%           70%           60% <br /> experimental data sets essential genes B.subtilis, E.coli,               B.subtilis versus E.coli               73%           44%           41% <br />                                                                                E.coli versus H.in&#175;uenzae              60%           35%           35% <br /> M.genitalium H.in&#175;uenzae make set cross- <br /> comparisons. In comparison, different                       A gene regarded essential experimentally essential <br /> analyses (Table 2). The &#174;rst analysis consists simply                       genomes. <br /> computing frequency leading strand essential non- <br /> essential genes genome, essentiality taken from <br /> the experimental data set [e.g. M.genitalium data set <br /> in analysis `M.genitalium (Bs)&apos;]. The second analysis                      leading strand character essential genes. For this, we <br /> consists computing frequency leading strand essential                compared B.subtilis genomes M.genitalium and <br /> genes genome essentiality assigned                          E.coli, H.in&#175;uenzae. We that <br /> homology genome [e.g. assigned                             genes essential genomes tend co-occur the <br /> homology B.subtilis analysis `M.genitalium (Bs)&apos;].                   leading strand higher frequency other <br /> The analysis consists computing frequency                      genes (Table 3). For example, comparison B.subtilis <br /> leading strand essential genes essentiality assigned                  E.coli, gene order <br /> only gene essential experimental data set              conservation (39), shows 73% essential genes leading <br /> its orthologue essential genome [e.g. gene              strand genomes. The similar analysis applied the <br /> is experimentally essential M.genitalium                   remaining genes indicates 44% genes conserved the <br /> and B.subtilis analysis `M.genitalium (Bs)&apos;]. The                    leading strand, expected frequency, taking into <br /> three analyses consistently results,                        account average gene strand composition genome, <br /> essential genes frequent leading strand                      41%. Between B.subtilis M.genitalium, difference is <br /> independently information used de&#174;ne essentiality.                   smaller (85 versus 70%), expected value much <br /> Only analysis H.in&#175;uenzae&apos;s experimental data set                   larger (60%). This conservation positioning essential <br /> indicates signi&#174;cant difference essential non-                  genes leading strand consistent existence of <br /> essential genes. The experimental analysis H.in&#175;uenzae                      selective forces displacement essential genes to <br /> identi&#174;ed 474 essential genes 1272 analysed (i.e. ~70%                   lagging strand. <br /> of genome) (2), twice essential <br /> genes B.subtilis (4), predicted H.in&#175;uenzae <br /> from computational genome analysis (38). In H.in&#175;uenzae,                       DISCUSSION <br /> essentiality determined transposon mutagenesis, which <br /> is likely overestimate number essential genes.                       Essentiality drives gene strand bias <br /> Unsurprisingly, use E.coli data set intersection             As discussed above, lines evidence indicate the <br /> between data sets indicates signi&#174;cant bias (73%                  positioning certain genes leading strand subject to <br /> essential leading strand genes latter). This general              selection. Brewer proposed biases caused <br /> trend: highest biases intersection               selection avoidance frequent head-on collisions <br /> data sets (Table 2), probably case likehood                DNAP RNAP (13). Here, con&#174;rmed our <br /> of falsely assigning essentiality minimised.                                previous analyses, showing bacteria, essentiality, not <br />                                                                                expressiveness, plays major role gene strand bias (19). <br /> Testing degree gene strand conservation. We used                   This compatible previous model emphasis is <br /> the experimental data sets test conservation                shifted rate collisions polymerases the <br />  6576   Nucleic Acids Research, 2003, Vol. 31, No. 22 <br />  <br />  <br /> transcript resulting collisions. Most importantly,     non-essential genes, differences typically high. <br /> is gene product, expression rate,          There are, however, exceptions analysis of <br /> important selective factor. Thus, understanding fate     highly expressed genes versus essential highly <br /> transcript cast light causes gene strand bias. A      expressed genes, concern obligatory parasites/ <br /> possibility that, transcription abortion occurs       symbionts, obligate intracellular (Fig. 1C). One <br /> lower rates co-oriented collisions head-on collisions    propose expression levels important in <br /> (14), truncated transcripts lead truncated, non-    genomes. This unlikely reasons. First, these <br /> functional, peptides. When peptides parts large         bacteria grow slowly expression levels <br /> complexes (e.g. ribosome DNAP), typically       important fast growers. Secondly, we <br /> basis dominant-negative phenotypes. This naturally           used CAI determine high levels expression. Yet, the <br /> most counter-selected essential functions.                    genomes spirochetes (30) Chlamydia (40) exhibit weak <br />    We extended previous observations concerning gene            (although signi&#174;cant) codon usage biases highly <br /> strand bias directions. First, shown       expressed genes, Buchnera genomes nearly devoid <br /> spite substantial rearrangements E.coli          biases (41). Our tests CAI two <br /> B.subtilis, leading strand positioning essential genes    genomes g-proteobacteria low G + C &#174;rmicutes <br /> conserved. It conserved average       lack signi&#174;cant codon usage bias ribosomal proteins: <br /> orthologue. Secondly, results putatively          T.tengcongensis Buchnera. If CAI bad predictor of <br /> identi&#174;ed essential genes preferably located leading     expression levels genomes, safely <br /> strand analysed genomes low G + C              attribute atypical bias expression levels. Thirdly, gene <br /> &#174;rmicutes g-proteobacteria. Hence, collision problems       strand bias putative highly expressed non-essential <br /> associated gene strand bias likely general        genes relatively small. Therefore, expression unlikely to <br /> importance bacterial world. Thirdly,       play important role bacteria in <br /> essentiality taken account, expression levels        B.subtilis E.coli. Genome rearrangements change the <br /> signi&#174;cantly contribute explain gene strand bias        leading strand character gene and, frequent, they <br /> groups. Although essential genes synthesised         lead lower gene strand bias. Although genomes of <br /> high levels, frequency highly expressed genes among <br />                                                                     obligatory intracellular bacteria stable, Buchnera <br /> essential genes higher average (e.g. nearly 30% <br />                                                                     suffered extensive rearrangements associated process <br /> among B.subtilis essential genes). It quite natural that <br />                                                                     genome reduction, lowered gene strand <br /> previous analyses using rDNA ribosomal proteins have <br />                                                                     bias. It possible bacteria, leading strand <br /> revealed expressiveness basis gene strand bias: <br />                                                                     positioning essential genes weaker selection, e.g. <br /> these genes simultaneously essential highly expressed. <br /> Here, essentiality taken account,         drift associated small population <br /> expression plays small, any, role determining           sizes. Many bacteria suffer relaxed selection of <br /> distribution genes replication strands. All        housekeeping functions (9). Also, depend host for <br /> results derived analyses orthology assignment       functions deemed essential. As such, the <br /> of essentiality orthologous relationships. As       assignment essentiality homology error <br /> shown, prone errors. Yet, degree             prone functions, especially metabolic ones. <br /> consistency results phylogenetic          On hand, intracellular bacteria some <br /> groups cross-validations performed suggest          essential genes (e.g. transporters metabolites) that <br /> that errors strongly biasing conclusions.          essential survival, essential in <br />                                                                     B.subtilis. A clearer insight gene strand bias these <br /> Understanding differences gene strand bias intensity             genomes require availability experimental data sets <br /> There signi&#174;cant differences terms overall gene           essential genes bacteria (note M.genitalium <br /> strand bias bacteria. As discussed (17),        does belong category an <br /> composition DNAP explain differences.          intracellular bacterium). <br /> This effect independent genome size                 In addition suggesting lower biases obligatory <br /> species ecology relate differential stability     intracellular bacteria, data indicate different levels of <br /> the replication fork. Bacillus subtilis higher general   gene strand bias different bacteria. The causes may <br /> gene strand bias (75%) higher essential gene strand bias      different stabilities replication forks, as <br /> (94%) E.coli (55 76%, respectively). Hence,            discussed above, different RNAP processivity, leading to <br /> effect essentiality gene strand bias adds caused   RNAP robust collisions, different ef&#174;ciency <br /> differences DNAP composition, effects            mechanisms degrading truncated peptides. When an <br /> likely related. For example, speculate         aborted transcript results stalled ribosomes, tmRNA tags the <br /> having DNAP a-subunits leads increased asym-              truncated peptide directs protease degradation <br /> metry outcome collisions DNAP RNAP.           machinery (42,43). tmRNA does direct elimination of <br /> This lead higher frequency leading strand genes       corresponding mRNA likely saturated <br /> in general, essential genes particular.                   replicating bacteria, large number collisions. <br />    Except Buchnera, essential genes           Nevertheless, tmRNA intrinsically ef&#174;cient, one <br /> more frequent leading strand remaining          expect lower levels gene strand bias bacteria. <br /> genes (Fig. 1A). When plots, putative highly              Experimental work necessary tackle these <br /> expressed genes, distribution essential versus               questions. <br />                                                                                           Nucleic Acids Research, 2003, Vol. 31, No. 22                 6577 <br />  <br />  <br /> SUPPLEMENTARY MATERIAL                                                          20. Louarn,J.M., Bouche,J.P., Legendre,F., Louarn,J. Patte,J. (1985) <br />                                                                                     Characterization properties large inversions E.coli <br /> Supplementary Material available NAR Online.                                  chromosome origin-to-terminus <span id='am-324' about='obo:NCBITaxon_9855' typeof='owl:Thing obo:NCBITaxon_9850 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_91561 obo:NCBITaxon_35500 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_34878 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314145 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_9845'><span id='am-325' property="rdfs:label" content="Axis" datatype="xsd:string"></span><span id='am-326' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-327'  ></span><span id='am-328' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>axis</span>. Mol. Gen. Genet., 201, <br />                                                                                     467&#177;476. <br />                                                                                 21. Segall,A., Mahan,M.J. Roth,J.R. (1988) Rearrangement the <br /> ACKNOWLEDGEMENTS                                                                    bacterial chromosome: forbidden inversions. Science, 241, 1314&#177;1318. <br />                                                                                 22. Sharp,P.M. Li,W.H. (1987) The codon adaptation index&#65533;?a measure <br /> We thank Joe&#200;l Pothier comments manuscript,                         directional synonymous codon usage bias potential <br /> the scientists disrupted studied genes                       applications. Nucleic Acids Res., 15, 1281&#177;1295. <br /> B.subtilis genome, particular                             23. Rocha,E.P.C. Danchin,A. (2001) Ongoing evolution strand <br /> enterprise happen S. D. Ehrlich, F. Kunst, N. Ogasawara                         composition bacterial genomes. Mol. Biol. Evol., 18, 1789&#177;1799. <br />                                                                                 24. Tatusov,R.L. Koonin,E.V. (1997) A genomic perspective protein <br /> H. Yoshikawa.                                                                       families. Science, 278, 631&#177;637. <br />                                                                                 25. Coghlan,A. Wolfe,K.H. (2000) Relationship codon bias mRNA <br />                                                                                     concentration protein length <span id='am-263' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-264' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-265' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-266'  ></span><span id='am-267' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-268' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-269' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-270' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-271' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-272' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-273' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-274' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span><span id='am-275' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-276' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-277' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-278' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-279' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span>Saccharomyces cerevisiae</span>. Yeast, 16, <br /> REFERENCES                                                                          1131&#177;1145. <br />  1. Hutchison,C.A., Peterson,S.N., Gill,S.R., Cline,R.T., White,O.,             26. Jansen,R., Bussemaker,H.J. Gerstein,M. (2003) Revisiting codon <br />     Fraser,C.M., Smith,H.O. Venter,J.C. (1999) Global transposon                adaptation index whole-genome perspective: analyzing the <br />     mutagenesis minimal Mycoplasma genome. Science, 286,                      relationship gene expression codon occurrence yeast <br />     2165&#177;2169.                                                                      using variety models. Nucleic Acids Res., 31, 2242&#177;2251. <br />  2. Akerley,B.J., Rubin,E.J., Novick,V.L., Amaya,K., Judson,N.              27. Sharp,P.M. Li,W.-H. (1986) An evolutionary perspective on <br />     Mekalanos,J.J. (2002) A genome-scale analysis identi&#174;cation              synonymous codon usage unicellular organisms. J. Mol. Evol., 24, <br />     genes required growth survival Haemophilus in&#175;uenzae.                 28&#177;38. <br />     Proc. Natl Acad. Sci. USA, 99, 966&#177;971.                                     28. Andersson,S.G.E. Kurland,C.G. (1990) Codon preferences in <br />  3. Forsyth,R.A., Haselbeck,R.J., Ohlsen,K.L., Yamamoto,R.T., Xu,H.,                free-living microorganisms. Microbiol. Rev., 54, 198&#177;210. <br />     Trawick,J.D., Wall,D., Wang,L., Brown-Driver,V., Froelich,J.M. et al.       29. Me&#194;digue,C., Rouxel,T., Vigier,P., Henaut,A. Danchin,A. (1991) <br />     (2002) A genome-wide strategy identi&#174;cation essential genes       Evidence horizontal gene transfer E.coli speciation. J. Mol. Biol., <br />     <span id='am-38' about='obo:NCBITaxon_1280' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1385 obo:NCBITaxon_90964 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_1279'><span id='am-39' property="oboInOwl:hasRelatedSynonym" content="Micrococcus aureus" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="PMID:8573498" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasRelatedSynonym" content="Staphilococcus aureus" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasRelatedSynonym" content="Staphlococcus pyogenes citreus" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="Staphylococcus aureus" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasRelatedSynonym" content="Micrococcus pyogenes" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasAlternativeId" content="NCBITaxon:325213" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasRelatedSynonym" content="Staphylococcus pyogenes aureus" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasRelatedSynonym" content="Staphylococus aureus" datatype="xsd:string"></span><span id='am-50'  ></span><span id='am-51' property="oboInOwl:hasRelatedSynonym" content="Streptococcus aureus" datatype="xsd:string"></span>Staphylococcus aureus</span>. Mol. Microbiol., 43, 1387&#177;1400.                          222, 851&#177;856. <br />  4. Kobayashi,K., Ehrlich,S.D., Albertini,A., Amati,G., Andersen,K.K.,          30. Perriere,G. Thioulouse,J. (2002) Use misuse correspondence <br />     Arnaud,M., Asai,K., Ashikaga,S., Aymerich,S., Bessieres,P. et al. (2003)        analysis codon usage studies. Nucleic Acids Res., 30, 4548&#177;4555. <br />     Essential Bacillus subtilis genes. Proc. Natl Acad. Sci. USA, 100,          31. Parkhill,J., Wren,B.W., Thomson,N.R., Titball,R.W., Holden,M.T., <br />     4678&#177;4683.                                                                      Prentice,M.B., Sebaihia,M., James,K.D., Churcher,C., Mungall,K.L. et al. <br />  5. Jordan,I.K., Rogozin,I.B., Wolf,Y.I. Koonin,E.V. (2002) Essential           (2001) Genome sequence <span id='am-507' about='obo:NCBITaxon_632' typeof='owl:Thing obo:NCBITaxon_629 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_91347 obo:NCBITaxon_1224 obo:NCBITaxon_2 obo:NCBITaxon_543 obo:NCBITaxon_1236'><span id='am-508' property="rdfs:label" content="Yersinia pestis" datatype="xsd:string"></span><span id='am-509' property="oboInOwl:hasRelatedSynonym" content="Bacterium pestis" datatype="xsd:string"></span><span id='am-510' property="oboInOwl:hasRelatedSynonym" content="Pasteurella pestis" datatype="xsd:string"></span><span id='am-511' property="oboInOwl:hasRelatedSynonym" content="Yersinia pseudotuberculosis subsp. pestis" datatype="xsd:string"></span><span id='am-512'  ></span><span id='am-513' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-514' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-515' property="oboInOwl:hasRelatedSynonym" content="Pestisella pestis" datatype="xsd:string"></span><span id='am-516' property="oboInOwl:hasRelatedSynonym" content="Bacillus pestis" datatype="xsd:string"></span>Yersinia pestis</span>, causative agent of <br />     genes evolutionarily conserved nonessential genes          plague. Nature, 413, 523&#177;527. <br />     bacteria. Genome Res., 12, 962&#177;968.                                         32. Glaser,P., Rusniok,C., Buchrieser,C., Chevalier,F., Frangeul,L., <br />  6. Pal,C., Papp,B. Hurst,L.D. (2003) Rate evolution gene                Msadek,T., Zouine,M., Couve,E., Lalioui,L., Poyart,C. et al. (2002) <br />     dispensability. Nature, 421, 496&#177;497.                                           Genome sequence <span id='am-63' about='obo:NCBITaxon_1311' typeof='obo:NCBITaxon_91061 owl:Thing obo:NCBITaxon_1301 obo:NCBITaxon_1239 obo:NCBITaxon_1 obo:NCBITaxon_1300 obo:NCBITaxon_131567 obo:NCBITaxon_2 obo:NCBITaxon_186826'><span id='am-64'  ></span><span id='am-65' property="oboInOwl:hasDbXref" content="PMID:15774692" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="PMID:8995807" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasRelatedSynonym" content="Streptococcus difficile" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasRelatedSynonym" content="Streptococcus mastitidis" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasRelatedSynonym" content="Streptococcus difficilis" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="Streptococcus agalactiae" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasAlternativeId" content="NCBITaxon:76757" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasRelatedSynonym" content="Streptococcus agalactiae contagiosae" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasRelatedSynonym" content="Streptoccocus de la mammite" datatype="xsd:string"></span>Streptococcus agalactiae</span>, pathogen causing <br />  7. Hirsh,A.E. (2003) Rate evolution gene dispensability&#65533;?reply.              invasive neonatal disease. Mol. Microbiol., 45, 1499&#177;1513. <br />     Nature, 421, 497&#177;498.                                                       33. Tillier,E.R. Collins,R.A. (2000) Genome rearrangement by <br />  8. Rocha,E.P.C. Danchin,A. (2003) An analysis determinants               replication-directed translocation. Nature Genet., 26, 195&#177;197. <br />     protein substitution rates bacteria. Mol. Biol. Evol., press.         34. Eisen,J.A., Heidelberg,J.F., White,O. Salzberg,S.L. (2000) Evidence <br />  9. Ochman,H. Moran,N.A. (2001) Genes lost genes found:                 <br /> </body></html>