<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:NCBITaxon="http://purl.obolibrary.org/obo/NCBITaxon#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:ncbitaxon="http://purl.obolibrary.org/obo/ncbitaxon#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Human EFO1p exhibits acetyltransferase activity <br /> and is a unique combination of linker histone and <br /> Ctf7p/Eco1p chromatid cohesion establishment <br /> domains <br />                                                                   pole opposite that of its sister chromatid. At anaphase onset, <br />                                                                            cohesion is inactivated, allowing each sister to move along <br /> Proper segregation of chromosomes during mitosis                           the spindle apparatus into the newly forming daughter cells <br /> requires that the products of chromosome replica-                          (2,3). <br /> tion are paired together&#208;termed sister chromatid                              In budding yeast, several structural cohesion proteins <br /> cohesion. In budding yeast, Ctf7p/Eco1p is an                              (cohesins) have been identi&#174;ed: Smc1p, Smc3p, Mcd1p/ <br /> essential protein that establishes cohesion between                        Scc1p, Irr1p/Scc3p and Pds5p (4&#177;10). One current model is <br /> sister chromatids during S phase. In &#174;ssion yeast,                         that these cohesins form rings that encompass or tether <br /> Eso1p also functions in cohesion establishment,                            together sister chromatids (11,12). Deposition cohesion fac- <br /> but is comprised of a Ctf7p/Eco1p domain fused to                          tors Scc2p (Mis4p in &#174;ssion yeast) and Scc4p are required for <br /> a Rad30p domain (a DNA polymerase) both of which                           the proper association of structural cohesins with chromatin, <br /> are independently expressed in budding yeast. In                           with deposition occurring from G1/S through most of mitosis <br /> <span id='am-62' about='obo:NCBITaxon_169495' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_169440 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_43738 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_169449 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_43750 obo:NCBITaxon_33208 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_169455 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-63' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-65'  ></span><span id='am-66' property="rdfs:label" content="This" datatype="xsd:string"></span>this</span> report, we identify and characterize the &#174;rst                         (7,13,14). Ctf7p/Eco1p is an acetyltransferase that represents a <br />                                                                            third <span id='am-52' about='obo:NCBITaxon_class' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-53' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="class" datatype="xsd:string"></span>class</span> of cohesion proteins, the establishment factors <br /> candidate human ortholog of Ctf7p/Eco1p, which <br />                                                                            (7,15,16). Although loss of Ctf7p/Eco1p (herein termed <br /> we term hEFO1p (human Establishment Factor                                 Ctf7p) function leads to precocious sister chromatid separ- <br /> Ortholog). As in &#174;ssion yeast Eso1p, the hEFO1p                            ation and cell death, Ctf7p is not required to maintain cohesion <br /> open reading frame extends well upstream of                                nor deposit cohesins onto DNA. Instead, budding yeast Ctf7p <br /> the C-terminal Ctf7p/Eco1p domain. However, this                           appears to establish cohesion by coupling the cohesion <br /> N-terminal extension in hEFO1p is unlike Rad30p,                           machinery to DNA replication. <br /> but instead exhibits signi&#174;cant homology to linker                            Several lines of evidence reveal that cohesion establishment <br /> histone proteins. Thus, hEFO1p is a unique fusion                          is coupled to DNA replication. First, Ctf7p is required <br /> of linker histone and cohesion establishment                               exclusively during S phase when DNA replication occurs <br /> domains. hEFO1p is widely expressed among the                              (7,16). Second, cells defective for Ctf7p function are rescued <br /> tissues tested. Consistent with a role in chromo-                          by elevated levels of PCNA (proliferating cell nuclear <br /> some segregation, hEFO1p localizes exclusively to                          antigen), a sliding clamp that promotes processive DNA <br />                                                                            replication (16,17). Third, Ctf7p physically associates with <br /> the nucleus when expressed in HeLa tissue culture <br />                                                                            each of three independent replication factor C (RFC) <br /> cells. Moreover, biochemical analyses reveal that                          complexes (18). RFC complexes load PCNA-like sliding <br /> hEFO1p exhibits acetyltransferase activity. These                          clamps onto double-stranded DNA, at least a subset of which <br /> &#174;ndings document the &#174;rst characterization of a                            are known to function in sister chromatid cohesion (18&#177;22). <br /> novel human acetyltransferase, hEFO1p, that is                             The characterization of Trf4p (Pol s) provided the &#174;rst <br /> comprised of both linker histone and Ctf7p/Eco1p                           evidence that a DNA polymerase could also function in sister <br /> domains.                                                                   chromatid cohesion (23,24). More recently, the large subunit <br />                                                                            of budding yeast DNA polymerase e (Pol2) was found to <br />                                                                            associate with Pol s and participate in cohesion (25). <br /> INTRODUCTION <br />                                                                               Despite the essential roles of budding yeast Ctf7p in <br /> Faithful transmission of chromosomes requires that chromo-                 chromosome segregation and cell viability, nothing is known <br /> somes are replicated and that the resulting sister chromatids              concerning the Ctf7p cohesion establishment pathway in <br /> are paired together. This pairing, or sister chromatid cohesion,           higher eukaryotes. Recent &#174;ndings from yeast reveal that <br /> occurs concomitantly with chromosome replication during                    Ctf7p function is conserved through evolution. For instance, in <br /> S phase (1). During mitosis, cohesion ensures that one                     budding yeast, CTF7 is located on chromosome VI and <br /> chromatid associates with microtubules from the spindle                    encodes 281 amino acids (7,16). RAD30, encoding a DNA <br />  <br /> *To whom correspondence should be addressed. Tel: +1 610 758 6268; Fax: +1 610 758 4004; Email rvs3@lehigh.edu <br />  <br /> Nucleic Acids Research, Vol. 31 No. 21 &#227; Oxford University Press 2003; all rights reserved <br />                                                                             Nucleic Acids Research, 2003, Vol. 31, No. 21    6335 <br />  <br />  <br /> repair polymerase (Pol h), is located on chromosome IV and          harboring hEFO1p in the reverse orientation were also <br /> encodes 632 amino acids (26&#177;28). In &#174;ssion yeast, CTF7 and          identi&#174;ed. To generate a C-terminal truncation of hEFO1p, <br /> RAD30 sequences are genetically fused to produce ESO1.              the GST&#177;hEFO1 construct was digested with BsmI to drop-out <br /> Eso1p is comprised of 872 amino acids and functions in both         a 497 nucleotide fragment containing the CTF7 core domain <br /> cohesion establishment and DNA polymerase functions,                coding region. The remaining construct was ligated back <br /> indicating that both Ctf7p and Rad30p activities are preserved      together, termed hEFO1-2, and encodes a protein harboring a <br /> (29,30). In this report, we identify the &#174;rst full-length human     C-terminal truncation. <br /> ortholog of Ctf7p and characterize its enzymatic activity, <br /> localization and novel genetic composition.                         Expression and detection of recombinant proteins <br />                                                                     pGST&#177;hEFO1p and control constructs were transformed into <br />                                                                     BL-21 bacterial cells and selected on LB plates containing <br /> MATERIALS AND METHODS                                               carbenicillin. Overnight liquid cultures were diluted 10-fold <br />                                                                     and incubated for 2 h at 37&#176;C to obtain log phase growth. The <br /> Cell growth and database methods <br />                                                                     cultures were then brought to a &#174;nal concentration of 100 mM <br /> Growth and sporulation media for yeast were described               isopropyl-b-D-thiogalactopyranoside (IPTG; Fisher) and <br /> previously (31). Yeast and bacterial strains and transform-         protein expression induced for 3&#177;4 h at 23&#176;C. Cells were <br /> ations were performed as described, with minor modi&#174;cations         then spun for 10 min at 9500 r.p.m. (Sorvall SS-34) and the <br /> (32&#177;36). HeLa cell cultures were maintained in MEM                  pellet resuspended in SDS/Laemmli buffer and boiled prior to <br /> supplemented with 10% fetal bovine serum, Earle&apos;s BSS,              resolving on polyacrylamide gels. Proteins were visualized by <br /> L-glutamine and antibiotics.                                        transferring onto PVDF membrane (Amersham/Pharmacia) <br />    Budding yeast Ctf7p and Rad30p sequences and &#174;ssion              and detected with a combination of a mouse antibody directed <br /> yeast Eso1p sequence were used to perform PSI-BLAST                 against GST (Santa Cruz), goat anti-mouse horseradish <br /> searches of non-redundant nucleic acid and protein sequence         peroxidase (HRP; Bio-Rad) and ECL+ (Amersham/ <br /> databases (GenBank CDS, PDB, SWISS-PROT, PIR and                    Pharmacia). <br /> PRF), expressed sequence tag (EST) databases and recent                For acetylation assays, the bacterial cell pellet was <br /> submissions to the human genonome project (http://www.              suspended in 400 ml of 120 ELB buffer (38) supplemented <br /> ncbi.nlm.nih.gov/BLAST). Clustal X and Clustal W were used          with EDTA-free protease inhibitors (Roche) and phenyl- <br /> to connect overlapping sequences to produce extended                methylsulfonyl &#175;uoride (PMSF). De-acetylase inhibitors <br /> contiguous (contig) coding sequences. A full-length cDNA            sodium butyrate (10 mM, Sigma) and nicotinamide (5 mM, <br /> containing human EFO1 was identi&#174;ed from the HUGE                   Sigma) were added prior to cell lysis. The cell suspensions <br /> (Human Unidenti&#174;ed Gene-Encoded Large Proteins) database            were lysed by sonication and brought to a &#174;nal concentration <br /> and obtained from the Kazusa DNA Research Institute in              of 50&#177;100 mM acetyl-CoA (Sigma). After 5&#177;10 min, the <br /> Japan (37).                                                         reaction was stopped by the addition of SDS/Laemmli buffer, <br />                                                                     boiled and resolved by polyacrylamide gel electrophoresis. <br /> Cloning and molecular methods <br />                                                                     Western blot detection of acetylated proteins was performed <br /> The human EFO1 opening reading frame (ORF) was                      using a combination of mouse antibody directed against <br /> independently derived by the HUGE database. hEFO1 coding            acetylated lysine residues (Cell Signaling Technology), goat <br /> sequence, &#175;anked by BamHI restriction sites, was generated          anti-mouse HRP (Bio-Rad) and ECL+ (Amersham/ <br /> by PCR using KIAA1911 as a template (obtained from Kazusa           Pharmacia). An additional step to enrich for GST-coupled <br /> DNA Research Institute) and oligos 5&#162;-CGCGGATCCATG-                 proteins was optionally performed using glutathione&#177; <br /> TCCATTCAGGAGAAATCAAAAGAG-3&#162; and 5&#162;-CGCG-                            Sepharose 4B beads (Amersham-Pharmacia) as previously <br /> GATCCGGTTGTTGCCAGTCCTGAGTTCATTGT-3&#162;. The                            described (18). <br /> resulting product contains hEFO1p in which the N-terminal              Transfection of pEGFP&#177;hEFO1p into human tissue culture <br /> methionine is deleted. The PCR product was then cloned into         HeLa cells was performed according to the manufacturer&apos;s <br /> pCR 2.1 (Invitrogen) to generate pBS1088. The entire EFO1           instructions (Roche). Brie&#175;y, cells were grown to ~30% <br /> region was sequenced, and two errors, presumably generated          con&#175;uence on coverslips housed in 6-well culture dishes. <br /> during PCR, were identi&#174;ed (Y93F and L384S). We termed              pEGFP&#177;hEFO1p and control <span id='am-101' about='obo:NCBITaxon_36549' typeof='owl:Thing obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-102' property="rdfs:label" content="plasmids" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasAlternativeId" content="NCBITaxon:2435" datatype="xsd:string"></span>plasmids</span> were diluted into <br /> this allele hEFO1-1. These two alterations are located well         serum-free medium supplemented with Fugene (Roche) and <br /> outside of the Ctf7p functional core domain (762&#177;831) and did       the resulting mixture added drop-wise into each of the wells <br /> not adversely affect either acetyltransferase activity or nuclear   before returning the cells to 37&#176;C. Transfection and recovery <br /> localization (see Results). To generate wild-type hEFO1, a          were assessed 24&#177;48 h later. <br /> BplI&#177;BsgI fragment obtained from KIAA1911 cDNA was                     For visualization of GFP&#177;hEFO1 protein in vivo, coverslips <br /> used to replace the short DNA region containing both PCR-           were dipped in warm PEM buffer and then &#174;xed by plunging <br /> based alterations. The resulting clone was con&#174;rmed as correct      into &#177;20&#176;C methanol for 2 min. The cells were rehydrated in <br /> by sequence analysis and used for further analyses.                 phosphate-buffered saline (PBS) and the coverslip mounted <br />    To generate in-frame green &#175;uorescent protein (GFP)&#177;             onto a glass slide using an antifade reagent to reduce <br /> hEFO1p and GST&#177;hEFO1p expression <span id='am-87' about='obo:NCBITaxon_29278' typeof='owl:Thing obo:NCBITaxon_81077 obo:NCBITaxon_1 obo:NCBITaxon_28384'><span id='am-88' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasDbXref" content="GC_ID:11" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="vectors" datatype="xsd:string"></span>vectors</span>, pCR 2.1-                  photobleaching (Vectastain). Images were collected on a <br /> hEFO1 was digested with BamHI and the resulting drop-out            Nikon microscope (Eclipse E800) coupled to a Photometrics <br /> fragment ligated into either pGEX 4T-3 linearized with              cooled CCD digital <span id='am-79' about='obo:NCBITaxon_274080' typeof='owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_33154 obo:NCBITaxon_65171 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_7408 obo:NCBITaxon_7401 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_231877 obo:NCBITaxon_172377 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7400 obo:NCBITaxon_33392 obo:NCBITaxon_7399 obo:NCBITaxon_50557'><span id='am-80' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Camera" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-83'  ></span>camera</span> (CoolSnap fx). <br /> BamHI (Amersham/Pharmacia) or pEGFP C.1 linearized with                For detection of GFP proteins by western blot analysis, <br /> BglII (Clontech). For both GST and GFP constructs, plasmids         coverslips were rinsed twice in cold PBS and the cells <br />  6336   Nucleic Acids Research, 2003, Vol. 31, No. 21 <br />  <br />  <br /> immediately lysed in SDS/Laemmli buffer, boiled and               alignment of the Ctf7p core domain indicates the high <br /> resolved by acrylamide gel electrophoresis. GFP-tagged            degree of conservation (Fig. 1B). We found full-length <br /> proteins were transferred onto PVDF membrane and detected         cDNAs for both Drosophila and human candidate orthologs <br /> with a combination of a mouse antibody directed against GFP       that contain coding regions that extend well upstream of the <br /> (Cell Signaling Technology), goat anti-mouse HRP (Bio-Rad)        Ctf7p domain (N-terminal extensions of 854 residues in <br /> and ECL+ (Amersham/Pharmacia).                                    Drosophila and 641 residues in humans). Neither of these <br />                                                                   N-terminal upstream domains exhibit similarity to Rad30p. <br /> Northern blot analyses <br />                                                                   These results show that, in both Drosophila and humans, the <br /> Expression of hEFO1p was assessed using a multi-tissue            highly conserved Ctf7p domain is fused to a domain other than <br /> human northern blot (Clontech) following the manufacturer&apos;s       Rad30p, indicating that Ctf7p is a modular element that has <br /> instructions. Two con&#174;rm expression, the membrane was             been genetically recombined to other elements through <br /> independently probed twice using distinct probes: a 705 nt        evolution. <br /> PCR product (oligos 5&#162;-GACCCACCATTGGATAATCAG- <br /> 3&#162; and 5&#162;TGGATATGTTCCGCAATTAGG-3&#162;) or a 690 bp                    Human EFO1p is one of four candidate proteins <br /> fragment released via EcoRI&#177;BamHI digestion. In both cases,       exhibiting homology to Ctf7p <br /> the DNA probes were labeled with [32P]dCTP by random <br /> priming and separated from unincorporated dNTPs prior to          A human EST that exhibits sequence homology to a single <br /> hybridization (39).                                               Ctf7p candidate ortholog in humans was previously reported <br />                                                                   (16). More recently, other short Ctf7p-like encoding sequen- <br />                                                                   ces have been identi&#174;ed (http://mendel.imp.univie.ac.at). <br />                                                                   Here, we extend each of these sequences, all of which reside <br /> RESULTS <br />                                                                   on different chromosomes and contain the highly conserved <br /> Ctf7p/Eco1p is highly conserved but genetically mobile            Ctf7p core domain (Fig. 2). At least one of these human ORFs <br /> through evolution                                                 contains an N-terminal extension that is absent from budding <br /> In budding yeast, Ctf7p is a 281 amino acid protein essential     yeast Ctf7p and unique to the N-terminal region of &#174;ssion <br /> for sister chromatid cohesion (7,16). In &#174;ssion yeast, Eso1p is   yeast Eso1p (see below). To differentiate between these <br /> an 872 amino acid protein that also functions in sister           human sequences in an unbiased way (ScCtf7p/Eco1p versus <br /> chromatin cohesion but is comprised of two distinct domains:      SpEso1p) and to re&#175;ect that at least one of these human coding <br /> a Rad30p domain as the N-terminus and a Ctf7p domain as the       regions contains a domain unique to any Ctf7p protein thus far <br /> C-terminus (29,30). To date, it is unknown whether higher         characterized, we name this <span id='am-44' about='obo:NCBITaxon_family' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-45' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="family" datatype="xsd:string"></span>family</span> human Establishment <br /> <span id='am-84' about='obo:NCBITaxon_order' typeof='owl:Thing NCBITaxon:_taxonomic_rank'><span id='am-85' property="rdfs:label" content="order" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>order</span> eukaryotes display the stand-alone Ctf7p of budding         Factor Orthologs (hEFO1s). In this report, we provide the full- <br /> yeast, the bi-functional Rad30p&#177;Ctf7p fusion of &#174;ssion yeast      length translation, expression pattern and characterization <br /> or a novel genetic fusion. Resolving this issue thus remains an   for one family member, hEFO1p (see below). hEFO1 <br /> important &#174;rst step in identifying the factors and elucidating    (KIAA1911, accession No. AB067498) resides on <br /> the molecular activities required for cohesion establishment in   chromosome XVIII. hEFO2&#177;hEFO4 exhibit signi&#174;cant <br /> human cells.                                                      homology to the C-terminus of hEFO1, although their <br />    We used computer-assisted database searches to assess          expression is currently unknown. hEFO2 resides on <br /> which form of Ctf7p predominates in higher eukaryotes. First,     chromosome VIII and represents the &#174;rst human coding <br /> we identi&#174;ed Rad30p-like proteins and then determined             sequence identi&#174;ed that exhibited signi&#174;cant homology to <br /> whether the Rad30p domain was coupled to a downstream             Ctf7p (16). We used database search methods to extend <br /> Ctf7p domain. The Rad30p domain of &#174;ssion yeast Eso1p was         the hEFO2 coding region, identifying two new overlapping <br /> used to query non-redundant databases, including ESTs and         ESTs that produce a conceptual contiguous amino acid <br /> newly released human genome sequences (Materials and              sequence of 334 residues (GenBank accession Nos N94144, <br /> Methods). We obtained multiple hits, consistent with previous     BG288675 and BE247544). hEFO3 (accession No. NT <br /> studies documenting the high degree of conservation of            005334.10) and hEFO4 (accession No. NT 033232.2) reside <br /> Rad30p proteins through evolution (40&#177;42). Sequence analy-        on chromosomes II and XI, respectively. Both hEFO3 and <br /> ses revealed that these Rad30p-like proteins failed to contain    hEFO4 coding sequences contain a 762 bp region that exhibits <br /> domains that exhibited signi&#174;cant homology to Ctf7p (<span id='am-96' about='obo:NCBITaxon_1369087' typeof='obo:NCBITaxon_37567 owl:Thing obo:NCBITaxon_33340 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_85604 obo:NCBITaxon_104431 obo:NCBITaxon_33154 obo:NCBITaxon_37570 obo:NCBITaxon_41191 obo:NCBITaxon_33317 obo:NCBITaxon_2759 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_33208 obo:NCBITaxon_41196 obo:NCBITaxon_197562 obo:NCBITaxon_85512 obo:NCBITaxon_197563 obo:NCBITaxon_95182 obo:NCBITaxon_41197 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_7088 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_7100 obo:NCBITaxon_33392 obo:NCBITaxon_50557'><span id='am-97'  ></span><span id='am-98' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-99' property="rdfs:label" content="Data" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span>data</span> not     signi&#174;cant homology to positions 3139&#177;3900 encoding <br /> shown). The inability to identify Rad30p proteins that also       hEFO1p. <br /> contain a Ctf7p domain in higher eukaryotes suggests that the        Conceptual translation of hEFO1 cDNA indicates that <br /> genetic fusion comprising Eso1p may be limited to close           hEFO1p is comprised of 840 amino acid residues (Fig. 3A) <br /> relatives of &#174;ssion yeast.                                        encoded by a 4.6 kb message containing 5&#162;- and 3&#162; <br />    Second, we asked the reciprocal question: is the Ctf7p         untranslated regions (37). To both test for hEFO1 expression <br /> domain coupled to an upstream Rad30p domain in higher             and assess its relative abundance among various cell types, we <br /> eukaryotes? The Ctf7p domain of &#174;ssion yeast Eso1p was used       probed a multi-tissue northern blot containing RNA isolated <br /> to query non-redundant and EST databases as described             from heart, brain, placenta, lung, liver, muscle, kidney and <br /> above. We then ascertained if any of the putative Ctf7p           pancreas tissues. 32P-labeled probes from two regions of the <br /> homologs contain a Rad30p domain. Database analyses               hEFO1 ORF were independently used and identi&#174;ed a single <br /> revealed that conceptual translations of putative Ctf7p           transcript (Fig. 3B). The results show that hEFO1 is expressed <br /> orthologs exist in several model organisms including              in the majority of tissues tested, but elevated expression was <br /> Drosophila, mice and humans (Fig. 1A). An amino acid              detected in skeletal muscle. <br />                                                                                           Nucleic Acids Research, 2003, Vol. 31, No. 21               6337 <br />  <br />  <br />  <br />  <br /> Figure 1. Identi&#174;cation of Ctf7p-like proteins in higher eukaryotes. (A) Schematic illustrating Eso1p and conceptual translations of cDNAs that encode <br /> Ctf7p-like proteins. E values (a measurement of similarity) for Ctf7p domains are shown in black. The N-terminal region of Eso1p that exhibits homology to <br /> Rad30p is shown in dark gray. Regions that extend upstream of the Ctf7p region in human and Drosophila are shown in light gray. (B) Amino acid align- <br /> ments reveal the high degree of conservation of the Ctf7p domain for a selection of coding sequences. Residues are numbered according to conceptual transla- <br /> tions of cDNA sequences, except for contig sequences which were assigned a starting residue number of 1. Accession numbers are as follows: <br /> <span id='am-30' about='obo:NCBITaxon_4895' typeof='obo:NCBITaxon_33154 owl:Thing obo:NCBITaxon_451864 obo:NCBITaxon_451866 obo:NCBITaxon_147554 obo:NCBITaxon_1 obo:NCBITaxon_4890 obo:NCBITaxon_34346 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4894 obo:NCBITaxon_2759'><span id='am-31' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Schizosaccharomyces" datatype="xsd:string"></span><span id='am-34'  ></span>Schizosaccharomyces</span> pombe Eso1p (BAA95122), <span id='am-6' about='obo:NCBITaxon_4932' typeof='owl:Thing obo:NCBITaxon_4930 obo:NCBITaxon_1 obo:NCBITaxon_4751 obo:NCBITaxon_131567 obo:NCBITaxon_4893 obo:NCBITaxon_33154 obo:NCBITaxon_451864 obo:NCBITaxon_4892 obo:NCBITaxon_4891 obo:NCBITaxon_4890 obo:NCBITaxon_716545 obo:NCBITaxon_147537 obo:NCBITaxon_2759'><span id='am-7' property="oboInOwl:hasExactSynonym" content="yeast" datatype="xsd:string"></span><span id='am-8'  ></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="lager beer yeast" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces capensis" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="brewer&apos;s yeast" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasRelatedSynonym" content="Sccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasRelatedSynonym" content="Saccharomyes cerevisiae" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces italicus" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces oviformis" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasRelatedSynonym" content="Candida robusta" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Saccharomyces cerevisiae" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasRelatedSynonym" content="Saccaromyces cerevisiae" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasRelatedSynonym" content="Saccharomyces uvarum var. melibiosus" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="baker&apos;s yeast" datatype="xsd:string"></span>Saccharomyces cerevisiae</span> Ctf7p/Eco1p (NP 116683), <span id='am-55' about='obo:NCBITaxon_7227' typeof='owl:Thing obo:NCBITaxon_7214 obo:NCBITaxon_32351 obo:NCBITaxon_33340 obo:NCBITaxon_7215 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_186285 obo:NCBITaxon_7147 obo:NCBITaxon_43733 obo:NCBITaxon_33154 obo:NCBITaxon_46879 obo:NCBITaxon_43738 obo:NCBITaxon_43845 obo:NCBITaxon_480117 obo:NCBITaxon_33317 obo:NCBITaxon_480118 obo:NCBITaxon_2759 obo:NCBITaxon_46877 obo:NCBITaxon_6072 obo:NCBITaxon_6960 obo:NCBITaxon_32341 obo:NCBITaxon_33208 obo:NCBITaxon_32346 obo:NCBITaxon_85512 obo:NCBITaxon_197562 obo:NCBITaxon_197563 obo:NCBITaxon_7203 obo:NCBITaxon_43746 obo:NCBITaxon_6656 obo:NCBITaxon_7496 obo:NCBITaxon_1206794 obo:NCBITaxon_88770 obo:NCBITaxon_33392 obo:NCBITaxon_50557 obo:NCBITaxon_43741'><span id='am-56' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-58'  ></span><span id='am-59' property="oboInOwl:hasRelatedSynonym" content="Drosophila melangaster" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="Drosophila melanogaster" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasExactSynonym" content="fruit fly" datatype="xsd:string"></span>Drosophila melanogaster</span> (AAF50579.1), Mus muscu- <br /> lus ORF (MGC:30637), <span id='am-67' about='obo:NCBITaxon_9606' typeof='owl:Thing obo:NCBITaxon_9605 obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-68' property="rdfs:label" content="Homo sapiens" datatype="xsd:string"></span><span id='am-69'  ></span><span id='am-70' property="oboInOwl:hasExactSynonym" content="human" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasExactSynonym" content="man" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Homo sapiens</span> KIAA1191 (AB067498) and <span id='am-74' about='obo:NCBITaxon_9605' typeof='owl:Thing obo:NCBITaxon_33213 obo:NCBITaxon_1 obo:NCBITaxon_9526 obo:NCBITaxon_9604 obo:NCBITaxon_32523 obo:NCBITaxon_131567 obo:NCBITaxon_32524 obo:NCBITaxon_117570 obo:NCBITaxon_32525 obo:NCBITaxon_89593 obo:NCBITaxon_7742 obo:NCBITaxon_117571 obo:NCBITaxon_33154 obo:NCBITaxon_1437010 obo:NCBITaxon_1338369 obo:NCBITaxon_40674 obo:NCBITaxon_8287 obo:NCBITaxon_2759 obo:NCBITaxon_314146 obo:NCBITaxon_6072 obo:NCBITaxon_33511 obo:NCBITaxon_33208 obo:NCBITaxon_7711 obo:NCBITaxon_314293 obo:NCBITaxon_314295 obo:NCBITaxon_376913 obo:NCBITaxon_7776 obo:NCBITaxon_9347 obo:NCBITaxon_207598 obo:NCBITaxon_9443'><span id='am-75'  ></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="Homo" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span>Homo</span> sapiens contig (comprised of N94144, BG288675 and BE247544). Other mouse <br /> (AAH08220, BAB26905 and AA881675) and human orthologs were also identi&#174;ed. <br />  <br />  <br />  <br />  <br /> Human EFO1p is comprised of linker histone protein                              both hEFO1p and Drosophila cDNA translations exhibit <br /> and Ctf7p domains                                                               signi&#174;cant homology to micronuclear linker histone proteins <br />                                                                                 identi&#174;ed in <span id='am-47' about='obo:NCBITaxon_5890' typeof='obo:NCBITaxon_291294 owl:Thing obo:NCBITaxon_31277 obo:NCBITaxon_37093 obo:NCBITaxon_1 obo:NCBITaxon_131567 obo:NCBITaxon_431838 obo:NCBITaxon_2759 obo:NCBITaxon_5878 obo:NCBITaxon_33630 obo:NCBITaxon_6020'><span id='am-48' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-49'  ></span><span id='am-50' property="oboInOwl:hasDbXref" content="GC_ID:6" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="Tetrahymena" datatype="xsd:string"></span>Tetrahymena</span> (Fig. 4). Based on similarities <br /> In both hEFO1 and in the Drosophila cDNA containing the                         between these unique genetic fusions, we term the Drosophila <br /> Ctf7p core domain that we identi&#174;ed by computer-assisted                        homolog dEFO1. Micronuclear linker histone proteins are <br /> database searches (Fig. 1), the C-terminal Ctf7p domain is                      unique from macronulcear histone H1 and are critical for <br /> fused to an N-terminal extension other than Rad30p. We next                     normal chromosome compaction (43,44). In Tetrahymena, the <br /> determined whether these N-terminal extensions are similar to                   71 kDa linker histone polyprotein is processed to produce <br /> each other or represent yet further novel genetic fusions.                      independent a, b, g and d linker histone proteins (a linker <br /> Database search methods reveal that both N-terminal exten-                      histone is an early cleavage product, but later processed to <br /> sions are conserved. Importantly, the N-terminal extensions of                  generate g and d linker histone proteins). Sequence alignments <br />  6338    Nucleic Acids Research, 2003, Vol. 31, No. 21 <br />  <br />  <br />  <br />  <br /> Figure 2. Amino acid alignments for conceptual translations of candidate human Ctf7p family members. See text for accession numbers and details. <br />  <br />  <br />  <br /> reveal that the N-terminal extension in hEFO1p is comprised                    Ctf7p is known to serve as its own substrate. We exploited this <br /> of d and b linker histone proteins (Fig. 4). These &#174;ndings                     autoacetylation activity to determine whether hEFO1p <br /> document a unique genetic fusion between linker histone                        also exhibits acetyltransferase activity. The hEFO1 coding <br /> protein and a Ctf7p domain.                                                    sequence was inserted in-frame behind that of GST in an <br />                                                                                inducible bacterial expression vector. The resulting GST&#177; <br /> EFO1p expressed in human cells is a nuclear protein                            hEFO1 construct was transformed into bacterial cells and <br /> In budding yeast, Ctf7p is a nuclear protein required for proper               protein expression induced using IPTG (Materials and <br /> chromosome segregation (7,16). It thus became important to                     Methods). As a positive control, GST&#177;CTF7 was indepen- <br /> test whether hEFO1p localizes to a speci&#174;c region in the                       dently transformed and expressed in bacteria. Post-induction, <br /> vertebrate cell. The hEFO1 coding cDNA was obtained from                       the bacterial strains were lysed and assessed for protein <br /> the Kazusa DNA Research Institute in Japan (37) and inserted                   expression. An optional protein enrichment step via <br /> in-frame behind GFP in a mammalian expression vector                           glutathione&#177;Sepharose bead pull-down was also used <br /> containing the <span id='am-1' about='obo:NCBITaxon_10358' typeof='owl:Thing obo:NCBITaxon_10357 obo:NCBITaxon_1 obo:NCBITaxon_548681 obo:NCBITaxon_35237 obo:NCBITaxon_10239 obo:NCBITaxon_10292'><span id='am-2' property="oboInOwl:hasDbXref" content="GC_ID:1" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="ncbi_taxonomy" datatype="xsd:string"></span><span id='am-4'  ></span><span id='am-5' property="rdfs:label" content="Cytomegalovirus" datatype="xsd:string"></span>cytomegalovirus</span> (CMV) promotor (Materials                       (Materials and Methods). Both GST&#177;hEFO1p and GST&#177; <br /> and Methods). The resulting GFP&#177;hEFO1p construct and GFP                       Ctf7p protein bands of the appropriate molecular weights were <br /> control plasmids were transfected into HeLa tissue culture                     readily detectable by western blot analyses (Fig. 6). <br /> cells. Within 24 h, GFP&#177;hEFO1p expression in HeLa cell                            We then tested whether bacterially expressed hEFO1p was <br /> extracts was readily detected by western blot at the appropriate               capable of acetyltransferase activity. As before, GST&#177;EFO1p <br /> molecular weight. A protein band of similar molecular weight                   and GST&#177;Ctf7p expression was induced, the bacterial cells <br /> was absent in HeLa extracts expressing GFP alone (data not                     lysed and GST-containing proteins enriched by glutathione&#177; <br /> shown).                                                                        Sepharose bead pull-downs. The resulting protein fractions <br />    We then determined the localization of GFP&#177;hEFO1p in                        were incubated for 10 min in buffer containing a &#174;nal <br /> human tissue culture cells. Transfected HeLa cells were grown                  concentration of 50 mM acetyl-CoA. The addition of acetyl <br /> on coverslips for 24&#177;48 h. Images were then collected using                    moieties onto GST&#177;hEFO1p or GST&#177;Ctf7p was then visualized <br /> both phase contrast microscopy to visualize cell morphology                    by western blot analysis using an acetylated lysine-speci&#174;c <br /> and epi-illumination to detect GFP-tagged proteins. GFP                        antibody previously documented to detect Ctf7p acetylation <br /> visualization revealed that hEFO1p localizes speci&#174;cally to                    reactions (15). As expected, GST&#177;Ctf7p exhibited a robust <br /> the human cell nucleus (Fig. 5). In contrast, GFP was                          autoacetylation activity. GST&#177;hEFO1p also exhibited a similar <br /> predominantly cytoplasmic. Untransfected cells were devoid                     autoacetylation activity (Fig. 6). No antibody cross-reactivity <br /> of GFP signal. These results reveal that hEFO1p localizes to                   was observed for GST alone. These &#174;ndings reveal that the <br /> the nucleus of human cells, consistent with a role in                          acetyltransferase activity of Ctf7p-like proteins is conserved <br /> chromosome segregation.                                                        through evolution. We next tested for the speci&#174;city of the <br />                                                                                hEFO1p acetylation reaction by generating a loss-of-function <br /> Human EFO1p exhibits acetyltransferase activity                                allele in which the C-terminal region containing the Ctf7p core <br /> Bacterial expressed budding yeast Ctf7p was recently shown                     domain was deleted (hEFO1-2p). The resulting construct was <br /> to possess acetyltransferase activity in vitro (15). Although                  expressed at levels similar to hEFO1p but was unable to <br /> physiologically relevant substrates have yet to be identi&#174;ed,                  catalyze the autoacetylation reaction (data not shown). <br />          Nucleic Acids Research, 2003, Vol. 31, No. 21              6339 <br />  <br />  <br />  <br />  <br /> Figure 3. Expression and conceptual translation of hEFO1p. (A) Nucleotide <br /> and corresponding amino acid sequence for hEFO1p. (B) Northern blot <br /> analysis reveals that hEFO1p is expressed in most of the human tissues <br /> tested. <br />  6340    Nucleic Acids Research, 2003, Vol. 31, No. 21 <br />  <br />  <br />  <br />  <br /> Figure 4. hEFO1p and Tetrahymena linker histone alignments. Top: the schematic depicts the region of conservation between Tetrahymena linker histone <br /> proteins and hEFO1p. The MLH polyprotein cleavage products a, d, g and b linker histones are shaded (AAC18874). Bottom: amino acid alignment reveals <br /> the high degree of conservation (E of 1 3 10 &#177;11) between hEFO1p and Tetrahymena g and b linker histone proteins. Identical residues are boxed in black; <br /> conserved residues are boxed in gray. <br />  <br />  <br />  <br />  <br /> DISCUSSION <br /> Sister chromatid cohesion plays a central role in chromosome <br /> transmission, cell cycle progression and genome maintenance <br /> (45,46). To date, Ctf7p-like proteins identi&#174;ed in yeast remain <br /> the only essential establishment factors identi&#174;ed: factors that <br /> couple sister chromatid cohesion to DNA replication without <br /> functioning in DNA replication per se (7,16,29). In this report, <br /> we identify and characterize the &#174;rst candidate Ctf7p ortholog <br /> in humans, hEFO1p. hEFO1p is present in almost all tissues <br /> tested. It is unknown whether the inability to detect hEFO1 in <br /> brain tissue is due to detection limitations or to the terminally <br /> differentiated nature of this tissue, but BLAST searches of <br /> EST databases identify CTF7-like cDNAs from brain libraries <br /> (A.Bellows and R.V.Skibbens, unpublished observations). We <br /> also identi&#174;ed three other chromosomal loci within the human                  Figure 5. Localization of hEFO1p in mammalian cells. GFP-tagged <br /> genome that encode Ctf7p-like proteins (hEFO2p&#177;hEFO4p).                       hEFO1p versus GFP vector alone constructs were transfected into HeLa <br /> One of the probes used to identify the hEFO1 transcript is co-                cells and localization assessed 24 h later. Cell images obtained using the <br /> incident with hEFO2&#177;hEFO4 sequences. However, we did not                      GFP channel reveal that hEFO1p speci&#174;cally localizes to the nucleus while <br />                                                                               GFP is uniformly distributed throughout the cell. Cell images obtained <br /> detect novel sized transcripts, suggesting that either the                    by differential interference contrast microscopy (DIC) are also shown <br /> reaction temperature precluded binding of similar but not                     (* denotes nuclei in transfected cells). GFP expression was not observed in <br /> identical sequences, that the other transcripts occur at very low             untransfected cells. <br />                                                                                     Nucleic Acids Research, 2003, Vol. 31, No. 21      6341 <br />  <br />  <br />                                                                             entail identifying physiologically relevant substrates for Ctf7p <br />                                                                             acetylation. <br />                                                                                Our &#174;ndings are novel in that human EFO1p is comprised of <br />                                                                             a Ctf7p core domain genetically fused to a domain that <br />                                                                             exhibits signi&#174;cant homology to structural nucleosome com- <br />                                                                             paction factors&#208;the linker histones. In Tetrahymena, the <br />                                                                             micronuclear linker histone gene (MLH) is translated into a <br />                                                                             single 71 kDa polypeptide which is then cleaved to produce <br />                                                                             four distinct linker histones: a and b in which a is further <br />                                                                             processed to produce d and g linker histone proteins. These <br />                                                                             four linker histone proteins are distinct from one another and <br />                                                                             unique from macronuclear histone H1 protein (43). In humans, <br />                                                                             EFO1p is comprised of a C-terminal Ctf7p domain fused to an <br />                                                                             N-terminal domain comprised of contiguous g and b linker <br />                                                                             histone proteins in which the predominant form appears <br />                                                                             uncleaved (Fig. 6). This genetic fusion is conserved through <br />                                                                             evolution in that a similar linker histone protein&#177;Ctf7p <br />                                                                             chimera exists in Drosophila (this study). At present, efforts <br /> Figure 6. Expression and acetyltransferase activity of human EFO1p and      to complement yeast ctf7 mutant phenotypes with hEFO1p <br /> budding yeast Ctf7p. (A) GST&#177;EFO1p and GST&#177;Ctf7p expressed in               have been unsuccessful (A.Bellows, M.Kenna and <br /> bacterial cells and visualized using an antibody directed against GST (a <br /> GST). Acetyltransferase activity of human EFO1p and budding yeast Ctf7p <br />                                                                             R.V.Skibbens, personal observation), although even close <br /> using an antibody directed against acetylated lysines (a Ac-K) previously   orthologs often fail to rescue essential functions across model <br /> described to detect Ctf7p acetylation reactions (15).                       organisms. <br />                                                                                Linker histones have been implicated in transcription <br />                                                                             regulation and chromosome compaction (47). For instance, <br /> levels, or that they are not expressed. Thus, while our analyses            in Tetrahymena MLH knockouts, loss of a, b, d and g linker <br /> of hEFO2&#177;hEFO4 greatly extend the limited sequences                         histone proteins resulted in micronuclei that were over twice <br /> previously described (15,16), the full-length conceptual                    the size of wild-type cell micronuclei (44). Similarly, in <br /> translations and expression patterns remain unknown. Given                  budding yeast, mutations in many cohesion factors, including <br /> the essential function of Ctf7p in both budding and &#174;ssion                  Mcd1p, Pds5p and Ctf7p, also exhibit defects in chromosome <br /> yeast and the apparent frequency through which CTF7                         condensation (4,9,16) (R.V.Skibbens and D.Koshland, unpub- <br /> recombines through evolution, it would not be surprising                    lished results). It is worth speculating that linker histone <br /> that multiple editions exist within the human genome.                       modi&#174;cations may help target the acetyltransferase activity of <br />    hEFO1p exhibits several characteristics consistent with a                hEFO1p to chromatin in a cell cycle-dependent manner. <br /> role in chromosome segregation and cohesion establishment.                  Previous studies revealed that Tetrahymena linker histones <br /> First, hEFO1p contains the highly conserved Ctf7p core                      contain cAMP-dependent protein kinase (PKA) target sequen- <br /> domain found in budding yeast Ctf7p and &#174;ssion yeast Eso1p                  ces, with evidence that at least d linker histone protein is a <br /> (this study) (7,16,29). Secondly, hEFO1p localizes speci&#174;c-                 PKA substrate. Thus, the phosphorylation state of d linker <br /> ally to the nucleus. We were unable to identify mitotic cells in            histone may regulate chromosome condensation/decondensa- <br /> which hEFO1p associated with condensed chromosomes                          tion (48). Future directions will relate to the role of <br /> (M.Kenna and R.V.Skibbens, personal observation).                           post-translational hEFO1p modi&#174;cations in chromosome <br /> Similarly, budding yeast Ctf7p is a nuclear protein that                    condensation and phenotypic analyses of cells lacking <br /> functions during S phase. Ctf7p is not required during mitosis,             hEFO1p. These endeavors may be dif&#174;cult given the possible <br /> nor is Ctf7p detectable within the bulk of the chromatin mass               functional redundancy of the hEFO family and the recent <br /> (7,16) (R.V.Skibbens and D.Koshland, unpublished results).                  observations that a separate class of proteins, called the <br /> Instead, Ctf7p is thought to transiently associate with the DNA             condensins, appears to perform the majority of chromosome <br /> replication fork during cohesion establishment (18). In                     condensation in humans (49,50). Similarly, there is an ever- <br /> combination, these &#174;ndings indicate that the cell cycle activity            growing number of cohesion proteins that, in addition to the <br /> and chromatin association of Ctf7p-like factors are probably                structural and deposition cohesion factors, include RFC <br /> conserved through evolution. Thirdly, hEFO1p exhibits                       factors, DNA polymerases and an acetyltransferase (15,18&#177; <br /> acetyltransferase activity and is capable of acting as its own              21,23&#177;25). Deciphering the extent to which these players <br /> substrate. A comparable activity has been reported for                      promote sister chromatid cohesion and chromosome con- <br /> budding yeast Ctf7p (15). Currently, physiologically relevant               densation will continue to be an intriguing endeavor for cell <br /> substrates of budding yeast Ctf7p acetylation have yet to be                biology. <br /> identi&#174;ed (15); thus, characterization of hEFO1p acetylation <br /> targets in human cells is beyond the scope of this report. <br />                                                                              <br /> However, the &#174;nding that acetyltransferase activities are <br /> conserved from yeast to humans strongly suggests that this               <br /> enzymatic activity will play a central role in establishing               <br /> cohesion. Future endeavors in solving the mystery of sister               <br /> chromatid pairing for both yeast and humans will probably                 <br /> </body></html>